<?xml version="1.0" encoding="UTF-8"?>
<div class="fig" id="PMC7788728/figure_2" xmlns="http://www.w3.org/1999/xhtml">
 <caption> Evolutionary history inferred for different genomic segments of S. invicta virus 14 (SINV-14). Midpoint-rooted Bayesian phylogenetic trees based on A full-length amino-acid sequences of RNA-dependent RNA polymerase (RdRp), B amino-acid sequence of conserved glycoprotein domain, C full-length amino-acid sequence of movement protein and D full-length amino-acid sequence of nucleocapsid protein. Bayesian posterior probabilities are shown at the nodes. Nodes with posterior probability values between &gt; 0.50 and ≤ 85 are indicated by empty circles, &gt; 0.85 and &lt; 0.95 are indicated by half-filled circles and those with values ≥ 0.95 are indicated by filled circles. The scale bar at the bottom indicates the number of amino acid substitutions per site. Branches highlighted in red indicate virus characterized in this study. Genera are indicated at the right side. Proposed species unaccepted yet by International Committee on Taxonomy of Viruses (ICTV) are followed by an asterisk. Proteins sequences encoded by cognates components for a given virus that do not have conserved domain according to Conserved Domain Database (CDD) were not included </caption>
 <div class="graphic"/>
 <p class="label">Fig. 2</p>
</div>
