<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Enzyme Inhib Med Chem</span><span class="iso-abbrev" title="iso-abbrev">J Enzyme Inhib Med Chem</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Enzyme Inhibition and Medicinal Chemistry</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1475-6366</span><span class="issn-epub" tagx="issn" title="issn-epub">1475-6374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7751408">7751408</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/33327795">33327795</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/14756366.2020.1861607">10.1080/14756366.2020.1861607</a></span><span class="publisher-id" title="publisher-id">1861607</span><div class="unknown" title="article-version" tag="article-version">
Version of Record</div> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Research Paper</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Production, purification and biochemical characterisation of a novel lipase from a newly identified lipolytic bacterium <i>Staphylococcus caprae</i> NCU S6</div> <div class="alt-title" title="alt-title">
J. Zhao et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Junxin</span></span><a href="#AF0001">a</a><a href="#AF0002">b</a><a href="#AF0003">c</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">Maomao</span></span><a href="#AF0001">a</a><a href="#AF0002">b</a><a href="#AF0003">c</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">Zheling</span></span><a href="#AF0001">a</a><a href="#AF0002">b</a><a href="#AF0004">d</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Ping</span></span><a href="#AF0001">a</a><a href="#AF0002">b</a><a href="#AF0004">d</a><a href="#AN0001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">Deming</span></span><a href="#AF0001">a</a><a href="#AF0002">b</a><a href="#AF0005">e</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">Shuguang</span></span><a href="#AF0002">b</a><a href="#AF0004">d</a><a href="#AF0006">f</a></span><span class="citation_author_institution" id="AF0001">[a], <span class="institution" title="institution">State Key Laboratory of Food Science and Technology, Nanchang University</span><span class="city" tagx="city" title="city">Nanchang</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="AF0002">[b], <span class="institution" title="institution">Jiangxi Province Key Laboratory of Edible and Medicinal Resources Exploitation, Nanchang University</span><span class="city" tagx="city" title="city">Nanchang</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="AF0003">[c], <span class="institution" title="institution">School of Food Science and Technology, Nanchang University</span><span class="city" tagx="city" title="city">Nanchang</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="AF0004">[d], <span class="institution" title="institution">School of Resource and Environmental and Chemical Engineering, Nanchang University</span><span class="city" tagx="city" title="city">Nanchang</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="AF0005">[e], <span class="institution" title="institution">New Zealand Institute of Natural Medicine Research</span><span class="city" tagx="city" title="city">Auckland</span><span class="country" tagx="country" title="country">New Zealand</span></span><span class="citation_author_institution" id="AF0006">[f], <span class="institution" title="institution">School for Engineering of Matter, Transport and Energy, Arizona State University</span><span class="city" tagx="city" title="city">Tempe</span><span class="state" tagx="state" title="state">AZ</span><span class="country" tagx="country" title="country">USA</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
CONTACT Ping Yu <span class="email" tagx="email" title="email">cpu_yuping@126.com</span><span class="institution" title="institution">State Key Laboratory of Food Science and Technology, Nanchang University</span>, <span class="addr-line" title="addr-line">235 Nanjing East Road</span>, <span class="state" tagx="state" title="state">Nanchang</span><span class="postal-code" tagx="postal-code" title="postal-code">330047</span>, <span class="country" tagx="country" title="country">China</span>; <span class="institution" title="institution">Jiangxi Province Key Laboratory of Edible and Medicinal Resources Exploitation, Nanchang University</span>, <span class="city" tagx="city" title="city">Nanchang</span><span class="postal-code" tagx="postal-code" title="postal-code">330031</span>, <span class="country" tagx="country" title="country">China</span>; <span class="institution" title="institution">School of Resource and Environmental and Chemical Engineering, Nanchang University</span>, <span class="city" tagx="city" title="city">Nanchang</span><span class="postal-code" tagx="postal-code" title="postal-code">330031</span>, <span class="country" tagx="country" title="country">China</span></div> </div><span class="pub-date-" title="pub-date-">: <span>2020-12-12</span></span><span class="pub-date-" title="pub-date-">: <span>2021</span></span><span class="volume" tagx="volume" title="volume">36</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">248</span><span class="lpage" tagx="lpage" title="lpage">256</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">The Author(s)</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="IENZ_36_1861607.pdf" title="self-uri">IENZ_36_1861607.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>A novel lipase, SCNL, was isolated from <i>Staphylococcus caprae</i> NCU S6 strain in the study. The lipase was purified to homogeneity with a yield of 6.13% and specific activity of 502.76â€‰U/mg, and its molecular weight was determined to be approximately 87â€‰kDa. SCNL maintained above 80% of its initial activity at a wide range of temperatures (20â€"50â€‰Â°C) and pH values (6â€"11), with an optimal temperature at 40â€‰Â°C and optimal pH at 9.0 with <i>p</i>-nitrophenyl palmitate as a substrate. SCNL exhibited a higher residual activity than the other staphylococcal lipases in the presence of common enzyme inhibitors and commercial detergents. The lipase activity was enhanced by organic solvents (isooctane, glycerol, DMSO and methanol) and metal ions (Na<sup>+</sup>, Ba<sup>2+</sup>, Ca<sup>2+</sup>, and Mn<sup>2+</sup>). The <i>K</i>m and <i>V</i>max values of SCNL were 0.695â€‰mM and 262.66â€‰s<sup>âˆ’1â€‰</sup>mM<sup>âˆ’1</sup>, respectively. The enzyme showed a preference for <i>p</i>-NP stearate, tributyrin and canola oil. These biochemical features of SCNL suggested that it may be an excellent novel lipase candidate for industrial and biotechnological applications.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd"><i>Staphylococcus caprae</i></div> <div class="kwd" title="kwd">
lipase</div> <div class="kwd" title="kwd">
purification</div> <div class="kwd" title="kwd">
biochemical characterisation</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1.</span><div class="title" tagx="title" title="title">
Introduction</div> <p>Lipolytic enzymes are composed of esterases (EC 3.1.1.1) and lipases (triacylglycerol hydrolases, EC 3.1.1.3), and the latter represent the most important enzymes that act on ester bonds between a carboxylic acid and an alcohol group at the lipid-water interface<a href="#CIT0001"><sup>1</sup></a>. Therefore, they are widely applied to various areas of industries, including foods, pharmaceuticals, biodiesel, detergents, cosmetics and chemicals<a href="#CIT0002"><sup>2</sup></a>. Lipases may come from a variety of sources, and are ubiquitous enzymes produced by some biological systems, such as plants, animals and microorganisms<a href="#CIT0003"><sup>3</sup></a>. Furthermore, microbial lipases are a kind of crucial enzymes in biotransformation due to its multifunction in applications and convenience for mass production, and are superior to the lipases from plants and animals in terms of activity, stability, ease of purification, molecular modifications, and continuous production independent of season<a href="#CIT0004"><sup>4</sup></a>. The special properties of these enzymes were given by the environments where microorganisms were isolated occasionally<a href="#CIT0005"><sup>5</sup></a>.</p> <p>Lipolytic bacteria are classified into different families based on their gene sequences and biochemical properties, and a high level of lipase is also a general feature of staphylococcus. The lipolytic enzymes in staphylococcus that have been studied for their applications in biotechnology<a href="#CIT0002"><sup>2</sup></a> are mainly extracellular lipases and expressed as prepropeptides, and several staphylococcal lipases were purified and biochemically characterised, with the molecular weight of approximately 70â€‰kDa<a href="#CIT0006"><sup>6</sup></a>. In recent years, our major research direction focussed on lipases from staphylococcal species, and to our knowledge no <i>Staphylococcus caprae</i> lipases have been reported, and thus purification and characterisation of these enzymes are necessary.</p> <p>In this study, we screened and identified a new lipolytic bacterium <i>Staphylococcus caprae</i> NCU S6 from sewerage, evaluated the growth characteristics and optimised the culture conditions of the strain for a maximum production of a novel lipase. The obtained lipase was purified and biochemically characterised.</p> </div> <div class="materialsandmethods" title="sec"><span class="label" tagx="label" title="label">2.</span><div class="title" tagx="title" title="title">
Materials and methods</div> <div class="materials" title="sec"><span class="label" tagx="label" title="label">2.1.</span><div class="title" tagx="title" title="title">
Materials</div> <p>All chemicals were purchased from Aladdin Chemical Co. (Shanghai, China) and Sigmaâ€"Aldrich (St. Louis, MO, USA). Sephadex G-50 and DEAE-Sepharose Fast Flow column were provided by Beijing Solarbio Science &amp;amp; Technology Co. <i>Staphylococcus epidermidis</i> ATCC 12228 and <i>Staphylococcus aureus</i> ATCC 25923 were obtained from Guangdong Huankai Microbial Science &amp;amp; Technology Co. (Guangzhou, China). All chemicals used in this study were of analytical grade.</p> </div> <div class="isolationandmolecularidentificationoflipase-producingstrains" title="sec"><span class="label" tagx="label" title="label">2.2.</span><div class="title" tagx="title" title="title">
Isolation and molecular identification of lipase-producing strains</div> <p>The samples from oil contaminated sewerages at Nanchang University were enriched in the 25â€‰ml lysogeny broth (LB) medium, pH 7.0 with shaking at 150â€‰rpm. Initial lipolytic bacteria were screened by using bromocresol purple (0.2%, <i>w/v</i>) and tributyrin (1.0%, <i>v/v</i>) agar plates<a href="#CIT0007"><sup>7</sup></a>. Positive isolates were rescreened by using Rhodamine B (0.5%, <i>w/v</i>) agar plate for lipase production<a href="#CIT0008"><sup>8</sup></a>.</p> <p>The bacteria were characterised by morphological, biochemical and molecular techniques. The biochemical tests, including coagulase test, novobiocin (5â€‰Î¼g/piece) sensitivity test and catalase production test, were carried out to discriminate staphylococci<a href="#CIT0009"><sup>9</sup></a>, and <i>Staphylococcus aureus</i> ATCC 25923 and <i>Staphylococcus epidermidis</i> ATCC 12228 were used as standard strains. The isolate was further identified by the 16S rRNA gene sequencing method. Then the genomic DNA of NCU (Nanchang University) S6 was obtained from colonies by protease K cleavage method and amplified as described previously<a href="#CIT0010"><sup>10</sup></a>. Evolutionary relationship was determined by MEGA version 7.0.</p> </div> <div class="cultureconditions" title="sec"><span class="label" tagx="label" title="label">2.3.</span><div class="title" tagx="title" title="title">
Culture conditions</div> <p>The <i>S. caprae</i> strain was cultured overnight at 37â€‰Â°C and 200â€‰rpm in 250â€‰ml Erlenmeyer flasks with 50â€‰ml medium A containing 1% (<i>w/v</i>) glucose, 0.5% yeast extract, 1% tryptone and 0.5% NaCl, pH 7.0. After preincubation, 4% (<i>v/v</i>) medium A was inoculated into 50â€‰ml medium B (fermentation medium, pH 7.0): 0.5% (<i>w/v</i>) glucose, 2.5% tryptic soy broth, 0.25% K<sub>2</sub>HPO<sub>4</sub>, 1% NaCl, 2% (<i>v/v)</i> olive oil and 1% Tween-80. Then 51â€‰h-incubation was carried out in an orbital shaker at 37â€‰Â°C and 200â€‰rpm.</p> </div> <div class="measurementoflipaseactivity" title="sec"><span class="label" tagx="label" title="label">2.4.</span><div class="title" tagx="title" title="title">
Measurement of lipase activity</div> <p>The lipase hydrolysis activity towards <i>p</i>-nitrophenyl palmitate (NPP) was measured according to the colorimetric method<a href="#CIT0011"><sup>11</sup></a>, with some modifications. Briefly, a reaction mixture contained 100â€‰Î¼L <i>p</i>-NPP (7.5â€‰mM, methanol solution) and 2.1â€‰ml buffer A (50â€‰mM Tris-HCl, pH 8.0). After heating of the samples in a water bath for 10â€‰min at 40â€‰Â°C, 100â€‰Î¼L appropriately diluted enzyme was added. The reaction mixture was then incubated at 40â€‰Â°C for 20â€‰min with a control group using thermally inactivated lipase to consider any spontaneous hydrolysis of <i>p</i>-NPP. The reaction was terminated by adding 100â€‰Î¼L ZnSO<sub>4</sub> (100â€‰mM, sterile water) on ice bath. Reaction solution was centrifuged at 8,000â€‰Ã—â€‰g for 5â€‰min to remove the insoluble substances, and the supernatant was used to determine the amount of liberated <i>p</i>-nitrophenol (NP) by its absorbance at 410â€‰nm. The lipolytic activities were expressed as international units. One unit of enzyme activity corresponded to one micromole of <i>p</i>-NP released from <i>p</i>-NPP per minute, and the specific activities were expressed as U/mg of protein.</p> </div> <div class="measurementofproteinconcentration" title="sec"><span class="label" tagx="label" title="label">2.5.</span><div class="title" tagx="title" title="title">
Measurement of protein concentration</div> <p>Protein concentration was determined according to Bradford<a href="#CIT0012"><sup>12</sup></a> by using bovine serum albumin (BSA) as a standard. The supernatant obtained from submerged culture medium after centrifugation at 4000â€‰Ã—â€‰g for 30â€‰min was used as crude extract for subsequent purification steps.</p> </div> <div class="evaluationofgrowthcharacteristics" title="sec"><span class="label" tagx="label" title="label">2.6.</span><div class="title" tagx="title" title="title">
Evaluation of growth characteristics</div> <p>After culturing the purified single strain in medium A (50â€‰ml) overnight, the inoculum at 8% (<i>v/v</i>) was transferred into medium B (50â€‰ml). The 5â€‰days-incubation was carried out in an orbital shaker at 37â€‰Â°C and 200â€‰rpm. Then, the absorption value of fermentation medium at 600â€‰nm was measured every 3â€‰h by using TU-1950 Double-Beam UV-Vis spectrophotometer (Beijing Purkinje General Instrument Co., Beijing, China). The growth characteristics of the strain were recorded as time-OD value curve. Meanwhile, the fermentation medium was taken out every 3â€‰h and centrifuged at 8000â€‰Ã—â€‰g for 15â€‰min to get the supernatant for lipase activity measurement.</p> </div> <div class="lipaseproductionandpurification" title="sec"><span class="label" tagx="label" title="label">2.7.</span><div class="title" tagx="title" title="title">
Lipase production and purification</div> <p>The <i>S. caprae</i> NCU S6 was cultivated under optimal reaction conditions for 51â€‰h at 37â€‰Â°C and 200â€‰rpm. The submerged culture medium was centrifuged at 4000â€‰Ã—â€‰g for 30â€‰min at 4â€‰Â°C to discard the cells. The clear supernatant containing extracellular lipase was used as crude lipase for subsequent purification. The lipase produced by <i>S. caprae</i> was purified by three steps and the purified lipase named as SCNL. The lipase activity and protein concentration were measured after each step.</p> <div class="ammoniumsulphateprecipitation(asp),ultrafiltrationandlyophilisation" title="sec"><span class="label" tagx="label" title="label">2.7.1.</span><div class="title" tagx="title" title="title">
Ammonium sulphate precipitation (ASP), ultrafiltration and lyophilisation</div> <p>Gradient concentration of ammonium sulphate (10â€‰âˆ’â€‰80%) was added to cell-free crude extract under magnetic stirrer at 4â€‰Â°C for overnight stratification. The precipitate at 80% saturation ammonium sulphate solution was obtained by centrifugation at 8000â€‰Ã—â€‰g for 30â€‰min and resuspended in Buffer A. The crude lipase solution was centrifuged at 8000â€‰Ã—â€‰g for 5â€‰min to remove the insoluble materials and then concentrated by ultrafiltration with Amicon-Ultra-15 (MWCO10kD, Millipore, Bedford, MO, USA). The concentrated lipase was lyophilised by a freeze drier after dialysing for follow-up experiments.</p> </div> <div class="gelfiltrationchromatography(gfc)" title="sec"><span class="label" tagx="label" title="label">2.7.2.</span><div class="title" tagx="title" title="title">
Gel filtration chromatography (GFC)</div> <p>The freeze-dried powder of enzyme solution (10â€‰ml, Buffer A) was loaded onto a Sephadex G-50 column (16â€‰Ã—â€‰300â€‰mm) previously equilibrated with Buffer A at a flow rate of 50â€‰ml/h for 1â€‰h. The fractions showing the highest lipase activity were collected.</p> </div> <div class="ion-exchangechromatography(iec)" title="sec"><span class="label" tagx="label" title="label">2.7.3.</span><div class="title" tagx="title" title="title">
Ion-exchange chromatography (IEC)</div> <p>The enzymatically active fractions (40â€‰ml) eluted from the Sephadex G-50 column were loaded onto a DEAE-Sepharose Fast Flow column (16â€‰Ã—â€‰300â€‰mm) preequilibrated with Buffer A. The lipase was absorbed by the column, and the impurities were washed away by Buffer A at a flow rate of 50â€‰ml/h until no lipase activity was detected in the washed fractions. Absorbed proteins were subjected to gradient elution, followed by a series of NaCl solutions (100â€‰ml of 0.02â€"1.0 M in sterile water) at a flow rate of 20â€‰ml/h. The fractions with the highest lipase activities were concentrated by ultrafiltration with Amicon-Ultra-15 (MWCO10kD, Millipore, Bedford, MO, USA), and then freeze-dried for subsequent analyses.</p> </div> </div> <div class="gelelectrophoresis" title="sec"><span class="label" tagx="label" title="label">2.8.</span><div class="title" tagx="title" title="title">
Gel electrophoresis</div> <p>Analytical polyacrylamide gel electrophoresis of SCNL was carried out in sodium dodecyl sulphate (SDS-PAGE) following a previous method<a href="#CIT0013"><sup>13</sup></a>.</p> </div> <div class="biochemicalcharacterisation" title="sec"><span class="label" tagx="label" title="label">2.9.</span><div class="title" tagx="title" title="title">
Biochemical characterisation</div> <div class="effectoftemperatureandphonlipaseactivityandstability" title="sec"><span class="label" tagx="label" title="label">2.9.1.</span><div class="title" tagx="title" title="title">
Effect of temperature and pH on lipase activity and stability</div> <p>To determine the optimal reaction temperature of SCNL, SCNL was incubated in Buffer A at the temperatures from 10 to 60â€‰Â°C, with a constant pH value at 7.0. The relative lipase activity and specific activity were determined under standard assay conditions. The highest lipase activity was taken as 100%. For thermostability determination, the purified lipase was pre-incubated in Buffer A at the same temperature range for 240â€‰min. The mixture was sampled every 20â€‰min and remaining lipase activity was determined as described previously. The initial lipase activity was taken as 100%.</p> <p>The optimal pH value for the lipase activity was also investigated by assaying the activity in various buffers at different pH values (3.0â€"12.0) for 30â€‰min at 40â€‰Â°C. Sodium acetate (50â€‰mM, pH 3.0â€"5.0), sodium phosphate monobasic (50â€‰mM, pH 6.0â€"7.0), Tris-HCl buffer (50â€‰mM, 8.0â€"9.0) and glycine/NaOH buffer (50â€‰mM, 10.0â€"12.0) were used as buffers. For pH stability test, the purified lipase was pre-incubated in different buffers with pH 3.0 to 12.0 for 7.0â€‰h. Then, the remaining activity was determined every hour as described previously.</p> </div> <div class="effectsoforganicsolventsonthestabilityofthelipase" title="sec"><span class="label" tagx="label" title="label">2.9.2.</span><div class="title" tagx="title" title="title">
Effects of organic solvents on the stability of the lipase</div> <p>The effect of organic solvents on the stability of the lipase was investigated by mixing different organic solvents (<a href="#t0001">Table 1</a>) with the enzyme in Buffer A at 10, 30, or 50% (<i>v/v</i>) for 1 and 28â€‰days. The mixture was incubated at 37â€‰Â°C and 200â€‰rpm, and the control group (without organic solvents) was also prepared in Buffer A. The residual activity of the lipase was measured under standard conditions, samples were withdrawn periodically and the final concentrations of organic solvents were less than 3% (<i>v/v</i>).</p> <div class="table-wrap_UNKNOWN" id="t0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>Effect of organic solvents on lipase activity.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup>  <thead> <tr> <th align="left">Solvents</th> <th align="center">Concentration (%, <i>v/v</i>)</th> <th align="center">Residual activity (%) 1â€‰day</th> <th align="center">28Â days</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Control</td> <td align="char" char=".">0</td> <td align="char" char="Â±">100.0â€‰Â±â€‰1.1</td> <td align="char" char="Â±">100.0â€‰Â±â€‰1.3</td> </tr> <tr> <td align="left">Acetone</td> <td align="char" char=".">10</td> <td align="char" char="Â±">96.9â€‰Â±â€‰3.3</td> <td align="char" char="Â±">89.6â€‰Â±â€‰2.1</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">30</td> <td align="char" char="Â±">85.4â€‰Â±â€‰1.1</td> <td align="char" char="Â±">77.9â€‰Â±â€‰1.4</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">59.2â€‰Â±â€‰2.5</td> <td align="char" char="Â±">50.3â€‰Â±â€‰1.0</td> </tr> <tr> <td align="left">Ethyl acetate</td> <td align="char" char=".">10</td> <td align="char" char="Â±">105.8â€‰Â±â€‰1.9</td> <td align="char" char="Â±">80.4â€‰Â±â€‰2.3</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">30</td> <td align="char" char="Â±">95.1â€‰Â±â€‰3.4</td> <td align="char" char="Â±">71.9â€‰Â±â€‰1.7</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">76.4â€‰Â±â€‰3.7</td> <td align="char" char="Â±">64.5â€‰Â±â€‰0.5</td> </tr> <tr> <td align="left">Isooctane</td> <td align="char" char=".">10</td> <td align="char" char="Â±">106.6â€‰Â±â€‰1.2</td> <td align="char" char="Â±">110.6â€‰Â±â€‰0.9</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">30</td> <td align="char" char="Â±">109.0â€‰Â±â€‰2.2</td> <td align="char" char="Â±">119.3â€‰Â±â€‰0.2</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">123.7â€‰Â±â€‰3.2</td> <td align="char" char="Â±">124.9â€‰Â±â€‰1.5</td> </tr> <tr> <td align="left">Diethyl ether</td> <td align="char" char=".">10</td> <td align="char" char="Â±">105.0â€‰Â±â€‰0.9</td> <td align="char" char="Â±">93.7â€‰Â±â€‰1.7</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">30</td> <td align="char" char="Â±">90.8â€‰Â±â€‰1.0</td> <td align="char" char="Â±">84.6â€‰Â±â€‰3.2</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">68.4â€‰Â±â€‰2.5</td> <td align="char" char="Â±">75.9â€‰Â±â€‰1.2</td> </tr> <tr> <td align="left">Glycerol</td> <td align="char" char=".">10</td> <td align="char" char="Â±">105.9â€‰Â±â€‰0.8</td> <td align="char" char="Â±">114.9â€‰Â±â€‰1.1</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">30</td> <td align="char" char="Â±">115.0â€‰Â±â€‰2.8</td> <td align="char" char="Â±">120.6â€‰Â±â€‰0.2</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">133.0â€‰Â±â€‰1.4</td> <td align="char" char="Â±">136.4â€‰Â±â€‰1.3</td> </tr> <tr> <td align="left">Ethylene glycol</td> <td align="char" char=".">10</td> <td align="char" char="Â±">105.0â€‰Â±â€‰2.1</td> <td align="char" char="Â±">94.3â€‰Â±â€‰0.6</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">30</td> <td align="char" char="Â±">81.8â€‰Â±â€‰1.8</td> <td align="char" char="Â±">75.1â€‰Â±â€‰1.1</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">58.2â€‰Â±â€‰0.2</td> <td align="char" char="Â±">60.3â€‰Â±â€‰1.4</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">10</td> <td align="char" char="Â±">106.6â€‰Â±â€‰4.7</td> <td align="char" char="Â±">100.7â€‰Â±â€‰1.6</td> </tr> <tr> <td align="left">Petroleum ether</td> <td align="char" char=".">30</td> <td align="char" char="Â±">98.3â€‰Â±â€‰1.7</td> <td align="char" char="Â±">89.1â€‰Â±â€‰0.8</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">81.4â€‰Â±â€‰0.7</td> <td align="char" char="Â±">73.6â€‰Â±â€‰0.5</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">10</td> <td align="char" char="Â±">105.8â€‰Â±â€‰1.0</td> <td align="char" char="Â±">86.2â€‰Â±â€‰1.1</td> </tr> <tr> <td align="left">Acetonitrile</td> <td align="char" char=".">30</td> <td align="char" char="Â±">85.1â€‰Â±â€‰3.4</td> <td align="char" char="Â±">72.3â€‰Â±â€‰1.2</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">76.4â€‰Â±â€‰0.5</td> <td align="char" char="Â±">60.1â€‰Â±â€‰1.5</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">10</td> <td align="char" char="Â±">113.7â€‰Â±â€‰1.6</td> <td align="char" char="Â±">105.6â€‰Â±â€‰0.2</td> </tr> <tr> <td align="left">N-hexane</td> <td align="char" char=".">30</td> <td align="char" char="Â±">98.3â€‰Â±â€‰1.3</td> <td align="char" char="Â±">90.5â€‰Â±â€‰1.7</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">78.2â€‰Â±â€‰0.9</td> <td align="char" char="Â±">69.5â€‰Â±â€‰3.1</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">10</td> <td align="char" char="Â±">120.4â€‰Â±â€‰1.4</td> <td align="char" char="Â±">109.5â€‰Â±â€‰1.0</td> </tr> <tr> <td align="left">N-heptane</td> <td align="char" char=".">30</td> <td align="char" char="Â±">105.7â€‰Â±â€‰0.9</td> <td align="char" char="Â±">95.2â€‰Â±â€‰2.5</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">98.2â€‰Â±â€‰0.4</td> <td align="char" char="Â±">86.4â€‰Â±â€‰1.8</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">10</td> <td align="char" char="Â±">114.1â€‰Â±â€‰0.7</td> <td align="char" char="Â±">110.1â€‰Â±â€‰0.9</td> </tr> <tr> <td align="left">Ethanol</td> <td align="char" char=".">30</td> <td align="char" char="Â±">94.7â€‰Â±â€‰2.3</td> <td align="char" char="Â±">91.2â€‰Â±â€‰1.3</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">84.8â€‰Â±â€‰1.9</td> <td align="char" char="Â±">79.6â€‰Â±â€‰2.1</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">10</td> <td align="char" char="Â±">106.3â€‰Â±â€‰1.7</td> <td align="char" char="Â±">108.3â€‰Â±â€‰0.6</td> </tr> <tr> <td align="left">DMSO</td> <td align="char" char=".">30</td> <td align="char" char="Â±">100.1â€‰Â±â€‰1.5</td> <td align="char" char="Â±">97.7â€‰Â±â€‰1.4</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">90.2â€‰Â±â€‰3.3</td> <td align="char" char="Â±">91.2â€‰Â±â€‰2.2</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">10</td> <td align="char" char="Â±">116.4â€‰Â±â€‰1.2</td> <td align="char" char="Â±">108.3â€‰Â±â€‰2.8</td> </tr> <tr> <td align="left">Methanol</td> <td align="char" char=".">30</td> <td align="char" char="Â±">95.6â€‰Â±â€‰1.8</td> <td align="char" char="Â±">98.1â€‰Â±â€‰3.5</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">71.9â€‰Â±â€‰4.5</td> <td align="char" char="Â±">80.4â€‰Â±â€‰1.9</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">10</td> <td align="char" char="Â±">104.1â€‰Â±â€‰0.7</td> <td align="char" char="Â±">105.7â€‰Â±â€‰1.5</td> </tr> <tr> <td align="left">Isopropanol</td> <td align="char" char=".">30</td> <td align="char" char="Â±">94.7â€‰Â±â€‰2.3</td> <td align="char" char="Â±">90.1â€‰Â±â€‰1.2</td> </tr> <tr> <td align="left">Â </td> <td align="char" char=".">50</td> <td align="char" char="Â±">69.2â€‰Â±â€‰1.1</td> <td align="char" char="Â±">76.4â€‰Â±â€‰0.9</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Enzyme samples were mixed with organic solvents (10, 30, 50%, <i>v/v</i>) and incubated for 1â€‰day or 28â€‰days in a rotary shaker set at 200â€‰rpm and 37â€‰Â°C prior to determining the residual activity. Results are presented as meansâ€‰Â±â€‰standard deviation (<i>n</i>â€‰=â€‰3).</p> </div> </div> </div> </div> <div class="effectsofmetalions,inhibitorsanddetergentsonthestabilityofthelipase" title="sec"><span class="label" tagx="label" title="label">2.9.3.</span><div class="title" tagx="title" title="title">
Effects of metal ions, inhibitors and detergents on the stability of the lipase</div> <p>The effects of metal ions, inhibitors and detergents on the stability of the purified lipase activity were investigated. The relative activity was measured by pre-incubating the lipase solution with each additive. All reagents were prepared in Buffer A, with each metal ion at a final concentration of 1 or 10â€‰mM (<a href="#t0002">Table 2</a>). Reaction mixtures containing 0.1 or 1% (<i>v/v</i> or <i>w/v</i>) of popular commercial detergents and inhibitors (<a href="#t0003">Table 3</a>) were also incubated at room temperature for 30â€‰min. The lipase activity without addition of metal ions, inhibitors or detergents was defined as 100%, and the relative activity was determined as compared to the control. Experiments were performed as described above, using <i>p</i>-NPP as a substrate. Each assay was carried out in triplicates.</p> <div class="table-wrap_UNKNOWN" id="t0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> <p>Effects of some metal ions on the lipase activity.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup>  <thead> <tr> <th align="left">Â </th> <th colspan="2" align="center">Relative activity (%) <hr /> </th> </tr> <tr> <th align="left">Â </th> <th align="center">1â€‰mM</th> <th align="center">10â€‰mM</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Control</td> <td align="char" char="Â±">100.0â€‰Â±â€‰1.8</td> <td align="char" char="Â±">100.0â€‰Â±â€‰1.1</td> </tr> <tr> <td align="left">Metal ions</td> <td align="char" char="Â±">Â </td> <td align="char" char="Â±">Â </td> </tr> <tr> <td align="left">â€ƒAg<sup>+</sup></td> <td align="char" char="Â±">96.2â€‰Â±â€‰1.5</td> <td align="char" char="Â±">101.7â€‰Â±â€‰0.9</td> </tr> <tr> <td align="left">â€ƒNa<sup>+</sup></td> <td align="char" char="Â±">100.7â€‰Â±â€‰2.6</td> <td align="char" char="Â±">120.8â€‰Â±â€‰1.1</td> </tr> <tr> <td align="left">â€ƒK<sup>+</sup></td> <td align="char" char="Â±">95.8â€‰Â±â€‰3.3</td> <td align="char" char="Â±">102.3â€‰Â±â€‰1.2</td> </tr> <tr> <td align="left">â€ƒSn<sup>2+</sup></td> <td align="char" char="Â±">13.4â€‰Â±â€‰0.8</td> <td align="char" char="Â±">12.1â€‰Â±â€‰0.5</td> </tr> <tr> <td align="left">â€ƒBa<sup>2+</sup></td> <td align="char" char="Â±">102.6â€‰Â±â€‰2.5</td> <td align="char" char="Â±">111.3â€‰Â±â€‰0.6</td> </tr> <tr> <td align="left">â€ƒZn<sup>2+</sup></td> <td align="char" char="Â±">85.3â€‰Â±â€‰2.1</td> <td align="char" char="Â±">53.9â€‰Â±â€‰1.4</td> </tr> <tr> <td align="left">â€ƒCa<sup>2+</sup></td> <td align="char" char="Â±">125.8â€‰Â±â€‰1.1</td> <td align="char" char="Â±">166.4â€‰Â±â€‰1.9</td> </tr> <tr> <td align="left">â€ƒCo<sup>2+</sup></td> <td align="char" char="Â±">101.9â€‰Â±â€‰0.3</td> <td align="char" char="Â±">72.3â€‰Â±â€‰1.3</td> </tr> <tr> <td align="left">â€ƒCu<sup>2+</sup></td> <td align="char" char="Â±">56.8â€‰Â±â€‰4.5</td> <td align="char" char="Â±">24.3â€‰Â±â€‰2.0</td> </tr> <tr> <td align="left">â€ƒFe<sup>2+</sup></td> <td align="char" char="Â±">42.0â€‰Â±â€‰3.1</td> <td align="char" char="Â±">25.6â€‰Â±â€‰1.4</td> </tr> <tr> <td align="left">â€ƒFe<sup>3+</sup></td> <td align="char" char="Â±">18.5â€‰Â±â€‰2.0</td> <td align="char" char="Â±">22.9â€‰Â±â€‰1.0</td> </tr> <tr> <td align="left">â€ƒMg<sup>2+</sup></td> <td align="char" char="Â±">89.3â€‰Â±â€‰2.4</td> <td align="char" char="Â±">83.6â€‰Â±â€‰0.1</td> </tr> <tr> <td align="left">â€ƒMn<sup>2+</sup></td> <td align="char" char="Â±">102.3â€‰Â±â€‰3.9</td> <td align="char" char="Â±">112.5â€‰Â±â€‰2.5</td> </tr> <tr> <td align="left">â€ƒPb<sup>2+</sup></td> <td align="char" char="Â±">45.4â€‰Â±â€‰2.1</td> <td align="char" char="Â±">74.3â€‰Â±â€‰4.8</td> </tr> <tr> <td align="left">â€ƒSb<sup>3+</sup></td> <td align="char" char="Â±">10.1â€‰Â±â€‰0.1</td> <td align="char" char="Â±">7.4â€‰Â±â€‰0.2</td> </tr> <tr> <td align="left">â€ƒAl<sup>3+</sup></td> <td align="char" char="Â±">67.6â€‰Â±â€‰1.5</td> <td align="char" char="Â±">47.3â€‰Â±â€‰1.7</td> </tr> <tr> <td align="left">â€ƒBi<sup>3+</sup></td> <td align="char" char="Â±">10.3â€‰Â±â€‰1.1</td> <td align="char" char="Â±">5.5â€‰Â±â€‰0.8</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Enzyme samples were incubated with various metal ions (1 and 10â€‰mM) for 30â€‰min, and the relative activity was assessed using the standard assay protocol. Results are presented as meansâ€‰Â±â€‰standard deviation (<i>n</i>â€‰=â€‰3).</p> </div> </div> </div> <div class="table-wrap_UNKNOWN" id="t0003" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3.</span> <p>Effects of various inhibitors and detergents on the activity of lipase.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup>  <thead> <tr> <th align="left">Â </th> <th colspan="2" align="center">Relative activity (%) <hr /> </th> <th align="center">Â </th> </tr> <tr> <th align="left">Â </th> <th align="center">1â€‰mM (0.1%, <i>v/v</i>)</th> <th align="center">10â€‰mM (1%, <i>v/v</i>)</th> <td style="">Â </td> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Control</td> <td align="char" char="Â±">100.0â€‰Â±â€‰1.5</td> <td align="char" char="Â±">100.0â€‰Â±â€‰1.0</td> </tr> <tr> <td align="left">Inhibitors</td> <td align="char" char="Â±">Â </td> <td align="char" char="Â±">Â </td> </tr> <tr> <td align="left">â€ƒDTT</td> <td align="char" char="Â±">85.3â€‰Â±â€‰2.1</td> <td align="char" char="Â±">71.5â€‰Â±â€‰1.7</td> </tr> <tr> <td align="left">â€ƒEDTA</td> <td align="char" char="Â±">66.5â€‰Â±â€‰2.4</td> <td align="char" char="Â±">32.9â€‰Â±â€‰3.3</td> </tr> <tr> <td align="left">â€ƒH<sub>2</sub>O<sub>2</sub></td> <td align="char" char="Â±">88.9â€‰Â±â€‰1.1</td> <td align="char" char="Â±">76.8â€‰Â±â€‰1.4</td> </tr> <tr> <td align="left">Detergents</td> <td align="char" char="Â±">Â </td> <td align="char" char="Â±">Â </td> </tr> <tr> <td align="left">â€ƒCTAB</td> <td align="char" char="Â±">107.8â€‰Â±â€‰2.0</td> <td align="char" char="Â±">91.5â€‰Â±â€‰0.6</td> </tr> <tr> <td align="left">â€ƒSDS</td> <td align="char" char="Â±">94.6â€‰Â±â€‰0.2</td> <td align="char" char="Â±">89.2â€‰Â±â€‰3.1</td> </tr> <tr> <td align="left">â€ƒSodium cholate</td> <td align="char" char="Â±">86.6â€‰Â±â€‰2.6</td> <td align="char" char="Â±">80.3â€‰Â±â€‰4.4</td> </tr> <tr> <td align="left">â€ƒTween-20</td> <td align="char" char="Â±">104.3â€‰Â±â€‰1.3</td> <td align="char" char="Â±">94.5â€‰Â±â€‰1.0</td> </tr> <tr> <td align="left">â€ƒTween-60</td> <td align="char" char="Â±">97.6â€‰Â±â€‰2.2</td> <td align="char" char="Â±">92.1â€‰Â±â€‰1.9</td> </tr> <tr> <td align="left">â€ƒTween-80</td> <td align="char" char="Â±">120.3â€‰Â±â€‰0.8</td> <td align="char" char="Â±">105.7â€‰Â±â€‰3.4</td> </tr> <tr> <td align="left">â€ƒTriton X-100</td> <td align="char" char="Â±">104.9â€‰Â±â€‰1.7</td> <td align="char" char="Â±">96.2â€‰Â±â€‰1.9</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Inhibitors and detergents (1 and 10â€‰mM, 0.1 and 1% <i>v/v</i>) were added in the reaction mixture and assayed using standard assay protocol. Results are presented as meansâ€‰Â±â€‰standard deviation (<i>n</i>â€‰=â€‰3).</p> </div> </div> </div> </div> <div class="determinationofreactionkineticsandsubstratespecificity" title="sec"><span class="label" tagx="label" title="label">2.9.4.</span><div class="title" tagx="title" title="title">
Determination of reaction kinetics and substrate specificity</div> <p>The maximum specific activity (<i>V</i><sub>max</sub>) and Michaelis-Menten constant (<i>K</i><sub>m</sub>) in reaction kinetics of SCNL was determined by Lineweaver-Burk plots using Origin software. The specific activity of SCNL was determined with different concentrations of <i>p</i>-NPP (0.1â€"5.0â€‰mM) under optimum conditions.</p> <p>The specificity of substrate with varying chain length and different saturation was investigated by the colorimetric and titrimetric assay. Furthermore, the specific activity and relative activity of SCNL towards the <i>p</i>-NP fatty acid esters (C<sub>2</sub>â€"C<sub>18</sub>), triglycerides (C<sub>2</sub>â€"C<sub>18</sub>) and nature oil were measured under different standard assay conditions according to a previous method<a href="#CIT0014"><sup>14</sup></a>, with minor modifications.</p> </div> </div> <div class="statisticalanalysis" title="sec"><span class="label" tagx="label" title="label">2.10.</span><div class="title" tagx="title" title="title">
Statistical analysis</div> <p>Results were expressed as meanâ€‰Â±â€‰standard deviation (SD). One-way analysis of variance (ANOVA), followed by Tukey's <i>post hoc</i> test, was used to determine statistically significant differences (<i>p</i>â€‰<div class="â€‰0.05)_UNKNOWN"> </div></p></div> </div> <div class="resultsanddiscussion" title="sec"><span class="label" tagx="label" title="label">3.</span><div class="title" tagx="title" title="title">
Results and discussion</div> <div class="isolationandidentificationofthelipolyticstrain" title="sec"><span class="label" tagx="label" title="label">3.1.</span><div class="title" tagx="title" title="title">
Isolation and identification of the lipolytic strain</div> <p>It was reported that the lipases were generally produced in the presence of lipid sources, such as olive oil<a href="#CIT0015"><sup>15</sup></a>. A total of 36 bacterial isolates were identified from the sewerage samples collected at Nanchang City, Jiangxi Province, China. In the preliminary screening, only 22 isolates showed clear halo on the bromocresol purple plates after 24-h cultivation at 37â€‰Â°C, among them 13 isolates also produced a zone of clearance surrounding each colony after 2-day incubation on tributyrin agar plates. However, the tributyrin was not only sensitive to hydrolysis by lipases, but also by esterases. Rescreening of the 13 isolates found that only 6 of them showed obvious effect of lipase production on Rhodamine B agar plate. NCU S6 was found to be the best lipolytic strain that showed the broadest orange fluorescence halos around each colony through UV irradiation.</p> <p>The 16S rDNA V1-V9 region amplified by PCR was sequenced, and subjected to the homology searches using NCBI BLAST analysis. The results revealed a 99% identity with different species belonging to <i>Staphylococcus</i> genus, and the strain was named <i>Staphylococcus caprae</i> NCU S6. The phylogenetic analysis (<a href="#F0001">Figure 1</a>) showed that the strain NCU S6 was the most closely (89%) related to <i>S. caprae</i> strain DSM 20608. The gram staining of NCU S6 showed gram positive. Biochemical approaches, including catalase, coagulase and novobiocin tests, were used to identify the Staphylococcus genus. As shown in <a href="#F0002">Figure 2(A)</a>, all three <i>Staphylococcus</i> strains instantly produced lots of bubbles, indicating catalase positive. <i>S. caprae</i> NCU S6 was found to be coagulase-negative and sensitive to novobiocin (radius of inhibition zone â‰¤10â€‰mm). The reference strains, <i>S. aureus</i> ATCC 25923 (negative control) and <i>S. epidermidis</i> ATCC 12228 (positive control) were coagulase-positive and coagulase-negative respectively, and both exhibited obvious resistance on novobiocin agar plate (radius of inhibition zones â‰¥18â€‰mm). The results have shown that NCU S6 strain belonged to <i>S. caprae</i> species which were not pathogens.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p><b>A p</b>hylogenetic tree displaying the relationship between strain NCU S6 and other <i>Staphylococcus</i> species based on partial 16S rDNA sequences of the strains of the genus. The tree was constructed using a neighbour joining algorithm in MEGA 7.0 software and the accession numbers are shown in parenthesis.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>(A) Results of coagulase tube test, novobiocin (5â€‰Î¼g/piece) sensitivity test, and catalase production test. (B) The result of 16S rDNA agarose gel electrophoresis analysis. (C) The image of Gram-staining result of <i>S. caprae</i> NCU S6.</p>   </div> </div> <div class="growthandlipaseactivitycharacteristicsofncus6" title="sec"><span class="label" tagx="label" title="label">3.2.</span><div class="title" tagx="title" title="title">
Growth and lipase activity characteristics of NCU S6</div> <p>NCU S6 strain was cultivated in growth medium A, and the cell metabolism was strongly linked to lipase activity. With the increase of incubation period, enzyme production was firstly increased and then decreased. As shown in <a href="#F0003">Figure 3</a>, dynamic growth condition and lipase activity of the cells in the fermentation medium was reflected by absorbance value. The logarithmic growth phase of NCU S6 started quickly after culturing in fermentation medium for 3â€‰h and lasted for approximately 21â€‰h. A slow growth from 24 to 39â€‰h was observed, and the strain subsequently entered a stationary phase from 42 to 111â€‰h. <a href="#F0003">Figure 3</a> also showed the dynamic changes of lipase activity, with an increase from the beginning of inoculation to 51â€‰h. The growth rate declined from 51 to 111â€‰h, which may result from the depletion of nutrients in the fermentation medium and a low metabolism level under aging phase of growth<a href="#CIT0016"><sup>16</sup></a>. Obviously, the peak (51â€‰h) of lipase activity (101â€‰U/mL) in the culture period was the optimal enzyme production time.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>The time courses of cell growth (â–²) of <i>S. caprae</i> NCU S6 and lipase production (â– ). The culture was carried out at 37â€‰Â°C and 200â€‰rpm for 111â€‰h and cell growth was monitored by measuring the absorbance at 600â€‰nm every 3â€‰h.</p>   </div> </div> <div class="purificationofs.capraelipase" title="sec"><span class="label" tagx="label" title="label">3.3.</span><div class="title" tagx="title" title="title">
Purification of <i>S. caprae</i> lipase</div> <p>The extracellular lipase from <i>S. caprae</i> NCU S6 (SCNL) was purified by three purification steps, ASP, GFC and IEC. As shown in <a href="#t0004">Table 4</a>, the activity (502.76â€‰U/mg) of the purified SCNL was increased by 332.95-fold, with approximately 6.13% recovery of overall yield. Among the three purification steps, GFC and IEC were found to be critical for obtaining highly purified lipase<a href="#CIT0017"><sup>17</sup></a>.</p> <div class="table-wrap_UNKNOWN" id="t0004" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4.</span> <p>Purification profile of lipase from <i>S. caprae</i>.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup>  <thead> <tr> <th align="left">Purification steps</th> <th align="center">Total activity (U<sup>a</sup>)</th> <th align="center">Total protein<sup>b</sup> (mg)</th> <th align="center">Protein yield (%)</th> <th align="center">Activity (%)</th> <th align="center">Specific activity (U/mg protein)</th> <th align="center">Purification (fold)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Crude extract</td> <td align="char" char=".">6154.11</td> <td align="char" char=".">4080.00</td> <td align="char" char=".">100.00</td> <td align="char" char=".">100.00</td> <td align="char" char=".">1.51</td> <td align="char" char=".">1.00</td> </tr> <tr> <td align="left">(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> precipitation (80%)</td> <td align="char" char=".">3692.47</td> <td align="char" char=".">652.00</td> <td align="char" char=".">39.33</td> <td align="char" char=".">54.67</td> <td align="char" char=".">5.66</td> <td align="char" char=".">3.75</td> </tr> <tr> <td align="left">Ultrafiltration</td> <td align="char" char=".">1891.06</td> <td align="char" char=".">24.00</td> <td align="char" char=".">15.29</td> <td align="char" char=".">30.73</td> <td align="char" char=".">78.79</td> <td align="char" char=".">52.18</td> </tr> <tr> <td align="left">Gel filtration chromatography</td> <td align="char" char=".">1250.09</td> <td align="char" char=".">6.45</td> <td align="char" char=".">5.31</td> <td align="char" char=".">20.31</td> <td align="char" char=".">193.81</td> <td align="char" char=".">128.35</td> </tr> <tr> <td align="left">Ion-exchange chromatography</td> <td align="char" char=".">626.67</td> <td align="char" char=".">1.85</td> <td align="char" char=".">3.67</td> <td align="char" char=".">10.18</td> <td align="char" char=".">338.74</td> <td align="char" char=".">224.33</td> </tr> <tr> <td align="left">Ultrafiltration</td> <td align="char" char=".">377.07</td> <td align="char" char=".">0.75</td> <td align="char" char=".">1.77</td> <td align="char" char=".">6.13</td> <td align="char" char=".">502.76</td> <td align="char" char=".">332.95</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>a</sup>One unit corresponds to 1â€‰Î¼mol of fatty acid released per min using <i>p</i>-NPP as substrate under standard conditions.</p> </div> <div class="fn-type-" title=""> <p><sup>b</sup>Protein concentration was determined by Bradford method.</p> </div> </div> </div> <p>SDS-PAGE analysis was used to determine the purity of SCNL obtained after purification. As shown in <a href="#F0004">Figure 4</a>, the purified lipase showed a single band, with a molecular weight of approximately 87â€‰kDa.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>SDSâ€"PAGE electrophoretogram of the lipase from different purification stages. Lane 1: crude enzyme, Lane 2: purified enzyme from ion-exchange chromatography, Lane 3: final purified enzyme from gel filtration chromatography.</p>   </div> </div> <div class="biochemicalcharacteristics" title="sec"><span class="label" tagx="label" title="label">3.4.</span><div class="title" tagx="title" title="title">
Biochemical characteristics</div> <div class="optimaltemperatureandthermostability" title="sec"><span class="label" tagx="label" title="label">3.4.1.</span><div class="title" tagx="title" title="title">
Optimal temperature and thermostability</div> <p>Temperature is one of the most important factors affecting the reaction rate of enzyme catalysis. As shown in <a href="#F0005">Figure 5(A)</a>, the lipase was active in the wide range of temperatures from 10 to 60â€‰Â°C. The maximum hydrolytic activity was found at 40â€‰Â°C, with the relative activity and specific activity of 100% and 503â€‰U/mg respectively. The optimal reaction temperature was lower than the lipase from <i>Aureobasidium pullulans</i> (55â€‰Â°C)<a href="#CIT0018"><sup>18</sup></a>. Activity analyses of the lipase at temperatures from 10 to 60â€‰Â°C for 240â€‰min showed that the remaining activity at 40â€‰Â°C was at the highest level (<a href="#F0005">Figure 5(B)</a>). Only the extremely high temperatures, such as 50 or 60â€‰Â°C, significantly inhibited the lipase activity, and prolonged incubation may rapidly inactivate the lipase. The highest remaining activity was at 40â€‰Â°C for 80â€‰min, and the lipase activity decreased with an increase of culture period. The decrease may be because the molecular structure of the enzyme was irreversibly changed, which may have altered the configuration of the active site, thereby decreasing interaction of the lipase with substrates<a href="#CIT0019"><sup>19</sup></a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Effect of temperature (A) and temperature-time (B), pH (C) and pH-time (D) on activity and stability of the lipase.</p>   </div> </div> <div class="optimalphandtoleranceoflipase" title="sec"><span class="label" tagx="label" title="label">3.4.2.</span><div class="title" tagx="title" title="title">
Optimal pH and tolerance of lipase</div> <p>The effect of pH on lipase activity and stability was determined by pre-incubating the enzyme in the pH values from 3.0 to 12.0 for 30â€‰min at 40â€‰Â°C. As shown in <a href="#F0005">Figure 5(C)</a>, the lipase was active (â‰¥50%) from pH 5.5 to 12.0, with the highest relative activity and residual activity at pH 9.0 and 8.0, respectively. The enzyme was similar to the lipase from <i>Serratia marcescens</i> N3, which showed maximum activity at pH 8.0<a href="#CIT0020"><sup>20</sup></a>. Investigation of pH tolerance of the enzyme (3.0â€"12.0, <a href="#F0005">Figure 5(D)</a>) showed that the enzyme reached the highest remaining activity at pH 9.0 after 5â€‰h. The lipase possessed a strong tolerance to alkaline environment, and the remaining activity was greater than 100% from pH 8.0 to 11.0. The stability of SCNL under alkaline conditions was also observed in <i>S. aureus</i> and <i>S. epidermis</i><a href="#CIT0021"><sup>21</sup></a>. The enzyme activities at pH 3.0â€"6.0 were lower than those at the neutral pH, with 60% of its initial activity being lost at pH 3.0. The results suggested that the lipase was suitable for applications under alkaline conditions.</p> </div> <div class="effectsoforganicsolventsonthestabilityofthelipase" title="sec"><span class="label" tagx="label" title="label">3.4.3.</span><div class="title" tagx="title" title="title">
Effects of organic solvents on the stability of the lipase</div> <p>Lipases with tolerance to organic solvents have received a great attention in recent years, as the enzymes could be used as desirable catalysts in various biotechnological fields, such as biofuels production, synthesis of catalysts and the optical resolution of chiral compounds<a href="#CIT0022"><sup>22</sup></a>. In the current study, the lipase showed a strong stability in several organic solvents at 10% (<i>v/v</i>) for 1â€‰day, and its activity slightly decreased in acetone, ethyl acetate, diethyl ether, ethylene glycol and acetonitrile after 28-day incubation. These results were consistent with a previous study showing that DMSO at 10% (<i>v/v</i>) improved the stability of the enzyme by dissolving the protein to some extent<a href="#CIT0023"><sup>23</sup></a>. Only some organic solvents enhanced the residual activity of the enzyme at 10â€"50% for 1 or 28â€‰days, and the effects of isooctane and glycerol on the enzyme activity were similar to the lipases from <i>Pseudomonas aeruginosa</i> PseA<a href="#CIT0024"><sup>24</sup></a>. Besides, even after preincubation in 50% (<i>v/v</i>) acetone for 28â€‰days the residual activity of the lipase was still 50% of its initial activity, indicating that it could be used as a functional catalyst for applications in the synthesis of esters.</p> </div> <div class="effectsofmetalionsonthestabilityofthelipase" title="sec"><span class="label" tagx="label" title="label">3.4.4.</span><div class="title" tagx="title" title="title">
Effects of metal ions on the stability of the lipase</div> <p>The tolerance of an enzyme to various metal ions is crucial not only for investigating its action mechanism, but also for its certain industrial applications. Several metal ions were reported to be involved in the lipase-catalyzed hydrolysis of oil by inducing the fatty acids to form their respective metal salts in the oilâ€"water interface and allowing lipase to act freely on oil molecules<a href="#CIT0025"><sup>25</sup></a>. The effects of monovalent, divalent and trivalent metal cations on lipase activity were analysed at 1 and 10â€‰mM. As shown in <a href="#t0002">Table 2</a>, the lipase activity was significantly improved in the presence of Na<sup>+</sup>, Ba<sup>2+</sup>, Ca<sup>2+</sup> and Mn<sup>2+</sup>. Especially, its relative activity in the presence of 10â€‰mM Ca<sup>2+</sup> was 1.66 folds higher than the control. Ca<sup>2+</sup> was reported to play an important role in conformational stability of the enzyme<a href="#CIT0026"><sup>26</sup></a>. Among the tested metal ions, Sn<sup>2+</sup>, Zn<sup>2+</sup>, Cu<sup>2+</sup>, Fe<sup>2+</sup>, Fe<sup>3+</sup>, Al<sup>3+</sup>, Sb<sup>3+</sup> and Bi<sup>3+</sup> significantly inhibited the relative activity.</p> </div> <div class="effectsofinhibitorsanddetergentsonthestabilityofthelipase" title="sec"><span class="label" tagx="label" title="label">3.4.5.</span><div class="title" tagx="title" title="title">
Effects of inhibitors and detergents on the stability of the lipase</div> <p>Commercial inhibitors and detergents were important surfactants for the preparation of emulsions in their practical applications. As shown in <a href="#t0003">Table 3</a>, ethylenediaminetetraacetic acid (EDTA, 1 and 10â€‰mM) exhibited significant effects on its relative activity (66.5 and 32.9%), indicating that SCNL was not only a metalloenzyme, but also a metal-activated enzyme. The inhibitors, H<sub>2</sub>O<sub>2</sub> and dithiothreitol (DTT) were reported to reduce the disulphide bonds of an enzyme, and prevent the formation of inter- or intramolecular disulphide bonds between cysteine residues of a lipase<a href="#CIT0027"><sup>27</sup></a>, but the relative activities of the lipase in the presence of these two inhibitors at 1 and 10â€‰mM still maintained more than 70%.</p> <p>The stability of lipases in the presence of detergents is the premise of being good washing additives in commercial detergents. In addition, surfactants can stimulate lipases in molecular bioimprinting, and play important roles in multifunctionality and various properties of enzymes<a href="#CIT0028"><sup>28</sup></a>. Interestingly, cationic detergent cetyltrimethylammonium bromide (CTAB) and anionic detergent (sodium dodecyl sulphate (SDS) slightly decreased the relative activity of SCNL, which disagreed with the lipase from <i>Burkholderia ubonensis</i> SL-4<a href="#CIT0029"><sup>29</sup></a>. As shown in <a href="#t0003">Table 3</a>, the non-ionic detergents slightly improved lipase activity. The non-ionic detergents did not weaken the hydrophobic interactions within the protein and damage the lipase<a href="#CIT0030"><sup>30</sup></a>. In addition, bile salts, such as sodium cholate, an universal commercial biosurfactant as a reducing agent in aqueous solutions slightly restrained the activity at 1 or 10â€‰mM, as the lipase still retained above 80% of initial activity. These findings were similar to a previous study showing that the bile salts did not restrain the activity of <i>S. xylosus</i> lipase even at higher concentrations<a href="#CIT0031"><sup>31</sup></a>. Therefore, SCNL may be a novel additive for the industrial applications of various multifunctional commercial detergents.</p> </div> <div class="km,vmaxandsubstratespecificity" title="sec"><span class="label" tagx="label" title="label">3.4.6.</span><div class="title" tagx="title" title="title"><i>K<sub>m</sub></i>, <i>V</i><sub>max</sub> and substrate specificity</div> <p>The <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> values of SCNL were determined to be 0.695â€‰mM and 262.66â€‰s<sup>âˆ’1â€‰</sup>mM<sup>âˆ’1</sup> respectively, when the substrate was <i>p</i>-NPP. The <i>K</i>m value was slightly higher than those of KM12 lipase (0.53â€‰mM)<a href="#CIT0032"><sup>32</sup></a> and the lipase from <i>B. licheniformis</i> strain SCD11501 (0.43â€‰mM)<a href="#CIT0033"><sup>33</sup></a>. The <i>V</i>max value of SCNL was higher than those of KM12 lipase (171.2â€‰s<sup>âˆ’1â€‰</sup>mM<sup>âˆ’1</sup>)<a href="#CIT0032"><sup>32</sup></a> and SML lipase (149.03â€‰s<sup>âˆ’1â€‰</sup>mM<sup>âˆ’1</sup>)<a href="#CIT0034"><sup>34</sup></a>. The results indicated the strong attraction of SCNL for the main substrate, and its high catalytic efficiency, and SCNL may be used as a novel lipase in pharmaceutical and biotech fields due to its notable economic and ecological benefits<a href="#CIT0035"><sup>35</sup></a>.</p> <p>The SCNL can hydrolyse <i>p</i>-NP esters with different acyl chain length (<a href="#t0005">Table 5</a>), with the highest specific activity of 616.54â€‰U/mg towards <i>p</i>NPS (18 carbons). The <i>p</i>-NP esters from C<sub>2</sub> to C<sub>16</sub> were hydrolysed moderately (45.46â€"54.54%) by SCNL apart from <i>p</i>NPB (4 carbons), indicating its preference for C<sub>4</sub> with a higher specific activity of 479.91â€‰U/mg. The lipase was similar to the lipases from <i>M. cinnamomea</i><a href="#CIT0036"><sup>36</sup></a> and <i>B. licheniformis</i> DSM 12369<a href="#CIT0024"><sup>24</sup></a>, possessed a strong hydrolysis efficiency for medium and long carbon chain length substrates, and had mild specificity towards short-chain <i>p</i>-NP esters. Among the triglycerides, SCNL showed a preference for short-chain and medium-chain fatty acids from C<sub>2</sub> to C<sub>12</sub>, with the maximum activity (196.00â€‰U/mg) in tributyrin (C<sub>4</sub>). The results indicated a lower hydrolytic activity of the enzyme for triglycerides with more than twelve carbons atoms (C<sub>14</sub>â€"C<sub>18</sub>), and that SCNL was a novel lipase different from esterases that only hydrolyse short-chain fatty acids.</p> <div class="table-wrap_UNKNOWN" id="t0005" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 5.</span> <p>Substrate specificity of SCNL.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup> <colgroup><col width="80pt" /></colgroup>  <thead> <tr> <th align="left">Substrate</th> <th align="center">Specific activity (U/mg)</th> <th align="center">Relative activity (%)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left"><i>p</i>NP ester<sup>a</sup></td> <td align="char" char="Â±">Â </td> <td align="char" char=".">Â </td> </tr> <tr> <td align="left"><i>â€ƒp</i>NPA (<i>p</i>-nitrophenyl acetate) (C<sub>2</sub>)</td> <td align="char" char="Â±">301.20â€‰Â±â€‰7.96</td> <td align="char" char=".">48.85</td> </tr> <tr> <td align="left"><i>â€ƒp</i>NPB (<i>p</i>-nitrophenyl butyrate) (C<sub>4</sub>)</td> <td align="char" char="Â±">479.91â€‰Â±â€‰1.79</td> <td align="char" char=".">77.84</td> </tr> <tr> <td align="left"><i>â€ƒp</i>NPH (<i>p</i>-nitrophenyl hexanoate) (C<sub>6</sub>)</td> <td align="char" char="Â±">281.98â€‰Â±â€‰3.08</td> <td align="char" char=".">45.74</td> </tr> <tr> <td align="left"><i>â€ƒp</i>NPC (<i>p</i>-nitrophenyl caprylate) (C<sub>8</sub>)</td> <td align="char" char="Â±">280.28â€‰Â±â€‰6.21</td> <td align="char" char=".">45.46</td> </tr> <tr> <td align="left"><i>â€ƒp</i>NPD (<i>p</i>-nitrophenyl decanoate) (C<sub>10</sub>)</td> <td align="char" char="Â±">336.23â€‰Â±â€‰6.23</td> <td align="char" char=".">54.54</td> </tr> <tr> <td align="left"><i>â€ƒp</i>NPL (<i>p</i>-nitrophenyl laurate) (C<sub>12</sub>)</td> <td align="char" char="Â±">312.72â€‰Â±â€‰1.92</td> <td align="char" char=".">50.72</td> </tr> <tr> <td align="left"><i>â€ƒp</i>NPM (<i>p</i>-nitrophenyl myristate) (C<sub>14</sub>)</td> <td align="char" char="Â±">310.45â€‰Â±â€‰2.50</td> <td align="char" char=".">50.35</td> </tr> <tr> <td align="left"><i>â€ƒp</i>NPP (<i>p</i>-nitrophenyl palmitate) (C<sub>16</sub>)</td> <td align="char" char="Â±">334.72â€‰Â±â€‰1.14</td> <td align="char" char=".">54.29</td> </tr> <tr> <td align="left"><i>â€ƒp</i>NPS (<i>p</i>-nitrophenyl stearate) (C<sub>18</sub>)</td> <td align="char" char="Â±">616.54â€‰Â±â€‰5.96</td> <td align="char" char=".">100.00<sup>c</sup></td> </tr> <tr> <td align="left">Triglyceride<sup>b</sup></td> <td align="char" char="Â±">Â </td> <td align="char" char=".">Â </td> </tr> <tr> <td align="left">â€ƒTriacetin (C<sub>2</sub>)</td> <td align="char" char="Â±">71.05â€‰Â±â€‰6.00</td> <td align="char" char=".">36.25</td> </tr> <tr> <td align="left">â€ƒTributyrin (C<sub>4</sub>)</td> <td align="char" char="Â±">196.00â€‰Â±â€‰5.29</td> <td align="char" char=".">100.00<sup>c</sup></td> </tr> <tr> <td align="left">â€ƒTricaproin (C<sub>6</sub>)</td> <td align="char" char="Â±">60.43â€‰Â±â€‰6.11</td> <td align="char" char=".">30.83</td> </tr> <tr> <td align="left">â€ƒTricaprylin (C<sub>8</sub>)</td> <td align="char" char="Â±">49.00â€‰Â±â€‰4.00</td> <td align="char" char=".">25.00</td> </tr> <tr> <td align="left">â€ƒTricaprin (C<sub>10</sub>)</td> <td align="char" char="Â±">24.01â€‰Â±â€‰3.42</td> <td align="char" char=".">12.25</td> </tr> <tr> <td align="left">â€ƒTrilaurin (C<sub>12</sub>)</td> <td align="char" char="Â±">29.40â€‰Â±â€‰4.00</td> <td align="char" char=".">15.00</td> </tr> <tr> <td align="left">â€ƒTrimyristin (C<sub>14</sub>)</td> <td align="char" char="Â±">28.58â€‰Â±â€‰3.06</td> <td align="char" char=".">14.58</td> </tr> <tr> <td align="left">â€ƒTripalmitin (C<sub>16</sub>)</td> <td align="char" char="Â±">24.50â€‰Â±â€‰5.29</td> <td align="char" char=".">12.50</td> </tr> <tr> <td align="left">â€ƒTristearin (C<sub>18</sub>)</td> <td align="char" char="Â±">22.87â€‰Â±â€‰4.62</td> <td align="char" char=".">11.67</td> </tr> <tr> <td align="left">Natural oils<sup>b</sup></td> <td align="char" char="Â±">Â </td> <td align="char" char=".">Â </td> </tr> <tr> <td align="left">â€ƒOlive oil</td> <td align="char" char="Â±">184.80â€‰Â±â€‰3.59</td> <td align="char" char=".">84.00</td> </tr> <tr> <td align="left">â€ƒSoybean oil</td> <td align="char" char="Â±">155.47â€‰Â±â€‰4.15</td> <td align="char" char=".">70.67</td> </tr> <tr> <td align="left">â€ƒRice bran oil</td> <td align="char" char="Â±">136.69â€‰Â±â€‰3.96</td> <td align="char" char=".">62.13</td> </tr> <tr> <td align="left">â€ƒFlaxseed oil</td> <td align="char" char="Â±">153.41â€‰Â±â€‰2.90</td> <td align="char" char=".">69.73</td> </tr> <tr> <td align="left">â€ƒPerilla oil</td> <td align="char" char="Â±">142.56â€‰Â±â€‰1.90</td> <td align="char" char=".">64.80</td> </tr> <tr> <td align="left">â€ƒCanola oil</td> <td align="char" char="Â±">220.00â€‰Â±â€‰7.19</td> <td align="char" char=".">100.00<sup>c</sup></td> </tr> <tr> <td align="left">â€ƒCorn oil</td> <td align="char" char="Â±">150.48â€‰Â±â€‰0.72</td> <td align="char" char=".">68.40</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>a</sup>Unit definition: 1 unit of activity is the amount of enzyme necessary to release 1â€‰Î¼mol of <i>p</i>-NP per minute at 37â€‰Â°C and pH 7.0 for SCNL.</p> </div> <div class="fn-type-" title=""> <p><sup>b</sup>Unit definition: 1 unit of activity is the amount of enzyme necessary to release 1â€‰Î¼mol of free fatty acids per minute at 37â€‰Â°C and pH 7.0 for SCNL.</p> </div> <div class="fn-type-" title=""> <p><sup>c</sup>The highest activity of SCNL was taken as 100% by using different substrates.</p> </div> </div> </div> <p>Among the tested natural oils, the canola oil was hydrolysed to a higher degree by SCNL at a high specific activity (220.00â€‰U/mg). The canola oil contains less than 2% erucic acid and less than 30â€‰mM glucosinolates, and has a relatively low level of saturated fat (â‰¤7%) and a high content of polyunsaturated fats, such as linoleic acid and Î±-linolenic acid (formally called 9,12,15-all-cis-octadecatrienoic acid). Î±-linolenic acid is an unsaturated omega-3 fatty acid available in many plant oils. The activity of SCNL was strong for natural oils with medium-chain to long-chain fatty acids [C<sub>6:0</sub>â€"C<sub>16:0</sub>]. Among them, both olive oil (184.80â€‰U/mg) and rice bran oil (176.69â€‰U/mg) contain long unsaturated fatty acids, such as oleic acid and linoleic acid. These results were in accordance with a previous report<a href="#CIT0014"><sup>14</sup></a>, indicating its preference to medium-chain fatty acids and long unsaturated fatty acids.</p> </div> </div> </div> <div class="conclusions" title="sec"><span class="label" tagx="label" title="label">4.</span><div class="title" tagx="title" title="title">
Conclusions</div> <p>A novel and multifunctional lipase, SCNL was purified from a newly isolated <i>S. caprae</i> NCU S6. The molecular weight of the lipase was determined to be approximately 87â€‰kDa through gel electrophoresis analysis. The lipase exhibited high temperature resistance and great pH stability. Furthermore, SCNL was highly active in the presence of a variety of industrial organic solvents and metal ions. The lipase also exhibited strong tolerance to widely used enzyme inhibitors and commercial detergents. Our results suggested that SCNL could be an excellent enzyme candidate for biotransformation in the food and pharmaceutical industries. Further studies would be focussed on its region-specificity and structural analysis, and its potential applications in biodiesel production and detergent industries.</p> </div> </div> <div class="back" title="back"> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>No potential conflict of interest was reported by the author(s).</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Javed</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azeem</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hussain</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial lipases: a review on purification and characterization</span>. <span class="source" tagx="source" title="source">Prog Biophys Mol Biol</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">132</span>:<span class="fpage" tagx="fpage" title="fpage">23</span>â€"<span class="lpage" tagx="lpage" title="lpage">34</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28774751">28774751</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarmah</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Revathi</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheelu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Recent advances on sources and industrial applications of lipases</span>. <span class="source" tagx="source" title="source">Biotechnol Prog</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">34</span>:<span class="fpage" tagx="fpage" title="fpage">5</span>â€"<span class="lpage" tagx="lpage" title="lpage">28</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29086509">29086509</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soni</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vohra</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Purification and characterisation of a thermostable alkaline lipase from a new thermophilic <i>Bacillus</i> sp. Rsj-1</span>. <span class="source" tagx="source" title="source">Process Biochemistry</span><span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">37</span>:<span class="fpage" tagx="fpage" title="fpage">1075</span>â€"<span class="lpage" tagx="lpage" title="lpage">84</span>.</span></li> <li tag="ref"><a name="CIT0004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lumor</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akoh</span><span class="given-names" tagx="given-names" title="given-names">CC.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Esterification and hydrolytic activities of candida rugosa lipase isoform 1 (lip1) immobilized on celite 545, duolite a7, and sephadex g-25</span>. <span class="source" tagx="source" title="source">J Agric Food Chem</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">56</span>:<span class="fpage" tagx="fpage" title="fpage">10396</span>â€"<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18850711">18850711</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Purification and characterization of an extracellular cold-adapted alkaline lipase produced by psychrotrophic bacterium <i>Yersinia enterocolitica</i> strain KM1</span>. <span class="source" tagx="source" title="source">J Basic Microbiol</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">55</span>:<span class="fpage" tagx="fpage" title="fpage">718</span>â€"<span class="lpage" tagx="lpage" title="lpage">28</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25677080">25677080</a></span></span></li> <li tag="ref"><a name="CIT0006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cadieux</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijayakumaran</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernards</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Role of lipase from community-associated methicillin-resistant staphylococcus aureus strain usa300 in hydrolyzing triglycerides into growth-inhibitory free fatty acids</span>. <span class="source" tagx="source" title="source">J Bacteriol</span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">196</span>:<span class="fpage" tagx="fpage" title="fpage">4044</span>â€"<span class="lpage" tagx="lpage" title="lpage">56</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25225262">25225262</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samad</span><span class="given-names" tagx="given-names" title="given-names">MYA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Razak</span><span class="given-names" tagx="given-names" title="given-names">CNA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salleh</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A plate assay for primary screening of lipase activity</span>. <span class="source" tagx="source" title="source">J Microbiol Meth</span><span class="year" tagx="year" title="year">1989</span>;<span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">51</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.</span></li> <li tag="ref"><a name="CIT0008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouker</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaeger</span><span class="given-names" tagx="given-names" title="given-names">KE.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Specific and sensitive plate assay for bacterial lipases</span>. <span class="source" tagx="source" title="source">Appl Environ Microbiol</span><span class="year" tagx="year" title="year">1987</span>;<span class="volume" tagx="volume" title="volume">53</span>:<span class="fpage" tagx="fpage" title="fpage">211</span>â€"<span class="lpage" tagx="lpage" title="lpage">3</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3103532">3103532</a></span></span></li> <li tag="ref"><a name="CIT0009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferreira</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonesso</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mondelli</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">da Cunha</span><span class="given-names" tagx="given-names" title="given-names">M. d L R d S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of <i>Staphylococcus saprophyticus</i> isolated from patients with urinary tract infection using a simple set of biochemical tests correlating with 16s-23s interspace region molecular weight patterns</span>. <span class="source" tagx="source" title="source">J Microbiol Methods</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">91</span>:<span class="fpage" tagx="fpage" title="fpage">406</span>â€"<span class="lpage" tagx="lpage" title="lpage">11</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23041266">23041266</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gricajeva</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BendikienÄ—</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KalÄ—dienÄ—</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lipase of bacillus stratosphericus l1: Cloning, expression and characterization</span>. <span class="source" tagx="source" title="source">Int J Biol Macromol</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">92</span>:<span class="fpage" tagx="fpage" title="fpage">96</span>â€"<span class="lpage" tagx="lpage" title="lpage">104</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27392776">27392776</a></span></span></li> <li tag="ref"><a name="CIT0011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gricajeva</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BikutÄ—</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KalÄ—dienÄ—</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Atypical organic-solvent tolerant bacterial hormone sensitive lipase-like homologue estag1 from <i>Staphylococcus saprophyticus</i> ag1: Synthesis and characterization</span>. <span class="source" tagx="source" title="source">Int J Biol Macromol</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">130</span>:<span class="fpage" tagx="fpage" title="fpage">253</span>â€"<span class="lpage" tagx="lpage" title="lpage">65</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30797006">30797006</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bradford</span><span class="given-names" tagx="given-names" title="given-names">MM.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span>. <span class="source" tagx="source" title="source">Anal Biochem</span><span class="year" tagx="year" title="year">1976</span>;<span class="volume" tagx="volume" title="volume">72</span>:<span class="fpage" tagx="fpage" title="fpage">248</span>â€"<span class="lpage" tagx="lpage" title="lpage">54</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/942051">942051</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laemmli</span><span class="given-names" tagx="given-names" title="given-names">UK.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cleavage of structural proteins during the assembly of the head of bacteriophage T4</span>. <span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1970</span>;<span class="volume" tagx="volume" title="volume">227</span>:<span class="fpage" tagx="fpage" title="fpage">680</span>â€"<span class="lpage" tagx="lpage" title="lpage">5</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5432063">5432063</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kao</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Two novel lipases purified from rice bran displaying lipolytic and esterification activities</span>. <span class="source" tagx="source" title="source">Int J Biol Macromol</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">139</span>:<span class="fpage" tagx="fpage" title="fpage">298</span>â€"<span class="lpage" tagx="lpage" title="lpage">306</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31386870">31386870</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boekema</span><span class="given-names" tagx="given-names" title="given-names">BKHL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beselin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breuer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hexadecane and tween 80 stimulate lipase production in <i>Burkholderia glumae</i> by different mechanisms</span>. <span class="source" tagx="source" title="source">Appl Environ Microbiol</span><span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">73</span>:<span class="fpage" tagx="fpage" title="fpage">3838</span>â€"<span class="lpage" tagx="lpage" title="lpage">44</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17468265">17468265</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rmili</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Achouri</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smichi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Purification and biochemical characterization of an organic solvent-tolerant and detergent-stable lipase from <i>Staphylococcus capitis</i></span>. <span class="source" tagx="source" title="source">Biotechnol Prog</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">35</span>:<span class="fpage" tagx="fpage" title="fpage">e2833</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31050178">31050178</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castro-Ochoa</span><span class="given-names" tagx="given-names" title="given-names">LD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RodrÃ­guez-GÃ³mez</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valerio-Alfaro</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ros</span><span class="given-names" tagx="given-names" title="given-names">RO.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Screening, purification and characterization of the thermoalkalophilic lipase produced by <i>Bacillus thermoleovorans</i> ccr11</span>. <span class="source" tagx="source" title="source">Enzyme Microbial Technology</span><span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">37</span>:<span class="fpage" tagx="fpage" title="fpage">648</span>â€"<span class="lpage" tagx="lpage" title="lpage">54</span>.</span></li> <li tag="ref"><a name="CIT0018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Screening, purification, and characterization of an extracellular lipase from <i>Aureobasidium pullulans</i> isolated from stuffed buns steamers</span>. <span class="source" tagx="source" title="source">J Zhejiang Univ Sci B</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">20</span>:<span class="fpage" tagx="fpage" title="fpage">332</span>â€"<span class="lpage" tagx="lpage" title="lpage">42</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30932378">30932378</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shivakumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharya</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Purification and characterization of a surfactant-compatible lipase from aspergillus tamarii jgif06 exhibiting energy-efficient removal of oil stains from polycotton fabric</span>. <span class="source" tagx="source" title="source">Biotechnol</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">131</span>.</span></li> <li tag="ref"><a name="CIT0020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neihaya</span><span class="given-names" tagx="given-names" title="given-names">HZ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saeed</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baho</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Production, purification and characterization of extra cellular lipase from <i>Serratia marcescens</i> and its potential activity for hydrolysis of edible oils</span>. <span class="source" tagx="source" title="source">Al Nahrain Univ</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">94</span>â€"<span class="lpage" tagx="lpage" title="lpage">102</span>.</span></li> <li tag="ref"><a name="CIT0021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khosasih</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suwanto</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">HK.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of lipases from <i>Staphylococcus aureus</i> and <i>Staphylococcus epidermidis</i> isolated from human facial sebaceous skin</span>. <span class="source" tagx="source" title="source">J Microbiol Biotechnol</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">22</span>:<span class="fpage" tagx="fpage" title="fpage">84</span>â€"<span class="lpage" tagx="lpage" title="lpage">91</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22297223">22297223</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dandavate</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jinjala</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keharia</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madamwar</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Production, partial purification and characterization of organic solvent tolerant lipase from <i>Burkholderia multivorans</i> v2 and its application for ester synthesis</span>. <span class="source" tagx="source" title="source">Bioresour Technol</span><span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">100</span>:<span class="fpage" tagx="fpage" title="fpage">3374</span>â€"<span class="lpage" tagx="lpage" title="lpage">81</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19285387">19285387</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J-H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biochemical properties and potential applications of an organic solvent-tolerant lipase isolated from <i>Serratia marcescens</i> ecu1010</span>. <span class="source" tagx="source" title="source">Process Biochem</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">43</span>:<span class="fpage" tagx="fpage" title="fpage">626</span>â€"<span class="lpage" tagx="lpage" title="lpage">33</span>.</span></li> <li tag="ref"><a name="CIT0024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaur</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khare</span><span class="given-names" tagx="given-names" title="given-names">SK.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Purification and characterization of lipase from solvent tolerant <i>Pseudomonas aeruginosa</i> psea</span>. <span class="source" tagx="source" title="source">Process Biochemy</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">43</span>:<span class="fpage" tagx="fpage" title="fpage">1040</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.</span></li> <li tag="ref"><a name="CIT0025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharon</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakazato</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogawa</span><span class="given-names" tagx="given-names" title="given-names">HI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kato</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lipase-induced hydrolysis of castor oil: effect of various metals</span>. <span class="source" tagx="source" title="source">J Indus Microbiol Biotechnol</span><span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">21</span>:<span class="fpage" tagx="fpage" title="fpage">292</span>â€"<span class="lpage" tagx="lpage" title="lpage">5</span>.</span></li> <li tag="ref"><a name="CIT0026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Invernizzi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papaleo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grandori</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Relevance of metal ions for lipase stability: structural rearrangements induced in the <i>Burkholderia glumae</i> lipase by calcium depletion</span>. <span class="source" tagx="source" title="source">J Struct Biol</span><span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">168</span>:<span class="fpage" tagx="fpage" title="fpage">562</span>â€"<span class="lpage" tagx="lpage" title="lpage">70</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19635571">19635571</a></span></span></li> <li tag="ref"><a name="CIT0027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phukon</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chourasia</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumari</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Production and characterisation of lipase for application in detergent industry from a novel <i>Pseudomonas helmanticensis</i> hs6</span>. <span class="source" tagx="source" title="source">Bioresource Technology</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">309</span>:<span class="fpage" tagx="fpage" title="fpage">123352</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32299046">32299046</a></span></span></li> <li tag="ref"><a name="CIT0028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukherjee</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">MN.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular bioimprinting of lipases with surfactants and its functional consequences in low water media</span>. <span class="source" tagx="source" title="source">Int J Biol Macromol</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">81</span>:<span class="fpage" tagx="fpage" title="fpage">544</span>â€"<span class="lpage" tagx="lpage" title="lpage">51</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26306412">26306412</a></span></span></li> <li tag="ref"><a name="CIT0029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A new extracellular thermo-solvent-stable lipase from <i>Burkholderia ubonensis</i> sl-4: identification, characterization and application for biodiesel production</span>. <span class="source" tagx="source" title="source">J Mol Catal B</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">126</span>:<span class="fpage" tagx="fpage" title="fpage">76</span>â€"<span class="lpage" tagx="lpage" title="lpage">89</span>.</span></li> <li tag="ref"><a name="CIT0030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kukreja</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bera</span><span class="given-names" tagx="given-names" title="given-names">MB.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lipase from pseudomonas aeruginosa mtcc 2488: partial purification, characterization and calcium dependent thermostability</span>. <span class="source" tagx="source" title="source">Ind J Biotechnol</span><span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">222</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.</span></li> <li tag="ref"><a name="CIT0031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouaziz</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horchani</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salem</span><span class="given-names" tagx="given-names" title="given-names">NB</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Expression, purification of a novel alkaline <i>Staphylococcus xylosus</i> lipase acting at high temperature</span>. <span class="source" tagx="source" title="source">Biochem Eng J</span><span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">54</span>:<span class="fpage" tagx="fpage" title="fpage">93</span>â€"<span class="lpage" tagx="lpage" title="lpage">102</span>.</span></li> <li tag="ref"><a name="CIT0032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malekabadi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Badoei-Dalfard</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karami</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biochemical characterization of a novel cold-active, halophilic and organic solvent-tolerant lipase from b. Licheniformis km12 with potential application for biodiesel production</span>. <span class="source" tagx="source" title="source">Int J Biol Macromol</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">109</span>:<span class="fpage" tagx="fpage" title="fpage">389</span>â€"<span class="lpage" tagx="lpage" title="lpage">98</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29258898">29258898</a></span></span></li> <li tag="ref"><a name="CIT0033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanwar</span><span class="given-names" tagx="given-names" title="given-names">SS.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Purification and bio-chemical characterization of a solvent-tolerant and highly thermostable lipase of <i>Bacillus licheniformis</i> strain scd11501</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci India Sect B Biol Sci</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">87</span>:<span class="fpage" tagx="fpage" title="fpage">411</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.</span></li> <li tag="ref"><a name="CIT0034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">JP.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Screening, purification and characterization of a novel cold-active and organic solvent-tolerant lipase from <i>Stenotrophomonas maltophilia</i> cgmcc 4254</span>. <span class="source" tagx="source" title="source">Bioresour Technol</span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">148</span>:<span class="fpage" tagx="fpage" title="fpage">114</span>â€"<span class="lpage" tagx="lpage" title="lpage">20</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24050922">24050922</a></span></span></li> <li tag="ref"><a name="CIT0035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarkar</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamasaki</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basak</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Purification and characterization of a new alkali-thermostable lipase from <i>Staphylococcus aureus</i> isolated from Arachis hypogaea rhizosphere</span>. <span class="source" tagx="source" title="source">Process Biochem</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">47</span>:<span class="fpage" tagx="fpage" title="fpage">858</span>â€"<span class="lpage" tagx="lpage" title="lpage">66</span>.</span></li> <li tag="ref"><a name="CIT0036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biochemical characterization of a novel lipase from <i>Malbranchea cinnamomea</i> suitable for production of lipolyzed milkfat flavor and biodegradation of phthalate esters</span>. <span class="source" tagx="source" title="source">Food Chem</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">297</span>:<span class="fpage" tagx="fpage" title="fpage">124925</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31253266">31253266</a></span></span></li> </ul> </div> </div>  </body></html>