<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Enzyme Inhib Med Chem</span><span class="iso-abbrev" title="iso-abbrev">J Enzyme Inhib Med Chem</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Enzyme Inhibition and Medicinal Chemistry</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1475-6366</span><span class="issn-epub" tagx="issn" title="issn-epub">1475-6374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7594867">7594867</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/33103497">33103497</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/14756366.2020.1837124">10.1080/14756366.2020.1837124</a></span><span class="publisher-id" title="publisher-id">1837124</span><div class="unknown" title="article-version" tag="article-version">
Version of Record</div> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Research Paper</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Pharmacophore-based virtual screening, synthesis, biological evaluation, and molecular docking study of novel pyrrolizines bearing urea/thiourea moieties with potential cytotoxicity and CDK inhibitory activities</div> <div class="alt-title" title="alt-title">
A. M. Shawky et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shawky</span><span class="given-names" tagx="given-names" title="given-names">Ahmed M.</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibrahim</span><span class="given-names" tagx="given-names" title="given-names">Nashwa A.</span></span><a href="#AF0002">b</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abourehab</span><span class="given-names" tagx="given-names" title="given-names">Mohammed A. S.</span></span><a href="#AF0003">c</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdalla</span><span class="given-names" tagx="given-names" title="given-names">Ashraf N.</span></span><a href="#AF0004">d</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-4527-8885</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">Ahmed M.</span></span><a href="#AF0002">b</a><a href="#AF0005">e</a><a href="#AN0001" /></span><span class="citation_author_institution" id="AF0001">[a], <span class="institution" title="institution">Science and Technology Unit (STU), Umm Al-Qura University</span><span class="city" tagx="city" title="city">Makkah</span><span class="country" tagx="country" title="country">Saudi Arabia</span></span><span class="citation_author_institution" id="AF0002">[b], <span class="institution" title="institution">Department of Pharmaceutical Chemistry, Faculty of Pharmacy, Umm Al-Qura University</span><span class="city" tagx="city" title="city">Makkah</span><span class="country" tagx="country" title="country">Saudi Arabia</span></span><span class="citation_author_institution" id="AF0003">[c], <span class="institution" title="institution">Department of Pharmaceutics, Faculty of Pharmacy, Umm Al-Qura University</span><span class="city" tagx="city" title="city">Makkah</span><span class="country" tagx="country" title="country">Saudi Arabia</span></span><span class="citation_author_institution" id="AF0004">[d], <span class="institution" title="institution">Department of Pharmacology and Toxicology, Faculty of Pharmacy, Umm Al-Qura University</span><span class="city" tagx="city" title="city">Makkah</span><span class="country" tagx="country" title="country">Saudi Arabia</span></span><span class="citation_author_institution" id="AF0005">[e], <span class="institution" title="institution">Department of Medicinal Chemistry, Faculty of Pharmacy, Beni-Suef University</span><span class="city" tagx="city" title="city">Beni-Suef</span><span class="country" tagx="country" title="country">Egypt</span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""> <p>Supplemental data for this article can be accessed <a href="https://doi.org/10.1080/14756366.2020.1837124"><u>here</u></a>.</p> </div> <div class="corresp" title="corresp"><b>CONTACT</b> Ahmed M. Gouda <span class="email" tagx="email" title="email">ahmed.gouda@pharm.bsu.edu.eg</span>, <span class="email" tagx="email" title="email">amsaid@uqu.edu.sa</span><span class="institution" title="institution">Department of Medicinal Chemistry, Faculty of Pharmacy, Beni-Suef University</span>, <span class="city" tagx="city" title="city">Beni-Suef</span><span class="postal-code" tagx="postal-code" title="postal-code">62514</span>, <span class="country" tagx="country" title="country">Egypt</span></div> </div><span class="pub-date-" title="pub-date-">: <span>2020-10-10</span></span><span class="pub-date-" title="pub-date-">: <span>2021</span></span><span class="volume" tagx="volume" title="volume">36</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">33</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">The Author(s)</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution-NonCommercial License (<a href="http://creativecommons.org/licenses/by-nc/4.0/">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="IENZ_36_1837124.pdf" title="self-uri">IENZ_36_1837124.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>In the current study, virtual screening of a small library of 1302 pyrrolizines bearing urea/thiourea moieties was performed. The top-scoring hits were synthesised and evaluated for their cytotoxicity against three cancer (MCF-7, A2780, and HT29) and one normal (MRC-5) cell lines. The results of the MTT assay revealed potent cytotoxic activities for most of the new compounds (IC<sub>50</sub> = 0.16â€"34.13â€‰Î¼M). The drug-likeness study revealed that all the new compounds conform to Lipinskiâ€™s rule. Mechanistic studies of compounds <b>18â€‰b</b>, <b>19a</b>, and <b>20a</b> revealed the induction of apoptosis and cell cycle arrest at the G<sub>1</sub> phase in MCF-7 cells. The three compounds also displayed potent inhibitory activity against CDK-2 (IC<sub>50</sub> = 25.53â€"115.30â€‰nM). Moreover, the docking study revealed a nice fitting of compound <b>19a</b> into the active sites of CDK-2/6/9. These preliminary results suggested that compound <b>19a</b> could serve as a promising scaffold in the discovery of new potent anticancer agents.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Graphical Abstract</div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/" /></div>  </div> <p> </p><ul> <div class="title" tagx="title" title="title">
Highlights</div> <li> <p>Virtual screening of 1302 pyrrolizines was done using the pharmacophore model of the multi-CDKI <b>3</b>.</p> </li> <li> <p>The top-scoring hits were synthesised and evaluated for their cytotoxic activities.</p> </li> <li> <p>Compound <b>19a</b> showed potent <i>inÂ vitro</i> cytotoxic activity against CDK-2.</p> </li> <li> <p>Compound <b>19a</b> induced apoptosis and cell cycle arrest at the G<sub>1</sub> phase in MCF-7 cells.</p> </li> <li> <p>The docking study revealed nice fitting of compound <b>19a</b> into CDK-2/6/9 with high binding affinities.</p> </li> </ul> <p /> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Pyrrolizine</div> <div class="kwd" title="kwd">
urea derivatives</div> <div class="kwd" title="kwd">
cytotoxicity</div> <div class="kwd" title="kwd">
apoptosis</div> <div class="kwd" title="kwd">
cell cycle</div> <div class="kwd" title="kwd">
CDK-2</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1.</span><div class="title" tagx="title" title="title">
Introduction</div> <p>Targeting the oncogenic protein kinases has emerged as a promising strategy in the development of new anticancer agents in the last three decades<a href="#CIT0001"><sup>1</sup></a><sup>,</sup><a href="#CIT0002"><sup>2</sup></a>. Currently, more than 40 kinase inhibitors were approved by the FDA for the treatment of different types of cancers<a href="#CIT0003"><sup>3â€"5</sup></a>. Among these inhibitors, the cyclin-dependent kinase inhibitors (CDKIs) attracted much attention which could be due to the important role of CDKs in cell division and differentiation<a href="#CIT0006"><sup>6</sup></a><sup>,</sup><a href="#CIT0007"><sup>7</sup></a>. Among the CDK family, CDK-2 plays an important role in the progression of cells from G<sub>1</sub> to S cell cycle phases<a href="#CIT0006"><sup>6</sup></a><sup>,</sup><a href="#CIT0007"><sup>7</sup></a>. The overexpression of CDK-2 was also reported in several solid tumours such as breast<a href="#CIT0008"><sup>8</sup></a>, colon<a href="#CIT0009"><sup>9</sup></a>, and ovarian cancers<a href="#CIT0010"><sup>10</sup></a>. In addition, the increase in CDK-2 expression was also associated with the induction of the radio-resistance in glioblastoma cells<a href="#CIT0011"><sup>11</sup></a><sup>,</sup><a href="#CIT0012"><sup>12</sup></a>, and metastasis in prostate cancer<a href="#CIT0013"><sup>13</sup></a>. CDK-2 also has a crucial role in DNA replication and apoptosis in different types of cancer<a href="#CIT0006"><sup>6</sup></a><sup>,</sup><a href="#CIT0012"><sup>12</sup></a>.</p> <p>These findings highlighted the importance of CDK-2 as a potential target in cancer chemotherapy. Several small molecule inhibitors of CDK-2 have proved potent anticancer activities<a href="#CIT0006"><sup>6</sup></a><sup>,</sup><a href="#CIT0014"><sup>14</sup></a>. However, many of these inhibitors have also displayed pan-CDKs inhibitory activity which could be attributed to the high sequence similarity between different members of the CDK family<a href="#CIT0006"><sup>6</sup></a>. Some of these inhibitors (roscovitine, dinaciclib, and Ro-3306) have succeeded to reach phase I/II clinical trials<a href="#CIT0006"><sup>6</sup></a><sup>,</sup><a href="#CIT0014"><sup>14</sup></a>.</p> <p>Our literature review<a href="#CIT0016"><sup>16â€"18</sup></a> revealed several CDKIs bearing similar pharmacophoric groups which include two aryl/heterocyclic rings, two carbonyl groups, and a five-membered pyrazole/imidazole core, <a href="#F0001">Figure 1</a>. Among these inhibitors, compound <b>1</b> exhibited moderate inhibitory activity against CDK-2 with an IC<sub>50</sub> value of 0.324â€‰Î¼M<sup>15</sup>. Compound <b>1</b> also exhibited inhibitory activity against CDK-1/4. The study of structureâ€"activity relationship of compound <b>1</b> revealed improvement in CDK-2 inhibitory activity on replacement of the ketone oxygen by sulfur<a href="#CIT0015"><sup>15</sup></a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Multi-CDK inhibitors <b>1</b>â€"<b>5</b> bearing similar pharmacophoric features.</p>   </div> <p>Compound <b>2</b> was reported among a series of pyrazole derivatives <b>2â€"4</b> with CDK-1/2 inhibitory activities<a href="#CIT0016"><sup>16</sup></a>. Compound <b>2</b> displayed inhibitory activity against CDK-2 with an IC<sub>50</sub> value of 140â€‰nM. Substitution on the benzoyl ring in compound <b>2</b> with 2,6-difluoro groups afforded compound <b>3</b> with a 46-fold increase in CDK-2 inhibitory activity, <a href="#F0001">Figure 1</a>.</p> <p>Moreover, compound <b>4</b> (AT7519) was obtained in an attempt to optimise the anticancer activity compound <b>3</b>. However, compound <b>4</b> exhibited weaker inhibitory activity against CDK-2 (IC<sub>50</sub> = 47â€‰nM) compared to the parent compound <b>3</b> which indicates that the aromatic 4-fluorophenyl moiety is favoured for CDKs inhibition<a href="#CIT0016"><sup>16</sup></a>, <a href="#F0001">Figure 1</a>. Mechanistic study of compound <b>4</b> also revealed high inhibitory activity against CDK-9, while weaker activity was observed against CDKs 1, 3, 4, and 6<sup>17</sup>. Moreover, replacement of the 4-piperidinyl ring in compound <b>4</b> by the <i>N</i>-4-((2-aminophenyl)carbamoyl)benzyl moiety afforded compound <b>5</b> with higher inhibitory activities against CDK-1/2<sup>18</sup>.</p> <p>In addition, several CDKIs were designed bearing substituted urea moiety<a href="#CIT0019"><sup>19â€"22</sup></a>. The importance of this moiety in the formation of hydrogen bonding network with the kinase domain which improves inhibitory activities against CDKs was discussed in several reports<a href="#CIT0019"><sup>19</sup></a><sup>,</sup><a href="#CIT0021"><sup>21</sup></a><sup>,</sup><a href="#CIT0022"><sup>22</sup></a>. Honma etÂ al. investigated the inhibitory activities of several diaryl urea derivatives against CDK-4<sup>19</sup>. The results revealed higher kinase inhibitory activity for the derivatives with the bulky aryl moiety when substituted with H-bond donner (HB<sub>D</sub>)/acceptor (HB<sub>A</sub>) groups. Among these derivatives, compound <b>6</b> with the bulky 7-hydroxynaphthalen-1-yl moiety inhibited CDK-4 with an IC<sub>50</sub> value of 7.6â€‰Î¼M, <a href="#F0002">Figure 2</a>. Compound <b>7</b> (CDKi 277, <a href="#F0002">Figure 2</a>) is also a thiazolyl urea derivative which exhibited potent ATP competitive inhibitory activity against CDKs 1, 2, and 5 with IC<sub>50</sub> values of 8, 4, and 5â€‰nM, respectively<a href="#CIT0020"><sup>20</sup></a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Diary urea derivatives <b>6</b>â€"<b>9</b> with their inhibitory activities against CDKs.</p>   </div> <p>On the other hand, compound <b>8</b> was reported with moderate inhibitory activity against CDK-2 (IC<sub>50</sub> = 14.3â€‰Î¼M)<a href="#CIT0021"><sup>21</sup></a>. However, extending the chemical structure of the diaryl urea <b>9</b> with methyl piperazine and aminopyrimidine moieties allowed the new compound to occupy a larger volume of the active site of CDK-2, <a href="#F0002">Figure 2</a>. Accordingly, compound <b>9</b> exhibited higher inhibitory activity against CDK-2 than compound <b>8</b><sup>22</sup>.</p> <div class="rationaleanddesign" title="sec"><span class="label" tagx="label" title="label">1.1.</span><div class="title" tagx="title" title="title">
Rationale and design</div> <p>Previously, we reported compound <b>10</b> among a series of pyrrolizine-5-carboxamide derivatives with potent cytotoxic activity against MCF-7 cells<a href="#CIT0023"><sup>23</sup></a>. Although compound <b>10</b> was able to activate caspase 3/7 in MCF-7 cells, but the exact mechanism of action of this compound was not investigated. Recently, we reported different pyrrolizine derivatives with weak to moderate inhibitory activities against 20 oncogenic kinases<a href="#CIT0024"><sup>24</sup></a><sup>,</sup><a href="#CIT0025"><sup>25</sup></a>. Among these kinases, CDK-2 was the most affected one by all the tested compounds.</p> <p>Based on the above-mentioned data, the current study was performed to optimise the cytotoxic potential, study the structure-activity relationship (SAR), and investigate the mechanism of action of the pyrrolizine-5-carboxamide <b>10</b>. In this study, scaffold A (<a href="#F0003">Figure 3</a>) was designed bearing the pharmacophoric groups of the lead compound <b>10</b> and the multi-CDKIs <b>3</b><sup>17</sup>. In addition, a small library of 1302 urea/thiourea derivatives was generated through different structural modifications in scaffold A, <a href="https://doi.org/10.1080/14756366.2020.1837124">Supplementary data (Figures S1â€"S42)</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>Rational design of scaffold A and general structure of the compound library.</p>   </div> <p>In the last two decades, more than 500 crystal structure of CDKs were released in the protein data bank<a href="#CIT0026"><sup>26</sup></a><sup>,</sup><a href="#CIT0027"><sup>27</sup></a>. Among these, a large number of CDK-2 crystal structures bound to inhibitors of diverse chemical scaffolds were reported<a href="#CIT0027"><sup>27</sup></a>. The use of the <i>in silico</i> studies (virtual screening, docking studies and molecular dynamics) in the discovery and identification of new potent CDKs inhibitors was previously discussed in several reports<a href="#CIT0028"><sup>28â€"30</sup></a>. Among these techniques, virtual screening was used in several studies to design/identify novel CDK-2 inhibitors<a href="#CIT0031"><sup>31</sup></a><sup>,</sup><a href="#CIT0032"><sup>32</sup></a>. Poulsen etÂ al. reported a nitrogen-linked macrocyclic CDK-2 inhibitor using structure-based design and docking studies<a href="#CIT0033"><sup>33</sup></a>.</p> <p>Moreover, the pharmacophore-based virtual screening was used widely in different steps of the drug discovery process. This technique depends on the generation of a 3D pharmacophore model based on a set of active ligands, a targetâ€"ligand complex or the apo target. The generated pharmacophore can then be used in screening virtual libraries of molecules to select/optimise the lead compounds<a href="#CIT0034"><sup>34</sup></a>. The application of virtual screening was also succeeded in the identification of potent CDK-2 inhibitors of diverse chemical nature<a href="#CIT0033"><sup>33</sup></a><sup>,</sup><a href="#CIT0035"><sup>35</sup></a>.</p> <p>In the current work, a pharmacophore-based virtual screening of the compound library was performed using the 3â€‰D pharmacophore model of the multi-CDKI <b>3</b>. The virtual screening was done using Pharmit (<a href="http://pharmit.csb.pitt.edu/">http://pharmit.csb.pitt.edu/</a>)<a href="#CIT0036"><sup>36</sup></a>. First, the pdb file of compound <b>3</b> (LZ9) bound to CDK-2 (pdb: 2VTP) was selected in the user interface. After loading the protein file, the program identified all the pharmacophoric features in the ligand (LZ9) based on the recorded ligand-protein interactions. This features included the aromatic, hydrophobic, hydrogen bond donor/acceptor, and positively/negatively charged moieties, <a href="#F0004">Figure 4</a>. In the current study, five of the pharmacophoric features in compound <b>3</b> which participate in the interactions with amino acids in the active site of CDK-2 were selected to generate the pharmacophoric model. These features included the aromatic phenyl, pyrazole ring (hydrophobic), and imidazole N<sup>2</sup> (HB<sub>A1</sub>), the anilide NH (HB<sub>D</sub>), and the carbonyl oxygen (HB<sub>A2</sub>), <a href="#F0004">Figure 4</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>Compound <b>3</b> (LZ9) bound to CDK-2 (pdb 2VTP); (A) pharmacophoric features of compound <b>3</b>; (B) pharmacophoric features selected to generate the 3D pharmacophore model used in the virtual screening.</p>   </div> <p>In the virtual screening, the maximum number of rotatable bonds was set to 9, while the radius of the pharmacophoric feature were set to the default values. Compounds with molecular weights &amp;gt; 500 were excluded to avoid violations from Lipinskiâ€™s rule. The top-scoring hits were ranked based on RMSD in <a href="#t0001">Table 1</a>.</p> <div class="table-wrap_UNKNOWN" id="t0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>The top-scoring hits of the compound library based on RMSD.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Rank</th> <th align="center">Hit<sup>a</sup></th> <th align="center">RMSD</th> <th align="center">Rotatable bond</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1</td> <td align="left">Naphthalen-1-yl-ureido</td> <td align="char" char=".">0.463</td> <td align="char" char=".">8</td> </tr> <tr> <td align="left">2</td> <td align="left">Naphthalen-1-yl-ureido</td> <td align="char" char=".">0.516</td> <td align="char" char=".">8</td> </tr> <tr> <td align="left">3</td> <td align="left">3-Fluorophenyl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">4</td> <td align="left">4-Methoxyphenyl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">5</td> <td align="left">Cyclohexyl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">6</td> <td align="left">4-Fluorophenyl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">7</td> <td align="left">Naphthalen-2-yl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">8</td> <td align="left">4-Bromophenyl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">9</td> <td align="left">Naphthalen-1-yl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">10</td> <td align="left">Tert-butyl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">11</td> <td align="left">Naphthalen-1-yl-thioureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">12</td> <td align="left">Cyclopentyl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">13</td> <td align="left">Cycloheptyl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">14</td> <td align="left">4-Methoxyphenyl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> <tr> <td align="left">15</td> <td align="left">3-Bromophenyl-ureido</td> <td align="char" char=".">0.522</td> <td align="char" char=".">9</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>a</sup>Hits named according to their aryl urea/thiourea fragment.</p> </div> </div> </div> <p>The results of the virtual screening of the compound library revealed the best score for the naphthalen-1-yl bearing hits (R<sub>1</sub> = H, CH<sub>3</sub>). The top 15-scoring-hits also included the naphthalen-1-yl-thioureido, two 4-methoxyphenyl-ureido and two fluorophenyl-ureido hits with the same RMSD, <a href="#t0001">Table 1</a>.</p> <p>Based on the above results, the SAR study of scaffold A was achieved through the following modifications in compound <b>10</b> (<a href="#F0005">Figure 5</a>): (i) replacement of the 4-chlorophenyl moiety by naphthalen-1-yl moiety; (ii) replacement of the urea moiety in the naphthalenyl derivatives by thiourea; (iii) replacement of the chloro group in compound <b>10</b> with electron donating (R<sub>2</sub> = 4-OCH<sub>3</sub>) or the electron withdrawing (R<sub>2</sub> = 4-F) groups to study the impact of electronic effect of these substituents on cytotoxicity of the new compounds; (iv) replacement of the 4-chlorophenyl in compound <b>10</b> moiety by the aliphatic hexyl group to compare the impact of the aliphatic versus aromatic (phenyl/naphthalenyl) moieties on cytotoxic activity of the new compounds; (v) substitution on ring A with electron donating (R<sub>1</sub> = 4-CH<sub>3</sub>) or with electron withdrawing (R<sub>1</sub> = 4-Cl) to study the impact of these substituents on activity.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Virtual screening and structural modifications of scaffold A.</p>   </div> </div> </div> <div class="materialandmethods" title="sec"><span class="label" tagx="label" title="label">2.</span><div class="title" tagx="title" title="title">
Material and methods</div> <div class="chemistry" title="sec"><span class="label" tagx="label" title="label">2.1.</span><div class="title" tagx="title" title="title">
Chemistry</div> <p>Chemicals and reagents were purchased from Sigma-Aldrich, Darmstadt, Germany. All solvents were analytical grade and were used without purification. Melting points (m.p.) were determined by IA 9100MK digital melting point apparatus. TLC was used to check the purity of the new compounds. the IR spectra were recorded by BRUKER TENSOR 37 spectrophotometer using KBr disc. The IR spectra were expressed in wavenumber (cm<sup>âˆ’1</sup>). The <sup>1</sup>H-NMR, <sup>13â€‰</sup>C-NMR, and DEPT C<sup>135</sup> spectra were recorded in the Faculty of Pharmacy, Umm Al-Qura University, KSA using BRUKER AVANCE III at 500 and 125â€‰MHz, respectively. The <i>J</i> constant was given in Hz. Shimadzu GCMS QP5050A spectrometer (EI, 70â€‰eV, regional centre for mycology and biotechnology, Al-Azhar University, Cairo, Egypt) was used to perform the mass analyses of the new compounds. The quantitative elemental analyses were done in the microanalytical centre, Cairo University. Compounds <b>12</b><a href="#CIT0037"><sup>37</sup></a>, <b>14aâ€"c</b><a href="#CIT0038"><sup>38â€"40</sup></a>, <b>15aâ€"c</b><a href="#CIT0041"><sup>41</sup></a> were prepared according to the previous reports. Copies of spectral data including IR, <sup>1</sup>H-NMR, <sup>13â€‰</sup>C-NMR, <sup>13â€‰</sup>C-NMR, DEPT C<sup>135</sup>, and mass spectra of the new compounds are provided in supplementary (<a href="https://doi.org/10.1080/14756366.2020.1837124">Figures S43â€"S137</a>). To facilitate the identification of different protons/carbons and their chemical shifts in each of the new compounds, the numbering of the atoms was illustrated in scaffold A, <a href="#F0005">Figure 5</a>.</p> <div class="generalprocedureforthepreparationofcompounds16â€" title="sec"><span class="label" tagx="label" title="label">2.1.1.</span><div class="title" tagx="title" title="title">
General procedure for the preparation of compounds 16â€"20aâ€"c</div> <p>A mixture of the staring material (2â€‰mmol), appropriate isocyanate/isothiocyanate (2.2â€‰mmol), TEA (0.5â€‰ml) in DCM (30â€‰ml) was stirred at rt for 12â€‰h. The solvent was evaporated, and the solid product was dissolved in chloroform-acetone (1:1). The formed precipitate was filtered, washed with acetone, and dried to give white amorphous solid product.</p> <div class="7-cyano-6-(3-hexylureido)-n-phenyl-2,3-dihydro-1h-pyrrolizine-5-carboxamide(16a)" title="sec"><span class="label" tagx="label" title="label">2.1.1.1.</span><div class="title" tagx="title" title="title">
7-Cyano-6-(3-hexylureido)-<i>N</i>-phenyl-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (16a)</div> <p>The title compound was prepared from the reaction of compound <b>15a</b> (0.53â€‰g, 2â€‰mmol) and hexyl isocyanate (0.28â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>16a</b> was obtained as white amorphous solid product, m.p. 217â€"19â€‰Â°C, yield 57%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3351 (NHs), 3057 (aromatic Câ€"H), 2934, 2872 (aliphatic Câ€"H), 2225 (CN), 1708, 1646 (COs), 1599, 1538 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1489, 1379, 1318, 1229 (Câ€"N, Câ€"O). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 0.84 (t, 3H, <i>J</i>â€‰=â€‰5.4â€‰Hz, hexyl CH<sub>3</sub>), 1.29â€"1.23 (m, 6H, hexyl CH<sub>2</sub>-3â€‰+â€‰CH<sub>2</sub>-4â€‰+â€‰CH<sub>2</sub>-5), 1.44â€"1.40 (m, 2H, hexyl CH<sub>2</sub>-2), 2.47â€"2.42 (m, 2H, pyrrolizine CH<sub>2</sub>-2), 2.97 (t, 2H, <i>J</i>â€‰=â€‰7.3â€‰Hz, pyrrolizine CH<sub>2</sub>-1), 3.13 (q, 2H, <i>J</i>â€‰=â€‰5.9â€‰Hz, hexyl CH<sub>2</sub>-1), 4.27 (t, 2H, <i>J</i>â€‰=â€‰6.6â€‰Hz, pyrrolizine CH<sub>2</sub>-3), 6.77 (s, 1H, N<u>H</u>CH<sub>2</sub>-), 7.08 (t, 1H, 6.7â€‰Hz, Ph C<u>H</u>-4), 7.33 (t, 2H, <i>J</i>â€‰=â€‰7.6â€‰Hz, Ph CH-3â€‰+â€‰CH-5), 7.56 (d, 2H, <i>J</i>â€‰=â€‰7.1â€‰Hz, Ph CH-2â€‰+â€‰CH-6), 8.36 (s, 1H, pyrrolizine-N<u>H</u>CO), 10.42 (s, 1H, Ph-N<u>H</u>CO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 14.4 (CH<sub>3</sub>), 22.5 (hexyl CH<sub>2</sub>-5), 24.9 (pyrrolizine CH<sub>2</sub>-2), 25.6 (pyrrolizine CH<sub>2</sub>-1), 26.4 (hexyl CH<sub>2</sub>-3), 30.2 (hexyl CH<sub>2</sub>-2), 31.5 (hexyl CH<sub>2</sub>-4), 40.1 (hexyl CH<sub>2</sub>-1), 49.7 (pyrrolizine CH<sub>2</sub>-3), 84.3 (pyrrolizine C-7), 115.4 (CN), 119.3 (pyrrolizine C-5), 119.4 (Ph CH-2â€‰+â€‰CH-6), 124.0 (Ph CH-4), 128.7 (pyrrolizine C-7a), 129.4 (Ph CH-3â€‰+â€‰CH-5), 139.1 (pyrrolizine C-6), 146.0 (Ph C-1), 157.6 (NH<u>C</u>ONH), 157.9 (Ph-NH<u>C</u>O). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 14.4 (CH<sub>3</sub>), 22.5 (hexyl CH<sub>2</sub>-5), 24.9 (pyrrolizine CH<sub>2</sub>-2), 25.6 (pyrrolizine CH<sub>2</sub>-1), 26.4 (hexyl CH<sub>2</sub>-3), 30.2 (hexyl CH<sub>2</sub>-2), 31.5 (hexyl CH<sub>2</sub>-4), 40.1 (hexyl CH<sub>2</sub>-1), 49.7 (pyrrolizine CH<sub>2</sub>-3), 119.4 (Ph CH-2â€‰+â€‰CH-6), 124.0 (Ph CH-4), 129.4 (Ph CH-3â€‰+â€‰CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 395 ([Mâ€‰+â€‰2]<sup>+</sup>, 5), 394 ([Mâ€‰+â€‰1]<sup>+</sup>, 33), 393 ([M]<sup>+</sup>, 100), 392 ([M-1]<sup>+</sup>, 2), 292 (6), 266 (5), 117 (2), 91 (7), 77 (3). Anal. Calcd. for C<sub>22</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub> (393.48): C, 67.15; H, 6.92; N, 17.80. Found: C, 66.81; H, 7.34; N, 17.93.</p> </div> <div class="7-cyano-6-(3-hexylureido)-n-(p-tolyl)-2,3-dihydro-1h-pyrrolizine-5-carboxamide(16b)" title="sec"><span class="label" tagx="label" title="label">2.1.1.2.</span><div class="title" tagx="title" title="title">
7-Cyano-6-(3-hexylureido)-<i>N</i>-(<i>p</i>-tolyl)-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (16b)</div> <p>The title compound was prepared from the reaction of compound <b>15b</b> (0.56â€‰g, 2â€‰mmol) and hexyl isocyanate (0.28â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>16</b> was obtained as white amorphous solid product, m.p. 231â€"33â€‰Â°C, yield 51%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3314 (NHs), 2954, 2926, 2857 (aliphatic Câ€"H), 2224 (CN), 1708, 1669, 1643 (COs), 1600, 1538 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1433, 1318, 1243 (Câ€"N, Câ€"O). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 0.84 (t, 3H, <i>J</i>â€‰=â€‰5.3â€‰Hz, hexyl CH<sub>3</sub>), 1.25â€"1.21 (m, 6H, hexyl CH<sub>2</sub>-3â€‰+â€‰CH<sub>2</sub>-4â€‰+â€‰CH<sub>2</sub>-5), 1.47â€"1.38 (m, 2H, hexyl CH<sub>2</sub>-2), 2.26 (s, 3H, tolyl CH<sub>3</sub>), 2.48â€"2.40 (m, 2H, pyrrolizine CH<sub>2</sub>-2), 2.97 (t, 2H, <i>J</i>â€‰=â€‰6.0â€‰Hz, pyrrolizine CH<sub>2</sub>-1), 3.13 (q, 2H, <i>J</i>â€‰=â€‰6.7â€‰Hz, hexyl CH<sub>2</sub>-1), 4.26 (t, 2H, <i>J</i>â€‰=â€‰7.6â€‰Hz, pyrrolizine CH<sub>2</sub>-3), 6.76 (s, 1H, N<u>H</u>CH<sub>2</sub>-), 7.13 (d, 2H, <i>J</i>â€‰=â€‰7.1â€‰Hz, Ph CH-3â€‰+â€‰CH-5), 7.45 (d, 2H, <i>J</i>â€‰=â€‰7.0â€‰Hz, Ph CH-2â€‰+â€‰CH-6), 8.34 (s, 1H, pyrrolizine-N<u>H</u>CO), 10.32 (s, 1H, Ph-N<u>H</u>CO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 14.4 (hexyl CH<sub>3</sub>), 20.9 (tolyl CH<sub>3</sub>), 22.5 (hexyl CH<sub>2</sub>-5), 24.9 (pyrrolizine CH<sub>2</sub>-2), 25.6 (pyrrolizine CH<sub>2</sub>-1), 26.4 (hexyl CH<sub>2</sub>-3), 30.2 (hexyl CH<sub>2</sub>-2), 31.5 (hexyl CH<sub>2</sub>-4), 40.1 (hexyl CH<sub>2</sub>-1), 49.7 (pyrrolizine CH<sub>2</sub>-3), 84.3 (pyrrolizine C-7), 115.4 (CN), 119.4 (Ph CH-2â€‰+â€‰CH-6), 119.7 (pyrrolizine C-5), 128.5 (pyrrolizine C-7a), 129.7 (Ph CH-3â€‰+â€‰CH-5), 133.0 (Ph C-1), 136.6 (pyrrolizine C-6), 145.9 (Ph C-4), 157.6 (NH<u>C</u>ONH), 157.7 (Ph-NH<u>C</u>O). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 14.4 (hexyl CH<sub>3</sub>), 20.9 (tolyl CH<sub>3</sub>), 22.5 (hexyl CH<sub>2</sub>-5), 24.9 (pyrrolizine CH<sub>2</sub>-2), 25.6 (pyrrolizine CH<sub>2</sub>-1), 26.4 (hexyl CH<sub>2</sub>-3), 30.2 (hexyl CH<sub>2</sub>-2), 31.5 (hexyl CH<sub>2</sub>-4), 40.1 (hexyl CH<sub>2</sub>-1), 49.7 (pyrrolizine CH<sub>2</sub>-3), 119.4 (Ph CH-2â€‰+â€‰CH-6), 129.7 (Ph CH-3â€‰+â€‰CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 409 ([Mâ€‰+â€‰2]<sup>+</sup>, 9), 408 ([Mâ€‰+â€‰1]<sup>+</sup>, 35), 407 ([M]<sup>+</sup>, 100), 406 ([M-1]<sup>+</sup>, 4), 405 ([M-2]<sup>+</sup>, 1), 395 (2), 368 (3), 306 (25), 280 (3), 276 (3), 173 (3), 117 (12), 106 (3), 104 (12), 91 (9), 77 (25). Anal. Calcd. for C<sub>23</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> (407.51): C, 67.79; H, 7.17; N, 17.19. Found: C, 68.14; H, 6.86; N, 16.86.</p> </div> <div class="n-(4-chlorophenyl)-7-cyano-6-(3-hexylureido)-2,3-dihydro-1h-pyrrolizine-5-carboxamide(16c)" title="sec"><span class="label" tagx="label" title="label">2.1.1.3.</span><div class="title" tagx="title" title="title"><i>N</i>-(4-Chlorophenyl)-7-cyano-6-(3-hexylureido)-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (16c)</div> <p>The title compound was prepared from the reaction of compound <b>15c</b> (0.60â€‰g, 2â€‰mmol) and hexyl isocyanate (0.28â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>16c</b> was obtained as white amorphous solid product, m.p. 240â€"2â€‰Â°C, yield 54%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3265 (NHs), 2954, 2923, 2857 (aliphatic Câ€"H), 2222 (CN), 1672, 1641 (COs), 1597, 1567 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1492, 1429, 1315, 1241 (Câ€"N, Câ€"O). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 0.83 (t, 3H, <i>J</i>â€‰=â€‰5.7â€‰Hz, hexyl CH<sub>3</sub>), 1.23â€"1.19 (m, 6H, hexyl CH<sub>2</sub>-3â€‰+â€‰CH<sub>2</sub>-4â€‰+â€‰CH<sub>2</sub>-5), 1.44â€"1.36 (m, 2H, hexyl CH<sub>2</sub>-2), 2.47â€"2.42 (m, 2H, pyrrolizine CH<sub>2</sub>-2), 2.98 (t, 2H, <i>J</i>â€‰=â€‰6.4â€‰Hz, pyrrolizine CH<sub>2</sub>-1), 3.12 (q, 2H, <i>J</i>â€‰=â€‰5.9â€‰Hz, hexyl CH<sub>2</sub>-1), 4.25 (t, 2H, <i>J</i>â€‰=â€‰6.8â€‰Hz, pyrrolizine CH<sub>2</sub>-3), 6.77 (s, 1H, N<u>H</u>CH<sub>2</sub>-), 7.40 (d, 2H, <i>J</i>â€‰=â€‰7.8â€‰Hz, Ph CH-3â€‰+â€‰CH-5), 7.58 (d, 2H, <i>J</i>â€‰=â€‰7.6â€‰Hz, Ph CH-2â€‰+â€‰CH-6), 8.37 (s, 1H, pyrrolizine-N<u>H</u>CO), 10.55 (s, 1H, Ph-N<u>H</u>CO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 14.4 (hexyl CH<sub>3</sub>), 22.5 (hexyl CH<sub>2</sub>-5), 24.9 (pyrrolizine CH<sub>2</sub>-2), 25.6 (pyrrolizine CH<sub>2</sub>-1), 26.4 (hexyl CH<sub>2</sub>-3), 30.1 (hexyl CH<sub>2</sub>-2), 31.5 (hexyl CH<sub>2</sub>-4), 40.1 (hexyl CH<sub>2</sub>-1), 49.7 (pyrrolizine CH<sub>2</sub>-3), 84.3 (pyrrolizine C-7), 115.3 (CN), 118.9 (Ph C-4), 121.0 (Ph CH-2â€‰+â€‰CH-6), 127.5 (pyrrolizine C-5), 129.0 (pyrrolizine C-7a), 129.3 (Ph CH-3â€‰+â€‰CH-5), 138.1 (pyrrolizine C-6), 146.2 (Ph C-1), 157.5 (NH<u>C</u>ONH), 158.0 (Ph-NH<u>C</u>O). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 14.4 (hexyl CH<sub>3</sub>), 22.5 (hexyl CH<sub>2</sub>-5), 24.9 (pyrrolizine CH<sub>2</sub>-2), 25.6 (pyrrolizine CH<sub>2</sub>-1), 26.4 (hexyl CH<sub>2</sub>-3), 30.2 (hexyl CH<sub>2</sub>-2), 31.5 (hexyl CH<sub>2</sub>-4), 40.1 (hexyl CH<sub>2</sub>-1), 49.7 (pyrrolizine CH<sub>2</sub>-3), 121.0 (Ph CH-2â€‰+â€‰CH-6), 129.3 (Ph CH-3â€‰+â€‰CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 430 ([Mâ€‰+â€‰3]<sup>+</sup>, 9), 429 ([Mâ€‰+â€‰2]<sup>+</sup>, 30), 428 ([Mâ€‰+â€‰1]<sup>+</sup>, 33), 427 ([M]<sup>+</sup>, 100), 426 ([M-1]<sup>+</sup>, 1), 401 (2), 327 (8), 326 (17), 300 (4), 271 (2), 125 (2), 117 (3), 90 (4), 77 (2). Anal. Calcd. for C<sub>22</sub>H<sub>26</sub>ClN<sub>5</sub>O<sub>2</sub> (427.93): C, 61.75; H, 6.12; N, 16.37. Found: C, 62.08; H, 6.43; N, 16.78.</p> </div> <div class="7-cyano-6-(3-(4-methoxyphenyl)ureido)-n-phenyl-2,3-dihydro-1h-pyrrolizine-5-carboxamide(17a)" title="sec"><span class="label" tagx="label" title="label">2.1.1.4.</span><div class="title" tagx="title" title="title">
7-Cyano-6-(3-(4-methoxyphenyl)ureido)-<i>N</i>-phenyl-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (17a)</div> <p>The title compound was prepared from the reaction of compound <b>15a</b> (0.53â€‰g, 2â€‰mmol) and 4-methoxyphenyl isocyanate (0.33â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>17a</b> was obtained as white amorphous solid product, m.p. 226â€"8â€‰Â°C, yield 62%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3412, 3309, 3262 (NHs), 2836 (aliphatic Câ€"H), 2229 (CN), 1670, 1643 (COs), 1600, 1559 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1445, 1322, 1248 (Câ€"N, Câ€"O). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.48â€"2.44 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.00 (t, 2H, <i>J</i>â€‰=â€‰7.1â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 3.71 (s, 3H, OC<u>H<sub>3</sub></u>), 4.29 (t, 2H, <i>J</i>â€‰=â€‰7.4â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 6.86 (d, 2H, <i>J</i>â€‰=â€‰8.6â€‰Hz, MeO-Ph CH-3â€‰+â€‰CH-5), 7.08 (t, 1H, <i>J</i>â€‰=â€‰6.7â€‰Hz, Ph CH-4), 7.37â€"7.34 (m, 4H, Ph CH-3â€‰+â€‰CH-5+ MeO-Ph CH-2â€‰+â€‰CH-6), 7.58 (d, 2H, <i>J</i>â€‰=â€‰7.5â€‰Hz, Ph CH-2â€‰+â€‰CH-6), 8.49 (s, 1H, MeO-Ph-N<u>H</u>), 9.06 (s, 1H, pyrrolizine-N<u>H</u>CO), 9.94 (s, 1H, Ph-N<u>H</u>CO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 55.6 (O<u>C</u>H<sub>3</sub>), 84.5 (pyrrolizine C-7), 114.4 (MeO-Ph <u>C</u>H-3+<u>C</u>H-5), 118.1 (CN), 119.9 (MeO-Ph CH-2â€‰+â€‰CH-6), 121.0 (Ph CH-2â€‰+â€‰CH-6), 124.2 (Ph CH-4), 129.4 (Ph CH-3â€‰+â€‰CH-5), 132.8 (MeO-Ph C-1), 133.4 (pyrrolizine C-5), 139.0 (pyrrolizine C-7a), 146.3 (pyrrolizine C-6), 153.4 (Ph C-1), 154.5 (NHCONH), 155.3 (MeO-Ph C-4), 158.3 (Ph-NH<u>C</u>O). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 55.6 (O<u>C</u>H<sub>3</sub>), 114.4 (MeO-Ph <u>C</u>H-3+<u>C</u>H-5), 119.9 (MeO-Ph CH-2â€‰+â€‰CH-6), 121.0 (Ph CH-2â€‰+â€‰CH-6), 124.2 (Ph CH-4), 129.4 (Ph CH-3â€‰+â€‰CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 294 ([M-123 (4-methoxyaniline)]<sup>+</sup>, 2), 293 ([M-124]<sup>+</sup>, 6), 266 (1), 264 (2), 200 (6), 173 (7), 145 (3), 123 (17), 122 (5), 121 (3), 117 (29), 108 (41), 91 (29), 77 (28), 65 (100). Anal. Calcd. for C<sub>23</sub>H<sub>21</sub>N<sub>5</sub>O<sub>3</sub> (415.44): C, 66.49; H, 5.09; N, 16.86. Found: C, 66.73; H, 4.77; N, 17.11.</p> </div> <div class="7-cyano-6-(3-(4-methoxyphenyl)ureido)-n-(p-tolyl)-2,3-dihydro-1h-pyrrolizine-5-carboxamide(17b)" title="sec"><span class="label" tagx="label" title="label">2.1.1.5.</span><div class="title" tagx="title" title="title">
7-Cyano-6-(3-(4-methoxyphenyl)ureido)-<i>N</i>-(<i>p</i>-tolyl)-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (17b)</div> <p>The title compound was prepared from the reaction of compound <b>15â€‰b</b> (0.56â€‰g, 2â€‰mmol) and 4-methoxyphenyl isocyanate (0.33â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>17â€‰b</b> was obtained as white amorphous solid product, m.p. 234â€"6â€‰Â°C, yield 63%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3401, 3275 (NHs), 3000 (aromatic Câ€"H), 2954, 2857, 2835 (aliphatic Câ€"H), 2222 (CN), 1670, 1644 (COs), 1600, 1563 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1430, 1319, 1247 (C-N, C-O). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.26 (s, 3H, CH<sub>3</sub>), 2.48â€"2.43 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.00 (t, 2H, <i>J</i>â€‰=â€‰6.4â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 3.72 (s, 3H, OC<u>H<sub>3</sub></u>), 4.29 (t, 2H, <i>J</i>â€‰=â€‰7.0â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 6.89 (d, 2H, <i>J</i>â€‰=â€‰7.4â€‰Hz, MeO-Ph CH-3â€‰+â€‰CH-5), 7.14 (d, 2H, <i>J</i>â€‰=â€‰7.3â€‰Hz, tolyl CH-3â€‰+â€‰CH-5), 7.37 (d, 2H, <i>J</i>â€‰=â€‰7.3â€‰Hz, MeO-Ph CH-2â€‰+â€‰CH-6), 7.47 (d, 2H, <i>J</i>â€‰=â€‰7.1â€‰Hz, tolyl CH-2â€‰+â€‰CH-6), 8.47 (s, 1H, MeO-Ph-N<u>H</u>CO), 9.05 (s, 1H, pyrrolizine-NHCO), 9.86 (s, 1H, tolyl-NHCO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 20.9 (CH<sub>3</sub>), 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.6 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 55.6 (O<u>C</u>H<sub>3</sub>), 84.5 (pyrrolizine C-7), 114.5 (MeO-Ph <u>C</u>H-3+<u>C</u>H-5), 115.5 (CN), 118.2 (MeO-Ph C-1), 119.9 (MeO-Ph CH-2â€‰+â€‰CH-6), 121.0 (tolyl CH-2â€‰+â€‰CH-6), 128.9 (pyrrolizine C-5), 129.7 (tolyl <u>C</u>H-3+<u>C</u>H-5), 132.8 (pyrrolizine C-7a), 133.2 (tolyl C-1), 136.5 (pyrrolizine C-6), 146.2 (tolyl C-4), 154.5 (NHCONH), 155.3 (MeO-Ph <u>C</u>-4), 158.1 (tolyl-NHCO). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 20.9 (CH<sub>3</sub>), 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 55.6 (O<u>C</u>H<sub>3</sub>), 114.5 (MeO-Ph <u>C</u>H-3+<u>C</u>H-5), 119.9 (MeO-Ph CH-2â€‰+â€‰CH-6), 121.0 (tolyl CH-2â€‰+â€‰CH-6), 129.7 (tolyl <u>C</u>H-3+<u>C</u>H-5). MS (EI): <i>m</i>/<i>z</i> (%) 431 ([Mâ€‰+â€‰2]<sup>+</sup>, 3), 429 ([M]<sup>+</sup>, 13), 392 (3), 322 (23), 307 (23), 306 ([M-123 (4-methoxyaniline)]<sup>+</sup>, 23), 305 (41), 304 (9), 280 (7), 278 (37), 266 (5), 250 (8), 200 (7), 172 (10), 132 (13), 123 (2), 108 (8), 92 (17), 77 (46), 65 (100). Anal. Calcd. for C<sub>24</sub>H<sub>23</sub>N<sub>5</sub>O<sub>3</sub> (429.47): C, 67.12; H, 5.40; N, 16.31. Found: C, 66.75; H, 4.89; N, 16.78.</p> </div> <div class="n-(4-chlorophenyl)-7-cyano-6-(3-(4-methoxyphenyl)ureido)-2,3-dihydro-1h-pyrrolizine-5-carboxamide(17c)" title="sec"><span class="label" tagx="label" title="label">2.1.1.6.</span><div class="title" tagx="title" title="title"><i>N</i>-(4-Chlorophenyl)-7-cyano-6-(3-(4-methoxyphenyl)ureido)-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (17c)</div> <p>The title compound was prepared from the reaction of compound <b>15c</b> (0.60â€‰g, 2â€‰mmol) with 4-methoxyphenyl isocyanate (0.33â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>17c</b> was obtained as white amorphous solid product, m.p. 245â€"7â€‰Â°C, yield 69%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3294, 3163 (NHs), 3088 (aromatic Câ€"H), 2212 (CN), 1663 (COs), 1631, 1605 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1489, 1403, 1322, 1210 (Câ€"N, Câ€"O), 790 (Câ€"Cl). <sup>1</sup>H-NMR (CDCl<sub>3</sub>, 500â€‰MHz, ppm) <i>Î´</i>: <i>Î´</i> 2.56â€"2.50 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 2.98 (t, 2H, <i>J</i>â€‰=â€‰7.5â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 3.80 (s, 3H, 4-OC<u>H<sub>3</sub></u>), 4.39 (t, 2H, <i>J</i>â€‰=â€‰7.2â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 6.20 (s, 1H, NH), 6.88 (d, 2H, <i>J</i>â€‰=â€‰8,9â€‰Hz, MeO-Ph CH-3â€‰+â€‰CH-5), 7.31â€"7.22 (m, 5H, NH, Cl-Ph CH-3â€‰+â€‰CH-5, MeO-Ph CH-2â€‰+â€‰CH-6â€‰+â€‰pyrrolizine-NHCO), 7.53 (d, 2H, <i>J</i> -= 8.8â€‰Hz, Cl-Ph CH-2â€‰+â€‰CH-6), 9.62 (s, 1H, Cl-Ph-NH). <sup>13â€‰</sup>C-NMR (CDCl<sub>3</sub>, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 55.7 (O<u>C</u>H<sub>3</sub>), 84.6 (pyrrolizine C-7), 114.6 (MeO-Ph <u>C</u>H-3+<u>C</u>H-5), 115.6 (CN), 117.0 (MeO-Ph C-1), 120.4 (MeO-Ph CH-2â€‰+â€‰CH-6), 120.9 (Cl-Ph CH-2â€‰+â€‰CH-6), 129.1 (4-Cl-Ph C-4), 129.8 (4-Cl-Ph <u>C</u>H-3+<u>C</u>H-5), 131.6 (pyrrolizine C-5), 133.0 (pyrrolizine C-7a), 136.7 (pyrrolizine C-6), 146.6 (4-Cl-Ph), 153.6 (NHCONH), 155.9 (MeO-Ph <u>C</u>-4), 159.0 (4-Cl-Ph-NH<u>C</u>O). DEPT C<sup>135</sup> (CDCl<sub>3</sub>, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 55.7 (O<u>C</u>H<sub>3</sub>), 114.6 (MeO-Ph <u>C</u>H-3+<u>C</u>H-5), 120.4 (MeO-Ph CH-2â€‰+â€‰CH-6), 120.9 (Cl-Ph CH-2â€‰+â€‰CH-6), 129.8 (4-Cl-Ph <u>C</u>H-3+<u>C</u>H-5). MS (EI): <i>m</i>/<i>z</i> (%) 451 ([Mâ€‰+â€‰2]<sup>+</sup>, 4), 450 ([Mâ€‰+â€‰1]<sup>+</sup>, 5), 447 ([M-2]<sup>+</sup>, 2), 445 (4), 434 (5), 406 (3), 395 (8), 368 (10), 352 (6), 301 (6), 273 (44), 272 (100), 271 (14), 248 (3), 134 (6), 123 (62), 122 (7), 108 (78), 95 (17), 91 (4), 77 (20). Anal. Calcd. for C<sub>23</sub>H<sub>20</sub>ClN<sub>5</sub>O<sub>3</sub> (449.89): C, 61.40; H, 4.48; N, 15.57. Found: C, 60.96; H, 4.81; N, 15.78.</p> </div> <div class="7-cyano-6-(3-(4-fluorophenyl)ureido)-n-phenyl-2,3-dihydro-1h-pyrrolizine-5-carboxamide(18a)" title="sec"><span class="label" tagx="label" title="label">2.1.1.7.</span><div class="title" tagx="title" title="title">
7-Cyano-6-(3-(4-fluorophenyl)ureido)-<i>N</i>-phenyl-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (18a)</div> <p>The title compound was prepared from the reaction of compound <b>15a</b> (0.53â€‰g, 2â€‰mmol) and 4-fluorophenyl isocyanate (0.3â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>18a</b> was obtained as white amorphous solid product, m.p. 242â€"4â€‰Â°C, yield 65%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3402, 3366, 3336, 3280 (NHs), 3058, 3043 (aromatic Câ€"H), 2969, 2895 (aliphatic Câ€"H), 2221 (CN), 1655 (COs), 1601, 1549 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1448, 1321, 1249 (Câ€"N, Câ€"O, Câ€"F). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.48â€"2.44 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.01 (t, 2H, <i>J</i>â€‰=â€‰7.1â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 4.30 (t, 2H, <i>J</i>â€‰=â€‰6.8â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 7.08 (t, 1H, <i>J</i>â€‰=â€‰6.9â€‰Hz, Ph C<u>H</u>-4), 7.14 (t, 2H, <i>J</i>â€‰=â€‰8.2â€‰Hz, F-Ph C<u>H</u>-2â€‰+â€‰C<u>H</u>-6), 7.33 (t, 2H, <i>J</i>â€‰=â€‰7.1â€‰Hz, Ph C<u>H</u>-3â€‰+â€‰C<u>H</u>-5), 7.47 (d, 2H, <i>J</i>â€‰=â€‰8.5â€‰Hz, F-Ph C<u>H</u>-3â€‰+â€‰C<u>H</u>-5), 7.59 (d, 2H, <i>J</i>â€‰=â€‰7.6â€‰Hz, Ph C<u>H</u>-2â€‰+â€‰C<u>H</u>-6), 8.55 (s, 1H, pyrrolizine-N<u>H</u>CO), 9.29 (s, 1H, F-Ph-NHCO), 9.84 (s, 1H, Ph-NHCO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 84.6 (pyrrolizine C-7), 115.4 (CN), 115.8 (d, <i>J</i>â€‰=â€‰22.3â€‰Hz, F-Ph <u>C</u>H-3+<u>C</u>H-5), 117.8 (pyrrolizine C-5), 120.0 (Ph CH-2â€‰+â€‰CH-6), 120.8 (d, <i>J</i>â€‰=â€‰7.7â€‰Hz, F-Ph CH-2â€‰+â€‰CH-6), 124.2 (Ph CH-4), 129.1 (pyrrolizine C-7a), 129.3 (Ph CH-3â€‰+â€‰CH-5), 136.2 (d, <i>J</i>â€‰=â€‰2.7â€‰Hz, F-Ph C-1), 139.0 (Ph CH-1), 146.3 (pyrrolizine C-6), 154.2 (NHCONH), 158.0 (d, <i>J</i>â€‰=â€‰238.4â€‰Hz, F-Ph C-4), 158.3 (PhNH<u>C</u>O). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 115.8 (d, <i>J</i>â€‰=â€‰22.3â€‰Hz, F-Ph <u>C</u>H-3+<u>C</u>H-5), 120.0 (Ph CH-2â€‰+â€‰CH-6), 120.8 (d, <i>J</i>â€‰=â€‰7.7â€‰Hz, F-Ph CH-2â€‰+â€‰CH-6), 124.2 (Ph CH-4), 129.3 (Ph CH-3â€‰+â€‰CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 402 ([M-1]<sup>+</sup>, 2), 377 (4), 376 (17), 375 (50), 374 (17), 340 (4), 316 (32), 315 (22), 314 (100), 267 (2), 187 (4), 156 (2), 79 (5), 77 (4). Anal. Calcd. for C<sub>22</sub>H<sub>18</sub>FN<sub>5</sub>O<sub>2</sub> (403.41): C, 65.50; H, 4.50; N, 17.36. Found: C, 65.83; H, 4.91; N, 17.62.</p> </div> <div class="7-cyano-6-(3-(4-fluorophenyl)ureido)-n-(p-tolyl)-2,3-dihydro-1h-pyrrolizine-5-carboxamide(18b)" title="sec"><span class="label" tagx="label" title="label">2.1.1.8.</span><div class="title" tagx="title" title="title">
7-Cyano-6-(3-(4-fluorophenyl)ureido)-<i>N</i>-(<i>p</i>-tolyl)-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (18b)</div> <p>The title compound was prepared from the reaction of compound <b>15b</b> (0.56â€‰g, 2â€‰mmol) with 4-fluorophenyl isocyanate (0.3â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>18b</b> was obtained as white amorphous solid product, m.p. 247â€"9â€‰Â°C, yield 72%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3405, 3329, 3292 (NHs), 3077, 3037, 3001 (aromatic C-H), 2967, 2919, 2868 (aliphatic Câ€"H), 2222 (CN), 1652 (COs), 1600, 1547 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1435, 1320, 1210 (Câ€"N, Câ€"O, Câ€"F). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.26 (s, 3H, CH<sub>3</sub>), 2.48â€"2.44 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.00 (t, 2H, <i>J</i>â€‰=â€‰6.4â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 4.29 (t, 2H, <i>J</i>â€‰=â€‰7.3â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 7.15â€"7.12 (m, 4H, tolyl CH-3â€‰+â€‰CH-5 and F-Ph CH-3â€‰+â€‰CH-5), 7.48â€"7.46 (m, 4H, tolyl CH-2â€‰+â€‰CH-6 and F-Ph CH-2â€‰+â€‰CH-6), 8.57 (s, 1H, F-Ph-NHCO), 9.29 (s, 1H, pyrrolizine-N<u>H</u>CO), 9.74 (s, 1H, tolyl-N<u>H</u>CO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 20.9 (CH<sub>3</sub>), 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 84.6 (pyrrolizine C-7), 115.4 (CN), 115.8 (d, <i>J</i>â€‰=â€‰22.3â€‰Hz, F-Ph <u>C</u>H-3+<u>C</u>H-5), 118.0 (pyrrolizine C-5), 120.0 (tolyl CH-2â€‰+â€‰CH-6), 120.8 (d, <i>J</i>â€‰=â€‰7.7â€‰Hz, F-Ph CH-2â€‰+â€‰CH-6), 128.9 (pyrrolizine C-7a), 129.7 (tolyl CH-3â€‰+â€‰CH-5), 133.2 (tolyl C-1), 136.2 (d, <i>J</i>â€‰=â€‰2.1â€‰Hz, F-Ph C-1), 136.4 (pyrrolizine C-6), 146.2 (tolyl C-4), 154.2 (NHCONH), 158.0 (d, <i>J</i>â€‰=â€‰238.9â€‰Hz, F-Ph C-4), 158.1 (tolyl-NHCO). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 20.9 (CH<sub>3</sub>), 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 115.8 (d, <i>J</i>â€‰=â€‰22.3â€‰Hz, F-Ph <u>C</u>H-3+<u>C</u>H-5), 120.0 (tolyl CH-2â€‰+â€‰CH-6), 120.7 (d, <i>J</i>â€‰=â€‰7.7â€‰Hz, F-Ph CH-2â€‰+â€‰CH-6), 129.7 (tolyl CH-3â€‰+â€‰CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 418 ([Mâ€‰+â€‰1]<sup>+</sup>, 2), 417 ([M]<sup>+</sup>, 6), 308 (4), 307 (20), 306 (82), 305 (9), 292 (6), 280 (8), 200 (16), 173 (11), 132 (9), 111 (54), 91 (43), 77 (92), 65 (100). Anal. Calcd. for C<sub>23</sub>H<sub>20</sub>FN<sub>5</sub>O<sub>2</sub> (417.44): C, 66.18; H, 4.83; N, 16.78. Found: C, 65.72; H, 4.43; N, 16.65.</p> </div> <div class="n-(4-chlorophenyl)-7-cyano-6-(3-(4-fluorophenyl)ureido)-2,3-dihydro-1h-pyrrolizine-5-carboxamide(18c)" title="sec"><span class="label" tagx="label" title="label">2.1.1.9.</span><div class="title" tagx="title" title="title"><i>N</i>-(4-Chlorophenyl)-7-cyano-6-(3-(4-fluorophenyl)ureido)-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (18c)</div> <p>The title compound was prepared from the reaction of compound <b>15c</b> (0.60â€‰g, 2â€‰mmol) with 4-fluorophenyl isocyanate (0.3â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>18c</b> was obtained as white amorphous solid product, m.p. 261â€"3â€‰Â°C, yield 75%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3403, 3325, 3278 (NHs), 3062, 3046 (aromatic Câ€"H), 2993, 2970 (aliphatic Câ€"H), 2224 (CN), 1666 (COs), 1598, 1551 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1431, 1315, 1233 (Câ€"N, Câ€"O, Câ€"F), 774 (Câ€"Cl). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.48â€"2.44 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.01 (t, 2H, <i>J</i>â€‰=â€‰6.7â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 4.29 (t, 2H, <i>J</i>â€‰=â€‰6.9â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 7.13 (t, 2H, <i>J</i>â€‰=â€‰7.8â€‰Hz, F-Ph CH-3â€‰+â€‰CH-5), 7.40 (d, 2H, <i>J</i>â€‰=â€‰7.5â€‰Hz, Cl-Ph CH-3â€‰+â€‰CH-5), 7.48â€"7.43 (m, 2H, F-Ph CH-2â€‰+â€‰CH-6), 7.62 (d, 2H, <i>J</i>â€‰=â€‰7.4â€‰Hz, Cl-Ph CH-2â€‰+â€‰CH-6), 8.56 (s, 1H, F-PH-N<u>H</u>CO), 9.27 (s, 1H, pyrrolizine-N<u>H</u>CO), 9.94 (s, 1H, Cl-Ph-N<u>H</u>CO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.6 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 84.5 (pyrrolizine C-7), 115.4 (CN), 115.8 (d, <i>J</i>â€‰=â€‰22.3â€‰Hz, F-Ph <u>C</u>H-3+<u>C</u>H-5), 117.3 (Cl-Ph C-4), 120.7 (d, <i>J</i>â€‰=â€‰7.7â€‰Hz, F-Ph CH-2â€‰+â€‰CH-6), 121.7 (Cl-Ph CH-2â€‰+â€‰CH-6), 127.7 (pyrrolizine C-5), 129.2 (Cl-Ph CH-3â€‰+â€‰CH-5), 129.5 (pyrrolizine C-7a), 136.2 (d, <i>J</i>â€‰=â€‰2.4â€‰Hz, F-Ph C-1), 138.0 (pyrrolizine C-6), 146.5 (Cl-Ph C-1), 153.9 (NHCONH), 158.0 (d, <i>J</i>â€‰=â€‰238.0â€‰Hz, F-Ph C-4), 158.4 (Cl-Ph-NHCO). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.6 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 115.8 (d, <i>J</i>â€‰=â€‰22.3â€‰Hz, F-Ph <u>C</u>H-3+<u>C</u>H-5), 120.7 (d, <i>J</i>â€‰=â€‰7.7â€‰Hz, F-Ph CH-2â€‰+â€‰CH-6), 121.7 (Cl-Ph CH-2â€‰+â€‰CH-6), 129.2 (Cl-Ph CH-3â€‰+â€‰CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 438 ([Mâ€‰+â€‰1]<sup>+</sup>, 5), 437 ([M]<sup>+</sup>, 15), 436 ([M-1]<sup>+</sup>, 2), 404 (6), 403 (14), 402 (2), 293 (14), 292 (82), 291 (7), 262 (2), 173 (5), 145 (3), 119 (15), 91 (43), 77 (37), 64 (100). Anal. Calcd. for C<sub>22</sub>H<sub>17</sub>ClFN<sub>5</sub>O<sub>2</sub> (437.85): C, 60.35; H, 3.91; N, 15.99. Found: C, 59.92; H, 4.27; N, 16.22.</p> </div> <div class="7-cyano-6-(3-(naphthalen-1-yl)ureido)-n-phenyl-2,3-dihydro-1h-pyrrolizine-5-carboxamide(19a)" title="sec"><span class="label" tagx="label" title="label">2.1.1.10.</span><div class="title" tagx="title" title="title">
7-Cyano-6-(3-(naphthalen-1-yl)ureido)-<i>N</i>-phenyl-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (19a)</div> <p>The title compound was prepared from the reaction of compound <b>15a</b> (0.53â€‰g, 2â€‰mmol) with 1-naphthyl isocyanate (0.37â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>19a</b> was obtained as white amorphous solid product, m.p. 258â€"61â€‰Â°C, yield 74%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3393, 3262 (NHs), 3047 (aromatic Câ€"H), 2987, 2951 (aliphatic Câ€"H), 2223 (CN), 1671, 1640 (COs), 1553, 1528 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1444, 1319, 1246 (Câ€"N, Câ€"O). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.48â€"2.44 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.02 (t, 2H, <i>J</i>â€‰=â€‰6.5â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 4.30 (t, 2H, <i>J</i>â€‰=â€‰7.5â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 7.08 (t, 1H, <i>J</i>â€‰=â€‰7.3â€‰Hz, Ph CH-4), 7.32 (t, 2H, <i>J</i>â€‰=â€‰6.4â€‰Hz, Ph C<u>H</u>-3â€‰+â€‰C<u>H</u>-5), 7.53-7.48 (m, 2H, naphthalenyl CH-2â€‰+â€‰pyrrolizine NH), 7.61 (d, 2H, <i>J</i>â€‰=â€‰7.1â€‰Hz, Ph CH-2â€‰+â€‰CH-6), 7.66 (t, 1H, <i>J</i>â€‰=â€‰8.8â€‰Hz, naphthalenyl CH-3), 7.71 (d, 1H, <i>J</i>â€‰=â€‰8.1â€‰Hz, naphthalenyl CH-4), 7.87 (d, 1H, <i>J</i>â€‰=â€‰7.7â€‰Hz, naphthalenyl CH-5), 7.96 (t, 1H, <i>J</i>â€‰=â€‰8.6â€‰Hz, naphthalenyl CH-6), 8.11 (t, 1H, <i>J</i>â€‰=â€‰7.5â€‰Hz, naphthalenyl CH-7), 8.26 (d, 1H, <i>J</i>â€‰=â€‰6.8â€‰Hz, naphthalenyl CH-8), 8.89 (s, 1H, naphthalenyl-NHCO), 9.96 (s, 1H, Ph-NHCO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 84.5 (pyrrolizine C-7), 117.9 (naphthalenyl CH-2), 118.1 (CN), 120.0 (Ph CH-2â€‰+â€‰CH-6), 121.9 (naphthalenyl CH-4), 122.3 (naphthalenyl CH-8), 124.1 (naphthalenyl CH-7), 124.5 (naphthalenyl CH-6), 126.3 (naphthalenyl CH-3), 126.5 (Ph CH-4), 127.3 (naphthalenyl C-8a), 128.8 (naphthalenyl CH-5), 129.0 (pyrrolizine C-5), 129.3 (Ph CH-3â€‰+â€‰CH-5), 134.2 (naphthalenyl C-4a), 134.5 (pyrrolizine C-7a), 134.8 (pyrrolizine C-6), 139.0 (Ph C-1), 146.2 (naphthalenyl C-1), 154.9 (NHCONH), 158.3 (Ph-NHCO). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 117.9 (naphthalenyl CH-2), 120.0 (Ph CH-2â€‰+â€‰CH-6), 121.9 (naphthalenyl CH-4), 122.3 (naphthalenyl CH-8), 124.1 (naphthalenyl CH-7), 124.5 (naphthalenyl CH-6), 126.3 (naphthalenyl CH-3), 126.5 (Ph CH-4), 128.8 (naphthalenyl CH-5), 129.3 (Ph CH-3â€‰+â€‰CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 436 ([Mâ€‰+â€‰1]<sup>+</sup>, 7), 432 ([M-3]<sup>+</sup>, 1), 424 (2), 412 (2), 331 (3), 312 (8), 287 (4), 189 (2), 178 (3), 169 (6), 143 (51), 116 (16), 115 (100), 114 (16), 91 (4), 90 (6), 75 (5). Anal. Calcd. for C<sub>26</sub>H<sub>21</sub>N<sub>5</sub>O<sub>2</sub> (435.48): C, 71.71; H, 4.86; N, 16.08. Found: C, 72.12; H, 4.54; N, 16.25.</p> </div> <div class="7-cyano-6-(3-(naphthalen-1-yl)ureido)-n-(p-tolyl)-2,3-dihydro-1h-pyrrolizine-5-carboxamide(19b)" title="sec"><span class="label" tagx="label" title="label">2.1.1.11.</span><div class="title" tagx="title" title="title">
7-Cyano-6-(3-(naphthalen-1-yl)ureido)-<i>N</i>-(<i>p</i>-tolyl)-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (19b)</div> <p>The title compound was prepared from the reaction of compound <b>15b</b> (0.56â€‰g, 2â€‰mmol) with 1-naphthyl isocyanate (0.37â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>19â€‰b</b> was obtained as white amorphous solid product, m.p. 267â€"9â€‰Â°C, yield 76%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3406, 3281 (NHs), 3050 (aromatic Câ€"H), 2919, 2867 (aliphatic Câ€"H), 2224 (CN), 1671, 1642 (COs), 1553, 1528 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1433, 1390, 1298 (Câ€"N, Câ€"O). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.26 (s, 3H, CH<sub>3</sub>), 2.48â€"2.44 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.01 (t, 2H, <i>J</i>â€‰=â€‰6.4â€‰Hz, pyrrolizine <u>H<sub>2</sub></u>-1), 4.29 (t, 2H, <i>J</i>â€‰=â€‰7.6â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 7.12 (d, 2H, <i>J</i>â€‰=â€‰7.7â€‰Hz, Ph C<u>H</u>-3â€‰+â€‰C<u>H</u>-5), 7.49 (d, 2H, <i>J</i>â€‰=â€‰8.1â€‰Hz, Ph C<u>H</u>-2â€‰+â€‰C<u>H</u>-6), 7.60â€"7.55 (m, 2H, naphthalenyl CH-2â€‰+â€‰pyrrolizine-NH), 7.66 (t, 1H, <i>J</i>â€‰=â€‰7.7â€‰Hz, naphthalenyl CH-3), 7.71 (d, 1H, <i>J</i>â€‰=â€‰8.0â€‰Hz, naphthalenyl CH-4), 7.86 (d, 1H, <i>J</i>â€‰=â€‰7.3â€‰Hz, naphthalenyl CH-6), 7.96 (t, 1H, <i>J</i>â€‰=â€‰7.9â€‰Hz, naphthalenyl CH-7), 8.12â€"8.09 (m, naphthalenyl CH-5), 8.26 (d, 1H, <i>J</i>â€‰=â€‰8.5â€‰Hz, naphthalenyl CH-8), 8.88 (s, 1H, naphthalenyl-NHCO), 9.87 (s, 1H, Ph-NHCO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 20.9 (CH<sub>3</sub>), 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 84.5 (pyrrolizine C-7), 115.5 (CN), 117.9 (naphthalenyl CH-2), 120.0 (Ph CH-2â€‰+â€‰CH-6), 121.9 (naphthalenyl CH-4), 123.4 (naphthalenyl CH-8), 124.4 (naphthalenyl CH-7), 126.2 (naphthalenyl C-8a), 126.3 (naphthalenyl CH-6), 126.4 (naphthalenyl CH-3), 128.8 (pyrrolizine C-5), 129.0 (naphthalenyl CH-5), 129.7 (Ph CH-3â€‰+â€‰CH-5), 133.2 (naphthalenyl C-4a), 134.2 (pyrrolizine C-7a), 134.5 (Ph C-1), 134.8 (pyrrolizine C-6), 136.5 (Ph C-4), 146.1 (naphthalenyl C-1), 155.0 (NHCONH), 158.1 (Ph-NHCO). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 20.9 (CH<sub>3</sub>), 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 117.9 (naphthalenyl CH-2), 120.0 (Ph CH-2â€‰+â€‰CH-6), 121.9 (naphthalenyl CH-4), 123.4 (naphthalenyl CH-8), 124.4 (naphthalenyl CH-7), 126.3 (naphthalenyl CH-6), 126.4 (naphthalenyl CH-3), 129.0 (naphthalenyl CH-5), 129.7 (Ph CH-3â€‰+â€‰CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 450 ([Mâ€‰+â€‰1]<sup>+</sup>, 5), 449 ([M]<sup>+</sup>, 8), 448 ([M-1]<sup>+</sup>, 2), 439 (5), 407 (3), 375 (3), 316 (8), 306 (49), 276 (3), 198 (3), 172 (4), 143 (16), 132 (16), 117 (33), 104 (17), 91 (17), 77 (88), 65 (100). Anal. Calcd. for C<sub>27</sub>H<sub>23</sub>N<sub>5</sub>O<sub>2</sub> (449.50): C, 72.14; H, 5.16; N, 15.58. Found: C, 71.72; H, 4.86; N, 16.05.</p> </div> <div class="n-(4-chlorophenyl)-7-cyano-6-(3-(naphthalen-1-yl)ureido)-2,3-dihydro-1h-pyrrolizine-5-carboxamide(19c)" title="sec"><span class="label" tagx="label" title="label">2.1.1.12.</span><div class="title" tagx="title" title="title"><i>N</i>-(4-Chlorophenyl)-7-cyano-6-(3-(naphthalen-1-yl)ureido)-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (19c)</div> <p>The title compound was prepared from the reaction of compound <b>15c</b> (0.6â€‰g, 2â€‰mmol) with 1-naphthyl isocyanate (0.37â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>19c</b> was obtained as white amorphous solid product, m.p. 274â€"6â€‰Â°C, yield 72%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3396, 3282 (NHs), 3059, 3008 (aromatic Câ€"H), 2225 (CN), 1672, 1641 (COs), 1597, 1556 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1492, 1386, 1245 (Câ€"N, Câ€"O), 791 (Câ€"Cl). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.47â€"2.44 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.02 (t, 2H, <i>J</i>â€‰=â€‰6.1â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 4.29 (t, 2H, <i>J</i>â€‰=â€‰6.8â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 7.38 (d, 2H, <i>J</i>â€‰=â€‰6.8â€‰Hz, Ph C<u>H</u>-3â€‰+â€‰C<u>H</u>-5), 7.55â€"7.47 (m, 3H, naphthalenyl CH-2â€‰+â€‰CH-3â€‰+â€‰CH-6), 7.63 (d, 2H, <i>J</i>â€‰=â€‰6.8â€‰Hz, Ph C<u>H</u>-2â€‰+â€‰C<u>H</u>-6), 7.70 (d, 1H, <i>J</i>â€‰=â€‰8.1â€‰Hz, naphthalenyl CH-4), 7.84 (d, 1H, <i>J</i>â€‰=â€‰6.8â€‰Hz, naphthalenyl CH-5), 7.96 (t, 1H, <i>J</i>â€‰=â€‰8.2â€‰Hz, naphthalenyl CH-7), 8.11 (d, 1H, <i>J</i>â€‰=â€‰6.7â€‰Hz, naphthalenyl CH-8), 8.89 (s, 1H, pyrrolizine-NHCO), 9.22 (s, 1H, naphthalenyl-NHCO), 10.07 (s, 1H, Ph-NHCO). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 84.4 (pyrrolizine C-7), 115.5 (CN), 117.6 (naphthalenyl C-8a), 117.9 (naphthalenyl CH-2), 119.4 (naphthalenyl CH-4), 121.6 (Ph CH-2â€‰+â€‰CH-6), 122.2 (naphthalenyl CH-8), 124.4 (naphthalenyl CH-7), 126.3 (naphthalenyl CH-6), 126.5 (naphthalenyl CH-3), 127.2 (Ph C-4), 127.65 (pyrrolizine C-5), 128.8 (naphthalenyl CH-5), 129.2 (Ph CH-3â€‰+â€‰CH-5), 129.4 (naphthalenyl C-4a), 134.2 (pyrrolizine C-7a), 134.5 (pyrrolizine C-6), 138.1 (Ph C-1), 146.4 (naphthalenyl C-1), 154.7 (NHCONH), 158.4 (Ph-NHCO). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 117.9 (naphthalenyl CH-2), 119.4 (naphthalenyl CH-4), 121.6 (Ph CH-2â€‰+â€‰CH-6), 122.2 (naphthalenyl CH-8), 124.4 (naphthalenyl CH-7), 126.3 (naphthalenyl CH-6), 126.5 (naphthalenyl CH-3), 128.8 (naphthalenyl CH-5), 129.2 (Ph CH-3â€‰+â€‰CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 471 ([Mâ€‰+â€‰2]<sup>+</sup>, 1), 469 ([M]<sup>+</sup>, 3), 434 (2), 424 (2), 326 (3), 314 (6), 313 (21), 312 (100), 311 (19), 293 (2), 210 (2), 169 (2), 143 (10), 115 (54), 90 (6), 77 (25). Anal. Calcd. for C<sub>26</sub>H<sub>20</sub>ClN<sub>5</sub>O<sub>2</sub> (469.92): C, 66.45; H, 4.29; N, 14.90. Found: C, 66.12; H, 4.67; N, 15.32.</p> </div> <div class="7-cyano-6-(3-(naphthalen-1-yl)thioureido)-n-phenyl-2,3-dihydro-1h-pyrrolizine-5-carboxamide(20a)" title="sec"><span class="label" tagx="label" title="label">2.1.1.13.</span><div class="title" tagx="title" title="title">
7-Cyano-6-(3-(naphthalen-1-yl)thioureido)-<i>N</i>-phenyl-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (20a)</div> <p>The title compound was prepared from the reaction of compound <b>15a</b> (0.53â€‰g, 2â€‰mmol) with 1-naphthyl isothiocyanate (0.41â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>20a</b> was obtained as white amorphous solid product, m.p. 264â€"6â€‰Â°C, yield 69%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3339, 3168 (NHs), 3054 (aromatic C-H), 2964, 2919 (aliphatic Câ€"H), 2229 (CN), 1694 (COs), 1596, 1523 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1488, 1398, 1331, 1216, 1189 (Câ€"N, Câ€"O, Câ€‰=â€‰S). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.47â€"2.39 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.07 (t, 2H, <i>J</i>â€‰=â€‰6.8â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 4.21 (t, 2H, <i>J</i>â€‰=â€‰7.0â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 7.20 (d, 1H, <i>J</i>â€‰=â€‰7.3â€‰Hz, naphthalenyl CH), 7.32 (t, 1H, <i>J</i>â€‰=â€‰7.5â€‰Hz, naphthalenyl CH), 7.41 (d, 1H, <i>J</i>â€‰=â€‰7.6â€‰Hz, naphthalenyl CH), 7.47 (t, 2H, <i>J</i>â€‰=â€‰7.30â€‰Hz, Ph CH-3â€‰+â€‰CH-5), 7.63â€"7.52 (m, 6H, 5 aromatics Hsâ€‰+â€‰pyrrolizine-NH), 7.87 (d, 1H, <i>J</i>â€‰=â€‰8.0â€‰Hz, naphthalenyl H), 7.97 (d, 1H, <i>J</i>â€‰=â€‰7.9â€‰Hz, aromatic CH), 8.04 (d, 1H, <i>J</i>â€‰=â€‰8.3â€‰Hz, naphthalenyl H), 9.85 (s, 1H, Ph-NH), 13.76 (s, 1H, naphthalenyl-NH). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.8 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.3 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 77.4 (pyrrolizine C-7), 108.1 (CN), 110.4 (naphthalenyl C-8a), 120.4 (naphthalenyl CH-2), 123.7 (Ph CH-2â€‰+â€‰CH-6), 126.1 (naphthalenyl CH-4), 126.3 (naphthalenyl CH-8), 126.6 (naphthalenyl CH-7), 126.6 (pyrrolizine C-5), 127.4 (naphthalenyl CH-6), 128.6 (Ph CH-3â€‰+â€‰CH-5), 129.1 (naphthalenyl CH-3), 129.4 (Ph CH-4), 129.6 (naphthalenyl CH-5), 130.7 (naphthalenyl C-4a), 134.4 (pyrrolizine C-7a), 135.8 (pyrrolizine C-6), 139.5 (Ph C-1), 146.8 (naphthalenyl C-1), 159.9 (Ph-NHCO), 175.9 (NHCSNH). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.8 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.3 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.7 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 120.4 (naphthalenyl CH-2), 123.7 (Ph CH-2â€‰+â€‰CH-6), 126.1 (naphthalenyl CH-4), 126.3 (naphthalenyl CH-8), 126.6 (naphthalenyl CH-7), 127.4 (naphthalenyl CH-6), 128.6 (Ph CH-3â€‰+â€‰CH-5), 129.1 (naphthalenyl CH-3), 129.4 (Ph CH-4), 129.6 (naphthalenyl CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 452 ([Mâ€‰+â€‰1]<sup>+</sup>, 1), 436 ([M-15]<sup>+</sup>, 2), 434 (7), 432 (13), 395 (2), 375 (4), 314 (100), 296 (4), 285 (5), 268 (2), 251 (2), 188 (6), 91 (6), 77 (11). Anal. Calcd. for C<sub>26</sub>H<sub>21</sub>N<sub>5</sub>OS (451.54): C, 69.16; H, 4.69; N, 15.51. Found: C, 69.46; H, 4.82; N, 15.21.</p> </div> <div class="7-cyano-6-(3-(naphthalen-1-yl)thioureido)-n-(p-tolyl)-2,3-dihydro-1h-pyrrolizine-5-carboxamide(20b)" title="sec"><span class="label" tagx="label" title="label">2.1.1.14.</span><div class="title" tagx="title" title="title">
7-Cyano-6-(3-(naphthalen-1-yl)thioureido)-<i>N</i>-(p-tolyl)-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (20b)</div> <p>The title compound was prepared from the reaction of compound <b>15b</b> (0.56â€‰g, 2â€‰mmol) with 1-naphthyl isothiocyanate (0.41â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>20â€‰b</b> was obtained as white amorphous solid product, m.p. 279â€"81â€‰Â°C, yield 72%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3379, 3281, 3132 (NHs), 3059 (aromatic C-H), 2929, 2837 (aliphatic Câ€"H), 2225 (CN), 1664, (Câ€‰=â€‰O), 1599, 1567, 1539 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1459, 1376, 1237, 1151 (Câ€"N, Câ€"O, Câ€‰=â€‰S). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.36 (s, 3H, CH<sub>3</sub>), 2.52â€"2.49 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.07 (t, 2H, <i>J</i>â€‰=â€‰6.8â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 4.21 (t, 2H, <i>J</i>â€‰=â€‰6.8â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 7.05 (d, 1H, <i>J</i>â€‰=â€‰7.1â€‰Hz, naphthalenyl CH), 7.26 (d, 1H, <i>J</i>â€‰=â€‰7.4â€‰Hz, naphthalenyl CH), 7.45 (broad s, 1H, naphthalenyl CH), 7.61â€"7.51 (m, 4H, aromatic Hs), 7.86 (d, 1H, <i>J</i>â€‰=â€‰7.8â€‰Hz, naphthalenyl CH), 7.96 (d, 2H, <i>J</i>â€‰=â€‰8.4â€‰Hz, tolyl CH-2â€‰+â€‰CH-6), 8.06â€"8.02 (m, 2H, aromatic Hs), 9.83 (s, 1H, pyrrolizine-NH), 10.56 (s, 1H, tolyl-NH), 13.71 (broad s, 1H, naphthalenyl-NH). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 21.3 (CH<sub>3</sub>), 25.1 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 26.2 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 48.8 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 109.1 (pyrrolizine C-7), 110.4 (CN), 118.8 (naphthalenyl CH-2), 123.7 (tolyl CH-2â€‰+â€‰CH-6), 126.1 (naphthalenyl CH-4), 126.3 (naphthalenyl CH-8), 126.6 (naphthalenyl C-8a), 126.6 (naphthalenyl CH-7), 127.0 (pyrrolizine C-5), 127.3 (naphthalenyl CH-6), 128.6 (tolyl CH-3â€‰+â€‰CH-5), 129.3 (naphthalenyl CH-3), 129.9 (naphthalenyl CH-5), 130.7 (naphthalenyl C-5a), 131.9 ((pyrrolizine C-7a), 134.4 (tolyl C-1), 135.8 (pyrrolizine C-6), 136.1 (tolyl C-4), 145.6 (naphthalenyl C-1), 158.3 (tolyl-NHCO), 176.0 (NHCSNH). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 21.3 (CH<sub>3</sub>), 25.1 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 26.2 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 48.8 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 118.8 (naphthalenyl CH-2), 123.7 (tolyl CH-2â€‰+â€‰CH-6), 126.1 (naphthalenyl CH-4), 126.3 (naphthalenyl CH-8), 126.6 (naphthalenyl CH-7), 127.3 (naphthalenyl CH-6), 128.6 (tolyl CH-3â€‰+â€‰CH-5), 129.3 (naphthalenyl CH-3), 129.9 (naphthalenyl CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 466 ([Mâ€‰+â€‰1]<sup>+</sup>, 28), 465 ([M]<sup>+</sup>, 16), 462 ([M-3]<sup>+</sup>, 18), 446 (15), 424 (17), 404 (37), 382 (57), 369 (42), 359 (22), 347 (15), 266 (46), 236 (27), 218 (18), 185 (16), 174 (25), 160 (19), 143 (4), 119 (22), 92 (100), 77 (29). Anal. Calcd. for C<sub>27</sub>H<sub>23</sub>N<sub>5</sub>OS (465.57): C, 69.65; H, 4.98; N, 15.04. Found: C, 69.54; H, 4.72; N, 14.87.</p> </div> <div class="n-(4-chlorophenyl)-7-cyano-6-(3-(naphthalen-1-yl)thioureido)-2,3-dihydro-1h-pyrrolizine-5-carboxamide(20c)" title="sec"><span class="label" tagx="label" title="label">2.1.1.15.</span><div class="title" tagx="title" title="title"><i>N</i>-(4-chlorophenyl)-7-cyano-6-(3-(naphthalen-1-yl)thioureido)-2,3-dihydro-1<i>H</i>-pyrrolizine-5-carboxamide (20c)</div> <p>The title compound was prepared from the reaction of compound <b>15c</b> (0.6â€‰g, 2â€‰mmol) with 1-naphthyl isothiocyanate (0.41â€‰g, 2.2â€‰mmol) according to the general procedure A. Compound <b>20c</b> was obtained as white amorphous solid product, m.p. 287â€"9â€‰Â°C, yield 68%. IRÊ‹<sub>max</sub>/cm<sup>âˆ’1</sup> 3279, 3244, 3179 (NHs), 3053 (aromatic Câ€"H), 2929 (aliphatic Câ€"H), 2224 (CN), 1662 (Câ€‰=â€‰O), 1600, 1567, 1542 (Câ€‰=â€‰C, Câ€‰=â€‰N), 1491, 1332, 1230, 1149 (Câ€"N, Câ€"O, Câ€‰=â€‰S), 792 (Câ€"Cl). <sup>1</sup>H-NMR (DMSO-<i>d<sub>6</sub></i>, 500â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 2.49â€"2.41 (m, 2H, pyrrolizine C<u>H<sub>2</sub></u>-2), 3.07 (t, 2H, <i>J</i>â€‰=â€‰6.1â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-1), 4.31 (t, 2H, <i>J</i>â€‰=â€‰7.8â€‰Hz, pyrrolizine C<u>H<sub>2</sub></u>-3), 7.31â€"7.17 (m, 2H), 7.60â€"7.49 (m, 5H), 7.90 (d, 1H, <i>J</i>â€‰=â€‰7.9â€‰Hz, naphthalenyl CH-4), 8.00 (d, 1H, <i>J</i>â€‰=â€‰7.0â€‰Hz, naphthalenyl CH-5), 8.05 (d, 1H, <i>J</i>â€‰=â€‰7.7â€‰Hz, naphthalenyl CH-8), 10.12 (s, 1H, pyrrolizine-NH), 10.69 (s, 1H, Cl-Ph-NH), 11.55 (s, 1H, naphthalenyl-NH). <sup>13â€‰</sup>C-NMR (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 24.9 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.6 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.6 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 84.7 (pyrrolizine C-7), 108.7 (CN), 111.0 (naphthalenyl C-8a), 120.4 (Cl-Ph CH-2â€‰+â€‰CH-6), 123.2 (naphthalenyl CH-2), 125.2 (naphthalenyl CH-4), 126.3 (naphthalenyl CH-8), 126.8 (Cl-Ph C-4), 127.1 (naphthalenyl CH-7), 127.2 (naphthalenyl CH-6), 128.0 (naphthalenyl CH-3), 128.8 (Cl-Ph CH-3â€‰+â€‰CH-5), 129.3 (naphthalenyl CH-5), 134.4 (pyrrolizine C-5), 134.6 (naphthalenyl C-4a), 135.6 (pyrrolizine C-7a), 136.8 (pyrrolizine C-6), 138.4 (Cl-Ph C-1), 146.0 (naphthalenyl C-1), 158.4 (Cl-Ph-NHCO), 173.4 (NH<u>C</u>SNH). DEPT C<sup>135</sup> (DMSO, 125â€‰MHz, <i>Î´</i> ppm): <i>Î´</i> 25.0 (pyrrolizine <u>C</u>H<sub>2</sub>-2), 25.7 (pyrrolizine <u>C</u>H<sub>2</sub>-1), 49.6 (pyrrolizine <u>C</u>H<sub>2</sub>-3), 120.4 (Cl-Ph CH-2â€‰+â€‰CH-6), 123.2 (naphthalenyl CH-2), 125.2 (naphthalenyl CH-4), 126.3 (naphthalenyl CH-8), 127.1 (naphthalenyl CH-7), 127.2 (naphthalenyl CH-6), 128.0 (naphthalenyl CH-3), 128.8 (Cl-Ph CH-3â€‰+â€‰CH-5), 129.3 (naphthalenyl CH-5). MS (EI): <i>m</i>/<i>z</i> (%) 487 ([Mâ€‰+â€‰2]<sup>+</sup>, 28), 486 ([Mâ€‰+â€‰1]<sup>+</sup>, 47), 472 (25), 445 (52), 435 (85), 412 (58), 367 (74), 336 (93), 277 (100), 262 (37), 202 (52), 156 (59), 128 (24), 91 (23), 77 (4). Anal. Calcd. for C<sub>26</sub>H<sub>20</sub>ClN<sub>5</sub>OS (485.99): C, 64.26; H, 4.15; N, 14.41. Found: C, 64.56; H, 4.62; N, 14.12.</p> </div> </div> </div> <div class="biologicalevaluation" title="sec"><span class="label" tagx="label" title="label">2.2.</span><div class="title" tagx="title" title="title">
Biological evaluation</div> <div class="cytotoxicactivity" title="sec"><span class="label" tagx="label" title="label">2.2.1.</span><div class="title" tagx="title" title="title">
Cytotoxic activity</div> <div class="cellculture" title="sec"><span class="label" tagx="label" title="label">2.2.1.1.</span><div class="title" tagx="title" title="title">
Cell culture</div> <p>The cancer/normal cell lines used in this study were obtained from the American Type Culture Collection (ATCC, Manassas, VA). The cell culture conditions were the same as in our previous reports<a href="#CIT0025"><sup>25</sup></a>.</p> </div> <div class="mttassay" title="sec"><span class="label" tagx="label" title="label">2.2.1.2.</span><div class="title" tagx="title" title="title">
MTT assay</div> <p>MTT assay was used to evaluate the cytotoxicity of the new compounds. The cancer/normal cells were incubated for 72â€‰h with the test compounds (dissolved in DMSO). The cytotoxic activities (IC<sub>50</sub> values) of the new compounds were determined according to the previous reports<a href="#CIT0042"><sup>42</sup></a>.</p> </div> </div> <div class="annexinv-fitc/pistainingassay" title="sec"><span class="label" tagx="label" title="label">2.2.2.</span><div class="title" tagx="title" title="title">
Annexin V-FITC/PI staining assay</div> <p>The apoptotic effect of compounds <b>18b, 19a</b>, and <b>20a</b> on MCF-7 cells was evaluated after treatment for 24â€‰h at 0, 1, 5, and 10â€‰ÂµM. The assay was done using Annexin V-FITC/PI staining assay. Flow cytometry (Bechman Coulter, FC500, Brea, CA) was used differentiate apoptotic/necrotic cells from viable cells<a href="#CIT0043"><sup>43</sup></a>.</p> </div> <div class="cellcycleanalysis" title="sec"><span class="label" tagx="label" title="label">2.2.3.</span><div class="title" tagx="title" title="title">
Cell cycle analysis</div> <p>The MCF-7 cells were incubated with compounds <b>18b, 19a</b>, and <b>20a</b> at 0, 1, 5, and 10â€‰ÂµM. After 24â€‰h treatment. The PI-stained cells was used to analyse cells using flow cytometer (BC, FC500)<sup>44</sup>.</p> </div> <div class="cdk-2inhibitionassay" title="sec"><span class="label" tagx="label" title="label">2.2.4.</span><div class="title" tagx="title" title="title">
Cdk-2 inhibition assay</div> <p>The CDK-2 inhibitory activities of compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> were determined <i>inÂ vitro</i> using ADP-Glo<sup>TM</sup> kinase assay (Promega, Madison, WI). The assay was done according to manufacturerâ€™s instructions and as described in the previous report<a href="#CIT0045"><sup>45</sup></a>. The results were represented as IC<sub>50</sub> values, <a href="#t0004">Table 4</a>.</p> </div> </div> <div class="computationalstudies" title="sec"><span class="label" tagx="label" title="label">2.3.</span><div class="title" tagx="title" title="title">
Computational studies</div> <div class="moleculardockingstudies" title="sec"><span class="label" tagx="label" title="label">2.3.1.</span><div class="title" tagx="title" title="title">
Molecular docking studies</div> <p>The docking studies were done using AutoDock 4.2<sup>46</sup>. Preparation of ligands, proteins, grid, and docking parameter files was done following our previous reports<a href="#CIT0047"><sup>47â€"49</sup></a>. The study was performed according to the previous report<a href="#CIT0025"><sup>25</sup></a>. Discovery studio visualiser was used to visualise the binding interaction<a href="#CIT0050"><sup>50</sup></a>. The pdb files of CDK-2 (pdb: 2VTP)<a href="#CIT0016"><sup>16</sup></a>, CDCK-6 (pdb: 2EUF)<a href="#CIT0051"><sup>51</sup></a>, and CDK-9 (pdb: 3TNH)<a href="#CIT0052"><sup>52</sup></a> were downloaded from protein data bank (<a href="http://www.rcsb.org/pdb">http://www.rcsb.org/pdb</a>). The results of the docking study of compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> were represented in <a href="#t0005">Tables 5</a> and <a href="#t0006">6</a> and <a href="#F0009 F0010 F0011">Figures 9â€"11</a>. Moreover, the results of the docking study of the remaining compounds into CDKs are provided in <a href="https://doi.org/10.1080/14756366.2020.1837124">supplementary data</a> (<a href="https://doi.org/10.1080/14756366.2020.1837124">Figures S138â€"S152</a>).</p> </div> <div class="drug-likenessandadmestudies" title="sec"><span class="label" tagx="label" title="label">2.3.2.</span><div class="title" tagx="title" title="title">
Drug-likeness and ADME studies</div> <p>SwissADME webserver (<a href="http://www.swissadme.ch/">http://www.swissadme.ch/</a>)<a href="#CIT0053"><sup>53</sup></a> was used to predict the physicochemical characters related to pharmacokinetic parameters of the new compounds. The study was performed following the previous report<a href="#CIT0039"><sup>39</sup></a>. The molecular weights, ClogP, number of hydrogen bond donner (HBD), and hydrogen bond acceptor (HBA) of the new compounds were calculated checked for any violation from Lipinskiâ€™s rule<a href="#CIT0054"><sup>54</sup></a>. The chemical structures were sketched and calculation start following submission of the chemical structures. MVs, TPSA, and DLSs were calculated using Molsoft webserver (<a href="http://molsoft.com/mprop/">http://molsoft.com/mprop/</a>). The expected GIT absorption percent (<b>%</b>Abs) of the new compounds was also calculated according to the previous report<a href="#CIT0055"><sup>55</sup></a>.</p> </div> </div> </div> <div class="resultsanddiscussion" title="sec"><span class="label" tagx="label" title="label">3.</span><div class="title" tagx="title" title="title">
Results and discussion</div> <div class="chemistry" title="sec"><span class="label" tagx="label" title="label">3.1.</span><div class="title" tagx="title" title="title">
Chemistry</div> <p>Preparation of the starting compounds <b>12</b><sup>37</sup> and <b>14aâ€"c</b><a href="#CIT0038"><sup>38â€"40</sup></a> was performed according to the previous reports, <a href="#SCH0001">Scheme 1</a>. Briefly, the lactam derivative <b>11</b> was reacted with dimethyl sulphate followed by reaction with malononitrile to give compound <b>12</b>. On the other hand, the anilides <b>14aâ€"c</b> were prepared form the reaction of (un)substituted-anilines with chloroacetyl chloride. The starting materials <b>15aâ€"c</b> were obtained from the reaction of compound <b>12</b> with the acetanilides <b>14aâ€"c</b> in refluxing acetone<a href="#CIT0041"><sup>41</sup></a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Scheme 1.</span></a></div>   <p>Reagents and conditions: (a) (CH<sub>3</sub>)<sub>2</sub>SO<sub>4</sub>, benzene, CH<sub>2</sub>(CN)<sub>2</sub>; (b) ClCH<sub>2</sub>COCl, g. acetic acid, CH<sub>2</sub>COONa; (c) acetone, K<sub>2</sub>CO<sub>3</sub>, reflux, 24â€‰h.</p>   </div> <p>The target urea derivatives <b>16â€"20aâ€"c</b> (<a href="#SCH0002">Scheme 2</a>) were prepared from the reaction of compounds 1<b>5aâ€"c</b> with the appropriate isocyanate in DCM according to the previous report<a href="#CIT0023"><sup>23</sup></a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Scheme 2.</span></a></div>   <p>Reagents and conditions: (d) hexyl isocyanate, TEA, DCM, stir, rt, 12â€‰h; (e) appropriate isocyanate, TEA, DCM, stir, rt, 12â€‰h; (f) appropriate isocyanate/isothiocyanate, TEA, DCM, stir, rt, 12â€‰h.</p>   </div> <p>The crude white solid product was dissolved in chloroform/acetone and the white amorphous solid formed was collected. The chemical structural of the new compounds was confirmed by spectral and elemental analyses, <a href="https://doi.org/10.1080/14756366.2020.1837124">Supplementary data (Figures S43â€"S137)</a>.</p> </div> <div class="biologicalevaluation" title="sec"><span class="label" tagx="label" title="label">3.2.</span><div class="title" tagx="title" title="title">
Biological evaluation</div> <div class="cytotoxicactivity" title="sec"><span class="label" tagx="label" title="label">3.2.1.</span><div class="title" tagx="title" title="title">
Cytotoxic activity</div> <div class="evaluationofcytotoxicactivity" title="sec"><span class="label" tagx="label" title="label">3.2.1.1.</span><div class="title" tagx="title" title="title">
Evaluation of cytotoxic activity</div> <p>All the new compounds were evaluated for their cytotoxicity against three cancer (MCF-7, A2780, and HT29) cell lines. Selection of these cell lines was based on previous findings which reported an overexpression of CDK-2 in breast, ovarian, and colon cancers<a href="#CIT0008"><sup>8â€"10</sup></a>.</p> <p>The investigation of cytotoxicity of the new compounds was performed using the MTT assay<a href="#CIT0042"><sup>42</sup></a>. The results were expressed in IC<sub>50</sub> values (the concentration required to inhibit the growth of cancer cells by 50% compared to the untreated control), <a href="#t0002">Table 2</a>.</p> <div class="table-wrap_UNKNOWN" id="t0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> <p>Cytotoxic activity of compounds <b>10</b>, <b>16â€"20aâ€"c</b> and lapatinib against MCF-7, A2780, and HT29 cell lines. <span class="inline-graphic" title="inline-graphic" /></p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th rowspan="2" align="center">Comp. no.</th> <th rowspan="2" align="center">R1</th> <th rowspan="2" align="center">R2</th> <th rowspan="2" align="center">X</th> <th colspan="3" align="center">(IC50â€‰Â±â€‰SD Î¼M)<sup>a</sup><hr /> </th> <th rowspan="2" align="center">Average Î¼M</th> </tr> <tr> <th align="center">MCF-7</th> <th align="center">A2780</th> <th align="center">HT29</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left"><b>16a</b></td> <td align="left">H</td> <td align="left">-C<sub>6</sub>H<sub>13</sub></td> <td align="left">O</td> <td align="char" char=".">0.59â€‰Â±â€‰0.01</td> <td align="char" char=".">0.35â€‰Â±â€‰0.18</td> <td align="char" char=".">0.85â€‰Â±â€‰0.07</td> <td align="char" char=".">0.60</td> </tr> <tr> <td align="left"><b>16b</b></td> <td align="left">CH<sub>3</sub></td> <td align="left">-C<sub>6</sub>H<sub>13</sub></td> <td align="left">O</td> <td align="char" char=".">1.17â€‰Â±â€‰0.31</td> <td align="char" char=".">0.23â€‰Â±â€‰0.04</td> <td align="char" char=".">1.56â€‰Â±â€‰0.14</td> <td align="char" char=".">0.99</td> </tr> <tr> <td align="left"><b>16c</b></td> <td align="left">Cl</td> <td align="left">-C<sub>6</sub>H<sub>13</sub></td> <td align="left">O</td> <td align="char" char=".">34.13â€‰Â±â€‰4.61</td> <td align="char" char=".">0.53â€‰Â±â€‰0.10</td> <td align="char" char=".">3.10â€‰Â±â€‰0.10</td> <td align="char" char=".">12.59</td> </tr> <tr> <td align="left"><b>17a</b></td> <td align="left">H</td> <td align="left">4-MeOPh</td> <td align="left">O</td> <td align="char" char=".">0.90â€‰Â±â€‰0.11</td> <td align="char" char=".">0.44â€‰Â±â€‰0.03</td> <td align="char" char=".">5.29â€‰Â±â€‰1.51</td> <td align="char" char=".">2.21</td> </tr> <tr> <td align="left"><b>17b</b></td> <td align="left">CH<sub>3</sub></td> <td align="left">4-MeOPh</td> <td align="left">O</td> <td align="char" char=".">2.85â€‰Â±â€‰0.31</td> <td align="char" char=".">0.42â€‰Â±â€‰0.07</td> <td align="char" char=".">1.44â€‰Â±â€‰0.17</td> <td align="char" char=".">1.57</td> </tr> <tr> <td align="left"><b>17c</b></td> <td align="left">Cl</td> <td align="left">4-MeOPh</td> <td align="left">O</td> <td align="char" char=".">3.36â€‰Â±â€‰0.83</td> <td align="char" char=".">0.54â€‰Â±â€‰0.02</td> <td align="char" char=".">0.42â€‰Â±â€‰0.16</td> <td align="char" char=".">1.44</td> </tr> <tr> <td align="left"><b>18a</b></td> <td align="left">H</td> <td align="left">4-F-Ph</td> <td align="left">O</td> <td align="char" char=".">10.76â€‰Â±â€‰0.07</td> <td align="char" char=".">10.58â€‰Â±â€‰2.83</td> <td align="char" char=".">9.13â€‰Â±â€‰0.85</td> <td align="char" char=".">10.16</td> </tr> <tr> <td align="left"><b>18b</b></td> <td align="left">CH<sub>3</sub></td> <td align="left">4-F-Ph</td> <td align="left">O</td> <td align="char" char=".">0.19â€‰Â±â€‰0.02</td> <td align="char" char=".">0.97â€‰Â±â€‰0.14</td> <td align="char" char=".">9.71â€‰Â±â€‰2.25</td> <td align="char" char=".">3.62</td> </tr> <tr> <td align="left"><b>18c</b></td> <td align="left">Cl</td> <td align="left">4-F-Ph</td> <td align="left">O</td> <td align="char" char=".">0.29â€‰Â±â€‰0.19</td> <td align="char" char=".">0.21â€‰Â±â€‰0.10</td> <td align="char" char=".">1.38â€‰Â±â€‰0.02</td> <td align="char" char=".">0.63</td> </tr> <tr> <td align="left"><b>19a</b></td> <td align="left">H</td> <td align="left">1-naphthalenyl</td> <td align="left">O</td> <td align="char" char=".">0.40â€‰Â±â€‰0.42</td> <td align="char" char=".">0.32â€‰Â±â€‰0.17</td> <td align="char" char=".">0.20â€‰Â±â€‰0.01</td> <td align="char" char=".">0.31</td> </tr> <tr> <td align="left"><b>19b</b></td> <td align="left">CH<sub>3</sub></td> <td align="left">1-naphthalenyl</td> <td align="left">O</td> <td align="char" char=".">0.44â€‰Â±â€‰0.09</td> <td align="char" char=".">5.39â€‰Â±â€‰1.13</td> <td align="char" char=".">0.68â€‰Â±â€‰0.14</td> <td align="char" char=".">2.17</td> </tr> <tr> <td align="left"><b>19c</b></td> <td align="left">Cl</td> <td align="left">1-naphthalenyl</td> <td align="left">O</td> <td align="char" char=".">1.27â€‰Â±â€‰0.01</td> <td align="char" char=".">3.07â€‰Â±â€‰1.50</td> <td align="char" char=".">2.56â€‰Â±â€‰0.70</td> <td align="char" char=".">2.30</td> </tr> <tr> <td align="left"><b>20a</b></td> <td align="left">H</td> <td align="left">1-naphthalenyl</td> <td align="left">S</td> <td align="char" char=".">0.42â€‰Â±â€‰0.15</td> <td align="char" char=".">0.16â€‰Â±â€‰0.01</td> <td align="char" char=".">0.72â€‰Â±â€‰0.12</td> <td align="char" char=".">0.43</td> </tr> <tr> <td align="left"><b>20b</b></td> <td align="left">CH<sub>3</sub></td> <td align="left">1-naphthalenyl</td> <td align="left">S</td> <td align="char" char=".">4.97â€‰Â±â€‰0.42</td> <td align="char" char=".">2.04â€‰Â±â€‰0.95</td> <td align="char" char=".">3.83â€‰Â±â€‰0.58</td> <td align="char" char=".">3.61</td> </tr> <tr> <td align="left"><b>20c</b></td> <td align="left">Cl</td> <td align="left">1-naphthalenyl</td> <td align="left">S</td> <td align="char" char=".">1.76â€‰Â±â€‰0.95</td> <td align="char" char=".">2.81â€‰Â±â€‰0.98</td> <td align="char" char=".">3.67â€‰Â±â€‰1.65</td> <td align="char" char=".">2.75</td> </tr> <tr> <td align="left"><b>10</b></td> <td align="left">3,5-diMe<sup>c</sup></td> <td align="left">4-Cl-Ph</td> <td align="left">O</td> <td align="char" char=".">1.12â€‰Â±â€‰0.10</td> <td align="center">â€"</td> <td align="center">â€"</td> <td align="center">â€"</td> </tr> <tr> <td align="left"><b>Lap.</b></td> <td align="left">â€"</td> <td align="left">â€"</td> <td align="left">â€"</td> <td align="char" char=".">6.77â€‰Â±â€‰1.16</td> <td align="char" char=".">10.52â€‰Â±â€‰0.76</td> <td align="char" char=".">12.46â€‰Â±â€‰1.27</td> <td align="char" char=".">9.92</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>a</sup>IC<sub>50</sub> is the concentration of test compounds which reduce cellular growth to 50% after treatment for 72â€‰h.</p> </div> <div class="fn-type-" title=""> <p><sup>b</sup>Results represent mean IC<sub>50</sub> valueâ€‰Â±â€‰SD (<i>n</i>â€‰=â€‰3); Lap, lapatinib.</p> </div> <div class="fn-type-" title=""> <p><sup>c</sup>CI<sub>50</sub> value quoted from our previous publication<a href="#CIT0023"><sup>23</sup></a>.</p> </div> </div> </div> <p>The results revealed the ability of the new compounds to inhibit the growth of the selected cancer cell lines with IC<sub>50</sub> values in the range of 0.16â€"34.13â€‰Î¼M. Among these derivatives, compound <b>18b</b> was the most active against MCF-7 cells (IC<sub>50</sub> = 0.19â€‰Î¼M), while compounds <b>19a</b> and <b>20a</b> were the most potent in inhibiting the growth of HT29 and A2780 cell line, respectively.</p> <p>The hexyl urea derivatives <b>16aâ€"c</b> displayed weak to potent cytotoxic activity against the three cell lines (IC<sub>50</sub> = 0.23â€"34.13â€‰Î¼M). Among the three derivatives, compound <b>16b</b> was the most active against A2780, while compound <b>16a</b> showed the highest cytotoxic activity against MCF-7 and HT29 cell lines, <a href="#t0002">Table 2</a>.</p> <p>Furthermore, the 4-methoxyphenyl-urea derivatives <b>17aâ€"c</b> showed potent cytotoxic activity against the three cancer cell lines with IC<sub>50</sub> values in the range of 0.42â€"5.29â€‰Î¼M. Compound <b>17c</b> was more active against MCF-7 and HT29 cell lines compared to its hexyl urea <b>16c</b>. However, compounds <b>17a</b>,<b>b</b> were less active against MCF-7 and A2780 cell lines than their hexyl analogs <b>16a</b>,<b>b</b>, <a href="#t0002">Table 2</a>.</p> <p>The 4-fluorophenyl urea derivatives <b>18aâ€"c</b> showed moderate to potent cytotoxic activities against the three cancer cell lines (IC<sub>50</sub> = 0.19â€"10.76â€‰Î¼M). Among these derivatives compound <b>18b</b> was the most active against MCF-7 cells. Moreover, compound <b>18c</b> exhibited higher cytotoxic activities towards MCF-7 and A2780 cell lines compared to the hexyl <b>16c</b> and methoxyphenyl <b>17c</b> analogues, <a href="#t0002">Table 2</a>.</p> <p>The naphthalenyl urea derivatives <b>19aâ€"c</b> exhibited the highest cytotoxicity with IC<sub>50</sub> in the range of 0.20â€"5.39â€‰Î¼M. Among these derivatives, compound <b>19a</b> displayed the highest cytotoxic activity against HT29 cell line. On the other hand, the thiourea derivatives <b>20aâ€"c</b> exhibited higher cytotoxic activities against A2780 cells than their urea analogues <b>19aâ€"c</b>. However, they (<b>20aâ€"c</b>) were less active against MCF-7 and HT29 cells compared to compounds <b>19aâ€"c</b>, <a href="#t0002">Table 2</a>.</p> <p>In conclusion, the results of MTT assay of most of the new compounds revealed higher cytotoxicity compared to lapatinib (IC<sub>50</sub> = 6.77â€"12.46â€‰Î¼M). In addition, most of the new compounds (<b>16a</b>, <b>17a</b>, <b>18b</b>,<b>c</b>, <b>19aâ€"c</b>) exhibited higher cytotoxic activities against MCF-7 than the parent compound <b>10</b>, <a href="#t0002">Table 2</a>.</p> </div> <div class="evaluationofcytotoxicselectivity" title="sec"><span class="label" tagx="label" title="label">3.2.1.2.</span><div class="title" tagx="title" title="title">
Evaluation of cytotoxic selectivity</div> <p>Beside the cytotoxic activity, toxicity and selectivity towards normal cells are also critical factors in the discovery of new anticancer agents<a href="#CIT0056"><sup>56</sup></a>. Accordingly, cytotoxicity of the new reported compounds (<b>16</b>â€"<b>20</b>) was also evaluated in the current study using the normal human foetal lung fibroblast (MRC-5). The investigation was performed using the MTT assay according to the previous reports<a href="#CIT0025"><sup>25</sup></a>. The results were presented in <a href="#t0003">Table 3</a>.</p> <div class="table-wrap_UNKNOWN" id="t0003" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3.</span> <p>Cytotoxicity of compounds <b>16â€"20aâ€"c</b>, and lapatinib against normal MRC-5 cells and their selectivity indices (SIs).</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th rowspan="2" align="left">Comp. no.</th> <th rowspan="2" align="center">IC<sub>50</sub> (ÂµMâ€‰Â±â€‰SD)<sup>a</sup> MRC-5</th> <th colspan="3" align="center">Selectivity index<sup>b</sup><hr /> </th> </tr> <tr> <th align="center">MCF-7</th> <th align="center">A2780</th> <th align="center">HT29</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left"><b>16a</b></td> <td align="left">11.86â€‰Â±â€‰0.87</td> <td align="left">20.10</td> <td align="left">33.89</td> <td align="char" char=".">13.95</td> </tr> <tr> <td align="left"><b>16b</b></td> <td align="left">2.65â€‰Â±â€‰0.29</td> <td align="left">2.26</td> <td align="left">11.52</td> <td align="char" char=".">1.70</td> </tr> <tr> <td align="left"><b>16c</b></td> <td align="left">0.75â€‰Â±â€‰0.14</td> <td align="left">0.02</td> <td align="left">1.42</td> <td align="char" char=".">0.24</td> </tr> <tr> <td align="left"><b>17a</b></td> <td align="left">2.30â€‰Â±â€‰0.15</td> <td align="left">2.56</td> <td align="left">5.23</td> <td align="char" char=".">0.43</td> </tr> <tr> <td align="left"><b>17b</b></td> <td align="left">6.20â€‰Â±â€‰0.88</td> <td align="left">2.18</td> <td align="left">14.76</td> <td align="char" char=".">4.31</td> </tr> <tr> <td align="left"><b>17c</b></td> <td align="left">3.80â€‰Â±â€‰0.25</td> <td align="left">1.13</td> <td align="left">7.04</td> <td align="char" char=".">9.05</td> </tr> <tr> <td align="left"><b>18a</b></td> <td align="left">10.70â€‰Â±â€‰0.61</td> <td align="left">0.99</td> <td align="left">1.01</td> <td align="char" char=".">1.17</td> </tr> <tr> <td align="left"><b>18b</b></td> <td align="left">1.46â€‰Â±â€‰0.42</td> <td align="left">7.68</td> <td align="left">1.51</td> <td align="char" char=".">0.15</td> </tr> <tr> <td align="left"><b>18c</b></td> <td align="left">0.10â€‰Â±â€‰0.01</td> <td align="left">0.34</td> <td align="left">0.48</td> <td align="char" char=".">0.07</td> </tr> <tr> <td align="left"><b>19a</b></td> <td align="left">2.67â€‰Â±â€‰0.37</td> <td align="left">6.68</td> <td align="left">8.34</td> <td align="char" char=".">13.35</td> </tr> <tr> <td align="left"><b>19b</b></td> <td align="left">1.54â€‰Â±â€‰0.20</td> <td align="left">3.50</td> <td align="left">0.29</td> <td align="char" char=".">2.26</td> </tr> <tr> <td align="left"><b>19c</b></td> <td align="left">2.68â€‰Â±â€‰0.37</td> <td align="left">2.11</td> <td align="left">0.87</td> <td align="char" char=".">1.05</td> </tr> <tr> <td align="left"><b>20a</b></td> <td align="left">0.43â€‰Â±â€‰0.13</td> <td align="left">1.02</td> <td align="left">2.69</td> <td align="char" char=".">0.60</td> </tr> <tr> <td align="left"><b>20b</b></td> <td align="left">2.00â€‰Â±â€‰0.75</td> <td align="left">0.40</td> <td align="left">0.98</td> <td align="char" char=".">0.52</td> </tr> <tr> <td align="left"><b>20c</b></td> <td align="left">2.92â€‰Â±â€‰0.60</td> <td align="left">1.66</td> <td align="left">1.04</td> <td align="char" char=".">0.80</td> </tr> <tr> <td align="left"><b>Lapatinib</b></td> <td align="left">6.74â€‰Â±â€‰1.22</td> <td align="left">1.00</td> <td align="left">0.64</td> <td align="char" char=".">0.54</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>a</sup>IC<sub>50</sub> against MRC-5 cells after 72â€‰h treatment with the test compound, results represent mean IC<sub>50</sub> Â± SD (<i>n</i>â€‰=â€‰3).</p> </div> <div class="fn-type-" title=""> <p><sup>b</sup>Selectively index (SI) = IC<sub>50</sub> value against normal MRC-5 cells/IC<sub>50</sub> value against cancer cell line.</p> </div> </div> </div> <p>The results revealed IC<sub>50</sub> values in the range of 0.10â€"11.86â€‰ÂµM. To evaluate the cytotoxic selectivity of the new compounds, the selectivity index (SI) for each compound was calculated. The SI was obtained by dividing its IC<sub>50</sub> value of the compound against normal MRC-5 cells by its IC<sub>50</sub> value against the corresponding cancer cell line, <a href="#t0003">Table 3</a>.</p> <p>Among the new compounds, the hexyl urea derivatives <b>16aâ€"c</b> inhibited the growth of MRC-5 cells with IC<sub>50</sub> values in the range of 0.75â€"11.86â€‰ÂµM. among these derivatives, compound <b>16a</b> showed the highest selectivity towards the three cell lines with SIs in the range of 13.95â€"33.89.</p> <p>Moreover, the 4-methoxyphenyl urea derivatives <b>17aâ€"c</b> exhibited higher selectivity (SI = 5.23â€"14.76) towards the ovarian cancer cell line (A2780) compared to their 4-fluorophenyl analogues <b>18aâ€"c</b>. However, compound <b>18b</b> displayed more than seven times higher selectivity towards the cancerous MCF-7 cells compared to lapatinib. In addition, the naphthalenyl urea derivatives <b>19a</b>â€"<b>c</b> displayed higher selectivity towards MCF-7 and HT29 cells compared to their thiourea derivatives <b>20a</b>â€"<b>c</b>, <a href="#t0003">Table 3</a>.</p> </div> <div class="structureâ€" title="sec"><span class="label" tagx="label" title="label">3.2.1.3.</span><div class="title" tagx="title" title="title">
Structureâ€"activity relationship (SAR)</div> <p>The relationship between the chemical structure of the new compounds and their cytotoxic activity/selectivity was presented in <a href="#F0006">Figure 6</a>. Compound <b>16a</b> which exhibited potent submicromolar cytotoxicity (IC<sub>50</sub> = 0.35â€"0.85â€‰Î¼M) and was the most selective towards three cancer cell lines.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>SAR of cytotoxicity and selectivity of the new compounds against MCF-7, A2780, and HT29 cells.</p>   </div> <p>Substitution on the phenyl ring in compound <b>16a</b> with 4-CH<sub>3</sub> increased the cytotoxic activity against A2780 cells. On the other hand, significant decreases in cytotoxic activities of compound <b>16a</b> against both MCF-7 and HT29 cell lines were observed on substitution with either 4-CH<sub>3</sub>/Cl groups on the phenyl ring. Moreover, substitution with CH<sub>3</sub>/Cl groups at the <i>para</i>-position of the phenyl ring in compound <b>16a</b> was associated with decrease in selectivity towards the three cancer cell lines, <a href="#F0006">Figure 6</a>.</p> <p>Replacement of the hexyl group in compound <b>16a</b> by 4-methoxyphenyl afforded compound <b>17a</b> with slight decrease in cytotoxicity against both MCF-7 and A2780 cells, while significant reduction in activity against HT29 cells was observed. Substitution on the phenyl ring (A) in compound <b>17a</b> with 4-CH<sub>3</sub>/Cl resulted in increase in cytotoxic activity against HT29 cells, while cytotoxicity against MCF-7 was decreases, <a href="#F0006">Figure 6</a>.</p> <p>In addition, replacement of the OCH<sub>3</sub> group in compound <b>17a</b> by F afforded compound <b>18a</b> with sharp decrease in cytotoxicity against the three cancer cell lines. On the other hand, substitution on the phenyl ring of compound <b>18a</b> with either the electron donating 4-CH<sub>3</sub> or the electron withdrawing 4-Cl groups improved cytotoxicity against both MCF-7 and A2780 cell lines, <a href="#F0006">Figure 6</a>.</p> <p>Replacement of the hexyl group in compound <b>16a</b> by 1-naphthalenyl group afforded compound <b>19a</b> which was more active as cytotoxic agent against the three cancer cell lines. Although cytotoxicity of compound <b>19a</b> was decreased on substitution with 4-CH<sub>3</sub>/Cl groups on ring A, but the produced derivatives <b>19b</b>,<b>c</b> were more active than their hexyl analogues <b>16b</b>,<b>c</b> against MCF-7 and HT29 cells, <a href="#F0006">Figure 6</a>.</p> <p>Replacement of urea oxygen in compound <b>19a</b> by sulphur yielded compound <b>20a</b> with higher cytotoxicity against A2780 cells but lower cytotoxic activities towards HT29 cells. Substitution on ring A of compound <b>20a</b> with the 4-CH<sub>3</sub>/Cl groups resulted in weaker cytotoxic activities against the three cancer cell lines, <a href="#F0006">Figure 6</a>.</p> </div> </div> <div class="annexinv-fitc/piapoptosisassay" title="sec"><span class="label" tagx="label" title="label">3.2.2.</span><div class="title" tagx="title" title="title">
Annexin V-FITC/PI apoptosis assay</div> <p>Previously, compound <b>10</b>, the lead compound in this study was reported to induce apoptosis in MCF-7 cells through activation of caspase 3/7<sup>23</sup>. Accordingly, the ability of the new compounds to induce apoptosis in the same cells (MCF-7) was investigated in the current study. Compounds <b>18â€‰b</b>, <b>19a</b>, and <b>20a</b> the most active against MCF-7 cells were selected for this study. The ability of these compounds to induce apoptosis was evaluated using annexin V fluorescein isothiocyanate (FITC)/Propidium Iodide (PI) staining assay according to the previous report<a href="#CIT0043"><sup>43</sup></a>. MCF-7 cells were treated with each the test compounds for 24â€‰h. The results were presented in <a href="#F0007">Figure 7</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p>Annexin V phases of MCF-7 treated with compound <b>18b, 19a</b>, and <b>20a</b> at 0, 1, 5, and 10â€‰ÂµM (24â€‰h, <i>x</i>-axis: Annexin V; <i>y</i>-axis: PI). C1: necrotic cells; C2: late apoptosis; C3: live cells; C4: early apoptosis. Data shown is mean % cell numberâ€‰Â±â€‰SD (<i>n</i>â€‰=â€‰3). Experiment was repeated x3 times.</p>   </div> <p>The results revealed a significant dose dependent increase of the apoptotic events by compound <b>18â€‰b</b> from 0% in the control, to 20%, 23%, and 37% at 1, 5, and 10â€‰ÂµM, respectively. Similarly, compound <b>20a</b> caused increase of 38%, 39%, and 47% at 1, 5, and 10â€‰ÂµM, respectively. in addition, compound <b>19a</b> caused the highest increase in apoptotic events with 68%, 88%, and 90% at 1, 5, and 10â€‰ÂµM, respectively, <a href="#F0007">Figure 7</a>.</p> </div> <div class="cell-cycleanalysis" title="sec"><span class="label" tagx="label" title="label">3.2.4.</span><div class="title" tagx="title" title="title">
Cell-cycle analysis</div> <p>The effect of compounds <b>18â€‰b</b>, <b>19a</b>, and <b>20a</b>, on cell-cycle perturbation in MCF-7 cells was investigated based on previous report<a href="#CIT0044"><sup>44</sup></a>. The MCF-7 cells were treated with each of the three compounds at 0, 1, 5, and 10â€‰ÂµM for 24â€‰h. Following this treatment, the propidium iodide (PI)-stained cells were analysed. The results are outlined in <a href="#F0008">Figure 8</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8.</span></a></div>   <p>Flow cytometry bar chart showing the effect of compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> at 0, 1, 5, and 10â€‰Î¼M on cell cycle distribution of MCF-7 cells after treatment for 24â€‰h, values represent the meanâ€‰Â±â€‰SD (<i>n</i>â€‰=â€‰3), experiment was repeated Ã—3.</p>   </div> <p>The results of the cell cycle analysis revealed G<sub>1</sub> -S cell cycle arrest by the three compounds (<b>18b</b>, <b>19a</b>, and <b>20a</b>). Compound <b>19a</b> caused the highest increase in the number of cells in the G<sub>1</sub> cell-cycle phase (70%) at the expense of the other phases, compared to <b>18b</b> (53%) and <b>20a</b> (57%), in <a href="#F0008">Figure 8</a>. These results agree with previous reports which indicate that CDK-2 inhibitors arrest cell cycle at the G<sub>1</sub> phase<a href="#CIT0006"><sup>6</sup></a><sup>,</sup><a href="#CIT0007"><sup>7</sup></a>.</p> </div> <div class="cdk-2inhibitoryactivity" title="sec"><span class="label" tagx="label" title="label">3.2.4.</span><div class="title" tagx="title" title="title">
Cdk-2 inhibitory activity</div> <p>Targeting CDKs is one of the promising strategies in cancer chemotherapy<a href="#CIT0006"><sup>6</sup></a>. The success of this strategy was confirmed by many CDK inhibitors which reached phase I/II clinical trials as potential anticancer agents<a href="#CIT0014"><sup>14</sup></a>. Previously, several compounds with pyrrolizine-5-carboxamide scaffold exhibited weak to moderate inhibitory activity against CDK-2<sup>24,</sup><a href="#CIT0025"><sup>25</sup></a>.</p> <p>In the current study, compounds <b>18b</b>, <b>19a</b>, and <b>20a</b>, the most active in cytotoxic assay (<a href="#t0002">Table 2</a>) were selected to evaluate for their inhibitory activity against CDK-2 <i>inÂ vitro</i>. The assay was performed using Adenosine diphosphate (ADP)-Glo<sup>TM</sup> kinase assay kit (Promega, Madison, WI) following the previous report<a href="#CIT0045"><sup>45</sup></a>. Briefly, the ability of CDK-2 to convert ATP to ADP was used as a measure of the kinase activity. The inhibition in activity of the kinase was determined after treatment with the test compounds relative to the control (DMSO, 5%). Staurosporine was used as positive control. The results of this assay are presented as IC<sub>50</sub> in <a href="#t0004">Table 4</a>.</p> <div class="table-wrap_UNKNOWN" id="t0004" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4.</span> <p>Inhibition of CDK-2 by compounds <b>18b</b>, <b>19a,</b> and <b>20a</b>.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Comp. no.</th> <th align="center">IC<sub>50</sub> (nM)<sup>a</sup> Â±SEM</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left"><b>18b</b></td> <td align="left">115.3â€‰Â±â€‰3.20</td> </tr> <tr> <td align="left"><b>19a</b></td> <td align="left">25.53â€‰Â±â€‰1.7</td> </tr> <tr> <td align="left"><b>20a</b></td> <td align="left">77.90â€‰Â±â€‰2.2</td> </tr> <tr> <td align="left"><b>Staurosporine</b></td> <td align="left">26.39â€‰Â±â€‰0.9</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>IC<sub>50</sub>, concentration which decrease kinase activity to 50%.</p> </div> <div class="fn-type-" title=""> <p><sup>a</sup>Average of three determinations;.</p> </div> </div> </div> <p>The results revealed high <i>inÂ vitro</i> inhibitory activity against CDK-2 by the tested compounds with IC<sub>50</sub> in the nanomolar range. Compound <b>19a</b> exhibited the highest inhibitory activity against CDK-2, which was comparable to that of the positive control (staurosporine). The inhibitory activity of compound <b>19a</b> was nearly three-fold higher than its thiourea analogue <b>20a</b>. Moreover, the inhibitory activities of the two naphthalenyl derivatives <b>19a</b> and <b>20a</b> against CDK-2 was higher than that of the 4-methoxyphenyl urea <b>18b</b>, <a href="#t0004">Table 4</a>.</p> </div> </div> <div class="computationalstudies" title="sec"><span class="label" tagx="label" title="label">3.3.</span><div class="title" tagx="title" title="title">
Computational studies</div> <div class="moleculardockingstudies" title="sec"><span class="label" tagx="label" title="label">3.3.1.</span><div class="title" tagx="title" title="title">
Molecular docking studies</div> <p>To identify the potential mechanism of action and understand the results of the CDK-2 inhibitory assay (<a href="#t0004">Table 4</a>), a molecular docking study was performed into the active site of CDK-2 protein. The study was performed to evaluate the binding free energies (affinities) and inhibition constants of the new compounds into CDK-2. The crystal structure of CDK-2 protein (pdb: 2VTP)<a href="#CIT0016"><sup>16</sup></a> was obtained from protein data bank (<a href="https://www.rcsb.org/structure/3TNW">https://www.rcsb.org/structure/3TNW</a>). AutoDock 4.2 was used to carry out the docking study<a href="#CIT0046"><sup>46</sup></a>. The docking procedures were performed according to the previous report<a href="#CIT0025"><sup>25</sup></a>. The grid and docking parameter files were prepared according to the previous reports<a href="#CIT0047"><sup>47â€"49</sup></a>. Visualisation of the binding interactions of the new compounds in the active sites of CDK-2 was performed by Discovery studio visualizer<a href="#CIT0050"><sup>50</sup></a>.</p> <div class="dockingstudyintocdk-2" title="sec"><span class="label" tagx="label" title="label">3.3.1.1.</span><div class="title" tagx="title" title="title">
Docking study into CDK-2</div> <p>Initially, the validation of the docking procedures into the active site of CDK-2 was performed. The native ligand, compound <b>3</b> (LZ9) was re-docked into the active site of CDK-2. The results revealed superposition of the re-docked ligand <b>3</b> over the position of the co-crystallized ligand with RMSD of 0.75â€‰Ã… (<a href="https://doi.org/10.1080/14756366.2020.1837124">Supplementary data, Figure S138</a>). The results of the docking study of compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> revealed nice fitting into CDK-2 with binding free energy in the range of âˆ’9.16 to âˆ’9.77â€‰kcal/mol. The binding affinities of the three compounds towards CDK-2 was also higher than that of the native ligand LZ9, <a href="#t0005">Table 5</a>. The results of the docking study of the remaining derivatives are provided in <a href="https://doi.org/10.1080/14756366.2020.1837124">supplementary data (Table S1 and Figures S139â€"S150</a>).</p> <p>Among the three derivatives, compound <b>19a</b> showed higher binding affinity towards CDK-2 compared to compounds <b>18b</b> and <b>20a</b> which was matched with their inhibitory activities against CDK-2, <a href="#t0005">Table 5</a>. In addition, the thiourea <b>20a</b> exhibited also higher affinity to CDK-2 compared to the 4-flourophenyl derivative <b>18b</b> which was also matched with their IC<sub>50</sub> values towards CDK-2, <a href="#t0004">Table 4</a>. To explain these results, the binding modes, orientations, and interactions of the three compounds (<b>18b, 18a</b>, and, <b>20a</b>) were investigated in comparison to the native ligand to understand their inhibitory activities against CDK-2.</p> <div class="table-wrap_UNKNOWN" id="t0005" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 5.</span> <p>Docking results of compounds <b>18b</b>, <b>19a</b>, <b>20a</b> and LZ9 into CDK-2 (pdb: 2VTP)</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th rowspan="2" align="left">Ligand</th> <th rowspan="2" align="center"><i>Î"</i>G<i><sub>b</sub></i><sup>a</sup></th> <th rowspan="2" align="center"><i>K<sub>i</sub></i><sup>b</sup></th> <th rowspan="2" align="center">HBs<sup>c</sup></th> <th colspan="2" align="center">Atoms in H-bonding <hr /> </th> <th rowspan="2" align="center">Length <sup>d</sup>(Ã…)</th> </tr> <tr> <th align="center">In ligand</th> <th align="center">In protein</th> </tr> </thead> <tbody valign="top"> <tr> <td rowspan="3"><b>18â€‰b</b></td> <td rowspan="3" align="char" char=".">âˆ’9.16</td> <td rowspan="3" align="char" char=".">193.5â€‰nM</td> <td rowspan="3" align="char" char=".">3</td> <td align="left">Urea N<u>H</u></td> <td align="left">CO of Leu83</td> <td align="char" char=".">1.88</td> </tr> <tr> <td align="left">C<u>N</u></td> <td align="left">NH of Leu83</td> <td align="char" char=".">2.09</td> </tr> <tr> <td align="left">Ph-NH</td> <td align="left">CO of Gln131</td> <td align="char" char=".">2.06</td> </tr> <tr> <td rowspan="3"><b>19a</b></td> <td rowspan="3" align="char" char=".">âˆ’9.97</td> <td rowspan="3" align="char" char=".">49.38â€‰nM</td> <td rowspan="3" align="char" char=".">3</td> <td align="left">C<u>N</u></td> <td align="left">NH of Leu83</td> <td align="char" char=".">1.97</td> </tr> <tr> <td align="left">Urea N<u>H</u></td> <td align="left">CO of Leu83</td> <td align="char" char=".">1.80</td> </tr> <tr> <td align="left">Ph-NH</td> <td align="left">CO of Gln131</td> <td align="char" char=".">2.93</td> </tr> <tr> <td rowspan="4"><b>20a</b></td> <td rowspan="4" align="char" char=".">âˆ’9.54</td> <td rowspan="4" align="char" char=".">101.28â€‰nM</td> <td rowspan="4" align="char" char=".">4</td> <td align="left">C<u>N</u></td> <td align="left">NH of Leu83</td> <td align="char" char=".">1.90</td> </tr> <tr> <td align="left">Thiourea S</td> <td align="left">CO of Asp86</td> <td align="char" char=".">2.07</td> </tr> <tr> <td align="left">Thiourea S</td> <td align="left">CO of Gln131</td> <td align="char" char=".">2.60</td> </tr> <tr> <td align="left">Ph-NH</td> <td align="left">CO of Gln131</td> <td align="char" char=".">2.76</td> </tr> <tr> <td rowspan="3"><b>LZ9</b></td> <td rowspan="3" align="char" char=".">âˆ’7.56</td> <td rowspan="3" align="char" char=".">2.81â€‰Î¼M</td> <td rowspan="3" align="char" char=".">3</td> <td align="left">Pyrazole-C<u>O</u></td> <td align="left">NH of Glu81</td> <td align="char" char=".">2.21</td> </tr> <tr> <td align="left">pyrazole NH</td> <td align="left">CO of Leu83</td> <td align="char" char=".">2.05</td> </tr> <tr> <td align="left">pyrazole NH<b><i><sup>e</sup></i></b></td> <td align="left">CH of Phe82</td> <td align="left">Â </td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>a</sup>Binding free energy (kcal/mol).</p> </div> <div class="fn-type-" title=""> <p><sup>b</sup>Inhibition constant (n/Î¼M).</p> </div> <div class="fn-type-" title=""> <p><sup>c</sup>HBs, number of hydrogen bonds.</p> </div> <div class="fn-type-" title=""> <p><sup>d</sup>Length in angstrom (Ã…).</p> </div> <div class="fn-type-" title=""> <p><sup>e</sup>Carbon hydrogen bond.</p> </div> </div> </div> <p>Investigation of the binding mode of the best fit conformation of compound <b>19a</b> revealed partial superposition of the pyrrole ring over the pyrazole ring in compound <b>3</b>. Moreover, the naphthalenyl moiety in compound <b>19a</b> superposed over the 4-fluorophenyl moiety of compound <b>3</b>, <a href="#F0009">Figure 9</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9.</span></a></div>   <p>Binding mode and interactions of compound <b>19a</b> into CDK-2 (pdb: 2VTP)): (A) 3D binding mode of compound <b>19a</b> into the active site of CDK-2, the native ligand LZ9 shown as yellow sticks, receptor shown as hydrogen-bond surface; (B) 2D binding mode of compound <b>19a</b> into CDK-2 showing three conventional hydrogen bonds, one pi-cation, and several hydrophobic interactions, hydrogen atoms were omitted for clarity.</p>   </div> <p>Compound <b>19a</b> exhibited three conventional hydrogen bonds with Leu83 (2 HBs) and Gln131 with bond length in the range of 1.80â€"2.93â€‰Ã…, <a href="#t0005">Table 5</a>. The pyrrolidine ring in compound <b>19a</b> also formed one carbon hydrogen bond with Asp145. Compound <b>19a</b> also showed one pi-anion interaction with Asp145 and a cluster of hydrophobic interactions with Val18, Ala31, Val64, Phe80, Leu134, and Ala144, <a href="#F0009">Figure 9</a>. These binding mode and interactions of compound <b>19a</b> with CDK-2 could account for its high inhibitory activity against CDK-2.</p> <p>Compound <b>20a</b> also adopted binding orientations in which the naphthalenyl-thiourea side chain superpose the position of 4-fluorophenyl-carboxamide moiety in compound <b>3</b>, <a href="#F0010">Figure 10</a>. Moreover, the phenyl-carboxamide side chain in compound <b>20a</b> adopted an orientation superposing over the position of the difluorophenyl-carboxamide chain in LZ9, while the pyrrole ring in compound <b>20a</b> overlaid partially over the pyrazole ring in LZ9.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 10.</span></a></div>   <p>Binding modes and interactions of compounds <b>18b/20a</b> into CDK-2 (pdb: 2VTP)): A) 3â€‰D binding mode of compound <b>18b</b> into CDK-2, the native ligand LZ9 shown as yellow sticks, receptor shown as hydrogen-bond surface; B) 3â€‰D binding mode of compound <b>20a</b> into CDK-2, the native ligand LZ9 shown as yellow sticks; C) 2â€‰D binding mode of compound <b>18b</b> into CDK-2 showing three conventional hydrogen bonds and several hydrophobic interactions; (C) 2D binding mode of compound <b>20a</b> into CDK-2 showing four conventional hydrogen bonds and one pi-anion, and several hydrophobic interactions, hydrogen atoms were omitted for clarity.</p>   </div> <p>The binding interactions of compound <b>20a</b> with CDK-2 included four conventional hydrogen bonds with Leu83, Asp86, and Gln131 (2 HBs) amino acids. In addition, one pi-anion and several hydrophobic interactions were also formed between compound <b>20a</b> and amino acids in the active site of CDK-2, <a href="#F0010">Figure 10</a>.</p> <p>Although compound <b>20a</b> showed higher number of hydrogen bonds with CDK-2, but it exhibited lower affinity than compound <b>19a</b> towards CDK-2. This could be attributed to the stronger (shorter) hydrogen bonds formed by compound <b>19a</b> with CDK-2 compared to compound <b>20a</b>, <a href="#t0005">Table 5</a>.</p> <p>On the other hand, compound <b>18b</b> exhibited three conventional hydrogen bonds with Leu83 and Gln131 and one carbon hydrogen bond with Asp145. Several hydrophobic interactions of the alkyl and pi-alkyl types were also observed between compound <b>18b</b> and amino acids in the active site of CDK-2, <a href="#F0010">Figure 10</a>.</p> <p>The bulkier size of compounds <b>19a</b> and <b>20a</b> compared to compound <b>18b</b> could account even partially to their higher affinities and higher inhibitory activities towards CDK-2. In addition, the higher binding affinities of compounds <b>19a</b> and <b>20a</b> compared to compound <b>18b</b> could be due to higher number and/or stronger hydrogen bonding interactions compared to compound <b>18b</b>. Moreover, each of the two compounds (<b>19a</b> and <b>20a</b>) exhibited one electrostatic interaction of pi-anion type which was absent with compound <b>18b</b>. these findings could account even partially for the higher inhibitory activity of compounds <b>19a</b> and <b>20a</b> compared to compound <b>18b</b>.</p> <p>Moreover, the results of the docking study into CDK-2 indicated the important role of the urea/thiourea moieties in the formation of hydrogen bonding interactions between the tested pyrrolizines (<b>18b</b>, <b>19a</b>, and <b>20a</b>) and amino acids in the active site of CDK-2. In addition, the pyrrolidine ring in the three compounds also contributed hydrophobically in the binding interactions into CDK-2 through the formation of a cluster of hydrophobic interactions with Ala31, Phe80, Leu134, and Ala144.</p> <p>However, compounds <b>18b</b>, <b>19a</b> and <b>20a</b> were designed based on the chemical structure of the multi-CDKI <b>3</b>. Accordingly, a molecular docking study was performed to investigate other potential targets which could mediate their cytotoxic activities.</p> </div> <div class="dockingintocdk-6/9" title="sec"><span class="label" tagx="label" title="label">3.3.1.2.</span><div class="title" tagx="title" title="title">
Docking into CDK-6/9</div> <p>The multi-CDKIs <b>1</b>â€"<b>4</b> exhibited potent anticancer activities against breast, ovarian, and colon cancers which could be attributed to their pan-CDKs inhibitory activities<a href="#CIT0015"><sup>15â€"18</sup></a><sup>,</sup><a href="#CIT0057"><sup>57</sup></a>. The ability of these compounds to inhibit multi-CDKs could be attributed to the shared pharmacophoric groups between the four compounds, <a href="#F0001">Figure 1</a>. In addition, high sequence similarities were confirmed between different members in the CDK family. CDK-6 and CDK-9 have sequence similarity of 44% and 32% with CDK-2, respectively<a href="#CIT0006"><sup>6</sup></a>. These similarities could also allow the CDK-2 inhibitors to target other members in the CDK family.</p> <p>Based on these findings, compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> which were designed bearing the pharmacophoric features of the multi-CDKI <b>3</b> were docked into the active sites of CDK-6 (pdb: 2EUF)<a href="#CIT0051"><sup>51</sup></a> and CDK-9 (pdb: 3TNH)<a href="#CIT0052"><sup>52</sup></a>. The aim of this study was to evaluate the ability of the three compounds to bind to and inhibit these two kinases. Selection of these three compounds was based on their high CDK-2 inhibitory activities, Table 4. In addition, the two CDKs 6/9 were selected based on the results of the cell cycle analysis which revealed G<sub>1</sub> to S cell-cycle arrest by the three compounds (<b>18b</b>, <b>19a</b>, and <b>20a</b>), <a href="#F0008">Figure 8</a>.</p> <p>The docking procedures into CDK-6/9 were essentially the same as those applied in the docking study into CDK-2<sup>25</sup>. Validation of docking study into CDK-6 revealed superposition of the re-docked ligand, palbociclib over the co-crystallized ligand with RMSD of 0.61â€‰Ã…, while the re-docked CAN508 into CDK-9 superposed the position of the native ligand with RMSD of 1.06â€‰Ã… (<a href="https://doi.org/10.1080/14756366.2020.1837124">Supplementary data, Figures S151 and S152</a>). The docking results were presented in Table 6.</p> <p>The results of the docking study into CDK-6 revealed good fitting of the three compounds with binding free energy in the range of âˆ’9.84 to âˆ’10.52â€‰kcal/mol compared to âˆ’11.54 for palbociclib. Among the three pyrrolizines, the 4-fluorophenyl urea <b>19a</b> displayed the highest affinity towards CDKs-6, while compound <b>18b</b> showed the lowest binding free energy, Table 6.</p> <p>Investigation of the binding interaction of compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> with CDK-6 revealed the formation of 1â€"3 conventional hydrogen bonds compared to one conventional hydrogen bond for palbociclib. The three compounds also exhibited several types of hydrophobic interactions with amino acids into the active site of CDK-6, <a href="#F0011">Figure 11</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 11.</span></a></div>   <p>3D binding modes, hydrogen bonding (shown as green dotted lines), and electrostatic interactions (shown as orang dotted lines) of compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> (shown as sticks) into CDK-6 (pdb: 2EUF) and CDK-9 (pdb: 3TNH): (A) compound <b>18b</b> into CDK-6; (B) compound <b>19a</b> into CDK-6; (C) compound <b>20a</b> into CDK-6; (D) compound <b>18b</b> into CDK-9; (E) compound <b>19a</b> into CDK-9; (F) compound <b>20a</b> into CDK-9, amino acid residues shown as lines coloured by element, hydrogen atoms were omitted for clarity.</p>   </div> <p>The tested compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> exhibited higher binding affinities (Î"G<sub>b</sub> = âˆ’9.81 to âˆ’10.40â€‰kcal/mol) towards CDK-9 compared to the native ligand, CAN508 (Î"G<sub>b</sub> = âˆ’6.29â€‰kcal/mol). The high affinities of the new compounds could be attributed to their bulkier size compared to CAN508 which allow them to occupy larger volume of the active site in CDK-9.</p> <p>Compound <b>19a</b> also showed the highest affinity towards CDK-9. Investigation of the binding interactions of compound <b>18b</b> into CDK-9 revealed one conventional hydrogen bond with Ile25, while compound <b>19a</b> showed three hydrogen conventional bonds Thr29, Lys48, and Ala153, <a href="#F0011">Figure 11</a>. The results of the docking study into CDKs-6/9 indicated that the three compounds (<b>18b</b>, <b>19a</b>, and <b>20a</b>) could have potential inhibitory activities against the two kinases.</p> <p>In attempt to compare the binding affinities of the three compounds against the CDKs, fold selectivity was calculated, <a href="#t0006">Table 6</a>. The three compounds displayed higher affinities towards CDK-6/9 than CDK-2 with fold selectivity in the range of <b>2.06</b>â€"3.58. These results suggest that the three compounds could have better inhibitory activities against CDK-6/9 than CDK-2.</p> <div class="table-wrap_UNKNOWN" id="t0006" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 6.</span> <p>Docking results of compounds, <b>18b</b>, <b>19a</b>, and <b>20a</b> into CDK-6 (pdb: 2EUF) and CDK-9 (pdb: 3TNH) in comparison to the native ligands, palbociclib (LQQ) and CAN508 (F18).</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th rowspan="2" align="left">Ligand</th> <th colspan="3" align="center">CDK-6 <hr /> </th> <th colspan="3" align="center">CDK-9 <hr /> </th> </tr> <tr> <th align="center"><i>Î"G<sub>b</sub></i><sup>a</sup></th> <th align="center"><i>K<sub>i</sub></i><sup>b</sup></th> <th align="center"><i>Fs</i><sup>c</sup></th> <th align="center"><i>Î"G<sub>b</sub></i><sup>a</sup></th> <th align="center"><i>K<sub>i</sub></i><sup>b</sup></th> <th align="center"><i>Fs</i><sup>c</sup></th> </tr> </thead> <tbody valign="top"> <tr> <td align="left"><b>18b</b></td> <td align="left">âˆ’9.84</td> <td align="left">61.08â€‰nM</td> <td align="left">3.17</td> <td align="left">âˆ’9.81</td> <td align="left">64.89â€‰nM</td> <td align="char" char=".">2.98</td> </tr> <tr> <td align="left"><b>19a</b></td> <td align="left">âˆ’10.52</td> <td align="left">19.41â€‰nM</td> <td align="left">2.54</td> <td align="left">âˆ’10.40</td> <td align="left">23.93â€‰nM</td> <td align="char" char=".">2.06</td> </tr> <tr> <td align="left"><b>20a</b></td> <td align="left">âˆ’10.30</td> <td align="left">28.30â€‰nM</td> <td align="left">3.58</td> <td align="left">âˆ’9.99</td> <td align="left">47.27â€‰nM</td> <td align="char" char=".">2.14</td> </tr> <tr> <td align="left"><b>LQQ<i><sup>d</sup></i></b></td> <td align="left">âˆ’11.54</td> <td align="left">3.48â€‰nM</td> <td align="left">â€"</td> <td align="left">â€"</td> <td align="left">â€"</td> <td align="center">â€"</td> </tr> <tr> <td align="left"><b>F18<i><sup>e</sup></i></b></td> <td align="left">â€"</td> <td align="left">â€"</td> <td align="left">â€"</td> <td align="left">âˆ’6.29</td> <td align="left">24.67â€‰Î¼M</td> <td align="center">â€"</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>a</sup>Binding free energy (kcal/mol).</p> </div> <div class="fn-type-" title=""> <p><sup>b</sup>Inhibition constant.</p> </div> <div class="fn-type-" title=""> <p><sup>c</sup>Fold selectivity towards CDK-6/9, calculated by dividing the K<sub>i</sub> value against CDK-2 by the K<sub>i</sub> value against CDK-6/9.</p> </div> <div class="fn-type-" title=""> <p><sup>d</sup>LQQ, palbociclib/.</p> </div> <div class="fn-type-" title=""> <p><sup>e</sup>F18, CAN508.</p> </div> </div> </div> </div> </div> <div class="insilicoadmeanddrug-likenessstudies" title="sec"><span class="label" tagx="label" title="label">3.3.2.</span><div class="title" tagx="title" title="title"><i>In silico</i> ADME and drug-likeness studies</div> <p>One of the major problems in drug discovery and development is the poor pharmacokinetic profile of a drug candidate<a href="#CIT0058"><sup>58</sup></a><sup>,</sup><a href="#CIT0059"><sup>59</sup></a>. The success of a drug candidate to pass the clinical investigation depends mainly on its pharmacokinetic as well as pharmacodynamic profile. Accordingly, the <i>in silico</i> studies of the physicochemical properties related to the pharmacokinetics of drug candidates have attracted considerable attention in the last two decades<a href="#CIT0060"><sup>60</sup></a><sup>,</sup><a href="#CIT0061"><sup>61</sup></a>.</p> <p>In the current study, the physicochemical properties related to the pharmacokinetic (PK) profiles of the new compounds <b>16â€"20aâ€"c</b> were evaluated using SwissADME (<a href="http://www.swissadme.ch/">http://www.swissadme.ch/</a>)<a href="#CIT0053"><sup>53</sup></a>. The new compounds were also evaluated for their drug-likeness score using the online Molsoft tools (<a href="http://www.molsoft.com/mprop/">http://www.molsoft.com/mprop/</a>). To predict the oral activity, the new compounds were also evaluated for any violation from Lipinskiâ€™s rule<a href="#CIT0054"><sup>54</sup></a>. The calculated values of the new compounds were compared with those of the multi-CDKIs, flavopiridol, palbociclib, compounds <b>3</b> and <b>10</b>. The results of this study are presented in <a href="#t0007">Table 7</a>.</p> <div class="table-wrap_UNKNOWN" id="t0007" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 7.</span> <p>Molecular properties related to PKs and DLS of the new compounds in comparison to compounds <b>3</b>, <b>10</b>, flavopiridol and palbociclib.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Â </th> <th colspan="7" align="center">Physicochemical properties <hr /> </th> <th rowspan="2" align="center">Lipinskiâ€™s rule</th> <th rowspan="2" align="center">%Abs <i><sup>a</sup></i></th> <th rowspan="2" align="center">BS</th> <th rowspan="2" align="center">DLS <i><sup>b</sup></i></th> </tr> <tr> <th align="left">Comp.</th> <th align="center">MW</th> <th align="center">MV</th> <th align="center">TPSA</th> <th align="center">MlogP</th> <th align="center">RBs</th> <th align="center">HB<sub>A</sub></th> <th align="center">HB<sub>D</sub></th> </tr> </thead> <tbody valign="top"> <tr> <td align="left"><b>16a</b></td> <td align="left">393.48</td> <td align="left">431.15</td> <td align="left">73.39</td> <td align="left">1.97</td> <td align="left">11</td> <td align="left">3</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">83.68</td> <td align="left">0.55</td> <td align="left">0.24</td> </tr> <tr> <td align="left"><b>16b</b></td> <td align="left">407.51</td> <td align="left">452.09</td> <td align="left">73.39</td> <td align="left">2.18</td> <td align="left">11</td> <td align="left">3</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">83.68</td> <td align="left">0.55</td> <td align="left">0.35</td> </tr> <tr> <td align="left"><b>16c</b></td> <td align="left">427.93</td> <td align="left">448.35</td> <td align="left">73.39</td> <td align="left">2.45</td> <td align="left">11</td> <td align="left">3</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">83.68</td> <td align="left">0.55</td> <td align="left">0.84</td> </tr> <tr> <td align="left"><b>17a</b></td> <td align="left">415.44</td> <td align="left">427.60</td> <td align="left">79.50</td> <td align="left">1.71</td> <td align="left">8</td> <td align="left">4</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">81.57</td> <td align="left">0.55</td> <td align="left">0.45</td> </tr> <tr> <td align="left"><b>17b</b></td> <td align="left">429.47</td> <td align="left">448.54</td> <td align="left">79.50</td> <td align="left">1.92</td> <td align="left">8</td> <td align="left">4</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">81.57</td> <td align="left">0.55</td> <td align="left">0.48</td> </tr> <tr> <td align="left"><b>17c</b></td> <td align="left">449.89</td> <td align="left">444.79</td> <td align="left">79.50</td> <td align="left">2.19</td> <td align="left">8</td> <td align="left">4</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">81.57</td> <td align="left">0.55</td> <td align="left">1.01</td> </tr> <tr> <td align="left"><b>18a</b></td> <td align="left">403.41</td> <td align="left">401.67</td> <td align="left">71.96</td> <td align="left">2.39</td> <td align="left">7</td> <td align="left">4</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">84.17</td> <td align="left">0.55</td> <td align="left">0.65</td> </tr> <tr> <td align="left"><b>18b</b></td> <td align="left">417.44</td> <td align="left">422.61</td> <td align="left">71.96</td> <td align="left">2.61</td> <td align="left">7</td> <td align="left">4</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">84.17</td> <td align="left">0.55</td> <td align="left">0.71</td> </tr> <tr> <td align="left"><b>18c</b></td> <td align="left">437.85</td> <td align="left">418.86</td> <td align="left">71.96</td> <td align="left">2.87</td> <td align="left">7</td> <td align="left">4</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">84.17</td> <td align="left">0.55</td> <td align="left">0.86</td> </tr> <tr> <td align="left"><b>19a</b></td> <td align="left">435.48</td> <td align="left">446.29</td> <td align="left">70.99</td> <td align="left">2.68</td> <td align="left">7</td> <td align="left">3</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">84.51</td> <td align="left">0.55</td> <td align="left">0.14</td> </tr> <tr> <td align="left"><b>19b</b></td> <td align="left">449.50</td> <td align="left">467.23</td> <td align="left">70.99</td> <td align="left">2.88</td> <td align="left">7</td> <td align="left">3</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">84.51</td> <td align="left">0.55</td> <td align="left">0.30</td> </tr> <tr> <td align="left"><b>19c</b></td> <td align="left">469.92</td> <td align="left">463.49</td> <td align="left">70.99</td> <td align="left">3.15</td> <td align="left">7</td> <td align="left">3</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">84.51</td> <td align="left">0.55</td> <td align="left">0.71</td> </tr> <tr> <td align="left"><b>20a</b></td> <td align="left">451.54</td> <td align="left">460.28</td> <td align="left">57.51</td> <td align="left">2.67</td> <td align="left">7</td> <td align="left">2</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">89.16</td> <td align="left">0.55</td> <td align="left">0.11</td> </tr> <tr> <td align="left"><b>20b</b></td> <td align="left">465.57</td> <td align="left">481.22</td> <td align="left">57.51</td> <td align="left">2.87</td> <td align="left">7</td> <td align="left">2</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">89.16</td> <td align="left">0.55</td> <td align="left">0.28</td> </tr> <tr> <td align="left"><b>20c</b></td> <td align="left">485.99</td> <td align="left">477.48</td> <td align="left">57.51</td> <td align="left">3.14</td> <td align="left">7</td> <td align="left">2</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">89.16</td> <td align="left">0.55</td> <td align="left">0.69</td> </tr> <tr> <td align="left"><b>10</b></td> <td align="left">447.92</td> <td align="left">469.04</td> <td align="left">58.48</td> <td align="left">2.92</td> <td align="left">7</td> <td align="left">2</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">88.82</td> <td align="left">0.55</td> <td align="left">0.71</td> </tr> <tr> <td align="left"><b>3</b></td> <td align="left">360.29</td> <td align="left">306.76</td> <td align="left">70.18</td> <td align="left">2.64</td> <td align="left">6</td> <td align="left">6</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">84.79</td> <td align="left">0.55</td> <td align="left">0.98</td> </tr> <tr> <td align="left"><b>Flavopiridol</b></td> <td align="left">401.8</td> <td align="left">382.75</td> <td align="left">71.93</td> <td align="left">1.75</td> <td align="left">2</td> <td align="left">5</td> <td align="left">3</td> <td align="left">Yes</td> <td align="left">84.18</td> <td align="left">0.55</td> <td align="left">1.48</td> </tr> <tr> <td align="left"><b>Palbociclib</b></td> <td align="left">447.53</td> <td align="left">479.27</td> <td align="left">84.30</td> <td align="left">1.13</td> <td align="left">5</td> <td align="left">6</td> <td align="left">2</td> <td align="left">Yes</td> <td align="left">79.92</td> <td align="left">0.55</td> <td align="left">0.62</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>MW: molecular weight; MV: molecular volume (A<sup>3</sup>); TPSA: topological polar surface area (A<sup>2</sup>); MlogP: logP (Moriguchi etal.); RBs: number of rotatable bonds; HB<sub>A</sub>: number of hydrogen bond acceptors; HB<sub>D</sub>: number of hydrogen bond donors.</p> </div> <div class="fn-type-" title=""> <p><sup>a</sup>%Abs (% absorbed orally) = 109âˆ’(0.345 Ã— TPSA; BS: bioavailability score.</p> </div> <div class="fn-type-" title=""> <p><sup>b</sup>DLS: drug-likeness score; Lipinskiâ€™s rule, yes mean no violation; MV, TPSA, and DLS were calculated using Molsoft (<a href="http://www.molsoft.com/mprop/">http://www.molsoft.com/mprop/</a>); other parameters calculated using SwissADME (<a href="http://www.swissadme.ch/">http://www.swissadme.ch/</a>).</p> </div> </div> </div> <p>The molecular weights of the new compounds were in the range 393.48â€"485.99â€‰Da which was less than 500. Moreover, the calculated molecular volumes of the new compounds were in the range of 401.67â€‰âˆ’â€‰481.22 A<sup>3</sup>, which was comparable to multi-CDKIs, flavopiridol and palbociclib. In addition, the new compounds showed topological polar surface areas (TPSA) in the range of 57.51â€"79.50 A<sup>2</sup> compared to 71.93 and 84.30 A<sup>2</sup> for flavopiridol and palbociclib, respectively, <a href="#t0007">Table 7</a>.</p> <p>The Clog P (MlogP) values of the new compounds were in the range of 1.71â€"3.15, which was higher than those of flavopiridol and palbociclib but was comparable with that compound <b>3</b>. Compounds <b>19c</b> and <b>20c</b> were the most lipophilic with ClogP values of 3.15 and 3.14, respectively. The chemical structure of the new compounds have 2â€"4 hydrogen bond acceptors (HB<sub>A</sub> &amp;lt;10) and 3 hydrogen bond donors (HB<sub>D</sub> &amp;lt; 5). These results indicated that all the new compounds conform to Lipinskiâ€™s rule. In addition, the new compounds showed drug-likeness score (DLS) in the range of 0.11â€"1.01, compared to 0.62 for palbociclib, <a href="https://doi.org/10.1080/14756366.2020.1837124">Supplementary data (Table S2)</a>. Among the new compounds, compound <b>17c</b> displayed the highest DLS.</p> <p>The expected GIT absorption percent (%Abs) of the new compounds was also calculated according to the previous report<a href="#CIT0055"><sup>55</sup></a>. The new compounds exhibited %Abs in the range of 81.57â€"89.16% compared to 84.79â€"84.18% for flavopiridol.</p> <p>The study also revealed that compounds <b>17a</b>,<b>b</b>, <b>18a</b>,<b>b</b>, <b>19a</b>, and <b>20a</b>,<b>b</b> are expected to be substrates for P-glycoprotein and predicted to be effluated from the CNS by P-glycoprotein, <a href="https://doi.org/10.1080/14756366.2020.1837124">Supplementary data (Table S3)</a>.</p> </div> </div> </div> <div class="conclusions" title="sec"><span class="label" tagx="label" title="label">4.</span><div class="title" tagx="title" title="title">
Conclusions</div> <p>In the current study, a library of 1302 pyrrolizine derivatives was designed based on the pyrrolizine-5-carboxamide scaffold <b>10</b>. The 3D pharmacophore model of the multi-CDKI <b>3</b> was used in the virtual screening of this library. The top-scoring hits were synthesised and evaluated for their cytotoxic activities against three cancer (MCF-7, A2780, and HT29) cell lines. The results revealed cytotoxic activities against the three cancer cell lines with IC<sub>50</sub> values in the range of 0.16â€"34.13â€‰Î¼M. Among the new derivatives, compound <b>18b</b> was the most active against MCF-7 cells (IC<sub>50</sub> = 0.19â€‰Î¼M). In addition, compounds <b>19a</b> and <b>20a</b> were the most potent in inhibiting the growth of and HT29 and A2780 cell lines, respectively. The results of the MTT assay of the new compounds against normal MRC-5 cells also revealed IC<sub>50</sub> values in the range of 0.10â€"11.86â€‰ÂµM. Compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> increased the apoptotic events and arrested MCF-7 cells at the G<sub>1</sub> phase. Mechanistic study of compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> revealed potent <i>inÂ vitro</i> inhibition in CDK-2 activity (IC<sub>50</sub> = 25.53â€"115.3â€‰nM). Among the three derivatives, compound <b>19a</b> exhibited the highest inhibitory activity against CDK-2. <i>In silico</i> ADME study revealed that all new compounds conform to Lipinskiâ€™s rule. In addition, the molecular docking analyses indicated that compounds <b>18b</b>, <b>19a</b>, and <b>20a</b> fit snugly into the active sites of CDK-2/6/9. Among the three derivatives, compound <b>19a</b> also displayed the highest binding scores and exhibited several hydrogen bonding interactions with the three kinases. These preliminary results suggested that compound <b>19a</b> could serve as a promising scaffold in the discovery and development of new potent anticancer agents.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="IENZ_A_1837124_SM9339.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>No potential conflict of interest was reported by the author(s).</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">NS.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Kinase inhibitors: the road ahead</span>. <span class="source" tagx="source" title="source">Nat Rev Drug Discov</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">17</span>:<span class="fpage" tagx="fpage" title="fpage">353</span>â€"<span class="lpage" tagx="lpage" title="lpage">77</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29545548">29545548</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhullar</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lagaron</span><span class="given-names" tagx="given-names" title="given-names">NO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGowan</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Kinase-targeted cancer therapies: progress, challenges and future directions</span>. <span class="source" tagx="source" title="source">Mol Cancer</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">17</span>:<span class="fpage" tagx="fpage" title="fpage">48</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29455673">29455673</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clausen</span><span class="given-names" tagx="given-names" title="given-names">MH.</span></span></span><span class="mixed-article-title" title="mixed-article-title">FDA-approved small-molecule kinase inhibitors</span>. <span class="source" tagx="source" title="source">Trends Pharmacol Sci</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">36</span>:<span class="fpage" tagx="fpage" title="fpage">422</span>â€"<span class="lpage" tagx="lpage" title="lpage">39</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25975227">25975227</a></span></span></li> <li tag="ref"><a name="CIT0004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roskoski</span><span class="given-names" tagx="given-names" title="given-names">R</span><span class="suffix" tagx="suffix" title="suffix">Jr.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Properties of FDA-approved small molecule protein kinase inhibitors: a 2020 update</span>. <span class="source" tagx="source" title="source">Pharmacol Res</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">152</span>:<span class="fpage" tagx="fpage" title="fpage">104609</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31862477">31862477</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doroshow</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kummar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">United States Food and Drug Administration approved oral kinase inhibitors for the treatment of malignancies</span>. <span class="source" tagx="source" title="source">Curr Probl Cancer</span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">37</span>:<span class="fpage" tagx="fpage" title="fpage">110</span>â€"<span class="lpage" tagx="lpage" title="lpage">44</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23972982">23972982</a></span></span></li> <li tag="ref"><a name="CIT0006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tadesse</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caldon</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tilley</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cyclin-dependent kinase 2 inhibitors in cancer therapy: an update</span>. <span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">62</span>:<span class="fpage" tagx="fpage" title="fpage">4233</span>â€"<span class="lpage" tagx="lpage" title="lpage">51</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30543440">30543440</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lees</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faha</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The cdk2 kinase is required for the G1-to-S transition in mammalian cells</span>. <span class="source" tagx="source" title="source">Oncogene</span><span class="year" tagx="year" title="year">1993</span>;<span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">1593</span>â€"<span class="lpage" tagx="lpage" title="lpage">602</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8502482">8502482</a></span></span></li> <li tag="ref"><a name="CIT0008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keyomarsi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pardee</span><span class="given-names" tagx="given-names" title="given-names">AB.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Redundant cyclin overexpression and gene amplification in breast cancer cells</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="year" tagx="year" title="year">1993</span>;<span class="volume" tagx="volume" title="volume">90</span>:<span class="fpage" tagx="fpage" title="fpage">1112</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8430082">8430082</a></span></span></li> <li tag="ref"><a name="CIT0009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cyclin-dependent kinase 2 is an ideal target for ovary tumors with elevated cyclin E1 expression</span>. <span class="source" tagx="source" title="source">Oncotarget</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">20801</span>â€"<span class="lpage" tagx="lpage" title="lpage">12</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26204491">26204491</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamamoto</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monden</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyoshi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cdk2/cdc2 expression in colon carcinogenesis and effects of cdk2/cdc2 inhibitor in colon cancer cells</span>. <span class="source" tagx="source" title="source">Int J Oncol</span><span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">233</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9664116">9664116</a></span></span></li> <li tag="ref"><a name="CIT0011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cyclin-dependent kinase 2 promotes tumor proliferation and induces radio resistance in glioblastoma</span>. <span class="source" tagx="source" title="source">Transl Oncol</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">548</span>â€"<span class="lpage" tagx="lpage" title="lpage">56</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27863310">27863310</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faber</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiles</span><span class="given-names" tagx="given-names" title="given-names">TC.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of cyclin-dependent kinase-2 induces apoptosis in human diffuse large B-cell lymphomas</span>. <span class="source" tagx="source" title="source">Cell Cycle</span><span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">2982</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18156799">18156799</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of CDK2 as a novel target in treatment of prostate cancer</span>. <span class="source" tagx="source" title="source">Future Oncol</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">14</span>:<span class="fpage" tagx="fpage" title="fpage">709</span>â€"<span class="lpage" tagx="lpage" title="lpage">18</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29323532">29323532</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">ZP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">YL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratner</span><span class="given-names" tagx="given-names" title="given-names">ES.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Targeting cyclin-dependent kinases for treatment of gynecologic cancers</span>. <span class="source" tagx="source" title="source">Front Oncol</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">303</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30135856">30135856</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamdouchi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keyser</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Collins</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The discovery of a new structural class of cyclin-dependent kinase inhibitors, aminoimidazo[1,2-a]pyridines</span>. <span class="source" tagx="source" title="source">Mol Cancer Ther</span><span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14749470">14749470</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyatt</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodhead</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berdini</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of N-(4-piperidinyl)-4-(2,6-dichlorobenzoylamino)-1H-pyrazole-3-carboxamide (AT7519), a novel cyclin dependent kinase inhibitor using fragment-based X-ray crystallography and structure based drug design</span>. <span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">51</span>:<span class="fpage" tagx="fpage" title="fpage">4986</span>â€"<span class="lpage" tagx="lpage" title="lpage">99</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18656911">18656911</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seftel</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuruvilla</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouroukis</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The CDK inhibitor AT7519M in patients with relapsed or refractory chronic lymphocytic leukemia (CLL) and mantle cell lymphoma. A Phase II study of the Canadian Cancer Trials Group</span>. <span class="source" tagx="source" title="source">Leuk Lymphoma</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">58</span>:<span class="fpage" tagx="fpage" title="fpage">1358</span>â€"<span class="lpage" tagx="lpage" title="lpage">65</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27750483">27750483</a></span></span></li> <li tag="ref"><a name="CIT0018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yun</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ullah</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery of novel cyclin-dependent kinase (CDK) and histone deacetylase (HDAC) dual inhibitors with potent inÂ vitro and inÂ vivo anticancer activity</span>. <span class="source" tagx="source" title="source">Eur J Med Chem</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">189</span>:<span class="fpage" tagx="fpage" title="fpage">112073</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31991336">31991336</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Honma</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayashi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aoyama</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure-based generation of a new class of potent Cdk4 inhibitors: new de novo design strategy and library design</span>. <span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">44</span>:<span class="fpage" tagx="fpage" title="fpage">4615</span>â€"<span class="lpage" tagx="lpage" title="lpage">27</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11741479">11741479</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Payton</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yakowec</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery and evaluation of dual CDK1 and CDK2 inhibitors</span>. <span class="source" tagx="source" title="source">Cancer Res</span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">66</span>:<span class="fpage" tagx="fpage" title="fpage">4299</span>â€"<span class="lpage" tagx="lpage" title="lpage">308</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16618755">16618755</a></span></span></li> <li tag="ref"><a name="CIT0021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">WrÃ³bel</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KieÅ‚bus</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaczor</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery of nitroaryl urea derivatives with antiproliferative properties</span>. <span class="source" tagx="source" title="source">J Enzyme Inhib Med Chem</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">31</span>:<span class="fpage" tagx="fpage" title="fpage">608</span>â€"<span class="lpage" tagx="lpage" title="lpage">18</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26114307">26114307</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alexander</span><span class="given-names" tagx="given-names" title="given-names">LT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MÃ¶bitz</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drueckes</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Type II inhibitors targeting CDK2</span>. <span class="source" tagx="source" title="source">ACS Chem Biol</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">2116</span>â€"<span class="lpage" tagx="lpage" title="lpage">25</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26158339">26158339</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelazeem</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arafa</span><span class="given-names" tagx="given-names" title="given-names">e-SA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdellatif</span><span class="given-names" tagx="given-names" title="given-names">KR.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Design, synthesis and pharmacological evaluation of novel pyrrolizine derivatives as potential anticancer agents</span>. <span class="source" tagx="source" title="source">Bioorg Chem</span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">53</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24462996">24462996</a></span></span></li> <li tag="ref"><a name="CIT0024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelazeem</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Omar</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pyrrolizines: design, synthesis, anticancer evaluation and investigation of the potential mechanism of action</span>. <span class="source" tagx="source" title="source">Bioorg Med Chem</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">25</span>:<span class="fpage" tagx="fpage" title="fpage">5637</span>â€"<span class="lpage" tagx="lpage" title="lpage">51</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28916158">28916158</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shawky</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abourehab</span><span class="given-names" tagx="given-names" title="given-names">MAS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdalla</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Optimization of pyrrolizine-based Schiff bases with 4-thiazolidinone motif: design, synthesis and investigation of cytotoxicity and anti-inflammatory potency</span>. <span class="source" tagx="source" title="source">Eur J Med Chem</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">185</span>:<span class="fpage" tagx="fpage" title="fpage">111780</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31655429">31655429</a></span></span></li> <li tag="ref"><a name="CIT0026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Recent development of CDK inhibitors: an overview of CDK/inhibitor co-crystal structures</span>. <span class="source" tagx="source" title="source">Eur J Med Chem</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">164</span>:<span class="fpage" tagx="fpage" title="fpage">615</span>â€"<span class="lpage" tagx="lpage" title="lpage">39</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30639897">30639897</a></span></span></li> <li tag="ref"><a name="CIT0027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tutone</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almerico</span><span class="given-names" tagx="given-names" title="given-names">AM.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Recent advances on CDK inhibitors: an insight by means of in silico methods</span>. <span class="source" tagx="source" title="source">Eur J Med Chem</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">142</span>:<span class="fpage" tagx="fpage" title="fpage">300</span>â€"<span class="lpage" tagx="lpage" title="lpage">15</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28802482">28802482</a></span></span></li> <li tag="ref"><a name="CIT0028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ece</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sevin</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The discovery of potential cyclin A/CDK2 inhibitors: a combination of 3D QSAR pharmacophore modeling, virtual screening, and molecular docking studies</span>. <span class="source" tagx="source" title="source">Med Chem Res</span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">22</span>:<span class="fpage" tagx="fpage" title="fpage">5832</span>â€"<span class="lpage" tagx="lpage" title="lpage">43</span>.</span></li> <li tag="ref"><a name="CIT0029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohammad</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Batra</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahiya</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of high-affinity inhibitors of cyclin-dependent kinase 2</span><span class="source" tagx="source" title="source">towards anticancer therapy. Molecules</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">24</span>:<span class="fpage" tagx="fpage" title="fpage">4589</span>.</span></li> <li tag="ref"><a name="CIT0030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakanishi</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murata</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagata</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of protein kinase CK2 inhibitors using solvent dipole ordering virtual screening</span>. <span class="source" tagx="source" title="source">Eur J Med Chem</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">96</span>:<span class="fpage" tagx="fpage" title="fpage">396</span>â€"<span class="lpage" tagx="lpage" title="lpage">404</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25912672">25912672</a></span></span></li> <li tag="ref"><a name="CIT0031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muegge</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Scaffold hopping through virtual screening using 2D and 3D similarity descriptors: ranking, voting, and consensus scoring</span>. <span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">49</span>:<span class="fpage" tagx="fpage" title="fpage">1536</span>â€"<span class="lpage" tagx="lpage" title="lpage">48</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16509572">16509572</a></span></span></li> <li tag="ref"><a name="CIT0032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">J-W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M-Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of novel CDK2 inhibitors by a multistage virtual screening method based on SVM, pharmacophore and docking model</span>. <span class="source" tagx="source" title="source">J Enzyme Inhib Med Chem</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">35</span>:<span class="fpage" tagx="fpage" title="fpage">235</span>â€"<span class="lpage" tagx="lpage" title="lpage">44</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31760818">31760818</a></span></span></li> <li tag="ref"><a name="CIT0033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poulsen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">William</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanchard</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure-based design of nitrogen-linked macrocyclic kinase inhibitors leading to the clinical candidate SB1317/TG02, a potent inhibitor of cyclin dependant kinases (CDKs), Janus kinase 2 (JAK2), and Fms-like tyrosine kinase-3 (FLT3)</span>. <span class="source" tagx="source" title="source">J Mol Model</span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">119</span>â€"<span class="lpage" tagx="lpage" title="lpage">30</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22820730">22820730</a></span></span></li> <li tag="ref"><a name="CIT0034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horvath</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pharmacophore-based virtual screening</span>. <span class="source" tagx="source" title="source">Methods Mol Biol</span><span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">672</span>:<span class="fpage" tagx="fpage" title="fpage">261</span>â€"<span class="lpage" tagx="lpage" title="lpage">98</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20838973">20838973</a></span></span></li> <li tag="ref"><a name="CIT0035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery of novel nonpeptide allosteric inhibitors interrupting the interaction of CDK2/cyclin A3 by virtual screening and bioassays</span>. <span class="source" tagx="source" title="source">Bioorg Med Chem Lett</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">25</span>:<span class="fpage" tagx="fpage" title="fpage">4069</span>â€"<span class="lpage" tagx="lpage" title="lpage">73</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26316466">26316466</a></span></span></li> <li tag="ref"><a name="CIT0036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sunseri</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koes</span><span class="given-names" tagx="given-names" title="given-names">DR.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pharmit: interactive exploration of chemical space</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">44</span>:<span class="fpage" tagx="fpage" title="fpage">W442</span>â€"<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27095195">27095195</a></span></span></li> <li tag="ref"><a name="CIT0037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">HI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almalki</span><span class="given-names" tagx="given-names" title="given-names">WH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azim</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Design, synthesis, and biological evaluation of some novel pyrrolizine derivatives as COX inhibitors with anti-inflammatory/analgesic activities and low ulcerogenic liability</span>. <span class="source" tagx="source" title="source">Molecules</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">21</span>:<span class="fpage" tagx="fpage" title="fpage">201</span>.</span></li> <li tag="ref"><a name="CIT0038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Attalah</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdalla</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aslam</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ethyl benzoate bearing pyrrolizine/indolizine moieties: design, synthesis and biological evaluation of anti-inflammatory and cytotoxic activities</span>. <span class="source" tagx="source" title="source">Bioorg Chem</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">94</span>:<span class="fpage" tagx="fpage" title="fpage">103371</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31708230">31708230</a></span></span></li> <li tag="ref"><a name="CIT0039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelazeem</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdalla</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pyrrolizine-5-carboxamides: exploring the impact of various substituents on anti-inflammatory and anticancer activities</span>. <span class="source" tagx="source" title="source">Acta Pharm</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">68</span>:<span class="fpage" tagx="fpage" title="fpage">251</span>â€"<span class="lpage" tagx="lpage" title="lpage">73</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31259695">31259695</a></span></span></li> <li tag="ref"><a name="CIT0040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belal</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdel Gawad</span><span class="given-names" tagx="given-names" title="given-names">NM.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis of novel indolizine, diazepinoindolizine and pyrimidoindolizine derivatives as potent and selective anticancer agents</span>. <span class="source" tagx="source" title="source">Res Chem Intermed</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">41</span>:<span class="fpage" tagx="fpage" title="fpage">9687</span>â€"<span class="lpage" tagx="lpage" title="lpage">701</span>.</span></li> <li tag="ref"><a name="CIT0041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elsaady</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edrees</span><span class="given-names" tagx="given-names" title="given-names">FH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gawad</span><span class="given-names" tagx="given-names" title="given-names">NMA.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis and biological evaluation of some novel Schiff base derivatives as potential anticancer agents</span>. <span class="source" tagx="source" title="source">J Chem Pharm Res</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">273</span>â€"<span class="lpage" tagx="lpage" title="lpage">82</span>.</span></li> <li tag="ref"><a name="CIT0042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malki</span><span class="given-names" tagx="given-names" title="given-names">WH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">HEA</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structural-based design, synthesis, and antitumor activity of novel alloxazine analogues with potential selective kinase inhibition</span>. <span class="source" tagx="source" title="source">Eur J Med Chem</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">152</span>:<span class="fpage" tagx="fpage" title="fpage">31</span>â€"<span class="lpage" tagx="lpage" title="lpage">52</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29684708">29684708</a></span></span></li> <li tag="ref"><a name="CIT0043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alkahtani</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alanazi</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aleanizy</span><span class="given-names" tagx="given-names" title="given-names">FS</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis, anticancer, apoptosis-inducing activities and EGFR and VEGFR2 assay mechanistic studies of 5,5-diphenylimidazolidine-2,4-dione derivatives: molecular docking studies</span>. <span class="source" tagx="source" title="source">Saudi Pharm J</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">682</span>â€"<span class="lpage" tagx="lpage" title="lpage">93</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31297023">31297023</a></span></span></li> <li tag="ref"><a name="CIT0044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdalla</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaheen</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdallah</span><span class="given-names" tagx="given-names" title="given-names">QMA</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Proapoptotic activity of Achillea membranacea essential oil and its major constituent 1,8-cineole against A2780 ovarian cancer cells</span>. <span class="source" tagx="source" title="source">Molecules</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">25</span>:<span class="fpage" tagx="fpage" title="fpage">1582</span>.</span></li> <li tag="ref"><a name="CIT0045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Design, synthesis and biological evaluation of pyrimidine derivatives as novel CDK2 inhibitors that induce apoptosis and cell cycle arrest in breast cancer cells</span>. <span class="source" tagx="source" title="source">Bioorg Med Chem</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">26</span>:<span class="fpage" tagx="fpage" title="fpage">3491</span>â€"<span class="lpage" tagx="lpage" title="lpage">501</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29853338">29853338</a></span></span></li> <li tag="ref"><a name="CIT0046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huey</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindstrom</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility</span>. <span class="source" tagx="source" title="source">J Comput Chem</span><span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">30</span>:<span class="fpage" tagx="fpage" title="fpage">2785</span>â€"<span class="lpage" tagx="lpage" title="lpage">91</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19399780">19399780</a></span></span></li> <li tag="ref"><a name="CIT0047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almalki</span><span class="given-names" tagx="given-names" title="given-names">FA.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Carprofen: a theoretical mechanistic study to investigate the impact of hydrophobic interactions of alkyl groups on modulation of COX-1/2 binding selectivity</span>. <span class="source" tagx="source" title="source">SN Appl Sci</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">1</span>:<span class="fpage" tagx="fpage" title="fpage">332</span>.</span></li> <li tag="ref"><a name="CIT0048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almalki</span><span class="given-names" tagx="given-names" title="given-names">FA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bin Ali</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almehmadi</span><span class="given-names" tagx="given-names" title="given-names">OM.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Profens: a comparative molecular docking study into cyclooxygenase-1/2</span>. <span class="source" tagx="source" title="source">Drug Invent Today</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">480</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>.</span></li> <li tag="ref"><a name="CIT0049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Attallah</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibrahim</span><span class="given-names" tagx="given-names" title="given-names">IT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abouzeid</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Design, synthesis, 99mTc labeling, and biological evaluation of a novel pyrrolizine derivative as potential anti-inflammatory agent</span>. <span class="source" tagx="source" title="source">Radiochemistry</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">59</span>:<span class="fpage" tagx="fpage" title="fpage">630</span>â€"<span class="lpage" tagx="lpage" title="lpage">8</span>.</span></li> <li tag="ref"><a name="CIT0050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: Dassault Systems BIOVIA,</span></span><span class="source" tagx="source" title="source">Discovery studio visualizer</span>, v16.1.0.15350. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Diego (CA)</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Dassault Systems</span>; <span class="year" tagx="year" title="year">2016</span>.</span></li> <li tag="ref"><a name="CIT0051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulze-Gahmen</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Toward understanding the structural basis of cyclin-dependent kinase 6 specific inhibition</span>. <span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">49</span>:<span class="fpage" tagx="fpage" title="fpage">3826</span>â€"<span class="lpage" tagx="lpage" title="lpage">31</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16789739">16789739</a></span></span></li> <li tag="ref"><a name="CIT0052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baumli</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hole</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noble</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endicott</span><span class="given-names" tagx="given-names" title="given-names">JA.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The CDK9 C-helix exhibits conformational plasticity that may explain the selectivity of CAN508</span>. <span class="source" tagx="source" title="source">ACS Chem Biol</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">811</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22292676">22292676</a></span></span></li> <li tag="ref"><a name="CIT0053" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daina</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michielin</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zoete</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span>. <span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">42717</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28256516">28256516</a></span></span></li> <li tag="ref"><a name="CIT0054" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipinski</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lombardo</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dominy</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feeney</span><span class="given-names" tagx="given-names" title="given-names">PJ.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <span class="source" tagx="source" title="source">Adv Drug Deliv Rev</span><span class="year" tagx="year" title="year">1997</span>;<span class="volume" tagx="volume" title="volume">23</span>:<span class="fpage" tagx="fpage" title="fpage">3</span>â€"<span class="lpage" tagx="lpage" title="lpage">25</span>.</span></li> <li tag="ref"><a name="CIT0055" /><span class="label" tagx="label" title="label">55</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abraham</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Rate-limited steps of human oral absorption and QSAR studies</span>. <span class="source" tagx="source" title="source">Pharm Res</span><span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">1446</span>â€"<span class="lpage" tagx="lpage" title="lpage">57</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12425461">12425461</a></span></span></li> <li tag="ref"><a name="CIT0056" /><span class="label" tagx="label" title="label">56</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atkins</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gershell</span><span class="given-names" tagx="given-names" title="given-names">LJ.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Selective anticancer drugs</span>. <span class="source" tagx="source" title="source">Nat Rev Drug Discov</span><span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">1</span>:<span class="fpage" tagx="fpage" title="fpage">491</span>â€"<span class="lpage" tagx="lpage" title="lpage">2</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12120255">12120255</a></span></span></li> <li tag="ref"><a name="CIT0057" /><span class="label" tagx="label" title="label">57</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asghar</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Witkiewicz</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knudsen</span><span class="given-names" tagx="given-names" title="given-names">ES.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The history and future of targeting cyclin-dependent kinases in cancer therapy</span>. <span class="source" tagx="source" title="source">Nat Rev Drug Discov</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">14</span>:<span class="fpage" tagx="fpage" title="fpage">130</span>â€"<span class="lpage" tagx="lpage" title="lpage">46</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25633797">25633797</a></span></span></li> <li tag="ref"><a name="CIT0058" /><span class="label" tagx="label" title="label">58</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">UrbÃ¡n</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The impact of early ADME profiling on drug discovery and development strategy</span>. <span class="source" tagx="source" title="source">DDW Drug Discov World</span><span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">73</span>â€"<span class="lpage" tagx="lpage" title="lpage">86</span>.</span></li> <li tag="ref"><a name="CIT0059" /><span class="label" tagx="label" title="label">59</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">AP.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Screening for human ADME/Tox drug properties in drug discovery</span>. <span class="source" tagx="source" title="source">Drug Discov Today</span><span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">357</span>â€"<span class="lpage" tagx="lpage" title="lpage">66</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11267922">11267922</a></span></span></li> <li tag="ref"><a name="CIT0060" /><span class="label" tagx="label" title="label">60</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gleeson</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hersey</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hannongbua</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">In-silico ADME models: a general assessment of their utility in drug discovery applications</span>. <span class="source" tagx="source" title="source">Curr Top Med Chem</span><span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">358</span>â€"<span class="lpage" tagx="lpage" title="lpage">81</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21320065">21320065</a></span></span></li> <li tag="ref"><a name="CIT0061" /><span class="label" tagx="label" title="label">61</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">In-silico ADME studies for new drug discovery: from chemical compounds to Chinese herbal medicines</span>. <span class="source" tagx="source" title="source">Curr Drug Metab</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">18</span>:<span class="fpage" tagx="fpage" title="fpage">535</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28302029">28302029</a></span></span></li> </ul> </div> </div>  </body></html>