<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Front Plant Sci</span><span class="iso-abbrev" title="iso-abbrev">Front Plant Sci</span><span class="publisher-id" title="publisher-id">Front. Plant Sci.</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Plant Science</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1664-462X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7785825">7785825</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fpls.2020.610599">10.3389/fpls.2020.610599</a></span><div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span>: <span class="subject" title="subject">Original Research</span></div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Reprogramming of Tomato Leaf Metabolome by the Activity of Heat Stress Transcription Factor HsfB1</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PaupiÃ¨re</span><span class="given-names" tagx="given-names" title="given-names">Marine Josephine</span></span><a href="#aff1"><sup>1</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/93726/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tikunov</span><span class="given-names" tagx="given-names" title="given-names">Yury</span></span><a href="#aff1"><sup>1</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/534322/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schleiff</span><span class="given-names" tagx="given-names" title="given-names">Enrico</span></span><a href="#aff2"><sup>2</sup></a><a href="#aff3"><sup>3</sup></a><a href="#aff4"><sup>4</sup></a><a href="#c001"><sup>*</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/32517/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bovy</span><span class="given-names" tagx="given-names" title="given-names">Arnaud</span></span><a href="#aff1"><sup>1</sup></a><a href="#c002"><sup>*</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/43163/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fragkostefanakis</span><span class="given-names" tagx="given-names" title="given-names">Sotirios</span></span><a href="#aff2"><sup>2</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/91085/overview" /></span></span></div><span class="citation_author_institution" id="aff1">[], <sup>1</sup><span class="institution" title="institution">Plant Breeding, Wageningen University</span><span class="addr-line" title="addr-line">Wageningen</span><span class="country" tagx="country" title="country">Netherlands</span></span><span class="citation_author_institution" id="aff2">[], <sup>2</sup><span class="institution" title="institution">Faculty of Biological Sciences, Molecular Cell Biology of Plants, Goethe University</span><span class="addr-line" title="addr-line">Frankfurt</span><span class="country" tagx="country" title="country">Germany</span></span><span class="citation_author_institution" id="aff3">[], <sup>3</sup><span class="institution" title="institution">Buchmann Institute for Molecular Life Sciences (BMLS), Goethe University</span><span class="addr-line" title="addr-line">Frankfurt</span><span class="country" tagx="country" title="country">Germany</span></span><span class="citation_author_institution" id="aff4">[], <sup>4</sup><span class="institution" title="institution">Frankfurt Institute of Advanced Studies (FIAS)</span><span class="addr-line" title="addr-line">Frankfurt</span><span class="country" tagx="country" title="country">Germany</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Vasileios Fotopoulos, Cyprus University of Technology, Cyprus</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Christine Helen Foyer, University of Birmingham, United Kingdom; Ertugrul Filiz, Duzce University, Turkey</p> </div> <div class="corresp" title="corresp">
*Correspondence: Enrico Schleiff, <span class="email" tagx="email" title="email">schleiff@bio.uni-frankfurt.de</span></div> <div class="corresp" title="corresp">
Arnaud Bovy, <span class="email" tagx="email" title="email">arnaud.bovy@wur.nl</span></div> <div class="fn-type-other" title="other"> <p>This article was submitted to Plant Abiotic Stress, a section of the journal Frontiers in Plant Science</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-12-12</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">610599</span><span class="history" title="history"><span class="received" title="received">received: 2020-9-26</span><span class="accepted" title="accepted">accepted: 2020-11-30</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">PaupiÃ¨re, Tikunov, Schleiff, Bovy and Fragkostefanakis</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Plants respond to high temperatures with global changes of the transcriptome, proteome, and metabolome. Heat stress transcription factors (Hsfs) are the core regulators of transcriptome responses as they control the reprogramming of expression of hundreds of genes. The thermotolerance-related function of Hsfs is mainly based on the regulation of many heat shock proteins (HSPs). Instead, the Hsf-dependent reprogramming of metabolic pathways and their contribution to thermotolerance are not well described. In tomato (<i>Solanum lycopersicum</i>), manipulation of HsfB1, either by suppression or overexpression (OE) leads to enhanced thermotolerance and coincides with distinct profile of metabolic routes based on a metabolome profiling of wild-type (WT) and HsfB1 transgenic plants. Leaves of HsfB1 knock-down plants show an accumulation of metabolites with a positive effect on thermotolerance such as the sugars sucrose and glucose and the polyamine putrescine. OE of HsfB1 leads to the accumulation of products of the phenylpropanoid and flavonoid pathways, including several caffeoyl quinic acid isomers. The latter is due to the enhanced transcription of genes coding key enzymes in both pathways, in some cases in both non-stressed and stressed plants. Our results show that beyond the control of the expression of Hsfs and HSPs, HsfB1 has a wider activity range by regulating important metabolic pathways providing an important link between stress response and physiological tomato development.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
flavonoid</div> <div class="kwd" title="kwd">
heat stress</div> <div class="kwd" title="kwd">
metabolomics</div> <div class="kwd" title="kwd"><i>Solanum lycopersicum</i></div> <div class="kwd" title="kwd">
transcription</div> <div class="kwd" title="kwd">
phenylpropanoid</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
FP7 People: Marie-Curie Actions<span class="named-content" title="named-content">10.13039/100011264</span></div> </div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Plants trigger a cascade of reactions to acclimate to and survive heat stress (HS) conditions (<a href="#B37">Zhu, 2016</a>). The transcriptional reprogramming marked by the induction or repression of hundreds of genes is central for HS response (HSR) (<a href="#B20">Ohama et al., 2017</a>). Changes in mRNA levels lead to alterations in the proteome and consequently in the metabolome landscape. These changes are required for the accumulation of proteins and metabolites with protective functions (<a href="#B22">PaupiÃ¨re et al., 2014</a>; <a href="#B6">Chaturvedi et al., 2016</a>; <a href="#B9">EchevarrÃ­a-ZomeÃ±o et al., 2016</a>). For instance, antioxidants (e.g., flavonoids) are required for ROS detoxification and osmolytes (e.g., sugars and proline) contribute to the cellular water content adjustment and protection of membrane integrity (<a href="#B23">Rizhsky et al., 2004</a>; <a href="#B14">Guy et al., 2008</a>). The accumulation of specific metabolites prior to HS can also affect the molecular HSR such as Î³-aminobutyric acid (GABA) which has a thermoprotective role (<a href="#B28">Taylor et al., 2010</a>).</p> <p>Heat stress transcription factors (Hsf) control the majority of transcriptional alterations that occur in heat stressed cells including many heat shock proteins (Hsp) (<a href="#B25">Scharf et al., 2012</a>). In addition, Hsfs regulate genes coding for proteins with various functions (<a href="#B4">Brooks et al., 2014</a>; <a href="#B10">Fragkostefanakis et al., 2016</a>, <a href="#B11">2018</a>; <a href="#B15">Huang et al., 2016</a>; <a href="#B29">Tian et al., 2019</a>). Some of the genes required for metabolite production are regulated by Hsfs such as the Hsf1b-controlled galactinol synthase (GolS1) involved in raffinose biosynthesis in <i>Arabidopsis thaliana</i> (<a href="#B21">Panikulangara et al., 2004</a>). HsfA2 knock-down in tomato anthers altered the abundance of several metabolites including the suppression of putrescine and GABA accumulation (<a href="#B10">Fragkostefanakis et al., 2016</a>).</p> <p>Tomato HsfB1 acts as repressor of several Hsfs, but also as co-activator of the master regulator HsfA1a (<a href="#B3">Bharti et al., 2004</a>; <a href="#B11">Fragkostefanakis et al., 2018</a>). HsfB1 suppression leads to the accumulation of several Hsfs and consequently to an increased leaf photosynthetic rate in response to HS (<a href="#B11">Fragkostefanakis et al., 2018</a>). Enhanced expression of HsfB1 has a positive impact on thermotolerance as well, most likely due to an enhanced co-activation of HsfA1a, which leads to the synthesis of Hsps (<a href="#B11">Fragkostefanakis et al., 2018</a>). However, the induction of HS-responsive genes and proteins already under non-stress conditions in HsfB1 overexpression (OE) lines leads to growth and developmental defects including reduced growth and flower abortion. Therefore, both HsfB1 knockdown and OE lines improve thermotolerance, although mediated through different regulatory routes. The importance of HsfB1 for the metabolome status of tomato plants under non-stress and HS conditions has not been explored, although the significant proteome alterations in HsfB1 OE and knock-down plants point to this direction. Here, comparison of the metabolome profile of wild-type (WT) and transgenic plants before and after HS allowed us to identify distinct and common pathways potentially related to enhanced thermotolerance in these lines and provide insights into new levels of regulation of HS resilience by the important HsfB1. Our results uncover an important link between the activity of this particular Hsf and HS-dependent metabolome changes.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterialandtreatments" title="sec"> <div class="title" tagx="title" title="title">
Plant Material and Treatments</div> <p>Wild type <i>Solanum lycopersicum</i> cv. Moneymaker and homozygous plants of T4 transgenic lines overexpressing HsfB1 (B1OE-5, -41, and -45), antisense (B1AS-18, -21), and co-suppression (B1CS-39; <a href="#B11">Fragkostefanakis et al., 2018</a>) were cultivated in greenhouse in pots with soil/perlite (3:1 ratio) under a 16/8 h day/night cycle (25Â°C day/20Â°C night) with light intensity at canopy level of approximately 120 Î¼mol m<sup>â€"2</sup> s<sup>â€"1</sup>. Eight-week-old plants were transferred to a CLF Plant Climatics chamber (120 Î¼mol m<sup>â€"2</sup> s<sup>â€"1</sup> light intensity), exposed to HS (1 h, 38Â°C) and then allowed to recover at 25Â°C after 1.5 h for gene expression analysis or 2 and 6 h for metabolome analysis. The stress was applied at 12.00 am and included a 30 min gradual increase while a 30 min gradual decreased from 38â€"25Â°C took place in the recovery phase which took place in the same chamber that the stress was applied. Control plants (25Â°C) were harvested prior to HS. The two youngest fully emerged leaves were pooled from five independent plants for each condition. The experiment was repeated five times for metabolome profiling and independently three for gene expression analysis (biological replicates), from different set of plants at different days.</p> </div> <div class="extractionofleafmetabolites" title="sec"> <div class="title" tagx="title" title="title">
Extraction of Leaf Metabolites</div> <p>Extraction of polar and semi-polar metabolites was carried out at 20Â°C using water:methanol:choloroform separation (<a href="#B5">Carreno-Quintero et al., 2012</a>; <a href="#B35">Wahyuni et al., 2013</a>). 700 Î¼L of 100% methanol and 300 Î¼L of distilled water were added to 100 mg of frozen homogenized leaf material followed by sonication (20 min) and centrifugation (17,000 Ã— <i>g</i>, 10 min). Supernatant was divided into two portions A and B for GC-TOF-MS and LC-QTOF-MS analyses, respectively. 300 Î¼L of portion (A) was supplemented with 600 Î¼L distilled water and 400 Î¼L chloroform followed by shaking (5 min) and centrifugation (17,000 Ã— <i>g</i>, 10 min). Supernatant (40 Î¼L) was transferred to a chromatographic glass vial (1.5 mL) with a glass insert and dried overnight in a centrifugal evaporator, the vials were then crimp capped under argon flow and subjected to a GC-TOF-MS analysis. Portion B was filtered through 0.2 Î¼m polytetrafluoroethylene (PTFE) filter into 1.5 mL chromatographic glass vials, crimp capped and subjected to a LC-QTOF-MS analysis.</p> </div> <div class="metabolomicprofiling" title="sec"> <div class="title" tagx="title" title="title">
Metabolomic Profiling</div> <p>Semi-polar metabolites (extract B) were profiled by LC-QTOF-MS (<a href="#B30">Tikunov et al., 2010</a>) using a Waters Alliance 2795 HT HPLC system equipped with a Luna C18(2) precolumn (2.0 mm Ã— 4 mm) and an analytical column (2.0 mm Ã— 150 mm, 100 Ã…, particle size 3 Î¼m; Phenomenex) connected to an Ultima V4.00.00 QTOF mass spectrometer (Waters, MS Technologies). Degassed formic acid:ultrapure water (1:1,000, v/v; eluent A) and formic acid:acetonitrile (1:1,000, v/v; eluent B) were pumped at 190 Î¼L min<sup>â€"1</sup> with a linear gradient from 5â€"35% eluent B (45 min), followed by washing and re-equilibration. The column and sample were kept at 40 and 20Â°C, respectively. Ionization was performed using an electrospray ionization source, and masses were detected in negative mode (collision energy: 10 eV; m/z range: 100â€"1,500). Data were recorded with MassLynx 4.0 software (Waters).</p> <p>Polar primary metabolites (extract A) were analyzed with an Optic 3 HP-injector (ATAS) and an Agilent 6890 gas chromatograph coupled to a Pegasus III TOF-MS (Leco; <a href="#B5">Carreno-Quintero et al., 2012</a>). For that 12.5 Î¼L <i>O</i>-methylhydroxylamine hydrochloride (20 mg mL<sup>â€"1</sup> pyridine) was added to the dried samples and incubated (30 min, 40Â°C). Samples were derivatized with 17.5 Î¼L <i>N</i>-methyl-<i>N</i>-trimethylsilyltrifluoroacetamide for 60 min. Two microliters were introduced into the injector at 70Â°C with a split ratio of 20 mL min<sup>â€"1</sup> followed by heating to 240Â°C (6Â°C s<sup>â€"1</sup>). Chromatographic separation was performed using a VF-5ms capillary column (Varian; 30 m Ã— 0.25 mm Ã— 0.25 Î¼m) including a 10-m guardian column (carrier-gas: helium; flow rate: 1 mL min<sup>â€"1</sup>) with a temperature ramp [2 min at 70Â°C, increase to 310Â°C (10Â°C min<sup>â€"1</sup>), 5 min at 310Â°C; transfer line temperature: 270Â°C]. The column effluent was ionized by electron impact at 70 eV. Mass spectra were acquired at 20 scans s<sup>â€"1</sup> within m/z range of 50â€"600, 200Â°C source temperature, 295 s solvent delay, and 1,700 V detector voltage. The data were recorded with ChromaTOF software 2.0.</p> </div> <div class="metabolitedataprocessing" title="sec"> <div class="title" tagx="title" title="title">
Metabolite Data Processing</div> <p>Data were processed in MetAlign software<sup><a href="#footnote1">1</a></sup> to correct for the baseline, eliminate noise, and perform mass spectral alignment of chromatograms (<a href="#B33">Tikunov et al., 2005</a>; <a href="#B8">De Vos et al., 2007</a>). Values lower than the detection threshold, 50 for GC-TOF-MS and 20 for LC-QTOF-MS, in more than three leaf samples once the estimated noise signal was subtracted from the estimated peak signal were omitted. Signals with m/z values &amp;lt;85 Da were considered non-specific and deleted. Compound mass spectra and quantitative ions were extracted from the modified MetAlign outputs as previously described (<a href="#B31">Tikunov et al., 2012</a>) by MsClust (see text footnote 1). Only primary and secondary metabolite quantitatively present in all replicates of at least one of the experimental conditions were used. If a quantitative ion automatically selected by MSClust showed saturation of the MS detector, this ion was replaced by its second or third isotopic ion. Mass spectra produced by MSClust was used for compound annotation.</p> </div> <div class="metaboliteannotation" title="sec"> <div class="title" tagx="title" title="title">
Metabolite Annotation</div> <p>Annotation of semi-polar ions was performed by comparing the detected mass spectra and retention time of compounds with the MoTo metabolite database (<a href="#B19">Moco et al., 2006</a>) and metabolite online databases (Dictionary of Natural Products<sup><a href="#footnote2">2</a></sup>; METLIN<sup><a href="#footnote3">3</a></sup>) according to the Metabolomics Standards Initiative requirements (<a href="#B27">Sumner et al., 2007</a>). Annotation of GC-TOF-MS metabolites was done by automatic mass spectra matching to the Golm metabolome database<sup><a href="#footnote4">4</a></sup> using MS Search v2.0 software of the National Institute of Standards and Technology (NIST). Three compound annotation hits were derived. Compound annotation was based on match factor and the difference of retention index between the library entries and the data. The Identified compounds were annotated level 1 when NMR was performed on the corresponding compounds of the library; level 2 was assigned when an analytical standard was used to annotate the compounds from the library or when a tandem mass spectrometry was performed; and level 3 was assigned when compounds were annotated based on their mass based on existing literature (<a href="#B19">Moco et al., 2006</a>; <a href="#B16">Iijima et al., 2008</a>; <a href="#B12">GÃ³mez-Romero et al., 2010</a>; <a href="#B30">Tikunov et al., 2010</a>, <a href="#B32">2013</a>; <a href="#B34">VallverdÃº-Queralt et al., 2010</a>; <a href="#B24">Roldan et al., 2014</a>).</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical Analysis</div> <p>The metabolite abundance values were normalized by fresh weight. The statistical analyses were performed with the IBM-SPSS statistic software package 20<sup><a href="#footnote5">5</a></sup>. The univariate analysis was performed with one-way ANOVA and a Duncanâ€™s <i>post hoc</i> test (<i>p</i> &amp;lt; 0.05). The multivariate analysis was performed by principal component analysis on log2 transformed and mean centered values with GeneMaths XT (version 2.12). Paired <i>t</i>-tests were used for comparison of gene expression in WT and transgenic lines under same conditions.</p> </div> <div class="transcriptanalysis" title="sec"> <div class="title" tagx="title" title="title">
Transcript Analysis</div> <p>Total RNA was extracted (E.Z.N.A. Plant RNA Kit; Omega BioTek, Norcross, GA, United States) from tomato leaves and cDNA was synthesized with Revert Aid reverse transcriptase (Thermo Scientific) following the manufacturerâ€™s protocols. Expression of genes was determined using quantitative real-time PCR (qRT-PCR) using gene-specific oligonucleotides (<a href="#T1">Table 1</a>), PowerUp SYBR Green Mastermix (ThermoFischer, Carlsbad, CA, United States) and cDNA template, under conditions previously described (<a href="#B11">Fragkostefanakis et al., 2018</a>). Data were analyzed by 2<sup>â€"Î"Î"Ct</sup> method (<a href="#B18">Livak and Schmittgen, 2001</a>) using <i>EF1</i>Î± (Solyc06g005060) as reference gene.</p> <div class="table-wrap_UNKNOWN" id="T1" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p>List of oligonucleotides used for qRT-PCR analysis.</p>  <table frame="hsides" rules="groups" cellspacing="5" cellpadding="5"> <thead> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><b>Gene</b></td> <td valign="top" align="left" rowspan="1" colspan="1"><b>Accession number</b></td> <td valign="top" align="left" rowspan="1" colspan="1"><b>Forward oligonucleotide (5â€²-3â€²)</b></td> <td valign="top" align="left" rowspan="1" colspan="1"><b>Reverse oligonucleotide (5â€²-3â€²)</b></td> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1">HCT</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc03g117600</td> <td valign="top" align="left" rowspan="1" colspan="1">TATGGCACGAGGACTCACTC</td> <td valign="top" align="left" rowspan="1" colspan="1">ATCACCAGCAACAGCAACAG</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">C3H1</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc01g096670</td> <td valign="top" align="left" rowspan="1" colspan="1">CTGTTGCTCGCGACCTCAAG</td> <td valign="top" align="left" rowspan="1" colspan="1">ACCTAATTGTGCCCCTGGAC</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">C3H2</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc10g078240</td> <td valign="top" align="left" rowspan="1" colspan="1">CTGTGGCTCGTGATCCAGCA</td> <td valign="top" align="left" rowspan="1" colspan="1">TGACCCAACATAGAAGTGACCA</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">HQT</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc07g005760</td> <td valign="top" align="left" rowspan="1" colspan="1">AGTCCAGTACCACGACCATG</td> <td valign="top" align="left" rowspan="1" colspan="1">TGGGCTGCGAGGATTTCATA</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">CHS1</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc09g091510</td> <td valign="top" align="left" rowspan="1" colspan="1">GTTCCGTGGACCCAGTGAAT</td> <td valign="top" align="left" rowspan="1" colspan="1">AAAAGGGCTTGGCCTACCA</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">CHS2</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc05g053550</td> <td valign="top" align="left" rowspan="1" colspan="1">GGCCGGCGATTCTAGATCA</td> <td valign="top" align="left" rowspan="1" colspan="1">TTTCGGGCTTTAGGCTCAGTT</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">CHI-1</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc05g010320</td> <td valign="top" align="left" rowspan="1" colspan="1">GAAGCAGTGCTCGATTCCATAAT</td> <td valign="top" align="left" rowspan="1" colspan="1">GTTTTTCACAAACCAACAGTTCTGAT</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">CHI-L</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc05g052240</td> <td valign="top" align="left" rowspan="1" colspan="1">GCGATAGAAGGTAAGGA</td> <td valign="top" align="left" rowspan="1" colspan="1">AGCCAAAGAAGAAATAGTTGT</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">F3H</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc02g083860</td> <td valign="top" align="left" rowspan="1" colspan="1">CACACCGATCCAGGAACCAT</td> <td valign="top" align="left" rowspan="1" colspan="1">GCCCACCAACTTGGTCTTGTA</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">F3â€²H</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc03g115220</td> <td valign="top" align="left" rowspan="1" colspan="1">GCACCACGAATGCACTTGC</td> <td valign="top" align="left" rowspan="1" colspan="1">CGTTAGTACCGTCGGCGAAT</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">FLS</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc11g013110</td> <td valign="top" align="left" rowspan="1" colspan="1">GAGCATGAAGTTGGGCCAAT</td> <td valign="top" align="left" rowspan="1" colspan="1">TGGTGGGTTGGCCTCATTAA</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">3GT</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc07g052640</td> <td valign="top" align="left" rowspan="1" colspan="1">CGAACGACGAAACACTGTTGA</td> <td valign="top" align="left" rowspan="1" colspan="1">TGCAGCATAGATGGCATTGG</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">PAL1</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc10g086180</td> <td valign="top" align="left" rowspan="1" colspan="1">GACTGCAGGAAGGAATCCAA</td> <td valign="top" align="left" rowspan="1" colspan="1">TTAAGCCCAAGGAGTTCACG</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">PAL2</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc05g056170</td> <td valign="top" align="left" rowspan="1" colspan="1">GATCGTTACGCGCTTAGGAC</td> <td valign="top" align="left" rowspan="1" colspan="1">CCTATTGGTGTCCCTTGGAA</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">C4L</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc03g097030</td> <td valign="top" align="left" rowspan="1" colspan="1">CAATGACGATGAAGCCACTG</td> <td valign="top" align="left" rowspan="1" colspan="1">GCAGCATCTGCAATATCTGG</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">C4H</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc05g047530</td> <td valign="top" align="left" rowspan="1" colspan="1">GCTGTTGCTGGTGGTAAGGT</td> <td valign="top" align="left" rowspan="1" colspan="1">GCTTTGCAATGTGCAAGCTA</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">EF1a</td> <td valign="top" align="left" rowspan="1" colspan="1">Solyc06g005060</td> <td valign="top" align="left" rowspan="1" colspan="1">GGAACTTGAGAAGGAGCCTAAG</td> <td valign="top" align="left" rowspan="1" colspan="1">CAACACCCACAGCAACAGTTT</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"><span class="attrib" tagx="attrib" title="attrib"><i>Oligonucleotides were designed using PRIMER3 (<a href="http://bioinfo.ut.ee/primer3-0.4.0/">bioinfo.ut.ee/primer3-0.4.0/</a>).</i></span></div> </div> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical Analysis</div> <p>The abundance values of both platforms were normalized by fresh weight and then log2 transformed. All the statistical analyses were performed with the IBM SPSS statistic software package 20 (see text footnote 5). The univariate analysis was performed with a one-way ANOVA for each compound followed by Duncanâ€™s <i>post hoc</i> test (<i>p</i> &amp;lt; 0.05). The multivariate analysis was performed with a principal component analysis on log2 transformed and mean centered value with GeneMaths XT version 2.12 software.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="profileofprimaryandsecondarymetabolitesintomatoleavesafterhs" title="sec"> <div class="title" tagx="title" title="title">
Profile of Primary and Secondary Metabolites in Tomato Leaves After HS</div> <p>To study the impact of a HS treatment on the tomato leaf metabolome, GC-TOF-MS and LC-QTOF-MS analyses were used to determine the qualitative and quantitative composition of primary and secondary metabolites, respectively. Young leaves of eight-week old WT <i>Solanum lycopersicum</i> cv. Moneymaker were harvested under control temperature (CO, 25Â°C), after an exposure at 38Â°C for 1 h (HS), or following a 2 (2R) or 6 h (6R) recovery from HS at 25Â°C. In total 37 polar metabolites were detected of which 27 were annotated including amino acids, sugars, organic acids, amines, and phenolic acids (<a href="#F1">Figure 1A</a>). In addition, 66 semi-polar metabolites were detected of which 41 were annotated including alkaloids, flavonoids, steroids, phenolic acids, organic acids, and triterpenes (<a href="#F1">Figure 1A</a> and <a href="#DS1">Supplementary Datasets 1</a>, <a href="#DS2">2</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Metabolite profile of wild-type tomato leaves exposed to HS. <b>(A)</b> Categories of metabolites identified in this study. <b>(Bâ€"F)</b> Primary and <b>(Gâ€"P)</b> secondary metabolite levels in leaves of plants exposed to 38Â°C for 1 h (HS), and then allowed to recover for 2 (R2) or 6 h (R6), or kept for the same time at 25Â°C as control sample (CO). Values are MS detector responses â€" arbitrary units and represent the average of four or five independent biological replicates Â± SE. Letters denote statistically significant differences (<i>p</i> &amp;lt; 0.05) based on one-way ANOVA and a <i>post hoc</i> Duncan test.</p>   </div> <p>The profiles of 15 annotated metabolites revealed a significant change in abundance among the samples. Only the alkaloid (Î²-tomatine) shows a significant reduction upon HS when compared to the control (<a href="#F1">Figure 1I</a>), while all others exhibited an accumulation either directly after HS or during the recovery phase. Serine (<a href="#F1">Figure 1C</a>), a triterpene glycoside (<a href="#F1">Figure 1H</a>), and three unknown steroid-like compound (<a href="#F1">Figures 1Nâ€"P</a>) accumulated significantly after 1 h of HS. GABA, myo-inositol, sucrose, a-tomatine, and feruloylquinic acid-<i>O</i>-hexoside significantly accumulated after 2 h of recovery from stress and then returned to control levels (<a href="#F1">Figures 1B,E,F,G,K</a>). Putrescine, dehydrotomatine and another unknown steroid-like compound showed no significant alteration upon 1 h of HS, but a transient increase after 2 h of recovery when compared to HS sample (<a href="#F1">Figures 1D,L,P</a>). Therefore, despite the short stress treatment, we observed a dynamic metabolome alteration during HS and recovery.</p> </div> <div class="hsfb1isimportantforthemetabolitestatusintomatoleavesinresponsetohs" title="sec"> <div class="title" tagx="title" title="title">
HsfB1 Is Important for the Metabolite Status in Tomato Leaves in Response to HS</div> <p>We performed the same analysis on one representative HsfB1 OE (B1OE-5) and one AS (B1AS-21) lines to study the impact of HsfB1 activity on the tomato leaf metabolome. The multivariate analysis including both primary and secondary metabolites showed that most of the variation of the metabolome (PC1 = 67% of the variation) could be attributed to the genetic background. A clear distinction between B1OE samples on the one hand and the WT and B1AS samples on the other hand was observed, independent of the treatment (<a href="#F2">Figure 2</a>). PC2 (12% of variation) separated control from heat stressed samples for all genotypes (<a href="#F2">Figure 2</a>). The variation of the metabolome between control and 6 h recovery state (6R) was the smallest in B1OE and the largest in B1AS samples (<a href="#F2">Figure 2</a>). In the latter PC2 was comparable for HS, 2 h recovery (2R) and 6R samples, while PC2 for WT and B1OE reflected the attenuation of HS and the tendency to recover back to the control status. These results suggest that the activity of HsfB1 may facilitate in the return of the metabolome profile after HS to control levels.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Principal component analysis (PCA). PCA of samples integrating both primary and secondary metabolites with the components 1 (PC1, 67%) and 2 (PC2, 12%) in WT, B1OE and B1AS lines in control (CO) and heat stressed (HS) leaves as well as after 2 (2R) or 6 h (6R) of recovery.</p>   </div> <p>The similarity of the profiles of the 68 annotated metabolites was examined by Pearson distance determination and metabolites were grouped based on complete linkage clustering method using Heatmapper (<a href="#B2">Babicki et al., 2016</a>). This analysis yielded five major groups (<a href="#F3">Figure 3</a>). Groups 1, 2, and 3 contained metabolites that have a diverse profile but have in general moderately or significantly lower levels in B1OE leaves, especially either after 2 or 6 h of recovery (2R, 6R). GABA and steroid_3 levels were significantly reduced in B1OE leaves in HS, 2R, and 6R samples when compared to WT, while, sucrose and glucose content was significantly reduced in 2R and 6R samples of B1OE leaves, respectively.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 3</span></a></div>   <p>Effect of HsfB1 overexpression or suppression on abundance of metabolites in tomato leaves exposed to heat stress. Annotated metabolites were grouped based on their profile in all samples and genotypes using complete linkage clustering method based on Pearson distance measurement. The dendrogram showing similarity in metabolite profiles was constructed by complete linkage clustering and distances were calculated by Pearson index. Metabolite levels are depicted as heat map constructed using heatmapper (<a href="#B2">Babicki et al., 2016</a>). Annotated metabolites and groups are depicted on the right.</p>   </div> <p>In group 1 several metabolites were significantly more abundant in B1AS control leaves when compared to WT, including glucose, sucrose, two sugar isoforms, putrescine, and ethanolamine (<a href="#F3">Figure 3</a>). Metabolites of group 2 such as citric acid, threonine, and malic acid were more abundant in B1AS during HS. In group 3, a triterpene glycoside and an Î±-tomatine were less abundant in B1AS 2R sample when compared to WT corresponding sample. Group 4 contained metabolites which did not show any change upon HS in any of the genotypes, but all of them having significantly reduced levels in B1AS leaves after 2 and 6 h of recovery when compared to WT and B1OE samples. These metabolites included mainly glycoalkaloids, steroids, and triterpenes. Group 5 was comprised of 29 compounds which show constitutively enhanced levels in all B1OE samples when compared to WT or B1AS. The majority of these metabolites are flavonoids, phenolic compounds, and organic acids. In conclusion, silencing and OE of HsfB1 had a different effect on the metabolic profile of tomato leaves in different stages of the HSR.</p> </div> <div class="analysisoftranscriptlevelsofflavonoidpathwaygenes" title="sec"> <div class="title" tagx="title" title="title">
Analysis of Transcript Levels of Flavonoid Pathway Genes</div> <p>The strong constitutive accumulation of flavonoids in HsfB1 OE line prompted the investigation of the expression profile of genes of the corresponding flavonoid pathway in all lines. Here, in addition to B1OE-5 and B1AS-21 used for metabolome analysis, two more independent transgenic HsfB1 antisense lines, one AS (B1AS-18) and one co-suppression (B1CS-39) (both further denoted as AS/CS) and two OE lines (B1OE-41 and B1OE-45) were used and compared to WT (<a href="#B11">Fragkostefanakis et al., 2018</a>). Control and HS samples were treated as described for the metabolome analysis, but recovery samples were harvested 1.5 h after they were returned to 25Â°C, supposing that transcriptional control precedes regulation at protein and metabolite levels. The expression of sixteen genes of the phenylpropanoid/flavonoid pathway (<a href="#B36">Zhang et al., 2015</a>; <a href="#F4">Figure 4A</a>) was studied by qRT-PCR (<a href="#F4">Figure 4B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 4</span></a></div>   <p>Effect of HsfB1 overexpression and suppression on transcript levels of phenylalanine pathway <b>(A)</b>. Transcript levels <b>(B)</b> were determined by qRT-PCR and normalized to wild-type control sample for each gene. Values are the average for three independent OE lines (B1OE-5, -41, -46), and three independent suppression lines (B1AS-18, B1AS-21, B1CS-39), Â± SE. Transcript levels are expressed relative to WT control sample. Asterisk denotes statistically significant difference (<i>p</i> &amp;lt; 0.05) when compared to WT for the same temperature, based on paired <i>t</i>-test. Conditions: C, control; H, heat stress; R, recovery. Genotypes: WT, wild-type; OE, overexpression; AS/CS, antisense/co-suppression.</p>   </div> <p>Phenylalanine to cinnamate conversion (the first reaction in the flavonoid pathway) is catalyzed by phenylalanine ammonia lyase (PAL). In WT plants, the abundance of <i>PAL1</i> transcript was reduced during HS, but enhanced during recovery to levels higher than observed in control sample (<a href="#F4">Figure 4B</a>). In contrast, <i>PAL2</i> was not altered in WT in response to HS (<a href="#F4">Figure 4B</a>). <i>PAL1</i> was enhanced in control leaves, HS and recovering leaves of AS/CS, but not of the OE lines, while <i>PAL2</i> showed increased levels in recovering leaves of both AS/CS and OE lines when compared to WT.</p> <p>Cinnamate 4-hydroxylase (C4H) gene catalyzing the conversion of cinnamate to 4-coumarate was reduced in WT leaves after HS treatment. However, <i>C4H</i> was enhanced in HS and recovering leaves of HsfB1 AS lines when compared to WT. The transcripts of the gene coding for cinnamate 4-hydroxylase (C4L) catalyzing the further conversion of 4-coumarate to 4-coumaroyl-Co-A were enhanced in both OE and AS/CS lines compared to WT.</p> <p>4-coumaroyl-Co-A can be routed either to lignin biosynthesis by convertion to caffeoyl quinate or to the flavonol biosynthesis pathway by conversion to naringenin chalcone, which is mediated by the chalcone synthase enzyme (CHS). Naringenin chalcone is then further converted to naringenin and dehydrokaempferol by chalcone isomerase (CHI) and the flavanone-3-hydroxylase (F3H), respectively. Finally, the activity of the flavonol synthase (FLS), the flavonoid-3â€²-hydroxylase (F3â€²H) and the flavonoid-3-<i>O</i>-glucosyltransferase (3GT) is required for the synthesis of flavonols and their glycosides. All eight tested genes, were downregulated during and/or after HS treatment in WT, AS/CS and OE plants. With the exception of <i>CHI-L</i> and <i>FLS</i>, all genes were affected by HsfB1 OE or knock-down.</p> <p>Under non-stress conditions OE of HsfB1 led to higher levels of <i>CHI</i>, but reduced levels of <i>F3H</i> and <i>F3</i>â€²<i>H</i>. AS/CS of HsfB1 had no effect on these genes prior to HS application. After 1 h of HS, an increased transcript abundance of <i>CHS1</i>, <i>CHS2</i>, and <i>CHI</i> in comparison to WT was observed, while <i>F3H</i> expression was decreased. <i>CHS1</i> and <i>CHS2</i> were also more abundant in the HsfB1 AS/CS lines. After 1.5 h recovery, <i>CHS2</i> remained at the higher level in OE when compared to WT, while the transcript level of the different <i>CHI</i> genes were lower than WT in both, AS/CS and OE lines. <i>3GT</i> was abundant upon recovery of AS/CS lines. Summarizing, both OE and AS/CS of HsfB1 enhanced the levels of <i>CHS</i> genes after HS and recovery, while transcript levels of <i>CHI</i> were suppressed during recovery. In addition, OE of HsfB1 seems to have a negative impact on the expression levels of <i>F3H</i> and <i>F3</i>â€²<i>H.</i></p> <p>The integration of 4-coumaroyl-Co-A into the caffeoyl quinic acid biosynthesis pathway starts with the production of <i>p</i>-coumaroyl-shikimate or <i>p</i>-coumaroyl-quinate by the multifunctional enzyme hydroxycinnamoyl-CoA shikimate/quinate transferase (HCT). Depending on the precursor, the <i>p</i>-coumaroyl ester 3-hydroxylase encoded by the two genes C3H1 and C3H2 either directly produces caffeoyl-quinate or caffeoyl shikimate. The latter is converted to caffeoyl-quinate by the action of the HCT and the hydroxycinnamoyl-CoA quinate transferase (HQT). While <i>HCT</i> and <i>C3H1</i> showed steady state levels after 1 h HS treatment in WT leaves, <i>C3H2</i> and <i>HQT</i> were reduced. Following HS, <i>C3H1</i>, <i>C3H2</i>, and <i>HQT</i> transcript levels were enhanced after 1.5 h recovery at even higher levels than the control WT sample.</p> <p>All four genes showed enhanced transcript levels in the OE lines in HS and both <i>C3H</i> genes in recovery samples when compared to WT. In contrast, only <i>HCT</i> showed increased levels after 1 h of HS in the HsfB1 AC/CS lines. Thus, the branch of caffeoyl quinate synthesis seems to be enhanced by HsfB1 OE.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Exposure to high temperatures leads to major metabolic alterations which support protection and maintenance of housekeeping cellular functions. Out of 68 annotated compounds we observed significant changes in the abundance of 15 metabolites upon a HS treatment in tomato leaves of WT plants (<a href="#F1">Figure 1</a>). While Î²-tomatine levels were reduced after HS treatment, ten compounds had increased levels either directly after the HS treatment or upon recovery when compared to the control leaves (<a href="#F1">Figure 1</a>). Among the accumulated metabolites we detected many that have been previously shown to have a positive impact on thermotolerance such as GABA, sugars, steroids, and several phenylpropanoid compounds and flavonoids (<a href="#B7">Commisso et al., 2016</a>; <a href="#B17">Liu et al., 2019</a>). Alterations in the profile of metabolites indicate either a variance in thermosensitivity levels of key enzymes and/or a variation in the levels of regulation of the relevant genes and proteins.</p> <p>HsfB1 is minimally expressed in non-stressed leaves and therefore its suppression has a lower impact on the variation of metabolome when compared to the effect of the OE (<a href="#F2">Figure 2</a>). These results are in agreement with the proteome variation in HsfB1 transgenic lines under similar conditions (<a href="#B11">Fragkostefanakis et al., 2018</a>). After HS, a recovery of the metabolome back to control levels is observed (<a href="#F2">Figure 2</a>). Thus, protective metabolites are either degraded or used for further processing as observed e.g., for serine or steroid_1 (<a href="#F1">Figure 1</a>). In B1AS leaves the recovery at metabolite level is strongly reduced (<a href="#F2">Figure 2</a>). This holds especially true for metabolites of group 4 (<a href="#F3">Figure 3</a>). The metabolome profile of B1OE line is in general different from that of WT and B1AS suggesting a significant impact of HsfB1 accumulation already under control conditions (<a href="#F2">Figure 2</a>; PC1). Interestingly, as shown by PC2, the metabolome composition of B1OE control is more related to the WT after 6 h of recovery than to WT control (<a href="#F2">Figure 2</a>). Thus, our results highlight an impact on leaf metabolome by HsfB1 accumulation and the importance of the regulation of HsfB1 homeostatic levels particularly for the fine-tuned recovery of the cellular state after HS.</p> <p>B1AS leaves accumulated several sugars including glucose and sucrose in control leaves (<a href="#F3">Figure 3</a>). Sugars can enhance thermotolerance by acting as signaling molecules, osmolytes or contribute via their carbon to oligosaccharide pathways (<a href="#B23">Rizhsky et al., 2004</a>; <a href="#B26">Serrano et al., 2019</a>). In addition, the polyamine putrescine which acts as a radical scavenger (<a href="#B1">Asthir et al., 2012</a>) also accumulated in non-stressed tomato leaves in B1AS line (<a href="#F3">Figure 3</a>). Therefore, suppression of HsfB1 led to an activation of specific metabolic pathways upon HS which can potentially enhance thermotolerance. We assume that HsfB1 may act as repressor of genes coding for enzymes of these pathways.</p> <p>HsfB1 OE led to a higher accumulation of compounds mainly belonging to the phenylpropanoid pathway. B1OE plants accumulate caffeoyl quinic acids and flavonoids, such as kaempferol- and quercetin glycosides, either already in non-stressed leaves or during HS (<a href="#F3">Figure 3</a>). These compounds showed steady state levels in WT and B1AS leaves, indicating that HsfB1 OE shifts metabolism toward specific metabolic routes.</p> <p>Phenolics have important antioxidant activity under stress conditions (<a href="#B13">Grace and Logan, 2000</a>). We did not observe any significant changes in the levels of phenolics upon HS treatment in WT (<a href="#F3">Figure 3</a>). Genes encoding for enzymes in this pathway were downregulated during HS, while the transcripts fully or partially returned to control levels already after 1.5 h of recovery, or even accumulated to higher levels in WT leaves (<a href="#F4">Figure 4</a>). For several genes, HsfB1 OE or suppression led to either a weaker reduction in transcript levels or even an accumulation when compared to WT (<a href="#F4">Figure 4</a>), suggesting that as proposed for Hsf-dependent networks like Hsps (<a href="#B11">Fragkostefanakis et al., 2018</a>), a shift from the homeostatic levels of HsfB1, either up or down, can result in a similar molecular response. Nevertheless, HsfB1 OE resulted in significantly higher levels of the four genes involved in the production of several caffeoyl quinic acid (chlorogenic acid) isomers especially after HS (<a href="#F4">Figure 4</a>), which were found to accumulate in the B1OE line (<a href="#F3">Figure 3</a>). In contrast, <i>F3H</i> and <i>F3</i>â€²<i>H</i> which encode for enzymes required for synthesis of flavonoids are downregulated in B1OE leaves, while the respective flavonol glycosides are increased in these samples (<a href="#F3">Figures 3</a>, <a href="#F4">4</a>). Therefore, while for the phenylpropanoid branch an apparent transcriptional regulation contributes to gene expression of biosynthetic enzymes, flavonoid pathway is not generally affected the same way. We assume that either an enhanced expression of a gene responsible for a rate-limiting step such as <i>CHI</i> may be sufficient to cause an accumulation of flavonoids, or the regulation of this pathway might occur at the posttranscriptional level, for example in a chaperone-dependent manner. We did not identify the corresponding proteins in our previous proteomics study (<a href="#B11">Fragkostefanakis et al., 2018</a>) and thus we cannot fully explain the basis of such differences. The phenylpropanoid pathway is the source for lignin biosynthesis. An enhanced lignification, important for secondary growth can, at least in part, explain the stunted growth observed in B1OE plants (<a href="#B11">Fragkostefanakis et al., 2018</a>).</p> <p>While the impact of HS on the metabolome profile of plants is documented, currently, the integration of metabolic pathways into the central HSR mechanisms, mainly regulated by Hsfs, is not well understood. Here we show that suppression of HsfB1 affects different metabolic pathways compared to the OE. We propose that this cannot be fully dependent on the chaperone activity of HSPs, as we have previously shown that both suppression and OE of HsfB1 lead to enhanced HSP synthesis compared to WT. However, whether HsfB1 directly regulates relevant genes for the respective pathways needs to be experimentally addressed in the future. Nevertheless, our results suggest the diverse functions of Hsf-dependent networks for thermotolerance and expand the current chaperone-oriented models to highlight the importance of readjustment of metabolism to meet cellular and organismic needs for protection and recovery from stress.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability Statement</div> <p>The original contributions presented in the study are included in the article/<a href="#DS1">Supplementary Material</a>, further inquiries can be directed to the corresponding author/s.</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>MP performed the metabolome analysis with the assistance of YT. SF performed the stress treatments and the gene expression analysis. SF, AB, and ES conceptualized and headed the project. SF and ES wrote the manuscript with contribution from all authors. All authors have read and approved the manuscript.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Conflict of Interest</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Funding.</b> This project was funded by SPOT-ITN (grant no. 289220).</p> </div> </div> <div class="ack" title="ack"> <p>We would like to thank all SPOT-ITN members and particularly Klaus-Dieter Scharf for his guidance and insights into the field of heat stress response.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">1</span><p><a href="http://www.metalign.nl">www.metalign.nl</a></p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">2</span><p><a href="http://dnp.chemnetbase.com">http://dnp.chemnetbase.com</a></p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">3</span><p><a href="http://metlin.scripps.edu">http://metlin.scripps.edu</a></p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">4</span><p><a href="http://gmd.mpimp-golm.mpg.de/">http://gmd.mpimp-golm.mpg.de/</a></p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">5</span><p><a href="http://www.ibm.com">www.ibm.com</a></p> </div> </div> <div class="supplementary material" title="supplementary material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fpls.2020.610599/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fpls.2020.610599/full#supplementary-material</a></p> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplemental Dataset 1</span> <p>Metabolites identified by GC-TOF-MS analysis.</p>  <div class="media" title="media"><a href="Data_Sheet_1.XLSX">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplemental Dataset 2</span> <p>Metabolites identified by LC-QTOF-MS analysis.</p>  <div class="media" title="media"><a href="Data_Sheet_2.XLSX">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asthir</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koundal</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bains</span><span class="given-names" tagx="given-names" title="given-names">N. S.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Putrescine modulates antioxidant defense response in wheat under high temperature stress.</span><span class="source" tagx="source" title="source"><i>Biol. Plant.</i></span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">757</span>â€"<span class="lpage" tagx="lpage" title="lpage">761</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10535-012-0209-1">10.1007/s10535-012-0209-1</a></span></span></li> <li tag="ref"><a name="B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babicki</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arndt</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marcu</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grant</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maciejewski</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Heatmapper: web-enabled heat mapping for all.</span><span class="source" tagx="source" title="source"><i>Nucleic Acids Res.</i></span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">W147</span>â€"<span class="lpage" tagx="lpage" title="lpage">W153</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw419">10.1093/nar/gkw419</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27190236">27190236</a></span></span></li> <li tag="ref"><a name="B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bharti</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Von Koskull-DÃ¶ring</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bharti</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tintschl-KÃ¶rbitzer</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Treuter</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Tomato heat stress transcription factor HsfB1 represents a novel type of general transcription coactivator with a histone-like motif interacting with the plant CREB binding protein ortholog HAC1.</span><span class="source" tagx="source" title="source"><i>Plant Cell</i></span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">1521</span>â€"<span class="lpage" tagx="lpage" title="lpage">1535</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.019927">10.1105/tpc.019927</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15131252">15131252</a></span></span></li> <li tag="ref"><a name="B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooks</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nekrasov</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lippman</span><span class="given-names" tagx="given-names" title="given-names">Z. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Eck</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Efficient Gene Editing in Tomato in the First Generation Using the Clustered Regularly Interspaced Short Palindromic Repeats/CRISPR-Associated9 System.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">166</span><span class="fpage" tagx="fpage" title="fpage">1292</span>â€"<span class="lpage" tagx="lpage" title="lpage">1297</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.114.247577">10.1104/pp.114.247577</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25225186">25225186</a></span></span></li> <li tag="ref"><a name="B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carreno-Quintero</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Acharjee</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maliepaard</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bachem</span><span class="given-names" tagx="given-names" title="given-names">C. W. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mumm</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouwmeester</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Untargeted metabolic quantitative trait loci analyses reveal a relationship between primary metabolism and potato tuber quality.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">158</span><span class="fpage" tagx="fpage" title="fpage">1306</span>â€"<span class="lpage" tagx="lpage" title="lpage">1318</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.111.188441">10.1104/pp.111.188441</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22223596">22223596</a></span></span></li> <li tag="ref"><a name="B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaturvedi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghatak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weckwerth</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Pollen proteomics: from stress physiology to developmental priming.</span><span class="source" tagx="source" title="source"><i>Plant Reprod.</i></span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">119</span>â€"<span class="lpage" tagx="lpage" title="lpage">132</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00497-016-0283-9">10.1007/s00497-016-0283-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27271282">27271282</a></span></span></li> <li tag="ref"><a name="B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Commisso</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toffali</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strazzer</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stocchero</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Impact of Phenylpropanoid Compounds on Heat Stress Tolerance in Carrot Cell Cultures.</span><span class="source" tagx="source" title="source"><i>Front. Plant Sci.</i></span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">16</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2016.01439">10.3389/fpls.2016.01439</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26858731">26858731</a></span></span></li> <li tag="ref"><a name="B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Vos</span><span class="given-names" tagx="given-names" title="given-names">R. C. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moco</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lommen</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keurentjes</span><span class="given-names" tagx="given-names" title="given-names">J. J. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bino</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">R. D.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Untargeted large-scale plant metabolomics using liquid chromatography coupled to mass spectrometry.</span><span class="source" tagx="source" title="source"><i>Nat. Protoc.</i></span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">778</span>â€"<span class="lpage" tagx="lpage" title="lpage">791</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2007.95">10.1038/nprot.2007.95</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17446877">17446877</a></span></span></li> <li tag="ref"><a name="B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">EchevarrÃ­a-ZomeÃ±o</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FernÃ¡ndez-Calvino</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castro-Sanz</span><span class="given-names" tagx="given-names" title="given-names">A. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LÃ³pez</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VÃ¡zquez</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castellano</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Dissecting the proteome dynamics of the early heat stress response leading to plant survival or death in Arabidopsis.</span><span class="source" tagx="source" title="source"><i>Plant Cell Environ.</i></span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">1264</span>â€"<span class="lpage" tagx="lpage" title="lpage">1278</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/pce.12664">10.1111/pce.12664</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26580143">26580143</a></span></span></li> <li tag="ref"><a name="B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fragkostefanakis</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mesihovic</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simm</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PaupiÃ¨re</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">HsfA2 controls the activity of developmentally and stress-regulated heat stress protection mechanisms in tomato male reproductive tissues.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">170</span><span class="fpage" tagx="fpage" title="fpage">2461</span>â€"<span class="lpage" tagx="lpage" title="lpage">2477</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.15.01913">10.1104/pp.15.01913</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26917685">26917685</a></span></span></li> <li tag="ref"><a name="B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fragkostefanakis</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simm</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El-Shershaby</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bublak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mesihovic</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">The repressor and co-activator HsfB1 regulates the major heat stress transcription factors in tomato Short title: Regulation of tomato heat stress response by HsfB1.</span><span class="source" tagx="source" title="source"><i>Plant Cell Environ.</i></span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">874</span>â€"<span class="lpage" tagx="lpage" title="lpage">890</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/pce.13434">10.1111/pce.13434</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30187931">30187931</a></span></span></li> <li tag="ref"><a name="B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ³mez-Romero</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Segura-Carretero</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FernÃ¡ndez-GutiÃ©rrez</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Metabolite profiling and quantification of phenolic compounds in methanol extracts of tomato fruit.</span><span class="source" tagx="source" title="source"><i>Phytochemistry</i></span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">1848</span>â€"<span class="lpage" tagx="lpage" title="lpage">1864</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phytochem.2010.08.002">10.1016/j.phytochem.2010.08.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20810136">20810136</a></span></span></li> <li tag="ref"><a name="B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grace</span><span class="given-names" tagx="given-names" title="given-names">S. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Logan</span><span class="given-names" tagx="given-names" title="given-names">B. A.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Energy dissipation and radical scavenging by the plant phenylpropanoid pathway.</span><span class="source" tagx="source" title="source"><i>Philos. Trans. R. Soc. B Biol. Sci.</i></span><span class="volume" tagx="volume" title="volume">355</span><span class="fpage" tagx="fpage" title="fpage">1499</span>â€"<span class="lpage" tagx="lpage" title="lpage">1510</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2000.0710">10.1098/rstb.2000.0710</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11128003">11128003</a></span></span></li> <li tag="ref"><a name="B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guy</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaplan</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kopka</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selbig</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hincha</span><span class="given-names" tagx="given-names" title="given-names">D. K.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Metabolomics of temperature stress.</span><span class="source" tagx="source" title="source"><i>Physiol. Plant.</i></span><span class="volume" tagx="volume" title="volume">132</span><span class="fpage" tagx="fpage" title="fpage">220</span>â€"<span class="lpage" tagx="lpage" title="lpage">235</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1399-3054.2007.00999.x">10.1111/j.1399-3054.2007.00999.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18251863">18251863</a></span></span></li> <li tag="ref"><a name="B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">C.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">C.-R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jinn</span><span class="given-names" tagx="given-names" title="given-names">T.-L.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The heat-stress factor HSFA6b connects ABA signaling and ABA-mediated heat responses.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">172</span>:<span class="issue" tagx="issue" title="issue">00860</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.16.00860">10.1104/pp.16.00860</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27493213">27493213</a></span></span></li> <li tag="ref"><a name="B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iijima</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakamura</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogata</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakurai</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suda</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Metabolite annotations based on the integration of mass spectral information.</span><span class="source" tagx="source" title="source"><i>Plant J.</i></span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">949</span>â€"<span class="lpage" tagx="lpage" title="lpage">962</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-313X.2008.03434.x">10.1111/j.1365-313X.2008.03434.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18266924">18266924</a></span></span></li> <li tag="ref"><a name="B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Regulation of Heat Shock Factor Pathways by Î³-aminobutyric Acid (GABA) Associated with Thermotolerance of Creeping Bentgrass.</span><span class="source" tagx="source" title="source"><i>Int. J. Mol. Sci.</i></span><span class="volume" tagx="volume" title="volume">20</span>:<span class="issue" tagx="issue" title="issue">4713</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms20194713">10.3390/ijms20194713</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31547604">31547604</a></span></span></li> <li tag="ref"><a name="B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Livak</span><span class="given-names" tagx="given-names" title="given-names">K. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmittgen</span><span class="given-names" tagx="given-names" title="given-names">T. D.</span></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Analysis of relative gene expression data using real-time quantitative PCR and.</span><span class="source" tagx="source" title="source"><i>Methods</i></span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">402</span>â€"<span class="lpage" tagx="lpage" title="lpage">408</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1006/meth.2001.1262">10.1006/meth.2001.1262</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a></span></span></li> <li tag="ref"><a name="B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moco</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bino</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vorst</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verhoeven</span><span class="given-names" tagx="given-names" title="given-names">H. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Groot</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Beek</span><span class="given-names" tagx="given-names" title="given-names">T. A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">A liquid chromatography-mass spectrometry-based metabolome database for tomato.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">141</span><span class="fpage" tagx="fpage" title="fpage">1205</span>â€"<span class="lpage" tagx="lpage" title="lpage">1218</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.106.078428">10.1104/pp.106.078428</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16896233">16896233</a></span></span></li> <li tag="ref"><a name="B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohama</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shinozaki</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Transcriptional Regulatory Network of Plant Heat Stress Response.</span><span class="source" tagx="source" title="source"><i>Trends Plant Sci.</i></span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">53</span>â€"<span class="lpage" tagx="lpage" title="lpage">65</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tplants.2016.08.015">10.1016/j.tplants.2016.08.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27666516">27666516</a></span></span></li> <li tag="ref"><a name="B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panikulangara</span><span class="given-names" tagx="given-names" title="given-names">T. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eggers-Schumacher</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wunderlich</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stransky</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¶ffl</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Galactinol synthase1. A novel heat shock factor target gene responsible for heat-induced synthesis of raffinose family oligosaccharides in arabidopsis.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">136</span><span class="fpage" tagx="fpage" title="fpage">3148</span>â€"<span class="lpage" tagx="lpage" title="lpage">3158</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.104.042606">10.1104/pp.104.042606</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15466240">15466240</a></span></span></li> <li tag="ref"><a name="B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PaupiÃ¨re</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Heusden</span><span class="given-names" tagx="given-names" title="given-names">A. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bovy</span><span class="given-names" tagx="given-names" title="given-names">A. G.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">The metabolic basis of pollen thermo-tolerance: Perspectives for breeding.</span><span class="source" tagx="source" title="source"><i>Metabolites</i></span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">889</span>â€"<span class="lpage" tagx="lpage" title="lpage">920</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/metabo4040889">10.3390/metabo4040889</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25271355">25271355</a></span></span></li> <li tag="ref"><a name="B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rizhsky</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shuman</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shulaev</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davletova</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mittler</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">When defense pathways collide. The response of Arabidopsis to a combination of drought and heat stress.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">134</span><span class="fpage" tagx="fpage" title="fpage">1683</span>â€"<span class="lpage" tagx="lpage" title="lpage">1696</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.103.033431.1">10.1104/pp.103.033431.1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15047901">15047901</a></span></span></li> <li tag="ref"><a name="B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roldan</span><span class="given-names" tagx="given-names" title="given-names">M. V. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engel</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vos</span><span class="given-names" tagx="given-names" title="given-names">R. C. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vereijken</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Astola</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Groenenboom</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Metabolomics reveals organ-specific metabolic rearrangements during early tomato seedling development.</span><span class="source" tagx="source" title="source"><i>Metabolomics</i></span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">958</span>â€"<span class="lpage" tagx="lpage" title="lpage">974</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11306-014-0625-2">10.1007/s11306-014-0625-2</a></span></span></li> <li tag="ref"><a name="B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scharf</span><span class="given-names" tagx="given-names" title="given-names">K. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berberich</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ebersberger</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nover</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">The plant heat stress transcription factor (Hsf) family: structure, function and evolution.</span><span class="source" tagx="source" title="source"><i>Biochim. Biophys. Acta</i></span><span class="volume" tagx="volume" title="volume">1819</span><span class="fpage" tagx="fpage" title="fpage">104</span>â€"<span class="lpage" tagx="lpage" title="lpage">119</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbagrm.2011.10.002">10.1016/j.bbagrm.2011.10.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22033015">22033015</a></span></span></li> <li tag="ref"><a name="B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serrano</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ling</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahieldin</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahfouz</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Thermopriming reprograms metabolic homeostasis to confer heat tolerance.</span><span class="source" tagx="source" title="source"><i>Sci. Rep.</i></span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">14</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-36484-z">10.1038/s41598-018-36484-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30626917">30626917</a></span></span></li> <li tag="ref"><a name="B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sumner</span><span class="given-names" tagx="given-names" title="given-names">L. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amberg</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beale</span><span class="given-names" tagx="given-names" title="given-names">M. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beger</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daykin</span><span class="given-names" tagx="given-names" title="given-names">C. A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Proposed minimum reporting standards for chemical analysis.</span><span class="source" tagx="source" title="source"><i>Metabolomics</i></span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">211</span>â€"<span class="lpage" tagx="lpage" title="lpage">221</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11306-007-0082-2">10.1007/s11306-007-0082-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24039616">24039616</a></span></span></li> <li tag="ref"><a name="B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kinnersley</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turano</span><span class="given-names" tagx="given-names" title="given-names">F. J.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Gamma Aminobutyric Acid (GABA) and Plant Responses to Stress.</span><span class="source" tagx="source" title="source"><i>Crit. Rev. Plant Sci.</i></span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">479</span>â€"<span class="lpage" tagx="lpage" title="lpage">509</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07352680091139277">10.1080/07352680091139277</a></span></span></li> <li tag="ref"><a name="B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Heat shock transcription factor A1b regulates heat tolerance in wheat and Arabidopsis through OPR3 and jasmonate signalling pathway.</span><span class="source" tagx="source" title="source"><i>Plant Biotechnol. J.</i></span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">1109</span>â€"<span class="lpage" tagx="lpage" title="lpage">1111</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/pbi.13268">10.1111/pbi.13268</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31559685">31559685</a></span></span></li> <li tag="ref"><a name="B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tikunov</span><span class="given-names" tagx="given-names" title="given-names">Y. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vos</span><span class="given-names" tagx="given-names" title="given-names">R. C. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ParamÃ¡s</span><span class="given-names" tagx="given-names" title="given-names">A. M. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">R. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bovy</span><span class="given-names" tagx="given-names" title="given-names">A. G.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">A role for differential glycoconjugation in the emission of phenylpropanoid volatiles from tomato fruit discovered using a metabolic data fusion approach.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">152</span><span class="fpage" tagx="fpage" title="fpage">55</span>â€"<span class="lpage" tagx="lpage" title="lpage">70</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.109.146670">10.1104/pp.109.146670</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19889876">19889876</a></span></span></li> <li tag="ref"><a name="B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tikunov</span><span class="given-names" tagx="given-names" title="given-names">Y. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laptenok</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">R. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bovy</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vos</span><span class="given-names" tagx="given-names" title="given-names">R. C. H.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">MSClust: A tool for unsupervised mass spectra extraction of chromatography-mass spectrometry ion-wise aligned data.</span><span class="source" tagx="source" title="source"><i>Metabolomics</i></span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">714</span>â€"<span class="lpage" tagx="lpage" title="lpage">718</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11306-011-0368-2">10.1007/s11306-011-0368-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22833709">22833709</a></span></span></li> <li tag="ref"><a name="B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tikunov</span><span class="given-names" tagx="given-names" title="given-names">Y. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Molthoff</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vos</span><span class="given-names" tagx="given-names" title="given-names">R. C. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beekwilder</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Houwelingen</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Hooft</span><span class="given-names" tagx="given-names" title="given-names">J. J. J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Non-smoky GLYCOSYLTRANSFERASE1 prevents the release of smoky aroma from tomato fruit.</span><span class="source" tagx="source" title="source"><i>Plant Cell</i></span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">3067</span>â€"<span class="lpage" tagx="lpage" title="lpage">3078</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.113.114231">10.1105/tpc.113.114231</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23956261">23956261</a></span></span></li> <li tag="ref"><a name="B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tikunov</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lommen</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Vos</span><span class="given-names" tagx="given-names" title="given-names">C. H. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verhoeven</span><span class="given-names" tagx="given-names" title="given-names">H. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bino</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">R. D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">A novel approach for nontargeted data analysis for metabolomics. Large-scale profiling of tomato fruit volatiles.</span><span class="source" tagx="source" title="source"><i>Plant Physiol.</i></span><span class="volume" tagx="volume" title="volume">139</span><span class="fpage" tagx="fpage" title="fpage">1125</span>â€"<span class="lpage" tagx="lpage" title="lpage">1137</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.105.068130">10.1104/pp.105.068130</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16286451">16286451</a></span></span></li> <li tag="ref"><a name="B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VallverdÃº-Queralt</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JÃ¡uregui</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medina-RemÃ³n</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AndrÃ©s-Lacueva</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamuela-RaventÃ³s</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Improved characterization of tomato polyphenols using liquid chromatography/electrospray ionization linear ion trap quadrupole Orbitrap mass spectrometry and liquid chromatography/electrospray ionization tandem mass spectrometry.</span><span class="source" tagx="source" title="source"><i>Rapid Commun. Mass Spectrom.</i></span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">2986</span>â€"<span class="lpage" tagx="lpage" title="lpage">2992</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/rcm.4731">10.1002/rcm.4731</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20872631">20872631</a></span></span></li> <li tag="ref"><a name="B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wahyuni</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ballester</span><span class="given-names" tagx="given-names" title="given-names">A. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tikunov</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vos</span><span class="given-names" tagx="given-names" title="given-names">R. C. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pelgrom</span><span class="given-names" tagx="given-names" title="given-names">K. T. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maharijaya</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Metabolomics and molecular marker analysis to explore pepper (Capsicum sp.) biodiversity.</span><span class="source" tagx="source" title="source"><i>Metabolomics</i></span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">130</span>â€"<span class="lpage" tagx="lpage" title="lpage">144</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11306-012-0432-6">10.1007/s11306-012-0432-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23335867">23335867</a></span></span></li> <li tag="ref"><a name="B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butelli</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alseekh</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tohge</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rallapalli</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Multi-level engineering facilitates the production of phenylpropanoid compounds in tomato.</span><span class="source" tagx="source" title="source"><i>Nat. Commun</i></span>. <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">11</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms9635">10.1038/ncomms9635</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26497596">26497596</a></span></span></li> <li tag="ref"><a name="B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">J. K.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Abiotic Stress Signaling and Responses in Plants.</span><span class="source" tagx="source" title="source"><i>Cell</i></span><span class="volume" tagx="volume" title="volume">167</span><span class="fpage" tagx="fpage" title="fpage">313</span>â€"<span class="lpage" tagx="lpage" title="lpage">324</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2016.08.029">10.1016/j.cell.2016.08.029</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27716505">27716505</a></span></span></li> </ul> </div> </div>  </body></html>