<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Plants (Basel)</span><span class="iso-abbrev" title="iso-abbrev">Plants (Basel)</span><span class="publisher-id" title="publisher-id">plants</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Plants</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2223-7747</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7589717">7589717</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/plants9101392">10.3390/plants9101392</a></span><span class="publisher-id" title="publisher-id">plants-09-01392</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Phenotypic Variability and Genetic Diversity of <i>Phragmites australis</i> in Quebec and Kashmir Reveal Contrasting Population Structure</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-2276-9633</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wani</span><span class="given-names" tagx="given-names" title="given-names">Gowher A.</span></span><a href="#af1-plants-09-01392">1</a><a href="#af2-plants-09-01392">2</a><a href="#c1-plants-09-01392">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">Manzoor A.</span></span><a href="#af1-plants-09-01392">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-1192-7429</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tekeu</span><span class="given-names" tagx="given-names" title="given-names">Honor√©</span></span><a href="#af2-plants-09-01392">2</a><a href="#af3-plants-09-01392">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reshi</span><span class="given-names" tagx="given-names" title="given-names">Zafar A.</span></span><a href="#af1-plants-09-01392">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atangana</span><span class="given-names" tagx="given-names" title="given-names">Alain R.</span></span><a href="#af2-plants-09-01392">2</a><a href="#af4-plants-09-01392">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khasa</span><span class="given-names" tagx="given-names" title="given-names">Damase P.</span></span><a href="#af2-plants-09-01392">2</a></span></div><span class="citation_author_institution" id="af1-plants-09-01392">[1], <span class="email" tagx="email" title="email">mashah@uok.edu.in</span><span class="email" tagx="email" title="email">zreshi@kashmiruniversity.ac.in</span></span><span class="citation_author_institution" id="af2-plants-09-01392">[2], <span class="email" tagx="email" title="email">honore.tekeu.1@ulaval.ca</span><span class="email" tagx="email" title="email">A.Atangana@cgiar.org</span><span class="email" tagx="email" title="email">Damase.Khasa@ibis.ulaval.ca</span></span><span class="citation_author_institution" id="af3-plants-09-01392">[3], </span><span class="citation_author_institution" id="af4-plants-09-01392">[4], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">gowher.wani@skuastkashmir.ac.in</span>; Tel.: +91-700-601-1834</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-10-10</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-10</span></span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1392</span><span class="history" title="history"><span class="received" title="received">received: 2020-9-14</span><span class="accepted" title="accepted">accepted: 2020-10-13</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The origin of differences in traits influencing competitive success between invasive and native wild populations of alien species is subject of debate. Herbarium-based information sources from 2005 onwards about nativity and distributional range of <i>Phragmites australis</i> were used to survey putative native populations of the species in Quebec, and chloroplast DNA (cpDNA) PCR-RFLP analyses identified only one native population, whereas the same analyses revealed that the Kashmir populations are invasive. We compared the native population of <i>P. australis</i> in Quebec (QN), ten populations invasive to Quebec (QE), and five populations invasive in Kashmir, India (KE) using morphometric traits. Using nine cpDNA microsatellite loci, we also compared nine KE populations, ten QE populations, and the QN population. Phenotypic variation was observed among and within populations. Only dry mass of flowers varied across regions. Characterization of morphotypes defined three distinct haplotypes. A bimodal distribution of stem diameter (SD), internode length (IL), leaf length (LL), and leaf width (LW) suggests that a major gene may control growth traits or occurrence of co-selection. High genetic differentiation was observed between populations (<i>R</i><sub>ST</sub> = 0.353) and haplotypes (<i>R</i><sub>ST</sub> = 0.133 to 0.418), indicating limited gene flow and probable local adaptation. Principal coordinates analysis and the neighbor-joining phylogenetic tree clearly distinguished the three haplotypes. Among-populations phenotypic difference (<i>P<sub>S</sub></i><sub>T</sub>) was lower than overall <i>R</i><sub>ST</sub> for plant height, SD, and fresh and dry mass of flowers and seeds, whereas <i>P</i><sub>ST</sub> estimates for LL and LW exceeded among-populations <i>R</i><sub>ST</sub>, suggesting divergent selection, while local adaptation might have occurred in IL, LL, and flower masses. Genetic drift probably influenced among-populations IL differences. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
common reed</div> <div class="kwd" title="kwd">
CpDNA microsatellite loci</div> <div class="kwd" title="kwd">
ecological competitive success</div> <div class="kwd" title="kwd">
haplotypes</div> <div class="kwd" title="kwd">
phenotypic and genetic differentiation</div> <div class="kwd" title="kwd">
plant invasion</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>The spread of invasive alien species (IAS) has received a great deal of attention, given the adverse ecological and economic effects that they can impose on invaded ecosystems [<a href="#B1-plants-09-01392">1</a>,<a href="#B2-plants-09-01392">2</a>]. Why invasive species expand their range faster in non-native regions, but not in their native range, and how these species become dominant components of non-native habitats while being minor constituents of their native ecosystems [<a href="#B3-plants-09-01392">3</a>] are still open questions. Alien plants are those plants that have been introduced to an area from their native range, either accidentally or intentionally, whereas invasive plants are those plants that are non-native to an ecosystem and which may cause a threat to native biodiversity and ecosystem integrity in many ecologically sensitive parts of the world. Very recently, the International Union for Conservation of Nature has come up with a standard Environmental Impact Classification of Alien Taxa (EICAT), based on the magnitude of the detrimental environmental impacts. Earlier studies have indicated that invasive species often possess specific demographic or physiological traits, such as faster growth rates, greater productivity, high fecundity, greater tolerance to environmental extreme events, or very effective dispersal compared to native and non-invasive introduced species [<a href="#B4-plants-09-01392">4</a>,<a href="#B5-plants-09-01392">5</a>,<a href="#B6-plants-09-01392">6</a>,<a href="#B7-plants-09-01392">7</a>]. These traits enable introduced species to spread very extensively and achieve high abundance. The origin of such variations in traits that influence success in competition between invasive and native species under natural conditions may include geographic isolation, genetic drift, divergent selection, phenotypic plasticity [<a href="#B8-plants-09-01392">8</a>], and rapid adaptive evolution [<a href="#B9-plants-09-01392">9</a>,<a href="#B10-plants-09-01392">10</a>]. Whether phenotypic divergence among and between populations of alien species is due to one or more of these factors is a debate that remains to be resolved. </p> <p>The phenotypic differences within and among populations of invasive species in their native and non-native ranges are widely speculated [<a href="#B11-plants-09-01392">11</a>,<a href="#B12-plants-09-01392">12</a>]. The colonization of new areas by invasive species would be associated theoretically with founder diversity [<a href="#B13-plants-09-01392">13</a>], phenotypic plasticity, and environmental conditions at the points of origin [<a href="#B14-plants-09-01392">14</a>], various ecological and evolutionary processes associated with geographical zone [<a href="#B15-plants-09-01392">15</a>], and population bottlenecks that reduce within-population genetic diversity and increase genetic differentiation among populations [<a href="#B16-plants-09-01392">16</a>]. These population genetic differentiations have important effects on species invasiveness, which is apparently linked to increased genetic variation and evolutionary potential [<a href="#B17-plants-09-01392">17</a>]. Moreover, population genetic analyses can increase our understanding of the origin and expansion of invasive species and invasive populations [<a href="#B18-plants-09-01392">18</a>,<a href="#B19-plants-09-01392">19</a>], or haplotypes (i.e., individuals with a specific group of alleles that are inherited within a species), and the same could have useful management implications as well. Previous research has typically focused on detecting how invasive haplotypes have invaded and replaced native haplotypes [<a href="#B20-plants-09-01392">20</a>,<a href="#B21-plants-09-01392">21</a>]. This replacement is most often due to better performance of invasive haplotypes compared to native haplotypes with respect to morphological traits, such as plant height and size, and to reproductive success. However, little attention has been paid to understanding the variation patterns of such invasive haplotypes in their native regions. Further, the causes of morphological differences between invasive and natural populations of IAS are not well understood. Thus, investigating phenotypic and genotypic differentiation between invasive and native haplotypes in wild populations should contribute to the understanding of mechanisms behind the invasion of alien species. </p> <p>Among-population phenotypic differentiation in the wild is effectively estimated using <i>P</i><sub>ST</sub> (i.e., the phenotypic analogue of <i>Q</i><sub>ST</sub>); [<a href="#B22-plants-09-01392">22</a>,<a href="#B23-plants-09-01392">23</a>,<a href="#B24-plants-09-01392">24</a>]. Lande (1992) [<a href="#B25-plants-09-01392">25</a>] stated that genetic variance in phenotypic traits between populations is expected to equal that of variation in neutral molecular loci under the influence of the forces that drive evolution. Indeed, the level of differentiation of quantitative traits should be similar to the level of differentiation at neutral loci, as divergent selection in quantitative traits induces more morphological differences than would be expected for neutral molecular markers [<a href="#B24-plants-09-01392">24</a>]. For this reason, a valuable method for investigating morphological differences between invasive and natural populations of IAS is to compare divergence in allele frequencies to divergence in quantitative traits. Whether selection or genetic drift is responsible for phenotypic divergence is a complex task and requires the initial step of comparing genetic differentiation with phenotypic differentiation. Phenotypic differences between populations are often assumed to be adaptive, and epigenetics are reported to play key role on adaptation of invasive species [<a href="#B26-plants-09-01392">26</a>]. Both molecular and quantitative variations can be influenced by non-adaptive processes that involve random genetic drift and gene flow [<a href="#B27-plants-09-01392">27</a>,<a href="#B28-plants-09-01392">28</a>]. Under these conditions, it would be especially challenging and interesting to determine whether phenotypic differences between populations represent the outcomes of adaptive or non-adaptive processes.</p> <p>Common reed (<i>Phragmites australis</i> [Cav.] Trin. ex Steud; Poaceae) is a polyploid perennial grass species with clonal architecture [<a href="#B29-plants-09-01392">29</a>,<a href="#B30-plants-09-01392">30</a>,<a href="#B31-plants-09-01392">31</a>]. The species is distributed worldwide and is highly invasive in Kashmir Himalayan aquatic habitats [<a href="#B32-plants-09-01392">32</a>]. In Kashmir, India, this species was earlier reported as a South American native [<a href="#B33-plants-09-01392">33</a>] and European native [<a href="#B32-plants-09-01392">32</a>], but substantial molecular evidence for the nativity of this species is lacking. Saltonstall (2002) [<a href="#B29-plants-09-01392">29</a>] sampled individuals of <i>P. australis</i> worldwide and recovered 27 haplotypes of this species on the basis of the sequences of two chloroplast DNA (hereafter, cpDNA) fragments. In North America, both native and exotic haplotypes of the species are found. Introduction of an exotic genotype of <i>P. australis</i> (Haplotype M) and its rapid spread, aided by anthropogenic disturbance, has contributed significantly to its invasion and range expansion in North America [<a href="#B29-plants-09-01392">29</a>]. Recently, Guo et al. [<a href="#B34-plants-09-01392">34</a>] showed that evolutionary mechanisms act differently in the native and introduced ranges of <i>P. australis</i> and that invasive populations of European origin have evolved to adapt to a different climate and to human-made habitats in North America. In Quebec, Canada, the native haplotype is characterized by a red stem and ligule base (i.e., the junction of the leaf blade with the leaf sheath) with fewer hairs, compared to the green stem of invasive haplotypes, which have more hairs on the ligule. Moreover, the inflorescence of the native haplotype is less dense relative to that of the invasive haplotype, containing fewer seeds [<a href="#B35-plants-09-01392">35</a>,<a href="#B36-plants-09-01392">36</a>]. The invasive haplotype in Quebec tends to grow preferentially along roads and in other human-made habitats, while the native haplotype prefers undisturbed, pristine habitats [<a href="#B37-plants-09-01392">37</a>,<a href="#B38-plants-09-01392">38</a>,<a href="#B39-plants-09-01392">39</a>,<a href="#B40-plants-09-01392">40</a>]. In India, the invasive <i>P. australis</i> haplotype is found to occur widely in various aquatic habitats, especially in Kashmir Himalayan wetlands. </p> <p>The objectives of this study were: (i) to assess and compare the phenotypic and genetic diversity of invasive populations of <i>P. australis</i> in Kashmir and Quebec, together with native haplotypes of this species in Quebec, and (ii) to assess evolutionary similarities between them.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
2. Results</div> <p>Restriction fragment length polymorphism (hereafter, RFLP) analysis confirmed that there was only one native population (QN1) in our sampled populations, instead of five as thought based on morphological variations. In utilizing the cpDNA PCR-RFLP analyses, we found that the Quebec population QN1 shared the same restriction pattern as native populations in USA [<a href="#B41-plants-09-01392">41</a>]. Further, we found that ten populations (QE1 to QE12) exhibited the same restriction sites as haplotype M [<a href="#B42-plants-09-01392">42</a>]. Results for the sequence <i>trnT</i>-<i>trnL</i> in the native Quebec QN1 population was 99% similar to haplotype E2 (NCBI accession no. AY016325; [<a href="#B42-plants-09-01392">42</a>]). Sequence <i>trnT</i>-<i>trnL</i> in the invasive Quebec QE1 population and sequence <i>trnT</i>-<i>trnL</i> in the Kashmir KE1 population were 99% similar to haplotype 4 (NCBI accession no. AY016327; [<a href="#B42-plants-09-01392">42</a>]). BLAST analysis was done in GenBank NCBI database for the <i>rbcL</i>-<i>psa</i>I intergenic spacer, and for native Quebec QN1, the sequence was identical to haplotype E2 (100% identity with NCBI accession no. JQ409547; Freeland and Vachon Unpublished data). Sequence <i>rbcL</i>-<i>psa</i>I in the invasive Quebec QE1 population was 99% similar to haplotype 4, while Kashmir KE1 population was identical (100% similar) to the same sequence of haplotype 4 (NCBI accession no. AY016335, [<a href="#B42-plants-09-01392">42</a>]). All nine populations from Kashmir displayed the same restriction pattern as haplotype M. Based on this information, we considered the putative native Quebec populations of Lac St-Louis, Parc Louis-Racine, Sabli√®re Colette, and La Pocati√®re as invasive <i>P. australis</i> populations, referring to them instead as QE9, QE10, QE11, and QE12.</p> <div class="2.1.phenotypicdiversity" title="sec"> <div class="title" tagx="title" title="title">
2.1. Phenotypic Diversity</div> <p>The invasive <i>P. australis</i> plants of the Ange-Gardien population, Quebec (QE4) were the tallest (Mean ¬± SE: 230.74 ¬± 55.54 cm height), followed by the invasive haplotypes of Berni√®re, Quebec (QE8: 226.72 ¬± 34.63 cm height), and the invasive haplotypes of Eastman, Quebec (QE6: 212.44 ¬± 68.96 cm height; <a href="#plants-09-01392-t001">Table 1</a>). The thickest plant stems were recorded for invasive <i>P. australis</i> plants from Kuhunus Wullar populations (KE5: 7.83 ¬± 1.362 cm), Ashaibagh (KE2: 7.78 ¬± 1.332 cm) and Rangharstop (KE4: 7.74 ¬± 1.264 cm; Table 5). The heaviest flowers in terms of fresh and dry mass were also recorded in Ashaibagh, Ange-Gardien, and Berni√®re populations (<a href="#plants-09-01392-t001">Table 1</a>). The highest intermodal length was recorded in plants from Berni√®re (QE8: 18.00 ¬± 3.749 cm), La Pocati√®re (QE1: 16.16 ¬± 2.407 cm), and Princeville (QE7: 16.77 ¬± 2.266 cm; <a href="#plants-09-01392-t001">Table 1</a>). The longest leaves were recorded in Ashaibagh, Rangharstop, Ange-Gardien, and Eastmann <i>P. australis</i> populations, whereas the widest leaves were recorded in invasive <i>P. australis</i> plants from Kashmir (<a href="#plants-09-01392-t001">Table 1</a>).</p> <p>Highly significant variation was observed among plants, whereas significant to highly significant variation was found among populations for the measured characters, except for leaf width (<a href="#plants-09-01392-t002">Table 2</a>). The variation between individual plants and between populations most often contributed to the bulk of total variation in the measured characters, except for leaf width (<a href="#plants-09-01392-t002">Table 2</a>). Additionally, among-populations variation was higher than between-haplotypes variation for all the characters that were measured (<a href="#plants-09-01392-t002">Table 2</a>). While <i>P. australis</i> plants from Kashmir had more highly significant (<i>p</i> &amp;lt; 0.001) heavier flower dry mass (3.59 ¬± 0.59 g) than did Quebec plants (2.55 ¬± 0.116 g), invasive <i>P. australis</i> plants had significantly wider leaves (2.04 ¬± 0.036 cm) than did native plants (1.65 ¬± 0.05 cm). Additionally, except for flower and dry seeds mass, no variation was found among regions for all measured characters (<a href="#plants-09-01392-t002">Table 2</a>).</p> </div> <div class="2.2.characterizationofphenotypes" title="sec"> <div class="title" tagx="title" title="title">
2.2. Characterization of Phenotypes</div> <p>To investigate the effects of morphological traits in distinguishing the haplotypes that were investigated, we analyzed the distribution of each trait among haplotypes. Though stem diameter, plant height, leaf length, and width distinguished the haplotypes that were studied, the differences were not significant (<a href="#plants-09-01392-f001">Figure 1</a>). </p> <p>To explore descriptors, we measured seven phenotypes (plant height, stem diameter, fresh flowers, dry flowers, internode length, and leaf length and width) in <i>P. australis</i>. A bimodal distribution was observed for stem diameter, internode length, leaf length, and width (<a href="#plants-09-01392-f002">Figure 2</a>). Normal distributions without any significant skewness were observed for the rest of traits.</p> <p>Principal component analysis (PCA) indicated two principal components with eigen-values greater than 1. The PCA grouped the seven <i>P. australis</i> descriptors into various components, with the first two explaining 79.33% of the variation, while the first principal axis (PC1) alone explained 61.84% of the variation (<a href="#plants-09-01392-f003">Figure 3</a>). The PC1 had high loading for leaf width. The second axis (PC2), explaining about 17.49% of the total variation, was correlated with plant height, stem diameter, fresh flowers and seeds, dry flowers, and seeds, internode length, and leaf length, indicating the usefulness of descriptors in <i>P. australis</i> (<a href="#plants-09-01392-t003">Table 3</a>).</p> </div> <div class="2.3.phenotypicdifferencesandheritability" title="sec"> <div class="title" tagx="title" title="title">
2.3. Phenotypic Differences and Heritability</div> <p>Moderate <i>P</i><sub>ST</sub> estimates were observed among populations, ranging from 0.12 for flower and seed fresh mass to 0.41 for leaf length (<a href="#plants-09-01392-t002">Table 2</a>). The measured traits were weakly (leaf width) to moderately (internode length, leaf length, and flower masses) heritable (<a href="#plants-09-01392-t002">Table 2</a>). </p> </div> <div class="2.4.geneticdiversity" title="sec"> <div class="title" tagx="title" title="title">
2.4. Genetic Diversity </div> <p>The polymorphism information content (PIC) varied from 0.4 (PaGT11) to 0.9, with an average of 0.8 (<a href="#plants-09-01392-t004">Table 4</a>). Based on the number of alleles, it was found that <i>P. australis</i> in Kashmir shows hexaploidy as the number of alleles from each population ranged from 1‚Äì6; for Quebec populations, it shows tetraploidy as the number of alleles ranged from 1‚Äì4. However, Quebec native population plants seem to be diploid, having only 1‚Äì2 alleles (<a href="#plants-09-01392-t004">Table 4</a>). All loci that were surveyed were variable in each population. Within-population genetic diversity indices are summarized in <a href="#plants-09-01392-t005">Table 5</a>. The mean number of alleles per locus ranged from 1.89 to 4.56, and the overall mean number of alleles per locus amounted to 8.56. The mean number of alleles per polymorphic locus ranged from 1.61 to 3.37, whereas the overall mean number of alleles per polymorphic locus amounted to 4.16 (<a href="#plants-09-01392-t005">Table 5</a>). For <i>P. australis</i> populations in Kashmir, <i>A</i> ranged from 2.00 to 4.56 and <i>A</i><sub>p</sub> ranged from 2.04 to 3.37. The range of values for <i>A</i> and <i>A</i><sub>p</sub> was not as wide in the Quebec populations, and respectively, 1.89 to 3.56, and 1.61 to 2.32. Mean percentages of polymorphic loci ranged from 66.67% (KE9, Ganderbal, Kashmir) to 100% (KE1, KE4, KE5, KE6, KE7, and KE8) for Kashmir populations. For the Quebec populations, <i>P</i> ranged from 88.89% (QE1, QE3, QE6, QE8, QE9, QE12, and QN1) to 100% (QE4, QE7, QE10, and QE11). The overall percentage of polymorphic loci was 92.78% for all study populations. Allelic richness for Kashmiri and Quebec populations ranged from 1.81 to 2.54, and 1.54 to 2.18, respectively. Observed and expected heterozygosities for Kashmir populations were 0.70 and 0.54, respectively; for Quebec populations, <i>H</i><sub>O</sub> was 0.38 and <i>H</i><sub>E</sub> was 0.39. Overall mean observed and expected estimates of heterozygosity were 0.52 and 0.46, respectively, while <i>H</i><sub>O</sub> and <i>H</i><sub>E</sub> per population ranged from 0.23 (QN1; Lac St Fran√ßois, Quebec) to 0.78 (KE6; Saderkote wullar, Kashmir) and from 0.20 (QN) to 0.66 (KE6), respectively (<a href="#plants-09-01392-t005">Table 5</a>). </p> </div> <div class="2.5.populationgeneticstructure" title="sec"> <div class="title" tagx="title" title="title">
2.5. Population Genetic Structure</div> <p>High genetic differentiation was observed among populations (overall <i>R</i><sub>ST</sub> = 0.353), ranging from 0.133 between invasive populations from Kashmir and Quebec, to 0.418 between Quebec native and invasive populations. These differences were best illustrated by the results of principal coordinate analysis (<a href="#plants-09-01392-f004">Figure 4</a>). The <i>P. australis</i> plants separated into three groups: Kashmir invasive, Quebec natives, and Quebec invasive, which each formed one group (<a href="#plants-09-01392-f004">Figure 4</a>).</p> <p>The phylogenetic tree (<a href="#plants-09-01392-f005">Figure 5</a>) was congruent with principal coordinate analysis results (<a href="#plants-09-01392-f004">Figure 4</a>), as the populations of native and non-native <i>P. australis</i> that were studied were found to be very structured. Clearly, ten populations belonging to Quebec invasive (QE1, QE2, QE4, QE6, QE7, QE8, QE9, QE10, QE11, and QE12) were grouped together, completely distant from the population of Quebec native (QN1; <a href="#plants-09-01392-f005">Figure 5</a>). Kashmir invasive populations were clustered in the same group, whereas KE9 were close to QN1 population following by a sub-group containing height populations (KE1, KE2, KE3, KE4, KE5, KE6, KE7, and KE8; <a href="#plants-09-01392-f005">Figure 5</a>). </p> </div> <div class="2.6.rst-pstcomparisons" title="sec"> <div class="title" tagx="title" title="title">
2.6. R<sub>ST</sub>- P<sub>ST</sub> Comparisons</div> <p>Among-population phenotypic differentiation estimates for plant height (0.23), plant stem diameter (0.25), and fresh and dry mass of flowers (0.12 and 0.14, respectively) were less than overall <i>R</i><sub>ST</sub> (0.35), whereas <i>P</i><sub>ST</sub> estimate for leaf width (0.39) and leaf length (0.41) exceeded the genetic differentiation at nine cpDNA microsatellite loci among the <i>P. australis</i> populations that were surveyed in the present study. <i>P</i><sub>ST</sub> estimate for internode length (0.33) was comparable with among-population <i>R</i><sub>ST</sub> estimate (0.35).</p> </div> <div class="2.7.relationshipsbetweengeneticandgeographicdistances" title="sec"> <div class="title" tagx="title" title="title">
2.7. Relationships between Genetic and Geographic Distances</div> <p>All the populations from both regions revealed a weak positive relationship between genetic distances and environmental variables (r<sup>2</sup> = 0.389, <i>p</i> &amp;lt; 0.001). When Mantel test was calculated within each region it again showed a weak but significant correlation between the two variables in Quebec (r<sup>2</sup> = 0.158, <i>p</i> ‚â§ 0.05) and non-significant correlation in Kashmir (r<sup>2</sup> = 8.06 √ó 10<sup>‚àí3</sup>, <i>p</i> = 0.5570).</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
3. Discussion</div> <p>The behavior of <i>P. australis</i> in various lakes, wetlands, and other water bodies of Kashmir is that of a typical invader in terms of forming the monotypic stands, attaining great abundance, and outcompeting native species [<a href="#B43-plants-09-01392">43</a>,<a href="#B44-plants-09-01392">44</a>]. Previous studies that were carried out in 2018 [<a href="#B45-plants-09-01392">45</a>] based on field observations reported this species at the stage V (widespread and dominant) on the stage-based hierarchical model (CM model) propounded by Colautti and MacIsaac [<a href="#B46-plants-09-01392">46</a>]. Similarly, earlier studies conducted in 2007 [<a href="#B33-plants-09-01392">33</a>] reported this species as a South American native and Shah et al. [<a href="#B32-plants-09-01392">32</a>] in 2014 reported it as a European native based on published, web-based sources as well as phytogeographical distribution of this species, but substantial molecular evidence for the nativity of this species is lacking. Based on the above evidence, many authors declared that <i>P. australis</i> was introduced in India, especially in Kashmir Himalayan region [<a href="#B44-plants-09-01392">44</a>,<a href="#B45-plants-09-01392">45</a>]. There is hardly any study conducted in Kashmir, India, or elsewhere, that suggests native range of <i>P. australis</i> in this part (Kashmir Himalaya) of Asia. The present study aimed at increasing our understanding of the phenotypic and genetic diversity of invasive populations of <i>P. australis</i> in Kashmir and Quebec. Only one native <i>P. australis</i> population in Quebec was identified during field surveys on reported natural distribution of the species across Quebec province and all the populations from Kashmir turned out to be invasive as revealed by cpDNA PCR-RFLP analyses. Such findings indicate that invasive haplotypes are becoming more dominant in the natural distribution range of the species in the sites of study. This is the first study in India, especially in Kashmir Himalaya, that provides molecular evidence regarding the invasive status of <i>P. australis</i> in this region. Such successful invasion of alien haplotypes in their non-native range is due to the fact that invaders usually perform better than natives in terms of life-history traits such as mature plant size and fecundity, thereby reducing the size of, or even replacing, native populations [<a href="#B4-plants-09-01392">4</a>,<a href="#B7-plants-09-01392">7</a>]. Further studies in this direction need to be carried out by taking samples from a larger global range, including native European populations of <i>P. australis</i>, in order to provide greater understanding of the origin and phylogeography of this species.</p> <div class="3.1.phenotypicvariation" title="sec"> <div class="title" tagx="title" title="title">
3.1. Phenotypic Variation</div> <p>Given the premise that the invasive populations perform better than native ones in terms of various growth-related traits, we found the highest values of seven growth-related traits of <i>P. australis</i> within invasive populations. These results are congruent with that of other studies [<a href="#B47-plants-09-01392">47</a>,<a href="#B48-plants-09-01392">48</a>,<a href="#B49-plants-09-01392">49</a>], although DeVrie et al. [<a href="#B50-plants-09-01392">50</a>] indicated that morphological characters cannot discriminate reliably between invasive and native <i>P. australis</i> individuals in North America. Further, local environment highly affects phenotypic traits in common reed [<a href="#B51-plants-09-01392">51</a>], explaining the difficulty in differentiating invasive and native individuals from these traits. However, plasticity in common reed is trait-specific [<a href="#B51-plants-09-01392">51</a>,<a href="#B52-plants-09-01392">52</a>]. Similarly, Pysek et al. [<a href="#B53-plants-09-01392">53</a>] compared the functional traits of 89 populations in a common garden experiment belonging to distinct clones of different origin: North America, Europe, Australia (two groups including native and invasive populations introduced from Europe), South Africa, and Far East. In general, both native and invasive populations with origins in Europe showed highest performance-related traits than other groups. North American native populations performed more poorly than both European-related groups and were similar in this respect to Australian and South African populations [<a href="#B53-plants-09-01392">53</a>]. Invasive European populations in North America do suppress the native North American populations wherever growing in direct competition [<a href="#B54-plants-09-01392">54</a>]. The weaker performance of Australian and South African populations in the Northern Hemisphere is due to weak genetic differences or environmental similarities [<a href="#B53-plants-09-01392">53</a>].</p> <p>Plant height plays an important role in invasiveness, as Crawley et al. [<a href="#B55-plants-09-01392">55</a>] and Canavan et al. [<a href="#B56-plants-09-01392">56</a>] found that tall-statured grasses harbor a useful functional trait for invasion success. In their review, Mozdzer et al. [<a href="#B57-plants-09-01392">57</a>] clearly indicated that introduced and native <i>P. australis</i> lineages differ both physiologically and morphologically. Introduced plants are generally taller and occur in greater densities, which results in greater productivity in the introduced lineage in nearly every study.</p> <p>In the present study, plant stem diameter appeared to be linked to invasiveness of <i>P. australis,</i> given that the highest mean values were recorded for invasive populations. Yet Muth and Pigliucci [<a href="#B58-plants-09-01392">58</a>], in comparing six growth-related traits, found very little variation among introduced invasive and introduced non-invasive plant species within two genera of Asteraceae (<i>Crepis</i> and <i>Centaurea</i>), with respect to stem diameter. As our study included one native population, we suggest that any assumptions need to be confirmed with fine-scale studies over a broader distribution range of <i>P. australis</i>, taking into account an even larger number of populations. </p> <p>Abundant flowering may also play an important role in reproductive success, which obviously influences propagule pressure and invasiveness. Our results that invasive plants in both regions significantly produce more flowers and seeds than native plants are quite striking. These traits can be taken as the indicator/s of plant invasiveness. In a survey of 46 native and 45 introduced populations of <i>Solidago gigantea</i>, Jakobs et al. [<a href="#B59-plants-09-01392">59</a>] found that total plant biomass was larger for invasive than native plants. Some genetic studies have shown that abundant seed production and seed-based recruitment are more common in <i>P. australis</i> compared to dispersal by stolons and rhizomes [<a href="#B60-plants-09-01392">60</a>,<a href="#B61-plants-09-01392">61</a>,<a href="#B62-plants-09-01392">62</a>]. Albert et al. [<a href="#B63-plants-09-01392">63</a>], however, attributed the success of this invader in North America to a reproduction strategy combining both the advantages of generative and vegetative propagations.</p> <p>Comparing the leaf characteristics of invasive and native populations showed that the former exhibit higher values for leaf area than the latter. Bimodal distribution observed for traits related to stem diameter, internode distance, leaf length, and width suggest that a major gene controls those four important characters related to leaf and stem sizes in <i>P. australis</i>. Although there was not significant among-population variation in leaf width, results from the present study revealed maximum leaf length and leaf width for invasive populations, while the lowest values were encountered in native population. Furthermore, variation in leaf width was very highly significant between haplotypes in the present study, with invaders appearing to perform better with respect to this quantitative trait as well. Successful invaders are characterized by a wider range of leaf areas and by greater phenotypic plasticity [<a href="#B64-plants-09-01392">64</a>]. Several studies have shown that invasive species have higher resource capture-related traits and possess more efficient photosynthetic machinery compared to native or non-invasive species [<a href="#B11-plants-09-01392">11</a>,<a href="#B65-plants-09-01392">65</a>,<a href="#B66-plants-09-01392">66</a>,<a href="#B67-plants-09-01392">67</a>]. Invasiveness is strongly related to leaf traits that are associated with rapid C capture and net CO<sub>2</sub> assimilation [<a href="#B68-plants-09-01392">68</a>,<a href="#B69-plants-09-01392">69</a>] and relative growth rate [<a href="#B70-plants-09-01392">70</a>]. </p> </div> <div class="3.2.characterizationofphenotypes" title="sec"> <div class="title" tagx="title" title="title">
3.2. Characterization of Phenotypes</div> <p>The characterization of among-haplotype variation could provide clues on co-selection of traits in case hitchhiking occurs [<a href="#B71-plants-09-01392">71</a>]. Indeed, a group of characters may distinguish closely related taxa, which are geographically distant, because of genetic drift or hitchhiking. Similar effects are also observed when a major gene controls the characters of interest. Results of the present study revealed that at least stem diameter, leaf length, and width may undergo co-selection or the traits are under the control of the major gene. Genome-wide association studies in the species may provide more insights on the issue. </p> </div> <div class="3.3.genotypicstudies" title="sec"> <div class="title" tagx="title" title="title">
3.3. Genotypic Studies</div> <p>After obtaining useful insights into phenotypic trait-based variations in native and invasive populations‚Äô vis-√†-vis invasiveness in <i>P. australis</i>, our objective was to confirm such variations at molecular level. Results from the present study indicate that in general, highest levels of genetic diversity are found in invasive populations, while the lowest levels of diversity are recorded in native population, consistent with [<a href="#B72-plants-09-01392">72</a>]. The highest levels of genetic diversity in invasive populations may explain adaptability of invaders in non-native habitats [<a href="#B9-plants-09-01392">9</a>,<a href="#B10-plants-09-01392">10</a>]. The low levels of genetic diversity in the only one native population that was identified in the present study are congruent with the results of [<a href="#B73-plants-09-01392">73</a>], which observed lower levels of genetic diversity in native <i>P. australis</i> populations than in invaders at many of cpDNA loci. This is contrary to what was observed by Zhao et al. [<a href="#B74-plants-09-01392">74</a>], who reported highest genetic diversity among native Mexican populations of <i>Solanum rostratum</i> Dunal and significantly reduced diversity in invasive populations within the USA and China, using ten microsatellite markers. Since cpDNA PCR-RFLP haplotype identification analyses in the present study revealed only one native population, instead of five as thought on the basis of morphological variations, and all the other populations are exotic, it seems that invasion has either replaced the native populations or intraspecific hybridization might have occurred between native and introduced lineages of a species that has increased invasiveness and lead to the decline of native lineages [<a href="#B54-plants-09-01392">54</a>]. We suggest that a broad-scale study including more continents than what has been done in our study, after identifying more native populations, be carried out to confirm or refute our findings.</p> <p>Being a cryptic invader, <i>P. australis</i> has emerged as an important model system for understanding mechanisms of invasion. There have been few population-level studies to directly evaluate differences in genetic diversity, reproductive mode, and dispersal between native and non-native <i>P. australis</i>, particularly in western North America [<a href="#B62-plants-09-01392">62</a>]. In view of the lack of any population genetics studies on <i>P. australis</i> in India, we have attempted here to assess its genetic diversity through allelic and population variation of microsatellite loci. It is noteworthy that the mean number of alleles per locus in the present study (<i>A</i> = 8.56) was comparable to the findings of Saltonstall [<a href="#B73-plants-09-01392">73</a>] on European <i>P. australis</i> populations (<i>A</i> = 8.0), although it was greater than the average (<i>A</i> = 7.6) as reported by Paul et al. [<a href="#B75-plants-09-01392">75</a>] on <i>P. australis</i> in the United Kingdom. In the present study, we most frequently came across tetraploidy and hexaploidy among regional groupings of invasive populations of <i>P. australis</i> that were collected across Quebec and Kashmir, although the native population in Quebec seems to be diploid. Hence, it appears that polyploidy might have played some role in invasiveness of this species. Clevering and Lissner [<a href="#B76-plants-09-01392">76</a>] found that tetraploidy was the most common ploidy of <i>P. australis</i> in Europe and North America, while octaploids (8x) were predominated in Asia. Such results suggest that invasive plants are more likely to be polyploids, while native plants are more likely to be diploids as was reported in some other studies [<a href="#B77-plants-09-01392">77</a>,<a href="#B78-plants-09-01392">78</a>]. Pandit et al. [<a href="#B79-plants-09-01392">79</a>] showed that invasive plant species are generally found to have high chromosome counts and to be polyploids, in contrast to rare species that are diploids and have lower ploidy ratios. In a related study, it was shown that the highly invasive plant species in Singapore were all polyploids [<a href="#B80-plants-09-01392">80</a>], thereby indicating that polyploidy contributes to invasion success. Interestingly, invasive Kashmir and invasive Quebec populations of <i>P. australis</i> were more similar to one another, both in terms of the number of alleles and fragment size, than to the native Quebec population that showed unique numbers of alleles and fragment sizes. With an increased native population size, more differences might be seen. However, Saltonstall (2003a) [<a href="#B30-plants-09-01392">30</a>] reported the same allelic variation among native <i>P. australis</i>, while highest allelic diversity was observed among introduced <i>P. australis</i> that were collected in North America.</p> <p>Our results indicate that <i>P. australis</i> populations that were studied are structured, which is in agreement with Guo et al. [<a href="#B34-plants-09-01392">34</a>]. While the latter study revealed the presence of substructures to delineate the population structure in common reed, Guo et al. [<a href="#B81-plants-09-01392">81</a>] postulated that the climatic niche had shifted between the native and introduced populations of the invasive European lineage. The authors reported strong effects of geography and environment on the genetic structure of populations in the native range, but these effects were counteracted by human-related factors manifested through colonization of anthropogenic habitats in the introduced range. Another possible explanation of the substructures observed in our study may be borrowed from Albert et al. [<a href="#B63-plants-09-01392">63</a>], who while investigating the reproduction of <i>P. australis</i> in eastern North America proposed that long-distance seed dispersal is imperative for common reed along roadsides and in marshes; whereas, plant fragments as well as seeds were shown to be important for dispersal to short distances along roads at least in the areas of its abundance. Further, robust reproductive strategy, both through sexual and asexual modes, was shown to mainly contribute to the invasive success of this species in North America. Our phylogenetic analysis easily identified three mains groups, with each group showing clear specificity in terms of the investigated traits.</p> </div> <div class="3.4.phenotypicvs.genotypicdifferentiation" title="sec"> <div class="title" tagx="title" title="title">
3.4. Phenotypic vs. Genotypic Differentiation</div> <p>Our results suggest that environmental conditions in Quebec and Kashmir are not as much contrasting to induce divergent selection on quantitative traits through strong local population pressures [<a href="#B82-plants-09-01392">82</a>]. This is because there was no major variation in genetic structure between Quebec and Kashmir <i>P. australis</i> populations, and no significant phenotypic variation was recorded between Quebec and Kashmir populations, except for flower and seed dry mass. Such non-spatial geographic differences in phenotypic traits are not common and may be caused by probable multiple introductions [<a href="#B17-plants-09-01392">17</a>,<a href="#B83-plants-09-01392">83</a>]. However, difference in flower and seed dry mass between the regions indicates that there might be differential effect of selection on quantitative characters.</p> <p>The among-population phenotypic divergence that was observed in this study regardless of region is not surprising and may be explained by several factors, including plasticity, genetic drift, and divergent selection. As <i>R</i><sub>ST</sub> differed to <i>P</i><sub>ST</sub> for six out of seven studied characters, divergent selection appears as the most probable cause of among-populations difference in quantitative traits [<a href="#B84-plants-09-01392">84</a>]. No apparent physical barrier to gene flow existed between sampled populations within the region, suggesting that genetic barriers such as different ploidy levels or the high rate of local lateral vegetative propagation of the species may explain the reduced gene flow among populations. Results from the present study are in contrast with that of [<a href="#B30-plants-09-01392">30</a>], which investigated patterns of differentiation in nine microsatellite loci in <i>P. australis</i>. Whether the observed phenotypic divergence in <i>P. australis</i> populations is heritable is not known. Investigating fine-scale local spatial invasions (i.e., within geographic regions) would certainly shed more light on the possible contributions of plasticity to patterns of among-population phenotypic variation in <i>P. australis</i>.</p> <p>The <i>R</i><sub>ST</sub>-<i>P</i><sub>ST</sub> contrast is useful for evaluating the relative contributions of non-adaptive and adaptive processes in shaping quantitative traits. Local adaptation and evolutionary potential have been advocated as greatly influencing the outcomes of invasion by alien species [<a href="#B10-plants-09-01392">10</a>,<a href="#B17-plants-09-01392">17</a>]. Heritability estimates of the characters that were studied may provide clues on local adaptation. Results of the present study indicate local adaptation of invasive haplotypes for internode length, leaf length, and flower masses, indicating that these traits may influence invasion success in the species. On the other hand, isolation by distance or divergent selection are the main hypotheses that explain phenotypic divergence in nature within species [<a href="#B8-plants-09-01392">8</a>]. As we observed a contrast between <i>R</i><sub>ST</sub> and <i>P</i><sub>ST</sub> estimates for six out of the seven traits that were investigated, we tested the correlation between genetic distances and geographical distances. Our results showed no or weak relationships between genetic and geographic distances in the regions that were surveyed. Thus, the hypothesis that could explain the phenotypic divergence observed between populations in traits that were investigated in our study is divergent selection for plant height, plant stem diameter, fresh and dry flower mass, leaf length, and leaf width coupled with local adaptation in internode length, leaf length, and flower masses. We also postulate that internode length variation between populations are explained by genetic drift. However, a large fine-scale study would confirm these findings.</p> </div> </div> <div class="4.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
4. Materials and Methods</div> <div class="4.1.studyareaandtargetspecies" title="sec"> <div class="title" tagx="title" title="title">
4.1. Study Area and Target Species</div> <p>The target species for the present study was <i>P. australis</i>. We specifically studied wild, alien invasive <i>P. australis</i> populations in the province of Quebec, Canada and Kashmir Himalaya, India (<a href="#plants-09-01392-f006">Figure 6</a> and <a href="#plants-09-01392-f007">Figure 7</a>), in addition to five putative native populations of this species in Quebec that were assumed to be native. Information about nativity and distributional range of the species was obtained from web-based sources, such as the Germplasm Resource Information Network (GRIN; <span class="uri" title="uri"><a href="http://www.ars-grin.gov/">http://www.ars-grin.gov/</a></span>), the United States Department of Agriculture (USDA Agricultural Research Service 2015 <span class="uri" title="uri"><a href="http://plants.usda.gov/">http://plants.usda.gov/</a></span> [<a href="#B85-plants-09-01392">85</a>]), and the Integrated Taxonomic Information System (USDA Germplasm Resources Information Network <span class="uri" title="uri"><a href="https://doi.org/10.15482/USDA.ADC/1212393">https://doi.org/10.15482/USDA.ADC/1212393</a></span> [<a href="#B86-plants-09-01392">86</a>]). However, information about the native populations in Quebec were obtained from the specimens of the target species that were stored in the two main herbaria of Quebec, viz., Louis-Marie Herbarium (QFA, Universit√© Laval) and Marie-Victorin Herbarium (MT, Institut de recherche en biologie v√©g√©tale, Universit√© de Montr√©al), together with those held in the Kashmir University Herbarium (KASH), and the list of native populations provided by Lavoie et al. [<a href="#B87-plants-09-01392">87</a>] onwards were also consulted.</p> </div> <div class="4.2.samplingprocedure" title="sec"> <div class="title" tagx="title" title="title">
4.2. Sampling Procedure </div> <p>A total of 16 populations (<a href="#plants-09-01392-t006">Table 6</a>) were investigated in Kashmir and Quebec for morphological traits, including five populations that were invasive to Kashmir; six putative populations that were invasive to Quebec, and five putative native populations in Quebec on the basis of morphometric traits [<a href="#B88-plants-09-01392">88</a>]. Twenty-five individual plants, which were separated by at least 10 m from one another to increase the likelihood that they were distinct genets [<a href="#B62-plants-09-01392">62</a>], were collected in each population. Morphological traits that were targeted in the present study included stem diameter, plant height, leaf length, leaf width, and internode length (<a href="#plants-09-01392-t007">Table 7</a>). The status of populations (invasive or native) was recorded in the field based on morphological differences between plants [<a href="#B88-plants-09-01392">88</a>].</p> <p>A total of 20 populations were sampled in this study for genetic study, including nine populations from Kashmir and eleven populations from Quebec (<a href="#plants-09-01392-t006">Table 6</a>). We designated only one population out of 20 populations as native, using cpDNA PCR-RFLP analysis. Collection sites in each region were separated by distances of 10‚Äì100 km. At each site, we collected one population, and, within each population, 25 individual plants were collected for molecular analysis. Fresh leaf samples (each about 1 cm<sup>2</sup> in area) were collected in tubes with desiccant and kept frozen at ‚àí20 ¬∞C prior to DNA extraction. Genomic DNA from plant species was extracted using the CTAB method [<a href="#B89-plants-09-01392">89</a>].</p> </div> <div class="4.3.genotyping" title="sec"> <div class="title" tagx="title" title="title">
4.3. Genotyping</div> <p>To verify whether <i>P. australis</i> individuals belonged to native or to introduced genotypes, a cpDNA PCR-RFLP analysis that had been developed by Saltonstall [<a href="#B41-plants-09-01392">41</a>] was used. Briefly, two non-coding regions of the chloroplast genome [<i>trnT</i> (UGU)-<i>trnL</i> (UAA) and <i>rbcL</i>-<i>psa</i>I] were amplified using the primers described by Saltonstall [<a href="#B41-plants-09-01392">41</a>]. PCR products were digested using the restriction enzyme <i>Rsa</i>I for the <i>trnL</i>b region and <i>Hha</i>I for the <i>rbcL</i> region following the manufacturer‚Äôs protocol (Promega, Madison, WI). Restriction fragments were electrophoresed in ethidium bromide-stained 3% TAE agarose gels and visualized using UV light. We also searched for matches with a Blast tool provided by NCBI. To confirm the results, PCR products were sequenced on an ABI 16-capillary genetic analyzer 3130XL (Applied Biosystems, Foster City, CA, USA). Sequences were edited and aligned using BioEdit version 7.0.5 [<a href="#B90-plants-09-01392">90</a>]. The BLASTn algorithm [<a href="#B91-plants-09-01392">91</a>] was used to query GenBank (NCBI) to determine the closest <i>P. austalis</i> haplotype. Phylogenetic analyses were conducted using MEGA 7 software [<a href="#B92-plants-09-01392">92</a>], and the Neighbour-Joining (NJ) method [<a href="#B93-plants-09-01392">93</a>].</p> <p>To assess the genetic diversity within and among populations of native and invasive <i>P. australis</i> across regions, we used a set of 9 cpDNA microsatellite loci, <i>viz</i>., PaGT-4, PaGT-8, PaGT-9, PaGT-11, PaGT-12, PaGT-13, PaGT-14, PaGT-16, and PaGT-22, which were developed by Saltonstall (<a href="#plants-09-01392-t008">Table 8</a>) [<a href="#B30-plants-09-01392">30</a>]. Amplifications were performed as two multiplexes and five single reactions, each in a total volume of 15 ¬µl containing 50 ng of template DNA, 0.5 mM each of the two forward and reverse primers, 0.2mM dNTPs (Applied Biosystems, Life Technologies, New York, NY, USA), 1.5 mM MgCl<sub>2</sub>,1x PCR buffer (10 mMTris, pH 8.0, 50 m MKCl, and 50 mM ammonium sulphate; Sigma Aldrich, St. Louis, MO, USA), 0.5 mM Fluorochrome (Applied Biosystems, Life Technologies), and 1 unit of Taq DNA polymerase (Sigma-Aldrich). Multiplexes PCR employed an annealing temperature of 55 ¬∞C and amplified loci PaGT 4 with PaGT 14, and loci PaGT 8 with PaGT 13. Single PCR employed an annealing temperature of 55 ¬∞C for loci PaGT 9 and PaGT 12, and 56 ¬∞C for loci PaGT 11, PaGT 16, and PaGT 22. Single PCRs were subjected to the same reaction conditions as the multiplexes (<a href="#plants-09-01392-t008">Table 8</a>).</p> <p>Two PCR protocols with different annealing temperatures were used during this study (<a href="#plants-09-01392-t008">Table 8</a>): (1) Amplification conditions were as follows: 94 ¬∞C for 3 min followed by 35 cycles of 94 ¬∞C for 30 s, 55 ¬∞C annealing for 30 s, 72 ¬∞C for 30 s, and a final extension of 72 ¬∞C for 5 min for loci PaGT 4 with PaGT 14, and loci PaGT 8 with PaGT 13, PaGT 9, and PaGT 12; (2) Amplification conditions were as follows: 94 ¬∞C for 4 min followed by 31 cycles of 94 ¬∞C for 45 s, 56 ¬∞C annealing for 2 min, 72 ¬∞C for 1 min, and a final extension of 72 ¬∞C for 2 min for loci PaGT 11, PaGT 16, and PaGT 22. Fragments were measured using an ABI PRISM 3130XL Analyzer (Applied Biosystems, Carlsbad, CA, USA) and scored using Gene mapper v4.0 software (Applied Biosystems).</p> </div> <div class="4.4.analysisofphenotypicdata" title="sec"> <div class="title" tagx="title" title="title">
4.4. Analysis of Phenotypic Data</div> <p>Variance components were estimated in the Statistical Analysis System [<a href="#B94-plants-09-01392">94</a>] with Proc Mixed using the restricted maximum likelihood analysis. Haplotype effects were fixed whereas region, population, and plant effects were assumed to be random, and data analyses were performed using the statistical model for a three-stage nested design [<a href="#B95-plants-09-01392">95</a>]: </p><div class="disp-formula" title="disp-formula">
Y<i><sub>ijkl</sub></i> = Œº + œÑ<i><sub>i</sub></i> + Œ≤<i><sub>j(k)</sub></i> + Œ¥<i><sub>(ij)k</sub></i> + e<i><sub>(ijk)l</sub></i><span class="label" tagx="label" title="label">(1)</span></div> where Y<i><sub>ijkl</sub></i> is the average value for the dependent variable for the <i>i</i>th region (Quebec or Kashmir), for the <i>j</i>th population nested in the <i>i</i>th region, for the <i>k</i>th haplotype (invasive or native) nested in the <i>j</i>th population and further in the <i>i</i>th region, and for the <i>l</i>th plant nested in the <i>k</i>th <i>P. australis</i> haplotype nested in the <i>j</i>th population further nested in the <i>i</i>th region; Œº is the overall mean; œÑ<i><sub>i</sub></i>, Œ≤<i><sub>j(k)</sub></i>, Œ¥<i><sub>(ij)k</sub></i> and e<i><sub>(ijk)l</sub></i> corresponded, respectively, to region effect (i.e., the between-region variance), to population effect (i.e., the among-population variance), to haplotype effect (i.e., the between-haplotype variance), and to plant effect (i.e., the within-population variance), the last term being the source of experimental error in the model. Post hoc multiple comparison of means was performed using Duncan Multiple Range Tests. <i>P</i><sub>ST</sub> values for each measured character were estimated using the following formula [<a href="#B22-plants-09-01392">22</a>,<a href="#B96-plants-09-01392">96</a>]: <div class="disp-formula" title="disp-formula"><i>P</i><sub>ST</sub> = <i>V</i><sub>AP</sub>/(<i>V</i><sub>AP</sub> + 2<i>V</i><sub>WP</sub>)<span class="label" tagx="label" title="label">(2)</span></div> where <i>V</i><sub>AP</sub> is the among-population variance and <i>V</i><sub>WP</sub> the within-population variance.<p /> <p>To determine whether putative phenotypic differences are due to adaptive processes or not, an analogous of narrow-sense heritability (<i>h</i><sup>2</sup>) estimate was computed as follows, as <i>P. australis</i> dispersal most often occurs through seeds than stolon and rhizomes [<a href="#B60-plants-09-01392">60</a>,<a href="#B62-plants-09-01392">62</a>]: </p><div class="disp-formula" title="disp-formula"><i>h</i><sup>2</sup> = among-plant variance/total variance<span class="label" tagx="label" title="label">(3)</span></div> <p /> <p>Similarity among different <i>P. australis</i> populations and population groups was calculated using Euclidean distance performed on standardized variables [<a href="#B97-plants-09-01392">97</a>], while hierarchical cluster analysis between populations was carried out on distance values using the R package pvclust [<a href="#B98-plants-09-01392">98</a>]. We characterized haplotypes and plotted the phenotypic distribution for each trait using the R ggpubr program [<a href="#B99-plants-09-01392">99</a>]. Principal components analysis (PCA) with phenotypic traits was applied to examine the pattern of phylogeographic groups, and how close they were to each other. PCA was performed using the Facto Mine R package [<a href="#B100-plants-09-01392">100</a>].</p> </div> <div class="4.5.populationgeneticsanalyses" title="sec"> <div class="title" tagx="title" title="title">
4.5. Population Genetics Analyses</div> <p>Given that <i>P. australis</i> plants sampled in this study were of different ploidy levels, descriptive statistics were performed using Polysat version 1.3 [<a href="#B101-plants-09-01392">101</a>]. These statistics included the percentage of polymorphic loci (<i>P</i>), the mean number of alleles (<i>A</i>), the mean number of alleles per polymorphic locus (<i>A</i><sub>p</sub>), allelic richness (<i>A</i><sub>R</sub>), and observed and expected heterozygosities (<i>H</i><sub>O</sub> and <i>H</i><sub>E</sub>) for each population [<a href="#B102-plants-09-01392">102</a>], together with unbiased estimates of population differentiation for microsatellites (<i>R</i><sub>ST</sub>; [<a href="#B103-plants-09-01392">103</a>]). Prior to analysis of molecular data, gametic disequilibrium verification between loci was performed using Fisher‚Äôs exact tests. Bruvo distances [<a href="#B104-plants-09-01392">104</a>] were estimated between genotypes using Polysat version 1.3 and principal coordinates analysis was used to display population structure, using R package [<a href="#B105-plants-09-01392">105</a>]. To measure the informative character of the SSR markers that were used in the present study, the polymorphism information contents (PIC) for each marker was calculated using as follows [<a href="#B106-plants-09-01392">106</a>]: </p><div class="disp-formula" title="disp-formula">
PIC = 1 ‚àí ∆©<sub>i=1</sub><sup>k</sup> P<sub>i</sub><sup>2</sup><span class="label" tagx="label" title="label">(4)</span></div> where k is the total number of alleles detected per locus and P<sub>i</sub> the frequency of the allele i in all populations. Genetic similarity (GS) [<a href="#B107-plants-09-01392">107</a>] was calculated as follows: <div class="disp-formula" title="disp-formula">
GS = 2N<sub>ij</sub>/(N<sub>i</sub> + N<sub>j</sub>)<span class="label" tagx="label" title="label">(5)</span></div> where N<sub>ij</sub> is the number of fragments common to individuals i and j, and (Ni + Nj) is the total number of fragments in both individuals.<p /> <p>To assess the phylogenetic relationships between individuals, the distance matrix was computed as genetic distance (GD) among <i>P. australis</i> populations through the software microsatellite analyser (MSA) version 4.05, using the formula of Nei and Li [<a href="#B108-plants-09-01392">108</a>], as follows: </p><div class="disp-formula" title="disp-formula">
GD<sub>xy</sub> = 1 ‚àí (2N<sub>xy</sub>/N<sub>x</sub> + N<sub>y</sub>).<span class="label" tagx="label" title="label">(6)</span></div> <p /> <p>Where N<sub>xy</sub> is the number of fragments common to individuals x and y, and (Nx + Ny) is the total number of fragments in both individuals.</p> <p>Then, the phylogenetic tree was constructed using genetic distance (neighbor joining algorithm) of the 20 populations of <i>P. australis</i> using Phylip and Mega 6 software. Relationships between genetic and geographic distance were investigated using Mantel tests in the Isolation By Distance Web Service (IBDWS 3.23; [<a href="#B109-plants-09-01392">109</a>]). Mantel Statistics was performed using 10,000 permutations to evaluate correlation between genetic and geographic distance.</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
5. Conclusions</div> <p>In conclusion, it appears that invasive populations of <i>P. australis</i> exhibit better growth-related traits, especially fresh and dry mass of flowers and seeds. PCoA based on Bruvo distances clearly distinguished invasive and native populations from the sample regions. Highest levels of genetic diversity were found in invasive populations of <i>P. australis</i> in Kashmir Himalayan wetlands. The observed among-population phenotypic differences in the surveyed <i>P. australis</i> are explained by divergent selection, local adaptation, and genetic drift. Understanding the phylogeographic basis of such a diversity gradient remains an open challenge. Cluster and structure analysis delineated all populations of native and invasive <i>P. australis</i> from Quebec and Kashmir into five main groups, shedding some light on their evolutionary relationships. The important level of the genetic diversity reported in the present study should be considered in managing the native and invasive <i>P. australis</i> populations in Quebec and Kashmir. The results provide a first step for further investigations based on sampling from a larger global range, including native European populations of <i>P. australis</i>.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The authors acknowledge the financial support of the Canadian Bureau for International Education (CBIE) in funding G.A.W. and a Natural Sciences and Engineering Research Council of Canada (NSERC) Discovery Grant to D.P.K. We thank Marie-Eve Beaulieu (Centre for Forest Research, Universit√© Laval) for assistance in laboratory work and reading of previous drafts of the present manuscript. The grant to MAS under the DBT supported Indo-Canadian project No. BT/IN/IC-IMPACTS/30/MAS/2015-2016 and the joint IC-Impacts/DBT-Laval #01 grant to DK and MAS under IC-Impacts on ‚Äò<i>Biomonitoring of water quality in relation to human health using biosensors and nanoparticle based purification system</i>‚Äô are well acknowledged. We also thank Claude Lavoie (Universit√© Laval) for providing the list of native population regions in Quebec.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher‚Äôs Note:</b> MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>This work is part of G.A.W.‚Äôs Ph.D. thesis research in the laboratories of M.A.S. and Z.A.R. at University of Kashmir, India and of D.P.K. at Universit√© Laval, Canada. G.A.W. collected leaf materials in Kashmir and Quebec, performed laboratory work, and contributed to drafting the manuscript. M.A.S., Z.A.R. and D.P.K. conceived the project and contributed to the writing of the present manuscript. A.R.A. contributed to the collection of leaf samples in Quebec, analyzed data, and contributed to the writing of the present manuscript. H.T. contributed to statistical analysis and writing of the manuscript. Together, the authors revised and corrected the manuscript versions. All authors have read and agreed to the published version of the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was financially supported by the DBT Indo-Canadian project No. BT/IN/IC-IMPACTS/30/MAS/2015-2016 to MAS and the joint IC-Impacts/DBT-Laval #01 to DPK.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare that they have no conflicts of interest.</p> <div class="title" tagx="title" title="title">
Data Accessibility</div> <p>The primer sequences (PaGT8-PaGT22) that were used in the present study have GenBank Accession Nos. AY230868-AY230876. We agree to deposit our data sequences to a public repository.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-plants-09-01392" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pimentel</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zuniga</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrison</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Update on the environmental and economic costs associated with alien-invasive species in the United States</span><span class="source" tagx="source" title="source">Ecol. Econ.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">52</span><span class="fpage" tagx="fpage" title="fpage">273</span><span class="lpage" tagx="lpage" title="lpage">288</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ecolecon.2004.10.002">10.1016/j.ecolecon.2004.10.002</a></span></span></li> <li tag="ref"><a name="B2-plants-09-01392" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyerson</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mooney</span><span class="given-names" tagx="given-names" title="given-names">H.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Invasive alien species in an era of globalization</span><span class="source" tagx="source" title="source">Front Ecol. Environ.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">199</span><span class="lpage" tagx="lpage" title="lpage">208</span><span class="pub-id"><a href="https://dx.doi.org/10.1890/1540-9295(2007)5[199:IASIAE]2.0.CO;2">10.1890/1540-9295(2007)5[199:IASIAE]2.0.CO;2</a></span></span></li> <li tag="ref"><a name="B3-plants-09-01392" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callaway</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maron</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">What have exotic plant invasions taught us over the past 20 years?</span><span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">369</span><span class="lpage" tagx="lpage" title="lpage">374</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tree.2006.04.008">10.1016/j.tree.2006.04.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16815436">16815436</a></span></span></li> <li tag="ref"><a name="B4-plants-09-01392" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rejmanek</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">What attributes make some plant species more invasive?</span><span class="source" tagx="source" title="source">Ecology</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">1655</span><span class="lpage" tagx="lpage" title="lpage">1661</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/2265768">10.2307/2265768</a></span></span></li> <li tag="ref"><a name="B5-plants-09-01392" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ellstrand</span><span class="given-names" tagx="given-names" title="given-names">N.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schierenbeck</span><span class="given-names" tagx="given-names" title="given-names">K.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hybridization as a stimulus for the evolution in plants?</span><span class="source" tagx="source" title="source">Proc. Nat. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">97</span><span class="fpage" tagx="fpage" title="fpage">7043</span><span class="lpage" tagx="lpage" title="lpage">7050</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.97.13.7043">10.1073/pnas.97.13.7043</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10860969">10860969</a></span></span></li> <li tag="ref"><a name="B6-plants-09-01392" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daehler</span><span class="given-names" tagx="given-names" title="given-names">C.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Performance comparisons of co-occurring native and invasive alien plants: Implications for conservation and restoration</span><span class="source" tagx="source" title="source">Annu. Rev. Ecol. Syst.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">183</span><span class="lpage" tagx="lpage" title="lpage">211</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.ecolsys.34.011802.132403">10.1146/annurev.ecolsys.34.011802.132403</a></span></span></li> <li tag="ref"><a name="B7-plants-09-01392" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bossdorf</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Auge</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lafuma</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogers</span><span class="given-names" tagx="given-names" title="given-names">W.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siemann</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prati</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phenotypic and genetic differentiation between native and introduced plant populations</span><span class="source" tagx="source" title="source">Oecologia</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">144</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00442-005-0070-z">10.1007/s00442-005-0070-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15891837">15891837</a></span></span></li> <li tag="ref"><a name="B8-plants-09-01392" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Funk</span><span class="given-names" tagx="given-names" title="given-names">W.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Testing evolutionary hypotheses for phenotypic divergence using landscape genetics</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">427</span><span class="lpage" tagx="lpage" title="lpage">430</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2009.04466.x">10.1111/j.1365-294X.2009.04466.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20331764">20331764</a></span></span></li> <li tag="ref"><a name="B9-plants-09-01392" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maron</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vila</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bommarco</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elmendorf</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beardsley</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid evolution of an invasive plant</span><span class="source" tagx="source" title="source">Ecol. Monogr.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">74</span><span class="fpage" tagx="fpage" title="fpage">261</span><span class="lpage" tagx="lpage" title="lpage">280</span><span class="pub-id"><a href="https://dx.doi.org/10.1890/03-4027">10.1890/03-4027</a></span></span></li> <li tag="ref"><a name="B10-plants-09-01392" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cano</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escarre</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fleck</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanco-Moreno</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sans</span><span class="given-names" tagx="given-names" title="given-names">F.X.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Increased fitness and plasticity of an invasive species in its introduced range: A study using <i>Senecio pterophorus</i></span><span class="source" tagx="source" title="source">J. Ecol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">96</span><span class="fpage" tagx="fpage" title="fpage">468</span><span class="lpage" tagx="lpage" title="lpage">476</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2745.2008.01363.x">10.1111/j.1365-2745.2008.01363.x</a></span></span></li> <li tag="ref"><a name="B11-plants-09-01392" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Kleunen</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weber</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A meta-analysis of trait differences between invasive and non-invasive plant species</span><span class="source" tagx="source" title="source">Ecol. Lett.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">235</span><span class="lpage" tagx="lpage" title="lpage">245</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1461-0248.2009.01418.x">10.1111/j.1461-0248.2009.01418.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20002494">20002494</a></span></span></li> <li tag="ref"><a name="B12-plants-09-01392" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">D‚ÄôAntonio</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">How much do phenotypic plasticity and local genetic variation contribute to phenotypic divergences along environmental gradients in widespread invasive plants? A meta-analysis</span><span class="source" tagx="source" title="source">Oikos</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">125</span><span class="fpage" tagx="fpage" title="fpage">907</span><span class="lpage" tagx="lpage" title="lpage">917</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/oik.02372">10.1111/oik.02372</a></span></span></li> <li tag="ref"><a name="B13-plants-09-01392" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forsman</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of genotypic and phenotypic variation on establishment are important for conservation, invasion and infection biology</span><span class="source" tagx="source" title="source">Proc. Nat. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">111</span><span class="fpage" tagx="fpage" title="fpage">302</span><span class="lpage" tagx="lpage" title="lpage">307</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1317745111">10.1073/pnas.1317745111</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24367109">24367109</a></span></span></li> <li tag="ref"><a name="B14-plants-09-01392" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leiblein-Wild</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tackenberg</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phenotypic variation of 38 European <i>Ambrosia artemisiifolia</i> populations measured in a common garden experiment</span><span class="source" tagx="source" title="source">Biol. Invasions</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">2003</span><span class="lpage" tagx="lpage" title="lpage">2015</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-014-0644-y">10.1007/s10530-014-0644-y</a></span></span></li> <li tag="ref"><a name="B15-plants-09-01392" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monty</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bizoux</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escarr√©</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahy</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid plant invasions in distinct climates involves different sources of phenotypic variation</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e55627</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0055627">10.1371/journal.pone.0055627</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23383251">23383251</a></span></span></li> <li tag="ref"><a name="B16-plants-09-01392" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amsellem</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noyer</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le Bourgeois</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossaert-McKey</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of genetic diversity of the invasive weed <i>Rubus alceifolius</i> Poir. (Rosaceae) in its native range and in areas of introduction, using amplified fragment length polymorphism (AFLP) markers</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">443</span><span class="lpage" tagx="lpage" title="lpage">455</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-294x.2000.00876.x">10.1046/j.1365-294x.2000.00876.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10736047">10736047</a></span></span></li> <li tag="ref"><a name="B17-plants-09-01392" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavergne</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Molofsky</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Increased genetic variation and evolutionary potential drive the success of an invasive grass</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">104</span><span class="fpage" tagx="fpage" title="fpage">3883</span><span class="lpage" tagx="lpage" title="lpage">3888</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0607324104">10.1073/pnas.0607324104</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17360447">17360447</a></span></span></li> <li tag="ref"><a name="B18-plants-09-01392" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsutsui</span><span class="given-names" tagx="given-names" title="given-names">N.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suarez</span><span class="given-names" tagx="given-names" title="given-names">A.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holway</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">T.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Reduced genetic variation and the success of an invasive species</span><span class="source" tagx="source" title="source">Proc. Nat. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">97</span><span class="fpage" tagx="fpage" title="fpage">5948</span><span class="lpage" tagx="lpage" title="lpage">5953</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.100110397">10.1073/pnas.100110397</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10811892">10811892</a></span></span></li> <li tag="ref"><a name="B19-plants-09-01392" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakai</span><span class="given-names" tagx="given-names" title="given-names">A.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allendorf</span><span class="given-names" tagx="given-names" title="given-names">F.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lodge</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Molofsky</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">With</span><span class="given-names" tagx="given-names" title="given-names">K.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baughman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cabin</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ellstrand</span><span class="given-names" tagx="given-names" title="given-names">N.C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The population biology of invasive species</span><span class="source" tagx="source" title="source">Annu. Rev. Ecol. Syst.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">305</span><span class="lpage" tagx="lpage" title="lpage">332</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.ecolsys.32.081501.114037">10.1146/annurev.ecolsys.32.081501.114037</a></span></span></li> <li tag="ref"><a name="B20-plants-09-01392" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">N.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hulme</span><span class="given-names" tagx="given-names" title="given-names">P.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoelzel</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Population genetics of an invasive species, <i>Heracleum mantegazzianum</i>: Implications for the role of life history, demographics and independent introductions</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">1747</span><span class="lpage" tagx="lpage" title="lpage">1756</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-294X.2003.01866.x">10.1046/j.1365-294X.2003.01866.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12803628">12803628</a></span></span></li> <li tag="ref"><a name="B21-plants-09-01392" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Genton</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shykoff</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giraud</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">High genetic diversity in French invasive populations of common ragweed, <i>Ambrosia artemisiifolia</i>, as a result of multiple sources of introduction</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">4275</span><span class="lpage" tagx="lpage" title="lpage">4285</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2005.02750.x">10.1111/j.1365-294X.2005.02750.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16313592">16313592</a></span></span></li> <li tag="ref"><a name="B22-plants-09-01392" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spitze</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Population structure in <i>Daphnia obtusa</i>‚Äìquantitative genetic and allozymic variation</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">135</span><span class="fpage" tagx="fpage" title="fpage">367</span><span class="lpage" tagx="lpage" title="lpage">374</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8244001">8244001</a></span></span></li> <li tag="ref"><a name="B23-plants-09-01392" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meril√§</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Quantitative trait and allozyme divergence in the greenfinch (<i>Carduelis chloris</i>, Aves: Fringillidae)</span><span class="source" tagx="source" title="source">Biol. J. Linn. Soc.</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">243</span><span class="lpage" tagx="lpage" title="lpage">266</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/bijl.1996.0120">10.1006/bijl.1996.0120</a></span></span></li> <li tag="ref"><a name="B24-plants-09-01392" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernatchez</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ecological theory of adaptive radiation: An empirical assessment from coregoninefishes (Salmoniformes)</span><span class="source" tagx="source" title="source">Evolution Illuminated: Salmon and Their Relatives</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hendry</span><span class="given-names" tagx="given-names" title="given-names">A.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stearns</span><span class="given-names" tagx="given-names" title="given-names">S.C.</span></span></span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford, UK</span><span class="year" tagx="year" title="year">2004</span><span class="fpage" tagx="fpage" title="fpage">157</span><span class="lpage" tagx="lpage" title="lpage">207</span></span></li> <li tag="ref"><a name="B25-plants-09-01392" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lande</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Neutral theory of quantitative genetic variance in an island model with local extinction and colonization</span><span class="source" tagx="source" title="source">Evolution</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">381</span><span class="lpage" tagx="lpage" title="lpage">389</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.1992.tb02046.x">10.1111/j.1558-5646.1992.tb02046.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28564025">28564025</a></span></span></li> <li tag="ref"><a name="B26-plants-09-01392" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stapley</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santure</span><span class="given-names" tagx="given-names" title="given-names">A.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dennis</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transposable elements as agents of rapid adaptation may explain the genetic paradox of invasive species</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">2241</span><span class="lpage" tagx="lpage" title="lpage">2252</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.13089">10.1111/mec.13089</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25611725">25611725</a></span></span></li> <li tag="ref"><a name="B27-plants-09-01392" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The interpretation of population structure by F-statistics with special regard to systems of mating</span><span class="source" tagx="source" title="source">Evolution</span><span class="year" tagx="year" title="year">1965</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">395</span><span class="lpage" tagx="lpage" title="lpage">420</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.1965.tb01731.x">10.1111/j.1558-5646.1965.tb01731.x</a></span></span></li> <li tag="ref"><a name="B28-plants-09-01392" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falconer</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mackay</span><span class="given-names" tagx="given-names" title="given-names">T.F.C.</span></span></span><span class="source" tagx="source" title="source">Introduction to Quantitative Genetics</span><span class="edition" tagx="edition" title="edition">4th ed.</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Longman</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Essex, UK</span><span class="year" tagx="year" title="year">1996</span></span></li> <li tag="ref"><a name="B29-plants-09-01392" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cryptic invasion by a non-native genotype of <i>Phragmites australis</i> into North America</span><span class="source" tagx="source" title="source">Proc. Nat. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">2445</span><span class="lpage" tagx="lpage" title="lpage">2449</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.032477999">10.1073/pnas.032477999</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11854535">11854535</a></span></span></li> <li tag="ref"><a name="B30-plants-09-01392" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Microsatellite variation within and among North American lineages of <i>Phragmites australis</i></span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">1689</span><span class="lpage" tagx="lpage" title="lpage">1702</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-294X.2003.01849.x">10.1046/j.1365-294X.2003.01849.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12803624">12803624</a></span></span></li> <li tag="ref"><a name="B31-plants-09-01392" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambertini</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gustafsson</span><span class="given-names" tagx="given-names" title="given-names">M.H.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frydenberg</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lissner</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Speranza</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brix</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A phylogeographic study of the cosmopolitan genus <i>Phragmites</i> (Poaceae) based on AFLPs</span><span class="source" tagx="source" title="source">Plant Syst. Evol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">258</span><span class="fpage" tagx="fpage" title="fpage">161</span><span class="lpage" tagx="lpage" title="lpage">182</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00606-006-0412-2">10.1007/s00606-006-0412-2</a></span></span></li> <li tag="ref"><a name="B32-plants-09-01392" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reshi</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of alien aquatic flora of Kashmir Himalaya: Implications for invasion management</span><span class="source" tagx="source" title="source">Trop. Ecol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">143</span><span class="lpage" tagx="lpage" title="lpage">157</span></span></li> <li tag="ref"><a name="B33-plants-09-01392" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khuroo</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rashid</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reshi</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dar</span><span class="given-names" tagx="given-names" title="given-names">G.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wafai</span><span class="given-names" tagx="given-names" title="given-names">B.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The alien flora of Kashmir Himalaya</span><span class="source" tagx="source" title="source">Biol. Invasions</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">269</span><span class="lpage" tagx="lpage" title="lpage">292</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-006-9032-6">10.1007/s10530-006-9032-6</a></span></span></li> <li tag="ref"><a name="B34-plants-09-01392" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">W.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambertini</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Py≈°ek</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyerson</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brix</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary mechanisms act differently in the native and introduced ranges of an invasive plant</span><span class="source" tagx="source" title="source">Ecol. Evol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">2440</span><span class="lpage" tagx="lpage" title="lpage">2452</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.3869">10.1002/ece3.3869</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29531666">29531666</a></span></span></li> <li tag="ref"><a name="B35-plants-09-01392" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blossey</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Phragmites</i>: Common reed. Morphological differences between native and introduced genotypes. Cornell University Ecology and Management of Invasive Plants Program, Department of Natural Resources</span><span class="year" tagx="year" title="year">2012</span><span class="comment">Available online: <a href="http://www.invasiveplants.net/phragmites/morphology.htm">http://www.invasiveplants.net/phragmites/morphology.htm</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 15 June 2013)</span></span></li> <li tag="ref"><a name="B36-plants-09-01392" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">P.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soreng</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Recognition of <i>Phragmites australis</i> subsp. <i>americanus</i> (Poaceae: Arundinoideae) in North America: Evidence from morphological and genetic analyses</span><span class="source" tagx="source" title="source">Sida Contrib. Bot.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">683</span><span class="lpage" tagx="lpage" title="lpage">692</span></span></li> <li tag="ref"><a name="B37-plants-09-01392" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambert</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casagrande</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Distribution of native and exotic <i>Phragmites australis</i> in Rhode Island</span><span class="source" tagx="source" title="source">Northeast. Nat.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">551</span><span class="lpage" tagx="lpage" title="lpage">560</span><span class="pub-id"><a href="https://dx.doi.org/10.1656/1092-6194(2006)13[551:DONAEP]2.0.CO;2">10.1656/1092-6194(2006)13[551:DONAEP]2.0.CO;2</a></span></span></li> <li tag="ref"><a name="B38-plants-09-01392" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lelong</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavoie</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jodoin</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belzile</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Expansion pathways of the exotic common reed (<i>Phragmites australis</i>): A historical and genetic analysis</span><span class="source" tagx="source" title="source">Divers. Distrib.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">430</span><span class="lpage" tagx="lpage" title="lpage">437</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1472-4642.2007.00351.x">10.1111/j.1472-4642.2007.00351.x</a></span></span></li> <li tag="ref"><a name="B39-plants-09-01392" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jodoin</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavoie</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villeneuve</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Theriault</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beaulieu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belzile</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Highways as corridors and habitats for the invasive common reed <i>Phragmites australis</i> in Quebec, Canada</span><span class="source" tagx="source" title="source">J. Appl. Ecol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">459</span><span class="lpage" tagx="lpage" title="lpage">466</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2664.2007.01362.x">10.1111/j.1365-2664.2007.01362.x</a></span></span></li> <li tag="ref"><a name="B40-plants-09-01392" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meadows</span><span class="given-names" tagx="given-names" title="given-names">R.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Distribution of native and introduced <i>Phragmites australis</i> in freshwater and oligohaline tidal marshes of the Delmarva peninsula and southern New Jersey</span><span class="source" tagx="source" title="source">J. Torrey Bot. Soc.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">134</span><span class="fpage" tagx="fpage" title="fpage">99</span><span class="lpage" tagx="lpage" title="lpage">107</span><span class="pub-id"><a href="https://dx.doi.org/10.3159/1095-5674(2007)134[99:DONAIP]2.0.CO;2">10.3159/1095-5674(2007)134[99:DONAIP]2.0.CO;2</a></span></span></li> <li tag="ref"><a name="B41-plants-09-01392" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A rapid method for identifying the origin of North American <i>Phragmites</i> populations using RFLP analysis</span><span class="source" tagx="source" title="source">Wetlands</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">1043</span><span class="lpage" tagx="lpage" title="lpage">1047</span><span class="pub-id"><a href="https://dx.doi.org/10.1672/0277-5212(2003)023[1043:ARMFIT]2.0.CO;2">10.1672/0277-5212(2003)023[1043:ARMFIT]2.0.CO;2</a></span></span></li> <li tag="ref"><a name="B42-plants-09-01392" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salstontall</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A set of primers for amplification of noncoding regions of chloroplast DNA in the grasses</span><span class="source" tagx="source" title="source">Mol. Ecol. Notes</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">76</span><span class="lpage" tagx="lpage" title="lpage">78</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1471-8278.2001.00031.x">10.1046/j.1471-8278.2001.00031.x</a></span></span></li> <li tag="ref"><a name="B43-plants-09-01392" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reshi</span><span class="given-names" tagx="given-names" title="given-names">Z.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavoie</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Predicting plant invasiveness from native range size: Clues from Kashmir Himalaya</span><span class="source" tagx="source" title="source">J. Plant Ecol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">167</span><span class="lpage" tagx="lpage" title="lpage">173</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jpe/rtr021">10.1093/jpe/rtr021</a></span></span></li> <li tag="ref"><a name="B44-plants-09-01392" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callaway</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Houseman</span><span class="given-names" tagx="given-names" title="given-names">G.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pal</span><span class="given-names" tagx="given-names" title="given-names">R.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosche</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reshi</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khasa</span><span class="given-names" tagx="given-names" title="given-names">D.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Conyza canadensis</i> suppresses plant diversity in its nonnative ranges but not at home: A transcontinental comparison</span><span class="source" tagx="source" title="source">New Phytol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">202</span><span class="fpage" tagx="fpage" title="fpage">1286</span><span class="lpage" tagx="lpage" title="lpage">1296</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/nph.12733">10.1111/nph.12733</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24548283">24548283</a></span></span></li> <li tag="ref"><a name="B45-plants-09-01392" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wani</span><span class="given-names" tagx="given-names" title="given-names">G.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reshi</span><span class="given-names" tagx="given-names" title="given-names">Z.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dar</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Polyploidy determines the stage of invasion: Clues from Kashmir Himalayan aquatic flora</span><span class="source" tagx="source" title="source">Acta Physiol. Plant.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">58</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11738-018-2629-4">10.1007/s11738-018-2629-4</a></span></span></li> <li tag="ref"><a name="B46-plants-09-01392" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colautti</span><span class="given-names" tagx="given-names" title="given-names">R.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacIsaac</span><span class="given-names" tagx="given-names" title="given-names">H.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A neutral terminology to define ‚Äòinvasive‚Äô species</span><span class="source" tagx="source" title="source">Divers. Distrib.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">135</span><span class="lpage" tagx="lpage" title="lpage">141</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1366-9516.2004.00061.x">10.1111/j.1366-9516.2004.00061.x</a></span></span></li> <li tag="ref"><a name="B47-plants-09-01392" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meadows</span><span class="given-names" tagx="given-names" title="given-names">R.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Above Ground Competition between Native and Introduced <i>Phragmites</i> in Two Tidal Marsh Basins in Delaware</span><span class="source" tagx="source" title="source">Master‚Äôs Thesis</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Delaware State University</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Dover, UK</span><span class="year" tagx="year" title="year">2006</span></span></li> <li tag="ref"><a name="B48-plants-09-01392" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mozdzer</span><span class="given-names" tagx="given-names" title="given-names">T.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zieman</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ecophysiological differences between genetic lineages facilitate the invasion of non-native <i>Phragmites australis</i> in North American Atlantic coast wetlands</span><span class="source" tagx="source" title="source">J. Ecol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">98</span><span class="fpage" tagx="fpage" title="fpage">451</span><span class="lpage" tagx="lpage" title="lpage">458</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2745.2009.01625.x">10.1111/j.1365-2745.2009.01625.x</a></span></span></li> <li tag="ref"><a name="B49-plants-09-01392" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mozdzer</span><span class="given-names" tagx="given-names" title="given-names">T.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Megonigal</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Jack-and-Master trait responses to elevated CO<sub>2</sub> and N: A Comparison of native and introduced <i>Phragmites australis</i></span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e42794</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0042794">10.1371/journal.pone.0042794</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23118844">23118844</a></span></span></li> <li tag="ref"><a name="B50-plants-09-01392" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeVries</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalski</span><span class="given-names" tagx="given-names" title="given-names">K.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bickford</span><span class="given-names" tagx="given-names" title="given-names">W.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Growth and Behavior of North American Microbes on <i>Phragmites australis</i> Leaves</span><span class="source" tagx="source" title="source">Microorganisms</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">690</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/microorganisms8050690">10.3390/microorganisms8050690</a></span></span></li> <li tag="ref"><a name="B51-plants-09-01392" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambertini</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brix</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eller</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Intraspecific variation in <i>Phragmites australis</i>: Clinal adaptation of functional traits and phenotypic plasticity vary with latitude of origin</span><span class="source" tagx="source" title="source">J. Ecol.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1365-2745.13401">10.1111/1365-2745.13401</a></span></span></li> <li tag="ref"><a name="B52-plants-09-01392" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bellavance</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brisson</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spatial dynamics and morphological plasticity of common reed (<i>Phragmites australis</i>) and cattails (<i>Typha</i> sp.) in freshwater marshes and roadside ditches</span><span class="source" tagx="source" title="source">Aquat. Bot.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">129</span><span class="lpage" tagx="lpage" title="lpage">134</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.aquabot.2010.04.003">10.1016/j.aquabot.2010.04.003</a></span></span></li> <li tag="ref"><a name="B53-plants-09-01392" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Py≈°ek</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sk√°lov√°</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ƒåuda</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">W.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dole≈æal</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kauz√°l</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambertini</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Py≈°kov√°</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brix</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyerson</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Physiology of a plant invasion: Biomass production, growth and tis-sue chemistry of invasive and native <i>Phragmites australis</i> populations</span><span class="source" tagx="source" title="source">Preslia</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">51</span><span class="lpage" tagx="lpage" title="lpage">75</span><span class="pub-id"><a href="https://dx.doi.org/10.23855/preslia.2019.051">10.23855/preslia.2019.051</a></span></span></li> <li tag="ref"><a name="B54-plants-09-01392" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyerson</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">David</span><span class="given-names" tagx="given-names" title="given-names">V.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rebecca</span><span class="given-names" tagx="given-names" title="given-names">N.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hybridization of invasive <i>Phragmites australis</i> with a native subspecies in North America</span><span class="source" tagx="source" title="source">Biol. Invasions</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">103</span><span class="lpage" tagx="lpage" title="lpage">111</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-009-9434-3">10.1007/s10530-009-9434-3</a></span></span></li> <li tag="ref"><a name="B55-plants-09-01392" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crawley</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harvey</span><span class="given-names" tagx="given-names" title="given-names">P.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purvis</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative ecology of the native and alien floras of the British Isles</span><span class="source" tagx="source" title="source">Philos. Trans. Biol. Sci.</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">351</span><span class="fpage" tagx="fpage" title="fpage">1251</span><span class="lpage" tagx="lpage" title="lpage">1259</span></span></li> <li tag="ref"><a name="B56-plants-09-01392" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Canavan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyerson</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Packer</span><span class="given-names" tagx="given-names" title="given-names">J.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Py≈°ek</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maurel</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lozano</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brundu</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Canavan</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cicatelli</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Tall-statured grasses: A useful functional group for invasion science</span><span class="source" tagx="source" title="source">Biol. Invasions</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">37</span><span class="lpage" tagx="lpage" title="lpage">58</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-018-1815-z">10.1007/s10530-018-1815-z</a></span></span></li> <li tag="ref"><a name="B57-plants-09-01392" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mozdzer</span><span class="given-names" tagx="given-names" title="given-names">T.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brisson</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hazelton</span><span class="given-names" tagx="given-names" title="given-names">E.L.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Physiological ecology and functional traits of North American native and Eurasian introduced <i>Phragmites australis</i> lineages</span><span class="source" tagx="source" title="source">AoB Plants</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">plt048</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/aobpla/plt048">10.1093/aobpla/plt048</a></span></span></li> <li tag="ref"><a name="B58-plants-09-01392" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muth</span><span class="given-names" tagx="given-names" title="given-names">N.Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pigliucci</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Traits of invasives reconsidered: Phenotypic comparisons of introduced invasive and introduced non-invasive plant species within two closely related clades</span><span class="source" tagx="source" title="source">Am. J. Bot.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">188</span><span class="lpage" tagx="lpage" title="lpage">196</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/ajb.93.2.188">10.3732/ajb.93.2.188</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21646179">21646179</a></span></span></li> <li tag="ref"><a name="B59-plants-09-01392" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jakobs</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weber</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Introduced plants of the invasive <i>Solidago gigantea</i> (Asteraceae) are larger and grow denser than conspecifics in the native range</span><span class="source" tagx="source" title="source">Divers. Distrib.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">19</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1472-4642.2004.00052.x">10.1111/j.1472-4642.2004.00052.x</a></span></span></li> <li tag="ref"><a name="B60-plants-09-01392" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belzile</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Labbe</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leblanc</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavoie</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Seeds contribute strongly to the spread of the invasive genotype of the common reed (<i>Phragmites australis</i>)</span><span class="source" tagx="source" title="source">Biol. Invasions</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">2243</span><span class="lpage" tagx="lpage" title="lpage">2250</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-009-9634-x">10.1007/s10530-009-9634-x</a></span></span></li> <li tag="ref"><a name="B61-plants-09-01392" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCormick</span><span class="given-names" tagx="given-names" title="given-names">M.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kettenring</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baron</span><span class="given-names" tagx="given-names" title="given-names">H.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whigham</span><span class="given-names" tagx="given-names" title="given-names">D.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spread of invasive <i>Phragmites australis</i> in estuaries with differing degrees of development: Genetic patterns, Allee effects and interpretation</span><span class="source" tagx="source" title="source">J. Ecol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">98</span><span class="fpage" tagx="fpage" title="fpage">1369</span><span class="lpage" tagx="lpage" title="lpage">1378</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2745.2010.01712.x">10.1111/j.1365-2745.2010.01712.x</a></span></span></li> <li tag="ref"><a name="B62-plants-09-01392" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kettenring</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mock</span><span class="given-names" tagx="given-names" title="given-names">K.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic diversity, reproductive mode, and dispersal differ between the cryptic invader, <i>Phragmites australis</i> and its native conspecific</span><span class="source" tagx="source" title="source">Biol. Invasions</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">2489</span><span class="lpage" tagx="lpage" title="lpage">2504</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-012-0246-5">10.1007/s10530-012-0246-5</a></span></span></li> <li tag="ref"><a name="B63-plants-09-01392" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brisson</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belzile</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turgeon</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavoie</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Strategies for a successful plant invasion: The reproduction of <i>Phragmites australis</i> in north-eastern North America</span><span class="source" tagx="source" title="source">J. Ecol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">1529</span><span class="lpage" tagx="lpage" title="lpage">1537</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1365-2745.12473">10.1111/1365-2745.12473</a></span></span></li> <li tag="ref"><a name="B64-plants-09-01392" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">Z.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scheepens</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">R.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Y.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Y.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biomass reallocation and increased plasticity might contribute to successful invasion of <i>Chromolaena odorata</i></span><span class="source" tagx="source" title="source">Flora</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">256</span><span class="fpage" tagx="fpage" title="fpage">79</span><span class="lpage" tagx="lpage" title="lpage">84</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.flora.2019.05.004">10.1016/j.flora.2019.05.004</a></span></span></li> <li tag="ref"><a name="B65-plants-09-01392" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Y.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.-F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sang</span><span class="given-names" tagx="given-names" title="given-names">W.-G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biomass allocation, morphology and photosynthesis of invasive and non-invasive exotic species grown at four irradiance levels</span><span class="source" tagx="source" title="source">Acta Oecol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">40</span><span class="lpage" tagx="lpage" title="lpage">47</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.actao.2006.03.009">10.1016/j.actao.2006.03.009</a></span></span></li> <li tag="ref"><a name="B66-plants-09-01392" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pysek</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Traits associated with invasiveness in allien plants</span><span class="source" tagx="source" title="source">J. Biogeogr.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">2040</span><span class="lpage" tagx="lpage" title="lpage">2050</span></span></li> <li tag="ref"><a name="B67-plants-09-01392" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlaepfer</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Billeter</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Why only tetraploid <i>Solidago gigantean</i> (Asteraceae) became invasive: A common garden comparison of ploidy levels</span><span class="source" tagx="source" title="source">Oecologia</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">163</span><span class="fpage" tagx="fpage" title="fpage">661</span><span class="lpage" tagx="lpage" title="lpage">673</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00442-010-1595-3">10.1007/s00442-010-1595-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20238128">20238128</a></span></span></li> <li tag="ref"><a name="B68-plants-09-01392" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baruch</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldstein</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Leaf construction cost, nutrient concentration, and net CO<sub>2</sub> assimilation of native and invasive species in Hawaii</span><span class="source" tagx="source" title="source">Oecologia</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">121</span><span class="fpage" tagx="fpage" title="fpage">183</span><span class="lpage" tagx="lpage" title="lpage">192</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s004420050920">10.1007/s004420050920</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28308558">28308558</a></span></span></li> <li tag="ref"><a name="B69-plants-09-01392" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shipley</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Net assimilation rate, specific leaf area and leaf mass ratio: Which is most closely correlated with relative growth rate: A meta-analysis</span><span class="source" tagx="source" title="source">Funct. Ecol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">565</span><span class="lpage" tagx="lpage" title="lpage">574</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2435.2006.01135.x">10.1111/j.1365-2435.2006.01135.x</a></span></span></li> <li tag="ref"><a name="B70-plants-09-01392" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rejmanek</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">S.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pitcairn</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grotkopp</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ecology of invasive plants: State of the art</span><span class="source" tagx="source" title="source">Invasive Alien Species: A New Synthesis</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mooney</span><span class="given-names" tagx="given-names" title="given-names">H.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mack</span><span class="given-names" tagx="given-names" title="given-names">R.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNeely</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neville</span><span class="given-names" tagx="given-names" title="given-names">L.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schei</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waage</span><span class="given-names" tagx="given-names" title="given-names">J.K.</span></span></span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Island Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington, DC, USA</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">104</span><span class="lpage" tagx="lpage" title="lpage">161</span></span></li> <li tag="ref"><a name="B71-plants-09-01392" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snowdon</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Haplotype hitchhiking promotes trait coselection in <i>Brassica napus</i></span><span class="source" tagx="source" title="source">Plant Biot. J.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">15781588</span></span></li> <li tag="ref"><a name="B72-plants-09-01392" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plut</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciotir</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Major</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freeland</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Origin of non-native <i>Phragmites australis</i> in North America, a common wetland invader</span><span class="source" tagx="source" title="source">Fundam. Appl. Limnol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">179</span><span class="fpage" tagx="fpage" title="fpage">121</span><span class="lpage" tagx="lpage" title="lpage">129</span><span class="pub-id"><a href="https://dx.doi.org/10.1127/1863-9135/2011/0179-0121">10.1127/1863-9135/2011/0179-0121</a></span></span></li> <li tag="ref"><a name="B73-plants-09-01392" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic variation among North American populations of <i>Phragmites australis</i>: Implications for management</span><span class="source" tagx="source" title="source">Estuaries</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">444</span><span class="lpage" tagx="lpage" title="lpage">451</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02823721">10.1007/BF02823721</a></span></span></li> <li tag="ref"><a name="B74-plants-09-01392" /><span class="label" tagx="label" title="label">74.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solis-Montero</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lou</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vallejo-Marin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Population structure and genetic diversity of native and invasive populations of <i>Solanum rostratum</i> (Solanaceae)</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e79807</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0079807">10.1371/journal.pone.0079807</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24224008">24224008</a></span></span></li> <li tag="ref"><a name="B75-plants-09-01392" /><span class="label" tagx="label" title="label">75.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kirk</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freeland</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic diversity and differentiation of fragmented reed beds (<i>Phragmites australis</i>) in the United Kingdom</span><span class="source" tagx="source" title="source">Hydrobiologia</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">665</span><span class="fpage" tagx="fpage" title="fpage">107</span><span class="lpage" tagx="lpage" title="lpage">115</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10750-011-0608-5">10.1007/s10750-011-0608-5</a></span></span></li> <li tag="ref"><a name="B76-plants-09-01392" /><span class="label" tagx="label" title="label">76.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clevering</span><span class="given-names" tagx="given-names" title="given-names">O.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lissner</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Taxonomy, chromosome numbers, clonal diversity and population dynamics of <i>Phragmites australis</i></span><span class="source" tagx="source" title="source">Aquat. Bot.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">185</span><span class="lpage" tagx="lpage" title="lpage">208</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0304-3770(99)00059-5">10.1016/S0304-3770(99)00059-5</a></span></span></li> <li tag="ref"><a name="B77-plants-09-01392" /><span class="label" tagx="label" title="label">77.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowry</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lester</span><span class="given-names" tagx="given-names" title="given-names">S.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The biogeography of plant reproduction: Potential determinants of species‚Äô range sizes</span><span class="source" tagx="source" title="source">J. Biogeogr.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">1975</span><span class="lpage" tagx="lpage" title="lpage">1982</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2699.2006.01562.x">10.1111/j.1365-2699.2006.01562.x</a></span></span></li> <li tag="ref"><a name="B78-plants-09-01392" /><span class="label" tagx="label" title="label">78.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Te Beest</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le Roux</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brysting</span><span class="given-names" tagx="given-names" title="given-names">A.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suda</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kubesova</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pysek</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The more the better? The role of polyploidy in facilitating plant invasions</span><span class="source" tagx="source" title="source">Ann. Bot.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">19</span><span class="lpage" tagx="lpage" title="lpage">45</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/aob/mcr277">10.1093/aob/mcr277</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22040744">22040744</a></span></span></li> <li tag="ref"><a name="B79-plants-09-01392" /><span class="label" tagx="label" title="label">79.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandit</span><span class="given-names" tagx="given-names" title="given-names">M.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pocock</span><span class="given-names" tagx="given-names" title="given-names">M.J.O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kunin</span><span class="given-names" tagx="given-names" title="given-names">W.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ploidy influences rarity and invasiveness in plants</span><span class="source" tagx="source" title="source">J. Ecol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">1108</span><span class="lpage" tagx="lpage" title="lpage">1115</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2745.2011.01838.x">10.1111/j.1365-2745.2011.01838.x</a></span></span></li> <li tag="ref"><a name="B80-plants-09-01392" /><span class="label" tagx="label" title="label">80.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandit</span><span class="given-names" tagx="given-names" title="given-names">M.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">H.T.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bisht</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Polyploidy in invasive plant species of Singapore</span><span class="source" tagx="source" title="source">Bot. J. Linn. Soc.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">151</span><span class="fpage" tagx="fpage" title="fpage">395</span><span class="lpage" tagx="lpage" title="lpage">403</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1095-8339.2006.00515.x">10.1111/j.1095-8339.2006.00515.x</a></span></span></li> <li tag="ref"><a name="B81-plants-09-01392" /><span class="label" tagx="label" title="label">81.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">W.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambertini</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.-Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyerson</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brix</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Invasion of Old World <i>Phragmites australis</i> in the New World: Precipitation and temperature patterns combined with human influences redesign the invasive niche</span><span class="source" tagx="source" title="source">Global Chang. Biol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">3406</span><span class="lpage" tagx="lpage" title="lpage">3422</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/gcb.12295">10.1111/gcb.12295</a></span></span></li> <li tag="ref"><a name="B82-plants-09-01392" /><span class="label" tagx="label" title="label">82.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKay</span><span class="given-names" tagx="given-names" title="given-names">J.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Latta</span><span class="given-names" tagx="given-names" title="given-names">R.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Adaptive population divergence: Markers, QTL and traits</span><span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">285</span><span class="lpage" tagx="lpage" title="lpage">291</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0169-5347(02)02478-3">10.1016/S0169-5347(02)02478-3</a></span></span></li> <li tag="ref"><a name="B83-plants-09-01392" /><span class="label" tagx="label" title="label">83.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolbe</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glor</span><span class="given-names" tagx="given-names" title="given-names">R.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schettino</span><span class="given-names" tagx="given-names" title="given-names">L.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lara</span><span class="given-names" tagx="given-names" title="given-names">A.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larson</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Losos</span><span class="given-names" tagx="given-names" title="given-names">J.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Multiple sources, admixture, and genetic variation in introduced Anolis lizard populations</span><span class="source" tagx="source" title="source">Conserv. Biol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">1612</span><span class="lpage" tagx="lpage" title="lpage">1625</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1523-1739.2007.00826.x">10.1111/j.1523-1739.2007.00826.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18173485">18173485</a></span></span></li> <li tag="ref"><a name="B84-plants-09-01392" /><span class="label" tagx="label" title="label">84.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Latta</span><span class="given-names" tagx="given-names" title="given-names">R.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gene flow, adaptive population divergence and comparative population structure across loci</span><span class="source" tagx="source" title="source">New Phytol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">161</span><span class="fpage" tagx="fpage" title="fpage">51</span><span class="lpage" tagx="lpage" title="lpage">58</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1469-8137.2003.00920.x">10.1046/j.1469-8137.2003.00920.x</a></span></span></li> <li tag="ref"><a name="B85-plants-09-01392" /><span class="label" tagx="label" title="label">85.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: (Dataset) USDA Agricultural Research Service</span></span><span class="mixed-article-title" title="mixed-article-title">Germplasm Resources Information Network (GRIN). Ag Data Commons</span><span class="year" tagx="year" title="year">2015</span><span class="comment">Available online: </span><span class="pub-id"><a href="https://dx.doi.org/10.15482/USDA.ADC/1212393">10.15482/USDA.ADC/1212393</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 23 February 2004)</span></span></li> <li tag="ref"><a name="B86-plants-09-01392" /><span class="label" tagx="label" title="label">86.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: Integrated Taxonomic Information System [Online]</span></span><span class="mixed-article-title" title="mixed-article-title">United States Department of Agriculture (Producer)</span><span class="year" tagx="year" title="year">2002</span><span class="comment">Available online: <a href="http://www.itis.usda.gov/">http://www.itis.usda.gov/</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 23 February 2004)</span></span></li> <li tag="ref"><a name="B87-plants-09-01392" /><span class="label" tagx="label" title="label">87.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavoie</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jean</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delisle</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letourneau</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exotic plant species of the St Lawrence River wetlands: A spatial and historical analysis</span><span class="source" tagx="source" title="source">J. Biogeogr.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">537</span><span class="lpage" tagx="lpage" title="lpage">549</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-2699.2003.00854.x">10.1046/j.1365-2699.2003.00854.x</a></span></span></li> <li tag="ref"><a name="B88-plants-09-01392" /><span class="label" tagx="label" title="label">88.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">D.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambertini</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jampeetong</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brix</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Clone-specifics differences in <i>Phragmites australis</i>: Effects of ploidy level and geographic origin</span><span class="source" tagx="source" title="source">Aquat. Bot.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">269</span><span class="lpage" tagx="lpage" title="lpage">279</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.aquabot.2006.11.005">10.1016/j.aquabot.2006.11.005</a></span></span></li> <li tag="ref"><a name="B89-plants-09-01392" /><span class="label" tagx="label" title="label">89.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doyle</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doyle</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A rapid DNA isolation procedure for small quantities of fresh leaf tissue</span><span class="source" tagx="source" title="source">Phytochem. Bull.</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">15</span></span></li> <li tag="ref"><a name="B90-plants-09-01392" /><span class="label" tagx="label" title="label">90.</span><span class="element-citation'"><span class="mixed-article-title" title="mixed-article-title">BioEdit</span><span class="comment">Available online: <a href="http://www.mbio.ncsu.edu/BioEdit/bioedit.html/">http://www.mbio.ncsu.edu/BioEdit/bioedit.html/</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 10 January 2018)</span></span></li> <li tag="ref"><a name="B91-plants-09-01392" /><span class="label" tagx="label" title="label">91.</span><span class="element-citation'"><span class="mixed-article-title" title="mixed-article-title">BLASTn</span><span class="comment">Available online: <a href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 11 January 2018)</span></span></li> <li tag="ref"><a name="B92-plants-09-01392" /><span class="label" tagx="label" title="label">92.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Filipski</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MEGA6: Molecular Evolutionary Genetics Analysis version 6.0</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">2725</span><span class="lpage" tagx="lpage" title="lpage">2729</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst197">10.1093/molbev/mst197</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122">24132122</a></span></span></li> <li tag="ref"><a name="B93-plants-09-01392" /><span class="label" tagx="label" title="label">93.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saitou</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The neighbor-joining method a new method for reconstructing phylogenetic trees</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">406</span><span class="lpage" tagx="lpage" title="lpage">425</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015">3447015</a></span></span></li> <li tag="ref"><a name="B94-plants-09-01392" /><span class="label" tagx="label" title="label">94.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: SAS, Institute Inc.</span></span><span class="source" tagx="source" title="source">SAS 9.3 Software</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">SAS Institute</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cary, NC, USA</span><span class="year" tagx="year" title="year">2012</span></span></li> <li tag="ref"><a name="B95-plants-09-01392" /><span class="label" tagx="label" title="label">95.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montgomery</span><span class="given-names" tagx="given-names" title="given-names">D.C.</span></span></span><span class="source" tagx="source" title="source">Design and Analysis of Experiments</span><span class="edition" tagx="edition" title="edition">10th ed.</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Wiley</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span><span class="year" tagx="year" title="year">2007</span><span class="size" title="size">size: 688ppages</span></span></li> <li tag="ref"><a name="B96-plants-09-01392" /><span class="label" tagx="label" title="label">96.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raeymaekers</span><span class="given-names" tagx="given-names" title="given-names">J.A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Houdt</span><span class="given-names" tagx="given-names" title="given-names">J.K.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larmuseau</span><span class="given-names" tagx="given-names" title="given-names">M.H.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geldof</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volckaert</span><span class="given-names" tagx="given-names" title="given-names">F.A.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Divergent selection as revealed by PST and QTL-based FST in three-spined stickleback (<i>Gasterosteus aculeatus</i>) populations along a coastal-inland gradient</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">891</span><span class="lpage" tagx="lpage" title="lpage">905</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2006.03190.x">10.1111/j.1365-294X.2006.03190.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17284219">17284219</a></span></span></li> <li tag="ref"><a name="B97-plants-09-01392" /><span class="label" tagx="label" title="label">97.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Legendre</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Legendre</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="source" tagx="source" title="source">Numerical Ecology</span><span class="edition" tagx="edition" title="edition">2nd ed.</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Amsterdam, The Netherlands</span><span class="year" tagx="year" title="year">1998</span></span></li> <li tag="ref"><a name="B98-plants-09-01392" /><span class="label" tagx="label" title="label">98.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimodaira</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pvclust: An R package for assessing the uncertainty in hierarchical clustering</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">1540</span><span class="lpage" tagx="lpage" title="lpage">1542</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btl117">10.1093/bioinformatics/btl117</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16595560">16595560</a></span></span></li> <li tag="ref"><a name="B99-plants-09-01392" /><span class="label" tagx="label" title="label">99.</span><span class="element-citation'"><span class="mixed-article-title" title="mixed-article-title">ggpubr: "ggplot2" Based Publication Ready Plots. R Package Version 0.1 7</span><span class="comment">Available online: <a href="https://rpkgs.datanovia.com/ggpubr/">https://rpkgs.datanovia.com/ggpubr/</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 27 June 2020)</span></span></li> <li tag="ref"><a name="B100-plants-09-01392" /><span class="label" tagx="label" title="label">100.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lƒô</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Josse</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Husson</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">FactoMine R: An R package for multivariate analysis</span><span class="source" tagx="source" title="source">J. Stat. Softw.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="https://dx.doi.org/10.18637/jss.v025.i01">10.18637/jss.v025.i01</a></span></span></li> <li tag="ref"><a name="B101-plants-09-01392" /><span class="label" tagx="label" title="label">101.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">L.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jasieniuk</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Polysat: An R package for polyploidy microsatellite analysis</span><span class="source" tagx="source" title="source">Mol. Ecol. Resour.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">562</span><span class="lpage" tagx="lpage" title="lpage">566</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1755-0998.2011.02985.x">10.1111/j.1755-0998.2011.02985.x</a></span></span></li> <li tag="ref"><a name="B102-plants-09-01392" /><span class="label" tagx="label" title="label">102.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weir</span><span class="given-names" tagx="given-names" title="given-names">B.S.</span></span></span><span class="source" tagx="source" title="source">Genetic Data Analysis II</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Sinauer Associates</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Sunderland, MA, USA</span><span class="year" tagx="year" title="year">1996</span><span class="fpage" tagx="fpage" title="fpage">445</span></span></li> <li tag="ref"><a name="B103-plants-09-01392" /><span class="label" tagx="label" title="label">103.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slatkin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A measure of population subdivision based on microsatellite allele frequencies</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">139</span><span class="fpage" tagx="fpage" title="fpage">457</span><span class="lpage" tagx="lpage" title="lpage">462</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7705646">7705646</a></span></span></li> <li tag="ref"><a name="B104-plants-09-01392" /><span class="label" tagx="label" title="label">104.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bruvo</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michels</span><span class="given-names" tagx="given-names" title="given-names">N.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Souza</span><span class="given-names" tagx="given-names" title="given-names">T.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulen-Burg</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A simple method for the calculation of microsatellite genotype distances irrespective of ploidy level</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">2101</span><span class="lpage" tagx="lpage" title="lpage">2106</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2004.02209.x">10.1111/j.1365-294X.2004.02209.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15189230">15189230</a></span></span></li> <li tag="ref"><a name="B105-plants-09-01392" /><span class="label" tagx="label" title="label">105.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: R Core Team</span></span><span class="source" tagx="source" title="source">A Language and Environment for Statistical Computing</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">R Foundation for Statistical Computing</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Vienna, Austria</span><span class="year" tagx="year" title="year">2016</span></span></li> <li tag="ref"><a name="B106-plants-09-01392" /><span class="label" tagx="label" title="label">106.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of gene diversity in subdivided populations</span><span class="source" tagx="source" title="source">Proc. Nat. Acad. Sci. USA</span><span class="year" tagx="year" title="year">1973</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">3321</span><span class="lpage" tagx="lpage" title="lpage">3323</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.70.12.3321">10.1073/pnas.70.12.3321</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4519626">4519626</a></span></span></li> <li tag="ref"><a name="B107-plants-09-01392" /><span class="label" tagx="label" title="label">107.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dice</span><span class="given-names" tagx="given-names" title="given-names">L.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Measures of the amount of ecologic association between species</span><span class="source" tagx="source" title="source">Ecology</span><span class="year" tagx="year" title="year">1945</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">297</span><span class="lpage" tagx="lpage" title="lpage">302</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/1932409">10.2307/1932409</a></span></span></li> <li tag="ref"><a name="B108-plants-09-01392" /><span class="label" tagx="label" title="label">108.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mathematical model for studying genetic variation in terms of endonucleases</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">1979</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">5269</span><span class="lpage" tagx="lpage" title="lpage">5273</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.76.10.5269">10.1073/pnas.76.10.5269</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/291943">291943</a></span></span></li> <li tag="ref"><a name="B109-plants-09-01392" /><span class="label" tagx="label" title="label">109.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bohonak</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelley</span><span class="given-names" tagx="given-names" title="given-names">S.T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolationbydistance, web service</span><span class="source" tagx="source" title="source">BMC Genet.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">13</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2156-6-13">10.1186/1471-2156-6-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15760479">15760479</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Biplot analysis of the studied 16 groups of <i>Phragmites australis</i> plants from three environments (Kashmir invasive, Quebec invasive, and Quebec native) based on seven morphological traits.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Distribution of phenotypes for the plant height (<b>upper left</b>), stem diameter (<b>upper right</b>), leaf length (<b>bottom left</b>), and leaf width (<b>bottom right</b>) are represented by histogram based on the average phenotype value of each <i>Phragmites</i> plants across different environments.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Principal component analysis plot showing different haplotypes for 378 <i>Phragmites</i> plants based of seven phenotypic traits.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Principal coordinate analysis with Bruvo distances, based on molecular data displaying population structure of Kashmir and Quebec invasives and Quebec native <i>Phragmites</i> between principal component analysis (PCA) 1 and PCA 2 (Blue‚ÄîQuebec invasive populations; Green‚ÄîQuebec native population; Red‚ÄîKashmir invasive populations).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Phylogenetic tree of <i>Phragmites australis</i> populations in native and invasive ranges based on SSR markers using the neighbor-joining method. Colors on branches classified subpopulations, mainly differentiated by geographical origin and ranges.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Map of the study area showing the distribution of sampling sites in different aquatic habitats of Kashmir Himalaya.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Map of the study area showing the distribution of sampling sites in different habitats of Quebec, Canada.</p>   </div> <div class="table-wrap_UNKNOWN" id="plants-09-01392-t001" orientation="portrait" position="float">
plants-09-01392-t001_Table 1<span class="label" tagx="label" title="label">Table 1</span> <p>The summary statistics (mean and standard error (¬± SE)) of measured seven phenotypic traits of invasive and native species of <i>Phragmites australis</i> in 16 populations from Kashmir, India and Quebec, Canada (QN1 = native population from Quebec; QE = invasive population in Quebec; KE = invasive population in Kashmir).</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">PopulAtion</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Plant Height (cm) ¬± SE</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Plant Stem Diameter (mm) ¬± SE</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Fresh Mass Flowers and Seeds (g) Per Plant ¬± SE</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Dry Mass Flowers and Seeds (g) Per Plant ¬± SE</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Internodal Length (cm) Per Plant ¬± SE</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Leaf Length (cm) ¬± SE</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Leaf Width (cm) ¬± SE</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QN1</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">191.64 ¬± 30.388 <sup>cde</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">5.06 ¬± 1.794 <sup>fg</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">0.71 ¬± 0.594 <sup>h</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">0.17 ¬± 0.102 <sup>f</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">8.86 ¬± 2.69 <sup>h</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">30.90 ¬± 3.453 <sup>fg</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.94 ¬± 0.220 <sup>d</sup></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QE12</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">135.20 ¬± 24.643 <sup>f</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">5.27 ¬± 1.315 <sup>efg</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">2.96 ¬± 2.44 <sup>fg</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.61 ¬± 1.072 <sup>e</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">12.52 ¬± 1.921 <sup>ef</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">28.99 ¬± 6.632 <sup>g</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.64 ¬± 0.377 <sup>fg</sup></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QE1</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">176.39 ¬± 35.788 <sup>e</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">6.22 ¬± 1.479 <sup>cd</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">6.07 ¬± 2.929 <sup>bcd</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">3.15 ¬± 1.443 <sup>bc</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">16.16 ¬± 2.407 <sup>b</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">33.48 ¬± 8.262 <sup>ef</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.80 ¬± 0.317 <sup>def</sup></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QE3</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">193.88 ¬± 22.643 <sup>cde</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">5.84 ¬± 0.828 <sup>def</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">6.29 ¬± 3.408 <sup>bcd</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">3.49 ¬± 1.952 <sup>bc</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">12.63 ¬± 2.383 <sup>ef</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">28.56 ¬± 4.827 <sup>g</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.32 ¬± 0.204 <sup>h</sup></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QE4</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">230.74 ¬± 55.540 <sup>a</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">6.73 ¬± 1.110 <sup>bc</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">6.84 ¬± 3.155 <sup>abc</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">3.64 ¬± 1.592 <sup>bc</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">14.40 ¬± 1.854 <sup>cd</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">46.80 ¬± 6.018 <sup>ab</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.70 ¬± 0.257 <sup>efg</sup></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QE6</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">212.44 ¬± 68.965 <sup>abc</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">6.36 ¬± 0.970 <sup>cd</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">3.84 ¬± 3.053 <sup>efg</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.71 ¬± 1.527 <sup>e</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">13.89 ¬± 2.577 <sup>cde</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">46.52 ¬± 6.170 <sup>ab</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.88 ¬± 0.212 <sup>de</sup></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QE7</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">202.36 ¬± 37.629 <sup>cd</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">6.29 ¬± 1.398 <sup>cd</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">5.20 ¬± 3.428 <sup>cde</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">2.99 ¬± 1.941 <sup>bc</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">16.77 ¬± 2.266 <sup>ab</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">42.50 ¬± 6.013 <sup>cd</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.76 ¬± 0.235 <sup>def</sup></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QE8</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">226.72 ¬± 34.632 <sup>ab</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">6.85 ¬± 1.408 <sup>bc</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">7.90 ¬± 4.116 <sup>ab</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">3.91 ¬± 2.029 <sup>ab</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">18.00 ¬± 3.749 <sup>a</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">41.57 ¬± 4.219 <sup>d</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.77 ¬± 0.218 <sup>def</sup></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QE9</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">173.32 ¬± 26.353 <sup>e</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">4.90 ¬± 1.034 <sup>g</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">3.91 ¬± 1.981 <sup>efg</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.88 ¬± 1.046 <sup>de</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">7.54 ¬± 1.233 <sup>h</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">34.13 ¬± 4.088 <sup>ef</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.59 ¬± 0.279 <sup>fg</sup></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QE10</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">176.46 ¬± 30.863 <sup>e</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">6.05 ¬± 1.343 <sup>cde</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">5.94 ¬± 4.227 <sup>bcd</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">2.76 ¬± 1.933 <sup>cd</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">10.89 ¬± 3.365 <sup>g</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">34.81 ¬± 5.410 <sup>e</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.50 ¬± 0.158 <sup>gh</sup></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>QE11</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">126.48 ¬± 23.901 <sup>f</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">4.99 ¬± 0.937 <sup>g</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">2.65 ¬± 2.049 <sup>g</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.40 ¬± 1.111 <sup>e</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">12.63 ¬± 2.383 <sup>ef</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">28.56 ¬± 4.827 <sup>g</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.32 ¬± 0.203 <sup>h</sup></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>KE1</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">206.56 ¬± 29.509 <sup>bcd</sup></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7.24 ¬± 1.556 <sup>ab</sup></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">5.89 ¬± 2.507 <sup>cd</sup></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">3.40 ¬± 1.558 <sup>bc</sup></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">13.88 ¬± 2.850 <sup>cde</sup></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">41.74 ¬± 4.601 <sup>d</sup></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">3.00 ¬± 0.706 <sup>a</sup></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Means that are followed by the same letter in columns are not significantly different using Duncan‚Äôs Multiple Range Test; native population (QN1) in bold.</p> </div> </div> </div> <div class="table-wrap_UNKNOWN" id="plants-09-01392-t002" orientation="portrait" position="float">
plants-09-01392-t002_Table 2<span class="label" tagx="label" title="label">Table 2</span> <p>Variance in <i>Phragmites australis</i> morphological characters from sixteen natural stands in Qu√©bec, Canada and Kashmir, India (SE = Standard Error).</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Source of Variation</th> <th colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Plant Height</th> <th colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Plant Stem Diameter</th> <th colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Fresh Mass of Flowers and Seeds</th> <th colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Dry Mass of Flowers and Seeds</th> <th colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Leaf Length</th> <th colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Internodal Length</th> <th colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Leaf Width</th> </tr> <tr> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Variance ¬± SE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><i>p</i>-Value</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Variance ¬± SE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><i>p</i>-Value</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Variance ¬± SE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><i>p</i>-Value</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Variance ¬± SE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><i>p</i>-Value</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Variance ¬± SE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><i>p</i>-Value</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Variance ¬± SE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><i>p</i>-Value</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Variance ¬± SE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><i>p</i>-Value</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Region</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.965 ¬± 1.458</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.2541</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.366 ¬± 1.092</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.37</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.37 ¬± 0.01</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.001</td> <td align="center" valign="middle" rowspan="1" colspan="1">31.5 ¬± 52.66</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.2766</td> <td align="center" valign="middle" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.69 ¬± 0.989</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.242</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Population (Region)</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">730.06 ¬± 304.25</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.0082</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.371 ¬± 0.174</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.0165</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.160 ¬± 1.059</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.0207</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.577 ¬± 0.29</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.001</td> <td align="center" valign="middle" rowspan="1" colspan="1">38.83 ¬± 5.754</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.007</td> <td align="center" valign="middle" rowspan="1" colspan="1">6.07 ¬± 2.406</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.0058</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.0298 ¬± 0.0228</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.095</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Haplotype (Population (Region))</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">20.76 ¬± 0.00</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.140 ¬± 0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.051 ¬± 0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.0142 ¬± 0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.0233 ¬± 0.002</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.001</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Plant (Haplotype (Population (Region)))</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">1227.30 ¬± 91.39</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.0001</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.555 ¬± 0.125</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.001</td> <td align="center" valign="middle" rowspan="1" colspan="1">7.846 ¬± 0.654</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.001</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.743 ¬± 0.195</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.0001</td> <td align="center" valign="middle" rowspan="1" colspan="1">27.55 ¬± 2.34</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.001</td> <td align="center" valign="middle" rowspan="1" colspan="1">6.08 ¬± 0.518</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.001</td> <td align="center" valign="middle" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Residual</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.0096</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">1.123 ¬± 0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.932 ¬± 0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.879 ¬± 0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">1.155 ¬± 0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.884 ¬± 0</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.1144 ¬± 0.0085</td> <td align="center" valign="middle" rowspan="1" colspan="1">&amp;lt;0.001</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>P</i><sub>ST</sub></b></td> <td align="center" valign="middle" rowspan="1" colspan="1">0.23</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.25</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.12</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.14</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.41</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.33</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">0.39</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b><i>h</i><sup>2</sup></b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.094</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.184</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.685</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.481</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.278</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.466</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="plants-09-01392-t003" orientation="portrait" position="float">
plants-09-01392-t003_Table 3<span class="label" tagx="label" title="label">Table 3</span> <p>Principal component analysis of seven <i>Phragmites australis</i> descriptors.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1" /> <th colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Eigen Vectors</th> </tr> <tr> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Component Number</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">PC1</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">PC2</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Standard Deviation</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.09</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.11</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Proportion of Variance (%)</td> <td align="center" valign="middle" rowspan="1" colspan="1">61.84</td> <td align="center" valign="middle" rowspan="1" colspan="1">17.49</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Cumulative Proportion (%</td> <td align="center" valign="middle" rowspan="1" colspan="1">61.84</td> <td align="center" valign="middle" rowspan="1" colspan="1">79.33</td> </tr> <tr> <td align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1" /> <td colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1"><b>Eigen Values</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Plant_height</td> <td align="center" valign="middle" rowspan="1" colspan="1">‚àí0.33</td> <td align="center" valign="middle" rowspan="1" colspan="1">-0.02</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Stem_diameter</td> <td align="center" valign="middle" rowspan="1" colspan="1">‚àí0.45</td> <td align="center" valign="middle" rowspan="1" colspan="1">-0.22</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Fresh_flowers</td> <td align="center" valign="middle" rowspan="1" colspan="1">‚àí0.40</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.37</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Dry_flowers</td> <td align="center" valign="middle" rowspan="1" colspan="1">‚àí0.41</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.31</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Internode_distance</td> <td align="center" valign="middle" rowspan="1" colspan="1">‚àí0.30</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.48</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Leaf_length</td> <td align="center" valign="middle" rowspan="1" colspan="1">‚àí0.41</td> <td align="center" valign="middle" rowspan="1" colspan="1">-0.31</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Leaf_width</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">‚àí0.31</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">-0.62</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="plants-09-01392-t004" orientation="portrait" position="float">
plants-09-01392-t004_Table 4<span class="label" tagx="label" title="label">Table 4</span> <p>Microsatellite loci and allelic diversity measures in invasive and native <i>Phragmites australis</i> populations from Quebec, Canada and Kashmir, India.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1" /> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Loci</th> <th colspan="3" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Allele Size Range (bp)</th> <th colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Number of Alleles</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1" /> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">PIC</th> </tr> <tr> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">KE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">QE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">QN1</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">KE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">QE</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">QN1</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 4</td> <td align="center" valign="middle" rowspan="1" colspan="1">291‚Äì295</td> <td align="center" valign="middle" rowspan="1" colspan="1">293‚Äì297</td> <td align="center" valign="middle" rowspan="1" colspan="1">285</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì3</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì3</td> <td align="center" valign="middle" rowspan="1" colspan="1">1</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.8</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 14</td> <td align="center" valign="middle" rowspan="1" colspan="1">191‚Äì209</td> <td align="center" valign="middle" rowspan="1" colspan="1">201‚Äì208</td> <td align="center" valign="middle" rowspan="1" colspan="1">199</td> <td align="center" valign="middle" rowspan="1" colspan="1">2‚Äì3</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì3</td> <td align="center" valign="middle" rowspan="1" colspan="1">1</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.9</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 8</td> <td align="center" valign="middle" rowspan="1" colspan="1">192‚Äì196</td> <td align="center" valign="middle" rowspan="1" colspan="1">194‚Äì196</td> <td align="center" valign="middle" rowspan="1" colspan="1">196</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì3</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì2</td> <td align="center" valign="middle" rowspan="1" colspan="1">1</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.7</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 13</td> <td align="center" valign="middle" rowspan="1" colspan="1">225‚Äì229</td> <td align="center" valign="middle" rowspan="1" colspan="1">227‚Äì229</td> <td align="center" valign="middle" rowspan="1" colspan="1">227‚Äì237</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì3</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì3</td> <td align="center" valign="middle" rowspan="1" colspan="1">1</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.5</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 9</td> <td align="center" valign="middle" rowspan="1" colspan="1">209‚Äì247</td> <td align="center" valign="middle" rowspan="1" colspan="1">209‚Äì229</td> <td align="center" valign="middle" rowspan="1" colspan="1">231</td> <td align="center" valign="middle" rowspan="1" colspan="1">2‚Äì4</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì2</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì2</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.9</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 11</td> <td align="center" valign="middle" rowspan="1" colspan="1">160‚Äì166</td> <td align="center" valign="middle" rowspan="1" colspan="1">160‚Äì166</td> <td align="center" valign="middle" rowspan="1" colspan="1">160‚Äì162</td> <td align="center" valign="middle" rowspan="1" colspan="1">2‚Äì6</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì3</td> <td align="center" valign="middle" rowspan="1" colspan="1">1</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.4</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 22</td> <td align="center" valign="middle" rowspan="1" colspan="1">201‚Äì222</td> <td align="center" valign="middle" rowspan="1" colspan="1">190‚Äì226</td> <td align="center" valign="middle" rowspan="1" colspan="1">201‚Äì215</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì4</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì3</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì2</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.9</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 12</td> <td align="center" valign="middle" rowspan="1" colspan="1">183‚Äì189</td> <td align="center" valign="middle" rowspan="1" colspan="1">183‚Äì190</td> <td align="center" valign="middle" rowspan="1" colspan="1">183‚Äì190</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì4</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì3</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì2</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.9</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 16</td> <td align="center" valign="middle" rowspan="1" colspan="1">248‚Äì325</td> <td align="center" valign="middle" rowspan="1" colspan="1">275‚Äì309</td> <td align="center" valign="middle" rowspan="1" colspan="1">282</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì4</td> <td align="center" valign="middle" rowspan="1" colspan="1">1‚Äì4</td> <td align="center" valign="middle" rowspan="1" colspan="1">1</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.9</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Mean</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.8</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>PIC: Polymorphic information content; KE, QE, QN are invasive haplotype from Kashmir, invasive haplotype from Quebec, and native haplotype from Quebec, respectively.</p> </div> </div> </div> <div class="table-wrap_UNKNOWN" id="plants-09-01392-t005" orientation="portrait" position="float">
plants-09-01392-t005_Table 5<span class="label" tagx="label" title="label">Table 5</span> <p>Population genetic diversity parameters estimated in 19 invasive <i>Phragmites australis</i> populations of Quebec (Canada) and Kashmir (India) and one native population of the species in Quebec, based on nine microsatellite markers.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1" /> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Popn</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><i>n</i></th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><i>P</i></th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><i>A</i></th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><i>A<sub>p</sub></i></th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><i>A<sub>R</sub></i></th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><i>H</i><sub>O</sub></th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><i>H</i><sub>E</sub></th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">KE1</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">100</td> <td align="center" valign="middle" rowspan="1" colspan="1">4.22</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.82</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.39</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.693</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.595</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">KE2</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">77.78</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.00</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.04</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.82</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.689</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.434</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">KE3</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">88.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.33</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.29</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.98</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.715</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.511</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">KE4</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">100</td> <td align="center" valign="middle" rowspan="1" colspan="1">3.33</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.63</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.16</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.722</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.543</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">KE5</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">100</td> <td align="center" valign="middle" rowspan="1" colspan="1">3.78</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.72</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.25</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.737</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.577</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">KE6</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">100</td> <td align="center" valign="middle" rowspan="1" colspan="1">4.56</td> <td align="center" valign="middle" rowspan="1" colspan="1">3.37</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.54</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.779</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.665</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">KE7</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">100</td> <td align="center" valign="middle" rowspan="1" colspan="1">3.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.43</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.15</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.641</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.543</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">KE8</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">100</td> <td align="center" valign="middle" rowspan="1" colspan="1">3.78</td> <td align="center" valign="middle" rowspan="1" colspan="1">3.10</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.4</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.731</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.598</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">KE9</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">66.67</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.11</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.17</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.81</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.569</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.406</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QE1</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">88.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">3.56</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.16</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.06</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.375</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.445</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QE3</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">88.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.44</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.79</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.81</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.463</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.380</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QE4</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">100</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.56</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.93</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.95</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.495</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.441</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QE6</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">88.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.44</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.00</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.05</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.451</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QE7</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">100</td> <td align="center" valign="middle" rowspan="1" colspan="1">3.00</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.92</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.94</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.463</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.408</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QE8</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">88.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.33</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.96</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.509</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.445</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QE9</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">88.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.44</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.80</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.77</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.393</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.354</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QE10</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">100</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.78</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.07</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.93</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.366</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.424</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QE11</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">100</td> <td align="center" valign="middle" rowspan="1" colspan="1">3.56</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.32</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.18</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.579</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.521</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QE12</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">88.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.61</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.59</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.321</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.291</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">QN1</td> <td align="center" valign="middle" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" rowspan="1" colspan="1">88.89</td> <td align="center" valign="middle" rowspan="1" colspan="1">2.22</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.72</td> <td align="center" valign="middle" rowspan="1" colspan="1">1.54</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.228</td> <td align="center" valign="middle" rowspan="1" colspan="1">0.204</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"> Overall Mean</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">92.78 </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">8.56</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4.16</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2.00</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.525</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.461</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Abbreviations: Popn = population name; <i>n</i> = sample size; <i>P</i> = percentage of polymorphic loci; <i>A</i> = mean number of alleles per locus; <i>A<sub>p</sub></i> = mean number of alleles per polymorphic locus; <i>A<sub>R</sub></i> = Allelic richness; <i>H</i><sub>O</sub> = mean observed heterozygosity within populations; <i>H</i><sub>E</sub> = mean expected heterozygosity within populations diver.</p> </div> </div> </div> <div class="table-wrap_UNKNOWN" id="plants-09-01392-t006" orientation="portrait" position="float">
plants-09-01392-t006_Table 6<span class="label" tagx="label" title="label">Table 6</span> <p>Geographic locations of <i>Phragmites australis</i> populations that were sampled in a study assessing and comparing the phenotypic and genetic diversity of the species in Quebec and Kashmir (India) with their putative population type as identified by morphometric characters (KE: alien invasive haplotype from Kashmir (India); QE: alien invasive haplotype from Quebec (Canada); QN1: native haplotype from Quebec; * populations that were sampled for phenotypic study; ‚Ä°: populations that were sampled for genotypic study; a.s.l = above sea level).</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Region</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Site of Collection</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Population</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Type</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Latitude (N)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Longitude</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Elevation (m a.s.l)</th> </tr> </thead> <tbody> <tr> <td rowspan="9" align="center" valign="middle" colspan="1"><b>Kashmir</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">Forshore</td> <td align="center" valign="middle" rowspan="1" colspan="1">KE1 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">34¬∞5‚Ä≤19‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">E 74¬∞30‚Ä≤44‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">1597</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Ashaibagh</td> <td align="center" valign="middle" rowspan="1" colspan="1">KE2 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">34¬∞4‚Ä≤11‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">E 74¬∞30‚Ä≤5‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">1579</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Nageen</td> <td align="center" valign="middle" rowspan="1" colspan="1">KE3 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">34¬∞3‚Ä≤51‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">E 74¬∞29‚Ä≤59‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">1595</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Rangharstop</td> <td align="center" valign="middle" rowspan="1" colspan="1">KE4 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">34¬∞3‚Ä≤46‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">E 74¬∞29‚Ä≤15‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">1583</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">KuhumusWullar</td> <td align="center" valign="middle" rowspan="1" colspan="1">KE5 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">34¬∞12‚Ä≤1‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">E 74¬∞21‚Ä≤35‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">1580</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Saderkotewullar</td> <td align="center" valign="middle" rowspan="1" colspan="1">KE6 <sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">34¬∞13‚Ä≤40‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">E 74¬∞27‚Ä≤43‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">1578</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Shalimar</td> <td align="center" valign="middle" rowspan="1" colspan="1">KE7 <sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">34¬∞50‚Ä≤5‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">E 74¬∞30‚Ä≤52‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">1583</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Dal Lake</td> <td align="center" valign="middle" rowspan="1" colspan="1">KE8 <sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">34¬∞37‚Ä≤19‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">E 74¬∞30‚Ä≤57‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">1585</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Ganderbal</td> <td align="center" valign="middle" rowspan="1" colspan="1">KE9 <sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">34¬∞8‚Ä≤9‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">E 74¬∞27‚Ä≤57‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">1572</td> </tr> <tr> <td rowspan="11" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">Quebec</td> <td align="center" valign="middle" rowspan="1" colspan="1">La Pocati√®re</td> <td align="center" valign="middle" rowspan="1" colspan="1">QE1 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">47¬∞20‚Ä≤27‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">W 70¬∞5‚Ä≤41‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">6.7 </td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">St Gr√©goire</td> <td align="center" valign="middle" rowspan="1" colspan="1">QE3 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">46¬∞14‚Ä≤27‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">W 72¬∞33‚Ä≤6‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">32.3</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Ange - Gardien</td> <td align="center" valign="middle" rowspan="1" colspan="1">QE4 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">45¬∞21‚Ä≤49‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">W 72¬∞55‚Ä≤50‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">97</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Eastman</td> <td align="center" valign="middle" rowspan="1" colspan="1">QE6 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">45¬∞17‚Ä≤53‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">W 72¬∞18‚Ä≤32‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">273</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Princeville</td> <td align="center" valign="middle" rowspan="1" colspan="1">QE7 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">46¬∞11‚Ä≤45‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">W 71¬∞54‚Ä≤40‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">123.7</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Berni√®re</td> <td align="center" valign="middle" rowspan="1" colspan="1">QE8 *<sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">46¬∞41‚Ä≤47‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">W 71¬∞18‚Ä≤13‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">76.8</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Lac St Francois</td> <td align="center" valign="middle" rowspan="1" colspan="1">QN1 <sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">Native</td> <td align="center" valign="middle" rowspan="1" colspan="1">45¬∞2‚Ä≤29‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">W 74¬∞27‚Ä≤47‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">47</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Lac St Louis</td> <td align="center" valign="middle" rowspan="1" colspan="1">QE9 <sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">45¬∞23‚Ä≤57‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">W 73¬∞45‚Ä≤43‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">28.6 </td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Parc Louis Racine</td> <td align="center" valign="middle" rowspan="1" colspan="1">QE10 <sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">45¬∞30‚Ä≤54‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">W73¬∞18‚Ä≤16‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">20.3 </td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Sabli√®re Colette</td> <td align="center" valign="middle" rowspan="1" colspan="1">QE11 <sup>‚Ä°</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" rowspan="1" colspan="1">45¬∞54‚Ä≤36‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">W73¬∞10‚Ä≤35‚Ä≥</td> <td align="center" valign="middle" rowspan="1" colspan="1">8</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">La Pocati√®re</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">QE12 <sup>‚Ä°</sup></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">invasive</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">47¬∞20‚Ä≤27‚Ä≥</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">W70¬∞5‚Ä≤46‚Ä≥</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">6.7</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="plants-09-01392-t007" orientation="portrait" position="float">
plants-09-01392-t007_Table 7<span class="label" tagx="label" title="label">Table 7</span> <p>Traits measured and methods that were used in a study aimed at investigating the phenotypic and genetic diversity of <i>P. australis</i> in Kashmir and Quebec.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Traits</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Method Used</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Stem diameter</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">Stem diameter was determined by measuring stem circumference at 25 cm above the ground surface using a metre tape graduated in 0.1 cm increments. </td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Plant height</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">Plant height was measured from soil surface to the apical node of the tallest shoot in each plant using measuring ruler tape.</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Leaf length and leaf width</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">Leaf length and leaf width were measured at 2 cm from the axil of leaf using plastic measuring scale. These measurements were taken six times per individual plant</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Internode length</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">Internode length was measured (¬±1 mm) from top to bottom of the plant using plastic measuring scale. These measurements were taken six times per individual plant</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Fresh mass of flowers and seeds</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">Flowers and seeds from the entire seed heads were cut off from each individual plant were sampled to measure fresh mass flowers and seeds using a portable electronic scale that was graduated to 0.001 g (Mitutoyo, Japan)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>Dry mass of flowers and seeds</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">The collected flowers and seeds were oven-dried at 70 ¬∞C for 3 to 4 days to determine the biomass of entire seed head from individual plant using a portable electronic scale that was graduated to 0.001 g </td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="plants-09-01392-t008" orientation="portrait" position="float">
plants-09-01392-t008_Table 8<span class="label" tagx="label" title="label">Table 8</span> <p>Microsatellite loci optimized for population genetics analysis.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Conditions</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Loci</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><sup>1</sup> Primer Sequences (5‚Ä≤‚ÄìR‚Ä≤)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Dyes</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">PCR<br /><br />Protocol Tested </th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"> T<i><sub>a</sub></i>‚ÄÉ<sub>(¬∞C)</sub></th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Multiplex 1</td> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 4</td> <td align="left" valign="middle" rowspan="1" colspan="1">‚ÄÇF: TGCTCCCTGCCAGTTTCTTG</td> <td align="center" valign="middle" rowspan="1" colspan="1">VIC</td> <td align="center" valign="middle" rowspan="1" colspan="1">McCormick et al. (2010) [<a href="#B61-plants-09-01392">61</a>]</td> <td align="center" valign="middle" rowspan="1" colspan="1">55</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="left" valign="middle" rowspan="1" colspan="1">‚ÄÇR: TATCCACCCTTCGAAGGCAC</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 14</td> <td align="left" valign="middle" rowspan="1" colspan="1">‚ÄÇF: GTTGCAGCAAGTATTTGG</td> <td align="center" valign="middle" rowspan="1" colspan="1">VIC</td> <td align="center" valign="middle" rowspan="1" colspan="1">Idem</td> <td align="center" valign="middle" rowspan="1" colspan="1">55</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="left" valign="middle" rowspan="1" colspan="1">‚ÄÇR: CAAGCATTCTAGTAGTAGC</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Multiplex 2</td> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 8</td> <td align="left" valign="middle" rowspan="1" colspan="1">‚ÄÇF: TCTGAACATAATCCTGGTGG</td> <td align="center" valign="middle" rowspan="1" colspan="1">FAM</td> <td align="center" valign="middle" rowspan="1" colspan="1">Idem</td> <td align="center" valign="middle" rowspan="1" colspan="1">55</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="left" valign="middle" rowspan="1" colspan="1">‚ÄÇR: TCTGTGTGAAGCAGTTCTGC</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 13</td> <td align="left" valign="middle" rowspan="1" colspan="1">‚ÄÇF: CTCATGCATCACTTCACAGG</td> <td align="center" valign="middle" rowspan="1" colspan="1">FAM</td> <td align="center" valign="middle" rowspan="1" colspan="1">Idem</td> <td align="center" valign="middle" rowspan="1" colspan="1">56</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="left" valign="middle" rowspan="1" colspan="1">‚ÄÇR: ACACGGACCTAACATCAACC</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Simplex 1</td> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 9</td> <td align="left" valign="middle" rowspan="1" colspan="1">‚ÄÇF: CCATGTGTTAATGTTGTCC</td> <td align="center" valign="middle" rowspan="1" colspan="1">PET</td> <td align="center" valign="middle" rowspan="1" colspan="1">Idem</td> <td align="center" valign="middle" rowspan="1" colspan="1">55</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">R: ATTGAATCCACACGTTTCCG</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Simplex 2</td> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 11</td> <td align="center" valign="middle" rowspan="1" colspan="1">F CAACTCCGTGAATGACATGC</td> <td align="center" valign="middle" rowspan="1" colspan="1">NED</td> <td align="center" valign="middle" rowspan="1" colspan="1">Belzile et al. (2010) [<a href="#B60-plants-09-01392">60</a>]</td> <td align="center" valign="middle" rowspan="1" colspan="1">56</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">R: CAGTTTGTGCACTAATGGAC</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Simplex 3</td> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 22</td> <td align="center" valign="middle" rowspan="1" colspan="1">F: TTGAGTGCCTGGTGTATTCG</td> <td align="center" valign="middle" rowspan="1" colspan="1">VIC</td> <td align="center" valign="middle" rowspan="1" colspan="1">Idem</td> <td align="center" valign="middle" rowspan="1" colspan="1">56</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">R AAGCTTCTGTCATGGAACCG</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Simplex 4</td> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 12</td> <td align="center" valign="middle" rowspan="1" colspan="1">F: CTTCCTAGGTCAGTATCATCC</td> <td align="center" valign="middle" rowspan="1" colspan="1">PET</td> <td align="center" valign="middle" rowspan="1" colspan="1">McCormick et al. (2010) [<a href="#B61-plants-09-01392">61</a>]</td> <td align="center" valign="middle" rowspan="1" colspan="1">55</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1">R: GTGGCAGCTGATTGATTTGG</td> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> <td align="center" valign="middle" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Simplex 5</td> <td align="center" valign="middle" rowspan="1" colspan="1">PaGT 16</td> <td align="center" valign="middle" rowspan="1" colspan="1">F: ACCAATCAGTCAGACTAGCC</td> <td align="center" valign="middle" rowspan="1" colspan="1">FAM</td> <td align="center" valign="middle" rowspan="1" colspan="1">Idem</td> <td align="center" valign="middle" rowspan="1" colspan="1">56</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">R: GTTCTCATGTTGGAGAAGCC</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>1</sup> Primer sequences were obtained from Saltonstall (2003a) [<a href="#B30-plants-09-01392">30</a>].</p> </div> </div> </div> </div>  </body></html>