<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Enzyme Inhib Med Chem</span><span class="iso-abbrev" title="iso-abbrev">J Enzyme Inhib Med Chem</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Enzyme Inhibition and Medicinal Chemistry</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1475-6366</span><span class="issn-epub" tagx="issn" title="issn-epub">1475-6374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7599001">7599001</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/14756366.2020.1841182">10.1080/14756366.2020.1841182</a></span><span class="publisher-id" title="publisher-id">1841182</span><div class="unknown" title="article-version" tag="article-version">
Version of Record</div> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Brief Report</span>: <span class="subject" title="subject">Short Communication</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
The SNAP-<i>tag</i> technology revised: an effective <i>chemo-enzymatic approach</i> by using a universal azide-based substrate</div> <div class="alt-title" title="alt-title">
R. Merlo et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merlo</span><span class="given-names" tagx="given-names" title="given-names">Rosa</span></span><a href="#AF0001">a</a><a href="#FN0001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caprioglio</span><span class="given-names" tagx="given-names" title="given-names">Diego</span></span><a href="#AF0002">b</a><a href="#FN0001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cillo</span><span class="given-names" tagx="given-names" title="given-names">Michele</span></span><a href="#AF0003">c</a><a href="#FN0001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valenti</span><span class="given-names" tagx="given-names" title="given-names">Anna</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mattossovich</span><span class="given-names" tagx="given-names" title="given-names">Rosanna</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrone</span><span class="given-names" tagx="given-names" title="given-names">Castrese</span></span><a href="#AF0002">b</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Massarotti</span><span class="given-names" tagx="given-names" title="given-names">Alberto</span></span><a href="#AF0002">b</a><a href="#AF0004">d</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossi</span><span class="given-names" tagx="given-names" title="given-names">Franca</span></span><a href="#AF0002">b</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miggiano</span><span class="given-names" tagx="given-names" title="given-names">Riccardo</span></span><a href="#AF0002">b</a><a href="#AF0004">d</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leonardi</span><span class="given-names" tagx="given-names" title="given-names">Antonio</span></span><a href="#AF0003">c</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minassi</span><span class="given-names" tagx="given-names" title="given-names">Alberto</span></span><a href="#AF0002">b</a><a href="#AN0002" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perugino</span><span class="given-names" tagx="given-names" title="given-names">Giuseppe</span></span><a href="#AF0001">a</a><a href="#AN0001" /></span><span class="citation_author_institution" id="AF0001">[a], <span class="institution" title="institution">Institute of Biosciences and BioResources, National Research Council of Italy</span><span class="city" tagx="city" title="city">Naples</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="AF0002">[b], <span class="institution" title="institution">Department of Pharmaceutical Sciences, University of Piemonte Orientale</span><span class="city" tagx="city" title="city">Novara</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="AF0003">[c], <span class="institution" title="institution">Department of Molecular Medicine and Medical Biotechnology, University of Naples â€œFederico IIâ€�</span><span class="city" tagx="city" title="city">Naples</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="AF0004">[d], <span class="institution" title="institution">IXTAL srl</span><span class="city" tagx="city" title="city">Novara</span><span class="country" tagx="country" title="country">Italy</span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">*</span><p>These Authors contributed equally and joined as First Author.</p> </div> <div class="fn-type-" title=""> <p>Supplemental data for this article can be accessed <a href="https://doi.org/10.1080/14756366.2020.1841182"><u>here</u></a>.</p> </div> <div class="corresp" title="corresp"><b>CONTACT</b> Giuseppe Perugino <span class="email" tagx="email" title="email">giuseppe.perugino@ibbr.cnr.it</span><span class="institution" title="institution">Institute of Biosciences and BioResources, National Research Council of Italy</span>, <span class="addr-line" title="addr-line">Via Pietro Castellino 111</span>, <span class="city" tagx="city" title="city">Naples</span><span class="postal-code" tagx="postal-code" title="postal-code">80131</span>, <span class="country" tagx="country" title="country">Italy</span></div> <div class="corresp" title="corresp">
Alberto Minassi <span class="email" tagx="email" title="email">alberto.minassi@uniupo.it</span><span class="institution" title="institution">Department of Pharmaceutical Sciences, University of Piemonte Orientale</span>, <span class="addr-line" title="addr-line">Largo Donegani 2/3</span>, <span class="city" tagx="city" title="city">Novara</span><span class="postal-code" tagx="postal-code" title="postal-code">28100</span>, <span class="country" tagx="country" title="country">Italy</span></div> </div><span class="pub-date-" title="pub-date-">: <span>2020-10-10</span></span><span class="pub-date-" title="pub-date-">: <span>2021</span></span><span class="volume" tagx="volume" title="volume">36</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">97</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">The Author(s)</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="IENZ_36_1841182.pdf" title="self-uri">IENZ_36_1841182.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>SNAP-<i>tag</i><sup>Â®</sup> is a powerful technology for the labelling of protein/enzymes by using benzyl-guanine (BG) derivatives as substrates. Although commercially available or ad hoc produced, their synthesis and purification are necessary, increasing time and costs. To address this limitation, here we suggest a revision of this methodology, by performing a <i>chemo-enzymatic approach</i>, by using a BG-substrate containing an azide group appropriately distanced by a spacer from the benzyl ring. The SNAP-<i>tag</i><sup>Â®</sup> and its relative thermostable version (<i>Ss</i>OGT-<i>H<sup>5</sup></i>) proved to be very active on this substrate. The stability of these <i>tags</i> upon enzymatic reaction makes possible the exposition to the solvent of the azide-moiety linked to the catalytic cysteine, compatible for the subsequent conjugation with DBCO-derivatives by azide-alkyne Huisgen cycloaddition. Our studies propose a strengthening and an improvement in terms of biotechnological applications for this self-labelling <i>protein-tag</i>.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd"><i>Protein-tag</i></div> <div class="kwd" title="kwd">
protein labelling</div> <div class="kwd" title="kwd">
enzymatic reaction</div> <div class="kwd" title="kwd">
click chemistry</div> <div class="kwd" title="kwd">
biotechnology</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1.</span><div class="title" tagx="title" title="title">
Introduction</div> <p>The advent of the self-labelling <i>protein-tags</i> (SLPs) has led to a huge push in modern biotechnology, especially in the field of cell biology, where auto-fluorescent proteins (AFPs) for a long time dominated for their versatility in the localisation experiments of proteins, organelles, and membranes<a href="#CIT0001"><sup>1</sup></a>. But the use of SLPs clearly goes beyond: they catalyse the covalent, highly specific and irreversible attachment of a part of their synthetic ligands upon reaction. This offers the opportunity to label them by conjugation to those ligands of an infinite number of chemical groups, such as fluorescent dyes, affinity molecules, or solid surfaces, expanding the application fields<a href="#CIT0002"><sup>2</sup></a>. Among SLPs, of particular note are the <i>Halotag</i><sup>Â®</sup>, the <i>SpyTag</i><a href="#CIT0003"><sup>3</sup></a> the SNAP- and the CLIP-<i>tag</i><sup>Â®</sup>. The Promega <i>Halotag</i><sup>Â®</sup> is a halo-alkane dehalogenase with a genetically modified active site, which reacts irreversibly with primary alkyl-halides<a href="#CIT0004"><sup>4</sup></a><sup>,</sup><a href="#CIT0005"><sup>5</sup></a>.</p> <p>SNAP-<i>tag</i><sup>Â®</sup> from New England Biolabs (NEB) is the engineered variant of the natural suicide human <i>O<sup>6</sup></i>-methylguanine DNA-methyltransferase protein (hMGMT). Alkylated DNA-alkyl-transferases (AGTs, MGMTs or OGTs, E.C. 2.1.1.63) are ubiquitous and conserved proteins involved in the repair of the DNA alkylation damage, in particular, they remove alkyl adducts at the level of <i>O<sup>6</sup></i>-position on guanine base<a href="#CIT0006"><sup>6</sup></a><sup>,</sup><a href="#CIT0007"><sup>7</sup></a>. The peculiar single-step mechanism are called â€œ<i>suicide enzymes</i>,â€� in which the alkylated base is directly repaired by the irreversible transfer of the alkylic group from the damaged guanine to the catalytic cysteine in the protein active site<a href="#CIT0008"><sup>8</sup></a>. The protein is permanently inactivated upon the trans-alkylation reaction and susceptible to <i>inÂ vivo</i> degradation via the proteasome.</p> <p>In 2003, the group of Kai Johnsson developed a new strategy to exploit the hMGMT suicidal reaction in biotechnology, adopting a directed-evolution approach to engineer a variant to be used as an innovative <i>protein-tag</i>, that is, the SNAP-<i>tag</i><sup>Â®</sup>. The rationale behind the SNAP-<i>tag</i> technology is the low substrate specificity of some AGT proteins, being able to efficiently recognise also the <i>O<sup>6</sup></i>-benzyl-guanine (BG) nucleobase<a href="#CIT0009"><sup>9</sup></a>. Likely, the reaction of these enzymes with BG-derivatives could happen: upon the irreversible transfer to the catalytic cysteine, they indeed demonstrated the specific labelling of the hMGMT with molecules, as fluorophores, previously conjugated to the 4-position of the BG benzyl ring. Because of the small dimension of this protein, it was mutagenized to abolish any DNA binding activity and utilised as <i>protein-tag</i> for the indirect labelling of proteins of interest genetically fused to it (<a href="#F0001">Figure 1</a>)<a href="#CIT0009"><sup>9â€"13</sup></a>. Later, the same group further engineered the SNAP-<i>tag</i><sup>Â®</sup> to obtain the CLIP-<i>tag</i><sup>Â®</sup>, which specifically reacts with <i>O<sup>2</sup></i>-benzyl-cytosine derivatives, expanding that technology for <i>inÂ vivo</i> and <i>inÂ vitro</i> multi-protein labelling<a href="#CIT0014"><sup>14</sup></a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Single-step reaction vs chemo-enzymatic approach. (A) The SNAP-<i>tag</i><sup>Â®</sup> technology is based on BG-derivatives singularly synthetised and purified, and not excluding that the conjugated chemical group (<i>green sphere</i>) could affect the enzymatic reaction rate. (B) The SNAP-<i>tag</i><sup>Â®</sup> technology revised uses a unique and universal azide BG-derivative, converting SNAP-<i>tag</i><sup>Â®</sup> in a clickable form, prone to perform a fast and efficient cycloaddition with DBCO-based chemical groups. POI, protein of interest genetically fused to the SNAP-<i>tag</i><sup>Â®</sup>.</p>   </div> <p>Apart from cell biology and fluorescence imaging, hundreds of papers are present in the literature showing many applications of SNAP-<i>tag</i><sup>Â®</sup> in several fields, among which RNA-editing<a href="#CIT0015"><sup>15</sup></a>, the development of SNAP-based sensors for small molecules<a href="#CIT0016"><sup>16â€"18</sup></a> and ions<a href="#CIT0019"><sup>19</sup></a><sup>,</sup><a href="#CIT0020"><sup>20</sup></a>, and protein-DNA complexes in â€œDNA Origamiâ€� structures<a href="#CIT0021"><sup>21</sup></a>.</p> <p>Following the same approach, Perugino and co-workers expanded this technology to extremophilic organisms and to all the applications which require harsh reaction conditions, not fully suitable for the employing of the mesophilic SNAP-<i>tag</i><sup>Â®</sup>. To this aim, they developed a â€œ<i>thermo</i>-SNAP-<i>tag</i>â€� by the production of a variant of the OGT from <i>Saccharolobus solfataricus</i> (previously <i>Sulfolobus solfataricus</i>, <i>Ss</i>OGT-<i>H<sup>5</sup></i>, hereinafter <i>H<sup>5</sup></i>), an enzyme which revealed extremely resistant to high temperature, high ionic strength, proteases attack, and, in general, to common physical and chemical denaturants<a href="#CIT0022"><sup>22</sup></a><sup>,</sup><a href="#CIT0023"><sup>23</sup></a>. The intrinsic stability of <i>H<sup>5</sup></i> made it compatible with expression and utilisation <i>inÂ vivo</i> as <i>protein-tag</i> in thermophilic organisms, as <i>Thermus thermophilus</i><a href="#CIT0024"><sup>24</sup></a> and <i>Sulfolobus islandicus</i><a href="#CIT0025"><sup>25</sup></a> as well as in an <i>inÂ vitro</i> expression system using <i>Sulfolobus</i> lysates<a href="#CIT0026"><sup>26</sup></a>. Recently, <i>H<sup>5</sup></i> became a part of the new ASL<i><sup>tag</sup></i> system<a href="#CIT0027"><sup>27</sup></a>, which was particularly useful for the <i>inÂ vivo</i> immobilisation and contemporary labelling of proteins and enzymes of interest, stabilising them without any purification procedures needed<a href="#CIT0028"><sup>28</sup></a>.</p> <p>SNAP-<i>tag</i><sup>Â®</sup> technology is essentially based on BG-substrates: although many of them are commercially available, the possibility of conjugation of infinite desired molecules to the 4-position on BG leads to the synthesis of <i>ad hoc</i> substrates. This is generally possible through the crosslinking reaction of the so-called â€œBG-building blockâ€� (such as the amine-reactive BG-NH<sub>2</sub>) with NHS-ester derivative compounds. The main disadvantage is the need to purify the final compounds before the reaction with the enzyme, increasing the times and costs of the experiments (<a href="#F0001">Figure 1(A)</a>). Furthermore, the presence of chemical groups conjugated to the benzyl moiety of the BG could affect the reaction efficiency of the SNAP-<i>tag</i><sup>Â®</sup><a href="#CIT0029"><sup>29â€"33</sup></a>, sometimes making this enzyme not fully applicable to particular requests.</p> <p>In this work, we analysed and confirmed the catalytic dependence of SNAP-<i>tag</i><sup>Â®</sup> and <i>H<sup>5</sup></i> by several substrates having different chemical groups conjugated to the <i>O<sup>6</sup></i>-position of the guanine. To overcome these limitations, in the current study we suggest a further improvement of this technology with the application of a <i>chemo-enzymatic approach</i>, by using a unique and universal azide decorated BG-derivative, to obtain the specific labelling of the <i>tag</i> (<i>clickable</i>-SNAP), that can be easily coupled with a potentially infinite number of commercially available di-benzo-cyclo-octyl (DBCO)-based molecules, through the copper-free azide-alkyne Huisgen cycloaddition (<a href="#F0001">Figure 1(B)</a>). This approach could mainly offer the advantage to take into account of a unique reaction rate for the enzyme (with the azide-based BG), saving costs and times for the linking to the <i>tag</i> of an infinite number of commercially available DBCO-molecules. Here, we successfully proved the labelling of the SNAP-<i>tag</i><sup>Â®</sup> with several DBCO-based fluorophores and the covalent immobilisation of this protein on alkyne-coated surface sensors.</p> </div> <div class="materialsandmethods" title="sec"><span class="label" tagx="label" title="label">2.</span><div class="title" tagx="title" title="title">
Materials and methods</div> <div class="reagents" title="sec"><span class="label" tagx="label" title="label">2.1.</span><div class="title" tagx="title" title="title">
Reagents</div> <p><b>BG</b> was from Activate Scientific GmbH (UK), whereas <b>MGPA</b> was a gift of Prof D. Prosperi (University of Bicocca, Milan, Italy). SNAP-Vista<sup>Â®</sup> Green (<b>SVG</b>), SNAP Cell<sup>Â®</sup> Block (<b>SCB</b>), SNAP Cell<sup>Â®</sup> 430 (<b>SC430</b>), BG-PEG-NH<sub>2</sub> (<b>BGPA</b>), pSNAP-<i>tag</i>(m) plasmid, DNA restriction endonucleases and DNA modification enzymes were purchased from New England Biolabs (USA). Molecular biology kits for plasmid preparations were from Macherey-Nagel GmbH (Germany). Oligonucleotides synthesis and DNA sequencing service were performed by Eurofins Genomics (Germany). <b>BDP FL alkyne, BDP FL DBCO</b>, <b>Cy5 DBCO</b> were purchased from Lumiprobe GmbH (Germany). <b>DBCO-PEG<sub>4</sub>-Fluor 545</b>, Tris(2-carboxyethyl)phosphin (TCEP), Tris [(1-benzyl-1H-1,2,3-triazol-4-yl)-methyl]amine (TBTA) were from Sigma-Aldrich (St. Louis, MO). Pierceâ"¢ Premium Grade Sulfo-NHS (N-hydroxy-sulfo-succinimide) and Pierceâ"¢ Premium Grade 1-ethyl-3-(3-dimethyl-amino-propyl)-carbodiimide hydrochloride (EDC) were from Thermo Fisher Scientific (Carlsbad, CA).</p> </div> <div class="compoundssynthesis:generalprocedures" title="sec"><span class="label" tagx="label" title="label">2.2.</span><div class="title" tagx="title" title="title">
Compounds synthesis: general procedures</div> <p><sup>1</sup>H (400â€‰MHz) and <sup>13</sup>C (100â€‰MHz) NMR spectra were measured on Bruker Advance Neo 400â€‰MHz spectrometer. Chemical shifts were referenced to the residual solvent signal (CDCl<sub>3</sub>: <i>Î´</i><sub>H</sub> = 7.26, <i>Î´</i><sub>C</sub> = 77.0; DMSO: <i>Î´</i><sub>H</sub> = 2.50 <i>Î´</i><sub>C</sub> = 39.5). Low-resolution ESI-MS were obtained on an LTQ OrbitrapXL (Thermo Scientific) mass spectrometer. IR spectra were registered on Shimadzu DR 8001 spectrophotometer. Silica gel 60 (70â€"230 mesh) used for gravity column chromatography (CC) was purchased from Machereyâ€"Nagel. Reactions were monitored by TLC on Merck 60 F254 (0.25â€‰mm) plates, visualised by staining with 5% H<sub>2</sub>SO<sub>4</sub> in ethanol or KMnO<sub>4</sub> and heating. Organic phases were dried with Na<sub>2</sub>SO<sub>4</sub> before evaporation. Chemical reagents and solvents were from Aldrich, Alfa Aesar, and TCI and were used without any further purification unless stated otherwise.</p> </div> <div class="synthesisofbgn3" title="sec"><span class="label" tagx="label" title="label">2.3.</span><div class="title" tagx="title" title="title">
Synthesis of BGN3</div> <p><b>BGN3</b> was synthesised according to the method of Zhang etÂ al.<a href="#CIT0034"><sup>34</sup></a>, whose experimental spectra were comparable. White solid. <sup>1</sup>H NMR (400â€‰MHz, DMSO-d<sub>6</sub>) <i>Î´</i> 7.84 (s, 1H), 7.53 (d, <i>J</i>â€‰=â€‰7.8â€‰Hz, 2H), 7.39 (d, <i>J</i>â€‰=â€‰7.8â€‰Hz, 2H), 6.28 (bs, 2H), 5.49 (s, 2H), 4.45 (s, 2H) (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S1(A)</a>). <sup>13</sup>C NMR (100â€‰MHz, DMSO-d<sub>6</sub>) <i>Î´</i> 159.69, 136.80, 135.43, 128.82, 128.58, 66.44, 53.37. IR (KBr) cm<sup>âˆ’1</sup>: 3638, 3462, 3322, 2799, 2132, 1424, 1257, 1163, 912, 790, 656, 514. ESI/MS: <i>m/z</i> [Mâ€‰+â€‰H<sup>+</sup>] 297 (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S1(B)</a>).</p> </div> <div class="synthesisofbgsn3" title="sec"><span class="label" tagx="label" title="label">2.4.</span><div class="title" tagx="title" title="title">
Synthesis of BGSN3</div> <p><b>BGSN3</b> was synthesised by following the scheme in <a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S2</a>.</p> <div class="synthesisof4-azido-n-(4-(hydroxymethyl)benzyl)butanamide(compound3)" title="sec"><span class="label" tagx="label" title="label">2.4.1</span><div class="title" tagx="title" title="title">
Synthesis of 4-azido-N-(4-(hydroxymethyl) benzyl) butanamide (compound 3)</div> <p>A stirred solution of compound <b>1</b> (see <a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S2</a>; 1.176â€‰g, 9.115â€‰mmol, 1 eq/mol) was prepared according to the method by Huang etÂ al.<a href="#CIT0035"><sup>35</sup></a> in DCM (30â€‰ml), compound <b>2</b> (1.500â€‰g, 10.939â€‰mmol, 1.2 eq/mol; prepared according to the method by Leng etÂ al.<a href="#CIT0036"><sup>36</sup></a> and TEA (5.08â€‰ml, 36.460â€‰mmol, 4 eq/mol) were added. The mixture was stirred for 10â€‰min at room temperature, then T3P (50% solution in EtOAc, 10.85â€‰ml, 18.230â€‰mmol, 2 eq/mol) was slowly added dropwise, and the stirred reaction was left overnight at room temperature until the complete conversion of the starting material (TLC: PE-EtOAc 4:6; Rf 1â€‰=â€‰0.47; Rf 3â€‰=â€‰0.16). The reaction was quenched by the addition of BRINE and extraction with DCM. After drying (Na<sub>2</sub>SO<sub>4</sub>) and evaporation, the residue was purified by gravity column chromatography on silica gel (gradient PE-EtOAc from 6:4 to 3:7) to afford compound <b>3</b> as a white solid (660â€‰mg, 30%). <sup>1</sup>H NMR (400â€‰MHz, CDCl<sub>3</sub>) <i>Î´</i> 7.28 (d, <i>J</i>â€‰=â€‰8.0â€‰Hz, 2H), 7.20 (d, <i>J</i>â€‰=â€‰7.9â€‰Hz, 2H), 6.36 (t, <i>J</i>â€‰=â€‰5.6â€‰Hz, 1H), 4.63 (s, 2H), 4.35 (d, <i>J</i>â€‰=â€‰5.7â€‰Hz, 2H), 3.31 (t, <i>J</i>â€‰=â€‰6.6â€‰Hz, 2H), 2.27 (t, <i>J</i>â€‰=â€‰7.3â€‰Hz, 2H), 1.89 (p, <i>J</i>â€‰=â€‰6.9â€‰Hz, 2H) (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S3(A)</a>).<sup>13</sup>C NMR (100â€‰MHz, CDCl<sub>3</sub>) <i>Î´</i> 171.93, 140.30, 137.28, 127.69, 127.17, 64.53, 50.65, 43.19, 32.96, 24.69. IR (KBr) cm<sup>âˆ’1</sup>: 3276, 3055, 2921, 2880, 2103, 1635, 1540, 1418, 1257, 1015, 827, 747, 677, 553. ESI/MS: <i>m/z</i> [Mâ€‰+â€‰H<sup>+</sup>] 249 (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S3(B)</a>).</p> </div> <div class="synthesisof(n-(4-(((2-amino-9h-purin-6-yl)oxy)methyl)benzyl)-4-azidobutanamide)(bgsn3)" title="sec"><span class="label" tagx="label" title="label">2.4.2.</span><div class="title" tagx="title" title="title">
Synthesis of (N-(4-(((2-amino-9H-purin-6-yl)oxy)methyl)benzyl)-4-azidobutanamide) (BGSN3)</div> <p>To a cooled solution (0â€‰Â°C) of compound <b>3</b> (400â€‰mg, 1.611â€‰mmol, 1 eq/mol) in dry DMF (10â€‰ml) in a dry flask under N<sub>2</sub> atmosphere, NaH (60% dispersion in mineral oil, 202â€‰mg, 5.059â€‰mmol, 3.14 eq/mol) was slowly added. The mixture was stirred at 0â€‰Â°C for 10â€‰min, then DMAP (16â€‰mg, 0.129â€‰mmol, 0.08 eq/mol) and compound <b>4</b> (451â€‰mg, 1.772â€‰mmol, 1.1 eq/mol; prepared according to the method by Kindermann etÂ al.<a href="#CIT0037"><sup>37</sup></a> were sequentially added. The reaction was then heated at room temperature and stirred for 4â€‰h until the complete conversion of the starting material (TLC: DCM-MeOH 9:1; Rf 4â€‰=â€‰0.70; Rf BGSN= 0.55), then quenched by slow addition of BRINE and extraction with EtOAc. After drying (Na<sub>2</sub>SO<sub>4</sub>) and evaporation, the residue was purified by gravity column chromatography on silica gel (gradient DCM-MeOH from pure DCM to 20:1) to afford <b>BGSN3</b> as a white solid (413â€‰mg, 67%). <sup>1</sup>H NMR (400â€‰MHz, DMSO-d<sub>6</sub>) <i>Î´</i> 12.48 (bs, NH purine, 1H), 8.43 (t, <i>J</i>â€‰=â€‰5.9â€‰Hz, 1H), 7.85 (s, 1H), 7.49 (d, <i>J</i>â€‰=â€‰7.7â€‰Hz, 2H), 7.30 (d, <i>J</i>â€‰=â€‰7.8â€‰Hz, 2H), 6.31 (bs, NH2 purine 2H), 5.50 (s, 2H), 4.30 (d, <i>J</i>â€‰=â€‰5.9â€‰Hz, 2H), 3.36 (t, <i>J</i>â€‰=â€‰6.8â€‰Hz, 2H), 2.26 (t, <i>J</i>â€‰=â€‰7.4â€‰Hz, 2H), 1.81 (p, <i>J</i>â€‰=â€‰7.1â€‰Hz, 2H) (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S4(A)</a>). <sup>13</sup>C NMR (100â€‰MHz, DMSO-d<sub>6</sub>) <i>Î´</i> 171.38, 159.91, 159.69, 155.23, 139.48, 137.90, 135.31, 128.60, 127.34, 113.57, 66.59, 50.36, 41.95, 32.23, 24.58. IR (KBr) cm<sup>âˆ’ 1</sup>: 3647, 3484, 3379, 3282, 2794, 2100, 1580, 1403, 1282, 1163, 938, 835, 789, 635, 553. ESI/MS: <i>m/z</i> [Mâ€‰+â€‰H<sup>+</sup>] 382 (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S4(B)</a>).</p> </div> </div> <div class="plasmidsandproteinpurification" title="sec"><span class="label" tagx="label" title="label">2.5</span><div class="title" tagx="title" title="title">
Plasmids and protein purification</div> <p>The cloning procedures in the pQE31 expression vector (Qiagen, Germany) were similar for both proteins. In particular, the pSNAP-<i>tag</i>(m) Vector was used as a template to amplify the DNA fragment relative to the SNAP-<i>tag</i><sup>Â®</sup> gene, by using QE_SNAP-Fwd/QE_SNAP-Rev oligonucleotides pairs (5â€²-ATGGCAGGATCCAA TGGACAAAGACTGCGAAATG-3â€²/5â€²-CTATCAAAGCTTAACCCAGCCCAGGCTTGCCCA G-3â€²; <i>BamH</i> I and <i>Hind</i> III sites, respectively, are underlined). Afterwards, the resulting fragment and the pQE31 vector were digested with <i>BamH</i> I and <i>Hind</i> III restriction enzymes and ligated, leading to the final pQE-SNAP plasmid. The final SNAP-<i>tag</i><sup>Â®</sup> protein was expressed with an extra N-terminal aminoacidic sequence, comprising a His<sub>6</sub>-<i>tag</i> (MRGSHHHHHHTDP-). The ligation mixture was used to transform <i>E. coli</i> KRX competent cells and positive colonies were confirmed by PCR and DNA sequence analyses.</p> <p><i>H<sup>5</sup></i> was cloned as previously described<a href="#CIT0022"><sup>22</sup></a>. SNAP-<i>tag</i><sup>Â®</sup> and <i>H<sup>5</sup></i> proteins were expressed in <i>E. coli</i> ABLE C cells, grown at 37â€‰Â°C in Luriaâ€"Bertani (LB) medium supplemented with 50â€‰mg/l kanamycin and 100â€‰mg/l ampicillin. The protein expression was induced with 1â€‰mM isopropyl-thio-Î²-D-galactoside (IPTG) at an absorbance value of 0.5â€"0.6 A<sub>600</sub> nm. After overnight growth, cells were collected and resuspended 1:3 (w/v) in purification buffer (50â€‰mM phosphate, 300â€‰mM NaCl; pH 8.0) supplemented with 1% Triton X-100 and stored overnight at âˆ’20â€‰Â°C. Subsequently, the biomass was treated in ice with lysozyme and DNAse for 60â€‰min and then sonicated as described (Perugino etÂ al., 2012). After centrifugation of 30â€‰min at 60,000â€‰Ã—â€‰<i>g</i>, the cell extract was recovered and applied to a Protino Niâ€"NTA Column 1â€‰ml (Machereyâ€"Nagel) for His<sub>6</sub>-tag affinity chromatography. The eluted fractions containing the protein were collected and dialysed against phosphate-buffered saline (PBS 1Ã—, 20â€‰mM phosphate buffer, NaCl 150â€‰mM, pH 7.3). Pooled fractions were concentrated and protein purification was confirmed by SDS-PAGE analysis. Aliquots were finally stored at âˆ’20â€‰Â°C.</p> </div> <div class="agtsâ€™substrateassaybycompetitiveinhibitionmethod" title="sec"><span class="label" tagx="label" title="label">2.6</span><div class="title" tagx="title" title="title">
AGTsâ€™ substrate assay by competitive inhibition method</div> <p>Competitive inhibition assay was performed as described<a href="#CIT0023"><sup>23</sup></a><sup>,</sup><a href="#CIT0038"><sup>38</sup></a>. Briefly, by using a fixed concentration of the fluorescent <b>SVG</b> (5â€‰ÂµM) and enzymes (5â€‰ÂµM), an increasing amount of guanine-derivatives (0â€"2â€‰mM) was added to the mixtures. The reactions were incubated for 30â€‰min at 25â€‰Â°C and 50â€‰Â°C for SNAP-<i>tag</i><sup>Â®</sup> and <i>H<sup>5</sup></i> respectively, and loaded on SDS-PAGE. Subsequently, fluorescent bands were measured by <i>gel-imaging</i> on a VersaDoc 4000â"¢ system (Bio-Rad), by applying a blue LED/530 bandpass filter. Obtained data were finally plotted by <a href="#M0001">Equation (1)</a>, </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="alternatives" title="alternatives"> <div class="graphic" position="float" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
RF</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
100</div> <div class="mi_UNKNOWN">
%</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
]</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
IC</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
50</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
]</div> </div> </div> </div> </div> </div></div></div> </div> where RF is the obtained Relative Fluorescence, [<i>I</i>] and [<i>S</i>] are the concentration of the inhibitor and the substrate, respectively, and finally the IC<sub>50</sub> is the concentration needed to reduce by 50% the fluorescence intensity of the protein band.<p /> <p>We evaluated the activity of SNAP-<i>tag</i><sup>Â®</sup> and <i>H<sup>5</sup></i> enzymes on <b>BGN3</b> and <b>BGSN3</b> by the afore-mentioned IC<sub>50</sub> method (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S5(A,B)</a>) and other <i>O<sup>6</sup></i>-guanine-derivatives (<a href="#t0001">Table 1</a>).</p> <div class="table-wrap_UNKNOWN" id="t0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>Substrate specificity of SNAP-<i>tag</i><sup>Â®</sup> and <i>H<sup>5</sup></i> by competitive inhibition method (IC<sub>50</sub>) by using <b>SVG</b> as substrate, and second order rate constant of the enzymatic reaction of these <i>protein-tags</i> only on the <b>BGSN3</b> substrate.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th rowspan="2" align="left">Structure</th> <th rowspan="2" align="center">Name</th> <th colspan="2" align="center">SNAP-<i>tag</i><sup>Â®</sup><hr /> </th> <th colspan="2" align="center"><i>Ss</i>OGT-<i>H<sup>5</sup></i><hr /> </th> <th rowspan="2" align="center">Note</th> </tr> <tr> <th align="center">IC<sub>50</sub> (ÂµM)</th> <th align="center"><i>k</i><sup>a</sup> (s<sup>âˆ’1</sup>Â M<sup>âˆ’1</sup>)</th> <th align="center">IC<sub>50</sub> (ÂµM)</th> <th align="center"><i>k</i> (s<sup>âˆ’1</sup>Â M<sup>âˆ’1</sup>)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="left">SVG</td> <td align="center">â€"</td> <td align="center">2.8â€‰Ã—â€‰10<sup>4 b</sup></td> <td align="center">â€"</td> <td align="center">1.6â€‰Ã—â€‰10<sup>4</sup></td> <td align="left">[14,24]</td> </tr> <tr> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="left">BG</td> <td align="center">36.8â€‰Â±â€‰5.6</td> <td align="center">â€"</td> <td align="center">10.1â€‰Â±â€‰1.0</td> <td align="center">â€"</td> <td align="left">This work</td> </tr> <tr> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="left">SCB</td> <td align="center">2.1â€‰Â±â€‰0.5</td> <td align="center">â€"</td> <td align="center">4.4â€‰Â±â€‰0.8</td> <td align="center">â€"</td> <td align="left">This work</td> </tr> <tr> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="left">BGN3</td> <td align="center">15.6â€‰Â±â€‰0.3</td> <td align="center">â€"</td> <td align="center">23.5â€‰Â±â€‰1.0</td> <td align="center">â€"</td> <td align="left">This work</td> </tr> <tr> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="left">BG430</td> <td align="center">ND<sup>c</sup></td> <td align="center">â€"</td> <td align="center">ND</td> <td align="center">â€"</td> <td align="left">This work</td> </tr> <tr> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="left">BGPA</td> <td align="center">86.0â€‰Â±â€‰6.7</td> <td align="center">â€"</td> <td align="center">14.3â€‰Â±â€‰1.9</td> <td align="center">â€"</td> <td align="left">This work</td> </tr> <tr> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="left">MGPA<sup>d</sup></td> <td align="center">â€"</td> <td align="center">â€"</td> <td align="center">268.9â€‰Â±â€‰19.1<sup>e</sup></td> <td align="center">â€"</td> <td align="left">This work</td> </tr> <tr> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="left">BGSN3</td> <td align="center">17.8â€‰Â±â€‰1.1</td> <td align="center">4.64â€‰Â±â€‰1.04â€‰Ã—â€‰10<sup>5</sup></td> <td align="center">10.0â€‰Â±â€‰0.7</td> <td align="center">1.40â€‰Â±â€‰0.47â€‰Ã—â€‰10<sup>4</sup></td> <td align="left">This work</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>For each compound, the guanine moiety is drawn <i>in black</i> and the chemical group conjugated to the benzyl ring <i>in blue</i>. The fluorescein moiety of the <b>SVG</b> is <i>in green</i>, whereas <b>SCB</b> differs from the other derivatives by the presence of a benzylic ring (<i>in red</i>). Azide group is conventionally coloured <i>in magenta</i>.</p> </div> <div class="fn-type-" title=""> <p><sup>a</sup>Reaction rates at 25â€‰Â°C; <sup>b</sup>this value was obtained by using a BG-fluorescein substrate (BG-FL) very similar to SVG; <sup>c</sup>not determined; <sup>d</sup>this molecule is a <i>O<sup>6</sup></i>-methyl-guanine derivative; <sup>e</sup>competitive assay for <i>H<sup>5</sup></i> was performed at 65â€‰Â°C.</p> </div> </div> </div> </div> <div class="invitrohuisgencu(i)-catalysedcycloadditionreaction" title="sec"><span class="label" tagx="label" title="label">2.7.</span><div class="title" tagx="title" title="title">
In vitro Huisgen Cu(I)-catalysed cycloaddition reaction</div> <p>The Huisgen chemical reaction was evaluated on SNAP-<i>tag</i><sup>Â®</sup> and <i>H<sup>5</sup></i> previously incubated with <b>BGN3</b> and <b>BGSN3</b>. An opportune amount of purified proteins was incubated within an equimolar ratio of these substrates for 60â€"120â€‰min at 25â€‰Â°C and 37â€‰Â°C respectively, to ensure the complete enzymatic labelling reaction. Later, we performed the subsequent cycloaddition using 5â€‰ÂµM of an alkyne-derivative of the fluorescein (<b>BDP FL alkyne</b>), in the presence of copper (1â€‰mM), TCEP (1â€‰mM), TBTA (0.1â€‰mM) and, where indicated, of SDS (0.05%). Finally, mixtures were loaded on SDS-PAGE and analysed as described in Section 4 (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S5(C,D)</a>).</p> </div> <div class="molecularmodelling" title="sec"><span class="label" tagx="label" title="label">2.8.</span><div class="title" tagx="title" title="title">
Molecular modelling</div> <p>All molecular modelling studies were performed on a Tesla workstation equipped with two Intel Xeon X5650 2.67â€‰GHz processors and Ubuntu 14.04 (<a href="http://www.ubuntu.com">http://www.ubuntu.com</a>). The protein structures and 3D chemical structures were generated in PyMOL (The PyMOL Molecular Graphics System, version 2.2.3, SchrÃ¶dinger LLC, 2019).</p> </div> <div class="moleculardynamics(md)simulation" title="sec"><span class="label" tagx="label" title="label">2.9.</span><div class="title" tagx="title" title="title">
Molecular dynamics (MD) simulation</div> <p>The MD simulations were carried out using the Desmond simulation package of SchrÃ¶dinger LLC (SchrÃ¶dinger Release 2019â€"1: Desmond Molecular Dynamics System; D. E. Shaw Research: New York, NY, 2019; Maestro-Desmond Interoperability Tools, SchrÃ¶dinger, New York, NY, 2019). The X-ray structure of the <i>H<sup>5</sup></i> covalently bound to <b>SVG</b> was used in this study, entry code 6GA0<a href="#CIT0039"><sup>39</sup></a>, water molecules were removed, and all hydrogen atoms and charges were added. The NPT ensemble with the temperature of 300â€‰K and a pressure 1â€‰bar was applied in all runs. The simulation length was 100â€‰ns with relaxation time 1â€‰ps. The OPLS_2005 force field parameters were used in all simulations<a href="#CIT0040"><sup>40</sup></a>. The long-range electrostatic interactions were calculated using the particle mesh Ewald method<a href="#CIT0041"><sup>41</sup></a>. The cut-off radius in Coulomb interactions was 9.0â€‰Ã…. The water molecules were explicitly described using the simple point charge model<a href="#CIT0042"><sup>42</sup></a>. The Martynaâ€"Tuckermanâ€"Klein chain coupling scheme<a href="#CIT0043"><sup>43</sup></a> with a coupling constant of 2.0â€‰ps was used for the pressure control and the NosÃ©â€"Hoover chain coupling scheme<a href="#CIT0044"><sup>44</sup></a> for the temperature control. Non-bonded forces were calculated using an r-RESPA integrator where the short-range forces were updated every step and the long-range forces were updated every three steps. The trajectory sampling was done at an interval of 1.0â€‰ps. The behaviour and interactions between the ligands and protein were analysed using the Simulation Interaction Diagram tool implemented in the Desmond MD package. The stability of MD simulations was monitored by looking at the RMSD of the ligand and protein atom positions in time.</p> </div> <div class="determinationoftherateconstantsofthechemo-enzymaticlabellingapproach" title="sec"><span class="label" tagx="label" title="label">2.10.</span><div class="title" tagx="title" title="title">
Determination of the rate constants of the chemo-enzymatic labelling approach</div> <p>Rate constants of the enzymatic reactions with the only <b>BGSN3</b> were determined by the method of Gautier etÂ al.<a href="#CIT0014"><sup>14</sup></a>. In this case, purified proteins (5â€‰ÂµM) were incubated with the substrate (5â€‰ÂµM) in PBS 1Ã— buffer at 25â€‰Â°C. Aliquots were taken at different times, the reactions were immediately stopped in Leammli Buffer 1Ã— in addition with 10â€‰ÂµM of <b>Cy5 DBCO</b> fluorophore and placing tubes on ice.</p> <p>Rate constants for the chemical reaction needed for the preliminary achievement of the <i>clickable</i>-SNAP and <i>clickable</i>-<i>H<sup>5</sup></i> with <b>BGSN3</b>, which was obtained by the afore-described protocol, in order to get the complete labelling. Then, to each aliquot of 5â€‰ÂµM of <i>clickable</i> proteins, 20â€‰ÂµM of <b>DBCO-PEG<sub>4</sub>-Fluor 545</b> fluorophore was added. At different times, an excess of sodium azide (NaN<sub>3</sub>, 300â€‰mM) was immediately added to each aliquot and then placing tubes on ice, in order to stop the click reaction between the azide group on the <b>BGSN3</b> and the <b>DBCO-PEG<sub>4</sub>-Fluor 545</b> molecule.</p> <p>Finally, for both the experiments, all aliquots were boiled in an SDS buffer for 5â€‰min, and immediately loaded on a SDS-PAGE, for the <i>gel-imaging</i> and <i>coomassie staining</i> analyses, as previously described. Data were fitted to a pseudo-first-order reaction model using the GraFit 5.0 software package (Erithacus Software Ltd.). Second-order rate constants <i>k</i> (in s<sup>âˆ’1</sup>Â M<sup>âˆ’1</sup>) were then obtained by dividing the pseudo-first-order constant by the concentration of the substrate (<a href="#F0002">Figure 2</a> and <a href="#t0001">Table 1</a>). Values given are an average of at least three independent measurements.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Reaction rates of the chemo-enzymatic approach. Pseudo-first-order reaction of <i>protein</i>-<i>tags</i> for (A) the enzymatic reaction with <b>BGSN3</b> (see <i>k</i> values also in <a href="#t0001">Table 1</a>), and of <i>clickable</i>-<i>tags</i> for (B) Huisgen reaction with <b>DBCO-PEG<sub>4</sub>-Fluor 545</b> (see values in the main text). Values given are an average of three independent measurements. The reaction scheme was an exemplification of <a href="#F0001">Figure 1(B)</a> in the main text. Data are represented as meanâ€‰Â±â€‰SEM.</p>   </div> </div> <div class="invitrohuisgencopper-freecycloadditionreactionwithdifferentdbco-fluorophores" title="sec"><span class="label" tagx="label" title="label">2.11.</span><div class="title" tagx="title" title="title">
In vitro Huisgen copper-free cycloaddition reaction with different DBCO-fluorophores</div> <p>For the copper-free click reaction, aliquots of 5â€‰ÂµM of each <i>clickable</i>-protein were incubated for 60â€‰min at room temperature in the dark with 5â€‰ÂµM of fluorescent DBCO-derivative substrates (<b>BDP FL DBCO</b>, <b>Cy5 DBCO</b>, and <b>DBCO-PEG<sub>4</sub>-Fluor 545</b>) in a total volume of 10â€‰Âµl of PBS 1Ã— buffer (<a href="#F0004">Figure 4</a> and <a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S6</a>). The reactions were finally stopped in Leammli Buffer 3Ã—, loaded on SDS-PAGE, and analysed as described in Section 4, by applying a blue LED/530 bandpass filter, red LED/695 bandpass filter and green LED/605 bandpass filter as excitation/emission parameters for each DBCO-fluorophores, respectively. The click reaction was also performed on 5â€‰ÂµM of both the enzymes, but in the presence of an <i>Ec</i>CFE diluted in PBS 1Ã— buffer.</p> </div> <div class="procedureforproteinimmobilisationonbiolayerinterferometry(bli),byfollowingthechemo-enzymaticapproach" title="sec"><span class="label" tagx="label" title="label">2.12.</span><div class="title" tagx="title" title="title">
Procedure for protein immobilisation on bio layer interferometry (BLI), by following the chemo-enzymatic approach</div> <p>OctetRED96â"¢ (ForteBio, Fremont, CA) was used to immobilise specifically SNAP-<i>tag</i><sup>Â®</sup> and <i>H<sup>5</sup></i> with the <i>chemo-enzymatic approach</i> (<a href="#F0005">Figure 5(A,B)</a>). Samples and reaction buffers were located in black 96-well plates (OptiPlate-96 Black, Black Opaque 96-well Microplate, PerkinElmer, Billerica, MA) in a maximum reaction volume of 300â€‰Âµl per well with 800â€‰rpm shaking for each step. For the immobilisation procedure, AR2G sensors were first wetted in 200â€‰Âµl of pure water for at least 15â€‰min, followed by an equilibration step (3â€‰min) in acetate buffer 0.1â€‰M, pH 5.0. Afterwards, they were activated with 20â€‰mM 1-ethyl-3-(3-dimethyl-amino-propyl) carbodiimide hydrochloride (EDC)/20â€‰mM N-hydroxy-sulfo-succinimide (sulfo-NHS) mixture in acetate buffer (60â€‰min) and covered with 2â€‰mM <b>propargyl-PEG<sub>3</sub>-amine</b> bifunctional linker (BroadPharm, San Diego, CA) in Loading step (20â€‰min). To avoid the presence of any free amine groups on the biosensors, a Blocking step with Ethanolamine 1â€‰M (30â€‰min) was performed. Subsequently, a Washing step (15â€‰min) with water and an Equilibration step in click-reaction buffer (15â€‰min) are followed.</p> <p>During the afore-described procedure, proteins were labelled with <b>BGSN3</b>. Finally, the immobilisation step for each sample via Huisgen reaction was carried out at 30â€‰Â°C for 80â€‰min, followed by a Washing step (20â€‰min), in order to remove all the unbound molecules. This procedure was the same in the presence of the <i>Ec</i>CFE. All measurements were performed in triplicates.</p> </div> <div class="permeabilityofeukaryoticandprokaryoticcellstobgsn3" title="sec"><span class="label" tagx="label" title="label">2.13.</span><div class="title" tagx="title" title="title">
Permeability of eukaryotic and prokaryotic cells to BGSN3</div> <p>HEK293T cells were maintained at 37â€‰Â°C with 5% CO<sub>2</sub> in Dulbeccoâ€™s Modified Essential Medium (Invitrogen, Carlsbad, CA) supplemented with 10% Foetal Bovine Serum (FBS) (Invitrogen) and 100â€‰U/ml Penicillin/Streptomycin (Roche, Switzerland). HEK293T cells were transfected with SNAP-<i>tag</i><sup>Â®</sup> plasmid by using Lipofectamine 2000 (Invitrogen) following manufacturerâ€™s protocol. The treatment with <b>BGSN3</b> were performed, at the concentration and time indicated for each experiment. Twenty-four hours after transfection, we treated cells with <b>BGSN3</b> for 2â€‰h at different concentrations ranging (from 1 to 25â€‰ÂµM), directly dissolving the compound in complete culture medium. Then cells were harvested, washed with PBS 1Ã— buffer and lysed with 50â€‰mM Tris-HCl pH 7.4, 150â€‰mM NaCl, 0.5â€‰mM EDTA, 0.1% Triton X-100 supplemented with complete protease (Roche, Switzerland) and phosphatase (SERVA Electrophoresis, Germany) inhibitors. Afterwards, transfected cells were treated with a fixed concentration of <b>BGSN3</b> (10â€‰ÂµM) at different time points (from 30 to 120â€‰min). Again, HEK293T cells were washed and lysed as described before. To confirm the reaction with <b>BGSN3</b>, the same amount of protein extract (0.91â€‰Âµg/ÂµL for each sample) was incubated for 30â€‰min at 25â€‰Â°C with <b>SVG</b>. Subsequently, proteins were loaded on SDS-PAGE and analysed by <i>gel-imaging</i> on a VersaDoc 4000â"¢ system (Bio-Rad), by applying a blue LED/530 bandpass filter (<a href="#F0006">Figure 6</a>).</p> <p>For flow cytometry analysis, HeLa cells were seeded in 24-well plates and transfected with SNAP-<i>tag</i><sup>Â®</sup> plasmid by using Lipofectamine 2000 (Invitrogen, USA) following manufacturerâ€™s protocol. Twenty-four hours after the transfection, cells were treated with 25â€‰ÂµM <b>BGSN3</b> for 1â€‰h, and the excess of the substrate was washed out by 2â€‰Ã—â€‰15â€‰min, followed by 1â€‰Ã—â€‰30â€‰min washes. Cells where then treated with 2.5â€‰ÂµM <b>BDP FL DBCO</b> for 30â€‰min and unbound fluorophore was removed by following the same procedure performed for the <b>BGSN3</b>. All treatments and washes were performed at 37â€‰Â°C in a complete culture medium. Lastly, cells were harvested by trypsinization, and fluorescence was measured using FACS CANTO II instrument. The analysis was performed on live singlet cells using FlowJo software (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S7(A)</a>).</p> <p><i>E. coli</i> ABLE C strain was transformed with SNAP-<i>tag</i><sup>Â®</sup> plasmid and protein expressed as previously described. After overnight growth, samples of 2â€‰ml were treated with 100â€‰ÂµM of <b>BGSN3</b> for 2â€‰h at 25â€‰Â°C and then collected by centrifugation at 2000â€‰Ã—â€‰<i>g</i>. Cell pellets of 0.05â€‰g were resuspended 1:3 (w/v) in PBS 1Ã— supplemented with 1% Triton X-100 and subjected to cell lysis, by applying 5 cycles of freeze-thawing. After a centrifugation at 13,000â€‰Ã—â€‰<i>g</i>, the supernatants containing the protein extract were incubated 30â€‰min at 25â€‰Â°C with <b>SVG,</b> and proteins were loaded on SDS-PAGE. Finally, fluorescent bands were analysed by <i>gel-imaging</i> techniques (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S7(B)</a>).</p> </div> </div> <div class="resultanddiscussion" title="sec"><span class="label" tagx="label" title="label">3.</span><div class="title" tagx="title" title="title">
Result and discussion</div> <div class="substratespecificityofagtsonbg-basedsubstrates" title="sec"><span class="label" tagx="label" title="label">3.1.</span><div class="title" tagx="title" title="title">
Substrate specificity of AGTs on BG-based substrates</div> <p>Following the irreversible reaction shown in <a href="#F0001">Figure 1</a>, we evaluated the activity of two enzymes in our possession on several <i>O<sup>6</sup></i>-guanine-derivatives (<a href="#t0001">Table 1</a>). Because most of them are non-fluorescent compounds, we performed an AGTsâ€™ competitive inhibition assay by using the fluorescein-derivative SNAP-Vista<sup>Â®</sup> Green as substrate (<b>SVG</b>), as previously described<a href="#CIT0022"><sup>22â€"24</sup></a><sup>,</sup><a href="#CIT0045"><sup>45</sup></a>. Briefly, the reaction of an AGT with <b>SVG</b> led to a fluoresceinated protein, which can be visualised as a fluorescent band in <i>gel-imaging</i> analysis after SDS-PAGE. The presence of increasing amounts of a non-fluorescent competitor in the reaction causes a decrease of the fluorescent signals, which can be measured and plotted for the IC<sub>50</sub> values determination<a href="#CIT0023"><sup>23</sup></a><sup>,</sup><a href="#CIT0046"><sup>46</sup></a>. As shown in <a href="#t0001">Table 1</a>, SNAP-<i>tag</i><sup>Â®</sup> and <i>H<sup>5</sup></i> displayed different behaviours versus these competitors, without any rationale for the dimension and/or polarity of the conjugated chemical groups. While SNAP-Cell<sup>Â®</sup> 430 (<b>SC430</b>) completely lost the competition with <b>SVG</b>, both the enzymes are extremely active on the SNAP Cell<sup>Â®</sup> Block (<b>SCB</b>), displaying the lowest IC<sub>50</sub> value measured. This result was expected, because <b>SCB</b> has a structure very similar to the Lomeguatrib, one of the most efficient inhibitors of the hMGMT protein, employed in the cancer treatment in combination with alkylating agents-based chemotherapeutics<a href="#CIT0047"><sup>47</sup></a>.</p> <p>In general, all commercially available products used (<b>SVG</b>, <b>SCB</b>, <b>BG430</b>, and <b>BG-PEG-NH2</b>, <b>BGPA</b>) are good substrates for the SNAP-<i>tag</i><sup>Â®</sup> and <i>H<sup>5</sup></i> enzymes, completing their labelling reaction in few hours (data not shown). However, based on our results, the choice of the chemical group to be conjugated to the <i>O<sup>6</sup></i>-guanine for zcustomized substrates may present risks, with consequent decreases in the reaction rate for these <i>protein-tags</i>. This was the case of methyl-guanine-PEG-NH2 (<b>MGPA</b>), which is an <i>O<sup>6</sup></i>-methyl-guanine derivative, used for the immobilisation of SNAP-<i>tag</i><sup>Â®</sup> on nanoparticles<a href="#CIT0048"><sup>48</sup></a>. The latter is not a preferred substrate, probably because of the absence of the benzyl ring, which leads to complete labelling of the SNAP-<i>tag</i><sup>Â®</sup> and <i>H<sup>5</sup></i> after over-night incubation at 4â€‰Â°C<a href="#CIT0048"><sup>48</sup></a> and 65â€‰Â°C (data not shown), respectively.</p> </div> <div class="invitroenzymaticreactionofengineeredagtswithbg-azidesubstrates" title="sec"><span class="label" tagx="label" title="label">3.2.</span><div class="title" tagx="title" title="title">
In vitro enzymatic reaction of engineered AGTs with BG-azide substrates</div> <p>Recent studies were focussed on the synthesis of alternative â€œBG-building blocks,â€� which offer the opportunity to produce SNAP-substrates by following easier and faster protocols: an alkyne substituted <i>O<sup>6</sup></i>-BG was employed in the synthesis of compounds by the Huisgen cycloaddition with azide-based fluorescent probes<a href="#CIT0049"><sup>49</sup></a> or, inversely, by using the <i>O<sup>6</sup></i>-BG-N<sub>3</sub> (<b>BGN3</b>, <a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S1</a>) for the conjugation with alkyne-based chemical groups<a href="#CIT0034"><sup>34</sup></a>. We evaluated the enzymatic reaction of the <i>H<sup>5</sup></i> and the SNAP-<i>tag</i><sup>Â®</sup><i>directly</i> on <b>BGN3</b> and a synthesised BG-derivative containing a benzyl ring opportunely spaced from the azide group (<b>BGSN3</b>, <a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S4</a>): after the reaction, no fluorescent signal was obtained on SDS-PAGE <i>gel-imaging</i> upon the addition of <b>SVG</b> (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S5(A,B)</a>). This indicates that the catalytic cysteine was completely blocked by the benzyl-azide moiety, impeding the access of the fluorescent substrate to the active site. Compared to the classical BG-derivatives, these <i>protein-tags</i> showed a reasonable activity on both these BG-azides, as resulted by the calculated IC<sub>50</sub> (<a href="#t0001">Table 1</a> and <a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S5(A,B)</a>).</p> <p>After the enzymatic reaction of <i>H<sup>5</sup></i> with <b>BGN3</b> and <b>BGSN3</b>, we performed the subsequent cycloaddition using an alkyne-derivative of the fluorescein (<b>BDP FL alkyne</b>): however, the chemical reaction was less efficient using the former substrate (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S5(C)</a>, lane 2). In this case, the complete fluorescein labelling of the protein was achieved only in the presence of a small amount of SDS during the cycloaddition step (lane 3), suggesting that the protein is still folded after the enzymatic reaction and the azide is hidden in the active site core. The addition of the denaturant could have slightly opened the protein structure, favouring a better exposure of the azide group to the solvent, and allowing the click reaction to occur.</p> <p>On the contrary, using <b>BGSN3</b> as substrate, the labelling of both the enzymes was comparable to the classical reaction with <b>SVG</b> without any denaturing agent, likely the longer spacer of <b>BGSN3</b> could sufficiently move away from the azide group from the protein surface for the Huisgen reaction (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S5(D)</a>, lanes 2 and 4). From now on, experiments were only performed by using the longer BG-azide. We first calculated the rate of the enzymatic reaction, demonstrating that both <i>protein-tags</i> show a high catalytic activity comparable to the commercial BG-derivatives currently used (<a href="#F0002">Figure 2(A)</a> and <a href="#t0001">Table 1</a>), also indicating that the complete protein labelling in less of an hour can be performed<a href="#CIT0013"><sup>13</sup></a><sup>,</sup><a href="#CIT0014"><sup>14</sup></a><sup>,</sup><a href="#CIT0024"><sup>24</sup></a>.</p> </div> <div class="molecularmodellingontheh5withbg-azides" title="sec"><span class="label" tagx="label" title="label">3.3.</span><div class="title" tagx="title" title="title">
Molecular modelling on the H<sup>5</sup> with BG-azides</div> <p><b>BGN3</b> and <b>BGSN3</b> differ in length since the chemical spacer between the benzyl ring and the active azide makes the latter potentially more prone to the labelling reaction. It could be assumed that this aspect alone influences the availability of the azide moiety to react. However, proteins are not a static system, the amino acids side-chain movements could mask the azide and prevent the â€œclickâ€� chemistry reaction. The covalent complexes of these compounds with <i>H<sup>5</sup></i> were analysed with Molecular Dynamics (MD) simulations using the Desmond package (see Experimental Section). The complexes were simulated for 100â€¯nsâ€¯at 300â€¯K using a standard protocol. The protein structure has been stabilised, as shown in the RMSDs for both the IDO1 CÎ± and the ligand (<a href="#F0003">Figure 3(A,C)</a>). The MD results were analysed in terms of Solvent Accessible Surface Area (SASA) of the compounds: more time the compounds are exposed to the solvent, the higher is the possibility to react<a href="#CIT0050"><sup>50</sup></a>. In <a href="#F0003">Figure 3</a> is reported the fluctuation of the SASA values over the simulation time together with the structure model of the <i>H<sup>5</sup></i> protein in complex with <b>BGN3</b> and <b>BSGN3</b>, respectively. The former is less exposed to the solvent with a SASA value of 32.967â€‰Â±â€‰18.573 Ã…<sup>2</sup> compared to <b>BGSN3</b>, which shows a higher SASA value 68.302â€‰Â±â€‰32.455 Ã…<sup>2</sup>. This simulation confirmed our biochemical data, proposing the BG-derivative with the spacer as a better substrate for our <i>chemo-enzymatic approach</i>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>Molecular modelling on <i>H<sup>5</sup></i> with BG-azides. (A) RMSD of the atomic positions for the compound <b>BGN3</b> (Lig fit Prot, <i>in red</i>) and the protein <i>H<sup>5</sup></i> (CÎ± positions, <i>in blue</i>) of the 100â€‰ns molecular dynamics simulations using Desmond package. (B) A timeline representation of the interactions and contacts (H-bonds, Hydrophobic, Ionic, Water bridges). (C) RMSD of the atomic positions for the compound <b>BGSN3</b> (Lig fit Prot, <i>in red</i>) and the protein <i>H<sup>5</sup></i> (CÎ± positions, <i>in blue</i>) of the 100â€‰ns molecular dynamics simulations using Desmond package. (D) A timeline representation of the interactions and contacts (H-bonds, Hydrophobic, Ionic, Water bridges). (E) Solvent Accessible Surface Area (SASA) of <b>BGN3</b>/<i>H<sup>5</sup></i> (<i>in orange</i>) and <b>BGSN3</b>/<i>H<sup>5</sup></i> (<i>in cyan</i>) complexes over the MD simulation time (mean values are depicted as dot lines). Frames of <i>H<sup>5</sup></i>-probe complexes with lower (F, H) and higher (G, I) SASA value for <b>BGN3</b> (F, G) and <b>BGSN3</b> (H, I), respectively.</p>   </div> </div> <div class="specificityandversatilityofthechemo-enzymaticreaction" title="sec"><span class="label" tagx="label" title="label">3.4.</span><div class="title" tagx="title" title="title">
Specificity and versatility of the chemo-enzymatic reaction</div> <p>The <i>O<sup>6</sup></i>-BG-based <b>BGSN3</b> is a good substrate for the two <i>protein-tags</i> used (<a href="#t0001">Table 1</a> and <a href="#F0002">Figure 2(A)</a>) and offering the advantage to sufficiently expose the azide group for the Huisgen reaction. This was the starting point to examine: (i) the labelling efficiency of the <i>clickable</i>-SNAP and <i>clickable</i>-<i>H<sup>5</sup></i> by using different DBCO-based fluorophores; (ii) the specificity of the â€œclickâ€� reaction.</p> <p>Upon the reaction with <b>BGSN3</b>, all cycloaddition reactions with three different DBCO-based fluorophores were complete in ca. 30â€"45â€‰min in PBS 1Ã— buffer (<a href="#F0004">Figure 4</a>, lanes 2â€"4), with a protein-labelling as efficient as the enzymatic reaction using the sole <b>SVG</b> (lane 1). We quantitatively evaluated the rate (<i>k</i>) of the click reaction by using the <b>DBCO-PEG4-Fluor 545</b> fluorophore: as expected, both the <i>clickable-tags</i> were labelled with the same efficiency (1.83â€‰Â±â€‰0.41â€‰Ã—â€‰10<sup>3</sup> s<sup>âˆ’1</sup>Â M<sup>âˆ’1</sup> for SNAP-<i>tag</i><sup>Â®</sup>; 1.54â€‰Â±â€‰0.39â€‰Ã—â€‰10<sup>3</sup> s<sup>âˆ’1</sup>Â M<sup>âˆ’1</sup> for <i>H<sup>5</sup></i>), demonstrating that the chemical reaction is sufficiently fast and independent from the <i>tags</i> (<a href="#F0002">Figure 2(B)</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>Specificity of the Huisgen reaction. <i>Gel-imaging</i> analysis of SNAP-<i>tag</i><sup>Â®</sup> labelling by a <i>chemo-enzymatic approach</i> with <b>BGSN3</b> and three different DBCO-derivative fluorophores. Protein (5â€‰ÂµM) was incubated with 5â€‰ÂµM of the azide-based BG for 60â€‰min at 25â€‰Â°C; then, an equimolar amount of DBCO-based substrate was added for the chemical click reaction, keeping the same time and temperature conditions. As control, SNAP-<i>tag</i><sup>Â®</sup> was incubated only with <b>SVG</b> (lane 1, signal marked with an asterisk).</p>   </div> <p>Concerning the specificity, we added a crude protein extract from <i>Escherichia coli</i> ABLE C (<i>Ec</i>CFE), without any AGT activity at the <i>gel-imaging</i> analysis (<a href="#F0004">Figure 4</a>, lane 5). In this context, the only presence of the free <i>protein-tag</i> and the DBCO-fluorophore also did not result in any fluorescent signal (lanes 6), whereas the previously purified <i>clickable</i>-SNAP (lane 7), as well as its free form in the presence of <b>BGSN3</b> (lane 8), was specifically able to complete the chemo-enzymatic reaction, giving an evident fluorescent signal. The high specificity of our approach was also confirmed by using the <i>H<sup>5</sup></i> enzyme, which displays a better labelling reaction than the mesophilic SNAP-<i>tag</i><sup>Â®</sup> (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S6</a>). Probably, something in the extract might impede SNAP-<i>tag</i><sup>Â®</sup> activity. These results clearly demonstrated the high efficiency of our <i>chemo-enzymatic approach</i> for the labelling of both the <i>protein-tags</i> used.</p> </div> <div class="applicationtothebiolayerinterferometry" title="sec"><span class="label" tagx="label" title="label">3.5.</span><div class="title" tagx="title" title="title">
Application to the bio layer interferometry</div> <p>The possibility to apply the SNAP-<i>tag</i><sup>Â®</sup> technology to the Surface Plasmon Resonance (SPR) for the covalent immobilisation of a protein of interest was first explored by the group of Kai Johnsson<a href="#CIT0037"><sup>37</sup></a>, followed by other groups with the same substrate<a href="#CIT0051"><sup>51</sup></a> or a biotin BG-derivative<a href="#CIT0052"><sup>52</sup></a>. Their approaches, again, required preliminarily the synthesis and the purification of a compatible substrate to cover the sensor chip surface. We used, instead, the <b>BGSN3</b> substrate for the immobilisation of the SNAP-<i>tag</i><sup>Â®</sup> directly on an alkyne-derived sensor chip of the bio layer interferometry (BLI) equipment, as shown in <a href="#F0005">Figure 5</a>. This technique is more advantageous with respect to the SPR because: (i) it needs a smaller amount of sample, making it more compatible to higher throughput (the capacity of running up to 96 samples in a parallel); (ii) the possibility to reuse samples, and (iii) of the total independency from any microfluidic issues.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Covalent immobilisation of clickable-tags on the BLI sensor. (A) Covering of the BLI sensor with a bi-functional linker, exposing alkyne groups for the Huisgen cycloaddition reaction; (B) reaction of the SNAP-<i>tag</i><sup>Â®</sup> with <b>BGSN3</b>; (C) chemo-enzymatic SNAP-<i>tag</i><sup>Â®</sup> immobilisation on BLI. The alkyne-covered sensor (<i>silver cylinder</i>) was immersed in wells containing the buffer (<i>in black</i>), the free SNAP-<i>tag</i><sup>Â®</sup> (<i>in blue</i>) and the <i>clickable</i>-SNAP (<i>in magenta</i>); (D) column chart relative to the BLI immobilisation of purified <i>protein-tags</i> alone (<i>black-bordered bars</i>) or in the presence of <b>BGSN3</b> (<i>magenta-bordered bars</i>). Filled magenta bars represent the BLI immobilisation using the <i>Ec</i>CFE upon heterologous expression of <i>protein-tags</i>. Standard deviations were obtained from three independent experiments. Data are represented as meanâ€‰Â±â€‰SEM.</p>   </div> <p>Given the lack of any available BLI alkyne-derived sensors, we first activated the AR2G type by a bi-functional linker (<b>propargyl-PEG 3-amine</b>) in order to expose an alkyne group on the surface (<a href="#F0005">Figure 5(A)</a>). This modified protocol provides first the coating of the sensor tips with alkyne groups (approx. 80â€‰min), during that the reaction between the <i>protein-tag</i> and <b>BGSN3</b> inside the 96-wells rack takes place (<a href="#F0005">Figure 5(B)</a>). Only the contemporary presence of the <i>clickable</i>-SNAP and the alkyne-coated sensor led to a measurable response (<a href="#F0005">Figure 5(C)</a>). After washing procedures, the signal did not significantly drop-down, given the covalent reaction between the protein and the sensor. We successfully achieved results with both the enzymes, although temperature and times of the enzymatic reaction on BLI (30â€‰Â°C) favoured the SNAP-<i>tag</i><sup>Â®</sup> respect to the thermophilic <i>H</i><sup>5</sup><a href="#CIT0013"><sup>13</sup></a><sup>,</sup><a href="#CIT0024"><sup>24</sup></a>. Furthermore, in <i>Ec</i>CFEs where both the enzymes were expressed, a specific and efficient immobilisation on BLI sensor tips occurred (<a href="#F0005">Figure 5(D)</a>), although the SNAP-<i>tag</i><sup>Â®</sup> displayed a lower labelling efficiency in the <i>Ec</i>CFE, as expected (compare lane 8 in <a href="#F0004">Figure 4</a> and <a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S6</a>). As for other techniques, this specific surface immobilisation of SNAP-<i>tag</i><sup>Â®</sup> gives the opportunity to perform a directly <i>on-chip purification</i> of a tagged-POI from a crude lysate. without any purification step, in an indirect manner, which favours a better orientation of the POI for its biological activities.</p> </div> <div class="permeabilityofeukaryoticandprokaryoticcellstobg-azides" title="sec"><span class="label" tagx="label" title="label">3.6.</span><div class="title" tagx="title" title="title">
Permeability of eukaryotic and prokaryotic cells to BG-azides</div> <p>One of the major applications of the SNAP-<i>tag</i><sup>Â®</sup> technology concerns the field of cell biology, where detecting fluorescent-tagged-POIs in living cells represents an important tool to study protein functions and locations<a href="#CIT0053"><sup>53</sup></a>. To test our <i>chemo-enzymatic approach</i>, we first investigated the permeability of <b>BGSN3</b>. Lysates of HEK293T cells pre-treated with <b>BGNS3</b> were then incubated with the <b>SVG</b> substrate: the absence of any fluorescent signal by <i>gel-imaging</i> only in BG-azide treated lysates demonstrated that the internalisation of <b>BGSN3</b> was fast (ca. 30â€‰min; <a href="#F0006">Figure 6</a>, lane 3) and at concentrations comparable with commercial cell biology BG-substrates (in the range of &amp;lt;5â€‰ÂµM; <a href="#F0006">Figure 6</a>, lane 8). Preliminary experiments by FACS analysis confirmed that the <i>inÂ vivo</i> cycloaddition between <b>BGSN3</b> and the <b>BDP-FL DBCO</b> occurred (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S7(A)</a>). This was also confirmed for <i>E. coli</i> bacterial cells (<a href="https://doi.org/10.1080/14756366.2020.1841182">Figure S7(B)</a>, lane 2).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>Eukaryotic permeability to BGSN3. SDS-PAGE analysis by <i>gel-imaging</i> and <i>coomassie staining</i> of HEK293T cell lysates. After <b>BGSN3</b> in medium treatment, lysates were incubated with <b>SVG</b>.</p>   </div> </div> </div> <div class="conclusionsandperspectives" title="sec"><span class="label" tagx="label" title="label">4.</span><div class="title" tagx="title" title="title">
Conclusions and perspectives</div> <p>We developed an innovative modification of the SNAP-<i>tag</i><sup>Â®</sup> technology, in order to overcome times and costs relative to the production and the utilisation of commercial or purified customised BG-derivatives. Although they are compatible in terms of catalytic activity as for the SNAP-<i>tag</i><sup>Â®</sup>, as well as for the others AGTs<a href="#CIT0022"><sup>22â€"24</sup></a><sup>,</sup><a href="#CIT0037"><sup>37</sup></a><sup>,</sup><a href="#CIT0046"><sup>46</sup></a><sup>,</sup><a href="#CIT0054"><sup>54</sup></a> the risk of lowering the catalytic activity of these <i>tags</i> with customised BG-derivatives should not be underestimated (<a href="#t0001">Table 1</a>). We started by the knowledge that: (i) self-labelling <i>protein-tags</i> are still folded and enough stability in their benzylated form after the enzymatic reaction<a href="#CIT0013"><sup>13</sup></a><sup>,</sup><a href="#CIT0024"><sup>24</sup></a>; (ii) the Huisgen cycloaddition is extremely versatile, fast and specific. Recently, the latter was used for the entrapment of catalytic activities by azide-based pseudo-substrates in a well-known powerful method, the <i>inÂ vivo</i> activity-based protein profiling (ABPP)<a href="#CIT0055"><sup>55</sup></a>. For these reasons, a <i>chemo-enzymatic approach</i> (<a href="#F0001">Figure 1(B)</a>) with an opportunely selected azide-based BG-substrate (<b>BGSN3</b>) was set up: the efficient exposition of the azide outside the protein surface allows the reaction with a huge number of commercially DBCO-based molecules, more than those BG-derivatives, keeping high the specificity in the presence of <i>inÂ vitro</i> â€œperturbingâ€� proteins (like in cell lysates) and the <i>inÂ vivo</i> labelling of expressed SNAP-<i>tag</i><sup>Â®</sup> in eukaryotic cells. Finally, <b>BGSN3</b> proved to be a good substrate for the direct immobilisation of these <i>tags</i> on solid surfaces. We demonstrated that splitting the SNAP-<i>tag</i><sup>Â®</sup> reaction into two fast steps, as experimentally measured (<a href="#F0002">Figure 2(A,B)</a>), does not affect the overall rate and efficiency of the protein labelling<a href="#CIT0013"><sup>13</sup></a><sup>,</sup><a href="#CIT0024"><sup>24</sup></a>, thus opening new perspectives and widening the applications of this powerful biotechnology.</p> </div> <div class="authorsâ€™contributions" title="sec"> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>Conceptualisation, G.P. and A.Mi.; Methodology, R.Me., G.P.; Investigation, R.Me., D.C. and M.C.; Formal Analysis, C.M., R.Mi. and A.Ma.; Validation, R.Me., R.Ma., A.Mi. and G.P.; Writing â€" Original Draft, R.Me. and G.P.; Writing â€" Review &amp;amp; Editing, C.M., F.R. and A.V and G.P.; Funding Acquisition, A.V. and G.P.; Resources, R.Me, R.Ma. and D.C.; Supervision, A.L., A.Mi. and G.P.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="IENZ_A_1841182_SM7385.zip">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgement</div> <p>We are grateful to prof. Davide Prosperi (University of Milan Bicocca) for the kind gift of the <b>MGPA</b> substrate.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>No potential conflict of interest was reported by the author(s).</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chudakov</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matz</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lukyanov</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lukyanov</span><span class="given-names" tagx="given-names" title="given-names">KA.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fluorescent proteins and their applications in imaging living cells and tissues</span>. <span class="source" tagx="source" title="source">Physiol Rev</span><span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">90</span>:<span class="fpage" tagx="fpage" title="fpage">1103</span>â€"<span class="lpage" tagx="lpage" title="lpage">63</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20664080">20664080</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liss</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barlag</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nietschke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hensel</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Self-labelling enzymes as universal tags for fluorescence microscopy, super-resolution microscopy and electron microscopy</span>. <span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">17740</span>â€"<span class="lpage" tagx="lpage" title="lpage">52</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26643905">26643905</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hatlem</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trunk</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linke</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leo</span><span class="given-names" tagx="given-names" title="given-names">JC.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Catching a SPY: using the SpyCatcher-SpyTag and related systems for labeling and localizing bacterial proteins</span>. <span class="source" tagx="source" title="source">Int J Mol Sci</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">20</span>:<span class="fpage" tagx="fpage" title="fpage">2129</span>â€"<span class="lpage" tagx="lpage" title="lpage">39</span>.</span></li> <li tag="ref"><a name="CIT0004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Los</span><span class="given-names" tagx="given-names" title="given-names">GV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Encell</span><span class="given-names" tagx="given-names" title="given-names">LP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McDougall</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">HaloTag: a novel protein labeling technology for cell imaging and protein analysis</span>. <span class="source" tagx="source" title="source">ACS Chem Biol</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">373</span>â€"<span class="lpage" tagx="lpage" title="lpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18533659">18533659</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">England</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">HaloTag technology: a versatile platform for biomedical applications</span>. <span class="source" tagx="source" title="source">Bioconjug Chem</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">26</span>:<span class="fpage" tagx="fpage" title="fpage">975</span>â€"<span class="lpage" tagx="lpage" title="lpage">86</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25974629">25974629</a></span></span></li> <li tag="ref"><a name="CIT0006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pegg</span><span class="given-names" tagx="given-names" title="given-names">AE.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Repair of O(6)-alkylguanine by alkyltransferases</span>. <span class="source" tagx="source" title="source">Mutat Res</span><span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">462</span>:<span class="fpage" tagx="fpage" title="fpage">83</span>â€"<span class="lpage" tagx="lpage" title="lpage">100</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10767620">10767620</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daniels</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mol</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arvai</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Active and alkylated human AGT structures: a novel zinc site, inhibitor and extrahelical base binding</span>. <span class="source" tagx="source" title="source">Embo J</span><span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">1719</span>â€"<span class="lpage" tagx="lpage" title="lpage">30</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10747039">10747039</a></span></span></li> <li tag="ref"><a name="CIT0008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daniels</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woo</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luu</span><span class="given-names" tagx="given-names" title="given-names">KX</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">DNA binding and nucleotide flipping by the human DNA repair protein AGT</span>. <span class="source" tagx="source" title="source">Nat Struct Mol Biol</span><span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">714</span>â€"<span class="lpage" tagx="lpage" title="lpage">20</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15221026">15221026</a></span></span></li> <li tag="ref"><a name="CIT0009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keppler</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gendreizig</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gronemeyer</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A general method for the covalent labeling of fusion proteins with small molecules <i>inÂ vivo</i></span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span><span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">21</span>:<span class="fpage" tagx="fpage" title="fpage">86</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12469133">12469133</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keppler</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pick</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arrivoli</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Labeling of fusion proteins with synthetic fluorophores in live cells</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">9955</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.</span></li> <li tag="ref"><a name="CIT0011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juillerat</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gronemeyer</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keppler</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Directed evolution of <i>O6</i>-alkylguanine-DNA alkyltransferase for efficient labeling of fusion proteins with small molecules <i>inÂ vivo</i></span>. <span class="source" tagx="source" title="source">Chem Biol</span><span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">313</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12725859">12725859</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gronemeyer</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chidley</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juillerat</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Directed evolution of O6-alkylguanine-DNA alkyltransferase for applications in protein labeling</span>. <span class="source" tagx="source" title="source">Protein Eng Des Sel</span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">309</span>â€"<span class="lpage" tagx="lpage" title="lpage">16</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16638797">16638797</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mollwitz</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunk</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmitt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Directed evolution of the suicide protein <i>O6</i>-alkylguanine-DNA alkyltransferase for Increased reactivity results in an alkylated protein with exceptional stability</span>. <span class="source" tagx="source" title="source">Biochemistry</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">51</span>:<span class="fpage" tagx="fpage" title="fpage">986</span>â€"<span class="lpage" tagx="lpage" title="lpage">94</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22280500">22280500</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gautier</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juillerat</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heinis</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">An engineered protein tag for multiprotein labeling in living cells</span>. <span class="source" tagx="source" title="source">Chem Biol</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">128</span>â€"<span class="lpage" tagx="lpage" title="lpage">36</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18291317">18291317</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vogel</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moschref Merkle</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Efficient and precise editing of endogenous transcripts with SNAP-tagged ADARs</span>. <span class="source" tagx="source" title="source">Nat Methods</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">535</span>â€"<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29967493">29967493</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srikun</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albers</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nam</span><span class="given-names" tagx="given-names" title="given-names">CI</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Organelle-targetable fluorescent probes for imaging hydrogen peroxide in living cells via SNAP-Tag protein labeling</span>. <span class="source" tagx="source" title="source">J Am Chem Soc</span><span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">132</span>:<span class="fpage" tagx="fpage" title="fpage">4455</span>â€"<span class="lpage" tagx="lpage" title="lpage">65</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20201528">20201528</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minakami</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyano</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Visualization of phagosomal hydrogen peroxide production by a novel fluorescent probe that is localized via SNAP-tag labeling</span>. <span class="source" tagx="source" title="source">Anal Chem</span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">86</span>:<span class="fpage" tagx="fpage" title="fpage">5983</span>â€"<span class="lpage" tagx="lpage" title="lpage">90</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24862209">24862209</a></span></span></li> <li tag="ref"><a name="CIT0018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griss</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schena</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reymond</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bioluminescent sensor proteins for point-of-care therapeutic drug monitoring</span>. <span class="source" tagx="source" title="source">Nat Chem Biol</span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">598</span>â€"<span class="lpage" tagx="lpage" title="lpage">603</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24907901">24907901</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tomat</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nolan</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaworski</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lippard</span><span class="given-names" tagx="given-names" title="given-names">SJ.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Organelle-specific zinc detection using zinpyr-labeled fusion proteins in live cells</span>. <span class="source" tagx="source" title="source">J Am Chem Soc</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">130</span>:<span class="fpage" tagx="fpage" title="fpage">15776</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18973293">18973293</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamiya</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnsson</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Localizable and highly sensitive calcium indicator based on a BODIPY fluorophore</span>. <span class="source" tagx="source" title="source">Anal Chem</span><span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">82</span>:<span class="fpage" tagx="fpage" title="fpage">6472</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20590099">20590099</a></span></span></li> <li tag="ref"><a name="CIT0021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SaccÃ </span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erkelenz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Orthogonal protein decoration of DNA origami</span>. <span class="source" tagx="source" title="source">Angew Chem Int Ed Engl</span><span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">49</span>:<span class="fpage" tagx="fpage" title="fpage">9378</span>â€"<span class="lpage" tagx="lpage" title="lpage">83</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21031395">21031395</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perugino</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vettone</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Illiano</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Activity and regulation of archaeal DNA alkyltransferase: conserved protein involved in repair of DNA alkylation damage</span>. <span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">287</span>:<span class="fpage" tagx="fpage" title="fpage">4222</span>â€"<span class="lpage" tagx="lpage" title="lpage">31</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22167184">22167184</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perugino</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miggiano</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serpe</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure-function relationships governing activity and stability of a DNA alkylation damage repair thermostable protein</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">43</span>:<span class="fpage" tagx="fpage" title="fpage">8801</span>â€"<span class="lpage" tagx="lpage" title="lpage">16</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26227971">26227971</a></span></span></li> <li tag="ref"><a name="CIT0024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vettone</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serpe</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hidalgo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A novel thermostable protein-tag: optimization of the <i>Sulfolobus solfataricus DNA- alkyl-transferase by protein engineering</i></span>. <span class="source" tagx="source" title="source">Extremophiles</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">20</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">13</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26499124">26499124</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Visone</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perugino</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title"><i>In vivo</i> and <i>inÂ vitro</i> protein imaging in thermophilic archaea by exploiting a novel protein tag</span>. <span class="source" tagx="source" title="source">PloS One</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">12</span>:<span class="fpage" tagx="fpage" title="fpage">e0185791</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28973046">28973046</a></span></span></li> <li tag="ref"><a name="CIT0026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo Gullo</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mattossovich</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perugino</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Optimization of an <i>inÂ vitro</i> transcription/translation system based on Sulfolobus solfataricus cell lysate</span>. <span class="source" tagx="source" title="source">Archaea</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">2019</span>:<span class="fpage" tagx="fpage" title="fpage">9848253</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30886540">30886540</a></span></span></li> <li tag="ref"><a name="CIT0027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merlo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Del Prete</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valenti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">An AGT-based protein-tag system for the labelling and surface immobilization of enzymes on <i>E. coli</i> outer membrane</span>. <span class="source" tagx="source" title="source">J Enzyme Inhib Med Chem</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">34</span>:<span class="fpage" tagx="fpage" title="fpage">490</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30724623">30724623</a></span></span></li> <li tag="ref"><a name="CIT0028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Del Prete</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merlo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valenti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Thermostability enhancement of the Î±-carbonic anhydrase from <i>Sulfurihydrogenibium yellowstonense</i> by using the anchoring-and-self-labelling-protein-tag system (ASL<sup>tag</sup>)</span>. <span class="source" tagx="source" title="source">J Enzyme Inhib Med Chem</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">34</span>:<span class="fpage" tagx="fpage" title="fpage">946</span>â€"<span class="lpage" tagx="lpage" title="lpage">54</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31039618">31039618</a></span></span></li> <li tag="ref"><a name="CIT0029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moschel</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McDougall</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dolan</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structural features of substituted purine derivatives compatible with depletion of human <i>O6</i>-alkylguanine-DNA alkyltransferase</span>. <span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">1992</span>;<span class="volume" tagx="volume" title="volume">35</span>:<span class="fpage" tagx="fpage" title="fpage">4486</span>â€"<span class="lpage" tagx="lpage" title="lpage">91</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1447749">1447749</a></span></span></li> <li tag="ref"><a name="CIT0030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chae</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McDougall</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dolan</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Substituted <i>O6</i>-benzylguanine derivatives and their inactivation of human <i>O6</i>-alkylguanine-DNA alkyltransferase</span>. <span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">1994</span>;<span class="volume" tagx="volume" title="volume">37</span>:<span class="fpage" tagx="fpage" title="fpage">342</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8308861">8308861</a></span></span></li> <li tag="ref"><a name="CIT0031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Terashima</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawate</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakumi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Substrate specificity of human <i>O6</i>-methylguanine-DNA methyltransferase for <i>O6</i>-benzylguanine derivatives in oligodeoxynucleotides</span>. <span class="source" tagx="source" title="source">Chem Res Toxicol</span><span class="year" tagx="year" title="year">1997</span>;<span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">1234</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9403175">9403175</a></span></span></li> <li tag="ref"><a name="CIT0032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McElhinney</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donnelly</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCormick</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inactivation of <i>O6</i>-alkylguanine-DNA alkyltransferase. 1. Novel O6-(hetarylmethyl)guanines having basic rings in the side chain</span>. <span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">41</span>:<span class="fpage" tagx="fpage" title="fpage">5265</span>â€"<span class="lpage" tagx="lpage" title="lpage">71</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9857094">9857094</a></span></span></li> <li tag="ref"><a name="CIT0033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of the structural features of guanine derivatives as MGMT inhibitors using 3D-QSAR modeling combined with molecular docking</span>. <span class="source" tagx="source" title="source">Molecules</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">21</span>:<span class="fpage" tagx="fpage" title="fpage">823</span>â€"<span class="lpage" tagx="lpage" title="lpage">44</span>.</span></li> <li tag="ref"><a name="CIT0034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">GYJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">One- and two-photon live cell imaging using a mutant SNAP-Tag protein and its FRET substrate pairs</span>. <span class="source" tagx="source" title="source">Org Lett</span><span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">4160</span>â€"<span class="lpage" tagx="lpage" title="lpage">3</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21786765">21786765</a></span></span></li> <li tag="ref"><a name="CIT0035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis and biological evaluation of salinomycin triazole analogues as anticancer agents</span>. <span class="source" tagx="source" title="source">Eur J Med Chem</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">127</span>:<span class="fpage" tagx="fpage" title="fpage">900</span>â€"<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27876192">27876192</a></span></span></li> <li tag="ref"><a name="CIT0036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leng</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiao</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miao</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A wash-free SNAP-tag fluorogenic probe based on the additive effects of quencher release and environmental sensitivity</span>. <span class="source" tagx="source" title="source">Chem Commun</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">53</span>:<span class="fpage" tagx="fpage" title="fpage">6448</span>â€"<span class="lpage" tagx="lpage" title="lpage">51</span>.</span></li> <li tag="ref"><a name="CIT0037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kindermann</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">George</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnsson</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnsson</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Covalent and selective immobilization of fusion proteins</span>. <span class="source" tagx="source" title="source">J Am Chem Soc</span><span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">125</span>:<span class="fpage" tagx="fpage" title="fpage">7810</span>â€"<span class="lpage" tagx="lpage" title="lpage">1</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12822993">12822993</a></span></span></li> <li tag="ref"><a name="CIT0038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mattossovich</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merlo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fontana</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A journey down to hell: new thermostable protein-tags for biotechnology at high temperatures</span>. <span class="source" tagx="source" title="source">Extremophiles</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">24</span>:<span class="fpage" tagx="fpage" title="fpage">81</span>â€"<span class="lpage" tagx="lpage" title="lpage">91</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31555904">31555904</a></span></span></li> <li tag="ref"><a name="CIT0039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrone</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Massarotti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Crystal structure of a thermophilic <i>O6</i>-alkylguanine-DNA alkyltransferase-derived self-labeling protein-tag in covalent complex with a fluorescent probe</span>. <span class="source" tagx="source" title="source">Biochem Biophys Res Commun</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">500</span>:<span class="fpage" tagx="fpage" title="fpage">698</span>â€"<span class="lpage" tagx="lpage" title="lpage">703</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29684348">29684348</a></span></span></li> <li tag="ref"><a name="CIT0040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banks</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beard</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Integrated Modeling Program, Applied Chemical Theory (IMPACT)</span>. <span class="source" tagx="source" title="source">J Comput Chem</span><span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">26</span>:<span class="fpage" tagx="fpage" title="fpage">1752</span>â€"<span class="lpage" tagx="lpage" title="lpage">80</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16211539">16211539</a></span></span></li> <li tag="ref"><a name="CIT0041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toukmaji</span><span class="given-names" tagx="given-names" title="given-names">AY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Board</span><span class="given-names" tagx="given-names" title="given-names">JA.</span><span class="suffix" tagx="suffix" title="suffix">Jr.</span></span></span>, <span class="mixed-article-title" title="mixed-article-title">Ewald summation techniques in perspective: a survey</span>. <span class="source" tagx="source" title="source">Comput Phys Commun</span><span class="year" tagx="year" title="year">1996</span>;<span class="volume" tagx="volume" title="volume">95</span>:<span class="fpage" tagx="fpage" title="fpage">73</span>â€"<span class="lpage" tagx="lpage" title="lpage">92</span>.</span></li> <li tag="ref"><a name="CIT0042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zielkiewicz</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural properties of water: comparison of the SPC, SPCE, TIP4P, and TIP5P models of water</span>. <span class="source" tagx="source" title="source">J Chem Phys</span><span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">123</span>:<span class="fpage" tagx="fpage" title="fpage">104501</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16178604">16178604</a></span></span></li> <li tag="ref"><a name="CIT0043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martyna</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klein</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuckerman</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Reversible multiple time scale molecular dynamics</span>. <span class="source" tagx="source" title="source">J Chem Phys</span><span class="year" tagx="year" title="year">1992</span>;<span class="volume" tagx="volume" title="volume">97</span>:<span class="fpage" tagx="fpage" title="fpage">2635</span>â€"<span class="lpage" tagx="lpage" title="lpage">43</span>.</span></li> <li tag="ref"><a name="CIT0044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evans</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holian</span><span class="given-names" tagx="given-names" title="given-names">BL.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Noseâ€"Hoover thermostat</span>. <span class="source" tagx="source" title="source">J Chem Phys</span><span class="year" tagx="year" title="year">1985</span>;<span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">4069</span>.</span></li> <li tag="ref"><a name="CIT0045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miggiano</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perugino</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciaramella</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Crystal structure of <i>Mycobacterium tuberculosis O6</i>-methylguanine-DNA methyltransferase protein clusters assembled on to damaged DNA</span>. <span class="source" tagx="source" title="source">Biochem J</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">473</span>:<span class="fpage" tagx="fpage" title="fpage">123</span>â€"<span class="lpage" tagx="lpage" title="lpage">33</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26512127">26512127</a></span></span></li> <li tag="ref"><a name="CIT0046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrone</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miggiano</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serpe</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Interdomain interactions rearrangements control the reaction steps of a thermostable DNA alkyltransferase</span>. <span class="source" tagx="source" title="source">Biochim Biophys Acta Gen Subj</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">1861</span>:<span class="fpage" tagx="fpage" title="fpage">86</span>â€"<span class="lpage" tagx="lpage" title="lpage">96</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27777086">27777086</a></span></span></li> <li tag="ref"><a name="CIT0047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middleton</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bridgewater</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Lomeguatrib, a potent inhibitor of <i>O6</i>-alkylguanine-DNA-alkyltransferase: phase I safety, pharmacodynamic, and pharmacokinetic trial and evaluation in combination with temozolomide in patients with advanced solid tumors</span>. <span class="source" tagx="source" title="source">Clin Cancer Res</span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">12</span>:<span class="fpage" tagx="fpage" title="fpage">1577</span>â€"<span class="lpage" tagx="lpage" title="lpage">84</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16533784">16533784</a></span></span></li> <li tag="ref"><a name="CIT0048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colombo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazzucchelli</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montenegro</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Protein oriented ligation on nanoparticles exploiting <i>O6</i>-alkylguanine-DNA transferase (SNAP) genetically encoded fusion</span>. <span class="source" tagx="source" title="source">Small</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">1492</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22431243">22431243</a></span></span></li> <li tag="ref"><a name="CIT0049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Terminal alkyne substituted <i>O6</i>-benzylguanine for versatile and effective syntheses of fluorescent labels to genetically encoded SNAP-tags</span>. <span class="source" tagx="source" title="source">RSC Adv</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">23646</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.</span></li> <li tag="ref"><a name="CIT0050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferraris</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spallek</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oehlmann</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure of <i>Thermococcus litoralis Î"</i><sup>1</sup>â€�pyrrolineâ€�2â€�carboxylate reductase in complex with NADH and Lâ€�proline</span>. <span class="source" tagx="source" title="source">Proteins</span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">82</span>:<span class="fpage" tagx="fpage" title="fpage">2268</span>â€"<span class="lpage" tagx="lpage" title="lpage">74</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24615888">24615888</a></span></span></li> <li tag="ref"><a name="CIT0051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perspicace</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kohler</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SPR-based interaction studies with small molecular weight ligands using hAGT fusion proteins</span>. <span class="source" tagx="source" title="source">Anal Biochem</span><span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">333</span>:<span class="fpage" tagx="fpage" title="fpage">280</span>â€"<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15450803">15450803</a></span></span></li> <li tag="ref"><a name="CIT0052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niesen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sack</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seidel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SNAP-Tag technology: a useful tool to determine affinity constants and other functional parameters of novel antibody fragments</span>. <span class="source" tagx="source" title="source">Bioconjug Chem</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">1931</span>â€"<span class="lpage" tagx="lpage" title="lpage">41</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27391930">27391930</a></span></span></li> <li tag="ref"><a name="CIT0053" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Testa</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wurm</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medda</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Multicolor fluorescence nanoscopy in fixed and living cells by exciting conventional fluorophores with a single wavelength</span>. <span class="source" tagx="source" title="source">Biophys J</span><span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">99</span>:<span class="fpage" tagx="fpage" title="fpage">2686</span>â€"<span class="lpage" tagx="lpage" title="lpage">94</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20959110">20959110</a></span></span></li> <li tag="ref"><a name="CIT0054" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miggiano</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valenti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Every OGT is illuminated â€¦ by fluorescent and synchrotron lights</span>. <span class="source" tagx="source" title="source">Int J Mol Sci</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">18</span>:<span class="fpage" tagx="fpage" title="fpage">2613</span>â€"<span class="lpage" tagx="lpage" title="lpage">31</span>.</span></li> <li tag="ref"><a name="CIT0055" /><span class="label" tagx="label" title="label">55</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zweerink</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kallnik</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ninck</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Activity-based protein profiling as a robust method for enzyme identification and screening in extremophilic Archaea</span>. <span class="source" tagx="source" title="source">Nat Commun</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">15352</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28480883">28480883</a></span></span></li> </ul> </div> </div>  </body></html>