<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Plant Commun</span><span class="iso-abbrev" title="iso-abbrev">Plant Commun</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Plant Communications</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2590-3462</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7747965">7747965</a></span><span class="pii" title="pii">S2590-3462(20)30149-8</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/j.xplc.2020.100118">10.1016/j.xplc.2020.100118</a></span><span class="publisher-id" title="publisher-id">100118</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Editorial</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Plant Evolutionary Adaptation</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rieseberg</span><span class="given-names" tagx="given-names" title="given-names">Loren H.</span></span><span class="email" tagx="email" title="email">lriesebe@mail.ubc.ca</span><a href="#aff1">1</a><a href="#cor1">∗</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">Lexuan</span></span><a href="#aff2">2</a></span><span class="citation_author_institution" id="aff1">[1], </span><span class="citation_author_institution" id="aff2">[2], </span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">∗</span>Corresponding author <span class="email" tagx="email" title="email">lriesebe@mail.ubc.ca</span></div> </div><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-10-10</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-11-11</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-10-10</span></span><span class="volume" tagx="volume" title="volume">1</span><span class="issue" tagx="issue" title="issue">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">100118</span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).</span></span></div> </div> </div> <div class="body" title="body"> <p id="p0010">In 1973, evolutionary biologist Theodosius Dobzhansky penned a famous essay titled "Nothing in Biology Makes Sense Except in the Light of Evolution" (<a href="#bib4">Dobzhansky, 1973</a>). In the essay, he described how many features of biology, such as the adaptive radiations of <i>Drosophila</i> fruit flies and the similarity of embryo development across animal species, are best explained by evolution. Although Dobzhansky's examples largely came from the animal kingdom, evolution is also the chief unifying theory in plant biology, linking molecular, cell, and whole organism-level phenomena. <i>Plant Communications</i> is intended to represent all of the plant sciences, and evolution can provide connections between the diverse studies published in the journal. This is illustrated by this special issue on <i>Plant Evolutionary Adaptation</i>, in which evolutionary approaches are employed to explain phenomena ranging from the geographic distribution of genomic variation to patterns of plant dispersal to the success of invasive species. Here we provide background for the topics addressed by these studies and highlight their main conclusions, moving from molecular to organismal-level studies.</p> <p id="p0015">Early studies of molecular adaptation in plants (and animals) focused on changes in amino acid sequences of proteins. Only with the genomic revolution in the 21st century did it become clear that variation in gene number plays a profoundly important role in plant adaptation as well. However, the focus to date has mainly been on gene duplications rather than gene losses or pseudogene formation. Studies of plant domestication have been an exception to this rule, and loss-of-function (LOF) mutations are known to underlie a number of key domestication phenotypes. Whether such mutations also contribute importantly to adaptation in natural populations is less clear, since LOF mutations in domesticates are established by artificial rather than natural selection. In this issue, <a href="#bib10">Xu and Guo (2020)</a> clarify this topic by reviewing what we know about LOF mutations. They describe the evolutionary events that lead to LOF mutations and show that such mutations are abundant in natural populations. Whereas most LOF mutations are likely neutral or deleterious, the authors describe a number of cases in which such mutations are advantageous, and evidence suggests that these examples represent the "tip of the iceberg." The authors further note that LOF mutations represent a valuable resource for functional studies of genes in non-model organisms, as well as for crop improvement.</p> <p id="p0020">Several studies employ evolutionary theory to understand patterns of sequence and structural variation within and between species. At the most basic level, such patterns will depend on the action of evolutionary forces such as mutation, genetic drift, gene flow, and recombination. Evolutionist biologists are particularly interested in identifying the footprints of natural selection. This is surprisingly challenging because demographic processes such as population bottlenecks and population expansions can create extreme drift, leading to patterns of genomic variation that are similar to those caused by positive selection. An especially convincing sign of natural selection is the repetition of evolutionary changes in similar environments (i.e., convergent evolution), as such a pattern is unlikely to result from drift. In this issue, <a href="#bib9">Xu et al. (2020)</a> explore evidence for convergent genomic evolution due to repeated adaptation to extreme environments. They point out that convergence can occur at different levels, from individual sites to genes to genetic pathways to genome-wide changes such as amino acid usage and GC content. They report that convergent evolution at individual sites is rare and prone to false positives, but that multiple forms of convergence at higher genetic levels are common. For example, mangroves exhibit repeated changes in amino acid composition, as well as convergent reductions in transposable element content and genome size. The authors note that with the rapid accumulation of genome sequence data for extremophile taxa, tests for convergent changes will become increasingly powerful.</p> <p id="p0025">Selection acting on the sporophytic stage of the seed plant life cycle is straightforward to envision and has been the main focus of plant evolutionary biologists. However, it is important to keep in mind that selection can also act on the gametophytic stage. Furthermore, as pointed out in a review of the genomics of gametophytic selection (<a href="#bib1">Beaudry et al., 2020</a>) in this issue, gametophytes are haploid and lack heterozygosity, so selection will act more efficiently on both deleterious and dominant mutations than in diploid sporophytes. Beaudry et al. describe the different ways gametophytes may experience selection, including competition among male gametophytes (i.e., pollen competition), as well as sexual conflict, coevolution, and pleiotropy between male and female gametophytes and the sporophyte. The authors compile evidence from experimental and expression studies, which supports the likely importance of these different kinds of selection on gene, genome, and phenotypic evolution. They also provide useful suggestions for future studies, noting that population genomic contrasts involving different mating systems, ploidy levels, and sex chromosomes offer a means of assessing the strength and evolutionary importance of gametophytic selection.</p> <p id="p0030">Whereas the impact of selection on genomic variation has arguably attracted the greatest attention from evolutionary biologists, the effects of genetic drift are a close second in attention received. This is because of the influence of neutral theory, which holds that most molecular variants are selectively neutral and their distribution within and between species is governed by random genetic drift rather than selection. As a consequence, there is considerable interest in how demographic processes such as population bottlenecks, population expansions, admixture, and mating system affect the strength of genetic drift and efficacy of selection. However, it can be challenging to disentangle the role of these different processes. Using resequencing data from <i>Arabis lyrata</i> (a close relative of <i>A. thaliana</i>), <a href="#bib8">Willi et al. (2020, this issue)</a> were able to exploit variation in mating system and in the geographic distribution of populations following postglacial expansion to assess the impact of these different processes on levels of genetic diversity and signatures of positive selection. They found that, as predicted by theory, genetic diversity was significantly reduced in selfing populations and populations furthest from glacial refugia. Also, evidence of positive selection increased with admixture, which might imply that genetic variation is limiting rates of evolution in non-admixed populations. Overall, these demographic processes accounted for approximately 60% of variation in genomic diversity and &amp;gt;50% of variation in signatures of positive selection.</p> <p id="p0035">A third and much more neglected evolutionary process that also influences the distribution of genetic diversity is gene flow. In outcrossing species, gene flow requires both the dispersal of individuals (gametophytes or sporophytes) to a different population and successful mating with resident individuals. Levels of gene flow can be affected by geographic distance, as well as by features of the landscape. Landscape genetics refers to studies that combine population genetics with landscape ecology. In this issue, <a href="#bib3">Cruzan and Hendrickson (2020)</a> describe strategies for incorporating the separate contributions of pollen (male gametophytes) and seed (sporophytes) dispersal, as well as plant habitat quality, into landscape genetic studies. Application of their approach to the analysis of seed dispersal in three plant species revealed that landscape resistance explains patterns of gene flow better than geographic distance. Counterintuitively, rates of gene flow were lower in high-quality than in low-quality habitat, implying that much remains to be understood about dispersal processes.</p> <p id="p0040">Two studies in this issue examine the evolution of polyploid species. Polyploidy or whole-genome duplication is a favorite subject of plant evolutionary biologists because it represents an extreme macromutation with profound ecological and evolutionary consequences. Polyploids in which duplicated genomes derive from the same species are called autopolyploids, whereas polyploids with duplicated genomes from different species are termed allopolyploids. Whereas autopolyploids arise more frequently than allopolyploids, the latter are more likely to become established due to hybrid vigor, although other factors likely contribute as well. <a href="#bib6">Meeus et al. (2020)</a> use natural and synthetic allopolyploids in <i>Mimulus</i> monkeyflowers to explore the evolutionary consequences of allopolyploidy. They report that whole-genome duplication does result in the recovery of hybrid fertility and in the generation of a reproductive barrier with the parental species (i.e., speciation). The polyploids also exhibit predicted increases in size and development time. Interestingly, the natural allopolyploid outperforms the synthetic polyploid and parental species in the field, suggestive of an important role for adaptive evolution in successful polyploid establishment.</p> <p id="p0045">The second study of polyploidy takes a closer look at the origin and evolution of three tetraploid species of eyebrights (<i>Euphrasia</i>), which are semi-parasites of grass species (<a href="#bib2">Becher et al., 2020</a>). The authors employ a common garden study to show that the phenotypic differences defining the tetraploid species are genetically controlled. They also generate whole-genome sequence data for the three species, which they use to demonstrate that the tetraploids are of hybrid origin (i.e., allopolyploidy). The species are not strongly reproductively isolated, and heterogeneous patterns of genomic differentiation combined with significant deviations in Patterson's <i>D</i> statistic are consistent with a reticulate evolutionary history. The authors conclude that the exchange of ancient variation through introgression may contribute to adaptive divergence and reproductive isolation in this group.</p> <p id="p0050">Invasive species are another favorite topic of evolutionary biologists because they represent evolutionary experiments taking place on ecological timescales. A common observation is that plants from the introduced range of a species (i.e., invasive populations) are larger, grow faster, and are more fecund than plants from the native range. A widely held explanation for this pattern, as well as for the overall success of the invaders, is that invasive plants trade off tolerance to abiotic and biotic stresses for increased performance. In this issue, <a href="#bib5">Hodgins et al. (2020)</a> investigate this hypothesis in the noxious weed, Canada thistle (<i>Cirsium arvense</i>), which is native to Eurasia, but has invaded all other continents except Antarctica. Using a common garden approach, the performance of native European and introduced North American populations was tested under five different stresses, as well under benign conditions. Although the invaders did outperform native populations as predicted, the study failed to find evidence of trade-offs, despite the large number of stresses that were tested.</p> <p id="p0055">The final paper in the special issue investigates the evolution of a mutualism using the classic model legume–rhizobia system. Mutualisms refer to interactions between two (or more) species in which both species benefit. Evolutionary theory predicts that traits that enhance mutualisms will be favored by selection, reducing variation in those traits. Nonetheless, studies of mutualisms often report considerable variation in fitness-enhancing traits. A possible solution to this conundrum is that genotype-by-environment (G × E) interactions are common in mutualisms. Under this assumption, genotypes that perform well in one environment do poorly in another, ensuring that variation is maintained. <a href="#bib7">Vaidya and Stinchcombe (2020)</a> tested this hypothesis by examining the performance of 50 legume genotypes in different light environments. Changes in genotype fitness rank were observed across light environments, confirming that, as predicted, G × E is responsible for the maintenance of variation in mutualism-enhancing traits in this system.</p> <p id="p0060">In conclusion, this special issue illustrates the explanatory power of evolution and how it can serve to unify disparate topics in plant biology. We hope that <i>Plant Communications</i> will continue to attract high-quality papers in plant evolutionary biology, as well as papers that employ an evolutionary approach to tackle other critical issues in the plant sciences.</p> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beaudry</span><span class="given-names" tagx="given-names" title="given-names">F.E.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rifkin</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">S.C.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">S.I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary genomics of plant gametophytic selection</span><span class="source" tagx="source" title="source">Plant Commun.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.xplc.2020.100115">10.1016/j.xplc.2020.100115</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Becher</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">M.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powell</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metherell</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riddiford</span><span class="given-names" tagx="given-names" title="given-names">N.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Twyford</span><span class="given-names" tagx="given-names" title="given-names">A.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Maintenance of species differences in closely related tetraploid parasitic <i>Euphrasia</i> (Orobanchaceae) on an isolated island</span><span class="source" tagx="source" title="source">Plant Commun.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.xplc.2020.100105">10.1016/j.xplc.2020.100105</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cruzan</span><span class="given-names" tagx="given-names" title="given-names">M.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hendrickson</span><span class="given-names" tagx="given-names" title="given-names">E.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Landscape genetics of plants: challenges and opportunities</span><span class="source" tagx="source" title="source">Plant Commun.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.xplc.2020.100100">10.1016/j.xplc.2020.100100</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobzhansky</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nothing in biology makes sense except in the light of evolution</span><span class="source" tagx="source" title="source">Am. Biol. Teach.</span><span class="volume" tagx="volume" title="volume">35</span><span class="year" tagx="year" title="year">1973</span><span class="fpage" tagx="fpage" title="fpage">125</span><span class="lpage" tagx="lpage" title="lpage">129</span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodgins</span><span class="given-names" tagx="given-names" title="given-names">K.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guggisberg</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nurkowski</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rieseberg</span><span class="given-names" tagx="given-names" title="given-names">L.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetically based trait differentiation but lack of trade-offs between stress tolerance and performance in introduced Canada thistle</span><span class="source" tagx="source" title="source">Plant Commun.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.xplc.2020.100116">10.1016/j.xplc.2020.100116</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meeus</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Šemberová</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Storme</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geelen</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vallejo-Marín</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of whole-genome duplication on the evolutionary rescue of sterile hybrid monkeyflowers</span><span class="source" tagx="source" title="source">Plant Commun.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.xplc.2020.100093">10.1016/j.xplc.2020.100093</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaidya</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stinchcombe</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The potential for genotype-by-environment interactions to maintain genetic variation in a model legume-rhizobia mutualism</span><span class="source" tagx="source" title="source">Plant Commun.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.xplc.2020.100114">10.1016/j.xplc.2020.100114</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Willi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fracassetti</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bachmann</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Buskirk</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Demographic processes linked to genetic diversity and positive selection across a species' range</span><span class="source" tagx="source" title="source">Plant Commun.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.xplc.2020.100111">10.1016/j.xplc.2020.100111</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic convergence in the adaptation to extreme environments</span><span class="source" tagx="source" title="source">Plant Commun.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.xplc.2020.100117">10.1016/j.xplc.2020.100117</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y.-L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Less is more, natural loss-of-function mutation is a strategy for adaptation</span><span class="source" tagx="source" title="source">Plant Commun.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.xplc.2020.100103">10.1016/j.xplc.2020.100103</a></span></span></li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p id="p0065">Richard Abbott, Nolan Kane, Sophie Karrenberg, Daniel Ortiz-Barrientos, and Suhua Shi were members of the special issue organizing committee. We thank them for their contributions. No conflict of interest declared.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p id="ntpara0010">Published by the Plant Communications Shanghai Editorial Office in association with Cell Press, an imprint of Elsevier Inc., on behalf of CSPB and IPPE, CAS.</p> </div> </div> </div>  </body></html>