<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7809279">7809279</a></span><span class="publisher-id" title="publisher-id">81087</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-021-81087-w">10.1038/s41598-021-81087-w</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
The specific DNA barcodes based on chloroplast genes for species identification of <i>Orchidaceae</i> plants</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Huili</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">Wenjun</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">Tie</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Yongliang</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Meng</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Xiaoxia</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">Xiaoxiao</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Qun</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Xinhong</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">gxh@hnu.edu.cn</span></span><a href="#Aff1" /></span><span class="citation_author_institution" id="Aff1">[], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.67293.39</span><span class="institution" title="institution">College of Biology, </span><span class="institution" title="institution">Hunan University, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-1-1</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1424</span><span class="history" title="history"><span class="received" title="received">received: 2019-12-17</span><span class="accepted" title="accepted">accepted: 2021-1-4</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">DNA barcoding is currently an effective and widely used tool that enables rapid and accurate identification of plant species. The <i>Orchidaceae</i> is the second largest family of flowering plants, with more than 700 genera and 20,000 species distributed nearly worldwide. The accurate identification of <i>Orchids</i> not only contributes to the safe utilization of these plants, but also it is essential to the protection and utilization of germplasm resources. In this study, the DNA barcoding of 4 chloroplast genes (<i>matK, rbcL, ndhF</i> and <i>ycf1</i>) were used to provide theoretical basis for species identification, germplasm conservation and innovative utilization of <i>orchids</i>. By comparing the nucleotide replacement saturation of the single or combined sequences among the 4 genes, we found that these sequences reached a saturation state and were suitable for phylogenetic relationship analysis. The phylogenetic analyses based on genetic distance indicated that <i>ndhF</i> and <i>ycf1</i> sequences were competent to identification at genus and species level of <i>orchids</i> in a single gene. In the combined sequences, <i>matK</i>â€‰+â€‰<i>ycf1</i> and <i>ndhF</i>â€‰+â€‰<i>ycf1</i> were qualified for identification at the genera and species levels, suggesting the potential roles of <i>ndhF, ycf1</i>, <i>matK</i>â€‰+â€‰<i>ycf1</i> and <i>ndhF</i>â€‰+â€‰<i>ycf1</i> as candidate barcodes for <i>orchids</i>. Based on the SNP sites, candidate genes were used to obtain the specific barcode of <i>orchid</i> plant species and generated the corresponding DNA QR code ID card that could be immediately recognized by electronic devices. This study provides innovative research methods for efficient species identification of <i>orchids</i>. The standardized and accurate barcode information of <i>Orchids</i> is provided for researchers. It lays the foundation for the conservation, evaluation, innovative utilization and protection of <i>Orchidaceae</i> germplasm resources.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Computational biology and bioinformatics</div> <div class="kwd" title="kwd">
Evolution</div> <div class="kwd" title="kwd">
Genetics</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Key Research and Development Program</span></div><span class="award-id" tagx="award-id" title="award-id">2017YFF0210301</span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Natural Science Foundation of China</span></div><span class="award-id" tagx="award-id" title="award-id">31872866</span><span class="award-id" tagx="award-id" title="award-id">31540064</span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2"><i>Orchidaceae</i> is the second largest family after <i>Composite</i>, and the largest family of monocotyledonous plants<sup><a href="#CR1">1</a>â€"<a href="#CR3">3</a></sup>. More than 700 genera and more than 20,000 species were identified in the family <i>Orchidaceae</i>, which account for 8 percent of all flowering plants<sup><a href="#CR1">1</a>,<a href="#CR2">2</a>,<a href="#CR4">4</a>â€"<a href="#CR7">7</a></sup>. <i>Orchids</i> mainly distribute in the tropical and subtropical regions of the world, and a few species grow in the temperate regions<sup><a href="#CR2">2</a>â€"<a href="#CR4">4</a>,<a href="#CR8">8</a>,<a href="#CR9">9</a></sup>.</p> <p id="Par3">The <i>Orchidaceae</i> plants exhibit important ornamental, medicinal, research and ecological value<sup><a href="#CR2">2</a>,<a href="#CR10">10</a>â€"<a href="#CR12">12</a></sup>. Many <i>Orchidaceae</i> plants with beautiful flowers and rich fragrance are ornamental plants, such as <i>Cymbidium</i>, <i>Phalaenopsis</i>, <i>Cypripedium</i><sup><a href="#CR2">2</a>,<a href="#CR12">12</a>â€"<a href="#CR14">14</a></sup>. Numerous species containing active ingredients, like polysaccharides, alkaloids, phenanthrene and dibenzyl also are served as traditional herbal medicines for treatment of the diseases<sup><a href="#CR2">2</a>,<a href="#CR7">7</a>,<a href="#CR10">10</a>,<a href="#CR12">12</a>,<a href="#CR15">15</a></sup>. These traits that is able to bring great economic benefits make <i>Orchidaceae</i> plants on raising market demand. In the past decades, over-exploitation and habitat destruction by humans caused serious extinction threats to a large number of <i>Orchidaceae</i> plants<sup><a href="#CR2">2</a>,<a href="#CR10">10</a>,<a href="#CR15">15</a></sup>. Additionally, more and more counterfeit and shoddy <i>Orchidaceae</i>-related products emerge. This is not only likely to threaten drug safety, but also caused damage to biodiversity<sup><a href="#CR2">2</a>,<a href="#CR7">7</a>,<a href="#CR11">11</a>,<a href="#CR12">12</a></sup>.</p> <p id="Par4">Given that, the accurate identification of <i>Orchidaceae</i> plants is of great significance for their safe utilization, biodiversity and the protection of genetic resources<sup><a href="#CR2">2</a>,<a href="#CR7">7</a>,<a href="#CR12">12</a>,<a href="#CR16">16</a>,<a href="#CR17">17</a></sup>. It is known that traditional identification methods are based on morphological features. Some <i>Orchidaceae</i> plants, however, alomst exhibit no morphological differences before flowering, and the orphological features are susceptible to environmental factors<sup><a href="#CR2">2</a>,<a href="#CR7">7</a>, <a href="#CR16">16</a>,<a href="#CR18">18</a></sup>. In addition, there are fewer and fewer experienced experts in morphological identification<sup><a href="#CR2">2</a>,<a href="#CR7">7</a>,<a href="#CR12">12</a>,<a href="#CR16">16</a>,<a href="#CR18">18</a>â€"<a href="#CR20">20</a></sup>. Totally, this makes the accurate identification to be a time consuming and labor intensive job. Therefore, we are badly in need of a rapid, accessible and accurate identification method.</p> <p id="Par5">The DNA barcode technology is a novel molecular recognition technology that uses short and standard DNA fragments for species identification<sup><a href="#CR7">7</a>,<a href="#CR16">16</a>,<a href="#CR17">17</a>,<a href="#CR19">19</a>,<a href="#CR21">21</a>,<a href="#CR22">22</a></sup>. DNA barcodes were originally utilized to identify microorganisms<sup><a href="#CR23">23</a></sup>, but now it is able to quickly and accurately identify species at the level of species with unlimited reasons for development stage, internal morphological diversity, environmental factors and user's professional level<sup><a href="#CR2">2</a>,<a href="#CR7">7</a>,<a href="#CR16">16</a>,<a href="#CR18">18</a>,<a href="#CR22">22</a>,<a href="#CR23">23</a></sup>. Thus, the DNA barcoding technology has been rapidly applied in species identification, biosystematics, biodiversity, ecological community evolution, species protection, archaeological sample identification and other aspects<sup><a href="#CR1">1</a>,<a href="#CR7">7</a>,<a href="#CR18">18</a>,<a href="#CR24">24</a>â€"<a href="#CR27">27</a></sup>. Mitochondrial cytochrome oxidase I gene proposed by Hebert et al. in 2003 had been widely used in animal species identification and phylogenetic development<sup><a href="#CR28">28</a>,<a href="#CR29">29</a></sup>. However, due to the low mutation rate of mitochondrial DNA, mitochondrial cytochrome oxidase I can not be used in plants<sup><a href="#CR21">21</a>,<a href="#CR23">23</a>,<a href="#CR30">30</a>,<a href="#CR31">31</a></sup>. In the past decades, many researchers have made great contributions to the search and application of barcode in plants. Subsequently, many scientists performed a great deal of phylogenetic analyses among numerous families or subfamilies of the <i>orchid</i> family based on two plastid genes <i>matK</i> or <i>rbcL</i><sup><a href="#CR24">24</a>,<a href="#CR30">30</a>,<a href="#CR32">32</a>â€"<a href="#CR34">34</a></sup>. Many efforts have been made to discover the core barcodes for different land plant taxa, whereas a consensus has not been reached<sup><a href="#CR35">35</a>,<a href="#CR36">36</a></sup>. After that, CBOL Plant Working Group compared the performance of seven leading candidate plasome DNA regions (<i>atpF</i>-<i>atpH</i> interval, <i>matK</i> gene, <i>rbcL</i> gene, <i>rpoB</i> gene, <i>rpoC1</i> gene, <i>psbK-psbI</i> interval and <i>trnH-psbA</i> interval) and recommended the 2-site combination of <i>rbcL</i>â€‰+â€‰<i>matK</i> as a plant barcode based on the evaluation of recoverability, sequence quality and species identification level<sup><a href="#CR23">23</a></sup>. The generality of medicinal plants species identification were assessed according to <i>matK</i> and <i>rbcL</i> genes<sup><a href="#CR16">16</a>,<a href="#CR27">27</a>,<a href="#CR37">37</a>â€"<a href="#CR40">40</a></sup>. The molecular taxonomic identification of the <i>Canarian</i> oceanic hotspot was studied based on <i>matK</i>â€‰+â€‰<i>rbcL</i><sup><a href="#CR41">41</a></sup>. Chen et al. found that <i>ycf1</i> showed high identification ability at the species level of rare and protected medicinal plants. The chloroplast gene <i>ndhF</i> was foundÂ to be able to identify 100% solanum species by Zhang et al.<sup><a href="#CR42">42</a>,<a href="#CR43">43</a></sup>. Although DNA barcoding has been widely studied in phylogeny and species identification of <i>Orchids</i>, it has not been reported that DNA barcoding genes can be used to develop specific identification segments of different species<sup><a href="#CR2">2</a>,<a href="#CR7">7</a>,<a href="#CR9">9</a>,<a href="#CR16">16</a>,<a href="#CR17">17</a>,<a href="#CR44">44</a>â€"<a href="#CR48">48</a></sup>.</p> <p id="Par6">Here, we used four chloroplast gene sequences (<i>matK</i>, <i>rbcL, ndhF</i> and <i>ycf1</i>) and three combined sequences including <i>matK</i>â€‰+â€‰<i>rbcL</i>, <i>matK</i>â€‰+â€‰<i>ycf1, ndhF</i>â€‰+â€‰<i>ycf1</i> of <i>Orchidaceae</i> species to develop unique identification fragments of a certain species of <i>Orchidaceae</i> based on phylogenetic analyses and SNP site analyses. Furthermore, the barcode genes were comprehensively analyzed to obtain standard DNA marker fragments of <i>Orchidaceae.</i> Therefore, this study provided a novel approach, based on the SNP barcode, to accurately and rapidly identify <i>Orchidaceae</i> plants. This technology replenishes traditional methods of identification in <i>Orchidaceae</i> plants. This is the first study to report a strategy for developing specific DNA barcodes of <i>Orchidaceae</i> plants, laying the foundation for the conservation, evaluation, innovative utilization and protection of <i>Orchidaceae</i> germplasm resources.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="dnasequencesanalysis" title="sec"> <div class="title" tagx="title" title="title">
DNA sequences analysis</div> <p id="Par7">In this study, the sequences including 3040 <i>matK</i> sequences (307 genera, 1900 species), 641 <i>rbcL</i> sequences (55 genera, 192 species), 225 <i>ndhF</i> sequences (102 species, 29 genera), and 384 <i>ycf1</i> sequences (48 genera, 173 species) of <i>Orchids</i> were obtained from the NCBI Nucleotide database (<a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>) for further analyses.</p> <p id="Par8">After blasting and editing, the consensus length of <i>matK</i>, <i>rbcL, ndhF and ycf1</i> were 2169Â bp, 1524Â bp, 2953Â bp, 8145Â bp respectively, and that of combined sequence including <i>matK</i>â€‰+â€‰<i>rbcL, matK</i>â€‰+â€‰<i>ycf1, ndhF</i>â€‰+â€‰<i>ycf1</i> were 3348Â bp, 9731Â bp, 9701Â bp, respectively.</p> <p id="Par9">The overall mean nucleotide base frequencies observed for candidate nucleotide sequences and the distribution of the four bases of candidate nucleotide sequences at different coding positions of codons were showed in Table <a href="#Tab1">1</a>. The average number of identical pairs (ii) for candidate nucleotide sequences was showed in Table <a href="#Tab2">2</a>. The account of transitional pairs (si) and transversional pairs (sv) of nucleotide sequences was showed in Table <a href="#Tab2">2</a>. The transitional and transversional of bases in the sequences may be related to the species difference. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>The nucleotide base frequencies analysis of candidate nucleotide sequences in <i>Orchidaceae</i> plants.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2">Sequences</th> <th align="left" colspan="11">Base content</th> </tr> <tr> <th align="left">A</th> <th align="left">T</th> <th align="left">C</th> <th align="left">G</th> <th align="left">GC</th> <th align="left">AT-1</th> <th align="left">GC-1</th> <th align="left">AT-2</th> <th align="left">GC-2</th> <th align="left">AT-3</th> <th align="left">GC-3</th> </tr> </thead> <tbody> <tr> <td align="left"><i>matK</i></td> <td char="." align="char">30.8</td> <td char="." align="char">37.8</td> <td char="." align="char">16.4</td> <td char="." align="char">15.0</td> <td char="." align="char">31.4</td> <td char="." align="char">68.5</td> <td char="." align="char">31.5</td> <td char="." align="char">68.9</td> <td char="." align="char">31.2</td> <td char="." align="char">68.3</td> <td char="." align="char">31.7</td> </tr> <tr> <td align="left"><i>rbcL</i></td> <td char="." align="char">28.0</td> <td char="." align="char">29.2</td> <td char="." align="char">18.4</td> <td char="." align="char">24.5</td> <td char="." align="char">42.9</td> <td char="." align="char">62.7</td> <td char="." align="char">37.3</td> <td char="." align="char">53.2</td> <td char="." align="char">46.7</td> <td char="." align="char">55.4</td> <td char="." align="char">44.6</td> </tr> <tr> <td align="left"><i>ndhF</i></td> <td char="." align="char">27.3</td> <td char="." align="char">39.4</td> <td char="." align="char">16.1</td> <td char="." align="char">17.2</td> <td char="." align="char">33.3</td> <td char="." align="char">66.9</td> <td char="." align="char">33.1</td> <td char="." align="char">64.9</td> <td char="." align="char">35.1</td> <td char="." align="char">68.4</td> <td char="." align="char">31.6</td> </tr> <tr> <td align="left"><i>ycf1</i></td> <td char="." align="char">40.4</td> <td char="." align="char">30.0</td> <td char="." align="char">13.9</td> <td char="." align="char">15.7</td> <td char="." align="char">29.6</td> <td char="." align="char">69.3</td> <td char="." align="char">30.8</td> <td char="." align="char">71.8</td> <td char="." align="char">28.2</td> <td char="." align="char">60.2</td> <td char="." align="char">29.8</td> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> <p>The analysis of nucleotide pair frequencies of candidate nucleotide sequences of <i>Orchidaceae</i> plants.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2">Sequence</th> <th align="left" colspan="4">ii</th> <th align="left" colspan="4">si</th> <th align="left" colspan="4">sv</th> <th align="left" colspan="4">R</th> </tr> <tr> <th align="left">Avg</th> <th align="left">1st</th> <th align="left">2nd</th> <th align="left">3rd</th> <th align="left">Avg</th> <th align="left">1st</th> <th align="left">2nd</th> <th align="left">3rd</th> <th align="left">Avg</th> <th align="left">1st</th> <th align="left">2nd</th> <th align="left">3rd</th> <th align="left">Avg</th> <th align="left">1st</th> <th align="left">2nd</th> <th align="left">3rd</th> </tr> </thead> <tbody> <tr> <td align="left"><i>matK</i></td> <td align="left">1268</td> <td align="left">431</td> <td align="left">423</td> <td align="left">414</td> <td align="left">49</td> <td align="left">16</td> <td align="left">15</td> <td align="left">18</td> <td align="left">43</td> <td align="left">13</td> <td align="left">15</td> <td align="left">15</td> <td char="." align="char">1.1</td> <td char="." align="char">1.3</td> <td char="." align="char">1.0</td> <td char="." align="char">1.2</td> </tr> <tr> <td align="left"><i>rbcL</i></td> <td align="left">1378</td> <td align="left">462</td> <td align="left">458</td> <td align="left">458</td> <td align="left">29</td> <td align="left">10</td> <td align="left">10</td> <td align="left">9</td> <td align="left">14</td> <td align="left">4</td> <td align="left">5</td> <td align="left">5</td> <td char="." align="char">2.1</td> <td char="." align="char">2.6</td> <td char="." align="char">1.9</td> <td char="." align="char">1.0</td> </tr> <tr> <td align="left"><i>ndhF</i></td> <td align="left">1234</td> <td align="left">410</td> <td align="left">419</td> <td align="left">404</td> <td align="left">38</td> <td align="left">14</td> <td align="left">12</td> <td align="left">12</td> <td align="left">34</td> <td align="left">11</td> <td align="left">10</td> <td align="left">12</td> <td char="." align="char">1.1</td> <td char="." align="char">1.3</td> <td char="." align="char">1.1</td> <td char="." align="char">0.9</td> </tr> <tr> <td align="left"><i>ycf1</i></td> <td align="left">4521</td> <td align="left">1514</td> <td align="left">1509</td> <td align="left">1498</td> <td align="left">205</td> <td align="left">61</td> <td align="left">72</td> <td align="left">72</td> <td align="left">200</td> <td align="left">68</td> <td align="left">62</td> <td align="left">70</td> <td char="." align="char">1.0</td> <td char="." align="char">0.9</td> <td char="." align="char">1.2</td> <td char="." align="char">1.0</td> </tr> <tr> <td align="left"><i>matK</i>â€‰+â€‰<i>rbcL</i></td> <td align="left">2835</td> <td align="left">953</td> <td align="left">936</td> <td align="left">946</td> <td align="left">73</td> <td align="left">19</td> <td align="left">32</td> <td align="left">22</td> <td align="left">56</td> <td align="left">15</td> <td align="left">21</td> <td align="left">21</td> <td char="." align="char">1.3</td> <td char="." align="char">1.3</td> <td char="." align="char">1.6</td> <td char="." align="char">1.1</td> </tr> <tr> <td align="left"><i>matK</i>â€‰+â€‰<i>ycf1</i></td> <td align="left">6015</td> <td align="left">1990</td> <td align="left">2008</td> <td align="left">2017</td> <td align="left">247</td> <td align="left">94</td> <td align="left">76</td> <td align="left">77</td> <td align="left">239</td> <td align="left">82</td> <td align="left">81</td> <td align="left">75</td> <td char="." align="char">1.0</td> <td char="." align="char">1.2</td> <td char="." align="char">0.9</td> <td char="." align="char">1.0</td> </tr> <tr> <td align="left"><i>ndhF</i>â€‰+â€‰<i>ycf1</i></td> <td align="left">5718</td> <td align="left">1905</td> <td align="left">1914</td> <td align="left">1899</td> <td align="left">189</td> <td align="left">63</td> <td align="left">61</td> <td align="left">65</td> <td align="left">187</td> <td align="left">61</td> <td align="left">61</td> <td align="left">65</td> <td char="." align="char">1.0</td> <td char="." align="char">1.0</td> <td char="." align="char">1.0</td> <td char="." align="char">1.0</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><i>ii</i> Identical Pairs, <i>si</i> Transitionsal Pairs, <i>sv</i> Transversional Pairs, <i>R</i> si/sv.</p> </div> </div> <p /> <p id="Par10">Polymorphism site analysis of the candidate nucleotide sequences revealed in Table <a href="#Tab3">3</a>. Among the single sequence and the combination sequence <i>rbcL</i> sequence had the least proportion of mutation sites, accounting for 34.8%, while the conservative sites in the corresponding <i>rbcL</i> sequence accounted for 64.7%. The sequence <i>matK</i> had the highest proportion of mutation sites (70.2%), and the corresponding <i>matK</i> sequence had the lowest proportion of conservative sites (18.9%). <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>The analysis of variation of candidate barcode sequences in <i>Orchidaceae</i> plants.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Sequence</th> <th align="left">Conserved site</th> <th align="left">Variable site</th> <th align="left">Parsimony-informative site</th> <th align="left">Sigon site</th> </tr> </thead> <tbody> <tr> <td align="left"><i>matK</i></td> <td align="left">411 (18.9%)</td> <td align="left">1523 (70.2%)</td> <td align="left">1275</td> <td align="left">222</td> </tr> <tr> <td align="left"><i>rbcL</i></td> <td align="left">986 (64.7%)</td> <td align="left">530 (34.8%)</td> <td align="left">504</td> <td align="left">26</td> </tr> <tr> <td align="left"><i>ndhF</i></td> <td align="left">1031 (34.9%)</td> <td align="left">1790 (60.6%)</td> <td align="left">1492</td> <td align="left">297</td> </tr> <tr> <td align="left"><i>ycf1</i></td> <td align="left">3291 (40.4%)</td> <td align="left">4732 (58.1%)</td> <td align="left">4578</td> <td align="left">154</td> </tr> <tr> <td align="left"><i>matK</i>â€‰+â€‰<i>rbcL</i></td> <td align="left">1856 (55.4%)</td> <td align="left">1455 (43.5%)</td> <td align="left">1369</td> <td align="left">86</td> </tr> <tr> <td align="left"><i>matK</i>â€‰+â€‰<i>ndhF</i></td> <td align="left">2017 (44.5%)</td> <td align="left">2377 (52.4%)</td> <td align="left">2027</td> <td align="left">348</td> </tr> <tr> <td align="left"><i>matK</i>â€‰+â€‰<i>ycf1</i></td> <td align="left">3996 (41.1%)</td> <td align="left">5506 (56.6%)</td> <td align="left">5247</td> <td align="left">259</td> </tr> <tr> <td align="left"><i>ndhF</i>â€‰+â€‰<i>ycf1</i></td> <td align="left">4217 (43.5%)</td> <td align="left">5299 (54.6%)</td> <td align="left">4696</td> <td align="left">592</td> </tr> </tbody> </table>  <p /> </div> <div class="geneticdiversity" title="sec"> <div class="title" tagx="title" title="title">
Genetic diversity</div> <p id="Par11">There must be some genetic variation based on their species differences since the data used to analyze were obtained from different species. The basic indicators of genetic diversity, displayed in Table <a href="#Tab4">4</a>, worked out in accordance with pairwise nucleotide differences and nucleotide diversity, and the validity of these indexes were verified by two neutrality tests, like Fuâ€™s <i>Fs</i><sup><a href="#CR49">49</a></sup> and Tajimaâ€™s <i>D</i><sup><a href="#CR50">50</a></sup>. The <i>matK</i>â€‰+â€‰<i>ycf1</i> sequences had revealed maximum genetic diversity cumulatively on the base of Eta value, revealed 2314 mutations within all sequences. While the <i>rbcL</i> sequences only had 322 mutations variations in all sequences. The significance of genetic diversity was verified by both neutrality tests, which confirmed that all sequences had significant difference but no very significant difference based on the probability value (p-value) of Fuâ€™s <i>Fs</i> test and Tajimaâ€™s <i>D</i> test (Table <a href="#Tab4">4</a>). <div class="table-wrap_UNKNOWN" id="Tab4"><span class="label" tagx="label" title="label">Table 4</span> </div></p><p>Genetic diversity caculation of <i>Orchidaceae</i> plants based on candidate barcode sequences by the DnaSP v5 software.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2">Sequences</th> <th align="left" rowspan="2">n</th> <th align="left" colspan="5">Nucleotide diversity</th> <th align="left" rowspan="2">Ï€</th> <th align="left" colspan="4">Neutrality tests</th> </tr> <tr> <th align="left">S</th> <th align="left">k</th> <th align="left">Eta</th> <th align="left">Hd</th> <th align="left">Î¸</th> <th align="left">Fuâ€™s <i>Fs</i></th> <th align="left">p-value</th> <th align="left">D</th> <th align="left">p-value</th> </tr> </thead> <tbody> <tr> <td align="left"><i>matK</i></td> <td align="left">3050</td> <td align="left">1288</td> <td align="left">3.12596</td> <td align="left">1523</td> <td align="left">0.9050</td> <td align="left">0.24339</td> <td align="left">0.07270</td> <td align="left">âˆ’Â 2.57476</td> <td align="left"><div class="â€‰0.05_UNKNOWN" /></td> <td align="left">âˆ’Â 1.84286</td> <td align="left"><div class="â€‰0.05_UNKNOWN" /></td> </tr> <tr> <td align="left"><i>rbcL</i></td> <td align="left">643</td> <td align="left">259</td> <td align="left">15.997</td> <td align="left">322</td> <td align="left">0.9779</td> <td align="left">0.07155</td> <td align="left">0.02503</td> <td align="left">âˆ’Â 0.51015</td> <td align="left">&amp;gt;â€‰0.10</td> <td align="left">âˆ’Â 1.92689</td> <td align="left"><div class="â€‰0.05_UNKNOWN" /></td> </tr> <tr> <td align="left"><i>ndhF</i></td> <td align="left">234</td> <td align="left">233</td> <td align="left">13.952</td> <td align="left">340</td> <td align="left">0.9660</td> <td align="left">0.18546</td> <td align="left">0.04589</td> <td align="left">âˆ’Â 2.96843</td> <td align="left"><div class="â€‰0.05_UNKNOWN" /></td> <td align="left">âˆ’Â 2.37565</td> <td align="left"><div class="â€‰0.01_UNKNOWN" /></td> </tr> <tr> <td align="left"><i>ycf1</i></td> <td align="left">384</td> <td align="left">906</td> <td align="left">95.110</td> <td align="left">1470</td> <td align="left">0.9921</td> <td align="left">0.17379</td> <td align="left">0.07339</td> <td align="left">0.25100</td> <td align="left">&amp;gt;â€‰0.10</td> <td align="left">âˆ’Â 1.78584</td> <td align="left"><div class="â€‰0.05_UNKNOWN" /></td> </tr> <tr> <td align="left"><i>matK</i>â€‰+â€‰<i>rbcL</i></td> <td align="left">372</td> <td align="left">559</td> <td align="left">54.729</td> <td align="left">821</td> <td align="left">0.9924</td> <td align="left">0.12616</td> <td align="left">0.05462</td> <td align="left">âˆ’Â 0.11121</td> <td align="left">&amp;gt;â€‰0.10</td> <td align="left">âˆ’Â 1.75132</td> <td align="left"><div class="â€‰0.05_UNKNOWN" /></td> </tr> <tr> <td align="left"><i>matK</i>â€‰+â€‰<i>ndhF</i></td> <td align="left">216</td> <td align="left">687</td> <td align="left">59.444</td> <td align="left">943</td> <td align="left">0.9853</td> <td align="left">0.12276</td> <td align="left">0.04604</td> <td align="left">âˆ’Â 1.32301</td> <td align="left">&amp;gt;â€‰0.10</td> <td align="left">âˆ’Â 2.00131</td> <td align="left"><div class="â€‰0.05_UNKNOWN" /></td> </tr> <tr> <td align="left"><i>matK</i>â€‰+â€‰<i>ycf1</i></td> <td align="left">378</td> <td align="left">1495</td> <td align="left">150.984</td> <td align="left">2314</td> <td align="left">0.9932</td> <td align="left">0.15463</td> <td align="left">0.06542</td> <td align="left">0.08057</td> <td align="left">&amp;gt;â€‰0.10</td> <td align="left">âˆ’Â 1.79023</td> <td align="left"><div class="â€‰0.05_UNKNOWN" /></td> </tr> <tr> <td align="left"><i>ndhF</i>â€‰+â€‰<i>ycf1</i></td> <td align="left">228</td> <td align="left">494</td> <td align="left">39.916</td> <td align="left">712</td> <td align="left">0.9740</td> <td align="left">0.15585</td> <td align="left">0.05191</td> <td align="left">âˆ’Â 1.47948</td> <td align="left">&amp;gt;â€‰0.10</td> <td align="left">âˆ’Â 2.13392</td> <td align="left"><div class="â€‰0.01_UNKNOWN" /></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><i>Eta</i> Total number of mutations, <i>n</i> number of sequences, <i>k</i> Average number of nucleotide difference, <i>S</i> Number of segregating sites, <i>Î¸</i> nucleotide substitution rate, <i>Ï€</i> nucleotide diversity, <i>Hd</i> haplotype diversity, <i>Fuâ€™s</i><i>Fs</i> is variation among different haplotypes in the population, <i>D</i> is the Tajima test statistic.</p> </div>  <p /> <p id="Par12">Like the neutrality testes of the Tajima test statistic (D value) in the sequences, the genetic variation for <i>ndhF</i> sequences was negatively little higher (âˆ’Â 2.37565) with respect to <i>rbcL</i> sequence, consisting value up to âˆ’Â 0.51015. And for combined sequences, the genetic variation for <i>ndhF</i>â€‰+â€‰<i>ycf1</i> sequences was negatively little higher (âˆ’Â 2.13392) with respect to <i>matK</i>â€‰+â€‰<i>rbcL</i> sequence, consisting value up to âˆ’Â 1.75132. With respect to Fuâ€™s <i>Fs</i> value for sequences variation, the <i>ndhF</i> sequences was higher sequences variation (âˆ’Â 2.96843), shown in Table <a href="#Tab4">4</a>, in comparison with <i>rbcL</i> sequence (âˆ’Â 0.51015). In order to observe nucleotide mismatch distribution among different sequences of <i>Orchidaceae</i> species, DNA sequences were analyzed for population size changes which was enriched the results of genetic diversity among species. All results showed significant genetic variation in <i>Orchidaceae</i> species for candidate nucleotide sequences (Fig.Â <a href="#Fig1">1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Pairwise mismatch distributions, based on <i>matK</i>, <i>rbcL</i>, <i>ndhF</i>, <i>ycf1</i> and the combined sequences by DnaSP v5. Note: The X-axis shows the observed distribution of pairwise genetic variation, and the Y-axis shows the frequency. <i>R</i><sup><i>2</i></sup> Ramos-Onsins and Rozas statistics, <i>r</i> Raggedness statistic, <i>Tau </i>Date of the Growth or Decline measured of mutational time, <i>C.V. </i>Coefficient of variation.</p>   </div> <p /> </div> <div class="phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis</div> <p id="Par13">In this study, we used the MEGA7.0 software based on the Neighbor Joining method and Kimura 2-parameter model to identify <i>rbcL, ndhF</i> and <i>ycf1</i> sequence of the evolutionary tree, and we compressed the same genera or the same subtribes of <i>Orchid</i> with the MEGA 7.0 own Compress Subtree. In the light of the topological structure of the evolutionary tree, species in several subfamilies are not be well identified based on <i>matK</i>, <i>rbcL</i> and the combined sequence <i>matK</i>â€‰+â€‰<i>rbcL</i>. In contrast, the <i>ndhF</i>, <i>ycf1</i> sequences and the combined sequences <i>matK</i>â€‰+â€‰<i>ycf1</i> and <i>ndhF</i>â€‰+â€‰<i>ycf1</i> of chloroplast genes exhibit better identification ability at the generic level (Figs. <a href="#Fig2">2</a>, <a href="#Fig3">3</a>, <a href="#Fig4">4</a>, <a href="#Fig5">5</a>, <a href="#Fig6">6</a>, <a href="#Fig7">7</a>, <a href="#Fig8">8</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>The NJ tree of <i>Orchidaceae</i> coming from analysis of the cp DNA <i>matK</i> sequence based on the K2P model. Names tagged in red indicates the genus, tagged in green showed the subtribe and tagged in blue showed the subfamily; The Numbers on the branches represent more than or equal to 50 percent support after the 1000 bootstrap replications test; Numbers following taxon names showed the number of species.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>The NJ tree of <i>Orchidaceae</i> coming from analysis of the cp DNA <i>rbcL</i> sequence based on the K2P model. Names tagged in red indicates the genus and tagged in green showed the subtribe; The Numbers on the branches represent more than or equal to 50 percent support after the 1000 bootstrap replications test; Numbers following taxon names showed the number of species.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>The NJ tree of <i>Orchidaceae</i> coming from analysis of the cp DNA <i>ndhF</i> sequence based on the K2P model. Names tagged in red indicates the genus and tagged in green showed the subtribe; The Numbers on the branches represent more than or equal to 50 percent support after the 1000 bootstrap replications test; Numbers following taxon names showed the number of species.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>The NJ tree of <i>Orchidaceae</i> coming from analysis of the cp DNA <i>ycf1</i> sequence based on the K2P model. Names tagged in red indicates the genus and tagged in green showed the subtribe; The Numbers on the branches represent more than or equal to 50 percent support after the 1000 bootstrap replications test; Numbers following taxon names showed the number of species.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>The NJ tree of <i>Orchidaceae</i> coming from analysis of the <i>matK</i>â€‰+â€‰<i>rbcL</i> sequence based on the K2P model. Names tagged in red indicates the genus and tagged in green showed the subtribe. The Numbers on the branches represent more than or equal to 50 percent support after the 1000 bootstrap replications test. The Numbers following taxon names showed the number of species.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>The NJ tree of <i>Orchidaceae</i> from analysis of the <i>matK</i>â€‰+â€‰<i>ycf1</i> sequence based on the K2P model. Names tagged in red indicates the genus and tagged in green showed the subtribe; The Numbers on the branches represent more than or equal to 50 percent support after the 1000 bootstrap replications test; Numbers following taxon names showed the number of species.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <p>The NJ tree of <i>Orchidaceae</i> from analysis of the <i>ndhF</i>â€‰+â€‰<i>ycf1</i> sequence based on the K2P model. Names tagged in red indicates the genus and tagged in green showed the subtribe.</p>   </div> <p /> </div> <div class="analysisofbarcodinggap" title="sec"> <div class="title" tagx="title" title="title">
Analysis of barcoding gap</div> <p id="Par14">An ideal DNA barcoding sequence for species identification should satisfy that inter-specific genetic variation is significantly greater than intra-specific genetic variation. In order to more accurately assess individual chloroplast genes and combined sequences in the <i>Orchid</i> genus species, and to verify the applicability of candidate sequences, the barcoding gap was analyzed according to frequency distribution showed in Fig.Â <a href="#Fig9">9</a>. The results revealed that the <i>ndhF</i> gene showed better performance in a single gene, while the combined sequences of <i>ndhF</i>â€‰+â€‰<i>ycf1</i> showed the best performance. The results of the Best Close Match of several candidate barcodes based on genetic distance are showed in Table <a href="#Tab5">5</a>. Among the single genes, the accuracy rate of <i>ycf1</i> gene for <i>orchid</i> plant identification is 89.32%, with 3.38% fuzzy identification rate and 6.25% error identification rate. The <i>ndhF</i> gene exhibits the highest identification rate and lower error rate of <i>matK</i>â€‰+â€‰<i>ycf1</i>, followed by <i>ndhF</i>â€‰+â€‰<i>ycf1</i> sequence. The accuracy of <i>matK</i>â€‰+â€‰<i>ycf1</i> sequence was 89.6%, with 2.8% fuzzy identification rate and 1.12% error identification rate. The accuracy rate of <i>ndhF</i>â€‰+â€‰<i>ycf1</i> sequence was 88.78%, with 2.33% fuzzy identification rate and 2.8% error identification rate. The data indicated that <i>ndhF</i> and <i>ycf1</i> were suitable for the identification of <i>Orchids</i> at the level of genus and species, while the combined sequences of <i>matK</i>â€‰+â€‰<i>ycf1</i> and <i>ndhF</i>â€‰+â€‰<i>ycf1</i> were qualified at the genera and species levels. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9</span></a></div>   <p>Histogram of frequency of intra-species (black) and inter-species (red) of <i>Orchidaceae</i> based on K2P distance of candidate genes. The X-axis represents the genetic distance, and the Y-axis represents the frequency.</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab5"><span class="label" tagx="label" title="label">Table 5</span> <p>Best Close Match test results based on genetic distance.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Sequences</th> <th align="left">Correct</th> <th align="left">Fuzzy</th> <th align="left">Error</th> <th align="left">Did not identify</th> </tr> </thead> <tbody> <tr> <td align="left"><i>ndhF</i></td> <td align="left">88.65%</td> <td align="left">3.45%</td> <td align="left">2.50%</td> <td align="left">5.48%</td> </tr> <tr> <td align="left"><i>ycf1</i></td> <td align="left">89.32%</td> <td align="left">3.38%</td> <td align="left">6.25%</td> <td align="left">1.05%</td> </tr> <tr> <td align="left"><i>matK</i>â€‰+â€‰<i>ycf1</i></td> <td align="left">89.60%</td> <td align="left">2.80%</td> <td align="left">1.12%</td> <td align="left">6.48%</td> </tr> <tr> <td align="left"><i>ndhF</i>â€‰+â€‰<i>ycf1</i></td> <td align="left">88.78%</td> <td align="left">2.33%</td> <td align="left">2.80%</td> <td align="left">6.09%</td> </tr> </tbody> </table> </div> <p /> </div> <div class="specificbarcodesbasedonsnpsites" title="sec"> <div class="title" tagx="title" title="title">
Specific barcodes based on SNP sites</div> <p id="Par15">Based on SNP sites, species-specific barcodes were developed and the appropriate fragments were blasted into the NCBI database. Based on the <i>ndhF</i> sequence, the specific barcode of species <i>Dendrobium scoriarum</i> was obtained. Knowledge about specific barcodes of species <i>Neuwiedia thelymitra</i>, <i>Spiranthes sinensis</i> and <i>Epiblema cocflorum</i> based on <i>ycf1</i> sequence was obtained. Based on the combined sequence <i>ndhF</i>â€‰+â€‰<i>ycf1</i>, the specific barcodes of <i>Liparis loeselii</i>, <i>Cremastraa ppendiculata</i>, <i>Spiranthes siensis</i> and <i>Anathallis obovata</i> were obtained, whereas <i>Liparis loeselii</i> and <i>Cremastra appendiculata</i> had two specific barcodes. Two-Dimensional code can be scanned by electronic equipment from DNA fragments that can be used for species identification. It can provide theoretical support for subsequent researchers. Using the Two-Dimensional code coding method, the species-specific barcode obtained was converted into two-dimensional barcode image, which was conducive to the conversion of barcode information (Figs. <a href="#Fig10">10</a>, <a href="#Fig11">11</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 10</span></a></div>   <p>DNA barcodes and two-dimensional DNA barcodes of <i>Orchidaceae</i> species based on <i>ndhF</i> and <i>ycf1</i> genes. Base A in green, base T in red, base C in blue, and base G in black.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 11</span></a></div>   <p>DNA barcodes and two-dimensional DNA barcodes of <i>Orchidaceae</i> species based on <i>matK</i>â€‰+â€‰<i>ycf1</i> and <i>ndhF</i>â€‰+â€‰<i>ycf1</i> genes. Base A in green, base T in red, base C in blue, and base G in black.</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par16">DNA barcode is able to be utilized for species identification by means of a DNA fragment that is common to all species. The fragment must simultaneously contain adequate variability to allow for species identification and enough conservative area for the design of universal primers<sup><a href="#CR21">21</a>,<a href="#CR23">23</a></sup>. So far, DNA barcoding have been widely used in many genera of <i>Orchidaceae</i><sup><a href="#CR1">1</a>,<a href="#CR2">2</a>,<a href="#CR7">7</a>,<a href="#CR16">16</a></sup>. As far as we know, it is the first time that multi-aspect analysis in species identification of <i>Orchidaceae</i> with such a well-rounded species size, based on <i>matK</i> and <i>rbcL</i> regions.</p> <p id="Par17">The results of sequences analyses on average GC content showed that the GC content of candidate sequences of <i>Orchids</i> was far less than AT content, while significantly less than the GC content of about 50% in common angiosperms. Of sequence variation situation analysis, the candidate gene mutations exist base insert and missing phenomenon. We performed the analysis of the genetic diversity by the DnasP 5.0 software. The higher haploid type diversity and relatively low haploid type diversity of nucleotide diversity demonstrated that the candidate sequences had certain polymorphisms.</p> <p id="Par18">The CBOL recommends <i>matK</i> and <i>rbcL</i> as universal barcodes in plant kingdom<sup><a href="#CR23">23</a></sup>. With the development of science and technology, many subsequent scientists have evaluated the discriminability of different DNA barcoding genes in different families or genera, but the discriminability of a candidate gene in different plants was different.</p> <p id="Par19">On the basis of phylogenetic relationship, the Barcoding Gap and the Best Close Match with the genetic distance in evaluating candidate barcode identification capability in <i>Orchid</i>, the phylogenetic analyses showed that the identification ability of <i>matK</i> and <i>rbcL</i> was low on the genus level. The possible reason was that there were more species in this study, which made the species in the related genus unable to form branches alone. The sequences of <i>ndhF</i> and <i>ycf1</i> were suitable for identification of genus and species of <i>Orchids</i>, and the combined sequences <i>matK</i>â€‰+â€‰<i>ycf1</i> and <i>ndhF</i>â€‰+â€‰<i>ycf1</i> were qualified at the genera and species levels. The Barcoding Gap test indicated that these candidate genes all contained Barcoding Gap, and the variation between species and within species had clear boundaries. The test results of Best Close Match revealed that the all combined sequences exhibited high genus identification rate, which was suitable for the identification of orchids at the level of genus and species.</p> <p id="Par20">Based on the SNP sites, the species level specific DNA barcodes of <i>Orchid</i> were successfully developed. Combinatorial sequences were able to develop more species-specific barcodes than chloroplast genes, which might be the result of combination sequences could provide more mutation sites and SNP sites. There were some differences in the specificities of different combination genes in <i>Orchidaceae</i> plants. Compared with <i>ndhF</i>â€‰+â€‰<i>ycf1</i>, the combined sequences of <i>matK</i>â€‰+â€‰<i>ycf1</i> could be developed more specific barcodes, which might be related to the species identification accuracy of <i>matK</i>â€‰+â€‰<i>ycf1</i> in <i>Orchids.</i></p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par21">In summary, <i>ndhF</i>, <i>ycf1</i>, <i>matK</i>â€‰+â€‰<i>ycf1</i> and <i>ndhF</i>â€‰+â€‰<i>ycf1</i> sequences are competent to develop species-specific barcodes to identify <i>Orchidaceae</i> plants at the molecular level. Cluster analysis using the <i>ndhF</i>, <i>ycf1</i>, <i>matK</i>â€‰+â€‰<i>ycf1</i> and <i>ndhF</i>â€‰+â€‰<i>ycf1</i> sequences in <i>Orchid</i> are nearly consistent with traditional plant morphology. Additionally, this study not only broadens the application of the <i>matK</i> and <i>rbcL</i> sequences in the barcode field, but also provides a novel thought to expand species identification method in a wide range of plant at the species level.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="nucleotidesequences" title="sec"> <div class="title" tagx="title" title="title">
Nucleotide sequences</div> <p id="Par22">For species identification, we retrieved the chloroplast DNA reference sequences including <i>matK</i>, <i>rbcL, ndhF and ycf1</i> from the NCBI Gene database (<a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>). We obtained the combined sequence including <i>matK</i>â€‰+â€‰<i>rbcL</i>, <i>matK</i>â€‰+â€‰<i>ycf1, ndhF</i>â€‰+â€‰<i>ycf1</i> by supermat's function in R Phylotools package. After manual screening, the short nucleotide sequences were deleted, and the sequences with different directions were modified manually.</p> </div> <div class="dataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Data analysis</div> <p id="Par23">We performed the sequences alignment by the Muscle in the MEGA 7.0 software<sup><a href="#CR51">51</a></sup> (<a href="https://www.megasoftware.net/">https://www.megasoftware.net/</a>) with the default alignment parameters for multiple sequences alignment parameters. In the pairwise distances analyses, the positions containing gaps and missing were eliminated from the data set (complete deletion option). Phylogenetic trees constructed with the Neighbor-joining (NJ) method according to Kimura 2-Parameter (K2P) model was assessed by the MEGA 7.0<sup><a href="#CR9">9</a>,<a href="#CR46">46</a>,<a href="#CR52">52</a></sup>. The clade reliability in these trees using the NJ methods was tested by bootstrapping, in which 1000 repeated sampling tests were performed to obtain the support values of the clade nodes. Polymorphic site, genetic diversity indices and neutrality tests [Fuâ€™s <i>Fs</i><sup><a href="#CR49">49</a></sup> and Tajimaâ€™s <i>D</i><sup><a href="#CR50">50</a></sup>] were performed by the DnaSP v5<sup><a href="#CR53">53</a></sup> (<a href="http://www.ub.edu/dnasp/index_v5.html">http://www.ub.edu/dnasp/index_v5.html</a>).</p> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>These authors contributed equally: Huili Li and Wenjun Xiao.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This research was supported by grants from National Key Research and Development Program (2017YFF0210301), National Natural Science Foundation of China (31872866 and 31540064), Key Research &amp;amp; Development Project of Hunan Provincial Department of Science and Technology (2019NK2081 and 2017SK2182).</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>G.X. and L.H. designed the research. L.H., X.W. and L.Y. conducted experiments. G.X., L.H., X.W., T.T., L.Y., Z.M., X.L., Z.X., and W.Q. analyzed the data. L.H. and X.W. drafted the manuscript and all authors reviewed and approved the manuscript.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par24">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References </div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chase</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cameron</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freudenstein</span><span class="given-names" tagx="given-names" title="given-names">JV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pridgeon</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AndrÃ©</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">An updated classification of <i>Orchidaceae</i></span><span class="source" tagx="source" title="source">Bot. J. Linn Soc.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">177</span><span class="fpage" tagx="fpage" title="fpage">151</span><span class="lpage" tagx="lpage" title="lpage">174</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/boj.12234">10.1111/boj.12234</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oh</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhandari</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span></span><span class="mixed-article-title" title="mixed-article-title">DNA barcoding of <i>Orchidaceae</i> in Korea</span><span class="source" tagx="source" title="source">Mol. Ecol. Resour.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">499</span><span class="lpage" tagx="lpage" title="lpage">507</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1755-0998.12207">10.1111/1755-0998.12207</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24267156">24267156</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gamarra</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cela</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">OrtÃºÃ±ez</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Orchidaceae</i> in equatorial guinea (west tropical Africa): Nomenclatural and taxonomic notes, new records and critical taxa</span><span class="source" tagx="source" title="source">Kew Bull.</span><span class="year" tagx="year" title="year">2019</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s12225-018-9787-9">10.1007/s12225-018-9787-9</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atwood</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span></span><span class="mixed-article-title" title="mixed-article-title">The size of the <i>Orchidaceae</i> and the systematic distribution of epiphytic <i>Orchids</i></span><span class="source" tagx="source" title="source">Selbyana</span><span class="year" tagx="year" title="year">1986</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">171</span><span class="lpage" tagx="lpage" title="lpage">186</span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arditti</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="source" tagx="source" title="source">Fundamentals of Orchid Biology,</span><span class="year" tagx="year" title="year">1992</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Wiley</span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douzery</span><span class="given-names" tagx="given-names" title="given-names">EJP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pridgeon</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kores</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linder</span><span class="given-names" tagx="given-names" title="given-names">HP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chase</span><span class="given-names" tagx="given-names" title="given-names">KMW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular phylogenetics of disease (<i>Orchidaceae</i>): A contribution from nuclear ribosomal ITS sequences</span><span class="source" tagx="source" title="source">Am. J. Bot.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">887</span><span class="lpage" tagx="lpage" title="lpage">899</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/2656709">10.2307/2656709</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10371730">10371730</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular identification of <i>Dendrobium</i> species (<i>Orchidaceae</i>) based on the DNA barcode ITS2 region and its application for phylogenetic study</span><span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">21975</span><span class="lpage" tagx="lpage" title="lpage">21988</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms160921975">10.3390/ijms160921975</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26378526">26378526</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schuiteman</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Devogelia (<i>Orchidaceae</i>): A new genus from the moluccas and new guinea</span><span class="source" tagx="source" title="source">Blumea</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">361</span><span class="lpage" tagx="lpage" title="lpage">366</span><span class="pub-id"><a href="https://dx.doi.org/10.3767/000651904X484324">10.3767/000651904X484324</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simo</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plunkett</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Droissart</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title">New phylogenetic insights toward developing a natural generic classification of African angraecoid <i>Orchids</i> (<i>Vandeae Orchidaceae</i>)</span><span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">126</span><span class="fpage" tagx="fpage" title="fpage">241</span><span class="lpage" tagx="lpage" title="lpage">249</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2018.04.021">10.1016/j.ympev.2018.04.021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29673694">29673694</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PÃ©rez</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Orchids: A review of uses in traditional medicine, its phytochemistry and pharmacology</span><span class="source" tagx="source" title="source">J. Med. Plants Res.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">592</span><span class="lpage" tagx="lpage" title="lpage">638</span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacquemyn</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merckx</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brys</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of network architecture reveals phylogenetic constraints on mycorrhizal specificity in the genus <i>Orchis</i> (<i>Orchidaceae</i>)</span><span class="source" tagx="source" title="source">New Phytol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">192</span><span class="fpage" tagx="fpage" title="fpage">518</span><span class="lpage" tagx="lpage" title="lpage">528</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1469-8137.2011.03796.x">10.1111/j.1469-8137.2011.03796.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21668874">21668874</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vij</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atwood</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span></span><span class="mixed-article-title" title="mixed-article-title">The size of the <i>Orchidaceae</i> and the systematic distribution of epiphytic <i>Orchids</i></span><span class="source" tagx="source" title="source">Selbyana</span><span class="year" tagx="year" title="year">1986</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">171</span><span class="lpage" tagx="lpage" title="lpage">186</span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoshikawa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murakami</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Novel indole S, O-bisdesmoside, calanthoside, the precursor glycoside of tryptanthrin, indirubin, and isatin, with increasing skin blood flow promoting effects, from two <i>Calanthe</i> species (<i>Orchidaceae</i>)</span><span class="source" tagx="source" title="source">Chem. Pharm. Bull.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">886</span><span class="lpage" tagx="lpage" title="lpage">888</span><span class="pub-id"><a href="https://dx.doi.org/10.1248/cpb.46.886">10.1248/cpb.46.886</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakurai</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure of cymbidine A, a monomeric peptidoglycan-related compound with hypotensive and diuretic activities, isolated from a higher plant, <i>Cymbidium goeringii</i> (<i>Orchidaceae</i>)</span><span class="source" tagx="source" title="source">Chem. Pharm. Bull.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">780</span><span class="lpage" tagx="lpage" title="lpage">783</span><span class="pub-id"><a href="https://dx.doi.org/10.1248/cpb.55.780">10.1248/cpb.55.780</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eda</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Budak</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detection and quantification of salep with real time PCR utilizing the nr-its2 region</span><span class="source" tagx="source" title="source">J. Sci. Food Agric.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">2447</span><span class="lpage" tagx="lpage" title="lpage">2454</span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asahina</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shinozaki</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masuda</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of medicinal <i>Dendrobium</i> species by phylogenetic analyses using <i>matK</i> and <i>rbcL</i> sequences</span><span class="source" tagx="source" title="source">J. Nat. Med.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">133</span><span class="lpage" tagx="lpage" title="lpage">138</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11418-009-0379-8">10.1007/s11418-009-0379-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20140532">20140532</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Boer</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghorbani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manzanilla</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raclariu</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kreziou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ounjai</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">DNA metabarcoding of orchid derived products reveals widespread illegal orchid trade</span><span class="source" tagx="source" title="source">Proc. R. Soc. Lond. B Biol. Sci.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">284</span><span class="fpage" tagx="fpage" title="fpage">1863</span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Techen</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parveen</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">IA</span></span></span><span class="mixed-article-title" title="mixed-article-title">DNA barcoding of medicinal plant material for identifcation</span><span class="source" tagx="source" title="source">Curr. Opin. Biotech.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">103</span><span class="lpage" tagx="lpage" title="lpage">110</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.copbio.2013.09.010">10.1016/j.copbio.2013.09.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24484887">24484887</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devos</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oh</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raspe</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacquemart</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manos</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nuclear ribosomal DNA sequence variation and evolution of spotted marsh <i>Orchids</i> (<i>Dactylorhiza maculata</i> group)</span><span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">568</span><span class="lpage" tagx="lpage" title="lpage">580</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2005.04.014">10.1016/j.ympev.2005.04.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15939630">15939630</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">GQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lohaus</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsiao</span><span class="given-names" tagx="given-names" title="given-names">YY</span></span></span><span class="mixed-article-title" title="mixed-article-title">The <i>Apostasia</i> genome and the evolution of orchids</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">549</span><span class="fpage" tagx="fpage" title="fpage">379</span><span class="lpage" tagx="lpage" title="lpage">383</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature23897">10.1038/nature23897</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28902843">28902843</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hebert</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gregory</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span></span><span class="mixed-article-title" title="mixed-article-title">The promise of DNA barcoding for taxonomy</span><span class="source" tagx="source" title="source">Syst. Bot.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">852</span><span class="lpage" tagx="lpage" title="lpage">859</span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coissac</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hollingsworth</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavergne</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">From barcodes to genomes: Extending the concept of DNA barcoding</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">423</span><span class="lpage" tagx="lpage" title="lpage">1428</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.13549">10.1111/mec.13549</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: CBOL Plant Working Group</span></span><span class="mixed-article-title" title="mixed-article-title">A DNA barcode for land plants</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">12794</span><span class="lpage" tagx="lpage" title="lpage">12797</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0905845106">10.1073/pnas.0905845106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19666622">19666622</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cameron</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chase</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitten</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span></span><span class="mixed-article-title" title="mixed-article-title">A phylogenetic analysis of the <i>Orchidaceae</i>: evidence from <i>rbcL</i> nucleotide sequences</span><span class="source" tagx="source" title="source">Am. J. Bot.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">208</span><span class="lpage" tagx="lpage" title="lpage">224</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/2656938">10.2307/2656938</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21680360">21680360</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lahaye</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van der, Bank, M., Bogarin, D., Warner, J., Pupulin, F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">DNA barcoding the floras of biodiversity hotspots</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">2923</span><span class="lpage" tagx="lpage" title="lpage">2928</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0709936105">10.1073/pnas.0709936105</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18258745">18258745</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">ZB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">XM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Species identification of the medicinal plant <i>Tulipa edulis</i> (<i>Liliaceae</i>) by DNA barcode marker</span><span class="source" tagx="source" title="source">Biochem. Syst. Ecol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">362</span><span class="lpage" tagx="lpage" title="lpage">368</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bse.2014.03.038">10.1016/j.bse.2014.03.038</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of the DNA barcodes in <i>Dendrobium</i> (<i>Orchidaceae</i>) from mainland Asia</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e0115168</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0115168">10.1371/journal.pone.0115168</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25602282">25602282</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hebert</span><span class="given-names" tagx="given-names" title="given-names">PDN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratnasingham</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De</span><span class="given-names" tagx="given-names" title="given-names">WJR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Barcoding animal life: Cytochrome c oxidase subunit 1 divergences among closely related species</span><span class="source" tagx="source" title="source">Proc. R. Soc. Lond. B Biol. Sci.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">270</span><span class="fpage" tagx="fpage" title="fpage">S96</span><span class="lpage" tagx="lpage" title="lpage">S99</span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hebert</span><span class="given-names" tagx="given-names" title="given-names">PDN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoeckle</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zemlak</span><span class="given-names" tagx="given-names" title="given-names">TS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francis</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of birds through DNA barcodes</span><span class="source" tagx="source" title="source">PLoS Biol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">e312</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.0020312">10.1371/journal.pbio.0020312</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15455034">15455034</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salazar</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chase</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ingrouille</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenetics of <i>Cranichideae</i> with emphasis on <i>Spiranthina</i>e (<i>Orchidaceae</i>, <i>Orchidoideae</i>): Evidence from plastid and nuclear DNA sequences</span><span class="source" tagx="source" title="source">Am. J. Bot.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">777</span><span class="lpage" tagx="lpage" title="lpage">795</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/ajb.90.5.777">10.3732/ajb.90.5.777</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21659175">21659175</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kress</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Use of DNA barcodes to identify flowering plants</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">102</span><span class="fpage" tagx="fpage" title="fpage">8369</span><span class="lpage" tagx="lpage" title="lpage">8374</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0503123102">10.1073/pnas.0503123102</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15928076">15928076</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldman</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freudenstein</span><span class="given-names" tagx="given-names" title="given-names">JV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kores</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenetics of <i>Arethuseae</i> (<i>Orchidaceae</i>) based on plastid <i>matK</i> and <i>rbcL</i> sequences</span><span class="source" tagx="source" title="source">Syst. Bot.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">670</span><span class="lpage" tagx="lpage" title="lpage">695</span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kores</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Molvray</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weston</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hopper</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chase</span><span class="given-names" tagx="given-names" title="given-names">CMW</span></span></span><span class="mixed-article-title" title="mixed-article-title">A phylogenetic analysis of <i>Diurideae</i> (<i>Orchidaceae</i>) based on plastid DNA sequence data</span><span class="source" tagx="source" title="source">Am. J. Bot.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">1903</span><span class="lpage" tagx="lpage" title="lpage">2191</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/3558366">10.2307/3558366</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21669623">21669623</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kocyan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiu</span><span class="given-names" tagx="given-names" title="given-names">YL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endress</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conti</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">A phylogenetic analysis of <i>Apostasioideae</i> (<i>Orchidaceae</i>) based on <i>ITS</i>, <i>trnL-F</i> and <i>matK</i> sequences</span><span class="source" tagx="source" title="source">Plant Syst. Evol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">247</span><span class="fpage" tagx="fpage" title="fpage">203</span><span class="lpage" tagx="lpage" title="lpage">213</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00606-004-0133-3">10.1007/s00606-004-0133-3</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lahaye</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van der Bank</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bogarin</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warner</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pupulin</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">DNA barcoding the floras of biodiversity hotspots</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">2923</span><span class="lpage" tagx="lpage" title="lpage">2928</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0709936105">10.1073/pnas.0709936105</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18258745">18258745</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farrington</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacGillivray</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faast</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Austin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Investigating DNA barcoding options for the identification of <i>Caladenia</i> (<i>Orchidaceae</i>) species</span><span class="source" tagx="source" title="source">Aust. J. Bot.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">276</span><span class="lpage" tagx="lpage" title="lpage">286</span><span class="pub-id"><a href="https://dx.doi.org/10.1071/BT08138">10.1071/BT08138</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Using DNA barcoding to identify species within <i>Euphorbiaceae</i></span><span class="source" tagx="source" title="source">Planta Med.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">1784</span><span class="lpage" tagx="lpage" title="lpage">1786</span><span class="pub-id"><a href="https://dx.doi.org/10.1055/s-0030-1249806">10.1055/s-0030-1249806</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20379956">20379956</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sui</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular authentication of the ethnomedicinal plant <i>Sabia parviflora</i> and its adulterants by DNA barcoding technique</span><span class="source" tagx="source" title="source">Planta Med.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">492</span><span class="lpage" tagx="lpage" title="lpage">496</span><span class="pub-id"><a href="https://dx.doi.org/10.1055/s-0030-1250468">10.1055/s-0030-1250468</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20979018">20979018</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">DNA barcodes for discriminating the medicinal plant <i>Scutellaria baicalensis</i> (<i>Lamiaceae</i>) and its adulterants</span><span class="source" tagx="source" title="source">Biol. Pharm. Bull.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">1198</span><span class="lpage" tagx="lpage" title="lpage">1203</span><span class="pub-id"><a href="https://dx.doi.org/10.1248/bpb.34.1198">10.1248/bpb.34.1198</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21804206">21804206</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Do</span><span class="given-names" tagx="given-names" title="given-names">HDK</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The newly developed single nucleotide polymorphism (SNP) markers for a potentially medicinal plant, <i>Crepidiastrum denticulatum</i> (<i>Asteraceae</i>), inferred from complete chloroplast genome data</span><span class="source" tagx="source" title="source">Mol. Biol. Rep.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">3287</span><span class="lpage" tagx="lpage" title="lpage">3297</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11033-019-04789-5">10.1007/s11033-019-04789-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30980269">30980269</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JaÃ©n</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marrero</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reyes</span><span class="given-names" tagx="given-names" title="given-names">BJA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santos</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naranjo</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">CaujapÃ©</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular taxonomic identification in the absence of a â€˜barcoding gapâ€™: A test with the endemic flora of the canarian oceanic hotspot</span><span class="source" tagx="source" title="source">Mol. Ecol. Resour.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">42</span><span class="lpage" tagx="lpage" title="lpage">56</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1755-0998.12292">10.1111/1755-0998.12292</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24919404">24919404</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span></span><span class="mixed-article-title" title="mixed-article-title">DNA barcoding of 21 rare and protected medicinal plants in Guangdong Province</span><span class="source" tagx="source" title="source">Chin. Med. Clin. Pharma</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">979</span><span class="lpage" tagx="lpage" title="lpage">984</span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">XH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Species-specific identification from incomplete sampling: Applying DNA barcodes to monitoring invasive Solanum plants</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e55927</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0055927">10.1371/journal.pone.0055927</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23409092">23409092</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Batista</span><span class="given-names" tagx="given-names" title="given-names">JAN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mota</span><span class="given-names" tagx="given-names" title="given-names">ACM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Proite</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bianchetti</span><span class="given-names" tagx="given-names" title="given-names">LDB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gustavo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romerogonzalez</span><span class="given-names" tagx="given-names" title="given-names">HH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular phylogenetics of neotropical cyanaeorchis (<i>Cymbidieae</i>, <i>Epidendroideae</i>, <i>Orchidaceae</i>): Geographical rather than morphological similarities plus a new species</span><span class="source" tagx="source" title="source">Phytotaxa</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">251</span><span class="lpage" tagx="lpage" title="lpage">272</span><span class="pub-id"><a href="https://dx.doi.org/10.11646/phytotaxa.156.5.1">10.11646/phytotaxa.156.5.1</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">WT</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular systematics of subtribe <i>Orchidinae</i> and Asian taxa of <i>Habenariinae</i> (<i>Orchideae</i>, <i>Orchidaceae</i>) based on plastid <i>matK</i>, <i>rbcL</i> and nuclear ITS</span><span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">41</span><span class="lpage" tagx="lpage" title="lpage">53</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2014.04.004">10.1016/j.ympev.2014.04.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24747003">24747003</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Jin, W. T., Schuiteman, A., Chase, M. W., Li, J. W., Chung, S. W., Hsu, T. C. Phylogenetics of subtribe <i>Orchidinae</i> s.l. (<i>Orchidaceae</i>; <i>Orchidoideae</i>) based on seven markers (plastid <i>matK</i>, <i>psab</i>, <i>rbcL</i>, <i>trnL-F</i>, <i>trnH-psbA</i>, and nuclear <i>nrITS</i>, <i>xdh</i>): Implications for generic delimitation. <i>BMC Plant Biol.</i><b>17</b>, 222 (2017).</span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verlynde</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular phylogeny of the genus <i>Bolusiella</i> (<i>Orchidaceae</i>, <i>Angraecinae</i>)</span><span class="source" tagx="source" title="source">Plant Syst. Evol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">304</span><span class="fpage" tagx="fpage" title="fpage">269</span><span class="lpage" tagx="lpage" title="lpage">279</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00606-017-1474-z">10.1007/s00606-017-1474-z</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular systematics of <i>Goodyerinae</i> (Cranichideae, Orchidoideae, <i>Orchidaceae</i>) based on multiple nuclear and plastid regions</span><span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">106542</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2019.106542">10.1016/j.ympev.2019.106542</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">YX</span></span></span><span class="mixed-article-title" title="mixed-article-title">Statistical tests of neutrality of mutations against population growth, hitchhiking and background selection</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">147</span><span class="fpage" tagx="fpage" title="fpage">915</span><span class="lpage" tagx="lpage" title="lpage">925</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9335623">9335623</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tajima</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">1989</span><span class="volume" tagx="volume" title="volume">123</span><span class="fpage" tagx="fpage" title="fpage">585</span><span class="lpage" tagx="lpage" title="lpage">595</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2513255">2513255</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">MEGA7: Molecular evolutionary genetics analysis Version 7.0 for bigger datasets</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">1870</span><span class="lpage" tagx="lpage" title="lpage">1874</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msw054">10.1093/molbev/msw054</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27004904">27004904</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span></span><span class="mixed-article-title" title="mixed-article-title">A molecular phylogeny of the bladed <i>Bangiales</i> (<i>Rhodophyta</i>) in China provides insights into biodiversity and biogeography of the genus, Pyropia</span><span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">120</span><span class="fpage" tagx="fpage" title="fpage">94</span><span class="lpage" tagx="lpage" title="lpage">102</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2017.11.009">10.1016/j.ympev.2017.11.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29180103">29180103</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Librado</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozas</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">DnaSP v5: A software for comprehensive analysis of DNA polymorphism data</span><span class="source" tagx="source" title="source">Bioinform</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1451</span><span class="lpage" tagx="lpage" title="lpage">1452</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp187">10.1093/bioinformatics/btp187</a></span></span></li> </ul> </div> </div>  </body></html>