<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Front Chem</span><span class="iso-abbrev" title="iso-abbrev">Front Chem</span><span class="publisher-id" title="publisher-id">Front. Chem.</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Chemistry</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2296-2646</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7680914">7680914</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fchem.2020.596479">10.3389/fchem.2020.596479</a></span><div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemistry</span>: <span class="subject" title="subject">Mini Review</span></div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Nature's Chemists: The Discovery and Engineering of Phytochemical Biosynthesis</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eljounaidi</span><span class="given-names" tagx="given-names" title="given-names">Kaouthar</span></span><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/1065755/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lichman</span><span class="given-names" tagx="given-names" title="given-names">Benjamin R.</span></span><a href="#c001"><sup>*</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/1064321/overview" /></span></span></div><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Centre for Novel Agricultural Products, Department of Biology, University of York</span><span class="addr-line" title="addr-line">York</span><span class="country" tagx="country" title="country">United Kingdom</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Chun Xu, The University of Queensland, Australia</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Anne Osbourn, John Innes Centre, United Kingdom; Paripok Phitsuwan, King Mongkut's University of Technology Thonburi, Thailand</p> </div> <div class="corresp" title="corresp">
*Correspondence: Benjamin R. Lichman <span class="email" tagx="email" title="email">benjamin.lichman@york.ac.uk</span></div> <div class="fn-type-other" title="other"> <p>This article was submitted to Chemical Biology, a section of the journal Frontiers in Chemistry</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">596479</span><span class="history" title="history"><span class="received" title="received">received: 2020-8-19</span><span class="accepted" title="accepted">accepted: 2020-10-09</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Eljounaidi and Lichman</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Plants produce a diverse array of natural products, many of which have high pharmaceutical value or therapeutic potential. However, these compounds often occur at low concentrations in uncultivated species. Producing phytochemicals in heterologous systems has the potential to address the bioavailability issues related to obtaining these molecules from their natural source. Plants are suitable heterologous systems for the production of valuable phytochemicals as they are autotrophic, derive energy and carbon from photosynthesis, and have similar cellular context to native producer plants. In this review we highlight the methods that are used to elucidate natural product biosynthetic pathways, including the approaches leading to proposing the sequence of enzymatic steps, selecting enzyme candidates and characterizing gene function. We will also discuss the advantages of using plant chasses as production platforms for high value phytochemicals. In addition, through this report we will assess the emerging metabolic engineering strategies that have been developed to enhance and optimize the production of natural and novel bioactive phytochemicals in heterologous plant systems.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
phytochemicals</div> <div class="kwd" title="kwd">
gene discovery</div> <div class="kwd" title="kwd">
biosynthetic pathway</div> <div class="kwd" title="kwd">
bioactive chemicals</div> <div class="kwd" title="kwd">
synthetic biology</div> <div class="kwd" title="kwd">
metabolic engineering</div> <div class="kwd" title="kwd">
natural products</div> <div class="kwd" title="kwd">
biosynthesis</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
UK Research and Innovation<span class="named-content" title="named-content">10.13039/100014013</span></div><span class="award-id" tagx="award-id" title="award-id">MR/S01862X/1</span></div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Plants have a remarkable capacity to produce a wide array of specialized metabolites (also referred to as secondary metabolites or natural products), to support their defense and ecological adaptation. These phytochemicals exhibit a variety of bioactivities and many of them are of considerable pharmaceutical importance. Based on their biosynthetic origins, plant secondary metabolites can be divided into three major groups: terpenoids, alkaloids, and phenolics.</p> <p>Terpenoids or isoprenoids are a structurally diverse class of plant specialized metabolites. They are derived by the repetitive fusion of branched five-carbon isopentanes, usually referred to as isoprene units. Many terpenoids have biological activities and are utilized as valuable pharmaceuticals. The most renowned terpenoid-based drugs include the antimalarial medicine artemisinin (<i>Artemisia annua</i>) and the anticancer drug Taxol (<i>Taxus brevifolia</i>) (<a href="#F1">Figure 1A</a>). Another major class of secondary metabolites are alkaloids, nitrogen containing low-molecular-weight compounds, typically derived from amino acids (Lichman, <a href="#B33">2020</a>). Numerous alkaloids (or their derivatives) are employed as high value drugs, such as vincristine and vinblastine (anticancer) from Madagascar periwinkle (<i>Catharanthus roseus</i>), camptothecin (anticancer) from <i>Camptotheca acuminata</i> and morphine (analgesic drug) from opium poppy (<a href="#F1">Figure 1B</a>). Phenolics are a large and diverse group of aromatic compounds which include the tannins, phenylpropanoid, anthocyanin and lignan subgroups. Etoposide and teniposide are among the most pharmaceutically important polyphenols (<a href="#F1">Figure 1C</a>). These compounds are derivatives of podophyllotoxin, a lignan from mayapple (<i>Podophyllum peltatum</i>), and are widely used in chemotherapies. Alongside alkaloids, terpenoids, and phenolics, plants also produce other types of secondary metabolites, such as polyketides, cyanogenic glycosides and glucosinolates, of which many have pharmaceutical relevance.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Examples of plant specialized metabolites of pharmaceutical significance. <b>(A)</b> Terpenoids: artemisinin and taxol. Isoprene units are highlighted with colors. <b>(B)</b> Alkaloids: camptothecin, vinblastine, and vincristine with tryptophan precursor highlighted; morphine tyrosine precursors highlighted. <b>(C)</b> Phenolics: etoposide and teniposide with phenylpropanoid units highlighted.</p>   </div> <p>Sourcing bioactive natural products from plants has many challenges. These metabolites are often found in low abundance or are produced in rare or slow-growing plant species, which limits the availability and accessibility of the active pharmaceutical agent and impacts market prices. In many cases, these molecules are very complex and are recalcitrant to chemical synthesis, as they contain many chiral centers or polycyclic rings (Nicolaou and Rigol, <a href="#B44">2020</a>). Even when synthesis is possible, it is not sustainable as it often relies on petrochemical feedstocks and/or environmentally unfriendly production processes (Lechner et al., <a href="#B27">2016</a>). This makes finding alternative sources for the supply of these valuable molecules necessary.</p> <p>Increased access to phytochemicals can be achieved through a number of approaches. For instance, the biosynthetic capabilities of the natural producer may be enhanced through selective breeding programmes (Townsend et al., <a href="#B59">2013</a>) or through genetic modification and the creation of transgenic lines (Shen et al., <a href="#B56">2018</a>). Cell cultures may also enable greater control over the production of the compound of interest. For example, cambial meristematic cell cultures derived from <i>Taxus cuspidata</i> have been shown to produce high yields of the anticancer drug, paclitaxel (Taxol), while circumventing the obstacles routinely associated with the commercial growth of dedifferentiated plant cells (Lee et al., <a href="#B29">2010</a>).</p> <p>Another alternative and increasingly popular approach is the transfer of biosynthetic pathways into heterologous systems such as plants (e.g., <i>Nicotiana benthamiana</i>) or microbes (e.g., <i>Saccharomyces cerevisiae</i> and <i>Escherichia coli</i>). The advantages of using microbes as production platforms for valuable chemicals include fast growth cycles, ease of genetic modification, and the fact that they offer a simplified product purification pipeline. The synthetic biology approaches used to produce plant metabolites in microbial organisms have been recently reviewed (Moses et al., <a href="#B40">2017</a>; Li et al., <a href="#B32">2018</a>). Plants, on the other hand, are photoautotrophs and do not require exogenous carbon sources, unlike typical microbial platforms. Plants also provide a cellular context similar to the native producer, which make them an attractive platform for the production of valuable chemicals.</p> <p>In order to engineer valuable phytochemicals in microbe or plants, the first step is to uncover the biosynthetic pathway of the metabolite of interest. In this review we highlight the recent methods that have been used to elucidate natural product biosynthetic pathways, including the approaches leading to proposing the sequence of enzymatic steps, assigning enzyme family, as well as, gene function elucidation. We will also discuss the advantages of using plant chassis as a production platform for high value phytochemicals. In addition, through this report we will assess the emerging metabolic engineering strategies that have been developed to enhance and optimize the production of natural and novel bioactive phytochemicals in heterologous plant systems.</p> </div> <div class="discoveryofspecializedmetabolicpathways" title="sec"> <div class="title" tagx="title" title="title">
Discovery of Specialized Metabolic Pathways</div> <div class="proposingabiosyntheticpathway" title="sec"> <div class="title" tagx="title" title="title">
Proposing a Biosynthetic Pathway</div> <p>In order to identify the enzymes that underlie the biosynthesis of valuable plant natural products, it is first necessary to hypothesize a plausible sequence of enzyme catalyzed reactions that can lead from primary metabolism to the molecule of interest. Generally, biomimetic syntheses tend to use mild conditions and harness cascade-like reactions commonly found in biosynthetic pathways, hence providing clues for pathway elucidation (Yoder and Johnston, <a href="#B61">2005</a>).</p> <p>If the metabolic pathway is incompletely described, isotope-labeling studies can be used to identify the unknown steps. Primary metabolites, such as amino acids, sugars, or even dioxygen and carbon dioxide, labeled with stable isotopes (e.g. <sup>2</sup>H, <sup>13</sup>C, <sup>15</sup>N, and <sup>18</sup>O), can be fed to the plant. The target specialized metabolite can then be assessed for isotope labeling by mass spectrometry (MS) or nuclear magnetic resonance (NMR) (Freund and Hegeman, <a href="#B17">2017</a>). This allows us to establish the identity of the starting material and identify its mode of incorporation. Strategically positioned isotopic labels can be used to probe the transformations and structural rearrangements that occur along the biosynthetic pathway.</p> <p>For example, the incompletely described biosynthesis of camptothecin, a monoterpene indole alkaloid (MIA) from <i>Camptotheca acuminata</i>, was investigated using metabolite profiling and isotope labeling studies. The approach led to the identification of plausible intermediates for missing pathway steps and demonstrated that nearly all camptothecin pathway intermediates were present as multiple isomers (Sadre et al., <a href="#B54">2016</a>). Therefore, these metabolite focused strategies can lead to a hypothetical biosynthetic pathway made up of a set of chemically reasonable enzymatic conversions.</p> <p>The next step is to assign possible enzyme classes to each hypothesized reaction based on characterized enzyme activities. For instance, a hydroxylation step will probably involve a cytochrome P450 enzyme or a 2-oxoglutarate dependent oxygenase (Mitchell and Weng, <a href="#B38">2019</a>), whilst a transfer of an amine group onto a carbonyl is likely to involve a PLP-dependent enzyme (Lee and Facchini, <a href="#B28">2011</a>). Once the hypothetical pathway has been populated with proposed enzyme classes, the gene discovery effort can begin.</p> </div> <div class="integrativeapproachesforgeneidentification" title="sec"> <div class="title" tagx="title" title="title">
Integrative Approaches for Gene Identification</div> <p>The discovery of genes involved in the biosynthesis of phytochemicals typically requires the use of multi-omics technologies such as genomics, transcriptomics, proteomics, and metabolomics. Analysis of these data enable the identification of candidate genes that encode enzymes that catalyze biosynthetic steps (<a href="#F2">Figure 2</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Overview of the approaches that can be employed to discover unknown enzymes in a biosynthetic pathway of interest. <b>(A)</b> Identification of candidate genes in plant material with difference in metabolites accumulation, through co-expression analysis or phylogenetic analysis to examine whether a gene clade is specific to the native plant producer. <b>(B)</b> Mining genomic data to look for genes that are physically localized in the vicinity of previously characterized enzymes (gene clusters). <b>(C)</b> Total protein purification and fractionation from the native plant material, followed by functional assays and proteomic identification of the active fraction.</p>   </div> <p>The biosynthesis of natural products is regulated both during development and in response to various environmental stimuli. Transcriptomics can pinpoint differentially expressed genes across different types of tissues (or cells), different developmental stages or in elicitor-treated material. Candidate genes involved in biosynthesis can be identified through correlation of expression with metabolite accumulation, or through identification of co-expressed clusters in which genes involved in the same pathway share expression patterns across tissues (<a href="#F2">Figure 2A</a>).</p> <p>The hormone methyl jasmonate (MeJA) is known to elicit various species-specific specialized metabolic pathways. This feature has often been used to identify genes involved in the biosynthesis of various natural products. Cytochrome P450 (CYP728B70) catalyzing the oxidation step in triptolide biosynthesis, an abietane-type diterpenoid from <i>Tripterygium wilfordii</i>, was identified through an integrative gene prioritization approach (Tu et al., <a href="#B60">2020</a>). The prioritized candidate genes were highly expressed in MeJA-induced cells and/or in the root bark (main accumulation site of the diterpene), as well as, exhibited similar expression patterns to already characterized enzymes in the pathway; copalyl diphosphate synthase and miltiradiene synthase (Tu et al., <a href="#B60">2020</a>).</p> <p>Various valuable plant natural products have been shown to accumulate in specific specialized structures, such as artemisinin in the glandular trichomes of <i>Artemisia annua</i> (Duke et al., <a href="#B15">1994</a>) and morphine in the laticifers of the aerial organs of opium poppy (<i>Papaver somniferum</i>) (Facchini and De Luca, <a href="#B16">1995</a>). Generating multi-omic data from these structures allows the identification of new biosynthetic enzymes involved in pathways of interest. For example, comparative proteomic analysis of trichomes and trichome-depleted leaves in catmint (<i>Nepeta mussinii</i>) led to the discovery of the unusual nepetalactol-related short-chain dehydrogenase enzymes (NEPS), which are enriched in the trichome and involved in the biosynthesis of volatile nepetalactones (Lichman et al., <a href="#B35">2019</a>).</p> <p>Phylogenetic analysis can provide valuable information to further guide the candidate gene prioritization process. Genes involved in a biosynthetic pathway unique to a particular species are likely to be phylogenetically distinct. Therefore, building phylogenetic trees containing candidate genes and homologous genes from other species may reveal subclades unique to the producer plant. Key oxidases in limonoid biosynthesis in <i>Melia azedarach</i>; CYP71BQ5 and CYP71CD2 were identified through this strategy. These cytochrome P450 enzymes were found in a distinct subclade within the phylogenetic tree, comprising only <i>Melia azedarach</i> sequences, suggesting their involvement in this lineage-specific pathway (Hodgson et al., <a href="#B22">2019</a>).</p> <p>Candidate gene selection may also be supplemented by mining whole genomes or partial genome sequences for genes physically localized in the vicinity of previously characterized enzymes (<a href="#F2">Figure 2B</a>). It has been reported that genes involved in biosynthetic pathways of a number of natural products are organized in functional clusters within plant genomes (Roselli et al., <a href="#B52">2017</a>; Lichman et al., <a href="#B34">2020</a>; Liu et al., <a href="#B36">2020</a>). Features of plant biosynthetic gene clusters, how they form and how they are regulated have been recently reviewed (NÃ¼tzmann et al., <a href="#B46">2016</a>, <a href="#B45">2018</a>). Another approach is total protein purification and fractionation, from the native plant material, followed by functional assays (<a href="#F2">Figure 2C</a>). Protein fractions demonstrating the desired enzyme activity can be analyzed through protein-mass spectrometry and the identity of the enzyme determined by comparison with a predicted peptide database. The formation of thebaine, the first opiate alkaloid in the biosynthesis of codeine and morphine in opium poppy, can occur spontaneously from (7<i>S</i>)-salutaridinol 7-O-acetate. However, functional assays with total soluble protein isolated from opium poppy latex resulted in a 10-fold increase in the formation of thebaine. After protein fractionation and subjecting the active protein fraction to LC-MS/MS proteomics analysis, candidate genes were identified and were then tested through enzymatic assays, leading to the functional characterization of thebaine synthase (Chen et al., <a href="#B10">2018</a>).</p> <p>Most of the aforementioned strategies, such as co-expression analysis, phylogenetics and genome mining, tend to be employed in an integrative way to allow efficient prioritization of candidate genes responsible for metabolic traits of interest. The next step is to verify the identity of the gene through functional assays.</p> </div> <div class="functionalcharacterizationofcandidategenes" title="sec"> <div class="title" tagx="title" title="title">
Functional Characterization of Candidate Genes</div> <p>Two types of experiments are typically required to ascribe a gene to a biosynthetic pathway: enzyme activity assays and reverse genetics gene function validation. The activity of the encoded enzyme must be characterized; this is typically achieved using recombinant expression. However, activity alone is not sufficient to determine <i>in planta</i> function. Here, reverse genetics approaches such as gene silencing are necessary to verify the role of the gene in a pathway.</p> <div class="pathwayelucidationinmicrobialplatforms" title="sec"> <div class="title" tagx="title" title="title">
Pathway Elucidation in Microbial Platforms</div> <p>Recombinant protein expression in <i>Escherichia coli</i> has been extensively used for the enzymatic characterization of biosynthetic genes. The advantages of employing this prokaryotic organism include fast growth kinetics and well-established molecular tools. Protein expression in <i>E. coli</i> is often followed by enzyme purification and <i>in vitro</i> assays. This enables the assessment of the biochemical activities of the candidate enzymes outside the complex cellular context (Caputi et al., <a href="#B9">2018</a>; Torrens-Spence et al., <a href="#B58">2018</a>; Lichman et al., <a href="#B35">2019</a>; Kim et al., <a href="#B26">2020</a>) (<a href="#F3">Figure 3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Overview of the functional characterization methods that can be utilized to determine gene function; enzymatic assays, heterologous expression of candidate biosynthetic genes in microbial host (yeast cells), and heterologous expression <i>in planta</i> (<i>Nicotiana benthamiana</i>).</p>   </div> <p>However, several enzyme families frequently involved in specialized metabolism in plants are membrane-associated, such as cytochrome P450s and membrane-bound prenyltransferases, or require specific eukaryotic-type post-translational modifications. For expression of these proteins, <i>Saccharomyces cerevisiae</i> is preferred over <i>E. coli</i> because of its eukaryotic cell architecture, including the availability of more suitable protein post-translational mechanisms. Microsomal fractions of yeast cells containing active recombinant proteins can be prepared and utilized for subsequent biochemical assays (Levsh et al., <a href="#B30">2019</a>).</p> <p>Alternatively, it is possible to exploit endogenous yeast metabolism and examine the enzymatic activity in an <i>in vivo</i> context, without purification. This method relies on the available precursors and substrates within native yeast pathways (<a href="#F3">Figure 3</a>). This approach may be preferred over <i>in vitro</i> assays if enzyme purification is challenging or higher throughput is required. If the substrate of interest is not available in yeast, an exogenously supplied substrate can be used to build the desired secondary metabolite (Tu et al., <a href="#B60">2020</a>). Another option is to engineer yeast strains through genome engineering (e.g., CRISPR or homologous recombination) to accumulate specific precursors that are not naturally present in yeast (Luo et al., <a href="#B37">2019</a>; Munakata et al., <a href="#B42">2019</a>), or to shutdown endogenous competitive branches that deplete the precursor of interest (Moses et al., <a href="#B41">2014</a>).</p> </div> <div class="pathwayelucidationinnicotianabenthamiana" title="sec"> <div class="title" tagx="title" title="title">
Pathway Elucidation in <i>Nicotiana benthamiana</i></div> <p><i>Nicotiana benthamiana</i> is a practical heterologous expression system for plant natural product biosynthetic pathways (<a href="#F3">Figure 3</a>). It is highly amenable to <i>Agrobacterium</i>-mediated transformation, allowing the transient expression of one or, simultaneously, multiple gene(s) of interest. This plant also provides a cellular context that is similar to the native producer plant. This characteristic allows the expressed proteins to be properly folded and targeted to the correct subcellular compartment, as well as provides a natural supply of chemical precursors; coenzymes, and cofactors.</p> <p>Heterologous expression of candidate genes in <i>N. benthama</i> has led to the discovery of a variety of enzymes underlying the biosynthesis of diverse high value plant-derived compounds (Levsh et al., <a href="#B30">2019</a>; Pluskal et al., <a href="#B47">2019</a>). The plant can also be used to reconstitute whole pathways for preparation of high-value compounds. For example, eight genes from <i>Gloriosa superba</i> involved in the biosynthesis of the alkaloid colchicine were discovered using <i>N. benthamiana</i> as a screening tool. Subsequently, the newly discovered genes plus eight previously described genes were used to engineer <i>N. benthamiana</i> to synthesise <i>N</i>-formyldemecolcine, a colchicine precursor, from phenylalanine and tyrosine (Nett et al., <a href="#B43">2020</a>). The use of <i>N. benthamiana</i> for the heterologous production of whole pathways is discussed further in section Plants as Chasses for Production of High-Value Phytochemicals.</p> </div> <div class="virusinducedgenesilencing" title="sec"> <div class="title" tagx="title" title="title">
Virus Induced Gene Silencing</div> <p>Virus-induced gene silencing (VIGS) is a reverse genetics approach commonly employed for the <i>in vivo</i> functional characterization of candidate biosynthetic genes (Moglia et al., <a href="#B39">2016</a>). VIGS provides direct evidence of the biological role of the gene through a loss-of-function mechanism. The system takes advantage of the plant's homology-dependent defense mechanisms in response to attack by viruses (Hileman et al., <a href="#B21">2005</a>). Inoculating the plant with a construct containing viral RNA and a fragment of the target gene will trigger the degradation or the inhibition of translation of the correspondent RNA, leading to the silencing of the gene of interest. The knock-down of specific genes through this system will reveal their involvement in the biosynthesis of the target natural product. The system was established and used in a variety of plants such as Madagascan periwinkle (<i>Catharanthus roseus</i>) (Qu et al., <a href="#B48">2019</a>) and opium poppy (Chen et al., <a href="#B10">2018</a>). However, a species-specific set up of the method is needed for each newly studied plant by, for instance, identifying suitable virus for the plant and efficient inoculation methods (Courdavault et al., <a href="#B11">2020</a>).</p> </div> </div> </div> <div class="plantsaschassesfortheproductionofhigh-valuephytochemicals" title="sec"> <div class="title" tagx="title" title="title">
Plants as Chasses for the Production of High-Value Phytochemicals</div> <p>Expressing genes in heterologous plant systems not only enables pathway elucidation and gene function determination, but also provides new opportunities to increase and diversify the production of high-value bioactive phytochemicals. As mentioned above, <i>N. benthamiana</i> represents a valuable tool for the heterologous production of phytochemicals. For example, 16 biosynthetic genes from Himalayan mayapple (<i>Podophyllum hexandrum</i>) were transferred to <i>N. benthamiana</i> to produce precursors of the chemotherapeutic drug etoposide at milligram-scale (Schultz et al., <a href="#B55">2019</a>).</p> <p>A major advantage of using <i>N. benthamina</i> as a production platform is the rapidity of the process; no transgenic plants need to be generated. Turnaround times are comparable to microbial systems, with the detection of products possible a few days after agroinfiltration. The process can be scaled up using vacuum agroinfiltration to reach gram-scale production of phytochemicals (Reed et al., <a href="#B50">2017</a>).</p> <div class="optimizingplant-basedproductionofphytochemicals" title="sec"> <div class="title" tagx="title" title="title">
Optimizing Plant-Based Production of Phytochemicals</div> <p>Diverse strategies have been used to enhance and optimize the production of phytochemicals <i>in planta</i>. Among these strategies include overexpressing yield-boosting enzymes such as tHMGR (truncated 3-hydroxy-3-methylglutaryl-CoA reductase, key rate-limiting enzyme in mevalonate pathway) (Reed et al., <a href="#B50">2017</a>) or 1-deoxy-d-xylulose 5-phosphate synthase (DXS), the first committed MEP pathway enzyme (BrÃ¼ckner and Tissier, <a href="#B6">2013</a>). Another method that can potentially lead to increased production is the suppression of competitive pathways through inactivation of endogenous genes by virus-induced gene silencing (Hasan et al., <a href="#B19">2014</a>) or RNA interference (Cankar et al., <a href="#B8">2015</a>).</p> <p>The biosynthesis of natural products in plants is highly compartmentalized with different steps of biosynthetic pathways occurring in different subcellular localization (Heinig et al., <a href="#B20">2013</a>). To enhance the production of phytochemicals of interest, an alternative method is to alter the subcellular location of heterologously expressed enzymes by addition, removal or modification of target peptides (Reed and Osbourn, <a href="#B49">2018</a>). Various studies have shown the potential of this strategy in enhancing the production, with targeting biosynthetic enzymes in different subcellular compartments. Dong et al. (<a href="#B14">2016</a>) reported that the targeting of geraniol synthase (GES) from <i>Valeriana officinalis</i> in the chloroplasts of <i>N. benthamina</i> increased the production of GES products compared to the mitochondrial- and cytosolic-targeted GES.</p> </div> <div class="emergingmetabolicengineeringstrategies" title="sec"> <div class="title" tagx="title" title="title">
Emerging Metabolic Engineering Strategies</div> <p>The ability of plant specialized structures, such as glandular trichomes, to synthesize and store hydrophobic and toxic metabolites has the potential to make engineering the production of these metabolites in plants more advantageous (Huchelmann et al., <a href="#B23">2017</a>). Conditions in trichomes and other structures may even be vital for the formation of the target compound: the final oxidative steps in the biosynthesis of artemisinin are non-enzymatic and require the non-aqueous environment of the subapical cavity of glandular trichomes to proceed (Czechowski et al., <a href="#B12">2016</a>). These findings highlight the challenges of producing specific secondary metabolites in microbial platforms that lack this level of structural complexity.</p> <p>In addition, engineering metabolic pathways in plants is sometimes hindered by toxicity issues and growth defects, due to the cytotoxicity of the engineered compounds or the depleted pools of precursors, necessary for central metabolism. Expressing these pathways in a cell-type specific way, by taking advantage of the separation of trichomes from the rest of the plant, might offer a solution for these issues (Tissier et al., <a href="#B57">2012</a>). Efforts to produce secondary metabolites in trichomes through the use of trichome-specific promoters have been reported, such as; taxadiene in <i>N. sylvestris</i>, and casbene (diterpene from <i>Ricinus communis</i>), in <i>N. tabacum</i> (<a href="#T1">Table 1</a>) (Rontein et al., <a href="#B51">2008</a>; Tissier et al., <a href="#B57">2012</a>).</p> <div class="table-wrap_UNKNOWN" id="T1" position="float"><span class="label" tagx="label" title="label">Table 1</span> <p>Examples of engineered cellular/subcellular localization of phytochemical production in <i>Nicotiana</i> sp.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th valign="top" align="left" rowspan="1" colspan="1"><b>Host plant</b></th> <th valign="top" align="left" rowspan="1" colspan="1"><b>Localization</b></th> <th valign="top" align="left" rowspan="1" colspan="1"><b>Engineering strategy</b></th> <th valign="top" align="left" rowspan="1" colspan="1"><b>Compound class</b></th> <th valign="top" align="left" rowspan="1" colspan="1"><b>Compound accumulation</b></th> <th valign="top" align="left" rowspan="1" colspan="1"><b>References</b></th> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><i>N. benthamiana</i></td> <td valign="top" align="left" rowspan="1" colspan="1">Synthetic hydrophobic organelles</td> <td valign="top" align="left" rowspan="1" colspan="1">Transient expression; fusing terpenoid enzymes to a microalgal lipid droplet surface protein</td> <td valign="top" align="left" rowspan="1" colspan="1">Diterpenoids</td> <td valign="top" align="left" rowspan="1" colspan="1">2.5 fold increase of target diterpenoids/diterpenoids acids</td> <td valign="top" align="left" rowspan="1" colspan="1">Sadre et al., <a href="#B53">2019</a></td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><i>N. benthamiana</i></td> <td valign="top" align="left" rowspan="1" colspan="1">Lipid bodies</td> <td valign="top" align="left" rowspan="1" colspan="1">Transient expression; Co-expression of Î±-bisabolol synthase with fatty acids biosynthesis regulators</td> <td valign="top" align="left" rowspan="1" colspan="1">Sesquiterpenoids</td> <td valign="top" align="left" rowspan="1" colspan="1">2â€"4 fold increase of; Î±-bisabolol, (E)-Î² -caryophyllene and Î±-barbatene.</td> <td valign="top" align="left" rowspan="1" colspan="1">Delatte et al., <a href="#B13">2018</a></td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><i>N. tabacum</i></td> <td valign="top" align="left" rowspan="1" colspan="1">Plastids</td> <td valign="top" align="left" rowspan="1" colspan="1">Stable transformation; plastome and nuclear transformation with artemisinic acid biosynthetic genes and other enzymes known to affect the metabolic flux</td> <td valign="top" align="left" rowspan="1" colspan="1">Sesquiterpenoids (artemisinic acid)</td> <td valign="top" align="left" rowspan="1" colspan="1">Accumulation of more than 120 mg/kg FW of Artemisinic acid.</td> <td valign="top" align="left" rowspan="1" colspan="1">Fuentes et al., <a href="#B18">2016</a></td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><i>N. tabacum</i></td> <td valign="top" align="left" rowspan="1" colspan="1">Glandular trichomes</td> <td valign="top" align="left" rowspan="1" colspan="1">Stable transformation; Casbene synthase/Trichome-specific promoters</td> <td valign="top" align="left" rowspan="1" colspan="1">Diterpenoids (casbene)</td> <td valign="top" align="left" rowspan="1" colspan="1">Accumulation of 1 mg/g FW of casbene</td> <td valign="top" align="left" rowspan="1" colspan="1">Tissier et al., <a href="#B57">2012</a></td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><i>N. sylvestris</i></td> <td valign="top" align="left" rowspan="1" colspan="1">Glandular trichomes</td> <td valign="top" align="left" rowspan="1" colspan="1">Stable transformation; taxadiene synthase/Trichome-specific promoters</td> <td valign="top" align="left" rowspan="1" colspan="1">Diterpenoids (Taxadiene)</td> <td valign="top" align="left" rowspan="1" colspan="1">Accumulation of 100 Î¼g/g FW of taxadiene</td> <td valign="top" align="left" rowspan="1" colspan="1">Tissier et al., <a href="#B57">2012</a></td> </tr> </tbody> </table> </div> <p>Engineering plastids in order to optimize the yields of valuable phytochemicals has been explored. Plastid engineering has various advantages over nuclear genome engineering including an efficient homologous recombination machinery, as well as the potential for greater gene expression levels (Bock, <a href="#B4">2015</a>; Boehm and Bock, <a href="#B5">2019</a>). To enhance the yield of artemisinic acid production (the precursor for the anti-malaria drug: artemisinin), a synthetic biology strategy has been developed combining plastome and nuclear transformation (Fuentes et al., <a href="#B18">2016</a>). The artemisinic acid metabolic pathway was introduced into the chloroplast genome of <i>N. tabacum</i> plants. Subsequently, enzymes known to affect the metabolic flux were introduced through nuclear transformation (<a href="#T1">Table 1</a>). This strategy led to identifying plants that produce high amounts of artemisinic acid (more than 120 mg of per kg biomass) (Fuentes et al., <a href="#B18">2016</a>; Jensen and Scharff, <a href="#B24">2019</a>).</p> <p>A newly emerging metabolic engineering strategy is engineering synthetic hydrophobic droplets within cells that enable the accumulation and storage of lipophilic compounds such as terpenoids. For example, the synthesis of lipid droplets in transient <i>N. benthamiana</i> system was enhanced through the ectopic production of a regulator of plastid fatty acid biosynthesis and a microalgal lipid droplet surface protein. Biosynthetic steps were anchored onto the surface of the lipid droplets and high-value sesqui- or diterpenoids were efficiently produced. These engineered lipid droplets may potentially facilitate terpenoid extraction from the plant, through â€œtrappingâ€� of the target molecules in the oil fraction (<a href="#T1">Table 1</a>) (Sadre et al., <a href="#B53">2019</a>). Another study demonstrated the potential of lipid bodies as hydrophobic storage organelle for sesquiterpenes molecules such as Î±-bisabolol (Delatte et al., <a href="#B13">2018</a>).</p> <p>CRISPR and Cas9-associated protein systems are emerging as powerful tools to study gene function and to improve specific traits in plant species. Knocking out biosynthetic genes using CRISPR/Cas9 system has been reported in <i>Salvia miltiorrhiza</i> (tanshinone, diterpenoid) (Li et al., <a href="#B31">2017</a>) and <i>Papaver somniferum</i> L. (benzylisoquinoline alkaloids) (Alagoz et al., <a href="#B1">2016</a>). Setting up CRISPR/Cas9 systems in diverse plant species will be useful for gene function elucidation studies and will also guide future efforts to improve the yield of phytochemicals of interest in valuable medicinal plants. CRISPR/Cas9 systems also have the potential to shut down competing side branches of metabolic pathways within plants, leading to improved yields of the target compound.</p> </div> <div class="combinatorialbiosynthesis" title="sec"> <div class="title" tagx="title" title="title">
Combinatorial Biosynthesis</div> <p>Engineering new-to-nature metabolic pathways in plants offers the possibility to expand the natural chemical diversity of plants and create a subset of novel chemical structures with new or improved bioactivities. Combinatorial biosynthesis is an engineering strategy, based on combining biosynthetic genes from different sources in a single host, thereby establishing new enzymeâ€"substrate combinations, that enables the generation of a suite of related compounds including many that are new-to-nature (Arendt et al., <a href="#B3">2016</a>). This strategy takes advantage of the hypothesis that enzymes involved in plant natural product biosynthesis have inherent relaxed substrate specificity and are often able to transform non-native substrates. Different studies aiming to explore the potential of this approach in creating novel compounds have been reported (<a href="#T2">Table 2</a>).</p> <div class="table-wrap_UNKNOWN" id="T2" position="float"><span class="label" tagx="label" title="label">Table 2</span> <p>Examples of novel compounds produced through combinatorial biosynthesis using transient expression in <i>N. benthamiana</i>.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th valign="top" align="left" rowspan="1" colspan="1"><b>Source of enzymes</b></th> <th valign="top" align="left" rowspan="1" colspan="1"><b>Enzymes</b></th> <th valign="top" align="left" rowspan="1" colspan="1"><b>Compound class</b></th> <th valign="top" align="left" rowspan="1" colspan="1"><b>Obtained compounds</b></th> <th valign="top" align="left" rowspan="1" colspan="1"><b>References</b></th> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><i>Tanacetum parthenium/Artemisia annua</i></td> <td valign="top" align="left" rowspan="1" colspan="1">Germacrene A synthase (GAS) Germacrene A oxidase (GAO) <br /><br /> Costunolide synthase (COS) <br /><br /> Parthenolide synthase (PTS) <br /><br /> aldehyde Î"11(13) double bond reductase2 (DBR2)</td> <td valign="top" align="left" rowspan="1" colspan="1">Sesquiterpenoids</td> <td valign="top" align="left" rowspan="1" colspan="1">Costunolide and parthenolide derivatives</td> <td valign="top" align="left" rowspan="1" colspan="1">Kashkooli Beyraghdar et al., <a href="#B25">2019</a></td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Oat, licorice, soy bean, barrel clover</td> <td valign="top" align="left" rowspan="1" colspan="1">Î²-amyrin synthase and Î²-amyrin-oxidizing P450s</td> <td valign="top" align="left" rowspan="1" colspan="1">Triterpenoids</td> <td valign="top" align="left" rowspan="1" colspan="1">Î²-amyrin derivatives</td> <td valign="top" align="left" rowspan="1" colspan="1">Reed et al., <a href="#B50">2017</a></td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Various species</td> <td valign="top" align="left" rowspan="1" colspan="1">Class I/II diterpene synthases</td> <td valign="top" align="left" rowspan="1" colspan="1">Diterpenoids</td> <td valign="top" align="left" rowspan="1" colspan="1">Diverse diterpene skeletons</td> <td valign="top" align="left" rowspan="1" colspan="1">Andersen-Ranberg et al., <a href="#B2">2016</a></td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><i>Setaria italica, Sorghum bicolor, Lotus japonicus, Brassica rapa</i></td> <td valign="top" align="left" rowspan="1" colspan="1">Cytochrome P450s <br /><br /> Glucosinolate biosynthetic genes</td> <td valign="top" align="left" rowspan="1" colspan="1">Isothiocyanate</td> <td valign="top" align="left" rowspan="1" colspan="1">Crucifalexins</td> <td valign="top" align="left" rowspan="1" colspan="1">Calgaro-Kozina et al., <a href="#B7">2020</a></td> </tr> </tbody> </table> </div> <p>Co-agroinfiltration of <i>N. benthamiana</i> with different expression constructs is an efficient method to test numerous enzyme combinations (Reed and Osbourn, <a href="#B49">2018</a>). This approach was used to generate novel Î²-amyrin derivatives through the expression of combinations of Î²-amyrin synthase and Î²-amyrin-oxidizing P450s from various plant species (Reed et al., <a href="#B50">2017</a>). In another study, the stereoselective biosynthesis of over 50 diterpene skeletons including natural variants and novel compounds was achieved <i>via</i> transient expression of combinations of class I and II diterpene synthases in <i>N. benthamiana</i> (Andersen-Ranberg et al., <a href="#B2">2016</a>). These terpene synthases were isolated from seven different diterpenoids producing plant species.</p> <p>The generation of costunolide and parthenolide derivatives was achieved by applying a combinatorial metabolic engineering approach expressing different enzymes involved in sesquiterpenoids biosynthesis from <i>Tanacetum parthenium</i> and <i>Artemisia annua</i> (<a href="#T2">Table 2</a>) (Kashkooli Beyraghdar et al., <a href="#B25">2019</a>). In a recent study, diverse cytochrome P450s from <i>Setaria italica, Sorghum bicolor, Lotus japonicus</i>, as well as glucosinolate biosynthetic genes from <i>Brassica rapa</i>, were heterologously expressed in <i>N. benthamiana</i>. This expression generated new-to-nature compounds with potent antifungal activity, called crucifalexins. This report showed that by broadening the set of primary metabolites that can be utilized by the core biosynthetic pathway of brassinin (isothiocyanate), it is possible to synthesize novel molecules with enhanced proprieties (Calgaro-Kozina et al., <a href="#B7">2020</a>).</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Phytochemicals were foundational to the emergence of organic chemistry and the early development of pharmaceuticals. The structural complexity and bioactivity of phytochemicals will ensure their continued relevance in synthetic chemistry and drug discovery. With the maturity of genetic and biochemical tools for gene discovery, characterization and engineering, we are entering into a new synthetic era of phytochemistry, in which biosynthetic pathways are elucidated and high-value phytochemicals can be produced in synthetic biological chasses.</p> <p>An integrative approach to gene prioritization, combining genomic transcriptomic and metabolomics, has become an efficient method for the identification of candidate biosynthetic genes. Characterization of genes through <i>in vitro</i> assays, coupled with <i>in vivo</i> validation through RNAi or VIGS, subsequently enables verification of candidate gene function. Once the pathway is uncovered, the heterologous <i>in planta</i> expression of the biosynthetic pathway is made possible, with many emerging engineering strategies available to enhance or diversify the production of the target compounds. These advances open the door to a future where diverse phytochemicals with new bioactivities and unusual structures can be routinely produced and purified from heterologous plant systems, aiding drug discovery efforts and leading to the scalable and sustainable production of valuable phytochemicals.</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>KE conceived of the manuscript and wrote the first draft. Both authors prepared figures, edited text, and approved the manuscript for submission.</p> </div> <div class="conflictofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Funding.</b> This work was supported by a UK Research and Innovation Future Leaders Fellowship to BRL (MR/S01862X/1).</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alagoz</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurkok</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Unver</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Manipulating the biosynthesis of bioactive compound alkaloids for next-generation metabolic engineering in opium poppy Using CRISPR-Cas 9 genome editing technology</span>. <span class="source" tagx="source" title="source">Sci. Rep.</span><span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">30910</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/srep30910">10.1038/srep30910</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27483984">27483984</a></span></span></li> <li tag="ref"><a name="B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen-Ranberg</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kongstad</span><span class="given-names" tagx="given-names" title="given-names">K. T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">M. T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">N. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pateraki</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bach</span><span class="given-names" tagx="given-names" title="given-names">S. S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Expanding the landscape of diterpene structural diversity through stereochemically controlled combinatorial biosynthesis</span>. <span class="source" tagx="source" title="source">Angew. Chemie Int. Ed.</span><span class="volume" tagx="volume" title="volume">55</span>, <span class="fpage" tagx="fpage" title="fpage">2142</span>â€"<span class="lpage" tagx="lpage" title="lpage">2146</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/anie.201510650">10.1002/anie.201510650</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26749264">26749264</a></span></span></li> <li tag="ref"><a name="B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arendt</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pollier</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callewaert</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goossens</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Synthetic biology for production of natural and new-to-nature terpenoids in photosynthetic organisms</span>. <span class="source" tagx="source" title="source">Plant J.</span><span class="volume" tagx="volume" title="volume">87</span>, <span class="fpage" tagx="fpage" title="fpage">16</span>â€"<span class="lpage" tagx="lpage" title="lpage">37</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/tpj.13138">10.1111/tpj.13138</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26867713">26867713</a></span></span></li> <li tag="ref"><a name="B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bock</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Engineering plastid genomes: methods, tools, and applications in basic research and biotechnology</span>. <span class="source" tagx="source" title="source">Annu. Rev. Plant Biol.</span><span class="volume" tagx="volume" title="volume">66</span>, <span class="fpage" tagx="fpage" title="fpage">211</span>â€"<span class="lpage" tagx="lpage" title="lpage">241</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-arplant-050213-040212">10.1146/annurev-arplant-050213-040212</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25494465">25494465</a></span></span></li> <li tag="ref"><a name="B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boehm</span><span class="given-names" tagx="given-names" title="given-names">C. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bock</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Recent advances and current challenges in synthetic biology of the plastid genetic system and metabolism</span>. <span class="source" tagx="source" title="source">Plant Physiol.</span><span class="volume" tagx="volume" title="volume">179</span>, <span class="fpage" tagx="fpage" title="fpage">794</span>â€"<span class="lpage" tagx="lpage" title="lpage">802</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.18.00767">10.1104/pp.18.00767</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30181342">30181342</a></span></span></li> <li tag="ref"><a name="B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BrÃ¼ckner</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tissier</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">High-level diterpene production by transient expression in <i>Nicotiana benthamiana</i></span>. <span class="source" tagx="source" title="source">Plant Methods</span><span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">46</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1746-4811-9-46">10.1186/1746-4811-9-46</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24330621">24330621</a></span></span></li> <li tag="ref"><a name="B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calgaro-Kozina</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vuu</span><span class="given-names" tagx="given-names" title="given-names">K. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keasling</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LoquÃ©</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sattely</span><span class="given-names" tagx="given-names" title="given-names">E. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shih</span><span class="given-names" tagx="given-names" title="given-names">P. M.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Engineering plant synthetic pathways for the biosynthesis of novel antifungals</span>. <span class="source" tagx="source" title="source">ACS Cent. Sci.</span><span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">1394</span>â€"<span class="lpage" tagx="lpage" title="lpage">1400</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acscentsci.0c00241">10.1021/acscentsci.0c00241</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32875080">32875080</a></span></span></li> <li tag="ref"><a name="B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cankar</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jongedijk</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klompmaker</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Majdic</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mumm</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouwmeester</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">(+)-Valencene production in <i>Nicotiana benthamiana</i> is increased by down-regulation of competing pathways</span>. <span class="source" tagx="source" title="source">Biotechnol. J.</span><span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">180</span>â€"<span class="lpage" tagx="lpage" title="lpage">189</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/biot.201400288">10.1002/biot.201400288</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25159317">25159317</a></span></span></li> <li tag="ref"><a name="B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caputi</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franke</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farrow</span><span class="given-names" tagx="given-names" title="given-names">S. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Payne</span><span class="given-names" tagx="given-names" title="given-names">R. M. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">T. D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Missing enzymes in the biosynthesis of the anticancer drug vinblastine in Madagascar periwinkle</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">360</span>, <span class="fpage" tagx="fpage" title="fpage">1235</span>â€"<span class="lpage" tagx="lpage" title="lpage">1239</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aat4100">10.1126/science.aat4100</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29724909">29724909</a></span></span></li> <li tag="ref"><a name="B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hagel</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tucker</span><span class="given-names" tagx="given-names" title="given-names">J. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shiigi</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yelpaala</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">A pathogenesis-related 10 protein catalyzes the final step in thebaine biosynthesis article</span>. <span class="source" tagx="source" title="source">Nat. Chem. Biol.</span><span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">738</span>â€"<span class="lpage" tagx="lpage" title="lpage">743</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41589-018-0059-7">10.1038/s41589-018-0059-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29807982">29807982</a></span></span></li> <li tag="ref"><a name="B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Courdavault</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Besseau</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oudin</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papon</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Connor</span><span class="given-names" tagx="given-names" title="given-names">S. E.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Virus-induced gene silencing: hush genes to make them talk</span>. <span class="source" tagx="source" title="source">Trends Plant Sci.</span><span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">714</span>â€"<span class="lpage" tagx="lpage" title="lpage">715</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tplants.2020.02.013">10.1016/j.tplants.2020.02.013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32526175">32526175</a></span></span></li> <li tag="ref"><a name="B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Czechowski</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larson</span><span class="given-names" tagx="given-names" title="given-names">T. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Catania</span><span class="given-names" tagx="given-names" title="given-names">T. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harvey</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">G. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">I. A.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title"><i>Artemisia annua</i> mutant impaired in Artemisinin synthesis demonstrates importance of nonenzymatic conversion in terpenoid metabolism</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span><span class="volume" tagx="volume" title="volume">113</span>, <span class="fpage" tagx="fpage" title="fpage">15150</span>â€"<span class="lpage" tagx="lpage" title="lpage">15155</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1611567113">10.1073/pnas.1611567113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27930305">27930305</a></span></span></li> <li tag="ref"><a name="B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delatte</span><span class="given-names" tagx="given-names" title="given-names">T. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scaiola</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Molenaar</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Sousa Farias</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alves Gomes Albertti</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Busscher</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Engineering storage capacity for volatile sesquiterpenes in <i>Nicotiana benthamiana</i> leaves</span>. <span class="source" tagx="source" title="source">Plant Biotechnol. J.</span><span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">1997</span>â€"<span class="lpage" tagx="lpage" title="lpage">2006</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/pbi.12933">10.1111/pbi.12933</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29682901">29682901</a></span></span></li> <li tag="ref"><a name="B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jongedijk</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouwmeester</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Der Krol</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Monoterpene biosynthesis potential of plant subcellular compartments</span>. <span class="source" tagx="source" title="source">New Phytol.</span><span class="volume" tagx="volume" title="volume">209</span>, <span class="fpage" tagx="fpage" title="fpage">679</span>â€"<span class="lpage" tagx="lpage" title="lpage">690</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/nph.13629">10.1111/nph.13629</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26356766">26356766</a></span></span></li> <li tag="ref"><a name="B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duke</span><span class="given-names" tagx="given-names" title="given-names">M. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">R. N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elsohly</span><span class="given-names" tagx="given-names" title="given-names">H. N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sturtz</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duke</span><span class="given-names" tagx="given-names" title="given-names">S. O.</span></span></span> (<span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Localization of artemisinin and artemisitene in foliar tissues of glanded and glandless biotypes of <i>Artemisia annua</i> L</span>. <span class="source" tagx="source" title="source">Int. J. Plant Sci.</span><span class="volume" tagx="volume" title="volume">155</span>, <span class="fpage" tagx="fpage" title="fpage">365</span>â€"<span class="lpage" tagx="lpage" title="lpage">372</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1086/297173">10.1086/297173</a></span></span></li> <li tag="ref"><a name="B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Facchini</span><span class="given-names" tagx="given-names" title="given-names">P. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Luca</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Phloem-specific expression of tyrosine/dopa decarboxylase genes and the biosynthesis of isoquinoline alkaloids in opium poppy</span>. <span class="source" tagx="source" title="source">Plant Cell</span><span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">1811</span>â€"<span class="lpage" tagx="lpage" title="lpage">1821</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2307/3870189">10.2307/3870189</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12242361">12242361</a></span></span></li> <li tag="ref"><a name="B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freund</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hegeman</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Recent advances in stable isotope-enabled mass spectrometry-based plant metabolomics</span>. <span class="source" tagx="source" title="source">Curr. Opin. Biotechnol.</span><span class="volume" tagx="volume" title="volume">43</span>, <span class="fpage" tagx="fpage" title="fpage">41</span>â€"<span class="lpage" tagx="lpage" title="lpage">48</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.copbio.2016.08.002">10.1016/j.copbio.2016.08.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27610928">27610928</a></span></span></li> <li tag="ref"><a name="B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuentes</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erban</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karcher</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kopka</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bock</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">A new synthetic biology approach allows transfer of an entire metabolic pathway from a medicinal plant to a biomass crop</span>. <span class="source" tagx="source" title="source">Elife</span><span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">e13664</span>. <span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.13664.020">10.7554/eLife.13664.020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27296645">27296645</a></span></span></li> <li tag="ref"><a name="B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasan</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">H. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeon</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moon</span><span class="given-names" tagx="given-names" title="given-names">B. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J. Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Metabolic engineering of <i>Nicotiana benthamiana</i> for the increased production of taxadiene</span>. <span class="source" tagx="source" title="source">Plant Cell Rep.</span><span class="volume" tagx="volume" title="volume">33</span>, <span class="fpage" tagx="fpage" title="fpage">895</span>â€"<span class="lpage" tagx="lpage" title="lpage">904</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00299-014-1568-9">10.1007/s00299-014-1568-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24463610">24463610</a></span></span></li> <li tag="ref"><a name="B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heinig</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gutensohn</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudareva</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aharoni</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">The challenges of cellular compartmentalization in plant metabolic engineering</span>. <span class="source" tagx="source" title="source">Curr. Opin. Biotechnol.</span><span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">239</span>â€"<span class="lpage" tagx="lpage" title="lpage">246</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.copbio.2012.11.006">10.1016/j.copbio.2012.11.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23246154">23246154</a></span></span></li> <li tag="ref"><a name="B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hileman</span><span class="given-names" tagx="given-names" title="given-names">L. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drea</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Martino</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Litt</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irish</span><span class="given-names" tagx="given-names" title="given-names">V. F.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Virus-induced gene silencing is an effective tool for assaying gene function in the basal eudicot species <i>Papaver somniferum</i> (opium poppy)</span>. <span class="source" tagx="source" title="source">Plant J.</span><span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">334</span>â€"<span class="lpage" tagx="lpage" title="lpage">341</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-313X.2005.02520.x">10.1111/j.1365-313X.2005.02520.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16212610">16212610</a></span></span></li> <li tag="ref"><a name="B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodgson</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De La PeÃ±a</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephenson</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thimmappa</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sattely</span><span class="given-names" tagx="given-names" title="given-names">E. S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of key enzymes responsible for protolimonoid biosynthesis in plants: opening the door to azadirachtin production</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span><span class="volume" tagx="volume" title="volume">116</span>, <span class="fpage" tagx="fpage" title="fpage">17096</span>â€"<span class="lpage" tagx="lpage" title="lpage">17104</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1906083116">10.1073/pnas.1906083116</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31371503">31371503</a></span></span></li> <li tag="ref"><a name="B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huchelmann</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boutry</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hachez</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Plant glandular trichomes: natural cell factories of high biotechnological interest</span>. <span class="source" tagx="source" title="source">Plant Physiol.</span><span class="volume" tagx="volume" title="volume">175</span>, <span class="fpage" tagx="fpage" title="fpage">6</span>â€"<span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.17.00727">10.1104/pp.17.00727</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28724619">28724619</a></span></span></li> <li tag="ref"><a name="B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">P. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scharff</span><span class="given-names" tagx="given-names" title="given-names">L. B.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Engineering of plastids to optimize the production of high-value metabolites and proteins</span>. <span class="source" tagx="source" title="source">Curr. Opin. Biotechnol.</span><span class="volume" tagx="volume" title="volume">59</span>, <span class="fpage" tagx="fpage" title="fpage">8</span>â€"<span class="lpage" tagx="lpage" title="lpage">15</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.copbio.2019.01.009">10.1016/j.copbio.2019.01.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30798145">30798145</a></span></span></li> <li tag="ref"><a name="B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kashkooli Beyraghdar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Krol</span><span class="given-names" tagx="given-names" title="given-names">A. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabe</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dickschat</span><span class="given-names" tagx="given-names" title="given-names">J. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouwmeester</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Substrate promiscuity of enzymes from the sesquiterpene biosynthetic pathways from <i>Artemisia annua</i> and <i>Tanacetum parthenium</i> allows for novel combinatorial sesquiterpene production</span>. <span class="source" tagx="source" title="source">Metab. Eng.</span><span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">12</span>â€"<span class="lpage" tagx="lpage" title="lpage">23</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ymben.2019.01.007">10.1016/j.ymben.2019.01.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30822491">30822491</a></span></span></li> <li tag="ref"><a name="B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">C. Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glinkerman</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pluskal</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weng</span><span class="given-names" tagx="given-names" title="given-names">J. K.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">The chloroalkaloid (â€")-acutumine is biosynthesized <i>via</i> a Fe(II)- and 2-oxoglutarate-dependent halogenase in Menispermaceae plants</span>. <span class="source" tagx="source" title="source">Nat. Commun.</span><span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">1867</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-020-15777-w">10.1038/s41467-020-15777-w</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32313070">32313070</a></span></span></li> <li tag="ref"><a name="B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lechner</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunk</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keasling</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The need for integrated approaches in metabolic engineering</span>. <span class="source" tagx="source" title="source">Cold Spring Harb. Perspect. Biol.</span><span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">a023903</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/cshperspect.a023903">10.1101/cshperspect.a023903</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27527588">27527588</a></span></span></li> <li tag="ref"><a name="B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">E. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Facchini</span><span class="given-names" tagx="given-names" title="given-names">P. J.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Tyrosine aminotransferase contributes to benzylisoquinoline alkaloid biosynthesis in opium poppy</span>. <span class="source" tagx="source" title="source">Plant Physiol.</span><span class="volume" tagx="volume" title="volume">157</span>, <span class="fpage" tagx="fpage" title="fpage">1067</span>â€"<span class="lpage" tagx="lpage" title="lpage">1078</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.111.185512">10.1104/pp.111.185512</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21949209">21949209</a></span></span></li> <li tag="ref"><a name="B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">E. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Y. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoo</span><span class="given-names" tagx="given-names" title="given-names">Y. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">S. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amir</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Cultured cambial meristematic cells as a source of plant natural products</span>. <span class="source" tagx="source" title="source">Nat. Biotechnol.</span><span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">1213</span>â€"<span class="lpage" tagx="lpage" title="lpage">1217</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.1693">10.1038/nbt.1693</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972422">20972422</a></span></span></li> <li tag="ref"><a name="B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levsh</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pluskal</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carballo</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weng</span><span class="given-names" tagx="given-names" title="given-names">J. K.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Independent evolution of rosmarinic acid biosynthesis in two sister families under the Lamiids clade of flowering plants</span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="volume" tagx="volume" title="volume">294</span>, <span class="fpage" tagx="fpage" title="fpage">15193</span>â€"<span class="lpage" tagx="lpage" title="lpage">15205</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.RA119.010454">10.1074/jbc.RA119.010454</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31481469">31481469</a></span></span></li> <li tag="ref"><a name="B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhan</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Targeted mutagenesis in the medicinal plant <i>Salvia miltiorrhiza</i></span>. <span class="source" tagx="source" title="source">Sci. Rep.</span><span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">43320</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/srep43320">10.1038/srep43320</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28256553">28256553</a></span></span></li> <li tag="ref"><a name="B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smolke</span><span class="given-names" tagx="given-names" title="given-names">C. D.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Strategies for microbial synthesis of high-value phytochemicals</span>. <span class="source" tagx="source" title="source">Nat. Chem.</span><span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">395</span>â€"<span class="lpage" tagx="lpage" title="lpage">404</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41557-018-0013-z">10.1038/s41557-018-0013-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29568052">29568052</a></span></span></li> <li tag="ref"><a name="B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lichman</span><span class="given-names" tagx="given-names" title="given-names">B. R.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">The scaffold-forming steps of plant alkaloid biosynthesis</span>. <span class="source" tagx="source" title="source">Nat. Prod. Rep</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/D0NP00031K">10.1039/D0NP00031K</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32745157">32745157</a></span></span></li> <li tag="ref"><a name="B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lichman</span><span class="given-names" tagx="given-names" title="given-names">B. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Godden</span><span class="given-names" tagx="given-names" title="given-names">G. T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamilton</span><span class="given-names" tagx="given-names" title="given-names">J. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palmer</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamileen</span><span class="given-names" tagx="given-names" title="given-names">M. O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">The evolutionary origins of the cat attractant nepetalactone in catnip</span>. <span class="source" tagx="source" title="source">Sci. Adv.</span><span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">eaba0721</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/sciadv.aba0721">10.1126/sciadv.aba0721</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32426505">32426505</a></span></span></li> <li tag="ref"><a name="B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lichman</span><span class="given-names" tagx="given-names" title="given-names">B. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamileen</span><span class="given-names" tagx="given-names" title="given-names">M. O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Titchiner</span><span class="given-names" tagx="given-names" title="given-names">G. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevenson</span><span class="given-names" tagx="given-names" title="given-names">C. E. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawson</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Connor</span><span class="given-names" tagx="given-names" title="given-names">S. E. O.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Uncoupled activation and cyclization in catmint reductive terpenoid biosynthesis</span>. <span class="source" tagx="source" title="source">Nat. Chem. Biol</span>. <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">71</span>â€"<span class="lpage" tagx="lpage" title="lpage">79</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41589-018-0185-2">10.1038/s41589-018-0185-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30531909">30531909</a></span></span></li> <li tag="ref"><a name="B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suarez Duran</span><span class="given-names" tagx="given-names" title="given-names">H. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harnvanichvech</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephenson</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schranz</span><span class="given-names" tagx="given-names" title="given-names">M. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Drivers of metabolic diversification: how dynamic genomic neighbourhoods generate new biosynthetic pathways in the Brassicaceae</span>. <span class="source" tagx="source" title="source">New Phytol.</span><span class="volume" tagx="volume" title="volume">227</span>, <span class="fpage" tagx="fpage" title="fpage">1109</span>â€"<span class="lpage" tagx="lpage" title="lpage">1123</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/nph.16338">10.1111/nph.16338</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31769874">31769874</a></span></span></li> <li tag="ref"><a name="B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reiter</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">d'Espaux</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Denby</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lechner</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Complete biosynthesis of cannabinoids and their unnatural analogues in yeast</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">567</span>, <span class="fpage" tagx="fpage" title="fpage">123</span>â€"<span class="lpage" tagx="lpage" title="lpage">126</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-019-0978-9">10.1038/s41586-019-0978-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30814733">30814733</a></span></span></li> <li tag="ref"><a name="B38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weng</span><span class="given-names" tagx="given-names" title="given-names">J. K.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Unleashing the synthetic power of plant oxygenases: from mechanism to application</span>. <span class="source" tagx="source" title="source">Plant Physiol.</span><span class="volume" tagx="volume" title="volume">179</span>, <span class="fpage" tagx="fpage" title="fpage">813</span>â€"<span class="lpage" tagx="lpage" title="lpage">829</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.18.01223">10.1104/pp.18.01223</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30670605">30670605</a></span></span></li> <li tag="ref"><a name="B39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moglia</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Acquadro</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eljounaidi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milani</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cagliero</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rubiolo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification of BAHD acyltransferases and <i>in vivo</i> characterization of HQT-like enzymes involved in caffeoylquinic acid synthesis in globe artichoke</span>. <span class="source" tagx="source" title="source">Front. Plant Sci.</span><span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">1424</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2016.01424">10.3389/fpls.2016.01424</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27721818">27721818</a></span></span></li> <li tag="ref"><a name="B40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moses</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mehrshahi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">A. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goossens</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Synthetic biology approaches for the production of plant metabolites in unicellular organisms</span>. <span class="source" tagx="source" title="source">J. Exp. Bot.</span><span class="volume" tagx="volume" title="volume">68</span>, <span class="fpage" tagx="fpage" title="fpage">4057</span>â€"<span class="lpage" tagx="lpage" title="lpage">4074</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/erx119">10.1093/jxb/erx119</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28449101">28449101</a></span></span></li> <li tag="ref"><a name="B41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moses</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pollier</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almagro</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buyst</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Montagu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PedreÃ±o</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Combinatorial biosynthesis of sapogenins and saponins in <i>Saccharomyces cerevisiae</i> using a C-16Î± hydroxylase from <i>Bupleurum falcatum</i></span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span><span class="volume" tagx="volume" title="volume">111</span>, <span class="fpage" tagx="fpage" title="fpage">1634</span>â€"<span class="lpage" tagx="lpage" title="lpage">1639</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1323369111">10.1073/pnas.1323369111</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24434554">24434554</a></span></span></li> <li tag="ref"><a name="B42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munakata</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takemura</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatsumi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moriyoshi</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yanagihara</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugiyama</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation of <i>Artemisia capillaris</i> membrane-bound di-prenyltransferase for phenylpropanoids and redesign of artepillin C in yeast</span>. <span class="source" tagx="source" title="source">Commun. Biol.</span><span class="volume" tagx="volume" title="volume">2</span>:<span class="fpage" tagx="fpage" title="fpage">384</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s42003-019-0630-0">10.1038/s42003-019-0630-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31646187">31646187</a></span></span></li> <li tag="ref"><a name="B43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nett</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sattely</span><span class="given-names" tagx="given-names" title="given-names">E. S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Discovery and engineering of colchicine alkaloid biosynthesis</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">584</span>, <span class="fpage" tagx="fpage" title="fpage">148</span>â€"<span class="lpage" tagx="lpage" title="lpage">153</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2546-8">10.1038/s41586-020-2546-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32699417">32699417</a></span></span></li> <li tag="ref"><a name="B44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicolaou</span><span class="given-names" tagx="given-names" title="given-names">K. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigol</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Perspectives from nearly five decades of total synthesis of natural products and their analogues for biology and medicine</span>. <span class="source" tagx="source" title="source">Nat. Prod. Rep.</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/D0NP00003E">10.1039/D0NP00003E</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32319494">32319494</a></span></span></li> <li tag="ref"><a name="B45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">NÃ¼tzmann</span><span class="given-names" tagx="given-names" title="given-names">H.-W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scazzocchio</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osbourn</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Metabolic gene clusters in eukaryotes</span>. <span class="source" tagx="source" title="source">Annu. Rev. Genet</span>. <span class="volume" tagx="volume" title="volume">52</span>, <span class="fpage" tagx="fpage" title="fpage">159</span>â€"<span class="lpage" tagx="lpage" title="lpage">183</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-genet-120417-031237">10.1146/annurev-genet-120417-031237</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30183405">30183405</a></span></span></li> <li tag="ref"><a name="B46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">NÃ¼tzmann</span><span class="given-names" tagx="given-names" title="given-names">H. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osbourn</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Plant metabolic clusters â€" from genetics to genomics</span>. <span class="source" tagx="source" title="source">New Phytol</span>. <span class="volume" tagx="volume" title="volume">211</span>, <span class="fpage" tagx="fpage" title="fpage">771</span>â€"<span class="lpage" tagx="lpage" title="lpage">789</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/nph.13981">10.1111/nph.13981</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27112429">27112429</a></span></span></li> <li tag="ref"><a name="B47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pluskal</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torrens-Spence</span><span class="given-names" tagx="given-names" title="given-names">M. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fallon</span><span class="given-names" tagx="given-names" title="given-names">T. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Abreu</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">C. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weng</span><span class="given-names" tagx="given-names" title="given-names">J. K.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">The biosynthetic origin of psychoactive kavalactones in kava</span>. <span class="source" tagx="source" title="source">Nat. Plants</span><span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">867</span>â€"<span class="lpage" tagx="lpage" title="lpage">878</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41477-019-0474-0">10.1038/s41477-019-0474-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31332312">31332312</a></span></span></li> <li tag="ref"><a name="B48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Safonova</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Luca</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Completion of the canonical pathway for assembly of anticancer drugs vincristine/vinblastine in <i>Catharanthus roseus</i></span>. <span class="source" tagx="source" title="source">Plant J.</span><span class="volume" tagx="volume" title="volume">97</span>, <span class="fpage" tagx="fpage" title="fpage">257</span>â€"<span class="lpage" tagx="lpage" title="lpage">266</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/tpj.14111">10.1111/tpj.14111</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30256480">30256480</a></span></span></li> <li tag="ref"><a name="B49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reed</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osbourn</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Engineering terpenoid production through transient expression in <i>Nicotiana benthamiana</i></span>. <span class="source" tagx="source" title="source">Plant Cell Rep.</span><span class="volume" tagx="volume" title="volume">37</span>, <span class="fpage" tagx="fpage" title="fpage">1431</span>â€"<span class="lpage" tagx="lpage" title="lpage">1441</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00299-018-2296-3">10.1007/s00299-018-2296-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29786761">29786761</a></span></span></li> <li tag="ref"><a name="B50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reed</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephenson</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miettinen</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brouwer</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leveau</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brett</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">A translational synthetic biology platform for rapid access to gram-scale quantities of novel drug-like molecules</span>. <span class="source" tagx="source" title="source">Metab. Eng.</span><span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">185</span>â€"<span class="lpage" tagx="lpage" title="lpage">193</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ymben.2017.06.012">10.1016/j.ymben.2017.06.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28687337">28687337</a></span></span></li> <li tag="ref"><a name="B51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rontein</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Onillon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herbette</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lesot</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Werck-Reichhart</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sallaud</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">CYP725A4 from yew catalyzes complex structural rearrangement of taxa-4(5),11(12)-diene into the cyclic ether 5(12)-oxa-3(11)-cyclotaxane</span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="volume" tagx="volume" title="volume">283</span>, <span class="fpage" tagx="fpage" title="fpage">6067</span>â€"<span class="lpage" tagx="lpage" title="lpage">6075</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M708950200">10.1074/jbc.M708950200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18167342">18167342</a></span></span></li> <li tag="ref"><a name="B52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roselli</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olry</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vautrin</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coriton</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ritchie</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galati</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">A bacterial artificial chromosome (BAC) genomic approach reveals partial clustering of the furanocoumarin pathway genes in parsnip</span>. <span class="source" tagx="source" title="source">Plant J.</span><span class="volume" tagx="volume" title="volume">89</span>, <span class="fpage" tagx="fpage" title="fpage">1119</span>â€"<span class="lpage" tagx="lpage" title="lpage">1132</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/tpj.13450">10.1111/tpj.13450</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27943460">27943460</a></span></span></li> <li tag="ref"><a name="B53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sadre</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banerjee</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benning</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Cytosolic lipid droplets as engineered organelles for production and accumulation of terpenoid biomaterials in leaves</span>. <span class="source" tagx="source" title="source">Nat. Commun.</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">853</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-019-08515-4">10.1038/s41467-019-08515-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30787273">30787273</a></span></span></li> <li tag="ref"><a name="B54" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sadre</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magallanes-Lundback</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pradhan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salim</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mesberg</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Metabolite diversity in alkaloid biosynthesis: a multilane (diastereomer) highway for camptothecin synthesis in <i>camptotheca acuminata</i></span>. <span class="source" tagx="source" title="source">Plant Cell</span><span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">1926</span>â€"<span class="lpage" tagx="lpage" title="lpage">1944</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.16.00193">10.1105/tpc.16.00193</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27432874">27432874</a></span></span></li> <li tag="ref"><a name="B55" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schultz</span><span class="given-names" tagx="given-names" title="given-names">B. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S. Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sattely</span><span class="given-names" tagx="given-names" title="given-names">E. S.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Total biosynthesis for milligram-scale production of etoposide intermediates in a plant chassis</span>. <span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span><span class="volume" tagx="volume" title="volume">141</span>, <span class="fpage" tagx="fpage" title="fpage">19231</span>â€"<span class="lpage" tagx="lpage" title="lpage">19235</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jacs.9b10717">10.1021/jacs.9b10717</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31755709">31755709</a></span></span></li> <li tag="ref"><a name="B56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">The genome of <i>Artemisia annua</i> provides insight into the evolution of asteraceae family and artemisinin biosynthesis</span>. <span class="source" tagx="source" title="source">Mol. Plant</span><span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">776</span>â€"<span class="lpage" tagx="lpage" title="lpage">788</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molp.2018.03.015">10.1016/j.molp.2018.03.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29703587">29703587</a></span></span></li> <li tag="ref"><a name="B57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tissier</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sallaud</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rontein</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Tobacco trichomes as a platform for terpenoid biosynthesis engineering</span>, in <span class="source" tagx="source" title="source">Isoprenoid Synthesis in Plants and Microorganisms</span>, eds. <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bach</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohmer</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>). p. <span class="fpage" tagx="fpage" title="fpage">271</span>â€"<span class="lpage" tagx="lpage" title="lpage">283</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-4614-4063-5_18">10.1007/978-1-4614-4063-5_18</a></span></span></li> <li tag="ref"><a name="B58" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torrens-Spence</span><span class="given-names" tagx="given-names" title="given-names">M. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pluskal</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carballo</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weng</span><span class="given-names" tagx="given-names" title="given-names">J. K.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Complete pathway elucidation and heterologous reconstitution of rhodiola salidroside biosynthesis</span>. <span class="source" tagx="source" title="source">Mol. Plant</span><span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">205</span>â€"<span class="lpage" tagx="lpage" title="lpage">217</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molp.2017.12.007">10.1016/j.molp.2017.12.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29277428">29277428</a></span></span></li> <li tag="ref"><a name="B59" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Townsend</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Segura</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chigeza</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Penfield</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rae</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harvey</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">The use of combining ability analysis to identify elite parents for <i>Artemisia annua</i> F1 hybrid production</span>. <span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">e61989</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0061989">10.1371/journal.pone.0061989</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23626762">23626762</a></span></span></li> <li tag="ref"><a name="B60" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Genome of <i>Tripterygium wilfordii</i> and identification of cytochrome P450 involved in triptolide biosynthesis</span>. <span class="source" tagx="source" title="source">Nat. Commun.</span><span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">971</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-020-14776-1">10.1038/s41467-020-14776-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32080175">32080175</a></span></span></li> <li tag="ref"><a name="B61" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoder</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnston</span><span class="given-names" tagx="given-names" title="given-names">J. N.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">A case study in biomimetic total synthesis: polyolefin carbocyclizations to terpenes and steroids</span>. <span class="source" tagx="source" title="source">Chem. Rev.</span><span class="volume" tagx="volume" title="volume">105</span>, <span class="fpage" tagx="fpage" title="fpage">4730</span>â€"<span class="lpage" tagx="lpage" title="lpage">4756</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/cr040623l">10.1021/cr040623l</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16351060">16351060</a></span></span></li> </ul> </div> </div>  </body></html>