<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">eLife</span><span class="iso-abbrev" title="iso-abbrev">Elife</span><span class="publisher-id" title="publisher-id">eLife</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">eLife</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2050-084X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">eLife Sciences Publications, Ltd</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7935491">7935491</a></span><span class="publisher-id" title="publisher-id">56279</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7554/eLife.56279">10.7554/eLife.56279</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Evolutionary Biology</span>: <span class="subject" title="subject">Genetics and Genomics</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Emergence and diversification of a highly invasive chestnut pathogen lineage across southeastern Europe</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stauber</span><span class="given-names" tagx="given-names" title="given-names">Lea</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-8367-6150</span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#con1" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Badet</span><span class="given-names" tagx="given-names" title="given-names">Thomas</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-6130-441X</span><a href="#aff2">2</a><a href="#con2" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feurtey</span><span class="given-names" tagx="given-names" title="given-names">Alice</span></span><a href="#aff2">2</a><a href="#aff3">3</a><a href="#con3" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">Simone</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-9129-8556</span><a href="#aff1">1</a><a href="#fund1" /><a href="#con4" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croll</span><span class="given-names" tagx="given-names" title="given-names">Daniel</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-2072-380X</span><span class="email" tagx="email" title="email">daniel.croll@unine.ch</span><a href="#aff2">2</a><a href="#fund2" /><a href="#con5" /><a href="#conf1" /></span><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Swiss Federal Institute for Forest, Snow and Landscape Research (WSL)</span><span class="addr-line" title="addr-line">Birmensdorf</span><span class="country" tagx="country" title="country">Switzerland</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Laboratory of Evolutionary Genetics, Institute of Biology, University of NeuchÃ¢tel</span><span class="addr-line" title="addr-line">NeuchÃ¢tel</span><span class="country" tagx="country" title="country">Switzerland</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">Plant Pathology, Institute of Integrative Biology, ETH ZÃ¼rich</span><span class="addr-line" title="addr-line">ZÃ¼rich</span><span class="country" tagx="country" title="country">Switzerland</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castric</span><span class="given-names" tagx="given-names" title="given-names">Vincent</span></span><span class="role" tagx="role" title="role">Reviewing Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">UniversitÃ© de Lille</span><span class="country" tagx="country" title="country">France</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weigel</span><span class="given-names" tagx="given-names" title="given-names">Detlef</span></span><span class="role" tagx="role" title="role">Senior Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Max Planck Institute for Developmental Biology</span><span class="country" tagx="country" title="country">Germany</span></span></span></div><span class="pub-date-" title="pub-date-">: <span>2021-3-3</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e56279</span><span class="history" title="history"><span class="received" title="received">received: 2020-2-22</span><span class="accepted" title="accepted">accepted: 2021-2-17</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Stauber et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2021</span></span><span class="license" title="license"> <div tag="license_ref">
http://creativecommons.org/licenses/by/4.0/</div><span class="license-p" title="license-p">This article is distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</span></span></div><a class="self-uri" href="elife-56279.pdf" title="self-uri">elife-56279.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Invasive microbial species constitute a major threat to biodiversity, agricultural production and human health. Invasions are often dominated by one or a small number of genotypes, yet the underlying factors driving invasions are poorly understood. The chestnut blight fungus <i>Cryphonectria parasitica</i> first decimated the North American chestnut, and a more recent outbreak threatens European chestnut stands. To unravel the chestnut blight invasion of southeastern Europe, we sequenced 230 genomes of predominantly European strains. Genotypes outside of the invasion zone showed high levels of diversity with evidence for frequent and ongoing recombination. The invasive lineage emerged from the highly diverse European genotype pool rather than a secondary introduction from Asia or North America. The expansion across southeastern Europe was mostly clonal and is dominated by a single mating type, suggesting a fitness advantage of asexual reproduction. Our findings show how an intermediary, highly diverse bridgehead population gave rise to an invasive, largely clonally expanding pathogen.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
Cryphonectria parasitica</div> <div class="kwd" title="kwd">
fungal pathogen</div> <div class="kwd" title="kwd">
invasive species</div> <div class="kwd" title="kwd">
population genomics</div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Research organism</div> <div class="kwd" title="kwd">
Other</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001711</span><span class="institution" title="institution">Schweizerischer Nationalfonds zur FÃ¶rderung der Wissenschaftlichen Forschung</span></div> </div><span class="award-id" tagx="award-id" title="award-id">170188</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">Simone</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Fondation Pierre Mercier pour la science</span></div> </div><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croll</span><span class="given-names" tagx="given-names" title="given-names">Daniel</span></span></span></div> <div class="funding-statement" title="funding-statement">
The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Author impact statement" value="The chestnut blight fungus has successfully established an intermediary, diverse European bridgehead population, giving rise to a highly invasive clonal lineage spreading across southeastern Europe." /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Over the past century, a multitude of invasive species have emerged as threats to forest and agricultural ecosystems worldwide (<a href="#bib91">Santini et al., 2013</a>; <a href="#bib111">Wingfield et al., 2010</a>). Increased human activities, such as global trade, enabled invasive species to cause economic damage through reduced agricultural production, degradation of ecosystems, and negative impacts on human health (<a href="#bib67">Marbuah et al., 2014</a>). A key group of invasive species in forests are fungal pathogens, which are often accidentally introduced via living plants and plant products (<a href="#bib89">Rossman, 2001</a>). To successfully colonize a new environment, fungal pathogens have to overcome several invasion barriers including effective dispersal abilities, changes in available hosts, competition with other fungi, and niche availability (<a href="#bib40">Gladieux et al., 2015</a>; <a href="#bib45">Hayes and Barry, 2008</a>). This may be achieved with plastic phenotypic changes followed by rapid genetic adaptation (<a href="#bib36">Garbelotto et al., 2015</a>). The invasive potential is also influenced by pre-adaptation arising outside of the invasion zone. Pre-adaptation can be a critical factor for environmental filtering by abiotic or host-related factors in the new environment (<a href="#bib59">Krasnov et al., 2015</a>). As the number of initial founders is often low, invasive populations are frequently of low genetic diversity, which reduces adaptive genetic variation (<a href="#bib2">Allendorf and Lundquist, 2003</a>; <a href="#bib113">Yang et al., 2012</a>). Yet, many fungal plant pathogen invasions were successful despite low genetic diversity within founding populations (<a href="#bib32">Fontaine et al., 2013</a>; <a href="#bib86">Raboin et al., 2007</a>; <a href="#bib112">Wuest et al., 2017</a>).</p> <p>A major model explaining the successful expansion of invasive populations despite low initial genetic diversity is the so-called bridgehead effect (<a href="#bib66">Lombaert et al., 2010</a>). In this model, highly adapted lineages emerge through recombination among genotypes established in an area of first introduction. Hence, the primary introduction serves as the bridgehead for a secondary and more expansive invasion. Although the bridgehead effect has been proposed as a scenario for many biological invasions (<a href="#bib38">Gau et al., 2013</a>; <a href="#bib104">van Boheemen et al., 2017</a>), empirical evidence for the creation of highly adaptive genotypes within bridgehead populations is still largely missing (<a href="#bib7">Bertelsmeier and Keller, 2018</a>). An alternative explanation suggests that primary bridgehead populations simply serve as repeated sources of inoculum for secondary invasions without selecting for the invasive (i.e., adapted) genotypes in situ (<a href="#bib7">Bertelsmeier and Keller, 2018</a>). This alternative scenario implies that initial populations are already composed of genotypes adapted to the new environment or have high phenotypic plasticity (<a href="#bib10">Bock et al., 2015</a>; <a href="#bib40">Gladieux et al., 2015</a>; <a href="#bib105">VukoviÄ‡ et al., 2019</a>). A conceptually similar model to the bridgehead effect was proposed for bacteria, where the success of epidemic clones would be contingent on the generation of recombinants in the original populations (<a href="#bib95">Smith et al., 1993</a>). Dissecting whether invasive species were pre-adapted to the new environment or gained adaptation through a bridgehead effect is crucial for effective containment strategies, but requires a deep sampling of genotypes during the early invasion process.</p> <p>The pace of adaptive evolution leading to successful invasions is determined by several lifeÂ history traits including the mating system (i.e., sexual, asexual, or mixed). Sexual reproduction can potentially promote the emergence of more virulent strains through the reshuffling of genetic variants, enhancing invasive success (<a href="#bib81">Philibert et al., 2011</a>). In contrast, a switch to asexual reproduction can limit genetic diversification and adaptive potential in newly introduced species (<a href="#bib26">Drenth et al., 2019</a>; <a href="#bib101">Taylor et al., 2015</a>). However, the low availability of mating partners at the invasion front can be a major cost for sexual reproduction. To avoid the cost of mate search, invasive fungal pathogens often switch from sexual to predominantly asexual reproduction (<a href="#bib48">Heitman et al., 2013</a>; <a href="#bib100">Suehs et al., 2004</a>). Consequently, asexual reproduction can promote rapid colonization and dissemination of adapted genotypes (<a href="#bib39">Gladieux et al., 2011</a>). Beyond invasive species, many fungal pathogens undergo asexual reproduction during the colonization of habitats, butÂ they switch to sexual reproduction for the production of survival propagules prior to unfavorable environmental conditions (<a href="#bib93">Schoustra et al., 2010</a>). Populations of some highly successful invaders, such as the ascomycete <i>Ophiostoma novo-ulmi</i> causing Dutch elm disease (<a href="#bib80">Paoletti et al., 2006</a>), or the oomycete <i>Phytophthora ramorum</i> (<a href="#bib43">GrÃ¼nwald et al., 2012</a>), are dominated by a single mating type. Low diversity in invasive lineages may be breached by secondary invasions introducing the opposite mating type as observed in <i>Phytophthora infestans</i> in Europe. Prior to the 1980s, only the presence of the A1 mating type was clearly established, followed by the secondary introduction of the A2 mating type leading to sexual reproduction (<a href="#bib68">Mariette et al., 2016</a>). Introgression from closely related species can also reintroduce a missing mating type. <i>O. novo-ulmi</i> acquired the missing mating type from <i>O. ulmi</i> (<a href="#bib14">Brasier and Webber, 2019</a>; <a href="#bib80">Paoletti et al., 2006</a>). Although switches in reproductive modes can be a key factor for invasion success (<a href="#bib81">Philibert et al., 2011</a>), mechanisms underlying such switches remain poorly understood (<a href="#bib8">Billiard et al., 2012</a>). Beyond the impact on genetic diversity, reproductive modes are often tied to specific environmental adaptations including the mode of spore dispersal. Sexual spores are often highly durable and can be wind dispersed over long distances (<a href="#bib81">Philibert et al., 2011</a>; <a href="#bib87">Rigling and Prospero, 2018</a>). Even rare long-distance dispersal events can contribute to the geographic expansion of invasive pathogens (<a href="#bib83">Prospero and Cleary, 2017</a>). In contrast, asexual spores serve mostly short distance dispersal in fungal pathogens with mixed reproduction (<a href="#bib73">Mehrotra, 2013</a>; <a href="#bib83">Prospero and Cleary, 2017</a>).</p> <p>Some invasive pathogens were discovered to harbor specific characteristics in the architecture of the genome, which may have facilitated the generation of adaptive genetic variation. For instance, genomes of <i>Ophiostoma</i> species causing Dutch elm disease show signals of frequent hybridization and introgression events, giving rise to lineages with increased virulence or growth at higher temperatures (<a href="#bib49">Hessenauer et al., 2020</a>). Genomes of a number of crop pathogens including the oomycete <i>P. infestans</i> and the fungal pathogens <i>Leptosphaeria maculans</i> and <i>Zymoseptoria tritici</i> show compartmentalization into gene-dense and gene-poor regions with virulence factor being preferentially encoded in gene-poor regions (<a href="#bib25">Dong et al., 2015</a>; <a href="#bib34">Frantzeskakis et al., 2019</a>; <a href="#bib102">Torres et al., 2020</a>). The gene-poor regions are enriched in transposable elements (TEs), and the encoded virulence factors (i.e., effectors) show epigenetic regulation, rapid sequence turnover, and weak phylogenetic conservation (<a href="#bib33">FouchÃ© et al., 2018</a>). In pathogen systems such as <i>Cryptococcus gattii</i> or <i>P. infestans</i>, copy number variation (CNV)Â has created highly successful outbreak lineages (<a href="#bib57">Knaus et al., 2020</a>; <a href="#bib99">Steenwyk et al., 2016</a>). Beyond potential roles in adaptation, the genome structure of plant pathogens is likely influenced by neutral processes including migratory bottlenecks possibly affecting the activity of TEs (<a href="#bib79">Oggenfuss et al., 2020</a>). Whether virulence factors are associated with genome compartmentalization and how chromosomal rearrangements could underpin the creation of invasive pathogen genotypes remain largely unknown.</p> <p>The ascomycete <i>Cryphonectria parasitica</i> (Murr.) Barr. is the causal agent of chestnut blight, a lethal bark disease of <i>Castanea</i> species (<a href="#bib87">Rigling and Prospero, 2018</a>). The pathogen is native to eastern Asia and was first observed in 1904 in North America on the American chestnut (<i>Castanea dentata</i>). In the following years, the disease rapidly spread throughout the distribution range of <i>C. dentata</i>, causing the vast decimation of this native tree species (<a href="#bib29">Elliott and Swank, 2008</a>). In 1938, the fungus was first detected on the European chestnut (<i>Castanea sativa</i>) near Genoa (Italy) and is now established in all major chestnut-growing countries in Europe (<a href="#bib87">Rigling and Prospero, 2018</a>). The damage to the European chestnut may have been reduced by the presence of the <i>Cryphonectria hypovirus 1</i> (CHV1), which acts as a biological control agent of chestnut blight (<a href="#bib87">Rigling and Prospero, 2018</a>). The virus can be transmitted both vertically to asexual spores (conidia) or horizontally through hyphal anastomoses between virus-infected and virus-free strains (<a href="#bib47">Heiniger and Rigling, 1994</a>). Hyphal anastomoses are controlled by a vegetative compatibility system, and the virus spreads most effectively between strains of the same vegetative compatibility type (<a href="#bib21">Cortesi et al., 2001</a>).</p> <p>Population genetic analyses showed that the invasion of Europe occurred through multiple introductions both from native populations in Asia and from populations in North America (<a href="#bib28">Dutech et al., 2012</a>). Invasive European <i>C. parasitica</i> populations are characterized by lower vegetative compatibility diversity than North American and Asian populations (<a href="#bib76">Milgroom and Cortesi, 1999</a>). Furthermore, European populations exhibit lower recombination rates (<a href="#bib27">Dutech et al., 2010</a>; <a href="#bib41">GonzÃ¡lez-Varela et al., 2011</a>; <a href="#bib84">Prospero and Rigling, 2012</a>). Mating in <i>C. parasitica</i> is controlled by a single mating type (MAT) locus (<a href="#bib70">Marra and Milgroom, 2001</a>), and natural populations can reproduce both sexually and asexually (<a href="#bib69">Marra et al., 2004</a>). Previous analyses of southeastern European <i>C. parasitica</i> populations based on sequence characterized amplified region (SCAR) markers suggested that the region was largely colonized by a single and likely asexual lineage also identified as S12 (<a href="#bib75">Milgroom et al., 2008</a>). The lineage belongs to the vegetative compatibility type EU-12. Within the distribution range of the lineage, sexual structures (i.e., perithecia) are rarely found (<a href="#bib75">Milgroom et al., 2008</a>; <a href="#bib96">Sotirovski et al., 2004</a>). Based on SCAR marker and field records, <a href="#bib75">Milgroom et al., 2008</a> suggested that the invasive S12 lineage originated in northern Italy and subsequently spread across southeastern Europe (<a href="#bib3">Avolio, 1978</a>; <a href="#bib9">Biraghi, 1946</a>; <a href="#bib17">Buccianti and Feliciani, 1966</a>; <a href="#bib56">KaradÅ¾iÄ‡ et al., 2019</a>; <a href="#bib77">Myteberi et al., 2013</a>; <a href="#bib88">Robin and Heiniger, 2001</a>). However, due to the low molecular marker resolution and challenges in relying on observational data, the origin, invasion route, and genetic diversification of <i>C. parasitica</i> in southeastern Europe remain largely unknown.</p> <p>In this study, we sequenced complete genomes of a comprehensive collection of European <i>C. parasitica</i> strains with a fine-scaled sampling throughout the S12 invasion zone. Based on high-confidence genome-wide single nucleotide polymorphisms (SNPs), we identified the most likely origin and recapitulated the invasion process across southeastern Europe. We show that the invasive <i>C. parasitica</i> lineage arose through an intermediary, highly diverse bridgehead population. During the expansion, the lineage became dominated by a single mating type but retained the ability to reproduce sexually.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="genome-widepolymorphismanalysesofglobalc.parasiticaisolates" title="sec"> <div class="title" tagx="title" title="title">
Genome-wide polymorphism analyses of global <i>C. parasitica</i> isolates</div> <p>We analyzed complete genomes of 230 <i>C. parasitica</i> isolates covering the global chestnut blight distribution range, as well as the European outbreak region of the invasive S12 lineage (<a href="#fig1">Figure 1A</a>, <a href="#supp1">Supplementary file 1</a>). Isolates were sequenced at a mean depth of 8â€"53Ã— to detect high-confidence genome-wide SNPs. A region of 179,501â€"2,084,312 bp on scaï¬€old 2 was associated to the mating type locus based on association mapping p-values (<a href="#fig1s1">Figure 1â€"figure supplement 1</a>). This region encoded known mating type-associated genes and was characterized by a high SNP density consistent with observations in other fungi (<a href="#bib53">Idnurm et al., 2015</a>; <a href="#bib101">Taylor et al., 2015</a>). We removed SNPs within the mating type-associated region to avoid confounding genetic structure with mating type divergence. We retained 80,700 SNPs and performed a principal component analysis (PCA) and constructed a SplitsTree phylogenetic network (<a href="#fig1">Figure 1B,Â C</a>). The PCA revealed four major clusters (CL1â€"CL4, <a href="#fig1">Figure 1B</a>), showing weak geographic clustering except for isolates of Chinese origin (cluster CL4 in <a href="#fig1">Figure 1B</a>). The largest cluster contained the majority of sequenced European isolates, including the S12 lineage, as well as all North American and two Georgian isolates (cluster CL1 in <a href="#fig1">Figure 1B</a>). The phylogenetic network analysis revealed large phylogenetic distances outside the main European/North American CL1 cluster (<a href="#fig1">Figure 1B,Â C</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <div class="title" tagx="title" title="title">
Genome-wide analyses of global <i>Cryphonectria parasitica</i> isolates.</div> <p>(<b>A</b>) Map of global sampling locations and (<b>B</b>) principal component analysis (PCA) of the 230 sequenced <i>C. parasitica</i> isolates. Colors indicate PCA clusters (CL1â€"CL4) and are as in (<b>A</b>). (<b>C</b>) SplitsTree phylogenetic network of the global <i>C. parasitica</i> sample set. Colors are as in (<b>A</b>) and (<b>B</b>). Isolates belonging to the S12 lineage are marked with an arrow in (<b>B</b>) and (<b>C</b>).</p>   </div> </div> <div class="reconstructionoftheeuropeandemographichistory" title="sec"> <div class="title" tagx="title" title="title">
Reconstruction of the European demographic history</div> <p>We implemented a demographic model based on each cluster identified by the PCA (<a href="#fig1">Figure 1B</a>), merging isolates from the two smaller clusters (i.e., CL3 and CL4, <a href="#fig1">Figure 1B</a>) to obtain a sufficient sample size. The folded site frequency spectrum (SFS) for each of the three groups contained a large fraction of singletons, suggesting a population expansion (<a href="#fig2">Figure 2</a>). We used the software <i>dadi</i> (<a href="#bib44">Gutenkunst et al., 2009</a>) to identify the best fitting model considering no population size change or a single instantaneous population size change, as well as more complex models represented in <a href="#fig2">Figure 2</a> (<a href="#supp2">Supplementary file 2</a>). For the isolates outside the main European/North American group, we found support for a recent population expansion (CL2â€"CL4 clusters, <a href="#fig1">Figure 1B</a>, <a href="#fig2">Figure 2</a>). Indeed, for both clusters, the best Akaike information criterion (AIC) value was obtained for modeling a simple instantaneous expansion (AIC of 70.4 and 115.4 for the CL3+CL4 and CL2, respectively; <a href="#supp2">Supplementary file 2</a>). The second-best model was a more complex version of an expansion scenario including an initial bottleneck followed by exponential growth (AIC of 75.0 and 152.7 for the â€˜bottlegrowthâ€™ model, for the CL3+CL4 and CL2, respectively). However, for the large European/North American CL1 cluster, despite removing all but a single isolate of the S12 lineage, parameter optimization was not successful for any model or any randomly drawn initial parameters (see <a href="#supp2">Supplementary file 2</a>). This is likely explained by the unusual frequency spectra. As shown in <a href="#fig2">Figure 2</a>, the observed folded SFS for the European/North American cluster (CL1 <a href="#fig1">Figure 1B</a>, <a href="#fig2">Figure 2</a>) is W-shaped (red line in <a href="#fig2">Figure 2</a>) instead of a monotonously declining curve as is the case in the empirical spectra for other populations as well as in the simulations (<a href="#fig2">Figure 2</a>). The unusual frequency spectrum of the European/North American cluster potentially stems from the presence of multiple small groups of related genotypes due to inbreeding.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <div class="title" tagx="title" title="title">
Demographic models for global <i>Cryphonectria parasitica</i> populations.</div> <p>Constant, expansion, bottleneck, bottleneck-expansion, and bottlegrowth population models were tested for three clusters (i.e., European/North American CL1, mixed European/Asian CL2, and Asian CL3 and CL4 clusters shown by color highlights). Clusters were inferred by principal component analysisÂ (PCA) (see <a href="#fig1">Figure 1B</a>). The two smaller Asian clusters (top-right PCA panel) highlighted in red and blue were combined for demographic analyses to obtain a sufficient sample size. Panels in columns 2â€"4 show modelâ€"data comparison plots for the three tested scenarios (i.e., constant, expansion and bottleneck, bottleneck-expansion and bottlegrowth) as inferred by <i>dadi</i>. Blue data points and line represent the model, andÂ red lines show the empirical data. Best fitting models according to theÂ Akaike information criterion (AIC) model comparisons are marked in black boxes. A detailed list of initial and best-fit parameters, as well as likelihood and AIC values, is shown in <a href="#supp2">Supplementary file 2</a>.</p>   </div> </div> <div class="phylogenomicanalysesofeuropeanandnorthamericanisolates" title="sec"> <div class="title" tagx="title" title="title">
Phylogenomic analyses of European and North American isolates</div> <p>To further investigate the European/North American CL1 cluster containing the S12 lineage (<a href="#fig1">Figure 1B</a>), we performed a SplitsTree phylogenetic network analysis to account for reticulation caused by recombination in this cluster specifically. The network showed a substantial diversification, with both long branching and reticulation (<a href="#fig3">Figure 3A</a>, <a href="#fig3s1">Figure 3â€"figure supplement 1</a>). Genotypes within the cluster show significant evidence for recombination (pairwise homoplasy indexÂ [PHI]Â test; p&amp;lt;0.0001, <a href="#fig3s2">Figure 3â€"figure supplement 2</a>). Despite the high level of genetic diversity, we found no evidence for geographic structure. Moreover, we found no clustering of isolates belonging to the same vegetative compatibility type with the exception of some EU-01, EU-02, and EU-12 (S12) genotypes of predominantly Balkan origin (<a href="#fig3">Figure 3A</a>, <a href="#fig3s1">Figure 3â€"figure supplement 1</a>). Vegetative compatibility-type diversity has been widely used as a proxy for genetic diversity in populations (<a href="#bib47">Heiniger and Rigling, 1994</a>). Our findings confirm the clear limitation of these markers as proxies for diversity (<a href="#bib54">JeÅ¾iÄ‡ et al., 2012</a>). Nevertheless, vegetative compatibilityÂ type diversity impacts mycovirus transmission dynamics and remains, hence, a relevant population trait. Nearly all <i>C. parasitica</i> isolates representing the S12 lineage showed almost identical genotypes and tight clustering. The most tightly clustered S12 genotypes were all of mating type MAT-1 (<i>n</i>Â =Â 105). Consistent with analyses by <a href="#bib75">Milgroom et al., 2008</a>, this group represents the invasive S12 lineage at the origin of the expansion of <i>C. parasitica</i> across southeastern Europe. Additionally, the phylogenetic network revealed closely related but not identical S12 genotypes of mating type MAT-2 (<i>n</i>Â =Â 7, <a href="#fig3">Figure 3A</a>). Hence, S12 outbreak strains of MAT-2 connect the nearly uniform cluster of S12 MAT-1 strains with the remaining genetic diversity of the major European subgroup of <i>C. parasitica</i>. The S12 cluster was furthermore connected with the remaining genotypes of the major clade by two EU-12 isolates from Bosnia (M1808 with MAT-1) and Georgia (MAK23 with MAT-2) (<a href="#fig3">Figure 3A</a>,Â <a href="#fig3s1">Figure 3â€"figure supplement 1</a>). Consistent with previous findings (<a href="#bib75">Milgroom et al., 2008</a>), we also found non-S12 genotypes in the invasion zone of the S12 outbreak including the vegetative compatibility types EU-01 and EU-02 in southern Italy, Greece, and Turkey (<a href="#fig3">Figure 3A,Â B</a>). Additionally, we found the S12 lineage in otherwise more diverse regions of Croatia and Bosnia (<a href="#fig3">Figure 3A,Â B</a>). Surprisingly, we also detected one Georgian (GEZ45) and one Portuguese (M2135) isolate within the S12 cluster, suggesting human-mediated dissemination of the lineage outside of southeastern Europe (<a href="#fig3">Figure 3B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <div class="title" tagx="title" title="title">
Phylogenetic network structure of the main European/North American subgroup and spatial distribution of S12 across Europe.</div> <p>(<b>A</b>) The highlighted branches represent the most abundant vegetative compatibility types. Isolates belonging to the S12 outbreak lineage (EU-12; mating type MAT-1, <i>n</i>Â =Â 105) are marked with a red hexagon and match symbols in (<b>B</b>). S12 isolates of mating type MAT-2 are highlighted in light red. Additional EU-12 isolates not belonging to the S12 lineage are highlighted in blue with information on the country of origin. Isolates sharing ancestry with the S12 lineage as inferred by fineSTRUCTURE (<a href="#fig4">Figure 4</a>, <a href="#fig4s2">Figure 4â€"figure supplement 2</a>) are marked with dark red diamonds. (<b>B</b>) Geographic map of European sampling locations. Light red hexagons mark locations where S12 was found, andÂ yellow circles indicate locations containing other genotypes from the main European/North American CL1 cluster (<a href="#fig1">Figure 1B</a>). Black circles show the location of highly distinct genotypes outside of the CL1 cluster (<a href="#fig1">Figure 1A,Â B</a>). (<b>C</b>) Genome-wide scan for selective sweeps (RAiSD). Effects due to the unknown demographic history of the European/North American CL1 cluster (<a href="#fig1">Figure 1B</a>) were mitigated by implementing population simulations under constant, expansion, and bottleneck scenarios. The three dashed lines show the maximum values observed in the simulated datasets following constant (gray) expansion (blue) and bottleneck (green) demographic models. Encoded protein functions overlapping with the top regions are shown as summaries.</p>   </div> </div> <div class="evidenceforselectionineuropeanpopulations" title="sec"> <div class="title" tagx="title" title="title">
Evidence for selection in European populations</div> <p>We investigated potential selective sweeps in the focal European/North American cluster. To avoid a bias by the deep sampling of the S12 lineage, we excluded all but one of the S12 MAT-1 isolates (see 'MaterialsÂ andÂ methods'). We used RAiSD (<a href="#bib1">Alachiotis and Pavlidis, 2018</a>) to produce a composite score of selective sweep signals and identified two strong outlier loci. Despite no clear demographic model fitting the genomic data of the focal population, we nevertheless attempted to control for potential demographic history effects. We performed 10,000 simulations of a population under three different models without selection: no population size change (i.e., constant), expansion, and bottleneck. This was followed by a genome-wide selection scan on the simulated datasets using RAiSD. We assessed above what Âµ values selection signals are most likely exceeding effects by demographic events. The highest values of Âµ were obtained for the simulations with the expansion scenarios (ÂµÂ =Â 37). Consequently, we used this value as a conservative threshold to identify the most robust signatures of selection. Two outlier loci obtained in the selection scan on the observed data have Âµ values above 37. The strongest sweep locus was located at the boundary of the mating type locus on scaffold 2 (<a href="#fig3">Figure 3C</a>). The second sweep locus was detected on scaffold 1 and encompassed an âˆ¼120 kb locus at positions 2099â€"3309 kb. The region contains 335 genes of which 107 encode conserved protein domains (<a href="#fig3">Figure 3C</a>, <a href="#supp3">Supplementary file 3</a>). Hence, genetic variation in the two selection sweep regions is potentially underpinning recent adaptation of <i>C. parasitica</i> to the European environment.</p> </div> <div class="analysesofcoancestryfortheinvasivelineage" title="sec"> <div class="title" tagx="title" title="title">
Analyses of coancestry for the invasive lineage</div> <p>The SplitsTree network revealed that the invasive S12 lineage has closely related genotypes occurring in Europe. Thus, to dissect the genetic ancestry of S12, we performed a coancestry matrix analysis using fineSTRUCTURE (<a href="#bib62">Lawson et al., 2012</a>) considering all isolates of the major European/North American cluster (CL1, <a href="#fig1">Figure 1B</a>), including the S12 lineage (<i>n</i>Â =Â 197). Within this cluster, KÂ =Â 43 genetic subclusters were identified with fineSTRUCTURE (<a href="#fig4">Figure 4</a>, <a href="#fig4s1">Figure 4â€"figure supplement 1A</a>). The large number of identified clusters likely stems from the presence of multiple small groups of highly related individuals beyond the S12 lineage. Furthermore, populations only showed weak geographic structuring (<a href="#fig4s2">Figure 4â€"figure supplement 2</a>). The highest level of coancestry for any given S12 isolate was found within the S12 population (<a href="#fig4">Figure 4</a>). Moreover, we identified the highest degree of shared ancestry between S12 and populations from the northern Balkans (Bosnia and Croatia), southern Switzerland, but also Georgia and Spain, with coancestry values between 35.3Â andÂ 49.7 (<a href="#fig4">Figure 4</a>, <a href="#fig4s2">Figure 4â€"figure supplement 2</a>). As Central European <i>C. parasitica</i> populations were likely established from North American sources alone (<a href="#bib28">Dutech et al., 2012</a>), the coancestry analysis points to a European origin of the invasive S12 lineage rather than a novel introduction from North America.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <div class="title" tagx="title" title="title">
Analysis of donors to the S12 lineage genotypes.</div> <p>Averaged coancestry matrix and population tree of the North American/European subgroup estimated by fineSTRUCTURE. The heatmap indicates averaged coancestry between populations. Populations sharing ancestry with S12 are marked in red-dashed boxes. S12 ancestry-sharing populations are additionally marked in <a href="#fig3">Figure 3A</a>. Detailed coancestry matrix with extensive population information is shown in <a href="#fig4s2">Figure 4â€"figure supplement 2</a>.</p>   </div> </div> <div class="telandscapeandcnvinthes12lineage" title="sec"> <div class="title" tagx="title" title="title">
TE landscape and CNV in the S12 lineage</div> <p>Invasive pathogen lineages may have undergone crucial genomic rearrangements producing more fit genotypes. Here, we generated a de novo identification and annotation of TEs for the <i>C. parasitica</i> genome. We found that 12% of the genome was composed of TEs with striking variation along the assembled scaffolds (i.e., quasi-chromosomes; <a href="#fig5">Figure 5A</a>). In particular, regions on scaffold 2 matching the mating type locus are highly enriched in TEs, suggesting that the large non-recombining region has undergone substantial degeneration (<a href="#fig5">Figure 5A</a>). In contrast, the vegetative incompatibility (<i>vic</i>) loci are located in regions devoid of TEs. In fungal pathogens, effector genes and TEs are often co-localized in fast-evolving compartments of the so-called â€˜two-speed genomeâ€™ (<a href="#bib25">Dong et al., 2015</a>). However, the <i>C. parasitica</i> genome shows no apparent compartmentalization into gene-sparse and gene-rich regions. We used machine learning to predict secreted proteins most likely acting as effectors to manipulate the host. In contrast to other plant pathogens, effector gene candidates showed no tendency to localize in gene-sparse regions of the genome (<a href="#fig5">Figure 5B</a>). Non-repressed TEs can potentially create additional copies in the genome, leading to intra-species variability in TE content. To detect such TE activity, we performed genome-wide scans of <i>C. parasitica</i> isolates for presence or absence of TEs based on split read and target site duplication information. At loci with detectable TE presence/absence polymorphism, we found an over twofold variation in total TE counts across all isolates (<a href="#fig5">Figure 5C</a>). The total TE count variation among the genetically diverse non-S12 isolates (North America and Europe only, <a href="#fig3">Figure 3A</a>) was larger than the clonal S12. Nevertheless, the count of TE insertions at polymorphic TE insertion sites ranged between 12Â andÂ 52 among S12 isolates, which was surprisingly high given their recent emergence and extremely high similarity across the genome (<a href="#fig5">Figure 5C</a>). Across TE insertion loci, we found only a marginally significant difference in the TE frequency among S12 compared to non-S12 isolates (p=0.03, <a href="#fig5s1">Figure 5â€"figure supplement 1</a>). This is in strong contrast to segregating SNPs. We found pairwise differences of 0â€"45 SNPs among S12 isolates and 0â€"3770 among non-S12 isolates (test statistic t-test mean difference p&amp;lt;0.001; <a href="#fig5s2">Figure 5â€"figure supplement 2</a>). For a direct comparison of TE and SNP diversity, we assessed the coefficient of variation (CV) of normalized TE and SNP counts. For TEs, we found a CVÂ =Â 0.24 for S12 and CVÂ =Â 0.33 for non-S12 isolates. For SNPs, we found a CVÂ =Â 0.007 for S12 and CVÂ =Â 0.69 for non-S12. Hence, our findings indicate a higher TE diversity among isolates compared to SNP diversity within the S12 lineage. We found that 44% and 49% of the polymorphic TE insertion sites are specific to the S12 and non-S12 group of isolates, respectively. Given the much more recent origin of the S12 lineage, this suggests increased TE activity since the emergence of the S12 lineage contributing to the creation ofÂ de novo genetic variation. For SNPs but not for TEs, the minor allele frequency spectrum of the S12 lineage was significantly skewed toward low-frequency variants compared to non-S12 isolates (Fisher's exact test, p&amp;lt;0.001; <a href="#fig5s2">Figure 5â€"figure supplement 2</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <div class="title" tagx="title" title="title">
Transposable element (TE) landscape and copyÂ number variation.</div> <p>(<b>A</b>) Genome-wide coverage of transposable elements (in 10 kb windows) matched by with normalized read depth for the S12 and non-S12 lineages (North America and Europe only). <i>vic</i> loci: vegetative incompatibility loci. (<b>B</b>) Genome-wide distribution of intergenic distances according to the length of 5â€² and 3â€² flanking regions. Red dots represent genes encoding predicted eï¬€ector proteins. (<b>C</b>) Counts of detected TE sequences across S12 and non-S12 isolates using split reads and target site duplication information. (<b>D</b>) Proportion of normalized read depth windows (800 bp) with evidence for duplications (normalized read depthÂ &amp;gt;1.6) or deletions (&amp;lt;0.4) overlapping with genes and TEs. (<b>E</b>) Heatmap showing the number of windows (800 bp) with duplications and deletions. The dendrogram shows the similarity in duplication or deletion profiles for S12 and non-S12 isolates (North America and Europe only). (<b>F</b>) Molecular functions (based on gene ontology) enriched in duplicated and deleted regions (upper and lower panel, respectively). Enrichment was tested by hypergeometric tests, and significance was established for a Bonferroni threshold at alphaÂ =Â 0.05. The numbers represent the number of genes with the matching gene ontology term in a duplicated or deleted region, and across the genome, respectively.</p>   </div> <p>Repetitive sequences such as TEs can trigger non-homologous recombination leading to CNV including deletions. We used normalized read coverage to assess CNV across the species including the S12 lineage (<a href="#fig5">Figure 5A</a>). Compared to the reference genome, we found that most CNVs in the S12 lineage are nearly fixed. Non-S12 isolates segregate most CNVs at intermediate frequencies (comparison between S12 and non-S12; Fisher's exact test p&amp;lt;0.001, <a href="#fig5s1">Figure 5â€"figure supplement 1</a>). Genes tend to overlap duplications rather than deletions (odds ratios of 0.37 and 0.13, respectively; two-sided Fisher test, p-value&amp;lt;0.001). TEs tend to overlap deletions rather than duplications (odds ratios of 5.03 and 1.76, respectively; two-sided Fisher's test, p-value&amp;lt;0.001) (<a href="#fig5">Figure 5D</a>). The mating type region on scaffold 2 and the rDNA locus on scaffold 6 show particularly high levels of CNV (<a href="#fig5">Figure 5A</a>). In a joint analysis of all isolates from the main European/North American cluster (CL1, <a href="#fig1">Figure 1B</a>), we found that coding sequences overlapping with duplications and deletions are enriched for gene ontology terms associated with protein binding functions and protein phosphorylation activity, respectively (<a href="#fig5">Figure 5F</a>). Overall, our results show that the S12 lineage underwent specific gene deletion and duplication patterns compared to the broad diversity of non-S12 isolates.</p> </div> <div class="theevolutionaryhistoryofeuropeanpopulations" title="sec"> <div class="title" tagx="title" title="title">
The evolutionary history of European populations</div> <p>To gain insights into evolutionary forces shaping polymorphism in the outbreak S12 mating type MAT-1 lineage versus other populations in the European/North American cluster (non-S12, CL1 cluster; <a href="#fig1">Figure 1B</a>, <a href="#fig3">Figure 3A</a>), we first analyzed allele frequencies across the genome in both groups (<a href="#fig6">Figure 6A</a>). The S12 lineage segregated virtually no intermediate allele frequencies in the range of 0.05â€"0.95. In contrast, the non-S12 genotypes showed overall a wide spectrum of allele frequencies across the genome. Genome-wide nucleotide diversity was extremely reduced in the S12 compared to non-S12 populations (<a href="#fig6">Figure 6B</a>). We analyzed the predicted impact on protein functions of segregating mutations in the S12 lineage and non-S12 populations (<a href="#fig6">Figure 6C</a>). We found 4 highly and 104 moderately deleterious mutations segregating within S12 compared to 32 high and 972 moderately deleterious mutations in non-S12 groups (<a href="#fig6">Figure 6C</a>). Three of the high-impact SNPs in the S12 lineage were classified as stop gain mutations, as well as one splice acceptor variant (insertion variant). Two of these high-impact mutations aï¬€ect proteins of the major facilitator superfamily, as well as a protein containing a LCCL domain and an ecdysteroid kinase. Non-S12 populations showed an over-representation of low-frequency high-impact mutations (Fisher contingency table test odds ratio of 3.7 and p=0.002028; <a href="#fig6">Figure 6C</a>). This is consistent with purifying selection reducing the frequency of these mutations due to fitness costs. Within the S12 lineage nearly all segregating mutations were at very low frequency. We found only modifier (i.e., nearly neutral) mutations rising to higher frequency within the lineage, suggesting that deleterious mutations in the S12 lineage can still be removed through low levels of recombination and purifying selection. The extremely low level of polymorphism segregating within the S12 lineage prevents strong inferences of selective sweeps in the lineage.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <div class="title" tagx="title" title="title">
Polymorphism segregating in the S12 lineage.</div> <p>(<b>A</b>) Alternative allele frequencies spectra across the genome for the S12 lineage (<i>n</i>Â =Â 105) compared to all other analyzed European (non-S12; <i>n</i>Â =Â 92). (<b>B</b>) Genome-wide nucleotide diversity (Pi) for the S12 lineage and non-S12 lineages in 10 kb windows. (<b>C</b>) Minor allele frequency spectra of high, moderate, and modifier (i.e., near neutral) impact mutations as identified by SnpEï¬€.</p>   </div> </div> <div class="retracings12invasionroutes" title="sec"> <div class="title" tagx="title" title="title">
Retracing S12 invasion routes</div> <p>To infer potential invasion routes of the S12 outbreak lineage (<i>n</i>Â =Â 105; <a href="#fig3">Figure 3A</a>), we investigated intra-lineage genetic diversity across southeastern Europe. The closely related genotypes segregated 529 high-confidence SNPs across the genome, of which 459 were identified as singletons. The phylogenetic network revealed a star-like structure, consistent with a predominantly clonal population structure (PHIÂ test p&amp;gt;0.05; <a href="#fig7">Figure 7A</a>). The genetic structure assessed by a PCA suggests a differentiation into two to three groups with a weak geographic association (<a href="#fig7">Figure 7B</a>). The majority of isolates were found in a cluster along the principal component axis 2 (<a href="#fig7">Figure 7B</a>) without apparent geographic structure within the group. The second group consisted of eight isolates from Turkey and one isolate from Georgia consistent with the geographic proximity of the countries (<a href="#fig7">Figure 7B</a>). The most diverse group consisted exclusively of isolates from Greece showing reticulation in the SplitsTree (<a href="#fig7">Figure 7A</a>). Nucleotide diversity for the S12 outbreak lineage was Ï€Â =Â 2.90Ã—10<sup>âˆ’7</sup> compared to Ï€Â =Â 2.62Ã—10<sup>âˆ’7</sup> for the largest subgroup (<i>nÂ =</i>Â 89) and Ï€Â =Â 1.80Ã—10<sup>âˆ’7</sup> for the Turkish/Georgian subgroup (<i>n</i>Â =Â 8) excluding the diverse group of Greek isolates. The spread of the Greek isolates suggests that these might have recently undergone sexual reproduction and admixture events. As we identified non-S12 genotypes in Greece, either within-lineage recombination or exchange with other lineages may underlie the reticulation and increased diversity.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <div class="title" tagx="title" title="title">
Fine-scale genetic diversity analyses of the S12 lineage.</div> <p>(<b>A</b>) SplitsTree and (<b>B</b>) principal component analysis (PCA) of the S12 mating type MAT-1 outbreak isolates (<i>n</i>Â =Â 105). Symbols and colors are as in (<b>A</b>). (<b>C</b>) Scheme of successful mating pairs of S12 mating type MAT-1 isolates crossed with isolates from the opposite mating type of the same geographic origin. Symbols are as in (<b>A</b>). (<b>D</b>) Photographic images of sexual <i>C. parasitica</i> fruiting bodies (i.e., perithecia) emerging from crosses of S12 mating type MAT-1 isolates with isolates of the opposite mating type after 5 months of incubation under controlled conditions. Left: perithecia embedded in a yellow-orange stromatic tissue. Right: cross-section of perithecia and chestnut bark. Flask-shaped structures with a long cylindrical neck develop in yellow-orange stromatic tissue and are embedded in the bark (except for the upper part). The ascospores are formed in sac-like structures (asci) in the basal part of the perithecium. When mature, the ascospores are actively ejected into the air through a small opening (ostiole) at the end of the perithecial neck.</p>   </div> </div> <div class="experimentaltestsofs12matingcompetence" title="sec"> <div class="title" tagx="title" title="title">
Experimental tests of S12 mating competence</div> <p>We tested experimentally whether S12 mating type MAT-1 isolates were still able to reproduce sexually. We selected S12 isolates from populations in Bosnia, Kosovo, and Italy matching locations where the opposite mating type was also detected (<a href="#supp4">Supplementary file 4</a>). After 5 months of incubation, we confirmed outcrossing of isolates of opposite mating type within the S12 lineage by pairing isolates from Molliq (Kosovo) and Nebrodi (southern Italy) (<a href="#fig7">Figure 7C</a>). Mating pairs from Molliq and Nebrodi grew numerous perithecia, which are the fruiting bodies specific to sexual reproduction (<a href="#fig7">Figure 7D</a>). Pairings of Bosnian isolates showed no perithecia formation. This could have been caused by undetected variation in isolate viability affecting mating competence, mutation accumulation, or environmental sensitivity. Using molecular mating type assays, we recovered both mating types among the ascospores produced from successful matings.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>We analyzed the invasion of the chestnut blight pathogen <i>C. parasitica</i> across southeastern Europe after a first establishment on the continent. The European population is characterized by the presence of a small number of highly distinct genotypes. Deep sampling of recent invasive genotypes showed that the secondary invasion in southeastern Europe was caused by a single, highly homogeneous lineage consisting nearly exclusively of a single mating type. Given the likely origin from the European gene pool with clear evidence for sexual reproduction, the invasive lineage likely transitioned from biparental mating populations to a single mating type. The spread across southeastern Europe left no clear geographic imprint. This lack of genetic differentiation along the invasion route may be due to high levels of gene flow after the initial colonization or a very rapid spread from the center of origin to multiple locations across southeastern Europe. We show that ongoing TE activity created unexpected levels of insertion polymorphism within the invasive lineage. In addition, the lineage carries a distinct gene deletion and duplication profile compared to the European and North American pool of <i>C. parasitica.</i></p> <div class="theestablishmentofaeuropeanbridgeheadpopulation" title="sec"> <div class="title" tagx="title" title="title">
The establishment of a European bridgehead population</div> <p>Central and southeastern European <i>C. parasitica</i> populations likely originated from North American lineages. Clusters of European and North American genotypes were largely overlapping. Shared genotype clusters between the continents are consistent with historic gene flow from North America to Europe. These genome-scale analyses are consistent with previous findings documenting multiple North American introductions into Central Europe, but also directly from Asia to Western Europe (<a href="#bib24">DemenÃ© et al., 2019</a>; <a href="#bib28">Dutech et al., 2012</a>; <a href="#bib74">Milgroom et al., 1996</a>). Our results show that Central and southeastern European <i>C. parasitica</i> populations were largely established from North American sources alone consistent with previous studies. We were unable to sufficiently sample the North American source populations, and, hence, demographic history analyses of North American and European populations remained inconclusive. However, the high diversity at the genetic level with polymorphism at the level of SNPs, TEs, and CNV, as well as at the level of vegetative compatibility types, suggests that Europe was repeatedly colonized over the past century. Although genetic diversity could also have accumulated in situ in Europe, the establishment of a large set of genotypes and vegetative compatibility types seems diï¬ƒcult to explain with population age alone. Sexual recombination between the three most common vegetative compatibility types in Europe (i.e., EU-01, EU-02, and EU-05; <a href="#bib88">Robin and Heiniger, 2001</a>) could not account for the observed vegetative compatibility-type diversity. Observational records date the first introductions into Europe to the 1930s (<a href="#bib88">Robin and Heiniger, 2001</a>). Hence, based on the observed genetic diversity, a scenario of repeated introductions of different vegetative compatibility types since theÂ 1930s seems most plausible.</p> <p>Within Europe, populations from southern Switzerland, Slovenia, Croatia, and Bosnia are highly diverse and have nearly balanced mating type ratios. We also found no clear genetic structure according to vegetative compatibility types. This strongly suggests frequent outcrossing and population admixture, consistent with reports of perithecia in the field (<a href="#bib54">JeÅ¾iÄ‡ et al., 2012</a>; <a href="#bib82">Prospero et al., 2006</a>; <a href="#bib84">Prospero and Rigling, 2012</a>; <a href="#bib103">Trestic et al., 2001</a>). Low vegetative compatibility-type diversity in most European <i>C. parasitica</i> populations was thought to have contributed to low population admixture within Europe compared to Asia and North America (<a href="#bib20">Cortesi et al., 1996</a>; <a href="#bib28">Dutech et al., 2012</a>; <a href="#bib84">Prospero and Rigling, 2012</a>). However, our genome-wide analyses revealed frequent and ongoing in situ admixture in Europe. Thus, vegetative compatibility-type diversity does not necessarily underpin population admixture frequency and genetic diversity in sexually recombining populations. Our findings show that in asexually reproducing populations, such as in the S12 lineage, genotypes tend to cluster according to vegetative compatibility types.</p> </div> <div class="emergenceofaninvasivelineagefromaeuropeanbridgehead" title="sec"> <div class="title" tagx="title" title="title">
Emergence of an invasive lineage from a European bridgehead</div> <p>The invasive lineage S12 most likely arose from existing genotypes established in Europe. The closest genotypes to the dominant S12 MAT-1 were S12 MAT-2 isolates found in Bosnia, Kosovo, and southern Italy. Analyses based on a coancestry matrix identified populations from Bosnia, Croatia, Switzerland, and Georgia, but also Spain sharing the highest ancestry with the S12 lineage. Although we cannot exclude the existence of yet undescribed genetic diversity in North American populations, our results strongly indicate that introductions from outside of Europe are unlikely to explain the emergence of S12. The lineage carries a unique set of copyÂ number variants compared to other European genotypes underlining the observation of a recombinant S12 genotype. Furthermore, the emergence of S12 was accompanied by a striking evolutionary transition from mixed mating type populations to single mating type outbreak populations. Human activity may have contributed to the shift toward single mating type populations. Shipments of infected chestnut seedlings from Northern Italy and other trading activities could have disseminated the invasive lineage further South. This would have exposed the pathogen to the geographically more fragmented chestnut forests typically found in southeastern Europe where asexuality or selfing may be advantageous in theÂ absence of mating partners. Although <i>C. parasitica</i> is able to produce asexual conidia in large quantities, these specific spores are thought to be mainly splash dispersed by rain over short distances (<a href="#bib42">Griffin, 1986</a>). Accounting for occasional dispersal by birds or insects (<a href="#bib46">Heald and Studhalter, 1914</a>), conidia dispersal is unlikely to contribute substantially to the colonization of new areas. Moreover, human-mediated dispersal could have introduced the lineage to other regions including the Iberian Peninsula and Georgia.</p> <p>Despite the loss of a mating type in the S12 lineage, we found limited evidence for reticulation indicating at least low levels of recombination. If mating in <i>C. parasitica</i> follows the canonical process found in many ascomycetes, isolates of opposite mating type are required. Hence, S12 isolates of mating type MAT-1 may sporadically mate with rare S12 isolates of mating type MAT-2, which are comparatively more diverse. The emergence of the opposite mating type at low frequency could be the result of recombination with other genotypes and subsequent backcrossing. Combined with experimental evidence, we show that the dominant S12 mating type MAT-1 has retained the ability for sexual reproduction. Furthermore, in Bosnia, Croatia, Italy (Sicily), and Turkey, the S12 lineage coexists with other genotypes (i.e., vegetative compatibility types EU-01 and EU-02) of both mating types, potentially enabling sexual recombination and diversification in situ. The invasive S12 lineage was potentially pre-adapted to the southeastern European niche as we traced the origins to a likely bridgehead population located in southern Switzerland, northern Italy, or the northern Balkans. Niche availability and benefits associated with asexual reproduction to colonize new areas may have pre-disposed the European <i>C. parasitica</i> bridgehead population to produce a highly invasive lineage. Such potential pre-adaptation to the southeastern European environmental conditions, as well as reduced niche availability, may also explain why genotypes belonging to the S12 lineage have rarely been found elsewhere in Europe.</p> </div> <div class="expansionandmutationaccumulationwithintheinvasivelineage" title="sec"> <div class="title" tagx="title" title="title">
Expansion and mutation accumulation within the invasive lineage</div> <p>The S12 lineage diversified largely through mutation accumulation as nearly all high-confidence SNPs were identified as singletons. Mutation accumulation in theÂ absence of substantial recombination resulted in star-like phylogenetic relationships. We found a surprisingly high degree of differentiation among S12 genomes at the level of TE insertion polymorphism. Hence, active transposition of TEs is an important factor in diversifying the invasive lineage and possibly underpin future adaptive evolution. We found no evidence for a preferential association of candidate effector genes with TEs. Even though genome compartmentalization is a feature in many fungal genomes, the association of effector genes with repeat-rich regions is not consistent among pathogens (<a href="#bib102">Torres et al., 2020</a>). Necrotrophic pathogens like <i>C. parasitica</i> often rely less on effector-based gene-for-gene interactions and tend to deploy toxins or cell-wall degrading enzymes (<a href="#bib35">Friesen et al., 2008</a>; <a href="#bib102">Torres et al., 2020</a>). Interestingly, <i>C. parasitica</i> has recently experienced a loss of genes associated with carbohydrate metabolism and cell-wall degradation, which may have reduced exposure to the host immunity system and increased virulence (<a href="#bib98">Stauber et al., 2020</a>). The lack of genes with confirmed pathogenicity functions largely prevents thorough investigations on factors facilitating virulence evolution (e.g., epigenetic silencing or sequence rearrangements).</p> <p>Analyses of allele frequency spectra suggested that the broader European <i>C. parasitica</i> populations effectively removed the most deleterious mutations through purifying selection. In contrast, the S12 lineage shows strong skews toward very low minor allele frequencies of all mutation categories. Interestingly, we found a broader spread in allele frequencies for nearly neutral mutations in the S12 lineages. This suggests that despite the largely clonal population structure deleterious mutations can still be removed through low levels of recombination and purifying selection. Using accumulated mutations as markers to retrace the spatial expansion of the invasive S12 lineage, we found no indication for a step-wise geographic expansion along potential invasion routes. A lack of genetic clustering across southeastern Europe may be a consequence of high levels of potentially human-mediated gene flow frequently introducing new genotypes over large distances. However, the lack of geographic structure could also have its origins from substantial population bottlenecks during the spread of S12 across southeastern Europe. Finally, the largely clonal lineage may also become exposed to processes such as Mullerâ€™s Ratchet fixing deleterious mutations over time (<a href="#bib30">Felsenstein, 1974</a>).</p> <p>We show how a highly invasive fungal pathogen lineage in Europe can emerge from an intermediate, genetically diverse bridgehead population in Europe. This is in line with the self-reinforcement invasion model where initial introductions promote secondary spread (<a href="#bib6">Bertelsmeier et al., 2018</a>; <a href="#bib37">Garnas et al., 2016</a>). However, empirical evidence for adaptation in bridgehead populations is often elusive. Additionally, human activity (<a href="#bib4">Banks et al., 2015</a>) and host naivety of the European chestnut could have contributed substantially to the rapid spread without the need for diversification and adaptation in the bridgehead population. To date, there is no evidence for the emergence of tolerance or resistance against <i>C. parasitica</i> in European chestnut populations. However, fungal virulence may be severely reduced by the naturally occurring or artificially introduced parasitic mycovirus CHV1 (<a href="#bib87">Rigling and Prospero, 2018</a>). Interestingly, mycovirus spread is most efficient in asexual <i>C. parasitica</i> populations such as the invasive S12 lineage. This is because isolates of the same lineage lack vegetative incompatibility barriers among each other. The mycovirus has been reported in southeastern Europe where it shows population structure in contrast to the fungal host (<a href="#bib16">Bryner and Rigling, 2012</a>). Our study included only CHV1-free S12 isolates, hence potential associations of viral load, genetic identity, and geographic structure remain to be investigated. Outcrossing populations often harbor many different vegetative compatibility groups slowing viral transmission (<a href="#bib88">Robin and Heiniger, 2001</a>). Hence, the presence of the mycovirus may confer a selective advantage to retain sexual cycles and diversify vegetative compatibility types. In turn, diversification may reduce the evolutionary advantage of the invasive lineage over time.</p> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="samplesofc.parasitica" title="sec"> <div class="title" tagx="title" title="title">
Samples of <i>C. parasitica</i></div> <p>A total of 230 <i>C. parasitica</i> were sequenced. A majority of 182 isolates originated from Albania, Bosnia, Bulgaria, Croatia, Greece, Italy, Kosovo, Macedonia, Serbia, Slovenia, Switzerland, and Turkey (<a href="#fig3">Figure 3B</a>, <a href="#supp1">Supplementary file 1</a>). All 182 isolates originating from Central and southeastern Europe tested to be virulent and mycovirus-free. The 48 other isolates were from China (<i>n</i> = 7), South Korea (<i>n</i> = 8), Japan (<i>n</i> = 8), North America (<i>n</i> = 8), Azerbaijan (<i>n</i> = 3), Georgia (<i>n</i> = 4), Portugal (<i>n</i> = 3), Spain (<i>n</i> = 3), and France (<i>n</i> = 5) (<a href="#supp1">Supplementary file 1</a>). A total of 125 European isolates belonged to the vegetative compatibility type EU-12, whereas 57 isolates represented other vegetative compatibility types (EU-types) occurring in Central and southeastern Europe. We additionally selected 45 isolates from Bulgaria, Greece, Italy, and Macedonia, which were already included in a previous population-wide study on southeastern European <i>C. parasitica</i> diversity by <a href="#bib75">Milgroom et al., 2008</a>. All samples were collected between 1951 and 2018 and are stored as glycerol stocks at â€"80Â°C in the culture collection of the Swiss Federal Research Institute WSL.</p> </div> <div class="dnaextractionandgenotyping" title="sec"> <div class="title" tagx="title" title="title">
DNA extraction and genotyping</div> <p>All isolates were first inoculated onto cellophane-covered potato dextrose agar plates (PDA, 39 g/L; BD Becton, Dickinson and Company, Franklin Lakes, USA) (<a href="#bib50">Hoegger et al., 2000</a>) and incubated for a minimum of 1 week at 24Â°C, at a 14 hr light and 10 hr darkness cycle. After a sufficient amount of mycelium and spores had grown, the isolates were harvested by scratching the mycelial mass oï¬€ the cellophane, transferring it into 2 mL tubes and freeze-drying it for 24 hr. For DNA extraction, 15â€"20 mg of dried material was weighted and single-tube extraction was performed using the DNeasy Plant Mini Kit (Qiagen, Hilden, Germany). DNA quantity was measured using the Invitrogen Qubit 3.0 Fluorometer (Thermo Fisher Scientific, Waltham, MA), and DNA quality was assessed using the Nanodrop 1000 Spectrophotometer (Thermo Fisher Scientific). Prior to sequencing, all isolates were screened for their genotype at 10 microsatellite markers (<a href="#bib84">Prospero and Rigling, 2012</a>). Additionally, the isolates were screened for their vegetative compatibility and mating type alleles in two multiplex PCRs, as described in <a href="#bib19">Cornejo et al., 2019</a>. Allele sizes for the genotyping ofÂ vegetative compatibility and mating types were scored with GeneMapper 5 (Thermo Fisher Scientific).</p> </div> <div class="illuminawhole-genomesequencing,variantcalling,andfiltration" title="sec"> <div class="title" tagx="title" title="title">
Illumina whole-genome sequencing, variant calling, and filtration</div> <p>Isolates were prepared for sequencing using the TrueSeq Nano DNA HT Library Preparation kit (Illumina, San Diego, CA). The libraries were sequenced at the Functional Genomics Centre Zurich (ETH Zurich and University of Zurich) on Illumina platforms (Illumina). The obtained sequences were trimmed with Trimmomatic v0.36 (<a href="#bib11">Bolger et al., 2014</a>) and aligned with Bowtie 2 v2.3.5.1 (<a href="#bib61">Langmead and Salzberg, 2012</a>) and SAMtools v1.9 (<a href="#bib63">Li et al., 2009</a>) to the <i>C. parasitica</i> reference genome (43.9 Mb) EP155 v2.0 (<a href="#bib22">Crouch et al., 2020</a>) available at the Joint Genome Institute (<a href="http://jgi.doe.gov/">http://jgi.doe.gov/</a>). Variant calling, selection, and filtration were conducted with the Genome Analysis Toolkit GATK v3.8 and v4.0.2.0 (<a href="#bib71">McKenna et al., 2010</a>). We retained variants satisfying the following filtration parameters: QUAL:â‰¥100, MQRankSum (lower):â‰¥âˆ’2.0, QD:â‰¥20.0, MQRankSum (upper):â‰¤2.0, MQ:â‰¥20.0, BaseQRankSum (lower): â‰¥ - 2.0, ReadPosRankSum (lower):â‰¥âˆ’2.0, ReadPosRankSum (upper):â‰¤2.0, BaseQRankSumÂ (upper):â‰¤2.0 (<a href="#fig1s2">Figure 1â€"figure supplement 2</a>). Furthermore, we used BCFtools v1.9 (<a href="#bib78">Narasimhan et al., 2016</a>) and the R package vcfR (<a href="#bib58">Knaus and GrÃ¼nwald, 2017</a>) for querying and VCFtools v0.1.16 (<a href="#bib23">Danecek et al., 2011</a>) for downstream variant filtering. Variants were additionally filtered for minor allele countÂ â‰¥1, excluding all missing data. Sites were also filtered per genotype, only keeping biallelic SNPs with a minimum depth of 3 and a genotyping quality of 99. To exclude SNPs associated with the mating type, we ran an association study with TASSEL 5 (<a href="#bib13">Bradbury et al., 2007</a>) and retrieved p-values for each SNP across the genome. We set a p-value threshold of pâ‰¤1Ã—10<sup>âˆ’10</sup> to remove all mating type-associatedÂ SNPs for further analysis. The SNPs showing strong association with the mating type were located on scaï¬€old 2 between positions 179,552 â€" 2,084,312 bp (<a href="#fig1s1">Figure 1â€"figure supplement 1</a>).</p> </div> <div class="phylogeneticreconstruction" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic reconstruction</div> <p>The filtered whole-genome SNP dataset was used to build unrooted phylogenetic networks using SplitsTree v4.14.6 (<a href="#bib52">Huson and Bryant, 2006</a>). SplitsTree was also used for calculating the PHI test (<a href="#bib15">Bruen et al., 2006</a>) to test for recombination. The required file conversions for SplitsTree (i.e., from VCF to FASTA format) were done with PGDSpider v2.1.1.5 (<a href="#bib65">Lischer and Excoffier, 2012</a>). PCA wasÂ performed as implemented in the R package ade4 (<a href="#bib12">Bougeard and Dray, 2018</a>).</p> </div> <div class="inferenceofs12donorpopulations" title="sec"> <div class="title" tagx="title" title="title">
Inference of S12 donor populations</div> <p>We generated an averaged coancestry matrix as inferred by fineSTRUCTURE v2.1.3 (<a href="#bib62">Lawson et al., 2012</a>). The software uses a MCMC-based algorithm to infer ancestral contributions based on patterns of haplotype similarity. For analysis, we selected the unlinked model and generated the required input files (â€˜phaseâ€™ files) in R. We ran the fineSTRUCTURE pipeline in â€˜automatic modeâ€™, with -s3iters 500000, -s4iters 100000, -maxretained 10000, -s2chunksperregion 20, and ploidy set to 1. Convergence was assessed through MCMC traces (<a href="#fig4s1">Figure 4â€"figure supplement 1</a>) and population assignments (i.e., pairwise coincidence matrix; <a href="#fig4s1">Figure 4â€"figure supplement 1</a>).</p> </div> <div class="populationgeneticanalysesofdemographyandselectionsignatures" title="sec"> <div class="title" tagx="title" title="title">
Population genetic analyses of demography and selection signatures</div> <p>We computed allele frequencies and estimated the allele spectrum using VCFtools. We used RStudio (<a href="#bib90">RStudio Team, 2015</a>) and ggplot2 (<a href="#bib108">Wickham, 2016</a>) for visualizations. Synonymous and non-synonymous sites were identified and annotated using SnpEï¬€ v4.3t (<a href="#bib18">Cingolani et al., 2012</a>). Variants that were classified by SnpEï¬€ as having a â€˜highâ€™, â€˜moderateâ€™, â€˜lowâ€™, or â€˜modifyingâ€™ impact on the encoded protein sequences. The data was summarized in R using the packages dplyr (<a href="#bib109">Wickham et al., 2018</a>), reshape2 (<a href="#bib107">Wickham, 2007</a>), tidyr (<a href="#bib110">Wickham and Henry, 2018</a>), andggplot2. Furthermore, we performed a genome-wide association study with TASSEL 5 (<a href="#bib13">Bradbury et al., 2007</a>) for identifying highly associated SNPs with genetic groups such as S12. Nucleotide diversity was calculated with vcftools -site-pi function using vcf-files that were converted to diploid.</p> <p>The demographic history of the populations was investigated based on the folded SFS using dadi (<a href="#bib44">Gutenkunst et al., 2009</a>). The vcf file was filtered to remove variants predicted to have a â€˜highâ€™ to â€˜moderateâ€™ effect by SnpEff and thinned with a 5 kb distance in order to obtain a set of most likely neutral SNPs. We grouped the isolates of the two smaller clusters of the PCA together (isolates from China, South Korea, Japan, and Georgia) in order to obtain a sufficiently large number of samples. For the European cluster, we kept only one randomly selected S12 isolate to avoid a bias from the large number of S12 clones. We compared the observed SFS against the SFS generated under several demographic models implemented in <i>dadi</i> (sub-module dadi.Demographics1D). For all models, we ran the optimization 50 times with initial parameters drawn randomly from the bounded distributions (specified for each model below) and the optimization method <i>optimiselog</i> with the maximum number of iterations set to 100 (see <a href="#supp2">Supplementary file 2</a> for initial parameters). Models were compared with the AIC. The AIC penalizes models with a higher number of parameters in order to identify the model explaining the largest amount of variation using the fewest possible independent variables. The compared models are described below and include the following parameters: nuF representing the ratio of contemporary to ancient population size, nuB representing the ratio of intermediate population size to ancient population size (relevant for some models only), and T representing the time at which the instantaneous size change happened (in units of 2*Na generations).</p> <ul> <li> <p>Standard neutral (i.e., constant) model (snm function) without population size change.</p> </li> <li> <p>Sudden expansion model (two_epoch function) with nuF bounded between 1 and 1000 and T between 0 and 2000.</p> </li> <li> <p>Sudden contraction model (two_epoch function) with nuF bounded between 1 and 1000 and T between 0 and 2000.</p> </li> <li> <p>Model implementing a single instantaneous change in population size at a specific point in time followed by exponential growth (bottlegrowth function) with nuF bounded between 0.01 and 1000, nuB between 0.01 and 100, and T between 0 and 500.</p> </li> <li> <p>Model implementing a sudden population contraction followed by a sudden expansion (i.e., a bottleneck with recovery; three-epoch function) with nuF bounded between 1 and 2000, nuB between 0.01 and 1, TB (length of the bottleneck) between 0 and 100, and TF (time since the bottleneck) between 0 and 500.</p> </li> </ul> <p>Datasets and scripts for all demographic analyses are accessible here: <a href="https://github.com/crolllab/datasets">https://github.com/crolllab/datasets</a>Â (<a href="#bib60">Laboratory of Evolutionary Genetics @ UNINE, 2021</a>; copy archived at <a href="https://archive.softwareheritage.org/swh:1:dir:e3c098b0dcd2bba3e201d3d13293e544c2d58232;origin=https://github.com/crolllab/datasets;visit=swh:1:snp:7e5189913de8400eed7cc2e7b8041677ecfefb2a;anchor=swh:1:rev:fa908efdb71d5ba90823c70bf2467e10ce6f12a7/">swh:1:rev:fa908efdb71d5ba90823c70bf2467e10ce6f12a7</a>).</p> <p>Identification of putative selective sweeps was performed using RAiSD v2.5 software that is based on three diï¬€erent genetic signatures of positive selection (with -y 1 -M 0 -w 50 -c 1 parameters) (<a href="#bib1">Alachiotis and Pavlidis, 2018</a>). The Manhattan plot shows the distribution of the computed composite <i>Âµ</i> statistic of positive selection. To investigate potential false positive selection signals due to the demographic history of the populations, we performed 10,000 ms simulations to model three demographic scenarios in the European population: constant, expansion, and bottleneck. We used the average theta value obtained from the demographic simulations with <i>dadi</i> on the two populations for which we could estimate the best fitting demographic history model. The recombination rate was estimated with LDhat v2.2 (<a href="#bib72">McVean and Auton, 2007</a>), with -missfreqcut 0, -samp 2000, -its 5000000, -bpen 20, and -burn 50 (<a href="#fig3s2">Figure 3â€"figure supplement 2</a>). The average values of phi and of pairwise SNP distance across the 11 scaffolds were used for ms simulations (<a href="#bib51">Hudson, 2002</a>). The bottleneck scenario was simulated given the parameters â€˜ms 10 10000- t 558 -r 0.0032457 12980 -eN .10000000000000000000 0.01â€™. For the expansion, we considered population doubling â€˜ms 10 10000 -t 558 -r 0.0032457 12980 -eN .10000000000000000000 2â€™, and the constant population model was performed given identical theta and phi parameters â€˜ms 10 10000 -t 558 -r 0.0032457 12980â€™. We performed RAiSD selection scans on each of the three simulated datasets of different demographic scenarios and extracted the highest value of the <i>Âµ</i> statistic.</p> </div> <div class="denovoidentificationoftesandeffectorprediction" title="sec"> <div class="title" tagx="title" title="title">
De novo identification of TEs and effector prediction</div> <p>We performed a de novo identification of TEs in the <i>C. parasitica</i> reference genome EP155 v2.0 using RepeatModeler v2.0.1 (<a href="#bib31">Flynn et al., 2019</a>). The consensus sequences were merged with the RepBase library (RepBaseRepeatMaskerEdition-20181026) and then used for repeat annotation using RepeatMasker v4.0.7 with a blast cutoï¬€ of 250 (<a href="#bib94">Smit et al., 2015</a>). Repeats were then filtered out for low complexity and simple repeats, and parsed using the parseRM_merge_interrupted.pl script from <a href="https://github.com/4ureliek/Parsing-RepeatMasker-Outputs">https://github.com/4ureliek/Parsing-RepeatMasker-Outputs</a>. We only retained TEs longer than 100 bp, and overlapping identical TEs were merged into single elements for the final annotation. In addition, TEs of the same family separated by less than 200 bp were considered as part of the same TE and merged into a single element. We analyzed population-level presence/absence variation of TEs using the R-based tool ngs_te_mapper, using trimmed reads as above and bwa version 0.7.17-r1188 (<a href="#bib5">Bergman, 2012</a>; <a href="#bib64">Li and Durbin, 2009</a>). Overlapping genes and TEs were identified using the intersect function from the bedtools suite v2.29 (<a href="#bib85">Quinlan and Hall, 2010</a>). For effector prediction, we first identified putatively secreted proteins for the <i>C. parasitica</i> reference genome with SignalP and TMHMM as implemented in InterProScan v5.31â€"70.0 (<a href="#bib55">Jones et al., 2014</a>). Then, protein sequences with a predicted secretion signal predicted by both tools were extracted with samtools v1.9 (<a href="#bib63">Li et al., 2009</a>) and used as input for effector prediction with EffectorP v2.0 (<a href="#bib97">Sperschneider et al., 2018</a>). EffectorP comes pre-trained on gene sequences encoding functionally confirmed fungal effectors.</p> </div> <div class="genomecompartmentalizationandcnvanalyses" title="sec"> <div class="title" tagx="title" title="title">
Genome compartmentalization and CNV analyses</div> <p>We investigated the genome architecture of <i>C. parasitica</i> following the protocol described in <a href="#bib92">Saunders et al., 2014</a>. Briefly, we computed intergenic distances with the Calculate_FIR_length.pl script using the gene prediction for <i>C. parasitica</i> reference genome EP155 v2.0. We defined 40 bins given the range of 3â€² and 5â€² intergenic distances and calculated the number of genes that fell within each bin. To infer CNV, we computed the normalized read depth (NRD)Â for all isolates using the CNVcaller pipeline (<a href="#bib106">Wang et al., 2017</a>). For this, the reference genome was first split into 800 bp overlapping kmers that were re-aligned to the reference using blasrÂ (-m 5 -noSplitSubreads -minMatch 15 -maxMatch 20 -advanceHalf -advanceExactMatches 10 -fastMaxInterval -fastSDP -aggressiveIntervalCut -bestn 10) to identify duplicated windows (python3 0.2.Kmer_Link.py ref.genome.kmer.aln 800 ref.genome.800.window.link). The resulting windows were then used to calculate the NRD for each isolate from the aligned reads (bam format) in 800 bp windows (Individual.Process.sh -b $bam -h ${i% .bam} -d ref.genome.800.window.link -s scaï¬€old_1). For all further analyses, we used the read depth normalized for the absolute copy number and GC content of each sample (RD_normalized output). Given the observed distribution of the NRD across all isolates, we considered windows with NRDÂ &amp;gt;1.6 to be duplicated and windows with NRDÂ &amp;lt;0.4 as deleted considering the 5th and 95th percentiles of the normalized read coverage (<a href="#fig5s3">Figure 5â€"figure supplement 3</a>). To investigate the relative diversity within the S12 and non-S12 groups, we resampled 80 random isolates from each lineage 100 times and investigated SNP, TE, and CNV diversity. We quantified the minor allele frequency spectrum in each of the two groups and tested for the group effect (S12 vs. non-S12) on the genetic diversity using Fisher's exact tests in R. We simulated datasets using 2000 resampling replicates to generate expectations for random distributions (<a href="#fig5s1">Figure 5â€"figure supplement 1</a>).</p> </div> <div class="matingexperiments" title="sec"> <div class="title" tagx="title" title="title">
Mating experiments</div> <p>The ability of <i>C. parasitica</i> isolates belonging to S12 with mating type MAT-1 to sexually outcross with isolates of opposite mating type was assessed in an inoculation experiment. For this, we randomly selected four S12 and one non-S12 isolate with mating type MAT-1, as well as five isolates of opposite mating type MAT-2 from populations in Italy (Nebrodi), Kosovo (Molliq), and Bosnia (Konic, VrnograÄ�, Projsa) (<a href="#supp4">Supplementary file 4</a>). All isolates of both mating types belonged to the vegetative compatibility type EU-12, and mating pairs were only formed between isolates from the same geographic source populations. As substrate for the pairings, 40-mm-long segments of dormant chestnut (<i>C. sativa</i>) stems (15â€"20 mm in diameter) were split longitudinally in half. The wood pieces were autoclaved twiceÂ and placed onto sterile Petri dishes (90 mm diameter), which were filled with 1.5% water agar to enclose the pieces. Mating pairs were inoculated on opposite sides of each halved stem with three replicates per pairing. The inoculated plates were then incubated at 25Â°C under a 16 hr photoperiod (2500 lux) for 14 days. After 2 weeks, mating was stimulated by adding 5 mL sterile water to the plates to suspend and distribute the conidia produced by both isolates over the stem segment. Any excess water was subsequently removed and the plates were incubated at 18Â°C under an 8 hr photoperiod (2500 lux). After 5 months of incubation, perithecia formation was assessed under a dissecting microscope. To confirm successful outcrossing, single perithecia were carefully extracted from the stromata and crushed in a drop of sterile distilled water. The resulting ascospore suspensions were plated on PDA and incubated at 25Â°C for 24â€"36 hr. Afterwards, single germinating ascospores were transferred to PDA and incubated in the dark for 3â€"5 days at 25Â°C. DNA was then extracted from 10 mg of lyophilized mycelium using the kit and instructions by KingFisher (Thermo Fisher Scientific). All single ascospore cultures were screened for mating types by performing a multiplex PCR following the protocol described in <a href="#bib19">Cornejo et al., 2019</a>.</p> </div> </div> </div> <div class="back" title="back"> <div class="funding-information" title="funding-information"> <div class="title" tagx="title" title="title">
Funding Information</div> <p>This paper was supported by the following grants:</p> <ul> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001711</span><span class="institution" title="institution">Schweizerischer Nationalfonds zur FÃ¶rderung der Wissenschaftlichen Forschung</span></div> </div><span class="award-id" tagx="award-id" title="award-id">170188</span> to Simone Prospero.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Fondation Pierre Mercier pour la science</span></div> </div> to Daniel Croll.<p /> </li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We are grateful to Ursula Oggenfuss for helpful comments on a previous manuscript version. We thank Sandrine Fattore, HÃ©lÃ¨ne Blauenstein, Quirin Kupper, Eva Augustiny, Dario RÃ¼egg, and Silvia Kobel for laboratory assistance. We acknowledge the Genetic Diversity Centre (GDC), ETH Zurich, for technical support and facility access. Ludwig Beenken and Martin Wrann helped with documenting mating experiments. Paolo Cortesi, Michael Milgroom, Kiril Sotirovski, Mihajlo Risteski, Marin JeÅ¾iÄ‡, and SeÃ§il Akilli kindly provided samples. We thank Pierre Gladieux and Nikhil Kumar Singh for discussions and sharing scripts for data analyses. We are grateful for theÂ insightful discussions with Daniel Rigling. Funding was awarded to SP by the Swiss National Science Foundation (grant 170188) and to DC by the Fondation Pierre Mercier pour la science.</p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional information</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Competing interests</b></div> <div class="fn-type-COI-statement" title="COI-statement"> <p>No competing interests declared.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Author contributions</b></div> <div class="fn-type-con" title="con"> <p>Conceptualization, Data curation, Formal analysis, Investigation, Visualization, Writing - original draft, Writing - review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Formal analysis, Investigation, Visualization, Writing - review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Formal analysis, Writing - review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Supervision, Funding acquisition, Writing - review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Supervision, Funding acquisition, Visualization, Writing - original draft, Writing - review and editing.</p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 1.</span> <div class="title" tagx="title" title="title">
Information on all analyzed isolates including sampling location, vegetative compatibility type (EU-type), (SSR)Â simple sequence repeats genotyping outcome, collector information, collection years, and NCBI accessions.</div>  <div class="media" title="media"><a href="elife-56279-supp1.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 2.</span> <div class="title" tagx="title" title="title">
Initial and best-fit parameters as well as likelihoods and Akaike information criterion (AIC) values as inferred by <i>dadi</i> for demographic analyses.</div> <p>Cluster names correspond to clusters as shown in <a href="#fig1">Figure 1A</a>.</p>  <div class="media" title="media"><a href="elife-56279-supp2.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 3.</span> <div class="title" tagx="title" title="title">
List of the genes in regions with signatures of recent positive selection (based on RAiSD).</div> <p>Putative functions are shown by (PFAM)Â Protein families (database) and Gene Ontology (GO) annotations. Single nucleotide polymorphismsÂ (SNPs) and the corresponding composite Î¼ statistic are shown.</p>  <div class="media" title="media"><a href="elife-56279-supp3.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary file 4.</span> <div class="title" tagx="title" title="title">
Summary of MAT-1 and MAT-2 isolate pairings used for testing sexual recombination in S12 populations.</div> <p>*Isolates from the S12 MAT-1 lineage.</p>  <div class="media" title="media"><a href="elife-56279-supp4.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Transparent reporting form</span><div class="media" title="media"><a href="elife-56279-transrepform.pdf">LINK</a></div> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>All raw sequencing data is available on the NCBI Short Read Archive (BioProjects PRJNA604575 and PRJNA644891).</p> <p>The following datasets were generated:</p> <p><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stauber</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croll</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2020</span></span></p><div tag="data-title">
Population sequencing of Cryphonectria parasitica</div><span class="source" tagx="source" title="source">NCBI BioProject</span><span class="pub-id" /><p /> <p><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stauber</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croll</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2020</span></span></p><div tag="data-title">
Population sequencing of Cryphonectria species</div><span class="source" tagx="source" title="source">NCBI BioProject</span><span class="pub-id" /><p /> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alachiotis</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pavlidis</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">RAiSD detects positive selection based on multiple signatures of a selective sweep and SNP vectors</span><span class="source" tagx="source" title="source">Communications Biology</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s42003-018-0085-8">10.1038/s42003-018-0085-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29809203">29809203</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allendorf</span><span class="given-names" tagx="given-names" title="given-names">FW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lundquist</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Introduction: population biology, evolution, and control of invasive species</span><span class="source" tagx="source" title="source">Conservation Biology</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">24</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1523-1739.2003.02365.x">10.1046/j.1523-1739.2003.02365.x</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avolio</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">1978</span><span class="mixed-article-title" title="mixed-article-title">La castanicoltura in Calabria</span><span class="source" tagx="source" title="source">Aspetti Selvicolturali E Linee Di Intervento Lâ€™Italia for E Mont</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">278</span><span class="lpage" tagx="lpage" title="lpage">296</span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banks</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paini</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodda</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">The role of global trade and transport network topology in the human-mediated dispersal of alien species</span><span class="source" tagx="source" title="source">Ecology Letters</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">188</span><span class="lpage" tagx="lpage" title="lpage">199</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/ele.12397">10.1111/ele.12397</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25529499">25529499</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bergman</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">A proposal for the reference-based annotation of de novo transposable element insertions</span><span class="source" tagx="source" title="source">Mobile Genetic Elements</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">51</span><span class="lpage" tagx="lpage" title="lpage">54</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/mge.19479">10.4161/mge.19479</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22754753">22754753</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertelsmeier</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ollier</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liebhold</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brockerhoff</span><span class="given-names" tagx="given-names" title="given-names">EG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ward</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keller</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Recurrent bridgehead effects accelerate global alien ant spread</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">5486</span><span class="lpage" tagx="lpage" title="lpage">5491</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1801990115">10.1073/pnas.1801990115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29735696">29735696</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertelsmeier</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keller</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Bridgehead effects and role of adaptive evolution in invasive populations</span><span class="source" tagx="source" title="source">Trends in Ecology &amp;amp; Evolution</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">527</span><span class="lpage" tagx="lpage" title="lpage">534</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tree.2018.04.014">10.1016/j.tree.2018.04.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29764688">29764688</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Billiard</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LÃ³pez-Villavicencio</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hood</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giraud</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Sex, outcrossing and mating types: unsolved questions in fungi and beyond</span><span class="source" tagx="source" title="source">Journal of Evolutionary Biology</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1020</span><span class="lpage" tagx="lpage" title="lpage">1038</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1420-9101.2012.02495.x">10.1111/j.1420-9101.2012.02495.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22515640">22515640</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biraghi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1946</span><span class="source" tagx="source" title="source">Cancro Della Corteccia Del Castagno</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Ramo Editoriale Degli Agricoltori</span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bock</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caseys</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cousens</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heredia</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HÃ¼bner</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">KG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitney</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rieseberg</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">What we still don't know about invasion genetics</span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">2277</span><span class="lpage" tagx="lpage" title="lpage">2297</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.13032">10.1111/mec.13032</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25474505">25474505</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolger</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lohse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usadel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Trimmomatic: a flexible trimmer for illumina sequence data</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">2114</span><span class="lpage" tagx="lpage" title="lpage">2120</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu170">10.1093/bioinformatics/btu170</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404">24695404</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bougeard</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dray</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Supervised multiblock analysis in R with the ade4 package</span><span class="source" tagx="source" title="source">Journal of Statistical Software</span><span class="volume" tagx="volume" title="volume">86</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">i01</span><span class="pub-id"><a href="https://dx.doi.org/10.18637/jss.v086.i01">10.18637/jss.v086.i01</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bradbury</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kroon</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casstevens</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramdoss</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buckler</span><span class="given-names" tagx="given-names" title="given-names">ES</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">TASSEL: software for association mapping of complex traits in diverse samples</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">2633</span><span class="lpage" tagx="lpage" title="lpage">2635</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btm308">10.1093/bioinformatics/btm308</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586829">17586829</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brasier</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webber</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Is there evidence for post-epidemic attenuation in the dutch elm disease pathogen <i>Ophiostoma novo-ulmi</i>?</span><span class="source" tagx="source" title="source">Plant Pathology</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">921</span><span class="lpage" tagx="lpage" title="lpage">929</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/ppa.13022">10.1111/ppa.13022</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bruen</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Philippe</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bryant</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">A simple and robust statistical test for detecting the presence of recombination</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">172</span><span class="fpage" tagx="fpage" title="fpage">2665</span><span class="lpage" tagx="lpage" title="lpage">2681</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.105.048975">10.1534/genetics.105.048975</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16489234">16489234</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bryner</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigling</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Hypovirus virulence and vegetative incompatibility in populations of the chestnut blight fungus</span><span class="source" tagx="source" title="source">Phytopathology</span><span class="volume" tagx="volume" title="volume">102</span><span class="fpage" tagx="fpage" title="fpage">1161</span><span class="lpage" tagx="lpage" title="lpage">1167</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/PHYTO-01-12-0013-R">10.1094/PHYTO-01-12-0013-R</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22857516">22857516</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buccianti</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feliciani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1966</span><span class="mixed-article-title" title="mixed-article-title">LA SITUAZIONE ATTUALE DEI CASTAGNETI IN ITALIA (The present situation of chestnut stands in Italy)</span><span class="conf-name" tagx="conf-name" title="conf-name">AttiConvegno Internazionale Sul Castagno, Cuneo</span><span class="fpage" tagx="fpage" title="fpage">61</span><span class="lpage" tagx="lpage" title="lpage">85</span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cingolani</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Platts</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">leL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coon</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Land</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruden</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">A program for annotating and predicting the effects of single nucleotideÂ polymorphisms, SnpEff: SNPs in the genome of <i>Drosophila melanogaster</i> strain w1118; iso-2; iso-3</span><span class="source" tagx="source" title="source">Fly</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">80</span><span class="lpage" tagx="lpage" title="lpage">92</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/fly.19695">10.4161/fly.19695</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22728672">22728672</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cornejo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Å ever</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kupper</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigling</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">A multiplexed genotyping assay to determine vegetative incompatibility and mating type in <i>Cryphonectria parasitica</i></span><span class="source" tagx="source" title="source">European Journal of Plant Pathology</span><span class="volume" tagx="volume" title="volume">155</span><span class="fpage" tagx="fpage" title="fpage">81</span><span class="lpage" tagx="lpage" title="lpage">91</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10658-019-01751-w">10.1007/s10658-019-01751-w</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cortesi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bisiach</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Distribution and diversity of vegetative compatibility types in subpopulations of <i>Cryphonectria parasitica</i> in Italy</span><span class="source" tagx="source" title="source">Mycological Research</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">1087</span><span class="lpage" tagx="lpage" title="lpage">1093</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0953-7562(96)80218-8">10.1016/S0953-7562(96)80218-8</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cortesi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCulloch</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Genetic control of horizontal virus transmission in the chestnut blight fungus, <i>Cryphonectria parasitica</i></span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">159</span><span class="fpage" tagx="fpage" title="fpage">107</span><span class="lpage" tagx="lpage" title="lpage">118</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11560890">11560890</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crouch</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dawe</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aerts</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barry</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Churchill</span><span class="given-names" tagx="given-names" title="given-names">ACL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grimwood</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hillman</span><span class="given-names" tagx="given-names" title="given-names">BI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pangilinan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salamov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmutz</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grigoriev</span><span class="given-names" tagx="given-names" title="given-names">IV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nuss</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span></span><span class="year" tagx="year" title="year">2020</span><span class="mixed-article-title" title="mixed-article-title">Genome sequence of the chestnut blight fungus <i>Cryphonectria parasitica</i> EP155: A fundamentalÂ resource for an archetypical invasive plant pathogen</span><span class="source" tagx="source" title="source">Phytopathology</span><span class="volume" tagx="volume" title="volume">110</span><span class="fpage" tagx="fpage" title="fpage">1180</span><span class="lpage" tagx="lpage" title="lpage">1188</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/PHYTO-12-19-0478-A">10.1094/PHYTO-12-19-0478-A</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32207662">32207662</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danecek</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Auton</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abecasis</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albers</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banks</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DePristo</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Handsaker</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lunter</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marth</span><span class="given-names" tagx="given-names" title="given-names">GT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sherry</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McVean</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durbin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="collab'">collab: 1000 Genomes Project Analysis Group</span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">The variant call format and VCFtools</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">2156</span><span class="lpage" tagx="lpage" title="lpage">2158</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr330">10.1093/bioinformatics/btr330</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522">21653522</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DemenÃ©</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Legrand</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouzy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debuchy</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saint-Jean</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fabreguettes</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutech</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Whole-genome sequencing reveals recent and frequent genetic recombination between clonal lineages of <i>Cryphonectria parasitica</i> in western Europe</span><span class="source" tagx="source" title="source">Fungal Genetics and Biology</span><span class="volume" tagx="volume" title="volume">130</span><span class="fpage" tagx="fpage" title="fpage">122</span><span class="lpage" tagx="lpage" title="lpage">133</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fgb.2019.06.002">10.1016/j.fgb.2019.06.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31175938">31175938</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raffaele</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamoun</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">The two-speed genomes of filamentous pathogens: waltz with plants</span><span class="source" tagx="source" title="source">Current Opinion in Genetics &amp;amp; Development</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">57</span><span class="lpage" tagx="lpage" title="lpage">65</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gde.2015.09.001">10.1016/j.gde.2015.09.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26451981">26451981</a></span></span></li> <li tag="ref"><a name="bib26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drenth</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McTaggart</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wingfield</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Fungal clones win the battle, but recombination wins the war</span><span class="source" tagx="source" title="source">IMA Fungus</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">18</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s43008-019-0020-8">10.1186/s43008-019-0020-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32647622">32647622</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutech</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fabreguettes</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Capdevielle</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Multiple introductions of divergent genetic lineages in an invasive fungal pathogen, <i>Cryphonectria parasitica</i>, in France</span><span class="source" tagx="source" title="source">Heredity</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">220</span><span class="lpage" tagx="lpage" title="lpage">228</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/hdy.2009.164">10.1038/hdy.2009.164</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19997121">19997121</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutech</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BarrÃ¨s</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bridier</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RavignÃ©</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">The chestnut blight fungus world tour: successive introduction events from diverse origins in an invasive plant fungal pathogen</span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">3931</span><span class="lpage" tagx="lpage" title="lpage">3946</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2012.05575.x">10.1111/j.1365-294X.2012.05575.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22548317">22548317</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elliott</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swank</span><span class="given-names" tagx="given-names" title="given-names">WT</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Long-term changes in forest composition and diversity following early logging (1919â€"1923) and the decline of American chestnut (<i>Castanea dentata</i>)</span><span class="source" tagx="source" title="source">Plant Ecology</span><span class="volume" tagx="volume" title="volume">197</span><span class="fpage" tagx="fpage" title="fpage">155</span><span class="lpage" tagx="lpage" title="lpage">172</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11258-007-9352-3">10.1007/s11258-007-9352-3</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felsenstein</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1974</span><span class="mixed-article-title" title="mixed-article-title">The evolutionary advantage of recombination</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">737</span><span class="lpage" tagx="lpage" title="lpage">756</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/genetics/78.2.737">10.1093/genetics/78.2.737</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4448362">4448362</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flynn</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubley</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goubert</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feschotte</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smit</span><span class="given-names" tagx="given-names" title="given-names">AFA</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">RepeatModeler2: automated genomic discovery of transposable element families</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/856591">10.1101/856591</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fontaine</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gladieux</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hood</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giraud</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">History of the invasion of the anther smut pathogen on <i>Silene latifolia</i> in North America</span><span class="source" tagx="source" title="source">New Phytologist</span><span class="volume" tagx="volume" title="volume">198</span><span class="fpage" tagx="fpage" title="fpage">946</span><span class="lpage" tagx="lpage" title="lpage">956</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/nph.12177">10.1111/nph.12177</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23406496">23406496</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FouchÃ©</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plissonneau</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croll</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">The birth and death of effectors in rapidly evolving filamentous pathogen genomes</span><span class="source" tagx="source" title="source">Current Opinion in Microbiology</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">34</span><span class="lpage" tagx="lpage" title="lpage">42</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mib.2018.01.020">10.1016/j.mib.2018.01.020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29455143">29455143</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frantzeskakis</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kusch</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panstruga</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">The need for speed: compartmentalized genome evolution in filamentous phytopathogens</span><span class="source" tagx="source" title="source">Molecular Plant Pathology</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mpp.12738">10.1111/mpp.12738</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30557450">30557450</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friesen</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faris</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solomon</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliver</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Host-specific toxins: effectors of necrotrophic pathogenicity</span><span class="source" tagx="source" title="source">Cellular Microbiology</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1421</span><span class="lpage" tagx="lpage" title="lpage">1428</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1462-5822.2008.01153.x">10.1111/j.1462-5822.2008.01153.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18384660">18384660</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garbelotto</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rocca</span><span class="given-names" tagx="given-names" title="given-names">GD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osmundson</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">di Lonardo</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danti</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">An increase in transmission-related traits and in phenotypic plasticity is documented during a fungal invasion</span><span class="source" tagx="source" title="source">Ecosphere</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">art180</span><span class="pub-id"><a href="https://dx.doi.org/10.1890/ES14-00426.1">10.1890/ES14-00426.1</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garnas</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Auger-Rozenberg</span><span class="given-names" tagx="given-names" title="given-names">M-A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roques</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertelsmeier</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wingfield</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saccaggi</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slippers</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Complex patterns of global spread in invasive insects: eco-evolutionary and management consequences</span><span class="source" tagx="source" title="source">Biological Invasions</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">935</span><span class="lpage" tagx="lpage" title="lpage">952</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-016-1082-9">10.1007/s10530-016-1082-9</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gau</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merz</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falloon</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunner</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Global genetics and invasion history of the potato powdery scab pathogen, <i>Spongospora subterranea</i> f.sp. <i>subterranea</i></span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e67944</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0067944">10.1371/journal.pone.0067944</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23840791">23840791</a></span></span></li> <li tag="ref"><a name="bib39" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gladieux</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Byrnes III</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aguileta</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heitman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giraud</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Epidemiology and evolution of fungal pathogens in plants and animals</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tibayrenc</span><span class="given-names" tagx="given-names" title="given-names">M. ichel</span></span></span><span class="source" tagx="source" title="source">Genetics and Evolution of Infectious Disease</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span><span class="fpage" tagx="fpage" title="fpage">59</span><span class="lpage" tagx="lpage" title="lpage">132</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/B978-0-12-799942-5.00004-4">10.1016/B978-0-12-799942-5.00004-4</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gladieux</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feurtey</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hood</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snirc</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clavel</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutech</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giraud</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">The population biology of fungal invasions</span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1969</span><span class="lpage" tagx="lpage" title="lpage">1986</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.13028">10.1111/mec.13028</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25469955">25469955</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GonzÃ¡lez-Varela</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GonzÃ¡lez</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Clonal population structure and introductions of the chestnut blight fungus, <i>Cryphonectria parasitica</i>, in Asturias, northern Spain</span><span class="source" tagx="source" title="source">European Journal of Plant Pathology</span><span class="volume" tagx="volume" title="volume">131</span><span class="fpage" tagx="fpage" title="fpage">67</span><span class="lpage" tagx="lpage" title="lpage">79</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10658-011-9788-0">10.1007/s10658-011-9788-0</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griffin</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span></span><span class="year" tagx="year" title="year">1986</span><span class="mixed-article-title" title="mixed-article-title">Chestnut blight and its control</span><span class="source" tagx="source" title="source">Horticultural Reviews</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">291</span><span class="lpage" tagx="lpage" title="lpage">336</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/9781118060810.ch8">10.1002/9781118060810.ch8</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GrÃ¼nwald</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garbelotto</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goss</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heungens</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Emergence of the sudden oak death pathogen <i>Phytophthora ramorum</i></span><span class="source" tagx="source" title="source">Trends in Microbiology</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">131</span><span class="lpage" tagx="lpage" title="lpage">138</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2011.12.006">10.1016/j.tim.2011.12.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22326131">22326131</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gutenkunst</span><span class="given-names" tagx="given-names" title="given-names">RN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hernandez</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williamson</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bustamante</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data</span><span class="source" tagx="source" title="source">PLOS Genetics</span><span class="volume" tagx="volume" title="volume">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1000695</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1000695">10.1371/journal.pgen.1000695</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19851460">19851460</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayes</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barry</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Are there any consistent predictors of invasion success?</span><span class="source" tagx="source" title="source">Biological Invasions</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">483</span><span class="lpage" tagx="lpage" title="lpage">506</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-007-9146-5">10.1007/s10530-007-9146-5</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heald</span><span class="given-names" tagx="given-names" title="given-names">FD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Studhalter</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="year" tagx="year" title="year">1914</span><span class="mixed-article-title" title="mixed-article-title">Birds as carriers of the chestnut blight fungus</span><span class="source" tagx="source" title="source">Journal of Agricultural Research</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">405</span><span class="lpage" tagx="lpage" title="lpage">422</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/4071651">10.2307/4071651</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heiniger</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigling</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Biological control of chestnut blight in Europe</span><span class="source" tagx="source" title="source">Annual Review of Phytopathology</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">581</span><span class="lpage" tagx="lpage" title="lpage">599</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.py.32.090194.003053">10.1146/annurev.py.32.090194.003053</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heitman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">TY</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Evolution of fungal sexual reproduction</span><span class="source" tagx="source" title="source">Mycologia</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">27</span><span class="pub-id"><a href="https://dx.doi.org/10.3852/12-253">10.3852/12-253</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23099518">23099518</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hessenauer</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fijarczyk</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prunier</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charron</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chapuis</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernier</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanguay</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamelin</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Landry</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span></span><span class="year" tagx="year" title="year">2020</span><span class="mixed-article-title" title="mixed-article-title">Hybridization and introgression drive genome evolution of Dutch elm disease pathogens</span><span class="source" tagx="source" title="source">Nature Ecology &amp;amp; Evolution</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">626</span><span class="lpage" tagx="lpage" title="lpage">638</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41559-020-1133-6">10.1038/s41559-020-1133-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32123324">32123324</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoegger</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigling</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holdenrieder</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heiniger</span><span class="given-names" tagx="given-names" title="given-names">U</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Genetic structure of newly established populations of <i>Cryphonectria parasitica</i></span><span class="source" tagx="source" title="source">Mycological Research</span><span class="volume" tagx="volume" title="volume">104</span><span class="fpage" tagx="fpage" title="fpage">1108</span><span class="lpage" tagx="lpage" title="lpage">1116</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0953756299002397">10.1017/S0953756299002397</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hudson</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Generating samples under a Wright-Fisher neutral model of genetic variation</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">337</span><span class="lpage" tagx="lpage" title="lpage">338</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/18.2.337">10.1093/bioinformatics/18.2.337</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11847089">11847089</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huson</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bryant</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Application of phylogenetic networks in evolutionary studies</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">254</span><span class="lpage" tagx="lpage" title="lpage">267</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msj030">10.1093/molbev/msj030</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16221896">16221896</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Idnurm</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hood</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johannesson</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giraud</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Contrasted patterns in mating-type chromosomes in fungi: hotspots versus coldspots of recombination</span><span class="source" tagx="source" title="source">Fungal Biology Reviews</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">220</span><span class="lpage" tagx="lpage" title="lpage">229</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fbr.2015.06.001">10.1016/j.fbr.2015.06.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26688691">26688691</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JeÅ¾iÄ‡</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krstin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigling</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">CurkoviÄ‡-Perica</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">High diversity in populations of the introduced plant pathogen, <i>Cryphonectria parasitica</i>, due to encounters between genetically divergent genotypes</span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">87</span><span class="lpage" tagx="lpage" title="lpage">99</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2011.05369.x">10.1111/j.1365-294X.2011.05369.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22092517">22092517</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Binns</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McAnulla</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McWilliam</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maslen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nuka</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pesseat</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quinn</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sangrador-Vegas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scheremetjew</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yong</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunter</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">InterProScan 5: genome-scale protein function classification</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">1236</span><span class="lpage" tagx="lpage" title="lpage">1240</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu031">10.1093/bioinformatics/btu031</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451626">24451626</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KaradÅ¾iÄ‡</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RaduloviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sikora</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">StanivukoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GoluboviÄ‡ Ä†urguz</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oszako</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MilenkoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Characterisation and pathogenicity of <i>Cryphonectria parasitica</i> on sweet chestnut and sessile oak trees in Serbia</span><span class="source" tagx="source" title="source">Plant Protection Science</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">191</span><span class="lpage" tagx="lpage" title="lpage">201</span><span class="pub-id"><a href="https://dx.doi.org/10.17221/38/2018-PPS">10.17221/38/2018-PPS</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knaus</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tabima</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shakya</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Judelson</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GrÃ¼nwald</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span></span><span class="year" tagx="year" title="year">2020</span><span class="mixed-article-title" title="mixed-article-title">Genome-Wide increased copy number is associated with emergence of dominant clones of the Irish potato famine pathogen <i>Phytophthora infestans</i></span><span class="source" tagx="source" title="source">mBio</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e00326-20</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.00326-20">10.1128/mBio.00326-20</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32576669">32576669</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knaus</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GrÃ¼nwald</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Vcfr: a package to manipulate and visualize variant call format data in R</span><span class="source" tagx="source" title="source">Molecular Ecology Resources</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">44</span><span class="lpage" tagx="lpage" title="lpage">53</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1755-0998.12549">10.1111/1755-0998.12549</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27401132">27401132</a></span></span></li> <li tag="ref"><a name="bib59" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krasnov</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shenbrot</span><span class="given-names" tagx="given-names" title="given-names">GI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khokhlova</span><span class="given-names" tagx="given-names" title="given-names">IS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanko</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morand</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mouillot</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Assembly rules of ectoparasite communities across scales: combining patterns of abiotic factors, host composition, geographic space, phylogeny and traits</span><span class="source" tagx="source" title="source">Ecography</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">184</span><span class="lpage" tagx="lpage" title="lpage">197</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/ecog.00915">10.1111/ecog.00915</a></span></span></li> <li tag="ref"><a name="bib60" /><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: Laboratory of Evolutionary Genetics @ UNINE</span></span><span class="year" tagx="year" title="year">2021</span><div tag="data-title">
Datasets</div><span class="source" tagx="source" title="source">Software Heritage</span><span class="version" tagx="version" title="version">swh:1:rev:fa908efdb71d5ba90823c70bf2467e10ce6f12a7</span><a href="https://archive.softwareheritage.org/swh:1:dir:e3c098b0dcd2bba3e201d3d13293e544c2d58232;origin=https://github.com/crolllab/datasets;visit=swh:1:snp:7e5189913de8400eed7cc2e7b8041677ecfefb2a;anchor=swh:1:rev:fa908efdb71d5ba90823c70bf2467e10ce6f12a7/">https://archive.softwareheritage.org/swh:1:dir:e3c098b0dcd2bba3e201d3d13293e544c2d58232;origin=https://github.com/crolllab/datasets;visit=swh:1:snp:7e5189913de8400eed7cc2e7b8041677ecfefb2a;anchor=swh:1:rev:fa908efdb71d5ba90823c70bf2467e10ce6f12a7/</a></span></li> <li tag="ref"><a name="bib61" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langmead</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with bowtie 2</span><span class="source" tagx="source" title="source">Nature Methods</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">357</span><span class="lpage" tagx="lpage" title="lpage">359</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a></span></span></li> <li tag="ref"><a name="bib62" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawson</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hellenthal</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falush</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Inference of population structure using dense haplotype data</span><span class="source" tagx="source" title="source">PLOS Genetics</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1002453</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1002453">10.1371/journal.pgen.1002453</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22291602">22291602</a></span></span></li> <li tag="ref"><a name="bib63" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Handsaker</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wysoker</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fennell</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Homer</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marth</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abecasis</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durbin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="collab'">collab: 1000 Genome Project Data Processing Subgroup</span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">The sequence alignment/Map format and SAMtools</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">2078</span><span class="lpage" tagx="lpage" title="lpage">2079</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a></span></span></li> <li tag="ref"><a name="bib64" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durbin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1754</span><span class="lpage" tagx="lpage" title="lpage">1760</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp324">10.1093/bioinformatics/btp324</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168">19451168</a></span></span></li> <li tag="ref"><a name="bib65" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lischer</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Excoffier</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">PGDSpider: an automated data conversion tool for connecting population genetics and genomics programs</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">298</span><span class="lpage" tagx="lpage" title="lpage">299</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr642">10.1093/bioinformatics/btr642</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22110245">22110245</a></span></span></li> <li tag="ref"><a name="bib66" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lombaert</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guillemaud</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cornuet</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malausa</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Facon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Estoup</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Bridgehead effect in the worldwide invasion of the biocontrol harlequin ladybird</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e9743</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0009743">10.1371/journal.pone.0009743</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20305822">20305822</a></span></span></li> <li tag="ref"><a name="bib67" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marbuah</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gren</span><span class="given-names" tagx="given-names" title="given-names">I-M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKie</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Economics of harmful invasive species: a review</span><span class="source" tagx="source" title="source">Diversity</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">500</span><span class="lpage" tagx="lpage" title="lpage">523</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/d6030500">10.3390/d6030500</a></span></span></li> <li tag="ref"><a name="bib68" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mariette</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mabon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">CorbiÃ¨re</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boulard</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glais</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marquer</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pasco</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montarry</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andrivon</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Phenotypic and genotypic changes in French populations of <i>Phytophthora infestans</i> : are invasive clones the most aggressive?</span><span class="source" tagx="source" title="source">Plant Pathology</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">577</span><span class="lpage" tagx="lpage" title="lpage">586</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/ppa.12441">10.1111/ppa.12441</a></span></span></li> <li tag="ref"><a name="bib69" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marra</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cortesi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bissegger</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Mixed mating in natural populations of the chestnut blight fungus, <i>Cryphonectria parasitica</i></span><span class="source" tagx="source" title="source">Heredity</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">189</span><span class="lpage" tagx="lpage" title="lpage">195</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/sj.hdy.6800492">10.1038/sj.hdy.6800492</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15241462">15241462</a></span></span></li> <li tag="ref"><a name="bib70" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marra</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">The mating system of the fungus <i>Cryphonectria parasitica</i>: selfing and self-incompatibility</span><span class="source" tagx="source" title="source">Heredity</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">134</span><span class="lpage" tagx="lpage" title="lpage">143</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-2540.2001.00784.x">10.1046/j.1365-2540.2001.00784.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11380658">11380658</a></span></span></li> <li tag="ref"><a name="bib71" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKenna</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanna</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banks</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sivachenko</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cibulskis</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kernytsky</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garimella</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altshuler</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gabriel</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daly</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DePristo</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span><span class="source" tagx="source" title="source">Genome Research</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">1297</span><span class="lpage" tagx="lpage" title="lpage">1303</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.107524.110">10.1101/gr.107524.110</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199">20644199</a></span></span></li> <li tag="ref"><a name="bib72" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McVean</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Auton</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2007</span><div tag="data-title">
LDhat 2.1: A Package for the Population Genetic Analysis of Recombination</div><span class="version" tagx="version" title="version">2.1</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Dep Stat</span><a href="http://www.stats.ox.ac.uk/~mcvean/LDhat/manual.pdf">http://www.stats.ox.ac.uk/~mcvean/LDhat/manual.pdf</a></span></li> <li tag="ref"><a name="bib73" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mehrotra</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="source" tagx="source" title="source">Fundamentals of Plant Pathology</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Tata McGraw-Hill Education</span></span></li> <li tag="ref"><a name="bib74" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipari</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaneko</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Intercontinental population structure of the chestnut blight fungus, <i>Cryphonectria parasitica</i></span><span class="source" tagx="source" title="source">Mycologia</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">179</span><span class="lpage" tagx="lpage" title="lpage">190</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/00275514.1996.12026642">10.1080/00275514.1996.12026642</a></span></span></li> <li tag="ref"><a name="bib75" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sotirovski</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spica</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brewer</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milev</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cortesi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Clonal population structure of the chestnut blight fungus in expanding ranges in southeastern Europe</span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">4446</span><span class="lpage" tagx="lpage" title="lpage">4458</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2008.03927.x">10.1111/j.1365-294X.2008.03927.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18803594">18803594</a></span></span></li> <li tag="ref"><a name="bib76" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cortesi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Analysis of population structure of the chestnut blight fungus based on vegetative incompatibility genotypes</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">96</span><span class="fpage" tagx="fpage" title="fpage">10518</span><span class="lpage" tagx="lpage" title="lpage">10523</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.96.18.10518">10.1073/pnas.96.18.10518</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10468641">10468641</a></span></span></li> <li tag="ref"><a name="bib77" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myteberi</span><span class="given-names" tagx="given-names" title="given-names">IF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lushaj Arvjen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KeÄ�a</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lushaj Arnisa</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lushaj</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Diversity of <i>Cryphonectria parasitica</i>, hypovirulence, and possibilities for biocontrol of chestnut canker in Albania</span><span class="source" tagx="source" title="source">International Journal of Microbiology Research and Reviews</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">21</span></span></li> <li tag="ref"><a name="bib78" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narasimhan</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danecek</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scally</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tyler-Smith</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durbin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">BCFtools/RoH: a hidden Markov model approach for detecting autozygosity from next-generation sequencing data</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">1749</span><span class="lpage" tagx="lpage" title="lpage">1751</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btw044">10.1093/bioinformatics/btw044</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26826718">26826718</a></span></span></li> <li tag="ref"><a name="bib79" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oggenfuss</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Badet</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wicker</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartmann</span><span class="given-names" tagx="given-names" title="given-names">FE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">NK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abraham</span><span class="given-names" tagx="given-names" title="given-names">LN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karisto</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vonlanthen</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mundt</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McDonald</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span></span><span class="year" tagx="year" title="year">2020</span><span class="mixed-article-title" title="mixed-article-title">A population-level invasion by transposable elements in a fungal pathogen</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.02.11.944652">10.1101/2020.02.11.944652</a></span></span></li> <li tag="ref"><a name="bib80" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paoletti</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buck</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brasier</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Selective acquisition of novel mating type and vegetative incompatibility genes via interspecies gene transfer in the globally invading eukaryote <i>Ophiostoma novo-ulmi</i></span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">249</span><span class="lpage" tagx="lpage" title="lpage">262</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2005.02728.x">10.1111/j.1365-294X.2005.02728.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16367844">16367844</a></span></span></li> <li tag="ref"><a name="bib81" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Philibert</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desprez-Loustau</span><span class="given-names" tagx="given-names" title="given-names">M-L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fabre</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halkett</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Husson</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lung-Escarmant</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MarÃ§ais</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vacher</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makowski</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Predicting invasion success of forest pathogenic fungi from species traits</span><span class="source" tagx="source" title="source">Journal of Applied Ecology</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">1381</span><span class="lpage" tagx="lpage" title="lpage">1390</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2664.2011.02039.x">10.1111/j.1365-2664.2011.02039.x</a></span></span></li> <li tag="ref"><a name="bib82" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conedera</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heiniger</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigling</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Saprophytic activity and sporulation of <i>Cryphonectria parasitica</i> on dead chestnut wood in forests with naturally established hypovirulence</span><span class="source" tagx="source" title="source">Phytopathology</span><span class="volume" tagx="volume" title="volume">96</span><span class="fpage" tagx="fpage" title="fpage">1337</span><span class="lpage" tagx="lpage" title="lpage">1344</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/PHYTO-96-1337">10.1094/PHYTO-96-1337</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18943666">18943666</a></span></span></li> <li tag="ref"><a name="bib83" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleary</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Effects of host variability on the spread of invasive forest diseases</span><span class="source" tagx="source" title="source">Forests</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">80</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/f8030080">10.3390/f8030080</a></span></span></li> <li tag="ref"><a name="bib84" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigling</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Invasion genetics of the chestnut blight fungus <i>Cryphonectria parasitica</i> in Switzerland</span><span class="source" tagx="source" title="source">Phytopathology</span><span class="volume" tagx="volume" title="volume">102</span><span class="fpage" tagx="fpage" title="fpage">73</span><span class="lpage" tagx="lpage" title="lpage">82</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/PHYTO-02-11-0055">10.1094/PHYTO-02-11-0055</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21848397">21848397</a></span></span></li> <li tag="ref"><a name="bib85" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quinlan</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">BEDTools: a flexible suite of utilities for comparing genomic features</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">841</span><span class="lpage" tagx="lpage" title="lpage">842</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btq033">10.1093/bioinformatics/btq033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278">20110278</a></span></span></li> <li tag="ref"><a name="bib86" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raboin</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selvi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliveira</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paulet</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calatayud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zapater</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brottier</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luzaran</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garsmeur</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlier</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">D'Hont</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Evidence for the dispersal of a unique lineage from Asia to America and Africa in the sugarcane fungal pathogen Ustilago scitaminea</span><span class="source" tagx="source" title="source">Fungal Genetics and Biology</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">64</span><span class="lpage" tagx="lpage" title="lpage">76</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fgb.2006.07.004">10.1016/j.fgb.2006.07.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16979360">16979360</a></span></span></li> <li tag="ref"><a name="bib87" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigling</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title"><i>Cryphonectria parasitica</i> , the causal agent of chestnut blight: invasion history, population biology and disease control</span><span class="source" tagx="source" title="source">Molecular Plant Pathology</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="lpage" tagx="lpage" title="lpage">20</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mpp.12542">10.1111/mpp.12542</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28142223">28142223</a></span></span></li> <li tag="ref"><a name="bib88" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heiniger</span><span class="given-names" tagx="given-names" title="given-names">U</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Chestnut blight in Europe: diversity of <i>Cryphonectria parasitica</i>, hypovirulence and biocontrol</span><span class="source" tagx="source" title="source">Forest Snow and Landscape Research</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">361</span><span class="lpage" tagx="lpage" title="lpage">367</span></span></li> <li tag="ref"><a name="bib89" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossman</span><span class="given-names" tagx="given-names" title="given-names">AMYY</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">A special issue on global movement of invasive plants and fungi</span><span class="source" tagx="source" title="source">BioScience</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">93</span><span class="lpage" tagx="lpage" title="lpage">94</span><span class="pub-id"><a href="https://dx.doi.org/10.1641/0006-3568(2001)051[0093:ASIOGM]2.0.CO;2">10.1641/0006-3568(2001)051[0093:ASIOGM]2.0.CO;2</a></span></span></li> <li tag="ref"><a name="bib90" /><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: RStudio Team</span></span><span class="year" tagx="year" title="year">2015</span><div tag="data-title">
RStudio: Integrated Development Environment for R</div><span class="source" tagx="source" title="source">RStudio</span><span class="version" tagx="version" title="version">3.2.1</span><a href="http://www.rstudio.org/">http://www.rstudio.org/</a></span></li> <li tag="ref"><a name="bib91" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santini</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghelardini</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Pace</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desprez-Loustau</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Capretti</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandelier</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cech</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chira</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diamandis</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaitniekis</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hantula</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holdenrieder</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jankovsky</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jurc</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kirisits</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kunca</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lygis</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malecka</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marcais</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmitz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schumacher</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solheim</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SzabÃ²</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsopelas</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vannini</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vettraino</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webber</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodward</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stenlid</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Biogeographical patterns and determinants of invasion by forest pathogens in Europe</span><span class="source" tagx="source" title="source">New Phytologist</span><span class="volume" tagx="volume" title="volume">197</span><span class="fpage" tagx="fpage" title="fpage">238</span><span class="lpage" tagx="lpage" title="lpage">250</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1469-8137.2012.04364.x">10.1111/j.1469-8137.2012.04364.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23057437">23057437</a></span></span></li> <li tag="ref"><a name="bib92" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saunders</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Win</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamoun</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raffaele</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Two-dimensional data binning for the analysis of genome architecture in filamentous plant pathogens and other eukaryotes</span><span class="source" tagx="source" title="source">Methods in Molecular Biology</span><span class="volume" tagx="volume" title="volume">1127</span><span class="fpage" tagx="fpage" title="fpage">29</span><span class="lpage" tagx="lpage" title="lpage">51</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-62703-986-4_3">10.1007/978-1-62703-986-4_3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24643550">24643550</a></span></span></li> <li tag="ref"><a name="bib93" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoustra</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rundle</span><span class="given-names" tagx="given-names" title="given-names">HD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dali</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kassen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Fitness-associated sexual reproduction in a filamentous fungus</span><span class="source" tagx="source" title="source">Current Biology</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">1350</span><span class="lpage" tagx="lpage" title="lpage">1355</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2010.05.060">10.1016/j.cub.2010.05.060</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20598542">20598542</a></span></span></li> <li tag="ref"><a name="bib94" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smit</span><span class="given-names" tagx="given-names" title="given-names">AFA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubley</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2015</span><div tag="data-title">
RepeatMasker Open</div><span class="source" tagx="source" title="source">RepeatMasker</span><span class="version" tagx="version" title="version">4.0</span><a href="http://www.repeatmasker.org/cgi-bin/WEBRepeatMasker">http://www.repeatmasker.org/cgi-bin/WEBRepeatMasker</a></span></li> <li tag="ref"><a name="bib95" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">NH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Rourke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spratt</span><span class="given-names" tagx="given-names" title="given-names">BG</span></span></span><span class="year" tagx="year" title="year">1993</span><span class="mixed-article-title" title="mixed-article-title">How clonal are bacteria?</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">4384</span><span class="lpage" tagx="lpage" title="lpage">4388</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.90.10.4384">10.1073/pnas.90.10.4384</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8506277">8506277</a></span></span></li> <li tag="ref"><a name="bib96" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sotirovski</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papazova-Anakieva</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grunwald</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milgroom</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Low diversity of vegetative compatibility types and mating type of <i>Cryphonectria parasitica</i> in the southern Balkans</span><span class="source" tagx="source" title="source">Plant Pathology</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">325</span><span class="lpage" tagx="lpage" title="lpage">333</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.0032-0862.2004.01006.x">10.1111/j.0032-0862.2004.01006.x</a></span></span></li> <li tag="ref"><a name="bib97" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sperschneider</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dodds</span><span class="given-names" tagx="given-names" title="given-names">PN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardiner</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Improved prediction of fungal effector proteins from secretomes with EffectorP 2.0</span><span class="source" tagx="source" title="source">Molecular Plant Pathology</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">2094</span><span class="lpage" tagx="lpage" title="lpage">2110</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mpp.12682">10.1111/mpp.12682</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29569316">29569316</a></span></span></li> <li tag="ref"><a name="bib98" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stauber</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prospero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croll</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2020</span><span class="mixed-article-title" title="mixed-article-title">Comparative genomics analyses of lifestyle transitions at the origin of an invasive fungal pathogen in the genus <i>Cryphonectria</i></span><span class="source" tagx="source" title="source">mSphere</span><span class="volume" tagx="volume" title="volume">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e00737-20</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/mSphere.00737-20">10.1128/mSphere.00737-20</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/33055257">33055257</a></span></span></li> <li tag="ref"><a name="bib99" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steenwyk</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soghigian</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perfect</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbons</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Copy number variation contributes to cryptic genetic variation in outbreak lineages of <i>Cryptococcus gattii</i> from the North American Pacific Northwest</span><span class="source" tagx="source" title="source">BMC Genomics</span><span class="volume" tagx="volume" title="volume">17</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">700</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-016-3044-0">10.1186/s12864-016-3044-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27590805">27590805</a></span></span></li> <li tag="ref"><a name="bib100" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suehs</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Affre</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MÃ©dail</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Invasion dynamics of two alien Carpobrotus (Aizoaceae) taxa on a Mediterranean Island: ii. reproductive strategies</span><span class="source" tagx="source" title="source">Heredity</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">550</span><span class="lpage" tagx="lpage" title="lpage">556</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/sj.hdy.6800454">10.1038/sj.hdy.6800454</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15100707">15100707</a></span></span></li> <li tag="ref"><a name="bib101" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hann-Soden</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Branco</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sylvain</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ellison</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Clonal reproduction in fungi</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">112</span><span class="fpage" tagx="fpage" title="fpage">8901</span><span class="lpage" tagx="lpage" title="lpage">8908</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1503159112">10.1073/pnas.1503159112</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26195774">26195774</a></span></span></li> <li tag="ref"><a name="bib102" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torres</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oggenfuss</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croll</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seidl</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span></span><span class="year" tagx="year" title="year">2020</span><span class="mixed-article-title" title="mixed-article-title">Genome evolution in fungal plant pathogens: looking beyond the two-speed genome model</span><span class="source" tagx="source" title="source">Fungal Biology Reviews</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">136</span><span class="lpage" tagx="lpage" title="lpage">143</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fbr.2020.07.001">10.1016/j.fbr.2020.07.001</a></span></span></li> <li tag="ref"><a name="bib103" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trestic</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uscuplic</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colinas</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rolland</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giraud</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Vegetative compatibility type diversity of <i>Cryphonectria parasitica</i> populations in Bosnia-Herzegovina, Spain and France</span><span class="source" tagx="source" title="source">Forest Snow and Landscape Research</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">391</span><span class="lpage" tagx="lpage" title="lpage">396</span></span></li> <li tag="ref"><a name="bib104" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Boheemen</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lombaert</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nurkowski</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gauffre</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rieseberg</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodgins</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Multiple introductions, admixture and bridgehead invasion characterize the introduction history of <i>Ambrosia artemisiifolia</i> in Europe and Australia</span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">5421</span><span class="lpage" tagx="lpage" title="lpage">5434</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.14293">10.1111/mec.14293</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28802079">28802079</a></span></span></li> <li tag="ref"><a name="bib105" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VukoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liber</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JeÅ¾iÄ‡</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sotirovski</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ä†urkoviÄ‡-Perica</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Link between epigenetic diversity and invasive status of south-eastern European populations of phytopathogenic fungus <i>Cryphonectria parasitica</i></span><span class="source" tagx="source" title="source">Environmental Microbiology</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">4521</span><span class="lpage" tagx="lpage" title="lpage">4536</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1462-2920.14742">10.1111/1462-2920.14742</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31314941">31314941</a></span></span></li> <li tag="ref"><a name="bib106" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">CNVcaller: highly efficient and widely applicable software for detecting copy number variations in large populations</span><span class="source" tagx="source" title="source">GigaScience</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">gix115</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/gigascience/gix115">10.1093/gigascience/gix115</a></span></span></li> <li tag="ref"><a name="bib107" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wickham</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Reshaping data with the reshape package</span><span class="source" tagx="source" title="source">Journal of Statistical Software</span><span class="volume" tagx="volume" title="volume">21</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">i12</span><span class="pub-id"><a href="https://dx.doi.org/10.18637/jss.v021.i12">10.18637/jss.v021.i12</a></span></span></li> <li tag="ref"><a name="bib108" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wickham</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="source" tagx="source" title="source">Ggplot2: Elegant Graphics for Data Analysis</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer-Verlag</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-0-387-98141-3">10.1007/978-0-387-98141-3</a></span></span></li> <li tag="ref"><a name="bib109" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wickham</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FranÃ§ois</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henry</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MÃ¼ller</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2018</span><div tag="data-title">
dplyr: A Grammar of Data Manipulation</div><span class="source" tagx="source" title="source">Github</span><span class="version" tagx="version" title="version">1.0.5</span><a href="https://github.com/tidyverse/dplyr">https://github.com/tidyverse/dplyr</a></span></li> <li tag="ref"><a name="bib110" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wickham</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henry</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2018</span><div tag="data-title">
tidyr: Easily Tidy Data with â€œspread()â€� and â€œgather()â€� Functions</div><span class="source" tagx="source" title="source">Tidyr</span><span class="version" tagx="version" title="version">1.1.2</span><a href="https://tidyr.tidyverse.org/">https://tidyr.tidyverse.org/</a></span></li> <li tag="ref"><a name="bib111" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wingfield</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slippers</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roux</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wingfield</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Fifty years of tree pest and pathogen invasions, increasingly threatening world forests</span><span class="conf-name" tagx="conf-name" title="conf-name">Fifty Years Invasion Ecol Leg Charles Elt</span><span class="fpage" tagx="fpage" title="fpage">89</span><span class="lpage" tagx="lpage" title="lpage">99</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/9781444329988.ch8">10.1002/9781444329988.ch8</a></span></span></li> <li tag="ref"><a name="bib112" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wuest</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrington</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraedrich</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yun</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Genetic variation in native populations of the laurel wilt pathogen, <i>Raffaelea lauricola</i>, in Taiwan and Japan and the introduced population in the United States</span><span class="source" tagx="source" title="source">Plant Disease</span><span class="volume" tagx="volume" title="volume">101</span><span class="fpage" tagx="fpage" title="fpage">619</span><span class="lpage" tagx="lpage" title="lpage">628</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/PDIS-10-16-1517-RE">10.1094/PDIS-10-16-1517-RE</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30677356">30677356</a></span></span></li> <li tag="ref"><a name="bib113" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">XM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">XF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Invasion genetics of the western flower Thrips in China: evidence for genetic bottleneck, hybridization and bridgehead effect</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e34567</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0034567">10.1371/journal.pone.0034567</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22509325">22509325</a></span></span></li> </ul> </div> </div>  </body></html>