<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Int J Mol Sci</span><span class="iso-abbrev" title="iso-abbrev">Int J Mol Sci</span><span class="publisher-id" title="publisher-id">ijms</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">International Journal of Molecular Sciences</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1422-0067</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7830504">7830504</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/33467107">33467107</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/ijms22020864">10.3390/ijms22020864</a></span><span class="publisher-id" title="publisher-id">ijms-22-00864</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Mass Spectral Filtering by Mass-Remainder Analysis (MARA) at High Resolution and Its Application to Metabolite Profiling of Flavonoids</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-8568-914X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">Tibor</span></span><a href="#af1-ijms-22-00864">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RÃ³th</span><span class="given-names" tagx="given-names" title="given-names">GergÅ‘</span></span><a href="#af1-ijms-22-00864">1</a><a href="#af2-ijms-22-00864">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuki</span><span class="given-names" tagx="given-names" title="given-names">Ã�kos</span></span><a href="#af1-ijms-22-00864">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zsuga</span><span class="given-names" tagx="given-names" title="given-names">MiklÃ³s</span></span><a href="#af1-ijms-22-00864">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-5274-6117</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ©ki</span><span class="given-names" tagx="given-names" title="given-names">SÃ¡ndor</span></span><a href="#af1-ijms-22-00864">1</a><a href="#c1-ijms-22-00864">*</a></span></div><span class="citation_author_institution" id="af1-ijms-22-00864">[1], <span class="email" tagx="email" title="email">nagy.tibor@science.unideb.hu</span><span class="email" tagx="email" title="email">roth.gergo@science.unideb.hu</span><span class="email" tagx="email" title="email">kuki.akos@science.unideb.hu</span><span class="email" tagx="email" title="email">zsuga.miklos@science.unideb.hu</span></span><span class="citation_author_institution" id="af2-ijms-22-00864">[2], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">keki.sandor@science.unideb.hu</span>; Fax: +36-52-518662</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021-1</span></span><span class="volume" tagx="volume" title="volume">22</span><span class="issue" tagx="issue" title="issue">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">864</span><span class="history" title="history"><span class="received" title="received">received: 2020-12-27</span><span class="accepted" title="accepted">accepted: 2021-1-14</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2021</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Flavonoids represent an important class of secondary metabolites because of their potential health benefits and functions in plants. We propose a novel method for the comprehensive flavonoid filtering and screening based on direct infusion mass spectrometry (DIMS) analysis. The recently invented data mining procedure, the multi-step mass-remainder analysis (M-MARA) technique is applied for the effective mass spectral filtering of the peak rich spectra of natural herb extracts. In addition, our flavonoid-filtering algorithm facilitates the determination of the elemental composition. M-MARA flavonoid-filtering uses simple mathematical and logical operations and thus, it can easily be implemented in a regular spreadsheet software. A huge benefit of our method is the high speed and the low demand for computing power and memory that enables the real time application even for tandem mass spectrometric analysis. Our novel method was applied for the electrospray ionization (ESI) DIMS spectra of various herb extract, and the filtered mass spectral data were subjected to chemometrics analysis using principal component analysis (PCA).</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
metabolomics</div> <div class="kwd" title="kwd">
flavonoid</div> <div class="kwd" title="kwd">
direct infusion mass spectrometry</div> <div class="kwd" title="kwd">
mass spectral filtering</div> <div class="kwd" title="kwd">
mass-remainder analysis</div> <div class="kwd" title="kwd">
principal component analysis</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Metabolomics is a comprehensive analysis that reveals the metabolome, i.e., all the metabolites of a biological system under study [<a href="#B1-ijms-22-00864">1</a>]. Metabolomics plays an increasingly important role in plant science [<a href="#B2-ijms-22-00864">2</a>]. In metabolic analysis different analytical approaches have been designed on several levels, such as â€œmetabolite fingerprintingâ€�, â€œmetabolite profilingâ€�, and â€œmetabolite target analysisâ€� [<a href="#B1-ijms-22-00864">1</a>,<a href="#B3-ijms-22-00864">3</a>]. Metabolic profiling focuses on the analysis of a group of metabolites or a class of compounds. An important class of plant metabolites is the flavonoids because of their potential health benefits and functions in plants [<a href="#B4-ijms-22-00864">4</a>]. Nearly 20,000 different flavonoids have now been characterized [<a href="#B5-ijms-22-00864">5</a>], with many of them showing interesting antioxidant, anti-inflammatory, anti-mutagenic and anti-carcinogenic properties [<a href="#B6-ijms-22-00864">6</a>,<a href="#B7-ijms-22-00864">7</a>]. They are active contributors to the health benefit of foods and beverages of plant origin, such as fruits, vegetables, tea, cocoa, and wine. Flavonoids have a C<sub>6</sub>-C<sub>3</sub>-C<sub>6</sub> (A, C, B ring) core structure and depending on the position of the linkage between the C and B rings they can be divided into classes, while based on the degree of oxidation and saturation in the heterocyclic C-ring the flavonoids may be divided into several groups. For example, flavones, flavonols, flavanones, flavanonols, flavanols. In addition, the huge diversity of flavonoids arises from the number and complexity of substituents and presence of additional heterocyclic rings [<a href="#B6-ijms-22-00864">6</a>,<a href="#B7-ijms-22-00864">7</a>]. The structures of the main flavonoid groups are shown in <a href="#ijms-22-00864-sch001">Scheme 1</a>.</p> <p>The identification and quantification of flavonoids requires sophisticated, advanced analytical techniques. Mass spectrometry (MS) and nuclear magnetic resonance (NMR) spectroscopy, especially when they are coupled to chromatographic techniques, particularly liquid chromatography (LC), are the two leading analytical methods used for metabolic profiling of natural extracts. Significant advances in hyphenated techniques (LCâ€"MS, LCâ€"NMR), as well as in combined MS/NMR methods, have occurred over the last decades [<a href="#B3-ijms-22-00864">3</a>,<a href="#B8-ijms-22-00864">8</a>,<a href="#B9-ijms-22-00864">9</a>,<a href="#B10-ijms-22-00864">10</a>,<a href="#B11-ijms-22-00864">11</a>,<a href="#B12-ijms-22-00864">12</a>]. On the other hand, the mass accuracy and resolving power of high resolution mass spectrometry (HRMS) has dramatically improved the detection and identification of compounds in complex natural samples and, therefore, has increased the relevance of non-hyphenated MS methods. Direct infusion mass spectrometry (DIMS) can significantly increase the analytical throughput compared to LCâ€"MS [<a href="#B3-ijms-22-00864">3</a>,<a href="#B13-ijms-22-00864">13</a>]. However, the rapid identification of hundreds or thousands of mass peaks arising in direct injection of complex natural samples requires advanced data mining techniques. Mass defect filtering (MDF) of complex MS data has been used for selectively detecting compound classes having similar mass defects (where mass defect is the difference between a compoundâ€™s exact mass and its nominal mass). The MDF process can facilitate the detection of the target compounds by the selective removal of all ions that fall outside of the preset filter mass range window [<a href="#B14-ijms-22-00864">14</a>,<a href="#B15-ijms-22-00864">15</a>,<a href="#B16-ijms-22-00864">16</a>,<a href="#B17-ijms-22-00864">17</a>]. However, if the aim is the comprehensive analysis of the flavonoids in complex natural samples, a wide mass defect window should be applied that would likely include many false peaks. The flavonoids fall into the mass defect segment where numerous possible chemical compositions may be valid [<a href="#B18-ijms-22-00864">18</a>].</p> <p>Besides the MDF process, the Kendrick Mass Defect (KMD) [<a href="#B19-ijms-22-00864">19</a>] and related methods [<a href="#B20-ijms-22-00864">20</a>,<a href="#B21-ijms-22-00864">21</a>,<a href="#B22-ijms-22-00864">22</a>] (Re-KMD) are applied for the analysis of complex mass spectra, however, it is not widespread in metabolomics.</p> <p>Recently, we have developed a data mining algorithm, called mass-remainder analysis (MARA) and its improved variant, multi-step mass-remainder analysis (M-MARA) for the processing and assessment of complex mass spectra [<a href="#B23-ijms-22-00864">23</a>,<a href="#B24-ijms-22-00864">24</a>,<a href="#B25-ijms-22-00864">25</a>,<a href="#B26-ijms-22-00864">26</a>,<a href="#B27-ijms-22-00864">27</a>]. In this paper, we propose a novel method for the comprehensive filtering and screening of flavonoids in the DIMS spectra of natural extracts.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
2. Results and Discussion</div> <p>As it was detailed in our previous works [<a href="#B23-ijms-22-00864">23</a>,<a href="#B24-ijms-22-00864">24</a>,<a href="#B25-ijms-22-00864">25</a>,<a href="#B26-ijms-22-00864">26</a>], mass-remainder analysis (MARA) and its multistep extension (M-MARA) are able to produce simplified graphical representations of the mass spectra, facilitating thereby the recognition, filtering, and grouping of the detected compounds. M-MARA simply performs sequential modulo operations on the measured <i>m/z</i> values in order to eliminate the homologous connections present in a complex system. Equation (1) reveals the general formula for the three-step M-MARA [<a href="#B26-ijms-22-00864">26</a>]: </p><div class="disp-formula" title="disp-formula">
MR<sub>3</sub> = [(<i>m/z</i> MOD R<sub>1</sub>) MOD R<sub>2</sub>] MOD R<sub>3</sub><span class="label" tagx="label" title="label">(1)</span></div> where the modulo (MOD) operation finds the mass remainder (MR<sub>n</sub>) after the division by R<sub>n</sub> (<i>n</i> = 1, 2, 3). It can easily be seen, that MARA is highly effective for data mining of mass spectra containing numerous peaks with periodicities generated by e.g., a repeat unit (polymers) or the CH<sub>2</sub> group (hydrocarbons). Therefore, at a first glance, the application of MARA might seem to be challenging and questionable for processing the mass spectra of complex natural extracts composed of very diverse flavonoid constituents. However, by careful selection of the M-MARA base units (R<sub>n</sub> in Equation (1)), most of the differences in the elemental composition of the flavonoids can be eliminated, and a single characteristic difference can be obtained and used further for grouping and effective filtering of the constituents. An interesting and promising approach is to sort the flavonoids according to their double bond equivalent (DBE) values. The main steps of our analysis are summarized and an example is detailed later. In the first step, the exact mass of the oxygen atom was chosen as the divisor of the modulo operation (R<sub>1</sub> = 15.994915). As a simplified example, <a href="#ijms-22-00864-f001">Figure 1</a>a shows the MR<sub>1</sub> versus <i>m/z</i> plot of eight flavonoids detected in the extract of the medicinal plant yarrow (<i>Achillea millefolium</i> L.).<p /> <p>As it can be seen in <a href="#ijms-22-00864-f001">Figure 1</a>a, compounds differing only by the number of oxygen atoms have the same MR<sub>1</sub> values and are aligned in a line. Moreover, <a href="#ijms-22-00864-f001">Figure 1</a>a also reveals another homologous connection that can be eliminated in the second MARA step. A characteristic difference of MR<sub>1</sub> = 1.97927 appears between the neighboring lines (dots) that corresponds to the mass difference between oxygen and CH<sub>2</sub> group. Using R<sub>2</sub> = 1.97927 in the next step, MARA assigns the same MR<sub>2</sub> value to most of the compounds differing only by the number of oxygen atoms and CH<sub>2</sub> groups (see <a href="#ijms-22-00864-f001">Figure 1</a>b). Using the MR<sub>2</sub> difference in <a href="#ijms-22-00864-f001">Figure 1</a>b as the base unit of the third MARA step (R<sub>3</sub> = 0.160795, i.e., corresponding to the mass of 8CH<sub>2</sub>-7O), the <i>m/z</i> range of the filtering can be extended, and thereby the MR<sub>3</sub> versus <i>m/z</i> plot (<a href="#ijms-22-00864-f001">Figure 1</a>c) shows merely a single line. <a href="#ijms-22-00864-f001">Figure 1</a>d shows the MR<sub>3</sub> versus <i>m/z</i> plot including the theoretical <i>m/z</i> values of more than one hundred flavonoids. Accordingly, three groups were distinguished based on the DBE values, flavans plus flavanols, flavanones plus flavanonols, and flavones plus flavonols, having the DBE values 9, 10, and 11, respectively. However, if the goal is a comprehensive mass spectral filtering of the flavonoid compounds, a wider DBE range should be considered. Due to the large number and complexity of substituents and presence of additional heterocyclic rings, the flavonoid species cover a DBE range of 9â€"30 [<a href="#B28-ijms-22-00864">28</a>,<a href="#B29-ijms-22-00864">29</a>]. (See <a href="#ijms-22-00864-f002">Figure 2</a>, which will be discussed later). By using this DBE range, our MARA filtering method removes the interference mass spectral peaks, which do not correspond to the elemental composition C<sub>x</sub>H<sub>2x+2-2*DBE</sub>O<sub>y</sub>, where DBE = 9â€"30, x and y are arbitrary integers. Now we will use the acronym M-MARA(<i>DBE</i>) for this first M-MARA run, which utilizes the DBE value as the characteristic difference for grouping and filtering of the flavonoid compounds. In order to decrease the number of false peaks remaining in the mass spectrum after filtering, a second M-MARA process using a different base unit set (R<sub>1</sub>, R<sub>2</sub>, and R<sub>3</sub>) was performed. M-MARA(<i>O</i>) calculates MR<sub>3</sub> values, which are independent from the elemental composition, except for the number of oxygen atoms in the flavonoid constituents. The divisors of M-MARA(<i>O</i>) are R<sub>1</sub> = 14.01565, R<sub>2</sub> = 2.01565, and R<sub>3</sub> = 0.0939 corresponding to the mass of CH<sub>2</sub>, H<sub>2</sub> and 7H<sub>2</sub>-CH<sub>2</sub>, respectively. <a href="#ijms-22-00864-f002">Figure 2</a>a shows the output of M-MARA(<i>O</i>) algorithm, applying the same <i>m/z</i> values (about one hundred flavonoids) as M-MARA(<i>DBE</i>) (see <a href="#ijms-22-00864-f001">Figure 1</a>d). </p> <p>Although the grouping of compounds does not correspond to the usual classification of the flavonoids (e.g., flavones, flavonols, flavans, flavanols, etc.), the MR<sub>3</sub> values of M-MARA(<i>O</i>) can be used for refining the filtering algorithm and for assigning the number of oxygens in a detected compound. By rotating the MR<sub>3</sub> plot of M-MARA(<i>O</i>) and combined with that of M-MARA(<i>DBE</i>), an overlay map can be constructed (see <a href="#ijms-22-00864-f002">Figure 2</a>b). <a href="#ijms-22-00864-f002">Figure 2</a>b illustrates our filtering algorithm: if the [MR<sub>3</sub>(<i>O</i>); MR<sub>3</sub>(<i>DBE</i>)] point, calculated from the <i>m/z</i> value of a mass spectral peak, does not coincides with an intersection of <a href="#ijms-22-00864-f002">Figure 2</a>b, this mass peak is filtered out. Here, we would like to emphasize again that the calculation of the MR<sub>3</sub> values requires simple divisions. The lower and upper DBE and O values of the filter (shown in <a href="#ijms-22-00864-f002">Figure 2</a>b) were determined based on the flavonoid database [<a href="#B28-ijms-22-00864">28</a>,<a href="#B29-ijms-22-00864">29</a>]. The smallest differences between the MR<sub>3</sub>(O) and MR<sub>3</sub>(DBE) values are 0.00113 and 0.00588, respectively. Our novel M-MARA method can be used if the applied instrument is capable of distinguishing these differences. The effect of accuracy on the efficiency of elemental composition determination was discussed by Kim et al. [<a href="#B18-ijms-22-00864">18</a>].</p> <p>The correlation between the DBE value and the number of oxygen atoms of the flavonoids offers an additional filtering criterion. </p> <p>As <a href="#ijms-22-00864-f003">Figure 3</a> shows, higher DBE values imply higher number of oxygen atoms (the Pearsonâ€™s correlation coefficient r = 0.76), decreasing thus the number of valid intersections in <a href="#ijms-22-00864-f002">Figure 2</a>b, e.g., the number of [MR<sub>3</sub>(<i>O</i>); MR<sub>3</sub>(<i>DBE</i>)] points of the mass peak filter. Furthermore, there are also strong positive correlations between the DBE versus <i>m/z</i> (r = 0.85) and the number of oxygen atoms versus <i>m/z</i> values (r = 0.97) (see <a href="#app1-ijms-22-00864">Figures S1 and S2</a>). For example, at <i>m/z</i> 600, the DBE value and the number of oxygen atoms of the flavonoid compounds are delimited between 11â€"19 and 8â€"17, respectively. </p> <p>In order to demonstrate the usefulness and capability of our novel mass filtering method, we applied it for screening of flavonoids in various herb plant extracts. As a representative example, <a href="#ijms-22-00864-f004">Figure 4</a>b shows the MR<sub>3</sub>(<i>O</i>)â€"MR<sub>3</sub>(<i>DBE</i>) plot constructed from the ESI-DIMS mass spectrum (shown in <a href="#ijms-22-00864-f004">Figure 4</a>a) of a yarrow (<i>Achillea millefolium</i> L.) extract (sample 1).</p> <p>Each point in <a href="#ijms-22-00864-f004">Figure 4</a>b corresponds to one or more peaks of the mass spectrum presented in <a href="#ijms-22-00864-f004">Figure 4</a>a. By applying our MR<sub>3</sub>(<i>O</i>)â€"MR<sub>3</sub>(<i>DBE</i>) filter (see <a href="#ijms-22-00864-f002">Figure 2</a>b) algorithm, the interference mass spectral peaks can be removed, and therefore, <a href="#ijms-22-00864-f004">Figure 4</a>c reveals only the points, which correspond to possible flavonoid constituents. For example, the mass peaks at <i>m/z</i> 285.0405 and 299.0562 belong to the [0.0389; 0.1258] point in the [MR<sub>3</sub>(<i>O</i>); MR<sub>3</sub>(<i>DBE</i>)] plane, corresponding to six oxygens in the molecule and DBE = 11 (see <a href="#ijms-22-00864-f004">Figure 4</a>c). The exact masses enable the assignment of the elemental compositions, such as [C<sub>15</sub>H<sub>9</sub>O<sub>6</sub>]<sup>1âˆ’</sup> (<i>m/z</i> 285.0405) and [C<sub>16</sub>H<sub>11</sub>O<sub>6</sub>]<sup>1âˆ’</sup> (<i>m/z</i> 299.0562), and suggest a possible identification as luteolin and chrysoeriol (a methoxy derivative of luteolin), respectively [<a href="#B30-ijms-22-00864">30</a>]. </p> <p>To summarize, the steps of our M-MARA flavonoid filtering algorithm are the following (see <a href="#ijms-22-00864-sch002">Scheme 2</a>): </p><ul> <li><span class="label" tagx="label" title="label">(1)</span><p>Calculation of the <i>Mass-remainder</i> values by Equation (1). MR<sub>3</sub>(<i>O</i>) and MR<sub>3</sub>(<i>DBE</i>) are calculated for all of the observed mass peaks (above an intensity threshold) of the ESI-DIMS mass spectrum. The divider base units (R<sub>1</sub>, R<sub>2</sub>, R<sub>3</sub> in Equation (1)) are for M-MARA(<i>O</i>): 14.01565, 2.01565, and 0.0939, and for M-MARA(<i>DBE</i>): 15.994915, 1.97927, and 0.160795.</p> </li> <li><span class="label" tagx="label" title="label">(2)</span><p>Rough filtering. The potential flavonoid MR<sub>3</sub>(<i>O</i>) and MR<sub>3</sub>(<i>DBE</i>) values are compiled in the MR-Table (see <a href="#ijms-22-00864-f002">Figure 2</a>b, <a href="#app1-ijms-22-00864">Supplementary Table S1</a>). A mass peak is removed, if its calculated MR<sub>3</sub> values have not been found in the table.</p> </li> <li><span class="label" tagx="label" title="label">(3)</span><p>Number of oxygen atom assignment and DBE value assignment. There is one-to-one correspondence between MR<sub>3</sub>(<i>O</i>) and the number of oxygen atoms in the molecule, and between MR<sub>3</sub>(<i>DBE</i>) and DBE value of the compound. The MR-Table (<a href="#app1-ijms-22-00864">Table S1</a>) can easily be used for these assignments.</p> </li> <li><span class="label" tagx="label" title="label">(4)</span><p>Fine filtering. The correlations between the DBE versus <i>m/z</i> and number of oxygen atoms versus <i>m/z</i> values (see <a href="#app1-ijms-22-00864">Figures S1 and S2</a>) offer additional filtering criteria. Based on the flavonoid database [<a href="#B28-ijms-22-00864">28</a>,<a href="#B29-ijms-22-00864">29</a>], an <i>m</i><i>/z</i> range was determined for every number of oxygen atoms and DBE value. If the <i>m/z</i> value of a mass peak does not fall into the ranges corresponding to its O and DBE values, the peak is eliminated.</p> </li> <li><span class="label" tagx="label" title="label">(5)</span><p>Elemental composition determination. Once the number of oxygen atoms (O) and the DBE values have been assigned, the general formula of the compound follows as C<sub>x</sub>H<sub>2x+2-2*DBE</sub>O<sub>O</sub>, where <i>x</i> is the only unknown variable. Knowing the exact <i>m/z</i> of the deprotonated [M-H<sup>+</sup>]<sup>â€"</sup> ion, <i>x</i> can easily be calculated by Equation (2): </p><div class="disp-formula" title="disp-formula"><i>x</i> = (<i>m/z</i> + 1.007276 âˆ’ <i>O</i> Ã— 15.994915 + 2 Ã— <i>DBE</i> Ã— 1.007825 âˆ’ 2 Ã— 1.007825)/14.01565<span class="label" tagx="label" title="label">(2)</span></div> where 1.007825, 1.007276, 15.994915, and 14.01565 are the exact masses of H, H<sup>+</sup>, oxygen and CH<sub>2</sub>, respectively.<p /> </li> </ul> <p /> <p>In the following, we will show, through a simple example, how our algorithm works. Let the <i>m/z</i> value of the compound in question be 285.0405. The steps of our M-MARA flavonoid-filtering approach are as follows: </p><ul> <li><span class="label" tagx="label" title="label">(1)</span><p>MR<sub>3</sub>(<i>O</i>) = 0.0389, MR<sub>3</sub>(<i>DBE</i>) = 0.1258</p> </li> <li><span class="label" tagx="label" title="label">(2)</span><p>The search in the MR-Table have found the calculated MR<sub>3</sub> values, the compound is a potential flavonoid.</p> </li> <li><span class="label" tagx="label" title="label">(3)</span><p>The number of oxygen atom and DBE values are O = 6, DBE = 11.</p> </li> <li><span class="label" tagx="label" title="label">(4)</span><p>The <i>m/z</i> ranges corresponding to O = 6, DBE = 11 are [283.0236; 557.2921] and [221.0596; 655.2256], respectively. The measured <i>m/z</i> = 285.0405 falls into these intervals, the compound is accepted as a flavonoid.</p> </li> <li><span class="label" tagx="label" title="label">(5)</span><p>Equation (2) gives <i>x</i> = 15, so the elemental composition of the compound is C<sub>15</sub>H<sub>10</sub>O<sub>6</sub> (see <a href="#ijms-22-00864-f004">Figure 4</a>c)</p> </li> </ul> <p /> <p>As the detailed steps and the example above demonstrate, our M-MARA flavonoid-filtering does not require any sophisticated, advanced mathematical and software tools, it needs simple mathematics, table searches and <i>IF</i> conditions. These operations can easily be implemented in a regular spreadsheet software.</p> <p>The suggested application area of our M-MARA filtering method is the comprehensive metabolite profiling of flavonoids in herb extract, foods and beverages of plant origin. As a pilot study to test our approach, the flavonoid analysis of altogether 15 commercially available brands of three herbs, such as yarrow (<i>Achillea millefolium</i> L.), elder (<i>Sambucus nigra</i> L.), and birch (<i>Betula</i> L.) were performed. <a href="#ijms-22-00864-t001">Table 1</a> summarizes the very basic statistics related to the number of mass peaks. </p> <p>The last row in <a href="#ijms-22-00864-t001">Table 1</a> reports the number of mass peaks identified by the time-consuming search in the flavonoid database [<a href="#B28-ijms-22-00864">28</a>,<a href="#B29-ijms-22-00864">29</a>]. For example, processing the ESI-DIMS spectra of Sample No. 1 (yarrow extract), M-MARA filtered 66 peaks as potential flavonoids, while the database search, based on the exact mass, found 51 peaks. It does not mean, however, that all the 66 âˆ’ 51 = 15 peaks are false hits, these peaks may include, perhaps novel, flavonoids or flavonoid derivatives. Furthermore, the mass peaks found by M-MARA and not included in the flavonoid database typically have very low peak intensities, as seen in <a href="#ijms-22-00864-f004">Figure 4</a>a.</p> <p>Furthermore, once the mass peaks of the flavonoids are selected by M-MARA-filtering, multivariate analysis such as principal component analysis (PCA) can serve as an efficient tool for discrimination of the type or brand of natural origin samples based on their flavonoid contents.</p> <p>The PCA highlights the differences among the samples creating principal components. Applying the intensities only do not reflect the concentration ratios of the flavonoids and flavonoid glycosides, due to the different ionization efficiencies and/or matrix effects. In order to overcome such cumbersome, isotope dilution [<a href="#B31-ijms-22-00864">31</a>], with deuterium labeled rutin and quercetin as internal standard (IS), was applied and all the measured intensities were corrected based on the labeled IS peaks. The ratio of the ionization efficiencies was around 2:1 quercetin to rutin.</p> <p><a href="#ijms-22-00864-f005">Figure 5</a>a demonstrates the two-dimensional PCA score scatter plot for the ESI-DIMS mass spectra, after filtered by M-MARA, of the fifteen herb extract (Sample No 1â€"No. 15). As seen in <a href="#ijms-22-00864-f005">Figure 5</a>a, significant clustering can be observed based on the herb type. In order to examine whether PCA is able to discriminate between the different brands of the same herb plant, each of the seven yarrow samples was measured five times. As it can be seen in <a href="#ijms-22-00864-f005">Figure 5</a>b, that PCA grouped the yarrow samples according their origin. </p> </div> <div class="3.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
3. Materials and Methods</div> <div class="3.1.chemicals" title="sec"> <div class="title" tagx="title" title="title">
3.1. Chemicals</div> <p>Ethanol 96% (puriss.) was purchased from Molar Chemicals (HalÃ¡sztelek, Hungary). Quercetin dihydrate 97%, ACN (acetonitrile) and deuterium oxide were received from VWR International (Leuven, Belgium). Sodium hydroxide (puriss.) was obtained from Sigma Aldrich (Taufkirchen, Germany). Rutin was isolated from a Japanese pagoda tree (<i>Sophora japonica</i>) through an alcohol extraction. Water was purified by a Direct-Q water system (Millipore, Molsheim, France).</p> </div> <div class="3.2.synthesisofdeuteriumlabeledrutinandquercetin" title="sec"> <div class="title" tagx="title" title="title">
3.2. Synthesis of Deuterium Labeled Rutin and Quercetin</div> <p>The synthesis of the labeled rutin and quercetin was done based on the method described by Hiraoka et al. [<a href="#B32-ijms-22-00864">32</a>]. Rutin (400 mg) and quercetin (62 mg) were weighed into a round-bottomed flask. The compounds were dissolved in 4.0 and 12.6 g D<sub>2</sub>O containing 10 mg/g NaOH. The solutions were heated up to 95 Â°C and stirred with a magnetic stirrer for 8 h under argon atmosphere. In the next step, the reaction mixtures were freeze-dried. The obtained powders were dissolved again in 4.0 and 12.6 g D<sub>2</sub>O and additional heating (95 Â°C) was applied for 8 h, stirred with a magnetic stirrer. After cooling down, the reaction mixture was treated by 13.3 and 20.7 mL acetic acid solution (10 <i>v/v</i> %) and stored at 4 Â°C overnight. The precipitate was filtered and washed with cold water and dried under vacuum. The HPLC-UV chromatograms and the mass spectra of the deuterated samples can be found in the Supporting Information as <a href="#app1-ijms-22-00864">Figures S4 and S5</a>.</p> </div> <div class="3.3.extractionofsamples" title="sec"> <div class="title" tagx="title" title="title">
3.3. Extraction of Samples</div> <p>15 commercially available brands of three herbs, such as yarrow (<i>Achillea millefolium</i> L.), elder (<i>Sambucus nigra</i> L.), and birch (<i>Betula</i> L.) were investigated. The manufacturer and the location of collections are in the Supporting Information as <a href="#app1-ijms-22-00864">Table S2</a>. Plant samples were ground, after which the powders were swollen in ethanol: water-7:3 (<i>v/v</i>) for 16 h. After the swelling process, the diluted samples were sonicated for an hour and then filtered. The filtrates were dried under vacuum to constant weights.</p> </div> <div class="3.4.electrosprayquadrupoletime-of-flightmassspectrometry(esi-qtofms)" title="sec"> <div class="title" tagx="title" title="title">
3.4. Electrospray Quadrupole Time-of-Flight Mass Spectrometry (ESI-QTOF MS)</div> <p>The high resolution mass spectrometric measurements were performed using a Maxis II type Qq-TOF MS instrument (Bruker Daltonics, Bremen, Germany) equipped with an electrospray ion-source where the spray voltage was 3.5 kV. N<sub>2</sub> was applied as drying (200 Â°C, 4.0 L/min) and nebulizer gas (0.5 bar). The mass spectra were recorded by means of a digitizer at a sampling rate of 2 GHz. The mass accuracy of the instrument is &amp;lt;600 ppb (internal calibration), the resolution is 40,000 at <i>m/z</i> 400 (FWHM). Each spectrum was calibrated internally with the sodium formate clusters. The spectra were evaluated with the Compass DataAnalysis 4.4 software (Bruker Daltonics, Bremen, Germany). The sample solutions were injected by a 7100 model capillary electrophoresis instrument without any separation. A fused silica capillary (70 cm Ã— 7.5 Âµm (id)) was applied, with 130 s, 3 bar preconditioning, 40 s, 2 bar injection and 100 s, 3 bar as postconditioning. The samples were dissolved in ethanol: water 1:1 (<i>v/v</i>) with 0.02% formic acid at a concentration of 60 Âµg/mL on a dry basis. The concentrations of the deuterated internal standards were 0.146 and 0.210 Âµg/mL quercetin and rutin, respectively.</p> </div> <div class="3.5.liquidchromatography(hplc-uv-ms)" title="sec"> <div class="title" tagx="title" title="title">
3.5. Liquid Chromatography (HPLC-UV-MS)</div> <p>The characterization of the deuterated standard materials was performed by liquid chromatographâ€"mass spectrometry measurements. For chromatography, a Waters 2695 Separations Module (Waters, Milford, MA, USA) equipped with a thermostable autosampler (5 Â°C) and a column module (40 Â°C), a Waters 2996 photodiode-array detector and a VDSphere PUR 100 C18-M-SE (4.6 Ã— 150 cm, 5 Âµm) column. Gradient elution was used with 1.5 mL/min flow rate. Mobile phase A was ACN (acetonitrile) containing 0.02% formic acid and mobile phase B was water containing 0.02% formic acid. The gradient condition was as follows: initially 20% A and 80% B, 0â€"20 min linear change to 50% A and 50% B, 20â€"30 min linear change to 90% A and 10% B and held this setting for additional 10 min. The analytes were detected at Î» = 240 nm. Mass spectrometric detection was carried out by a MicroTOF-Q type Qq-TOF MS instrument (Bruker Daltoniks) using an ESI-source with negative ion mode. The spray voltage was set to 4.5 kV. The nebulizer pressure was 1.6 bar, while, the temperature of the drying gas (N<sub>2</sub>, 7 L/min) was kept at 200 Â°C. The data evaluation was performed by the DataAnalysis 3.4 software from Bruker.</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
4. Conclusions</div> <p>Multi-step Mass-remainder analysis (M-MARA) was successfully applied for data mining of complex, peak-rich ESI-DIMS mass spectra of various herb extracts. M-MARA eliminated most of the differences in the elemental composition of the flavonoids, the target class of our study, and a single characteristic difference was highlighted and used for filtering. Moreover, our flavonoid-filtering algorithm performs the mass remainder calculation twice, using different divisors, decreasing thereby the number of false hits of the mass filter, and facilitating the determination of the elemental composition. A huge benefit of our method is the high speed and the low demand for computing power and memory that enables the real time application. For example, the <i>Auto MSMS</i> feature of the mass spectrometerâ€™s acquisition software can be extended by a filtering script or module, and applied for the automatic recognition, isolation and fragmentation of the flavonoids. In addition to the elemental composition, tandem mass spectra enable the distinction and identification of the flavonoid isomers as well.</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note:</b> MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>The following are available online at <span class="uri" title="uri"><a href="https://www.mdpi.com/1422-0067/22/2/864/s1">https://www.mdpi.com/1422-0067/22/2/864/s1</a></span>.</p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="ijms-22-00864-s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>Conceptualization; T.N., Ã�.K., S.K., Methodology; T.N., G.R., Ã�.K., Data curation; T.N., G.R., Ã�.K., Formal analysis; T.N., G.R., Ã�.K., Supervision; T.N., Ã�.K., M.Z., S.K., Visualization; T.N., G.R., Ã�.K., Roles/Writingâ€"original draft; Ã�.K., T.N., Writingâ€"review &amp;amp;editing; M.Z., S.K. All authors have read and agreed to the published version of the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>The work was supported by the GINOP-2.3.2-15-2016-00041 and GINOP-2.3.3-15-2016-00004 grants. The projects were co-financed by the European Union and the European Regional Development Fund. Furthermore, this paper was also supported by Grant No. FK-132385 from National Research, Development and Innovation Office (NKFI) and by the ÃšNKP-20-5 new national excellence program of the ministry for innovation and technology from the source of the national research, development and innovation fund, and the by the JÃ¡nos Bolyai Research Scholarship of the Hungarian Academy of Sciences.</p> <div class="title" tagx="title" title="title">
Data Availability Statement</div> <p>Data is contained within the article or <a href="#app1-ijms-22-00864">supplementary material</a>.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-ijms-22-00864" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fiehn</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolomicsâ€"The link between genotypes and phenotypes</span><span class="source" tagx="source" title="source">Plant Mol. Biol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">155</span><span class="lpage" tagx="lpage" title="lpage">171</span><span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1013713905833">10.1023/A:1013713905833</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11860207">11860207</a></span></span></li> <li tag="ref"><a name="B2-ijms-22-00864" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guy</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kopka</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moritz</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant metabolomics coming of age</span><span class="source" tagx="source" title="source">Physiol. Plant.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">132</span><span class="fpage" tagx="fpage" title="fpage">113</span><span class="lpage" tagx="lpage" title="lpage">116</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1399-3054.2007.01020.x">10.1111/j.1399-3054.2007.01020.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18251854">18251854</a></span></span></li> <li tag="ref"><a name="B3-ijms-22-00864" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolfender</span><span class="given-names" tagx="given-names" title="given-names">J.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marti</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertrand</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Current approaches and challenges for the metabolite profiling of complex natural extracts</span><span class="source" tagx="source" title="source">J. Chromatogr. A</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">1382</span><span class="fpage" tagx="fpage" title="fpage">136</span><span class="lpage" tagx="lpage" title="lpage">164</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chroma.2014.10.091">10.1016/j.chroma.2014.10.091</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25464997">25464997</a></span></span></li> <li tag="ref"><a name="B4-ijms-22-00864" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">O.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markham</span><span class="given-names" tagx="given-names" title="given-names">K.R.</span></span></span><span class="source" tagx="source" title="source">Flavonoids: Chemistry, Biochemistry and Applications</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">CRC Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Boca Raton, FL, USA</span><span class="year" tagx="year" title="year">2005</span></span></li> <li tag="ref"><a name="B5-ijms-22-00864" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buckingham</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munasinghe</span><span class="given-names" tagx="given-names" title="given-names">V.R.N.</span></span></span><span class="source" tagx="source" title="source">Dictionary of Flavonoids with CD-ROM</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Abingdon, UK</span><span class="year" tagx="year" title="year">2015</span></span></li> <li tag="ref"><a name="B6-ijms-22-00864" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grotewold</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="source" tagx="source" title="source">The Science of Flavonoids</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin/Heidelberg, Germany</span><span class="year" tagx="year" title="year">2007</span></span></li> <li tag="ref"><a name="B7-ijms-22-00864" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panche</span><span class="given-names" tagx="given-names" title="given-names">A.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diwan</span><span class="given-names" tagx="given-names" title="given-names">A.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandra</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Flavonoids: An overview</span><span class="source" tagx="source" title="source">J. Nutr. Sci.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e47</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/jns.2016.41">10.1017/jns.2016.41</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28620474">28620474</a></span></span></li> <li tag="ref"><a name="B8-ijms-22-00864" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akimoto</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ara</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakajima</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suda</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikeda</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahashi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muneto</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shibata</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">FlavonoidSearch: A system for comprehensive flavonoid annotation by mass spectrometry</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">1243</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-01390-3">10.1038/s41598-017-01390-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28455528">28455528</a></span></span></li> <li tag="ref"><a name="B9-ijms-22-00864" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kachlicki</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piasecka</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stobiecki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marczak</span><span class="given-names" tagx="given-names" title="given-names">Å�.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural Characterization of Flavonoid Glycoconjugates and Their Derivatives with Mass Spectrometric Techniques</span><span class="source" tagx="source" title="source">Molecules</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">21</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1494</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules21111494">10.3390/molecules21111494</a></span></span></li> <li tag="ref"><a name="B10-ijms-22-00864" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jia</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span><span class="mixed-article-title" title="mixed-article-title">High-throughput mass spectrometry scheme for screening and quantification of flavonoids in antioxidant nutraceuticals</span><span class="source" tagx="source" title="source">J. Chromatogr. A</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">1608</span><span class="fpage" tagx="fpage" title="fpage">460408</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chroma.2019.460408">10.1016/j.chroma.2019.460408</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31378531">31378531</a></span></span></li> <li tag="ref"><a name="B11-ijms-22-00864" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bingol</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bruschweiler-Li</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somogyi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BrÃ¼schweiler</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolomics Beyond Spectroscopic Databases: A Combined MS/NMR Strategy for the Rapid Identification of New Metabolites in Complex Mixtures</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">3864</span><span class="lpage" tagx="lpage" title="lpage">3870</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ac504633z">10.1021/ac504633z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25674812">25674812</a></span></span></li> <li tag="ref"><a name="B12-ijms-22-00864" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TimÃ¡ri</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costa dos Santos</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">S.O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bruschweiler-Li</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BrÃ¼schweiler</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Real-Time Pure Shift HSQC NMR for Untargeted Metabolomics</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">2304</span><span class="lpage" tagx="lpage" title="lpage">2311</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.analchem.8b04928">10.1021/acs.analchem.8b04928</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30608652">30608652</a></span></span></li> <li tag="ref"><a name="B13-ijms-22-00864" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuhrer</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heer</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Begemann</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zamboni</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">High-Throughput, Accurate Mass Metabolome Profiling of Cellular Extracts by Flow Injectionâ€"Time-of-Flight Mass Spectrometry</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">7074</span><span class="lpage" tagx="lpage" title="lpage">7080</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ac201267k">10.1021/ac201267k</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21830798">21830798</a></span></span></li> <li tag="ref"><a name="B14-ijms-22-00864" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sleno</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The use of mass defect in modern mass spectrometry</span><span class="source" tagx="source" title="source">J. Mass Spectrom.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">226</span><span class="lpage" tagx="lpage" title="lpage">236</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jms.2953">10.1002/jms.2953</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22359333">22359333</a></span></span></li> <li tag="ref"><a name="B15-ijms-22-00864" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ray</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Humphreys</span><span class="given-names" tagx="given-names" title="given-names">W.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skiles</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanders</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detection and Characterization of Metabolites in Biological Matrices Using Mass Defect Filtering of Liquid Chromatography/High Resolution Mass Spectrometry Data</span><span class="source" tagx="source" title="source">Drug Metab. Dispos.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">1722</span><span class="lpage" tagx="lpage" title="lpage">1733</span><span class="pub-id"><a href="https://dx.doi.org/10.1124/dmd.106.009241">10.1124/dmd.106.009241</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16815965">16815965</a></span></span></li> <li tag="ref"><a name="B16-ijms-22-00864" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ray</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mass defect filter technique and its applications to drug metabolite identification by high-resolution mass spectrometry</span><span class="source" tagx="source" title="source">J. Mass Spectrom.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">999</span><span class="lpage" tagx="lpage" title="lpage">1016</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jms.1610">10.1002/jms.1610</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19598168">19598168</a></span></span></li> <li tag="ref"><a name="B17-ijms-22-00864" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ekanayaka</span><span class="given-names" tagx="given-names" title="given-names">E.A.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Celiz</span><span class="given-names" tagx="given-names" title="given-names">M.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">A.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Relative Mass Defect Filtering of Mass Spectra: A Path to Discovery of Plant Specialized Metabolites</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">167</span><span class="fpage" tagx="fpage" title="fpage">1221</span><span class="lpage" tagx="lpage" title="lpage">1232</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.114.251165">10.1104/pp.114.251165</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25659383">25659383</a></span></span></li> <li tag="ref"><a name="B18-ijms-22-00864" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodgers</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marshall</span><span class="given-names" tagx="given-names" title="given-names">A.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Truly â€œexactâ€� mass: Elemental composition can be determined uniquely from molecular mass measurement at âˆ¼0.1 mDa accuracy for molecules up to âˆ¼500 Da</span><span class="source" tagx="source" title="source">Int. J. Mass Spectrom.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">251</span><span class="fpage" tagx="fpage" title="fpage">260</span><span class="lpage" tagx="lpage" title="lpage">265</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijms.2006.02.001">10.1016/j.ijms.2006.02.001</a></span></span></li> <li tag="ref"><a name="B19-ijms-22-00864" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kendrick</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A Mass Scale Based on CH2 = 14.0000 for High Resolution Mass Spectrometry of Organic Compounds</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">1963</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">2146</span><span class="lpage" tagx="lpage" title="lpage">2154</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ac60206a048">10.1021/ac60206a048</a></span></span></li> <li tag="ref"><a name="B20-ijms-22-00864" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouquet</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satoh</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">First Gut Instincts Are Always Right: The Resolution Required for a Mass Defect Analysis of Polymer Ions Can Be as Low as Oligomeric</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">2404</span><span class="lpage" tagx="lpage" title="lpage">2408</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.analchem.7b04518">10.1021/acs.analchem.7b04518</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29336551">29336551</a></span></span></li> <li tag="ref"><a name="B21-ijms-22-00864" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouquet</span><span class="given-names" tagx="given-names" title="given-names">T.N.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Kendrick analysis for polymer mass spectrometry</span><span class="source" tagx="source" title="source">J. Mass Spectrom.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">933</span><span class="lpage" tagx="lpage" title="lpage">947</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jms.4480">10.1002/jms.4480</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31758605">31758605</a></span></span></li> <li tag="ref"><a name="B22-ijms-22-00864" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouquet</span><span class="given-names" tagx="given-names" title="given-names">T.N.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pizzala</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rollet</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crozet</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giusti</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charles</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mass Spectrometry-Based Analytical Strategy for Comprehensive Molecular Characterization of Biodegradable Poly(lactic-co-glycolic Acid) Copolymers</span><span class="source" tagx="source" title="source">J. Am. Soc. Mass. Spectrom.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">1554</span><span class="lpage" tagx="lpage" title="lpage">1562</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/jasms.0c00137">10.1021/jasms.0c00137</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32510219">32510219</a></span></span></li> <li tag="ref"><a name="B23-ijms-22-00864" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuki</span><span class="given-names" tagx="given-names" title="given-names">Ã�.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zsuga</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ©ki</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mass-Remainder Analysis (MARA): A New Data Mining Tool for Copolymer Characterization</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">3892</span><span class="lpage" tagx="lpage" title="lpage">3897</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.analchem.7b04730">10.1021/acs.analchem.7b04730</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29443513">29443513</a></span></span></li> <li tag="ref"><a name="B24-ijms-22-00864" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuki</span><span class="given-names" tagx="given-names" title="given-names">Ã�.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zsuga</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ©ki</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mass-Remainder Analysis (MARA): An Improved Method for Elemental Composition Assignment in Petroleomics</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">6479</span><span class="lpage" tagx="lpage" title="lpage">6486</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.analchem.8b04976">10.1021/acs.analchem.8b04976</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30912635">30912635</a></span></span></li> <li tag="ref"><a name="B25-ijms-22-00864" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuki</span><span class="given-names" tagx="given-names" title="given-names">Ã�.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hashimov</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zsuga</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ©ki</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Multistep Mass-Remainder Analysis and its Application in Copolymer Blends</span><span class="source" tagx="source" title="source">Macromolecules</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">1199</span><span class="lpage" tagx="lpage" title="lpage">1204</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.macromol.9b02409">10.1021/acs.macromol.9b02409</a></span></span></li> <li tag="ref"><a name="B26-ijms-22-00864" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hashimov</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuki</span><span class="given-names" tagx="given-names" title="given-names">Ã�.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zsuga</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ©ki</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tandem Mass-Remainder Analysis of Industrially Important Polyether Polyols</span><span class="source" tagx="source" title="source">Polymers</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">2768</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/polym12122768">10.3390/polym12122768</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/33255196">33255196</a></span></span></li> <li tag="ref"><a name="B27-ijms-22-00864" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuki</span><span class="given-names" tagx="given-names" title="given-names">Ã�.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hashimov</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">File</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zsuga</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ©ki</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mass Spectrometric Characterization of Epoxidized Vegetable Oils</span><span class="source" tagx="source" title="source">Polymers</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">394</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/polym11030394">10.3390/polym11030394</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30960377">30960377</a></span></span></li> <li tag="ref"><a name="B28-ijms-22-00864" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="mixed-article-title" title="mixed-article-title">Flavonoid Database, Arita Laboratory, National Institute of Genetics</span><span class="comment">Available online: <a href="http://metabolomics.jp/wiki/Main_Page">http://metabolomics.jp/wiki/Main_Page</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 5 October 2020)</span></span></li> <li tag="ref"><a name="B29-ijms-22-00864" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arita</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suwa</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Search extension transforms Wiki into a relational system: A case for flavonoid metabolite database</span><span class="source" tagx="source" title="source">BioData Mining</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1756-0381-1-7">10.1186/1756-0381-1-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18822113">18822113</a></span></span></li> <li tag="ref"><a name="B30-ijms-22-00864" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huo</span><span class="given-names" tagx="given-names" title="given-names">C.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">M.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.-F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Q.-W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiyota</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cytotoxic flavonoids from the flowers of Achillea millefolium</span><span class="source" tagx="source" title="source">Chem. Nat. Compd.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">958</span><span class="lpage" tagx="lpage" title="lpage">962</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10600-013-0438-y">10.1007/s10600-013-0438-y</a></span></span></li> <li tag="ref"><a name="B31-ijms-22-00864" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kruve</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Strategies for Drawing Quantitative Conclusions from Nontargeted Liquid Chromatographyâ€"High-Resolution Mass Spectrometry Analysis</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">4691</span><span class="lpage" tagx="lpage" title="lpage">4699</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.analchem.9b03481">10.1021/acs.analchem.9b03481</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32134258">32134258</a></span></span></li> <li tag="ref"><a name="B32-ijms-22-00864" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiraoka</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyamoto</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baba</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furuta</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Preparation of deuterium-labeled rutin by hydrogen exchange reaction</span><span class="source" tagx="source" title="source">J. Label. Compd. Radiopharm.</span><span class="year" tagx="year" title="year">1981</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">613</span><span class="lpage" tagx="lpage" title="lpage">619</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jlcr.2580180502">10.1002/jlcr.2580180502</a></span></span></li> </ul> </div> <div class="display-objects" title="display-objects"> <div class="title" tagx="title" title="title">
Figures, Schemes and Table</div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Scheme 1</span></a></div>   <p>The structures of main flavonoid groups and their double bond equivalent (DBE) values.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>(<b>a</b>) MR<sub>1</sub> (<i>DBE</i>) versus <i>m/z</i> plot for eight flavonoids detected in the extract of the medical plant yarrow (<i>Achillea millefolium</i> L.). (<b>b</b>) MR<sub>2</sub> (<i>DBE</i>) <i>vers</i>us <i>m/z</i> plot for eight flavonoids detected in the extract of the medical plant yarrow (<i>Achillea millefolium</i> L.). (<b>c</b>) MR<sub>3</sub> (<i>DBE</i>) versus <i>m/z</i> plot for eight flavonoids detected in the extract of the medical plant yarrow (<i>Achillea millefolium</i> L.). (<b>d</b>) MR<sub>3</sub> (<i>DBE</i>) versus <i>m/z</i> plot calculated from the theoretical <i>m/z</i> values of about one hundred flavonoids. (<i>DBE</i> in parenthesis indicates that the double bond equivalent values were revealed as the characteristic difference.).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>(<b>a</b>) MR<sub>3</sub> versu<i>s m/z</i> plot calculated from the theoretical <i>m/z</i> values of about one hundred flavonoids (applied for <a href="#ijms-22-00864-f001">Figure 1</a>d). (<i>O</i> in parenthesis indicates that the number of oxygen atoms were revealed as the characteristic difference.) (<b>b</b>) MR<sub>3</sub>(<i>DBE</i>)â€"MR<sub>3</sub>(<i>O</i>) mesh used as a mass spectral filter.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>The DBE values and the number of oxygen atoms of the compounds in the flavonoid database [<a href="#B28-ijms-22-00864">28</a>,<a href="#B29-ijms-22-00864">29</a>]. The Pearsonâ€™s correlation coefficient r = 0.76.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>(<b>a</b>) ESI-DIMS spectrum of yarrow (<i>Achillea millefolium</i> L.) extract. (<b>b</b>) MR<sub>3</sub>(<i>O</i>)â€"MR<sub>3</sub>(<i>DBE</i>) plot of a yarrow (<i>Achillea millefolium</i> L.) extract. (<b>c</b>) Filtered MR<sub>3</sub>(<i>O</i>)â€"MR<sub>3</sub>(<i>DBE</i>) plot of a yarrow extract.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Scheme 2</span></a></div>   <p>Flowchart of the Multistep Mass Remainder Analysis (M-MARA) flavonoid filtering algorithm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Principal Component Analysis (PCA) score plots for the filtered ESI-DIMS mass spectra of (<b>a</b>) the different herbs (variance % PC1: 43.4%, PC2: 31.5%), (<b>b</b>) different brands of yarrow (variance % PC1: 66.0%, PC2: 32.0%).</p>   </div> <div class="table-wrap_UNKNOWN" id="ijms-22-00864-t001" orientation="portrait" position="float">
ijms-22-00864-t001_Table 1<span class="label" tagx="label" title="label">Table 1</span> <p>The number of <i>m/z</i> peaks in the ESI-DIMS spectra of herb extracts before and after M-MARA filtering, and identified by database search.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th colspan="2" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Type</th> <th colspan="7" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Yarrow (<i>Achillea</i><i>millefolum</i> L.)</th> <th colspan="4" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Elder (<i>Sambucus nigra</i> L.)</th> <th colspan="4" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1">Birch (<i>Betula</i> L.)</th> </tr> <tr> <th colspan="2" align="center" valign="middle" style="border-bottom:solid thin" rowspan="1">No.</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">3</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">5</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">6</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">8</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">9</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">10</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">11</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">12</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">13</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">14</th> <th align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">15</th> </tr> </thead> <tbody> <tr> <td rowspan="3" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">No. of peaks</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">in the spectra</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1258</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">889</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">883</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1254</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">842</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1070</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1516</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1251</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1293</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1067</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1033</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1578</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1180</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1662</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1590</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">M-MARA</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">66</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">15</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">52</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">65</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">53</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">58</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">56</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">49</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">35</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">52</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">40</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">60</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">57</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">67</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">63</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">database search</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">51</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">9</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">36</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">45</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">40</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">45</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">43</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">37</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">28</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">41</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">31</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">41</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">45</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">45</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">46</td> </tr> </tbody> </table> </div> </div> </div>  </body></html>