<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Bioenerg Biomembr</span><span class="iso-abbrev" title="iso-abbrev">J Bioenerg Biomembr</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Bioenergetics and Biomembranes</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0145-479X</span><span class="issn-epub" tagx="issn" title="issn-epub">1573-6881</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer US</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7704527">7704527</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/33247393">33247393</a></span><span class="publisher-id" title="publisher-id">9858</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1007/s10863-020-09858-0">10.1007/s10863-020-09858-0</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
QSAR, molecular docking, design, and pharmacokinetic analysis of 2-(4-fluorophenyl) imidazol-5-ones as anti-breast cancer drug compounds against MCF-7 cell line</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-5086-2344</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawal</span><span class="given-names" tagx="given-names" title="given-names">Hadiza Abdulrahman</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">azeezalawal@gmail.com</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uzairu</span><span class="given-names" tagx="given-names" title="given-names">Adamu</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">adamuuzairu@yahoo.com</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uba</span><span class="given-names" tagx="given-names" title="given-names">Sani</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">saniuba10@yahoo.com</span></span><a href="#Aff1" /></span><span class="citation_author_institution" id="Aff1">[], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.411225.1</span><span class="institution-id" title="institution-id">0000 0004 1937 1493</span><span class="institution" title="institution">Department of Chemistry, </span><span class="institution" title="institution">Ahmadu Bello University, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-11-11</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-11-11</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020</span></span><span class="volume" tagx="volume" title="volume">52</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">475</span><span class="lpage" tagx="lpage" title="lpage">494</span><span class="history" title="history"><span class="received" title="received">received: 2020-8-5</span><span class="accepted" title="accepted">accepted: 2020-10-28</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">The anti-proliferative activities of Novel series of 2-(4-fluorophenyl) imidazol-5-ones against MCF-7 breast cancer cell line were explored via in-slico studies which includes Quantitative structureâ€"activity relationship QSAR, molecular docking studies, designing new compounds, and analyzing the pharmacokinetics properties of the designed compounds. From the QSAR analysis, model number one emerged the best as seen from the arithmetic assessments of (R<sup>2</sup>)â€‰=â€‰0.6981, (R<sup>2</sup><sub>adj</sub>)â€‰=â€‰0.6433, (Q<sup>2</sup>)â€‰=â€‰0.5460 and (R<sup>2</sup><sub>pred</sub>) of 0.5357. Model number one was used in designing new derivative compounds, with higher effectiveness against estrogen positive breast cancer (MCF-7 cell line). The Molecular docking studies between the derivatives and Polo-like kinases (Plk1) receptor proved that the derivatives of 2-(4-fluorophenyl) imidazol-5-ones bind tightly to the receptor, thou ligand 24 and 27 had the highest binding affinities of âˆ’8.8 andâ€‰âˆ’â€‰9.1Â kcal/mol, which was found to be higher than Doxorubicin with a docking score of âˆ’8.0Â kcal/mol. These new derivatives of 2-(4-fluorophenyl) imidazol-5-ones shall be excellent inhibitors against (plk1). The pharmacokinetics analysis performed on the new structures revealed that all the structures passed the test and also the Lipinski rule of five, and they could further proceed to pre-clinical tests. They both revealed a revolution in medicine for developing novel anti-breast cancer drugs against MCF-7 cell line.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
QSAR analysis</div> <div class="kwd" title="kwd">
2-(4-fluorophenyl) imidazol-5-ones</div> <div class="kwd" title="kwd">
Ligand-based design</div> <div class="kwd" title="kwd">
Pharmacokinetics</div> <div class="kwd" title="kwd">
Breast cancer</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© Springer Science+Business Media, LLC, part of Springer Nature 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Cancer is described by uncontrolled cell proliferation, thereby affecting the surrounding tissue, and over again spreading throughout the body. It is a complicated disease. (Bajaj et al. <a href="#CR5">2018</a>). Despite the vast high-tech and social enhancement, cancer remains the most alarming disease and a leading cause of pain and death in humans (Bhaumik et al. <a href="#CR6">2019</a>).</p> <p id="Par3">Among women, after lung cancer, cancer of the breast is the second source of mortality. About 40,610 women passed away from cancer of the breast and about 252,710 more diagnoses were projected in 2017 (Bajaj et al. <a href="#CR5">2018</a>). Cancer of the breast accounts for about 24% of all cancer types in females (Xiao et al. <a href="#CR26">2018</a>). A likely population of people with breast cancer has similar features such as older age, lack of prolonged breast feeding, adding weight, overdue age at first birth, lack of exercise, etc. (Liu et al. <a href="#CR18">2018</a>). Detailed investigation of pathways and mechanisms on how cancer spreads and discovering many anti-cancer agents have made a breakthrough in the treatment of cancer (Bhaumik et al. <a href="#CR6">2019</a>).</p> <p id="Par4">Luminal type of breast cancer is Estrogen receptor (ER)âˆ’/progesterone receptor (PR) - the positive type which is caused by the overexpression of estrogen receptor Î± (ERÎ±). It is made up of about 70% of the mammary tumor patients tagged as ER-positive (ER+). The endless stimulation of ERÎ± by estrogens induce the multiplication of cancer cells, MCF-7 cell line (Jordan et al. <a href="#CR16">2007</a>). The master mitotic regulator, Polo-like kinase 1 (Plk1), is an important gene cell division and a known cancer drug target. It is found overexpressed in a large collection of different cancer types and this tumoral overexpression often correlates with poor patient prognosis (De CÃ¡rcer <a href="#CR9">2019</a>).</p> <p id="Par5">In the study of (Sanhaji et al. <a href="#CR23">2012</a>), it showed that Polo-like kinases (Plk1) causes destructive proliferation in tumor cells and strongly stimulates the development of cell circle. Plk1 overexpression allows cells to supersede barriers, causing genomic uncertainty and stimulating the alteration of mammalian cells. Plk1 was proven as amongst the utmost striking receptor for breast cancer treatment. Plk1 mediates estrogen receptor (ER) which regulates gene overexpression in human breast cancer cells. Recently, (Abo-Elanwar et al. <a href="#CR4">2019</a>) reported novel Thirty-nine derivatives of Imidazolones connected to chalone moiety which showed a great anti-breast cancer activity against MCF-7 cell line. Imidazole and its derivatives have much significance in both natural products and synthetic molecules. The exclusive molecules have great electron-rich features which allow them to bind easily with diverse enzymes and receptors, thus showing wide anti-proliferative activities (Zhang et al. <a href="#CR28">2020</a>). Several activities such as anticancer, antimicrobial (Premakumari et al. <a href="#CR21">2014</a>), cardio-activity, and angiotensin II receptor antagonistic activity have been described explicitly in compounds containing imidazolone moiety (Abo-Elanwar et al. <a href="#CR4">2019</a>).</p> <p id="Par6">Chemotherapy remains one of the best and fast clinical options though it is often limited due to undesirable toxic effect including weight loss, fatigue, nausea, loss of appetite, and so on, making it urgent to develop more effective drug candidates with less toxicity to eradicate this disease (Iqbal et al. <a href="#CR15">2019</a>). The computer-aided drug design approach saves time and ensures better effectiveness of the drug candidate. This research would be aimed at exploring the novel derivatives of imidazolone by building a mathematical model (QSAR) that predicts the anti-proliferative activities from its compound and using plk1 receptor with the derivatives to understand their interactions via molecular docking studies towards anti-breast cancer drug discovery, concentrating on breast cancer treatment with less toxicity and more effectiveness.</p> </div> <div class="methodology" title="sec"> <div class="title" tagx="title" title="title">
Methodology</div> <div class="computerapplications" title="sec"> <div class="title" tagx="title" title="title">
Computer applications</div> <p id="Par7">The software includes; Chemdraw version 12.0.2, Spartanâ€™14 (version 1.1.2), Material studio (V8) software, Pyrex software, PADEL V2.20, DTC data lab software version, and Auto dock visualizer version 4.2.</p> </div> <div class="qsarstudies" title="sec"> <div class="title" tagx="title" title="title">
QSAR studies</div> <div class="dataset" title="sec"> <div class="title" tagx="title" title="title">
Dataset</div> <p id="Par8">39 derivatives of Imidazolones connected to chalone moiety with anti-proliferative activities (IC<sub>50</sub>) on MCF-7 cancer cell lines were obtained from (Abo-Elanwar et al. <a href="#CR4">2019</a>) writings. The anti-proliferative activities were measured in inhibitory concentration (IC<sub>50</sub>) and then converted to the logarithm scale (pIC<sub>50</sub>). The tabulated form of the IC<sub>50</sub> is measured in concentration of micromolar (Î¼M) and the pIC<sub>50</sub> is shown in Table <a href="#Tab1">1</a>. </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\mathrm{pIC}}_{50}=-\log 10\ \left({\mathrm{IC}}_{50}\ \mathrm{x}\ {10}^{-6}\right). $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
pIC</div> <div class="mn_UNKNOWN">
50</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mo_UNKNOWN">
log</div> <div class="mn_UNKNOWN">
10</div> <div class="mspace_UNKNOWN" /> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
IC</div> <div class="mn_UNKNOWN">
50</div> </div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
x</div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
6</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div></div><div class="graphic" position="anchor" /> </div> </div> <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> <p>2-(4-fluorophenyl) imidazol-5-ones derivatives and its activities</p>  <div class="graphic" position="anchor" id="d30e386" /> <div class="graphic" position="anchor" id="d30e387" /> <div class="graphic" position="anchor" id="d30e388" /> <div class="graphic" position="anchor" id="d30e389" /> <div class="graphic" position="anchor" id="d30e390" /> <div class="graphic" position="anchor" id="d30e391" /> <div class="graphic" position="anchor" id="d30e392" /> </div> <p /> </div> <div class="molecularoptimization" title="sec"> <div class="title" tagx="title" title="title">
Molecular optimization</div> <p id="Par9">The geometric optimization is executed such that the countable electronic and molecular parameters could depict the original physicochemical properties of the observed molecule. (Putri et al. <a href="#CR22">2019</a>). The derivatives were sketched using Chemdraw (V12.0.2) software in 2D format. They were then converted to 3D format and further optimized using Spartan 14 (V1.1.4) software, with the parameters, Density Functional Theory (DFT) at B3LYP, 6-31G/ basis set (Ibrahim et al. <a href="#CR14">2018</a>; Abdullahi et al. <a href="#CR2">2019</a>).</p> </div> <div class="moleculardescriptors" title="sec"> <div class="title" tagx="title" title="title">
Molecular descriptors</div> <p id="Par10">Pharmaceutical Data Exploration Laboratory Software V (2.20) was used in calculating molecular descriptors for the thirty-nine (39) optimized compounds of Imidazolones derivatives which were then converted to SDF format after optimization. (Yap <a href="#CR27">2011</a>).</p> </div> <div class="pretreatmentanddivisionofdataset" title="sec"> <div class="title" tagx="title" title="title">
Pretreatment and division of data set</div> <p id="Par11">The outcomes obtained from PADEL-software were pretreated to remove persistent values and undesirable descriptors using Data Pre-treatment software GUI 1.2. (Abdulrahman et al. <a href="#CR3">2020</a>). Kennard-Stone algorithm (Kennard and Stone <a href="#CR17">1969</a>) method divided the derivatives into 27 calibration and 12 validation sets to get a mathematical equation.</p> </div> <div class="qsarequation" title="sec"> <div class="title" tagx="title" title="title">
QSAR equation</div> <p id="Par12">Material studio software (V8) was used in constructing a model with Genetic Function Approximation (GFA) procedure. The dependent variable is the anti-proliferative activities (pIC<sub>50</sub>) and the independent variable is model parameters.</p> </div> <div class="validatingtheequation(internal)" title="sec"> <div class="title" tagx="title" title="title">
Validating the equation (internal)</div> <p id="Par13">The validity of the built model should be tested on an external set of data that has not yet been used during the process of developing the model (Tropsha et al. <a href="#CR24">2003</a>). The built equations were evaluated using the Friedman formula; (Friedman <a href="#CR11">1991</a>). </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \mathrm{LOF}=\kern0.5em \frac{SEE}{M{\left[1-\beta \left(\frac{c+d\times p}{M}\right)\right]}^2} $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
LOF</div> <div class="mo_UNKNOWN">
=</div> <div class="mspace_UNKNOWN" /> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
SEE</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
Î²</div> <div class="mfenced_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
Ã—</div> <div class="mi_UNKNOWN">
p</div> </div> <div class="mi_UNKNOWN">
M</div> </div> </div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par14">Where: <i>SEE</i> is the Standard Estimated Error. If SEE is small, it suggests an enhanced equation. SEE is expressed below; </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \mathrm{SEE}=\sqrt{\frac{{\left({Y}_{exp}-{Y}_{pred}\right)}^2}{N-P-1}} $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
SEE</div> <div class="mo_UNKNOWN">
=</div> <div class="msqrt_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
exp</div> </div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mtext_UNKNOWN">
pred</div> </div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par15"><i>d</i> equals user-defined smoothing parameter, <i>C</i> equals to the sum of the model terms, <i>M</i> equals the sum of train set compounds and <i>p</i> is the equation parameters (Ibrahim et al. <a href="#CR14">2018</a>). The correlation coefficient (R<sup>2</sup>) accounts for the fragment of the total difference of the equation. The nearer the R<sup>2</sup> value is to 1, the more enhanced the model is built. R<sup>2</sup> is given as: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\mathrm{R}}^2=1-\left[\frac{\sum {\left({Y}_{exp}-{Y}_{pred}\right)}^2}{\sum {\left({Y}_{exp}-{Y}_{training}\right)}^2}\right] $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mfenced_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
exp</div> </div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mtext_UNKNOWN">
pred</div> </div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
exp</div> </div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mtext_UNKNOWN">
training</div> </div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par16">Where <i>Y</i><sub><i>exp</i></sub> and <i>Y</i><sub><i>pred</i></sub> are the means of biological and calculated activities of the calibration set (Tropsha et al. <a href="#CR24">2003</a>).</p> <p id="Par17">R<sup>2</sup> rises as the number of descriptors increase. Hence R<sup>2</sup> does not assure the effectiveness of the equations. R<sup>2</sup><sub>adj</sub> is used to reconfirm the strength and effectiveness of the equation. </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M9">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\mathrm{R}}_{\mathrm{adj}}^2=\frac{R^2-P\left(n-1\right)}{n-p+1} $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mtext_UNKNOWN">
adj</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
P</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par18">Where p and n are the descriptors from the equation and calibration set. Validation coefficient test (Q<sup>2</sup><sub>cv</sub>) was used in assessing the robustness of the model and prediction power of the derivatives, itâ€™s given as: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M11">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\mathrm{Q}}_{\mathrm{cv}}^2=1-\left[\frac{\sum {\left({Y}_{pred}-{Y}_{exp}\right)}^2}{\sum {\left({Y}_{exp}-{Y}_{mint\mathrm{r} aining}\right)}^2}\right] $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div> <div class="mtext_UNKNOWN">
cv</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mfenced_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mtext_UNKNOWN">
pred</div> </div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
exp</div> </div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mi_UNKNOWN">
exp</div> </div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Y</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
mint</div> <div class="mi_UNKNOWN">
r</div> <div class="mtext_UNKNOWN">
aining</div> </div> </div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par19">Y<sub>mintraining</sub> Y<sub>exp</sub>, and Y<sub>pred</sub> equals to the mean activities (pIC<sub>50</sub>) of calibration set, bio-activities (IC<sub>50</sub>) and calculated of the calibration set (Brandon and Orr <a href="#CR7">2015</a>).</p> </div> <div class="externalmodelvalidation" title="sec"> <div class="title" tagx="title" title="title">
External model validation</div> <div class="meaneffect" title="sec"> <div class="title" tagx="title" title="title">
Mean effect</div> <p id="Par20">The mean effect shows the descriptors or model parameters that influence the generated equation.</p> <p id="Par21">The symbols on the model parameters show the various impact of each parameter in the overall derived equation, either an increase or decrease of the model parameter. Thus itâ€™s expressed as; </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M13">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ Mean\ effect=\frac{\beta_j{\sum}_i^n{D}_j}{\sum m\left({\beta}_j{\sum}_i^n{D}_j\right)} $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mtext_UNKNOWN">
Mean effect</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="msubsup_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mi_UNKNOWN">
m</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="msubsup_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mi_UNKNOWN">
j</div> </div> </div> </div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par22">Where <i>m</i> equals the model parameters, <i>B</i><sub><i>j</i></sub> equals to descriptors coefficient <i>j</i>, <i>n</i> equals to the prediction set molecules and <i>D</i><sub><i>j</i></sub> is the matrix value of the model parameter in the prediction set (Minovski et al. <a href="#CR19">2013</a>).</p> </div> <div class="varianceinflationfactor(vif)" title="sec"> <div class="title" tagx="title" title="title">
Variance inflation factor (VIF)</div> <p id="Par23">The VIF takes into account the amount of co-linearity amongst the descriptors in an equation. It is calculated as </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M15">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ VIF=\frac{1}{\left(1-{R}^2\right)} $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
VIF</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par24">R<sup>2</sup> is the correlation coefficient. (Myers <a href="#CR20">1990</a>).</p> <p id="Par25">The greater the value, the bigger the link amongst the model parameters. The VIF values of less than 10 show the equation is stable while the values above 10 indicate the equation is not effective and cannot be used.</p> </div> <div class="applicabilitydomain" title="sec"> <div class="title" tagx="title" title="title">
Applicability domain</div> <p id="Par26">The applicability domain approach is aimed at estimating independently, the reliability of every generated equation (Eriksson et al. <a href="#CR10">2003</a>). Model validation should be within the training domain and the compounds need to be assessed as fitting within the domain to ascertain the model. The applicability domain is evaluated by the leverage values for each compound. Leverage defines the applicability domain of the built equation (Veerasamy et al. <a href="#CR25">2011</a>). It is given as; </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M17">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {H}_i={x}_i\ {\left({X}^TX\right)}^{-k}\ {x_i}^T\ \left(i={K}^{\dots },P\right) $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
H</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mi_UNKNOWN">
X</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="msup_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mspace_UNKNOWN" /> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> <div class="mo_UNKNOWN">
â€¦</div> </div> </div> <div class="mi_UNKNOWN">
P</div> </div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par27">Where <i>X</i><sup><i>T</i></sup> is the matrix transpose of <i>X</i> used in constructing the equation, <i>X</i><sub><i>i</i></sub> is the matrix of calibration set of <i>I</i> and <i>X</i> is the <i>n x k</i> matrix of train set descriptors. (<i>d*</i>) is the warning leverage, <i>d*</i> searches for outliers. It is shown as; </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M19">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \mathrm{d}\ast =\frac{3\left(k+1\right)}{n} $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> <div class="mi_UNKNOWN">
n</div> </div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par28"><i>k</i> equals to the total model parameters and <i>n</i> equals to the calibration compounds. Williamâ€™s plot (A plot of standardized values vs. leverages) of the calibration and validation compounds. Molecules found in the warning leverages within the graph are the calculated molecules.</p> </div> </div> <div class="qualityassurancemodelgenerated" title="sec"> <div class="title" tagx="title" title="title">
Quality assurance model generated</div> <p id="Par29">Table <a href="#Tab2">2</a> shows the least required values in assessing the mathematical equation (Ibrahim et al. <a href="#CR14">2018</a>). The table parameters were used in conforming to the effectiveness and prediction power of the derived equations. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>Recommended values for evaluating QSAR equations</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Character</th> <th>Name</th> <th>Value</th> </tr> </thead> <tbody> <tr> <td>R<sup>2</sup></td> <td>Coefficient of determination</td> <td>â‰¥0.6</td> </tr> <tr> <td>P<sub>(95%)</sub></td> <td>Confidence interval at 95% confidence level</td> <td>&amp;lt;0.05</td> </tr> <tr> <td>Q<sup>2</sup></td> <td>Squared cross-validation coefficient</td> <td>â‰¥0.5</td> </tr> <tr> <td>R<sup>2</sup>-Q<sup>2</sup></td> <td>Difference between R<sup>2</sup> and Q<sup>2</sup></td> <td>&amp;lt;0.3</td> </tr> <tr> <td>N<sub>test set</sub></td> <td>Least number of the test set</td> <td>â‰¥5</td> </tr> <tr> <td>R<sup>2</sup><sub>ext</sub></td> <td>Coefficient of determination of external validation</td> <td>â‰¥0.5</td> </tr> </tbody> </table>  <p /> </div> </div> <div class="moleculardockingstudies" title="sec"> <div class="title" tagx="title" title="title">
Molecular docking studies</div> <p id="Par30">Five compounds with high pIC<sub>50</sub> underwent molecular docking studies with the receptor Polo-like kinase 1(PKL1) in complex with B16727. The receptor used was obtained from Protein Data Bank (Code: 3FC2) and was set using Discovery studio software, the ligand (compounds) were also converted to PBD format as shown in Fig.Â <a href="#Fig4">4</a>. Autodock Vina of Pyrx software was employed in calculating the binding affinity of the ligand and receptor (Abdulfatai et al. <a href="#CR1">2018</a>).</p> </div> <div class="pharmacokinetics(drug-likeness)" title="sec"> <div class="title" tagx="title" title="title">
Pharmacokinetics (drug-likeness)</div> <p id="Par31">SwissADME; an online tool, used in investigating the ADME property physicochemical, pharmacokinetic, and medicinal chemistry responsiveness of smalls compounds (Daina et al. <a href="#CR8">2017</a>) was employed in assessing the Pharmacokinetic parameters of the new structures.</p> <p id="Par32">Again, the designed compounds were checked for their adaptability with Lipinskiâ€™s rule of five (Hou et al. <a href="#CR13">2019</a>), well-used criteria to comprehend if a compound can be taken orally or not, such as molecular weight (MW)â€‰â‰¤â€‰500, octanol/water partition coefficient (AlogP)â€‰â‰¤â€‰5, number of hydrogen bond donors (HBDs)â€‰â‰¤â€‰5 and number of hydrogen bond acceptors (HBAs)â€‰â‰¤â€‰10.6. According to the rule of five, a compound cannot be taken orally if it does not meet up to two or rules out of the rules of five (Guan et al. <a href="#CR12">2019</a>).</p> </div> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and discussion</div> <div class="qsarof2-(4-fluorophenyl)imidazol-5-onesderivatives" title="sec"> <div class="title" tagx="title" title="title">
QSAR of 2-(4-fluorophenyl) imidazol-5-ones derivatives</div> <p id="Par33">QSAR examination was used to verify the relationship of 2-(4-fluorophenyl) imidazol-5-ones derivatives with its anti-proliferative activities. Using the Genetic Function Approximation (GFA) method, four QSAR equations were built to predict the anti-proliferative activities of imidazole derivatives. From both internal and external validation parameters, model number 1 passed with correlation coefficient squared (R<sup>2</sup>) of 0.6981, correlation coefficient adjusted squared (R<sup>2</sup><sub>adj</sub>) of 0.6433, cross-validation coefficient (Q<sup>2</sup>) of 0.5460 and external validation (R<sup>2</sup><sub>pred</sub>) of 0.5390. TablesÂ <a href="#Tab3">3</a> and <a href="#Tab4">4</a> shows how the external validation of model 1 was calculated using the validation set (test set) and model descriptors. </p><ul> <li> <p id="Par34">Model 1</p> </li> </ul> <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> <p>External validation of equation 1</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Name</th> <th>pIC<sub>50</sub></th> <th>MATS4e</th> <th>GATS5e</th> <th>SpMax4_Bhs</th> <th>RDF150u</th> <th>Y<sub>pred</sub></th> <th>Y<sub>pred</sub>-Y<sub>obs</sub></th> </tr> </thead> <tbody> <tr> <td>23</td> <td>5.1203</td> <td>âˆ’0.1068</td> <td>0.8685</td> <td>3.9563</td> <td>5.5912</td> <td>4.9935</td> <td>0.1268</td> </tr> <tr> <td>24</td> <td>5.1062</td> <td>âˆ’0.1214</td> <td>0.8301</td> <td>3.9563</td> <td>4.7605</td> <td>4.9494</td> <td>0.1568</td> </tr> <tr> <td>27</td> <td>5.2027</td> <td>âˆ’0.1666</td> <td>0.9511</td> <td>3.8468</td> <td>6.6068</td> <td>4.7426</td> <td>0.4601</td> </tr> <tr> <td>33</td> <td>4.5882</td> <td>âˆ’0.1687</td> <td>0.8792</td> <td>3.8317</td> <td>0.8057</td> <td>4.4399</td> <td>0.1483</td> </tr> <tr> <td>35</td> <td>4.2447</td> <td>âˆ’0.1646</td> <td>0.9304</td> <td>3.8245</td> <td>0.0040</td> <td>4.2702</td> <td>âˆ’0.0255</td> </tr> <tr> <td>36</td> <td>4.7537</td> <td>âˆ’0.1093</td> <td>0.8139</td> <td>3.9606</td> <td>1.24E-28</td> <td>4.6337</td> <td>0.1199</td> </tr> <tr> <td>37</td> <td>4.5769</td> <td>âˆ’0.1197</td> <td>0.8164</td> <td>3.9607</td> <td>0.3669</td> <td>4.6078</td> <td>âˆ’0.0309</td> </tr> <tr> <td>38</td> <td>4.4583</td> <td>âˆ’0.1590</td> <td>0.9474</td> <td>3.7874</td> <td>1.90E-17</td> <td>4.3050</td> <td>0.1533</td> </tr> <tr> <td>39</td> <td>4.6570</td> <td>âˆ’0.1725</td> <td>0.9504</td> <td>3.7880</td> <td>1.6951</td> <td>4.3798</td> <td>0.2772</td> </tr> <tr> <td>6</td> <td>4.6407</td> <td>âˆ’0.1135</td> <td>0.7806</td> <td>3.9553</td> <td>4.89E-47</td> <td>4.7065</td> <td>âˆ’0.0658</td> </tr> <tr> <td>7</td> <td>4.0286</td> <td>âˆ’0.1231</td> <td>0.8095</td> <td>3.9553</td> <td>2.53E-33</td> <td>4.5857</td> <td>âˆ’0.5571</td> </tr> <tr> <td>8</td> <td>4.5252</td> <td>âˆ’0.1680</td> <td>0.8066</td> <td>3.9661</td> <td>0.9960</td> <td>4.4381</td> <td>0.0871</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="Tab4"><span class="label" tagx="label" title="label">Table 4</span> <p>Calculation on external validation of equation 1 (continued)</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>(Y<sub>pred</sub>-Y<sub>obs</sub>)<sup>2</sup></th> <th>Y<sub>mintrn</sub></th> <th>(Y<sub>mintrn</sub>-Y<sub>obs</sub>)</th> <th>(Y<sub>mintrn</sub>-Y<sub>obs</sub>)<sup>2</sup></th> </tr> </thead> <tbody> <tr> <td>0.0161</td> <td>4.5364</td> <td>0.5839</td> <td>0.3409</td> </tr> <tr> <td>0.0246</td> <td>4.5364</td> <td>0.5698</td> <td>0.3247</td> </tr> <tr> <td>0.2117</td> <td>4.5364</td> <td>0.6663</td> <td>0.4430</td> </tr> <tr> <td>0.0210</td> <td>4.5364</td> <td>0.0518</td> <td>0.0027</td> </tr> <tr> <td>0.0007</td> <td>4.5364</td> <td>âˆ’0.2917</td> <td>0.0851</td> </tr> <tr> <td>0.0144</td> <td>4.5364</td> <td>0.2173</td> <td>0.0472</td> </tr> <tr> <td>0.0009</td> <td>4.5364</td> <td>0.0405</td> <td>0.0016</td> </tr> <tr> <td>0.0235</td> <td>4.5364</td> <td>âˆ’0.0781</td> <td>0.0061</td> </tr> <tr> <td>0.0769</td> <td>4.5364</td> <td>0.1206</td> <td>0.0145</td> </tr> <tr> <td>0.0043</td> <td>4.5364</td> <td>0.1043</td> <td>0.0109</td> </tr> <tr> <td>0.3104</td> <td>4.5364</td> <td>âˆ’0.5078</td> <td>0.2579</td> </tr> <tr> <td>0.0076</td> <td>4.5364</td> <td>âˆ’0.0112</td> <td>0.0001</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>âˆ‘(Y<sub>ob</sub> âˆ’ Y<sub>pred</sub>)<sup>2</sup>â€‰=â€‰0.713008 âˆ‘(Y<sub>obs</sub> âˆ’ Y Â®<sub>train</sub>)<sup>2</sup>â€‰=â€‰1.535707âˆ´ R<sup>2</sup><i>test</i> = 1â€‰âˆ’â€‰(0.7130/ 1.5357)â€‰=â€‰0.5357</p> </div> </div> <p /> <p id="Par35"> </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M21">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\mathrm{pIC}}_{50}=4.888518176\ast \mathrm{MATS}4\mathrm{e}-2.570261057\ast \mathrm{GATS}5\mathrm{e}-1.514002889\ast \mathrm{SpMax}4\_\mathrm{Bhs}+0.086137333\ast \mathrm{RDF}150\mathrm{u}+13.256220911 $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
pIC</div> <div class="mn_UNKNOWN">
50</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
4.888518176</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
MATS</div> <div class="mn_UNKNOWN">
4</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
2.570261057</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
GATS</div> <div class="mn_UNKNOWN">
5</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1.514002889</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
SpMax</div> <div class="mn_UNKNOWN">
4</div> <div class="mo_UNKNOWN">
_</div> <div class="mi_UNKNOWN">
Bhs</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0.086137333</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mi_UNKNOWN">
RDF</div> <div class="mn_UNKNOWN">
150</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
13.256220911</div></div><div class="graphic" position="anchor" /> </div> </div> <ul> <li> <p id="Par36">Model 2</p> </li> </ul> <p /> <p id="Par37"> </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M23">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\mathrm{pIC}}_{50}=-0.018183153\ast \mathrm{ALogp}2+4.669978294\ast \mathrm{MATS}4\mathrm{e}-1.245497827\ast \mathrm{GATS}5\mathrm{e}+0.065416696\ast \mathrm{RDF}150\mathrm{u}+6.326485838 $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
pIC</div> <div class="mn_UNKNOWN">
50</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
0.018183153</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
ALogp2</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
4.669978294</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
MATS</div> <div class="mn_UNKNOWN">
4</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1.245497827</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
GATS</div> <div class="mn_UNKNOWN">
5</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0.065416696</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mi_UNKNOWN">
RDF</div> <div class="mn_UNKNOWN">
150</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
6.326485838</div></div><div class="graphic" position="anchor" /> </div> </div> <ul> <li> <p id="Par38">Model 3</p> </li> </ul> <p /> <p id="Par39"> </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M25">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\mathrm{pIC}}_{50}=-0.021417358\ast \mathrm{ALogp}2+2.673511925\ast \mathrm{MATS}4\mathrm{e}-1.523695399\ast \mathrm{GATS}4\mathrm{e}+0.061648777\ast \mathrm{RDF}150\mathrm{u}+6.593640317 $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
pIC</div> <div class="mn_UNKNOWN">
50</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
0.021417358</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
ALogp2</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
2.673511925</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
MATS</div> <div class="mn_UNKNOWN">
4</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1.523695399</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
GATS</div> <div class="mn_UNKNOWN">
4</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0.061648777</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mi_UNKNOWN">
RDF</div> <div class="mn_UNKNOWN">
150</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
6.593640317</div></div><div class="graphic" position="anchor" /> </div> </div> <ul> <li> <p id="Par40">Model 4</p> </li> </ul> <p /> <p id="Par41"> </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M27">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\mathrm{pIC}}_{50}=4.242442241\ast \mathrm{MATS}4\mathrm{e}-1.572590496\ast \mathrm{GATS}5\mathrm{e}+0.065174802\ast \mathrm{RDF}150\mathrm{u}-0.392069439\ast \mathrm{P}1\mathrm{m}+6.739118905 $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
pIC</div> <div class="mn_UNKNOWN">
50</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
4.242442241</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
MATS</div> <div class="mn_UNKNOWN">
4</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1.572590496</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mtext_UNKNOWN">
GATS</div> <div class="mn_UNKNOWN">
5</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0.065174802</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mi_UNKNOWN">
RDF</div> <div class="mn_UNKNOWN">
150</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
0.392069439</div> <div class="mo_UNKNOWN">
âˆ—</div> <div class="mi_UNKNOWN">
P</div> <div class="mn_UNKNOWN">
1</div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
6.739118905</div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par42">The biological, calculated and the residual values of 2-(4-fluorophenyl) imidazol-5-ones compounds are seen in Table <a href="#Tab5">5</a>. <div class="table-wrap_UNKNOWN" id="Tab5"><span class="label" tagx="label" title="label">Table 5</span> </div></p><p>The bio-activities (pIC<sub>50</sub>), prediction inhibition and residual of model 1</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Name</th> <th>pIC<sub>50</sub></th> <th>Predicted IC<sub>50</sub></th> <th>Residual</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>4.3202</td> <td>4.3219</td> <td>âˆ’0.0017</td> </tr> <tr> <td>2</td> <td>4.9846</td> <td>4.7403</td> <td>0.2443</td> </tr> <tr> <td>3</td> <td>4.8011</td> <td>4.6737</td> <td>0.1274</td> </tr> <tr> <td>4</td> <td>5.0262</td> <td>5.0411</td> <td>0.0209</td> </tr> <tr> <td>5</td> <td>4.0206</td> <td>4.2734</td> <td>âˆ’0.2529</td> </tr> <tr> <td>6*</td> <td>4.6407</td> <td>4.7065</td> <td>âˆ’0.0658</td> </tr> <tr> <td>7*</td> <td>4.0286</td> <td>4.5857</td> <td>âˆ’0.5571</td> </tr> <tr> <td>8*</td> <td>4.5252</td> <td>4.4381</td> <td>0.0871</td> </tr> <tr> <td>9</td> <td>4.0543</td> <td>4.1766</td> <td>âˆ’0.1223</td> </tr> <tr> <td>10</td> <td>4.0434</td> <td>4.1436</td> <td>âˆ’0.1002</td> </tr> <tr> <td>11</td> <td>4.2142</td> <td>4.5352</td> <td>âˆ’0.3210</td> </tr> <tr> <td>12</td> <td>4.4309</td> <td>4.0666</td> <td>0.3643</td> </tr> <tr> <td>13</td> <td>4.4865</td> <td>4.4213</td> <td>0.0652</td> </tr> <tr> <td>14</td> <td>4.4026</td> <td>4.2774</td> <td>0.1252</td> </tr> <tr> <td>15</td> <td>4.0829</td> <td>4.1101</td> <td>âˆ’0.0272</td> </tr> <tr> <td>16</td> <td>4.9000</td> <td>4.8921</td> <td>0.0079</td> </tr> <tr> <td>17</td> <td>4.8133</td> <td>4.8557</td> <td>âˆ’0.0424</td> </tr> <tr> <td>18</td> <td>4.7247</td> <td>4.9270</td> <td>âˆ’0.2023</td> </tr> <tr> <td>19</td> <td>4.5607</td> <td>4.3625</td> <td>0.1982</td> </tr> <tr> <td>20</td> <td>4.3428</td> <td>4.1947</td> <td>0.1481</td> </tr> <tr> <td>21</td> <td>4.1388</td> <td>4.4897</td> <td>âˆ’0.3509</td> </tr> <tr> <td>22</td> <td>4.9311</td> <td>4.9916</td> <td>âˆ’0.0605</td> </tr> <tr> <td>23*</td> <td>5.1203</td> <td>4.9935</td> <td>0.1268</td> </tr> <tr> <td>24*</td> <td>5.1062</td> <td>4.9494</td> <td>0.1568</td> </tr> <tr> <td>25</td> <td>4.6955</td> <td>4.7353</td> <td>âˆ’0.0398</td> </tr> <tr> <td>26</td> <td>5.0132</td> <td>4.6610</td> <td>0.3522</td> </tr> <tr> <td>27*</td> <td>5.2027</td> <td>4.7426</td> <td>0.4601</td> </tr> <tr> <td>28</td> <td>5.0232</td> <td>4.7887</td> <td>0.2345</td> </tr> <tr> <td>29</td> <td>4.6269</td> <td>4.7330</td> <td>âˆ’0.1061</td> </tr> <tr> <td>30</td> <td>4.8645</td> <td>4.7518</td> <td>0.1127</td> </tr> <tr> <td>31</td> <td>4.3002</td> <td>4.3924</td> <td>âˆ’0.0922</td> </tr> <tr> <td>32</td> <td>4.0947</td> <td>4.3760</td> <td>âˆ’0.2813</td> </tr> <tr> <td>33*</td> <td>4.5882</td> <td>4.4399</td> <td>0.1483</td> </tr> <tr> <td>34</td> <td>4.5508</td> <td>4.5509</td> <td>âˆ’0.0005</td> </tr> <tr> <td>35*</td> <td>4.2447</td> <td>4.2702</td> <td>âˆ’0.0255</td> </tr> <tr> <td>36*</td> <td>4.7537</td> <td>4.6338</td> <td>0.1199</td> </tr> <tr> <td>37*</td> <td>4.5769</td> <td>4.6078</td> <td>âˆ’0.0309</td> </tr> <tr> <td>38*</td> <td>4.4583</td> <td>4.3050</td> <td>0.1533</td> </tr> <tr> <td>39</td> <td>4.6570</td> <td>4.3798</td> <td>0.2773</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>The compounds with (*) are the validation compounds while the compounds without (*) are the calibration set</p> </div>  <p /> <p id="Par43">The descriptors obtained from the mathematical model 1 were defined and classified as seen in Table <a href="#Tab6">6</a>. <div class="table-wrap_UNKNOWN" id="Tab6"><span class="label" tagx="label" title="label">Table 6</span> </div></p><p>Definition of descriptors and their classes for model 1</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Name</th> <th>Definition</th> <th>Class</th> </tr> </thead> <tbody> <tr> <td>MATS4e</td> <td>Moran autocorrelation - lag 4 / weighted by Sanderson electronegativities.</td> <td>2D</td> </tr> <tr> <td>GATS5e</td> <td>Geary autocorrelation - lag 5 / weighted by Sanderson electronegativities.</td> <td>2D</td> </tr> <tr> <td>SPMAX4_Bhs</td> <td>The largest absolute eigenvalue of Burden modified matrix - n 4 / weighted by relative I-state.</td> <td>2D</td> </tr> <tr> <td>RDF150U</td> <td>Radial distribution function-150 / unweighted.</td> <td>2D</td> </tr> </tbody> </table>  <p /> <p id="Par44">Further statistical analysis was carried out on the model parameters to find the correlation between the individual descriptors and also the impact of each descriptor in the model. The results are shown in Table <a href="#Tab7">7</a>. <div class="table-wrap_UNKNOWN" id="Tab7"><span class="label" tagx="label" title="label">Table 7</span> </div></p><p>Statistical analysis of model 1 parameters</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th /> <th>MATS4e</th> <th>GATS5e</th> <th>SpMax4_Bhs</th> <th>RDF150u</th> <th>VIF</th> <th>Mean Effect</th> </tr> </thead> <tbody> <tr> <td>MATS4e</td> <td>1</td> <td>âˆ’0.36547</td> <td>0.511519</td> <td>0.208789</td> <td>1.38903</td> <td>0.07868</td> </tr> <tr> <td>GATS5e</td> <td>âˆ’0.36547</td> <td>1</td> <td>âˆ’0.60954</td> <td>0.310998</td> <td>3.140572</td> <td>0.268682</td> </tr> <tr> <td>SpMax4_Bhs</td> <td>0.511519</td> <td>âˆ’0.60954</td> <td>1</td> <td>0.338777</td> <td>3.267216</td> <td>0.669708</td> </tr> <tr> <td>RDF150u</td> <td>0.208789</td> <td>0.310998</td> <td>0.338777</td> <td>1</td> <td>2.220033</td> <td>âˆ’0.01707</td> </tr> </tbody> </table>  <p /> <p id="Par45">A graph of the calculated activities against the biological activities was drawn to show the relationship between derivatives as seen in Fig. <a href="#Fig1">1</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Plot of calculated activity versus inhibition concentration (biological activities)</p>   </div> <p /> <p id="Par46">Figure <a href="#Fig2">2</a> shows a plot of standardized residual values versus the experimental activities of the derivatives. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Plot of standardized residual against bioactivities (Experimental activity).</p>   </div> <p /> <p id="Par47">A graph of standardized residual against the leverages was plotted to show the derivatives that fell within the applicability domain as seen in Figs. <a href="#Fig3">3</a> and <a href="#Fig4">4</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>The Williamâ€™s plot</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>3D representation of prepared ligand and receptor</p>   </div> <p /> </div> <div class="moleculardockinganalysis" title="sec"> <div class="title" tagx="title" title="title">
Molecular docking analysis</div> <p id="Par48">A summary of the relationship between some 2-(4-fluorophenyl) imidazole-5-ones derivatives and the receptor was shown in Table <a href="#Tab8">8</a>. The pictorial analysis of docked compounds was accessed by assessing the hydrogen bond interactions, hydrogen bond length, and hydrophobic interactions. Both 2D representations of the binding pose of compounds 24 and 27 to the active pocket of the protein target are shown in Figs.Â <a href="#Fig5">5</a> and <a href="#Fig6">6</a> respectively. <div class="table-wrap_UNKNOWN" id="Tab8"><span class="label" tagx="label" title="label">Table 8</span> </div></p><p>molecular docking interaction in some complexes</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Complex</th> <th>Binding affinity (kcal/mol)</th> <th>Amino acid</th> <th>Bond type</th> <th>Interaction</th> <th>Distance(A<sup>0</sup>)</th> </tr> </thead> <tbody> <tr> <td>27</td> <td>âˆ’9.1</td> <td>LYS82</td> <td>Hydrogen Bond</td> <td>Hydrogen Bond Interaction</td> <td>2.76853</td> </tr> <tr> <td /> <td /> <td>LEU59</td> <td>Hydrogen Bond</td> <td>Hydrogen Bond Interaction</td> <td>2.08894</td> </tr> <tr> <td /> <td /> <td>LYS82</td> <td>Electrostatic</td> <td>Pi-Cation</td> <td>4.57869</td> </tr> <tr> <td /> <td /> <td>GLU101</td> <td>Electrostatic</td> <td>Pi-Anion</td> <td>4.83015</td> </tr> <tr> <td /> <td /> <td>ASP194</td> <td>Electrostatic</td> <td>Pi-Anion</td> <td>3.54145</td> </tr> <tr> <td /> <td /> <td>PHE64</td> <td> <p>Hydrogen</p> <p>Bond</p> </td> <td>Pi-Donor Hydrogen Bond</td> <td>2.84892</td> </tr> <tr> <td /> <td /> <td>LEU59</td> <td>Hydrophobic</td> <td>Pi-Sigma</td> <td>3.8345</td> </tr> <tr> <td /> <td /> <td>LEU59</td> <td>Hydrophobic</td> <td>Pi-Sigma</td> <td>3.93488</td> </tr> <tr> <td /> <td /> <td>GLY62</td> <td>Hydrophobic</td> <td>Amide-Pi Stacked</td> <td>3.80969</td> </tr> <tr> <td /> <td /> <td>LEU59</td> <td>Hydrophobic</td> <td>Alkyl</td> <td>4.89515</td> </tr> <tr> <td /> <td /> <td>LEU132</td> <td>Hydrophobic</td> <td>Alkyl</td> <td>4.61692</td> </tr> <tr> <td /> <td /> <td>ARG136</td> <td>Hydrophobic</td> <td>Pi-Alkyl</td> <td>4.58314</td> </tr> <tr> <td /> <td /> <td>ALA65</td> <td>Hydrophobic</td> <td>Pi-Alkyl</td> <td>4.75156</td> </tr> <tr> <td>24</td> <td>âˆ’8.8</td> <td>ARG136</td> <td>Hydrogen Bond</td> <td>Conventional Hydrogen Bond</td> <td>2.60978</td> </tr> <tr> <td /> <td /> <td>ASP194</td> <td>Hydrogen Bond</td> <td>Conventional Hydrogen Bond</td> <td>1.99045</td> </tr> <tr> <td /> <td /> <td>LYS82</td> <td>Electrostatic</td> <td>Pi-Cation</td> <td>3.69094</td> </tr> <tr> <td /> <td /> <td>GLU101</td> <td>Electrostatic</td> <td>Pi-Anion</td> <td>4.16479</td> </tr> <tr> <td /> <td /> <td>CYS67</td> <td>Others</td> <td>Pi-Sulfur</td> <td>4.37876</td> </tr> <tr> <td /> <td /> <td>PHE64</td> <td>Hydrophobic</td> <td>Pi-Pi T-shaped</td> <td>5.25545</td> </tr> <tr> <td /> <td /> <td>LEU89</td> <td>Hydrophobic</td> <td>Alkyl</td> <td>4.82227</td> </tr> <tr> <td /> <td /> <td>MET98</td> <td>Hydrophobic</td> <td>Alkyl</td> <td>4.1282</td> </tr> <tr> <td /> <td /> <td>PHE64</td> <td>Hydrophobic</td> <td>Pi-Alkyl</td> <td>4.85386</td> </tr> <tr> <td /> <td /> <td>CYS67</td> <td>Hydrophobic</td> <td>Pi-Alkyl</td> <td>5.47383</td> </tr> <tr> <td /> <td /> <td>ALA65</td> <td>Hydrophobic</td> <td>Pi-Alkyl</td> <td>5.17103</td> </tr> <tr> <td /> <td /> <td>MET98</td> <td>Hydrophobic</td> <td>Pi-Alkyl</td> <td>5.26908</td> </tr> <tr> <td /> <td /> <td>ALA80</td> <td>Hydrophobic</td> <td>Pi-Alkyl</td> <td>5.10978</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>2D and 3D representation of complex 27</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>3D and 2D representation of complex 24</p>   </div> <p /> </div> <div class="ligandbasedrugdesign" title="sec"> <div class="title" tagx="title" title="title">
Ligand Base drug design</div> <p id="Par49">Ligand based approach was used in designing 18 new imidazole derivative compounds with higher calculated activities than that of the template compounds as shown in Table <a href="#Tab9">9</a>. <div class="table-wrap_UNKNOWN" id="Tab9"><span class="label" tagx="label" title="label">Table 9</span> </div></p><p>Newly designed imidazole derivative compounds with their new predicted activities (pIC<sub>50</sub>)</p>  <div class="graphic" position="anchor" id="d30e3406" /> <div class="graphic" position="anchor" id="d30e3407" /> <div class="graphic" position="anchor" id="d30e3408" />  <p /> </div> <div class="pharmacokineticsofdesigned2-(4-fluorophenyl)imidazole-5-onescompounds" title="sec"> <div class="title" tagx="title" title="title">
Pharmacokinetics of designed 2-(4-fluorophenyl) imidazole-5-ones compounds</div> <p id="Par50">The newly designed compounds were further explored to ascertain their drug-friendliness. The pharmacokinetic analysis of the new compounds are shown in Table <a href="#Tab10">10</a>, all the compounds pass the Lipinski rule of five test. The bioavailability radar of molecules 11, 13, and 17 are shown in Fig.Â <a href="#Fig7">7</a>. <div class="table-wrap_UNKNOWN" id="Tab10"><span class="label" tagx="label" title="label">Table 10</span> </div></p><p>Pharmacokinetics of designed 2-(4-fluorophenyl) imidazole-5-ones compounds</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>S/No.</th> <th>MW (mg/mol)</th> <th>nAH</th> <th>nRB</th> <th>HBA</th> <th>HBD</th> <th>MR</th> <th>TPSA (Ã…<sup>2</sup>)</th> <th>iLOGP</th> <th>BBB</th> <th>PAIN</th> <th>Brenk</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>415.24</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>114.8</td> <td>45.81</td> <td>3.81</td> <td>Yes</td> <td>1</td> <td>2</td> </tr> <tr> <td>2</td> <td>415.24</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>114.8</td> <td>3.72</td> <td>3.72</td> <td>Yes</td> <td>1</td> <td>2</td> </tr> <tr> <td>3</td> <td>504.15</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>120.18</td> <td>45.81</td> <td>3.93</td> <td>Yes</td> <td>1</td> <td>1</td> </tr> <tr> <td>4</td> <td>382.34</td> <td>17</td> <td>3</td> <td>6</td> <td>0</td> <td>104.69</td> <td>45.81</td> <td>3.52</td> <td>Yes</td> <td>1</td> <td>1</td> </tr> <tr> <td>5</td> <td>415.24</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>114.80</td> <td>45.81</td> <td>3.70</td> <td>Yes</td> <td>1</td> <td>1</td> </tr> <tr> <td>6</td> <td>415.24</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>114.80</td> <td>45.81</td> <td>3.72</td> <td>Yes</td> <td>1</td> <td>2</td> </tr> <tr> <td>7</td> <td>504.15</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>120.18</td> <td>45.81</td> <td>3.80</td> <td>Yes</td> <td>1</td> <td>2</td> </tr> <tr> <td>8</td> <td>490.12</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>115.21</td> <td>45.81</td> <td>3.51</td> <td>Yes</td> <td>1</td> <td>1</td> </tr> <tr> <td>9</td> <td>401.22</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>109.83</td> <td>45.81</td> <td>3.49</td> <td>Yes</td> <td>1</td> <td>2</td> </tr> <tr> <td>10</td> <td>490.12</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>115.21</td> <td>45.81</td> <td>3.51</td> <td>Yes</td> <td>1</td> <td>1</td> </tr> <tr> <td>11</td> <td>490.12</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>115.21</td> <td>45.81</td> <td>3.74</td> <td>Yes</td> <td>1</td> <td>2</td> </tr> <tr> <td>12</td> <td>401.22</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>109.83</td> <td>45.81</td> <td>3.35</td> <td>Yes</td> <td>1</td> <td>1</td> </tr> <tr> <td>13</td> <td>490.12</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>115.21</td> <td>45.81</td> <td>3.70</td> <td>Yes</td> <td>1</td> <td>2</td> </tr> <tr> <td>14</td> <td>401.22</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>109.83</td> <td>45.81</td> <td>3.49</td> <td>Yes</td> <td>1</td> <td>2</td> </tr> <tr> <td>15</td> <td>472.25</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>117.49</td> <td>45.81</td> <td>3.67</td> <td>Yes</td> <td>1</td> <td>2</td> </tr> <tr> <td>16</td> <td>425.25</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>112.48</td> <td>45.81</td> <td>3.76</td> <td>Yes</td> <td>1</td> <td>1</td> </tr> <tr> <td>17</td> <td>380.80</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>109.79</td> <td>45.81</td> <td>3.72</td> <td>Yes</td> <td>1</td> <td>1</td> </tr> <tr> <td>18</td> <td>366.77</td> <td>17</td> <td>3</td> <td>4</td> <td>0</td> <td>104.82</td> <td>45.81</td> <td>3.29</td> <td>Yes</td> <td>1</td> <td>1</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><i>MW</i> Molecular weight (&amp;lt;500Â mg/mol), <i>nAH</i> number of aromatic heavy atoms, <i>nRB</i> rotatable bonds, <i>HBA</i> Hydrogen bond acceptors, <i>HBD</i> Hydrogen bond donors, <i>MR</i> molecular refractivity, <i>TPSA</i> Topological polar surface area, <i>BBB</i> blood-brain barrier</p> </div>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <p>The bioavailability radar for molecules 11, 13, and 17</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="qsarof2-(4-fluorophenyl)imidazol-5-onesderivatives" title="sec"> <div class="title" tagx="title" title="title">
QSAR of 2-(4-fluorophenyl) imidazol-5-ones derivatives</div> <p id="Par51">The results obtained from the QSAR analysis showed that both internal and external validation of the model were in agreement with the minimum proposed values used in assessing the equation as seen in Table <a href="#Tab2">2</a> above. The external validation of model 1 was achieved using the model parameters from the validation compounds as seen in Tables <a href="#Tab3">3</a> and <a href="#Tab4">4</a>. The effectiveness of the equation was measured by the reliability of the validation compounds and calculated pIC<sub>50</sub> of the calibration compounds. The biological, calculated and the residual values of 2-(4-fluorophenyl) imidazol-5-ones compounds are seen in Table <a href="#Tab5">5</a>. The low residual values are obtained from the difference between the biological and calculated activities, showing the high prediction power of eq. 1. Both internal and external validation conforms eq. 1 to be greatly effective, strong, and extremely predictive. Table <a href="#Tab6">6</a> shows the definition of the model 1 parameters. The mean effect obtained from the model parameters, shows (GATS5e, MATS4e, and SpMax4_Bhs) carries a positive coefficient showing that an increase in those factors would increase the bioactivities of the derivatives while (RDF150u) carrying a negative coefficient indicates a decrease in the descriptor would also increase the experimental activities of 2-(4-fluorophenyl) imidazol-5-ones derivative compounds. The statistical analysis shows that there is no much collinearity amongst the model parameters ensuring that the equation is highly robust as seen in Table <a href="#Tab7">7</a>.</p> <p id="Par52">The graph of calculated activities (pIC<sub>50</sub>) against biological activities (IC<sub>50</sub>) as shown in Fig. <a href="#Fig1">1</a> indicates the pIC<sub>50</sub> has been in good agreement with the biological activities as seen in Table <a href="#Tab3">3</a>. Figure <a href="#Fig2">2</a> shows the values of both calibration and validation compounds spread on both sides of the graph, showing no systematic errors between the standardized residual versus bio-activities (Experimental activity) (Jalali-Heravi and Kyani 2004). Fig. <a href="#Fig3">3</a> shows Williamâ€™s graph (standardized residuals against leverages), indicating that all the molecules fell in the warning leverage area, calculated to be (hâ€‰=â€‰0.56).</p> </div> <div class="moleculardockinganalysis" title="sec"> <div class="title" tagx="title" title="title">
Molecular docking analysis</div> <p id="Par53">The docking analysis on compounds of 2-(4-fluorophenyl) imidazole-5-ones derivatives with the protein target, Polo-like kinase 1(PKL1) in complex with B16727 was performed. 5 compounds with high pIC<sub>50</sub> were chosen for these studies, amongst the 5, compound 24 and 27 had the highest docking score of âˆ’8.8 andâ€‰âˆ’â€‰9.1Â kcal/mol as seen in Table <a href="#Tab8">8</a>.</p> <p id="Par54">Compound 27 showed backbone conventional hydrogen bonding interaction between -NH group with LYS82 (2.7685A<sup>0</sup>) and carbonyl group with LEU59 (2.0889A<sup>0</sup>). Three amino acids showed electrostatic interaction at LYS82 (4.57869A<sup>0</sup>) which is a pi-orbital cation interaction then GLU101 (4.83015A<sup>0</sup>), ASP194 (3.54145A<sup>0</sup>) are pi-orbital anion interaction. Furthermore, the compound formed a hydrophobic bond with three amino acids of LEU59 at distance 3.8345A<sup>0</sup>, 3.93488A<sup>0</sup>, and 4.89515A<sup>0</sup>, then GLY62 (3.80969A<sup>0</sup>), ARG136 (4.58314A<sup>0</sup>), and ALA65 (4.75156A<sup>0</sup>).</p> <p id="Par55">Again in compound 24, ARG136 and ASP 139 gave covalent hydrogen interaction at a distance of 2.60978A<sup>0</sup> and 1.99045A<sup>0</sup>. Two electrostatic bonds occur with the compound at LYS82 (4.16479A<sup>0</sup>) and ASP194 (1.99045A<sup>0</sup>). It formed a hydrophobic bond with PHE64 (5.25545A<sup>0</sup>), MET98 (4.1282A<sup>0</sup>) also, the pi-orbital containing delocalized electrons in the benzene ring interact with the alkyl groups of PHE64 (4.85386A<sup>0</sup>), CYS67 (5.47383A<sup>0</sup>), ALA65 (5.17103A<sup>0</sup>), MET98 (5.26908A<sup>0</sup>), ALA80 (5.26908A<sup>0</sup>) three amino acids of PRO384 (5.1107A<sup>0</sup>, 4.7845A<sup>0</sup> and 4.7531A<sup>0</sup>) to form a hydrophobic bond. Both the hydrogen bond and the hydrophobic interactions in the complexes showed that ligand 24 and 27 of 2-(4-fluorophenyl) imidazole-5-ones derivatives are most active against Polo-like kinase 1(PKL1) in complex with B16727 respectively.</p> </div> <div class="ligandbaseddesign" title="sec"> <div class="title" tagx="title" title="title">
Ligand based design</div> <p id="Par56">Eighteen (18) new 2-(4-fluorophenyl) imidazole-5-ones derivative compounds were designed, their predicted activities were higher than that of the chosen template (compounds 4 with pIC<sub>50âˆ’</sub>5.0620 and compound 2 with pIC<sub>50</sub>â€"4.9846) as shown in Table <a href="#Tab9">9</a>. From the mean effect of the descriptors, SpMax4_Bhs had a greater positive impact followed by GATS5e and MATS4e while RDF150u had the least negative impact on the model. According to MATS4e (Moran autocorrelation - lag 4 / weighted by Sanderson electronegativities) and GATS5e (Geary autocorrelation - lag 5 / weighted by Sanderson electronegativities) descriptors, adding more electronegative elements (GATS5e and MATS4e) would increase the potency of the designed compounds. The modification occurred by adding more electronegative elements to the template (compounds 2 and 4).</p> </div> <div class="pharmacokineticsofdesigned2-(4-fluorophenyl)imidazole-5-onescompounds" title="sec"> <div class="title" tagx="title" title="title">
Pharmacokinetics of designed 2-(4-fluorophenyl) imidazole-5-ones compounds</div> <p id="Par57">The designed compounds were assessed for their drug-friendliness. The molecules passed the drug- friendliness assessment (ADME and physicochemical properties) as shown in Table <a href="#Tab10">10</a>, none of the designed compounds violated two rules out of the Lipinski rule of five; a famous benchmark utilized in invalidating the drug-likeness of a molecule (as stated in â€œ<a href="#Sec13">Molecular Docking Studies</a>â€� section). The bioavailability radar of molecules 11, 13, and 17 are shown in Fig. <a href="#Fig7">7</a>. The bioavailability radar gives a quick and easy summary of the pharmacokinetic properties of a compound. The pink area signifies the ideal ranges for each property (lipophilicity: XLOGP3 from âˆ’0.7 to +5.0, size: molecular weight from 150 to 500Â g/mol, polarity: TPSA from 20 to 130Â Ã…<sup>2</sup>, solubility: log S less than 6, saturation: the fraction of carbons in the sp<sup>3</sup> hybridization not higher than 0.25, and flexibility: less than 9 rotatable bonds) (Daina et al. <a href="#CR8">2017</a>).</p> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par58">2-(4-fluorophenyl) imidazole-5-ones derivatives showed a more reliable anti-cancer drug candidate against MCF-7 cell line using QSAR analysis, molecular docking assessment, and pharmacokinetics analysis. The model 1 parameters obtained from QSAR showed that increasing MATS4e and RDF150u, and decreasing GATS5e and SPMAX4_Bhs would proliferation the biological activities of the inhibitors 2-(4-fluorophenyl) imidazol-5-ones derivatives as an effective drug for curing breast cancer. The strength and predictive capability of the generated equation was explored for both internal and external validation assessment which conforms with the least approved values, indicating that model number one parameters could be used in developing new 2-(4-fluorophenyl) imidazol-5-ones drug compounds with higher effectiveness. The model parameters (MATS4e and GATS5e) had more significant and based on their mean effect, adjustment were made on the fragments of the lead compounds (2 and 4) to design 18 new imidazole derivative compounds with a higher calculated activity against MCF-7 cell line.</p> <p id="Par59">The molecular docking result showed that compound 24 and 27 had the highest docking score of âˆ’8.8 andâ€‰âˆ’â€‰9.1Â kcal/mol. From the research it is proved that some series of 2-(4-fluorophenyl) imidazol-5-ones derivatives compounds bind tightly to the binding pose of the target, stabilizing the receptor Polo-like kinase 1(PKL1) in complex with B16727 which is proven from the complexes as seen above. The compounds would serve as the most capable inhibitors against (PKL1) and this shows a revolution in medicine to design and develop new estrogen-positive (MCF-7 cell line) breast cancer drugs.</p> <p id="Par60">Additionally, the pharmacokinetics analysis (drug-likeliness test) executed on the designed molecules revealed that all the compounds can move on to the next step of pre-clinical trial because they passed drug-friendliness analysis (ADME and other physicochemical properties) and they also adhered to the Rule of Five: a benchmark used in assessing the drug-likeness of compounds. This shows a great discovery for medicine in finding permanent solutions to breast cancer (MCF-7 cell line).</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <p>The authors acknowledge the technical effort of the physical chemistry Department, Ahmadu Bello University Zaria for their immense advice and various contribution toward the successful completion of the research.</p> </div> <div class="title" tagx="title" title="title">
Authorâ€™s contributions</div> <p>HAL worked on the data set computationally by the methodology to get a mathematical model with high predictive activity and was a major contributor in drafting the manuscript, AU carried out statistical analysis on the built model to ensure its effectiveness and also participated in the write-up and SU re-edited the work to ensure it conforms with the manuscriptâ€™s guide. All the Arthurs read and approved the final manuscript.</p> <div class="title" tagx="title" title="title">
Data availability</div> <p>The data set was obtained from (Abo-Elanwar et al. <a href="#CR4">2019</a>).</p> <div class="title" tagx="title" title="title">
Compliance with ethical standards</div> <div class="title" tagx="title" title="title">
Conflict of interest</div> <p id="Par61">Not applicable.</p> <div class="title" tagx="title" title="title">
Code availability</div> <p id="Par62">The softwareâ€™s used includes; Chemdraw software version 12.0.2, Spartanâ€™14 (version 1.1.2), Material studio (V8) software, Pyrex software version, PADEL-Descriptor Software V2.20, and DTC data lab software, Microsoft word Office Excel 2013 version and Auto dock visualizer version 4.2,</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdulfatai</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uzairu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uba</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular docking and quantitative structure-activity relationship study of anticonvulsant activity of aminobenzothiazole derivatives</span><span class="source" tagx="source" title="source">Beni-Suef Univ JBasic App Sci</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">204</span><span class="lpage" tagx="lpage" title="lpage">214</span></span></li> <li tag="ref"><a name="CR2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Abdullahi M, Uzairu A, Shallangwa GA, Mamza P, Arthur DE, Ibrahim MT (2019) In-silico modeling studies on some C14-urea-tetrandrine derivatives as potent anti-cancer agents against prostate (PC3) cell line. J King Saud Univ-Sci</span></li> <li tag="ref"><a name="CR3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Abdulrahman HL, Uzairu A, Uba S (2020) In silico studies of some 2-anilinopyrimidine derivatives as anti-triple-negative breast cancer agents. Beni-Suef University Journal of Basic and Applied Sciences 9(1):1â€"12</span></li> <li tag="ref"><a name="CR4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abo-Elanwar</span><span class="given-names" tagx="given-names" title="given-names">YA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mostafa</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El-Sayed</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nasr</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis and biological evaluation of new 2-(4-fluorophenyl) imidazol-5-ones as anticancer agents</span><span class="source" tagx="source" title="source">J Appl Pharm Sci</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">05</span><span class="fpage" tagx="fpage" title="fpage">001</span><span class="lpage" tagx="lpage" title="lpage">011</span><span class="pub-id"><a href="https://dx.doi.org/10.7324/JAPS.2019.90501">10.7324/JAPS.2019.90501</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bajaj</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis, thymidine phosphorylase inhibitory, and computational study of novel 1, 3, 4-oxadiazole-2-thione derivatives as potential anticancer agents</span><span class="source" tagx="source" title="source">Comput Biol Chem</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">151</span><span class="lpage" tagx="lpage" title="lpage">160</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.compbiolchem.2018.05.013">10.1016/j.compbiolchem.2018.05.013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30015176">30015176</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhaumik</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pal</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debnath</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karmakar</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jana</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misra</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Natural products inspired allicin analogs as novel anti-cancer agents</span><span class="source" tagx="source" title="source">Bioorg Chem</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">259</span><span class="lpage" tagx="lpage" title="lpage">272</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bioorg.2019.01.057">10.1016/j.bioorg.2019.01.057</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30731359">30731359</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Brandon V, Orr A (2015) Comprehensive R archive network (CRAN): <a href="http://cran.rproject.org">http://CRAN.Rproject.org</a></span></li> <li tag="ref"><a name="CR8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daina</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michielin</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zoete</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness, and medicinal chemistry friendliness of small molecules</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">42717</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep42717">10.1038/srep42717</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28256516">28256516</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De CÃ¡rcer</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">The mitotic cancer target polo-like kinase 1: oncogene or tumor suppressor?</span><span class="source" tagx="source" title="source">Genes</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">208</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/genes10030208">10.3390/genes10030208</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eriksson</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaworska</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Worth</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cronin</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McDowell</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gramatica</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Methods for reliability and uncertainty assessment and applicability evaluations of classification-and regression-based QSARs</span><span class="source" tagx="source" title="source">Environ Health Perspect</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">111</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">1361</span><span class="lpage" tagx="lpage" title="lpage">1375</span><span class="pub-id"><a href="https://dx.doi.org/10.1289/ehp.5758">10.1289/ehp.5758</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12896860">12896860</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Friedman JH (1991) Multivariate adaptive regression splines<i>.</i> Ann Stat 1â€"67</span></li> <li tag="ref"><a name="CR12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Guan L, Yang H, Cai Y, Sun L, Di P, Li W, ..., Tang Y (2019). ADMET-scoreâ€"a comprehensive scoring function for the evaluation of chemical drug-likeness. MedChemComm 10(1): 148â€"157</span></li> <li tag="ref"><a name="CR13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hou Y, Zhu L, Li Z, Shen Q, Xu Q, Li W, â€¦, Gong P (2019). Design, synthesis and biological evaluation of novel 7-amino-[1,2,4] triazolo [4,3-f] pteridinone and 7-aminotetrazolo [1,5-f] pteridinone derivative as potent antitumor agents. Eur J Med Chem 163: 690â€"709</span></li> <li tag="ref"><a name="CR14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ibrahim MT, Uzairu A, Shallangwa GA, Ibrahim A (2018) In-silico studies of some oxadiazoles derivatives as anti-diabetic compounds. J King Saud Univ-Sci</span></li> <li tag="ref"><a name="CR15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Iqbal J, Ejaz SA, Khan I, Ausekle E, Miliutina M, Langer P (2019) Exploration of quinolone and quinoline derivatives as potential anticancer agents. DARU J Pharm Sci 1â€"14</span></li> <li tag="ref"><a name="CR16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Jordan VC, Brodie AM (2007) Development and evolution of therapies targeted to the estrogen receptor for the treatment and prevention of breast cancer. Steroids 72(1):7â€"25</span></li> <li tag="ref"><a name="CR17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kennard</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Computer-aided design of experiments</span><span class="source" tagx="source" title="source">Technometrics</span><span class="year" tagx="year" title="year">1969</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">137</span><span class="lpage" tagx="lpage" title="lpage">148</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/00401706.1969.10490666">10.1080/00401706.1969.10490666</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">You</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis and biological evaluation of 4, 6-diaryl-2-pyrimidinamine derivatives as anti-breast cancer agents</span><span class="source" tagx="source" title="source">Bioorg Med Chem Lett</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">28</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1138</span><span class="lpage" tagx="lpage" title="lpage">1142</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bmcl.2017.12.066">10.1016/j.bmcl.2017.12.066</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29482944">29482944</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minovski</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ZË‡uperl</span><span class="given-names" tagx="given-names" title="given-names">Å </span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drgan</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">NovicË‡</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Assessment of applicability domain for multivariate counter-propagation artificial neural network predictive models by minimum Euclidean distance space analysis: a casestudy</span><span class="source" tagx="source" title="source">Anal Chim Acta</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">759</span><span class="fpage" tagx="fpage" title="fpage">28</span><span class="lpage" tagx="lpage" title="lpage">42</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.aca.2012.11.002">10.1016/j.aca.2012.11.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23260674">23260674</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Myers RH (1990) Classical and modern regression application. Duxbury Press</span></li> <li tag="ref"><a name="CR21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Premakumari</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muralikrishna</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padmaja</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padmavathi</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reddy</span><span class="given-names" tagx="given-names" title="given-names">GD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis, antimicrobial and anticancer activities of amido sulfonamido methane linked bis heterocycles</span><span class="source" tagx="source" title="source">Arab J Chem</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">385</span><span class="lpage" tagx="lpage" title="lpage">395</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.arabjc.2013.10.024">10.1016/j.arabjc.2013.10.024</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Putri DE, Pranowo HD, Haryadi WINARTO (2019) Study on anti-tumor activity of novel 3-substituted 4 Anilino-Coumarin derivatives using quantitative structure-activity relationship (QSAR). In <i>Materials Science Forum</i> (Vol. 948, pp. 101-108). Trans Tech Publications</span></li> <li tag="ref"><a name="CR23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanhaji</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kreis</span><span class="given-names" tagx="given-names" title="given-names">NN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimmer</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berg</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Louwen</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">p53 is not directly relevant to the response of polo-like kinase 1 inhibitors</span><span class="source" tagx="source" title="source">Cell Cycle</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">543</span><span class="lpage" tagx="lpage" title="lpage">553</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/cc.11.3.19076">10.4161/cc.11.3.19076</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22262171">22262171</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tropsha</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gramatica</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gombar</span><span class="given-names" tagx="given-names" title="given-names">VK</span></span></span><span class="mixed-article-title" title="mixed-article-title">The importance of being earnest: validation is the absolute essential for successful application and interpretation of QSPR models</span><span class="source" tagx="source" title="source">Mol Inf</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">69</span><span class="lpage" tagx="lpage" title="lpage">77</span></span></li> <li tag="ref"><a name="CR25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veerasamy</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajak</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sivadasan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Varghese</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agrawal</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Validation of QSAR models-strategies and importance</span><span class="source" tagx="source" title="source">Int J Drug Des Discov</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">511</span><span class="lpage" tagx="lpage" title="lpage">519</span></span></li> <li tag="ref"><a name="CR26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Xiao X, Xu M, Yang C, Yao Y, Liang LN, Chung PE, ..., Ben-David Y (2018) Novel racemosin B derivatives as new therapeutic agents for aggressive breast cancer. Bioorg Med Chem 26(23â€"24): 6096â€"6104</span></li> <li tag="ref"><a name="CR27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yap</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span></span><span class="mixed-article-title" title="mixed-article-title">PaDEL-descriptor: an open-source software to calculate molecular descriptors and fingerprints</span><span class="source" tagx="source" title="source">J Comput Chem</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">1466</span><span class="lpage" tagx="lpage" title="lpage">1474</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.21707">10.1002/jcc.21707</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21425294">21425294</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Design of novel dopamine D2 and serotonin 5-HT2A receptors dual antagonists toward schizophrenia: an integrated study with QSAR, molecular docking, virtual screening, and molecular dynamics simulations</span><span class="source" tagx="source" title="source">J Biomol Struct Dyn</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">38</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">860</span><span class="lpage" tagx="lpage" title="lpage">885</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2019.1590244">10.1080/07391102.2019.1590244</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30916624">30916624</a></span></span></li> </ul> </div> </div>  </body></html>