<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Virulence</span><span class="iso-abbrev" title="iso-abbrev">Virulence</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Virulence</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">2150-5594</span><span class="issn-epub" tagx="issn" title="issn-epub">2150-5608</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7781676">7781676</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/33315533">33315533</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/21505594.2020.1859820">10.1080/21505594.2020.1859820</a></span><span class="publisher-id" title="publisher-id">1859820</span><div class="unknown" title="article-version" tag="article-version">
Version of Record</div> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Research Paper</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
A novel phosphoinositide kinase Fab1 regulates biosynthesis of pathogenic aflatoxin in <i>Aspergillus flavus</i></div> <div class="alt-title" title="alt-title">
M. YANG ET AL.</div> <div class="alt-title" title="alt-title">
VIRULENCE</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Mingkun</span></span><a href="#aff0001"><sup>a</sup></a><a href="#aff0002"><sup>b</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Zhuo</span></span><a href="#aff0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">Youhuang</span></span><a href="#aff0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhuang</span><span class="given-names" tagx="given-names" title="given-names">Zhenhong</span></span><a href="#aff0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">Feng</span></span><a href="#aff0002"><sup>b</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Mingzhu</span></span><a href="#aff0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Shihua</span></span><a href="#aff0001"><sup>a</sup></a><a href="#an0001" /></span><span class="citation_author_institution" id="aff0001">[a], <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Fujian Agriculture and Forestry University</span></div><span class="city" tagx="city" title="city">Fuzhou</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="aff0002">[b], <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Institute of Hydrobiology, Chinese Academy of Sciences</span></div><span class="city" tagx="city" title="city">Wuhan</span><span class="country" tagx="country" title="country">China</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
CONTACT Shihua Wang <span class="email" tagx="email" title="email">wshyyl@sina.com</span></div> </div><span class="pub-date-" title="pub-date-">: <span>2020-12-12</span></span><span class="pub-date-" title="pub-date-">: <span>2021</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-12-12</span></span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">96</span><span class="lpage" tagx="lpage" title="lpage">113</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">The Author(s)</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"> <div tag="license_ref">
http://creativecommons.org/licenses/by/4.0/</div><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="KVIR_12_1859820.pdf" title="self-uri">KVIR_12_1859820.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
ABSTRACT</div> <p><i>Aspergillus flavus</i> (<i>A. flavus</i>) is one of the most important model environmental fungi which can produce a potent toxin and carcinogen known as aflatoxin. Aflatoxin contamination causes massive agricultural economic loss and a critical human health issue each year. Although a functional vacuole has been highlighted for its fundamental importance in fungal virulence, the molecular mechanisms of the vacuole in regulating the virulence of <i>A. flavus</i> remain largely unknown. Here, we identified a novel vacuole-related protein in <i>A. flavus</i>, the ortholog of phosphatidylinositol-3-phosphate-5-kinase (Fab1) in <i>Saccharomyces cerevisiae</i>. This kinase was located at the vacuolar membrane, and loss of <i>fab1</i> function was found to affect the growth, conidia and sclerotial development, cellular acidification and metal ion homeostasis, aflatoxin production and pathogenicity of <i>A. flavus</i>. Further functional analysis revealed that Fab1 was required to maintain the vacuole size and cell morphology. Additional quantitative proteomic analysis suggested that Fab1 was likely to play an important role in maintaining vacuolar/cellular homeostasis, with vacuolar dysregulation upon <i>fab1</i> deletion leading to impaired aflatoxin synthesis in this fungus. Together, these results provide insight into the molecular mechanisms by which this pathogen produces aflatoxin and mediates its pathogenicity, and may facilitate dissection of the vacuole-mediated regulatory network in <i>A. flavus</i>.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
KEYWORDS</div> <div class="kwd" title="kwd"><i>Aspergillus flavus</i></div> <div class="kwd" title="kwd">
Fab1</div> <div class="kwd" title="kwd">
vacuole</div> <div class="kwd" title="kwd">
aflatoxin production</div> <div class="kwd" title="kwd">
pathogenicity</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p><i>Aspergillus flavus</i> (<i>A. flavus</i>) is one of the most important species in the <i>Aspergillus</i> genus because it can cause both noninvasive and invasive systematic aspergillosis in immunocompromised individuals, animals and economically important crops [<a href="#cit0001">1</a>,<a href="#cit0002">2</a>]. <i>A. flavus</i> produces a range of potent carcinogens and toxins collectively known as aflatoxins, creating concerns regarding their potential for environmental contamination [<a href="#cit0003">3</a>]. It has been estimated that approximately 10% of food samples collected between 2007 and 2012 were estimated to contain aflatoxins [<a href="#cit0004">4</a>]. When these foods are ingested or when <i>A. flavus</i> is able to grow without immunological constraint, acute toxicity in addition to a long-term increase in cancer risk can occur in affected individuals [<a href="#cit0005">5</a>]. Owing to the economic and food safety issues in agriculture and medicine, much attention has been directed at the study of aflatoxin synthesis and pathway regulation through the genetics, molecular biology and biochemistry methods. Understanding the molecular mechanisms of aflatoxin biosynthesis and pathway regulation will enable the development of innovative approaches for reducing aflatoxin production in important agricultural crops and human exposure to aflatoxins in populations at high risk for aflatoxicosis. Although a wide variety of factors can induce or inhibit aflatoxin synthesis in <i>Aspergillus</i> [<a href="#cit0006">6–10</a>], the detailed regulatory mechanisms controlling aflatoxin production in <i>A. flavus</i> are still fragmentary.</p> <p>Recent studies have shown that fungal vacuoles/vesicles are versatile organelles, involved in regulating various cellular processes [<a href="#cit0011">11</a>]. Mutations affecting vacuole function and vacuole membrane homeostasis can lead to defects in fungal virulence [<a href="#cit0012">12–14</a>]. In <i>Candida albicans</i>, mutations of a subset of vacuolar proteins affected hyphal morphogenesis, growth, and virulence [<a href="#cit0015">15</a>]. Additionally, a lack of the vacuolar transport chaperone Vtc4 in <i>Ustilago maydis</i> significantly decreased its virulence on maize [<a href="#cit0016">16</a>], and deletion of the vesicle-vacuole fusion protein Vb1 influences aflatoxin synthesis in <i>A. parasiticus</i> [<a href="#cit0017">17</a>]. Notably, previous efforts have focused on the role of vacuoles and vesicles in the secondary metabolism of <i>A. flavus</i> [<a href="#cit0017">17</a>], suggesting that a functional vacuole plays an important role in fungal growth and virulence. However, very little is known about the regulatory mechanism of vacuoles in regulating <i>Aspergillus</i> metabolism, particularly the functions of vacuole-related genes.</p> <p>Accumulating evidence has revealed that phosphatidylinositol lipids play important roles in various cellular processes [<a href="#cit0018">18</a>]. Among them, phosphatidylinositol 3,5-bisphosphate (PtdIns-3,5-P<sub>2</sub>), which predominantly locates in the endosome and vacuole, has been implicated in membrane homeostasis in various types of eukaryotic cells [<a href="#cit0019">19</a>,<a href="#cit0020">20</a>]. PtdIns-3,5-P<sub>2</sub> is generated from phosphatidylinositol 3-phosphate (PI-3-P) by the phosphoinositide phosphate kinase Fab1. Previous effects have stated that Fab1 kinase is highly conserved in many fungi and may play an important regulatory role in maintaining the normal vacuole function and morphology by regulating the PtdIns-3,5-P<sub>2</sub> levels in phosphatidylinositol metabolism [<a href="#cit0021">21</a>,<a href="#cit0022">22</a>]. The ability of Fab1 to regulate vacuole size, protein sorting, osmotic balance and the trafficking of some substrates in yeast has been well established [<a href="#cit0019">19</a>,<a href="#cit0020">20</a>]. Notably, defects in the dynamic regulation of PtdIns-3,5-P<sub>2</sub> are linked to human diseases [<a href="#cit0022">22</a>]. In the human pathogenic fungus <i>C. glabrata</i>, PtdIns-3,5-P<sub>2</sub> levels regulated by Fab1 are closely related to membrane trafficking and vacuole homeostasis, and <i>C. glabrata</i> Fab1 plays important roles in the intracellular survival and virulence of this fungus [<a href="#cit0023">23</a>]. Despite the studies of Fab1 having had some success, the molecular details of how this enzyme regulates vacuolar functions and aflatoxin synthesis in <i>Aspergillus</i> are still lacking.</p> <p>To fill this gap, we have made an attempt to unravel the functional role of the <i>A. flavus</i> putative orthologue of <i>S. cerevisiae</i> PtdIns-3,5-P<sub>2</sub> kinase Fab1 in a variety of cellular processes. Our functional studies revealed that deletion of Fab1 influences the growth, development, aflatoxin production, pathogenicity, and vacuole/vesicle function of <i>A. flavus</i>. Further quantitative proteomic analysis suggested that Fab1 may maintain vacuolar/cellular homeostasis to facilitate aflatoxin synthesis in <i>A. flavus</i>. Together, these results provide insight into the mechanisms of <i>A. flavus</i> pathogenicity and aflatoxin biosynthesis.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="discoveryandvalidationofanovelvacuole-relatedprotein" title="sec"> <div class="title" tagx="title" title="title">
Discovery and validation of a novel vacuole-related protein</div> <p>Through NCBI BLAST (Basic Local Alignment Search Tool) searching, we identified an <i>A. flavus</i> protein with significant similarity to yeast (<i>Saccharomyces cerevisiae</i>) PtdIns-3,5-P<sub>2</sub> kinase (Fab1 protein) (<b>Fig. S1</b>). The coordinates of the putative <i>fab1</i> were 4,215,543–4,220,895 from NW_002477237.1 fragment in the <i>A. flavus</i> genome, with no annotation information (<b>Fig. S2</b>). To further confirm the existence of the putative <i>fab1</i>, we performed transcriptional expression analysis by using previously published RNA-Seq data. As shown in <b>Figure S2</b>, the additional transcriptomic data also supported the presence of this novel gene in the <i>A. flavus</i> genome. Notably, a highly similar homolog of this protein was discovered in <i>A. oryzae</i> and designated as a putative phosphatidylinositol-3-phosphate 5-kinase (<b>Fig. S3A</b>). Moreover, analysis of the protein structural domain revealed that this family of proteins contains a number of highly conserved domains, including a zinc finger structure (FYVE), as well as Cpn60_TCP1 and phosphatidylinositol phosphate kinase domains (PIPKc) (<a href="#f0001">Figure 1(a</a>)), which correspond to phosphatidylinositol phosphate kinases in most eukaryotic cells [<a href="#cit0019">19</a>]. These analyses thus suggested that this novel protein may function as a phosphatidylinositol phosphate kinase (named as Fab1) involved in phosphatidylinositol metabolism. We hypothesized that Fab1 in <i>A. flavus</i> may catalyze the synthesis of phosphatidylinositol 4,5-bisphosphate and phosphatidylinositol 3,5-bisphosphate, which are essential lipid molecules in a range of cellular processes [<a href="#cit0021">21</a>]. Phylogenetic analysis further revealed that this protein was highly conserved among different species, including <i>Aspergillus</i> species, <i>Elaphomyces granulatus, Penicillium arizonense, Penicillium coprophilum, Penicillium polonicum, Penicillium vulpinum</i> and <i>Talaromyces atroroseus</i> (<b>Fig. S3B</b>). To further validate the novel protein, we assessed <i>fab1</i> expression at the transcript level and our result revealed significant up-regulation under three distinct growth conditions (<a href="#f0001">Figure 1(b</a>)). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Fab1 identification and validation. (a) Domain analysis of Fab1 in different species. (b) Relative <i>fab1</i> mRNA levels under different growth conditions. Data are presented as means ±S.D. and represent the results of three independent experiments. Statistically significant differences are indicated: *<i>p</i> &amp;lt; 0.05. (c) Validation of <i>fab1</i> transcript quantification via northern blotting. (d) PCR validation of <i>fab1</i> fragment clones by 3’RACE. The second round of PCR was performed with the gene-specific forward primers and the RACE adaptor primers. The PCR products were analyzed by electrophoresis. Fragment1 and Fragment2 are two independent biological replicates. (e) Sequencing validation of amplified fragment from <i>fab1</i> by 3’RACE. The amplified fragment by 3’RACE included the splice site and ploy (A) of <i>fab1.</i></p>   </div> <p /> </div> <div class="fab1regulatesgrowth,conidia,andsclerotialdevelopment" title="sec"> <div class="title" tagx="title" title="title">
Fab1 regulates growth, conidia, and sclerotial development</div> <p>To elucidate the functional role of Fab1, we disrupted <i>fab1</i> by gene replacement mutagenesis using a fusion PCR approach (<b>Fig. S4A</b>). Mutant (∆fab1) and complementation (∆fab1::fab1) strains of <i>A. flavus</i> were generated and validated by PCR and sequencing (<b>Fig. S4B</b>). We also confirmed whether <i>fab1</i> was expressed in these strains <i>via</i> northern blotting (<a href="#f0001">Figure 1(c</a>)). In order to verify the <i>fab1</i> 3’ untranslated region (UTR), total <i>A. flavus</i> RNA was subjected to a 3’-RACE (rapid amplification of cDNA ends) experiment and the resultant 3’-RACE PCR product was sequenced to verify the splicing sites and poly(A) tail of <i>fab1</i> (<a href="#f0001">Figure 1(d,e</a>)).</p> <p>We next measured the growth and morphology of WT, ∆fab1 and ∆fab1::fab1 strains of <i>A. flavus</i> grown on yeast extract-sucrose agar (YES) and potato dextrose agar (PDA) agar media. After growth on YES and PDA media for 5 days, the ∆fab1 strain exhibited smaller diameter colonies relative to the WT and ∆fab1::fab1 strains (<a href="#f0002">Figure 2(a,b</a>) <b>and Fig. S5A, B</b>), suggesting that <i>fab1</i> deletion reduced the growth rate of this fungus. Consistently, the ∆fab1 strain failed to form conidiophores according to microscopic examination, whereas this effect was restored in the ∆fab1::fab1 strain (<a href="#f0002">Figure 2(c</a>) <b>and Fig. S5C</b>), suggesting that <i>fab1</i> is important for conidial formation in <i>A. flavus</i>. We additionally confirmed this result <i>via</i> measuring the expression of two transcription factor-encoding genes important for conidiation (<i>brlA</i> and <i>abaA)</i> [<a href="#cit0024">24</a>,<a href="#cit0025">25</a>] <i>via</i> qPCR during vegetative growth. We found the expression of both of these genes was significantly reduced by &amp;gt;90% in the ∆fab1 strain relative to in the WT and ∆fab1::fab1 strains (<b>Fig. S5D</b>). These results thus indicated that the conidiation pathway was affected by <i>fab1</i> deletion, leading to downregulation of <i>brlA</i> and <i>abaA</i> expression. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Morphological phenotypes of different <i>A. flavus</i> strains. Colony morphology of WT, ∆fab1 and ∆fab1::fab1 strains grown on YES (a) or PDA (b) media at 37°C for 5 days. (c) Conidiophores morphology of WT, ∆fab1 and ∆fab1::fab1 strains following a 12 h incubation. (d) Phenotypic characterization of WT, ∆fab1 and ∆fab1::fab1 strains grown on YPD medium at 37°C for 9 days. (e) TLC assay-mediated assessment of aflatoxin B<sub>1</sub> (AFB<sub>1</sub>) production by the WT, ∆fab1 and ∆fab1::fab1 strains in YES liquid media cultured at 37°C for 5 days. (f) MS analysis of aflatoxin production by WT, ∆fab1 and ∆fab1::fab1 strains. All data represent the results of three independent experiments</p>   </div> <p /> <p>As sclerotia are important means of resisting stress conditions, we assessed sclerotial formation when the strains were grown on sclerotia-inducing YPD medium at 37°C for 9 days. Unlike in the WT strain, sclerotia in the ∆fab1 strain were completely absent; this phenotype was reversed in the ∆fab1::fab1 strain (<a href="#f0002">Figure 2(d</a>) <b>and Fig. S5E</b>), suggesting that a loss of <i>fab1</i> led to impairing sclerotial formation. We additionally assayed the expression of the sclerotia-associated <i>nsdC, nsdD</i> and <i>nsdR</i> and found that both <i>nsdC</i> and <i>nsdR</i> were twofold downregulated in the ∆fab1 strain, while <i>nsdD</i> was downregulated by 10-fold relative to in the WT and ∆fab1::fab1 strains (<b>Fig. S5F</b>). These results thus together clearly indicated that Fab1 is essential for <i>A. flavus</i> growth, conidial formation, and sclerotial production.</p> </div> <div class="fab1isessentialforaflatoxinbiosynthesisandpathogenicity" title="sec"> <div class="title" tagx="title" title="title">
Fab1 is essential for aflatoxin biosynthesis and pathogenicity</div> <p>The synthesis of phosphatidylinositol bisphosphate by Fab1 on the vacuole has the potential to mediate homeostatic regulation of the vacuolar membrane and to regulate proximal ion channels within these vacuoles [<a href="#cit0026">26</a>,<a href="#cit0027">27</a>]. As such, we hypothesized that vacuolar dysregulation arising as a result of the loss of <i>fab1</i> would result in impaired aflatoxin export. To test this hypothesis, we assessed aflatoxin production via thin layer chromatography (TLC) and mass spectrometry (MS) analysis. TLC analyses indicated markedly reduced aflatoxin B<sub>1</sub> production from the ∆fab1 strain relative to from the WT and ∆fab1::fab1 strains (<a href="#f0002">Figure 2(e</a>)), with the levels reduced by &amp;gt;10-fold relative to the WT strain (<b>Fig. S5G</b>). We then validated this result via high-performance liquid chromatography (HPLC) coupled to a Xevo TQ MS machine. This analysis confirmed that only trace levels of aflatoxin B<sub>1</sub> were detectable and minimal aflatoxin B<sub>2</sub> was evident in the ∆fab1 strain, whereas normal production of these aflatoxins was observed for the WT and ∆fab1::fab1 strains (<a href="#f0002">Figure 2(f</a>) <b>and Fig. S5H</b>). This thus further suggested that Fab1 was likely to influence the biosynthesis of aflatoxins and their consequent export by controlling vacuolar membrane homeostasis.</p> <p>As aflatoxin contamination of food sources is a significant source of concern, we next explored the impact of <i>fab1</i> deletion on the pathogenicity of <i>A. flavus</i>. We observed difference growth dynamics for these strains when they were used to inoculate peanut seeds; the growth, sporulation, and invasion of the ∆fab1 strain were markedly less effective than in the WT and ∆fab1::fab1 strains (<a href="#f0003">Figure 3(a</a>)). Consistent with our previous results, the ∆fab1 strain tended to produce fewer conidia than the WT and ∆fab1::fab1 strains (<a href="#f0003">Figure 3(b</a>)). Previous studies showed that aflatoxin is important for seed infection [<a href="#cit0028">28–30</a>]. We subsequently assessed aflatoxin production in these samples, revealing that the WT and ∆fab1::fab1 strains, but not the ∆fab1 strain produced aflatoxin B<sub>1</sub> (<a href="#f0003">Figure 3(c</a>)). Further MS analyses confirmed that the ∆fab1 strain failed to produce detectable aflatoxins, whereas normal aflatoxin production was evident for the WT and ∆fab1::fab1 strains (<a href="#f0003">Figure 3(d</a>)). We similarly analyzed the invasive growth of these <i>A. flavus</i> strains on maize seeds, revealing that aflatoxin production and conidial formation were significantly reduced for the ∆fab1 strain relative to the WT and ∆fab1::fab1 strains (<a href="#f0003">Figure 3(e–h)</a>). Together, these findings indicated that <i>fab1</i> in <i>A. flavus</i> may play a regulatory role in seed infection. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>Infection analysis of seeds with different <i>A. flavus</i> strains. (a) Host colonization of WT, ∆fab1 and ∆fab1::fab1 strains. Peanut seeds were inoculated with WT, ∆fab1 and ∆fab1::fab1 strains and cultured at 29°C for 7 days. (b) Conidia amounts produced from the infected peanut seeds. (c) TLC assay results for aflatoxin B<sub>1</sub> (AFB<sub>1</sub>) levels in extracts from infected peanut seeds. (d) MS analysis of aflatoxin production from infected peanut seeds. (e) Maize seeds were inoculated with WT, ∆fab1, or ∆fab1::fab1 strains and cultured at 29°C for 7 days. (f) Conidia amounts produced from the infected maize seeds. (g) TLC was used to detect the aflatoxin B<sub>1</sub> (AFB<sub>1</sub>) production in the infected maize seeds. (h) MS analysis of aflatoxin production from the infected maize seeds. Data are presented as means ±S.D. and represent the results of three independent experiments. **<i>p</i> &amp;lt; 0.01</p>   </div> <p /> <p>It has been reported that enzymes involved in aflatoxin biosynthesis including Nor-1, Ver-1, and OmtA are localized to the vacuoles and vesicles in <i>A. parasiticus</i> [<a href="#cit0031">31</a>]. Thus, we hypothesized that <i>fab1</i> deletion would similarly impact aflatoxin biosynthesis in <i>A. flavus</i>. By staining these cells with neutral red dye, while we observed normal vesicle formation in the WT strain, we saw no vesicles formation in ∆fab1 cells when grown on aflatoxin-inducing medium (YES) for 3 days (<a href="#f0004">Figure 4(a</a>) <b>and B</b>). To further confirm this finding, we isolated the vesicle-vacuole fraction from <i>A. flavus</i> cells as previously described [<a href="#cit0032">32</a>]. We then performed TLC to measure aflatoxin levels in these subcellular fractions, further confirming that <i>fab1</i> deletion led to a near total loss of aflatoxin levels in this vesicle-vacuole fraction. Our results suggested that aflatoxins biosynthesis and transport may be disrupted in the ∆fab1 strain (<a href="#f0004">Figure 4(c</a>) <b>and Fig. S6</b>). This same finding was also evident in the cytosolic and secreted fractions obtained from these cells. Based on these findings and previous reports, we, therefore, concluded that Fab1 is likely to play a central role in maintaining vacuolar/cellular homeostasis, with vacuolar dysregulation upon <i>fab1</i> deletion leading to impaired aflatoxin synthesis, storage, and export, in agreement with previous reports [<a href="#cit0017">17</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>Aflatoxin production from different fraction of <i>A. flavus</i> strains. (a-b) Microscopic analysis of vacuole-vesicles stained with neutral red commonly used for staining of the vacuoles <i>in vivo</i>. (c) TLC assessment of aflatoxin B<sub>1</sub> (AFB<sub>1</sub>) production from different fractions of WT, ∆fab1, and ∆fab1::fab1 strains</p>   </div> <p /> </div> <div class="deletionoffab1impactsvacuolarphenotype" title="sec"> <div class="title" tagx="title" title="title">
Deletion of fab1 impacts vacuolar phenotype</div> <p>In yeast, Fab1 is associated with the vacuolar and pre-vacuolar compartments [<a href="#cit0033">33</a>]. In order to assess Fab1 subcellular localization in <i>A. flavus</i>, we first produced a Fab1-GFP fusion protein that was validated by PCR (<b>Fig. S7A</b>) and western-blotting analysis with an anti-GFP antibody (<b>Fig. S7B</b>). Subsequently, the WT strain expressing the Fab1-GFP fusion protein was stained with the membrane-specific dye FM4-64. As illustrated in <a href="#f0005">Figure 5(a</a>), strong fluorescence was detected in the fungal cells by laser scanning confocal microscopy (LSCM) analysis during conidial germination and hyphal growth. Notably, the green fluorescence signals (indicated by GFP) were clearly observed on the vacuolar membrane of the wild-type strain, whereas hyphal cells also showed obvious red signals (fluorescence of FM4-64) on the vacuolar membrane when the strains were stained with the membrane-specific dye. As a result, these results indicated that Fab1 protein is localized to the vacuolar membrane (<a href="#f0005">Figure 5(a</a>)), in agreement with previous fungal studies [<a href="#cit0033">33</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Localization of Fab1 in <i>A. flavus</i> strains. (a) Microscopic images of the WT strains expressing Fab1-GFP fusion protein. The strains were cultured in YES medium for 6, 24, 36, 48 and 60 h. The vacuolar membranes were stained with fluorochrome FM4-64. The bright, expressed, stained and merged images were shown. Scale bars represented 15 µm. (b) Bright-field microscopy of WT and ∆fab1 strains stained with neutral red commonly used for staining of the vacuole <i>in vivo</i>. The WT and ∆fab1 strains were cultured in YES medium for 6, 24, 36, 48 and 60 h. Scale bars represented 15 µm</p>   </div> <p /> <p>Vacuoles were previously shown to be essential for fungal growth, differentiation, symbiosis, and pathogenesis [<a href="#cit0034">34</a>]. We observed marked increases in the vacuole size in ∆fab1 fungi, with the vacuoles exhibiting swelling to fill most of the cell volume when the strains were grown under aflatoxin inducing conditions (YES) (<b>Fig. S8</b>). We next confirmed this vacuolar phenotype <i>via</i> staining the cells with neutral red dye when the cells were grown on aflatoxin-inducing medium (YES) during different growth stages. As shown in <a href="#f0005">Figure 5(b</a>), we observed the presence of significantly more large vacuoles in the ∆fab1 strain relative to in the WT control during conidial germination and hyphal growth. Surprisingly, the vacuole occupied most of the total volume of the fungal cell in ∆fab1 strain grown on YES medium for 60 h. The substantial vacuolar enlargement observed in the ∆fab1 strain had the potential to drive substantial cellular swelling. Our results suggested that the formation of large vacuoles was linked to <i>fab1</i> deletion, and the swelling was presumably associated with the well-documented role of proteins in this family as regulators of vacuolar/cellular homeostasis and turnover of the vacuolar membrane [<a href="#cit0033">33</a>,<a href="#cit0034">34</a>]. We speculated that Fab1 is vital for normal vacuole function and morphology in <i>A. flavus</i>.</p> </div> <div class="deletionoffab1impactscellularacidificationandmetalionhomeostasis" title="sec"> <div class="title" tagx="title" title="title">
Deletion of fab1 impacts cellular acidification and metal ion homeostasis</div> <p>Previous studies have proven that fungal vacuoles are acidic organelles, involved in many aspects of cellular processes, including cellular homeostasis, membrane trafficking, signaling, and nutrition, as well as stress responses [<a href="#cit0034">34</a>]. To further determine whether deletion of <i>fab1</i> would affect the cellular acidification, we measured the vacuolar and cytosolic pH in the WT, ∆fab1 and ∆fab1::fab1 strains. These strains were stained with a specific fluorescent dye (BCECF, 2,7-bis(2-carboxyethyl)-5,6-carboxyfluorescein-acetoxymethyl ester) to compare the changes of pH in different strains, and the vacuolar pH was quantified from the collected hyphal cells as previous reports [<a href="#cit0035">35</a>]. As expected, the vacuolar pH in the stained ∆fab1 strain was quantified as 5.76, which was significantly more acidic than that of the WT (pH 6.13) and ∆fab1::fab1 (pH 6.13) strains (<i>p</i> &amp;lt; 0.01) (<a href="#f0006">Figure 6(a</a>–c)). Furthermore, we observed the similar changes of the cytosolic pH in the WT (pH 7.15), ∆fab1 (pH 5.79) and ∆fab1::fab1 (pH 6.95) strains (<a href="#f0006">Figure 6(a</a>–c)). These results suggested a significant acidification of vacuole and cytoplasm in the absence of <i>fab1</i>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>Impacts of <i>fab1</i> deletion on cellular acidification and metal ion homeostasis. (a) Calibration curve showing the ratio of fluorescence intensity from excitation at 490 nm to intensity from excitation at 450 nm at different pH values. (b) Calibration curve showing the ratio of fluorescence intensity from excitation at 405 nm to intensity from excitation at 485 nm at different pH values. Wild-type cells were stained with BCECF-AM fluorescent dye and then equilibrated with buffers at varied pH for 60 min. A linear trendline is shown. (c) Vacuolar and cytosolic pH in WT, ∆fab1 and ∆fab1::fab1 strains. All strains were stained with BCECF-AM fluorescent dye and the fluorescence ratios were measured and converted to pH using the calibration curves. (d-f) Relative levels of intracellular Ca<sup>2+</sup>, Na<sup>+</sup> and K<sup>+</sup> in WT, ∆fab1 and ∆fab1::fab1 strains. All strains were stained with the specific fluorescent indicator dyes Fura-2-AM, SBFI-AM and PBFI-AM, respectively. The relative concentrations of Ca<sup>2+</sup>, Na<sup>+</sup> and K<sup>+</sup> in the cytoplasm were calculated based on the fluorescence ratio (Ratio 340/380). The unstained strains were used as controls. Data are presented as means ±S.D. and represent the results of three independent experiments. **<i>p</i> &amp;lt; 0.01</p>   </div> <p /> <p>In addition, cellular metal ions (Ca<sup>2+</sup>, Na<sup>+</sup> and K<sup>+</sup>) in the WT, ∆fab1 and ∆fab1::fab1 strains were assessed by using specific fluorescent indicator dyes, including Ca<sup>2+</sup>-specific fluorescent probe Fura-2-AM (Fura-2 acetoxymethyl ester), Na<sup>+</sup>-specific fluorescent probe SBFI-AM (sodium-binding benzofuran isophthalate acetoxymethyl ester) and K<sup>+</sup>-specific fluorescent probe PBFI-AM (potassium-binding benzofuran isophthalate acetoxymethyl ester), according to previously described methods [<a href="#cit0036">36–39</a>]. Upon binding metal ions, these specific fluorescence dyes can exhibit a fluorescence absorption shift from 380 to 340 nm of excitation. As a result, the relative Ca<sup>2+</sup>, Na<sup>+</sup> and K<sup>+</sup> concentrations were evaluated and quantified as the ratio of fluorescence intensities assessed at the excitation wavelengths of 340/380 nm. As shown in <a href="#f0006">Figure 6(d</a>–f), we observed obvious differences of intracellular metal ions concentrations in the WT, ∆fab1 and ∆fab1::fab1 strains. The ∆fab1 strain showed significantly decreased relative levels of intracellular Ca<sup>2+</sup>, Na<sup>+</sup> and K<sup>+</sup> concentrations compared to the WT and ∆fab1::fab1 strains (<i>p</i> &amp;lt; 0.01), indicating significant changes in cellular homeostasis in the absence of <i>fab1</i>. Together, our observations suggested that <i>fab1</i> deletion resulted in severe defect in vacuolar and cellular acidification, as well as cellular metal ion homeostasis.</p> </div> <div class="identificationofproteinsregulatedbyfab1" title="sec"> <div class="title" tagx="title" title="title">
Identification of proteins regulated by Fab1</div> <p>A variety of methodologies now support relative quantification measurements in proteomics, and one of the most commonly used strategies through MS is stable isotope labeling of peptides using isobaric reagents, such as tandem mass tag (TMT). To this end, we next examined the molecular basis of Fab1 function by TMT-based quantitative proteomic analysis to identify proteins regulated by Fab1 (<a href="#f0007">Figure 7(a</a>)). Finally, a total of 4,928 proteins were identified by proteomic analysis. Among them, 4,292 proteins were identified based upon the presence of at least two unique peptides and 3,821 proteins (&amp;gt;89%) were quantifiable (<a href="#f0007">Figure 7(b</a>) <b>and Table S1</b>). The large number of identified proteins reproducibly quantified in all samples led to very strong correlations for these fungal strains (<a href="#f0007">Figure 7(c</a>)). We constructed a heatmap showing the relative levels of quantifiable proteins in the WT and ∆fab1 <i>A. flavus</i> strains, with 289 proteins exhibiting a &amp;gt; 1.2-fold change between samples (∆fab1/WT) (<a href="#f0007">Figure 7(d</a>) <b>and Table S2</b>). Of these differentially expressed proteins (DEPs), 163 were up-regulated and 126 were down-regulated. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p>TMT-based quantitative proteomic analysis. (a) TMT strategy overview. (b) Venn diagram of identified and quantified proteins. (c) Comparison of TMT-based quantification intensities within three replicates, with correlations calculated <i>via</i> a Pearson correlation analysis. (d) Heatmap showing the ratio of quantifiable proteins between WT with Δfab1 strains</p>   </div> <p /> </div> <div class="functionalannotationandcellularlocalizationoffab1-regulatedproteins" title="sec"> <div class="title" tagx="title" title="title">
Functional annotation and cellular localization of Fab1-regulated proteins</div> <p>To further investigate the function of Fab1-regulated proteins, we next explored the biological roles of the proteins based upon their gene ontology (GO) functional annotations. A large portion of these proteins were associated with "metabolic processes," including phosphorus metabolism. As Fab1 appears to be associated with the vacuolar and pre-vacuolar compartments, and is a key regulator of vacuolar membrane homeostasis in <i>A. flavus</i>, it is perhaps unsurprising that these Fab1-regulated proteins were involved in transport and stress-response activities (<a href="#f0008">Figure 8(a</a>) <b>and Table S3</b>). Consistent with such roles, most of these proteins were found in the cytoplasm, cell membrane, and endoplasmic reticulum (<a href="#f0008">Figure 8(a</a>) <b>and Table S3</b>). Over 50 up-regulated DEPs were located on the integral component of the membrane, further indicating the importance of Fab1 in normal membrane homeostasis. Further enrichment analyses demonstrated that these DEPs were significantly enriched in metabolic processes, including metabolism of secondary metabolite glycerolipids and the metabolism of glycerophospholipid (<b>Table S4</b>). Interestingly, we also observed enrichment for the protein export, peroxisome, and MAPK signaling pathways, supporting a role for Fab1 in counteracting stress, due to the disruption of vacuolar/cellular homeostasis (<b>Table S4</b>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8.</span></a></div>   <p>Functional analysis of Fab1-regulated proteins. (a) Functional annotation of differentially expressed proteins (DEPs). (b) Heatmap highlighting the distribution of DEPs associated with development and secondary metabolite production in WT and Δfab1 strains</p>   </div> <p /> <p>Using the MultiLoc tool to assess protein subcellular localization, we found that most of the DEPs were located in the cytosol (Up: 64, Down: 63), followed by the extracellular (Up: 30, Down: 12) and mitochondria (Up: 17, Down: 8) compartments. Interestingly, many of these DEPs were located in the peroxisomes (Up: 14, Down: 12), vacuole (Up: 17, Down: 5), endoplasmic reticulum (Up: 6, Down: 4), and Golgi (Up: 3, Down: 0) (<b>Fig. S9A and Table S5</b>). A previous study showed that Fab1 is linked not only to vacuoles, but also to the pre-vacuolar compartment, which is important for protein trafficking from the Golgi and the plasma membrane to the vacuole in eukaryotic cells [<a href="#cit0040">40</a>]. In the present work, we observed robust localization of Fab1 to the vacuolar membrane of <i>A. flavus</i>, in line with a previous fungal study [<a href="#cit0033">33</a>]. Therefore, as unexpected, proteins involved in such trafficking activity were found to be significantly regulated by vacuolar dysregulation upon <i>fab1</i> deletion.</p> </div> <div class="fab1-regulatedproteinsinvolvedinsecondarymetaboliteproduction" title="sec"> <div class="title" tagx="title" title="title">
Fab1-regulated proteins involved in secondary metabolite production</div> <p>Consistent with the functional annotations, the DEPs were mapped to KEGG pathways and we found a subset of DEPs to be involved in development and secondary metabolite production in <i>A. flavus</i> (<a href="#f0008">Figure 8(b</a>)). For example, proteins associated with development, such as conidial hydrophobin RodB/HypB [<a href="#cit0041">41</a>], glucose-methanol-choline oxidoreductase [<a href="#cit0042">42</a>], spore-specific catalase CatA [<a href="#cit0043">43</a>], a TCTP family protein [<a href="#cit0044">44</a>], were also differentially expressed in the ∆fab1 strain. We further identified 14 proteins involved in aflatoxin biosynthesis, including monooxygenase, polyketide synthase, o-methyltransferase, alcohol dehydrogenase, aldehyde reductase, acetate kinase, lipase/esterase, Spt-Ada-Gcn5 acetyltransferase (Ada2) [<a href="#cit0045">45</a>], and cysteine-rich secreted protein [<a href="#cit0046">46</a>]. We also observed significant changes in the levels of two vacuole-associated proteins and four proteins linked with phosphatidylinositol metabolism, further supporting the roles of these proteins in vacuolar homeostasis.</p> </div> <div class="confirmationofdifferentiallyregulatedproteinsbyparallelreactionmonitoringandrt-pcranalysis" title="sec"> <div class="title" tagx="title" title="title">
Confirmation of differentially regulated proteins by parallel reaction monitoring and RT-PCR analysis</div> <p>Owing to the high sensitivity and robustness, parallel reaction monitoring (PRM) has recently emerged as the "gold standard" MS technique for targeted proteomic approaches, which are routinely used for quantitative analysis of targeted proteins [<a href="#cit0047">47</a>,<a href="#cit0048">48</a>]. To independently validate our quantitative results, we next conducted a PRM assay for 15 of the identified DEPs. Most of the selected DEPs were represented by either one or two unique peptides, with 28 total peptides represented (<b>Table S6</b>). Relative to the WT strain, five of these DEPs were down-regulated and the remaining 10 were significantly upregulated in the ∆fab1 strain (<b>Fig. S9B and Table S7</b>). The PRM analysis revealed results nearly identical to our TMT-based analysis, confirming quality and quantitative accuracy of our MS data.</p> <p>To further validate these findings, we subjected 41 of the DEPs to RT-PCR-mediated comparisons of transcript abundance between the WT and ∆fab1 <i>A. flavus</i> strains. We observed expression patterns consistent with our proteomic results for more than half of these genes, whereas discrepancies between the mRNA and protein abundance were observed for the remaining genes (<b>Fig. S9C</b>). This weak correlation between the mRNA and protein abundance suggests the potential for the proteins posttranscriptional regulation of these genes. Together, these results revealed that Fab1 may influence aflatoxin biosynthesis via maintaining vacuolar/cellular homeostasis by regulating diverse vacuole-related proteins.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Fungal vacuoles/vesicles play a crucial role in the regulation of various cellular processes. In this study, we identified a novel phosphoinositide kinase (Fab1) in the vacuole of <i>A. flavus</i>. Experimental evidence revealed that Fab1 was essential for the growth, development, cellular acidification and metal ion homeostasis, aflatoxin production and pathogenicity of <i>A. flavus</i>. Further functional studies of Fab1 suggested that this kinase plays a crucial role in the maintenance of vacuole size and cell morphology in this fungus. Our results provide insight into the mechanisms of aflatoxin synthesis and <i>A. flavus</i> pathogenicity, highlighting potential novel strategies for controlling aflatoxin contamination.</p> <p>The Fab1 protein identified in this work is highly conserved among different fungi (<a href="#f0001">Figure 1(a</a>) <b>and Fig. S3B</b>). Originally identified in yeast [<a href="#cit0049">49</a>], this kinase can phosphorylate phosphatidylinositol-3-phosphate (PI-3-P) to phosphatidylinositol 3,5-bisphosphate (PtdIns-3,5-P<sub>2</sub>), which is closely linked to cellular stress responses [<a href="#cit0033">33</a>]. PtdIns-3,5-P<sub>2</sub> further regulates vacuole and cellular homeostasis [<a href="#cit0026">26</a>,<a href="#cit0027">27</a>], with reduced levels of PtdIns-3,5-P<sub>2</sub> being associated with serious neurodegenerative diseases [<a href="#cit0050">50–52</a>]. Notably, this lipid kinase appears to be an increasingly attractive target for drug development, particularly in the fields of cancer and other proliferative diseases, and in suppressing inflammatory and immune responses [<a href="#cit0053">53</a>]. As such, Fab1 is necessary for regulating PtdIns-3,5-P<sub>2</sub> levels <i>in vivo</i>, potentially playing essential roles in a wide range of processes. In agreement with previous results [<a href="#cit0033">33</a>], we found that Fab1 localized to the vacuole membrane in <i>A. flavus</i>, where it appeared to play a key regulatory role in this fungus. Given the importance of PtdIns(3,5)P<sub>2</sub>, loss of <i>fab1</i> in <i>A. flavus</i> led to significant defects in growth and development, as expected and observed in other organisms [<a href="#cit0020">20</a>,<a href="#cit0054">54–58</a>]. Interestingly, <i>fab1</i> deletion results in altered cellular morphology without affecting the virulence of <i>Candia albicans</i> [<a href="#cit0059">59</a>]. In this study, we generated evidence that Fab1 was able to impact aflatoxin production, suggesting this gene to be associated with the regulation of <i>A. flavus</i> virulence.</p> <p>Based on our findings and past reports, we were able to construct a hypothetical model in which Fab1 regulates aflatoxin synthesis (<a href="#f0009">Figure 9</a>). Peroxisomes and mitochondria can serve as a source of the large amounts of acetyl-CoA necessary for aflatoxin synthesis, flowing from peroxisomes into vacuoles [<a href="#cit0005">5</a>]. Early aflatoxin synthesis may also occur in peroxisomes (<a href="#f0009">Figure 9</a>). Consequently, acetyl-CoA synthesis and flux may be regulated by enzymes within these organelles, influencing early aflatoxin biosynthesis. The most important enzymes for the mid- and late stages of aflatoxin biosynthesis are also known to transit to the vacuoles from the nucleus, Golgi, and endoplasmic reticulum via autophagic and cytoplasm-to-vacuole-targeting (Cvt) pathways [<a href="#cit0005">5</a>]. The DEPs in these organelles may also impact gene expression and transport mechanisms, thereby altering aflatoxin synthesis. Notably, enzymes linked with secondary metabolite production are often localized to vesicles and vacuoles, and the same is true for those associated with fungal aflatoxin production [<a href="#cit0031">31</a>,<a href="#cit0060">60</a>,<a href="#cit0061">61</a>]. In <i>Beauveria bassiana</i>, the vacuole localized protein 4 (VLP4) are required for the autophagy, development, and virulence, suggesting the potential role of vacuolar protein in fungal virulence [<a href="#cit0062">62</a>]. In this study, altered expression of the vacuole/vesicle-associated proteins and of other transport-associated proteins thus can impact aflatoxin biosynthesis, transport, and excretion. Together, we conclude that <i>fab1</i> deletion may alter PtdIns-3,5-P<sub>2</sub> levels, further disrupting vacuolar homeostasis and impacting the whole metabolic network in <i>A. flavus</i>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9.</span></a></div>   <p>The proposed model for the role of Fab1 in aflatoxin biosynthesis. Fab1 may alter PtdIns-3,5-P<i><sub>2</sub></i> levels, further disrupting vacuolar homeostasis and impacting the entire metabolic network in <i>A. flavus</i>. Red and green correspond to the number of upregulated and downregulated proteins, respectively</p>   </div> <p /> <p>In summary, a novel phosphoinositide kinase (Fab1) was identified in <i>A. flavus</i>. In order to further explore its biological roles, we revealed it to have previously undocumented roles in the regulation of pathogenicity and aflatoxin production. Our results can serve as a valuable resource for future efforts to explore the synthesis, storage, transport, and excretion of secondary metabolites including aflatoxins.</p> </div> <div class="experimentalprocedures" title="sec"> <div class="title" tagx="title" title="title">
Experimental procedures</div> <div class="cellcultureandproteinextraction" title="sec"> <div class="title" tagx="title" title="title">
Cell culture and protein extraction</div> <p>All <i>A. flavus</i> strains used in this study are given in <b>Table S8</b>. <i>A. flavus</i> was cultured on liquid yeast extract-sucrose agar (YES) and potato dextrose agar (PDA) and the <i>A. flavus</i> were then grown under several conditions. For the low temperature and pH stresses, 10<sup>6</sup> conidia were resuspended in fresh YES media at 29°C or YES media (pH 5, with hydrochloride) at 37°C. For the oxidative and hyperosmotic stresses, 10<sup>6</sup> conidia were resuspended in fresh YES media containing additional H<sub>2</sub>O<sub>2</sub> (0.8 M) or sodium chloride (1 M), respectively.</p> <p>The cultures were then collected by filtration and washed three times with phosphate-buffered saline (PBS) buffer. The mycelia were ground into powders and reconstituted in buffer [20 mM Tris-Cl (pH 7.5), 150 mM NaCl, 1× phenylmethanesulfonyl fluoride (PMSF)]. The mixtures were shaken at 30°C for 1 h and the debris was removed by centrifugation at 5,000 × g at 4°C for 1 h. Finally, the protein concentration was measured by bicinchonininc acid (BCA) method (Tiangen, Beijing, China).</p> </div> <div class="real-timepcrand3’rapidamplificationofcdnaends(3’race)validations" title="sec"> <div class="title" tagx="title" title="title">
Real-Time PCR and 3’ rapid amplification of cDNA ends (3’RACE) validations</div> <p>Total RNA was extracted from the mycelia grown in different conditions (37°C, 29°C, salt and H<sub>2</sub>O<sub>2</sub> stresses) using TRIzol reagent (Invitrogen) as the manufacturer’s protocols. The concentration of total RNA was quantitated using the Nanodrop 2000 spectrophotometer. RT-PCR validation of novel genes was performed using the SYBR Green PCR Master Mix (Applied Biosystems) and the LightCycler 480 Real-Time PCR System (Roche). The actin gene was used as the endogenous control and three independent biological replicates were carried out for duplicate samples. The gene-specific primers were compiled in <b>Table S9</b>.</p> <p>3’RACE procedures were performed by using 3’-Full RACE Core Set kit (Code No. 6106, Takara) as the manufacturer’s instructions. The first-strand cDNA was synthesized by using 1 μg RNA at the poly(A) tail of the mRNA. Two rounds of nested PCR were performed with the gene-specific forward primers and the RACE adaptor primers. Touch-down PCR protocol was used to confirm the appropriate annealing temperature. Amplicons were detected by electrophoresis on 1% agarose gel to determine the length, and then cloned to T-vector and further sequenced to confirm the specificity of amplification. The gene-specific forward primers were: fab1-RACE1: 5’-GTTTTACGCTGAACAGTTCGATGCTTT-3’ and fab1-RACE2: 5’-CAAGAGTCGGTTCAGGGAAGCAATGGC-3’.</p> </div> <div class="northernblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
Northern blot analysis</div> <p>Northern blotting analysis was carried out by using DIG Northern Starter Kit (Roche) following the manufacturer’s protocol. The integral RNA was hybridized with a digoxigenin (DIG)-labeled RNA probe of the <i>fab1</i> gene overnight at 42°C. The double-stranded DNA probes specific for <i>fab1</i> was synthesized <i>via</i> PCR using primers 5’-TGAACCTTCCTCGTCCTC-3’ and 5’-CCACATTCGCTAGTCACAG-3’. The <i>actin</i> gene was used as control and the probe was generated <i>via</i> PCR using primers 5’-TGATGAGGCACAGTCCAAGCG-3’ and 5’-CACCGATCCAGACGGAGTATTT-3’. Chemiluminescent signals were detected using a fluorescence scanner (ImageQuant TL, GE Healthcare).</p> </div> <div class="westernblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
Western blot analysis</div> <p>The proteins were extracted as mentioned in the protein extraction section and about 50 μg of proteins were used for western blotting analysis as previously described [<a href="#cit0063">63</a>].</p> </div> <div class="constructionofthefab1deletionmutant" title="sec"> <div class="title" tagx="title" title="title">
Construction of the fab1 deletion mutant</div> <p>A gene deletion mutant was constructed and confirmed by previously described methods [<a href="#cit0064">64</a>]. Briefly, the upstream (<i>AP</i>) and downstream (<i>BP</i>) sequences of the <i>fab1</i> gene, as well as the <i>pyrG</i> gene of <i>A. fumigatus</i> were fused into the interruption fragment (<i>AP-pyrG-BP</i>) using a fusion PCR strategy. The fused fragment was then transformed into <i>A. flavus</i> CA14 PTs protoplasts to construct the <i>fab1</i> deletion strain (∆fab1). For the construction of the complementary strain, <i>fab1</i> gene with its upstream promoter region was firstly cloned into the pPTRI vector (Takara) and the recombinant plasmid was then transformed into the ∆fab1 mutant protoplasts to construct the complementary strain (∆fab1::fab1). All mutant and complementary strains were confirmed by PCR and sequencing.</p> </div> <div class="mycelialgrowth,conidiationandsclerotiaquantification" title="sec"> <div class="title" tagx="title" title="title">
Mycelial growth, conidiation and sclerotia quantification</div> <p>The YES and PDA media were point inoculated with 10<sup>5</sup> conidia and kept at 37°C for 5 days in dark to evaluate the growth rate. Then, the colony morphology was recorded and colony diameter was measured. After homogenization and dilution with distilled water, conidia amount was counted using a hemocytometer and a microscope as previously reported [<a href="#cit0065">65</a>]. For the sclerotia analysis, 10<sup>5</sup> conidia were inoculated on Yeast Peptone Dextrose (YPD) media and kept for 9 day at 37°C in the dark and the sclerotial formation was counted. In order to wash away conidia to aid in enumeration of the sclerotia, the plates were washed with 75% ethanol. Each experiment was carried out with three independent biological replicates.</p> </div> <div class="seedinfections" title="sec"> <div class="title" tagx="title" title="title">
Seed infections</div> <p>To measure the pathogenicity of <i>A. flavus</i>, mature peanut and maize seeds were washed with 0.05% sodium hypochlorite, 75% ethanol and sterile water, and then inoculated by immersion in a 10<sup>7</sup> spores suspension for 30 min. The infected seeds were then placed in culture dishes lined with three pieces of moist sterile filter paper to maintain humidity. After 5 days of incubation, the infected seeds were pictured, collected, and mixed with 0.05% Tween 80, followed by vortexing to release the spores. About 100 µl spore suspension was collected, diluted, and counted for further quantification. Aflatoxins were extracted and analyzed by thin layer chromatography (TLC) and mass spectrometer assays according to the protocol described below. Each treatment was performed with three independent biological replicates.</p> </div> <div class="vesicle-vacuoleandnon-vesicle-vacuolefractions" title="sec"> <div class="title" tagx="title" title="title">
Vesicle-vacuole and non-vesicle-vacuole fractions</div> <p><i>A. flavus</i> was cultured in YES media at 37°C for 2 days and the protoplasts were prepared using previously described protocols [<a href="#cit0064">64</a>]. The vesicle-vacuole fraction was purified as described [<a href="#cit0032">32</a>].</p> </div> <div class="thinlayerchromatography(tlc)andmsquantificationofaflatoxins" title="sec"> <div class="title" tagx="title" title="title">
Thin layer chromatography (TLC) and MS quantification of aflatoxins</div> <p>Aflatoxins from different cultures were extracted by equal volume chloroform with vibration for 30 min and purified by 0.45 u filter. The chloroform layer containing aflatoxins was then dried at 70°C and the precipitates were finally redissolved in 50 ul chloroform for TLC analysis and the fluorescence signal of aflatoxin was analyzed and quantified with Image J suite (<a href="http://rsbweb.nih.gov/ij/">http://rsbweb.nih.gov/ij/</a>).</p> <p>Mass spectrometry was also used to detect aflatoxins. Briefly, the samples were analyzed by an ACQUITY H-Class UPLC system coupled to a Xevo TQ (Waters, Milford, MA). Aflatoxins were separated with an analytical C<sub>18</sub>-nanocapillary LC column (1.7 μm particle, 2.1 × 50 mm) and eluted with a linear gradient elution program as follows: 0–3 min, 15%-50% solvent B (0.1% formic acid/80% acetonitrile, v/v); 4–5 min, 50%-70%; 6.5–8 min, 70%-100%; 8–10 min, 100%-50%; 10–11 min, 50%-15%; 11–15 min, 15%; at 200 nL/min flow rate. The MS was operated in selected reaction monitoring (SRM) mode. MS source conditions included a cone voltage optimized to 30 V, a capillary voltage set to 3 kV, collision energy set to 35 V, nitrogen gas flow at 50 L/h, a source temperature of 150°C, desolvation temperature at 350°C. Data was acquired using MassLynx 4.0. The precursor ions of aflatoxins (AFB<sub>1</sub> and AFB<sub>2</sub>) were set to 312.92 m/z and 314.78 m/z, respectively.</p> </div> <div class="microscopicanalysis" title="sec"> <div class="title" tagx="title" title="title">
Microscopic analysis</div> <p><i>A. flavus</i> was cultured in YES media at 37°C for 6, 24, 36, 48, or 60 h. After washing three times with PBS to remove the media, hyphal filaments were separated from the mycelia pellet by using forceps, and then stained with neutral red staining solution (C0125, Beyotime, Jiangsu, China) for 5–8 min. In addition, all samples were stained with 50 µm FM4-64 staining solution (N-3-triethy lammoniumpropyl 4-p-diethylamino phenyl-hexa-trienyl pyridinium dibromide) (DE-N4073, BioDee, Beijing, China) for 20–30 min. After washing three times with PBS, the hyphal filaments were placed on a microscope slide, and observed using a Nikon Ti-U microscope. Fluorescent photographs were observed on a Leica TCS SP8 laser scanning confocal microscope (LSCM). Green fluorescent protein (GFP) was excited with a 488 nm laser and FM4-64 with a 558 nm laser. Fluorescence emissions were measured at 507 nm and 734 nm for GFP and FM4-64, respectively.</p> </div> <div class="measurementofvacuolarandcytosolicph" title="sec"> <div class="title" tagx="title" title="title">
Measurement of vacuolar and cytosolic pH</div> <p>The vacuolar and cytosolic pH values were measured as previously described [<a href="#cit0035">35</a>,<a href="#cit0066">66</a>]. Briefly, all <i>A. flavus</i> strains were cultured in YES media, and then collected and washed three times with PBS buffer to remove the media. The hyphal filaments were then resuspended in PBS buffer containing 50 µM BCECF-AM [2,7-bis(2-carboxyethyl)-5,6-carboxyfluorescein-acetoxymethyl ester] fluorescent dye (GC14603, GLPBIO, Montclair, CA, USA) and incubated at 37°C for 20–30 min. After washing three times with PBS buffer, the fluorescence intensities from different stained hyphal filaments were quantified at the respective excitation/emission wavelengths of 450/535 and 490/535 nm for vacuolar fluorescence, while cytosolic fluorescence intensities were measured at the excitation wavelengths of 405 and 485 nm, with an emission wavelength of 508 nm using an automatic hybrid multi-mode microplate reader (Synergy H1, BioTek, Winooski, VT, USA). The calibration curves for vacuolar and cytosolic pH were established according to previously described [<a href="#cit0060">60</a>,<a href="#cit0061">61</a>], and the vacuolar and cytosolic pH values were calculated based on the calibration equations: <i>y</i> = −7.1084 + 1.8409<i>x</i> (<i>r</i><sup>2</sup> = 0.9785, for vacuolar pH) and <i>y</i> = −1.8817 + 0.4496<i>x</i> (<i>r</i><sup>2</sup> = 0.9948, for cytosolic pH), where <i>y</i> is the ratio and <i>x</i> is pH value. Each treatment was performed with three independent biological replicates. Statistical comparisons were performed by Student’s <i>t</i>-test.</p> </div> <div class="measurementofcytosolicca2+/na+/k+levels" title="sec"> <div class="title" tagx="title" title="title">
Measurement of cytosolic Ca<sup>2+</sup>/Na<sup>+</sup>/K<sup>+</sup> levels</div> <p>The metal ions in <i>A. flavus</i> were assayed by staining with the Ca<sup>2+</sup>-specific fluorescence dye Fura-2-AM (Fura-2 acetoxymethyl ester, 40702ES50, Yeasen, Shanghai, China), Na<sup>+</sup>-specific fluorescence dye SBFI-AM (sodium-binding benzofuran isophthalate acetoxymethyl ester, GC44876, GLPBIO, Montclair, CA, USA) and K<sup>+</sup>-specific fluorescence dye PBFI-AM (potassium-binding benzofuran isophthalate acetoxymethyl ester, GC18477, GLPBIO, Montclair, CA, USA) as previous reports [<a href="#cit0036">36–39</a>]. All <i>A. flavus</i> strains were cultured in YES media, then collected and washed three times with PBS buffer to remove the media. The hyphal filaments were resuspended in PBS buffer containing 10 mM fluorescence dyes (Fura-2-AM, SBFI-AM, or PBFI-AM) and incubated at 37°C for 20–30 min. After washing three times with PBS buffer, the fluorescence intensities from the different stained hyphal filaments were measured at excitation wavelengths of 340/380 nm and an emission wavelength of 505 nm by using a Synergy H1 Hybrid Multi-Mode Microplate Reader (BioTek, Winooski, VT). The relative metal ions concentrations in different hyphal filaments were calculated based on the ratio of fluorescence intensity after dual-wavelength excitation. The unstained strains were used as controls. Each treatment was performed with three independent biological replicates. Statistical comparisons were performed by Student’s <i>t</i>-test.</p> </div> <div class="proteinextraction,digestionandtmtlabelingofwildtype(wt)andmutants" title="sec"> <div class="title" tagx="title" title="title">
Protein extraction, digestion and TMT labeling of wild type (WT) and mutants</div> <p>Protein lysates from WT and ∆fab1 strains were extracted and quantified as the protocol described in protein extraction section. We precipitated the lysate using 80% ice-cold acetone and then washed three times with ice-cold acetone to remove the pigment and other small molecules. The protein lysates were finally redissolved in 50 mM ammonium bicarbonate. Equal amounts of protein extracts were then in-solution digested by trypsin (Promega) as previously described [<a href="#cit0063">63</a>,<a href="#cit0067">67</a>] and each sample was prepared with three biological replicates. After trypsin digestion, a total of 100 µg of each sample was further processed as the manufacturer’s protocol for TMT 6plex isobaric label kit (Thermo Fisher Scientific). The labeled peptides were mixed at equal amounts and desalted with C<sub>18</sub> (40 μm, 60-Å particle; Agilent Technologies) columns and dried with a vacuum centrifuge.</p> </div> <div class="hplcfractionationandlc-ms/msanalysis" title="sec"> <div class="title" tagx="title" title="title">
HPLC fractionation and LC-MS/MS analysis</div> <p>TMT-labeled peptides were resuspended in 1% (v/v) formic acid and separated into 60 fractions by high pH reverse-phase HPLC using a 4.6 mm × 25 cm C<sub>18</sub> column (5 μm particles, Agilent) with a 60 min gradient of 8–32% acetonitrile (10 mM ammonium bicarbonate, pH 9). Finally, the peptides were combined into nine fractions and dried with a vacuum centrifuge.</p> <p>After dissolution with 0.1% formic acid, the fractioned peptides were directly separated with an online nanoflow EASY-nLC 1200 system (Thermo Fisher Scientific). Peptides were eluted with a gradient of an increase from 6% to 22% of solvent B (0.1% formic acid/90% acetonitrile, v/v) over 38 min, 22%-32% in 4 min, 32%-80% in 4 min, and holding at 80% for the last 4 min at 450 nL/min flow rate. A full MS scan from m/z 350 to 1,600 was acquired at 120,000 resolution. The top 20 most intense precursor ions were selected for following MS/MS fragmentation by HCD with NCE of 28% and analyzed with a resolution of 30,000 in the Orbitrap. The dynamic exclusion was set to 30 s and maximum injection times for both MS and MS/MS were 50 ms and 60 ms, respectively. The isolation width of precursor ion was set to 1.4 m/z and the intensity threshold was set at 8.3E4. AGC for both MS and MS/MS were 3E6 and 1E5, respectively.</p> </div> <div class="proteinidentificationandquantification" title="sec"> <div class="title" tagx="title" title="title">
Protein identification and quantification</div> <p>For the identification, the RAW data were searched against the <i>A. flavus</i> protein database concatenated with the protein sequences of common contaminants using MaxQuant software (version 1.6.3.4). Trypsin was used as a protease with two maximum missed cleavages and fixed modification was carbamidomethylation of cysteine. Dynamic modifications were set as deamidation (Asn/Gln) and oxidation (Met). Reporter ion was set as 6plexTMT for quantification. The mass errors for precursor ions and fragment ions were set to 20 ppm and 0.02 Da, respectively. Peptide for quantification was set as unique and razor. The FDR thresholds for peptide and protein identification were specified at maximum 1% and minimum score for peptides was set &amp;gt;40. Proteins identified based upon the presence of at least two unique peptides were accepted and reported.</p> <p>For the TMT-labeling quantification, the ProteinGroup file from MaxQuant result was loaded to Perseus software (version 1.5.6.0) to facilitate statistical analysis. Only proteins that were identified and quantifiable in three biological replicates were used for relative quantification. The intensity of the TMT reporter ion for each protein was normalized against the total intensity of each sample and the corrected TMT reporter ion intensities in each sample were used to calculate fold changes between samples. A two-sample Student’s <i>t</i>-test was used for the statistical evaluation and the coefficient of variation (CV) was computed from the three biological replicates. The differentially expressed proteins were defined as fold-change ≥1.2 or ≤0.83, and <i>p</i> &amp;lt; 0.05, CV &amp;lt; 20%.</p> </div> <div class="ddaandparallelreactionmonitoring(prm)experiments" title="sec"> <div class="title" tagx="title" title="title">
DDA and parallel reaction monitoring (PRM) experiments</div> <p>The same protein extracts derived from quantitative proteomics were tryptic digested as previously described [<a href="#cit0063">63</a>,<a href="#cit0067">67</a>]. Each experiment was carried out with three independent biological replicates. The digested peptides were separated on an online nano-flow EASY-nLC 1000 system (Thermo Fisher Scientific) with a gradient of 7–25% solvent B (90% acetonitrile/0.1% formic acid, v/v) over 40 min, 25%-35% in 8 min, 35%-80% in 4 min and holding at 80% for the last 4 min at 350 nL/min flow rate.</p> <p>For DDA-based experiments, a full MS scan from m/z 350 to 1,800 was acquired at 70,000 resolution. The top 10 most intense precursor ions were selected for following MS/MS fragmentation by HCD with NCE of 28% and analyzed with a resolution of 17,500 in the Orbitrap. The electrospray voltage applied was 2.1 kV and dynamic exclusion was set to 30 s. The maximum injection times for both full MS and MS/MS were 20 ms and 100 ms, respectively. The isolation width of precursor ion was set to 1.6 m/z. AGC for both MS and MS/MS were set at 3E6 and 5E4, respectively.</p> <p>For PRM experiments, the mass spectrometer was operated in PRM mode with MS1 scan from m/z 400 to 950 at 70,000 mass resolution following by 1 MS/MS acquisitions. The remaining parameters were set as same as DDA-based experiments. PRM data sets were acquired in a time scheduled mode and isolation lists for each performed method can be found in <b>Table S6</b>.</p> </div> <div class="dataanalysisofprm" title="sec"> <div class="title" tagx="title" title="title">
Data analysis of PRM</div> <p>For the identification of DDA data, raw data files were processed as the protocol described in Protein identification and quantification. For the acquired PRM data, all raw files were imported into the target quantitative software Skyline daily (v.3.5.1.9942). Spectral libraries were built using MS/MS search files from MaxQuant. The precursor of targeted peptide and the 3–10 most intense fragment ions were monitored and used for quantification. Peaks were manually checked for correct integration and the area under the curve of targeted peptide was obtained from the summed AUCs of each transition. The abundance of each peptide was normalized against the average of abundance of each protein. For the protein quantification, the mean of targeted peptide abundance was used to calculate fold changes between samples.</p> </div> <div class="bioinformaticsanalysis" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatics analysis</div> <p>The functional annotation of all the identified proteins was performed by using Blast2GO tool and the subcellular localization was analyzed by CELLO web tool [<a href="#cit0068">68</a>]. Conservation analysis of the <i>A. flavus</i> proteome was carried out by using reciprocal BLAST [<a href="#cit0069">69</a>]. Sequence alignment was also performed using ClustalX2 [<a href="#cit0070">70</a>] and visualized by CLC sequence viewer (<a href="http://www.qiagenbioinformatics.com/product-downloads/">http://www.qiagenbioinformatics.com/product-downloads/</a>). Domain structures were made by DOG software [<a href="#cit0071">71</a>] and a neighbor-joining tree was constructed by using MEGA 5.0 software [<a href="#cit0072">72</a>]. The transcripts were retrieved from NCBI (SRX237459, SRX237295, SRX1330586, SRX396791). For RNA-seq data analysis, the retrieved low-quality RNA reads were filtered and then mapped to the <i>A. flavus</i> genome using TopHat (version 2.1.1) [<a href="#cit0073">73</a>]. R scripts and Excel were used for the statistical analyses. Statistical significance was analyzed by Student’s t-tests and expressed as a p value. <i>p</i> &amp;lt; 0.05 was considered to be statistically significant.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>All data represent results from at least three independent experiments. Statistical analysis was performed to assess differences between different experiments. Student’s <i>t</i>-test was used to analyze the statistical significance. <i>p</i> &amp;lt; 0.05 was considered to be statistically significant. One asterisk and two asterisks indicate <i>p</i> &amp;lt; 0.05 and <i>p</i> &amp;lt; 0.01, respectively.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="KVIR_A_1859820_SM9268.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="KVIR_A_1859820_SM9258.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="KVIR_A_1859820_SM9228.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="KVIR_A_1859820_SM9223.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="KVIR_A_1859820_SM9219.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="KVIR_A_1859820_SM9218.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="KVIR_A_1859820_SM9211.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="KVIR_A_1859820_SM9208.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="KVIR_A_1859820_SM9207.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>This work was supported by the National Key Research and Development Program of China (2016YFA0501304), National Natural Science Foundation of China (Grant No. 31772105 and Grant No. 31601069) and CAS Key Technology Talent Program (to M.K.Y). The authors would like to thank the Analysis and Testing Center of Institute of Hydrobiology and the Wuhan Branch, Supercomputing Center, Chinese Academy of Sciences.</p> </div> <div class="authors’contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p>S.H.W., F.G. and M.K.Y. conceived and designed the study. M.K.Y., Z.Z. performed the proteomics experiments. M.K.Y., Z.Z., Y.H.B. M.Z.L and F.G. analyzed and interpreted the data. The paper was written by M.K.Y., F.G. and S.H.W. and was edited by all authors.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>The authors have declared that no competing interests exist.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>All of the raw MS data, TMT-based quantitative proteomic results and the results of PRM validation were uploaded to the public access iProX database (<a href="http://www.iprox.org">http://www.iprox.org</a>) with the identifier IPX0001753000.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary material</div> <p>Supplemental data for this article can be accessed <a href="https://doi.org/10.1080/21505594.2020.1859820">here</a>.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="cit0001" /><span class="label" tagx="label" title="label">[1]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Denning</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Follansbee</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scolaro</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Pulmonary aspergillosis in the acquired immunodeficiency syndrome</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="year" tagx="year" title="year">1991</span>;<span class="volume" tagx="volume" title="volume">324</span>:<span class="fpage" tagx="fpage" title="fpage">654</span>–<span class="lpage" tagx="lpage" title="lpage">662</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1994248">1994248</a></span></span></li> <li tag="ref"><a name="cit0002" /><span class="label" tagx="label" title="label">[2]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleveland</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nierman</span><span class="given-names" tagx="given-names" title="given-names">WC</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Aspergillus flavus genomics: gateway to human and animal health, food safety, and crop resistance to diseases</span>. <span class="source" tagx="source" title="source">Rev Iberoam Micología</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">22</span>:<span class="fpage" tagx="fpage" title="fpage">194</span>–<span class="lpage" tagx="lpage" title="lpage">202</span>.</span></li> <li tag="ref"><a name="cit0003" /><span class="label" tagx="label" title="label">[3]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banziger</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Workgroup report: public health strategies for reducing aflatoxin exposure in developing countries</span>. <span class="source" tagx="source" title="source">Environ Health Perspect</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">114</span>:<span class="fpage" tagx="fpage" title="fpage">1898</span>–<span class="lpage" tagx="lpage" title="lpage">1903</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17185282">17185282</a></span></span></li> <li tag="ref"><a name="cit0004" /><span class="label" tagx="label" title="label">[4]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: Authority EFS</span></span><span class="mixed-article-title" title="mixed-article-title">Aflatoxins (sum of B1, B2, G1, G2) in cereals and cereal-derived food products</span>. <span class="source" tagx="source" title="source">Efsa Supporting Publications</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">10</span>:<span class="page-range" tagx="page-range" title="page-range">n/a-n/a</span>.</span></li> <li tag="ref"><a name="cit0005" /><span class="label" tagx="label" title="label">[5]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roze</span><span class="given-names" tagx="given-names" title="given-names">LV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linz</span><span class="given-names" tagx="given-names" title="given-names">JE.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Aflatoxin biosynthesis: current frontiers</span>. <span class="source" tagx="source" title="source">Annu Rev Food Sci Technol</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">293</span>–<span class="lpage" tagx="lpage" title="lpage">311</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23244396">23244396</a></span></span></li> <li tag="ref"><a name="cit0006" /><span class="label" tagx="label" title="label">[6]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiseman</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buchanan</span><span class="given-names" tagx="given-names" title="given-names">RL.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Determination of glucose level needed to induce aflatoxin production in aspergillus-parasiticus</span>. <span class="source" tagx="source" title="source">Can J Microbiol</span>. <span class="year" tagx="year" title="year">1987</span>;<span class="volume" tagx="volume" title="volume">33</span>:<span class="fpage" tagx="fpage" title="fpage">828</span>–<span class="lpage" tagx="lpage" title="lpage">830</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3690425">3690425</a></span></span></li> <li tag="ref"><a name="cit0007" /><span class="label" tagx="label" title="label">[7]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buchanan</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stahl</span><span class="given-names" tagx="given-names" title="given-names">HG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ability of various carbon-sources to induce and support aflatoxin synthesis by aspergillus-parasiticus</span>. <span class="source" tagx="source" title="source">J Food Saf</span>. <span class="year" tagx="year" title="year">1984</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">271</span>–<span class="lpage" tagx="lpage" title="lpage">279</span>.</span></li> <li tag="ref"><a name="cit0008" /><span class="label" tagx="label" title="label">[8]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gourama</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bullerman</span><span class="given-names" tagx="given-names" title="given-names">LB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of growth and aflatoxin production of aspergillus-flavus by lactobacillus species</span>. <span class="source" tagx="source" title="source">J Food Protect</span>. <span class="year" tagx="year" title="year">1995</span>;<span class="volume" tagx="volume" title="volume">58</span>:<span class="fpage" tagx="fpage" title="fpage">1249</span>–<span class="lpage" tagx="lpage" title="lpage">1256</span>.</span></li> <li tag="ref"><a name="cit0009" /><span class="label" tagx="label" title="label">[9]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gourama</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bullerman</span><span class="given-names" tagx="given-names" title="given-names">LB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Aspergillus flavus and Aspergillus parasiticus: aflatoxigenic fungi of concern in foods and feeds: a review</span>. <span class="source" tagx="source" title="source">J Food Protect</span>. <span class="year" tagx="year" title="year">1995</span>;<span class="volume" tagx="volume" title="volume">58</span>:<span class="fpage" tagx="fpage" title="fpage">1395</span>–<span class="lpage" tagx="lpage" title="lpage">1404</span>.</span></li> <li tag="ref"><a name="cit0010" /><span class="label" tagx="label" title="label">[10]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klich</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Environmental and developmental factors influencing aflatoxin production by Aspergillus flavus and Aspergillus parasiticus</span>. <span class="source" tagx="source" title="source">Mycoscience</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">48</span>:<span class="fpage" tagx="fpage" title="fpage">71</span>–<span class="lpage" tagx="lpage" title="lpage">80</span>.</span></li> <li tag="ref"><a name="cit0011" /><span class="label" tagx="label" title="label">[11]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veses</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richards</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gow</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vacuoles and fungal biology</span>. <span class="source" tagx="source" title="source">Curr Opin Microbiol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">503</span>–<span class="lpage" tagx="lpage" title="lpage">510</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18935977">18935977</a></span></span></li> <li tag="ref"><a name="cit0012" /><span class="label" tagx="label" title="label">[12]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">A novel role of the vacuolar calcium channel Yvc1 in stress response, morphogenesis and pathogenicity of Candida albicans</span>. <span class="source" tagx="source" title="source">Int J Med Microbiol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">304</span>:<span class="fpage" tagx="fpage" title="fpage">339</span>–<span class="lpage" tagx="lpage" title="lpage">350</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24368068">24368068</a></span></span></li> <li tag="ref"><a name="cit0013" /><span class="label" tagx="label" title="label">[13]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rees</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiele</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">From aging to virulence: forging connections through the study of copper homeostasis in eukaryotic microorganisms</span>. <span class="source" tagx="source" title="source">Curr Opin Microbiol</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">175</span>–<span class="lpage" tagx="lpage" title="lpage">184</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15063856">15063856</a></span></span></li> <li tag="ref"><a name="cit0014" /><span class="label" tagx="label" title="label">[14]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilty</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smulian</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newman</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Histoplasma capsulatum vacuolar ATPase is required for iron homeostasis, intracellular replication in macrophages and virulence in a murine model of histoplasmosis</span>. <span class="source" tagx="source" title="source">Mol Microbiol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">70</span>:<span class="fpage" tagx="fpage" title="fpage">127</span>–<span class="lpage" tagx="lpage" title="lpage">139</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18699866">18699866</a></span></span></li> <li tag="ref"><a name="cit0015" /><span class="label" tagx="label" title="label">[15]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veses</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richards</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gow</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vacuole inheritance regulates cell size and branching frequency of Candida albicans hyphae</span>. <span class="source" tagx="source" title="source">Mol Microbiol</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">71</span>:<span class="fpage" tagx="fpage" title="fpage">505</span>–<span class="lpage" tagx="lpage" title="lpage">519</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19040629">19040629</a></span></span></li> <li tag="ref"><a name="cit0016" /><span class="label" tagx="label" title="label">[16]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boyce</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kretschmer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kronstad</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span></span><span class="mixed-article-title" title="mixed-article-title">The vtc4 gene influences polyphosphate storage, morphogenesis, and virulence in the maize pathogen Ustilago maydis</span>. <span class="source" tagx="source" title="source">Eukaryot Cell</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">1399</span>–<span class="lpage" tagx="lpage" title="lpage">1409</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16896223">16896223</a></span></span></li> <li tag="ref"><a name="cit0017" /><span class="label" tagx="label" title="label">[17]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chanda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roze</span><span class="given-names" tagx="given-names" title="given-names">LV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">A key role for vesicles in fungal secondary metabolism</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">106</span>:<span class="fpage" tagx="fpage" title="fpage">19533</span>–<span class="lpage" tagx="lpage" title="lpage">19538</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19889978">19889978</a></span></span></li> <li tag="ref"><a name="cit0018" /><span class="label" tagx="label" title="label">[18]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dove</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Phosphatidylinositol 3,5-bisphosphate and Fab1p/PIKfyve underPPIn endo-lysosome function</span>. <span class="source" tagx="source" title="source">Biochem J</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">419</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">13</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19272020">19272020</a></span></span></li> <li tag="ref"><a name="cit0019" /><span class="label" tagx="label" title="label">[19]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michell</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heath</span><span class="given-names" tagx="given-names" title="given-names">VL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemmon</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Phosphatidylinositol 3,5-bisphosphate: metabolism and cellular functions</span>. <span class="source" tagx="source" title="source">Trends Biochem Sci</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">31</span>:<span class="fpage" tagx="fpage" title="fpage">52</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16364647">16364647</a></span></span></li> <li tag="ref"><a name="cit0020" /><span class="label" tagx="label" title="label">[20]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicot</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fares</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Payrastre</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">The phosphoinositide kinase PIKfyve/Fab1p regulates terminal lysosome maturation in Caenorhabditis elegans</span>. <span class="source" tagx="source" title="source">Mol Biol Cell</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">17</span>:<span class="fpage" tagx="fpage" title="fpage">3062</span>–<span class="lpage" tagx="lpage" title="lpage">3074</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16801682">16801682</a></span></span></li> <li tag="ref"><a name="cit0021" /><span class="label" tagx="label" title="label">[21]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishihara</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shibasaki</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kizuki</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Type I phosphatidylinositol-4-phosphate 5-kinases - Cloning of the third isoform and deletion/substitution analysis of members of this novel lipid kinase family</span>. <span class="source" tagx="source" title="source">J Biol Chem</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">273</span>:<span class="fpage" tagx="fpage" title="fpage">8741</span>–<span class="lpage" tagx="lpage" title="lpage">8748</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9535851">9535851</a></span></span></li> <li tag="ref"><a name="cit0022" /><span class="label" tagx="label" title="label">[22]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lang</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strunk</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azad</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">An intramolecular interaction within the lipid kinase Fab1 regulates cellular phosphatidylinositol 3,5-bisphosphate lipid levels</span>. <span class="source" tagx="source" title="source">Mol Biol Cell</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">28</span>:<span class="fpage" tagx="fpage" title="fpage">858</span>–<span class="lpage" tagx="lpage" title="lpage">864</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28148651">28148651</a></span></span></li> <li tag="ref"><a name="cit0023" /><span class="label" tagx="label" title="label">[23]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choudhary</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhakt</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaur</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Essential role for the phosphatidylinositol 3,5-bisphosphate synthesis complex in caspofungin tolerance and virulence in Candida glabrata</span>. <span class="source" tagx="source" title="source">Antimicrob Agents Chemother</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">63:e00886–19</span>.</span></li> <li tag="ref"><a name="cit0024" /><span class="label" tagx="label" title="label">[24]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boylan</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Timberlake</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span></span><span class="mixed-article-title" title="mixed-article-title">brlA is necessary and sufficient to direct conidiophore development in Aspergillus nidulans</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">1988</span>;<span class="volume" tagx="volume" title="volume">54</span>:<span class="fpage" tagx="fpage" title="fpage">353</span>–<span class="lpage" tagx="lpage" title="lpage">362</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3293800">3293800</a></span></span></li> <li tag="ref"><a name="cit0025" /><span class="label" tagx="label" title="label">[25]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sewall</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mims</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Timberlake</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span></span><span class="mixed-article-title" title="mixed-article-title">abaA controls phialide differentiation in Aspergillus nidulans</span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">1990</span>;<span class="volume" tagx="volume" title="volume">2</span>:<span class="fpage" tagx="fpage" title="fpage">731</span>–<span class="lpage" tagx="lpage" title="lpage">739</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2152124">2152124</a></span></span></li> <li tag="ref"><a name="cit0026" /><span class="label" tagx="label" title="label">[26]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bridges</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Phosphatidylinositol 3,5-bisphosphate plays a role in the activation and subcellular localization of mechanistic target of rapamycin 1</span>. <span class="source" tagx="source" title="source">Mol Biol Cell</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">23</span>:<span class="fpage" tagx="fpage" title="fpage">2955</span>–<span class="lpage" tagx="lpage" title="lpage">2962</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22696681">22696681</a></span></span></li> <li tag="ref"><a name="cit0027" /><span class="label" tagx="label" title="label">[27]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Natsuko</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kai</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yui</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Roles for PI(3,5)P2 in nutrient sensing through TORC1</span>. <span class="source" tagx="source" title="source">Mol Biol Cell</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">25</span>:<span class="fpage" tagx="fpage" title="fpage">1171</span>–<span class="lpage" tagx="lpage" title="lpage">1185</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24478451">24478451</a></span></span></li> <li tag="ref"><a name="cit0028" /><span class="label" tagx="label" title="label">[28]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dolezal</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">OBrian</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Aspergillus flavus infection induces transcriptional and physical changes in developing maize kernels</span>. <span class="source" tagx="source" title="source">Front Microbiol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">384</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25132833">25132833</a></span></span></li> <li tag="ref"><a name="cit0029" /><span class="label" tagx="label" title="label">[29]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaike</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keller</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Aspergillus flavus</span>. <span class="source" tagx="source" title="source">Annu Rev Phytopathol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">49</span>:<span class="fpage" tagx="fpage" title="fpage">107</span>–<span class="lpage" tagx="lpage" title="lpage">133</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21513456">21513456</a></span></span></li> <li tag="ref"><a name="cit0030" /><span class="label" tagx="label" title="label">[30]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Groopman</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="mixed-article-title" title="mixed-article-title">DNA damage by mycotoxins</span>. <span class="source" tagx="source" title="source">Mutat Res</span>. <span class="year" tagx="year" title="year">1999</span>;<span class="volume" tagx="volume" title="volume">424</span>:<span class="fpage" tagx="fpage" title="fpage">167</span>–<span class="lpage" tagx="lpage" title="lpage">181</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10064859">10064859</a></span></span></li> <li tag="ref"><a name="cit0031" /><span class="label" tagx="label" title="label">[31]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">LW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiou</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klomparens</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Subcellular localization of aflatoxin biosynthetic enzymes Nor-1, Ver-1, and OmtA in time-dependent fractionated colonies of Aspergillus parasiticus</span>. <span class="source" tagx="source" title="source">Arch Microbiol</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">181</span>:<span class="fpage" tagx="fpage" title="fpage">204</span>–<span class="lpage" tagx="lpage" title="lpage">214</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14722624">14722624</a></span></span></li> <li tag="ref"><a name="cit0032" /><span class="label" tagx="label" title="label">[32]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chanda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roze</span><span class="given-names" tagx="given-names" title="given-names">LV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linz</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Purification of a vesicle-vacuole (V) fraction from Aspergillus</span>. <span class="source" tagx="source" title="source">Methods Mol Biol</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">944</span>:<span class="fpage" tagx="fpage" title="fpage">259</span>–<span class="lpage" tagx="lpage" title="lpage">266</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23065623">23065623</a></span></span></li> <li tag="ref"><a name="cit0033" /><span class="label" tagx="label" title="label">[33]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gary</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wurmser</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonangelino</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Fab1p is essential for PtdIns(3)P 5-kinase activity and the maintenance of vacuolar size and membrane homeostasis</span>. <span class="source" tagx="source" title="source">J Cell Biol</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">143</span>:<span class="fpage" tagx="fpage" title="fpage">65</span>–<span class="lpage" tagx="lpage" title="lpage">79</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9763421">9763421</a></span></span></li> <li tag="ref"><a name="cit0034" /><span class="label" tagx="label" title="label">[34]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veses</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richards</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gow</span><span class="given-names" tagx="given-names" title="given-names">NAR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vacuoles and fungal biology</span>. <span class="source" tagx="source" title="source">Curr Opin Microbiol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">503</span>–<span class="lpage" tagx="lpage" title="lpage">510</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18935977">18935977</a></span></span></li> <li tag="ref"><a name="cit0035" /><span class="label" tagx="label" title="label">[35]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ying</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Na+/H+ antiporter Nhx1 controls vacuolar fusion indispensible for life cycles in vitro and in vivo in a fungal insect pathogen</span>. <span class="source" tagx="source" title="source">Environ Microbiol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">18</span>:<span class="fpage" tagx="fpage" title="fpage">3884</span>–<span class="lpage" tagx="lpage" title="lpage">3895</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27112237">27112237</a></span></span></li> <li tag="ref"><a name="cit0036" /><span class="label" tagx="label" title="label">[36]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mouhoumed</span><span class="given-names" tagx="given-names" title="given-names">AZ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mou</span><span class="given-names" tagx="given-names" title="given-names">YN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Three proline rotamases involved in calcium homeostasis play differential roles in stress tolerance, virulence and calcineurin regulation of Beauveria bassiana</span>. <span class="source" tagx="source" title="source">Cell Microbiol</span>. <span class="year" tagx="year" title="year">2020;22:e13239</span>.</span></li> <li tag="ref"><a name="cit0037" /><span class="label" tagx="label" title="label">[37]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meier</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovalchuk</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Properties of the new fluorescent Na+ indicator CoroNa Green: comparison with SBFI and confocal Na+ imaging</span>. <span class="source" tagx="source" title="source">J Neurosci Meth</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">155</span>:<span class="fpage" tagx="fpage" title="fpage">251</span>–<span class="lpage" tagx="lpage" title="lpage">259</span>.</span></li> <li tag="ref"><a name="cit0038" /><span class="label" tagx="label" title="label">[38]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindberg</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">In-situ determination of intracellular concentrations of K+ in Barley (Hordeum-Vulgare L Cv Kara) Using the K+-binding fluorescent dye benzofuran isophthalate</span>. <span class="source" tagx="source" title="source">Planta</span>. <span class="year" tagx="year" title="year">1995</span>;<span class="volume" tagx="volume" title="volume">195</span>:<span class="fpage" tagx="fpage" title="fpage">525</span>–<span class="lpage" tagx="lpage" title="lpage">529</span>.</span></li> <li tag="ref"><a name="cit0039" /><span class="label" tagx="label" title="label">[39]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasner</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ganz</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation of intracellular potassium in mesangial cells - a fluorescence analysis using the dye, Pbfi</span>. <span class="source" tagx="source" title="source">Am J Physiol</span>. <span class="year" tagx="year" title="year">1992</span>;<span class="volume" tagx="volume" title="volume">262</span>:<span class="fpage" tagx="fpage" title="fpage">F462</span>–<span class="lpage" tagx="lpage" title="lpage">F7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1558163">1558163</a></span></span></li> <li tag="ref"><a name="cit0040" /><span class="label" tagx="label" title="label">[40]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, et al</span><span class="source" tagx="source" title="source">Plant prevacuolar compartments and endocytosis</span>; <span class="year" tagx="year" title="year">2006</span>.</span></li> <li tag="ref"><a name="cit0041" /><span class="label" tagx="label" title="label">[41]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paris</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debeaupuis</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crameri</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Conidial hydrophobins of aspergillus fumigatus</span>. <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>. <span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">69</span>:<span class="fpage" tagx="fpage" title="fpage">1581</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12620846">12620846</a></span></span></li> <li tag="ref"><a name="cit0042" /><span class="label" tagx="label" title="label">[42]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Etxebeste</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrero-García</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cortese</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">GmcA is a putative glucose-methanol-choline oxidoreductase required for the induction of asexual development in aspergillus nidulans</span>. <span class="source" tagx="source" title="source">Plos One</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">e40292</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22792266">22792266</a></span></span></li> <li tag="ref"><a name="cit0043" /><span class="label" tagx="label" title="label">[43]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Navarro</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stringer</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansberg</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">catA, a new Aspergillus nidulans gene encoding a developmentally regulated catalase</span>. <span class="source" tagx="source" title="source">Curr Genet</span>. <span class="year" tagx="year" title="year">1996</span>;<span class="volume" tagx="volume" title="volume">29</span>:<span class="fpage" tagx="fpage" title="fpage">352</span>–<span class="lpage" tagx="lpage" title="lpage">359</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8598056">8598056</a></span></span></li> <li tag="ref"><a name="cit0044" /><span class="label" tagx="label" title="label">[44]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oh</span><span class="given-names" tagx="given-names" title="given-names">YT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahn</span><span class="given-names" tagx="given-names" title="given-names">C-S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeong</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Aspergillus nidulans translationally controlled tumor protein has a role in the balance between asexual and sexual differentiation and normal hyphal branching</span>. <span class="source" tagx="source" title="source">Fems Microbiol Lett</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">343</span>:<span class="fpage" tagx="fpage" title="fpage">20</span>–<span class="lpage" tagx="lpage" title="lpage">25</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23480775">23480775</a></span></span></li> <li tag="ref"><a name="cit0045" /><span class="label" tagx="label" title="label">[45]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraskevi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lockington</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelly</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">The Spt-Ada-Gcn5 acetyltransferase (SAGA) complex in aspergillus nidulans</span>. <span class="source" tagx="source" title="source">Plos One</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">e65221</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23762321">23762321</a></span></span></li> <li tag="ref"><a name="cit0046" /><span class="label" tagx="label" title="label">[46]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Prediction of pathogenesis-related secreted proteins from Stemphylium lycopersici</span>. <span class="source" tagx="source" title="source">BMC Microbiol</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">18:191</span>.</span></li> <li tag="ref"><a name="cit0047" /><span class="label" tagx="label" title="label">[47]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Dongen</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niessen</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span></span><span class="mixed-article-title" title="mixed-article-title">LC-MS systems for quantitative bioanalysis</span>. <span class="source" tagx="source" title="source">Bioanalysis</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">2391</span>–<span class="lpage" tagx="lpage" title="lpage">2399</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23088465">23088465</a></span></span></li> <li tag="ref"><a name="cit0048" /><span class="label" tagx="label" title="label">[48]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Integrating MS1 and MS2 scans in high-resolution parallel reaction monitoring assays for targeted metabolite quantification and dynamic (13)C-labeling metabolism analysis</span>. <span class="source" tagx="source" title="source">Anal Chem</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">89</span>:<span class="fpage" tagx="fpage" title="fpage">877</span>–<span class="lpage" tagx="lpage" title="lpage">885</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27966897">27966897</a></span></span></li> <li tag="ref"><a name="cit0049" /><span class="label" tagx="label" title="label">[49]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dove</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooke</span><span class="given-names" tagx="given-names" title="given-names">FT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douglas</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Osmotic stress activates phosphatidylinositol-3,5-bisphosphate synthesis</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">1997</span>;<span class="volume" tagx="volume" title="volume">390</span>:<span class="fpage" tagx="fpage" title="fpage">187</span>–<span class="lpage" tagx="lpage" title="lpage">192</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9367158">9367158</a></span></span></li> <li tag="ref"><a name="cit0050" /><span class="label" tagx="label" title="label">[50]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zolov</span><span class="given-names" tagx="given-names" title="given-names">SN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dave</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yanling</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">In vivo, Pikfyve generates PI(3,5)P2, which serves as both a signaling lipid and the major precursor for PI5P</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">109</span>:<span class="fpage" tagx="fpage" title="fpage">17472</span>–<span class="lpage" tagx="lpage" title="lpage">17477</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23047693">23047693</a></span></span></li> <li tag="ref"><a name="cit0051" /><span class="label" tagx="label" title="label">[51]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mccartney</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zolov</span><span class="given-names" tagx="given-names" title="given-names">SN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kauffman</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Activity-dependent PI(3,5)P2 synthesis controls AMPA receptor trafficking during synaptic depression</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">111</span>:<span class="fpage" tagx="fpage" title="fpage">4896</span>–<span class="lpage" tagx="lpage" title="lpage">4905</span>.</span></li> <li tag="ref"><a name="cit0052" /><span class="label" tagx="label" title="label">[52]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bharadwaj</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cunningham</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">FIG4 regulates lysosome membrane homeostasis independent of phosphatase function</span>. <span class="source" tagx="source" title="source">Hum Mol Genet</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">25</span>:<span class="fpage" tagx="fpage" title="fpage">681</span>–<span class="lpage" tagx="lpage" title="lpage">692</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26662798">26662798</a></span></span></li> <li tag="ref"><a name="cit0053" /><span class="label" tagx="label" title="label">[53]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stein</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterfield</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span></span><span class="mixed-article-title" title="mixed-article-title">PI3-kinase inhibition: a target for drug development?</span><span class="source" tagx="source" title="source">Mol Med Today</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">347</span>–<span class="lpage" tagx="lpage" title="lpage">357</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10954868">10954868</a></span></span></li> <li tag="ref"><a name="cit0054" /><span class="label" tagx="label" title="label">[54]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lang</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strunk</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azad</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">An intramolecular interaction within the lipid kinase Fab1 regulates cellular phosphatidylinositol 3,5-bisphosphate lipid levels</span>. <span class="source" tagx="source" title="source">Mol Biol Cell</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">28</span>:<span class="fpage" tagx="fpage" title="fpage">858</span>–<span class="lpage" tagx="lpage" title="lpage">864</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28148651">28148651</a></span></span></li> <li tag="ref"><a name="cit0055" /><span class="label" tagx="label" title="label">[55]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirano</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diverse physiological functions of FAB1 and phosphatidylinositol 3,5-bisphosphate in plants</span>. <span class="source" tagx="source" title="source">Front Plant Sci</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">10</span>.</span></li> <li tag="ref"><a name="cit0056" /><span class="label" tagx="label" title="label">[56]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rusten</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodahl</span><span class="given-names" tagx="given-names" title="given-names">LMW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pattni</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Fab1 phosphatidylinositol 3-Phosphate 5-kinase controls trafficking but not silencing of endocytosed receptors</span>. <span class="source" tagx="source" title="source">Mol Biol Cell</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">17</span>:<span class="fpage" tagx="fpage" title="fpage">3989–4001</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16837550">16837550</a></span></span></li> <li tag="ref"><a name="cit0057" /><span class="label" tagx="label" title="label">[57]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikonomov</span><span class="given-names" tagx="given-names" title="given-names">OC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sbrissa</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delvecchio</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">The phosphoinositide kinase PIKfyve is vital in early embryonic development preimplantation lethality of Pikfyve(-/-) embryos but normality of Pikfyve(±) mice</span>. <span class="source" tagx="source" title="source">J Biol Chem</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">286</span>:<span class="fpage" tagx="fpage" title="fpage">13404</span>–<span class="lpage" tagx="lpage" title="lpage">13413</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21349843">21349843</a></span></span></li> <li tag="ref"><a name="cit0058" /><span class="label" tagx="label" title="label">[58]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takasuga</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horie</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasaki</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Critical roles of type III phosphatidylinositol phosphate kinase in murine embryonic visceral endoderm and adult intestine</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">110</span>:<span class="fpage" tagx="fpage" title="fpage">1726</span>–<span class="lpage" tagx="lpage" title="lpage">1731</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23322734">23322734</a></span></span></li> <li tag="ref"><a name="cit0059" /><span class="label" tagx="label" title="label">[59]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Augsten</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubner</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Defective hyphal induction of a Candida albicans phosphatidylinositol 3-phosphate 5-kinase null mutant on solid media does not lead to decreased virulence</span>. <span class="source" tagx="source" title="source">Infect Immun</span>. <span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">70</span>:<span class="fpage" tagx="fpage" title="fpage">4462</span>–<span class="lpage" tagx="lpage" title="lpage">4470</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12117957">12117957</a></span></span></li> <li tag="ref"><a name="cit0060" /><span class="label" tagx="label" title="label">[60]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sung-Yong</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linz</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functional expression and subcellular localization of the aflatoxin pathway enzyme Ver-1 fused to enhanced green fluorescent protein</span>. <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">74</span>:<span class="fpage" tagx="fpage" title="fpage">6385</span>–<span class="lpage" tagx="lpage" title="lpage">6396</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18757582">18757582</a></span></span></li> <li tag="ref"><a name="cit0061" /><span class="label" tagx="label" title="label">[61]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sung-Yong</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linz</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functional expression and sub-cellular localization of the early aflatoxin pathway enzyme Nor-1 in Aspergillus parasiticus</span>. <span class="source" tagx="source" title="source">Mycol Res</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">113</span>:<span class="fpage" tagx="fpage" title="fpage">591</span>–<span class="lpage" tagx="lpage" title="lpage">601</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19217941">19217941</a></span></span></li> <li tag="ref"><a name="cit0062" /><span class="label" tagx="label" title="label">[62]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">ZJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">HH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">XG</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Discovery of a new intravacuolar protein required for the autophagy, development and virulence of Beauveria bassiana</span>. <span class="source" tagx="source" title="source">Environ Microbiol</span>. <span class="year" tagx="year" title="year">2017;19:2806-2818</span>.</span></li> <li tag="ref"><a name="cit0063" /><span class="label" tagx="label" title="label">[63]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">XH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Genome annotation of a model diatom phaeodactylum tricornutum using an integrated proteogenomic pipeline</span>. <span class="source" tagx="source" title="source">Mol Plant</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">1292</span>–<span class="lpage" tagx="lpage" title="lpage">1307</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30176371">30176371</a></span></span></li> <li tag="ref"><a name="cit0064" /><span class="label" tagx="label" title="label">[64]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scharfenstein</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Development and refinement of a high-efficiency gene-targeting system for Aspergillus flavus</span>. <span class="source" tagx="source" title="source">J Microbiol Methods</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">81</span>:<span class="fpage" tagx="fpage" title="fpage">240</span>–<span class="lpage" tagx="lpage" title="lpage">246</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20298723">20298723</a></span></span></li> <li tag="ref"><a name="cit0065" /><span class="label" tagx="label" title="label">[65]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Adenylate cyclase AcyA regulates development, aflatoxin biosynthesis and fungal virulence in aspergillus flavus</span>. <span class="source" tagx="source" title="source">Front Cell Infect Microbiol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">190</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28066725">28066725</a></span></span></li> <li tag="ref"><a name="cit0066" /><span class="label" tagx="label" title="label">[66]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diakov</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tarsio</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kane</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Measurement of vacuolar and cytosolic ph in vivo in yeast cell suspensions</span>. <span class="source" tagx="source" title="source">Jove-J Vis Exp</span>. <span class="year" tagx="year" title="year">2013;19:50261</span>.</span></li> <li tag="ref"><a name="cit0067" /><span class="label" tagx="label" title="label">[67]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Proteogenomic analysis and global discovery of posttranslational modifications in prokaryotes</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">111</span>:<span class="fpage" tagx="fpage" title="fpage">E5633</span>–<span class="lpage" tagx="lpage" title="lpage">E42</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25512518">25512518</a></span></span></li> <li tag="ref"><a name="cit0068" /><span class="label" tagx="label" title="label">[68]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hwang</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Predicting subcellular localization of proteins for Gram-negative bacteria by support vector machines based on n-peptide compositions</span>. <span class="source" tagx="source" title="source">Protein Sci</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">1402</span>–<span class="lpage" tagx="lpage" title="lpage">1406</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15096640">15096640</a></span></span></li> <li tag="ref"><a name="cit0069" /><span class="label" tagx="label" title="label">[69]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rastogi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maheswari</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dorrell</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Integrative analysis of large scale transcriptome data draws a comprehensive landscape of Phaeodactylum tricornutum genome and evolutionary origin of diatoms</span>. <span class="source" tagx="source" title="source">Sci Rep</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">8:4834</span>.</span></li> <li tag="ref"><a name="cit0070" /><span class="label" tagx="label" title="label">[70]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larkin</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blackshields</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">Clustal W and clustal X version 2.0</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">23</span>:<span class="fpage" tagx="fpage" title="fpage">2947</span>–<span class="lpage" tagx="lpage" title="lpage">2948</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036">17846036</a></span></span></li> <li tag="ref"><a name="cit0071" /><span class="label" tagx="label" title="label">[71]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">DOG 1.0: illustrator of protein domain structures</span>. <span class="source" tagx="source" title="source">Cell Res</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">271</span>–<span class="lpage" tagx="lpage" title="lpage">273</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19153597">19153597</a></span></span></li> <li tag="ref"><a name="cit0072" /><span class="label" tagx="label" title="label">[72]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">28</span>:<span class="fpage" tagx="fpage" title="fpage">2731</span>–<span class="lpage" tagx="lpage" title="lpage">2739</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353">21546353</a></span></span></li> <li tag="ref"><a name="cit0073" /><span class="label" tagx="label" title="label">[73]</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pertea</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trapnell</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, et al</span><span class="mixed-article-title" title="mixed-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions</span>. <span class="source" tagx="source" title="source">Genome Biol</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">14:R36</span>.</span></li> </ul> </div> </div>  </body></html>