<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Beni Suef Univ J Basic Appl Sci</span><span class="iso-abbrev" title="iso-abbrev">Beni Suef Univ J Basic Appl Sci</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Beni-Suef University Journal of Basic and Applied Sciences</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">2314-8535</span><span class="issn-epub" tagx="issn" title="issn-epub">2314-8543</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer Berlin Heidelberg</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin/Heidelberg</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7841026">7841026</a></span><span class="publisher-id" title="publisher-id">95</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s43088-021-00095-x">10.1186/s43088-021-00095-x</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Virtual screening of phytochemical compounds as potential inhibitors against SARS-CoV-2 infection</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-8104-9046</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kothandan</span><span class="given-names" tagx="given-names" title="given-names">Ram</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">ram.k.bt@kct.ac.in</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajan</span><span class="given-names" tagx="given-names" title="given-names">Cashlin Anna Suveetha Gnana</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arjun</span><span class="given-names" tagx="given-names" title="given-names">Janamitra</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raj</span><span class="given-names" tagx="given-names" title="given-names">Rejoe Raymond Michael</span></span><a href="#Aff1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Syed</span><span class="given-names" tagx="given-names" title="given-names">Sowfia</span></span><a href="#Aff1" /></span><span class="citation_author_institution" id="Aff1">[], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.252262.3</span><span class="institution-id" title="institution-id">0000 0001 0613 6919</span><span class="institution" title="institution">Bioinformatics Laboratory, Department of Biotechnology, </span><span class="institution" title="institution">Kumaraguru College of Technology, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-1-1</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-1-1</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2021</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">1</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">9</span><span class="history" title="history"><span class="received" title="received">received: 2020-9-2</span><span class="accepted" title="accepted">accepted: 2021-1-4</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1">The present pandemic situation due to coronavirus has led to the search for newer prevention, diagnostic, and treatment methods. The onset of the corona infection in a human results in acute respiratory illness followed by death if not diagnosed and treated with suitable antiretroviral drugs. With the unavailability of the targeted drug treatment, several repurposed drugs are being used for treatment. However, the side-effects of the drugs urges us to move to a search for newer synthetic- or phytochemical-based drugs. The present study investigates the use of various phytochemicals virtually screened from various plant sources in Western Ghats, India, and subsequently molecular docking studies were performed to identify the efficacy of the drug in retroviral infection particularly coronavirus infection.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par2">Out of 57 phytochemicals screened initially based on the structural and physicochemical properties, 39 were effectively used for the docking analysis. Finally, 5 lead compounds with highest hydrophobic interaction and number of H-bonds were screened. Results from the interaction analysis suggest Piperolactam A to be pocketed well with good hydrophobic interaction with the residues in the binding region R1. ADME and toxicity profiling also reveals Piperolactam A with higher LogS values indicating higher permeation and hydrophilicity. Toxicity profiling suggests that the 5 screened compounds to be relatively safe.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par3">The in silico methods used in this study suggests that the compound Piperolactam A to be the most effective inhibitor of S-protein from binding to the GRP78 receptor. By blocking the binding of the S-protein to the CS-GRP78 cell surface receptor, they can inhibit the binding of the virus to the host.</p> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>The online version contains supplementary material available at 10.1186/s43088-021-00095-x.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
In silico drug design</div> <div class="kwd" title="kwd">
Virtual screening</div> <div class="kwd" title="kwd">
Retroviral infection</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par12">COVID-19, an infectious disease caused by Severe Acute Respiratory Syndrome Corona Virus-2 (SARS-CoV-2) has become an unexpected threat to the human population. With the exponential raising of the infection by followed increasing mortality rate day by day, the World Health Organization ((WHO)) has reported active cases 14,348,858 and death of 603,691 humans on 20th July 2020 [<a href="#CR1">1</a>]. The threat has forced a major responsibility to scientific society in the search for new diagnostic methods, treatment, and preventive solutions. Existing repurposed drugs have been neglected by the medical community due to associated side effects [<a href="#CR2">2</a>].</p> <p id="Par13">CoV belongs to the family <i>Coronaviridae</i> with a large RNA genome of 30 kb, positive-sensed and non-segmented [<a href="#CR3">3</a>]. The virion structure contains four parts of protein which are Spike proteins (S-proteins), membrane proteins (M-proteins), envelope proteins (E-proteins), and nucleocapsid proteins (N-proteins). Of these S-proteins play a major role in transfer of virion particles from the virus to the host cells using ACE2 receptor [<a href="#CR4">4</a>]. Although various literature has highlighted the importance of other proteins; the S-protein is currently well studied and has been as a potent drug target [<a href="#CR5">5</a>]. The S-protein exhibit in a homo-trimeric state with N-terminal glycosylation in which the S-polypeptide chain cleaved into two subunits S1 and S2 (Fig. <a href="#Fig1">1</a>)Â [<a href="#CR3">3</a>]. The S1 subunit with the larger receptor binding domain and the S2 subunit which induces the membrane fusion polypeptides forming the six helix bundle which results in complete fusion and insertion of viral genome to the host cell cytosol [<a href="#CR6">6</a>]. Hence, structural analysis on the S-protein could reveal possible mechanisms by which the viral replication could be terminated/controlled [<a href="#CR7">7</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>3D structure of 6X6P (Spike Protein, Trimeric structure) with chain AÂ (Red), chain B (green), and chain C (magenta). Binding region 1 with its amino acids is highlighted</p>   </div> <p /> <p id="Par14">Till date, many studies are involved in targeting ACE2 receptor (angiotensin converting enzyme-2), an amino peptidase which found to act as a cellular gateway for the entry of viral RNA into the cells using the S-proteins present in the surface of the virus [<a href="#CR5">5</a>, <a href="#CR8">8</a>]. But ACE2 receptor is not only the gateway present on cell membrane to carry this virion into the cells, there are other cellular surface proteins which could also take this role. One such protein is glucose regulated protein 78 (GRP78) residing in lumen of endoplasmic reticulum (ER), a heat shock protein with a molecular weight of 78 kDa and a molecular chaperone present in all eukaryotic cells. The major role of GRP78 is protein folding, unfolding, and resisting aggregation of proteins in cytosol [<a href="#CR9">9</a>]. The ER-protein when overexpressed escape the KDEL motif receptors which responsible for retention in ER due to its saturation or down regulation and get translocated in the cell membrane and act as cell surface GRP78 receptor (CS-GRP78 receptor) which pose the potential threat to our system for viral entry. Hypoxia, glucose starvation, and tumor causes ER stress which thus upregulate the GRP78 genes and result in over production of GRP78 [<a href="#CR10">10</a>]. Misfolded protein accumulation also results in overexpression of this protein mostly reported in neurodegenerative disorders and conditions like Alzheimer, Parkinsonâ€™s disease, and prion protein disease [<a href="#CR11">11</a>].</p> <p id="Par15">CS-GRP78 acts as a multifunctional receptor and binds plenty of proteins and other compounds and activates several pathways which have negative impact on the cells like apoptosis [<a href="#CR12">12</a>]. Several studies reveal that this protein has been responsible for many types of viral entry. In entry of CoxsackieVirus A9 CS-GRP78 involved as the co-receptor along with MHC Class 1 molecule on cell surface [<a href="#CR13">13</a>]. GRP78 studies reported an important host cell factor in the entry of Japanese Encephalitis Virus into the host cells [<a href="#CR14">14</a>]. Studies report that along with Dipeptidyl-peptidase 4 (DPP4), CS-GRP78 involved in the entry of MERS-CoV and reported that introduction of MERS-CoV into cells upregulate the GRP78 genes. Upregulation of GRP78 genes results in overproduction of GRP78 which help in both viral entry and development [<a href="#CR15">15</a>]. Conditions in which the primary receptor expression in host is low CS-GRP78 act as the main alternate door for the viral entry. Thus, inhibiting or reducing CS-GRP78 mediated entry of SARS-CoV-2 is essentially equivalent to inhibiting the ACE2 mediated receptor entry [<a href="#CR16">16</a>]. Protein-protein docking studies established the possible binding site where these two proteins could form complex with good HADDOCK Score, PRODIGY Binding affinity, H-bonds, and hydrophobic interaction. It would be a good approach, if we could block the region of S-protein which is responsible for binding to the CS-GRP78 receptor.</p> <p id="Par16">Considering the pandemic situation, several health organization, and nations in the aim of developing antiviral agents are repurposing drugs like Lopinavir (HIV), Hydroxychloroquinone (Malaria) against COVID [<a href="#CR17">17</a>, <a href="#CR18">18</a>]. In general, repurposed pose variable target mechanisms and lack effectiveness for coronavirus strain. On the other hand, antiviral drugs also exhibit adverse side effects, which directly and indirectly affect human health [<a href="#CR19">19</a>]. To overcome this shortfall, the development of plant-based drugs and treatment strategies with minimal side-effects are expected [<a href="#CR20">20</a>]. Potentially phytochemicals, which serves as an infinite resource for drug development and novel pharmacophore may be benefited as a therapeutic agent against corona viruses. Their therapeutic applications against diverse viruses can be explained by various antiviral mechanisms such as inhibition of replication process [<a href="#CR21">21</a>] and blocking the binding of virus to the host [<a href="#CR22">22</a>].</p> <p id="Par17">Ancient Indian traditional techniques used several plant parts to treat various infections and disease conditions [<a href="#CR23">23</a>]. Plant parts used in the treatment was experimentally proven to pose effective medicinal properties, further studies on their crude extract and compounds extracted from the plant proved that the extracts are contained with secondary metabolites like alkaloids, flavonoids, phenols, and chalcones and were termed as phytochemicals. A large number of phytochemicals pose antimicrobial, antiviral, and antioxidant properties [<a href="#CR24">24</a>]. These plant parts which were used to treat the disease or infection were later experimentally proven for their medicinal properties [<a href="#CR25">25</a>]. Several literature studies and promising diverse biological assays reveal that a large amount of phytochemicals can be utilized as an effective treatment for retroviral infections. Recently, phytochemicals are used as an inhibitory material for viral infection including HIV, influenza, common cold, etc. [<a href="#CR26">26</a>, <a href="#CR27">27</a>].</p> <p id="Par18">In this study, an attempt has been made to virtually screen potential phytochemicals with high effectiveness particularly from plants originated from Western Ghats, India. Intensive literature survey has been made prior to the screening of phytochemicals. Effectiveness of the screened compounds were validated based on the in silico docking technique by binding to the S-protein of SARS-CoV-2 and ranked based on the interaction analysis between the protein-receptor complex.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="computationalplatform" title="sec"> <div class="title" tagx="title" title="title">
Computational platform</div> <p id="Par19">All computational analyses were performed on Linux Mint 18 platform in a Dell T20 server on Intel Xeon Quad Core 3.2 Hz.</p> </div> <div class="receptorpreparation" title="sec"> <div class="title" tagx="title" title="title">
Receptor preparation</div> <p id="Par20">The SARS-CoV-2 S-protein (PDB ID: 6X6P at 3.2 <i>Ã…</i>) with the spike glycoprotein was downloaded from RCSB Protein Bank (PDB). The protein is a trimeric viral fusion protein with subunits S1 and S2. S1 contains the receptor binding protein (RBD) responsible for host cell receptor binding and the S2 subunit facilitates the membrane fusion between the viral and host cell membranes. The viral protein contains 1274 amino acids and weighs 434.87 kDa. For the purpose of the study, we utilized only Chain A from the S-protein for docking analysis. Protein visualization and manipulation of the structures was done using UCSF Chimera [<a href="#CR28">28</a>].</p> </div> <div class="binding-sitepreparation" title="sec"> <div class="title" tagx="title" title="title">
Binding-site preparation</div> <p id="Par21">The prediction of binding site was done with respect to the available literature [<a href="#CR29">29</a>] and then verified based on the presence of hydrophobic regions on the surface of the protein. For the purpose of the study, we utilized the same binding site between the S-protein and CS-GRP78, since our main aim is to block the binding of S-protein to the ER protein [<a href="#CR29">29</a>]. The hydrophobicity of the protein is calculated using GRAVY server [<a href="#CR30">30</a>]. The S-protein contains four hydrophobic regions in the receptor binding domain (RBD) ranging from 318 to 510 as of the S1 subunit. The binding energy values obtained from protein binding energy prediction (PRODIGY) [<a href="#CR29">29</a>].</p> </div> <div class="screeningforphytochemicals" title="sec"> <div class="title" tagx="title" title="title">
Screening for phytochemicals</div> <p id="Par22">Phytochemicals with antiviral, antimicrobial, and antioxidant properties were screened based on the previous literature and were sorted, filtered based on their geographical origin [<a href="#CR31">31</a>â€"<a href="#CR33">33</a>]. More than 100 compounds were initially screened. 3D and 2D structures were downloaded from PUBCHEM database [<a href="#CR34">34</a>] and drugbank.ca [<a href="#CR35">35</a>] in both SDF and Simplified Molecular Input Line Entry System (SMILES) format [<a href="#CR36">36</a>] for further screening procedures. In order to select the potential drug-like compounds, several filters were applied on the structures.</p> <p id="Par23">Initially, LIPINSKI rule of five (RO5) was applied to screen potential drug-like compounds [<a href="#CR37">37</a>]. Drug-like compounds screened from RO5 was further fed into SWISS-ADME server for screening based on the pharmacokinetics, drug-likeliness, and medicinal chemistry friendliness of small molecules [<a href="#CR38">38</a>]. In addition, the physicochemical properties and the drugability of the selected compounds were also predicted with SWISS-ADME server. Hydroxychloroquinone (HCQ) (PUBCHEM ID: 3652), a well-known retro viral drug was used as a standard throughout the studies [<a href="#CR39">39</a>].</p> </div> <div class="multipleliganddocking" title="sec"> <div class="title" tagx="title" title="title">
Multiple ligand docking</div> <p id="Par24">For the purpose of study, Autodock was employed for docking simulations [<a href="#CR40">40</a>]. Initially, water molecules and hetero atoms were removed from the structures [<a href="#CR41">41</a>]. Addition of gasteiger charges and H-bond was carried out prior to the docking protocol. A grid box of 25 <i>Ã—</i> 22 <i>Ã—</i> 19 <i>Ã…</i> was assigned on the surface of the receptor with respect to the identified active site. In this present study, multiple ligands have been screened against the receptor. Multiple ligand docking was carried out using PyRx in the Autodock environment [<a href="#CR42">42</a>]. The docked structures were analyzed for root mean square deviation (RMSD), lowest energy conformer, and hydrogen bond (HB) interaction. 2D protein-ligand interaction profile was obtained using Protein Plus Server [<a href="#CR43">43</a>, <a href="#CR44">44</a>] and Protein Ligand Interaction Profiler [<a href="#CR45">45</a>].</p> </div> <div class="physicochemicalpropertyoftheboundligand" title="sec"> <div class="title" tagx="title" title="title">
Physicochemical property of the bound ligand</div> <p id="Par25">To validate the bound ligand with the receptor, the physicochemical and pharmacokinetic properties of the drug are analyzed using SWISS-ADME server [<a href="#CR38">38</a>]. The toxicity profile is analyzed using admetSAR 2.0 [<a href="#CR46">46</a>], which provides information on the ADMET properties. Screened compounds were subjected to Ghose filter [<a href="#CR47">47</a>] and Muegge Filter [<a href="#CR48">48</a>].</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par26">In the present study, 57 phytochemicals were initially screened based on their structural and physicochemical properties, later only 39 compounds were utilized for the docking analysis based on the various filters utilized during the course of the study. Binding sites were initially chosen based on the literature survey made on the target protein 6X6P. Potential binding sites were further selected based on the hydrophobic index obtained from GRAVY server. Results from GRAVY server indicate significant values for the four-binding region R1-R4 as âˆ’ 0.24, âˆ’ 0.3, âˆ’ 0.28, and âˆ’ 0.08, respectively. Binding sites were also considered based on their predicted binding energy by PRODIGY. Results from PRODIGY for the four binding regions R1â€"R4 are âˆ’ 12.4, âˆ’ 9.8, âˆ’ 10.1, and â€" 14 kcal/mol, respectively. Region 4 (R4) [âˆ’ 9.8 kcal/mol] and Region 1 (R1) [âˆ’ 12.4 kcal/mol] of the S-protein pose a higher hydrophobicity. However, R1 was considered for the potential binding site.</p> <p id="Par27">Prior to the actual binding, the ligands are screened based on the RO5 and SWISS-ADME filters. Among the 39 finally screened compounds, only 5 compounds exhibited docking energy values above âˆ’ 8.9 to 8.3 kcal/mol for SARS-CoV-2 S-protein Region 1Â (Fig. <a href="#Fig2">2</a>). Among them, Tanshinone I exhibited an average docking score of âˆ’ 8.9 kcal/mol and Piperolactam A with a docking score of âˆ’ 8.3 kcal/mol ranked the strongest both in terms of binding energy and HB interaction. A list of all other ranked ligands based on the binding energy is listed in the Table <a href="#Tab1">1</a>. The standard ligand, HCQ resulted in a docking score of âˆ’ 4.7 kcal/mol and was used to compare other phytochemicals docked with the S-protein. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Dock Scores obtained from Autodock Pyrex of Screened Ligands</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Ligand</th> <th>Binding energy<br /><br />(kcal/mol)</th> <th>Molecular weight<br /><br />(g/mol)</th> <th>Rotatable bonds</th> <th>HB acceptor</th> <th>HB Donar</th> <th>TPSA (Â Ã…<sup>2</sup>)</th> </tr> </thead> <tbody> <tr> <td>Tanshinone I</td> <td>-8.9</td> <td>276.29</td> <td>0</td> <td>3</td> <td>0</td> <td>47.28</td> </tr> <tr> <td>Ellipticine</td> <td>-8.4</td> <td>246.31</td> <td>0</td> <td>1</td> <td>1</td> <td>28.68</td> </tr> <tr> <td>Anabsinthin</td> <td>-8.4</td> <td>496.64</td> <td>0</td> <td>6</td> <td>1</td> <td>82.06</td> </tr> <tr> <td>Camptothecin</td> <td>-8.3</td> <td>348.35</td> <td>1</td> <td>5</td> <td>1</td> <td>81.42</td> </tr> <tr> <td>Piperolactam A</td> <td>-8.3</td> <td>265.26</td> <td>1</td> <td>3</td> <td>2</td> <td>62.32</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Virtually screened ligands from PubChem compounds</p>   </div> <p /> <p id="Par28">The HCQ-S-protein complex has two HB interactions with ARG355 and TYR396 which contribute to the hydrophobic interactions. In comparison with HCQ, Tanshinone I has the highest binding affinity of âˆ’ 8.9 kcal/mol compared to all other compounds and has one HB of distance 2.38 <i>Ã…</i> with the acceptor atom O2 of ASN343. Tanshinone I also formed a good hydrophobic interaction with the neighboring amino acids <i>viz.,</i> PHE338, PHE342, VAL367, LEU368, and PHE374 (Table <a href="#Tab2">2</a>), whereas the second lead compound Piperolactam A with binding affinity âˆ’ 8.3 kcal/mol is found to form two HB interaction with the S-protein in which both bonds are from ASP364 to the compound with a distance of 1.99 <i>Ã…</i> and 2.05 <i>Ã…Â </i>(Fig. <a href="#Fig3">3</a>). <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>HB Interaction with the Screened Ligands</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Ligands</th> <th>Hydrogen bonds</th> <th>Hydrogen Bond Interaction</th> <th>Distance H-A (A<sup>2</sup>)</th> </tr> </thead> <tbody> <tr> <td>Tanshinone I</td> <td>1</td> <td>ASN A: 343</td> <td>2.3</td> </tr> <tr> <td>Ellipticine</td> <td>0</td> <td>-</td> <td>-</td> </tr> <tr> <td>Anabsinthin</td> <td>1</td> <td>ASP A:364</td> <td>2.68</td> </tr> <tr> <td>Camptothecin</td> <td>2</td> <td>GLY A:339, ASP A:364</td> <td>2.83,3.21</td> </tr> <tr> <td>Piperolactam A</td> <td>2</td> <td>ASP A:364, ASP A:364</td> <td>1.99, 2.05</td> </tr> <tr> <td>Hydroxychloroquine</td> <td>2</td> <td>355 A:ARG,466 A:ARG</td> <td>2.27,2.37</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Piperolactam A docked with S-protein. <b>a</b> Surface image of Piperolactam A docked with the Region 1 of 6X6P S-protein. <b>b</b> 2D Interaction map of the Piperolactam A with the S-protein. Interaction profiles indicate two H-bond with ASP364</p>   </div> <p /> <p id="Par29">Compounds with good ADME parameters are suitable for the drug discovery process. Analysis of the lipophilicity of the screened compounds showed positive values which indicates enhanced rate of adsorption [<a href="#CR49">49</a>] as described in <a href="#MOESM1">supplementary material</a>.</p> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par30">In this present study, although during the selection and validation of the predicted binding sites, both Region R1 and R4 resulted as potential binding sites among the four predicted sites. Based on the hydrophobicity index, R4 is marked with higher index values; however, the site fails to show minimum interaction with the known receptor CS-GRP78 [<a href="#CR10">10</a>]. The binding affinity of R4 is â€" 14 kcal/mol which is much lower than the R1 with âˆ’ 12.4 kcal/mol. Based on the binding affinity values, R1 is considered to be a potential binding site for the screened ligands. The binding site involves the residues of ASN343, ASP364, GLY339, ARG355, and ARG466. Additional amino acids at the active site are PHE338, VAL367, LEU368, PHE342, PHE337, and TYR396 (Fig. <a href="#Fig1">1</a>).</p> <p id="Par31">With respect to screened ligands, the hydrophilicity factor (LogS) of Camptothecin (âˆ’ 3.49) and Piperolactam A (âˆ’ 3.87) showed higher LogS values (high permeation and hydrophilicity) than the other screened compounds [<a href="#CR50">50</a>]. Among the screened compounds, Ellipticine (âˆ’ 5.05) showed a less LogS value indicating a poor permeation. Similarly, anabsinthin acts a good substrate for P-glycoprotein, which is highly susceptible to changes in pharmacokinetics by acting as either P-glycoprotein inducer or inhibitors [<a href="#CR51">51</a>]. Thus, anabsinthin could cross the blood-brain barrier (BBB) and it may render the compound functionally inactive [<a href="#CR52">52</a>]. Further, anabsinthin could not satisfy Ghose rule [<a href="#CR47">47</a>] of drug-likeness with three violations and Muegge rule [<a href="#CR48">48</a>] of drug-likeness with one violation. Piperolactam A showed a good pharmacokinetics property with good GI absorption, cross BBB, affects only two cytochrome P450 enzymes compared to others and is also not a substrate for P-glycoprotein. The phytochemical compound Piperolactam A has synthetic accessibility of 2.02 which makes it at ease of synthesis of the compound in spite of an alert in the Brenk filter. Whereas Camptothecin was found with no violations but its synthetic accessibility is of higher value than that of Piperolactam A. Toxicity analysis is essential for a drug to know the potential hazard to be produced by it. The predicted toxicity profile from our study reveals that all the compounds are relatively safe.</p> <p id="Par32">Results from the docking analysis suggest that the Piperolactam A pocketed well with good hydrophobic interaction with residues PHE338, PHE342, and VAL367as well Â (Fig. <a href="#Fig3">3</a>). Based on the results obtained, the study suggests use of Piperolactam A as a potential antiretroviral drug against SARS-CoV-2. Piperolactam, an aristolactam isolated from Piper betle Linn and its abundant availability in Western Ghats [<a href="#CR53">53</a>].</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par33">In this study, five phytochemical compounds with antiretroviral properties were screened against the receptor 6X6P. These compounds displayed appreciable pharmacokinetic and physicochemical properties. ADMET analyses of these compounds reveal that the compound Piperolactam A from <i>Piperaceae</i> family to be most effective in the inhibition of S-protein from binding to the CS-GRP78 receptor.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="43088_2021_95_MOESM1_ESM.pdf">LINK</a> <p><b>Additional file 1.</b></p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
ACE2</div> <div class="def" title="def"> <p id="Par4">Angiotensin converting enzyme-2</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ER</div> <div class="def" title="def"> <p id="Par5">Endoplasmic reticulum</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DPP4</div> <div class="def" title="def"> <p id="Par6">Dipeptidyl-peptidase 4</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CS-GRP78</div> <div class="def" title="def"> <p id="Par7">Cell surface glucose receptor protein 78</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HIV</div> <div class="def" title="def"> <p id="Par8">Human immuno-deficiency virus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PRODIGY</div> <div class="def" title="def"> <p id="Par9">Protein energy binding prediction</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SMILES</div> <div class="def" title="def"> <p id="Par10">Simplified Molecular Input Line Entry System</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BBB</div> <div class="def" title="def"> <p id="Par11">Blood-brain barrier</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Not applicable.</p> </div> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>RK designed the study, carried out research, and wrote manuscript. CSA contributed to study design and manuscript writing. JA and SS advised on phytochemical virtual screening and editing manuscript. RR contributed to the docking analysis and interpretation. All authors have read and approved the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>Not applicable.</p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>All data used in the study is given as supplementary data.</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par34">Not applicable.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p id="Par35">Not applicable.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par36">The authors declare that they have no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Labrague</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Los Santos</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">COVID-19 anxiety among frontline nurses: predictive role of organisational support, personal resilience and social support</span><span class="source" tagx="source" title="source">J Nurs Manag.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">28</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">1653</span><span class="lpage" tagx="lpage" title="lpage">1661</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/jonm.13121">10.1111/jonm.13121</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32770780">32770780</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saini</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parihar</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soni</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title">Drug repurposing: an overview</span><span class="source" tagx="source" title="source">Asian J Pharm Res Dev</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">194</span><span class="lpage" tagx="lpage" title="lpage">212</span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Astuti</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2): an overview of viral structure and host response</span><span class="source" tagx="source" title="source">Diabetes Metab Syndr</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">407</span><span class="lpage" tagx="lpage" title="lpage">412</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.dsx.2020.04.020">10.1016/j.dsx.2020.04.020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32335367">32335367</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belouzard</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Millet</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Licitra</span><span class="given-names" tagx="given-names" title="given-names">BN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whittaker</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mechanisms of coronavirus cell entry mediated by the viral spike protein</span><span class="source" tagx="source" title="source">Viruses.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1011</span><span class="lpage" tagx="lpage" title="lpage">1033</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v4061011">10.3390/v4061011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22816037">22816037</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roshanravan</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghaffari</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedayati</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Angiotensin converting enzyme-2 as therapeutic target in COVID-19</span><span class="source" tagx="source" title="source">Diabetes Metab Syndr</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">637</span><span class="lpage" tagx="lpage" title="lpage">639</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.dsx.2020.05.022">10.1016/j.dsx.2020.05.022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32428864">32428864</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalathiya</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padariya</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayordomo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lisowska</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicholson</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Highly conserved homotrimer cavity formed by the SARS-CoV-2 spike glycoprotein: a novel binding site</span><span class="source" tagx="source" title="source">J Clin Med.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">1473</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/jcm9051473">10.3390/jcm9051473</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7290299">7290299</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prajapat</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarma</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shekhar</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avti</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinha</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaur</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Drug targets for corona virus: a systematic review</span><span class="source" tagx="source" title="source">Indian J Pharmacol.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">52</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">56</span><span class="pub-id"><a href="https://dx.doi.org/10.4103/ijp.IJP_115_20">10.4103/ijp.IJP_115_20</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32201449">32201449</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Penninger</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slutsky</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Angiotensin-converting enzyme 2 (ACE2) as a SARS-CoV-2 receptor: molecular mechanisms and potential therapeutic target</span><span class="source" tagx="source" title="source">Intensive Care Med.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">46</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">586</span><span class="lpage" tagx="lpage" title="lpage">590</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00134-020-05985-9">10.1007/s00134-020-05985-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32125455">32125455</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cell surface GRP78 as a death receptor and an anticancer drug target</span><span class="source" tagx="source" title="source">Cancers.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">11</span><span class="fpage" tagx="fpage" title="fpage">1787</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/cancers11111787">10.3390/cancers11111787</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6896222">6896222</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibrahim</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelmalek</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span></span><span class="mixed-article-title" title="mixed-article-title">GRP78: a cellâ€™s response to stress</span><span class="source" tagx="source" title="source">Life Sci</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">226</span><span class="fpage" tagx="fpage" title="fpage">156</span><span class="lpage" tagx="lpage" title="lpage">163</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.lfs.2019.04.022">10.1016/j.lfs.2019.04.022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30978349">30978349</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="mixed-article-title" title="mixed-article-title">The ER chaperone and signaling regulator GRP78/BiP as a monitor of endoplasmic reticulum stress</span><span class="source" tagx="source" title="source">Methods.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">35</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">373</span><span class="lpage" tagx="lpage" title="lpage">381</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ymeth.2004.10.010">10.1016/j.ymeth.2004.10.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15804610">15804610</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¶nthal</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Endoplasmic reticulum stress: its role in disease and novel prospects for therapy</span><span class="source" tagx="source" title="source">Scientifica.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">2012</span><span class="fpage" tagx="fpage" title="fpage">857516</span><span class="pub-id"><a href="https://dx.doi.org/10.6064/2012/857516">10.6064/2012/857516</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24278747">24278747</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ni</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Beyond the endoplasmic reticulum: atypical GRP78 in cell viability, signalling and therapeutic targeting</span><span class="source" tagx="source" title="source">Biochem J</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">434</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">188</span><span class="pub-id"><a href="https://dx.doi.org/10.1042/BJ20101569">10.1042/BJ20101569</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21309747">21309747</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nain</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukherjee</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karmakar</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paton</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paton</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">GRP78 is an important host factor for Japanese encephalitis virus entry and replication in mammalian cells</span><span class="source" tagx="source" title="source">J Virol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">91</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">e02274</span><span class="lpage" tagx="lpage" title="lpage">e02216</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02274-16">10.1128/JVI.02274-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28053106">28053106</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Middle East respiratory syndrome coronavirus and bat coronavirus HKU9 both can utilize GRP78 for attachment onto host cells</span><span class="source" tagx="source" title="source">J Biol Chem.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">293</span><span class="issue" tagx="issue" title="issue">30</span><span class="fpage" tagx="fpage" title="fpage">11709</span><span class="lpage" tagx="lpage" title="lpage">11726</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.RA118.001897">10.1074/jbc.RA118.001897</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29887526">29887526</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ha</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Krieken</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlos</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="mixed-article-title" title="mixed-article-title">The stress-inducible molecular chaperone GRP78 as potential therapeutic target for coronavirus infection</span><span class="source" tagx="source" title="source">J Infect</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">81</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">452</span><span class="lpage" tagx="lpage" title="lpage">482</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jinf.2020.06.017">10.1016/j.jinf.2020.06.017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32535155">32535155</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Muralidharan N, Sakthivel R, Velmurugan D, Gromiha MM (2020) Computational studies of drug repurposing and synergism of lopinavir, oseltamivir and ritonavir binding with SARS-CoV-2 Protease against COVID-19. J Biomol Struct Dyn 16:1â€"6. 10.1080/07391102.2020.1752802. Epub ahead of print. PMID: 32248766.</span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tripathy</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dassarma</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chabalala</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsabisa</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span></span><span class="mixed-article-title" title="mixed-article-title">A review on possible modes of actions of Chloroquine/Hydroxychloroquine: Repurposing against SAR-COV-2 (COVID 19) pandemic</span><span class="source" tagx="source" title="source">Int J Antimicrob Agents</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">106028</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijantimicag.2020.106028">10.1016/j.ijantimicag.2020.106028</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32450198">32450198</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ghildiyal R, Prakash V, Chaudhary V, Gupta V, Gabrani R (2020) Phytochemicals as antiviral agents: recent updates. In: Plant-derived Bioactives; Singapore: Springer, pp 279â€"295</span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biswas</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nandy</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukherjee</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dey</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Moringa oleifera Lam. and derived phytochemicals as promising antiviral agents: a review</span><span class="source" tagx="source" title="source">South Afr J Botany</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">272</span><span class="lpage" tagx="lpage" title="lpage">282</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.sajb.2019.07.049">10.1016/j.sajb.2019.07.049</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kapoor</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanwar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antiviral phytochemicals: an overview</span><span class="source" tagx="source" title="source">Biochem Physiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="pub-id"><a href="https://dx.doi.org/10.4172/2168-9652.1000220">10.4172/2168-9652.1000220</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subudhi</span><span class="given-names" tagx="given-names" title="given-names">BB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chattopadhyay</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mishra</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Current strategies for inhibition of chikungunya infection</span><span class="source" tagx="source" title="source">Viruses.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">235</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v10050235">10.3390/v10050235</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5977228">5977228</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rastogi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rawat</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Indian traditional ayurvedic system of medicine and nutritional supplementation</span><span class="source" tagx="source" title="source">Evid Based Complement Alternat Med.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">2013</span><span class="fpage" tagx="fpage" title="fpage">376327</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23864888">23864888</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Naithani R, Mehta RG, Shukla D, Chandersekera SN, Moriarty RM (2010) Antiviral activity of phytochemicals: a current perspective. In: Dietary Components and Immune Function. Humana Press, Totowa: Springer, pp 421â€"468</span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrovska</span><span class="given-names" tagx="given-names" title="given-names">BB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Historical review of medicinal plants usage</span><span class="source" tagx="source" title="source">Pharmacogn Rev</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">11</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="pub-id"><a href="https://dx.doi.org/10.4103/0973-7847.95849">10.4103/0973-7847.95849</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22654398">22654398</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meher</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purohit</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dash</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span></span><span class="source" tagx="source" title="source">Probing phytochemicals as prospective antiviral agents against HIV-1 protease through structure-based virtual screening and molecular dynamics simulations</span><span class="year" tagx="year" title="year">2020</span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hota DK, Nayak D, Swain S, Bhattacharyay D (2020) Capsicum derived phytochemicals against Shikimate dehydrogenase of haemophilus influenza causing bronchitis. J Pharm Res Intern 32(6):128â€"131</span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pettersen</span><span class="given-names" tagx="given-names" title="given-names">EF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goddard</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Couch</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenblatt</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">UCSF Chimeraa visualization system for exploratory research and analysis</span><span class="source" tagx="source" title="source">J Comput Chem</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">13</span><span class="fpage" tagx="fpage" title="fpage">1605</span><span class="lpage" tagx="lpage" title="lpage">1612</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.20084">10.1002/jcc.20084</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15264254">15264254</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibrahim</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelmalek</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elshahat</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span></span><span class="mixed-article-title" title="mixed-article-title">COVID-19 spike-host cell receptor GRP78 binding site prediction</span><span class="source" tagx="source" title="source">J Infect</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">80</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">554</span><span class="lpage" tagx="lpage" title="lpage">562</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jinf.2020.02.026">10.1016/j.jinf.2020.02.026</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32169481">32169481</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ul Qamar</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alqahtani</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alamri</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural basis of SARS-CoV-2 3CLpro and anti-COVID-19 drug discovery from medicinal plants</span><span class="source" tagx="source" title="source">J Pharm Anal</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">313</span><span class="lpage" tagx="lpage" title="lpage">319</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jpha.2020.03.009">10.1016/j.jpha.2020.03.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32296570">32296570</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silambarasan</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sureshkumar</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krupa</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amalraj</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayyanar</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Traditional herbal medicines practiced by the ethnic people in Sathyamangalam forests of Western Ghats, India</span><span class="source" tagx="source" title="source">Eur J Integrative Med</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">61</span><span class="lpage" tagx="lpage" title="lpage">72</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.eujim.2017.10.010">10.1016/j.eujim.2017.10.010</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishnan</span><span class="given-names" tagx="given-names" title="given-names">PN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decruse</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radha</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Conservation of medicinal plants of Western Ghats, India and its sustainable utilization through in vitro technology</span><span class="source" tagx="source" title="source">In Vitro Cell Dev Biology-Plant.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">47</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">110</span><span class="lpage" tagx="lpage" title="lpage">122</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11627-011-9344-9">10.1007/s11627-011-9344-9</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venkatachalapathi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sangeeth</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamilselvi</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paulsamy</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Hemaidc</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ethnomedicinal assessment of Irula tribes of Walayar valley of Southern Western Ghats, India</span><span class="source" tagx="source" title="source">Saudi J Biol Sci</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">760</span><span class="lpage" tagx="lpage" title="lpage">775</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.sjbs.2016.10.011">10.1016/j.sjbs.2016.10.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29740242">29740242</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gindulyte</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">PubChem 2019 update: improved access to chemical data</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">47</span><span class="issue" tagx="issue" title="issue">D1</span><span class="fpage" tagx="fpage" title="fpage">D1102</span><span class="lpage" tagx="lpage" title="lpage">D1109</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gky1033">10.1093/nar/gky1033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30371825">30371825</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wishart</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feunang</span><span class="given-names" tagx="given-names" title="given-names">YD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marcu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grant</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">DrugBank 5.0: a major update to the DrugBank database for 2018</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">46</span><span class="issue" tagx="issue" title="issue">D1</span><span class="fpage" tagx="fpage" title="fpage">D1074</span><span class="lpage" tagx="lpage" title="lpage">D1082</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkx1037">10.1093/nar/gkx1037</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29126136">29126136</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weininger</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">SMILES, a chemical language and information system. 1. Introduction to methodology and encoding rules</span><span class="source" tagx="source" title="source">J Chem Inform Comput Sci.</span><span class="year" tagx="year" title="year">1988</span><span class="volume" tagx="volume" title="volume">28</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">31</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ci00057a005">10.1021/ci00057a005</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipinski</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lead-and drug-like compounds: the rule-of-five revolution</span><span class="source" tagx="source" title="source">Drug Discov Today.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">1</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">337</span><span class="lpage" tagx="lpage" title="lpage">341</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ddtec.2004.11.007">10.1016/j.ddtec.2004.11.007</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daina</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michielin</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zoete</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">42717</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep42717">10.1038/srep42717</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28256516">28256516</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naghipour</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghodousi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahsepar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elyasi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Repurposing of well-known medications as antivirals: hydroxychloroquine and chloroquine; from HIV-1 infection to COVID-19</span><span class="source" tagx="source" title="source">Expert Rev Anti Infect Ther</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">18</span><span class="issue" tagx="issue" title="issue">11</span><span class="fpage" tagx="fpage" title="fpage">1119</span><span class="lpage" tagx="lpage" title="lpage">1133</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/14787210.2020.1792291">10.1080/14787210.2020.1792291</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32631083">32631083</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huey</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindstrom</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanner</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belew</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodsell</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility</span><span class="source" tagx="source" title="source">J Comput Chem</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">16</span><span class="fpage" tagx="fpage" title="fpage">2785</span><span class="lpage" tagx="lpage" title="lpage">2791</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.21256">10.1002/jcc.21256</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19399780">19399780</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huggins</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tidor</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Systematic placement of structural water molecules for improved scoring of proteinâ€"ligand interactions</span><span class="source" tagx="source" title="source">Protein Eng Des Sel</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">24</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">777</span><span class="lpage" tagx="lpage" title="lpage">789</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/protein/gzr036">10.1093/protein/gzr036</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21771870">21771870</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dallakyan S, Olson AJ (2015) Swmall-molecule library screening by docking with PyRx. In: Chemical biology. Humana Press, New York: Springer, pp 243â€"250</span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FÃ¤hrrolfes</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bietz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flachsenberg</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyder</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nittinger</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Otto</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Proteins Plus: a web portal for structure analysis of macromolecules</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">45</span><span class="issue" tagx="issue" title="issue">W1</span><span class="fpage" tagx="fpage" title="fpage">W337</span><span class="lpage" tagx="lpage" title="lpage">W343</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkx333">10.1093/nar/gkx333</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28472372">28472372</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stierand</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MaaÃŸ</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rarey</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular complexes at a glance: automated generation of two-dimensional complex diagrams</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">22</span><span class="issue" tagx="issue" title="issue">14</span><span class="fpage" tagx="fpage" title="fpage">1710</span><span class="lpage" tagx="lpage" title="lpage">1716</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btl150">10.1093/bioinformatics/btl150</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16632493">16632493</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salentin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schreiber</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haupt</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adasme</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schroeder</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">PLIP: fully automated proteinâ€"ligand interaction profiler</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">43</span><span class="issue" tagx="issue" title="issue">W1</span><span class="fpage" tagx="fpage" title="fpage">W443</span><span class="lpage" tagx="lpage" title="lpage">W447</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkv315">10.1093/nar/gkv315</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25873628">25873628</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lou</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">admetSAR 2.0: web-service for prediction and optimization of chemical ADMET properties</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">35</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1067</span><span class="lpage" tagx="lpage" title="lpage">1069</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bty707">10.1093/bioinformatics/bty707</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30165565">30165565</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghose</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viswanadhan</span><span class="given-names" tagx="given-names" title="given-names">VN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wendoloski</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">A knowledge-based approach in designing combinatorial or medicinal chemistry libraries for drug discovery. 1. A qualitative and quantitative characterization of known drug databases</span><span class="source" tagx="source" title="source">J Comb Chem</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">1</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">55</span><span class="lpage" tagx="lpage" title="lpage">68</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/cc9800071">10.1021/cc9800071</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10746014">10746014</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muegge</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heald</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brittelli</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Simple selection criteria for drug-like chemical matter</span><span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">44</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">1841</span><span class="lpage" tagx="lpage" title="lpage">1846</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/jm015507e">10.1021/jm015507e</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11384230">11384230</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnott</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Planey</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="mixed-article-title" title="mixed-article-title">The influence of lipophilicity in drug discovery and design</span><span class="source" tagx="source" title="source">Expert Opin Drug Discov</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">863</span><span class="lpage" tagx="lpage" title="lpage">875</span><span class="pub-id"><a href="https://dx.doi.org/10.1517/17460441.2012.714363">10.1517/17460441.2012.714363</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22992175">22992175</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hildebrand</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Solubility. xii. Regular solutions1</span><span class="source" tagx="source" title="source">J Am Chem Soc.</span><span class="year" tagx="year" title="year">1929</span><span class="volume" tagx="volume" title="volume">51</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">66</span><span class="lpage" tagx="lpage" title="lpage">80</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ja01376a009">10.1021/ja01376a009</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure, function and regulation of P-glycoprotein and its clinical relevance in drug disposition</span><span class="source" tagx="source" title="source">Xenobiotica.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">38</span><span class="issue" tagx="issue" title="issue">7-8</span><span class="fpage" tagx="fpage" title="fpage">802</span><span class="lpage" tagx="lpage" title="lpage">832</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/00498250701867889">10.1080/00498250701867889</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18668431">18668431</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amin</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span></span><span class="mixed-article-title" title="mixed-article-title">P-glycoprotein inhibition for optimal drug delivery</span><span class="source" tagx="source" title="source">Drug Target Insights</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">DTI</span><span class="lpage" tagx="lpage" title="lpage">S12519</span><span class="pub-id"><a href="https://dx.doi.org/10.4137/DTI.S12519">10.4137/DTI.S12519</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khushbu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roshni</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anar</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carol</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayuree</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phytochemical and therapeutic potential of Piper longum Linn a review</span><span class="source" tagx="source" title="source">Intern J Res Ayurveda Pharm</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">157</span><span class="lpage" tagx="lpage" title="lpage">161</span></span></li> </ul> </div> </div>  </body></html>