<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Pharmacogn Mag</span><span class="iso-abbrev" title="iso-abbrev">Pharmacogn Mag</span><span class="publisher-id" title="publisher-id">PM</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Pharmacognosy Magazine</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0973-1296</span><span class="issn-epub" tagx="issn" title="issn-epub">0976-4062</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Medknow Publications &amp;amp; Media Pvt Ltd</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">India</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5669121">5669121</a></span><span class="publisher-id" title="publisher-id">PM-13-715</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.4103/pm.pm_82_17">10.4103/pm.pm_82_17</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Molecular Simulation-based Combinatorial Modeling and Antioxidant Activities of Zingiberaceae Family Rhizomes</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usha</span><span class="given-names" tagx="given-names" title="given-names">Talambedu</span></span><a href="#aff1">1</a><a href="#aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pradhan</span><span class="given-names" tagx="given-names" title="given-names">Sushen</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">Arvind Kumar</span></span><a href="#aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhivya</span><span class="given-names" tagx="given-names" title="given-names">Shanmugarajan</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">H. P. Prashanth</span></span><a href="#aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">Manoj Kumar</span></span><a href="#aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joshi</span><span class="given-names" tagx="given-names" title="given-names">Neelu</span></span><a href="#aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basistha</span><span class="given-names" tagx="given-names" title="given-names">Bharat Chandra</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murthy</span><span class="given-names" tagx="given-names" title="given-names">K. R. Siddalinga</span></span><a href="#aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selvaraj</span><span class="given-names" tagx="given-names" title="given-names">Saravanakumar</span></span><a href="#aff8">8</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middha</span><span class="given-names" tagx="given-names" title="given-names">Sushil Kumar</span></span><a href="#aff1">1</a><a href="#aff9">9</a><a href="#cor1" /></span></div><span class="citation_author_institution" id="aff1">[1], </span><span class="citation_author_institution" id="aff2">[2], </span><span class="citation_author_institution" id="aff3">[3], </span><span class="citation_author_institution" id="aff4">[4], </span><span class="citation_author_institution" id="aff5">[5], </span><span class="citation_author_institution" id="aff6">[6], </span><span class="citation_author_institution" id="aff7">[7], </span><span class="citation_author_institution" id="aff8">[8], </span><span class="citation_author_institution" id="aff9">[9], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><b>Correspondence:</b> Dr. Sushil Kumar Middha, Department of Biotechnology, Biotechnology Finishing School, Science Research Centre, DBT-BIF Facility, Maharani Lakshmi Ammanni College for Women, Bengaluru - 560 012, Karnataka, India. E-mail: <span class="email" tagx="email" title="email">drsushilmiddha@gmail.com</span></div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2017-10</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-10-10</span></span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">Suppl 3</span><span class="fpage" tagx="fpage" title="fpage">S715</span><span class="lpage" tagx="lpage" title="lpage">S722</span><span class="history" title="history"><span class="received" title="received">received: 2017-3-05</span><span class="rev-recd" title="rev-recd">rev-recd: 2017-4-27</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the Creative Commons Attribution-NonCommercial-ShareAlike 3.0 License, which allows others to remix, tweak, and build upon the work non-commercially, as long as the author is credited and the new creations are licensed under the identical terms.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="objective:" title="sec"> <div class="title" tagx="title" title="title">
Objective:</div> <p>The main aim of this scientific report was to investigate a series of phytochemicals <i>in silico</i> and the pharmacology of four plants found at higher altitude in the ginger family, Zingiberaceae (incl. Costaceae) from North-East India, particularly Sikkim. First, the goal was to determine the biological activities of the four herbs (used under Zingiberaceae family) using antioxidant assays to identify the best species. Second, previously reported compounds <i>in litero</i> were subsequently screened for their anticancerous activities using <i>in silico</i> methods.</p> </div> <div class="materialsandmethods:" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods:</div> <p>Using the methanolic extracts of herbs, quantitative detection of phytochemicals such as total phenols and total flavonoids was detected, and the free radical scavenging activity was also studied using 2,2-diphenyl-1-picryl-hydrazyl (DPPH) assay. Docking process was studied, using Discovery Studio version 3.5, to identify suitable molecules at the protein-binding sites through annealing and genetic simulation algorithms. Grids centered on active sites were obtained with spacing of 54 × 55 × 56, and 0.503 grid spacing was calculated. The methods adopted and used in this study were comparisons of Global and Local Search Methods to determine the parameters such as maximum number of 250,000 energy evaluations as well as generations of 27,000, followed by mutation and crossover rates of 0.02 and 0.80. The number of docking runs was set to 10. Molecular dynamics study was done to check the stability of the complex.</p> </div> <div class="results:" title="sec"> <div class="title" tagx="title" title="title">
Results:</div> <p>Among all the genus of Zingiberaceae family investigated in this study, <i>Curcuma angustifolia</i> and <i>Hedychium sp</i>. exhibited the highest 537 ± 12.45; 292 ± 9.16 mg gallic acid equivalent/g total polyphenols and 38 ± 1.54; 75 ± 6.75 mg quercetin equivalent/g flavonoids, respectively. Depending on the concentration, the <i>Hedychium</i> sp. extract exerted the highest scavenging activity on DPPH radical (IC<sub>50</sub> 36.4 μg/mL). <i>In silico</i> result demonstrated that the synergetic effects of β-phellandrene with other compounds might be responsible for its anticancerous activity. β-phellandrene and farnesene epoxide showed bonding with Leu<sup>298</sup>, Ala<sup>302</sup>, Met<sup>336</sup>, Leu<sup>339</sup>, Leu<sup>343</sup>, Phe<sup>356</sup>, Ala<sup>302</sup>, Glu<sup>305</sup>, Met<sup>340</sup>, Leu<sup>343</sup>, Arg<sup>346</sup>, Phe<sup>356</sup>, Ile<sup>373</sup>, Ile<sup>376</sup>, Leu<sup>380</sup>, His<sup>475</sup>, Leu<sup>476</sup>, and Leu<sup>491</sup>.</p> </div> <div class="conclusion:" title="sec"> <div class="title" tagx="title" title="title">
Conclusion:</div> <p>Based on the current available literature, this is the first study to understand the interaction of compounds found in the rhizomes of Zingiberaceae family.</p> </div> <div class="summary" title="sec"> <div class="title" tagx="title" title="title">
SUMMARY</div> <p> </p><ul> <li> <p>The aqueous methanolic extract of Zingiberaceae family <i>Curcuma angustifolia</i> and <i>Hedychium sp</i>. has potent antioxidant activity as assessed by 2,2-diphenyl-1-picryl-hydrazyl assays</p> </li> <li> <p><i>Hedychium</i> sp. is understood to possess more active compounds than other varieties</p> </li> <li> <p><i>In silico</i> studies indicated synergetic effects of β-phellandrene and other compounds for its anticancerous activity.</p> </li> </ul> <p /> <p><span class="inline-graphic" title="inline-graphic" /></p> <p><b>Abbreviations used:</b> CADD: Computer-aided drug designing; ROS: Reactive oxygen species; ADMET: Absorption, distribution, metabolism, and excretion-toxicity; FeCl<sub>3</sub>: Ferric chloride; DPPH: 2,2-diphenyl-1-picryl-hydrazyl; NaNO<sub>2</sub>: Sodium nitrite; TCA: Trichloroacetic acid; K<sub>2</sub>HPO<sub>4</sub>: Di-potassium hydrogen phosphate; H<sub>2</sub>O<sub>2</sub>: Hydrogen peroxide; KH<sub>2</sub>PO<sub>4</sub>: Potassium di-hydrogen phosphate, K<sub>2</sub>Fe (CN)<sub>6</sub>: Potassium ferricyanide; KOH: Potassium hydroxide; NaOH: Sodium hydroxide; Na<sub>2</sub>CO<sub>3</sub>: Sodium carbonate; CH<sub>3</sub>COONa: Sodium acetate; AlCl<sub>3</sub>: Aluminum chloride.</p> </div> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
Zingiberaceae</div> <div class="kwd" title="kwd"><i>Curcuma</i></div> <div class="kwd" title="kwd">
cancer</div> <div class="kwd" title="kwd">
CADD</div> <div class="kwd" title="kwd">
molecular dynamics</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>Lack of scientific knowledge to find a cure for health ailments leads to the diversion of research toward new strategies such as exploration of traditional ethnobotanical knowledge. Plants constitute an important source of various secondary metabolites.[<a href="#ref1">1</a>] High-altitude medicinal plants acquire a unique collection of secondary metabolites that can act as a potential therapeutic targets for diseases.[<a href="#ref2">2</a>] Bioactive natural inventions with remedial potential have relied on external circumstances and some of these compounds are beyond exploration by traditional means. Therefore, the efficiency of computational tools to facilitate drug discovery are is known and has been recognized even in case of natural products.[<a href="#ref1">1</a><a href="#ref3">3</a><a href="#ref4">4</a><a href="#ref5">5</a>] Computer-aided drug designing (CADD) utilizes the existing information from the ethnobotanical data to provide valuable understanding to address the existing therapeutics and for the development of new nature-derived drugs. In addition, more than a few medicinal plants have been continuously utilized in various conventional medicinal systems since olden days throughout the globe. However, their mechanisms have not yet been explored properly. Therefore, the CADD approaches for plant-based drug discovery would yield immense gain in advancing the world's health population.</p> <p>Antioxidants are the counterparts that can restrain single oxygen or free radicals prior to any noteworthy oxidant force taking place.[<a href="#ref2">2</a><a href="#ref6">6</a>] Increasing knowledge and lookout for a healthier product having fewer preservatives and synthetic additives are motivating research endeavors toward the natural antioxidants. Plant-related dietary products are gaining consideration due to their increasing scientific evidence, having indicated their richness in several phenolic compounds; therefore, preventing pathological conditions such as diabetes, cancer, and neurological diseases.</p> <p>Zingiberaceae family, an Angiosperm (flowering plants) group which contains 52 plant genera including genus such as <i>Curcuma angustifolia</i>, <i>Curcuma longa</i>, <i>Hedychium</i> sp., and <i>Kaempferia rotunda</i> Linn., is widely distributed in Asian and African regions, known to have diverse pharmacological activities (<a href="http://www.theplantlist.org/1.1/browse/A/zingiberaceae/">http://www.theplantlist.org/1.1/browse/A/zingiberaceae/</a>).</p> <p><i>Curcuma</i> and Ginger are the most imperative known crops in the hill state of Sikkim, India, at North-East Himalayas (27º 00’ 46" N–28º 07’ 48" N latitudes and 88º 00’ 58" E–88º 55’ 25" E longitudes) next to <i>Citrus</i> sp. and <i>Amomum subulatum</i>. The Sikkimese tribes, namely, Lepcha and Bhutia use them for their religious activities besides culinary and medicinal use.[<a href="#ref7">7</a>] Pradhan and Badola[<a href="#ref8">8</a>] stated 118 medicinal plant species at the high altitude of Sikkim practiced by Lepcha tribe of Dzongu valley, distributed across 71 families and 108 genera. Sharma[<a href="#ref9">9</a>] reported that <i>C. angustifolia</i> plant is used for curing worms and stomach ache. Curcumin isolated from <i>C. longa</i> is well known for its pharmacological properties. The antioxidant, antibacterial, antifungal, cytotoxic, and chemopreventive effects are the well-known pharmacological properties of <i>Hedychium</i> sp. (butterfly ginger) in addition to anti-urolithiasis, anti-angiogenic, neuro-pharmacological, fibrinogenolytic, coagulant, anti-allergic, larvicidal, anthelminthic, analgesic, anti-inflammatory, and hepatoprotective activities.[<a href="#ref10">10</a>] The phytochemistry of <i>Hedychium sp</i>. and <i>K. rotunda</i> Linn. (Indian crocus) is still unknown, which makes these species studying worthwhile. Though the pharmacological properties of these rhizomes have been previously reported, the mechanisms of therapeutic principles are still unknown. Hence, the objective of this study was to investigate the pharmacological properties of four indigenous rhizomes in addition to interaction against an estrogen receptor and a series of previously reported compounds (phytochemicals) <i>in silico</i> found in Zingiberaceae (incl. Costaceae) family from Sikkim, India, reported at a higher altitude.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
MATERIALS AND METHODS</div> <div class="chemicalsandreagents" title="sec"> <div class="title" tagx="title" title="title">
Chemicals and reagents</div> <p>Ascorbic acid, 2,2-diphenyl-1-picryl-hydrazyl (DPPH), ferric chloride, gallic acid, naphthyl ethylenediamine dihydrochloride, quercetin, sodium nitrite (NaNO<sub>2</sub>), sulphanilamide, and trichloroacetic acid were obtained from HiMedia Laboratories Pvt. Ltd., Mumbai, India. Di-potassium hydrogen phosphate, hydrogen peroxide, methanol, potassium di-hydrogen phosphate, potassium ferricyanide, potassium hydroxide, sodium hydroxide (NaOH), sodium carbonate (Na<sub>2</sub>CO<sub>3</sub>), sodium acetate, and sodium nitroprusside were procured from Merck, Mumbai, India, and Folin–Ciocalteu (FC) reagent was purchased from Sisco Research Laboratories Pvt. Ltd., Mumbai, India. Aluminum chloride (AlCl<sub>3</sub>) and ortho-phosphoric acid were obtained from SD Fine Chemicals Limited, Mumbai, India. All chemicals and solvents were of analytical grade.</p> </div> <div class="plantmaterialsandextraction" title="sec"> <div class="title" tagx="title" title="title">
Plant materials and extraction</div> <p>The fresh rhizomes of <i>C. angustifolia</i>, <i>C. longa</i>, <i>Hedychium</i> sp., and <i>K. rotunda</i> Linn. were collected in June 2015 from Sajong, Sikkim, India. The specimen vouchers have been identified by a plant taxonomist and deposited at Sikkim State Council of Science and Technology, Sikkim, India, and the given voucher numbers are GT07, GT08, GT09, and GT10. The washed rhizomes were air-dried and macerated using a grinder. The powdered rhizomes were defatted with 1:1 v/v petroleum ether (b.p. 40°C–60°C) in a Soxhlet apparatus. Then, 20 g of each processed sample was extracted using Soxhlet apparatus with 70% aqueous methanol in 1:15 w/v ratio at 70°C for 4 h in three cycles.[<a href="#ref11">11</a>] The lyophilized extracts were stored at 4°C until used. The samples were dissolved in double-distilled water (DDW) in the required aliquots just before use.</p> </div> <div class="preliminaryphytochemicalscreening" title="sec"> <div class="title" tagx="title" title="title">
Preliminary phytochemical screening</div> <p>All four freshly prepared extracts were subjected to preliminary phytochemical analysis according to the standard protocols.[<a href="#ref12">12</a>]</p> </div> <div class="determinationoftheyieldofplantextract" title="sec"> <div class="title" tagx="title" title="title">
Determination of the yield of plant extract</div> <p>The yields of evaporated rhizome extracts were calculated using the author's previous reports.[<a href="#ref3">3</a>]</p> </div> <div class="determinationofbiochemicalconstituents" title="sec"> <div class="title" tagx="title" title="title">
Determination of biochemical constituents</div> <div class="determinationoftotalphenoliccontent" title="sec"> <div class="title" tagx="title" title="title">
Determination of total phenolic content</div> <p>FC reagent method[<a href="#ref13">13</a>] with slight alteration was used to assess total phenolic content. A volume of 0.5 mL of the extracts was mixed with 0.5 mL of FC reagent (previously diluted 1:1 with DDW) and incubated at room temperature (RT) for 5 min. Followed by the addition of 1 mL of 20% Na<sub>2</sub>CO<sub>3</sub> solution, the mixture was incubated at RT for 10 min. The absorbance was measured at 730 nm. Gallic acid monohydrate was used as a standard and the total phenolic content was defined as Gram gallic acid equivalents per 100 g of the extract.</p> </div> <div class="determinationoftotalflavonoidcontent" title="sec"> <div class="title" tagx="title" title="title">
Determination of total flavonoid content</div> <p>The AlCl<sub>3</sub> method was used to determine the total flavonoid content of samples.[<a href="#ref14">14</a>] The reactive mixture was prepared by adding the extract with 1.25 mL DDW, followed by 75 μl of NaNO<sub>2</sub> (5%). After 5 min of incubation at RT, 0.15 mL of AlCl<sub>3</sub> (10%) was added and kept at RT for 6 min. Then, the mixture was treated with 0.5 mL of 1 mM NaOH. Finally, the sample was diluted with 275 μl of DDW followed by incubation at RT for 20 min. The absorbance was measured at 510 nm. The absorbance was measured at 510 nm and the total flavonoid content was determined from quercetin standard curve.</p> </div> <div class="2,2-diphenyl-1-picryl-hydrazylradicalscavengingactivity" title="sec"> <div class="title" tagx="title" title="title">
2,2-diphenyl-1-picryl-hydrazyl radical scavenging activity</div> <p>DPPH method was used to determine the free radical scavenging activity of the sample and the standard, ascorbic acid.[<a href="#ref15">15</a>] A volume of 0.5 mL of 100–1000 μg of the extracts was mixed with freshly prepared 0.5 mL DPPH solution and the final volume was made up to 2 mL. A control sample was the same volume without any extract. Methanol was used as a blank. The reaction mixture is allowed to stand for 30 min in dark. The discoloration was measured at 517 nm. The experiment was performed in triplicates. DPPH scavenging activity (%) was determined using the following equation: DPPH scavenging activity (%) = A<sub>0</sub> − A<sub>1</sub>/A<sub>0</sub> × 100</p> <p>Where A<sub>0</sub> is the absorbance of the control and A<sub>1</sub> is the sample.</p> </div> </div> <div class="dockingstudies" title="sec"> <div class="title" tagx="title" title="title">
Docking studies</div> <p>An <i>in silico</i> study was performed to understand the protein–ligand interaction similarly as per our previously designed protocol.[<a href="#ref16">16</a>]</p> <div class="generationofdataset" title="sec"> <div class="title" tagx="title" title="title">
Generation of dataset</div> <p>Eighty-one compounds of Zingiberaceae (incl. Costaceae) were selected from PubMed (<a href="http://pubmed.ncbi.nlm.nih.gov/">http://pubmed.ncbi.nlm.nih.gov/</a>) and downloaded from PubChem database (<a href="http://pubchem.ncbi.nlm.nih.gov/">http://pubchem.ncbi.nlm.nih.gov/</a>) as ligands considering their chemical and physical properties. The 2D structures of all these compounds were drawn using MarvinSketch, and biological activities were calculated using Molinspiration (<a href="http://www.molinspiration.org">www.molinspiration.org</a>) server.</p> </div> <div class="preparationoftheligands" title="sec"> <div class="title" tagx="title" title="title">
Preparation of the ligands</div> <p>Drawn ligands were processed during the ligand preparation step for various stages having included addition and deletion of hydrogen atoms and heteroatom.[<a href="#ref16">16</a><a href="#ref17">17</a>] In addition, charged group neutralization, low-energy ring conformers, geometry optimization, and saving of output files were performed[<a href="#ref5">5</a><a href="#ref18">18</a>] using Discovery Studio (DS) version 3.5 (Biovia, USA). Lipinski's filter of 5 with various parameters such as hydrogen donors, acceptors, log <i>P</i> value, molecular weight, and the reactive filter were followed to discard the ligands with poor pharmacological properties and reactive functional groups.[<a href="#ref19">19</a>]</p> </div> <div class="preparationofthereceptors" title="sec"> <div class="title" tagx="title" title="title">
Preparation of the receptors</div> <p>The three-dimensional (3D) structure of the receptor (Protein Data Bank [PDB] ID-1U3Q) was downloaded from PDB (<a href="http://www.rcsb.org">http://www.rcsb.org</a>). The X-ray crystal structure of the estrogen receptor has 2.4 Å resolution, <i>R</i>-value free: 0.279, <i>R</i>-value work: 0.227, and is biologically active as well as in a stable state. PDB structure was processed as per the standard accessible protocol of DS version 3.5.[<a href="#ref5">5</a>]</p> </div> <div class="absorption,distribution,metabolism,andexcretion/toxicitypredictionbykomputer-assistedtechnology" title="sec"> <div class="title" tagx="title" title="title">
Absorption, distribution, metabolism, and excretion/TOxicity Prediction by Komputer-assisted Technology</div> <p>The energy-minimized natural compounds were subjected to absorption, distribution, metabolism, and excretion (ADME)/Tox (ADMET) calculations using ADME-TOxicity Prediction by Komputer-assisted Technology (TOPKAT) (DS 3.5). The TOPKAT predicts structurally significant candidates and pharmaceutically relevant properties such as Ames mutagenicity, carcinogenicity, rat oral LD<sub>50</sub>, rat maximum tolerated dose, rat inhalation toxicity LC<sub>50</sub>, and Log <i>P</i> of the probable candidates. ADMET descriptor includes various parameters such as human intestinal absorption, CYP2D6 binding, plasma protein binding, blood–brain barrier (BBB) penetration, and aqueous solubility.</p> </div> <div class="simulationstudies" title="sec"> <div class="title" tagx="title" title="title">
Simulation studies</div> <p>The study was followed to explore the protein–ligand interactions. Docking allows virtually screening compounds and predicting the strongest binders based on their scoring functions. The software used here to do docking simulation was iGEMDOCK against all prepared compounds. This system's total energy alias Gibbs energy (kcal/mol) is calculated and recorded by the system.</p> </div> <div class="ligandfit/discoverystudio3.5" title="sec"> <div class="title" tagx="title" title="title">
LigandFit/Discovery Studio 3.5</div> <p>Ligand fit is a shape-directed docking method of ligands to bind with the specific active sites of the protein.[<a href="#ref20">20</a>] To initiate docking study in DS 3.5, receptor or target protein was prepared by removing heteroatoms, water molecules. The Chemistry at HARvard Macromolecular Mechanics (CHARMM) force field was applied using the simulation tool DS 3.5. The active site was identified using cavity-based method from receptor cavities. A virtually screened compound after ADME and TOPKAT was subjected to docking process. Top 20 poses/scores were generated for each interaction. Each pose was saved, evaluated, and ranked based on DS consensus scoring function that includes JAIN, Ligscore 1, Ligscore 2, PLP1, PLP2, and potential of mean force. Hydrogen bond interactions between the ligands and active site residues were also assessed.</p> </div> <div class="moleculardynamicsimulation" title="sec"> <div class="title" tagx="title" title="title">
Molecular dynamic simulation</div> <p>The final screened compound having the best pose of docked complex from the docking studies (1U3Q-apo protein complex) was subjected to 200 picosecond (ps) of MD simulations using leap-frog verlet integration algorithm in DS 3.5. In addition, SHAKE constraint was introduced to fix all bonds involving hydrogen in the simulation. The executed simulation cascades were minimization with CHARMm force field followed by minimization with steepest descent of 500 steps and conjugate gradient of 500 steps. Consequently, the docked complex is heated upto 300K with 4 ps of simulation time. Finally, production in the canonical ensemble was subjected with equal T<sub>mass</sub> and P<sub>mass</sub> at 300 k for 200 ps. In addition, spherical cutoff method of electrostatics was implemented to study all nonbonded energy without using periodic boundary condition.</p> </div> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>All experiments reported were done in triplicates. The Tukey range test was used to check their statistical significance at <i>P</i> &amp;lt; 0.05.</p> </div> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
RESULTS AND DISCUSSION</div> <p>In the modern era, Zingiberaceae family has attained an interest as a potential treatment for many health ailments,[<a href="#ref21">21</a>] including anti-inflammatory disorders[<a href="#ref22">22</a>] and cancer.[<a href="#ref23">23</a>] Till March 2016, only 56 scientific articles were found for the keyword "Zingiberaceae" and "Phytochemistry in the pubmed repository. Most of the species reported were <i>Zingiber</i> sp. (23), <i>Curcuma</i> sp. (9), <i>Kaempferia</i> sp. (3), <i>Hedychium</i> sp. (3), <i>Aframomum</i> sp. (1), <i>Alpinia</i> sp. (1), <i>A. subulatum</i> (1), <i>Boesenbergia</i> sp. (1), <i>Renealmia</i> sp. (1), <i>Siphonochilus</i> sp. (1), and <i>Xiphidium</i> sp. (1) with 73% of all the available publications. However, the authors were not able to find any report with regard to <i>A. purpurata</i> and <i>H. coronarium</i> from North-East India, specifically at high latitude (27.20 N) and longitude (88.40 E). Moreover, there were lots of scientific lacunae in case of any works addressing the phytochemistry of <i>Hedychium</i> sp., <i>K. rotunda</i> Linn., and <i>Curcuma</i> sp. To the authors, it was important to notice that well-documented species of this Zingiberaceae family only represent 5%–10% of the species, and the vast majority belonging to this family remain poorly studied or biochemically unknown. Hence, the present study focused on the phytochemistry and <i>in silico</i> findings of selected species of Zingiberaceae family.</p> <div class="biochemicalinvestigation" title="sec"> <div class="title" tagx="title" title="title">
Biochemical investigation</div> <p>Antioxidants are the molecules to quench the free radicals synthesized in the plants as the inevitable byproducts of turning food into energy and formidable threats faced by them in disease condition.[<a href="#ref3">3</a>] Polyphenols, naturally occurring in our food, are scientifically proven for the major antioxidant activities.[<a href="#ref24">24</a>] The preliminary tests indicated the presence of various constituents such as phenols, saponins, carbohydrates, tannins, flavonoids, and alkaloids in different edible rhizome extracts. The phenols and flavonoids are the most investigated phytochemicals with regard to curing health ailments. Several reports suggested that these phytochemicals are active against diabetes,[<a href="#ref1">1</a><a href="#ref25">25</a>] cancer,[<a href="#ref16">16</a>] oxidative diseases,[<a href="#ref6">6</a><a href="#ref15">15</a>] and inflammatory diseases.[<a href="#ref3">3</a><a href="#ref26">26</a>] The present study aimed to analyze the comparative antioxidative properties of the herbs classified under Zingiberaceae family and their possible anticancer potentials against an estrogen receptor. The extracting solvent used was methanol, and the conditions were based on previous reports.[<a href="#ref27">27</a><a href="#ref28">28</a>] Total extract yield was ranging between 1.65% and 3.20% (w/w) for all the four rhizomes [<a href="#T1">Table 1</a>].</p> <div class="table-wrap_UNKNOWN" id="T1" position="float"><span class="label" tagx="label" title="label">Table 1</span> <p>Quantitative phytochemical assay in different rhizomes</p>  <div class="graphic" /> </div> <p>A low-to-high pattern such as <i>K. rotunda</i> Linn. &amp;lt; <i>Hedychium</i> sp. &amp;lt; <i>C. longa &amp;lt; C. angustifolia</i> was seen in case of water contents [<a href="#T1">Table 1</a>]. <i>Hedychium</i> sp. was found to be the richest in flavonoids, followed by both <i>Curcuma</i> species, though in case of the total phenol, <i>C. angustifolia</i> showed highest phenols followed by <i>Hedychium</i> sp., <i>C. longa</i>, and <i>K. rotunda</i> Linn [<a href="#T1">Table 1</a>].</p> </div> <div class="freeradicalscavengingactivity(2,2-diphenyl-1-picryl-hydrazylmethod)" title="sec"> <div class="title" tagx="title" title="title">
Free radical scavenging activity (2,2-diphenyl-1-picryl-hydrazyl method)</div> <p>All the rhizomes showed free radical scavenging activity as measured by DPPH with the concentrations ranging from 100 to 500 μg/mL [<a href="#F1">Figure 1</a>]. A dose-dependent approach in DPPH scavenging activity was noticed in the methanolic extract of different rhizomes of Zingiberaceae family. The maximum percentage inhibition was observed at a concentration of 500 μg/mL with 79.5% inhibition by <i>Hedychium</i> sp. (IC<sub>50</sub> 36.4 μg/mL), whereas that of the standard (ascorbic acid) was found to be 81.76%. The highest radical scavenging activity suggests that all the indigenous rhizomes of Zingiberaceae family might be full of potent-free radical inhibitors, which can act as potential antioxidants.[<a href="#ref29">29</a>] Therefore, as a result, these extracts can scavenge the DPPH*radicals to form constantly reduced DPPH molecules.[<a href="#ref27">27</a>] The methanolic extract of all the four species found to have high DPPH scavenging activities and have already been reported in comparison to the aqueous and acetone extract by other researchers in different plant materials.[<a href="#ref15">15</a><a href="#ref16">16</a><a href="#ref24">24</a><a href="#ref27">27</a>] The DPPH activity in a dose-dependent manner suggested that these rhizomes can serve as a potential targets for several health problems.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Comparative 2,2-diphenyl-1-picryl-hydrazyl scavenging activity of methanolic extract of different rhizomes of Zingiberaceae family</p>   </div> </div> <div class="insilicostudies" title="sec"> <div class="title" tagx="title" title="title"><i>In silico</i> studies</div> <p>Molecular docking, a vital tool, has gained a lot of popularity in structural molecular biology and CADD. The aim of the ligand–protein docking is to determine the interaction of a chemical entity (ligand) with a receptor (protein) of already existing 3D structure. First, the purpose of this stochastic interaction study is hypothesized to know a better candidate and second how the drug like molecules (ligands) inhibit the target.</p> <p>The prediction methods use scoring functions to approximately calculate the binding energies of envisaged ligand–receptor complex. The estimated binding energy (combination of binding constant [Kd] and the Gibbs-free energy [ΔGL]) is carried out by evaluating the most vital physical–chemical phenomena involved in inter-molecular interactions and entropic effects.[<a href="#ref29">29</a>] Based on scoring function, a probable molecule would be ranked.</p> <p>In this study, we have collected common molecules found in Zingiberaceae family <i>in litero</i>. Lead optimization was done using Lipinski rule of 5 for the molecules reported earlier.[<a href="#ref5">5</a>] Only 71 entities have passed out of 81 compounds.</p> <p>Lowest energy stabilization is directly proportional to maximum hydrogen bonding interactions with regard to protein–ligand complex and selected as the best pose possible. β-phellandrene (−96.8) has shown the best docking energy score as followed by t-linalool oxide (−87.6) and zingerone (−83.4) [<a href="#T2">Table 2</a>]. There are previous studies showing β-phellandrene as a potent candidate for cancer studies.[<a href="#ref30">30</a><a href="#ref31">31</a>] Song <i>et al</i>.[<a href="#ref32">32</a>] indicated zingerone to ameliorate lipopolysaccharide-induced acute kidney injury by inhibiting toll-like receptor-4 signaling pathway. Chang <i>et al</i>.[<a href="#ref33">33</a>] showed the potency of linalool oxide against cancer cell lines. They reported that linalool can promote gene expression for p53, p21, p27, p16, and p18 genes that can arrest the G0/G1 phase as well as G2/M phase.</p> <div class="table-wrap_UNKNOWN" id="T2" position="float"><span class="label" tagx="label" title="label">Table 2</span> <p>Interaction score of the best molecule with bonding pattern using iGemDock</p>  <div class="graphic" /> </div> <p>A hydrogen bonding interaction could be pragmatic between the β-phellandrene and the hydroxyl group of Arg<sup>346</sup> [<a href="#T2">Table 2</a>]. Therefore, this could only functional as a weak hydrogen bond acceptor, which is not very physiologically relevant. Few van der Wall interactions are incidental and formed. The major three ligands such as β-phellandrene, trans linalool oxide, and zingerone mutually interacted with amino acids such as Arg<sup>346</sup>, Leu<sup>298</sup>, Leu<sup>349/343/476</sup>, Phe<sup>356</sup>, and Leu<sup>476</sup> [<a href="#T2">Table 2</a>].</p> </div> <div class="moleculardynamics(simulations)studies" title="sec"> <div class="title" tagx="title" title="title">
Molecular dynamics (simulations) studies</div> <p>The best pose of 1U3Q – β-phellandrene – complex from virtual screening and LigandFit was selected, and the apo-form of protein (1U3Q) was subjected to 200 ps molecular dynamics (MD) simulations using DS version 3.5. The stability of the protein was analyzed and confirmed by plotting root mean square deviation (RMSD) and radius of gyration (Rg) of protein [<a href="#F2">Figure 2a</a>]. The RMSD is a measure of the deviation of the conformational stability of the proteins from backbone structure to the early starting structure and fundamental property investigation in MD studies.[<a href="#ref34">34</a>] The RMSD plot of apo-form of protein and 1U3Q – β-phellandrene complex structure has shown deviation between 2.15 and 2.25 Å [<a href="#F2">Figure 2b</a>]. Root mean square fluctuation (RMSF) of main chain Ca atoms is also calculated with reference to a primary structure for average fluctuation of all residues during the simulation and to locate the flexibility of each amino acid residue along the polypeptide chain. An insight of computed results of RMSF showed that the major alpha-helices are more stable and stabilized [<a href="#F2">Figure 2c</a>]. Perhaps, slight changes in deviation were observed in C-terminal alpha helices of apo-form of protein and the loop regions are flexible in both structures. The Rg is also calculated to analyze the folding pattern of the protein during simulation[<a href="#ref35">35</a>] and is defined as the root mean square distance of the collection of atoms from their common center of gravity. The results of gyration showed not much of deviation in folding pattern of protein [<a href="#F2">Figure 2d</a>]. The overall simulation of 1U3Q – β-phellandrene complex showed more stability (−12585.238 kcal/mol) than an apo-form of protein (−12438.166 kcal/mol) variation in energy parameters with respect to time and temperature [<a href="#T3">Table 3</a>]. <a href="#F3">Figure 3a</a>–<a href="#F3">d</a> shows the two best molecules, namely, β-phellandrene and farnesene epoxide interacting with 1U3Q.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>(a) Molecular interaction of β-phellandrene, (b) molecular interaction of farnesene epoxide, (c) absorption, distribution, metabolism, and excretion-Tox plot, (d) 1U3Q-farnesene epoxide complex</p>   </div> <div class="table-wrap_UNKNOWN" id="T3" position="float"><span class="label" tagx="label" title="label">Table 3</span> <p>Energy parameter variations in different stages of molecular dynamics</p>  <div class="graphic" /> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>(a) 1U3Q-β-phellandrene complexes, (b) root mean square deviation plot of 1U3Q-β-phellandrene complexes and 1U3Q-apo protein, during 200 ps simulations. (c) Root mean square fluctuation plot of 1U3Q-β-phellandrene complexes and 1U3Q-apo protein, during 200 ps simulations. (d) Radius of gyration plot of 1U3Q-β-phellandrene complexes and 1U3Q-apo protein</p>   </div> </div> <div class="absorption,distribution,metabolism,andexcretionandtoxicitystudy" title="sec"> <div class="title" tagx="title" title="title">
Absorption, distribution, metabolism, and excretion and toxicity study</div> <p>ADME-T plot [<a href="#F3">Figure 3c</a>] shows 95% and 99% confidence ellipsoids for absorption and BBB, respectively. Farnesene epoxide (55.86) solubility is lower than β-phellandrene (36.61) with good absorption, nonhepatotoxic activity, and high penetration across BBB [<a href="#T1">Table 1</a>]. Unlike other compounds, β-phellandrene had showed a nontoxic effect in developmental toxicity potential. Hence, authors considered β-phellandrene a better target for estrogen receptor. The two best molecules, i.e., β-phellandrene and farnesene epoxide are forming 11 and 14 favorable nonbond interactions, respectively [<a href="#F3">Figure 3a</a> and <a href="#F3">b</a>]. The bond-forming residues in case of compound β-phellandrene are Leu<sup>298</sup>, Ala<sup>302</sup>, Met<sup>336</sup>, Leu<sup>339</sup>, Leu<sup>343</sup>, and Phe<sup>356</sup> and farnesene epoxide are Ala<sup>302</sup>, Glu<sup>305</sup>, Met<sup>340</sup>, Leu<sup>343</sup>, Arg<sup>346</sup>, Phe<sup>356</sup>, Ile<sup>373</sup>, Ile<sup>376</sup>, Leu<sup>380</sup>, His<sup>475</sup>, Leu<sup>476</sup>, and Leu<sup>491</sup> [<a href="#F3">Figure 3a</a> and <a href="#F3">b</a>]. <a href="#F3">Figure 3d</a> depicts 1U3Q-farnesene epoxide complex. TOPKAT employs robust and cross-validated Quantitative Structure Toxicity Relationship representation for assessing various measures of toxicity and utilizing the patented optimal predictive space validation method to assist in interpreting the results.[<a href="#ref16">16</a>] Screened compounds showed a nonmutagenic effect when subjected to Ames test [<a href="#T4">Table 4</a>]. Trans linalool oxide and zingerone showed optimal solubility and were less favorable to cross BBB while E, E alpha-farnesene had a moderate absorption.</p> <div class="table-wrap_UNKNOWN" id="T4" position="float"><span class="label" tagx="label" title="label">Table 4</span> <p>Absorption, distribution, metabolism, and excretion and toxicity descriptors for top five drug-like compounds</p>  <div class="graphic" /> </div> </div> </div> <div class="conclusionandfutureprospects" title="sec"> <div class="title" tagx="title" title="title">
CONCLUSION AND FUTURE PROSPECTS</div> <p>To conclude, among all the genus of Zingiberaceae family, <i>Hedychium</i> sp. possess more active compounds than other varieties, followed by <i>K. rotunda Linn</i>., <i>C. angustifolia</i>, and <i>C. longa</i>. This study is an effort to investigate the interaction of Zingiberaceae family compounds and predict the best active compound for anticancerous activity. The bioassay-guided isolation and identification of the bioactive components are still need to be explored. In addition, a detailed research is also required to reveal the structure–activity relationship of these active compounds. The outcome of this research in the aforementioned areas will provide a convincing support for the future clinical uses of these rhizomes in modern medicine.</p> <div class="financialsupportandsponsorship" title="sec"> <div class="title" tagx="title" title="title">
Financial support and sponsorship</div> <p>Nil.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Conflicts of interest</div> <p>There are no conflicts of interest.</p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors thank DBT-BIF Facility, Government of India (GOI); BT-Finishing School, GOK, Biotech hub Sikkim, DBT, GOI; and Maharani Lakshmi Ammanni College for Women management, for providing the necessary facility.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="ref1" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middha</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faizan</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanghamitra</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basistha</span><span class="given-names" tagx="given-names" title="given-names">BC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usha</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">In silico-based combinatorial pharmacophore modelling and docking studies of GSK-3β and GK inhibitors of hippophae</span><span class="source" tagx="source" title="source">J Biosci</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">805</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24287660">24287660</a></span></span></li> <li tag="ref"><a name="ref2" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasote</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katyare</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hegde</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bae</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Significance of antioxidant potential of plants and its relevance to therapeutic applications</span><span class="source" tagx="source" title="source">Int J Biol Sci</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">982</span><span class="lpage" tagx="lpage" title="lpage">91</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26157352">26157352</a></span></span></li> <li tag="ref"><a name="ref3" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middha</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lokesh</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prashanth</span><span class="given-names" tagx="given-names" title="given-names">HP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhardwaj</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamal</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">In silico exploration of cyclooxygenase inhibitory activity of natural compounds found in <i>Myrica nagi</i> using LC-MS</span><span class="source" tagx="source" title="source">Symbiosis</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="lpage" tagx="lpage" title="lpage">78</span></span></li> <li tag="ref"><a name="ref4" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usha</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lubna</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prashanth</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohan</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pande</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of anti-cancer targets of eco-friendly waste <i>Punica granatum</i> peel by dual reverse virtual screening and binding analysis</span><span class="source" tagx="source" title="source">Asian Pac J Cancer Prev</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">10345</span><span class="lpage" tagx="lpage" title="lpage">50</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25556473">25556473</a></span></span></li> <li tag="ref"><a name="ref5" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usha</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middha</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karthik</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manoj</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faizan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular docking studies of anti-cancerous candidates in <i>Hippophae rhamnoides</i> and <i>Hippophae salicifolia</i></span><span class="source" tagx="source" title="source">J Biomed Res</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">406</span><span class="lpage" tagx="lpage" title="lpage">15</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25332713">25332713</a></span></span></li> <li tag="ref"><a name="ref6" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middha</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usha</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ameliorating reactive oxygen species-induced <i>in vitro</i> lipid peroxidation in liver, carbohydrate and DNA damage by <i>Dendrocalamus hamiltonii</i> different leaf extracts</span><span class="source" tagx="source" title="source">Chiang Mai J Sci</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">1131</span><span class="lpage" tagx="lpage" title="lpage">42</span></span></li> <li tag="ref"><a name="ref7" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basistha</span><span class="given-names" tagx="given-names" title="given-names">BC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Handique</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pradhan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Determination of genetic fidelity of Micropropagate d plants of <i>Zingiber officinale</i> cv- Majhauley of Sikkim Himalaya using RAPD markers</span><span class="source" tagx="source" title="source">Int J Fund Appl Sci</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">20</span><span class="lpage" tagx="lpage" title="lpage">3</span></span></li> <li tag="ref"><a name="ref8" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pradhan</span><span class="given-names" tagx="given-names" title="given-names">BK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Badola</span><span class="given-names" tagx="given-names" title="given-names">HK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ethnomedicinal plant use by Lepcha tribe of Dzongu valley, bordering Khangchendzonga biosphere reserve, in North Sikkim, India</span><span class="source" tagx="source" title="source">J Ethnobiol Ethnomed</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">22</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18826658">18826658</a></span></span></li> <li tag="ref"><a name="ref9" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Traditional processing of Shotti (<i>Curcuma angustifolia</i> Roxb.). - A rhizome based ethnic weaning food</span><span class="source" tagx="source" title="source">Indian J Tradit Knowl</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">154</span><span class="lpage" tagx="lpage" title="lpage">5</span></span></li> <li tag="ref"><a name="ref10" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">EW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phytochemistry and pharmacology of ornamental gingers <i>Hedychium coronarium</i> and <i>Alpinia purpurata</i>: A review</span><span class="source" tagx="source" title="source">J Integr Med</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">368</span><span class="lpage" tagx="lpage" title="lpage">79</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26559362">26559362</a></span></span></li> <li tag="ref"><a name="ref11" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandra</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhat</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arora</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nailwal</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pande</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>In vitro</i> antioxidant and free radical scavenging activity of <i>Macrotyloma uniflorum</i> (Gahat dal.). from Kumauni region</span><span class="source" tagx="source" title="source">Int J Fund Appl Sci</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="lpage" tagx="lpage" title="lpage">10</span></span></li> <li tag="ref"><a name="ref12" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middha</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of DPPH radical scavenging activity, total phenols and antioxidant activities in Indian wild <i>Bambusa vulgaris</i>’Vittata’methanolic leaf extract</span><span class="source" tagx="source" title="source">J Nat Pharm</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">40</span><span class="lpage" tagx="lpage" title="lpage">5</span></span></li> <li tag="ref"><a name="ref13" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ainsworth</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gillespie</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Estimation of total phenolic content and other oxidation substrates in plant tissues using Folin-Ciocalteu reagent</span><span class="source" tagx="source" title="source">Nat Protoc</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">875</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17446889">17446889</a></span></span></li> <li tag="ref"><a name="ref14" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhishen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mengcheng</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jianming</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">The determination of flavonoid content in mulberry and their scavenging effects on superoxide radicals</span><span class="source" tagx="source" title="source">Food Chem</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">555</span><span class="lpage" tagx="lpage" title="lpage">9</span></span></li> <li tag="ref"><a name="ref15" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basistha</span><span class="given-names" tagx="given-names" title="given-names">BC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middha</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antioxidant profiling of <i>Hippophae salicifolia</i> growing in sacred forests of Sikkim, India</span><span class="source" tagx="source" title="source">Funct Plant Biol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">697</span><span class="lpage" tagx="lpage" title="lpage">701</span></span></li> <li tag="ref"><a name="ref16" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usha</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tripathi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pande</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middha</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular docking and quantum mechanical studies on pelargonidin-3-glucoside as renoprotective ACE inhibitor</span><span class="source" tagx="source" title="source">ISRN Comput Biol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">2013</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">4</span></span></li> <li tag="ref"><a name="ref17" /><span class="label" tagx="label" title="label">17</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suri</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naik</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Elucidating the precise interaction of reduced and oxidized states of neuroglobin with Ubc12 and Cop9 using molecular mechanics studies</span><span class="source" tagx="source" title="source">Int J Fund Appl Sci</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">74</span><span class="lpage" tagx="lpage" title="lpage">7</span></span></li> <li tag="ref"><a name="ref18" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maggiora</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mezey</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chou</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span></span><span class="mixed-article-title" title="mixed-article-title">A new chiral feature in α-helical domains of proteins</span><span class="source" tagx="source" title="source">Biopolymers</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">211</span><span class="lpage" tagx="lpage" title="lpage">4</span></span></li> <li tag="ref"><a name="ref19" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipinski</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lombardo</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dominy</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feeney</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span><span class="source" tagx="source" title="source">Adv Drug Deliv Rev</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="lpage" tagx="lpage" title="lpage">26</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11259830">11259830</a></span></span></li> <li tag="ref"><a name="ref20" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venkatachalam</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oldfield</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waldman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ligand Fit: A novel method for the shape-directed rapid docking of ligands to protein active sites</span><span class="source" tagx="source" title="source">J Mol Graph Model</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">289</span><span class="lpage" tagx="lpage" title="lpage">307</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12479928">12479928</a></span></span></li> <li tag="ref"><a name="ref21" /><span class="label" tagx="label" title="label">21</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lakhan</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ford</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tepper</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Zingiberaceae extracts for pain: A systematic review and meta-analysis</span><span class="source" tagx="source" title="source">Nutri J</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">50</span></span></li> <li tag="ref"><a name="ref22" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wohlmuth</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deseo</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brushett</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacFarlane</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterman</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevenson</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biological activity and novel cytotoxic curcuminoid from <i>Pleuranthodium racemigerum</i>–An Australian <i>Zingiberaceae</i></span><span class="source" tagx="source" title="source">Planta Medica</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">73</span><span class="fpage" tagx="fpage" title="fpage">947</span></span></li> <li tag="ref"><a name="ref23" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anasamy</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdul</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sukari</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelwahab</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamalidehghan</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A phenylbutenoid dimer, cis-3-(3¢,4¢-dimethoxyphenyl)-4-[(E)-3¢¢¢,4¢¢¢-dimethoxystyryl] cyclohex-1-ene, exhibits apoptogenic properties in T-acute lymphoblastic leukemia cells via induction of p53-independent mitochondrial signalling pathway</span><span class="source" tagx="source" title="source">Evid Based Complement Altern Med</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">2013</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">14</span></span></li> <li tag="ref"><a name="ref24" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brglez</span><span class="given-names" tagx="given-names" title="given-names">Mojzer E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knez</span><span class="given-names" tagx="given-names" title="given-names">Hrnčič M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Škerget</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knez</span><span class="given-names" tagx="given-names" title="given-names">Ž</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bren</span><span class="given-names" tagx="given-names" title="given-names">U</span></span></span><span class="mixed-article-title" title="mixed-article-title">Polyphenols: Extraction methods, antioxidative action, bioavailability and anticarcinogenic effects</span><span class="source" tagx="source" title="source">Molecules</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">901</span></span></li> <li tag="ref"><a name="ref25" /><span class="label" tagx="label" title="label">25</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middha</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usha</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misra</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lokesh</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of antioxidative, analgesic and anti-inflammatory activities of methanolic extract of <i>Myrica nagi</i> leaves-an animal model approach</span><span class="source" tagx="source" title="source">Symbiosis</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">179</span><span class="lpage" tagx="lpage" title="lpage">84</span></span></li> <li tag="ref"><a name="ref26" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pande</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rizvi</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant polyphenols as dietary antioxidants in human health and disease</span><span class="source" tagx="source" title="source">Oxid Med Cell Longev</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">270</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20716914">20716914</a></span></span></li> <li tag="ref"><a name="ref27" /><span class="label" tagx="label" title="label">27</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Surojanametakul</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satmalee</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saengprakai</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siliwan</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wattanasiritham</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Preparation of curcuminoid powder from turmeric root (<i>Curcuma longa</i> Linn) for food ingredient use</span><span class="source" tagx="source" title="source">Kasetsart J Nat Sci</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">123</span><span class="lpage" tagx="lpage" title="lpage">30</span></span></li> <li tag="ref"><a name="ref28" /><span class="label" tagx="label" title="label">28</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devaki</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nirupama</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nirupama</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yajurvedi</span><span class="given-names" tagx="given-names" title="given-names">HN</span></span></span><span class="mixed-article-title" title="mixed-article-title">Protective effect of rhizome extracts of the herb, vacha (<i>Acorus calamus</i>) against oxidative damage: An <i>in vivo</i> and <i>in vitro</i> study</span><span class="source" tagx="source" title="source">Food Sci Hum Well</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">76</span><span class="lpage" tagx="lpage" title="lpage">84</span></span></li> <li tag="ref"><a name="ref29" /><span class="label" tagx="label" title="label">29</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span></span><span class="mixed-article-title" title="mixed-article-title">Scoring functions for protein-ligand docking</span><span class="source" tagx="source" title="source">Curr Protein Pept Sci</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">407</span><span class="lpage" tagx="lpage" title="lpage">20</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17073693">17073693</a></span></span></li> <li tag="ref"><a name="ref30" /><span class="label" tagx="label" title="label">30</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hakkim</span><span class="given-names" tagx="given-names" title="given-names">FL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Buloshi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Sabahi</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Frankincense derived heavy terpene cocktail boosting breast cancer cell (MDA-MB-231) death <i>in vitro</i></span><span class="source" tagx="source" title="source">Asian Pacif J Trop Biomed</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">824</span><span class="lpage" tagx="lpage" title="lpage">8</span></span></li> <li tag="ref"><a name="ref31" /><span class="label" tagx="label" title="label">31</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piras</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosa</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marongiu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atzeri</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dessì</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falconieri</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Extraction and separation of volatile and fixed oils from seeds of <i>Myristica fragrans</i> by supercritical CO<sub>2</sub>: Chemical composition and cytotoxic activity on CaCo-2 cancer cells</span><span class="source" tagx="source" title="source">J Food Sci</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">C448</span><span class="lpage" tagx="lpage" title="lpage">53</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22429024">22429024</a></span></span></li> <li tag="ref"><a name="ref32" /><span class="label" tagx="label" title="label">32</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Zingerone ameliorates lipopolysaccharide-induced acute kidney injury by inhibiting toll-like receptor 4 signaling pathway</span><span class="source" tagx="source" title="source">Eur J Pharmacol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">772</span><span class="fpage" tagx="fpage" title="fpage">108</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26698392">26698392</a></span></span></li> <li tag="ref"><a name="ref33" /><span class="label" tagx="label" title="label">33</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shieh</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeh</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">HP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Linalool induces cell cycle arrest and apoptosis in leukemia cells and cervical cancer cells through CDKIs</span><span class="source" tagx="source" title="source">Int J Mol Sci</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">28169</span><span class="lpage" tagx="lpage" title="lpage">79</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26703569">26703569</a></span></span></li> <li tag="ref"><a name="ref34" /><span class="label" tagx="label" title="label">34</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuzmanic</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zagrovic</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Determination of ensemble-average pairwise root mean-square deviation from experimental B-factors</span><span class="source" tagx="source" title="source">Biophys J</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">98</span><span class="fpage" tagx="fpage" title="fpage">861</span><span class="lpage" tagx="lpage" title="lpage">71</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20197040">20197040</a></span></span></li> <li tag="ref"><a name="ref35" /><span class="label" tagx="label" title="label">35</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindahl</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hess</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Spoel</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">GROMACS 3.0: A package for molecular simulation and trajectory analysis</span><span class="source" tagx="source" title="source">J Mol Model</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">306</span><span class="lpage" tagx="lpage" title="lpage">17</span></span></li> </ul> </div> </div>  </body></html>