<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Biotechnol Biofuels</span><span class="iso-abbrev" title="iso-abbrev">Biotechnol Biofuels</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Biotechnology for Biofuels</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1754-6834</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7874482">7874482</a></span><span class="publisher-id" title="publisher-id">1880</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s13068-021-01880-7">10.1186/s13068-021-01880-7</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
CRISPRi screens reveal genes modulating yeast growth in lignocellulose hydrolysate</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gutmann</span><span class="given-names" tagx="given-names" title="given-names">Friederike</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jann</span><span class="given-names" tagx="given-names" title="given-names">Cosimo</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">jann@embl.de</span></span><a href="#Aff3">3</a><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pereira</span><span class="given-names" tagx="given-names" title="given-names">Filipa</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">filipa.pereira@embl.de</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johansson</span><span class="given-names" tagx="given-names" title="given-names">Andreas</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinmetz</span><span class="given-names" tagx="given-names" title="given-names">Lars M.</span></span><a href="#Aff3">3</a><a href="#Aff5">5</a><a href="#Aff6">6</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-6166-8640</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patil</span><span class="given-names" tagx="given-names" title="given-names">Kiran R.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">patil@mrc-tox.cam.ac.uk</span></span><a href="#Aff1">1</a><a href="#Aff7">7</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.4709.a</span><span class="institution-id" title="institution-id">0000 0004 0495 846X</span><span class="institution" title="institution">European Molecular Biology Laboratory (EMBL), Structural and Cell Biology Unit, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.419491.0</span><span class="institution-id" title="institution-id">0000 0001 1014 0849</span><span class="institution" title="institution">Max DelbrÃ¼ck Center for Molecular Medicine in the Helmholtz Association (MDC), </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.4709.a</span><span class="institution-id" title="institution-id">0000 0004 0495 846X</span><span class="institution" title="institution">European Molecular Biology Laboratory (EMBL), Genome Biology Unit, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.5801.c</span><span class="institution-id" title="institution-id">0000 0001 2156 2780</span><span class="institution" title="institution">Department of Biology, Institute of Biochemistry, </span><span class="institution" title="institution">ETH Zurich, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.168010.e</span><span class="institution-id" title="institution-id">0000000419368956</span><span class="institution" title="institution">Department of Genetics, </span><span class="institution" title="institution">Stanford University School of Medicine, </span></div></span><span class="citation_author_institution" id="Aff6">[6], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.168010.e</span><span class="institution-id" title="institution-id">0000000419368956</span><span class="institution" title="institution">Stanford Genome Technology Center, </span></div></span><span class="citation_author_institution" id="Aff7">[7], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.5335.0</span><span class="institution-id" title="institution-id">0000000121885934</span><span class="institution" title="institution">MRC Toxicology Unit, </span><span class="institution" title="institution">University of Cambridge, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-2-2</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-2-2</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">14</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">41</span><span class="history" title="history"><span class="received" title="received">received: 2020-9-17</span><span class="accepted" title="accepted">accepted: 2021-1-7</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1">Bakerâ€™s yeast is a widely used eukaryotic cell factory, producing a diverse range of compounds including biofuels and fine chemicals. The use of lignocellulose as feedstock offers the opportunity to run these processes in an environmentally sustainable way. However, the required hydrolysis pretreatment of lignocellulosic material releases toxic compounds that hamper yeast growth and consequently productivity.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par2">Here, we employ CRISPR interference in <i>S. cerevisiae</i> to identify genes modulating fermentative growth in plant hydrolysate and in presence of lignocellulosic toxins. We find that at least one-third of hydrolysate-associated gene functions are explained by effects of known toxic compounds, such as the decreased growth of <i>YAP1</i> or <i>HAA1</i>, or increased growth of <i>DOT6</i> knock-down strains in hydrolysate.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par3">Our study confirms previously known genetic elements and uncovers new targets towards designing more robust yeast strains for the utilization of lignocellulose hydrolysate as sustainable feedstock, and, more broadly, paves the way for applying CRISPRi screens to improve industrial fermentation processes.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Lignocellulose hydrolysate</div> <div class="kwd" title="kwd">
Growth-inhibitor compounds</div> <div class="kwd" title="kwd">
CRISPR interference screen</div> <div class="kwd" title="kwd">
Yeast fermentation</div> <div class="kwd" title="kwd">
Sustainable biotechnology</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100010663</span><span class="institution" title="institution">H2020 European Research Council</span></div> </div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001659</span><span class="institution" title="institution">Deutsche Forschungsgemeinschaft</span></div> </div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100008662</span><span class="institution" title="institution">Joachim Herz Stiftung</span></div> </div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Projekt DEAL</span></div> </div> <div class="unknown" title="open-access" tag="open-access"> <p>Open Access funding enabled and organized by Projekt DEAL.</p> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par11">The bakerâ€™s yeast <i>S. cerevisiae</i> is the most frequently used eukaryotic cell factory [<a href="#CR9">9</a>, <a href="#CR35">35</a>, <a href="#CR53">53</a>, <a href="#CR59">59</a>]. The competitive production of biofuels and many other value-added compounds with yeast requires the use of cheap substrates that do not compete with food, feed and arable land. Lignocellulosic materials represent an economic and environmentally sustainable alternative feedstock. Spruce softwood is a promising lignocellulose source in the northern hemisphere [<a href="#CR85">85</a>, <a href="#CR87">87</a>] and an abundant side product of the lumber, pulp and paper industry [<a href="#CR71">71</a>]. Lignocellulose has a complex structure, largely consisting of cellulose, hemicellulose and lignin. The extraction of fermentable mono-saccharide sugars (glucose) and hemicellulose polymers (pentose and hexose sugars) from cellulose biomass requires hydrolysis pretreatment. During this processing step, toxic compounds are released to the soluble hydrolysate which represent a major challenge in using this feedstock in an industrial setting [<a href="#CR2">2</a>, <a href="#CR5">5</a>, <a href="#CR64">64</a>, <a href="#CR66">66</a>, <a href="#CR82">82</a>]. These compounds can be classified in three main groups: aliphatic acids, furan aldehydes and phenolic/aromatic derivativesÂ [<a href="#CR37">37</a>]. Aliphatic acids lower the intracellular pH and interfere with nutrient uptake [<a href="#CR61">61</a>]. Furans inhibit dehydrogenases and raise levels of reactive oxygen species [<a href="#CR1">1</a>, <a href="#CR7">7</a>]. Phenolics perturb plasma membrane composition and potential, resulting in a disruption of cell signalling and sorting processes [<a href="#CR40">40</a>]. Understanding the impact of these substances on yeast growth and the cellular mechanisms of tolerance will enhance the use of lignocellulose-based biotechnological processes.</p> <p id="Par12">Several efforts have been made to characterize transcriptome changes [<a href="#CR75">75</a>] and to improve substrate utilization and tolerance to lignocellulosic inhibitors in industrial yeast strains [<a href="#CR13">13</a>, <a href="#CR73">73</a>, <a href="#CR86">86</a>], including deletion collection screens for synthetic and straw lignocellulosic hydrolysates which linked tolerance mechanisms to ATPase activity and pH, pentose phosphate metabolism, lipid metabolism and the biosynthesis of amino acids [<a href="#CR65">65</a>, <a href="#CR77">77</a>]. The traditional generation of deletion collections for industrial strains is laborious, since it requires the change of genomic sequence in multiple alleles, and cannot assess the effects of transcript down-regulation. The emerging CRISPR-based knock-out, interference and activation systems thus offer genetic screens with broader phenotypic scope.</p> <p id="Par13">Here, we established CRISPR interference (CRISPRi) screens to identify genetic functions that tune yeast growth in spruce hydrolysate. CRISPRi is emerging as a powerful tool to study genotypeâ€"phenotype relations via precise transcriptional repression [<a href="#CR25">25</a>, <a href="#CR26">26</a>, <a href="#CR55">55</a>, <a href="#CR56">56</a>, <a href="#CR78">78</a>]. We employed an in-house developed single plasmid CRISPRi system [<a href="#CR78">78</a>] which is inducible by anhydrotetracycline (ATc) [<a href="#CR18">18</a>] to repress <i>S. cerevisiae</i> transcription factors (TFs, <i>n</i>â€‰=â€‰161) [<a href="#CR14">14</a>] and protein kinases (PKs, <i>n</i>â€‰=â€‰129) [<a href="#CR10">10</a>], key players in the regulation of cellular mechanisms and pathways. We detect genes capable of modulating yeast growth in hydrolysate, as well as in presence of toxic lignocellulose compounds to understand toxicity-dependent effects. We confirm previously known links to growth in hydrolysate and discover novel genetic associations that can be directly applied to advance sustainable bioprocesses (Fig.Â <a href="#Fig1">1</a>a). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Study schematic and yeast tolerance to ethanol, hydrolysate and growth inhibitors. <b>a</b> Aim of the study. Schematic showing the hydrolysis of lignocellulosic material to convert large polymeric carbohydrates to mono-, di- and oligosaccharides, at the same time releasing toxic compounds that repel yeast growth (growth inhibitors). CRISPR interference or activation screens in hydrolysate allow for the identification of gene functions that contribute to stress sensitivity and resistance to enable the generation of robust strains for biotechnology applications. Schematic is inspired by PÃ©rez et al. [<a href="#CR66">66</a>] and Patel et al. [<a href="#CR63">63</a>]. <b>b</b> Yeast growth in synthetic complete media with 2% glucose (SCM), in SCMâ€‰+â€‰10% spruce hydrolysate, and in SCMâ€‰+â€‰45% inhibitor cocktail (1â€‰Ã—â€‰IC stock mixture diluted to 45%). The optical density at 600Â nm (OD<sub>600</sub>) (y-axis) of <i>S. cerevisiae</i> BY4743 strain cultures was measured in 50-mL flask cultures over time (x-axis). Curves denote the average of <i>n</i>â€‰=â€‰3 biological replicates. Error bars denote standard deviations. <b>c</b> Ethanol yield obtained in different fermentation conditions is shown as g [EtOH produced] / g [glucose consumed], as calculated from HPLC measurements in <i>n</i>â€‰=â€‰3 biological replicates</p>   </div> <p /> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="characterizationofyeastfermentationinlignocellulosichydrolysateandinpresenceofgrowthinhibitorcompounds" title="sec"> <div class="title" tagx="title" title="title">
Characterization of yeast fermentation in lignocellulosic hydrolysate and in presence of growth inhibitor compounds</div> <p id="Par14">In this study, we make use of CRISPR interference to profile genetic functions affecting growth of the diploid BY4743 yeast in spruce hydrolysate-containing media and in presence of growth-inhibitory compounds typically present in spruce and other hydrolysates (Fig.Â <a href="#Fig1">1</a>a). The laboratory BY4743 strain is well characterized, genetically amenable and comes with lots of available high-quality datasets on genome sequencing, annotation and function, in contrast to most process-specialized and frequently polyploid industrial yeast strains [<a href="#CR22">22</a>]. We characterized growth of BY4743 across different media in order to identify conditions suitable for genetic screens, and further quantified glucose consumption and metabolite secretion profiles under the selected screening conditions.</p> <p id="Par15">To assess the general feasibility of employing BY4743 for fermentation, we were interested in the strainsâ€™ ability to grow the presence of high amounts of ethanol (EtOH). As expected, high EtOH concentrations of 5% or 10% decreased maximum growth rate by approximately 30% or 50%, respectively (Additional file <a href="#MOESM1">1</a>: Fig. S1a). We then went on to characterize tolerance to lignocellulosic hydrolysate and to a cocktail of eight selected growth inhibitors (inhibitor cocktail, referred to as IC) which are commonly found in lignocellulosic hydrolysates, based on previous studies (Table <a href="#Tab1">1</a> and Methods). Comparing growth in different dilutions of an industrially derived and widely used spruce hydrolysate [<a href="#CR15">15</a>, <a href="#CR36">36</a>, <a href="#CR46">46</a>, <a href="#CR47">47</a>, <a href="#CR81">81</a>, <a href="#CR86">86</a>], we found that the BY4743 strain grew well in media supplemented with up to 14% hydrolysate without considerable changes in growth rate, while no growth was observed in hydrolysate concentrations of 20% and more (Additional file <a href="#MOESM1">1</a>: Fig. S1b). Culturing yeast in 50-ml flasks, we measured decreased growth in the 10% hydrolysate dilution (Fig.Â <a href="#Fig1">1</a>b). We further measured growth in SCM supplemented with different concentrations of inhibitor cocktail. Growth profiles in 15â€"35% of inhibitor cocktail were similar to those in SCM in well-plate format (Additional file <a href="#MOESM1">1</a>: Fig. S1c). For 50-ml flask cultures, we observed decreased growth in 45% IC with comparable impact to 10% hydrolysate (Fig.Â <a href="#Fig1">1</a>b) and therefore selected these conditions for further experiments, including the CRISPRi-based competition assays (Fig.Â <a href="#Fig1">1</a>a). <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Composition of toxic compounds of the inhibitor cocktail</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2">Inhibitor</th> <th align="left" colspan="2">1â€‰Ã—â€‰IC mixture (stock)</th> <th align="left" colspan="2">SCM 45% IC (used for screens)</th> </tr> <tr> <th align="left">Concentration (mM)</th> <th align="left">Concentration (g/L)</th> <th align="left">Concentration (mM)</th> <th align="left">Concentration (g/L)</th> </tr> </thead> <tbody> <tr> <td align="left">Furfural</td> <td char="." align="char">11.4</td> <td char="." align="char">1.1</td> <td char="." align="char">5.13</td> <td char="." align="char">0.495</td> </tr> <tr> <td align="left">5-HMF</td> <td char="." align="char">5.6</td> <td char="." align="char">0.7</td> <td char="." align="char">2.52</td> <td char="." align="char">0.315</td> </tr> <tr> <td align="left">Formaldehyde</td> <td char="." align="char">5.8</td> <td char="." align="char">0.2</td> <td char="." align="char">2.61</td> <td char="." align="char">0.09</td> </tr> <tr> <td align="left">Acetic acid</td> <td char="." align="char">45.6</td> <td char="." align="char">2.7</td> <td char="." align="char">20.52</td> <td char="." align="char">1.215</td> </tr> <tr> <td align="left">Formic acid</td> <td char="." align="char">15.5</td> <td char="." align="char">0.7</td> <td char="." align="char">6.975</td> <td char="." align="char">0.315</td> </tr> <tr> <td align="left">Vanillin</td> <td char="." align="char">1.6</td> <td char="." align="char">0.2</td> <td char="." align="char">0.72</td> <td char="." align="char">0.09</td> </tr> <tr> <td align="left">Coniferyl aldehyde</td> <td char="." align="char">1.6</td> <td char="." align="char">0.03</td> <td char="." align="char">0.72</td> <td char="." align="char">0.0135</td> </tr> <tr> <td align="left">p-Hydroxybenzaldehyde</td> <td char="." align="char">1.6</td> <td char="." align="char">0.2</td> <td char="." align="char">0.72</td> <td char="." align="char">0.09</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Compound concentrations are listed in mM and g/L for IC mixtures of the 1â€‰Ã—â€‰stock and the mixture used for genetic screens</p> </div>  <p /> <p id="Par16">We next quantified the concentration of extracellular metabolites during yeast cultivation in synthetic complete media (SCM), 10% hydrolysate and 45% inhibitor compound cocktail (IC). Remarkably, ethanol yields were increased byâ€‰~â€‰65% in hydrolysate-supplemented medium compared to SCM and IC conditions (Fig.Â <a href="#Fig1">1</a>c). The increased ethanol production can partially be explained by theâ€‰~â€‰25% higher initial glucose levels (Additional file <a href="#MOESM2">2</a>: Table S1) and the breakdown of large sugar polymers during hydrolysis pretreatment that presumably increased the concentration of fermentable carbohydrates [<a href="#CR75">75</a>], such as galactose and mannose [<a href="#CR20">20</a>], see Methods). We further quantified acetic acid, a common growth inhibitor, and found its concentration to decrease by approximately 50% during fermentation, while glucose was fully consumed (Additional file <a href="#MOESM3">3</a>: Fig. S2). Interestingly, this suggests that acetic acid, and potentially also other growth-inhibitory compounds, can be metabolized under the given conditions. Taken together, the quantified dynamics in growth and metabolite abundances during fermentations in hydrolysate and in the presence of an lignocellulosic inhibitor mixture which showed that growth-inhibitory substances can be metabolized and enabled us to derive optimal conditions for functional genomics assays.</p> </div> <div class="crisprieffectsarereproducibleandcapturepositivecontrols" title="sec"> <div class="title" tagx="title" title="title">
CRISPRi effects are reproducible and capture positive controls</div> <p id="Par17">To identify genes that modulate yeast growth in the presence of lignocellulose hydrolysate and inhibitor cocktail, we employed a plasmid-based CRISPRi system that allows for continuous expression of a nuclease-deactivated (d)Cas9 protein fused to the potent Mxi1 repressor domain [<a href="#CR78">78</a>], as well as the inducible expression of a gRNA from an anhydrotetracycline (ATc)-controlled promoter. For gene dosage screens, we used a library of 1573 gRNAs to repress 161 transcription factors and 129 protein kinases which are highly involved in the regulation of growth adaptation and cellular signalling. While the centromeric plasmids enable repression of only a single gene in each cell, we designed libraries with up to six gRNA locations per gene target [<a href="#CR34">34</a>].</p> <p id="Par18">Repression effects on cellular fitness were assayed in different media (SCM, SCM 10% hydrolysate and SCM 45% IC), performing selections ofâ€‰~â€‰25 generations for all screens to enable direct comparison of effects on doubling time (Fig.Â <a href="#Fig2">2</a>a). Each condition was assayed in triplicate in CRISPRi-inducing (+â€‰ATc) as well as in reference (-ATc) conditions. After selection in different media, plasmids were extracted and gRNA barcodes quantified by sequencing to compare strain abundances betweenâ€‰+â€‰ATc and -ATc populations. Multidimensional scaling (MDS) of read count samples indicate high similarity between replicates and allow to estimate sources of variability of CRISPRi effect size (Fig.Â <a href="#Fig2">2</a>b). Samples of the inhibitor cocktail (SCMâ€‰+â€‰45% IC) were positioned closely to hydrolysate samples (SCMâ€‰+â€‰10% Hydrolysate), suggesting higher similarity to this condition than compared with SCM (Fig.Â <a href="#Fig2">2</a>b). For a detailed comparison of fitness effects, we provide correlations of read counts (Additional file <a href="#MOESM4">4</a>: Fig. S3), guide RNA (Additional file <a href="#MOESM5">5</a>: Fig. S4) and gene log2-fold changes (log2FCs) across all screen conditions (Additional file <a href="#MOESM6">6</a>: Fig. S5). Read counts of the three biological replicates profiled for each condition are highly correlated, giving confidence in the derived effects (Additional file <a href="#MOESM4">4</a>: Fig. S3). Notably, we further found gRNA fold changes highly correlated with those of a previous study where the TF and PK libraries have been profiled individually in SCM with different transformation batches and without oxygen limitation [<a href="#CR34">34</a>], indicating high reproducibility of CRISPRi effects (Pearson <i>R</i><sup>2</sup>â€‰=â€‰0.67, <i>p</i>-valueâ€‰<div class="â€‰2.2eâˆ’16,_UNKNOWN" href="#Fig2">
2c). Accordingly, 67% of variation between CRISPRi effects observed in one of the screens were explained by the other. Interestingly, we measured increased fitness for the repression of three genes (<i>HAP1</i>, <i>RIM11</i> and <i>RME1</i>) that were not significant in the previous study [<a href="#CR34">34</a>], presumably due to the oxygen-limited conditions applied here to focus on fermentation. This is supported by the haeme-activated protein 1 (Hap1) TF which represses Rox1 in non-stress conditions [<a href="#CR88">88</a>]. In response to hypoxia, Hap1 is inhibited to de-repress Rox1 and induce hypoxic stress gene expression [<a href="#CR39">39</a>, <a href="#CR49">49</a>, <a href="#CR88">88</a>] which would be enhanced in CRISPRi strains. The Rme1 TF and the Rim11 PK (one of four glutathione synthetase kinase 3 homologs) regulate meiosis, and their knock-down effects may hint to growth-antagonizing roles in oxygen-limited environments. As anticipated from a competitive fitness assay, genes essential for viability (based on the <i>Saccharomyces</i> Genome Database SGD) [<a href="#CR12">12</a>], showed higher depletion compared to non-essential genes and represent positive controls that validate the experimental setup (Fig.Â <a href="#Fig2">2</a>d). </div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>CRISPRi screens are reproducible and capture positive controls. <b>a</b> Schematic of screen procedure and selection conditions. All selections are performed with populations grown with ATc to induce CRISPRi and without ATc as reference, and in biological triplicate cultures. <b>b</b> MDS-plot of read count samples, depicting Euclidian distance variation in two dimensions (x- and y-axis) to estimate (dis)similarity of replicate samples and effect size between CRISPRi-induced (+â€‰ATc) and reference samples (-ATc) across all screen conditions. The 2-dimensional MDS-plot was generated with the default edgeR function to illustrate similarity between samples. <b>c</b> Correlation of guide log2FCs from this study to a previous screen from Jann et al. [<a href="#CR34">34</a>]. CRISPRi effects of both screens report on fitness in SC medium. Jann et al. [<a href="#CR34">34</a>] phenotyped the TF and PK libraries separately with two replicates while in the presented screens we phenotyped a single consisting of a combined TF and PK libraries and measured triplicates. <b>d</b> Distribution of log2 gene fold changes for non-essential (blue) and essential genes (gold)</p>   </div> <p /> <p id="Par19">Having validated gene dosage effects in SCM and their reliability across studies, we screened CRISPRi populations in 10% spruce hydrolysate (Fig.Â <a href="#Fig2">2</a>a). This revealed fitness effects (gene fold change with false discovery rate (FDR)â€‰<div class="â€‰0.05_UNKNOWN" href="#Fig3">
3a and Additional file <a href="#MOESM7">7</a>: Table S2). Ten of these also showed growth effects in SCM (brown dots in Fig.Â <a href="#Fig3">3</a>a). Four more genes also caused general fitness defects although missing the strict significance requirements in the SCM screen. This left ten genes with hydrolysate-associated roles and minor or no impact on growth (labelled in Fig.Â <a href="#Fig3">3</a>a). Repression of seven genes decreased (<i>YAP1</i>, <i>HAA1</i>, <i>HOG1, PBS2, UGA3, CDC15</i> and <i>UME6)</i> and of three genes increased growth (<i>DOT6</i>, <i>SKO1</i> and <i>BUB1</i>) specifically in hydrolysate. Most of these have well-known roles in adaptation to diverse kinds of stress. </div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Genes with specific functions in hydrolysate fitness. <b>a</b> Scatter of gene log2FCs in SCM versus SCMâ€‰+â€‰10% hydrolysate conditions. Dots denote essential (brown) and non-essential genes (green). Genes with strong hydrolysate-specific effect on fitness are labelled. <b>b</b> Dilution spot plating of CRISPRi strains grown with or without 250Â ng/ÂµL ATc for 24Â h and plated on SCM or SCMâ€‰+â€‰10% hydrolysate agar plates. The no gRNA control strain (noGuideCtr) harbours a non-functional gRNA. The <i>HOG1</i> repression strain (<i>HOG1</i>_rep) expresses a <i>HOG1</i>-targeting gRNA. <b>c</b> Venn diagram of genes modulating fitness in SCM, SCMâ€‰+â€‰10% hydrolysate and SCMâ€‰+â€‰45% inhibitor cocktail conditions. The overlap of significant genes is shown together with volcano plots to illustrate perturbation strength and confidence of individual hits. Volcano plot of gene log2FCs and -log10 Benjaminiâ€"Hochberg FDRs for CRISPRi effects in <b>d</b> SCM, <b>e</b> SCMâ€‰+â€‰10% hydrolysate, and <b>f</b> in SCMâ€‰+â€‰45% inhibitor cocktail. The dashed blue line marks an FDR of 0.05. Some but not all significant modulators are labelled for clarity. <b>g</b> Guide RNA log2FCs for selected genes across conditions. Dots denote gRNA log2FCs and are coloured by FDR for single genes (not for non-essential and essential gene panels). Diamonds denote the means and are coloured in green for SCM, blue for SCMâ€‰+â€‰10% hydrolysate and yellow for SCMâ€‰+â€‰45% inhibitor compound conditions akin to colours used in <b>d</b>â€"<b>f</b>. For genes, the mean gRNA log2FCs (diamonds) correspond to their gene log2FC</p>   </div> <p /> <p id="Par20">Repression of <i>YAP1</i> results in decreased growth in SCMâ€‰+â€‰10% hydrolysate medium (Fig.Â <a href="#Fig3">3</a>a). The Yap1 TF induces gene expression in response to oxidative stress [<a href="#CR48">48</a>, <a href="#CR60">60</a>] and is known to be activated by toxic compounds including furans [<a href="#CR41">41</a>] and phenolic molecules [<a href="#CR58">58</a>]. In line with CRISPRi effects, the <i>yap1Î"</i> mutant has decreased fitness in hydrolysate generated from <i>Miscanthus</i> plants [<a href="#CR77">77</a>], and a recent study also demonstrated that <i>YAP1</i> overexpression increases growth in spruce hydrolysate [<a href="#CR86">86</a>], thus providing a direct validation of screen effects and illustrating how gene functions can be applied for strain design in biotechnology.</p> <p id="Par21">Likewise, we measured decreased fitness in hydrolysate for <i>HAA1</i> in CRISPRi screening (Fig.Â <a href="#Fig3">3</a>a). The Haa1 TF is required for adaptation to mildly acidic environments [<a href="#CR19">19</a>], so that effects are most likely explained by the acidic pH of spruce hydrolysate (pH 4.5 of the 10% hydrolysate medium, compared to pH 5.5 of SCM). Akin to <i>YAP1,</i> the overexpression of <i>HAA1</i> in an industrial yeast strain increased growth rate in hardwood hydrolysate and additionally improved ethanol production [<a href="#CR13">13</a>].</p> <p id="Par22">We further found reduced hydrolysate fitness upon repression of <i>UME6</i> (Fig.Â <a href="#Fig3">3</a>a), which encodes a regulator of meiotic and translation-related genes [<a href="#CR50">50</a>]. Accordingly, the <i>ume6Î"</i> mutant is sensitive to oxidative [<a href="#CR11">11</a>], hyperosmotic [<a href="#CR16">16</a>] high temperature stress [<a href="#CR34">34</a>] and to more than 20 diverse chemicals some of which may have properties similar to lignocellulosic compounds (based on SGD [<a href="#CR12">12</a>]). The <i>ume6Î"</i> strain has also been measured with decreased fitness in <i>Miscanthus</i> plant-derived hydrolysate [<a href="#CR77">77</a>].</p> <p id="Par23">We also measured decreased growth in hydrolysate upon repression of <i>HOG1</i> (Fig.Â <a href="#Fig3">3</a>a). The mitogen-activated protein kinase (MAPKs) is a central component of the high osmolarity glycerol (HOG) pathway which mediates adaptation to hyperosmotic environments. We confirmed effects on <i>HOG1</i> by dilution spot plating of an individual CRISPRi strain expressing a <i>HOG1</i>-targeting gRNA (AGGATCTTCGAAGGGAAGGA) with strong effect size in the screen (log2FCs of -3.35 in SCMâ€‰+â€‰10% Hydrolysate, compared to -0.62 in SCM) (Fig.Â <a href="#Fig3">3</a>b). In line with these results, the deletion of <i>HOG1</i> in different strain backgrounds has been reported with reduced fitness in hydrolysate derived from corn stover [<a href="#CR84">84</a>]. In addition, we identified repression of the <i>HOG1</i>-activating kinase <i>PBS2</i> to result in decreased fitness in spruce hydrolysate, and mild growth defects are also known from the <i>pbs2Î"</i> mutant in <i>Miscanthus</i> plant hydrolysate [<a href="#CR77">77</a>]. HOG pathway effects could be due to the high osmolarity of spruce hydrolysate as a result of dissolved compatible solutes which are released from lignocellulosic material during hydrolysis, including salts and carbohydrates.</p> <p id="Par24">In contrast, repression of the HOG-downstream TF <i>SKO1</i> increased cell growth in the presence of hydrolysate. Sko1 is constitutively nuclear and bound to promoters for repression in non-stress condition [<a href="#CR62">62</a>, <a href="#CR68">68</a>, <a href="#CR72">72</a>]. Upon hyperosmotic stress which likely prevails in hydrolysate, Hog1 is activated and transitions into the nucleus where it phosphorylates Sko1 which is then exported to the cytosol, activating transcription via de-repression [<a href="#CR62">62</a>, <a href="#CR68">68</a>, <a href="#CR72">72</a>]. Interestingly, genes induced in lignocellulose hydrolysate are enriched for Sko1 target genes [<a href="#CR75">75</a>] which supports the relevance of Sko1-controlled transcripts in hydrolysate.</p> <p id="Par25">We notably observed increased growth in hydrolysate for repression of <i>DOT6</i> (Fig.Â <a href="#Fig3">3</a>a) which encodes a transcriptional repressor that responds to osmotic and oxidative stress [<a href="#CR52">52</a>, <a href="#CR62">62</a>]. The stimulated fitness in hydrolysate upon <i>DOT6</i> repression can, akin to <i>SKO1</i>, be explained through de-repression<i>.</i> Proteins encoded by genes that affect growth in hydrolysate form physical interaction networks with bundled interactions at components of the osmotic (Hog1, Pbs2, Sko1) and oxidative stress response (Yap1, Dot6) and connect to growth regulators (Additional file <a href="#MOESM8">8</a>: Fig. S6).</p> <p id="Par26">The mechanisms underlying further hydrolysate-specific effects of <i>BUB1, CDC15</i> and <i>UGA3</i> seem not directly linked to their known functions and offer leads for future investigation (Fig.Â <a href="#Fig3">3</a>a). <i>CDC15</i> and <i>BUB1</i> encode protein kinases involved in cell cycle control, e.g. with Bub1 hindering cell cycle progression if the spindle apparatus is damaged [<a href="#CR29">29</a>, <a href="#CR30">30</a>]. <i>UGA3</i> encodes a transcriptional activator of the gamma-aminobutyrate (GABA) pathway. Notably, the <i>uga3</i> mutant was found with increased cellular fitness in presence of ethanol [<a href="#CR69">69</a>].</p> <p id="Par27">To better understand the cellular processes remodelled in hydrolysate, we determined the target genes of TFs that modulated fitness in hydrolysate using chromatin immunoprecipitation (ChIP) data [<a href="#CR74">74</a>] which were enriched for functions in translation, RNA binding, cyclic compound binding, the regulation of carbon metabolism (glycolysis and gluconeogenesis) and nuclear export of non-coding RNAs (Additional file <a href="#MOESM9">9</a>: Fig. S7). We further searched for the phosphorylation targets of PKs modulating hydrolysate fitness, and found that these were enriched for functions in general kinase activity, Mitogen-activated protein kinase (MAPK) and target of rapamycin (TOR) signalling, mitophagy, stress granule components, chronic cell aging, as well as the cellular responses to osmotic stress, organic substances and acidic chemicals (Additional file <a href="#MOESM10">10</a>: Fig. S8). Toxic chemicals, low pH and hyperosmotic conditions thus represent major stressors that yeast cells genetically brace against during growth in spruce hydrolysate.</p> </div> <div class="contributionsoflignocellulosicgrowthinhibitors" title="sec"> <div class="title" tagx="title" title="title">
Contributions of lignocellulosic growth inhibitors</div> <p id="Par28">We next screened for growth effects in the presence of eight lignocellulosic inhibitor compounds to determine genetic effects linked to their toxicity (IC composition in Table <a href="#Tab1">1</a>). After 25 generations selection in SCMâ€‰+â€‰45% Inhibitor Cocktail, we identified seven genes with inhibitor-specific functions compared to SCM, five of which overlap with hydrolysate-specific effects (Fig.Â <a href="#Fig3">3</a>c, Additional file <a href="#MOESM7">7</a>: Table S2). Fitness effects observed for repression of <i>HAA1</i>, <i>HOG1, PBS2</i>, <i>SKO1</i> and <i>UGA3</i> in hydrolysate were not measured in presence of inhibitor compounds and are therefore likely independent of the toxicity caused by the used substances and concentrations (Fig.Â <a href="#Fig3">3</a>g). In contrast, the strong hydrolysate-specific growth effects of <i>BUB1, DOT6</i>, <i>UME6</i> and <i>YAP1</i> CRISPRi strains were reproduced with the inhibitor cocktail and were thus caused by one or multiple compounds in the cocktail (Fig.Â <a href="#Fig3">3</a>g). In addition, we find that the repression of <i>STB5</i> reduced growth in inhibitor-supplemented and hydrolysate media although the gene barely missed the strict significance requirements in the hydrolysate screen (Fig.Â <a href="#Fig3">3</a>g). <i>STB5</i> encodes a zinc cluster activator of pleiotropic multidrug resistance genes [<a href="#CR4">4</a>]. In accord, <i>stb5Î"</i> strains are reported with decreased fitness in <i>Miscanthus</i> plant hydrolysate [<a href="#CR77">77</a>], while its overexpression was shown to increase growth in spruce hydrolysate [<a href="#CR86">86</a>]. Three genes with inhibitor-specific effects are transcribed from bidirectional promoters which are hard to dissect by CRISPRi (<i>CKA2|SLD7, STE7|DHH1, FKH2|YNL067W-A</i>) since effects can be caused by the perturbation of either of the two genes, or their combined impact. Two of these loci (<i>CKA2|SLD7, STE7|DHH1</i>) were also significant in the hydrolysate screen. Clustering of gene fold changes across screens allows for the identification of specific and shared gene functions between conditions (Additional file <a href="#MOESM11">11</a>: Fig. S9). Finally, we confirmed five strong loss of function fitness effects of our CRISPRi screens by using gene deletion strains of the prototrophic haploid library (BY4741 background) [<a href="#CR57">57</a>]. We found hydrolysate growth decreased in <i>hog1Î"</i>, <i>pbs2Î"</i> and <i>stb5Î"</i> strains, and increased in <i>dot6Î"</i> and <i>sko1Î"</i> strains, recapitulating CRISPRi screen results (Additional file <a href="#MOESM12">12</a>: Fig. S10).</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par29">Here, we employed CRISPR interference screens to identify regulatory genes capable of adjusting the growth of bakerâ€™s yeast in spruce hydrolysate and in the presence of lignocellulosic toxins. This allowed us to explain contributions of toxicity in hydrolysate growth conditions, and how genetic screens can be utilized towards overcoming current challenges in hydrolysate fermentation.</p> <p id="Par30">CRISPRi perturbations are powerful to probe genotypeâ€"phenotype relations in high throughput, with low cost and labour, and with high reproducibility. Our single plasmid CRISPRi system has been deeply characterized [<a href="#CR33">33</a>, <a href="#CR34">34</a>, <a href="#CR78">78</a>, <a href="#CR79">79</a>], is freely available on addgene (#73796) and supported by an in-house developed gRNA design platform (<a href="http://lp2.github.io/yeast-crispri/">http://lp2.github.io/yeast-crispri/)</a> [<a href="#CR78">78</a>] and a customizable computational analysis pipeline (Methods)Â [<a href="#CR34">34</a>]. The plasmid system can be transformed into any strain background, including polyploid and industrial strains, to devise strategies for improving process performance.</p> <p id="Par31">The presented CRISPRi screens on yeast growth in hydrolysate complement genetic screens with deletion mutants [<a href="#CR65">65</a>, <a href="#CR77">77</a>]. We found that hydrolysate-specific functions are frequently connected to stress adaptation in the responses to oxidative (Yap1, Stb5), osmotic (Hog1, Pbs2, Sko1), acidic (Haa1) and general stress (Dot6, Ume6). These mirror the environmental conditions prevailing in the hydrolysate which are presumably perceived as stressful by yeast cells. Overcoming them can partially be tackled by the pretreatment of hydrolysate by, e.g. de-salting, pH adjustment or by addition of reducing agents to quench reactive oxygen species. Alternatively, or in addition, yeast strains can be genetically engineered to enhance growth, stress tolerance and productivity. Here we demonstrate that CRISPRi screens provide the opportunity to identify gene targets for strain optimization and, notably, multiple of our hits were already known to affect yeast growth in hydrolysate and have been successfully applied, including Yap1, Stb5 and Haa1. Overexpression of these three genes has been shown to increase tolerance to hydrolysate [<a href="#CR13">13</a>, <a href="#CR86">86</a>] which not only validates their genetic functions, but also demonstrates their potential to improve bioprocesses.</p> <p id="Par32">To our knowledge, the presented study presents the first functional genetic screens of yeast in spruce hydrolysate. These screens suggest novel genes, expression of which can be modified to optimize hydrolysate fermentation, such as by overexpression of <i>CDC15</i> and <i>UGA3,</i> or by decreasing expression levels of <i>BUB1</i>, <i>DOT6</i> and <i>SKO1.</i> In addition, our finding that repression of the two key HOG signalling components (<i>PBS2</i> and <i>HOG1</i>) result in growth defects in hydrolysate suggests an important regulatory role of this pathway during hydrolysate fermentation. We employed five haploid BY4741 deletion collection strains [<a href="#CR57">57</a>] to validate a handful of CRISPRi screen effects, demonstrating that hydrolysate fitness is enhanced in <i>dot6Î"</i> and <i>sko1Î"</i> mutants, and reduced in <i>hog1Î"</i>, <i>pbs2Î"</i> and <i>stb5Î"</i> backgrounds (Additional file <a href="#MOESM12">12</a>: Fig. S10). These preliminary results warrant further investigation into the role of these genes in hydrolysate tolerance.</p> <p id="Par33">While our aim was to characterize genetics underlying fermentative growth, employing the screen under other selection conditions, such as low pH [<a href="#CR42">42</a>], high ethanol [<a href="#CR21">21</a>], and high temperatures [<a href="#CR6">6</a>, <a href="#CR17">17</a>, <a href="#CR31">31</a>, <a href="#CR34">34</a>, <a href="#CR83">83</a>], could be used to further improve specific processes.</p> <p id="Par34">Since growth-inhibiting agents are described as major challenge for utilizing plant hydrolysates for fermentation, we screened for fitness effects in presence of eight such compounds covering aliphatic acids, furan aldehydes and phenolic/aromatic molecules [<a href="#CR38">38</a>]. While our cocktail was defined to largely cover these prevalent impacts, there are presumably additional lignocellulosic substances with yet other mechanisms of toxicity. Our finding that at least one-third of hydrolysate-specific gene functions were explained by toxicity effects therefore represents a lower bound and likely an underestimation. Due to the high prevalence of toxicity effects, relieving them through genetic regulation is an attractive avenue to facilitate the fermentation of lignocellulosic material.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par35">Taken together, we show how CRISPRi screens can be used to identify genetic elements underlying complex environmental conditions encountered by cells in industrial bioprocesses. Our study pinpoints genetic functions that can be engineered to facilitate utilization of lignocellulose as feedstock for yeast fermentation, and thereby hopefully motivates and contributes to the establishment of environmentally sustainable procedures in biotechnology.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="yeaststrains,bacterialstrainsandplasmids" title="sec"> <div class="title" tagx="title" title="title">
Yeast strains, bacterial strains and plasmids</div> <p id="Par36">Two gRNA libraries targeting sets of 129 protein kinases [<a href="#CR10">10</a>] with 688 gRNAs and 161 transcription factors [<a href="#CR14">14</a>] with 885 gRNAs were used as described in Jann et al. [<a href="#CR34">34</a>]. These DNA oligonucleotide libraries were integrated in the dCas9-Mxi1 plasmid system at the NotI restriction site by Gibson Assembly [<a href="#CR23">23</a>] as described in Smith et al. [<a href="#CR78">78</a>], transformed into the diploid BY4743 strain background as outlined by Gietz and Schiestl [<a href="#CR24">24</a>] and pooled together, yielding a final library with 1573 gRNAs. Growth phenotypes of five mutants (<i>dot6Î"</i>, <i>hog1Î"</i>, <i>pbs2Î"</i>, <i>sko1Î"</i>, <i>stb5Î"</i>) from the haploid BY4741 deletion collection [<a href="#CR57">57</a>] and the BY4741 wild type were measured<i>.</i> (Additional file <a href="#MOESM13">13</a>: AF1 lists all chemical compounds, oligonucleotides, plasmids, bacterial and yeast strains, as well as all gRNAs used in this study).</p> </div> <div class="growthmedia" title="sec"> <div class="title" tagx="title" title="title">
Growth media</div> <p id="Par37">Filter-sterilized synthetic complete uracil-dropout medium (SCM-Ura,Â for simplicity referred to as â€œSCMâ€� in the manuscript) containing 20Â g/L glucose, 6.7Â g/L yeast nitrogen base without amino acids and with ammonium sulphate and 2Â g/L amino acids uracil-dropout mix, pH 5.5, was used for fermentation. SCM-Ura was supplemented with hydrolysate or inhibitor cocktail mixture as indicated. Spruce hydrolysate was kindly provided by SEKAB (Ã–rnskÃ¶ldsvik, Sweden).</p> <p id="Par38">The sawdust raw material used to generate the hydrolysate contains 13Â g/kg dry solid/dry weight (DS) arabinose, 20Â g/kg DS galactose, 408Â g/kg DS glucose, 50Â g/kg DS xylose, 108Â g/kg DS mannose, 0.26% ash (525Â Â°C), 2.3% cyclohexane/acetone soluble matter and 27% DS lignin (Klason method) as reported by SEKAB. The sawdust with a dry matter content of 50% was pretreated with sulphur dioxide, enzymatically hydrolysed and filtered to remove solids.</p> <p id="Par39">The resulting hydrolysate is of defined composition and has been chemically characterized with 83Â g/L glucose and higher order carbohydrates (26Â g/L mannose, 9Â g/L xylose, and less than 4Â g/L galactose and arabinose), as well as toxic compounds that include, among others, 4.7Â g/L acetic acid, 3.4Â g/L 5-hydroxymethyl-furfural, 1.2Â g/L furfural and less than 1Â g/L of phenolic derivatives, levulinic and formic acid [<a href="#CR3">3</a>]. In CRISPRi screening, SCM media was supplemented with 10% of the described hydrolysate of pH 4.5.</p> <p id="Par40">We defined an inhibitor cocktail (Table <a href="#Tab1">1</a>) based on previous characterizations of spruce and other plant hydrolysates, focussing particularly on medium produced by SEKAB [<a href="#CR15">15</a>, <a href="#CR36">36</a>, <a href="#CR46">46</a>, <a href="#CR47">47</a>, <a href="#CR81">81</a>, <a href="#CR86">86</a>] or generated under similar technical and chemical conditions [<a href="#CR51">51</a>, <a href="#CR54">54</a>, <a href="#CR67">67</a>]. The inhibitor mix was generated to impose growth defects of all three major inhibitor compound types (aliphatic acids, furan aldehydes and phenolic/aromatic molecules) and their concentrations are not proportional to the ones of the used hydrolysate from SEKAB. Inhibitor compounds were chosen based on described effect size and occurrence in hydrolysates based on literature. Inhibitor compounds were purchased from Sigma-Aldrich, and mixed to a 5Ã—â€‰concentrated cocktail before addition to SCM (Table <a href="#Tab1">1</a>). Coniferyl aldehyde, vanillin and p-hydroxybenzaldehyde were diluted in 3Â ml 0.1Â M NaOH prior to addition. Inhibitor-supplemented SCM-Ura was adjusted to pH 5 with 0.1Â M NaOH.</p> </div> <div class="analyticalmethods" title="sec"> <div class="title" tagx="title" title="title">
Analytical methods</div> <p id="Par41">Quantitative determination of acetate, ethanol and glucose was performed by high-performance liquid chromatography (HPLC). The HPLC system was equipped with a refractive index detector (Alliance HPLC with 2414 RID, Waters, Eschborn, Germany) and a Rezex ROA-Organic Acid Hâ€‰+â€‰(8%) column (Phenomenex, Aschaffenburg, Germany) which was held at 65Â Â°C during all measurements. 0.5Â mM sulfuric acid was used as mobile phase carrying the samples at a flow rate of 0.5Â ml/min. Samples were stored at 6Â Â°C prior to each run which took 15Â min. Hydrolysate-containing samples were diluted four times. Compounds were quantified by comparing the metabolite peak in the sample with a mixture of standards with known concentrations of each metabolite.</p> </div> <div class="dilutionspotplating" title="sec"> <div class="title" tagx="title" title="title">
Dilution spot plating</div> <p id="Par42">CRISPRi cultures were grown in SCM and SCMâ€‰+â€‰10% hydrolysate, either with or without addition of 250Â ng/mL ATc for 24Â h at 30Â Â°C. A dilution series was prepared in medium without ATc, and 10Â ÂµL of the dilutions were plated on SCM or SCMâ€‰+â€‰10% hydrolysate agar plates. Photographs were taken after 2Â days incubation at 30Â Â°C to report on colony size.</p> </div> <div class="cultureconditions" title="sec"> <div class="title" tagx="title" title="title">
Culture conditions</div> <p id="Par43">For HPLC experiments, yeast cultures were grown in 50Â mL in shake flasks at 30Â Â°C with 175Â rpm. Flasks were sealed with rubber plugs to mimic oxygen-limiting conditions, stimulating fermentation rather than respiration. Plugs were pierced with sterile needles (27, BD Microlance, Becton Dickinson) and cotton plugs to enable CO<sub>2</sub> exchange. Plate reader assays were performed under oxygen-limited conditions in 96-well plates (Nunclon Delta Surface, Thermo Scientific) with Synergy HTX Multi-Detection Microplate Readers (BioTek) at 30Â Â°C. Plates were inoculated with an optical density at wavelength 600Â nm (OD<sub>600</sub>) of 0.005. OD<sub>600</sub> was measured in 15Â min intervals and shaken at 800Â rpm for 10Â s before measurements. For screens, yeast was cultured in sealed falcon tubes as described below.</p> </div> <div class="competitivegrowthassays" title="sec"> <div class="title" tagx="title" title="title">
Competitive growth assays</div> <p id="Par44">For screens, defrosted hydrolysate and inhibitor cocktail were diluted in SCM-Ura to 10% and 45%, respectively. Screens were performed in triplicate samples, each with addition of 250Â ng/mL ATc to induce gRNA expression and without ATc as reference. Yeast cultures were profiled in 15Â mL falcon tubes containing 11Â mL medium that was inoculated with overnight cultures at OD 0.005. Falcon tubes were sealed to mimic oxygen-limited conditions, and a needle (20 G, BD Microlance) stuffed with cotton was pierced through the lid. Cells were then grown in a shaking incubator at 30Â Â°C and 180Â rpm. The tubes were opened only to assess growth stage. Cells were transferred to fresh medium in late mid-exponential phase (before reaching OD<sub>600</sub>â€‰=â€‰1). During the screen, two such transfers were performed to maintain exponential growth. Cell samples were centrifuged, and pellets were washed and used to extract plasmid DNA.</p> </div> <div class="sequencing" title="sec"> <div class="title" tagx="title" title="title">
Sequencing</div> <p id="Par45">Plasmid DNA was purified using the Miniprep kit (QIAprep Spin, Qiagen, Hilden, Germany) with modified protocol. Cell pellets were resuspended in P1 solution (accordingly to kit manufacturer) and incubated with 9U lyticase (Sigma-Aldrich) at 37Â Â°C for 30Â min, followed by harsh vortexing for 2Â min. The remaining steps were performed following the provided kit protocol. Illumina sequencing adapters and inline barcodes were introduced to DNA barcodes by PCR, creating a specific double index of samples. All PCR products were analysed by gel electrophoresis and then purified (QIAquick, Quiagen). Samples were pooled to similar amounts. The resulting sequencing library was concentrated by performing another PCR purification, and a 1% agarose gel electrophoresis was performed for size-selection and purification. DNA quality was checked with a DNA-Bioanalyzer, and a diluted sample was sequenced on an Illumina Next-Seq500 machine in paired-end mode with 75 bases read length.</p> </div> <div class="dataprocessingandanalysis" title="sec"> <div class="title" tagx="title" title="title">
Data processing and analysis</div> <p id="Par46">Raw Illumina sequencing reads were demultiplexed using Jemultiplexer. Trimmed reads were aligned to a gRNA sequence reference genome with Burrows-Wheeler aligner to compute read counts. Read counts were processed in R code with the edgeR package, using a generalized linear model to compute the log2 guide and gene fold change (log2FC) betweenâ€‰+â€‰ATc and -ATc populations that both went through selection. Significant genes were required to have a gene log2FC with FDRâ€‰<div class="â€‰0.05_UNKNOWN"> </div></p></div> <div class="enrichmentanalyses" title="sec"> <div class="title" tagx="title" title="title">
Enrichment analyses</div> <p id="Par47">Gene ontology-enrichment was performed using the gProfiler2 R packageÂ [<a href="#CR43">43</a>]. TF target genes were determined using Chromatin Immuno-Precipitation on chip (ChIP-chip) data from GonÃ§alves et al. [<a href="#CR28">28</a>]. Genes bound by two or more TFs from the set of significant modulators identified in CRISPRi screens were used for enrichment analysis with the <i>S. cerevisiae</i> genome as statistic background. Phosphorylation targets of protein kinases were determined with data from phosphogrid 2.0 [<a href="#CR74">74</a>]. The PK targets of significant modulators of CRISPRi screens were used for enrichment analysis, using all protein kinase targets measured in the phosphoproteomics data set as statistic background.</p> </div> </div> <div class="datavisualization" title="sec"> <div class="title" tagx="title" title="title">
Data visualization</div> <p id="Par48">Plots were generated in R (V. 3.4.1) [<a href="#CR70">70</a>] with the ggplot2 (V. 3.1.0) [<a href="#CR27">27</a>], ggally (V. 1.3) [<a href="#CR76">76</a>] and pheatmap (V. 1.0.10) packages [<a href="#CR45">45</a>]. In boxplots, the middle line denotes the median, and lower and upper hinges denote the first and third quartiles, respectively. Figures were designed in Adobe Illustrator 2019.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM1_ESM.png">LINK</a> <p><b>Additional file 1: Figure S1.</b> Yeast growth across media in 96-well plate format. The optical density at 600 nm (OD<sub>600</sub>, on y-axis) was quantified over time (x-axis) during growth of BY4743 in synthetic complete media with 2% glucose (SCM), as well as in SCM that was supplemented with different concentrations of (a) ethanol, (b) spruce hydrolysate or (c) inhibitor compound cocktail. The respective supplement concentrations are indicated individually. For the IC mixture, the 1x IC stock was diluted to the indicated percentages. The curves denote the average of <i>n</i> = 4 wells measured in 96-well format, normalized by subtraction of media background.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM2_ESM.png">LINK</a> <p><b>Additional file 2: Table S1.</b> HPLC measurements. Metabolites concentrations (g L<sup>âˆ’1</sup>), measured by HPLC, of yeast cultures grown in SCM, SCM+10% hydrolysate and SCM+45% inhibitor cocktail. Glucose, ethanol and acetic acid concentrations were measured in the growth medium (Initial) and at the end of fermentations (Final). Three biological triplicates were performed for the three tested conditions, error represents the standard deviation between replicates.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM3_ESM.png">LINK</a> <p><b>Additional file 3: Figure S2. </b>Acetic acid metabolization. Changes of acetic acid (in g L<sup>âˆ’1</sup>) concentration during fermentation in different growth conditions (indicated in figure legend) at cultivation start and end points, measured by HPLC. Initial data corresponds to media used to inoculate, while each point in â€œFinalâ€� correspond to acetic acid concentration of three biological replicates.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM4_ESM.png">LINK</a> <p><b>Additional file 4: Figure S3.</b> Read count correlation. Spearman correlations of read count samples across screens.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM5_ESM.png">LINK</a> <p><b>Additional file 5: Figure S4.</b> Guide RNA fold changes across conditions. Scatter plots with dots denoting gRNAs, density distributions and Pearson correlations of gRNA log2 fold changes across screen conditions.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM6_ESM.png">LINK</a> <p><b>Additional file 6: Figure S5.</b> Gene fold changes across conditions. Scatter plots with dots denoting genes, density distributions and Pearson correlations of gene log2 fold changes across screen conditions. Line denotes smoothed linear fits.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM7_ESM.png">LINK</a> <p><b>Additional file 7: Table S2.</b> CRISPRi effects across media. Significant genes (gene level fold change with FDR &amp;lt; 0.05 and at least two gRNAs with absolute log2FC â‰¥ 1 and FDR &amp;lt; 0.05) are shown across screens with their mean log2FC, their maximum gRNA log2FC and ID to specify if a TF or PK is targeted. For target genes transcribed from bidirectional promoters, both genes are reported (separated with a vertical dash). Tables are ordered by gene log2FC. Rows of essential genes as defined by in-viable knock-out mutants [<a href="#CR12">12</a>] are in green color. One and two asterisks (*, **) behind a gene name indicate that repression caused hydrolysate-specific or inhibitor-specific effects, respectively (not measured in SCM).</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM8_ESM.png">LINK</a> <p><b>Additional file 8: Figure S6.</b> Proteinâ€"protein interaction network between modulators of hydrolysate growth. Experimental proteinâ€"protein interactions of genes modulating cellular fitness in hydrolysate, obtained from STRING [<a href="#CR76">76</a>]. Dots denote genes, coloured by gradients from light to dark by increased strength in either positively (green) or negatively (red) modulating hydrolysate fitness, obtained from screen log2-fold changes. Dot and gene label size denote the multiple-testing adjusted FDRs obtained in the screen. Line thickness indicates confidence of the physical interaction obtained from the STRING database. Network visualization was performed with Gephi [<a href="#CR8">8</a>], using the Force Atlas 2 algorithm for clustering with standard parameters [<a href="#CR32">32</a>].</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM9_ESM.png">LINK</a> <p><b>Additional file 9: Figure S7.</b> Hydrolysate-specific TF target gene functions. GO enrichment of TF target genes determined from ChIP-chip (GonÃ§alves et al., 2017) of TFs which modulate hydrolysate growth, generated using the gProfiler2 R package (Reimand et al. 2019).</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM10_ESM.png">LINK</a> <p><b>Additional file 10: Figure S8.</b> Hydrolysate-specific PK interactor functions. GO enrichment of PK phosphorylation targets determined from Phospho-proteomics data [<a href="#CR71">71</a>] of PKs which modulate hydrolysate growth, generated using the gProfiler2 R package (Reimand et al. 2019).</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM11_ESM.png">LINK</a> <p><b>Additional file 11: Figure S9.</b> CRISPRi effects across screens. Log2 gene fold changes compared between SC medium, SCM + 10% Hydrolysate and SCM + 45% Inhibitor Cocktail. The heatmap was generated with the pheatmap R package (Kolde 2019).</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM12_ESM.png">LINK</a> <p><b>Additional file 12: Figure S10.</b> Growth profile in SCM and in SCM+10% Hydrolysate of prototrophic gene deletion strains. The optical density at 600 nm (OD<sub>600</sub>, on y-axis) was quantified over time (hour, x-axis) during growth of prototrophic BY4741 WT (grey) and the prototrophic BY4741 deletion strains (orange) in SCM and in SCM supplemented with 10% spruce hydrolysate. The curves denote the average of <i>n</i> = 3 wells measured in 96-well format, normalized by subtraction of media background.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM13_ESM.xlsx">LINK</a> <p><b>Additional file 13: AF1.Â List of all chemical compounds, oligonucleotides, plasmids, bacterial and yeast strains, as well as all gRNAs sequences used in this study.</b></p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM14_ESM.xlsx">LINK</a> <p><b>Additional file 14.</b></p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="13068_2021_1880_MOESM15_ESM.xlsx">LINK</a> <p><b>Additional file 15.</b></p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
ATc</div> <div class="def" title="def"> <p id="Par4">Anhydrotetracycline</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CRISPRi</div> <div class="def" title="def"> <p id="Par5">Clustered Regularly Interspaced Short Palindromic Repeats interference</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
(d)Cas9</div> <div class="def" title="def"> <p id="Par6">(Dead) Clustered Regularly Interspaced Short Palindromic Repeats-associated protein 9</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HPLC</div> <div class="def" title="def"> <p id="Par7">High-performance liquid chromatography</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
IC</div> <div class="def" title="def"> <p id="Par8">Inhibitor compound cocktail</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MDS</div> <div class="def" title="def"> <p id="Par9">Multidimensional scaling</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SCM</div> <div class="def" title="def"> <p id="Par10">Synthetic complete medium</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>Friederike Gutmann and Cosimo Jann contributed equally to this work</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>The online version contains supplementary material available at 10.1186/s13068-021-01880-7.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors wish to thank Antonius J.A. van Maris (Royal Institute of Technology, Stockholm, Sweden) for his supervisory support of the Masterâ€™s Thesis of F.G. Adnan Cavka from SEKAB (Ã–rnskÃ¶ldsvik, Sweden) is gratefully acknowledged for the kind provision of pretreated lignocellulose hydrolysate and technical advice. The authors very much appreciated the useful discussions with Maurizio Bettiga (Chalmers University of Technology, Gothenburg, Sweden) regarding the experimental setup and Justin Smith (Stanford University) for CRISPRi construct setup.</p> </div> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>CJ, FG, FP and KRP conceived the study. CJ, FG and FP designed experiments. CJ and FG performed experiments and data analysis. AJ and LMS provided essential insight and advice. CJ, FP and KRP supervised the study. CJ wrote the manuscript with help from FG, FP and KRP. All authors reviewed the manuscript and approved the final version.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>Open Access funding enabled and organized by Projekt DEAL.Â This study was supported by an Advanced Investigator grant from the European Research Council (ERC) under the European Unionâ€™s Horizon 2020 research and innovation programme (AdG-742804 to L.M.S.), by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation; project STE 1422/4-1 to L.M.S.), and a Joachim Herz Add-On Fellowship (to C.J.).</p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>Demultiplexed sequencing data, read counts, gRNA fold changes and gene fold changes were deposited at Gene Expression Omnibus and are accessible under GSE155590 with the token <i>chkxawwunlgrjeb</i>. Computed gRNA barcode and gene level fold changes are provided in the Additional file <a href="#MOESM14">14</a>: AF2 and Additional file <a href="#MOESM15">15</a>: AF3, respectively.</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par49">Not applicable.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p id="Par50">All authors approved the manuscript.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par51">The authors declare that they have no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCaffery</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanctot</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slininger</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis Liu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorsich</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Furfural induces reactive oxygen species accumulation and cellular damage in <i>Saccharomyces cerevisiae</i></span><span class="source" tagx="source" title="source">Biotechnol Biofuels</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">2</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1754-6834-3-2">10.1186/1754-6834-3-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20150993">20150993</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almeida</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Modig</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petersson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HÃ¤hn-HÃ¤gerdal</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LidÃ©n</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorwa-Grauslund</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Increased tolerance and conversion of inhibitors in lignocellulosic hydrolysates by <i>Saccharomyces cerevisiae</i></span><span class="source" tagx="source" title="source">J Chem Technol Biotechnol Int Res Process Environ Clean Technol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">82</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">340</span><span class="lpage" tagx="lpage" title="lpage">349</span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alriksson</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavka</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JÃ¶nsson</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Improving the fermentability of enzymatic hydrolysates of lignocellulose through chemical in-situ detoxification with reducing agents</span><span class="source" tagx="source" title="source">Biores Technol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">102</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">1254</span><span class="lpage" tagx="lpage" title="lpage">1263</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biortech.2010.08.037">10.1016/j.biortech.2010.08.037</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akache</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacPherson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sylvain</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turcotte</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Complex interplay among regulators of drug resistance genes in <i>Saccharomyces cerevisiae</i></span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">279</span><span class="issue" tagx="issue" title="issue">27</span><span class="fpage" tagx="fpage" title="fpage">27855</span><span class="lpage" tagx="lpage" title="lpage">27860</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M403487200">10.1074/jbc.M403487200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15123673">15123673</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azhar</span><span class="given-names" tagx="given-names" title="given-names">SHM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdulla</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jambo</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marbawi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gansau</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faik</span><span class="given-names" tagx="given-names" title="given-names">AAM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigues</span><span class="given-names" tagx="given-names" title="given-names">KF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Yeasts in sustainable bioethanol production: a review</span><span class="source" tagx="source" title="source">Biochem Biophys Rep</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">52</span><span class="lpage" tagx="lpage" title="lpage">61</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29114570">29114570</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banat</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nigam</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marchant</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McHale</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ethanol production at elevated temperatures and alcohol concentrations: Part I-Yeasts in general</span><span class="source" tagx="source" title="source">World J Microbiol Biotechnol</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">809</span><span class="lpage" tagx="lpage" title="lpage">821</span><span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1008802704374">10.1023/A:1008802704374</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banerjee</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatnagar</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viswanathan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of glycolysis by furfural in Saccharomyces cerevisiae</span><span class="source" tagx="source" title="source">Eur J Appl Microbiol Biotechnol</span><span class="year" tagx="year" title="year">1981</span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">226</span><span class="lpage" tagx="lpage" title="lpage">231</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00505872">10.1007/BF00505872</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bastian M, Heymann S, Jacomy M. Gephi: an open source software for exploring and manipulating networks. In: Third international AAAI conference on weblogs and social media; 2009.</span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berry</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stewart</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span></span><span class="source" tagx="source" title="source">Yeast biotechnology</span><span class="year" tagx="year" title="year">2012</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer Science &amp;amp; Business Media</span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breitkreutz</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharom</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boucher</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neduva</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">ZY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breitkreutz</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stark</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahn</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">A global protein kinase and phosphatase interaction network in yeast</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">328</span><span class="issue" tagx="issue" title="issue">5981</span><span class="fpage" tagx="fpage" title="fpage">1043</span><span class="lpage" tagx="lpage" title="lpage">1046</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1176495">10.1126/science.1176495</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20489023">20489023</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sherlock</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burrows</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">HI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCann</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Troyanskaya</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Global analysis of gene function in yeast by quantitative phenotypic profiling</span><span class="source" tagx="source" title="source">Mol Syst Biol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">2006-0001</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/msb4100043">10.1038/msb4100043</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16738548">16738548</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cherry</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adler</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ball</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chervitz</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dwight</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hester</span><span class="given-names" tagx="given-names" title="given-names">ET</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jia</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juvik</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roe</span><span class="given-names" tagx="given-names" title="given-names">TY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schroeder</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weng</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Botstein</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">SGD: Saccharomyces genome database</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">26</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">73</span><span class="lpage" tagx="lpage" title="lpage">79</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/26.1.73">10.1093/nar/26.1.73</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9399804">9399804</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cunha</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costa</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferraz</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RomanÃ­</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johansson</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SÃ¡-Correia</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domingues</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">HAA1 and PRS3 overexpression boosts yeast tolerance towards acetic acid improving xylose or glucose consumption: unravelling the underlying mechanisms</span><span class="source" tagx="source" title="source">Appl Microbiol Biotechnol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">102</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">4589</span><span class="lpage" tagx="lpage" title="lpage">4600</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00253-018-8955-z">10.1007/s00253-018-8955-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29607452">29607452</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Boer</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span></span><span class="mixed-article-title" title="mixed-article-title">YeTFaSCo: a database of evaluated yeast transcription factor sequence specificities</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">40</span><span class="issue" tagx="issue" title="issue">D1</span><span class="fpage" tagx="fpage" title="fpage">D169</span><span class="lpage" tagx="lpage" title="lpage">D179</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkr993">10.1093/nar/gkr993</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22102575">22102575</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demeke</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dietz</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FoulquiÃ©-Moreno</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mutturi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deprez</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abt</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonini</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liden</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dumortier</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verplaetse</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boles</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verplaetse</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Development of a D-xylose fermenting and inhibitor tolerant industrial Saccharomyces cerevisiae strain with high performance in lignocellulose hydrolysates using metabolic and evolutionary engineering</span><span class="source" tagx="source" title="source">Biotechnol Biofuels</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">89</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1754-6834-6-89">10.1186/1754-6834-6-89</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23800147">23800147</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudley</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janse</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanay</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shamir</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Church</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span></span><span class="mixed-article-title" title="mixed-article-title">A global view of pleiotropy and phenotypically derived gene function in yeast</span><span class="source" tagx="source" title="source">Mol Syst Biol</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">1</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">2005-0001</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/msb4100004">10.1038/msb4100004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16729036">16729036</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgardo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carolina</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manuel</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juanita</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baeza</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Selection of thermotolerant yeast strains Saccharomyces cerevisiae for bioethanol production</span><span class="source" tagx="source" title="source">Enzyme Microbial Technol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">43</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">120</span><span class="lpage" tagx="lpage" title="lpage">123</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.enzmictec.2008.02.007">10.1016/j.enzmictec.2008.02.007</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farzadfard</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perli</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tunable and multifunctional eukaryotic transcription factors based on CRISPR/Cas</span><span class="source" tagx="source" title="source">ACS Synth Biol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">604</span><span class="lpage" tagx="lpage" title="lpage">613</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/sb400081r">10.1021/sb400081r</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23977949">23977949</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandes</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mira</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vargas</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Canelhas</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SÃ¡-Correia</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Saccharomyces cerevisiae adaptation to weak acids involves the transcription factor Haa1p and Haa1p-regulated genes</span><span class="source" tagx="source" title="source">Biochem Biophys Res Commun</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">337</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">95</span><span class="lpage" tagx="lpage" title="lpage">103</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbrc.2005.09.010">10.1016/j.bbrc.2005.09.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16176797">16176797</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francois</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alkim</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morin</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">Engineering microbial pathways for production of bio-based chemicals from lignocellulosic sugars: current status and perspectives</span><span class="source" tagx="source" title="source">Biotechnol Biofuels</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">23</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13068-020-01744-6">10.1186/s13068-020-01744-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31911817">31911817</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujita</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuyama</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwahashi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">The genome-wide screening of yeast deletion mutants to identify the genes required for tolerance to ethanol and other alcohols</span><span class="source" tagx="source" title="source">FEMS Yeast Res</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">744</span><span class="lpage" tagx="lpage" title="lpage">750</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1567-1364.2006.00040.x">10.1111/j.1567-1364.2006.00040.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16879425">16879425</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giaever</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ni</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Connelly</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riles</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VÃ©ronneau</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dow</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucau-Danila</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AndrÃ©</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arkin</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Astromoff</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El Bakkoury</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bangham</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benito</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brachat</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campanaro</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Curtiss</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deutschbauer</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Entian</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flaherty</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foury</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garfinkel</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerstein</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gotte</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ¼ldener</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hegemann</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hempel</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herman</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaramillo</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelly</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelly</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ¶tter</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LaBonte</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamb</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lan</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lussier</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menard</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooi</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Revuelta</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roberts</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ross-Macdonald</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scherens</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schimmack</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shafer</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shoemaker</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sookhai-Mahadeo</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Storms</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strathern</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valle</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Voet</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volckaert</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ward</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilhelmy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winzeler</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yen</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Youngman</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bussey</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boeke</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snyder</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Philippsen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnston</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functional profiling of the Saccharomyces cerevisiae genome</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">418</span><span class="issue" tagx="issue" title="issue">6896</span><span class="fpage" tagx="fpage" title="fpage">387</span><span class="lpage" tagx="lpage" title="lpage">391</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature00935">10.1038/nature00935</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12140549">12140549</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chuang</span><span class="given-names" tagx="given-names" title="given-names">RY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venter</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hutchison</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">HO</span></span></span><span class="mixed-article-title" title="mixed-article-title">Enzymatic assembly of DNA molecules up to several hundred kilobases</span><span class="source" tagx="source" title="source">Nat Methods</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">343</span><span class="lpage" tagx="lpage" title="lpage">345</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.1318">10.1038/nmeth.1318</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19363495">19363495</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gietz</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schiestl</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span></span><span class="mixed-article-title" title="mixed-article-title">High-efficiency yeast transformation using the LiAc/SS carrier DNA/PEG method</span><span class="source" tagx="source" title="source">Nat Protoc</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">31</span><span class="lpage" tagx="lpage" title="lpage">34</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2007.13">10.1038/nprot.2007.13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17401334">17401334</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larson</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morsut</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brar</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torres</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stern-Ginossar</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brandmann</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitehead</span><span class="given-names" tagx="given-names" title="given-names">EH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doudna</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">WA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weissman</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span></span><span class="mixed-article-title" title="mixed-article-title">CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">154</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">442</span><span class="lpage" tagx="lpage" title="lpage">451</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2013.06.044">10.1016/j.cell.2013.06.044</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23849981">23849981</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horlbeck</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adamson</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villalta</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitehead</span><span class="given-names" tagx="given-names" title="given-names">EH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guimaraes</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panning</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ploegh</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bassik</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kampmann</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weissman</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genome-scale CRISPR-mediated control of gene repression and activation</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">159</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">647</span><span class="lpage" tagx="lpage" title="lpage">661</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2014.09.029">10.1016/j.cell.2014.09.029</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25307932">25307932</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ginestet</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">ggplot2: elegant graphics for data analysis</span><span class="source" tagx="source" title="source">J R Stat Soc Ser A Stat Soc</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">174</span><span class="fpage" tagx="fpage" title="fpage">245</span><span class="lpage" tagx="lpage" title="lpage">246</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1467-985X.2010.00676_9.x">10.1111/j.1467-985X.2010.00676_9.x</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GonÃ§alves</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakic</span><span class="given-names" tagx="given-names" title="given-names">ZR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zampieri</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagih</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ochoa</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauer</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beltrao</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saez-Rodriguez</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Systematic analysis of transcriptional and post-transcriptional regulation of metabolism in yeast</span><span class="source" tagx="source" title="source">PLoS Comput Biol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">e1005297</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1005297">10.1371/journal.pcbi.1005297</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28072816">28072816</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goto</span><span class="given-names" tagx="given-names" title="given-names">GH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mishra</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdulle</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slaughter</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitagawa</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bub1-mediated adaptation of the spindle checkpoint</span><span class="source" tagx="source" title="source">PLoS Genet</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">e1001282</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1001282">10.1371/journal.pgen.1001282</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21298086">21298086</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartwell</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Culotti</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reid</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic control of the cell-division cycle in yeast, I. Detection of mutants</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci</span><span class="year" tagx="year" title="year">1970</span><span class="volume" tagx="volume" title="volume">66</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">352</span><span class="lpage" tagx="lpage" title="lpage">359</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.66.2.352">10.1073/pnas.66.2.352</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5271168">5271168</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasunuma</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kondo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Consolidated bioprocessing and simultaneous saccharification and fermentation of lignocellulose to ethanol with thermotolerant yeast strains</span><span class="source" tagx="source" title="source">Process Biochem</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">47</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">1287</span><span class="lpage" tagx="lpage" title="lpage">1294</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.procbio.2012.05.004">10.1016/j.procbio.2012.05.004</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacomy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venturini</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heymann</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bastian</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">ForceAtlas2, a continuous graph layout algorithm for handy network visualization designed for the Gephi software</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">e98679</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0098679">10.1371/journal.pone.0098679</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24914678">24914678</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaffe</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dziulko</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Onge</span><span class="given-names" tagx="given-names" title="given-names">RPS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levy</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sherlock</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Improved discovery of genetic interactions using CRISPRiSeq across multiple environments</span><span class="source" tagx="source" title="source">Genome Res</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">668</span><span class="lpage" tagx="lpage" title="lpage">681</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.246603.118">10.1101/gr.246603.118</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30782640">30782640</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jann</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johansson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parts</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinmetz</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gene dosage screens in yeast reveal core signalling pathways controlling heat adaptation</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.08.26.267674">10.1101/2020.08.26.267674</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jansen</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bracher</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papapetridis</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verhoeven</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Bruijn</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Waal</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Maris</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klaassen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pronk</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span></span><span class="mixed-article-title" title="mixed-article-title">Saccharomyces cerevisiae strains for second-generation ethanol production: from academic exploration to industrial implementation</span><span class="source" tagx="source" title="source">FEMS Yeast Res</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">fox044</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/femsyr/fox044">10.1093/femsyr/fox044</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5812533">5812533</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johansson</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiros</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsson</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fermentation performance and physiology of two strains of Saccharomyces cerevisiae during growth in high gravity spruce hydrolysate and spent sulphite liquor</span><span class="source" tagx="source" title="source">BMC Biotechnol</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">47</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1472-6750-14-47">10.1186/1472-6750-14-47</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24885359">24885359</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JÃ¶nsson</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alriksson</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nilvebrant</span><span class="given-names" tagx="given-names" title="given-names">N-O</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bioconversion of lignocellulose: inhibitors and detoxification</span><span class="source" tagx="source" title="source">Biotechnology for Biofuels.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">16</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1754-6834-6-16">10.1186/1754-6834-6-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23356676">23356676</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JÃ¶onsson</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alriksson</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nilvebrant</span><span class="given-names" tagx="given-names" title="given-names">N-O</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bioconversion of lignocellulose: inhibitors and detoxification</span><span class="source" tagx="source" title="source">Biotechnol Biofuels</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">16</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1754-6834-6-16">10.1186/1754-6834-6-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23356676">23356676</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keng</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">HAP1 and ROX1 form a regulatory pathway in the repression of HEM13 transcription in Saccharomyces cerevisiae</span><span class="source" tagx="source" title="source">Mol Cell Biol</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">2616</span><span class="lpage" tagx="lpage" title="lpage">2623</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/MCB.12.6.2616">10.1128/MCB.12.6.2616</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1588959">1588959</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keweloh</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weyrauch</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rehm</span><span class="given-names" tagx="given-names" title="given-names">H-J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phenol-induced membrane changes in free and immobilized <i>Escherichia coli</i></span><span class="source" tagx="source" title="source">Appl Microbiol Biotechnol</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">33</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">66</span><span class="lpage" tagx="lpage" title="lpage">71</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00170572">10.1007/BF00170572</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1366564">1366564</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Roles of the Yap1 transcription factor and antioxidants in Saccharomyces cerevisiae's tolerance to furfural and 5-hydroxymethylfurfural, which function as thiol-reactive electrophiles generating oxidative stress</span><span class="source" tagx="source" title="source">Appl Environ Microbiol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">79</span><span class="issue" tagx="issue" title="issue">16</span><span class="fpage" tagx="fpage" title="fpage">5069</span><span class="lpage" tagx="lpage" title="lpage">5077</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.00643-13">10.1128/AEM.00643-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23793623">23793623</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koivistoinen</span><span class="given-names" tagx="given-names" title="given-names">OM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuivanen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barth</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turkia</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PitkÃ¤nen</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PenttilÃ¤</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richard</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Glycolic acid production in the engineered yeasts Saccharomyces cerevisiae and <i>Kluyveromyces lactis</i></span><span class="source" tagx="source" title="source">Microb Cell Fact</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">82</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1475-2859-12-82">10.1186/1475-2859-12-82</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24053654">24053654</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolberg</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raudvere</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuzmin</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vilo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">gprofiler2 - an R package for gene list functional enrichment analysis and namespace conversion toolset g:Profiler</span><span class="source" tagx="source" title="source">F1000Research.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">709</span><span class="pub-id"><a href="https://dx.doi.org/10.12688/f1000research.24956.2">10.12688/f1000research.24956.2</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kolde, R. Package `pheatmapâ€™. Bioconductor. 2012.</span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kolde R. Pheatmap: Pretty Heatmaps R. Package. Bioconductor; CRAN-project.org/package=pheatmap; 2012.</span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koppram</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albers</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olsson</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary engineering strategies to enhance tolerance of xylose utilizing recombinant yeast to inhibitors derived from spruce biomass</span><span class="source" tagx="source" title="source">Biotechnol Biofuels</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">32</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1754-6834-5-32">10.1186/1754-6834-5-32</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22578262">22578262</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koppram</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olsson</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Combined substrate, enzyme and yeast feed in simultaneous saccharification and fermentation allow bioethanol production from pretreated spruce biomass at high solids loadings</span><span class="source" tagx="source" title="source">Biotechnol Biofuels</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">54</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1754-6834-7-54">10.1186/1754-6834-7-54</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24713027">24713027</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuge</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nomoto</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation of YAP-1 nuclear localization in response to oxidative stress</span><span class="source" tagx="source" title="source">EMBO J</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">1710</span><span class="lpage" tagx="lpage" title="lpage">1720</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/emboj/16.7.1710">10.1093/emboj/16.7.1710</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9130715">9130715</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwast</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">PV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poyton</span><span class="given-names" tagx="given-names" title="given-names">RO</span></span></span><span class="mixed-article-title" title="mixed-article-title">Oxygen sensing and the transcriptional regulation of oxygen-responsive genes in yeast</span><span class="source" tagx="source" title="source">J Exp Biol</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">201</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">1177</span><span class="lpage" tagx="lpage" title="lpage">1195</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9510529">9510529</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lardenois</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Becker</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walther</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Law</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demougin</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strich</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Primig</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Global alterations of the transcriptional landscape during yeast growth and development in the absence of Ume6-dependent chromatin modification</span><span class="source" tagx="source" title="source">Mol Genet Genomics</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">290</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">2031</span><span class="lpage" tagx="lpage" title="lpage">2046</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00438-015-1051-5">10.1007/s00438-015-1051-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25957495">25957495</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reimann</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nilvebrant</span><span class="given-names" tagx="given-names" title="given-names">NO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JÃ¶nsson</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of different methods for the detoxification of lignocellulose hydrolyzates of spruce</span><span class="source" tagx="source" title="source">Appl Biochem Biotechnol</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">77</span><span class="issue" tagx="issue" title="issue">1â€"3</span><span class="fpage" tagx="fpage" title="fpage">91</span><span class="lpage" tagx="lpage" title="lpage">103</span><span class="pub-id"><a href="https://dx.doi.org/10.1385/ABAB:77:1-3:91">10.1385/ABAB:77:1-3:91</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lippman</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broach</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Protein kinase A and TORC1 activate genes for ribosomal biogenesis by inactivating repressors encoded by Dot6 and its homolog Tod6</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">106</span><span class="issue" tagx="issue" title="issue">47</span><span class="fpage" tagx="fpage" title="fpage">19928</span><span class="lpage" tagx="lpage" title="lpage">19933</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0907027106">10.1073/pnas.0907027106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19901341">19901341</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopes</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosa</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hector</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dien</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mertens</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayub</span><span class="given-names" tagx="given-names" title="given-names">MAZ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influence of genetic background of engineered xylose-fermenting industrial Saccharomyces cerevisiae strains for ethanol production from lignocellulosic hydrolysates</span><span class="source" tagx="source" title="source">J Ind Microbiol Biotechnol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">44</span><span class="issue" tagx="issue" title="issue">11</span><span class="fpage" tagx="fpage" title="fpage">1575</span><span class="lpage" tagx="lpage" title="lpage">1588</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10295-017-1979-z">10.1007/s10295-017-1979-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28891041">28891041</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MartÃ­n</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stagge</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JÃ¶nsson</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Formation of microbial inhibitors in steam-explosion pretreatment of softwood impregnated with sulfuric acid and sulfur dioxide</span><span class="source" tagx="source" title="source">Biores Technol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">262</span><span class="fpage" tagx="fpage" title="fpage">242</span><span class="lpage" tagx="lpage" title="lpage">250</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biortech.2018.04.074">10.1016/j.biortech.2018.04.074</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGlincy</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meacham</span><span class="given-names" tagx="given-names" title="given-names">ZA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swain</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baum</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ingolia</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">A genome-scale CRISPR interference guide library enables comprehensive phenotypic profiling in yeast</span><span class="source" tagx="source" title="source">BiorXiv</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.03.11.988105">10.1101/2020.03.11.988105</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Momen-Roknabadi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oikonomou</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tavazoie</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">An inducible CRISPR-interference library for genetic interrogation of Saccharomyces cerevisiae biology</span><span class="source" tagx="source" title="source">bioRxiv.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">2</span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MÃ¼lleder</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Capuano</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pir</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauer</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliver</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ralser</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">A prototrophic deletion mutant collection for yeast metabolomics and systems biology</span><span class="source" tagx="source" title="source">Nat Biotechnol</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">1176</span><span class="lpage" tagx="lpage" title="lpage">1178</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.2442">10.1038/nbt.2442</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23222782">23222782</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">TTM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwaki</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohya</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Izawa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vanillin causes the activation of Yap1 and mitochondrial fragmentation in Saccharomyces cerevisiae</span><span class="source" tagx="source" title="source">J Biosci Bioeng</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">117</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="lpage" tagx="lpage" title="lpage">38</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jbiosc.2013.06.008">10.1016/j.jbiosc.2013.06.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23850265">23850265</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ostergaard</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olsson</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic engineering of Saccharomyces cerevisiae</span><span class="source" tagx="source" title="source">Microbiol Mol Biol Rev</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">64</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">34</span><span class="lpage" tagx="lpage" title="lpage">50</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/MMBR.64.1.34-50.2000">10.1128/MMBR.64.1.34-50.2000</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10704473">10704473</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okazaki</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tachibana</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naganuma</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mano</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuge</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Multistep disulfide bond formation in Yap1 is required for sensing and transduction of H2O2 stress signal</span><span class="source" tagx="source" title="source">Mol Cell</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">27</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">675</span><span class="lpage" tagx="lpage" title="lpage">688</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molcel.2007.06.035">10.1016/j.molcel.2007.06.035</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17707237">17707237</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pampulha</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loureiro-Dias</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Combined effect of acetic acid, pH and ethanol on intracellular pH of fermenting yeast</span><span class="source" tagx="source" title="source">Appl Microbiol Biotechnol</span><span class="year" tagx="year" title="year">1989</span><span class="volume" tagx="volume" title="volume">31</span><span class="issue" tagx="issue" title="issue">5â€"6</span><span class="fpage" tagx="fpage" title="fpage">547</span><span class="lpage" tagx="lpage" title="lpage">550</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00270792">10.1007/BF00270792</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pascual-Ahuir</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posas</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serrano</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Proft</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Multiple levels of control regulate the yeast cAMP-response element-binding protein repressor Sko1p in response to stress</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">276</span><span class="issue" tagx="issue" title="issue">40</span><span class="fpage" tagx="fpage" title="fpage">37373</span><span class="lpage" tagx="lpage" title="lpage">37378</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M105755200">10.1074/jbc.M105755200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11500510">11500510</a></span></span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arora</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sartaj</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pruthi</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pruthi</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sustainable biodiesel production from oleaginous yeasts utilizing hydrolysates of various non-edible lignocellulosic biomasses</span><span class="source" tagx="source" title="source">Renew Sustain Energy Rev</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">62</span><span class="fpage" tagx="fpage" title="fpage">836</span><span class="lpage" tagx="lpage" title="lpage">855</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.rser.2016.05.014">10.1016/j.rser.2016.05.014</a></span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parawira</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tekere</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biotechnological strategies to overcome inhibitors in lignocellulose hydrolysates for ethanol production</span><span class="source" tagx="source" title="source">Crit Rev Biotechnol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">31</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">20</span><span class="lpage" tagx="lpage" title="lpage">31</span><span class="pub-id"><a href="https://dx.doi.org/10.3109/07388551003757816">10.3109/07388551003757816</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20513164">20513164</a></span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pereira</span><span class="given-names" tagx="given-names" title="given-names">FB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RomanÃ­</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teixeira</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domingues</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Industrial robust yeast isolates with great potential for fermentation of lignocellulosic biomass</span><span class="source" tagx="source" title="source">Biores Technol</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">161</span><span class="fpage" tagx="fpage" title="fpage">192</span><span class="lpage" tagx="lpage" title="lpage">199</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biortech.2014.03.043">10.1016/j.biortech.2014.03.043</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PÃ©rez</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munoz-Dorado</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De la Rubia</span><span class="given-names" tagx="given-names" title="given-names">TDLR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinez</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biodegradation and biological treatments of cellulose, hemicellulose and lignin: an overview</span><span class="source" tagx="source" title="source">Int Microbiol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">53</span><span class="lpage" tagx="lpage" title="lpage">63</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10123-002-0062-3">10.1007/s10123-002-0062-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12180781">12180781</a></span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petersson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LidÃ©n</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fed-batch cultivation of Saccharomyces cerevisiae on lignocellulosic hydrolyzate</span><span class="source" tagx="source" title="source">Biotech Lett</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">219</span><span class="lpage" tagx="lpage" title="lpage">225</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10529-006-9227-z">10.1007/s10529-006-9227-z</a></span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Proft</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serrano</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Repressors and upstream repressing sequences of the stress-regulated ENA1 gene in Saccharomyces cerevisiae: bZIP protein Sko1p confers HOG-dependent osmotic regulation</span><span class="source" tagx="source" title="source">Mol Cell Biol</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">19</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">537</span><span class="lpage" tagx="lpage" title="lpage">546</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/MCB.19.1.537">10.1128/MCB.19.1.537</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9858577">9858577</a></span></span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">The genomic landscape and evolutionary resolution of antagonistic pleiotropy in yeast</span><span class="source" tagx="source" title="source">Cell Rep</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">1399</span><span class="lpage" tagx="lpage" title="lpage">1410</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.celrep.2012.09.017">10.1016/j.celrep.2012.09.017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23103169">23103169</a></span></span></li> <li tag="ref"><a name="CR70" /><span class="label" tagx="label" title="label">70.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">R Core Team. R: a language and environment for statistical computing. R Foundation for Statistical Computing; 2018.</span></li> <li tag="ref"><a name="CR71" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ragauskas</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eckert</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hallett</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liotta</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span></span><span class="mixed-article-title" title="mixed-article-title">From wood to fuels: integrating biofuels and pulp production</span><span class="source" tagx="source" title="source">Ind Biotechnol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">55</span><span class="lpage" tagx="lpage" title="lpage">65</span><span class="pub-id"><a href="https://dx.doi.org/10.1089/ind.2006.2.55">10.1089/ind.2006.2.55</a></span></span></li> <li tag="ref"><a name="CR72" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rep</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Proft</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Remize</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TamÃ¡s</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serrano</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thevelein</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hohmann</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Saccharomyces cerevisiae Sko1p transcription factor mediates HOG pathway-dependent osmotic regulation of a set of genes encoding enzymes implicated in protection from oxidative damage</span><span class="source" tagx="source" title="source">Mol Microbiol</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">40</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">1067</span><span class="lpage" tagx="lpage" title="lpage">1083</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-2958.2001.02384.x">10.1046/j.1365-2958.2001.02384.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11401713">11401713</a></span></span></li> <li tag="ref"><a name="CR73" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RomanÃ­</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pereira</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johansson</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domingues</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic engineering of Saccharomyces cerevisiae ethanol strains PE-2 and CAT-1 for efficient lignocellulosic fermentation</span><span class="source" tagx="source" title="source">Biores Technol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">179</span><span class="fpage" tagx="fpage" title="fpage">150</span><span class="lpage" tagx="lpage" title="lpage">158</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biortech.2014.12.020">10.1016/j.biortech.2014.12.020</a></span></span></li> <li tag="ref"><a name="CR74" /><span class="label" tagx="label" title="label">74.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sadowski</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breitkreutz</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stark</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahabieh</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raithatha</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernhard</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oughtred</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dolinski</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barreto</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tyers</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">The PhosphoGRID Saccharomyces cerevisiae protein phosphorylation site database: version 2.0 update</span><span class="source" tagx="source" title="source">Database</span><span class="year" tagx="year" title="year">2013</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s42003-020-01452-9">10.1038/s42003-020-01452-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23674503">23674503</a></span></span></li> <li tag="ref"><a name="CR75" /><span class="label" tagx="label" title="label">75.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sardi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rovinskiy</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasch</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Leveraging genetic-background effects in Saccharomyces cerevisiae to improve lignocellulosic hydrolysate tolerance</span><span class="source" tagx="source" title="source">Appl Environ Microbiol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">82</span><span class="issue" tagx="issue" title="issue">19</span><span class="fpage" tagx="fpage" title="fpage">5838</span><span class="lpage" tagx="lpage" title="lpage">5849</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.01603-16">10.1128/AEM.01603-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27451446">27451446</a></span></span></li> <li tag="ref"><a name="CR76" /><span class="label" tagx="label" title="label">76.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Schloerke B, Crowley J, Cook D, Hofmann H, Hadley W, Briatte F, Marbach M, Thoen E, Elberg A, Larmarange J. GGally: Extension to â€˜ggplot2â€™. R Package version 1.3.2 <a href="https://CRAN.R-project.org/package=GGally">https://CRAN.R-project.org/package=GGally</a>. 2017.</span></li> <li tag="ref"><a name="CR77" /><span class="label" tagx="label" title="label">77.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skerker</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leon</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mar</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tarjan</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wetmore</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deutschbauer</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baumohl</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bauer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">IbÃ¡nez</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">VD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dissecting a complex chemical stress: chemogenomic profiling of plant hydrolysates</span><span class="source" tagx="source" title="source">Mol Syst Biol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">674</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/msb.2013.30">10.1038/msb.2013.30</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23774757">23774757</a></span></span></li> <li tag="ref"><a name="CR78" /><span class="label" tagx="label" title="label">78.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suresh</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlecht</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagih</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peltz</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinmetz</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parts</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Onge</span><span class="given-names" tagx="given-names" title="given-names">RPS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Quantitative CRISPR interference screens in yeast identify chemical-genetic interactions and new rules for guide RNA design</span><span class="source" tagx="source" title="source">Genome Biol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">45</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-016-0900-9">10.1186/s13059-016-0900-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26956608">26956608</a></span></span></li> <li tag="ref"><a name="CR79" /><span class="label" tagx="label" title="label">79.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlecht</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suresh</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horecka</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Proctor</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aijar</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennet</span><span class="given-names" tagx="given-names" title="given-names">RAO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">YF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinmetz</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyman</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levy</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Onge</span><span class="given-names" tagx="given-names" title="given-names">RPS</span></span></span><span class="mixed-article-title" title="mixed-article-title">A method for high-throughput production of sequence-verified DNA libraries and strain collections</span><span class="source" tagx="source" title="source">Mol Syst Biol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">913</span><span class="pub-id"><a href="https://dx.doi.org/10.15252/msb.20167233">10.15252/msb.20167233</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28193641">28193641</a></span></span></li> <li tag="ref"><a name="CR80" /><span class="label" tagx="label" title="label">80.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lehmann</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bork</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huynen</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">STRING: a web-server to retrieve and display the repeatedly occurring neighbourhood of a gene</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">28</span><span class="issue" tagx="issue" title="issue">18</span><span class="fpage" tagx="fpage" title="fpage">3442</span><span class="lpage" tagx="lpage" title="lpage">3444</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/28.18.3442">10.1093/nar/28.18.3442</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10982861">10982861</a></span></span></li> <li tag="ref"><a name="CR81" /><span class="label" tagx="label" title="label">81.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soudham</span><span class="given-names" tagx="given-names" title="given-names">VP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brandberg</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mikkola</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsson</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detoxification of acid pretreated spruce hydrolysates with ferrous sulfate and hydrogen peroxide improves enzymatic hydrolysis and fermentation</span><span class="source" tagx="source" title="source">Biores Technol</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">166</span><span class="fpage" tagx="fpage" title="fpage">559</span><span class="lpage" tagx="lpage" title="lpage">565</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biortech.2014.05.096">10.1016/j.biortech.2014.05.096</a></span></span></li> <li tag="ref"><a name="CR82" /><span class="label" tagx="label" title="label">82.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hydrolysis of lignocellulosic materials for ethanol production: a review</span><span class="source" tagx="source" title="source">Biores Technol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">83</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0960-8524(01)00212-7">10.1016/S0960-8524(01)00212-7</a></span></span></li> <li tag="ref"><a name="CR83" /><span class="label" tagx="label" title="label">83.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Vleet</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeffries</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Yeast metabolic engineering for hemicellulosic ethanol production</span><span class="source" tagx="source" title="source">Curr Opin Biotechnol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">20</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">300</span><span class="lpage" tagx="lpage" title="lpage">306</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.copbio.2009.06.001">10.1016/j.copbio.2009.06.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19545992">19545992</a></span></span></li> <li tag="ref"><a name="CR84" /><span class="label" tagx="label" title="label">84.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagner</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coon</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasch</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="mixed-article-title" title="mixed-article-title">PKA and HOG signaling contribute separable roles to anaerobic xylose fermentation in yeast engineered for biofuel production</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">e0212389</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0212389">10.1371/journal.pone.0212389</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31112537">31112537</a></span></span></li> <li tag="ref"><a name="CR85" /><span class="label" tagx="label" title="label">85.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">WillfÃ¶r</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmbom</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterisation of water soluble polysaccharides from Norway spruce and Scots pine</span><span class="source" tagx="source" title="source">Wood Sci Technol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">38</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">173</span><span class="lpage" tagx="lpage" title="lpage">179</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00226-003-0200-x">10.1007/s00226-003-0200-x</a></span></span></li> <li tag="ref"><a name="CR86" /><span class="label" tagx="label" title="label">86.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JÃ¶nsson</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Profiling of Saccharomyces cerevisiae transcription factors for engineering the resistance of yeast to lignocellulose-derived inhibitors in biomass conversion</span><span class="source" tagx="source" title="source">Microb Cell Fact</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">199</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12934-017-0811-9">10.1186/s12934-017-0811-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29137634">29137634</a></span></span></li> <li tag="ref"><a name="CR87" /><span class="label" tagx="label" title="label">87.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">XJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gleisner</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sulfite pretreatment (SPORL) for robust enzymatic saccharification of spruce and red pine</span><span class="source" tagx="source" title="source">Biores Technol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">100</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">2411</span><span class="lpage" tagx="lpage" title="lpage">2418</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biortech.2008.10.057">10.1016/j.biortech.2008.10.057</a></span></span></li> <li tag="ref"><a name="CR88" /><span class="label" tagx="label" title="label">88.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zitomer</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowry</span><span class="given-names" tagx="given-names" title="given-names">CV</span></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation of gene expression by oxygen in Saccharomyces cerevisiae</span><span class="source" tagx="source" title="source">Microbiol Mol Biol Rev</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">56</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span></span></li> </ul> </div> </div>  </body></html>