<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6825121">6825121</a></span><span class="publisher-id" title="publisher-id">52416</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-019-52416-x">10.1038/s41598-019-52416-x</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Reference gene analysis and its use for kinase expression profiling in <i>Fasciola hepatica</i></div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Houhou</span><span class="given-names" tagx="given-names" title="given-names">Hicham</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puckelwaldt</span><span class="given-names" tagx="given-names" title="given-names">Oliver</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strube</span><span class="given-names" tagx="given-names" title="given-names">Christina</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haeberlein</span><span class="given-names" tagx="given-names" title="given-names">Simone</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">simone.haeberlein@vetmed.uni-giessen.de</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2165 8627</span><span class="institution-id" title="institution-id">grid.8664.c</span><span class="institution" title="institution">Institute of Parasitology, </span><span class="institution" title="institution">BFS, Justus Liebig University, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0126 6191</span><span class="institution-id" title="institution-id">grid.412970.9</span><span class="institution" title="institution">Institute for Parasitology, Centre for Infection Medicine, </span><span class="institution" title="institution">University of Veterinary Medicine Hanover, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-11-11</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">15867</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-20</span><span class="accepted" title="accepted">accepted: 2019-10-17</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">The liver fluke <i>Fasciola hepatica</i> causes fasciolosis, a foodborne zoonosis affecting humans and livestock worldwide. A reliable quantification of gene expression in all parasite life stages relevant for targeting by anthelmintics in the mammalian host is fundamental. The aim of this study was to define a set of stably expressed reference genes for qRT-PCR in <i>Fasciola</i> studies. We determined the expression stabilities of eight candidate reference genes by the algorithms NormFinder, geNorm, BestKeeper, and comparative ΔCT method. The most stably expressed reference genes for the comparison of intra-mammalian life stages were glutamyl-prolyl-tRNA synthetase (Fh<i>eprs</i>) and tubulin-specific chaperone D (Fh<i>tbcd</i>). The two best reference genes for analysis of <i>in vitro</i>-cultured juveniles were Fh<i>tbcd</i> and proteasome subunit beta type-7 (Fh<i>psmb7</i>). These genes should replace the housekeeping gene <i>gapdh</i> which is used in most <i>Fasciola</i> studies to date, but in fact was differentially expressed in our analysis. Based on the new reference genes, we quantified expression of five kinases (Abl1, Abl2, PKC, Akt1, Plk1) discussed as targets in other parasitic flatworms. Distinct expression patterns throughout development were revealed and point to interesting biological functions. We like to motivate using this set of validated reference genes for future <i>F</i>. <i>hepatica</i> research, such as studies on drug targets or parasite development.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Drug discovery</div> <div class="kwd" title="kwd">
Molecular biology</div> <div class="kwd" title="kwd">
Zoology</div> <div class="kwd" title="kwd">
Diseases</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">The liver fluke <i>Fasciola hepatica</i> is a cosmopolitan parasitic flatworm causing zoonotic disease in humans and tremendous economic losses by infecting livestock<sup><a href="#CR1">1</a></sup>. The life cycle of <i>F</i>. <i>hepatica</i> is complex and includes a snail as an intermediate host and a mammal as a definitive host. The fluke develops through multiple stages: from eggs to miracidia, sporocysts, rediae, cercariae, metacercariae, newly excysted juveniles (NEJs), and immature flukes which eventually reach the adult stage. Molecular research on this and other helminths has considerably advanced in recent years, with genome data and tools such as RNA interference (RNAi) becoming available for these complex multicellular organisms. Relative quantification of gene expression by quantitative real-time PCR (qRT-PCR) is an essential component of many experimental approaches. The accuracy of such relative gene expression analyses is largely dependent on the stable expression of the reference genes used for normalisation. Housekeeping genes are typically used as reference in qRT-PCR, although in some cases their expression is known to vary in helminths depending on the experimental condition, the parasite stage, or the parasite’s sex<sup><a href="#CR2">2</a>–<a href="#CR4">4</a></sup>. Classical housekeeping genes such as glyceraldehyde-3-phosphate dehydrogenase (<i>gapdh</i>) and <i>ß-actin</i> have often been used based on tradition rather than being experimentally validated as stably expressed genes for the species or parasite stage of interest. However, the validation of expression stabilities of reference genes under the desired experimental conditions is an essential step, which should precede any comparative studies on expression levels of target genes.</p> <p id="Par3">Comparative gene expression analysis in different life-stages is of major interest in <i>F</i>. <i>hepatica</i> research. This includes for instance the validation of expression of potential drug target genes. A new active compound should ideally target all life stages within the final host, including NEJs emerging from metacercariae in the host’s intestine, immature flukes migrating through the body cavity to the liver capsule and through the liver parenchyma causing acute fascioliasis, and adult flukes, which trigger chronic fascioliasis while residing inside the bile ducts where egg deposition occurs<sup><a href="#CR5">5</a></sup>. Gene expression often varies between intra-mammalian life stages of <i>F</i>. <i>hepatica</i> as metabolic, nutritional and locomotor demands vary between stages<sup><a href="#CR6">6</a>,<a href="#CR7">7</a></sup>. Other research approaches include the <i>in vitro</i> culture of flukes, such as the recently established <i>in vitro</i> maturation of NEJs to immature flukes in order to study early development<sup><a href="#CR8">8</a></sup>, or RNAi-mediated knockdown of gene expression which might involve culture for more than 3 weeks<sup><a href="#CR9">9</a></sup>. Gene expression stability in parasites during <i>in vitro</i> culture is likely to differ from non-cultured parasites, because of the known differences from the <i>in vitro</i> conditions compared to the natural environment provided by the host. This motivated us to identify reference genes with stable expression in the intra-mammalian life stages and during <i>in vitro</i> culture of <i>F</i>. <i>hepatica</i>. To this end, we used four different algorithms to assess eight different candidate reference genes and their expression stability in the relevant <i>in vivo</i>-stages (NEJs, 4 week-old immature flukes, 12-week old adults), and in juvenile flukes cultured for 4 weeks <i>in vitro</i>. Out of these candidate genes, three were identified as the most stably expressed reference genes.</p> <p id="Par4">As a first application example, we utilised these selected reference genes to determine the expression level of potentially druggable target genes in the different life stages and during <i>in vitro</i> culture. Kinases are discussed as promising drug targets in parasitic flatworms<sup><a href="#CR10">10</a>,<a href="#CR11">11</a></sup>. For instance, promising anthelmintic effects were observed in immature and adult schistosomes by RNAi or inhibitor treatment against Abelson (Abl) tyrosine kinases and polo-like kinases (PLK)<sup><a href="#CR12">12</a>–<a href="#CR15">15</a></sup>. Surprisingly, in <i>Fasciola</i> research, kinases as drug targets have been largely neglected so far although new anthelmintics are urgently needed facing the spread of triclabendazole resistance around the globe<sup><a href="#CR1">1</a>,<a href="#CR16">16</a>,<a href="#CR17">17</a></sup>. We have identified five kinase orthologs in <i>F</i>. <i>hepatica</i> and characterised their expression by qRT-PCR. Based on the new reference genes, an interesting change in kinase expression patterns during fluke development was obtained. This may substantiate further research activity on kinases in <i>F</i>. <i>hepatica</i>.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="selectionofcandidatereferencegenes" title="sec"> <div class="title" tagx="title" title="title">
Selection of candidate reference genes</div> <p id="Par5">Eight different candidate reference genes were chosen among which we assumed to find stably expressed reference genes suitable for the quantification of gene expression independent from the parasite stage or <i>in vitro</i> cultivation (Fig. <a href="#Fig1">1</a>). The candidates were selected based on already available transcriptome datasets or literature on related trematode species. The candidate reference gene names, biological function and accession numbers are listed in Table <a href="#Tab1">1</a>. In detail, orthologs of two of the genes were previously shown to be stably expressed in selected life stages of the Chinese liver fluke <i>Clonorchis sinensis</i><sup><a href="#CR2">2</a></sup>: β-actin (<i>actb</i>) and small nuclear ribonucleoprotein (<i>snrpa1</i>). Another two candidate genes were chosen because previous transcriptome analyses suggested a stable expression of their orthologs in all life stages of the blood fluke <i>Schistosoma mansoni</i><sup><a href="#CR18">18</a></sup>: tubulin-specific chaperone D (<i>tbcd</i>) and protein phosphatase 1 catalytic subunit beta (<i>ppp1cb</i>). Two candidates were selected because we found them to be most stably expressed during <i>in vitro</i> culture of <i>S</i>. <i>mansoni</i><sup><a href="#CR4">4</a></sup>: leucine zipper and EF-hand containing transmembrane protein 1 (<i>letm1</i>) and proteasome subunit beta type-7 (<i>psmb7</i>). The ortholog of a glutamyl-prolyl-tRNA synthetase (<i>eprs</i>) was included based on its stable expression in different strains of <i>F</i>. <i>hepatica</i><sup><a href="#CR19">19</a></sup>. Finally, for comparison, we included the well-known housekeeping gene <i>gapdh</i>, which is currently widely used for normalisation also in <i>F</i>. <i>hepatica</i> studies<sup><a href="#CR9">9</a>,<a href="#CR20">20</a>,<a href="#CR21">21</a></sup>. Thus, all reference gene candidates play roles in conserved cellular processes like mRNA splicing and translation, cytoskeleton arrangement, glycolysis, mitochondrial organisation, and the regulation of cell growth (Table <a href="#Tab1">1</a>). Orthologs for all eight genes were identified by BLASTp search of known genes in <i>Homo sapiens</i> (accession numbers see Table <a href="#Tab1">1</a>) against the genome of <i>F</i>. <i>hepatica</i>. The presence of relevant conserved protein domains was confirmed by SMART analysis (see Supplementary Fig. <a href="#MOESM1">S1</a>). The identity of the orthologs was further confirmed by multiple alignment of the amino acid sequences with those of model species (see Supplementary Table <a href="#MOESM1">S1</a> and Fig. <a href="#MOESM1">S8</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p><i>In vivo</i> and <i>in vitro</i> stages of <i>F</i>. <i>hepatica</i> under investigation. Stably expressed reference genes were identified for the comparison of gene expression of (<b>A</b>) newly excysted juveniles (NEJs), immature and adult worms, i.e. stages relevant for the mammalian host; and (<b>B</b>) NEJs and <i>in vitro</i>-cultured juvenile worms, which are frequently studied as part of drug testing or knockdown of gene expression.</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> <p>Overview of candidate reference genes for the study of gene expression in <i>F</i>. <i>hepatica</i>.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Gene name</th> <th>Gene ID</th> <th>Homology (e-value)*</th> <th>Protein function</th> </tr> </thead> <tbody> <tr> <td>Fh<i>tbcd</i></td> <td>maker-scaffold10x_815_pilon-snap-gene-1.87</td> <td>Tubulin-specific chaperone D [<i>H</i>. <i>sapiens</i>, NP_005984.3] (7e-177)</td> <td>Cofactor D is one of four proteins involved in the pathway leading to correctly folded beta-tubulin from folding intermediates.</td> </tr> <tr> <td>Fh<i>eprs</i></td> <td>maker-scaffold10x_14_pilon-snap-gene-0.109</td> <td>Glutamyl-prolyl-tRNA synthetase [<i>H</i>. <i>sapiens</i>, NP_004437.2] (0.0)</td> <td>The protein encoded by this gene is a multifunctional aminoacyl-tRNA synthetase that catalyses the aminoacylation of glutamic acid and proline tRNA species.</td> </tr> <tr> <td>Fh<i>letm1</i></td> <td>maker-scaffold10x_721_pilon-snap-gene-0.10</td> <td> <p>Leucine zipper and EF-hand containing transmembrane protein 1 [<i>Homo sapiens</i>, NP_036450.1]</p> <p>(2e-85)</p> </td> <td>The protein functions to maintain the mitochondrial tubular shapes and is required for normal mitochondrial morphology and cellular viability.</td> </tr> <tr> <td>Fh<i>actb</i></td> <td>augustus_masked-scaffold10x_269_pilon-processed-gene-0.18</td> <td>Actin, cytoplasmic 1 [<i>H</i>. <i>sapiens</i>, NP_001092.1] (0.0)</td> <td>Actins are highly conserved proteins that are involved in cell motility, structure, integrity and intracellular signaling. The encoded protein is a major constituent of the contractile apparatus and one of the two non-muscle cytoskeletal actins that are ubiquitously expressed.</td> </tr> <tr> <td>Fh<i>snrpa1</i></td> <td>maker-scaffold10x_234_pilon-snap-gene-0.20</td> <td>U2 small nuclear ribonucleoprotein A’ [<i>H</i>. <i>sapiens</i>, NP_003081.2] (5e-72)</td> <td>This gene encodes a protein which is a component of the spliceosome and it is involved in pre-mRNA splicing.</td> </tr> <tr> <td>Fh<i>ppp1cb</i></td> <td>maker-scaffold10x_238_pilon-snap-gene-0.95</td> <td>Protein phosphatase 1 catalytic subunit beta [<i>H</i>. <i>sapiens</i>, NP_002700.1] (0.0)</td> <td>The protein encoded by this gene is one of the three catalytic subunits of protein phosphatase 1 (PP1). PP1 is a serine/threonine specific protein phosphatase known to be involved in the regulation of a variety of cellular processes.</td> </tr> <tr> <td>Fh<i>psmb7</i></td> <td>maker-scaffold10x_1452_pilon-augustus-gene-0.11</td> <td>Proteasome subunit beta type-7 [<i>H</i>. <i>sapiens</i>, NP_002790.1] (1e-79)</td> <td>Important component of the cellular protein degradation complex.</td> </tr> <tr> <td>Fh<i>gapdh</i></td> <td>maker-scaffold10x_2706_pilon-snap-gene-0.15</td> <td>Glyceraldehyde-3-phosphate dehydrogenase [<i>H</i>. <i>sapiens</i>, NP_001276674.1] (0.0)</td> <td>GAPDH catalyzes the sixth step of the glycolysis by converting D-glyceraldehyde 3-phosphate to 3-phospho-D-glyceroyl phosphate.</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>*</sup>Determined by NCBI BLAST.</p> </div> </div> <p /> </div> <div class="rnaqualityofparasitesamplesandperformanceofqrt-pcrprimers" title="sec"> <div class="title" tagx="title" title="title">
RNA quality of parasite samples and performance of qRT-PCR primers</div> <p id="Par6">Isolation of sufficient amounts of high-quality RNA from low numbers of NEJs is often problematic because of their small size. Using the Monarch RNA Extraction Kit, we managed to isolate and analyse RNA from as little as 10 NEJs. Representative electropherograms from BioAnalyzer analysis of RNA quantity and quality are shown in Supplementary Fig. <a href="#MOESM1">S2</a>. From 10 and 20 NEJs, we obtained in average 4–6 ng and 9–14 ng RNA, respectively. A good RNA integrity was reflected by the distinct 18 S RNA peak. Primer specificity was confirmed by PCR yielding one specific amplification product of expected size (see Supplementary Table <a href="#MOESM1">S2</a>) for each candidate reference gene, without primer-dimer formation or genomic DNA contamination. The primer sequences and amplicon lengths can be found in Supplementary Table <a href="#MOESM1">S2</a>. Absence of unspecific products was also confirmed by melt-curve analysis, which showed a single peak. The expression level of the eight candidate reference genes was assessed by qRT-PCR by determining the Ct value for each sample. With average Ct values of 15.27 to 27.63, all candidate genes were within the range of an acceptable reference gene expression level (15 &amp;lt; Ct &amp;lt; 30) (Fig. <a href="#Fig2">2</a>). The highest expression was found for Fh<i>gapdh</i>, and the lowest for Fh<i>ppp1cb</i>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Distribution of threshold cycle (Ct) values of 8 candidate reference genes across all samples. The solid line represents the median, boxes indicate the 25<sup>th</sup> and 75<sup>th</sup> percentile, and the whiskers represent the minimum and maximum values of averaged qRT-PCR expression data from (<b>A</b>) 9 samples of intra-mammalian <i>in vivo</i> stages (NEJs, immature and adult flukes), and (<b>B</b>) 6 samples of <i>in vitro</i> cultured juveniles (NEJs and 4-week cultured juveniles).</p>   </div> <p /> </div> <div class="identificationofthemoststablyexpressedreferencegenes" title="sec"> <div class="title" tagx="title" title="title">
Identification of the most stably expressed reference genes</div> <p id="Par7">Expression stabilities of the eight candidate reference genes were determined by four different algorithms: NormFinder, geNorm, BestKeeper, and the comparative ΔCt method. As input data, we used expression values either of different intra-mammalian stages (NEJs, 4 week-old immature flukes, 12 week-old adults), or of NEJs prior and after 4 weeks of <i>in vitro</i>-culture (then called 4 week-old juveniles) (Fig. <a href="#Fig1">1</a>). Transcript levels for all samples were determined by absolute quantification against a standard curve. Input data for stability analysis were raw Ct values (BestKeeper), relative Ct values (geNorm, ΔCt method), and calculated concentrations of amplification products (NormFinder), respectively. The most stably expressed reference gene has a low stability value (NormFinder, geNorm, ΔCt method), or a high coefficient of correlation (BestKeeper).</p> <div class="expressionstabilityofcandidatereferencegenesindifferentflukestages" title="sec"> <div class="title" tagx="title" title="title">
Expression stability of candidate reference genes in different fluke stages</div> <p id="Par8">In a first step, the expression stabilities of the selected candidate genes between three different fluke stages relevant for the final host were determined. The obtained ranking of each algorithm is summarised in Fig. <a href="#Fig3">3</a> and Supplementary Table <a href="#MOESM1">S3</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Stability of expression of eight candidate reference genes in three different intra-mammalian life stages of <i>F</i>. <i>hepatica</i>. Stability values were obtained for NEJs, 4 week-old immature, and 12 week-old adult worms using NormFinder (<b>A</b>), geNorm (<b>B</b>), BestKeeper (<b>C</b>), and the comparative ΔCT method (<b>D</b>). Genes were ranked from the least stable (on the left) to the most stable (on the right).</p>   </div> <p /> <p id="Par9">NormFinder ranks genes according to their stability value M, which is based on the size of intra- and inter-group expression variations, i.e. the variation within an experimental group (here: biological replicates) and between different experimental groups (here: fluke stages). A good reference gene is characterised by an M value below 1 in heterogeneous cell or tissue sample sets (Vandesompele 2002). NormFinder identified the tRNA-synthetase Fh<i>eprs</i> and the proteasome subunit Fh<i>psmb7</i> as the two best candidates, whereas Fh<i>gapdh</i> turned out to be the least stably transcribed gene in a combined analysis of all fluke stages (Fig. <a href="#Fig3">3A</a>). Additionally, expression stabilities of the eight reference gene candidates were calculated using the geNorm algorithm and ranked from the most stable to least stable candidate gene (Fig. <a href="#Fig3">3B</a>). geNorm analysis ranked the tubulin chaperone Fh<i>tbcd</i> and Fh<i>eprs</i> as the two best reference genes for fluke stage comparison and, again, Fh<i>gapdh</i> as least suitable candidate. In BestKeeper analysis, the average of the pairwise variations of each gene with all other genes is used to create the stability value M: the lower M is, the more stable are the transcript levels of a gene. The calculated standard deviation [±CP] should not exceed 1. Indeed, standard deviations of all eight genes were &amp;lt;1. BestKeeper identified Fh<i>tbcd</i> and the actin Fh<i>actb</i> as the best reference genes, and the protein phosphatase subunit Fh<i>ppp1cb</i> as worst candidate (Fig. <a href="#Fig3">3C</a>). The fourth algorithm, the ΔCt method, revealed Fh<i>eprs</i> and Fh<i>tbcd</i> as the best candidates and Fh<i>gapdh</i> as the least stably expressed gene (Fig. <a href="#Fig3">3D</a>). These results were equivalent to those generated by geNorm. For some genes, such as Fh<i>actb</i> and Fh<i>gapdh</i>, the predicted expression stability (here: coefficient of correlation) according to BestKeeper was fairly good, whereas these genes were among the least or only average stably expressed genes in the three other analyses. This argues for using more than just one algorithm when performing studies of reference gene validation.</p> <p id="Par10">Because of the slightly heterogeneous rankings produced by the four algorithms, a final global ranking was obtained by assigning the numbers 1–8 to each stability coefficient (with 1 as the most stable and 8 as the least stable gene), and creating the geometric mean of these ranks for each gene. This procedure is equivalent to the global ranking done by the RefFinder tool<sup><a href="#CR22">22</a></sup>. This final ranking revealed Fh<i>eprs</i> and Fh<i>tbcd</i> as the two most stably expressed genes and, therefore, the most suitable reference genes for comparisons of intra-mammalian fluke stages. The commonly used housekeeping gene <i>gapdh</i> ranked last. The final ranking from the lowest to highest calculated average rank was as follows: Fh<i>eprs</i> &amp;lt; Fh<i>tbcd</i> &amp;lt; Fh<i>psmb7</i> &amp;lt; Fh<i>actb</i> &amp;lt; Fh<i>snrpa1</i> &amp;lt; Fh<i>letm1</i> &amp;lt; Fh<i>ppp1cb</i> &amp;lt; Fh<i>gapdh</i> (Fig. <a href="#Fig5">5A</a>).</p> </div> <div class="expressionstabilityofcandidatereferencegenesduringinvitrocultureofjuvenileflukes" title="sec"> <div class="title" tagx="title" title="title">
Expression stability of candidate reference genes during <i>in vitro</i> culture of juvenile flukes</div> <p id="Par11">Next to quantification of gene expression in different fluke stages, gene expression analyses in <i>in vitro</i>-cultured NEJs is another standard approach for various research questions such as expression changes during maturation of juveniles or characterisation of gene function by RNAi<sup><a href="#CR9">9</a>,<a href="#CR20">20</a>,<a href="#CR23">23</a></sup>. Analogous to the stability ranking for the intra-mammalian stages, we used the four different algorithms to identify the most stably expressed reference genes for NEJs before and after 28 days of <i>in vitro</i> culture in an established long-term culture medium<sup><a href="#CR8">8</a></sup>. The individual ranking for each algorithm is summarised in Fig. <a href="#Fig4">4</a> and Supplementary Table <a href="#MOESM1">S4</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Stability of expression of eight candidate reference genes during <i>in vitro</i> culture of juvenile <i>F</i>. <i>hepatica</i>. Stability values were obtained for 1-day old NEJs and juveniles grown for 4 weeks in serum-rich medium using NormFinder (<b>A</b>), geNorm (<b>B</b>), BestKeeper (<b>C</b>), and the comparative ΔCT method (<b>D</b>). Genes were ranked from the least stable (on the left) to the most stable (on the right).</p>   </div> <p /> <p id="Par12">NormFinder, BestKeeper, and the comparative ΔCT method identified Fh<i>tbcd</i> and Fh<i>psmb7</i> as the two most stably expressed genes. geNorm identified Fh<i>tbcd</i> and Fh<i>eprs</i> as most suitable reference genes, as for the analysis of intra-mammalian stages before, followed by Fh<i>psmb7</i>. The least stably expressed gene was Fh<i>ppp1cb</i> (geNorm and ΔCT method) or Fh<i>gapdh</i> (NormFinder and BestKeeper).</p> <p id="Par13">Accordingly, the global ranking based on the geometric mean of individual ranks revealed Fh<i>tbcd</i> and Fh<i>psmb7</i> as the most stably expressed reference genes. The average calculated ranks were as follows: Fh<i>tbcd</i> &amp;lt; Fh<i>psmb7</i> &amp;lt; Fh<i>eprs</i> &amp;lt; Fh<i>snrpa1</i> &amp;lt; Fh<i>actb</i> &amp;lt; Fh<i>letm1</i> &amp;lt; Fh<i>ppp1cb</i> &amp;lt; Fh<i>gapdh</i> (Fig. <a href="#Fig5">5B</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Mean rank of expression stability of eight candidate reference genes in <i>F</i>. <i>hepatica</i>. A number (from 1 to 8) was assigned to each stability coefficient obtained from four algorithms. The mean rank with SEM is shown for (<b>A</b>) the analysis of three intra-mammalian stages (NEJs, immature and adult flukes), and (<b>B</b>) the analysis of <i>in vitro</i>-cultured juveniles (NEJs before and after 28 days culture). Genes were ranked from the least stable (on the left) to the most stable (on the right).</p>   </div> <p /> </div> </div> <div class="relativeexpressionlevelsofreferencegenecandidatesduringdevelopmentandanthelmintictreatment" title="sec"> <div class="title" tagx="title" title="title">
Relative expression levels of reference gene candidates during development and anthelmintic treatment</div> <p id="Par14">To clarify in how far the previously used reference gene Fh<i>gapdh</i><sup><a href="#CR9">9</a>,<a href="#CR20">20</a>,<a href="#CR21">21</a></sup> is differentially regulated during <i>in vivo</i> development or <i>in vitro</i> culture, we relatively quantified its expression compared to the geometric mean of the two best reference genes. Fh<i>gapdh</i> was clearly differentially expressed with a significant upregulation during <i>in vitro</i> culture and during development from NEJs to immature and adult flukes (Fig. <a href="#Fig6">6A,C</a>). A differential expression was also found for most other reference gene candidates positioned on number 4 to 7 in the global ranking (see Supplementary Fig. <a href="#MOESM1">S3</a>). On the contrary, Fh<i>psmb7</i> and Fh<i>eprs</i> were not differentially expressed, being in line with their fairly good stability rank (number 3 in the global rankings) (Fig. <a href="#Fig6">6B,D</a>). Taken together, Fh<i>gapdh</i> as well as all genes from global stability rank 4 and higher appear not to be suitable as reference genes for inter-stage comparisons. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Relative expression levels of a stable and the least stable reference gene candidate in different stages of <i>F</i>. <i>hepatica</i>. Relative quantification is based on normalisation against the geometric mean of the two most stably expressed reference genes identified for (<b>A</b>,<b>B</b>) intra-mammalian stages (<i>tbcd</i> and <i>eprs</i>), or for (<b>C</b>,<b>D</b>) <i>in vitro</i> cultured juveniles (<i>tbcd</i> and <i>psmb7</i>). <i>gapdh</i> (<b>A</b>,<b>C</b>) was previously revealed as the least stable gene, <i>psmb7</i> and <i>eprs</i> (<b>B</b>,<b>D</b>) were among the three most stably expressed genes. Average values of 3–4 biological replicates with SEM are shown. Significant differences are indicated with **<i>p</i> &amp;lt; 0.01, ***<i>p</i> &amp;lt; 0.001 (t-test).</p>   </div> <p /> <p id="Par15">Next, we addressed whether selected reference genes are also stably expressed during anthelmintic treatment of flukes. To this end, adult <i>F</i>. <i>hepatica</i> were cultured with different sublethal concentrations of triclabendazole for 2 days <i>in vitro</i>. Because triclabendazole is known to affect tubulin in <i>F</i>. <i>hepatica</i><sup><a href="#CR24">24</a></sup>, it was of particular interest whether the tubulin-specific chaperone D Fh<i>tbcd</i> would be stably expressed. The expression of all genes investigated (Fh<i>tbcd</i>, Fh<i>psmb7</i>, Fh<i>gapdh</i>) did not change by drug exposure (Fig. <a href="#Fig7">7</a>) and thus appear suitable as reference genes for this type of experimental setting. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Relative expression of selected reference genes after culture with triclabendazole. Adult <i>F</i>. <i>hepatica</i> were cultured for 2 days with 10 µM and 20 µM triclabendazole, or in an equivalent concentration of DMSO as control. Expression of <i>tbcd</i> (<b>A</b>) was normalised against the geometric mean of <i>eprs</i> and <i>psmb7</i>, expression of <i>psmb7</i> (<b>B</b>) and <i>gapdh</i> (<b>C</b>) were normalised against the geometric mean of <i>eprs</i> and <i>tbcd</i>. Average values of 4 biological replicates with SEM are shown.</p>   </div> <p /> </div> <div class="validationofselectedreferencegenesforquantificationoftargetgeneexpressionindifferentflukestages" title="sec"> <div class="title" tagx="title" title="title">
Validation of selected reference genes for quantification of target gene expression in different fluke stages</div> <p id="Par16">The performance of the selected reference genes was validated by quantification of relative expression levels of five orthologs of kinase genes in <i>F</i>. <i>hepatica</i>. Kinases are discussed as promising anthelmintic target of inhibitors<sup><a href="#CR10">10</a>,<a href="#CR25">25</a></sup>. In particular, Abl kinases and polo-like kinase 1 have been studied in the past as potential targets in schistosomes, tape-worms, and filariae<sup><a href="#CR12">12</a>,<a href="#CR26">26</a>–<a href="#CR28">28</a></sup>. Furthermore, the protein kinases B (also called Akt) and C have been investigated as potential targets in <i>S</i>. <i>mansoni</i><sup><a href="#CR13">13</a>,<a href="#CR29">29</a>,<a href="#CR30">30</a></sup>. For all these kinases, RNAi or inhibitor treatment <i>in vitro</i> revealed anthelmintic effects on immature or adult schistosome stages.</p> <p id="Par17">All the more surprising is that neither of these kinases has been studied to date in <i>Fasciola</i>. Knowledge on kinase expression in all life stages relevant for drug targeting is desirable, as well as a reliable quantification of expression during <i>in vitro</i> culture, for instance as part of knockdown experiments. Therefore, we identified orthologs of above mentioned kinases by BLASTp search (Table <a href="#Tab2">2</a>): Fh<i>abl1</i> and Fh<i>abl2</i> as orthologs of the protein tyrosine kinases <i>abl1</i> and <i>abl2</i>, which play roles in a variety of cellular processes including cell differentiation and cytoskeletal rearrangements<sup><a href="#CR31">31</a></sup>; Fh<i>akt1</i> and Fh<i>pkc</i> as orthologs of the serine/threonine-protein kinases B and C, which are known to regulate many processes including cell metabolism, proliferation, and survival<sup><a href="#CR32">32</a>,<a href="#CR33">33</a></sup>; Fh<i>plk1</i> as an ortholog of polo-like kinase 1 with important roles during cell cycle progression<sup><a href="#CR34">34</a></sup>. The presence of conserved protein domains was confirmed by SMART analysis (see Supplementary Fig. <a href="#MOESM1">S4</a>). For instance, Fh<i>plk1</i> contained a typical kinase domain and two polo-box domains. The identity of the kinase orthologs was further confirmed by multiple alignment of the amino acid sequences against several model species (see Supplementary Fig. <a href="#MOESM1">S9</a>). <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>Overview of genes of interest for the study of gene expression in <i>F</i>. <i>hepatica</i>.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Gene of interest</th> <th>Gene ID</th> <th>Homology (e-value)<sup>*</sup></th> <th>Protein function</th> </tr> </thead> <tbody> <tr> <td>Fh<i>abl1</i></td> <td>maker-scaffold10x_1995_pilon-snap-gene-0.46</td> <td>ABL proto-oncogene 1, non-receptor tyrosine kinase [<i>H</i>. <i>sapiens</i>, NP_005148.2] (2e-119)</td> <td>Protein tyrosine kinase involved in a variety of cellular processes, including cell division, adhesion, differentiation, and response to stress.</td> </tr> <tr> <td>Fh<i>abl2</i></td> <td>maker-scaffold10x_873_pilon-snap-gene-0.69</td> <td>ABL proto-oncogene 2, non-receptor tyrosine kinase isoform e [<i>H</i>. <i>sapiens</i>, NP_001129473.1] (2e-135)</td> <td>Protein tyrosine kinase with a role in cytoskeletal rearrangements through its F-actin- and microtubule-binding sequences.</td> </tr> <tr> <td>Fh<i>akt1</i></td> <td>maker-scaffold10x_205_pilon-augustus-gene-0.40</td> <td>Rac-alpha serine/threonine-protein kinase [<i>H</i>. <i>sapiens</i>, NP_001014431.1] (9e-137)</td> <td>The serine/threonine-protein kinase AKT1 is also known as protein kinase B. AKT kinases regulate many processes including metabolism, proliferation, cell survival, and growth.</td> </tr> <tr> <td>Fh<i>pkc</i></td> <td>maker-scaffold10x_608_pilon-snap-gene-0.5</td> <td>Protein kinase C iota [<i>H</i>. <i>sapiens</i>, NP_002731.4] (0.0)</td> <td>A serine/threonine protein kinase involved in cell survival, differentiation and polarity. It plays a role in microtubule dynamics in the early secretory pathway.</td> </tr> <tr> <td>Fh<i>plk1</i></td> <td>maker-scaffold10x_784_pilon-snap-gene-0.36</td> <td>Polo-like kinase 1 [<i>H</i>. <i>sapiens</i>, NP_005021.2] (0.0)</td> <td>The serine/threonine protein kinase is highly expressed during mitosis and performs several important functions throughout the M phase of the cell cycle.</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>*</sup>Determined by NCBI BLAST.</p> </div>  <p /> <p id="Par18">For both experimental groups (<i>in vivo</i> and <i>in vitro</i> stages), we quantified kinase expression using the geometric mean of the two most stably expressed reference genes for normalisation, which were Fh<i>eprs</i> and Fh<i>tbcd</i>, and Fh<i>psmb7</i> and Fh<i>tbcd</i>, respectively. The five kinase genes were found to be expressed in all intra-mammalian life stages. The highest relative expression was observed for Fh<i>abl1</i> and Fh<i>pkc</i>. Interestingly, two types of expression patterns during fluke development were revealed: while Fh<i>plk1</i> was expressed highest in adult flukes and low in NEJs and immature flukes, expression of all other kinases was highest in NEJs and significantly downregulated during development to immature and adult flukes (Fig. <a href="#Fig8">8</a>). <i>In vitro</i> maturation of NEJs to immature flukes by 4-week culture revealed very similar expression patterns (Fig. <a href="#Fig9">9</a>) as seen during <i>in vivo</i> development. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <p>Relative expression levels of kinases in three different intra-mammalian life stages of <i>F</i>. <i>hepatica</i>. Expression data from NEJs, 4 week-old immature, and 12 week-old adult worms were normalised against the geometric mean of the two most stably expressed reference genes (<i>eprs</i> and <i>tbcd</i>). Average values of 3–4 biological replicates with SEM are shown. Significant differences are indicated with **<i>p</i> &amp;lt; 0.01, ***<i>p</i> &amp;lt; 0.001 (t-test). <i>abl1</i>, tyrosine-protein kinase Abl1; <i>abl2</i>, tyrosine-protein kinase Abl2; <i>pkc</i>, protein kinase C; <i>akt1</i>, Rac-alpha serine/threonine-protein kinase 1; <i>plk1</i>, polo-like kinase 1</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9</span></a></div>   <p>Relative expression levels of kinases during <i>in vitro</i> culture of juvenile <i>F</i>. <i>hepatica</i>. Expression data from NEJs and juvenile worms grown for 4 weeks in serum-rich medium were normalised against the geometric mean of the two most stably expressed reference genes (<i>tbcd</i> and <i>psmb7</i>). Average values of 3–4 biological replicates with SEM are shown. Significant differences are indicated with **<i>p</i> &amp;lt; 0.01, ***<i>p</i> &amp;lt; 0.001 (t-test). <i>abl1</i>, tyrosine-protein kinase Abl1; <i>abl2</i>, tyrosine-protein kinase Abl2; <i>pkc</i>, protein kinase C; <i>akt1</i>, Rac-alpha serine/threonine-protein kinase 1; <i>plk1</i>, polo-like kinase 1.</p>   </div> <p /> <p id="Par19">As a proof of principle, we also used Fh<i>gapdh</i> for normalisation of kinase gene expressions to assess whether using a suboptimal gene would yield different results compared to using the top ranked reference genes. While the overall kinase expression patterns were largely similar, a differing expression pattern was revealed for some kinase genes and some stages. As example, Fh<i>gapdh</i> suggested a significant downregulation of Fh<i>abl1</i> and Fh<i>akt1</i> expression from NEJs to immature flukes (Supplementary Fig. <a href="#MOESM1">S5</a>), while there was no expression difference based on the stable expressed reference genes (Fig. <a href="#Fig8">8</a>). On the other hand, the significantly upregulated expression of Fh<i>plk1</i> from NEJs to immature flukes was not evident when normalised against Fh<i>gapdh</i>. Even more striking, Fh<i>gapdh</i> suggested a significant downregulation of Fh<i>plk1</i> during <i>in vitro</i> culture of juveniles (Supplementary Fig. <a href="#MOESM1">S6</a>), while in fact a trend for increased expression was demonstrated before (Fig. <a href="#Fig9">9</a>). This clearly shows that using a suboptimal reference gene for normalisation might give differing or even opposite expression results for a gene of interest.</p> <p id="Par20">The developmental expression changes for kinase genes found in <i>F</i>. <i>hepatica</i> matched in parts with expression patterns of orthologs in the related blood fluke <i>S</i>. <i>mansoni</i>. Previous studies showed that the schistosome <i>abl</i> kinase genes Sm<i>abl1</i> and Sm<i>abl2</i> were downregulated during maturation from schistosomula to adults, and <i>plk1</i> was strongly upregulated, at least in adult females<sup><a href="#CR3">3</a>,<a href="#CR18">18</a></sup> (Supplementary Fig. <a href="#MOESM1">S7</a>). Taken together, various kinases are expressed in intra-mammalian life stages of <i>F</i>. <i>hepatica</i>, and expression changes during <i>in vivo</i> development are mimicked by <i>in vitro</i> culture.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par21">Proteomic studies revealed striking differences of gene expression among the life stages of <i>F</i>. <i>hepatica</i> occurring in the mammalian host<sup><a href="#CR6">6</a>,<a href="#CR7">7</a></sup>. This has important practical implication for vaccine development and drug target research. Knowledge on target gene expression in all life stages relevant for anthelmintics development is desirable as well as reliable methods for the quantification of gene expression during <i>in vitro</i> culture, for instance as part of target gene validation using knockdown experiments.</p> <p id="Par22">Per definition, housekeeping genes are essential for maintaining the cellular function and therefore, in theory, should be stably expressed. In practice, however, they may turn out to be regulated to some extent depending on the organisms, developmental stages, tissue types, and experimental settings<sup><a href="#CR35">35</a></sup>. This requires an accurate validation of candidate genes as reference genes for e.g. qRT-PCR studies. As classical housekeeping genes, <i>gapdh</i> and <i>ß-actin</i> have been used for normalisation in gene expression studies in all kind of organisms, including helminths<sup><a href="#CR9">9</a>,<a href="#CR20">20</a>,<a href="#CR21">21</a>,<a href="#CR36">36</a></sup>. At least for the oriental liver fluke, <i>Clonorchis sinensis</i>, a stable expression of <i>ß-actin</i> was demonstrated in the comparison of two life stages, metacercariae and adults. In the same study, however, <i>gapdh</i> exhibited poor expression stability<sup><a href="#CR2">2</a></sup>. Furthermore, a <i>gapdh</i> ortholog was among the least stable candidate genes for gene expression studies in adult schistosomes cultured <i>in vitro</i><sup><a href="#CR4">4</a></sup>. In our study with <i>F</i>. <i>hepatica</i>, <i>gapdh</i> showed the lowest and <i>ß-actin</i> only a moderate expression stability among eight candidate genes tested. The transcript levels of <i>gapdh</i> appeared to be significantly upregulated during maturation of the fluke. This is of particular relevance as <i>gapdh</i> is currently a standard gene used for normalisation of gene expression in <i>F</i>. <i>hepatica</i> studies<sup><a href="#CR9">9</a>,<a href="#CR20">20</a>,<a href="#CR21">21</a></sup>, but based on our results, <i>gapdh</i> may not be the most suitable reference gene for inter-stage comparison of gene expression by qRT-PCR. This implies that without additional validation, the selection of reference genes for gene expression studies in one species should not be based on results obtained in a related species because there is no guarantee for comparable expression profiles.</p> <p id="Par23">We aimed at identifying stably expressed genes among a selection of eight candidate reference genes for two different experimental settings often used in <i>Fasciola</i> research: the comparison of life stages relevant for the mammalian host, and the <i>in vitro</i> culture of juvenile flukes. The four algorithms resulted in a slightly divergent ranking of the genes, which was expected from previous studies on other organisms, and which can be explained by the different type of input data and data processing used by the algorithms<sup><a href="#CR4">4</a>,<a href="#CR37">37</a></sup>. Therefore, a global ranking based on the results of all algorithms was performed. In the three intra-mammalian stages, the glutamyl-prolyl-tRNA synthetase Fh<i>eprs</i> and tubulin-specific chaperone D Fh<i>tbcd</i> were most stably expressed. In cultured parasites, this applied to Fh<i>tbcd</i> and the proteasome subunit beta type-7 Fh<i>psmb7</i>.</p> <p id="Par24">The tubulin-specific chaperone TBCD is one of four proteins involved in the pathway leading to correctly folded beta-tubulin from folding intermediates. Being involved in the regulation of microtubule polymerisation and depolymerisation<sup><a href="#CR38">38</a>,<a href="#CR39">39</a></sup>, it is required for crucial cellular processes such as proper assembly of the mitotic spindle and correct progression of mitosis. The glutamyl-prolyl-tRNA synthetase EPRS belongs to the family of aminoacyl-tRNA synthetases, which charge tRNAs with their corresponding amino acids. Accordingly, EPRS catalyses the aminoacylation of proline and glutamic acid tRNA species<sup><a href="#CR40">40</a></sup>. The proteasome subunit beta type-7 PSMB7 is part of the 20S and 26S proteasome complexes and thus involved in the proteolytic degradation of most intracellular proteins. The proteasome plays a key role in the maintenance of protein homeostasis by removing unneeded proteins, and damaged or misfolded proteins that could impair cellular functions<sup><a href="#CR41">41</a></sup>. Because microtubule function, tRNA synthesis, and proteolytic degradation are essential processes for all cells, and presumably independent of any developmental stage, it was not surprising that <i>tbcd</i>, <i>eprs</i> and <i>psmb7</i> turned out as the most stably expressed genes in our study.</p> <p id="Par25">The stability ranking obtained for <i>in vitro</i>-cultured liver flukes is in part similar to the ranking of a related study in <i>S</i>. <i>mansoni</i>. Here the ortholog for proteasome subunit beta type-7, Sm<i>psmb7</i>, ranked also second best during <i>in vitro</i> culture of adult worms, and Sm<i>gapdh</i> was among the least stably expressed genes<sup><a href="#CR4">4</a></sup>. In contrast, the most stable gene in adult schistosomes, <i>letm1</i>, was among the least stable genes investigated in <i>Fasciola</i>. Heterogenous rankings were also obtained for <i>tbcd</i> and <i>ppp1cb</i>: both genes were revealed to be stably expressed in the different life stages of <i>S</i>. <i>mansoni</i> by a meta-analysis study<sup><a href="#CR18">18</a></sup>, while in <i>F</i>. <i>hepatica</i>, only <i>tbcd</i> was. Taken together, the most stably expressed reference genes among the tested candidates in <i>F</i>. <i>hepatica</i> are housekeeping genes belonging to the family of tRNA synthetases, proteasome subunits, and the microtubule machinery.</p> <p id="Par26">For a first application of the newly identified reference genes in qRT-PCR experiments, we focused on kinase genes since they are discussed as potential druggable targets in various helminth species<sup><a href="#CR10">10</a>,<a href="#CR26">26</a>,<a href="#CR27">27</a></sup>. Surprisingly, kinases as drug targets have been largely neglected so far in <i>Fasciola</i> research. In the past, selected kinases have been in focus mainly as vaccine candidate (phosphoglycerate kinase) or as marker gene for discriminating hybrids of <i>Fasciola spp</i>. (phosphoenolpyruvate carboxykinase)<sup><a href="#CR42">42</a>,<a href="#CR43">43</a></sup>. Phosphofructokinase seems to be the only kinase of <i>F</i>. <i>hepatica</i> studied as potential chemotherapeutic target, in work by Mansour dating back as far as 1962<sup><a href="#CR44">44</a>,<a href="#CR45">45</a></sup>, but was not further followed because of suboptimal <i>in vivo</i> efficacy of an phosphofructokinase inhibitor<sup><a href="#CR46">46</a></sup>. To move kinases more into the spotlight of <i>Fasciola</i> anthelmintics research, we identified five kinase genes in <i>F</i>. <i>hepatica</i> and quantified their expression during development at the transcriptional level, <i>in vivo</i> and <i>in vitro</i>. For orthologs of all these kinases, promising anthelmintic effects have been obtained <i>in vitro</i> by knockdown of kinase gene expression or kinase inhibitor treatment in other parasitic flatworms including <i>S</i>. <i>mansoni</i><sup><a href="#CR12">12</a>–<a href="#CR15">15</a>,<a href="#CR26">26</a>,<a href="#CR28">28</a>–<a href="#CR30">30</a></sup>. Our analyses showed that these kinases were expressed in all intra-mammalian stages of <i>F</i>. <i>hepatica</i>. Furthermore, interesting expression patterns were detected throughout development. The potential polo-like kinase 1 ortholog Fh<i>plk1</i> was found to be highly expressed in adults but low in NEJs or immature flukes. This might suggest a role of Fh<i>plk1</i> particularly for the mature stage. A similar expression pattern was observed in <i>S</i>. <i>mansoni</i>, where Sm<i>plk1</i> expression was mainly found in germinal cells of adult worms<sup><a href="#CR15">15</a></sup>. Accordingly, inhibition or RNAi of Sm<i>plk1</i> affected egg production and gonad morphology<sup><a href="#CR13">13</a>,<a href="#CR15">15</a></sup>. Opposite to Fh<i>plk1</i>, orthologs of the two Abl kinases and two serine/threonine-protein kinases showed a peak of expression in NEJ. In other organisms, these kinases are amongst others involved in cytoskeleton remodeling in response to extracellular stimuli such as growth factors, and in the regulation of cell metabolism<sup><a href="#CR31">31</a>–<a href="#CR34">34</a></sup>. Thus, these kinases might play important roles during early growth and development of flukes, which still has to be substantiated in functional studies in the future.</p> <p id="Par27">That kinases were found to be expressed in all intra-mammalian stages appears as a prerequisite for any novel target in <i>F</i>. <i>hepatica</i>, because new compounds should preferably be able to hit all developmental stages in the final host, as does the current gold standard triclabendazole<sup><a href="#CR17">17</a></sup>. Whether the significantly different mRNA expression levels between parasite stages found for most kinases will lead to a difference in susceptibility to target inhibition should be part of future studies. A first target gene validation can be achieved by knockdown using RNAi. For such an <i>in vitro</i> culture experiment, it should be taken into account that according to our findings, kinase transcript levels significantly change during 4 weeks of culture. Furthermore, for all studied kinases, inhibitor treatment <i>in vitro</i> has revealed anthelmintic effects on other parasitic flatworms<sup><a href="#CR12">12</a>,<a href="#CR14">14</a>,<a href="#CR15">15</a>,<a href="#CR26">26</a>,<a href="#CR28">28</a>–<a href="#CR30">30</a></sup>. Thus, it is certainly worth testing several of the known kinase inhibitors, such as imatinib and BI 2536, against the different stages of <i>F</i>. <i>hepatica</i> in near future. To this end, first results obtained by us indicate that imatinib has also the potential of killing <i>Fasciola in vitro</i> (Haeberlein, unpublished results).</p> <p id="Par28">To conclude, for future expression analyses by qRT-PCR in <i>F</i>. <i>hepatica</i> we propose using the glutamyl-prolyl-tRNA synthetase Fh<i>eprs</i> and tubulin-specific chaperone D Fh<i>tbcd</i> as reference genes for studies dealing with the different intra-mammalian fluke stages. Beyond that, we suggest using Fh<i>tbcd</i> and the proteasome subunit beta type-7 Fh<i>psmb7</i> for studies on <i>in vitro</i>-cultured juvenile flukes, such as for RNAi experiments. Especially for inter-stage comparisons, these new reference genes have the potential to replace the traditional housekeeping gene <i>gapdh</i> which is used in many <i>Fasciola</i> studies to date<sup><a href="#CR9">9</a>,<a href="#CR20">20</a>,<a href="#CR21">21</a></sup>, but turned out to be differentially expressed during fluke development in our analysis. We also like to motivate, as a good laboratory practice, to newly validate the suitability of reference genes for studies that use different experimental setups than ours, such as extended drug treatment studies. Using the newly defined reference genes from our study, we quantified expression of kinase orthologs in all relevant intra-mammalian life stages important for drug targeting, which revealed distinct expression patterns throughout development pointing to interesting biological functions. Together with the previously identified broad anthelminthic activity of some kinase inhibitors, this motivates for validation experiments on kinases as potential targets in <i>F</i>. <i>hepatica</i>.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Material and Methods</div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics statement</div> <p id="Par29">Animal experiments were performed in accordance with the German Animal Welfare act in addition to national and international guidelines for animal welfare and were approved by the ethics commission of the Institutional Animal Care and Use Committee (IACUC) of the German Lower Saxony State Office for Consumer Protection and Food Safety (<i>Niedersaechsisches Landesamt für Verbraucherschutz und Lebensmittelsicherheit</i>) under reference number 33.8-42502-05-118A336.</p> </div> <div class="parasites" title="sec"> <div class="title" tagx="title" title="title">
Parasites</div> <p id="Par30">Metacercariae from an Italian strain of <i>F</i>. <i>hepatica</i> were purchased from Ridgeway Research (UK). Excystment was done as previously described with some modifications<sup><a href="#CR9">9</a></sup>. Briefly, the outer layer of the metacercariae was physically removed using a scalpel followed by 3–5 min exposure to 10% bleach (v/v). Metacercariae were then incubated in excystment solution (0.6% w/v sodium bicarbonate, 0.45% w/v sodium chloride, 0.4% w/v sodium tauroglycocholate, 0.025 M HCl, 0.4% w/v L-cysteine) for at least 1–2 h at 37 °C and 5% CO<sub>2</sub> until NEJs started to hatch. NEJs were collected in complete RPMI medium (containing 1% ABAM-solution (10,000 units penicillin, 10 mg streptomycin and 25 mg amphotericin B per ml)) and snap-frozen in liquid nitrogen at 24 h after excystment. Immature and adult worms were harvested from livers of sheep experimentally infected with 250 metacercariae at week 4 and 12 post-infection, respectively. Worms were kept for 1 h in 0.9% NaCl (w/v) to allow clearance of gut contents. All parasite stages were snap-frozen in liquid nitrogen. Samples were stored at −80 °C until further usage.</p> </div> <div class="invitroculture" title="sec"> <div class="title" tagx="title" title="title"><i>In vitro</i> culture</div> <p id="Par31">In order to grow juvenile <i>F</i>. <i>hepatica in vitro</i>, NEJs were incubated on day 1 post excystment in complete RPMI1640 medium supplemented with 50% chicken serum at 37 °C and 5% CO<sub>2</sub><sup><a href="#CR8">8</a></sup>. Medium was changed regularly (2–3 times per week). Juveniles were incubated in density of 10 juveniles per ml. At week 4 of culture, juveniles were harvested, snap-frozen in liquid nitrogen and stored at −80 °C until further usage. To study stability of reference gene expression after anthelmintic exposure, adult <i>F</i>. <i>hepatica</i> were cultured for 2 days in complete RPMI1640 with 5% chicken serum and supplemented with 10 µM and 20 µM triclabendazole (dissolved in DMSO), or supplemented with DMSO as present in the highest drug concentration as a negative control. Medium and compounds were refreshed after 24 h and worms snap-frozen in liquid nitrogen and stored at −80 °C until RNA extraction.</p> </div> <div class="rnaisolationandcdnasynthesis" title="sec"> <div class="title" tagx="title" title="title">
RNA isolation and cDNA synthesis</div> <p id="Par32">Total RNA from all life stages was extracted using the Monarch total RNA Miniprep kit (New England BioLabs) following the manufacturer’s protocol. In brief, NEJs, <i>in vitro</i>-grown juveniles, and immature worms were incubated in 300 µl of 1x RNA/DNA protection buffer. Adult worms were chopped in pieces and incubated in 600 µl of the reagent. All samples were subjected to mechanical homogenisation using pestles. Sample sizes ranged between 30–40 NEJs per replicate, 5–10 <i>in vitro</i>-grown juvenile worms, and 1 each for immature and adult worms. RNA quality and quantity were checked by electropherogram analysis using the BioAnalyzer 2100 and an Agilent RNA 6000 Pico or Nano Chip according to the manufacturer’s instructions (Agilent Technologies, USA). Synthesis of cDNA was performed using the QuantiTect Reverse Transcription Kit (QIAGEN, Germany) comprising a genomic DNA wipeout step and 11 ng of total RNA per reaction. cDNAs were diluted 1:10 before being used as template in qRT-PCR.</p> </div> <div class="quantitativereal-timepcr" title="sec"> <div class="title" tagx="title" title="title">
Quantitative real-time PCR</div> <p id="Par33">All primers used for qRT-PCR experiments were designed for a melting temperature of 60 °C and an amplicon size of 140–200 bp (Supplementary Table <a href="#MOESM1">S2</a>), using the Primer3Plus software tool<sup><a href="#CR47">47</a></sup>. When possible, primer pairs were located on different exons of a gene to distinguish amplification of contaminating genomic DNA by size. Prior to qRT-PCR, all primer pairs were tested under standard PCR conditions using the FirePol taq polymerase (Solis BioDyne, Estonia). PCR products were checked for specificity and occurrence of primer dimers on a 2% agarose gel. Only primer pairs yielding in one specific product with no primer dimers were further used for qRT-PCR. Appropriate PCR products were gel extracted (GeneJET gel extraction kit; Thermo Scientific, USA) and used to prepare a standard-curve with 1:10 dilution steps to test primer efficiencies<sup><a href="#CR48">48</a></sup>. Only primers with an efficiency of 90–100% were used for subsequent analyses. Primers were commercially synthesised by Integrated DNA Technologies IDT (USA).</p> <p id="Par34">The 2x PerfeCTa SYBR Green SuperMix (Quantabio, USA) was used in qRT-PCRs for the detection of synthesised DNA double strands in a final volume of 10 µl and 400 nM of each primer. Analysis was performed on a Rotorgene Q cycler (QIAGEN, Germany) with the following conditions: initial denaturation step at 95 °C for 3 min, 45 cycles at 95 °C for 10 sec, 60 °C for 15 sec, and 72 °C for 20 sec. Melting point analyses were performed for each primer pair to verify primer specificity and to exclude the generation of primer dimers or unspecific side-products. All qRT-PCRs were performed in three to four biological replicates with three technical replicates for each sample. Amplified PCR products were calculated by absolute quantification against a standard curve<sup><a href="#CR49">49</a></sup>. The expression of genes of interest was determined by relative quantification against the geometric mean of two selected reference genes. Relative expression levels were calculated by expressing the data as n-fold difference by the formula: relative expression = 2<sup>−delta Ct</sup> × f, with f = 1000 as an arbitrary factor.</p> </div> <div class="evaluationofexpressionstabilityofreferencegenes" title="sec"> <div class="title" tagx="title" title="title">
Evaluation of expression stability of reference genes</div> <p id="Par35">Four different software algorithms were used to determine the transcription stability of selected candidate reference genes: NormFinder, geNorm, BestKeeper, and the comparative ΔCT method<sup><a href="#CR50">50</a>–<a href="#CR53">53</a></sup>. Two separate sets of analyses were performed. On the one hand, all samples from NEJs, immature, and adult worms were analysed to obtain the most stably transcribed genes for studies dealing with different life stages of the parasite. On the other hand, all samples from NEJs and <i>in vitro</i>-grown juvenile worms were used to reveal the best reference genes for gene expression studies in <i>in vitro</i>-culture experiments.</p> <p id="Par36">The algorithm NormFinder determines intra- and inter-group variations across the different samples to calculate a stability value (M). Low variations give a low stability value, which indicates stable transcription of a gene. As input data, the calculated concentrations of qPCR amplification were used<sup><a href="#CR50">50</a></sup>. The geNorm algorithm calculates pairwise variations of each reference gene when compared with the other genes based on relative Ct values. The stability value (M) is based on the average of these pairwise variations. Again, a stable transcription is reflected by a low M value. BestKeeper analysis was performed on raw Ct values. This algorithm assumes that stable reference genes should display similar transcription patterns, i.e. are highly correlated to each other. This is reflected by a high coefficient of correlation (r), whereby the most stably transcribed genes exhibit values closest to 1. The comparative ΔCT method compares the difference in Ct values of reference genes in pairs. Ranking is based on the variability of averaged standard deviations<sup><a href="#CR53">53</a></sup>.</p> </div> <div class="insilicoanalyses" title="sec"> <div class="title" tagx="title" title="title"><i>In silico</i> analyses</div> <p id="Par37">Eight candidate reference genes and five kinase genes of <i>F</i>. <i>hepatica</i> were identified by BLASTp search of the known human orthologs against the genome of <i>F</i>. <i>hepatica</i> (Centre for Genomic Research, University of Liverpool, BioProject ID PRJEB25283) using the public domain tool WormBase ParaSite, version WBPS13 (<a href="https://parasite.wormbase.org">https://parasite.wormbase.org</a>)<sup><a href="#CR54">54</a></sup>. Gene names, biological function, and accession numbers of <i>H</i>. <i>sapiens</i> and <i>F</i>. <i>hepatica</i> are listed in Tables <a href="#Tab1">1</a> and <a href="#Tab2">2</a>. The BLASTp cutoff for the identification of potential orthologs was 5E-72. The identity of the potential <i>F</i>. <i>hepatica</i> orthologs was confirmed by analysis of conserved protein domains using SMART (<a href="http://smart.embl-heidelberg.de/">http://smart.embl-heidelberg.de/</a>)<sup><a href="#CR55">55</a></sup>, and by multiple alignment of amino acid sequences against the sequences of several model species or related species (<i>H</i>. <i>sapiens</i>, <i>M</i>. <i>musculus</i>, <i>C</i>. <i>elegans</i>, <i>D</i>. <i>melanogaster</i>, <i>S</i>. <i>mansoni</i>) using CLUSTALW. The accession numbers for all species used for multiple alignment are listed in Supplementary Table <a href="#MOESM1">S1</a>.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par38">Statistically significant differences between samples were determined by t-test. Error bars represent the standard error of the mean (SEM). <i>p</i>-values &amp;lt; 0.05 were considered significant.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2019_52416_MOESM1_ESM.pdf">LINK</a> <p>Supplementary</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p>is available for this paper at 10.1038/s41598-019-52416-x.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors thank Marcel Haas for excellent technical assistance as well as Aaron Maule and his team at Queen’s University Belfast for introduction to the <i>F</i>. <i>hepatica</i> culture model. The work was funded by the excellence initiative of the Hessian Ministry of Science, Higher Education and Art (HMWK) supporting the LOEWE Centre DRUID and by a stipend of the Faculty of Veterinary Medicine of the Justus Liebig University Giessen awarded to S.H. URL: <a href="https://wissenschaft.hessen.de/wissenschaft/landesprogramm-loewe">https://wissenschaft.hessen.de/wissenschaft/landesprogramm-loewe</a>.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>H.H. contributed to acquisition, analysis, and interpretation of data, and prepared figures and tables. O.P. contributed to acquisition of data. C.S. provided parasite material. S.H. designed the study, contributed to analysis and interpretation of data, drafted the manuscript, and provided funding. All authors have reviewed the manuscript.</p> <div class="title" tagx="title" title="title">
Data availability</div> <p>All data generated or analysed during this study are included in this published article and its Supplementary Information Files.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par39">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webb</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cabada</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Recent developments in the epidemiology, diagnosis, and treatment of <i>Fasciola</i> infection</span><span class="source" tagx="source" title="source">Current opinion in infectious diseases</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">409</span><span class="lpage" tagx="lpage" title="lpage">414</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/qco.0000000000000482">10.1097/qco.0000000000000482</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30113327">30113327</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoo</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Reference genes for quantitative analysis on <i>Clonorchis sinensis</i> gene expression by real-time PCR</span><span class="source" tagx="source" title="source">Parasitology research</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">104</span><span class="fpage" tagx="fpage" title="fpage">321</span><span class="lpage" tagx="lpage" title="lpage">328</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00436-008-1195-x">10.1007/s00436-008-1195-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18815810">18815810</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Schistosome sex matters: a deep view into gonad-specific and pairing-dependent transcriptomes reveals a complex gender interplay</span><span class="source" tagx="source" title="source">Scientific reports</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">31150</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep31150">10.1038/srep31150</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27499125">27499125</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haeberlein</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of a new panel of reference genes to study pairing-dependent gene expression in <i>Schistosoma mansoni</i></span><span class="source" tagx="source" title="source">International journal for parasitology</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">615</span><span class="lpage" tagx="lpage" title="lpage">624</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijpara.2019.01.006">10.1016/j.ijpara.2019.01.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31136746">31136746</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dawes</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fascioliasis: the invasive stages in mammals</span><span class="source" tagx="source" title="source">Advances in parasitology</span><span class="year" tagx="year" title="year">1970</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">259</span><span class="lpage" tagx="lpage" title="lpage">274</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0065-308X(08)60257-2">10.1016/S0065-308X(08)60257-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4934032">4934032</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Maggio</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Across intra-mammalian stages of the liver fluke <i>Fasciola hepatica</i>: a proteomic study</span><span class="source" tagx="source" title="source">Scientific reports</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">32796</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep32796">10.1038/srep32796</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27600774">27600774</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donnelly</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dalton</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranganathan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">An integrated transcriptomics and proteomics analysis of the secretome of the helminth pathogen <i>Fasciola hepatica</i>: proteins associated with invasion and infection of the mammalian host</span><span class="source" tagx="source" title="source">Molecular &amp;amp; cellular proteomics: MCP</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1891</span><span class="lpage" tagx="lpage" title="lpage">1907</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/mcp.M900045-MCP200">10.1074/mcp.M900045-MCP200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19443417">19443417</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCusker</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Stimulating neoblast-like cell proliferation in juvenile <i>Fasciola hepatica</i> supports growth and progression towards the adult phenotype <i>in vitro</i></span><span class="source" tagx="source" title="source">PLoS neglected tropical diseases</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e0004994</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0004994">10.1371/journal.pntd.0004994</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27622752">27622752</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McVeigh</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">RNAi dynamics in juvenile <i>Fasciola</i> spp. liver flukes reveals the persistence of gene silencing <i>in vitro</i></span><span class="source" tagx="source" title="source">PLoS neglected tropical diseases</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e3185</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0003185">10.1371/journal.pntd.0003185</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25254508">25254508</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gelmedin</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dissous</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grevelding</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Re-positioning protein-kinase inhibitors against schistosomiasis</span><span class="source" tagx="source" title="source">Future medicinal chemistry</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">737</span><span class="lpage" tagx="lpage" title="lpage">752</span><span class="pub-id"><a href="https://dx.doi.org/10.4155/fmc.15.31">10.4155/fmc.15.31</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25996067">25996067</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grevelding</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langner</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dissous</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Kinases: molecular stage directors for schistosome development and differentiation</span><span class="source" tagx="source" title="source">Trends in parasitology</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">246</span><span class="lpage" tagx="lpage" title="lpage">260</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pt.2017.12.001">10.1016/j.pt.2017.12.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29276074">29276074</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beckmann</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grevelding</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Imatinib has a fatal impact on morphology, pairing stability and survival of adult <i>Schistosoma mansoni in vitro</i></span><span class="source" tagx="source" title="source">International journal for parasitology</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">521</span><span class="lpage" tagx="lpage" title="lpage">526</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijpara.2010.01.007">10.1016/j.ijpara.2010.01.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20149792">20149792</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guidi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Application of RNAi to genomic drug target validation in schistosomes</span><span class="source" tagx="source" title="source">PLoS neglected tropical diseases</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e0003801</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0003801">10.1371/journal.pntd.0003801</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25992548">25992548</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure-bioactivity relationship for benzimidazole thiophene inhibitors of polo-like kinase 1 (PLK1), a potential drug target in <i>Schistosoma mansoni</i></span><span class="source" tagx="source" title="source">PLoS neglected tropical diseases</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e0004356</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0004356">10.1371/journal.pntd.0004356</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26751972">26751972</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Schistosoma mansoni</i> polo-like kinase 1: A mitotic kinase with key functions in parasite reproduction</span><span class="source" tagx="source" title="source">International journal for parasitology</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">1075</span><span class="lpage" tagx="lpage" title="lpage">1086</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijpara.2010.03.002">10.1016/j.ijpara.2010.03.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20350550">20350550</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fairweather</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Triclabendazole progress report, 2005-2009: an advancement of learning?</span><span class="source" tagx="source" title="source">Journal of helminthology</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">139</span><span class="lpage" tagx="lpage" title="lpage">150</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/s0022149x09321173">10.1017/s0022149x09321173</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19366485">19366485</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelley</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Current threat of triclabendazole resistance in <i>Fasciola hepatica</i></span><span class="source" tagx="source" title="source">Trends in parasitology</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">458</span><span class="lpage" tagx="lpage" title="lpage">469</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pt.2016.03.002">10.1016/j.pt.2016.03.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27049013">27049013</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A gene expression atlas of adult <i>Schistosoma mansoni</i> and their gonads</span><span class="source" tagx="source" title="source">Scientific data</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">170118</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/sdata.2017.118">10.1038/sdata.2017.118</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28829433">28829433</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radio</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pleiotropic alterations in gene expression in Latin American <i>Fasciola hepatica</i> isolates with different susceptibility to drugs</span><span class="source" tagx="source" title="source">Parasites &amp;amp; vectors</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">56</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13071-017-2553-2">10.1186/s13071-017-2553-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29368659">29368659</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cwiklinski</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Infection by the helminth parasite <i>Fasciola hepatica</i> requires rapid regulation of metabolic</span><span class="source" tagx="source" title="source">virulence, and invasive factors to adjust to its mammalian host. Molecular &amp;amp; cellular proteomics: MCP</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">792</span><span class="lpage" tagx="lpage" title="lpage">809</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/mcp.RA117.000445">10.1074/mcp.RA117.000445</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29321187">29321187</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rinaldi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Development of functional genomic tools in trematodes: RNA interference and luciferase reporter gene activity in <i>Fasciola hepatica</i></span><span class="source" tagx="source" title="source">PLoS neglected tropical diseases</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">e260</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0000260">10.1371/journal.pntd.0000260</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18612418">18612418</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Xie, F., Xiao, P., Chen, D., Xu, L. &amp;amp; Zhang, B. miRDeepFinder: a miRNA analysis tool for deep sequencing of plant small RNAs. <i>Plant molecular biology</i>, 10.1007/s11103-012-9885-2 (2012).</span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCammick</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Calmodulin disruption impacts growth and motility in juvenile liver fluke</span><span class="source" tagx="source" title="source">Parasites &amp;amp; vectors</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">46</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13071-016-1324-9">10.1186/s13071-016-1324-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26817678">26817678</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trudgett</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoey</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fairweather</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Triclabendazole-resistant <i>Fasciola hepatica</i>: beta-tubulin and response to <i>in vitro</i> treatment with triclabendazole</span><span class="source" tagx="source" title="source">Parasitology</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">124</span><span class="fpage" tagx="fpage" title="fpage">325</span><span class="lpage" tagx="lpage" title="lpage">338</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/s003118200100124x">10.1017/s003118200100124x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11922434">11922434</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giuliani</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Computationally-guided drug repurposing enables the discovery of kinase targets and inhibitors as new schistosomicidal agents</span><span class="source" tagx="source" title="source">PLoS computational biology</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">e1006515</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1006515">10.1371/journal.pcbi.1006515</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30346968">30346968</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hemer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brehm</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>In vitro</i> efficacy of the anticancer drug imatinib on <i>Echinococcus multilocularis</i> larvae</span><span class="source" tagx="source" title="source">International journal of antimicrobial agents</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">458</span><span class="lpage" tagx="lpage" title="lpage">462</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijantimicag.2012.07.007">10.1016/j.ijantimicag.2012.07.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22947125">22947125</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Connell</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennuru</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dolan</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nutman</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Targeting filarial Abl-like kinases: orally available, food and drug administration-approved tyrosine kinase inhibitors are microfilaricidal and macrofilaricidal</span><span class="source" tagx="source" title="source">The Journal of infectious diseases</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">212</span><span class="fpage" tagx="fpage" title="fpage">684</span><span class="lpage" tagx="lpage" title="lpage">693</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiv065">10.1093/infdis/jiv065</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25657255">25657255</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schubert</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Targeting <i>Echinococcus multilocularis</i> stem cells by inhibition of the Polo-like kinase EmPlk1</span><span class="source" tagx="source" title="source">PLoS neglected tropical diseases</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e2870</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0002870">10.1371/journal.pntd.0002870</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24901228">24901228</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ressurreicao</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Protein kinase C and extracellular signal-regulated kinase regulate movement, attachment, pairing and egg release in <i>Schistosoma mansoni</i></span><span class="source" tagx="source" title="source">PLoS neglected tropical diseases</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e2924</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0002924">10.1371/journal.pntd.0002924</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24921927">24921927</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Compound library screening identified Akt/PKB kinase pathway inhibitors as potential key molecules for the development of new chemotherapeutics against schistosomiasis. <i>International journal for parasitology</i></span><span class="source" tagx="source" title="source">Drugs and drug resistance</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">256</span><span class="lpage" tagx="lpage" title="lpage">266</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijpddr.2014.09.004">10.1016/j.ijpddr.2014.09.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25516836">25516836</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">ZM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation of DNA damage-induced apoptosis by the c-Abl tyrosine kinase</span><span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">1437</span><span class="lpage" tagx="lpage" title="lpage">1440</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.94.4.1437">10.1073/pnas.94.4.1437</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9037071">9037071</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coffer</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodgett</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular cloning and characterisation of a novel putative protein-serine kinase related to the cAMP-dependent and protein kinase C families</span><span class="source" tagx="source" title="source">European journal of biochemistry</span><span class="year" tagx="year" title="year">1991</span><span class="volume" tagx="volume" title="volume">201</span><span class="fpage" tagx="fpage" title="fpage">475</span><span class="lpage" tagx="lpage" title="lpage">481</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1432-1033.1991.tb16305.x">10.1111/j.1432-1033.1991.tb16305.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1718748">1718748</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selbie</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmitz-Peiffer</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biden</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular cloning and characterization of PKC iota, an atypical isoform of protein kinase C derived from insulin-secreting cells</span><span class="source" tagx="source" title="source">The Journal of biological chemistry</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">268</span><span class="fpage" tagx="fpage" title="fpage">24296</span><span class="lpage" tagx="lpage" title="lpage">24302</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8226978">8226978</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lane</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nigg</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antibody microinjection reveals an essential role for human polo-like kinase 1 (Plk1) in the functional maturation of mitotic centrosomes</span><span class="source" tagx="source" title="source">The Journal of cell biology</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">135</span><span class="fpage" tagx="fpage" title="fpage">1701</span><span class="lpage" tagx="lpage" title="lpage">1713</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.135.6.1701">10.1083/jcb.135.6.1701</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8991084">8991084</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bustin</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benes</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nolan</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfaffl</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Quantitative real-time RT-PCR–a perspective</span><span class="source" tagx="source" title="source">Journal of molecular endocrinology</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">597</span><span class="lpage" tagx="lpage" title="lpage">601</span><span class="pub-id"><a href="https://dx.doi.org/10.1677/jme.1.01755">10.1677/jme.1.01755</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15956331">15956331</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahnel</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Gonad RNA-specific qRT-PCR analyses identify genes with potential functions in schistosome reproduction such as SmFz1 and SmFGFRs</span><span class="source" tagx="source" title="source">Frontiers in genetics</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">170</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fgene.2014.00170">10.3389/fgene.2014.00170</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24959172">24959172</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeLorenzo</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moon</span><span class="given-names" tagx="given-names" title="given-names">TS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Selection of stable reference genes for RT-qPCR in <i>Rhodococcus opacus</i> PD630</span><span class="source" tagx="source" title="source">Scientific reports</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">6019</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-24486-w">10.1038/s41598-018-24486-w</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29662144">29662144</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fanarraga</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aloria</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zabala</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tubulin folding cofactor D is a microtubule destabilizing protein</span><span class="source" tagx="source" title="source">FEBS letters</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">470</span><span class="fpage" tagx="fpage" title="fpage">93</span><span class="lpage" tagx="lpage" title="lpage">95</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/s0014-5793(00)01293-x">10.1016/s0014-5793(00)01293-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10722852">10722852</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cowan</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of TBCD and its regulatory interactor Arl2 on tubulin and microtubule integrity</span><span class="source" tagx="source" title="source">Cytoskeleton (Hoboken, N.J.)</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">706</span><span class="lpage" tagx="lpage" title="lpage">714</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cm.20480">10.1002/cm.20480</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cerini</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A component of the multisynthetase complex is a multifunctional aminoacyl-tRNA synthetase</span><span class="source" tagx="source" title="source">The EMBO journal</span><span class="year" tagx="year" title="year">1991</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">4267</span><span class="lpage" tagx="lpage" title="lpage">4277</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/j.1460-2075.1991.tb05005.x">10.1002/j.1460-2075.1991.tb05005.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1756734">1756734</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rivett</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mason</span><span class="given-names" tagx="given-names" title="given-names">GG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murray</span><span class="given-names" tagx="given-names" title="given-names">RZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reidlinger</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation of proteasome structure and function</span><span class="source" tagx="source" title="source">Molecular biology reports</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">99</span><span class="lpage" tagx="lpage" title="lpage">102</span><span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1006814306401">10.1023/A:1006814306401</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9228289">9228289</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wesolowska</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Immune responses in rats and sheep induced by a DNA vaccine containing the phosphoglycerate kinase gene of <i>Fasciola hepatica</i> and liver fluke infection</span><span class="source" tagx="source" title="source">Acta parasitologica</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">212</span><span class="lpage" tagx="lpage" title="lpage">220</span><span class="pub-id"><a href="https://dx.doi.org/10.1515/ap-2016-0030">10.1515/ap-2016-0030</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27078643">27078643</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shoriki</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Novel methods for the molecular discrimination of <i>Fasciola</i> spp. on the basis of nuclear protein-coding genes</span><span class="source" tagx="source" title="source">Parasitology international</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">180</span><span class="lpage" tagx="lpage" title="lpage">183</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.parint.2015.12.002">10.1016/j.parint.2015.12.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26680160">26680160</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mansour</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mansour</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of serotonin (5-hydroxytryptamine) and adenosine 3’,5’-phosphate on phosphofructokinase from the liver fluke <i>Fasciola hepatica</i></span><span class="source" tagx="source" title="source">The Journal of biological chemistry</span><span class="year" tagx="year" title="year">1962</span><span class="volume" tagx="volume" title="volume">237</span><span class="fpage" tagx="fpage" title="fpage">629</span><span class="lpage" tagx="lpage" title="lpage">634</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14469439">14469439</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fairweather</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Threadgold</span><span class="given-names" tagx="given-names" title="given-names">LT</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Fasciola hepatica</i>: motility response to fasciolicides <i>in vitro</i></span><span class="source" tagx="source" title="source">Experimental parasitology</span><span class="year" tagx="year" title="year">1984</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="lpage" tagx="lpage" title="lpage">224</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0014-4894(84)90094-8">10.1016/0014-4894(84)90094-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6723893">6723893</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulman</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valentino</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cifelli</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ostlind</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dose-dependent pharmacokinetics and efficacy of MK-401 against old, and young-mature infections of <i>Fasciola hepatica</i> in the rat</span><span class="source" tagx="source" title="source">The Journal of parasitology</span><span class="year" tagx="year" title="year">1982</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">603</span><span class="lpage" tagx="lpage" title="lpage">608</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/3280917">10.2307/3280917</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7119988">7119988</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Untergasser</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Primer3–new capabilities and interfaces</span><span class="source" tagx="source" title="source">Nucleic acids research</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">e115</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gks596">10.1093/nar/gks596</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22730293">22730293</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dorak, M. <i>Real-time PCR</i>. 58–62 (Oxford: Taylor &amp;amp; Francis, 2008).</span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leutner</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Combinatory microarray and SuperSAGE analyses identify pairing-dependently transcribed genes in <i>Schistosoma mansoni</i> males, including follistatin</span><span class="source" tagx="source" title="source">PLoS neglected tropical diseases</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">e2532</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0002532">10.1371/journal.pntd.0002532</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24244773">24244773</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandesompele</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes</span><span class="source" tagx="source" title="source">Genome biology</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">Research0034</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2002-3-7-research0034">10.1186/gb-2002-3-7-research0034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12184808">12184808</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfaffl</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tichopad</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prgomet</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neuvians</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Determination of stable housekeeping genes, differentially regulated target genes and sample integrity: BestKeeper–Excel-based tool using pair-wise correlations</span><span class="source" tagx="source" title="source">Biotechnology letters</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">509</span><span class="lpage" tagx="lpage" title="lpage">515</span><span class="pub-id"><a href="https://dx.doi.org/10.1023/B:BILE.0000019559.84305.47">10.1023/B:BILE.0000019559.84305.47</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15127793">15127793</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orntoft</span><span class="given-names" tagx="given-names" title="given-names">TF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Normalization of real-time quantitative reverse transcription-PCR data: a model-based variance estimation approach to identify genes suited for normalization, applied to bladder and colon cancer data sets</span><span class="source" tagx="source" title="source">Cancer research</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">5245</span><span class="lpage" tagx="lpage" title="lpage">5250</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.Can-04-0496">10.1158/0008-5472.Can-04-0496</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15289330">15289330</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silver</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Best</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thein</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Selection of housekeeping genes for gene expression studies in human reticulocytes using real-time PCR</span><span class="source" tagx="source" title="source">BMC molecular biology</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2199-7-33">10.1186/1471-2199-7-33</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17026756">17026756</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howe</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolt</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shafie</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kersey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berriman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">WormBase ParaSite - a comprehensive resource for helminth genomics</span><span class="source" tagx="source" title="source">Molecular and biochemical parasitology</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">215</span><span class="fpage" tagx="fpage" title="fpage">2</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molbiopara.2016.11.005">10.1016/j.molbiopara.2016.11.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27899279">27899279</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letunic</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doerks</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bork</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">SMART: recent updates, new developments and status in 2015</span><span class="source" tagx="source" title="source">Nucleic acids research</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">D257</span><span class="lpage" tagx="lpage" title="lpage">260</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gku949">10.1093/nar/gku949</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25300481">25300481</a></span></span></li> </ul> </div> </div>  </body></html>