<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">BMC Biotechnol</span><span class="iso-abbrev" title="iso-abbrev">BMC Biotechnol</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">BMC Biotechnology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1472-6750</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7523293">7523293</a></span><span class="publisher-id" title="publisher-id">643</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12896-020-00643-w">10.1186/s12896-020-00643-w</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Novel kinase platform for the validation of the anti-tubercular activities of <i>Pelargonium sidoides</i> (Geraniaceae)</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lukman</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Odeyemi</span><span class="given-names" tagx="given-names" title="given-names">S. W.</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roth</span><span class="given-names" tagx="given-names" title="given-names">R. L.</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mbabala</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tshililo</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vlok</span><span class="given-names" tagx="given-names" title="given-names">N. M.</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dewar</span><span class="given-names" tagx="given-names" title="given-names">M. J. B.</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1590-7230</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenyon</span><span class="given-names" tagx="given-names" title="given-names">C. P.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">kenyon@sun.ac.za</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.412801.e</span><span class="institution-id" title="institution-id">0000 0004 0610 3238</span><span class="institution" title="institution">Department of Life and Consumer Sciences, College of Agriculture and Environmental Sciences, </span><span class="institution" title="institution">University of South Africa, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.7327.1</span><span class="institution-id" title="institution-id">0000 0004 0607 1766</span><span class="institution" title="institution">Council for Scientific and Industrial Research, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.11956.3a</span><span class="institution-id" title="institution-id">0000 0001 2214 904X</span><span class="institution" title="institution">DST-NRF Centre of Excellence for Biomedical Tuberculosis Research, South African Medical Research Council Centre for Tuberculosis Research, Division of Molecular Biology and Human Genetics, </span><span class="institution" title="institution">Faculty of Medicine and Health Sciences, Stellenbosch University, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.11956.3a</span><span class="institution-id" title="institution-id">0000 0001 2214 904X</span><span class="institution" title="institution">Proteomics Spectrometry Unit, Central Analytical Facility, </span><span class="institution" title="institution">University of Stellenbosch, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-9-9</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">20</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">50</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-15</span><span class="accepted" title="accepted">accepted: 2020-8-28</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1"><i>Pelargonium sidoides</i> is an important traditional medicine in South Africa with a well-defined history of both traditional and documented use of an aqueous-ethanolic formulation of the roots of <i>P. sidoides</i> (EPs 7630), which is successfully employed for the treatment of respiratory tract infections. There is also historical evidence of use in the treatment of tuberculosis. The aim of this study was to develop a platform of <i>Mycobacterium tuberculosis</i> (Mtb) kinase enzymes that may be used for the identification of therapeutically relevant ethnobotanical extracts that will allow drug target identification, as well as the subsequent isolation of the active compounds.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par2">Mtb kinases, Nucleoside diphosphokinase, Homoserine kinase, Acetate kinase, Glycerol kinase, Thiamine monophosphate kinase, Ribokinase, Aspartokinase and Shikimate kinase were cloned, produced in <i>Escherichia coli</i> and characterized. HPLC-based assays were used to determine the enzyme activities and subsequently the inhibitory potentials of varying concentrations of a <i>P. sidoides</i> extract against the produced enzymes. The enzyme activity assays indicated that these enzymes were active at low ATP concentrations. The 50% inhibitory concentration (IC<sub>50</sub>) of an aqueous root extract of <i>P. sidoides</i> against the kinases indicated SK has an IC<sub>50</sub> of 1.2â€‰Î¼g/ml and GK 1.4â€‰Î¼g/ml. These enzyme targets were further assessed for compound identification from the <i>P. sidoides</i> literature.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par3">This study suggests <i>P. sidoides</i> is potentially a source of anti-tubercular compounds and the Mtb kinase platform has significant potential as a tool for the subsequent screening of <i>P. sidoides</i> extracts and plant extracts in general, for compound identification and elaboration by selected extract target inhibitor profiling.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd"><i>Mycobacterium tuberculosis</i></div> <div class="kwd" title="kwd">
Kinases</div> <div class="kwd" title="kwd">
Anti-tubercular</div> <div class="kwd" title="kwd">
Target identification</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">MRC (South Africa)</span></div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">University of South Africa</span></div> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par14">The prevalence in tuberculosis (TB), together with the recent increase in the incidence of multidrug-resistance (MDR) cases, has led to the search for new drug targets and new drugs that are effective against Mtb. TB is often a fatal disease, and one that poses a global threat to human health [<a href="#CR5">5</a>, <a href="#CR6">6</a>]. Globally, infection associated with TB is second only to HIV/AIDS as the greatest killer due to a single initiating infectious agent [<a href="#CR25">25</a>]. Out of the 9 million people infected with TB in a year, 3 million are left untreated, acting as a reservoir for further infection. Many of these 3 million untreated cases live in poverty with minimal access to healthcare. Over 95% of TB cases and deaths are in developing countries such as South Africa, often where the percentage of HIV/AIDS co-infection is high, and therefore occurs in individuals with compromised immune systems (WHO, 2020) [<a href="#CR25">25</a>].</p> <p id="Par15">TB is caused by various strains of <i>Mycobacterium tuberculosis</i>, commonly affects the lungs, and is transferrable from person to person through the air. Mtb is an intra-cellular parasite normally residing in the human macrophages, where its survival and growth depends on complex networks of the attenuation of macrophage activity by the Mtb bacilli. Mtb has not only developed a number of mechanisms to evade onslaught from such host macrophage immune responses as reactive oxygen and nitrogen species, but it has also evolved a metabolism that has allowed it to survive in this very specialized niche environment [<a href="#CR27">27</a>]. It has become evident that one of the mechanisms to arrest the progression of Mtb into full-blown infection is a multi-targeted approach [<a href="#CR23">23</a>]. The key question which arises is, â€œCan a single plant extract be used to screen multiple targets as a means of identifying novel synergistic anti-infective properties?â€� Medicinal plants have been used for the treatment of several diseases and the plant extract identified for analysis was that from <i>Pelargonium sidoides,</i> as this plant has a 200â€‰year documented history of ethnobotanical use in the treatment of tuberculosis and other infections [<a href="#CR1">1</a>, <a href="#CR2">2</a>, <a href="#CR12">12</a>, <a href="#CR13">13</a>, <a href="#CR15">15</a>, <a href="#CR18">18</a>, <a href="#CR20">20</a>]. This includes well documented reports in literatures that suggest the medicinal properties and efficacy of <i>P. sidoides</i> against <i>M. tuberculosis</i> and other bacterial infections [<a href="#CR16">16</a>, <a href="#CR9">9</a>, <a href="#CR17">17</a>, <a href="#CR22">22</a>]<i>.</i> The mechanisms and targets of this anti-tubercular activity are, however, not defined. The aqueous-ethanolic extract of <i>P. sidoides</i> (EPÂ® 7630) roots have been used to treat bacterial infections, as well as to induce the production of the pro-inflammatory cytokines TNF-Î± and IL-6 in human blood human immune cells, alleviating symptoms associated with acute bronchitis [<a href="#CR12">12</a>, <a href="#CR13">13</a>]. This extract is licensed in Germany as herbal medicine for the treatment of upper respiratory tract infections [<a href="#CR26">26</a>].</p> <p id="Par16">This investigation was therefore set up to target a functionally diverse range of Mtb kinase enzymes as a mechanism of identifying potential Mtb drug targets using the complexity found in medicinal plant extracts as the source of chemical diversity. It was also decided to select a range of Mtb kinases which, where possible, do not occur within mammalian biochemistry. Kinases have been classified into 25 families of homologous proteins, with the families assembled into 12 fold-groups based on the similarity of their structural folds [<a href="#CR3">3</a>, <a href="#CR4">4</a>]. It has further been demonstrated that each of the 12 fold-groups has a distinct phosphoryl transfer mechanism [<a href="#CR11">11</a>], and it was therefore decided to select the kinase enzyme targets from 6 of the 12 fold-groups, thereby representing 6 distinct phosphoryl transfer mechanisms. The six identified phosphoryl transfer mechanisms all have distinct ATP binding motifs. The selection of these kinases should therefore identify different classesâ€™ compounds capable of binding ATP. The enzymes selected are crucial for the metabolism and survival of Mtb. As relatively large number of enzymes was to be comparatively simultaneously assessed it was envisaged to keep the enzyme purification and assays as simple as possible.</p> <p id="Par17">The eight specific Mtb kinases targeted are Nucleoside diphosphokinase (NDK, EC 2.7.4.6), Homoserine kinase (HSK, EC 2.7.1.39), Acetate kinase (AK, EC 2.7.2.1), Glycerol kinase (GK, EC 2.7.1.30), Thiamine monophosphate kinase (ThiL, EC 2.7.4.1), Ribokinase (RBKS, EC 2.7.1.15), Aspartokinase (AsK, EC 2.7.2.4), and Shikimate kinase (SK, EC 2.7.1.71) [<a href="#CR7">7</a>, <a href="#CR10">10</a>, <a href="#CR14">14</a>, <a href="#CR21">21</a>]. The kinases were expressed in <i>Escherichia coli,</i> purified and the activity determined through HPLC-based assays. The inhibitory properties of a <i>P. sidoides</i> extract were then investigated against the characterized kinases.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par18">The enzymes were expressed in <i>E. coli</i> with His-tags to facilitate the purification of these enzymes, and their functionality was validated by determining their enzyme activity before carrying out the <i>P. sidoides</i> inhibitory experiments.</p> <div class="cloning,expressionandpurificationofenzymes" title="sec"> <div class="title" tagx="title" title="title">
Cloning, expression and purification of enzymes</div> <p id="Par19">The Mtb kinase genes were PCR-amplified from <i>M. tuberculosis</i> H37Rv genomic DNA, yielding amplicons of 415, 952, 1162, 1558, 1006, 919, 1268 and 520â€‰bp for <i>ndkA</i> (nucleoside diphosphate kinase)<i>, ThrB</i> (homoserine kinase)<i>, ackA</i> (acetate kinase)<i>, glpK</i> (glycerol kinase)<i>, thil</i> (thiamine monophosphate kinase)<i>, rbks</i> (ribokinase)<i>, ask</i> alpha and <i>ask</i> beta (aspartokinase) and <i>aroK</i> gene (shikimate kinase), respectively. These were subcloned into the selected plasmids and confirmed via sequencing.</p> <p id="Par20">All enzymes were expressed in <i>E. coli</i> BL21 (DE3) subsequent to IPTG induction. Following nickel-affinity purification using either native or denaturing means, the proteins was verified by SDS-PAGE as outlined in Fig.Â <a href="#Fig1">1</a> and in AdditionalÂ fileÂ <a href="#MOESM1">1</a> for the detailed break-down of the fractions obtained. Acetate kinase (AK) and glycerol kinase (GK) were both well expressed however the final eluate yielded lower levels of protein. They were both however still used in the screening as sufficiently high levels of enzyme activity were obtained. It was decided to keep these enzymes in the screen panel as one of the primary aims of setting up the screen panel was to assess if the panel may be used to identify compound target selectivity from mixtures such as plant extracts. It is envisaged that a higher fidelity secondary screen will be set up, using purer enzyme, once the target has been identified. The secondary screen will be used for compound identification. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>SDS-PAGE gels of the Mtb his-tagged kinases purified from <i>E. coli</i> BL21 (DE3). M represents the molecular mass marker (PageRulerâ"¢ Plus Pre-stained Protein Ladder, Thermo Scientific, USA) with the sizes of the bands indicated to the left of the gels in kDa. <b>a</b> NDK, 14,4â€‰kDa. <b>b</b> HSK, 33.4â€‰kDa. <b>c</b> AK, 43.7â€‰kDa. <b>d</b> GK, 58.2â€‰kDa. <b>e</b> ThiL, 36.4â€‰kDa. <b>f</b> RBKS, 32.3â€‰kDa. <b>g</b> AsK alpha, 44.6â€‰kDa and AsK beta, 18â€‰kDa. <b>h</b> SK, 20.7â€‰kDa</p>   </div> <p /> </div> <div class="enzymeactivityassays" title="sec"> <div class="title" tagx="title" title="title">
Enzyme activity assays</div> <p id="Par21">The effect of the ATP concentration on the steady state specific activity of the <i>M. tuberculosis</i> kinases was expressed over a concentration gradient of ATP (Fig.Â <a href="#Fig2">2</a>). As the enzymes were recombinantly expressed in <i>E. coli</i>, the specific activity of the individual enzymes referred to throughout the document is the recombinant specific activity as the enzymes were not obtained from their native host. The best-fit to the data was obtained for the specified kinetic model using the non-linear regression algorithms as outlined using the GraphPad PrismÂ® 5 software. The variation in the maximum specific enzyme activity for each enzyme was vast, ranging from approximately 0.14â€‰nM/minute/nM protein for AsK to in excess of 3000â€‰nM/minute/nM protein for SK, indicative of the great variation in binding affinity for ATP of the selected enzymes. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>The effect of the ATP concentration on the specific enzyme activities of the purified kinase enzymes</p>   </div> <p /> </div> <div class="p.sidoidesinhibitoryassay" title="sec"> <div class="title" tagx="title" title="title"><i>P. sidoides</i> inhibitory assay</div> <p id="Par22">The inhibitory activities of various dilutions of <i>P. sidoides</i> extracts were concentration-dependent in all the kinases except HSK, RBKS and AsK (Fig.Â <a href="#Fig3">3</a>). No inhibition was observed at 1â€‰Ã—â€‰10<sup>âˆ’â€‰6</sup>â€‰mg/ml of <i>P. sidoides</i> on ThiL and RBKS kinases. The most susceptible enzymes to <i>P. sidoides</i> extract were SK and GK with the lowest IC<sub>50</sub> values of 1.17â€‰mg/ml and 1.4â€‰mg/ml, respectively, when compared to other kinases (TableÂ <a href="#Tab1">1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>The inhibitory activities of P. sidoides against the purified kinases</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> <p>Kinases and their respective IC50 values derived from the dose-response curves</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Kinase</th> <th>IC<sub>50</sub> value (mg/ml)</th> </tr> </thead> <tbody> <tr> <td>NDK</td> <td>9.28</td> </tr> <tr> <td>HSK</td> <td>13.72</td> </tr> <tr> <td>AK</td> <td>212.3</td> </tr> <tr> <td>GK</td> <td>1.40</td> </tr> <tr> <td>ThiL</td> <td>16.01</td> </tr> <tr> <td>RBKS</td> <td>73.42</td> </tr> <tr> <td>AsK</td> <td>31.16</td> </tr> <tr> <td>SK</td> <td>1.17</td> </tr> </tbody> </table> </div> <p /> <p id="Par23">The validation for presence of the enzymes to complement the enzyme functionality data was demonstrated using peptide mass spectroscopy (TableÂ <a href="#Tab2">2</a>). The data and the diagnostic mass fragmentation patterns for selected peptides are outlined in Additional file <a href="#MOESM1">1</a>. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>Kinases and their respective MS validation</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Kinase</th> <th>Peptide</th> <th>Probability (%)</th> <th>Unique Peptides</th> <th>Unique Spectra</th> </tr> </thead> <tbody> <tr> <td>NDK</td> <td>(R)KGLTIAALQLR(T)</td> <td>100</td> <td>11</td> <td>18</td> </tr> <tr> <td>HSK</td> <td>(K)GFAVTELTVGEAVR(W)</td> <td>100</td> <td>65</td> <td>100</td> </tr> <tr> <td>AK</td> <td>(K)MLAEDGIDLQTcGLVAVGHR</td> <td>99.7</td> <td>34</td> <td>47</td> </tr> <tr> <td>GK</td> <td>(R)DQLGIISGAAQSEALAR(Q)</td> <td>99.7</td> <td>55</td> <td>68</td> </tr> <tr> <td>ThiL</td> <td>(R)TVVSTDMLVQDSHFR(L)</td> <td>100</td> <td>3</td> <td>3</td> </tr> <tr> <td>RBKS</td> <td>ND</td> <td /> <td /> <td /> </tr> <tr> <td>AsK</td> <td>(R)cVEYARRHnIP(V)</td> <td>99.7</td> <td>3</td> <td>3</td> </tr> <tr> <td>SK</td> <td>ND</td> <td /> <td /> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><i>ND</i> not detected</p> </div>  <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par24">A range of kinases was selected for screening, based on representing some of the 12 fold-groups that have distinct phosphoryl transfer mechanisms, with a total of 6 distinct phosphoryl transfer mechanisms represented in the 8 enzymes used in this study [<a href="#CR11">11</a>]. The purification of the proteins was challenging, but a variety of different techniques were investigated in order to acquire sufficient amounts of relatively pure protein (as estimated by PAGE) to run all assays simultaneously as part of a preliminary medicinal plant extract screen. The presence of each kinase was demonstrated by enzyme activity, SDS-PAGE and/or MS analysis. The competitive inhibition of the kinase reaction may manifest either as inhibiting the binding of ATP or by the inhibition of the binding of the enzyme substrate that is to be phosphorylated. As these enzymes were selected based on the fact that they all have distinct phosphoryl transfer reactions as well as distinct substrates, it was envisaged that the plant extract may demonstrate a significant variation on the IC<sub>50</sub>s obtained. This was found to be the case, with SK and GK having IC<sub>50</sub> values of 1.17â€‰mg/ml and 1.4â€‰mg/ml, respectively, with all the other enzymes having IC<sub>50</sub>s of at least one order of magnitude higher. Clearly, the kinases selected could be used in a primary selection to identify targets for a plant extract. The selected enzymes could then be used in a more stringent secondary screen to identify the active compounds. What is significant is one of the major active ingredients of <i>P. sidoides</i> is gallic acid, which is a shikimic acid mimic (the substrate of SK) (Fig.Â <a href="#Fig4">4</a>). Gallic acid and a range of <i>O</i>-galloylated compounds have been demonstrated to be present in the extracts of <i>P. sidoides</i> [<a href="#CR12">12</a>]. SK has been identified as being a potential target to develop antimicrobial agents for Mtb [<a href="#CR8">8</a>, <a href="#CR19">19</a>]. An associated species of <i>Pelargonium</i> used in South Africa for medicinal purposes is <i>Pelargonium reniforme</i>. <i>P. reniforme</i> produces <i>O</i>-galloylated glycerol (glycerol-1-gallate) which could be a potential bi-functional inhibitor of both SK and GK. If only low levels of glycerol-1-gallate are synthesized in <i>P</i>. <i>sidoides</i>, however if the binding constants of glycerol-1-gallate for SK and GK is high enough inhibition will still occur and probably synergistically. As the enzymes selected are all kinase enzymes it is realistic to believe that one of the binding mechanisms is in the kinase ATP binding site. These results clearly demonstrate that this platform of Mtb kinase enzymes can be used as a primary selection strategy for the identification of active ingredients in plant extracts that allows for the stratification of the inhibition of Mtb kinase enzymes and the validation of the extracts potential medicinal properties. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Structural similarity between gallic acid and a few o-galloyl derivatives which are components of P. sidoides and shikimic acid, the substrate to SK</p>   </div> <p /> <p id="Par25">The traditional use of the plant root of <i>P. sidoides</i> was for a wide range of ailments including tuberculosis (for well researched review on the ethnobotanical and medicinal use of <i>P. sidoides</i> and other Pelargonium specices see reference [<a href="#CR2">2</a>];). <i>P. sidoides</i> forms the basis of Charles Henry Stevens secret cure for tuberculosis, â€œStevens Cureâ€� and was called â€œUmckaloaboâ€�. Modern aqueous-ethanolic formulation of the roots of P. sidoides (EPÂ® 7630) has been successfully employed for the treatment of ear, nose and throat disorders as well as respiratory tract infections [<a href="#CR12">12</a>, <a href="#CR13">13</a>].</p> <p id="Par26">The data exhibited only moderate direct antibacterial capabilities against a spectrum of Gram-positive and Gram-negative bacteria, although convincing data was provided in support for the improvement of immune functions at various levels, hence, validating the medicinal uses of EPÂ® 7630. The concentrations and nature of the active compounds in the extract are unknown. This data however, provided support to validate the medicinal use of <i>P. sidoides</i>. However, the remedial effects are not yet associated with mechanistic structure and function analyses and therefore further investigations are required in order to study the functional relationships between the <i>O</i>-gallolylated compounds and SK and GK. Phytochemical studies show the presence of a large number of other secondary metabolites in the plant, such as tannins, coumarins, phenolic acids, phenylpropanoid derivatives and other chemical constituents [<a href="#CR12">12</a>, <a href="#CR17">17</a>]. This identified platform of Mtb kinases could serve as a screen to allow for mechanistic studies to be carried out on ethnobotanical plant extracts. The traditional use of <i>P. sidoides</i> and the present enzyme results indicate the potential use of this kinase platform to directly relate traditional use to target mechanistic investigations thereby identifying the potential drug target. These data should eventually contribute to evidence-based traditional medicines.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par27">In conclusion, selected Mtb kinases were successfully expressed in <i>E. coli</i> and purified and validated by SDS-PAGE, enzyme activity and/or MS spectroscopy. The enzyme functionality was validated through the enzyme activity of the purified proteins, and the effect of a <i>P. sidoides</i> plant extract on their activities was determined. The most susceptible enzymes tested were SK and GK, with the lowest IC<sub>50</sub> values. This suggests that both SK and GK could be used as targets through which <i>P. sidoides</i> extracts could be characterized in terms of the specific chemistry of the inhibitors. As these two enzymes have different phosphoryl transfer mechanisms is it probable that different classes of compounds will be selected. The biosynthesis of the aromatic amino acids occurs via chorismate, the precursor to which is shikimate. As mammals do not have the biochemistry for the synthesis of chorismate or any of its intermediates, SK is a good validated target for Mtb. The human essential amino acids, tyrosine, tryptophan and phenylalanine are all synthesized using chorismate as the precursor. <i>P. sidoides</i> contains a broad range of O-galloylated compounds all of which are potential inhibitors of SK and in the case of glycerol-1-gallate, GK. Having identified the potential targets for <i>P. sidoides</i> inhibition SK and GK will therefore serve as a good screen for compound identification and validation from <i>P. sidoides</i> extracts.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="materials" title="sec"> <div class="title" tagx="title" title="title">
Materials</div> <p id="Par28">H37Rv genomic DNA was received from Professor Ian Wiid, University of Stellenbosch, South Africa. The Mtb <i>aroK</i> gene (encoding Shikimate Kinase) in pET15-b was received from the laboratory of Chris Abell, University of Cambridge. All other genes were cloned in-house as outlined in section 5.3. All PCR reagents were from Kapa Biosystems (KAPA HiFi for gene amplification and KAPA 2G Fast for screening), and cloning materials were purchased from Epicentre Technologies, USA (Fast-Linkâ"¢ DNA Ligation Kit) or Zymo Research, USA (Zyppyâ"¢ Plasmid Miniprep kit and Zymocleanâ"¢ Gel DNA Recovery Kit). Oligonucleotides for gene amplification were obtained from IDT Inc. (USA). All other chemicals used were at least analytical grade and were obtained from Sigma-Aldrich.</p> </div> <div class="plantmaterial" title="sec"> <div class="title" tagx="title" title="title">
Plant material</div> <p id="Par29">The <i>P. sidoides</i> fresh plant material was supplied by the Natural Plants and Agroprocessing (NPA) division of CSIR Biosciences through the CSIR Enterprise Creation for Development (ECD) division, Pretoria, South Africa. In an endeavor to limit the environmental destruction due to the uncontrolled wild harvesting of plants the ECD of the CSIR has facilitated the cultivation of a number of important ethnobotanical plants of known high usage, <i>P. sidoides</i> being one of them. The plants used were cultivated by Rodene Nursery, (ECD Sample number, ECD-MP-0252; Extract number, PEL-223-48448A). All plant taxonomy at the ECD was done in conjunction with South African National Biodiversity Institute (SANBI) which forms part of the The Plant List protocol.</p> <p id="Par30">The primary pre-processing post agricultural harvesting involved washing of the biomass (stalks and leaves cut into smaller pieces) prior to drying at 40-50â€‰Â°C, over 3â€"5â€‰days. The material was then milled using a hammer mill before carrying out the batch extraction process. The extraction was carried out in a glass jacketed percolation column. The biomass material was held in place within the column using mutton cloth. The percolation process was then carried out over 7â€‰h. The extraction solvent used was ethanol (43% v/v EtOH, 1.25â€‰L) and effective dried raw material loading (0.25â€‰kg). The solvent was re-circulated through the column reactor over the period of 7â€‰h. The solvent is pumped in at the top of the percolation column and allowed to diffuse through the column under gravitational force. The flow rate was estimated to be 36â€‰ml/min. The percolator column dimensions (out diameter 11â€‰cm, column lengthâ€‰~â€‰32â€‰cm). The recovered filtrate (ex. percolation) was then collected and ethanol removed using a Buchi evaporator (40-60â€‰Â°C, âˆ’â€‰85â€‰kPa over 1â€"2â€‰h). The extract yield was 16.917â€‰g (yield 6.76% m/m). The crude extract was stored at 4â€‰Â°C.</p> </div> <div class="cloningofthekinasegenesfromm.tuberculosis" title="sec"> <div class="title" tagx="title" title="title">
Cloning of the kinase genes from <i>M. tuberculosis</i></div> <p id="Par31">The genes that were cloned were <i>ndka</i> (nucleoside diphosphate kinase NDK; Rv2445c), <i>thrB</i> (homoserine kinase HSK; Rv1296), <i>ackA</i> (acetate kinase AK; Rv0409), glpK (glycerol kinase GK; Rv3696c), <i>thiL</i> (thiamine monophosphate kinase Thil; Rv2977c), <i>rbks</i> (ribokinase RBKS; Rv2436) and <i>ask</i> (aspartokinase AsK; Rv3709c). The <i>aroK</i> gene (encoding Shikimate Kinase) was obtained from the laboratory of Chris Abell, University of Cambridge. The genes were amplified from H37Rv genomic DNA using the oligonucleotide primers shown in TableÂ <a href="#Tab3">3</a>, and the PCR products subcloned into the selected pET vector (Novagen, Germany) using the applicable restriction enzymes. <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Forward and reverse primers used to amplify specific kinase genes. Note also preferred vector for each construct</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Kinase</th> <th>Gene and Rv identifier</th> <th>Forward and Reverse primers (5â€²â€‰â†’â€‰3â€²)</th> <th>Restri-ction Enzyme Recogni-tion site (underli-ned)</th> <th>Selected pET vector</th> </tr> </thead> <tbody> <tr> <td rowspan="4">NDK</td> <td rowspan="4"><i>ndkA</i> Rv2445c</td> <td><i>ndkA</i>-Fwd</td> <td /> <td>pET-16b</td> </tr> <tr> <td>GG<u>CATATG</u>ACCGAACGGACTCTGGTACTG</td> <td><i>Nde</i>I</td> <td /> </tr> <tr> <td colspan="3"><i>ndkA</i>-Rev</td> </tr> <tr> <td>GT<u>GGATCC</u>TTAGGCGCCGGGAAACCAG</td> <td><i>Bam</i>HI</td> <td /> </tr> <tr> <td rowspan="4">HSK</td> <td rowspan="4"><i>thrB</i> Rv1296</td> <td><i>thrB</i>-Fwd</td> <td /> <td>pET-20a</td> </tr> <tr> <td>GG<u>CATATG</u>GTGACTCAAGCATTG</td> <td><i>Nde</i>I</td> <td /> </tr> <tr> <td colspan="3"><i>thrB</i>-Rev</td> </tr> <tr> <td>GT<u>CTCGAG</u>ACCGGGAACTCTTACTG</td> <td><i>Xho</i>I</td> <td /> </tr> <tr> <td rowspan="4">AK</td> <td rowspan="4"> <p><i>ackA</i></p> <p>Rv0409</p> </td> <td><i>ackA</i>-Fwd</td> <td /> <td>pET-16b</td> </tr> <tr> <td>GG<u>CATATG</u>GAGTAGCACCGTGCTGGTGATCAA</td> <td><i>Nde</i>I</td> <td /> </tr> <tr> <td colspan="3"><i>ackA</i>-Rev</td> </tr> <tr> <td>GT<u>GGATCC</u>TTACGCTCGGCGTCCGCCCAG</td> <td><i>Bam</i>HI</td> <td /> </tr> <tr> <td rowspan="4">GK</td> <td rowspan="4"> <p><i>glpK</i></p> <p>Rv3696c</p> </td> <td><i>glpK</i>-Fwd</td> <td /> <td>pET-16b</td> </tr> <tr> <td>(GG<u>CATATG</u>TCCGACGCCATCCTAG</td> <td><i>Nde</i>I</td> <td /> </tr> <tr> <td colspan="3"><i>glpK</i>-Rev</td> </tr> <tr> <td>CAT<u>GTCGAC</u>TTAGGACACGTCAACCCAATCC</td> <td><i>Sal</i>I</td> <td /> </tr> <tr> <td rowspan="4">ThiL</td> <td rowspan="4"> <p><i>thil</i></p> <p>Rv2977c</p> </td> <td><i>thil</i>-Fwd</td> <td /> <td>pET-28a</td> </tr> <tr> <td>GGTA<u>CATATG</u>ACCACTAAAGATCACTC</td> <td><i>Nde</i>I</td> <td /> </tr> <tr> <td colspan="3"><i>thil</i>-Rev</td> </tr> <tr> <td>GAT<u>CTCGAG</u>TTACCCTAGCGAACCTTG</td> <td><i>Xho</i>I</td> <td /> </tr> <tr> <td rowspan="4">RBKS</td> <td rowspan="4"> <p><i>rbks</i></p> <p>Rv2436</p> </td> <td><i>rbks</i>-Fwd</td> <td /> <td>pET-28a</td> </tr> <tr> <td>GTA<u>CATATG</u>GCAAACGCCAGTGAG</td> <td><i>Nde</i>I</td> <td /> </tr> <tr> <td colspan="3"><i>rbks</i>-Rev</td> </tr> <tr> <td>GAT<u>CTCGAG</u>TTATGAACCGTTGTG</td> <td><i>Xho</i>I</td> <td /> </tr> <tr> <td rowspan="8">AsK<sup>a</sup></td> <td rowspan="8"> <p><i>ask</i></p> <p>Rv3709c</p> </td> <td><i>ask</i> alpha-Fwd</td> <td /> <td>pCDF-</td> </tr> <tr> <td>GATTA<u>CATATG</u>GCGCTCGTCGTGCAG</td> <td><i>Nde</i>I</td> <td>Duet-1</td> </tr> <tr> <td><i>ask</i> alpha-Rev</td> <td /> <td>(alpha)</td> </tr> <tr> <td>GAT<u>GTCGAC</u>TTACCGTCCCGTCCCCG-3â€™</td> <td><i>Sal</i>I</td> <td /> </tr> <tr> <td><i>ask</i> beta-Fwd</td> <td /> <td>pET-26a</td> </tr> <tr> <td>GAT<u>CATATG</u>GAAGACCCCATCCTGACCG</td> <td><i>Nde</i>I</td> <td>(beta)</td> </tr> <tr> <td><i>ask</i> beta-Rev</td> <td /> <td /> </tr> <tr> <td>GCCCCTGCCCTGCC<u>CAGCTG</u>TATG</td> <td><i>Sal</i>I</td> <td /> </tr> <tr> <td>SK</td> <td> <p><i>aroK</i></p> <p>Rv2539c</p> </td> <td>Primers were not designed as the <i>aroK</i> plasmid was received from the laboratory of Chris Abell, University of Cambridge</td> <td /> <td>pET-15b</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup> AsK consists of 2 hetero-monomers (alpha and beta), to be co-expressed</p> </div>  <p /> </div> <div class="expressionofmtbkinases" title="sec"> <div class="title" tagx="title" title="title">
Expression of Mtb kinases</div> <p id="Par32"><i>E. coli</i> BL21(DE3) (Novagen, USA) was used as production host. For AsK production, two co-transformed plasmids were used for co-expression of the alpha and beta monomers. The recombinant strains were cultivated in 250â€‰ml LB broth supplemented with the appropriate antibiotic(s) at 37â€‰Â°C with shaking at 200â€‰rpm and, at OD<sub>600</sub>â€‰~â€‰0.6, induced with 1â€‰mM Isopropyl Î²-D-1-thiogalactopyranoside (IPTG) and incubated overnight at 28â€‰Â°C. Production of SK was carried as described by Kenyon et al. [<a href="#CR10">10</a>]. The cells were harvested by centrifugation at 4080â€‰<i>g</i> for 10â€‰min at 4â€‰Â°C.</p> </div> <div class="purificationofmtbkinases" title="sec"> <div class="title" tagx="title" title="title">
Purification of Mtb kinases</div> <p id="Par33">The biomass pellets were resuspended in 20â€‰ml Binding Buffer (1â€‰M NaCl, 20â€‰mM Tris-HCl and 5â€‰mM Imidazole: pHâ€‰7.9), lysed by sonication and re-centrifuged to separate the soluble and insoluble fractions.</p> <p id="Par34">The soluble kinases NDK, ThiL, RBKS, AsK and SK were purified using the Profiniaâ"¢ Affinity Chromatography Protein Purification System (Bio-Rad, USA) with a 1â€‰ml column containing nickel-iminodiacetic acid (Ni-IDA) resin. The Standard Native conditions and protocols were followed according to the manufacturerâ€™s instructions. For AK and GK, MagReSynâ"¢ NTA (ReSynâ"¢ Biosciences. South Africa) was used for purification. The manufacturerâ€™s scaled-up protocol was followed. The eluates were dialysed overnight in each selected dialysis buffer (TableÂ <a href="#Tab4">4</a>). <div class="table-wrap_UNKNOWN" id="Tab4"><span class="label" tagx="label" title="label">Table 4</span> </div></p><p>Dialysis buffers used for each Mtb kinase</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Kinase</th> <th>Dialysis buffer</th> </tr> </thead> <tbody> <tr> <td rowspan="4">NDK</td> <td>50â€‰mM Tris pHâ€‰8.0</td> </tr> <tr> <td>100â€‰mM KCl</td> </tr> <tr> <td>1â€‰mM DTT</td> </tr> <tr> <td>1â€‰mM MgCl<sub>2</sub></td> </tr> <tr> <td rowspan="4">HSK</td> <td>50â€‰mM MOPS pHâ€‰8.0</td> </tr> <tr> <td>150â€‰mM NaCl</td> </tr> <tr> <td>1â€‰mM DTT</td> </tr> <tr> <td>10â€‰mM MgCl<sub>2</sub></td> </tr> <tr> <td rowspan="4">AK and GK</td> <td>50â€‰mM Tris pHâ€‰7.5</td> </tr> <tr> <td>150â€‰mM NaCl</td> </tr> <tr> <td>1â€‰mM DTT</td> </tr> <tr> <td>5â€‰mM MgCl<sub>2</sub></td> </tr> <tr> <td rowspan="4">ThiL and RBKS</td> <td>50â€‰mM MOPS pHâ€‰7.6</td> </tr> <tr> <td>150â€‰mM NaCl</td> </tr> <tr> <td>1â€‰mM DTT</td> </tr> <tr> <td>10â€‰mM MgCl<sub>2</sub></td> </tr> <tr> <td rowspan="4">AsK</td> <td>50â€‰mM Tris pHâ€‰6.0</td> </tr> <tr> <td>200â€‰mM NaCl</td> </tr> <tr> <td>1â€‰mM DTT</td> </tr> <tr> <td>10â€‰mM MgCl<sub>2</sub></td> </tr> <tr> <td rowspan="2">SK</td> <td>50â€‰mM Tris pHâ€‰7.5</td> </tr> <tr> <td>1â€‰M NaCl</td> </tr> </tbody> </table>  <p /> <p id="Par35">HSK was insoluble and was purified using an Ã"KTA Avant (GE Healthcare, USA). The biomass pellet was resuspended in 40â€‰ml Denaturation Solublisation Buffer (DSB; 50â€‰mM NaH<sub>2</sub>PO<sub>4</sub>, 300â€‰mM NaCl, 8â€‰M urea; pHâ€‰7.9) and incubated for 2â€‰h at 37â€‰Â°C with shaking at 50â€‰rpm. This was then lysed by sonication and centrifuged at 4080â€‰g for 10â€‰min at 4â€‰Â°C. The supernatant was clarified through a 0.45â€‰Î¼m syringe filter and loaded onto a 25â€‰ml bed volume Ni-NTA (nickel-nitrilotriacetic acid) column on the Ã"KTA Avant, pre-equilibrated with DSB. After loading, the column was washed with DSB, and HSK was refolded on the column using a linear gradient from 100% DSB to 100% of the urea-free Lysis Equilibration Buffer (50â€‰mM NaH<sub>2</sub>PO<sub>4</sub>, 300â€‰mM NaCl; pHâ€‰7.9) before being eluted off the resin using Elution Buffer (50â€‰mM NaH<sub>2</sub>PO<sub>4</sub>, 300â€‰mM NaCl, and 250â€‰mM Imidazole; pHâ€‰8.0). The eluate was dialysed overnight in HSKâ€™s selected dialysis buffer (Table <a href="#Tab4">4</a>), and concentrated five-fold through a Vivaspin 10â€‰kDa MWCO column (Sartorius).</p> <p id="Par36">The concentration of the proteins was determined using the QubitÂ® 2.0 Fluorometer (Life Technologies. USA) and Qubit Protein Assay Kit, as recommended by the manufacturer. A volume of 50â€‰Î¼l of all proteins except AK and GK, were snap-frozen in liquid nitrogen and stored at âˆ’â€‰80â€‰Â°C until assayed. For AK and GK, aliquots of 50â€‰Î¼l of the dialysed protein were mixed with 50% [v/v] glycerol before storage at âˆ’â€‰80â€‰Â°C.</p> </div> <div class="determinationofenzymeactivity" title="sec"> <div class="title" tagx="title" title="title">
Determination of enzyme activity</div> <p id="Par37">The kinase samples were thawed on ice prior to setting up the enzyme assays. The HPLC assay reactions were carried out in 100â€‰Î¼l volumes and incubated at 37â€‰Â°C. The assay was carried out as described by [<a href="#CR10">10</a>]. Briefly, the assay reactions consisted of 90â€‰Î¼l of the prepared reaction mixture (TableÂ <a href="#Tab5">5</a>) with either 10â€‰Î¼l of enzyme, prepared in triplicate or 10â€‰Î¼l distilled water, prepared in duplicate, which served as a control blank. A range of ATP concentrations was assayed, with ATP and MgCl<sub>2</sub> concentrations always kept at a 1:1 ratio [<a href="#CR24">24</a>]. After the pre-determined reaction time (Table <a href="#Tab5">5</a>), the reactions were stopped with 5% [v/v] 200â€‰mM EDTA.2Na.2H<sub>2</sub>O and subsequently loaded onto an Agilent 1100 HPLC to measure the adenosine diphosphate (ADP) product formation and the reduction of the ATP substrate. The HPLC automatically injected 0.2â€‰Î¼l of each sample reaction mix onto a Phenomenex 5â€‰Î¼ LUNA C18 column with the mobile phase containing PIC AÂ® (Waters Corporation. USA), 250â€‰ml acetonitrile and 7â€‰g KH<sub>2</sub>PO<sub>4</sub> per litre of water. The flow rate of the mobile phase was 1â€‰ml/min and the separated reactants were detected using a UV detector to measure absorbance at a wavelength of 259â€‰nm. AMP, ADP and ATP standards were used to calibrate the HPLC and the levels of ADP in each sample were determined by using Agilent ChemStation (Revision B.02.01) software (Agilent Technologies. USA). Absorbance values obtained for the control containing distilled water were subtracted from the enzyme reactions. Favorable enzyme activity, in this study, was defined by achieving linearity to demonstrate a constant rate, as well as attaining percentage conversions (of ATP to ADP) within the range of 5â€"15%. <div class="table-wrap_UNKNOWN" id="Tab5"><span class="label" tagx="label" title="label">Table 5</span> </div></p><p>Details of assay reactions for determination of enzyme activity and inhibition assays in the presence of various dilutions of P. sidoides extract</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Enzyme</th> <th>Enzyme Activity Assay reaction mixtures</th> <th><i>P. sidoides</i> inhibition Assay reaction mixtures</th> <th>Incubation time</th> </tr> </thead> <tbody> <tr> <td>NDK</td> <td> <p>100â€‰mMâ€‰K-PO<sub>4</sub> buffer (pHâ€‰6.8), 250â€‰mM KCl</p> <p>5â€‰nM enzyme, 0.2â€‰M Thymidine diphosphate</p> </td> <td> <p>100â€‰mMâ€‰K-PO<sub>4</sub> buffer (pHâ€‰6.8), 250â€‰mM KCl</p> <p>5â€‰nM enzyme, 0.2â€‰M Thymidine diphosphate</p> </td> <td>40 mins</td> </tr> <tr> <td>HSK</td> <td> <p>50â€‰mM HEPES buffer (pHâ€‰7.0), 450â€‰mM KCl</p> <p>704â€‰nM enzyme, 10â€‰mM Homoserine</p> </td> <td> <p>50â€‰mM HEPES buffer (pHâ€‰7.0), 450â€‰mM KCl</p> <p>704â€‰nM enzyme, 10â€‰mM Homoserine</p> </td> <td>4â€‰h</td> </tr> <tr> <td>AK</td> <td> <p>100â€‰mM Tris buffer (pHâ€‰7.0), 250â€‰mM KCl</p> <p>223â€‰nM enzyme, 10â€‰mM Na-acetate</p> </td> <td> <p>100â€‰mM Tris buffer (pHâ€‰7.0), 250â€‰mM KCl</p> <p>223â€‰nM enzyme, 10â€‰mM Na-acetate</p> </td> <td>24â€‰h</td> </tr> <tr> <td>GK</td> <td> <p>100â€‰mM Tris buffer (pHâ€‰7.0), 250â€‰mM KCl</p> <p>208.6â€‰nM enzyme, 100â€‰mM Glycerol</p> </td> <td> <p>100â€‰mM Tris buffer (pHâ€‰7.0), 250â€‰mM KCl</p> <p>208.6â€‰nM enzyme, 100â€‰mM Glycerol</p> </td> <td>24â€‰h</td> </tr> <tr> <td>ThiL</td> <td> <p>100â€‰mM Tris buffer (pHâ€‰8.0), 250â€‰mM KCl</p> <p>2074â€‰nM enzyme, 1â€‰mM Thiamine monophosphate</p> </td> <td> <p>100â€‰mM Tris buffer (pHâ€‰8.0), 250â€‰mM KCl</p> <p>2074â€‰nM enzyme, 1â€‰mM Thiamine monophosphate</p> </td> <td>5â€‰h</td> </tr> <tr> <td>RBKS</td> <td> <p>100â€‰mM Tris buffer (pHâ€‰7.2), 100â€‰mM KCl</p> <p>250â€‰nM enzyme, 10â€‰mM <div class="sc_UNKNOWN">
d</div>-ribose</p> </td> <td> <p>100â€‰mM Tris buffer (pHâ€‰7.2), 100â€‰mM KCl</p> <p>250â€‰nM enzyme, 10â€‰mM <div class="sc_UNKNOWN">
d</div>-ribose</p> </td> <td>4â€‰h</td> </tr> <tr> <td>AsK</td> <td> <p>100â€‰mM Tris-HCl buffer (pHâ€‰7.5), 178.2â€‰nM enzyme</p> <p>10â€‰mMâ€‰ <div class="sc_UNKNOWN">
l</div>-Aspartic acid</p> </td> <td> <p>100â€‰mM Tris-HCl buffer (pHâ€‰7.5), 178.2â€‰nM enzyme</p> <p>10â€‰mMâ€‰ <div class="sc_UNKNOWN">
l</div>-Aspartic acid</p> </td> <td>6â€‰h</td> </tr> <tr> <td>SK</td> <td> <p>100â€‰mMâ€‰K-PO<sub>4</sub> buffer (pHâ€‰6.8), 500â€‰mM KCl</p> <p>10â€‰nM enzyme, 8â€‰mM shikimic acid</p> </td> <td> <p>100â€‰mMâ€‰K-PO<sub>4</sub> buffer (pHâ€‰6.8), 500â€‰mM KCl</p> <p>10â€‰nM enzyme, 8â€‰mM shikimic acid</p> </td> <td>20 mins</td> </tr> </tbody> </table>  <p /> </div> <div class="pelargoniumsidoidesinhibitoryactivity" title="sec"> <div class="title" tagx="title" title="title"><i>Pelargonium sidoides</i> inhibitory activity</div> <p id="Par38">The Mtb kinases were thawed on ice. A 100â€‰mg/ml stock solution of the <i>P. sidoides</i> crude extract was prepared in distilled water. A 10-fold serial dilution of the aqueous plant extracts, ranging from 1â€‰Ã—â€‰10<sup>1</sup>â€‰mg/ml to 1â€‰Ã—â€‰10<sup>âˆ’â€‰5</sup>â€‰mg/ml, was then prepared before being stored at âˆ’â€‰20â€‰Â°C. The inhibitory activity determination was carried out using HPLC enzyme assays as described earlier, at a single ATP and MgCl<sub>2</sub> concentration (1â€‰mM each), and with the addition of varying concentrations of the <i>P. sidoides</i> plant extract. A water-only control was run in parallel, to serve as a negative control for activity comparison analysis. The reaction mixtures of each kinase (Table <a href="#Tab5">5</a>) were incubated at 37â€‰Â°C for a specific time and thereafter stopped with 5% [v/v] 200â€‰mM EDTA.2Na.2H<sub>2</sub>O. The reactions were subsequently analysed as above. All assays were carried out in triplicate and the standard deviation determined and plotted as part of the data. The IC<sub>50</sub> values were calculated, with the aid of GraphPad Prism 5 (GraphPad Software Inc. USA) as specified by the software when plotting log [Inhibitor] concentration versus the enzyme activity.</p> <p id="Par39">Enzyme acticvity assays contained 0.25â€"1.5â€‰mM ATP and 2â€‰mM MgCl<sub>2</sub>. Dose response assays contained 0â€"1â€‰mg/ml <i>P. sidoides</i> plant extract, 1â€‰mM ATP and 1â€‰mM MgCl<sub>2</sub>.</p> </div> <div class="proteinexpressionvalidation" title="sec"> <div class="title" tagx="title" title="title">
Protein expression validation</div> <p id="Par40">Each purified protein was then proteolytically fragmented using the Thermo Scientificâ"¢ SMART Digestâ"¢ kit as per the manufacturerâ€™s instructions. The peptide fragments were lyophilized and made up in 20â€‰Î¼l 2% v/v acetonitrile containing 0.1% v/v formic acid for mass spectroscopy analysis.</p> <div class="liquidchromatography(dionexnano-rslc)" title="sec"> <div class="title" tagx="title" title="title">
Liquid chromatography (Dionex nano-RSLC)</div> <p id="Par41">Liquid chromatography was performed on a Thermo Scientific Ultimate 3000 RSLC equipped with a 5â€‰mmâ€‰Ã—â€‰300â€‰mm C<sub>18</sub> trap column (Thermo Scientific) and a CSH 25â€‰cmâ€‰Ã—â€‰75â€‰Î¼m 1.7â€‰Î¼m particle size C<sub>18</sub> column (Waters) analytical column. The solvent system employed was loading: 2% acetonitrile:water; 0.1% formic acid; Solvent A: 2% acetonitrile:water; 0.1% formic acid and Solvent B: 100% acetonitrile:water, 0.1% formic acid. The samples were loaded onto the trap column using loading solvent at a flow rate of 10 Î¼L/min from a temperature controlled autosampler set at 7â€‰Â°C. Loading was performed for 5â€‰min before the sample was eluted onto the analytical column. Flow rate was set to 325â€‰nL/minute and the gradient generated as follows: 2.0 -10% B for 4â€‰min; followed by 10â€"35% B from 4 to 60â€‰min and finally 35â€"50% B from 60 to 70â€‰min. Chromatography was performed at 40â€‰Â°C and the outflow delivered to the mass spectrometer through a stainless steel nano-bore emitter.</p> </div> <div class="massspectrometry" title="sec"> <div class="title" tagx="title" title="title">
Mass spectrometry</div> <p id="Par42">Mass spectrometry was performed using a Thermo Scientific Fusion mass spectrometer equipped with a Nanospray Flex ionization source. The sample was introduced through a stainless steel emitter. Data was collected in positive mode with spray voltage set to 1.8â€‰kV and ion transfer capillary set to 280â€‰Â°C. Spectra were internally calibrated using polysiloxane ions at m/zâ€‰=â€‰445.12003 and 371.10024. MS1 scans were performed using the orbitrap detector set at 120000 resolution over the scan range <i>m/z</i>â€‰=â€‰350â€"1650 with Adaptive Gain Control (AGC) target at 5â€‰Ã—â€‰10<sup>4</sup> and maximum injection time of 40â€‰ms. Data was acquired in profile mode.</p> <p id="Par43">MS2 acquisitions were performed using monoisotopic precursor selection for ion with charges +â€‰2â€‰âˆ’â€‰+â€‰7 with error tolerance set to +/âˆ’â€‰10â€‰ppm. Precursor ions were excluded from fragmentation once for a period of 60â€‰s. Precursor ions were selected for fragmentation in High Energy Dissociation (HCD) mode using the quadrupole mass analyser with HCD energy set to 30%. Fragment ions were detected in the orbitrap mass analyzer set to 15,000 resolution. The AGC target was set to 5â€‰Ã—â€‰10<sup>4</sup> and the maximum injection time to 30â€‰ms. These data was acquired in centroid mode.</p> <p id="Par44">The raw files generated by the mass spectrometer were imported into Proteome Discoverer v1.4 (Thermo Scientific) and processed using the Sequest algorithm. Database interrogation was performed against a concatenated database created using the Uniprot <i>M. tuberculosis</i> database with the cRAP contaminant database. Semi-tryptic cleavage with 2 missed cleavages was allowed for. Precursor mass tolerance was set to 10â€‰ppm and fragment mass tolerance set to 0.05â€‰Da. Demamidation (arginine and glutamine), oxidation (methionine) and acetylation of protein N-terminal was allowed as dynamic modifications and thiomethyl of cysteine as static modification. Peptide validation was performed using the Target-Decoy PSM validator node. The output files from Proteome Discoverer were imported in to Scaffold Q+ and the assignments validated using X1Tandem and the PeptideProphet and ProteinProphet algorithms.</p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="12896_2020_643_MOESM1_ESM.docx">LINK</a> <p><b>Additional file 1 </b>SDS-PAGE gels of the Mtb his-tagged kinases purified from <i>E. coli</i> BL21 (DE3). A) Nucleotide diphosphate kinase. B) Histidine kinase. C) Acetate kinase and Glycerol kinase. D) Thiamine monophosphate kianse (T) and Ribokinase (R). E) Aspartokinase. F) Skikimate kinase. Protein concentrations of purified enzymes.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
Mtb</div> <div class="def" title="def"> <p id="Par4"><i>Mycobacterium tuberculosis</i></p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NDK</div> <div class="def" title="def"> <p id="Par5">Nucleoside diphosphokinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HSK</div> <div class="def" title="def"> <p id="Par6">Homoserine kinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
AK</div> <div class="def" title="def"> <p id="Par7">Acetate kinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GK</div> <div class="def" title="def"> <p id="Par8">Glycerol kinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ThiL</div> <div class="def" title="def"> <p id="Par9">Thiamine monophosphate kinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RBKS </div> <div class="def" title="def"> <p id="Par10">Ribokinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
AsK</div> <div class="def" title="def"> <p id="Par11">Aspartokinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SK</div> <div class="def" title="def"> <p id="Par12">Shikimate kinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
IPTG </div> <div class="def" title="def"> <p id="Par13">mM Isopropyl Î²-D-1-thiogalactopyranoside</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p><b>Supplementary information</b> accompanies this paper at 10.1186/s12896-020-00643-w.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Not applicable.</p> </div> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>V. Luckman and R. Roth designed and conducted the molecular biology and enzymology experiments. S. Odeyemi drafted the manuscript. N. Tshililo, L. Mbalala and M. Vlok designed and conducted the proteomics and mass spectroscopy experiments. J. Dewar provided the funding and supervised the student. C. Kenyon devised the project, supervised the experiments, did data interpretation and drafted the manuscript. The author(s) read and approved the final manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>The authors will like to acknowledge University of South Africa and Council for Scientific and Industrial Research (CSIR) for funding and supporting this work. This research was partially funded by the South African government through the South African Medical Research Council. The content is solely the responsibility of the authors and does not necessarily represent the official views of the South African Medical Research Council.</p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>The datasets used and/or analysed during the current study are available from the corresponding author on reasonable request.</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par45">This study was primarily a chemistry investigation and involved no human or animal participants as a result no ethics approval was required in the respective institutions.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p id="Par46">No consent was required for the publication of any of the data.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par47">We wish to confirm that there are no known conflicts of interest associated with this publication and there has been no significant financial support for this work that could have influenced its outcome. All corroborating data may be obtained from the corresponding author.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bladt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagner</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">From the Zulu medicine to the European phytomedicine Umckaloabos</span><span class="source" tagx="source" title="source">Phytomedicine.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">2</span><span class="lpage" tagx="lpage" title="lpage">4</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phymed.2006.11.030">10.1016/j.phymed.2006.11.030</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brendler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Wyk</span><span class="given-names" tagx="given-names" title="given-names">B-E</span></span></span><span class="mixed-article-title" title="mixed-article-title">A historical, scientific and commercial perspective on the medicinal use of Pelargonium sidoides (Geraniaceae)</span><span class="source" tagx="source" title="source">J Ethnopharmacol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">420</span><span class="lpage" tagx="lpage" title="lpage">433</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jep.2008.07.037">10.1016/j.jep.2008.07.037</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18725280">18725280</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheek</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grishin</span><span class="given-names" tagx="given-names" title="given-names">NV</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sequence and structure classification of kinases</span><span class="source" tagx="source" title="source">J Mol Biol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">320</span><span class="issue" tagx="issue" title="issue">2002</span><span class="fpage" tagx="fpage" title="fpage">855</span><span class="lpage" tagx="lpage" title="lpage">881</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0022-2836(02)00538-7">10.1016/S0022-2836(02)00538-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12095261">12095261</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheek</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ginalski</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grishin</span><span class="given-names" tagx="given-names" title="given-names">NV</span></span></span><span class="mixed-article-title" title="mixed-article-title">A comprehensive update of the sequence and structure classification of kinases</span><span class="source" tagx="source" title="source">BMC Struct Biol</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">6</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1472-6807-5-6">10.1186/1472-6807-5-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15771780">15771780</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cole</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brosch</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parkhill</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garnier</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Deciphering the biology of Mycobacterium tuberculosis from the complete genome sequence</span><span class="source" tagx="source" title="source">Nature.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">393</span><span class="fpage" tagx="fpage" title="fpage">537</span><span class="lpage" tagx="lpage" title="lpage">544</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/31159">10.1038/31159</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9634230">9634230</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Health and related indicators</span><span class="source" tagx="source" title="source">South Afr Health Rev</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">2017</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">217</span><span class="lpage" tagx="lpage" title="lpage">339</span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Pascale</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griffiths</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shakya</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nazi</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">GD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification and characterization of new inhibitors of fungal Homoserine kinase</span><span class="source" tagx="source" title="source">ChemBioChem.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">1179</span><span class="lpage" tagx="lpage" title="lpage">1182</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cbic.201100121">10.1002/cbic.201100121</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21538764">21538764</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reshetnikova</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Crystal structure of Shikimate kinase from Mycobacterium tuberculosis reveals the dynamic role of the LID domain in catalysis</span><span class="source" tagx="source" title="source">J Mol Biol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">319</span><span class="fpage" tagx="fpage" title="fpage">779</span><span class="lpage" tagx="lpage" title="lpage">789</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0022-2836(02)00339-X">10.1016/S0022-2836(02)00339-X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12054870">12054870</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Helfer M, Koppensteiner H, Schneider M, Rebensburg S, Forcisi S, MÃ¼ller C, Schmitt-Kopplin P, Schindler M, Brack-Werner R. The root extract of the medicinal plant Pelargonium sidoides is a potent HIV-1 attachment inhibitor. PLoS One. 2014;9. 10.1371/journal.pone.0087487.</span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenyon</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steyn</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roth</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steenkamp</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nkosi</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oldfield</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span></span><span class="mixed-article-title" title="mixed-article-title">The role of the C8 proton of ATP in the regulation of phosphoryl transfer within kinases and synthetases</span><span class="source" tagx="source" title="source">BMC Biochem</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">36</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2091-12-36">10.1186/1471-2091-12-36</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21749731">21749731</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenyon</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roth</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Westhuyzen</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parkinson</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Conserved phosphoryl transfer mechanisms within kinase families and the role of the C8 proton of ATP in the activation of phosphoryl transfer</span><span class="source" tagx="source" title="source">BMC Res Notes</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">131</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1756-0500-5-131">10.1186/1756-0500-5-131</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22397702">22397702</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolodziej</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fascinating metabolic pools of Pelargonium sidoides and Pelargonium reniforme, traditional and phytomedicinal sources of the herbal medicine UmckaloaboÂ®</span><span class="source" tagx="source" title="source">Phytomedicine.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">9</span><span class="lpage" tagx="lpage" title="lpage">17</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phymed.2006.11.021">10.1016/j.phymed.2006.11.021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17188477">17188477</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolodziej</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antimicrobial, antiviral and immunomodulatory activity studies of Pelargonium sidoides (EPsÂ® 7630) in the context of health promotion</span><span class="source" tagx="source" title="source">Pharmaceuticals (Basel)</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">1295</span><span class="lpage" tagx="lpage" title="lpage">1314</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ph4101295">10.3390/ph4101295</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27721327">27721327</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishna</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srinivasan</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ajitkumar</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Varshney</span><span class="given-names" tagx="given-names" title="given-names">U</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mycobacterium tuberculosis and Escherichia coli nucleoside diphosphate kinases lack multifunctional activities to process uracil containing DNA</span><span class="source" tagx="source" title="source">DNA Repair (Amst)</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">1483</span><span class="lpage" tagx="lpage" title="lpage">1492</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/J.DNAREP.2004.06.007">10.1016/J.DNAREP.2004.06.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15380104">15380104</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lahlou</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">The success of natural products in drug discovery</span><span class="source" tagx="source" title="source">Pharmacol Pharm</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">17</span><span class="lpage" tagx="lpage" title="lpage">31</span><span class="pub-id"><a href="https://dx.doi.org/10.4236/pp.2013.43A003">10.4236/pp.2013.43A003</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mativandlela</span><span class="given-names" tagx="given-names" title="given-names">SPN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lall</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">JJM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antibacterial, antifungal and antitubercular activity of (the roots of) Pelargonium reniforme (CURT) and Pelargonium sidoides (DC) (Geraniaceae) root extracts</span><span class="source" tagx="source" title="source">SA J Bot</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">72</span><span class="fpage" tagx="fpage" title="fpage">232</span><span class="lpage" tagx="lpage" title="lpage">237</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/13880200701538716">10.1080/13880200701538716</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mativandlela</span><span class="given-names" tagx="given-names" title="given-names">SPN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">JJM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hussein</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lall</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antitubercular activity of compounds isolated from Pelargonium sidoides</span><span class="source" tagx="source" title="source">Pharm Biol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">645</span><span class="lpage" tagx="lpage" title="lpage">650</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/13880200701538716">10.1080/13880200701538716</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Odeyemi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Afolayan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bradley</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phytochemical analysis and anti-oxidant activities of Albuca bracteata Jacq. and Albuca setosa Jacq bulb extracts used for the management of diabetes in the eastern cape, South Africa, Asian Pac</span><span class="source" tagx="source" title="source">J Trop Biomed</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">577</span><span class="lpage" tagx="lpage" title="lpage">584</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.apjtb.2017.05.013">10.1016/j.apjtb.2017.05.013</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pereira</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Oliveira</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Canduri</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dias</span><span class="given-names" tagx="given-names" title="given-names">MVB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palma</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basso</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santos</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Azevedo</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure of shikimate kinase from <i>Mycobacterium tuberculosis</i> reveals the binding of shikimic acid</span><span class="source" tagx="source" title="source">Acta Crystallogr Sect D Biol Crystallogr</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">2310</span><span class="lpage" tagx="lpage" title="lpage">2319</span><span class="pub-id"><a href="https://dx.doi.org/10.1107/S090744490402517X">10.1107/S090744490402517X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15583379">15583379</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnitzler</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneider</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stintzing</span><span class="given-names" tagx="given-names" title="given-names">FC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carle</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reichling</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Efficacy of an aqueous Pelargonium sidoides extract against herpesvirus</span><span class="source" tagx="source" title="source">Phytomedicine.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">1108</span><span class="lpage" tagx="lpage" title="lpage">1116</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phymed.2008.06.009">10.1016/j.phymed.2008.06.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18691858">18691858</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sikarwar</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaushik</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinha</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaur</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cloning, expression, and purification of nucleoside Diphosphate kinase from Acinetobacter baumannii</span><span class="source" tagx="source" title="source">Enzyme Res</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">2013</span><span class="fpage" tagx="fpage" title="fpage">597028</span><span class="pub-id"><a href="https://dx.doi.org/10.1155/2013/597028">10.1155/2013/597028</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23662205">23662205</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antimycobacterial activity of indigenous south African plants</span><span class="source" tagx="source" title="source">African Med J</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">904</span><span class="lpage" tagx="lpage" title="lpage">907</span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tomioka</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatano</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yasumoto</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimizu</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Recent advances in antituberculous drug development and novel drug targets</span><span class="source" tagx="source" title="source">Expert Rev Respir Med</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">455</span><span class="lpage" tagx="lpage" title="lpage">471</span><span class="pub-id"><a href="https://dx.doi.org/10.1586/17476348.2.4.455">10.1586/17476348.2.4.455</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20477209">20477209</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walaas</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walaas</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bridgwater</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Briggs</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haslewood</span><span class="given-names" tagx="given-names" title="given-names">GAD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flood</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">The activation of muscle hexokinase by divalent metal ions</span><span class="source" tagx="source" title="source">Acta Chem Scand</span><span class="year" tagx="year" title="year">1962</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">1682</span><span class="lpage" tagx="lpage" title="lpage">1694</span><span class="pub-id"><a href="https://dx.doi.org/10.3891/acta.chem.scand.16-1682">10.3891/acta.chem.scand.16-1682</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO. Fact sheet on Tuberculosis: World Heal. Organ; 2020. <a href="http://www.who.int/en/news-room/fact-sheets/detail/tuberculosis">http://www.who.int/en/news-room/fact-sheets/detail/tuberculosis</a>.</span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Witte</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koch</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volk</span><span class="given-names" tagx="given-names" title="given-names">HD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolk</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabat</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Pelargonium sidoides extract EPs 7630 drives the innate immune defense by activating selected MAP kinase pathways in human monocytes</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">13</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0138075">10.1371/journal.pone.0138075</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Z-G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Physical and functional interaction between d-ribokinase and topoisomerase I has opposite effects on their respective activity in Mycobacterium smegmatis and Mycobacterium tuberculosis</span><span class="source" tagx="source" title="source">Arch Biochem Biophys</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">512</span><span class="fpage" tagx="fpage" title="fpage">135</span><span class="lpage" tagx="lpage" title="lpage">142</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/J.ABB.2011.05.018">10.1016/J.ABB.2011.05.018</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21683681">21683681</a></span></span></li> </ul> </div> </div>  </body></html>