<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Ecol Evol</span><span class="iso-abbrev" title="iso-abbrev">Ecol Evol</span><span class="doi" title="doi">10.1002/(ISSN)2045-7758</span><span class="publisher-id" title="publisher-id">ECE3</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Ecology and Evolution</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-7758</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7920766">7920766</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1002/ece3.7139">10.1002/ece3.7139</a></span><span class="publisher-id" title="publisher-id">ECE37139</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Research</span>: <span class="subject" title="subject">Original Research</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Optimization and validation of a costâ€�effective protocol for biosurveillance of invasive alien species</div> <div class="alt-title" title="alt-title">
MILIÃ�Nâ€�GARCÃ�A et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MiliÃ¡nâ€�GarcÃ­a</span><span class="given-names" tagx="given-names" title="given-names">Yoamel</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-6596-5063</span><a href="#ece37139-aff-0001"><sup>1</sup></a><span class="address" title="address"><span class="email" tagx="email" title="email">ymiliang@uoguelph.ca</span><span class="email" tagx="email" title="email">yoamelmg@gmail.com</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">Robert</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-6731-2506</span><a href="#ece37139-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madden</span><span class="given-names" tagx="given-names" title="given-names">Mary</span></span><a href="#ece37139-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bullasâ€�Appleton</span><span class="given-names" tagx="given-names" title="given-names">Erin</span></span><a href="#ece37139-aff-0002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanner</span><span class="given-names" tagx="given-names" title="given-names">Robert H.</span></span><a href="#ece37139-aff-0001"><sup>1</sup></a></span></div><span class="citation_author_institution" id="ece37139-aff-0001">[1], <span class="named-content" title="named-content">Department of Integrative Biology</span><span class="institution" title="institution">University of Guelph</span><span class="city" tagx="city" title="city">Guelph</span><span class="named-content" title="named-content">ON</span><span class="country" tagx="country" title="country">Canada</span></span><span class="citation_author_institution" id="ece37139-aff-0002">[2], <span class="named-content" title="named-content">Canadian Food Inspection Agency</span><span class="institution" title="institution">Government of Canada</span><span class="city" tagx="city" title="city">Ottawa</span><span class="named-content" title="named-content">ON</span><span class="country" tagx="country" title="country">Canada</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><b>Correspondence</b><br /><br /> Yoamel MiliÃ¡nâ€�GarcÃ­a, Department of Integrative Biology, University of Guelph, 50 Stone Rd E, Guelph, ON N1G 2W1, Canada.<br /><br /> Emails: <span class="email" tagx="email" title="email">ymiliang@uoguelph.ca</span>; <span class="email" tagx="email" title="email">yoamelmg@gmail.com</span><br /><br /></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-2-2</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021-3</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">5</span><span class="issue-id" tagx="issue-id" title="issue-id">10.1002/ece3.v11.5</span><span class="fpage" tagx="fpage" title="fpage">1999</span><span class="lpage" tagx="lpage" title="lpage">2014</span><span class="history" title="history"><span class="received" title="received">received: 2020-9-08</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-11-23</span><span class="accepted" title="accepted">accepted: 2020-11-26</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="file:ECE3-11-1999.pdf" title="self-uri">file:ECE3-11-1999.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Environmental DNA (eDNA) metabarcoding has revolutionized biodiversity monitoring and invasive pest biosurveillance programs. The introduction of insect pests considered invasive alien species (IAS) into a nonâ€�native range poses a threat to native plant health. The early detection of IAS can allow for prompt actions by regulating authorities, thereby mitigating their impacts. In the present study, we optimized and validated a fast and costâ€�effective eDNA metabarcoding protocol for biosurveillance of IAS and characterization of insect and microorganism diversity. Fortyâ€�eight traps were placed, following the CFIA's annual forest insect trapping survey, at four locations in southern Ontario that are high risk for forest IAS. We collected insects and eDNA samples using Lindgren funnel traps that contained a saturated salt (NaCl) solution in the collection jar. Using cytochrome c oxidase I (COI) as a molecular marker, a modified Illumina protocol effectively identified 2,535 Barcode Index Numbers (BINs). BINs were distributed among 57 Orders and 304 Families, with the vast majority being arthropods. Two IAS (<i>Agrilus planipennis</i> and <i>Lymantria dispar</i>) are regulated by the Canadian Food Inspection Agency (CFIA) as plant health pests, are known to occur in the study area, and were identified through eDNA in collected traps. Similarly, using 16S ribosomal RNA and nuclear ribosomal internal transcribed spacer (ITS), five bacterial and three fungal genera, which contain species of regulatory concern across several Canadian jurisdictions, were recovered from all sampling locations. Our study results reaffirm the effectiveness and importance of integrating eDNA metabarcoding as part of identification protocols in biosurveillance programs.</p> </div> <div class="abstract" title="abstract"> <p>In the present study, we optimized and validated a fast and costâ€�effective eDNA metabarcoding protocol for biosurveillance of plant IAS and characterization of insect and microorganism diversity. Fortyâ€�eight traps were placed, following the CFIA's annual forest insect trapping survey, at four locations in southern Ontario that are high risk for forest IAS. We collected insects and eDNA samples using Lindgren funnel traps that contained a saturated salt (NaCl) solution in the collection jar. Our study results reaffirm the effectiveness and importance of integrating eDNA metabarcoding as part of identification protocols in biosurveillance programs. </p><div class="boxed-text" tagx="boxed-text" title="boxed-text"> <div class="graphic" position="anchor" id="nlm-graphic-1" orientation="portrait" /> </div> <p /> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
16S</div> <div class="kwd" title="kwd">
biomonitoring</div> <div class="kwd" title="kwd">
biosurveillance</div> <div class="kwd" title="kwd">
COI</div> <div class="kwd" title="kwd">
eDNA metabarcoding</div> <div class="kwd" title="kwd">
invasive alien species</div> <div class="kwd" title="kwd">
ITS</div> <div class="kwd" title="kwd">
salt trap solution</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Canadian Food Inspection Agency </span><span class="institution-id" title="institution-id">10.13039/100009837</span></div> </div> </div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="cover-date" value="March 2021" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_JATSPMC version:5.9.9 mode:remove_FC converted:01.03.2021" /> </div> </div> <p content-type="self-citation"><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">MiliÃ¡nâ€�GarcÃ­a</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Madden</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bullasâ€�Appleton</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hanner</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Optimization and validation of a costâ€�effective protocol for biosurveillance of invasive alien species</span>. <span class="source" tagx="source" title="source">Ecol Evol</span>. <span class="year" tagx="year" title="year">2021</span>;<span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">1999</span>â€"<span class="lpage" tagx="lpage" title="lpage">2014</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.7139">10.1002/ece3.7139</a></span></span></p> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>Metabarcoding has become an effective method for molecularâ€�based biomonitoring programs and biosurveillance for invasive alien species (IAS; Makiola etÂ al.,Â <a href="#ece37139-bib-0026">2020</a>). This molecular technique involves highâ€�throughput sequencing (HTS), which facilitates the identification of multiple species using environmental DNA (eDNA) extracted from complex ecological samples (Creer etÂ al.,Â <a href="#ece37139-bib-0009">2016</a>; Taberlet etÂ al.,Â <a href="#ece37139-bib-0040">2012</a>). Incorporating HTS into biosurveillance programs is advantageous as it can provide results faster than conventional identification methods, thereby allowing for early detection of species of concern, including IAS (Ruppert etÂ al.,Â <a href="#ece37139-bib-0036">2019</a>). The ability to obtain results fast, the existence of curated DNA reference libraries (Nilsson etÂ al.,Â <a href="#ece37139-bib-0027">2019</a>; Quast etÂ al.,Â <a href="#ece37139-bib-0031">2013</a>; Ratnasingham &amp;amp; Hebert,Â <a href="#ece37139-bib-0033">2007</a>), and the continued drop in the cost of HTS platforms have contributed to metabarcoding's rise in popularity in environmental monitoring (Cristescu,Â <a href="#ece37139-bib-0010">2014</a>; de Kerdrel etÂ al.,Â <a href="#ece37139-bib-0011">2020</a>). Furthermore, as DNA reference libraries with associated morphological species identifications continue to grow, the time, effort, and resources that would otherwise be put toward morphological identification of specimens can be saved through the use of molecular identification methods. This is significant since the number of taxonomists available to conduct species identifications is declining. Reference libraries, therefore, provide a permanent repository of traditional taxonomic expertise that can be used with the appropriate molecular identification tools as needed (Cristescu,Â <a href="#ece37139-bib-0010">2014</a>).</p> <p>The mitochondrial DNA region coding for cytochrome c oxidase I (COI) enzyme has been recognized as the primary marker for metabarcoding in the animal kingdom (Hebert etÂ al.,Â <a href="#ece37139-bib-0020">2003</a>). Similarly, the nuclear ribosomal internal transcribed spacer (ITS) and the 16S ribosomal RNA gene have been adopted as the fungal and bacterial markers, respectively (Klindworth etÂ al.,Â <a href="#ece37139-bib-0025">2013</a>; Seifert,Â <a href="#ece37139-bib-0037">2009</a>). Multiple reference databases and workbenches for data collection and analysis are currently available for these markers as well as a growing number of sequence records (e.g., BOLD has 8,099,249 records for COI as of November 20, 2020; Nilsson etÂ al.,Â <a href="#ece37139-bib-0027">2019</a>; Quast etÂ al.,Â <a href="#ece37139-bib-0031">2013</a>; Ratnasingham &amp;amp; Hebert,Â <a href="#ece37139-bib-0033">2007</a>, <a href="#ece37139-bib-0034">2013</a>, and UNITE has 2,480,043 for ITS as of November 20, 2020). However, despite the plethora of protocols available for eDNA metabarcoding and the drop in the cost of HTS over the years, there is limited information available on costâ€�effectiveness (Elbrecht &amp;amp; Steinke,Â <a href="#ece37139-bib-0014">2019</a>; Ji etÂ al.,Â <a href="#ece37139-bib-0024">2013</a>; de Kerdrel etÂ al.,Â <a href="#ece37139-bib-0011">2020</a>). Biodiversity monitoring and biosurveillance programs would benefit from the inclusion of HTS protocols to accelerate the detection of species of concern and reduce costs associated with processing large numbers of individual specimens (Giovani etÂ al.,Â <a href="#ece37139-bib-0016">2020</a>; Piper etÂ al.,Â <a href="#ece37139-bib-0030">2019</a>). On a broader scale, costâ€�effectiveness is essential given that only a fraction of the global biodiversity (including IAS and other pest species) has been described to date, and several billions of dollars may be needed to complete this enormous endeavor (Carbayo &amp;amp; Marques,Â <a href="#ece37139-bib-0006">2011</a>).</p> <p>Pest insects have a negative impact on Canada's forests and are second only to wildfires in their effect ("The State of Canada's Forests. Annual Report 2018.", <a href="#ece37139-bib-0041">n.d.</a>). Pest insects that are IAS are considered a byproduct of anthropogenic activities and can provoke significant economic and biodiversity losses (Westphal etÂ al.,Â <a href="#ece37139-bib-0045">2008</a>). For instance, IAS are capable of destroying about 400,000Â ha of forest every year in Canada (Government of Canada, <a href="#ece37139-bib-0017">2013</a>). Nonmanufactured wood packaging and loose wood dunnage are highâ€�risk pathways for the introduction of IAS, particularly woodâ€�boring beetles (e.g., Cerambycidae and Buprestidae families), like the Asian longâ€�horned beetle (<i>Anoplophora glabripennis</i>) and the emerald ash borer beetle (<i>Agrilus planipennis</i>) (CFIA,Â <a href="#ece37139-bib-0007">2017</a>). Current pathwayâ€�based biosurveillance programs led by the Canadian Food Inspection Agency (CFIA) includes placing traps at sites that are at a high risk for IAS, such as industrial zones receiving international commodities associated with nonmanufactured wood packaging and dunnage. The trapped insects remain in the fluid of collection jars until specimens are decanted and referred to the CFIA Entomology laboratory for morphological identification. As organisms interact with their environment, whether it be the fluid of a collection jar or a plant, they shed DNA into it (Adams etÂ al.,Â <a href="#ece37139-bib-0002">2019</a>; TabÂ erlet etÂ al.,Â <a href="#ece37139-bib-0039">2018</a>). The eDNA extracted from the collection fluid can then be used to identify insects considered IAS, native pests, and accompanying microorganisms using molecular methods such as eDNA metabarcoding (Pawlowski etÂ al.,Â <a href="#ece37139-bib-0029">2020</a>; Taberlet etÂ al.,Â <a href="#ece37139-bib-0039">2018</a>).</p> <p>Traditionally, plant health trapping survey protocols for insect pest detection have used alcoholâ€�based collection fluids (e.g., ethanol or propylene glycol) to preserve biological material for morphological identification (CFIA,Â <a href="#ece37139-bib-0007">2017</a>). Recent protocols have replaced alcoholâ€�based fluids with saturated salt solutions due to advantages over the previous chemistries including lower cost, less storage space requirements, low toxicity to humans, fewer regulatory constraints for laboratories and inspectors, nonflammability, and lower evaporation rate (Young etÂ al.,Â <a href="#ece37139-bib-0047">2020</a>). Salt solutions have also proven to be satisfactory for preserving morphological structures of captured specimens (Young etÂ al.,Â <a href="#ece37139-bib-0047">2020</a>) and for the preservation of water samples for eDNA analysis (Williams etÂ al.,Â <a href="#ece37139-bib-0046">2016</a>). Therefore, the standardization and validation of a costâ€�effective protocol for eDNA metabarcoding analysis using salt trap solutions are of crucial interest. The current study focusses on the optimization and validation of a protocol for eDNA metabarcoding of organisms captured during regulatory plant health monitoring surveys in southern Ontario, Canada. Special attention will be placed on the woodâ€�boring beetles as well as regulated pathogenic bacteria and fungi inadvertently collected in the traps.</p> </div> <div class="materials-and-methods" title="materials-and-methods"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
MATERIALS AND METHODS</div> <div class="targetpests" title="sec"><span class="label" tagx="label" title="label">2.1</span><div class="title" tagx="title" title="title">
Target pests</div> <p>These included all insect IAS found in Canada as well as pests that impact plant health and are regulated by the CFIA (30 taxa; <a href="https://www.inspection.gc.ca/plant-health/plant-pests-invasive-species/insects/eng/1307077188885/1307078272806">https://www.inspection.gc.ca/plantâ€�health/plantâ€�pestsâ€�invasiveâ€�species/insects/eng/1307077188885/1307078272806</a>). The surveillance protocol targeted longâ€�horned beetles (<i>A.Â glabripennis</i>, <i>Anoplophora chinensis</i>, <i>Tetropium fuscum</i>, <i>Tetropium castaneum</i>, and <i>Aromia bungii</i>) which are woodâ€�boring IAS (CFIA,Â <a href="#ece37139-bib-0007">2017</a>). Also, we explored 39 fungi and eight bacteria recognized as plant pests and regulated by the Canadian government (<a href="https://www.inspection.gc.ca/plant-health/plant-pests-invasive-species/regulated-pests/eng/1363317115207/1363317187811">https://www.inspection.gc.ca/plantâ€�health/plantâ€�pestsâ€�invasiveâ€�species/regulatedâ€�pests/eng/1363317115207/1363317187811</a>). The eight bacterial species represent seven different genera: <i>Brenneria salicis</i>, <i>Clavibacter michiganensis</i>Â subsp.Â <i>sepedonicus</i>, <i>Pseudomonas syringaeÂ pv</i>.Â <i>cannabina</i>, <i>Ralstonia solanacearum</i>, <i>Xanthomonas campestrisÂ pv</i>.Â <i>cannabis</i>, <i>Xanthomonas populi</i>, <i>Xylella fastidiosa</i>, and <i>Xylophilus ampelinus</i>.</p> </div> <div class="collectionlocations" title="sec"><span class="label" tagx="label" title="label">2.2</span><div class="title" tagx="title" title="title">
Collection locations</div> <p>Lindgren funnel traps were placed at four locations in Southern Ontario, Canada, with six sample sites (traps) at each location (FigureÂ <a href="#ece37139-fig-0001">1</a>). The traps represented a subset of those deployed following the CFIA's annual regulatory survey program (CFIA's Plant Health business line), which is aimed at detecting insects introduced through highâ€�risk pathways. Locations were selected based on susceptibility to IAS, accessibility, sufficient area to accommodate six traps spaced approximately 25â€"30 meters apart, and limited public access to avoid vandalism. Six traps were then distributed at these sites near species of trees known to be hosts to the target IAS and that were showing evidence of stress/decline, or damage, indicating the possible presence of IAS. One location was in Halton Hills (GT) and had traps situated near a Municipal landfill and railroad track. A second location was in a 241â€�hectare park of Carolinian forest that is situated along Lake Erie in Chathamâ€�Kent (WP). A third location, Barrie (BA), was within a wooded area close to railway tracks that experiences a high volume of import traffic. A fourth and final location was in Woodstock (TO), a city that includes manufacturing facilities that import commodities packaged using wood materials. All four locations are less than 215Â km from the USâ€"Canada border and could presumably be exposed to traffic of imported wood materials carrying IAS and other pests.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Map showing collection locations in Ontario, Canada. Inset indicates Southern Ontario. Barrie (BA), Halton Hills (GT), Woodstock (TO), and Chathamâ€�Kent (WP)</p>   </div> </div> <div class="collectionprotocol" title="sec"><span class="label" tagx="label" title="label">2.3</span><div class="title" tagx="title" title="title">
Collection protocol</div> <p>Lindgren funnel traps were equipped with Ultra High Release Ethanol (UHR) Lures (Synergy Semiochemical Corp), a type of lure that targets all longhorn beetles, and a collection cup that contained approximately 100Â ml of a table saltâ€�saturated solution (2Â kg of sodium chloride per 5Â L of water). Traps were placed at sampling sites during the summer following CFIA's Survey Protocol: Invasive Alien Species Forestry Trapping (<a href="#ece37139-bib-0007">2017</a>). The first set of traps was collected in July 2018 and the second in August 2018, approximately 4Â weeks after having been placed. With four locations, six sample sites, and two collection periods, a total of 48 field samples were collected for the study. All specimens captured in the traps were decanted from the solution and sent to the CFIA's entomology Laboratory. Salt trap solutions were transported to the Hanner Laboratory at the University of Guelph, Ontario, Canada, for temporary storage at âˆ’80â"ƒ followed by processing and analysis.</p> </div> <div class="samplefiltration" title="sec"><span class="label" tagx="label" title="label">2.4</span><div class="title" tagx="title" title="title">
Sample filtration</div> <p>Salt solutions were filtered through a nitrocellulose mixed ester membrane filter (pore size 1Â Âµm, diameter 47Â mm, Sterlitech). The filter was mounted onto a magnetic filtration cup (Pall) and secured to a 3â€�piece manifold connected to an GAST vacuum pump (GAST Manufactured, Inc). All supplies were sterilized with 50% bleach or ELIMINase (Decon Labs) before filtration.</p> <p>The amount of debris within the salt solutions varied greatly across samples and would sometimes cause clogging of membrane pores. Therefore, most samples required multiple membranes to filter the entire volume, resulting in a total of 110 membranes for the 48 samples. The membranes were stored at âˆ’80Â°C until DNA extraction could be completed. As a negative control, one sample of saturated table salt solution prepared in the laboratory was also filtered. All membranes were stored in new Ziploc bags at âˆ’80Â°C until processing for eDNA extraction.</p> </div> <div class="ednaextraction" title="sec"><span class="label" tagx="label" title="label">2.5</span><div class="title" tagx="title" title="title">
eDNA extraction</div> <p>We used a modified CTAB buffer (Coyne etÂ al.,Â <a href="#ece37139-bib-0008">2005</a>; Dempster et al., <a href="#ece37139-bib-0012">1999</a>) (2% w/v cetyltrimethylammonium bromide, 2% w/v polyvinylpyrrolidone, 1.4M NaCl, 100Â mM Trisâ€�HCl, 20Â nM EDTA). This buffer was used due to reported success in retrieving good quality eDNA yield (Barnes etÂ al.,Â <a href="#ece37139-bib-0003">2014</a>; Dougherty etÂ al.,Â <a href="#ece37139-bib-0013">2016</a>; Renshaw etÂ al.,Â <a href="#ece37139-bib-0035">2015</a>; Turner etÂ al.,Â <a href="#ece37139-bib-0042">2014</a>).</p> <p>The eDNA extraction steps described below were performed on the 110 membranes and the negative control. To extract the eDNA, we used the Dougherty etÂ al.Â (<a href="#ece37139-bib-0013">2016</a>) protocol with some minor modifications. Each filter paper was allowed to thaw and then cut into quarters using new razor blades. Each quarter was placed into individual 2â€�ml microcentrifuge tubes containing ~250Â mg of 1â€�mmâ€�diameter glass beads, and 500Â Î¼l of CTAB buffer prewarmed to 65Â°C in a heat block. The filter paper quarters were pulped using the TissueLyser II (Qiagen) at a frequency of 30Â Hz for one minute and then incubated at 65Â°C for one hour in a heat block. Following incubation, each tube received 500Â Î¼l of 24:1 chloroformâ€"isoamyl alcohol and was briefly vortexed. The aqueous phase containing the eDNA was separated from the chloroform phase by centrifuging the tubes at 13,000Â <i>g</i> for 15Â min to enable phase separation. After two passes with 24:1 chloroformâ€"isoamyl alcohol, the aqueous phase (approximately 500Â Î¼l) was transferred to new 2.0â€�ml Eppendorf tubes and mixed with 500Â Î¼l of isopropanol, and 200Â Î¼l of a 5Â M NaCl solution. Tubes were briefly vortex and stored at âˆ’20Â°C overnight to facilitate precipitation of eDNA. Tubes were then centrifugated at 13,000Â <i>g</i> for another 15Â min to pellet the eDNA. The supernatant was removed by pipetting, and 200Â Î¼l of 70% ethanol was added to wash the pellet. After centrifugation at 13,000Â <i>g</i> for 15Â min, the ethanol was removed by pipetting and replaced anew. The contents were again briefly vortexed and centrifuged at 13,000Â <i>g</i> for 15Â min. The ethanol was removed for a final time, and the tubes were then placed in a fume hood to allow any remaining ethanol to evaporate at room temperature (~1Â hr). Once the eDNA pellet was dry, it was then resuspended in 25Â Î¼l 1X TE buffer (prewarmed at 70Â°C) to favor DNA dilution. eDNA extracts from each quarter belonging to the same trap were pooled. A 5Â Î¼l subsample was used to visually assess the presence and quality of eDNA on a 1% agarose gel electrophoresis. The concentration of eDNA (ng/Î¼l) was determined by fluorometry (Qubit). All eDNA extracts with concentrations over 5Â ng/Î¼l were diluted to 5Â ng/Î¼l with 10Â mM Tris pHÂ =Â 8.5 This was done to normalize the amount of starting material per sample and to reduce the effect of potential PCR inhibitors in the samples. Eight samples exhibited signals of inhibition by showing no amplification during the first PCR of library preparation (see below). These samples were treated to remove inhibitors using a 1x NGS magnetic beads (Machereyâ€�Nagel) ratio, following manufacturer's protocols. Following quality control steps, the remaining volume of solution containing eDNA extract was placed in 1.5â€�ml LoBind Eppendorf tubes and stored at âˆ’20Â°C.</p> </div> <div class="librarypreparation" title="sec"><span class="label" tagx="label" title="label">2.6</span><div class="title" tagx="title" title="title">
Library preparation</div> <p>For COI and ITS library preparation, we adapted the â€œ16S Metagenomic Sequencing Library Preparation Protocolâ€� (Illumina) while no modifications were needed for 16S (â€œ16S Metagenomic Sequencing Library Preparationâ€�, <a href="#ece37139-bib-0001">2013</a>).</p> <div class="firstpcr" title="sec"><span class="label" tagx="label" title="label">2.6.1</span><div class="title" tagx="title" title="title">
First PCR</div> <p>The first PCR was conducted in 25Â Î¼l reaction volumes on an Eppendorf Mastercycler thermal cycler with each reaction containing 2.5Â Î¼l of eDNA template, 12.5Â Î¼l of 2X KAPA HiFi HotStart Ready Mix (Roche Diagnostics), and 0.2Â Î¼M of primers with Illumina adaptors (TableÂ <a href="#ece37139-tbl-0001">1</a>). The eDNA target fragment lengths to amplify were ~407 base pairs (bp) for COI, 460Â bp for 16S, and 500â€"600Â bp for ITS. Cycling conditions were as follows: COI: 94Â°C (120Â s) followed by five cycles at 94Â°C (40Â s), 45Â°C (40Â s), 72Â°C (60Â s); then 35 cycles of 94Â°C (40Â s), 51Â°C (40Â s), 72Â°C (60Â s); and a final extension of 72Â°C (5Â min) (Braukmann etÂ al.,Â <a href="#ece37139-bib-0005">2019</a>); 16S: 95Â°C (180Â s), 25 cycles of 95Â°C (30Â s), 55Â°C (30Â s), 72Â°C (30Â s), and a final extension of 72Â°C (300Â s) (â€œ16S Metagenomic Sequencing Library Preparationâ€�, <a href="#ece37139-bib-0001">2013</a>); ITS: 95Â°C (120Â s), 40 cycles at 95Â°C (30Â s), 55Â°C (30Â s), 72Â°C (60Â s), and a final extension of 72Â°C (600Â s) (Beeck etÂ al.,Â <a href="#ece37139-bib-0004">2014</a>). A negative control was included for every batch of samples amplified. All PCR products were visualized on 1% agarose gel to check for proper amplification and fragment size of the amplicons. PCR products were purified using a 0.8x NGS magnetic beads (Machereyâ€�Nagel) ratio following the manufacturer's protocols.</p> <div class="table-wrap_UNKNOWN" id="ece37139-tbl-0001" content-type="TABLE" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p>Illumina adapters and primers (sequence 5â€²â€�3â€²) from the literature used in HTS for COI, 16S, and ITS</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <tbody> <tr> <td align="left" colspan="3" rowspan="1">Adapter overhang sequences</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">Forward overhang</td> <td align="left" rowspan="1" colspan="1">5â€² TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGâ€�[locusâ€�specific sequence]</td> <td align="left" rowspan="2" colspan="1">â€œ16S Metagenomic Sequencing Library Preparationâ€� (<a href="#ece37139-bib-0001">2013</a>)</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">Reverse overhang</td> <td align="left" rowspan="1" colspan="1">5â€² GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGâ€�[locusâ€�specific sequence]</td> </tr> <tr> <td align="left" colspan="3" rowspan="1">COI Primers</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">MLepF1</td> <td align="left" rowspan="1" colspan="1">GCTTTCCCACGAATAAATAATA</td> <td align="left" rowspan="1" colspan="1">Hajibabaei etÂ al.Â (<a href="#ece37139-bib-0019">2006</a>)</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">RonMWASPdeg</td> <td align="left" rowspan="1" colspan="1">GGWTCWCCWGATATAKCWTTTCC</td> <td align="left" rowspan="1" colspan="1">M. A. Smith (unpublished); HernÃ¡ndezâ€�Triana etÂ al.Â (<a href="#ece37139-bib-0023">2014</a>)</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">LepR1</td> <td align="left" rowspan="1" colspan="1">TAAACTTCTGGATGTCCAAAAAATCA</td> <td align="left" rowspan="1" colspan="1">Hebert etÂ al.Â (<a href="#ece37139-bib-0021">2004</a>); HernÃ¡ndezâ€�Triana etÂ al.Â (<a href="#ece37139-bib-0023">2014</a>)</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">HCO2198</td> <td align="left" rowspan="1" colspan="1">TAAACTTCAGGGTGACCAAAAAATCA</td> <td align="left" rowspan="1" colspan="1">Folmer etÂ al.Â (<a href="#ece37139-bib-0015">1994</a>)</td> </tr> <tr> <td align="left" colspan="3" rowspan="1">16S Primers</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">16S Forward</td> <td align="left" rowspan="1" colspan="1">CCTACGGGNGGCWGCAG</td> <td align="left" rowspan="2" colspan="1">â€œ16S Metagenomic Sequencing Library Preparationâ€� (<a href="#ece37139-bib-0001">2013</a>)</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">16S Reverse</td> <td align="left" rowspan="1" colspan="1">GACTACHVGGGTATCTAATCC</td> </tr> <tr> <td align="left" colspan="3" rowspan="1">ITS Primers</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">ITS86_F</td> <td align="left" rowspan="1" colspan="1">GTGAATCATCGAATCTTTGAA</td> <td align="left" rowspan="2" colspan="1">Beeck etÂ al.Â (<a href="#ece37139-bib-0004">2014</a>)</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">ITS4_R</td> <td align="left" rowspan="1" colspan="1">TCCTCCGCTTATTGATATGC</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="ece37139-ntgp-0001"> <div class="title" tagx="title" title="title">
Note</div> <div class="fn-type-" title=""> <p>COI primers have been used as cocktails, combining 1 to 1 volume of the two forwards (forward cocktail) and the two reverse (reverse cocktail) primers. References are also shown.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> </div> <div class="secondpcr" title="sec"><span class="label" tagx="label" title="label">2.6.2</span><div class="title" tagx="title" title="title">
Second PCR</div> <p>The purified products resulting from the first PCR were used as the template for the second PCR. As per standard methods for eDNA library preparation, the second PCR was conducted in a separate room and using different PCR workstations than the first PCR. The second PCR used unique index primer combinations for each sample. The sequences for the index primers were equivalent to the Nextera XT Index Kit (Illumina), but synthesized de novo using the services from Integrated DNA Technologies (IDT) and prepared at Advanced Analysis Centre (AAC) at the University of Guelph. This PCR was conducted in 50Â Î¼l reaction volumes on an Eppendorf Mastercycler using a unique index primer combination for each sample. Each reaction contained 5Â Î¼l of previously cleaned PCR product, 5Â Î¼l of each index primer (10Â Î¼M), 25Â Î¼l of 2X KAPA HiFi HotStart Ready Mix (Roche Diagnostics), and 10Â Î¼l of molecular biology grade water. Cycling conditions were as follows: 95Â°C (180Â s), eight cycles at 95Â°C (30Â s), 55Â°C (40Â s), 72Â°C (30Â s), and a final extension at 72Â°C (300Â s) (â€œ16S Metagenomic Sequencing Library Preparationâ€�, <a href="#ece37139-bib-0001">2013</a>). PCR products were visualized on 1% agarose gel first and purified using a 0.6x NGS magnetic beads ratio (Machereyâ€�Nagel), following manufacturer's protocols.</p> </div> </div> <div class="highâ€�throughputsequencing" title="sec"><span class="label" tagx="label" title="label">2.7</span><div class="title" tagx="title" title="title">
Highâ€�throughput sequencing</div> <p>Sequencing of COI, 16S, and ITS marker regions was performed at the Genomics Facility of AAC at the University of Guelph. For quality control purposes, each sequence library was first normalized using SequalPrep Normalization Kit (Thermo Fisher Scientific), pooled, and quantified with the Qubit dsDNA High Sensitivity assay kit (Thermo Fisher Scientific), and finally checked for fragment size in a Bioanalyzer High Sensitivity DNA Chip (Agilent). After passing quality control, libraries were sequenced on an Illumina MiSeq System using a MiSeq reagent kit, version 3 (600 cycles). Each sample was analyzed based on retaining 1% of the total capacity of the run. Sequencing reads were demultiplexed, and the adapters trimmed with the MiSeq Reporter software generating two pairedâ€�end FASTQ raw data files.</p> </div> <div class="costanalysis" title="sec"><span class="label" tagx="label" title="label">2.8</span><div class="title" tagx="title" title="title">
Cost analysis</div> <p>We conducted a simple cost analysis that compared the cost of processing one sample (from eDNA extraction to MiSeq sequencing) based on the modified protocol described here versus the original â€œ16S Metagenomic Sequencing Library Preparation Protocolâ€� (Illumina) (TableÂ <a href="#ece37139-tbl-0002">2</a>).</p> <div class="table-wrap_UNKNOWN" id="ece37139-tbl-0002" content-type="TABLE" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 2</span> <p>Approximate cost analysis for processing one sample, including filtering, eDNA extraction, library preparation, and MiSeq sequencing at the Genomics Facility of the Advanced Analysis Centre (AAC) at the University of Guelph</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Laboratory steps for processing one sample</th> <th align="left" valign="top" rowspan="1" colspan="1">Cost per step CAD/USD<a href="#ece37139-note-0005"><sup>b</sup></a> (this study)</th> <th align="left" valign="top" rowspan="1" colspan="1">Cost per step CAD/USD<a href="#ece37139-note-0005"><sup>b</sup></a> (Illumina protocol)</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">Filters (pore size 1Â Î¼m, diameter 47Â mm; an average of 2.5 per trap)</td> <td align="left" rowspan="1" colspan="1">2.5</td> <td align="left" rowspan="1" colspan="1">2.5</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">CTAB extraction method (4 per filter)</td> <td align="left" rowspan="1" colspan="1">2.5</td> <td align="left" rowspan="1" colspan="1">2.5</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">DNA quantification (Qubit) and normalization</td> <td align="left" rowspan="1" colspan="1">2.5</td> <td align="left" rowspan="1" colspan="1">2.5</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">First PCR (pooled extracts per filter)</td> <td align="left" rowspan="1" colspan="1">5</td> <td align="left" rowspan="1" colspan="1">5</td> </tr> <tr> <td align="left" style="background-color:#A5A5A5" rowspan="1" colspan="1">Magnetic beads (at all stages)</td> <td align="left" style="background-color:#A5A5A5" rowspan="1" colspan="1">0.7</td> <td align="left" style="background-color:#A5A5A5" rowspan="1" colspan="1">7</td> </tr> <tr> <td align="left" style="background-color:#A5A5A5" rowspan="1" colspan="1">Second PCR (Indexing reaction and primers<a href="#ece37139-note-0004"><sup>a</sup></a>)</td> <td align="left" style="background-color:#A5A5A5" rowspan="1" colspan="1">7</td> <td align="left" style="background-color:#A5A5A5" rowspan="1" colspan="1">25</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">HTS Sequencing (Illumina MiSeq)</td> <td align="left" rowspan="1" colspan="1">30</td> <td align="left" rowspan="1" colspan="1">30</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Overall</td> <td align="left" rowspan="1" colspan="1">50.2 CAD/36.7 USD</td> <td align="left" rowspan="1" colspan="1">74.5 CAD/54.5 USD</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="ece37139-ntgp-0002"> <div class="title" tagx="title" title="title">
Note</div> <div class="fn-type-" title=""> <p>Processing steps are compared to library preparation costs of Illumina's 16S Metagenomic Sequencing Library Preparation (â€œ<a href="#ece37139-bib-0001">16S</a> Metagenomic Sequencing Library Preparationâ€�,Â <a href="#ece37139-bib-0001">2013</a>). Shaded cells represent steps common to both protocols where modifications were completed to reduce the protocol's total cost.</p> </div> <div class="fn-type-" title=""> <p>Nonhighlighted fields represent steps common to both protocols where no modifications were needed. The costs apply to all three molecular markers: COI, 16S, and ITS.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label"><sup>a</sup></span><p>Illumina protocol adapted by the AAC at the University of Guelph.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label"><sup>b</sup></span><p>Currency exchange rates (CAD/USD) as of 09 August 2020:1 CADÂ =Â 0.73105.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> </div> <div class="dataanalysis" title="sec"><span class="label" tagx="label" title="label">2.9</span><div class="title" tagx="title" title="title">
Data analysis</div> <p>Quality control of the FASTQ raw data files was explored using FastQC as per base sequence quality scores, per base sequence content, sequence length distribution, sequence duplication levels, overrepresented sequences, and adapter content. (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>).</p> <div class="coi" title="sec"><span class="label" tagx="label" title="label">2.9.1</span><div class="title" tagx="title" title="title">
COI</div> <p>HTS data visualization, validation, analysis, and storage were conducted in the multiuser platform Multiplex Barcode Research And Visualization Environment (mBRAVE; <a href="http://mbrave.net/">http://mbrave.net/</a>). The parameters in the online platform were set as follow: (a) trimming: trim front: 30Â bp; trim end: 30Â bp; trim length 450Â bp; (b) filtering: min QV: 20Â qv; min length: 350Â bp; max bases with low QV [Ë‚20]: 25%; max bases with ultralow QV [Ë‚10]: 5%; (c) preclustering threshold: none; ID distance threshold: 3%; minimum OTU size: 5; operational taxonomic unit (OTU) threshold: 2%; (d) assembler min overlap: 20Â bp, assembler max subs: 5Â bp. An mBRAVE algorithm for removing chimeras was also applied to the data; (e) queries against Barcode of Life (BOLD) libraries in the following order: 1.â€"SYSâ€�MBRAVEC: System Reference Libraryâ€"Standard Contaminants Based on Reagent Production; 2.â€"SYSâ€�HUMC: System Reference Libraryâ€"Human Contamination Check; 3.â€"SYSâ€�CRLBACTERIA: System Reference Library for mBRAVE ID Engineâ€"Bacteria COI; 4.â€"SYSâ€�CRLPROTISTA: System Reference Library for mBRAVE ID Engineâ€"Protista COI; 5.â€"SYSâ€�CRLCHORDATA: System Reference Library for mBRAVE ID Engineâ€"Chordata; 6.â€"SYSâ€�CRLAVES: System Reference Library for mBRAVE ID Engineâ€"Aves; 7.â€"DSâ€�PLANTP20: CFIAâ€�Plant pestsâ€�invasive species (insects) update February 2020; 8.â€"DSâ€�VBEETLES: Vectors of <i>Bretziella fagacearum</i> update February 2020; 9.â€"DSâ€�CUL2020: <i>Culicoides</i> Database Update: February 2020; 10.â€"SYSâ€�CRLINSECTA: System Reference Library for mBRAVE ID Engineâ€"Insecta; 11.â€"SYSâ€�CRLNONINSECTARTH: System Reference Library for mBRAVE ID Engineâ€"Nonâ€�Insect Arthropoda; and 12.â€"SYSâ€�CRLNONARTHINVERT: System Reference Library for mBRAVE ID Engineâ€"Nonâ€�Arthropoda Invertebrates. Sequences were matched to BINs in mBRAVE. Essentially, a BIN (Barcode Index Number) is an alphanumeric code that corresponds to a tight cluster of closely related species haplotypes. BINs are a good proxy for actual biological species (Ratnasingham &amp;amp; Hebert,Â <a href="#ece37139-bib-0034">2013</a>).</p> <p>A second pipeline for data analysis was conducted to validate the identity of the IAS recognized by mBRAVE. COI FASTQ raw data files were analyzed sequentially in Geneious Prime, version 2020.1.1 (Biomatters, Ltd.) as follows: (a) Paired reads were set; (b) sequences were trimmed with BBDuk version 37.25, keeping a minimum sequence quality of 20 Phred (i.e., 99% base call accuracy), and a minimum length of 200Â bp; (c) pairedâ€�end reads were merged with BBMerge version 37.25; (d) duplicates were removed with Dedupe version 37.25; (e) clustering was completed by de novo assembly using Geneious assembler, as well as minimum overlap identity of 98%; and (f) The Basic Local Alignment Search Tool (BLAST) was used to compare our sequence library against the National Center for Biotechnology Information (NCBI) database. Rarefaction curves expressed as the number of operational taxonomic units (OTU) identified per trap versus the number of sequences related to the OTUs identified were built in R version 3.6.1 (R Core Team,Â <a href="#ece37139-bib-0032">2019</a>).</p> </div> <div class="16s" title="sec"><span class="label" tagx="label" title="label">2.9.2</span><div class="title" tagx="title" title="title">
16S</div> <p>16S molecular marker data were analyzed sequentially using Geneious Prime, version 2020.1.1 (Biomatters, Ltd.) as follows: (a) Paired reads were set; (b) sequences were trimmed with BBDuk version 37.25, keeping a minimum sequence quality of 20 Phred (i.e., 99% base call accuracy), and a minimum length of 200Â bp; (c) pairedâ€�end reads were merged with BBMerge version 37.25; (d) duplicate sequences were removed with Dedupe version 37.25; and (e) exact sequence variants were classified using the 16S Biodiversity tool implemented in Geneious Prime, version 2020.1.1 (Biomatters, Ltd.). This tool runs the Ribosomal Database Project (RDP) Classifier, version 2.12, which assigns a taxonomy to the genus level (Wang etÂ al.,Â <a href="#ece37139-bib-0044">2007</a>) and includes a bootstrap confidence estimate for each sequence by comparing them to a bacterial and archaeal 16S rRNA database (Wang etÂ al.,Â <a href="#ece37139-bib-0044">2007</a>). Search results were queried against the list of bacterial species regulated by Canada. Graphs of bacterial diversity were produced using the Krona metagenomic visualization tool (Ondov etÂ al.,Â <a href="#ece37139-bib-0028">2011</a>).</p> </div> <div class="its" title="sec"><span class="label" tagx="label" title="label">2.9.3</span><div class="title" tagx="title" title="title">
ITS</div> <p>Similar to 16S, data analysis of ITS was carried out in Geneious Prime, version 2020.1.1 (Biomatters, Ltd.) as follows: (a) Paired reads were set; (b) sequences were trimmed with BBDuk version 37.25, keeping a minimum sequence quality of 20 Phred (i.e., 99% base call accuracy), and a minimum length of 200Â bp; (c) pairedâ€�end reads were merged with BBMerge version 37.25; (d) duplicates were removed with Dedupe version 37.25; (e) chimera reads were removed using UNITE as a reference library; (f) clustering was done by de novo assembly using Geneious assembler and custom sensitivity, as well as minimum overlap identity of 98%; and (g) The Basic Local Alignment Search Tool (BLAST) was used to compare our sequence library against the Internal Transcribed Spacer database of the National Center for Biotechnology Information (NCBI). Search results were ordered by Eâ€�value (in ascending order) and then filtered by sequence length (â‰¥200Â bp) and percentage of pairwise identity (â‰¥98%). The filtered list was queried against the list of fungal species regulated by Canada. Graphs of fungal diversity were generated for the most speciesâ€�rich genera (â‰¥10 species) using R (R Core Team,Â <a href="#ece37139-bib-0032">2019</a>).</p> </div> </div> </div> <div class="results" title="results"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
RESULTS</div> <div class="coi" title="sec"><span class="label" tagx="label" title="label">3.1</span><div class="title" tagx="title" title="title">
COI</div> <p>All samples amplified successfully with the primer combination chosen. Fragment sizes were at the expected length of approximately 407 nucleotides for all PCR steps during library preparation. The Illumina MiSeq run generated a total of 9,034,668 reads for the 48 traps analyzed in this study with the number of reads per sample ranging from 102,571 to 338,642 (Table <a href="#ece37139-tbl-0003">3</a>). The percentage of filtered reads was lower than 25% for all the traps except for two (WPâ€�1â€�18â€�7â€�18_COI, 60.43% and BAâ€�6_19â€�7â€�18_COI, 25.73%) (TableÂ <a href="#ece37139-tbl-0003">3</a>). The number of suspected species present through the identification of unique BINs per trap varied from 60 to 313. The lowest number of BINs was attributed to traps placed at the Barrie (BA) location (BAâ€�1_24â€�8â€�18_COI, BAâ€�3_24â€�8â€�18_COI, and BAâ€�6_24â€�8â€�18_COI collected on August 2018) (Table <a href="#ece37139-tbl-0003">3</a>). The highest number of BINs was recovered in traps placed at the Woodstock (TO) location (TOâ€�1â€�30â€�07â€�2018_COI, TOâ€�5â€�30â€�07â€�2018_COI, and TOâ€�6â€�30â€�07â€�2018_COI collected on July 2018). The number of remaining OTUs inferred from the data ranged from 36 to 469 (Table <a href="#ece37139-tbl-0003">3</a>). Rarefaction curves of total OTU counts versus the number of sequences calculated for each trap reached asymptote in all cases (FigureÂ <a href="#ece37139-fig-0002">2</a>).</p> <div class="table-wrap_UNKNOWN" id="ece37139-tbl-0003" content-type="TABLE" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 3</span> <p>The number of reads, BINs, and sequences associated with BIN identification, remaining OTU counts, percentage of filtered reads (% Filtered), and percentage of dereplicated reads (% Dereplicated) per sample for COI</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Sample name</th> <th align="left" valign="top" rowspan="1" colspan="1">Reads</th> <th align="left" valign="top" rowspan="1" colspan="1">BINs</th> <th align="left" valign="top" rowspan="1" colspan="1">Sequences</th> <th align="left" valign="top" rowspan="1" colspan="1">OTU Counts</th> <th align="left" valign="top" rowspan="1" colspan="1">% Filtered</th> <th align="left" valign="top" rowspan="1" colspan="1">% Dereplicated</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�1â€�03â€�07â€�2018â€�COI</td> <td align="char" char="." rowspan="1" colspan="1">148,806</td> <td align="char" char="." rowspan="1" colspan="1">181</td> <td align="char" char="." rowspan="1" colspan="1">125,219</td> <td align="char" char="." rowspan="1" colspan="1">218</td> <td align="left" rowspan="1" colspan="1">2.45</td> <td align="char" char="." rowspan="1" colspan="1">69.19</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�2â€�03â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">152,732</td> <td align="char" char="." rowspan="1" colspan="1">217</td> <td align="char" char="." rowspan="1" colspan="1">124,416</td> <td align="char" char="." rowspan="1" colspan="1">163</td> <td align="left" rowspan="1" colspan="1">2.43</td> <td align="char" char="." rowspan="1" colspan="1">70.22</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�3â€�03â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">135,306</td> <td align="char" char="." rowspan="1" colspan="1">187</td> <td align="char" char="." rowspan="1" colspan="1">89,030</td> <td align="char" char="." rowspan="1" colspan="1">165</td> <td align="left" rowspan="1" colspan="1">1.18</td> <td align="char" char="." rowspan="1" colspan="1">69.2</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�4â€�03â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">137,956</td> <td align="char" char="." rowspan="1" colspan="1">177</td> <td align="char" char="." rowspan="1" colspan="1">115,974</td> <td align="char" char="." rowspan="1" colspan="1">117</td> <td align="left" rowspan="1" colspan="1">0.7</td> <td align="char" char="." rowspan="1" colspan="1">67.74</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�5â€�03â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">135,926</td> <td align="char" char="." rowspan="1" colspan="1">203</td> <td align="char" char="." rowspan="1" colspan="1">99,710</td> <td align="char" char="." rowspan="1" colspan="1">145</td> <td align="left" rowspan="1" colspan="1">1.77</td> <td align="char" char="." rowspan="1" colspan="1">66.23</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�6â€�03â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">148,600</td> <td align="char" char="." rowspan="1" colspan="1">195</td> <td align="char" char="." rowspan="1" colspan="1">112,480</td> <td align="char" char="." rowspan="1" colspan="1">163</td> <td align="left" rowspan="1" colspan="1">4.05</td> <td align="char" char="." rowspan="1" colspan="1">68.96</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�1â€�18â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">160,544</td> <td align="char" char="." rowspan="1" colspan="1">83</td> <td align="char" char="." rowspan="1" colspan="1">5,132</td> <td align="char" char="." rowspan="1" colspan="1">36</td> <td align="left" rowspan="1" colspan="1">60.43</td> <td align="char" char="." rowspan="1" colspan="1">78.78</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�2â€�18â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">170,393</td> <td align="char" char="." rowspan="1" colspan="1">159</td> <td align="char" char="." rowspan="1" colspan="1">115,071</td> <td align="char" char="." rowspan="1" colspan="1">117</td> <td align="left" rowspan="1" colspan="1">1.71</td> <td align="char" char="." rowspan="1" colspan="1">70.7</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�3â€�18â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">171,886</td> <td align="char" char="." rowspan="1" colspan="1">205</td> <td align="char" char="." rowspan="1" colspan="1">109,977</td> <td align="char" char="." rowspan="1" colspan="1">188</td> <td align="left" rowspan="1" colspan="1">4</td> <td align="char" char="." rowspan="1" colspan="1">67.16</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�4â€�18â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">144,228</td> <td align="char" char="." rowspan="1" colspan="1">190</td> <td align="char" char="." rowspan="1" colspan="1">90,246</td> <td align="char" char="." rowspan="1" colspan="1">105</td> <td align="left" rowspan="1" colspan="1">2.3</td> <td align="char" char="." rowspan="1" colspan="1">66.64</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�5â€�18â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">141,555</td> <td align="char" char="." rowspan="1" colspan="1">104</td> <td align="char" char="." rowspan="1" colspan="1">85,013</td> <td align="char" char="." rowspan="1" colspan="1">37</td> <td align="left" rowspan="1" colspan="1">11.43</td> <td align="char" char="." rowspan="1" colspan="1">79.07</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�6â€�18â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">156,955</td> <td align="char" char="." rowspan="1" colspan="1">172</td> <td align="char" char="." rowspan="1" colspan="1">96,160</td> <td align="char" char="." rowspan="1" colspan="1">60</td> <td align="left" rowspan="1" colspan="1">8.5</td> <td align="char" char="." rowspan="1" colspan="1">69.37</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�1_19â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">188,612</td> <td align="char" char="." rowspan="1" colspan="1">123</td> <td align="char" char="." rowspan="1" colspan="1">129,004</td> <td align="char" char="." rowspan="1" colspan="1">233</td> <td align="left" rowspan="1" colspan="1">8.24</td> <td align="char" char="." rowspan="1" colspan="1">70.17</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�2_19â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">200,026</td> <td align="char" char="." rowspan="1" colspan="1">125</td> <td align="char" char="." rowspan="1" colspan="1">142,661</td> <td align="char" char="." rowspan="1" colspan="1">300</td> <td align="left" rowspan="1" colspan="1">13.69</td> <td align="char" char="." rowspan="1" colspan="1">69.76</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�3_19â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">189,626</td> <td align="char" char="." rowspan="1" colspan="1">191</td> <td align="char" char="." rowspan="1" colspan="1">112,161</td> <td align="char" char="." rowspan="1" colspan="1">259</td> <td align="left" rowspan="1" colspan="1">18.34</td> <td align="char" char="." rowspan="1" colspan="1">67.4</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�4_19â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">200,985</td> <td align="char" char="." rowspan="1" colspan="1">166</td> <td align="char" char="." rowspan="1" colspan="1">134,391</td> <td align="char" char="." rowspan="1" colspan="1">194</td> <td align="left" rowspan="1" colspan="1">13.02</td> <td align="char" char="." rowspan="1" colspan="1">72.98</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�5_19â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">308,781</td> <td align="char" char="." rowspan="1" colspan="1">116</td> <td align="char" char="." rowspan="1" colspan="1">237,742</td> <td align="char" char="." rowspan="1" colspan="1">160</td> <td align="left" rowspan="1" colspan="1">9.48</td> <td align="char" char="." rowspan="1" colspan="1">78.77</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�6_19â€�7â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">338,642</td> <td align="char" char="." rowspan="1" colspan="1">189</td> <td align="char" char="." rowspan="1" colspan="1">190,311</td> <td align="char" char="." rowspan="1" colspan="1">410</td> <td align="left" rowspan="1" colspan="1">25.73</td> <td align="char" char="." rowspan="1" colspan="1">70.82</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�1â€�30â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">149,318</td> <td align="char" char="." rowspan="1" colspan="1">217</td> <td align="char" char="." rowspan="1" colspan="1">106,971</td> <td align="char" char="." rowspan="1" colspan="1">254</td> <td align="left" rowspan="1" colspan="1">3.84</td> <td align="char" char="." rowspan="1" colspan="1">67.71</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�2â€�30â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">133,566</td> <td align="char" char="." rowspan="1" colspan="1">195</td> <td align="char" char="." rowspan="1" colspan="1">76,130</td> <td align="char" char="." rowspan="1" colspan="1">358</td> <td align="left" rowspan="1" colspan="1">8.76</td> <td align="char" char="." rowspan="1" colspan="1">66.99</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�3â€�30â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">139,968</td> <td align="char" char="." rowspan="1" colspan="1">128</td> <td align="char" char="." rowspan="1" colspan="1">83,041</td> <td align="char" char="." rowspan="1" colspan="1">320</td> <td align="left" rowspan="1" colspan="1">5.04</td> <td align="char" char="." rowspan="1" colspan="1">70.31</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�4â€�30â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">153,723</td> <td align="char" char="." rowspan="1" colspan="1">135</td> <td align="char" char="." rowspan="1" colspan="1">102,129</td> <td align="char" char="." rowspan="1" colspan="1">243</td> <td align="left" rowspan="1" colspan="1">3.82</td> <td align="char" char="." rowspan="1" colspan="1">69.19</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�5â€�30â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">138,016</td> <td align="char" char="." rowspan="1" colspan="1">284</td> <td align="char" char="." rowspan="1" colspan="1">112,626</td> <td align="char" char="." rowspan="1" colspan="1">224</td> <td align="left" rowspan="1" colspan="1">3.4</td> <td align="char" char="." rowspan="1" colspan="1">67.78</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�6â€�30â€�07â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">154,558</td> <td align="char" char="." rowspan="1" colspan="1">313</td> <td align="char" char="." rowspan="1" colspan="1">107,918</td> <td align="char" char="." rowspan="1" colspan="1">316</td> <td align="left" rowspan="1" colspan="1">10.94</td> <td align="char" char="." rowspan="1" colspan="1">66.3</td> </tr> </tbody> </table> <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" colspan="7" style="border-bottom:solid 1px #000000" valign="top" rowspan="1">Number of reads, BINs, sequences (Seqs) associated with BIN identification, remaining OTU counts, percentage of filtered reads (Filtered %), and percentage of dereplicated reads (Dereplicated %) per sample for COI</th> </tr> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Sample name</th> <th align="left" valign="top" rowspan="1" colspan="1">Reads</th> <th align="left" valign="top" rowspan="1" colspan="1">BINs</th> <th align="left" valign="top" rowspan="1" colspan="1">Seqs</th> <th align="left" valign="top" rowspan="1" colspan="1">OTU Counts</th> <th align="left" valign="top" rowspan="1" colspan="1">Filtered %</th> <th align="left" valign="top" rowspan="1" colspan="1">Dereplicated %</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�1_16â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">172,717</td> <td align="char" char="." rowspan="1" colspan="1">194</td> <td align="char" char="." rowspan="1" colspan="1">140,527</td> <td align="char" char="." rowspan="1" colspan="1">158</td> <td align="char" char="." rowspan="1" colspan="1">6.11</td> <td align="char" char="." rowspan="1" colspan="1">68.83</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�2_16â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">212,186</td> <td align="char" char="." rowspan="1" colspan="1">213</td> <td align="char" char="." rowspan="1" colspan="1">165,688</td> <td align="char" char="." rowspan="1" colspan="1">203</td> <td align="char" char="." rowspan="1" colspan="1">5.27</td> <td align="char" char="." rowspan="1" colspan="1">68.54</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�3_16â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">254,985</td> <td align="char" char="." rowspan="1" colspan="1">188</td> <td align="char" char="." rowspan="1" colspan="1">204,726</td> <td align="char" char="." rowspan="1" colspan="1">194</td> <td align="char" char="." rowspan="1" colspan="1">3.58</td> <td align="char" char="." rowspan="1" colspan="1">69.25</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�4_16â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">253,413</td> <td align="char" char="." rowspan="1" colspan="1">217</td> <td align="char" char="." rowspan="1" colspan="1">212,061</td> <td align="char" char="." rowspan="1" colspan="1">135</td> <td align="char" char="." rowspan="1" colspan="1">2.7</td> <td align="char" char="." rowspan="1" colspan="1">69.08</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�5_16â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">200,655</td> <td align="char" char="." rowspan="1" colspan="1">134</td> <td align="char" char="." rowspan="1" colspan="1">144,174</td> <td align="char" char="." rowspan="1" colspan="1">98</td> <td align="char" char="." rowspan="1" colspan="1">15.66</td> <td align="char" char="." rowspan="1" colspan="1">72</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">GTâ€�6_16â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">142,747</td> <td align="char" char="." rowspan="1" colspan="1">142</td> <td align="char" char="." rowspan="1" colspan="1">99,940</td> <td align="char" char="." rowspan="1" colspan="1">162</td> <td align="char" char="." rowspan="1" colspan="1">14.57</td> <td align="char" char="." rowspan="1" colspan="1">68.85</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�1_22â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">263,437</td> <td align="char" char="." rowspan="1" colspan="1">157</td> <td align="char" char="." rowspan="1" colspan="1">143,486</td> <td align="char" char="." rowspan="1" colspan="1">347</td> <td align="char" char="." rowspan="1" colspan="1">4.13</td> <td align="char" char="." rowspan="1" colspan="1">69.94</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�2_22â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">330,301</td> <td align="char" char="." rowspan="1" colspan="1">160</td> <td align="char" char="." rowspan="1" colspan="1">225,989</td> <td align="char" char="." rowspan="1" colspan="1">321</td> <td align="char" char="." rowspan="1" colspan="1">0.45</td> <td align="char" char="." rowspan="1" colspan="1">68.26</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�3_22â€�8â€�18_COI_</td> <td align="char" char="." rowspan="1" colspan="1">276,045</td> <td align="char" char="." rowspan="1" colspan="1">205</td> <td align="char" char="." rowspan="1" colspan="1">103,801</td> <td align="char" char="." rowspan="1" colspan="1">452</td> <td align="char" char="." rowspan="1" colspan="1">3.7</td> <td align="char" char="." rowspan="1" colspan="1">66.22</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�4_22â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">267,173</td> <td align="char" char="." rowspan="1" colspan="1">139</td> <td align="char" char="." rowspan="1" colspan="1">177,975</td> <td align="char" char="." rowspan="1" colspan="1">393</td> <td align="char" char="." rowspan="1" colspan="1">3.37</td> <td align="char" char="." rowspan="1" colspan="1">69.25</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�5_22â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">256,504</td> <td align="char" char="." rowspan="1" colspan="1">158</td> <td align="char" char="." rowspan="1" colspan="1">191,390</td> <td align="char" char="." rowspan="1" colspan="1">239</td> <td align="char" char="." rowspan="1" colspan="1">1.34</td> <td align="char" char="." rowspan="1" colspan="1">71.55</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WPâ€�6_22â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">243,309</td> <td align="char" char="." rowspan="1" colspan="1">121</td> <td align="char" char="." rowspan="1" colspan="1">123,239</td> <td align="char" char="." rowspan="1" colspan="1">82</td> <td align="char" char="." rowspan="1" colspan="1">3.56</td> <td align="char" char="." rowspan="1" colspan="1">76.1</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�1_24â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">145,523</td> <td align="char" char="." rowspan="1" colspan="1">72</td> <td align="char" char="." rowspan="1" colspan="1">62,197</td> <td align="char" char="." rowspan="1" colspan="1">387</td> <td align="char" char="." rowspan="1" colspan="1">14.63</td> <td align="char" char="." rowspan="1" colspan="1">66.75</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�2_24â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">212,628</td> <td align="char" char="." rowspan="1" colspan="1">88</td> <td align="char" char="." rowspan="1" colspan="1">151,481</td> <td align="char" char="." rowspan="1" colspan="1">313</td> <td align="char" char="." rowspan="1" colspan="1">7.74</td> <td align="char" char="." rowspan="1" colspan="1">71.34</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�3_24â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">115,902</td> <td align="char" char="." rowspan="1" colspan="1">63</td> <td align="char" char="." rowspan="1" colspan="1">13,586</td> <td align="char" char="." rowspan="1" colspan="1">469</td> <td align="char" char="." rowspan="1" colspan="1">23.08</td> <td align="char" char="." rowspan="1" colspan="1">62.55</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�4_24â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">102,571</td> <td align="char" char="." rowspan="1" colspan="1">89</td> <td align="char" char="." rowspan="1" colspan="1">69,278</td> <td align="char" char="." rowspan="1" colspan="1">277</td> <td align="char" char="." rowspan="1" colspan="1">7.83</td> <td align="char" char="." rowspan="1" colspan="1">69.16</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�5_24â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">188,706</td> <td align="char" char="." rowspan="1" colspan="1">89</td> <td align="char" char="." rowspan="1" colspan="1">129,043</td> <td align="char" char="." rowspan="1" colspan="1">312</td> <td align="char" char="." rowspan="1" colspan="1">8.02</td> <td align="char" char="." rowspan="1" colspan="1">69.69</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">BAâ€�6_24â€�8â€�18_COI</td> <td align="char" char="." rowspan="1" colspan="1">133,081</td> <td align="char" char="." rowspan="1" colspan="1">60</td> <td align="char" char="." rowspan="1" colspan="1">58,736</td> <td align="char" char="." rowspan="1" colspan="1">339</td> <td align="char" char="." rowspan="1" colspan="1">12.07</td> <td align="char" char="." rowspan="1" colspan="1">67.85</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�1_28â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">175,589</td> <td align="char" char="." rowspan="1" colspan="1">105</td> <td align="char" char="." rowspan="1" colspan="1">139,710</td> <td align="char" char="." rowspan="1" colspan="1">179</td> <td align="char" char="." rowspan="1" colspan="1">1.7</td> <td align="char" char="." rowspan="1" colspan="1">71.14</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�2_28â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">167,549</td> <td align="char" char="." rowspan="1" colspan="1">112</td> <td align="char" char="." rowspan="1" colspan="1">100,841</td> <td align="char" char="." rowspan="1" colspan="1">271</td> <td align="char" char="." rowspan="1" colspan="1">9.11</td> <td align="char" char="." rowspan="1" colspan="1">68.75</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�3_28â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">189,358</td> <td align="char" char="." rowspan="1" colspan="1">103</td> <td align="char" char="." rowspan="1" colspan="1">120,406</td> <td align="char" char="." rowspan="1" colspan="1">356</td> <td align="char" char="." rowspan="1" colspan="1">1.51</td> <td align="char" char="." rowspan="1" colspan="1">68.69</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�4_28â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">167,074</td> <td align="char" char="." rowspan="1" colspan="1">106</td> <td align="char" char="." rowspan="1" colspan="1">145,180</td> <td align="char" char="." rowspan="1" colspan="1">160</td> <td align="char" char="." rowspan="1" colspan="1">1.44</td> <td align="char" char="." rowspan="1" colspan="1">69.33</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�5_28â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">231,638</td> <td align="char" char="." rowspan="1" colspan="1">112</td> <td align="char" char="." rowspan="1" colspan="1">189,424</td> <td align="char" char="." rowspan="1" colspan="1">224</td> <td align="char" char="." rowspan="1" colspan="1">4.32</td> <td align="char" char="." rowspan="1" colspan="1">72.79</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOâ€�6_28â€�8â€�2018_COI</td> <td align="char" char="." rowspan="1" colspan="1">230,869</td> <td align="char" char="." rowspan="1" colspan="1">112</td> <td align="char" char="." rowspan="1" colspan="1">181,460</td> <td align="char" char="." rowspan="1" colspan="1">102</td> <td align="char" char="." rowspan="1" colspan="1">10.65</td> <td align="char" char="." rowspan="1" colspan="1">71.69</td> </tr> </tbody> </table> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Rarefaction curves representing the number of operational taxonomic units (OTUs) versus the number of corresponding sequences for each sample site in July 2018 (a) and August 2018 (b)</p>   </div> <p>Overall, 2,535 BINs were identified across the 48 traps, representing 57 Orders and 304 Families, the vast majority of them being arthropods (AppendixÂ <a href="#ece37139-sup-0001">S1</a>). A total of 5,997,851 sequences were associated with a BIN and together represented an average quality value (QV) score of over 37 out of a total possible score of 40.</p> </div> <div class="comparativebinanalysis" title="sec"><span class="label" tagx="label" title="label">3.2</span><div class="title" tagx="title" title="title">
Comparative BIN analysis</div> <div class="trapscollectedinjuly2018" title="sec"><span class="label" tagx="label" title="label">3.2.1</span><div class="title" tagx="title" title="title">
Traps collected in July 2018</div> <p>A total of 727 BINs were identified across traps placed at the Halton Hills (GT) location, representing 27 Orders and 139 Families. Eleven out of the 727 BINs were shared among the six traps (Figure <a href="#ece37139-fig-0003">3</a>). At the Chathamâ€�Kent (WP) location, a total of 583 BINs were identified for a total of 29 Orders and 132 Families. Nine out of the 583 BINs were shared among the six traps (Figure <a href="#ece37139-fig-0003">3</a>). At the Barrie (BA) location, a total number of 604 BINs were recognized and distributed among 30 Orders and 140 Families, with nine BINs shared among traps. At Woodstock (TO), 870 BINs were identified; five of them were common among the six traps (Figure <a href="#ece37139-fig-0003">3</a>). Among the 870 BINs, a total of 43 Orders and 183 Families were recovered. Overall, 1,876 BINs were classified in traps placed on southern Ontario, Canada, sites in July 2018. The BINs were distributed among 49 Orders and 261 Families.</p> </div> <div class="trapscollectedinaugust2018" title="sec"><span class="label" tagx="label" title="label">3.2.2</span><div class="title" tagx="title" title="title">
Traps collected in August 2018</div> <p>A total of 701 BINs were identified across traps placed at the Halton Hills (GT) location, with 12 of them shared among traps. Overall, they represented 33 Orders and 141 Families (Figure 3). At Chathamâ€�Kent (WP), ten BINs were common among traps, with a total of 600 BINs identified and distributed across 34 Orders and 149 Families (Figure 3). A total of 314 BINs were recognized at the Barrie (BA) location, with two of them shared among traps. The 314 BINs were distributed among 26 Orders and 85 Families (Figure <a href="#ece37139-fig-0003">3</a>). At Woodstock (TO), 432 BINs were identified, representing 32 Orders and 112 Families. Six out of the 432 BINs were common among the traps (Figure <a href="#ece37139-fig-0003">3</a>).</p> <p>In general, for the samples collected in August 2018, a total of 1,360 BINs were identified, and all of them were distributed among 45 Orders and 218 Families.</p> </div> </div> <div class="identificationofplantpestsandias" title="sec"><span class="label" tagx="label" title="label">3.3</span><div class="title" tagx="title" title="title">
Identification of plant pests and IAS</div> <div class="regulatedinsects" title="sec"><span class="label" tagx="label" title="label">3.3.1</span><div class="title" tagx="title" title="title">
Regulated insects</div> <p>Speciesâ€�specific sequence identification was completed with over 97% average mean similarity and over a minimum COI sequence length of 392Â bp (Table <a href="#ece37139-tbl-0004">4</a>). DNA sequences of two invasive plant pest species regulated by the CFIA (<i>Lymantria dispar</i> [gypsy moth] and <i>A.Â planipennis</i> [emerald ash borer beetle]) were detected at the sampling locations, which is not unexpected as the species are known to occur at these locations. DNA sequences of the gypsy moth were also recovered at every sampling location in both collection months. On the contrary, DNA sequences of the emerald ash borer beetle were recovered at only two sampling locations in July (Barrie [BA] and Woodstock [TO]), as well as in August (Halton Hills [GT] and Barrie [BA]) (Table <a href="#ece37139-tbl-0004">4</a>). The highest number of gypsy moth sequences was recovered in August at the Halton Hills location. For the emerald ash borer beetle, the highest number of DNA sequences was obtained at the Barrie location in July 2018.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 3</span></a></div>   <p>Number of BINs identified per trap and overlapping BINs. Each colored triangle represents one of six traps per location (Halton Hills [GT], Chathamâ€�Kent [WP], Barrie [BA], and Woodstock [TO]) and date sampled in July and August 2018. The number of overlapping BINs among traps per location and date is indicated in the intercepting area for each representation</p>   </div> <div class="table-wrap_UNKNOWN" id="ece37139-tbl-0004" content-type="TABLE" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 4</span> <p>Taxonomy of IAS that impact plant health identified per location.</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">BIN.Taxon.ID</th> <th align="left" valign="top" rowspan="1" colspan="1">Phylum</th> <th align="left" valign="top" rowspan="1" colspan="1">Class</th> <th align="left" valign="top" rowspan="1" colspan="1">Order</th> <th align="left" valign="top" rowspan="1" colspan="1">Family</th> <th align="left" valign="top" rowspan="1" colspan="1">Genus</th> <th align="left" valign="top" rowspan="1" colspan="1">Species</th> <th align="left" valign="top" rowspan="1" colspan="1">Seq</th> <th align="left" valign="top" rowspan="1" colspan="1">% MS</th> <th align="left" valign="top" rowspan="1" colspan="1">Length</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="10" rowspan="1">GT 3â€�7â€�2018</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">BOLD:AAA2052</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Lepidoptera</td> <td align="left" rowspan="1" colspan="1">Erebidae</td> <td align="left" rowspan="1" colspan="1"><i>Lymantria</i></td> <td align="left" rowspan="1" colspan="1"><i>Lymantria dispar</i></td> <td align="char" char="." rowspan="1" colspan="1">14,434</td> <td align="char" char="." rowspan="1" colspan="1">99.78</td> <td align="char" char="." rowspan="1" colspan="1">396.04</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">TAX:177812</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Lepidoptera</td> <td align="left" rowspan="1" colspan="1">Erebidae</td> <td align="left" rowspan="1" colspan="1"><i>Lymantria</i></td> <td align="left" rowspan="1" colspan="1"><i>Lymantria dispar</i></td> <td align="char" char="." rowspan="1" colspan="1">1</td> <td align="char" char="." rowspan="1" colspan="1">98.39</td> <td align="char" char="." rowspan="1" colspan="1">395</td> </tr> <tr> <td align="left" colspan="10" rowspan="1">WP 18â€�7â€�2018</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">BOLD:AAA2052</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Lepidoptera</td> <td align="left" rowspan="1" colspan="1">Erebidae</td> <td align="left" rowspan="1" colspan="1"><i>Lymantria</i></td> <td align="left" rowspan="1" colspan="1"><i>Lymantria dispar</i></td> <td align="char" char="." rowspan="1" colspan="1">54</td> <td align="char" char="." rowspan="1" colspan="1">99.34</td> <td align="char" char="." rowspan="1" colspan="1">395.19</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">TAX:24220</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Lepidoptera</td> <td align="left" rowspan="1" colspan="1">Erebidae</td> <td align="left" rowspan="1" colspan="1"><i>Lymantria</i></td> <td align="left" rowspan="1" colspan="1"><i>Lymantria dispar</i></td> <td align="char" char="." rowspan="1" colspan="1">1</td> <td align="char" char="." rowspan="1" colspan="1">98.73</td> <td align="char" char="." rowspan="1" colspan="1">395</td> </tr> <tr> <td align="left" colspan="10" rowspan="1">BA 19â€�7â€�2018</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">BOLD:AAE6821</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Coleoptera</td> <td align="left" rowspan="1" colspan="1">Buprestidae</td> <td align="left" rowspan="1" colspan="1"><i>Agrilus</i></td> <td align="left" rowspan="1" colspan="1"><i>Agrilus planipennis</i></td> <td align="char" char="." rowspan="1" colspan="1">214</td> <td align="char" char="." rowspan="1" colspan="1">99.55</td> <td align="char" char="." rowspan="1" colspan="1">394.94</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">TAX:27304</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Coleoptera</td> <td align="left" rowspan="1" colspan="1">Buprestidae</td> <td align="left" rowspan="1" colspan="1"><i>Agrilus</i></td> <td align="left" rowspan="1" colspan="1"><i>Agrilus planipennis</i></td> <td align="char" char="." rowspan="1" colspan="1">4,234</td> <td align="char" char="." rowspan="1" colspan="1">99.73</td> <td align="char" char="." rowspan="1" colspan="1">394.93</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">BOLD:AAA2052</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Lepidoptera</td> <td align="left" rowspan="1" colspan="1">Erebidae</td> <td align="left" rowspan="1" colspan="1"><i>Lymantria</i></td> <td align="left" rowspan="1" colspan="1"><i>Lymantria dispar</i></td> <td align="char" char="." rowspan="1" colspan="1">1</td> <td align="char" char="." rowspan="1" colspan="1">98.48</td> <td align="char" char="." rowspan="1" colspan="1">395</td> </tr> <tr> <td align="left" colspan="10" rowspan="1">TO 30â€�7â€�2018</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">TAX:27304</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Coleoptera</td> <td align="left" rowspan="1" colspan="1">Buprestidae</td> <td align="left" rowspan="1" colspan="1"><i>Agrilus</i></td> <td align="left" rowspan="1" colspan="1"><i>Agrilus planipennis</i></td> <td align="char" char="." rowspan="1" colspan="1">19</td> <td align="char" char="." rowspan="1" colspan="1">99.92</td> <td align="char" char="." rowspan="1" colspan="1">394.89</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">BOLD:AAE6821</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Coleoptera</td> <td align="left" rowspan="1" colspan="1">Buprestidae</td> <td align="left" rowspan="1" colspan="1"><i>Agrilus</i></td> <td align="left" rowspan="1" colspan="1"><i>Agrilus planipennis</i></td> <td align="char" char="." rowspan="1" colspan="1">1</td> <td align="char" char="." rowspan="1" colspan="1">98.73</td> <td align="char" char="." rowspan="1" colspan="1">395</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">BOLD:AAA2052</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Lepidoptera</td> <td align="left" rowspan="1" colspan="1">Erebidae</td> <td align="left" rowspan="1" colspan="1"><i>Lymantria</i></td> <td align="left" rowspan="1" colspan="1"><i>Lymantria dispar</i></td> <td align="char" char="." rowspan="1" colspan="1">138</td> <td align="char" char="." rowspan="1" colspan="1">99.52</td> <td align="char" char="." rowspan="1" colspan="1">398.38</td> </tr> <tr> <td align="left" colspan="10" rowspan="1">GT 16â€�8â€�2018</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">TAX:27304</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Coleoptera</td> <td align="left" rowspan="1" colspan="1">Buprestidae</td> <td align="left" rowspan="1" colspan="1"><i>Agrilus</i></td> <td align="left" rowspan="1" colspan="1"><i>Agrilus planipennis</i></td> <td align="char" char="." rowspan="1" colspan="1">1</td> <td align="char" char="." rowspan="1" colspan="1">97.46</td> <td align="char" char="." rowspan="1" colspan="1">392</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">BOLD:AAA2052</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Lepidoptera</td> <td align="left" rowspan="1" colspan="1">Erebidae</td> <td align="left" rowspan="1" colspan="1"><i>Lymantria</i></td> <td align="left" rowspan="1" colspan="1"><i>Lymantria dispar</i></td> <td align="char" char="." rowspan="1" colspan="1">29,252</td> <td align="char" char="." rowspan="1" colspan="1">99.67</td> <td align="char" char="." rowspan="1" colspan="1">395.04</td> </tr> <tr> <td align="left" colspan="10" rowspan="1">WP 22â€�8â€�2018</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">BOLD:AAA2052</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Lepidoptera</td> <td align="left" rowspan="1" colspan="1">Erebidae</td> <td align="left" rowspan="1" colspan="1"><i>Lymantria</i></td> <td align="left" rowspan="1" colspan="1"><i>Lymantria dispar</i></td> <td align="char" char="." rowspan="1" colspan="1">13</td> <td align="char" char="." rowspan="1" colspan="1">99.93</td> <td align="char" char="." rowspan="1" colspan="1">395</td> </tr> <tr> <td align="left" colspan="10" rowspan="1">BA 24â€�8â€�2018</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">TAX:27304</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Coleoptera</td> <td align="left" rowspan="1" colspan="1">Buprestidae</td> <td align="left" rowspan="1" colspan="1"><i>Agrilus</i></td> <td align="left" rowspan="1" colspan="1"><i>Agrilus planipennis</i></td> <td align="char" char="." rowspan="1" colspan="1">1</td> <td align="char" char="." rowspan="1" colspan="1">99.49</td> <td align="char" char="." rowspan="1" colspan="1">395</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">BOLD:AAA2052</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Lepidoptera</td> <td align="left" rowspan="1" colspan="1">Erebidae</td> <td align="left" rowspan="1" colspan="1"><i>Lymantria</i></td> <td align="left" rowspan="1" colspan="1"><i>Lymantria dispar</i></td> <td align="char" char="." rowspan="1" colspan="1">2</td> <td align="char" char="." rowspan="1" colspan="1">99.50</td> <td align="char" char="." rowspan="1" colspan="1">395</td> </tr> <tr> <td align="left" colspan="10" rowspan="1">TO 28â€�8â€�2018</td> </tr> <tr> <td align="left" style="padding-left:10%" rowspan="1" colspan="1">BOLD:AAA2052</td> <td align="left" rowspan="1" colspan="1">Arthropoda</td> <td align="left" rowspan="1" colspan="1">Insecta</td> <td align="left" rowspan="1" colspan="1">Lepidoptera</td> <td align="left" rowspan="1" colspan="1">Erebidae</td> <td align="left" rowspan="1" colspan="1"><i>Lymantria</i></td> <td align="left" rowspan="1" colspan="1"><i>Lymantria dispar</i></td> <td align="char" char="." rowspan="1" colspan="1">2</td> <td align="char" char="." rowspan="1" colspan="1">99.37</td> <td align="char" char="." rowspan="1" colspan="1">395.5</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="ece37139-ntgp-0003"> <div class="title" tagx="title" title="title">
Note</div> <div class="fn-type-" title=""> <p>Included are the reference library's BIN taxon identifier (BIN.Taxon.ID), number of corresponding DNA sequences (Seq), average percent mean similarity (% MS), and average bp length analyzed (Length).</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> </div> <div class="bacteria" title="sec"><span class="label" tagx="label" title="label">3.3.2</span><div class="title" tagx="title" title="title">
Bacteria</div> <p>Five (<i>Brenneria</i>, <i>Pseudomonas</i>, <i>Xanthomonas, Xylella</i>, and <i>Xylophilus</i>) out of the seven genera containing species regulated by Canada were identified across collection sites in Ontario and comprised 2,061,117 sequences (<a href="https://16s.geneious.com/16s/results/c8f7bebb-1095-4a1f-86e5-0262cd49e7ab.html">https://16s.geneious.com/16s/results/c8f7bebbâ€�1095â€�4a1fâ€�86e5â€�0262cd49e7ab.html</a>). Among the five genera, sequence count and mean confidence ranged from 47 to 103,730 and 27.09 and 93.08, respectively (FigureÂ <a href="#ece37139-fig-0004">4</a>). <i>Pseudomonas</i> and <i>Xanthomonas</i> exhibited the highest mean confidence values for sequence classification, while <i>Xylella</i> showed the lowest value. <i>Pseudomonas</i> was also the most represented genus in terms of sequence count (FigureÂ <a href="#ece37139-fig-0004">4</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 4</span></a></div>   <p>Graph representing bacterial diversity per sampling location and collection date. The color gradient represents mean confidence, as indicated in the gradient bar. An interactive pie chart showing the classification of the sequences and including detailed information for each section of the pie can be viewed online. The list of links per collection sites is indicated below. a: GT 3â€�7â€�2018 (<a href="https://16s.geneious.com/16s/results/fa8ab1e3-4f50-4985-ac59-7302a31b8451.html">https://16s.geneious.com/16s/results/fa8ab1e3â€�4f50â€�4985â€�ac59â€�7302a31b8451.html</a>); b: wp 18â€�7â€�2018 (<a href="https://16s.geneious.com/16s/results/e7d15b36-86bd-493f-8ccf-36ac439842c3.html%23">https://16s.geneious.com/16s/results/e7d15b36â€�86bdâ€�493fâ€�8ccfâ€�36ac439842c3.html#</a>); c: ba 19â€�7â€�2018 (<a href="https://16s.geneious.com/16s/results/57cc0b9e-f720-4476-9afb-897e7fcb89fd.html">https://16s.geneious.com/16s/results/57cc0b9eâ€�f720â€�4476â€�9afbâ€�897e7fcb89fd.html</a>); d: to 30â€�7â€�2018 (<a href="https://16s.geneious.com/16s/results/12fee165-8bf0-409f-adb3-3501ece47a8d.html%23">https://16s.geneious.com/16s/results/12fee165â€�8bf0â€�409fâ€�adb3â€�3501ece47a8d.html#</a>); e: gt 16â€�8â€�2018 (<a href="https://16s.geneious.com/16s/results/386dbe57-590b-44ef-89f6-f068d64dabff.html%23">https://16s.geneious.com/16s/results/386dbe57â€�590bâ€�44efâ€�89f6â€�f068d64dabff.html#</a>); f: wp 22â€�8â€�2018 (<a href="https://16s.geneious.com/16s/results/a2631a48-9dd8-4c6a-9358-8f357f012baa.html%23">https://16s.geneious.com/16s/results/a2631a48â€�9dd8â€�4c6aâ€�9358â€�8f357f012baa.html#</a>); g: ba 24â€�8â€�2018 (<a href="https://16s.geneious.com/16s/results/2dd31759-6c6c-458a-bcc7-0478626d21d9.html%23">https://16s.geneious.com/16s/results/2dd31759â€�6c6câ€�458aâ€�bcc7â€�0478626d21d9.html#</a>); h: to 28â€�8â€�2018 (<a href="https://16s.geneious.com/16s/results/f30255aa-9f0c-4e4d-95f1-8d782cef0f6b.html%23">https://16s.geneious.com/16s/results/f30255aaâ€�9f0câ€�4e4dâ€�95f1â€�8d782cef0f6b.html#</a>)</p>   </div> </div> <div class="fungi" title="sec"><span class="label" tagx="label" title="label">3.3.3</span><div class="title" tagx="title" title="title">
Fungi</div> <p>No regulated fungal species were identified across collection sites. A total of 36 genera consisting of ten or more species were detected across sampling locations (FigureÂ <a href="#ece37139-fig-0005">5</a>). Three out of the 36 genera (<i>Alternaria, Diaporthe,</i> and <i>Fusarium</i>) which have species represented on Canadian regulatory lists were successfully detected with a high percentage of pairwise genetic identity (â‰¥ 98%) in all location sites (FigureÂ <a href="#ece37139-fig-0005">5</a>). The number of fungal genera comprising regulated species per location ranged from three to six, with the traps placed in Barrie in August 2018 containing the lowest fungal diversity (FigureÂ <a href="#ece37139-fig-0005">5</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 5</span></a></div>   <p>Graphical representation of the fungal genera with ten or more species per sampling location and date. Each genus is represented by a different color, and the scale represents the number of species found</p>   </div> </div> </div> </div> <div class="discussion" title="discussion"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
DISCUSSION</div> <div class="performanceofoptimizedprotocol" title="sec"><span class="label" tagx="label" title="label">4.1</span><div class="title" tagx="title" title="title">
Performance of optimized protocol</div> <p>eDNA metabarcoding allows rapid species identification from complex environmental samples, with higher sensitivity, lower environmental disturbance, and shorter processing times than traditional identification methods (Ruppert etÂ al.,Â <a href="#ece37139-bib-0036">2019</a>). These attributes make the integration of eDNA methods into biosurveillance programs for the detecting IAS an ideal choice. The optimized protocol presented here, using a saturated salt trap solution successfully detected IAS and allowed sample processing from eDNA extraction to HTS at a fraction of the cost compared with the Illumina reference protocol (Table <a href="#ece37139-tbl-0002">2</a>). Our results suggest the stability of the eDNA in the collection fluid as well as the preservation of arthropod specimens allowing for morphological species identification and validation. Due to the characteristics of our experimental design, such as eDNA potentially remaining in the salt solution for up to 4Â weeks, there is an expected level of eDNA degradation. Despite possible degradation, the target fragment lengths (above 400Â bp) were effectively amplified for all the molecular markers studied, regardless of the origin of the genetic material (i.e., animal, bacterial, or fungal).</p> <p>Considering that every step of an adapted eDNA metabarcoding protocol can have a particular impact on species detectability, protocol validation is essential (Ruppert etÂ al.,Â <a href="#ece37139-bib-0036">2019</a>). The asymptotic rarefaction curves presented here may suggest that the read depths of our optimized protocol were sufficient to recover the full OTU diversity present in each trap. Nevertheless, it is important to highlight that the curves reflect OTU diversity based on the specific primer sets used in our study (TableÂ <a href="#ece37139-tbl-0001">1</a>). Although we selected primers to maximize OTU representation, due to issues such as â€œPCR dropout,â€� primer binding and therefore species detection may have been less than 100% (Griswold,Â <a href="#ece37139-bib-0018">2019</a>).</p> <p>Contrary to eDNA metabarcoding of samples collected in ethanol (Zenker etÂ al.,Â <a href="#ece37139-bib-0048">2020</a>), our protocol was successful in generating highâ€�quality library preparations of the three targeted molecular markers for all of our samples. This validates that saltâ€�saturated solutions can act as a reservoir for eDNA in conventional trapping surveys (Young etÂ al.,Â <a href="#ece37139-bib-0047">2020</a>). If this were not the case, any logistical advantages over the alcoholâ€�based fluids listed above would more likely be irrelevant. However, additional studies, specifically addressing eDNA preservation in saturated salt solutions versus alcoholâ€�based solutions, will be required in samples of known taxonomic composition to compare the methods' efficiency comprehensively.</p> <p>Our procedure was effective in detecting IAS from highâ€�risk sites in southern Ontario. Among the 2,535 BINs identified, two regulated insect species known to be established in the area, <i>A.Â planipennis</i> (order Coleoptera) and <i>L.Â dispar</i> (order Lepidoptera), were identified and also validated by a more than 97% mean similarity over a minimum of 392Â bp (TableÂ <a href="#ece37139-tbl-0004">4</a>). Species were identified using two independent data analysis pipelines to significantly reduce the likelihood of false positives.</p> <p>Despite the lack of detection of regulated bacterial and fungal species, there were five bacterial (FigureÂ <a href="#ece37139-fig-0004">4</a>) and three fungal genera (FigureÂ <a href="#ece37139-fig-0005">5</a>) comprising regulated species that were successfully recovered across all sampling sites. These results validate our modified protocol for detecting regulated bacterial and fungal taxa based on 16S and ITS and reinforce its utility for the identification of a wide variety of regulated taxa. However, whether or not some taxa are underrepresented in salt trapâ€�saturated solutions, as has been demonstrated for alcoholâ€�based collection fluids (Zenker etÂ al.,Â <a href="#ece37139-bib-0048">2020</a>), should be further evaluated.</p> </div> <div class="applicationsforbiomonitoringandbiosurveillance" title="sec"><span class="label" tagx="label" title="label">4.2</span><div class="title" tagx="title" title="title">
Applications for biomonitoring and biosurveillance</div> <p>Since its introduction into North America, the emerald ash borer beetle (order Coleoptera) has been responsible for killing several millions of ash trees across the continent (Herms &amp;amp; McCullough,Â <a href="#ece37139-bib-0022">2014</a>). Current management plans are mostly focused on biological and insecticidal control of infected areas and do not typically include early detection of the species as it moves into new areas. This approach mainly reflects difficulties in early detection via conventional surveys (Herms &amp;amp; McCullough,Â <a href="#ece37139-bib-0022">2014</a>). In that respect, present eDNA metabarcoding protocols are promising and allow for not only early detection but also species identifications at any life stage. The containment of pest insect outbreaks is also greatly benefited from the development of nondestructive eDNA metabarcoding protocols that allow early detection. By delineating the boundaries of an outbreak as clearly as possible, actions can be taken on infected areas while avoiding the spread of the pest. The gypsy moth (order Lepidoptera) is listed among the 100 worst IAS by the International Union for Conservation of Nature (IUCN), and it is considered among the worst hardwood defoliators (Vivek etÂ al.,Â <a href="#ece37139-bib-0043">2020</a>). Our sampling protocol was able to trap a regulated Lepidopteran species, even if the specific traps and lures targeted mainly Coleopterans, particularly woodâ€�boring beetles. This demonstrates the effectiveness of the protocol in detecting IAS and its potential wider applicability beyond our target taxa.</p> <p>Developing efficient and costâ€�effective protocols for the early detection of IAS is key in decreasing tree deaths and related economic losses. In the recent past, the high cost of HTS platforms prohibited their application in routine biodiversity monitoring and biosurveillance programs (Shokralla etÂ al.,Â <a href="#ece37139-bib-0038">2012</a>). Although HTS has become more economically accessible, it constitutes just one element in multistep metabarcoding protocols. In our study, we used alternative costâ€�effective chemistries without compromising protocol efficiency and were able to process samples at a reduced cost (Table <a href="#ece37139-tbl-0002">2</a>). This costâ€�effectiveness if applied to similar programs, could open opportunities for increased replication and thus more accurate species diversity delineations. Although technical replicates were not included in our experimental design, these could be incorporated in the future to evaluate the variability of the testing protocol. Nevertheless, the multiple traps per location could be considered biological replicates when it comes to the evaluation of the presence/absence of IAS in a given location.</p> <p>Also, as a nondestructive method, our present protocol allows for the morphological identification of the pests in the collection jars. However, even in the absence of morphological confirmation, environmental managers and regulators could direct attention and resources to areas where positive detection of IAS eDNA has been made. In these instances, temporal (multiyear) sampling protocols would be desirable to ensure proper decisionâ€�making and implementation of eradication strategies in infected areas.</p> <p>It is worth mentioning that morphological identification of arthropods remains a challenge due to factors such as unidentifiable early life stages, specimens escaping from the trap, and advanced or total degradation of specimens in collection jars. In all these cases, the eDNA metabarcoding protocol described here would allow the detection of the species, whereas morphological confirmation may not be possible, leading to false negatives. However, the eDNA of an invasive species that has not been captured can be released with a specimenâ€™s feces and result in potential falseâ€�positive results for a particular location while at the same time signalling the presence of the IAS in the wider region. The eDNA from a nonâ€�captured species can also be found when the invasive speciesâ€™ material persists in the trapped speciesâ€™ gut content. These scenarios illustrate how eDNA and morphological identification might yield different species' detection results.</p> </div> </div> <div class="conclusions" title="conclusions"><span class="label" tagx="label" title="label">5</span><div class="title" tagx="title" title="title">
CONCLUSIONS</div> <p>Here, we standardized, optimized, and validated a protocol for metabarcoding using eDNA collected from salt trap solutions. We demonstrate the validity of employing eDNAâ€�based detection of governmentâ€�regulated pests as a potential regulatory and management tool. Also, the current protocol supports the utility of saltâ€�saturated solutions as reservoirs of eDNA and as a valid substitute for alcoholâ€�based collection fluids in biosurveillance studies.</p> <p>In an environmental setting, direct applications of the protocol at the national level could include contributing to pest and pathogen databases maintained by the CFIA, the Ministry of Environment and Climate Change Canada, Ministry of Agriculture and Agriâ€�Food, and Ministry of Health. The protocol may also find utility with similar agencies internationally. Our protocol is not only relevant to CFIA's Plant Health business line but also to their Animal Health business line and to other stakeholders, such as industry, environmental consultants, researchers, and conservation managers. Further, our metabarcoding protocol has potential diagnostic applications in a human health clinical setting for bacterialâ€� and fungalâ€�borne diseases. Importantly, the costâ€�effectiveness of the optimized protocol could be useful in both environmental and human health settings where the costs of pathogen detection and identification can be prohibitive. The protocol could be used to investigate and describe pathogen interactions with each other and with their environment (i.e., the pathobiome). Altogether, the validated and costâ€�effective protocol may be fundamental in characterizing interactions among humans, animals, plants, microorganisms, and their shared environment to achieve better health results for all (e.g., One Health approach).</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
CONFLICT OF INTEREST</div> <p>None declared.</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
AUTHOR CONTRIBUTIONS</div> <p><b>Yoamel MiliÃ¡nâ€�GarcÃ­a:</b> Data curation (lead); formal analysis (lead); investigation (equal); methodology (equal); writing â€" original draft (lead); writing â€" review and editing (equal). <b>Robert Young:</b> Conceptualization (equal); investigation (equal); methodology (equal); writing â€" review and editing (equal). <b>Mary Madden:</b> Investigation (equal); methodology (equal); writing â€" review and editing (equal). <b>Erin Bullasâ€�Appleton:</b> Funding acquisition (equal); resources (equal); writing â€" review and editing (equal). <b>Robert H. Hanner:</b> Conceptualization (equal); funding acquisition (lead); investigation (equal); project administration (lead); resources (equal); supervision (lead); writing â€" review and editing (equal).</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data">  <p>Appendix S1</p>  <div class="media" title="media"><a href="ECE3-11-1999-s001.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGMENTS</div> <p>We are thankful to the team from the Genomics Facility of the AAC at the University of Guelph, and particularly to Jeffrey Gross for all suggestions and recommendations during library preparation and MiSeq runs. We thank Jocelyn Kelly and Mireille Marcotte for kindly reviewing earlier versions of the manuscript. We are also grateful to Jessica Castellanosâ€�Labarcena for helping with the analyses in R and the scripting. We acknowledge the assistance of all members of the Hanner Laboratory during data collection.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
DATA AVAILABILITY STATEMENT</div> <p>Data available from the University of Guelph Research Data Repository <a href="https://doi.org/10.5683/SP2/HTSZNS">https://doi.org/10.5683/SP2/HTSZNS</a> will become available upon acceptance of the manuscript for publication.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="ece37139-bib-0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: 16S Metagenomic Sequencing Library Preparation</span> (<span class="year" tagx="year" title="year">2013</span>). <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">28</span>. Retrieved from <a href="https://support.illumina.com/downloads/16s_metagenomic_sequencing_library_preparation.html">https://support.illumina.com/downloads/16s_metagenomic_sequencing_library_preparation.html</a></span></li> <li tag="ref"><a name="ece37139-bib-0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Adams</span>, <span class="given-names" tagx="given-names" title="given-names">C. I. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Knapp</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gemmell</span>, <span class="given-names" tagx="given-names" title="given-names">N. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jeunen</span>, <span class="given-names" tagx="given-names" title="given-names">G.â€�J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bunce</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lamare</span>, <span class="given-names" tagx="given-names" title="given-names">M. D.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Taylor</span>, <span class="given-names" tagx="given-names" title="given-names">H. R.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Beyond biodiversity: Can environmental DNA (eDNA) cut it as a population genetics tool?</span><span class="source" tagx="source" title="source">Genes</span>, <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="elocation-id" tagx="elocation-id" title="elocation-id">192</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/genes10030192">10.3390/genes10030192</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Barnes</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span>, <span class="given-names" tagx="given-names" title="given-names">C. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jerde</span>, <span class="given-names" tagx="given-names" title="given-names">C. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Renshaw</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chadderton</span>, <span class="given-names" tagx="given-names" title="given-names">W. L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lodge</span>, <span class="given-names" tagx="given-names" title="given-names">D. M.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Environmental conditions influence eDNA persistence in aquatic systems</span>. <span class="source" tagx="source" title="source">Environmental Science &amp;amp; Technology</span>, <span class="volume" tagx="volume" title="volume">48</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">1819</span>â€"<span class="lpage" tagx="lpage" title="lpage">1827</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/es404734p">10.1021/es404734p</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24422450">24422450</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Beeck</span>, <span class="given-names" tagx="given-names" title="given-names">M. O. D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lievens</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Busschaert</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Declerck</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vangronsveld</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Colpaert</span>, <span class="given-names" tagx="given-names" title="given-names">J. V.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Comparison and validation of some ITS primer pairs useful for fungal metabarcoding studies</span>. <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">6</span>), <span class="elocation-id" tagx="elocation-id" title="elocation-id">e97629</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0097629">10.1371/journal.pone.0097629</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24933453">24933453</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Braukmann</span>, <span class="given-names" tagx="given-names" title="given-names">T. W. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ivanova</span>, <span class="given-names" tagx="given-names" title="given-names">N. V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Prosser</span>, <span class="given-names" tagx="given-names" title="given-names">S. W. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Elbrecht</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Steinke</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ratnasingham</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">de Waard</span>, <span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sones</span>, <span class="given-names" tagx="given-names" title="given-names">J. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zakharov</span>, <span class="given-names" tagx="given-names" title="given-names">E. V.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hebert</span>, <span class="given-names" tagx="given-names" title="given-names">P. D. N.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Metabarcoding a diverse arthropod mock community</span>. <span class="source" tagx="source" title="source">Molecular Ecology Resources</span>, <span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">711</span>â€"<span class="lpage" tagx="lpage" title="lpage">727</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1755-0998.13008">10.1111/1755-0998.13008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30779309">30779309</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Carbayo</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Marques</span>, <span class="given-names" tagx="given-names" title="given-names">A. C.</span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">The costs of describing the entire animal kingdom</span>. <span class="source" tagx="source" title="source">Trends in Ecology &amp;amp; Evolution</span>, <span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">154</span>â€"<span class="lpage" tagx="lpage" title="lpage">155</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tree.2011.01.004">10.1016/j.tree.2011.01.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21296450">21296450</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: CFIA</span> (<span class="year" tagx="year" title="year">2017</span>). <span class="source" tagx="source" title="source">Survey protocol: Invasive alien species forestry trapping</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">CFIA</span>. Retrieved from <a href="https://www.inspection.gc.ca/plant-health/plant-pests-invasive-species/plant-pest-surveillance/eng/1344466499681/1344466638872#p3">https://www.inspection.gc.ca/plant-health/plant-pests-invasive-species/plant-pest-surveillance/eng/1344466499681/1344466638872#p3</a></span></li> <li tag="ref"><a name="ece37139-bib-0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Coyne</span>, <span class="given-names" tagx="given-names" title="given-names">K. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Handy</span>, <span class="given-names" tagx="given-names" title="given-names">S. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Demir</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Whereat</span>, <span class="given-names" tagx="given-names" title="given-names">E. B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hutchins</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Portune</span>, <span class="given-names" tagx="given-names" title="given-names">K. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Doblin</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cary</span>, <span class="given-names" tagx="given-names" title="given-names">S. C.</span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Improved quantitative realâ€�time PCR assays for enumeration of harmful algal species in field samples using an exogenous DNA reference standard</span>. <span class="source" tagx="source" title="source">Limnology and Oceanography: Methods</span>, <span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">381</span>â€"<span class="lpage" tagx="lpage" title="lpage">391</span>. <span class="pub-id"><a href="https://dx.doi.org/10.4319/lom.2005.3.381">10.4319/lom.2005.3.381</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0009" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Creer</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deiner</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Frey</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Porazinska</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Taberlet</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thomas</span>, <span class="given-names" tagx="given-names" title="given-names">W. K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Potter</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bik</span>, <span class="given-names" tagx="given-names" title="given-names">H. M.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The ecologist's field guide to sequenceâ€�based identification of biodiversity</span>. <span class="source" tagx="source" title="source">Methods in Ecology and Evolution</span>, <span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">1008</span>â€"<span class="lpage" tagx="lpage" title="lpage">1018</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/2041-210X.12574">10.1111/2041-210X.12574</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cristescu</span>, <span class="given-names" tagx="given-names" title="given-names">M. E.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">From barcoding single individuals to metabarcoding biological communities: Towards an integrative approach to the study of global biodiversity</span>. <span class="source" tagx="source" title="source">Trends in Ecology &amp;amp; Evolution</span>, <span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">10</span>), <span class="fpage" tagx="fpage" title="fpage">566</span>â€"<span class="lpage" tagx="lpage" title="lpage">571</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tree.2014.08.001">10.1016/j.tree.2014.08.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25175416">25175416</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">de Kerdrel</span>, <span class="given-names" tagx="given-names" title="given-names">G. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Andersen</span>, <span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kennedy</span>, <span class="given-names" tagx="given-names" title="given-names">S. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gillespie</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Krehenwinkel</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Rapid and costâ€�effective generation of single specimen multilocus barcoding data from whole arthropod communities by multiple levels of multiplexing</span>. <span class="source" tagx="source" title="source">Scientific Reports</span>, <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">12</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-019-54927-z">10.1038/s41598-019-54927-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31913322">31913322</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dempster</span>, <span class="given-names" tagx="given-names" title="given-names">E. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pryor</span>, <span class="given-names" tagx="given-names" title="given-names">K. V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Francis</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Young</span>, <span class="given-names" tagx="given-names" title="given-names">J. E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rogers</span>, <span class="given-names" tagx="given-names" title="given-names">H. J.</span></span> (<span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Rapid DNA extraction from ferns for PCRâ€"based analyses</span>. <span class="source" tagx="source" title="source">BioTechniques</span>, <span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">66</span>â€"<span class="lpage" tagx="lpage" title="lpage">68</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2144/99271bm13">10.2144/99271bm13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10407666">10407666</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dougherty</span>, <span class="given-names" tagx="given-names" title="given-names">M. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Larson</span>, <span class="given-names" tagx="given-names" title="given-names">E. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Renshaw</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gantz</span>, <span class="given-names" tagx="given-names" title="given-names">C. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Egan</span>, <span class="given-names" tagx="given-names" title="given-names">S. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Erickson</span>, <span class="given-names" tagx="given-names" title="given-names">D. M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lodge</span>, <span class="given-names" tagx="given-names" title="given-names">D. M.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Environmental DNA (eDNA) detects the invasive rusty crayfish Orconectes rusticus at low abundances</span>. <span class="source" tagx="source" title="source">Journal of Applied Ecology</span>, <span class="volume" tagx="volume" title="volume">53</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">722</span>â€"<span class="lpage" tagx="lpage" title="lpage">732</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1365-2664.12621">10.1111/1365-2664.12621</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Elbrecht</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Steinke</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Scaling up DNA metabarcoding for freshwater macrozoobenthos monitoring</span>. <span class="source" tagx="source" title="source">Freshwater Biology</span>, <span class="volume" tagx="volume" title="volume">64</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">380</span>â€"<span class="lpage" tagx="lpage" title="lpage">387</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/fwb.13220">10.1111/fwb.13220</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Folmer</span>, <span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Black</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hoeh</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lutz</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vrijenhoek</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">DNA primers for amplification of mitochondrial cytochrome c oxidase subunit I from diverse metazoan invertebrates</span>. <span class="source" tagx="source" title="source">Molecular Marine Biology and Biotechnology</span>, <span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">294</span>â€"<span class="lpage" tagx="lpage" title="lpage">299</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7881515">7881515</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Giovani</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">BlÃ¼mel</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lopian</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Teulon</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bloem</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Galeano MartÃ­nez</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">BeltrÃ¡n Montoya</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Urias Morales</span>, <span class="given-names" tagx="given-names" title="given-names">C. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dharmapuri</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Timote</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Horn</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chouibani</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mezui M'Ella</span>, <span class="given-names" tagx="given-names" title="given-names">J. G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Herrera</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Castinel</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Goletsos</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moeller</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Naumann</span>, <span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stancanelli</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, â€¦ <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xia</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Science diplomacy for plant health</span>. <span class="source" tagx="source" title="source">Nature Plants</span>, <span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">902</span>â€"<span class="lpage" tagx="lpage" title="lpage">905</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41477-020-0744-x">10.1038/s41477-020-0744-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32782407">32782407</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: Government of Canada, N. R. of C</span> (<span class="year" tagx="year" title="year">2013</span>). <span class="source" tagx="source" title="source">About Forest Invasive Alien Species (FIAS)</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Government of Canada</span>. <a href="https://www.exoticpests.gc.ca/definition">https://www.exoticpests.gc.ca/definition</a></span></li> <li tag="ref"><a name="ece37139-bib-0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Griswold</span>, <span class="given-names" tagx="given-names" title="given-names">C. K.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Properties of samples with segregating polymerase chain reaction (PCR) dropout mutations within a species</span>. <span class="source" tagx="source" title="source">Evolutionary Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">1176934319883612</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1177/1176934319883612">10.1177/1176934319883612</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31723319">31723319</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hajibabaei</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Janzen</span>, <span class="given-names" tagx="given-names" title="given-names">D. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Burns</span>, <span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hallwachs</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hebert</span>, <span class="given-names" tagx="given-names" title="given-names">P. D. N.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">DNA barcodes distinguish species of tropical Lepidoptera</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">103</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">968</span>â€"<span class="lpage" tagx="lpage" title="lpage">971</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0510466103">10.1073/pnas.0510466103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16418261">16418261</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hebert</span>, <span class="given-names" tagx="given-names" title="given-names">P. D. N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cywinska</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ball</span>, <span class="given-names" tagx="given-names" title="given-names">S. L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">deWaard</span>, <span class="given-names" tagx="given-names" title="given-names">J. R.</span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Biological identifications through DNA barcodes</span>. <span class="source" tagx="source" title="source">Proceedings of the Royal Society of London. Series B: Biological Sciences</span>, <span class="volume" tagx="volume" title="volume">270</span>(<span class="issue" tagx="issue" title="issue">1512</span>), <span class="fpage" tagx="fpage" title="fpage">313</span>â€"<span class="lpage" tagx="lpage" title="lpage">321</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2002.2218">10.1098/rspb.2002.2218</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12614582">12614582</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hebert</span>, <span class="given-names" tagx="given-names" title="given-names">P. D. N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Penton</span>, <span class="given-names" tagx="given-names" title="given-names">E. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Burns</span>, <span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Janzen</span>, <span class="given-names" tagx="given-names" title="given-names">D. H.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hallwachs</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Ten species in one: DNA barcoding reveals cryptic species in the neotropical skipper butterfly Astraptes fulgerator</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">101</span>(<span class="issue" tagx="issue" title="issue">41</span>), <span class="fpage" tagx="fpage" title="fpage">14812</span>â€"<span class="lpage" tagx="lpage" title="lpage">14817</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0406166101">10.1073/pnas.0406166101</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15465915">15465915</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Herms</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McCullough</span>, <span class="given-names" tagx="given-names" title="given-names">D. G.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Emerald ash borer invasion of North America: History, biology, ecology, impacts, and management</span>. <span class="source" tagx="source" title="source">Annual Review of Entomology</span>, <span class="volume" tagx="volume" title="volume">59</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">13</span>â€"<span class="lpage" tagx="lpage" title="lpage">30</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-ento-011613-162051">10.1146/annurev-ento-011613-162051</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">HernÃ¡ndezâ€�Triana</span>, <span class="given-names" tagx="given-names" title="given-names">L. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Prosser</span>, <span class="given-names" tagx="given-names" title="given-names">S. W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">RodrÃ­guezâ€�Perez</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chaverri</span>, <span class="given-names" tagx="given-names" title="given-names">L. G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hebert</span>, <span class="given-names" tagx="given-names" title="given-names">P. D. N.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gregory</span>, <span class="given-names" tagx="given-names" title="given-names">T. R.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Recovery of DNA barcodes from blackfly museum specimens (Diptera: Simuliidae) using primer sets that target a variety of sequence lengths</span>. <span class="source" tagx="source" title="source">Molecular Ecology Resources</span>, <span class="volume" tagx="volume" title="volume">14</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">508</span>â€"<span class="lpage" tagx="lpage" title="lpage">518</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1755-0998.12208">10.1111/1755-0998.12208</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24299419">24299419</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ji</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ashton</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pedley</span>, <span class="given-names" tagx="given-names" title="given-names">S. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Edwards</span>, <span class="given-names" tagx="given-names" title="given-names">D. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tang</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nakamura</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kitching</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dolman</span>, <span class="given-names" tagx="given-names" title="given-names">P. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Woodcock</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Edwards</span>, <span class="given-names" tagx="given-names" title="given-names">F. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Larsen</span>, <span class="given-names" tagx="given-names" title="given-names">T. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hsu</span>, <span class="given-names" tagx="given-names" title="given-names">W. W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Benedick</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hamer</span>, <span class="given-names" tagx="given-names" title="given-names">K. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wilcove</span>, <span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bruce</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Levi</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lott</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, â€¦ <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yu</span>, <span class="given-names" tagx="given-names" title="given-names">D. W.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Reliable, verifiable and efficient monitoring of biodiversity via metabarcoding</span>. <span class="source" tagx="source" title="source">Ecology Letters</span>, <span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">10</span>), <span class="fpage" tagx="fpage" title="fpage">1245</span>â€"<span class="lpage" tagx="lpage" title="lpage">1257</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/ele.12162">10.1111/ele.12162</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23910579">23910579</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Klindworth</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pruesse</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schweer</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Peplies</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Quast</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Horn</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">GlÃ¶ckner</span>, <span class="given-names" tagx="given-names" title="given-names">F. O.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Evaluation of general 16S ribosomal RNA gene PCR primers for classical and nextâ€�generation sequencingâ€�based diversity studies</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>, <span class="volume" tagx="volume" title="volume">41</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">e1</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gks808">10.1093/nar/gks808</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22933715">22933715</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Makiola</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Compson</span>, <span class="given-names" tagx="given-names" title="given-names">Z. G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Baird</span>, <span class="given-names" tagx="given-names" title="given-names">D. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Barnes</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Boerlijst</span>, <span class="given-names" tagx="given-names" title="given-names">S. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bouchez</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brennan</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bush</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Canard</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cordier</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Creer</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Curry</span>, <span class="given-names" tagx="given-names" title="given-names">R. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">David</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dumbrell</span>, <span class="given-names" tagx="given-names" title="given-names">A. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gravel</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hajibabaei</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hayden</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">van der Hoorn</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jarne</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, â€¦ <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bohan</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Key Questions for Nextâ€�Generation Biomonitoring</span>. <span class="source" tagx="source" title="source">Frontiers in Environmental Science</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fenvs.2019.00197">10.3389/fenvs.2019.00197</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nilsson</span>, <span class="given-names" tagx="given-names" title="given-names">R. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Larsson</span>, <span class="given-names" tagx="given-names" title="given-names">K.â€�H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Taylor</span>, <span class="given-names" tagx="given-names" title="given-names">A. F. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bengtssonâ€�Palme</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jeppesen</span>, <span class="given-names" tagx="given-names" title="given-names">T. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schigel</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kennedy</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Picard</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">GlÃ¶ckner</span>, <span class="given-names" tagx="given-names" title="given-names">F. O.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tedersoo</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Saar</span>, <span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">KÃµljalg</span>, <span class="given-names" tagx="given-names" title="given-names">U.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Abarenkov</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">The UNITE database for molecular identification of fungi: Handling dark taxa and parallel taxonomic classifications</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>, <span class="volume" tagx="volume" title="volume">47</span>(<span class="issue" tagx="issue" title="issue">D1</span>), <span class="fpage" tagx="fpage" title="fpage">D259</span>â€"<span class="lpage" tagx="lpage" title="lpage">D264</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gky1022">10.1093/nar/gky1022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30371820">30371820</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ondov</span>, <span class="given-names" tagx="given-names" title="given-names">B. D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bergman</span>, <span class="given-names" tagx="given-names" title="given-names">N. H.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Phillippy</span>, <span class="given-names" tagx="given-names" title="given-names">A. M.</span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Interactive metagenomic visualization in a Web browser</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">12</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">385</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-12-385">10.1186/1471-2105-12-385</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21961884">21961884</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pawlowski</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">ApothÃ©lozâ€�Perretâ€�Gentil</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Altermatt</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Environmental DNA: What's behind the term? Clarifying the terminology and recommendations for its future use in biomonitoring</span>. <span class="source" tagx="source" title="source">Molecular Ecology</span>, <span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">22</span>), <span class="fpage" tagx="fpage" title="fpage">4258</span>â€"<span class="lpage" tagx="lpage" title="lpage">4264</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.15643">10.1111/mec.15643</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32966665">32966665</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Piper</span>, <span class="given-names" tagx="given-names" title="given-names">A. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Batovska</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cogan</span>, <span class="given-names" tagx="given-names" title="given-names">N. O. I.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weiss</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cunningham</span>, <span class="given-names" tagx="given-names" title="given-names">J. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rodoni</span>, <span class="given-names" tagx="given-names" title="given-names">B. C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Blacket</span>, <span class="given-names" tagx="given-names" title="given-names">M. J.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Prospects and challenges of implementing DNA metabarcoding for highâ€�throughput insect surveillance</span>. <span class="source" tagx="source" title="source">GigaScience</span>, <span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/gigascience/giz092">10.1093/gigascience/giz092</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Quast</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pruesse</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yilmaz</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gerken</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schweer</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yarza</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Peplies</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">GlÃ¶ckner</span>, <span class="given-names" tagx="given-names" title="given-names">F. O.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">The SILVA ribosomal RNA gene database project: Improved data processing and webâ€�based tools</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>, <span class="volume" tagx="volume" title="volume">41</span>(<span class="issue" tagx="issue" title="issue">D1</span>), <span class="fpage" tagx="fpage" title="fpage">D590</span>â€"<span class="lpage" tagx="lpage" title="lpage">D596</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gks1219">10.1093/nar/gks1219</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193283">23193283</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: R Core Team</span> (<span class="year" tagx="year" title="year">2019</span>). <span class="source" tagx="source" title="source">R: A language and environment for statistical computing</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">R Core Team</span>. Retrieved from <a href="https://www.R-project.org/">https://www.Râ€�project.org/</a></span></li> <li tag="ref"><a name="ece37139-bib-0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ratnasingham</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hebert</span>, <span class="given-names" tagx="given-names" title="given-names">P. D. N.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">bold: The barcode of life data system (http://www.barcodinglife.org)</span>. <span class="source" tagx="source" title="source">Molecular Ecology Notes</span>, <span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">355</span>â€"<span class="lpage" tagx="lpage" title="lpage">364</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1471-8286.2007.01678.x">10.1111/j.1471-8286.2007.01678.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18784790">18784790</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0034" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ratnasingham</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hebert</span>, <span class="given-names" tagx="given-names" title="given-names">P. D. N.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">A DNAâ€�based registry for all animal species: The Barcode Index Number (BIN) system</span>. <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">7</span>), <span class="elocation-id" tagx="elocation-id" title="elocation-id">e66213</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0066213">10.1371/journal.pone.0066213</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23861743">23861743</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0035" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Renshaw</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Olds</span>, <span class="given-names" tagx="given-names" title="given-names">B. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jerde</span>, <span class="given-names" tagx="given-names" title="given-names">C. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McVeigh</span>, <span class="given-names" tagx="given-names" title="given-names">M. M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lodge</span>, <span class="given-names" tagx="given-names" title="given-names">D. M.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">The room temperature preservation of filtered environmental DNA samples and assimilation into a phenolâ€"chloroformâ€"isoamyl alcohol DNA extraction</span>. <span class="source" tagx="source" title="source">Molecular Ecology Resources</span>, <span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">168</span>â€"<span class="lpage" tagx="lpage" title="lpage">176</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1755-0998.12281">10.1111/1755-0998.12281</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24834966">24834966</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ruppert</span>, <span class="given-names" tagx="given-names" title="given-names">K. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kline</span>, <span class="given-names" tagx="given-names" title="given-names">R. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rahman</span>, <span class="given-names" tagx="given-names" title="given-names">M. S.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Past, present, and future perspectives of environmental DNA (eDNA) metabarcoding: A systematic review in methods, monitoring, and applications of global eDNA</span>. <span class="source" tagx="source" title="source">Global Ecology and Conservation</span>, <span class="volume" tagx="volume" title="volume">17</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e00547</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gecco.2019.e00547">10.1016/j.gecco.2019.e00547</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Seifert</span>, <span class="given-names" tagx="given-names" title="given-names">K. A.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Progress towards DNA barcoding of fungi</span>. <span class="source" tagx="source" title="source">Molecular Ecology Resources</span>, <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">s1</span>), <span class="fpage" tagx="fpage" title="fpage">83</span>â€"<span class="lpage" tagx="lpage" title="lpage">89</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1755-0998.2009.02635.x">10.1111/j.1755-0998.2009.02635.x</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shokralla</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Spall</span>, <span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gibson</span>, <span class="given-names" tagx="given-names" title="given-names">J. F.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hajibabaei</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Nextâ€�generation sequencing technologies for environmental DNA research</span>. <span class="source" tagx="source" title="source">Molecular Ecology</span>, <span class="volume" tagx="volume" title="volume">21</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">1794</span>â€"<span class="lpage" tagx="lpage" title="lpage">1805</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2012.05538.x">10.1111/j.1365-294X.2012.05538.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22486820">22486820</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Taberlet</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bonin</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Coissac</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zinger</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="source" tagx="source" title="source">Environmental DNA: For biodiversity research and monitoring</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span>.</span></li> <li tag="ref"><a name="ece37139-bib-0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Taberlet</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Coissac</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pompanon</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brochmann</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Willerslev</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Towards nextâ€�generation biodiversity assessment using DNA metabarcoding</span>. <span class="source" tagx="source" title="source">Molecular Ecology</span>, <span class="volume" tagx="volume" title="volume">21</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">2045</span>â€"<span class="lpage" tagx="lpage" title="lpage">2050</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2012.05470.x">10.1111/j.1365-294X.2012.05470.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22486824">22486824</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: The State of Canada's Forests. Annual Report 2018</span> (n.d.). <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">80</span>. Retrieved from <a href="https://cfs.nrcan.gc.ca/pubwarehouse/pdfs/39336.pdf">https://cfs.nrcan.gc.ca/pubwarehouse/pdfs/39336.pdf</a></span></li> <li tag="ref"><a name="ece37139-bib-0042" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Turner</span>, <span class="given-names" tagx="given-names" title="given-names">C. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Barnes</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xu</span>, <span class="given-names" tagx="given-names" title="given-names">C. C. Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jones</span>, <span class="given-names" tagx="given-names" title="given-names">S. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jerde</span>, <span class="given-names" tagx="given-names" title="given-names">C. L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lodge</span>, <span class="given-names" tagx="given-names" title="given-names">D. M.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Particle size distribution and optimal capture of aqueous macrobial eDNA</span>. <span class="source" tagx="source" title="source">Methods in Ecology and Evolution</span>, <span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">676</span>â€"<span class="lpage" tagx="lpage" title="lpage">684</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/2041-210X.12206">10.1111/2041-210X.12206</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0043" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vivek</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Griess</span>, <span class="given-names" tagx="given-names" title="given-names">V. C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Keena</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Assessing the potential distribution of Asian Gypsy moth in Canada: A comparison of two methodological approaches</span>. <span class="source" tagx="source" title="source">Scientific Reports (Nature Publisher Group); London</span>, <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">10</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-019-57020-7">10.1038/s41598-019-57020-7</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0044" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Garrity</span>, <span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tiedje</span>, <span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cole</span>, <span class="given-names" tagx="given-names" title="given-names">J. R.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy</span>. <span class="source" tagx="source" title="source">Applied and Environmental Microbiology</span>, <span class="volume" tagx="volume" title="volume">73</span>(<span class="issue" tagx="issue" title="issue">16</span>), <span class="fpage" tagx="fpage" title="fpage">5261</span>â€"<span class="lpage" tagx="lpage" title="lpage">5267</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.00062-07">10.1128/AEM.00062-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586664">17586664</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0045" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Westphal</span>, <span class="given-names" tagx="given-names" title="given-names">M. I.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Browne</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">MacKinnon</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Noble</span>, <span class="given-names" tagx="given-names" title="given-names">I.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">The link between international trade and the global distribution of invasive alien species</span>. <span class="source" tagx="source" title="source">Biological Invasions</span>, <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">391</span>â€"<span class="lpage" tagx="lpage" title="lpage">398</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-007-9138-5">10.1007/s10530-007-9138-5</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0046" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Williams</span>, <span class="given-names" tagx="given-names" title="given-names">K. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huyvaert</span>, <span class="given-names" tagx="given-names" title="given-names">K. P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Piaggio</span>, <span class="given-names" tagx="given-names" title="given-names">A. J.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">No filters, no fridges: A method for preservation of water samples for eDNA analysis</span>. <span class="source" tagx="source" title="source">BMC Research Notes</span>, <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">298</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s13104-016-2104-5">10.1186/s13104-016-2104-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27278936">27278936</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0047" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Young</span>, <span class="given-names" tagx="given-names" title="given-names">R. G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">MiliÃ¡nâ€�GarcÃ­a</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yu</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bullasâ€�Appleton</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hanner</span>, <span class="given-names" tagx="given-names" title="given-names">R. H.</span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Biosurveillance for invasive insect pest species using an environmental DNA metabarcoding approach and a high salt trap collection fluid</span>. <span class="source" tagx="source" title="source">Ecology and Evolution</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.7113">10.1002/ece3.7113</a></span></span></li> <li tag="ref"><a name="ece37139-bib-0048" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zenker</span>, <span class="given-names" tagx="given-names" title="given-names">M. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Specht</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fonseca</span>, <span class="given-names" tagx="given-names" title="given-names">V. G.</span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Assessing insect biodiversity with automatic light traps in Brazil: Pearls and pitfalls of metabarcoding samples in preservative ethanol</span>. <span class="source" tagx="source" title="source">Ecology and Evolution</span>, <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">2352</span>â€"<span class="lpage" tagx="lpage" title="lpage">2366</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.6042">10.1002/ece3.6042</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32184986">32184986</a></span></span></li> </ul> </div> </div>  </body></html>