<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Nat Commun</span><span class="iso-abbrev" title="iso-abbrev">Nat Commun</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Nature Communications</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2041-1723</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5585233">5585233</a></span><span class="publisher-id" title="publisher-id">626</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41467-017-00626-0">10.1038/s41467-017-00626-0</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Lineage overwhelms environmental conditions in determining rhizosphere bacterial community structure in a cosmopolitan invasive plant</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowen</span><span class="given-names" tagx="given-names" title="given-names">Jennifer L.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">je.bowen@northeastern.edu</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kearns</span><span class="given-names" tagx="given-names" title="given-names">Patrick J.</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Byrnes</span><span class="given-names" tagx="given-names" title="given-names">Jarrett E. K.</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wigginton</span><span class="given-names" tagx="given-names" title="given-names">Sara</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">Warwick J.</span></span><a href="#Aff4">4</a><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenwood</span><span class="given-names" tagx="given-names" title="given-names">Michael</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tran</span><span class="given-names" tagx="given-names" title="given-names">Khang</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Jennifer</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cronin</span><span class="given-names" tagx="given-names" title="given-names">James T.</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyerson</span><span class="given-names" tagx="given-names" title="given-names">Laura A.</span></span><a href="#Aff3">3</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2173 3359</span><span class="institution-id" title="institution-id">grid.261112.7</span><span class="institution" title="institution">Department of Marine and Environmental Sciences, Marine Science Center, </span><span class="institution" title="institution">Northeastern University, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0386 3207</span><span class="institution-id" title="institution-id">grid.266685.9</span><span class="institution" title="institution">University of Massachusetts Boston, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0416 2242</span><span class="institution-id" title="institution-id">grid.20431.34</span><span class="institution" title="institution">Department of Natural Resources Science, </span><span class="institution" title="institution">University of Rhode Island, Woodward Hall, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0662 7451</span><span class="institution-id" title="institution-id">grid.64337.35</span><span class="institution" title="institution">Department of Biological Sciences, </span><span class="institution" title="institution">Louisiana State University, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0385 8571</span><span class="institution-id" title="institution-id">grid.16488.33</span><span class="institution" title="institution">Present Address: The Bio-Protection Research Centre, </span><span class="institution" title="institution">Lincoln University, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-9-9</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2017-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">433</span><span class="history" title="history"><span class="received" title="received">received: 2017-2-13</span><span class="accepted" title="accepted">accepted: 2017-7-13</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Plant–microbe interactions play crucial roles in species invasions but are rarely investigated at the intraspecific level. Here, we study these interactions in three lineages of a globally distributed plant, <i>Phragmites australis</i>. We use field surveys and a common garden experiment to analyze bacterial communities in the rhizosphere of <i>P. australis</i> stands from native, introduced, and Gulf lineages to determine lineage-specific controls on rhizosphere bacteria. We show that within-lineage bacterial communities are similar, but are distinct among lineages, which is consistent with our results in a complementary common garden experiment. Introduced <i>P. australis</i> rhizosphere bacterial communities have lower abundances of pathways involved in antimicrobial biosynthesis and degradation, suggesting a lower exposure to enemy attack than native and Gulf lineages. However, lineage and not rhizosphere bacterial communities dictate individual plant growth in the common garden experiment. We conclude that lineage is crucial for determination of both rhizosphere bacterial communities and plant fitness.</p> </div> <div class="abstract" title="abstract"> <p id="Par2">Environmental factors often outweigh host heritable factors in structuring host-associated microbiomes. Here, Bowen et al. show that host lineage is crucial for determination of rhizosphere bacterial communities in <i>Phragmites australis</i>, a globally distributed invasive plant.</p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2017" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par3">Interactions between plants and soil microorganisms that occur in the rhizosphere create microhabitats that result in plant species-specific rhizosphere microbial communities. These interactions are highly localized in the region immediately adjacent to plant roots and play a critical role in plant fitness<sup><a href="#CR1">1</a>, <a href="#CR2">2</a></sup>. There is extensive literature on host–microbe interactions<sup><a href="#CR3">3</a>, <a href="#CR4">4</a></sup>, much of which suggests that environmental factors typically outweigh heritable factors in structuring microbiomes<sup><a href="#CR5">5</a></sup>, despite genome wide association studies that link host loci to microbial community structure<sup><a href="#CR6">6</a></sup>. However, we lack an understanding of the extent of intraspecific variation in plant–soil microbe interactions, particularly among invasive plant populations, which could provide insights into why some genotypes are more successful than others. While there is evidence of intraspecific variation in microbial communities among cultivated plant species<sup><a href="#CR7">7</a>–<a href="#CR12">12</a></sup>, it is unclear whether intraspecific forcing is sufficient to overwhelm environmental influences to structure rhizosphere microbial communities in wild plant species.</p> <p id="Par4">Micallef et al.<sup><a href="#CR8">8</a></sup> documented unique rhizosphere microbial communities among different genotypes of <i>Arabidopsis thaliana</i> when grown in common soil and attributed those changes to genotype-specific variation in root exudates. Similarly, different wheat<sup><a href="#CR9">9</a></sup>, corn<sup><a href="#CR10">10</a>, <a href="#CR11">11</a></sup>, and rice<sup><a href="#CR12">12</a></sup> cultivars all demonstrate unique microbial signatures in their rhizosphere soils. However, most studies of this nature are typically executed either in a laboratory setting with common soil or in a small number of similar field plots<sup><a href="#CR9">9</a></sup>. Exceptions, however, include a study of corn cultivars grown across multiple locations that suggests geographical differences are more important than cultivar differences in structuring microbial communities<sup><a href="#CR7">7</a></sup>, a pattern that is duplicated in natural populations of the cottonwood tree (<i>Populus deltoides</i>)<sup><a href="#CR13">13</a></sup>. Very few studies have evaluated the interplay between biogeography and intraspecific genetic factors in structuring the rhizosphere microbiome.</p> <p id="Par5">Refining our understanding of the interactions between plant genotype, biogeography, and the microbiome is a critically important research priority to promote sustainable agriculture<sup><a href="#CR14">14</a></sup> and is also crucial to our understanding of plant invasions. Escape from native soil plant pathogens is often invoked as one reason for the success of exotic invasive species<sup><a href="#CR15">15</a></sup>. Reduced pathogen loads could decrease the need of invaders to allocate resources to pathogen defense, for example via root exudates that stimulate antibiotic production in the rhizosphere<sup><a href="#CR8">8</a></sup>. These resources, instead, could be reallocated to traits that enhance the competitive ability of the invasive plant<sup><a href="#CR16">16</a></sup>. If true, we might expect to see a greater homogeneity in the microbial community structure of native populations compared to introduced populations because of the need to maintain robust pathogen defenses. In contrast, invasive populations that do not use resources for pathogen defense would be more likely to have divergent microbial communities that are dominated by local soil generalists<sup><a href="#CR16">16</a></sup>, therefore showing decreasing similarity with distance among invasive lineages.</p> <p id="Par6"><i>Phragmites australis</i> is a globally distributed plant that is also a harmful invader in North America. It is among the best-studied uncultivated plants<sup><a href="#CR17">17</a></sup> and is considered a model species for plant invasions<sup><a href="#CR18">18</a></sup>. In North America there are at least three lineages of <i>P. australis</i> that grow sympatrically, including an invasive lineage (haplotype M, hereafter "Introduced") that was introduced to North America from Europe<sup><a href="#CR19">19</a></sup> at least 150 years ago<sup><a href="#CR20">20</a></sup>. Other lineages of <i>P. australis</i> include multiple haplotypes of the native North American lineage (hereafter "Native"), and one Gulf coast lineage, (haplotype I, hereafter "Gulf"), whose origin is undetermined but appears to have been present in North America for thousands of years<sup><a href="#CR19">19</a>–<a href="#CR26">26</a></sup>. Across North America, Introduced <i>P. australis</i> is highly dominant in coastal and inland marshes, affects multiple trophic levels and ecosystem processes, and is costly to eradicate<sup><a href="#CR27">27</a></sup>. Because the three different lineages of <i>P. australis</i> are widespread and grow sympatrically across North America, this species provides a unique opportunity to study the similarities and differences in the rhizosphere bacterial communities both among plant lineages with known phylogenies and across a wide range of geographical distances.</p> <p id="Par7">Here, we investigated whether there were distinct rhizosphere bacterial communities associated with different <i>P. australis</i> lineages in both a field survey and common garden experiment. We hypothesized that if plant–microbe interactions facilitated invasions, Introduced <i>P. australis</i> populations would have more heterogeneous rhizosphere bacterial communities than Native or Gulf populations. Since soil microbes are typically controlled by local edaphic characteristics<sup><a href="#CR28">28</a>–<a href="#CR31">31</a></sup>, homogeneity of rhizosphere bacteria within <i>P. australis</i> lineages across the sampling range, along with heterogeneity among lineages in our field experiment, would be consistent with our hypothesis that lineage-specific forcing is structuring rhizosphere bacterial communities. Our results, confirmed in both a field survey and a common garden experiment, indicate that the rhizosphere bacterial community is structured by plant lineage, with the Introduced lineage demonstrating evidence of decreased enemy attack compared to the Gulf and Native lineages.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="overviewofexperiments" title="sec"> <div class="title" tagx="title" title="title">
Overview of experiments</div> <p id="Par8">We performed a field survey in which we collected rhizosphere soils from multiple <i>P. australis</i> populations across three lineages at 21 locations spanning the continental United States (Supplementary Table <a href="#MOESM1">1</a>, Supplementary Fig. <a href="#MOESM1">1</a>). At three locations (Block Island, RI; Great Bay, NH; and Falmouth, MA), we collected rhizosphere soil samples from co-occurring Native and Introduced <i>P. australis</i> populations, allowing us to more closely examine the relative strength of both lineage and geographic proximity in structuring rhizosphere bacterial communities. It is possible, however, that homogeneity in the bacterial community within lineages could also arise from different microhabitat preferences maintained by each lineage, so we confirmed the role of lineage as a driver of bacterial community structure in a common garden experiment. We grew multiple populations from each lineage individually from surface sterilized rhizomes in homogenized (though not sterilized) potting soil under identical greenhouse conditions. For both the field and common garden experiment we used high throughput sequencing of the 16S rRNA gene to examine the entire bacterial community and its gene product, 16S rRNA, to assess which bacteria were actively synthesizing proteins and were therefore more likely to be influencing plant fitness (see Methods).</p> </div> <div class="fieldsurvey" title="sec"> <div class="title" tagx="title" title="title">
Field survey</div> <p id="Par9">Even at continental scales, our results reveal a remarkable influence of <i>P. australis</i> lineage on the structure of the bacterial community (Fig. <a href="#Fig1">1a</a>). A principal coordinates analysis of Bray–Curtis similarity indicates that the rhizosphere bacterial communities of the Native, Gulf, and Introduced lineages were distinct from one another, regardless of where in North America they were collected (permutational multivariate analysis of variance (PERMANOVA); <i>F</i><sub>2,107</sub> = 22.38, <i>p</i> &amp;lt; 0.001). We also found similar results when we examined the potentially active community (Supplementary Fig. <a href="#MOESM1">2</a>; PERMANOVA, <i>F</i><sub>2,75</sub> = 8.13, <i>p</i> &amp;lt; 0.001), indicating that lineage may be a sufficiently strong determinant of bacterial community structure that it overwhelms local environmental conditions even among the active community, which would be more likely to respond to local environmental variation. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Differences in microbial community structure among lineages and as a function of distance between sampling locations. <b>a</b> Principal coordinates analysis of Bray–Curtis dissimilarity for the microbial communities associated with Native (<i>green</i>), Gulf (<i>blue</i>) and Introduced (<i>orange</i>) lineages of <i>Phragmites australis</i>, collected from across the United States (Supplementary Fig. <a href="#MOESM1">1</a>). <b>b</b> Bray–Curtis dissimilarity plotted as a function of the distance (km) between sampling locations. Regression statistics are found in Supplementary Table <a href="#MOESM1">2</a>. <i>n</i> = 5 replicates per <i>P. australis</i> population (eight Introduced populations, five Gulf populations, eight Native populations)</p>   </div> <p /> <p id="Par10">The rhizosphere bacterial communities within the Gulf and Native lineages demonstrated stronger within-lineage similarity than the Introduced lineage (Fig. <a href="#Fig1">1b</a>, Supplementary Table <a href="#MOESM1">2</a>). The lower overall similarity in community structure among Introduced populations ultimately resulted in an overall higher bacterial richness in the Introduced lineage compared to the other lineages (Supplementary Fig. <a href="#MOESM1">3</a>; ANOVA, <i>F</i><sub>5,206</sub> = 56.5, <i>p</i> &amp;lt; 0.01). Remarkably, bacterial communities associated with sympatric Native and Introduced populations were less similar to one another than populations from within the same lineage collected &amp;gt;3000 km apart (Fig. <a href="#Fig1">1b</a>). The similarity among the bacterial communities within the Introduced lineage decreased with distance, a pattern not observed in the Native and Gulf lineages and indicative of a higher rate of bacterial turnover in the Introduced lineage rhizospheres.</p> <p id="Par11">We examined the core microbiome of each lineage (taxa present in 100% of samples collected from across the continent). Despite striking differences in community structure among lineages (Fig. <a href="#Fig1">1a</a>), bacterial taxa within lineages displayed a core set of taxa that were consistent across large geographic regions (Fig. <a href="#Fig2">2</a>). Core microbiome members were predominantly heterotrophic taxa likely able to take advantage of the high primary productivity of <i>P. australis</i>. Surprisingly, while core Native and Gulf bacteria were sometimes present in the Introduced rhizospheres, abundances were typically low (Fig. <a href="#Fig2">2</a>). For example, the deltaproteobacterial order Myxococcales was among the most abundant orders identified but 77% of all Myxococcales in the total community were found in Native rhizospheres and only 4% were found in Introduced rhizospheres (Supplementary Table <a href="#MOESM1">3</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Core microbiome analysis for each lineage. Abundance of taxa belonging to the core microbiome of different <i>Phragmites australis</i> lineages. <i>Colored bars</i> indicate the taxon was present in 100% of the samples for that lineage, whereas <i>gray bars</i> indicate that it was only present in some samples</p>   </div> <p /> </div> <div class="commongardenexperiment" title="sec"> <div class="title" tagx="title" title="title">
Common garden experiment</div> <p id="Par12">To investigate experimentally how different <i>P. australis</i> lineages shape the rhizosphere community and whether this ecosystem engineering contributes to plant fitness, we performed a common garden experiment on a subset of the field populations (Supplementary Table <a href="#MOESM1">1</a>). We also sought to tease apart the effects of lineage on plant performance as mediated by lineage-specific effects on the soil bacterial community. We hypothesized that the three lineages grown in homogenized Metromix potting soil would develop distinct rhizosphere bacterial communities, and that these communities would influence plant productivity. Although similar initially (Fig. <a href="#Fig3">3a</a>, <i>purple</i>), 4 months of conditioning by the three different lineages caused soil bacterial communities to diverge dramatically (Fig. <a href="#Fig3">3</a>, Supplementary Tables <a href="#MOESM1">3</a> and <a href="#MOESM1">4</a>). The total bacterial community (Fig. <a href="#Fig3">3a</a>, <i>open symbols</i>) formed three clusters; one from the Gulf, one exclusively from Introduced, and one that was a mix of Introduced and Native. Consistent with our field results, the active bacterial community (Fig. <a href="#Fig3">3b</a>) was strongly influenced by both lineage and source population. Active bacterial communities associated with the three Native populations (Fig. <a href="#Fig3">3b</a>, <i>blue</i>) were highly similar to each other, but divergent from Introduced and Gulf communities. In contrast, each individual population of active bacteria from both the Gulf (Fig. <a href="#Fig3">3b</a>, <i>green</i>) and Introduced (Fig. <a href="#Fig3">3b</a>, <i>orange</i>) lineages formed distinct non-overlapping clusters of points that were not only different from Native populations, but also different from each other. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Microbial community structure of different lineages of <i>Phragmites australis</i> grown in a common garden experiment. <b>a</b> Principal coordinates analysis of Bray–Curtis dissimilarity for the entire (<i>open symbols</i>) and active (<i>closed symbols</i>) microbial community, along with the pre-incubation community (<i>purple</i>). <b>b</b> Principal coordinates analysis of the Bray–Curtis dissimilarity values for just the active microbial community, indicating the importance of population (PERMANOVA, <i>F</i><sub>59,14</sub> = 20.56, <i>p</i> &amp;lt; 0.001), and lineage (<i>F</i><sub>71,2</sub> = 19.79, <i>p</i> &amp;lt; 0.001)</p>   </div> <p /> <p id="Par13">To assess whether the lineage-specific differences in bacterial community structure could potentially play a role in plant fitness or invasion success, we used an ancestral state reconstruction model to infer the function of the bacterial community<sup><a href="#CR32">32</a></sup>. We assessed three different KEGG pathways relevant to plant–microbe interactions, including Toxin and Metal Detoxification, Antimicrobial Biosynthesis, and Antimicrobial Degradation. All three pathways were significantly less abundant in the rhizospheres of the Introduced lineage after 4 months growing in our common garden experiment (Fig. <a href="#Fig4">4</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Abundance of KEGG pathways potentially associated with plant–microbe interactions in the rhizosphere soils of Introduced, Gulf, and Native lineages. In each of the three pathways, data are the mean ± standard error of the mean and different letters denote significantly different means within each pathway. The abundance of each pathway was significantly lower in the Introduced lineage rhizosphere microbial communites than in those communities associated with the Gulf and Native lineages. Toxin/metal detoxification: ANOVA, <i>F</i><sub>2,75</sub> = 47.56, <i>p</i> &amp;lt; 0.001; antimicrobial biosynthesis: ANOVA, <i>F</i><sub>2,75</sub> = 35.23, <i>p</i> &amp;lt; 0.001; antimicrobial degradation: ANOVA, <i>F</i><sub>2,75</sub> = 29.33, <i>p</i> &amp;lt; 0.001)</p>   </div> <p /> </div> <div class="structuralequationmodeling" title="sec"> <div class="title" tagx="title" title="title">
Structural equation modeling</div> <p id="Par14">Whether changes in the bacterial community affect invasion success ultimately depends on how these changes influence plant fitness related traits, measured here as biomass. However, plant fitness might also be driven by other modified soil properties or solely by plant lineage. We teased apart these direct and indirect effects of <i>P. australis</i> lineage using Structural Equation Modeling<sup><a href="#CR33">33</a>–<a href="#CR35">35</a></sup>. Common garden Introduced populations had lower bacterial activity and metabolism than Native and Gulf <i>P. australis</i> populations. Surprisingly, this lineage effect on bacterial community structure did not translate into altered biomass of <i>P. australis</i> plants (Fig. <a href="#Fig5">5</a>, Supplementary Table <a href="#MOESM1">5</a> and <a href="#MOESM1">6</a>). Rather, while <i>P. australis</i> lineage shaped the bacterial community, its activity, and belowground soil properties, the influence of lineage on aboveground and belowground biomass was solely via direct effects. Thus, <i>Phragmites australis</i> lineage appears to be the primary driver of differences in the rhizosphere ecosystem and in plant biomass, at least for the 4-month-old plants in this common garden experiment. It is worth noting, however, that the observed differences in in the rhizosphere bacterial community could have indirect effects on co-occurring plants through spillover of generalist pathogens or mutualists, or by altering competitive interactions<sup><a href="#CR36">36</a></sup>, effects we did not evaluate because each <i>P. australis</i> plant was grown individually. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Structural Equation Model fitted with range-standardized coefficients. <i>Solid lines</i> indicate that the driver influences the likelihood of the model via a <i>χ</i><sup>2</sup> Likelihood ratio test. <i>Dashed paths</i> indicate no detectable influence of the driver (<i>p</i> &amp;gt; 0.05). Standardized coefficients are presented for each path. Letters <i>a</i> and <i>b</i> denote groupings via post-hoc tests. <i>χ</i><sup>2</sup> test statistics can be found in Supplementary Tables <a href="#MOESM1">5</a> and <a href="#MOESM1">6</a></p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par15">Our results expand on findings from lab-based studies of model organisms and agricultural cultivars<sup><a href="#CR7">7</a>–<a href="#CR12">12</a></sup> to document the important role of plant lineage in structuring the bacterial community of a wild plant species whose lineages diverged thousands of years ago<sup><a href="#CR37">37</a>, <a href="#CR38">38</a></sup>. We demonstrate that different lineages of <i>P. australis</i> had unique bacterial signatures when samples were collected from across the continental United States. Numerous studies have documented the importance of local environmental features in structuring soil microbial communities<sup><a href="#CR7">7</a>, <a href="#CR13">13</a>, <a href="#CR28">28</a>–<a href="#CR31">31</a></sup>, thus the divergence in bacterial communities between sympatric lineages growing in the same system suggests that lineage overwhelms local environmental conditions in determining rhizosphere bacterial community structure—results we were able to confirm with a common garden experiment.</p> <p id="Par16">Differences between Native and Introduced <i>P. australis</i> have been observed in associated oomycete<sup><a href="#CR39">39</a></sup> and archaeal communities<sup><a href="#CR40">40</a></sup>. These consistent differences between Native and Introduced lineages across multiple microbial domains support the view that lineage exerts a dominant control on soil rhizosphere communities, one that may promote invasion success through plant–soil feedbacks, spillover, and soil legacies<sup><a href="#CR36">36</a></sup>. In our common garden experiment, we found no evidence that lineage-driven bacterial communities directly affected plant fitness in the absence of competitors. Therefore, any effects of lineage-specific bacterial communities on invasion success likely comes from effects on competitors via plant–soil feedbacks. An extensive body of literature documents the important role of plant–soil feedbacks, particularly the escape from pathogens in the native range, in promoting invasion success<sup><a href="#CR15">15</a>, <a href="#CR41">41</a>, <a href="#CR42">42</a></sup>. These conclusions, however, are typically inferred from measuring plant growth responses when grown in soils that have been preconditioned with either native or invasive plants<sup><a href="#CR43">43</a></sup>, or from soils collected from native and invasive ranges<sup><a href="#CR44">44</a></sup>. Thus, in most cases the actual microbial mechanism for observed negative soil feedbacks is implied rather than documented. Although we have yet to test how the bacterial community changes after cross transplanting native and invasive plants into preconditioned soils, our work nonetheless sheds light on possible bacterial underpinnings of plant–soil feedbacks.</p> <p id="Par17">We hypothesized that if native lineages were under constant enemy attack they would have evolved specific associations with specialist rhizosphere bacteria that could aid in their defense from pathogens<sup><a href="#CR15">15</a>, <a href="#CR45">45</a>, <a href="#CR46">46</a></sup>. As a result, the bacterial communities associated with the native lineage should be similar regardless of geographical location. The much greater similarity observed within the Native and Gulf lineages, compared to the Introduced lineage, both in our field and our common garden experiment, supports this conclusion. The different abundances of important protein pathways that we observe (Fig. <a href="#Fig4">4</a>) add additional support for the enemy release hypothesis. Introduced <i>P. australis</i> rhizosphere microbial communities had significantly lower abundances of pathways involved in antimicrobial biosynthesis and degradation, suggesting a lower exposure to enemy attack. Introduced <i>P. australis</i>, when freed from pathogens that were present in its native range, cease to dedicate resources to cultivation of a pathogen defense system. Thus, the bacterial community in the rhizosphere soils should show a greater contribution of local soil generalists, rather than specific taxa cultivated for pathogen defense. The greater change in bacterial turnover with distance in the Introduced lineage compared to the other lineages suggests that Introduced populations have a greater reliance on novel associations in soils from the introduced range that are derived from the pool of local microflora, a pool that would be expected to decrease in similarity with distance from the source population<sup><a href="#CR47">47</a></sup>.</p> <p id="Par18">Our results support that the <i>P. australis</i> lineage played a critical role in structuring bacterial communities across large spatial scales. The importance of lineage overwhelmed the importance of local environmental factors in structuring bacterial communities both in the field and in a common garden greenhouse experiment. Globally, the spread of invasive species has reduced biodiversity<sup><a href="#CR48">48</a></sup> and altered ecosystem function<sup><a href="#CR49">49</a></sup>, and this work highlights another potential impact of species invasions—lineage-specific reshaping of endemic microbial communities. The higher belowground biomass of the Introduced <i>P. australis</i> lineage, compared to the Native lineage and to other native wetland plants it displaces, suggests that invasion-induced changes in microbial community could be widespread, though further work is needed to determine how far into the bulk soil these changes propagate. Microbes play key roles in ecosystem function and this bottom-up engineering of the rhizosphere microbial community assembly by plant lineage could have repercussions for important ecosystem services provided by microbes, such as nutrient cycling and carbon sequestration. It is therefore essential that we take the next step to determine how these lineage-specific changes in the rhizosphere bacterial community translate to changes in <i>P. australis</i> associated ecosystem function.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="fieldsampling" title="sec"> <div class="title" tagx="title" title="title">
Field sampling</div> <p id="Par19">During the summer and fall of 2015, we collected soil samples to assess the bacterial communities associated with Native, Introduced, and Gulf <i>P. australis</i> rhizosphere (Supplementary Table <a href="#MOESM1">1</a>, Supplementary Fig. <a href="#MOESM1">1</a>). We ran 20 m transects through the middle of previously identified stands of <i>P. australis</i> and excavated belowground material every 5 m, starting with an initial sampling at 0 m and collecting additional samples at 5, 10, 15 and 20 m distances (<i>n</i> = 5 samples per transect, which we have previously shown to be robust to capture the dynamics within and between different plant rhizosphere bacterial communities<sup><a href="#CR50">50</a></sup>). At each sampling location we collected the soil directly associated with the roots. Soil samples were preserved in RNAlater and stored at −20 °C. At the same time, aboveground plant tissue was collected to verify lineage and for measurement of carbon, nitrogen, and gallic acid concentration. Haplotypes of <i>P. australis</i> lineages were confirmed by cpDNA sequencing<sup><a href="#CR23">23</a></sup>.</p> </div> <div class="commongardenexperimentaldesign" title="sec"> <div class="title" tagx="title" title="title">
Common garden experimental design</div> <p id="Par20">We maintain multiple populations of <i>P. australis</i>, whose lineages have been confirmed by RFLP analysis according to the methods of Saltonstall<sup><a href="#CR23">23</a></sup>, in a laboratory at the University of Rhode Island. For this experiment, we selected 15 different source populations that had been growing in the Rhode Island common garden for at least 3 years. Three source populations were from the Native lineage, two were from the Gulf lineage, and ten were from the Introduced lineage. The original plants were collected from across Eastern North America, spanning a latitudinal gradient from Quebec to Louisiana. Multiple individuals of each plant were propagated in water in the greenhouse from same length stem cuttings so replicates of each population were genetically identical and of standard size. When the aerial shoots of the propagules were sufficiently large (&amp;gt;2.5 cm in length) they were separated from the larger stem on which they were growing by cutting the stem 2.5 cm on either side of the node from which the stem emerged. The surfaces of the plants and roots were dipped in a 10% bleach solution to remove any surface associated microbes and each propagule was individually planted in a clean, bleached cone-shaped "deepot" container (2.5" × 10", D40L, Stuewe and Sons, Inc.) filled with sifted and homogenized Metromix potting soil. Individual deepots were randomly positioned in racks and racks were rotated regularly to avoid location-specific effects on plant growth. Plants were grown in the greenhouse under identical conditions of water and light for 4 months, at which point individual plants were harvested. To harvest, we gently removed the entire plant from the cone. After allowing loose soil to shake free, we then collected remaining root associated soil into individual cryovials that were immediately frozen in liquid nitrogen. Remaining roots were cleaned of any residual soil and roots, shoots, and residual soils were separately bagged and dried for later analysis.</p> </div> <div class="soilandplanttissueanalysis" title="sec"> <div class="title" tagx="title" title="title">
Soil and plant tissue analysis</div> <p id="Par21">Plant shoots and roots were oven-dried at 70 °C to a constant biomass and weighed to determine the total aboveground and belowground biomass for each plant. Leaves were lyophilized and ground to a fine powder in the laboratory. Carbon and nitrogen content of both the aboveground and belowground plant tissue was determined on a Perkin Elmer 2400 Series Elemental Analyzer (Waltham, MA) at the University of Massachusetts Environmental Analytical Facility following standard operating procedures. The coefficient of variation in replicate analysis was 0.016 for %C and 0.025 for %N. Total phenolics (nM g<sup>−1</sup> of dried plant tissue) were estimated using a microplate modified version of the Folin–Ciocalteu method<sup><a href="#CR51">51</a></sup>. Colorimetric methods were used to determine NO<sub>3</sub><sup>−</sup><sup><a href="#CR52">52</a></sup>) and NH<sub>4</sub><sup>+</sup><sup><a href="#CR53">53</a></sup>) concentrations using a Bio-Tek microplate reader (Powerwave 340, Winooski, VT).</p> </div> <div class="nucleicacidextraction,amplification,andsequencing" title="sec"> <div class="title" tagx="title" title="title">
Nucleic acid extraction, amplification, and sequencing</div> <p id="Par22">Five replicates from each population were haphazardly selected for molecular analysis of the total and active bacterial communities. DNA was extracted from approximately 0.25 g of soil using the MoBio PowerSoil DNA Isolation Kit (Carlsbad, CA) following manufacturer’s instructions. RNA was extracted from an additional 0.5 g of soil following a protocol originally developed by Mettle et al.<sup><a href="#CR54">54</a></sup> and modified as described by Kearns et al.<sup><a href="#CR50">50</a></sup>. Briefly, cells were lysed by vortexing with 0.17 mm glass beads in 700 µl PBL buffer (5 mM Tris-HCl (pH 5.0), 5 mM Na<sub>2</sub>EDTA, 0.1% (wt/vol) sodium dodecyl sulfate, and 6% (vol/vol) water-saturated phenol) for 10 min. Following centrifugation, the supernatant was transferred to a new centrifuge tube and the remaining soil and glass beads were resuspended in 700 µl TPM buffer (50 mM Tris-HCl (pH 5.0), 1.7% (wt/vol) polyvinylpyrrolidone, and 20 mM MgCl<sub>2</sub>) and vortexed at maximum speed for an additional 10 min. Combined supernatant was treated with an equal volume of phenol:choloroform:isoamyl alcohol (25:24:1 v/v/v; pH 5.5) before nucleic acids were precipitated with 100% isopropanol and 3 M sodium acetate and washed with 70% ethanol. The RNA was loaded on an Illustra Autoseq G-50 Spin Column (GE Healthcare, Pittsburgh, PA) containing 500 µl prewashed Q-Sepharose (GE Healthcare). Samples were spun at 650 × <i>g</i> and eluted five times with 80 µl of 1.5 M NaCl (pH 5.5). The eluate was again precipitated with 100% isopropanol and 3 M sodium acetate, and washed with 70% ethanol before being eluted into 50 µl di-ethyl pyrocarbonate treated water. Any DNA contamination in the RNA was removed with DNase I (New England Biolabs, Ipswich, MA) following manufacturer’s instructions and RNA was reverse transcribed to cDNA with random hexamer primers using the Invitrogen Superscript RT III cDNA synthesis kit (Grand Island, NY). cDNA synthesis was checked by amplification with general bacterial primers and products were visualized via gel electrophoresis on an ethidium bromide stained 1.5% agarose gel.</p> <p id="Par23">DNA and cDNA from each sample was quantified via Qubit 2.0 (ThermoFisher, Waltham, MA, USA) and Picogreen (Invitrogen, Grand Island, NY) and all samples were normalized to a nucleic acid concentration of 3 ng μl<sup>−1</sup> for subsequent PCR. We amplified a 291 base pair fragment of the V4 region of the 16S rRNA and 16S rRNA gene using primers 515 F and 806R<sup><a href="#CR55">55</a></sup> that were adapted for sequencing<sup><a href="#CR56">56</a></sup> on an Illumina MiSeq (San Diego, CA). Each sample was amplified in triplicate with a unique 12 base barcode to enable pooling of multiple samples per sequencing run. PCR products were cleaned via gel purification using the Qiagen QIAquick gel extraction kit (Valencia, CA) and were fluorometrically quantified using a ThermoFisher Qubit (Waltham, MA). Final PCR products were pooled in equimolar ratios and sequenced using the Illumina MiSeq platform with a 300 cycle V2 chemistry sequencing kit.</p> </div> <div class="dataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Data analysis</div> <p id="Par24">After sequencing, fastq-join<sup><a href="#CR57">57</a></sup> was used to match paired end reads using default parameters and the resulting data were demultiplexed and quality filtered<sup><a href="#CR58">58</a></sup> in QIIME<sup><a href="#CR59">59</a></sup> (version 1.9.1). We used closed-reference picking with uclust<sup><a href="#CR60">60</a></sup> to determine operational taxonomic units (OTUs) at a 97% sequence identity using the Greengenes 13.8 Core Reference Alignment<sup><a href="#CR61">61</a></sup>. Chimeric sequences were identified and removed using Chimera Slayer<sup><a href="#CR62">62</a></sup> and singletons were also removed prior to downstream analysis. We compared the bacterial community composition of each sample by calculating the Bray–Curtis dissimilarity metric and we visualized the resulting distance metric using a principal coordinates analysis in QIIME. Alpha diversity was calculated based on data rarified to the lowest sequencing depth, 5394 sequences per sample. We calculated a core microbiome, defined as taxa that were present in 100% of the samples within a given lineage using QIIME, and we assessed significant differences in taxonomic abundance among the different lineages using a Kruskal–Wallis test followed by a Benjamini–Hochberg correction to correct for false discovery.</p> <p id="Par25">We assessed differences in bacterial community structure among Native, Gulf, and Introduced <i>P. australis</i> lineages using Adonis, which is a PERMANOVA<sup><a href="#CR63">63</a></sup> implemented in QIIME, with significance assessed at an alpha of 0.01. We assessed changes in community structure with distance by plotting the pairwise Bray–Curtis dissimilarity coefficient against pairwise distance between locations and calculated the linear regression between the two in R. We used PiCrust<sup><a href="#CR64">64</a></sup>, which assigns function based on 16S rRNA marker sequences using an ancestral state reconstruction approach, to determine bacterial metabolisms. To assess differences in potential bacterial metabolisms among lineages, we implemented PiCrust on Galaxy<sup><a href="#CR64">64</a>–<a href="#CR66">66</a></sup>, using closed-reference OTU picking in QIIME. The resulting biom file was uploaded to Galaxy (<a href="http://huttenhower.sph.harvard.edu/galaxy/">http://huttenhower.sph.harvard.edu/galaxy/</a>) and data were normalized to account for multiple copies of 16S rRNA in some bacteria. After normalization PiCrust predicted the metagenomic content of each sample. We then used HUManN<sup><a href="#CR67">67</a></sup> to generate pathway summaries and separated out those pathways associated with bacterial metabolisms for use in the structural equation model. We tested for significant differences in three specific pathway associated with plant–microbe interactions, toxin and metal detoxification, antimicrobial biosynthesis, and antimicrobial degradation using analysis of variance, followed by Tukey’s post-hoc test in R.</p> </div> <div class="structuralequationmodeling" title="sec"> <div class="title" tagx="title" title="title">
Structural equation modeling</div> <p id="Par26">We used mixed model Structural Equation Modeling (SEM)<sup><a href="#CR45">45</a>–<a href="#CR47">47</a></sup> to assess the interactions among <i>P. australis</i> lineage, bacterial processes, and plant and soil properties. All data and code for the SEM is available at: at <a href="https://github.com/jebyrnes/phrag_common_garden_sem">https://github.com/jebyrnes/phrag_common_garden_sem</a>. Parameters in the model included <i>P. australis</i> lineage, above and belowground plant biomass, belowground gallic acid concentration (phenolics), soil percents and nitrogen, and three bacterial parameters: number of OTUs, activity (per taxon ratio of 16S rRNA to the 16S rRNA gene), and metabolism (calculated from KEGG pathways indicated by PiCrust). We allowed <i>P. australis</i> lineage to affect bacterial richness (number of taxa), activity (16S rRNA:16S rRNA gene ratio), and metabolism (based on ancestral state reconstruction of metabolic pathways). Lineage and all ecosystem variables could affect aboveground and belowground biomass, and all variables influenced soil carbon and nitrogen pools, which could also covary.</p> <p id="Par27">We fit and evaluated the model using restricted maximum likelihood with the <i>nlme</i> and <i>piecewiseSEM</i> package<sup><a href="#CR47">47</a>, <a href="#CR68">68</a></sup> in R (version 3.31). Given that lineage was categorical, we evaluated whether different variables influenced one another using <i>χ</i><sup>2</sup> Likelihood ratio tests for individual model pieces. We then report the estimated path coefficient. We compared individual lineage effects using post-hoc means adjusted Tukey tests<sup><a href="#CR69">69</a></sup>. Exploration of non-additive models with <i>P. australis</i> lineages interacting with different endogenous variables showed no nonlinearities.</p> </div> <div class="dataavailability" title="sec"> <div class="title" tagx="title" title="title">
Data availability</div> <p id="Par28">All sequences associated with this study are available from the Sequence Read Archive under accession numbers SRR4419841 (greenhouse associated sequences) and SRR4420130 (field collected sequences). All data and code associated with the structural equation modeling are available at <a href="https://github.com/jebyrnes/phrag_common_garden_sem">https://github.com/jebyrnes/phrag_common_garden_sem</a>.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41467_2017_626_MOESM1_ESM.pdf">LINK</a> <p>Supplementary Information</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41467_2017_626_MOESM2_ESM.pdf">LINK</a> <p>Peer Review File</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Electronic supplementary material</b></p> <p><b>Supplementary Information</b> accompanies this paper at doi:10.1038/s41467-017-00626-0.</p> </div> <div class="fn-type-" title=""> <p><b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Petr Pyšek, A. Randall Hughes and José Amador for comments on earlier versions of this manuscript. Funding was provided by NSF grants to J.L.B. (DEB1350491), L.A.M. (DEB1049914), J.T.C. (DEB1050084), and J.T.C. and W.J.A. (DEB1501775). Additional support was provided by the University of Rhode Island to L.A.M. (RI00H-332, 311000-6044), the Louisiana Environmental Education Commission to W.J.A. and MIT Sea Grant to J.E.K.B. (2014-R/RCM-36). We thank Rose Martin and Serena Moseman-Valtierra for early discussions on this project and Jeff Dusenberry and the Research Computing Center at University of Massachusetts Boston.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>J.L.B., L.A.M., W.J.A. and J.T.C. designed the experiments, J.E.K.B. performed structural equation modeling, S.W. and W.J.A. collected field samples, L.A.M. and S.W. maintained the greenhouse experiment, P.J.K., M.G., K.T., S.W. and J.Y. performed all sequence processing and analysis under the guidance of J.L.B., J.T.C. provided logistical support for field operations, J.L.B., L.A.M., W.J.A., J.T.C., J.E.K.B. and P.J.K. wrote the manuscript.</p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par29">The authors declare no competing financial interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamerlynck</span><span class="given-names" tagx="given-names" title="given-names">EP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Interactions between the salt marsh grass <i>Spartina patens</i>, arbuscular mycorrhizal fungi, and sediment bacteria during the growing season</span><span class="source" tagx="source" title="source">Soil Biol. Biochem.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">501</span><span class="lpage" tagx="lpage" title="lpage">511</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0038-0717(03)00004-X">10.1016/S0038-0717(03)00004-X</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bever</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Platt</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morton</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span></span><span class="mixed-article-title" title="mixed-article-title">Microbial population and community dynamics on plant roots and their feedbacks on plant communities</span><span class="source" tagx="source" title="source">Annu. Rev. Microbiol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">265</span><span class="lpage" tagx="lpage" title="lpage">283</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-micro-092611-150107">10.1146/annurev-micro-092611-150107</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22726216">22726216</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brucker</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bordenstein</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span></span><span class="mixed-article-title" title="mixed-article-title">The roles of host evolutionary relationships (Genus: <i>Nasonia</i>) and development in structuring microbial communities</span><span class="source" tagx="source" title="source">Evolution</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">349</span><span class="lpage" tagx="lpage" title="lpage">362</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.2011.01454.x">10.1111/j.1558-5646.2011.01454.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22276533">22276533</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Org</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic and environmental control of host-gut microbiota interactions</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1558</span><span class="lpage" tagx="lpage" title="lpage">1569</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.194118.115">10.1101/gr.194118.115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26260972">26260972</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davenport</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span></span><span class="mixed-article-title" title="mixed-article-title">Elucidating the role of the host genome in shaping microbiome composition</span><span class="source" tagx="source" title="source">Gut Microbes</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">178</span><span class="lpage" tagx="lpage" title="lpage">184</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/19490976.2016.1155022">10.1080/19490976.2016.1155022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26939746">26939746</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Horton, M. W. et al. Genome-wide association study of <i>Arabidopsis thaliana</i> leaf microbial community. <i>Nat. Comm</i>. <b>5</b>, 5320 (2014).</span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peiffer</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diversity and heritability of the maize rhizosphere microbiome under field conditions</span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">110</span><span class="fpage" tagx="fpage" title="fpage">6548</span><span class="lpage" tagx="lpage" title="lpage">6553</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1302837110">10.1073/pnas.1302837110</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23576752">23576752</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Micallef</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shiaris</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colón-Carmona</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influence of <i>Arabidopsis thaliana</i> accessions on rhizobacterial communities and natural variation in root exudates</span><span class="source" tagx="source" title="source">J. Exp. Bot.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">1729</span><span class="lpage" tagx="lpage" title="lpage">1742</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/erp053">10.1093/jxb/erp053</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19342429">19342429</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahoney</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hulbert</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Community structure, species variation, and functional characterization of rhizosphere-associated bacteria of different winter wheat (<i>Triticum aestivum</i>) cultivars</span><span class="source" tagx="source" title="source">Front. Plant Sci.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">132</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2017.00132">10.3389/fpls.2017.00132</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28243246">28243246</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aira</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gómez-Brandón</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lazcano</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bååth</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domínguez</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant genotype strongly modifies the structure and growth of maize rhizosphere microbial communities</span><span class="source" tagx="source" title="source">Soil. Biol. Biochem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">2276</span><span class="lpage" tagx="lpage" title="lpage">2281</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2010.08.029">10.1016/j.soilbio.2010.08.029</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouffaud</span><span class="given-names" tagx="given-names" title="given-names">M-L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Is diversification history of maize influencing selection of soil bacteria by roots?</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">195</span><span class="lpage" tagx="lpage" title="lpage">206</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2011.05359.x">10.1111/j.1365-294X.2011.05359.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22126532">22126532</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure, variation, and assembly of the root-associated microbiomes of rice</span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">112</span><span class="fpage" tagx="fpage" title="fpage">E911</span><span class="lpage" tagx="lpage" title="lpage">E920</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1414592112">10.1073/pnas.1414592112</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25605935">25605935</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shakya</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A multifactor analysis of fungal and bacterial community structure in the root microbiome of mature <i>Populus deltoids</i> trees</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e76382</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0076382">10.1371/journal.pone.0076382</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24146861">24146861</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Busby</span><span class="given-names" tagx="given-names" title="given-names">PE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Research priorities for harnessing plant microbiomes in sustainable agriculture</span><span class="source" tagx="source" title="source">PLoS Biol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">e2001793</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.2001793">10.1371/journal.pbio.2001793</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28350798">28350798</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reinhart</span><span class="given-names" tagx="given-names" title="given-names">KO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callaway</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Soil biota and invasive plants</span><span class="source" tagx="source" title="source">New. Phytol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">170</span><span class="fpage" tagx="fpage" title="fpage">445</span><span class="lpage" tagx="lpage" title="lpage">457</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1469-8137.2006.01715.x">10.1111/j.1469-8137.2006.01715.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16626467">16626467</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blossey</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Notzold</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolution of increased competitive ability in invasive nonindigenous plants: a hypothesis</span><span class="source" tagx="source" title="source">J. Ecol.</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">887</span><span class="lpage" tagx="lpage" title="lpage">899</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/2261425">10.2307/2261425</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hulme</span><span class="given-names" tagx="given-names" title="given-names">PE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bias and error in understanding plant invasion impacts</span><span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">212</span><span class="lpage" tagx="lpage" title="lpage">218</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tree.2012.10.010">10.1016/j.tree.2012.10.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23153723">23153723</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyerson</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cronin</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyšek</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Phragmites australis</i> as a model organism for studying plant invasions</span><span class="source" tagx="source" title="source">Biol. Invasions</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">2421</span><span class="lpage" tagx="lpage" title="lpage">2431</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-016-1132-3">10.1007/s10530-016-1132-3</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambertini</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Tracing the origin of Gulf Coast <i>Phragmites</i> (Poaceae): a story of long-distance dispersal and hybridization</span><span class="source" tagx="source" title="source">Am. J. Bot.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">538</span><span class="lpage" tagx="lpage" title="lpage">551</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/ajb.1100396">10.3732/ajb.1100396</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22334449">22334449</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cryptic invasion by a non-native genotype of the common reed, <i>Phragmites australis</i>, into North America</span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">2445</span><span class="lpage" tagx="lpage" title="lpage">2449</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.032477999">10.1073/pnas.032477999</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11854535">11854535</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauber</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powers</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeFrancesch</span><span class="given-names" tagx="given-names" title="given-names">FR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isozyme variation and correspondence with unusual infrared reflectance pattern in <i>Phragmites australis</i> (Poaceae)</span><span class="source" tagx="source" title="source">Plant Syst. Evol.</span><span class="year" tagx="year" title="year">1991</span><span class="volume" tagx="volume" title="volume">178</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00937978">10.1007/BF00937978</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pellgrin</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauber</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isozyme variation among populations of the clonal species <i>Phragmites australis</i> (Cav.) Trin. ex Steudel</span><span class="source" tagx="source" title="source">Aquat. Bot.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">241</span><span class="lpage" tagx="lpage" title="lpage">259</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0304-3770(98)00120-X">10.1016/S0304-3770(98)00120-X</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">A rapid method for identifying the origin of North American <i>Phragmites</i> populations using RFLP analysis</span><span class="source" tagx="source" title="source">Wetlands</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">1043</span><span class="lpage" tagx="lpage" title="lpage">1047</span><span class="pub-id"><a href="https://dx.doi.org/10.1672/0277-5212(2003)023[1043:ARMFIT]2.0.CO;2">10.1672/0277-5212(2003)023[1043:ARMFIT]2.0.CO;2</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauber</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Notes on <i>Phragmites australis</i> (Poaceae: Arundinoideae) in North America</span><span class="source" tagx="source" title="source">J. Bot. Res. Inst. Texas</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">385</span><span class="lpage" tagx="lpage" title="lpage">388</span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauber</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hood</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic variation in the common reed <i>Phragmites australis</i> in the Mississippi River Delta marshes: evidence for multiple introductions</span><span class="source" tagx="source" title="source">Estuar. Coasts</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">851</span><span class="lpage" tagx="lpage" title="lpage">862</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s12237-011-9391-9">10.1007/s12237-011-9391-9</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eguiarte</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogeographic analyses and genetic structure illustrate the complex evolutionary history of <i>Phragmites australis</i> in Mexico</span><span class="source" tagx="source" title="source">Am. J. Bot.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">876</span><span class="lpage" tagx="lpage" title="lpage">887</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/ajb.1500399">10.3732/ajb.1500399</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27208356">27208356</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyerson</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambert</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saltonstall</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">A tale of three lineages: expansion of common reed (<i>Phragmites australis</i>) in the U.S. Southwest and Gulf Coast</span><span class="source" tagx="source" title="source">Invasive Plant Sci. Manag.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">515</span><span class="lpage" tagx="lpage" title="lpage">520</span><span class="pub-id"><a href="https://dx.doi.org/10.1614/IPSM-D-09-00052.1">10.1614/IPSM-D-09-00052.1</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warren</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Shifts in diversity and community structure of endophytic bacteria and archaea across root, stem and leaf tissues in the common reed, <i>Phragmites australis</i>, along a salinity gradient in a marine tidal wetland of northern China</span><span class="source" tagx="source" title="source">Antonie Van Leeuwenhoek</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">104</span><span class="fpage" tagx="fpage" title="fpage">759</span><span class="lpage" tagx="lpage" title="lpage">768</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10482-013-9984-3">10.1007/s10482-013-9984-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23897211">23897211</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fierer</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span></span><span class="mixed-article-title" title="mixed-article-title">The diversity and biogeography of soil bacterial communities</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">626</span><span class="lpage" tagx="lpage" title="lpage">631</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0507535103">10.1073/pnas.0507535103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16407148">16407148</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauber</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamady</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knight</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fierer</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pyrosequencing-based assessment of soil pH as a predictor of soil bacterial community structure at the continental scale</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">5111</span><span class="lpage" tagx="lpage" title="lpage">5120</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.00335-09">10.1128/AEM.00335-09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19502440">19502440</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borruso</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bacci</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mengoni</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DePhilippis</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brusetti</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rhizosphere effect and salinity competing to shape microbial communities in <i>Phragmites australis</i> (Cav.) Trin. ex-Steud</span><span class="source" tagx="source" title="source">FEMS Microbiol. Lett.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">359</span><span class="fpage" tagx="fpage" title="fpage">193</span><span class="lpage" tagx="lpage" title="lpage">200</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1574-6968.12565">10.1111/1574-6968.12565</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25131902">25131902</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langille</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Predictive functional profiling of microbial communities using 16S rRNA marker gene sequences</span><span class="source" tagx="source" title="source">Nat. Biotech.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">814</span><span class="lpage" tagx="lpage" title="lpage">821</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.2676">10.1038/nbt.2676</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shipley</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Confirmatory path analysis in a generalized multilevel context</span><span class="source" tagx="source" title="source">Ecology</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">363</span><span class="lpage" tagx="lpage" title="lpage">368</span><span class="pub-id"><a href="https://dx.doi.org/10.1890/08-1034.1">10.1890/08-1034.1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19323220">19323220</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grace</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Guidelines for a graph-theoretic implementation of structural equation modeling</span><span class="source" tagx="source" title="source">Ecosphere</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="https://dx.doi.org/10.1890/ES12-00048.1">10.1890/ES12-00048.1</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lefcheck</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">piecewiseSEM: Piecewise structural equation modeling in R for ecology, evolution, and systematics</span><span class="source" tagx="source" title="source">Methods Ecol. Evol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">573</span><span class="fpage" tagx="fpage" title="fpage">573</span><span class="lpage" tagx="lpage" title="lpage">579</span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Allen, W. J., Meyerson L. A., Flick, A. J. &amp;amp; Cronin J. T. Indirect effects of plant-soil feedbacks on competition between native and invasive wetland plants. Preprint at <a href="http://www.biorxiv.org/content/early/2017/07/07/160523">http://www.biorxiv.org/content/early/2017/07/07/160523</a> (2017).</span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Shasta ground sloth food habits, Ramparts Cave, Arizona</span><span class="source" tagx="source" title="source">Paleobiology.</span><span class="year" tagx="year" title="year">1978</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">302</span><span class="lpage" tagx="lpage" title="lpage">319</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0094837300006011">10.1017/S0094837300006011</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orson</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="mixed-article-title" title="mixed-article-title">A paleoecological assessment of <i>Phragmites australis</i> in New England tidal marshes: changes in plant community structure during the last few millennia</span><span class="source" tagx="source" title="source">Biol. Invasions</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">149</span><span class="lpage" tagx="lpage" title="lpage">158</span><span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1010047731369">10.1023/A:1010047731369</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">EB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karp</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Soil pathogen communities associated with native and non-native <i>Phragmites australis</i>, populations in freshwater wetlands</span><span class="source" tagx="source" title="source">Ecol. Evol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">5254</span><span class="lpage" tagx="lpage" title="lpage">5267</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.900">10.1002/ece3.900</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24455153">24455153</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yarwood</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldwin</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mateu</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buyer</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Archaeal rhizosphere communitites differ between the native and invasive lineages of the wetland plant <i>Phragmites australis</i> (common reed) in a Chesapeake Bay subestuary</span><span class="source" tagx="source" title="source">Biol. Invasions</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">2717</span><span class="lpage" tagx="lpage" title="lpage">2728</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-016-1144-z">10.1007/s10530-016-1144-z</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klironomos</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span></span><span class="mixed-article-title" title="mixed-article-title">Feedback with soil biota contributes to plant rarity and invasiveness in communities</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">417</span><span class="fpage" tagx="fpage" title="fpage">67</span><span class="lpage" tagx="lpage" title="lpage">70</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/417067a">10.1038/417067a</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11986666">11986666</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maron</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callaway</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pal</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Do exotic plants lose resistance to pathogenic soil biota from their native range? A test with <i>Solidago gigantea</i></span><span class="source" tagx="source" title="source">Oecologia</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">179</span><span class="fpage" tagx="fpage" title="fpage">447</span><span class="lpage" tagx="lpage" title="lpage">454</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00442-015-3350-2">10.1007/s00442-015-3350-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26003309">26003309</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulmatiski</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beard</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevens</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cobbold</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant-soil feedbacks: a meta-analytical review</span><span class="source" tagx="source" title="source">Ecol. Lett.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">980</span><span class="lpage" tagx="lpage" title="lpage">992</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1461-0248.2008.01209.x">10.1111/j.1461-0248.2008.01209.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18522641">18522641</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maron</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Staged invasions across disparate grasslands: effects of seed provenance, consumers and disturbance on productivity and species richness</span><span class="source" tagx="source" title="source">Ecol. Lett.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">499</span><span class="lpage" tagx="lpage" title="lpage">507</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/ele.12250">10.1111/ele.12250</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24467348">24467348</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Elton, C. S. <i>The Ecology of Invasions by Animals and Plants</i> (University of Chicago Press, 1958).</span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keane</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crawley</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exotic plant invasions and the enemy release hypothesis</span><span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">164</span><span class="lpage" tagx="lpage" title="lpage">170</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0169-5347(02)02499-0">10.1016/S0169-5347(02)02499-0</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martiny</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Microbial biogeography: putting microorganisms on the map</span><span class="source" tagx="source" title="source">Nat. Rev. Microbiol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">102</span><span class="lpage" tagx="lpage" title="lpage">112</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro1341">10.1038/nrmicro1341</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16415926">16415926</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyšek</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Invasive species, environmental change and management, and health</span><span class="source" tagx="source" title="source">Annu. Rev. Environ. Resour.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="lpage" tagx="lpage" title="lpage">55</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-environ-033009-095548">10.1146/annurev-environ-033009-095548</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pejchar</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mooney</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Invasive species, ecosystem services, and human well-being</span><span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">497</span><span class="lpage" tagx="lpage" title="lpage">504</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tree.2009.03.016">10.1016/j.tree.2009.03.016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19577817">19577817</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kearns</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Nutrient enrichment induces high rates of dormancy and decreases diversity of active bacterial taxa in salt marshes</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">12881</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms12881">10.1038/ncomms12881</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27666199">27666199</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cronin</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattarai</span><span class="given-names" tagx="given-names" title="given-names">GP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyerson</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biogeography of a plant invasion: plant–herbivore interactions</span><span class="source" tagx="source" title="source">Ecology</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">96</span><span class="fpage" tagx="fpage" title="fpage">1115</span><span class="lpage" tagx="lpage" title="lpage">1127</span><span class="pub-id"><a href="https://dx.doi.org/10.1890/14-1091.1">10.1890/14-1091.1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26230031">26230031</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doane</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horwath</span><span class="given-names" tagx="given-names" title="given-names">WR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spectrophotometric determination of nitrate with a single reagent</span><span class="source" tagx="source" title="source">Anal. Lett.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">2713</span><span class="lpage" tagx="lpage" title="lpage">2722</span><span class="pub-id"><a href="https://dx.doi.org/10.1081/AL-120024647">10.1081/AL-120024647</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weatherburn</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phenol-hypochlorite reaction for determination of ammonia</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">1967</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">971</span><span class="lpage" tagx="lpage" title="lpage">974</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ac60252a045">10.1021/ac60252a045</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mettel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shrestha</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liesack</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Extraction of mRNA from soil</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">5995</span><span class="lpage" tagx="lpage" title="lpage">6000</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.03047-09">10.1128/AEM.03047-09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20622132">20622132</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caporaso</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Global patterns of 16S rRNA diversity at a depth of millions of seuqences per sample</span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">4516</span><span class="lpage" tagx="lpage" title="lpage">4522</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1000080107">10.1073/pnas.1000080107</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20534432">20534432</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caporaso</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ultra-high-throughput microbial community analysis on the Illumina HiSeq and MiSeq platforms</span><span class="source" tagx="source" title="source">ISME J.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">1621</span><span class="lpage" tagx="lpage" title="lpage">1624</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ismej.2012.8">10.1038/ismej.2012.8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22402401">22402401</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Aronesty, E. <i>ea-utils</i>: Command-line tools for processing biological sequencing data, <a href="http://code.google.com/p/ea-utils">http://code.google.com/p/ea-utils</a> (2011).</span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bokulich</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Quality-filtering vastly improves diversity estimates from Illumina amplicon sequencing</span><span class="source" tagx="source" title="source">Nat. Meth.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">57</span><span class="lpage" tagx="lpage" title="lpage">59</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.2276">10.1038/nmeth.2276</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caporaso</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">QIIME allows analysis of high-throughput community sequencing data</span><span class="source" tagx="source" title="source">Nat. Meth.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">335</span><span class="lpage" tagx="lpage" title="lpage">336</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.f.303">10.1038/nmeth.f.303</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Search and clustering orders of magnitude faster than BLAST</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">2460</span><span class="lpage" tagx="lpage" title="lpage">2461</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btq461">10.1093/bioinformatics/btq461</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20709691">20709691</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeSantis</span><span class="given-names" tagx="given-names" title="given-names">TZ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with ARB</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">72</span><span class="fpage" tagx="fpage" title="fpage">5069</span><span class="lpage" tagx="lpage" title="lpage">5072</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.03006-05">10.1128/AEM.03006-05</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16820507">16820507</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Chimeric 16S rRNA sequence formation and detection in Sanger and 454-pyrosequenced PCR amplicons</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">494</span><span class="lpage" tagx="lpage" title="lpage">504</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.112730.110">10.1101/gr.112730.110</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21212162">21212162</a></span></span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">A new method for non-parametric multivariate analysis of variance</span><span class="source" tagx="source" title="source">Austral Ecol.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">32</span><span class="lpage" tagx="lpage" title="lpage">46</span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Blankenberg, D. et al. Galaxy: a web-based genome analysis tool for experimentalists. <i>Curr. Prot. Mol. Biol</i>. 89:19.10 19.10.1–19.10.21 (2010).</span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giardine</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Galaxy: a platform for interactive large-scale genome analysis</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">1451</span><span class="lpage" tagx="lpage" title="lpage">1455</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.4086505">10.1101/gr.4086505</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16169926">16169926</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goecks</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences</span><span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">R86</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2010-11-8-r86">10.1186/gb-2010-11-8-r86</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20738864">20738864</a></span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abubucker</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic reconstruction for metagenomic data and its application to the human microbiome</span><span class="source" tagx="source" title="source">PLoS Comput. Biol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e1002358</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1002358">10.1371/journal.pcbi.1002358</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22719234">22719234</a></span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Pinheiro, J. C. &amp;amp; Bates, D. M. <i>Mixed-effects models in S and S-PLUS</i> (Springer, 2000).</span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lenth</span><span class="given-names" tagx="given-names" title="given-names">RV</span></span></span><span class="mixed-article-title" title="mixed-article-title">Least-squares means: the R package Lsmeans</span><span class="source" tagx="source" title="source">J. Stat. Softw.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">33</span><span class="pub-id"><a href="https://dx.doi.org/10.18637/jss.v069.i01">10.18637/jss.v069.i01</a></span></span></li> </ul> </div> </div>  </body></html>