<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Metabolites</span><span class="iso-abbrev" title="iso-abbrev">Metabolites</span><span class="publisher-id" title="publisher-id">metabolites</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Metabolites</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2218-1989</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7822174">7822174</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/33375173">33375173</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/metabo11010002">10.3390/metabo11010002</a></span><span class="publisher-id" title="publisher-id">metabolites-11-00002</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Untargeted Metabolomics Studies on Drug-Incubated <i>Phragmites australis</i> Profiles</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-7627-8779</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wahman</span><span class="given-names" tagx="given-names" title="given-names">Rofida</span></span><a href="#af1-metabolites-11-00002">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-3611-0346</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SauvÃªtre</span><span class="given-names" tagx="given-names" title="given-names">Andres</span></span><a href="#af2-metabolites-11-00002">2</a><a href="#fn1-metabolites-11-00002">â€ </a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-7042-8094</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchrÃ¶der</span><span class="given-names" tagx="given-names" title="given-names">Peter</span></span><a href="#af2-metabolites-11-00002">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-7881-5697</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moser</span><span class="given-names" tagx="given-names" title="given-names">Stefan</span></span><a href="#af3-metabolites-11-00002">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letzel</span><span class="given-names" tagx="given-names" title="given-names">Thomas</span></span><a href="#af1-metabolites-11-00002">1</a><a href="#af4-metabolites-11-00002">4</a><a href="#c1-metabolites-11-00002">*</a></span></div><span class="citation_author_institution" id="af1-metabolites-11-00002">[1], <span class="email" tagx="email" title="email">rofida.wahman@tum.de</span></span><span class="citation_author_institution" id="af2-metabolites-11-00002">[2], <span class="email" tagx="email" title="email">andre.sauvetre@umontpellier.fr</span><span class="email" tagx="email" title="email">peter.schroeder@helmholtz-muenchen.de</span></span><span class="citation_author_institution" id="af3-metabolites-11-00002">[3], <span class="email" tagx="email" title="email">stefan_moser@web.de</span></span><span class="citation_author_institution" id="af4-metabolites-11-00002">[4], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">T.letzel@tum.de</span></div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">â€ </span><p>Current address: HydroSciences Montpellier, UMR 5569, FacultÃ© de Pharmacie, University of Montpellier, Avenue Charles Flahault 15, 34000 Montpellier, France.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-12-12</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021-1</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">1</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">2</span><span class="history" title="history"><span class="received" title="received">received: 2020-11-12</span><span class="accepted" title="accepted">accepted: 2020-12-21</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Plants produce a huge number of functionally and chemically different natural products that play an important role in linking the plant with the adjacent environment. Plants can also absorb and transform external organic compounds (xenobiotics). Currently there are only a few studies concerning the effects of xenobiotics and their transformation products on plant metabolites using a mass spectrometric untargeted screening strategy. This study was designed to investigate the changes of the <i>Phragmites australis</i> metabolome following/after diclofenac or carbamazepine incubation, using a serial coupling of reversed-phase liquid chromatography (RPLC) and hydrophilic interaction liquid chromatography (HILIC) combined with accurate high-resolution time-of-flight mass spectrometer (TOF-MS). An untargeted screening strategy of metabolic fingerprints was developed to purposefully compare samples from differently treated <i>P. australis</i> plants, revealing that <i>P. australis</i> responded to each drug differently. When solvents with significantly different polarities were used, the metabolic profiles of <i>P. australis</i> were found to change significantly. For instance, the production of polyphenols (such as quercetin) in the plant increased after diclofenac incubation. Moreover, the pathway of unsaturated organic acids became more prominent, eventually as a reaction to protect the cells against reactive oxygen species (ROS). Hence, <i>P. australis</i> exhibited an adaptive mechanism to cope with each drug. Consequently, the untargeted screening approach is essential for understanding the complex response of plants to xenobiotics.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd"><i>P. australis</i> metabolic profile</div> <div class="kwd" title="kwd">
untargeted metabolomics</div> <div class="kwd" title="kwd">
diclofenac</div> <div class="kwd" title="kwd">
carbamazepine</div> <div class="kwd" title="kwd">
orthogonal partial least square-discriminant analysis (OPLS-DA)</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Throughout human history, plants have been the most important source for pioneering medicines, flavors and industrial materials. Plants produce up to 200,000 natural products with a vast chemical diversity using a wide range of enzymes and substrates [<a href="#B1-metabolites-11-00002">1</a>]. Among them, primary metabolites are relatively few and are defined as fundamental to plant physiology. In contrast, secondary metabolites (specialized metabolites) are essentially diverse. The high diversity of specialized metabolites leads to functional variation. Specialized metabolites play important roles in the interactions between the plant and its environment since they are involved in protection against environmental stress, competition or pollinator attraction and some are involved in vegetative or floral development. Furthermore, they play a vital role in the conception of environmental signals and their translation in biography traits. Therefore, they may play major roles in allowing the organism to sustain environmental constraints.</p> <p>Among many other uses, plants have been used to restore environmental quality via adsorption and accumulation of organic xenobiotics (such as pesticides, dyes, drugs, etc.) inside their tissues. In the process of phytoremediation, plants can degrade these external compounds through a specific enzymatic scaffold consisting of three phases [<a href="#B2-metabolites-11-00002">2</a>,<a href="#B3-metabolites-11-00002">3</a>]. In this process, they utilize enzymes like cytochromes P450 or peroxidases to activate the xenobiotics in phase I and conjugating enzymes as glucosyl-transferases, malonyl-transferases or glutathione S-transferases in phase II for detoxification [<a href="#B4-metabolites-11-00002">4</a>,<a href="#B5-metabolites-11-00002">5</a>].</p> <p>Diclofenac (DCF) and carbamazepine (CBZ) belong to the problematic xenobiotics in European wastewater treatment plants since they can harm the ecotoxicological equilibrium. They have not only been detected in wastewater but also in biosolids and wastewater effluents [<a href="#B6-metabolites-11-00002">6</a>,<a href="#B7-metabolites-11-00002">7</a>,<a href="#B8-metabolites-11-00002">8</a>]. Recently, the maximal measured concentrations of DCF and CBZ in municipal wastewater has ranged between 440 and 7100â€‰ng Â· L<sup>âˆ’1</sup> and 1075â€"6300 ng Â· L<sup>âˆ’1</sup> of, respectively [<a href="#B9-metabolites-11-00002">9</a>,<a href="#B10-metabolites-11-00002">10</a>]. DCF is one of the most commonly used non-steroidal anti-inflammatory drugs, while CBZ has been used widely as an antiepileptic and mood stabilizer since the 1970s. Both DCF and CBZ are taken up and translocated into the aerial parts of plants, where they can be accumulated or metabolized into more or less toxic products [<a href="#B7-metabolites-11-00002">7</a>,<a href="#B11-metabolites-11-00002">11</a>].</p> <p>One of these species is <i>P. australis</i>, known as â€œcommon reedâ€�. <i>P. australis</i> belongs to the Poaceae family and is an invasive plant that spreads worldwide [<a href="#B12-metabolites-11-00002">12</a>]. It has been used for a long time in wetlands to remove pollutants, reduce nitrogen loads and provide oxygen to the rhizosphere [<a href="#B5-metabolites-11-00002">5</a>,<a href="#B13-metabolites-11-00002">13</a>]. Furthermore, it is used as a resource for traditional crafts and fodder. In some regions like Northern China, it is grown as a crop and its leaves are used in the treatment of bronchitis and cholera [<a href="#B12-metabolites-11-00002">12</a>].</p> <p>The chemical diversity (such as polarity, stability, reactivity or ionization) of plant molecules prohibits investigation of the metabolitesÂ´ full picture and understanding changes in their occurrence. The analysis purpose and chemical nature of plant metabolites determine the analytical technique. Also, the untargeted concept demands an analytical device, which can separate different classes of metabolites with a wide range of polarities. For untargeted metabolite analysis, several analytical techniques are available. These approaches apply liquid chromatography-mass spectrometry (LC-MS) [<a href="#B14-metabolites-11-00002">14</a>], gas chromatography-mass spectrometry (GC-MS) [<a href="#B15-metabolites-11-00002">15</a>] or nuclear magnetic resonance (NMR) [<a href="#B16-metabolites-11-00002">16</a>] to analyze a large number of different chemical metabolites classes within one single analysis. NMR has analytical reproducibility and a non-destructive nature; however, it has relatively low sensitivity compared to MS. Besides, a mass spectrometric untargeted screening strategy can identify a large number of molecules independently to provide a preferably holistic picture of the plantâ€™s metabolome. Also, the thermal stability of the stationary phase, metabolites and their derivatives, which might introduce artifacts, limit the metabolome coverage derived by GC-MS. Thus, the usage of LC-MS has expanded rapidly over the past ten years in untargeted metabolomics analysis [<a href="#B17-metabolites-11-00002">17</a>].</p> <p>The serial coupling of reversed-phase liquid chromatography (RPLC) and hydrophilic interaction liquid chromatography (HILIC) is often used to separate compounds with differing polarities in a single run [<a href="#B18-metabolites-11-00002">18</a>]. Further, connecting the serial RPLC-HILIC coupling with an accurate high-resolution time-of-flight mass spectrometer (HRMS) provides the detection of a wide range of metabolites.</p> <p>Recently, the awareness of untargeted metabolomics analysis has increased due to its capabilities in the assessment of xenobiotics exposure/specific biomarkers and the risk of contaminants to living organisms. For example, the metabolic fingerprints of <i>Plantago lanceolata</i> showed various chemical changes as a response to different stresses [<a href="#B19-metabolites-11-00002">19</a>].</p> <p>Metabolic fingerprinting experiments aim to determine relative differences between two or more systems elucidating a biological relationship. Therefore, statistical strategies are typically used in a chemometrics style. Univariate and multivariate statistics can be used as standard approaches to extract relevant information from complex datasets [<a href="#B20-metabolites-11-00002">20</a>,<a href="#B21-metabolites-11-00002">21</a>,<a href="#B22-metabolites-11-00002">22</a>,<a href="#B23-metabolites-11-00002">23</a>].</p> <p>This study aims to investigate the effect of DCF, CBZ and their transformation products on <i>P. australis</i> metabolites using mass spectrometric untargeted screening analysis. By studying the fingerprint of <i>P. australis</i> leaves, rhizomes and roots with/without incubation at two different concentrations, 10 and 100 ÂµM and 10 and 50 ÂµM of DCF, as well as CBZ, respectively. Also, a statistical workflow was used to discriminate against the changes in <i>P. australis</i> metabolites and detect the differentiating metabolic profiles.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
2. Results</div> <p>Diclofenac and carbamazepine are considered problematic environmental pollutants because they can be found in surface waters (mainly in wastewater treatment plant effluents) in high concentrations. Several plant species have been found capable of absorbing and detoxifying them. Despite this, the effects of diclofenac and carbamazepine and their transformation products on plant metabolic profiles need more investigation. Consequently, <i>P. australis</i> was incubated with 10 and 100 ÂµM DCF and with 10 and 50 ÂµM CBZ, respectively. Each plant (i.e., leaves, rhizomes and roots, respectively) was extracted with four different solvents. The different samples were injected into a robust and reproducible serial coupling RPLC-HILIC in hyphenation with electron spray ionization- time of flight- mass spectrometer (ESI-TOF-MS). Both robustness and reproducibility of the analytical system was checked and proven with a mixture of 13 reference standards (<a href="#app1-metabolites-11-00002">Tables S1 and S2</a>). The statistical evaluation discriminated between the different extraction solvents, plant parts and incubation. Moreover, the statistical analysis was enabled to assign the changes in <i>P. australis</i> metabolic profiles and the differentiating metabolic profiles (DMF) between the control and incubated samples. Furthermore, different DCF and CBZ transformation products could be identified or suspected. Thus, changes in <i>P. australis</i> biosynthetic pathways could be predicted.</p> <div class="2.1.metabolicprofilingelucidationinphragmitesaustralisextractswithrplc-hilic-esi-tof-ms" title="sec"> <div class="title" tagx="title" title="title">
2.1. Metabolic Profiling Elucidation in Phragmites australis Extracts with RPLC-HILIC-ESI-TOF-MS</div> <p>A mixture of 13 reference standards was injected at the beginning/end and fixed intervals during the experimental sets (i.e., after each extract batch). The results of the standard mixture proved the accuracy, repeatability and reproducibility of the analytical system. Mass and RT of the standards during the experiment showed an acceptable deviation of less than 8 ppm (with a ToF system from the year 2012) and % RSD less than 2 except for gabapentin, carbetamid and sotalol. For more information, readers are referred to Wahman et al., 2019 [<a href="#B18-metabolites-11-00002">18</a>].</p> <p>The four different extracts 100% MeOH, acidic 90% MeOH, 50% MeOH and 100% H<sub>2</sub>O of <i>P. australis</i> leaf, rhizome and root were analyzed similarly with RPLC-HILIC-ESI-TOF-MS coupling as previously described [<a href="#B24-metabolites-11-00002">24</a>,<a href="#B25-metabolites-11-00002">25</a>,<a href="#B26-metabolites-11-00002">26</a>]. The obtained mass spectrometric total ion chromatograms (TICs) were interpreted to extract the feature (extracted masses, RT and signal abundance) according to the parameters mentioned in <a href="#sec4dot6dot1-metabolites-11-00002">Section 4.6.1</a>. Background signals (i.e., all the peaks detected in the corresponding blank) were deleted to avoid false positives. The Retention Time (RT)/Mass plots of the different background were presented in (<a href="#app1-metabolites-11-00002">Figure S1</a>). Lastly, the features found in the triplicate injections were considered for further analysis. Exemplarily, <a href="#metabolites-11-00002-f001">Figure 1</a> represents (RT)/Mass plots for 100% MeOH extracts of <i>P. australis</i> leaf, rhizome and root in positive ion mode. The highly polar to polar compounds eluted at RT &amp;lt; 15 min, with logD values below zero (HILIC part). The nonpolar compounds were eluted at RT &amp;gt; 15 min, with logD values above zero (RPLC part). The differences in chromatographic fingerprints reflected the variability in metabolite profiles (and composition) in the leaf, rhizome and root samples. Detailed information and a description of data analysis have already been discussed in previous publications [<a href="#B18-metabolites-11-00002">18</a>,<a href="#B26-metabolites-11-00002">26</a>].</p> </div> <div class="2.2.differentextractsofphragmitesaustralisâ€™smetabolicfingerprints" title="sec"> <div class="title" tagx="title" title="title">
2.2. Different Extracts of Phragmites australisâ€™s Metabolic Fingerprints</div> <p>Untargeted metabolomics analysis demands minimal pretreatment methods to allow the detection of almost all the sample metabolites. Different leaf, rhizome and root samples were extracted with the described four solvents (i.e., 100% MeOH, acidic 90% MeOH, 50% MeOH and 100% H<sub>2</sub>O), individually, which ensured the extraction of a wide range of metabolites from nonpolar to highly polar metabolites relative to the solvents, which were used in the extraction process. All the extracts were analyzed along a metabolitesâ€™ fingerprint strategy. In orthogonal partial least square-discriminant analysis (OPLS-DA), the variables were <i>P. australis</i> metabolites, which were plotted in the loading score plot (as in <a href="#metabolites-11-00002-f002">Figure 2</a>a). The OPLS-DA model described the variables according to the solvents of the extraction class. Of the data variations, 18.3% (R<sup>2</sup>X(cum)) are responsible to distinguish between the classes that were previously established based on solvents. The rest of the variation (orthogonal components) describes the variation within the solvent classes. The high value of those parameters indicates that the OPLS-DA model had a good classification and prediction efficiency to distinguish between different extracts, even though it described one variation (i.e., solvent type). This is owing to the accuracy of variables, which were separated with a robust and reproducible LC-system. In (<a href="#metabolites-11-00002-f002">Figure 2</a>b), the samples are distributed according to t1 (predicative component) and t2 (orthogonal component). The predictive component (t1) separated the samples into two groups, the first group (negative side) contained the acidic 90% MeOH extracts and the second group (positive side) consisted of 100% MeOH, 50% MeOH and 100% H<sub>2</sub>O extracts (<a href="#metabolites-11-00002-f002">Figure 2</a>b). Moreover, the orthogonal component (t2) described the differences within the group. Consequently, it separated the second group into 100%MeOH and 50%MeOH in the positive part and the 100% H<sub>2</sub>O extracts in the negative part (<a href="#metabolites-11-00002-f002">Figure 2</a>b). Further, in (<a href="#metabolites-11-00002-f002">Figure 2</a>c), good separation was reached according to the cumulative goodness of fit and the cross-validation parameters of each variable R<sup>2</sup> and Q<sup>2</sup>, respectively. Therefore, the model had no risk of overfitting.</p> </div> <div class="2.3.phragmitesaustralisleaf,rhizomeandrootmetabolicfingerprints" title="sec"> <div class="title" tagx="title" title="title">
2.3. Phragmites australis Leaf, Rhizome and Root Metabolic Fingerprints</div> <p>In this section, the 11,442 variables were differentiated according to the plant part that they originated from. The variables were in the loading score plot (<a href="#metabolites-11-00002-f002">Figure 2</a>d). This step was performed to improve the statistical significance of the dataset through additional cross-validation. Also, the OPLS-DA analysis was conducted to investigate the structure of the data. Another 18% of the data was investigated through the variation between X (metabolites) and Y (plant parts) given by R<sup>2</sup>X(cum) and explained approximately 94% of the variations in the various samples (R<sup>2</sup>Y(cum)). We found the predictive value of the model was (Q<sup>2</sup>(cum) = 74%), which was created by OPLS-DA. The cross-validation performance was confirmed by analysis of variance (ANOVA). OPLS-DA discriminates the different samples of <i>P. australis</i> leaf, rhizome and root regardless of incubation, with or without DCF or CBZ and regardless of the extraction solvent composition according to the plant part (<a href="#metabolites-11-00002-f002">Figure 2</a>e).</p> <p>The predictive component t1 differentiated between leaf extracts in one group (negative part) and the root and rhizomes extracts in the second group (positive part), as shown in (<a href="#metabolites-11-00002-f002">Figure 2</a>e). However, the t2 (orthogonal component) differentiated rhizomes sample in the negative part and the roots in the positive one.</p> <p>The quality of the OPLS-DA module was expressed by the cumulative value of the goodness of fit and the cross-validation for each value R<sup>2</sup> and Q<sup>2</sup>, respectively, as shown in the Q<sup>2</sup>/R<sup>2</sup> Overview plot (<a href="#app1-metabolites-11-00002">Figure S2A</a>). The high values of the previous parameters indicated a good classification and prediction efficiency to distinguish between different plant parts.</p> </div> <div class="2.4.untargetedmetabolomicsanalysisofphragmitesaustralisincubatedwithdcforcbz" title="sec"> <div class="title" tagx="title" title="title">
2.4. Untargeted Metabolomics Analysis of Phragmites australis Incubated with DCF or CBZ</div> <p>The untargeted metabolomics analysis of <i>P. australis</i> incubated with/without DCF or CBZ, respectively, was performed using OPLS-DA to assign the changes in its metabolic fingerprint. The <i>P. australis</i> metabolic fingerprints of different extraction solvents with various plant parts were investigated in <a href="#sec2dot2-metabolites-11-00002">Section 2.2</a> and <a href="#sec2dot3-metabolites-11-00002">Section 2.3</a>, respectively, to test the organization and reliability of the data. Then, the large data set was used to perform the untargeted analysis and assign a list of metabolites that determined the distance between different groups. Also, the metabolic markers of <i>P. australis</i> incubated with/without DCF or CBZ were plotted each by the OPLS-DA, which represented the variability in metabolic patterns due to the different incubation (<a href="#metabolites-11-00002-f003">Figure 3</a>, that is, the loading plot in <a href="#metabolites-11-00002-f003">Figure 3</a>a and the OPLS-DA score plot in 3B). OPLS-DA analysis showed the identified and unidentified metabolites, which distinguished the different clusters according to the characteristic change of control or incubated samples metabolite profiles. It was used to enhance the quality of pairwise classification analysis. The relative high goodness of fit indicated the good separation of different incubation groups of <i>P. australis</i> R<sup>2</sup> and Q<sup>2</sup>, respectively, (<a href="#app1-metabolites-11-00002">Figure S2B</a>). To facilitate the interpretation and visualization of OPLS-DA, the S-plot was drawn to illustrate the modelâ€™s influence with accuracy in the search for differentiating metabolic profile (DMF). S-plot analysis represented the highest contributing signals for the control and incubation of <i>P. australis</i> with DCF or CBZ (<a href="#metabolites-11-00002-f004">Figure 4</a>a,b, respectively). The DMF was extracted from S-plots, which were marked in red color. They were selected based on their contribution to the variation and correlation within the data set between the X-variables and the predictive component t1 (p (corr) vector). Hence, they were considered functionally in the combined form of a metabolic profile, which was distinguished between the control and the incubated <i>P. australis</i>. In <a href="#metabolites-11-00002-f004">Figure 4</a>c,d, the contribution plots extend the data to better visualization and indicate regions responsible for sample clustering [<a href="#B22-metabolites-11-00002">22</a>]. The contribution plot summarized the changing trends in the metabolites in the pairwise groups (i.e., control and incubated samples) through expressing the fold change of the metabolites between the control and incubated samples. As examples, succinic acid (DM_4), propane-1,2,3-triol (DM_6) and 2- hydroxypropanoic acid (DM_7) were detected as transformation products of DCF (DMs, see <a href="#metabolites-11-00002-t001">Table 1</a>), thus they appeared significant in the positive part of the plot, (<a href="#metabolites-11-00002-f004">Figure 4</a>c). Also, quercetin (compound 1 in <a href="#metabolites-11-00002-f004">Figure 4</a>c,d) was found in elevated signal heights after <i>P. australis</i> exposure to DCF. Quercetin is a flavonoid that protects the plant against reactive oxygen species (ROS) during exposure to pharmaceuticals.</p> <p>In CBZ incubation, the contribution plot showed that in incubated samples 2,3-dihydro-2,3-dihydroxy-carbamazepine (compound <b>2</b> in <a href="#metabolites-11-00002-f004">Figure 4</a>d) and carbamazepine-10,11-epoxide (compound <b>3</b> in <a href="#metabolites-11-00002-f004">Figure 4</a>d) could be identified as metabolites (evaluated and proven with reference standards) and appeared with significant intensity. The carbamazepine-10,11-epoxide was statistically significant variable important (VIP) &amp;gt; 1 and <i>p</i> &amp;lt; 0.05. However, in this case quercetin (compound <b>1</b> in <a href="#metabolites-11-00002-f004">Figure 4</a>d) intensity was found to decrease significantly.</p> </div> <div class="2.5.metabolismofdiclofenacinphragmitesaustralis" title="sec"> <div class="title" tagx="title" title="title">
2.5. Metabolism of Diclofenac in Phragmites australis</div> <p><i>P. australis</i> was exposed for 96 h to diclofenac in the two concentrations of 10 and 100 ÂµM, respectively. Thereafter, the data was processed with MassHunter Workstation Software Profinder B.06.00 (Agilent Technologies, Waldbronn, Germany) to detect the DCF molecule and its metabolites (including background subtraction). Diclofenac and its hydroxylated metabolites were detected in the roots and rhizomes of <i>P. australis</i>. Diclofenac was identified at 295.0173 Da with 2 ppm deviation from the monoisotopic mass, which eluted at 26 min (logD (pH7) &amp;gt; 0). The EICs of diclofenac in extracts and standard are shown in <a href="#app1-metabolites-11-00002">Figure S3</a>. Further, hydroxylated metabolites were suspected according to the previously mentioned criteria at 311.0114 Da with 0.6 ppm deviation from the monoisotopic mass, which eluted at 24 min (logD (pH7) &amp;gt; 0) [<a href="#B27-metabolites-11-00002">27</a>]. Moreover, the analysis of different extracts of <i>P australis</i> samples based on the mass, RT and LogD (pH7) revealed seven proposed metabolites of diclofenac, as summarized in <a href="#metabolites-11-00002-t001">Table 1</a> and <a href="#app1-metabolites-11-00002">Figure S3</a>. The comparison of mass spectra from leaf, rhizome and root resulted in evidence for five metabolites of diclofenac in all parts of the treated samples of <i>P. australis</i>. However, DM_6 and DM_7 were detected in leaf extracts (<a href="#metabolites-11-00002-t001">Table 1</a>).</p> </div> <div class="2.6.metabolismofcarbamazepineinphragmitesaustralis" title="sec"> <div class="title" tagx="title" title="title">
2.6. Metabolism of Carbamazepine in Phragmites australis</div> <p><i>P. australis</i> was exposed for 96 h to carbamazepine in two CBZ concentrations of 10 and 50 ÂµM, respectively. Thereafter, the data was processed with MassHunter Workstation Software Profinder B.06.00 to detect the compound and its metabolites (including background subtraction). According to SauvÃªtre and co-workers (2018), four different pathways for carbamazepine metabolism (PCM) were investigated in the plant [<a href="#B28-metabolites-11-00002">28</a>].</p> <p>The different transformation products were identified in different <i>P. australis</i> samples using reference standards. The mean monoisotopic mass of standards and different samples and their absolute variation were tabulated, as well as the retention time in both standards and samples and the variation between them (<a href="#app1-metabolites-11-00002">Table S3</a>, <a href="#app1-metabolites-11-00002">Figure S4</a>). Carbamazepine-10,11-epoxide, 10,11-dihydro-10,11-dihydroxy-carbamazepine, 10,11-dihydro-10-hydroxy-carbamazepine, 9-acridine carboxaldehyde and 2,3-dihydro-2,3-dihydroxy-carbamazepine were identified in <i>P. australis</i> incubated samples. The mass deviation was less than 5 ppm and the deviation in the RT was less than 0.3 min. Carbamazepine-10,11-epoxide and 9-acridine carboxaldehyde have been identified in leaf, rhizome and root in both incubation levels 10 and 50 ÂµM. 10,11-dihydro-10,11-dihydroxy-carbamazepine and 2,3-dihydro-2,3-dihydroxy-carbamazepine were identified in root and rhizome extracts in both incubation concentrations. Moreover, the first was identified in leaf extracts of 50 ÂµM carbamazepine; however, the latter was not detectable in leaf extracts, respectively.</p> </div> <div class="2.7.impactsofdcfandcbzonphragmitesaustralismetabolicpathways" title="sec"> <div class="title" tagx="title" title="title">
2.7. Impacts of DCF and CBZ on Phragmites australis Metabolic Pathways</div> <p>Metabolic pathway analysis was performed to identify the pathways that were induced upon the incubation of <i>P. australis</i> with DCF or CBZ via MetaboAnalyst 4.0 software (Montreal, QC, Canada) based on the Kyoto Encyclopedia of Genes and Genomes (KEGG) database. The results are established on OPLS-DA and S-plot analysis of different metabolites with/without DCF or CBZ incubation depending on the p (corr) vector in the OPLS-DA module. Consequently, these metabolites represented the metabolic profile (DMF) which differentiate between control and incubated samples.</p> <p>For the investigation, the mummichog algorithm was the first implementation of this concept to infer pathway activities from a ranked list of MS peaks identified by untargeted metabolomics. The DMF was applied to test their participation in pathways using a combination of network analysis and functional enrichment analysis and listed in <a href="#app1-metabolites-11-00002">Tables S4 and S5</a>. The pathways exhibiting <i>p</i> &amp;lt; 0.05 were considered a statistically significant metabolic pathway, meaning that they were affected with DCF or CBZ or their transformation products via MetaboAnalyst 4.0 software based on the Kyoto Encyclopedia of Genes and Genomes (KEGG) database.</p> <p>KEGG pathway analysis of the analyzed metabolites using the pathway data set of <i>A. thaliana</i> matched 27 pathways in <i>P. australis</i> incubated with DCF. Further, 11 were significantly biologically active and are listed in the table in (<a href="#metabolites-11-00002-f005">Figure 5</a>a). They were identified as glycolysis/gluconeogenesis, ascorbate and aldarate metabolism, fructose and mannose metabolism, galactose metabolism, the pentose phosphate pathway, arginine biosynthesis, alanine, aspartate and glutamate metabolism, purine metabolism, pyrimidine metabolism, glutathione metabolism and phenylalanine metabolism.</p> <p>After CBZ incubation, 22 pathways were significantly altered. Twelve pathways have a more significant <i>p</i> value, relating them to the effect of CBZ and/or its transformation products (<a href="#metabolites-11-00002-f005">Figure 5</a>b). They were the pentose phosphate pathway, purine metabolism, pyrimidine metabolism, fatty acid biosynthesis, arachidonic acid metabolism, tyrosine metabolism, tryptophan metabolism, Î²-Alanine metabolism, arginine and proline metabolism, pantothenate and CoA biosynthesis, carbon fixation in photosynthetic organisms and folate biosynthesis. The previous pathways were induced after incubation with DCF or CBZ.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
3. Discussion</div> <p><i>P. australis</i> metabolic profiles were investigated in different samples with RPLC-HILIC-ESI-TOF-MS. The system accuracy and reproducibility were checked using a mixture of 13 reference standards. The results of the quality control mixture revealed that the serial coupling produces robust and reproducible data, which was used for further statistical analysis. <i>P.australis</i> metabolomicsâ€™ statistical analysis of the four different extracts reveals that the 90% acidic MeOH extracts are very different from the other extracts of various <i>P. australis</i> leaf, rhizome and root samples. However, the 100% MeOH, 50% MeOH and 100% H<sub>2</sub>O extracts are more related to each other. 100% MeOH was generally located in the middle between 50% MeOH and 100% H<sub>2</sub>O extracts, however, representing a similar solution behavior to all three extract types. A large t2 range value was observed mostly in acidic 90% MeOH extracts, that is, a value far above the critical limits in the score space. Hence, this is likely to be an outlying observation. The elimination of acidic 90% MeOH extracts might improve the model. Also, most of the metabolites extracted with acidic 90% MeOH have been detected in three other solvents. The same observation was made for <i>Lemna minor</i> extracts and can be described with the different occurrence of charged molecules caused by different pH values [<a href="#B26-metabolites-11-00002">26</a>].</p> <p>The statistical software allowed cross-validation between the datasets. Furthermore, an additional step of cross-validation of the data was done through the differentiation between the x (metabolites) and the plant parts. The results reveal the significance of the data organization. Hence, the compounds extracted from roots and rhizomes are more correlated than compounds in leaves, which supports the physiological similarities between them [<a href="#B29-metabolites-11-00002">29</a>]. After the additional cross-validation step, the OPLS-DA model was built to assign the variables, which differentiate between control and incubated samples. OPLS-DA is a supervised approach that tends to improve the separation between (two or more) groups of samples. For this reason, it is widely used for classification purposes and biomarker identification and/or differentiating metabolic profiles (DMF) in metabolomics studies [<a href="#B21-metabolites-11-00002">21</a>].</p> <p>The DMF of <i>P. australis</i> was extracted. Further, the fold change of the different variables in the control and incubated samples show an increase in quercetin in DCF incubation; however, in this case of CBZ, it was found to decrease significantly. <i>P. australis</i> responds to the incubation with different classes of compounds, with the potential to protect it against ROS, such as polyphenols and flavonoids. Also, it might be that CBZ does not cause oxidative stress to the same extent as DCF. Furthermore, succinic acid (DM_4), propane-1,2,3-triol (DM_6) and 2-hydroxypropanoic acid (DM_7) appeared significant in the positive part of the plot with VIP &amp;gt; 1 and <i>p</i> &amp;lt; 0.05 (<a href="#metabolites-11-00002-f004">Figure 4</a>c). They were shifted toward the middle of the figure more than CBZ transformation products, indicating that they originated partially from diclofenac. The same was in CBZ transformation products, which were not produced by <i>P. australis.</i> The carbamazepine-10,11-epoxide was statistically significant: VIP &amp;gt; 1 and <i>p</i> &amp;lt; 0.05. Carbamazepine-10,11-epoxide is considered a main transformed product of carbamazepine in plants (compound <b>3</b> in <a href="#metabolites-11-00002-f004">Figure 4</a>d). It is also the first metabolite of carbamazepine in tomato plants and <i>Armoracia rusticana</i> root cultures [<a href="#B28-metabolites-11-00002">28</a>,<a href="#B30-metabolites-11-00002">30</a>]. Hence, the increase of succinic acid (DM_4) might affect the TCA cycle (tricarboxylic acid cycle) of <i>P. australis.</i></p> <p>Also, diclofenac and its hydroxylated metabolites were identified in <i>P. australis</i> extracts. It has been reported that formation of hydroxylated metabolites is the first step in the detoxification of diclofenac in the plant. These results reveal that P450 monooxygenases or peroxidases were involved to detoxify diclofenac [<a href="#B11-metabolites-11-00002">11</a>].</p> <p>In CBZ incubation, different transformation products were identified, which originated from different metabolism pathways (PCM). The 10,11-diOH pathway (PCM) has been investigated comprehensively in plants like cucumber, tomato, sweet potato, lettuce, carrot and horseradish [<a href="#B28-metabolites-11-00002">28</a>,<a href="#B31-metabolites-11-00002">31</a>]. The main transformed product is carbamazepine-10,11-epoxide (MartÃ­nez-Piernas et al., 2019), which has been identified in leaf, rhizome and root in both incubations levels 10 and 50 ÂµM, as shown in <a href="#app1-metabolites-11-00002">Table S3</a> [<a href="#B32-metabolites-11-00002">32</a>]. This is the first oxidation step, which was observed in different organisms from bacteria and fungi to mammals. It was conducted by cytochrome P450 and/or peroxidases, leading to different sub-pathways (PCM).</p> <p>The first sub-pathway (PCM) started with cleavage of the epoxide bond and the hydroxylation to form 10,11-dihydro-10,11-dihydroxy-carbamazepine. This step is catalyzed by epoxide hydrolase enzymes [<a href="#B33-metabolites-11-00002">33</a>]. A recent study reported further metabolism of 10,11-dihydro-10,11-dihydroxy-carbamazepine through hydroxylation of the benzene ring [<a href="#B32-metabolites-11-00002">32</a>]. 10,11-dihydro-10,11-dihydroxy-carbamazepine was identified in root and rhizome extracts in both incubation concentrations, as shown in <a href="#app1-metabolites-11-00002">Table S3</a>. However, 10,11-dihydro-10-hydroxy-carbamazepine was identified in leaf extracts in the 50 ÂµM carbamazepine. It seems that the 10,11-dihydro-10,11-dihydroxy-carbamazepine was transferred to rhizome and leaf for further metabolism. This explains why it reaches the leaves in high concentration from the highly loaded rhizome tissue. Thus, <i>P. australis</i> can degrade carbamazepine through a 10,11-diOH pathway (PCM), initiated in the root and completed in leaf and rhizome.</p> <p>Further, <i>P. australis</i> metabolized the carbamazepine-10,11-epoxide into 9-acridine carboxaldehyde, which is analogous to other studies [<a href="#B28-metabolites-11-00002">28</a>,<a href="#B31-metabolites-11-00002">31</a>,<a href="#B32-metabolites-11-00002">32</a>]. The 9-acridine carboxaldehyde is a reactive compound that was identified in leaf, rhizome and root extracts of plants exposed to 10 and 50 ÂµM carbamazepine. Further metabolites downstream from 9-acridine carboxaldehyde could not be detected. 9-acridine carboxaldehyde is typically transformed into acridone, which is a non-toxic compound with the formation of the two intermediates acridine and 9-hydroxy acridine [<a href="#B28-metabolites-11-00002">28</a>]. In our conditions, they were below the detectable limit or did not arise. Because 9-acridine carboxaldehyde was identified in leaf, rhizome and root extracts in both concentrations, it may be assumed that there are further steps in the acridine metabolic pathway (PCM) that could be detected, like in lettuce, with longer incubation periods [<a href="#B32-metabolites-11-00002">32</a>].</p> <p>The last sub-pathway (PCM) is similar to the 10, 11-diOH pathway in the first steps. It includes a consecutive oxidation reaction on the aromatic group. Carbamazepine was metabolized to 2,3-dihydro-2,3-dihydroxy-carbamazepine which comprises two steps with the participation of a cytochrome P450 or a peroxidase and an epoxide hydrolase [<a href="#B33-metabolites-11-00002">33</a>]. 2,3-dihydro-2,3-dihydroxy-carbamazepine was identified in the <i>P. australis</i> rhizome and root in both incubation levels. However, 2,3-dihydro-2,3-dihydroxy-carbamazepine was not detectable in leaf extracts. It is moreover possible that the leaf metabolizes CBZ to 4-hydroxy-carbamazepine as in the leaves of the tomato plant. It was identified formerly in lettuce after 4 days of exposure; however, the researchers spiked the lettuce with 1 mg L<sup>âˆ’1</sup> carbamazepine [<a href="#B30-metabolites-11-00002">30</a>,<a href="#B32-metabolites-11-00002">32</a>].</p> <p>Thus, it is possible that <i>P. australis</i> takes up and metabolizes carbamazepine in different mechanisms. Still, all the metabolites resemble phase I metabolism in plant metabolism and not the glycosylated transformation products (phase II) [<a href="#B3-metabolites-11-00002">3</a>].</p> <p>The mummichog algorithm of the DMF between control and incubated samples shows an induction in the eleven and twelve pathways due to incubation with DCF or CBZ, respectively. They were identified as glycolysis/gluconeogenesis, ascorbate and aldarate metabolism, fructose and mannose metabolism, galactose metabolism, the pentose phosphate pathway, arginine biosynthesis, alanine, aspartate and glutamate metabolism, purine metabolism, pyrimidine metabolism, glutathione metabolism and phenylalanine metabolism in DCF incubation.</p> <p>After CBZ incubation, the pentose phosphate pathway, purine metabolism, pyrimidine metabolism, fatty acid biosynthesis, arachidonic acid metabolism, tyrosine metabolism, tryptophan metabolism, Î²-Alanine metabolism, arginine and proline metabolism, pantothenate and CoA biosynthesis, carbon fixation in photosynthetic organisms and folate biosynthesis pathways were induced after the incubation of CBZ.</p> <p>The glucosinolate biosynthesis pathway provides the plant with defense compounds against biting insects and worms. Furthermore, the cytochrome P450 enzyme controls glucosinolate biosynthesis, which is involved in the metabolism of DCF and CBZ during phase I [<a href="#B11-metabolites-11-00002">11</a>,<a href="#B34-metabolites-11-00002">34</a>]. The decline in sugars after plant exposure to DCF and CBZ indicates an increase in energy consumption [<a href="#B35-metabolites-11-00002">35</a>].</p> <p>Pyrimidine metabolism could be proposed to be a result of glucosinolate biosynthesis pathway alteration, which up-regulates the biosynthesis of other stress-induced pathways [<a href="#B36-metabolites-11-00002">36</a>]. Pyrimidine metabolism is enhanced and leads to products that could be used in the case of salvage, that is, recovery of infections and subsequent synthesis of secondary products with specific functions in defense mechanisms. Additionally, pyrimidine metabolism provides a source of Î²-alanine or Î²-aminobutyrate, which might be an important source for the pantothenate of coenzyme A [<a href="#B37-metabolites-11-00002">37</a>].</p> <p>Furthermore, fatty acid biosynthesis and the related pyruvate metabolism were induced after incubation of <i>P. australis</i> with DCF and CBZ. After incubation with DCF, <i>P. australis</i> exhibits some responses differently from incubation with CBZ, since it showed an increase in phenylalanine metabolism which aligned with the increase of quercetin content. The flavonoid 3â€²-monooxygenase and the flavonoid 3â€²,5â€²-hydroxylase enzymes responsible for the conversion of kaempferol to quercetin are cytochrome P450 plant types, which are triggered after incubation with DCF. However, the quercetin levels were lower after CBZ incubation (<a href="#metabolites-11-00002-f004">Figure 4</a>d).</p> <p>Glutathione metabolism was affected in DCF incubation but not under the influence of CBZ. Glutathione metabolism is a central part of the antioxidative ascorbateâ€"glutathione cycle. Further, glutathione is critical for the detoxification of xenobiotics, environmental stress tolerance and, in the form of phytochelatins, also the retention of heavy metals [<a href="#B38-metabolites-11-00002">38</a>]. The metabolism of DCF required Nicotinamide-Adenine Dinucleotide Phosphate (NADPH) as reductants [<a href="#B11-metabolites-11-00002">11</a>]. Upon DCF incubation, succinic acid formed from the degradation of diclofenac might have fueled the TCA cycle, which has been detected significantly in extracts incubated with DCF (<a href="#metabolites-11-00002-f004">Figure 4</a>c).</p> <p>In 2019, Sivaram and coworkers reported a high impact on the TCA cycle in maize leaves exposed to pyrene [<a href="#B39-metabolites-11-00002">39</a>]. Consequently, the Î³-Aminobutyric acid (GABA) shunt bypasses two steps of the TCA cycle. Moreover, it has an important pathway under stress conditions and it is associated with numerous physiological responses, including the regulation of cytosolic pH, carbon fluxes into the TCA cycle, nitrogen metabolism, osmoregulation and plant-pathogen interaction. Increased GABA levels also occur in response to changing environmental conditions and represent another adaptive mechanism in the attempt to maintain the rate of respiration under certain harmful conditions (AraÃºjo et al., 2012) [<a href="#B40-metabolites-11-00002">40</a>].</p> <p>The same result was observed in lettuce crops, which were exposed to different contaminants of emerging concern (CEC) concentrations [<a href="#B35-metabolites-11-00002">35</a>]. Also, it was reported that aromatic hydrocarbons altered the osmotic balance in maize [<a href="#B39-metabolites-11-00002">39</a>]. The alteration of previously mentioned biosynthetic pathways enhances the <i>P. australis</i> defense mechanisms. They also seem to be involved in the transformation of DCF and CBZ.</p> <p>Finally, <i>P. australis</i> used the glutathione metabolism pathway to defend itself against the DCF and seemingly, used the unsaturated fatty acid pathway to protect itself during the incubation with CBZ. Hence, <i>P. australis</i> responded differently to the DCF and CBZ through changing its metabolic pathway regardless of the type of drug to some extent. Consequently, specific changes in several common metabolic pathways can be considered as a marker for pollutant exposure in <i>P. australis</i>. However, each drug has fingerprints of the alteration of distinct metabolic pathways, which might be connected to its metabolites and the enzymes involved in metabolism. Therefore, the induced or changed pathways could be used as indicators for the exposure of the plant to DCF or CBZ.</p> </div> <div class="4.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
4. Materials and Methods</div> <div class="4.1.reagentsandchemicals" title="sec"> <div class="title" tagx="title" title="title">
4.1. Reagents and Chemicals</div> <p>LC-MS grade methanol and water were obtained from VWR, Darmstadt, Germany. Quercetin dehydrate (â‰¥95%, high performance liquid chromatography (HPLC)) was purchased from Enzo Life Sciences GmbH, LÃ¶rrach, Germany. Diclofenac (&amp;gt;99%) was obtained from Cayman Chemical Company, Ann Arbor, Michigan, MI, USA. Glyphosate (100 quality level, HPLC), gabapentin (200 quality level, HPLC), monuron (100 quality level, HPLC), chloridazon (100 quality level, HPLC), carbetamide (100 quality level, HPLC), metobromuron (100 quality level, HPLC), sotalol (â‰¥98%), quinoxyfen (100 quality level, HPLC), metconazol (100 quality level, HPLC) and fenofibrate (â‰¥99%) were obtained from Sigma, Darmstadt, Germany. Metformin (300 quality level, HPLC) was obtained from Fluka, Buchs, Switzerland. Furthermore, chlorbromuron (99.24%) and diazinon (99.53%) were obtained from Dr. Ehrenstorfer, Augsburg, Germany. Carbamazepine, 2,3-dihydro-2,3-dihydroxycarbamazepine, 10,11-dihydro-10,11-dihydroxy-carbamazepine, 10,11-dihydro-10-hydroxy-carbamazepine, 9-acridine carboxaldehyde and carbamazepine-10,11-epoxide were kindly provided by the German Research Center for Environmental Health, Comparative Microbiome Analysis (COMI), Helmholtz Centrum of Munich, Munich, Germany.</p> </div> <div class="4.2.plantsamples" title="sec"> <div class="title" tagx="title" title="title">
4.2. Plant Samples</div> <p>Twelve <i>P. australis</i> plants were grown in semi-hydroponic conditions in the greenhouse as described by SauvÃªtre and SchrÃ¶der earlier, in 2015 [<a href="#B7-metabolites-11-00002">7</a>]. Plants were grown in Hoagland solution made of (in mg/L) 472.30 Ca(NO<sub>3</sub>)<sub>2</sub>Â·4H<sub>2</sub>O, 202.22 KNO<sub>3</sub>, 492.96 MgSO<sub>4</sub>Â·7H<sub>2</sub>O, 68.04 KH<sub>2</sub>PO<sub>4</sub>, 80.04 NH<sub>4</sub>NO<sub>3</sub>, H<sub>3</sub>BO<sub>3</sub>, 1.8 MnCl<sub>2</sub>Â·4H<sub>2</sub>O, 0.2 ZnSO<sub>4</sub>Â·7H<sub>2</sub>O, 0.1 CuSO<sub>4</sub>Â·5H<sub>2</sub>O, 0.025 NaMoO<sub>4</sub> and 3.67 FeNa-Ethylenediaminetetraacetic acid. Plants (approximately 0.8 m in height) of uniform size were selected and placed into individual pots containing 2 L of spiked Hoagland solution Each pot contained one plant and was arranged in the greenhouse following a completely randomized design. The nutrient medium was spiked with a stock solution to reach the desired final concentration (diclofenac, 10 and 100 ÂµM and carbamazepine, 10 and 50 ÂµM, respectively). Control plants growing in Hoagland solution (spiked with the same amount of solvent as incubated plants) were used to obtain a reference plant matrix. Two pots were set up for each of the four exposure concentrations. Each assay consisted of duplicates arranged in the greenhouse following a randomized design. To compensate for water losses by evapo-transpiration, distilled water was added daily to the pots for a final volume of 2 L. Plants were exposed for 4 days before they were harvested. Harvested material was divided into roots, rhizomes and leaves, frozen in liquid nitrogen and stored at âˆ’81 Â°C until further processing. <i>P. australis</i> was kindly provided by the German Research Center for Environmental Health, Comparative Microbiome Analysis (COMI), Helmholtz Center Munich, Munich, Germany.</p> </div> <div class="4.3.extraction" title="sec"> <div class="title" tagx="title" title="title">
4.3. Extraction</div> <p>Twelve samples of <i>P. australis</i> leaves, rhizomes and roots, respectively, were collected, frozen under liquid N2. Then, the samples were freeze-dried and milled (Retsch S1 planetary ball mill, Retsch GmbH, Haan, Germany). Duplicates of each plant part were extracted with (a) 100% methanol (MeOH), (b) 90% MeOH (MeOH-water-formic acid (FAC) (90:9.5:0.5, <i>v</i>/<i>v</i>/<i>v</i>), (c) MeOH-water (50:50, <i>v</i>/<i>v</i>) and (d) 100% water (H<sub>2</sub>O), respectively. The solvents containing 500 mg plant powder were sonicated (Sonorex super RK 106, Bandelin, Germany) for 10 min at 4 Â°C with 35 kHz frequency. Then, samples were centrifuged (Z 200 A Universal Compact Centrifuge, Hermle LaborTechnik GmbH, Wehingen, Germany) at 1500 rpm/261.6Ã— <i>g</i> for 20 min and the supernatants were transferred to clean glass test tubes. The extraction process was triplicated in identical experimental conditions. Finally, the extracts were evaporated to dryness (using a SpeedVac, Fischer Scientific, GÃ¶teborg, Sweden) and dissolved in (50:50 (<i>v</i>/<i>v</i> %)) MeOH: H<sub>2</sub>O [<a href="#B18-metabolites-11-00002">18</a>,<a href="#B26-metabolites-11-00002">26</a>].</p> </div> <div class="4.4.instruments" title="sec"> <div class="title" tagx="title" title="title">
4.4. Instruments</div> <p>Filtered samples and standards (with 22 Âµm filter, Analytics Shops, Munich, Germany) were separated by LC (Agilent 1260 Infinity) consisting of an autosampler, two columns, two binary pumps, an online degasser, a mixing chamber and a UV detector. The LC-system was used to perform reversed-phase and zwitterion hydrophilic interaction liquid chromatography (HILIC) in its serial coupling. The reversed-phase separation column was a Poroshell 120 EC-C18 (50.0 Ã— 3.0 mm, 2.7 Î¼m, Agilent Technologies Waldbronn, Germany). The HILIC column was a ZIC-HILIC column (150 Ã— 2.1 mm, 5 Î¼m, 200 Ã…, Merck Sequant, Umea, Sweden). Columns were coupled through a T-piece (Upchurch, IDEX Europe GmbH, Erlangen, Germany). The third port of the T-piece was connected to the HILIC flow pump. The injection volume was 10 Î¼L. Further details, like the mobile phase of the RPLC-HILIC serial coupling and other settings, are described in References [<a href="#B24-metabolites-11-00002">24</a>,<a href="#B25-metabolites-11-00002">25</a>,<a href="#B26-metabolites-11-00002">26</a>].</p> <p>Samples and reference mixes were analyzed with a HRMS â€œtime-of-flightâ€� (TOF) mass spectrometer equipped with a Jet Stream ESI interface (Agilent Technologies, Waldbronn, Germany) and hyphenated with RPLC-HILIC chromatography. The parameters were as follows: 325 Â°C gas temperature, 10 L/min drying gas flow, 325 Â°C sheath gas temperature, 7.5 L/min sheath gas flow, 45-psi nebulizer operating pressure and 100 V fragmentor voltage. Ions were detected in positive ionization mode with a mass range of 50â€"2100 Daltons. The resolution of the instrument was better than 10,000 at <i>m/z</i> 922.</p> </div> <div class="4.5.qualitycontroloftherplc-hilic-esi-tof-mssystem" title="sec"> <div class="title" tagx="title" title="title">
4.5. Quality Control of the RPLC-HILIC-ESI-TOF-MS System</div> <p>The robustness and reproducibility of the RPLC-HILIC-ESI-TOF-MS system were tested with a standard mixture containing 13 different reference standards with a 20 ÂµM final concentration. The mixture consisted of metformin, glyphosate, gabapentin, monuron, chloridazon, carbetamide, metobromuron, sotalol, chlorbromuron, diazinon, quinoxyfen, metconazol and fenofibrate. The mixture was injected at the beginning/end of the experiment series and at fixed intervals during the experiment. It was injected after each extraction batch.</p> <p>The absolute variation between the literature monoisotopic mass and the mean of measured isotopic masses (Î"ppm) was computed according to the following equation: </p><div class="disp-formula" title="disp-formula">
Î"ppm = (monoisotopic mass of standard âˆ’mean of standard masses)/monoisotopic mass of standard Ã— 10<sup>6</sup><span class="label" tagx="label" title="label">(1)</span></div> <p /> <p>Moreover, the standard deviations (SD) of RT and Relative standard deviations (RSD) were calculated: </p><div class="disp-formula" title="disp-formula">
% of RSD = SD of compound RTs in different injection/Mean of compound RTs<span class="label" tagx="label" title="label">(2)</span></div> <p /> <p>The results are summarized in (<a href="#app1-metabolites-11-00002">Tables S1 and S2</a>). The absolute mass deviation ranged from 0.2 (Da) to 7 (Da). The RT standard deviation was less than 1%. Moreover, the relative standard deviation (%RSD) ranged from 0.5% to 3.6%. Thus, the results indicate the accuracy, repeatability and reproducibility of the LC system as previously reported in the investigation of plant metabolites in <i>Lemna minor</i> samples [<a href="#B18-metabolites-11-00002">18</a>].</p> </div> <div class="4.6.dataevaluation" title="sec"> <div class="title" tagx="title" title="title">
4.6. Data Evaluation</div> <div class="4.6.1.spectrometricdataevaluation" title="sec"> <div class="title" tagx="title" title="title">
4.6.1. Spectrometric Data Evaluation</div> <p>Data acquired with MassHunter Workstation LC/MS Data Acquisition software B 05.00, (Agilent Technologies, Waldbronn, Germany) was subsequently analyzed with Profinder B.06.00 (Agilent Technologies, Waldbronn, Germany) to extract the so-called â€œfeaturesâ€� by their retention times (RT), molecular mass and their peak intensity in various <i>P. australis</i> extracts. This was performed in a combination of the 3-fold injections of each sample after removing the features found in the corresponding blank samples. The parameters are set to a peak filter of 1000 counts peak height, ion species to â€œpositive ionsâ€� with H<sup>+</sup>, Na<sup>+</sup>, K<sup>+</sup> and NH4<sup>+</sup>, â€œcharge stateâ€� to 1, the â€œexpected RTâ€� to Â±3.00 min and the mass to Â±10 ppm. The extracted ion chromatograms (EICs) were smoothed with a Gaussian function using 9 points function and 5000 points Gaussian width. This limits the result finally to 2000 compound groups.</p> </div> <div class="4.6.2.dcfandcbztransformationproductsdetection" title="sec"> <div class="title" tagx="title" title="title">
4.6.2. DCF and CBZ Transformation Products Detection</div> <p>DCF and CBZ transformation product standards were analyzed using Agilent Profinder B.06.00 (Agilent Technologies). They were detected in the <i>P. australis</i> extracts, with masses estimated at Â±10 ppm and RT Â±0.3 min of the exact mass and RT of the standards, respectively. The metabolites were identified and suspected (when analytical standards were not available) after RPLC-HILIC-ESI-TOF-MS separation in the suspect analysis. (Suspects screening typically is performed with accurate and high-resolution mass spectrometers to observe the empirical formula of each molecule present and/or with tandem-mass spectrometry to observe specific fragment spectra). A local database was built using MassHunter PCDL Manager B.04.00 (Agilent Technologies, Waldbronn, Germany). Further, the logD (pH7) was the third parameter used to certify the identity of metabolites. The highly polar to polar compounds eluted from the HILIC column at RT &amp;lt; 15 min, with logD values below zero. The nonpolar compounds were eluted from the RP column at RT &amp;gt; 15 min, with logD values above zero. Metabolites within the criteria of mass, RT and logD (pH7) in the suspect analysis were considered.</p> </div> <div class="4.6.3.statisticaldataanalysis" title="sec"> <div class="title" tagx="title" title="title">
4.6.3. Statistical Data Analysis</div> <p>Data statistical analyses were conducted with SIMCA 16 software (MalmÃ¶, Sweden). Further analysis and data evaluation were performed with Microsoft Access and Excel 2016 (Redmond, Washington, WA, USA) and OriginPro 2019, Origin Lab cooperation, Northampton, MA, USA.</p> <p>The preprocessed data is a matrix. The rows are the exact masses, retention times (RTs) and abundances of each sample, which were listed in a Microsoft Excel Sheet. For statistical analysis, it is common to handle the data matrices with rows as observations and columns as compounds [<a href="#B36-metabolites-11-00002">36</a>]. Therefore, the data was organized in the Microsoft Access Database file (DBF), which was exclusively built to be suitable for SIMCA 16. The main advantage of the data matrix is the inherent support to align quantitative data (plant part, plant number, extraction solvent and drug incubation) along with related metadata (i.e., feature annotations/abundance as columns and sample annotations as rows). In DBF, the RTs, masses and abundances were connected to the corresponding plant part (i.e., leaf, rhizome and root), plant number (i.e., plants 1 and 2), extraction solvent and drug incubation. Once the matrix was built, comprehensive statistical analyses could be performed by using the vast range of functions provided by the software. The matrix consisted of 432 observations (i.e., the incubation with/without DCF or CBZ) and 11,442 variables (features). Furthermore, the plant part, plant number and extraction solvent were used as secondary observations. The data was not transformed and centered; however, it was scaled. The data was analyzed according to the following two strategies, considering the statistical analyses in untargeted metabolomics. Further information about the data setup can be found in the <a href="#app1-metabolites-11-00002">supplementary material</a>. By default, SIMCA provides an algorithm called â€œcross-validationâ€� to get the most valid model by calculating the adequate number of principal components to prevent overfitting of the data in the model. Furthermore, the software provides a large number of visual diagrams (score plots (with Hotellingâ€™s ellipse), DModX (Distant to Model) and statistic tables to assess the quality of the model in addition to the R<sup>2</sup> and Q<sup>2</sup>.</p> <ol> <li> <p>Metabolite fingerprinting was used to capture metabolite patterns across metabolite profiles. They are characterized without further identification steps (i.e., without need for standard reference material). Partial Least Squares (PLS) and Orthogonal Partial Least Squares regression-Discriminant Analysis (OPLS-DA) were used to relate sets of X-variables (such as plant part, plant number, extraction solvent and drug incubation) to the metabolites matrix. SIMCA 16 has a tool called Multiblock Orthogonal Component Analysis (MOCA). MOCAâ€™s concept is used to accomplish a fast and accurate analysis of multiple blocks of data (variables) registered for the same set of observations. MOCA aims at extracting the information in complex multi-block data analytics. Furthermore, it will extract two sets of components: the joint and the unique components. The quality of the models is described by R<sup>2</sup> and Q<sup>2</sup> values, where R<sup>2</sup> is the proportion of variance in the data explained by the models and indicates the goodness of fit and Q<sup>2</sup> is the proportion of variance in the data predictable by the model and expresses predictability [<a href="#B41-metabolites-11-00002">41</a>].</p> </li> <li> <p>Metabolite profiling which uses sets of predefined metabolites were studied in different samples of <i>P. australis</i> and differences in metabolites were usually related to the incubation with DCF or CBZ. Metabolite/variable selection was conducted to observe only the most significant metabolite candidates that explain the differences between the samples using S- and contribution-plots. The statistical models were built with confidence limits at 95%. Also, the differentiating metabolic profile (DMF) was chosen based on their contribution to the variation and correlation within the data sets. The related metabolic pathways were analyzed using MetaboAnalyst 4.0. Moreover, their contributions and biological clarifications were described based on the Kyoto Encyclopedia of Genes and Genomes (KEGG) database. The KEGG pathway analysis tool was used by the Arabidopsis thaliana database. The pathway analysis module combines the enrichment analysis and topology analysis based on KEGG. Fisherâ€™s test was used to generate <i>p</i> values. The <i>p</i> value was equal to 0.05, which indicates the fundamental connection of the identified metabolite with their respective metabolite and not due to the random chance [<a href="#B42-metabolites-11-00002">42</a>,<a href="#B43-metabolites-11-00002">43</a>].</p> </li> </ol> </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
5. Conclusions</div> <p>Metabolites of <i>P. australis</i> influenced by pharmaceuticals were investigated using RPLC-HILIC-ESI-TOF-MS. The experimental data and the statistical analysis revealed a change in the metabolitesâ€™ fingerprint between the different extracts, different plant parts and upon incubation with DCF and CBZ. The PLS and OPLS-DA identified the statistically significant clusters between the different groups. Further, significant DMF was determined in <i>P. australis</i> after the incubation with DCF or CBZ, individually. Different metabolic pathways were predicted from the statically identified DMF. These pathways were related mainly to the defense of the plant against stressful environmental conditions. <i>P. australis</i> adapted to each drug differently. <i>P. australis</i> could putatively use the glutathione metabolism pathway and unsaturated fatty acid pathway to protect itself during the incubation with DCF and CBZ, respectively. The results reveal insights into the metabolic profile of the speciesÂ´ adaptation to different pollutants. This study may set a cornerstone for understanding the changes in the plant metabolism after incubation with DCF and CBZ. Also, the mass spectrometric untargeted metabolomics strategy has a substantial role in investigating the biochemical changes and metabolic adaptation of plants in xenobiotics exposure cases.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>This work was assisted by the German Research Foundation (DFG) and the Technical University of Munich (TUM) in the scope of the Open Access Publishing Program.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note:</b> MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>The supplementary materials are available online at <span class="uri" title="uri"><a href="https://www.mdpi.com/2218-1989/11/1/2/s1">https://www.mdpi.com/2218-1989/11/1/2/s1</a></span>, Figure S1: Retention time (RT)/Mass plot of the background was analyzed by RPLC-HILIC-ESI-TOF-MS in positive electrospray ionization mode, Figure S2: The Q<sup>2</sup>/ R<sup>2</sup> Overview plot displays the individual cumulative R<sup>2</sup> (green columns) and Q<sup>2</sup> (blue columns) and Q<sup>2</sup> for the goodness of fits and cross-validation parameters (a) <i>P. australis</i> different parts. (b) <i>P. australis</i> different incubation, Figure S3: EICs were corresponding to measured diclofenac (right) and the reference standard (left), which were identified in the extracts of <i>Phragmites australis</i> leaf, rhizome, and roots incubated with 10 and 100 Î¼M diclofenac. Also, EICs relative to transformed products are suspected in the extracts of <i>Phragmites australis</i> leaf, rhizome, and roots incubated with 10 and 100 Î¼M diclofenac, Figure S4: EICs were corresponding to carbamazepine (CBZ) and its transformed product standards (left), which were identified in the extracts of <i>Phragmites australis</i> leaf, rhizome, and roots incubated with 10 and 50Î¼M carbamazepine (measured right), Table S1: The standards compounds of the quality control external calibration mixture, monoisotopic mass in the literature (L), monoisotopic in different injection and the mean of them, the variation between monoisotopic mass in the literature (L), and the mean of measured monoisotopic mass, and mean mass standard deviation (SD) are listed, Table S2: The standards compounds of the quality control external calibration mixture, the single RT, mean RT of the different injections, mean RT standard deviation (SD), and relative standard deviation (RSD) of the standards are listed, Table S3: List of carbamazepine (CBZ) transformed product identified in <i>Phragmites australis</i> different samples with the mean monoisotopic mass in the standards (S), the mean monoisotopic mass of <i>Phragmites australis</i> (Ph), the variation between them, mean RT of standards (S), mean RT of <i>Phragmites australis</i> (Ph), and the variation between them were listed, Table S4: The differentiating metabolic profile (DMF) metabolites of <i>Phragmites australis</i> due to incubation with 10 &amp;amp; 100 ÂµM diclofenac were extracted from the S-plot, monoisotopic mass, <i>p</i>-value, and t. score are listed, Table S5. The differentiating metabolic profile (DMF) metabolites of <i>Phragmites australis</i> due to incubation with 10 &amp;amp; 50 ÂµM carbamazepine were extracted from the S-plot, monoisotopic mass, <i>p</i>-value, and t.score are listed.</p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="metabolites-11-00002-s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>R.W. and T.L. designed the metabolomics study. R.W. prepared the samples, performed the untargeted metabolomics analysis and ran the analyses in the metabolomics platform. S.M. created the statistical design for data evaluation and data interpretation and performed the realization together with R.W. and T.L., A.S. and P.S. created the test set and performed the plant growing and incubation together with R.W. They also supported R.W. in biological data interpretation. R.W. and T.L. conceived and drafted the manuscript. All the authors contributed with critical intellectual input. All authors have read and agreed to the published version of the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This research was partially funded by the Bavarian State Ministry of the Environment and Consumer Protection as well as the Cultural Affairs and Mission Sector of the Egyptian Ministry of Higher Education.</p> <div class="title" tagx="title" title="title">
Institutional Review Board Statement</div> <p>Not applicable.</p> <div class="title" tagx="title" title="title">
Informed Consent Statement</div> <p>Not applicable.</p> <div class="title" tagx="title" title="title">
Data Availability Statement</div> <p>The data is available at <span class="uri" title="uri"><a href="https://www.ebi.ac.uk/metabolights/">https://www.ebi.ac.uk/metabolights/</a></span> with IDâ€"sMTBLS2321.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare that they have no conflict of interest. There is no connection between AFIN-TS GMbH and the subject of this manuscript.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-metabolites-11-00002" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tohge</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishiyama</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirai</span><span class="given-names" tagx="given-names" title="given-names">M.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yano</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakajima</span><span class="given-names" tagx="given-names" title="given-names">J.-I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Awazuhara</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inoue</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahashi</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodenowe</span><span class="given-names" tagx="given-names" title="given-names">D.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitayama</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="source" tagx="source" title="source">Identification of Genes Involved in Anthocyanin Accumulation by Integrated Analysis of Metabolome and Transcriptome in Pap1-Overexpressing Arabidopsis Plants</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Dordrecht, The Netherlands</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">159</span><span class="lpage" tagx="lpage" title="lpage">168</span></span></li> <li tag="ref"><a name="B2-metabolites-11-00002" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sandermann</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diesperger</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scheel</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="source" tagx="source" title="source">Metabolism of Xenobiotics by Plant Cell Cultures</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin/Heidelberg, Germany</span><span class="year" tagx="year" title="year">1977</span><span class="fpage" tagx="fpage" title="fpage">178</span><span class="lpage" tagx="lpage" title="lpage">196</span></span></li> <li tag="ref"><a name="B3-metabolites-11-00002" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchrÃ¶der</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scheer</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belford</span><span class="given-names" tagx="given-names" title="given-names">E.J.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolism of organic xenobiotics in plants: Conjugating enzymes and metabolic endpoints</span><span class="source" tagx="source" title="source">Minerva Biotecnol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">91</span></span></li> <li tag="ref"><a name="B4-metabolites-11-00002" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchrÃ¶der</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maier</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debus</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detoxification of Herbicides in <i>Phragmites australis</i></span><span class="source" tagx="source" title="source">Z. Nat. C</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">317</span><span class="pub-id"><a href="https://dx.doi.org/10.1515/znc-2005-3-417">10.1515/znc-2005-3-417</a></span></span></li> <li tag="ref"><a name="B5-metabolites-11-00002" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villette</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maurer</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wanko</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heintz</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Xenobiotics metabolization in Salix alba leaves uncovered by mass spectrometry imaging</span><span class="source" tagx="source" title="source">Metabolomics</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">122</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11306-019-1572-8">10.1007/s11306-019-1572-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31471668">31471668</a></span></span></li> <li tag="ref"><a name="B6-metabolites-11-00002" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letzel</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metzner</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letzel</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exposure assessment of the pharmaceutical diclofenac based on long-term measurements of the aquatic input</span><span class="source" tagx="source" title="source">Environ. Int.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">363</span><span class="lpage" tagx="lpage" title="lpage">368</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.envint.2008.09.002">10.1016/j.envint.2008.09.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19027956">19027956</a></span></span></li> <li tag="ref"><a name="B7-metabolites-11-00002" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SauvÃªtre</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchrÃ¶der</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Uptake of carbamazepine by rhizomes and endophytic bacteria of <i>Phragmites australis</i></span><span class="source" tagx="source" title="source">Front. Plant Sci.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">83</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2015.00083">10.3389/fpls.2015.00083</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25750647">25750647</a></span></span></li> <li tag="ref"><a name="B8-metabolites-11-00002" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thelusmond</span><span class="given-names" tagx="given-names" title="given-names">J.-R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawka</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strathmann</span><span class="given-names" tagx="given-names" title="given-names">T.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cupples</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diclofenac, carbamazepine and triclocarban biodegradation in agricultural soils and the microorganisms and metabolic pathways affected</span><span class="source" tagx="source" title="source">Sci. Total Environ.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">640â€"641</span><span class="fpage" tagx="fpage" title="fpage">1393</span><span class="lpage" tagx="lpage" title="lpage">1410</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.scitotenv.2018.05.403">10.1016/j.scitotenv.2018.05.403</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30021306">30021306</a></span></span></li> <li tag="ref"><a name="B9-metabolites-11-00002" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vieno</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SillanpÃ¤Ã¤</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fate of diclofenac in municipal wastewater treatment plantâ€"A review</span><span class="source" tagx="source" title="source">Environ. Int.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">28</span><span class="lpage" tagx="lpage" title="lpage">39</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.envint.2014.03.021">10.1016/j.envint.2014.03.021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24791707">24791707</a></span></span></li> <li tag="ref"><a name="B10-metabolites-11-00002" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hai</span><span class="given-names" tagx="given-names" title="given-names">F.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asif</span><span class="given-names" tagx="given-names" title="given-names">M.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sencadas</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shawkat</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanderson-Smith</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorman</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Z.-Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamamoto</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Carbamazepine as a Possible Anthropogenic Marker in Water: Occurrences, Toxicological Effects, Regulations and Removal by Wastewater Treatment Technologies</span><span class="source" tagx="source" title="source">Water</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">107</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/w10020107">10.3390/w10020107</a></span></span></li> <li tag="ref"><a name="B11-metabolites-11-00002" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartha</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchrÃ¶der</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolism of diclofenac in plantsâ€"hydroxylation is followed by glucose conjugation</span><span class="source" tagx="source" title="source">J. Hazard. Mater.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">243</span><span class="fpage" tagx="fpage" title="fpage">250</span><span class="lpage" tagx="lpage" title="lpage">256</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jhazmat.2012.10.023">10.1016/j.jhazmat.2012.10.023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23137548">23137548</a></span></span></li> <li tag="ref"><a name="B12-metabolites-11-00002" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">M.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blossey</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Importance of plant traits, and herbivory for invasiveness of <i>Phragmites australis</i> (Poaceae)</span><span class="source" tagx="source" title="source">Am. J. Bot.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">95</span><span class="fpage" tagx="fpage" title="fpage">1557</span><span class="lpage" tagx="lpage" title="lpage">1568</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/ajb.0800023">10.3732/ajb.0800023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21628163">21628163</a></span></span></li> <li tag="ref"><a name="B13-metabolites-11-00002" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">K.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biddlestone</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Engineered reed-bed systems for wastewater treatment</span><span class="source" tagx="source" title="source">Trends Biotechnol.</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">248</span><span class="lpage" tagx="lpage" title="lpage">252</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0167-7799(00)88958-1">10.1016/S0167-7799(00)88958-1</a></span></span></li> <li tag="ref"><a name="B14-metabolites-11-00002" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuli</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ressom</span><span class="given-names" tagx="given-names" title="given-names">H.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">LC-MS-based metabolomics</span><span class="source" tagx="source" title="source">Mol. Biosyst.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">470</span><span class="lpage" tagx="lpage" title="lpage">481</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/C1MB05350G">10.1039/C1MB05350G</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22041788">22041788</a></span></span></li> <li tag="ref"><a name="B15-metabolites-11-00002" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beale</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinu</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouremenos</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poojary</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narayana</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boughton</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanojia</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dayalan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dias</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Review of recent developments in GCâ€"MS approaches to metabolomics-based research</span><span class="source" tagx="source" title="source">Metabolomics</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">152</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11306-018-1449-2">10.1007/s11306-018-1449-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30830421">30830421</a></span></span></li> <li tag="ref"><a name="B16-metabolites-11-00002" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emwas</span><span class="given-names" tagx="given-names" title="given-names">A.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKay</span><span class="given-names" tagx="given-names" title="given-names">R.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tenori</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saccenti</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gowda</span><span class="given-names" tagx="given-names" title="given-names">G.A.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raftery</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alahmari</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaremko</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaremko</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">NMR Spectroscopy for Metabolomics Research</span><span class="source" tagx="source" title="source">Metabolites</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">123</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/metabo9070123">10.3390/metabo9070123</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31252628">31252628</a></span></span></li> <li tag="ref"><a name="B17-metabolites-11-00002" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aretz</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meierhofer</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Advantages, and pitfalls of mass spectrometry-based metabolome profiling in systems biology</span><span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">17</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">632</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms17050632">10.3390/ijms17050632</a></span></span></li> <li tag="ref"><a name="B18-metabolites-11-00002" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wahman</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grassmann</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchrÃ¶der</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letzel</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant metabolomic workflows using reversed-phase LC and HILIC with ESI-TOF-MS</span><span class="source" tagx="source" title="source">LCGC N. Am.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">8</span><span class="lpage" tagx="lpage" title="lpage">15</span></span></li> <li tag="ref"><a name="B19-metabolites-11-00002" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riach</span><span class="given-names" tagx="given-names" title="given-names">A.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perera</span><span class="given-names" tagx="given-names" title="given-names">M.V.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Florance</span><span class="given-names" tagx="given-names" title="given-names">H.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Penfield</span><span class="given-names" tagx="given-names" title="given-names">S.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">J.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of plant leaf metabolites reveals no common response to insect herbivory by <i>Pieris rapae</i> in three related host-plant species</span><span class="source" tagx="source" title="source">J. Exp. Bot.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">2547</span><span class="lpage" tagx="lpage" title="lpage">2556</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/erv045">10.1093/jxb/erv045</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25711707">25711707</a></span></span></li> <li tag="ref"><a name="B20-metabolites-11-00002" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gromski</span><span class="given-names" tagx="given-names" title="given-names">P.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kotze</span><span class="given-names" tagx="given-names" title="given-names">H.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Correa</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ellis</span><span class="given-names" tagx="given-names" title="given-names">D.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Armitage</span><span class="given-names" tagx="given-names" title="given-names">E.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">M.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodacre</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influence of missing values substitutes on multivariate analysis of metabolomics data</span><span class="source" tagx="source" title="source">Metabolites</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">433</span><span class="lpage" tagx="lpage" title="lpage">452</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/metabo4020433">10.3390/metabo4020433</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24957035">24957035</a></span></span></li> <li tag="ref"><a name="B21-metabolites-11-00002" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamichhane</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sen</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dickens</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HyÃ¶tylÃ¤inen</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">OreÅ¡iÄ�</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chapter Fourteenâ€"An Overview of metabolomics data analysis: Current tools and future perspectives</span><span class="source" tagx="source" title="source">Compr. Anal. Chem.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">82</span><span class="fpage" tagx="fpage" title="fpage">387</span><span class="lpage" tagx="lpage" title="lpage">413</span></span></li> <li tag="ref"><a name="B22-metabolites-11-00002" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prinsloo</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vervoort</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identifying anti-HSV compounds from unrelated plants using NMR and LC-MS metabolomic analysis</span><span class="source" tagx="source" title="source">Metabolomics</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">134</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11306-018-1432-y">10.1007/s11306-018-1432-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30830404">30830404</a></span></span></li> <li tag="ref"><a name="B23-metabolites-11-00002" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Worley</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powers</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Multivariate Analysis in Metabolomics</span><span class="source" tagx="source" title="source">Curr. Metab.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">92</span><span class="lpage" tagx="lpage" title="lpage">107</span></span></li> <li tag="ref"><a name="B24-metabolites-11-00002" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bieber</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greco</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grosse</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letzel</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">RPLC-HILIC, and SFC with mass spectrometry: Polarity-extended organic molecule screening in environmental (Water) samples</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">7907</span><span class="lpage" tagx="lpage" title="lpage">7914</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.analchem.7b00859">10.1021/acs.analchem.7b00859</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28650149">28650149</a></span></span></li> <li tag="ref"><a name="B25-metabolites-11-00002" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greco</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grosse</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letzel</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Serial coupling of reversed-phase and zwitterionic hydrophilic interaction LC/MS for the analysis of polar and nonpolar phenols in wine</span><span class="source" tagx="source" title="source">J. Sep. Sci.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">1379</span><span class="lpage" tagx="lpage" title="lpage">1388</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jssc.201200920">10.1002/jssc.201200920</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23505207">23505207</a></span></span></li> <li tag="ref"><a name="B26-metabolites-11-00002" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wahman</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GraÃŸmann</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SauvÃªtre</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchrÃ¶der</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letzel</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Lemna minor</i> studies under various storage periods using extended-polarity extraction and metabolite non-target screening analysis</span><span class="source" tagx="source" title="source">J. Pharm. Biomed. Anal.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">188</span><span class="fpage" tagx="fpage" title="fpage">113362</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jpba.2020.113362">10.1016/j.jpba.2020.113362</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32526623">32526623</a></span></span></li> <li tag="ref"><a name="B27-metabolites-11-00002" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajab</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greco</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heim</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helmreich</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letzel</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Serial coupling of RP and zwitterionic hydrophilic interaction LC-MS: Suspects screening of diclofenac transformation products by oxidation with a boron-doped diamond electrode</span><span class="source" tagx="source" title="source">J. Sep. Sci.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">3011</span><span class="lpage" tagx="lpage" title="lpage">3018</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jssc.201300562">10.1002/jssc.201300562</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23857646">23857646</a></span></span></li> <li tag="ref"><a name="B28-metabolites-11-00002" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauvetre</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harpaintner</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poschenrieder</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchrÃ¶der</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolism of carbamazepine in plant roots and endophytic rhizobacteria isolated from <i>Phragmites australis</i></span><span class="source" tagx="source" title="source">J. Hazard. Mater.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">342</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">95</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jhazmat.2017.08.006">10.1016/j.jhazmat.2017.08.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28823920">28823920</a></span></span></li> <li tag="ref"><a name="B29-metabolites-11-00002" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hess</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="source" tagx="source" title="source">Plant Physiology</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin/Heidelberg, Germany</span><span class="year" tagx="year" title="year">1975</span><span class="fpage" tagx="fpage" title="fpage">334</span></span></li> <li tag="ref"><a name="B30-metabolites-11-00002" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riemenschneider</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seiwert</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwarz</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reemtsma</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Extensive Transformation of the pharmaceutical carbamazepine following uptake into intact tomato plants</span><span class="source" tagx="source" title="source">Environ. Sci. Technol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">6100</span><span class="lpage" tagx="lpage" title="lpage">6109</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.est.6b06485">10.1021/acs.est.6b06485</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28506063">28506063</a></span></span></li> <li tag="ref"><a name="B31-metabolites-11-00002" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riemenschneider</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seiwert</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldstein</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Raggad</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salameh</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chefetz</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reemtsma</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">An LC-MS/MS method for the determination of 28 polar environmental contaminants and metabolites in vegetables irrigated with treated municipal wastewater</span><span class="source" tagx="source" title="source">Anal. Methods</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">1273</span><span class="lpage" tagx="lpage" title="lpage">1281</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/C6AY02984A">10.1039/C6AY02984A</a></span></span></li> <li tag="ref"><a name="B32-metabolites-11-00002" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MartÃ­nez-Piernas</span><span class="given-names" tagx="given-names" title="given-names">A.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nahim-Granados</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Polo-LÃ³pez</span><span class="given-names" tagx="given-names" title="given-names">M.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FernÃ¡ndez-IbÃ¡Ã±ez</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murgolo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mascolo</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AgÃ¼era</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of transformation products of carbamazepine in lettuce crops irrigated with Ultraviolet-C treated water</span><span class="source" tagx="source" title="source">Environ. Pollut.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">247</span><span class="fpage" tagx="fpage" title="fpage">1009</span><span class="lpage" tagx="lpage" title="lpage">1019</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.envpol.2019.02.001">10.1016/j.envpol.2019.02.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30823329">30823329</a></span></span></li> <li tag="ref"><a name="B33-metabolites-11-00002" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tybring</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Bahr</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertilsson</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Collste</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glaumann</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solbrand</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolism of carbamazepine and its epoxide metabolite in human and rat liver in vitro</span><span class="source" tagx="source" title="source">Drug Metab. Dispos.</span><span class="year" tagx="year" title="year">1981</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">561</span><span class="lpage" tagx="lpage" title="lpage">564</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6120817">6120817</a></span></span></li> <li tag="ref"><a name="B34-metabolites-11-00002" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">D.-K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahn</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">H.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yun</span><span class="given-names" tagx="given-names" title="given-names">H.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">J.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwon</span><span class="given-names" tagx="given-names" title="given-names">S.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic response induced by parasitic plant-fungus interactions hinder amino sugar and nucleotide sugar metabolism in the host</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep37434">10.1038/srep37434</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28442746">28442746</a></span></span></li> <li tag="ref"><a name="B35-metabolites-11-00002" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurtado</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parastar</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matamoros</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PiÃ±a</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tauler</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayona</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Linking the morphological and metabolomic response of <i>Lactuca sativa</i> L exposed to emerging contaminants using GCâ€‰Ã—â€‰GC-MS and chemometric tools</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">6546</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-06773-0">10.1038/s41598-017-06773-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28747703">28747703</a></span></span></li> <li tag="ref"><a name="B36-metabolites-11-00002" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tantikanjana</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mikkelsen</span><span class="given-names" tagx="given-names" title="given-names">M.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hussain</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halkier</span><span class="given-names" tagx="given-names" title="given-names">B.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sundaresan</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functional Analysis of the tandem-duplicated P450 genes SPS/BUS/CYP79F1 and CYP79F2 in glucosinolate biosynthesis and plant development by Ds transposition-generated double mutants</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">135</span><span class="fpage" tagx="fpage" title="fpage">840</span><span class="lpage" tagx="lpage" title="lpage">848</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.104.040113">10.1104/pp.104.040113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15194821">15194821</a></span></span></li> <li tag="ref"><a name="B37-metabolites-11-00002" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zrenner</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stitt</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sonnewald</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boldt</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pyrimidine and purine biosynthesis and degradation in plants</span><span class="source" tagx="source" title="source">Annu. Rev. Plant Biol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">805</span><span class="lpage" tagx="lpage" title="lpage">836</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.arplant.57.032905.105421">10.1146/annurev.arplant.57.032905.105421</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16669783">16669783</a></span></span></li> <li tag="ref"><a name="B38-metabolites-11-00002" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jing</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Glutathione Metabolism and Its Function in Higher Plants Adapting to Stress</span><span class="source" tagx="source" title="source">Antioxidants and Antioxidant Enzymes in Higher Plants</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer International Publishing</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Heidlberg, Germany</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">205</span></span></li> <li tag="ref"><a name="B39-metabolites-11-00002" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sivaram</span><span class="given-names" tagx="given-names" title="given-names">A.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subashchandrabose</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Logeshwaran</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lockington</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naidu</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Megharaj</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolomics reveals defensive mechanisms adapted by maize on exposure to high molecular weight polycyclic aromatic hydrocarbons</span><span class="source" tagx="source" title="source">Chemosphere</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">214</span><span class="fpage" tagx="fpage" title="fpage">771</span><span class="lpage" tagx="lpage" title="lpage">780</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chemosphere.2018.09.170">10.1016/j.chemosphere.2018.09.170</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30296765">30296765</a></span></span></li> <li tag="ref"><a name="B40-metabolites-11-00002" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AraÃºjo</span><span class="given-names" tagx="given-names" title="given-names">W.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nunes-Nesi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikoloski</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sweetlove</span><span class="given-names" tagx="given-names" title="given-names">L.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernie</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic control and regulation of the tricarboxylic acid cycle in photosynthetic and heterotrophic plant tissues</span><span class="source" tagx="source" title="source">Plant Cell Environ.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-3040.2011.02332.x">10.1111/j.1365-3040.2011.02332.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21477125">21477125</a></span></span></li> <li tag="ref"><a name="B41-metabolites-11-00002" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LÃ¶fstedt</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trygg</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">OnPLSâ€"A novel multiblock method for the modeling of predictive and orthogonal variation</span><span class="source" tagx="source" title="source">J. Chemom.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">441</span><span class="lpage" tagx="lpage" title="lpage">455</span></span></li> <li tag="ref"><a name="B42-metabolites-11-00002" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chong</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wishart</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Using MetaboAnalyst 4.0 for Comprehensive and Integrative Metabolomics Data Analysis</span><span class="source" tagx="source" title="source">Curr. Protoc. Bioinform.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">e86</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cpbi.86">10.1002/cpbi.86</a></span></span></li> <li tag="ref"><a name="B43-metabolites-11-00002" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chong</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamamoto</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MetaboAnalystR 2.0: From Raw Spectra to Biological Insights</span><span class="source" tagx="source" title="source">Metabolites</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">57</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/metabo9030057">10.3390/metabo9030057</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Retention time (RT)/Mass plot of <i>Phragmites australis</i> 100% Methanol extracts analyzed by RPLC-HILIC-ESI-TOF-MS in positive electrospray ionization mode. (<b>a</b>) Leaf; (<b>b</b>) Rhizome; (<b>c</b>) Root, which showed the featuresâ€™ separation according to their polarity and detected according to their <i>m/z</i>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>(<b>a</b>) The OPLS-DA score plot of different <i>Phragmites australis</i> extracts with a confidence limit of 95% discriminating according to the solvent used in the extraction. The variables were plotted according to the first principal component (t1) and the orthogonal component (t2). The triangles represent 100% methanol extracts, circles represent acidic 90% methanol extracts, the squares represent 50% methanol extracts and the stars represent aqueous extracts, respectively. The green color represents leaf samples, the light green color represents rhizome samples and the brown color represents root samples, respectively. Each symbol represents one observation of <i>P. australis</i> leaf, rhizome and root plant part; (<b>b</b>) loading scatter plot for the selected principal components; (<b>c</b>) the Q<sup>2</sup>/R<sup>2</sup> Overview plot displays the individual cumulative R<sup>2</sup> (green columns) and Q<sup>2</sup> (blue columns) for the goodness of fits and cross-validation parameters; (<b>d</b>) the loading scatter plot for the selected principal components displays the relation between the different <i>Phragmites australis</i> samples and the chosen metabolites; (<b>e</b>) the OPLS-DA score plot of the different parts of <i>Phragmites australis</i> with a confidence limit of 95%, discriminating according to the plant part.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>(<b>a</b>) The loading plot displays the relation between the different <i>Phragmites australis</i> samples and the chosen metabolite; (<b>b</b>) the OPLS-DA score plot of different <i>Phragmites australis</i> samples incubated with 10 and 50 Î¼M carbamazepine, 10 and 100 Î¼M diclofenac, individually. The confidence limit is 95%. For carbamazepine incubation the purple color represents the control group, the light blue represents a sample incubated with 10 Î¼M carbamazepine and the blue color represents samples incubated with 50 Î¼M carbamazepine. For diclofenac incubation the light purple color represents the control group, the yellow color represents samples incubated with 10 Î¼M diclofenac and the orange color represents samples incubated with 100 Î¼M diclofenac.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>(<b>a</b>) S-plot of <i>Phragmites australis</i> control and incubated with 10 and 100 Î¼M diclofenac samples; (<b>b</b>) S-plot of <i>Phragmites australis</i> control and incubated with 10 and 50 Î¼M carbamazepine samples. The S-plot provides the visualization of the loading components (green color) of OPLS-DA to enable the interpretation of the data. The red-labeled compounds represent the differentiating metabolic profile (DMF) of each incubation; (<b>c</b>) the contribution plot shows the up- and down-regulated compounds due to the incubation of <i>Phragmites australis</i> with 10 and 100 Î¼M diclofenac, individually. Down-regulated compounds have negative values, while up-regulated compounds have positive values. (1) Quercetin, DM_6, DM_4 and DM_7 have up-regulated in <i>Phragmites australis</i> due to incubation with diclofenac at (8820, 0.0385581), (9968, 0101411), (10,817, 0.279365) and (10,985, 0.389), respectively; (<b>d</b>) the contribution plot of <i>Phragmites australis</i> with 10 and 50 Î¼M carbamazepine, individually. Down-regulated compounds have negative values as compound 1 which was quercetin at (2033, âˆ’0.118706). Up-regulated compounds have positive values as compounds 2 and 3 which were 2,3-dihydro-2,3-dihydroxycarbamazepine at (10,921, 0.288274) and carbamazepine-10,11-epoxide at (11,270, 1.27941), respectively.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Overview of pathway analysis using MetaboAnalyst 4.0. For metabolite set enrichment analysis of <i>Phragmites australis</i> differentiating metabolic profile (DMF) after incubation with (<b>a</b>) 10 and 100 Î¼M diclofenac; (<b>b</b>) 10 and 50 Î¼M carbamazepine. The overview displays all matched pathways as circles. The color and size of each circle are based on the <i>p</i> value and the impact of the pathway value, respectively. The names of the metabolic pathways are listed in the table.</p>   </div> <div class="table-wrap_UNKNOWN" id="metabolites-11-00002-t001" orientation="portrait" position="float">
metabolites-11-00002-t001_Table 1<span class="label" tagx="label" title="label">Table 1</span> <p>List of diclofenac (DCF) transformation products detected in different <i>Phragmites australis</i> leaf, rhizome and roots incubated with 10 and 100 ÂµM diclofenac samples with the monoisotopic mass in the literature (L), the mean monoisotopic mass of <i>Phragmites australis</i> (Ph), the variation between them, mean RT of <i>Phragmites australis</i> (Ph), standard deviation, relative standard deviation and LogD (pH = 7.4) were listed. The logD values were predicted from ChemAxon software (<span class="uri" title="uri"><a href="https://disco.chemaxon.com/apps/demos/logd/">https://disco.chemaxon.com/apps/demos/logd/</a></span>).</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">DCF Transformed Products</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Name</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Mono <br /><br />Isotopic Mass (Da) <br /><br />(L) (Rajab, Greco et al. 2013)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Mean <br /><br />Mono <br /><br />Isotopic Mass <br /><br />(Da) <br /><br />(Ph) </th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Î"ppm </th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Mean RT (Min)<br /><br />(Ph) </th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">SD <br /><br />of RT<br /><br /> (Min) </th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">RSD </th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">LogD <br /><br />(pH = 7.4)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Leaf</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Rhizome</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Root</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>DM_1</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2-Hydroxypropanoic acid</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">152.0473</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">152.0475</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’0.99</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">8.1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.05</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.67</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’1.86</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>DM_2</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2-(Hydroxymethyl)benzene-1,4-diol</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">140.0473</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">140.0472</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.71</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">12.5</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.06</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.51</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.60</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>DM_3</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2-Hydroxysuccinic acid</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">134.0215</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">134.0215</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.07</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">12.6</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.03</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.23</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’6.81</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>DM_4</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Succinic acid</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">118.0266</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">118.0271</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’3.95</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">6.8</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.04</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.55</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’1.99</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>DM_5</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Fumaric acid</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">116.0101</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">116.01</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.49</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">12.6</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.05</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.42</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’2.00</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>DM_6</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Propane-1,2,3-triol</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">92.0473</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">92.04703</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2.9</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7.2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.07</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.99</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’1.84</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Ã—</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Ã—</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>DM_7</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2-Hydroxypropanoic acid</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">90.0317</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">90.03207</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’4.07</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">12.1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.04</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.32</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’1.00</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆš</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Ã—</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Ã—</td> </tr> </tbody> </table> </div> </div>  </body></html>