<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Mol Divers</span><span class="iso-abbrev" title="iso-abbrev">Mol Divers</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Molecular Diversity</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1381-1991</span><span class="issn-epub" tagx="issn" title="issn-epub">1573-501X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer International Publishing</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cham</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7778709">7778709</a></span><span class="publisher-id" title="publisher-id">10169</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1007/s11030-020-10169-0">10.1007/s11030-020-10169-0</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
CoViTris2020 and ChloViD2020: a striking new hope in COVID-19 therapy</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3681-114X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabie</span><span class="given-names" tagx="given-names" title="given-names">Amgad M.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">amgadpharmacist1@yahoo.com</span><span class="email" tagx="email" title="email">dr.amgadrabie@gmail.com</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="citation_author_institution" id="Aff1">[1], </span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.10251.37</span><span class="institution-id" title="institution-id">0000000103426662</span><span class="institution" title="institution">Pharmaceutical Organic Chemistry Department, Faculty of Pharmacy, </span><span class="institution" title="institution">Mansoura University, </span></div></span><span class="citation_author_institution" id="Aff3">[3], </span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-1-1</span></span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">16</span><span class="history" title="history"><span class="received" title="received">received: 2020-8-7</span><span class="accepted" title="accepted">accepted: 2020-12-4</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted research re-use and secondary analysis in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="abstract" title="sec"> <div class="title" tagx="title" title="title">
Abstract</div> <p id="Par1">Designing anticoronavirus disease 2019 (anti-COVID-19) agents is the primary concern of medicinal chemists/drug designers nowadays. Repurposing of known active compounds against the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a new effective and time-saving trend in anti-COVID-19 drug discovery. Thorough inhibition of the coronaviral-2 proteins (i.e., multitarget inhibition) is a possible powerful favorable strategy for developing effectively potent drugs for COVID-19. In this new research study, I succeeded to repurpose the two antioxidant polyhydroxy-1,3,4-oxadiazole compounds <b>CoViTris2020</b> and <b>ChloViD2020</b> as the first multitarget coronaviral protein blockers with extremely higher potencies (reach about 65 and 304 times, for <b>CoViTris2020</b>, and 20 and 93 times, for <b>ChloViD2020</b>, more potent than remdesivir and favipiravir, respectively). These two 2,5-disubstituted-1,3,4-oxadiazoles were computationally studied (through molecular docking in almost all SARS-CoV-2 proteins) and biologically assessed (through a newly established robust in vitro anti-COVID-19 assay) for their anticoronaviral-2 bioactivities. The data obtained from the docking investigation showed that both ligands promisingly exhibited very strong inhibitory binding affinities with almost all docked enzymes (e.g., they displayed extremely lower binding energies of âˆ’â€‰12.00 and âˆ’â€‰9.60Â kcal/mol, respectively, with the SARS-CoV-2 RNA-dependent RNA polymerase â€œRdRpâ€�). The results of the biological assay revealed that <b>CoViTris2020</b> and <b>ChloViD2020</b> significantly displayed very high anti-COVID-19 activities (anti-SARS-CoV-2 EC<sub>50</sub>â€‰=â€‰0.31 and 1.01Â Î¼M, respectively). Further in vivo/clinical studies for the development of <b>CoViTris2020</b> and <b>ChloViD2020</b> as anti-COVID-19 medications are required. In brief, the ascent of <b>CoViTris2020</b> and <b>ChloViD2020</b> as the two lead members of the novel family of anti-COVID-19 polyphenolic 2,5-disubstituted-1,3,4-oxadiazole derivatives represents a promising hope in COVID-19 therapy.</p> </div> <div class="graphicabstract" title="sec"> <div class="title" tagx="title" title="title">
Graphic abstract</div> <p id="Par150"><b>CoViTris2020</b> and <b>ChloViD2020</b> inhibit SARS-CoV-2 life cycle with surprising EC<sub>50</sub> values of 0.31 and 1.01 Î¼M, respectively. <b>CoViTris2020</b> strongly inhibits coronaviral-2 RdRp with exceptionally lower inhibitory binding energy of âˆ’Â 12.00 kcal/mol. </p><div class="graphic" position="anchor" id="MO9" /> <p /> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Anti-COVID-19 drug</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
Coronavirus</div> <div class="kwd" title="kwd">
Coronaviral-2</div> <div class="kwd" title="kwd">
RNA-dependent RNA polymerase (RdRp)</div> <div class="kwd" title="kwd">
Papain-like protease (PLpro)</div> <div class="kwd" title="kwd">
Polyphenolic 2,5-disubstituted-1,3,4-oxadiazole</div> <div class="kwd" title="kwd">
Remdesivir</div> <div class="kwd" title="kwd">
Ivermectin</div> <div class="kwd" title="kwd">
Favipiravir</div> <div class="kwd" title="kwd">
CoViTris2020</div> <div class="kwd" title="kwd">
ChloViD2020</div> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">A broad variety of tactics and strategies was employed to fight the current worldwide coronavirus disease 2019 (COVID-19) epidemic as an urgent result of the novelty of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections, and the lack of effective medications [<a href="#CR1">1</a>â€"<a href="#CR3">3</a>]. Many plans of these emerging strategies depend on reevaluating and repurposing existent drugs, and some others are totally new [<a href="#CR4">4</a>]. The design in each case depends on present scientific proof and evidence of logic mechanistic approaches that are efficient against either analogous/identical viral infections or the severe signs/symptoms that are caused by COVID-19 [<a href="#CR1">1</a>â€"<a href="#CR4">4</a>]. Comprehensive inhibition of the coronaviral-2 proteins (i.e., multitarget inhibition) can establish one of the most promising plans for developing effectively potent drugs for COVID-19. However, all the discovered blockers of the main stages of SARS-CoV-2 life cycle lack the required maximal potencies against the major SARS-CoV-2 enzymes (e.g., RNA-dependent RNA polymerase â€œRdRpâ€�, papain-like protease â€œPLproâ€�, and main protease â€œMproâ€�) [<a href="#CR3">3</a>, <a href="#CR4">4</a>].</p> <p id="Par3">Many known drugs are currently under anti-COVID-19 therapeutic investigation, such as remdesivir (antiviral), ivermectin (antiparasitic), favipiravir (antiinfluenza), hydroxychloroquine (antimalarial and antirheumatic disorders), and arbidol (antiinfluenza) [<a href="#CR4">4</a>â€"<a href="#CR17">17</a>]. Perspectives of many medicinal chemists are based on utilizing the in silico computational molecular modeling studies as the starting points for designing novel potent compounds active against SARS-CoV-2, and most of these studies have been collected in currently published reviews [<a href="#CR18">18</a>, <a href="#CR19">19</a>]. This current direction in searching for effective safe known compounds having the structural flexibility to be successfully redirected and repurposed as potential agents against COVID-19 triggered me to screen our library of known compounds/drugs (including my previously synthesized active compounds). Surprisingly, the predictive computational molecular search and screening gave rise to the discovery of two promising compounds in respect to the anti-SARS-CoV-2 activities, 1,2,3-tris[5-(3,4,5-trihydroxyphenyl)-1,3,4-oxadiazol-2-yl]propan-2-ol (<b>CoViTris2020</b>) and 5-[5-(7-chloro-4-hydroxyquinolin-3-yl)-1,3,4-oxadiazol-2-yl]benzene-1,2,3-triol (<b>ChloViD2020</b>) (Fig.Â <a href="#Fig1">1</a>). It is worth noticing that both compounds were previously synthesized and evaluated as effective antioxidant agents [<a href="#CR20">20</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Chemical structures of <b>CoViTris2020</b> and <b>ChloViD2020</b> molecules</p>   </div> <p /> <p id="Par4">The adaptable and flexible scaffolds of <b>CoViTris2020</b> and <b>ChloViD2020</b> molecules are characterized by carrying significantly considerable number of hydrogen bond donors/acceptors and aromatic rings, which are the principal key moieties required for the best/ideal/strongest binding (hydrogen bonding and hydrophobic interactions) with the binding domains of almost all the COVID-19 protein targets in SARS-CoV-2 and human. It is worth mentioning that <b>CoViTris2020</b> and <b>ChloViD2020</b> are chemically very analogous to RNA nucleos(t)ides, giving them the expected abilities of being efficient bioisosteric antimetabolites in COVID-19 treatment. Additionally, <b>CoViTris2020</b> and <b>ChloViD2020</b> structures are also characterized by their abilities to strongly bound to zinc atoms, where they have many active zincophoric centers (i.e., they are rich in oxygenous and nitrogenous moieties) allowing them to act as strong multizincophores. Zinc atom/ion carriers have been extensively studied especially for their potent antiviral activities and they have been found to effectively inhibit the replication/reproduction machinery of several viruses in vitro [<a href="#CR21">21</a>]. Specifically, Zn<sup>2+</sup> ions in certain exceedingly higher/lower concentrations can impair and disrupt the coronaviral replication and transcription and, therefore, excessive displacement or transport of Zn<sup>2+</sup> ions by zincophoric agents (e.g., <b>CoViTris2020</b> and <b>ChloViD2020</b>) will undoubtedly affect the coronaviral replication processes (zinc ionophores have been found to successfully and significantly inhibit the replication process of coronaviruses intracellularly in cell cultures) [<a href="#CR10">10</a>, <a href="#CR15">15</a>, <a href="#CR22">22</a>]. The considerable chemical resonance of both structures aids in boosting the comprehensive potential anti-COVID-19 bioactivities through intensifying the inhibitory chemical stabilities of the complexes of each compound with the active site residues of the coronaviral-2/human target proteins [<a href="#CR20">20</a>]. It is also worth noticing that the potent antioxidant actions of both compounds may aid their anti-SARS-CoV-2/anti-COVID-19 bioactivities through reducing almost all the oxidizing moieties of all the target proteins, which logically causes deactivation of the catalytic and noncatalytic biological activities of these important active proteins, meantime, these antioxidant properties aid in quenching the free radicals and fighting the oxidative stress inside the body of the COVID-19 patient [<a href="#CR3">3</a>, <a href="#CR20">20</a>]. The existence of the bioactive antiviral 1,3,4-oxadiazole rings in the backbones of the two molecules is suggested to significantly enhance the anti-COVID-19 actions of both of them [<a href="#CR20">20</a>]. The two compounds have very balanced (ideal) lipophilic/hydrophilic properties (they have log <i>P</i>â€‰<div class="â€‰5,_UNKNOWN" href="#CR23">
23]. The two molecules were docked in all the available SARS-CoV-2 nonstructural and structural protein targets (including functional enzymes and proteins) along with one human structural enzyme, with an emphasis on the most important enzymes like the RdRp.</div></p> <p id="Par5">After deep investigative analysis of the SARS-CoV-2 constructure, researchers have been able to identify all the possible targets of the SARS-CoV-2 in a detailed way [<a href="#CR24">24</a>]. The promising results of the computational molecular docking of both compounds in almost all targets, in comparison with the reference potent drugs, theoretically confirmed their exceptional multitarget inhibitory activity against coronaviral-2 particles (to the best of my knowledge, the two compounds are considered the first potent multitarget anti-COVID-19 compounds). This interesting outcome inspired me to experimentally bioevaluate them. Precisely as the hypothetical predictions, the in vitro bioassay outcomes were also very promising. Based on all the previous facts and results, we can expect that <b>CoViTris2020</b> and <b>ChloViD2020</b> will successfully act as very potent anti-COVID-19 drug candidates through several and distinct mechanisms of action (i.e., through a very effective anti-COVID-19 mode of multiaction). Detailed illustrations of all the discovered target COVID-19 therapy nonstructural proteins (nsps) and structural proteins (sps) present in SARS-CoV-2 and human (up to date) are presented in Figs. <a href="#Fig2">2</a> and <a href="#Fig3">3</a>, respectively. In this research paper, I report the successful repurposing of the two previously synthesized antioxidant 1,3,4-oxadiazole compounds, <b>CoViTris2020</b> and <b>ChloViD2020</b>, as effective and potent anti-COVID-19 agents (as the first potent antidotal multitarget anti-SARS-CoV-2/anti-COVID-19 drugs). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>A detailed illustration of all the discovered target nsps present in SARS-CoV-2 and involved in COVID-19 therapy (up to date)</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>A detailed illustration of all the discovered target sps present in SARS-CoV-2/human and involved in COVID-19 therapy (up to date)</p>   </div> <p /> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="computationalmoleculardockingstudiesofcovitris2020andchlovid2020(predictiveanti-covid-19propertiesevaluation)" title="sec"> <div class="title" tagx="title" title="title">
Computational molecular docking studies of CoViTris2020 and ChloViD2020 (predictive anti-COVID-19 properties evaluation)</div> <p id="Par6">To expectedly assess the anti-COVID-19 activities of the two target antioxidant compounds of the current research, <b>CoViTris2020</b> and <b>ChloViD2020</b>, prior to their actual practical anti-COVID-19 biological evaluation (preliminary in vitro/in vivo assays and subsequent preclinical/clinical trials), molecular docking of the molecules of both compounds in the target SARS-CoV-2 nsps and sps (along with one target human sp, ACE2) has been carried out using the docking engines of the most known and credible international molecular docking software programs (e.g., Discovery Studio, LeDock, GemDock, and GOLD). Different docking software programs were used to confidently and doubtlessly ensure the results and to assess and guarantee their reproducibility. Integrating the expected pharmacophoric features with the interaction energy analysis disclosed functionally pivotal amino acid residues in the binding pockets/cavities of the active and/or allosteric sites of the target SARS-CoV-2 enzymes (mainly) along with in silico predicted common inhibitory binding modes with the highly potent reference repurposed compounds (e.g., remdesivir, ivermectin, and favipiravir). The docking results were very brilliant to encourage me to utilize a new specific web-based server designed in 2020, which is specialized in precise molecular docking of all COVID-19 target nsps/sps and prediction of anti-COVID-19 activities and potencies of tested compounds. For this objective, the newly programmed COVID-19 Docking Server was used [<a href="#CR25">25</a>].</p> <p id="Par7">COVID-19 Docking Server web-based software (AutoDock Vina is used as the main docking engine) is an interactive web server for docking small molecules, peptides, or antibodies against potential protein targets of COVID-19 in order to predict the binding modes between COVID-19 targets and the ligands along with screening and evaluating the anti-COVID-19 activities of these ligands (i.e., the platform provides a free interactive modern tool using a very precise knowledge-based scoring function to evaluate the candidate binding poses for the specific prediction of COVID-19 target-ligand interactions and the following drug discovery for COVID-19). Generally, almost all SARS-CoV-2 nsps (enzymes) and very few SARS-CoV-2/human sps as COVID-19 therapy targets are recommended for small molecule docking. The structures of all the functional or structural protein targets involved in the SARS-CoV-2 replication life cycle were collected by direct downloading from international web databases (e.g., from the Protein Data Bank â€œPDBâ€�) or constructed based on their known homologs of coronaviruses (by using homology modeling module of Maestro 10, website: <a href="http://www.schrodinger.com">www.schrodinger.com</a>), and completely prepared ready for direct docking on this web-based software. The utmost effective and influenced nsps and sps (almost are functional enzymes) among all the detected and recognized proteins involved in COVID-19 therapy (previously presented in Figs. <a href="#Fig2">2</a> and <a href="#Fig3">3</a>) were accurately opted to be targeted and docked. This includes fifteen various nCoV protein targets and one human protein target (the human enzyme ACE2). For docking of only one small molecule, the â€œDockingâ€� mode box should be specifically selected as the computational module (type) for every specific target (this is the selection used in the current case). To get the best accurate results, an average exhaustiveness option of â€œ12â€� was used. The detailed results of these estimations (binding energies in kcal/mol) are shown in Table <a href="#Tab1">1</a>. After docking, the structure of each enzyme/protein-compound complex was further examined and accurately analyzed for characterization by the help of the fully automated comprehensive interactive tool of the famous Proteinâ€"Ligand Interaction Profiler (PLIP) web server (<a href="https://projects.biotec.tu-dresden.de/plip-web">https://projects.biotec.tu-dresden.de/plip-web</a>) [<a href="#CR26">26</a>], and results were marginally tabulated for comparison, explication of the previous docking results, and then placing conclusions (see Table <a href="#Tab2">2</a> as an example). <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Score values of the sixteen computationally predicted pharmacological anti-COVID-19-related activities (against SARS-CoV-2 nsps/sps and human sp ACE2) of the target 1,3,4-oxadiazoles (<b>CoViTris2020</b> and <b>ChloViD2020</b>) and the three reference drugs (remdesivir, ivermectin, and favipiravir), respectively, using COVID-19 Docking Server methodology (the table demonstrates the top docking binding model score value, i.e., the least predicted binding energy value, in kcal/mol for each compound with each target protein)</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2">Classification</th> <th align="left" rowspan="2">SARS-CoV-2/Human target protein</th> <th align="left" colspan="5">Top pose score value for docking of nCoV protein targets<br /><br />Inhibitory binding energies/affinities (kcal/mol)</th> </tr> <tr> <th align="left"><b>CoViTris2020</b></th> <th align="left"><b>ChloViD2020</b></th> <th align="left"><b>Remdesivir</b></th> <th align="left"><b>Ivermectin</b><br /><br />(<b><i>B</i></b><sub><b><i>1a</i></b></sub><b><i> form</i></b>)</th> <th align="left"><b>Favipiravir</b></th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="14"><i>Nsps</i></td> <td align="left">Mpro</td> <td align="left">âˆ’â€‰9.50</td> <td align="left">âˆ’â€‰8.20</td> <td align="left">âˆ’â€‰7.70</td> <td align="left">âˆ’â€‰6.50</td> <td align="left">âˆ’â€‰5.40</td> </tr> <tr> <td align="left">RdRp (RTP site)</td> <td align="left">âˆ’â€‰12.00</td> <td align="left">âˆ’â€‰9.60</td> <td align="left">âˆ’â€‰8.30</td> <td align="left">âˆ’â€‰7.10</td> <td align="left">âˆ’â€‰6.90</td> </tr> <tr> <td align="left">RdRp (RNA site)</td> <td align="left">âˆ’â€‰9.40</td> <td align="left">âˆ’â€‰7.90</td> <td align="left">âˆ’â€‰7.10</td> <td align="left">âˆ’â€‰6.60</td> <td align="left">âˆ’â€‰6.10</td> </tr> <tr> <td align="left">PLpro (dimer)</td> <td align="left">âˆ’â€‰10.60</td> <td align="left">âˆ’â€‰9.30</td> <td align="left">âˆ’â€‰8.10</td> <td align="left">âˆ’â€‰6.00</td> <td align="left">âˆ’â€‰5.40</td> </tr> <tr> <td align="left">Nsp3 (207-379, AMP site)</td> <td align="left">âˆ’â€‰9.70</td> <td align="left">âˆ’â€‰7.70</td> <td align="left">âˆ’â€‰7.10</td> <td align="left">âˆ’â€‰5.90</td> <td align="left">âˆ’â€‰5.40</td> </tr> <tr> <td align="left">Nsp3 (207-379, MES site)</td> <td align="left">âˆ’â€‰9.90</td> <td align="left">âˆ’â€‰9.90</td> <td align="left">âˆ’â€‰8.40</td> <td align="left">âˆ’â€‰6.40</td> <td align="left">âˆ’â€‰5.50</td> </tr> <tr> <td align="left">Helicase (ADP site)</td> <td align="left">âˆ’â€‰8.80</td> <td align="left">âˆ’â€‰8.00</td> <td align="left">âˆ’â€‰7.00</td> <td align="left">âˆ’â€‰5.80</td> <td align="left">âˆ’â€‰5.30</td> </tr> <tr> <td align="left">Helicase (NCB site)</td> <td align="left">âˆ’â€‰9.90</td> <td align="left">âˆ’â€‰8.90</td> <td align="left">âˆ’â€‰7.50</td> <td align="left">âˆ’â€‰6.10</td> <td align="left">âˆ’â€‰5.40</td> </tr> <tr> <td align="left">Nsp14 (ExoN)</td> <td align="left">âˆ’â€‰8.10</td> <td align="left">âˆ’â€‰7.50</td> <td align="left">âˆ’â€‰7.70</td> <td align="left">âˆ’â€‰5.70</td> <td align="left">âˆ’â€‰4.90</td> </tr> <tr> <td align="left">Nsp14 (N7-MTase)</td> <td align="left">âˆ’â€‰11.40</td> <td align="left">âˆ’â€‰9.30</td> <td align="left">âˆ’â€‰9.10</td> <td align="left">âˆ’â€‰7.20</td> <td align="left">âˆ’â€‰6.10</td> </tr> <tr> <td align="left">Nsp15 (endoribonuclease)</td> <td align="left">âˆ’â€‰8.50</td> <td align="left">âˆ’â€‰8.10</td> <td align="left">âˆ’â€‰8.30</td> <td align="left">âˆ’â€‰6.00</td> <td align="left">âˆ’â€‰4.80</td> </tr> <tr> <td align="left">Nsp16 (GTA site)</td> <td align="left">âˆ’â€‰10.10</td> <td align="left">âˆ’â€‰8.70</td> <td align="left">âˆ’â€‰8.30</td> <td align="left">âˆ’â€‰6.80</td> <td align="left">âˆ’â€‰5.60</td> </tr> <tr> <td align="left">Nsp16 (MGP site)</td> <td align="left">âˆ’â€‰9.70</td> <td align="left">âˆ’â€‰7.70</td> <td align="left">âˆ’â€‰7.30</td> <td align="left">âˆ’â€‰6.20</td> <td align="left">âˆ’â€‰5.10</td> </tr> <tr> <td align="left">Nsp16 (SAM site)</td> <td align="left">âˆ’â€‰10.10</td> <td align="left">âˆ’â€‰8.70</td> <td align="left">âˆ’â€‰8.10</td> <td align="left">âˆ’â€‰6.70</td> <td align="left">âˆ’â€‰5.50</td> </tr> <tr> <td align="left" rowspan="2"><i>Sps</i></td> <td align="left">N protein (NCB site)</td> <td align="left">âˆ’â€‰10.00</td> <td align="left">âˆ’â€‰8.90</td> <td align="left">âˆ’â€‰7.40</td> <td align="left">âˆ’â€‰6.50</td> <td align="left">âˆ’â€‰5.20</td> </tr> <tr> <td align="left">Human ACE2</td> <td align="left">âˆ’â€‰10.20</td> <td align="left">âˆ’â€‰9.00</td> <td align="left">âˆ’â€‰7.90</td> <td align="left">âˆ’â€‰6.70</td> <td align="left">âˆ’â€‰5.60</td> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> <p>Summary of the main active amino acid residues of chains A and C â€œnsp12/7â€� (of the SARS-CoV-2 RdRp) interacted with <b>CoViTris2020</b>, <b>ChloViD2020</b>, and remdesivir (active form) molecules, respectively (pivotal catalytic residues of the expected active site are shown in italics)</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2">Compound</th> <th align="left" colspan="3">SARS-CoV-2 RdRp amino acid residues</th> </tr> <tr> <th align="left">Hydrogen bonds<br /><br />(of all types)</th> <th align="left">Hydrophobic interactions</th> <th align="left">Ï€-Cation/Halogen interactions</th> </tr> </thead> <tbody> <tr> <td align="left"><b>CoViTris2020</b></td> <td align="left"><b>Chain A</b>: ARG553, TYR619, LYS621 (2 H bonds), <i>CYS622, ASP623, SER682, THR687 (2 H bonds), ALA688, ASN691</i>, SER759 (2 H bonds), <i>ASP760</i>, SER795, LYS798</td> <td align="left"><b>Chain A</b>: PRO620, <i>ASP623</i>, ARG624, LYS798</td> <td align="left"><b>Chain A</b>: ARG553</td> </tr> <tr> <td align="left"><b>ChloViD2020</b></td> <td align="left"><b>Chain A</b>: <i>ASP623, ASN691</i>, ASP846, LYS849; <b>Chain C</b>: VAL12, SER15, GLN18, GLN19, MET90</td> <td align="left"><b>Chain A</b>: MET87, LYS411, ASN414, LYS417; <b>Chain C</b>: MET90</td> <td align="left"><b>Chain A</b>: LYS417, ASP418</td> </tr> <tr> <td align="left"><b>Remdesivir</b></td> <td align="left"><b>Chain A</b>: ARG555 (2 H bonds), <i>CYS622, ASP623, SER682, THR687, ALA688, ASN691, ASP760</i></td> <td align="left">â€"</td> <td align="left"><b>Chain A</b>: ARG555</td> </tr> </tbody> </table> </div> <p /> </div> <div class="anti-covid-19biologicalactivities(invitroassay)ofcovitris2020andchlovid2020" title="sec"> <div class="title" tagx="title" title="title">
Anti-COVID-19 biological activities (in vitro assay) of CoViTris2020 and ChloViD2020</div> <p id="Par8">This novel and highly reliable anti-COVID-19 in vitro assay is based upon the authentic procedures of Chu and coworkers with very slight modifications [<a href="#CR5">5</a>, <a href="#CR27">27</a>]. The complete procedures were carried out in a specialized biosafety level 3 (BSL-3) laboratory (SARS-CoV-2 is classified as a BSL-3 pathogen by the WHO and FDA) in Hong Kong SAR (China). The assayed SARS-CoV-2 virus, BetaCoV/Hong Kong/VM20001061/2020, was isolated from the fresh nasopharynx aspirate and throat swab of a confirmed fifty-years-aged COVID-19 male patient in Hong Kong using Vero E6 cells (ATCC CRL-1586). Stock virus (10<sup>7.25</sup> TCID<sub>50</sub>/mL) was prepared after three serial passages in Vero E6 cells in infection media (DMEM supplemented with 4.5Â g/L D-glucose, 100Â mg/L sodium pyruvate, 2% FBS, 100,000 U/L Penicillinâ€"Streptomycin, and 25Â mM HEPES). Following the original procedures in the literature, <b>CoViTris2020</b> and <b>ChloViD2020</b> compounds were synthesized (starting from gallic acid), purified (&amp;gt;â€‰97% purity), fully characterized, and put in small dark brown glass containers to be ready for the assay [<a href="#CR20">20</a>]. The pure three reference compounds were purchased from MedChemExpress (remdesivir), Sigma-Aldrich (ivermectin, B<sub>1a</sub> form), and Toyama Chemical â€œFujifilmÂ group, Japanâ€� (favipiravir). The stocks of the five tested compounds were accurately prepared by dissolving each of them in dimethylsulfoxide â€œDMSOâ€� (to get a concentration of 100Â mM of each of <b>CoViTris2020</b>, <b>ChloViD2020</b>, remdesivir, ivermectin, and favipiravir). To evaluate the in vitro anti-SARS-CoV-2 activities of the two target new compounds (<b>CoViTris2020</b> and <b>ChloViD2020</b>) in comparison with those of the standard three reference compounds (mentioned above), Vero E6 cells were pretreated with the five compounds diluted in infection media for 1Â h prior to infection by SARS-CoV-2 virus at MOIâ€‰=â€‰0.02. The five tested compounds were maintained with the virus inoculum during the 2-h incubation period. The inoculum was removed after incubation, and the cells were overlaid with infection media containing the diluted test compounds. After 48Â h incubation at 37Â Â°C, supernatants were immediately collected to quantify viral loads by TCID<sub>50</sub> assay or quantitative real-time RT-PCR (TaqManâ"¢ Fast Virus 1-Step Master Mix) [<a href="#CR5">5</a>, <a href="#CR27">27</a>]. Viral loads in this assay were fitted in logarithm scale (log<sub>10</sub> TCID<sub>50</sub>/mL and log<sub>10</sub> viral RNA copies/mL), not in linear scale, under increasing concentrations of the tested compounds [<a href="#CR5">5</a>, <a href="#CR9">9</a>, <a href="#CR27">27</a>]. Four-parameter logistic regression (GraphPad Prism) was used to fit the doseâ€"response curves and determine the EC<sub>50</sub> of the tested compounds that inhibit SARS-CoV-2 viral replication (CPEIC<sub>100</sub> was also determined for each compound). Cytotoxicity of each of the five tested compounds was evaluated in Vero E6 cells using the CellTiter-GloÂ® Luminescent Cell Viability Assay (Promega) [<a href="#CR5">5</a>, <a href="#CR28">28</a>]. The detailed values resulted from the previous assays (compound concentrations in Î¼M) are shown in Table <a href="#Tab3">3</a>. Final results were represented as the meanâ€‰Â±â€‰SD from the triplicate biological experiments. Statistical analysis was performed using SkanIt 4.0 Research Edition software (ThermoFisher Scientific) and Prism V5 software (GraphPad). All reported data were significant at pâ€‰<div class="â€‰0.05._UNKNOWN"><div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></div></p><p>Anti-COVID-19 (anticoronaviral-2) activities (along with human/mammalian cells toxicities) of <b>CoViTris2020</b> and <b>ChloViD2020</b> (using remdesivir, ivermectin, and favipiravir, respectively, as the reference drugs) against SARS-CoV-2 in Vero E6 cells</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2">Classification</th> <th align="left" rowspan="2">Compound Name</th> <th align="left" rowspan="2">CC<sub>50</sub><sup>a</sup><br /><br />(Î¼M)</th> <th align="left" colspan="3">Inhibition of SARS-CoV-2 in vitro (Î¼M)</th> </tr> <tr> <th align="left">100% CPE Inhibitory concentration (CPEIC<sub>100</sub>)<sup>b</sup></th> <th align="left">50% Reduction in infectious virusÂ (EC<sub>50</sub>)<sup>c</sup></th> <th align="left">50% Reduction in viral RNA copyÂ (EC<sub>50</sub>)<sup>d</sup></th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="2">Target compounds</td> <td align="left"><b>CoViTris2020</b></td> <td align="left">&amp;gt;â€‰100</td> <td char="." align="char">0.99â€‰Â±â€‰0.11</td> <td char="." align="char">0.31â€‰Â±â€‰0.02</td> <td align="left">0.33â€‰Â±â€‰0.02</td> </tr> <tr> <td align="left"><b>ChloViD2020</b></td> <td align="left">&amp;gt;â€‰100</td> <td char="." align="char">1.97â€‰Â±â€‰0.23</td> <td char="." align="char">1.01â€‰Â±â€‰0.07</td> <td align="left">1.23â€‰Â±â€‰0.09</td> </tr> <tr> <td align="left" rowspan="3">Reference compounds</td> <td align="left"><b>Remdesivir</b></td> <td align="left">&amp;gt;â€‰100</td> <td char="." align="char">22.50â€‰Â±â€‰3.02</td> <td char="." align="char">20.17â€‰Â±â€‰1.72</td> <td align="left">23.88â€‰Â±â€‰1.96</td> </tr> <tr> <td align="left"><b>Ivermectin</b></td> <td align="left">&amp;gt;â€‰100</td> <td char="." align="char">83.05â€‰Â±â€‰7.15</td> <td char="." align="char">53.00â€‰Â±â€‰3.99</td> <td align="left">62.44â€‰Â±â€‰5.08</td> </tr> <tr> <td align="left"><b>Favipiravir</b></td> <td align="left">&amp;gt;â€‰100</td> <td char="." align="char">98.82â€‰Â±â€‰8.64</td> <td char="." align="char">94.09â€‰Â±â€‰6.79</td> <td align="left">&amp;gt;â€‰100</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup>CC<sub>50</sub>Â or 50% cytotoxic concentration is the concentration of the tested compound that kills half the cells in an uninfected cell culture. CC<sub>50</sub> was determined with serially diluted compounds in Vero E6 cells at 48Â h postincubation using CellTiter-Glow Luminescent Cell Viability Assay (Promega)</p> <p><sup>b</sup>CPEIC<sub>100</sub> or 100% CPE inhibitory concentration is the lowest concentration of the tested compound that causes 100% inhibition of the cytopathic effects (CPE) of SARS-CoV-2 virus in Vero E6 cells under increasing concentrations of the tested compound at 48Â h postinfection. Compounds were serially twofold or fourfold diluted from 100Â Î¼M concentration</p> <p><sup>c</sup>EC<sub>50</sub>Â or 50% effective concentration is the concentration of the tested compound that is required for 50% reduction in infectious SARS-CoV-2 virus particles in vitro. EC<sub>50</sub> is determined by infectious virus yield in culture supernatant at 48Â h postinfection (log<sub>10</sub>Â TCID<sub>50</sub>/mL)</p> <p><sup>d</sup>EC<sub>50</sub>Â or 50% effective concentration is the concentration of the tested compound that is required for 50% reduction in SARS-CoV-2 viral RNA copies in vitro. EC<sub>50</sub>Â is determined by viral RNA copies number in culture supernatant at 48Â h postinfection (log<sub>10</sub>Â RNA copies/mL)</p> </div>  <p /> </div> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and discussion</div> <div class="computationalmoleculardockingstudiesofcovitris2020andchlovid2020(predictiveanti-covid-19propertiesevaluation)" title="sec"> <div class="title" tagx="title" title="title">
Computational molecular docking studies of CoViTris2020 and ChloViD2020 (predictive anti-COVID-19 properties evaluation)</div> <p id="Par9">The deep understanding of the COVID-19 target-ligand interactions represents an extremely crucial key challenge in drug discovery for COVID-19.Â The computational simulative prediction of the SARS-CoV-2 proteins-inhibiting properties of the newly repurposed target compounds <b>CoViTris2020</b> and <b>ChloViD2020</b> greatly helped me to have a comprehensive overview of the expected anti-COVID-19 activities of the two compounds. This prediction also aided me to have a detailed conception about the two target compounds anti-COVID-19 modes of action along with the compounds degrees of efficacy and potency.</p> <p id="Par10">On close inspection of the top pose score values (present in Table <a href="#Tab1">1</a>) of docking most nsps and sps of SARS-CoV-2 (along with ACE2 of human, which is almost the only human entry point for the virus into the human body until now, see Introduction section and Fig.Â <a href="#Fig3">3</a>) using COVID-19 Docking Server, it is clearly observed that <b>CoViTris2020</b> is distinguishably ranked first in its inhibitory binding affinities and potencies with an excellent range of binding energies of âˆ’â€‰8.10 to âˆ’â€‰12.00Â kcal/mol. The binding affinities of <b>CoViTris2020</b> dramatically exceed those of all the three reference drugs (they are about to be twice as those of favipiravir) and <b>ChloViD2020</b>, as the potent antioxidant <b>CoViTris2020</b> molecule powerfully binds to the respective proteins forming very stable inhibited (deactivated) complexes with relatively amazing binding energies which are the lowest among all (i.e., significantly lower than the binding energies of all the other four compounds in their complexes with the respective nsps and sps). Specifically, <b>CoViTris2020</b> molecule gives its best inhibitory binding affinities with the three nsps RdRp-RNA (RTP site) (âˆ’â€‰12.00Â kcal/mol), nsp14 (N7-MTase) (âˆ’â€‰11.40Â kcal/mol), and PLpro (dimer) (âˆ’â€‰10.60Â kcal/mol), respectively (Fig.Â <a href="#Fig4">4</a>aâ€"d), which are three of the most effective protein targets in hindering and stopping the life cycle of the SARS-CoV-2 through inhibition and deactivation of their active sites [<a href="#CR24">24</a>]. These exceptional results are very promising as they indicate the high possibility of <b>CoViTris2020</b> to be a very potent inhibitor (blocker) of RdRp, nsp14, and/or PLpro. Other proteins affording very encouraging excellent binding affinities with <b>CoViTris2020</b> molecule include ACE2, nsp16 (all sites), N protein (NCB site), nsp3 (both sites), helicase (both sites), Mpro, and RdRp without RNA (RNA site) (Fig.Â <a href="#Fig4">4</a>b), respectively. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Screenshots of COVID-19 Docking Server outputs of the top predicted binding model of docking of <b>CoViTris2020</b> molecule (colored pink) in: <b>a</b> SARS-CoV-2 RdRp-RNA â€œRTP siteâ€� (<i>PDB code</i>: 7BV2; colored with other various colors; Cartoon Style). <b>b</b> SARS-CoV-2 RdRp â€œRNA siteâ€� (<i>PDB code</i>: 7BV2; colored with other various colors; Trace Style). <b>c</b> SARS-CoV-2 nsp14 â€œN7-MTaseâ€� (<i>PDB code</i>: 5C8S, 1J53 â€œfor active site homologyâ€�; colored with other various colors; Cartoon Style). <b>d</b> SARS-CoV-2 PLpro â€œdimerâ€� (<i>PDB code</i>: 6WUU; colored with other various colors; Trace Style)</p>   </div> <p /> <p id="Par11"><b>ChloViD2020</b> comes second, among all the five compounds, in its inhibitory binding energies and affinities which are ranged from âˆ’â€‰7.50 to âˆ’â€‰9.90Â kcal/mol. The binding affinities of <b>ChloViD2020</b> surpass those of all the three reference drugs (except for those of each of exoribonuclease and endoribonuclease with remdesivir, as there is binding energies difference ofâ€‰âˆ’â€‰0.20Â kcal/mol in favor of remdesivir in both cases), as the potent antioxidant <b>ChloViD2020</b> molecule strongly binds to the respective proteins forming very stable inhibited complexes with relatively lower binding energies (i.e., significantly lower than the binding energies of all the three reference molecules in their complexes with the respective nsps and sps). <b>ChloViD2020</b> molecule specifically gives its best inhibitory binding affinities with the four nsps nsp3 (207-379, MES site) (âˆ’â€‰9.90Â kcal/mol), RdRp-RNA (RTP site) (âˆ’â€‰9.60Â kcal/mol), nsp14 (N7-MTase) (âˆ’â€‰9.30Â kcal/mol), and PLpro (dimer) (âˆ’â€‰9.30Â kcal/mol), respectively (Fig.Â <a href="#Fig5">5</a>aâ€"d). Interestingly, <b>ChloViD2020</b> approaches and reaches the lower binding affinity of <b>CoViTris2020</b> in only one enzyme which is N7-MTase (it is also one of the most effective protein targets in stopping the coronaviral-2 life cycle through blocking of its active site [<a href="#CR24">24</a>]) with the same binding energy value of -9.90Â kcal/mol. These promising results indicate the significant possibility of <b>ChloViD2020</b> to be a very potent inhibitor ligand or antagonist of nsp3, RdRp, nsp14, and/or PLpro. Other proteins giving very encouraging binding affinities with <b>ChloViD2020</b> molecule include ACE2, N protein (NCB site), helicase (both sites), nsp16 (all sites), Mpro, Nsp15 (endoribonuclease), and RdRp without RNA (RNA site), respectively. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Screenshots of COVID-19 Docking Server outputs of the top predicted binding model of docking of <b>ChloViD2020</b> molecule (colored pink) in: <b>a</b> SARS-CoV-2 nsp3 â€œ207-379, MES siteâ€� (<i>PDB code</i>: 6W6Y; colored with other various colors; Cartoon Style). <b>b</b> SARS-CoV-2 RdRp-RNA â€œRTP siteâ€� (<i>PDB code</i>: 7BV2; colored with other various colors; Cartoon Style). <b>c</b> SARS-CoV-2 nsp14 â€œN7-MTaseâ€� (<i>PDB code</i>: 5C8S, 1J53 â€œfor active site homologyâ€�; colored with other various colors; Cartoon Style). <b>d</b> SARS-CoV-2 PLpro â€œdimerâ€� (<i>PDB code</i>: 6WUU; colored with other various colors; Trace Style)</p>   </div> <p /> <p id="Par12">The chemical structures of <b>CoViTris2020</b> and <b>ChloViD2020</b> molecules are exceptionally characterized by higher degrees of balanced orientational andÂ conformationalÂ flexibility when compared to those of the other or reference anticoronaviral drugs (e.g., favipiravir, remdesivir, hydroxychloroquine, ivermectin, and arbidol). These extraordinary unique flexibilities of <b>CoViTris2020</b> and <b>ChloViD2020</b> structures are apparently observed in the docking poses in SARS-CoV-2 target proteins (along with human target ACE2) as shown in both Figs. <a href="#Fig6">6</a> and <a href="#Fig7">7</a>, respectively. <b>CoViTris2020</b> molecule (a trisubstituted or three armed bulky derivative of citric acid) has higher degree of flexibility than <b>ChloViD2020</b> molecule (a typical 2,5-disubstituted derivative of 1,3,4-oxadiazole ring) due to, mainly, its larger topological polar surface area (TPSA) and its higher number of atoms. This highly balanced flexibility is generally required for excellent and extreme positioning of the drug molecule to be more superimposable in the active binding pockets and cavities of any enzyme or protein (i.e., required for extreme lock-and-key positioning of the drug molecule after hitting and striking any protein molecule), leading to more adequate and potent inhibition (i.e., antagonism or blocking) of the actions (either catalytic or whatever) done or mediated by the protein. Consequently, the highly balanced flexibilities of <b>CoViTris2020</b> and <b>ChloViD2020</b> molecules greatly increase their abilities to be very potent anti-COVID-19 agents, respectively. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Collective screenshots of COVID-19 Docking Server outputs of the top predicted binding models resulted from the docking of <b>CoViTris2020</b> molecule (colored pink) in different SARS-CoV-2 proteins (colored with other various colors; trace and cartoon styles), showing the extremely balanced high degrees of orientational and conformational flexibility of the molecule during the hitting attempts against all target coronaviral-2 proteins</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <p>Collective screenshots of COVID-19 Docking Server outputs of the top predicted binding models resulted from the docking of <b>ChloViD2020</b> molecule (colored pink) in different SARS-CoV-2 proteins (colored with other various colors; trace and cartoon styles), showing the extremely balanced high degrees of orientational and conformational flexibility of the molecule during the hitting attempts against all target coronaviral-2 proteins</p>   </div> <p /> <p id="Par13">Remdesivir, computationally, comes first among the three reference drugs (and third among all tested compounds in general) in anti-COVID-19 activities with binding energies range ofâ€‰âˆ’â€‰7.00 toâ€‰âˆ’â€‰9.10Â kcal/mol. Remdesivir shows its best binding affinity with the target enzyme nsp14 (N7-MTase), as it binds to this important SARS-CoV-2 enzyme with a very good binding energy ofâ€‰âˆ’â€‰9.10Â kcal/mol. Ivermectin and favipiravir come last, with their best binding affinities observed with the two coronaviral-2 enzymes nsp14 (N7-MTase;â€‰âˆ’â€‰7.20Â kcal/mol) and RdRp (RTP site;â€‰âˆ’â€‰6.90Â kcal/mol), respectively.</p> <p id="Par14">The results of the inhibitory binding affinities of <b>CoViTris2020</b> with the SARS-CoV-2 RdRp (RTP site) are relatively the best among all the COVID-19 targets, thus the next goal was to extensively investigate the specific interactions with the amino acids of the active site(s) of this crucial polymerase in deeper details (using the data files obtained from the same web server). These deep investigations revealed the high degree of similarity of the expected anti-RdRp mode of action of the two ligands <b>CoViTris2020</b> and <b>ChloViD2020</b> as compared to that of remdesivir and its active metabolite (some of the interacting residues of the active pockets are the same and some others are very close) as shown in Fig.Â <a href="#Fig8">8</a>aâ€"c. It was also found that <b>CoViTris2020</b> molecule has an apparent superiority over <b>ChloViD2020</b> and remdesivir (in its active metabolic form) molecules (together with the other two reference molecules) in the strength of inhibitory binding forces with RdRp (e.g., <b>CoViTris2020</b> has at least sixteen hydrogen bonds, more than four hydrophobic interactions, and at least one Ï€-cation interaction; <b>ChloViD2020</b> has at least nine hydrogen bonds, five hydrophobic interactions, and two halogen interactions; while on the other hand, remdesivir active metabolite (the most active drug among the three references) has only less than nine hydrogen bonds, no considerable hydrophobic interactions, and only one Ï€-cation interaction), supporting its expected promising comprehensive anti-SAR-CoV-2 activities. Exceptionally, <b>CoViTris2020</b> is the only inhibitor among all the investigated compounds that strongly interacts with the RdRp complex structure (especially chain A) with that large number of hydrogen bonds (of all types) which exceeds sixteen bonds. According to the previous literature, <b>CoViTris2020</b> can be considered the only compound that strongly binds with the SARS-CoV-2 nsp12 in this extensively effective inhibitory mode, making it an optimal potent SARS-CoV-2 RdRp inhibitor candidate. Furthermore, this potential unique property gives <b>CoViTris2020</b> an extra advantage of having potent blocking nature in its action on the active site of the polymerase. On the other hand, <b>ChloViD2020</b> is the only inhibitor among all the investigated compounds that interacts with both chains A (nsp12) and C (nsp7) of the RdRp complex structure, since it, additionally, creates interaction forces of the hydrogen-bond/hydrophobic types with the polymerase cofactor chain C (mainly with the residues VAL12, SER15, GLN18, GLN19, and MET90). To the best of my knowledge, <b>ChloViD2020</b> can be predictably considered one of the rarest ligands that interacts with and inhibits more than one protein chain of the SARS-CoV-2 RdRp complex (i.e., acts through a dual mode of action). <b>ChloViD2020</b> is also characterized by its unique interaction forces created between its chlorine atom and the residues LYS417 and ASP418 of chain A. Table <a href="#Tab2">2</a> summarizes all the main active amino acid residues involved in the inhibitory interactions of <b>CoViTris2020</b>, <b>ChloViD2020</b>, and remdesivir molecules, respectively. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 8</span></a></div>   <p>The inhibitory binding interactions, of <b>a CoViTris2020</b>; <b>b ChloViD2020</b>; <b>c</b> Remdesivir (active metabolite form), with the active amino acids of the SARS-CoV-2 RdRp (2D and 3D representations, respectively)</p>   </div> <p /> <p id="Par15">In short, the previous computational results of the predicted binding modes of the two inhibitors <b>CoViTris2020</b> and <b>ChloViD2020</b> with the SARS-CoV-2/human proteins extremely comply with and support my suggested hypothetical multitarget mechanism of anti-COVID-19 action of each of the two ligands.</p> </div> <div class="anti-covid-19biologicalactivities(invitroassay)ofcovitris2020andchlovid2020" title="sec"> <div class="title" tagx="title" title="title">
Anti-COVID-19 biological activities (in vitro assay) of CoViTris2020 and ChloViD2020</div> <p id="Par16">The results demonstrated in Table <a href="#Tab3">3</a> obviously and directly revealed the extremely higher and amazing anti-COVID-19 effectiveness and potency of <b>CoViTris2020</b> and <b>ChloViD2020</b> (as two of the most potent anti-SARS-CoV-2 compounds ever) relative to those of each of the reference drugs. All the five tested compounds were found to inhibit SARS-CoV-2 replication in Vero E6 cells with EC<sub>50</sub>Â under 100Â Î¼M. Surprisingly, <b>CoViTris2020</b> (EC<sub>50</sub>â€‰=â€‰0.31Â Î¼M) was found to be about 65, 171, and 303.5 times more potent than remdesivir (EC<sub>50</sub>â€‰=â€‰20.17Â Î¼M), ivermectin (EC<sub>50</sub>â€‰=â€‰53.00Â Î¼M), and favipiravir (EC<sub>50</sub>â€‰=â€‰94.09Â Î¼M), respectively, in the anti-SARS-CoV-2 activity (in vitro). With the same interesting results, <b>ChloViD2020</b> (EC<sub>50</sub>â€‰=â€‰1.01Â Î¼M) was found to be about 20, 52.5, and 93 times more potent than remdesivir, ivermectin, and favipiravir, respectively, in the same assay. According to this assay, CC<sub>50</sub> of <b>CoViTris2020</b> and <b>ChloViD2020</b> is much larger than 100Â Î¼M, thus both compounds are expected to have very high safety margin and clinical selectivity index (SI; SIâ€‰=â€‰CC<sub>50</sub>/EC<sub>50</sub>, SI<sub><b>CoViTris2020</b></sub>â€‰&amp;gt;â€‰322.58 and SI<sub><b>ChloViD2020</b></sub>â€‰&amp;gt;â€‰99.01), while on the other hand, the references remdesivir, ivermectin, and favipiravir are expected to have narrow safety margin and clinical therapeutic index (SI<sub>Remdesivir</sub>â€‰&amp;gt;â€‰4.96, SI<sub>Ivermectin</sub>â€‰&amp;gt;â€‰1.89, and SI<sub>Favipiravir</sub>â€‰&amp;gt;â€‰1.06). <b>CoViTris2020</b> and <b>ChloViD2020</b> are also having interestingly very small values of the concentration that causes 100% inhibition of the SARS-CoV-2 cytopathic effects in vitro (<b>CoViTris2020</b> has the best CPEIC<sub>100</sub> value, among all the tested compounds, of 0.99Â Î¼M, directly followed by <b>ChloViD2020</b> which has CPEIC<sub>100</sub> value of 1.97Â Î¼M) and of the concentration that is required for 50% reduction in the number of SARS-CoV-2 RNA copies in vitro (<b>CoViTris2020</b> has the best EC<sub>50</sub> value, among all the tested compounds, of 0.33Â Î¼M, directly followed by <b>ChloViD2020</b> which has EC<sub>50</sub> value of 1.23Â Î¼M).</p> <p id="Par17">We should put into consideration the possibility that <b>CoViTris2020</b> and <b>ChloViD2020</b> may undergo prior intracellular metabolic transformation into more active forms (e.g., their triphosphate forms) by human cellular enzymes (e.g., kinases and transferases), which may differ among various cell types, thus evaluation of the pharmacological actions of these two target compounds in primary human airway epithelial cells will clearly facilitate the interpretation and clarification of the previous results. The metabolic activation would almost add extra anti-COVID-19 activities to the two drugs through incorporation of human cell-biocompatible and human cell-bioavailable moieties into the chemical structures of both of them, and this would consequently increase the net clinical effectiveness and efficacy of these two potent compounds. Importantly, the three reference drugs (remdesivir, ivermectin, and favipiravir) are currently undergoing extensive and broad clinical trials as anti-SARS-CoV-2/anti-COVID-19 agents worldwide. The very high values of CC<sub>50</sub> of <b>CoViTris2020</b> and <b>ChloViD2020</b> indicate that both compounds would be predictably well tolerated in the human body. The significantly desirable high values of SI (i.e., the extremely minute values of anti-SARS-CoV-2 EC<sub>50</sub> along with the very high values of mammalian cells CC<sub>50</sub>) of <b>CoViTris2020</b> and <b>ChloViD2020</b> reflect that both compounds evidently favor resistant RNA virus over DNA virus and mammalian cells, and this, in turn, expectedly indicates and expresses the selective specificity of these two compounds as anti-COVID-19 drugs (specifically, <b>CoViTris2020</b> can be considered as a unique superpowerful SARS-CoV-2 antidote/killer). <b>CoViTris2020</b> is apparently more potent and more promising than <b>ChloViD2020</b> as SARS-CoV-2 inhibitor. Using a combination formula of <b>CoViTris2020</b> and remdesivir is a possible good experimental choice, as it may have exceptional and striking combinational synergistic anti-COVID-19 action (comprising the desirable advantages of both potent SARS-CoV-2 inhibitors) in further assays (in vivo) and preclinical/clinical trials. Almost all the factual experimental results obtained and concluded, here, in the anti-COVID-19 antiviral biological evaluation are complying with the previous speculative theoretical results suggested and extracted from the anti-SARS-CoV-2 computational molecular pharmacological predictions for the two newly repurposed biologically reevaluated compounds, <b>CoViTris2020</b> and <b>ChloViD2020</b>, and the three reference drugs.</p> </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par18">Thinking outside the box isÂ very important to win our current challenge against the scary COVID-19 pandemic. Potent multiblockade of, mainly, the novel SARS-CoV-2 proteins including enzymes and receptors could be seen as one of the most effective fertile approaches for comprehensive COVID-19 therapy through designing, discovering, and searching for multitarget SARS-CoV-2 inhibitors, thus much efforts in drug discovery in 2020 were focused on trying to successfully repurpose known drugs and compounds in order to effectively inhibit this extremely resistant coronavirus. My specific efforts led to the discovery of two very promising potent multitarget SARS-CoV-2 inhibitors through the successful reevaluation and repurposing of the known antioxidant 1,3,4-oxadiazole compounds, previously synthesized by me, <b>CoViTris2020</b> (1,2,3-tris[5-(3,4,5-trihydroxyphenyl)-1,3,4-oxadiazol-2-yl]propan-2-ol) and <b>ChloViD2020</b> (5-[5-(7-chloro-4-hydroxyquinolin-3-yl)-1,3,4-oxadiazol-2-yl]benzene-1,2,3-triol), which effectively inhibited SARS-CoV-2 life cycle with IC<sub>50</sub> values of 0.31 (according to the used anti-COVID-19 assay, <b>CoViTris2020</b> is the most potent SARS-CoV-2 inhibitor discovered till now) and 1.01Â Î¼M, and amazingly presented about 65-/171-/303.5-fold and 20-/52.5-/93-fold anti-SARS-CoV-2 activities and potencies more than remdesivir/ivermectin/favipiravir, respectively. On the other hand, the discovery of the very potent anti-COVID-19 activities of <b>CoViTris2020</b> and <b>ChloViD2020</b> molecules through the successful biological reevaluation and repurposing opens the door for us to establish the first class of anti-COVID-19 agents of the type â€œpolyphenolic 1,3,4-oxadiazolesâ€� which will specifically comprise a series of 2,5-disubstituted-1,3,4-oxadiazole derivatives (beginning with the first two effective members, <b>CoViTris2020</b> and <b>ChloViD2020</b>). Prior extensive computational molecular studies showed that <b>CoViTris2020</b> and <b>ChloViD2020</b> have the ideal and balanced values of the pharmacokinetic and druglikeness parameters required to be effectively potent anti-COVID-19 drugs inside the human body. Computational molecular modeling analysis of the best inhibitory docking binding modes of <b>CoViTris2020</b> and <b>ChloViD2020</b> molecules with the SARS-CoV-2/human proteins showed that the 3,4,5-trihydroxyphenyl moieties greatly increase the blocking affinities and potencies at the active and/or allosteric sites of the SARS-CoV-2/human enzymes or proteins (binding energies reachâ€‰âˆ’â€‰12.00Â kcal/mol for <b>CoViTris2020</b> andâ€‰âˆ’â€‰9.90Â kcal/mol for <b>ChloViD2020</b>) when compared to those of the three references (which lack this effective antioxidant 3,4,5-trihydroxyphenyl moiety), remdesivir, ivermectin, and favipiravir (binding energies maximally reachâ€‰âˆ’â€‰9.10,â€‰âˆ’â€‰7.20, andâ€‰âˆ’â€‰6.90Â kcal/mol, respectively). Surprisingly, <b>CoViTris2020</b> surpassed <b>ChloViD2020</b> and the three moderately to highly potent reference drugs in the values of almost all compared computational and experimental anti-COVID-19 parameters, scores, and activities. Specifically, <b>CoViTris2020</b> and <b>ChloViD2020</b> give their best inhibitory binding affinities results with the catalytic active site of the SARS-CoV-2 RdRp. If the <b>CoViTris2020</b> compound successfully passes the in vivo bioassays and then the preclinical/clinical trials with highly effective and satisfactorily significant results as anti-COVID-19 agent, a possible combination therapy (e.g., as an oral tablet/capsule, an intravenous/intramuscular parenteral vial/ampoule, or a nasal/oral/ocular prophylactic gel/drops) with a second highly potent old anti-RNA virus drug, such as remdesivir, may be a recommended available choice for ultimate COVID-19 treatment in the near future. In brief, in this new research paper, the antioxidant <b>CoViTris2020</b> and <b>ChloViD2020</b> molecules were successfully reevaluated, repurposed, and reported as very promising hit molecules (they could also be considered as the first extremely potent anticoronaviral-2 polyphenolic 1,3,4-oxadiazole compounds â€œCoronavirus-2 Killersâ€�) with general multitarget and very potent successful inhibition against SARS-CoV-2 enzymes (mainly), and consequently, both compounds are two of the first known promising under-investigation candidate drugs for the effective and complete treatment of COVID-19.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>I gratefully thank and deeply acknowledge anyone who gave a hand to make this new discovery and work coming out to light.</p> </div> <div class="title" tagx="title" title="title">
Compliance with ethical standards</div> <div class="title" tagx="title" title="title">
Conflict of interest</div> <p id="Par19">I hereby declare that I totally have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this new research paper.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azhar</span><span class="given-names" tagx="given-names" title="given-names">EI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madani</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ntoumi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kock</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dar</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ippolito</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mchugh</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Memish</span><span class="given-names" tagx="given-names" title="given-names">ZA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zumla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petersen</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">The continuing 2019-nCoV epidemic threat of novel coronaviruses to global healthâ€"the latest 2019 novel coronavirus outbreak in Wuhan, China</span><span class="source" tagx="source" title="source">Int J Infect Dis</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">264</span><span class="lpage" tagx="lpage" title="lpage">266</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijid.2020.01.009">10.1016/j.ijid.2020.01.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31953166">31953166</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">You</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">ZJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span></span><span class="mixed-article-title" title="mixed-article-title">The epidemic of 2019-novel-coronavirus (2019-nCoV) pneumonia and insights for emerging infectious diseases in the future</span><span class="source" tagx="source" title="source">Microbes Infect</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">80</span><span class="lpage" tagx="lpage" title="lpage">85</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.micinf.2020.02.002">10.1016/j.micinf.2020.02.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32087334">32087334</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Jiang S, Du L, Shi Z (2020) An emerging coronavirus causing pneumonia outbreak in Wuhan, China: calling for developing therapeutic and prophylactic strategies. Emerg Microbes Infect 9:275â€"277. 10.1080/22221751.2020.1723441. Erratum: Jiang S (2020) Correction. Emerg Microbes Infect 9:539. 10.1080/22221751.2020.1737363</span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Discovering drugs to treat coronavirus disease 2019 (COVID-19)</span><span class="source" tagx="source" title="source">Drug Discov Ther</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">58</span><span class="lpage" tagx="lpage" title="lpage">60</span><span class="pub-id"><a href="https://dx.doi.org/10.5582/ddt.2020.01012">10.5582/ddt.2020.01012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32147628">32147628</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choy</span><span class="given-names" tagx="given-names" title="given-names">KT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">AY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaewpreedee</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sia</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">KPY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">DKW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">MCW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheung</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peiris</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yen</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Remdesivir, lopinavir, emetine, and homoharringtonine inhibit SARS-CoV-2 replication in vitro</span><span class="source" tagx="source" title="source">Antiviral Res</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">178</span><span class="fpage" tagx="fpage" title="fpage">104786</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2020.104786">10.1016/j.antiviral.2020.104786</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32251767">32251767</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Anti-HCV, nucleotide inhibitors, repurposing against COVID-19</span><span class="source" tagx="source" title="source">Life Sci</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">248</span><span class="fpage" tagx="fpage" title="fpage">117477</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.lfs.2020.117477">10.1016/j.lfs.2020.117477</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32119961">32119961</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Elfiky AA (2020) Ribavirin, remdesivir, sofosbuvir, galidesivir, and tenofovir against SARS-CoV-2 RNA dependent RNA polymerase (RdRp): a molecular docking study. Life Sci 253:117592. 10.1016/j.lfs.2020.117592. Erratum: Elfiky AA (2020) Corrigendum. Life Sci 258:118350. 10.1016/j.lfs.2020.118350</span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shannon</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">NT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selisko</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eydoux</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alvarez</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guillemot</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decroly</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peersen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferron</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Canard</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Remdesivir and SARS-CoV-2: structural requirements at both nsp12 RdRp and nsp14 exonuclease active-sites</span><span class="source" tagx="source" title="source">Antiviral Res</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">178</span><span class="fpage" tagx="fpage" title="fpage">104793</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2020.104793">10.1016/j.antiviral.2020.104793</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32283108">32283108</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</span><span class="source" tagx="source" title="source">Cell Res</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">269</span><span class="lpage" tagx="lpage" title="lpage">271</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41422-020-0282-0">10.1038/s41422-020-0282-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32020029">32020029</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">HW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">368</span><span class="fpage" tagx="fpage" title="fpage">1499</span><span class="lpage" tagx="lpage" title="lpage">1504</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.abc1560">10.1126/science.abc1560</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32358203">32358203</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caly</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Druce</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Catton</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jans</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagstaff</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span></span><span class="mixed-article-title" title="mixed-article-title">The FDA-approved drug ivermectin inhibits the replication of SARS-CoV-2 in vitro</span><span class="source" tagx="source" title="source">Antiviral Res</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">178</span><span class="fpage" tagx="fpage" title="fpage">104787</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2020.104787">10.1016/j.antiviral.2020.104787</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32251768">32251768</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Experimental treatment with favipiravir for COVID-19: an open-label control study</span><span class="source" tagx="source" title="source">Engineering</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">1192</span><span class="lpage" tagx="lpage" title="lpage">1198</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.eng.2020.03.007">10.1016/j.eng.2020.03.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32346491">32346491</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">YX</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">XP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Favipiravir: pharmacokinetics and concerns about clinical trials for 2019-nCoV infection</span><span class="source" tagx="source" title="source">Clin Pharmacol Ther</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">242</span><span class="lpage" tagx="lpage" title="lpage">247</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cpt.1844">10.1002/cpt.1844</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32246834">32246834</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shiraki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daikoku</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Favipiravir, an anti-influenza drug against life-threatening RNA virus infections</span><span class="source" tagx="source" title="source">Pharmacol Ther</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">209</span><span class="fpage" tagx="fpage" title="fpage">107512</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pharmthera.2020.107512">10.1016/j.pharmthera.2020.107512</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32097670">32097670</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Derwand</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scholz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Does zinc supplementation enhance the clinical efficacy of chloroquine/hydroxychloroquine to win today's battle against COVID-19?</span><span class="source" tagx="source" title="source">Med Hypotheses</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">142</span><span class="fpage" tagx="fpage" title="fpage">109815</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mehy.2020.109815">10.1016/j.mehy.2020.109815</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32408070">32408070</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hydroxychloroquine, a less toxic derivative of chloroquine, is effective in inhibiting SARS-CoV-2 infection in vitro</span><span class="source" tagx="source" title="source">Cell Discov</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">16</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41421-020-0156-0">10.1038/s41421-020-0156-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32194981">32194981</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">The anti-influenza virus drug, arbidol is an efficient inhibitor of SARS-CoV-2 in vitro</span><span class="source" tagx="source" title="source">Cell Discov</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">28</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41421-020-0169-8">10.1038/s41421-020-0169-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32373347">32373347</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdul Amin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jha</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fight against novel coronavirus: a perspective of medicinal chemists</span><span class="source" tagx="source" title="source">Eur J Med Chem</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">201</span><span class="fpage" tagx="fpage" title="fpage">112559</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ejmech.2020.112559">10.1016/j.ejmech.2020.112559</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32563814">32563814</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Targeting the dimerization of the main protease of coronaviruses: a potential broad-spectrum therapeutic strategy</span><span class="source" tagx="source" title="source">ACS Comb Sci</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">297</span><span class="lpage" tagx="lpage" title="lpage">305</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acscombsci.0c00058">10.1021/acscombsci.0c00058</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32402186">32402186</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabie</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tantawy</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Badr</span><span class="given-names" tagx="given-names" title="given-names">SMI</span></span></span><span class="mixed-article-title" title="mixed-article-title">Design, synthesis, and biological evaluation of novel 5-substituted-2-(3,4,5-trihydroxyphenyl)-1,3,4-oxadiazoles as potent antioxidants</span><span class="source" tagx="source" title="source">Am J Org Chem</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">54</span><span class="lpage" tagx="lpage" title="lpage">80</span><span class="pub-id"><a href="https://dx.doi.org/10.5923/j.ajoc.20160602.02">10.5923/j.ajoc.20160602.02</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishida</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Review on the role of Zn<sup>2+</sup> ions in viral pathogenesis and the effect of Zn<sup>2+</sup> ions for host cell-virus growth inhibition</span><span class="source" tagx="source" title="source">Am J Biomed Sci Res</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">28</span><span class="lpage" tagx="lpage" title="lpage">37</span><span class="pub-id"><a href="https://dx.doi.org/10.34297/AJBSR.2019.02.000566">10.34297/AJBSR.2019.02.000566</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">te Velthuis</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van den Worm</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sims</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snijder</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Hemert</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Zn<sup>2+</sup> inhibits coronavirus and arterivirus RNA polymerase activity in vitro and zinc ionophores block the replication of these viruses in cell culture</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">e1001176</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1001176">10.1371/journal.ppat.1001176</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21079686">21079686</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipinski</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lombardo</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dominy</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feeney</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span><span class="source" tagx="source" title="source">Adv Drug Deliv Rev</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="lpage" tagx="lpage" title="lpage">25</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0169-409X(96)00423-1">10.1016/S0169-409X(96)00423-1</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of therapeutic targets for SARS-CoV-2 and discovery of potential drugs by computational methods</span><span class="source" tagx="source" title="source">Acta Pharm Sin B</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">766</span><span class="lpage" tagx="lpage" title="lpage">788</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.apsb.2020.02.008">10.1016/j.apsb.2020.02.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32292689">32292689</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">COVID-19 Docking Server Web-based Software, COVID-19 Docking Server (Homepage on the Internet), Shan Change Lab., Institute of Bioinformatics and Medical Engineering, School of Electrical and Information Engineering, Jiangsu University of Technology, Changzhou 213001, China, 2020; Available from <i>COVID-19 Docking Server on the Web</i> (homepage: <a href="http://ncov.schanglab.org.cn">http://ncov.schanglab.org.cn</a>); Results obtained through using interactive docking tool in COVID-19 Docking Server (CopyrightÂ© 2018â€"2023, Shan Chang; Version 2020) on this website (accessed and cited in 2020, 11â€"22 June), and references cited therein.</span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salentin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schreiber</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haupt</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adasme</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schroeder</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">PLIP: fully automated protein-ligand interaction profiler</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">W443</span><span class="lpage" tagx="lpage" title="lpage">W447</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkv315">10.1093/nar/gkv315</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25873628">25873628</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">DKW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">SMS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">KPY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishnan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">DYM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">CKC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peiris</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">LLM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular diagnosis of a novel coronavirus (2019-nCoV) causing an outbreak of pneumonia</span><span class="source" tagx="source" title="source">Clin Chem</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">549</span><span class="lpage" tagx="lpage" title="lpage">555</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/clinchem/hvaa029">10.1093/clinchem/hvaa029</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32031583">32031583</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kusov</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nian</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Brunn</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leyssen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanko</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neyts</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Wilde</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snijder</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilgenfeld</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Î±-Ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication: structure-based design, synthesis, and activity assessment</span><span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">4562</span><span class="lpage" tagx="lpage" title="lpage">4578</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jmedchem.9b01828">10.1021/acs.jmedchem.9b01828</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32045235">32045235</a></span></span></li> </ul> </div> </div>  </body></html>