<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7801432">7801432</a></span><span class="publisher-id" title="publisher-id">79320</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-020-79320-z">10.1038/s41598-020-79320-z</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Bioactive diterpenoids impact the composition of the root-associated microbiome in maize (<i>Zea mays</i>)</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-8891-4274</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">Katherine M.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">kmmurphy@ucdavis.edu</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">Joseph</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Louie</span><span class="given-names" tagx="given-names" title="given-names">Katherine B.</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowen</span><span class="given-names" tagx="given-names" title="given-names">Benjamin P.</span></span><a href="#Aff3">3</a><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sundaresan</span><span class="given-names" tagx="given-names" title="given-names">Venkatesan</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Northen</span><span class="given-names" tagx="given-names" title="given-names">Trent R.</span></span><a href="#Aff3">3</a><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zerbe</span><span class="given-names" tagx="given-names" title="given-names">Philipp</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.27860.3b</span><span class="institution-id" title="institution-id">0000 0004 1936 9684</span><span class="institution" title="institution">Department of Plant Biology, </span><span class="institution" title="institution">University of California-Davis, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.89336.37</span><span class="institution-id" title="institution-id">0000 0004 1936 9924</span><span class="institution" title="institution">Integrative Biology, </span><span class="institution" title="institution">University of Texas, Austin, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.184769.5</span><span class="institution-id" title="institution-id">0000 0001 2231 4551</span><span class="institution" title="institution">Environmental Genomics and Systems Biology, </span><span class="institution" title="institution">Lawrence Berkeley National Laboratory, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.184769.5</span><span class="institution-id" title="institution-id">0000 0001 2231 4551</span><span class="institution" title="institution">Joint Genome Institute, </span><span class="institution" title="institution">Lawrence Berkeley National Laboratory, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-1-1</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">333</span><span class="history" title="history"><span class="received" title="received">received: 2020-5-10</span><span class="accepted" title="accepted">accepted: 2020-11-23</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Plants deploy both primary and species-specific, specialized metabolites to communicate with other organisms and adapt to environmental challenges, including interactions with soil-dwelling microbial communities. However, the role of specialized metabolites in modulating plant-microbiome interactions often remains elusive. In this study, we report that maize (<i>Zea mays</i>) diterpenoid metabolites with known antifungal bioactivities also influence rhizosphere bacterial communities. Metabolite profiling showed that dolabralexins, antibiotic diterpenoids that are highly abundant in roots of some maize varieties, can be exuded from the roots. Comparative 16S rRNA gene sequencing determined the bacterial community composition of the maize mutant <i>Zman2</i> (<i>anther ear 2</i>), which is deficient in dolabralexins and closely related bioactive kauralexin diterpenoids. The <i>Zman2</i> rhizosphere microbiome differed significantly from the wild-type sibling with the most significant changes observed for Alphaproteobacteria of the order Sphingomonadales. Metabolomics analyses support that these differences are attributed to the diterpenoid deficiency of the <i>Zman2</i> mutant, rather than other large-scale metabolome alterations. Together, these findings support physiological functions of maize diterpenoids beyond known chemical defenses, including the assembly of the rhizosphere microbiome.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Secondary metabolism</div> <div class="kwd" title="kwd">
Microbiome</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000001</span><span class="institution" title="institution">National Science Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Graduate Research Fellowship Program</span><span class="award-id" tagx="award-id" title="award-id">1758976</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">Katherine M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zerbe</span><span class="given-names" tagx="given-names" title="given-names">Philipp</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">UC Davis Innovation Institute for Food and Health (IIFH)</span></div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">USDA NIFA</span></div><span class="award-id" tagx="award-id" title="award-id">2019-67011-29544</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">Katherine M.</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">DOE Joint Genome Institute</span></div><span class="award-id" tagx="award-id" title="award-id">DE-AC02-05CH11231</span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100006235</span><span class="institution" title="institution">Lawrence Berkeley National Laboratory</span></div> </div><span class="award-id" tagx="award-id" title="award-id">DE-AC02-05CH11231</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Northen</span><span class="given-names" tagx="given-names" title="given-names">Trent R.</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Extensive research in recent years has demonstrated the composition and importance of rhizosphere microbial communities to plant health and fitness<sup><a href="#CR1">1</a>â€"<a href="#CR3">3</a></sup>. The cooperative partnership of microbes and plants has been attributed largely to an exudation of photosynthate sugars from the plant in exchange for nutrient supply and protection against biotic and abiotic stress, ultimately contributing to increased plant vigor and yield<sup><a href="#CR4">4</a>,<a href="#CR5">5</a></sup>. Despite extensive characterization of species- and tissue-specific microbial communities and how these vary in their ecological and genetic contexts, our understanding of the mechanisms by which plants recruit and maintain root-associated microbial communities is still limited<sup><a href="#CR5">5</a>â€"<a href="#CR11">11</a></sup>. </p> <p id="Par3">As the most economically important crop in the United States, maize (<i>Zea mays</i>) has been the subject of long-standing research to improve crop yield and stress resilience traits<sup><a href="#CR12">12</a></sup>. The â€œcoreâ€� microbiome of maize has provided insights into the presence of specific phyla in and near maize roots<sup><a href="#CR6">6</a>,<a href="#CR13">13</a>,<a href="#CR14">14</a></sup>. A study of 27 maize inbred lines across developmental stages and geographical locations demonstrated maize genotype as a replicable factor in defining the microbiome, with significant variation due to the different genetic backgrounds among inbred lines. Five â€œcoreâ€� Operational Taxonomic Units (OTUs), all in the phyla Proteobacteria, were found to be present in all samples<sup><a href="#CR14">14</a></sup>. Drawing on these detailed insights into the maize microbiome, further research has ventured into determining the chemical signaling factors that determine microbiome composition and variation across genotypes. Specialized metabolites mediate various plant interactions with the environment and other organisms, and several major bioactive metabolite groups have been identified in maize, including diterpenoids, sesquiterpenoids, oxylipins, and benzoxazinoids<sup><a href="#CR15">15</a>â€"<a href="#CR20">20</a></sup>. Given their distinct structures, bioactivities and abundance across different maize tissues and developmental stages, maize specialized metabolites can be hypothesized to play a critical role in the plant communication with root-borne microbes and thus help determine the root microbiome. Recently, benzoxazinoids were shown to influence the maize response to stress via altered rhizosphere microbial communities mediated by 6-methoxy-benzoxazolin-2-one (MBOA)<sup><a href="#CR21">21</a></sup>. Additional benzoxazinoid mutant studies showed changes to fungal and bacterial communities as a result of benzoxazinoid deficiency<sup><a href="#CR22">22</a></sup>. Furthermore, benzoxazinoid-deficient mutants were shown to feature substantially altered root metabolomes, suggesting alterations in the microbiome may not be solely attributed to a lack of specific benzoxazinoids, but rather global changes in root metabolites in mutant plants<sup><a href="#CR23">23</a></sup>. Across different studies, there further is a large degree of variation in the observed differences in alpha and beta diversity in benzoxazinoid mutants, largely dependent on developmental stage, environmental conditions, and mutational and genetic background<sup><a href="#CR21">21</a>â€"<a href="#CR23">23</a></sup>.</p> <p id="Par5">The diverse group of terpenoid metabolites also has shown to be critical in mediating above- and belowâ€�ground interactions between plants and other organisms, including microbes<sup><a href="#CR24">24</a></sup>. Kauralexins and dolabralexins are two major diterpenoid groups in maize that have demonstrated or predicted roles in biotic and abiotic stress responses<sup><a href="#CR16">16</a>â€"<a href="#CR18">18</a>,<a href="#CR25">25</a></sup>. Kauralexins show stress-elicited accumulation in several tissues, including stems and scutellum, and mediate quantitative defenses against fungal pathogens such as species of <i>Fusarium</i>, <i>Aspergillus</i> and <i>Cochliobolus</i>, as well as insect pests including the European corn borer (<i>Ostrinia nubilalis</i>)<sup><a href="#CR17">17</a>,<a href="#CR26">26</a>â€"<a href="#CR28">28</a></sup>. The more recently discovered group of dolabralexins shows pathogen-inducible accumulation predominantly in roots and, like kauralexins, have strong growth-inhibitory activity against <i>Fusarium</i> pathogens<sup><a href="#CR18">18</a></sup>. In addition to their defensive potential against biotic stressors, both kauralexin and dolabralexin production was shown to increase in response to abiotic stress, such as drought or oxidative stress<sup><a href="#CR16">16</a>,<a href="#CR18">18</a></sup>. Kauralexins and dolabralexins derive from a common precursor, <i>ent</i>-copalyl pyrophosphate (<i>ent</i>-CPP), which is also shared with the gibberellin (GA) biosynthetic pathway critical for plant growth (Fig.Â <a href="#Fig1">1</a>)<sup><a href="#CR17">17</a>,<a href="#CR29">29</a></sup>. Two catalytically redundant diterpene synthase (diTPS) enzymes, ANTHER EAR 1 (ZmAn1) and ANTHER EAR 2 (ZmAn2) control <i>ent</i>-CPP formation in maize<sup><a href="#CR29">29</a>,<a href="#CR30">30</a></sup>. Genetic studies revealed that ZmAN1 is critical for GA biosynthesis, whereas ZmAN2 feeds <i>ent</i>-CPP into kauralexin and dolabralexin biosynthesis, thus enabling a pathway partition separating precursor flux toward primary and secondary (i.e. specialized) diterpenoid pathways<sup><a href="#CR29">29</a>â€"<a href="#CR31">31</a></sup> (Fig.Â <a href="#Fig1">1</a>). This pathway separation is supported by the phenotype of the <i>Zman2</i> mutant, which features a loss of function in the <i>Zman2</i> gene through a stable Ds insertion from the Activator (Ac) and Dissociation (Ds) system<sup><a href="#CR16">16</a>,<a href="#CR29">29</a></sup>. <i>Zman2</i> has kauralexin and dolabralexin deficiency but normal GA levels<sup><a href="#CR16">16</a></sup>. <i>Zman2</i> has a normal growth phenotype, but is more susceptible to biotic and abiotic stress than its wild type (WT) sibling, which is consistent with the protective bioactivity of kauralexins and dolabralexins<sup><a href="#CR16">16</a></sup> and suggests a possible role of these metabolites in broader plantâ€"microbe interactions, including root microbiota. To test this hypothesis, we combined microbial 16S rRNA sequencing and metabolite profiling of the diterpenoid-deficient <i>Zman2</i> mutant compared to its WT sibling to investigate the role of diterpenoids in shaping the root microbial communities.</p> <p id="Par4"> </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Schematic overview of key enzymes involved in diterpenoid biosynthesis in maize. Abbreviations: GGPP, (<i>E</i>,<i>E</i>,<i>E</i>)-geranyl geranyl diphosphate; <i>ent</i>-CPP, copalyl diphosphate; An, anther ear; KSL, kaurene synthase-like. Orange arrows represent pathways en route to specialized, defensive metabolites. Green pathways represent gibberellin hormone biosynthesis.</p>   </div> <p /> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="dolabralexinsaresecretedfromofmaizeroots" title="sec"> <div class="title" tagx="title" title="title">
Dolabralexins are secreted from of maize roots</div> <p id="Par6">To examine a possible role of maize diterpenoids in plant-microbiome interactions, we utilized the <i>Zman2</i> mutant genotype in comparison to its isogenic WT sibling<sup><a href="#CR16">16</a>,<a href="#CR29">29</a></sup>. Previous studies showed that 30-day-old WT maize plants significantly accumulate dolabralexins, and to a lesser extent kauralexins, in roots<sup><a href="#CR16">16</a></sup>, whereas the <i>Zman2</i> mutant genotype is almost completely devoid of these diterpenoids<sup><a href="#CR16">16</a>â€"<a href="#CR18">18</a></sup>. Despite the deficiency of both kauralexins and dolabralexins in <i>Zman2</i> root tissue, mutant plants did not show an apparent phenotype under well-watered conditions (Fig.Â <a href="#Fig2">2</a>A), consistent with prior reports describing largely unaltered root and shoot weight, developmental features, and GA and zealexin levels in the <i>Zman2</i> mutant<sup><a href="#CR16">16</a></sup>. <i>Zman2</i> plants were used as a control for analyzing root metabolite exudation due to the deficiency of diterpenoids, and subsequent analysis of the microbiome and metabolome. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Phenotype and metabolite abundance in maize <i>Zman2</i> mutants and WT plants. (<b>A</b>) Representative images of <i>Zman2</i> mutant plants and the corresponding WT sibling used in this study. (<b>B</b>) Average intensity of metabolites in maize root exudate samples using positive mode LCâ€"MS/MS analysis. Peak area given for epoxydolabrene [Mâ€‰+â€‰H], trihydroxydolabrene [Mâ€‰+â€‰H-H2O], and benzoxazinoids, with the last based on detection of metabolites containing BOA (1,3-benzoxazol-2-one (BOA) [Mâ€‰+â€‰H]). * represents significant difference between two sample types, pâ€‰â‰¤â€‰0.05. Two-tailed t-tests were used for benzoxazinoids; one-tailed t-tests were used for dolabralexins, since they are predicted to be enriched in WT and deficient in <i>Zman2.</i> Error bars represent standard error. nâ€‰=â€‰5 (WT, <i>Zman2</i>); Control represents one extraction of nutrient water, and demonstrates LCâ€"MS/MS background.</p>   </div> <p /> <p id="Par7">To determine a possible ability of maize diterpenoids to affect the rhizosphere microbiome, we first tested if diterpenoids can be exuded from maize roots. For this purpose, 38-day-old <i>Zman2</i> and WT maize plants were grown on soil, then gently cleaned and suspended for 48Â h in nutrient water. After removing the plants, metabolites were extracted from the nutrient water using an equal volume of ethyl acetate and analyzed by LCâ€"MS/MS against authentic metabolite standards. As a positive control, the benzoxazinoid 1,3-benzoxazol-2-one (BOA), known to be secreted from maize roots, was measured and used as a standard to detect BOA and other benzoxazinoids metabolites with similar mass spectra. Benzoxazinoids were found to be present in both <i>Zman2</i> and WT plant exudates, while absent in nutrient water without plants, as expected (Fig.Â <a href="#Fig2">2</a>B). Of the known dolabralexins for which standards were availableâ€"epoxydolabrene and trihydroxydolabreneâ€"only trace amounts were detected in nutrient water after incubation of the <i>Zman2</i> mutant (Fig.Â <a href="#Fig2">2</a>B), as expected based on the known mutant phenotype<sup><a href="#CR18">18</a></sup>. Trihydroxydolabrene and epoxydolabrene were both significantly enriched in the WT root exudate samples than <i>Zman2</i> or nutrient water without plants (Fig.Â <a href="#Fig2">2</a>B). Epoxydolabranol, the third dolabralexin for which a standard was available, was not detected in mutant nor WT plant exudates.</p> </div> <div class="maizerootmicrobialcommunitiesaredistinctbycompartment" title="sec"> <div class="title" tagx="title" title="title">
Maize root microbial communities are distinct by compartment</div> <p id="Par8">The <i>Zman2</i> mutant genotype and its corresponding WT sibling serve as a tool to investigate the effect of diterpenoids, or the lack thereof, on the maize root microbial community<sup><a href="#CR16">16</a>,<a href="#CR18">18</a></sup>. Based on the growth conditions of previous research on the mutant genotype, showing an enrichment of dolabralexins and to a lesser extent kauralexins, we used one-month-old <i>Zman2</i> and isogenic WT sibling plants to comparatively examine the impact of diterpenoid-deficiency on the maize root microbiome. Representative plant images are shown in Fig.Â <a href="#Fig2">2</a>A.</p> <p id="Par9">Microbiomes of the rhizosphere (1â€"2Â mm of soil outside the root) and endosphere (inside the root), the latter representing root samples after removal of rhizosphere and rhizoplane microbes through washing and sonication of the roots, were analyzed. Bulk soil without plants was used as a control to examine background soil microbial communities. The 16S rRNA gene (V4 region) was sequenced using Illumina MiSeq and sequences were clustered into operational taxonomic units (OTUs) using the QIIME pipeline and the Greengenes database<sup><a href="#CR32">32</a></sup>. After filtering to remove mitochondrial and chloroplast OTUs, 4,259 distinct OTUs remained. Following standard protocols for analyzing data with a binomial distribution, such as the MiSeq data generated in this study, OTU counts were normalized by relative abundance, in which the counts were taken as a percentage of the total number of OTU counts for a sample<sup><a href="#CR33">33</a>â€"<a href="#CR35">35</a></sup>. This method was used rather than rarefaction methods, which are at risk for discarding so as not to discard low abundance OTUs<sup><a href="#CR36">36</a></sup>. The DESeq2 package in R, which is used to analyze data with a binomial distribution such as the data generated in this study, was used to determine which OTUs were enriched or depleted in the WT and <i>Zman2</i> samples<sup><a href="#CR37">37</a></sup>.</p> <p id="Par10">Consistent with previous research in maize and other plant species<sup><a href="#CR7">7</a>,<a href="#CR38">38</a></sup>, the microbial communities of the two plant compartments and the bulk soil were all statistically distinct. The alpha diversity, as measured by the Shannon index, showed the greatest diversity of microbes in bulk soil, with reduced diversity in the rhizosphere and further reduction in the endosphere (Fig.Â <a href="#Fig3">3</a>, statistics reported in Supplemental Table 2). Sequencing depth was used as a covariate to account for variance due to experimental variables and not any possible variance due to sequencing depth. A permutational multivariate analysis of variance (PERMANOVA) was used to measure the diversity between samples (beta diversity) and showed that, when accounting for all factors, compartment accounts for 31% of the variation between samples (pâ€‰<div class="â€‰0.001)_UNKNOWN" href="#Fig4">
4A). </div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Alpha diversity of each sample type, as measured by the Shannonâ€™s H index. Letters represent significantly different measurements, pâ€‰â‰¤â€‰0.05. All statistics are available in Supplemental Table 2. nâ€‰=â€‰5 (bulk soil) or nâ€‰=â€‰6 (all plant samples).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Principal Coordinate Analysis (PCoA) using Bray distances for (<b>A</b>) all samples, (<b>B</b>) endosphere only, and (<b>C</b>) the rhizosphere only. Each point represents an individual plant or soil sample. Percentage in the axis labels represents the eigenvalue, or percent of variation explained by that axis. nâ€‰=â€‰5 (bulk soil) or nâ€‰=â€‰6 (all plant samples).</p>   </div> <p /> <p id="Par11">A total of 547 OTUs, in 19 phyla (out of 34 total phyla represented by all data points), were enriched in the rhizosphere as compared to the endosphere, whereas 63 OTUs in 8 phyla were enriched in the endosphere as compared to the rhizosphere, as determined using the DESeq2 package in R<sup><a href="#CR37">37</a></sup>. Among the 10 most abundant phyla plotted for each sample type, some phyla were found to be enriched in both compartments, whereas the rhizosphere was predominantly enriched for OTUs in the phyla Actinobacteria, Acidobacteria, and Alphaproteobacteria (Fig.Â <a href="#Fig5">5</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Distribution of the ten most abundant phyla for each sample type. Bars represent the relative abundance of all OTUs within each of the top 10 most abundant phyla. nâ€‰=â€‰5 (bulk soil) or nâ€‰=â€‰6 (all plant samples).</p>   </div> <p /> <p id="Par12">The endosphere did not demonstrate significant differences in beta diversity or alpha diversity attributed to genotype, as determined by PERMANOVA and PCoA or Shannon index, respectively (Fig.Â <a href="#Fig4">4</a>B). No individual OTUs were significantly enriched or depleted in regards to genotype in the endosphere as analyzed by generalized linear models using the DESeq2 package in R<sup><a href="#CR37">37</a></sup>. Because there was no apparent difference, all further analysis focuses on the rhizosphere compartment only, and the bulk soil and endosphere samples were omitted from further analyses. </p> </div> <div class="thezman2mutantfeaturesadistinctmicrobialcommunitycomposition" title="sec"> <div class="title" tagx="title" title="title">
The <i>Zman2</i> mutant features a distinct microbial community composition</div> <p id="Par13">Next, the impact of genotype on the rhizosphere microbiome composition was assessed. Significant differences in the microbiome composition were observed between WT and <i>Zman2</i> plants, with genotype accounting for 10.7% of the variation in the rhizosphere (pâ€‰<div class="â€‰0.05)_UNKNOWN">
Zman2 plants harbor a more diverse microbiome as determined by a greater alpha diversity compared to the WT sibling in the rhizosphere (Fig.Â <a href="#Fig3">3</a>, Supplemental Table 2). Six OTUs were more abundant in WT plants, whereas none were enriched in <i>Zman2</i> samples in the rhizosphere (Supplemental Fig. 1; OTU abundances by sample type plotted in Supplemental Fig. 2). Of the six OTUs, all were assigned to Alphaproteobacteria belonging to the order Sphingomonadales, three of which were assigned to the genus Sphingobium, whereas the remaining OTUs were unclassified at the genus level.</div></p> </div> <div class="wildtypeandmutantplantshavelargelyindistinguishablemetabolomes" title="sec"> <div class="title" tagx="title" title="title">
Wild type and mutant plants have largely indistinguishable metabolomes</div> <p id="Par14">To verify that differences in microbiome composition can be attributed to a deficiency in diterpenoids in the roots of <i>Zman2</i> plants, metabolite profiling using both targeted and untargeted LCâ€"MS/MS analysis was performed on the same root samples used for microbial composition analysis. Targeted metabolite analysis of the major dolabralexin metabolite, trihydroxydolabrene (THD), confirmed via a standard that THD was near absent in the <i>Zman2</i> mutant while present in WT (Fig.Â <a href="#Fig6">6</a>). Epoxydolabranol was not found in either mutant nor WT plants (Supplemental Fig. 3), while epoxydolabrene was found to be lowly abundant in both mutant and WT plants (Fig.Â <a href="#Fig6">6</a>), presumably because of their conversion to THD. Mirror plots demonstrate these identifications, as well as their absence in the mutant plants (Supplemental Fig. 3). This observation is consistent with previous research, showing low levels of dolabralexin and kauralexin metabolites in <i>Zman2</i>, predictably due to <i>ent</i>-CPP derived from ZmAn1 activity<sup><a href="#CR16">16</a></sup>. As a control, benzoxazinoid abundance was calculated using BOA as a standard, and found to be present in both <i>Zman2</i> and WT roots without significant differences in abundance. Using BOA for LCâ€"MS/MS generates multiple peaks with similar mass spectra due to various benzoxazinoids compounds, and their total area was analyzed here (Supplemental Fig. 3). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Average intensity of metabolites in maize root samples using positive mode LCâ€"MS/MS peak area. Peak area based on epoxydolabrene [Mâ€‰+â€‰H], trihydroxydolabrene [Mâ€‰+â€‰H-H2O], and benzoxazinoids, with the last based on detection of metabolites containing BOA (1,3-benzoxazol-2-one (BOA) [Mâ€‰+â€‰H]). * represents significant difference between two samples pâ€‰â‰¤â€‰0.1. Two-tailed t-tests were used for benzoxazinoids; one-tailed t-tests were used for the remaining metabolites, since they are predicted to be enriched in WT and deficient in <i>Zman2</i>; nâ€‰=â€‰4 (<i>Zman2</i>) or nâ€‰=â€‰5 (WT). Error bars represent standard error.</p>   </div> <p /> <p id="Par15">Parallel untargeted metabolomics analysis also did not indicate significant variance in the global metabolite profiles of the WT and <i>Zman2</i> plants, determined by principle component analyses (Fig.Â <a href="#Fig7">7</a>A,B). Furthermore, PERMANOVA analysis based on all features (dominant mass ions and corresponding specific retention times) demonstrated genotype did not significantly impact the metabolome using either positive or negative ionization modes (Supplemental Table 3). Although the PERMANOVA and PCoA demonstrated no significant difference overall between genotypes, a generalized linear model was used to identify individual metabolites that may be significantly enriched or depleted. Performing analyses in both positive and negative ionization modes, a total of 102 and 46 metabolites were enriched in WT as compared to 79 and 38 enriched in <i>Zman2,</i> respectably by ionization mode (Fig.Â <a href="#Fig7">7</a>C,D, Supplemental Table 4). Annotation of the remaining metabolites with distinct abundance in WT and <i>Zman2</i> roots by comparison to mass spectral databases identified significant matches for 16 compounds (Supplemental Table 4). Consistent with the targeted metabolite profiling, THD was identified among the 10 most abundant compounds in the 102 enriched metabolites in WT roots (ID positive-2360). The remaining metabolites could not be annotated with high confidence, but probably represent so far uncharacterized dolabralexin- or kauralexin-derived metabolites (Supplemental Fig. 4, Supplemental Table 4). The metabolite group containing THD (ID positive-2360), represented as connected nodes in the metabolic network generated in Cytoscape, is connected to two other features that were not significantly enriched or depleted and unannotated, but may represent dolabralexin-type molecules given the similarity of their mass spectra. An additional compound enriched in WT was annotated as mesterolone, a triterpenoid, yet features mass spectra highly similar to that of epoxydolabrene (Supplemental Table 4). The remaining metabolites were at least one order of magnitude less abundant as compared to THD in WT plants (Supplemental Fig. 4, Supplemental Table 4). Although the linear models show some metabolites enriched or depleted in WT versus <i>Zman2</i> plants (Fig.Â <a href="#Fig7">7</a>C,D), the global metabolome is not significantly altered and THD represents one of the most significantly different metabolites in its abundance. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Principal Coordinate Analysis (PCoA) of the metabolome for all root samples in (<b>A</b>) positive mode or (<b>B</b>) negative mode using LCâ€"MS/MS. Each point represents an individual plant sample. Percentage in the axis labels represents the eigenvalue, or percent of variation explained by that axis. Number of enriched metabolites detected using LCâ€"MS/MS in positive mode (<b>C</b>) or negative mode (<b>D</b>), as measured by a generalized linear model at the level of pâ€‰â‰¤â€‰0.05, for each genotype. Overlap are not significantly different; nâ€‰=â€‰4 (<i>Zman2</i>) or nâ€‰=â€‰5 (WT).</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par16">The dynamic interrelations between plants and their species-specific root microbiota directly influence plant health and stress tolerance<sup><a href="#CR4">4</a></sup>. Despite the importance of these mutualistic relationships, the complex chemical mechanisms coordinating inter-organismal interactions remain largely elusive. In particular, limited knowledge exists on how specific metabolites, blends thereof, and the corresponding pathways impact plantâ€"microbe interactions and microbial community assembly. For example, recent maize studies illustrated that mutant genotypes deficient in benzoxazinoid metabolites (specifically MBOA) showed an altered stress response mediated by the influence of MBOA on the below-ground microbial community, thus underscoring the importance of these metabolite-guided plantâ€"microbe interactions on plant health<sup><a href="#CR21">21</a>â€"<a href="#CR23">23</a></sup>. The microbiome and metabolome analyses performed in this study support the hypothesis that specific groups of bioactive diterpenoids in maize, namely dolabralexins and/or kauralexins, contribute to the assembly of the rhizosphere microbiome.</p> <p id="Par17">Although the underlying secretion mechanisms require further study, presence of dolabralexins in maize root exudates supports a role of these compounds in below-ground plantâ€"microbe interactions (Fig.Â <a href="#Fig2">2</a>). Microbiome analysis of the root microbial communities showed no significant influence of genotype on endosphere communities using distance-based methods (Fig.Â <a href="#Fig4">4</a>B). It appears plausible that diterpenoids do not impact endophytic microbes due to the spatial separation of endophytic microbes that predominantly colonize the apoplast<sup><a href="#CR39">39</a>,<a href="#CR40">40</a></sup>, whereas functionalized diterpenoids accumulate intracellularly as demonstrated in several plant species, and are exuded into the rhizosphere as shown here<sup><a href="#CR41">41</a>â€"<a href="#CR43">43</a></sup>. Our results showing distinct rhizosphere microbial communities of <i>Zman2</i> and its WT sibling with a more diverse root microbiome alpha diversity in <i>Zman2</i> (Fig.Â <a href="#Fig3">3</a>), provide evidence supporting a role of diterpenoids in the microbiome assembly by reducing the community diversity. This difference in diversity supports the hypothesis that dolabralexins directly inhibit the growth and/or propagation of specific rhizobia bacteria. This is further supported by the distinct beta diversity between <i>Zman2</i> and WT (Fig.Â <a href="#Fig4">4</a>C), with genotype accounting for 10.7% of the variation between samples. Notably, the significant differences between the two genotypes were defined by only a few OTUs, most of which were assigned to the order Sphingomonadales (Supplemental Figs. 1 and 2). Sphingomonads have been reported to degrade phenolic compounds and utilize them as carbon sources<sup><a href="#CR44">44</a></sup>, and were among the OTUs displaying the greatest heritable variation (H<sup>2</sup>) across maize lines of the NAM (Nested Association Mapping) diversity panel<sup><a href="#CR14">14</a></sup>. Considering the variation of dolabralexins across selected maize inbred lines<sup><a href="#CR18">18</a></sup>, it can be speculated that not only phenolics, but also maize-specific diterpenoids mediate the interaction with species of Sphingomonadales.</p> <p id="Par18">The overall increased microbiome diversity in the <i>Zman2</i> mutant differs from previous research supporting that a greater microbial diversity promotes crop resistance to soil pathogens (Fig.Â <a href="#Fig3">3</a>)<sup><a href="#CR45">45</a>,<a href="#CR46">46</a></sup>, given that previous work has shown <i>Zman2</i> to be more susceptible to fungal pathogens<sup><a href="#CR29">29</a></sup>. Considering these findings in association with the demonstrated anti-microbial activity of both kauralexins and dolabralexins<sup><a href="#CR17">17</a>,<a href="#CR18">18</a></sup>, the relationship between the disease-preventative properties of dolabralexins and bacterial diversity remains more complex, and the influence of dolabralexins may not be directly on the bacterial communities, but possibly indirect via fungal communities and other microbe-microbe interactions, in addition to or instead of direct impacts on growth of rhizobia bacteria.</p> <p id="Par19">Recent research provided insight into the effect of benzoxazinoids on the microbiome and their importance to plant health. Contrasting the significant role of diterpenoids in determining alpha diversity shown in this study (Fig.Â <a href="#Fig3">3</a>), benzoxazinoids did not change alpha diversity, but impacted bacterial beta diversity and specific phyla and OTUs to varying degrees depending on the experimental conditions and genotypes used<sup><a href="#CR21">21</a>â€"<a href="#CR23">23</a></sup>. Thus, it appears plausible that the impact of plant age, soil type, environmental stimuli, genetic background, and/or sampling on metabolite-microbiome interactions contribute to these contrasting results. The level of variation in beta diversity are comparable to the 10.7% of beta diversity explained by diterpenoids in this study. Moreover, the difference in the impact of benzoxazinoids and diterpenoids on the rhizosphere alpha diversity and community composition point toward distinct functionalities of different metabolite classes in maize-microbiome interactions. Recent analysis of two rice mutants, <i>Oscps2</i> and <i>Oscps4</i>, deficient in rice-specific diterpenoids showed indistinguishable rhizosphere microbiomes from their WT siblings, suggesting that maize diterpenoids are distinct in their ability to modify the rhizosphere microbiome<sup><a href="#CR47">47</a></sup>.</p> <p id="Par20">The lack of changes in the global metabolome of WT and <i>Zman2</i> roots is consistent with WT phenotype of <i>Zman2</i> mutant plants under healthy conditions (Fig.Â <a href="#Fig7">7</a>)<sup><a href="#CR16">16</a></sup>, and supports a major role of dolabralexin and/or kauralexin bioactivity, rather than other large metabolic perturbations caused by the ZmAN2 loss of function, on the observed microbiome alterations. While the overall metabolomes were not significantly different, PERMANOVA and PCoA studies identified a number of unidentified metabolites with distinct abundances in WT and <i>Zman2</i> plants. Here were select metabolites not yet identified as dolabralexins or kauralexins that were altered in their abundance between the mutant and WT. These could be not-yet-identified dolabralexins or kauralexins products, breakdown products of these metabolites, or other fluctuations. Interestingly, several of these significantly enriched or depleted compounds featured mass fragmentation patterns suggesting that they represent yet unknown dolabralexin- and kauralexin-type compounds with possible functions in the plants interaction with the rhizosphere microbiome. Notably, recent research investigating the root metabolome and microbiome in maize mutants deficient in the biosynthesis of selected benzoxazinoid compounds found a different scenario, where mutant plants displayed significant metabolic changes across many pathways in PCoA and other statistical methods<sup><a href="#CR23">23</a></sup>. These results underscore the importance of metabolite analysis to understand the broader metabolic implications of pathway mutations especially within complex, branching metabolite networks, so as to not attribute microbiome changes to a single absent metabolite, but possibly due to larger changes throughout the metabolic network in the mutant plants.</p> <p id="Par21">Using a defined pathway mutant, the present study supports a role of species-specific bioactive maize diterpenoids in shaping the rhizosphere microbiome diversity and composition. These findings expand our insight into diterpenoid functions in Poaceous crops beyond well-established anti-microbial and anti-feedant bioactivities. Such deeper knowledge of the mechanisms underlying natural plantâ€"microbe interactions will be critical for ultimately enabling broader agricultural applications.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="rootexudateanalysis" title="sec"> <div class="title" tagx="title" title="title">
Root exudate analysis</div> <p id="Par22">Seeds of <i>Zman2</i> and its isogenic wild type sibling (both in the W22 background) were obtained from Dr. Eric Schmelz (UC San Diego). Seeds (nâ€‰=â€‰6 for each genotype) were planted and grown in potting soil (Sunshine Mix #1, Sungro Horticulture, 75â€"85% Canadian Sphagnum peat moss, 25â€"15% perlite) in 1 gallon pots in a greenhouse and watered with nutrient water, containing calcium nitrate (0.6Â g/L water), Growmore 4â€"13-38 (0.3Â g/L water), and magnesium sulfate (0.3Â g/L water). Growmore contains the following nutrients by weight: 4% total nitrogen, 18% phosphoric acid, 38% soluble potash, 0.5% magnesium, 0.2% boron, 0.1% copper, 0.4% iron, 0.2% manganese, 0.01% molybdenum, and 0.1% zinc.</p> <p id="Par23">After 38Â days, plants were removed from pots and the roots were gently washed with deionized water so as not to cause tissue damage. The nutrient solution pH was 3.36, determined using an Ohaus Starter2000 pH meter. The plants were then placed in 2.8 L Erlenmeyer flasks with the nutrient water previously described and suspended with tape such that only the roots were in the nutrient water. Flasks were wrapped in aluminum foil to prevent light stress to the roots and placed in a growth chamber (conditions as detailed below). After 48Â h, plants were removed, and the nutrient water was filtered through a metal strainer to remove any possible tissue debris. Metabolites were extracted from the exudate water by adding 700Â mL of ethyl acetate to 700Â mL exudate water and leaving at 4Â Â°C for 24Â h. The organic solvent layer was then separated and concentrated using a rotary evaporator for metabolite analysis. Nutrient water containing no plants was used as a control.</p> </div> <div class="plantgrowthconditions" title="sec"> <div class="title" tagx="title" title="title">
Plant growth conditions</div> <p id="Par24">Soil was collected at a UC Davis research field site (coordinates 38.531152, -121.783182) by collecting approximately 6 inches of top soil using bleached shovels and collecting the soil in sterile bags. The field site had grown maize for one year, and at the time of collection (November 19, 2016) was fallow and had the stover previously turned under after the summer harvest. The soil at this field site is a silty clay loam, as determined using the standard â€œtexture by feelâ€� method for soil classification<sup><a href="#CR48">48</a></sup>. The soil was then mixed in the sterile bags by using the sterile shovel to stir and mix until the soil appeared homogenous. The soil was then distributed to 2.37 L pots that were sterilized using 3% bleach wash. The soil was not sieved, and any large rocks or soil chunks were removed by hand during pot filling.</p> <p id="Par25">Plants were grown in a growth chamber in the pots in order to control for all other environmental conditions. The growth chamber was set to a 16/8Â hÂ day/night cycle, with a 26/22Â Â°C day/night temperature cycle. Seeds of <i>Zman2</i> and WT plants were sterilized in 3% (v/v) bleach for one hour, then washed five times with deionized water, and planted approximately 3Â cm deep in the pots with maize field soil. Pots were distributed in the growth chamber in a block design to mitigate location effects. <i>Zman2</i> and WT plant microbiomes and corresponding metabolomes were measured for six biological replicates each using bulk soil (no plants) as a control. Pots were watered every other day with 175Â mL of nutrient water (see contents in root secretion assay methods), and tissue was harvested on the 45<sup>th</sup> day as described below.</p> </div> <div class="microbiomesamplecollection" title="sec"> <div class="title" tagx="title" title="title">
Microbiome sample collection</div> <p id="Par26">Sample collection and processing was adapted from Edwards et al.<sup><a href="#CR7">7</a></sup>. In brief, plants were carefully removed from the soil, and gently shaken untilâ€‰~â€‰2Â mm of soil adhering to the root remained. nâ€‰=â€‰5 (bulk soil) or nâ€‰=â€‰6 (all plant samples). The roots were then transferred to a 50Â mL falcon tube contained sterile phosphate buffer saline (PBS, 137Â mM NaCl, 2.7Â mM KCl, 10Â mM Na<sub>2</sub>HPO<sub>4</sub>, 1.8Â mM KH<sub>2</sub>PO<sub>4</sub>) and placed on ice. For analysis of the rhizosphere microbiome, these roots in the PBS in falcon tubes on ice were shaken using sterile forceps to remove the soil from the root surface and soil samples were stored at 4Â Â°C until further processing the next day. Gentle shaking with the forceps and careful observation ensured that no roots were broken and in the rhizosphere sample. For analysis of the endosphere microbiome, the above root samples were placed into fresh PBS buffer in a new 50Â mL falcon tube and sonicated three 3 times for 10Â s each, followed by placing the roots in fresh PBS buffer again to remove any rhizoplane microbes. Using these roots,â€‰~â€‰4Â cm sections of the primary root (beginning 2Â cm below the root-shoot junction) was cut, placed in a new tube, frozen in liquid N<sub>2</sub>, and stored at âˆ’ 80Â Â°C until further processing. Bulk soil samples from soil 2Â cm below the surface were collected using a sterile scoop and stored in PBS buffer at 4Â Â°C until sample processing the next day.</p> </div> <div class="dnaextraction" title="sec"> <div class="title" tagx="title" title="title">
DNA extraction</div> <p id="Par27">All DNA was extracted using the MoBio PowerSoil DNA extraction kit and eluted in 50 ÂµL of DEPC-treated water. The rhizosphere samples were concentrated by pipetting 1Â mL of the rhizosphere soil in PBS into a 2Â mL tube and centrifuged for 30Â s at 10,000Ã—<i>g</i>. The supernatant was discarded and the soil was used for DNA extraction. The endosphere samples were homogenized and ground in liquid N<sub>2</sub> for DNA extraction with the MoBio Powersoil DNA kit.</p> </div> <div class="16srrnageneamplification,quantitation,andsequencing" title="sec"> <div class="title" tagx="title" title="title">
16S rRNA gene amplification, quantitation, and sequencing</div> <p id="Par28">The V4 region (515 to 806Â bp of the 16S rRNA gene) of the 16S rRNA gene was amplified according to Edwards et al.<sup><a href="#CR49">49</a></sup>. In brief, PCR was performed using Qiagen HotStart HiFidelity polymerase with the following parameters for each mix: 6.25 ÂµL water, 2.5 ÂµL buffer, 1.25 ÂµL of 10Â ÂµM forward primer, 1.25 ÂµL of 10Â ÂµM reverse primer, 0.25 ÂµL HotStart polymerase, and 1 ÂµL of DNA. Specific primer pairs, containing unique 12Â bp barcode adaptors on each end of the forward and reverse primers were used for each reaction. Samples without DNA were used as negative controls. A touchdown PCR program was used with the following parameters: 95Â Â°C for 5Â min; 7 cycles of 95Â Â°C for 45Â s, 65Â Â°C for 1Â min decreasing at 2Â Â°C per cycle, and 72Â Â°C for 90Â s; 30 cycles of 95Â Â°C for 45Â s, 50Â Â°C for 30Â s, 72Â Â°C for 90Â s; a final extension of 72Â Â°C for 5Â min; and samples were held at 4Â Â°C. Only samples producing single amplicon bands as verified by agarose gel electrophoresis were considered for further analysis, and all samples were maintained for downstream analysis.</p> <p id="Par29">Amplicons were purified to remove primers using AmPure XP beads (Beckman Coulter). Here, beads were added to each PCR reaction, incubated at room temperature for 5Â min, and placed on a magnet for 2Â min to separate the beads. After removal of the supernatant, the beads were washed with 70% ethanol twice. The ethanol was then allowed to evaporate and the beads were resuspended in 50 ÂµL water, mixed well, and placed again on a magnet to remove the supernatant containing the desired PCR products. DNA concentrations were measured using a Qubit, and pooled to reach samples of equimolar concentrations. The pooled samples were cleaned as described above, separated by agarose gel electrophoresis and the 400Â bp amplicons were extracted using a NucleoSpin Gel and PCR Clean-up kit (Machereyâ€"Nagel). Libraries were made and sequencing was performed at the UC Davis Genome Center using 250â€‰Ã—â€‰250 paired end, dual index Illumina MiSeq sequencing.</p> </div> <div class="sequenceanalysis" title="sec"> <div class="title" tagx="title" title="title">
Sequence analysis</div> <p id="Par30">Sequences were analyzed as previously described by Edwards et al.<sup><a href="#CR7">7</a></sup>. In brief, sequences were demultiplexed based on individual barcodes using a custom R script, and assembled into single sequences using Pandaseq. Sequences were then clustered into OTUs with the NINJA-OPS pipeline using 97% pairwise sequence identity referenced against the Greengenes 16S rRNA sequence database (version 13_8)<sup><a href="#CR32">32</a></sup>.</p> <p id="Par31">In total, 1,802,959 high-quality sequences were obtained with a median read count of 31,204 per sample, and a range of 1699â€"87,795 (All data is available in Sequence Read Archive, Sequence Read Archive repository, BioProject ID PRJNA600272 [<a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA600272">https://www.ncbi.nlm.nih.gov/sra/PRJNA600272</a>]). Using the QIIME pipeline, reads were clustered based on 97% sequence identity into operational taxonomic units (OTUs) and were annotated using QIIME and the Greengenes database<sup><a href="#CR32">32</a></sup>, resulting in 7181 microbial OTUs. Chloroplast and mitochondrial OTUs represented 65 OTUs and were removed, along with low-abundance OTUs (less than 5% of the total sample), leaving 4258 total OTUs. OTU counts were then normalized by relative abundance, which was used rather than rarefaction methods so as not to discard low abundance OTUs<sup><a href="#CR36">36</a></sup>.</p> <p id="Par32">All statistical analysis of the OTU table generated by QIIME<sup><a href="#CR32">32</a></sup> were analyzed using custom R scripts (version 3.6.1)<sup><a href="#CR50">50</a></sup>. Alpha-diversity was measured using the â€œShannonâ€� method in the R package vegan<sup><a href="#CR51">51</a></sup>. Principle coordinate analysis (PCoA) were conducted using unconstrained principles and Bray distances in the R package vegan<sup><a href="#CR51">51</a></sup>. PERMANOVA (permutational multivariate analyses of variance) analysis was performed using the R package vegan function adonis to measure beta-diversity<sup><a href="#CR51">51</a></sup>. The DESeq2 package<sup><a href="#CR52">52</a></sup> was used to identify OTUs and phyla whose abundance was differentially affected by our experimental variables. Phyla counts were derived by aggregating raw counts for OTUs at the phylum level within each sample. After analysis by DESeq2, the results were compiled and tidied using the biobroom package<sup><a href="#CR53">53</a></sup>. Plots were visualized using the ggplot2 in the tidyverse package<sup><a href="#CR54">54</a></sup>. All scripts generated in this study have been deposited to GitHub (<a href="https://github.com/kmurphy61/maizemicrobiome">https://github.com/kmurphy61/maizemicrobiome</a>).</p> </div> <div class="metaboliteextraction" title="sec"> <div class="title" tagx="title" title="title">
Metabolite extraction</div> <p id="Par33">The remaining roots (~â€‰1Â g fresh weight) used for endosphere microbiome analysis (see microbiome sample collection) were homogenized and ground in liquid nitrogen. Because of availability of tissue, the number of plant samples were reduced for metabolite extraction as compared to microbiome DNA extraction; nâ€‰=â€‰4 for <i>Zman2</i>, nâ€‰=â€‰5 for WT. Samples were then placed in a 2Â mL glass vial and metabolites extracted by incubation in 2Â mL methanol overnight at 4Â Â°C with gentle rocking. Samples were centrifuged for 10Â min at 4000â€‰Ã—â€‰g and the methanol phase transferred to a new vial using a glass pipette, air-dried, and resuspended in 100 ÂµL methanol.</p> </div> <div class="metaboliteanalysis" title="sec"> <div class="title" tagx="title" title="title">
Metabolite analysis</div> <p id="Par34">For metabolite analysis by liquid chromatography tandem mass spectrometry (LCâ€"MS/MS), samples were spiked with 4Â ÂµM internal standard mixture of deuterium-labeled lipids (Cat# 110899, 857463P, 861809O, 110922, 110922, 110921, 110918, 110579, 110544, Avanti Polar Lipids, Inc) and 1Â Âµg/mL ABMBA (2-Amino-3-bromo-5-methylbenzoic acid, Sigma). UHPLC reverse phase chromatography was performed using an Agilent 1290 LC coupled with a QExactive Orbitrap MS (QEâ€‰=â€‰139) (Thermo Scientific, San Jose, CA). Chromatography was performed using a C18 column (Agilent ZORBAX Eclipse Plus C18, Rapid Resolution HD, 2.1â€‰Ã—â€‰50Â mm, 1.8Â Âµm) at a flow rate of 0.4Â mL/min and injection volume varied from 0.9 to 3.5 ÂµL to normalize against sample dry weight. Samples were run on the C18 column at 60 ÂºC equilibrated with 100% buffer A (100% LCâ€"MS water w/ 0.1% formic acid) for 1Â min, following by a linear dilution of buffer A down to 0% with buffer B (100% acetonitrile w/ 0.1% formic acid) over 7Â min, and followed by isocratic elution in 100% buffer B for 1.5Â min. Full MS spectra were collected ranging from <i>m</i>/<i>z</i> 80â€"2,000 at 60,000 to 70,000 resolution in both positive and negative mode, with MS/MS fragmentation data acquisition using an average of stepped 10â€"20-40 and 20â€"50-60Â eV collision energies at 17,500 resolution. For targeted analysis, product identification by comparison to standards was performed where authentic standards were available.</p> <p id="Par35">For untargeted analysis, exact mass and retention time coupled with MS/MS fragmentation spectra were used to identify compounds. Featuresâ€"high intensity signals narrowly contained at a given retention time and <i>m</i>/<i>z</i>â€"were detected using the MZMine software v 2.24 (<a href="http://dx.doi.org/10.1093/bioinformatics/btk039">http://dx.doi.org/10.1093/bioinformatics/btk039</a>). Data was filtered to remove MS/MS fragment ions withinâ€‰Â±â€‰17Â Da of the precursor m/z, and subsequently filtered to remove all but the top six ions in theâ€‰Â±â€‰50Â Da throughout the spectrum. Precursor and fragment ion tolerance was 0.05Â Da. Features that showed a significantly different abundance (peak height) using generalized linear models, calculated using custom R scripts and the lm() function<sup><a href="#CR50">50</a></sup>, with statistical analysis results in Supplemental Table 3 and significantly encriched or depleted features listed in Supplementary Table 4. Generalized linear models are linear regression models used to determine if a particular feature is significantly different in abundance between two genotypes. All features were annotated using Global Natural Products Social Molecular Networking (GNPS)<sup><a href="#CR55">55</a>â€"<a href="#CR59">59</a></sup>. In short, a Feature-Based Molecular Networking workflow was used to assign features to a molecular network with a cosine score above 0.7 and more than six matched peaks. The maximum size of a molecular family was 100, and low scoring edges were removed to meet this threshold. The spectra were then searched against GNPS spectral libraries and annotated with the top hit, if there was one. Complete annotations, features present, and Cytoscape visualization networks are available online (<a href="https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=a748e519752249e2a912dd3d466db98d">https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=a748e519752249e2a912dd3d466db98d</a>). All scripts are available on GitHub (<a href="https://github.com/kmurphy61/maizemicrobiome.git">https://github.com/kmurphy61/maizemicrobiome.git</a><u>)</u>. Lists of significantly different features enriched or depleted in each sample type are available in Supplemental Table 4.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_79320_MOESM1_ESM.docx">LINK</a> <p>Supplementary Legends.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_79320_MOESM2_ESM.pdf">LINK</a> <p>Supplementary Tables.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_79320_MOESM3_ESM.xlsx">LINK</a> <p>Supplementary Tables.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_79320_MOESM4_ESM.pdf">LINK</a> <p>Supplementary Figures.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>The online version contains supplementary material available at 10.1038/s41598-020-79320-z.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors gratefully acknowledge Dr. Eric Schmelz (UC San Diego), for providing seeds of <i>Zman2</i> and WT plants used in this study, and for providing purified trihydroxydolabrene (THD) for use as a standard. They also acknowledge Zach Liechty (UC Davis) and Christian Santos-MedellÃ­n (UC Davis) for their advising in data analysis and computation. Financial support for this work was provided by the National Science Foundation (NSF) Plant-Biotic Interactions Program (Award# 1758976 to PZ), the NSF Graduate Research Fellowship Program (to KMM), the UC Davis Innovation Institute for Food and Health (IIFH) Fellowship Program (to KMM &amp;amp; PZ), and the USDA NIFA Predoctoral Fellowship Program (Award# 2019-67011-29544 to KMM). JE is supported by funding from the USDA NIFA Postdoctoral Fellowship program (Award# 2019-67012-29713 to JE). The work conducted by the US Department of Energy Joint Genome Institute, a DOE Office of Science User Facility, is supported under contract no. DE-AC02-05CH11231.Â This manuscript has been authored by an author at Lawrence Berkeley National Laboratory under Contract No. DE-AC02-05CH11231 with the U.S. Department of Energy. The U.S. Government retains, and the publisher, by accepting the article for publication, acknowledges, that the U.S. Government retains a non-exclusive, paid-up, irrevocable, world-wide license to publish or reproduce the published form of this manuscript, or allow others to do so, for U.S. Government purposes.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>K.M.M. and P.Z. designed the experiments; T.R.N. advised on metabolomics experiments; K.M.M. performed most experiments; K.M.M. and J.E. performed microbiome analyses; K.B.L. and B.P.B. performed LC/MS experiments and data analysis; K.M.M. and P.Z. wrote the article with contributions of all other authors.</p> <div class="title" tagx="title" title="title">
Data availability</div> <p>The datasets generated during and/or analyzed during this study are available in the Sequence Read Archive repository, BioProject ID PRJNA600272 [<a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA600272">https://www.ncbi.nlm.nih.gov/sra/PRJNA600272</a>]. Complete metabolite annotations and Cytoscape visualization networks are available (<a href="https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=a748e519752249e2a912dd3d466db98d">https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=a748e519752249e2a912dd3d466db98d</a>). The code used to analyze these datasets are available in the GitHub repository, <a href="https://github.com/kmurphy61/maizemicrobiome.git">https://github.com/kmurphy61/maizemicrobiome.git</a>.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par36">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korenblum</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aharoni</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phytobiome metabolism: beneficial soil microbes steer crop plants' secondary metabolism</span><span class="source" tagx="source" title="source">Pest Manag. Sci.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">2378</span><span class="lpage" tagx="lpage" title="lpage">2384</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ps.5440">10.1002/ps.5440</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30973666">30973666</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitter</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brader</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfaffenbichler</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sessitsch</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Next generation microbiome applications for crop production - limitations and the need of knowledge-based solutions</span><span class="source" tagx="source" title="source">Curr. Opin. Microbiol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">59</span><span class="lpage" tagx="lpage" title="lpage">65</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mib.2019.10.006">10.1016/j.mib.2019.10.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31731227">31731227</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coleman-Derr</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Causes and consequences of a conserved bacterial root microbiome response to drought stress</span><span class="source" tagx="source" title="source">Curr. Opin. Microbiol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">6</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mib.2019.07.003">10.1016/j.mib.2019.07.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31454709">31454709</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berendsen</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pieterse</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakker</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span></span><span class="mixed-article-title" title="mixed-article-title">The rhizosphere microbiome and plant health</span><span class="source" tagx="source" title="source">Trends Plant Sci.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">478</span><span class="lpage" tagx="lpage" title="lpage">486</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tplants.2012.04.001">10.1016/j.tplants.2012.04.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22564542">22564542</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castrillo</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Root microbiota drive direct integration of phosphate stress and immunity</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">543</span><span class="fpage" tagx="fpage" title="fpage">513</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature21417">10.1038/nature21417</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28297714">28297714</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paulson</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">XQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolter</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Simplified and representative bacterial community of maize roots</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">114</span><span class="fpage" tagx="fpage" title="fpage">E2450</span><span class="lpage" tagx="lpage" title="lpage">E2459</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1616148114">10.1073/pnas.1616148114</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28275097">28275097</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure, variation, and assembly of the root-associated microbiomes of rice</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">112</span><span class="fpage" tagx="fpage" title="fpage">E911</span><span class="lpage" tagx="lpage" title="lpage">920</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1414592112">10.1073/pnas.1414592112</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25605935">25605935</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aira</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomez-Brandon</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lazcano</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baath</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dominguez</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant genotype strongly modifies the structure and growth of maize rhizosphere microbial communities</span><span class="source" tagx="source" title="source">Soil. Biol. Biochem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">2276</span><span class="lpage" tagx="lpage" title="lpage">2281</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2010.08.029">10.1016/j.soilbio.2010.08.029</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia-Salamanca</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial diversity in the rhizosphere of maize and the surrounding carbonate-rich bulk soil</span><span class="source" tagx="source" title="source">Microb. Biotechnol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">36</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1751-7915.2012.00358.x">10.1111/j.1751-7915.2012.00358.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22883414">22883414</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Oliveira</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial diversity in the rhizosphere of maize genotypes contrasting for phosphorus use efficiency</span><span class="source" tagx="source" title="source">Pesqui Agropecu Bras.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">1473</span><span class="lpage" tagx="lpage" title="lpage">1482</span><span class="pub-id"><a href="https://dx.doi.org/10.1590/S0100-204X2009001100015">10.1590/S0100-204X2009001100015</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haichar</span><span class="given-names" tagx="given-names" title="given-names">FE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Plant host habitat and root exudates shape soil bacterial community structure</span><span class="source" tagx="source" title="source">Isme J.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">1221</span><span class="lpage" tagx="lpage" title="lpage">1230</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ismej.2008.80">10.1038/ismej.2008.80</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18754043">18754043</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savary</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The global burden of pathogens and pests on major food crops</span><span class="source" tagx="source" title="source">Nat. Ecol. Evol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">430</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41559-018-0793-y">10.1038/s41559-018-0793-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30718852">30718852</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lundberg</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Defining the core <i>Arabidopsis thaliana</i> root microbiome</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">488</span><span class="fpage" tagx="fpage" title="fpage">86</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature11237">10.1038/nature11237</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22859206">22859206</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walters</span><span class="given-names" tagx="given-names" title="given-names">WA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Large-scale replicated field study of maize rhizosphere identifies heritable microbes</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2018</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1800918115">10.1073/pnas.1800918115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29941552">29941552</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christensen</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Maize death acids, 9-lipoxygenase-derived cyclopente(a)nones, display activity as cytotoxic phytoalexins and transcriptional mediators</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">112</span><span class="fpage" tagx="fpage" title="fpage">11407</span><span class="lpage" tagx="lpage" title="lpage">11412</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1511131112">10.1073/pnas.1511131112</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26305953">26305953</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaughan</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Accumulation of terpenoid phytoalexins in maize roots is associated with drought tolerance</span><span class="source" tagx="source" title="source">Plant Cell Environ.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">2195</span><span class="lpage" tagx="lpage" title="lpage">2207</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/pce.12482">10.1111/pce.12482</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25392907">25392907</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmelz</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identity, regulation, and activity of inducible diterpenoid phytoalexins in maize</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">5455</span><span class="lpage" tagx="lpage" title="lpage">5460</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1014714108">10.1073/pnas.1014714108</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21402917">21402917</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mafu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery, biosynthesis and stress-related accumulation of dolabradiene-derived defenses in maize</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2018</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.17.01351">10.1104/pp.17.01351</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29475898">29475898</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huffaker</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Novel acidic sesquiterpenoids constitute a dominant class of pathogen-induced phytoalexins in maize</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">156</span><span class="fpage" tagx="fpage" title="fpage">2082</span><span class="lpage" tagx="lpage" title="lpage">2097</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.111.179457">10.1104/pp.111.179457</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21690302">21690302</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmad</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Benzoxazinoid metabolites regulate innate immunity against aphids and fungi in maize</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">157</span><span class="fpage" tagx="fpage" title="fpage">317</span><span class="lpage" tagx="lpage" title="lpage">327</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.111.180224">10.1104/pp.111.180224</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21730199">21730199</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Root exudate metabolites drive plantâ€"soil feedbacks on growth and defense by shaping the rhizosphere microbiota</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">2738</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-018-05122-7">10.1038/s41467-018-05122-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30013066">30013066</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kudjordjie</span><span class="given-names" tagx="given-names" title="given-names">EN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sapkota</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steffensen</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fomsgaard</span><span class="given-names" tagx="given-names" title="given-names">IS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicolaisen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Maize synthesized benzoxazinoids affect the host associated microbiome</span><span class="source" tagx="source" title="source">Microbiome</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">59</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s40168-019-0677-7">10.1186/s40168-019-0677-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30975184">30975184</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotton</span><span class="given-names" tagx="given-names" title="given-names">TEA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic regulation of the maize rhizobiome by benzoxazinoids</span><span class="source" tagx="source" title="source">ISME J.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">1647</span><span class="lpage" tagx="lpage" title="lpage">1658</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41396-019-0375-2">10.1038/s41396-019-0375-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30796337">30796337</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osbourn</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant terpenes that mediate below-ground interactions: prospects for bioengineering terpenoids for plant protection</span><span class="source" tagx="source" title="source">Pest Manag. Sci.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">2368</span><span class="lpage" tagx="lpage" title="lpage">2377</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ps.5410">10.1002/ps.5410</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30884099">30884099</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Block</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaughan</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmelz</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christensen</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biosynthesis and function of terpenoid defense compounds in maize (<i>Zea mays</i>)</span><span class="source" tagx="source" title="source">Planta</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">249</span><span class="fpage" tagx="fpage" title="fpage">21</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00425-018-2999-2">10.1007/s00425-018-2999-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30187155">30187155</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christensen</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Commercial hybrids and mutant genotypes reveal complex protective roles for inducible terpenoid defenses in maize</span><span class="source" tagx="source" title="source">J. Exp. Bot.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">1693</span><span class="lpage" tagx="lpage" title="lpage">1705</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/erx495">10.1093/jxb/erx495</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29361044">29361044</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christie</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Systems genetics reveals a transcriptional network associated with susceptibility in the maize-grey leaf spot pathosystem</span><span class="source" tagx="source" title="source">Plant J.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">746</span><span class="lpage" tagx="lpage" title="lpage">763</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/tpj.13419">10.1111/tpj.13419</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27862526">27862526</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmelz</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biosynthesis, elicitation and roles of monocot terpenoid phytoalexins</span><span class="source" tagx="source" title="source">Plant J.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">79</span><span class="fpage" tagx="fpage" title="fpage">659</span><span class="lpage" tagx="lpage" title="lpage">678</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/tpj.12436">10.1111/tpj.12436</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24450747">24450747</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The maize An2 gene is induced by Fusarium attack and encodes an ent-copalyl diphosphate synthase</span><span class="source" tagx="source" title="source">Plant Mol. Biol.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">881</span><span class="lpage" tagx="lpage" title="lpage">894</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11103-005-1674-8">10.1007/s11103-005-1674-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16307364">16307364</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bensen</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of the maize An1 gene</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">75</span><span class="lpage" tagx="lpage" title="lpage">84</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.7.1.75">10.1105/tpc.7.1.75</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7696880">7696880</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Multiple genes recruited from hormone pathways partition maize diterpenoid defences</span><span class="source" tagx="source" title="source">Nat. Plants</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1043</span><span class="lpage" tagx="lpage" title="lpage">1056</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41477-019-0509-6">10.1038/s41477-019-0509-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31527844">31527844</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caporaso</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">QIIME allows analysis of high-throughput community sequencing data</span><span class="source" tagx="source" title="source">Nat. Methods</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">335</span><span class="lpage" tagx="lpage" title="lpage">336</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.f.303">10.1038/nmeth.f.303</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20383131">20383131</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Soil domestication by rice cultivation results in plant-soil feedback through shifts in soil microbiota</span><span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">221</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-019-1825-x">10.1186/s13059-019-1825-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31651356">31651356</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Compositional shifts in root-associated bacterial and archaeal microbiota track the plant life cycle in field-grown rice</span><span class="source" tagx="source" title="source">PLoS Biol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">e2003862</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.2003862">10.1371/journal.pbio.2003862</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29474469">29474469</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liechty</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative analysis of root microbiomes of rice cultivars with high and low methane emissions reveals differences in abundance of methanogenic archaea and putative upstream fermenters</span><span class="source" tagx="source" title="source">mSystems.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/mSystems.00897-19">10.1128/mSystems.00897-19</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32071162">32071162</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMurdie</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Waste not, want not: why rarefying microbiome data is inadmissible</span><span class="source" tagx="source" title="source">PLoS Comput. Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e1003531</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1003531">10.1371/journal.pcbi.1003531</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24699258">24699258</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anders</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Count-based differential expression analysis of RNA sequencing data using R and bioconductor</span><span class="source" tagx="source" title="source">Nat. Protoc.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1765</span><span class="lpage" tagx="lpage" title="lpage">1786</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2013.099">10.1038/nprot.2013.099</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23975260">23975260</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naylor</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeGraaf</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purdom</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coleman-Derr</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Drought and host selection influence bacterial community dynamics in the grass root microbiome</span><span class="source" tagx="source" title="source">ISME J.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">2691</span><span class="lpage" tagx="lpage" title="lpage">2704</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ismej.2017.118">10.1038/ismej.2017.118</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28753209">28753209</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">EK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poole</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span></span><span class="mixed-article-title" title="mixed-article-title">The plant microbiome</span><span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2013-14-6-209">10.1186/gb-2013-14-6-209</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23805896">23805896</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inner plant values: diversity, colonization and benefits from endophytic bacteria</span><span class="source" tagx="source" title="source">Front. Microbiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">2552</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2017.02552">10.3389/fmicb.2017.02552</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29312235">29312235</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lange</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Integrative approaches for the identification and localization of specialized metabolites in tripterygium roots</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">173</span><span class="fpage" tagx="fpage" title="fpage">456</span><span class="lpage" tagx="lpage" title="lpage">469</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.15.01593">10.1104/pp.15.01593</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27864443">27864443</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pateraki</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Manoyl oxide (13R), the biosynthetic precursor of forskolin, is synthesized in specialized root cork cells in <i>Coleus forskohlii</i></span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">164</span><span class="fpage" tagx="fpage" title="fpage">1222</span><span class="lpage" tagx="lpage" title="lpage">1236</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.113.228429">10.1104/pp.113.228429</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24481136">24481136</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaughan</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Formation of the unusual semivolatile diterpene rhizathalene by the Arabidopsis class I terpene synthase TPS08 in the root stele is involved in defense against belowground herbivory</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1108</span><span class="lpage" tagx="lpage" title="lpage">1125</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.112.100057">10.1105/tpc.112.100057</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23512856">23512856</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">44Kim, S. J., Chun, J., Bae, K. S. &amp;amp; Kim, Y. C. Polyphasic assignment of an aromatic-degrading Pseudomonas sp., strain DJ77, in the genus Sphingomonas as Sphingomonas chungbukensis sp. nov. <i>Int. J. Syst. Evol. Microbiol.</i><b>50 Pt 4</b>, 1641â€"1647, doi:10.1099/00207713-50-4-1641 (2000).</span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Elsas</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Microbial diversity determines the invasion of soil by a bacterial pathogen</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">1159</span><span class="lpage" tagx="lpage" title="lpage">1164</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1109326109">10.1073/pnas.1109326109</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22232669">22232669</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Trophic network architecture of root-associated bacterial communities determines pathogen invasion and plant health</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2015</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms9413">10.1038/ncomms9413</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26666832">26666832</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inferring roles in defense from metabolic allocation of rice diterpenoids</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">2018</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.18.00205">10.1105/tpc.18.00205</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30373760">30373760</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salley</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A comparison of soil texture-by-feel estimates: implications for the citizen soil scientist</span><span class="source" tagx="source" title="source">Soil Sci. Soc. Am. J.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">82</span><span class="fpage" tagx="fpage" title="fpage">1526</span><span class="lpage" tagx="lpage" title="lpage">1537</span><span class="pub-id"><a href="https://dx.doi.org/10.2136/sssaj2018.04.0137">10.2136/sssaj2018.04.0137</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">49Edwards, J., Santos-MedellÃ­n, C. &amp;amp; Sundaresan, V. Extraction and 16S rRNA sequence analysis of microbiomes associated with rice roots. <b>8(</b>12), e2884, doi: 10.21769/BioProtoc.2884 (2018).</span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">50Team, R. C. <i>R: A language and environment for statistical computing. R Foundation for Statistical Computing</i>, &amp;lt;<a href="https://www.R-project.org/">https://www.R-project.org/</a>&amp;gt; (2019).</span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">51Oksanen, J. <i>et al. vegan: Community Ecology Package. R package version 2.5â€"6.</i>, &amp;lt;<a href="https://CRAN.R-project.org/package=vegan">https://CRAN.R-project.org/package=vegan</a>&amp;gt; (2019).</span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Love</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anders</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span><span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">550</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-014-0550-8">10.1186/s13059-014-0550-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25516281">25516281</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">53Bass, A. J. <i>et al. biobroom: Turn Bioconductor objects into tidy data frames. R package version 1.16.0.</i>, &amp;lt;<a href="https://github.com/StoreyLab/biobroom">https://github.com/StoreyLab/biobroom</a>&amp;gt; (2019).</span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">54Wickham, H. <i>tidyverse: Easily Install and Load the 'Tidyverse'. R package version 1.2.1.</i>, &amp;lt;<a href="https://CRAN.R-project.org/package=tidyverse">https://CRAN.R-project.org/package=tidyverse</a>&amp;gt; (2017).</span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pluskal</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castillo</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villar-Briones</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oresic</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">MZmine 2: modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data</span><span class="source" tagx="source" title="source">BMC Bioinformatics</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">395</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-11-395">10.1186/1471-2105-11-395</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20650010">20650010</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katajamaa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miettinen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">OreÅ¡iÄ�</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">MZmine: toolbox for processing and visualization of mass spectrometry based molecular profile data</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">634</span><span class="lpage" tagx="lpage" title="lpage">636</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btk039">10.1093/bioinformatics/btk039</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16403790">16403790</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">57Nothias, L. F. <i>et al.</i> Feature-based Molecular Networking in the GNPS Analysis Environment. <i>bioRxiv</i>, 812404, doi:10.1101/812404 (2019).</span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking</span><span class="source" tagx="source" title="source">Nat. Biotechnol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">828</span><span class="lpage" tagx="lpage" title="lpage">837</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.3597">10.1038/nbt.3597</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27504778">27504778</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">59Ono, K., Demchak, B. &amp;amp; Ideker, T. Cytoscape tools for the web age: D3.js and Cytoscape.js exporters. <i>F1000Res</i><b>3</b>, 143, doi:10.12688/f1000research.4510.2 (2014).</span></li> </ul> </div> </div>  </body></html>