<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Molecules</span><span class="iso-abbrev" title="iso-abbrev">Molecules</span><span class="publisher-id" title="publisher-id">molecules</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Molecules</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1420-3049</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7504347">7504347</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/molecules25173822">10.3390/molecules25173822</a></span><span class="publisher-id" title="publisher-id">molecules-25-03822</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
In Silico Identification of Potential Natural Product Inhibitors of Human Proteases Key to SARS-CoV-2 Infection</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-3232-3299</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vivek-Ananth</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span><a href="#af1-molecules-25-03822">1</a><a href="#af2-molecules-25-03822">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-5854-0710</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rana</span><span class="given-names" tagx="given-names" title="given-names">Abhijit</span></span><a href="#af2-molecules-25-03822">2</a><a href="#af3-molecules-25-03822">3</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-8115-6634</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajan</span><span class="given-names" tagx="given-names" title="given-names">Nithin</span></span><a href="#af1-molecules-25-03822">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-0791-2259</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biswal</span><span class="given-names" tagx="given-names" title="given-names">Himansu S.</span></span><a href="#af2-molecules-25-03822">2</a><a href="#af3-molecules-25-03822">3</a><a href="#c1-molecules-25-03822">*</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-6796-9604</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samal</span><span class="given-names" tagx="given-names" title="given-names">Areejit</span></span><a href="#af1-molecules-25-03822">1</a><a href="#af2-molecules-25-03822">2</a><a href="#c1-molecules-25-03822">*</a><a href="#fn1-molecules-25-03822">â€ </a></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carta</span><span class="given-names" tagx="given-names" title="given-names">Antonio</span></span><span class="role" tagx="role" title="role">Academic Editor</span></span></div><span class="citation_author_institution" id="af1-molecules-25-03822">[1], <span class="email" tagx="email" title="email">vivekananth@imsc.res.in</span><span class="email" tagx="email" title="email">nithinofficial2912@gmail.com</span></span><span class="citation_author_institution" id="af2-molecules-25-03822">[2], <span class="email" tagx="email" title="email">abhijit.rana@niser.ac.in</span></span><span class="citation_author_institution" id="af3-molecules-25-03822">[3], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">himansu@niser.ac.in</span> (H.S.B.); <span class="email" tagx="email" title="email">asamal@imsc.res.in</span> (A.S.)</div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">â€ </span><p>Lead Contact.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-8-8</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-9</span></span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">17</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">3822</span><span class="history" title="history"><span class="received" title="received">received: 2020-7-27</span><span class="accepted" title="accepted">accepted: 2020-8-16</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Presently, there are no approved drugs or vaccines to treat COVID-19, which has spread to over 200 countries and at the time of writing was responsible for over 650,000 deaths worldwide. Recent studies have shown that two human proteases, TMPRSS2 and cathepsin L, play a key role in host cell entry of SARS-CoV-2. Importantly, inhibitors of these proteases were shown to block SARS-CoV-2 infection. Here, we perform virtual screening of 14,011 phytochemicals produced by Indian medicinal plants to identify natural product inhibitors of TMPRSS2 and cathepsin L. AutoDock Vina was used to perform molecular docking of phytochemicals against TMPRSS2 and cathepsin L. Potential phytochemical inhibitors were filtered by comparing their docked binding energies with those of known inhibitors of TMPRSS2 and cathepsin L. Further, the ligand binding site residues and non-covalent interactions between protein and ligand were used as an additional filter to identify phytochemical inhibitors that either bind to or form interactions with residues important for the specificity of the target proteases. This led to the identification of 96 inhibitors of TMPRSS2 and 9 inhibitors of cathepsin L among phytochemicals of Indian medicinal plants. Further, we have performed molecular dynamics (MD) simulations to analyze the stability of the protein-ligand complexes for the three top inhibitors of TMPRSS2 namely, qingdainone, edgeworoside C and adlumidine, and of cathepsin L namely, ararobinol, (+)-oxoturkiyenine and 3Î±,17Î±-cinchophylline. Interestingly, several herbal sources of identified phytochemical inhibitors have antiviral or anti-inflammatory use in traditional medicine. Further in vitro and in vivo testing is needed before clinical trials of the promising phytochemical inhibitors identified here.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
TMPRSS2</div> <div class="kwd" title="kwd">
cathepsin L</div> <div class="kwd" title="kwd">
molecular docking</div> <div class="kwd" title="kwd">
molecular dynamics</div> <div class="kwd" title="kwd">
non-covalent interactions</div> <div class="kwd" title="kwd">
phytochemical inhibitors</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>In December 2019, a new respiratory disease with unknown cause with clinical symptoms of fever, cough, shortness of breath, fatigue and pneumonia was first reported in Wuhan, (China) [<a href="#B1-molecules-25-03822">1</a>,<a href="#B2-molecules-25-03822">2</a>,<a href="#B3-molecules-25-03822">3</a>]. While most cases of this new disease show mild to moderate symptoms, a small fraction of cases, especially those with comorbid conditions like diabetes and hypertension, can develop fatal conditions such as acute respiratory distress syndrome (ARDS) due to severe lung damage [<a href="#B4-molecules-25-03822">4</a>]. In January 2020, a novel betacoronavirus, initially named 2019-nCoV, was discovered to be the etiological agent of this new disease [<a href="#B1-molecules-25-03822">1</a>,<a href="#B2-molecules-25-03822">2</a>,<a href="#B3-molecules-25-03822">3</a>]. Subsequently, human-to-human transmission of this disease was confirmed [<a href="#B4-molecules-25-03822">4</a>]. By 30 January 2020, the 2019-nCoV had spread to more than 20 countries and the World Health Organization (WHO) declared a public health emergency of international concern. On 11 February 2020, the international committee on taxonomy of viruses permanently named 2019-nCoV as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and the WHO named the associated disease as coronavirus disease 2019 (COVID-19). By 11 March 2020, COVID-19 had spread to more than 150 countries across six continents and the WHO upgraded the status of the epidemic to pandemic. As of 27 July 2020, the number of laboratory confirmed COVID-19 cases and deaths have already surpassed 16 million and 650,000, respectively, with the worst affected countries as USA, Brazil, UK, Mexico, Italy, India, France and Spain (<span class="uri" title="uri"><a href="https://www.worldometers.info/coronavirus/">https://www.worldometers.info/coronavirus/</a></span>). In short, the COVID-19 pandemic is an unprecedented public health and economic emergency for humankind.</p> <p>Coronaviruses are enveloped, positive-sense, single-stranded RNA viruses with large viral genomes. The publication of SARS-CoV-2 genome in January 2020 led to its taxonomic classification into the family <i>Coronaviridae</i> and genus <i>Betacoronavirus</i> [<a href="#B1-molecules-25-03822">1</a>,<a href="#B2-molecules-25-03822">2</a>,<a href="#B3-molecules-25-03822">3</a>]. Bats are natural reservoirs of coronaviruses [<a href="#B5-molecules-25-03822">5</a>]. The SARS-CoV-2 genome shares 96% nucleotide identity with bat coronavirus RaTG13, which suggests a probable zoonotic transfer to humans via an intermediate animal host [<a href="#B6-molecules-25-03822">6</a>]. Prior to SARS-CoV-2 epidemic, there are two precedence of zoonotic transfer of <i>betacoronaviruses</i> to humans, namely the severe acute respiratory syndrome coronavirus (SARS-CoV) and the middle east respiratory syndrome coronavirus (MERS-CoV), which had also led to outbreaks of severe respiratory disease [<a href="#B7-molecules-25-03822">7</a>]. In 2002â€"2003, SARS-CoV emerged in China and led to 8098 infections and 774 deaths across the world [<a href="#B7-molecules-25-03822">7</a>]. Interestingly, the SARS-CoV-2 genome shares ~80% nucleotide identity with SARS-CoV [<a href="#B1-molecules-25-03822">1</a>,<a href="#B2-molecules-25-03822">2</a>,<a href="#B3-molecules-25-03822">3</a>]. In 2012, MERS-CoV emerged in Saudi Arabia and led to 2521 infections and 866 deaths that were largely limited to Middle Eastern countries [<a href="#B7-molecules-25-03822">7</a>]. Unlike SARS and MERS, the geographic spread of COVID-19 and the ensuing mortality is significantly higher. To date, there are no approved antiviral drugs or vaccines against <i>betacoronavirus</i> infections including COVID-19 [<a href="#B8-molecules-25-03822">8</a>]. Hence, the current measures to contain COVID-19 include social distancing, aggressive testing, patient isolation, contact tracing and travel restrictions. In present circumstances, an immediate goal of biomedical research is to develop antivirals or anti-COVID therapeutics for SARS-CoV-2 [<a href="#B8-molecules-25-03822">8</a>].</p> <p>The SARS-CoV-2 genome comprises ~30,000 bases with 14 open reading frames (ORFs) coding for 27 proteins [<a href="#B9-molecules-25-03822">9</a>]. The genome organization of SARS-CoV-2 is similar to other <i>betacoronaviruses</i> with the 5â€²-region coding for non-structural proteins and the 3â€²-region coding for structural proteins. Important structural proteins of SARS-CoV-2 coded by the 3â€²-region include the spike (S) surface glycoprotein, the envelope (E) protein, the matrix (M) protein and the nucleocapsid (N) protein [<a href="#B9-molecules-25-03822">9</a>]. The 5â€²-region of the SARS-CoV-2 genome contains the replicase gene which codes for two overlapping polyproteins, pp1a and pp1ab, which are proteolytically cleaved by two important non-structural proteins, 3-chymotrypsin like protease (3CL<sup>pro</sup>) and papain-like protease (PL<sup>pro</sup>), to produce functional (non-structural) proteins. Other important non-structural proteins of SARS-CoV-2 for the viral life cycle include the RNA-dependent RNA polymerase (RdRp) and helicase. Accordingly, the 4 non-structural proteins, 3CL<sup>pro</sup>, PL<sup>pro</sup>, RdRp and helicase, along with the spike glycoprotein of SARS-CoV-2 are among the most attractive targets for anti-COVID drugs [<a href="#B8-molecules-25-03822">8</a>].</p> <p>Rather than targeting important SARS-CoV-2 proteins for viral life cycle, an alternative approach to anti-COVID drugs involves targeting host factors key to SARS-CoV-2 infection [<a href="#B8-molecules-25-03822">8</a>]. For host cell entry, SARS-CoV-2 employs the spike (S) protein whose S1 subunit has a receptor binding domain (RBD) that specifically recognizes the cell surface receptor angiotensin converting enzyme 2 (ACE2) [<a href="#B10-molecules-25-03822">10</a>,<a href="#B11-molecules-25-03822">11</a>,<a href="#B12-molecules-25-03822">12</a>,<a href="#B13-molecules-25-03822">13</a>,<a href="#B14-molecules-25-03822">14</a>]. Notably, both SARS-CoV-2 and SARS-CoV employ ACE2 as the cell entry receptor [<a href="#B10-molecules-25-03822">10</a>,<a href="#B11-molecules-25-03822">11</a>,<a href="#B12-molecules-25-03822">12</a>,<a href="#B13-molecules-25-03822">13</a>,<a href="#B14-molecules-25-03822">14</a>]. After attachment of S protein to ACE2 receptor, the membrane fusion of virus and host cell depends on proteolytic activation of S protein by host proteases which involves cleavage of S1 subunit at S1/S2 and S2â€² sites resulting in dissociation of S1 subunit and structural change in S2 subunit of S protein [<a href="#B10-molecules-25-03822">10</a>,<a href="#B11-molecules-25-03822">11</a>,<a href="#B12-molecules-25-03822">12</a>,<a href="#B13-molecules-25-03822">13</a>,<a href="#B14-molecules-25-03822">14</a>]. Hoffmann et al. [<a href="#B10-molecules-25-03822">10</a>] showed that the host cell proteases, Transmembrane Protease Serine 2 (TMPRSS2) and cathepsin L or cathepsin B, can carry out S protein priming required for SARS-CoV-2 entry. Hoffmann et al. [<a href="#B10-molecules-25-03822">10</a>] also showed that TMPRSS2 is more essential for S protein priming and SARS-CoV-2 entry. In parallel, Ou et al. [<a href="#B11-molecules-25-03822">11</a>] used specific inhibitors of cathepsin L and cathepsin B to show that cathepsin L rather than cathepsin B is essential for S protein priming of SARS-CoV-2 and membrane fusion in lysosomes. These studies highlight at least two alternate pathways for host cell entry of SARS-CoV-2. On the one hand, after SARS-CoV-2 attachment to ACE2, the membrane fusion and cytoplasmic entry can occur at the plasma membrane provided the cell surface protease TMPRSS2 is available to carry out S protein priming. On the other hand, after SARS-CoV-2 attachment to ACE2, the virus can be internalized as part of endosomes in the endocytic pathway, and later, the membrane fusion and cytoplasmic entry will occur in lysosomes provided the lysosomal protease cathepsin L is available to carry out S protein priming [<a href="#B10-molecules-25-03822">10</a>,<a href="#B11-molecules-25-03822">11</a>,<a href="#B12-molecules-25-03822">12</a>]. Depending on the target cell and associated expression of host cell proteases, SARS-CoV-2 may use one of the alternative pathways for host cell entry. Importantly, the above-mentioned studies also showed that known inhibitors camostat mesylate and nafamostat mesylate of TMPRSS2 and E-64d and PC-0626568 (SID26681509) of cathepsin L can block or significantly reduce the host cell entry of SARS-CoV-2 [<a href="#B10-molecules-25-03822">10</a>,<a href="#B11-molecules-25-03822">11</a>,<a href="#B12-molecules-25-03822">12</a>]. In conclusion, human proteases TMPRSS2 and cathepsin L are key factors for host cell entry and are important targets for anti-COVID drugs [<a href="#B10-molecules-25-03822">10</a>,<a href="#B11-molecules-25-03822">11</a>,<a href="#B12-molecules-25-03822">12</a>,<a href="#B15-molecules-25-03822">15</a>].</p> <p>To expedite this search for anti-COVID drugs, several computational studies have used homology modeling or published crystal structures of SARS-CoV-2 proteins, molecular docking and diverse small molecule libraries, to predict potential inhibitors of SARS-CoV-2 proteins including among existing approved drugs for repurposing and natural compounds (see e.g., [<a href="#B16-molecules-25-03822">16</a>,<a href="#B17-molecules-25-03822">17</a>,<a href="#B18-molecules-25-03822">18</a>,<a href="#B19-molecules-25-03822">19</a>,<a href="#B20-molecules-25-03822">20</a>,<a href="#B21-molecules-25-03822">21</a>,<a href="#B22-molecules-25-03822">22</a>,<a href="#B23-molecules-25-03822">23</a>,<a href="#B24-molecules-25-03822">24</a>]). In comparison, fewer computational studies [<a href="#B16-molecules-25-03822">16</a>,<a href="#B19-molecules-25-03822">19</a>] have focussed on identification of potential inhibitors of host factors. In this work, we perform virtual screening of a large phytochemical library specific to Indian medicinal plants to identify potential natural product inhibitors of TMPRSS2 and cathepsin L.</p> <p>Plant-based natural products have made immense contributions to drug discovery [<a href="#B25-molecules-25-03822">25</a>]. Specifically, ~40% of the small-molecule drugs approved to date by the US Food and Drug Administration (FDA) are either natural products or natural product derivatives. Recently, there are reports from China of successful use of traditional Chinese medicine and associated herbs in treatment of COVID-19 patients [<a href="#B26-molecules-25-03822">26</a>]. On similar lines, there have been suggestions to tap the rich legacy of traditional Indian medicine and information on phytochemicals of Indian herbs in the search for anti-COVID drugs [<a href="#B27-molecules-25-03822">27</a>]. Previously, some of us have built IMPPAT, the largest resource to date on phytochemicals of Indian herbs [<a href="#B28-molecules-25-03822">28</a>]. In this work, we perform molecular docking using a large library of 14,011 phytochemicals compiled mainly from IMPPAT to identify potential natural product inhibitors of TMPRSS2 and cathepsin L. Subsequently, we have also performed molecular dynamics (MD) simulations to investigate the stability of protein-ligand complexes for the top phytochemical inhibitors of TMPRSS2 and cathepsin L identified in this work.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
2. Results</div> <div class="2.1.workflowforvirtualscreening" title="sec"> <div class="title" tagx="title" title="title">
2.1. Workflow for Virtual Screening</div> <p>This computational study aims to predict potential phytochemical inhibitors of human proteases, TMPRSS2 and cathepsin L, that are important for priming of S protein and cell entry of SARS-CoV-2 [<a href="#B10-molecules-25-03822">10</a>,<a href="#B11-molecules-25-03822">11</a>,<a href="#B12-molecules-25-03822">12</a>]. Briefly, the workflow for this virtual screening is as follows (<a href="#molecules-25-03822-f001">Figure 1</a>).</p> <p>In the first stage, we prepared the ligands for molecular docking with target proteases. We compiled a library of 14,011 phytochemicals produced by medicinal plants used in traditional Indian medicine, and the main source of this compilation was IMPPAT [<a href="#B28-molecules-25-03822">28</a>], the largest resource on phytochemicals of Indian herbs to date (see <a href="#sec3-molecules-25-03822">Section 3</a>). Next, the standard drug-likeness measure, Lipinskiâ€™s rule of five (RO5) [<a href="#B29-molecules-25-03822">29</a>], was used to filter a subset of 10,510 drug-like phytochemicals. Next, the filtered phytochemicals were prepared for docking by retrieving their 3D structures from Pubchem [<a href="#B30-molecules-25-03822">30</a>] followed by energy-minimization using OpenBabel [<a href="#B31-molecules-25-03822">31</a>] (<a href="#sec3-molecules-25-03822">Section 3</a>).</p> <p>In the second stage, we prepared the target proteins for docking with prepared ligands. For TMPRSS2, the 3D structure is yet to be determined experimentally, and thus, we built a homology model of TMPRSS2 using SWISS-MODEL [<a href="#B32-molecules-25-03822">32</a>] which was used for docking after energy-minimization using UCSF Chimera [<a href="#B33-molecules-25-03822">33</a>] (<a href="#molecules-25-03822-f002">Figure 2</a>; <a href="#sec3-molecules-25-03822">Section 3</a>). <a href="#molecules-25-03822-f002">Figure 2</a> displays the TMPRSS2 model structure with the catalytic triad S441, H296 and D345 and the substrate binding residue D435 in the S1 subsite. For cathepsin L, the crystal structure (PDB 5MQY) with 1.13 Ã… resolution was used for docking after energy-minimization using UCSF Chimera (<a href="#molecules-25-03822-f003">Figure 3</a>; <a href="#sec3-molecules-25-03822">Section 3</a>). <a href="#molecules-25-03822-f003">Figure 3</a> displays the cathepsin L structure with the catalytic dyad C25 and H163 in S1 subsite, and other important residues in S2 and S1â€² subsites.</p> <p>In the third stage, we performed protein-ligand docking using AutoDock Vina [<a href="#B34-molecules-25-03822">34</a>]. For protein-ligand docking, an appropriate grid box was manually determined for TMPRSS2 and cathepsin L (<a href="#sec3-molecules-25-03822">Section 3</a>). To decide on a stringent binding energy cut off for the identification of potential inhibitors, docking was first performed for known inhibitors of target proteins (<a href="#sec3-molecules-25-03822">Section 3</a>). Based on the docking binding energies of the known inhibitors, camostat and nafamostat, to TMPRSS2, we decided on a stringent criteria of binding energy â‰¤âˆ’8.5 kcal/mol for the best docked pose of screened ligands to identify potential inhibitors of TMPRSS2 (<a href="#sec3-molecules-25-03822">Section 3</a>). Similarly, based on the docking binding energies of the known inhibitors, E-64d and PC-0626568, to cathepsin L, we decided on a stringent criteria of binding energy â‰¤âˆ’8.0 kcal/mol for the best docked pose of screened ligands to identify potential inhibitors of cathepsin L (<a href="#sec3-molecules-25-03822">Section 3</a>). Thereafter, docking was performed for the prepared ligands in the phytochemical library against the prepared structures of TMPRSS2 and cathepsin L (<a href="#sec3-molecules-25-03822">Section 3</a>). Lastly, we filtered the subset of phytochemicals whose binding energy in the best docked pose with TMPRSS2 (respectively, cathepsin L) is â‰¤âˆ’8.5 kcal/mol (respectively, â‰¤âˆ’8.0 kcal/mol). Moreover, the best docked pose with TMPRSS2 or cathepsin L of each filtered phytochemical was separated from AutoDock Vina output file, and then, combined with the target protein structure to obtain the docked protein-ligand complex in .pdb format (<a href="#sec3-molecules-25-03822">Section 3</a>). At the end of third stage, we obtained 101 phytochemicals whose binding energy in the best docked pose with TMPRSS2 is â‰¤âˆ’8.5 kcal/mol and 16 phytochemicals whose binding energy in the best docked pose with cathepsin L is â‰¤âˆ’8.0 kcal/mol.</p> <p>In the fourth stage, the structure of docked protein-ligand complex in .pdb format for each filtered phytochemical from third stage was used to determine the ligand binding site residues in the target protein and different non-covalent interactions such as hydrogen bond, halogen bond, hydrophobic interactions, etc. between ligand and target protein (<a href="#sec3-molecules-25-03822">Section 3</a>). In case of TMPRSS2, the specificity of this trypsin-like protease is determined by the conserved substrate binding residue D435 in the S1 pocket [<a href="#B35-molecules-25-03822">35</a>] (<a href="#sec3-molecules-25-03822">Section 3</a>), and therefore, a potent inhibitor should either bind to or form non-covalent interactions with D435. In case of cathepsin L, the specificity of this cysteine protease is dependent on the catalytic residues, C25 and H163, in the S1 subsite (<a href="#sec3-molecules-25-03822">Section 3</a>), and therefore, a potent inhibitor should either bind to or form non-covalent interactions with the catalytic residues. In this work, we consider a phytochemical to be a potential inhibitor of TMPRSS2 (respectively, cathepsin L) only if the ligand binding energy in the best docked pose is â‰¤âˆ’8.5 kcal/mol (respectively, â‰¤âˆ’8.0 kcal/mol) and the ligand binds to or forms non-covalent interactions with the residue D435 in TMPRSS2 (respectively, residues C25 and H163 in cathepsin L).</p> <p>At the end of fourth stage, we obtained 96 phytochemicals (labelled T1â€"T96; Figure 5; <a href="#app1-molecules-25-03822">Supplementary Table S1</a>) as potential inhibitors of TMPRSS2 and 9 phytochemicals (labelled C1â€"C9; Figure 6; <a href="#app1-molecules-25-03822">Supplementary Table S2</a>) as potential inhibitors of cathepsin L. Using IMPPAT [<a href="#B28-molecules-25-03822">28</a>], we provide a list of Indian medicinal plants that can produce the identified phytochemical inhibitors of TMPRSS2 and cathepsin L (<a href="#molecules-25-03822-t001">Table 1</a> and <a href="#molecules-25-03822-t002">Table 2</a>; <a href="#app1-molecules-25-03822">Supplementary Table S3</a>). Furthermore, we have also compiled information on potential antiviral or anti-inflammatory use in traditional medicine of the herbal sources of the identified phytochemical inhibitors of TMPRSS2 and cathepsin L (<a href="#molecules-25-03822-t001">Table 1</a> and <a href="#molecules-25-03822-t002">Table 2</a>; <a href="#app1-molecules-25-03822">Supplementary Table S3</a>). In <a href="#app1-molecules-25-03822">Supplementary Tables S4 and S5</a>, we list the ligand binding site residues and non-covalent protein-ligand interactions for the identified phytochemical inhibitors of TMPRSS2 and cathepsin L. We have also predicted the physicochemical and ADMET properties of the identified phytochemical inhibitors of TMPRSS2 and cathepsin L (<a href="#sec3-molecules-25-03822">Section 3</a>; <a href="#app1-molecules-25-03822">Supplementary Tables S6 and S7</a>).</p> <p>Finally, in the fifth stage, for the top three inhibitors of TMPRSS2 namely, T1 (qingdainone), T2 (edgeworoside C) and T3 (adlumidine), and of cathepsin L namely, C1 (ararobinol), C2 ((+)-oxoturkiyenine) and C3 (3Î±,17Î±-cinchophylline), their respective protein-ligand complexes were analyzed using 180 ns MD simulation (<a href="#sec3-molecules-25-03822">Section 3</a>; Figures 8 and 9; <a href="#app1-molecules-25-03822">Supplementary Figures S1 and S2</a>) and their interaction binding energy was computed using MM-PBSA method (<a href="#sec3-molecules-25-03822">Section 3</a>; <a href="#molecules-25-03822-t003">Table 3</a>).</p> </div> <div class="2.2.potentialphytochemicalinhibitorsoftmprss2" title="sec"> <div class="title" tagx="title" title="title">
2.2. Potential Phytochemical Inhibitors of TMPRSS2</div> <p>As mentioned above, we have identified 96 potential natural product inhibitors of TMPRSS2 by computational screening of 14,011 phytochemicals produced by Indian medicinal plants, and these 96 compounds labelled T1-T96 are listed in <a href="#app1-molecules-25-03822">Supplementary Table S1</a> along with their PubChem identifier, common name, IUPAC name and structure in SMILES format. In this section, we provide a detailed discussion of the top nine phytochemical inhibitors (labelled as T1â€"T9) whose binding energies in the best docked poses with TMPRSS2 are â‰¤âˆ’9.2 kcal/mol. <a href="#molecules-25-03822-f004">Figure 4</a> displays the structure of these top 9 phytochemical inhibitors of TMPRSS2 and <a href="#molecules-25-03822-t001">Table 1</a> provides a list of Indian medicinal plants that can produce them.</p> <p>Phytochemical T1, qingdainone, has a binding energy of âˆ’9.6 kcal/mol. T1 is a quinazoline alkaloid produced by <i>Strobilanthes cusia</i>, a herb with antiviral activity [<a href="#B36-molecules-25-03822">36</a>]. <a href="#molecules-25-03822-f005">Figure 5</a>a shows the TMPRSS2 residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with T1. TMPRSS2 residue D440 forms C-Hâ‹¯O type hydrogen bond with T1 whereas residue A399 forms C-Hâ‹¯N type hydrogen bond with T1. Further, T1 forms hydrophobic contacts with residues I381, S382, T387, E388, N398, A400, D440, C465 and A466.</p> <p>Phytochemical T2, edgeworoside C, also has a binding energy of âˆ’9.6 kcal/mol. T2 is a coumarin produced by <i>Edgeworthia gardneri</i>, a medicinal plant consumed as a herbal tea in Tibet [<a href="#B37-molecules-25-03822">37</a>]. In traditional medicine, <i>Edgeworthia gardneri</i> has been used to treat metabolic disorders including diabetes [<a href="#B38-molecules-25-03822">38</a>,<a href="#B39-molecules-25-03822">39</a>]. <a href="#molecules-25-03822-f005">Figure 5</a>b shows the TMPRSS2 residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with T2. T2 forms 12 hydrogen bonds with residues A386, N398, A399, V434, D435 and D440 of TMPRSS2. The phenolic hydroxyl group of T2 acts as both acceptor and donor forming O-Hâ‹¯O and N-Hâ‹¯O type hydrogen bonds with the substrate binding residue D435 and C-Hâ‹¯O type hydrogen bond with residue V434. The hydroxyl groups attached to the pyran ring of T2 form hydrogen bonds with residues A386, N398 and D440. Further, T2 forms hydrophobic contacts with residues E260, I381, A400, N433, and A466.</p> <p>Phytochemical T3, adlumidine, also has a binding energy of âˆ’9.6 kcal/mol. T3 is produced by <i>Fumaria indica</i>, a herb used in traditional medicine to treat fever, cough, skin ailments and urinary diseases [<a href="#B40-molecules-25-03822">40</a>]. Adlumidine has also been reported to be an inhibitor of GABA receptor [<a href="#B41-molecules-25-03822">41</a>]. <a href="#molecules-25-03822-f005">Figure 5</a>c shows the TMPRSS2 residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with T3. The two 1,3-dioxole groups present in T3 facilitate the formation of an extensive hydrogen bond network with E388, E389, S436, C465 and A466. T3 also forms C-Hâ‹¯S type hydrogen bond [<a href="#B42-molecules-25-03822">42</a>] with C437. Further, T3 forms hydrophobic contacts with residues E260, I381, S382, T387, N398, A399 and A400.</p> <p>Phytochemicals T4 (pseudo-Î±-colubrine), T6 (strychnine <i>N</i>-oxide), T7 (Î±-colubrine) and T9 (2-hydroxy-3-methoxystrychnine) have binding energies of âˆ’9.3 kcal/mol, âˆ’9.3 kcal/mol, âˆ’9.2 kcal/mol and âˆ’9.2 kcal/mol, respectively. These four phytochemicals are monoterpenoid indole alkaloids produced by <i>Strychnos nux-vomica</i>. The herb <i>Strychnos nux-vomica</i> is used in traditional Indian medicine and its alkaloids have been shown to exhibit anti-inflammatory, anti-oxidant, antitumor and hepatoprotective activities [<a href="#B43-molecules-25-03822">43</a>]. Note that <i>Strychnos nux-vomica</i> is a poisonous plant whose seeds are extensively used in Ayurveda only after proper detoxification procedure called <i>Shodhana</i> described in Ayurvedic texts [<a href="#B43-molecules-25-03822">43</a>]. <a href="#molecules-25-03822-f005">Figure 5</a>d shows the TMPRSS2 residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with T4. T4 forms C-Hâ‹¯O type hydrogen bonds with residues A400, N433, D435 (substrate binding residue), C437 and D440. Further, T4 forms hydrophobic contacts with residues E260, I381, S382, T387, N398, A399, V434, D440 and A466. <a href="#molecules-25-03822-f005">Figure 5</a>f shows the TMPRSS2 residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with T6. T6 forms a C-Hâ‹¯N type hydrogen bond with residue N398. The substrate binding residue D435 also forms a C-Hâ‹¯O type hydrogen bond with T6. Further, T6 forms hydrophobic contacts with residues N398, A400, V434 and A466. <a href="#app1-molecules-25-03822">Supplementary Figure S3a</a> shows the TMPRSS2 residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with T7. T7 forms five C-Hâ‹¯O type hydrogen bonds with residues N433, D435 (substrate binding residue), C437 and D440. Further, T7 forms hydrophobic contacts with residues E260, T387, N398, A399, A400, V434 and A466. <a href="#app1-molecules-25-03822">Supplementary Figure S3c</a> shows the TMPRSS2 residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with T9. The phenolic hydroxyl group of T9 forms hydrogen bonds with residues S382 and A399. The substrate binding site D435 also forms a C-Hâ‹¯O type hydrogen bond with T9. Further, T9 forms hydrophobic contacts with residues E260, N398, A399, A400 and V434.</p> <p>Phytochemical T5, bicuculline, has a binding energy of âˆ’9.3 kcal/mol, and it is a stereoisomer of T3. T5 is an isoquinoline alkaloid and is produced by herbs <i>Corydalis govaniana</i>, <i>Nerium oleander</i> and <i>Fumaria indica</i>. Bicuculline has also been reported to be a GABA receptor inhibitor [<a href="#B44-molecules-25-03822">44</a>]. <a href="#molecules-25-03822-f005">Figure 5</a>e shows the TMPRSS2 residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with T5. The two 1,3-dioxole groups present in T5 facilitate the formation of an extensive hydrogen bond network with residues E388, E389 and A400. The Furan-2-one ring also forms a C-Hâ‹¯O type hydrogen bond with C437. Further, T5 forms hydrophobic contacts with residues E260, T387, E388, N398, A399 and A466.</p> <p>Phytochemical T8, egenine, has a binding energy of âˆ’9.2 kcal/mol. T8 is an isoquinoline alkaloid produced by <i>Fumaria vaillantii</i>. In traditional medicine, <i>Fumaria vaillantii</i> has been reported to exhibit antifungal, anti-inflammatory and anti-psychotic activities [<a href="#B45-molecules-25-03822">45</a>]. <a href="#app1-molecules-25-03822">Supplementary Figure S3b</a> shows the TMPRSS2 residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with T8. The two 1,3-dioxole groups present in T8 form hydrogen bonds with G383, E388, E389 and A400. One of the hydroxyl groups present in T8 forms C-Hâ‹¯O type hydrogen bond with residue C437. Further, T8 forms hydrophobic contacts with residues T387, A399, E388, N398, E260, and A466.</p> </div> <div class="2.3.potentialphytochemicalinhibitorsofcathepsinl" title="sec"> <div class="title" tagx="title" title="title">
2.3. Potential Phytochemical Inhibitors of Cathepsin L</div> <p>As mentioned above, we have identified nine potential natural product inhibitors of cathepsin L by computational screening of 14,011 phytochemicals produced by Indian medicinal plants, and these compounds labelled C1â€"C9 are listed in <a href="#app1-molecules-25-03822">Supplementary Table S2</a> along with their PubChem identifier, common name, IUPAC name and structure in SMILES format. <a href="#molecules-25-03822-f006">Figure 6</a> displays the structure of these top nine phytochemical inhibitors of cathepsin L and <a href="#molecules-25-03822-t002">Table 2</a> provides a list of Indian medicinal plants that produce them.</p> <p>Phytochemical C1, ararobinol, has a binding energy of âˆ’8.9 kcal/mol. C1 is a bianthraquinone produced by herb <i>Senna occidentalis</i> used in Ayurveda. In traditional medicine, <i>Senna occidentalis</i> has been reported for antibacterial, antifungal, anti-inflammatory, anti-diabetic and anti-cancer activities [<a href="#B46-molecules-25-03822">46</a>]. Interestingly, there is a Chinese patent application [<a href="#B47-molecules-25-03822">47</a>] on potential use of ararobinol to treat human influenza virus infections, however, this suggests only a potential antiviral activity of C1 not specific to SARS-CoV-2 which further needs to be verified through in vitro and in vivo experimental studies. <a href="#molecules-25-03822-f007">Figure 7</a>a shows the cathepsin L residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with C1. Residues Q19 and A138 form C-Hâ‹¯N and C-Hâ‹¯O type of hydrogen bonds, respectively, with C1. Also, the residue W189 forms both face-to-edge and face-to-face type of Ï€-Ï€ stacking interaction with C1. Further, C1 forms hydrophobic contacts with residues C25, G139, L144, H163 and W189.</p> <p>Phytochemical C2, (+)-oxoturkiyenine, has a binding energy of âˆ’8.3 kcal/mol. C2 is an isoquinoline-derived alkaloid produced by the herb <i>Hypecoum pendulum</i> [<a href="#B48-molecules-25-03822">48</a>]. <a href="#molecules-25-03822-f007">Figure 7</a>b shows the cathepsin L residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with C2. The 2,5-dihydro-furan ring present in C2 forms two N-Hâ‹¯O type hydrogen bonds with residue Q19 and W189. The catalytic residue H163 forms N-Hâ‹¯O type hydrogen bond with C2. Also, the residue W189 forms a face-to-edge type of Ï€-Ï€ stacking interaction with C2. Further, C2 forms hydrophobic contacts with residues G139, H140, H163 and W189.</p> <p>Phytochemical C3, 3Î±,17Î±-cinchophylline, has a binding energy of âˆ’8.3 kcal/mol. C3 is a cinchophylline-type of alkaloid produced by the herb <i>Cinchona calisaya</i>. The <i>Cinchona</i> alkaloids have been reported for their antimicrobial, antiparasitic and anti-inflammatory activities [<a href="#B49-molecules-25-03822">49</a>]. <a href="#molecules-25-03822-f007">Figure 7</a>c shows the cathepsin L residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with C3. C3 forms 8 hydrogen bonds with residues of cathepsin L. One of the catalytic residue C25 forms C-Hâ‹¯S and N-Hâ‹¯S type hydrogen bonds with C3. The other catalytic residue H163 forms C-Hâ‹¯N and N-Hâ‹¯N type hydrogen bonds with C3. Further, one of the two pyrrole rings present in C3 forms hydrogen bond with residue G23. Lastly, M70 forms a C-Hâ‹¯S type hydrogen bond with C3. Further, C3 forms hydrophobic contacts with residues Q21, C22, L69, M70, A135 and W189.</p> <p>Phytochemical C4, rugosanine B, has a binding energy of âˆ’8.2 kcal/mol. C4 is a cyclopeptide alkaloid produced by the bark of <i>Ziziphus rugosa</i> [<a href="#B50-molecules-25-03822">50</a>]. Various parts of <i>Ziziphus rugosa</i> have been reported for their antibacterial, antifungal, anti-inflammatory and analgesic activities [<a href="#B51-molecules-25-03822">51</a>]. <a href="#molecules-25-03822-f007">Figure 7</a>d shows the cathepsin L residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with C4. The pyrrole ring present in C4 forms a N-Hâ‹¯O type hydrogen bond with residue D162. Moreover, C4 forms C-Hâ‹¯O type hydrogen bonds with A138, D162 and L69. Also, the residue W189 forms a face-to-edge type of Ï€-Ï€ stacking interaction with C4. Further, C4 forms hydrophobic contacts with residues G23, C25, G67, M70, A135, A138, D162, H163, G164, W189 and A214.</p> <p>Phytochemical C5, trichotomine, has a binding energy of âˆ’8.2 kcal/mol. C5 is a bisindole alkaloid present in <i>Clerodendrum trichotomum</i>. <i>Clerodendrum trichotomum</i> has been reported for its use to treat cold, hypertension, rheumatism, dysentery and other inflammatory diseases [<a href="#B52-molecules-25-03822">52</a>]. <a href="#molecules-25-03822-f007">Figure 7</a>e shows the cathepsin L residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with C5. The carboxylic acid group present in C5 forms hydrogen bonds with residues Q19 and H163. The indole ring of C5 forms a N-Hâ‹¯N type hydrogen bond with residue G68. Further, C5 forms hydrophobic contacts with residues G23, C25, G67, G68, L69 and Y72.</p> <p>Phytochemical C6, tectol, has binding energy of âˆ’8.1 kcal/mol. C6 is a naphthoquinone derivative [<a href="#B53-molecules-25-03822">53</a>] present in <i>Tectona grandis</i> and <i>Tecomella undulata</i>. <i>Tectona grandis</i> has been reported to have anti-inflammatory and antiparasitic activities [<a href="#B45-molecules-25-03822">45</a>]. <i>Tecomella undulata</i> has been used to treat syphilis and also reported to have anti-inflammatory and anti-HIV activities [<a href="#B54-molecules-25-03822">54</a>]. Additionally, Tectol has been reported to inhibit farnesyltransferase enzyme [<a href="#B55-molecules-25-03822">55</a>]. <a href="#molecules-25-03822-f007">Figure 7</a>f shows the cathepsin L residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with C6. The pyran group of C6 is involved in a C-Hâ‹¯O type hydrogen bond with residue L144. The catalytic residue C25 forms a C-Hâ‹¯S type hydrogen bond with C6. The other catalytic residue H163 forms a N-Hâ‹¯O type hydrogen bond with C6. Also, the residue W189 forms both face-to-face and face-to-edge type of Ï€-Ï€ stacking interaction with C6. Further, C6 forms hydrophobic contacts with G23, L144 and W189.</p> <p>Phytochemical C7, silymonin, has a binding energy of âˆ’8.1 kcal/mol. C7 is a flavanolignan [<a href="#B56-molecules-25-03822">56</a>] present in <i>Silybum marianum</i>. <i>Silybum marianum</i> has been used as a hepatoprotective agent and is reported to have anti-oxidant and anti-inflammatory activities [<a href="#B57-molecules-25-03822">57</a>]. <a href="#app1-molecules-25-03822">Supplementary Figure S4a</a> shows the cathepsin L residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with C7. C7 has four hydroxyl groups which help in the formation of an extensive network of hydrogen bonds with residues Q21, A138 and G139. C7 also forms two N-Hâ‹¯O type hydrogen bonds with H163 and W189. Also, the residue W189 forms a face-to-edge type of Ï€-Ï€ stacking interaction with C7. Further, C7 forms hydrophobic contacts with residues G23, A138, L144, H163 and W189.</p> <p>Phytochemical C8, picrasidine M, has a binding energy of âˆ’8.0 kcal/mol. C8 is a Î²-carboline alkaloid present in <i>Picrasma quassioides</i>. <i>Picrasma quassioides</i> has been reported to have antiviral and antifungal activities [<a href="#B28-molecules-25-03822">28</a>]. <a href="#app1-molecules-25-03822">Supplementary Figure S4b</a> shows the cathepsin L residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with C8. The carboxylic group of residue D162 forms two C-Hâ‹¯O type hydrogen bonds with C8. Also, residues M70 and G23 form hydrogen bonds of type C-Hâ‹¯S and C-Hâ‹¯O, respectively, with C8. Also, the residue W189 forms a face-to-edge type of Ï€-Ï€ stacking interaction with C8. Further, C8 forms hydrophobic contacts with residues G23, L69, D162 and W189.</p> <p>Phytochemical C9, trisjuglone, has a binding energy of âˆ’8.0 kcal/mol. C9 is a naphthoquinone present in <i>Juglans regia</i> (i.e., walnut). In traditional medicine, <i>Juglans regia</i> has been reported to have anti-inflammatory, antifungal and antimicrobial activities [<a href="#B45-molecules-25-03822">45</a>]. <a href="#app1-molecules-25-03822">Supplementary Figure S4c</a> shows the cathepsin L residues that form hydrogen bonds or Ï€-Ï€ stacking interactions with C9. The benzoquinone moiety present in C9 forms two C-Hâ‹¯O type hydrogen bonds with residue Q21 and one N-Hâ‹¯O type hydrogen bond with W189. In contrast, the other catalytic residue H163 forms a N-Hâ‹¯O type hydrogen bond with C9. Also, the residue W189 forms a face-to-edge type of Ï€-Ï€ stacking interaction with C9. Further, C9 forms hydrophobic contacts with residues Q21, G23, C25, L144 and W189.</p> </div> <div class="2.4.moleculardynamicssimulationoftopinhibitors" title="sec"> <div class="title" tagx="title" title="title">
2.4. Molecular Dynamics Simulation of Top Inhibitors</div> <p>In order to investigate the stability of the protein-ligand complexes of the identified inhibitors, MD simulation of 180 ns was performed for top three inhibitors of TMPRSS2 namely, qingdainone (T1), edgeworoside C (T2) and adlumidine (T3), and of cathepsin L namely, ararobinol (C1), (+)-oxoturkiyenine (C2) and 3Î±,17Î±-cinchophylline (C3). Specifically, we have performed six 180 ns MD runs for protein-ligand complexes (TMPRSS2-T1, TMPRSS2-T2, TMPRSS2-T3, cathepsin L-C1, cathepsin L-C2 and cathepsin L-C3) and two 180 ns MD runs for free TMPRSS2 and cathepsin L proteins (<a href="#sec3-molecules-25-03822">Section 3</a>). To access the stability of the six protein-ligand complexes, we have computed radius of gyration (R<sub>g</sub>) of the protein, root mean square deviation (RMSD) of the CÎ± atoms of the protein, root mean square fluctuations (RMSF) of the CÎ± atoms of the protein, RMSD of the ligand and finally distance of the center of mass of the ligand from the center of mass of the catalytic residues or substrate binding residues of the protein in complex with the ligand (<a href="#molecules-25-03822-f008">Figure 8</a> and <a href="#molecules-25-03822-f009">Figure 9</a>).</p> <p>The R<sub>g</sub> value of TMPRSS2 in complex with T1, T2 and T3 remains largely stable throughout the MD simulation (<a href="#molecules-25-03822-f008">Figure 8</a>a). This implies that the top inhibitors of TMPRSS2 namely, T1, T2 and T3 do not induce any major structural changes to TMPRSS2 and TMPRSS2 remains structurally stable in complex with these inhibitors. TMPRSS2 in complex with T1, T2 and T3 has an average R<sub>g</sub> value of 2.110 Â± 0.022 nm, 2.096 Â± 0.024 nm and 2.091 Â± 0.021 nm, respectively. Further, RMSD value of the CÎ± atoms of TMPRSS2 in complex with T1, T2 and T3 become stable after 80 ns (<a href="#molecules-25-03822-f008">Figure 8</a>b). Over the 80 ns to 180 ns time interval, TMPRSS2 in complex with T1, T2 and T3 has an average RMSD value of 0.525 Â± 0.023 nm, 0.501 Â± 0.016 nm and 0.634 Â± 0.018 nm, respectively. Lastly, <a href="#molecules-25-03822-f008">Figure 8</a>c shows the RMSF value per residue for TMPRSS2 in complex with T1, T2 and T3. R<sub>g</sub>, RMSD and RMSF values of TMPRSS2 in complex with T1, T2 and T3 closely follow R<sub>g</sub>, RMSD and RMSF values of TMPRSS2 free protein (<a href="#molecules-25-03822-f008">Figure 8</a>aâ€"c; <a href="#app1-molecules-25-03822">Supplementary Figure S1aâ€"c</a>). <a href="#app1-molecules-25-03822">Supplementary Figure S2aâ€"c</a> show the superimposed snapshots at 120 ns, 140 ns and 160 ns of TMPRSS2-T1, TMPRSS2-T2 and TMPRSS2-T3 complexes, respectively. To further quantify the stability of the inhibitors T1, T2 and T3 bound to TMPRSS2, we have computed the RMSD of T1, T2 and T3 (<a href="#molecules-25-03822-f008">Figure 8</a>d) and distance of the center of mass of T1, T2 and T3 from the center of mass of the substrate binding residue D435 in TMPRSS2 (<a href="#molecules-25-03822-f008">Figure 8</a>e). Both RMSD of T1, T2 and T3 bound with TMPRSS2 and distance of the center of mass of T1, T2 and T3 from the center of mass of the substrate binding residue D435 becomes largely stable after 100 ns of the MD simulation (<a href="#molecules-25-03822-f008">Figure 8</a>d,e).</p> <p>Also, R<sub>g</sub> value of cathepsin L in complex with C1, C2 and C3 is stable throughout the MD simulation implying C1, C2 and C3 do not induce any major structural changes to cathepsin L and cathepsin L remains structurally stable in complex with these inhibitors (<a href="#molecules-25-03822-f009">Figure 9</a>a). Cathepsin L in complex with C1, C2 and C3 has an average R<sub>g</sub> value of 1.700 Â± 0.010 nm, 1.706 Â± 0.007 nm and 1.705 Â± 0.008 nm, respectively. Similarly, RMSD value of the CÎ± atoms of cathepsin L in complex with C1, C2 and C3 become largely stable after 80 ns (<a href="#molecules-25-03822-f009">Figure 9</a>b). Over the 80 ns to 180 ns time interval, cathepsin L in complex with C1, C2 and C3 has an average RMSD value of 0.281 Â± 0.028 nm, 0.276 Â± 0.022 nm and 0.270 Â± 0.014 nm, respectively. <a href="#molecules-25-03822-f009">Figure 9</a>c shows the RMSF value per residue for cathepsin L in complex with C1, C2 and C3. As in the case of TMPRSS2, R<sub>g</sub>, RMSD and RMSF values of cathepsin L in complex with C1, C2 and C3 closely follow R<sub>g</sub>, RMSD and RMSF values of cathepsin L free protein (<a href="#molecules-25-03822-f009">Figure 9</a>aâ€"c; <a href="#app1-molecules-25-03822">Supplementary Figure S1dâ€"f</a>). <a href="#app1-molecules-25-03822">Supplementary Figure S2dâ€"f</a> show the superimposed snapshots at 120 ns, 140 ns and 160 ns of cathepsin L-C1, cathepsin L-C2 and cathepsin L-C3 complexes, respectively. In order to quantify the stability of the inhibitors C1, C2 and C3 bound to cathepsin L we have also computed the RMSD of C1, C2 and C3 (<a href="#molecules-25-03822-f009">Figure 9</a>d) and distance of the center of mass of C1, C2 and C3 from the center of mass of the catalytic residues C25 (<a href="#molecules-25-03822-f009">Figure 9</a>e) and H163 (<a href="#molecules-25-03822-f009">Figure 9</a>f) in cathepsin L.</p> <p>C1 has a largely stable RMSD after 120 ns of the MD simulation, C2 has the lowest and most stable RMSD in comparison with C1 and C3, and C3 shows a largely stable RMSD from 50 ns to 130 ns and from 150 ns to 170 ns of the MD simulation (<a href="#molecules-25-03822-f009">Figure 9</a>d). Distance of the center of mass of C1, C2 and C3 from the center of mass of the catalytic residues C25 and H163 in cathepsin L also remains largely consistent after 120 ns of the MD simulation (<a href="#molecules-25-03822-f009">Figure 9</a>e,f).</p> </div> <div class="2.5.mm-pbsabindingenergyoftopinhibitors" title="sec"> <div class="title" tagx="title" title="title">
2.5. MM-PBSA Binding Energy of Top Inhibitors</div> <p>Molecular Mechanics Poissonâ€"Boltzmann Surface Area (MM-PBSA) is a widely used method to compute the binding energy of small molecules with biological macromolecules such as proteins [<a href="#B58-molecules-25-03822">58</a>]. Notably, the protein-ligand binding energy obtained using MM-PBSA method has been found to be more accurate than that obtained from protein-ligand docking [<a href="#B58-molecules-25-03822">58</a>]. Therefore, we have computed the binding energy for the top three inhibitors of TMPRSS2 and cathepsin L using MM-PBSA method. From the 180 ns MD simulation of the six protein-ligand complexes (TMPRSS2-T1, TMPRSS2-T2, TMPRSS2-T3, cathepsin L-C1, cathepsin L-C2 and cathepsin L-C3), 80 snapshots were obtained between 100 ns to 180 ns of the simulation at an interval of 1 ns along the trajectory, and thereafter, the 80 snapshots were used to compute the binding energy using g_mmpbsa (<a href="#sec3-molecules-25-03822">Section 3</a>) [<a href="#B59-molecules-25-03822">59</a>,<a href="#B60-molecules-25-03822">60</a>]. The final binding energy is the sum of contributions from van der Waals, electrostatic, polar solvation, and solvent accessible surface area (SASA) energy components. The contribution of each of the above components to the binding energy of the top inhibitors is shown in <a href="#molecules-25-03822-t003">Table 3</a>.</p> <p>Although T1, T2 and T3 have the same docked binding energy value of âˆ’9.6 kcal/mol to TMPRSS2, there are differences in their binding energy computed using MM-PBSA method. While TMPRSS2-T1 complex has the lowest binding energy value of âˆ’39.15 Â± 2.799 kcal/mol, TMPRSS2-T2 and TMPRSS2-T3 complexes have binding energy value of âˆ’30.284 Â± 3.585 kcal/mol and âˆ’27.386 Â± 2.077 kcal/mol, respectively (<a href="#molecules-25-03822-t003">Table 3</a>). In case of cathepsin L, cathepsin L-C1, cathepsin L-C2 and cathepsin L-C3 complexes have binding energy value of âˆ’22.384 Â± 3.420 kcal/mol, âˆ’20.577 Â± 3.600 kcal/mol and âˆ’26.156 Â± 3.433 kcal/mol, respectively (<a href="#molecules-25-03822-t003">Table 3</a>). Moreover, in comparison to binding energy obtained from docking using AutoDock Vina, binding energy for the TMPRSS2 and cathepsin L complexes with their top inhibitors computed using MM-PBSA method were found to be 2-fold to 4-fold lower, signifying even stronger binding (<a href="#molecules-25-03822-t001">Table 1</a>, <a href="#molecules-25-03822-t002">Table 2</a> and <a href="#molecules-25-03822-t003">Table 3</a>)</p> </div> </div> <div class="3.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
3. Materials and Methods</div> <div class="3.1.phytochemicallibraryanddrug-likenessevaluation" title="sec"> <div class="title" tagx="title" title="title">
3.1. Phytochemical Library and Drug-Likeness Evaluation</div> <p>Previously, some of us have built the Indian Medicinal Plants, Phytochemistry And Therapeutics (IMPPAT) database [<a href="#B28-molecules-25-03822">28</a>] which is the largest resource on phytochemicals of Indian herbs to date. For this study, we compiled a ligand library of 14,011 phytochemicals by augmenting the information in IMPPAT [<a href="#B28-molecules-25-03822">28</a>] with additional information compiled from other literature sources [<a href="#B61-molecules-25-03822">61</a>,<a href="#B62-molecules-25-03822">62</a>,<a href="#B63-molecules-25-03822">63</a>,<a href="#B64-molecules-25-03822">64</a>,<a href="#B65-molecules-25-03822">65</a>,<a href="#B66-molecules-25-03822">66</a>,<a href="#B67-molecules-25-03822">67</a>,<a href="#B68-molecules-25-03822">68</a>,<a href="#B69-molecules-25-03822">69</a>,<a href="#B70-molecules-25-03822">70</a>]. Thereafter, the widely used drug-likeness measure, Lipinskiâ€™s rule of five (RO5) [<a href="#B29-molecules-25-03822">29</a>], was employed to filter the potential drug-like molecules within the ligand library of 14,011 phytochemicals. Specifically, 10,510 phytochemicals passed the R05 drug-likeness filter. We then retrieved the three-dimensional (3D) structures of these phytochemicals from Pubchem [<a href="#B30-molecules-25-03822">30</a>]. Next the 3D structures of the drug-like phytochemicals were energy-minimized using <i>obminimize</i> within the OpenBabel toolbox [<a href="#B31-molecules-25-03822">31</a>]. Finally, the energy-minimized 3D structures of ligands in .sdf format were converted to .pdb format using OpenBabel.</p> </div> <div class="3.2.homologymodelingoftmprss2structure" title="sec"> <div class="title" tagx="title" title="title">
3.2. Homology Modeling of TMPRSS2 Structure</div> <p>TMPRSS2 is a trypsin-like serine protease whose catalytic site consists of the triad Ser441 (S441), His296 (H296) and Asp345 (D345) [<a href="#B35-molecules-25-03822">35</a>]. It is well established that trypsin-like serine proteases cleave peptide bonds following positively charged amino acid residues such as arginine or lysine, and this specificity of the enzyme is determined by a negatively charged aspartate residue located at the bottom of its S1 pocket [<a href="#B71-molecules-25-03822">71</a>]. In TMPRSS2, this specificity is determined by the conserved negatively charged residue Asp435 (D435) at the bottom of the S1 pocket [<a href="#B35-molecules-25-03822">35</a>].</p> <p>To date the 3D structure of TMPRSS2 has not been experimentally determined, and thus, we have used SWISS-MODEL [<a href="#B32-molecules-25-03822">32</a>] webserver (<span class="uri" title="uri"><a href="https://swissmodel.expasy.org/interactive">https://swissmodel.expasy.org/interactive</a></span>) to perform homology modeling of TMPRSS2. We submitted the TMPRSS2 protein sequence (NCBI reference sequence NP_005647.3) to SWISS-MODEL and selected the crystal structure of human protein hepsin (PDB 1Z8G) [<a href="#B72-molecules-25-03822">72</a>] as the template to build the model structure (<a href="#molecules-25-03822-f002">Figure 2</a>a). Note that hepsin (PDB 1Z8G) is also a Type II transmembrane trypsin-like serine protease, and it shares 38% sequence similarity with TMPRSS2 (NP_005647.3) (<a href="#molecules-25-03822-f002">Figure 2</a>b). Subsequently, UCSF Chimera [<a href="#B33-molecules-25-03822">33</a>] was used to minimize the energy of the TMPRSS2 model structure obtained from SWISS-MODEL. Thereafter, the energy-minimized TMPRSS2 model structure was assessed using the structure assessment tool within SWISS-MODEL. In the TMPRSS2 model, 94.19% of the amino acid residues were found to be in the Ramachandran favoured regions in the Ramachandran plot (<a href="#molecules-25-03822-f002">Figure 2</a>c) and the model structure has a MolProbity [<a href="#B73-molecules-25-03822">73</a>] score of 1.50.</p> </div> <div class="3.3.proteinstructureofcathepsinl" title="sec"> <div class="title" tagx="title" title="title">
3.3. Protein Structure of Cathepsin L</div> <p>We use the crystal structure (PDB 5MQY) [<a href="#B74-molecules-25-03822">74</a>] of human cathepsin L with 1.13Ã… resolution obtained from Protein Data Bank for virtual screening. UCSF Chimera was used to minimize the energy of the cathepsin L structure. <a href="#molecules-25-03822-f003">Figure 3</a> displays the cathepsin L structure with important residues in S1, S2 and S1â€² subsites of the enzyme [<a href="#B75-molecules-25-03822">75</a>]. Previous research has also revealed that S1 and S2 subsites of cathepsin L are important for the specificity of the enzyme [<a href="#B75-molecules-25-03822">75</a>,<a href="#B76-molecules-25-03822">76</a>]. In cathepsin L, the catalytic site consists of Cys25 (C25) and His163 (H163) in the S1 subsite, and Trp189 (W189) is at the center of the S1â€² subsite [<a href="#B75-molecules-25-03822">75</a>] (<a href="#molecules-25-03822-f003">Figure 3</a>). In cathepsin L, the S2 subsite with important residues Asp162 (D162), Met161 (M161), Ala135 (A135), Met70 (M70) and Leu69 (L69) forms a deep hydrophobic pocket, and lastly, the conserved residue Gly68 (G68) is at the center of the S3 subsite [<a href="#B75-molecules-25-03822">75</a>,<a href="#B77-molecules-25-03822">77</a>] (<a href="#molecules-25-03822-f003">Figure 3</a>).</p> </div> <div class="3.4.moleculardocking" title="sec"> <div class="title" tagx="title" title="title">
3.4. Molecular Docking</div> <p>AutoDock Vina [<a href="#B34-molecules-25-03822">34</a>] was used to perform the molecular docking of energy-minimized 3D structures of ligands with energy-minimized structure of target proteins. Accordingly, the 3D structures of prepared ligands in .pdb format were converted to .pdbqt format using the python script <i>prepare_ligand4.py</i> from AutoDockTools [<a href="#B78-molecules-25-03822">78</a>]. Similarly, the energy-minimized structure of TMPRSS2 and cathepsin L in .pdb format were converted to .pdbqt format using the python script <i>prepare_receptor4.py</i> from AutoDockTools [<a href="#B78-molecules-25-03822">78</a>].</p> <p>For protein-ligand docking, the appropriate grid box specified by the search space centre and search space size for TMPRSS2 and cathepsin L was manually determined by considering the key residues in target proteins such as the catalytic residues and substrate binding residues, which are important for the function and specificity of the considered proteases as reported in the literature. For TMPRSS2, the grid box was defined by the search space centre (40.50, âˆ’6.00, 25.70) and search space size of 25 Ã… Ã— 25 Ã… Ã— 25 Ã…. For cathepsin L, the grid box was defined by the search space centre (55.06, 48.65, 18.12) and search space size of 25 Ã… Ã— 25 Ã… Ã— 25 Ã….</p> <p>For both target proteins, the molecular docking of prepared ligands was performed using AutoDock Vina with the exhaustiveness set as 8. The top conformation of the docked ligand with lowest binding energy, i.e., the best docked pose, was obtained from the output of AutoDock Vina using the associated script <i>vina_split</i>. Subsequently, a combined structure file in .pdb format of the docked protein-ligand complex (with ligand in the best docked pose) was prepared using a custom script and pdb-tools [<a href="#B79-molecules-25-03822">79</a>].</p> </div> <div class="3.5.identificationofprotein-ligandinteractions" title="sec"> <div class="title" tagx="title" title="title">
3.5. Identification of Protein-Ligand Interactions</div> <p>We searched the combined structure file of the docked protein-ligand complex for ligand binding residues in protein and different non-covalent interactions that can facilitate the binding of the ligand with the protein. These non-covalent protein-ligand interactions were identified using different geometric criteria which are specific to different types of interactions:</p> <p>Binding site residue. Ligand binding site residues are defined as amino acids in protein which have at least one non-hydrogen atom in the proximity of at least one non-hydrogen atom of the ligand. The distance cut off to determine this proximity between non-hydrogen atoms of protein and ligand is taken to be the sum of their van der Waals radius plus 0.5Ã… [<a href="#B80-molecules-25-03822">80</a>].</p> <p>Hydrogen bonds. The accepted geometric criteria for hydrogen bonds of type D-Hâ‹¯A are as follows. Firstly, the distance between the hydrogen (H) and acceptor (A) atom should be less than the sum of their van der Waals radii. Secondly, the angle formed by donor (D), H and A atoms should be &amp;gt;90Â° (<a href="#app1-molecules-25-03822">Supplementary Figure S5a</a>). Moreover, carbon (C), nitrogen (N), oxygen (O) or sulfur (S) atoms can be donors while N, O or S atoms can be acceptors [<a href="#B42-molecules-25-03822">42</a>,<a href="#B81-molecules-25-03822">81</a>,<a href="#B82-molecules-25-03822">82</a>].</p> <p>Chalcogen bonds. In contrast to hydrogen bonds, chalcogen bonds are of type C-Yâ‹¯A, where Y can be a S or selenium (Se) atom and A can be a N, O or S atom. The accepted geometric criteria for chalcogen interactions are as follows. Firstly, the distance between Y and A should be less than the sum of their van der Waals radii. Secondly, the angle formed by the triad, that is âˆ C-Yâ‹¯A, should lie in the range 150Â° to 180Â° (<a href="#app1-molecules-25-03822">Supplementary Figure S5b</a>) [<a href="#B83-molecules-25-03822">83</a>].</p> <p>Halogen bonds. Halogen bonds are of type C-Yâ‹¯A-B, where halogen Y can be a Fluorine (F), Chlorine (Cl), Bromine (Br) or Iodine (I) atom and A can be a N, O or S atom. The formation of the halogen bond is favoured when the distance between Y and A is â‰¤3.7 Ã… and the angle Î¸<sub>1</sub> of the A atom relative to the C-Y bond, and the angle Î¸<sub>2</sub> of the halogen Y relative to the A-B bond should be â‰¥90Â° (<a href="#app1-molecules-25-03822">Supplementary Figure S5c</a>) [<a href="#B84-molecules-25-03822">84</a>].</p> <p>Ï€-Ï€ stacking. This interaction occurs between two aromatic rings and can be majorly classified into two types, namely, face-to-face and face-to-edge. In the case of face-to-face type of Ï€-Ï€ interaction, the distance between the centroids of the two participating aromatic rings should be &amp;lt;4.4 Ã… and the angle between their ring planes should be &amp;lt;30Â°. In the case of face-to-edge type of Ï€-Ï€ interaction, the distance between the centroids of the two participating aromatic rings should be &amp;lt;5.5 Ã… and the angle formed by the ring planes should be in the range 60Â° to 120Â° (<a href="#app1-molecules-25-03822">Supplementary Figure S5d,e</a>).</p> <p>Hydrophobic interactions. The geometric criteria for the formation of hydrophobic interactions between atoms in protein and ligand are as follows [<a href="#B85-molecules-25-03822">85</a>]. The distance between a carbon atom in protein or ligand and a carbon, halogen or sulfur atom in ligand or protein, respectively, should be â‰¤4 Ã…. Furthermore, we ensure that the involved atoms in a hydrophobic interaction between protein and ligand do not form hydrogen, chalcogen or halogen bonds between them [<a href="#B85-molecules-25-03822">85</a>].</p> <p>In order to detect the above-mentioned protein-ligand interactions, an in-house Python program was written to enable batch processing of combined structure files containing docked protein-ligand complexes for our large phytochemical library.</p> </div> <div class="3.6.comparisonwithreferenceinhibitorsoftmprss2andcathepsinl" title="sec"> <div class="title" tagx="title" title="title">
3.6. Comparison with Reference Inhibitors of TMPRSS2 and Cathepsin L</div> <p>In order to identify potent phytochemical inhibitors of target proteins, we decided to compare the binding energy of the best docked pose of ligands with binding energies of the best docked pose of known inhibitors of TMPRSS2 and cathepsin L obtained from AutoDock Vina.</p> <p>Recent experiments have shown that both camostat mesylate and nafamostat mesylate, which are approved for human use in Japan, can block the TMPRSS2-dependent cell entry of SARS-CoV-2 [<a href="#B10-molecules-25-03822">10</a>,<a href="#B15-molecules-25-03822">15</a>]. By docking these two inhibitors to TMPRSS2 using AutoDock Vina with exhaustiveness set at 50, the predicted binding energies of camostat and nafamostat was found to be âˆ’7.4 kcal/mol and âˆ’8.5 kcal/mol, respectively. <a href="#app1-molecules-25-03822">Supplementary Figure S6a,b</a> show the best docked poses of nafamostat and camostat with TMPRSS2, and it is seen that both molecules form hydrogen bonds with the substrate binding residue D435. Importantly, in comparison to camostat mesylate, nafamostat mesylate in a recent experiment was shown to inhibit the TMPRSS2-dependent cell entry with 15-fold higher efficiency and an EC<sub>50</sub> value in lower nanomolar range [<a href="#B15-molecules-25-03822">15</a>], and thus, the docked binding energies of these two known inhibitors are in line with experiments. Based on above observations, we decided on a stringent criteria of docked binding energy â‰¤âˆ’8.5 kcal/mol for screened ligands to be identified as potential inhibitors of TMPRSS2.</p> <p>Recent experiments have shown that the small molecules E-64d and PC-0626568 (SID26681509) can block the cathepsin L-dependent cell entry of SARS-CoV-2 [<a href="#B10-molecules-25-03822">10</a>,<a href="#B11-molecules-25-03822">11</a>,<a href="#B12-molecules-25-03822">12</a>]. Note that cathepsin L is one of 11 cysteine cathepsin proteases encoded by the human genome, and the cathepsins share a high sequence similarity to papain, a non-specific plant protease [<a href="#B86-molecules-25-03822">86</a>]. E64-d is a broad spectrum inhibitor which can inhibit proteases cathepsins B, H, L and calpain, while PC-0626568 is a specific inhibitor of cathepsin L [<a href="#B11-molecules-25-03822">11</a>]. Moreover, a recent study [<a href="#B11-molecules-25-03822">11</a>] used the specific inhibitor PC-0626568 of cathepsin L to conclude that cathepsin L rather than cathepsin B is important for cell entry of SARS-CoV-2. As both cathepsin L and cathepsin B are expressed in several mammalian tissues, it is important to design specific inhibitors of cathepsin L [<a href="#B75-molecules-25-03822">75</a>,<a href="#B76-molecules-25-03822">76</a>] to avoid any off-target toxicity. Along with the above-mentioned two inhibitors of cathepsin L, we have also considered the co-crystallized inhibitor GH4 present in the crystal structure of cathepsin L (PDB 5MQY) as another reference inhibitor of cathepsin L. We remark that while recent experiments [<a href="#B10-molecules-25-03822">10</a>,<a href="#B11-molecules-25-03822">11</a>,<a href="#B12-molecules-25-03822">12</a>] have shown that E-64d and PC-0626568 can block the cathepsin L-dependent cell entry of SARS-CoV-2, similar experimental data specific to SARS-CoV-2 infection is lacking for cathepsin L inhibitor GH4. By docking theseâ€"three known inhibitors to cathepsin L using AutoDock Vina with exhaustiveness set at 50, the predicted binding energies of E-64d, PC-0626568 and GH4 were found to be âˆ’5.0 kcal/mol, âˆ’8.0 kcal/mol and âˆ’6.3 kcal/mol, respectively. Notably, the docked binding energies are in line with known specificity of E-64d and PC-0626568 to cathepsin L. <a href="#app1-molecules-25-03822">Supplementary Figure S6câ€"e</a> show the best docked poses of PC-0626568, E-64d and GH4 with cathepsin L. It is seen that both E-64d and PC-0626568 form hydrogen bonds with both catalytic residues C25 and H163. Based on above observations, we decided on a stringent criteria of docked binding energy â‰¤âˆ’8.0 kcal/mol for screened ligands to be identified as potential inhibitors of cathepsin L. We have also compared the docked pose of GH4 obtained from AutoDock Vina with the pose of GH4 in the co-crystallized structure of cathepsin L, and the RMSD between the heavy atoms in the two poses was found to be 0.786 Ã…. <a href="#app1-molecules-25-03822">Supplementary Figure S7</a> shows the superimposed structures of the docked pose of GH4 from AutoDock Vina and the pose of GH4 in the co-crystallized structure of cathepsin L. Apart from PC-0626568 and E-64d, seven other cathepsin-L inhibitors namely, dec-RVKR-CMK, K11777, small molecule 5705213, MDL28170, SSAA09E1, EST and oxocarbazate, have been reported in literature to exhibit ant-coronavirus activity [<a href="#B87-molecules-25-03822">87</a>].</p> </div> <div class="3.7.moleculardynamicssimulations" title="sec"> <div class="title" tagx="title" title="title">
3.7. Molecular Dynamics Simulations</div> <p>Using GROMACS 5.1.5 [<a href="#B88-molecules-25-03822">88</a>] along with GROMOS96 54a7 force field [<a href="#B89-molecules-25-03822">89</a>], we have performed MD simulations for the top inhibitors of TMPRSS2 and cathepsin L to assess the stability of their protein-ligand complexes. The Automated Topology Builder (ATB) version 3.0 (<span class="uri" title="uri"><a href="https://atb.uq.edu.au/">https://atb.uq.edu.au/</a></span>) was used to generate topology for the top inhibitors [<a href="#B90-molecules-25-03822">90</a>]. The protein-ligand complex was placed in the center of a cubic periodic box and was solvated by the addition of simple point charge (SPC) waters. Then net charge of the system was neutralized by addition of Na<sup>+</sup> and Cl<sup>âˆ’</sup> ions. Energy minimization was performed using the steepest descent algorithm. Then the system was heated to 310 K during a constant number of particles, volume, and temperature (NVT) simulation of 500 ps with 2 fs time step. Then the pressure was equilibrated to 1 bar during a constant number of particles, pressure, and temperature (NPT) simulation of 500 ps with 2 fs time step. In both the above simulations, protein and ligand were position restrained. Thereafter, the position restraint was removed and a production MD run was performed for a period of 180 ns with 2 fs time step. During the MD simulation, the structural coordinates were saved after every 2 ps. Temperature was maintained at 310 K using the v-rescale temperature [<a href="#B91-molecules-25-03822">91</a>]. Pressure was maintained at 1 bar using Parrinello-Rahman pressure coupling method [<a href="#B92-molecules-25-03822">92</a>]. Time constant for the temperature and pressure coupling were kept at 0.1 ps and 2 ps, respectively. The short-range interactions were computed for the atom pairs within the cutoff of 1.4 nm, whereas the long-range electrostatic interactions were calculated using Particle mesh Ewald (PME) method with fourth order cubic interpolation and 0.16 nm grid spacing. All bonds were constrained using the LINCS method. The trajectories obtained from the MD simulations were then used to compute and analyze the radius of gyration of the protein (R<sub>g</sub>), root mean square deviation (RMSD) of the protein backbone CÎ± atoms and root mean square fluctuation (RMSF) of the protein backbone CÎ± atoms using GROMACS scripts.</p> </div> <div class="3.8.mm-pbsacalculation" title="sec"> <div class="title" tagx="title" title="title">
3.8. MM-PBSA Calculation</div> <p>Molecular Mechanics Poisson-Boltzmann Surface Area (MM-PBSA) method was used to compute the binding energy of the interactions between the protein-ligand complexes for the top inhibitors of TMPRSS2 and cathepsin L. From the 180 ns trajectory obtained from the MD simulation, 80 snapshots were obtained between 100 ns to 180 ns of the simulation at an interval of 1 ns along the trajectory, and thereafter, the binding energy was computed using g_mmpbsa [<a href="#B59-molecules-25-03822">59</a>,<a href="#B60-molecules-25-03822">60</a>]. g_mmpbsa computes the Gibbs free energy of binding (Î"G<sub>binding</sub>) using MM-PBSA method as described by the following equation: </p><div class="disp-formula" title="disp-formula">
Î"G<sub>binding</sub> = G<sub>complex</sub> âˆ’ (G<sub>protein</sub> + G<sub>ligand</sub>)</div> where G<sub>complex,</sub> G<sub>protein</sub> and G<sub>ligand</sub> represent the total binding energy of the protein-ligand complex, the free protein and the unbounded ligand, respectively.<p /> </div> <div class="3.9.validationofthetmprss2modelstructure" title="sec"> <div class="title" tagx="title" title="title">
3.9. Validation of the TMPRSS2 Model Structure</div> <p>In order to validate the model structure of TMPRSS2 built using SWISS-MODEL, we have used two approaches. Firstly, we have run a 180 ns MD simulation of the TMPRSS2 free protein and have computed radius of gyration (R<sub>g</sub>), RMSD and RMSF for the TMPRSS2 free protein from its MD trajectory (<a href="#app1-molecules-25-03822">Supplementary Figure S1aâ€"c</a>). The R<sub>g</sub> of the TMPRSS2 free protein remains largely stable throughout the 180 ns MD simulation (<a href="#app1-molecules-25-03822">Supplementary Figure S1a</a>). This signifies that the TMPRSS2 model structure remains structurally intact and stable during the MD simulation. The RMSD of the TMPRSS2 free protein becomes stable after 60 ns (<a href="#app1-molecules-25-03822">Supplementary Figure S1b</a>).</p> <p>Secondly, we have used the MD trajectory of TMPRSS2 in complex with the top three potential inhibitors (T1, T2 or T3) from 40 ns to 80 ns to validate the TMPRSS2 model structure. The MD trajectory of TMPRSS2 in complex with the inhibitors from 40 ns to 80 ns was used for validation, 80 ns to 100 ns was used for equilibration, and from 100 ns to 180 ns for finding the binding energy of T1, T2 or T3 with TMPRSS2 using MM-PBSA method. To validate the TMPRSS2 model structure, the binding energy computed using MM-PBSA method of T1, T2 or T3 with TMPRSS2 during 40 ns to 80 ns was compared with binding energy of T1, T2 or T3 with TMPRSS2 during 100 ns to 180 ns of the MD simulation. We find that T1, T2 and T3 have average binding energies of âˆ’37.507 Â± 2.537 kcal/mol, âˆ’30.383 Â± 4.005 kcal/mol and âˆ’26.586 Â± 2.506 kcal/mol, respectively, with TMPRSS2 during 40 ns to 80 ns of the MD simulation. The above reported binding energies of T1, T2 and T3 with TMPRSS2 during 40 ns to 80 ns are very close to the binding energies computed for the same inhibitors in complex with TMPRSS2 during 100 ns to 180 ns of the MD simulation as reported in <a href="#molecules-25-03822-t003">Table 3</a>. These results further validate the TMPRSS2 model structure used for binding energy computations during 100 ns to 180 ns of the MD simulations.</p> </div> <div class="3.10.predictionofadmetproperties" title="sec"> <div class="title" tagx="title" title="title">
3.10. Prediction of ADMET Properties</div> <p>In order to assess the pharmacokinetic properties and potential toxicity of the inhibitors of TMPRSS2 and cathepsin L predicted from this study, we have also computed the Absorption, Distribution, Metabolism, Excretion and Toxicity (ADMET) properties of the inhibitors using SwissADME [<a href="#B93-molecules-25-03822">93</a>] and vNN-ADMET [<a href="#B94-molecules-25-03822">94</a>].</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
4. Conclusions</div> <p>The current COVID-19 pandemic is a serious threat to humankind. As of 27 July 2020, COVID-19 has led to more than 650,000 deaths worldwide. Due to the absence of approved therapeutics or vaccines against SARS-CoV-2, several countries have been forced to implement partial or complete lockdown measures to restrict infection spread; however, such measures have in turn resulted in an economic catastrophe. Consequently, there is an urgent need to develop antivirals and vaccines against SARS-CoV-2 to protect humankind. In this direction, protein-ligand docking and MD simulation are powerful computational methods to expedite the search for anti-COVID drugs by rapid identification of promising lead molecules. Here, we have used molecular docking and MD simulations in the search of natural compound inhibitors of two human proteases, TMPRSS2 and cathepsin L, that are key host factors in SARS-CoV-2 infection [<a href="#B10-molecules-25-03822">10</a>,<a href="#B11-molecules-25-03822">11</a>,<a href="#B12-molecules-25-03822">12</a>].</p> <p>Since early civilization, humans have used medicinal plants in different systems of traditional medicine to treat various ailments [<a href="#B95-molecules-25-03822">95</a>]. Specifically, traditional systems of Indian medicine including Ayurveda, Siddha and Unani have over centuries acquired invaluable knowledge on medicinal plants spanning the rich biodiversity of the subcontinent for treating various ailments including viral infections [<a href="#B28-molecules-25-03822">28</a>]. As plant-based natural products have been an indomitable source of lead molecules in the course of modern drug discovery [<a href="#B25-molecules-25-03822">25</a>], it is worthwhile to search for potential anti-COVID drugs among phytochemicals of Indian medicinal plants. In this direction, some of us have built IMPPAT [<a href="#B28-molecules-25-03822">28</a>], the largest resource on phytochemicals of Indian medicinal plants to facilitate natural product-based drug discovery. In this work, we have performed virtual screening of 14,011 phytochemicals that are produced by Indian medicinal plants to identify potential inhibitors of key host factors, TMPRSS2 and cathepsin L, for SARS-CoV-2 infection. For the identified top inhibitors of TMPRSS2 and cathepsin L, we have performed MD simulation, and thereafter, employed MM-PBSA method to compute binding energies of the protein-ligand complexes.</p> <p>We have predicted 96 potential phytochemical inhibitors of TMPRSS2, of which the top three candidates, namely, qingdainone (T1), edgeworoside C (T2) and adlumidine (T3), have binding energy value of âˆ’39.15 Â± 2.799 kcal/mol, âˆ’30.284 Â± 3.585 kcal/mol and âˆ’27.386 Â± 2.077 kcal/mol, respectively (<a href="#molecules-25-03822-t003">Table 3</a>). We have also predicted nine potential phytochemical inhibitors of cathepsin L, of which the top three candidates, namely, ararobinol (C1), (+)-oxoturkiyenine (C2) and 3Î±,17Î±-cinchophylline (C3), have binding energy value of âˆ’22.384 Â± 3.420 kcal/mol, âˆ’20.577 Â± 3.600 kcal/mol and âˆ’26.156 Â± 3.433 kcal/mol, respectively (<a href="#molecules-25-03822-t003">Table 3</a>). Furthermore, most of the herbal sources of the identified phytochemical inhibitors of TMPRSS2 and cathepsin L have been reported to have antiviral or anti-inflammatory use in traditional medicine (<a href="#molecules-25-03822-t001">Table 1</a> and <a href="#molecules-25-03822-t002">Table 2</a>). Additional in vitro and in vivo testing of the identified phytochemical inhibitors of TMPRSS2 and cathepsin L is needed before these molecules can enter clinical trials against COVID-19. In conclusion, we expect the natural product inhibitors identified in this computational study for TMPRSS2 and cathepsin L will likely inform future research toward natural product-based anti-COVID therapeutics.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>Authors thank S. Krishnaswamy for guidance and help with MD simulations. Authors also thank Dhiraj Kumar, Vinay Nandicoori and Pinaki Saha for discussions. A.S. and R.P.V.-A. Thank the computational staff of IMSc for maintaining resources during the Covid-19 associated lockdown. H.S.B. and A.R. acknowledge the computational facilities of NISER. A.S. would like to acknowledge financial support from the Science and Engineering Research Board (SERB) India through the award of a Ramanujan fellowship (SB/S2/RJN-006/2014), Max Planck Society Germany through the award of a Max Planck Partner Group in Mathematical Biology, and the Abdus Salam International Centre for Theoretical Physics Italy for the award of a Simons Associateship. The funders have no role in study design, data collection, data analysis, manuscript preparation or decision to publish.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Sample Availability:</b> Samples of the compounds are not available from the authors.</p> </div> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>The following are available online: Figure S1: (<b>a</b>) Radius of gyration, (<b>b</b>) RMSD, and (<b>c</b>) RMSF for TMPRSS2 free protein. (<b>d</b>) Radius of gyration, (<b>e</b>) RMSD, and (<b>f</b>) RMSF for cathepsin L free protein, Figure S2: Superimposition of the snapshots at 120 ns, 140 ns and 160 ns of (<b>a</b>) TMPRSS2-T1 complex, (<b>b</b>) TMPRSS2-T2 complex, (<b>c</b>) TMPRSS2-T3 complex, (<b>d</b>) cathepsin L-C1 complex, (<b>e</b>) cathepsin L-C2 complex and (<b>f</b>) cathepsin L-C3 complex obtained from their respective MD simulation trajectories, Figure S3: Cartoon representation of the protein-ligand interactions of the phytochemical inhibitors of TMPRSS2. Interactions of TMPRSS2 residues with atoms of (<b>a</b>) T7, (<b>b</b>) T8, and (<b>c</b>) T9. The carbon atoms of the ligand are shown in green colour while the carbon atoms of the amino acid residues in TMPRSS2 are shown in cyan colour. TMPRSS2 residues interacting with the ligand atoms via hydrogen bonds or Ï€-Ï€ stacking are labelled with the corresponding single letter residue code along with their position in the protein sequence. The hydrogen bonds and Ï€-Ï€ stacking are displayed using yellow and red dotted lines, respectively, Figure S4: Cartoon representation of the protein-ligand interactions of the phytochemical inhibitors of cathepsin L. Interactions of cathepsin L residues with atoms of (<b>a</b>) C7, (<b>b</b>) C8, and (<b>c</b>) C9. The carbon atoms of the ligand are shown in green colour while the carbon atoms of the amino acid residues in cathepsin L are shown in cyan colour. Cathepsin L residues interacting with the ligand atoms via hydrogen bonds or Ï€-Ï€ stacking are labelled with the corresponding single letter residue code along with their position in the protein sequence. The hydrogen bonds and Ï€-Ï€ stacking are displayed using yellow and red dotted lines, respectively, Figure S5: Geometric criteria for the identification of protein-ligand interactions. (<b>a</b>) Hydrogen bond, (<b>b</b>) Chalcogen bond, (<b>c</b>) Halogen bond, (<b>d</b>) face-to-face Ï€-Ï€ stacking, and (<b>e</b>) face-to-edge Ï€-Ï€ stacking, Figure S6: Cartoon representation of the protein-ligand interactions of the known inhibitors of TMPRSS2 and cathepsin L. Interactions of TMPRSS2 residues with atoms of (<b>a</b>) nafamostat, and (<b>b</b>) camostat. Interactions of cathepsin L residues with atoms of (<b>c</b>) PC-0626568, (<b>d</b>) E-64d, and (<b>e</b>) GH4. The carbon atoms of the ligand are shown in green colour while the carbon atoms of the amino acid residues in TMPRSS2 or cathepsin L are shown in cyan colour. TMPRSS2 or cathepsin L residues interacting with the ligand atoms via hydrogen bonds or Ï€-Ï€ stacking or halogen bonds are labelled with the corresponding single letter residue code along with their position in the protein sequence. The hydrogen bonds, Ï€-Ï€ stacking and halogen bonds are displayed using yellow, red and black dotted lines, respectively, Figure S7: Superimposition of the docked pose of GH4 with cathepsin L obtained from AutoDock Vina (shown in green colour) and the pose of GH4 in the co-crystallized structure with cathepsin L (shown in red colour), Table S1: Top list of 96 phytochemical inhibitors of TMPRSS2. For each phytochemical, the table gives the symbol, docked binding energy, Pubchem identifier, common name, IUPAC name and SMILES, Table S2: Top list of 9 phytochemical inhibitors of cathepsin L. For each phytochemical, the table gives the symbol, docked binding energy, Pubchem identifier, common name, IUPAC name and SMILES, Table S3: Herbal sources of top 96 phytochemical inhibitors of TMPRSS2. For each phytochemical, the table gives the symbol, docked binding energy, common name and plant source. Plant sources which have been reported to have anti-viral or anti-inflammatory therapeutic use in traditional medicine literature are shown in bold and marked with an [*] sign, Table S4: The table lists the ligand binding site residues and non-covalent protein-ligand interactions namely, Hydrogen bond interactions, Ï€-Ï€ stacking interactions of face-to-face type and face-to-edge type and Hydrophobic interactions that were identified from the docked protein-ligand complexes of the top 96 phytochemical inhibitors of TMPRSS2, Table S5: The table lists the ligand binding site residues and non-covalent protein-ligand interactions namely, Hydrogen bond interactions, Ï€-Ï€ stacking interactions of face-to-face type and face-to-edge type and Hydrophobic interactions that were identified from the docked protein-ligand complexes of the top 9 phytochemical inhibitors of cathepsin L, Table S6: ADMET properties of the top list of 96 potential phytochemical inhibitors of TMPRSS2. For each phytochemical, the table gives the symbol, Pubchem identifier; several physicochemical properties such as Molecular weight in g/mol, LogP (Partition coefficient), MolarRefractivity, TPSA (Topological Surface Area) in Ã…<sup>2</sup>, Number of hydrogen bond acceptors, Number of hydrogen bond donors, Total number of atoms, Number of rotatable bonds, Shape complexity (fraction of carbon atoms that are sp<sup>3</sup> hybridized), Stereochemical complexity (fraction of carbon atoms which are stereogenic); several drug-likeness properties such as Lipinski RO5 filter (Lipinskiâ€™s Rule of 5 filter), QEDw (weighted quantitative estimate of drug-likeness), QEDuw (unweighted quantitative estimate of drug-likeness), Leadlikeness-number of violations predicted by SwissADME, Synthetic accessibility predicted by SwissADME; several absorption and distribution properties such as Solubility class predicted by SwissADME using ESOL method, Solubility class predicted by SwissADME using method by Ali et al., Solubility class predicted by SwissADME using Silicos-IT, Gasterointestinal absorption predicted by SwissADME, Skin permeation predicted by SwissADME (log Kp cm/s), Blood Brain Barrier permeation predicted by SwissADME, Blood Brain Barrier permeation predicted by vNNADMET, Bioavailability Score predicted by SwissADME; several metabolism properties such as P-glycoprotein substrate predicted by SwissADME, P-glycoprotein Substrate predicted by vNNADMET, P-glycoprotein inhibitor predicted by vNNADMET, Cytochrome P450 1A2 inhibitor predicted by SwissADME, Cytochrome P450 1A2 inhibitor predicted by vNNADMET, Cytochrome P450 2C19 inhibitor predicted by SwissADME, Cytochrome P450 2C19 inhibitor predicted by vNNADMET, Cytochrome P450 2C9 inhibitor predicted by SwissADME, Cytochrome P450 2C9 inhibitor predicted by vNNADMET, Cytochrome P450 2D6 inhibitor predicted by SwissADME, Cytochrome P450 2D6 inhibitor predicted by vNNADMET, Cytochrome P450 3A4 inhibitor predicted by SwissADME, Cytochrome P450 3A4 inhibitor predicted by vNNADMET, Human liver microsomal stability predicted by vNNADMET (HLM); several toxicity properties such as PAINS-Number of alerts predicted by SwissADME, Brenk-Number of alerts predicted by SwissADME, Drug-induced liver injury predicted by vNNADMET (DILI), Cytotoxicity predicted by vNNADMET, hERG Blocker predicted by vNNADMET, Mitochondrial toxicity predicted by vNNADMET (MMP) and Mutagenecity predicted by vNNADMET (AMES), Table S7: ADMET properties of the top list of 9 potential phytochemical inhibitors of cathepsin L. For each phytochemical, the table gives the symbol, Pubchem identifier; several physicochemical properties such as Molecular weight in g/mol, LogP (Partition coefficient), MolarRefractivity, TPSA (Topological Surface Area) in Ã…<sup>2</sup>, Number of hydrogen bond acceptors, Number of hydrogen bond donors, Total number of atoms, Number of rotatable bonds, Shape complexity (fraction of carbon atoms that are sp<sup>3</sup> hybridized), Stereochemical complexity (fraction of carbon atoms which are stereogenic); several drug-likeness properties such as Lipinski RO5 filter (Lipinskiâ€™s Rule of 5 filter), QEDw (weighted quantitative estimate of drug-likeness), QEDuw (unweighted quantitative estimate of drug-likeness), Leadlikeness-number of violations predicted by SwissADME, Synthetic accessibility predicted by SwissADME; several absorption and distribution properties such as Solubility class predicted by SwissADME using ESOL method, Solubility class predicted by SwissADME using method by Ali et al., Solubility class predicted by SwissADME using Silicos-IT, Gasterointestinal absorption predicted by SwissADME, Skin permeation predicted by SwissADME (log Kp cm/s), Blood Brain Barrier permeation predicted by SwissADME, Blood Brain Barrier permeation predicted by vNNADMET, Bioavailability Score predicted by SwissADME; several metabolism properties such as P-glycoprotein substrate predicted by SwissADME, P-glycoprotein Substrate predicted by vNNADMET, P-glycoprotein inhibitor predicted by vNNADMET, Cytochrome P450 1A2 inhibitor predicted by SwissADME, Cytochrome P450 1A2 inhibitor predicted by vNNADMET, Cytochrome P450 2C19 inhibitor predicted by SwissADME, Cytochrome P450 2C19 inhibitor predicted by vNNADMET, Cytochrome P450 2C9 inhibitor predicted by SwissADME, Cytochrome P450 2C9 inhibitor predicted by vNNADMET, Cytochrome P450 2D6 inhibitor predicted by SwissADME, Cytochrome P450 2D6 inhibitor predicted by vNNADMET, Cytochrome P450 3A4 inhibitor predicted by SwissADME, Cytochrome P450 3A4 inhibitor predicted by vNNADMET, Human liver microsomal stability predicted by vNNADMET (HLM); several toxicity properties such as PAINS-Number of alerts predicted by SwissADME, Brenk-Number of alerts predicted by SwissADME, Drug-induced liver injury predicted by vNNADMET (DILI), Cytotoxicity predicted by vNNADMET, hERG Blocker predicted by vNNADMET, Mitochondrial toxicity predicted by vNNADMET (MMP) and Mutagenecity predicted by vNNADMET (AMES).</p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="molecules-25-03822-s001.zip">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>Conceptualization, R.P.V.-A. and A.S.; Data curation, R.P.V.-A., N.R. and A.S.; Formal analysis, R.P.V.-A., A.R., H.S.B. and A.S.; Methodology, R.P.V.-A., A.R., H.S.B. and A.S.; Supervision, H.S.B. and A.S.; Writingâ€"original draft, R.P.V.-A., A.R., H.S.B. and A.S.; Writingâ€"review &amp;amp; editing, R.P.V.-A., H.S.B and A.S. All authors have read and agreed to the published version of the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This research has received no specific external funding.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest.</p> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div><span class="array" tagx="array" title="array">   MD Molecular dynamic   COVID-19 Coronavirus disease 2019   SARS-CoV-2 Severe acute respiratory syndrome coronavirus 2   SARS-CoV Severe acute respiratory syndrome coronavirus   MERS-CoV Middle East respiratory syndrome coronavirus   TMPRSS2 Transmembrane Protease Serine 2   RBD Receptor binding domain   ACE-2 Angiotensin converting enzyme 2   ORF Open reading frame   FDA Food and drug administration   ADMET Absorption, distribution, metabolism, excretion and toxicity   RMSD Root mean square deviation   RMSF Root mean square fluctuation   MM-PBSA Molecular Mechanics Poisson-Boltzmann Surface Area  </span></div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-molecules-25-03822" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">395</span><span class="fpage" tagx="fpage" title="fpage">497</span><span class="lpage" tagx="lpage" title="lpage">506</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30183-5">10.1016/S0140-6736(20)30183-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31986264">31986264</a></span></span></li> <li tag="ref"><a name="B2-molecules-25-03822" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horby</span><span class="given-names" tagx="given-names" title="given-names">P.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayden</span><span class="given-names" tagx="given-names" title="given-names">F.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">G.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A novel coronavirus outbreak of global health concern</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">395</span><span class="fpage" tagx="fpage" title="fpage">470</span><span class="lpage" tagx="lpage" title="lpage">473</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30185-9">10.1016/S0140-6736(20)30185-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31986257">31986257</a></span></span></li> <li tag="ref"><a name="B3-molecules-25-03822" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">382</span><span class="fpage" tagx="fpage" title="fpage">727</span><span class="lpage" tagx="lpage" title="lpage">733</span><span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa2001017">10.1056/NEJMoa2001017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31978945">31978945</a></span></span></li> <li tag="ref"><a name="B4-molecules-25-03822" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">J.F.-W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kok</span><span class="given-names" tagx="given-names" title="given-names">K.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">To</span><span class="given-names" tagx="given-names" title="given-names">K.K.-W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yip</span><span class="given-names" tagx="given-names" title="given-names">C.C.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">R.W.-S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: A study of a family cluster</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">395</span><span class="fpage" tagx="fpage" title="fpage">514</span><span class="lpage" tagx="lpage" title="lpage">523</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30154-9">10.1016/S0140-6736(20)30154-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31986261">31986261</a></span></span></li> <li tag="ref"><a name="B5-molecules-25-03822" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bats Are Natural Reservoirs of SARS-Like Coronaviruses</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">310</span><span class="fpage" tagx="fpage" title="fpage">676</span><span class="lpage" tagx="lpage" title="lpage">679</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1118391">10.1126/science.1118391</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16195424">16195424</a></span></span></li> <li tag="ref"><a name="B6-molecules-25-03822" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.-G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Si</span><span class="given-names" tagx="given-names" title="given-names">H.-R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.-L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">579</span><span class="fpage" tagx="fpage" title="fpage">270</span><span class="lpage" tagx="lpage" title="lpage">273</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2012-7">10.1038/s41586-020-2012-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32015507">32015507</a></span></span></li> <li tag="ref"><a name="B7-molecules-25-03822" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Wit</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Doremalen</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falzarano</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munster</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">SARS and MERS: Recent insights into emerging coronaviruses</span><span class="source" tagx="source" title="source">Nat. Rev. Genet.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">523</span><span class="lpage" tagx="lpage" title="lpage">534</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro.2016.81">10.1038/nrmicro.2016.81</a></span></span></li> <li tag="ref"><a name="B8-molecules-25-03822" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Clercq</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Therapeutic options for the 2019 novel coronavirus (2019-nCoV)</span><span class="source" tagx="source" title="source">Nat. Rev. Drug Discov.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">149</span><span class="lpage" tagx="lpage" title="lpage">150</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/d41573-020-00016-0">10.1038/d41573-020-00016-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32127666">32127666</a></span></span></li> <li tag="ref"><a name="B9-molecules-25-03822" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</span><span class="source" tagx="source" title="source">Cell Host Microbe</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">325</span><span class="lpage" tagx="lpage" title="lpage">328</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chom.2020.02.001">10.1016/j.chom.2020.02.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32035028">32035028</a></span></span></li> <li tag="ref"><a name="B10-molecules-25-03822" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoffmann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kleine-Weber</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schroeder</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KrÃ¼ger</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrler</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erichsen</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schiergens</span><span class="given-names" tagx="given-names" title="given-names">T.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrler</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">N.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nitsche</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">181</span><span class="fpage" tagx="fpage" title="fpage">271</span><span class="lpage" tagx="lpage" title="lpage">280</span><span class="page-range" tagx="page-range" title="page-range">271â€"280.e8</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2020.02.052">10.1016/j.cell.2020.02.052</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32142651">32142651</a></span></span></li> <li tag="ref"><a name="B11-molecules-25-03822" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ou</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mi</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of spike glycoprotein of SARS-CoV-2 on virus entry and its immune cross-reactivity with SARS-CoV</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">1620</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-020-15562-9">10.1038/s41467-020-15562-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32221306">32221306</a></span></span></li> <li tag="ref"><a name="B12-molecules-25-03822" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geng</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Auerbach</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cell entry mechanisms of SARS-CoV-2</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">117</span><span class="fpage" tagx="fpage" title="fpage">11727</span><span class="lpage" tagx="lpage" title="lpage">11734</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.2003138117">10.1073/pnas.2003138117</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32376634">32376634</a></span></span></li> <li tag="ref"><a name="B13-molecules-25-03822" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aihara</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geng</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Auerbach</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural basis of receptor recognition by SARS-CoV-2</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">581</span><span class="fpage" tagx="fpage" title="fpage">221</span><span class="lpage" tagx="lpage" title="lpage">224</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2179-y">10.1038/s41586-020-2179-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32225175">32225175</a></span></span></li> <li tag="ref"><a name="B14-molecules-25-03822" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural basis for the recognition of SARS-CoV-2 by full-length human ACE2</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">367</span><span class="fpage" tagx="fpage" title="fpage">1444</span><span class="lpage" tagx="lpage" title="lpage">1448</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.abb2762">10.1126/science.abb2762</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32132184">32132184</a></span></span></li> <li tag="ref"><a name="B15-molecules-25-03822" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoffmann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schroeder</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kleine-Weber</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MÃ¼ller</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PÃ¶hlmann</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nafamostat Mesylate Blocks Activation of SARS-CoV-2: New Treatment Option for COVID-19</span><span class="source" tagx="source" title="source">Antimicrob. Agents Chemother.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">64</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AAC.00754-20">10.1128/AAC.00754-20</a></span></span></li> <li tag="ref"><a name="B16-molecules-25-03822" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of therapeutic targets for SARS-CoV-2 and discovery of potential drugs by computational methods</span><span class="source" tagx="source" title="source">Acta Pharm. Sin. B</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">766</span><span class="lpage" tagx="lpage" title="lpage">788</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.apsb.2020.02.008">10.1016/j.apsb.2020.02.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32292689">32292689</a></span></span></li> <li tag="ref"><a name="B17-molecules-25-03822" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ElFiky</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ribavirin, Remdesivir, Sofosbuvir, Galidesivir, and Tenofovir against SARS-CoV-2 RNA dependent RNA polymerase (RdRp): A molecular docking study</span><span class="source" tagx="source" title="source">Life Sci.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">253</span><span class="fpage" tagx="fpage" title="fpage">117592</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.lfs.2020.117592">10.1016/j.lfs.2020.117592</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32222463">32222463</a></span></span></li> <li tag="ref"><a name="B18-molecules-25-03822" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parves</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uddin</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al Mamun</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halim</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A molecular modeling approach to identify effective antiviral phytochemicals against the main protease of SARS-CoV-2</span><span class="source" tagx="source" title="source">J. Biomol. Struct. Dyn.</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1761883">10.1080/07391102.2020.1761883</a></span></span></li> <li tag="ref"><a name="B19-molecules-25-03822" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basharat</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yousuf</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castaldo</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rastrelli</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Virtual Screening of Natural Products against Type II Transmembrane Serine Protease (TMPRSS2), the Priming Agent of Coronavirus 2 (SARS-CoV-2)</span><span class="source" tagx="source" title="source">Molecules</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">25</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">2271</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules25102271">10.3390/molecules25102271</a></span></span></li> <li tag="ref"><a name="B20-molecules-25-03822" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Modi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sagar</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">In silico studies on therapeutic agents for COVID-19: Drug repurposing approach</span><span class="source" tagx="source" title="source">Life Sci.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">252</span><span class="fpage" tagx="fpage" title="fpage">117652</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.lfs.2020.117652">10.1016/j.lfs.2020.117652</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32278693">32278693</a></span></span></li> <li tag="ref"><a name="B21-molecules-25-03822" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choudhir</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shukla</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tyagi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhushan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rathore</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of phytochemical inhibitors against main protease of COVID-19 using molecular modeling approaches</span><span class="source" tagx="source" title="source">J. Biomol. Struct. Dyn.</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1772112">10.1080/07391102.2020.1772112</a></span></span></li> <li tag="ref"><a name="B22-molecules-25-03822" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Potential covalent drugs targeting the main protease of the SARS-CoV-2 coronavirus</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">3295</span><span class="lpage" tagx="lpage" title="lpage">3298</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btaa224">10.1093/bioinformatics/btaa224</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32239142">32239142</a></span></span></li> <li tag="ref"><a name="B23-molecules-25-03822" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olubiyi</span><span class="given-names" tagx="given-names" title="given-names">O.O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olagunju</span><span class="given-names" tagx="given-names" title="given-names">M.O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keutmann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loschwitz</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strodel</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">High Throughput Virtual Screening to Discover Inhibitors of the Main Protease of the Coronavirus SARS-CoV-2</span><span class="source" tagx="source" title="source">Molecules</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">25</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">3193</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules25143193">10.3390/molecules25143193</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32668701">32668701</a></span></span></li> <li tag="ref"><a name="B24-molecules-25-03822" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmad</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dwivedy</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mariadasse</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tiwari</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kar</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeyakanthan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biswal</span><span class="given-names" tagx="given-names" title="given-names">B.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Prediction of Small Molecule Inhibitors Targeting the Severe Acute Respiratory Syndrome Coronavirus-2 RNA-dependent RNA Polymerase</span><span class="source" tagx="source" title="source">ACS Omega</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acsomega.0c02096">10.1021/acsomega.0c02096</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32743211">32743211</a></span></span></li> <li tag="ref"><a name="B25-molecules-25-03822" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newman</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cragg</span><span class="given-names" tagx="given-names" title="given-names">G.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Natural Products As Sources of New Drugs over the 30 Years from 1981 to 2010</span><span class="source" tagx="source" title="source">J. Nat. Prod.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">311</span><span class="lpage" tagx="lpage" title="lpage">335</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/np200906s">10.1021/np200906s</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22316239">22316239</a></span></span></li> <li tag="ref"><a name="B26-molecules-25-03822" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">J.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">A.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.-J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Traditional Chinese medicine for COVID-19 treatment</span><span class="source" tagx="source" title="source">Pharmacol. Res.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">155</span><span class="fpage" tagx="fpage" title="fpage">104743</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phrs.2020.104743">10.1016/j.phrs.2020.104743</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32145402">32145402</a></span></span></li> <li tag="ref"><a name="B27-molecules-25-03822" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vellingiri</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jayaramayya</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iyer</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narayanasamy</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Govindasamy</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giridharan</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ganesan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venugopal</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venkatesan</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ganesan</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">COVID-19: A promising cure for the global panic</span><span class="source" tagx="source" title="source">Sci. Total Environ.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">725</span><span class="fpage" tagx="fpage" title="fpage">138277</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.scitotenv.2020.138277">10.1016/j.scitotenv.2020.138277</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32278175">32278175</a></span></span></li> <li tag="ref"><a name="B28-molecules-25-03822" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohanraj</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karthikeyan</span><span class="given-names" tagx="given-names" title="given-names">B.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vivek-Ananth</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chand</span><span class="given-names" tagx="given-names" title="given-names">R.P.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aparna</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mangalapandi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samal</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">IMPPAT: A curated database of Indian Medicinal Plants, Phytochemistry And Therapeutics</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">4329</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-22631-z">10.1038/s41598-018-22631-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29531263">29531263</a></span></span></li> <li tag="ref"><a name="B29-molecules-25-03822" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipinski</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lombardo</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dominy</span><span class="given-names" tagx="given-names" title="given-names">B.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feeney</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span><span class="source" tagx="source" title="source">Adv. Drug Deliv. Rev.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="lpage" tagx="lpage" title="lpage">26</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0169-409X(00)00129-0">10.1016/S0169-409X(00)00129-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11259830">11259830</a></span></span></li> <li tag="ref"><a name="B30-molecules-25-03822" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gindulyte</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shoemaker</span><span class="given-names" tagx="given-names" title="given-names">B.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiessen</span><span class="given-names" tagx="given-names" title="given-names">P.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">PubChem 2019 update: Improved access to chemical data</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">D1102</span><span class="lpage" tagx="lpage" title="lpage">D1109</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gky1033">10.1093/nar/gky1033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30371825">30371825</a></span></span></li> <li tag="ref"><a name="B31-molecules-25-03822" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oâ€™Boyle</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banck</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morley</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandermeersch</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hutchison</span><span class="given-names" tagx="given-names" title="given-names">G.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Open Babel: An open chemical toolbox</span><span class="source" tagx="source" title="source">J. Cheminformatics</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1758-2946-3-33">10.1186/1758-2946-3-33</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21982300">21982300</a></span></span></li> <li tag="ref"><a name="B32-molecules-25-03822" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterhouse</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertoni</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bienert</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Studer</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tauriello</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gumienny</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heer</span><span class="given-names" tagx="given-names" title="given-names">F.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beer</span><span class="given-names" tagx="given-names" title="given-names">T.A.P.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rempfer</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bordoli</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SWISS-MODEL: Homology modelling of protein structures and complexes</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">W296</span><span class="lpage" tagx="lpage" title="lpage">W303</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gky427">10.1093/nar/gky427</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29788355">29788355</a></span></span></li> <li tag="ref"><a name="B33-molecules-25-03822" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pettersen</span><span class="given-names" tagx="given-names" title="given-names">E.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goddard</span><span class="given-names" tagx="given-names" title="given-names">T.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Couch</span><span class="given-names" tagx="given-names" title="given-names">G.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenblatt</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">E.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferrin</span><span class="given-names" tagx="given-names" title="given-names">T.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">UCSF Chimera--a visualization system for exploratory research and analysis</span><span class="source" tagx="source" title="source">J. Comput. Chem.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1605</span><span class="lpage" tagx="lpage" title="lpage">1612</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.20084">10.1002/jcc.20084</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15264254">15264254</a></span></span></li> <li tag="ref"><a name="B34-molecules-25-03822" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trott</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span><span class="source" tagx="source" title="source">J. Comput. Chem.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">455</span><span class="lpage" tagx="lpage" title="lpage">461</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.21334">10.1002/jcc.21334</a></span></span></li> <li tag="ref"><a name="B35-molecules-25-03822" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paoloni-Giacobino</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peitsch</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossier</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antonarakis</span><span class="given-names" tagx="given-names" title="given-names">S.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cloning of the TMPRSS2 Gene, Which Encodes a Novel Serine Protease with Transmembrane, LDLRA, and SRCR Domains and Maps to 21q22.3</span><span class="source" tagx="source" title="source">Genomics</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">309</span><span class="lpage" tagx="lpage" title="lpage">320</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/geno.1997.4845">10.1006/geno.1997.4845</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9325052">9325052</a></span></span></li> <li tag="ref"><a name="B36-molecules-25-03822" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">Y.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">C.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yen</span><span class="given-names" tagx="given-names" title="given-names">H.-R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">Y.-S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Y.-P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">S.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">C.-W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antiviral Action of Tryptanthrin Isolated from Strobilanthes cusia Leaf against Human Coronavirus NL63</span><span class="source" tagx="source" title="source">Biomolecules</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">366</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/biom10030366">10.3390/biom10030366</a></span></span></li> <li tag="ref"><a name="B37-molecules-25-03822" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">D.-G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">A.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">Y.-Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coumarins with Î±-glucosidase and Î±-amylase inhibitory activities from the flower of Edgeworthia gardneri</span><span class="source" tagx="source" title="source">Fitoterapia</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">107</span><span class="fpage" tagx="fpage" title="fpage">122</span><span class="lpage" tagx="lpage" title="lpage">127</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fitote.2015.10.012">10.1016/j.fitote.2015.10.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26529177">26529177</a></span></span></li> <li tag="ref"><a name="B38-molecules-25-03822" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Y.-X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">H.-W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">In vitro evaluation of dual agonists for PPARÎ³/Î² from the flower of Edgeworthia gardneri (wall.) Meisn</span><span class="source" tagx="source" title="source">J. Ethnopharmacol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">162</span><span class="fpage" tagx="fpage" title="fpage">14</span><span class="lpage" tagx="lpage" title="lpage">19</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jep.2014.12.034">10.1016/j.jep.2014.12.034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25557029">25557029</a></span></span></li> <li tag="ref"><a name="B39-molecules-25-03822" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Q.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">Z.-N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The flower of Edgeworthia gardneri (wall.) Meisn. suppresses adipogenesis through modulation of the AMPK pathway in 3T3-L1 adipocytes</span><span class="source" tagx="source" title="source">J. Ethnopharmacol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">191</span><span class="fpage" tagx="fpage" title="fpage">379</span><span class="lpage" tagx="lpage" title="lpage">386</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jep.2016.06.059">10.1016/j.jep.2016.06.059</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27350007">27350007</a></span></span></li> <li tag="ref"><a name="B40-molecules-25-03822" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">P.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">C.V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A review on ethnobotany, phytochemistry and pharmacology of Fumaria indica (Fumitory)</span><span class="source" tagx="source" title="source">Asian Pac. J. Trop. Biomed.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">665</span><span class="lpage" tagx="lpage" title="lpage">669</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S2221-1691(12)60117-8">10.1016/S2221-1691(12)60117-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23569991">23569991</a></span></span></li> <li tag="ref"><a name="B41-molecules-25-03822" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kardos</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BlaskÃ³</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kerekes</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KovÃ¡cs</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonyi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of [3H]GABA binding to rat brain synaptic membranes by bicuculline related alkaloids</span><span class="source" tagx="source" title="source">Biochem. Pharmacol.</span><span class="year" tagx="year" title="year">1984</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">3537</span><span class="lpage" tagx="lpage" title="lpage">3545</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0006-2952(84)90134-5">10.1016/0006-2952(84)90134-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6095852">6095852</a></span></span></li> <li tag="ref"><a name="B42-molecules-25-03822" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chand</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sahoo</span><span class="given-names" tagx="given-names" title="given-names">D.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rana</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jena</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biswal</span><span class="given-names" tagx="given-names" title="given-names">H.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Prodigious Hydrogen Bonds with Sulfur and Selenium in Molecular Assemblies, Structural Biology, and Functional Materials</span><span class="source" tagx="source" title="source">Acc. Chem. Res.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.accounts.0c00289">10.1021/acs.accounts.0c00289</a></span></span></li> <li tag="ref"><a name="B43-molecules-25-03822" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Acharya</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitra</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shukla</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of Shodhana (processing) on Kupeelu (Strychnos nux-vomica Linn.) with special reference to strychnine and brucine content</span><span class="source" tagx="source" title="source">AYU (An. Int. Q. J. Res. Ayurveda)</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">402</span><span class="lpage" tagx="lpage" title="lpage">407</span><span class="pub-id"><a href="https://dx.doi.org/10.4103/0974-8520.93923">10.4103/0974-8520.93923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22529660">22529660</a></span></span></li> <li tag="ref"><a name="B44-molecules-25-03822" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ueno</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bracamontes</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zorumski</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiss</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinbach</span><span class="given-names" tagx="given-names" title="given-names">J.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bicuculline and Gabazine Are Allosteric Inhibitors of Channel Opening of the GABAA Receptor</span><span class="source" tagx="source" title="source">J. Neurosci.</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">625</span><span class="lpage" tagx="lpage" title="lpage">634</span><span class="pub-id"><a href="https://dx.doi.org/10.1523/JNEUROSCI.17-02-00625.1997">10.1523/JNEUROSCI.17-02-00625.1997</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8987785">8987785</a></span></span></li> <li tag="ref"><a name="B45-molecules-25-03822" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khare</span><span class="given-names" tagx="given-names" title="given-names">C.P.</span></span></span><span class="source" tagx="source" title="source">Indian Medicinal Plants: An Illustrated Dictionary</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span><span class="year" tagx="year" title="year">2007</span></span></li> <li tag="ref"><a name="B46-molecules-25-03822" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arya</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panghal</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhankhar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cassia occidentalis L.: A review on its ethnobotany, phytochemical and pharmacological profile</span><span class="source" tagx="source" title="source">Fitoterapia</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">223</span><span class="lpage" tagx="lpage" title="lpage">230</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fitote.2009.09.008">10.1016/j.fitote.2009.09.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19796670">19796670</a></span></span></li> <li tag="ref"><a name="B47-molecules-25-03822" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">S.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z.Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">X.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Application of Anthraqinone Derivative in Resisting Influenza Virus and Bird Flu Virus H5N1</span><span class="source" tagx="source" title="source">China Patent</span><span class="patent" tagx="patent" title="patent">CN20051134677</span><span class="day" tagx="day" title="day">21</span><span class="month" tagx="month" title="month">12</span><span class="year" tagx="year" title="year">2005</span></span></li> <li tag="ref"><a name="B48-molecules-25-03822" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mete</span><span class="given-names" tagx="given-names" title="given-names">I.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ¶zler</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">(+)-Oxoturkiyenine: An Isoquinoline-Derived Alkaloid from Hypecoum pendulum</span><span class="source" tagx="source" title="source">J. Nat. Prod.</span><span class="year" tagx="year" title="year">1988</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">272</span><span class="lpage" tagx="lpage" title="lpage">274</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/np50056a013">10.1021/np50056a013</a></span></span></li> <li tag="ref"><a name="B49-molecules-25-03822" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurung</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spectrum of biological properties of cinchona alkaloids: A brief review</span><span class="source" tagx="source" title="source">J. Pharmacogn. Phytochem.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">162</span><span class="lpage" tagx="lpage" title="lpage">166</span></span></li> <li tag="ref"><a name="B50-molecules-25-03822" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tripathi</span><span class="given-names" tagx="given-names" title="given-names">Y.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maurya</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cyclopeptide alkaloids from Zizyphus rugosa bark</span><span class="source" tagx="source" title="source">Phytochemistry</span><span class="year" tagx="year" title="year">1989</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">1563</span><span class="lpage" tagx="lpage" title="lpage">1565</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0031-9422(00)97797-5">10.1016/S0031-9422(00)97797-5</a></span></span></li> <li tag="ref"><a name="B51-molecules-25-03822" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analgesic and anti-inflammatory activities of Zizyphus rugosa root barks</span><span class="source" tagx="source" title="source">J. Chem. Pharm. Res.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">255</span><span class="lpage" tagx="lpage" title="lpage">259</span></span></li> <li tag="ref"><a name="B52-molecules-25-03822" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luan</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">X.-D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M.-X.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Traditional uses and pharmacological properties of Clerodendrum phytochemicals</span><span class="source" tagx="source" title="source">J. Tradit. Complement. Med.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">24</span><span class="lpage" tagx="lpage" title="lpage">38</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtcme.2017.04.001">10.1016/j.jtcme.2017.04.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29321986">29321986</a></span></span></li> <li tag="ref"><a name="B53-molecules-25-03822" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sumthong</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero-GonzÃ¡lez</span><span class="given-names" tagx="given-names" title="given-names">R.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verpoorte</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of Anti-Wood Rot Compounds in Teak (Tectona grandisL.f.) Sawdust Extract</span><span class="source" tagx="source" title="source">J. Wood Chem. Technol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">247</span><span class="lpage" tagx="lpage" title="lpage">260</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/02773810802452592">10.1080/02773810802452592</a></span></span></li> <li tag="ref"><a name="B54-molecules-25-03822" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kapadia</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jadeja</span><span class="given-names" tagx="given-names" title="given-names">R.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thounaojam</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devkar</span><span class="given-names" tagx="given-names" title="given-names">R.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mishra</span><span class="given-names" tagx="given-names" title="given-names">S.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Traditional uses, phytochemistry and pharmacology of Tecomella undulataâ€"A review</span><span class="source" tagx="source" title="source">Asian Pac. J. Trop. Biomed.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">1918</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S2221-1691(12)60521-8">10.1016/S2221-1691(12)60521-8</a></span></span></li> <li tag="ref"><a name="B55-molecules-25-03822" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cadelis</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourguet-Kondracki</span><span class="given-names" tagx="given-names" title="given-names">M.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dubois</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valentin</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barker</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Copp</span><span class="given-names" tagx="given-names" title="given-names">B.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery and preliminary structureâ€"activity relationship studies on tecomaquinone I and tectol as novel farnesyltransferase and plasmodial inhibitors</span><span class="source" tagx="source" title="source">Bioorganic Med. Chem.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">3102</span><span class="lpage" tagx="lpage" title="lpage">3107</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bmc.2016.05.024">10.1016/j.bmc.2016.05.024</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27240468">27240468</a></span></span></li> <li tag="ref"><a name="B56-molecules-25-03822" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosisio</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of the flavanolignans of Silybum marianum L. On lipid peroxidation in rat liver microsomes and freshly isolated hepatocytes</span><span class="source" tagx="source" title="source">Pharmacol. Res.</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">147</span><span class="lpage" tagx="lpage" title="lpage">165</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/1043-6618(92)91383-R">10.1016/1043-6618(92)91383-R</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1635893">1635893</a></span></span></li> <li tag="ref"><a name="B57-molecules-25-03822" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahmani</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirzad</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rafieian</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rafieian-Kopaei</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Silybum marianum: Beyond Hepatoprotection</span><span class="source" tagx="source" title="source">J. Evid. -Based Complementary Altern. Med.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">292</span><span class="lpage" tagx="lpage" title="lpage">301</span><span class="pub-id"><a href="https://dx.doi.org/10.1177/2156587215571116">10.1177/2156587215571116</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25686616">25686616</a></span></span></li> <li tag="ref"><a name="B58-molecules-25-03822" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Genheden</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryde</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities</span><span class="source" tagx="source" title="source">Expert Opin. Drug Discov.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">449</span><span class="lpage" tagx="lpage" title="lpage">461</span><span class="pub-id"><a href="https://dx.doi.org/10.1517/17460441.2015.1032936">10.1517/17460441.2015.1032936</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25835573">25835573</a></span></span></li> <li tag="ref"><a name="B59-molecules-25-03822" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">N.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sept</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joseph</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holst</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCammon</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Electrostatics of nanosystems: Application to microtubules and the ribosome</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">98</span><span class="fpage" tagx="fpage" title="fpage">10037</span><span class="lpage" tagx="lpage" title="lpage">10041</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.181342398">10.1073/pnas.181342398</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11517324">11517324</a></span></span></li> <li tag="ref"><a name="B60-molecules-25-03822" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumari</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lynn</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Open Source Drug Discovery Consortium g_mmpbsaâ€"A GROMACS Tool for High-Throughput MM-PBSA Calculations</span><span class="source" tagx="source" title="source">J. Chem. Inf. Model.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">1951</span><span class="lpage" tagx="lpage" title="lpage">1962</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ci500020m">10.1021/ci500020m</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24850022">24850022</a></span></span></li> <li tag="ref"><a name="B61-molecules-25-03822" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="source" tagx="source" title="source">The Wealth of India the wealth of India: A Dictionary of Indian Raw Materials &amp;amp; Industrial Products. First Supplement Series (Raw Materials)</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Publications &amp;amp; Information Directorate, Council of Scientific &amp;amp; Industrial Research</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi, India</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">Volume 1</span></span></li> <li tag="ref"><a name="B62-molecules-25-03822" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="source" tagx="source" title="source">The Wealth of India the Wealth of India: A Dictionary of Indian Raw Materials &amp;amp; Industrial Products. First Supplement Series (Raw Materials)</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Publications &amp;amp; Information Directorate, Council of Scientific &amp;amp; Industrial Research</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi, India</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">Volume 2</span></span></li> <li tag="ref"><a name="B63-molecules-25-03822" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="source" tagx="source" title="source">The Wealth of India the wealth of India: A Dictionary of Indian Raw Materials &amp;amp; Industrial Products. First Supplement Series (Raw Materials)</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Publications &amp;amp; Information Directorate, Council of Scientific &amp;amp; Industrial Research</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi, India</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">Volume 3</span></span></li> <li tag="ref"><a name="B64-molecules-25-03822" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="source" tagx="source" title="source">The Wealth of India the Wealth of India: A Dictionary of Indian Raw Materials &amp;amp; Industrial Products. First Supplement Series (Raw Materials)</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Publications &amp;amp; Information Directorate, Council of Scientific &amp;amp; Industrial Research</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi, India</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">Volume 4</span></span></li> <li tag="ref"><a name="B65-molecules-25-03822" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="source" tagx="source" title="source">The Wealth of India the Wealth of India: A Dictionary of Indian Raw Materials &amp;amp; Industrial Products. First Supplement Series (Raw Materials)</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Publications &amp;amp; Information Directorate, Council of Scientific &amp;amp; Industrial Research</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New Delhi, India</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">Volume 5</span></span></li> <li tag="ref"><a name="B66-molecules-25-03822" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rastogi</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Drug Research Perspectives</span><span class="source" tagx="source" title="source">Compendium of Indian Medicinal Plants</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Central Drug Research Institute</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Lucknow, India</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">Volume 1</span><span class="fpage" tagx="fpage" title="fpage">1960</span><span class="lpage" tagx="lpage" title="lpage">1969</span></span></li> <li tag="ref"><a name="B67-molecules-25-03822" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rastogi</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Drug Research Perspectives</span><span class="source" tagx="source" title="source">Compendium of Indian Medicinal Plants</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Central Drug Research Institute</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Lucknow, India</span><span class="year" tagx="year" title="year">1991</span><span class="volume" tagx="volume" title="volume">Volume 2</span><span class="fpage" tagx="fpage" title="fpage">1970</span><span class="lpage" tagx="lpage" title="lpage">1979</span></span></li> <li tag="ref"><a name="B68-molecules-25-03822" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rastogi</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Drug Research Perspectives</span><span class="source" tagx="source" title="source">Compendium of Indian Medicinal Plants</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Central Drug Research Institute</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Lucknow, India</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">Volume 3</span><span class="fpage" tagx="fpage" title="fpage">1980</span><span class="lpage" tagx="lpage" title="lpage">1984</span></span></li> <li tag="ref"><a name="B69-molecules-25-03822" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rastogi</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Drug Research Perspectives</span><span class="source" tagx="source" title="source">Compendium of Indian Medicinal Plants</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Central Drug Research Institute</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Lucknow, India</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">Volume 4</span><span class="fpage" tagx="fpage" title="fpage">1985</span><span class="lpage" tagx="lpage" title="lpage">1989</span></span></li> <li tag="ref"><a name="B70-molecules-25-03822" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rastogi</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Drug Research Perspectives</span><span class="source" tagx="source" title="source">Compendium of Indian Medicinal Plants</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Central Drug Research Institute</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Lucknow, India</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">Volume 5</span><span class="fpage" tagx="fpage" title="fpage">1990</span><span class="lpage" tagx="lpage" title="lpage">1994</span></span></li> <li tag="ref"><a name="B71-molecules-25-03822" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evnin</span><span class="given-names" tagx="given-names" title="given-names">L.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasquez</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Craik</span><span class="given-names" tagx="given-names" title="given-names">C.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Substrate specificity of trypsin investigated by using a genetic selection</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">6659</span><span class="lpage" tagx="lpage" title="lpage">6663</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.87.17.6659">10.1073/pnas.87.17.6659</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2204062">2204062</a></span></span></li> <li tag="ref"><a name="B72-molecules-25-03822" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herter</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piper</span><span class="given-names" tagx="given-names" title="given-names">D.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aaron</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gabriele</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cutler</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatt</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choe</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Craik</span><span class="given-names" tagx="given-names" title="given-names">C.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatocyte growth factor is a preferred in vitro substrate for human hepsin, a membrane-anchored serine protease implicated in prostate and ovarian cancers</span><span class="source" tagx="source" title="source">Biochem. J.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">390</span><span class="fpage" tagx="fpage" title="fpage">125</span><span class="lpage" tagx="lpage" title="lpage">136</span><span class="pub-id"><a href="https://dx.doi.org/10.1042/BJ20041955">10.1042/BJ20041955</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15839837">15839837</a></span></span></li> <li tag="ref"><a name="B73-molecules-25-03822" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">V.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arendall</span><span class="given-names" tagx="given-names" title="given-names">W.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Headd</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keedy</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Immormino</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kapral</span><span class="given-names" tagx="given-names" title="given-names">G.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murray</span><span class="given-names" tagx="given-names" title="given-names">L.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">D.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MolProbity: All-atom structure validation for macromolecular crystallography</span><span class="source" tagx="source" title="source">Acta Crystallogr. Sect. D Biol. Crystallogr.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">12</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1107/S0907444909042073">10.1107/S0907444909042073</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20057044">20057044</a></span></span></li> <li tag="ref"><a name="B74-molecules-25-03822" /><span class="label" tagx="label" title="label">74.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuhn</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TichÃ½</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">R.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuglstatter</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benz</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giroud</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schirmeister</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abel</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Prospective Evaluation of Free Energy Calculations for the Prioritization of Cathepsin L Inhibitors</span><span class="source" tagx="source" title="source">J. Med. Chem.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">2485</span><span class="lpage" tagx="lpage" title="lpage">2497</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jmedchem.6b01881">10.1021/acs.jmedchem.6b01881</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28287264">28287264</a></span></span></li> <li tag="ref"><a name="B75-molecules-25-03822" /><span class="label" tagx="label" title="label">75.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujishima</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imai</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nomura</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujisawa</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamamoto</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugawara</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The crystal structure of human cathepsin L complexed with E-64</span><span class="source" tagx="source" title="source">FEBS Lett.</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">407</span><span class="fpage" tagx="fpage" title="fpage">47</span><span class="lpage" tagx="lpage" title="lpage">50</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0014-5793(97)00216-0">10.1016/S0014-5793(97)00216-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9141479">9141479</a></span></span></li> <li tag="ref"><a name="B76-molecules-25-03822" /><span class="label" tagx="label" title="label">76.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turk</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoka</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasiljeva</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renko</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turk</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turk</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cysteine cathepsins: From structure, function and regulation to new frontiers</span><span class="source" tagx="source" title="source">Biochim. Biophys. Acta (BBA)-Proteins Proteom.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">1824</span><span class="fpage" tagx="fpage" title="fpage">68</span><span class="lpage" tagx="lpage" title="lpage">88</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbapap.2011.10.002">10.1016/j.bbapap.2011.10.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22024571">22024571</a></span></span></li> <li tag="ref"><a name="B77-molecules-25-03822" /><span class="label" tagx="label" title="label">77.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams-Cioaba</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krupa</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mort</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Min</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural basis for the recognition and cleavage of histone H3 by cathepsin L.</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">197</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms1204">10.1038/ncomms1204</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21326229">21326229</a></span></span></li> <li tag="ref"><a name="B78-molecules-25-03822" /><span class="label" tagx="label" title="label">78.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huey</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindstrom</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanner</span><span class="given-names" tagx="given-names" title="given-names">M.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belew</span><span class="given-names" tagx="given-names" title="given-names">R.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodsell</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span><span class="source" tagx="source" title="source">J. Comput. Chem.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">2785</span><span class="lpage" tagx="lpage" title="lpage">2791</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.21256">10.1002/jcc.21256</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19399780">19399780</a></span></span></li> <li tag="ref"><a name="B79-molecules-25-03822" /><span class="label" tagx="label" title="label">79.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigues</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mc Teixeira</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trellet</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonvin</span><span class="given-names" tagx="given-names" title="given-names">A.M.J.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">pdb-tools: A swiss army knife for molecular structures</span><span class="source" tagx="source" title="source">F1000Research</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">1961</span><span class="pub-id"><a href="https://dx.doi.org/10.12688/f1000research.17456.1">10.12688/f1000research.17456.1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30705752">30705752</a></span></span></li> <li tag="ref"><a name="B80-molecules-25-03822" /><span class="label" tagx="label" title="label">80.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cassarino</span><span class="given-names" tagx="given-names" title="given-names">T.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwede</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Assessment of ligand-binding residue predictions in CASP9</span><span class="source" tagx="source" title="source">Proteins Struct. Funct. Bioinform.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">79</span><span class="fpage" tagx="fpage" title="fpage">126</span><span class="lpage" tagx="lpage" title="lpage">136</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.23174">10.1002/prot.23174</a></span></span></li> <li tag="ref"><a name="B81-molecules-25-03822" /><span class="label" tagx="label" title="label">81.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarkhel</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desiraju</span><span class="given-names" tagx="given-names" title="given-names">G.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">N-Hâ€¦O, O-Hâ€¦O, and C-Hâ€¦O hydrogen bonds in protein-ligand complexes: Strong and weak interactions in molecular recognition</span><span class="source" tagx="source" title="source">Proteins Struct. Funct. Bioinform.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">247</span><span class="lpage" tagx="lpage" title="lpage">259</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.10567">10.1002/prot.10567</a></span></span></li> <li tag="ref"><a name="B82-molecules-25-03822" /><span class="label" tagx="label" title="label">82.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Geometric characteristics of hydrogen bonds involving sulfur atoms in proteins</span><span class="source" tagx="source" title="source">Proteins Struct. Funct. Bioinform.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">151</span><span class="lpage" tagx="lpage" title="lpage">163</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.22327">10.1002/prot.22327</a></span></span></li> <li tag="ref"><a name="B83-molecules-25-03822" /><span class="label" tagx="label" title="label">83.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÅ™Ã­Å¾</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fanfrlik</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepsik</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chalcogen Bonding in Proteinâˆ’Ligand Complexes: PDB Survey and Quantum Mechanical Calculations</span><span class="source" tagx="source" title="source">Chem. Phys. Chem.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">2540</span><span class="lpage" tagx="lpage" title="lpage">2548</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cphc.201800409">10.1002/cphc.201800409</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30003638">30003638</a></span></span></li> <li tag="ref"><a name="B84-molecules-25-03822" /><span class="label" tagx="label" title="label">84.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borozan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">StojanoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">S.Ä�.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Halogen bonding in complexes of proteins and non-natural amino acids</span><span class="source" tagx="source" title="source">Comput. Biol. Chem.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">231</span><span class="lpage" tagx="lpage" title="lpage">239</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.compbiolchem.2013.10.002">10.1016/j.compbiolchem.2013.10.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24200696">24200696</a></span></span></li> <li tag="ref"><a name="B85-molecules-25-03822" /><span class="label" tagx="label" title="label">85.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Freitas</span><span class="given-names" tagx="given-names" title="given-names">R.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schapira</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A systematic analysis of atomic proteinâ€"ligand interactions in the PDBâ€ â€ Electronic supplementary information (ESI) available. See doi:10.1039/c7md00381a</span><span class="source" tagx="source" title="source">Med. Chem. Comm.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1970</span><span class="lpage" tagx="lpage" title="lpage">1981</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/C7MD00381A">10.1039/C7MD00381A</a></span></span></li> <li tag="ref"><a name="B86-molecules-25-03822" /><span class="label" tagx="label" title="label">86.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turk</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turk</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turk</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">NEW EMBO MEMBERSâ€™ REVIEW: Lysosomal cysteine proteases: Facts and opportunities</span><span class="source" tagx="source" title="source">EMBO J.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">4629</span><span class="lpage" tagx="lpage" title="lpage">4633</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/emboj/20.17.4629">10.1093/emboj/20.17.4629</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11532926">11532926</a></span></span></li> <li tag="ref"><a name="B87-molecules-25-03822" /><span class="label" tagx="label" title="label">87.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Libby</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cathepsin L-selective inhibitors: A potentially promising treatment for COVID-19 patients</span><span class="source" tagx="source" title="source">Pharmacol. Ther.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">213</span><span class="fpage" tagx="fpage" title="fpage">107587</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pharmthera.2020.107587">10.1016/j.pharmthera.2020.107587</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32470470">32470470</a></span></span></li> <li tag="ref"><a name="B88-molecules-25-03822" /><span class="label" tagx="label" title="label">88.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abraham</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murtola</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulz</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PÃ¡ll</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hess</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindahl</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">GROMACS: High performance molecular simulations through multi-level parallelism from laptops to supercomputers</span><span class="source" tagx="source" title="source">SoftwareX</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">19</span><span class="lpage" tagx="lpage" title="lpage">25</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.softx.2015.06.001">10.1016/j.softx.2015.06.001</a></span></span></li> <li tag="ref"><a name="B89-molecules-25-03822" /><span class="label" tagx="label" title="label">89.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmid</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eichenberger</span><span class="given-names" tagx="given-names" title="given-names">A.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choutko</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riniker</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winger</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mark</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Gunsteren</span><span class="given-names" tagx="given-names" title="given-names">W.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Definition and testing of the GROMOS force-field versions 54A7 and 54B7</span><span class="source" tagx="source" title="source">Eur. Biophys. J.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">843</span><span class="lpage" tagx="lpage" title="lpage">856</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00249-011-0700-9">10.1007/s00249-011-0700-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21533652">21533652</a></span></span></li> <li tag="ref"><a name="B90-molecules-25-03822" /><span class="label" tagx="label" title="label">90.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stroet</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caron</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Visscher</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geerke</span><span class="given-names" tagx="given-names" title="given-names">D.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malde</span><span class="given-names" tagx="given-names" title="given-names">A.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mark</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Automated Topology Builder Version 3.0: Prediction of Solvation Free Enthalpies in Water and Hexane</span><span class="source" tagx="source" title="source">J. Chem. Theory Comput.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">5834</span><span class="lpage" tagx="lpage" title="lpage">5845</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jctc.8b00768">10.1021/acs.jctc.8b00768</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30289710">30289710</a></span></span></li> <li tag="ref"><a name="B91-molecules-25-03822" /><span class="label" tagx="label" title="label">91.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berendsen</span><span class="given-names" tagx="given-names" title="given-names">H.J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Postma</span><span class="given-names" tagx="given-names" title="given-names">J.P.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Gunsteren</span><span class="given-names" tagx="given-names" title="given-names">W.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DiNola</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haak</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular dynamics with coupling to an external bath</span><span class="source" tagx="source" title="source">J. Chem. Phys.</span><span class="year" tagx="year" title="year">1984</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">3684</span><span class="lpage" tagx="lpage" title="lpage">3690</span><span class="pub-id"><a href="https://dx.doi.org/10.1063/1.448118">10.1063/1.448118</a></span></span></li> <li tag="ref"><a name="B92-molecules-25-03822" /><span class="label" tagx="label" title="label">92.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parrinello</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Polymorphic transitions in single crystals: A new molecular dynamics method</span><span class="source" tagx="source" title="source">J. Appl. Phys.</span><span class="year" tagx="year" title="year">1981</span><span class="volume" tagx="volume" title="volume">52</span><span class="fpage" tagx="fpage" title="fpage">7182</span><span class="pub-id"><a href="https://dx.doi.org/10.1063/1.328693">10.1063/1.328693</a></span></span></li> <li tag="ref"><a name="B93-molecules-25-03822" /><span class="label" tagx="label" title="label">93.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daina</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michielin</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zoete</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">SwissADME: A free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">42717</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep42717">10.1038/srep42717</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28256516">28256516</a></span></span></li> <li tag="ref"><a name="B94-molecules-25-03822" /><span class="label" tagx="label" title="label">94.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schyman</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desai</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallqvist</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">vNN Web Server for ADMET Predictions</span><span class="source" tagx="source" title="source">Front. Pharmacol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">889</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fphar.2017.00889">10.3389/fphar.2017.00889</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29255418">29255418</a></span></span></li> <li tag="ref"><a name="B95-molecules-25-03822" /><span class="label" tagx="label" title="label">95.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piao</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Traditional Medicine and Modern Medicine from Natural Products</span><span class="source" tagx="source" title="source">Molecules</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">21</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">559</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules21050559">10.3390/molecules21050559</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27136524">27136524</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Virtual screening workflow to identify potential phytochemical inhibitors of human proteases TMPRSS2 and cathepsin L.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>(<b>a</b>) Cartoon representation of the homology model structure of TMPRSS2 which has been energy-minimized using UCSF Chimera. The figure zooms into the region containing the catalytic triad Ser441 (S441), His296 (H296) and Asp345 (D345), and the substrate binding residue Asp435 (D435) in the S1 subsite of the enzyme. (<b>b</b>) Alignment of protein sequences for TMPRSS2 and hepsin (PDB 1Z8G) which was used as a template to model the structure of TMPRSS2. (<b>c</b>) General Ramachandran plot of the energy-minimized model structure of TMPRSS2, which displays the torsional angles, phi (Ï†) and psi (Ïˆ), of the amino acid residues in the protein.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Cartoon representation of the crystal structure of human cathepsin L (PDB 5MQY). The figure zooms into the region containing the catalytic residues Cys (C25) and His163 (H163) in the S1 subsite, residues Asp162 (D162), Met161 (M161), Ala135 (A135), Met70 (M70) and Leu (L69) in the S2 subsite, Trp189 (W189) at the centre of S1â€² subsite and the conserved residue Gly68 (G68) in the S3 subsite of the enzyme.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Molecular structures of the top 9 phytochemical inhibitors (compounds <b>T1</b>â€"<b>T9</b>) of TMPRSS2. For each inhibitor, the figure shows the 2D structure, common name and docked binding energy of the ligand with TMPRSS2.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Cartoon representation of the protein-ligand interactions of the phytochemical inhibitors of TMPRSS2. Interactions of TMPRSS2 residues with atoms of (<b>a</b>) T1, (<b>b</b>) T2, (<b>c</b>) T3, (<b>d</b>) T4, (<b>e</b>) T5, and (<b>f</b>) T6. The carbon atoms of the ligand are shown in green colour while the carbon atoms of the amino acid residues in TMPRSS2 are shown in cyan colour. TMPRSS2 residues interacting with the ligand atoms via hydrogen bonds or Ï€-Ï€ stacking are labelled with the corresponding single letter residue code along with their position in the protein sequence. The hydrogen bonds and Ï€-Ï€ stacking are displayed using yellow and red dotted lines, respectively.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Molecular structures of the top 9 phytochemical inhibitors (compounds <b>C1</b>â€"<b>C9</b>) of cathepsin L. For each inhibitor, the figure shows the 2D structure, common name and docked binding energy of the ligand with cathepsin L.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Cartoon representation of the protein-ligand interactions of the phytochemical inhibitors of cathepsin L. Interactions of cathepsin L residues with atoms of (<b>a</b>) C1, (<b>b</b>) C2, (<b>c</b>) C3, (<b>d</b>) C4, (<b>e</b>) C5, and (<b>f</b>) C6. The carbon atoms of the ligand are shown in green colour while the carbon atoms of the amino acid residues in cathepsin L are shown in cyan colour. Cathepsin L residues interacting with the ligand atoms via hydrogen bonds or Ï€-Ï€ stacking are labelled with the corresponding single letter residue code along with their position in the protein sequence. The hydrogen bonds and Ï€-Ï€ stacking are displayed using yellow and red dotted lines, respectively.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <p>(<b>a</b>) Radius of gyration for TMPRSS2 in complex with T1, T2 and T3, (<b>b</b>) RMSD for TMPRSS2 in complex with T1, T2 and T3, (<b>c</b>) RMSF for TMPRSS2 in complex with T1, T2 and T3, (<b>d</b>) RMSD of T1, T2 and T3, and (<b>e</b>) Distance of the center of mass of T1, T2 and T3 from the substrate binding residue D435 in TMPRSS2.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9</span></a></div>   <p>(<b>a</b>) Radius of gyration for cathepsin L in complex with C1, C2 and C3, (<b>b</b>) RMSD for cathepsin L in complex with C1, C2 and C3, (<b>c</b>) RMSF for cathepsin L in complex with C1, C2 and C3, (<b>d</b>) RMSD of C1, C2 and C3, (<b>e</b>) Distance of the center of mass of C1, C2 and C3 from the catalytic residue C25 in cathepsin L, and (<b>f</b>) Distance of the center of mass of C1, C2 and C3 from the catalytic residue H163 in cathepsin L.</p>   </div> <div class="table-wrap_UNKNOWN" id="molecules-25-03822-t001" orientation="portrait" position="float">
molecules-25-03822-t001_Table 1<span class="label" tagx="label" title="label">Table 1</span> <p>Herbal sources of top 9 phytochemical inhibitors of TMPRSS2. For each phytochemical, the table gives the symbol, docked binding energy, common name and plant source. Plant sources which have been reported to have antiviral or anti-inflammatory use in traditional medicine literature are shown in bold and marked with an [*] sign.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Phytochemical Symbol</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Binding Energy (kcal/mol)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Common Name</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Plant Source</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">T1</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’9.6</td> <td align="center" valign="middle" rowspan="1" colspan="1">Qingdainone</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Strobilanthes cusia</i></b><b>[*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">T2</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’9.6</td> <td align="center" valign="middle" rowspan="1" colspan="1">Edgeworoside C</td> <td align="center" valign="middle" rowspan="1" colspan="1"><i>Edgeworthia gardneri</i></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">T3</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’9.6</td> <td align="center" valign="middle" rowspan="1" colspan="1">Adlumidine</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Fumaria indica</i></b><b>[*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">T4</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’9.3</td> <td align="center" valign="middle" rowspan="1" colspan="1">Pseudo-Î±-Colubrine</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Strychnos nux-vomica</i></b><b>[*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">T5</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’9.3</td> <td align="center" valign="middle" rowspan="1" colspan="1">Bicuculline</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Fumaria indica</i></b><b>[*]<i>, Corydalis govaniana</i> [*]<i>, Nerium oleander</i> [*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">T6</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’9.3</td> <td align="center" valign="middle" rowspan="1" colspan="1">Strychnine <i>N</i>-oxide</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Strychnos nux-vomica</i></b><b>[*]<i>, Strychnos ignatii</i> [*]<i>, Strychnos colubrina</i> [*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">T7</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’9.2</td> <td align="center" valign="middle" rowspan="1" colspan="1">Î±-Colubrine</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Strychnos nux-vomica</i></b><b>[*]<i>, Strychnos ignatii</i> [*]<i>, Strychnos colubrina</i> [*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">T8</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’9.2</td> <td align="center" valign="middle" rowspan="1" colspan="1">Egenine</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Fumaria vaillantii</i></b><b>[*]</b></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">T9</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’9.2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2-Hydroxy-3-methoxystrychnine</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b><i>Strychnos nux-vomica</i></b><b>[*]</b></td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="molecules-25-03822-t002" orientation="portrait" position="float">
molecules-25-03822-t002_Table 2<span class="label" tagx="label" title="label">Table 2</span> <p>Herbal sources of top 9 phytochemical inhibitors of Cathepsin L. For each phytochemical, the table gives the symbol, docked binding energy, common name and plant source. Plant sources which have been reported to have antiviral or anti-inflammatory use in traditional medicine literature are shown in bold and marked with an [*] sign.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Phytochemical Symbol</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Binding Energy (kcal/mol)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Common Name</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Plant Source</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">C1</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’8.9</td> <td align="center" valign="middle" rowspan="1" colspan="1">Ararobinol</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Senna occidentalis</i></b><b>[*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">C2</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’8.3</td> <td align="center" valign="middle" rowspan="1" colspan="1">(+)-Oxoturkiyenine</td> <td align="center" valign="middle" rowspan="1" colspan="1"><i>Hypecoum pendulum</i></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">C3</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’8.3</td> <td align="center" valign="middle" rowspan="1" colspan="1">3Alpha,17Alpha-Cinchophylline</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Cinchona calisaya</i></b><b>[*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">C4</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’8.2</td> <td align="center" valign="middle" rowspan="1" colspan="1">Rugosanine B</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Ziziphus rugosa</i></b><b>[*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">C5</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’8.2</td> <td align="center" valign="middle" rowspan="1" colspan="1">Trichotomine</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Clerodendrum trichotomum</i></b><b>[*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">C6</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’8.1</td> <td align="center" valign="middle" rowspan="1" colspan="1">Tectol</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Tectona grandis</i></b><b>[*]<i>, Tecomella undulata</i> [*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">C7</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’8.1</td> <td align="center" valign="middle" rowspan="1" colspan="1">Silymonin</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Silybum marianum</i></b><b>[*]</b></td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">C8</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’8</td> <td align="center" valign="middle" rowspan="1" colspan="1">Picrasidine M</td> <td align="center" valign="middle" rowspan="1" colspan="1"><b><i>Picrasma quassioides</i></b><b>[*]</b></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">C9</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’8</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Trisjuglone</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b><i>Juglans regia</i></b><b>[*]</b></td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="molecules-25-03822-t003" orientation="portrait" position="float">
molecules-25-03822-t003_Table 3<span class="label" tagx="label" title="label">Table 3</span> <p>MM-PBSA based binding energy for top three inhibitors of TMPRSS2 and cathepsin L.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Protein-Ligand Complex</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Binding Energy (kcal/mol)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Van Der Waals Energy (kcal/mol)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Electrostatic Energy (kcal/mol)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Polar Solvation Energy (kcal/mol)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">SASA Energy (kcal/mol)</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">TMPRSS2-T1</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’39.15 Â± 2.799</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’54.285 Â± 2.903</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’3.031 Â±1.439</td> <td align="center" valign="middle" rowspan="1" colspan="1">22.844 Â± 2.44</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’4.678 Â± 0.237</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">TMPRSS2-T2</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’30.284 Â±3.585</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’49.048 Â± 3.838</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’12.501 Â± 4.884</td> <td align="center" valign="middle" rowspan="1" colspan="1">35.978 Â± 5.226</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’4.712 Â± 0.320</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">TMPRSS2-T3</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’27.386 Â± 2.077</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’39.379 Â± 2.109</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’8.846 Â± 1.423</td> <td align="center" valign="middle" rowspan="1" colspan="1">24.359 Â± 2.157</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’3.52 Â± 0.210</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">cathepsin L-C1</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’22.384 Â± 3.420</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’25.296 Â± 3.127</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’2.214 Â± 1.661</td> <td align="center" valign="middle" rowspan="1" colspan="1">7.988 Â± 4.103</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’2.861 Â± 0.366</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">cathepsin L-C2</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’20.577 Â± 3.600</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’30.129 Â± 3.154</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’4.572 Â± 2.138</td> <td align="center" valign="middle" rowspan="1" colspan="1">16.891 Â± 3.533</td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’2.767 Â± 0.234</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">cathepsin L-C3</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’26.156 Â± 3.433</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’37.165 Â± 3.308</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’2.093 Â± 1.379</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">16.958 Â± 4.513</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’3.856 Â± 0.319</td> </tr> </tbody> </table> </div> </div>  </body></html>