<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Mol Plant Pathol</span><span class="iso-abbrev" title="iso-abbrev">Mol Plant Pathol</span><span class="doi" title="doi">10.1111/(ISSN)1364-3703</span><span class="publisher-id" title="publisher-id">MPP</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Molecular Plant Pathology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1464-6722</span><span class="issn-epub" tagx="issn" title="issn-epub">1364-3703</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7694671">7694671</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/33084136">33084136</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1111/mpp.13003">10.1111/mpp.13003</a></span><span class="publisher-id" title="publisher-id">MPP13003</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Article</span>: <span class="subject" title="subject">Original Articles</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Chorismate mutase and isochorismatase, two potential effectors of the migratory nematode <i>Hirschmanniella oryzae</i>, increase host susceptibility by manipulating secondary metabolite content of rice</div> <div class="alt-title" title="alt-title">
BAUTERS et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bauters</span><span class="given-names" tagx="given-names" title="given-names">Lander</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-5456-9305</span><a href="#mpp13003-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kyndt</span><span class="given-names" tagx="given-names" title="given-names">Tina</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-5267-5013</span><a href="#mpp13003-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Meyer</span><span class="given-names" tagx="given-names" title="given-names">Tim</span></span><a href="#mpp13003-aff-0002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morreel</span><span class="given-names" tagx="given-names" title="given-names">Kris</span></span><a href="#mpp13003-aff-0003"><sup>3</sup></a><a href="#mpp13003-aff-0004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boerjan</span><span class="given-names" tagx="given-names" title="given-names">Wout</span></span><a href="#mpp13003-aff-0003"><sup>3</sup></a><a href="#mpp13003-aff-0004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lefevere</span><span class="given-names" tagx="given-names" title="given-names">Hannes</span></span><a href="#mpp13003-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gheysen</span><span class="given-names" tagx="given-names" title="given-names">Godelieve</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-1929-5059</span><a href="#mpp13003-aff-0001"><sup>1</sup></a><span class="address" title="address"><span class="email" tagx="email" title="email">Godelieve.Gheysen@UGent.be</span></span></span></div><span class="citation_author_institution" id="mpp13003-aff-0001">[1], <span class="named-content" title="named-content">Department of Biotechnology</span><span class="named-content" title="named-content">Faculty of Bioscience Engineering</span><span class="institution" title="institution">Ghent University</span><span class="city" tagx="city" title="city">Ghent</span><span class="country" tagx="country" title="country">Belgium</span></span><span class="citation_author_institution" id="mpp13003-aff-0002">[2], <span class="named-content" title="named-content">Department of Data Analysis and Mathematical Modelling</span><span class="named-content" title="named-content">Faculty of Bioscience Engineering</span><span class="institution" title="institution">Ghent University</span><span class="city" tagx="city" title="city">Ghent</span><span class="country" tagx="country" title="country">Belgium</span></span><span class="citation_author_institution" id="mpp13003-aff-0003">[3], <span class="institution" title="institution">VIBâ€�UGent Center for Plant Systems Biology</span><span class="city" tagx="city" title="city">Ghent</span><span class="country" tagx="country" title="country">Belgium</span></span><span class="citation_author_institution" id="mpp13003-aff-0004">[4], <span class="named-content" title="named-content">Department of Plant Biotechnology and Bioinformatics</span><span class="named-content" title="named-content">Faculty of Sciences</span><span class="institution" title="institution">Ghent University</span><span class="city" tagx="city" title="city">Ghent</span><span class="country" tagx="country" title="country">Belgium</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><b>Correspondence</b><br /><br /> Godelieve Gheysen, Department of Biotechnology, Faculty of Bioscience Engineering, Ghent University, Ghent, Belgium.<br /><br /> Email: <span class="email" tagx="email" title="email">Godelieve.Gheysen@UGent.be</span><br /><br /></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-10-10</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-12</span></span><span class="volume" tagx="volume" title="volume">21</span><span class="issue" tagx="issue" title="issue">12</span><span class="issue-id" tagx="issue-id" title="issue-id">10.1002/mpp.v21.12</span><span class="fpage" tagx="fpage" title="fpage">1634</span><span class="lpage" tagx="lpage" title="lpage">1646</span><span class="history" title="history"><span class="received" title="received">received: 2020-6-28</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-9-07</span><span class="accepted" title="accepted">accepted: 2020-9-07</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="file:MPP-21-1634.pdf" title="self-uri">file:MPP-21-1634.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p><i>Hirschmanniella oryzae</i> is one of the most devastating nematodes on rice, leading to substantial yield losses. Effector proteins aid the nematode during the infection process by subduing plant defence responses. In this research we characterized two potential <i>H.Â oryzae</i> effector proteins, chorismate mutase (HoCM) and isochorismatase (HoICM), and investigated their enzymatic activity and their role in plant immunity. Both HoCM and HoICM proved to be enzymatically active in complementation tests in mutant <i>Escherichia coli</i> strains. Infection success by the migratory nematode <i>H.Â oryzae</i> was significantly higher in transgenic rice lines constitutively expressing <i>HoCM</i> or <i>HoICM</i>. Expression of <i>HoCM</i>, but not <i>HoICM</i>, increased rice susceptibility against the sedentary nematode <i>Meloidogyne graminicola</i> also. Transcriptome and metabolome analyses indicated reductions in secondary metabolites in the transgenic rice plants expressing the potential nematode effectors. The results presented here demonstrate that both HoCM and HoICM suppress the host immune system and that this may be accomplished by lowering secondary metabolite levels in the plant.</p> </div> <div class="abstract" title="abstract"> <p>Chorismate mutase and isochorismatase from the plant parasitic nematode <i>Hirschmanniella oryzae</i> are able to decrease immunity of rice by interfering with the phenylpropanoid pathway. </p><div class="boxed-text" tagx="boxed-text" title="boxed-text"> <div class="graphic" position="anchor" id="nlm-graphic-1" orientation="portrait" /> </div> <p /> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
chorismate mutase</div> <div class="kwd" title="kwd">
defense</div> <div class="kwd" title="kwd"><i>Hirschmanniella oryzae</i></div> <div class="kwd" title="kwd">
isochorismatase</div> <div class="kwd" title="kwd">
nematode</div> <div class="kwd" title="kwd"><i>Oryza sativa</i></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Fonds Wetenschappelijk Onderzoek </span><span class="institution-id" title="institution-id">10.13039/501100003130</span></div> </div><span class="award-id" tagx="award-id" title="award-id">3G009829W</span></div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="cover-date" value="December 2020" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_JATSPMC version:5.9.4 mode:remove_FC converted:27.11.2020" /> </div> </div> <p content-type="self-citation"><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bauters</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kyndt</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De Meyer</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Chorismate mutase and isochorismatase, two potential effectors of the migratory nematode <i>Hirschmanniella oryzae</i>, increase host susceptibility by manipulating secondary metabolite content of rice</span>. <span class="source" tagx="source" title="source">Mol. Plant Pathol.</span><span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">21</span>:<span class="fpage" tagx="fpage" title="fpage">1634</span>â€"<span class="lpage" tagx="lpage" title="lpage">1646</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/mpp.13003">10.1111/mpp.13003</a></span></span></p> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
INTRODUCTION</div> <p><i>Hirschmanniella oryzae</i> is one of the most devastating pests in rice fields worldwide. Because <i>H.Â oryzae</i> is well adapted to flooded conditions, it is a major problem in flooded rice ecosystems (Babatola, <a href="#mpp13003-bib-0002">1981</a>). This migratory nematode penetrates the root and migrates through the aerenchyma, completing its life cycle in about 33Â days (Karakas, <a href="#mpp13003-bib-0035">2004</a>). Plantâ€�parasitic nematodes are equipped with an arsenal of cell wallâ€�modifying proteins that are secreted through the stylet into the host tissue to hydrolyse cell walls (Davis et al., <a href="#mpp13003-bib-0008">2011</a>). Sedentary nematodes establish a feeding site, whereby it is important to remain undetected by the plant to prevent an adequate defence response. Effectors are secreted by this type of nematode to alter plant cell metabolism to their advantage and to attenuate the defence mechanisms of the host (Postma et al., <a href="#mpp13003-bib-0050">2012</a>; Hewezi and Baum, <a href="#mpp13003-bib-0027">2013</a>; Jaouannet et al., <a href="#mpp13003-bib-0031">2013</a>). Migratory nematodes, like <i>H.Â oryzae</i>, stay mobile during their entire life cycle. They keep on migrating through the root whilst feeding. Although this lifestyle might enable them to outrun local plant defences and effector proteins inhibiting plant defence might seem unnecessary, several potential effector proteins have been predicted from the transcriptome of plantâ€�parasitic migratory nematodes (Haegeman et al., <a href="#mpp13003-bib-0024">2011</a>; Nicol et al., <a href="#mpp13003-bib-0045">2012</a>; Bauters et al., <a href="#mpp13003-bib-0003">2014</a>). This implies that migratory nematode species also invest energy in attenuating the defence responses of their host.</p> <p>The plant hormone salicylic acid (SA) plays an important role as a signalling molecule during the defence reaction on pathogen infection. SA can be produced through two distinct pathways starting from chorismate, the final product of the shikimate pathway: the isochorismate synthase (ICS) pathway and the phenylalanine ammonia lyase (PAL) pathway (leading to SA and phenylpropanoids). Several chorismateâ€�derived compounds play a role in plant physiology and the defence system, for example, SA, auxin, lignin, and flavonoids. Changes in concentrations of these compounds can affect the plant defence against pathogen infection (Jones et al., <a href="#mpp13003-bib-0034">2007</a>; Nahar et al., <a href="#mpp13003-bib-0044">2012</a>). Interestingly, transcripts encoding chorismate mutase and isochorismatase were detected in the transcriptome of <i>H.Â oryzae</i> (Bauters et al., <a href="#mpp13003-bib-0003">2014</a>). Both enzymes can interfere in the host production of salicylic acid and/or phenylpropanoids. Chorismate mutase catalyses the conversion of chorismate to prephenate. This enzyme is present in plants and bacteria but had not been reported in animals until its discovery in <i>Meloidogyne javanica</i> (Lambert et al., <a href="#mpp13003-bib-0039">1999</a>). Since then it has been reported in several other plantâ€�parasitic nematodes (Jones et al., <a href="#mpp13003-bib-0033">2003</a>; Huang et al., <a href="#mpp13003-bib-0028">2005</a>; Vanholme et al., <a href="#mpp13003-bib-0060">2009</a>). It has been shown that chorismate mutase, secreted by plantâ€�pathogenic organisms, is able to alter plant cell development by impairing the development of lateral roots and vascular tissue (Doyle and Lambert, <a href="#mpp13003-bib-0013">2003</a>). More recent research provided experimental evidence that a secreted chorismate mutase originating from <i>Ustilago maydis</i> reduced SA levels in host plants (Djamei et al., <a href="#mpp13003-bib-0011">2011</a>). A decrease in SA content was also observed in tobacco leaves transiently expressing a chorismate mutase from <i>MeloidogyneÂ incognita</i> (<i>Miâ€�CM3</i>) upon infection by the oomycete <i>Phytophthora capsici</i>. In addition, constitutive expression of <i>Miâ€�CM3</i> increased susceptibility of tobacco plants to <i>M.Â incognita</i> infection (Wang et al., <a href="#mpp13003-bib-0064">2018</a>). The gene encoding isochorismatase has not been characterized in nematodes before, but it was reported to be present in the genome of <i>Meloidogyne hapla</i> and the transcriptome of <i>Rotylenchulus reniformis</i> (Opperman et al., <a href="#mpp13003-bib-0047">2008</a>; Wubben et al., <a href="#mpp13003-bib-0065">2010</a>). Moreover, isochorismatase was detected in the secretome of plantâ€�pathogenic fungi, but not in nonpathogenic species (Soanes et al., <a href="#mpp13003-bib-0057">2008</a>) and it was reported to reduce SA content in plants on infection. Isochorismatase converts isochorismate to 2,3â€�dihydroxyâ€�2,3â€�dihydrobenzoate and pyruvate. Isochorismate is an intermediate in the biosynthesis of SA, hence isochorismatase is capable of reducing the pool of isochorismate available for SA synthesis (Liu et al., <a href="#mpp13003-bib-0042">2014</a>).</p> <p>In this research, the genetic structure and protein properties of chorismate mutase (<i>HoCM</i>) and isochorismatase (<i>HoICM</i>) isolated from <i>H.Â oryzae</i> are described. Their activity is demonstrated by complementation of chorismate mutase (<i>CM</i>) and isochorismatase (<i>ICM</i>) deficient <i>Escherichia coli</i> strains, and infection assays with <i>H.Â oryzae</i> on transgenic plants showed that both proteins increase susceptibility of the host plant to nematode infection. Moreover, <i>HoCM,</i> but not <i>HoICM</i> expression, had an effect on infection with the sedentary nematode <i>Meloidogyne graminicola</i>. Finally, RNAâ€�Seq performed on rice plants expressing <i>HoICM</i> or <i>HoCM</i> revealed that both proteins interfere with secondary metabolic pathways, a result supported by metabolome analysis.</p> </div> <div class="results" title="results"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
RESULTS</div> <div class="geneticstructureofhocmandhoicm" title="sec"><span class="label" tagx="label" title="label">2.1</span><div class="title" tagx="title" title="title">
Genetic structure of <i>HoCM</i> and <i>HoICM</i></div> <p>The genomic sequence of chorismate mutase (<i>HoCM,</i> KP297892) consists of 942 nucleotides from start to stop codon, and contains two introns of 72 and 123 nucleotides. The isochorismatase gene (<i>HoICM,</i> KP297893) consists of 822 nucleotides from start to stop codon and includes two introns of 69 and 81 base pairs (FigureÂ <a href="#mpp13003-fig-0001">1a,b</a>). All exon/intron transitions contain the consensus splice site GT/AG, except for one <i>HoICM</i> noncanonical splice site GT/CG. The first 60 nucleotides of <i>HoCM</i> code for a secretion signal, targeting the encoded protein to the classical secretory pathway. Expression in the secretory glands was shown by in situ hybridization in previous research (Bauters et al., <a href="#mpp13003-bib-0003">2014</a>). No secretion signal was detected in <i>HoICM</i>. Several attempts were made to perform an in situ hybridization for <i>HoICM</i> transcripts, but all of them were negative or showed ambiguous results. <i>HoICM</i> was PCRâ€�amplified from genomic DNA extracted from a single nematode, three times, from an independent single handâ€�picked nematode. <i>HoICM</i> could be amplified in all three experiments, supporting that this is an endogenous <i>H.Â oryzae</i> gene rather than being derived from any contaminating material. The fact that this type of isochorismatase (ICM) is specific to plantâ€�parasitic species (Bauters et al., <a href="#mpp13003-bib-0003">2014</a>) and that it is secreted by filamentous pathogens through a nonclassical secreted pathway (Liu et al., <a href="#mpp13003-bib-0042">2014</a>) points towards an effector function in nematodes as well.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Gene structure and partial protein sequence of <i>HoCM</i> and <i>HoICM</i>. (a) <i>HoCM</i> has a total length of 942 base pairs at the genomic level, containing two small introns. The protein consists of three different domains: an Nâ€�terminal signal peptide (SP), a serine/histidineâ€�rich domain (S/HRD), and a chorismate mutase domain (CMD) (PF01817). The alignment shows the conserved catalytic residues in several plantâ€�parasitic nematode species (Mjav, <i>Meloidogyne javanica</i>; Mare, <i>Meloidogyne arenaria</i>; Minc, <i>Meloidogyne incognita</i>; GPLIN, <i>Globodera pallida</i>). Eight conserved putative catalytic residues characterized in chorismate mutase of <i>Mycobacterium tuberculosis</i> and/or <i>Escherichia coli</i> are marked with filled dots. (b) <i>HoICM</i> is 822Â bp long and contains two small introns. It contains an isochorismatase domain (PF00857), but no predicted Nâ€�terminal signal peptide. The three putative catalytic residues are marked with filled dots. Sequence names correspond to the protein sequences encoded by different nematodes, downloaded from the Wormbase Parasite database (Ho, <i>Hirschmanniella oryzae</i>, ALI53582; Dd, <i>Ditylenchus destructor</i>, Dd_10059; Hg, <i>Heterodera glycines</i>, Hetgly.G000001356; Gr, <i>Globodera rostochiensis</i>, GROS_g01640.t1; Mf, <i>Meloidogyne floridiensis</i>, scf7180000418840.g2997; Mi, <i>M. incognita</i>, Minc3s04290g35947; Mj, <i>M. javanica</i>, M.javanica_Scaff754g009582; Ma, <i>M. arenaria</i>, M.arenaria_Scaff10008g072685; Me, <i>Meloidogyne enterolobii</i>, scaffold11435_cov287.g14711; Mh, <i>Meloidogyne hapla</i>, MhA1_Contig1915.frz3.gene2). *Sequence adjusted (part of intron used as exon) due to probable mistake in gene prediction</p>   </div> </div> <div class="importantcatalyticresiduesareconservedinbothhocmandhoicm" title="sec"><span class="label" tagx="label" title="label">2.2</span><div class="title" tagx="title" title="title">
Important catalytic residues are conserved in both HoCM and HoICM</div> <p>The HoCM protein consists of three domains, including the signal peptide. The Nâ€�terminal domain of the mature protein has no similarities with other domains in public databases, but it has two motifs, rich in serine and histidine, respectively, hence the name serine/histidineâ€�rich domain (S/HRD). The Câ€�terminal part is similar to the Pfam chorismate mutase (CM) type II domain (PF01817) (CMD). Alignment of HoCM and other CM sequences deduced from the genomes of cyst and rootâ€�knot nematodes showed conservation of catalytic residues as initially characterized in <i>E.Â coli</i> and <i>Mycobacterium tuberculosis</i> (Lee et al., <a href="#mpp13003-bib-0040">1995</a>; Ã–kvist et al., <a href="#mpp13003-bib-0046">2006</a>). The eight conserved catalytic residues are highlighted in FigureÂ <a href="#mpp13003-fig-0001">1a</a>. All residues are conserved in <i>H.Â oryzae</i>, but far less in cyst and rootâ€�knot nematodes.</p> <p>An ICM domain (PF00857) is predicted in the Câ€�terminal part of the HoICM protein. Homologous sequences are present in the expressed sequence tag (EST) database of <i>R. reniformis</i>, <i>Radopholus similis</i>, and <i>Heterodera glycines</i>. No homologues were found in the genome of <i>Globodera pallida</i>, but several homologous sequences were retrieved from the genomes of different <i>Meloidogyne</i> spp. and <i>Globodera Â rostochiensis</i>. Catalytic residues in ICMs have been identified in <i>Oleispira antarctica</i> and <i>Pseudomonas aeruginosa</i>, and are conserved in <i>H.Â oryzae</i> and other plantâ€�parasitic nematodes (FigureÂ <a href="#mpp13003-fig-0001">1b</a>) (Parsons et al., <a href="#mpp13003-bib-0048">2003</a>; Goral et al., <a href="#mpp13003-bib-0021">2012</a>).</p> </div> <div class="predictingtheproteinstructureofhocmandhoicm" title="sec"><span class="label" tagx="label" title="label">2.3</span><div class="title" tagx="title" title="title">
Predicting the protein structure of HoCM and HoICM</div> <p>CMs are classified into two main groups according to their structure: the AroH and AroQ classes. Proteins of the rare AroH class, represented by the CM of <i>Bacillus subtilis</i>, have both Î±â€�helices and Î²â€�sheets in their structure (Chook et al., <a href="#mpp13003-bib-0007">1993</a>). On the other hand, proteins of the AroQ class are more abundant and are represented by the CM structure of <i>E.Â coli</i>, containing only Î±â€�helices (Lee et al., <a href="#mpp13003-bib-0040">1995</a>). Secondary and tertiary structure predictions showed that HoCM is composed of Î±â€�helices and loops, without Î²â€�sheets, indicating that HoCM is a member of the AroQ class. A 3D model was constructed using Phyre2 with the CM structures of <i>M.Â tuberculosis</i> (PDB code 2FP1) and <i>Yersinia pestis</i> (2GBB) as a template. The predicted tertiary structure of HoCM is shown in FigureÂ <a href="#mpp13003-fig-0002">2a</a>. The CM domain of HoCM has the same predicted topology as the global structure of an AroQ<sub>Î³</sub> protein, which consists of six Î±â€�helices connected by loops. The S/HRD region (Nâ€�terminal) of HoCM lacked homology and was modelled ab initio with only 72% of the residues modelled with over 90% confidence.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Predicted tertiary structure of HoCM and HoICM. (a) HoCM. The structure of the mature protein is shown (without secretion signal). It was constructed using homology modelling with chorismate mutase models of <i>Mycobacterium tuberculosis</i> and <i>Yersinia pestis</i> as templates. The Câ€�terminal chorismate mutase domain (grey) consists of two large Î±â€�helices in the middle, surrounded by four smaller Î±â€�helices. The Nâ€�terminal part (black) is predicted to have two Î±â€�helices. Eight catalytic residues are shown as stick models. (b) HoICM. The sixâ€�stranded parallel Î²â€�sheet (black) is in the centre of the molecule, surrounded by three Î±â€�helices at one side and two at the other side. At the bottom (three) and the top left corner (one) in this view of the molecule there are four 3<sub>10</sub>â€�helices. The catalytic triad (Cys164â€� Asp42â€�Lys131) is labelled</p>   </div> <p>ICMâ€�like hydrolases (ILH) are a large family of enzymes divided into several structural subgroups by the Structural Classification of Proteins (SCOP release 2.03), that is, YecD, <i>N</i>â€�carbamoylsarcosine amidohydrolase, phenazine biosynthesis protein (PhzD), pyrazinamidase/nicotinamidase, ribonuclease MAR1, and YcaC. These proteins belong to the class of â€œalphaâ€�betaâ€�alpha sandwichâ€� according to the CATH (Class Architecture Topology/fold Homologous superfamily) structural classification, adopting a Rossman fold. The 3D model of HoICM was constructed using six different ICM structures (PDB code: 1NF9, 1NBA, 2FQ1, 3TB4, 3IRV, and 3KL2), creating a model for which 95% of the residues were modelled with more than 90% confidence with Phyre2. The model shows a molecule with a Rossman fold; a sixâ€�stranded parallel Î²â€�sheet flanked by three big Î±â€�helices at one side and two at the other. Next to these Î±â€�helices, there are four putative 3<sub>10</sub>â€�helices present. The presumed catalytic triad is shown in FigureÂ <a href="#mpp13003-fig-0002">2b</a>. The catalytic centre where isochorismate can bind has been described in <i>Oleispira antarctica</i> and is conserved in other plantâ€�parasitic nematodes (PPNs) (Goral et al., <a href="#mpp13003-bib-0021">2012</a>). PyMOL (v. 1.6) structural alignment assigns HoICM to the structural subgroup of the phenazine biosynthesis proteins (root mean square deviation of 0.437).</p> </div> <div class="hocmandhoicmcomplemente.Â colimutants" title="sec"><span class="label" tagx="label" title="label">2.4</span><div class="title" tagx="title" title="title"><i>HoCM</i> and <i>HoICM</i> complement <i>E.Â coli</i> mutants</div> <p>The activity of two different <i>HoCM</i> constructs was assessed by a complementation assay in a CMâ€�deficient <i>E.Â coli</i> strain (KA12/pKIMPâ€�UAUC) grown on dropout medium without phenylalanine. One construct contained the full <i>CM</i> coding sequence without the predicted signal peptide, the second only consisted of the catalytic region. The positive control was the <i>E.Â coli</i> mutant expressing a <i>B.Â subtilis CM</i>. The results confirmed CM activity of the HoCM protein (FigureÂ <a href="#mpp13003-fig-0003">3a</a>). The fact that mutants with both protein constructs grew equally well reveals that the S/HRD region is not necessary for CM activity.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 3</span></a></div>   <p>Complementation tests of HoCM and HoICM. (a) Complementation assay of the chorismate mutase (CM)â€�deficient <i>Escherichia coli</i> strain KA12/pKIMPâ€�UAUC with different constructs of HoCM. The left Petri dish shows bacterial growth on M9cY medium supplemented with phenylalanine, the right one bacterial growth on M9cY medium without additions. +, positive control (CM of <i>Bacillus subtilis</i>); âˆ’, negative control (empty vector pQE30â€�UA); Cat, catalytic domain of CM; Full, mature CM protein. (b) Complementation assay of <i>entB</i>â€�mutated <i>E.Â coli</i> AN192. AN192 cells were complemented with an empty vector (pDEST17) (white) or HoICM (grey). Cells were grown in liquid medium with different concentrations of bipyridyl to create ironâ€�limiting conditions. Bacterial growth was observed by measuring optical density (<i>y</i> axis) at three different time points (an average was taken from four bacterial cultures). Treatments were compared with the control (no bipyridyl) and statistically analysed by a Mannâ€"Whitney test. Each experiment was performed twice with similar results. Different letters above the graph indicate the significant differences between the different treatments</p>   </div> <p>The activity of HoICM was tested using an <i>ICM</i> (<i>entB</i>) deficient strain (AN192) incapable of producing the siderophore enterobactin, an iron chelator. The growth of AN192 and AN192 complemented with <i>HoICM</i> under ironâ€�limiting conditions, created by adding bipyridyl, was monitored by measuring optical density. Results showed that AN192 cells harbouring <i>HoICM</i> grew much better under ironâ€�limiting conditions compared to the empty vector control (FigureÂ <a href="#mpp13003-fig-0003">3b</a>). Fifteen hours postinoculation, there still was a significant difference (<i>p</i> valueÂ <div class="â_UNKNOWN">
HoICM grew as well as the cells under conditions without bipyridyl. AN192 cells that carried the empty vector could not grow well under ironâ€�limiting conditions, the growth rate of this control remaining far below the growth rate of <i>HoICM</i>â€�complemented cells under the two most stringent conditions (150 and 200Â ÂµM bipyridyl). This result proves that HoICM can complement the lack of ICM activity in the AN192 strain.</div></p> </div> <div class="ricelinesexpressinghocmorhoicmaremoresusceptibletonematodeinfection" title="sec"><span class="label" tagx="label" title="label">2.5</span><div class="title" tagx="title" title="title">
Rice lines expressing <i>HoCM</i> or <i>HoICM</i> are more susceptible to nematode infection</div> <p>Transgenic rice lines constitutively expressing one of two versions of <i>HoCM</i> (<i>HoCM</i> without signal peptide (<i>HoCM_FULL</i>) or only the catalytic domain of <i>HoCM</i> (<i>HoCM_CAT</i>)) or <i>HoICM</i> were generated starting from <i>O.Â sativa</i> "Nipponbare." Lines transformed with a vector without insert (empty vector) were used as control. Five and two independent lines were obtained for <i>HoCM_CAT</i> and <i>HoCM_FULL</i>, respectively. For <i>HoICM</i>, eight independent lines were obtained, but only three of them produced enough viable seeds to perform further experiments. No clear phenotypes were observed between independent lines during the entire life cycle of the plant. The presence of the construct was verified at the genomic level by PCR with geneâ€�specific primers. Expression was validated by quantitative reverse transcription PCR (RTâ€�qPCR; see TableÂ <a href="#mpp13003-sup-0005">S1</a>a,b). The transgenic lines were tested for their susceptibility to nematode infection (FigureÂ <a href="#mpp13003-fig-0004">4</a>). Constitutive expression of the nematode <i>HoCM</i> or <i>HoICM</i> enhanced susceptibility of the rice plants against <i>H.Â oryzae</i>. All three different constructs showed similar results. The transgenic lines were used in an infection assay with the sedentary rootâ€�knot nematode <i>M.Â graminicola</i> as well. In contrast with the results obtained for <i>H.Â oryzae</i>, two lines of <i>HoICM</i>â€�expressing rice plants showed no difference in <i>M.Â graminicola</i> infection levels with the control. On the other hand, both HoCM constructs enhanced susceptibility against <i>M.Â graminicola</i>, similar to the results obtained for <i>H.Â oryzae</i> (FigureÂ <a href="#mpp13003-fig-0004">4</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 4</span></a></div>   <p>Effect of expression of HoICM, HoCM_FULL, and HoCM_CAT in rice on susceptibility against <i>Hirschmanniella oryzae</i> (a) and <i>Meloidogyne graminicola</i> (b). Bars represent the average number of nematodes (<i>H.Â oryzae</i>) or galls (<i>M.Â graminicola</i>) of eight infected plants per line. The number of galls was expressed relative to the control, which was set to 1. Asterisks indicate values significantly different from the control (empty vector) according to a nonparametric Mannâ€"Whitney test (<i>p</i>Â <div class="â_UNKNOWN">
p = .052. Error bars represent the standard error of the mean. Data for <i>M.Â graminicola</i> infection are expressed as relative values. The two repetitions of each experiment are shown. The infection was conducted on one line per construct</div></p>   </div> </div> <div class="severalgenesinvolvedinplantdefencearedownâ€�regulatedinricelinesconstitutivelyexpressingthepotentialnematodeeffectors" title="sec"><span class="label" tagx="label" title="label">2.6</span><div class="title" tagx="title" title="title">
Several genes involved in plant defence are downâ€�regulated in rice lines constitutively expressing the potential nematode effectors</div> <p>RNAâ€�Seq was performed on one line per construct to study transcriptional changes in rice plants expressing <i>HoCM_CAT</i>, <i>HoCM_FULL</i>, or <i>HoICM</i>. Almost 150 million sequence fragments were generated. Almost 95% of all reads could be mapped to the rice genome (IRGSPâ€�1.0, see TableÂ <a href="#mpp13003-sup-0006">S2</a>).</p> <p>Expression levels of all genes in the transgenic lines were compared to those in the line transformed with the empty vector. Differentially expressed genes (DEGs) with a false discovery rate (FDR) &amp;lt;0.05 were compared using blastp to protein data for Viridiplantae available in the SwissProt database for annotation purposes. Compared to the empty vector line, the transgenic line expressing <i>HoCM_FULL</i> had 55 DEGs (13 upâ€�regulated, 42 downâ€�regulated). When <i>HoCM_CAT</i> was expressed, 196 DEGs were detected (67 up, 129 down), and 105 DEGs (36 up, 69 down) were found in plants expressing <i>HoICM</i>. Approximately half of these genes had a significant (bitâ€�scoreÂ &amp;gt;Â 50) hit when compared with blast against the SwissProt database. These annotated genes were classified into six groups according to their putative function (FileÂ <a href="#mpp13003-sup-0011">S1</a>). Expression of 20 genes was downâ€�regulated in the transgenic plants expressing any of the three constructs, among which were two glutathione <i>S</i>â€�transferases, a monoâ€�oxygenase, and geranylgeranyl pyrophosphate synthase. A nonproteinâ€�coding transcript was upâ€�regulated in all three constructs. Nine genes had the same trend in transgenic plants with either CM construct, among which were a downâ€�regulation of genes involved in steroid biosynthesis and auxin transport. It is interesting to note that many genes involved in signalling, stress response, and secondary metabolite production were differentially expressed upon in planta expression of both nematode genes. An overview of the significantly differentially regulated genes can be found in FileÂ <a href="#mpp13003-sup-0011">S1</a>.</p> <p>The reliability of the RNAâ€�Seq results was validated by evaluating the expression of 12 randomly chosen DEGs by RTâ€�qPCR. Eleven of the genes confirmed RNAâ€�Seq results; only one gene showed a contrasting expression pattern: the gene coding for histone H2B.3 was significantly upâ€�regulated according to the RNAâ€�Seq data (log<sub>2</sub>FCÂ =Â 1.6) in plants expressing <i>HoCM_CAT</i>, while it was downâ€�regulated according to the RTâ€�qPCR data. The expression of two genes, coding for glutathione <i>S</i>â€�transferase and a receptorâ€�like protein kinase, was so strongly downâ€�regulated that it was undetectable by RTâ€�qPCR in the transgenic lines (see FigureÂ <a href="#mpp13003-sup-0001">S1</a>). The clear correspondence with RTâ€�qPCR results for these 12 genes (<i>p</i>Â =Â .006, binomial test) supports the reliability of the generated RNAâ€�Seq data.</p> </div> <div class="geneontologyenrichmentanalysisshowsadecreaseindefencecompoundbiosynthesisterms" title="sec"><span class="label" tagx="label" title="label">2.7</span><div class="title" tagx="title" title="title">
Gene ontology enrichment analysis shows a decrease in defence compound biosynthesis terms</div> <p>Parametric analysis of gene set enrichment (PAGE) was conducted using the log<sub>2</sub> fold changes of all genes as input. All levels of gene ontology (GO) in the category â€œBiological Processâ€� were considered, with a significance level of 0.05 (FDR). GO terms with a calculated absolute <i>Z</i>â€�score higher than 4 for at least one of the three constructs are visualized in FigureÂ <a href="#mpp13003-fig-0005">5</a>. The expression of each different construct induces a repression of the â€œsecondary metabolic processâ€� GO in rice. Looking at the child terms of this GO term, the repression was due to a reduction in the GO term â€œPhenylpropanoid metabolic processâ€�. The GO term â€œLignin catabolic processâ€� is reduced in <i>HoCM_FULLâ€�</i> and <i>HoICM</i>â€�expressing plants. â€œDiterpene phytoalexin metabolic processâ€� is suppressed in plants expressing <i>HoCM</i>. Rice plants expressing either one of the <i>HoCM</i> constructs are also repressed in the general â€œDefense responseâ€�, while all three lines are reduced in â€œOxidation reductionâ€�.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 5</span></a></div>   <p>Gene ontology (GO) term enrichment analysis, showing the significantly enriched GOterms in the category â€œBiological processâ€�. GO terms for which at least one of the three constructs resulted in plant transcripts with significant difference with an absolute <i>Z</i>â€�score (<i>y</i> axis) value of 4 are shown in the graph. a and aÊ¹ indicate an ancestor term (a) with its child terms (aÊ¹). All GO terms represented in this graph are significantly less abundant in the transgenic lines (<i>Z</i>â€�scoreÂ <div class="â_UNKNOWN">   </div></p></div> </div> <div class="metabolomeanalysissuggeststhathocmdownâ€�regulatesphenylpropanoidbiosynthesis" title="sec"><span class="label" tagx="label" title="label">2.8</span><div class="title" tagx="title" title="title">
Metabolome analysis suggests that HoCM downâ€�regulates phenylpropanoid biosynthesis</div> <p>No differences in SA content in shoots or roots were detected in transgenic lines compared to the control (FigureÂ <a href="#mpp13003-sup-0002">S2</a>) using the protocol described by Haeck et al. (<a href="#mpp13003-bib-0023">2018</a>). To obtain more insight into the effects of either <i>ICM</i> or <i>CM</i> on phenolic metabolism in rice roots, a comparative liquid chromatographyâ€�mass spectrometry (LCâ€�MS, see Experimental procedures) profiling was performed on the methanol extracts from <i>HoICM</i>â€�expressing plants and the control line (referred to as the <i>HoICM</i> set) and from <i>HoCM_FULL</i>, <i>HoCM_CAT</i>, and the control line (referred to as the <i>HoCM</i> set). Taking both sets together, approximately 13,884 compounds (as estimated by the number of <i>m</i>/<i>z</i> feature groups) were profiled. The abundances of these compounds were analysed by applying an analysis of variance (ANOVA) model separately for each set, and correcting for the large number of tests by calculating the FDR (Î±Â =Â .01). The numbers of soâ€�obtained differential compounds for the <i>HoICM</i> and <i>HoCM</i> sets were 53 (TableÂ <a href="#mpp13003-sup-0007">S3</a>) and 17 (TableÂ <a href="#mpp13003-sup-0008">S4</a>), respectively. In the <i>HoCM</i> set, 1 and 16 compounds displayed higher and lower levels in the transgenic lines as compared to the control lines. However, all of them were of low abundance, hindering the recording of goodâ€�quality MS/MS spectra for structural elucidation (TableÂ <a href="#mpp13003-sup-0009">S5</a>). For the <i>HoICM</i> set, 7 and 46 compounds showed higher and lower levels in the transgenic lines as compared to the control lines, respectively. The most differentially accumulating compound in the <i>HoICM</i> line was structurally characterized as 1â€�<i>O</i>â€�ethylâ€�6â€�<i>O</i>â€�glycoloyl hexose. In addition, another of the top accumulating compounds was very similar in the MS/MS spectrum and was identified as (1â€�<i>O</i>â€�ethyl)hexosyl(6â€�&amp;gt;)ethylene glycol glycolate. Both compounds indicate that expression of <i>HoICM</i> leads to the accumulation of glycolic acid. In addition, many of the compounds that showed diminished levels in the <i>HoICM</i> lines were probably fatty acidâ€�derived molecules that contained a glycine moiety based on the chemical formula and the detection of glycineâ€�representing MS/MS product ions (TableÂ <a href="#mpp13003-sup-0009">S5</a> and FiguresÂ <a href="#mpp13003-sup-0003">S3</a> and <a href="#mpp13003-sup-0004">S4</a>).</p> <p>Because it proved to be difficult to characterize the unknown compounds that were statistically different in abundance between treatment and control, as an alternative approach only those metabolites that could be matched with a known compound were taken into consideration. To increase confidence in the proposed matches, only compounds with a fragmentation score higher than 0.8 were kept. This pipeline resulted in a list of only 17 compounds, 16 of which were linked to the phenylpropanoid pathway. Although not significantly different, for <i>HoCM</i>â€�expressing lines 13 compounds had lower levels in the two independent lines per construct, compared to the control, suggesting downâ€�regulation of the phenylpropanoid pathway. A twoâ€�way ANOVA using â€œrelative abundance of each componentâ€� as variable and â€œcontrol vs transgenicâ€� and â€œindependent lineâ€� as fixed and random factors, respectively, was conducted. A general lower abundance of compounds from the phenylpropanoid pathway in <i>HoCM</i>â€�expressing lines (<i>p</i> = .008) was observed without any effect of the â€œindependent lineâ€� factor (<i>p</i> = .5). Because there was an interaction effect between the fixed and the random factor for <i>HoICM</i>â€�expressing plants, no clear conclusion could be drawn (see TableÂ <a href="#mpp13003-tbl-0001">1</a>).</p> <div class="table-wrap_UNKNOWN" id="mpp13003-tbl-0001" content-type="TABLE" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p>Relative values for characterized secondary metabolite abundance in rice lines expressing HoCM or HoICM compared to the control line</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1" /> <th align="left" valign="top" rowspan="1" colspan="1">Compound ID</th> <th align="left" valign="top" rowspan="1" colspan="1">HoCM_FULL2</th> <th align="left" valign="top" rowspan="1" colspan="1">HoCM_CAT2</th> <th align="left" valign="top" rowspan="1" colspan="1">HoCM_CAT3</th> <th align="left" valign="top" rowspan="1" colspan="1">HoICM1</th> <th align="left" valign="top" rowspan="1" colspan="1">HoICM4</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">Tricin(4'â€�<i>O</i>â€�8)Gâ€�5/7â€�<i>O</i>â€�hexoside 3</td> <td align="char" char="." rowspan="1" colspan="1">0.4023</td> <td align="char" char="." rowspan="1" colspan="1">0.2582</td> <td align="char" char="." rowspan="1" colspan="1">1.1936</td> <td align="char" char="." rowspan="1" colspan="1">4.6489</td> <td align="char" char="." rowspan="1" colspan="1">1.2003</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">Vanillic acid 1</td> <td align="char" char="." rowspan="1" colspan="1">0.3996</td> <td align="char" char="." rowspan="1" colspan="1">0.5069</td> <td align="char" char="." rowspan="1" colspan="1">0.4375</td> <td align="char" char="." rowspan="1" colspan="1">1.2130</td> <td align="char" char="." rowspan="1" colspan="1">0.4053</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">Syringic acid 4â€�<i>O</i>â€�hexoside</td> <td align="char" char="." rowspan="1" colspan="1">0.3837</td> <td align="char" char="." rowspan="1" colspan="1">0.4415</td> <td align="char" char="." rowspan="1" colspan="1">0.4575</td> <td align="char" char="." rowspan="1" colspan="1">1.2970</td> <td align="char" char="." rowspan="1" colspan="1">0.7224</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">Vanilloyl hexose</td> <td align="char" char="." rowspan="1" colspan="1">0.1163</td> <td align="char" char="." rowspan="1" colspan="1">0.1129</td> <td align="char" char="." rowspan="1" colspan="1">0.1869</td> <td align="char" char="." rowspan="1" colspan="1">0.6548</td> <td align="char" char="." rowspan="1" colspan="1">0.6579</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">4â€�Hydroxybenzoic acid</td> <td align="char" char="." rowspan="1" colspan="1">0.3824</td> <td align="char" char="." rowspan="1" colspan="1">0.4140</td> <td align="char" char="." rowspan="1" colspan="1">0.6249</td> <td align="char" char="." rowspan="1" colspan="1">2.1176</td> <td align="char" char="." rowspan="1" colspan="1">1.6102</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">Feruloyl quinate 1</td> <td align="char" char="." rowspan="1" colspan="1">0.3513</td> <td align="char" char="." rowspan="1" colspan="1">0.6880</td> <td align="char" char="." rowspan="1" colspan="1">0.5078</td> <td align="char" char="." rowspan="1" colspan="1">1.2792</td> <td align="char" char="." rowspan="1" colspan="1">0.5786</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">Ferulic acid 4â€�<i>O</i>â€�hexoside 2 (â€�dehydrated hexose)</td> <td align="char" char="." rowspan="1" colspan="1">1.0309</td> <td align="char" char="." rowspan="1" colspan="1">0.9846</td> <td align="char" char="." rowspan="1" colspan="1">0.5018</td> <td align="char" char="." rowspan="1" colspan="1">1.4663</td> <td align="char" char="." rowspan="1" colspan="1">0.5709</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">Sinapoyl hexose</td> <td align="char" char="." rowspan="1" colspan="1">0.4679</td> <td align="char" char="." rowspan="1" colspan="1">0.2185</td> <td align="char" char="." rowspan="1" colspan="1">0.6017</td> <td align="char" char="." rowspan="1" colspan="1">1.0430</td> <td align="char" char="." rowspan="1" colspan="1">1.0901</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1"><i>p</i>â€�Coumaric acid</td> <td align="char" char="." rowspan="1" colspan="1">0.3871</td> <td align="char" char="." rowspan="1" colspan="1">0.4913</td> <td align="char" char="." rowspan="1" colspan="1">0.6304</td> <td align="char" char="." rowspan="1" colspan="1">1.7322</td> <td align="char" char="." rowspan="1" colspan="1">1.0924</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">Caffeoyl shikimic acid 1</td> <td align="char" char="." rowspan="1" colspan="1">0.3424</td> <td align="char" char="." rowspan="1" colspan="1">0.6480</td> <td align="char" char="." rowspan="1" colspan="1">0.5111</td> <td align="char" char="." rowspan="1" colspan="1">0.5509</td> <td align="char" char="." rowspan="1" colspan="1">0.3112</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1"><i>t</i>â€�Ferulic acid</td> <td align="char" char="." rowspan="1" colspan="1">0.3413</td> <td align="char" char="." rowspan="1" colspan="1">0.3526</td> <td align="char" char="." rowspan="1" colspan="1">0.3254</td> <td align="char" char="." rowspan="1" colspan="1">0.6086</td> <td align="char" char="." rowspan="1" colspan="1">0.0763</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1"><i>m</i>â€�Hydroxycinnamic acid</td> <td align="char" char="." rowspan="1" colspan="1">0.3302</td> <td align="char" char="." rowspan="1" colspan="1">0.4654</td> <td align="char" char="." rowspan="1" colspan="1">0.5502</td> <td align="char" char="." rowspan="1" colspan="1">1.7059</td> <td align="char" char="." rowspan="1" colspan="1">1.1033</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">G(red8â€�5)GÂ +Â hexose</td> <td align="char" char="." rowspan="1" colspan="1">0.2210</td> <td align="char" char="." rowspan="1" colspan="1">0.8074</td> <td align="char" char="." rowspan="1" colspan="1">0.6627</td> <td align="char" char="." rowspan="1" colspan="1">0.6986</td> <td align="char" char="." rowspan="1" colspan="1">0.2126</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">G(8â€�8)G hexoside 1</td> <td align="char" char="." rowspan="1" colspan="1">0.3728</td> <td align="char" char="." rowspan="1" colspan="1">0.9640</td> <td align="char" char="." rowspan="1" colspan="1">0.5749</td> <td align="char" char="." rowspan="1" colspan="1">0.5736</td> <td align="char" char="." rowspan="1" colspan="1">0.2960</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">G(red8â€�5)G 8â€�<i>O</i>â€�hexoside</td> <td align="char" char="." rowspan="1" colspan="1">0.3414</td> <td align="char" char="." rowspan="1" colspan="1">0.3420</td> <td align="char" char="." rowspan="1" colspan="1">0.6841</td> <td align="char" char="." rowspan="1" colspan="1">3.3869</td> <td align="char" char="." rowspan="1" colspan="1">0.7431</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Phenylpropanoid</td> <td align="left" rowspan="1" colspan="1">G(8â€�5)G [âˆ’H2O] 2</td> <td align="char" char="." rowspan="1" colspan="1">0.3202</td> <td align="char" char="." rowspan="1" colspan="1">0.2526</td> <td align="char" char="." rowspan="1" colspan="1">0.5464</td> <td align="char" char="." rowspan="1" colspan="1">1.1624</td> <td align="char" char="." rowspan="1" colspan="1">0.3864</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="char" char="." colspan="2" rowspan="1"><i>p</i> = .008</td> <td align="char" char="." rowspan="1" colspan="1" /> <td align="char" char="." colspan="2" rowspan="1"><i>p</i> = .865</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Jasmonate</td> <td align="left" rowspan="1" colspan="1">12â€�Hydroxy jasmonic acid sulphate</td> <td align="char" char="." rowspan="1" colspan="1">0.4016</td> <td align="char" char="." rowspan="1" colspan="1">0.5912</td> <td align="char" char="." rowspan="1" colspan="1">0.4852</td> <td align="char" char="." rowspan="1" colspan="1">0.8332</td> <td align="char" char="." rowspan="1" colspan="1">0.7970</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="mpp13003-ntgp-0001"> <div class="title" tagx="title" title="title">
Note</div> <div class="fn-type-" title=""> <p>Compounds that have a lower abundance compared to the control have a value &amp;lt; 1. <i>p</i> values show the result of a twoâ€�way analysis of variance comparing relative abundances of compounds from the phenylpropanoid pathway between transgenic lines and control.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> </div> </div> <div class="discussion" title="discussion"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
DISCUSSION</div> <p>Although the presence of a gene encoding CM has been reported in transcriptomes of migratory PPNs before (Bauters et al., <a href="#mpp13003-bib-0003">2014</a>; Haegeman et al., <a href="#mpp13003-bib-0024">2011</a>), this is the first time it has been functionally characterized in a migratory nematode. Chorismate is the final product of the shikimate pathway, a pathway that has only been reported in plants and microorganisms, but not in animals (Herrmann, <a href="#mpp13003-bib-0026">1995</a>). The first report of CM from a nematode was in <i>M.Â javanica</i> (Lambert et al., <a href="#mpp13003-bib-0039">1999</a>), where it was thought to be involved in the establishment of a feeding site within the host. Indoleâ€�3â€�acetic acid (auxin, IAA) levels were reported to be lowered due to expression of this <i>CM</i> in plant roots, which had an influence on root development. This observation is in contradiction with the findings that IAA is important for feeding site initiation, while the nematodeâ€�secreted CM was hypothesized to be a vital element in this process (Goverse et al., <a href="#mpp13003-bib-0022">2000</a>).</p> <p>The CM protein produced by <i>H.Â oryzae</i> contains a unique Nâ€�terminal serine/histidineâ€�rich domain and a conserved catalytic domain with CM activity. Rootâ€�knot nematodes lack an Nâ€�terminal domain preceding the CM catalytic domain. Cyst nematode CMs do have an Nâ€�terminal domain, but lack homology to the serine/histidine rich domain found in <i>H.Â oryzae</i>. Vanholme et al. (<a href="#mpp13003-bib-0060">2009</a>) proved that the Nâ€�terminal sequence was not needed for CM activity in <i>HeteroderaÂ schachtii</i>, similar to the hereâ€�obtained results for HoCM. More research is needed to elucidate the function of the Nâ€�terminal region present in cyst nematodes and <i>H.Â oryzae</i>. Because it is not needed for enzymatic activity, it is possible that this region has another function, for example interaction with host proteins.</p> <p>A geneencoding ICM was reported in the transcriptome of <i>H.Â oryzae</i> (Bauters et al., <a href="#mpp13003-bib-0003">2014</a>), in <i>R. reniformis</i> (Wubben et al., <a href="#mpp13003-bib-0065">2010</a>), and in the genome of the cyst nematode <i>G. rostochiensis</i> (Evesâ€�van den Akker et al., <a href="#mpp13003-bib-0017">2016</a>). As Opperman et al. (<a href="#mpp13003-bib-0047">2008</a>) suggested while studying the genome of <i>M.Â hapla</i>, <i>ICM</i> has probably been acquired by horizontal gene transfer from bacteria. The sequence of <i>ICM</i> does not contain a predicted Nâ€�terminal secretion signal, but the fact that the gene is conserved in other PPNs, while being absent in nematodes with a different lifestyle (Bauters et al., <a href="#mpp13003-bib-0003">2014</a>), led us to believe it might be involved in plant parasitism. This hypothesis is strengthened by the fact that ICM is also present in the secretome and in the culture supernatants of phytopathogenic fungi, helping to increase susceptibility of the host (Soanes et al., <a href="#mpp13003-bib-0057">2008</a>; Elâ€�Bebany et al., <a href="#mpp13003-bib-0016">2010</a>; Ismail et al., <a href="#mpp13003-bib-0030">2014</a>). ICM is probably secreted by a nonclassical secretory pathway in fungi (Liu et al., <a href="#mpp13003-bib-0042">2014</a>), but some fungal ICMs are predicted to have a secretion signal (Ismail et al., <a href="#mpp13003-bib-0030">2014</a>). Further research is necessary to provide experimental evidence of ICM secretion by nematodes.</p> <p>Although it is generally assumed that SA mediates resistance against biotrophic pathogens (Glazebrook, <a href="#mpp13003-bib-0020">2005</a>) and <i>H.Â oryzae</i> kills the cells it is feeding from, in contrast to sedentary nematodes (De Vleesschauwer et al., <a href="#mpp13003-bib-0009">2013</a>), SA was shown to be important for an adequate defence response against <i>H.Â oryzae</i> infection in rice (Nahar et al., <a href="#mpp13003-bib-0044">2012</a>). On the other hand, the SA biosynthesis pathway appears changed after <i>H.Â oryzae</i> infection with upâ€�regulation of <i>OsPAL1</i> and downâ€�regulation of <i>OsICS</i> in shoots (Kyndt et al., <a href="#mpp13003-bib-0038">2012</a>). SA biosynthesis in plants can occur through two different pathways, both starting from chorismate as a precursor (Vlot et al., <a href="#mpp13003-bib-0063">2009</a>; reviewed in Lefevere et al., <a href="#mpp13003-bib-0041">2020</a>). A secreted CM from <i>U. maydis</i> was able to reduce SA accumulation upon pathogen infection (Djamei et al., <a href="#mpp13003-bib-0011">2011</a>). ICM, secreted by two fungal pathogens (<i>Verticillium dahliae</i> and <i>Phytophthora sojae</i>), reduces the SA content at least 3â€�fold upon transient expression in <i>Nicotiana Â benthamiana</i> leaves, rendering the plant more vulnerable to various pathogens (Liu et al., <a href="#mpp13003-bib-0042">2014</a>). According to the results presented in this research, HoICM and HoCM do not have an effect on SA. However, upon interpreting these results, one has to keep in mind that the RNAâ€�Seq and metabolome data presented here came from plants that were not challenged with a pathogen, in contrast to the data of Liu et al. (<a href="#mpp13003-bib-0042">2014</a>) and Djamei et al. (<a href="#mpp13003-bib-0011">2011</a>). A GO enrichment analysis performed on RNAâ€�Seq data from <i>HoICMâ€�</i>, <i>HoCM_FULLâ€�</i>, and <i>HoCM_CAT</i>â€�expressing rice lines, together with a metabolome analysis, points towards changes in phenylpropanoid content. The GO term â€œsecondary metabolic processâ€� and its child term â€œphenylpropanoid metabolic processâ€� were reduced in the three different transgenic lines. Both ICM and CM act upon chorismate (or chorismateâ€�derived metabolites), which is the starting point of the phenylpropanoid pathway (Dempsey et al., <a href="#mpp13003-bib-0010">2011</a>). These data were backed up by results extracted from the metabolome analysis on the transgenic lines. Looking at identified compounds, phenylpropanoid content was lower in plants expressing <i>HoCM</i>, whereas results for plants expressing <i>HoICM</i> were less clear. The fact that the <i>PAL</i> genes, considered as the starting point of the phenylpropanoid pathway, are often upâ€� and downâ€�regulated in resistant and susceptible interactions, respectively, shows that phenylpropanoids are important defence molecules. This was shown in several plantâ€"pathogen interaction experiments (Gao et al., <a href="#mpp13003-bib-0019">2008</a>; Uehara et al., <a href="#mpp13003-bib-0059">2010</a>; Feng et al., <a href="#mpp13003-bib-0018">2011</a>; Khanam et al., <a href="#mpp13003-bib-0036">2018</a>). In contrast to our observations, Djamei and colleagues (<a href="#mpp13003-bib-0011">2011</a>) reported an increase in phenylpropanoid biosynthesis products in plants infected with <i>U.Â maydis</i>, which indicates that chorismate is directed into the phenylpropanoid pathway. CM originating from <i>U.Â maydis</i> was also able to reduce the SA content of maize upon infection (Djamei et al., <a href="#mpp13003-bib-0011">2011</a>). A recent study showed that transient expression of CM originating from the PPN <i>M. incognita</i> was able to reduce the SA content of tobacco leaves after <i>Phytophthora</i> infection (Wang et al., <a href="#mpp13003-bib-0064">2018</a>). Our results showed that unchallenged rice plants expressing <i>HoCM</i> or <i>HoICM</i> did not change their SA content in roots or shoots. This observation was strengthened by the fact that no SAâ€�responsive genes (e.g., <i>OsWRKY45</i> or <i>OsNPR1</i>) were differentially regulated according to the transcriptome analysis.</p> <p>Genes involved in â€œDiterpenoid phytoalexin metabolic processâ€� were inhibited in <i>HoCM_FULLâ€�</i> and <i>HoCM_CAT</i>â€�expressing plants. Diterpenoid phytoalexins (DPs) have several functions in plants. Their antimicrobial properties have been well documented. DPs accumulate in infection spots where necrotic tissue starts to develop (Huffaker et al., <a href="#mpp13003-bib-0029">2011</a>). Transgenic lines deficient in <i>OsCPS4</i>, a phytoalexin biosynthetic gene, showed increased susceptibility to rice blast fungus (Toyomasu et al., <a href="#mpp13003-bib-0058">2014</a>). Also in nematodes it was shown that induction of DP biosynthesis increases resistance against nematodes (Verbeek et al., <a href="#mpp13003-bib-0062">2019</a>). Some terpenoids might be involved in nematode resistance by accumulation in lesions near the infection site (Veech, <a href="#mpp13003-bib-0061">1982</a>), so a reduced terpenoid content could influence susceptibility to nematodes. The mechanism by which expression of <i>HoCM</i> or <i>HoICM</i> influences the diterpenoid phytoalexin biosynthesis pathway remains unclear. So far, it was only suggested that the phenylpropanoid and terpenoid pathway compete for carbon and that carbon flow into these two pathways is tightly regulated (Xie et al., <a href="#mpp13003-bib-0066">2008</a>).</p> <p>The data generated in this paper suggest that the immune system of the host is altered by lowering secondary metabolite content upon secretion of CM and ICM by <i>H.Â oryzae</i>. Further research is needed to pinpoint the exact metabolites responsible for the drop in immunity and the mechanism by which HoCM and HoICM are able to do this.</p> </div> <div class="experimentalprocedures" title="sec"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
EXPERIMENTAL PROCEDURES</div> <div class="nematodednaextractionandgeneamplification" title="sec"><span class="label" tagx="label" title="label">4.1</span><div class="title" tagx="title" title="title">
Nematode DNA extraction and gene amplification</div> <p>Genomic DNA was extracted from a batch of mixed stages of <i>H.Â oryzae</i> or a single handâ€�picked nematode as previously described by Bolla et al. (<a href="#mpp13003-bib-0006">1988</a>), with some minor modifications. Briefly, nematodes were sonicated three times for 10Â s to break cell membranes. Samples were incubated for 1Â hr at 65Â°C after which DNA was isolated using phenol and chloroform. Isolated DNA and a cDNA library (Bauters et al., <a href="#mpp13003-bib-0003">2014</a>) were used to amplify <i>HoCM</i> and <i>HoICM</i> by PCR with geneâ€�specific primers (see TableÂ <a href="#mpp13003-sup-0010">S6</a>). PCR fragments were cloned into pGEMâ€�T (Promega) according to the standard thymine/adenine protocol and sequenced by the Sanger method (LGC) to verify the insert.</p> </div> <div class="bioinformaticanalysis" title="sec"><span class="label" tagx="label" title="label">4.2</span><div class="title" tagx="title" title="title">
Bioinformatic analysis</div> <p>Several blast programs (blastn and blastx), available on the NCBI server (<a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>), were used for sequence similarity searches in EST and nonredundant protein databases. Alignments were generated with BioEdit (Hall, <a href="#mpp13003-bib-0025">1999</a>). Putative Nâ€�terminal secretion signals were detected by SignalP 4.1 (Petersen et al., <a href="#mpp13003-bib-0049">2011</a>). Protein domains were identified by searching the Pfam 27.0 database (Punta et al., <a href="#mpp13003-bib-0051">2011</a>). Protein 3D models were constructed with the Protein Homology/anologY Recognition Engine v. 2.0 (Phyre2) (Punta et al., <a href="#mpp13003-bib-0051">2011</a>) and visualized by PyMOL 1.6 (<a href="http://www.pymol.org/">http://www.pymol.org/</a>).</p> </div> <div class="activityassays" title="sec"><span class="label" tagx="label" title="label">4.3</span><div class="title" tagx="title" title="title">
Activity assays</div> <p>Two <i>HoCM</i> fragments (HoCMFull and HoCMCat) were amplified by PCR using the full cDNA sequence as template (primer sequences can be found in TableÂ <a href="#mpp13003-sup-0010">S6</a>). The first fragment comprised the full sequence of CM without sequence encoding the signal peptide, the second fragment only contained the sequence encoding the catalytic domain (base pair 258 to 747). The complementation assay was performed as described by Vanholme et al. (<a href="#mpp13003-bib-0060">2009</a>).</p> <p>HoICM activity was tested using <i>entB</i>â€�mutated AN192 <i>E.Â coli</i> cells. AN192 cells are unable to grow in ironâ€�limiting conditions. <i>HoICM</i> was amplified by a twoâ€�step PCR starting from cDNA as template to add attB sites to the fragment (primer sequences can be found in TableÂ <a href="#mpp13003-sup-0010">S6</a>). The fragment was cloned into the destination vector pDEST17 using the standard Gateway technology protocol (Life Technologies). The pDEST17 vector carrying <i>HoICM</i> was transformed via heat shock into <i>E.Â coli</i> AN192 cells. <i>E.Â coli</i> AN192 cells transformed with the empty vector were used as negative control. These cells were grown overnight at 37Â°C in Luriaâ€�Bertani (LB) medium with 100Â mg/L carbenicillin. Optical density (OD) was measured at 600Â nm and equal amounts of bacteria were added to fresh medium containing different concentrations of 2,2Ê¹â€�bipyridyl to create ironâ€�limiting conditions (0, 100, 150, and 200Â ÂµM). These different types of media were inoculated with <i>E.Â coli</i> AN192 cells carrying <i>HoICM</i> or the empty vector. OD was measured at 600Â nm at 15, 24, and 48Â hr postinoculation. Results were statistically analysed with a Mannâ€"Whitney test. The experiment was performed twice.</p> </div> <div class="planttransformation" title="sec"><span class="label" tagx="label" title="label">4.4</span><div class="title" tagx="title" title="title">
Plant transformation</div> <p>cDNA sequences of <i>HoICM</i> and <i>HoCM</i> were amplified by PCR on a cDNA library (Bauters et al., <a href="#mpp13003-bib-0003">2014</a>). Primer sequences are provided in TableÂ <a href="#mpp13003-sup-0010">S6</a>. Expression vectors (pMBb7Fm21GWâ€�UBIL, obtained from the Center for Plant Systems Biology [VIB, Belgium]) were created for three different constructs: <i>HoICM</i> (containing the complete open reading frame), <i>HoCM</i> without secretion signal (HoCM_FULL), and only the catalytic domain of <i>HoCM</i> (<i>HoCM_CAT</i>). Vectors were created by standard Gateway cloning. Cloned sequences as well as position in the vector were verified by Sanger sequencing. Rice transformation was done as described by Ji et al. (<a href="#mpp13003-bib-0032">2015</a>). Leaf samples were taken and DNA was extracted to confirm the presence of the gene of interest by PCR using geneâ€�specific primers (TableÂ <a href="#mpp13003-sup-0010">S6</a>). DNA was extracted by heating a leaf sample at 95Â°C for 15Â min in 400Â Âµl of extraction buffer (200Â mM Tris.HCl, 250Â mM NaCl, 25Â mM ethylenediaminetetraacetic acid [EDTA] and 0.5% sodium dodecyl sulphate [SDS]). Afterwards 50Â Âµl chloroform was added. The sample was centrifuged and 200Â Âµl of isopropanol was added to 200Â Âµl of the upper aqueous phase to precipitate the DNA.</p> </div> <div class="rnaextraction,sequencing,andvalidation" title="sec"><span class="label" tagx="label" title="label">4.5</span><div class="title" tagx="title" title="title">
RNAextraction, sequencing, and validation</div> <p>Rice plants were grown under greenhouse conditions at 30Â°C. In the flowering stage, leaf samples were taken from three plants of a line transformed with the empty vector, and three plants of a line expressing <i>HoCM_FULL</i> or <i>HoCM_CAT</i>. Due to limited seed availability, the line for which most seeds per transformant were available was chosen. Because of technical reasons only two plants were used for lines expressing <i>HoICM</i>. RNA was extracted using the Nucleospin RNA extraction kit (Macheryâ€�Nagel) according to the manufacturer's guidelines with an additional DNase I (Thermo Scientific) treatment. At least 4Â Âµg of RNA of each sample was sent for RNA sequencing (Microsynth). An mRNA library was constructed using the Illumina TruSeq RNA sample preparation kit. The library was sequenced using a NextSeq platform (Illumina). To validate RNAâ€�Seq results, a set of genes was selected randomly to perform RTâ€�qPCR. RNA was extracted with the RNeasy Plant Mini kit (Qiagen). cDNA was synthesized and a RTâ€�qPCR was performed with <i>EIF5c</i> and <i>EXP NARCAI</i> as reference genes to normalize expression results. Primers can be found in TableÂ <a href="#mpp13003-sup-0010">S6</a>.</p> </div> <div class="mappingandgeneexpressionprofiling" title="sec"><span class="label" tagx="label" title="label">4.6</span><div class="title" tagx="title" title="title">
Mapping and gene expression profiling</div> <p>Reads from all libraries were mapped to the <i>O.Â sativa</i> ssp. <i>japonica</i> reference genome (IRGSPâ€�1.0) using the Spliced Transcripts Alignment to a Reference (STAR, v2.3.1) software with standard settings (Dobin et al., <a href="#mpp13003-bib-0012">2013</a>). The GenomicFeatures (relying on Ensemble's BioMart osativa_eg_gene dataset) and GenomicAlignment R/BioConductor packages were used to obtain an overview of all exons per gene (makeTranscriptDbFromBiomart and exonsBy functions), respectively, to quantify expression per sample for these genes starting from the STAR output (readGAlignmentsFromBam and countOverlaps functions). Additional gene annotation information was obtained from Ensembl (BioMart, canonical identifiers only). Differential gene expression was assessed using the Râ€�package â€œEdgeRâ€� version 3.8.5 (Robinson et al., <a href="#mpp13003-bib-0055">2010</a>). Data were analysed using the generalized linear model (GLM) approach. Data were normalized using normalization factors calculated by the Trimmed Median of Mâ€�values method with standard settings as implemented in the EdgeR package (Robinson and Oshlack, <a href="#mpp13003-bib-0056">2010</a>). The proposed GLM was fitted to the design matrix using the Coxâ€�Reid profileâ€�adjusted likelihood method (McCarthy et al., <a href="#mpp13003-bib-0043">2012</a>). Differential gene expression levels (Log<sub>2</sub> fold change) were calculated and Benjamini and Hochberg adjusted (Benjamini and Hochberg, <a href="#mpp13003-bib-0004">1995</a>) false discovery rates (FDR) were estimated to detect DEGs (FDRÂ <div class="â_UNKNOWN" href="#mpp13003-bib-0014">
2010). Gene identifiers with their corresponding Log<sub>2</sub>FC were used as input in a Parametric Analysis of Gene Set Enrichment (PAGE) (Kim and Volsky, <a href="#mpp13003-bib-0037">2005</a>). The Hochberg multitest adjustment method was performed (<i>p</i>Â &amp;lt; .05).</div></p> </div> <div class="phenolicprofiling" title="sec"><span class="label" tagx="label" title="label">4.7</span><div class="title" tagx="title" title="title">
Phenolic profiling</div> <p>Roots (approximately 5Â mg of dry weight) were homogenized in liquid nitrogen and extracted with 1Â ml of methanol. The methanol extract was then evaporated and the pellet dissolved in 200Â Î¼l of water/cyclohexane (1:1, vol/vol). Then, 10Â Î¼l of the aqueous phase was analysed via reverseâ€�phase ultrahigh performance liquid chromatography using an Acquity UPLC BEH C18 column. In addition to full MS analysis, a pooled sample was subjected to dataâ€�dependent MS/MS analysis (exclusion durationÂ =Â 10Â s). Integration and alignment of the <i>m</i>/<i>z</i> features were performed via Progenesis QI software v. 2.1 (Waters Corporation). The normalization was set on â€œexternal standardsâ€� and was based on the dry weight of the samples.</p> <p>For statistical analysis, the data were split into two sets. A first set comprised the two independent <i>HoICM</i> expression lines (HoICM1 and HoICM4) and the corresponding control line (<i>HoICM</i> set). In the second set, one <i>HoCM_FULL</i> expression line, two independent <i>HoCM_CAT</i> expression lines (HoCM_CAT2 and HoCM_CAT3), and the corresponding control line were included (<i>HoCM</i> set). Both sets were subjected to a oneâ€�way ANOVA using the <i>lm()</i> function in R v. 3.4.2 (R_Core_Team, <a href="#mpp13003-bib-0052">2017</a>) with a weighting factor included for the first set. For post hoc testing, the <i>pairwise.t.test(p.adjust</i> = <i>â€�bonferroniâ€�)</i> function was applied. The false discovery rate (Î±Â =Â .01) was computed on the ANOVA model significance using the <i>p.adjust(method</i> = <i>â€�fdrâ€�)</i> function. Additional filtering on the prerequisites of (a) a significant post hoc test for each of the transgenic lines and (b) abundance changes in the same direction for all transgenic lines as compared to the control lines yielded 54 and 17 differential <i>m</i>/<i>z</i> features (corresponding to 53 and 17 <i>m</i>/<i>z</i> feature groups) for the <i>HoICM</i> and <i>HoCM</i> sets, respectively.</p> <p>Structural annotation was performed using a retention time window of 1Â min, and both precursor ion and MS/MS identity searches. The precursor ion search (10Â ppm tolerance) was based on a compound database constructed via instant JChem (ChemAxon), whereas MS/MS identities were obtained by matching against an inâ€�house mass spectral database (200Â ppm fragment tolerance). As none of the MS/MS spectra of the differential <i>m</i>/<i>z</i> features could be identified via database matching, MS/MS spectral elucidation was attempted (see FileÂ <a href="#mpp13003-sup-0012">S2</a>) using gas phase fragmentation rules and in silico MS/MS elucidation software, that is, CSI:FingerID (BÃ¶cker and DÃ¼hrkop, <a href="#mpp13003-bib-0005">2016</a>; DÃ¼hrkop et al., <a href="#mpp13003-bib-0015">2015</a>) and CFMâ€�ID (Allen et al., <a href="#mpp13003-bib-0001">2015</a>). A more detailed description of the phenolic profiling protocol can be found in FileÂ <a href="#mpp13003-sup-0012">S2</a>.</p> </div> <div class="nematodeinfectionassay" title="sec"><span class="label" tagx="label" title="label">4.8</span><div class="title" tagx="title" title="title">
Nematode infection assay</div> <p>The <i>M.Â graminicola</i> culture, originally isolated in the Philippines, was maintained on <i>O.Â sativa</i> "Nipponbare" in potting soil at 27Â°C (16/8Â hr light regime). <i>H.Â oryzae</i> was obtained from infected rice roots sampled from fields in Myanmar. Nematodes were extracted from rice roots using a modified Baerman funnel technique. Rice seeds were first germinated on wet tissue paper at 27Â°C in the dark for 3Â days. Germinated seedlings were transferred to PVC tubes containing sand and absorbent polymer substrate (Reversat et al., <a href="#mpp13003-bib-0054">1999</a>). Twoâ€�week old plants (T<sub>1</sub> or T<sub>2</sub> generation) were infected with 200 (<i>M.Â graminicola</i>) or 300 (<i>H.Â oryzae</i>) nematodes/plant. Two weeks after infection, nematode susceptibility was assessed by counting the number of galls (<i>M.Â graminicola</i>) or nematodes (<i>H.Â oryzae</i>) per root. Galls/nematodes were visualized by boiling roots in 0.8% acetic acid and 0.013% acid fuchsin for 3Â min, after which they were washed under running tap water and destained in acid glycerol. Infection assays with <i>H.Â oryzae</i> were only conducted on a single line per construct due to difficulties in culturing <i>H.Â oryzae</i> under laboratory conditions.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data">  <p><b>FIGURE S1</b> Validation of RNAâ€�Seq results. Expression values were calculated compared to expression in the empty vector control (expression level set at 1). Normalization was done with two reference genes (EIF5C and EXP NARCAI). Expression values are plotted on a Log2â€�scale. Expression values are a mean of two biological replicates (calculated by taking the average of three technical replicates). Statistical analysis was performed with REST 2009 software (asterisk indicates a significant difference, <i>p</i>Â <div class="â_UNKNOWN">
Sâ€�transferase; ser/thr_kin, serine/threonineâ€�protein kinase; hist, histone H2B.3; LRR, LRR receptorâ€�like serine/threonineâ€�protein kinase; Râ€�like PK, receptorâ€�like protein kinase; Î±â€�galac, Î±â€�1,2â€�galactosyltransferase; RIK, protein RIK; PM_ATPase, plasma membrane ATPase; gluc6, glucoseâ€�6â€�phosphate/phosphate translocator 2. In between brackets the log2FC according to RNAâ€�Seq is given. Bars represent standard errors, calculated with REST2009 software using Taylorâ€™s series</div></p>  <div class="media" title="media"><a href="MPP-21-1634-s001.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>FIGURE S2</b> Root (A) and shoot (B) tissue of 3â€�weekâ€�old plants was harvested to quantify SA content on a ultra high performance liquid chromatography (Uâ€�HPLC) system. Plant material was homogenized by grinding in liquid nitrogen (three individual plants per line were used). Extraction of 100Â mg of plant material was performed using the modified Bieleski solvent after which it was filtrated and evaporated. Chromatographic separation was performed on a Uâ€�HPLC system equipped with a Nucleodur C18 column (50Â Ã—Â 2Â mm, 1.8Â Î¼mÂ dp), using a mobile phase gradient consisting of acidified methanol and water. Mass spectrometric analysis was carried out in selectedâ€�ion monitoring (SIM) mode with a Q Exactiveâ"¢ Orbitrap mass spectrometer (Thermo Scientific), operating in both positive and negative electrospray ionization mode at a resolution of 70,000 fullâ€�width at half maximum. Data were analysed with a nonparametric pairwise Kruskalâ€"Wallis test. No significant differences were detected among the different lines. Ctrl, empty vector line</p>  <div class="media" title="media"><a href="MPP-21-1634-s002.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>FIGURE S3</b> Representation of the fragmentation pathways of the compound eluting at 3.26Â min (id: 3.26_265.0921m/z)</p>  <div class="media" title="media"><a href="MPP-21-1634-s003.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>FIGURE S4</b> Representation of the fragmentation pathways of the compound eluting at 4.33Â min (id: 4.33_309.1185m/z)</p>  <div class="media" title="media"><a href="MPP-21-1634-s004.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>TABLE S1</b> Table showing the Ct values for two control genes (Exp Narcai and EIF5C) and the transgene (A, HoCM; B, HoICM) that is expressed in the different transgenic lines and controls (wildâ€�type Nipponbare and empty vector line). Values that are underlined indicate that there was no expression for that gene. NA, not available</p>  <div class="media" title="media"><a href="MPP-21-1634-s005.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>TABLE S2</b> Overview of the output of RNAâ€�Seq on the different transgenic lines</p>  <div class="media" title="media"><a href="MPP-21-1634-s006.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>TABLE S3</b> Differential compounds in the transgenic lines overexpressing HoICM. FC, fold change; FDR, false discovery rate</p>  <div class="media" title="media"><a href="MPP-21-1634-s007.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>TABLE S4</b> Differential compounds in the transgenic lines overexpressing HoCM. FC, fold change; FDR, false discovery rate</p>  <div class="media" title="media"><a href="MPP-21-1634-s008.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>TABLE S5</b> MS/MS spectral information for a selection of compounds which are differentially abundant in HoCM or HoICM expressing lines. ID, compound identifier; tR, retention time; <i>m</i>/<i>z</i>, mass to charge ratio; Î"ppm, deviation of measured mass; MS/MS, relative abundances to the base peak of the MS/MS product ions are indicated between parentheses</p>  <div class="media" title="media"><a href="MPP-21-1634-s009.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>TABLE S6</b> Table with primer sequences used in this research</p>  <div class="media" title="media"><a href="MPP-21-1634-s010.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>FILE S1</b> List of differentially expressed genes in transgenic lines compared to the control according to the RNAâ€�Seq data</p>  <div class="media" title="media"><a href="MPP-21-1634-s011.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p><b>FILE S2</b> Detailed protocol describing the metabolomics experiment</p>  <div class="media" title="media"><a href="MPP-21-1634-s012.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGEMENTS</div> <p>We want to acknowledge the Research Foundation Flanders FWO for financial support (FWO grant 3G009829W). Thanks to Professor Bryan Bellaire for providing us with the isochorismateâ€�deficient strain AN192. We want to acknowledge Professor Kristof Demeestere for his help with the analysis of salicylic acid levels. We thank the VIB Metabolomics Core (VIBâ€�UGent, Belgium) for performing the untargeted metabolic profiling, LCâ€�MS data processing and basic statistical analysis. The authors declare that there are no conflicts of interest.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
DATA AVAILABILITY STATEMENT</div> <p>Sequence data are available in GenBank at <a href="http://www.ncbi.nlm.nih.gov">www.ncbi.nlm.nih.gov</a> with accession numbers HoCM: KP297892, and HoICM: KP297893. The data that support the findings of this study are available from the corresponding author upon reasonable request.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="mpp13003-bib-0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Allen</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Greiner</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wishart</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Competitive fragmentation modeling of ESIâ€�MS/MS spectra for putative metabolite identification</span>. <span class="source" tagx="source" title="source">Metabolomics</span>, <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">98</span>â€"<span class="lpage" tagx="lpage" title="lpage">110</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Babatola</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">1981</span>) <span class="mixed-article-title" title="mixed-article-title">Effect of pH, oxygen and temperature on the activity and survival of Hirschmanniella spp</span>. <span class="source" tagx="source" title="source">Nematologica</span>, <span class="volume" tagx="volume" title="volume">27</span>, <span class="fpage" tagx="fpage" title="fpage">285</span>â€"<span class="lpage" tagx="lpage" title="lpage">291</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bauters</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Haegeman</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kyndt</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gheysen</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Analysis of the transcriptome of <i>Hirschmanniella oryzae</i> to explore potential survival strategies and hostâ€"nematode interactions</span>. <span class="source" tagx="source" title="source">Molecular Plant Pathology</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">352</span>â€"<span class="lpage" tagx="lpage" title="lpage">363</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24279397">24279397</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Benjamini</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hochberg</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span> (<span class="year" tagx="year" title="year">1995</span>) <span class="mixed-article-title" title="mixed-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</span>. <span class="source" tagx="source" title="source">Journal of the Royal Statistical Society. Series B (Methodological)</span>, <span class="volume" tagx="volume" title="volume">57</span>, <span class="fpage" tagx="fpage" title="fpage">289</span>â€"<span class="lpage" tagx="lpage" title="lpage">300</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">BÃ¶cker</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">DÃ¼hrkop</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Fragmentation trees reloaded</span>. <span class="source" tagx="source" title="source">Journal of Cheminformatics</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">5</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26807156">26807156</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bolla</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weaver</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Winter</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">1988</span>) <span class="mixed-article-title" title="mixed-article-title">Genomic differences among pathotypes of <i>Bursaphelenchus xylophilus</i></span>. <span class="source" tagx="source" title="source">Journal of Nematology</span>, <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">309</span>â€"<span class="lpage" tagx="lpage" title="lpage">316</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19290214">19290214</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chook</span>, <span class="given-names" tagx="given-names" title="given-names">Y.M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ke</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lipscomb</span>, <span class="given-names" tagx="given-names" title="given-names">W.N.</span></span> (<span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">Crystal structures of the monofunctional chorismate mutase from <i>Bacillus subtilis</i> and its complex with a transition state analog</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">90</span>, <span class="fpage" tagx="fpage" title="fpage">8600</span>â€"<span class="lpage" tagx="lpage" title="lpage">8603</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8378335">8378335</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Davis</span>, <span class="given-names" tagx="given-names" title="given-names">E.L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Haegeman</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kikuchi</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="chapter-title" tagx="chapter-title" title="chapter-title">Degradation of the plant cell wall by nematodes</span> In: <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">J.T.</span></span></span>,<span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gheysen</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> and <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fenoll</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (Eds.) <span class="source" tagx="source" title="source">Genomics and Molecular Genetics of Plantâ€�Nematode Interactions</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Dordrecht</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>, pp. <span class="fpage" tagx="fpage" title="fpage">255</span>â€"<span class="lpage" tagx="lpage" title="lpage">272</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0009" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De Vleesschauwer</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gheysen</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">HÃ¶fte</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Hormone defense networking in rice: tales from a different world</span>. <span class="source" tagx="source" title="source">Trends in Plant Science</span>, <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">555</span>â€"<span class="lpage" tagx="lpage" title="lpage">565</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23910453">23910453</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dempsey</span>, <span class="given-names" tagx="given-names" title="given-names">D.M.A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vlot</span>, <span class="given-names" tagx="given-names" title="given-names">A.C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wildermuth</span>, <span class="given-names" tagx="given-names" title="given-names">M.C.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Klessig</span>, <span class="given-names" tagx="given-names" title="given-names">D.F.</span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Salicylic acid biosynthesis and metabolism</span>. <span class="source" tagx="source" title="source">The Arabidopsis Book/American Society of Plant Biologists</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">24</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Djamei</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schipper</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rabe</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ghosh</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vincon</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kahnt</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> et al. (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Metabolic priming by a secreted fungal effector</span>. <span class="source" tagx="source" title="source">Nature</span>, <span class="volume" tagx="volume" title="volume">478</span>, <span class="fpage" tagx="fpage" title="fpage">395</span>â€"<span class="lpage" tagx="lpage" title="lpage">400</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21976020">21976020</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dobin</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Davis</span>, <span class="given-names" tagx="given-names" title="given-names">C.A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schlesinger</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Drenkow</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zaleski</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jha</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> et al. (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">STAR: ultrafast universal RNAâ€�seq aligner</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">15</span>â€"<span class="lpage" tagx="lpage" title="lpage">21</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886">23104886</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Doyle</span>, <span class="given-names" tagx="given-names" title="given-names">E.A.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lambert</span>, <span class="given-names" tagx="given-names" title="given-names">K.N.</span></span> (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title"><i>Meloidogyne javanica</i> chorismate mutase 1 alters plant cell development</span>. <span class="source" tagx="source" title="source">Molecular Plantâ€�Microbe Interactions</span>, <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">123</span>â€"<span class="lpage" tagx="lpage" title="lpage">131</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12575746">12575746</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Du</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ling</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Su</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">agriGO: a GO analysis toolkit for the agricultural community</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>, <span class="volume" tagx="volume" title="volume">38</span>(<span class="issue" tagx="issue" title="issue">suppl_2</span>), <span class="fpage" tagx="fpage" title="fpage">W64</span>â€"<span class="lpage" tagx="lpage" title="lpage">W70</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20435677">20435677</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">DÃ¼hrkop</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shen</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Meusel</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rousu</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">BÃ¶cker</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Searching molecular structure databases with tandem mass spectra using CSI: FingerID</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">112</span>, <span class="fpage" tagx="fpage" title="fpage">12580</span>â€"<span class="lpage" tagx="lpage" title="lpage">12585</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26392543">26392543</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Elâ€�Bebany</span>, <span class="given-names" tagx="given-names" title="given-names">A.F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rampitsch</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Daayf</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Proteomic analysis of the phytopathogenic soilborne fungus <i>Verticillium dahliae</i> reveals differential protein expression in isolates that differ in aggressiveness</span>. <span class="source" tagx="source" title="source">Proteomics</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">289</span>â€"<span class="lpage" tagx="lpage" title="lpage">303</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20017145">20017145</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Evesâ€�van den Akker</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Laetsch</span>, <span class="given-names" tagx="given-names" title="given-names">D.R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thorpe</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lilley</span>, <span class="given-names" tagx="given-names" title="given-names">C.J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Danchin</span>, <span class="given-names" tagx="given-names" title="given-names">E.G.J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Da Rocha</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, et al. (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">The genome of the yellow potato cyst nematode, Globodera rostochiensis, reveals insights into the basis of parasitism and virulence</span>. <span class="source" tagx="source" title="source">Genome Biology</span>, <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">124</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27286965">27286965</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Feng</span>, <span class="given-names" tagx="given-names" title="given-names">J.â€�X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cao</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Duan</span>, <span class="given-names" tagx="given-names" title="given-names">C.â€�J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Luo</span>, <span class="given-names" tagx="given-names" title="given-names">X.â€�M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Le</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span> et al. (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Involvement of OsNPR1/NH1 in rice basal resistance to blast fungus <i>Magnaporthe oryzae</i></span>. <span class="source" tagx="source" title="source">European Journal of Plant Pathology</span>, <span class="volume" tagx="volume" title="volume">131</span>, <span class="fpage" tagx="fpage" title="fpage">221</span>â€"<span class="lpage" tagx="lpage" title="lpage">235</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gao</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Starr</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">GÃ¶bel</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Engelberth</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Feussner</span>, <span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tumlinson</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> et al. (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Maize 9â€�lipoxygenase ZmLOX3 controls development, rootâ€�specific expression of defense genes, and resistance to rootâ€�knot nematodes</span>. <span class="source" tagx="source" title="source">Molecular Plantâ€�Microbe Interactions</span>, <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">98</span>â€"<span class="lpage" tagx="lpage" title="lpage">109</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18052887">18052887</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Glazebrook</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Contrasting mechanisms of defense against biotrophic and necrotrophic pathogens</span>. <span class="source" tagx="source" title="source">Annual Review of Phytopathology</span>, <span class="volume" tagx="volume" title="volume">43</span>, <span class="fpage" tagx="fpage" title="fpage">205</span>â€"<span class="lpage" tagx="lpage" title="lpage">227</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Goral</span>, <span class="given-names" tagx="given-names" title="given-names">A.M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tkaczuk</span>, <span class="given-names" tagx="given-names" title="given-names">K.L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chruszcz</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kagan</span>, <span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Savchenko</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Minor</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Crystal structure of a putative isochorismatase hydrolase from <i>Oleispira antarctica</i></span>. <span class="source" tagx="source" title="source">Journal of Structural and Functional Genomics</span>, <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">27</span>â€"<span class="lpage" tagx="lpage" title="lpage">36</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22350524">22350524</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Goverse</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Overmars</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Engelbertink</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schots</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bakker</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Helder</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Both induction and morphogenesis of cyst nematode feeding cells are mediated by auxin</span>. <span class="source" tagx="source" title="source">Molecular Plantâ€�Microbe Interactions</span>, <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">1121</span>â€"<span class="lpage" tagx="lpage" title="lpage">1129</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11043473">11043473</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Haeck</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Van Langenhove</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Harinck</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kyndt</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gheysen</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">HÃ¶fte</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> et al. (<span class="year" tagx="year" title="year">2018</span>) <span class="mixed-article-title" title="mixed-article-title">Trace analysis of multiâ€�class phytohormones in <i>Oryza sativa</i> using different scan modes in highâ€�resolution Orbitrap mass spectrometry: method validation, concentration levels, and screening in multiple accessions</span>. <span class="source" tagx="source" title="source">Analytical and Bioanalytical Chemistry</span>, <span class="volume" tagx="volume" title="volume">410</span>, <span class="fpage" tagx="fpage" title="fpage">4527</span>â€"<span class="lpage" tagx="lpage" title="lpage">4539</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29796899">29796899</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Haegeman</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Joseph</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gheysen</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Analysis of the transcriptome of the root lesion nematode <i>Pratylenchus coffeae</i> generated by 454 sequencing technology</span>. <span class="source" tagx="source" title="source">Molecular and Biochemical Parasitology</span>, <span class="volume" tagx="volume" title="volume">178</span>, <span class="fpage" tagx="fpage" title="fpage">7</span>â€"<span class="lpage" tagx="lpage" title="lpage">14</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21513748">21513748</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hall</span>, <span class="given-names" tagx="given-names" title="given-names">T.A.</span></span> (<span class="year" tagx="year" title="year">1999</span>) <span class="source" tagx="source" title="source">BioEdit: A Userâ€�Friendly Biological Sequence Alignment Editor and Analysis Program for Windows 95/98/NT. Nucleic Acids Symposium Series</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Information Retrieval Ltd.</span>, c1979â€"c2000, <span class="fpage" tagx="fpage" title="fpage">95</span>â€"<span class="lpage" tagx="lpage" title="lpage">98</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Herrmann</span>, <span class="given-names" tagx="given-names" title="given-names">K.M.</span></span> (<span class="year" tagx="year" title="year">1995</span>) <span class="mixed-article-title" title="mixed-article-title">The shikimate pathway: early steps in the biosynthesis of aromatic compounds</span>. <span class="source" tagx="source" title="source">The Plant Cell</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">907</span>â€"<span class="lpage" tagx="lpage" title="lpage">919</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12242393">12242393</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hewezi</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Baum</span>, <span class="given-names" tagx="given-names" title="given-names">T.J.</span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Manipulation of plant cells by cyst and rootâ€�knot nematode effectors</span>. <span class="source" tagx="source" title="source">Molecular Plantâ€�Microbe Interactions</span>, <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">9</span>â€"<span class="lpage" tagx="lpage" title="lpage">16</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22809272">22809272</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huang</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dong</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Allen</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Davis</span>, <span class="given-names" tagx="given-names" title="given-names">E.L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Baum</span>, <span class="given-names" tagx="given-names" title="given-names">T.J.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hussey</span>, <span class="given-names" tagx="given-names" title="given-names">R.S.</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Two chorismate mutase genes from the rootâ€�knot nematode <i>Meloidogyne incognita</i></span>. <span class="source" tagx="source" title="source">Molecular Plant Pathology</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">23</span>â€"<span class="lpage" tagx="lpage" title="lpage">30</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20565635">20565635</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huffaker</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kaplan</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vaughan</span>, <span class="given-names" tagx="given-names" title="given-names">M.M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dafoe</span>, <span class="given-names" tagx="given-names" title="given-names">N.J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ni</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rocca</span>, <span class="given-names" tagx="given-names" title="given-names">J.R.</span></span> et al. (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Novel acidic sesquiterpenoids constitute a dominant class of pathogenâ€�induced phytoalexins in maize</span>. <span class="source" tagx="source" title="source">Plant Physiology</span>, <span class="volume" tagx="volume" title="volume">156</span>, <span class="fpage" tagx="fpage" title="fpage">2082</span>â€"<span class="lpage" tagx="lpage" title="lpage">2097</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21690302">21690302</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ismail</span>, <span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Godfrey</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Able</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Fungal growth, proteinaceous toxins and virulence of <i>Pyrenophora teres</i> f. <i>teres</i> on barley</span>. <span class="source" tagx="source" title="source">Australasian Plant Pathology</span>, <span class="volume" tagx="volume" title="volume">43</span>, <span class="fpage" tagx="fpage" title="fpage">535</span>â€"<span class="lpage" tagx="lpage" title="lpage">546</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jaouannet</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Magliano</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Arguel</span>, <span class="given-names" tagx="given-names" title="given-names">M.J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gourgues</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Evangelisti</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Abad</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span> et al. (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">The rootâ€�knot nematode calreticulin Miâ€�CRT is a key effector in plant defense suppression</span>. <span class="source" tagx="source" title="source">Molecular Plantâ€�Microbe Interactions</span>, <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">97</span>â€"<span class="lpage" tagx="lpage" title="lpage">105</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22857385">22857385</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ji</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gheysen</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ullah</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Verbeek</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shang</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De Vleesschauwer</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span> et al. (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">The role of thionins in rice defence against root pathogens</span>. <span class="source" tagx="source" title="source">Molecular Plant Pathology</span>, <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">870</span>â€"<span class="lpage" tagx="lpage" title="lpage">881</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25676661">25676661</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jones</span>, <span class="given-names" tagx="given-names" title="given-names">J.T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Furlanetto</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bakker</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Banks</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Blok</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chen</span>, <span class="given-names" tagx="given-names" title="given-names">Q.</span></span> et al. (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Characterization of a chorismate mutase from the potato cyst nematode <i>Globodera pallida</i></span>. <span class="source" tagx="source" title="source">Molecular Plant Pathology</span>, <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">43</span>â€"<span class="lpage" tagx="lpage" title="lpage">50</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20569361">20569361</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0034" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jones</span>, <span class="given-names" tagx="given-names" title="given-names">J.T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Furlanetto</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Phillips</span>, <span class="given-names" tagx="given-names" title="given-names">M.S.</span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">The role of flavonoids produced in response to cyst nematode infection of <i>Arabidopsis thaliana</i></span>. <span class="source" tagx="source" title="source">Nematology</span>, <span class="volume" tagx="volume" title="volume">9</span>(), <span class="fpage" tagx="fpage" title="fpage">671</span>â€"<span class="lpage" tagx="lpage" title="lpage">677</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0035" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Karakas</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Life cycle and mating behavior of <i>Hirschmanniella oryzae</i> (nematoda: Pratylenchidae) on excised <i>Oryzae sativa</i> roots</span>. <span class="source" tagx="source" title="source">Fen Bilimleri Dergisi</span>, <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Khanam</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bauters</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Singh</span>, <span class="given-names" tagx="given-names" title="given-names">R.R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Verbeek</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Haeck</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sultan</span>, <span class="given-names" tagx="given-names" title="given-names">S.M.</span></span> et al. (<span class="year" tagx="year" title="year">2018</span>) <span class="mixed-article-title" title="mixed-article-title">Mechanisms of resistance in the rice cultivar Manikpukha to the rice stem nematode <i>Ditylenchus angustus</i></span>. <span class="source" tagx="source" title="source">Molecular Plant Pathology</span>, <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">1391</span>â€"<span class="lpage" tagx="lpage" title="lpage">1402</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28990717">28990717</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kim</span>, <span class="given-names" tagx="given-names" title="given-names">S.â€�Y.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Volsky</span>, <span class="given-names" tagx="given-names" title="given-names">D.J.</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">PAGE: parametric analysis of gene set enrichment</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">144</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15941488">15941488</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kyndt</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nahar</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Haegeman</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De Vleesschauwer</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">HÃ¶fte</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gheysen</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Comparing systemic defenceâ€�related gene expression changes upon migratory and sedentary nematode attack in rice</span>. <span class="source" tagx="source" title="source">Plant Biology</span>, <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">73</span>â€"<span class="lpage" tagx="lpage" title="lpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22188265">22188265</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lambert</span>, <span class="given-names" tagx="given-names" title="given-names">K.N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Allen</span>, <span class="given-names" tagx="given-names" title="given-names">K.D.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sussex</span>, <span class="given-names" tagx="given-names" title="given-names">I.M.</span></span> (<span class="year" tagx="year" title="year">1999</span>) <span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of an esophagealâ€�glandâ€�specific chorismate mutase from the phytoparasitic nematode <i>Meloidogyne javanica</i></span>. <span class="source" tagx="source" title="source">Molecular Plantâ€�Microbe Interactions</span>, <span class="volume" tagx="volume" title="volume">12</span>, <span class="fpage" tagx="fpage" title="fpage">328</span>â€"<span class="lpage" tagx="lpage" title="lpage">336</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10188271">10188271</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lee</span>, <span class="given-names" tagx="given-names" title="given-names">A.Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Karplus</span>, <span class="given-names" tagx="given-names" title="given-names">P.A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ganem</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Clardy</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">1995</span>) <span class="mixed-article-title" title="mixed-article-title">Atomic structure of the buried catalytic pocket of <i>Escherichia coli</i> chorismate mutase</span>. <span class="source" tagx="source" title="source">Journal of the American Chemical Society</span>, <span class="volume" tagx="volume" title="volume">117</span>, <span class="fpage" tagx="fpage" title="fpage">3627</span>â€"<span class="lpage" tagx="lpage" title="lpage">3628</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lefevere</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bauters</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gheysen</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">2020</span>) <span class="mixed-article-title" title="mixed-article-title">Salicylic acid biosynthesis in plants</span>. <span class="source" tagx="source" title="source">Frontiers in Plant Science</span>, <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">338</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32362901">32362901</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0042" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liu</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Song</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span>, <span class="given-names" tagx="given-names" title="given-names">X,</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yuan</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Su</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span>, et al. (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Unconventionally secreted effectors of two filamentous pathogens target plant salicylate biosynthesis</span>. <span class="source" tagx="source" title="source">Nature Communications</span>, <span class="volume" tagx="volume" title="volume">5</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">4686</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0043" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McCarthy</span>, <span class="given-names" tagx="given-names" title="given-names">D.J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chen</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Smyth</span>, <span class="given-names" tagx="given-names" title="given-names">G.K.</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Differential expression analysis of multifactor RNAâ€�Seq experiments with respect to biological variation</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>, <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">4288</span>â€"<span class="lpage" tagx="lpage" title="lpage">4297</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22287627">22287627</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0044" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nahar</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kyndt</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nzogela</span>, <span class="given-names" tagx="given-names" title="given-names">Y.B.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gheysen</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Abscisic acid interacts antagonistically with classical defense pathways in riceâ€"migratory nematode interaction</span>. <span class="source" tagx="source" title="source">New Phytologist</span>, <span class="volume" tagx="volume" title="volume">196</span>, <span class="fpage" tagx="fpage" title="fpage">901</span>â€"<span class="lpage" tagx="lpage" title="lpage">913</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22985247">22985247</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0045" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nicol</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gill</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fosuâ€�Nyarko</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jones</span>, <span class="given-names" tagx="given-names" title="given-names">M.G.</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">de novo analysis and functional classification of the transcriptome of the root lesion nematode, <i>Pratylenchus thornei</i>, after 454 GS FLX sequencing</span>. <span class="source" tagx="source" title="source">International Journal for Parasitology</span>, <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">225</span>â€"<span class="lpage" tagx="lpage" title="lpage">237</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22309969">22309969</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0046" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ã–kvist</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dey</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sasso</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Grahn</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kast</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Krengel</span>, <span class="given-names" tagx="given-names" title="given-names">U.</span></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">1.6 Ã… crystal structure of the secreted chorismate mutase from <i>Mycobacterium tuberculosis</i>: novel fold topology revealed</span>. <span class="source" tagx="source" title="source">Journal of Molecular Biology</span>, <span class="volume" tagx="volume" title="volume">357</span>, <span class="fpage" tagx="fpage" title="fpage">1483</span>â€"<span class="lpage" tagx="lpage" title="lpage">1499</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16499927">16499927</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0047" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Opperman</span>, <span class="given-names" tagx="given-names" title="given-names">C.H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bird</span>, <span class="given-names" tagx="given-names" title="given-names">D.M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Williamson</span>, <span class="given-names" tagx="given-names" title="given-names">V.M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rokhsar</span>, <span class="given-names" tagx="given-names" title="given-names">D.S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Burke</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cohn</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> et al. (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Sequence and genetic map of <i>Meloidogyne hapla</i>: a compact nematode genome for plant parasitism</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">105</span>, <span class="fpage" tagx="fpage" title="fpage">14802</span>â€"<span class="lpage" tagx="lpage" title="lpage">14807</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18809916">18809916</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0048" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Parsons</span>, <span class="given-names" tagx="given-names" title="given-names">J.F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Calabrese</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Eisenstein</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ladner</span>, <span class="given-names" tagx="given-names" title="given-names">J.E.</span></span> (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Structure and mechanism of <i>Pseudomonas aeruginosa</i> PhzD, an isochorismatase from the phenazine biosynthetic pathway</span>. <span class="source" tagx="source" title="source">Biochemistry</span>, <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">5684</span>â€"<span class="lpage" tagx="lpage" title="lpage">5693</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12741825">12741825</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0049" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Petersen</span>, <span class="given-names" tagx="given-names" title="given-names">T.N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brunak</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">von Heijne</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nielsen</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions</span>. <span class="source" tagx="source" title="source">Nature Methods</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">785</span>â€"<span class="lpage" tagx="lpage" title="lpage">786</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21959131">21959131</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0050" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Postma</span>, <span class="given-names" tagx="given-names" title="given-names">W.J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Slootweg</span>, <span class="given-names" tagx="given-names" title="given-names">E.J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rehman</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Finkersâ€�Tomczak</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tytgat</span>, <span class="given-names" tagx="given-names" title="given-names">T.O.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">van Gelderen</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span> et al. (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">The effector SPRYSECâ€�19 of <i>Globodera rostochiensis</i> suppresses CCâ€�NBâ€�LRRâ€�mediated disease resistance in plants</span>. <span class="source" tagx="source" title="source">Plant Physiology</span>, <span class="volume" tagx="volume" title="volume">160</span>, <span class="fpage" tagx="fpage" title="fpage">944</span>â€"<span class="lpage" tagx="lpage" title="lpage">954</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22904163">22904163</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0051" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Punta</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Coggill</span>, <span class="given-names" tagx="given-names" title="given-names">P.C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Eberhardt</span>, <span class="given-names" tagx="given-names" title="given-names">R.Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mistry</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tate</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Boursnell</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span> et al. (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">The Pfam protein families database</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>, <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">D290</span>â€"<span class="lpage" tagx="lpage" title="lpage">D301</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127870">22127870</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0052" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: R_Core_Team</span> (<span class="year" tagx="year" title="year">2017</span>) <span class="source" tagx="source" title="source">R: A Language and Environment for Statistical Computing</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Vienna, Austria</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">R Foundation for Statistical Computing</span> [Online]. Available at: <a href="https://www.R-project.org/">https://www.Râ€�project.org/</a>. [Accessed 9 October 2020].</span></li> <li tag="ref"><a name="mpp13003-bib-0054" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Reversat</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Boyer</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sannier</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pandoâ€�Bahuon</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span> (<span class="year" tagx="year" title="year">1999</span>) <span class="mixed-article-title" title="mixed-article-title">Use of a mixture of sand and waterâ€�absorbent synthetic polymer as substrate for the xenic culturing of plantâ€�parasitic nematodes in the laboratory</span>. <span class="source" tagx="source" title="source">Nematology</span>, <span class="volume" tagx="volume" title="volume">1</span>, <span class="fpage" tagx="fpage" title="fpage">209</span>â€"<span class="lpage" tagx="lpage" title="lpage">212</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0055" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Robinson</span>, <span class="given-names" tagx="given-names" title="given-names">M.D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McCarthy</span>, <span class="given-names" tagx="given-names" title="given-names">D.J.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Smyth</span>, <span class="given-names" tagx="given-names" title="given-names">G.K.</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">edgeR: a bioconductor package for differential expression analysis of digital gene expression data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">139</span>â€"<span class="lpage" tagx="lpage" title="lpage">140</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308">19910308</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0056" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Robinson</span>, <span class="given-names" tagx="given-names" title="given-names">M.D.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Oshlack</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">A scaling normalization method for differential expression analysis of RNAâ€�seq data</span>. <span class="source" tagx="source" title="source">Genome Biology</span>, <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">R25</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20196867">20196867</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0057" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Soanes</span>, <span class="given-names" tagx="given-names" title="given-names">D.M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alam</span>, <span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cornell</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wong</span>, <span class="given-names" tagx="given-names" title="given-names">H.M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hedeler</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Paton</span>, <span class="given-names" tagx="given-names" title="given-names">N.W.</span></span> et al. (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Comparative genome analysis of filamentous fungi reveals gene family expansions associated with fungal pathogenesis</span>. <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">3</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e2300</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18523684">18523684</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0058" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Toyomasu</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Usui</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sugawara</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Otomo</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hirose</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Miyao</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span> et al. (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Reverseâ€�genetic approach to verify physiological roles of rice phytoalexins: characterization of a knockdown mutant of <i>OsCPS4</i> phytoalexin biosynthetic gene in rice</span>. <span class="source" tagx="source" title="source">Physiologia Plantarum</span>, <span class="volume" tagx="volume" title="volume">150</span>, <span class="fpage" tagx="fpage" title="fpage">55</span>â€"<span class="lpage" tagx="lpage" title="lpage">62</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23621683">23621683</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0059" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Uehara</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sugiyama</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Matsuura</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Arie</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Masuta</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Resistant and susceptible responses in tomato to cyst nematode are differentially regulated by salicylic acid</span>. <span class="source" tagx="source" title="source">Plant and Cell Physiology</span>, <span class="volume" tagx="volume" title="volume">51</span>, <span class="fpage" tagx="fpage" title="fpage">1524</span>â€"<span class="lpage" tagx="lpage" title="lpage">1536</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20660227">20660227</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0060" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vanholme</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kast</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Haegeman</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jacob</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Grunewald</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gheysen</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Structural and functional investigation of a secreted chorismate mutase from the plantâ€�parasitic nematode <i>Heterodera schachtii</i> in the context of related enzymes from diverse origins</span>. <span class="source" tagx="source" title="source">Molecular Plant Pathology</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">189</span>â€"<span class="lpage" tagx="lpage" title="lpage">200</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19236568">19236568</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0061" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Veech</span>, <span class="given-names" tagx="given-names" title="given-names">J.A.</span></span> (<span class="year" tagx="year" title="year">1982</span>) <span class="mixed-article-title" title="mixed-article-title">Phytoalexins and their role in the resistance of plants to nematodes</span>. <span class="source" tagx="source" title="source">Journal of Nematology</span>, <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">2</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19295667">19295667</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0062" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Verbeek</span>, <span class="given-names" tagx="given-names" title="given-names">R.E.M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Van Buyten</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alam</span>, <span class="given-names" tagx="given-names" title="given-names">M.Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De Vleesschauwer</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Van Bockhaven</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Asano</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, et al. (<span class="year" tagx="year" title="year">2019</span>) <span class="mixed-article-title" title="mixed-article-title">Jasmonateâ€�induced defense mechanisms in the belowground antagonistic interaction between <i>Pythium arrhenomanes</i> and <i>Meloidogyne graminicola</i> in Rice</span>. <span class="source" tagx="source" title="source">Frontiers in Plant Science</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">1515</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31824540">31824540</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0063" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vlot</span>, <span class="given-names" tagx="given-names" title="given-names">A.C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dempsey</span>, <span class="given-names" tagx="given-names" title="given-names">D.M.A.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Klessig</span>, <span class="given-names" tagx="given-names" title="given-names">D.F.</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Salicylic acid, a multifaceted hormone to combat disease</span>. <span class="source" tagx="source" title="source">Annual Review of Phytopathology</span>, <span class="volume" tagx="volume" title="volume">47</span>, <span class="fpage" tagx="fpage" title="fpage">177</span>â€"<span class="lpage" tagx="lpage" title="lpage">206</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0064" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xue</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dai</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Qin</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liu</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chi</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span> et al. (<span class="year" tagx="year" title="year">2018</span>) <span class="mixed-article-title" title="mixed-article-title">A novel <i>Meloidogyne incognita</i> chorismate mutase effector suppresses plant immunity by manipulating the salicylic acid pathway and functions mainly during the early stages of nematode parasitism</span>. <span class="source" tagx="source" title="source">Plant Pathology</span>, <span class="volume" tagx="volume" title="volume">67</span>, <span class="fpage" tagx="fpage" title="fpage">1436</span>â€"<span class="lpage" tagx="lpage" title="lpage">1448</span>.</span></li> <li tag="ref"><a name="mpp13003-bib-0065" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wubben</span>, <span class="given-names" tagx="given-names" title="given-names">M.J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Callahan</span>, <span class="given-names" tagx="given-names" title="given-names">F.E.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Scheffler</span>, <span class="given-names" tagx="given-names" title="given-names">B.S.</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Transcript analysis of parasitic females of the sedentary semiâ€�endoparasitic nematode <i>Rotylenchulus reniformis</i></span>. <span class="source" tagx="source" title="source">Molecular and Biochemical Parasitology</span>, <span class="volume" tagx="volume" title="volume">172</span>, <span class="fpage" tagx="fpage" title="fpage">31</span>â€"<span class="lpage" tagx="lpage" title="lpage">40</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20346373">20346373</a></span></span></li> <li tag="ref"><a name="mpp13003-bib-0066" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xie</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kapteyn</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> and <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gang</span>, <span class="given-names" tagx="given-names" title="given-names">D.R.</span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">A systems biology investigation of the MEP/terpenoid and shikimate/phenylpropanoid pathways points to multiple levels of metabolic control in sweet basil glandular trichomes</span>. <span class="source" tagx="source" title="source">The Plant Journal</span>, <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">349</span>â€"<span class="lpage" tagx="lpage" title="lpage">361</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18248593">18248593</a></span></span></li> </ul> </div> </div>  </body></html>