<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Commun Biol</span><span class="iso-abbrev" title="iso-abbrev">Commun Biol</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Communications Biology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2399-3642</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7809426">7809426</a></span><span class="publisher-id" title="publisher-id">1562</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s42003-020-01562-4">10.1038/s42003-020-01562-4</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Shifts in honeybee foraging reveal historical changes in floral resources</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-8197-1970</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">Laura</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brennan</span><span class="given-names" tagx="given-names" title="given-names">Georgina L.</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowe</span><span class="given-names" tagx="given-names" title="given-names">Abigail</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3124-3550</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Creer</span><span class="given-names" tagx="given-names" title="given-names">Simon</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-5881-4707</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ford</span><span class="given-names" tagx="given-names" title="given-names">Col R.</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-9593-6925</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vere</span><span class="given-names" tagx="given-names" title="given-names">Natasha</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">natasha.devere@gardenofwales.org.uk</span></span><a href="#Aff1">1</a><a href="#Aff4">4</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.499399.7</span><span class="institution-id" title="institution-id">0000 0000 9539 5260</span><span class="institution" title="institution">National Botanic Garden of Wales, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.7362.0</span><span class="institution-id" title="institution-id">0000000118820937</span><span class="institution" title="institution">MEFGL, School of Natural Sciences, Bangor University, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.4514.4</span><span class="institution-id" title="institution-id">0000 0001 0930 2361</span><span class="institution" title="institution">Centre for Environmental and Climate Research / Aquatic Ecology, Lund University, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.493538.0</span><span class="institution-id" title="institution-id">0000 0001 2222 015X</span><span class="institution" title="institution">IBERS, Aberystwyth University, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-1-1</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">37</span><span class="history" title="history"><span class="received" title="received">received: 2019-11-22</span><span class="accepted" title="accepted">accepted: 2020-12-3</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articleâ€™s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Decreasing floral resources as a result of habitat loss is one of the key factors in the decline of pollinating insects worldwide. Understanding which plants pollinators use is vital to inform the provision of appropriate floral resources to help prevent pollinator loss. Using a globally important pollinator, the honeybee, we show how changes in agricultural intensification, crop use and the spread of invasive species, have altered the nectar and pollen sources available in the UK. Using DNA metabarcoding, we analysed 441 honey samples from 2017 and compared these to a nationwide survey of honey samples from 1952. We reveal that shifts in major plants foraged by honeybees are driven by changes in the availability of these plants within the landscape. Improved grasslands are the most widespread habitat type in the UK, and management changes within this habitat have the greatest potential to increase floral resource availability.</p> </div> <div class="abstract" title="abstract"> <p id="Par2">Laura Jones et al. compare honey samples across the UK from 2017 to a nationwide survey of honey samples from 1952 in order to determine how nectar and pollen sources have changed over time. They find that shifts in major plants foraged by honeybees are driven by changes in the availability of these plants within the landscape, and focus on improved grasslands as the most widespread habitat type and a potential target for management efforts to increase floral resource availability.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Biodiversity</div> <div class="kwd" title="kwd">
Molecular biology</div> <div class="kwd" title="kwd">
Animal behaviour</div> <div class="kwd" title="kwd">
Plant ecology</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Funding to support this research was provided by the National Botanic Garden of Wales and a bequest from Dr Quentin Kay. LJ and AL were supported by Knowledge Economy Skills Scholarships (KESS2), part funded by the Welsh Governmentâ€™s European Social Fund (ESF).</span></div> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par3">Widespread declines in insect pollinators and the associated impacts on crops and biodiversity is a global problem<sup><a href="#CR1">1</a>â€"<a href="#CR3">3</a></sup>. One of the major factors implicated in pollinator declines is the reduction of floral resources due to agricultural intensification and habitat loss<sup><a href="#CR4">4</a>â€"<a href="#CR7">7</a></sup>. Across Europe, the dominant land use is agriculture, with over half of the European landscape being managed agriculturally<sup><a href="#CR8">8</a></sup>. Agricultural land is therefore a focus of conservation efforts to prevent the loss of associated biodiversity<sup><a href="#CR8">8</a>,<a href="#CR9">9</a></sup>. The input of inorganic nitrogen, re-seeding of grassland leys, high levels of grazing, and herbicide application can all cause species-rich, semi-natural grassland to become improved grassland, with a corresponding reduction in the diversity and availability of flowers used by pollinating insects for nectar and pollen<sup><a href="#CR10">10</a></sup>. In England and Wales, the proportion of lowland semi-natural grassland has been estimated to be 3% of what was present prior to 1939<sup><a href="#CR11">11</a></sup>.</p> <p id="Par4">It is possible to track changes in the availability of nectar resources over time by combining vegetation surveys and direct nectar measurements<sup><a href="#CR12">12</a></sup>, but it is difficult to relate this to changes in pollinator foraging. Honeybees are an ideal model to assess landscape changes in forage availability and usage as they have a widespread distribution and long foraging range<sup><a href="#CR13">13</a></sup>. Managed honeybees can be geolocated exactly and so by characterising the pollen found within honey, we can determine the floral resources used for nectar and pollen in the area surrounding the hive<sup><a href="#CR14">14</a></sup>.</p> <p id="Par5">Here, honey provided by beekeepers as part of a nationwide UK campaign in 2017 has been characterised and compared to honey sampled in 1952, enabling us to investigate whether landscape-scale changes in the floral resource are leading to changes in honeybee foraging. DNA metabarcoding, using two complementary DNA barcode markers (<i>rbcL</i> and ITS2), was used to identify the plant taxa within contemporary honey samples, extracted from hives between April and October, across the latitudinal and longitudinal range of the UK in 2017. We compared the plant composition of the 2017 honey with the last UK wide survey of honey samples from 1952, characterised using melissopalynology<sup><a href="#CR15">15</a>,<a href="#CR16">16</a></sup>. DNA metabarcoding leverages a higher taxonomic resolution of the plant taxa present in the honey when compared to microscopic identification and so a conservative approach was taken to compare the data.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par6">In 2017, we analysed 441 honey samples, with most samples provided from England and Wales in July (147 samples) and August (155 samples) (Fig.Â <a href="#Fig1">1</a>). The habitat type surrounding the hives reflected the composition of habitats of the UK with a positive correlation between the proportion of habitats in the UK and the proportion in a 2-km radius around the hives (Fig.Â <a href="#Fig1">1</a>, <i>r</i><sup><i>2</i></sup>â€‰=â€‰0.8, <i>P</i>â€‰=â€‰0.0002). Improved grassland, arable and horticulture, broadleaved woodland and suburban were the top habitats within the locality of the hives (Fig.Â <a href="#Fig1">1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <div class="title" tagx="title" title="title">
Distribution of honey samples (<i>n</i>â€‰=â€‰441) collected in 2017 and analysed with DNA metabarcoding, along with the habitat types hives were found within.</div> <p><b>a</b> The month when honey was extracted from the hive, April (4) to October (10), is indicated by colours. <b>b</b> The percentage area of different habitats is presented for the UK as a whole and for within a 2-km radius of honey samples, characterised using the 2015 CEH Land Cover map (NERC CEH).</p>   </div> <p /> <div class="ukhoneybeeforaging" title="sec"> <div class="title" tagx="title" title="title">
UK honeybee foraging</div> <p id="Par7">A total of 157 plant taxa were identified from the 441 honey samples, using the <i>rbcL</i> and ITS2 barcode regions combined (<a href="#MOESM3">Supplementary Data</a>). The total frequency of occurrence for each plant taxon was calculated as the presence of the taxon across all 2017 honey samples (Fig.Â <a href="#Fig2">2</a>). Of the 157 identified taxa, only 44 occurred in over 5% of the honey samples and only four taxa were identified in over 50% of samples (Fig.Â <a href="#Fig2">2</a>). For the abundance of each plant taxon within a sample, the proportion of sequences returned was placed into classes. Plant taxa represented by over 45% of sequences were designated predominant for that sample; between 15 and 45% were secondary; between 1 and 15% were important minor taxa and &amp;lt;1% of reads were classed as minor taxa (Fig.Â <a href="#Fig2">2</a>). Major taxa were defined as taxa returned at a predominant or secondary level. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <div class="title" tagx="title" title="title">
Plant taxa found in over 5% of honey samples analysed using DNA metabarcoding (<i>n</i>â€‰=â€‰441).</div> <p>The abundance of a plant taxon within a honey sample is indicated by the colour, with four abundance classes: Predominant: &amp;gt;45% of sequences returned in a sample, secondary: 15â€"45%, important minor: 1â€"15% and minor &amp;lt;1%.</p>   </div> <p /> <p id="Par8">Brambles (<i>Rubus</i> spp.) were both the most frequently found and abundant species within the honey samples, followed by white clover (<i>Trifolium repens</i>) and <i>Brassica</i> species. The <i>Brassica</i> species include the crop, oilseed rape (<i>Brassica napus</i>), along with other wild and cultivated <i>Brassica</i> species. The high frequency of these taxa across the honey reflects their long flowering period, with these plant groups appearing at high levels from May to September (Fig.Â <a href="#Fig3">3</a>). The next most frequently found and abundant plants were spring-flowering shrubs and trees, including hawthorn (<i>Cratageus monogyna</i>), apple (<i>Malus</i> spp.), <i>Cotoneaster</i> spp., sycamore and maples (<i>Acer</i> spp.), and cherries and plums (<i>Prunus</i> spp.). Towards the end of the season (peaking in September), heather <i>(Calluna vulgaris</i>) and the non-native invasive species, Himalayan balsam (<i>Impatiens glandulifera</i>) were found abundantly within honey samples (Fig.Â <a href="#Fig3">3</a>). Pollen identified in honey collected in 2017 reflects the seasonal changes in the plants available to the honeybees, with calendar month (Aprilâ€"October), being a good predictor of plant taxa composition (Fig.Â <a href="#Fig3">3</a>; LR<sub>428, 1</sub>â€‰=â€‰454.8, <i>P</i>â€‰=â€‰0.001). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <div class="title" tagx="title" title="title">
The most abundantly found plant taxa in the honey samples analysed using DNA metabarcoding are summarised as a proportion of samples through the season.</div> <p>Plant taxa illustrated are the most frequently found plant taxa in 2017 at a predominant and secondary level. Samples collected in April (<i>n</i>â€‰=â€‰3) and October (<i>n</i>â€‰=â€‰7) were excluded. Sample sizes: May (<i>n</i>â€‰=â€‰39), June (<i>n</i>â€‰=â€‰71), July (<i>n</i>â€‰=â€‰147), August (<i>n</i>â€‰=â€‰155), September (<i>n</i>â€‰=â€‰43). Predominant: &amp;gt;45% of sequences returned in a sample, secondary: 15â€"45%, important minor: 1â€"15% and minor &amp;lt;1%. The overall height of the bar indicates the total proportion of samples with that taxon.</p>   </div> <p /> <p id="Par9">There were no overall regional differences between England, Scotland and Wales, in the most frequently found taxa in 2017 (Supplementary Fig.Â <a href="#MOESM2">1</a>; Latitude LR<sub>427, 1</sub>â€‰=â€‰272.2, <i>P</i>â€‰=â€‰0.086; Longitude LR<sub>426, 1</sub>â€‰=â€‰352.3, <i>P</i>â€‰=â€‰0.092). While latitude and longitude were not significant predictors when assessing the overall honey composition, at the individual taxa level there was some evidence of spatial autocorrelation in 22 of the 157 taxa identified (using Moranâ€™s I; <a href="#MOESM3">Supplementary Data</a>). However, after Bonferroniâ€™s correction for multiple testing none of the 22 taxa remained significant. The relationship between the plant composition of the honey and the dominant surrounding habitat class was significant however habitat class explained only 3% of the total variation (Supplementary Fig.Â <a href="#MOESM2">2</a>; <i>r</i><sup>2</sup>â€‰=â€‰0.037, <i>P</i>â€‰=â€‰0.001).</p> <p id="Par10">There were significant relationships between the distribution of insect attractive crops, field beans (<i>Vicia faba</i>) and oilseed rape (<i>Brassica napus</i>), and their presence within honey samples (Fig.Â <a href="#Fig4">4</a>). <i>Vicia</i> species were more likely to be detected in honey within a 2â€‰km radius of field beans (<i>x</i><sup>2</sup>â€‰=â€‰52.83, d.f.â€‰=â€‰4, <i>P</i>â€‰<div class="â€‰0.001)_UNKNOWN">
Brassica species within 2â€‰km of oilseed rape (<i>x</i><sup>2</sup>â€‰=â€‰50.71, d.f.â€‰=â€‰4, <i>P</i>â€‰<div class="â€‰0.001)._UNKNOWN" /></div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <div class="title" tagx="title" title="title">
Distribution of field beans (<i>Vicia faba</i>) and oilseed rape (<i>Brassica napus</i>) in the UK and their presence within honey samples (<i>n</i>â€‰=â€‰424) detected using DNA metabarcoding.</div> <p>The distribution of field beans (<b>a</b>) and oil seed rape (<b>b</b>) in 2017 from CEH Land Cover plus: Crops map is shown under honey samples with <i>Vicia</i> spp. (<b>a</b>) and <i>Brassica</i> spp. (<b>b</b>) present, as indicated by coloured squares. Black circles: not detected. <b>c</b><i>Vicia</i> spp. were more likely to be detected in honey within a 2â€‰km radius of field beans (<i>x</i><sup>2</sup>â€‰=â€‰52.83, d.f.â€‰=â€‰4, <i>p</i>â€‰<div class="â€‰0.0001)_UNKNOWN">
Brassica spp. within 2â€‰km of oil seed rape (<i>x</i><sup>2</sup>â€‰=â€‰50.71, d.f.â€‰=â€‰4, <i>p</i>â€‰<div class="â€‰0.0001)_UNKNOWN" href="#MOESM2">
1).</div></div></p>   </div> <p /> </div> <div class="comparisionwith1952" title="sec"> <div class="title" tagx="title" title="title">
Comparision with 1952</div> <p id="Par11">In 1952, 855 honey samples, from throughout the UK, were analysed using melissopalynology<sup><a href="#CR15">15</a>,<a href="#CR16">16</a></sup>. A total of 66 plant taxa were identified, 47 of which matched with the plants found in 2017 (Supplementary Fig.Â <a href="#MOESM2">3</a>, <a href="#MOESM2">Supplementary Data and Discussion</a>).</p> <p id="Par12">Overall, there was a positive correlation between the frequency of occurrence for the 47 plant taxa between the two collection dates of 1952 and 2017 (Supplementary Fig.Â <a href="#MOESM2">4</a>; Kendallâ€™s Ï" correlation coefficient Ï"â€‰=â€‰0.389, <i>P</i>â€‰<div class="â€‰0.001)._UNKNOWN" href="#Fig5">
5; LR<sub>125, 1</sub>â€‰=â€‰93.16, <i>P</i>â€‰=â€‰0.001), while no significant difference was found between sampling locations (Supplementary Fig.Â <a href="#MOESM2">5</a>; LR<sub>37, 88</sub>â€‰=â€‰508.0, <i>P</i>â€‰=â€‰0.944). Of the nine plant taxa returned as major taxa for honeybees in both 1952 and 2017, and present in over 1% of samples, seven of these plants show significant differences in use. This corresponds to differences in their frequency within the landscape as measured by the Countryside Survey (Fig.Â <a href="#Fig5">5</a>). The top forage found in 1952, white clover (<i>Trifolium repens</i>), was reported as a major plant in 74% of honey samples, decreasing to 31% in 2017 (<i>x</i><sup>2</sup>â€‰=â€‰229.51, d.fâ€‰=â€‰1, <i>P</i>â€‰<div class="â€‰0.001)._UNKNOWN">
Trifolium pratense) also decreased in use from 5% of honey samples to 1% (<i>x</i><sup>2</sup>â€‰=â€‰11.18, d.fâ€‰=â€‰1, <i>P</i>â€‰=â€‰0.027). Based on the Countryside Survey, <i>Trifolium repens</i> decreased in the landscape by 13% and <i>T. pratense</i> by 27% between 1978 and 2007. </div></div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <div class="title" tagx="title" title="title">
Change (%) in plant taxa used by honeybees from 1952 to 2017 along with changes in the abundance of those taxa in the Countryside Survey (%).</div> <p>The taxa included are those found as predominant and secondary within honey samples (&amp;gt;15% of pollen grains in mellisopalynology or &amp;gt;15% of DNA sequences) for more than 1% of samples in both surveys. The <i>p</i>-value for the chi-squared tests used Bonferroni correction for multiple testing. The Countryside Survey represents changes in the abundance of the taxa within 1577 fixed plots between 1978 and 2007.</p>   </div> <p /> <p id="Par13">Contrasting the decline in the <i>Trifolium</i> species, brambles (<i>Rubus</i> spp.) have seen an increase in forage use compared to 1952 and are now the most foraged genus for honeybees in the UK. In 1952, <i>Rubus</i> was the major taxa in only 5% of honey samples, compared to 31% in 2017 (<i>x</i><sup>2</sup>â€‰=â€‰367.07, d.fâ€‰=â€‰1, <i>P</i>â€‰<div class="â€‰0.001),_UNKNOWN">
Rubus fruticosus by 21% between 1978 and 2007.</div></p> <p id="Par14"><i>Brassica</i> species were the major taxa source in only 1% of honey samples in 1952 compared with 21% in 2017 (<i>x</i><sup>2</sup>â€‰=â€‰131.46, d.fâ€‰=â€‰1, <i>P</i>â€‰<div class="â€‰0.001),_UNKNOWN">
Brassica napus). No significant difference was found between the honey surveys for <i>Vicia</i> species, despite an increase in production of field beans (<i>Vicia faba</i>) since 1945 (<i>x</i><sup>2</sup>â€‰=â€‰7.15, d.fâ€‰=â€‰1, <i>P</i>â€‰=â€‰0.255)<sup><a href="#CR17">17</a>â€"<a href="#CR19">19</a></sup>. In contrast, the Countryside Survey shows a 26% decrease in <i>Vicia</i> species reflecting reductions in the availability of wild vetches.</div></p> <p id="Par15">Himalayan balsam (<i>Impatiens glandulifera</i>) is a non-native invasive species first introduced into the UK in 1839 which, after an initial lag phase, started to increase rapidly in distribution from 1940 to 1960<sup><a href="#CR20">20</a>,<a href="#CR21">21</a></sup>. <i>Impatiens glandulifera</i> increased as a major taxon from 1% of honey samples in 1952 to 6% in 2017 (<i>x</i><sup>2</sup>â€‰=â€‰22.17, d.fâ€‰=â€‰1, <i>P</i>â€‰<div class="â€‰0.001)._UNKNOWN" /></p> <p id="Par16">All analyses were additionally run on rarefied sequencing data (<a href="#MOESM2">Supplementary Results</a>). Using rarefied data did not change the conclusions from the statistical analyses completed here.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par17">Improved grassland under agricultural management is the most widespread habitat of the UK (Fig.Â <a href="#Fig1">1</a>) and has been estimated to provide the greatest contribution to nationwide nectar resource, with <i>Trifolium repens</i> as the dominant source of nectar<sup><a href="#CR12">12</a></sup>. However, the presence of flowering <i>Trifolium</i> species has reduced substantially within managed grasslands, due to decreasing use of clover leys in crop rotation and the increased application of inorganic nitrogen fertilizers and herbicides<sup><a href="#CR5">5</a>,<a href="#CR22">22</a></sup>. Furthermore, the clover that is present in modern grasslands may not be contributing to landscape estimates of nectar, as management that includes multiple cuts or intensive grazing can prevent flowering. Nevertheless, <i>T. repens</i> still represents the second most foraged plant in the 2017 survey suggesting that, despite declines in land cover, honeybees are still actively seeking out white clover.</p> <p id="Par18">White clover (<i>Trifolium repens</i>) and bramble (<i>Rubus fruticosus</i>) have similar flowering periods and the increased predominance of <i>Rubus</i> spp. within the honey may reflect the reduced availability of <i>Trifolium repens</i> during the same flowering period. Records show that <i>R. fruticosus</i> increased in local frequency between 1978 and 2007, however its distribution across the UK was not found to have changed between 1962 and 2001<sup><a href="#CR23">23</a></sup>. <i>R. fruticosus</i> and <i>T. repens</i> both offer pollen and nectar, however, the protein content and proportion of essential amino acids is lower in <i>R. fruticosus</i> compared to <i>T. repens</i><sup><a href="#CR24">24</a></sup> meaning honeybees may not be gaining the same nutritional benefits if substituting <i>T. repens</i> with <i>R. fruticosus</i>.</p> <p id="Par19">A shift in landscape forage availability occurred with the increase of oilseed rape as a UK crop since the 1970s (4884â€‰ha were grown in 1969 compared to 279,030 in 1988<sup><a href="#CR5">5</a>,<a href="#CR25">25</a>,<a href="#CR26">26</a></sup>), this is reflected in the increase in <i>Brassica</i> species within the 2017 honey samples. Field beans (<i>Vicia faba</i>) are another insect attractive species that have increased in production. However, the overall frequency of <i>Vicia</i> species within the honey was found to have decreased from 32 to 23%, reflecting the decline in wild vetches, which are a common component of species-rich grasslands<sup><a href="#CR27">27</a></sup>. As a major component of the honey, <i>Vicia</i> species have increased in frequency from 2 to 5%. with one possibility being that where there has been an increase in the availability of the field bean crop, honeybees will use them as a major honey source.</p> <p id="Par20">Himalayan balsam (<i>Impatiens glandulifera</i>) increased significantly as a major plant within the honey, representing a non-native invasive species that has increased in availability in the landscape. <i>I. glandulifera</i> is extremely attractive to pollinators with a higher nectar production when compared to other plant species associated with the same habitat<sup><a href="#CR28">28</a></sup>. Although attractive to pollinators, <i>I. glandulifera</i> can have a negative impact on native plant diversity by outcompeting other species for both space and pollinators, leading to a reduction in seed-set in co-occurring species<sup><a href="#CR28">28</a></sup>.</p> <p id="Par21">While the dominant habitat class surrounding the hives showed a significant relationship with the plant composition of the honey, it explained a limited amount of the total variation. The plants found here to be the most frequently used by honeybees are widely distributed in the UK. Honeybees may be selecting the same frequently found plants across different habitat classes, with the time of year being a better predictor for plant choice. Strong seasonal variation unrelated to the surrounding landscape diversity has been seen in the pollen collected by honeybees<sup><a href="#CR29">29</a></sup>. In addition, the foraging distances for pollen-collecting bees have been shown to vary both with the complexity of surrounding landscapes and season<sup><a href="#CR30">30</a></sup>, suggesting honeybees may be increasing their foraging range for certain forage plants. While the overall plant composition of the honey was found to unrelated to the location of the hive, further work could investigate the potential geographic patterns present in the spatially restricted plant species found at lower levels within the honey.</p> <p id="Par22">Agricultural intensification, changes in crop species and the spread of non-native invasive plants all contribute to changes in the available forage for honeybees and have wider implications for pollinator habitat management, since the key taxa identified represent the plant species which provide the greatest abundance of nectar nationally within the UK<sup><a href="#CR8">8</a></sup>.</p> <p id="Par23">On a landscape scale, the management recommendation that has the greatest potential to increase the quantity of nectar on a UK-wide basis is to increase the presence and diversity of nectar-rich species within improved grasslands, including flowering clover (<i>Trifolium repens</i> and <i>T. pratense</i>). Improved grasslands represent the most extensive habitat type of the UK and changes to increase plant diversity and flower availability within this habitat will have the greatest impact on nectar and pollen provision.</p> <p id="Par24">Honeybees as a model provide an overview of the availability of these widespread foraging resources on an otherwise unachievable scale. However, this information should be set within the context of the wider pollinator community when discussing increasing forage provision. Honeybee foraging differs in comparison to other wild pollinators including solitary bees, bumblebees, and hoverflies who are not able to reach resources at the same scale. Honeybeesâ€™ larger forage range comes from their large social structure coupled with their ability to communicate forage location<sup><a href="#CR31">31</a></sup>. In addition to behavioural differences, honeybees have physiological differences to other pollinators, including aspects such as tongue length, which can restrict access to other important sources of forage<sup><a href="#CR32">32</a></sup>. Recommendations for specific forage plants should therefore consider the needs of the wider pollinator cohort and the appropriate diversity of flowers required to meet the needs of a diverse and resilient pollinator community.</p> <p id="Par25">Here we show a significant correlation in the overall presence and absence of plant taxa found by both the DNA metabarcoding in 2017 and by melissopalynology in 1952, with significant differences when predominant and secondary foraging is examined. The robustness of DNA metabarcoding in accurately representing the abundance of biomass within a system is debated, and while positive correlations between the proportion of DNA sequences and relative abundance have been found, this can be accompanied by a high degree of variance between markers<sup><a href="#CR33">33</a>â€"<a href="#CR36">36</a></sup>.</p> <p id="Par26">To our knowledge, this study represents the first UK-wide floral analysis of honey samples since the melissopalynology survey of 1952. The changes observed in honeybee foraging between 1952 and 2017 are evidencing the impacts of widespread changes in available forage from shifts in agricultural management, the presence of crop species and the spread of invasive species. Charting these changes has management implications for providing nectar and pollen forage nationally.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="2017honeysampling" title="sec"> <div class="title" tagx="title" title="title">
2017 honey sampling</div> <p id="Par27">Beekeepers were invited to provide honey for analysis via a nationwide campaign publicised on the gardening programme, BBC Gardenerâ€™s World (broadcast July 2017). Participating beekeepers were asked to supply ~30â€‰ml of honey from any date in 2017, reporting the date of sample collection and the location of the apiary, using a grid reference or postcode. In total 441 honey samples were processed from beekeepers.</p> </div> <div class="honeydnaextraction" title="sec"> <div class="title" tagx="title" title="title">
Honey DNA extraction</div> <p id="Par28">Any wax was removed using sterile forceps and DNA was extracted from 10â€‰g of honey using a modified version of the DNeasy Plant Mini extraction kit (Qiagen). Firstly, the 10â€‰g of honey was made up to 30â€‰ml with molecular grade water and incubated in a water bath at 65â€‰Â°C for 30â€‰min. Samples were then centrifuged (Sorvall RC-5B) for 30â€‰min at 15,000â€‰rpm, the supernatant was discarded, and the pellet resuspended in 400â€‰Î¼L of a buffer made from a mix of 400â€‰Î¼L AP1 from the DNeasy Plant Mini Kit (Qiagen), 80â€‰Î¼L proteinase K (1â€‰mg/ml) (Sigma) and 1â€‰Î¼L RNase A (Qiagen). This was incubated again for 60â€‰min at 65â€‰Â°C in a water bath and then disrupted using a TissueLyser II (Qiagen) for 4â€‰min at 30â€‰Hz with 3â€‰mm tungsten carbide beads. The remaining steps were carried out according to the manufacturerâ€™s protocol, excluding the use of the QIAshredder and the second wash stage. The extracted DNA was purified using the OneStep PCR Inhibitor Removal Kit (Zymo Research) and diluted 1 in 10.</p> </div> <div class="pcrandlibrarypreparation" title="sec"> <div class="title" tagx="title" title="title">
PCR and library preparation</div> <p id="Par29">Illumina MiSeq paired-end indexed amplicon libraries were created via a two-step PCR protocol. Two libraries were prepared for the DNA barcode regions, <i>rbcL</i> and ITS2. Initial amplification used the template specific primers <i>rbcL</i>af and <i>rbcLr</i>506<sup><a href="#CR37">37</a></sup>, and ITS2F and ITS3R, with universal tails designed to attach custom indices in the second-round PCR. To improve clustering on the Illumina MiSeq, a 6N sequence was also added between the forward template specific primer and the universal tail.</p> <p id="Par30">Forward universal tail, 6N sequence and <i>rbcL</i>af: [<b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</b>]NNNNNN[ATGTCACCACAAACAGAGACTAAAGC]</p> <p id="Par31">Reverse universal tail and <i>rbcL</i>r506: [<b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT</b>][AGGGGACGACCATACTTGTTCA]</p> <p id="Par32">Forward universal tail, 6N sequence and ITS2F: [<b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</b>]NNNNNN[ATGCGATACTTGGTGTGAAT]</p> <p id="Par33">Reverse universal tail and ITS3R: [<b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT</b>][GACGCTTCTCCAGACTACAAT]</p> <p id="Par34">This first PCR used a final volume of 20â€‰Î¼l: 2â€‰Î¼l template DNA, 10â€‰Î¼l of 2Ã— Phusion Hot Start II High-Fidelity Mastermix (New England Biolabs UK), 0.4â€‰Î¼l (2.5â€‰ÂµM) forward and reverse primers, and 7.2â€‰Î¼l of PCR grade water. Thermal cycling conditions for <i>rbcL</i> were: 98â€‰Â°C for 30â€‰s, 95â€‰Â°C for 2â€‰min; 95â€‰Â°C for 30â€‰s, 50â€‰Â°C for 30â€‰s, 72â€‰Â°C for 40â€‰s (40 cycles); 72â€‰Â°C for 5â€‰min, 30â€‰Â°C for 10â€‰s. Thermal cycling conditions for the first ITS2 PCR were: 98â€‰Â°C for 30â€‰s 94â€‰Â°C for 5â€‰min; 94â€‰Â°C for 30â€‰s, 56â€‰Â°C for 30â€‰s, 72â€‰Â°C for 40â€‰s (40 cycles); 72â€‰Â°C for 10â€‰min, 30â€‰Â°C for 1â€‰min. The initial PCR was carried out three times and pooled.</p> <p id="Par35">The pooled products from the first PCR were purified following Illuminaâ€™s 16S Metagenomic Sequencing Library Preparation protocol using Agencourt AMPure XP beads (Beckman Coulter). The purified PCR product from round one was followed by a second round of amplification to anneal custom unique and identical i5 and i7 indices to each sample (Ultramer, Integrated DNA Technologies).</p> <p id="Par36">This index PCR stage used a final volume of 25â€‰Î¼l reaction (12.5â€‰Î¼l of 2Ã— Phusion Hot Start II High-Fidelity Mastermix, 1â€‰Î¼l of i7 Index Primer and i5 Index Primer, 6.5â€‰Î¼l of PCR grade water, and 5â€‰Î¼l of purified first-round PCR product). Thermal cycling conditions were: 98â€‰Â°C for 30â€‰s; 95â€‰Â°C for 30â€‰s, 55â€‰Â°C for 30â€‰s, 72â€‰Â°C for 30â€‰s (8 cycles); 72â€‰Â°C for 5â€‰min, 4â€‰Â°C for 10â€‰min. Following the index PCR, a 1% gel was run to verify its success. The index PCR product was then purified following the PCR clean-up two sections of the Illumina protocol. The purified products of the index PCR were quantified using a Qubit 3.0 fluorescence spectrophotometer (Thermo Fisher Scientific) and pooled at equal concentrations to produce the final library. Positive and negative controls were amplified and sequenced alongside honey samples. The positive control was made from a mixture of five tropical tree species that were not present in the survey site. The species <i>Baccaurea stipulata</i>, <i>Colona serratifolia</i>., <i>Dillenia excelsa</i>, <i>Kleinhovia hospita</i>, and <i>Pterospermum macrocarpum</i> were used, taking 5â€‰Î¼l from each separate DNA extraction and mixing, before following the protocol as with the honey samples. All five species were detected within the sequencing results.</p> </div> <div class="bioinformaticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatic analysis</div> <p id="Par37">Sequence data were processed using a modified data analysis pipeline<sup><a href="#CR14">14</a>,<a href="#CR38">38</a></sup>. Raw reads were trimmed to remove low-quality regions (Trimmomatic v. 0.33), paired, and then merged (FLASH v. 1.2.11), with merged reads shorter than 450â€‰bp discarded. Identical reads were dereplicated within samples and then clustered at 100% identity across all samples (vsearch v. 2.3.2), with singletons (sequence reads that occurred only once across all samples) discarded.</p> <p id="Par38">The Barcode Wales and Barcode UK projects provide 98% coverage for the native flowering plants and conifers of the UK<sup><a href="#CR37">37</a></sup>. This reference library was supplemented with a curated library of the non-native and horticultural species, downloaded from GenBank. This UK species list was generated using the list of native species of the UK from Stace (2010)<sup><a href="#CR39">39</a></sup>, 505 naturalised alien species (BSBI), and horticultural species from the IRIS BG database at the National Botanic Garden of Wales.</p> <p id="Par39">The sequence data from the honey samples were compared against the reference database using blastn, using the script vsearch-pipe.py. The top BLAST hits were then summarised using the script vsearch_blast_summary.py. Sequences with bit scores below the 1st percentile were excluded. If the top bit scores of a sequence matched to a single species, then the sequence was identified to that species. If the top bit scores matched to different species within the same genus, then the result was attributed to the genus level. If the top bit score belonged to multiple genera within the same family then a family level designation was made. Sequences that returned families from different clades were excluded. These automated identifications were then checked manually for botanical veracity. To check identified plant species against their availability across the UK, species records from the BSBI (Botanical Society of Britain and Ireland) were used for native species, while commercial availability for horticultural species was verified with the RHS Plant Finder<sup><a href="#CR40">40</a></sup>. Within each sample, the number of sequences returned from <i>rbcL</i> and ITS2 for each plant taxon was summed to combine the results of each marker.</p> <p id="Par40">The proportion of sequences was used in the analysis, which has been shown to be an appropriate method to control for differences in read number<sup><a href="#CR41">41</a></sup>. Alternatively, the sequencing data can be rarefied, but this has been criticised as a statistical technique, due to requiring the removal of valid data<sup><a href="#CR41">41</a></sup>. To investigate the impact of rarefying on the conclusions drawn from the data, all analyses were rerun with rarefied data (<a href="#MOESM2">Supplementary Results</a>).</p> </div> <div class="1952honeysampling" title="sec"> <div class="title" tagx="title" title="title">
1952 Honey sampling</div> <p id="Par41">In 1952, 855 honey samples were characterised from 66 counties across the UK and Ireland using melissopalynology<sup><a href="#CR15">15</a>,<a href="#CR16">16</a></sup>. The methods reported for the research conducted in 1952 are described here fully for comparison. Samples were obtained via a general appeal and were all collected during the honey season of 1952. For each honey sample, ~200 pollen grains were identified using the morphology of the pollen under the microscope, following a standardised protocol<sup><a href="#CR42">42</a></sup>. To extract the pollen, 10â€‰g of honey was dissolved in 20â€‰ml of distilled water, from which 10â€‰ml was taken and centrifuged at ~2000â€‰rpm for one minute. The supernatant was discarded, and the sediment retained, and then the process was repeated for the remaining liquid. From the sediment, a drop was transferred to a glass slide and spread out over an area of 1â€‰cm<sup>2</sup>, before being stained with fuchsin and dried. Euparal vert was used as a final mounting medium. Pollen was identified by comparison with a reference library of pollen preparations and available pollen morphological data<sup><a href="#CR43">43</a>,<a href="#CR44">44</a></sup>. Each plant taxon found in the sampled honey was reported according to the proportion of pollen grains found and classed into predominant (&amp;gt;45% of pollen grains), secondary (15â€"45% of pollen grains) and important minor (1â€"15% of pollen grains). The location data for the honey samples were restricted to the county level, and summary data tables were presented for each UK county that returned honey.</p> </div> <div class="comparingthe1952and2017honeysamples" title="sec"> <div class="title" tagx="title" title="title">
Comparing the 1952 and 2017 honey samples</div> <p id="Par42">The plants detected using DNA metabarcoding and melissopalynology have been compared in previous studies with concordance found between the two methods<sup><a href="#CR45">45</a>â€"<a href="#CR48">48</a></sup>. Both methods detect the same major taxa, but rarer species in a sample are less likely to be found consistently, both when comparing methods and also during replicates of the same method<sup><a href="#CR45">45</a>â€"<a href="#CR47">47</a></sup>. DNA metabarcoding is often able to detect more taxa when compared to melissopalynology, by identifying rarer species in the sample and by achieving higher taxonomic resolution in certain cases. While melissopalynology uses counts of pollen grains to provide a starting point for quantitative analysis, DNA metabarcoding as a process is semi-quantitative, with biases associated with the process of DNA extraction, PCR and sequencing<sup><a href="#CR33">33</a>,<a href="#CR45">45</a></sup>. To allow for these considerations we placed the proportion of DNA sequence reads and pollen counts into four broad abundance classes matching the classifications used in melissopalynology (predominant, secondary, important minor and minor) and focus our analyses and conclusions on changes in the frequency of occurrence of the major taxa, classed as predominant and secondary. Both methods capture information on both nectar and pollen plants within the honey, however, certain species can be over or under represented in pollen analysis compared to their relative nectar contribution<sup><a href="#CR49">49</a></sup>. Both pollen and nectar plants are required to meet the foraging requirements of pollinators.</p> </div> <div class="statisticsandreproducibility" title="sec"> <div class="title" tagx="title" title="title">
Statistics and reproducibility</div> <div class="statisticalanalysisofdnametabarcodingdata" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis of DNA metabarcoding data</div> <p id="Par43">To understand how the plant taxa composition within the honey sample was structured in space and time, the effect of time (measured as the calendar month number in 2017), latitude and longitude of sampling location were included in a single, two-tailed generalized linear model using the â€˜manyglmâ€™ function in the package â€˜mvabundâ€™<sup><a href="#CR50">50</a></sup>. Honey samples with missing metadata were excluded, giving a sample size of 428. An abundance table of taxa (number of sequence reads) found in each sample was set as the multivariate response variable and a common set of predictor variables (month, latitude and longitude) were fit using a negative binomial distribution. The number of sequence reads per sample was included as an â€œoffsetâ€� in the model in order to control for differences in the number of sequence reads between samples. Monte Carlo resampling was used to test for significant community-level responses to our predictors. The strong mean-variance relationship in the data (Supplementary Fig.Â <a href="#MOESM2">6</a>) and the distribution of the count data (Supplementary Figs.Â <a href="#MOESM2">7</a>,Â <a href="#MOESM2">8</a>) support the use of a negative binomial distribution in the model. The appropriateness of the models was checked by visual inspection of the residuals against predicted values from the models (Supplementary Figs.Â <a href="#MOESM2">9</a>â€"<a href="#MOESM2">11</a>).</p> <p id="Par44">We completed a spatial eigenfunction analysis using distance-based Moranâ€™s eigenvectors. Moranâ€™s Eigenvector Maps were computed using the â€˜memâ€™ function from the adespatial package. Moranâ€™s I was computed for each taxa using the â€˜moran.randtestâ€™, with Bonferroni correction for multiple testing. The direction of autocorrelation (positive and negative) was tested using the â€˜moranNP.randtestâ€™ function, using the adespatial package in R.</p> </div> <div class="statisticalanalysisofthe1952and2017honeysamples" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis of the 1952 and 2017 honey samples</div> <p id="Par45">Abundance classes were assigned based on the percentage of reads returned for the two DNA regions <i>rbcL</i> and ITS2, matching the classifications used in melissopalynology. Plant taxa represented by over 45% of reads were designated <i>predominant</i> for that sample; between 15 and 45% were <i>secondary</i>; between 1 and 15% were <i>important minor</i> taxa, and &amp;lt;1% of reads were classed as <i>minor</i> taxa. The number of times each taxon occurred at each level of abundance was then calculated, with the sum of this giving the frequency of occurrence across all the samples.</p> <p id="Par46">The results of the 2017 analysis were then compared with 855 honey samples characterised in 1952, from across the UK and Ireland using melissopalynology<sup><a href="#CR15">15</a>,<a href="#CR16">16</a></sup>. The relationship between the frequency of occurrence for the matched plant taxa between 1952 and 2017 was assessed using Kendallâ€™s rank correlation.</p> <p id="Par47">To compare the major taxa (classed as predominant and secondary) between 2017 and 1952, the effect of sample location (UK county name) and sample year (2017 or 1952) were included in a two-tailed generalized linear model using the â€˜manyglmâ€™ function in the package â€˜mvabundâ€™<sup><a href="#CR50">50</a></sup>. In the absence of latitude and longitude for honey samples collected in 1952, UK ceremonial county names were used as a proxy for the location for both 2017 and 1952 honey samples. Honey samples from 2017 were assigned their ceremonial county based on latitude and longitude and matched to the counties listed in 1952. Using a binomial distribution, the effect of county location and year (1952 and 2017) were included as explanatory variables in the model and the presence or absence of each taxa was set as the response variable. The appropriateness of the models was checked by visual inspection of the residuals against predicted values from the models (Supplementary Fig.Â <a href="#MOESM2">11</a>).</p> <p id="Par48">The change in proportion of major (predominant and secondary) taxa between 1952 and 2017 was examined for the plant taxa that occurred as major taxa in more than 1% of samples for both honey surveys. Chi-squared contingency tests were used to assess differences, with Bonferroni correction for multiple testing. All statistical analyses were carried out using <i>R</i> (v. 3.5.2).</p> </div> <div class="countrysidesurveyvegetationplotdatafrequencychanges" title="sec"> <div class="title" tagx="title" title="title">
Countryside Survey vegetation plot data frequency changes</div> <p id="Par49">Changes in the local frequency of the major plant forage species found in both 1952 and 2017 were assessed using the Countryside Survey data from 1978 to 2007<sup><a href="#CR51">51</a>â€"<a href="#CR53">53</a></sup>. In 1978, the survey looked at 256 1â€‰km squares within which fixed plots were established, representing fields and unenclosed land (200â€‰m<sup>2</sup>) as well as linear features including hedgerows, streams and roadsides (10â€‰m<sup>2</sup>). In each plot, a list of all vascular plants was recorded. Where possible, squares and plots were then revisited in 2007, representing 236 1â€‰km squares containing 1577 plots. For these revisited plots, the percentage change in plot frequency was calculated<sup><a href="#CR54">54</a></sup>.</p> </div> <div class="landscapedata" title="sec"> <div class="title" tagx="title" title="title">
Landscape data</div> <p id="Par50">The Land Cover 2017 map was used to characterise habitat in a 2â€‰km radius of the hives<sup><a href="#CR55">55</a></sup> while the 2017 CEH Land Cover Plus: Crops map was used to assess the presence and absence of crop species, <i>Brassica napus</i> (oilseed rape) and <i>Vicia faba</i> (field beans), within a 2â€‰km radius of each hive. A chi-squared contingency test was used to analyse the differences between the presence of the crop species in the honey and the presence and absence of the crop within the landscape. Non-metric multidimensional scaling (NMDS) ordination was used to visualise differences in the composition of the honey relating to the dominant habitat type in a 2â€‰km radius, based on the proportion of reads returned for each taxon. Ordinations were carried out using the metaMDS function in the vegan package<sup><a href="#CR56">56</a></sup> in R using Bray-Curtis dissimilarity indices. The differences in plant community composition and surrounding dominant habitat type were tested using the <i>adonis</i> function from vegan, with 999 permutations. Analyses and maps were generated in <i>R</i> (v. 3.5.2).</p> </div> </div> <div class="reportingsummary" title="sec"> <div class="title" tagx="title" title="title">
Reporting summary</div> <p id="Par51">Further information on research design is available in theÂ <a href="#MOESM4">Nature Research Reporting Summary</a> linked to this article.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="42003_2020_1562_MOESM1_ESM.pdf">LINK</a> <p>Peer Review File</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="42003_2020_1562_MOESM2_ESM.pdf">LINK</a> <p>Supplementary Information</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="42003_2020_1562_MOESM3_ESM.xlsx">LINK</a> <p>Supplementary Data</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="42003_2020_1562_MOESM4_ESM.pdf">LINK</a> <p>Reporting Summary</p>  </div> </div> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p><b>Supplementary information</b> is available for this paper at 10.1038/s42003-020-01562-4.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Funding to support this research was provided by the National Botanic Garden of Wales and a bequest from Dr Quentin Kay. L.J. and A.L. were supported by Knowledge Economy Skills Scholarships (KESS2), part funded by the Welsh Governmentâ€™s European Social Fund (ESF). Thank you to beekeepers throughout the UK for generously providing honey samples. Thank you to Adam Leitch for making A.S.C. Deansâ€™ 1958 NDB thesis available to us.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>The study was conceived by NdV and L.J. The lab work was carried out by L.J. The data were compiled by L.J. and A.L. and analysed by L.J., C.F. and G.B. with suggestions from N.dV. and S.C. The manuscript was written by L.J. and N.dV. with contributions from all of the authors.</p> <div class="title" tagx="title" title="title">
Data availability</div> <p>Sequence data are available at the Sequence Read Archive (SRA) accession number PRJNA577454. Summarised sequence data are provided inÂ <a href="#MOESM3">Supplementary Data</a>.</p> <div class="title" tagx="title" title="title">
Code availability</div> <p>The code for processing the sequencing data is available at <a href="https://github.com/colford/nbgw-plant-illumina-pipeline">https://github.com/colford/nbgw-plant-illumina-pipeline</a><sup><a href="#CR38">38</a></sup>.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par52">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Potts</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Global pollinator declines: trends, impacts and drivers</span><span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">345</span><span class="lpage" tagx="lpage" title="lpage">353</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tree.2010.01.007">10.1016/j.tree.2010.01.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20188434">20188434</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powney</span><span class="given-names" tagx="given-names" title="given-names">GD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Widespread losses of pollinating insects in Britain</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1018</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-019-08974-9">10.1038/s41467-019-08974-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30914632">30914632</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goulson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicholls</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Botias</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rotheray</span><span class="given-names" tagx="given-names" title="given-names">EL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bee declines driven by combined stress from parasites, pesticides, and lack of flowers</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">347</span><span class="fpage" tagx="fpage" title="fpage">1255957</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1255957">10.1126/science.1255957</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25721506">25721506</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevens</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The extent of semi-natural grassland communities in lowland England and Wales: a review of conservation surveys 1978â€"96</span><span class="source" tagx="source" title="source">Grass Forage Sci.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">18</span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutherland</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Post-war changes in arable farming and biodiversity in Great Britain</span><span class="source" tagx="source" title="source">J. Appl. Ecol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">157</span><span class="lpage" tagx="lpage" title="lpage">176</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-2664.2002.00695.x">10.1046/j.1365-2664.2002.00695.x</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petit</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuart</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gillespie</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barr</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Field boundaries in Great Britain: Stock and change between 1984, 1990 and 1998</span><span class="source" tagx="source" title="source">J. Environ. Manag.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">229</span><span class="lpage" tagx="lpage" title="lpage">238</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0301-4797(02)00176-7">10.1016/S0301-4797(02)00176-7</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">BH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Agricultural intensification and the loss of habitat, species and amenity in British grasslands: a review of historical change and assessment of future prospects</span><span class="source" tagx="source" title="source">Grass Forage Sci.</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">365</span><span class="lpage" tagx="lpage" title="lpage">372</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2494.1990.tb01961.x">10.1111/j.1365-2494.1990.tb01961.x</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BatÃ¡ry</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dicks</span><span class="given-names" tagx="given-names" title="given-names">LV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kleijn</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutherland</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">The role of agri-environment schemes in conservation and environmental management</span><span class="source" tagx="source" title="source">Conserv. Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">1006</span><span class="lpage" tagx="lpage" title="lpage">1016</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/cobi.12536">10.1111/cobi.12536</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25997591">25997591</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benton</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vickery</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Farmland biodiversity: Is habitat heterogeneity the key?</span><span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">182</span><span class="lpage" tagx="lpage" title="lpage">188</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0169-5347(03)00011-9">10.1016/S0169-5347(03)00011-9</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">French</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Species composition determines forage quality and medicinal value of high diversity grasslands in lowland England</span><span class="source" tagx="source" title="source">Agriculture, Ecosyst. Environ.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">241</span><span class="fpage" tagx="fpage" title="fpage">193</span><span class="lpage" tagx="lpage" title="lpage">204</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.agee.2017.03.012">10.1016/j.agee.2017.03.012</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuller</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="mixed-article-title" title="mixed-article-title">The changing extent and conservation interest of lowland grasslands in England and Wales: a review of grassland surveys 1930â€"1984</span><span class="source" tagx="source" title="source">Biol. Conserv.</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">281</span><span class="lpage" tagx="lpage" title="lpage">300</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0006-3207(87)90121-2">10.1016/0006-3207(87)90121-2</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baude</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Historical nectar assessment reveals the fall and rise of floral resources in Britain</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">530</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">88</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature16532">10.1038/nature16532</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26842058">26842058</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Couvillon</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¼rch</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratnieks</span><span class="given-names" tagx="given-names" title="given-names">FLW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dancing bees communicate a foraging preference for rural lands in high-level agri-environment schemes</span><span class="source" tagx="source" title="source">Curr. Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1212</span><span class="lpage" tagx="lpage" title="lpage">1215</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2014.03.072">10.1016/j.cub.2014.03.072</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24856213">24856213</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vere</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Using DNA metabarcoding to investigate honey bee foraging reveals limited flower use despite high floral availability</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">42838</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep42838">10.1038/srep42838</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28205632">28205632</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Deans, A. S. C. <i>Survey of British Honey Sources</i>. (Bee Research Association, 1957).</span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Deans, A. S. C. The Pollen Analysis of some British Honeys. (Thesis for the National Bee Diploma, 1958).</span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">DEFRA. Agriculture in the United Kingdom 2017. (2018).</span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breeze</span><span class="given-names" tagx="given-names" title="given-names">TDD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailey</span><span class="given-names" tagx="given-names" title="given-names">APP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balcombe</span><span class="given-names" tagx="given-names" title="given-names">KGG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Potts</span><span class="given-names" tagx="given-names" title="given-names">SGG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pollination services in the UK: How important are honeybees?</span><span class="source" tagx="source" title="source">Agriculture, Ecosyst. Environ.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">142</span><span class="fpage" tagx="fpage" title="fpage">137</span><span class="lpage" tagx="lpage" title="lpage">143</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.agee.2011.03.020">10.1016/j.agee.2011.03.020</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Williams, I. H. &amp;amp; Carreck, N. L. Land use changes and honey bee forage plants. In <i>Forage for bees in an agricultural landscape</i>. 7â€"20 (1994).</span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usher</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kornberg</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horwood</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Southwood</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Invasibility and wildlife conservation: invasive species on nature reserves</span><span class="source" tagx="source" title="source">Philos. Trans. R. Soc. B: Biol. Sci.</span><span class="year" tagx="year" title="year">1986</span><span class="volume" tagx="volume" title="volume">314</span><span class="fpage" tagx="fpage" title="fpage">695</span><span class="lpage" tagx="lpage" title="lpage">710</span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rich</span><span class="given-names" tagx="given-names" title="given-names">TCG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodruff</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span></span><span class="mixed-article-title" title="mixed-article-title">Changes in the vascular plant floras of England and Scotland between 1930â€"1960 and 1987â€"1988: the BSBI monitoring scheme</span><span class="source" tagx="source" title="source">Biol. Conserv.</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">217</span><span class="lpage" tagx="lpage" title="lpage">229</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0006-3207(95)00077-1">10.1016/0006-3207(95)00077-1</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arroyo</span><span class="given-names" tagx="given-names" title="given-names">BE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bradbury</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span></span><span class="mixed-article-title" title="mixed-article-title">A review of the abundance and diversity of invertebrate and plant foods of granivorous birds in northern Europe in relation to agricultural change</span><span class="source" tagx="source" title="source">Agriculture, Ecosyst. Environ.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">13</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0167-8809(99)00064-X">10.1016/S0167-8809(99)00064-X</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Preston, C. D., Pearman, D. A. &amp;amp; Dines, T. D. New Atlas of the British and Irish Flora: An Atlas of the Vascular Plants of Britain, Ireland, The Isle of Man and the Channel Islands. (Oxford University Press, 2002).</span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franco</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Breeding system, pollinator choice and variation in pollen quality in British herbaceous plants</span><span class="source" tagx="source" title="source">Funct. Ecol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">592</span><span class="lpage" tagx="lpage" title="lpage">598</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2435.2008.01415.x">10.1111/j.1365-2435.2008.01415.x</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chamberlain</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuller</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Local extinctions and changes in species richness of lowland farmland birds in England and Wales in relation to recent changes in agricultural land-use</span><span class="source" tagx="source" title="source">Agriculture, Ecosyst. Environ.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">17</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0167-8809(99)00105-X">10.1016/S0167-8809(99)00105-X</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutcliffe</span><span class="given-names" tagx="given-names" title="given-names">OL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kay</span><span class="given-names" tagx="given-names" title="given-names">QON</span></span></span><span class="mixed-article-title" title="mixed-article-title">Changes in the arable flora of central southern England since the 1960s</span><span class="source" tagx="source" title="source">Biol. Conserv.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0006-3207(99)00119-6">10.1016/S0006-3207(99)00119-6</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Stevens, D., Smith, S., Blackstock, T., Bosanquet, S. &amp;amp; Stevens, J. <i>Grasslands of Wales: A Survey of Lowland Species Rich Grasslands 1987â€"2004</i>. (University of Wales Press, 2010).</span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chittka</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¼rkens</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Successful invasion of a floral market</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">411</span><span class="fpage" tagx="fpage" title="fpage">653</span><span class="lpage" tagx="lpage" title="lpage">653</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/35079676">10.1038/35079676</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11395755">11395755</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danner</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keller</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HÃ¤rtel</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steffan-Dewenter</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Honey bee foraging ecology: season but not landscape diversity shapes the amount and diversity of collected pollen</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">9</span><span class="lpage" tagx="lpage" title="lpage">12</span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steffan-Dewenter</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuhn</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Honeybee foraging in differentially structured landscapes</span><span class="source" tagx="source" title="source">Proc. R. Soc. B: Biol. Sci.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">270</span><span class="fpage" tagx="fpage" title="fpage">569</span><span class="lpage" tagx="lpage" title="lpage">575</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2002.2292">10.1098/rspb.2002.2292</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Couvillon</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Honey bee foraging distance depends on month and forage type</span><span class="source" tagx="source" title="source">Apidologie</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">61</span><span class="lpage" tagx="lpage" title="lpage">70</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s13592-014-0302-5">10.1007/s13592-014-0302-5</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Comba</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corbet</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunt</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warren</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Flowers, nectar and insect visits: evaluating british plant species for pollinator-friendly gardens</span><span class="source" tagx="source" title="source">Ann. Bot.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">369</span><span class="lpage" tagx="lpage" title="lpage">383</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/anbo.1998.0835">10.1006/anbo.1998.0835</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deagle</span><span class="given-names" tagx="given-names" title="given-names">BE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Counting with DNA in metabarcoding studies: how should we convert sequence reads to dietary data?</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">391</span><span class="lpage" tagx="lpage" title="lpage">406</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.14734">10.1111/mec.14734</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29858539">29858539</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Quantitative and qualitative assessment of pollen DNA metabarcoding using constructed species mixtures</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">431</span><span class="lpage" tagx="lpage" title="lpage">455</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.14840">10.1111/mec.14840</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30118180">30118180</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keller</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluating multiplexed next-generation sequencing as a method in palynology for mixed pollen samples</span><span class="source" tagx="source" title="source">Plant Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">558</span><span class="lpage" tagx="lpage" title="lpage">566</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/plb.12251">10.1111/plb.12251</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25270225">25270225</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Rankâ€�based characterization of pollen assemblages collected by honey bees using a multiâ€�locus metabarcoding approach</span><span class="source" tagx="source" title="source">Appl. Plant Sci.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">1500043</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/apps.1500043">10.3732/apps.1500043</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vere</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">DNA barcoding the native flowering plants and conifers of wales</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">e37945</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0037945">10.1371/journal.pone.0037945</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22701588">22701588</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ford, C. R. &amp;amp; Jones, L. nbgw-plant-illumina-pipeline: Shifts in honeybee foraging reveal historical changes in floral resources. <a href="https://zenodo.org/record/4166248#.X6A5l4j7RhE">https://zenodo.org/record/4166248#.X6A5l4j7RhE</a> (2020).</span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Stace, C. <i>New Flora of the British Isles</i>. (Cambridge University Press, 2010).</span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Cubey, J. <i>RHS Plant Finder 2017</i>. (2017).</span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">McMurdie, P. J. &amp;amp; Holmes, S. Waste Not, Want Not: Why Rarefying Microbiome Data Is Inadmissible. <i>PLoS Comput. Biol.</i><b>10</b> (2014).</span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maurizio</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodges</span><span class="given-names" tagx="given-names" title="given-names">FED</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pollen analysis of honey</span><span class="source" tagx="source" title="source">Bee World</span><span class="year" tagx="year" title="year">1951</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">5</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/0005772X.1951.11094660">10.1080/0005772X.1951.11094660</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hodges, D. <i>The Pollen Loads of the Honeybee</i>. (Bee Research Association, 1952).</span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Faegri, K. &amp;amp; Iversen, J. <i>Text-book of Pollen Analysis</i>. (E. Munksgaard, 1950).</span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hawkins, J. et al. Using DNA metabarcoding to identify the floral composition of honey: a new tool for investigating honey bee foraging preferences. <i>PLOS ONE</i><b>10</b> (2015).</span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Richardson, R. T. et al. Application of ITS2 metabarcoding to determine the provenance of pollen collected by honey bees in an agroecosystem. <i>Appl. Plant Sci.</i><b>3</b> (2015).</span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smart</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A comparison of honey bee-collected pollen from working agricultural lands using light microscopy and its metabarcoding</span><span class="source" tagx="source" title="source">Environ. Entomol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">38</span><span class="lpage" tagx="lpage" title="lpage">49</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28062536">28062536</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sponsler</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A screening-level assessment of the pollinator-attractiveness of ornamental nursery stock using a honey bee foraging assay</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">831</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-020-57858-2">10.1038/s41598-020-57858-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31965017">31965017</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bryant</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">GD</span></span></span><span class="mixed-article-title" title="mixed-article-title">The r-values of honey: pollen coefficients</span><span class="source" tagx="source" title="source">Palynology</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">28</span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naumann</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warton</span><span class="given-names" tagx="given-names" title="given-names">DI</span></span></span><span class="mixed-article-title" title="mixed-article-title">mvabund - an R package for model-based analysis of multivariate abundance data</span><span class="source" tagx="source" title="source">Methods Ecol. Evolution</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">471</span><span class="lpage" tagx="lpage" title="lpage">474</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.2041-210X.2012.00190.x">10.1111/j.2041-210X.2012.00190.x</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Long-term vegetation monitoring in Great Britain - The Countryside Survey 1978â€"2007 and beyond</span><span class="source" tagx="source" title="source">Earth Syst. Sci. Data</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">445</span><span class="lpage" tagx="lpage" title="lpage">459</span><span class="pub-id"><a href="https://dx.doi.org/10.5194/essd-9-445-2017">10.5194/essd-9-445-2017</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bunce, R. G. H. et al. Countryside Survey 2007 vegetation plot data. 10.5285/57f97915-8ff1-473b-8c77-2564cbd747bc (2014).</span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">C. J., B., Bunce, R. G., Smart, S. M. &amp;amp; Whittaker H. A. Countryside Survey 1978 vegetation plot data. 10.5285/67bbfabb-d981-4ced-b7e7-225205de9c96 (2014).</span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carvell</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Declines in forage availability for bumblebees at a national scale</span><span class="source" tagx="source" title="source">Biol. Conserv.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">132</span><span class="fpage" tagx="fpage" title="fpage">481</span><span class="lpage" tagx="lpage" title="lpage">489</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biocon.2006.05.008">10.1016/j.biocon.2006.05.008</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Rowland, C. S. et al. Land Cover Map 2015 (vector, GB). <i>NERC Environmental Information Data Centre</i>10.5285/6c6c9203-7333-4d96-88ab-78925e7a4e73 (2017).</span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dixon</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Computer program review VEGAN, a package of R functions for community ecology</span><span class="source" tagx="source" title="source">J. Vegetation Sci.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">927</span><span class="lpage" tagx="lpage" title="lpage">930</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1654-1103.2003.tb02228.x">10.1111/j.1654-1103.2003.tb02228.x</a></span></span></li> </ul> </div> </div>  </body></html>