<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">ACS Omega</span><span class="iso-abbrev" title="iso-abbrev">ACS Omega</span><span class="publisher-id" title="publisher-id">ao</span><span class="coden" title="coden">acsodf</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">ACS Omega</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2470-1343</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">American Chemical Society</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7841945">7841945</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1021/acsomega.0c05472">10.1021/acsomega.0c05472</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Endophytic <i>Akanthomyces</i> sp. LN303 from Edelweiss Produces Emestrin and Two New 2-Hydroxy-4 Pyridone Alkaloids</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oberhofer</span><span class="given-names" tagx="given-names" title="given-names">Martina</span></span><a href="#cor1">*</a><a href="#aff1">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wackerlig</span><span class="given-names" tagx="given-names" title="given-names">Judith</span></span><a href="#aff2">â€¡</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zehl</span><span class="given-names" tagx="given-names" title="given-names">Martin</span></span><a href="#aff3">Â§</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ¼yÃ¼k</span><span class="given-names" tagx="given-names" title="given-names">Havva</span></span><a href="#aff2">â€¡</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Jia Jian</span></span><a href="#aff2">â€¡</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prado-Roller</span><span class="given-names" tagx="given-names" title="given-names">Alexander</span></span><a href="#aff4">âˆ¥</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Urban</span><span class="given-names" tagx="given-names" title="given-names">Ernst</span></span><a href="#aff2">â€¡</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zotchev</span><span class="given-names" tagx="given-names" title="given-names">Sergey B.</span></span><a href="#aff1">â€ </a></span><span class="citation_author_institution" id="aff1">[â€ ], <span class="institution" title="institution">University of Vienna</span><span class="country" tagx="country" title="country">Austria</span></span><span class="citation_author_institution" id="aff2">[â€¡], <span class="institution" title="institution">University of Vienna</span><span class="country" tagx="country" title="country">Austria</span></span><span class="citation_author_institution" id="aff3">[Â§], <span class="institution" title="institution">University of Vienna</span><span class="country" tagx="country" title="country">Austria</span></span><span class="citation_author_institution" id="aff4">[âˆ¥], <span class="institution" title="institution">University of Vienna</span><span class="country" tagx="country" title="country">Austria</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Email: <span class="email" tagx="email" title="email">martina.oberhofer@univie.ac.at</span>. Phone: <span class="phone" tagx="phone" title="phone">+43-1-4277-55294</span>.</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-01-01</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021-01-01</span></span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">2184</span><span class="lpage" tagx="lpage" title="lpage">2191</span><span class="history" title="history"><span class="received" title="received">received: 2020-11-09</span><span class="accepted" title="accepted">accepted: 2020-12-07</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">The Authors</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2021</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article published under a Creative Commons Non-Commercial No Derivative Works (CC-BY-NC-ND) Attribution <a href="http://pubs.acs.org/page/policy/authorchoice_ccbyncnd_termsofuse.html">License</a>, which permits copying and redistribution of the article, and creation of adaptations, all for non-commercial purposes.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p content-type="toc-graphic"> </p><div class="graphic" id="ab-tgr1" /> <p /> <p>In the search for new antibiotics, several fungal endophytes were isolated from the medicinal plant <i>Leontopodium nivale</i> subsp. <i>alpinum</i> (Edelweiss). The extract from one of these fungi classified as <i>Akanthomyces</i> sp. displayed broad-spectrum antibiotic activity against gram-negative bacteria and fungi. Further investigation into the composition of this extract using bioactivity-guided fractionation, HRMS, and nuclear magnetic resonance revealed two new 4-hydroxy-2-pyridone alkaloids (<b>1</b>, <b>2</b>) and emestrin (<b>3</b>), an epidithiodioxopiperazine not previously known to be produced by a member of Cordycipitaceae. Further testing of purified compounds <b>1</b> and <b>2</b> proved that they are devoid of antibiotic activity, and all the activities observed in the crude extract could be assigned to emestrin (<b>3</b>), whose configuration was confirmed by crystallographic data. This study demonstrates, for the first time, that endophytic fungi from Edelweiss can produce new compounds, prompting further investigation into them for drug discovery.</p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="document-id-old-9" value="ao0c05472" /> <meta name="document-id-new-14" value="ao0c05472" /> <meta name="ccc-price" value="" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>New antibiotics are urgently needed to combat infectious diseases caused by multiresistant pathogens. Discovery rates of antibiotic compounds have dramatically dropped in the past decades. High costs of drug discovery and development along with frequent rediscovery of known compounds and the rapid emergence of new resistances among pathogens<sup><a href="#ref1">1</a></sup> have a strong negative impact on the number of new molecules entering clinical trials.<sup><a href="#ref2">2</a></sup> Considering this, innovative approaches to isolate underexplored microorganisms and test their ability to produce antibiotically active compounds emerged. One example is endophytes, microorganisms causing no harm to their plant host while inhabiting their tissues. Endophytic fungi, and particularly those of the phylum <i>Ascomycota</i>, were demonstrated to be prolific producers of antibiotic compounds.<sup><a href="#ref3">3</a></sup> Endophytic fungi are also known to biosynthesize the same or similar secondary metabolites previously identified in their host plant<sup><a href="#ref4">4</a></sup> and thus provide the potential for biotechnological production of such compounds using microorganisms instead of relying on the plant material.<sup><a href="#ref5">5</a>,<a href="#ref6">6</a></sup></p> <p>Edelweiss (<i>Leontopodium nivale</i> subsp. <i>alpinum</i>) is a traditional medicinal plant in Central Europe, which was widely applied in human and veterinary medicine before the species was overexploited and became endangered. All parts of the plant were used to treat gastrointestinal, pulmonary, and heart conditions, most likely because of its antiinflammatory and antimicrobial properties.<sup><a href="#ref7">7</a></sup> Besides the traditional use of the <i>L. nivale</i> subsp. <i>alpinum</i>, only one study investigated its antibacterial properties in vitro, while in vivo studies and clinical tests are not available.<sup><a href="#ref7">7</a>,<a href="#ref8">8</a></sup></p> <p>Here, we present the investigation of the antibiotically active extract from an entomopathogenic fungal endophyte <i>Akanthomyces</i> sp. (<i>Ascomycota</i>, Cordycipitaceae) isolated from <i>L. nivale</i> subsp. <i>alpinum</i> affording two novel 4-hydroxy-2-pyridone alkaloids and the known epidithiodioxopiperazine emestrin (<a href="#cht1">Chart </a><a href="#cht1">1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Chart 1</span></a></div>   <div class="title" tagx="title" title="title">
Chemical Structures of the Novel Compounds Akanthenin A (<b>1</b>) and Akanthenin B (<b>2</b>) and the Structurally Related Compound YCM1008A along with the known Compound Emestrin (<b>3</b>).</div>   </div> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <p>Eleven fungal strains with similar morphology were isolated from three <i>L. nivale</i> subsp. <i>alpinum</i> plants (<a href="#fig1">Figures </a><a href="#fig1">1</a>, <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">S1</a>). Genomic DNA was isolated from pure fungal cultures, and the large ribosomal subunit region (LSU) was sequenced and referenced to the GenBank database using the Basic Logical Alignment Search Tool (BLASTn). Phylogenetic analysis including closely related reference strains from the family Cordycipitaceae identified the fungal isolates as <i>Akanthomyces</i> spp.<sup><a href="#ref9">9</a></sup> (Figure S1 in the <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Supporting Information</a>). Based on this phylogeny, no resolution was possible between the two species <i>Lecanicillium attenuatum</i> and <i>Akanthomyces muscarius</i>, where <i>Lecanicillium</i> is an outdated synonym name for <i>Akanthomyces</i> sp.<sup><a href="#ref10">10</a></sup><i>Akanthomyces</i> spp. are known as entomopathogenic fungi which symbiotically defend their host plant against insect herbivores. <i>Akanthomyces</i> spp. can colonize insects rapidly and form fruiting bodies (stromata) with sexual spores.<sup><a href="#ref11">11</a></sup> Entomopathogenic fungi known to occur as endophytes, are prolific producers of secondary metabolites<sup><a href="#ref9">9</a>,<a href="#ref12">12</a></sup> and some may be consumed as medicinal fungi.<sup><a href="#ref13">13</a></sup> The frequent occurrence of <i>Akanthomyces</i> sp. within the studied plant specimens, along with an interesting antibiotic profile of the total extract against <i>Escherichia coli</i>, <i>Saccharomyces cerevisiae,</i> and filamentous fungi advocated its choice for further investigation.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Microscopic and macroscopic morphology of <i>Akanthomyces</i> sp. LN303. Fungal mycelia with conidia and conidiophores (a). Six day old colony of <i>Akanthomyces</i> sp. LN303 on potato dextrose agar (b).</p>   </div> <p>All analyses were performed on isolate LN303 and finally backed up by results from <i>Akanthomyces</i> spp. originating from two other plant individuals (LN276, LN528). Additionally, isolates LN401 and LN427 were screened for compounds <b>1â€"3</b> using liquid chromatographyâ€"mass spectrometry (LCâ€"MS) analyses. Emestrin was produced by all <i>Akanthomyces</i> isolates in different concentrations; however, akanthenin A and B remained exclusive to LN303. After fermentation on rice media, dichloromethane:methanol (2:1) extracts yielded 2.593 g of dry matter for LN303 (from 20 flasks), 0.412 g for LN276 (from 5 flasks), 0.229 g for LN528, 0.363 g for LN401, and 0.252 g for LN427 (from 1 flask each). Antibiotic bioassays with LN303 total extract yielded an interesting broad-spectrum antibiotic profile against all fungal test organismsâ€"<i>S. cerevisiae</i>, <i>Fusarium graminearum</i> ,and <i>Aspergillus niger</i>â€"and against the Gram-negative bacterium <i>E. coli</i>. No activity was detected against Gram-positive <i>Bacillus subtilis</i>. The total extract was subjected to analytical and semipreparative reversed-phase high pressure LC (HPLC) using a C-18 column and antibiotic bioassays. Initial fractionation yielded seven fractions, of which fraction 4 was antibiotically active (<a href="#tbl1">Table </a><a href="#tbl1">1</a>). Subfractionation of fraction 4 yielded several pure compounds, which were analyzed by LCâ€"MS and nuclear magnetic resonance spectroscopy (NMR).</p> <div class="table-wrap_UNKNOWN" id="tbl1" position="float"><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Antibiotic Activity of Total Extract, Fraction 4, and Pure Compounds <b>1â€"3</b> Measured as a Diameter of Inhibition Zones in the Disc Diffusion Assays</div>  <table frame="hsides" rules="groups" border="0"> <colgroup> <col align="left" /></colgroup> <colgroup><col align="char" char="." /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="char" char="." /></colgroup> <colgroup><col align="char" char="." /></colgroup>  <thead> <tr> <th style="border:none;" align="center">Â </th> <th colspan="7" align="center" char=".">inhibition zones (mm) <hr /> </th> </tr> <tr> <th style="border:none;" align="center">test organism</th> <th style="border:none;" align="center" char=".">total extract LN303 1.89Â mg</th> <th style="border:none;" align="center">fractionÂ 4 12.3Â Î¼g</th> <th style="border:none;" align="center">akantheninÂ A (<b>1</b>) 100Â Î¼g</th> <th style="border:none;" align="center">akantheninÂ B (<b>2</b>) 100Â Î¼g</th> <th style="border:none;" align="center">emestrinÂ A (<b>3</b>) 55Â Î¼g</th> <th style="border:none;" align="center" char=".">PC<a href="#t1fn2">b</a> tetracyclin 10Â mg<a href="#t1fn3">c</a>/Nystatin 500Â Î¼g<a href="#t1fn4">d</a></th> <th style="border:none;" align="center" char=".">NC<a href="#t1fn5">e</a> methanol 50Â Î¼L</th> </tr> </thead> <tbody> <tr> <td style="border:none;" align="left"><i>B. subtilis</i></td> <td style="border:none;" align="char" char=".">0</td> <td style="border:none;" align="left">n.a.</td> <td style="border:none;" align="left">0</td> <td style="border:none;" align="left">0</td> <td style="border:none;" align="left">0</td> <td style="border:none;" align="char" char=".">19</td> <td style="border:none;" align="char" char=".">0</td> </tr> <tr> <td style="border:none;" align="left"><i>E. coli</i></td> <td style="border:none;" align="char" char=".">9</td> <td style="border:none;" align="left">9</td> <td style="border:none;" align="left">n.a.</td> <td style="border:none;" align="left">n.a.</td> <td style="border:none;" align="left">11.3</td> <td style="border:none;" align="char" char=".">18.5</td> <td style="border:none;" align="char" char=".">0</td> </tr> <tr> <td style="border:none;" align="left"><i>S. cerevisiae</i></td> <td style="border:none;" align="char" char=".">7</td> <td style="border:none;" align="left">7</td> <td style="border:none;" align="left">0</td> <td style="border:none;" align="left">0</td> <td style="border:none;" align="left">13</td> <td style="border:none;" align="char" char=".">30</td> <td style="border:none;" align="char" char=".">0</td> </tr> <tr> <td style="border:none;" align="left"><i>F. graminearum</i></td> <td style="border:none;" align="char" char=".">25</td> <td style="border:none;" align="left">15</td> <td style="border:none;" align="left">n.a.</td> <td style="border:none;" align="left">n.a.</td> <td style="border:none;" align="left">15/21.5<a href="#t1fn1">a</a></td> <td style="border:none;" align="char" char=".">30.5</td> <td style="border:none;" align="char" char=".">0</td> </tr> <tr> <td style="border:none;" align="left"><i>A. niger</i></td> <td style="border:none;" align="char" char=".">10</td> <td style="border:none;" align="left">11</td> <td style="border:none;" align="left">n.a.</td> <td style="border:none;" align="left">n.a.</td> <td style="border:none;" align="left">11.5</td> <td style="border:none;" align="char" char=".">25</td> <td style="border:none;" align="char" char=".">0</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">a</span><p>Complete/partial inhibition.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">b</span><p>Positive control.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">c</span><p>Antibacterial.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">d</span><p>Antifungal.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">e</span><p>Negative control.</p> </div> </div> </div> <p>Investigation of the active fraction 4 afforded two novel 4-hydroxy-2-pyridone alkaloid compounds (<b>1</b>, <b>2</b>) along with emestrin (<b>3</b>). Emestrin was identified as the compound partly or fully responsible for all antibiotic activities, whereas the two pyridone alkaloids were devoid of any antibiotic activity, despite their abundance in the active extract (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S14</a>).</p> <p>LCâ€"MS analysis of <b>1</b> yielded a molecular formula of C<sub>22</sub>H<sub>31</sub>NO<sub>5</sub>, obtained by high-resolution electrospray ionization MS (HRESIMS) from the [M + H]<sup>+</sup> ion at <i>m</i>/<i>z</i> 390.2270 (calcd for C<sub>22</sub>H<sub>32</sub>NO<sub>5</sub><sup>+</sup>, <i>m</i>/<i>z</i> 390.2275, Î" = 1.2 ppm) and from the [M + Na]<sup>+</sup> ion at <i>m</i>/<i>z</i> 412.2089 (calcd for C<sub>22</sub>H<sub>31</sub>NO<sub>5</sub>Na<sup>+</sup>, <i>m</i>/<i>z</i> 412.2094, Î" = 1.3 ppm) (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figures S2â€"S4</a>). The [M + H]<sup>+</sup> ion showed strong source decay via sequential loss of H<sub>2</sub>O and CH<sub>2</sub>O, indicating a labile methoxy group attached to a nonaliphatic substructure (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S4</a>). The fragmentation pattern shows a stable substructure containing most of the heteroatoms (C<sub>7</sub>H<sub>7</sub>NO<sub>4</sub>) and did not fit to any of the 28 compounds with the above sum formula currently listed in the Dictionary of Natural Products (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S5, Schemes S1 and S2</a>).<sup><a href="#ref14">14</a></sup></p> <p>The <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of <b>1</b> displayed signals of a pyridone core and an unsaturated and branched side chain. In addition, a very deep field-shifted methoxy signal (<sup>1</sup>H at 4.07 ppm and <sup>13</sup>C at 65.05 ppm) was detected, which could be interpreted as a methyl ester of a hydroxamic acid moiety. The structure of the side chain was assigned by 2D NMR methods (correlation spectroscopy COSY, heteronuclear single quantum coherence HSQC, and heteronuclear multiple bond correlation HMBC). Thus, the position of the unsaturated and branched side chain at carbon 2 was proved by correlation signals from carbon 2 to protons at 4, 9, 1â€², and 3â€² in HMBC. In addition, the proton at 2 gave correlation signals to carbons 8a, 9, 1â€², and 3â€² in HMBC. Comparison with the literature data of the known <i>N</i>-methoxy-pyridone derivative YCM1008A proved the structure to be the same with the exception of an additional methyl and hydroxy group in the side chain<sup><a href="#ref15">15</a></sup> (<a href="#tbl2">Table </a><a href="#tbl2">2</a>) (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S16</a>). The position of the additional methyl group at carbon 6â€² was assigned due to the correlation signals of the protons at 5â€² and 7â€² with the carbon 11â€² in HMBC. The position of the hydroxy function was proved by the coupling signals of protons at carbon 8â€² (Î´<sub>H</sub> 2.61 ppm) and 12â€² (Î´<sub>H</sub> 3.60 and 3.43 ppm) in COSY. The configuration of the chiral centers could not be established by NMR methods.</p> <div class="table-wrap_UNKNOWN" id="tbl2" position="float"><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title"><sup>1</sup>H and <sup>13</sup>C NMR Data of YCM1008A<a href="#t2fn1">a</a> (Î´ in ppm, Methanol-<i>d</i><sub>4</sub>) in Comparison with 1 and 2 (Î´ in ppm, CDCl<sub>3</sub>)</div>  <table frame="hsides" rules="groups" border="0"> <colgroup> <col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup>  <thead> <tr> <th style="border:none;" align="center">Â </th> <th colspan="2" align="center">YCM1008A<a href="#t2fn1">a</a><hr /> </th> <th colspan="2" align="center">akantheninÂ A (<b>1</b>) <hr /> </th> <th colspan="2" align="center">akantheninÂ B (<b>2</b>) <hr /> </th> </tr> <tr> <th style="border:none;" align="center">position</th> <th style="border:none;" align="center">Î´<sub>H</sub> (<i>J</i>Â inÂ Hz)</th> <th style="border:none;" align="center">Î´<sub>C</sub>, type</th> <th style="border:none;" align="center">Î´<sub>H</sub> (<i>J</i>Â inÂ Hz)</th> <th style="border:none;" align="center">Î´<sub>C</sub>, type</th> <th style="border:none;" align="center">Î´<sub>H</sub> (<i>J</i>Â inÂ Hz)</th> <th style="border:none;" align="center">Î´<sub>C</sub>, type</th> </tr> </thead> <tbody> <tr> <td style="border:none;" align="left">2</td> <td style="border:none;" align="left">4.39Â d (11.3)</td> <td style="border:none;" align="left">84.9, CH</td> <td style="border:none;" align="left">4.58Â m</td> <td style="border:none;" align="left">78.14, CH</td> <td style="border:none;" align="left">4.59Â m</td> <td style="border:none;" align="left">78.03, CH</td> </tr> <tr> <td style="border:none;" align="left">3</td> <td style="border:none;" align="left">1.92Â m</td> <td style="border:none;" align="left">35.7, CH</td> <td style="border:none;" align="left">2.28Â m</td> <td style="border:none;" align="left">35.13, CH</td> <td style="border:none;" align="left">2.28Â m</td> <td style="border:none;" align="left">35.06, CH</td> </tr> <tr> <td style="border:none;" align="left">4</td> <td style="border:none;" align="left">4.66Â d (3.0)Â m</td> <td style="border:none;" align="left">63.0, CH</td> <td style="border:none;" align="left">4.61Â m</td> <td style="border:none;" align="left">65.60, CH</td> <td style="border:none;" align="left">4.60Â m</td> <td style="border:none;" align="left">65.52, CH</td> </tr> <tr> <td style="border:none;" align="left">4a</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">112.5,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">110.87,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">110.84,Â C</td> </tr> <tr> <td style="border:none;" align="left">5</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">160.8,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">161.14,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">159.82,Â C</td> </tr> <tr> <td style="border:none;" align="left">7</td> <td style="border:none;" align="left">7.22Â d (7.9)</td> <td style="border:none;" align="left">136.4, CH</td> <td style="border:none;" align="left">7.41Â d (7.9)</td> <td style="border:none;" align="left">134.23, CH</td> <td style="border:none;" align="left">7.41Â d (7.9)</td> <td style="border:none;" align="left">134.22, CH</td> </tr> <tr> <td style="border:none;" align="left">8</td> <td style="border:none;" align="left">5.98Â d (7.9)</td> <td style="border:none;" align="left">101.5, CH</td> <td style="border:none;" align="left">5.97Â d (7.9)</td> <td style="border:none;" align="left">99.79, CH</td> <td style="border:none;" align="left">5.97Â d (7.9)</td> <td style="border:none;" align="left">99.72, CH</td> </tr> <tr> <td style="border:none;" align="left">8a</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">163.8,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">159.82,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">161.28,Â C</td> </tr> <tr> <td style="border:none;" align="left">9</td> <td style="border:none;" align="left">0.93Â d (6.9)</td> <td style="border:none;" align="left">12.9, CH<sub>3</sub></td> <td style="border:none;" align="left">0.68Â d (7.4)</td> <td style="border:none;" align="left">9.64, CH<sub>3</sub></td> <td style="border:none;" align="left">0.67Â d (7.3)</td> <td style="border:none;" align="left">9.62, CH<sub>3</sub></td> </tr> <tr> <td style="border:none;" align="left">10</td> <td style="border:none;" align="left">3.99Â s</td> <td style="border:none;" align="left">65.5, CH<sub>3</sub></td> <td style="border:none;" align="left">4.07Â s</td> <td style="border:none;" align="left">65.05, CH<sub>3</sub></td> <td style="border:none;" align="left">4.07Â s</td> <td style="border:none;" align="left">65.05, CH<sub>3</sub></td> </tr> <tr> <td style="border:none;" align="left">1â€²</td> <td style="border:none;" align="left">1.76Â d (1.1)</td> <td style="border:none;" align="left">11.4, CH<sub>3</sub></td> <td style="border:none;" align="left">1.85Â s</td> <td style="border:none;" align="left">14.38, CH<sub>3</sub></td> <td style="border:none;" align="left">1.84Â s</td> <td style="border:none;" align="left">14.39, CH<sub>3</sub></td> </tr> <tr> <td style="border:none;" align="left">2â€²</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">133.6,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">132.50,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">132.95,Â C</td> </tr> <tr> <td style="border:none;" align="left">3â€²</td> <td style="border:none;" align="left">6.15Â d (11.0)</td> <td style="border:none;" align="left">132.4, CH</td> <td style="border:none;" align="left">6.32Â d (10.8)</td> <td style="border:none;" align="left">125.41, CH</td> <td style="border:none;" align="left">6.32Â d (11.2)</td> <td style="border:none;" align="left">125.26, CH</td> </tr> <tr> <td style="border:none;" align="left">4â€²</td> <td style="border:none;" align="left">6.41 dd (14.8, 11.0)</td> <td style="border:none;" align="left">126.8, CH</td> <td style="border:none;" align="left">6.47 dd (15.1, 10.8)</td> <td style="border:none;" align="left">122.44, CH</td> <td style="border:none;" align="left">6.50 dd (15.3, 11.2)</td> <td style="border:none;" align="left">123.35, CH</td> </tr> <tr> <td style="border:none;" align="left">5â€²</td> <td style="border:none;" align="left">6.28 dd (14.8, 10.8)</td> <td style="border:none;" align="left">136.3, CH</td> <td style="border:none;" align="left">6.36Â d (15.1)</td> <td style="border:none;" align="left">137.76, CH</td> <td style="border:none;" align="left">6.37Â d (15.3)</td> <td style="border:none;" align="left">137.14, CH</td> </tr> <tr> <td style="border:none;" align="left">6â€²</td> <td style="border:none;" align="left">6.28 dd (14.9, 10.8)</td> <td style="border:none;" align="left">130.4, CH</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">137.19,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">137.03,Â C</td> </tr> <tr> <td style="border:none;" align="left">7â€²</td> <td style="border:none;" align="left">6.28 dd (14.9, 8.0)</td> <td style="border:none;" align="left">142.7, CH</td> <td style="border:none;" align="left">5.26Â d (9.9)</td> <td style="border:none;" align="left">134.36, CH</td> <td style="border:none;" align="left">5.49Â d (9.6)</td> <td style="border:none;" align="left">129.26, CH</td> </tr> <tr> <td style="border:none;" align="left">8â€²</td> <td style="border:none;" align="left">2.10Â m</td> <td style="border:none;" align="left">40.0, CH</td> <td style="border:none;" align="left">2.61Â m</td> <td style="border:none;" align="left">43.25, CH</td> <td style="border:none;" align="left">3.36Â m</td> <td style="border:none;" align="left">45.83, CH</td> </tr> <tr> <td style="border:none;" align="left">9â€²</td> <td style="border:none;" align="left">1.34Â m</td> <td style="border:none;" align="left">30.8, CH<sub>2</sub></td> <td style="border:none;" align="left">1.52Â m</td> <td style="border:none;" align="left">24.67, CH<sub>2</sub></td> <td style="border:none;" align="left">1.86Â m</td> <td style="border:none;" align="left">26.10, CH<sub>2</sub></td> </tr> <tr> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">1.23Â m</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">1.62Â m</td> <td style="border:none;" align="left">Â </td> </tr> <tr> <td style="border:none;" align="left">10â€²</td> <td style="border:none;" align="left">0.87Â t (7.4)</td> <td style="border:none;" align="left">12.1, CH<sub>3</sub></td> <td style="border:none;" align="left">0.88Â t (7.4)</td> <td style="border:none;" align="left">11.69, CH<sub>3</sub></td> <td style="border:none;" align="left">0.94Â t (7.5)</td> <td style="border:none;" align="left">11.61, CH<sub>3</sub></td> </tr> <tr> <td style="border:none;" align="left">11â€²</td> <td style="border:none;" align="left">1.01Â d (6.7)</td> <td style="border:none;" align="left">20.5, CH<sub>3</sub></td> <td style="border:none;" align="left">1.89Â s</td> <td style="border:none;" align="left">13.22, CH<sub>3</sub></td> <td style="border:none;" align="left">1.89Â s</td> <td style="border:none;" align="left">12.96, CH<sub>3</sub></td> </tr> <tr> <td style="border:none;" align="left">12â€²</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">3.60 dd (10.5,Â 5.6)</td> <td style="border:none;" align="left">66.42, CH<sub>2</sub></td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">175.85,Â C</td> </tr> <tr> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">3.43 dd (10.5,Â 8.1)</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">Â </td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">a</span><p>Reference (<a href="#ref15">15</a>).</p> </div> </div> </div> <p>Compound <b>2</b> was assigned a molecular formula of C<sub>22</sub>H<sub>29</sub>NO<sub>6</sub> based on the HRESIMS data for the [M + H]<sup>+</sup> ion at <i>m</i>/<i>z</i> 404.2064 (calcd for C<sub>22</sub>H<sub>30</sub>NO<sub>6</sub><sup>+</sup>, <i>m</i>/<i>z</i> 404.2068, Î" = 0.8 ppm) and the [M + Na]<sup>+</sup> ion at <i>m</i>/<i>z</i> 426.1884 (calcd for C<sub>22</sub>H<sub>29</sub>NO<sub>6</sub>Na<sup>+</sup>, <i>m</i>/<i>z</i> 426.1887, Î" = 0.8 ppm). Tandem MS (MS/MS) data of the [M + Na]<sup>+</sup> ion showed high similarity to those of compound <b>1</b>, and the same stable C<sub>7</sub>H<sub>7</sub>NO<sub>4</sub>Na<sup>+</sup> fragment ion at <i>m</i>/<i>z</i> 192.0269, containing the <i>N</i>-methoxy-pyridone ring, was observed. In contrast to <b>1</b>, <b>2</b> was detected in negative ion mode, where the [M â€" H]<sup>âˆ’</sup> ion underwent complete in-source fragmentation, mainly because of the loss of CO<sub>2</sub>. Further fragmentation of the [Mâ€"CO<sub>2</sub>â€"H]<sup>âˆ’</sup> ion and several indicative fragment ions of the [M + Na]<sup>+</sup> ion pointed toward the presence of a carboxylic acid group in the side chain (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figures S6â€"S8, Schemes S3â€"S5</a>).</p> <p>The <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of <b>2</b> displayed signals of a pyridinone core and an unsaturated and branched side chain equal to that of <b>1</b>, but the position 12â€² was oxidized to a carboxylic group (Î´<sub>C</sub> 175.85 ppm) (<a href="#tbl2">Table </a><a href="#tbl2">2</a>) (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S17</a>).</p> <p>Compounds <b>1</b> and <b>2</b> are 4-oxy-3-alkoxy-2-pyridones belonging to the 4-hydroxy-2-pyridone alkaloids, which constitute an intensely researched class of natural products recognized for their diverse bioactivities. These compounds can be produced by plants, bacteria, and fungi and have been classified into four groups depending on the substituents in the 3-position of the characteristic pyridine ring.<sup><a href="#ref17">17</a></sup> Bioactivities of these compounds cover a wide range from cytotoxic and antibiotic<sup><a href="#ref17">17</a></sup> to insecticidal properties, but they can also have very specific effects, such as promoting neurite outgrowth formation.<sup><a href="#ref18">18</a></sup></p> <p>Compounds <b>1</b> and <b>2</b> were isolated from LN303, yielding 4.95 mg of compound <b>1</b> (purity 94%, estimated by analytical HPLC) and 0.83 mg of compound <b>2</b> (&amp;gt;90%). According to the LCâ€"MS analysis of the total extracts, both compounds were only detected in isolate LN303 (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S14</a>) and were absent in all other investigated fungal extracts under the given growing conditions. We propose the names akanthenin A (<b>1</b>) and B (<b>2</b>) for these new compounds. Akanthenin A and B were most similar to the described compound YCM1008A, which can suppress Ca<sup>2+</sup>-induced growth inhibition in <i>S. cerevisiae</i>.<sup><a href="#ref15">15</a>,<a href="#ref16">16</a></sup> Calcineurin inhibitors like, for example, cyclosporin, are valuable immunosuppressants used in the treatment of autoimmune diseases and organ transplants.<sup><a href="#ref19">19</a></sup> Akanthomycin is another pyridone alkaloid that acts as an antibiotic and was described previously as being produced by <i>Akanthomyces gracilis</i>;<sup><a href="#ref20">20</a></sup> however, none of our total extracts contained this compound in detectable concentrations.</p> <p>Compound <b>3</b> was preferentially detected as [M â€" H]<sup>âˆ’</sup> ions, from which a molecular formula of C<sub>27</sub>H<sub>22</sub>N<sub>2</sub>O<sub>10</sub>S<sub>2</sub> was deduced (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S9</a>). The only known natural product matching this accurate mass and sum formula is emestrin, alternatively named emestrin A, a mycotoxin produced by several unrelated fungi such as multiple <i>Emericella</i> species, <i>Armillaria tabescens</i>, and <i>Podospora australis</i>.<sup><a href="#ref21">21</a>âˆ’<a href="#ref26">26</a></sup> Structure elucidation of <b>3</b> by 1D NMR indicated a polycyclic ring system consisting of a sulfur-bridged diketopiperazine moiety, an unsaturated bicyclic ring system, and two trisubstituted benzene substructures. Further analysis by 2D NMR methods (COSY, HSQC, and HMBC) proved that the structure of <b>3</b> was identical to that of emestrin (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S18</a>). This compound belongs to the class of epidithiodiketopiperacines<sup><a href="#ref24">24</a></sup> (<a href="#fig1">Figure </a><a href="#fig1">1</a>), a well-studied group of alkaloids exhibiting potent bioactivities.<sup><a href="#ref26">26</a></sup> Published NMR data of emestrin were obtained either in deuterated dimethyl sulfoxide<sup><a href="#ref24">24</a>,<a href="#ref25">25</a></sup> or CDCl<sub>3</sub>.<sup><a href="#ref22">22</a></sup> We recorded NMR data in the latter solvent because it can be readily evaporated, and the compound can be used in bioassays (purity &amp;gt; 95%). The obtained <sup>1</sup>H and <sup>13</sup>C NMR data of <b>3</b> were in agreement with the data reported by Nursid et al.<sup><a href="#ref22">22</a></sup> within the expected experimental error (<a href="#tbl3">Table </a><a href="#tbl3">3</a>). To verify the absolute configuration of compound <b>3</b> and its identity to the published structure of emestrin, we also performed X-ray crystallographic structure elucidation. A single crystal X-ray analysis (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S15</a>) confirmed that compound <b>3</b> is indeed emestrin, as described by Seya et al.<sup><a href="#ref24">24</a>,<a href="#ref27">27</a></sup></p> <div class="table-wrap_UNKNOWN" id="tbl3" position="float"><span class="label" tagx="label" title="label">Table 3</span> <div class="title" tagx="title" title="title"><sup>1</sup>H and <sup>13</sup>C NMR Data of Emestrin (<b>3</b>) in Comparison with Literature Data (Î´ in ppm, CDCl<sub>3</sub>)</div>  <table frame="hsides" rules="groups" border="0"> <colgroup> <col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup>  <thead> <tr> <th style="border:none;" align="center">position</th> <th style="border:none;" align="center">Î´<sub>H</sub> (<i>J</i>Â inÂ Hz)</th> <th style="border:none;" align="center">Î´<sub>C</sub>, type</th> <th style="border:none;" align="center">Î´<sub>H</sub></th> <th style="border:none;" align="center">Î´<sub>C</sub></th> </tr> </thead> <tbody> <tr> <td style="border:none;" align="center">Â </td> <td colspan="2" align="center">emestrin (<b>3</b>)</td> <td colspan="2" align="center">emestrin<a href="#t3fn1">a</a></td> </tr> <tr> <td style="border:none;" align="left">1</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">166.97,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">167.1</td> </tr> <tr> <td style="border:none;" align="left">3</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">78.29,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">78.5</td> </tr> <tr> <td style="border:none;" align="left">4</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">163.71,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">163.8</td> </tr> <tr> <td style="border:none;" align="left">5a</td> <td style="border:none;" align="left">5.69 dd (8.0, 2.0)</td> <td style="border:none;" align="left">60.76, CH</td> <td style="border:none;" align="left">5.71</td> <td style="border:none;" align="left">60.9</td> </tr> <tr> <td style="border:none;" align="left">6</td> <td style="border:none;" align="left">4.89 ddd (8.0, 2.4, 2.2)</td> <td style="border:none;" align="left">74.98, CH</td> <td style="border:none;" align="left">4.88</td> <td style="border:none;" align="left">75.2</td> </tr> <tr> <td style="border:none;" align="left">7</td> <td style="border:none;" align="left">4.94 dd (8.3, 2.2)</td> <td style="border:none;" align="left">107.54, CH</td> <td style="border:none;" align="left">4.93</td> <td style="border:none;" align="left">107.7</td> </tr> <tr> <td style="border:none;" align="left">8</td> <td style="border:none;" align="left">6.34 dd (8.3, 2.4)</td> <td style="border:none;" align="left">138.51, CH</td> <td style="border:none;" align="left">6.32</td> <td style="border:none;" align="left">138.6</td> </tr> <tr> <td style="border:none;" align="left">10</td> <td style="border:none;" align="left">6.91Â d (2.0)</td> <td style="border:none;" align="left">143.06, CH</td> <td style="border:none;" align="left">6.91</td> <td style="border:none;" align="left">143.3</td> </tr> <tr> <td style="border:none;" align="left">10a</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">112.38,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">112.5</td> </tr> <tr> <td style="border:none;" align="left">11</td> <td style="border:none;" align="left">4.99 (s)</td> <td style="border:none;" align="left">76.67, CH</td> <td style="border:none;" align="left">4.98</td> <td style="border:none;" align="left">76.8</td> </tr> <tr> <td style="border:none;" align="left">11a</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">72.81,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">73.1</td> </tr> <tr> <td style="border:none;" align="left">2-NCH<sub>3</sub></td> <td style="border:none;" align="left">3.42 (s)</td> <td style="border:none;" align="left">27.84, CH<sub>3</sub></td> <td style="border:none;" align="left">3.40</td> <td style="border:none;" align="left">28</td> </tr> <tr> <td style="border:none;" align="left">1â€²</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">126.57,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">126.7</td> </tr> <tr> <td style="border:none;" align="left">2â€²</td> <td style="border:none;" align="left">7.90Â d (2.2)</td> <td style="border:none;" align="left">121.09, CH</td> <td style="border:none;" align="left">7.88</td> <td style="border:none;" align="left">121.1</td> </tr> <tr> <td style="border:none;" align="left">3â€²</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">145.24,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">145.2</td> </tr> <tr> <td style="border:none;" align="left">4â€²</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">148.83,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">148.9</td> </tr> <tr> <td style="border:none;" align="left">5â€²</td> <td style="border:none;" align="left">6.95Â d (8.3)</td> <td style="border:none;" align="left">115.03, CH</td> <td style="border:none;" align="left">6.92</td> <td style="border:none;" align="left">115.3</td> </tr> <tr> <td style="border:none;" align="left">6â€²</td> <td style="border:none;" align="left">7.08 dd (8.3, 2.2)</td> <td style="border:none;" align="left">125.50, CH</td> <td style="border:none;" align="left">7.07</td> <td style="border:none;" align="left">125.6</td> </tr> <tr> <td style="border:none;" align="left">7â€²</td> <td style="border:none;" align="left">5.42 (s)</td> <td style="border:none;" align="left">74.69, CH</td> <td style="border:none;" align="left">5.41</td> <td style="border:none;" align="left">74.9</td> </tr> <tr> <td style="border:none;" align="left">1â€³</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">122.54,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">122.7</td> </tr> <tr> <td style="border:none;" align="left">2â€³</td> <td style="border:none;" align="left">7.77Â d (2.0)</td> <td style="border:none;" align="left">121.99, CH</td> <td style="border:none;" align="left">7.75</td> <td style="border:none;" align="left">122.2</td> </tr> <tr> <td style="border:none;" align="left">3â€³</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">145.24,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">145.3</td> </tr> <tr> <td style="border:none;" align="left">4â€³</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">154.17,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">154.3</td> </tr> <tr> <td style="border:none;" align="left">5â€³</td> <td style="border:none;" align="left">7.06Â d (8.5)</td> <td style="border:none;" align="left">112.17, CH</td> <td style="border:none;" align="left">7.04</td> <td style="border:none;" align="left">112.3</td> </tr> <tr> <td style="border:none;" align="left">6â€³</td> <td style="border:none;" align="left">7.79 dd (8.5, 2.0)</td> <td style="border:none;" align="left">126.46, CH</td> <td style="border:none;" align="left">7.79</td> <td style="border:none;" align="left">126.5</td> </tr> <tr> <td style="border:none;" align="left">7â€³</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">165.14,Â C</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">165.3</td> </tr> <tr> <td style="border:none;" align="left">4â€³-OCH<sub>3</sub></td> <td style="border:none;" align="left">4.03 (s)</td> <td style="border:none;" align="left">56.28, CH<sub>3</sub></td> <td style="border:none;" align="left">4.0</td> <td style="border:none;" align="left">56.5</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">a</span><p>Reference (<a href="#ref22">22</a>).</p> </div> </div> </div> <p>LCâ€"MS analyses of the total extracts of the five studied <i>Akanthomyces</i> sp. strains demonstrated that emestrin is present as an abundant compound in all five isolates. Interestingly, an isomer was detected that eluted at minute 20.0 from the C-18 HPLC column shortly before emestrin did, which eluted after 20.2â€"20.4 min. The relative abundance of this isomer compared to emestrin decreased significantly in the order LN276 &amp;gt; LN303 &amp;gt; LN528 âˆ¼ LN427 &amp;gt; LN401 (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figures S10â€"S12</a>). Despite the identical sum formula and very similar polarity, the fragmentation patterns differed significantly between the two isomers (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S13, Scheme S6</a>). Unfortunately, attempts to isolate this isomer did not yield sufficient amounts for structure elucidation.</p> <p>Compounds <b>1</b>, <b>2,</b> and <b>3</b> were evaluated by disc diffusion assays using 100 Î¼g of pure compound per disk against all test organisms <i>B. subtilis</i>, <i>E. coli</i>, <i>S. cerevisiae</i>, <i>F. graminearum,</i> and <i>A. niger</i> (<a href="#tbl1">Table </a><a href="#tbl1">1</a>). <b>1</b> and <b>2</b> did not show any antibiotic activity, while the activities of emestrin (<b>3</b>) correlated qualitatively with the activity profile of the LN303 total extract. Moreover, our results fit the literature data for emestrin, for which strong antifungal activities and activity against <i>E. coli</i> were reported.<sup><a href="#ref23">23</a></sup></p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="generalexperimentalprocedures" title="sec"> <div class="title" tagx="title" title="title">
General Experimental Procedures</div> <p>Analytical HPLC separations were performed with a Shimadzu Prominence system consisting of a CBM-20A system controller, a LC-20A solvent delivery pump, a DGU-20A5 degasser, a SIL-20A autosampler, a CTO-20AC column oven, an SPD-M20A diode array detector, and an FRC-10A fraction collector. For semipreparative HPLC separations, a Shimadzu system consisting of a CBM-20A system controller, two LC-20AR solvent delivery pumps, a DGU-20A3R degasser, a manual injector, an SPD-10Ai UVâ€"Vis detector, and an FRC-10A fraction collector were applied. These systems were equipped with Shim-pack GIS columns (Shimadzu, Nakagyo-ku, KyoÌ…to, Japan) of 5 Î¼m; 4.6 Ã— 250 mm and 10 Î¼m, 20 Ã— 250 mm. Gradient elution with double-distilled water and ACN (LiChrosolv Reag. Ph Eur, gradient grade for LC, Merck KGaA) at room temperature with a flow of 1 mL/min and 20 mL/min and an injection volume of 10 Î¼L and 2 mL, respectively, was applied. Detection wavelength for fractionation was set to 254 nm.</p> <p>NMR spectra were recorded on a Bruker AVANCE 500 NMR spectrometer (UltraShield) using a 5 mm switchable probe (TCI Prodigy Kryo-probe head, 5 mm, triple resonance-inverse-detection probe head) with <i>z</i> axis gradients and automatic tuning and matching accessory (Bruker BioSpin). The resonance frequency for <sup>1</sup>H NMR was 500.13 MHz and for <sup>13</sup>C NMR 125.75 MHz. All measurements were performed for a solution in fully deuterated chloroform at 298 K. Standard 1D and gradient-enhanced 2D experiments, like double quantum filtered COSY, HSQC, and HMBC, were used as supplied by the manufacturer. Chemical shifts are referenced internally to the residual, nondeuterated solvent signal <sup>1</sup>H (Î´ 7.26 ppm) and to the carbon signal of chloroform <sup>13</sup>C (Î´ 77.00 ppm).</p> <p>LCâ€"MS analyses were performed on a Vanquish Horizon UHPLC system (Thermo Fisher Scientific) coupled to the ESI source of an LTQ Orbitrap Velos mass spectrometer (Thermo Fisher Scientific). Separation was carried out on an Acclaim 120 C18, 2.1 Ã— 150 mm, 3 Î¼m HPLC column (Thermo Fisher Scientific) using water and acetonitrile, both modified with 0.1% formic acid as mobile phases A and B, respectively. The sample components were separated and eluted with a linear gradient from 5 to 95% B in 45 min, followed by an isocratic column cleaning (9.5 min at 95% B) and re-equilibration step (10 min at 5% B). The flow rate was 0.45 mL/min, and the column oven temperature was set to 25 Â°C. High-resolution ESI-MS (HRESIMS) spectra were recorded in positive and negative ion mode in the range <i>m</i>/<i>z</i> 140â€"2000 with an FT resolution of 60.000. HRESIMS/MS spectra of the three most intense precursor ions in each MS<sup>1</sup> spectrum were obtained in automated data-dependent acquisition mode using helium as collision gas and the following settings: activation type: CID, isolation width: Î"<i>m</i>/<i>z</i> = 3, normalized collision energy of 35.0, activation <i>Q</i>: 0.250, activation time: 30 ms, and FT resolution: 15.000.</p> <div class="akanthenina(1)" title="sec"> <div class="title" tagx="title" title="title">
Akanthenin A (<b>1</b>)</div> <p>2-[(2<i>E</i>,4<i>E</i>,6<i>E</i>)-8-(Hydroxymethyl)-6-methyldeca-2,4,6-trien-2-yl]-2,3,4,6-tetrahydro-4-hydroxy-6-methoxy-3-methyl-5<i>H</i>-pyrano[3,2-<i>c</i>]pyridin-5-one, pale yellow amorphous solid; for <sup>1</sup>H and <sup>13</sup>C NMR data, see <a href="#tbl2">Table </a><a href="#tbl2">2</a> and Supporting Information <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S16</a>. HRESIMS <i>m</i>/<i>z</i>: 390.2270 [M + H]<sup>+</sup> (calcd for C<sub>22</sub>H<sub>32</sub>NO<sub>5</sub><sup>+</sup>, <i>m</i>/<i>z</i> 390.2275, Î" = 1.2 ppm) and <i>m</i>/<i>z</i>: 412.2089 [M + Na]<sup>+</sup> (calcd for C<sub>22</sub>H<sub>31</sub>NO<sub>5</sub>Na<sup>+</sup>, <i>m</i>/<i>z</i> 412.2094, Î" = 1.3 ppm).</p> </div> <div class="akantheninb(2)" title="sec"> <div class="title" tagx="title" title="title">
Akanthenin B (<b>2</b>)</div> <p>(3<i>E</i>,5<i>E</i>,7<i>E</i>)-2-Ethyl-8-(4-hydroxy-6-methoxy-3-methyl-5-oxo-3,4,5,6-tetrahydro-2<i>H</i>-pyrano[3,2-<i>c</i>]pyridin-2-yl)-4-methylnona-3,5,7-trienoic acid, pale yellow amorphous solid; for <sup>1</sup>H and <sup>13</sup>C NMR data, see <a href="#tbl2">Table </a><a href="#tbl2">2</a> and Supporting Information <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S17</a>. HRESIMS <i>m</i>/<i>z</i>: 404.2064 [M + H]<sup>+</sup> (calcd for C<sub>22</sub>H<sub>30</sub>NO<sub>6</sub><sup>+</sup>, <i>m</i>/<i>z</i> 404.2068, Î" = 0.8 ppm) and <i>m</i>/<i>z</i>: 426.1884 [M + Na]<sup>+</sup> (calcd for C<sub>22</sub>H<sub>29</sub>NO<sub>6</sub>Na<sup>+</sup>, <i>m</i>/<i>z</i> 426.1887, Î" = 0.8 ppm).</p> </div> <div class="emestrin(3)" title="sec"> <div class="title" tagx="title" title="title">
Emestrin (<b>3</b>)</div> <p>White amorphous solid; for <sup>1</sup>H and <sup>13</sup>C NMR data, see <a href="#tbl3">Table </a><a href="#tbl3">3</a> and in Supporting Information <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S18</a>. HRESIMS <i>m</i>/<i>z</i>: 597.0633 [M â€" H]<sup>âˆ’</sup> (calcd for C<sub>27</sub>H<sub>21</sub>N<sub>2</sub>O<sub>10</sub>S<sub>2</sub><sup>â€"</sup>, <i>m</i>/<i>z</i> 597.0643, Î" = 1.6 ppm).</p> </div> </div> <div class="x-raycrystallographicanalysisofemestrin" title="sec"> <div class="title" tagx="title" title="title">
X-ray Crystallographic Analysis of Emestrin</div> <p>Emestrin was crystallized in methanol at 10 Â°C, and a suitable crystal was chosen for the analysis. The X-ray intensity data were measured on a Bruker D8 Venture diffractometer equipped with a multilayer monochromator, a Mo K/Î± Incoatec micro-focus sealed tube, and an Oxford cooling system. The structure was resolved by direct methods. Nonhydrogen atoms were refined with anisotropic displacement parameters. Hydrogen atoms were inserted at calculated positions and refined with a riding model. Bruker SAINT v8.38B software packages (2005â€"2019, Bruker AXS) were used for analysis with a narrow-frame algorithm for frame integration, SADABS (Sheldrick, G. M., 1996, SADABS, University of GÃ¶ttingen, Germany) for absorption correction, OLEX2<sup><a href="#ref28">28</a></sup> for structure solution, refinement, molecular diagrams, and graphical user interface, Shelxle<sup><a href="#ref29">29</a></sup> for refinement and graphical user-interface, SHELXS-2015 for structure solution and SHELXL-2015 for refinement (Sheldrick 2015...), and PLATON<sup><a href="#ref30">30</a></sup> for symmetry check. Experimental data and CCDC-Codes (available online: <span class="uri" title="uri"><a href="http://www.ccdc.cam.ac.uk/conts/retrieving.html">http://www.ccdc.cam.ac.uk/conts/retrieving.html</a></span>) can be found in Supporting Information <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Table S1</a>. Crystal data, data on collection parameters, and structure refinement details are provided in Supporting Information <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Tables S2â€"S3</a>. The asymmetric crystal unit and its intermolecular and intramolecular interactions are given in Supporting Information <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S15</a> along with the crystallographic information file (CIF).</p> </div> <div class="plantmaterialandendophyteisolation" title="sec"> <div class="title" tagx="title" title="title">
Plant Material and Endophyte Isolation</div> <p>Five specimens of <i>L. nivale</i> subsp. <i>alpinum</i> were harvested at Rax Mountain in Austria on August 8, 2016, and have been identified by Dr. Martina Oberhofer (Department of Pharmacognosy, University of Vienna). A voucher specimen of LN2, the plant that hosted the fungal strain LN303, is deposited at the herbarium of the Department of Pharmacognosy, University of Vienna (LN2080816). After a cooled transport, plants were rinsed under tap water and surface-sterilized using a 0.1% HgCl<sub>2</sub> supplemented with 0.1% Tween (400 mL), whereby organ-specific sterilization protocols were applied. While the roots and rhizomes were exposed to HgCl<sub>2</sub> solution for 10 min, more delicate above-ground organs were treated for 5 min. After sterilization, plant parts were rinsed three times with sterile distilled water. 100 Î¼L from the last washing water was plated onto potato dextrose agar (PDA) and tryptic soy agar and incubated overnight to ensure successful sterilization. Plant material was dissected in the sterile bench, and 0.1 g was filled into maceration tubes. 1 mL of 20% sterile glycerol solution was added, and plant material was macerated applying a Precellys24 tissue homogenizer and bead tubes provided by the manufacturer (Bertin Instruments, France). Macerates were then diluted in four consecutive 1:10 dilutions; 100 Î¼L was dispersed on the surface of different nutrient media. Plates were monitored for 3 months, and new colonies were subcultured. <i>Akanthomyces</i> sp. was isolated repeatedly. Eleven isolates were each purified with single spore isolation three consecutive times. After pure cultures were obtained, DNA extraction and barcoding was performed using ITS3-LR3 primers (5â€²-3â€²GCATCGATGAAGAACGCAGC; CCGTGTTTCAAGACGGG) for the LSU region, and the resulting sequences were compared to those in the GenBank database (NCBI). Sequences are deposited at GenBank, and accession numbers are represented in <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">Figure S1</a>, a maximum likelihood analysis estimated by Mega7 software.<sup><a href="#ref31">31</a></sup> Three isolates LN276, LN303, and LN528 originating from the three host plant individuals were chosen for extraction and analyses along with two more isolates LN401 and LN427, both isolated from plant 5.</p> </div> <div class="fermentationandextraction" title="sec"> <div class="title" tagx="title" title="title">
Fermentation and Extraction</div> <p>Rice media contained 10 g of rice (Billa Himalaya Basmatireis organic) supplemented with 25 mL of distilled water and were autoclaved before inoculation. Each isolate was inoculated onto the media in 250 mL Erlenmeyer flasks and left for growing in the dark at 24 Â°C until fungal hyphae had permeated the entire media plus additionally 7 days. For upscales of the fermentation, the dimensions of flasks, rice, and water remained identical, but the number of flasks was increased. In total, LN303 was inoculated into 20 flasks, LN276 was grown in 5 flasks, and LN528, LN401, and LN427 each in one. During harvest, cultures were transferred to Schott flasks, frozen, lyophilized for 48 h, and subsequently extracted with 100 mL of dichloromethane:methanol (2:1) per rice culture for 1 h at 200 rpm at 28 Â°C. The extract was aliquoted into 50 mL Falcon tubes and centrifuged at 4000 rpm for a minimum of 10 min or until the supernatant was clear. The supernatant was then evaporated to dryness using a rotary evaporator at 40 Â°C. The dry matter yield was determined and then re-dissolved using appropriate amounts of methanol.</p> </div> <div class="discdiffusionassay" title="sec"> <div class="title" tagx="title" title="title">
Disc Diffusion Assay</div> <p>Antibiotic activity of extracts and fractions was evaluated using <i>B. subtilis</i> strain 168 (Gram-positive), <i>E. coli</i> strain DH5Î± (Gram-negative), <i>S. cerevisiae</i> BY4742 (yeast), and <i>F. graminearum</i> CBS 112.30 as well as <i>A. niger</i> CBS 110271 representing filamentous fungi. Methanolic total extracts, fractions, or pure compounds were applied onto a previously autoclaved paper disc (Whatman GE Healthcare Life Sciences, USA) and dried under sterile conditions before being placed onto specific growth media inoculated with the test organisms. Both bacteria were grown on luria agar at 37 Â°C for 12 h, yeast was cultivated on yeast extract peptone dextrose agar (YPD) at 28 Â°C for 12 h and both filamentous fungi were grown on PDA at 24 Â°C for 96 h before inspection. Activities were recorded as partial or complete inhibition zones and the respective diameter of inhibition zones along with positive (antibacterial bioassays: tetracyclin 10 Î¼g; antifungal bioassays: nystatin 500 Î¼g) and negative controls (methanol). 50 Î¼L of total extracts, fractions, and the negative control methanol was applied; in the case of pure compounds, 100 Î¼g was administered.</p> </div> <div class="extractpreparationandcompoundpurification" title="sec"> <div class="title" tagx="title" title="title">
Extract Preparation and Compound Purification</div> <p>The solid upscale extracts of LN303 were dissolved in methanol to concentrations of approx. 100 mg/mL. Samples were pretreated with water prior to semipreparative HPLC analyses. For that purpose, 1 mL of water was added to 1 mL of the extract, and the sample was left for 1 h at room temperature and was then centrifuged for 20 min at 4000 rpm (Eppendorf Centrifuge 5804 R) in order to remove the undissolved parts of the sample. Final sample concentrations were about 1 mg/mL for analytical and 50 mg/mL for semipreparative HPLC. Fractionation of supernatants was conducted on the semipreparative HPLC with a water and ACN gradient (0â€"10 min: 5â€"30% ACN; 10â€"40 min: 30â€"70% ACN; 40â€"45 min: 70â€"95% ACN; 45â€"85 min: 95% ACN). In total, seven time-depending fractions (fraction 1â€"7: min 0â€"7.5, 7.5â€"17.5, 17.5â€"25.0, 25.0â€"32.5, 32.5â€"40.0, 40.0â€"60.0, and 60.0â€"85, respectively) were collected and evaporated to dryness at 40 Â°C (Heidolph Rotary Evaporator VV2011). Residues were subsequently redissolved in methanol and tested for bioactivity. Fraction 4 (F4) was subfractionated with an optimized gradient (0â€"5 min: 50â€"52% ACN; 5â€"15 min: 52â€"53% ACN, 15â€"20 min: 53â€"95% ACN; 20â€"25 min: 95% ACN). In total, 11 fractions were collected in 2.5 min steps, whereas the main peaks were collected manually from minute 5.5 to 6.7 corresponding to akanthenin A, minute 9.9 to 10.7 corresponding to emestrin, and minute 15 to 15.8 corresponding to akanthenin B.</p> </div> </div> </div> <div class="back" title="back"> <div class="title" tagx="title" title="title">
Supporting Information Available</div> <p>The Supporting Information is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.0c05472?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acsomega.0c05472</a>. </p><ul> <li> <p>Phylogenetic analysis of endophytic <i>Akanthomyces</i> isolates; LCâ€"MS data of purified compounds <b>1</b> and <b>2</b>; HRESIMS spectrum of compound <b>3</b>; LCâ€"MS data of the total extract from the isolates LN276, LN303, LN528, LN401, and LN427; HRESIMS/MS spectra of the [M â€" H]<sup>âˆ’</sup> ion of emestrin (<b>3</b>) and the nonisolated emestrin isomer (S13); proposed fragmentation pathway of the [M â€" H]<sup>âˆ’</sup> ion of emestrin; experimental parameters of crystallography and CCDC code of emestrin; crystallographic data on emestrin; crystallographic data collection on emestrin; PLATON report; asymmetric unit of the emestrin crystal; NMR data on compound <b>1</b>; NMR data on compound <b>2</b>; and NMR data on compound <b>3</b> (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_001.pdf">PDF</a>)</p> </li> <li> <p>Crystallographic information file for emestrin (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c05472/suppl_file/ao0c05472_si_002.cif">CIF</a>)</p> </li> </ul> <p /> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="ao0c05472_si_001.pdf">LINK</a> <p>ao0c05472_si_001.pdf</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="ao0c05472_si_002.cif">LINK</a> <p>ao0c05472_si_002.cif</p>  </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>The manuscript was written through the contribution of all the authors. M.O., S.B.Z., J.W., and E.U. designed and conceptualized the study. M.O. collected the plant specimen and isolated the fungal endophytes. M.O., H.B., J.W., M.Z., E.U., A.P.-R., and J.J.C. collected analytical chemistry data. All the authors performed data analyses and have approved the final version of the manuscript.</p> <p>This study was funded by the University of Vienna.</p> <p>The authors declare no competing financial interest.</p> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We gratefully acknowledge Franz Tod, Claudia Haager and Christoph Wawrosch who helped collecting the plant specimens. Claudia Haager also participated in isolation of the endophytes. We are highly appreciative of the analytical lab assistance by Lisa Horvat from the Department of Pharmaceutical Chemistry, University of Vienna and thank Anna Fabisikova from the MS Centre of the Faculty of Chemistry, University of Vienna, for her assistance with LCâ€"MS data acquisition.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ref1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wohlleben</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mast</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stegmann</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ziemert</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="mixed-article-title" title="mixed-article-title">Antibiotic drug discovery</span>. <span class="source" tagx="source" title="source">Microbiol. Biotechnol.</span><span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">541</span>â€"<span class="lpage" tagx="lpage" title="lpage">548</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1751-7915.12388">10.1111/1751-7915.12388</a></span>.</span></li> <li tag="ref"><a name="ref2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ventola</span><span class="given-names" tagx="given-names" title="given-names">C. L.</span></span><span class="mixed-article-title" title="mixed-article-title">The antibiotic resistance crisis: part 1: causes and threats</span>. <span class="source" tagx="source" title="source">P. T.</span><span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">277</span>â€"<span class="lpage" tagx="lpage" title="lpage">283</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25859123">25859123</a></span></span></li> <li tag="ref"><a name="ref3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinez-Klimova</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RodrÃ­guez-PeÃ±a</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SÃ¡nchez</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title">Endophytes as sources of antibiotics</span>. <span class="source" tagx="source" title="source">Biochem. Pharmacol.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">134</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">17</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bcp.2016.10.010">10.1016/j.bcp.2016.10.010</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27984002">27984002</a></span></span></li> <li tag="ref"><a name="ref4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venieraki</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimou</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katinakis</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="mixed-article-title" title="mixed-article-title">Endophytic fungi residing in medicinal plants have the ability to produce the same or similar pharmacologically active secondary metabolites as their hosts</span>. <span class="source" tagx="source" title="source">Hell. Plant Protect. J.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">51</span>â€"<span class="lpage" tagx="lpage" title="lpage">66</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1515/hppj-2017-0006">10.1515/hppj-2017-0006</a></span>.</span></li> <li tag="ref"><a name="ref5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venugopalan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srivastava</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title">Endophytes as in vitro production platforms of high value plant secondary metabolites</span>. <span class="source" tagx="source" title="source">Biotechnol. Adv.</span><span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">33</span>, <span class="fpage" tagx="fpage" title="fpage">873</span>â€"<span class="lpage" tagx="lpage" title="lpage">887</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biotechadv.2015.07.004">10.1016/j.biotechadv.2015.07.004</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26225453">26225453</a></span></span></li> <li tag="ref"><a name="ref6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kusari</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spiteller</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Are we ready for industrial production of bioactive plant secondary metabolites utilizing endophytes?</span>. <span class="source" tagx="source" title="source">Nat. Prod. Rep.</span><span class="year" tagx="year" title="year">2011</span>, <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">1203</span>â€"<span class="lpage" tagx="lpage" title="lpage">1207</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/c1np00030f">10.1039/c1np00030f</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21629952">21629952</a></span></span></li> <li tag="ref"><a name="ref7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobner</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwaiger</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jenewein</span><span class="given-names" tagx="given-names" title="given-names">I. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuppner</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="mixed-article-title" title="mixed-article-title">Antibacterial activity of Leontopodium alpinum (Edelweiss)</span>. <span class="source" tagx="source" title="source">J. Ethnopharmacol.</span><span class="year" tagx="year" title="year">2003</span>, <span class="volume" tagx="volume" title="volume">89</span>, <span class="fpage" tagx="fpage" title="fpage">301</span>â€"<span class="lpage" tagx="lpage" title="lpage">303</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jep.2003.09.004">10.1016/j.jep.2003.09.004</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14611896">14611896</a></span></span></li> <li tag="ref"><a name="ref8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tauchen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kokoska</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="mixed-article-title" title="mixed-article-title">The chemistry and pharmacology of Edelweiss: a review</span>. <span class="source" tagx="source" title="source">Phytochem. Rev.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">295</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11101-016-9474-0">10.1007/s11101-016-9474-0</a></span>.</span></li> <li tag="ref"><a name="ref9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vinit</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doilom</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wanasinghe</span><span class="given-names" tagx="given-names" title="given-names">D. N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhat</span><span class="given-names" tagx="given-names" title="given-names">D. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brahmanage</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeewon</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyde</span><span class="given-names" tagx="given-names" title="given-names">K. D.</span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic placement of Akanthomyces muscarius, a new endophyte record from Nypa fruticans in Thailand</span>. <span class="source" tagx="source" title="source">Curr. Res. Environ. Appl. Mycol.</span><span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">404</span>â€"<span class="lpage" tagx="lpage" title="lpage">417</span>. <span class="pub-id"><a href="https://dx.doi.org/10.5943/cream/8/3/10">10.5943/cream/8/3/10</a></span>.</span></li> <li tag="ref"><a name="ref10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kepler</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luangsa-Ard</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hywel-Jones</span><span class="given-names" tagx="given-names" title="given-names">N. L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quandt</span><span class="given-names" tagx="given-names" title="given-names">C. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sung</span><span class="given-names" tagx="given-names" title="given-names">G.-H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rehner</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aime</span><span class="given-names" tagx="given-names" title="given-names">M. C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henkel</span><span class="given-names" tagx="given-names" title="given-names">T. W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanjuan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zare</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossman</span><span class="given-names" tagx="given-names" title="given-names">A. Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spatafora</span><span class="given-names" tagx="given-names" title="given-names">J. W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shrestha</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="mixed-article-title" title="mixed-article-title">A phylogenetically-based nomenclature for Cordycipitaceae (Hypocreales)</span>. <span class="source" tagx="source" title="source">imafungus</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">335</span>â€"<span class="lpage" tagx="lpage" title="lpage">353</span>. <span class="pub-id"><a href="https://dx.doi.org/10.5598/imafungus.2017.08.02.08">10.5598/imafungus.2017.08.02.08</a></span>.</span></li> <li tag="ref"><a name="ref11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sung</span><span class="given-names" tagx="given-names" title="given-names">G.-H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hywel-Jones</span><span class="given-names" tagx="given-names" title="given-names">N. L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sung</span><span class="given-names" tagx="given-names" title="given-names">J.-M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luangsa-Ard</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shrestha</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spatafora</span><span class="given-names" tagx="given-names" title="given-names">J. W.</span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic classification of Cordyceps and the clavicipitaceous fungi</span>. <span class="source" tagx="source" title="source">Stud. Mycol.</span><span class="year" tagx="year" title="year">2007</span>, <span class="volume" tagx="volume" title="volume">57</span>, <span class="fpage" tagx="fpage" title="fpage">5</span>â€"<span class="lpage" tagx="lpage" title="lpage">59</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3114/sim.2007.57.01">10.3114/sim.2007.57.01</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18490993">18490993</a></span></span></li> <li tag="ref"><a name="ref12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MolnÃ¡r</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krasnoff</span><span class="given-names" tagx="given-names" title="given-names">S. B.</span></span><span class="mixed-article-title" title="mixed-article-title">Secondary metabolites from entomopathogenic Hypocrealean fungi</span>. <span class="source" tagx="source" title="source">Nat. Prod. Rep.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">27</span>, <span class="fpage" tagx="fpage" title="fpage">1241</span>â€"<span class="lpage" tagx="lpage" title="lpage">1275</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/c001459c">10.1039/c001459c</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20601982">20601982</a></span></span></li> <li tag="ref"><a name="ref13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo</span><span class="given-names" tagx="given-names" title="given-names">H.-C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsieh</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">F.-Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">T.-H.</span></span><span class="mixed-article-title" title="mixed-article-title">A systematic review of the mysterious caterpillar fungus Ophiocordyceps sinensis in DongChongXiaCao (DoÌ…ng ChÃ³ng XiÃ CÇŽo) and related bioactive ingredients</span>. <span class="source" tagx="source" title="source">J. Tradit., Complementary Altern. Med.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">16</span>â€"<span class="lpage" tagx="lpage" title="lpage">32</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/s2225-4110(16)30164-x">10.1016/s2225-4110(16)30164-x</a></span>.</span></li> <li tag="ref"><a name="ref14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chassagne</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cabanac</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubert</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">David</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marti</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="mixed-article-title" title="mixed-article-title">The landscape of natural product diversity and their pharmacological relevance from a focus on the Dictionary of Natural Products</span>. <span class="source" tagx="source" title="source">Phytochem. Rev.</span><span class="year" tagx="year" title="year">2019</span>, <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">601</span>â€"<span class="lpage" tagx="lpage" title="lpage">622</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11101-019-09606-2">10.1007/s11101-019-09606-2</a></span>.</span></li> <li tag="ref"><a name="ref15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatsuta</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamaguchi</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsuda</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamaguchi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hattori</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagai</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hosokawa</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title">The first total synthesis and structural determination of YCM1008A</span>. <span class="source" tagx="source" title="source">Tetrahedron Lett.</span><span class="year" tagx="year" title="year">2007</span>, <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">4187</span>â€"<span class="lpage" tagx="lpage" title="lpage">4190</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tetlet.2007.04.074">10.1016/j.tetlet.2007.04.074</a></span>.</span></li> <li tag="ref"><a name="ref16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koizumi</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukumitsu</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chanklan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyakawa</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawahara</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwamoto</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kakita</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahayu</span><span class="given-names" tagx="given-names" title="given-names">E. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hosokawa</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatsuta</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ichimura</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">YCM1008A, a novel Ca2<sup>+</sup>-signaling inhibitor, produced by Fusarium sp. YCM1008</span>. <span class="source" tagx="source" title="source">J. Antibiot.</span><span class="year" tagx="year" title="year">2007</span>, <span class="volume" tagx="volume" title="volume">60</span>, <span class="fpage" tagx="fpage" title="fpage">455</span>â€"<span class="lpage" tagx="lpage" title="lpage">458</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/ja.2007.58">10.1038/ja.2007.58</a></span>.</span></li> <li tag="ref"><a name="ref17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jessen</span><span class="given-names" tagx="given-names" title="given-names">H. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gademann</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="mixed-article-title" title="mixed-article-title">4-Hydroxy-2-pyridone alkaloids: structures and synthetic approaches</span>. <span class="source" tagx="source" title="source">Nat. Prod. Rep.</span><span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">27</span>, <span class="fpage" tagx="fpage" title="fpage">1168</span>â€"<span class="lpage" tagx="lpage" title="lpage">1185</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/b911516c">10.1039/b911516c</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20532354">20532354</a></span></span></li> <li tag="ref"><a name="ref18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneider</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riese</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schubert</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamburger</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Farinosones Aâ€"C, Neurotrophic Alkaloidal Metabolites from the Entomogenous Deuteromycete Paecilomyces farinosus</span>. <span class="source" tagx="source" title="source">J. Nat. Prod.</span><span class="year" tagx="year" title="year">2004</span>, <span class="volume" tagx="volume" title="volume">67</span>, <span class="fpage" tagx="fpage" title="fpage">1854</span>â€"<span class="lpage" tagx="lpage" title="lpage">1858</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/np049761w">10.1021/np049761w</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15568775">15568775</a></span></span></li> <li tag="ref"><a name="ref19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azzi</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sayegh</span><span class="given-names" tagx="given-names" title="given-names">M. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallat</span><span class="given-names" tagx="given-names" title="given-names">S. G.</span></span><span class="mixed-article-title" title="mixed-article-title">Calcineurin inhibitors: 40 years later, canâ€™t live without</span>. <span class="source" tagx="source" title="source">J. Immunol.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">191</span>, <span class="fpage" tagx="fpage" title="fpage">5785</span>â€"<span class="lpage" tagx="lpage" title="lpage">5791</span>. <span class="pub-id"><a href="https://dx.doi.org/10.4049/jimmunol.1390055">10.4049/jimmunol.1390055</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24319282">24319282</a></span></span></li> <li tag="ref"><a name="ref20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagenaar</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clardy</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="mixed-article-title" title="mixed-article-title">Akanthomycin, a new antibiotic pyridone from the entomopathogenic fungus Akanthomyces gracilis</span>. <span class="source" tagx="source" title="source">Org. Lett.</span><span class="year" tagx="year" title="year">2002</span>, <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">671</span>â€"<span class="lpage" tagx="lpage" title="lpage">673</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ol016737q">10.1021/ol016737q</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11869098">11869098</a></span></span></li> <li tag="ref"><a name="ref21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yue</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krausert</span><span class="given-names" tagx="given-names" title="given-names">N. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">An</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gloer</span><span class="given-names" tagx="given-names" title="given-names">J. B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bills</span><span class="given-names" tagx="given-names" title="given-names">G. F.</span></span><span class="mixed-article-title" title="mixed-article-title">Emestrins: anti- Cryptococcus epipolythiodioxopiperazines from Podospora australis</span>. <span class="source" tagx="source" title="source">J. Nat. Prod.</span><span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">79</span>, <span class="fpage" tagx="fpage" title="fpage">2357</span>â€"<span class="lpage" tagx="lpage" title="lpage">2363</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jnatprod.6b00498">10.1021/acs.jnatprod.6b00498</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27557418">27557418</a></span></span></li> <li tag="ref"><a name="ref22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nursid</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chasanah</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murwantoko</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wahyuono</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and identification of emestrin from Emericella nidulans and investigation of its anticancer properties</span>. <span class="source" tagx="source" title="source">Microbiol. Indonesia</span><span class="year" tagx="year" title="year">2011</span>, <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">160</span>â€"<span class="lpage" tagx="lpage" title="lpage">169</span>. <span class="pub-id"><a href="https://dx.doi.org/10.5454/mi.5.4.3">10.5454/mi.5.4.3</a></span>.</span></li> <li tag="ref"><a name="ref23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herath</span><span class="given-names" tagx="given-names" title="given-names">H. M. T. B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacob</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abbas</span><span class="given-names" tagx="given-names" title="given-names">H. K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanayakkara</span><span class="given-names" tagx="given-names" title="given-names">N. P. D.</span></span><span class="mixed-article-title" title="mixed-article-title">New secondary metabolites from bioactive extracts of the fungus Armillaria tabescens</span>. <span class="source" tagx="source" title="source">Nat. Prod. Res.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">27</span>, <span class="fpage" tagx="fpage" title="fpage">1562</span>â€"<span class="lpage" tagx="lpage" title="lpage">1568</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/14786419.2012.738206">10.1080/14786419.2012.738206</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23140424">23140424</a></span></span></li> <li tag="ref"><a name="ref24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seya</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakajima</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawai</span><span class="given-names" tagx="given-names" title="given-names">K.-i.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Udagawa</span><span class="given-names" tagx="given-names" title="given-names">S.-i.</span></span><span class="mixed-article-title" title="mixed-article-title">Structure and absolute configuration of emestrin, a new macrocyclic epidithiodioxopiperazine from Emericella striata</span>. <span class="source" tagx="source" title="source">J. Chem. Soc., Chem. Commun.</span><span class="year" tagx="year" title="year">1985</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">657</span>â€"<span class="lpage" tagx="lpage" title="lpage">658</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/c39850000657">10.1039/c39850000657</a></span>.</span></li> <li tag="ref"><a name="ref25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooike</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nozawa</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawai</span><span class="given-names" tagx="given-names" title="given-names">K.-I.</span></span><span class="mixed-article-title" title="mixed-article-title">An epitetrathiodioxopiperazine related to emestrin from Emericella foveolata</span>. <span class="source" tagx="source" title="source">Phytochem</span><span class="year" tagx="year" title="year">1997</span>, <span class="volume" tagx="volume" title="volume">46</span>, <span class="fpage" tagx="fpage" title="fpage">123</span>â€"<span class="lpage" tagx="lpage" title="lpage">126</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/s0031-9422(97)00210-0">10.1016/s0031-9422(97)00210-0</a></span>.</span></li> <li tag="ref"><a name="ref26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">T. R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Epidithiodioxopiperazines. occurrence, synthesis and biogenesis</span>. <span class="source" tagx="source" title="source">Nat. Prod. Rep.</span><span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">1376</span>â€"<span class="lpage" tagx="lpage" title="lpage">1404</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/c3np70097f">10.1039/c3np70097f</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24816491">24816491</a></span></span></li> <li tag="ref"><a name="ref27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seya</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nozawa</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakajima</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawai</span><span class="given-names" tagx="given-names" title="given-names">K.-I.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Udagawa</span><span class="given-names" tagx="given-names" title="given-names">S.-I.</span></span><span class="mixed-article-title" title="mixed-article-title">Studies on fungal products. Part 8. Isolation and structure of emestrin, a novel antifungal macrocyclic epidithiodioxopiperazine from Emericella striata. X-Ray molecular structure of emestrin</span>. <span class="source" tagx="source" title="source">J. Chem. Soc., Perkin Trans. 1</span><span class="year" tagx="year" title="year">1986</span>, <span class="fpage" tagx="fpage" title="fpage">109</span>â€"<span class="lpage" tagx="lpage" title="lpage">116</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/p19860000109">10.1039/p19860000109</a></span>.</span></li> <li tag="ref"><a name="ref28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dolomanov</span><span class="given-names" tagx="given-names" title="given-names">O. V.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourhis</span><span class="given-names" tagx="given-names" title="given-names">L. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gildea</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howard</span><span class="given-names" tagx="given-names" title="given-names">J. A. K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puschmann</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="mixed-article-title" title="mixed-article-title">OLEX2: a complete structure solution, refinement and analysis program</span>. <span class="source" tagx="source" title="source">J. Appl. Crystallogr.</span><span class="year" tagx="year" title="year">2009</span>, <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">339</span>â€"<span class="lpage" tagx="lpage" title="lpage">341</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1107/s0021889808042726">10.1107/s0021889808042726</a></span>.</span></li> <li tag="ref"><a name="ref29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HÃ¼bschle</span><span class="given-names" tagx="given-names" title="given-names">C. B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheldrick</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dittrich</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="mixed-article-title" title="mixed-article-title">ShelXle: a Qt graphical user interface for SHELXL</span>. <span class="source" tagx="source" title="source">J. Appl. Crystallogr.</span><span class="year" tagx="year" title="year">2011</span>, <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">1281</span>â€"<span class="lpage" tagx="lpage" title="lpage">1284</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1107/S0021889811043202">10.1107/S0021889811043202</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22477785">22477785</a></span></span></li> <li tag="ref"><a name="ref30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spek</span><span class="given-names" tagx="given-names" title="given-names">A. L.</span></span><span class="mixed-article-title" title="mixed-article-title">PLATON SQUEEZE, a tool for the calculation of the disordered solvent contribution to the calculated structure factors</span>. <span class="source" tagx="source" title="source">Acta Crystallogr., Sect. C: Struct. Chem.</span><span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">71</span>, <span class="fpage" tagx="fpage" title="fpage">9</span>â€"<span class="lpage" tagx="lpage" title="lpage">18</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1107/S2053229614024929">10.1107/S2053229614024929</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25567569">25567569</a></span></span></li> <li tag="ref"><a name="ref31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="mixed-article-title" title="mixed-article-title">MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">33</span>, <span class="fpage" tagx="fpage" title="fpage">1870</span>â€"<span class="lpage" tagx="lpage" title="lpage">1874</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msw054">10.1093/molbev/msw054</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27004904">27004904</a></span></span></li> </ul> </div> </div>  </body></html>