<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PeerJ</span><span class="iso-abbrev" title="iso-abbrev">PeerJ</span><span class="publisher-id" title="publisher-id">peerj</span><span class="pmc" title="pmc">peerj</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PeerJ</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2167-8359</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">PeerJ Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Diego, USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7703384">7703384</a></span><span class="publisher-id" title="publisher-id">10392</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7717/peerj.10392">10.7717/peerj.10392</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Molecular Biology</span>: <span class="subject" title="subject">Mycology</span>: <span class="subject" title="subject">Drugs and Devices</span>: <span class="subject" title="subject">Infectious Diseases</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
A diverse global fungal library for drug discovery</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">Guodong</span></span><a href="#aff-1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Annamalai</span><span class="given-names" tagx="given-names" title="given-names">Thirunavukkarasu</span></span><a href="#aff-2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Xiaohong</span></span><a href="#aff-1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Sheng</span></span><a href="#aff-3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munga</span><span class="given-names" tagx="given-names" title="given-names">Stephen</span></span><a href="#aff-4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">Guomin</span></span><a href="#aff-5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tse-Dinh</span><span class="given-names" tagx="given-names" title="given-names">Yuk-Ching</span></span><a href="#aff-2">2</a><a href="#aff-6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Jun</span></span><span class="email" tagx="email" title="email">lij@fiu.edu</span><a href="#aff-1">1</a><a href="#aff-6">6</a></span><span class="citation_author_institution" id="aff-1">[1], <span class="institution" title="institution">Department of Biological Sciences, Florida International University</span><span class="city" tagx="city" title="city">Miami</span><span class="state" tagx="state" title="state">FL</span><span class="country" tagx="country" title="country">United States of America</span></span><span class="citation_author_institution" id="aff-2">[2], <span class="institution" title="institution">Department of Chemistry and Biochemistry, Florida International University</span><span class="city" tagx="city" title="city">Miami</span><span class="state" tagx="state" title="state">FL</span><span class="country" tagx="country" title="country">United States of America</span></span><span class="citation_author_institution" id="aff-3">[3], <span class="institution" title="institution">School of Public Health, City University of New York</span><span class="city" tagx="city" title="city">NY</span><span class="country" tagx="country" title="country">United States of America</span></span><span class="citation_author_institution" id="aff-4">[4], <span class="institution" title="institution">Center for Global Health Research, Kenya Medical Research Institute</span><span class="city" tagx="city" title="city">Kisumu</span><span class="country" tagx="country" title="country">Kenya</span></span><span class="citation_author_institution" id="aff-5">[5], <span class="institution" title="institution">Department of Hematology, Southern Medical University Affiliated Nanhai Hospital</span><span class="city" tagx="city" title="city">Foshan</span><span class="state" tagx="state" title="state">Guangdong</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="aff-6">[6], <span class="institution" title="institution">Biomolecular Sciences Institute, Florida International University</span><span class="city" tagx="city" title="city">Miami</span><span class="state" tagx="state" title="state">FL</span><span class="country" tagx="country" title="country">United States of America</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomez</span><span class="given-names" tagx="given-names" title="given-names">Shawn</span></span></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e10392</span><span class="history" title="history"><span class="received" title="received">received: 2020-6-25</span><span class="accepted" title="accepted">accepted: 2020-10-28</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Niu et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, reproduction and adaptation in any medium and for any purpose provided that it is properly attributed. For attribution, the original author(s), title, publication source (PeerJ) and either DOI or URL of the article must be cited.</span></span></div><a class="self-uri" href="https://peerj.com/articles/10392" title="self-uri">https://peerj.com/articles/10392</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Secondary fungal metabolites are important sources for new drugs against infectious diseases and cancers.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p>To obtain a library with enough diversity, we collected about 2,395 soil samples and 2,324 plant samples from 36 regions in Africa, Asia, and North America. The collection areas covered various climate zones in the world. We examined the usability of the global fungal extract library (GFEL) against parasitic malaria transmission, Gram-positive and negative bacterial pathogens, and leukemia cells.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>Nearly ten thousand fungal strains were isolated. Sequences of nuclear ribosomal internal transcribed spacer (ITS) from 40 randomly selected strains showed that over 80% were unique. Screening GFEL, we found that the fungal extract from <i>Penicillium thomii</i> was able to block <i>Plasmodium falciparum</i> transmission to <i>Anopheles gambiae</i>, and the fungal extract from <i>Tolypocladium album</i> was able to kill myelogenous leukemia cell line K562. We also identified a set of candidate fungal extracts against bacterial pathogens.</p> </div> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
Fungus</div> <div class="kwd" title="kwd">
Secondary metabolites</div> <div class="kwd" title="kwd">
Small molecule</div> <div class="kwd" title="kwd">
Natural product</div> <div class="kwd" title="kwd">
Drugs</div> <div class="kwd" title="kwd">
Anti-malaria</div> <div class="kwd" title="kwd">
Antibiotics</div> <div class="kwd" title="kwd">
Anti-leukemia</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
NIAID</div><span class="award-id" tagx="award-id" title="award-id">No. 1R01AI125657</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
NSF Career Award</div><span class="award-id" tagx="award-id" title="award-id">No. 1453287</span></div> <div class="funding-statement" title="funding-statement">
This work is supported by NIAID (No. 1R01AI125657) and NSF Career Award (No. 1453287). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Natural products, produced by living organisms in nature, have been used as medicine for thousands of years (<a href="#ref-3">Dias, Urban &amp;amp; Roessner, 2012</a>; <a href="#ref-2">Buyel, 2018</a>). For instance, the treatment of malaria was recorded in China with the Qinghao plant (<i>Artemisia annua</i>) as early as the second century. The active ingredient, qinghaosu (artemisinin), was isolated from the plant by Youyou Tu and her colleagues in 1971 (<a href="#ref-19">Luo &amp;amp; Shen, 1987</a>; <a href="#ref-12">Hsu, 2006</a>). The establishment of microbiology in the early modern era led to drug discoveries from microbes. The first antibiotic, penicillin, was discovered from a fungus by Alexander Fleming (<a href="#ref-6">Fleming, 1929</a>). Fungi have initially been and still are used to produce medicines to treat infectious diseases (<a href="#ref-4">Elder, 1944</a>). Furthermore, people use natural products to treat tumors. Indeed, more than half of anti-tumor drugs or leads in current clinical trials are from natural products (<a href="#ref-34">Wolfender &amp;amp; Queiroz, 2012</a>).</p> <p>Microbial metabolites are essential resources for drug discovery (<a href="#ref-16">Lenzi et al., 2018</a>). Compared with other natural product resources, fungi have the following advantages. First, there are enormous fungal species: about 120,000 fungal species have been described (<a href="#ref-10">Hawksworth &amp;amp; Lucking, 2017</a>) and 5.1 million fungal species are estimated (<a href="#ref-1">Blackwell, 2011</a>). Second, fungi produce broad and diverse secondary metabolites with a vast difference in chemical structures (<a href="#ref-27">Pham et al., 2019</a>). Third, large-scale fermentation can generate a large amount of fungal secondary metabolites, which was exampled by the production of alcohol and lactic acid. However, yield of many target fungal secondary metabolites is restricted by fungal growth and differentiation (<a href="#ref-21">Nielsen &amp;amp; Nielsen, 2017</a>; <a href="#ref-15">Keller, 2019</a>; <a href="#ref-27">Pham et al., 2019</a>), which is resolved by new technologies that enable us to engineer a fungus to produce a specific compound in high yield by modifying its metabolic pathways (<a href="#ref-31">Van Dijk &amp;amp; Wang, 2016</a>). Also, the recent development of genomic sequencing technology and the identification of more biosynthetic gene clusters accelerate the discovery and application of new compounds from fungi (<a href="#ref-13">Hussain et al., 2017</a>; <a href="#ref-15">Keller, 2019</a>).</p> <p>Fungi can be isolated from soil, water, air, plants, or other organisms. In particular, the endophytic fungi from the plants can generate similar secondary metabolites as their hosts (<a href="#ref-32">Venieraki, Dimou &amp;amp; Katinakis, 2017</a>). Thus, the heterologous expression can replace their hosts as the supplies of crude materials for some medicines (<a href="#ref-31">Van Dijk &amp;amp; Wang, 2016</a>). Diverse fungal libraries are critical for the research and industry communities (<a href="#ref-24">Niu et al., 2015</a>). At present, there are several specific fungal libraries for drug discovery (<a href="#ref-20">May et al., 2004</a>; <a href="#ref-29">Richards et al., 2012</a>; <a href="#ref-37">Zhang et al., 2014</a>), many of which focus on specific environments (<a href="#ref-17">Li et al., 2005</a>; <a href="#ref-8">Gonzalez-Menendez et al., 2018</a>; <a href="#ref-36">Zhang et al., 2018</a>). We focus on generating a global diverse fungal library to facilitate new drug discovery. The soil and plant samples were collected globally, currently including from Asia, Africa, and North America.</p> <p>To determine the usability of our fungal library, we screened the newly established fungal extract library for malaria transmission inhibitors, antibiotics, and drug leads against chronic myeloid leukemia. A set of positives were discovered.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials &amp;amp; Methods</div> <div class="collectingsamples" title="sec"> <div class="title" tagx="title" title="title">
Collecting samples</div> <p>We collected plant and soil samples from different regions around the world. The field collection was approved by the United States Department of Agriculture with permit number of P526P-18-03319. The soil samples were taken in 5â€"10Â cm depths under the surface. Plant samples consisted of the whole plant or separated plant parts such as roots, stems, leaves, flowers, fruits, or various combinations of components. Samples were stored on ice or in a 4Â Â°C fridge immediately after collection. No samples were more than 5 g. Most of samples collected by authors, residents, and friends were mailed to labs and processed locally. For instance, samples collected in China and Myanmar were shipped to Guomin Niuâ€™s lab in Guangdong, China to process. Samples collected in the US were processed at Jun Liâ€™s lab in Florida, USA. The fungal extracts form the GFEL library.</p> </div> <div class="isolationoffungifromsoilandplants" title="sec"> <div class="title" tagx="title" title="title">
Isolation of fungi from soil and plants</div> <p>For each soil sample, we transferred 50 mg soil to a 1.5 mL plastic tube, and one mL autoclaved distilled water was added. The sample was vortexed for 30 secs and centrifuged at 500 g for 2Â min (min) to get rid of the soil particles. For plant samples, the first step was to sterilize the plant surface by rinsing the sample with distilled H<sub>2</sub>O, then soaking in 70% ethanol for 10Â s (sec) and rinsing again with distilled H<sub>2</sub>O. After sterilization, the plants were cut into 0.5Â cm Ã— 0.5Â cm pieces and transferred into a sterile mortar. Then, two mL of distilled H<sub>2</sub>O was added, followed by grinding with a pestle for 1â€"3 min and the slurry was transferred to a 1.5 mL plastic tube and then centrifuged at 500 g for 2 min to get rid of the particles. About 100 ÂµL of upper supernatant from treated soil or plants was evenly spread onto a 100 Ã— 15Â mm Petri Dish plate containing 14 mL Malt Extract Agar (MEA), made of 10 g malt extract, 1 g yeast extract, 15 g agar, and 0.05 g chloramphenicol (Sigma-Aldrich, St. Louis, MO) in 1 L distilled H<sub>2</sub>O and autoclaved at 121Â Â°C for 20 min. The plates were sealed with parafilm, and the fungi were allowed to grow for 7â€"14 days at room temperature (RT) with cycles of 12Â h (hr) of darkness and 12 hr of light.</p> <p>The fungal colonies on the MEA medium plates were picked with a toothpick and inoculated in a new MEA plate by streaking. If the colonies were mixtures of two or more species, we kept inoculating and streaking until a single colony appeared. Finally, a piece of fungal agar containing mycelium or spores was cut and transferred to a 1.5 mL Eppendorf tube containing 500 ÂµL of sterile 20% glycerol in distilled H<sub>2</sub>O. We stored the cells in a âˆ’80Â Â°C freezer for long-term storage.</p> </div> <div class="metaboliteproductionandextraction" title="sec"> <div class="title" tagx="title" title="title">
Metabolite production and extraction</div> <p>Cereal based medium was used to grow fungi to produce secondary metabolites (<a href="#ref-24">Niu et al., 2015</a>). Briefly, six pieces of Cheerios Breakfast cereals (General Mills, Minneapolis, MN) were placed in a glass test tube, capped with a plastic lid and autoclaved for 20 min, and then two mL sterile sucrose water (3 g of sucrose and 50 mg chloramphenicol in 1 L distilled H<sub>2</sub>O) was added into the tube. Later, the fungal colony grown on the MEA plate was inoculated into the cereal medium and incubated at RT for one month to produce sufficient metabolites. A month late, two mL ethyl acetate was added into a tube to extract the fungal metabolites, mixed with a glass stirring rod, and placed overnight in a chemical hood with gentle shaking. The next day, one mL upper layer of supernatant was transferred to a 1.5 mL tube. After centrifugation (2,000 g for 2 min), around 950Â ÂµL clear supernatant was transferred to a pre-weighed 1.5 mL plastic tube and dried with SpeedVac concentrator (Thermo Fisher Scientific, Waltham, MA). Finally, the dry extracts were weighed and dissolved in an appropriate amount of dimethyl sulfoxide (DMSO) to prepare 10 mg/mL stock solution and stored in a âˆ’20Â Â°C freezer for future screening assays.</p> </div> <div class="determinationoffungalspecies" title="sec"> <div class="title" tagx="title" title="title">
Determination of fungal species</div> <p>Forty fungal isolates were randomly picked to evaluate the fungal libraryâ€™s diversity. The fungi were cultured with liquid malt extract medium at RT for one week, and mycelium was collected for DNA extraction using DNAzol (Thermo Fisher). Genomic DNA applied as PCR templates were isolated using DNAzol Reagent following the manual (Thermo Fisher Scientific). To identify the fungal species, nuclear ribosomal ITS regions were amplified by PCR with specific primers (<a href="#table-1">Table 1</a>) (<a href="#ref-26">Op De Beeck et al., 2014</a>) using the following approach: 94Â Â°C 2 min; 94Â Â°C 30Â s, 55Â Â°C 30Â s, 72Â Â°C 1 min, 35 cycles; 72Â Â°C 5 min. The amplified products were sequenced and blasted against the NCBI database to identify fungal species (<a href="#ref-28">Raja et al., 2017</a>).</p> <div class="table-wrap_UNKNOWN" id="table-1" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7717/peerj.10392/table-1">10.7717/peerj.10392/table-1</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
PCR Primers for fungal ITS regions.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1">Primer name</th> <th rowspan="1" colspan="1">sequences</th> <th rowspan="1" colspan="1">rDNA region</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">ITS1F (F)</td> <td rowspan="1" colspan="1">CTTGGTCATTTAGAGGAAGTAA</td> <td rowspan="1" colspan="1">18S</td> </tr> <tr> <td rowspan="1" colspan="1">ITS2 (R)</td> <td rowspan="1" colspan="1">GCTGCGTTCTTCATCGATGC</td> <td rowspan="1" colspan="1">5.8S</td> </tr> <tr> <td rowspan="1" colspan="1">ITS3 (F)</td> <td rowspan="1" colspan="1">GCATCGATGAAGAACGCAGC</td> <td rowspan="1" colspan="1">5.8S</td> </tr> <tr> <td rowspan="1" colspan="1">ITS4 (R)</td> <td rowspan="1" colspan="1">TCCTCCGCTTATTGATATGC</td> <td rowspan="1" colspan="1">28S</td> </tr> <tr> <td rowspan="1" colspan="1">ITS86F (F)</td> <td rowspan="1" colspan="1">GTGAATCATCGAATCTTTGAA</td> <td rowspan="1" colspan="1">5.8S</td> </tr> <tr> <td rowspan="1" colspan="1">ITS86R (R)</td> <td rowspan="1" colspan="1">TTCAAAGATTCGATGATTCAC</td> <td rowspan="1" colspan="1">5.8S</td> </tr> </tbody> </table> </div> </div> </div> <div class="screeningthefungalextractlibrarytoidentifymalariatransmission-blockingcandidateswithelisaassays" title="sec"> <div class="title" tagx="title" title="title">
Screening the fungal extract library to identify malaria transmission-blocking candidates with ELISA assays</div> <p>As described previously (<a href="#ref-24">Niu et al., 2015</a>), the red blood cells (iRBC) infected by <i>Plasmodium falciparum</i> (NF54 strain from MR4, Manassas, VA) were cultured in RPMI-1640 medium (Life Tech, Grand Island, NY) supplemented with 10% heat-inactivated (56Â Â°C for 45 min) human AB+ serum (Interstate blood bank, Memphis, TN), 12.5 Âµg/mL hypoxanthine and 4% hematocrit (O+ human blood) in a candle jar at 37Â Â°C for 15-17 days. The medium was replaced every day to provide sufficient nutrients. Blood smears stained with Giemsa (Sigma-Aldrich, St. Louis, MO) were used to examine parasitemia or gametocytemia every other day under a light microscope. Then, the cells were collected and washed three times with RPMI-1640 at 300Â Ã— g for 4 m. The cell pellets were re-suspended in PBST (PBS containing 0.2% Tween-20) and homogenized by ultra-sonication with six cycles of 10Â s pulse and 50Â s resting on ice for each period. The lysates were centrifuged at 8,000 g for 2 min to remove insoluble materials and cellular debris. With the iRBC lysate and insect cell-expressed recombinant FREP1, the ELISA assay was used to screen the fungal extract library to block FREP1-parasite interaction (<a href="#ref-24">Niu et al., 2015</a>). A 96-well ELISA plate was coated with 50 ÂµL iRBC lysate (2 mg/mL protein) and incubated overnight at 4Â Â°C. After coating, the plate was blocked with 100 ÂµL of PBS plus 0.2% bovine serum albumin (BSA) per well for 1.5 hr at RT. After removal of the blocking solution, FREP1 (10 Âµg/mL) in blocking buffer (PBS plus 0.2% BSA) was added to each well, and 1 ÂµL fungal extract was taken from a 96-well plate containing 2 mg/mL crude extract dissolved in DMSO in each well with a multiple-channel pipette and transferred to the ELISA plate, then incubated for 1 hr at RT with gentle shaking. After washing three times with PBST, 50 ÂµL rabbit anti-FREP1 polyclonal antibody (<a href="#ref-24">Niu et al., 2015</a>) (diluted 1: 5,000 in blocking buffer, 1 Âµg/mL) was added to each well and incubated for 1 hr at RT. About 50 ÂµL alkaline phosphatase-conjugated anti-rabbit IgG (diluted 1: 20,000 in blocking buffer) was added to each well and incubated for 45 min at RT. The wells were washed three times with PBST between incubations. After washing, each well was developed with 50 ÂµL pNPP substrate (Sigma-Aldrich) until the colors were visible, and absorbance at 405 nm was measured. The functional FREP1 supplemented with 1 ÂµL solvent (DMSO) was used as non-inhibition control, and the heat-inactivated FREP1 (65Â Â°C for 15 min) was used as a 100% inhibition control.</p> </div> <div class="determinationofthetransmission-blockingactivityofthefungalextractsinmosquitoes" title="sec"> <div class="title" tagx="title" title="title">
Determination of the transmission-blocking activity of the fungal extracts in mosquitoes</div> <p>Following the previous protocol (<a href="#ref-35">Zhang et al., 2015</a>), the 15- to 17-day old cultured <i>P.Â falciparum</i> containing 2â€"3% stage V gametocytes were collected and diluted with new O+ type human blood to get 0.2% stage V gametocytes in the blood. Then, the 150 ÂµL blood was mixed with the same volume of heat-inactivated AB+ human serum. Then, 3 ÂµL candidate fungal extract in DMSO (10 mg/mL or 2 mg/mL) was mixed with 297 ÂµL infected blood, the final fungal extract concentration in blood was 100 or 20Â Âµg/mL, respectively. SMFA was performed to feed about 100 3â€"5 days old <i>An.Â gambiae</i> G3 female mosquitoes for 15 min, and the engorged mosquitoes were maintained with 8% sugar in a BSL-2 insectary (28Â Â°C, 12-h light/dark cycle, 80% humidity). The midguts were dissected seven days post-infection and stained with 0.1% mercury dibromofluorescein disodium salt in PBS for 16 min. The oocysts in midguts were counted under a light microscope. The standard membrane feeding assays were conducted at least twice to confirm the results.</p> </div> <div class="screeningthefungalextractlibrarytoidentifyantibiotics" title="sec"> <div class="title" tagx="title" title="title">
Screening the fungal extract library to identify antibiotics</div> <p>A subset of fungal extracts, randomly selected, were tested for the antibacterial activity to inhibit the growth of <i>Shigella flexneri</i> (ATCC 9199), <i>Staphylococcus aureus</i> (ATCC 14775), methicillin-resistant <i>Staphylococcus aureus</i> (MRSA, ATCC BAA-44) and <i>E. coli</i> (AS17tolc). The drug-screening was performed in a 384-well microplate format using the following procedures. Bacteria were cultured in 50 mL Mueller Hinton broth (MHB) in a 150-mL flask overnight at 37Â Â°C. The next day, the cells were first OD<sub>600</sub> adjusted to 0.1 and then further diluted 1:100 in MHB, and a volume of 50Â ÂµL (âˆ¼10<sup>5</sup> CFU) is added to each well of the 384-well sterile microplates (Thermo Fisher Scientific). Fungal extracts (0.5Â ÂµL in DMSO) were then added to each test well at a final concentration of 40Â Âµg/mL. The plates were incubated for 18â€"20 hr at 37Â Â°C. At the end of this incubation, resazurin (Sigma-Aldrich) was added to the wells to determine the growth of bacteria. The final concentration was 0.02%, and the plates were further incubated for 4-6 hr at 37Â Â°C. In the presence of viable cells, resazurin was reduced to resorufin (pink) along with an increase in fluorescence (<a href="#ref-25">Oâ€™Brien et al., 2000</a>). Extracts showing antibiotic activity (hits) were scored as those that prevented the color change and also reduced the fluorescence (Ex 540, Em 590nm) by 90% when compared to the control wells containing no inhibitor. Ciprofloxacin was used as a positive control for bacterial growth inhibition. Three wells were used for each sample. For the positive candidate extracts, we repeated the experiments at least once to confirm the results.</p> </div> <div class="screeningthefungalextractlibrarytoidentifydrugsleadsagainstchronicmyeloidleukemiawithmttassays" title="sec"> <div class="title" tagx="title" title="title">
Screening the fungal extract library to identify drugs leads against chronic myeloid leukemia with MTT assays</div> <p>Cell proliferation was analyzed by the 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl tetrazolium bromide (MTT) using Vybrant<sup>Â®</sup> MTT Cell Proliferation Assay (Thermo Fisher Scientific) with the human immortalized myelogenous leukemia cell line K562. Around 2Â Ã—Â 10<sup>4</sup> cells in 100Â ÂµL culture medium (RPMI 1640 + 2mM glutamine + 10% fetal bovine serum) were seeded in 96-well microplates and incubated at 37Â Â°C with 5% CO<sub>2</sub>. The next day, oneÂ ÂµL fungal extract in DMSO was added (final concentration of the fungal extract was 20Â Âµg/mL), and the cells were incubated at 37Â Â°C with 5% CO<sub>2</sub> for another 24 hr. Next, the microplate was centrifuged at 500 g for 10 min to pellet the cells, the medium was carefully removed as much as possible, and 100Â ÂµL of fresh medium was then added. About 10Â ÂµL of the 12 mM MTT stock solution was added, mixed, and incubated for 4 hr at 37Â Â°C. The microplate was centrifuged again at 500 g for 10 m. After removing 75Â ÂµL of the medium from the wells with 25Â ÂµL medium with cells left, 50Â ÂµL of DMSO was added to each well, mixed, and incubated at 37Â Â°C for 10 min to dissolve formazan crystal for measurement. The same amount of DMSO without drugs was applied as a control. The optical density was measured at an absorbance wavelength of 540 nm. Cell growth inhibition rate (%) = (A<sub>540</sub> of control âˆ’A<sub>540</sub> of treatment/ A<sub>540</sub> of control)Â Ã—100%. Triplicates were conducted for each sample.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>All the experiments were independently repeated at least twice and analyzed with the Wilcoxon-Mannâ€"Whitney test using GraphPad Prism (GraphPad Software, CA, USA).</p> </div> <div class="sequenceavailability" title="sec"> <div class="title" tagx="title" title="title">
Sequence availability</div> <p>All fungal ITS sequences obtained in this project have been deposited into GenBank at NCBI (<a href="https://www.ncbi.nlm.nih.gov/genbank/sequenceids/">https://www.ncbi.nlm.nih.gov/genbank/sequenceids/</a>).</p> </div> <div class="constructionofphylogenetictree" title="sec"> <div class="title" tagx="title" title="title">
Construction of phylogenetic tree</div> <p>Forty fungal ITS sequences in FASTA format were input into an online multiple sequence alignment tool (<a href="https://www.ebi.ac.uk/Tools/msa/clustalo/">https://www.ebi.ac.uk/Tools/msa/clustalo/</a>) using Clustal Omega algorithm (<a href="#ref-11">Higgins &amp;amp; Sharp, 1988</a>). The parameter â€œDNAâ€� was selected for input sequence and â€œClustalWâ€� was selected as output format. All other parameters were kept as defaults. Multiple sequence alignment was conducted by clicking on â€œSubmitâ€�. After alignment was completed, the tab of â€œPhylogenetic Treeâ€� was clicked and the checkbox â€œRealâ€� was selected for â€œBranch lengthâ€� to visualize the phylogenetic tree. The phylogenetic tree was saved as a pdf file through â€œprintâ€� under â€œFileâ€�.</p> </div> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="extensivefungallibrarywithnearlytenthousandisolates" title="sec"> <div class="title" tagx="title" title="title">
Extensive fungal library with nearly ten thousand isolates</div> <p>A large and diverse fungal library is powerful in discovering new drugs. To achieve this goal, we collected samples globally. Current collection includes samples from Kenya, Myanmar, USA, and China. We received about 2395 soil samples and 2324 plant samples. We collected the whole plant or separated plant parts such as roots, stems, leaves, flowers, fruits, or various combinations of components. The samples were from 36 regions, including Nairobi in Kenya, 10 regions in the United States, Yangon in Myanmar, and 24 districts in China (<a href="#table-2">Table 2</a>). The collection places cover various climate zones.</p> <p>From these samples, 9,053 fungal isolates in total were cultured. Among them, 2,356 were from the plant samples, and 6,688 were from soil samples. About one fungal strain per plant-part sample and 2.8 fungal isolates per soil sample were obtained by average. Nearly 69.4% of fungal isolates were from the subtropical climate in China (Shanghai, Guangzhou, and Chongqing) and the USA (e.g.,Â Dallas, New Orleans, and Oklahoma City). About 8% of fungal isolates were from tropical climates such as Miami in the USA, Yangon in Myanmar, Kisumu in Kenya. Approximate 4% were from tropical/subtropical highland climate areas such as Nairobi in Kenya, the Lijiang National Park, and the Potatso National Park in China. A small portion (2.1%) was from the cold areas, such as Jiuzhaigou National Park in China, and Alaska in the USA (<a href="#fig-1">Fig. 1A</a>). We collected the samples from different landforms, including hills, mountains, plateaus, canyons, valleys, and bays. Notably, some fungi were isolated from samples collected from the mountains with an altitude over 3,000Â m, such as the Cang Mountains and Meili Snow Mountains in the Yunnan Province, China. The vast difference of sample collection in location, weather, climate, and altitude (<a href="#fig-1">Fig. 1B</a>) promises the diversity of fungal species and their genetic background.</p> <p>To maximize the diversity and quantity in our fungal library, we selected fungal colonies based on their location, color, and morphology on culture plates. Therefore, the fungal isolates in the library look strikingly different (<a href="#fig-1">Fig. 1C</a>). This visible criterion facilitates the fungal library construction. However, we also discarded many fungi that are different species with similar morphology. We examined the species diversity of our fungal library at the molecular level by randomly picking 40 fungal isolates. Their genomic DNA was isolated, the ITS regions were amplified, and PCR products were sequenced. These sequences have been deposited into GenBank at <a href="https://www.ncbi.nlm.nih.gov/genbank/sequenceids/">https://www.ncbi.nlm.nih.gov/genbank/sequenceids/</a>. Their Accession numbers are <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594355">MT594355</a>â€"<a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594393">MT594393</a> and <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT584204">MT584204</a>. We searched these sequences against NCBI DNA databases using blast. Results show that about 12.5% of total fungal isolates have identical ITS sequences to others in the library (<a href="#table-3">Table 3</a>). For instance, 126-G10, 117-B9, and 45-F10 have identical ITS to <i>Fusarium solani</i>, and 2 of 3 might be duplicates. Strains 99-H5 and 78-D10 have identical ITS to <i>Penicillium sclerotiorum</i>. More than 80% of fungal isolates belong to different species or strains, indicating the fungal library is highly diverse. A small portion (&amp;lt;12.5%) are duplications of the other. Based on these ITS sequences, a phylogenetic tree was constructed, displaying the fungal diversity in samples (<a href="#supplemental-information">File S1</a>). Three genera (<i>Trichoderma</i>, <i>Fusarium</i> and <i>Penicillium</i>) present in three big branches, which is consistent to our sampling sources, e.g.,Â soil and plants. <i>Trichoderma</i> and <i>Fusarium</i> are the most prevalent soil fungi and many are associated with plants (<a href="#ref-9">Harman et al., 2004</a>). <i>Penicillium</i> is ubiquitous genus with more than 350 species already identified (<a href="#ref-33">Visagie et al., 2014</a>).</p> <div class="table-wrap_UNKNOWN" id="table-2" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7717/peerj.10392/table-2">10.7717/peerj.10392/table-2</a><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Fungal strains collected from different locations of the world.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1"># of isolates</th> <th rowspan="1" colspan="1">Area/City/County</th> <th rowspan="1" colspan="1">State/Province</th> <th rowspan="1" colspan="1">Country</th> <th rowspan="1" colspan="1">Continent</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">188</td> <td rowspan="1" colspan="1">Norman</td> <td rowspan="1" colspan="1">Oklahoma</td> <td rowspan="1" colspan="1">USA</td> <td rowspan="1" colspan="1">North America</td> </tr> <tr> <td rowspan="1" colspan="1">287</td> <td rowspan="1" colspan="1">Oklahoma City</td> <td rowspan="1" colspan="1">Oklahoma</td> <td rowspan="1" colspan="1">USA</td> <td rowspan="1" colspan="1">North America</td> </tr> <tr> <td rowspan="1" colspan="1">208</td> <td rowspan="1" colspan="1">Stillwater</td> <td rowspan="1" colspan="1">Oklahoma</td> <td rowspan="1" colspan="1">USA</td> <td rowspan="1" colspan="1">North America</td> </tr> <tr> <td rowspan="1" colspan="1">76</td> <td rowspan="1" colspan="1">Dallas</td> <td rowspan="1" colspan="1">Texas</td> <td rowspan="1" colspan="1">USA</td> <td rowspan="1" colspan="1">North America</td> </tr> <tr> <td rowspan="1" colspan="1">68</td> <td rowspan="1" colspan="1">New Orleans</td> <td rowspan="1" colspan="1">Louisiana</td> <td rowspan="1" colspan="1">USA</td> <td rowspan="1" colspan="1">North America</td> </tr> <tr> <td rowspan="1" colspan="1">59</td> <td rowspan="1" colspan="1">Pensacola</td> <td rowspan="1" colspan="1">Florida</td> <td rowspan="1" colspan="1">USA</td> <td rowspan="1" colspan="1">North America</td> </tr> <tr> <td rowspan="1" colspan="1">47</td> <td rowspan="1" colspan="1">Tallahassee</td> <td rowspan="1" colspan="1">Florida</td> <td rowspan="1" colspan="1">USA</td> <td rowspan="1" colspan="1">North America</td> </tr> <tr> <td rowspan="1" colspan="1">33</td> <td rowspan="1" colspan="1">Lake City</td> <td rowspan="1" colspan="1">Florida</td> <td rowspan="1" colspan="1">USA</td> <td rowspan="1" colspan="1">North America</td> </tr> <tr> <td rowspan="1" colspan="1">401</td> <td rowspan="1" colspan="1">Miami</td> <td rowspan="1" colspan="1">Florida</td> <td rowspan="1" colspan="1">USA</td> <td rowspan="1" colspan="1">North America</td> </tr> <tr> <td rowspan="1" colspan="1">119</td> <td rowspan="1" colspan="1">Juneau</td> <td rowspan="1" colspan="1">Alaska</td> <td rowspan="1" colspan="1">USA</td> <td rowspan="1" colspan="1">North America</td> </tr> <tr> <td rowspan="1" colspan="1">702</td> <td rowspan="1" colspan="1">Nairobi</td> <td rowspan="1" colspan="1">NA</td> <td rowspan="1" colspan="1">Kenya</td> <td rowspan="1" colspan="1">Africa</td> </tr> <tr> <td rowspan="1" colspan="1">322</td> <td rowspan="1" colspan="1">Yangon</td> <td rowspan="1" colspan="1">NA</td> <td rowspan="1" colspan="1">Myanmar</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">181</td> <td rowspan="1" colspan="1">Shanghai</td> <td rowspan="1" colspan="1">NA</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">34</td> <td rowspan="1" colspan="1">Beijing</td> <td rowspan="1" colspan="1">NA</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">39</td> <td rowspan="1" colspan="1">Chongqing</td> <td rowspan="1" colspan="1">NA</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">1469</td> <td rowspan="1" colspan="1">Foshan</td> <td rowspan="1" colspan="1">Guangdong</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">472</td> <td rowspan="1" colspan="1">Guangzhou</td> <td rowspan="1" colspan="1">Guangdong</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">1093</td> <td rowspan="1" colspan="1">South China Botanical Garden</td> <td rowspan="1" colspan="1">Guangdong</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">667</td> <td rowspan="1" colspan="1">Taishan</td> <td rowspan="1" colspan="1">Guangdong</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">37</td> <td rowspan="1" colspan="1">Zhongshan</td> <td rowspan="1" colspan="1">Guangdong</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">45</td> <td rowspan="1" colspan="1">Jiangmen</td> <td rowspan="1" colspan="1">Guangdong</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">14</td> <td rowspan="1" colspan="1">Maoming</td> <td rowspan="1" colspan="1">Guangdong</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">351</td> <td rowspan="1" colspan="1">Shaoguan</td> <td rowspan="1" colspan="1">Guangdong</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">34</td> <td rowspan="1" colspan="1">Shenzhen</td> <td rowspan="1" colspan="1">Guangdong</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">356</td> <td rowspan="1" colspan="1">Laibin</td> <td rowspan="1" colspan="1">Guangxi</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">54</td> <td rowspan="1" colspan="1">Liuzhou</td> <td rowspan="1" colspan="1">Guangxi</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">39</td> <td rowspan="1" colspan="1">Rongan</td> <td rowspan="1" colspan="1">Guangxi</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">808</td> <td rowspan="1" colspan="1">Zhongshan</td> <td rowspan="1" colspan="1">Guangxi</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">46</td> <td rowspan="1" colspan="1">Qinzhou</td> <td rowspan="1" colspan="1">Guangxi</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">135</td> <td rowspan="1" colspan="1">Lijiang</td> <td rowspan="1" colspan="1">Yunnan</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">94</td> <td rowspan="1" colspan="1">Cang Mountain</td> <td rowspan="1" colspan="1">Yunnan</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">254</td> <td rowspan="1" colspan="1">XishuangbannaTropical Botanical Garden</td> <td rowspan="1" colspan="1">Yunnan</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">77</td> <td rowspan="1" colspan="1">Potatso National Park</td> <td rowspan="1" colspan="1">Yunnan</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">50</td> <td rowspan="1" colspan="1">Tongzi</td> <td rowspan="1" colspan="1">Guizhou</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">38</td> <td rowspan="1" colspan="1">Jiuzhaigou National Park</td> <td rowspan="1" colspan="1">Sichuan</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">132</td> <td rowspan="1" colspan="1">Xinglong Tropical Botanical Garden</td> <td rowspan="1" colspan="1">Hainan</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">Hohhot</td> <td rowspan="1" colspan="1">Inner Mongolia</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> <tr> <td rowspan="1" colspan="1">21</td> <td rowspan="1" colspan="1">Changzhi</td> <td rowspan="1" colspan="1">Shanxi</td> <td rowspan="1" colspan="1">China</td> <td rowspan="1" colspan="1">Asia</td> </tr> </tbody> </table> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10392/fig-1"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Worldwide localization of samples, distribution of collected fungi in different climate zones, and some fungal morphology.</div> <p>(A) Samples were collected worldwide, indicated by red dots. (B) Distribution of the collected fungi in different climates zones. The climate classification was based on the KÃ¶ppenâ€"Geiger Climate Classification. The codes of the climate are as the following: Aw: Tropical monsoon climate; Bsk: Cold semi-arid climate; Cfa: Humid subtropical climate; Cfb: Temperate oceanic climate; Cwa: Monsoon-influenced humid subtropical climate; Cwb: Subtropical highland climate or Monsoon-influenced temperate marine climate; Dfb: Humid continental climate; Dwa: Monsoon-influenced hot-summer humid continental climate; Dwb: Monsoon-influenced warm-summer humid continental climate. (C) Morphology of some isolated fungal colonies in the library.</p>   </div> </div> <div class="endophyticfungifromchinesemedicinalplants" title="sec"> <div class="title" tagx="title" title="title">
Endophytic fungi from Chinese medicinal plants</div> <p>Our fungal library includes many endophytic fungi. Since endophytic fungi produce many plant metabolites with medical functions, we collected plant samples from the three most extensive tropical botanic gardens in China, including South China, Hainan Xinglong, and Yunnan Tropical. There are many diverse plants in these gardens. We collected 27 well-known Chinese medicinal plants, such as Chinese black olive (<i>Canarium pimela</i>), Chinese croton (<i>Excoecaria cochinchinensis</i>), Lemon-scented gum (<i>Eucalyptus citriodora Hook</i>), Sweet osmanthus (<i>Osmanthus fragrans</i>), and Yellow cow wood (<i>Cratoxylum cochinchinense</i>) (<a href="#table-4">Table 4</a>). We separated different parts from each plant, e.g.,Â leaf, bark, stem, fruits, and seeds, and sterilized their surfaces with 75% ethanol. Following grinding masses and culturing on Malt Extract Agar (MEA) plates, more than 50 endophytic fungi were isolated on MEA plates. The colors and morphology of these endophytic fungi and their corresponding plants look strikingly different (<a href="#fig-2">Fig. 2</a>). The ITS of three fungi were PCR-amplified and sequenced (Accession # in GenBank are <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT994711">MT994711</a>, <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT994712">MT994712</a>, and <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594489">MT594489</a>). They were identified as <i>Stephanonectria keithii</i> (<a href="#fig-2">Fig. 2C</a>), <i>Aspergillus</i> sp<i>.</i> (<a href="#fig-2">Fig.Â 2E</a>), <i>and Tolypocladium album</i> (<a href="#fig-2">Fig. 2Z</a>), respectively.</p> <div class="table-wrap_UNKNOWN" id="table-3" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7717/peerj.10392/table-3">10.7717/peerj.10392/table-3</a><span class="label" tagx="label" title="label">Table 3</span> <div class="title" tagx="title" title="title">
The species of randomly sampled fungi from the library.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1">ID</th> <th rowspan="1" colspan="1">Species</th> <th rowspan="1" colspan="1">Length</th> <th rowspan="1" colspan="1">Coverage</th> <th rowspan="1" colspan="1">Identity</th> <th rowspan="1" colspan="1">Source</th> <th rowspan="1" colspan="1">Sample location</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">11-A5</td> <td rowspan="1" colspan="1"><i>Acremonium cellulolyticus</i></td> <td rowspan="1" colspan="1">581</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangxi, China</td> </tr> <tr> <td rowspan="1" colspan="1">S3/2</td> <td rowspan="1" colspan="1"><i>Albifimbria verrucaria</i></td> <td rowspan="1" colspan="1">582</td> <td rowspan="1" colspan="1">98%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">116-E12</td> <td rowspan="1" colspan="1"><i>Arthropsis hispanica</i></td> <td rowspan="1" colspan="1">455</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Florida, USA</td> </tr> <tr> <td rowspan="1" colspan="1">4-H9</td> <td rowspan="1" colspan="1"><i>Ascomycota</i> sp.</td> <td rowspan="1" colspan="1">586</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">plant</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">64-A1</td> <td rowspan="1" colspan="1"><i>Aspergillus flavus</i></td> <td rowspan="1" colspan="1">597</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">plant</td> <td rowspan="1" colspan="1">Oklahoma, USA</td> </tr> <tr> <td rowspan="1" colspan="1">79-B7</td> <td rowspan="1" colspan="1"><i>Aspergillus</i> sp.</td> <td rowspan="1" colspan="1">574</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">49-G11</td> <td rowspan="1" colspan="1"><i>Aspergillus sydowii</i></td> <td rowspan="1" colspan="1">569</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">air</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">58-C1</td> <td rowspan="1" colspan="1"><i>Aspergillus</i> sp.</td> <td rowspan="1" colspan="1">621</td> <td rowspan="1" colspan="1">92%</td> <td rowspan="1" colspan="1">77%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">3-D7</td> <td rowspan="1" colspan="1"><i>Cylindrocladium</i> sp.</td> <td rowspan="1" colspan="1">575</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">24-C5</td> <td rowspan="1" colspan="1"><i>Debaryomyces subglobosus</i></td> <td rowspan="1" colspan="1">634</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">116-H3</td> <td rowspan="1" colspan="1"><i>Epicoccum sorghinum</i></td> <td rowspan="1" colspan="1">589</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">plant</td> <td rowspan="1" colspan="1">Guangxi, China</td> </tr> <tr> <td rowspan="1" colspan="1">88-E10</td> <td rowspan="1" colspan="1"><i>Fusarium kyushuense</i></td> <td rowspan="1" colspan="1">540</td> <td rowspan="1" colspan="1">95%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">126-G10</td> <td rowspan="1" colspan="1"><i>Fusarium solani</i></td> <td rowspan="1" colspan="1">573</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangxi, China</td> </tr> <tr> <td rowspan="1" colspan="1">117-B9</td> <td rowspan="1" colspan="1"><i>Fusarium solani</i></td> <td rowspan="1" colspan="1">576</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Florida, USA</td> </tr> <tr> <td rowspan="1" colspan="1">45-F10</td> <td rowspan="1" colspan="1"><i>Fusarium solani</i></td> <td rowspan="1" colspan="1">573</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">3-F5</td> <td rowspan="1" colspan="1"><i>Fusarium verticillioides</i></td> <td rowspan="1" colspan="1">562</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangxi, China</td> </tr> <tr> <td rowspan="1" colspan="1">18-F5</td> <td rowspan="1" colspan="1"><i>Trichoderma</i> sp.</td> <td rowspan="1" colspan="1">619</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">92%</td> <td rowspan="1" colspan="1">plant</td> <td rowspan="1" colspan="1">Hainan, China</td> </tr> <tr> <td rowspan="1" colspan="1">66-B4</td> <td rowspan="1" colspan="1"><i>Metarhizium</i> sp.</td> <td rowspan="1" colspan="1">617</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Yunnan, China</td> </tr> <tr> <td rowspan="1" colspan="1">17-A1</td> <td rowspan="1" colspan="1"><i>Mycosphaerella</i> sp.</td> <td rowspan="1" colspan="1">573</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Alaska, USA</td> </tr> <tr> <td rowspan="1" colspan="1">HW</td> <td rowspan="1" colspan="1"><i>Neurospora</i> sp.</td> <td rowspan="1" colspan="1">586</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">air</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">74-F11</td> <td rowspan="1" colspan="1"><i>Penicillium rolfsii</i></td> <td rowspan="1" colspan="1">591</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">99-H4</td> <td rowspan="1" colspan="1"><i>Penicillium sclerotiorum</i></td> <td rowspan="1" colspan="1">586</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">plant</td> <td rowspan="1" colspan="1">Guangxi, China</td> </tr> <tr> <td rowspan="1" colspan="1">78-D10</td> <td rowspan="1" colspan="1"><i>Penicillium sclerotiorum</i></td> <td rowspan="1" colspan="1">583</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">81-D8</td> <td rowspan="1" colspan="1"><i>Penicillium</i> sp.</td> <td rowspan="1" colspan="1">575</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">107-H3</td> <td rowspan="1" colspan="1"><i>Penicillium soppii</i></td> <td rowspan="1" colspan="1">470</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Alaska, USA</td> </tr> <tr> <td rowspan="1" colspan="1">125-B10</td> <td rowspan="1" colspan="1"><i>Penicillium</i> sp.</td> <td rowspan="1" colspan="1">589</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">114-D12</td> <td rowspan="1" colspan="1"><i>Penicillium</i> sp.</td> <td rowspan="1" colspan="1">584</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">plant</td> <td rowspan="1" colspan="1">Yangon, Myanmer</td> </tr> <tr> <td rowspan="1" colspan="1">3-G10</td> <td rowspan="1" colspan="1"><i>Talaromyces stipitatus</i></td> <td rowspan="1" colspan="1">582</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangxi, China</td> </tr> <tr> <td rowspan="1" colspan="1">37-A6</td> <td rowspan="1" colspan="1"><i>Trichoderma</i> sp.</td> <td rowspan="1" colspan="1">612</td> <td rowspan="1" colspan="1">83%</td> <td rowspan="1" colspan="1">86%</td> <td rowspan="1" colspan="1">plant</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">95-C6</td> <td rowspan="1" colspan="1"><i>Trichoderma</i> sp.</td> <td rowspan="1" colspan="1">571</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangxi, China</td> </tr> <tr> <td rowspan="1" colspan="1">45-G11</td> <td rowspan="1" colspan="1"><i>Trichoderma atroviride</i></td> <td rowspan="1" colspan="1">629</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">CL</td> <td rowspan="1" colspan="1"><i>Trichoderma</i> sp.</td> <td rowspan="1" colspan="1">612</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Maasi Marla, Kenya</td> </tr> <tr> <td rowspan="1" colspan="1">16-D5</td> <td rowspan="1" colspan="1"><i>Trichoderma harzianum</i></td> <td rowspan="1" colspan="1">625</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">100-D3</td> <td rowspan="1" colspan="1"><i>Trichoderma</i> sp.</td> <td rowspan="1" colspan="1">554</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">104-D12</td> <td rowspan="1" colspan="1"><i>Trichoderma harzianum</i></td> <td rowspan="1" colspan="1">623</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangxi, China</td> </tr> <tr> <td rowspan="1" colspan="1">98-C8</td> <td rowspan="1" colspan="1"><i>Trichoderma</i> sp.</td> <td rowspan="1" colspan="1">610</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangxi, China</td> </tr> <tr> <td rowspan="1" colspan="1">80-G9</td> <td rowspan="1" colspan="1"><i>Trichoderma</i> sp.</td> <td rowspan="1" colspan="1">622</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">114-G8</td> <td rowspan="1" colspan="1"><i>Trichoderma</i> sp.Â </td> <td rowspan="1" colspan="1">617</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">99%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Guangdong, China</td> </tr> <tr> <td rowspan="1" colspan="1">117-H9</td> <td rowspan="1" colspan="1"><i>Trichoderma harzianum</i></td> <td rowspan="1" colspan="1">624</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">soil</td> <td rowspan="1" colspan="1">Florida, USA</td> </tr> <tr> <td rowspan="1" colspan="1">121-G6</td> <td rowspan="1" colspan="1"><i>Metarhizium carneum</i></td> <td rowspan="1" colspan="1">172</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">100%</td> <td rowspan="1" colspan="1">plant</td> <td rowspan="1" colspan="1">Hainan, China</td> </tr> </tbody> </table> </div> </div> <div class="table-wrap_UNKNOWN" id="table-4" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7717/peerj.10392/table-4">10.7717/peerj.10392/table-4</a><span class="label" tagx="label" title="label">Table 4</span> <div class="title" tagx="title" title="title">
Some fungal strains isolated from Chinese medicinal plants.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1">ID</th> <th rowspan="1" colspan="1">Sample collection location</th> <th rowspan="1" colspan="1">Medical plant/tissue</th> <th rowspan="1" colspan="1">Family</th> <th rowspan="1" colspan="1">Species</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">12A7</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Chinese black olive/stem</td> <td rowspan="1" colspan="1">Burseraceae</td> <td rowspan="1" colspan="1"><i>Canarium pimela</i></td> </tr> <tr> <td rowspan="1" colspan="1">30F12</td> <td rowspan="1" colspan="1">SCBG, Guanzhou, China</td> <td rowspan="1" colspan="1">Devilâ€™s Trumpet/leaf</td> <td rowspan="1" colspan="1">Solanaceae</td> <td rowspan="1" colspan="1"><i>Datura metel</i></td> </tr> <tr> <td rowspan="1" colspan="1">32F1</td> <td rowspan="1" colspan="1">SCBG, Guanzhou, China</td> <td rowspan="1" colspan="1">Japanese lantern</td> <td rowspan="1" colspan="1">Malvaceae</td> <td rowspan="1" colspan="1"><i>Hibiscus schizopetalus</i></td> </tr> <tr> <td rowspan="1" colspan="1">38H8</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Sweet osmanthus/leaf</td> <td rowspan="1" colspan="1">Oleaceae</td> <td rowspan="1" colspan="1"><i>Osmanthus fragrans (Thunb)</i></td> </tr> <tr> <td rowspan="1" colspan="1">42B7</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Lemon-scented gum/bark</td> <td rowspan="1" colspan="1">Myrtaceae</td> <td rowspan="1" colspan="1"><i>Eucalyptus citriodoraÂ Hook.f</i></td> </tr> <tr> <td rowspan="1" colspan="1">44G1</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Sausage Tree/leaf</td> <td rowspan="1" colspan="1">Bignoniaceae</td> <td rowspan="1" colspan="1"><i>Kigelia africana (Lam.)Â Benth.</i></td> </tr> <tr> <td rowspan="1" colspan="1">44G9</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Chinese croton/leaf</td> <td rowspan="1" colspan="1">Euphorbiaceae</td> <td rowspan="1" colspan="1"><i>Excoecaria cochinchinensis</i></td> </tr> <tr> <td rowspan="1" colspan="1">71G5</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1"><i>Maytenus hookeri</i>Â Loes/leaf</td> <td rowspan="1" colspan="1">Celastraceae</td> <td rowspan="1" colspan="1"><i>Maytenus hookeriÂ Loes.</i></td> </tr> <tr> <td rowspan="1" colspan="1">71G11</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Siris tree/leaf</td> <td rowspan="1" colspan="1">Fabaceae</td> <td rowspan="1" colspan="1"><i>Albizia lebbeck</i></td> </tr> <tr> <td rowspan="1" colspan="1">74E7</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Yellow cow wood/leaf</td> <td rowspan="1" colspan="1">Hypericaceae</td> <td rowspan="1" colspan="1"><i>Cratoxylum cochinchinense</i></td> </tr> <tr> <td rowspan="1" colspan="1">74E10</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1"><i>Dracontomelon duperreanum</i>/leaf</td> <td rowspan="1" colspan="1">Anacardiaceae</td> <td rowspan="1" colspan="1"><i>Dracontomelon duperreanum</i></td> </tr> <tr> <td rowspan="1" colspan="1">35H11</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Harland boxwood/leaf</td> <td rowspan="1" colspan="1">Buxaceae</td> <td rowspan="1" colspan="1"><i>Buxus microphylla</i></td> </tr> <tr> <td rowspan="1" colspan="1">9D1</td> <td rowspan="1" colspan="1">HXTBG, Haikou, China</td> <td rowspan="1" colspan="1">Sisal/leaf</td> <td rowspan="1" colspan="1">Asparagaceae</td> <td rowspan="1" colspan="1"><i>Agave sisalana</i></td> </tr> <tr> <td rowspan="1" colspan="1">9H3</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Candlenut/leaf</td> <td rowspan="1" colspan="1">Euphorbiaceae</td> <td rowspan="1" colspan="1"><i>Aleurites moluccanus</i></td> </tr> <tr> <td rowspan="1" colspan="1">9H7</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Kadsura vine/leaf</td> <td rowspan="1" colspan="1">Schisandraceae</td> <td rowspan="1" colspan="1"><i>Kadsura japonica</i></td> </tr> <tr> <td rowspan="1" colspan="1">21B3</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Castor oil plant/fruit</td> <td rowspan="1" colspan="1">Euphorbiaceae</td> <td rowspan="1" colspan="1"><i>Ricinus communis</i></td> </tr> <tr> <td rowspan="1" colspan="1">29A7</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Soft bollygum/leaf</td> <td rowspan="1" colspan="1">Lauraceae</td> <td rowspan="1" colspan="1"><i>Litsea glutinosa</i></td> </tr> <tr> <td rowspan="1" colspan="1">37E10</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Sweet osmanthus/leaf</td> <td rowspan="1" colspan="1">Oleaceae</td> <td rowspan="1" colspan="1"><i>Osmanthus fragrans</i></td> </tr> <tr> <td rowspan="1" colspan="1">79A7</td> <td rowspan="1" colspan="1">HXTBG, Haikou, China</td> <td rowspan="1" colspan="1">Thyme/leaf</td> <td rowspan="1" colspan="1">Lamiaceae</td> <td rowspan="1" colspan="1"><i>Thymus mongolicus Ronn</i></td> </tr> <tr> <td rowspan="1" colspan="1">79A11</td> <td rowspan="1" colspan="1">Taishan, China</td> <td rowspan="1" colspan="1"><i>Sterculia nobilis</i>/leaf</td> <td rowspan="1" colspan="1">Malvaceae</td> <td rowspan="1" colspan="1"><i>SterculiaÂ nobilisÂ Smith</i></td> </tr> <tr> <td rowspan="1" colspan="1">79F5</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Milk tree/leaf</td> <td rowspan="1" colspan="1">Moraceae</td> <td rowspan="1" colspan="1"><i>Ficus hispidaÂ L. f.</i></td> </tr> <tr> <td rowspan="1" colspan="1">82C1</td> <td rowspan="1" colspan="1">HXTBG, Haikou, China</td> <td rowspan="1" colspan="1">Sessileflower/leaf</td> <td rowspan="1" colspan="1">Araliaceae</td> <td rowspan="1" colspan="1"><i>Acanthopanax gracilistylus</i></td> </tr> <tr> <td rowspan="1" colspan="1">72E3</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Trumpet tree/leaf</td> <td rowspan="1" colspan="1">Urticaceae</td> <td rowspan="1" colspan="1"><i>Cecropia peltata</i></td> </tr> <tr> <td rowspan="1" colspan="1">81F7</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">Callicarpa brevipes/fruit</td> <td rowspan="1" colspan="1">Lamiaceae</td> <td rowspan="1" colspan="1"><i>Callicarpa brevipes</i></td> </tr> <tr> <td rowspan="1" colspan="1">78A11</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">bollygum /leaf</td> <td rowspan="1" colspan="1">Lauraceae</td> <td rowspan="1" colspan="1"><i>Litsea glutinosa</i></td> </tr> <tr> <td rowspan="1" colspan="1">78A10</td> <td rowspan="1" colspan="1">SCBG, Guangzhou, China</td> <td rowspan="1" colspan="1">African oil palm/leaf</td> <td rowspan="1" colspan="1">Arecaceae</td> <td rowspan="1" colspan="1"><i>Elaeis guineensis</i></td> </tr> <tr> <td rowspan="1" colspan="1">126H5</td> <td rowspan="1" colspan="1">HXTBG, Haikou, China</td> <td rowspan="1" colspan="1">Giant crepe-myrtle/bark</td> <td rowspan="1" colspan="1">Lythraceae</td> <td rowspan="1" colspan="1"><i>Lagerstroemia speciosa</i></td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><b>Notes.</b></p> </div> <div class="fn-type-other" title="other"> <p> </p><div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
 SCBG</div> <div class="def" title="def"> <p>South China Botanical Garden</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
 HXTBG</div> <div class="def" title="def"> <p>Hainan Xinglong Tropical Botanical Garden</p> </div> </div> </div> <p /> </div> </div> </div> </div> <div class="constructionofafungalextractlibrary" title="sec"> <div class="title" tagx="title" title="title">
Construction of a fungal extract library</div> <p>To generate a fungal metabolite library, we used the cereal-based medium to produce the secondary metabolites as reported (<a href="#ref-24">Niu et al., 2015</a>). Each fungus was cultured in a test tube with six small pieces of cereals. After culturing for one month, we used ethyl acetate to extract the secondary metabolites. We obtained 9,053 crude extracts in total, each of which corresponds to a specific fungal isolate. As anticipated, different fungal strains produced different amounts of secondary metabolites from 1 mg to 20 mg per gram culture, and have various physical features such as stickiness, odors, and solubility. More than 90% of the extracts have colors, including green, orange, red, yellow, purple, and others. The crude extracts were dissolved in DMSO to generate 2 mg/mL solution. For future reference, we named this library â€œGlobal Fungal Extract Libraryâ€� or GFEL in brief.</p> </div> <div class="screenthefungalextractlibraryagainstp.falciparumtransmissiontomosquitoes" title="sec"> <div class="title" tagx="title" title="title">
Screen the fungal extract library against <i>P. falciparum</i> transmission to mosquitoes</div> <p>Malaria remains a devastating disease, and <i>Anopheles</i> midgut protein fibrinogen-related protein 1 (FREP1) mediates <i>Plasmodium</i> transmission (<a href="#ref-18">Li et al. 2013</a>; <a href="#ref-35">Zhang et al., 2015</a>; <a href="#ref-22">Niu et al., 2017</a>). FREP1 mediates <i>Plasmodium</i> invasion in mosquitoes by binding to <i>P.Â falciparum</i> gametocytes or ookinetes (<a href="#ref-35">Zhang et al., 2015</a>).</p> <p>To examine the usability of the newly constructed fungal library, we screened 460 fungal extracts obtained in later 2016 and early 2017 for their inhibition activity against FREP1-<i>P.Â falciparum</i> interaction and found 4 extracts that prevented FREP1 from binding to <i>P.Â falciparum</i> lysates by over 90%. The fungal colonies of these four fungal candidates on MEA plates showed different colors and shapes (<a href="#fig-3">Figs. 3A</a>â€"<a href="#fig-3">3D</a>). Then, we determined the activities of the four candidates against <i>P.Â falciparum</i> to <i>An. gambiae in vivo</i> using the standard membrane feeding assays (SMFA). Results show that the fungal extracts of 37C6 and 22E8 could block malaria transmission at a concentration of 100Â Âµg/mL, and 100D3 and 45F10 did not (<a href="#fig-3">Fig. 3E</a>, <a href="#supplemental-information">File S2</a>). The oocyst numbers of 37C6 or 22E8 extract-treated mosquitoes were nearly zero, while the oocyst numbers in the 100D3 or 45F10 extract-treated mosquitoes were not significantly different from that of the DMSO control (<a href="#fig-3">Fig.Â 3E</a>). After further dilution of the two positive candidate fungal extracts (37C6 and 22E8) to 20 Âµg/mL, we found that the extract of 22E8 still significantly inhibited the activity in <i>P. falciparum</i> transmission to mosquitoes (<a href="#fig-3">Fig. 3F</a>, <a href="#supplemental-information">File S2</a>). We sequenced the ITS sequences of the four candidate fungi and their accession number are <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594486">MT594486</a>â€"<a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594488">MT594488</a> and <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT613342">MT613342</a> at GenBank at <a href="https://www.ncbi.nlm.nih.gov/genbank/sequenceids/">https://www.ncbi.nlm.nih.gov/genbank/sequenceids/</a>. According to ITS sequences of 22E8 (Acc #: <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT613342">MT613342</a>) and 37C6 (Acc#: <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594487">MT594487</a>), the fungal species of 22E8 and 37C6 were <i>Penicillium thomii</i> and <i>Penicillium pancosmium</i>, respectively (<a href="#table-5">Table 5</a>). Notably, this is the first report about <i>P. thomii</i> and <i>P. pancosmium</i> that produce secondary metabolites with antimalarial activities. An independent project in our lab identified <i>Asperaculane B</i> as an active compound from this GFEL that inhibited malaria transmission to mosquitoes (<a href="#ref-23">Niu et al., 2020</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10392/fig-2"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
The fungal isolates and the corresponding Chinese medicinal host plants.</div> <p>The images in Aâ€"I, Sâ€"AA and JJâ€"SS were the fugal isolates grown on MEA plates, and the photos below in Jâ€"R, BBâ€"II and TTâ€"BBB are the corresponding host plants where they were isolated.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10392/fig-3"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <div class="title" tagx="title" title="title">
The transmission-blocking activity of the extracts candidates by screening the fungal library with the <i>in vitro</i> FREP1-parasite interaction-based ELISA assays.</div> <p>(Aâ€"D) The morphology of fungal isolates 37C6, 100D3, 22E8T, and 45F10 on the MEA agar plate. (E) The final concentrations of the fungal extracts were 100 Âµg/mL and the results show that the fungal extracts (37C6 and 22E8) significantly reduced the oocyst number compared with the DMSO control while the oocyst number of the other two (100D3 and 45F10) was not significantly different with the DMSO control, respectively. (F) Further, the fungal extract of 22E8 continued to show a significant reduction of the oocyst number in midgut while the 37C6 fungal extract did not have significant effects on <i>P. falciparum</i> infection in mosquitoes when the concentration of the fungal extracts was decreased to 20 Âµg/mL. <b>N</b>: the number of mosquitoes for each treatment; <b>mean</b>: the average number of oocysts per midgut; <b>PR</b>: infection prevalence in mosquitoes. <b><i>p</i></b>: the p-value was calculated by the Mann-Whitney-Wilcoxon test. The experiments were repeated three times.</p>   </div> </div> <div class="examinetheusabilityoffungalextractlibraryinfindingantibioticleads" title="sec"> <div class="title" tagx="title" title="title">
Examine the usability of fungal extract library in finding antibiotic leads</div> <p>Antibiotic-resistant bacteria threaten public health (<a href="#ref-30">Todd, 2017</a>). We screened potential antibiotics from this newly established fungal library against antibiotic-resistant bacterial pathogens. We randomly picked 574 fungal extracts and examined their activity in inhibiting Gram-positive (methicillin-resistant <i>S. aureus</i> MRSA) and Gram-negative (<i>S. flexneri</i>) bacterial pathogens. These fungi were isolated from samples collected in late 2017. Among them, 47 inhibited the growth of <i>S. aureus</i> MRSA (hit rate of 10.8%), and one inhibited the growth of <i>S. flexneri</i> (hit rate of 0.17%, <a href="#table-6">Table 6</a>). The hit rate of antibiotics against Gram-positive bacteria was higher than that of the Gram-negative bacteria, which is consistent with the well-known challenges in antibiotic discovery against gram-negative pathogens. Gram-negative pathogens have unique outer membrane and efflux pumps (<a href="#ref-5">Fair &amp;amp; Tor, 2014</a>).</p> <p>We also examined the effect of 288 fungal extracts on non-methicillin-resistant <i>S. aureus</i>. The results showed that 22 prevented the growth of non-methicillin-resistant <i>S. aureus</i> (hit rate of 7.6%). Besides, we analyzed another two featured bacteria, <i>Mycobacterium smegmatis</i>, which is gram-positive and has acid-fast dye staining cell wall, and <i>E. coli</i>-AS17tolc, which has gram-negative cell wall, but more permeable than the wild type <i>E. coli</i>. We obtained eight candidates against <i>Mycobacterium smegmatis</i> and three candidates against <i>E. coli</i>-AS17tolc. The results showed 10.8% and 1% hit rates to <i>Mycobacterium smegmatis</i> and <i>E. coli</i>-AS17tolc, respectively (<a href="#table-6">Table 6</a> and <a href="#supplemental-information">File S3</a>). The results show that the secondary metabolites produced by different types of fungi in the GFEL contained biologically active compounds against drug-resistant and pathogenic strains of bacteria.</p> <div class="table-wrap_UNKNOWN" id="table-5" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7717/peerj.10392/table-5">10.7717/peerj.10392/table-5</a><span class="label" tagx="label" title="label">Table 5</span> <div class="title" tagx="title" title="title">
The fungal strains producing metabolites that inhibited malaria transmission.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1" /> <th colspan="2" rowspan="1"><a href="#table-5fn1"><sup>a</sup></a>Transmission blocking activity (%)</th> </tr> <tr> <th rowspan="1" colspan="1">ID</th> <th rowspan="1" colspan="1">Fungal species</th> <th rowspan="1" colspan="1">Access No</th> <th rowspan="1" colspan="1">100Â Âµg/mL</th> <th rowspan="1" colspan="1">20Â Âµg/mL</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">22E8</td> <td rowspan="1" colspan="1"><i>Penicillium thomii</i></td> <td rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT613342">MT613342</a></td> <td rowspan="1" colspan="1">99.4</td> <td rowspan="1" colspan="1">44.9</td> </tr> <tr> <td rowspan="1" colspan="1">100D3</td> <td rowspan="1" colspan="1"><i>Trichoderma harzianum</i></td> <td rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594486">MT594486</a></td> <td rowspan="1" colspan="1">NS</td> <td rowspan="1" colspan="1">NS</td> </tr> <tr> <td rowspan="1" colspan="1">37C6</td> <td rowspan="1" colspan="1"><i>Penicillium pancosmium</i></td> <td rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594487">MT594487</a></td> <td rowspan="1" colspan="1">99.3</td> <td rowspan="1" colspan="1">NS</td> </tr> <tr> <td rowspan="1" colspan="1">45F10</td> <td rowspan="1" colspan="1"><i>Fusarium solani</i></td> <td rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594488">MT594488</a></td> <td rowspan="1" colspan="1">NS</td> <td rowspan="1" colspan="1">NS</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><b>Notes.</b></p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">a</span><p>The transmission-blocking activity was calculated with the equation: (the mean oocyst number of the control groupâ€"the mean oocyst number of the treatment group)/the mean oocyst number of the control X100%.</p> </div> <div class="fn-type-other" title="other"> <p> </p><div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
 NS</div> <div class="def" title="def"> <p>Not significant</p> </div> </div> </div> <p /> </div> </div> </div> <div class="table-wrap_UNKNOWN" id="table-6" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7717/peerj.10392/table-6">10.7717/peerj.10392/table-6</a><span class="label" tagx="label" title="label">Table 6</span> <div class="title" tagx="title" title="title">
Antibiotic screening results show the positives and hit rates to various bacteria.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1"><i>Mycobacterium smegmatis</i></th> <th rowspan="1" colspan="1"><i>Staphylococcus aureus</i> (WT)</th> <th rowspan="1" colspan="1"><i>Staphylococcus aureus</i> (MRSA)</th> <th rowspan="1" colspan="1"><i>Shigella flexneri</i></th> <th rowspan="1" colspan="1"><i>E.coli</i> (AS17tolc)</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Total extracts screened</td> <td rowspan="1" colspan="1">574</td> <td rowspan="1" colspan="1">288</td> <td rowspan="1" colspan="1">574</td> <td rowspan="1" colspan="1">574</td> <td rowspan="1" colspan="1">288</td> </tr> <tr> <td rowspan="1" colspan="1">Number of positive hits</td> <td rowspan="1" colspan="1">62</td> <td rowspan="1" colspan="1">22</td> <td rowspan="1" colspan="1">47</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">3</td> </tr> <tr> <td rowspan="1" colspan="1">Hit rate (%)</td> <td rowspan="1" colspan="1">10.8</td> <td rowspan="1" colspan="1">7.6</td> <td rowspan="1" colspan="1">8.2</td> <td rowspan="1" colspan="1">0.17</td> <td rowspan="1" colspan="1">1</td> </tr> </tbody> </table> </div> </div> </div> <div class="examinetheusabilityoffungalextractlibraryinfindinganti-chronicmyeloidleukemiacandidates" title="sec"> <div class="title" tagx="title" title="title">
Examine the usability of fungal extract library in finding anti-chronic myeloid leukemia candidates</div> <p>Finally, we studied the possibility of obtaining drug candidate leads against chronic myelocytic leukemia (CML), a malignant tumor of the blood system (<a href="#ref-14">Kaleem et al., 2015</a>). A small subset of endophytic fungi from Chinese medicinal plants was used for this purpose. Fifty extracts were examined for their inhibition against the human immortalized myelogenous leukemia cell line K562 using the MTT method. The final concentrations of crude extracts were 20 Âµg/mL. The results from triplicates showed that 4 extracts (#17, #22, #29, #35) significantly inhibited the proliferation of K562 cells (<i>p</i>Â <div class="â_UNKNOWN" href="#fig-4">
Fig. 4, <a href="#supplemental-information">File S4</a>). The hit rate of the plant-fungal metabolite library was 8%. Notably, the survivorship of K562 cells with extract #29 was about 24.8% compared with the control, e.g., the inhibition of extract #29 on K562 proliferation was as high as 75.2%. The specimen of #29 was from the fungus (<a href="#fig-2">Fig. 2Z</a>) isolated from the plant <i>Litsea glutinosa</i> (<a href="#fig-2">Fig. 2HH</a>) collected from the Medical Botanical Garden of South China Botanical Garden in Guangzhou. <i>Litsea glutinosa</i> has been used to treat diarrhea, traumatic injuries, mumps, and rheumatism. It contains abundant flavonoids, terpenes, and alkaloids, and has good antibacterial activity, immunomodulatory effect, and anti-tumor effect. We PCR-amplified the ITS region of the fungal candidate (#29) and sequenced the region. Based on the ITS region sequence (Accession # at GenBank: <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594489">MT594489</a>), the candidate fungus was <i>Tolypocladium album</i>. Notably, a tetrameric acid from <i>Tolypocladium album</i> has been reported to inhibit the tumorâ€™s growth (<a href="#ref-7">Fukuda et al., 2015</a>). Further studies will be conducted to isolate and identify the bioactive compounds from this fungus.</div></p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10392/fig-4"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <div class="title" tagx="title" title="title">
Anti-chronic myeloid leukemia screening results.</div> <p>About 50 extracts of endophytic fungi isolated from the Chinese medicinal plant-fungal metabolite library were examined against K562 cells using the MTT method. The results show that 4 extracts (#17, #22, #29, #35) significantly inhibited the growth of K562 cells (<i>p</i>Â <div class="â_UNKNOWN">   </div></p></div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>We established a comprehensive fungal library that is diverse and useful for the scientific communities. We also demonstrated the usability of GFEL and identified a set of fungal strains that produce secondary metabolites to inhibit <i>Plasmodium falciparum</i>â€™s transmission, chronic pneumonia development, and bacteria proliferation. Further studies will identify the active compounds for drug development.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
 Supplemental Information</div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7717/peerj.10392/supp-1">10.7717/peerj.10392/supp-1</a><span class="label" tagx="label" title="label">Supplemental Information 1</span> <div class="title" tagx="title" title="title">
Phylogenetic tree of the 40 fungi</div> <p>The phylogenetic tree was constructed based on their ITS sequences. Three large genus groups predominantly presented in soils and associated with plants were labeled.</p>  <div class="media" title="media"><a href="peerj-08-10392-s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7717/peerj.10392/supp-2">10.7717/peerj.10392/supp-2</a><span class="label" tagx="label" title="label">Supplemental Information 2</span> <div class="title" tagx="title" title="title">
Numeric raw data show the number of oocysts in Fig 3B and 3C</div>  <div class="media" title="media"><a href="peerj-08-10392-s002.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7717/peerj.10392/supp-3">10.7717/peerj.10392/supp-3</a><span class="label" tagx="label" title="label">Supplemental Information 3</span> <div class="title" tagx="title" title="title">
Raw data show candidate fungal extracts against bacterial proliferation</div>  <div class="media" title="media"><a href="peerj-08-10392-s003.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7717/peerj.10392/supp-4">10.7717/peerj.10392/supp-4</a><span class="label" tagx="label" title="label">Supplemental Information 4</span> <div class="title" tagx="title" title="title">
Numeric raw data of Fig 4 show fungal extracts inhibiting K562 proliferation</div>  <div class="media" title="media"><a href="peerj-08-10392-s004.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7717/peerj.10392/supp-5">10.7717/peerj.10392/supp-5</a><span class="label" tagx="label" title="label">Supplemental Information 5</span> <div class="title" tagx="title" title="title">
Fungal ITS sequences</div> <p>All sequences but the accession number MT613342, stain name 22E8 are available at https://www.ncbi.nlm.nih.gov/nuccore/a ccess#. The ITS sequences for accession number MT613342 will be available in next GenBank release.</p>  <div class="media" title="media"><a href="peerj-08-10392-s005.txt">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank numerous residents and colleagues for collecting samples from various places and mailing the samples to us. We also appreciated many undergraduate students and technicians in culturing and isolating fungi.</p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional Information and Declarations</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Competing Interests</div> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors declare there are no competing interests.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Author Contributions</div> <div class="fn-type-con" title="con"> <p><a href="#author-1">Guodong Niu</a> and <a href="#author-8">Jun Li</a> conceived and designed the experiments, performed the experiments, analyzed the data, prepared figures and/or tables, authored or reviewed drafts of the paper, and approved the final draft.</p> </div> <div class="fn-type-con" title="con"> <p><a href="#author-2">Thirunavukkarasu Annamalai</a>, <a href="#author-3">Xiaohong Wang</a> and <a href="#author-6">Guomin Niu</a> performed the experiments, analyzed the data, prepared figures and/or tables, and approved the final draft.</p> </div> <div class="fn-type-con" title="con"> <p><a href="#author-4">Sheng Li</a> and <a href="#author-5">Stephen Munga</a> performed the experiments, authored or reviewed drafts of the paper, and approved the final draft.</p> </div> <div class="fn-type-con" title="con"> <p><a href="#author-7">Yuk-Ching Tse-Dinh</a> conceived and designed the experiments, analyzed the data, authored or reviewed drafts of the paper, and approved the final draft.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Field Study Permissions</div> <div class="fn-type-" title=""> <p>The following information was supplied relating to field study approvals (i.e., approving body and any reference numbers):</p> <p>Field collection was approved by the United States Department of Agriculture on 9/12/2018 (permit number P526-180522-001). This USDA permit allowed us to receive global samples legally.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
DNA Deposition</div> <div class="fn-type-" title=""> <p>The following information was supplied regarding the deposition of DNA sequences:</p> <p>All fungal ITS sequences are available at GenBank: <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT584204">MT584204</a>, <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594355">MT594355</a>â€"<a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594393">MT594393</a>, <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594486">MT594486</a>, <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594487">MT594487</a>, <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT594488">MT594488</a>, <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT994711">MT994711</a>â€"<a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT994712">MT994712</a>, and <a href="https://www.ncbi.nlm.nih.gov/nucleotide?term=MT613342">MT613342</a>.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Data Availability</div> <div class="fn-type-" title=""> <p>The following information was supplied regarding data availability:</p> <p>The ITS sequences, phylogenetic tree of 40 fungal isolates from <a href="#table-3">Table 3</a>, raw numeric measurements, and the inhibition results of fungal extracts against different bacterial strains are available in the <a href="#supplemental-information">Supplemental File</a>.</p> </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ref-1" /><span class="label" tagx="label" title="label">Blackwell (2011)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blackwell</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">The fungi: 1, 2, 3.... 5.1 million species?</span><span class="source" tagx="source" title="source">American Journal of Botany</span><span class="volume" tagx="volume" title="volume">98</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">426</span><span class="lpage" tagx="lpage" title="lpage">438</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/ajb.1000298">10.3732/ajb.1000298</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21613136">21613136</a></span></span></li> <li tag="ref"><a name="ref-2" /><span class="label" tagx="label" title="label">Buyel (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buyel</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Plants as sources of natural and recombinant anti-cancer agents</span><span class="source" tagx="source" title="source">Biotechnology Advances</span><span class="volume" tagx="volume" title="volume">36</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">506</span><span class="lpage" tagx="lpage" title="lpage">520</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biotechadv.2018.02.002">10.1016/j.biotechadv.2018.02.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29408560">29408560</a></span></span></li> <li tag="ref"><a name="ref-3" /><span class="label" tagx="label" title="label">Dias, Urban &amp;amp; Roessner (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dias</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Urban</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roessner</span><span class="given-names" tagx="given-names" title="given-names">U</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">A historical overview of natural products in drug discovery</span><span class="source" tagx="source" title="source">Metabolites</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">303</span><span class="lpage" tagx="lpage" title="lpage">336</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/metabo2020303">10.3390/metabo2020303</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24957513">24957513</a></span></span></li> <li tag="ref"><a name="ref-4" /><span class="label" tagx="label" title="label">Elder (1944)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elder</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span></span><span class="year" tagx="year" title="year">1944</span><span class="mixed-article-title" title="mixed-article-title">Fungi for penicillin production</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">99</span><span class="issue" tagx="issue" title="issue">2563</span><span class="fpage" tagx="fpage" title="fpage">119</span><span class="lpage" tagx="lpage" title="lpage">120</span></span></li> <li tag="ref"><a name="ref-5" /><span class="label" tagx="label" title="label">Fair &amp;amp; Tor (2014)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fair</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tor</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Antibiotics and bacterial resistance in the 21st century</span><span class="source" tagx="source" title="source">Perspectives in Medicinal Chemistry</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="lpage" tagx="lpage" title="lpage">64</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25232278">25232278</a></span></span></li> <li tag="ref"><a name="ref-6" /><span class="label" tagx="label" title="label">Fleming (1929)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fleming</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1929</span><span class="mixed-article-title" title="mixed-article-title">On the antibacterial action of cultures of a Penicillium with special reference to their use in the isolation of B. influenza</span><span class="source" tagx="source" title="source">British Journal of Experimental Pathology</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">226</span><span class="lpage" tagx="lpage" title="lpage">236</span></span></li> <li tag="ref"><a name="ref-7" /><span class="label" tagx="label" title="label">Fukuda etÂ al. (2015)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukuda</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sudoh</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsuchiya</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okuda</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuura</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Motojima</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oikawa</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Igarashi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Tolypoalbin, a new tetramic acid from Tolypocladium album TAMA 479</span><span class="source" tagx="source" title="source">Journal of Antibiotics</span><span class="volume" tagx="volume" title="volume">68</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">399</span><span class="lpage" tagx="lpage" title="lpage">402</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ja.2014.165">10.1038/ja.2014.165</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25627017">25627017</a></span></span></li> <li tag="ref"><a name="ref-8" /><span class="label" tagx="label" title="label">Gonzalez-Menendez etÂ al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalez-Menendez</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crespo</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeÂ Pedro</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diaz</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serrano</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mackenzie</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Justicia</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalez-Tejero</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casares</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vicente</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reyes</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tormo</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Genilloud</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Fungal endophytes from arid areas of Andalusia: high potential sources for antifungal and antitumoral agents</span><span class="source" tagx="source" title="source">Scientific Reports</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">9729</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-28192-5">10.1038/s41598-018-28192-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29950656">29950656</a></span></span></li> <li tag="ref"><a name="ref-9" /><span class="label" tagx="label" title="label">Harman etÂ al. (2004)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harman</span><span class="given-names" tagx="given-names" title="given-names">GE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howell</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viterbo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chet</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lorito</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title"><i>Trichoderma</i> speciesâ€"opportunistic, avirulent plant symbionts</span><span class="source" tagx="source" title="source">Nature Reviews. Microbiology</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">43</span><span class="lpage" tagx="lpage" title="lpage">56</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro797">10.1038/nrmicro797</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15035008">15035008</a></span></span></li> <li tag="ref"><a name="ref-10" /><span class="label" tagx="label" title="label">Hawksworth &amp;amp; Lucking (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hawksworth</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucking</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Fungal diversity revisited: 2.2 to 3.8 million species</span><span class="source" tagx="source" title="source">Microbiology Spectrum</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">FUNK-0052-2016</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/microbiolspec.FUNK-0052-2016">10.1128/microbiolspec.FUNK-0052-2016</a></span></span></li> <li tag="ref"><a name="ref-11" /><span class="label" tagx="label" title="label">Higgins &amp;amp; Sharp (1988)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharp</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span></span><span class="year" tagx="year" title="year">1988</span><span class="mixed-article-title" title="mixed-article-title">CLUSTAL: a package for performing multiple sequence alignment on a microcomputer</span><span class="source" tagx="source" title="source">Gene</span><span class="volume" tagx="volume" title="volume">73</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">237</span><span class="lpage" tagx="lpage" title="lpage">244</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0378-1119(88)90330-7">10.1016/0378-1119(88)90330-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3243435">3243435</a></span></span></li> <li tag="ref"><a name="ref-12" /><span class="label" tagx="label" title="label">Hsu (2006)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">The history of qing hao in the Chinese materia medica</span><span class="source" tagx="source" title="source">Transactions of the Royal Society of Tropical Medicine and Hygiene</span><span class="volume" tagx="volume" title="volume">100</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">505</span><span class="lpage" tagx="lpage" title="lpage">508</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.trstmh.2005.09.020">10.1016/j.trstmh.2005.09.020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16566952">16566952</a></span></span></li> <li tag="ref"><a name="ref-13" /><span class="label" tagx="label" title="label">Hussain etÂ al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hussain</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Sadi</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulz</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinert</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">IR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">A fruitful decade for fungal polyketides from 2007 to 2016: antimicrobial activity, chemotaxonomy and chemodiversity</span><span class="source" tagx="source" title="source">Future Medicinal Chemistry</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">14</span><span class="fpage" tagx="fpage" title="fpage">1631</span><span class="lpage" tagx="lpage" title="lpage">1648</span><span class="pub-id"><a href="https://dx.doi.org/10.4155/fmc-2017-0028">10.4155/fmc-2017-0028</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28884584">28884584</a></span></span></li> <li tag="ref"><a name="ref-14" /><span class="label" tagx="label" title="label">Kaleem etÂ al. (2015)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaleem</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shahab</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shamsi</span><span class="given-names" tagx="given-names" title="given-names">TS</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Chronic myeloid leukemiaâ€"prognostic value of mutations</span><span class="source" tagx="source" title="source">Asian Pacific Journal of Cancer Prevention</span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">17</span><span class="fpage" tagx="fpage" title="fpage">7415</span><span class="lpage" tagx="lpage" title="lpage">7423</span><span class="pub-id"><a href="https://dx.doi.org/10.7314/APJCP.2015.16.17.7415">10.7314/APJCP.2015.16.17.7415</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26625737">26625737</a></span></span></li> <li tag="ref"><a name="ref-15" /><span class="label" tagx="label" title="label">Keller (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keller</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Fungal secondary metabolism: regulation, function and drug discovery</span><span class="source" tagx="source" title="source">Nature Reviews. Microbiology</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">167</span><span class="lpage" tagx="lpage" title="lpage">180</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41579-018-0121-1">10.1038/s41579-018-0121-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30531948">30531948</a></span></span></li> <li tag="ref"><a name="ref-16" /><span class="label" tagx="label" title="label">Lenzi etÂ al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lenzi</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costa</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alberton</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goulart</span><span class="given-names" tagx="given-names" title="given-names">JAG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tavares</span><span class="given-names" tagx="given-names" title="given-names">LBB</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Medicinal fungi: a source of antiparasitic secondary metabolites</span><span class="source" tagx="source" title="source">Applied Microbiology and Biotechnology</span><span class="volume" tagx="volume" title="volume">102</span><span class="issue" tagx="issue" title="issue">14</span><span class="fpage" tagx="fpage" title="fpage">5791</span><span class="lpage" tagx="lpage" title="lpage">5810</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00253-018-9048-8">10.1007/s00253-018-9048-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29749562">29749562</a></span></span></li> <li tag="ref"><a name="ref-17" /><span class="label" tagx="label" title="label">Li etÂ al. (2005)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qing</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">YL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">ZW</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Screening for endophytic fungi with antitumour and antifungal activities from Chinese medicinal plants</span><span class="source" tagx="source" title="source">World Journal of Microbiology &amp;amp; Biotechnology</span><span class="volume" tagx="volume" title="volume">21</span><span class="issue" tagx="issue" title="issue">8â€"9</span><span class="fpage" tagx="fpage" title="fpage">1515</span><span class="lpage" tagx="lpage" title="lpage">1519</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11274-005-7381-4">10.1007/s11274-005-7381-4</a></span></span></li> <li tag="ref"><a name="ref-18" /><span class="label" tagx="label" title="label">Li etÂ al. (2013)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Githure</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Genome-block expression-assisted association studies discover malaria resistance genes in <i>Anopheles gambiae</i></span><span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span><span class="volume" tagx="volume" title="volume">110</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">20675</span><span class="lpage" tagx="lpage" title="lpage">20680</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1321024110">10.1073/pnas.1321024110</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24297936">24297936</a></span></span></li> <li tag="ref"><a name="ref-19" /><span class="label" tagx="label" title="label">Luo &amp;amp; Shen (1987)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">XD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span></span><span class="year" tagx="year" title="year">1987</span><span class="mixed-article-title" title="mixed-article-title">The chemistry, pharmacology, and clinical-applications of qinghaosu (artemisinin) and its derivatives</span><span class="source" tagx="source" title="source">Medicinal Research Reviews</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">29</span><span class="lpage" tagx="lpage" title="lpage">52</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/med.2610070103">10.1002/med.2610070103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3550324">3550324</a></span></span></li> <li tag="ref"><a name="ref-20" /><span class="label" tagx="label" title="label">May etÂ al. (2004)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milne</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shingles</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nwiah</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Interactive catalogue of Australian Fungi. C. R. B. Gardens</span><span class="source" tagx="source" title="source">Melbourne, Australian biological resources study</span></span></li> <li tag="ref"><a name="ref-21" /><span class="label" tagx="label" title="label">Nielsen &amp;amp; Nielsen (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Development of fungal cell factories for the production of secondary metabolites: linking genomics and metabolism</span><span class="source" tagx="source" title="source">Synthetic and Systems Biotechnology</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">5</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.synbio.2017.02.002">10.1016/j.synbio.2017.02.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29062956">29062956</a></span></span></li> <li tag="ref"><a name="ref-22" /><span class="label" tagx="label" title="label">Niu etÂ al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franca</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roobsoong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguitragool</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prachumsri</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butler</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The fibrinogen-like domain of FREP1 protein is a broad-spectrum malaria transmission-blocking vaccine antigen</span><span class="source" tagx="source" title="source">Journal of Biological Chemistry</span><span class="volume" tagx="volume" title="volume">292</span><span class="issue" tagx="issue" title="issue">28</span><span class="fpage" tagx="fpage" title="fpage">11960</span><span class="lpage" tagx="lpage" title="lpage">11969</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M116.773564">10.1074/jbc.M116.773564</a></span></span></li> <li tag="ref"><a name="ref-23" /><span class="label" tagx="label" title="label">Niu etÂ al. (2020)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2020</span><span class="mixed-article-title" title="mixed-article-title">Fungal metabolite Asperaculane B inhibits malaria infection and transmission</span><span class="source" tagx="source" title="source">Molecules</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">13</span><span class="fpage" tagx="fpage" title="fpage">3018</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules25133018">10.3390/molecules25133018</a></span></span></li> <li tag="ref"><a name="ref-24" /><span class="label" tagx="label" title="label">Niu etÂ al. (2015)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cichewicz</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Targeting mosquito FREP1 with a fungal metabolite blocks malaria transmission</span><span class="source" tagx="source" title="source">Scientific Reports</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">14694</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep14694">10.1038/srep14694</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26437882">26437882</a></span></span></li> <li tag="ref"><a name="ref-25" /><span class="label" tagx="label" title="label">Oâ€™Brien etÂ al. (2000)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oâ€™Brien</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orton</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pognan</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Investigation of the Alamar Blue (resazurin) fluorescent dye for the assessment of mammalian cell cytotoxicity</span><span class="source" tagx="source" title="source">European Journal of Biochemistry</span><span class="volume" tagx="volume" title="volume">267</span><span class="issue" tagx="issue" title="issue">17</span><span class="fpage" tagx="fpage" title="fpage">5421</span><span class="lpage" tagx="lpage" title="lpage">5426</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1432-1327.2000.01606.x">10.1046/j.1432-1327.2000.01606.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10951200">10951200</a></span></span></li> <li tag="ref"><a name="ref-26" /><span class="label" tagx="label" title="label">OpÂ DeÂ Beeck etÂ al. (2014)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">OpÂ DeÂ Beeck</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lievens</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Busschaert</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Declerck</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vangronsveld</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colpaert</span><span class="given-names" tagx="given-names" title="given-names">JV</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Comparison and validation of some ITS primer pairs useful for fungal metabarcoding studies</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e97629</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0097629">10.1371/journal.pone.0097629</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24933453">24933453</a></span></span></li> <li tag="ref"><a name="ref-27" /><span class="label" tagx="label" title="label">Pham etÂ al. (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pham</span><span class="given-names" tagx="given-names" title="given-names">JV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yilma</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feliz</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Majid</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maffetone</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reynolds</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">A review of the microbial production of bioactive natural products and biologics</span><span class="source" tagx="source" title="source">Frontiers in Microbiology</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1404</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31281299">31281299</a></span></span></li> <li tag="ref"><a name="ref-28" /><span class="label" tagx="label" title="label">Raja etÂ al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raja</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pearce</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oberlies</span><span class="given-names" tagx="given-names" title="given-names">NH</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Fungal identification using molecular tools: a primer for the natural products research community</span><span class="source" tagx="source" title="source">Journal of Natural Products</span><span class="volume" tagx="volume" title="volume">80</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">756</span><span class="lpage" tagx="lpage" title="lpage">770</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jnatprod.6b01085">10.1021/acs.jnatprod.6b01085</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28199101">28199101</a></span></span></li> <li tag="ref"><a name="ref-29" /><span class="label" tagx="label" title="label">Richards etÂ al. (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richards</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leonard</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bass</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Marine fungi: their ecology and molecular diversity</span><span class="source" tagx="source" title="source">Annual Review of Marine Science</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">495</span><span class="lpage" tagx="lpage" title="lpage">522</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-marine-120710-100802">10.1146/annurev-marine-120710-100802</a></span></span></li> <li tag="ref"><a name="ref-30" /><span class="label" tagx="label" title="label">Todd (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Todd</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Reconsidering antibiotic resistance</span><span class="source" tagx="source" title="source">American Journal of Nursing</span><span class="volume" tagx="volume" title="volume">117</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">66</span><span class="lpage" tagx="lpage" title="lpage">67</span></span></li> <li tag="ref"><a name="ref-31" /><span class="label" tagx="label" title="label">Van Dijk &amp;amp; Wang (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Dijk</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Heterologous expression of fungal secondary metabolite pathways in the Aspergillus nidulans host system</span><span class="source" tagx="source" title="source">Methods in Enzymology</span><span class="volume" tagx="volume" title="volume">575</span><span class="fpage" tagx="fpage" title="fpage">127</span><span class="lpage" tagx="lpage" title="lpage">142</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/bs.mie.2016.02.021">10.1016/bs.mie.2016.02.021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27417927">27417927</a></span></span></li> <li tag="ref"><a name="ref-32" /><span class="label" tagx="label" title="label">Venieraki, Dimou &amp;amp; Katinakis (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venieraki</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimou</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katinakis</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Endophytic fungi residing in medicinal plants have the ability to produce the same or similar pharmacologically active secondary metabolites as their hosts</span><span class="source" tagx="source" title="source">Hellenic Plant Protection Journal</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">51</span><span class="lpage" tagx="lpage" title="lpage">66</span><span class="pub-id"><a href="https://dx.doi.org/10.1515/hppj-2017-0006">10.1515/hppj-2017-0006</a></span></span></li> <li tag="ref"><a name="ref-33" /><span class="label" tagx="label" title="label">Visagie etÂ al. (2014)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Visagie</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Houbraken</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frisvad</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klaassen</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perrone</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seifert</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Varga</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yaguchi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samson</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Identification and nomenclature of the genus Penicillium</span><span class="source" tagx="source" title="source">Studies in Mycology</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">343</span><span class="lpage" tagx="lpage" title="lpage">371</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.simyco.2014.09.001">10.1016/j.simyco.2014.09.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25505353">25505353</a></span></span></li> <li tag="ref"><a name="ref-34" /><span class="label" tagx="label" title="label">Wolfender &amp;amp; Queiroz (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolfender</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Queiroz</span><span class="given-names" tagx="given-names" title="given-names">EF</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">New approaches for studying the chemical diversity of natural resources and the bioactivity of their constituents</span><span class="source" tagx="source" title="source">Chimia</span><span class="volume" tagx="volume" title="volume">66</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">324</span><span class="lpage" tagx="lpage" title="lpage">329</span><span class="pub-id"><a href="https://dx.doi.org/10.2533/chimia.2012.324">10.2533/chimia.2012.324</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22867545">22867545</a></span></span></li> <li tag="ref"><a name="ref-35" /><span class="label" tagx="label" title="label">Zhang etÂ al. (2015)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franca</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butler</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimopoulos</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Anopheles midgut FREP1 mediates plasmodium invasion</span><span class="source" tagx="source" title="source">Journal of Biological Chemistry</span><span class="volume" tagx="volume" title="volume">290</span><span class="issue" tagx="issue" title="issue">27</span><span class="fpage" tagx="fpage" title="fpage">16490</span><span class="lpage" tagx="lpage" title="lpage">16501</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M114.623165">10.1074/jbc.M114.623165</a></span></span></li> <li tag="ref"><a name="ref-36" /><span class="label" tagx="label" title="label">Zhang etÂ al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">ZZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">CQ</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Novel natural products from extremophilic fungi</span><span class="source" tagx="source" title="source">Marine Drugs</span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">194</span></span></li> <li tag="ref"><a name="ref-37" /><span class="label" tagx="label" title="label">Zhang etÂ al. (2014)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nong</span><span class="given-names" tagx="given-names" title="given-names">XH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Insights into deep-sea sediment fungal communities from the East Indian Ocean using targeted environmental sequencing combined with traditional cultivation</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e109118</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0109118">10.1371/journal.pone.0109118</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25272044">25272044</a></span></span></li> </ul> </div> </div>  </body></html>