<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">BMC Microbiol</span><span class="iso-abbrev" title="iso-abbrev">BMC Microbiol</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">BMC Microbiology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1471-2180</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7507641">7507641</a></span><span class="publisher-id" title="publisher-id">1977</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12866-020-01977-3">10.1186/s12866-020-01977-3</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Root-associated endophytic bacterial community composition and structure of three medicinal licorices and their changes with the growing year</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dang</span><span class="given-names" tagx="given-names" title="given-names">Hanli</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Tao</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Guifang</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mu</span><span class="given-names" tagx="given-names" title="given-names">Yudi</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">Xinhua</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Zhongke</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhuang</span><span class="given-names" tagx="given-names" title="given-names">Li</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">3033573705@qq.com</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.411680.a</span><span class="institution-id" title="institution-id">0000 0001 0514 4044</span><span class="institution" title="institution">College of Life Sciences, Key Laboratory of Xinjiang Phytomedicine Resource Utilization, Ministry of Education, </span><span class="institution" title="institution">Shihezi University, </span></div></span><span class="citation_author_institution" id="Aff2">[2], </span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-9-9</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">20</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">291</span><span class="history" title="history"><span class="received" title="received">received: 2020-6-23</span><span class="accepted" title="accepted">accepted: 2020-9-15</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1">The dried roots and rhizomes of medicinal licorices are widely used worldwide as a traditional medicinal herb, which are mainly attributed to a variety of bioactive compounds that can be extracted from licorice root. Endophytes and plants form a symbiotic relationship, which is an important source of host secondary metabolites.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par2">In this study, we used high-throughput sequencing technology and high-performance liquid chromatography to explore the composition and structure of the endophytic bacterial community and the content of bioactive compounds (glycyrrhizic acid, liquiritin and total flavonoids) in different species of medicinal licorices (<i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i>, and <i>Glycyrrhiza inflata</i>) and in different planting years (1â€"3â€‰years). Our results showed that the contents of the bioactive compounds in the roots of medicinal licorices were not affected by the species, but were significantly affected by the main effect growing year (1â€"3) (<i>P</i>â€‰<div class="â€‰0.05),_UNKNOWN">
Rhizobiaceae, <i>Pseudomonas</i>, <i>Novosphingobium</i>, and <i>Pantoea</i> were significantly dominant in the 27 samples. Distance-based redundancy analysis (db-RDA) showed that the content of total flavonoids explained the differences in composition and distribution of endophytic bacterial communities in roots of cultivated medicinal liquorices to the greatest extent. Total soil salt was the most important factor that significantly affected the endophytic bacterial community in soil factors, followed by ammonium nitrogen and nitrate nitrogen. Among the leaf nutrition factors, leaf water content had the most significant effect on the endophytic bacterial community, followed by total phosphorus and total potassium.</div></p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par3">This study not only provides information on the composition and distribution of endophytic bacteria in the roots of medicinal licorices, but also reveals the influence of abiotic factors on the community of endophytic bacteria and bioactive compounds, which provides a reference for improving the quality of licorice.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Endophytic bacteria</div> <div class="kwd" title="kwd">
Growing year</div> <div class="kwd" title="kwd">
High-throughput sequencing</div> <div class="kwd" title="kwd">
Liquorice species</div> <div class="kwd" title="kwd">
Plant secondary metabolites</div> <div class="kwd" title="kwd">
Soil physicochemical</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100012166</span><span class="institution" title="institution">National Basic Research Program of China (973 Program)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">2014CB954203</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhuang</span><span class="given-names" tagx="given-names" title="given-names">Li</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100014718</span><span class="institution" title="institution">Innovative Research Group Project of the National Natural Science Foundation of China</span></div> </div><span class="award-id" tagx="award-id" title="award-id">31360139</span><span class="award-id" tagx="award-id" title="award-id">41561010</span><span class="award-id" tagx="award-id" title="award-id">31560177</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Guifang</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhuang</span><span class="given-names" tagx="given-names" title="given-names">Li</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par27">The term endophyte describes those taxa that can live within plant tissues, either within or between host cells [<a href="#CR1">1</a>]. These endophytes include bacteria, fungi, archaea and unicellular eukaryotes [<a href="#CR2">2</a>]. The internal niches of plants provide a protective barrier from environmental influences that allow bacteria to live and reproduce [<a href="#CR3">3</a>]. In return, plants may benefit from endophytic associations. First, endophytic bacteria can promote the growth and development of host plants by producing a range of nutrients and promoting primary and secondary nutrient uptake through atmospheric nitrogen fixation [<a href="#CR4">4</a>]. Second, endophytic bacteria can participate in aspects of the plantâ€™s phosphate solubilization activity [<a href="#CR5">5</a>], such as osmotic adjustment [<a href="#CR6">6</a>] and stomatal regulation [<a href="#CR7">7</a>], to increase the plantâ€™s ecological adaptability. Furthermore, endophytic bacteria may overcome environmental stress conditions, such as drought and soil salinity stress [<a href="#CR8">8</a>, <a href="#CR9">9</a>], and improve plant growth. Also as biological control agents, endophytic bacteria can produce or promote the production of secondary metabolites by host plants to reduce or prevent damage caused by certain pathogens [<a href="#CR10">10</a>, <a href="#CR11">11</a>]. Numerous studies have shown that an in-depth understanding of the diversity of endophytic bacteria will elucidate the function of the interaction between microorganisms and plants, which will be conducive to the development of strategies for ecological environment restoration and sustainable agricultural, such as remediation of the soil environment [<a href="#CR12">12</a>] and increasing crop production [<a href="#CR13">13</a>].</p> <p id="Par28">Endophytic bacteria have been isolated from both monocotyledonous and dicotyledonous plants [<a href="#CR3">3</a>], ranging from woody plants [<a href="#CR14">14</a>], such as walnut and poplar, to herbaceous crop plants, such as rice and wheat. Modern molecular technology, especially high-throughput sequencing technology, has facilitated characterization of the composition and structure of microbial communities in different plants and has substantially increased our understanding of the composition and diversity of endophytic bacterial communities in the plants [<a href="#CR6">6</a>, <a href="#CR15">15</a>]. These techniques provide information more rapidly and accurately than conventional culturing methods, which are time-consuming and may not be applicable for microorganisms that cannot be cultured [<a href="#CR16">16</a>, <a href="#CR17">17</a>]. Although high-throughput sequencing technology is still subject to several challenges in terms of analysis and visualization of genomic data that relies heavily on the performance of automated pipelines [<a href="#CR18">18</a>], these techniques can be applied to comprehensive analyses of plant tissues RNA and DNA sequencing and are within the reach of most laboratories. High-throughput sequencing based on the targeted phylogenetic marker 16S rRNA can be used to effectively characterize the diversity of microbial communities [<a href="#CR19">19</a>], which makes it a practical approach to the isolation and identification of the community composition and diversity of endogenous bacteria in medicinal licorices.</p> <p id="Par29">The dried roots and rhizomes of three medicinal licorices (<i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza inflata</i>, and <i>Glycyrrhiza glabra</i>) are widely used worldwide as a traditional medicinal herb [<a href="#CR20">20</a>]. Many clinical and experimental studies have demonstrated that the root of medicinal licorices has a variety of pharmacological properties [<a href="#CR21">21</a>], such as anti-inflammatory, antiviral and antitumor, immunomodulatory, hepatoprotective activities, which are attributed to a variety of bioactive constituents extracted from licorice root, mainly including triterpene saponins and flavonoids [<a href="#CR22">22</a>]. Glycyrrhizic acid, which is present at the highest levels among the triterpene saponins, is an important pharmacologically bioactive constituent with anti-inflammatory [<a href="#CR23">23</a>], antiviral, immune regulation and other biological properties. Recent studies have shown that licorice flavonoids have a variety of pharmacological activities and have gradually become a hot spot in pharmacological research [<a href="#CR24">24</a>, <a href="#CR25">25</a>].</p> <p id="Par30">Bioactive compounds contained within medicinal plants are the material basis for clinical efficacy and an important index of the quality of medicinal materials. Furthermore, these materials are the evolutionary result of the interaction between plants and their natural environment [<a href="#CR26">26</a>, <a href="#CR27">27</a>]. Studies have shown that the accumulation of bioactive compounds in licorice roots is affected by both the ecological environment and regulation of nutrient elements, such as nitrogen sources and phosphate concentrations in the soil [<a href="#CR28">28</a>], to patterns of distribution in the root distribution pattern that are affected by factors such as year [<a href="#CR29">29</a>] and biomass [<a href="#CR30">30</a>].</p> <p id="Par31">Currently, there is little information about the composition of endophytic bacterial communities in the roots of medicinal licorices at different times in the planting year, and the factors influencing the community structure are not clear. Furthermore, the factors influencing the content of bioactive compounds of medicinal licorices also remain to be elucidated. Therefore, in this study, we conducted a three-year licorice planting experiment in the field and used Illumina high-throughput sequencing technology to detect the composition and diversity of the endophytic bacterial community of medicinal licorice roots to explore the factors affecting the accumulation of bioactive compounds and the structure of the endophytic bacterial community.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="theeffectofthegrowingyearonthecontentofbioactivecompoundsintherootsofmedicinallicorices" title="sec"> <div class="title" tagx="title" title="title">
The effect of the growing year on the content of bioactive compounds in the roots of medicinal licorices</div> <p id="Par32">The results of two-way ANOVA showed that the contents of the bioactive compounds (glycyrrhizic acid (GIA), liquiritin (LI) and total flavonoid (GTF)) were not significantly affected by the interaction effect between growing year (1â€"3) and plant species (<i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza inflata</i>, and <i>Glycyrrhiza glabra</i>) (<i>P</i>â€‰&amp;gt;â€‰0.05) (TableÂ <a href="#Tab1">1</a>). However, the contents of the bioactive compounds were significantly affected by the main effect growing year (1â€"3) (<i>P</i>â€‰<div class="â€‰0.05)_UNKNOWN" href="#Tab1">
1 and Fig.Â <a href="#Fig1">1</a>), with a trend of stable increase in the contents observed with each growing year. As shown in Fig. <a href="#Fig1">1</a>, the contents of GIA, LI and GTF in the year 3 were significantly higher than those in year 1, the contents of GIA and GTF in year 2 were significantly higher than those in year 1, and the contents of GIA and LI in year 3 were significantly higher than those in year 2. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></div></p><p>Effect of species and year on the bioactive compounds of licorice root</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Source</th> <th /> <th>Type III Sum of Squares</th> <th>Degrees of freedom</th> <th>Mean Square</th> <th>F</th> <th><b><i>p</i></b> value</th> </tr> </thead> <tbody> <tr> <td rowspan="3"><b>Years</b></td> <td>GIA</td> <td>8.307</td> <td>2</td> <td>4.153</td> <td>20.087</td> <td>0.000</td> </tr> <tr> <td>GTF</td> <td>10.381</td> <td>2</td> <td>5.191</td> <td>97.654</td> <td>0.000</td> </tr> <tr> <td>LI</td> <td>5.605</td> <td>2</td> <td>2.802</td> <td>19.053</td> <td>0.000</td> </tr> <tr> <td rowspan="3"><b>Species</b></td> <td>GIA</td> <td>0.231</td> <td>2</td> <td>0.116</td> <td>0.559</td> <td>0.581</td> </tr> <tr> <td>GTF</td> <td>0.244</td> <td>2</td> <td>0.122</td> <td>2.296</td> <td>0.129</td> </tr> <tr> <td>LI</td> <td>0.297</td> <td>2</td> <td>0.149</td> <td>1.010</td> <td>0.384</td> </tr> <tr> <td rowspan="3"><b>Years * Species</b></td> <td>GIA</td> <td>0.398</td> <td>4</td> <td>0.099</td> <td>0.481</td> <td>0.750</td> </tr> <tr> <td>GTF</td> <td>0.055</td> <td>4</td> <td>0.014</td> <td>0.260</td> <td>0.900</td> </tr> <tr> <td>LI</td> <td>0.284</td> <td>4</td> <td>0.071</td> <td>0.483</td> <td>0.748</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Description: <i>P</i>â€‰<div class="â€‰0.05_UNKNOWN">   </div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Effect of main effect growing year on the bioactive compounds of licorice roots</p> <p>Description: Ordinate is the content of GIA (<b>a</b>), GTF (<b>b</b>) and LI (<b>c</b>); abscissa is the group name (E, R and S: years 1, 2, and 3, respectively). Different letters indicated statistically significant difference (<i>P</i>â€‰<div class="â€‰0.05)._UNKNOWN" /></p>   </div> <p /> <p id="Par33">Details of the temperature, rainfall, leaves and soil factors during the experimental period are presented in Supplementary Table S<a href="#MOESM1">1</a>. In addition, Spearman correlation analysis showed that the content of bioactive compounds was significantly correlated with soil physicochemical properties and nutritional components of leaves (TableÂ <a href="#Tab2">2</a>). In terms of soil physicochemical factors, GIA and LI had a very significant positive correlation with soil ammonium nitrogen (SAN) (<i>R</i><sup><i>2</i></sup>â€‰&amp;gt;â€‰0; <i>P</i>â€‰<div class="â€‰0.01),_UNKNOWN">
R<sup><i>2</i></sup>â€‰<div class="â€‰0;_UNKNOWN"><i>P</i>â€‰<div class="â€‰0.01);_UNKNOWN">
R<sup><i>2</i></sup>â€‰&amp;gt;â€‰0; <i>P</i>â€‰<div class="â€‰0.01),_UNKNOWN">
R<sup><i>2</i></sup>â€‰<div class="â€‰0;_UNKNOWN"><i>P</i>â€‰<div class="â€‰0.01),_UNKNOWN">
R<sup><i>2</i></sup>â€‰<div class="â€‰0.01;_UNKNOWN"><i>P</i>â€‰<div class="â€‰0.05)._UNKNOWN" href="#Tab2">
2), GIA, LI and GTF were very negatively correlated with water content (PWC), total potassium (PTK) (Fig.Â <a href="#Fig2">2</a> a, b, c) and crude fiber (CF). In addition, GTF also showed a very significant negative correlation with total phosphorus (PTP) (Fig.Â <a href="#Fig2">2</a>d). There was a significant relationship between leaf nutrients and a small number of variables. Specifically, PWC was positively correlated with SNN, but negatively correlated with SAN; organic carbon (POC) was positively correlated with soil total potassium (STK); total nitrogen (PTN) was positively correlated with POC; total phosphorus (PTP) was very positively correlated with SNN, TS and PTN; PTK was very positively correlated with SNN, TS, PWC, PTN and PTP, but negatively correlated with SAN; CF was very positively correlated with PWC and PTK. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></div></div></div></div></div></div></div></div></p><p>Spearman correlation coefficient of the content of bioactive compounds with soil physicochemical properties and nutritional components of leaves</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th /> <th>GIA</th> <th>GTF</th> <th>LI</th> <th>SOM</th> <th>STN</th> <th>STP</th> <th>STK</th> <th>SNN</th> <th>SAN</th> <th>TS</th> <th>PWC</th> <th>POC</th> <th>PTN</th> <th>PTP</th> <th>PTK</th> <th>CF</th> </tr> </thead> <tbody> <tr> <td>GIA</td> <td>1.000</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>GTF</td> <td>0.774**</td> <td>1.000</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>LI</td> <td>0.964**</td> <td>0.716**</td> <td>1.000</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>SOM</td> <td>0.173</td> <td>0.156</td> <td>0.173</td> <td>1.000</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>STN</td> <td>0.262</td> <td>0.060</td> <td>0.233</td> <td>0.578**</td> <td>1.000</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>STP</td> <td>0.230</td> <td>0.032</td> <td>0.218</td> <td>0.438**</td> <td>0.713**</td> <td>1.000</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>STK</td> <td>âˆ’â€‰0.283</td> <td>âˆ’0.025</td> <td>âˆ’â€‰0.283</td> <td>0.147</td> <td>âˆ’â€‰0.051</td> <td>âˆ’â€‰0.131</td> <td>1.000</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>SNN</td> <td>âˆ’0.363</td> <td>âˆ’â€‰0.490**</td> <td>âˆ’â€‰0.359</td> <td>0.057</td> <td>âˆ’â€‰0.179</td> <td>âˆ’â€‰0.216</td> <td>âˆ’â€‰0.234</td> <td>1.000</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>SAN</td> <td>0.600**</td> <td>0.535**</td> <td>0.593**</td> <td>0.396*</td> <td>0.323</td> <td>0.433*</td> <td>0.027</td> <td>âˆ’0.233</td> <td char="." align="char">1.000</td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>TS</td> <td>âˆ’0.511**</td> <td>âˆ’â€‰0.449*</td> <td>âˆ’â€‰0.499**</td> <td>âˆ’0.294</td> <td>âˆ’â€‰0.474*</td> <td>âˆ’0.452*</td> <td>âˆ’â€‰0.259</td> <td>0.760**</td> <td char="." align="char">âˆ’â€‰0.490**</td> <td char="." align="char">1.000</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>PWC</td> <td>âˆ’â€‰731**</td> <td>âˆ’0.807**</td> <td>âˆ’0.737**</td> <td>0.077</td> <td>0.019</td> <td>âˆ’0.017</td> <td>0.073</td> <td>0.484*</td> <td char="." align="char">âˆ’0.405*</td> <td char="." align="char">0.343</td> <td char="." align="char">1.000</td> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>POC</td> <td>âˆ’0.331</td> <td>âˆ’0.114</td> <td>âˆ’â€‰0.311</td> <td>0.321</td> <td>0.013</td> <td>âˆ’0.142</td> <td>0.430*</td> <td>0.321</td> <td char="." align="char">0.077</td> <td char="." align="char">0.324</td> <td char="." align="char">0.111</td> <td char="." align="char">1.000</td> <td /> <td /> <td /> <td /> </tr> <tr> <td>PTN</td> <td>âˆ’0.261</td> <td>âˆ’0.297</td> <td>âˆ’0.192</td> <td>0.098</td> <td>0.101</td> <td>0.136</td> <td>0.028</td> <td>0.243</td> <td char="." align="char">0.059</td> <td char="." align="char">0.206</td> <td char="." align="char">0.245</td> <td char="." align="char">0.444*</td> <td char="." align="char">1.000</td> <td /> <td /> <td /> </tr> <tr> <td>PTP</td> <td>âˆ’0.298</td> <td>âˆ’0.537**</td> <td>âˆ’0.223</td> <td>âˆ’â€‰0.073</td> <td>âˆ’0.027</td> <td>âˆ’â€‰0.095</td> <td>âˆ’0.357</td> <td>0.654**</td> <td char="." align="char">âˆ’0.264</td> <td char="." align="char">0.656**</td> <td char="." align="char">0.368</td> <td char="." align="char">0.259</td> <td char="." align="char">0.608**</td> <td char="." align="char">1.000</td> <td /> <td /> </tr> <tr> <td>PTK</td> <td>âˆ’0.670**</td> <td>âˆ’0.707**</td> <td>âˆ’â€‰0.634**</td> <td>âˆ’0.074</td> <td>âˆ’â€‰0.013</td> <td>0.048</td> <td>âˆ’â€‰0.123</td> <td>0.528**</td> <td char="." align="char">âˆ’â€‰0.421*</td> <td char="." align="char">0.623**</td> <td char="." align="char">0.644**</td> <td char="." align="char">0.336</td> <td char="." align="char">0.587**</td> <td char="." align="char">0.792**</td> <td char="." align="char">1.000</td> <td /> </tr> <tr> <td>CF</td> <td>âˆ’0.684**</td> <td>âˆ’0.662**</td> <td>âˆ’â€‰0.699**</td> <td>âˆ’0.053</td> <td>0.070</td> <td>0.259</td> <td>0.062</td> <td>0.207</td> <td char="." align="char">âˆ’0.349</td> <td char="." align="char">0.173</td> <td char="." align="char">0.806**</td> <td char="." align="char">âˆ’0.024</td> <td char="." align="char">0.378</td> <td char="." align="char">0.227</td> <td char="." align="char">0.601**</td> <td char="." align="char">1.000</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Description: the values are the correlation coefficients. ** means <i>P</i>â€‰<div class="â€‰0.01;_UNKNOWN">
Pâ€‰<div class="â€‰0.05_UNKNOWN" /></div></p> </div>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Stepwise multiple linear regression model</p> <p>Description: The content of PTK (<b>a</b>, <b>b</b>, <b>c</b>) and PTP (<b>d</b>) were used as independent variables, and the content of GIA, GTF and LI were used as dependent variables.</p>   </div> <p /> </div></div> <div class="compositionofbacterialcommunityintherootofmedicinalliquorices" title="sec"> <div class="title" tagx="title" title="title">
Composition of bacterial community in the root of medicinal liquorices</div> <p id="Par34">In 27 samples, a total of 1,979,531 effective sequences were obtained after quality control. The sequencing results for each sample are listed in Supplementary Table S<a href="#MOESM1">2</a>. The effective sequences were clustered into operational taxonomic units (OTUs) with 97% identity, and a total of 2432 OTUs were obtained. The rarefaction curves were normalized to the lowest number of sequences, showed that the number of OTU in each sample increased gradually with quantity of sequence, thus confirming that the amount of sequencing data is adequate (Fig.Â <a href="#Fig3">3</a>). According to the OTUs, 28 phyla, 45 classes, 106 orders, 216 families, 508 genera and 313 species were annotated. FigureÂ <a href="#Fig4">4</a>a shows the 10 bacterial phyla with the greatest abundance in the bacterial community in the roots of medicinal licorices. Proteobacteria dominated the observed sequences at the phylum level, representing 79.917, 64.420, 80.265, 82.848, 65.733, 87.886, 64.544, 75.024 and 85.847% of the total number of species in E.W, R.W, S.W, E.D, R.D, S.D., E.G, R.G and S.G, respectively. In addition, Actinobacteria occupied a large part of the relative abundance in E.G (18.960%) and R.D (22.171%), respectively. In addition, the abundance of Bacteroidetes was high in the R.W, accounting for 19.742%; the abundance of Firmicutes was also high in the R.G, E.G and R.W samples, accounting for 8.735, 7.466 and 6.287%, respectively (Fig.Â <a href="#Fig4">4</a>a). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Rarefaction curves of bacterial community composition in 27 samples</p> <p>Description: The rarefaction curves different colors represent different samples (E, R and S: years 1, 2, and 3, respectively; W, G and D: <i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i>, and <i>Glycyrrhiza inflata</i>, respectively; the third number representing the replicate number).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Histograms of relative abundance of the top 10 bacteria at the phyla level of taxonomy (<b>a</b>), and significantly different species in the sample (<b>b</b>). Description: Ordinate is the relative abundance of bacteria phyla; others refers to are sequences with less or not be annotated; abscissa is the group name (E, R and S: years 1, 2, and 3, respectively; W, G and D: <i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i>, and <i>Glycyrrhiza inflata</i>, respectively). The mark * is significance test <i>p</i>â€‰<div class="â€‰0.05._UNKNOWN" /></p>   </div> <p /> <p id="Par35">In addition, <i>t</i>-test analysis of the two groups showed significant differences in the relative abundance of Tenericutes in samples E, G and W (<i>p</i>â€‰<div class="â€‰0.05)_UNKNOWN" href="#Fig4">
4b). As shown in Fig.Â <a href="#Fig4">4</a>b, the relative abundance of Tenericutes in E.G was significantly higher than that in E.D, E.W and S.G (<i>p</i>â€‰<div class="â€‰0.05)._UNKNOWN">
pâ€‰<div class="â€‰0.05)._UNKNOWN">
pâ€‰<div class="â€‰0.05)._UNKNOWN">
pâ€‰<div class="â€‰0.05)._UNKNOWN" /></div></div></div></div></p> <p id="Par36">In terms of genus, unidentified-<i>Rhizobiaceae</i> was more abundant than other genera (Fig.Â <a href="#Fig5">5</a>), with the abundance in individual samples ranging from 3.398% (E.D) to 33.985% (S.W). The abundance of <i>Pseudomonas</i> was high in the E.W, E.D, R.W, S.D. and R.G samples, accounting for 15.195, 7.165, 6.326, 16.584 and 7.772%, respectively. <i>Pantoea</i> (15.386%) and <i>Halomonas</i> (7.630%) were found to be the most dominant in E.D sample. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Histograms of relative abundance of the top 10 bacteria at the genera level of taxonomy</p> <p>Description: Ordinate is the relative abundance of bacteria genera; others refers to are sequences with less or not be annotated; abscissa is the group name (E, R and S: years 1, 2, and 3, respectively; W, G and D: <i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i>, and <i>Glycyrrhiza inflata</i>, respectively).</p>   </div> <p /> <p id="Par37">We used linear discriminant analysis (LDA) effect size (LEfSe) to identify discriminative taxa among different species of medicinal licorices and different growing years. As shown in Fig.Â <a href="#Fig6">6</a>, the results of LEfSe analysis of all years and species based on the rank sum test revealed a total of 16 biomarkers with significant differences that were contained in the E.D (5 taxa), E.W (7 taxa), R.G (2 taxa) and R.W (2 taxa) groups. No discriminative taxa were observed in year 3. These biomarkers included <i>Variovorax</i>, <i>Nocardiopsis</i>, <i>Methylophaga</i>, <i>Pelagibacterium</i>, <i>Halomonas</i> and <i>Sinomicrobium</i> at the genus taxonomic level (Fig.Â <a href="#Fig6">6</a>b). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Cladograms (<b>a</b>) and LDA value distribution histogram (<b>b</b>) in 27 samples. Description: In cladograms (<b>a</b>), the circle radiating from inside to outside represents the taxonomic level from the Phylum to the species. Each small circle at a different taxonomic level represents a taxonomic at that level, and the diameter of the small circle is proportionate to the relative abundance of species. The figure shows the species with LDA Score greater than the set value (default setting is 4) (<b>b</b>), that is, species with significant differences in different groups. The length of the histogram represents the size of the influence of species with significant differences. The English letters in the figure is the group name (E, R and S: years 1, 2, and 3, respectively; W, G and D: <i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i>, and <i>Glycyrrhiza inflata</i>, respectively).</p>   </div> <p /> <p id="Par38">Details of the composition of the top 10 dominant bacteria at other classification levels are listed in Supplementary Table S<a href="#MOESM1">3</a>. Specifically, Alphaproteobacteria, Gammaproteobacteria, unidentified-Actinobacteria dominate at the class taxonomic level; the dominant species at the order taxonomic level are Rhizobiales, Sphingomonadales and Gammaproteobacteria; the dominant species at the family taxonomic level are Rhizobiaceae, Pseudomonadaceae and Sphingomonadaceae; the dominant species at the species taxonomic level are <i>Pantoea-brenneri</i>, <i>Neorhizobium-huautlense</i>, and <i>Pseudomonas-psychrotolerans</i>.</p> </div> <div class="effectsofyearandspeciesonalphadiversityandbetadiversityinthebacterialcommunityofroots" title="sec"> <div class="title" tagx="title" title="title">
Effects of year and species on alpha diversity and beta diversity in the bacterial community of roots</div> <p id="Par39">The alpha diversity index of each group is shown in Supplementary Table S<a href="#MOESM1">4</a>. The results of two-way ANOVA showed that the alpha diversity indexes (Shannon, Simpson, Chao1 and ACE) were not significantly affected by the interaction between plant species and growing year and were not significantly affected by the main effect species and main effect growing year (TableÂ <a href="#Tab3">3</a>). <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Effects of growth years and species on endophytic bacterial community richness index and diversity index</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Source</th> <th /> <th>Type III Sum of Squares</th> <th>Degrees of freedom</th> <th>Mean Square</th> <th>F</th> <th><b><i>p</i></b> value</th> <th>Partial Eta Squared</th> </tr> </thead> <tbody> <tr> <td rowspan="4">year</td> <td>Shannon</td> <td>3.157</td> <td>2</td> <td>1.578</td> <td>1.540</td> <td>0.241</td> <td>0.146</td> </tr> <tr> <td>Simpson</td> <td>0.009</td> <td>2</td> <td>0.004</td> <td>1.401</td> <td>0.272</td> <td>0.135</td> </tr> <tr> <td>Chao1</td> <td>21,992.749</td> <td>2</td> <td>10,996.375</td> <td>0.659</td> <td>0.530</td> <td>0.068</td> </tr> <tr> <td>ACE</td> <td>21,955.833</td> <td>2</td> <td>10,977.917</td> <td>0.728</td> <td>0.496</td> <td>0.075</td> </tr> <tr> <td rowspan="4">species</td> <td>Shannon</td> <td>4.650</td> <td>2</td> <td>2.325</td> <td>2.269</td> <td>0.132</td> <td>0.201</td> </tr> <tr> <td>Simpson</td> <td>0.006</td> <td>2</td> <td>0.003</td> <td>0.934</td> <td>0.411</td> <td>0.094</td> </tr> <tr> <td>Chao1</td> <td>41,731.275</td> <td>2</td> <td>20,865.638</td> <td>1.250</td> <td>0.310</td> <td>0.122</td> </tr> <tr> <td>ACE</td> <td>32,540.035</td> <td>2</td> <td>16,270.017</td> <td>1.079</td> <td>0.361</td> <td>0.107</td> </tr> <tr> <td rowspan="4">year*species</td> <td>Shannon</td> <td>7.272</td> <td>4</td> <td>1.818</td> <td>1.774</td> <td>0.178</td> <td>0.283</td> </tr> <tr> <td>Simpson</td> <td>0.016</td> <td>4</td> <td>0.004</td> <td>1.246</td> <td>0.327</td> <td>0.217</td> </tr> <tr> <td>Chao1</td> <td>129,404.800</td> <td>4</td> <td>32,351.200</td> <td>1.938</td> <td>0.148</td> <td>0.301</td> </tr> <tr> <td>ACE</td> <td>121,953.430</td> <td>4</td> <td>30,488.357</td> <td>2.023</td> <td>0.134</td> <td>0.310</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Description: Description: <i>P</i>â€‰<div class="â€‰0.05_UNKNOWN">   </div></p> <p id="Par40">However, beta diversity analysis showed significant differences in the endophytic bacterial community among the different groups. As shown in Fig.Â <a href="#Fig7">7</a>, Wilcox rank sum test based on the UniFrac distances showed significant differences in beta diversity between E.W and S.W (<i>P</i>â€‰<div class="â€‰0.05),_UNKNOWN">
Pâ€‰<div class="â€‰0.05),_UNKNOWN">
Pâ€‰<div class="â€‰0.05),_UNKNOWN">
Pâ€‰<div class="â€‰0.05),_UNKNOWN">
Pâ€‰<div class="fig" title="fig"> </div></div></div></div></div></p><div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <p>The significance test of the differences of Beta diversity. Description: Ordinate is the Beta diversity; abscissa is the group name (E, R and S: years 1, 2, and 3, respectively; W, G and D: <i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i>, and <i>Glycyrrhiza inflata</i>, respectively). The mark * is significance test <i>p</i>â€‰<div class="â€‰0.05._UNKNOWN" /></p>   </div></div> <p /> </div> <div class="therelationshipbetweenthedominantphylumandgenusofendophyticbacteriaandthebioactivecompounds,soilphysicochemicalpropertiesandleafnutrition" title="sec"> <div class="title" tagx="title" title="title">
The relationship between the dominant phylum and genus of endophytic bacteria and the bioactive compounds, soil physicochemical properties and leaf nutrition</div> <p id="Par41">Spearman correlation analysis showed that there was a significant relationship between dominant bacteria phylum and bioactive compounds, soil physicochemical properties and leaf nutrition (TableÂ <a href="#Tab4">4</a>). Specifically, Proteobacteria showed a very significant negative correlation with PTN; Actinobacteria showed a significant negative correlation with SAN, GIA and LI (<i>R</i><sup><i>2</i></sup>â€‰<div class="â€‰0;_UNKNOWN"><i>P</i>â€‰<div class="â€‰0.05);_UNKNOWN">
R<sup><i>2</i></sup>â€‰&amp;gt;â€‰0; <i>P</i>â€‰<div class="â€‰0.05)._UNKNOWN"><div class="table-wrap_UNKNOWN" id="Tab4"><span class="label" tagx="label" title="label">Table 4</span> </div></div></div></div></p><p>Spearman correlation analyses testing the relationship between the relative abundance of dominant bacterial phyla in the root of licorice and bioactive compounds, soil physicochemical properties and leaf nutrition</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th /> <th>Proteobacteria</th> <th>Actinobacteria</th> <th>Bacteroidetes</th> <th>Firmicutes</th> <th>Unidentified-Bacteria</th> <th>Chlamydiae</th> <th>Tenericutes</th> <th>Acidobacteria</th> <th>Verrucomicrobia</th> <th>Planctomycetes</th> </tr> </thead> <tbody> <tr> <td>SOM</td> <td>0.098</td> <td>âˆ’0.219</td> <td>âˆ’0.096</td> <td>âˆ’0.190</td> <td>âˆ’0.253</td> <td>âˆ’0.167</td> <td>âˆ’0.305</td> <td>âˆ’0.246</td> <td>0.092</td> <td>âˆ’0.332</td> </tr> <tr> <td>STN</td> <td>0.242</td> <td>âˆ’0.358</td> <td>âˆ’0.004</td> <td>âˆ’0.178</td> <td>âˆ’0.236</td> <td>âˆ’0.207</td> <td>âˆ’â€‰0.131</td> <td>âˆ’0.252</td> <td>âˆ’â€‰0.020</td> <td>âˆ’0.155</td> </tr> <tr> <td>STP</td> <td>âˆ’0.024</td> <td>âˆ’0.236</td> <td>0.209</td> <td>0.099</td> <td>âˆ’0.129</td> <td>0.125</td> <td>0.165</td> <td>âˆ’0.319</td> <td>âˆ’0.070</td> <td>âˆ’â€‰0.033</td> </tr> <tr> <td>STK</td> <td>âˆ’0.220</td> <td>0.087</td> <td>0.216</td> <td>0.091</td> <td>âˆ’0.127</td> <td>0.060</td> <td>âˆ’0.225</td> <td>0.111</td> <td>0.256</td> <td>âˆ’0.116</td> </tr> <tr> <td>SNN</td> <td>0.075</td> <td>0.154</td> <td>âˆ’0.137</td> <td>âˆ’0.005</td> <td>âˆ’â€‰0.128</td> <td>âˆ’0.241</td> <td>0.292</td> <td>0.128</td> <td>âˆ’0.355</td> <td>âˆ’0.026</td> </tr> <tr> <td>SAN</td> <td>0.073</td> <td>âˆ’0.455*</td> <td>0.250</td> <td>0.138</td> <td>0.223</td> <td>0.107</td> <td>0.067</td> <td>âˆ’0.068</td> <td>0.208</td> <td>âˆ’0.222</td> </tr> <tr> <td>TS</td> <td>0.034</td> <td>0.274</td> <td>âˆ’0.228</td> <td>0.045</td> <td>âˆ’0.195</td> <td>âˆ’0.276</td> <td>0.311</td> <td>0.214</td> <td>âˆ’0.289</td> <td>0.211</td> </tr> <tr> <td>PWC</td> <td>âˆ’0.088</td> <td>0.111</td> <td>0.020</td> <td>0.173</td> <td>âˆ’0.270</td> <td>âˆ’0.094</td> <td>0.047</td> <td>0.038</td> <td>âˆ’0.204</td> <td>âˆ’0.046</td> </tr> <tr> <td>POC</td> <td>âˆ’0.178</td> <td>0.016</td> <td>0.181</td> <td>0.354</td> <td>âˆ’0.290</td> <td>âˆ’0.225</td> <td>0.074</td> <td>0.434*</td> <td>0.077</td> <td>0.151</td> </tr> <tr> <td>PTN</td> <td>âˆ’0.499**</td> <td>0.291</td> <td>0.187</td> <td>0.430*</td> <td>0.236</td> <td>0.008</td> <td>0.339</td> <td>0.263</td> <td>0.066</td> <td>0.184</td> </tr> <tr> <td>PTP</td> <td>0.022</td> <td>0.122</td> <td>âˆ’0.134</td> <td>0.090</td> <td>âˆ’0.162</td> <td>âˆ’â€‰0.177</td> <td>0.238</td> <td>0.130</td> <td>âˆ’0.199</td> <td>0.188</td> </tr> <tr> <td>PTK</td> <td>âˆ’0.202</td> <td>0.218</td> <td>âˆ’0.010</td> <td>0.267</td> <td>âˆ’0.339</td> <td>âˆ’0.172</td> <td>0.279</td> <td>0.117</td> <td>âˆ’0.186</td> <td>0.214</td> </tr> <tr> <td>CF</td> <td>âˆ’0.330</td> <td>0.206</td> <td>0.095</td> <td>0.408*</td> <td>âˆ’0.054</td> <td>0.170</td> <td>0.343</td> <td>0.121</td> <td>âˆ’0.022</td> <td>0.212</td> </tr> <tr> <td>GlA</td> <td>0.358</td> <td>âˆ’0.463*</td> <td>0.049</td> <td>âˆ’0.274</td> <td>0.185</td> <td>0.040</td> <td>âˆ’0.097</td> <td>âˆ’0.204</td> <td>âˆ’â€‰0.012</td> <td>âˆ’0.100</td> </tr> <tr> <td>GTF</td> <td>0.168</td> <td>âˆ’0.354</td> <td>0.031</td> <td>âˆ’0.241</td> <td>0.248</td> <td>0.083</td> <td>âˆ’0.038</td> <td>âˆ’0.165</td> <td>0.221</td> <td>âˆ’0.070</td> </tr> <tr> <td>LI</td> <td>0.331</td> <td>âˆ’0.458*</td> <td>0.079</td> <td>âˆ’0.242</td> <td>0.189</td> <td>0.074</td> <td>âˆ’0.111</td> <td>âˆ’0.218</td> <td>0.037</td> <td>âˆ’0.109</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Description: the values are the correlation coefficients. ** means <i>P</i>â€‰<div class="â€‰0.01;_UNKNOWN">
Pâ€‰<div class="â€‰0.05_UNKNOWN" /></div></p> </div>  <p /> <p id="Par42">As shown in Fig.Â <a href="#Fig8">8</a>, there was a significant relationship between the dominant bacteria genus and the bioactive compounds of roots, soil physicochemical factors and leaf nutrition. Specifically, unidentified-<i>Rhizobiaceae</i> had a significant positive correlation with SAN, GIA, GTF and LI, but a very significant negative correlation with CF; <i>Pantoea</i> had a significant positive correlation with PTK; <i>Pseudomonas</i> had a significant positive correlation with PTP; <i>Halomonas</i> had a significant positive correlation with SNN, TS, PWC, PTK and CF, but a very significant negative correlation with SAN, GIA, GTF and LI; <i>Nocardiopsis</i> had a significant positive correlation with TS, PWC, PTP, PTK and CF, but a very significant negative correlation with SAN, GIA, GTF and LI; <i>Novosphingobium</i> had a significant negative correlation with PWC, but a very significant positive correlation with GTF; and <i>Sinomicrobium</i> had significant positive correlation with SNN, PWC, PTK and CF, but a significant negative correlation with GIA, GTF and LI. It is worth noting that we found that the biomarkers with significant differences at the genus taxonomic level of the dominant bacterial community (including <i>Nocardiopsis</i>, <i>Halomonas</i> and <i>Sinomicrobium</i>) had a significant negative correlation with GIA, GTF, and LI (<i>R</i><sup><i>2</i></sup>â€‰<div class="â€‰0;_UNKNOWN"><i>P</i>â€‰<div class="â€‰0.05)._UNKNOWN" href="#MOESM1">
5. These results showed that the biomarkers with significant differences (except <i>Methylophaga</i>) at each classification level were significantly negatively correlated with GIA, GTF, and LI, while all showed a significant positive correlation with PTK. </div></div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 8</span></a></div>   <p>Heat maps of Spearman correlation analysis. Description: Ordinate is the information of environmental factors, and abscissa is the information of species at the genera level of taxonomy. The correlation coefficient r of Spearman is between âˆ’â€‰1 and 1, <i>r</i>â€‰<div class="â€‰0_UNKNOWN">
râ€‰&amp;gt;â€‰0 is positive correlation, and the mark * is significance test <i>p</i>â€‰<div class="â€‰0.05._UNKNOWN" /></div></p>   </div> <p /> </div> <div class="totalflavonoidsexplainedthedifferenceincompositionanddistributionofendophyticbacteriacommunityintherootsofplantinglicoricestothegreatestextent" title="sec"> <div class="title" tagx="title" title="title">
Total flavonoids explained the difference in composition and distribution of endophytic bacteria community in the roots of planting licorices to the greatest extent</div> <p id="Par43">Distance-based redundancy analysis (db-RDA) based on the Brayâ€"Curtis distance showed that the bioactive compounds, soil physicochemical and leaf nutrient components had significant effects on the endophytic bacterial community in the roots of medicinal licorices (Fig.Â <a href="#Fig9">9</a>, TableÂ <a href="#Tab5">5</a>). Specifically, the contents of GIA, GTF, and LI all had a significant effect on the endophytic bacterial community in the roots (<i>P</i>â€‰<div class="â€‰0.05)._UNKNOWN">
r<sup><i>2</i></sup>â€‰=â€‰0.638, <i>P</i>â€‰<div class="â€‰0.01)._UNKNOWN">
râ€‰=â€‰0.260; <i>P</i>â€‰=â€‰0.008) (Supplementary Table S<a href="#MOESM1">6</a>). </div></div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 9</span></a></div>   <p>Db-RDA for all groups. Description: Environmental factors are generally represented by arrows. The length of the arrow line represents the degree of correlation between a certain environmental factor and community and species distribution, and the longer the arrow, the greater the correlation. When the angle between the environmental factors is acute, it means that there is a positive correlation between the two environmental factors, while when the angle is obtuse, there is a negative correlation.</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab5"><span class="label" tagx="label" title="label">Table 5</span> <p>Results for db-RDA testing effects of soil physicochemical properties, leaf nutrients and bioactive compounds on the composition and distribution of bacterial community in licorice root</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th /> <th><b><i>r</i></b><sup><b><i>2</i></b></sup></th> <th><b><i>P</i></b> Value</th> </tr> </thead> <tbody> <tr> <td>SOM</td> <td>0.118</td> <td>0.213</td> </tr> <tr> <td>STN</td> <td>0.163</td> <td>0.113</td> </tr> <tr> <td>STP</td> <td>0.165</td> <td>0.110</td> </tr> <tr> <td>STK</td> <td>0.035</td> <td>0.666</td> </tr> <tr> <td>SNN</td> <td>0.296</td> <td>0.010</td> </tr> <tr> <td>SAN</td> <td>0.428</td> <td>0.002</td> </tr> <tr> <td>TS</td> <td>0.497</td> <td>0.000</td> </tr> <tr> <td>PWC</td> <td>0.498</td> <td>0.001</td> </tr> <tr> <td>POC</td> <td>0.014</td> <td>0.849</td> </tr> <tr> <td>PTN</td> <td>0.044</td> <td>0.607</td> </tr> <tr> <td>PTP</td> <td>0.260</td> <td>0.028</td> </tr> <tr> <td>PTK</td> <td>0.442</td> <td>0.001</td> </tr> <tr> <td>CF</td> <td>0.247</td> <td>0.033</td> </tr> <tr> <td>GlA</td> <td>0.555</td> <td>0.000</td> </tr> <tr> <td>GTF</td> <td>0.638</td> <td>0.000</td> </tr> <tr> <td>LI</td> <td>0.393</td> <td>0.005</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Description: <i>r</i><sup><i>2</i></sup> is the determinant coefficients of the distribution of the bacterial community by environmental factors</p> </div> </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par44">In this study, we explored the effects of planting year (years 1, 2, and 3) and species of medicinal licorices (<i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i>, and <i>Glycyrrhiza inflata</i>) on the accumulation of bioactive compounds. We found that the bioactive compounds were more affected by main effect planting year than main effect species (TableÂ <a href="#Tab1">1</a> and Fig.Â <a href="#Fig1">1</a>), which is consistent with the results of Fan et al. [<a href="#CR31">31</a>]. On the one hand, the bioactive compounds of medicinal plants are the products of physiological activities that occur in the process of cell specialization and maturation [<a href="#CR26">26</a>], and their contents increase with the growth and development of plants. On the other hand, most of the active ingredients in medicinal plants are secondary metabolites [<a href="#CR32">32</a>] (including phenolic compounds, total flavonoids and glycyrrhizin), which accumulate during plant growth and development to provide protection against pathogens and neighboring plants [<a href="#CR33">33</a>]. Furthermore, plant growth and development are influenced by climatic conditions (rainfall regimen and temperature). Zoe Bont et al. [<a href="#CR34">34</a>] demonstrated that plants can adapt to changing environments by adjusting the production and maintenance of compounds in their roots. Plants grown for 3 years in this study were subjected to two more years of climate variability than other sample plants, which can stimulate the accumulation of compounds in the roots and accelerate their resistance to environmental pressures. This is also one of reasons why growth year is a factor that influences the accumulation of bioactive compounds in the roots of medicinal plants.</p> <p id="Par45">Similarly, the accumulation of bioactive compounds in medicinal licorice roots is easily affected by many factors. In this study, our results showed the components in leaves (PWC, POC, PTN, PTP, PTK and CF) were negatively correlated with the contents of glycyrrhizic acid (GIA), liquiritin (LI) and total flavonoids (GTF) in roots (TableÂ <a href="#Tab2">2</a> and Fig.Â <a href="#Fig2">2</a>). Thus, these findings indicated that resource allocation and nutrient competition occur among roots and leaves of medicinal licorices. At the same time, we showed that the content of glycyrrhizic acid (GIA), liquiritin (LI) and total flavonoids (GTF) are negatively correlated with TS in soil (TableÂ <a href="#Tab2">2</a>). We speculate that this effect is mainly because the high concentration of salts (mostly ions like Na<sup>+</sup>) in the soil can decrease the uptake of water and nutrients by plants, as well as the rate of photosynthesis [<a href="#CR35">35</a>], resulting in ion and osmotic stress [<a href="#CR36">36</a>], thus inhibited the accumulation of bioactive compounds. However, appropriate salt stress can increase the accumulation of bioactive compounds in licorice by upregulating proteins involved in the related biosynthesis pathways to stimulate the production of glycyrrhizic acid, liquiritin and total flavonoids and other bioactive compounds [<a href="#CR37">37</a>, <a href="#CR38">38</a>]. Wan [<a href="#CR39">39</a>] and Yang [<a href="#CR40">40</a>] et al. demonstrated that appropriate salt stress stimulated the glycometabolism of <i>Glycyrrhiza</i>, accelerated the decomposition of substances, and promoted the formation and accumulation of glycyrrhizic acid, liquiritin and total flavonoids in medicinal licorices, while high concentration and long-term salt stress inhibited the production of bioactive compounds.</p> <p id="Par46">In addition, our results also showed that GIA, LI and GTF had a significant positive correlation with SAN, but have no significant correlation with SNN (TableÂ <a href="#Tab2">2</a>). We speculate that this is related to the concentration of ammonium nitrogen and nitrate nitrogen in soil and the efficiency of root absorption [<a href="#CR41">41</a>]. In addition, Pei et al. [<a href="#CR42">42</a>] reported the highest content of total flavonoids (GTF) in the root of licorices when SNN: SAN was 0/100, which was significantly higher than that the contents when SNN: SAN was 100/0 and 50/50. These findings showed that different forms and ratios of nitrogen can affect the quality of <i>Glycyrrhiza</i>, indicating that the nitrogen sources in the soil in the form of ammonium nitrogen and nitrate nitrogen contribute collectively to the accumulation of bioactive compounds in medicinal licorices. On the other hand, nitrogen fixation is a result of the symbiotic relationship between rhizobia and host plants, which can facilitate the absorption of nitrogen from the air and soil by host plants. Some studies [<a href="#CR43">43</a>, <a href="#CR44">44</a>] showed that rhizobia form nodules in the root of <i>glycyrrhiza</i> by symbiosis and facilitate nitrogen fixation by the hosts in exchange for nutrients. The amount of nitrogen fixation by rhizobia is affected by the hostâ€™s demand and yield potential for nitrogen, the compatibility of both symbiotic partners and the availability of nitrogen in the soil. Therefore, clarifying the role of soil factors in the growth of medicinal licorices will provide a theoretical basis for the synthesis of bioactive compounds and rational utilization of medicinal plants in production practice.</p> <p id="Par47">In this study, we investigated the composition and diversity of endophytic bacterial communities in three medicinal licorices roots using high-throughput sequencing technology, which provides a large amount of data with more accuracy than that obtained in previous studies using traditional technology [<a href="#CR45">45</a>, <a href="#CR46">46</a>]. Our study showed no significant differences in the alpha diversity index, which is used to describe the diversity and complexity of the bacterial community, between the interaction effect planting year and species of medicinal licorices (TableÂ <a href="#Tab3">3</a>). However, our results showed the indexes of endophytic bacterial diversity in year 3 were slightly lower than those in year 2 (Supplementary Table S<a href="#MOESM1">4</a>), which, combined with the observation that growth years had a significant impact on the contents of GIA, GTF and LI, and showed an upward trend with the increasing number of growing year (Fig.Â <a href="#Fig1">1</a>). We speculate that the accumulation of bioactive compounds in roots regulates the endophytic bacterial diversity and that this is mediated via mechanisms that reduce the relative abundance through the antibacterial effects of bioactive compounds or chemical signals that negatively affect growth [<a href="#CR10">10</a>]. Although the mechanism by which bioactive compounds regulate endophytic bacteria is still unclear, this discovery may form the basis of further in-depth research. In addition, studies have shown that plant organs, rainfall and temperature are the main drivers of endophytic components [<a href="#CR47">47</a>]. In this study, climatic conditions, especially rainfall, in year 2 were significantly higher than those in year 3 (Supplementary Table S<a href="#MOESM1">1</a>). This difference may change endophytic bacteria diversity, suggesting that these species are likely to be affected by climate changes in the near future.</p> <p id="Par48">Our results showed specific microbiomes in 27 samples of medicinal licorices. Proteobacteria was the dominant phylum in all samples, followed by Actinobacteria and Bacteroidetes (Fig.Â <a href="#Fig4">4</a>a), which is consistent with the results of Cheng et al. [<a href="#CR48">48</a>] The phylum Proteobacteria, which constitutes the largest and phenotypically most diverse phylogenetic lineage at present [<a href="#CR49">49</a>], is an important group of microorganisms in evolutionary, geological, and environmental terms [<a href="#CR50">50</a>]. Numerous studies have confirmed that Proteobacteria is the main dominant phylum of the endophytic bacterial communities associated with plants ranging from economic crops, such as soybean [<a href="#CR51">51</a>] and tomato [<a href="#CR52">52</a>], to ornamental plants, such as peony [<a href="#CR53">53</a>].</p> <p id="Par49">Among the 27 samples of medicinal licorices analyzed in this study, unidentified <i>Rhizobiaceae</i> was the dominant genus (Fig.Â <a href="#Fig5">5</a>). Unidentified<i>-Rhizobiaceae</i>, which may be non-culturable or not studied, may be a bacterium of potential importance in the endophytic bacterial community of licorice, because of the nitrogen fixation function of rhizobia, which has become a research hotspot in legumes [<a href="#CR54">54</a>]. Wei et al. [<a href="#CR55">55</a>] demonstrated high tolerance to NaCl, pH and temperature in a large number of isolated rhizobia from <i>Glycyrrhiza uralensis</i> Liu et al. [<a href="#CR49">49</a>] showed that inoculation with rhizobia improved the growth of <i>Glycyrrhiza uralensis</i> and increased the dry weight of the underground rhizome. These studies all showed that beneficial rhizobia coexist with and promote the growth of <i>Glycyrrhiza</i> medicinal licorices. In addition, our results also showed a significant positive correlation between unidentified-<i>Rhizobiaceae</i> and the content of GIA, GTF and LI (Fig.Â <a href="#Fig8">8</a>). Although the scope of beneficial and culturable <i>Rhizobiae</i> is still unclear at present, this information provides a direction for future research on improving the efficacy of medicinal licorices.</p> <p id="Par50">Although many studies the composition of plant endophytic bacterial communities are based on Illumina MiSeq technology [<a href="#CR56">56</a>], few studies have focused on the influence of planting year and species on the composition of endophytic bacterial communities in <i>Glycyrrhiza</i>. Our results show that the endophytic bacteria community is affected by the planting year and species of <i>Glycyrrhiza</i> (Fig.Â <a href="#Fig7">7</a>), which helps to fill the knowledge gap in this field. Endophytic bacteria exist in plant species as part of the root microbiome. The endophytic bacteria community (species diversity: abundance and relative abundance) in plants is dynamic and affected by abiotic and biological factors, such as soil conditions, biogeography and plant species [<a href="#CR57">57</a>]. In this study, the content of bioactive compounds in roots was shown to be the driving factor in differences in the composition and distribution of endophytic bacteria community (Fig.Â <a href="#Fig9">9</a>, TableÂ <a href="#Tab5">5</a>, Supplementary Table S<a href="#MOESM1">6</a>), with the total flavonoid content shown to explain the differences in the composition and distribution of endophytic bacteria community to the greatest extent. Flavonoids, which are a variety of polyphenolic compounds produced in plants by secondary metabolism, can act as a signal of interaction between plants and many microorganisms [<a href="#CR58">58</a>, <a href="#CR59">59</a>]. For example, flavonoids can act as chemoattractants and inducers of nodulation (nod) and other genes in rhizobia [<a href="#CR60">60</a>]. In recent years, a large number of studies including research by MÃ¡rton Szoboszlay et al. [<a href="#CR61">61</a>] have also shown that flavonoids have an impact on bacterial communities.</p> <p id="Par51">The endophytic bacterial composition differed among the samples, and our study also indicates that the endophytic bacterial community in the roots of medicinal licorices is determined by the bioactive compounds in the roots. In addition, this result is consistent with the hypothesis that the physical and chemical factors in the soil and leaf nutrition dictate the composition of the endophytic bacterial population, although further studies are required to characterize the roles of these endophytic bacteria. Our study provides useful information for the development of strategies to improve the production and quality of medicinal licorices.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par52">In this study, numerous endophytic bacteria communities were detected in the roots of medicinal licorices based on high-throughput sequencing. Furthermore, we identified significant differences in the relative abundance of some endophytic bacteria among plant species and growing year. Furthermore, the contents of glycyrrhizic acid, liquiritin and total flavonoids in the roots of medicinal licorices increased significantly with the main effect planting year, and the total flavonoid content shown to explain the differences in the composition and distribution of endophytic bacteria community to the greatest extent.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="4.1samplecollection" title="sec"> <div class="title" tagx="title" title="title">
4.1 Sample collection</div> <p id="Par53">In May 2016, the seeds of three species of medicinal licorices (<i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza inflata</i>, and <i>Glycyrrhiza glabra</i>) were purchased from Xinjiang Beiling licorice Technology Co., Ltd. (Xinjiang, China), and were sown in the south of Yanqi County (86Â°17â€²33â€³ E, 42Â°11â€²34â€³ N), Xinjiang, China. There were nine separate sample plots (4â€‰Ã—â€‰4â€‰m per plot, 3 plots Ã—â€‰3 replicates for each species of licorices). The fields were ploughed each year before sowing and each sample plot was harrowed to 20â€‰cm. The licorice plant seeds were watered immediately after sowing, in the middle growth period and at the stage of maturity. Seeds were sown in this way every year for 3 years. Details of the temperature, rainfall and soil factors during the experimental period are shown in Supplementary Table S<a href="#MOESM1">1</a>. Field sampling was conducted in August 2017, August 2018 and August 2019, when liquorice roots and rhizosphere soil (depth 0â€"40â€‰cm) were collected. To ensure that the experiment was representative, three well-grown licorice roots were randomly selected and cut out with sterile scissors. Each root was divided into two parts: one part was placed into a ziplocked bag for the determination of the bioactive compounds in the root, while the other part was placed into a sterile bag and quickly transported on a piece of ice to the laboratory. To eliminate the interference of other microorganisms, the surface of roots was sterilized in the laboratory by first rinsing soil from the roots under running water followed by washing with sterile distilled water. The roots were then soaked in 75% alcohol for 30â€‰s for surface disinfection, then washed five times with sterile distilled water before soaking in 5% sodium hypochlorite for 5â€‰min. Finally the roots were washed five times with sterile distilled water and air-dried under sterile conditions. To confirm that the surface sterilization process was successful, the last rinse solution was inoculated onto a Luriaâ€"Bertani (LB) medium and cultured at 28â€‰Â°C for 72â€‰h. No bacterial growth confirmed that the surface sterilization was successful. All root samples were immediately placed on ice and then stored at liquid nitrogen prior to total DNA extraction. Soil samples from the rhizosphere were air-dried and sieved through a 2-mm mesh. The following soil physicochemical characteristics were analyzed according to the methods described by the Bao et al. [<a href="#CR62">62</a>]: The content of organic matter (SOM) was determined by external heating with potassium dichromate. The total nitrogen (STN) content was determined using the perchloric acid-sulfuric acid digestion method. The total phosphorus (STP) content was determined by acid digestion (molybdenum-antimony colorimetry). The total potassium (STK) content was determined by acid digestion (atomic absorption spectrometry). The total salt (TS) content was determined by atomic absorption spectrometry. Nitrate nitrogen (SNN) and ammonium nitrogen (SAN) contents were analyzed using 0.01â€‰M calcium chloride extraction. In addition, leaf samples, were air-dried to constant weight, ground to a powder using a pestle and mortar and passed through a 60-mesh sieve. The following plant nutritional components were then analyzed: The content of organic carbon (POC) was determined by external heating with potassium dichromate. Water content (PWC) was determined by weighing. The total nitrogen (PTN) content was determined using the perchloric acid-sulfuric acid digestion method. The total phosphorus (PTP) content was determined by acid digestion (molybdenum-antimony colorimetry method). The total potassium (PTK) content was determined by acid digestion (atomic absorption spectrometry). The crude fiber (CF) content was determined using the acid-base detergent method. All the samples were labeled by combination with letters and numbers, with the first letter representing the age of the medicinal licorices (E, R and S: years 1, 2, and 3, respectively), the second letter representing the species (W, G and D: <i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i>, and <i>Glycyrrhiza inflata</i>, respectively), and the third number representing the replicate number. For example, E.W.3 represents the third repetition of <i>Glycyrrhiza uralensis</i> in the first year.</p> </div> <div class="determinationofactivecomponents" title="sec"> <div class="title" tagx="title" title="title">
Determination of active components</div> <p id="Par54">The root samples were dried at 60â€‰Â°C for 72â€‰h to constant weight (it has been confirmed that glycyrrhizic acid (GIA) and liquiritin (LI) do not decompose at this temperature [<a href="#CR63">63</a>]). The dried root samples were ground to a powder with a pestle and mortar and passed through a 60 mesh sieve. An aliquot (0.2â€‰g) of powdered root sample was extracted with 71% chromatographic methanol in an ultrasonic bath (250â€‰W, 40â€‰kHz) at room temperature [<a href="#CR64">64</a>]. The extract was then centrifuged at 12,000â€‰rpm for 10â€‰min and the supernatant was filtered (0.22-Î¼m pore size) (Agilent, USA). The GIA and LI contents in the dried root samples (0.2â€‰g) of the medicinal licorices were determined by high-performance liquid chromatography (HPLC, Agilent-1260 Infinity, USA) using an Agilent ZORBAX SB-C18 column (150â€‰mmâ€‰Ã—â€‰4.6â€‰mm, 5â€‰Î¼m) with mobile phase (chromatographic methanol: ultra-pure water: 36% glacial acetic acidâ€‰=â€‰71:28:1) and mobile phase (acetonitrile:0.5% glacial acetic acidâ€‰=â€‰1:4) respectively, and a gradient elution flow rate of 1.0â€‰mLâ€¢min-1. GIA and LI were detected at 254â€‰nm and 276â€‰nm, respectively. The injection volume was 5â€‰Î¼L and the column temperature was 30â€‰Â°C. The GIA and LI reference materials (CAS#1405-86-3 and CAS#551â€"15-5, respectively) were purchased from Solarbio and used for calibration purposes. The total flavonoid content (GTF) in medicinal licorices was determined by ultraviolet spectrophotometry at 334â€‰nm with the LI standard (CAS#551â€"15-5) as the control.</p> </div> <div class="dnaextractionandlibraryconstruction" title="sec"> <div class="title" tagx="title" title="title">
DNA extraction and library construction</div> <p id="Par55">After immersion in liquid nitrogen, genomic DNA was extracted from the samples using the DNA Quick Plant System kit (Tiangen, China) according to the manufacturerâ€™s instructions. The purity and concentration of DNA were evaluated using a NanoDrop2000 (Thermo Fisher Scientific, USA). According to the concentration, each DNA sample was diluted a final concentration to 1â€‰ng/Î¼L with sterile distilled water for use as a DNA template. The 16S rDNA genes of the V4 region were amplified using specific primers (515F: 5â€²-GTGCCAGCMGCCGCGGTAA-3â€² and 806R: 5â€²-GGACTACHVGGGTWTCTAAT-3â€²) with barcodes [<a href="#CR65">65</a>]. PCR analyses were carried out with PhusionÂ® High-Fidelity PCR Master Mix and GC Buffer (New England Biolabs) to ensure amplification efficiency and accuracy. PCR runs started at at 95â€‰Â°C for 3â€‰min, followed by 30â€‰cycles of 95â€‰Â°C for 30â€‰s, 55â€‰Â°C for 30â€‰s, 72â€‰Â°C for 30â€‰s and a final extension step at 72â€‰Â°C for 5â€‰min.</p> <p id="Par56">The PCR product was mixed with the same volume of 1Ã— loading buffer (containing SYBR green) and then was detected by 2% agarose gel electrophoresis. The PCR product was purified from the target strip using a Gel Extraction Kit (Qiagen, Germany). The libraries were constructed using a TruSeqÂ®DNA PCR-Free Sample Preparation Kit (Illumina, USA) according to the manufacturerâ€™s instructions, and index codes were added. The library quality was assessed on the QubitÂ® 2.0 Fluorometer (Thermo Scientific) and Agilent Bioanalyzer 2100 system. Finally, Amplicon sequencing was performed using the Illumina HiSeq2500 platforms at the Beijing Compass Biotechnology Co., Ltd. (Beijing, China).</p> </div> <div class="bioinformaticsanalysis" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatics analysis</div> <p id="Par57">Paired-end reads were assigned to samples based on their unique barcode and truncated by cutting off the barcode and primer sequence. The paired-end reads of each sample were spliced using FLASH (V1.2.7) [<a href="#CR66">66</a>] and then assembled to generate raw tags. To avoid the influence of non-microbiota sequences (such as, chloroplast and mitochondrial sequences), the raw sequences were further filtered by QIIME (V1.9.1) [<a href="#CR67">67</a>] to remove non-microbiota taxa before subsequent analysis. Then raw tags were subjected to a strict quality filtering process using QIIME to obtain high-quality clean tags [<a href="#CR68">68</a>]. Effective tags were obtained by comparison of the clean tags sequence with the UCHIME Algorithm [<a href="#CR69">69</a>] and Gold database to detect and remove chimeric sequences.</p> <p id="Par58">UPARSE software (UPARSE v7.0.1001) was used to cluster the effective tags of all samples into the same operational taxonomic units (OTUs) with â‰¥97% identity, and taking the sequence with the highest frequency as the representative sequence of each OTU. The taxonomic information for each representative sequence was annotated using the SILVA database (threshold value set to 0.8â€"1), and multiple sequence alignment was performed using MUSCLE (Version 3.8.31) software to study the phylogenetic relationship of the representative sequences of OTUs among the 27 root samples. OTU abundance information was normalized using a standard sequence number corresponding to the sample with the lowest sequences (49,096 reads for sample E.W.1). Subsequent analysis of alpha diversity and beta diversity were performed based on this output normalized data. The raw sequence reads have been deposited in the Sequence Read Archive (SRA) database of NCBI (SRR12483393 to SRR12483419)(<a href="https://dataview.ncbi.nlm.nih.gov/object/PRJNA658331?reviewer=pi4poruikv7bnsshn4kaq4n3tc">https://dataview.ncbi.nlm.nih.gov/object/PRJNA658331?reviewer=pi4poruikv7bnsshn4kaq4n3tc</a>).</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par59">The average of the three replicate samples was used to generate the histograms of relative abundance (phylum and genus). R software (Version 2.15.3) was used for alpha diversity index and beta diversity analyses, rarefaction curve generation, Spearman correlation analysis of heat maps, Mantel tests and distance-based redundancy analysis (db-RDA). LEfSe software was used for linear discriminant analysis (LDA) and effect size (LEfSe) analysis with the default filtering value of LDA score set at 4. Statistical analysis (including one-way analysis of variance (ANOVA) and Spearman correlation analysis) were performed with SPSS 19.0 (IBM Inc., Armonk, USA), two-way ANOVA was used to analyze the effects of different year and licorice species on the contents of bioactive compounds. Spearman correlations (r) were run among the bioactive compounds, the soil physicochemical properties and nutritional components of leaves.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="12866_2020_1977_MOESM1_ESM.docx">LINK</a> <p><b>Additional file 1: Table S1.</b> Soil and plant factors of samples. Group name (E, R and S: years 1, 2, and 3, respectively; W, G and D: <i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i>, and <i>Glycyrrhiza inflata,</i> respectively). <b>Table S2.</b> Sequencing results of each sample. Raw reads refers to the sequence filtering out low-quality bases; clean reads refers to the sequence finally used for subsequent analysis after filtering chimeras; base refers to the number of bases of final clean reads; Avglen refers to the average length of clean reads. Q20 refers to the percentage of bases whose quality value is greater than 20 (sequencing error rate is less than 1%); GC (%) refers to the content of GC bases in clean reads; effective (%) refers to the percentage of the number of clean reads and the number of raw reads. Sample name: E, R and S: years 1, 2, and 3, respectively; W, G and D: <i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i> and <i>Glycyrrhiza inflata</i>, respectively; the third number representing the replicate number. <b>Table S3.</b> Composition information of dominant bacteria at each classification level. Others: The sum of the undefined and unannotated parts. Group name: E, R and S: years 1, 2, and 3, respectively; W, G and D: <i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i> and <i>Glycyrrhiza inflata</i>, respectively; the third number representing the replicate number. <b>Table S4.</b> The alpha diversity indices in each group. Community richness was identified using the Chao1 and ACE estimator. Community diversity was identified using the Shannon and Simpson indexes. Sequencing depth was characterized by Goodâ€™s coverage, goodâ€™s coverage estimator values ranged from 99.6 to 99.8%, indicating that the sequence numbers per group were high enough. Group: E, R and S: years 1, 2, and 3, respectively; W, G and D: <i>Glycyrrhiza uralensis</i>, <i>Glycyrrhiza glabra</i> and <i>Glycyrrhiza inflata</i>, respectively. <b>Table S5.</b> Spearman correlation analyses testing the relationship between the Biomarker and the relationship between the physicochemical properties of soil, leaf nutrients and bioactive compounds of licorice root. The values are the correlation coefficients. ** means <i>P</i>â€‰<div class="â€‰0.01;_UNKNOWN">
Pâ€‰<div class="â€‰0.05._UNKNOWN"><b>Table S6.</b> Correlations between environmental factors and endophytic bacterial communities in each group. Variable is the information of environmental factors, <i>r</i> is the correlation coefficient, and <i>P</i> value is the <i>p</i>-value of significance test. The larger <i>r</i> value is, the greater the correlation between environmental factors and species abundance information is. <i>P</i>â€‰<div class="â€‰0.05_UNKNOWN">  </div></div></div></p></div> </div> <p /> </div> </div>  <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
HPLC</div> <div class="def" title="def"> <p id="Par4">High-performance liquid chromatography</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LB</div> <div class="def" title="def"> <p id="Par5">Luria-Bertani</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GIA</div> <div class="def" title="def"> <p id="Par6">Glycyrrhizic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LI</div> <div class="def" title="def"> <p id="Par7">Liquiritin</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GTF</div> <div class="def" title="def"> <p id="Par8">Total flavonoid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SOM</div> <div class="def" title="def"> <p id="Par9">Organic matter</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
STN</div> <div class="def" title="def"> <p id="Par10">Total nitrogen</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
STP</div> <div class="def" title="def"> <p id="Par11">Total phosphorus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
STK</div> <div class="def" title="def"> <p id="Par12">Total potassium</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TS</div> <div class="def" title="def"> <p id="Par13">Total salt</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SNN</div> <div class="def" title="def"> <p id="Par14">Nitrate nitrogen</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SAN</div> <div class="def" title="def"> <p id="Par15">Ammonium nitrogen</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
POC</div> <div class="def" title="def"> <p id="Par16">Organic carbon</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PWC</div> <div class="def" title="def"> <p id="Par17">Water content</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PTN</div> <div class="def" title="def"> <p id="Par18">Total nitrogen</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PTP</div> <div class="def" title="def"> <p id="Par19">Total phosphorus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PTK</div> <div class="def" title="def"> <p id="Par20">Total potassium</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CF</div> <div class="def" title="def"> <p id="Par21">Crude fiber</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
OTUs</div> <div class="def" title="def"> <p id="Par22">Operational taxonomic units</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
db-RDA</div> <div class="def" title="def"> <p id="Par23">Distance-based redundancy analysis</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LDA</div> <div class="def" title="def"> <p id="Par24">Linear discriminant analysis</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LEfSe</div> <div class="def" title="def"> <p id="Par25">Linear discriminant analysis effect size</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ANOVA</div> <div class="def" title="def"> <p id="Par26">Analysis of variance</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>Hanli Dang and Tao Zhang contributed equally to this work and should be considered co-first authors</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p><b>Supplementary information</b> accompanies this paper at 10.1186/s12866-020-01977-3.</p> </div> <div class="ack" title="ack"> <p>In this study, we would like to thank professor L.Z for hers guidance, all the authors for their joint efforts. We would like to thank many graduate students who are not listed as co-authors and to the staff involved in the maintenance of farmland and the collection of soil samples.</p> </div> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>L.Z., H.D. and T.Z., designed the experiment. H.D and T.Z. performed methodology, data analysis, investigated and were a major contributor in writing the manuscript. G. Li., Y. Mu, X. Lv. and Z. W. collected the samples and modified the manuscript. H.D. and T.Z. contributed equally to this work and should be considered co-first authors. All authors read and approved the final manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This study was financially supported by the Chinese National Basic Research Program (2014CB954203) and the National Natural Science Foundation of China (31360139, 41561010 and 31560177). All funding agencies provided financial support only and did not participate in the design of experiments, sampling and manuscript writing tasks.</p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>All data generated or analysed during this study are included in this published article and its supplementary information files.</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par60">Not applicable.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p id="Par61">Not applicable.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par62">The authors declare that they have no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrison</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griffin</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span></span><span class="mixed-article-title" title="mixed-article-title">The diversity and distribution of endophytes across biomes, plant phylogeny and host tissues: how far have we come and where do we go from here?</span><span class="source" tagx="source" title="source">Environ Microbiol</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">6</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1462-2920.14968">10.1111/1462-2920.14968</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hardoim</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Overbeek</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berg</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pirttil? AM, Compant S, Campisano a, D?Ring M, Sessitsch a. the hidden world within plants. Ecological and evolutionary considerations for defining functioning of microbial Endophytes</span><span class="source" tagx="source" title="source">Microbiol Mol Biol Rev</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">79</span><span class="fpage" tagx="fpage" title="fpage">293</span><span class="lpage" tagx="lpage" title="lpage">320</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/MMBR.00050-14">10.1128/MMBR.00050-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26136581">26136581</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryan</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kieran</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashley</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryan</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dowling</span><span class="given-names" tagx="given-names" title="given-names">DN</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial endophytes: recent developments and applications</span><span class="source" tagx="source" title="source">FEMS Microbiol Lett</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">278</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1574-6968.2007.00918.x">10.1111/j.1574-6968.2007.00918.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18034833">18034833</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulz</span><span class="given-names" tagx="given-names" title="given-names">BJE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boyle</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christine</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sieber</span><span class="given-names" tagx="given-names" title="given-names">TN</span></span></span><span class="source" tagx="source" title="source">Microbial root endophytes</span><span class="year" tagx="year" title="year">2006</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walia</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guleria</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chauhan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mehta</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="source" tagx="source" title="source">Endophytic Bacteria</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">61</span><span class="lpage" tagx="lpage" title="lpage">93</span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wani</span><span class="given-names" tagx="given-names" title="given-names">ZA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashraf</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohiuddin</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riyaz-Ul-Hassan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant-endophyte symbiosis, an ecological perspective</span><span class="source" tagx="source" title="source">Appl Microbiol Biotechnol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">2955</span><span class="lpage" tagx="lpage" title="lpage">2965</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00253-015-6487-3">10.1007/s00253-015-6487-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25750045">25750045</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jasim</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jimtha</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jyothis</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radhakrishnan</span><span class="given-names" tagx="given-names" title="given-names">EK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant growth promoting potential of endophytic bacteria isolated from Piper nigrum</span><span class="source" tagx="source" title="source">Plant Growth Regul</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10725-013-9802-y">10.1007/s10725-013-9802-y</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lata</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gond</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Induction of abiotic stress tolerance in plants by endophytic microbes</span><span class="source" tagx="source" title="source">Lett Appl Microbiol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">268</span><span class="lpage" tagx="lpage" title="lpage">276</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/lam.12855">10.1111/lam.12855</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29359344">29359344</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yaish</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antony</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glick</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of endophytic plant growth-promoting bacteria from date palm tree (<i>Phoenix dactylifera</i> L.) and their potential role in salinity tolerance</span><span class="source" tagx="source" title="source">Antonie Van Leeuwenhoek</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">107</span><span class="fpage" tagx="fpage" title="fpage">1519</span><span class="lpage" tagx="lpage" title="lpage">1532</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10482-015-0445-z">10.1007/s10482-015-0445-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25860542">25860542</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bibi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yasir</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">YR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diversity and characterization of Endophytic Bacteria associated with tidal flat plants and their antagonistic effects on Oomycetous plant pathogens</span><span class="source" tagx="source" title="source">Plant Pathol J</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">20</span><span class="lpage" tagx="lpage" title="lpage">31</span><span class="pub-id"><a href="https://dx.doi.org/10.5423/PPJ.OA.06.2011.0123">10.5423/PPJ.OA.06.2011.0123</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lodewyckx</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vangronsveld</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Porteous</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">ERB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taghavi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mezgeay</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Der Lelie</span><span class="given-names" tagx="given-names" title="given-names">DV</span></span></span><span class="mixed-article-title" title="mixed-article-title">Endophytic bacteria and their potential applications</span><span class="source" tagx="source" title="source">Crit Rev Plant Sci</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">583</span><span class="lpage" tagx="lpage" title="lpage">606</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/0735-260291044377">10.1080/0735-260291044377</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Afzal</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iqbal</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">QM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plantâ€"bacteria partnerships for the remediation of hydrocarbon contaminated soils</span><span class="source" tagx="source" title="source">Chemosphere.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">1317</span><span class="lpage" tagx="lpage" title="lpage">1332</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chemosphere.2012.09.045">10.1016/j.chemosphere.2012.09.045</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23058201">23058201</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senthilkumar</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anandham</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madhaiyan</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venkateswaran</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sa</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="source" tagx="source" title="source">Endophytic Bacteria: Perspectives and Applications in Agricultural Crop Production</span><span class="year" tagx="year" title="year">2011</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin Heidelberg</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chebotar</span><span class="given-names" tagx="given-names" title="given-names">VK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shcherbakov</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maslennikova</span><span class="given-names" tagx="given-names" title="given-names">SN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaplatkin</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanarskiy</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zavalin</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Endophytic bacteria of woody plants as the basis of complex microbial preparations for agriculture and forestry</span><span class="source" tagx="source" title="source">Russ Agric Sci</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">339</span><span class="lpage" tagx="lpage" title="lpage">342</span><span class="pub-id"><a href="https://dx.doi.org/10.3103/S1068367416050037">10.3103/S1068367416050037</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berg</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plantâ€"microbe interactions promoting plant growth and health: perspectives for controlled use of microorganisms in agriculture</span><span class="source" tagx="source" title="source">Appl Microbiol Biotechnol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">84</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00253-009-2092-7">10.1007/s00253-009-2092-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19568745">19568745</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartels</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sunkar</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Drought and salt tolerance in plants</span><span class="source" tagx="source" title="source">Crit Rev Plant Sci</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">23</span><span class="lpage" tagx="lpage" title="lpage">58</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/07352680590910410">10.1080/07352680590910410</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amann</span><span class="given-names" tagx="given-names" title="given-names">RI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ludwig</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schleifer</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic identification and in situ detection of individual microbial cells without cultivation</span><span class="source" tagx="source" title="source">Microbiol Rev</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">143</span><span class="lpage" tagx="lpage" title="lpage">169</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/MMBR.59.1.143-169.1995">10.1128/MMBR.59.1.143-169.1995</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7535888">7535888</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monot</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orgeur</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Camiade</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brehier</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dupuy</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">COV2HTML: a visualization and analysis tool of bacterial next generation sequencing (NGS) data for postgenomics life scientists</span><span class="source" tagx="source" title="source">Omics A J Integr Biol</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">184</span><span class="lpage" tagx="lpage" title="lpage">195</span><span class="pub-id"><a href="https://dx.doi.org/10.1089/omi.2013.0119">10.1089/omi.2013.0119</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akinsanya</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goh</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ting</span><span class="given-names" tagx="given-names" title="given-names">ASY</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metagenomics study of endophytic bacteria in Aloe vera using next-generation technology</span><span class="source" tagx="source" title="source">Genomics Data</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">159</span><span class="lpage" tagx="lpage" title="lpage">163</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gdata.2015.09.004">10.1016/j.gdata.2015.09.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26697361">26697361</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of three officinal Chinese pharmacopoeia species of Glycyrrhiza based on separation and quantification of triterpene saponins and chemometrics analysis</span><span class="source" tagx="source" title="source">Food Chem</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">141</span><span class="fpage" tagx="fpage" title="fpage">1681</span><span class="lpage" tagx="lpage" title="lpage">1689</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodchem.2013.05.073">10.1016/j.foodchem.2013.05.073</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23870878">23870878</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asl</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hosseinzadeh</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Review of pharmacological effects of Glycyrrhiza sp. and its bioactive compounds</span><span class="source" tagx="source" title="source">Phytother Res</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">709</span><span class="lpage" tagx="lpage" title="lpage">724</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ptr.2362">10.1002/ptr.2362</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xueyan</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wenquan</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shengli</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weidong</span><span class="given-names" tagx="given-names" title="given-names">LI</span></span></span><span class="mixed-article-title" title="mixed-article-title">Review of pharmacological effects of Glycyrrhiza radix and its bioactive compounds</span><span class="source" tagx="source" title="source">Chin J Chin Mater Med</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">2695</span><span class="lpage" tagx="lpage" title="lpage">2700</span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baltina</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemical modification of glycyrrhizic acid as a route to new bioactive compounds for medicine</span><span class="source" tagx="source" title="source">Curr Med Chem</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">155</span><span class="lpage" tagx="lpage" title="lpage">171</span><span class="pub-id"><a href="https://dx.doi.org/10.2174/0929867033368538">10.2174/0929867033368538</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12570715">12570715</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Honda S, Tominaga Y, Yokota S. Licorice Flavonoids. USA: Wiley-Blackwell; 2009.</span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aoki</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Honda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishida</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitano</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arai</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yokota</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakagawa</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asakura</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakai</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Suppression by licorice flavonoids of abdominal fat accumulation and body weight gain in high-fat diet-induced obese C57BL/6J mice</span><span class="source" tagx="source" title="source">Biosci Biotechnol Biochem</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">206</span><span class="lpage" tagx="lpage" title="lpage">214</span><span class="pub-id"><a href="https://dx.doi.org/10.1271/bbb.60463">10.1271/bbb.60463</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17213668">17213668</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">The research Progress of the correlation between growth development and dynamic accumulation of the effective components in medicinal plants</span><span class="source" tagx="source" title="source">Chinese Bull Bot</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">293</span><span class="lpage" tagx="lpage" title="lpage">304</span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Da-Cheng</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pei-Gen</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genomics and Evolution in Traditional Medicinal Plants: Road to a Healthier Life</span><span class="source" tagx="source" title="source">Evol Bioinformatics Online</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">197</span><span class="lpage" tagx="lpage" title="lpage">212</span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of nitrogen source and phosphate concentration on biomass and metabolites accumulation in adventitious root culture of Glycyrrhiza uralensis Fisch</span><span class="source" tagx="source" title="source">Acta Physiol Plant</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">915</span><span class="lpage" tagx="lpage" title="lpage">921</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11738-013-1470-z">10.1007/s11738-013-1470-z</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">ZR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">WQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The underground part growth distribution pattern of Glycyrrhiza uralensis and its effects on glycyrrhizinic acid content</span><span class="source" tagx="source" title="source">Chin J Chin Mater Med</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">305</span><span class="lpage" tagx="lpage" title="lpage">309</span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Wen-Zhi AN, Ling-Min Z, Jian-Jun X, Xi-Yu Z, Fa-Yuan Z, Yong-Gang L. Study on the root distribution pattern and biomass of the cultivated Glycyrrhiza uralensis. Pratacultural Ence. 2007;7:51â€"4.</span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ming</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ainong</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiaojun</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leijie</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">Growth and accumulation of active components of licorice in different growth years in semiarid regions of middle of Gansu Province</span><span class="source" tagx="source" title="source">Acta Agriculturae Boreali-occidentalis Sinica</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1522</span><span class="lpage" tagx="lpage" title="lpage">1528</span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Efferth</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koch</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Complex interactions between phytochemicals. The multi-target therapeutic concept of Phytotherapy</span><span class="source" tagx="source" title="source">Curr Drug Targets</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">122</span><span class="lpage" tagx="lpage" title="lpage">132</span><span class="pub-id"><a href="https://dx.doi.org/10.2174/138945011793591626">10.2174/138945011793591626</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20735354">20735354</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demain</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">The natural functions of secondary metabolites</span><span class="source" tagx="source" title="source">Adv Biochem Eng Biotechnol</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">39</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11036689">11036689</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bont Z, ZÃ¼st T, CCM A, Huber M, Erb M. Heritable variation in root secondary metabolites is associated with recent climate. J Ecol. 2020. 10.1111/1365-2745.13441.</span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verma</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shukla</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Impact of various factors responsible for fluctuation in plant secondary metabolites</span><span class="source" tagx="source" title="source">J Appl Res Med Aromatic Plants</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">105</span><span class="lpage" tagx="lpage" title="lpage">113</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jarmap.2015.09.002">10.1016/j.jarmap.2015.09.002</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakamoto</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of Root-Zone Temperature on Growth and Quality of Hydroponically Grown Red Leaf Lettuce (<i>Lactuca sativa</i> L. cv. Red Wave)</span><span class="source" tagx="source" title="source">Am J Plant Ences</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">2350</span><span class="lpage" tagx="lpage" title="lpage">2360</span><span class="pub-id"><a href="https://dx.doi.org/10.4236/ajps.2015.614238">10.4236/ajps.2015.614238</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zahra</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masoud</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sohani</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Triterpenoid gene expression and phytochemical content in Iranian licorice under salinity stress</span><span class="source" tagx="source" title="source">Protoplasma</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">256</span><span class="fpage" tagx="fpage" title="fpage">827</span><span class="lpage" tagx="lpage" title="lpage">837</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00709-018-01340-4">10.1007/s00709-018-01340-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30623261">30623261</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mei</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative proteomic analysis reveals the molecular mechanisms underlying the accumulation difference of bioactive constituents in Glycyrrhiza uralensis Fisch under salt stress</span><span class="source" tagx="source" title="source">J Agric Food Chem</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">1480</span><span class="lpage" tagx="lpage" title="lpage">1493</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jafc.9b04887">10.1021/acs.jafc.9b04887</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31899641">31899641</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chun-Yang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dan</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jun-Ling</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen-Quan</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei-Dong</span><span class="given-names" tagx="given-names" title="given-names">LI</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of NaCl Stress on Growth,Physiological Index and Content of Effective Composition of Glycyrrhiza uralensis</span><span class="source" tagx="source" title="source">Chin J Exp Tradit Med Formulae</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">118</span><span class="lpage" tagx="lpage" title="lpage">122</span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Xiu-Hong Y, Jian-Min LI, Xue-Hui D, Liu-Sheng D. Effects of Exogenous Glycyrrhizinic Acid on the Seedling Growth,Glycyrrhizinic Acid Content of Roots and Some Physiological Indexes of Glycyrrhiza uralensis Fisch. Seedling under NaCl Stress. Plant Physiol Commun. 2006;3:441â€"4.</span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anthony</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">M., glass. Nitrogen use efficiency of crop plants: physiological constraints upon nitrogen absorption</span><span class="source" tagx="source" title="source">Crit Rev Plant Sci</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">453</span><span class="lpage" tagx="lpage" title="lpage">470</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/07352680390243512">10.1080/07352680390243512</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Wenmei P, Senjun Z, Jing W. Effect of different nitrogen forms and ratios on yield and quality of licorice. Chin Agric Ence Bull. 2011;28:184â€"7.</span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Host legume-exuded antimetabolites optimize the symbiotic rhizosphere</span><span class="source" tagx="source" title="source">Mol Microbiol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">73</span><span class="fpage" tagx="fpage" title="fpage">507</span><span class="lpage" tagx="lpage" title="lpage">517</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2958.2009.06790.x">10.1111/j.1365-2958.2009.06790.x</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanna</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leone</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gehong</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kristina</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RL</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biogeography of symbiotic and other endophytic bacteria isolated from medicinal Glycyrrhiza species in China</span><span class="source" tagx="source" title="source">FEMS Microbiol Ecol</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">79</span><span class="fpage" tagx="fpage" title="fpage">46</span><span class="lpage" tagx="lpage" title="lpage">68</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1574-6941.2011.01198.x">10.1111/j.1574-6941.2011.01198.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22066910">22066910</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohamad</span><span class="given-names" tagx="given-names" title="given-names">OAA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin-Biao</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaimaa</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jian-Wei</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasulov</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yong-Hong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedlund</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen-Jun</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of the Antimicrobial Activity of Endophytic Bacterial Populations From Chinese Traditional Medicinal Plant Licorice and Characterization of the Bioactive Secondary Metabolites Produced by Bacillus atrophaeus Against Verticillium dahliae</span><span class="source" tagx="source" title="source">Front Microbiol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">924</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2018.00924">10.3389/fmicb.2018.00924</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29867835">29867835</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Xiao-Li R. Isolation of Endophytic Bacteria from Glycyrrhiza and identifying of antagonistic Bacteria. Microbiology. 2007;4:700â€"4.</span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomes</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pereira</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benhadi</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lino-Neto</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baptista</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Endophytic and epiphytic Phyllosphere fungal communities are shaped by different environmental factors in a Mediterranean ecosystem</span><span class="source" tagx="source" title="source">Microb Ecol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">668</span><span class="lpage" tagx="lpage" title="lpage">679</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00248-018-1161-9">10.1007/s00248-018-1161-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29500493">29500493</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lou</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Endophytic bacterial diversity in Glycyrrhiza inflata bat. From Xinjiang by culture-independent method</span><span class="source" tagx="source" title="source">Acta Microbiol Sin</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">718</span><span class="lpage" tagx="lpage" title="lpage">725</span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi-Hao</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai-Fu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen-Xin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Study on screening of high efficient strains of rhizobium on Glycyrrhiza</span><span class="source" tagx="source" title="source">J Heb Acad Ences</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">58</span><span class="lpage" tagx="lpage" title="lpage">61</span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MarÃ­n</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="source" tagx="source" title="source">Proteobacteria</span><span class="year" tagx="year" title="year">2015</span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okubo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikeda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaneko</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitsui</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tabata</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minamisawa</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nodulation-dependent communities of culturable bacterial endophytes from stems of field-grown soybeans</span><span class="source" tagx="source" title="source">Microbes Environ</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">253</span><span class="lpage" tagx="lpage" title="lpage">258</span><span class="pub-id"><a href="https://dx.doi.org/10.1264/jsme2.ME09125">10.1264/jsme2.ME09125</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21566381">21566381</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MarÃ­a</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pieckenstain</span><span class="given-names" tagx="given-names" title="given-names">FL</span></span></span><span class="mixed-article-title" title="mixed-article-title">The communities of tomato (Solanum lycopersicum L.) leaf endophytic bacteria, analyzed by 16S-ribosomal RNA gene pyrosequencing</span><span class="source" tagx="source" title="source">FEMS Microbiol Lett</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">187</span><span class="lpage" tagx="lpage" title="lpage">194</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1574-6968.12377">10.1111/1574-6968.12377</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruixian</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ping</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wenyu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Illumina-based analysis of endophytic bacterial diversity of tree peony (Paeonia sect. Moutan) roots and leaves</span><span class="source" tagx="source" title="source">Braz J Microbiol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">695</span><span class="lpage" tagx="lpage" title="lpage">705</span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Willems</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">The taxonomy of rhizobia: an overview</span><span class="source" tagx="source" title="source">Plant Soil</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">287</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11104-006-9058-7">10.1007/s11104-006-9058-7</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">GH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">ZX</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">YZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindsroem</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Strain Mesorhizobium sp. CCNWGX035: a stress-tolerant isolate from Glycyrrhiza glabra displaying a wide host range of nodulation</span><span class="source" tagx="source" title="source">Pedosphere.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">102</span><span class="lpage" tagx="lpage" title="lpage">112</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1002-0160(07)60108-8">10.1016/S1002-0160(07)60108-8</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Compant</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ClÃ©ment</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sessitsch</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant growth-promoting bacteria in the rhizo- and endosphere of plants: their role, colonization, mechanisms involved and prospects for utilization</span><span class="source" tagx="source" title="source">Soil Biol Biochem</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">669</span><span class="lpage" tagx="lpage" title="lpage">678</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2009.11.024">10.1016/j.soilbio.2009.11.024</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaiero</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mccall</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Best</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunfield</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inside the root microbiome: bacterial root endophytes and plant growth promotion</span><span class="source" tagx="source" title="source">Am J Bot</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">1738</span><span class="lpage" tagx="lpage" title="lpage">1750</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/ajb.1200572">10.3732/ajb.1200572</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23935113">23935113</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hooker</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Perception and modification of plant flavonoid signals by rhizosphere microorganisms</span><span class="source" tagx="source" title="source">Environ Microbiol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1867</span><span class="lpage" tagx="lpage" title="lpage">1880</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1462-2920.2006.01141.x">10.1111/j.1462-2920.2006.01141.x</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mathesius</span><span class="given-names" tagx="given-names" title="given-names">HU</span></span></span><span class="mixed-article-title" title="mixed-article-title">The role of flavonoids in root-rhizosphere signalling: opportunities and challenges for improving plant-microbe interactions</span><span class="source" tagx="source" title="source">J Exp Bot</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">3429</span><span class="lpage" tagx="lpage" title="lpage">3444</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/err430">10.1093/jxb/err430</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22213816">22213816</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Multiple responses of rhizobia to flavonoids during legume root infection</span><span class="source" tagx="source" title="source">Adv Bot Res</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">62</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0065-2296(04)41001-5">10.1016/S0065-2296(04)41001-5</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MÃ¡rton</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alison</span><span class="given-names" tagx="given-names" title="given-names">W-M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Effect of Root Exudate 7,4â€²-Dihydroxyflavone and Naringenin on Soil Bacterial Community Structure</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">16</span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bao SD. Soil Agro-chemistrical Analysis. China: China Agriculture Press; 2008. p. 22â€"196.</span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jing</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zuliang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yanpeng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guangxi</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chunsheng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiaojun</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of Abscisic acid on accumulation of five active components in root of Glycyrrhiza uralensis</span><span class="source" tagx="source" title="source">Molecules.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">1982</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules22010125">10.3390/molecules22010125</a></span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hao</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Improved phosphorus nutrition by arbuscular mycorrhizal symbiosis as a key factor facilitating glycyrrhizin and liquiritin accumulation in Glycyrrhiza uralensis</span><span class="source" tagx="source" title="source">Plant Soil</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">439</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">15</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11104-019-04022-z">10.1007/s11104-019-04022-z</a></span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berry</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ben Mahfoudh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loy</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Barcoded primers used in multiplex amplicon pyrosequencing Bias amplification</span><span class="source" tagx="source" title="source">Appl Environ Microbiol</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">612</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.07448-11">10.1128/AEM.07448-11</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanja</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steven</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">FLASH: fast length adjustment of short reads to improve genome assemblies</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">2957</span><span class="lpage" tagx="lpage" title="lpage">2963</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr507">10.1093/bioinformatics/btr507</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21903629">21903629</a></span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caporaso</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuczynski</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stombaugh</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bittinger</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bushman</span><span class="given-names" tagx="given-names" title="given-names">FD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costello</span><span class="given-names" tagx="given-names" title="given-names">EK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fierer</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PeÃ±a</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodrich</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gordon</span><span class="given-names" tagx="given-names" title="given-names">JI</span></span></span><span class="mixed-article-title" title="mixed-article-title">QIIME allows analysis of high-throughput community sequencing data</span><span class="source" tagx="source" title="source">Nat Methods</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">335</span><span class="lpage" tagx="lpage" title="lpage">336</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.f.303">10.1038/nmeth.f.303</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20383131">20383131</a></span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bokulich</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subramanian</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faith</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gevers</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gordon</span><span class="given-names" tagx="given-names" title="given-names">JI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knight</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mills</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caporaso</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Quality-filtering vastly improves diversity estimates from Illumina amplicon sequencing</span><span class="source" tagx="source" title="source">Nat Methods</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">57</span><span class="lpage" tagx="lpage" title="lpage">59</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.2276">10.1038/nmeth.2276</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23202435">23202435</a></span></span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robert</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brian</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jose Clemente</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">UCHIME improves sensitivity and speed of chimera detection</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">2194</span><span class="lpage" tagx="lpage" title="lpage">2200</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr381">10.1093/bioinformatics/btr381</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674">21700674</a></span></span></li> </ul> </div> </div>  </body></html>