<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PeerJ</span><span class="iso-abbrev" title="iso-abbrev">PeerJ</span><span class="publisher-id" title="publisher-id">peerj</span><span class="pmc" title="pmc">peerj</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PeerJ</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2167-8359</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">PeerJ Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Diego, USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7811785">7811785</a></span><span class="publisher-id" title="publisher-id">10687</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7717/peerj.10687">10.7717/peerj.10687</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Genomics</span>: <span class="subject" title="subject">Plant Science</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Comparative analysis and implications of the chloroplast genomes of three thistles (<i>Carduus</i> L., Asteraceae)</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">Joonhyung</span></span><a href="#aff-1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Do</span><span class="given-names" tagx="given-names" title="given-names">Hoang Dang Khoa</span></span><a href="#aff-1">1</a><a href="#aff-2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyun</span><span class="given-names" tagx="given-names" title="given-names">JongYoung</span></span><a href="#aff-1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Changkyun</span></span><a href="#aff-1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Joo-Hwan</span></span><span class="email" tagx="email" title="email">kimjh2009@gachon.ac.kr</span><a href="#aff-1">1</a></span><span class="citation_author_institution" id="aff-1">[1], <span class="institution" title="institution">Department of Life Science, Gachon University</span><span class="city" tagx="city" title="city">Seongnam</span><span class="state" tagx="state" title="state">Gyeonggi</span><span class="country" tagx="country" title="country">Korea</span></span><span class="citation_author_institution" id="aff-2">[2], <span class="institution" title="institution">Nguyen Tat Thanh Hi-Tech Institute, Nguyen Tat Thanh University</span><span class="city" tagx="city" title="city">Ho Chi Minh City</span><span class="country" tagx="country" title="country">Vietnam</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">Jonathan</span></span></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e10687</span><span class="history" title="history"><span class="received" title="received">received: 2020-2-28</span><span class="accepted" title="accepted">accepted: 2020-12-11</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Jung et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2021</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, reproduction and adaptation in any medium and for any purpose provided that it is properly attributed. For attribution, the original author(s), title, publication source (PeerJ) and either DOI or URL of the article must be cited.</span></span></div><a class="self-uri" href="https://peerj.com/articles/10687" title="self-uri">https://peerj.com/articles/10687</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p><i>Carduus</i>, commonly known as plumeless thistles, is a genus in the Asteraceae family that exhibits both medicinal value and invasive tendencies. However, the genomic data of <i>Carduus</i> (i.e., complete chloroplast genomes) have not been sequenced.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p>We sequenced and assembled the chloroplast genome (cpDNA) sequences of three <i>Carduus</i> species using the Illumina Miseq sequencing system and Geneious Prime. Phylogenetic relationships between <i>Carduus</i> and related taxa were reconstructed using Maximum Likelihood and Bayesian Inference analyses. In addition, we used a single nucleotide polymorphism (SNP) in the protein coding region of the <i>matK</i> gene to develop molecular markers to distinguish <i>C. crispus</i> from <i>C. acanthoides</i> and <i>C. tenuiflorus</i>.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>The cpDNA sequences of <i>C. crispus, C. acanthoides</i>, and <i>C. tenuiflorus</i> ranged from 152,342 bp to 152,617 bp in length. Comparative genomic analysis revealed high conservation in terms of gene content (including 80 protein-coding, 30 tRNA, and four rRNA genes) and gene order within the three focal species and members of subfamily Carduoideae. Despite their high similarity, the three species differed with respect to the number and content of repeats in the chloroplast genome. Additionally, eight hotspot regions, including <i>psbI-trnS_GCU</i>, <i>trnE_UUC-rpoB</i>, <i>trnR_UCU-trnG_UCC</i>, <i>psbC-trnS_UGA</i>, <i>trnT_UGU-trnL_UAA</i>, <i>psbT-psbN</i>, <i>petD-rpoA</i>, and <i>rpl16-rps3</i>, were identified in the study species. Phylogenetic analyses inferred from 78 protein-coding and non-coding regions indicated that <i>Carduus</i> is polyphyletic, suggesting the need for additional studies to reconstruct relationships between thistles and related taxa. Based on a SNP in <i>matK</i>, we successfully developed a molecular marker and protocol for distinguishing <i>C. crispus</i> from the other two focal species. Our study provides preliminary chloroplast genome data for further studies on plastid genome evolution, phylogeny, and development of species-level markers in <i>Carduus</i>.</p> </div> </div> <div class="kwd-group"> <div class="kwd" title="kwd"><i>Carduus crispus</i></div> <div class="kwd" title="kwd">
Chloroplast genome</div> <div class="kwd" title="kwd">
Invasive species</div> <div class="kwd" title="kwd">
Molecular markers</div> <div class="kwd" title="kwd">
Plumeless thistles</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Animal and Plant Quarantine Agency of the Republic of Korea and Gachon University</div> </div> <div class="funding-statement" title="funding-statement">
This research was funded by grants from scientific research (2017-2018) of the Animal and Plant Quarantine Agency of the Republic of Korea and the Gachon University research fund of 2019 (GCU-2019-0821). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p><i>Carduus</i> L. (subfamily Carduoideae; Asteraceae), commonly known as plumeless thistles, comprises 90 species native to Eurasia and Africa (<a href="#ref-1">Angiosperm Phylogeny Group, 2016</a>). Several <i>Carduus</i> species are invasive, noxious weeds on other continents (<a href="#ref-17">Doing, Biddiscombe &amp;amp; Knedlhans, 1969</a>). Four species, including <i>C. acanthoides</i> L. (spiny plumeless thistle), <i>C. tenuiflorus</i> Curtis (sheep thistle), <i>C. pycnocephalus</i> L. (Italian thistle), and <i>C. crispus</i> GuirÃ£o ex Nyman (welted thistle), all of which originate in Eurasia and Africa, are considered invasive in North America (<a href="#ref-19">Dunn, 1976</a>; <a href="#ref-65">Verloove, 2014</a>). <i>Carduus crispus</i>, also called curly plumeless thistle, is also considered an invasive species in Korea (<a href="#ref-32">Jung et al., 2017</a>). This species differs from other <i>Carduus</i> in having soft, sparsely arachnoid-hairy leaves with short marginal bristles, and apically recurved involucral bracts (<a href="#ref-61">Todorov et al., 2018</a>). Among <i>Carduus</i> species, <i>C. crispus</i> contains chemicals with the potential to treat various diseases (<a href="#ref-71">Xie, Li &amp;amp; Jia, 2005</a>; <a href="#ref-12">Davaakhuu, Sukhdolgor &amp;amp; Gereltu, 2010</a>; <a href="#ref-42">Lee et al., 2011</a>; <a href="#ref-63">Tunsag, Davaakhuu &amp;amp; Batsuren, 2011</a>). Specifically, certain compounds extracted from <i>C. crispus</i> have potential value in the treatment of obesity and cancer (<a href="#ref-12">Davaakhuu, Sukhdolgor &amp;amp; Gereltu, 2010</a>; <a href="#ref-42">Lee et al., 2011</a>). While <i>Carduus</i> has been studied from various perspectives (i.e.,Â invasion, phylogeny, and medicinal effects), its chloroplast genome has not been sequenced. It is therefore worthwhile to study the genome of <i>Carduus</i>, and particularly that of <i>C. crispus</i>, which has potential medicinal benefits.</p> <p>In most angiosperms, the chloroplast genome (cpDNA) contains genes essential to photosynthesis (<a href="#ref-60">Sugiura, 1992</a>). Genomic events (i.e.,Â gene deletion, inversion, or duplication) in cpDNA may provide information about speciesâ€™ evolutionary history (<a href="#ref-9">Cosner, Raubeson &amp;amp; Jansen, 2004</a>; <a href="#ref-15">Do &amp;amp; Kim, 2017</a>; <a href="#ref-25">Haberle et al., 2008</a>). For example, the Fabaceae includes clades that are characterised by large inversions and the loss of inverted repeat regions (<a href="#ref-6">Choi &amp;amp; Choi, 2017</a>). Inversions have also been recorded in the cpDNA of Asteraceae (<a href="#ref-35">Kim, Choi &amp;amp; Jansen, 2005</a>). Specifically, a large inversion comprising a 22.8Â kb sequence occurred simultaneously with a small inversion of a 3.3Â kb fragment; this event coincided with the split between major clades (excluding Barnadesioideae) in the evolution of Asteraceae. cpDNA data can also be used to develop molecular markers based on nucleotide polymorphisms (i.e.,Â single nucleotide polymorphism (SNP) markers and microsatellite markers). Molecular authentication has been reported for various plant species, with a focus on invasive plants, endangered species, and taxa with potential medicinal value (<a href="#ref-36">Kim et al., 2012</a>; <a href="#ref-30">Ishikawa, Sakaguchi &amp;amp; Ito, 2016</a>; <a href="#ref-44">Luo et al., 2016</a>; <a href="#ref-52">Park et al., 2016</a>; <a href="#ref-47">Marochio et al., 2017</a>; <a href="#ref-27">Han et al., 2018</a>; <a href="#ref-14">Do et al., 2019</a>). Specific regions of cpDNA have been identified for developing molecular markers in plants, including the commonly-used <i>matK</i> region (<a href="#ref-54">Poovitha et al., 2016</a>; <a href="#ref-66">Vu et al., 2017</a>). Among the Asteraceae, studies on molecular markers have been conducted for rubber dandelion (<i>Taraxacum kok-saghyz</i> LE Rodin), horseweed (<i>Conyza</i> sp.), Indian Chrysanthemum (<i>Chrysanthemum indicum</i> L.), the endemic herb <i>Aster savatieri</i> Makino, and the invasive plant <i>Tithonia diversifolia</i> (Hemsl.) A Gray (<a href="#ref-30">Ishikawa, Sakaguchi &amp;amp; Ito, 2016</a>; <a href="#ref-44">Luo et al., 2016</a>; <a href="#ref-74">Zhang et al., 2017</a>; <a href="#ref-47">Marochio et al., 2017</a>; <a href="#ref-27">Han et al., 2018</a>). In addition to the development of these molecular markers, complete cpDNA sequences have been reported for various Asteraceae species (<a href="#ref-35">Kim, Choi &amp;amp; Jansen, 2005</a>; <a href="#ref-7">Choi &amp;amp; Park, 2015</a>; <a href="#ref-68">Wang et al., 2015a</a>; <a href="#ref-67">Wang et al., 2015b</a>; <a href="#ref-73">Yun, Gil &amp;amp; Kim, 2017</a>; <a href="#ref-43">Liu et al., 2018</a>; <a href="#ref-45">Ma, Sun &amp;amp; Zhao, 2018</a>; <a href="#ref-59">Su et al., 2018</a>). cpDNA sequences may be used to elucidate the phylogeny of angiosperms from the clade that is basal to monocots and eudicots (<a href="#ref-1">Angiosperm Phylogeny Group, 2016</a>). Previous investigations into phylogenetic relationships among members of the Asteraceae have been conducted using a range of molecular data types, including <i>rbcL</i>, <i>ndhF</i>, <i>matK</i>, chloroplast DNA restriction sites, ITS sequence data, and nuclear loci (<a href="#ref-31">Jansen, Michaels &amp;amp; Palmer, 1991</a>; <a href="#ref-26">HÃ¤ffner &amp;amp; Hellwig, 1999</a>; <a href="#ref-22">Fu et al., 2016</a>; <a href="#ref-46">Mandel et al., 2019</a>). However, the paucity of available sequence data may have resulted in ambiguous relationships between <i>Carduus</i> and related taxa (<a href="#ref-26">HÃ¤ffner &amp;amp; Hellwig, 1999</a>; <a href="#ref-22">Fu et al., 2016</a>). In particular, ITS sequence data suggest that <i>C. leptacanthus</i> is sister to <i>Cirsium</i> and <i>Notobasis</i>, whereas another <i>Carduus</i> species is sister to <i>Cirsium</i> and <i>Tyrimmus</i> (<a href="#ref-26">HÃ¤ffner &amp;amp; Hellwig, 1999</a>). As such, clarification of relationships between <i>Carduus</i> and related species will require studies that include a larger number of <i>Carduus</i> species and different data types (i.e.,Â chloroplast and mitochondrial genomes).</p> <p>We used next-generation sequencing (NGS) to sequence and characterise the chloroplast genomes of <i>Carduus crispus</i>, <i>C. acanthoides</i>, and <i>C. tenuiflorus</i>, which exhibit both invasive tendencies and potential medical utility (particularly <i>C. crispus</i>). We then conducted comparative genomic analyses to explore genomic diversity among the three species with respect to highly variable regions, and the types and numbers of repeats. In addition, we reconstructed the formerly ambiguous relationship between <i>Carduus</i> and related taxa based on 78 protein-coding regions and non-coding sequences. Finally, we developed a specific molecular marker for <i>C. crispus</i> based on a SNP in the <i>matK</i> gene. This molecular marker provides useful information for managing <i>C. crispus</i> invasions, particularly with respect to the identification of immature (vegetative) individuals, which tend to be morphologically similar to other <i>Carduus</i> species (i.e.,Â having winged stems with apical spines and spiny leaves). This molecular marker may also support positive identification of <i>C. crispus</i> for medical usage.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials &amp;amp; Methods</div> <div class="taxonsampling,totaldnaextraction,chloroplastgenomeassembly,andcomparativeanalysis" title="sec"> <div class="title" tagx="title" title="title">
Taxon sampling, total DNA extraction, chloroplast genome assembly, and comparative analysis</div> <p> <div class="table-wrap_UNKNOWN" id="table-1" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7717/peerj.10687/table-1">10.7717/peerj.10687/table-1</a><span class="label" tagx="label" title="label">Table 1</span> </div></p><div class="title" tagx="title" title="title">
List of <i>Carduus</i> species for NGS and developing molecular marker.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1"><b>No.</b></th> <th rowspan="1" colspan="1"><b>Species</b></th> <th rowspan="1" colspan="1"><b>Voucher</b></th> <th rowspan="1" colspan="1"><b>Location</b></th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">Korea National Arboretum (LK0908)</td> <td rowspan="1" colspan="1">Mt. Seokbyeong, Imgye-myeon, Jeongseon-gun, Gangwon-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman*</td> <td rowspan="1" colspan="1">Korea National Arboretum (LK0943)</td> <td rowspan="1" colspan="1">Mt. cheongog, Hajang-myeon, Samcheok-si, Gangwon-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">Korea National Arboretum (LK1497)</td> <td rowspan="1" colspan="1">Mt. Nochu, Yeoryang-myeon, Jeongseon-gun, Gangwon-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">Korea National Arboretum (CNUFR0470)</td> <td rowspan="1" colspan="1">295, Sinseong-ri, Bukha-myeon, Jangseong-gun, Jeollanam-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">Korea National Arboretum (LK0430)</td> <td rowspan="1" colspan="1">Mt. cheongog, Imgye-myeon, Jeongseon-gun, Gangwon-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">6</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">National Institute of Biological Resources (NIBRVP0000619716)</td> <td rowspan="1" colspan="1">Mt. Jaam, Namhu-myeon, Andong-si, Gyeongsangbuk-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">7</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">National Institute of Biological Resources (NIBRVP0000601328)</td> <td rowspan="1" colspan="1">Sanghwa-ri, Danchon-myeon, Uiseong-gun, Gyeongsangbuk-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">8</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">National Institute of Biological Resources (NIBRVP0000524207)</td> <td rowspan="1" colspan="1">Mt. Beophwa, Yugu-eup, Gongju-si, Chungcheongnam-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">9</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">National Institute of Biological Resources (NIBRVP0000613325)</td> <td rowspan="1" colspan="1">Mt. Mani, Hwado-myeon, Ganghwa-gun, Incheon, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">10</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">National Institute of Biological Resources (NIBRVP0000580633)</td> <td rowspan="1" colspan="1">Mt. Jeonggwang, Mohyeon-eup, Cheoin-gu, Yongin-si, Gyeonggi-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">11</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">National Institute of Biological Resources (NIBRVP0000578538)</td> <td rowspan="1" colspan="1">Mt. Jangam, Pyeongchang-eup, Pyeongchang-gun, Gangwon-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">12</td> <td rowspan="1" colspan="1"><i>Carduus crispus</i> GuirÃ£o ex Nyman</td> <td rowspan="1" colspan="1">National Institute of Biological Resources (NIBRVP0000580501)</td> <td rowspan="1" colspan="1">Mt. Gwangdeok, Dongnam-gu, Cheonan-si, Chungcheongnam-do, Republic of Korea</td> </tr> <tr> <td rowspan="1" colspan="1">13</td> <td rowspan="1" colspan="1"><i>Carduus acanthoides</i> L.*</td> <td rowspan="1" colspan="1">Kim 2018-001, Nevada city, Ca, USA</td> <td rowspan="1" colspan="1">USA, California, Nevada city</td> </tr> <tr> <td rowspan="1" colspan="1">14</td> <td rowspan="1" colspan="1"><i>Carduus acanthoides</i> L.</td> <td rowspan="1" colspan="1">NewYork Botanical Garden (00532263)</td> <td rowspan="1" colspan="1">USA, Colorado, Pitkin Co., South side of State Highway 82, 5 miles W of Aspen, Airport., 2256 - 2256m</td> </tr> <tr> <td rowspan="1" colspan="1">15</td> <td rowspan="1" colspan="1"><i>Carduus acanthoides</i> L.</td> <td rowspan="1" colspan="1">NewYork Botanical Garden (00532244)</td> <td rowspan="1" colspan="1">USA, Wisconsin, Richland Co., 3 miles SE of Richland Center., 43.300443 -90.321019</td> </tr> <tr> <td rowspan="1" colspan="1">16</td> <td rowspan="1" colspan="1"><i>Carduus acanthoides</i> L.</td> <td rowspan="1" colspan="1">NewYork Botanical Garden (00532262)</td> <td rowspan="1" colspan="1">USA, Wyoming, Platte Co., 1402 - 1402m</td> </tr> <tr> <td rowspan="1" colspan="1">17</td> <td rowspan="1" colspan="1"><i>Carduus acanthoides</i> L.</td> <td rowspan="1" colspan="1">Carnegie Museum Herbarium (267884)</td> <td rowspan="1" colspan="1">Romania, Oltenia, distr. Dolj, inter vicos Lascar Catargiu et Popoveni ad Canalul colector, 75m</td> </tr> <tr> <td rowspan="1" colspan="1">18</td> <td rowspan="1" colspan="1"><i>Carduus acanthoides</i> L.</td> <td rowspan="1" colspan="1">Carnegie Museum Herbarium (528144)</td> <td rowspan="1" colspan="1">USA, Pennsylvania, Mifflin, Maple &amp;amp; Walnut Sts, Belleville</td> </tr> <tr> <td rowspan="1" colspan="1">19</td> <td rowspan="1" colspan="1"><i>Carduus acanthoides</i> L.</td> <td rowspan="1" colspan="1">United States National Herbarium (1944312)</td> <td rowspan="1" colspan="1">Czech Republic, Bohemia centralis: Paraha-Troja. In ruderatis</td> </tr> <tr> <td rowspan="1" colspan="1">20</td> <td rowspan="1" colspan="1"><i>Carduus acanthoides</i> L.</td> <td rowspan="1" colspan="1">United States National Herbarium (3419733)</td> <td rowspan="1" colspan="1">Ukraine, Prov. Czerkassy, prope opp. Umanj, in ruderatis</td> </tr> <tr> <td rowspan="1" colspan="1">21</td> <td rowspan="1" colspan="1"><i>Carduus tenuiflorus</i> Curtis*</td> <td rowspan="1" colspan="1">Kim 2018-002, Nevada city, Ca, USA</td> <td rowspan="1" colspan="1">USA, California, Nevada city</td> </tr> <tr> <td rowspan="1" colspan="1">22</td> <td rowspan="1" colspan="1"><i>Carduus tenuiflorus</i> Curtis</td> <td rowspan="1" colspan="1">NewYork Botanical Garden (00366662)</td> <td rowspan="1" colspan="1">Mexico, El Cercado. Santiago, N. L., 495â€"495 m</td> </tr> <tr> <td rowspan="1" colspan="1">23</td> <td rowspan="1" colspan="1"><i>Carduus tenuiflorus</i> Curtis</td> <td rowspan="1" colspan="1">Carnegie Museum Herbarium (519240)</td> <td rowspan="1" colspan="1">USA, California, Humboldt, Angels Ranch, toward Hungry Hollow, Bald Mountain</td> </tr> <tr> <td rowspan="1" colspan="1">24</td> <td rowspan="1" colspan="1"><i>Carduus tenuiflorus</i> Curtis</td> <td rowspan="1" colspan="1">Carnegie Museum Herbarium (282243)</td> <td rowspan="1" colspan="1">USA, California, Alameda, Berkeley</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><b>Notes.</b></p> </div> <div class="fn-type-other" title="other"> <p>Asterisks indicate samples for next generation sequencing (NGS) analysis.</p> </div> </div>  Leaves of <i>C. crispus</i>, <i>C. acanthoides</i>, and <i>C. tenuiflorus</i> were collected and dried in silica gel powder for NGS analysis. In addition, to test the efficiency of molecular markers (<a href="#table-1">TableÂ 1</a>), leaves of 22 individuals of the three species were sampled from herbaria at the Korea National Arboretum (KNA), National Institute of Biological Resources (NIBR), New York Botanical Garden (NYBG), Carnegie Museum Herbarium (CM), and the United States National Herbarium (US). A modified cetyl trimethylammonium bromide (CTAB) protocol was used to extract total DNA from collected samples (<a href="#ref-18">Doyle &amp;amp; Doyle, 1987</a>). High-quality DNA was used to conduct NGS using the Miseq platform and a Miseq Reagent Kit v3 (Illumina, Seoul, South Korea). Raw NGS data (2 Ã— 300 bp paired-end reads) were cleaned by cutting adapter sequences, removing duplicate and chimeric reads, and trimming ends with &amp;gt; 0.05 probability of error per base. Cleaning was conducted using Geneious Prime (<a href="#ref-33">Kearse et al., 2012</a>). Raw data were submitted to NCBI (accession number <a href="http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA645567">PRJNA645567</a>). Using Geneious Prime, filtered NGS data for each species were mapped to the reference chloroplast genome sequences of <i>Arctium lappa</i> (NCBI accession number <a href="http://www.ncbi.nlm.nih.gov/nuccore/MH375874">MH375874</a>), <i>Saussurea polylepis</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/MF695711">MF695711</a>), <i>Carthamus tinctorius</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KP404628">KP404628</a>), <i>Centaurea diffusa</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KJ690264">KJ690264</a>), <i>Silybum marianum</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KT267161">KT267161</a>), <i>Cirsium arvense</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KY562583">KY562583</a>), <i>Cynara humilis</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KP299292">KP299292</a>), and <i>Atractylodes chinensis</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/MG874805">MG874805</a>) to isolate cpDNA reads of which the similarity to reference was over 95%. Isolated cpDNA reads were then assembled using the de novo function in Geneious to create various contigs of chloroplast genome sequences. The newly created contigs were de novo re-assembled to construct complete cpDNA sequences for each focal species. We confirmed the results from Geneious Prime by reconstructing the complete cpDNA of <i>Carduus</i> using NOVOplasty and following Dierckxsens et al. (2017). Using Geneious Prime, gene content and order of sequenced cpDNA were annotated based on existing complete cpDNA sequences of other Asteraceae taxa. Annotations that had over 95% similarity in comparison with references were retained, and the start and stop codons in the protein coding regions were verified (<a href="#supp-2">DataÂ S1</a>). tRNA sequences were assessed using tRNA Scan-SE (<a href="#ref-5">Chan &amp;amp; Lowe, 2019</a>). Complete cpDNA sequences of the three species were submitted to GenBank; accession numbers were <a href="http://www.ncbi.nlm.nih.gov/nuccore/MK652229">MK652229</a> for <i>C. crispus</i>, <a href="http://www.ncbi.nlm.nih.gov/nuccore/MK652228">MK652228</a> for <i>C. acanthoides</i>, and <a href="http://www.ncbi.nlm.nih.gov/nuccore/MK652230">MK652230</a> for <i>C. tenuiflorus.</i> The cpDNA map was visualised using OGDraw (<a href="#ref-24">Greiner, Lehwark &amp;amp; Bock, 2019</a>). Complete chloroplast genomes of <i>Carduus</i> species were aligned with other Asteraceae and related species (<i>Nicotiana tabacum</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/NC_001879">NC_001879</a>) was used as an outgroup; <a href="#supp-4">Table S1</a>), and gene loss and rearrangement were identified using MAUVE (<a href="#ref-11">Darling, Mau &amp;amp; Perna, 2010</a>). In addition, Geneious Prime was used to calculate the pairwise identities of cpDNA sequences in the focal species. Small single repeats (SSRs) were analysed using Phobos embedded in Geneious Prime (<a href="#ref-8">Christoph, 2006â€"2010</a>). Minimum repeat numbers were 10, 5, 4, and 3 for mono-, di-, tri-, and tetra-nucleotides, respectively. REPuter (<a href="#ref-39">Kurtz et al., 2001</a>) was used to analyse the large repeat sequences (minimum length = 20 bp) in each species. To explore nucleotide diversity, 131 coding and non-coding regions were extracted from the complete cpDNA (<a href="#supp-4">Table S2</a>). Following alignment using MUSCLE embedded in Geneious Prime (<a href="#ref-20">Edgar, 2004</a>), aligned sequences were imported into DnaSP 6 (<a href="#ref-56">Rozas et al., 2017</a>) for calculation of Pi values.<p /> </div> <div class="phylogeneticanalysisofcarduusandrelatedtaxa" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis of <i>Carduus</i> and related taxa</div> <p>A total of 78 protein-coding regions were extracted from the complete cpDNA of the focal species and other related taxa (<a href="#supp-4">Table S1</a>). Sequences were aligned using MUSCLE embedded in Geneious Prime (<a href="#ref-20">Edgar, 2004</a>). We used jModeltest (<a href="#ref-53">Posada, 2008</a>) to find the best model for the aligned DNA sequences; GTR + I + R was selected as the most suitable model and was used in Maximum Likelihood (ML) and Bayesian Inference (BI) analyses. The ML analysis was conducted with the IQ-tree web server (<a href="http://iqtree.cibiv.univie.ac.at">http://iqtree.cibiv.univie.ac.at</a>), using 1,000 bootstrap replications to calculate branch support values (<a href="#ref-62">Trifinopoulos et al., 2016</a>). We used MrBayes v3.2 (<a href="#ref-55">Ronquist et al., 2012</a>) for BI analyses. The Markov chain Monte Carlo (MCMC) analysis was run for 1,000,000 generations, and a tree was assembled every 1000 generations. A 25% burn-in setting was used for summarising trees. Figtree v4.0 (<a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>) was used to visualise phylogenetic trees. Other datasets, including whole chloroplast genomes (excluding one IR region), non-coding regions of cpDNA, and hotspot regions derived from the cpDNA of <i>Carduus</i> species, were used in phylogenetic analysis in addition to protein-coding regions (<a href="#supp-4">TableÂ S1</a>). Analytical procedures for these additional datasets were identical to those used for the protein coding regions; however, we used the TVM+I+G model for the whole chloroplast genome (excluding one inverted repeat [IR] region) and all non-coding regions, and the TVM+G model for the hotspot regions dataset.</p> </div> <div class="snpidentification,primerdesign,andmultiplexpcr" title="sec"> <div class="title" tagx="title" title="title">
SNP identification, primer design, and multiplex PCR</div> <p>The complete <i>matK</i> gene, extracted from the cpDNA of the three focal species, was aligned using MUSCLE to identify SNPs (<a href="#ref-20">Edgar, 2004</a>; <a href="#supp-4">Fig. S1</a>). The selected SNP for <i>C. crispus</i> was then confirmed by aligning the available <i>matK</i> sequences of other <i>Carduus</i> species on NCBI to those of the focal species (<a href="#supp-4">Fig.Â S2</a>). Based on SNP data, primer pairs were designed using Primer3 to distinguish <i>C. crispus</i> from other <i>Carduus</i> species (<a href="#ref-64">Untergasser et al., 2012</a>). Primer sequences included matK_463F (5â€²-CATCTGGAAATCTTGGTTCAG-3â€²), matK_1162R (5â€²-GATGCCCCAATGCGTTACAA-3â€²), CD_SNP_F1 (5â€²-AATTCTTGCTTCAAAAGG GTCC- 3â€²), CD_SNP_R1 (5â€²-TTCCATTTATTCATCAA AAGATAC-3â€²), CD_SNP_F2 (5â€²-AATTCTTGCTTCAAAAGGGTCG-3â€²), and CD_SNP_R2 (5â€²-TTCCATTTATTCATCAAAAGATAG- 3â€²). The multiplex PCR of matK_463F, matK_1162R, CD_SNP_F1, and CD_SNP_R1 was designed to yield the 323 bp band for <i>C. crispus</i>, the 421 bp band for other <i>Carduus</i>, and the 700 bp band for all examined samples (<a href="#supp-4">Fig. S3</a>). By contrast, the combination of matK_463F, matK_1162R, CD_SNP_F2, and CD_SNP_R2 yielded a 421 bp PCR product for <i>C. crispus</i> and a 323 bp band for other <i>Carduus</i> species (<a href="#supp-4">Fig. S3</a>). Reactions were conducted in 25 Âµl solution consisting of 50 ng of template DNA, 2.5 Âµl of 10Ã— reaction buffer, 0.5 U of E-taq DNA polymerase, 50 mM MgCl<sub>2</sub>, and 5 mM dNTPs. Concentrations of outer primer pairs (matK_463F and matK_1162R) and inner primer pairs (CD_SNP_F1 and CD_SNP_R1, CD_SNP_F2 and CD_SNP_R2) were 0.75 pM and 0.5 pM, respectively. The PCR procedure consisted of 1 min at 94 Â°C, followed by denaturing for 1 min at 94 Â°C, annealing for 40 s at 55 Â°C, an extension stage of 50 s at 72 Â°C, and an additional extension of 7 min at 72 Â°C.</p> <div class="table-wrap_UNKNOWN" id="table-2" orientation="landscape" position="float"><a href="https://dx.doi.org/10.7717/peerj.10687/table-2">10.7717/peerj.10687/table-2</a><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Features of chloroplast genomes, assembly information, and pairwise identity among three <i>Carduus</i> species and related taxa.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1"><b>Species</b></th> <th rowspan="1" colspan="1"><i>Carduus crispus</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/MK652229">MK652229</a>)</th> <th rowspan="1" colspan="1"><i>C. tenuiflorus</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/MK652230">MK652230</a>)</th> <th rowspan="1" colspan="1"><i>C. acanthoides</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/MK652228">MK652228</a>)</th> <th rowspan="1" colspan="1"><i>Cynara humilis</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KP299292">KP299292</a>)</th> <th rowspan="1" colspan="1"><i>C. baetica</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KP842706">KP842706</a>)</th> <th rowspan="1" colspan="1"><i>C. cornigera</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KP842707">KP842707</a>)</th> <th rowspan="1" colspan="1"><i>C. cardunculus var. scolymus</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KP842708">KP842708</a>)</th> <th rowspan="1" colspan="1"><i>C. cardunculus var sylvestris</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KP842721">KP842721</a>)</th> <th rowspan="1" colspan="1"><i>Cirsium arvense</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/KY562583">KY562583</a>)</th> <th rowspan="1" colspan="1"><i>Helianthus annus</i> (<a href="http://www.ncbi.nlm.nih.gov/nuccore/NC007977">NC007977</a>)</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1"><b>Total reads</b></td> <td rowspan="1" colspan="1">21,118,624</td> <td rowspan="1" colspan="1">4,621,758</td> <td rowspan="1" colspan="1">4,300,959</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> </tr> <tr> <td rowspan="1" colspan="1"><b>Assemble read</b></td> <td rowspan="1" colspan="1">805,076 (3.8%)</td> <td rowspan="1" colspan="1">189,215 (4.1%)</td> <td rowspan="1" colspan="1">180,245 (4.2%)</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> </tr> <tr> <td rowspan="1" colspan="1"><b>Coverage</b></td> <td rowspan="1" colspan="1">1,585</td> <td rowspan="1" colspan="1">372</td> <td rowspan="1" colspan="1">354</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> </tr> <tr> <td rowspan="1" colspan="1"><b>Number of contigs</b></td> <td rowspan="1" colspan="1">126</td> <td rowspan="1" colspan="1">119</td> <td rowspan="1" colspan="1">14</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> </tr> <tr> <td rowspan="1" colspan="1"><b>N50 value (bp)</b></td> <td rowspan="1" colspan="1">95,199</td> <td rowspan="1" colspan="1">50,873</td> <td rowspan="1" colspan="1">128,044</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> <td rowspan="1" colspan="1">â€"</td> </tr> <tr> <td rowspan="1" colspan="1"><b>Total length</b></td> <td rowspan="1" colspan="1">152,342</td> <td rowspan="1" colspan="1">152,426</td> <td rowspan="1" colspan="1">152,617</td> <td rowspan="1" colspan="1">152,585</td> <td rowspan="1" colspan="1">152,548</td> <td rowspan="1" colspan="1">152,550</td> <td rowspan="1" colspan="1">152,529</td> <td rowspan="1" colspan="1">152,528</td> <td rowspan="1" colspan="1">152,855</td> <td rowspan="1" colspan="1">151,104</td> </tr> <tr> <td rowspan="1" colspan="1"><b>LSC</b></td> <td rowspan="1" colspan="1">83,254</td> <td rowspan="1" colspan="1">83,360</td> <td rowspan="1" colspan="1">83,532</td> <td rowspan="1" colspan="1">83,622</td> <td rowspan="1" colspan="1">83,599</td> <td rowspan="1" colspan="1">83,580</td> <td rowspan="1" colspan="1">83,578</td> <td rowspan="1" colspan="1">83,577</td> <td rowspan="1" colspan="1">83,859</td> <td rowspan="1" colspan="1">83,530</td> </tr> <tr> <td rowspan="1" colspan="1"><b>SSC</b></td> <td rowspan="1" colspan="1">18,706</td> <td rowspan="1" colspan="1">18,674</td> <td rowspan="1" colspan="1">18,693</td> <td rowspan="1" colspan="1">18,651</td> <td rowspan="1" colspan="1">18,639</td> <td rowspan="1" colspan="1">18,660</td> <td rowspan="1" colspan="1">18,641</td> <td rowspan="1" colspan="1">18,641</td> <td rowspan="1" colspan="1">18,633</td> <td rowspan="1" colspan="1">18,308</td> </tr> <tr> <td rowspan="1" colspan="1"><b>IR</b></td> <td rowspan="1" colspan="1">25,191</td> <td rowspan="1" colspan="1">25,196</td> <td rowspan="1" colspan="1">25,196</td> <td rowspan="1" colspan="1">25,156</td> <td rowspan="1" colspan="1">25,155</td> <td rowspan="1" colspan="1">25,155</td> <td rowspan="1" colspan="1">25,155</td> <td rowspan="1" colspan="1">25,155</td> <td rowspan="1" colspan="1">25,182</td> <td rowspan="1" colspan="1">24,633</td> </tr> <tr> <td rowspan="1" colspan="1"><b>Protein-coding genes</b></td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">80</td> <td rowspan="1" colspan="1">80</td> </tr> <tr> <td rowspan="1" colspan="1"><b>tRNA</b></td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> <td rowspan="1" colspan="1">30</td> </tr> <tr> <td rowspan="1" colspan="1"><b>rRNA</b></td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">4</td> </tr> <tr> <td rowspan="1" colspan="1"><b>LSC-IR junction</b></td> <td rowspan="1" colspan="1"><i>rps19</i> (60 bp)</td> <td rowspan="1" colspan="1"><i>rps19</i> (60 bp)</td> <td rowspan="1" colspan="1"><i>rps19</i> (61 bp)</td> <td rowspan="1" colspan="1"><i>rps19</i> (60 bp)</td> <td rowspan="1" colspan="1"><i>rps19</i> (60 bp)</td> <td rowspan="1" colspan="1"><i>rps19</i> (60 bp)</td> <td rowspan="1" colspan="1"><i>rps19</i> (60 bp)</td> <td rowspan="1" colspan="1"><i>rps19</i> (60 bp)</td> <td rowspan="1" colspan="1"><i>rps19</i> (60 bp)</td> <td rowspan="1" colspan="1"><i>rps19</i> (101 bp)</td> </tr> <tr> <td rowspan="1" colspan="1"><b>SSC-IR junction</b></td> <td rowspan="1" colspan="1"><i>ycf1</i> (565 bp)</td> <td rowspan="1" colspan="1"><i>ycf1</i> (565 bp)</td> <td rowspan="1" colspan="1"><i>ycf1</i> (568 bp)</td> <td rowspan="1" colspan="1"><i>ycf1</i> (567 bp) (<i>ycf1-ndhF</i> overlap 17 bp)</td> <td rowspan="1" colspan="1"><i>ycf1</i> (567 bp) (<i>ycf1-ndhF</i> overlap 17 bp)</td> <td rowspan="1" colspan="1"><i>ycf1</i> (567 bp) (<i>ycf1-ndhF</i> overlap 17 bp)</td> <td rowspan="1" colspan="1"><i>ycf1</i> (567 bp) (<i>ycf1-ndhF</i> overlap 17 bp)</td> <td rowspan="1" colspan="1"><i>ycf1</i> (567 bp) (<i>ycf1-ndhF</i> overlap 17 bp)</td> <td rowspan="1" colspan="1"><i>ycf1</i> (565 bp)</td> <td rowspan="1" colspan="1"><i>ycf1</i> (576 bp)</td> </tr> <tr> <td rowspan="1" colspan="1"><b>Pairwise identity (%)</b></td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1"><i>C. crispus</i></td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">98.5</td> <td rowspan="1" colspan="1">98.5</td> <td rowspan="1" colspan="1">98.5</td> <td rowspan="1" colspan="1">98.5</td> <td rowspan="1" colspan="1">98.5</td> <td rowspan="1" colspan="1">98.8</td> <td rowspan="1" colspan="1">92.9</td> </tr> <tr> <td rowspan="1" colspan="1"><i>C. tenuiflorus</i></td> <td rowspan="1" colspan="1">99.6</td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">98.5</td> <td rowspan="1" colspan="1">98.5</td> <td rowspan="1" colspan="1">98.5</td> <td rowspan="1" colspan="1">98.5</td> <td rowspan="1" colspan="1">98.5</td> <td rowspan="1" colspan="1">98.9</td> <td rowspan="1" colspan="1">92.3</td> </tr> <tr> <td rowspan="1" colspan="1"><i>C. acanthoides</i></td> <td rowspan="1" colspan="1">99.2</td> <td rowspan="1" colspan="1">99.3</td> <td rowspan="1" colspan="1">100</td> <td rowspan="1" colspan="1">98.7</td> <td rowspan="1" colspan="1">98.6</td> <td rowspan="1" colspan="1">98.7</td> <td rowspan="1" colspan="1">98.7</td> <td rowspan="1" colspan="1">98.7</td> <td rowspan="1" colspan="1">98.7</td> <td rowspan="1" colspan="1">92.3</td> </tr> <tr> <td rowspan="1" colspan="1" /> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><b>Notes.</b></p> </div> <div class="fn-type-other" title="other"> <p>The dashes (â€") mean no data in this study.</p> </div> </div> </div> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="comparativechloroplastgenomeanalysisofthefocalspecies" title="sec"> <div class="title" tagx="title" title="title">
Comparative chloroplast genome analysis of the focal species</div> <p>Differing numbers of reads were obtained from NGS data, resulting in varying cpDNA coverage rates among the three focal species (<a href="#table-2">Table 2</a>). Total cpDNA length differed among species, ranging from 152,342 bp to 152,617 bp, and included a large single copy (LSC), a small single copy (SSC), and two IR regions (<a href="#fig-1">Fig. 1</a>). By contrast, all three species had identical numbers of protein coding (80), tRNA (30), and rRNA (4) genes (<a href="#table-2">Table 2</a>, <a href="#supp-4">TableÂ S3</a>). The IR-LSC and IR-SSC junctions were located in the <i>rps19</i> and <i>ycf1</i> coding regions, respectively, but were longer in <i>C. acanthoides</i> (<i>rps19</i> = 61 bp and <i>ycf1</i> = 568 bp) than in the other two species (<i>rps19</i> = 60 bp and <i>ycf1</i> = 565 bp). In addition, pairwise identity indicated that <i>C. crispus</i> is more similar to <i>C. tenuiflorus</i> (99.6%) than <i>C. acanthoides</i> (99.2%). Observations of nucleotide diversity indicated that 119 of 131 surveyed regions differ among the three focal species (<a href="#supp-4">Table S2</a>, <a href="#fig-2">Fig. 2</a>). Compared to coding regions, non-coding sequences had higher Pi values (<a href="#fig-2">Fig. 2</a>). The highest Pi values were found in the <i>psbC-trnS</i> (0.0171) and <i>psbH-petB</i> (0.0161) regions. The highest value in coding regions was 0.00696 for <i>ycf1</i> (<a href="#fig-2">Fig. 2</a>). High nucleotide diversity regions (Pi values &amp;gt;0.008) included <i>psbI-trnS_GCU</i>, <i>trnE_UUC-rpoB</i>, <i>trnR_UCU-trnG_UCC</i>, <i>psbC-trnS_UGA</i>, <i>trnT_UGU-trnL_UAA</i>, <i>psbT-psbN</i>, <i>petD-rpoA</i>, and <i>rpl16-rps3</i>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10687/fig-1"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Map of the chloroplast genomes of <i>Carduus</i>.</div> <p>Genes inside the circle are transcribed clockwise whereas genes outside the circle are transcribed counterclockwise. LSC, large single copy; SSC, small single copy; IRAâ€"IRB, inverted repeat regions.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10687/fig-2"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
Nucleotide diversity (Pi values) among the three <i>Carduus</i> species.</div>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10687/fig-3"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <div class="title" tagx="title" title="title">
Repeats in the three <i>Carduus</i> species.</div> <p>(A) Distribution of repeats in the coding and non-coding regions, (B) composition of three types of repeat, and (C) unique and shared repeats in the three <i>Carduus</i> species.</p>   </div> </div> <div class="featuresofcpdnarepeats" title="sec"> <div class="title" tagx="title" title="title">
Features of cpDNA repeats</div> <p>Analysis of SSRs yielded 43 SSRs in <i>C. crispus</i>, 40 in <i>C. tenuiflorus</i>, and 31 in <i>C. acanthoides</i> (<a href="#supp-4">Table S4</a>). SSRs occupied the same position in all three species and were mostly located in non-coding regions. Although four types of SSR (i.e., mono-, di-, tri-, and tetra-nucleotides) were identified, most SSRs were mononucleotides composed of A and T nucleotides (<a href="#supp-4">TableÂ S4</a>). All 31 SSRs found in <i>C. acanthoides</i> were also present in the other two species (<a href="#supp-4">TableÂ S4</a>). By contrast, <i>C. crispus</i> had three unique SSRs and shared nine SSRs with <i>C. tenuiflorus</i>. There were no unique SSRs in <i>C. acanthoides</i> or specific shared SSRs between <i>C. acanthoides</i> and either <i>C. crispus</i> or <i>C. tenuiflorus</i>.</p> <p>Among the focal species, 16 repeats were identified for both <i>C. crispus</i> and <i>C. tenuiflorus,</i> compared to 15 for <i>C. acanthoides</i> (<a href="#fig-3">Fig. 3A</a>, <a href="#supp-4">Table S5</a>). There were more repeats in coding regions than in non-coding areas, with the exception of <i>C. acanthoides</i>. Three types of repeats (i.e., forward, reverse, and palindrome) were identified in <i>C. tenuiflorus</i> and <i>C. acanthoides</i>; by contrast, only forward and palindrome repeats were found in <i>C. crispus</i>. Forward repeats were more abundant than reverse and palindrome repeats (<a href="#fig-3">Fig. 3B</a>). Among recorded repeats, nine were common among all three species (<a href="#fig-3">Fig. 3C</a>). <i>Carduus acanthoides</i> had four unique repeats, whereas <i>C. tenuiflorus</i> and <i>C. crispus</i> each had a single unique repeat. <i>Carduus acanthoides</i> shared one specific repeat with both <i>C. crispus</i> and <i>C. tenuiflorus</i>, whereas <i>C. crispus</i> and <i>C. tenuiflorus</i> shared five specific repeat regions (<a href="#fig-3">Fig.Â 3C</a>).</p> </div> <div class="phylogeneticrelationshipsbetweencarduusandrelatedtaxa" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic relationships between <i>Carduus</i> and related taxa</div> <p>The ML and BI analyses, based on 78 protein-coding genes from <i>Carduus</i> and related taxa, yielded identical topologies (<a href="#fig-4">Fig. 4</a>). In particular, both analyses confirmed the monophyly of Asteraceae subfamilies (i.e., Carduoideae, Chichorioideae, and Asteroideae). In contrast to the high support for Carduoideae and Cichorioideae (Bootstrap = 100/Posterior Probability = 1), low support was found for Asteroideae clades (<a href="#fig-4">Fig. 4</a>). Notably, monophyly of the three <i>Carduus</i> species was not supported by either analysis. For example, <i>C. acanthoides</i> was sister to <i>Silybum marianum</i>, whereas <i>C. crispus</i> and <i>C. tenuiflorus</i> were sister to <i>Cirsium arvense</i>. Additional ML and BI analyses of full cpDNA sequences and non-coding regions suggested similar relationships (<a href="#supp-4">Fig. S4</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10687/fig-4"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <div class="title" tagx="title" title="title">
BI tree of <i>Carduus</i> and related taxa inferred from 78 protein coding cpDNA regions.</div> <p>Numbers indicate supporting values (BP, Bootstrap; PP, Posterior Probability). The asterisk indicates the <i>Scaevola porocarya</i> branch, which is compared with other species in the tree; this branch was reduced five times. Only supporting values under 100 (BP) or 1 (PP) are shown.</p>   </div> </div> <div class="multiplexpcrandspecificmarkersforc.crispus" title="sec"> <div class="title" tagx="title" title="title">
Multiplex PCR and specific markers for <i>C. crispus</i></div> <p>The results of multiplex PCR for the two groups of primer pairs yielded similar products, both of which were designed to identify <i>C. crispus</i>. In the first group, a 323 bp band was found in <i>C. crispus,</i> whereas a 421 bp band was identified in <i>C. acanthoides</i> and <i>C. tenuiflorus</i> (<a href="#fig-5">Fig. 5A</a>). By contrast, the combination of matK_463F, matK_1162R, CD_SNP_F2, and CD_SNP_R2 yielded a longer PCR product for <i>C. crispus</i> in comparison with the other species (<a href="#fig-5">Fig. 5B</a>). The designed primer pairs were specific to all <i>C. crispus</i> samples examined in this study (<a href="#supp-4">Figs. S5</a> and <a href="#supp-4">S6</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10687/fig-5"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <div class="title" tagx="title" title="title">
PCR results of specific primer pairs for <i>C. crispus</i>.</div> <p>(A) Combination of matK_463F, matK_1162R, CD_SNP_F1, and CD_SNP_R1, and (B) combination of matK_463F, matK_1162R, CD_SNP_F2, and CD_SNP_R2. Numbers 1â€"4 = <i>C. crispus</i>; 5â€"8 = <i>C. acanthoides</i>, and 9â€"12 = <i>C. tenuiflorus</i>.</p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="conservatismofcarduuscpdna" title="sec"> <div class="title" tagx="title" title="title">
Conservatism of <i>Carduus</i> cpDNA</div> <p>Chloroplast genomes are highly conserved in angiosperms with respect to gene content and order (<a href="#ref-60">Sugiura, 1992</a>). This conservative tendency was observed in the three newly sequenced <i>Carduus</i> cpDNA genomes, compared to other Asteraceae members (<a href="#table-2">Table 2</a>). Other cpDNA sequences have revealed unique genomic events in Asteraceae. For example, the <i>atpB</i> gene, which encodes the CF1 ATPase beta subunit, is annotated as a pseudogene in <i>Aster spathulifolius</i> due to a deletion within the coding region (Choi and Park, 2015). Similarly, <i>trnT_GGU</i> was completely deleted or pseudogenised in the tribe Gnaphalieae (<a href="#ref-41">Lee et al., 2017</a>). Duplication of <i>trnF_GAA</i> has been identified in <i>Taraxacum</i> (<a href="#ref-57">Salih et al., 2017</a>). No comparable genomic events are present in <i>Carduus</i> or other members of subfamily Carduoideae (<a href="#table-2">Table 2</a>, <a href="#supp-4">Fig. S7</a>). However, nucleotide diversity data pointed to potential regions for further study of phylogeny and population genetics, and the development of <i>Carduus</i>-specific molecular markers (<a href="#supp-4">Table S2</a>, <a href="#fig-2">Fig. 2</a>). The number of species we examined for this study was low relative to the approximately 32,000 known Asteraceae species. Therefore, additional studies that include the majority of Asteraceae species should be conducted to explore the overall evolutionary trends in the chloroplast genomes of this globally-distributed family.</p> <p>Chloroplast genomes provide useful molecular data for reconstructing phylogeny, exploring biogeography, and estimating divergence time in angiosperm lineages (<a href="#ref-16">Do, Kim &amp;amp; Kim, 2014</a>; <a href="#ref-48">Nguyen, Kim &amp;amp; Kim, 2015</a>; <a href="#ref-38">Kim, Kim &amp;amp; Kim, 2016</a>; <a href="#ref-37">Kim &amp;amp; Kim, 2018</a>). Repetitive sequences in the chloroplast genome provide useful information for studying genomic rearrangement and phylogeny (<a href="#ref-4">Cavalier-Smith, 2002</a>; <a href="#ref-49">Nie et al., 2012</a>; <a href="#ref-72">Yi et al., 2013</a>; <a href="#ref-37">Kim &amp;amp; Kim, 2018</a>). In addition, existing repeats might result in the accumulation of new repeats in cpDNA (<a href="#ref-3">Asano et al., 2004</a>). One of the crucial molecular data types in cpDNA is SSR sequences. Other studies have used SSRs to develop specific markers for different species and to study the genetic diversity of angiosperms (<a href="#ref-30">Ishikawa, Sakaguchi &amp;amp; Ito, 2016</a>; <a href="#ref-44">Luo et al., 2016</a>; <a href="#ref-47">Marochio et al., 2017</a>; <a href="#ref-27">Han et al., 2018</a>). In this study, although cpDNA sequences were highly conserved, the three <i>Carduus</i> species were found to have different numbers of SSRs (<a href="#supp-4">Table S4</a>). While we did not develop SSR markers or conduct population studies of <i>Carduus,</i> the SSR information we obtained may be useful in future studies on <i>Carduus</i> species. In addition to the repeats shared among the three species, <i>C. crispus</i> had three unique repetitive sequences (<a href="#supp-4">Table S4</a>), which may be useful in population studies, phylogenetic analyses, and the development of additional molecular markers.</p> </div> <div class="uncertainrelationshipsamongcarduus" title="sec"> <div class="title" tagx="title" title="title">
Uncertain relationships among <i>Carduus</i></div> <p>Phylogenetic analyses of the Asteraceae have identified ambiguous relationships between <i>Carduus</i>, <i>Cirsium</i>, and <i>Silybum</i> (<a href="#ref-22">Fu et al., 2016</a>; <a href="#ref-50">Panero, 2016</a>; <a href="#ref-2">Arnelas et al., 2018</a>); for example, ITS data suggests that <i>Carduus</i> is polyphyletic (<a href="#ref-26">HÃ¤ffner &amp;amp; Hellwig, 1999</a>). Although three coding regions (<i>matK</i>, <i>rbcL</i>, and <i>ndhF</i>) were used to reconstruct phylogenetic relationships, the position of <i>Carduus</i> remained unresolved (<a href="#ref-22">Fu et al., 2016</a>). We used 78 protein-coding regions to clarify these relationships; however, the phylogeny of <i>Carduus</i> and related taxa remains unclear (<a href="#fig-4">Fig. 4</a>). Specifically, <i>C. acanthoides</i> was found to be close to <i>Silybum marianum</i> whereas <i>C. crispus</i> and <i>C. tenuiflorus</i> form a clade with <i>Cirsium arvense</i>. While non-coding regions can be useful in reconstructing the phylogeny of lower taxa, we were unable to recover the monophyly of <i>Carduus</i> using data from non-coding regions, including the eight hotspot areas as well as the combined data from coding and non-coding regions (<a href="#supp-4">Fig. S4</a>). These issues suggest the need for additional studies on the phylogeny of <i>Carduus</i> and other members of the subfamily Carduoideae using supplementary molecular data and morphology.</p> </div> <div class="implicationsofsnpdatafordevelopingmolecularmarkersforcarduus" title="sec"> <div class="title" tagx="title" title="title">
Implications of SNP data for developing molecular markers for <i>Carduus</i></div> <p>SNPs are useful in population studies due to its extremely abundant presence in the angiosperms genomes (<a href="#ref-10">Cui et al., 2017</a>; <a href="#ref-21">Fischer et al., 2017</a>; <a href="#ref-51">Pantoja et al., 2017</a>), and are effective in phylogenetic analysis (<a href="#ref-40">LeachÃ© &amp;amp; Oaks, 2017</a>). In addition, various molecular markers have been developed for different angiosperm species based on SNP data from chloroplast genomes (<a href="#ref-34">Khlestkina &amp;amp; Salina, 2006</a>; <a href="#ref-68">Wang et al., 2015a</a>; <a href="#ref-67">Wang et al., 2015b</a>; <a href="#ref-29">Hyun et al., 2019</a>; <a href="#ref-70">Xia et al., 2019</a>). We successfully developed a molecular marker, inferred from SNP data, to distinguish <i>C. crispus</i> from <i>C. acanthoides</i> and <i>C. tenuiflorus</i> (<a href="#supp-4">Fig. S3</a>). Our marker demonstrates that nucleotide sequence variations can provide rapid molecular identification of <i>C. crispus</i>. We focused on <i>C. crispus</i> because it exhibits the characteristics of an invasive species (<a href="#ref-19">Dunn, 1976</a>; <a href="#ref-65">Verloove, 2014</a>; <a href="#ref-32">Jung et al., 2017</a>), and may also have value for the treatment of obesity and cancer (<a href="#ref-71">Xie, Li &amp;amp; Jia, 2005</a>; <a href="#ref-12">Davaakhuu, Sukhdolgor &amp;amp; Gereltu, 2010</a>; <a href="#ref-42">Lee et al., 2011</a>; <a href="#ref-63">Tunsag, Davaakhuu &amp;amp; Batsuren, 2011</a>). Various DNA-based markers (i.e., inter-simple sequence repeats [ISSRs], sequence characterisation of amplified regions [SCARs], and SSRs) have been developed to authenticate medicinal plants to ensure safety and efficacy (<a href="#ref-28">Hao et al., 2010</a>; <a href="#ref-58">Sarwat et al., 2012</a>; <a href="#ref-23">Ganie et al., 2015</a>; <a href="#ref-69">Ward, Gaskin &amp;amp; Wilson, 2008</a>). Additionally, molecular data are useful for understanding invasion processes of alien plants (Ward et al., 2008). We developed a SNP-based molecular marker for <i>C. crispus</i> (<a href="#fig-5">Fig. 5</a>, <a href="#supp-4">Figs. S5</a> and <a href="#supp-4">S6</a>) that may be used to detect <i>C. crispus</i> invasions in their early stages, and to develop suitable management strategies. Our alignment results also identified specific SNPs for <i>C. acanthoides</i> and <i>C. tenuiflorus</i>, which may be used to create molecular markers for these species (<a href="#supp-4">Fig. S1</a>). Although we only used the SNP in <i>matK</i>, use of the complete cpDNA sequences of <i>Carduus</i> will enable the mining of SNPs from other regions for developing molecular markers for <i>C. crispus</i> and related species.</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>In this study, we provided the first complete cpDNA sequences for <i>Carduus</i> species. Despite the absence of significant differences (i.e., inversions, deletions, and duplications) between the chloroplast genomes of <i>Carduus</i> and those of related taxa, the newly acquired cpDNA sequences have value as a resource in future studies of the evolution of the chloroplast genome in Carduoideae and Asteraceae. Additionally, the 78 protein-coding regions of the chloroplast genome revealed uncertainty regarding the position of <i>Carduus</i> within the subfamily Carduoideae, and suggested the need for additional studies to reconstruct relationships not only among thistles, but among other members of the Asteraceae as well. The methods and protocols used in developing molecular markers for <i>C. crispus</i> are easy to apply and may be useful as a standard method in other studies of Asteraceae species.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
 Supplemental Information</div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7717/peerj.10687/supp-1">10.7717/peerj.10687/supp-1</a><span class="label" tagx="label" title="label">Supplemental Information 1</span> <div class="title" tagx="title" title="title"><i>Carduus acanthoides</i> chloroplast genome sequence</div>  <div class="media" title="media"><a href="peerj-09-10687-s001.fasta">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7717/peerj.10687/supp-2">10.7717/peerj.10687/supp-2</a><span class="label" tagx="label" title="label">Data S1</span> <div class="title" tagx="title" title="title">
The GO annotation of <i>Carduus</i> species</div>  <div class="media" title="media"><a href="peerj-09-10687-s002.txt">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7717/peerj.10687/supp-3">10.7717/peerj.10687/supp-3</a><span class="label" tagx="label" title="label">Supplemental Information 3</span> <div class="title" tagx="title" title="title"><i>Carduus crispuss</i> chloroplast genome sequence</div>  <div class="media" title="media"><a href="peerj-09-10687-s003.fasta">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7717/peerj.10687/supp-4">10.7717/peerj.10687/supp-4</a><span class="label" tagx="label" title="label">Supplemental Information 4</span> <div class="title" tagx="title" title="title">
Supplemental tables and figures</div> <p>Table S1: List of species for phylogenomic analysis and whole cpDNA alignment. Table S2: List of regions for calculating Pi values. Table S3. Genes composition of the <i>Carduus</i> chloroplast genomes. Table S4: Feature of SSR in three <i>Carduus</i> species. Table S5: Feature of repeats in three <i>Carduus</i> species. Figure S1: The alignment of <i>matK</i> among three <i>Carduus</i>. The asterisk indicates the positions of single nucleotide polymorphism (SNP). The red square indicates SNP site for design primer pairs. Figure S2: The design of the primer pairs based on SNP site that is specific for <i>Carduus crispus</i>. Figure S3: The Bayesian Inference tree of <i>Carduus</i> and related taxa inferred from whole cpDNA sequences (A), non-coding regions of cpDNA (B), and eight hotspot regions (C).Figure S4: The PCR results of specific primer pairs for <i>Carduus crispus</i>. The combination of matK_463F, matK_1162R, CD_SNP_F1, and CD_SNP_R1. The number from 1 to 4: <i>Carduus crispus</i>; from 5 to 8: <i>Carduus acanthoides</i>; from 9 to 12: <i>Carduus tenuiflorus</i>. Figure S5: The PCR results of specific primer pairs for <i>Carduus crispus</i>. The combination of matK_463F, matK_1162R, CD_SNP_F2, and CD_SNP_R2. The number from 1 to 4: <i>Carduus crispus</i>; from 5 to 8: <i>Carduus acanthoides</i>; from 9 to 12: <i>Carduus tenuiflorus</i>. Figure S6: The MAUVE alignment of chloroplast genomes among <i>Carduus</i> and related species.</p>  <div class="media" title="media"><a href="peerj-09-10687-s004.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7717/peerj.10687/supp-5">10.7717/peerj.10687/supp-5</a><span class="label" tagx="label" title="label">Supplemental Information 5</span> <div class="title" tagx="title" title="title"><i>Carduus tenuiflorus</i> chloroplast genome sequence</div>  <div class="media" title="media"><a href="peerj-09-10687-s005.fasta">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>The authors thank the Korea National Arboretum (KNA), National Institute of Biological Resources (NIBR), New York Botanical Garden (NYBG), Carnegie Museum Herbarium (CM), and the United States National Herbarium (US) for providing materials in support of this study. We also appreciate the helpful comments of anonymous reviewers.</p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional Information and Declarations</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Competing Interests</div> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors declare there are no competing interests.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Author Contributions</div> <div class="fn-type-con" title="con"> <p><a href="#author-1">Joonhyung Jung</a> and <a href="#author-2">Hoang Dang Khoa Do</a> performed the experiments, analyzed the data, prepared figures and/or tables, authored or reviewed drafts of the paper, and approved the final draft.</p> </div> <div class="fn-type-con" title="con"> <p><a href="#author-3">JongYoung Hyun</a> performed the experiments, prepared figures and/or tables, and approved the final draft.</p> </div> <div class="fn-type-con" title="con"> <p><a href="#author-4">Changkyun Kim</a> and <a href="#author-5">Joo-Hwan Kim</a> conceived and designed the experiments, authored or reviewed drafts of the paper, and approved the final draft.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
DNA Deposition</div> <div class="fn-type-" title=""> <p>The following information was supplied regarding the deposition of DNA sequences:</p> <p>The complete cpDNA sequences of <i>C. crispus, C. acanthoides</i>, and <i>C. tenuiflorus</i> are available at GenBank: <a href="http://www.ncbi.nlm.nih.gov/nuccore/MK652229">MK652229</a>, <a href="http://www.ncbi.nlm.nih.gov/nuccore/MK652228">MK652228</a>, and <a href="http://www.ncbi.nlm.nih.gov/nuccore/MK652230">MK652230</a>, respectively.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Data Availability</div> <div class="fn-type-" title=""> <p>The following information was supplied regarding data availability:</p> <p>The three <i>Carduus</i> speciesâ€™ sequence data are available at NCBI: <a href="http://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA645567">PRJNA645567</a>. Specimen voucher numbers and deposition information are available in <a href="#table-1">Table 1</a>.</p> </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ref-1" /><span class="label" tagx="label" title="label">Angiosperm Phylogeny Group (2016)</span><span class="element-citation'"><span class="year" tagx="year" title="year">2016</span><span class="person-group'"><span class="collab'">collab: Angiosperm Phylogeny Group</span></span><span class="mixed-article-title" title="mixed-article-title">An update of the Angiosperm Phylogeny Group classification for the orders and families of flowering plants: APG IV</span><span class="source" tagx="source" title="source">Botanical Journal of the Linnean Society</span><span class="volume" tagx="volume" title="volume">181</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">20</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/boj.12385">10.1111/boj.12385</a></span></span></li> <li tag="ref"><a name="ref-2" /><span class="label" tagx="label" title="label">Arnelas et al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnelas</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PÃ©rez-Collazos</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devesa</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LÃ³pez</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Catalan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Phylogeny of highly hybridogenous Iberian <i>Centaurea</i> L. (Asteraceae) taxa and its taxonomic implications</span><span class="source" tagx="source" title="source">Plant Biosystems</span><span class="volume" tagx="volume" title="volume">152</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">1182</span><span class="lpage" tagx="lpage" title="lpage">1190</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/11263504.2018.1435569">10.1080/11263504.2018.1435569</a></span></span></li> <li tag="ref"><a name="ref-3" /><span class="label" tagx="label" title="label">Asano et al. (2004)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asano</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsudzuki</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahashi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimada</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kadowaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Complete nucleotide sequence of the sugarcane (<i>Saccharum officinarum</i>) chloroplast genome: a comparative analysis of four monocot chloroplast genomes</span><span class="source" tagx="source" title="source">DNA Research</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">93</span><span class="lpage" tagx="lpage" title="lpage">99</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/dnares/11.2.93">10.1093/dnares/11.2.93</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15449542">15449542</a></span></span></li> <li tag="ref"><a name="ref-4" /><span class="label" tagx="label" title="label">Cavalier-Smith (2002)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavalier-Smith</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Chloroplast evolution: secondary symbiogenesis and multiple losses</span><span class="source" tagx="source" title="source">Current Biology</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">R62</span><span class="lpage" tagx="lpage" title="lpage">R64</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0960-9822(01)00675-3">10.1016/S0960-9822(01)00675-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11818081">11818081</a></span></span></li> <li tag="ref"><a name="ref-5" /><span class="label" tagx="label" title="label">Chan &amp;amp; Lowe (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowe</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">tRNAscan-SE: searching for tRNA genes in genomic sequences</span><span class="source" tagx="source" title="source">Methods in Molecular Biology</span><span class="volume" tagx="volume" title="volume">1962</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-4939-9173-0_1">10.1007/978-1-4939-9173-0_1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31020551">31020551</a></span></span></li> <li tag="ref"><a name="ref-6" /><span class="label" tagx="label" title="label">Choi &amp;amp; Choi (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">IS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">BH</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The distinct plastid genome structure of <i>Maackia fauriei</i> (Fabaceae: Papilionoideae) and its systematic implications for genistoids and tribe Sophoreae</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0173766</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0173766">10.1371/journal.pone.0173766</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28399123">28399123</a></span></span></li> <li tag="ref"><a name="ref-7" /><span class="label" tagx="label" title="label">Choi &amp;amp; Park (2015)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">The complete chloroplast genome sequence of <i>Aster spathulifolius</i> (Asteraceae); genomic features and relationship with Asteracea</span><span class="source" tagx="source" title="source">Gene</span><span class="volume" tagx="volume" title="volume">572</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">214</span><span class="lpage" tagx="lpage" title="lpage">221</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gene.2015.07.020">10.1016/j.gene.2015.07.020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26164759">26164759</a></span></span></li> <li tag="ref"><a name="ref-8" /><span class="label" tagx="label" title="label">Christoph (2006â€"2010)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christoph</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2006â€"2010</span><span class="mixed-article-title" title="mixed-article-title">Phobos 3.3.11</span><span class="uri" title="uri"><a href="http://www.rub.de/ecoevo/cm/cm_phobos.htm">http://www.rub.de/ecoevo/cm/cm_phobos.htm</a></span></span></li> <li tag="ref"><a name="ref-9" /><span class="label" tagx="label" title="label">Cosner, Raubeson &amp;amp; Jansen (2004)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cosner</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raubeson</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jansen</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Chloroplast DNA rearrangements in Campanulaceae: phylogenetic utility of highly rearranged genomes</span><span class="source" tagx="source" title="source">BMC Evolutionary Biology</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">27</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-4-27">10.1186/1471-2148-4-27</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15324459">15324459</a></span></span></li> <li tag="ref"><a name="ref-10" /><span class="label" tagx="label" title="label">Cui et al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mei</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Genetic diversity, population structure, and linkage disequilibrium of an association-mapping panel revealed by genome-wide SNP markers in sesame</span><span class="source" tagx="source" title="source">Frontiers in Plant Science</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1189</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2017.01189">10.3389/fpls.2017.01189</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28729877">28729877</a></span></span></li> <li tag="ref"><a name="ref-11" /><span class="label" tagx="label" title="label">Darling, Mau &amp;amp; Perna (2010)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darling</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mau</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perna</span><span class="given-names" tagx="given-names" title="given-names">NT</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e11147</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0011147">10.1371/journal.pone.0011147</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20593022">20593022</a></span></span></li> <li tag="ref"><a name="ref-12" /><span class="label" tagx="label" title="label">Davaakhuu, Sukhdolgor &amp;amp; Gereltu (2010)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davaakhuu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sukhdolgor</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gereltu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Lipid lowering effect of ethanolic extract of <i>Carduus crispus</i> L. on hypercholesterolemic rats</span><span class="source" tagx="source" title="source">Mongolian Journal of Biological Sciences</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">49</span><span class="lpage" tagx="lpage" title="lpage">51</span></span></li> <li tag="ref"><a name="ref-13" /><span class="label" tagx="label" title="label">Dierckxsens, Mardulyn, Smits (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dierckxsens</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mardulyn</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smits</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">NOVOPlasty: de novo assembly of organelle genomes from whole genome data</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">45</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">e18</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw955">10.1093/nar/gkw955</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28204566">28204566</a></span></span></li> <li tag="ref"><a name="ref-14" /><span class="label" tagx="label" title="label">Do et al. (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Do</span><span class="given-names" tagx="given-names" title="given-names">HDK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyun</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J-H</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">The newly developed single nucleotide polymorphism (SNP) markers for a potentially medicinal plant, <i>Crepidiastrum denticulatum</i> (Asteraceae), inferred from complete chloroplast genome data</span><span class="source" tagx="source" title="source">Molecular Biology Reports</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">3287</span><span class="lpage" tagx="lpage" title="lpage">3297</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11033-019-04789-5">10.1007/s11033-019-04789-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30980269">30980269</a></span></span></li> <li tag="ref"><a name="ref-15" /><span class="label" tagx="label" title="label">Do &amp;amp; Kim (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Do</span><span class="given-names" tagx="given-names" title="given-names">HDK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J-H</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">A dynamic tandem repeat in monocotyledons inferred from a comparative analysis of chloroplast genomes in Melanthiaceae</span><span class="source" tagx="source" title="source">Frontiers in Plant Science</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">693</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2017.00693">10.3389/fpls.2017.00693</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28588587">28588587</a></span></span></li> <li tag="ref"><a name="ref-16" /><span class="label" tagx="label" title="label">Do, Kim &amp;amp; Kim (2014)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Do</span><span class="given-names" tagx="given-names" title="given-names">HDK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">A <i>trnI_CAU</i> triplication event in the complete chloroplast genome of <i>Paris verticillata</i> M.Bieb. (Melanthiaceae, Liliales)</span><span class="source" tagx="source" title="source">Genome Biology and Evolution</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">1699</span><span class="lpage" tagx="lpage" title="lpage">1706</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/gbe/evu138">10.1093/gbe/evu138</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24951560">24951560</a></span></span></li> <li tag="ref"><a name="ref-17" /><span class="label" tagx="label" title="label">Doing, Biddiscombe &amp;amp; Knedlhans (1969)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doing</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biddiscombe</span><span class="given-names" tagx="given-names" title="given-names">EF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knedlhans</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">1969</span><span class="mixed-article-title" title="mixed-article-title">Ecology and distribution of the <i>Carduus nutans</i> group (nodding thistles) in Australia</span><span class="source" tagx="source" title="source">Vegetatio</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">313</span><span class="lpage" tagx="lpage" title="lpage">351</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF01965915">10.1007/BF01965915</a></span></span></li> <li tag="ref"><a name="ref-18" /><span class="label" tagx="label" title="label">Doyle &amp;amp; Doyle (1987)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doyle</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doyle</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="year" tagx="year" title="year">1987</span><span class="mixed-article-title" title="mixed-article-title">A rapid DNA isolation procedure for small quantities of fresh leaf tissue</span><span class="source" tagx="source" title="source">Phytochemical Bulletin</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">15</span></span></li> <li tag="ref"><a name="ref-19" /><span class="label" tagx="label" title="label">Dunn (1976)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunn</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">1976</span><span class="mixed-article-title" title="mixed-article-title">Distribution of <i>Carduus nutans, C. acanthoides, C. pycnocephalus</i>, and <i>C. crispus</i>, in the United States</span><span class="source" tagx="source" title="source">Weed Science</span><span class="volume" tagx="volume" title="volume">24</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">518</span><span class="lpage" tagx="lpage" title="lpage">524</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0043174500066595">10.1017/S0043174500066595</a></span></span></li> <li tag="ref"><a name="ref-20" /><span class="label" tagx="label" title="label">Edgar (2004)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">32</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">1792</span><span class="lpage" tagx="lpage" title="lpage">1797</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkh340">10.1093/nar/gkh340</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a></span></span></li> <li tag="ref"><a name="ref-21" /><span class="label" tagx="label" title="label">Fischer et al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rellstab</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leuzinger</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roumet</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gugerli</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimizu</span><span class="given-names" tagx="given-names" title="given-names">KK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Widmer</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Estimating genomic diversity and population differentiationâ€"an empirical comparison of microsatellite and SNP variation in <i>Arabidopsis halleri</i></span><span class="source" tagx="source" title="source">BMC Genomics</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">69</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-016-3459-7">10.1186/s12864-016-3459-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28077077">28077077</a></span></span></li> <li tag="ref"><a name="ref-22" /><span class="label" tagx="label" title="label">Fu et al. (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiao</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">T-G</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="person-group'"><span class="collab'">collab: China Phylogeny Consortium</span></span><span class="mixed-article-title" title="mixed-article-title">A comprehensive generic-level phylogeny of the sunflower family: implications for the systematics of Chinese Asteraceae</span><span class="source" tagx="source" title="source">Jnl of Sytematics Evolution</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">416</span><span class="lpage" tagx="lpage" title="lpage">437</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/jse.12216">10.1111/jse.12216</a></span></span></li> <li tag="ref"><a name="ref-23" /><span class="label" tagx="label" title="label">Ganie et al. (2015)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ganie</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Upadhyay</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Authentication of medicinal plants by DNA markers</span><span class="source" tagx="source" title="source">Plant Gene</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">83</span><span class="lpage" tagx="lpage" title="lpage">99</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.plgene.2015.10.002">10.1016/j.plgene.2015.10.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32289060">32289060</a></span></span></li> <li tag="ref"><a name="ref-24" /><span class="label" tagx="label" title="label">Greiner, Lehwark &amp;amp; Bock (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greiner</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lehwark</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bock</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">OrganellarGenomeDRAW (OGDRAW) version 1.3.1: expanded toolkit for the graphical visualization of organellar genomes</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">W59</span><span class="lpage" tagx="lpage" title="lpage">W64</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkz238">10.1093/nar/gkz238</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30949694">30949694</a></span></span></li> <li tag="ref"><a name="ref-25" /><span class="label" tagx="label" title="label">Haberle et al. (2008)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haberle</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fourcade</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boore</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jansen</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Extensive rearrangements in the chloroplast genome of <i>Trachelium caeruleum</i> are associated with repeats and tRNA genes</span><span class="source" tagx="source" title="source">Journal of Molecular Evolution</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">350</span><span class="lpage" tagx="lpage" title="lpage">361</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00239-008-9086-4">10.1007/s00239-008-9086-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18330485">18330485</a></span></span></li> <li tag="ref"><a name="ref-26" /><span class="label" tagx="label" title="label">HÃ¤ffner &amp;amp; Hellwig (1999)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HÃ¤ffner</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hellwig</span><span class="given-names" tagx="given-names" title="given-names">FH</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Phylogeny of the tribe Cardueae (Compositae) with emphasis on the subtribe Carduinae: an analysis based on ITS sequence data</span><span class="source" tagx="source" title="source">Willdenowia</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">1/2</span><span class="fpage" tagx="fpage" title="fpage">27</span><span class="lpage" tagx="lpage" title="lpage">39</span><span class="pub-id"><a href="https://dx.doi.org/10.3372/wi.29.2902">10.3372/wi.29.2902</a></span></span></li> <li tag="ref"><a name="ref-27" /><span class="label" tagx="label" title="label">Han et al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhan</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">SSR marker development and intraspecific genetic divergence exploration of <i>Chrysanthemum indicum</i> based on transcriptome analysis</span><span class="source" tagx="source" title="source">BMC Genomics</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">291</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-018-4702-1">10.1186/s12864-018-4702-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29695227">29695227</a></span></span></li> <li tag="ref"><a name="ref-28" /><span class="label" tagx="label" title="label">Hao et al. (2010)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hao</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Authentication of medicinal plants by DNA-based markers and genomics</span><span class="source" tagx="source" title="source">Chinese Herbal Medicines</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">250</span><span class="lpage" tagx="lpage" title="lpage">261</span><span class="pub-id"><a href="https://dx.doi.org/10.3969/j.issn.1674-6384.2010.04.003">10.3969/j.issn.1674-6384.2010.04.003</a></span></span></li> <li tag="ref"><a name="ref-29" /><span class="label" tagx="label" title="label">Hyun et al. (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyun</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Do</span><span class="given-names" tagx="given-names" title="given-names">HDK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Development of molecular markers for invasive alien plants in Korea: a case study of a toxic weed, Cenchrus longispinus L., based on next generation sequencing data</span><span class="source" tagx="source" title="source">PeerJ</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">e7965</span><span class="pub-id"><a href="https://dx.doi.org/10.7717/peerj.7965">10.7717/peerj.7965</a></span></span></li> <li tag="ref"><a name="ref-30" /><span class="label" tagx="label" title="label">Ishikawa, Sakaguchi &amp;amp; Ito (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishikawa</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakaguchi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ito</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Development and characterization of SSR markers for <i>Aster savatieri</i> (Asteraceae)</span><span class="source" tagx="source" title="source">Applications in Plant Sciences</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">apps.1500143</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/apps.1500143">10.3732/apps.1500143</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27347451">27347451</a></span></span></li> <li tag="ref"><a name="ref-31" /><span class="label" tagx="label" title="label">Jansen, Michaels &amp;amp; Palmer (1991)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jansen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michaels</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palmer</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1991</span><span class="mixed-article-title" title="mixed-article-title">Phylogeny and character evolution in the asteraceae based on chloroplast dna restriction site mapping</span><span class="source" tagx="source" title="source">Systematic Botany</span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">98</span><span class="lpage" tagx="lpage" title="lpage">115</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/2418976">10.2307/2418976</a></span></span></li> <li tag="ref"><a name="ref-32" /><span class="label" tagx="label" title="label">Jung et al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shin</span><span class="given-names" tagx="given-names" title="given-names">HT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">An</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heo</span><span class="given-names" tagx="given-names" title="given-names">TI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="source" tagx="source" title="source">Invasive Alien Plants in South Korea</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Korea National Arboretum</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Pocheon</span></span></li> <li tag="ref"><a name="ref-33" /><span class="label" tagx="label" title="label">Kearse et al. (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kearse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moir</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stones-Havas</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheung</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sturrock</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">28</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">1647</span><span class="lpage" tagx="lpage" title="lpage">1649</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bts199">10.1093/bioinformatics/bts199</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22543367">22543367</a></span></span></li> <li tag="ref"><a name="ref-34" /><span class="label" tagx="label" title="label">Khlestkina &amp;amp; Salina (2006)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khlestkina</span><span class="given-names" tagx="given-names" title="given-names">EK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salina</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">SNP markers: methods of analysis, ways of development, and comparison on an example of common wheat</span><span class="source" tagx="source" title="source">Russian Journal of Genetics</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">585</span><span class="lpage" tagx="lpage" title="lpage">594</span><span class="pub-id"><a href="https://dx.doi.org/10.1134/S1022795406060019">10.1134/S1022795406060019</a></span></span></li> <li tag="ref"><a name="ref-35" /><span class="label" tagx="label" title="label">Kim, Choi &amp;amp; Jansen (2005)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">K-J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jansen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Two chloroplast DNA inversions originated simultaneously during the early evolution of the sunflower family (Asteraceae)</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">1783</span><span class="lpage" tagx="lpage" title="lpage">1792</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msi174">10.1093/molbev/msi174</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15917497">15917497</a></span></span></li> <li tag="ref"><a name="ref-36" /><span class="label" tagx="label" title="label">Kim et al. (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jang</span><span class="given-names" tagx="given-names" title="given-names">HW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J-H</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Molecular identification of <i>Schisandra chinensis</i> and its allied species using multiplex PCR based on SNPs</span><span class="source" tagx="source" title="source">Genes Genom</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">283</span><span class="lpage" tagx="lpage" title="lpage">290</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s13258-011-0201-3">10.1007/s13258-011-0201-3</a></span></span></li> <li tag="ref"><a name="ref-37" /><span class="label" tagx="label" title="label">Kim &amp;amp; Kim (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J-H</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Updated molecular phylogenetic analysis, dating and biogeographical history of the lily family (Liliaceae: Liliales)</span><span class="source" tagx="source" title="source">Botanical Journal of the Linnean Society</span><span class="volume" tagx="volume" title="volume">187</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">579</span><span class="lpage" tagx="lpage" title="lpage">593</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/botlinnean/boy031">10.1093/botlinnean/boy031</a></span></span></li> <li tag="ref"><a name="ref-38" /><span class="label" tagx="label" title="label">Kim, Kim &amp;amp; Kim (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Insight into infrageneric circumscription through complete chloroplast genome sequences of two <i>Trillium</i> species</span><span class="source" tagx="source" title="source">AoB PLANTS</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">plw015</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/aobpla/plw015">10.1093/aobpla/plw015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26933149">26933149</a></span></span></li> <li tag="ref"><a name="ref-39" /><span class="label" tagx="label" title="label">Kurtz et al. (2001)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurtz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choudhuri</span><span class="given-names" tagx="given-names" title="given-names">JV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohlebusch</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schleiermacher</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoye</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giegerich</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">REPuter: the manifold applications of repeat analysis on a genomic scale</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">22</span><span class="fpage" tagx="fpage" title="fpage">4633</span><span class="lpage" tagx="lpage" title="lpage">4642</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/29.22.4633">10.1093/nar/29.22.4633</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11713313">11713313</a></span></span></li> <li tag="ref"><a name="ref-40" /><span class="label" tagx="label" title="label">LeachÃ© &amp;amp; Oaks (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LeachÃ©</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oaks</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The utility of Single Nucleotide Polymorphism (SNP) data in phylogenetics</span><span class="source" tagx="source" title="source">Annual Review of Ecology, Evolution, and Systematics</span><span class="volume" tagx="volume" title="volume">48</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">69</span><span class="lpage" tagx="lpage" title="lpage">84</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-ecolsys-110316-022645">10.1146/annurev-ecolsys-110316-022645</a></span></span></li> <li tag="ref"><a name="ref-41" /><span class="label" tagx="label" title="label">Lee et al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">WB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">BH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Characterization of two complete chloroplast genomes in the tribe gnaphalieae (Asteraceae): gene loss or pseudogenization of <i>trn</i> T-GGU and implications for phylogenetic relationships</span><span class="source" tagx="source" title="source">Horticultural Science and Technoloy</span><span class="volume" tagx="volume" title="volume">35</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">769</span><span class="lpage" tagx="lpage" title="lpage">783</span><span class="pub-id"><a href="https://dx.doi.org/10.12972/kjhst.20170081.">10.12972/kjhst.20170081.</a></span></span></li> <li tag="ref"><a name="ref-42" /><span class="label" tagx="label" title="label">Lee et al. (2011)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joo</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">YN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">YS</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Inhibition of adipocyte differentiation by MeOH extract from <i>Carduus crispus</i> through ERK and p38 MAPK pathways</span><span class="source" tagx="source" title="source">Natural Product Sciences</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">273</span><span class="lpage" tagx="lpage" title="lpage">278</span></span></li> <li tag="ref"><a name="ref-43" /><span class="label" tagx="label" title="label">Liu et al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Sequencing and analysis of <i>Chrysanthemum carinatum</i> Schousb and <i>Kalimeris indica.</i> The complete chloroplast genomes reveal two inversions and <i>rbcL</i> as barcoding of the vegetable</span><span class="source" tagx="source" title="source">Molecules</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">1358</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules23061358">10.3390/molecules23061358</a></span></span></li> <li tag="ref"><a name="ref-44" /><span class="label" tagx="label" title="label">Luo et al. (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ou</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Development of EST-SSR markers for the invasive plant <i>Tithonia diversifolia</i> (Asteraceae)</span><span class="source" tagx="source" title="source">Applications in Plant Sciences</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">apps.1600011</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/apps.1600011">10.3732/apps.1600011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27437174">27437174</a></span></span></li> <li tag="ref"><a name="ref-45" /><span class="label" tagx="label" title="label">Ma, Sun &amp;amp; Zhao (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">YP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">The complete chloroplast genome of the marsh plant-<i>Leucanthemella linearis</i> (Asteraceae)</span><span class="source" tagx="source" title="source">Mitochondrial DNA Part B</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">237</span><span class="lpage" tagx="lpage" title="lpage">238</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/23802359.2018.1437801">10.1080/23802359.2018.1437801</a></span></span></li> <li tag="ref"><a name="ref-46" /><span class="label" tagx="label" title="label">Mandel et al. (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mandel</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dikow</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siniscalchi</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thapa</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watson</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Funk</span><span class="given-names" tagx="given-names" title="given-names">VA</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">A fully resolved backbone phylogeny reveals numerous dispersals and explosive diversifications throughout the history of Asteraceae</span><span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span><span class="volume" tagx="volume" title="volume">116</span><span class="issue" tagx="issue" title="issue">28</span><span class="fpage" tagx="fpage" title="fpage">14083</span><span class="lpage" tagx="lpage" title="lpage">14088</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1903871116">10.1073/pnas.1903871116</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31209018">31209018</a></span></span></li> <li tag="ref"><a name="ref-47" /><span class="label" tagx="label" title="label">Marochio et al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marochio</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bevilaqua</span><span class="given-names" tagx="given-names" title="given-names">MRR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takano</span><span class="given-names" tagx="given-names" title="given-names">HK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mangolim</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliveira Junior</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Machado</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silva</span><span class="given-names" tagx="given-names" title="given-names">FP</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Genetic admixture in species of <i>Conyza</i> (Asteraceae) as revealed by microsatellite markers</span><span class="source" tagx="source" title="source">Acta Scientiarum. Agronomy</span><span class="volume" tagx="volume" title="volume">39</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">437</span><span class="lpage" tagx="lpage" title="lpage">445</span><span class="pub-id"><a href="https://dx.doi.org/10.4025/actasciagron.v39i4.32947">10.4025/actasciagron.v39i4.32947</a></span></span></li> <li tag="ref"><a name="ref-48" /><span class="label" tagx="label" title="label">Nguyen, Kim &amp;amp; Kim (2015)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">PAT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J-H</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">The complete chloroplast genome of colchicine plants (<i>Colchicum autumnale</i> L. and <i>Gloriosa superba</i> L.) and its application for identifying the genus</span><span class="source" tagx="source" title="source">Planta</span><span class="volume" tagx="volume" title="volume">242</span><span class="fpage" tagx="fpage" title="fpage">223</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00425-015-2303-7">10.1007/s00425-015-2303-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25904477">25904477</a></span></span></li> <li tag="ref"><a name="ref-49" /><span class="label" tagx="label" title="label">Nie et al. (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biradar</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weining</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Complete chloroplast genome sequence of a major invasive species, crofton weed (<i>Ageratina adenophora</i>)</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e36869</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0036869">10.1371/journal.pone.0036869</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22606302">22606302</a></span></span></li> <li tag="ref"><a name="ref-50" /><span class="label" tagx="label" title="label">Panero (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panero</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic uncertainty and fossil calibration of Asteraceae chronograms</span><span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span><span class="volume" tagx="volume" title="volume">113</span><span class="issue" tagx="issue" title="issue">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">E411</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1517649113">10.1073/pnas.1517649113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26699490">26699490</a></span></span></li> <li tag="ref"><a name="ref-51" /><span class="label" tagx="label" title="label">Pantoja et al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pantoja</span><span class="given-names" tagx="given-names" title="given-names">PO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SimÃ³n-Porcar</span><span class="given-names" tagx="given-names" title="given-names">VI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puzey</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vallejo-MarÃ­n</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Genetic variation and clonal diversity in introduced populations of <i>Mimulus guttatus</i> assessed by genotyping at 62 single nucleotide polymorphism loci</span><span class="source" tagx="source" title="source">Plant Ecology &amp;amp; Diversity</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">5</span><span class="lpage" tagx="lpage" title="lpage">15</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/17550874.2017.1287785">10.1080/17550874.2017.1287785</a></span></span></li> <li tag="ref"><a name="ref-52" /><span class="label" tagx="label" title="label">Park et al. (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J-H</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Development of chloroplast microsatellite markers for the endangered <i>Maianthemum bicolor</i> (Asparagaceae s.l.)</span><span class="source" tagx="source" title="source">Applications in Plant Sciences</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">1600032</span><span class="pub-id"><a href="https://dx.doi.org/10.3732/apps.1600032">10.3732/apps.1600032</a></span></span></li> <li tag="ref"><a name="ref-53" /><span class="label" tagx="label" title="label">Posada (2008)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">jModelTest: phylogenetic model averaging</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">1253</span><span class="lpage" tagx="lpage" title="lpage">1256</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msn083">10.1093/molbev/msn083</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18397919">18397919</a></span></span></li> <li tag="ref"><a name="ref-54" /><span class="label" tagx="label" title="label">Poovitha et al. (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poovitha</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stalin</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balaji</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parani</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Multi-locus DNA barcoding identifies matK as a suitable marker for species identification in Hibiscus L</span><span class="source" tagx="source" title="source">Genome</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">59</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">1150</span><span class="lpage" tagx="lpage" title="lpage">1156</span><span class="pub-id"><a href="https://dx.doi.org/10.1139/gen-2015-0205">10.1139/gen-2015-0205</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27753524">27753524</a></span></span></li> <li tag="ref"><a name="ref-55" /><span class="label" tagx="label" title="label">Ronquist et al. (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teslenko</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van der Mark</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayres</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darling</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HÃ¶hna</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larget</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huelsenbeck</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space</span><span class="source" tagx="source" title="source">Systematic Biology</span><span class="volume" tagx="volume" title="volume">61</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">539</span><span class="lpage" tagx="lpage" title="lpage">542</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/sys029">10.1093/sysbio/sys029</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22357727">22357727</a></span></span></li> <li tag="ref"><a name="ref-56" /><span class="label" tagx="label" title="label">Rozas et al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozas</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferrer-Mata</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SÃ¡nchez-DelBarrio</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guirao-Rico</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Librado</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramos-Onsins</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SÃ¡nchez-Gracia</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">DnaSP 6: DNA sequence polymorphism analysis of large datasets</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">3299</span><span class="lpage" tagx="lpage" title="lpage">3302</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msx248">10.1093/molbev/msx248</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29029172">29029172</a></span></span></li> <li tag="ref"><a name="ref-57" /><span class="label" tagx="label" title="label">Salih et al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salih</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ä½</span><span class="given-names" tagx="given-names" title="given-names">MajeskÃ½</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwarzacher</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gornall</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heslop-Harrison</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Complete chloroplast genomes from apomictic <i>Taraxacum</i> (Asteraceae): identity and variation between three microspecies</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0168008</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0168008">10.1371/journal.pone.0168008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28182646">28182646</a></span></span></li> <li tag="ref"><a name="ref-58" /><span class="label" tagx="label" title="label">Sarwat et al. (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarwat</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nabi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srivastava</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular markers in medicinal plant biotechnology: past and present</span><span class="source" tagx="source" title="source">Critical Reviews in Biotechnology</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">32</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">74</span><span class="lpage" tagx="lpage" title="lpage">92</span><span class="pub-id"><a href="https://dx.doi.org/10.3109/07388551.2011.551872">10.3109/07388551.2011.551872</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21649550">21649550</a></span></span></li> <li tag="ref"><a name="ref-59" /><span class="label" tagx="label" title="label">Su et al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Comparative chloroplast genomics between the invasive weed <i>Mikania micrantha</i> and its indigenous congener <i>Mikania cordata</i>: Structure variation, identification of highly divergent regions, divergence time estimation, and phylogenetic analysis</span><span class="source" tagx="source" title="source">Molecular Phylogenetics and Evolution</span><span class="volume" tagx="volume" title="volume">126</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">195</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2018.04.015">10.1016/j.ympev.2018.04.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29684597">29684597</a></span></span></li> <li tag="ref"><a name="ref-60" /><span class="label" tagx="label" title="label">Sugiura (1992)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugiura</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="mixed-article-title" title="mixed-article-title">The chloroplast genome</span><span class="source" tagx="source" title="source">Plant Molecular Biology</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">149</span><span class="lpage" tagx="lpage" title="lpage">168</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00015612">10.1007/BF00015612</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1600166">1600166</a></span></span></li> <li tag="ref"><a name="ref-61" /><span class="label" tagx="label" title="label">Todorov et al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Todorov</span><span class="given-names" tagx="given-names" title="given-names">KT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheshmedzhiev</span><span class="given-names" tagx="given-names" title="given-names">IV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoyanov</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mladenova</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimitrova-Dyulgerova</span><span class="given-names" tagx="given-names" title="given-names">IZ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Taxonomic Structure of Genus Carduus L. in Bulgaria</span><span class="source" tagx="source" title="source">Ecologia Balkanica</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">229</span><span class="lpage" tagx="lpage" title="lpage">233</span></span></li> <li tag="ref"><a name="ref-62" /><span class="label" tagx="label" title="label">Trifinopoulos et al. (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trifinopoulos</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">LT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Von Haeseler</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minh</span><span class="given-names" tagx="given-names" title="given-names">BQ</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">W-IQ-TREE: a fast online phylogenetic tool for maximum likelihood analysis</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">44</span><span class="issue" tagx="issue" title="issue">W1</span><span class="fpage" tagx="fpage" title="fpage">W232</span><span class="lpage" tagx="lpage" title="lpage">W235</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw256">10.1093/nar/gkw256</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27084950">27084950</a></span></span></li> <li tag="ref"><a name="ref-63" /><span class="label" tagx="label" title="label">Tunsag, Davaakhuu &amp;amp; Batsuren (2011)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tunsag</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davaakhuu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Batsuren</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">New isoquinoline alkaloid from <i>Carduus crispus</i> L</span><span class="source" tagx="source" title="source">Mongolian Journal of Chemistry</span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">38</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">87</span></span></li> <li tag="ref"><a name="ref-64" /><span class="label" tagx="label" title="label">Untergasser et al. (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Untergasser</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cutcutache</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koressaar</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faircloth</span><span class="given-names" tagx="given-names" title="given-names">BC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Remm</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozen</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Primer3-new capabilities and interfaces</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">40</span><span class="issue" tagx="issue" title="issue">15</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e115</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gks596">10.1093/nar/gks596</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22730293">22730293</a></span></span></li> <li tag="ref"><a name="ref-65" /><span class="label" tagx="label" title="label">Verloove (2014)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verloove</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Carduus acanthoides (Asteraceae), a locally invasive alien species in Belgium</span><span class="source" tagx="source" title="source">Dumortiera</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">23</span><span class="lpage" tagx="lpage" title="lpage">28</span></span></li> <li tag="ref"><a name="ref-66" /><span class="label" tagx="label" title="label">Vu et al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vu</span><span class="given-names" tagx="given-names" title="given-names">THT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tran</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tran</span><span class="given-names" tagx="given-names" title="given-names">HD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Review on molecular markers for identification of Orchids</span><span class="source" tagx="source" title="source">Vietnam Journal of Science, Technology and Engineering</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">59</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">62</span><span class="lpage" tagx="lpage" title="lpage">75</span><span class="pub-id"><a href="https://dx.doi.org/10.31276/VJSTE.59(2).62">10.31276/VJSTE.59(2).62</a></span></span></li> <li tag="ref"><a name="ref-67" /><span class="label" tagx="label" title="label">Wang et al. (2015b)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="year" tagx="year" title="year">2015b</span><span class="mixed-article-title" title="mixed-article-title">Comparative analysis of asteraceae chloroplast genomes: structural organization, RNA editing and evolution</span><span class="source" tagx="source" title="source">Plant Molecular Biology Reporter</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">1526</span><span class="lpage" tagx="lpage" title="lpage">1538</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11105-015-0853-2">10.1007/s11105-015-0853-2</a></span></span></li> <li tag="ref"><a name="ref-68" /><span class="label" tagx="label" title="label">Wang et al. (2015a)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meinhardt</span><span class="given-names" tagx="given-names" title="given-names">LW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mischke</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsumoto</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2015a</span><span class="mixed-article-title" title="mixed-article-title">Developing single nucleotide polymorphism (SNP) markers from transcriptome sequences for identification of longan (<i>Dimocarpus longan</i>) germplasm</span><span class="source" tagx="source" title="source">Horticulture Research</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">14065</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/hortres.2014.65">10.1038/hortres.2014.65</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26504559">26504559</a></span></span></li> <li tag="ref"><a name="ref-69" /><span class="label" tagx="label" title="label">Ward, Gaskin &amp;amp; Wilson (2008)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ward</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaskin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ecological Genetics of Plant Invasion: What Do We Know?</span><span class="source" tagx="source" title="source">Invasive Plant Science and Management</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">1</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">98</span><span class="lpage" tagx="lpage" title="lpage">109</span><span class="pub-id"><a href="https://dx.doi.org/10.1614/IPSM-07-022.1">10.1614/IPSM-07-022.1</a></span></span></li> <li tag="ref"><a name="ref-70" /><span class="label" tagx="label" title="label">Xia et al. (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mason</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dou</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Development of high-density SNP markers and their application in evaluating genetic diversity and population structure in <i>Elaeis guineensis</i></span><span class="source" tagx="source" title="source">Frontiers in Plant Science</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">130</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2019.00130">10.3389/fpls.2019.00130</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30809240">30809240</a></span></span></li> <li tag="ref"><a name="ref-71" /><span class="label" tagx="label" title="label">Xie, Li &amp;amp; Jia (2005)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jia</span><span class="given-names" tagx="given-names" title="given-names">ZJ</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">A new flovone glycoside and other constituents from <i>Carduus crispus</i></span><span class="source" tagx="source" title="source">Pharmazie</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">233</span><span class="lpage" tagx="lpage" title="lpage">236</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15801681">15801681</a></span></span></li> <li tag="ref"><a name="ref-72" /><span class="label" tagx="label" title="label">Yi et al. (2013)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">Y-J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">The complete chloroplast genome sequence of <i>Cephalotaxus oliveri</i> (Cephalotaxaceae): evolutionary comparison of <i>Cephalotaxus</i> chloroplast DNAs and insights into the loss of inverted repeat copies in gymnosperms</span><span class="source" tagx="source" title="source">Genome Biology and Evolution</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">688</span><span class="lpage" tagx="lpage" title="lpage">698</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/gbe/evt042">10.1093/gbe/evt042</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23538991">23538991</a></span></span></li> <li tag="ref"><a name="ref-73" /><span class="label" tagx="label" title="label">Yun, Gil &amp;amp; Kim (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yun</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gil</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The complete chloroplast genome sequence of <i>Saussurea polylepis</i> (Asteraceae), a vulnerable endemic species of Korea</span><span class="source" tagx="source" title="source">Mitochondrial DNA Part B</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">650</span><span class="lpage" tagx="lpage" title="lpage">651</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/23802359.2017.1375881">10.1080/23802359.2017.1375881</a></span></span></li> <li tag="ref"><a name="ref-74" /><span class="label" tagx="label" title="label">Zhang et al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iaffaldano</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhuang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardina</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cornish</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Chloroplast genome resources and molecular markers differentiate rubber dandelion species from weedy relatives</span><span class="source" tagx="source" title="source">BMC Plant Biology</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">34</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12870-016-0967-1">10.1186/s12870-016-0967-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28152978">28152978</a></span></span></li> </ul> </div> </div>  </body></html>