<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Front Microbiol</span><span class="iso-abbrev" title="iso-abbrev">Front Microbiol</span><span class="publisher-id" title="publisher-id">Front. Microbiol.</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Microbiology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1664-302X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7736176">7736176</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fmicb.2020.601616">10.3389/fmicb.2020.601616</a></span><div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span>: <span class="subject" title="subject">Original Research</span></div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Antioxidant and Antiproliferative Activity of <i>Allium ursinum</i> and Their Associated Microbiota During Simulated <i>in vitro</i> Digestion in the Presence of Food Matrix</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">StanisavljeviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">Nemanja</span></span><a href="#aff1"><sup>1</sup></a><a href="#c001"><sup>*</sup></a><a href="#fn002"><sup>â€ </sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/1075530/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SokoviÄ‡ BajiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">Svetlana</span></span><a href="#aff1"><sup>1</sup></a><a href="#fn002"><sup>â€ </sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/314879/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JovanoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">Å½ivko</span></span><a href="#aff2"><sup>2</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/75189/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MatiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">Ivana</span></span><a href="#aff3"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TolinaÄ�ki</span><span class="given-names" tagx="given-names" title="given-names">Maja</span></span><a href="#aff1"><sup>1</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/269625/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PopoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">DuÅ¡anka</span></span><a href="#aff1"><sup>1</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/256881/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PopoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">Nikola</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TerziÄ‡-VidojeviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">Amarela</span></span><a href="#aff1"><sup>1</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/252610/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GoliÄ‡</span><span class="given-names" tagx="given-names" title="given-names">NataÅ¡a</span></span><a href="#aff1"><sup>1</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/77227/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BeÅ¡koski</span><span class="given-names" tagx="given-names" title="given-names">Vladimir</span></span><a href="#aff4"><sup>4</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/509563/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SamardÅ¾iÄ‡</span><span class="given-names" tagx="given-names" title="given-names">Jelena</span></span><a href="#aff5"><sup>5</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/536638/overview" /></span></span></div><span class="citation_author_institution" id="aff1">[], <sup>1</sup><span class="institution" title="institution">Laboratory for Molecular Microbiology, Institute of Molecular Genetics and Genetic Engineering, University of Belgrade</span><span class="addr-line" title="addr-line">Belgrade</span><span class="country" tagx="country" title="country">Serbia</span></span><span class="citation_author_institution" id="aff2">[], <sup>2</sup><span class="institution" title="institution">Faculty of Biology, University of Belgrade</span><span class="addr-line" title="addr-line">Belgrade</span><span class="country" tagx="country" title="country">Serbia</span></span><span class="citation_author_institution" id="aff3">[], <sup>3</sup><span class="institution" title="institution">Institute of Oncology and Radiology of Serbia</span><span class="addr-line" title="addr-line">Belgrade</span><span class="country" tagx="country" title="country">Serbia</span></span><span class="citation_author_institution" id="aff4">[], <sup>4</sup><span class="institution" title="institution">Faculty of Chemistry, University of Belgrade</span><span class="addr-line" title="addr-line">Belgrade</span><span class="country" tagx="country" title="country">Serbia</span></span><span class="citation_author_institution" id="aff5">[], <sup>5</sup><span class="institution" title="institution">Laboratory for Plant Molecular Biology, Institute of Molecular Genetics and Genetic Engineering, University of Belgrade</span><span class="addr-line" title="addr-line">Belgrade</span><span class="country" tagx="country" title="country">Serbia</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: MiklÃ³s TakÃ³, University of Szeged, Hungary</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Quang Duc Nguyen, Szent IstvÃ¡n University, Hungary; Svetoslav Todorov, University of SÃ£o Paulo, Brazil</p> </div> <div class="corresp" title="corresp">
*Correspondence: Nemanja StanisavljeviÄ‡, <span class="email" tagx="email" title="email">nemanjalab08@imgge.bg.ac.rs</span></div> <div class="fn-type-other" title="other"> <p><sup>â€ </sup>These authors have contributed equally to this work</p> </div> <div class="fn-type-other" title="other"> <p>This article was submitted to Food Microbiology, a section of the journal Frontiers in Microbiology</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-12-12</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">601616</span><span class="history" title="history"><span class="received" title="received">received: 2020-9-01</span><span class="accepted" title="accepted">accepted: 2020-11-09</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">StanisavljeviÄ‡, SokoviÄ‡ BajiÄ‡, JovanoviÄ‡, MatiÄ‡, TolinaÄ�ki, PopoviÄ‡, PopoviÄ‡, TerziÄ‡-VidojeviÄ‡, GoliÄ‡, BeÅ¡koski and SamardÅ¾iÄ‡</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>In this study, for the first time, the comprehensive analysis of antiproliferative and antioxidant activities of ramson, followed by the analysis of its associated microbiota and health-promoting effects of lactic acid bacteria (LAB), was performed. Ramson (<i>Allium ursinum</i>) is recognized as a medicinal plant with a long history of use in traditional medicine due to its antimicrobial and antioxidant activity. In this study the influence of <i>in vitro</i> gastrointestinal digestion on the cytotoxic activity of <i>A. ursinum</i> extracts against human malignant cell lines was demonstrated. Seven sulfur compounds, the degradation products of thiosulfinates, including diallyl disulfide were shown to inhibit proliferation of malignant cells by inducing accumulation within G2/M phase as well as to induce apoptosis through activation of caspase-3 and mitochondrial signaling pathway. Further, the <i>A. ursinum</i> microbiota, particularly LAB with potential probiotic effects, was analyzed by culture-dependent method and culture-independent method [denaturing gradient gel electrophoresis (DGGE)]. The obtained results revealed that the most abundant genera were <i>Streptococcus</i>, <i>Lactobacillus</i>, and <i>Bacillus</i>. The <i>Lactobacillus</i> genus was mainly represented by <i>L. fermentum.</i> The pulsed-field gel electrophoresis (PFGE) analysis revealed the presence of two PFGE pulsotypes. The probiotic potential of the strain <i>L. fermentum</i> BGSR163 belonging to PFGE pulsotype 1 and the strain <i>L. fermentum</i> BGSR227 belonging to the PFGE pulsotype 2 was characterized. The results revealed that both strains are safe for human use, successfully survive the simulated gastrointestinal conditions, have potential to transiently colonize the gastrointestinal tract (GIT) and have a protective immunomodulatory effect, inducing the production of proinflammatory cytokine IL17 and regulatory cytokine IL10, while decreasing the production of proinflammatory cytokine IFN-Î³. In conclusion, the results of this study suggest that consumption of <i>A. ursinum</i> might have health-promoting properties, including anticancer effects, while <i>L. fermentum</i> strains isolated from <i>A. ursinum</i> leaves could be used as probiotics for human consumption.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
ramson</div> <div class="kwd" title="kwd">
antiproliferative</div> <div class="kwd" title="kwd">
antioxidant â€"phytochemical studies</div> <div class="kwd" title="kwd">
probiotic</div> <div class="kwd" title="kwd">
immunomodulation</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Ministarstvo Prosvete, Nauke i TehnoloÅ¡kog Razvoja<span class="named-content" title="named-content">10.13039/501100004564</span></div><span class="award-id" tagx="award-id" title="award-id">451-03-68 / 2020-14 / 200042</span></div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Ramsons or wild garlic (<i>Allium ursinum</i>) belongs to medicinal plant species which has been recognized as an important source of biologically active compounds, with the long history of application in traditional medicine as a natural remedy for gastrointestinal, cardiovascular, and respiratory disorders (<a href="#B66">Sobolewska et al., 2015</a>). However, its chemical composition and biological activities have been rarely studied to date. The most important ramsons chemical constituents with pharmacological activity are undoubtedly sulfur compounds; among the most common are glutamyl peptides and sulfoxides, which are at the same time the most widely studied chemicals in this plant species. <i>Allium</i> species are characterized by high content of S-alk(en)yl-L-cysteine-sulfoxides which after hydrolysis yields many different volatile compounds including thiosulfinates and polysulfides responsible for characteristic odor and flavor (<a href="#B66">Sobolewska et al., 2015</a>). The quantitative profile of cysteine-sulfoxides which could be found in ramson is highly variable depending on plant organ and harvest time. The most abundant sulfoxides are: methiin [(+)-S-methyl-L-cysteine-sulfoxide], aliin [(+)-S-2-propenyl-L-cysteine-sulfoxide], isoaliin [(+)-S-(1-propenyl)-L-cysteine-sulfoxide], propiin [(+)-S-propyl-L-cysteine-sulfoxide], as well as ethiin (S-ethyl-cysteine-sulfoxide). Tissue damage induced by processing or pathogen attack leads to hydrolytic cleavage of sulfoxides by C,S-lyases and formation of thiosulfinates responsible for the characteristic odor, as well as pyruvic acid and ammonia. The most abundant thiosulfinates present in ramson extracts are allicin (diallylthiosulfinate) and methyl-allyl- or dimethyl thiosulfinates, regarded as unstable and reactive which could be easily decomposed to dithiins, sulfides, polysulfides, ajoenes and many other volatile or non-volatile compounds even at room temperature. Although the composition, stability, as well as bioavailability of thiosulfinates from fresh garlic and several garlic products have been already examined (<a href="#B37">Lawson and Gardner, 2005</a>; <a href="#B58">Rahman, 2007</a>; <a href="#B36">Lawson and Hunsaker, 2018</a>), this type of study has never been undertaken on <i>A. ursinum</i> fresh leaves. For that purpose, a standardized static <i>in vitro</i> digestion model described by <a href="#B40">Minekus et al. (2014)</a> has been applied in this study to simulate the digestion of ramsons leaves in the presence of complex food matrix. Further, bioaccessible fraction was examined using two-dimensional gas chromatography to assess decomposition of thiosulfinates. In addition, antiproliferative and antioxidant activities were determined in the same fractions.</p> <p>Despite the increased global interest in bioactive metabolites from medicinal plants, very little is known about the health-promoting effects of plant-associated microbiota. A range of bacterial species colonizes the rhizosphere, plant surfaces and internal tissues (epiphytes or endophytes) of various dietary and medicinal plants (<a href="#B15">Danhorn and Fuqua, 2007</a>; <a href="#B8">Borozan and Raba, 2016</a>; <a href="#B54">Presta et al., 2017</a>). The microbiota composition from leaves, roots and rhizospheric soil was followed in aromatic plants <i>Thymus vulgaris</i> and <i>Thymus citriodorus</i> and correlated with the composition of essential oils (<a href="#B10">Checcucci et al., 2017</a>). Further, the antimicrobial activities of bacteria isolated from root, leaf and fruit of medicinal plants <i>Echinacea purpurea</i> and <i>Peganum harmala</i> were tested (<a href="#B54">Presta et al., 2017</a>; <a href="#B7">Bibi, 2017</a>). The antimicrobial activities against various pathogenic bacteria were scored, such as <i>Escherichia coli</i>, methicillin-resistant <i>Staphylococcus aureus</i> (MRSA), <i>Enterococcus faecium</i>, <i>E. faecalis</i>, and <i>Pseudomonas aeruginosa</i>, indicating that medicinal plants could be a valuable source of bacteria with antimicrobial activity against clinically relevant pathogens (<a href="#B7">Bibi, 2017</a>). In addition, the roles of endophytic bacterial strains of <i>Bacillus</i> sp., <i>Enterobacter</i> sp., <i>Pantoea</i> sp., <i>Erwinia</i> sp., and <i>Stenotrophomonas</i> sp. in the biosynthesis of various metabolites and therapeutic potential of <i>Fagonia indica</i> was reported (<a href="#B58">Rahman, 2007</a>). Microbiota associated with <i>A. ursinum</i> has not been studied so far.</p> <p>The composition of lactic acid bacteria (LAB) from <i>A. ursinum</i> fresh leaves was studied. Moreover, the range of lactobacilli species have been isolated, and their probiotic potential was characterized.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="isolationandenumerationoflab" title="sec"> <div class="title" tagx="title" title="title">
Isolation and Enumeration of LAB</div> <p>The 10 g of fresh leaves was immersed in 90 ml of 0.85% saline and incubated for 3 h on 37Â°C with agitation. A volume of 1 ml of leaves and saline mixture was supplemented with 15% (w/v) glycerol, and later used for microbiological and microbiota analysis. After 3 h, 5 ml of mixture was added to 45 ml of 3 different media most suitable for isolation of LAB (MRS [Oxoid,], GM17 [Oxoid] and Nutrient broth[Oxoid]) and incubated for 72 h aerobically at 22Â°C and 30Â°C, and anaerobically (BD GasPak<sup>TM</sup> EZ CO<sub>2</sub> Container System, Becton, Dickinson and Company, Sparks, MD, United States) at 45Â°C. After incubation, serial dilutions (10<sup>1</sup>â€"10<sup>7</sup>) were prepared and plated onto MRS, GM17 and Nutrient agar with cycloheximide (1 Î¼l/ml) for colony count. After the incubation approx. 30 colonies from each plate were selected randomly, streaked on new MRS, GM17, and Nutrient agar plates, respectively and examined under a microscope. The pure cultures were stored at âˆ’80Â°C in GM17, MRS or Nutrient broth supplemented with 15% (w/v) glycerol.</p> <p>In parallel, 500 Î¼l of each culture (grown in MRS, GM17 and Nutrient and from each temperature), was added to 10 ml of reconstituted skim milk (Merck GmbH, Darmstadt, Germany) and incubated for 48 h on three different temperatures as previously mentioned. After 48 h, the serial dilutions (10<sup>1</sup>â€"10<sup>7</sup>) of the isolates that curdled the skimmed milk were prepared. The rest of the experimental procedure was the same as the procedure mentioned above.</p> <p>The 217 pure cultures were isolated and characterized according to their morphology by Gram staining and cell formation, catalase test and gelatinase production. Out of 217 isolates, 50 Gram-positive aerobically grown isolates were selected for further analysis. They were divided into 3 subgroups according to their growth on different mediums and different temperatures (23 isolates chosen from MRS plates incubated at 30Â°C, as well as 16 isolates chosen from GM17 plates and 11 isolates chosen from Nutrient-plates incubated at 22Â°C).</p> </div> <div class="dnaisolation" title="sec"> <div class="title" tagx="title" title="title">
DNA Isolation</div> <p>Total DNA from pure cultures and first serial dilution of leaves and saline mixture were extracted as described by <a href="#B25">Hopwood et al. (1985)</a>.</p> </div> <div class="16srdnasequencingoflabisolates" title="sec"> <div class="title" tagx="title" title="title">
16S rDNA Sequencing of LAB Isolates</div> <p>16S rDNA PCR amplification was performed with UNI16SF (5â€²-GAGAGTTTGATCCTGGC-3â€²) and UNI16SR (5â€²-AGGAGGTGATCCAGCCG-3â€²) primers, complementary to 16S rRNA gene, as described by <a href="#B27">Jovcic et al. (2009)</a>. The obtained PCR amplicons were purified (Qiagen, GmbH, Hilden, Germany) and sequenced (Macrogen, Amsterdam, The Netherlands). To determine sequence similarities the BLAST algorithm was used in the NCBI nucleotide sequence database<sup><a href="#footnote1">1</a></sup>.</p> </div> <div class="denaturinggradientgelelectrophoresisanalysisandfragmentsequencing" title="sec"> <div class="title" tagx="title" title="title">
Denaturing Gradient Gel Electrophoresis Analysis and Fragment Sequencing</div> <p>To investigate the dominant bacterial communities by denaturing gradient gel electrophoresis (DGGE) analysis, Quick-DNA Fecal/Soil Microbe Kit (Zymo Research, Irvine, CA, United States) was used for extraction of bacterial DNA from the first serial dilution of leaves and saline mixture. PCR amplification of the obtained DNA was performed using lactobacilli-specific primers Lab-0159f and Uni-0515-GCr (Invitrogen, Paisley, United Kingdom), as described by <a href="#B38">Lukic et al. (2013)</a>. The samples were amplified using KAPA <i>Taq</i> DNA polymerase (KAPA Biosystems, Cape Town, South Africa) in GeneAmp PCR System 2700 (Applied Biosystems, Foster City, CA, United States). DGGE analysis and gel manipulation after electrophoresis was performed according to <a href="#B38">Lukic et al. (2013)</a>. DGGE was performed on DGGE-2001 (C.B.S. Scientific, San Diego, CA, United States). Cloning of PCR fragments of interest, the transformation of the DH5Î± competent cells and analysis of selected transformants was performed as previously described <a href="#B35">Kulas et al. (2019)</a>. Sequence annotation and the database searches for sequence similarities were performed with the BLAST tool available online.<sup>1</sup>.</p> </div> <div class="pulsed-fieldgelelectrophoresis" title="sec"> <div class="title" tagx="title" title="title">
Pulsed-Field Gel Electrophoresis</div> <p><i>Lactobacillus</i> sp. isolates were analyzed by pulsed-field gel electrophoresis (PFGE) in order to determine differences among the strains. PFGE was carried out as previously described (<a href="#B70">Tenover et al., 1995</a>). Briefly, PFGE with <i>Not</i>I-digested genomic DNA was performed for 18 h at 300 V at 9Â°C using a 2015 Pulsafor unit apparatus (LKB Instruments, Bromma, Sweden). Grouping of isolates based on the PFGE results was done using SPSS software package, version 20 (IBM Inc., Chicago, IL, United States).</p> </div> <div class="invitrodigestion" title="sec"> <div class="title" tagx="title" title="title"><i>In vitro</i> Digestion</div> <p>Fresh leaves of <i>A. ursinum</i> were quickly homogenized in liquid nitrogen using mortar and pestle. (Oxoid, Hampshire, United Kingdom) medium, were harvested by centrifugation (4500 Ã— <i>g</i>, 10 min) and washed in 0.85% saline. Aliquots of <i>A. ursinum</i> powder (2 g) or bacterial pellets, obtained from 10 ml of overnight <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 cultures in MRS were placed in 50 ml conical tubes and mixed with 3 g of infant formula (Juvitana, Swisslion Product d.o.o. Indjija, Serbia) which contained 3% of protein, 10% of carbohydrate and 1% of fat according to manufacturerâ€™s specifications. The formula was prepared from boiled turkey meat, boiled potato and corn paste, rice flour, and water. Control was prepared by mixing 2 ml of distilled water with 3 g of infant formula. Obtained mixtures were subjected to standardized static <i>in vitro</i> digestion according to the method of <a href="#B40">Minekus et al. (2014)</a>. Oral phase of simulated digestion was started by addition of 5 g of previously obtained mixture to 3.5 ml of simulated salivary fluid (SSF), 0.5 ml of salivary Î±-amylase solution (1500 U/ml), 25 Î¼l of 0.3 M CaCl<sub>2</sub> and 975 Î¼l of water, shaken vigorously and incubated 2 min at 37Â°C. In the following gastric phase, 10 ml of previously treated oral bolus was mixed with 7.5 ml of simulated gastric fluid (SGF), 1.6 ml of pepsin solution (25000 U/ml), prepared in SGF, 5 Î¼l of 0.3 M CaCl<sub>2</sub> and pH was adjusted to 3.0 using 1M HCl. The mixture was supplemented with distilled water to achieve volume of 20 ml. Obtained mixture was incubated at 37Â°C for 2 h, using orbital shaker at 300 rpm. For the final intestinal phase 20 mL of obtained gastric chyme was mixed with 11 ml of simulated intestinal fluid (SIF), 5 ml of pancreatin solution 800 U/ml (trypsin activity) prepared in SIF, 2,5 ml of 160 mM bile salt solution and 40 Î¼l of 0.3 M CaCl<sub>2</sub>. The pH value of 7.0 was adjusted using 1M NaOH and the mixture was supplemented with distilled water to reach final volume of 40 ml. Additional incubation was performed with constant shaking at 300 rpm during 2h at 37Â°C. After completed digestion samples were centrifuged at 4500 Ã— <i>g</i> for 10 min at 4Â°C, supernatants were kept and their volume was accurately measured, then immediately frozen in liquid nitrogen and kept for further experiments at âˆ’80Â°C. Composition of SSF, SGF and SIF electrolyte solutions was presented in <a href="#T1">Table 1</a>. The results of <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 survival after <i>in vitro</i> digestion were expressed as Log CFU (Colony forming units)/ml and the survival rate was calculated from the viable cell count with respect to initial cell counts. The aliquots were taken at 0, 10, 120, and 220 min of the simulated digestion. The 10 times serial dilutions were prepared in 0.85% saline and plated on MRS agar plates and incubated anaerobically at 37Â°C for 48 h. The experiments were performed in triplicates.</p> <div class="table-wrap_UNKNOWN" id="T1" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p>Preparation of stock solutions and simulated digestive fluids.</p>  <table frame="hsides" rules="groups" cellspacing="5" cellpadding="5"> <thead> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Compound</td> <td valign="top" align="center" colspan="2" rowspan="1">Stock conc.</td> <td valign="top" align="center" rowspan="1" colspan="1">SSF <hr /> </td> <td valign="top" align="center" rowspan="1" colspan="1">SGF <hr /> </td> <td valign="top" align="center" rowspan="1" colspan="1">SIF <hr /> </td> </tr> <tr> <td valign="top" colspan="3" rowspan="1" /> <td valign="top" align="center" rowspan="1" colspan="1">pH 7.0 <hr /> </td> <td valign="top" align="center" rowspan="1" colspan="1">pH 3.0 <hr /> </td> <td valign="top" align="center" rowspan="1" colspan="1">pH 7.0 <hr /> </td> </tr> <tr> <td rowspan="1" colspan="1" /> <td valign="top" align="center" colspan="2" rowspan="1" /> <td valign="top" align="center" rowspan="1" colspan="1">Conc. in SSF</td> <td valign="top" align="center" rowspan="1" colspan="1">Conc. in SGF</td> <td valign="top" align="center" rowspan="1" colspan="1">Conc. in SSF</td> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1" /> <td valign="top" align="center" rowspan="1" colspan="1"><b>g/l</b></td> <td valign="top" align="center" rowspan="1" colspan="1"><b>mol/l</b></td> <td valign="top" align="center" rowspan="1" colspan="1"><b>mmol/l</b></td> <td valign="top" align="center" rowspan="1" colspan="1"><b>mmol/l</b></td> <td valign="top" align="center" rowspan="1" colspan="1"><b>mmol/l</b></td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">KCl</td> <td valign="top" align="center" rowspan="1" colspan="1">37.3</td> <td valign="top" align="center" rowspan="1" colspan="1">0.5</td> <td valign="top" align="center" rowspan="1" colspan="1">15.1</td> <td valign="top" align="center" rowspan="1" colspan="1">6.9</td> <td valign="top" align="center" rowspan="1" colspan="1">6.8</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">KH<sub>2</sub>PO<sub>4</sub></td> <td valign="top" align="center" rowspan="1" colspan="1">68</td> <td valign="top" align="center" rowspan="1" colspan="1">0.5</td> <td valign="top" align="center" rowspan="1" colspan="1">3.7</td> <td valign="top" align="center" rowspan="1" colspan="1">0.9</td> <td valign="top" align="center" rowspan="1" colspan="1">0.8</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">NaHCO<sub>3</sub></td> <td valign="top" align="center" rowspan="1" colspan="1">84</td> <td valign="top" align="center" rowspan="1" colspan="1">1</td> <td valign="top" align="center" rowspan="1" colspan="1">13.6</td> <td valign="top" align="center" rowspan="1" colspan="1">25</td> <td valign="top" align="center" rowspan="1" colspan="1">85</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">NaCl</td> <td valign="top" align="center" rowspan="1" colspan="1">117</td> <td valign="top" align="center" rowspan="1" colspan="1">2</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">47.2</td> <td valign="top" align="center" rowspan="1" colspan="1">38.4</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">MgCl<sub>2</sub>(H<sub>2</sub>0)<sub>6</sub></td> <td valign="top" align="center" rowspan="1" colspan="1">30.5</td> <td valign="top" align="center" rowspan="1" colspan="1">0.15</td> <td valign="top" align="center" rowspan="1" colspan="1">0.15</td> <td valign="top" align="center" rowspan="1" colspan="1">0.1</td> <td valign="top" align="center" rowspan="1" colspan="1">0.33</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">(NH<sub>4</sub>)<sub>2</sub>CO<sub>3</sub></td> <td valign="top" align="center" rowspan="1" colspan="1">48</td> <td valign="top" align="center" rowspan="1" colspan="1">0.5</td> <td valign="top" align="center" rowspan="1" colspan="1">0.06</td> <td valign="top" align="center" rowspan="1" colspan="1">0.5</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">NaOH</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">1</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">8.4</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">HCl</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> <td valign="top" align="center" rowspan="1" colspan="1">6</td> <td valign="top" align="center" rowspan="1" colspan="1">1.1</td> <td valign="top" align="center" rowspan="1" colspan="1">15.6</td> <td valign="top" align="center" rowspan="1" colspan="1">â€"</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">CaCl<sub>2</sub> (H<sub>2</sub>O)<sub>2</sub></td> <td valign="top" align="center" rowspan="1" colspan="1">44.1</td> <td valign="top" align="center" rowspan="1" colspan="1">0.3</td> <td valign="top" align="center" rowspan="1" colspan="1">(0.75)</td> <td valign="top" align="center" rowspan="1" colspan="1">(0.075)</td> <td valign="top" align="center" rowspan="1" colspan="1">(0.3)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"><span class="attrib" tagx="attrib" title="attrib"><i>SSF, simulated salivary fluid; SGF, simulated gastric fluid; SIF, simulated intestinal fluid.</i></span></div> </div> </div> <div class="comprehensivetwo-dimensionalgaschromatography/massspectrometry(gcÃ—gc-ms)" title="sec"> <div class="title" tagx="title" title="title">
Comprehensive Two-Dimensional Gas Chromatography/Mass Spectrometry (GC Ã— GC-MS)</div> <p>For GC Ã— GC-MS analysis, obtained samples after <i>in vitro</i> digestion of <i>A. ursinum</i> were extracted using dichloromethane (CH<sub>2</sub>Cl<sub>2</sub>) for 1 h by liquid/liquid extraction. Prepared extracts were filtered using membrane filter (0.45 Î¼m, Agilent) and after concentration directly analyzed by GC Ã— GC-MS GCMS-QP2010 Ultra (Shimadzu, Kyoto, Japan) equipped with ZX2 thermal modulation system (Zoex Corp.) as Total Ion Chromatograms (TIC). A Rtx<sup>Â®</sup>-1 (first column: RESTEK, Crossbond<sup>Â®</sup> 100% dimethyl polysiloxane, 30 m, 0.25 mm ID, df = 0.25 Î¼m) and a BPX50 (SGE Analytical Science, 1 m, 0.1 mm ID, df = 0.1 Î¼m) columns were connected using the GC Ã— GC modulator as the first and second capillary columns, respectively. The oven was programmed at an initial temperature of 40Â°C for 4 min, then ramped at 10Â°C/min to 310Â°C. Isothermal step at 310Â°C for 5 min was at the program end. The modulation period was 6 s with 300 ms of hot jet pulse. The MS data were collected using Shimadzu GC/MS Real-Time Analysis and GC Ã— GC-MS data were analyzed using GC Image R2.6 (GC Image LLC). Obtained MS spectra were compared with the MS libraries NIST 11, NIST 11s, and Wiley 8.</p> </div> <div class="totalphenoliccontent(tpc)" title="sec"> <div class="title" tagx="title" title="title">
Total Phenolic Content (TPC)</div> <p>Total phenolic content was determined according to the previously described method of <a href="#B28">Julkunen-Tiitto (1985)</a> with slight modifications. Aliquots of supernatants obtained after <i>in vitro</i> digestion (20 Î¼l) were mixed with 1580 Î¼l of distilled water and 100 Î¼l of Folinâ€"CiocÃ¢lteu reagent and shaken vigorously. After 15 min incubation at room temperature, 300 Î¼l of 20% sodium carbonate solution was added to the mixture which was then incubated for additional 2 h. After the incubation absorbance of the mixture was recorded at 765 nm. Total phenolic content was calculated using gallic acid standard curve and expressed as mg of gallic acid equivalents per ml of supernatant (mg GAE/ml).</p> </div> <div class="totalflavonoidcontent(tfc)" title="sec"> <div class="title" tagx="title" title="title">
Total Flavonoid Content (TFC)</div> <p>Supernatants collected after <i>in vitro</i> digestions were concentrated five times by evaporating under vacuum at 60Â°C. Obtained concentrates were then centrifuged at 5000 Ã— <i>g</i> for 15 min to precipitate insoluble particles. The total flavonoid content in the concentrated supernatants was then determined by aluminum chloride assay according to the previously described method of <a href="#B60">Ribeiro et al. (2015)</a> with modifications. Supernatants (125 Î¼l) were mixed with 625 Î¼l of distilled water and 37.5 Î¼l 5% NaNO<sub>2</sub>. After 6 min of incubation at room temperature, 75 Î¼l of 10% AlCl<sub>3</sub> solution was added and incubation was prolonged for 5 min. After the second incubation, 250 Î¼l of 1M NaOH was added and mixture was supplemented with distilled water to a final volume of 1.25 ml. Absorbance of the samples was recorded at 510 nm. The flavonoid content was calculated using quercetin standard curve (5â€"100 Î¼g/ml) and expressed as Î¼g of quercetin equivalents per mL of supernatant (Î¼g QE/ml).</p> </div> <div class="measurementofrelativedpphscavengingcapacity(rdsc)" title="sec"> <div class="title" tagx="title" title="title">
Measurement of Relative DPPH Scavenging Capacity (RDSC)</div> <p>Relative 2,2-Diphenyl-1-Picrylhydrazyl (DPPH) radical scavenging capacity measurement was determined according to <a href="#B11">Cheng et al. (2006)</a>. The reaction mixture consisted of 100 Î¼L of supernatant diluted with methanol. Five different concentrations (10, 20, 40, 60, and 80 Î¼L) of supernatant and Trolox were used. Then, 100 Î¼L of 200 Î¼M DPPH in methanol was added to each microplate and shaken. Blank contained 200 Î¼L of methanol and control sample was prepared by mixing 100 Î¼L of methanol and 100 Î¼L of DPPH solution. Absorbance was continuously recorded each minute for 1.5 h at 515 nm. The percent of DPPH quenched for each time point was calculated according to the equation:</p> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
%</div> <div class="unknown" title="mpadded" tag="mpadded"> <div class="mi_UNKNOWN">
DPPH</div> </div> <div class="mi_UNKNOWN">
quenched</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
sample</div> </div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
blank</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
/</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
control</div> </div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
blank</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
]</div> <div class="mo_UNKNOWN">
Ã—</div> <div class="mn_UNKNOWN">
100</div> </div></div></div> <p>The values of DPPH in percents quenched in different time points for each extract and standard were plotted against reaction time. The area under obtained curve (AUC) was determined according to the following equation:</p> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="unknown" title="mpadded" tag="mpadded"> <div class="mtext_UNKNOWN">
AUC</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
5</div> <div class="mo_UNKNOWN">
â�¢</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
3</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
â€¦</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
5</div> <div class="mo_UNKNOWN">
â�¢</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div></div></div> <p>Where <i>f</i><sub>0</sub> represents DPPH quenched at the start of the measurement and <i>f</i><sub><i>i</i></sub> is DPPH quenched at reaction time <i>i</i>. Relative DPPH radical scavenging capacity (RDSC) was calculated according to the equation:</p> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="unknown" title="mpadded" tag="mpadded"> <div class="mi_UNKNOWN">
RDSC</div> </div> <div class="mo_UNKNOWN">
â�¢</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="unknown" title="mpadded" tag="mpadded"> <div class="mi_UNKNOWN">
mM</div> </div> <div class="mo_UNKNOWN">
â�¢</div> <div class="mi_UNKNOWN">
TE</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
ml</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
AUC</div> <div class="mi_UNKNOWN">
sample</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
AUC</div> <div class="mi_UNKNOWN">
trolox</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
Ã—</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
molarity</div> <div class="mi_UNKNOWN">
trolox</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
volume</div> <div class="mi_UNKNOWN">
sample</div> </div> </div> </div></div></div> </div> <div class="ferrousionbindingcapacity" title="sec"> <div class="title" tagx="title" title="title">
Ferrous Ion Binding Capacity</div> <p>The ferrous ion binding capacity was monitored by the previously described method of <a href="#B16">Dinis et al. (1994)</a> with slight modifications. Briefly, 940 Î¼L of supernatant dilutions in distilled water were mixed with 20 Î¼L of 2 mM ferrous chloride and incubated for 30 min at room temperature. After incubation, 200 Î¼L of 5 mM ferrozine was added and the mixture was incubated for 10 min under the same conditions. Formation of a red ferrous ion-ferrozine complex was recorded at 562 nm. Control sample was prepared with distilled water instead of the sample. Percentage of ferrous ion binding was calculated according to the equation:</p> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
%</div> <div class="unknown" title="mpadded" tag="mpadded"> <div class="mi_UNKNOWN">
of</div> </div> <div class="unknown" title="mpadded" tag="mpadded"> <div class="mi_UNKNOWN">
bound</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
Fe</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
control</div> </div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
sample</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
control</div> </div> <div class="mo_UNKNOWN">
Ã—</div> <div class="mn_UNKNOWN">
100</div> </div></div></div> </div> <div class="invitrocytotoxicactivity" title="sec"> <div class="title" tagx="title" title="title"><i>In vitro</i> Cytotoxic Activity</div> <p>The cytotoxic effects of the undigested and digested <i>A. ursinum</i> extracts were investigated on human cervical adenocarcinoma HeLa and human colon adenocarcinoma LS174 cell lines (American Type Culture Collection, Manassas, VA, United States). The cells were cultured in the complete nutrient medium (RPMI 1640 supplemented with <div class="sc_UNKNOWN">
L</div>-glutamine (2 mM), streptomycin (100 Î¼g/ml) and penicillin (100 IU/ml), 10% heat-inactivated fetal bovine serum (FBS) and 25 mM HEPES), as described elsewhere (<a href="#B39">MatiÄ‡ et al., 2013</a>). HeLa (2,000 cells per well) and LS174 (7,000 cells per well) cells were seeded in 96-well microtiter plates. The cells were allowed to adhere for 24 h, and afterward, five concentrations of tested extracts were added to the cells ranging from 1.25% to 20% (five serial two-fold dilutions). The nutrient medium was added to the control cells. Cell survival was determined by MTT assay after 72 h treatment, according to the procedure firstly described by <a href="#B41">Mosmann (1983)</a> and modified by <a href="#B46">Ohno and Abe (1991)</a>. All experiments were done in triplicate. Cisplatin was used as a positive control. The IC<sub>50</sub> concentration was defined as the concentration of an extract causing the inhibition of cell survival by 50% compared with a control cell sample.</p> </div> <div class="cellcycleanalysis" title="sec"> <div class="title" tagx="title" title="title">
Cell Cycle Analysis</div> <p>HeLa and LS174 cells were exposed to extracts of <i>A. ursinum</i> for 24 h (applied concentrations were 5% for HeLa and 20% for LS174 cells). After incubation the cells were collected, washed in phosphate buffered saline (PBS) and fixed in 70% ethanol on ice, according to a standard protocol (<a href="#B46">Ohno and Abe, 1991</a>; <a href="#B47">Ormerod, 1994</a>). Cell samples were stored at âˆ’20Â°C for at least one week before staining. The cells were incubated with RNase A in PBS (final concentration 200 Î¼g/ml) for 30 min at 37Â°C. Next, the propidium iodide, at a concentration of 40 Î¼g/ml, was added to the cells. Cell cycle phase distributions were assessed by FACSCalibur Flow Cytometer (BD Biosciences, Franklin Lakes, NJ, United States). CELLQuest software (BD Biosciences) was used for data analyses.</p> </div> <div class="effectsofspecificcaspase-3inhibitor" title="sec"> <div class="title" tagx="title" title="title">
Effects of Specific Caspase-3 Inhibitor</div> <p>To examine whether <i>A. ursinum</i> extracts could activate apoptosis in HeLa cells, the specific irreversible peptide inhibitor of caspase-3 (Z-DEVD-FMK, R&amp;amp;D Systems, Minneapolis, United States) was used for cell cycle analysis, as described previously (<a href="#B39">MatiÄ‡ et al., 2013</a>). Briefly, HeLa cells were preincubated for 2 h with Z-DEVD-FMK at a final concentration of 40 Î¼M. Afterwards, the cells were treated with extracts (applied at a concentration of 5%). After 24 h treatment, the cells were harvested and fixed in 70% ethanol. Cell cycle analysis was performed as described in the previous section.</p> </div> <div class="gelatinaseactivity" title="sec"> <div class="title" tagx="title" title="title">
Gelatinase Activity</div> <p>Gelatinase activity was assayed according to the procedure described by <a href="#B69">Su et al. (1991)</a>. The bacterial isolates were plated on agar plates containing 3% gelatin (Oxoid) at 37Â°C for 48 h and flooded with a saturated solution of ammonium sulfate (Centrohem, Stara Pazova, Serbia). A transparent halo around cells and gelatin precipitates was used as an indication for gelatinase producers. <i>E. faecalis</i> V583 (<a href="#B50">Paulsen, 2003</a>) was used as a positive control.</p> </div> <div class="hemolyticactivity" title="sec"> <div class="title" tagx="title" title="title">
Hemolytic Activity</div> <p>The two <i>L. fermentum</i> (BGSR163 and BGSR227) isolates were tested for hemolytic activity. The strains were cultured in MRS broth and were streaked on Columbia agar plates containing 5% of sheep blood (Oxoid). The plates were incubated for 48 h at 37Â°C. Blood agar plates were examined for signs of Î²-hemolysis (clear zones around colonies), Î±-hemolysis (green-hued zones around colonies) or Î³-hemolysis (no zones around colonies) as described <a href="#B2">Argyri et al. (2013)</a>.</p> </div> <div class="minimalinhibitoryconcentration(mic)" title="sec"> <div class="title" tagx="title" title="title">
Minimal Inhibitory Concentration (MIC)</div> <p>Minimal inhibitory concentrations (MICs) of clinically relevant antibiotics were determined by microdilution testing following European Food Safety Authority (EFSA, Panel, 2012) guidance. Susceptibility was tested against: ampicillin (2 Î¼g/ml), gentamicin (16 Î¼g/ml), kanamycin (64 Î¼g/ml), streptomycin (64 Î¼g/ml), erythromycin (1 Î¼g/ml), clindamycin (1 Î¼g/ml), tetracycline (8 Î¼g/ml), and chloramphenicol (4 Î¼g/ml). Microdilution tests were performed in MRS (Oxoid). Cell density was monitored after 24 h incubation at 37Â°C at 595 nm using Plate Reader Infinite 200 pro (MTX Lab Systems, Vienna, Austria).</p> </div> <div class="cytotoxicityassayofprobioticstrainsandtheirspostbiotics" title="sec"> <div class="title" tagx="title" title="title">
Cytotoxicity Assay of Probiotic Strains and Theirs Postbiotics</div> <p>Lactate dehydrogenase (LDH) Cytotoxicity Assay Kit (Thermo Fisher Scientific, Waltham, MA, United States) was used for a measure of cytotoxicity level in the cell cultures through detection of LDH released from dead cells. After treatments of Caco-2 and mesenteric lymph node cells (MLNC) with selected <i>Lactobacillus</i> strains, LDH activity in supernatants was determined by following the manufacturerâ€™s instructions. The absorbance was measured at 450 nm using Plate Reader Infinite 200 pro (MTX Lab Systems).</p> </div> <div class="adherencetocaco-2cells" title="sec"> <div class="title" tagx="title" title="title">
Adherence to Caco-2 Cells</div> <p>The adhesion ability of <i>L. fermentum</i> isolates was performed on colonocyte-like Caco-2 cells. Caco-2 was purchased from the European Collection of Cell Cultures (ECACC No. 86010202). The culture and maintenance of the cells were done according to procedures described by <a href="#B61">SÃ¡nchez et al. (2010)</a> using Dulbeccoâ€™s Modified Eagle Medium (DMEM) supplemented with 10% fetal bovine serum (FBS), 100 U/ml penicillin and 100 mg/ml streptomycin and 2 mM l-glutamine. Media and reagents were purchased from Thermo Fisher Scientific. The adherence (expressed as a percentage) was calculated as: CFU adhered bacteria/CFU added bacteria. Experiments were performed in two replicated plates and in each plate; three wells were used per sample.</p> </div> <div class="isolationandtreatmentofmesentericlymphnodecells" title="sec"> <div class="title" tagx="title" title="title">
Isolation and Treatment of Mesenteric Lymph Node Cells</div> <p>Mesenteric lymph nodes (MLN) were isolated from Wistar rats and the suspension of cells (MLNC) was prepared by mechanical disruption of MLN, as described at <a href="#B3">BajiÄ‡ et al. (2020)</a>. MLNC were stimulated with concanavalin A (ConA, Sigma-Aldrich, 2.5 Î¼g/ml) and treated with cell culture supernatants originating from 2.5 Ã— 10<sup>7</sup> bacterial cells, with UV inactivated bacteria at a ratio of 10:1 (bacteria to eukaryotic cell), 1% of <i>A. ursinum</i> extract after passing through the gastrointestinal tract (GIT), and complex food matrix used as a carrier while passing through the GIT for 72 h. After 72 h of incubation, the cell culture supernatants were collected and subjected to cytotoxicity assay and cytokines quantification.</p> <p>All experimental procedures were approved by the Ethics committee of the Ministry of agriculture, forestry and water management of the Republic of Serbia (license number 119-01-5/14/2017-09).</p> </div> <div class="cytokinesquantification" title="sec"> <div class="title" tagx="title" title="title">
Cytokines Quantification</div> <p>Cytokines concentration in 72 h cell culture supernatants was determined by enzyme-linked immunosorbent assay (ELISA). For interleukin (IL)-10 and interferon (IFN)-Î³ detection, specific DuoSet ELISA kits were used according to the manufacturerâ€™s instructions (R&amp;amp;D Systems, Minneapolis, MN, United States). For IL-17A detection, rat IL-17A (homodimer) ELISA Ready-SET-Go was used according to the manufacturerâ€™s instructions (eBioscience, San Diego, CA, United States). Samples were analyzed in duplicates and the results were calculated using the standard curves.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical Analysis</div> <p>All data are presented as mean values Â± standard error of the mean. One-way ANOVA with the Dunnettâ€™s <i>post hoc</i> test was used to compare multiple groups. Values at <i>p</i> &amp;lt; 0.05 or less were considered statistically significant. All experiments were repeated at least three times. Statistical analysis and graph design were carried out using GraphPad Prism Software.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="totalphenolic,totalflavonoidcontentandantioxidantactivities" title="sec"> <div class="title" tagx="title" title="title">
Total Phenolic, Total Flavonoid Content and Antioxidant Activities</div> <p>Total phenolic content (TPC) of the supernatants obtained before and after digestion were presented in <a href="#T2">Table 2</a>. Bioaccessible fractions of undigested ramson SN1t<sub>0</sub>-SN3t<sub>0</sub> contained 187â€"221 Î¼g GAE/mL and food matrix alone had lower phenolic content of 92 Î¼g GAE/ml, while digested samples showed significantly higher TPC values of 235â€"264 Î¼g GAE/ml for ramson samples and 110 Î¼g GAE/ml for food matrix. In case of all studied samples <i>in vitro</i> digestion provoked a statistically significant increase in TPC. Total flavonoid content of bioaccessible fraction before the digestion ranged from 35.1 to 46.2 Î¼g QE/ml, while after the digestion, a slight increment was recorded in two ramsons samples SN1 and SN2. Bioaccessible fraction of undigested food matrix contained a significant content of flavonoids 15.8 Î¼g QE/ml which was also increased after the digestion as it can be seen from <a href="#T2">Table 2</a>. Antioxidant activity of undigested samples, measured by kinetic DPPH scavenging assay, followed the same trend as TFC, ranging from 0.35 to 0.45 Î¼M TE/ml. The digestion provoked a significant elevation of radical scavenging in SN1 and SN2 samples. Food matrix showed twice lower activity for undigested sample and the increment after digestion was not observed. Ferrous ion chelating capacities before digestion ranged from 42.5%, detected in food matrix, up to 71.3% in SN2 sample. Substantial increase of FCC after digestion was recorded in all samples including food matrix.</p> <div class="table-wrap_UNKNOWN" id="T2" position="float"><span class="label" tagx="label" title="label">TABLE 2</span> <p>Total phenolic, flavonoid content, and antioxidant activities before and after <i>in vitro</i> digestion.</p>  <table frame="hsides" rules="groups" cellspacing="5" cellpadding="5"> <thead> <tr> <td valign="top" align="left" rowspan="1" colspan="1" /> <td valign="top" align="left" rowspan="1" colspan="1">TPC (Î¼g GAE/ml)</td> <td valign="top" align="left" rowspan="1" colspan="1">TFC (Î¼g QE/ml)</td> <td valign="top" align="left" rowspan="1" colspan="1">DPPH scavenging (Î¼M TE/ml)</td> <td valign="top" align="center" rowspan="1" colspan="1">FCC (%)</td> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN1 t<sub>0</sub></td> <td valign="top" align="left" rowspan="1" colspan="1">221 Â± 2</td> <td valign="top" align="left" rowspan="1" colspan="1">46.2 Â± 1.6</td> <td valign="top" align="left" rowspan="1" colspan="1">0.45 Â± 0.05</td> <td valign="top" align="center" rowspan="1" colspan="1">68.2 Â± 0.9</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN1 â€" after digestion</td> <td valign="top" align="left" rowspan="1" colspan="1">264 Â± 1*</td> <td valign="top" align="left" rowspan="1" colspan="1">55.4 Â± 1.1*</td> <td valign="top" align="left" rowspan="1" colspan="1">0.59 Â± 0.07*</td> <td valign="top" align="center" rowspan="1" colspan="1">85.0 Â± 1.6*</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN2 t<sub>0</sub></td> <td valign="top" align="left" rowspan="1" colspan="1">205 Â± 3</td> <td valign="top" align="left" rowspan="1" colspan="1">35.1 Â± 0.8</td> <td valign="top" align="left" rowspan="1" colspan="1">0.40 Â± 0.06</td> <td valign="top" align="center" rowspan="1" colspan="1">71.3 Â± 1.9</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN2 â€" after digestion</td> <td valign="top" align="left" rowspan="1" colspan="1">235 Â± 2*</td> <td valign="top" align="left" rowspan="1" colspan="1">43.3 Â± 0.5*</td> <td valign="top" align="left" rowspan="1" colspan="1">0.56 Â± 0.05*</td> <td valign="top" align="center" rowspan="1" colspan="1">88.1 Â± 0.8*</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN3 t<sub>0</sub></td> <td valign="top" align="left" rowspan="1" colspan="1">187 Â± 4</td> <td valign="top" align="left" rowspan="1" colspan="1">38.0 Â± 1.6</td> <td valign="top" align="left" rowspan="1" colspan="1">0.35 Â± 0.07</td> <td valign="top" align="center" rowspan="1" colspan="1">66.5 Â± 1.4</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN3 â€" after digestion</td> <td valign="top" align="left" rowspan="1" colspan="1">239 Â± 2*</td> <td valign="top" align="left" rowspan="1" colspan="1">40.4 Â± 1.4</td> <td valign="top" align="left" rowspan="1" colspan="1">0.40 Â± 0.06</td> <td valign="top" align="center" rowspan="1" colspan="1">86.6 Â± 1.6*</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SNM t<sub>0</sub></td> <td valign="top" align="left" rowspan="1" colspan="1">92 Â± 2</td> <td valign="top" align="left" rowspan="1" colspan="1">15.8 Â± 0.9</td> <td valign="top" align="left" rowspan="1" colspan="1">0.19 Â± 0.05</td> <td valign="top" align="center" rowspan="1" colspan="1">42.5 Â± 2.1</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SNM â€" after digestion</td> <td valign="top" align="left" rowspan="1" colspan="1">110 Â± 1*</td> <td valign="top" align="left" rowspan="1" colspan="1">22.7 Â± 1.1*</td> <td valign="top" align="left" rowspan="1" colspan="1">0.24 Â± 0.05</td> <td valign="top" align="center" rowspan="1" colspan="1">76.8 Â± 1.8*</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"><span class="attrib" tagx="attrib" title="attrib"><i>SN, supernatants of <i>Allium ursinum</i> extracts obtained before and after digestion; SNM, solely food matrix; TPC, Total phenolic content; TFC, Total flavonoid content; DPPH, 2,2-Diphenyl-1-Picrylhydrazyl; FCC, ferrous ion chelating capacity; GAE, gallic acid equivalent; QE, quercetin equivalent; TE, trolox equivalent. Values are shown as means Â± SD (<i>n</i> = 3); statistical significance *<i>p</i> &amp;lt; 0.05 (Studentâ€™s <i>t</i>-test).</i></span></div> </div> </div> <div class="gcÃ—gc-msanalysis" title="sec"> <div class="title" tagx="title" title="title">
GC Ã— GC-MS Analysis</div> <p>Two-dimensional gas chromatography analysis of <i>A. ursinum</i> bioaccessible fraction obtained after digestion revealed presence of seven volatile compounds which were absent in supernatant of digested food matrix (<a href="#F1">Figures 1A,B</a>). The dominant compounds in the analyzed spectrum of samples containing <i>A. ursinum</i> were allicin decomposition products: 3,4-dimethylthiophene; 1,3-dithiane; diallyl disulfide; allyl methyl disulfide; 2-vinyl-1,3-dithiane; (E)-1-propenyl methyl disulfide; (E)-1-propenyl allyl disulfide (<a href="#F1">Figure 1B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Comprehensive two-dimensional gas chromatography/mass spectrometry (GC Ã— GC-MS). Figures depicting chromatograms for: <b>(A)</b> Supernatant of the digested food matrix; <b>(B)</b>. Supernatant of the digested sample SN1</p>   </div> </div> <div class="theeffectsoftheextractsoncellcyclephasedistribution" title="sec"> <div class="title" tagx="title" title="title">
The Effects of the Extracts on Cell Cycle Phase Distribution</div> <p>Investigation of the mechanisms of the cytotoxic activity of the <i>A. ursinum</i> extracts by cell cycle analysis revealed that all tested extracts applied at a concentration of 5% induced remarkable increase of HeLa cells in the subG1 phase in comparison with those determined in control, untreated cell sample after treatments that lasted 24, 48, and 72 h. The effects of the extracts on cell cycle phase distribution are presented in <a href="#F2">Figure 2A</a>. The differences between the percentages of HeLa cells at specific phases of the cell cycle exposed to undigested and digested <i>A. ursinum</i> extracts were observed. Undigested extracts <b>1</b> and <b>2</b> after 24 h incubation caused accumulation of HeLa cells within the G2/M cell cycle phase in comparison with control cells. The increase of cells in the S phase was observed in HeLa cells treated for 48 and 72 h with undigested extracts. Among undigested extracts, the extract 1 induced the most intensive accumulation of cells within the S phase, which was accompanied with G2/M phase arrest after 48 h treatment. The treatment with food matrix of HeLa cells did not significantly affect their cell cycle phase distribution (<a href="#F2">Figure 2A</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Changes in the cell cycle phase distribution of human cervical adenocarcinoma (HeLa) cells induced by the <i>Allium ursinum</i> extracts (<i>c</i> = 5%) after 24 h, 48 h, and 72 h of treatment <b>(A)</b>. Changes in the cell cycle phase distribution of human colon adenocarcinoma (LS174) cells induced by the <i>Allium ursinum</i> extracts (<i>c</i> = 20%) after 24 h, 48 h, and 72 h of treatment. <b>(B)</b> Effects of specific caspase-3 inhibitor on the percentages of subG1 HeLa cells exposed to <i>Allium ursinum</i> extracts for 24 h without caspase-3 inhibitor and pretreated with caspase-3 inhibitor before treatment with extracts <b>(C)</b> C-control HeLa or LS174 cells, extract 1 control for digestion, extract 1â€² â€" digested, extract 2 control for digestion, extract 2â€² â€" digested, extract 3 control for digestion, extract 3â€² â€" digested.</p>   </div> <p>In human colon adenocarcinoma LS174 cells exposure for 24, 48 and, 72 h to undigested and digested <i>A. ursinum</i> extracts applied at a concentration of 20% led to remarkable increase in the percentage of subG1 cells when compared with control cell sample, as shown in <a href="#F2">Figure 2B</a>. The increase of LS174 cells within the subG1 phase was remarkably higher in cells incubated with 20% undigested extracts. In addition, all three digested extracts caused G2/M cell cycle arrest in LS174 cells after 24, 48 and, 72 h of incubation. Accumulation of LS174 cells within the G2/M phase was the most prominent in cell samples exposed for 24 h to tested extracts. It should be noted that the food matrix at a concentration of 20% slightly increased the percentage of LS174 cells within the subG1 phase.</p> <p>Cytotoxic activity of <i>A. ursinum</i> tested extracts digested <i>in vitro</i> on HeLa and LS174 cells is presented in <a href="#T3">Table 3</a>.</p> <div class="table-wrap_UNKNOWN" id="T3" position="float"><span class="label" tagx="label" title="label">TABLE 3</span> <p>Cytotoxic activity of <i>Allium ursinum</i> extracts digested <i>in vitro.</i></p>  <table frame="hsides" rules="groups" cellspacing="5" cellpadding="5"> <thead> <tr> <td valign="top" align="left" rowspan="1" colspan="1" /> <td valign="top" align="center" rowspan="1" colspan="1">HeLa</td> <td valign="top" align="center" rowspan="1" colspan="1">LS174</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1" /> <td valign="top" align="left" colspan="2" rowspan="1"> <hr /> </td> </tr> <tr> <td rowspan="1" colspan="1" /> <td valign="top" align="center" colspan="2" rowspan="1"><i>IC<sub>50</sub> (mean</i> Â± <i>standard deviation)</i></td> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN1 t<sub>0</sub></td> <td valign="top" align="center" rowspan="1" colspan="1">2.75 Â± 0.20</td> <td valign="top" align="center" rowspan="1" colspan="1">20</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN1</td> <td valign="top" align="center" rowspan="1" colspan="1">4.02 Â± 0.13</td> <td valign="top" align="center" rowspan="1" colspan="1">&amp;gt; 20</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN2 t<sub>0</sub></td> <td valign="top" align="center" rowspan="1" colspan="1">3.18 Â± 0.26</td> <td valign="top" align="center" rowspan="1" colspan="1">20</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN2</td> <td valign="top" align="center" rowspan="1" colspan="1">4.38 Â± 0.21</td> <td valign="top" align="center" rowspan="1" colspan="1">&amp;gt; 20</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN3 t<sub>0</sub></td> <td valign="top" align="center" rowspan="1" colspan="1">4.15 Â± 0.02</td> <td valign="top" align="center" rowspan="1" colspan="1">19.69 Â± 0.44</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">SN3</td> <td valign="top" align="center" rowspan="1" colspan="1">4.90 Â± 0.09</td> <td valign="top" align="center" rowspan="1" colspan="1">&amp;gt; 20</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">cisplatin</td> <td valign="top" align="center" rowspan="1" colspan="1">2.46 Â± 0.08</td> <td valign="top" align="center" rowspan="1" colspan="1">13.99 Â± 2.00</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"><span class="attrib" tagx="attrib" title="attrib"><i>IC<sub>50</sub> values represent mean Â± standard deviation of three independent experiments. Values for extracts are shown in percentages, values for cisplatin are shown in Î¼M. SNt<sub>0</sub> â€" supernatants of <i>A. ursinum</i> extracts obtained before digestion, SN â€" after digestion; HeLa-human cervical adenocarcinoma cell lines; LS174-human colon adenocarcinoma cell lines.</i></span></div> </div> </div> <div class="effectsofthecaspase-3inhibitor" title="sec"> <div class="title" tagx="title" title="title">
Effects of the Caspase-3 Inhibitor</div> <p>To explore whether the observed increase of HeLa cells within the subG1 phase after 24 h exposure to undigested and digested <i>A. ursinum</i> extracts could be attributed to apoptotic cell death, the cell cycle analysis of cells pretreated with a specific inhibitor of caspase-3 was performed. The decrease in the percentage of subG1 HeLa cells pretreated with caspase-3 inhibitor before exposure to each of the tested extracts was found when compared with treated cells which were not preincubated with an inhibitor (<a href="#F2">Figure 2C</a>). These results point to the ability of both undigested and digested extracts of <i>A. ursinum</i> to induce apoptosis in HeLa cells through activation of main effector caspase-3.</p> </div> <div class="evaluationofa.ursinum-associatedmicrobiota" title="sec"> <div class="title" tagx="title" title="title">
Evaluation of <i>A. ursinum-</i>Associated Microbiota</div> <p>In order to determine the composition of LAB and closely related bacteria associated with <i>A. ursinum</i>, DGGE analysis of 16S rRNA amplicons obtained by using lactobacilli-specific primers Lab-0159f and Uni-0515-GCr was performed. DGGE analysis of the dominant microbiota present in the first dilution of <i>A. ursinum</i> leaves revealed the predominant presence of genera: <i>Streptococcus</i>, corresponded to the bands 5, 6, 7, 8, and 9 (69.2%), <i>Lactococcus</i>, corresponded to the bands 2, 3, and 4 (23.1%), and <i>Lactobacillus</i>, corresponded to the bands 11 and 13 (15.3%). Sequencing results of the re-amplified bands from DGGE gel indicated the domination of <i>Lactococcus lactis</i>, <i>Streptococcus thermophilus</i>, <i>Lactobacillus reuteri</i>, and <i>Lactobacillus gasseri</i> (<a href="#F3">Figure 3</a> and <a href="#TS1">Supplementary Table 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 3</span></a></div>   <p>Denaturing gradient gel electrophoresis (DGGE) profiles of rDNA amplicons obtained using a <i>Lactobacillus</i>-specific primer set on bacterial DNA isolated from <i>Allium ursinum</i> leaf tissue samples. Bands indicated by numbers (1â€"16) were excised, cloned, and sequenced.</p>   </div> </div> <div class="isolationandidentificationoflabfroma.ursinum" title="sec"> <div class="title" tagx="title" title="title">
Isolation and Identification of LAB From <i>A. ursinum</i></div> <p>LAB and other closely related bacteria were isolated from the first dilution of <i>A. ursinum</i> leaves. The 50 isolates selected on the basis of their growth on MRS, GM17, and Nutri agar were further analyzed by 16S rDNA sequencing. The most abundant genus belonged to <i>Staphylococcus</i> (34%), <i>Lactobacillus</i> all of them <i>L. fermentum</i> (30%), and <i>Bacillus</i> (14%) (<a href="#TS2">Supplementary Table 2</a>).</p> <p>In order to determine inter-strain differences among the <i>L. fermentum</i> isolates, 15 <i>L. fermentum</i> strains were analyzed by PFGE. The dendrogram produced by SPSS software demonstrated genomic similarity ranging from 75 to 100% (<a href="#F4">Figure 4</a>). Two clusters with differences up to 25% were observed. Cluster I comprise one pulsotype (14 isolates); while cluster II contains only one pulsotype with one isolate BGSR163. The band pattern revealed the presence of two different <i>L. fermentum</i> PFGE puslotypes (strains) (<a href="#F4">Figure 4</a>), hence the strains BGSR163 and BGSR227, belonging to different PFGE pulsotypes, were chosen for cytotoxicity assay, as well as for evaluation of probiotic potential â€"survival through simulated GIT, adhesion to intestinal epithelial cells (IEC), and immunomodulatory activity, as the main criteria for selection of probiotic strains.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 4</span></a></div>   <p>Dendrogram derived from <i>Not</i>I PFGE patterns showing the relatedness of <i>Lactobacillus fermentum</i> species isolated from <i>Allium ursinum</i>. The dendrogram was constructed using SPSS software. Letters I and II indicate two major clusters.</p>   </div> </div> <div class="determinationofthesafetystatusofthel.fermentumbgsr163andl.fermentumbgsr227isolates" title="sec"> <div class="title" tagx="title" title="title">
Determination of the Safety Status of the <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 Isolates</div> <p>In order to determine the safety status of the <i>L. fermentum</i> isolates, the cytotoxic, gelatinase, and hemolytic activities were assayed, and the susceptibility to minimal inhibitory concentrations of antibiotics was determined. The Caco-2 cells were exposed to live <i>L. fermentum</i> isolates to determine the possible cytotoxic effect of the strains. The results showed that <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 live bacterial strains exhibited cytotoxic activity to Caco-2 cells, while the cell culture supernatants from both strains did not exhibit cytotoxic activity on the same cells (<a href="#F5">Figure 5A</a>). MTT assay showed that cell culture supernatants from <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 do not decrease cell metabolic activity of Caco-2 cells after the treatment (<a href="#F5">Figure 5B</a>). Absence of gelatinase activity, for both <i>L. fermentum</i> strains (BGSR162 and BGSR227), on gelatin agar plates was noted, as well as absence of hemolysis zones, on blood agar plates. Both strains were susceptible to all clinically relevant antibiotics (ampicillin, gentamicin, kanamycin, streptomycin, erythromycin, clindamycin, tetracycline, and chloramphenicol).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 5</span></a></div>   <p>The effect of <i>Lactobacillus fermentum</i> BGSR163 and BGSR227 live strains and cell culture supernatants on toxicity, assessed by LDH assay <b>(A)</b> and the effect of <i>L. fermentum</i> BGSR163 and BGSR227 cell culture supernatant on metabolic activity of Caco-2 cells <b>(B)</b>. All values are presented as mean Â± SD from three independent experiments. DMEM â€" Dulbeccoâ€™s Modified Eagleâ€™s Medium.</p>   </div> </div> <div class="survivalofl.fermentumbgsr163andl.fermentumbgsr227inchemicallysimulatedgittransit" title="sec"> <div class="title" tagx="title" title="title">
Survival of <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 in Chemically Simulated GIT Transit</div> <p>The survival of the strains <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 in chemically simulated GIT transit was tested. The results revealed that both strains successfully survived the passage through simulated GIT conditions (<a href="#F6">Figure 6A</a>). The results showed that the number of viable cells of both <i>L. fermentum</i> strains after 10 min (simulated mouth conditions) was not significantly changed compared to that at the beginning of the experiment. However, 110 min exposure to simulated gastric conditions lowered the number of viable cells of both <i>L. fermentum</i> strains for approx.1 log unit (Î"LogBGSR163 = 0.8 and BGSR227 = 1), although the number of viable cells was rather high (8 Log CFU/ml). Interestingly, when the cells were exposed to the simulated small intestine conditions (100 min) with increased bile salts concentrations, the growth of the strains was restored (Î"LogBGSR163 = 0.7 and Î"LogBGSR227 = 0.8). The survival rate of the strain BGSR163 was 98.85%; Î"log CFU/ml = 0.1), while the survival rate of BGSR227 was 97.73%; Î"log CFU/ml = 0.2 (<a href="#F6">Figure 6A</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 6</span></a></div>   <p>Survival of selected <i>Lactobacillus fermentum</i> BGSR163 and BGSR227 isolates, as indicated on the right, under oral conditions (0â€"10 min) gastric conditions (10â€"120 min), and intestinal conditions (120â€"220 min). All values are presented as mean Â± SD from three independent experiments <b>(A)</b>. Adhesion of <i>Lactobacillus fermentum</i> BGSR163 and BGSR227 on Caco-2 cells. All values are presented as mean Â± SD from three independent experiments. <b>(B)</b>.</p>   </div> </div> <div class="adherencetocaco-2cells" title="sec"> <div class="title" tagx="title" title="title">
Adherence to Caco-2 Cells</div> <p>The adherence abilities of the <i>L. fermentum</i> isolates BGSR163 and <i>L. fermentum</i> BGSR227 were investigated on Caco-2 cells. Both strains were able to adhere to Caco-2 cells, although in different extent (<a href="#F6">Figure 6B</a>). The strain <i>L. fermentum</i> BGSR163 exhibited higher adherence ability (13.73%), comparing to the strain <i>L. fermentum</i> BGSR227 (4.45%) (<a href="#F6">Figure 6B</a>).</p> </div> <div class="immunomodulatoryactivity" title="sec"> <div class="title" tagx="title" title="title">
Immunomodulatory Activity</div> <p>In order to evaluate the immunomodulatory effect of potential probiotic bacteria (UV inactivated bacteria and cell culture supernatant) as well as <i>A. ursinum</i> extract, the production of proinflammatory cytokines IL-17 and IFN-Î³, and immunoregulatory cytokine IL-10 of MLNC stimulated with a ConA was tested. Importantly, none of the used treatments exhibited cytotoxic effects (data not shown). Treatments of MLNC with cell-free culture supernatants and UV inactivated bacteria <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 significantly stimulated the production of IL-17 (p &amp;lt; 0.01 and 0.001 respectively) (<a href="#F7">Figure 7A</a>), while the treatments of MLNC with cell-free culture supernatants of <i>L. fermentum</i> BGSR163 and UV inactivated bacteria of both strains led to the decreased production of IFN-Î³ (<i>p</i> &amp;lt; 0.01 and 0.001 respectively) (<a href="#F7">Figure 7B</a>). <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 cell culture supernatants as well as their UV inactivated bacteria statistically significantly stimulated the production of IL-10 in MLNC (<i>p</i> &amp;lt; 0.001) (<a href="#F7">Figure 7C</a>). Plant extract treatment and treatment with complex food matrix did not exert effects on cytokine production (<a href="#F7">Figures 7Aâ€"C</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 7</span></a></div>   <p>Effect of cell culture supernatant (spn) and UV inactivated <i>Lactobacillus. fermentum</i> strains BGSR163 and BGSR227, as well as <i>A. ursinum</i> and complex food matrix (commercial infant formula) after passing through the gastrointestinal tract on the production of IL-17 <b>(A)</b>, IFN-Î³ <b>(B)</b>, and IL-10 <b>(C)</b> on ConA stimulated MLNC after the 72 h treatment. All values are presented as mean Â± SD from three independent experiments. One-way ANOVA with the Dunnettâ€™s <i>post hoc</i> test was used to compare groups with control. The statistical significance of cytokine production is shown (**<i>p</i> &amp;lt; 0.01, ***<i>p</i> &amp;lt; 0.001).</p>   </div> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Ramson â€"<i>Allium ursinum</i> has been used as a medicinal plant for centuries. While its antimicrobial, cytotoxic, and antioxidant activity were previously shown, the ramson-associated microbiota was not characterized so far. The current study represents the first attempt to elucidate the effect of simulated digestion of ramson in the presence of complex food matrix on the bioaccessibility of phenolic and organosulfur compounds as well as their <i>in vitro</i> biological activities. Hence, this paper presents the characterization of antiproliferative and antioxidant activities of ramson, together with the health-promoting properties of its associated LAB. Namely, the number of studies in the last decade identified the complex plant-associated microbial communities, so-called plant microbiota (<a href="#B13">Compant et al., 2019</a>). Moreover, the beneficial effects of the plant microbiota on the plant growth and health was demonstrated (<a href="#B13">Compant et al., 2019</a>). However, the comprehensive analyses of the plant extracts together with the health-promoting effects of the plant microbiota are rare. Particularly the data related to the probiotic effects of the <i>A. ursinum-</i>associated microbiota are lacking.</p> <div class="totalphenolic,flavonoidcontent,andantioxidantactivities" title="sec"> <div class="title" tagx="title" title="title">
Total Phenolic, Flavonoid Content, and Antioxidant Activities</div> <p>Available literature data on TPC and TFC in <i>Allium ursinum</i> are very diverse and heterogeneous which is mainly caused by different procedures for extract preparation and chemical analysis. In previous study (<a href="#B22">Gitin et al., 2012</a>) reported TPC values of 27.9 g GAE/100 g (dry basis) in leaf extract obtained by 12-day maceration with 70% ethanol, while <a href="#B17">Djurdjevic et al. (2004)</a> determined 3.24 mg/g of total free phenolics in dry leaves extracted with 80% (v/v) boiling aqueous methanol, followed by boiling ethyl acetate. TFC content of fresh leaves was also strongly dependent on extraction method. According to <a href="#B22">Gitin et al. (2012)</a> flavonoid content determined using ultrasound assisted extraction was 7.3 mg Quercetin/kg fresh leaves compared to 2.7 mg QE/kg using conventional maceration. In another recent study (<a href="#B51">PavloviÄ‡ et al., 2017</a>), the authors applied five different extraction techniques followed by determination of TPC and TFC in dried extracts of <i>A. sativum</i> leaves. According to obtained data TPC varied from 24.32 to 43.97 mg of GAE/g of extract and values for TFC were from 7.46 to 38.56 mg of Rutin/g of extract. Several previous studies (<a href="#B44">Nakano et al., 1981</a>; <a href="#B9">Carotenuto et al., 1996</a>; <a href="#B72">Wu et al., 2009</a>) have shown that main polyphenols present in ramsons leaves are most certainly flavonoids, among them kaempferol derivatives were found to be predominant (<a href="#B48">OszmiaÅ"ski et al., 2013</a>). To the best of our knowledge not a single one of mentioned studies have not considered the bioaccessibility of ramsons polyphenol compounds. In current study total phenolic and flavonoid content have been examined in bioaccessible fraction obtained after <i>in vitro</i> digestion of fresh ramsons leaves in presence of complex food matrix. The aim of this experimental design was to mimic feeding conditions as closely as possible. Obtained results showed that the concentrations of total polyphenols and total flavonoids in bioaccessible fraction were far lower than those detected in fresh and dry leaves as well as their concentrated extracts. Determined content of TPC after the digestion in presence of food matrix was only 235â€"264 Î¼g GA/mL, from which 110 Î¼g GA/mL was present in bioacessible fraction of food matrix digested without <i>A. ursinum</i>. Similar results were obtained for TFC 40.4â€"55.4 Î¼g QE/mL in samples containing <i>A. ursinum</i> and 22.7 Î¼g QE/mL, in food matrix alone. Observed low yields were expected due to mild extraction conditions in aqueous medium of the mimicked digestive tract compared to organic solvent extraction applied in previous studies. However, it was not the only limiting factor. Various types of interactions between food constituents (proteins, carbohydrates, lipids or fibers) and polyphenols affecting their bioaccessibility have been documented previously (<a href="#B68">StanisavljeviÄ‡ et al., 2015</a>; <a href="#B53">PeÅ¡iÄ‡ et al., 2019</a>). When compared TPC and TFC values before (SN1t<sub>0</sub>â€"SN3t<sub>0</sub>) and after the digestion (SN1t<sub>0</sub>â€"SN3t<sub>0</sub>) a mild increase after completed digestion was observed in majority of samples containing <i>A. ursinum.</i> Similarly, in samples containing solely food matrix (SNM) increased TPC and TFC values have also been recorded upon digestion. Thus, it is not clear which portion of that increment might be attributed to release of phenolic compounds from <i>A. ursinum</i> during the digestion. Results obtained for TPC should be interpreted with the great caution especially in samples containing complex food matrices in aqueous media. It has been determined previously that Folin-Ciocalteuâ€™s reagent is also showing reactivity toward reducing sugars, ascorbate, certain organic and fatty acids, aromatic amines and peptides which could be released during the digestion of food constituents (<a href="#B55">Prior et al., 2005</a>). Total flavonoid content of food matrix was noticeably high, although approximately three times lower than in samples containing ramsons. This might be explained by the shortcomings of aluminum chloride based assay, which is dependent on the type of flavonoid present in the sample and are specific only for a limited number of flavonoids while it can react with phenolic acids giving non-specific absorbance at 510 nm (<a href="#B52">PÃªkal and Pyrzynska, 2014</a>). It has been shown previously that food matrix used in our study contains significant amount of phenolic acids and several flavonoids (<a href="#B53">PeÅ¡iÄ‡ et al., 2019</a>). Previous studies showed that composition of food matrix and the effect of co-digestion of polyphenols may affect their antioxidant activity as well. As it can be observed from <a href="#T4">Table 4</a>, radical scavenging activity measured by DPPH assay showed strong positive correlation with TPC (<i>r</i> = 0.93) and TFC (<i>r</i> = 0.94), while correlation of these parameters with ferrous ion chelating capacity (FCC) was far lower, <i>r</i> = 0.73 with TPC and <i>r</i> = 0.68 with TFC. DPPH scavenging activity was significantly elevated after the digestion in case of two samples SN1 an SN2 while remained unchanged for samples SN3 and SNM. Values of FCC were also increased upon completed digestion in all studied samples with the most prominent increment in food matrix digested alone (SNM). Considering these results, it can be concluded that bioaccessible constituents of <i>A. ursinum</i> did not substantially contribute to FCC, which in majority originates from food matrix and components of the digestive fluids. It can also be observed that the increase in FCC after digestion was higher in the food matrix alone than in combination with ramsons, indicating that some ramsons components interfere with the nutrient matrix components, resulting in a moderate increase in FCC. This situation has already been documented in the case of the combined digestion of polyphenols from grapes and the same type of nutrient matrix (<a href="#B53">PeÅ¡iÄ‡ et al., 2019</a>). Taken altogether the quantities of TPC and TFC in bioaccessible fraction of ramson, digested in presence of food matrix were moderate, compared to one recorded previously in dry or fresh leaves and their extracts. This was expected, having in mind mild extraction conditions of simulated digestive tract and interactions of phenolic compounds with food constituents. Nevertheless, determined values of TPC and TFC were approximately two times higher in samples containing ramson than in food matrix digested alone. Results of DPPH assay were positively and strongly correlated with TPC and TFC, while the contribution of phenolic compounds to FCC was less pronounced.</p> <div class="table-wrap_UNKNOWN" id="T4" position="float"><span class="label" tagx="label" title="label">TABLE 4</span> <p>Correlations between antioxidant parameters and total phenolic and flavonoid contents.</p>  <table frame="hsides" rules="groups" cellspacing="5" cellpadding="5"> <thead> <tr> <td valign="top" align="left" rowspan="1" colspan="1" /> <td valign="top" align="center" rowspan="1" colspan="1">TPC</td> <td valign="top" align="center" rowspan="1" colspan="1">TFC</td> <td valign="top" align="center" rowspan="1" colspan="1">DPPH</td> <td valign="top" align="center" rowspan="1" colspan="1">FCC</td> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1">TPC</td> <td valign="top" align="center" rowspan="1" colspan="1">1.0</td> <td valign="top" align="center" rowspan="1" colspan="1">0.96</td> <td valign="top" align="center" rowspan="1" colspan="1">0.93</td> <td valign="top" align="center" rowspan="1" colspan="1">0.73</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">TFC</td> <td rowspan="1" colspan="1" /> <td valign="top" align="center" rowspan="1" colspan="1">1.0</td> <td valign="top" align="center" rowspan="1" colspan="1">0.94</td> <td valign="top" align="center" rowspan="1" colspan="1">0.68</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">DPPH</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td valign="top" align="center" rowspan="1" colspan="1">1.0</td> <td valign="top" align="center" rowspan="1" colspan="1">0.72</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">FCC</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td valign="top" align="center" rowspan="1" colspan="1">1.0</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"><span class="attrib" tagx="attrib" title="attrib"><i>TPC, Total phenolic content; TFC, Total flavonoid content; DPPH, 2,2-Diphenyl-1-Picrylhydrazyl; FCC, ferrous ion chelating capacity. Numbers represent <i>R</i>-square values. All shown correlations are significant at <i>p</i> &amp;lt; 0.05 level.</i></span></div> </div> </div> <div class="qualitativeanalysisofvolatileorganosulfurcompounds" title="sec"> <div class="title" tagx="title" title="title">
Qualitative Analysis of Volatile Organosulfur Compounds</div> <p>According to the literature data dominant organosulfur compounds (cysteine sulfoxides) of <i>A. ursinum</i> are methiin and alliin, but it also contains certain amounts of isoalliin, propiin and ethiinas well (<a href="#B66">Sobolewska et al., 2015</a>). Thiosulfinates are formed from cysteine sulfoxides upon tissue mechanical injury by the hydrolyticaction of the alliinase (C,S-lyase). The optimal conditions for hydrolytic cleavage of sulfoxides are pH = 6 and temperature of 35Â°C. The primary products of this reaction are thiosulfinates, pyruvic acid, and ammonia. The most abundant (75â€"90%) thiosulfinates detected in various ramson extracts were: allicin (diallylthiosulfinate = di-2-propenyl thiosulfinate) and methyl-allyl- or dimethyl thiosulfinates (<a href="#B65">Sendl and Wagner, 1991</a>). Depending on the <i>Allium</i> species and specific conditions, thiosulfinates decomposition could yield a variety of constituents such as methyl allyl, diallyl diethyl mono-, di-, tri-, tetra-, penta-, and hexasulfides, vinyldithiins, as well as (E)- and (Z)-ajoene (<a href="#B4">Benkeblia and Lanzotti, 2007</a>). The most abundant thiosulfinate detected previously in ramson leaf extracts are represented by: allyl methyl disulfide, diallyldisulfide, allyl methyl trisulfide, allyl (E)-1-propenyl disulfide, dimethyl trisulfide, methyl (E)-1-propenyl disulfide, allyl propyl disulfide and allyl (Z)-1-propenyl disulfide (<a href="#B66">Sobolewska et al., 2015</a>; <a href="#B51">PavloviÄ‡ et al., 2017</a>).</p> <p>Our analysis determined presence of seven volatile compounds including: 3,4-dimethylthiophene; 1,3-dithiane; diallyl disulfide; allyl methyl disulfide; 2-vinyl-1,3-dithiane; (E)-1-propenyl methyl disulfide; (E)-1-propenyl allyl disulfide. At least to the best of our knowledge, there are no available literature data on of <i>A. ursinum</i> organosulfur compounds in bioaccessible fraction obtained after <i>in vitro</i> digestion. However, similar studies have been conducted on several other <i>Allium</i> species. It has been shown previously that hydrolytic action of alliinase can be preserved in stomach especially in presence moderate to high protein meal which can transiently raise pH to 4.4 or higher, which is sufficient to enable catalysis and produce allyl-thiosulfinates (<a href="#B36">Lawson and Hunsaker, 2018</a>). Diallyl disulfide detected in our study belongs to allyl polysulfides spontaneous transformation products of allyl-thiosulfinates (<a href="#B36">Lawson and Hunsaker, 2018</a>). The presence of 3,4-dimethylthiophene was previously detected in <i>Allium sativum</i> essential oil from Clevenger distillation as one of major sulfur-containing compounds (<a href="#B62">Satyal et al., 2017</a>) as well as in <i>A. ursinum</i> leaf extract as a minor compound (<a href="#B26">Ivanova et al., 2009</a>). According to literature data 2-vinyl-1,3-dithiane is being formed during thermal degradation of aliin on temperatures above 180Â°C (<a href="#B34">Kubec et al., 1997</a>) and it was also found in fried and baked garlic (<a href="#B75">Yu et al., 1994</a>), but there is also evidence that it could be formed at 40Â°C (<a href="#B30">KeleÅŸ et al., 2014</a>). This compound was recently detected in fresh leaf extracts of <i>A. ursinum</i> (<a href="#B26">Ivanova et al., 2009</a>). Cyclic organosulfur compound reported in our study-1,3-dithiane has been characterized previously as a minor constituent of garlic with the ability to stimulate production of reactive oxygen species in neutrophils (<a href="#B63">Schepetkin et al., 2019</a>). It was also detected in <i>Allium roseum</i>, (<a href="#B77">Zouari et al., 2012</a>) but this is the first report in <i>Allium ursinum</i>. Allyl methyl disulfide, (E)-1-propenyl methyl disulfide, (E)-1-propenyl allyl disulfide presence was reported previously in <i>A. sativum</i> and <i>A. vienale</i> essential oil (Prabodh <a href="#B62">Satyal et al., 2017</a>), as well in <i>A. ursinum</i> oil and leaf extracts (<a href="#B64">Schmitt et al., 2005</a>; <a href="#B23">GoÄ‘evac et al., 2008</a>; <a href="#B66">Sobolewska et al., 2015</a>).</p> </div> <div class="anticancerproperties" title="sec"> <div class="title" tagx="title" title="title">
Anticancer Properties</div> <p>The results of investigation of the anticancer properties of the <i>Allium ursinum</i> extracts are in accordance with data of <a href="#B73">Xu et al. (2013)</a> who demonstrated that watery extracts of <i>A. ursinum</i> caused apoptosis through activation of caspase-3, caspase-8 and caspase-9, and G2/M arrest in human gastric adenocarcinoma AGS cells. Our study is the first to report the influence of <i>in vitro</i> gastrointestinal digestion on the cytotoxic activity of extracts obtained from <i>A. ursinum</i> against human malignant cell lines. The extracts prepared from medicinal plant <i>A. ursinum</i> were reported to exert various pharmacological activities, such as antioxidant, antihypertensive, antidiabetic, antimicrobial, antifungal and anti-inflammatory activities in addition to anticancer activity (<a href="#B73">Xu et al., 2013</a>; <a href="#B71">Trio et al., 2014</a>; <a href="#B66">Sobolewska et al., 2015</a>, and references cited therein). Use in European traditional medicine and diverse biological activities exhibited by <i>A. ursinum</i> may be attributed to various sulfur-containing compounds, including thiosulfinates which are the most abundant chemical constituents of <i>A. ursinum</i> and phenolic compounds (<a href="#B73">Xu et al., 2013</a>; <a href="#B66">Sobolewska et al., 2015</a>). Only seven sulfur compounds, which represent the degradation products of thiosulfinates, were detected by 2D gas chromatography analysis in the examined extracts. The allyl sulfur compounds, including diallyl disulfide identified in the tested extracts, have been shown to inhibit proliferation of malignant cells by inducing accumulation within G2/M phase as well as to induce apoptosis (<a href="#B31">Knowles and Milner, 2001</a>; <a href="#B24">Herman-Antosiewicz and Singh, 2004</a>; <a href="#B49">Park et al., 2007</a>; <a href="#B74">Yi and Su, 2013</a>). In addition, it was demonstrated that diallyl disulfide triggered apoptosis in cancer cells through activation of caspase-3 and mitochondrial signaling pathway (<a href="#B42">Nagaraj et al., 2010</a>; <a href="#B74">Yi and Su, 2013</a>).</p> <p>Results of our <i>in vitro</i> study suggest that <i>A. ursinum</i> consumption might have health promoting properties, including anticancer effects. The extensive <i>in vivo</i> research is required to fully evaluate the possible cancer preventive efficacy of <i>A. ursinum</i> and its bioactive compounds in addition to further bioavailability and pharmacokinetic studies.</p> </div> <div class="beneficialeffectsofthea.ursinumassociatedmicrobiota" title="sec"> <div class="title" tagx="title" title="title">
Beneficial Effects of the <i>A. ursinum</i> Associated Microbiota</div> <p>Recent literature data revealed that many of the phytotherapeutic molecules originate from plant-associated microbiota, although it is still highly unexplored area (<a href="#B33">KÃ¶berl et al., 2013</a>). In particular, the microbiota of medicinal plants is highly specific, depending on the unique bioactive secondary metabolites produced by medicinal plants (<a href="#B57">Qi et al., 2012</a>). Plants are acknowledged as an important source of LAB, although not systematically investigated. Recently, <a href="#B32">KÃ¶berl et al. (2019)</a> identified the leaf surfaces and leaf endosphere of <i>Matricaria chamomilla</i> L. and <i>Calendula officinalis</i> L. as a source of LAB, with <i>Enterococcaceae</i> dominantly present in both plants, while <i>Lactobacillaceae</i> and <i>Carnobacteriaceae</i> were more abundant on <i>M. chamomilla</i>, and <i>Leuconostocaceae</i> and <i>Streptococcaceae</i> more abundant on <i>C. officinalis</i>. The specific aim of this study was the determination of <i>A. ursinum</i>-associated microbiota, as well as isolation and identification of LAB from <i>A. ursinum</i> with potential health-promoting effects.</p> <p>In general, the microbial communities could be evaluated by traditional and molecular methods. The number of studies related to evaluation of ecology of microbial communities demonstrated that the diversity and structure of the associated microbiota cannot be successfully studied by using only one of the above-mentioned approaches. The conventional culture-dependent methods are time consuming and require the isolation and cultivation of microorganisms prior to their identification as well as physiological tests. Hence, the lower abundant taxa might often be outcompeted on culture plates by numerically more abundant species. In contrast, the DGGE analysis, as one of the most frequently and highly reliable molecular techniques used for the study of complex microbial ecology is based on direct DNA isolation, although this method lack quantitative information related to the proportions of the present taxa. Indeed, it is highly appreciated that the combination of both culture-dependent and culture-independent approaches should be used to describe the diversity, structure and dynamic of the complex microbial communities (<a href="#B12">Cocolin et al., 2013</a>).</p> <p>In this study the DGGE analysis revealed the predominant presence of LAB species <i>Lc. lactis</i>, <i>S. thermophilus</i>, <i>L. reuteri</i>, and <i>L. gasseri.</i> Besides, <i>Leuconostoc mesenteroides/pseudomesenteroides</i> and <i>Macrococcus caseolyticus</i>, a large Gram-positive coccus, often associated with meat and sheep, goat and cattle milk, were identified. In addition, <i>Erwinia billingiae</i>, a Gram-negative bacteria belonging to the <i>Enterobacteriaceae</i> family, usually found as a plant pathogen, as well as <i>Staphylococcus warneri</i>, associated with bacteremia, were identified in the DGGE analysis (<a href="#B29">Kamath et al., 1992</a>; <a href="#B56">Prodâ€™homme et al., 2017</a>).</p> <p>However, when traditional microbiological cultivation-dependent approach was used, beside LAB mainly represented by <i>L. fermentum</i>, various other species were identified including soil bacterium <i>Bacillus niacini</i> (<a href="#B43">Nagel and Andreesen, 1991</a>), plant growth-promoting rhizobacterium <i>Bacillus aryabhattai</i> (<a href="#B6">Bhattacharyya et al., 2017</a>), <i>Paenibacillus amylolyticus</i>, previously isolated from fermented rice bran (<a href="#B1">Akuzawa et al., 2016</a>), plant-pathogen <i>Curtobacterium flaccumfaciens</i> (<a href="#B19">Evtushenko and Takeuchi, 2006</a>), plant and human pathogen <i>Bacillus pumilus</i> (<a href="#B76">Yuan and Gao, 2015</a>), opportunistic pathogen a main cause of staphylococcal infections <i>Staphylococcus haemolyticus</i> (<a href="#B14">Czekaj et al., 2015</a>), <i>Arthrobacter woluwensis</i>, associated with subacute infective endocarditis (<a href="#B5">Bernasconi et al., 2004</a>), as well as <i>Rothiaamarae</i> found previously in sludge of a foul water sewer (<a href="#B20">Fan, 2002</a>).</p> <p>Taken together the results obtained in this study revealed discrepancies between the two methods used, pointing that the less abundant species have not been entirely detected by culture-dependent method. The noticed discrepancies could be also related to the PCR detection of remaining DNA from dead cells that could not be detected by culture-dependant method, as well as related to the growth media used in the study, as some species have specific growth requirements that were not met in the experimental setup.</p> <p>Detailed analysis of <i>L. fermentum</i> isolates revealed that all 14 strains could be divided into two PFGE pulsotypes. Hence, the health promoting potential of the strain <i>L. fermentum</i> BGSR163, belonging to pulsotype 1 and <i>L. fermentum</i> BGSR227, belonging to pulsotype 2 were further characterized.</p> <p>According to (<a href="#B21">Food and Agriculture Organization of the United Nations,and World Health Organization, 2006</a>) probiotics are defined as â€œlive microorganisms that confer health benefits to the host when ingested in adequate amountsâ€�. The main criteria for determination of probiotics are their safety status, survival in the simulated GIT conditions, the ability to colonize GIT and immunomodulatory potential.</p> <p>The safety status of probiotic strains implies the absence of the virulence factors (e.g., hemolytic and/or gelatinase activity) and susceptibility to clinically relevant antibiotics (<a href="#B18">European Food Safety Authority, 2012</a>). The results obtained in this study confirmed that the strains <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 are safe to be used as probiotics since they do not exhibit hemolytic and gelatinase activity and are susceptible to all clinically relevant antibiotics.</p> <p>The survival in simulated gastrointestinal conditions is an important criterion for the selection of probiotic strains in order to exhibit their probiotic effects in the gut (<a href="#B21">Food and Agriculture Organization of the United Nations, and World Health Organization, 2006</a>). The results of this study revealed that the strains successfully survived the simulated GIT passage. It was known from literature data that bacteria better survive the GIT transit when applied in food matrix due to the buffering and protective effect of the food components on probiotic bacteria (<a href="#B59">Ranadheera et al., 2010</a>). Particularly, the strains successfully survived simulated mouth conditions indicating that both strains were resistant to saliva and Î±-amylase. The number of viable cells lowered in simulated gastric conditions, although the CFU number was still quite high, indicating that the strains are used to the acidic environment, which is expected for LAB strains (<a href="#B45">Nikolic et al., 2012</a>). Interestingly, the growth of the strains was restored in the simulated intestinal conditions, indicating the possible pre-adaptation in the natural environment since <i>L. fermentum</i> has been commonly found in the gut.</p> <p>Transitory colonization of the intestinal mucosa is another important criterion proposed by FAO/WHO guideline for selection of probiotic strains. In this study, the adhesion of <i>L. fermentum</i> strains BGSR163 and BGSR227 to Caco-2 intestinal epithelial cells (IEC) was investigated. Both strains exhibited the adherence ability to IEC in a range common for probiotic strains (<a href="#B67">Sokovic Bajic et al., 2019</a>), although the higher adhesion was scored in strain <i>L. fermentum</i> BGSR163 than in the strain <i>L. fermentum</i> BGSR227.</p> <p>Finally, the immunomodulatory effects of the strains <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227, as well as <i>A. ursinum</i> extract, were evaluated. Both strains <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR227 significantly stimulated the production of proinflammatory cytokine IL-17 (<i>p</i> &amp;lt; 0.01 and 0.001 respectively), as well as the production of immunoregulatory cytokine IL-10 (<i>p</i> &amp;lt; 0.001). The obtained results indicate the potential of these probiotics to stimulate the immune systemâ€™s response to extracellular pathogens. Cell culture supernatant of <i>L. fermentum</i> BGSR163 and UV inactivated bacteria <i>L. fermentum</i> BGSR163 and <i>L. fermentum</i> BGSR 227 decreased the production of proinflammatory cytokine IFN-Î³ (<i>p</i> &amp;lt; 0.01 and 0.001 respectively). Given that UV inactivated strains significantly reduced the production of IFN-Î³, the major Th1 cytokine, important for response to intracellular pathogens, the supernatants of cell cultures appear to better choice for use as postbiotics. Possible utilization of <i>L. fermentum</i> BGSR227 cell culture supernatant as postbiotic could be a good strategy to stimulate the response to pathogens, without reducing the ability of the immune system to respond to a concomitant viral infection. Plant extract treatment and treatment with food matrix did not exert effects on cytokine production.</p> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Taken together the results of this study suggest that <i>A. ursinum</i> consumption might have health-promoting properties, including anticancer effects. The safety status (the absence of gelatinase and hemolytic activity, susceptibility to the clinically relevant antibiotics) and health promoting potential (survival in the simulated GIT passage, the ability to colonize the intestinal mucosa and immunomodulatory potential) of <i>A. ursinum</i>-associated <i>L. fermentum</i> strains isolated in this study indicate that the strains might be used as probiotics for human consumption. However, the extensive <i>in vivo</i> research is required to fully evaluate the possible cancer preventive efficacy of <i>A. ursinum</i> and its bioactive compounds, as well as probiotic activity of its associated <i>L. fermentum</i> strains in addition to further bioavailability and pharmacokinetic studies.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability Statement</div> <p>The original contributions presented in the study are included in the article/<a href="#TS1">Supplementary Material</a>, further inquiries can be directed to the corresponding author.</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>NS, SB, and MT conceived and designed the study. NS and SB performed the main work. NP participated in the research of safety status and relationship between isolatesâ€™ origins, and performed experiments with the animals used. DP and AT-V participated in the isolation and enumeration of LAB. SB, MT, and DP participated in the probiotic characterization of the isolates and DGGE analysis. NS, SB, MT, and NG analyzed, interpreted, and critically revised the data. NS, SB, NG, and JS prepared the manuscript for submission. Å½J performed antioxidant activity assays. IM tested the effects of the extracts on cell cycle phase distribution. VB performed GC Ã— GC-MS. All authors finally approved the version to be published.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Conflict of Interest</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Funding.</b> This work was funded by the Ministry of Education, Science and Technological Development of the Republic of Serbia, registration numbers: 451-03-68/2020-14/200042 and 451-03-68/2020-14/200168 agreement on the implementation and financing of research work in 2020.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">1</span><p><a href="http://www.ncbi.nlm.nih.gov/BLAST">http://www.ncbi.nlm.nih.gov/BLAST</a></p> </div> </div> <div class="supplementary material" title="supplementary material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2020.601616/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fmicb.2020.601616/full#supplementary-material</a></p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="Table_1.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="Table_2.DOCX">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akuzawa</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagaoka</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanekatsu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kubota</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohtake</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Draft genome sequence of <i>Paenibacillus amylolyticus</i> Heshi-A3, isolated from fermented rice bran in a japanese fermented seafood dish.</span><span class="source" tagx="source" title="source"><i>Genome Announc</i>.</span><span class="volume" tagx="volume" title="volume">4</span>:<span class="issue" tagx="issue" title="issue">e00218-16</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/genomeA.00218-16">10.1128/genomeA.00218-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27034503">27034503</a></span></span></li> <li tag="ref"><a name="B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Argyri</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zoumpopoulou</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karatzas</span><span class="given-names" tagx="given-names" title="given-names">K.-A. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsakalidou</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nychas</span><span class="given-names" tagx="given-names" title="given-names">G.-J. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panagou</span><span class="given-names" tagx="given-names" title="given-names">E. Z.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Selection of potential probiotic lactic acid bacteria from fermented olives by <i>in vitro</i> tests.</span><span class="source" tagx="source" title="source"><i>Food Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">282</span>â€"<span class="lpage" tagx="lpage" title="lpage">291</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fm.2012.10.005">10.1016/j.fm.2012.10.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23200662">23200662</a></span></span></li> <li tag="ref"><a name="B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BajiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">S. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ã�okiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DiniÄ‡</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TomiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PopoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BrdariÄ‡</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">GABA potentiate the immunoregulatory effects of <i>Lactobacillus brevis</i> BGZLS10-17 via ATG5-dependent autophagy in vitro.</span><span class="source" tagx="source" title="source"><i>Sci. Rep.</i></span><span class="volume" tagx="volume" title="volume">10</span>:<span class="issue" tagx="issue" title="issue">1347</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-020-58177-2">10.1038/s41598-020-58177-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31992761">31992761</a></span></span></li> <li tag="ref"><a name="B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benkeblia</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanzotti</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Allium thiosulfinates: chemistry, biological properties and their potential utilization in food preservation.</span><span class="source" tagx="source" title="source"><i>Food</i></span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">193</span>â€"<span class="lpage" tagx="lpage" title="lpage">201</span>.</span></li> <li tag="ref"><a name="B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernasconi</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valsangiacomo</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peduzzi</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carota</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moccetti</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Funke</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title"><i>Arthrobacter woluwensis</i> subacute infective endocarditis: case report and review of the literature.</span><span class="source" tagx="source" title="source"><i>Clin. Infect. Dis</i>.</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">e27</span>â€"<span class="lpage" tagx="lpage" title="lpage">e31</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1086/381436">10.1086/381436</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14765360">14765360</a></span></span></li> <li tag="ref"><a name="B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharyya</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakshi</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallick</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukherji</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bera</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghosh</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-guided insights into the plant growth promotion capabilities of the physiologically versatile <i>Bacillus aryabhattai</i> Strain AB211.</span><span class="source" tagx="source" title="source"><i>Front. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">8</span>:<span class="issue" tagx="issue" title="issue">411</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2017.00411">10.3389/fmicb.2017.00411</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28377746">28377746</a></span></span></li> <li tag="ref"><a name="B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bibi</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Diversity of antagonistic bacteria isolated from medicinal plant <i>Peganum harmala</i> L.</span><span class="source" tagx="source" title="source"><i>Saudi J. Biol</i>. Sci.</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1288</span>â€"<span class="lpage" tagx="lpage" title="lpage">1293</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.sjbs.2015.09.021">10.1016/j.sjbs.2015.09.021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28855824">28855824</a></span></span></li> <li tag="ref"><a name="B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borozan</span><span class="given-names" tagx="given-names" title="given-names">A. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raba</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Evaluation of bacterial community and of <i>Actinomycetes</i> composition from the <i>Allium ursinum</i> L species rhizosphere.</span><span class="source" tagx="source" title="source"><i>LucrÇŽri ÅžtiinÅ£ifice Seria Agronomie</i></span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">31</span>â€"<span class="lpage" tagx="lpage" title="lpage">34</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11118-011-9244-y">10.1007/s11118-011-9244-y</a></span></span></li> <li tag="ref"><a name="B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carotenuto</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Feo</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fattorusso</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanzotti</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magno</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cicala</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">The flavonoids of <i>Allium ursinum</i>.</span><span class="source" tagx="source" title="source"><i>Phytochemistry</i></span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">531</span>â€"<span class="lpage" tagx="lpage" title="lpage">536</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0031-9422(95)00574-9">10.1016/0031-9422(95)00574-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8821433">8821433</a></span></span></li> <li tag="ref"><a name="B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Checcucci</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maida</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bacci</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ninno</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bilia</span><span class="given-names" tagx="given-names" title="given-names">A. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biffi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Is the plant-associated microbiota of <i>Thymus</i> spp. adapted to plant essential oil?</span><span class="source" tagx="source" title="source"><i>Res. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">168</span><span class="fpage" tagx="fpage" title="fpage">276</span>â€"<span class="lpage" tagx="lpage" title="lpage">282</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.resmic.2016.11.004">10.1016/j.resmic.2016.11.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27884782">27884782</a></span></span></li> <li tag="ref"><a name="B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu (Lucy)</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">High-throughput relative DPPH radical scavenging capacity assay.</span><span class="source" tagx="source" title="source"><i>J. Agric. Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">7429</span>â€"<span class="lpage" tagx="lpage" title="lpage">7436</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf0611668">10.1021/jf0611668</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17002404">17002404</a></span></span></li> <li tag="ref"><a name="B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cocolin</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alessandria</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dolci</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorra</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rantsiou</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Culture independent methods to assess the diversity and dynamics of microbiota during food fermentation.</span><span class="source" tagx="source" title="source"><i>Int. J. Food Microbiol.</i></span><span class="volume" tagx="volume" title="volume">162</span><span class="fpage" tagx="fpage" title="fpage">8</span>â€"<span class="lpage" tagx="lpage" title="lpage">12</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23340384">23340384</a></span></span></li> <li tag="ref"><a name="B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Compant</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samad</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faist</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sessitsch</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">A review on the plant microbiome: ecology, functions, and emerging trends in microbial application.</span><span class="source" tagx="source" title="source"><i>J. Adv. Res</i>.</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">29</span>â€"<span class="lpage" tagx="lpage" title="lpage">37</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jare.2019.03.004">10.1016/j.jare.2019.03.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31341667">31341667</a></span></span></li> <li tag="ref"><a name="B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Czekaj</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciszewski</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szewczyk</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title"><i>Staphylococcus haemolyticus</i> â€" an emerging threat in the twilight of the antibiotics age.</span><span class="source" tagx="source" title="source"><i>Microbiology</i></span><span class="volume" tagx="volume" title="volume">161</span><span class="fpage" tagx="fpage" title="fpage">2061</span>â€"<span class="lpage" tagx="lpage" title="lpage">2068</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/mic.0.000178">10.1099/mic.0.000178</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26363644">26363644</a></span></span></li> <li tag="ref"><a name="B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danhorn</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuqua</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Biofilm formation by plant-associated bacteria.</span><span class="source" tagx="source" title="source"><i>Annu. Rev. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">401</span>â€"<span class="lpage" tagx="lpage" title="lpage">422</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.micro.61.080706.093316">10.1146/annurev.micro.61.080706.093316</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17506679">17506679</a></span></span></li> <li tag="ref"><a name="B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinis</span><span class="given-names" tagx="given-names" title="given-names">T. C. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madeira</span><span class="given-names" tagx="given-names" title="given-names">V. M. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almeida</span><span class="given-names" tagx="given-names" title="given-names">L. M.</span></span></span> (<span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Action of phenolic derivatives (acetaminophen, salicylate, and 5-aminosalicylate) as inhibitors of membrane lipid peroxidation and as peroxyl radical scavengers.</span><span class="source" tagx="source" title="source"><i>Arch. Biochem. Biophys</i>.</span><span class="volume" tagx="volume" title="volume">315</span><span class="fpage" tagx="fpage" title="fpage">161</span>â€"<span class="lpage" tagx="lpage" title="lpage">169</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1006/abbi.1994.1485">10.1006/abbi.1994.1485</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7979394">7979394</a></span></span></li> <li tag="ref"><a name="B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Djurdjevic</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinic</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pavlovic</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitrovic</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karadzic</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesevic</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Allelopathic potential of <i>Allium ursinum</i> L.</span><span class="source" tagx="source" title="source"><i>Biochem. Syst. Ecol</i>.</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">533</span>â€"<span class="lpage" tagx="lpage" title="lpage">544</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bse.2003.10.001">10.1016/j.bse.2003.10.001</a></span></span></li> <li tag="ref"><a name="B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: European Food Safety Authority</span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Guidance on the assessment of bacterial susceptibility to antimicrobials of human and veterinary importance.</span><span class="source" tagx="source" title="source"><i>EFSA J</i>.</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="issue" tagx="issue" title="issue">2740</span>.</span></li> <li tag="ref"><a name="B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evtushenko</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takeuchi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). â€œ<span class="mixed-article-title" title="mixed-article-title">The family <i>Microbacteriaceae</i></span>,â€� in <span class="source" tagx="source" title="source"><i>The Prokaryotes</i></span>, <span class="edition" tagx="edition" title="edition">3 Edn</span>, <span class="role" tagx="role" title="role">eds</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dworkin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falkow</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosenberg</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schleifer</span><span class="given-names" tagx="given-names" title="given-names">K.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stackebrandt</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer Science</span>), <span class="fpage" tagx="fpage" title="fpage">1020</span>â€"<span class="lpage" tagx="lpage" title="lpage">1098</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/0-387-30743-5_43">10.1007/0-387-30743-5_43</a></span></span></li> <li tag="ref"><a name="B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title"><i>Rothia amarae</i> sp. nov., from sludge of a foul water sewer.</span><span class="source" tagx="source" title="source"><i>Int. J. Syst. Evol. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">52</span><span class="fpage" tagx="fpage" title="fpage">2257</span>â€"<span class="lpage" tagx="lpage" title="lpage">2260</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/ijs.0.02312-0">10.1099/ijs.0.02312-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12508895">12508895</a></span></span></li> <li tag="ref"><a name="B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: Food and Agriculture Organization of the United Nations, and World Health Organization</span><span class="role" tagx="role" title="role">(Eds)</span> (<span class="year" tagx="year" title="year">2006</span>). <span class="source" tagx="source" title="source"><i>Probiotics in Food: Health and Nutritional Properties and Guidelines for Evaluation, FAO Food and Nutrition Paper.</i></span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Rome</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Food and Agriculture Organization of the United Nations</span>.</span></li> <li tag="ref"><a name="B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gitin</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinica</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parnavel</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">The influence of extraction method on the apparent content of bioactive compounds in romanian <i>Allium</i> spp.</span><span class="source" tagx="source" title="source"><i>Leaves. Not. Bot. Horti Agrobot. Cluj Napoca</i></span><span class="volume" tagx="volume" title="volume">40</span>:<span class="issue" tagx="issue" title="issue">93</span><span class="pub-id"><a href="https://dx.doi.org/10.15835/nbha4017212">10.15835/nbha4017212</a></span></span></li> <li tag="ref"><a name="B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GoÄ‘evac</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VujisiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MojoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">IgnjatoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SpasojeviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vajs</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Evaluation of antioxidant capacity of <i>Allium ursinum</i> L. volatile oil and its effect on membrane fluidity.</span><span class="source" tagx="source" title="source"><i>Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">107</span><span class="fpage" tagx="fpage" title="fpage">1692</span>â€"<span class="lpage" tagx="lpage" title="lpage">1700</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodchem.2007.10.017">10.1016/j.foodchem.2007.10.017</a></span></span></li> <li tag="ref"><a name="B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herman-Antosiewicz</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">S. V.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Signal transduction pathways leading to cell cycle arrest and apoptosis induction in cancer cells by <i>Allium</i> vegetable-derived organosulfur compounds: a review.</span><span class="source" tagx="source" title="source"><i>Mutat. Res. Mol. Mech. Mutagen</i>.</span><span class="volume" tagx="volume" title="volume">555</span><span class="fpage" tagx="fpage" title="fpage">121</span>â€"<span class="lpage" tagx="lpage" title="lpage">131</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mrfmmm.2004.04.016">10.1016/j.mrfmmm.2004.04.016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15476856">15476856</a></span></span></li> <li tag="ref"><a name="B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hopwood</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bibb</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chater</span><span class="given-names" tagx="given-names" title="given-names">K. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kieser</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bruton</span><span class="given-names" tagx="given-names" title="given-names">C. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kieser</span><span class="given-names" tagx="given-names" title="given-names">H. M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">1985</span>). <span class="source" tagx="source" title="source"><i>Genetic Manipulation of Streptomyces: a Laboratory Manual.</i></span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Norwich</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Innes Foundation</span>.</span></li> <li tag="ref"><a name="B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ivanova</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mikhova</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Najdenski</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsvetkova</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kostova</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Chemical composition and antimicrobial activity of wild garlic <i>Allium ursinum</i> of bulgarian origin.</span><span class="source" tagx="source" title="source"><i>Nat. Prod. Commun</i>.</span><span class="volume" tagx="volume" title="volume">4</span>:<span class="issue" tagx="issue" title="issue">1934578X0900400</span><span class="pub-id"><a href="https://dx.doi.org/10.1177/1934578X0900400808">10.1177/1934578X0900400808</a></span></span></li> <li tag="ref"><a name="B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jovcic</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Begovic</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lozo</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Topisirovic</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kojic</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Dynamics of sodium dodecyl sulfate utilization andantibiotic susceptibility of strain <i>Pseudomonas sp</i>. ATCC19151.</span><span class="source" tagx="source" title="source"><i>Arch. Biol. Sci</i>.</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">159</span>â€"<span class="lpage" tagx="lpage" title="lpage">164</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2298/ABS0902159J">10.2298/ABS0902159J</a></span></span></li> <li tag="ref"><a name="B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Julkunen-Tiitto</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">1985</span>). <span class="mixed-article-title" title="mixed-article-title">Phenolic constituents in the leaves of northern willows: methods for the analysis of certain phenolics.</span><span class="source" tagx="source" title="source"><i>J. Agric. Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">213</span>â€"<span class="lpage" tagx="lpage" title="lpage">217</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf00062a013">10.1021/jf00062a013</a></span></span></li> <li tag="ref"><a name="B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamath</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singer</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Isenberg</span><span class="given-names" tagx="given-names" title="given-names">H. D.</span></span></span> (<span class="year" tagx="year" title="year">1992</span>). <span class="mixed-article-title" title="mixed-article-title">Clinical significance of <i>Staphylococcus warneri</i> bacteremia.</span><span class="source" tagx="source" title="source"><i>J. Clin. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">261</span>â€"<span class="lpage" tagx="lpage" title="lpage">264</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/jcm.30.2.261-264.1992">10.1128/jcm.30.2.261-264.1992</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1537891">1537891</a></span></span></li> <li tag="ref"><a name="B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KeleÅŸ</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TaÅŸkÄ±n</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baktemur</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kafkas</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ¼yÃ¼kalaca</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Comparitive study on volatile aroma compounds of two different garlic types (Kastamonu And Chinese) using gas chromatography mass spectrometry (Hs-Gc/Ms) technique.</span><span class="source" tagx="source" title="source"><i>Afr. J. Tradit. Complement. Altern. Med</i>.</span><span class="volume" tagx="volume" title="volume">11</span>:<span class="issue" tagx="issue" title="issue">217</span>. <span class="pub-id"><a href="https://dx.doi.org/10.4314/ajtcam.v11i3.30">10.4314/ajtcam.v11i3.30</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25371585">25371585</a></span></span></li> <li tag="ref"><a name="B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knowles</span><span class="given-names" tagx="given-names" title="given-names">L. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milner</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Possible mechanism by which allyl sulfides suppress neoplastic cell proliferation.</span><span class="source" tagx="source" title="source"><i>J. Nutr</i>.</span><span class="volume" tagx="volume" title="volume">131</span><span class="fpage" tagx="fpage" title="fpage">1061S</span>â€"<span class="lpage" tagx="lpage" title="lpage">1066S</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/jn/131.3.1061S">10.1093/jn/131.3.1061S</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11238817">11238817</a></span></span></li> <li tag="ref"><a name="B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ¶berl</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erschen</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Etemadi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El-Arabi</span><span class="given-names" tagx="given-names" title="given-names">T. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berg</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Deciphering the microbiome shift during fermentation of medicinal plants.</span><span class="source" tagx="source" title="source"><i>Sci. Rep</i>.</span><span class="volume" tagx="volume" title="volume">9</span>:<span class="issue" tagx="issue" title="issue">13461</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-019-49799-2">10.1038/s41598-019-49799-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31530872">31530872</a></span></span></li> <li tag="ref"><a name="B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ¶berl</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramadan</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bauer</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berg</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">The microbiome of medicinal plants: diversity and importance for plant growth, quality and health.</span><span class="source" tagx="source" title="source"><i>Front. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">4</span>:<span class="issue" tagx="issue" title="issue">400</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2013.00400">10.3389/fmicb.2013.00400</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24391634">24391634</a></span></span></li> <li tag="ref"><a name="B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kubec</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VelÃ­Å¡ek</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DoleÅ¾al</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kubelka</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Sulfur-containing volatiles arising by thermal degradation of alliin and deoxyalliin.</span><span class="source" tagx="source" title="source"><i>J. Agric. Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">3580</span>â€"<span class="lpage" tagx="lpage" title="lpage">3585</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf970071q">10.1021/jf970071q</a></span></span></li> <li tag="ref"><a name="B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulas</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mirkov</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tucovic</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zolotarevski</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glamoclija</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veljovic</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Pulmonary <i>Aspergillus fumigatus</i> infection in rats affects gastrointestinal homeostasis.</span><span class="source" tagx="source" title="source"><i>Immunobiology</i></span><span class="volume" tagx="volume" title="volume">224</span><span class="fpage" tagx="fpage" title="fpage">116</span>â€"<span class="lpage" tagx="lpage" title="lpage">123</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.imbio.2018.10.001">10.1016/j.imbio.2018.10.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30348457">30348457</a></span></span></li> <li tag="ref"><a name="B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawson</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunsaker</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Allicin bioavailability and bioequivalence from garlic supplements and garlic foods.</span><span class="source" tagx="source" title="source"><i>Nutrients</i></span><span class="volume" tagx="volume" title="volume">10</span>:<span class="issue" tagx="issue" title="issue">812</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/nu10070812">10.3390/nu10070812</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29937536">29937536</a></span></span></li> <li tag="ref"><a name="B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawson</span><span class="given-names" tagx="given-names" title="given-names">L. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardner</span><span class="given-names" tagx="given-names" title="given-names">C. D.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Composition, stability, and bioavailability of garlic products used in a clinical trial.</span><span class="source" tagx="source" title="source"><i>J. Agric. Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">6254</span>â€"<span class="lpage" tagx="lpage" title="lpage">6261</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf050536">10.1021/jf050536</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16076102">16076102</a></span></span></li> <li tag="ref"><a name="B38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lukic</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strahinic</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milenkovic</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golic</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kojic</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Topisirovic</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Interaction of <i>Lactobacillus fermentum</i> BGHI14 with rat colonic mucosa: implications for colitis induction.</span><span class="source" tagx="source" title="source"><i>Appl. Environ. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">79</span><span class="fpage" tagx="fpage" title="fpage">5735</span>â€"<span class="lpage" tagx="lpage" title="lpage">5744</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.01807-13">10.1128/AEM.01807-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23851097">23851097</a></span></span></li> <li tag="ref"><a name="B39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MatiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">I. Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AljanÄ�iÄ‡</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Å½iÅ¾ak</span><span class="given-names" tagx="given-names" title="given-names">Å½</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vajs</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jadranin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MilosavljeviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">In vitro antitumor actions of extracts from endemic plant <i>Helichrysum zivojinii</i>.</span><span class="source" tagx="source" title="source"><i>BMC Complement. Altern. Med</i>.</span><span class="volume" tagx="volume" title="volume">13</span>:<span class="issue" tagx="issue" title="issue">36</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1472-6882-13-36">10.1186/1472-6882-13-36</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23414290">23414290</a></span></span></li> <li tag="ref"><a name="B40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minekus</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alminger</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alvito</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ballance</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bohn</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourlieu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">A standardised static <i>in vitro</i> digestion method suitable for food â€" an international consensus.</span><span class="source" tagx="source" title="source"><i>Food Funct</i>.</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1113</span>â€"<span class="lpage" tagx="lpage" title="lpage">1124</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/c3fo60702j">10.1039/c3fo60702j</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24803111">24803111</a></span></span></li> <li tag="ref"><a name="B41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mosmann</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">1983</span>). <span class="mixed-article-title" title="mixed-article-title">Rapid colorimetric assay for cellular growth and survival: application to proliferation and cytotoxicity assays.</span><span class="source" tagx="source" title="source"><i>J. Immunol. Methods</i></span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">55</span>â€"<span class="lpage" tagx="lpage" title="lpage">63</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0022-1759(83)90303-4">10.1016/0022-1759(83)90303-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6606682">6606682</a></span></span></li> <li tag="ref"><a name="B42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagaraj</span><span class="given-names" tagx="given-names" title="given-names">N. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anilakumar</span><span class="given-names" tagx="given-names" title="given-names">K. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">O. V.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Diallyl disulfide causes caspase-dependent apoptosis in human cancer cells through a Bax-triggered mitochondrial pathway.</span><span class="source" tagx="source" title="source"><i>J. Nutr. Biochem</i>.</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">405</span>â€"<span class="lpage" tagx="lpage" title="lpage">412</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jnutbio.2009.01.015">10.1016/j.jnutbio.2009.01.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19423321">19423321</a></span></span></li> <li tag="ref"><a name="B43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagel</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andreesen</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span></span> (<span class="year" tagx="year" title="year">1991</span>). <span class="mixed-article-title" title="mixed-article-title"><i>Bacillus niacini</i> sp. nov., a nicotinate-metabolizing mesophile isolated from Soil.</span><span class="source" tagx="source" title="source"><i>Int. J. Syst. Bacteriol</i>.</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">134</span>â€"<span class="lpage" tagx="lpage" title="lpage">139</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/00207713-41-1-134">10.1099/00207713-41-1-134</a></span></span></li> <li tag="ref"><a name="B44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakano</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murakami</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nohara</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tomimatsu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">1981</span>). <span class="mixed-article-title" title="mixed-article-title">The constituents of Paris verticillata M.v.</span><span class="source" tagx="source" title="source"><i>Bieb. Chem. Pharm. Bull. (Tokyo)</i></span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">1445</span>â€"<span class="lpage" tagx="lpage" title="lpage">1451</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1248/cpb.29.1445">10.1248/cpb.29.1445</a></span></span></li> <li tag="ref"><a name="B45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikolic</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LÃ³pez</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strahinic</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SuÃ¡rez</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kojic</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FernÃ¡ndez-GarcÃ­a</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Characterisation of the exopolysaccharide (EPS)-producing <i>Lactobacillus paraplantarum</i> BGCG11 and its non-EPS producing derivative strains as potential probiotics.</span><span class="source" tagx="source" title="source"><i>Int. J. Food Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">158</span><span class="fpage" tagx="fpage" title="fpage">155</span>â€"<span class="lpage" tagx="lpage" title="lpage">162</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijfoodmicro.2012.07.015">10.1016/j.ijfoodmicro.2012.07.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22858135">22858135</a></span></span></li> <li tag="ref"><a name="B46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohno</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abe</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">1991</span>). <span class="mixed-article-title" title="mixed-article-title">Rapid colorimetric assay for the quantification of leukemia inhibitory factor (LIF) and interleukin-6 (IL-6).</span><span class="source" tagx="source" title="source"><i>J. Immunol. Methods</i></span><span class="volume" tagx="volume" title="volume">145</span><span class="fpage" tagx="fpage" title="fpage">199</span>â€"<span class="lpage" tagx="lpage" title="lpage">203</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0022-1759(91)90327-c">10.1016/0022-1759(91)90327-c</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1765652">1765652</a></span></span></li> <li tag="ref"><a name="B47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ormerod</span><span class="given-names" tagx="given-names" title="given-names">M.G. (Ed.).</span></span></span> (<span class="year" tagx="year" title="year">1994</span>). <span class="source" tagx="source" title="source"><i>Flow Cytometry: A Practical Approach, The Practical Approach Series</i></span>, <span class="edition" tagx="edition" title="edition">2 Edn</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span>.</span></li> <li tag="ref"><a name="B48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">OszmiaÅ"ski</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolniak-Ostek</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">WojdyÅ‚o</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Characterization and content of flavonol derivatives of <i>Allium ursinum</i> L.</span><span class="source" tagx="source" title="source"><i>Plant. J. Agric. Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">176</span>â€"<span class="lpage" tagx="lpage" title="lpage">184</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf304268e">10.1021/jf304268e</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23249145">23249145</a></span></span></li> <li tag="ref"><a name="B49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">K.-W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeong</span><span class="given-names" tagx="given-names" title="given-names">I.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Byun</span><span class="given-names" tagx="given-names" title="given-names">M.-W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">K.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Cytotoxic and antitumor activities of thiosulfinates from <i>Allium tuberosum</i> L.</span><span class="source" tagx="source" title="source"><i>J. Agric. Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">7957</span>â€"<span class="lpage" tagx="lpage" title="lpage">7961</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf0713051">10.1021/jf0713051</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17711341">17711341</a></span></span></li> <li tag="ref"><a name="B50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paulsen</span><span class="given-names" tagx="given-names" title="given-names">I. T.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Role of mobile DNA in the evolution of vancomycin-resistant <i>Enterococcus faecalis</i>.</span><span class="source" tagx="source" title="source"><i>Science</i></span><span class="volume" tagx="volume" title="volume">299</span><span class="fpage" tagx="fpage" title="fpage">2071</span>â€"<span class="lpage" tagx="lpage" title="lpage">2074</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1080613">10.1126/science.1080613</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12663927">12663927</a></span></span></li> <li tag="ref"><a name="B51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PavloviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">D. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VeljkoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">StojanoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">N. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GoÄ�manac-IgnjatoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mihailov-Krstev</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BrankoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Influence of different wild-garlic (<i>Allium ursinum</i>) extracts on the gastrointestinal system: spasmolytic, antimicrobial and antioxidant properties.</span><span class="source" tagx="source" title="source"><i>J. Pharm. Pharmacol</i>.</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">1208</span>â€"<span class="lpage" tagx="lpage" title="lpage">1218</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/jphp.12746">10.1111/jphp.12746</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28543032">28543032</a></span></span></li> <li tag="ref"><a name="B52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PÃªkal</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyrzynska</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Evaluation of aluminium complexation reaction for flavonoid content assay.</span><span class="source" tagx="source" title="source"><i>Food Anal. Methods</i>.</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">1776</span>â€"<span class="lpage" tagx="lpage" title="lpage">1782</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s12161-014-9814-x">10.1007/s12161-014-9814-x</a></span></span></li> <li tag="ref"><a name="B53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PeÅ¡iÄ‡</span><span class="given-names" tagx="given-names" title="given-names">M. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MilinÄ�iÄ‡</span><span class="given-names" tagx="given-names" title="given-names">D. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KostiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">A. Å½</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">StanisavljeviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">N. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VukotiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">G. N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KojiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">M. O.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title"><i>In vitro</i> digestion of meat- and cereal-based food matrix enriched with grape extracts: How are polyphenol composition, bioaccessibility and antioxidant activity affected?</span><span class="source" tagx="source" title="source"><i>Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">284</span><span class="fpage" tagx="fpage" title="fpage">28</span>â€"<span class="lpage" tagx="lpage" title="lpage">44</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodchem.2019.01.107">10.1016/j.foodchem.2019.01.107</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30744859">30744859</a></span></span></li> <li tag="ref"><a name="B54" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Presta</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosi</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fondi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maida</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perrin</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miceli</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Phenotypic and genomic characterization of the antimicrobial producer <i>Rheinheimera</i> sp. EpRS3 isolated from the medicinal plant <i>Echinacea purpurea</i>: insights into its biotechnological relevance.</span><span class="source" tagx="source" title="source"><i>Res. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">168</span><span class="fpage" tagx="fpage" title="fpage">293</span>â€"<span class="lpage" tagx="lpage" title="lpage">305</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.resmic.2016.11.001">10.1016/j.resmic.2016.11.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27884784">27884784</a></span></span></li> <li tag="ref"><a name="B55" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prior</span><span class="given-names" tagx="given-names" title="given-names">R. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schaich</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Standardized methods for the determination of antioxidant capacity and phenolics in foods and dietary supplements.</span><span class="source" tagx="source" title="source"><i>J. Agric. Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">4290</span>â€"<span class="lpage" tagx="lpage" title="lpage">4302</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf0502698">10.1021/jf0502698</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15884874">15884874</a></span></span></li> <li tag="ref"><a name="B56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prodâ€™homme</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Micol</span><span class="given-names" tagx="given-names" title="given-names">L. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weitsch</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gassend</span><span class="given-names" tagx="given-names" title="given-names">J.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinet</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bellini</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Cutaneous infection and bactaeremia caused by <i>Erwinia billingiae</i>: a case report.</span><span class="source" tagx="source" title="source"><i>New Microbes New Infect</i>.</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">134</span>â€"<span class="lpage" tagx="lpage" title="lpage">136</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.nmni.2017.07.006">10.1016/j.nmni.2017.07.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28831302">28831302</a></span></span></li> <li tag="ref"><a name="B57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Rhizosphere and non-rhizosphere bacterial community composition of the wild medicinal plant <i>Rumex patientia</i>.</span><span class="source" tagx="source" title="source"><i>World J. Microbiol. Biotechnol.</i></span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">2257</span>â€"<span class="lpage" tagx="lpage" title="lpage">2265</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11274-012-1033-2">10.1007/s11274-012-1033-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22806049">22806049</a></span></span></li> <li tag="ref"><a name="B58" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Allicin and other functional active components in garlic: health benefits and bioavailability.</span><span class="source" tagx="source" title="source"><i>Int. J. Food Prop</i>.</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">245</span>â€"<span class="lpage" tagx="lpage" title="lpage">268</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/10942910601113327">10.1080/10942910601113327</a></span></span></li> <li tag="ref"><a name="B59" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranadheera</span><span class="given-names" tagx="given-names" title="given-names">R. D. C. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baines</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">M. C.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Importance of food in probiotic efficacy.</span><span class="source" tagx="source" title="source"><i>Food Res. Int</i>.</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodres.2009.09.009">10.1016/j.foodres.2009.09.009</a></span></span></li> <li tag="ref"><a name="B60" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribeiro</span><span class="given-names" tagx="given-names" title="given-names">A. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berto</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribeiro</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freitas</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ChistÃ©</span><span class="given-names" tagx="given-names" title="given-names">R. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Visentainer</span><span class="given-names" tagx="given-names" title="given-names">J. V.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Stem bark and flower extracts of <i>Vismia cauliflora</i> are highly effective antioxidants to human blood cells by preventing oxidative burst in neutrophils and oxidative damage in erythrocytes.</span><span class="source" tagx="source" title="source"><i>Pharm. Biol</i>.</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">1691</span>â€"<span class="lpage" tagx="lpage" title="lpage">1698</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3109/13880209.2014.1001407">10.3109/13880209.2014.1001407</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25868622">25868622</a></span></span></li> <li tag="ref"><a name="B61" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SÃ¡nchez</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FernÃ¡ndez-GarcÃ­a</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Margolles</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de los Reyes-GavilÃ¡n</span><span class="given-names" tagx="given-names" title="given-names">C. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruas-Madiedo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Technological and probiotic selection criteria of a bile-adapted <i>Bifidobacterium animalis</i> subsp. <i>lactis</i> strain.</span><span class="source" tagx="source" title="source"><i>Int. Dairy J</i>.</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">800</span>â€"<span class="lpage" tagx="lpage" title="lpage">805</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.idairyj.2010.06.004">10.1016/j.idairyj.2010.06.004</a></span></span></li> <li tag="ref"><a name="B62" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satyal</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Craft</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dosoky</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Setzer</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">The chemical compositions of the volatile oils of garlic (<i>Allium sativum</i>) and wild garlic (<i>Allium vineale</i>).</span><span class="source" tagx="source" title="source"><i>Foods</i></span><span class="volume" tagx="volume" title="volume">6</span>:<span class="issue" tagx="issue" title="issue">63</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/foods6080063">10.3390/foods6080063</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28783070">28783070</a></span></span></li> <li tag="ref"><a name="B63" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schepetkin</span><span class="given-names" tagx="given-names" title="given-names">I. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kirpotina</span><span class="given-names" tagx="given-names" title="given-names">L. N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khlebnikov</span><span class="given-names" tagx="given-names" title="given-names">A. I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balasubramanian</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quinn</span><span class="given-names" tagx="given-names" title="given-names">M. T.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Neutrophil immunomodulatory activity of natural organosulfur compounds.</span><span class="source" tagx="source" title="source"><i>Molecules</i></span><span class="volume" tagx="volume" title="volume">24</span>:<span class="issue" tagx="issue" title="issue">1809</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/molecules24091809">10.3390/molecules24091809</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31083328">31083328</a></span></span></li> <li tag="ref"><a name="B64" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmitt</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulz</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Storsberg</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keusgen</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Chemical characterization of <i>Allium ursinum</i> L. depending on harvesting time.</span><span class="source" tagx="source" title="source"><i>J. Agric. Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">7288</span>â€"<span class="lpage" tagx="lpage" title="lpage">7294</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf0504768">10.1021/jf0504768</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16131144">16131144</a></span></span></li> <li tag="ref"><a name="B65" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sendl</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagner</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">1991</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation and identification of homologues of ajoene and alliin from bulb-extracts of <i>Allium ursinum</i>.</span><span class="source" tagx="source" title="source"><i>Planta Med</i>.</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">361</span>â€"<span class="lpage" tagx="lpage" title="lpage">362</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1055/s-2006-960118">10.1055/s-2006-960118</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17226171">17226171</a></span></span></li> <li tag="ref"><a name="B66" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sobolewska</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Podolak</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makowska-WaÌ§s</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title"><i>Allium ursinum</i>: botanical, phytochemical and pharmacological overview.</span><span class="source" tagx="source" title="source"><i>Phytochem. Rev</i>.</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">81</span>â€"<span class="lpage" tagx="lpage" title="lpage">97</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11101-013-9334-0">10.1007/s11101-013-9334-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25774103">25774103</a></span></span></li> <li tag="ref"><a name="B67" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokovic Bajic</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Djokic</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinic</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veljovic</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golic</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mihajlovic</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">GABA-producing natural dairy isolate from artisanal zlatar cheese attenuates gut inflammation and strengthens gut epithelial barrier in vitro.</span><span class="source" tagx="source" title="source"><i>Front. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="issue" tagx="issue" title="issue">527</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2019.00527">10.3389/fmicb.2019.00527</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30936860">30936860</a></span></span></li> <li tag="ref"><a name="B68" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">StanisavljeviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SamardÅ¾iÄ‡</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JankoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Å avikin</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mojsin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TopaloviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Antioxidant and antiproliferative activity of chokeberry juice phenolics during <i>in vitro</i> simulated digestion in the presence of food matrix.</span><span class="source" tagx="source" title="source"><i>Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">175</span><span class="fpage" tagx="fpage" title="fpage">516</span>â€"<span class="lpage" tagx="lpage" title="lpage">522</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodchem.2014.12.009">10.1016/j.foodchem.2014.12.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25577114">25577114</a></span></span></li> <li tag="ref"><a name="B69" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">Y. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sulavik</span><span class="given-names" tagx="given-names" title="given-names">M. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makinen</span><span class="given-names" tagx="given-names" title="given-names">K. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makinen</span><span class="given-names" tagx="given-names" title="given-names">P. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fiedler</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">1991</span>). <span class="mixed-article-title" title="mixed-article-title">Nucleotide sequence of the gelatinase gene (gelE) from <i>Enterococcus faecalis</i> subsp. <i>liquefaciens</i>.</span><span class="source" tagx="source" title="source"><i>Infect. Immun</i>.</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">415</span>â€"<span class="lpage" tagx="lpage" title="lpage">420</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/IAI.59.1.415-420.1991">10.1128/IAI.59.1.415-420.1991</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1846126">1846126</a></span></span></li> <li tag="ref"><a name="B70" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tenover</span><span class="given-names" tagx="given-names" title="given-names">F. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arbeit</span><span class="given-names" tagx="given-names" title="given-names">R. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goering</span><span class="given-names" tagx="given-names" title="given-names">R. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mickelsen</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murray</span><span class="given-names" tagx="given-names" title="given-names">B. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Persing</span><span class="given-names" tagx="given-names" title="given-names">D. H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Interpreting chromosomal DNA restriction patterns produced by pulsed-field gel electrophoresis: criteria for bacterial strain typing.</span><span class="source" tagx="source" title="source"><i>J. Clin. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">2233</span>â€"<span class="lpage" tagx="lpage" title="lpage">2239</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/jcm.33.9.2233-2239.1995">10.1128/jcm.33.9.2233-2239.1995</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7494007">7494007</a></span></span></li> <li tag="ref"><a name="B71" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trio</span><span class="given-names" tagx="given-names" title="given-names">P. Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">You</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakao</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">D. X.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Chemopreventive functions and molecular mechanisms of garlic organosulfur compounds.</span><span class="source" tagx="source" title="source"><i>Food Funct</i>.</span><span class="volume" tagx="volume" title="volume">5</span>:<span class="issue" tagx="issue" title="issue">833</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/c3fo60479a">10.1039/c3fo60479a</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24664286">24664286</a></span></span></li> <li tag="ref"><a name="B72" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dushenkov</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">C.-T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Novel acetylated flavonoid glycosides from the leaves of <i>Allium ursinum</i>.</span><span class="source" tagx="source" title="source"><i>Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">592</span>â€"<span class="lpage" tagx="lpage" title="lpage">595</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodchem.2008.12.058">10.1016/j.foodchem.2008.12.058</a></span></span></li> <li tag="ref"><a name="B73" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Apoptosis and G2/M arrest induced by <i>Allium ursinum</i> (ramson) watery extract in an AGS gastric cancer cell line.</span><span class="source" tagx="source" title="source"><i>OncoTargets Ther</i>.</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">779</span>â€"<span class="lpage" tagx="lpage" title="lpage">783</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2147/OTT.S45865">10.2147/OTT.S45865</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23836991">23836991</a></span></span></li> <li tag="ref"><a name="B74" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular mechanisms for the anti-cancer effects of diallyl disulfide.</span><span class="source" tagx="source" title="source"><i>Food Chem. Toxicol</i>.</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">362</span>â€"<span class="lpage" tagx="lpage" title="lpage">370</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fct.2013.04.001">10.1016/j.fct.2013.04.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23583486">23583486</a></span></span></li> <li tag="ref"><a name="B75" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">T. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">L. Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">C. T.</span></span></span> (<span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Volatile compounds of blanched, fried blanched, and baked blanched garlic slices.</span><span class="source" tagx="source" title="source"><i>J. Agric. Food Chem</i>.</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">1342</span>â€"<span class="lpage" tagx="lpage" title="lpage">1347</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jf00042a018">10.1021/jf00042a018</a></span></span></li> <li tag="ref"><a name="B76" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Genomic analysis of a ginger pathogen <i>Bacillus pumilus</i> providing the understanding to the pathogenesis and the novel control strategy.</span><span class="source" tagx="source" title="source"><i>Sci. Rep</i>.</span><span class="volume" tagx="volume" title="volume">5</span>:<span class="issue" tagx="issue" title="issue">10259</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/srep10259">10.1038/srep10259</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25989507">25989507</a></span></span></li> <li tag="ref"><a name="B77" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zouari</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Najjaa</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neffati</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ammar</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">A new essential oil chemotype of <i>Allium Roseum</i> analyzed by an apolar column.</span><span class="source" tagx="source" title="source"><i>Int. J. Food Prop</i>.</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">385</span>â€"<span class="lpage" tagx="lpage" title="lpage">397</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/10942912.2010.487629">10.1080/10942912.2010.487629</a></span></span></li> </ul> </div> </div>  </body></html>