<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Ecol Evol</span><span class="iso-abbrev" title="iso-abbrev">Ecol Evol</span><span class="doi" title="doi">10.1002/(ISSN)2045-7758</span><span class="publisher-id" title="publisher-id">ECE3</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Ecology and Evolution</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-7758</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7882976">7882976</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1002/ece3.7171">10.1002/ece3.7171</a></span><span class="publisher-id" title="publisher-id">ECE37171</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Research</span>: <span class="subject" title="subject">Original Research</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Identification of novel bacterial biomarkers to detect bird scavenging by invasive rats</div> <div class="alt-title" title="alt-title">
MULETZâ€�WOLZ et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muletzâ€�Wolz</span><span class="given-names" tagx="given-names" title="given-names">Carly R.</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-5047-9601</span><a href="#ece37171-aff-0001"><sup>1</sup></a><span class="address" title="address"><span class="email" tagx="email" title="email">muletzc@si.edu</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson Rankin</span><span class="given-names" tagx="given-names" title="given-names">Erin</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-7741-113X</span><a href="#ece37171-aff-0001"><sup>1</sup></a><a href="#ece37171-aff-0002"><sup>2</sup></a><a href="#ece37171-aff-0003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGrathâ€�Blaser</span><span class="given-names" tagx="given-names" title="given-names">Sarah</span></span><a href="#ece37171-aff-0001"><sup>1</sup></a><a href="#ece37171-aff-0004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venkatraman</span><span class="given-names" tagx="given-names" title="given-names">Madhvi</span></span><a href="#ece37171-aff-0001"><sup>1</sup></a><a href="#ece37171-aff-0002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maldonado</span><span class="given-names" tagx="given-names" title="given-names">JesÃºs E.</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-4282-1072</span><a href="#ece37171-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gruner</span><span class="given-names" tagx="given-names" title="given-names">Daniel S.</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-3153-4297</span><a href="#ece37171-aff-0002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fleischer</span><span class="given-names" tagx="given-names" title="given-names">Robert C.</span></span><a href="#ece37171-aff-0001"><sup>1</sup></a></span></div><span class="citation_author_institution" id="ece37171-aff-0001">[1], <span class="named-content" title="named-content">Center for Conservation Genomics</span><span class="institution" title="institution">Smithsonian Conservation Biology Institute</span><span class="institution" title="institution">National Zoological Park</span><span class="city" tagx="city" title="city">Washington</span><span class="named-content" title="named-content">DC</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="ece37171-aff-0002">[2], <span class="named-content" title="named-content">Department of Entomology</span><span class="institution" title="institution">University of Maryland</span><span class="city" tagx="city" title="city">College Park</span><span class="named-content" title="named-content">MD</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="ece37171-aff-0003">[3], <span class="named-content" title="named-content">Department of Entomology</span><span class="institution" title="institution">University of California</span><span class="city" tagx="city" title="city">Riverside</span><span class="named-content" title="named-content">CA</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="ece37171-aff-0004">[4], <span class="named-content" title="named-content">Department of Biology</span><span class="institution" title="institution">University of Florida</span><span class="city" tagx="city" title="city">Gainesville</span><span class="named-content" title="named-content">FL</span><span class="country" tagx="country" title="country">USA</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><b>Correspondence</b><br /><br /> Carly R. Muletzâ€�Wolz, Center for Conservation Genomics, Smithsonian Conservation Biology Institute, National Zoological Park, Washington, DC 20008, USA.<br /><br /> Email: <span class="email" tagx="email" title="email">muletzc@si.edu</span><br /><br /></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021-2</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">4</span><span class="issue-id" tagx="issue-id" title="issue-id">10.1002/ece3.v11.4</span><span class="fpage" tagx="fpage" title="fpage">1814</span><span class="lpage" tagx="lpage" title="lpage">1828</span><span class="history" title="history"><span class="received" title="received">received: 2020-7-31</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-12-11</span><span class="accepted" title="accepted">accepted: 2020-12-16</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="file:ECE3-11-1814.pdf" title="self-uri">file:ECE3-11-1814.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Rapid advances in genomic tools for use in ecological contexts and nonâ€�model systems allow unprecedented insight into interactions that occur beyond direct observation. We developed an approach that couples microbial forensics with molecular dietary analysis to identify species interactions and scavenging by invasive rats on native and introduced birds in Hawaii. First, we characterized bacterial signatures of bird carcass decay by conducting 16S rRNA highâ€�throughput sequencing on chicken (<i>Gallus gallus domesticus</i>) tissues collected over an 11â€�day decomposition study in natural Hawaiian habitats. Second, we determined if fieldâ€�collected invasive black rats (<i>Rattus rattus</i>; <i>n</i>Â =Â 51, stomach and fecal samples) had consumed birds using molecular diet analysis with two independent PCR assays (mitochondrial Cytochrome Oxidase I and Cytochrome b genes) and Sanger sequencing. Third, we characterized the gut microbiome of the same rats using 16S rRNA highâ€�throughput sequencing and identified 15 bacterial taxa that were (a) detected only in rats that consumed birds (<i>n</i>Â =Â 20/51) and (b) were indicative of decaying tissue in the chicken decomposition experiment. We found that 18% of rats (<i>n</i>Â =Â 9/51) likely consumed birds as carrion by the presence of bacterial biomarkers of decayed tissue in their gut microbiome. One species of native bird (<i>Myadestes obscurus</i>) and three introduced bird species (<i>Lophura leucomelanos</i>, <i>Meleagris gallopavo</i>, <i>Zosterops japonicus</i>) were detected in the ratsâ€™ diets, with individuals from these species (except <i>L. nycthemera</i>) likely consumed through scavenging. Bacterial biomarkers of bird carcass decay can persist through rat digestion and may serve as biomarkers of scavenging. Our approach can be used to reveal trophic interactions that are challenging to measure through direct observation.</p> </div> <div class="abstract" title="abstract"> <p>We developed a novel threeâ€�part approach using molecular dietary analysis and microbial forensics to reveal species interactions and scavenging by invasive rats on native and introduced birds in Hawaii. Our forensic genomic tool for use in ecological contexts is valuable for (a) detecting diet items in stomach and fecal samples, and (b) determining decomposition status of prey. Our method properly calibrated and replicated has great potential to detect and quantify species interactions that are remote, rare, and challenging to measure. </p><div class="boxed-text" tagx="boxed-text" title="boxed-text"> <div class="graphic" position="anchor" id="nlm-graphic-1" orientation="portrait" /> </div> <p /> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
biomarkers</div> <div class="kwd" title="kwd">
decay</div> <div class="kwd" title="kwd">
forensics</div> <div class="kwd" title="kwd">
genomics</div> <div class="kwd" title="kwd">
Hawaii</div> <div class="kwd" title="kwd">
microbiome</div> <div class="kwd" title="kwd">
noninvasive</div> <div class="kwd" title="kwd">
predation</div> <div class="kwd" title="kwd">
scavenging</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Robert &amp;amp; Arlene Kogod</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
University of Marylandâ€�Smithsonian Seed Grant program</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">National Science Foundation </span><span class="institution-id" title="institution-id">10.13039/100000001</span></div> </div><span class="award-id" tagx="award-id" title="award-id">DEBâ€�1019928</span><span class="award-id" tagx="award-id" title="award-id">DEBâ€�1020007</span><span class="award-id" tagx="award-id" title="award-id">DEBâ€�1020412</span></div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="cover-date" value="February 2021" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_JATSPMC version:5.9.7 mode:remove_FC converted:15.02.2021" /> </div> </div> <p content-type="self-citation"><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Muletzâ€�Wolz</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wilson Rankin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McGrathâ€�Blaser</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Identification of novel bacterial biomarkers to detect bird scavenging by invasive rats</span>. <span class="source" tagx="source" title="source">Ecol Evol</span>. <span class="year" tagx="year" title="year">2021</span>;<span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">1814</span>â€"<span class="lpage" tagx="lpage" title="lpage">1828</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.7171">10.1002/ece3.7171</a></span></span></p> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p>This article has been contributed to by US Government employees and their work is in the public domain in the USA.</p> </div> <div class="fn-type-" title=""> <p>This research was approved by the US Geological Bird Banding Laboratory (permit no. 22958), the State of Hawaii Dept of Land and Natural Resources Division of Forestry and Wildlife (Protected Wildlife permit no. WL13â€�02), the Michigan Technological Univ. Institutional Animal Care and Use Committee (IACUC, no. 332879â€�4), the USDA APHIS WS IACUC Protocol QAâ€�1776, and Stanford IACUC, no. 1776.</p> </div> <div class="fn-type-funding" title="funding"> <p><b>Funding information</b>This project was funded by the University of Marylandâ€�Smithsonian Seed Grant program to DSG, RCF, JEM, and by Robert and Arlene Kogod support of the Secretarial Scholar Program to CRMW. Additional funding included National Science Foundation grants (DEBâ€�1020007 to DSG; DEBâ€�1020412 to T. Fukami &amp;amp; C. Giardina; and DEBâ€�1019928 to D. Flaspohler), the USDA Forest Service Inst. of Pacific Islands Forestry for support in implementation and maintenance of the larger study. Any use of trade, firm, or product names is for descriptive purposes only and does not imply endorsement by the U.S. Government. None of the funders had any input into the content of the manuscript. None of the funders required their approval of the manuscript before submission or publication.</p> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>Interactions among species impact biodiversity patterns, ecosystem responses to change, and ecosystem services (Tylianakis etÂ al.,Â <a href="#ece37171-bib-0059">2008</a>). The classic approaches for measuring species interactions involve direct and indirect observation, experimental manipulations of species or functional guilds, and predator stomach content analysis (Novak &amp;amp; Wootton,Â <a href="#ece37171-bib-0042">2010</a>; Paine,Â <a href="#ece37171-bib-0043">1980</a>; Ruffino etÂ al.,Â <a href="#ece37171-bib-0052">2015</a>). However, trophic interactions may be cryptic or rare and are often logistically challenging to measure. An integrated battery of techniques is often necessary to quantify predation by consumers, resolve trophic links, and to parameterize food web network models (Birkhofer etÂ al.,Â <a href="#ece37171-bib-0006">2017</a>; Carreonâ€�Martinez &amp;amp; Heath,Â <a href="#ece37171-bib-0010">2010</a>; Traugott etÂ al.,Â <a href="#ece37171-bib-0058">2013</a>). To date, these methods do not provide a solution for determining if a consumer is intaking a diet item as prey through predation or as carrion through scavenging. Ecologists often use indirect approaches (e.g., molecular or isotopic analyses) to identify that a food item was consumed, and the detection of a food item is generally classified as a predation event. Yet, scavenging may be involved in up to 45% of food web links and represents a substantial form of energy transfer between trophic levels that is unique from predation (Wilson &amp;amp; Wolkovich,Â <a href="#ece37171-bib-0066">2011</a>). For instance, 124â€�fold more energy can be transferred per scavenging link than per predation link (Wilson &amp;amp; Wolkovich,Â <a href="#ece37171-bib-0066">2011</a>). Underestimating the role of scavenging in food webs likely impacts assessments of predatorâ€"prey interactions, energy flow, and important foodweb metrics. Likewise, the distinction between predation and scavenging has consequences for population and community dynamics (Moleon etÂ al.,Â <a href="#ece37171-bib-0038">2014</a>). Forensic genomics may be a useful technique to differentiate predation and scavenging to aid in our understanding of species interactions and food web ecology.</p> <p>Forensic genomics used in an ecological context is an increasingly valuable tool for (a) detecting and identifying consumed prey in predator diet samples (invasively or noninvasively collected) and (b) determining rates of decomposition of an animal carcass, among others. First, molecular analyses of fecal material or gastrointestinal (GI) samples represent a technique to study diet and elucidate trophic interactions (McInnes etÂ al.,Â <a href="#ece37171-bib-0034">2017</a>; Rytkonen etÂ al.,Â <a href="#ece37171-bib-0054">2019</a>; Zarzosoâ€�Lacoste etÂ al.,Â <a href="#ece37171-bib-0070">2016</a>). These methods are based on PCR amplification of targeted diet items, using universal or speciesâ€�specific primer pairs, in samples collected from the consumer. DNA sequencing is then performed on the amplified PCR products either through Sanger sequencing (Zarzosoâ€�Lacoste etÂ al.,Â <a href="#ece37171-bib-0070">2016</a>) or highâ€�throughput sequencing (Rytkonen etÂ al.,Â <a href="#ece37171-bib-0054">2019</a>) and the resulting DNA sequences are compared to customized and/or public DNA reference databases (e.g., BOLD, NCBI GenBank) to identify the diet item. Second, molecular analyses of microbial community change over carcass decomposition time represents a unique way to study time since death (Belk etÂ al.,Â <a href="#ece37171-bib-0004">2018</a>; Metcalf etÂ al.,Â <a href="#ece37171-bib-0037">2016</a>; Pechal etÂ al.,Â <a href="#ece37171-bib-0045">2014</a>). This method is based on PCR amplification with universal primer pairs to target microbiota at domainâ€�level scales (e.g., bacteria, fungi) to identify microbial biomarkers associated with different time periods of decomposition. Microbial succession of particular taxa is often predictable across soil types, seasons and host species (Belk etÂ al.,Â <a href="#ece37171-bib-0004">2018</a>; Metcalf etÂ al.,Â <a href="#ece37171-bib-0037">2016</a>). Uniting these molecular methods can provide important food web insight into what omnivores/carnivores are consuming as diet items and whether particular diet items were likely consumed as prey through predation or carrion through scavenging. Quantifying rates of predation versus scavenging can provide novel insights into food web ecology and foster a more integrated understanding of how complex web dynamics interact (e.g., how prey or scavenging affects invasive population size, influences predation on vulnerable prey or affects food web stability; Moleon etÂ al.,Â <a href="#ece37171-bib-0038">2014</a>; Wolkovich etÂ al.,Â <a href="#ece37171-bib-0068">2014</a>; Zou etÂ al.,Â <a href="#ece37171-bib-0073">2016</a>).</p> <p>Invasive rats have profound and wideâ€�ranging effects on island ecosystems and food webs (Clark,Â <a href="#ece37171-bib-0012">1982</a>; Shiels etÂ al.,Â <a href="#ece37171-bib-0055">2014</a>; Towns etÂ al.,Â <a href="#ece37171-bib-0057">2006</a>). For example, in Hawaii the invasive black rat (<i>Rattus rattus</i>) consumes a diversity of foods including seeds, fruits, arthropods, carrion, bird eggs and nestlings (Amarasekare,Â <a href="#ece37171-bib-0001">1993</a>; Cole etÂ al.,Â <a href="#ece37171-bib-0013">2000</a>; Levy,Â <a href="#ece37171-bib-0032">2003</a>). Over millions of years, Hawaiian island ecosystems have evolved in the absence of any functional analog to rats, with bats as the only native volant terrestrial mammal (Percy etÂ al.,Â <a href="#ece37171-bib-0046">2008</a>; Price &amp;amp; Clague,Â <a href="#ece37171-bib-0047">2002</a>). Thus, invasive black rats with their broad diet spanning the green and detrital food webs (i.e., multiâ€�channel omnivory: [Wolkovich etÂ al.,Â <a href="#ece37171-bib-0068">2014</a>]) may directly and indirectly impact key processes within invaded ecosystems.</p> <p>Invasive black rats can directly and indirectly impact the ecology and food web links for native and introduced bird species in Hawaii (FigureÂ <a href="#ece37171-fig-0001">1</a>). Estimates of nest predation by black rats in Hawaii have ranged from high (e.g., 87% [Stone et al., <a href="#ece37171-bib-0056">1984</a>] to extremely low (e.g., 4% [Amarasekare,Â <a href="#ece37171-bib-0001">1993</a>]), indicating that black rats can have direct negative effects on reproductive success of native birds via predation, but this may vary by location. Rats can also have indirect effects on native and introduced birds by altering their foraging behavior and vertical habitat use, in turn impacting arthropod community biomass in the upper canopy (Wilson Rankin etÂ al.,Â <a href="#ece37171-bib-0067">2018</a>). Determining if invasive rats are preying upon birds, consuming birds as carrion, or not consuming birds will help ascertain the extent of the direct and indirect effects the rats are having on bird communities. If rats are directly consuming eggs, nestlings, fledglings or healthy adult birds, then such consumption could have a devastating direct effect on bird populations. If rats are consuming birds as carrion or rarely consuming birds in general, then the direct impact of rats on bird populations will be little to none (Fukami etÂ al.,Â <a href="#ece37171-bib-0024">2006</a>), although negative indirect impacts are still likely to occur (Wilson Rankin etÂ al.,Â <a href="#ece37171-bib-0067">2018</a>). Quantifying interactions among native and introduced species is critical to predicting the longâ€�term impacts of biological invasions and to understanding mechanisms for coexistence in these modified communities.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Invasive black rats (<i>Rattus rattus</i>) can directly and indirectly affect the ecology of native and introduced birds in Hawaii, such as the (a) native Ê»Å�maÊ»o (<i>Myadestes obscurus</i>) and the (b) introduced Japanese whiteâ€�eye (<i>Zosterops japonicus</i>) through modification of food web links. Photo credit: Jack Jeffrey</p>   </div> <p>We developed a novel threeâ€�part approach to identify potentially informative biomarkers of diet item decay status, determine if invasive black rats (<i>R. rattus</i>; GI samples) consumed birds, and to determine if the biomarkers could distinguish bird predation from bird scavenging (i.e., carrion consumption; FigureÂ <a href="#ece37171-fig-0002">2</a>). First, we identified bacterial taxa that were associated with either fresh or decaying chicken tissue. Second, we determined whether rats had consumed birds using PCRâ€�based diet analysis. Third, we identified bacterial taxa that were both (a) detected only in rats that had consumed birds and (b) were indicative of fresh or decaying tissue in the experimental chicken model. We collected both rat stomachs and rat feces to validate that rat fecal samples, both less invasive and timeâ€�intensive, would provide similar biological conclusions as stomach samples. We present evidence that bacterial biomarkers, linked to the successional stage of bird carcass decay, can persist as biomarkers through digestion to be measurable as biomarkers in both rat stomachs and noninvasive fecal material. Properly replicated through time and space and calibrated for use in a local system, such forensic genomic tools can detect and quantify species interactions that are remote, rare, and highly challenging to measure.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Conceptual figure of our threeâ€�part approach to identify and validate informative bacterial biomarkers of bird consumption and carcass decay status (that are not part of the native rat GI microbiome). Asterisk indicates the potential to use informative bacterial taxa to resolve ambiguous status as to whether the rat consumed a bird or not</p>   </div> </div> <div class="methods" title="methods"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
METHODS</div> <div class="studylocation" title="sec"><span class="label" tagx="label" title="label">2.1</span><div class="title" tagx="title" title="title">
Study location</div> <p>We conducted a chicken (<i>Gallus gallus domesticus</i>) tissue decomposition study and collected black rat (<i>Rattus rattus</i>) fecal and stomach samples from rat traps in forest fragments, locally known as â€œkÄ«pukaâ€�. Hundreds of kÄ«puka were created by historical lava flows in 1855 and 1881 (Vaughn etÂ al.,Â <a href="#ece37171-bib-0062">2014</a>). The kÄ«puka we sampled are located on the NE slope of Mauna Loa Volcano in the Upper Waiakea Forest Reserve on the Island of Hawaii (19Â°40â€²N 155Â°20â€²W, 1,470â€"1,790Â m elevation). These wellâ€�replicated fragments vary in size, yet they share the same soils, origin, and a single dominant forest canopy species, â€˜Å�hiâ€˜a lehua (<i>Metrosideros polymorpha</i>). Both native and nonâ€�native animals inhabit these forests (Gruner,Â <a href="#ece37171-bib-0026">2004</a>), including birds endemic and introduced (TableÂ <a href="#ece37171-sup-0001">S1</a>) to Hawaii (Flaspohler etÂ al.,Â <a href="#ece37171-bib-0022">2010</a>). Hawaii has only one native terrestrial mammal <i>Aeorestes semotus</i> (Hawaiian hoary bat). All other terrestrial mammals present in the study system are nonâ€�native, including <i>Rattus exulans</i> (Polynesian rat), <i>Rattus norvegicus</i> (Norway rat), <i>Mus musculus</i> (house mouse), <i>Herpestes javanicus</i> (Asian mongoose) and the most commonly encountered, <i>Rattus rattus</i> (black rat). Of these carnivorous and omnivorous species, only <i>R. rattus</i> demonstrably climbs trees and forages in forest canopies with regularity (Shiels etÂ al.,Â <a href="#ece37171-bib-0055">2014</a>; Vanderwerf,Â <a href="#ece37171-bib-0061">2012</a>; Wilson Rankin etÂ al.,Â <a href="#ece37171-bib-0067">2018</a>).</p> </div> <div class="identifybacterialbiomarkersfromdecayingchickentissue" title="sec"><span class="label" tagx="label" title="label">2.2</span><div class="title" tagx="title" title="title">
Identify bacterial biomarkers from decaying chicken tissue</div> <p>To quantify microbial decay of bird tissue, we sampled decomposing chicken tissue at day 0 in the lab, and days 0, 1, 2, 4, 7, and 11 in the field at 10 locations (TableÂ <a href="#ece37171-sup-0001">S2</a>). First, we obtained a whole chicken carcass labeled antibiotic free from a grocery store (KTA Super Stores, Hilo HI). We split the chicken breast tissue into 70 equivalent samples weighing approximately 6Â g on a sterile work surface in the laboratory and surface sterilized the chicken samples with 10% bleach. We created 10 sets of seven tissues samples and randomly assigned each set of samples to one of ten locations. Prior to leaving the lab, we immediately collected one sample from each assigned location and those served as the lab control samples (Day 0 lab; <i>n</i>Â =Â 10). In the field, in two separate kÄ«puka, we placed the remaining six samples at each of ten specified forested locations. These kÄ«puka were a subset of the larger study on the interactive effects of predation and ecosystem size on kÄ«puka food webs (spanning 34 kÄ«puka within a ~35Â km<sup>2</sup> range; Knowlton etÂ al.,Â <a href="#ece37171-bib-0031">2017</a>; Wilson Rankin etÂ al.,Â <a href="#ece37171-bib-0067">2018</a>). The two kÄ«puka for this chicken decomposition study were chosen as easily accessible, of intermediate size (3.19 and 2.77Â ha for K18 and K19 respectively) and had rats present (rat removal controls in larger study). We chose to do the decomposition in the field to determine bacteria as biomarkers for decomposition in the same habitat as where the rats and birds occur. We placed each chicken tissue sample on a sterile piece of tin foil in a cleaned, closed rat trap (Havahart<sup>Â®</sup>) secured to the ground using clean, new tent stakes. Samples were placed in traps to avoid loss of experimental units during the decomposition experiment by rats. No ground scavenging insects were observed at the field sites, nor any evidence of such scavenging of tissue pieces. Immediately after placing the samples in the trap, we collected one sample from each location and those served as the field control samples (Day 0 field; <i>n</i>Â =Â 10). The remaining samples were collected during subsequent visits at Days 1, 2, 4, 7, and 11. We wore a fresh pair of sterile nylon gloves to collect each sample and placed it into 2Â ml tubes in Queens College Buffer (20% DMSO, 0.25Â M EDTA, 100Â mM Tris, pH 7.5, saturated with NaCl; Amos etÂ al.,Â <a href="#ece37171-bib-0002">1992</a>). All vials (<i>N</i>Â =Â 70) were shipped to the Center for Conservation Genomics (CCG; National Zoological Park, Washington, DC) and stored in a âˆ’20Â°C freezer until analysis.</p> <p>We conducted 454 pyrosequencing of 16S rRNA gene amplicons to characterize bacterial communities of the decomposing chicken tissue samples. We cut the tissue sample into 5Â mmÂ Ã—Â 5Â mm sections using scissors and tweezers sterilized with bleach and ethanol between each sample. We extracted DNA from 70 chicken tissue samples using the BioSprint 96 DNA kit (Qiagen) following the manufacturer's protocol for tissue extractions with an overnight incubation in lysis buffer and included negative extraction controls. We followed previously published methods (Muletz Wolz etÂ al.,Â <a href="#ece37171-bib-0039">2018</a>) to amplify the V3â€�V5 16S rRNA gene region with the universal gene primer set 515F and 939R and to conduct library preparation. Each 25â€�Î¼l PCR reaction consisted of 1.25Â U of AmpliTaq Gold DNA Polymerase (Thermo Fisher), 2.5Â Î¼M MgCl, 200Â nM dNTPs, 200Â nM reverse fusion primer, 400Â nM forward barcoded fusion primer and 3Â Î¼l DNA template. PCR conditions were 95Â°C for 7Â m, followed by 30 cycles of 95Â°C for 45Â s, 55Â°C for 30Â s, 72Â°C for 45Â s and a final extension (72Â°C for 7Â m). We used Speedâ€�beads (in a PEG/NaCl buffer; Rohland &amp;amp; Reich,Â <a href="#ece37171-bib-0051">2012</a>) to clean postâ€�PCR products, and pooled samples equimolar based on Qubit (Invitrogen) DNA concentration values. We conducted highâ€�throughput sequencing of samples using one Roche 454 GS Junior run and one GS FLXÂ +Â run. Negative extraction controls and negative PCR controls were run with each 454 sequencing run to monitor for and then remove from analyses potential contaminant bacteria introduced in the laboratory.</p> <p>We used R (Râ€�Coreâ€�Team,Â <a href="#ece37171-bib-0049">2019</a>) and â€˜dada2â€™ package (Callahan etÂ al.,Â <a href="#ece37171-bib-0009">2016</a>) to process the 454 reads following default parameters for 454 data, and to generate amplicon sequence variants (ASVs). We filtered the data to only contain ASVs that occurred at least once in at least two samples (i.e., filtered singletons). Four of the 70 samples (from Lab Day 0 and Field Day 0 samples) did not yield any sequences and were not included in analyses. To identify ASVs associated with particular sampling days across sites, we merged samples by Day across the ten replicate sites (see resultsâ€"no significant differences in microbiome composition among sites were observed). We identified ASVs that we termed â€˜fresh tissue biomarkersâ€™, defined as ASVs present in only Day 0 lab, Day 0 field, and/or Day 1. We identified ASVs that we termed â€˜decayed tissue biomarkersâ€™, defined as ASVs present only in Days 2, 4, 7, and/or 11.</p> </div> <div class="fieldmethodsforcapturingrats" title="sec"><span class="label" tagx="label" title="label">2.3</span><div class="title" tagx="title" title="title">
Field methods for capturing rats</div> <p>As part of a larger study on the interactive effects of predation and ecosystem size on kÄ«puka food webs (spanning 34 kÄ«puka within a ~35Â km<sup>2</sup> range), rats were removed from 16 kÄ«puka, while 18 other kÄ«puka served as control plots (Knowlton etÂ al.,Â <a href="#ece37171-bib-0031">2017</a>; Wilson Rankin etÂ al.,Â <a href="#ece37171-bib-0067">2018</a>). In rat removal kÄ«puka, Victor M326 Pro Rat snap traps were set out and baited with peanut butter or coconut to quickly and ethically kill rodents upon entry (Stanford IACUC, no. 1776). Freshly killed rats (<i>n</i>Â =Â 23 from 10 kÄ«puka in current study; TableÂ <a href="#ece37171-sup-0001">S3</a>) were dissected and their entire stomach was preserved in 100% ethanol in the field. All equipment was disinfected with 10% bleach between dissections. In control kÄ«puka, two feeding stations (Black Trakkaâ"¢, Gotcha Traps Limited) per hectare were set out and baited with peanut butter. Pilot studies conducted in 2012 demonstrated that each trap night yields 4â€"15 fecal pellets. One fecal pellet was preserved in 100% ethanol and frozen until processing (<i>n</i>Â =Â 28 from 8 kÄ«puka in current study; TableÂ <a href="#ece37171-sup-0001">S3</a>).</p> </div> <div class="dietanalysistodetermineifratsconsumedbird" title="sec"><span class="label" tagx="label" title="label">2.4</span><div class="title" tagx="title" title="title">
Diet analysis to determine if rats consumed bird</div> <p>We extracted DNA from rat fecal and stomach samples using the Qiagen QIAamp DNA stool mini kit with modifications from manufacturer's protocol following previously published methods (Eggert etÂ al.,Â <a href="#ece37171-bib-0020">2005</a>; Wilbert etÂ al.,Â <a href="#ece37171-bib-0065">2015</a>) to increase DNA yields from fecal samples. We included a negative extraction control with each set of extractions. We used two independent birdâ€�specific primer pairs targeting conserved regions of the mitochondrial Cytochrome Oxidase I (COI) and Cytochrome b (Cytb) genes to determine if rats had consumed bird.</p> <p>We analyzed a total of 23 rat stomach samples and 28 rat fecal samples, which were collected from different individuals. We used BIRDF1/AWCintR2 primer pair, which targets a ~350Â bp region in the COI gene (Zarzosoâ€�Lacoste etÂ al.,Â <a href="#ece37171-bib-0071">2013</a>). BIRDF1/AWCintR2 was selected as most relevant from a wider range of primers based on similar types of samples (Zarzosoâ€�Lacoste etÂ al.,Â <a href="#ece37171-bib-0071">2013</a>, <a href="#ece37171-bib-0070">2016</a>). We used CytbCorL (5â€²â€�ACTGCGACAAAATCCCATTCâ€�3â€²) and CytbCor3 (5â€²â€�GACTCCTCCTAGTTTATTTGGGâ€�3â€²), which targets a ~233Â bp region of the Cytb gene and was designed to target corvid DNA, but also previously amplified babbler DNA (Renner et al., <a href="#ece37171-bib-0050">2008</a>). Each 25â€�Î¼l PCR reaction consisted of 1.25Â U of AmpliTaq Gold DNA Polymerase (ThermoFisher), 1.5Â Î¼M MgCl, 200Â nM dNTPs, 200Â nM reverse primer, 200Â nM forward primer, 20Â ng/Î¼l BSA and 3Â Î¼l DNA template. PCR conditions were 95Â°C for 10Â m, followed by 40 cycles of 95Â°C for 15, 30Â s at the annealing temperature (57Â°C for BirdF1/AwCintR2 and 51Â°C for CytbCorL/CytbCor3 respectively), 72Â°C for 45Â s and a final extension (72Â°C for 5Â m). Any PCR products within the appropriate size range visualized via gel electrophoresis were then cleaned using ExoSAPâ€�IT (United States Bioâ€�chemical), and Sanger sequenced from the forward primer direction using the BigDye Terminator Cycle Sequencing Kit (Applied Biosystems, Inc). The sequenced products were column filtered, dried down, rehydrated with 10Â Î¼l of HPLC purified formamide, and then analyzed on an Applied Biosystems 3130xl DNA Analyzer. Individual sequences were assembled and edited in Sequencher 5.1 (GeneCodes). We sequenced 45 putative bird positive samples from BIRDF1/AWCintR2 primer pair and 23 putative bird positive samples from CytbCorL/CytbCor3 primer pair.</p> <p>Samples were assigned a category of â€˜consumed birdâ€™, â€˜not consumed birdâ€™, or â€˜ambiguousâ€™ based on PCR results and Sanger sequencing. Every sample was tested at least twice, once with the COI primer set and once with the Cytb primer set. We considered samples as â€˜consumed birdâ€™ if at least one of the bird primer sets produced a highâ€�quality sequence (&amp;gt;50% quality) from Sanger sequencing, and that sequence matched a bird species in GenBank that we would expect to detect in our study site (TableÂ <a href="#ece37171-sup-0001">S1</a>). We blasted individual sequences in NCBI using BLASTN 2.9.0+. When all top matches (eâ€�valueÂ <div class="â_UNKNOWN">
95%) we recorded that bird species as the bird species consumed by the rat. We considered a sample as â€˜not consumed birdâ€™ if both bird primer sets did not amplify DNA in either of the COI and Cytb PCR reactions. We replicated the PCRs for the â€˜not consumed birdâ€™ for each primer set a second time to reduce the probability of false negatives. The negative samples were all tested a total of four times (two replicates per primer pair). We considered a sample as â€˜ambiguousâ€™ if at least one bird primer set produced a PCR band of appropriate target bp size, but we were unable to get a highâ€�quality sequence matching a bird sequence from Sanger sequencing after replicate sequencing attempts. Ambiguous samples were either (a) repeatedly lowâ€�quality sequences, (b) of mixed sequences, or (c) the sequence was of high quality, but matched a nonâ€�bird taxon (often earthworms and moths). Ambiguous samples were excluded when identifying informative biomarkers in the rat GI microbiome (see <i>biomarker and microbiome matching</i> subsection below).</div></p> <p>We verified that the nine bird species found in the study sites (TableÂ <a href="#ece37171-sup-0001">S1</a>) had sequences in GenBank for either the COI gene, the Cytb gene, or both by custom searches. Two bird species (<i>C. sandwichensis</i>, <i>L. leucomelanos</i>) did not have reference sequences available for COI, but the <i>Lophura</i> genus had reference sequences. All bird species had reference sequences for the Cytb gene. In the â€˜consumed birdâ€™ samples, we did not find evidence of multiple sequencing (i.e., superposition of several sequences from different species in the same sample).</p> </div> <div class="microbiomesofratfecesandratstomachs(ratgimicrobiome)" title="sec"><span class="label" tagx="label" title="label">2.5</span><div class="title" tagx="title" title="title">
Microbiomes of rat feces and rat stomachs (rat GI microbiome)</div> <p>We conducted Illumina highâ€�throughput sequencing of 16S rRNA gene amplicons to characterize bacterial communities in rat feces and rat stomachs. We amplified DNA using the same primers as used for the chicken samples (515F, 939R). We used a twoâ€�step library preparation to generate 16S rRNA amplicons for sequencing. First, we performed duplicate PCR reactions for each sample, and included negative extraction controls and negative PCR controls. Each 25â€�Î¼l amplicon PCR reaction consisted of 1.25Â U of AmpliTaq Gold DNA Polymerase (ThermoFisher), 1.5Â Î¼M MgCl, 200Â nM dNTPs, 500Â nM reverse primer with overhang adapter, 500Â nM forward primer with overhang adapter, 20Â ng/Î¼l BSA and 3Â Î¼l DNA template. PCR conditions were 95Â°C for 7Â m, followed by 30 cycles of 95Â°C for 45Â s, 55Â°C for 30Â s, 72Â°C for 45Â s and a final extension (72Â°C for 7Â m). Then, we performed index PCR, adding custom i5 and i7 adaptors to the PCR amplicons to uniquely identify each sample. Each 50â€�Î¼l index PCR reaction consisted of 25Â Î¼l of 2Ã— Phusion Hot Start II HF Master Mix, 1Â mM i5 primer, 1Â mM i7 primer, and 5Â Î¼l amplicon PCR DNA template. PCR conditions were 98Â°C for 2Â m, followed by eight cycles of 98Â°C for 20Â s, 62Â°C for 30Â s, 72Â°C for 30Â s and a final extension (72Â°C for 2Â m). Between each PCR reaction, PCR products were cleaned with Speedâ€�beads (Rohland &amp;amp; Reich,Â <a href="#ece37171-bib-0051">2012</a>). We quantified samples using a Qubit Fluorometer (Invitrogen), and then pooled them equimolar to make a final library pool. We conducted size selection for the target band using Eâ€�Gel EX 2% agarose (Invitrogen) and Qiagen QIAquick Gel Extraction kit. We conducted Illumina MiSeq highâ€�throughput sequencing on one sequencing run (v3 chemistry: 2Â Ã—Â 300Â bp kit) to characterize the bacterial communities of each sample.</p> <p>We quality filtered and processed sequence reads using the program Quantitative Insights Into Microbial Ecology 2 (vQIIME2â€�2018.8; Bolyen etÂ al.,Â <a href="#ece37171-bib-0008">2019</a>). We imported demultiplexed reads from the Illumina MiSeq and filtered them using the following criteria: â€�â€�pâ€�truncâ€�lenâ€�f 270 â€�â€�pâ€�truncâ€�lenâ€�r 200 â€�â€�pâ€�trimâ€�leftâ€�f 19 â€�â€�pâ€�trimâ€�leftâ€�r 23. Sequences were then categorized into ASVs via the dada2 pipeline (Callahan etÂ al.,Â <a href="#ece37171-bib-0009">2016</a>) and taxonomy was assigned by aligning ASVs with the Greengenes 13_8 99% database (DeSantis etÂ al.,Â <a href="#ece37171-bib-0017">2006</a>) using a NaÃ¯ve Bayes classifier trained on the 515F/939R region. A phylogenetic tree was built using the fasttree algorithm (Price etÂ al.,Â <a href="#ece37171-bib-0048">2010</a>). These files were imported into R using the package â€˜phyloseqâ€™ (McMurdie &amp;amp; Holmes,Â <a href="#ece37171-bib-0035">2013</a>), where all subsequent analyses were conducted. We filtered the data to only contain ASVs that occurred at least once on at least two samples (i.e., filtered singletons). One fecal sample (Pâ€�173A) had low sequence coverage (1,012 sequences) and was not included in analyses.</p> </div> <div class="biomarkerandmicrobiomematching:informativebacterialtaxaofbirdconsumptionanddecaystatus" title="sec"><span class="label" tagx="label" title="label">2.6</span><div class="title" tagx="title" title="title">
Biomarker and microbiome matching: informative bacterial taxa of bird consumption and decay status</div> <p>We compared the sequences of our bacterial biomarker to the sequences of bacteria detected in rat fecal and stomach microbiomes to identify which bacteria were both biomarkers and also present in the GI tract of rats. We used a custom blast analysis in Geneious 8.1 to query our biomarker sequences against the microbiome sequences (Muletzâ€�Wolz etÂ al.,Â <a href="#ece37171-bib-0040">2017</a>). We used a megablast program, having Geneious return results as queryâ€�centered alignment data only and returning only the top hit. We considered a rat GI microbiome ASV as a match to a biomarker ASV if they matched at â‰¥97% sequence similarity. We caution that this does not necessarily mean these bacteria are the same species or strains, only that they are strong candidates particularly at â‰¥99% sequence similarity (which a majority of our biomarkers areâ€"see in results TableÂ <a href="#ece37171-tbl-0002">2</a>).</p> <p>We identified which bacterial biomarkers were only detected in rat samples that showed evidence of bird consumption. We did so to eliminate any potential biomarkers that may be part of the resident GI microbiome (i.e., present in rats that did not consume birds), and therefore uninformative. We subset our rat fecal and stomach microbiome data to (a) only contain ASVs that matched bacterial biomarkers, and (b) only contain rats that we had unambiguously determined to have consumed bird (<i>n</i>Â =Â 20) or to have not consumed bird (<i>n</i>Â =Â 11) from diet analysis (see <i>diet analysis of rat feces and rat stomachs</i> above). We exported a site (rat) by species (ASV) matrix from R of this subset data, and manually identified which biomarkers were only present in rat samples that showed evidence of bird consumption. We only considered biomarkers as informative when they were present in rats that consumed bird and absent from rats that did not consume bird.</p> </div> <div class="statisticalanalyses" title="sec"><span class="label" tagx="label" title="label">2.7</span><div class="title" tagx="title" title="title">
Statistical analyses</div> <p>All statistical analyses were performed in R version 3.5.3 (Râ€�Coreâ€�Team,Â <a href="#ece37171-bib-0049">2019</a>). In the rat microbiome dataset, we quantified microbiome structure with (a) two measures of alpha diversity (ASV richness and Faith's phylogenetic diversity [Faith's PD]), (b) three measures of beta diversity (Jaccard, unweighted UniFrac, and Brayâ€"Curtis), and (c) two measures of bacterial relative abundance using sequence counts (two taxonomic levels: bacterial ASV and bacterial phylum). Prior to conducting Brayâ€"Curtis analyses, we performed proportion normalization on the raw sequence counts (i.e., total sum scaling) to correct for biases associated with unequal sequencing depth on this abundanceâ€�weighted measures (McMurdie &amp;amp; Holmes,Â <a href="#ece37171-bib-0036">2014</a>; Weiss etÂ al.,Â <a href="#ece37171-bib-0064">2017</a>). Variation in sequencing depth was approximately 16x. Therefore, we included sequence count as an additional explanatory variable in our alpha and beta diversity analyses (Weiss etÂ al.,Â <a href="#ece37171-bib-0064">2017</a>) to account for variation in sequencing depth among samples and to avoid statistical issues associated with sequence count rarefaction (McMurdie &amp;amp; Holmes,Â <a href="#ece37171-bib-0036">2014</a>). If sequencing depth was significant, we performed analyses on a rarefied dataset to verify that sequencing depth was not driving the significance of the biological effect. We found that sequencing depth did not affect our statistical results or biological inference, and therefore we only report the statistics for the raw sequence data (nonâ€�rarefied) in the results. Our sequencing depth provided adequate sampling of the community (FigureÂ <a href="#ece37171-sup-0001">S1</a>).</p> <p>We analyzed rat stomach versus rat fecal samples to determine how the two sample types varied in microbiome structure. To determine if alpha diversity differed between sample types, we conducted ANOVAs with ASV richness or Faith's PD as the response variable and rat sample type as the explanatory variable. We used the <i>ANOVA</i> function in the â€˜carâ€™ package with type II sums of squares to determine significance. To determine if bacterial community composition (beta diversity) differed between sample type, we conducted PERMANOVAs with Jaccard, unweighted UniFrac or Brayâ€"Curtis distance as the response variable and rat sample type as the explanatory variable. We verified that beta dispersion was similar between sample types and not influencing the PERMANOVA results by conducting a PERMDISP. To determine if bacterial relative abundance differed between sample types, we used the package â€˜DAtestâ€™ to first rank various statistical methods used to test for differential abundance (Russel etÂ al.,Â <a href="#ece37171-bib-0053">2018</a>). We filtered low abundance ASVs (present in &amp;lt;7 samples) using the function <i>preDA</i>. Then, we input the raw sequence counts of the filtered ASV table (or phylum level table with no filtering) and used the function <i>testDA</i> to allow each statistical method to perform its default transformation on the data. We used the top two differential abundance tests that had the lowest False Positive Rate (Russel etÂ al.,Â <a href="#ece37171-bib-0053">2018</a>) in our statistical analyses with bacterial abundance at two taxonomic levels (bacterial ASV or bacterial phylum) as the response variable and rat sample type as the explanatory variable. We report only ASVs or phyla that were significant (after being corrected for multiple comparisons) in both of the top ranked differential abundance tests (in our analysesÂ =Â quasiâ€�poisson generalized linear model [function <i>DA.qpo</i>] and Welch <i>t</i>â€�test [function <i>DA.ttt</i>]).</p> <p>We determined if microbiome structure could predict whether a rat consumed a bird. We subset our rat GI microbiome dataset to only include rats that we had unambiguously determined to have consumed bird (<i>n</i>Â =Â 20) or to have not consumed bird (<i>n</i>Â =Â 11) from diet analysis. To determine if ASV richness or Faith's PD was predictive of bird consumption, we used a generalized linear model with a binomial distribution with bird in diet (yes or no) as the response variable and the alpha diversity metric (ASV richness or Faith's PD), sample type and their interaction as response variables. For beta diversity, we were unable to find a statistical test that can use a distance matrix as an explanatory variable. Therefore, we determined if bacterial community composition (beta diversity response variable: Jaccard, unweighted UniFrac or Brayâ€"Curtis distance) differed between rats that consumed birds and those that did not and if sample type mattered (bird in diet, sample type, and their interaction as explanatory variables). Finally, we conducted indicator species analysis to see if particular bacterial ASVs or particular bacterial genera (table merged at the genus level) could differentiate rat samples based on bird consumption. We filtered the ASV table to only contain bacterial taxa found in at least 15% of samples to reduce the number of comparisons and likelihood for false positives (ASVs <i>n</i>Â =Â 124). We used the <i>multipatt</i> function in the â€˜indicspeciesâ€™ package (De Caceres &amp;amp; Legendre,Â <a href="#ece37171-bib-0015">2009</a>) on a presenceâ€�absence matrix. We corrected for multiple comparisons using false discovery rate corrections. We considered an ASV as an indicator species if it had a <i>p</i>Â <div class="â_UNKNOWN" href="#ece37171-bib-0003">
2015; Castroâ€�Luna etÂ al.,Â <a href="#ece37171-bib-0011">2007</a>). An indicator value of 1 indicates that the ASV was observed in all the samples from one group and completely absent from the other group, while an indicator value of 0 indicates that the ASV was widely distributed across both groups.</div></p> </div> </div> <div class="results" title="results"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
RESULTS</div> <div class="identificationofbacterialbiomarkersfromdecayingchickentissue" title="sec"><span class="label" tagx="label" title="label">3.1</span><div class="title" tagx="title" title="title">
Identification of bacterial biomarkers from decaying chicken tissue</div> <p>We generated 215,369 highâ€�quality sequences from 66 chicken tissue samples. Average sequencing depth per biological sample was 3,193 sequences (minÂ =Â 5, maxÂ =Â 12,338). Many of the early sampling days (Day 0 lab, Day 0 field, Day 1) had low sequence coverage likely due to few microbes colonizing the freshly bleached tissue. We identified 79 ASVs present on chicken tissue belonging to three bacterial phyla (Proteobacteria, Bacteroidetes, Firmicutes), with 99% of sequences belonging to the Proteobacteria phylum. The majority of bacterial biomarker ASVs belonged to the genus <i>Pseudomonas</i> within the phylum Proteobacteria (FigureÂ <a href="#ece37171-sup-0001">S2</a>, 87% of sequences).</p> <p>We found similar microbiome composition on decomposing chicken tissue among the ten replicate sites (Jaccard PERMANOVA: <i>p</i>Â =Â .291; Brayâ€"Curtis: <i>p</i>Â =Â .37), but differences across sampling days (FigureÂ <a href="#ece37171-sup-0001">S2</a>). Site replicates were pooled together to identify fresh tissue biomarkers (present in only Day 0 lab, Day 0 field, and/or Day 1) and decayed tissue biomarkers (present only in Days 2, 4, 7, and/or 11). We identified 44 ASVs of the 70 chickenâ€�tissueâ€�associated ASVs as biomarkers. Two ASVs we identified as fresh tissue bacterial biomarkers, and 42 ASVs we identified as decayed tissue bacterial biomarkers (FigureÂ <a href="#ece37171-fig-0003">3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 3</span></a></div>   <p>Biomarkers identified from decaying chicken tissue sampled over 11Â days. We identified two bacterial ASVs as fresh tissue bacterial biomarkers and 42 ASVs as decayed tissue bacterial biomarkers</p>   </div> </div> <div class="dietanalysistodetermineifratsconsumedbird" title="sec"><span class="label" tagx="label" title="label">3.2</span><div class="title" tagx="title" title="title">
Diet analysis to determine if rats consumed bird</div> <p>We found that the rat samples we collected included rats that had consumed birds recently and rats that had not consumed bird recently (TableÂ <a href="#ece37171-tbl-0001">1</a>). We identified 20 rat samples that contained bird DNA (<i>n</i>Â =Â 7 feces, <i>n</i>Â =Â 13 stomachs). We identified 11 rat samples that contained no bird DNA (<i>n</i>Â =Â 9 feces, <i>n</i>Â =Â 2 stomachs). We had 20 samples that we could not unambiguously confirm the presence or absence of bird DNA; we considered these samples as ambiguous, and excluded these samples from when we were identifying the putative biomarkers. All sequences and their associated data are provided at <a href="https://doi.org/10.6084/m9.figshare.13274816">https://doi.org/10.6084/m9.figshare.13274816</a>.</p> <div class="table-wrap_UNKNOWN" id="ece37171-tbl-0001" content-type="TABLE" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p>Detection of bird DNA in rat samples</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Sample type</th> <th align="left" valign="top" rowspan="1" colspan="1">Bird in diet</th> <th align="left" valign="top" rowspan="1" colspan="1">Sample size (<i>n</i>)</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">feces</td> <td align="left" rowspan="1" colspan="1">Ambiguous</td> <td align="char" char="." rowspan="1" colspan="1">12</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">feces</td> <td align="left" rowspan="1" colspan="1">No</td> <td align="char" char="." rowspan="1" colspan="1">9</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">feces</td> <td align="left" rowspan="1" colspan="1">Yes</td> <td align="char" char="." rowspan="1" colspan="1">7</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1">Ambiguous</td> <td align="char" char="." rowspan="1" colspan="1">8</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1">No</td> <td align="char" char="." rowspan="1" colspan="1">2</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1">Yes</td> <td align="char" char="." rowspan="1" colspan="1">13</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">TotalÂ =</td> <td align="char" char="." rowspan="1" colspan="1">51</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="ece37171-ntgp-0002"> <div class="title" tagx="title" title="title">
Note</div> <div class="fn-type-" title=""> <p>Fecal and stomach samples were collected from different rats.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> </div> <div class="characterizationofratgimicrobiome" title="sec"><span class="label" tagx="label" title="label">3.3</span><div class="title" tagx="title" title="title">
Characterization of rat GI microbiome</div> <p>We generated 1,230,163 highâ€�quality sequences from 51 rat samples (<i>n</i>Â =Â 28 fecal samples, <i>n</i>Â =Â 23 stomach samples). Average sequencing depth per biological sample was 24,121 sequences (minÂ =Â 4,762, maxÂ =Â 74,921). We identified 987 ASVs present in the rat GI tract belonging to 11 bacterial phyla (FigureÂ <a href="#ece37171-fig-0004">4c</a>). The majority of ASVs belonged to the phyla Proteobacteria, Firmicutes and Bacteroidetes. Stomach and fecal samples cumulatively had 987 bacterial ASVs, with 407 ASVs detected in both sample types. The shared ASVs made up 84.5% of the sequences from stomach samples and 67.7% of sequences from fecal samples, indicating that many ASVs were shared between sample types.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 4</span></a></div>   <p>Microbiome structure by rat sample type. Fecal samples had (a) higher bacterial ASV richness and (b) dissimilar community composition (UniFrac distances shown here with 95% confidence ellipses: shapes indicative of bird in diet status) from stomach samples. The relative abundance of bacterial phyla was generally similar (c), except Bacteroidetes was higher in feces compared to stomach samples</p>   </div> <p>Microbiome structure generally differed between rat sample type. Rat fecal samples had higher bacterial ASV richness (FigureÂ <a href="#ece37171-fig-0004">4a</a>, ANOVA: <i>F</i><sub>1,49</sub>Â =Â 9.82, <i>p</i>Â =Â .003) and higher phylogenetic diversity (Faith's PD ANOVA: <i>F</i><sub>1,49</sub>Â =Â 7.02, <i>p</i>Â =Â .011) than rat stomach samples. Rat feces and rat stomachs differed in bacterial community composition (FigureÂ <a href="#ece37171-fig-0004">4b</a>, Jaccard PERMANOVA: Pseudoâ€� <i>F</i><sub>1,49</sub>Â =Â 4.30, <i>R</i><sup>2</sup>Â =Â 8%, <i>p</i>Â =Â .001; UniFrac: Pseudoâ€� <i>F</i><sub>1,49</sub>Â =Â 7.69, <i>R</i><sup>2</sup>Â =Â 14%, <i>p</i>Â =Â .001; Brayâ€"Curtis: Pseudoâ€� <i>F</i><sub>1,49</sub>Â =Â 2.85, <i>R</i><sup>2</sup>Â =Â 6%, <i>p</i>Â =Â .001), but had similar dispersion within their respective communities (Jaccard PERMDISP: <i>p</i>Â =Â .2, UniFrac: <i>p</i>Â =Â .7, Brayâ€"Curtis: <i>p</i>Â =Â .1). Bacterial relative abundances were similar between fecal and stomach samples at the ASV and phylum level, except the phylum Bacteriodetes was higher in rat fecal samples than in stomach samples (FigureÂ <a href="#ece37171-fig-0004">4c</a>, Welch <i>t</i>â€�test <i>p</i>Â =Â .007, log2FC âˆ’1.72; Quasiâ€�poisson GLM <i>p</i>Â =Â .006, log2FC âˆ’2.62). There was variation in microbiome structure across kÄ«puka, but the pattern of higher bacterial ASV richness and phylogenetic diversity, distinct bacterial community composition, and higher Bacteriodetes in fecal versus stomach samples was generally maintained across the kÄ«puka within each sample type (FigureÂ <a href="#ece37171-sup-0001">S3</a>).</p> </div> <div class="informativebacterialtaxaofbirdconsumptionanddecaystatus" title="sec"><span class="label" tagx="label" title="label">3.4</span><div class="title" tagx="title" title="title">
Informative bacterial taxa of bird consumption and decay status</div> <p>We determined if the bacterial biomarkers that we identified from decaying chicken tissue (FigureÂ <a href="#ece37171-fig-0003">3</a>) were able to predict bird consumption and decay status. We detected several decayed tissue bacterial biomarkers in rat GI samples that had consumed bird, but did not detect any fresh tissue bacterial biomarkers. We detected 40 bacterial ASVs in the fecal and stomach microbiomes of rats that matched 22 decayed tissue bacterial biomarkers (matched â‰¥97% sequence similarity). We then eliminated the bacterial ASVs that were present in rat microbiome samples of rats that did not consume birds. In other words, we eliminated any bacterial biomarker that may have been resident microbial taxa in GI tract of rats. After this filtering step, we had 15 informative bacterial ASVs, which matched seven decayed tissue biomarkers and were found only in rats that had consumed bird (TableÂ <a href="#ece37171-tbl-0002">2</a>). Of the 20 rats we confirmed consumed birds through diet analysis, we identified nine rats that potentially consumed birds as carrion (3/7 fecal samples; 6/13 stomach samples) with our 15 informative bacterial ASVs (TableÂ <a href="#ece37171-tbl-0002">2</a>). Combining diet and biomarker analysis, we found that rats consumed four different bird species, including one native and three introduced birds, and likely consumed three of those bird species through scavenging (TableÂ <a href="#ece37171-tbl-0003">3</a>).</p> <div class="table-wrap_UNKNOWN" id="ece37171-tbl-0002" content-type="TABLE" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 2</span> <p>Bacterial ASVs detected on decaying chicken tissue that matched bacterial ASVs in GI microbiomes of rats that consumed birds</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Decayed chicken tissue biomarker ASV ID</th> <th align="left" valign="top" rowspan="1" colspan="1">No. tissue replicates detected in</th> <th align="left" valign="top" rowspan="1" colspan="1">Rat GI ASV ID</th> <th align="left" valign="top" rowspan="1" colspan="1">Phylum</th> <th align="left" valign="top" rowspan="1" colspan="1">Family</th> <th align="left" valign="top" rowspan="1" colspan="1">Genus</th> <th align="left" valign="top" rowspan="1" colspan="1">Species</th> <th align="left" valign="top" rowspan="1" colspan="1">Similarity to decayed tissue biomarker (%)</th> <th align="left" valign="top" rowspan="1" colspan="1">No. rats detected as consuming bird as carrion</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="2" colspan="1">ASV100 (Day 11)</td> <td align="left" rowspan="2" colspan="1">3</td> <td align="left" rowspan="1" colspan="1">e15086c6f06b9ab2db9544a6573e930d</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Pseudomonadaceae</td> <td align="left" rowspan="1" colspan="1"><i>Pseudomonas</i></td> <td align="left" rowspan="1" colspan="1"><i>viridiflava</i></td> <td align="char" char="." rowspan="1" colspan="1">100</td> <td align="char" char="." rowspan="1" colspan="1">1</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">93ae0cb4f1180c5bc85da5a5ad99d4a2</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Pseudomonadaceae</td> <td align="left" rowspan="1" colspan="1"><i>Pseudomonas</i></td> <td align="left" rowspan="1" colspan="1"><i>viridiflava</i></td> <td align="char" char="." rowspan="1" colspan="1">99</td> <td align="char" char="." rowspan="1" colspan="1">3<sup>#</sup></td> </tr> <tr> <td align="left" rowspan="4" colspan="1">ASV103 (Day 11)</td> <td align="left" rowspan="4" colspan="1">2</td> <td align="left" rowspan="1" colspan="1">bd298d360cf7e61a2e1d2abe51467853</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Enterobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="char" char="." rowspan="1" colspan="1">99</td> <td align="char" char="." rowspan="1" colspan="1">3<sup>#</sup></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">9e7246ddc0eae83c1a74a316fe72f2f1</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Enterobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="char" char="." rowspan="1" colspan="1">99</td> <td align="char" char="." rowspan="1" colspan="1">2<sup>#</sup></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">910acad922c911e9b2270f96326b9bb1</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Enterobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="char" char="." rowspan="1" colspan="1">99</td> <td align="char" char="." rowspan="1" colspan="1">1</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">12e77d709989ae4da784b662fce24432</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Enterobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="char" char="." rowspan="1" colspan="1">99</td> <td align="char" char="." rowspan="1" colspan="1">1<sup>+</sup></td> </tr> <tr> <td align="left" rowspan="3" colspan="1">ASV135 (Day 4)</td> <td align="left" rowspan="3" colspan="1">2</td> <td align="left" rowspan="1" colspan="1">f00f91d78f49cfbe68e6554f920ae5d8</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Enterobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="char" char="." rowspan="1" colspan="1">99</td> <td align="char" char="." rowspan="1" colspan="1">3<sup>#</sup></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">e15e067244d294ff4e0a4bb7f3149dc9</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Enterobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="char" char="." rowspan="1" colspan="1">100</td> <td align="char" char="." rowspan="1" colspan="1">1</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">813e7aaa81207e1cf91364ee5691a605</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Enterobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="char" char="." rowspan="1" colspan="1">98</td> <td align="char" char="." rowspan="1" colspan="1">3<sup>#</sup></td> </tr> <tr> <td align="left" rowspan="2" colspan="1">ASV142 (Day 4)</td> <td align="left" rowspan="2" colspan="1">2</td> <td align="left" rowspan="1" colspan="1"><i>df</i>8da8cdf673547d89adba6a22f7d2ee</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Enterobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>Ewingella</i></td> <td align="left" rowspan="1" colspan="1"><i>americana</i></td> <td align="char" char="." rowspan="1" colspan="1">98</td> <td align="char" char="." rowspan="1" colspan="1">3*<sup>#</sup></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">239c51edf1db3559615306d8952fe27e</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Enterobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="left" rowspan="1" colspan="1"><i>â€"</i></td> <td align="char" char="." rowspan="1" colspan="1">98</td> <td align="char" char="." rowspan="1" colspan="1">1</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">ASV147 (Day 4)</td> <td align="left" rowspan="1" colspan="1">2</td> <td align="left" rowspan="1" colspan="1">36d75a97bbb0e49db0d56538ce946b2a</td> <td align="left" rowspan="1" colspan="1">Firmicutes</td> <td align="left" rowspan="1" colspan="1">Carnobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>Carnobacterium</i></td> <td align="left" rowspan="1" colspan="1"><i>NA</i></td> <td align="char" char="." rowspan="1" colspan="1">97</td> <td align="char" char="." rowspan="1" colspan="1">1</td> </tr> <tr> <td align="left" rowspan="2" colspan="1">ASV54 (Day 4)</td> <td align="left" rowspan="2" colspan="1">3</td> <td align="left" rowspan="1" colspan="1">9cd5c63214c4582ee6d84e0bfd903c71</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Pseudomonadaceae</td> <td align="left" rowspan="1" colspan="1"><i>Pseudomonas</i></td> <td align="left" rowspan="1" colspan="1"><i>NA</i></td> <td align="char" char="." rowspan="1" colspan="1">99</td> <td align="char" char="." rowspan="1" colspan="1">3*<sup>#</sup></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">21301da854c631af0c945541ce91efb8</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Pseudomonadaceae</td> <td align="left" rowspan="1" colspan="1"><i>Pseudomonas</i></td> <td align="left" rowspan="1" colspan="1"><i>viridiflava</i></td> <td align="char" char="." rowspan="1" colspan="1">98</td> <td align="char" char="." rowspan="1" colspan="1">1</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">ASV73 (Day 11)</td> <td align="left" rowspan="1" colspan="1">2</td> <td align="left" rowspan="1" colspan="1">868c3e66c9baedb1bf2c6b9328abf35c</td> <td align="left" rowspan="1" colspan="1">Proteobacteria</td> <td align="left" rowspan="1" colspan="1">Enterobacteriaceae</td> <td align="left" rowspan="1" colspan="1"><i>Morganella</i></td> <td align="left" rowspan="1" colspan="1"><i>morganii</i></td> <td align="char" char="." rowspan="1" colspan="1">100</td> <td align="char" char="." rowspan="1" colspan="1">2<sup>#</sup></td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="char" char="." rowspan="1" colspan="1">Total no. unique rats =</td> <td align="char" char="." rowspan="1" colspan="1">9</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="ece37171-ntgp-0003"> <div class="title" tagx="title" title="title">
Note</div> <div class="fn-type-" title=""> <p>Day in parentheses is the first sampling time point since start of decomposition that biomarker was detected on the chicken tissues. Bacterial biomarkers that were detected in both rat stomachs and rat fecal samples are denoted by an asterisk (*), those only detected in rat feces are denoted by plus sign (<sup>+</sup>), and those detected in multiple kÄ«puka are denoted by a hash mark (<sup>#</sup>).</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> <div class="table-wrap_UNKNOWN" id="ece37171-tbl-0003" content-type="TABLE" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 3</span> <p>Results of Sanger sequence assignment to bird species based on NCBI blast and results of detection of informative bacterial biomarkers of carrion consumption in rat GI microbiome</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000" span="1" /></colgroup> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Primer pair</th> <th align="left" valign="top" rowspan="1" colspan="1">Sample ID</th> <th align="left" valign="top" rowspan="1" colspan="1">Sample type</th> <th align="left" valign="top" rowspan="1" colspan="1">Sequence result</th> <th align="left" valign="top" rowspan="1" colspan="1">Likely consumed as carrion</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="14" colspan="1">BirdF1/AwCintR2 (COI gene region)</td> <td align="left" rowspan="1" colspan="1">Pâ€�165A</td> <td align="left" rowspan="1" colspan="1">feces</td> <td align="left" rowspan="1" colspan="1"><i>Lophura leucomelanos*</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�619</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�620</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�623</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Myadestes obscurus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�624</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�626</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Lophura leucomelanos*</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�627</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�631</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�637</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�640</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�642</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�661</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�662</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�665</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="left" rowspan="12" colspan="1">CytbCorL/CytbCor3 (Cytb gene region)</td> <td align="left" rowspan="1" colspan="1">Pâ€�002A</td> <td align="left" rowspan="1" colspan="1">feces</td> <td align="left" rowspan="1" colspan="1"><i>Meleagris gallopavo</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Pâ€�003A</td> <td align="left" rowspan="1" colspan="1">feces</td> <td align="left" rowspan="1" colspan="1"><i>Meleagris gallopavo</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Pâ€�018A</td> <td align="left" rowspan="1" colspan="1">feces</td> <td align="left" rowspan="1" colspan="1"><i>Myadestes obscurus</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Pâ€�023A</td> <td align="left" rowspan="1" colspan="1">feces</td> <td align="left" rowspan="1" colspan="1"><i>Myadestes obscurus</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Pâ€�025A</td> <td align="left" rowspan="1" colspan="1">feces</td> <td align="left" rowspan="1" colspan="1"><i>Myadestes obscurus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Pâ€�166A</td> <td align="left" rowspan="1" colspan="1">feces</td> <td align="left" rowspan="1" colspan="1"><i>Myadestes obscurus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�619</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�623</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Myadestes obscurus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�626</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Lophura leucomelanos*</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�627</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�631</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">Râ€�12â€�637</td> <td align="left" rowspan="1" colspan="1">stomach</td> <td align="left" rowspan="1" colspan="1"><i>Zosterops japonicus</i></td> <td align="left" rowspan="1" colspan="1">Yes</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="ece37171-ntgp-0004"> <div class="title" tagx="title" title="title">
Note</div> <div class="fn-type-" title=""> <p>Asterisk denotes that sequences matched to <i>Lophura nycthemera</i> (not found in Hawaii) and was considered to indicate rat consumption of the <i>Lophura</i> species present in our study sites (<i>Lophura leucomelanos</i>), which does not have a reference COI sequence in GenBank.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> <p>As part of an exploratory analysis, we used the informative bacterial ASVs for decayed tissue consumption to suggest the bird consumption status for some of the â€˜ambiguousâ€™ rat samples from diet analysis. We had 20 rat samples (12 fecal and eight stomach samples) that we originally assigned as ambiguous for bird consumption through diet analysis. We suggest that five of those 20 rats (three fecal and two stomach samples) may have consumed birds as carrion; those rat samples contained at least two of the 15 informative bacterial ASVs for decayed bird tissue in their microbiome.</p> </div> <div class="likelihoodratgimicrobiomestructurecanpredictbirdconsumption" title="sec"><span class="label" tagx="label" title="label">3.5</span><div class="title" tagx="title" title="title">
Likelihood rat GI microbiome structure can predict bird consumption</div> <p>We also tested if diversity measures of rat GI microbiome structure could predict bird composition. Rat GI microbiome structure did not predict bird consumption (alpha diversity GLMs: ASV richness <i>p</i>Â =Â .3, Faith's PD <i>p</i>Â =Â .3: beta diversity PERMANOVAs: Jaccard <i>p</i>Â =Â .2, UniFrac <i>p</i>Â =Â .2, Brayâ€"Curtis <i>p</i>Â =Â .5). We detected one ASV, <i>Prevotella copri</i>, that was predominately found in rats that did not consume birds (7/11 rats) while generally absent from rats that had consumed bird (2/20 rats; Indicator Species Analysis <i>p</i>Â =Â .041, stat valueÂ =Â 0.74).</p> </div> </div> <div class="discussion" title="discussion"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
DISCUSSION</div> <p>Scavenging is widespread and significant in most food webs, but is often significantly underestimated, producing inflated predation rates and underestimated indirect effects (Wilson &amp;amp; Wolkovich,Â <a href="#ece37171-bib-0066">2011</a>). We used a multiâ€�pronged approach of new molecular and classic ecological methods to document bird scavenging by invasive rats in Hawaii. We showed that bacterial biomarkers, linked to the successional stage of bird carcass decay, likely persist through digestion as biomarkers of carrion consumption in both rat stomachs and fecal material. Our forensic microbiology tool provides vital information to identify species in networks, detect cryptic linkages that occur beyond our observation (e.g., at night, in the forest canopy, in burrows or crevices) and can be used to help build weighted network models with predictions for food web stability (Deng etÂ al.,Â <a href="#ece37171-bib-0016">2012</a>; Evans etÂ al.,Â <a href="#ece37171-bib-0021">2016</a>; Traugott etÂ al.,Â <a href="#ece37171-bib-0058">2013</a>).</p> <p>We found that 39% of the rats we sampled (20/51) unambiguously consumed birds, and nearly half of thoseâ€"18% in total (9/51)â€"may have consumed birds as carrion by the presence of bacterial biomarkers in their GI microbiome. It is possible that the other 11 rats that we confirmed consumed bird may have consumed the birds as prey given the absence of decayed tissue biomarkers. We found that four of the nine birds repeatedly detected in our study area were detected in the diet of rats, which included one native bird species (<i>M. obscurus</i>) and three introduced bird species (<i>L. leucomelanos</i>, <i>M. gallopavo</i>, <i>Z. japonicus</i>). Two of the native bird species did not have reference sequences for our COI primer set, which may have reduced our likelihood to detect them; however, we did not detect these as diet items with the Cytb primer set for which there were reference sequences in GenBank. The Kalij pheasant (<i>L. nycthemera</i>) was the only species lacking biomarkers attributable to scavenging, while the other three species had individuals that may have been consumed through scavenging.</p> <p>The rats that we identified as likely consuming birds through scavenging contained at least one of 15 informative bacterial ASVs of carrion consumption in their GI microbiome. We caution that these biomarkers may be generally indicative of other vertebrate carrion consumption. In our study system, there are few other vertebrate diet items, but we cannot rule out the possibility that those were also consumed as carrion. In future work, if diet samples are analyzed more broadly, for example using a metabarcoding approach (Forinâ€�Wiart etÂ al.,Â <a href="#ece37171-bib-0023">2018</a>), and only bird species are identified, then identified bacterial biomarkers in the feces or guts most likely originated from the bird in the diet and not some other vertebrate. Additionally, future work should determine if bacterial biomarkers can be considered specific to a particular diet group (e.g., birds or mammals or reptiles) by conducting feeding trials (Zarzosoâ€�Lacoste etÂ al.,Â <a href="#ece37171-bib-0072">2011</a>).</p> <p>Microbial composition in the environment and on animals can often differ across the landscape (Bisson etÂ al.,Â <a href="#ece37171-bib-0007">2009</a>; Muletz Wolz etÂ al.,Â <a href="#ece37171-bib-0039">2018</a>). Across the kÄ«puka we sampled, we found that eight of the 15 informative bacterial biomarkers occurred in rats sampled in multiple kÄ«puka indicating reproducibility of bacterial signatures of decay within our study area. From studies of human and mouse carcass decomposition, evidence suggests that microbial signatures of decay are often predictable on a larger scale across soil types, seasons and host species (Belk etÂ al.,Â <a href="#ece37171-bib-0004">2018</a>; Metcalf etÂ al.,Â <a href="#ece37171-bib-0037">2016</a>). Occurrence of a specific subset of biomarkers from a microbial community offers the potential to develop cheaper, targeted assays (Yan etÂ al.,Â <a href="#ece37171-bib-0069">2015</a>). Replicating our study across a larger geographic region to determine if similar bacterial carrion biomarkers are identified would provide support for developing more targeted assays.</p> <p>We identified bacterial biomarkers of decayed tissue (i.e., carrion) consumption that we considered indicative of scavenging, but were unable to identify bacterial biomarkers of fresh tissue that could have been indicative of predation. In our system, it is most likely that predation with immediate consumption and scavenging are the primary feeding strategies as rats are the primary consumers of bird nestlings and fledglings in the kÄ«puka. However, in other systems where rats may consume large prey items over a series of days, it would be important to differentiate predation and consumption over several days from scavenging. We suggest that for future studies to conclusively differentiate (a) predation and immediate consumption, (b) predation and consumption over several days, and (c) scavenging, it will require direct observation or manipulation to identify and validate biomarkers for each category. In our study, we did find two bacterial ASVs in the decaying chicken tissue study indicative of fresh tissue (and potentially predation), but we did not detect those bacterial ASVs in rat GI microbiomes. The chicken tissue was sterilized with bleach prior to placing it in the field in order to remove any bacteria that accrued on the surface of the meat during its processing for sale. This may have impacted our ability to identify biomarkers of fresh bird tissue, and few bacterial taxa were present at early sampling time points. Likewise, the microbiome of commercially produced chicken is unlikely to resemble that of wild forest birds at initial time points, but during decomposition we expected that the decaying chicken microbiome would be representative of a general bird decomposition microbiome (Belk etÂ al.,Â <a href="#ece37171-bib-0004">2018</a>; Metcalf etÂ al.,Â <a href="#ece37171-bib-0037">2016</a>). We chose to use storeâ€�bought, antibioticâ€�free chicken as a proof of concept and to determine if using birds in Hawaii in a decomposition study would be warranted in the future, as many are vulnerable or endangered.</p> <p>In this study, we determined that invasive rats in Hawaii consumed tissues of both native and introduced birds as diet items. One of the birds identified was the Japanese whiteâ€�eye (<i>Z. japonicus</i>), an invasive bird that since its introduction in 1929 has proliferated to become the most abundant bird species on the islands (Berger,Â <a href="#ece37171-bib-0005">1981</a>; Van Riper,Â <a href="#ece37171-bib-0060">2000</a>). In future studies, we suggest using wholeÂ <i>Z. japonicus</i>Â carcasses in decomposition and/or feeding trials as they are invasive, abundant, and a known rat diet item. It would be pertinent to compare future studies using wild birds with the bacterial biomarkers identified in our chicken decomposition study to verify how the impact of whole unaltered carcasses impact biomarker discovery.</p> <p>The bacterial biomarkers we identified have the potential to be used to determine if rats had consumed birds or not, in cases where the results from diet analysis are ambiguous. However, with our currently limited knowledge on the specificity of bacterial biomarkers of decay to particular diet items, we provide an exploratory analysis with potential for use in future work. We had a relatively high number of ambiguous samples, and this number may have been reduced with more replication of PCR and sequencing (e.g., six PCR replicates per primer pair such as in Zarzosoâ€�Lacoste etÂ al.,Â <a href="#ece37171-bib-0070">2016</a>). However, we still would anticipate some ambiguity even with high laboratory replication and sequencing (Forinâ€�Wiart etÂ al.,Â <a href="#ece37171-bib-0023">2018</a>; Zarzosoâ€�Lacoste etÂ al.,Â <a href="#ece37171-bib-0070">2016</a>). Nonetheless, we were able to robustly identify 20 rats as â€˜consumed birdâ€™ and 11 rats as â€˜not consumed birdsâ€™ for analyses and identification of biomarkers, which was one of our primary goals of the research.</p> <p>Molecular diet analysis of fecal material is gaining traction among molecular ecologists as a noninvasive method to study diet and trophic interactions, particularly for invasive species (Egeter etÂ al.,Â <a href="#ece37171-bib-0019">2019</a>; Zarzosoâ€�Lacoste etÂ al.,Â <a href="#ece37171-bib-0070">2016</a>). However, molecular techniques in diet analysis are not without issue, and some ambiguity can exist such as repeatability in food item detection (Forinâ€�Wiart etÂ al.,Â <a href="#ece37171-bib-0023">2018</a>) and PCR bias (Nichols etÂ al.,Â <a href="#ece37171-bib-0041">2018</a>). Integrating diet analysis with forensic microbiology can provide a suite of tools to better inform conclusions on food webs, including resolving ambiguities in diet analysis and elucidating whether diet items were likely consumed through predation versus scavenging.</p> <p>In order to validate that rat fecal samples would provide similar biological conclusions as stomach samples, we utilized rat stomach and fecal samples from 18 different kÄ«puka that were collected as part of a larger study in which rat removals were performed and individuals were ethically euthanized (stomachs) or traps were placed in control nonâ€�rat removal plots (feces). We found that while rat stomachs and rat feces differed in their microbiome structure with reproducibility in those differences across kÄ«puka, two of 15 bacterial biomarkers were detected in both sample types (<i>Ewingella americana</i> and a <i>Pseudomonas</i> sp.). In paired samples from individual animals of other vertebrate species, fecal samples differed from stomach samples in microbiome structure, including in bats (Ingala etÂ al.,Â <a href="#ece37171-bib-0030">2018</a>) and birds (Drovetski etÂ al.,Â <a href="#ece37171-bib-0018">2018</a>; Videvall etÂ al.,Â <a href="#ece37171-bib-0063">2018</a>). Ingala et al. (<a href="#ece37171-bib-0030">2018</a>) proposed that fecal microbiomes may be more species rich than stomachs, because bacterial DNA from diet items can be retained in pockets of undigested material in feces. Recently produced fecal material may serve as a better sample type than stomachs for bacterial biomarkers of conditions prior to consumption, given the greater potential for retention of bacterial DNA from those diet items in feces. Likewise, collecting fecal material is less timeâ€�intensive and more humane than stomachs.</p> <p>Genomic forensics have many applications to wildlife conservation settings and to food web ecology. Rats (<i>Rattus</i> spp.) have been introduced to nearly 90% of the world's islands and are among the most detrimental invaders (Harper &amp;amp; Bunbury,Â <a href="#ece37171-bib-0028">2015</a>; Martin etÂ al.,Â <a href="#ece37171-bib-0033">2000</a>; Shiels etÂ al.,Â <a href="#ece37171-bib-0055">2014</a>). Knowing the magnitude of predation by invasive rats can inform management practices. Yet, monitoring large numbers of bird nests in the forest canopy to determine if rats caused significant nest failure through predation can be timeâ€� and costâ€�prohibitive. For instance, extensive previous monitoring of bird nests identified a general cause of nest failure (i.e., â€œpresumed predationâ€�) in only ~15% of cases (Knowlton, unpublished data). Similarly, observing foraging behavior to assess rat diets becomes impractical with increasing canopy or cliff height, and a massive effort is required to obtain the statistical power to detect subtle population and community impacts. Repeated temporal snapshots are needed to observe dietary changes that might result from behavioral shifts in foraging, but invasive techniques such as stomach sampling remove animals from the population and is impractical at a large scale. Our paired diet and microbial forensics approach using noninvasive fecal samples could be implemented as a means to screen larger geographic regions for rat predation versus scavenging; certain areas could then be identified for validation with direct observation. Direct observations would be informative to validate bacterial biomarkers that could conclusively differentiate predation and immediate consumption from predation and consumption over several days and from scavenging.</p> <p>Nest predation by black rats in Hawaii can vary across the landscape from high to extremely low (e.g., 87% [Stone et al., <a href="#ece37171-bib-0056">1984</a>] to 4% [Amarasekare,Â <a href="#ece37171-bib-0001">1993</a>]). Although dead biomass can subsidize predator populations, it does not contribute demographically to the dynamics of prey populations (Wolkovich etÂ al.,Â <a href="#ece37171-bib-0068">2014</a>). However, the availability of alternative resources, as live prey or as carrion, may be equally influential in supporting higher numerical response or population densities of invasive omnivores impacting populations of conservation concern (David etÂ al.,Â <a href="#ece37171-bib-0014">2017</a>; Grendelmeier etÂ al.,Â <a href="#ece37171-bib-0025">2018</a>; Holt &amp;amp; Bonsall,Â <a href="#ece37171-bib-0029">2017</a>). Thus, determining the degree of scavenging versus predation is key to determining the impact of invasive rats on bird populations. Our paired diet and microbial forensics approach could be used to examine regional patterns of rat predation versus scavenging on native and introduced bird species, where particular focus would be on identifying localities where rats or other invasive omnivores are shown to be more likely to be consuming native birds through predation. Rat removal is extremely time and labor intensive; therefore, being able to target rat populations that are having direct impacts on native Hawaiian avifauna can be integral to their conservation. Specifically, researchers could collect fecal material from a variety of localities with nonlethal sampling or trapping. Our molecularâ€�based approaches could then be used to detect bird DNA in diet and look for the informative bacterial biomarkers of predation or carrion consumption, as identified here (carrion consumption biomarkers only) or in future research. Specific areas could then be targeted for validation with on ground observations, and appropriate management decisions could be made. Forensic genomic and microbiology methods applied in wildlife ecology have great potential to detect and quantify species interactions that are remote, rare, and challenging to measure. However, there is a need for basic research, such as the research we present here and those by others (Guo etÂ al.,Â <a href="#ece37171-bib-0027">2016</a>; Pechal &amp;amp; Benbow,Â <a href="#ece37171-bib-0044">2016</a>), before this can be formalized as a widely accepted tool.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
CONFLICT OF INTEREST</div> <p>The authors have declared that no competing interests exist.</p> </div> <div class="authorcontribution" title="sec"> <div class="title" tagx="title" title="title">
AUTHOR CONTRIBUTION</div> <p><b>Carly R. Muletzâ€�Wolz:</b> Formal analysis (lead); Investigation (lead); Visualization (lead); Writingâ€�original draft (lead); Writingâ€�review &amp;amp; editing (lead). <b>Erin Wilson Rankin:</b> Conceptualization (equal); Investigation (equal); Resources (equal); Writingâ€�review &amp;amp; editing (equal). <b>Sarah McGrathâ€�Blaser:</b> Formal analysis (equal); Investigation (equal); Visualization (equal); Writingâ€�review &amp;amp; editing (equal). <b>Madhvi Venkatraman:</b> Investigation (equal); Resources (equal); Writingâ€�review &amp;amp; editing (equal). <b>JesÃºs E Maldonado:</b> Conceptualization (equal); Funding acquisition (equal); Resources (equal); Writingâ€�review &amp;amp; editing (equal). <b>Daniel S Gruner:</b> Conceptualization (equal); Funding acquisition (equal); Resources (equal); Supervision (equal); Writingâ€�review &amp;amp; editing (equal). <b>Robert C Fleischer:</b> Conceptualization (equal); Funding acquisition (equal); Resources (equal); Supervision (equal); Writingâ€�review &amp;amp; editing (equal).</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data">  <p>Supplementary Material</p>  <div class="media" title="media"><a href="ECE3-11-1814-s001.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGEMENTS</div> <p>We thank the Hawaii Division of Forestry and Wildlife for allowing access to the research sites. We thank W. Pitt of the US Dept of Agriculture (USDA) Animal and Plant Health Inspection Service for providing critical support for implementing and maintaining rat removal treatments, and field technicians B. Hwang, D. Leopold, K. Francisco, C. Morrison, L. Nakagawa, J. Pangâ€�Ching, and R. De Mattos. We thank the USDA Forest Service Inst. of Pacific Islands Forestry for providing essential logistical and vehicle support, office, and laboratory space. We thank the US Food and Drug Administration and M. Allard for providing a GS FLXÂ +Â to CCG, and Roche 454 Life Sciences for providing a GS Junior to CCG and additional support. We thank N. McInerney for guidance in molecular procedures, and S. Davies, N. Kurata, and K. Stewardson for help in the initial phase of the molecular project.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
DATA AVAILABILITY STATEMENT</div> <p>Demultiplexed highâ€�throughput sequence data and associated metadata have been deposited in the National Center for Biotechnology Information Sequence Read Archive (<a href="http://www.ncbi.nlm">www.ncbi.nlm</a>. nih.gov/sra) under BioProject IDs: PRJNA573692 (chicken decomposition study) and PRJNA573693 (rat GI microbiome). All Sanger sequences for diet analysis and associated metadata are provided at <a href="https://doi.org/10.6084/m9.figshare.13274816">https://doi.org/10.6084/m9.figshare.13274816</a></p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="ece37171-bib-0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Amarasekare</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span> (<span class="year" tagx="year" title="year">1993</span>). <span class="mixed-article-title" title="mixed-article-title">Potential impact of mammalian nest predators on endemic forest birds of western Maunaâ€�Kea, Hawaii</span>. <span class="source" tagx="source" title="source">Conservation Biology</span>, <span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">316</span>â€"<span class="lpage" tagx="lpage" title="lpage">324</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1523-1739.1993.07020316.x">10.1046/j.1523-1739.1993.07020316.x</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Amos</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Whitehead</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferrari</span>, <span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Payne</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gordon</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">1992</span>). <span class="mixed-article-title" title="mixed-article-title">Restrictable DNA from sloughed cetacean skin: Its potential for use in population analyses</span>. <span class="source" tagx="source" title="source">Marine Mammal Science</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">275</span>â€"<span class="lpage" tagx="lpage" title="lpage">283</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1748-7692.1992.tb00409.x">10.1111/j.1748-7692.1992.tb00409.x</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Becker</span>, <span class="given-names" tagx="given-names" title="given-names">M. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Walke</span>, <span class="given-names" tagx="given-names" title="given-names">J. B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cikanek</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Savage</span>, <span class="given-names" tagx="given-names" title="given-names">A. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mattheus</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Santiago</span>, <span class="given-names" tagx="given-names" title="given-names">C. N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Minbiole</span>, <span class="given-names" tagx="given-names" title="given-names">K. P. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Harris</span>, <span class="given-names" tagx="given-names" title="given-names">R. N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Belden</span>, <span class="given-names" tagx="given-names" title="given-names">L. K.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gratwicke</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Composition of symbiotic bacteria predicts survival in Panamanian golden frogs infected with a lethal fungus</span>. <span class="source" tagx="source" title="source">Proceedings of the Royal Society B: Biological Sciences</span>, <span class="volume" tagx="volume" title="volume">282</span>(<span class="issue" tagx="issue" title="issue">1805</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2014.2881">10.1098/rspb.2014.2881</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Belk</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xu</span>, <span class="given-names" tagx="given-names" title="given-names">Z. Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Carter</span>, <span class="given-names" tagx="given-names" title="given-names">D. O.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lynne</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bucheli</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Knight</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Metcalf</span>, <span class="given-names" tagx="given-names" title="given-names">J. L.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Microbiome data accurately predicts the postmortem interval using Random Forest Regression Models</span>. <span class="source" tagx="source" title="source">Genes</span>, <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/genes9020104">10.3390/genes9020104</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Berger</span>, <span class="given-names" tagx="given-names" title="given-names">A. J.</span></span> (<span class="year" tagx="year" title="year">1981</span>). <span class="source" tagx="source" title="source">Hawaiian birdlife</span>, <span class="edition" tagx="edition" title="edition">2</span>nd ed. <span class="publisher-name" tagx="publisher-name" title="publisher-name">University of Hawaii Press</span>.</span></li> <li tag="ref"><a name="ece37171-bib-0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Birkhofer</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bylund</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dalin</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferlian</span>, <span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gagic</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">HambÃ¤ck</span>, <span class="given-names" tagx="given-names" title="given-names">P. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Klapwijk</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mestre</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Roubinet</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schroeder</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stenberg</span>, <span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Porcel</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">BjÃ¶rkman</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jonsson</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Methods to identify the prey of invertebrate predators in terrestrial field studies</span>. <span class="source" tagx="source" title="source">Ecology and Evolution</span>, <span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">6</span>), <span class="fpage" tagx="fpage" title="fpage">1942</span>â€"<span class="lpage" tagx="lpage" title="lpage">1953</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.2791">10.1002/ece3.2791</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28331601">28331601</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bisson</span>, <span class="given-names" tagx="given-names" title="given-names">I. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Marra</span>, <span class="given-names" tagx="given-names" title="given-names">P. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Burtt</span>, <span class="given-names" tagx="given-names" title="given-names">E. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sikaroodi</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gillevet</span>, <span class="given-names" tagx="given-names" title="given-names">P. M.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Variation in plumage microbiota depends on season and migration</span>. <span class="source" tagx="source" title="source">Microbial Ecology</span>, <span class="volume" tagx="volume" title="volume">58</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">212</span>â€"<span class="lpage" tagx="lpage" title="lpage">220</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00248-009-9490-3">10.1007/s00248-009-9490-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19212698">19212698</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bolyen</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rideout</span>, <span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dillon</span>, <span class="given-names" tagx="given-names" title="given-names">M. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bokulich</span>, <span class="given-names" tagx="given-names" title="given-names">N. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Abnet</span>, <span class="given-names" tagx="given-names" title="given-names">C. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alâ€�Ghalith</span>, <span class="given-names" tagx="given-names" title="given-names">G. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alexander</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alm</span>, <span class="given-names" tagx="given-names" title="given-names">E. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Arumugam</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Asnicar</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bai</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bisanz</span>, <span class="given-names" tagx="given-names" title="given-names">J. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bittinger</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brejnrod</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brislawn</span>, <span class="given-names" tagx="given-names" title="given-names">C. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brown</span>, <span class="given-names" tagx="given-names" title="given-names">C. T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Callahan</span>, <span class="given-names" tagx="given-names" title="given-names">B. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Caraballoâ€�RodrÃ­guez</span>, <span class="given-names" tagx="given-names" title="given-names">A. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chase</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, â€¦ <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Caporaso</span>, <span class="given-names" tagx="given-names" title="given-names">J. G.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2</span>. <span class="source" tagx="source" title="source">Nature Biotechnology</span>, <span class="volume" tagx="volume" title="volume">37</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">852</span>â€"<span class="lpage" tagx="lpage" title="lpage">857</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41587-019-0209-9">10.1038/s41587-019-0209-9</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0009" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Callahan</span>, <span class="given-names" tagx="given-names" title="given-names">B. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McMurdie</span>, <span class="given-names" tagx="given-names" title="given-names">P. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rosen</span>, <span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Han</span>, <span class="given-names" tagx="given-names" title="given-names">A. W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Johnson</span>, <span class="given-names" tagx="given-names" title="given-names">A. J. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holmes</span>, <span class="given-names" tagx="given-names" title="given-names">S. P.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">DADA2: Highâ€�resolution sample inference from Illumina amplicon data</span>. <span class="source" tagx="source" title="source">Nature Methods</span>, <span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">581</span>â€"<span class="lpage" tagx="lpage" title="lpage">583</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.3869">10.1038/nmeth.3869</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27214047">27214047</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Carreonâ€�Martinez</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Heath</span>, <span class="given-names" tagx="given-names" title="given-names">D. D.</span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Revolution in food web analysis and trophic ecology: Diet analysis by DNA and stable isotope analysis</span>. <span class="source" tagx="source" title="source">Molecular Ecology</span>, <span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">25</span>â€"<span class="lpage" tagx="lpage" title="lpage">27</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2009.04412.x">10.1111/j.1365-294X.2009.04412.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20078768">20078768</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Castroâ€�Luna</span>, <span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sosa</span>, <span class="given-names" tagx="given-names" title="given-names">V. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Castilloâ€�Campos</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Quantifying phyllostomid bats at different taxonomic levels as ecological indicators in a disturbed tropical forest</span>. <span class="source" tagx="source" title="source">Acta Chiropterologica</span>, <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">219</span>â€"<span class="lpage" tagx="lpage" title="lpage">228</span>.</span></li> <li tag="ref"><a name="ece37171-bib-0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Clark</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span> (<span class="year" tagx="year" title="year">1982</span>). <span class="mixed-article-title" title="mixed-article-title">Foraging behavior of a vertebrate omnivore (<i>Rattus rattus</i>)â€"Meal structure, sampling and diet breadth</span>. <span class="source" tagx="source" title="source">Ecology</span>, <span class="volume" tagx="volume" title="volume">63</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">763</span>â€"<span class="lpage" tagx="lpage" title="lpage">772</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2307/1936797">10.2307/1936797</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cole</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Loope</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Medeiros</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Howe</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Anderson</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Food habits of introduced rodents in highâ€�elevation shrubland of Haleakala National Park, Maui, Hawai'i</span>. <span class="source" tagx="source" title="source">Pacific Science</span>, <span class="volume" tagx="volume" title="volume">54</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">313</span>â€"<span class="lpage" tagx="lpage" title="lpage">329</span>.</span></li> <li tag="ref"><a name="ece37171-bib-0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">David</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thebault</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Anneville</span>, <span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Duyck</span>, <span class="given-names" tagx="given-names" title="given-names">P. F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chapuis</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Loeuille</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Impacts of invasive species on food webs: A review of empirical data</span>. <span class="source" tagx="source" title="source">Networks of Invasion: A Synthesis of Concepts</span>, <span class="volume" tagx="volume" title="volume">56</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">60</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/bs.aecr.2016.10.001">10.1016/bs.aecr.2016.10.001</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De Caceres</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Legendre</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Associations between species and groups of sites: Indices and statistical inference</span>. <span class="source" tagx="source" title="source">Ecology</span>, <span class="volume" tagx="volume" title="volume">90</span>(<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">3566</span>â€"<span class="lpage" tagx="lpage" title="lpage">3574</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20120823">20120823</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deng</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jiang</span>, <span class="given-names" tagx="given-names" title="given-names">Y. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yang</span>, <span class="given-names" tagx="given-names" title="given-names">Y. F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">He</span>, <span class="given-names" tagx="given-names" title="given-names">Z. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Luo</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span>, <span class="given-names" tagx="given-names" title="given-names">J. Z.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular ecological network analyses</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">113</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-13-113">10.1186/1471-2105-13-113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214541">22214541</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">DeSantis</span>, <span class="given-names" tagx="given-names" title="given-names">T. Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hugenholtz</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Larsen</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rojas</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brodie</span>, <span class="given-names" tagx="given-names" title="given-names">E. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Keller</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huber</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dalevi</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hu</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Andersen</span>, <span class="given-names" tagx="given-names" title="given-names">G. L.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Greengenes, a chimeraâ€�checked 16S rRNA gene database and workbench compatible with ARB</span>. <span class="source" tagx="source" title="source">Applied and Environmental Microbiology</span>, <span class="volume" tagx="volume" title="volume">72</span>(<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">5069</span>â€"<span class="lpage" tagx="lpage" title="lpage">5072</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/aem.03006-05">10.1128/aem.03006-05</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16820507">16820507</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Drovetski</span>, <span class="given-names" tagx="given-names" title="given-names">S. V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">O'Mahoney</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ransome</span>, <span class="given-names" tagx="given-names" title="given-names">E. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Matterson</span>, <span class="given-names" tagx="given-names" title="given-names">K. O.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lim</span>, <span class="given-names" tagx="given-names" title="given-names">H. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chesser</span>, <span class="given-names" tagx="given-names" title="given-names">R. T.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Graves</span>, <span class="given-names" tagx="given-names" title="given-names">G. R.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Spatial organization of the gastrointestinal microbiota in urban Canada Geese</span>. <span class="source" tagx="source" title="source">Scientific Reports</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">10</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-21892-y">10.1038/s41598-018-21892-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29311619">29311619</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Egeter</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Roe</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Peixoto</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Puppo</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Easton</span>, <span class="given-names" tagx="given-names" title="given-names">L. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pinto</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bishop</span>, <span class="given-names" tagx="given-names" title="given-names">P. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Robertson</span>, <span class="given-names" tagx="given-names" title="given-names">B. C.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Using molecular diet analysis to inform invasive species management: A case study of introduced rats consuming endemic New Zealand frogs</span>. <span class="source" tagx="source" title="source">Ecology and Evolution</span>, <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">5032</span>â€"<span class="lpage" tagx="lpage" title="lpage">5048</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.4903">10.1002/ece3.4903</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31110660">31110660</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Eggert</span>, <span class="given-names" tagx="given-names" title="given-names">L. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Maldonado</span>, <span class="given-names" tagx="given-names" title="given-names">J. E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fleischer</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Nucleic acid isolation from ecological samplesâ€"Animal scat and other associated materials</span>. <span class="source" tagx="source" title="source">Methods in Enzymology</span>, <span class="volume" tagx="volume" title="volume">395</span>, <span class="fpage" tagx="fpage" title="fpage">73</span>â€"<span class="lpage" tagx="lpage" title="lpage">87</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15865962">15865962</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Evans</span>, <span class="given-names" tagx="given-names" title="given-names">D. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kitson</span>, <span class="given-names" tagx="given-names" title="given-names">J. J. N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lunt</span>, <span class="given-names" tagx="given-names" title="given-names">D. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Straw</span>, <span class="given-names" tagx="given-names" title="given-names">N. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pocock</span>, <span class="given-names" tagx="given-names" title="given-names">M. J. O.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Merging DNA metabarcoding and ecological network analysis to understand and build resilient terrestrial ecosystems</span>. <span class="source" tagx="source" title="source">Functional Ecology</span>, <span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">1904</span>â€"<span class="lpage" tagx="lpage" title="lpage">1916</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1365-2435.12659">10.1111/1365-2435.12659</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Flaspohler</span>, <span class="given-names" tagx="given-names" title="given-names">D. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Giardina</span>, <span class="given-names" tagx="given-names" title="given-names">C. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Asner</span>, <span class="given-names" tagx="given-names" title="given-names">G. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hart</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Price</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lyons</span>, <span class="given-names" tagx="given-names" title="given-names">C. K.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Castaneda</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Longâ€�term effects of fragmentation and fragment properties on bird species richness in Hawaiian forests</span>. <span class="source" tagx="source" title="source">Biological Conservation</span>, <span class="volume" tagx="volume" title="volume">143</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">280</span>â€"<span class="lpage" tagx="lpage" title="lpage">288</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biocon.2009.10.009">10.1016/j.biocon.2009.10.009</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Forinâ€�Wiart</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Poulle</span>, <span class="given-names" tagx="given-names" title="given-names">M. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Piry</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cosson</span>, <span class="given-names" tagx="given-names" title="given-names">J. F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Larose</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Galan</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Evaluating metabarcoding to analyse diet composition of species foraging in anthropogenic landscapes using Ion Torrent and Illumina sequencing</span>. <span class="source" tagx="source" title="source">Scientific Reports</span>, <span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">17091</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-34430-7">10.1038/s41598-018-34430-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30459313">30459313</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fukami</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wardle</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bellingham</span>, <span class="given-names" tagx="given-names" title="given-names">P. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mulder</span>, <span class="given-names" tagx="given-names" title="given-names">C. P. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Towns</span>, <span class="given-names" tagx="given-names" title="given-names">D. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yeates</span>, <span class="given-names" tagx="given-names" title="given-names">G. W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bonner</span>, <span class="given-names" tagx="given-names" title="given-names">K. I.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Durrett</span>, <span class="given-names" tagx="given-names" title="given-names">M. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Grantâ€�Hoffman</span>, <span class="given-names" tagx="given-names" title="given-names">M. N.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Williamson</span>, <span class="given-names" tagx="given-names" title="given-names">W. M.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Aboveâ€� and belowâ€�ground impacts of introduced predators in seabirdâ€�dominated island ecosystems</span>. <span class="source" tagx="source" title="source">Ecology Letters</span>, <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">1299</span>â€"<span class="lpage" tagx="lpage" title="lpage">1307</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1461-0248.2006.00983.x">10.1111/j.1461-0248.2006.00983.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17118004">17118004</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Grendelmeier</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Arlettaz</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pasinelli</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Numerical response of mammalian carnivores to rodents affects bird reproduction in temperate forests: A case of apparent competition?</span><span class="source" tagx="source" title="source">Ecology and Evolution</span>, <span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">23</span>), <span class="fpage" tagx="fpage" title="fpage">11596</span>â€"<span class="lpage" tagx="lpage" title="lpage">11608</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.4608">10.1002/ece3.4608</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30598759">30598759</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gruner</span>, <span class="given-names" tagx="given-names" title="given-names">D. S.</span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Attenuation of topâ€�down and bottomâ€�up forces in a complex terrestrial community</span>. <span class="source" tagx="source" title="source">Ecology</span>, <span class="volume" tagx="volume" title="volume">85</span>(<span class="issue" tagx="issue" title="issue">11</span>), <span class="fpage" tagx="fpage" title="fpage">3010</span>â€"<span class="lpage" tagx="lpage" title="lpage">3022</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1890/04-0020">10.1890/04-0020</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Guo</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fu</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liao</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hu</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Long</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yan</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ding</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zha</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Guo</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yan</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chang</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cai</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Potential use of bacterial community succession for estimating postâ€�mortem interval as revealed by highâ€�throughput sequencing</span>. <span class="source" tagx="source" title="source">Scientific Reports</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">24197</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep24197">10.1038/srep24197</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27052375">27052375</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Harper</span>, <span class="given-names" tagx="given-names" title="given-names">G. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bunbury</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Invasive rats on tropical islands: Their population biology and impacts on native species</span>. <span class="source" tagx="source" title="source">Global Ecology and Conservation</span>, <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">607</span>â€"<span class="lpage" tagx="lpage" title="lpage">627</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gecco.2015.02.010">10.1016/j.gecco.2015.02.010</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holt</span>, <span class="given-names" tagx="given-names" title="given-names">R. D.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bonsall</span>, <span class="given-names" tagx="given-names" title="given-names">M. B.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Apparent competition</span>. <span class="source" tagx="source" title="source">Annual Review of Ecology, Evolution, and Systematics</span>, <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">447</span>â€"<span class="lpage" tagx="lpage" title="lpage">471</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-ecolsys-110316-022628">10.1146/annurev-ecolsys-110316-022628</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ingala</span>, <span class="given-names" tagx="given-names" title="given-names">M. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Simmons</span>, <span class="given-names" tagx="given-names" title="given-names">N. B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wultsch</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Krampis</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Speer</span>, <span class="given-names" tagx="given-names" title="given-names">K. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Perkins</span>, <span class="given-names" tagx="given-names" title="given-names">S. L.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Comparing microbiome sampling methods in a wild mammal: Fecal and intestinal samples record different signals of host ecology, evolution</span>. <span class="source" tagx="source" title="source">Frontiers in Microbiology</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2018.00803">10.3389/fmicb.2018.00803</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29403456">29403456</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Knowlton</span>, <span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Flaspohler</span>, <span class="given-names" tagx="given-names" title="given-names">D. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Paxton</span>, <span class="given-names" tagx="given-names" title="given-names">E. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fukami</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Giardina</span>, <span class="given-names" tagx="given-names" title="given-names">C. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gruner</span>, <span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rankin</span>, <span class="given-names" tagx="given-names" title="given-names">E. E. W.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Movements of four native Hawaiian birds across a naturally fragmented landscape</span>. <span class="source" tagx="source" title="source">Journal of Avian Biology</span>, <span class="volume" tagx="volume" title="volume">48</span>(<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">921</span>â€"<span class="lpage" tagx="lpage" title="lpage">931</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/jav.00924">10.1111/jav.00924</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Levy</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Getting the drop on Hawaiian invasives</span>. <span class="source" tagx="source" title="source">BioScience</span>, <span class="volume" tagx="volume" title="volume">53</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">694</span>â€"<span class="lpage" tagx="lpage" title="lpage">699</span>.</span></li> <li tag="ref"><a name="ece37171-bib-0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Martin</span>, <span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thibault</span>, <span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bretagnolle</span>, <span class="given-names" tagx="given-names" title="given-names">V.</span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Black rats, island characteristics, and colonial nesting birds in the Mediterranean: Consequences of an ancient introduction</span>. <span class="source" tagx="source" title="source">Conservation Biology</span>, <span class="volume" tagx="volume" title="volume">14</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">1452</span>â€"<span class="lpage" tagx="lpage" title="lpage">1466</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1523-1739.2000.99190.x">10.1046/j.1523-1739.2000.99190.x</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0034" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McInnes</span>, <span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jarman</span>, <span class="given-names" tagx="given-names" title="given-names">S. N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lea</span>, <span class="given-names" tagx="given-names" title="given-names">M.â€�A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Raymond</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deagle</span>, <span class="given-names" tagx="given-names" title="given-names">B. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Phillips</span>, <span class="given-names" tagx="given-names" title="given-names">R. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Catry</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stanworth</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weimerskirch</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kusch</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gras</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cherel</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Maschette</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alderman</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">DNA metabarcoding as a marine conservation and management tool: A circumpolar examination of fishery discards in the diet of threatened albatrosses</span>. <span class="source" tagx="source" title="source">Frontiers in Marine Science</span>, <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fmars.2017.00277">10.3389/fmars.2017.00277</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0035" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McMurdie</span>, <span class="given-names" tagx="given-names" title="given-names">P. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holmes</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Phyloseq: An R package for reproducible interactive analysis and graphics of microbiome census data</span>. <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">12</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0061217">10.1371/journal.pone.0061217</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McMurdie</span>, <span class="given-names" tagx="given-names" title="given-names">P. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holmes</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Waste not, want not: Why rarefying microbiome data is inadmissible</span>. <span class="source" tagx="source" title="source">Plos Computational Biology</span>, <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">12</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1003531">10.1371/journal.pcbi.1003531</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Metcalf</span>, <span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xu</span>, <span class="given-names" tagx="given-names" title="given-names">Z. Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weiss</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lax</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Van Treuren</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hyde</span>, <span class="given-names" tagx="given-names" title="given-names">E. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Song</span>, <span class="given-names" tagx="given-names" title="given-names">S. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Amir</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Larsen</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sangwan</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Haarmann</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Humphrey</span>, <span class="given-names" tagx="given-names" title="given-names">G. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ackermann</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thompson</span>, <span class="given-names" tagx="given-names" title="given-names">L. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lauber</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bibat</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nicholas</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gebert</span>, <span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Petrosino</span>, <span class="given-names" tagx="given-names" title="given-names">J. F.</span></span>, â€¦ <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Knight</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial community assembly and metabolic function during mammalian corpse decomposition</span>. <span class="source" tagx="source" title="source">Science</span>, <span class="volume" tagx="volume" title="volume">351</span>(<span class="issue" tagx="issue" title="issue">6269</span>), <span class="fpage" tagx="fpage" title="fpage">158</span>â€"<span class="lpage" tagx="lpage" title="lpage">162</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aad2646">10.1126/science.aad2646</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26657285">26657285</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moleon</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sanchezâ€�Zapata</span>, <span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Selva</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Donazar</span>, <span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Owenâ€�Smith</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Interâ€�specific interactions linking predation and scavenging in terrestrial vertebrate assemblages</span>. <span class="source" tagx="source" title="source">Biological Reviews</span>, <span class="volume" tagx="volume" title="volume">89</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">1042</span>â€"<span class="lpage" tagx="lpage" title="lpage">1054</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/brv.12097">10.1111/brv.12097</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24602047">24602047</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Muletz Wolz</span>, <span class="given-names" tagx="given-names" title="given-names">C. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yarwood</span>, <span class="given-names" tagx="given-names" title="given-names">S. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Campbell Grant</span>, <span class="given-names" tagx="given-names" title="given-names">E. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fleischer</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lips</span>, <span class="given-names" tagx="given-names" title="given-names">K. R.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Effects of host species and environment on the skin microbiome of Plethodontid salamanders</span>. <span class="source" tagx="source" title="source">Journal of Animal Ecology</span>, <span class="volume" tagx="volume" title="volume">87</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">341</span>â€"<span class="lpage" tagx="lpage" title="lpage">353</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1365-2656.12726">10.1111/1365-2656.12726</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Muletzâ€�Wolz</span>, <span class="given-names" tagx="given-names" title="given-names">C. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">DiRenzo</span>, <span class="given-names" tagx="given-names" title="given-names">G. V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yarwood</span>, <span class="given-names" tagx="given-names" title="given-names">S. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Campbell Grant</span>, <span class="given-names" tagx="given-names" title="given-names">E. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fleischer</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lips</span>, <span class="given-names" tagx="given-names" title="given-names">K. R.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Antifungal bacteria on woodland salamander skin exhibit high taxonomic diversity and geographic variability</span>. <span class="source" tagx="source" title="source">Applied and Environment Microbiology</span>, <span class="volume" tagx="volume" title="volume">83</span>(<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.00186-17">10.1128/AEM.00186-17</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nichols</span>, <span class="given-names" tagx="given-names" title="given-names">R. V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vollmers</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Newsom</span>, <span class="given-names" tagx="given-names" title="given-names">L. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Heintzman</span>, <span class="given-names" tagx="given-names" title="given-names">P. D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Leighton</span>, <span class="given-names" tagx="given-names" title="given-names">M. K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Green</span>, <span class="given-names" tagx="given-names" title="given-names">R. E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shapiro</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Minimizing polymerase biases in metabarcoding</span>. <span class="source" tagx="source" title="source">Molecular Ecology Resources</span>, <span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">927</span>â€"<span class="lpage" tagx="lpage" title="lpage">939</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1755-0998.12895">10.1111/1755-0998.12895</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0042" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Novak</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wootton</span>, <span class="given-names" tagx="given-names" title="given-names">J. T.</span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Using experimental indices to quantify the strength of species interactions</span>. <span class="source" tagx="source" title="source">Oikos</span>, <span class="volume" tagx="volume" title="volume">119</span>(<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">1057</span>â€"<span class="lpage" tagx="lpage" title="lpage">1063</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1600-0706.2009.18147.x">10.1111/j.1600-0706.2009.18147.x</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0043" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Paine</span>, <span class="given-names" tagx="given-names" title="given-names">R. T.</span></span> (<span class="year" tagx="year" title="year">1980</span>). <span class="mixed-article-title" title="mixed-article-title">Food websâ€"Linkage, interaction strength and community infrastructureâ€"The 3rd Tansley lecture</span>. <span class="source" tagx="source" title="source">Journal of Animal Ecology</span>, <span class="volume" tagx="volume" title="volume">49</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">667</span>â€"<span class="lpage" tagx="lpage" title="lpage">685</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2307/4220">10.2307/4220</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0044" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pechal</span>, <span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Benbow</span>, <span class="given-names" tagx="given-names" title="given-names">M. E.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial ecology of the salmon necrobiome: Evidence salmon carrion decomposition influences aquatic and terrestrial insect microbiomes</span>. <span class="source" tagx="source" title="source">Environmental Microbiology</span>, <span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">1511</span>â€"<span class="lpage" tagx="lpage" title="lpage">1522</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1462-2920.13187">10.1111/1462-2920.13187</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26690563">26690563</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0045" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pechal</span>, <span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Crippen</span>, <span class="given-names" tagx="given-names" title="given-names">T. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Benbow</span>, <span class="given-names" tagx="given-names" title="given-names">M. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tarone</span>, <span class="given-names" tagx="given-names" title="given-names">A. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dowd</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tomberlin</span>, <span class="given-names" tagx="given-names" title="given-names">J. K.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">The potential use of bacterial community succession in forensics as described by high throughput metagenomic sequencing</span>. <span class="source" tagx="source" title="source">International Journal of Legal Medicine</span>, <span class="volume" tagx="volume" title="volume">128</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">193</span>â€"<span class="lpage" tagx="lpage" title="lpage">205</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00414-013-0872-1">10.1007/s00414-013-0872-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23749255">23749255</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0046" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Percy</span>, <span class="given-names" tagx="given-names" title="given-names">D. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Garver</span>, <span class="given-names" tagx="given-names" title="given-names">A. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wagner</span>, <span class="given-names" tagx="given-names" title="given-names">W. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">James</span>, <span class="given-names" tagx="given-names" title="given-names">H. F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cunningham</span>, <span class="given-names" tagx="given-names" title="given-names">C. W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Miller</span>, <span class="given-names" tagx="given-names" title="given-names">S. E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fleischer</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Progressive island colonization and ancient origin of Hawaiian Metrosideros (Myrtaceae)</span>. <span class="source" tagx="source" title="source">Proceedings of the Royal Society Bâ€�Biological Sciences</span>, <span class="volume" tagx="volume" title="volume">275</span>(<span class="issue" tagx="issue" title="issue">1642</span>), <span class="fpage" tagx="fpage" title="fpage">1479</span>â€"<span class="lpage" tagx="lpage" title="lpage">1490</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2008.0191">10.1098/rspb.2008.0191</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0047" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Price</span>, <span class="given-names" tagx="given-names" title="given-names">J. P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Clague</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">How old is the Hawaiian biota? Geology and phylogeny suggest recent divergence</span>. <span class="source" tagx="source" title="source">Proceedings of the Royal Society Bâ€�Biological Sciences</span>, <span class="volume" tagx="volume" title="volume">269</span>(<span class="issue" tagx="issue" title="issue">1508</span>), <span class="fpage" tagx="fpage" title="fpage">2429</span>â€"<span class="lpage" tagx="lpage" title="lpage">2435</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2002.2175">10.1098/rspb.2002.2175</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0048" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Price</span>, <span class="given-names" tagx="given-names" title="given-names">M. N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dehal</span>, <span class="given-names" tagx="given-names" title="given-names">P. S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Arkin</span>, <span class="given-names" tagx="given-names" title="given-names">A. P.</span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">FastTree 2â€�approximately maximumâ€�likelihood trees for large alignments</span>. <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">10</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0009490">10.1371/journal.pone.0009490</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0049" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: Râ€�Coreâ€�Team</span> . (<span class="year" tagx="year" title="year">2019</span>). <span class="source" tagx="source" title="source">R: A language and environment for statistical computing</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">R Foundation for Statistical Computing</span> Retrieved from <a href="https://www.R-project.org/">https://www.Râ€�project.org/</a></span></li> <li tag="ref"><a name="ece37171-bib-0050" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Renner</span>, <span class="given-names" tagx="given-names" title="given-names">S. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rappole</span>, <span class="given-names" tagx="given-names" title="given-names">J. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rasmussen</span>, <span class="given-names" tagx="given-names" title="given-names">P. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Aung</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Aung</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shwe</span>, <span class="given-names" tagx="given-names" title="given-names">N. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dumbacher</span>, <span class="given-names" tagx="given-names" title="given-names">J. P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fleischer</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">A new subspecies of Tesiaolivea (Sylviidae) from Chiang Mai province, northern Thailand</span>. <span class="source" tagx="source" title="source">Journal of Ornithology</span>, <span class="volume" tagx="volume" title="volume">149</span>, <span class="fpage" tagx="fpage" title="fpage">439</span>â€"<span class="lpage" tagx="lpage" title="lpage">450</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10336-008-0277-y">10.1007/s10336-008-0277-y</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0051" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rohland</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Reich</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Costâ€�effective, highâ€�throughput DNA sequencing libraries for multiplexed target capture</span>. <span class="source" tagx="source" title="source">Genome Research</span>, <span class="volume" tagx="volume" title="volume">22</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">939</span>â€"<span class="lpage" tagx="lpage" title="lpage">946</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.128124.111">10.1101/gr.128124.111</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22267522">22267522</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0052" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ruffino</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zarzosoâ€�Lacoste</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vidal</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Assessment of invasive rodent impacts on island avifauna: Methods, limitations and the way forward</span>. <span class="source" tagx="source" title="source">Wildlife Research</span>, <span class="volume" tagx="volume" title="volume">42</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">185</span>â€"<span class="lpage" tagx="lpage" title="lpage">195</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1071/WR15047">10.1071/WR15047</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0053" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Russel</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thorsen</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brejnrod</span>, <span class="given-names" tagx="given-names" title="given-names">A. D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bisgaard</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">SÃ¸rensen</span>, <span class="given-names" tagx="given-names" title="given-names">S. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">BurmÃ¸lle</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Datest: A framework for choosing differential abundance or expression method</span>. <span class="source" tagx="source" title="source">bioRxiv</span>, <span class="fpage" tagx="fpage" title="fpage">241802</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/241802">10.1101/241802</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0054" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">RytkÃ¶nen</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vesterinen</span>, <span class="given-names" tagx="given-names" title="given-names">E. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Westerduin</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">LeviÃ¤kangas</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vatka</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mutanen</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">VÃ¤limÃ¤ki</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hukkanen</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Suokas</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Orell</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">From feces to data: A metabarcoding method for analyzing consumed and available prey in a birdâ€�insect food web</span>. <span class="source" tagx="source" title="source">Ecology and Evolution</span>, <span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">631</span>â€"<span class="lpage" tagx="lpage" title="lpage">639</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ece3.4787">10.1002/ece3.4787</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30680143">30680143</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0055" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shiels</span>, <span class="given-names" tagx="given-names" title="given-names">A. B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pitt</span>, <span class="given-names" tagx="given-names" title="given-names">W. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sugihara</span>, <span class="given-names" tagx="given-names" title="given-names">R. T.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Witmer</span>, <span class="given-names" tagx="given-names" title="given-names">G. W.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Biology and impacts of pacific island invasive species. 11. <i>Rattus rattus</i>, the black rat (Rodentia: Muridae)</span>. <span class="source" tagx="source" title="source">Pacific Science</span>, <span class="volume" tagx="volume" title="volume">68</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">145</span>â€"<span class="lpage" tagx="lpage" title="lpage">184</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2984/68.2.1">10.2984/68.2.1</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0056" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stone</span>, <span class="given-names" tagx="given-names" title="given-names">C. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bank</span>, <span class="given-names" tagx="given-names" title="given-names">P. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Higashino</span>, <span class="given-names" tagx="given-names" title="given-names">P. K.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Howarth</span>, <span class="given-names" tagx="given-names" title="given-names">F. G.</span></span> (<span class="year" tagx="year" title="year">1984</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Interrelationships of alien and native plants and animals in Kipahulu valley, Haleakala National Park: A preliminary report</span> In <span class="person-group'"><span class="string-name" title="string-name"><span class="given-names" tagx="given-names" title="given-names">C. W.</span><span class="surname" tagx="surname" title="surname">Smith</span></span></span> (Ed.), <span class="source" tagx="source" title="source">Proceedings of the Fifth Conference in Natural Sciences Hawaii Volcanoes National Park</span> (pp. <span class="fpage" tagx="fpage" title="fpage">91</span>â€�<span class="lpage" tagx="lpage" title="lpage">105</span>). <span class="publisher-name" tagx="publisher-name" title="publisher-name">University of Hawaii at Manoa, Department of Botany</span>.</span></li> <li tag="ref"><a name="ece37171-bib-0057" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Towns</span>, <span class="given-names" tagx="given-names" title="given-names">D. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Atkinson</span>, <span class="given-names" tagx="given-names" title="given-names">I. A. E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Daugherty</span>, <span class="given-names" tagx="given-names" title="given-names">C. H.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Have the harmful effects of introduced rats on islands been exaggerated?</span><span class="source" tagx="source" title="source">Biological Invasions</span>, <span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">863</span>â€"<span class="lpage" tagx="lpage" title="lpage">891</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10530-005-0421-z">10.1007/s10530-005-0421-z</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0058" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Traugott</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kamenova</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ruess</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Seeber</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Plantegenest</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Empirically characterising trophic networks: What emerging DNAâ€�based methods, stable isotope and fatty acid analyses can offer</span>. <span class="source" tagx="source" title="source">Advances in Ecological Research</span>, <span class="volume" tagx="volume" title="volume">49</span>, <span class="fpage" tagx="fpage" title="fpage">177</span>â€"<span class="lpage" tagx="lpage" title="lpage">224</span>.</span></li> <li tag="ref"><a name="ece37171-bib-0059" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tylianakis</span>, <span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Didham</span>, <span class="given-names" tagx="given-names" title="given-names">R. K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bascompte</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wardle</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Global change and species interactions in terrestrial ecosystems</span>. <span class="source" tagx="source" title="source">Ecology Letters</span>, <span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">1351</span>â€"<span class="lpage" tagx="lpage" title="lpage">1363</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1461-0248.2008.01250.x">10.1111/j.1461-0248.2008.01250.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19062363">19062363</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0060" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Van Riper</span>, <span class="given-names" tagx="given-names" title="given-names">S. G.</span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Japanese whiteâ€�eye (<i>Zosterops japonicus</i>), version 2.0</span> In <span class="person-group'"><span class="string-name" title="string-name"><span class="given-names" tagx="given-names" title="given-names">A. F.</span><span class="surname" tagx="surname" title="surname">Poole</span></span></span>, &amp;amp; <span class="person-group'"><span class="string-name" title="string-name"><span class="given-names" tagx="given-names" title="given-names">F. B.</span><span class="surname" tagx="surname" title="surname">Gill</span></span></span> (Eds.), <span class="source" tagx="source" title="source">The birds of North America</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Cornell Lab of Ornithology</span><span class="pub-id"><a href="https://dx.doi.org/10.2173/bna.487">10.2173/bna.487</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0061" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vanderwerf</span>, <span class="given-names" tagx="given-names" title="given-names">E. A.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution of nesting height in an endangered Hawaiian forest bird in response to a nonâ€�native predator</span>. <span class="source" tagx="source" title="source">Conservation Biology</span>, <span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">905</span>â€"<span class="lpage" tagx="lpage" title="lpage">911</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1523-1739.2012.01877.x">10.1111/j.1523-1739.2012.01877.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22830652">22830652</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0062" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vaughn</span>, <span class="given-names" tagx="given-names" title="given-names">N. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Asner</span>, <span class="given-names" tagx="given-names" title="given-names">G. P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Giardina</span>, <span class="given-names" tagx="given-names" title="given-names">C. P.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Centennial impacts of fragmentation on the canopy structure of tropical montane forest</span>. <span class="source" tagx="source" title="source">Ecological Applications</span>, <span class="volume" tagx="volume" title="volume">24</span>(<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">1638</span>â€"<span class="lpage" tagx="lpage" title="lpage">1650</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1890/13-1568.1">10.1890/13-1568.1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29210228">29210228</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0063" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Videvall</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Strandh</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Engelbrecht</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cloete</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cornwallis</span>, <span class="given-names" tagx="given-names" title="given-names">C. K.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Measuring the gut microbiome in birds: Comparison of faecal and cloacal sampling</span>. <span class="source" tagx="source" title="source">Molecular Ecology Resources</span>, <span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">424</span>â€"<span class="lpage" tagx="lpage" title="lpage">434</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1755-0998.12744">10.1111/1755-0998.12744</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29205893">29205893</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0064" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weiss</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xu</span>, <span class="given-names" tagx="given-names" title="given-names">Z. Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Peddada</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Amir</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bittinger</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gonzalez</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lozupone</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zaneveld</span>, <span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">VÃ¡zquezâ€�Baeza</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Birmingham</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hyde</span>, <span class="given-names" tagx="given-names" title="given-names">E. R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Knight</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Normalization and microbial differential abundance strategies depend upon data characteristics</span>. <span class="source" tagx="source" title="source">Microbiome</span>, <span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">27</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s40168-017-0237-y">10.1186/s40168-017-0237-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28253908">28253908</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0065" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wilbert</span>, <span class="given-names" tagx="given-names" title="given-names">T. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Woollett</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Whitelaw</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dart</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hoyt</span>, <span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Galen</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ralls</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Meade</span>, <span class="given-names" tagx="given-names" title="given-names">D. E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Maldonado</span>, <span class="given-names" tagx="given-names" title="given-names">J. E.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Nonâ€�invasive baseline genetic monitoring of the endangered San Joaquin kit fox on a photovoltaic solar facility</span>. <span class="source" tagx="source" title="source">Endangered Species Research</span>, <span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">31</span>â€"<span class="lpage" tagx="lpage" title="lpage">41</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3354/esr00649">10.3354/esr00649</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0066" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wilson</span>, <span class="given-names" tagx="given-names" title="given-names">E. E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wolkovich</span>, <span class="given-names" tagx="given-names" title="given-names">E. M.</span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Scavenging: How carnivores and carrion structure communities</span>. <span class="source" tagx="source" title="source">Trends in Ecology &amp;amp; Evolution</span>, <span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">129</span>â€"<span class="lpage" tagx="lpage" title="lpage">135</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tree.2010.12.011">10.1016/j.tree.2010.12.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21295371">21295371</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0067" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wilson Rankin</span>, <span class="given-names" tagx="given-names" title="given-names">E. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Knowlton</span>, <span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gruner</span>, <span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Flaspohler</span>, <span class="given-names" tagx="given-names" title="given-names">D. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Giardina</span>, <span class="given-names" tagx="given-names" title="given-names">C. P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Leopold</span>, <span class="given-names" tagx="given-names" title="given-names">D. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Buckardt</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pitt</span>, <span class="given-names" tagx="given-names" title="given-names">W. C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fukami</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Vertical foraging shifts in Hawaiian forest birds in response to invasive rat removal</span>. <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">9</span>), <span class="elocation-id" tagx="elocation-id" title="elocation-id">e0202869</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0202869">10.1371/journal.pone.0202869</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30248110">30248110</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0068" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wolkovich</span>, <span class="given-names" tagx="given-names" title="given-names">E. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Allesina</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cottingham</span>, <span class="given-names" tagx="given-names" title="given-names">K. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moore</span>, <span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sandin</span>, <span class="given-names" tagx="given-names" title="given-names">S. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">de Mazancourt</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Linking the green and brown worlds: The prevalence and effect of multichannel feeding in food webs</span>. <span class="source" tagx="source" title="source">Ecology</span>, <span class="volume" tagx="volume" title="volume">95</span>(<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">3376</span>â€"<span class="lpage" tagx="lpage" title="lpage">3386</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1890/13-1721.1">10.1890/13-1721.1</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0069" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yan</span>, <span class="given-names" tagx="given-names" title="given-names">X. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yang</span>, <span class="given-names" tagx="given-names" title="given-names">M. X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liu</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gao</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hu</span>, <span class="given-names" tagx="given-names" title="given-names">J. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hu</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Discovery and validation of potential bacterial biomarkers for lung cancer</span>. <span class="source" tagx="source" title="source">American Journal of Cancer Research</span>, <span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">10</span>), <span class="fpage" tagx="fpage" title="fpage">3111</span>â€"<span class="lpage" tagx="lpage" title="lpage">3122</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26693063">26693063</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0070" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zarzosoâ€�Lacoste</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bonnaud</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Corse</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gilles</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Meglecz</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Costedoat</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gouni</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vidal</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Improving morphological diet studies with molecular ecology: An application for invasive mammal predation on island birds</span>. <span class="source" tagx="source" title="source">Biological Conservation</span>, <span class="volume" tagx="volume" title="volume">193</span>, <span class="fpage" tagx="fpage" title="fpage">134</span>â€"<span class="lpage" tagx="lpage" title="lpage">142</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biocon.2015.11.018">10.1016/j.biocon.2015.11.018</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0071" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zarzosoâ€�Lacoste</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Corse</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vidal</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Improving PCR detection of prey in molecular diet studies: Importance of groupâ€�specific primer set selection and extraction protocol performances</span>. <span class="source" tagx="source" title="source">Molecular Ecology Resources</span>, <span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">117</span>â€"<span class="lpage" tagx="lpage" title="lpage">127</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1755-0998.12029">10.1111/1755-0998.12029</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23134438">23134438</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0072" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zarzosoâ€�Lacoste</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ruffino</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vidal</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Limited predatory capacity of introduced black rats on bird eggs: An experimental approach</span>. <span class="source" tagx="source" title="source">Journal of Zoology</span>, <span class="volume" tagx="volume" title="volume">285</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">188</span>â€"<span class="lpage" tagx="lpage" title="lpage">193</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1469-7998.2011.00828.x">10.1111/j.1469-7998.2011.00828.x</a></span></span></li> <li tag="ref"><a name="ece37171-bib-0073" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zou</span>, <span class="given-names" tagx="given-names" title="given-names">K. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thebault</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lacroix</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Barot</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Interactions between the green and brown food web determine ecosystem functioning</span>. <span class="source" tagx="source" title="source">Functional Ecology</span>, <span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">1454</span>â€"<span class="lpage" tagx="lpage" title="lpage">1465</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1365-2435.12626">10.1111/1365-2435.12626</a></span></span></li> </ul> </div> </div>  </body></html>