<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7854645">7854645</a></span><span class="publisher-id" title="publisher-id">82238</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-021-82238-9">10.1038/s41598-021-82238-9</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Impact of root-associatedÂ strains of three <i>Paraburkholderia</i> species on primary and secondary metabolism of <i>Brassica oleracea</i></div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeon</span><span class="given-names" tagx="given-names" title="given-names">Je-Seung</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carreno-Quintero</span><span class="given-names" tagx="given-names" title="given-names">Natalia</span></span><a href="#Aff1">1</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Eekelen</span><span class="given-names" tagx="given-names" title="given-names">HenriÃ«tte D. L. M.</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Vos</span><span class="given-names" tagx="given-names" title="given-names">Ric C. H.</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raaijmakers</span><span class="given-names" tagx="given-names" title="given-names">Jos M.</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Etalo</span><span class="given-names" tagx="given-names" title="given-names">Desalegn W.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">D.Etalo@nioo.knaw.nl</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.418375.c</span><span class="institution-id" title="institution-id">0000 0001 1013 0288</span><span class="institution" title="institution">Department of Microbial Ecology, </span><span class="institution" title="institution">Netherlands Institute of Ecology NIOO-KNAW, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.5132.5</span><span class="institution-id" title="institution-id">0000 0001 2312 1970</span><span class="institution" title="institution">Institute of Biology, </span><span class="institution" title="institution">Leiden University, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.425600.5</span><span class="institution-id" title="institution-id">0000 0004 0501 5041</span><span class="institution" title="institution">KeyGene N.V., </span></div></span><span class="citation_author_institution" id="Aff4">[4], </span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-2-2</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-2-2</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">2781</span><span class="history" title="history"><span class="received" title="received">received: 2020-9-14</span><span class="accepted" title="accepted">accepted: 2021-1-14</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Several root-colonizing bacterial species can simultaneously promote plant growth and induce systemic resistance. How these rhizobacteria modulate plant metabolism to accommodate the carbon and energy demand from these two competing processes is largely unknown. Here, we show that strains of three <i>Paraburkholderia</i> species, <i>P. graminis</i> PHS1 (<i>Pbg</i>), <i>P. hospita</i> mHSR1 (<i>Pbh</i>), and <i>P. terricola</i> mHS1 (<i>Pbt</i>), upon colonization of the roots of two Broccoli cultivars led to cultivar-dependent increases in biomass, changes in primary and secondary metabolism and induced resistance against the bacterial leaf pathogen <i>Xanthomonas campestris.</i> Strains that promoted growth led to greater accumulation of soluble sugars in the shoot and particularly fructose levels showed an increase of up to 280-fold relative to the non-treated control plants. Similarly, a number of secondary metabolites constituting chemical and structural defense, including flavonoids, hydroxycinnamates, stilbenoids, coumarins and lignins, showed greater accumulation while other resource-competing metabolite pathways were depleted. High soluble sugar generation, efficient sugar utilization, and suppression or remobilization of resource-competing metabolites potentially contributed to curb the tradeoff between the carbon and energy demanding processes induced by <i>Paraburkholderia</i>-Broccoli interaction. Collectively, our results provide a comprehensive and integrated view of the temporal changes in plant metabolome associated with rhizobacteria-mediated plant growth promotion and induced resistance.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Biochemistry</div> <div class="kwd" title="kwd">
Chemical biology</div> <div class="kwd" title="kwd">
Ecology</div> <div class="kwd" title="kwd">
Microbiology</div> <div class="kwd" title="kwd">
Plant sciences</div> <div class="kwd" title="kwd">
Systems biology</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Plants allocate their photoassimilates for growth, storage, and defense, but they also release them into the rhizosphere feeding the microbial community<sup><a href="#CR1">1</a>,<a href="#CR2">2</a></sup>. Particularly, root exudation of soluble organic carbon is often omitted from the plant carbon balance as it is considered a minor component<sup><a href="#CR3">3</a></sup>. However, several studies suggested that 20â€"50% of the total fixed carbon can be released belowground from plant roots<sup><a href="#CR4">4</a>â€"<a href="#CR6">6</a></sup>. Low molecular weight organic compounds including amino acids, organic acids, sugars, phenolics, alcohols and polypeptides, are among the major constituents of root exudates, influencing the structure and function of the rhizosphere microbial community<sup><a href="#CR7">7</a></sup>. Root-associated bacteria, often referred as rhizobacteria, can form beneficial relationships with plants promoting growth and/or inducing defense<sup><a href="#CR8">8</a>,<a href="#CR9">9</a></sup>. Despite the vast amount of studies on plant growth promotion and induced systemic resistance by different rhizobacterial genera, relatively little is known to date about how rhizobacteria change plant chemistry and how these changes relate to the phenotypic changes in plant growth and resistance.</p> <p id="Par3">Plants produce various secondary metabolites that play key roles in the adaptation of plants to environmental changes, including tolerance to biotic stresses such as insect herbivory and pathogen infections<sup><a href="#CR10">10</a>â€"<a href="#CR13">13</a></sup>. Recent studies showed that a number of rhizobacteria elicit secondary metabolite accumulation, including metabolites involved in defense<sup><a href="#CR8">8</a>,<a href="#CR14">14</a>â€"<a href="#CR17">17</a></sup>. Such changes in plant secondary metabolism have costs associated with the biosynthesis, transport and storage of these molecules and with the competition for primary metabolites and energy needed for plant growth. The biosynthesis of primary and secondary metabolites depend on common precursors and have a trade-off at the biochemical level. Intriguingly, rhizobacteria are able to orchestrate balanced plant growth, plant defense and secondary metabolite production. Hence, investigation of how rhizobacteria influence plant primary and secondary metabolism can provide a road map of key metabolite targets that play a balancing act between plant growth and defense. Recent developments in metabolomics approaches provide valuable tools to assess the influence of rhizobacteria on the dynamics of primary and secondary metabolism and to identify key metabolite classes that have a broader impact on host plant growth and defense.</p> <p id="Par4">In the present study, we investigated the effects of root-colonizing strains of three <i>Paraburkholderia</i> species on the phenotypes of two Broccoli cultivars, in particular on growth and defense against the bacterial leaf pathogen <i>Xanthomonas campestris</i>. Broccoli (<i>Brassica oleracea</i> var. italica) is a crop plant known for high value natural compounds, such as glucosinolates and flavonoids<sup><a href="#CR18">18</a></sup>. In the present study, we selected two Broccoli cultivars, Coronado and Malibu, contrasting in their relative levels of glucosinolates (mainly glucoiberin, glucoraphanin and glucobrassicin). Glucosinolates are among the plant secondary metabolites involved in rhizobacteria-mediated bacterial pathogen resistance in the Brassica model species <i>Arabidopsis thaliana</i><sup><a href="#CR8">8</a></sup>. <i>Paraburkholderia</i> is a monophyletic clade diverged from the genus <i>Burkholderia</i><sup><a href="#CR19">19</a></sup>. A number of rhizospheric and endophytic <i>Paraburkhoderia</i> species, in particular <i>Paraburkholderia phytofirmans</i> PsJN, <i>P. fungorum</i> and <i>P. graminis,</i> have been shown to promote growth of maize, strawberry and Arabidopsis<sup><a href="#CR20">20</a>â€"<a href="#CR22">22</a></sup> and suppress pathogen infections<sup><a href="#CR23">23</a>,<a href="#CR24">24</a></sup>. In addition, various <i>Paraburkholderia</i> species are typically found in the mycosphere consuming organic acids released from fungi and using the hyphae as â€˜highwaysâ€™ for translocation<sup><a href="#CR25">25</a></sup>. The strains of <i>Paraburkholderia</i> species used in our study are <i>P. graminis</i> PHS1 (<i>Pbg</i>), <i>P. hospita</i> mHSR1 (<i>Pbh</i>), and <i>P. terricola</i> mHS1 (<i>Pbt</i>), which exhibited plant protection against the fungal root pathogen <i>Rhizoctonia solani.</i> For <i>P. graminis</i>, we further showed that the production of sulfurous volatiles was a key mechanism in disease suppression<sup><a href="#CR24">24</a></sup>. To begin to identify the bacterial traits associated with plant growth promotion, we screened live cells, heat-killed cells, cell-free culture supernatant and volatile compounds of <i>Pbg</i> and found that for growth promotion, Broccoli requires live <i>Pbg</i> cells.</p> <p id="Par5">Using non-targeted metabolomics, we assessed the impact of the strains of <i>Paraburkholderia</i> species on the temporal dynamics of shoot primary and secondary metabolisms of the two Broccoli cultivars. Our results showed common and specific signatures in both primary and secondary metabolism in the two Broccoli cultivars colonized by the strains of the <i>Paraburkholderia</i> species. The results also revealed that the enhanced accumulation of soluble sugars in shoots of Broccoli upon root colonization by <i>Paraburkholderia</i> coincided with distinct changes in secondary metabolism that in turn correlate with distinctive changes in plant growth and defense. The integrated strategy adopted in this study enhanced our fundamental understanding of metabolic changes associated with rhizobacteria-mediated plant growth and defense.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="paraburkholderiaspeciespromotebroccoligrowthinacultivar-dependentmanner" title="sec"> <div class="title" tagx="title" title="title"><i>Paraburkholderia</i> species promote Broccoli growth in a cultivar-dependent manner</div> <p id="Par6">Root tip inoculation of the two Broccoli cultivars with strains of three different <i>Paraburkholderia</i> species led to changes in leaf color (deep green leaves), shoot biomass, root biomass and root architecture (Fig.Â <a href="#Fig1">1</a>a). Percent change in biomass was used as a measure to assess the growth-promoting effects of the <i>Paraburkholderia</i> species in the two Broccoli cultivars. Two-way analysis of variance (ANOVA) was conducted to assess the influence of the two independent variables (strains of <i>Paraburkholderia</i> species and Broccoli cultivars) on both shoot and root biomass. The <i>Paraburkholderia</i> species included three levels (<i>Pbg</i>, <i>Pbh, Pbt</i>) and the Broccoli cultivars consisted of two levels (Coronado, Malibu). For shoots, all interactions, except <i>Pbt</i>-Malibu, resulted in significant increases in biomass relative to the non-treated control plants, while for roots all three <i>Paraburkholderia</i> species significantly increased the biomass in both Broccoli cultivars (Fig.Â <a href="#Fig1">1</a>b). In general, the relative impact of <i>Paraburkholderia</i> species was up to 3 times higher for root biomass than for shoot biomass (Fig.Â <a href="#Fig1">1</a>b). Two-way ANOVA showed highly significant interactions between the strains of <i>Paraburkholderia</i> species and Broccoli cultivars regarding the percent changes in shoot and root biomass (Supplementary Table <a href="#MOESM3">S1</a>). Overall, for cultivar Coronado the percent change in shoot biomass was about 40% compared to the control, and not significantly different between the different strains of <i>Paraburkholderia</i> species, whereas in cultivar Malibu the percent change in shoot biomass was significantly higher for <i>Pbg</i> (~â€‰70%) and <i>Pbh</i> (~â€‰90%) as compared to <i>Pbt</i>. Furthermore, inoculation with <i>Pbh</i> led to a significantly higher increase in shoot biomass in cultivar Malibu than in Coronado. Regarding the percent change in root biomass, only inoculation of <i>Pbt</i> showed significant differences between the two Broccoli cultivars. As indicated above, the shoot biomass of cultivar Malibu inoculated with <i>Pbt</i> was not significantly different from the control plants (Fig.Â <a href="#Fig1">1</a>b). Over a period of 11Â days, both <i>Pbg</i> and <i>Pbh-</i>treated Broccoli cultivars showed significantly higher shoot and root biomass from 7Â days post inoculation (dpi) onwards, while <i>Pbt</i>-treated plants showed higher shoot biomass in Coronado from 9 dpi onwards (Fig.Â <a href="#Fig1">1</a>c). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Biomass and phenotypic changes in Broccoli cultivars in response to root tip inoculation with strains ofÂ three <i>Paraburkholderia</i> species. (<b>a</b>) Pictures of MS agar plate with two Broccoli cultivars (Coronado and Malibu) at 11Â days post inoculation with strains ofÂ three <i>Paraburkholderia</i> speciesÂ (<i>Pbg</i>: <i>Paraburkholderia graminis</i> PHS1, <i>Pbh</i>: <i>P. hospita</i> mHSR1, and <i>Pbt</i>: <i>P. terricola</i> mHS1). (<b>b</b>) Percent changes in shoot and rootÂ biomass (meanâ€‰Â±â€‰standard error, nâ€‰=â€‰4Â (shoot) and n = 6 (root)) of two Broccoli cultivars inoculated with theÂ strains of the <i>Paraburkholderia</i> species. Treatments sharing the same letters are not significantly different (Two-way ANOVA, Tukeyâ€™s HSD post hoc test, <i>P</i>â€‰<div class="â€‰0.05)._UNKNOWN">
c) Temporal changes in shoot biomass of two Broccoli cultivars (Coronado and Malibu) inoculated with the <i>Paraburkholderia</i> species. AsterisksÂ in panels b and c denote significant differences from the non-treated control samplesÂ (two-tailed Studentâ€™s t test: *<i>P</i>â€‰<div class="â€‰0.05;_UNKNOWN">
Pâ€‰<div class="â€‰0.01)._UNKNOWN" /></div></div></p>   </div> <p /> </div> <div class="relationbetweenrootcolonizationandplantgrowthpromotion" title="sec"> <div class="title" tagx="title" title="title">
Relation between root colonization and plant growth promotion</div> <p id="Par7">The extent of root colonization of the strains of <i>Paraburkholderia</i> species was assessed for the two Broccoli cultivars at the early and late growth stages. The data was log-transformed, as they did not meet the ANOVA assumption for homogeneity of variance and normality. Three-way analysis of variance was conducted on the interaction effects of <i>Paraburkholderia</i> species strains, Broccoli cultivars and time after inoculation on root colonization. The <i>Paraburkholderia</i> species strains included three levels (<i>Pbg</i>, <i>Pbh</i> and <i>Pbt</i>), the Broccoli cultivars included two levels (Coronado, Malibu) and time after inoculation consisted of two levels (6 dpi, 11 dpi). There was a highly significant three-way interaction effect on root colonization (Supplementary Table <a href="#MOESM3">S2</a>). In general, <i>Pbg</i> showed significantly higher root colonization in both cultivars at both time points when compared to <i>Pbh</i> and <i>Pbt</i> (Fig.Â <a href="#Fig2">2</a> and Supplementary Table <a href="#MOESM3">S3</a>). In addition, <i>Pbg</i> showed significantly higher root colonization in cultivar Coronado at 6 dpi (2.1â€‰Â±â€‰0.1â€‰Ã—â€‰10<sup>8</sup> Cfu/mg roots) and at 11 dpi (8.1â€‰Â±â€‰0.3â€‰Ã—â€‰10<sup>7</sup> Cfu/mg roots). In cultivar Malibu root colonization by <i>Pbg</i> was not significantly different between the two time points (1.0â€‰Â±â€‰0.1â€‰Ã—â€‰10<sup>8</sup> Cfu/mg roots (6 dpi) and 1.0â€‰Â±â€‰0.1â€‰Ã—â€‰10<sup>8</sup> Cfu/mg roots (11 dpi)). <i>Pbt</i> showed significantly lower root colonization in cultivar Malibu at both time points. Furthermore, <i>Pbt</i> on Malibu showed a significant decline in root colonization at the later time point (1.7â€‰Â±â€‰0.2â€‰Ã—â€‰10<sup>6</sup> Cfu/mg roots (6 dpi) and 7.0â€‰Â±â€‰0.4â€‰Ã—â€‰10<sup>5</sup> Cfu/mg roots (11 dpi)). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Root colonization ability of strains ofÂ three <i>Paraburkholderia</i> species (sp)Â for two Broccoli cultivarsÂ (cv) at 6 and 11 days post inoculation (dpi). Means of 3 replicates are shown. Treatments sharing the same letter are not significantly different based on three-ways ANOVA (dpi, Broccoli cultivars and Bacteria species, Tukeyâ€™s HSD post hoc test,Â <i>P</i>â€‰<div class="â€‰0.05)._UNKNOWN">
Paraburkholderia species (<i>Pbg</i>: <i>Paraburkholderia graminis</i>Â PHS1, <i>Pbh</i>: <i>P. hospita</i>Â mHSR1, and <i>Pbt</i>: <i>P. terricola</i>Â mHS1).</div></p>   </div> <p /> </div> <div class="paraburkholderiaspeciesalteredprimaryandsecondarymetabolismofbroccolishoot" title="sec"> <div class="title" tagx="title" title="title"><i>Paraburkholderia</i> species altered primary and secondary metabolism of Broccoli shoot</div> <p id="Par8">Considering the extent of strain and cultivar-dependent variations in root colonization and plant growth promotion, we investigated the systemic effect of the bacterial strains on the shoot metabolome of the two Broccoli cultivars at 6 and 11 dpi. GCâ€"MS and LCâ€"MS-based non-targeted metabolomics analysis of shoot extracts were performed to profile the polar primary metabolites and semi-polar secondary metabolites, respectively. The data was subjected to ANOVA with correction for multiple testing (Benjamini-Hochberg) and metabolites that were significantly different (<i>P</i>â€‰<div class="â€‰0.05_UNKNOWN">
â€‰2) between at least two treatments were used for multivariate analysis. Principal Component Analysis (PCA) and Hierarchical Cluster Analysis (HCA) were used to reduce the dimensionality of the data and explore specific patterns of change in metabolomeÂ in the different plantâ€"rhizobacteria interactions.</div></p> <div class="effectsofparaburkholderiaonshootprimarymetabolism" title="sec"> <div class="title" tagx="title" title="title">
Effects of <i>Paraburkholderia</i> on shoot primary metabolism</div> <p id="Par9">GCâ€"MS-based non-targeted metabolomics demonstrated that out of the 138 polar metabolites detected, 68 (50%) were significantly different between at least two treatments. At 6 dpi, PCA indicated that the first three principal components (PCs) explained 62.8% of the total variance (Fig.Â <a href="#Fig3">3</a>a1). The first PC (PC1), explained 34.8% of the total variance and corresponded to the effect of the three <i>Paraburkholderia</i> treatments on the metabolome of both cultivars (Fig.Â <a href="#Fig3">3</a>b1, Clusters 1, 5, 8 and 9). <i>Pbg</i> had the greatest impact on shoot primary metabolism of both Broccoli cultivars, while inoculation with <i>Pbh</i> and <i>Pbt</i> resulted in changes in the shoot primary metabolome in a cultivar-dependent manner. <i>Pbh</i> had greater impact on shoot primary metabolome of Malibu, while <i>Pbt</i> had greater impact on shoot primary metabolome of Coronado (Fig.Â <a href="#Fig3">3</a>a1). The major changes in primary metabolism induced by <i>Paraburkholderia</i> included accumulation of sugars (Cluster 9) and depletion of amino acids (Cluster 5, phenylalanine, lysine and methionine) and phosphoenolpyruvate (PEP), a key intermediate in glycolysis and gluconeogenesis. Some of the representative metabolites in cluster 8 that showed accumulation in all interactions, except in the ineffective <i>Pbt</i>-Malibu interaction, include aspartic acid, mannonic acid and putrescine. The second principal component (PC2) explained 18.2% of the total variance and resulted from metabolites that showed variation between the two cultivars. Furthermore, treatment of the two cultivars with <i>Pbg</i> and <i>Pbh</i> widened the inherent variation in the level of some of the metabolites between the two Broccoli cultivars. (Fig.Â <a href="#Fig3">3</a>b1, Clusters 2 and 3). Amino acids such as glutamine, oxoproline (pyroglutamic acid), GABA (Î³-aminobutyric acid) and isoleucine were intrinsically higher in Coronado than in Malibu.</p> <p id="Par10">At the later seedling growth stage (11 dpi), the initial inoculation of the roots of the two Broccoli cultivars with the strains of the <i>Paraburkholderia</i> species continued to have substantial impact on shoot primary metabolism. PCA showed that the first three principal components explained 72.1% of the total variance (Fig.Â <a href="#Fig3">3</a>a2). Here, the impact of all the three strains of the <i>Paraburkholderia</i> species on the Broccoli shoot metabolome was cultivar dependent and was greater in Malibu (Fig.Â <a href="#Fig3">3</a>a2). The first principal component (PC1) explained 44.2% of the total variance and resulted from metabolites that were accumulated (Fig.Â <a href="#Fig3">3</a>b2, Cluster 5) or reduced (Cluster 2) in the <i>Paraburkholderia</i> treatments. The Broccoli metabolites that decreased upon inoculation with the strains of the <i>Paraburkholderia</i> species encompassed amino acids such as lysine, phenylalanine, methionine, the non-proteinogenic amino acids ornithine and GABA, as well as PEP. In all plantâ€"microbe combinations, except the ineffective partnership between <i>Pbt</i>-Malibu, PEP showed 11â€"14 fold decreases (Supplementary excel, Table <a href="#MOESM1">S6</a>). Sugars and other metabolites, including ascorbic acid and aspartic acid, represented the metabolites enhanced by the <i>Paraburkholderia</i> treatments when compared to the control plants (Cluster 5). Six days after treatment with <i>Paraburkholderia,</i> sugars showed greater abundance in cultivar Malibu than in cultivar Coronado (Fig.Â <a href="#Fig3">3</a>b2). However, at 11 dpi, sugars in <i>Paraburkholderia</i>-treated plants showed substantial depletion in cultivar Coronado as compared to 6 dpi (Fig.Â <a href="#Fig3">3</a>b1), whereas in cultivar Malibu, the temporal variation in the level of these sugars was less pronounced (Fig.Â <a href="#Fig3">3</a>b2, Cluster 5, Supplementary Figure <a href="#MOESM2">S3</a> and <a href="#MOESM2">S4</a>). PC2, representing 19.4% of the total variance, was associated with metabolites in cluster 1 including glycine, that were depleted in all treatment combinations except in the controls and in the ineffective partnership between <i>Pbt</i> and Malibu (Fig.Â <a href="#Fig3">3</a>b2, Cluster 1). Oxoproline and some other metabolites in cluster 3 were intrinsically abundant in the shoots of cultivar Coronado.</p> </div> <div class="paraburkholderiaimpactonbroccoliprimarymetabolismishighlyassociatedwithsolublesugars" title="sec"> <div class="title" tagx="title" title="title"><i>Paraburkholderia</i> impact on Broccoli primary metabolism is highly associated with soluble sugars</div> <p id="Par11">As sugars are the primary drivers of plant growth, we looked into their temporal dynamics, particularly related to sugar generation and utilization in the shoots of the two Broccoli cultivars treated with the strains of the <i>Paraburkholderia</i> species. The fold change in sugar level between <i>Paraburkholderia</i> treated and control plants at 6 dpi was used as a measure of sugar generation, while the fold change in sugar level of treated plants from 6 to 11 dpi was used as a measure of sugar utilization. In control plants, the sugar levels showed no significant difference between the two Broccoli cultivars at 6 dpi (supplementary Fig. <a href="#MOESM2">S3</a>). In contrast, treatment with the strains of the <i>Paraburkholderia</i> species showed substantial impact on the sugar generation in shoots of both Broccoli cultivars, resulting in significant increases in the level of fructose and its derivatives, glucose, sorbose, galactose and galactopyranose at 6 dpi. Moreover, the magnitude of sugar generation showed remarkable differences between the strains of the <i>Paraburkholderia</i> species-Broccoli cultivar combinations (Fig.Â <a href="#Fig3">3</a>c1). <i>Pbg</i> treatment resulted in the highest sugar generation when compared to <i>Pbh</i> and <i>Pbt,</i> and this ability was significantly higher in cultivar Malibu than in Coronado. The ineffective partnership between <i>Pbt</i> and Malibu had the least impact on sugar generation. Similarly, the utilization of sugar also showed noticeable differences among the strains of the <i>Paraburkholderia</i> species-Broccoli cultivar combinations. In Coronado, <i>Pbg</i> inoculation led to greater sugar utilization when compared to cultivar Malibu. The ineffective partnership between <i>Pbt</i> and Malibu showed reduced sugar utilization when compared to the effective partnership of <i>Pbt</i> with Coronado (Fig.Â <a href="#Fig3">3</a>c2). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p><i>Paraburkholderia</i>-mediated changes in shoot primary metabolites in two Broccoli cultivars. (<b>a</b>) Principal Component Analysis (PCA) and (<b>b</b>) Hierarchical Cluster Analysis (HCA) based on differentially regulated metabolites of the samples at 6 dpi (<b>1</b>) and 11 dpi (<b>2</b>). In the HCA, metabolite clusters are indicated by different colors. Information on the representative metabolites of each clusters is given on the right side, if the metabolites are annotated. (<b>c</b>) Impact of <i>Paraburkholderia</i> species on sugar generation (<b>1</b>)Â and utilizationÂ (<b>2</b>) of two Broccoli cultivars. Broccoli cultivars (Cor: Coronado, Mal: Malibu), Cont.: non-rhizobacteria treated control, <i>Pbg</i>: <i>Paraburkholderia graminis</i>Â PHS1, <i>Pbh</i>: <i>P. hospita</i> mHSR1,Â and <i>Pbt</i>: <i>P. terricola</i>Â mHS1.</p>   </div> <p /> </div> <div class="effectsofparaburkholderiaonshootsecondarymetabolism" title="sec"> <div class="title" tagx="title" title="title">
Effects of <i>Paraburkholderia</i> on shoot secondary metabolism</div> <p id="Par12">From the 1,868 metabolites detected by LCMS, 1,386 (74%) were significantly different between at least two treatments. PCA of the metabolites at 6 dpi demonstrated distinct clustering of the samples based on the strains of the <i>Paraburkholderia</i> species-Broccoli cultivar combination (Fig.Â <a href="#Fig4">4</a>a1). Here PC1 explained 33.2% of the total variation and was associated with sample differences due to metabolites that were intrinsically more abundant in one of the two Broccoli cultivars (Clusters 3, 5 and 12, Fig.Â <a href="#Fig4">4</a>b1). Metabolites that were intrinsically more abundant in Coronado included aliphatic glucosinolates such as 2-methylbutyl glucosinolate and glucoiberverin as well as the aromatic glucosinolates glucotropaeolin and gluconasturtiin. The levels of 2-methylbutyl glucosinolate and glucoiberverin were 147 and 209 times higher in Coronado than in Malibu, respectively (Cluster 3). In Malibu, on the other hand, a number of phenolic compounds were intrinsically more abundant (Clusters 5 and 12).</p> <p id="Par13">The second principal component (PC2) explained 20.9% of the total variance and was associated with metabolites that were reduced (Fig.Â <a href="#Fig4">4</a>b1) Clusters 2 and 4) or induced (Clusters 7 and 11) by the <i>Paraburkholderia</i> treatments. Treatment of both cultivars with the <i>Paraburkholderia</i> species also widened the intrinsic cultivar variation in metabolites. Inoculation of <i>Pbg</i> had the greatest impact on the shoot secondary metabolome profile of both Broccoli cultivars, whereas the ineffective partnership between <i>Pbt</i> and Malibu had less pronounced impact on the shoot metabolome. Metabolites in cluster 2, comprising amino acids such as arginine, asparagine, tryptophan and N-acetylated glutamic acid/fucosamine, showed greater reduction in their abundance upon treatment with strains of the <i>Paraburkholderia</i> species when compared to the control. Cluster 4 encompassed metabolites that were more abundant in Malibu than in Coronado and included ascorbic acid ethyl ester, N-acetyl-tryptophan, and terpenoids putatively annotated as such as S-furanopetasitin and sonchuionoside C. The metabolites in clusters 7 and 11 were induced by all the strains of <i>Paraburkholderia</i> species and were dominated by phenolic compounds. In Malibu, inoculation of <i>Pbg</i> led to greater accumulation of flavonoids glycosides (i.e. kaempferol-di/tri-(feruloyl/coumaroyl glycosides and robinin), hydroxycinnamates (ferulic acid and its derivatives, caffeic acid derivatives such as chlorogenic acid) and indole-3-acetic-acid-O-glucuronide when compared to the other two <i>Paraburkholderia</i> species.</p> <p id="Par14">PC3 explained 4.9% of the total variance and was represented by <i>Pbg</i>-induced (Clusters 8 and 10) or <i>Pbt</i>-induced (Cluster 13) metabolites in both Broccoli cultivars. <i>Pbg</i>-enhanced metabolites in cluster 8 consisted of the flavonoid kaempferol 3-sophorotrioside, whereas <i>Pbt</i>-enhanced metabolites in cluster 13 included the hydroxycinnamate O-sinapoyl-beta-D-glucoside and resveratrol-sulfoglucoside, a stilbenoid.</p> <p id="Par15">Similarly, at 11dpi, inoculation with the strains of the <i>Paraburkholderia</i> species led to substantial changes in the shoot metabolite profiles of the two Broccoli cultivars (Fig.Â <a href="#Fig4">4</a>a2,b2). In the PCA, the first three PCs explained 51.1% of the total variance. The first PC, explaining 29.1% of the total variance is associated with metabolites that accumulated or were reduced in response to <i>Paraburkholderia</i> and the change in these groups of metabolites was more pronounced in Malibu cultivar (Fig.Â <a href="#Fig4">4</a>b2, Clusters 1, 2, 3, 4 and 5: up; 10 and 11: down). The induced metabolites in the above-mentioned clusters included flavonoids i.e. kaempferol-di/tri-glycosides (feruloyl/caffeoyl/coumaroyl), robinin, medicarpin-O-glucoside-malonate, as well as hydroxycinnamates, i.e. ferulic acid, caffeic acid and various derivatives of these metabolites. Furthermore, <i>Paraburkholderia</i> also induced coumarins such as eupatoriochromene and mahaleboside and mevalonate, a precursor of mevalonate pathways that goes into terpenoid biosynthesis. The reduced metabolites in both Broccoli cultivars included amino acids such as arginine, asparagine and N-acetylglutamic acid (Cluster 10). Meanwhile, metabolites in cluster 11 were also reduced by the <i>Paraburkholderia</i> treatment and these metabolites were intrinsically more abundant in cultivar Coronado. Some of the metabolites in cluster 11 included sulfur-containing metabolites such as 2-methylbutyl glucosinolate and glucoiberverin, derivatives of sulfurous amino acids including leucyl-cysteine and methionyl-isoleucine, as well as precursor or breakdown products of glucosinolates, for instance 6-methylthiohexanaldoxime and 3-methylsulfinylpropyl isothiocyanate.</p> <p id="Par16">The second PC (PC2) explained 23.8% of the total variance and was due to metabolites that were intrinsically more abundant in cultivar Malibu (Fig.Â <a href="#Fig4">4</a>b2, Clusters 6, 7, 8 and 9). Metabolites in clusters 6, 7, 8 and 9 showed significant reduction in all effective partnerships. Tryptophan, a building block for indolic glucosinolate and the growth hormone indole-3-acetic acid, N-acetylated amino acids including N-acetyl phenylalanine/tryptophan, terpenoids, i.e. S-furanopetasitin and sonchuionoside C, and sulforaphane, an isothiocyanate, are some of the metabolites in these clusters worth mentioning.</p> <p id="Par17">PC3 explained 6.2% of the total variance and was associated with yet unknown metabolites that showed <i>Pbg</i> specific alteration in both Broccoli cultivars (Clusters 13). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Rhizobacteria-mediated changes in the shoot secondary metabolites in Broccoli cultivars. (<b>a</b>) Principal component analysis (PCA) and (<b>b</b>) Hierarchical cluster analysis (HCA) based on differentially regulated metabolites of the samples at 6 dpi (<b>1</b>) and 11 dpi (<b>2</b>). In the HCA, metabolite clusters are indicated by different colors. Information on the representative metabolites of each clusters is given on the right side, if the metabolites are annotated. Broccoli cultivars (Cor: Coronado, Mal: Malibu), Cont.: non-rhizobacteria treated control, <i>Pbg</i>: <i>Paraburkholderia graminis</i>Â PHS1, <i>Pbh</i>: <i>P. hospita</i> mHSR1Â and <i>Pbt</i>: <i>P. terricola</i> mHS1. *GLSâ€‰=â€‰glucosinolate, **Dâ€‰=â€‰derivative.Â Some primary metabolites were also detected in the semi-polar fraction of the shoot extract, eluting at the early stage of the chromatographic separation and are listed among the secondary metabolites in panels b1 and b2.</p>   </div> <p /> </div> </div> <div class="paraburkholderiainducessystemicresistanceagainstthebacterialleafpathogenxanthomonascampestrisinacultivar-dependentmanner" title="sec"> <div class="title" tagx="title" title="title"><i>Paraburkholderia</i> induces systemic resistance against the bacterial leaf pathogen <i>Xanthomonas campestris</i> in a cultivar-dependent manner</div> <p id="Par18">As shown above, the two Broccoli cultivars exhibited inherent differences in their shoot chemistry (Fig.Â <a href="#Fig4">4</a>). Furthermore, treatment of the plant roots with strains of the <i>Paraburkholderia</i> species led to substantial alteration of the shoot metabolome including metabolome signatures specific to the individual combination of <i>Paraburkholderia</i> species and Broccoli cultivar (Fig.Â <a href="#Fig4">4</a>). Based on this, we hypothesized that the inherent and induced differences in shoot chemistry between the two cultivars could contribute to a differential defense response against leaf pathogens. To address this hypothesis, treated and control plants of the two cultivars were challenged with two bacterial leaf pathogens, i.e. <i>Xanthomonas campestris</i> pv.Â armoraciae P4216 (<i>Xca</i>) and <i>Xanthomonas campestris</i> pv. campestris P4014 (<i>Xcc</i>).</p> <p id="Par19">The interaction effect of two independent variables (<i>Paraburkholderia</i> species and Broccoli cultivars) on disease severity of the two bacterial pathogens was assessed using beta regression. The strains of the <i>Paraburkholderia</i> species included three levels (<i>Pbg</i>, <i>Pbh</i> and <i>Pbt</i>) and the Broccoli cultivars consisted of two levels (Coronado and Malibu). There was a highly significant interaction effect of the strains of the <i>Paraburkholderia</i> species and Broccoli cultivars on disease severity on both <i>Xanthomonas</i> pathovars (Supplementary Table <a href="#MOESM3">S4</a>). No significant inherent variation in disease severity was observed between the two Broccoli cultivars when control plants were challenged with the two bacterial pathogens (Fig.Â <a href="#Fig5">5</a>). However, treatment of the roots with <i>Paraburkholderia</i> led to a clear reduction or enhancement of disease severity. For example, treatment with <i>Pbg</i> and <i>Pbh</i> enhanced disease severity by 18â€"28% in cultivar Coronado challenged by both bacterial pathogens, whereas <i>Pbh</i> and <i>Pbt</i> significantly reduced the disease severity in cultivar Malibu challenged by <i>Xca</i> (47% and 30%, respectively) and <i>Xcc</i> (33% and 28%, respectively) (Fig.Â <a href="#Fig5">5</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Impact of root-colonizing <i>Paraburkholderia</i> species on defense of Broccoli cultivars against two bacterial leaf pathogens. Disease severity index of two broccoli cultivars pretreated with either one of the three <i>Paraburkholderia</i> species and challenged with two bacterial leaf pathogens <i>Xanthomonas campestris</i> pv. armoraciae P4216 (<i>Xc</i>a) and <i>Xanthomonas campestris</i> pv. campestris P4014 (<i>Xc</i>c). Broccoli cultivars (Cor: Coronado, Mal: Malibu), Control: non-treated control, <i>Pbg</i>: <i>Paraburkholderia graminis</i>Â PHS1, <i>Pbh</i>: <i>P. hospita</i>Â mHSR and <i>Pbt</i>: <i>P. terricola</i> mHS1.Â Treatments sharing the same letter are not significantly different (Two-way ANOVA, Tukeyâ€™s HSD post hoc test, <i>P</i>â€‰<div class="â€‰0.05)._UNKNOWN" /></p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par20">Our results show that strains of root-colonizing <i>Paraburkholderia</i> species alter shoot primary and secondary metabolism of Broccoli seedlings, promote their growth and induce systemic defense against the bacterial leaf pathogen <i>Xanthomonas campestris</i>. The magnitude of the alteration of these traits is dependent on the <i>Paraburkholderia</i> strain-Broccoli cultivar combination. The widely accepted â€œgrowth-defense tradeoffâ€� concept asserts that activation of plant defense comes at the expense of plant growth due to resource limitations<sup><a href="#CR26">26</a></sup>. Here we showed that some rhizobacterial <i>Paraburkholderia</i> strains can promote plant growth and at the same time induce plant defense against biotic stress factors. Rhizobacteria-induced defense is considered to have lower cost when compared to the activation of direct defense<sup><a href="#CR27">27</a>,<a href="#CR28">28</a></sup>. However, assessing some key physiological processes such as seed production, number of flowers, pollen quality and number, and plant growth does not necessarily explain the energy and carbon costs associated with defense priming. Hence, to begin to understand the underlying mechanisms by which rhizobacteria can promote growth and simultaneously prime the plantâ€™s defense without compromising plant fitness, a comprehensive investigation of the temporal changes induced in the host metabolome network is needed.</p> <p id="Par21">The two Broccoli cultivars used in this study showed inherent differences in their shoot metabolome profile: phenolics were more abundant in cultivar Malibu and glucosinolates and other sulfur-containing compounds showed higher abundance in cultivar Coronado. The <i>Paraburkholderia</i> strains exerted a substantial impact on primary and secondary metabolism at both early and later stages of Broccoli seedling growth. Their biggest impact on primary metabolism was reflected in the generation and utilization of soluble sugars, and both parameters showed significant variation across the strains of the <i>Paraburkholderia</i> species-Broccoli cultivar combinations (Fig.Â <a href="#Fig3">3</a>c). All combinations, except <i>Pbt</i>-Malibu, resulted in an effective partnership, i.e. plant growth promotion. These effective partnerships showed high soluble sugar generation at the early stage of seedling growth and high sugar utilization at the later stage of seedling growth, whereas the ineffective partnership between Malibu and <i>Pbt</i> showed a lower sugar generation and utilization. Soluble sugars are fuel for plant growth and for the biosynthesis of secondary metabolites involved in defense<sup><a href="#CR29">29</a>â€"<a href="#CR31">31</a></sup>. Moreover, soluble sugars such as galactose, glucose, sorbose, fructose, sucrose and xylose have been reported to be effective chemotaxis agents for bacteria<sup><a href="#CR32">32</a>â€"<a href="#CR34">34</a></sup>. In general, low sugar concentration promotes â€˜sourceâ€™ activities such as photosynthesis, nutrient mobilization and export, while high sugar level enhances â€˜sinkâ€™ activities such as growth and sugar storage<sup><a href="#CR35">35</a></sup>. Hence, the enhancement of the levels of plant soluble sugars such as sorbose and fructose by beneficial <i>Paraburkholderia</i> strains can be considered as one of the metabolic signatures of an effective partnership. Furthermore, under effective partnership, there was significant depletion (&amp;gt;â€‰11-fold) of PEP, a key substrate for the TCA cycle and the shikimate pathway (Fig.Â <a href="#Fig6">6</a>), whereas PEP depletion was only about twofold in the ineffective partnership. Furthermore, greater depletion of GABA under the effective partnership suggests catabolism of GABA to succinyl semialdehyde followed by its conversion to succinate to feed the greater demand of pyruvate in the TCA cycle. Key intermediates in the TCA cycle such as citric acid and malic acid showed increased abundance under effective partnerships. In the TCA cycle, citrate is converted to malate and used in the mitochondria for energy production<sup><a href="#CR36">36</a></sup>. Hence, these observations suggest that the beneficial <i>Paraburkholderia</i>-Broccoli interactions most likely require a greater demand for carbon and energy needed for enhanced growth and defense priming. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Alteration of core primary and secondary metabolite pathways by strains ofÂ <i>Paraburkholderia</i> species in Broccoli. The metabolite pathways are organized as modules inside different colored boxes and the abundance of the significantly altered metabolites is depicted by the heat map, with each cells representing the abundance of a metabolite of a sample for <i>Paraburkholderia</i> species-Broccoli cultivar and time combinations. The metabolite ID corresponding to each metabolites is indicated at the top of the heat map and detailed information on the identity of the individual metabolites is provided in the supplementary excel Table <a href="#MOESM1">S9</a>. G-6P (Glucose 6-phosphate), F-6P (Fructose 6-phosphate), CHS (Chalcone synthase), cisZOG1 (Cis-zeatin O-glucosyltransferase 1), CYP (Cytochrome P450), GABA (Î³-aminobutyric acid), GLS (Glucosinolates), GSH (Glutathione), HCT (hydroxycinnamoyl-CoA shikimate/quinate hydroxycinnamoyl transferase), IAA (indole-3-acetic acid), IAN (indoleâ€�3â€�acetonitrile), IAOx (indole-3-acetaldoxime), IPP (isopentenyl pyrophosphate), MAM (methylthioalkylmalate synthase), and STS (stilbene synthase). Broccoli cultivars (Cor: Coronado, Mal: Malibu), Cont.: non-rhizobacteria treated control, <i>Pbg</i>: <i>Paraburkholderia graminis</i>Â PHS1, <i>Pbh</i>: <i>P. hospita</i>Â mHSR1, and <i>Pbt</i>: <i>P. terricola</i>Â mHS1. Multiple-headed arrows indicate hidden intermediate processes in the pathways.</p>   </div> <p /> <p id="Par22">The relationship between root colonization density of the <i>Paraburkholderia</i> strains and the corresponding growth promotion effect on Broccoli cultivars was not linear. Past studies on doseâ€"response relationships in bacteria-plant interactions indicated that increasing the rhizobacteria densities beyond the threshold density required to induce a given phenotype did not further enhanced the impact on the phenotype <sup><a href="#CR37">37</a>,<a href="#CR38">38</a></sup>. The <i>Pbg</i>-Malibu interaction showed the highest root colonization and had the highest impact on shoot fructose level (~â€‰&amp;gt;â€‰280 folds). In contrast, <i>Pbt</i>-Malibu and <i>Pbh</i>-Coronado interactions showed the lowest root colonization levels and had the lowest impact on shoot fructose generation at the early stages of seedling growth. Interestingly, <i>Pbg</i> also showed the biggest impact on secondary metabolism in general, and on phenolic compounds accumulation in particular, when compared to <i>Pbh</i> and <i>Pbt</i> at 6 dpi (Figs. <a href="#Fig4">4</a>a1,b1, <a href="#Fig6">6</a>). This suggests that the effects of elevated levels of soluble sugars might not only be limited to plant growth but also extended to secondary metabolites biosynthesis. For example, fructose is the primary substrate for fructose-6-phospate, a key substrate for the biosynthesis of both PEP and erythros-4-phosphate (Fig.Â <a href="#Fig6">6</a>). The latter two intermediates are channeled to the shikimate pathway that bridges carbohydrate metabolism to biosynthesis of aromatic primary and secondary metabolites<sup><a href="#CR29">29</a></sup>. The shikimate pathway provides all the important precursors for the biosynthesis of phenolic compounds including hydroxycinnamates, flavonoids, stilbenoids, coumarins and lignins, that all showed significant accumulation in plants treated with <i>Paraburkholderia</i> (Figs. <a href="#Fig4">4</a>b, <a href="#Fig6">6</a>). A study on the <i>pho3</i> mutant of Arabidopsis, which accumulates soluble sugars to high levels, showed large increases in the expression of transcription factors and enzymes involved in anthocyanin biosynthesis<sup><a href="#CR39">39</a></sup>. Another study, on quinoa cotyledons, also showed accumulation of both fructose (~â€‰tenfold) and flavonoids in response to UV-B radiation<sup><a href="#CR40">40</a></sup>. Considering our results and these previous findings by others, we postulate that one of the key mechanisms by which rhizobacteria modulate host secondary metabolism is by soluble sugar generation. </p> <p id="Par23">In addition to enhancement of key precursors for growth and secondary metabolite biosynthesis, treatment of Broccoli roots with <i>Paraburkholderia</i> also induced metabolite remobilization. The metabolite re-direction involved both suppression of resource competing metabolite pathways, such as amino acids, and rechanneling of existing primary metabolite-derivatives and other secondary metabolites to other metabolite pathways. For example, aromatic glucosinolates, amino acids and their derivatives, and some terpenoids showed more depletion under effective partnerships (Figs.Â <a href="#Fig4">4</a>b2, <a href="#Fig6">6</a>). Hence, re-direction of existing metabolites towards specific metabolite pathways could be an additional strategy used by rhizobacteria to reduce the costs of de novo biosynthesis of metabolites.</p> <p id="Par24">Our results also showed that rhizobacteria-mediated reorganization of the host metabolome landscape affected not only plant growth but also the defense response of Broccoli cultivars to the bacterial leaf pathogen <i>Xanthomonas</i>. In general, inoculation of the strains of the <i>Paraburkholderia</i> species showed greater suppression of pathogen proliferation in cultivar Malibu that intrinsically accumulates higher levels of phenolic compounds (Fig.Â <a href="#Fig5">5</a>). The phenolic pathway in leaves appears to be the central target by the <i>Paraburkholderia</i> species and was altered more in Malibu than in Coronado (Fig.Â <a href="#Fig6">6</a>). All metabolite classes belonging to this pathway including flavonoids, hydroxycinnamates, stilbenoids, coumarins and lignins showed substantial accumulation upon treatments with the strains of the three <i>Paraburkholderia</i> species (Fig.Â <a href="#Fig6">6</a>). These metabolites have direct antimicrobial effects and/or act as a physical barrier against pathogenic microorganisms<sup><a href="#CR41">41</a>,<a href="#CR42">42</a></sup>. For example, hydroxycinnamic acids and flavonoids were shown to negatively affect the disease symptom development in Chinese cabbage challenged by <i>Xanthomonas campestris</i> pv. campestris (<i>Xcc</i>)<sup><a href="#CR43">43</a></sup>. Of all the <i>Paraburkholderia</i> species-Broccoli cultivars combinations, treatment of roots of cultivar Coronado with <i>Pbg</i> and <i>Pbh</i> resulted in higher disease severity (Fig.Â <a href="#Fig5">5</a>). However, we could not trace metabolites that are specifically induced or reduced in Coronado cultivar that is primed with <i>Pbg</i> and <i>Pbh</i>. Priming is a complex process that alters various components of the plant immune system, including key plant hormones with a role in defence signalling (such as SA, JA and Ethylene), pathogenesis-related proteins and phytoalexins. Priming might have resulted in broad-spectrum/cross resistance or compromised resistance that is effective only against certain group of microbes/herbivores<sup><a href="#CR44">44</a>,<a href="#CR45">45</a></sup>. Hence, the case of Coronado cultivar primed with <i>Pbh</i> and <i>Pbg</i> might fall under compromised resistance, and perhaps those plants might be resistant against other fungal/bacterial pathogens. Moreover, induced resistance against a pathogen might not exclusively be explained by the associated change in the metabolome as priming alters wide ranges of components of the plant immune machinery. Even the combined untargeted metabolite profiling used in our study does not cover all the metabolites that are produced during the interaction of the plants with the rhizobacteria. For example, metabolites of paramount importance in plantâ€"microbe interactions, such as phospholipids, other non-polar metabolites and volatile organic compounds were not analysed in our study. Profiling of such a wider array of metabolite groups could provide other metabolite signatures of induced disease susceptibility. </p> <p id="Par25">Defense priming is not a low-cost defensive measure but could cost substantial amounts of energy and carbon resources. This is shown by the massive accumulation of phenolics and other defensive compounds in plants treated with the strains of the <i>Paraburkholderia</i> species even before the plants were challenged with the bacterial leaf pathogens. The integrated primary and secondary metabolome profiling of primed plants suggests that rhizobacteria could avert the negative impact of defense priming on the host fitness by generating substantial amounts of soluble sugars and remobilizing other metabolites to accommodate for the high energy and carbon skeleton demand associated with growth and defense priming. This suggests that defense costs can be regulated, if resources are not limiting. This hypothesis aligns with studies that showed that the inevitability of growth-defense trade-off occurs primarily under resource-limiting conditions<sup><a href="#CR46">46</a>,<a href="#CR47">47</a></sup>.</p> <p id="Par26">It should be noted that the physiological state of the plants grown on MS media in our experimental design may be different from that of Broccoli plants grown in agricultural fields. Nevertheless, we used proper controls for all experiments and the effect of the rhizobacteria on different phenotypic and metabolomic traits were evaluated by comparing them with mock-inoculated plants. Our data indicated that treatment of the plants with the strains of the <i>Paraburkholderia</i> species alter the baseline growth level of the plants, their metabolism and their response to biotic stress in a <i>Paraburkholderia</i> strain-Broccoli cultivar specific manner. Hence, our comprehensive metabolomic and phenotypic analyses provided the first essential step in understanding the temporal changes in plant metabolism induced by root-colonizing bacteria and their association with plant growth promotion and plant defense. The observed and discussed metabolic hallmarks associated with effective partnerships between the <i>Paraburkholderia</i> strains and Broccoli cultivars potentially play a regulatory role in carbon and energy economy of the plant to ensure sustained growth and defense. Further extensive transcriptional and mutational analyses of the bacterial strains and its host plant as well as greenhouse and field trials will be the next steps for understanding of the functional importance of the specific changes in phenotype and metabolome induced by these rhizobacteria. These elaborate analyses also involving the model Brassica plant species Arabidopsis could lay a blueprint for a new microbiome-based plant breeding strategy to produce crops in which high yielding and stress-resilience go hand in hand.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="bacterialstrainsandcultureconditions" title="sec"> <div class="title" tagx="title" title="title">
Bacterial strains and culture conditions</div> <p id="Par27">The epiphytic rhizobacterial strains <i>Paraburkholderia graminis</i> PHS1 (<i>Pbg</i>), <i>P. hospita</i> mHSR1(<i>Pbh</i>), and <i>P. terricola</i> mHS1 (<i>Pbt</i>) used in this study were originally isolated from rhizospheric soil of <i>Beta vulgaris</i> grown in <i>Rhizoctonia solani</i> suppressive soil<sup><a href="#CR24">24</a></sup>. Cultures of <i>Paraburkholderia</i> species were maintained in Luria Bertani (LB)-medium (Lennox, Carl Roth) at 25 â"ƒ. After incubation for 16Â h, bacteria cells pellets were recovered by centrifugation. These pellets were then washed three times with 10Â mM MgSO<sub>4</sub> and resuspended in 10Â mM MgSO<sub>4</sub> to a final density of OD<sub>600</sub>â€‰=â€‰1.0 (~â€‰10<sup>9</sup> cells per ml).</p> </div> <div class="plantmaterialsandgrowthconditions" title="sec"> <div class="title" tagx="title" title="title">
Plant materials and growth conditions</div> <p id="Par28">Seeds of two Broccoli cultivars (<i>Brassica oleracea</i> var. italica), Coronado and Malibu, were kindly provided by Bejo Seeds (Warmenhuizen, The Netherlands). The seeds were surface sterilized for 30Â min by immersing them in 1% (v/v) sodium hypochlorite amended with 0.1% (v/v) of Tween 20, and rinsed three times with ample sterile distilled water. Thereafter, six seeds were sown on 100 X 100Â mm square petri dishes containing 50Â ml of half-strength Murashige and Skoog (0.5 X MS) agar media with 0.5% sucrose (w/v) and 1.2% plant agar (w/v). Five days after germination and vegetative growth in petri dishes, the root tips of the seedlings from the two Broccoli cultivars were inoculated with 2Â Âµl cell suspension (Â±â€‰10<sup>9</sup> cells per ml) of each strain of the three <i>Paraburkholderia</i> species. Plants treated with 2Â Âµl 10Â mM MgSO<sub>4</sub> served as controls. The determination of the initial density was based on our earlier studies on plant growth promotion and induced systemic resistance by <i>Pseudomonas fluorescens</i> SS101<sup><a href="#CR8">8</a>,<a href="#CR9">9</a></sup>. Moreover, comparison of different initial inoculum density of <i>Pbg</i> on Arabidopsis roots showed no significant influence on the final root bacterial density and on plant growth promotion (data not shown). The plates with the control and inoculated plants were then sealed and incubated in a climate chamber (21Â Â°C / 21Â Â°CÂ day/night temperature; 180Â Âµmol light m<sup>-2</sup>Â s<sup>-1</sup> at plant level during 16Â h/d; 70% relative humidity) until harvest (11Â days post inoculation). Temporal changes in shoot fresh biomass were measured every two days until harvest.</p> </div> <div class="rhizobacteriarootcolonizationassay" title="sec"> <div class="title" tagx="title" title="title">
Rhizobacteria root colonization assay</div> <p id="Par29">Bacterial root colonization was determined at 6 and 11 dpi for each of the three <i>Paraburkholderia</i> species on each of the two Broccoli cultivars. Briefly, treated roots were collected at 6 and 11 dpi and placed in sterile 50Â mL Falcon tubes and its biomass was measured. Then the root samples were vortexed (60Â s) in 10Â mM MgSO<sub>4</sub>, sonicated (60Â s), and again vortexed (15Â s) to re-suspend the bacteria adhering to the root. The suspensions were serial dilution plated onto PSA plates containing 100Â ÂµgÂ ml<sup>-1</sup> delvocid (DSM) to inhibit fungal growth. Plates were incubated at 25Â Â°C in the dark for 3Â days, colonies were counted and the number of colony-forming units (Cfu) per gram of root fresh weight was calculated.</p> </div> <div class="plantphenotyping" title="sec"> <div class="title" tagx="title" title="title">
Plant phenotyping</div> <p id="Par30">Fresh biomass of the Broccoli shoots was measured to determine the effect of the rhizobacteria on plant growth. For Broccoli, shoot fresh biomass from the respective treatments was weighed every two days after bacterial inoculation until the last harvest at 11 dpi. The average weight of 6 Broccoli seedlings was considered as one independent biological replica. The roots were carefully removed from the MS-agar and washed with distilled water to eliminate adhering agar, blotted dry on filter paper and their fresh weight was recorded.</p> </div> <div class="inducedresistanceassays" title="sec"> <div class="title" tagx="title" title="title">
Induced resistance assays</div> <p id="Par31">To assess the impact of <i>Paraburkholderia</i> species on induced resistance, the two Broccoli cultivars were inoculated with the three <i>Paraburkholderia</i> species and grown for 11Â days. Thereafter, leaves were inoculated with the bacterial leaf pathogens <i>Xanthomonas campestris</i> pv. Armoraciae P4216 (<i>Xc</i>a) and <i>Xanthomonas campestris</i> pv. Campestris P4014 (<i>Xc</i>c). To do that, <i>Xc</i>a and <i>Xc</i>c were cultured in LB-medium at 25 â"ƒ. After 16Â h, bacterial cells were washed following a similar procedure for <i>Paraburkholderia</i> as described above. A 2Â Âµl suspension of <i>Xc</i>a or <i>Xc</i>c (~â€‰1 X 10<sup>9</sup> cell per ml) was inoculated on the first true leaf of the Broccoli seedlings after scratching the leaf surface with sterile 20Â Âµl pipet tips. Ten days after pathogens challenge, disease severity on shoot was assessed by determining the migration of lesion from the inoculation spot to the other parts of the shoot based on a 0-to-5 ordinal scale as shown in supplementary Fig. <a href="#MOESM2">S2</a>: 1â€‰=â€‰no necrosis or migration, 2â€‰=â€‰necrosis of the treated leaf, 3â€‰=â€‰migration of the lesions to the leafstalk of the treated leaf, 4â€‰=â€‰necrotic or water-soaked lesions of the neighboring (nontreated) leaf, and 5â€‰=â€‰infection of the entire seedling. Severity values were converted to 0 to 100 Disease severity index (DSI) according to the following equation used by<sup><a href="#CR48">48</a></sup>. DSI (%)â€‰=â€‰âˆ‘ (scores of all plants)/ [Maximum disease score x (total number of plants)]â€‰Ã—â€‰100.</p> </div> <div class="plantmetabolomics" title="sec"> <div class="title" tagx="title" title="title">
Plant metabolomics</div> <div class="samplecollection" title="sec"> <div class="title" tagx="title" title="title">
Sample collection</div> <p id="Par32">Shoots from the control plants and plants treated with three <i>Paraburkholderia</i> species were harvested at 6 and 11 dpi. For each plant cultivar x <i>Paraburkholderia</i> combination, 4 biological replicates of 6 plants each were considered. Briefly, shoots were snap frozen in liquid nitrogen and ground to fine powder under continuous cooling through addition of liquid nitrogen and the samples were kept at -80 â"ƒ until further use.</p> </div> <div class="polarprimarymetaboliteextractionandanalysis" title="sec"> <div class="title" tagx="title" title="title">
Polar primary metabolite extraction and analysis</div> <p id="Par33">Polar primary metabolite sample preparation was performed as described by Carreno-Quintero et al.<sup><a href="#CR49">49</a></sup>. A total of 1.4Â mL of methanol containing ribitol (0.2Â mg/mL) as an internal standard was added to a 2Â mL Eppendorf tube containing a total of 200Â mg Broccoli leaf powder. After vortexing (10Â s) and shaking in a thermomixer at 950Â rpm for 10Â min, the samples were centrifuged at maximum speed for 10Â min. 500 ÂµL of the supernatant was transferred to a new 2Â mL Eppendorf tube and 370Â Âµl of chloroform and 750Â Âµl of distilled water were added. The mixture was vigorously mixed by vortexing and centrifugation for 10Â min at maximum speed (14,000Â rpm) and 50Â Âµl of the upper polar phase was transferred to an insert in a 2Â mL vial. The solvent was then vacuum dried (speedvac) for 16Â h at room temperature and sealed under an argon atmosphere. The dried samples were derivatized online as described by Lisec et al.<sup><a href="#CR50">50</a></sup> using a Combi PAL autosampler (CTC Analytics). Initially, 12.5Â Âµl methoxyamine (20Â mgÂ mL<sup>-1</sup> pyridine) was added to each of the samples and incubated for 30Â min at 40Â Â°C under agitation. The samples were then derivatized for one hour by adding 17.5Â Âµl of <i>N</i>-methyl-<i>N</i>-(trimethylsilyl) trifluoroacetamide (MSTFA). An alkane mixture (C11-C21 and C24-C33) was added to determine the retention indices of metabolites. The derivatized samples were analyzed by a GC-TOFâ€"MS system consisting of an Optic 3 high-performance injector (ATAS) and an Agilent 6890 gas chromatograph (Agilent Technologies) coupled to a Pegasus III time-of-flight mass spectrometer (Leco Instruments). A 2 ÂµL of each sample was subjected to the injector at 70Â Â°C using a split flow of 19Â mLÂ min<sup>-1</sup>. The chromatographic separation was performed using a VF-5Â ms capillary column (Varian; 30Â mâ€‰Ã—â€‰0.25Â mmâ€‰Ã—â€‰0.25Â mm) including a 10-m guardian column with helium as carrier gas at a flow rate of 1Â mLÂ min<sup>-1</sup>. The temperature was isothermal for 2Â min at 70 â"ƒ, followed by a 10 â"ƒ min<sup>-1</sup> ramp to 310 â"ƒ, and was held for 5Â min. The transfer line temperature was set at 270 â"ƒ. The column effluent was ionized by electron impact at 70Â eV. Mass spectra were acquired at 20 scans s<sup>-1</sup> within a mass-to-charge ratio range of 50 to 600 at a source temperature of 200 â"ƒ. A solvent delay of 295Â s was set. The detector voltage was set to 1,400Â V.</p> </div> <div class="semi-polarsecondarymetaboliteextractionandanalysis" title="sec"> <div class="title" tagx="title" title="title">
Semi-polar secondary metabolite extraction and analysis</div> <p id="Par34">For extraction of semi-polar secondary metabolites, 300 ÂµL of 99.89% methanol containing 0.13% (v/v) formic acid was added to 100Â mg plant material in 2Â ml round bottom Eppendorf tubes, and sonicated for 15Â min followed by centrifugation for 15Â min at 20,000Â g. The supernatants containing predominantly the semi-polar metabolites were transferred to 96-well filter plate (AcroPrepTM, 350 ÂµL, 0.45Â Âµm, PALL), and vacuum filtrated into the 96-deep-well autosampler plates (Waters) using a Genesis Workstation (Tecan Systems).</p> <p id="Par35">LCMS profiling of these semi-polar extract was performed using an Ultimate 3000 U-HPLC (Dionex) coupled to a Q-Exactive Plus Orbitrap FTMS (Thermo), as described recently<sup><a href="#CR51">51</a></sup>. A Luna C18 column (2.0 X 150Â mm, 3Â Âµm; Phenomenex) maintained at 40Â Â°C and a 45Â min linear gradient of 5â€"35% acetonitrile in 0.1% formic acid in water at a flow rate of 0.19Â mlÂ min<sup>-1</sup> were used to separate the compounds present in 5Â Âµl of each extract<sup><a href="#CR52">52</a></sup>. Full scan MS data were generated with electrospray in switching positive/negative ionization mode at a mass resolution of 35,000 (FWHM at m/z 200) in the range of <i>m/z</i> 95â€"1350. Subsequent MS/MS experiments for identification of selected metabolites were performed with separate positive or negative electrospray ionization at a normalized collision energy of 27 and a mass resolution of 17,500. Some primary metabolites were also detected in the semi polar fraction of the shoot extract eluting at the early stage of the chromatographic separation and are listed among the secondary metabolites in the Fig.Â <a href="#Fig4">4</a>b1,b2, supplementary Table <a href="#MOESM1">S7</a> and <a href="#MOESM1">S8</a>.</p> </div> </div> <div class="dataprocessingandanalysis" title="sec"> <div class="title" tagx="title" title="title">
Data processing and analysis</div> <div class="preprocessing" title="sec"> <div class="title" tagx="title" title="title">
Preprocessing</div> <p id="Par36">The raw data from GC-TOFâ€"MS analysis was preprocessed by Chroma TOF software 2.0 (Leco Instruments). Baseline-corrected mass features of the raw GC-TOFâ€"MS and LCMS data were extracted and aligned using Metalign software<sup><a href="#CR53">53</a></sup>. The mass features were considered as a reproducible signal if they were detected in at least 3 biological replicates of any treatment with a signal intensity at least 3 times higher than the background noise value calculated by Metalign. Then, mass features originating from the same metabolite, as generated in the ion source, were clustered based on similarities in both retention time and relative abundance across all measured samples, using MSClust software<sup><a href="#CR54">54</a></sup>. This software removes metabolite signal redundancy and generates so-called centrotypes, representing reconstructed putative metabolites including their in-source mass spectra. The relative abundance of each compound in a given sample is represented by the Measured Ion Counts (MIC), which is the sum of the ion count values (corrected by their centrotype membership) for all clustered ions. The samples were batch corrected to reduce batch effect of large series of samples during the run according to<sup><a href="#CR55">55</a></sup>. The relative intensity of the detected metabolites by GC-TOFâ€"MS was normalized to the internal standard, ribitol.</p> </div> </div> <div class="multivariateanalysis" title="sec"> <div class="title" tagx="title" title="title">
Multivariate analysis</div> <p id="Par37">The preprocessed data from both analysis platforms were log-transformed and scaled and used for ANOVA. Metabolites that were significantly different between at least two treatments with a fold changesâ€‰&amp;gt;â€‰2.0 were used to perform Principal component analysis (PCA) and hierarchical cluster analysis (HCA). The HCA was performed using Pearsonâ€™s correlation coefficient and Unweighted Pair Group Method with Arithmetic Mean (UPGMA).</p> </div> <div class="metaboliteidentification" title="sec"> <div class="title" tagx="title" title="title">
Metabolite identification</div> <div class="gc-tofâ€" title="sec"> <div class="title" tagx="title" title="title">
GC-TOFâ€"MS</div> <p id="Par38">To automatically identify metabolites, the reconstructed mass spectra file resulting from the MSClust software was introduced to NIST MS search software (v 2.2) with both the Wiley spectral library and an in-house library constructed using standards, followed by comparison of the reported and observed retention indices as determined by a series of alkanes. Metabolite annotations of selected compounds were manually curated.</p> </div> <div class="lcâ€" title="sec"> <div class="title" tagx="title" title="title">
LCâ€"MS</div> <p id="Par39">Annotation of differentially regulated metabolites was performed based on selection of pseudomolecule ions from the masses in the MSClust-reconstructed metabolites, first by matching their accurate masses plus retention times to previously reported metabolites present in Arabidopsis and Broccoli on the same LCâ€"MS system and similar chromatographic conditions<sup><a href="#CR8">8</a>,<a href="#CR56">56</a></sup>. If compounds were not yet present in this experimentally obtained database, detected masses were matched with compound libraries, including Metabolomics Japan (<a href="http://www.metabolomics.jp">www.metabolomics.jp</a>), the Dictionary of Natural Products (<a href="http://dnp.chemnetbase.com">http://dnp.chemnetbase.com</a>), KNApSAcK (<a href="http://kanaya.naist.jp/KNApSAcK">http://kanaya.naist.jp/KNApSAcK</a>), and Metlin (http:// metlin.scripps.edu/) using a maximum deviation of observed mass from calculated mass of 5Â ppm. The identity of potential candidate metabolites was further verified from the MSMS data using the online Magma tool<sup><a href="#CR57">57</a></sup> that compares the Insilico fragmentation patterns of a given metabolite to the experimentally obtained MSMS fragmentation pattern.</p> </div> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par40">The relative changes in shoot biomass, root biomass in the combinations of the two Broccoli cultivars and <i>Paraburkholderia</i> species was analyzed with R Studio software (Version 3.6.1). First, the normality and homogeneity of variance of the data was assessed and when the two assumptions were not met, the data was transformed using Box-Cox or log transformation using a package MASS. Differences were tested by two-way analysis of variance (ANOVA). A Tukey-HSD test was used to separate group mean values when the ANOVA was significant atÂ <i>p</i>â€‰<div class="â€‰0.05._UNKNOWN" href="#MOESM3">
S1 Differences in phenotypic parameters between the rhizobacterial treatments and non-treated controls were assessed by Studentâ€™s <i>t</i>-Test.</div></p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2021_82238_MOESM1_ESM.xlsx">LINK</a> <p>Supplementary Information 1.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2021_82238_MOESM2_ESM.docx">LINK</a> <p>Supplementary Information 2.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2021_82238_MOESM3_ESM.docx">LINK</a> <p>Supplementary Information 3.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>The online version contains supplementary material available at 10.1038/s41598-021-82238-9.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Seeds of two Broccoli cultivars (<i>Brassica oleracea</i> var. italic), Coronado and Malibu, were kindly provided by Bejo seed company (Warmenhuizen, The Netherlands). We are grateful to Bert Schipper for his assistance with the LC MS analysis.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>J.M.R., R.C.H.D.V. and D.W.E. conceived the project; D.W.E. and J.M.R. supervised the experiments; J.J. performed the experiments; H.D.L.M.V.E., N.C.Q., J.J., R.D.V. and D.W.E. analyzed the data; J.J., D.W.E. and J.M.R. wrote the manuscript with contributions of all the authors. D.W.E. agrees to serve as the author responsible for contact and ensures communication.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>Part of the project was supported by Korean government scholarship program of the National Institute for International Education (NIIED).</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par41">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butler</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bottomley</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myrold</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Microbial community dynamics associated with rhizosphere carbon flow</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">6793</span><span class="lpage" tagx="lpage" title="lpage">6800</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/aem.69.11.6793-6800.2003">10.1128/aem.69.11.6793-6800.2003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14602642">14602642</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okubo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsusaka</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugiyama</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Impacts of root symbiotic associations on interspecific variation in sugar exudation rates and rhizosphere microbial communities: a comparison among four plant families</span><span class="source" tagx="source" title="source">Plant Soil</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">399</span><span class="fpage" tagx="fpage" title="fpage">345</span><span class="lpage" tagx="lpage" title="lpage">356</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11104-015-2703-2">10.1007/s11104-015-2703-2</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dilkes</span><span class="given-names" tagx="given-names" title="given-names">NB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farrar</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Temporal dynamics of carbon partitioning and rhizodeposition in wheat</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">134</span><span class="fpage" tagx="fpage" title="fpage">706</span><span class="lpage" tagx="lpage" title="lpage">715</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.103.032045">10.1104/pp.103.032045</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14764904">14764904</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warembourg</span><span class="given-names" tagx="given-names" title="given-names">FR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roumet</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lafont</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Differences in rhizosphere carbon-partitioning among plant species of different families</span><span class="source" tagx="source" title="source">Plant Soil</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">256</span><span class="fpage" tagx="fpage" title="fpage">347</span><span class="lpage" tagx="lpage" title="lpage">357</span><span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1026147622800">10.1023/A:1026147622800</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">ZA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Allocation of photosynthesized carbon in an intensively farmed winter wheat-soil system as revealed by (CO2)-C-14 pulse labelling</span><span class="source" tagx="source" title="source">Sci. Rep. UK</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-21547-y">10.1038/s41598-018-21547-y</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helmisaari</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makkonen</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kellomaki</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valtonen</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malkonen</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Below- and above-ground biomass, production and nitrogen use in Scots pine stands in eastern Finland</span><span class="source" tagx="source" title="source">For. Ecol. Manag.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">165</span><span class="fpage" tagx="fpage" title="fpage">317</span><span class="lpage" tagx="lpage" title="lpage">326</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0378-1127(01)00648-X">10.1016/S0378-1127(01)00648-X</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haichar</span><span class="given-names" tagx="given-names" title="given-names">FE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santaella</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heulin</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Achouak</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Root exudates mediated interactions belowground</span><span class="source" tagx="source" title="source">Soil Biol. Biochem.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">69</span><span class="lpage" tagx="lpage" title="lpage">80</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2014.06.017">10.1016/j.soilbio.2014.06.017</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van de Mortel</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic and transcriptomic changes induced in Arabidopsis by the rhizobacterium Pseudomonas fluorescens SS101</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">160</span><span class="fpage" tagx="fpage" title="fpage">2173</span><span class="lpage" tagx="lpage" title="lpage">2188</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.112.207324">10.1104/pp.112.207324</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23073694">23073694</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis of bacterial determinants of plant growth promotion and induced systemic resistance by Pseudomonas fluorescens</span><span class="source" tagx="source" title="source">Environ. Microbiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">4638</span><span class="lpage" tagx="lpage" title="lpage">4656</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1462-2920.13927">10.1111/1462-2920.13927</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28892231">28892231</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rattan</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mechanism of action of insecticidal secondary metabolites of plant origin</span><span class="source" tagx="source" title="source">Crop Prot.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">913</span><span class="lpage" tagx="lpage" title="lpage">920</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cropro.2010.05.008">10.1016/j.cropro.2010.05.008</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourgaud</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gravot</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milesi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gontier</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Production of plant secondary metabolites: a historical perspective</span><span class="source" tagx="source" title="source">Plant Sci.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">161</span><span class="fpage" tagx="fpage" title="fpage">839</span><span class="lpage" tagx="lpage" title="lpage">851</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0168-9452(01)00490-3">10.1016/S0168-9452(01)00490-3</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boulogne</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petit</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ozier-Lafontaine</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desfontaines</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loranger-Merciris</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Insecticidal and antifungal chemicals produced by plants: a review</span><span class="source" tagx="source" title="source">Environ. Chem. Lett.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">325</span><span class="lpage" tagx="lpage" title="lpage">347</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10311-012-0359-1">10.1007/s10311-012-0359-1</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennett</span><span class="given-names" tagx="given-names" title="given-names">RN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallsgrove</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Secondary metabolites in plant defence mechanisms</span><span class="source" tagx="source" title="source">New Phytol.</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">127</span><span class="fpage" tagx="fpage" title="fpage">617</span><span class="lpage" tagx="lpage" title="lpage">633</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1469-8137.1994.tb02968.x">10.1111/j.1469-8137.1994.tb02968.x</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Host plant secondary metabolite profiling shows a complex, strain-dependent response of maize to plant growth-promoting rhizobacteria of the genus Azospirillum</span><span class="source" tagx="source" title="source">New Phytol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">189</span><span class="fpage" tagx="fpage" title="fpage">494</span><span class="lpage" tagx="lpage" title="lpage">506</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1469-8137.2010.03484.x">10.1111/j.1469-8137.2010.03484.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20946131">20946131</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valette</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rey</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerin</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Comte</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wisniewski-DyÃ©</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">A common metabolomic signature is observed upon inoculation of rice roots with various rhizobacteria</span><span class="source" tagx="source" title="source">J. Integr. Plant Biol.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">62</span><span class="fpage" tagx="fpage" title="fpage">228</span><span class="lpage" tagx="lpage" title="lpage">246</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/jipb.12810">10.1111/jipb.12810</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30920733">30920733</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stringlis</span><span class="given-names" tagx="given-names" title="given-names">IA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">MYB72-dependent coumarin exudation shapes root microbiome assembly to promote plant health</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">E5213</span><span class="lpage" tagx="lpage" title="lpage">E5222</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1722335115">10.1073/pnas.1722335115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29686086">29686086</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mhlongo</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piater</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steenkamp</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Labuschagne</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dubery</span><span class="given-names" tagx="given-names" title="given-names">IA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic profiling of PGPR-treated tomato plants reveal priming-related adaptations of secondary metabolites and aromatic amino acids</span><span class="source" tagx="source" title="source">Metabolites</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/metabo10050210">10.3390/metabo10050210</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/33207638">33207638</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Naguib, A. E.-M. M. <i>et al.</i> Enhancement of phenolics, flavonoids and glucosinolates of Broccoli (Brassica olaracea, var. Italica) as antioxidants in response to organic and bio-organic fertilizers. <i>J. Saudi Soc. Agric. Sci.</i><b>11</b>, 135â€"142 (2012).</span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sawana, A., Adeolu, M. &amp;amp; Gupta, R. S. Molecular signatures and phylogenomic analysis of the genus Burkholderia: proposal for division of this genus into the emended genus Burkholderia containing pathogenic organisms and a new genus Paraburkholderia gen. nov. harboring environmental species. <i>Front. Genet.</i><b>5</b>, 429 (2014).</span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Plant probiotic bacteria Bacillus and Paraburkholderia improve growth, yield and content of antioxidants in strawberry fruit</span><span class="source" tagx="source" title="source">Sci. Rep. UK</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">2504</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-20235-1">10.1038/s41598-018-20235-1</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitter</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A new approach to modify plant microbiomes and traits by introducing beneficial bacteria at flowering into progeny seeds</span><span class="source" tagx="source" title="source">Front. Microbiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28167932">28167932</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ledger</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Volatile-mediated effects predominate in Paraburkholderia phytofirmans growth promotion and salt stress tolerance of Arabidopsis thaliana</span><span class="source" tagx="source" title="source">Front. Microbiol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">1838</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2016.01838">10.3389/fmicb.2016.01838</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27909432">27909432</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Timmermann</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Paraburkholderia phytofirmans PsJN protects Arabidopsis thaliana against a virulent strain of Pseudomonas syringae through the activation of induced resistance</span><span class="source" tagx="source" title="source">Mol. Plant Microbe Interact.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">215</span><span class="lpage" tagx="lpage" title="lpage">230</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/MPMI-09-16-0192-R">10.1094/MPMI-09-16-0192-R</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28118091">28118091</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">CarriÃ³n</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Involvement of Burkholderiaceae and sulfurous volatiles in disease-suppressive soils</span><span class="source" tagx="source" title="source">ISME J</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">2307</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41396-018-0186-x">10.1038/s41396-018-0186-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29899517">29899517</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nazir</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warmink</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boersma</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Elsas</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mechanisms that promote bacterial fitness in fungal-affected soil microhabitats</span><span class="source" tagx="source" title="source">FEMS Microbiol. Ecol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="lpage" tagx="lpage" title="lpage">185</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1574-6941.2009.00807.x">10.1111/j.1574-6941.2009.00807.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20002182">20002182</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huot</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montgomery</span><span class="given-names" tagx="given-names" title="given-names">BL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span></span><span class="mixed-article-title" title="mixed-article-title">Growth-defense tradeoffs in plants: a balancing act to optimize fitness</span><span class="source" tagx="source" title="source">Mol. Plant</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">1267</span><span class="lpage" tagx="lpage" title="lpage">1287</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/mp/ssu049">10.1093/mp/ssu049</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24777989">24777989</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinez-Medina</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Recognizing plant defense priming</span><span class="source" tagx="source" title="source">Trends Plant Sci.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">818</span><span class="lpage" tagx="lpage" title="lpage">822</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tplants.2016.07.009">10.1016/j.tplants.2016.07.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27507609">27507609</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Hulten</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pelser</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Loon</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pieterse</span><span class="given-names" tagx="given-names" title="given-names">CMJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ton</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Costs and benefits of priming for defense in <i>Arabidopsis</i></span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">5602</span><span class="lpage" tagx="lpage" title="lpage">5607</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0510213103">10.1073/pnas.0510213103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16565218">16565218</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrmann</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weaver</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span></span><span class="mixed-article-title" title="mixed-article-title">The shikimate pathway</span><span class="source" tagx="source" title="source">Annu. Rev. Plant Phys.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">473</span><span class="lpage" tagx="lpage" title="lpage">503</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.arplant.50.1.473">10.1146/annurev.arplant.50.1.473</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Payyavula</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic profiling reveals altered sugar and secondary metabolism in response to UGPase overexpression in Populus</span><span class="source" tagx="source" title="source">BMC Plant Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12870-014-0265-8">10.1186/s12870-014-0265-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25287590">25287590</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lastdrager</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanson</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smeekens</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sugar signals and the control of plant growth and development</span><span class="source" tagx="source" title="source">J. Exp. Bot.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">799</span><span class="lpage" tagx="lpage" title="lpage">807</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/ert474">10.1093/jxb/ert474</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24453229">24453229</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ordal</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villani</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosendahl</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemotaxis towards sugars by bacillus-subtilis</span><span class="source" tagx="source" title="source">J. Gen. Microbiol.</span><span class="year" tagx="year" title="year">1979</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">167</span><span class="lpage" tagx="lpage" title="lpage">172</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/00221287-115-1-167">10.1099/00221287-115-1-167</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/119030">119030</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adler</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hazelbauer</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahl</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemotaxis toward sugars in <i>Escherichia coli</i></span><span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">1973</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">824</span><span class="lpage" tagx="lpage" title="lpage">847</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.115.3.824-847.1973">10.1128/JB.115.3.824-847.1973</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4580570">4580570</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Feng, H. C. <i>et al.</i> Identification of chemotaxis compounds in root exudates and their sensing chemoreceptors in plant-growth-promoting rhizobacteria bacillus amyloliquefaciens SQR9. <i>Mol. Plant Microbe Interact.</i><b>31</b>, 995â€"1005 .10.1094/Mpmi-01-18-0003-R (2018).</span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rolland</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baena-Gonzalez</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sugar sensing and signaling in plants: conserved and novel mechanisms</span><span class="source" tagx="source" title="source">Annu. Rev. Plant Biol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">675</span><span class="lpage" tagx="lpage" title="lpage">709</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.arplant.57.032905.105441">10.1146/annurev.arplant.57.032905.105441</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16669778">16669778</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernie</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrari</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sweetlove</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Respiratory metabolism: glycolysis, the TCA cycle and mitochondrial electron transport</span><span class="source" tagx="source" title="source">Curr. Opin. Plant Biol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">254</span><span class="lpage" tagx="lpage" title="lpage">261</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pbi.2004.03.007">10.1016/j.pbi.2004.03.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15134745">15134745</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dose-response relationships and inundative biological-control</span><span class="source" tagx="source" title="source">Phytopathology</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">84</span><span class="fpage" tagx="fpage" title="fpage">780</span><span class="lpage" tagx="lpage" title="lpage">784</span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raaijmakers</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Dose-response relationships in biological-control of Fusarium-Wilt of radish by <i>Pseudomonas</i> Spp</span><span class="source" tagx="source" title="source">Phytopathology</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">1075</span><span class="lpage" tagx="lpage" title="lpage">1081</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/Phyto-85-1075">10.1094/Phyto-85-1075</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lloyd</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zakhleniuk</span><span class="given-names" tagx="given-names" title="given-names">OV</span></span></span><span class="mixed-article-title" title="mixed-article-title">Responses of primary and secondary metabolism to sugar accumulation revealed by microarray expression analysis of the Arabidopsis mutant, pho3</span><span class="source" tagx="source" title="source">J. Exp. Bot.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">1221</span><span class="lpage" tagx="lpage" title="lpage">1230</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/erh143">10.1093/jxb/erh143</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15133053">15133053</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilal</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epidermal lignin deposition in quinoa cotyledons in response to UV-B radiation</span><span class="source" tagx="source" title="source">Photochem. Photobiol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">79</span><span class="fpage" tagx="fpage" title="fpage">205</span><span class="lpage" tagx="lpage" title="lpage">210</span><span class="pub-id"><a href="https://dx.doi.org/10.1562/0031-8655(2004)079&lt;0205:Eldiqc&gt;2.0.Co;2">10.1562/0031-8655(2004)079&amp;lt;0205:Eldiqc&amp;gt;2.0.Co;2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15068034">15068034</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dixon</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The phenylpropanoid pathway and plant defenceâ€"a genomics perspective</span><span class="source" tagx="source" title="source">Mol. Plant. Pathol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">371</span><span class="lpage" tagx="lpage" title="lpage">390</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1364-3703.2002.00131.x">10.1046/j.1364-3703.2002.00131.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20569344">20569344</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cvikrova</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mala</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hrubcova</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eder</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Soluble and cell wall-bound phenolics and lignin in Ascocalyx abietina infected Norway spruces</span><span class="source" tagx="source" title="source">Plant Sci.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">170</span><span class="fpage" tagx="fpage" title="fpage">563</span><span class="lpage" tagx="lpage" title="lpage">570</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.plantsci.2005.10.011">10.1016/j.plantsci.2005.10.011</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Islam, M. T. <i>et al.</i> Characterization of p-Coumaric acid-induced soluble and cell wall-bound phenolic metabolites in relation to disease resistance to Xanthomonas campestris pv. campestris in Chinese cabbage. <i>Plant Physiol. Biochem.</i><b>125</b>, 172â€"177 .10.1016/j.plaphy.2018.02.012 (2018).</span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heil</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bostock</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Induced systemic resistance (ISR) against pathogens in the context of induced plant defences</span><span class="source" tagx="source" title="source">Ann. Bot. Lond.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">503</span><span class="lpage" tagx="lpage" title="lpage">512</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/aob/mcf076">10.1093/aob/mcf076</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pieterse</span><span class="given-names" tagx="given-names" title="given-names">CMJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Induced systemic resistance by beneficial microbes</span><span class="source" tagx="source" title="source">Annu. Rev. Phytopathol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">52</span><span class="fpage" tagx="fpage" title="fpage">347</span><span class="lpage" tagx="lpage" title="lpage">375</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-phyto-082712-102340">10.1146/annurev-phyto-082712-102340</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24906124">24906124</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Dam</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldwin</span><span class="given-names" tagx="given-names" title="given-names">IT</span></span></span><span class="mixed-article-title" title="mixed-article-title">Costs of jasmonate-induced responses in plants competing for limited resources</span><span class="source" tagx="source" title="source">Ecol. Lett.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">30</span><span class="lpage" tagx="lpage" title="lpage">33</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1461-0248.1998.00010.x">10.1046/j.1461-0248.1998.00010.x</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bergelson</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purrington</span><span class="given-names" tagx="given-names" title="given-names">CB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Surveying patterns in the cost of resistance in plants</span><span class="source" tagx="source" title="source">Am. Nat.</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">148</span><span class="fpage" tagx="fpage" title="fpage">536</span><span class="lpage" tagx="lpage" title="lpage">558</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/285938">10.1086/285938</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vieira</span><span class="given-names" tagx="given-names" title="given-names">RF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paula</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carneiro</span><span class="given-names" tagx="given-names" title="given-names">JES</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teixeira</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Queiroz</span><span class="given-names" tagx="given-names" title="given-names">TFN</span></span></span><span class="mixed-article-title" title="mixed-article-title">Management of white mold in type III common bean with plant spacing and fungicide</span><span class="source" tagx="source" title="source">Trop. Plant Pathol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">95</span><span class="lpage" tagx="lpage" title="lpage">101</span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carreno-Quintero</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Untargeted metabolic quantitative trait loci analyses reveal a relationship between primary metabolism and potato tuber quality</span><span class="source" tagx="source" title="source">Plant Physiol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">158</span><span class="fpage" tagx="fpage" title="fpage">1306</span><span class="lpage" tagx="lpage" title="lpage">1318</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.111.188441">10.1104/pp.111.188441</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22223596">22223596</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lisec</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schauer</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kopka</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Willmitzer</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernie</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gas chromatography mass spectrometryâ€"based metabolite profiling in plants</span><span class="source" tagx="source" title="source">Nat. Protoc.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">387</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2006.59">10.1038/nprot.2006.59</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17406261">17406261</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pyrethric acid of natural pyrethrin insecticide: complete pathway elucidation and reconstitution in Nicotiana benthamiana</span><span class="source" tagx="source" title="source">New Phytol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">223</span><span class="fpage" tagx="fpage" title="fpage">751</span><span class="lpage" tagx="lpage" title="lpage">765</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/nph.15821">10.1111/nph.15821</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30920667">30920667</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Vos</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Untargeted large-scale plant metabolomics using liquid chromatography coupled to mass spectrometry</span><span class="source" tagx="source" title="source">Nat. Protoc.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">778</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2007.95">10.1038/nprot.2007.95</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17446877">17446877</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lommen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">MetAlign: interface-driven, versatile metabolomics tool for hyphenated full-scan mass spectrometry data preprocessing</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">3079</span><span class="lpage" tagx="lpage" title="lpage">3086</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ac900036d">10.1021/ac900036d</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19301908">19301908</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tikunov</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laptenok</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bovy</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Vos</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">MSClust: a tool for unsupervised mass spectra extraction of chromatography-mass spectrometry ion-wise aligned data</span><span class="source" tagx="source" title="source">Metabolomics</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">714</span><span class="lpage" tagx="lpage" title="lpage">718</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11306-011-0368-2">10.1007/s11306-011-0368-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22833709">22833709</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wehrens</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Improved batch correction in untargeted MS-based metabolomics</span><span class="source" tagx="source" title="source">Metabolomics</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">88</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11306-016-1015-8">10.1007/s11306-016-1015-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27073351">27073351</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beekwilder</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The impact of the absence of aliphatic glucosinolates on insect herbivory in Arabidopsis</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">e2068</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0002068">10.1371/journal.pone.0002068</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18446225">18446225</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ridder</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Automatic chemical structure annotation of an LCâ€"MS n based metabolic profile from green tea</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">6033</span><span class="lpage" tagx="lpage" title="lpage">6040</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ac400861a">10.1021/ac400861a</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23662787">23662787</a></span></span></li> </ul> </div> </div>  </body></html>