<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PeerJ</span><span class="iso-abbrev" title="iso-abbrev">PeerJ</span><span class="publisher-id" title="publisher-id">peerj</span><span class="pmc" title="pmc">peerj</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PeerJ</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2167-8359</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">PeerJ Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Diego, USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7751431">7751431</a></span><span class="publisher-id" title="publisher-id">10512</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7717/peerj.10512">10.7717/peerj.10512</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Agricultural Science</span>: <span class="subject" title="subject">Microbiology</span>: <span class="subject" title="subject">Taxonomy</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Antimicrobial activity of <i>Streptomyces</i> spp. isolated from <i>Apis dorsata</i> combs against some phytopathogenic bacteria</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Promnuan</span><span class="given-names" tagx="given-names" title="given-names">Yaowanoot</span></span><span class="email" tagx="email" title="email">faasynp@ku.ac.th</span><a href="#aff-1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Promsai</span><span class="given-names" tagx="given-names" title="given-names">Saran</span></span><a href="#aff-1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meelai</span><span class="given-names" tagx="given-names" title="given-names">Sujinan</span></span><a href="#aff-2">2</a></span><span class="citation_author_institution" id="aff-1">[1], <span class="institution" title="institution">Department of Microbiology, Faculty of Liberal Arts and Science, Kasetsart University-Kamphaeng Saen campus</span><span class="city" tagx="city" title="city">Kamphaeng Saen</span><span class="state" tagx="state" title="state">Nakhon Pathom</span><span class="country" tagx="country" title="country">Thailand</span></span><span class="citation_author_institution" id="aff-2">[2], <span class="institution" title="institution">Department of Microbiology, Faculty of Science, Silpakorn University-Sanam Chandra Palace campus</span><span class="city" tagx="city" title="city"> Nakhon Pathom</span><span class="state" tagx="state" title="state">Nakhon Pathom</span><span class="country" tagx="country" title="country">Thailand</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gillespie</span><span class="given-names" tagx="given-names" title="given-names">Joseph</span></span></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-12-12</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e10512</span><span class="history" title="history"><span class="received" title="received">received: 2020-4-30</span><span class="accepted" title="accepted">accepted: 2020-11-16</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Promnuan et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, reproduction and adaptation in any medium and for any purpose provided that it is properly attributed. For attribution, the original author(s), title, publication source (PeerJ) and either DOI or URL of the article must be cited.</span></span></div><a class="self-uri" href="https://peerj.com/articles/10512" title="self-uri">https://peerj.com/articles/10512</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The aim of this study was to investigate the antimicrobial potential of actinomycetes isolated from combs of the giant honey bee, <i>Apis dorsata</i>. In total, 25 isolates were obtained from three different media and were screened for antimicrobial activity against four plant pathogenic bacteria (<i>Ralstonia solanacearum</i>, <i>Xanthomonas campestris</i> pv. <i>campestris</i>, <i>Xanthomonas oryzae</i> pv. <i>oryzae</i> and<i> Pectobacterium carotovorum</i>). Following screening using a cross-streaking method, three isolates showed the potential to inhibit the growth of plant pathogenic bacteria. Based on a 96-well microtiter assay, the crude extract of DSC3-6 had minimum inhibitory concentration (MIC) values against <i>X. oryzae</i> pv. <i>oryzae</i>, <i>X. campestris</i> pv. <i>campestris</i>, <i>R. solanacearum</i> and<i> P. carotovorum</i> of 16, 32, 32 and 64 mg L<sup>âˆ’1</sup>, respectively. The crude extract of DGA3-20 had MIC values against <i>X. oryzae</i> pv. <i>oryzae</i>, <i>X. campestris</i> pv. <i>campestris</i>, <i>R. solanacearum</i> and<i> P. carotovorum</i> of 32, 32, 32 and 64 mg L<sup>âˆ’1</sup>, respectively. The crude extract of DGA8-3 at 32 mgL<sup>âˆ’1</sup> inhibited the growth of <i>X. oryzae</i> pv. <i>oryzae</i>, <i>X. campestris</i> pv. <i>campestris</i>, <i>R. solanacearum</i> and<i> P. carotovorum</i>. Based on their 16S rRNA gene sequences, all isolates were identified as members of the genus <i>Streptomyces</i>. The analysis of 16S rRNA gene sequence similarity and of the phylogenetic tree based on the maximum likelihood algorithm showed that isolates DSC3-6, DGA3-20 and DGA8-3 were closely related to <i>Streptomyces ramulosus</i> (99.42%), <i>Streptomyces axinellae</i> (99.70%) and <i>Streptomyces drozdowiczii</i> (99.71%), respectively. This was the first report on antibacterial activity against phytopathogenic bacteria from actinomycetes isolated from the giant honey bee.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd"><i>Streptomyces</i></div> <div class="kwd" title="kwd">
Bees</div> <div class="kwd" title="kwd">
Plant pathogens</div> <div class="kwd" title="kwd"><i>Ralstonia solanacearum</i></div> <div class="kwd" title="kwd"><i>Xanthomonas campestris</i> pv. <i>campestris</i></div> <div class="kwd" title="kwd"><i>Xanthomonas oryzae</i> pv. <i>oryzae</i></div> <div class="kwd" title="kwd"><i>Pectobacterium carotovorum</i></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Department of Microbiology, Faculty of Liberal Arts and Science, Kasetsart University of the year 2019, by the Research Promotion and Technology Transfer Center (RPTTC) of the Faculty of Liberal Arts and Science, Kasetsart University Kamphaeng Sean campus, Thailand and by Grant SRIF-JRG-2562-04 from the Faculty of Science, Silpakorn University, Nakhon Pathom, Thailand</div> </div> <div class="funding-statement" title="funding-statement">
This research was supported by the Department of Microbiology, Faculty of Liberal Arts and Science, Kasetsart University of the year 2019, by the Research Promotion and Technology Transfer Center (RPTTC) of the Faculty of Liberal Arts and Science, Kasetsart University Kamphaeng Sean campus, Thailand and by Grant SRIF-JRG-2562-04 from the Faculty of Science, Silpakorn University, Nakhon Pathom, Thailand. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The Gram-negative bacteria <i>Xanthomonas campestris</i> pv. <i>campestris</i>, <i>Xanthomonas oryzae</i> pv. <i>oryzae</i>, <i>Ralstonia solanacearum</i> and <i>Pectobacterium carotovorum</i> are known to cause significant losses in many crop plants worldwide. <i>X.Â campestris</i> pv. <i>campestris</i> is a seed-borne pathogen that causes black rot disease in a large number of species of the Brassicaceae, including the genera <i>Brassica</i> and <i>Arabidopsis</i>. The typical disease symptoms include V-shaped yellow lesions starting from the leaf margins and blackening of the veins (<a href="#ref-34">Vicente &amp;amp; Holub, 2013</a>). <i>X.Â oryzae</i> pv. <i>oryzae</i> causes devastating bacterial bright leaf (BLB) disease, which is one of the major diseases of rice in Asian countries. This bacterial pathogen grows in the xylem vessel, causing yellow/white lesions along the leaf veins (<a href="#ref-40">Xie et al., 2018</a>). <i>R.Â solanacearum</i> is the causal agent of bacterial wilt, which is one of the most devastating plant diseases worldwide. This soil-borne vascular pathogen can cause disease to many economically important crops, including tomato, potato, eggplant, tobacco and banana. The bacterium infects plants via wounds or root tips; it invades the xylem vessels and systematically spreads to the aerial parts of the plant through the vascular system (<a href="#ref-21">Ombiro et al., 2018</a>). <i>P.Â carotovorum</i>, formerly known as <i>Erwinia carotovora</i>, is one of the most destructive diseases of postharvest vegetables worldwide, especially potatoes, green peppers and Chinese cabbages (<a href="#ref-41">Zhao et al., 2013</a>). The bacterium is found on plant surfaces and in soil, where it may enter the plant via wound sites or through natural openings on the plant surface. Once inside the plant, it resides in the vascular tissue and intracellular spaces, where it remains until environmental conditions, including free water, oxygen availability and temperature, become suitable for disease development (<a href="#ref-8">Itoh et al., 2003</a>). Soft rot pathogens cause general tissue maceration, termed soft rot disease, through the production of enzymes that degrade plant cell walls.</p> <p>The management of plant diseases is difficult. The use of resistant cultivars to control bacterial wilt disease is the most economical, environmentally friendly and effective method (<a href="#ref-21">Ombiro et al., 2018</a>). The disease control of black rot relies on the use of pathogen-free seed and planting material and the elimination of other potential inoculum sources (<a href="#ref-34">Vicente &amp;amp; Holub, 2013</a>). Currently, the control of postharvest bacterial soft rot depends mainly upon the use of bactericides, such as hypochlorite, formaldehyde solution and antibiotics (<a href="#ref-41">Zhao et al., 2013</a>). However, the use of chemical bactericides and antibiotics to control phytopathogenic bacteria could cause serious damage to the environment and human health. Moreover, some emerging strains have shown strong resistance to all these products (<a href="#ref-29">Sabir et al., 2017</a>; <a href="#ref-18">Mougou &amp;amp; Boughalleb-Mâ€™hamdi, 2018</a>; <a href="#ref-38">Wu et al., 2019</a>). Therefore, many researchers have focused on the development of alternative methods of controlling plant diseases. The use of antibacterial compounds from plant extracts (<a href="#ref-31">Satish, Raveesha &amp;amp; Jandrdhana, 1999</a>; <a href="#ref-9">Kaur et al., 2016</a>), validamycin A (<a href="#ref-7">Ishikawa et al., 2004</a>), xantho-oligosaccharide (<a href="#ref-28">Qian et al., 2006</a>) and ralhibitins (<a href="#ref-21">Ombiro et al., 2018</a>) to inhibit the growth of phytopathogens has been studied and reported. There have been several studies of antagonistic microorganisms, such as <i>Bacillus</i> spp. and <i>Pseudomonas</i> spp., endophytic actinomycetes and melanogenic actinomycetes to inhibit the growth of bacteria causing black rot and bacterial bright leaf disease (<a href="#ref-39">Wulff et al., 2002</a>; <a href="#ref-17">Mishra &amp;amp; Arora, 2011</a>; <a href="#ref-41">Zhao et al., 2013</a>; <a href="#ref-19">Muangham, Pathom-aree &amp;amp; Duangmal, 2015</a>). Several <i>Streptomyces</i> species, such as <i>Streptomyces aureofaciens</i>, <i>Streptomyces avermitilis</i>, <i>Streptomyces humidus</i>, <i>Streptomyces hygroscopicus</i>, <i>Streptomyces lividans</i>, <i>Streptomyces lydicus</i>, <i>Streptomyces olivaceoviridis</i>, <i>Streptomyces plicatus</i>, <i>Streptomyces roseoflavus</i>, <i>Streptomyces scabies</i> and <i>Streptomyces violaceusniger</i>, have been used to control soil-borne diseases for their intense antagonistic activities by the production of various antimicrobial substances (<a href="#ref-42">Zheng et al., 2019</a>).</p> <p>Actinomycetes, especially <i>Streptomyces</i> species, are well known for producing bioactive compounds which suggests that actinobacteria have the potential to produce antimicrobial compounds against phytobacterial pathogens (<a href="#ref-33">Viaene et al., 2016</a>). Actinomycetes isolated from different habitats have been investigated for antimicrobial activities against plant pathogens; for example, melanogenic <i>Streptomyces</i> isolated from rhizospheric soils had the ability to inhibit the growth of rice pathogenic bacteria (<i>X.Â oryzae</i> pv. <i>oryzae</i> and <i>X.Â oryzae</i> pv. <i>oryzicola</i>). Among these, isolate TY68-3 had the highest antibacterial activity and siderophore production and had 99.6% 16S rDNA sequence similarity to <i>S.Â indiaensis</i> (<a href="#ref-19">Muangham, Pathom-aree &amp;amp; Duangmal, 2015</a>). In addition, <i>S.Â caeruleatus</i> isolated from the rhizosphere soil of <i>Cassia fistula</i> had the highest activity against the soybean pathogen <i>X.Â campestris</i> pv. <i>glycine</i> (<a href="#ref-16">Mingma et al., 2014</a>). <a href="#ref-6">Hastuti et al. (2012)</a> reported that endophytic <i>Streptomyces</i> reduced <i>X. oryzae</i> pv. <i>oryzae</i> infection in rice. Some isolates were able to improve the growth of rice seedlings, plant height and dry weight. In addition, <i>Streptomyces</i> strain LBR02 had the highest inhibitory activity (25Â mm diameter inhibition zone) against <i>X.Â oryzae</i> pv. <i>oryzae in vitro</i>. Furthermore, <i>S.Â violaceusnige</i> (strain A5) was isolated from chitin-rich partially decomposed molted snakeskin and had maximum inhibitory activity (0.625â€"1.25 mg mL<sup>âˆ’1</sup>) against <i>X.Â axonopodis</i> pv. <i>punicae</i>, the causative agent of oily spot disease in pomegranate (<a href="#ref-3">Chavan et al., 2016</a>). These reports indicated that actinomycetes, especially <i>Streptomyces</i>, may provide a new approach for the use of actinomycetes for biocontrol in agriculture.</p> <p>The giant honey bees, consisting of the species <i>Apis dorsata</i>, <i>Apis laboriosa</i> and <i>Apis breviligula</i>, are distributed over a vast geographic area in South and Southeast Asia. In Thailand, only <i>A.Â dorsata</i> is found. Bees of this species build a massive single comb attached under the surface of a stout tree branch or an overhang of a rock face, or sometimes to the eves of buildings or other urban structures (<a href="#ref-37">Wongsiri et al., 1996</a>). The actinomycetes associated with <i>A.Â dorsata</i> have never been studied or reported. However, there have been several studies of actinomycetes associated with bees and stingless bees in Thailand. The novel actinomycete species <i>Actinomadura apis</i> was isolated from the honey bee (<i>Apis mellifera</i>) (<a href="#ref-26">Promnuan, Kudo &amp;amp; Chantawannakul, 2011</a>). Two novel species of the genus <i>Streptomyces</i> (<i>Streptomyces chiangmaiensis</i> and <i>Streptomyces lannensis</i>) were isolated from stingless bee (<i>Tetragonilla collina</i>) collected from Chiang Mai Province, northern Thailand (<a href="#ref-27">Promnuan, Kudo &amp;amp; Chantawannakul, 2013</a>). Thirty-two actinobacteria isolates were obtained from honey bees (<i>A.Â mellifera</i>, <i>Apis cereana</i> and <i>Apis florea</i>). Most of the isolates belonged to the genus <i>Streptomyces</i>. Some less frequent isolates were classified in the genera <i>Nonomuraea</i>, <i>Nocardiopsis</i> and <i>Actinomadura</i>. Moreover, some of these isolates produced antimicrobial compounds that inhibited the growth of the honey bee pathogens <i>Paenibacillus larvae</i> and <i>Melisococcus plutonius</i>, which cause American and European foulbrood diseases in honey bees, respectively (<a href="#ref-25">Promnuan, Kudo &amp;amp; Chantawannakul, 2009</a>). These studies indicated that actinomycetes associated with bees have the potential to produce antimicrobial compounds to combat disease in agriculture. However, the rate of discovery of new antibiotics from actinomycetes from common habitats has slowed down; therefore, novel antibiotics must be found from actinomycetes in unexplored habitats (<a href="#ref-2">Berdy, 2005</a>).</p> <p>This study focused on actinomycetes isolated from <i>A.Â dorsata</i> combs and evaluated their antibacterial activity against plant pathogenic bacteria. According to data obtained in the current study, the antibacterial activity of metabolites from actinomycetes could be implemented against phytopathogenic bacteria to assist in crop protection.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials &amp;amp; Methods</div> <div class="samplecollection" title="sec"> <div class="title" tagx="title" title="title">
Sample collection</div> <p>Three combs of the giant honey bee (<i>A.Â dorsata</i>) were collected from the Mae-rim district, Chiang Mai Province, Thailand in April 2014. The three hive samples were collected from local villages in private areas. Verbal permission was acquired from each of the owners: 1. Mr. Ton Tatiya (2 hives) 25/2 Moo2, Pong Yaeng Sub-district, Mae-Rim district, Chiang Mai Province, Thailand. 2. Mr. Ma Madamun (1 hive) 93/3 Moo2, Pong Yaeng Sub-district, Mae-Rim district, Chiang Mai Province, Thailand. Adult bees, pollen and honey were collected and kept in sterile tubes and stored at âˆ’20Â Â°C until the isolation process.</p> </div> <div class="actinomyceteisolation" title="sec"> <div class="title" tagx="title" title="title">
Actinomycete isolation</div> <p>Three adult bees were surface-sterilized using a triple surface-sterilization technique and ground aseptically following the method modified from <a href="#ref-22">Photita et al. (2004)</a> and <a href="#ref-25">Promnuan, Kudo &amp;amp; Chantawannakul (2009)</a>. Isolation of actinomycetes from pollen and honey was obtained using a standard dilution plate method on starch casein nitrate agar (<a href="#ref-13">KÃ¼ster &amp;amp; Williams, 1964</a>), glycerol-asparagine (ISP5) (<a href="#ref-24">Pridham &amp;amp; Lyons, 1961</a>) and Czapekâ€™s agar (<a href="#ref-35">Waksman, 1950</a>) supplemented with 25Â Âµg mL<sup>âˆ’1</sup> nystatin and nalidixic acid. Plates were incubated at 30Â Â°C for 7â€"21 days and examined periodically. The actinobacterial colonies were isolated, purified and maintained in yeast extract-malt extract agar (ISP2) (<a href="#ref-32">Shirling &amp;amp; Gottlieb, 1966</a>) slants and stored at 4Â Â°C.</p> </div> <div class="testorganisms" title="sec"> <div class="title" tagx="title" title="title">
Test organisms</div> <p><i>X.Â campestris</i> pv. <i>campestris</i>, <i>X.Â oryzae</i> pv. <i>oryzae</i>, <i>R.Â solanacearum</i> and <i>P.Â carotovorum</i> were obtained from the Department of Agriculture, Ministry of Agriculture and Cooperative, Thailand. The phytobacterial pathogens were activated and maintained in nutrient agar (NA) and ISP2 agar for 24â€"48Â h at 30Â Â°C before use.</p> </div> <div class="screeningofantagonisticactinomycetes" title="sec"> <div class="title" tagx="title" title="title">
Screening of antagonistic actinomycetes</div> <p>In total, 25 actinomycete isolates were evaluated for their activity toward four plant pathogenic bacteria: <i>X. campestris</i> pv. <i>campestris, X. oryzae</i> pv. <i>oryzae, R. solanacearum</i> and <i>P.Â carotovorum</i> using a modified cross streak method (<a href="#ref-14">Lemos, Toranzo &amp;amp; Barja, 1985</a>). Each isolate was streaked on the center of ISP2 and a glucose yeast extract (GYE) (<a href="#ref-1">Agate &amp;amp; Bhat, 1963</a>) agar plate and incubated at 30Â Â°C for 7 days. Each test organism was streaked across the actinomycete line and incubated at 30Â Â°C for 24Â h. Then, the inhibition zones were observed and measured. The experiment was conducted in triplicate.</p> </div> <div class="extractionofbioactivecompounds" title="sec"> <div class="title" tagx="title" title="title">
Extraction of bioactive compounds</div> <p>The actinobacterial isolates that showed potent activity against the growth of the test organisms in the previous method were grown on ISP2 agar plates and incubated at 30Â Â°C for 14 days. The metabolites were extracted using a modified extraction method as described by <a href="#ref-12">Kumar et al. (2012)</a>. The culture medium of each isolate was cut into small pieces (approximately 0.5Â Ã—Â 0.5Â cm), extracted with 200 mL of ethyl acetate, shaken vigorously on a flask shaker at 150 rpm and 30Â Â°C for 48Â h and then filtered using filter paper (Whatman No.1). The ethyl acetate fractions were concentrated using a rotary vacuum evaporator at 40Â Â°C. The crude extracts were resuspended in 1 mL of sterile dimethyl sulfoxide (DMSO) and stored at âˆ’20Â Â°C until testing for antimicrobial activity.</p> </div> <div class="determinationoftheminimuminhibitoryconcentration(mic)ofselectedactinomycetes" title="sec"> <div class="title" tagx="title" title="title">
Determination of the minimum inhibitory concentration (MIC) of selected actinomycetes</div> <p>The MIC of crude extract was determined according to <a href="#ref-36">Wiegand, Hilpert &amp;amp; Hancock (2008)</a> with some modifications. The concentrations of the test organisms, <i>X.Â campestris</i> pv. <i>campestris, X. oryzae</i> pv. <i>oryzae, R. solanacearum</i> and <i>P.Â carotovorum</i>, were adjusted to the equivalent of 0.5 McFarland standard. The crude extract was dissolved in sterile DMSO to obtain an initial concentration of 5,120 mg L<sup>âˆ’1</sup>. Sterile DMSO was used as a negative control. The MIC of each extract was determined using serial two-fold dilutions in ISP2 broth in a 96-well microtiter plate (concentration range from 0.5 to 256 mg L<sup>âˆ’1</sup>). The experiment was performed in triplicate. Each microtiter plate was incubated at 30Â Â°C for 24Â h. After incubation, the suspension from each well was streaked onto separate ISP2 agar plates and incubated at 30Â Â°C for 24Â h, after which any growth was observed. The minimum concentrations of the extracts that showed no turbidity and had no bacterial growth on the ISP2 agar plate were recorded as the MIC values.</p> </div> <div class="identificationofactinomycetesusing16srdnasequencing" title="sec"> <div class="title" tagx="title" title="title">
Identification of actinomycetes using 16S rDNA sequencing</div> <p>The selected isolates that showed potent activity against the test organisms were grown in 50 mL of ISP2 broth and incubated in a shaker (120 revolutions min<sup>âˆ’1</sup>) at 30Â Â°C for 7 days. Then, cells were collected via centrifugation (91,000 g) for 5Â min and were washed three times using sterile distilled water. Genomic DNA was extracted, and the 16S rRNA gene was amplified using the methods described by <a href="#ref-20">Nakajima et al. (1999)</a>. The primers used for amplification were 20F (5â€²-AGTTTGATCCTGGCTC) and 1540R (5â€²-AAGGAGGTGATCCAGCC). Then, PCR products were purified using an Invitrogenâ"¢ PureLinkâ"¢ PCR Purification Kit (Thermo Fisher Scientific, USA) according to the manufacturerâ€™s instructions. Purified PCR products were sequenced by the Sanger method at 1st BASE, Singapore. The highest similarity of actinomycetes with the reference species was confirmed using the NCBI BLAST tool (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). The sequences of closely related type strains retrieved from the GenBank database were multiple aligned using Clustal_W in BioEdit Sequence Alignment Editor 7.2.5 (<a href="#ref-5">Hall, 1999</a>). After multiple alignments, a phylogenetic tree was constructed using the maximum likelihood (ML) method in MEGA X version 10.1.8 (<a href="#ref-11">Kumar etÂ al., 2018</a>) based on a comparison of 1,332â€"1,374 nucleotides present in all the strains used after elimination of gaps and ambiguous nucleotides from the sequences. <i>Streptomyces thermocarboxydus</i> DSM 44293<sup>T</sup> was used as an outgroup. Confidence values for branches of the phylogenetic tree were determined using bootstrap analyses based on 1,000 resamplings (<a href="#ref-4">Felsenstein, 1985</a>). The sequence similarity values were calculated from the pairwise alignments obtained using BioEdit 7.2.5 (<a href="#ref-5">Hall, 1999</a>).</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="isolationofactinomycetesfroma.Â dorsata" title="sec"> <div class="title" tagx="title" title="title">
Isolation of actinomycetes from <i>A.Â dorsata</i></div> <p>The samples (adult bees, pollen and honey) were collected from three hives of <i>A.Â dorsata</i>. Twenty-five morphologically different actinobacterial isolates were obtained from three different media, with 60% from ISP5 agar followed by starch casein nitrate agar and Czapekâ€™s agar. Most of the actinomycetes were isolated from pollen (84%), followed by honey (12%) and adult bees (4%) (<a href="#table-1">Table 1</a>).</p> </div> <div class="antimicrobialactivityagainstplantpathogens" title="sec"> <div class="title" tagx="title" title="title">
Antimicrobial activity against plant pathogens</div> <p>Based on screening for antimicrobial activity using the cross-streaking method, three actinomycete isolates (DSC3-6, DGA3-20 and DGA8-3) showed potent activity (&amp;gt;10 mm diameter inhibition zones) against the growth of four phytopathogenic bacteria (<i>X. campestris</i> pv<i>. campestris, X. oryzae</i> pv<i>. oryzae, R. solanacearum</i> and <i>P. carotovorum</i>) on ISP2 agar plates (<a href="#fig-1">Fig. 1</a>). The crude extracts of the three isolates were subsequently tested for their MIC levels against the growth of phytopathogenic bacteria. Based on the 96-well microtiter assay, the MIC values of the crude extract of the three actinobacterial strains are shown in <a href="#table-2">Table 2</a>. The MIC value of actinobacterial strain DSC3-6 against <i>X. oryzae</i> pv<i>. oryzae</i> was 16 mg L<sup>âˆ’1</sup>, and the MIC values of the crude extracts of DGA3-20 and DGA8-3 were both 32 mg L<sup>âˆ’1</sup>. All isolates inhibited the growth of <i>X. campestris</i> pv. <i>campestris</i>, <i>R. solanacearum</i> and <i>P. carotovorum</i>, with MIC values of 32, 32 and 64 mg L<sup>âˆ’1</sup>, respectively.</p> <div class="table-wrap_UNKNOWN" id="table-1" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7717/peerj.10512/table-1">10.7717/peerj.10512/table-1</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Numbers of actinomycetes isolated from <i>A. dorsata</i> using glycerol asparagine agar (ISP5), starch casein nitrate agar (SC) and Czapekâ€™s agar (CZ).</div>  <div class="alternatives" title="alternatives"> <div class="graphic" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1"><b>Sample</b></th> <th align="center" colspan="3" rowspan="1"><b>Isolation medium</b></th> <th rowspan="1" colspan="1"><b>Total (%)</b></th> </tr> <tr> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1"><b>ISP5</b></th> <th rowspan="1" colspan="1"><b>SC</b></th> <th rowspan="1" colspan="1"><b>CZ</b></th> <th rowspan="1" colspan="1" /> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Adults</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1"><b>1 (4%)</b></td> </tr> <tr> <td rowspan="1" colspan="1">Pollen</td> <td rowspan="1" colspan="1">11</td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1"><b>21 (84%)</b></td> </tr> <tr> <td rowspan="1" colspan="1">Honey</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1"><b>3 (12%)</b></td> </tr> <tr> <td rowspan="1" colspan="1"><b>Total (%)</b></td> <td rowspan="1" colspan="1"><b>15 (60%)</b></td> <td rowspan="1" colspan="1"><b>5 (20%)</b></td> <td rowspan="1" colspan="1"><b>5 (20%)</b></td> <td rowspan="1" colspan="1"><b>25 (100%)</b></td> </tr> </tbody> </table> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10512/fig-1"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Inhibitory effects of actinomycetes.</div> <p>Inhibitory effects of actinomycetes against the growth of 1. <i>R. solanacearum</i>; 2. <i>X. campestris</i> pv. <i>campestris</i>;<i /> 3. <i>X. oryzae</i> pv. <i>oryzae</i> and 4. <i>P. carotovorum</i> on ISP2 agar plates: (A) DSC3-6, (B) DGA3-20 and (C) DGA8-3.</p>   </div> <div class="table-wrap_UNKNOWN" id="table-2" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7717/peerj.10512/table-2">10.7717/peerj.10512/table-2</a><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Characterization and identification of actinomycetes using the 16S rDNA gene sequence and minimum inhibitory concentration (MIC) values of actinomycetes against the growth of 1, <i>X. campestris</i> pv. <i>campestris</i>; 2, <i>X. oryzae</i> pv. <i>oryzae</i>; 3, <i>R. solanacearum</i> and 4, <i>P. carotovorum</i>.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup> <colgroup><col span="1" /></colgroup>  <thead> <tr> <th rowspan="1" colspan="1"><b>Isolate No.</b></th> <th rowspan="1" colspan="1"><b>Morphological characteristic</b></th> <th rowspan="1" colspan="1"><b>Source</b></th> <th rowspan="1" colspan="1"><b>Accession No.</b></th> <th align="center" colspan="4" rowspan="1"><b>MIC</b><b>(mg L</b><sup>âˆ’1</sup><b>)</b></th> <th rowspan="1" colspan="1"><b>16S rDNA gene identification</b><b>(% similarity)</b></th> </tr> <tr> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1"><b>1</b></th> <th rowspan="1" colspan="1"><b>2</b></th> <th rowspan="1" colspan="1"><b>3</b></th> <th rowspan="1" colspan="1"><b>4</b></th> <th rowspan="1" colspan="1" /> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">DSC3-6</td> <td rowspan="1" colspan="1">Powdery colonies</td> <td rowspan="1" colspan="1">Pollen</td> <td rowspan="1" colspan="1"><a href="http://www.ncbi.nlm.nih.gov/nuccore/LC536753">LC536753</a></td> <td rowspan="1" colspan="1">32</td> <td rowspan="1" colspan="1">16</td> <td rowspan="1" colspan="1">32</td> <td rowspan="1" colspan="1">64</td> <td rowspan="1" colspan="1"><i>Streptomyces ramulosus</i> (99.42%)</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">Substrate mycelium: cream</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">Aerial spore mass: grey or black</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">Produce yellow pigment</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">DGA3-20</td> <td rowspan="1" colspan="1">Powdery colonies</td> <td rowspan="1" colspan="1">Pollen</td> <td rowspan="1" colspan="1"><a href="http://www.ncbi.nlm.nih.gov/nuccore/LC536752">LC536752</a></td> <td rowspan="1" colspan="1">32</td> <td rowspan="1" colspan="1">32</td> <td rowspan="1" colspan="1">32</td> <td rowspan="1" colspan="1">64</td> <td rowspan="1" colspan="1"><i>Streptomyces axinellae</i> (99.70%)</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">Substrate mycelium: cream or grey</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">Aerial spore mass: grey or black</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1">DGA8-3</td> <td rowspan="1" colspan="1">Powdery colonies</td> <td rowspan="1" colspan="1">Honey</td> <td rowspan="1" colspan="1"><a href="http://www.ncbi.nlm.nih.gov/nuccore/LC536754">LC536754</a></td> <td rowspan="1" colspan="1">32</td> <td rowspan="1" colspan="1">32</td> <td rowspan="1" colspan="1">32</td> <td rowspan="1" colspan="1">64</td> <td rowspan="1" colspan="1"><i>Streptomyces drozdowiczii</i> (99.71%)</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">Substrate mycelium: cream</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">Aerial spore mass: grey or black</td> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1" /> </tr> </tbody> </table> </div> </div> </div> <div class="identificationofactinomycetesusing16srdna" title="sec"> <div class="title" tagx="title" title="title">
Identification of actinomycetes using 16S rDNA</div> <p>The three actinobacterial isolates that showed inhibition of all four phytobacterial pathogens were identified using 16S rDNA sequencing. 16S rRNA gene sequences for strains DSC3-6 (<a href="http://www.ncbi.nlm.nih.gov/nuccore/LC536753">LC536753</a>), DGA3-20 (<a href="http://www.ncbi.nlm.nih.gov/nuccore/LC536752">LC536752</a>) and DSC8-3 (<a href="http://www.ncbi.nlm.nih.gov/nuccore/LC536754">LC536754</a>) were analyzed by BLAST using the GenBank database. The results showed that all strains had high similarity to members of the genus <i>Streptomyces</i>. The almost complete 16S rRNA gene sequences for strains DSC3-6, DGA3-20 and DGA8-3 were compared with the corresponding sequences of closely related strains of the genus <i>Streptomyces</i>. The maximum likelihood tree (<a href="#fig-2">Fig. 2</a>) revealed that strains DSC3-6, DGA3-20 and DGA8-3 were closely related to <i>S. ramulosus</i>, <i>S. axinellae</i> and <i>S. drozdowiczii</i>, respectively.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7717/peerj.10512/fig-2"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
Maximum likelihood (ML) tree based on 16S rRNA gene sequences showing the phylogenetic positions of DSC3-6, DGA3-20 and DSC8-3 relative to type strains of other <i>Streptomyces</i> species.</div> <p><i>S. thermocarboxydus</i> DSM 44293<sup>T</sup> was used as an outgroup. The number at each node is the bootstrap support value (%) based on 1,000 replicates (only values &amp;gt; 50% are shown). The scale bar shows 0.010 substitutions per nucleotide position.</p>   </div> <p>The sequence similarity value between each actinomycete isolate and its closely related type strain was aligned and calculated from the pairwise alignment. The results showed that DSC3-6, DGA3-20 and DGA8-3 were closely related to <i>S. ramulosus</i> (99.42%), <i>S. axinellae</i> (99.70%) and <i>S. drozdowiczii</i> (99.71%), respectively (<a href="#table-2">Table 2</a>).</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>This study investigated antagonistic activity against phytopathogenic bacteria of actinomycetes isolated from the giant honey bee (<i>A. dorsata</i>). Using ISP5 agar, DGA3-20 and DGA8-3 were obtained from pollen and honey samples. However, isolate DSC3-6, which had the highest activity against <i>X. oryzae</i> pv. <i>oryzae</i>, was obtained from pollen using starch casein nitrate agar. This indicated that using different isolation media may increase the opportunity of finding potential actinomycete strains. The three <i>Streptomyces</i> strains capable of inhibiting phytopathogens were obtained from pollen and honey stored in combs. The actinomycetes may be taken into hives by the worker bees collecting food and/or water from environmental sources outside the hives (<a href="#ref-25">Promnuan, Kudo &amp;amp; Chantawannakul, 2009</a>). <i>Streptomyces</i> were isolated from strawberry flowers and pollen cultivated in high-bed greenhouses in Jinju, Republic of Korea. This study showed that honey bees (<i>A. mellifera</i>) can transfer <i>Streptomyces</i> bacteria among flowers and strawberry plants. In addition, these endophytic <i>Streptomyces</i> had the ability to protect both plant and honey bees from phytopathogenic fungi (<i>Botrytis cinerea</i>) and entomopathogens, respectively (<a href="#ref-10">Kim et al., 2019</a>).</p> <p>Based on 16S rDNA sequence analysis, all potent isolates belonged to the genus <i>Streptomyces</i>. Actinomycetes, especially <i>Streptomyces</i> are well known for the production of secondary metabolites with antagonistic activity against phytopathogens (<a href="#ref-33">Viaene et al., 2016</a>). There have been reports of actinobacteria associated with insects. For example, <i>Streptomyces</i>, <i>Micromonospora</i> and <i>Actinoplanes</i> isolated from nests of the paper wasp <i>Polistes dominulus</i> could inhibit the growth of <i>Pseudomonas aeruginosa, Escherichia coli</i>, <i>Staphylococcus aureus</i>, <i>Serratia marcescens</i> and <i>Bacillus subtilis</i> (<a href="#ref-15">Madden et al., 2013</a>). The <i>Streptomyces</i> spp. isolated from solitary wasp mud nests showed activity against various drug-resistant bacterial pathogens. The isolate MN 9(V) showed activity against both <i>E. coli</i> and <i>P. aeruginosa</i> at a concentration of 25 mg mL<sup>âˆ’1</sup> (<a href="#ref-12">Kumar et al., 2012</a>). The novel macrocyclic lactam sceliphrolactam was isolated from mud dauber wasps (<i>Chalybion californicum</i> and <i>Sceliphron caementarium</i>) and could act as an antifungal by destabilizing fungal cell membrane functions (<a href="#ref-23">Poulsen et al., 2011</a>). Actinomycetes associated with bees (<i>A. mellifera</i>, <i>A. cereana</i> and <i>A. florea</i>) produce antimicrobial compounds that inhibit the growth of bacterial pathogens causing American and European foulbrood diseases in honey bees (<a href="#ref-25">Promnuan, Kudo &amp;amp; Chantawannakul, 2009</a>). Furthermore, <i>Streptomyces</i> spp. isolated from black dwarf honey bee (<i>A. andreniformis</i>), showed high activity in decreasing the egg hatch rate and increasing the infective second-stage juvenile mortality rate of the root-knot nematode (<i>Meloidogyne incognita</i>) <i>in vitro</i> and reduced root gall of chili <i>in vivo</i> (<a href="#ref-30">Santisuk et al., 2018</a>). These results indicated that actinomycetes associated with insects can provide novel antimicrobial products for use in agriculture.</p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>Bacterial bright leaf disease, which is caused by <i>X. oryzae</i> pv. <i>oryzae</i>, is one of the major diseases of rice in Asian countries. This study is the first report of the antibacterial activity of actinomycete species isolated from giant honey bee (<i>A. dorsata</i>) combs. <i>In vitro</i>, the crude extract of <i>S. ramulosus</i> (DSC3-6) had the highest activity against the growth of <i>X. oryzae</i> pv<i>. oryzae.</i> According to the current results, actinomycetes associated with the giant honey bee could be a good source of bioactive compounds for use in agriculture. In further studies, potent actinomycete strains will be investigated as biocontrol agents with plants under greenhouse conditions.</p> </div> </div> <div class="back" title="back"> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional Information and Declaration</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Competing Interests</div> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors declare there are no competing interests.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Author Contributions</div> <div class="fn-type-con" title="con"> <p><a href="#author-1">Yaowanoot Promnuan</a> conceived and designed the experiments, performed the experiments, analyzed the data, prepared figures and/or tables, authored or reviewed drafts of the paper, and approved the final draft.</p> </div> <div class="fn-type-con" title="con"> <p><a href="#author-2">Saran Promsai</a> and <a href="#author-3">Sujinan Meelai</a> conceived and designed the experiments, authored or reviewed drafts of the paper, and approved the final draft.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Field Study Permissions</div> <div class="fn-type-" title=""> <p>The following information was supplied relating to field study approvals (i.e., approving body and any reference numbers):</p> <p>The three hive samples were collected from local villages in private areas. Verbal permission was acquired from each of the owners:</p> <p>- Mr. Ton Tatiya (2 hives) 25/2 Moo2, Pong Yaeng Sub-district, Mae-Rim district, Chiang Mai Province, Thailand.</p> <p>- Mr. Ma Madamun (1 hive) 93/3 Moo2, Pong Yaeng Sub-district, Mae-Rim district, Chiang Mai Province, Thailand.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
DNA Deposition</div> <div class="fn-type-" title=""> <p>The following information was supplied regarding the deposition of DNA sequences:</p> <p>The almost-complete 16S rRNA gene sequences described here are accessible via GenBank: <a href="http://www.ncbi.nlm.nih.gov/nuccore/LC536752">LC536752</a> to <a href="http://www.ncbi.nlm.nih.gov/nuccore/LC536754">LC536754</a>.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title">
Data Availability</div> <div class="fn-type-" title=""> <p>The following information was supplied regarding data availability:</p> <p>The phylogenetic trees are available in <a href="#fig-2">Fig. 2</a>.</p> </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ref-1" /><span class="label" tagx="label" title="label">Agate &amp;amp; Bhat (1963)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agate</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhat</span><span class="given-names" tagx="given-names" title="given-names">JV</span></span></span><span class="year" tagx="year" title="year">1963</span><span class="mixed-article-title" title="mixed-article-title">A method for the preferential isolation of actinomycetes from soils</span><span class="source" tagx="source" title="source">Antonie van Leeuwenhoek</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">297</span><span class="lpage" tagx="lpage" title="lpage">304</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02046072">10.1007/BF02046072</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14068462">14068462</a></span></span></li> <li tag="ref"><a name="ref-2" /><span class="label" tagx="label" title="label">Berdy (2005)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berdy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Bioactive Microbial Metabolites</span><span class="source" tagx="source" title="source">The Journal of Antibiotics</span><span class="volume" tagx="volume" title="volume">58</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">26</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ja.2005.1">10.1038/ja.2005.1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15813176">15813176</a></span></span></li> <li tag="ref"><a name="ref-3" /><span class="label" tagx="label" title="label">Chavan etÂ al. (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chavan</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nawani</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanda</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tandon</span><span class="given-names" tagx="given-names" title="given-names">DD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khetmalas</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Biocontrol potential of actinomycetes against <i>Xanthomonas axonopodis</i> pv. <i>punicae</i>, a causative agent for oily spot disease of pomegranate</span><span class="source" tagx="source" title="source">Biocontrol Science and Technology</span><span class="volume" tagx="volume" title="volume">26</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">351</span><span class="lpage" tagx="lpage" title="lpage">372</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/09583157.2015.1116057">10.1080/09583157.2015.1116057</a></span></span></li> <li tag="ref"><a name="ref-4" /><span class="label" tagx="label" title="label">Felsenstein (1985)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felsenstein</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1985</span><span class="mixed-article-title" title="mixed-article-title">Confidence limits on phylogenies: an approach using the bootstrap</span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">783</span><span class="lpage" tagx="lpage" title="lpage">791</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.1985.tb00420.x">10.1111/j.1558-5646.1985.tb00420.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28561359">28561359</a></span></span></li> <li tag="ref"><a name="ref-5" /><span class="label" tagx="label" title="label">Hall (1999)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT</span><span class="source" tagx="source" title="source">Nucleic Acids Symposium Series</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">95</span><span class="lpage" tagx="lpage" title="lpage">98</span></span></li> <li tag="ref"><a name="ref-6" /><span class="label" tagx="label" title="label">Hastuti etÂ al. (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hastuti</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lestari</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suwanto</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saraswati</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Endophytic <i>Streptomyces</i> spp. as biocontrol agents of rice bacterial leaf blight pathogen (<i>Xanthomonas oryzae</i> pv. <i>oryzae</i>)</span><span class="source" tagx="source" title="source">Hayati</span><span class="volume" tagx="volume" title="volume">19</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">155</span><span class="lpage" tagx="lpage" title="lpage">162</span></span></li> <li tag="ref"><a name="ref-7" /><span class="label" tagx="label" title="label">Ishikawa etÂ al. (2004)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishikawa</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki-Nishimito</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukuchi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Effective control of cabbage black rot by Validamycin A and its effect on extracellular polysaccharide - production of <i>Xanthomonas campestris</i> pv. <i>campestris</i></span><span class="source" tagx="source" title="source">Journal of Pesticide Science</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="lpage" tagx="lpage" title="lpage">213</span><span class="pub-id"><a href="https://dx.doi.org/10.1584/jpestics.29.209">10.1584/jpestics.29.209</a></span></span></li> <li tag="ref"><a name="ref-8" /><span class="label" tagx="label" title="label">Itoh etÂ al. (2003)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Itoh</span><span class="given-names" tagx="given-names" title="given-names">IK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holeva</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birch</span><span class="given-names" tagx="given-names" title="given-names">PRJ</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Soft rot erwiniae: from genes to genomes</span><span class="source" tagx="source" title="source">Molecular Plant Pathology</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">17</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1364-3703.2003.00149.x">10.1046/j.1364-3703.2003.00149.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20569359">20569359</a></span></span></li> <li tag="ref"><a name="ref-9" /><span class="label" tagx="label" title="label">Kaur etÂ al. (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaur</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nyochembeng</span><span class="given-names" tagx="given-names" title="given-names">LN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mentreddy</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banerjee</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cebert</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Assessment of the antimicrobial activity of Lentinula edodes against <i>Xanthomonas campestris</i> pv. <i>vesicatoria</i></span><span class="source" tagx="source" title="source">Crop Protection</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">284</span><span class="lpage" tagx="lpage" title="lpage">288</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cropro.2016.08.001">10.1016/j.cropro.2016.08.001</a></span></span></li> <li tag="ref"><a name="ref-10" /><span class="label" tagx="label" title="label">Kim etÂ al. (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeon</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weller</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomashow</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paulitz</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwak</span><span class="given-names" tagx="given-names" title="given-names">YS</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">A mutualistic interaction between <i>Streptomyces</i> bacteria, strawberry plants and pollinating bees</span><span class="source" tagx="source" title="source">Nature Communications</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">4802</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-019-12785-3">10.1038/s41467-019-12785-3</a></span></span></li> <li tag="ref"><a name="ref-11" /><span class="label" tagx="label" title="label">Kumar etÂ al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knyaz</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">MEGA X: molecular evolutionary genetics analysis across computing platforms</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">1547</span><span class="lpage" tagx="lpage" title="lpage">1549</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msy096">10.1093/molbev/msy096</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29722887">29722887</a></span></span></li> <li tag="ref"><a name="ref-12" /><span class="label" tagx="label" title="label">Kumar etÂ al. (2012)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bharti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">VK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O</span><span class="given-names" tagx="given-names" title="given-names">Gusain</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bisht</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Actinomycetes from solitary wasp mud nest and swallow bird mud nest: isolation and screening for their antibacterial activity</span><span class="source" tagx="source" title="source">World Journal of Microbiology &amp;amp; Biotechnology</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">871</span><span class="lpage" tagx="lpage" title="lpage">880</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11274-011-0884-2">10.1007/s11274-011-0884-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22805807">22805807</a></span></span></li> <li tag="ref"><a name="ref-13" /><span class="label" tagx="label" title="label">KÃ¼ster &amp;amp; Williams (1964)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ¼ster</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span></span><span class="year" tagx="year" title="year">1964</span><span class="mixed-article-title" title="mixed-article-title">Selection of media for isolation of streptomycetes</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">202</span><span class="fpage" tagx="fpage" title="fpage">928</span><span class="lpage" tagx="lpage" title="lpage">929</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/202928a0">10.1038/202928a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14190108">14190108</a></span></span></li> <li tag="ref"><a name="ref-14" /><span class="label" tagx="label" title="label">Lemos, Toranzo &amp;amp; Barja (1985)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemos</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toranzo</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barja</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="year" tagx="year" title="year">1985</span><span class="mixed-article-title" title="mixed-article-title">Antibiotic activity of epiphytic bacteria isolated from intertidal seaweeds</span><span class="source" tagx="source" title="source">Microbial Ecology</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">149</span><span class="lpage" tagx="lpage" title="lpage">163</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02010487">10.1007/BF02010487</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24221303">24221303</a></span></span></li> <li tag="ref"><a name="ref-15" /><span class="label" tagx="label" title="label">Madden etÂ al. (2013)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madden</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grassetti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soriano</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Starks</span><span class="given-names" tagx="given-names" title="given-names">PT</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Actinomycetes with antimicrobial activity isolated from paper wasp (Hymenoptera: Vespidae: Polistinae) Nests</span><span class="source" tagx="source" title="source">Environmental Entomology</span><span class="volume" tagx="volume" title="volume">42</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">703</span><span class="lpage" tagx="lpage" title="lpage">710</span><span class="pub-id"><a href="https://dx.doi.org/10.1603/EN12159">10.1603/EN12159</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23905732">23905732</a></span></span></li> <li tag="ref"><a name="ref-16" /><span class="label" tagx="label" title="label">Mingma etÂ al. (2014)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mingma</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pathom-aree</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trakulnaleamsai</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thamchaipenet</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duangmal</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Isolation of rhizospheric and roots endophytic actinomycetes from Leguminosae plant and their activities to inhibit soybean pathogen, <i>Xanthomonas campestris</i> pv. <i>glycine</i></span><span class="source" tagx="source" title="source">World Journal of Microbiology &amp;amp; Biotechnology</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">271</span><span class="lpage" tagx="lpage" title="lpage">280</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11274-013-1451-9">10.1007/s11274-013-1451-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23913026">23913026</a></span></span></li> <li tag="ref"><a name="ref-17" /><span class="label" tagx="label" title="label">Mishra &amp;amp; Arora (2011)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mishra</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arora</span><span class="given-names" tagx="given-names" title="given-names">NK</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Evaluation of rhizospheric <i>Pseudomonas</i> and <i>Bacillus</i> as biocontrol tool for <i>Xanthomonas campestris</i> pv <i>campestris.</i></span><span class="source" tagx="source" title="source">World Journal of Microbiology &amp;amp; Biotechnology</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">693</span><span class="lpage" tagx="lpage" title="lpage">702</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11274-011-0865-5">10.1007/s11274-011-0865-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22806865">22806865</a></span></span></li> <li tag="ref"><a name="ref-18" /><span class="label" tagx="label" title="label">Mougou &amp;amp; Boughalleb-Mâ€™hamdi (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mougou</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boughalleb-Mâ€™hamdi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Biocontrol of <i>Pseudomonas syringae</i> pv. <i>syringae</i> affecting citrus orchards in Tunisia by using indigenous <i>Bacillus</i> spp. and garlic extract</span><span class="source" tagx="source" title="source">Egyptian Journal of Biological Pest Control</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">60</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s41938-018-0061-0">10.1186/s41938-018-0061-0</a></span></span></li> <li tag="ref"><a name="ref-19" /><span class="label" tagx="label" title="label">Muangham, Pathom-aree &amp;amp; Duangmal (2015)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muangham</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pathom-aree</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duangmal</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Melanogenic actinomycetes from rhizosphere soil-antagonistic activity against <i>Xanthomonas oryzae</i> and plant-growth-promoting traits</span><span class="source" tagx="source" title="source">Canadian Journal of Microbiology</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">164</span><span class="lpage" tagx="lpage" title="lpage">170</span><span class="pub-id"><a href="https://dx.doi.org/10.1139/cjm-2014-0645">10.1139/cjm-2014-0645</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25620622">25620622</a></span></span></li> <li tag="ref"><a name="ref-20" /><span class="label" tagx="label" title="label">Nakajima etÂ al. (1999)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakajima</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitpreechavanich</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kudo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title"><i>Microbispora coralline</i> sp. nov. a new species of the genus <i>Microbispora</i> isolated from Thai soil</span><span class="source" tagx="source" title="source">International Journal of Systematic Bacteriology</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">1761</span><span class="lpage" tagx="lpage" title="lpage">1767</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10555358">10555358</a></span></span></li> <li tag="ref"><a name="ref-21" /><span class="label" tagx="label" title="label">Ombiro etÂ al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ombiro</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sawai</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noutoshi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishina</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsui</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamanota</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyada</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ichinose</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Specific growth inhibitors of <i>Ralstonia solanacearum</i>, <i>Xanthomonas oryzae</i> pv. <i>oryzae</i>, <i>X.Â campestris</i> pv. <i>campestris</i>, and <i>Clavibacter michiganensis</i> subsp. <i>michiganensis</i></span><span class="source" tagx="source" title="source">Microbiology Research</span><span class="volume" tagx="volume" title="volume">215</span><span class="fpage" tagx="fpage" title="fpage">29</span><span class="lpage" tagx="lpage" title="lpage">35</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.micres.2018.06.005">10.1016/j.micres.2018.06.005</a></span></span></li> <li tag="ref"><a name="ref-22" /><span class="label" tagx="label" title="label">Photita etÂ al. (2004)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Photita</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lumyong</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lumyong</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKenzie</span><span class="given-names" tagx="given-names" title="given-names">EHC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyde</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Are some endophytes of Musa acuminate latent pathogens?</span><span class="source" tagx="source" title="source">Fungal Diversity</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">131</span><span class="lpage" tagx="lpage" title="lpage">140</span></span></li> <li tag="ref"><a name="ref-23" /><span class="label" tagx="label" title="label">Poulsen etÂ al. (2011)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poulsen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oh</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clardy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Currie</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Chemical analyses of wasp-associated <i>Streptomyces</i> bacteria reveal a prolific potential for natural products discovery</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e16763</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0016763">10.1371/journal.pone.0016763</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21364940">21364940</a></span></span></li> <li tag="ref"><a name="ref-24" /><span class="label" tagx="label" title="label">Pridham &amp;amp; Lyons (1961)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pridham</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyons</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="year" tagx="year" title="year">1961</span><span class="mixed-article-title" title="mixed-article-title"><i>Streptomyces albus</i> (Rossi Doria) Waskman et Henrici: taxonomic study of strains labeled <i>Streptomyces albus</i></span><span class="source" tagx="source" title="source">Journal of Bacteriology</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">431</span><span class="lpage" tagx="lpage" title="lpage">441</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.81.3.431-441.1961">10.1128/JB.81.3.431-441.1961</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13737991">13737991</a></span></span></li> <li tag="ref"><a name="ref-25" /><span class="label" tagx="label" title="label">Promnuan, Kudo &amp;amp; Chantawannakul (2009)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Promnuan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kudo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chantawannakul</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Actinomycetes isolated from beehives in Thailand</span><span class="source" tagx="source" title="source">World Journal of Microbiology &amp;amp; Biotechnology</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1685</span><span class="lpage" tagx="lpage" title="lpage">1689</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11274-009-0051-1">10.1007/s11274-009-0051-1</a></span></span></li> <li tag="ref"><a name="ref-26" /><span class="label" tagx="label" title="label">Promnuan, Kudo &amp;amp; Chantawannakul (2011)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Promnuan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kudo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chantawannakul</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title"><i>Actinomadura apis</i> sp. nov. isolated from a honey bee (<i>Apis mellifera</i>) hive, and the reclassification of <i>Actinomadura cremea</i> subsp. <i>rifamycini</i> Gauze et al.1987 as <i>Actinomadura rifamycini</i> (Gauze et al.1987) sp. nov. comb. nov</span><span class="source" tagx="source" title="source">International Journal of Systematic Bacteriology</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">2271</span><span class="lpage" tagx="lpage" title="lpage">2277</span></span></li> <li tag="ref"><a name="ref-27" /><span class="label" tagx="label" title="label">Promnuan, Kudo &amp;amp; Chantawannakul (2013)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Promnuan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kudo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chantawannakul</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title"><i>Streptomyces chiangmaiensis</i> sp. nov. and <i>Streptomyces lannensis</i> sp. nov. isolated from the South-East Asian stingless bee (<i>Tetragonilla collina</i>)</span><span class="source" tagx="source" title="source">International Journal of Systematic Bacteriology</span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">1896</span><span class="lpage" tagx="lpage" title="lpage">1901</span></span></li> <li tag="ref"><a name="ref-28" /><span class="label" tagx="label" title="label">Qian etÂ al. (2006)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">An</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Antibacterial activity of xantho-oligosaccharide cleaved from xanthan against phytopathogenic <i>Xanthomonas campestris</i> pv. <i>campestris.</i></span><span class="source" tagx="source" title="source">Process Biochemistry</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">1582</span><span class="lpage" tagx="lpage" title="lpage">1588</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.procbio.2006.03.003">10.1016/j.procbio.2006.03.003</a></span></span></li> <li tag="ref"><a name="ref-29" /><span class="label" tagx="label" title="label">Sabir etÂ al. (2017)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabir</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El-Khalfi</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Errachidi</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chemsi</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serrano</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soukri</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Evaluation of the potential of some essential oils in biological control against phytopathogenic agent <i>Pseudomonas syringae</i> pv. <i>tomato</i></span><span class="source" tagx="source" title="source">Journal of Plant Pathology &amp;amp; Microbiology</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1000420</span></span></li> <li tag="ref"><a name="ref-30" /><span class="label" tagx="label" title="label">Santisuk etÂ al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santisuk</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Promnuan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khun-In</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nimnoi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruanpanun</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Efficiency of actinomycetes isolated from black dwarf honey bee (<i>Apis andreniformis</i>) in controlling root-knot nematode, <i>Meloidogyne incognita</i> causes root knot disease of chili in greenhouse</span><span class="source" tagx="source" title="source">Journal of Agriculture</span><span class="volume" tagx="volume" title="volume">34</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">481</span><span class="lpage" tagx="lpage" title="lpage">490</span></span></li> <li tag="ref"><a name="ref-31" /><span class="label" tagx="label" title="label">Satish, Raveesha &amp;amp; Jandrdhana (1999)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satish</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raveesha</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jandrdhana</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Antibacterial activity of plant extracts on phytopathogenic <i>Xanthomonas campestris</i> pathovars</span><span class="source" tagx="source" title="source">Letters in Applied Microbiology</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">145</span><span class="lpage" tagx="lpage" title="lpage">147</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-2672.1999.00479.x">10.1046/j.1365-2672.1999.00479.x</a></span></span></li> <li tag="ref"><a name="ref-32" /><span class="label" tagx="label" title="label">Shirling &amp;amp; Gottlieb (1966)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirling</span><span class="given-names" tagx="given-names" title="given-names">EB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gottlieb</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">1966</span><span class="mixed-article-title" title="mixed-article-title">Methods for characterization of <i>Streptomyces</i> species</span><span class="source" tagx="source" title="source">International Journal of Systematic Bacteriology</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">313</span><span class="lpage" tagx="lpage" title="lpage">340</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/00207713-16-3-313">10.1099/00207713-16-3-313</a></span></span></li> <li tag="ref"><a name="ref-33" /><span class="label" tagx="label" title="label">Viaene etÂ al. (2016)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viaene</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langendries</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beirinckx</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maes</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goormachtig</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title"><i>Streptomyces</i> as a plantâ€™s best friend?</span><span class="source" tagx="source" title="source">FEMS Microbiology Ecology</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">fiw119</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/femsec/fiw119">10.1093/femsec/fiw119</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27279415">27279415</a></span></span></li> <li tag="ref"><a name="ref-34" /><span class="label" tagx="label" title="label">Vicente &amp;amp; Holub (2013)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vicente</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holub</span><span class="given-names" tagx="given-names" title="given-names">EB</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title"><i>Xanthomonas campestris</i> pv. <i>campestris</i> (cause of black rot of crucifers) in the genomic era is still a worldwide threat to brassica crops</span><span class="source" tagx="source" title="source">Molecular Plant Pathology</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">2</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1364-3703.2012.00833.x">10.1111/j.1364-3703.2012.00833.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23051837">23051837</a></span></span></li> <li tag="ref"><a name="ref-35" /><span class="label" tagx="label" title="label">Waksman (1950)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waksman</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span></span><span class="year" tagx="year" title="year">1950</span><span class="mixed-article-title" title="mixed-article-title">The actinomycetes: their nature, occurrence, activities and importance</span><span class="source" tagx="source" title="source">Annales Cryptogamici Et Phytopathologici</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">230</span></span></li> <li tag="ref"><a name="ref-36" /><span class="label" tagx="label" title="label">Wiegand, Hilpert &amp;amp; Hancock (2008)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiegand</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilpert</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hancock</span><span class="given-names" tagx="given-names" title="given-names">REW</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Agar and broth dilution methods to determine the minimal inhibitory concentration (MIC) of antimicrobial substances</span><span class="source" tagx="source" title="source">Nature Protocols</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">163</span><span class="lpage" tagx="lpage" title="lpage">175</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2007.521">10.1038/nprot.2007.521</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18274517">18274517</a></span></span></li> <li tag="ref"><a name="ref-37" /><span class="label" tagx="label" title="label">Wongsiri etÂ al. (1996)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wongsiri</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lekprayoon</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thapa</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thirakupt</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rinderer</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sylvester</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oldroyd</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Booncham</span><span class="given-names" tagx="given-names" title="given-names">U</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Comparative biology of <i>Apis andreniformis</i> and <i>Apis florea</i> in Thailand</span><span class="source" tagx="source" title="source">Bee World</span><span class="volume" tagx="volume" title="volume">77</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">23</span><span class="lpage" tagx="lpage" title="lpage">35</span></span></li> <li tag="ref"><a name="ref-38" /><span class="label" tagx="label" title="label">Wu etÂ al. (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">The critical role of cytochrome c maturation (CCM) system in the tolerance of <i>Xanthomonas campestris</i> pv. <i>campestris</i> to phenazines</span><span class="source" tagx="source" title="source">Pesticide Biochemistry &amp;amp; Physiology</span><span class="volume" tagx="volume" title="volume">156</span><span class="fpage" tagx="fpage" title="fpage">63</span><span class="lpage" tagx="lpage" title="lpage">71</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pestbp.2019.02.003">10.1016/j.pestbp.2019.02.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31027582">31027582</a></span></span></li> <li tag="ref"><a name="ref-39" /><span class="label" tagx="label" title="label">Wulff etÂ al. (2002)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wulff</span><span class="given-names" tagx="given-names" title="given-names">EG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mguni</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mortensen</span><span class="given-names" tagx="given-names" title="given-names">CN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keswani</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hockenhull</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Biological control of black rot (<i>Xanthomonas campestris</i> pv. <i>campestris</i>) of brassicas with an antagonistic strain of <i>Bacillus subtilis</i> in Zimbabwe</span><span class="source" tagx="source" title="source">European Journal of Plant Pathology</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">317</span><span class="lpage" tagx="lpage" title="lpage">25</span></span></li> <li tag="ref"><a name="ref-40" /><span class="label" tagx="label" title="label">Xie etÂ al. (2018)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajer</span><span class="given-names" tagx="given-names" title="given-names">FU</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Antibacterial effects of volatiles produced by Bacillus strain D13 against <i>Xanthomonas oryzae</i> pv. <i>oryzae</i></span><span class="source" tagx="source" title="source">Molecular Plant Pathology</span><span class="volume" tagx="volume" title="volume">19</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">49</span><span class="lpage" tagx="lpage" title="lpage">58</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mpp.12494">10.1111/mpp.12494</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27682316">27682316</a></span></span></li> <li tag="ref"><a name="ref-41" /><span class="label" tagx="label" title="label">Zhao etÂ al. (2013)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Control of postharvest soft rot caused by <i>Erwinia carotovora</i> of vegetables by a strain of <i>Bacillus amyloliquefaciens</i> and its potential modes of action</span><span class="source" tagx="source" title="source">World Journal of Microbiology &amp;amp; Biotechnology</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">411</span><span class="lpage" tagx="lpage" title="lpage">420</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11274-012-1193-0">10.1007/s11274-012-1193-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23117674">23117674</a></span></span></li> <li tag="ref"><a name="ref-42" /><span class="label" tagx="label" title="label">Zheng etÂ al. (2019)</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Y</span><span class="given-names" tagx="given-names" title="given-names">Zhu</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">A <i>Streptomyces</i> sp. strain: isolation, identification, and potential as a biocontrol agent against soilborne diseases of tomato plants</span><span class="source" tagx="source" title="source">Biological Control</span><span class="volume" tagx="volume" title="volume">136</span><span class="fpage" tagx="fpage" title="fpage">104004</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biocontrol.2019.104004">10.1016/j.biocontrol.2019.104004</a></span></span></li> </ul> </div> </div>  </body></html>