<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">BMC Plant Biol</span><span class="iso-abbrev" title="iso-abbrev">BMC Plant Biol</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">BMC Plant Biology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1471-2229</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7825178">7825178</a></span><span class="publisher-id" title="publisher-id">2825</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12870-021-02825-w">10.1186/s12870-021-02825-w</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Transcriptional and proteomic insights into phytotoxic activity of interspecific potato hybrids with low glycoalkaloid contents</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szajko</span><span class="given-names" tagx="given-names" title="given-names">Katarzyna</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciekot</span><span class="given-names" tagx="given-names" title="given-names">JarosÅ‚aw</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wasilewicz-Flis</span><span class="given-names" tagx="given-names" title="given-names">Iwona</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marczewski</span><span class="given-names" tagx="given-names" title="given-names">Waldemar</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7072-7630</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SoÅ‚tys-Kalina</span><span class="given-names" tagx="given-names" title="given-names">Dorota</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">d.soltys@ihar.edu.pl</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.425508.e</span><span class="institution-id" title="institution-id">0000 0001 2323 609X</span><span class="institution" title="institution">Plant Breeding and Acclimatization Institute, MÅ‚ochÃ³w Research Centre, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.418769.5</span><span class="institution-id" title="institution-id">0000 0001 1089 8270</span><span class="institution" title="institution">Ludwik Hirszfeld Institute of Immunology and Experimental Therapy, Laboratory of Biomedical Chemistry, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2021-1-1</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2021-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2021</span></span><span class="volume" tagx="volume" title="volume">21</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">60</span><span class="history" title="history"><span class="received" title="received">received: 2020-9-11</span><span class="accepted" title="accepted">accepted: 2021-1-5</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1">Glycoalkaloids are bioactive compounds that contribute to the defence response of plants against herbivore attack and during pathogenesis. Solanaceous plants, including cultivated and wild potato species, are sources of steroidal glycoalkaloids. <i>Solanum</i> plants differ in the content and composition of glycoalkaloids in organs. In wild and cultivated potato species, more than 50 steroidal glycoalkaloids were recognized. Steroidal glycoalkaloids are recognized as potential allelopathic/phytotoxic compounds that may modify the growth of target plants. There are limited data on the impact of the composition of glycoalkaloids on their phytotoxic potential.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par2">The presence of Î±-solasonine and Î±-solamargine in potato leaf extracts corresponded to the high phytotoxic potential of the extracts. Among the differentially expressed genes between potato leaf bulks with high and low phytotoxic potential, the most upregulated transcripts in sample of high phytotoxic potential were anthocyanin 5-aromatic acyltransferase-like and subtilisin-like protease SBT1.7-transcript variant X2. The most downregulated genes were carbonic anhydrase chloroplastic-like and miraculin-like. An analysis of differentially expressed proteins revealed that the most abundant group of proteins were those related to stress and defence, including glucan endo-1,3-beta-glucosidase acidic isoform, whose expression level was 47.96Ã— higher in potato leaf extract with low phytotoxic.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par3">The phytotoxic potential of potato leaf extract possessing low glycoalkaloid content is determined by the specific composition of these compounds in leaf extract, where Î±-solasonine and Î±-solamargine may play significant roles. Differentially expressed gene and protein profiles did not correspond to the glycoalkaloid biosynthesis pathway in the expression of phytotoxic potential. We cannot exclude the possibility that the phytotoxic potential is influenced by other compounds that act antagonistically or may diminish the glycoalkaloids effect.</p> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>The online version contains supplementary material available at 10.1186/s12870-021-02825-w.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Allelopathy</div> <div class="kwd" title="kwd">
Diploid hybrid</div> <div class="kwd" title="kwd">
Glucan endo-1,3-beta-glucosidase</div> <div class="kwd" title="kwd">
Leptine II</div> <div class="kwd" title="kwd">
Solamargine</div> <div class="kwd" title="kwd"><i>Solanum chacoense</i></div> <div class="kwd" title="kwd"><i>Solanum tuberosum</i></div> <div class="kwd" title="kwd">
Solasonine</div> <div class="kwd" title="kwd">
Threonine deaminase</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100004442</span><span class="institution" title="institution">Narodowym Centrum Nauki</span></div> </div><span class="award-id" tagx="award-id" title="award-id">UMO-2016/23/D/NZ9/02672</span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2021" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par15">Allelopathy is a broadly understood phenomenon that refers to multidirectional interactions among organisms (plants, bacteria, viruses and fungi) that involve the release of compounds called allelochemicals into the environment [<a href="#CR1">1</a>]. Plant allelopathy between donor and acceptor plants is mainly negative in nature and impairs plant growth, development and/or germination. The ability to synthetize and release allelopathic compounds, especially in plant-plant and plant-pathogen interactions, is an important aspect of allelopathy since it determines plant survival and proper development during biotic stresses [<a href="#CR2">2</a>]. To distinguish allelopathy in ecosystems from research on allelopathic interactions in laboratory, term â€˜phytotoxicityâ€™ is used, describing negative interactions between donors and acceptors [<a href="#CR1">1</a>]. A lot of laboratory research focused on recognition of phytotoxic potential use water extracts as phytotoxic factor. Water extracts most closely resemble leaching of compounds from plant organs, that occurs under natural conditions (leaching by rain, dew) [<a href="#CR3">3</a>, <a href="#CR4">4</a>].</p> <p id="Par16">An integral component of allelopathic interactions are allelochemicals that are secondary metabolites derived from three biosynthetic pathways: the shikimate, isoprenoid/mevalonate and polyketide pathways [<a href="#CR5">5</a>]. Based on their origin, secondary metabolites can be divided into three groups: phenylpropanoids, terpenoids, and polyketides. These compounds are distinguished from primary metabolites by their characteristic structure (unique carbon skeleton), which makes them specialized for responses to environmental conditions and biotic stresses. Secondary metabolites are involved in allelopathic interactions but principally in plant defence against pathogens [<a href="#CR6">6</a>].</p> <p id="Par17">Solanaceous species have been integral parts of human civilizations as food sources and drugs for thousands of years. All of the approximately 180 tuber-bearing <i>Solanum</i> species are indigenous to Latin America. They occur in a wide range of environmental conditions from Mexico in the north to Chile in the south and occupy various habitats [<a href="#CR7">7</a>]. The ability of wild potatoes to adapt and acclimatize to environmental conditions makes them a rich source of variability in biotic and abiotic stress resistance [<a href="#CR8">8</a>]. Steroidal glycoalkaloids, nitrogen-containing steroidal glycosides, are secondary metabolites that occur naturally in most plant organs of <i>Solanum</i> species [<a href="#CR9">9</a>]. They are biosynthesized by the sterol branch of the mevalonic acid/isoprenoid pathway [<a href="#CR10">10</a>]. There are many types of glycoalkaloids (GAs) in potato germplasm. Fifty-six GAs, including Î±-solanine, Î±-solasonine, Î±-solamargine, Î±-chaconine, and leptine II, were revealed in tubers of wild species and cultivated potato [<a href="#CR11">11</a>]. The GA content in tubers is markedly lower than that in potato leaves [<a href="#CR12">12</a>].</p> <p id="Par18">Î±-Solanine and Î±-chaconine possess antibacterial, fungicidal and insecticidal properties [<a href="#CR13">13</a>], can be classified as phytoalexins and are synthesized in response to pathogen infection [<a href="#CR14">14</a>]. Their biological activity depends mostly on their chemical structures, and Î±-chaconine is more active than Î±-solanine on the growth of fungi such as <i>Alternaria Brassicicola, Phytophthora. medicaginis,</i> and <i>Rhizoctonia solani</i> [<a href="#CR15">15</a>]. In our previous paper, we confirmed the potato phytotoxic potential (PP) against the test plant mustard (<i>Sinapis alba</i> L.), specie often used as aftercorp [<a href="#CR16">16</a>]. We demonstrated a significant role of total glycoalkaloid (TGA) content in the expression of PP among wild potato species and potato hybrids, and presented that some clones with low TGA content may possess PP.</p> <p id="Par19">For better understanding of processes/phenomena that may be directly or indirectly implicated with PP of potato under low TGA content, we used transcriptomic and proteomic approaches. It is known that allelopathic/phytotoxic interactions are determined directly by allelopathic compounds however, their content or profile in plantâ€™s organs is regulated not only at the genetic level but also undergo coordination of metabolic pathways. Transcriptomic and proteomic studies may provide complementary knowledge to integrative metabolite profiling and better understanding of allelopathy/phytotoxicity phenomena at plant system level [<a href="#CR17">17</a>]. In the present study, we showed that potato PP under low TGA content is directly related with GAs composition.</p> <p id="Par20">Contrasting F1 individuals differing in PP and GAs from a cross of an interspecific <i>Solanum</i> hybrid with <i>S. chacoense</i> were used for comprehensive analyses of transcriptomic and proteomic profiles.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="evaluationoftotalglycoalkaloid,glycoalkaloid,totalphenolicandtotalflavonoidcontents" title="sec"> <div class="title" tagx="title" title="title">
Evaluation of total glycoalkaloid, glycoalkaloid, total phenolic and total flavonoid contents</div> <p id="Par21">Two bulk samples, C and D, that had similar total glycoalkaloid (TGA) contents (2.7 and 2.6â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup>) and various PP were used (TableÂ <a href="#Tab1">1</a>). Both samples had similar concentrations of Î±-solanine (0.23 and 0.26â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup>) and Î±-chaconine (0.14 and 0.18â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup>) (Table <a href="#Tab1">1</a>). Î±-Solasonine was detected in only C at a concentration of 1.69â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup> and consisted 40.8% of GAs, while the most frequent glycoalkaloid in D was leptine II (66.7%, at a concentration of 1.22â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup>). The total flavonoids (TF) were significantly higher in D, while total phenolics (TP) were at similar levels in both samples (Table <a href="#Tab1">1</a>). The average retention time and mass of GAs found in the samples are presented in AdditionalÂ FileÂ <a href="#MOESM1">1</a>, and mass spectra are presented in AdditionalÂ FileÂ <a href="#MOESM2">2</a>. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Concentration, frequency of glycoalkaloids, total phenolics and total flavonoids in potato leaf extract of C and D samples</p>  <table frame="hsides" rules="groups"> <tbody> <tr> <td><b>Bulks</b></td> <td colspan="2"><b>C</b></td> <td colspan="2"><b>D</b></td> </tr> <tr> <td><b>Glycoalkaloids</b><sup><b>a</b></sup></td> <td> <p>Frequency</p> <p>[%]</p> </td> <td> <p>Concentration</p> <p>[Î¼g ml<sup>-1</sup>]</p> </td> <td> <p>Frequency</p> <p>[%]</p> </td> <td> <p>Concentration</p> <p>[Î¼g ml<sup>-1</sup>]</p> </td> </tr> <tr> <td>â€ƒÎ±-Solasonine</td> <td>40.8</td> <td>1.69Â±0.06</td> <td>0.0</td> <td>0.00</td> </tr> <tr> <td>â€ƒÎ±-Solamargine</td> <td>39.9</td> <td>1.65Â±0.05</td> <td>9.3</td> <td>0.17Â±0.04</td> </tr> <tr> <td>â€ƒÎ±-Solanine</td> <td>5.6</td> <td>0.23Â±0.02</td> <td>14.2</td> <td>0.26Â±0.06</td> </tr> <tr> <td>â€ƒÎ±-Chaconine</td> <td>3.4</td> <td>0.14Â±0.07</td> <td>9.8</td> <td>0.18Â±0.04</td> </tr> <tr> <td>â€ƒLeptine II<sup>b</sup></td> <td>10.4</td> <td>0.43Â±0.01</td> <td>66.7</td> <td>1.22Â±0.22</td> </tr> <tr> <td /> <td colspan="2"><b>C</b></td> <td colspan="2"><b>D</b></td> </tr> <tr> <td><b>TGA concentration [Î¼gml</b><sup><b>-1</b></sup><b>]</b><sup><b>c</b></sup></td> <td char="Â±" align="char" colspan="2">2.7Â±0.4</td> <td char="Â±" align="char" colspan="2">2.6Â±0.6</td> </tr> <tr> <td><b>TP concentration</b><sup><b>c</b></sup></td> <td char="Â±" align="char" colspan="2">23.7Â±2.7</td> <td char="Â±" align="char" colspan="2">29.4Â±4.7</td> </tr> <tr> <td><b>TF concentration</b><sup><b>c</b></sup></td> <td char="Â±" align="char" colspan="2">4.2Â±0.1</td> <td char="Â±" align="char" colspan="2">4.9<sup>*</sup>Â±0.2</td> </tr> <tr> <td><b>Phytotoxic potential [%]</b></td> <td char="Â±" align="char" colspan="2">40.6Â±7.2</td> <td char="Â±" align="char" colspan="2">0.0<sup>d*</sup>Â±5.8</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup> measured using mass spectrometry</p> <p><sup>b</sup> counted as equivalent of Î±-solanine</p> <p><sup>c</sup> colorimetric measurement (Â±SD)</p> <p><sup>d</sup> length of PLE-treated plants the same as in control</p> <p><sup>e</sup>deviated from bulk C (<i>t-student test</i>)</p> </div>  <p /> </div> <div class="profilesofdifferentiallyexpressedgeneandprotein" title="sec"> <div class="title" tagx="title" title="title">
Profiles of differentially expressed gene and protein</div> <p id="Par22">GAs have been recognized as compounds with protective activity against pathogens, pests and herbivores. Recently, we confirmed that GAs present in potato leaf extract (PLE) exhibit PP. PLE of wild species and hybrids inhibited mustard growth, and TGA content was negatively correlated with mustard root and seedling lengthÂ [<a href="#CR16">16</a>]. Based on this finding, we addressed the question of which factors/phenomena may play a significant role in the expression of PP under low TGA (2.7â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup>) content in PLE?Â We analysed the PP of potato in relation to gene and protein expression in potato leaves and GA content in PLE.</p> <p id="Par23">Leaf RNA of bulk samples C and D was analysed using the BGISEQ-500 (BGI, China) platform. The total number of raw reads ranged from 28,723,117 to 28,804,382. A total of 23,836 differentially expressed transcripts were identified between libraries of C and D. We analysed the most differentially regulated transcripts with false discovery rates (FDR-adjusted <i>p</i>-value <div class="â€‰0.05)_UNKNOWN" href="#MOESM3">
3. The top 10 most up- and downregulated transcripts are presented in TableÂ <a href="#Tab2">2</a>. The most upregulated transcripts in C were anthocyanin 5-aromatic acyltransferase-like and subtilisin-like protease SBT1.7, transcript variant X2, with log2 fold changes (FCs) of 9.45 and 9.19, respectively. The most downregulated genes were carbonic anhydrase chloroplastic-like and miraculin-like, with log2 FCs of âˆ’â€‰9.94 andâ€‰âˆ’â€‰9.28, respectively. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></div></p><p>The top 10 most abundant transcripts in down-regulated and up-regulated genes from RNA-seq data, after comparison mRNA samples D vs. C</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Gene name</th> <th>Locus</th> <th>log2 FC<sup><b>a</b></sup></th> <th>FDR<br /><br /><b><i>p</i></b>-value<sup><b>b</b></sup></th> </tr> </thead> <tbody> <tr> <td colspan="4"><b>Up-regulated DEGs</b></td> </tr> <tr> <td>â€ƒAnthocyanin 5-aromatic acyltransferase-like</td> <td>LOC102605147</td> <td>9.45</td> <td>9.20E-14</td> </tr> <tr> <td>â€ƒSubtilisin-like protease SBT1.7, transcript variant X2</td> <td>LOC102596363</td> <td>9.19</td> <td>9.51E-13</td> </tr> <tr> <td>â€ƒPutative uncharacterized protein YER190C-A</td> <td>LOC107057927</td> <td>8.24</td> <td>5.49E-10</td> </tr> <tr> <td>â€ƒProbable disease resistance protein At1g61310, transcript variant X1</td> <td>LOC102600040</td> <td>6.68</td> <td>7.34E-06</td> </tr> <tr> <td>â€ƒ1-Aminocyclopropane-1-carboxylate oxidase 5-like</td> <td>LOC102589195</td> <td>6.67</td> <td>1.66E-06</td> </tr> <tr> <td>â€ƒTrans-resveratrol di-O-methyltransferase-like</td> <td>LOC107057698</td> <td>6.65</td> <td>1.61E-06</td> </tr> <tr> <td>â€ƒUncharacterized LOC102590955</td> <td>LOC102590955</td> <td>6.58</td> <td>1.09E-05</td> </tr> <tr> <td>â€ƒTwo-component response regulator ARR2</td> <td>LOC102596771</td> <td>6.51</td> <td>1.90E-05</td> </tr> <tr> <td>â€ƒUncharacterized LOC107058900</td> <td>LOC107058900</td> <td>6.48</td> <td>1.65E-05</td> </tr> <tr> <td>â€ƒMiraculin-like</td> <td>LOC102589829</td> <td>6.27</td> <td>6.09E-05</td> </tr> <tr> <td colspan="4"><b>Down-regulated DEGs</b></td> </tr> <tr> <td>â€ƒCarbonic anhydrase, chloroplastic-like</td> <td>LOC102589374</td> <td>âˆ’9.94</td> <td>3.32E-20</td> </tr> <tr> <td>â€ƒMiraculin-like</td> <td>LOC107061746</td> <td>âˆ’9.28</td> <td>3.78E-13</td> </tr> <tr> <td>â€ƒCannabidiolic acid synthase-like 2</td> <td>LOC102604287</td> <td>âˆ’9.17</td> <td>1.02E-12</td> </tr> <tr> <td>â€ƒ21â€‰kDa protein-like</td> <td>LOC102592643</td> <td>âˆ’9.00</td> <td>2.55E-12</td> </tr> <tr> <td>â€ƒPutative UPF0481 protein At3g02645</td> <td>LOC102605914</td> <td>âˆ’9.00</td> <td>3.09E-12</td> </tr> <tr> <td>â€ƒPeroxidase 16</td> <td>LOC102581043</td> <td>âˆ’8.40</td> <td>1.77E-10</td> </tr> <tr> <td>â€ƒProtein detoxification 29-like</td> <td>LOC102581213</td> <td>âˆ’8.39</td> <td>8.42E-42</td> </tr> <tr> <td>â€ƒMetallothionein-like protein type 2</td> <td>LOC102589950</td> <td>âˆ’8.32</td> <td>3.98E-10</td> </tr> <tr> <td>â€ƒNon-specific lipid-transfer protein 1-like</td> <td>LOC102599380</td> <td>âˆ’8.22</td> <td>5.72E-10</td> </tr> <tr> <td>â€ƒThreonine dehydratase biosynthetic, chloroplastic-like</td> <td>LOC102583664</td> <td>âˆ’8.21</td> <td>5.59E-10</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup>log2 estimated fold change</p> <p><sup>b</sup>FDR adjusted <i>p</i>-value</p> </div>  <p /> <p id="Par24">Liquid chromatographyâˆ’mass spectrometry analysis of the samples C amd D resulted in 48,836 spectra that were linked to matched peptides, 2125 accessions, and annotated as 3054 proteins. All quantitatively abundant proteins are presented on Fig.Â <a href="#Fig1">1</a>. Thirty-three differentially expressed proteins (DEPs) were found using the established criteria (q-value<div class="â€‰0.05)_UNKNOWN" href="#Tab3">
3). The DEPs were divided into five categories according to the system of [<a href="#CR18">18</a>] and assigned based on the UniProt database: primary metabolism, amino acid metabolism, cell structure, protein turnover and stress and defence. The most abundant group of proteins were those related to stress and defence (12 proteins) with glucan endo-1,3-betaglucosidase acidic isoform, whose expression was 47.96Ã— higher in D than in C. Among the DEPs, 3 were characteristic of C: in the category of primary metabolism, there were formamidase and non-symbiotic haemoglobin 2, and in the category of cell structure, there was tetratricopeptide repeat superfamily protein. For D, 3 DEPs were characteristic: 2 in the category cell structure (protein trichome birefringence-like, lysine histidine transporter 1) and 1 in stress and defence (basic endochitinase). </div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Potato leaf proteins that varied after comparison of bulks D vs. C. The volcanoÂ plot shows quantitative abundanceÂ of proteins. Only the blue and red dots possess the q-value <div class="â€‰0.05._UNKNOWN" href="#Tab2">
2</div></p>   </div> <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> <p>List of 33 differentially expressed proteins after comparison samples D vs. C</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>ID</th> <th>Protein accession numbers<sup><b>a</b></sup></th> <th>Protein</th> <th>q-value</th> <th>Peptides<sup><b>b</b></sup></th> <th>Fold change<br /><br />(D/C)<sup><b>c</b></sup></th> </tr> </thead> <tbody> <tr> <td colspan="6"><b>Primary metabolism</b></td> </tr> <tr> <td>1</td> <td>M1CC20 XP_006339814.1</td> <td>Similar to: formamidase</td> <td>only in C</td> <td>14</td> <td /> </tr> <tr> <td>2</td> <td>M1BEV1 XP_006364443.1</td> <td>Similar to: non-symbiotic hemoglobin 2</td> <td>only in C</td> <td>4</td> <td /> </tr> <tr> <td>3</td> <td>M1AL00 XP_006363768.1</td> <td>Similar to: ferredoxin-dependent glutamate synthase, chloroplastic</td> <td>0.00015</td> <td>457</td> <td>0.81</td> </tr> <tr> <td>4</td> <td>M1CK21, M1CK23 XP_006347295.1</td> <td>Similar to: chloroplast stem-loop binding protein of 41â€‰kDa b, chloroplastic</td> <td>0.00028</td> <td>163</td> <td>0.71</td> </tr> <tr> <td>5</td> <td>M1AZB4 XP_006349675.1</td> <td>Similar to: Chloroplast stem-loop binding protein of 41â€‰kDa a, chloroplastic</td> <td>0.00106</td> <td>152</td> <td>0.72</td> </tr> <tr> <td>6</td> <td>P04045 NP_001275215.1</td> <td>Alpha-1,4 glucan phosphorylase L-1 isozyme, chloroplastic/amyloplastic</td> <td>0.02116</td> <td>93</td> <td>1.50</td> </tr> <tr> <td>7</td> <td>P32811 NP_001275118.1</td> <td>Alpha-glucan phosphorylase, H isozyme</td> <td>0.01682</td> <td>89</td> <td>0.61</td> </tr> <tr> <td>8</td> <td>M0ZYC1</td> <td>Protein disulfide-isomerase</td> <td>0.02141</td> <td>115</td> <td>1.45</td> </tr> <tr> <td>9</td> <td>M1AIV9</td> <td>Pectinesterase</td> <td>0.00106</td> <td>94</td> <td>1.72</td> </tr> <tr> <td colspan="6"><b>Amino acid metabolism</b></td> </tr> <tr> <td>10</td> <td>M1BCZ5, P54260 NP_001275291.1</td> <td>Aminomethyltransferase, mitochondrial</td> <td>0.04385</td> <td>247</td> <td>0.78</td> </tr> <tr> <td>11</td> <td><b>M1AZP7, P31212 XP_006366786.1</b></td> <td><b>Threonine dehydratase biosynthetic (Fragment)</b></td> <td><b>0.00162</b></td> <td><b>153</b></td> <td><b>1.37</b></td> </tr> <tr> <td>12</td> <td>M1BH63</td> <td>Similar to: acetylornithine deacetylase</td> <td>0.00015</td> <td>74</td> <td>2.43</td> </tr> <tr> <td colspan="6"><b>Cell structure</b></td> </tr> <tr> <td>13</td> <td>M1AJ37, M1AJ38 XP_006351160.1</td> <td>Similar to: Tetratricopeptide repeat superfamily protein</td> <td>only in C</td> <td>3</td> <td /> </tr> <tr> <td>14</td> <td>M1AN84 XP_006362437.1, M1BTR0 XP_006348847.1</td> <td>Similar to: Protein trichome birefringence-like</td> <td>only in D</td> <td>6</td> <td /> </tr> <tr> <td>15</td> <td>M1BT25 XP_006338671.1</td> <td>Similar to: lysine histidine transporter 1</td> <td>only in D</td> <td>5</td> <td /> </tr> <tr> <td>16</td> <td>M1A9U0 XP_006357191.1</td> <td>Similar to: 30S ribosomal protein S21</td> <td>0.02471</td> <td>24</td> <td>3.28</td> </tr> <tr> <td colspan="6"><b>Protein turnover</b></td> </tr> <tr> <td>17</td> <td>M1AMY3, M1AMZ0 XP_006353922.1</td> <td>Similar to: Kunitz-type protease inhibitor D</td> <td>0.00015</td> <td>42</td> <td>7.82</td> </tr> <tr> <td>18</td> <td> <p>P58521,</p> <p>P58518,</p> <p>P58520,</p> <p>M1AKE5,</p> <p>P16348,</p> <p>P17979,</p> </td> <td> <p>Aspartic protease inhibitor 9</p> <p>Aspartic protease inhibitor 3 (Fragment)</p> <p>Aspartic protease inhibitor 6 (Fragment)</p> <p>Similar to: Aspartic protease inhibitor 5</p> <p>Aspartic protease inhibitor 11</p> <p>Aspartic protease inhibitor 8</p> </td> <td>0.00161</td> <td>8</td> <td>2.94</td> </tr> <tr> <td>19</td> <td>P37842, M1A5P8, M1A5Q3, M1A5P9</td> <td>Multicystatin, Cysteine proteinase inhibitor</td> <td>0.00015</td> <td>87</td> <td>10.07</td> </tr> <tr> <td>20</td> <td>M1C4F2, M1C4F3 XP_006364268.1</td> <td>Similar to: Aspartyl protease family protein</td> <td>0.00131</td> <td>57</td> <td>2.87</td> </tr> <tr> <td>21</td> <td>P31427 NP_001305566.1</td> <td>Leucine aminopeptidase, chloroplastic</td> <td>0.00015</td> <td>80</td> <td>2.58</td> </tr> <tr> <td colspan="6"><b>Stress and defense</b></td> </tr> <tr> <td>22</td> <td>M1AGK5</td> <td>Similar to: basic endochitinase</td> <td>only in D</td> <td>13</td> <td /> </tr> <tr> <td>23</td> <td>M0ZMG2</td> <td>Similar to: Acidic endochitinase Q</td> <td>0.00015</td> <td>54</td> <td>3.17</td> </tr> <tr> <td>24</td> <td>M1D578</td> <td>Peroxidase</td> <td>0.00132</td> <td>53</td> <td>2.11</td> </tr> <tr> <td>25</td> <td>M1APC7, M1APC8, M1APC9 XP_006365633.2</td> <td>Similar to: Glucan endo-1,3-beta-glucosidase, acidic</td> <td>0.00015</td> <td>104</td> <td>2.42</td> </tr> <tr> <td>26</td> <td>M1APC4</td> <td>Similar to: Glucan endo-1,3-beta-glucosidase, acidic</td> <td>0.00015</td> <td>13</td> <td>47.96</td> </tr> <tr> <td>27</td> <td>M1CX91</td> <td>Similar to: glucan endo-1,3-beta-glucosidase, acidic</td> <td>0.00162</td> <td>50</td> <td>2.32</td> </tr> <tr> <td>28</td> <td>M1APC5</td> <td>Similar to: glucan endo-1,3-beta-glucosidase, acidic</td> <td>0.00527</td> <td>16</td> <td>2.42</td> </tr> <tr> <td>29</td> <td>Q941G6 NP_001275095.1</td> <td>Pathogenesis-related protein 1b</td> <td>0.00233</td> <td>42</td> <td>2.60</td> </tr> <tr> <td>30</td> <td>M1BPP7 AFW90570.1</td> <td>Pathogenesis-related protein P2</td> <td>0.00015</td> <td>36</td> <td>5.70</td> </tr> <tr> <td>31</td> <td>M0ZMA9 XP_006340889.1</td> <td>Similar to: pathogenesis-related protein STH-2</td> <td>0.00015</td> <td>80</td> <td>2.74</td> </tr> <tr> <td>32</td> <td>M1CBM0 XP_006349319.1</td> <td>Similar to: stromal 70â€‰kDa heat shock-related protein, chloroplastic</td> <td>0.00713</td> <td>197</td> <td>0.77</td> </tr> <tr> <td>33</td> <td>P32111, M0ZQ21, M0ZQ26 XP_006367669.1</td> <td>Probable glutathione S-transferase</td> <td>0.04449</td> <td>63</td> <td>1.71</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup> Accession number according to UniProt/NCBI</p> <p><sup>b</sup> Number of peptides matched to predicted protein sequence</p> <p><sup>c</sup> Fold change (D/C) derived by comparison of relative protein intensity between the bulk D and the bulk C, values above fold change &amp;gt;â€‰1 are characteristic for D sample</p> <p>ns - non significant</p> <p>bolded â€" protein that abundance is correlated with its gene expression from RNA-seq experiment</p> </div> </div> <p /> </div> <div class="functionalenrichmentanalysis" title="sec"> <div class="title" tagx="title" title="title">
Functional enrichment analysis</div> <p id="Par25">Based on the GO analysis, three functional groups were categorized: molecular function (MF), biological process (BP) and cellular component (CC). We identified 34 main GO categories, in three levels (16 GO terms in Biological process, 7 GO terms in Molecular function and 11 GO terms in Cellular component) (Fig.Â <a href="#Fig2">2</a>). In the MF group, the most significantly enriched GO terms were catalytic activity, binding followed byÂ transporter activity. For the BP group, metabolic process, cellular process and response to stimulus were the GO terms most significantly enriched. With regard to CC, most enriched GO terms were cell, cell part and organelle. All unprocessed GO terms identified from the comparison of D vs. C are shown in AdditionalÂ FileÂ <a href="#MOESM4">4</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Gene ontology (GO) analysis of potato D vs. C differentially expressed genes. The relative frequencies of GO hits for target genes assigned to the GO functional categories: cellular components, biological processes and molecular functions</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par26">We previously demonstrate significant involvement of GAs in potato phytotoxic interactions. However, we observed that potato clones with low GAs content also possess phytotoxic abilities. We suspected that PP of clones with low GAs content is determined by various GAs composition. In this purpose we derived a diploid potato population with <i>S. chacoense</i> as a paternal donor of PP, to diminish effect of genetic background and selected clones of contrasting PP. <i>S. chacoense</i> is a potato wild relative and is known as a source of various steroidal glycoalkaloids, including leptine glycoalkaloids [<a href="#CR19">19</a>]. Depending on the <i>S. chacoense</i> accession a lot of glycoalkaloids can be found including leptine I and II, leptinine I and II, Î±-solanine, Î±- and Î²-chaconine, commersonine and demissine; other alkaloids as calystegines and clavepictine, or steroidal saponins as saponin, tuberoside, torvoside or graeculin [<a href="#CR20">20</a>â€"<a href="#CR23">23</a>]. In the present study, the low TGA content in C and D accompanied various GA patterns among the samples. Most frequently, the GAs Î±-solasonine and Î±-solamargine together may play a significant role in PP expression in C. Both compounds were previously recognized in ripe fruits of grey bitter-apple (<i>Solanum incanum</i> L.) as an inhibitorsÂ of lettuce (<i>Lactuca sativa</i> L.) root growth with comparable strength at a concentration of 100â€‰mgâ€‰dm<sup>âˆ’â€‰3</sup> [<a href="#CR24">24</a>]. Moreover, they may act synergistically when applied jointly in a 1:1 ratio, as was demonstrated on cucumber (<i>Cucumis sativus</i> L.) radicle growth when these two compounds were applied separately [<a href="#CR25">25</a>].</p> <p id="Par27">The biological activity of GAs was found to be strictly influenced by the configuration and occurrence of sugar moieties [<a href="#CR26">26</a>]. Thus, the composition of GAs may shape the expression and strength of PP. The phytotoxic effect of GAs and their derivatives on cucumber root growth increased with increasing concentration and was dependent on their type [<a href="#CR25">25</a>]. The strongest effect was observed for Î±-chaconine, which was followed in descending order by Î±-solanine, Î±-solamargine, Î±-solasonine, 6-O-sulfated chaconine and 6-O-sulfated solamargine. To our knowledge, there are no data regarding leptine II allelopathic abilities. The occurrence of foliar leptine glycoalkaloids is directly related to the degree of feeding deterrence of Colorado potato beetle (CPB) [<a href="#CR20">20</a>, <a href="#CR27">27</a>]. Leptine II has the monoacetylated (C23) aglycone leptinidine. This chemical modification increases leptine toxicity against CPB and human tumour cell lines; however, its phytotoxicity has not yet been evaluated [<a href="#CR20">20</a>, <a href="#CR28">28</a>].</p> <p id="Par28">Glycoalkaloid biosynthesis can be described as a series of actions of glycoalkyloid metabolism (GAME) genes that promote steroidal alkaloid (aglycone) yieldsÂ [<a href="#CR29">29</a>]. Then, the aglycones undergo glycosylation by different UDP-glycosyltransferases to synthesize various GAs [<a href="#CR30">30</a>]. We found different expression patterns of various UDP-glycosyltransferases (see AdditionalÂ FileÂ <a href="#MOESM5">5</a>), especially UDP-glycosyltransferase 90A1 (log2 FCâ€‰=â€‰-3.01), which confirmed different GA landscapes between the samples and may affect PP of PLE. It was recently demonstrated that the pattern of GAs in progeny derived from crosses with <i>S. chacoense</i> segregates and may be regulated by 38 genes that are located on chromosome VIII and coexpressed with GAME genes [<a href="#CR31">31</a>]. Comprehensive analysis of DEGs, the GO enrichment, pointing significant participation of metabolic processes (BP-30.9%) and catalytic activity (MF-62.2%) as most differentiating the bulks. This may suggest significant involvement of metabolic pathways in regulation of phytotoxic potential.</p> <p id="Par29">We cannot exclude the possibility that PP is influenced by other compounds that act antagonistically or may diminish the GAs effect. As was shown by [<a href="#CR15">15</a>], Î±-solanine applied together with gallic acid and quercetin diminished the inhibitory effect of Î±-solanine applied alone on mustard root growth. In the present study, D had a significantly higher TF content than C, which may influence the strength of the phytotoxic effect; however, this difference in concentration was slight.</p> <p id="Par30">We suggested that PP expression may be directly related to the qualitative and/or quantitative content of each GA recognized in PLE under low TGA content. Since GAs are involved in plant resistance events and the phytotoxicity of PLE, we assumed that PP of C will be accompanied by increased expression of genes and proteins involved in plant resistance to biotic stress. However, comparison of gene and protein expression profiles in D vs. C did not provide definite evidence to support this concept. Patterns of DEGs involved in the defence response, e.g., endopeptidases, were found, but they were shared between samples. Notably, a gene whose expression is increased in C encodes trans-resveratrol di-O-methyltransferase-like (pterostilbene synthase, EC 2.1.1.240), an enzyme responsible for synthesis of stilbenes, a group of phytoalexins. Most DEGs for this enzyme, which were positioned on different loci, were upregulated in C (see Additional File <a href="#MOESM3">3</a>). Transgenic tobacco (<i>Nicotiana tabacum</i> L.) lines overexpressing stilbene synthase exhibited increased synthesis of phytoalexins, which was accompanied by resistance to <i>Botrytis cinerea</i> infection [<a href="#CR32">32</a>].</p> <p id="Par31">The protein profiles of D vs. C do not correspond toÂ gene expression data. The most numerous group of DEPs was involved with stress and defence responses in D. An extremely high fold change was noted for glucan endo-1,3-Î²-glucosidase acidic isoform (47.96). Plant glucan endo-1,3-Î²-glucosidases are enzymes that can participate in resistance against pathogens and degradation of cell wall components (Î²-glucans) during plant development [<a href="#CR33">33</a>, <a href="#CR34">34</a>]. However, their importance in pathogenesis was debated, since antisense transformants of <i>N. sylvestris</i> and tobacco with Î²-1,3-glucanase deficiency could compensate for the activity of this enzyme by producing a functionally equivalent replacement during infection by tobacco mosaic virus [<a href="#CR35">35</a>].</p> <p id="Par32">Protein abundance is controlled by variation at the coding gene itself and by variation mapping to other regions of the genome [<a href="#CR36">36</a>]. Protein synthesis is regulated at many levels, ranging from splicing and mRNA degradation to protein modification to ubiquitination and proteolysis in proteasomes [<a href="#CR37">37</a>]. Therefore, it is not surprising that transcriptome data did not correspond to proteomic profiles (Table <a href="#Tab2">2</a> and Table <a href="#Tab3">3</a>). We found one DEG, threonine dehydratase biosynthetic (FCâ€‰=â€‰1.37), whose protein expression is significantly increased in D. Threonine dehydratase, also called threonine deaminase, is an enzyme for isoleucine biosynthesis [<a href="#CR38">38</a>] and acts together with protease inhibitors in the herbivore gut, decreasing dietary proteins and threonine availability. In <i>Solanum</i> species, the threonine dehydratase gene has a duplicated paralogue that is coexpressed with genes engaged in herbivore resistance. In lupin (<i>Lupinus</i> sp. L.), a high content of threonine was accompanied by an increased level of alkaloids [<a href="#CR39">39</a>]. On the other hand, a higher level of threonine induced the synthesis of alkaloid-pilocarpine in <i>Pilocarpus vahl</i> [<a href="#CR40">40</a>], is a good precursor of pyrrolizidine alkaloids in <i>Senecio douglasii</i> and participates in the direct biosynthesis of strigosine [<a href="#CR41">41</a>]. One of the proteins characterized for only C was formamidase (EC 3.5.1.49), which is engaged in nitrogen metabolism [<a href="#CR42">42</a>]. The gene encoding this enzyme was also recognized as drought-responsive and regulated by abscisic acid (ABA) [<a href="#CR43">43</a>]. Both enzymes may participate indirectly in GA synthesis due to their involvement in the metabolism of nitrogen, which is inserted into the aglycone skeleton.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par33">Complex analysis of GAs, together with trancriptomic and proteomic results provided new insight in understanding the expression of phytotoxic abilities of potato. We demonstrated that PP expression under low TGA content may be connected with the GAs composition in PLE. We pointed potential involvement of metabolic pathways (GO enrichment) in direct regulation of GAs biosynthesis and accumulation (genes of GAs main biosynthesis pathway) or indirect regulation by other factors/phenomena, e.g., nitrogen metabolism, and is a result of plant reactions to biotic and abiotic stresses. Comparison of transcriptomic with proteomic revealed gene and protein of threonine dehydratase bisynthetic as common element bonding these two omics and important in expression of potato phytotoxic abilities.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterial" title="sec"> <div class="title" tagx="title" title="title">
Plant material</div> <p id="Par34">Plant material consisted of the potato diploid population 15â€"1 (F1 progeny, <i>N</i>â€‰=â€‰Â 166) from a cross of <i>Solanum</i> hybrid DG 88â€"89 (seed parent) and a wild species <i>S. chacoense</i> (pollen parent). The maternal clone was generated within the diploid potato breeding program at the Plant Breeding and Acclimatization Instituteâ€"National Research Institute, MÅ‚ochÃ³w, Poland. Paternal specie was obtained from National Centre for Plant Genetic Resources, RadzikÃ³w, Poland (accession POL003:333133). DG 88â€"89 was the multigenerational hybrid originating from crosses of diploid potato clones. In terms of the genomes, the percentage of <i>S. tuberosum</i> in DG 88â€"89 was 78.3% and that of <i>S. chacoense</i> was 15.7%. Maternal clone and paternal species differ with TGA concentration in the PLE and PP. DG 88â€"89 exhibited a low TGA concentration (5.2â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup>) and nondetectable PP, while <i>S. chacoense</i> had a high TGA concentration (55.6â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup>) and high PP (70%) [<a href="#CR15">15</a>]. Three replicates of each of the progeny were grown in a greenhouse from May to October 2016. In full anthesis, leaflets were collected, mixed, portioned into 0.5 and 1â€‰g portions, frozen in liquid nitrogen and stored until use at âˆ’â€‰80â€‰Â°C.</p> </div> <div class="preparationofpotatoleafextract" title="sec"> <div class="title" tagx="title" title="title">
Preparation of potato leaf extract</div> <p id="Par35">PLE was prepared as previously described by [<a href="#CR16">16</a>]. Briefly, 0.5â€‰g of frozen leaves was ground in liquid nitrogen, supplemented with 50â€‰ml of distilled water (1% w/v) and shaken for 24â€‰h on a laboratory shaker. Freshly prepared and filtered extract was used for biochemical and mass spectrometry (MS) analysis.</p> </div> <div class="evaluationoftotalglycoalkaloid,totalphenolicandtotalflavonoidcontentsinpotatoleafextract" title="sec"> <div class="title" tagx="title" title="title">
Evaluation of total glycoalkaloid, total phenolic and total flavonoid contents in potato leaf extract</div> <p id="Par36">The TGA concentration was measured for all 166 individuals in 2016 using the colorimetric method by [<a href="#CR44">44</a>] with modification as described by [<a href="#CR16">16</a>]. Briefly, 1% PLE was concentrated fourfold in a vacuum rotary evaporator (SpeedVac Appligene Refrigerated Aspirator, Germany). TGA was extracted using 10% acetic acid and precipitated with 5â€‰M ammonium hydroxide. Samples were suspended in 100% methanol. The colour reaction was carried out using 98% sulfuric acid and 1% paraformaldehyde.</p> <p id="Par37">Absorbance was measured on a Hitachi U-1900 (Japan) spectrophotometer at a wavelength of 562â€‰nm against a blank sample. The concentration of TGA was expressed in equivalents of Î±-solanine (Sigma-Aldrich, S3757).</p> <p id="Par38">TP content was determined as in [<a href="#CR16">16</a>] with a modification of the method described by [<a href="#CR45">45</a>]. Analysis was performed on twofold-concentrated PLE using freshly prepared tenfold-diluted Folinâ€"Ciocalteu reagent (Sigma, F9252) and 7.5% sodium carbonate (w/v), and the absorbance was measured at 760â€‰nm against a blank sample. The concentration of TP was expressed as equivalents of gallic acid (PhytoLab, 89,198).</p> <p id="Par39">TF was determined as in [<a href="#CR16">16</a>] with a modified method described by [<a href="#CR46">46</a>]. Analysis was performed on twofold-concentrated PLE using 10% aluminium chloride (w/v) and 1â€‰M potassium acetate, and the absorbance was measured at 415â€‰nm against a blank sample. The concentration of TF was expressed as equivalents of quercetin (PhytoLab, No. 89262).</p> <p id="Par40">TGA, TP and TF concentrations in selected genotypes were measured in 2016â€"2018. All measurements were performed in three biological repetitions, and each repetition had two technical replicates.</p> </div> <div class="evaluationofphytotoxicpotentialofpotatoleafextract" title="sec"> <div class="title" tagx="title" title="title">
Evaluation of phytotoxic potential of potato leaf extract</div> <p id="Par41">PP was measured against the test plant - mustard cv. Rota (Vera-Agra Breeding Company, CieszkÃ³w, Poland) in three biological repetitions for all 166Â individuals in 2016 and for the selected plants in 2017â€"2018 as previously described by [<a href="#CR16">16</a>]. Briefly, 15 mustard seeds after radicle protrusion (appx. 3â€‰mm long) were transferred into Petri dishes (square, 12â€‰cm) filled with filter paper and moisture with distilled water or 1% PLE (PLE-treated plants). After 5â€‰days of incubation, the lengths of the control and PLE-treated mustard seedlings were measured. PP was expressed in % as the degree of seedling length inhibition/stimulation in relation to the length of control plants (grown in water) according to the formula </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ Inhibition\ \left(\%\right)=\left(1-\frac{\mathrm{Treated}\ \mathrm{seedling}\ \mathrm{length}\ }{\mathrm{Control}\ \mathrm{seedling}\ \mathrm{length}}\right)x\ 100 $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mtext_UNKNOWN">
Inhibition</div> <div class="mspace_UNKNOWN" /> <div class="mfenced_UNKNOWN"> <div class="mo_UNKNOWN">
%</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Treated seedling length</div> <div class="mspace_UNKNOWN" /> </div> <div class="mtext_UNKNOWN">
Control seedling length</div> </div> </div> </div> <div class="mi_UNKNOWN">
x</div> <div class="mspace_UNKNOWN" /> <div class="mn_UNKNOWN">
100</div></div><div class="graphic" position="anchor" /> </div> </div> <p /> </div> <div class="constructionofbulksamples" title="sec"> <div class="title" tagx="title" title="title">
Construction of bulk samples</div> <p id="Par42">Based on TGA concentration and PP (AdditionalÂ FileÂ <a href="#MOESM6">6</a>), bulk samples C and D were constructed, each with three biological replications. Bulked sample analysis allows for more effective identification of genes underlying a trait. In this approach, contrasting individuals from a segregating population are pooled and then commonly screened to identify specific markers [<a href="#CR47">47</a>]. Sample C exhibited low TGA content (2.7â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup> in PLE) and high PP (40%); D exhibited a low TGA concentration (2.6â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup> in PLE) and nondetectable PP (PLE-treated plants were the same length as the control). In each sample, equal amounts of frozen leaves in liquid nitrogen from three F1 individuals were ground, mixed together and stored at âˆ’â€‰80â€‰Â°C.</p> </div> <div class="rnaisolationandrna-seqanalysis" title="sec"> <div class="title" tagx="title" title="title">
RNA isolation and RNA-seq analysis</div> <p id="Par43">RNA was isolated from samples C and D according to the protocol described in [<a href="#CR48">48</a>] using TRIzol reagent. Briefly, 0.1â€‰g of tissue ground in liquid nitrogen was supplemented with 1â€‰ml of TRIzol reagent. Extraction was performed twice in chloroform. The RNA was precipitated in 0.3â€‰ml of salt solution (0.8â€‰M sodium citrate and 1.2â€‰M sodium chloride) and 0.3â€‰ml of isopropanol and resuspended in sterile water. The quality and quantity of RNA were determined using a NanoDrop spectrophotometer (Thermo Scientific) at 260â€‰nm and 280â€‰nm and on a 2% agarose gel. Next, RNA was treated with DNase I (Thermo Scientific, EN0521) to degrade double-stranded and single-stranded DNA contaminants in RNA samples.</p> <p id="Par44">The mRNA was isolated using the DynabeadsÂ® mRNA Purification Kit for mRNA enrichment (Ambion, 61,006), and a library was prepared using the MGIEasy RNA Directional Library Prep Set (MGI, 1000006386), both according to the manufacturerâ€™s protocols.</p> <p id="Par45">The established cDNA libraries were sequenced on the BGISEQ-500 sequencing platform (BGI Genomics, China) to generate 100-bp paired-end reads. RNA-seq reads were generated by GenomedÂ® (Warsaw, Poland). After filtering of adaptor sequences and low-quality reads, data were obtained for subsequent analysis. Then, the index of the reference genome (<a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000226075.1">https://www.ncbi.nlm.nih.gov/assembly/GCF_000226075.1</a>) was built using Bowtie v2.1.0, and clean reads obtained for samples C and D were aligned to the reference genome using TopHat v2.0.9 (Broad Institute, Boston, MA). Next, HTSeq v0.5.3 was used to count the number of reads mapped to each gene. The DEGs were identified by the DESeq package.</p> </div> <div class="analysisofgeneontologytermenrichment" title="sec"> <div class="title" tagx="title" title="title">
Analysis of gene ontology term enrichment</div> <p id="Par46">To study the biological functions of the DEGs, gene set enrichment with GO terms was performed using the topGO package. To extract the significant GO categories, Fisherâ€™s exact test was performed with the <i>elim</i> algorithm. To prepare circle diagram of all significant GO terms, we used as query for finding the ontology in various functional categories on the basis of GOslim categories as: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \frac{\mathrm{annotations}\ \mathrm{to}\ \mathrm{terms}\ \mathrm{in}\ \mathrm{GOslim}\ \mathrm{category}\ }{\mathrm{total}\ \mathrm{annotations}\ \mathrm{to}\ \mathrm{terms}\ \mathrm{in}\ \mathrm{this}\ \mathrm{ontology}} \times 100. $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
annotations to terms in GOslim category</div> <div class="mspace_UNKNOWN" /> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
total annotations to terms in this ontology</div> <div class="mspace_UNKNOWN" /> </div> </div> <div class="mo_UNKNOWN">
Ã—</div> <div class="mn_UNKNOWN">
100</div> <div class="mo_UNKNOWN">
.</div></div><div class="graphic" position="anchor" /> </div> </div> <p /> </div> <div class="analysisoftheglycoalkaloidsprofileinpotatoleafextract" title="sec"> <div class="title" tagx="title" title="title">
Analysis of the glycoalkaloids profile in potato leaf extract</div> <p id="Par47">The GA fraction was isolated from 1% PLE of bulked samples C and D using a solid-phase extraction method (QuEChERS). First, for each PLE sample, Î±-solamarine (ChemFaces, CFN93102) dissolved in methanol was added as an internal standard to a final concentration of 10â€‰ngâ€‰Î¼l<sup>âˆ’â€‰1</sup> to calculate the percentage of recovery of GA. In the control sample, Î±-solamarine was added to distilled water to the same final concentration. Both types of samples (control and PLE) were passed through sterilizing filters (0.2â€‰Î¼m, Nalgeneâ"¢). To 750â€‰Î¼l of a sample, an equal amount of acetonitrile (ACN) with 1% formic acid was added, and the sample was applied onto the solid phase of QuEChERS (UTC, ECQUCHL12CT) and shaken for 30â€‰s on a vortex mixer. Then, the supernatant obtained after GA isolation was diluted 10-fold with methanol. HPLC-MS analysis was performed on a Dionex 3000 RS-HPLC equipped with a DGP-3600 pump, a WPS-3000 TLS TRS autosampler, a TCC-3000 RS column compartment (Dionex Corporation, USA) and a Bruker micrOTOF-QII mass spectrometer (Bruker Daltonics, Germany). The chromatography column was a 50â€‰Ã—â€‰3.1 (i.d)-millimetre Thermo Scientific Hyperil GOLD with 1,9-Î¼m particles (Part No. 25002â€"052130, Serial No. 0110796A6, Lot No. 10922).</p> <p id="Par48">Chromatographic conditions: For the mobile phase, solvent A was water, and solvent B was ACN. The flow program was as follows: 0â€‰min â€" 5% solvent B; 1.4â€‰min â€" 5% solvent B; 22.9â€‰min â€" 95% solvent B; 24.4â€‰min â€" 95% solvent B; 24.5â€‰min â€" 5% solvent B; 29â€‰min â€" 5% solvent B. The injection sample volume was 1.5â€‰Î¼L. The flow rate was 0.2â€‰mlâ€‰min<sup>âˆ’â€‰1</sup>, and the eluent was monitored by MS. The analysis was performed in negative ESI mode. Scan range: 50â€"1500â€‰m/z, end plate offset: âˆ’â€‰500â€‰V, capillary: 4500â€‰V, nebulizer gas (N2): 1,2â€‰bar, dry gas (N2): 10â€‰L/min, dry temperature: 220â€‰Â°C.</p> <p id="Par49">Qualitative analysis of GA content was performed using the frequency of each compound in the TGA found in the sample. The qualitative GA profile was calculated using the following formula: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ F\left[\%\right]=\frac{A_n}{A_{A\iota \iota}}\times 100\%, $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mfenced_UNKNOWN"> <div class="mo_UNKNOWN">
%</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
AÎ¹Î¹</div> </div> </div> <div class="mo_UNKNOWN">
Ã—</div> <div class="mn_UNKNOWN">
100</div> <div class="mo_UNKNOWN">
%</div> <div class="mo_UNKNOWN">
,</div></div><div class="graphic" position="anchor" /> </div> </div> <p /> <p id="Par50">where</p> <p id="Par51"><i>F</i> â€" Frequency of compounds</p> <p id="Par52"><i>A</i><sub><i>n</i></sub> â€" Area of analysis compound</p> <p id="Par53"><i>A</i><sub><i>All</i></sub> â€" Area of all GAs in the sample.</p> <p id="Par54">Quantitative analysis was performed using the GA standards Î±-solanine (ChemFaces, CFN90560), Î±-chaconine (ChemFaces, CFN00450), Î±-solamargine (ChemFaces, CFN90159), and Î±-solasonine (PhytoLab 83,271). To quantify the compounds, calibration curves for each GA standard were generated over the concentration range of 0.1â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup> to 10â€‰Î¼gâ€‰ml<sup>âˆ’â€‰1</sup>. For leptine II, a curve for Î±-solanine was used due to the lack of a standard. The results are expressed in Î¼g ml<sup>âˆ’â€‰1</sup> and take into account the percent recovery of each compound in relation to the control sample concentration.</p> </div> <div class="proteinextraction" title="sec"> <div class="title" tagx="title" title="title">
Protein extraction</div> <p id="Par55">Proteins were isolated from C and D as described in [<a href="#CR49">49</a>] with a minor modification. Then, 0.1â€‰g of powdered tissue was suspended in 350â€‰Î¼l of the extraction buffer and incubated on ice for 30â€‰min. Then, phenol solution (RotiÂ®-Aqua-Phenol) was added in a 1:1 (v/v) ratio and incubated at room temperature for 10â€‰min. The phenol phase was recovered twice by centrifugation at 4â€‰Â°C, transferred to new tubes with extraction buffer 1:1 (v/v) and precipitated in cold methanol containing 0.1â€‰M ammonium acetate 1:4 (v/v). The mixture was incubated overnight at âˆ’â€‰20â€‰Â°C and centrifuged at room temperature. The liquid phase was removed, and the pellet was washed once with 100% methanol pre-chilled to âˆ’â€‰20â€‰Â°C, centrifuged with 80% acetone, and centrifuged at the highest speed. The final protein pellet was air-dried and dissolved in 200â€‰Î¼l of 25â€‰mM ammonium bicarbonate. The sample protein content was determined according to the method described by [<a href="#CR50">50</a>] using the bicinchoninic acid assay and bovine serum albumin as a standard. Five independent biological replicates were analysed in this study. A total of 120â€‰Î¼g of protein from each probe was sent to the Mass Spectrometry Laboratory at the Institute of Biochemistry and Biophysics, Polish Academy of Sciences (Warsaw, Poland), for nano-LC-MS-MS/MS (nanoliquid chromatography coupled to tandem mass spectrometry) analysis.</p> </div> <div class="comparativeanalysisofdifferentiallyexpressedproteins" title="sec"> <div class="title" tagx="title" title="title">
Comparative analysis of differentially expressed proteins</div> <p id="Par56">Peptide mixtures were analysed by nano-LC-MS-MS/MS using a nano-Acquity (Waters) LC system and a Q-Exactive mass spectrometer (Thermo Electron Corp., San Jose, CA) with the same equipment, buffers and parameters like in [<a href="#CR51">51</a>]. The raw data were processed by Mascot Distiller followed by Mascot Search (Matrix Science, London, UK, on-site licence) against the UniProt <i>Solanum tuberosum</i> database (February 2018 release). The search parameters for precursor and product ion mass tolerances were 30â€‰ppm and 0.1â€‰Da, respectively enzyme specificity: trypsin and missed cleavage sites. Peptides with Mascot scores exceeding the threshold value corresponding to <div class="â€‰5%_UNKNOWN" href="#CR52">
52]. Shortly, the mass calibration and data filtering described above were carried out like in [<a href="#CR51">51</a>]. At the end, lists of identified peptides with corresponding abundances were exported for statistical analysis carried out with in-house developed Diffprot Software (version 1.5.19; 3.01.2013) [<a href="#CR53">53</a>]. Prior to analysis, abundances were normalized with LOWESS. Proteins with more than 90% common peptides were clustered, and only peptides unique for the cluster were used for statistical analysis. Only proteins with q-values â‰¤0.05 were considered differentially expressed.</div></p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2021_2825_MOESM1_ESM.docx">LINK</a> <p><b>Additional file 1: Supplementary Table S1.</b> Average retention time (RT) and mass of compounds found in samples D and C.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2021_2825_MOESM2_ESM.pdf">LINK</a> <p><b>Additional file 2: Supplementary Figure S1.</b> Mass spectra of GAs found in samples C and D.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2021_2825_MOESM3_ESM.xlsx">LINK</a> <p><b>Additional file 3: Supplementary Table S2.</b> All DEGs recognized after comparison D vs. C bulks, sorted by log2 FC.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2021_2825_MOESM4_ESM.xlsx">LINK</a> <p><b>Additional file 4: Supplementary Table S3.</b> GO enrichement of DEGs in molecular function, biological processes and cellular compartments after comparison D vs C.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2021_2825_MOESM5_ESM.docx">LINK</a> <p><b>Additional file 5: Supplementary Table S4.</b> Expression pattern of UDP-glucosytransferases and other genes engaged in GAs biosynthesis between the bulks D vs. C, sorted by Log2 FC.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2021_2825_MOESM6_ESM.xlsx">LINK</a> <p><b>Additional file 6: Supplementary Table S5.</b> TGA content and PP in individuals of 15-1 population.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
CPB</div> <div class="def" title="def"> <p id="Par4">Colorato Potato Beetle</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DEGs</div> <div class="def" title="def"> <p id="Par5">differentially expressed genes</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DEPs</div> <div class="def" title="def"> <p id="Par6">differentially expressed proteins</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GAME</div> <div class="def" title="def"> <p id="Par7">glycoalkaloids metabolism genes</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GO</div> <div class="def" title="def"> <p id="Par8">gene ontology</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PLE</div> <div class="def" title="def"> <p id="Par9">potato leaf extract</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PP</div> <div class="def" title="def"> <p id="Par10">phytotoxic potential</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GAs</div> <div class="def" title="def"> <p id="Par11">glycoalkaloids (measured by mass spectrometry)</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TF</div> <div class="def" title="def"> <p id="Par12">total flavonoids</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TGA</div> <div class="def" title="def"> <p id="Par13">total glycoalkaloids (measured colorimetrically)</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TP</div> <div class="def" title="def"> <p id="Par14">total phenolics</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Authors thank to Konrad Kowalski from Masdiag laboratory (Warsaw, Poland) for useful directions and technical support in GA recognition and determination.</p> </div> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>KS performed most of the experiments, JC performed quantitative analysis of GA, IW-S provide and collected plant material, WM contributed useful suggestions and wrote part of the manuscript, DSK designed and conducted experiments, wrote the manuscript. All authors have read and approved the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>The research was financially supported by grants from The National Science Center in Poland, (UMO-2016/23/D/NZ9/02672). The funding organizations provided the financial support to the research projects, but were not involved in the design of the study, data collection, analysis of the data, or the writing of the manuscript.</p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>All data analysed in this study are included in this published article and its supplementary files. All raw and processed RNA-seq data have been deposited in the Gene Expression Omnibus [GEO] repository under the link [<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE155583">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE155583</a>] and Sequence Read Archive [SRA] repository under the link [<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA650400">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA650400</a>].</p> <p>The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par57">Not applicable.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p id="Par58">Not applicable.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par59">The authors declare that they have no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macias</span><span class="given-names" tagx="given-names" title="given-names">FA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marin</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliveros-Bastidas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Varela</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonet</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrera</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Allelopathy as a new strategy for sustainable ecosystems development</span><span class="source" tagx="source" title="source">Biol Sci Space</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">18</span><span class="lpage" tagx="lpage" title="lpage">23</span><span class="pub-id"><a href="https://dx.doi.org/10.2187/bss.17.18">10.2187/bss.17.18</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12897457">12897457</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Trezzi MM, Vidal RA, Balbinot AA, Bittencourt H von H, Souza Filho APS. Allelopathy: driving mechanisms governing its activity in agriculture. J Plant Int 2016;11:53â€"60.</span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gniazdowska</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bogatek</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Allelopathic interactions between plants. Multi site action of allelochemicals</span><span class="source" tagx="source" title="source">Acta Physiol Plant</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">395</span><span class="lpage" tagx="lpage" title="lpage">407</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11738-005-0017-3">10.1007/s11738-005-0017-3</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Findura</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hara</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szparaga</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kocira</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">CzerwiÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BartoÅ¡</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of the effects of allelopathic aqueous plant extracts, as potential preparations for seed dressing, on the modulation of cauliflower seed germination</span><span class="source" tagx="source" title="source">Agriculture.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">122</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/agriculture10040122">10.3390/agriculture10040122</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">BÃ¶ttger A, Vothknecht U, Bolle C, Wolf A. Plant secondary metabolites and their general function in plants. In: Lessons on caffeine, cannabis &amp;amp; co. Learning Materials Biosci. (eds). Cham: Springer; 2018. p. 3â€"17. <a href="http://doiorg-443.webvpn.fjmu.edu.cn/10.1007/978-3-319-99546-5_1">http://doiorg-443.webvpn.fjmu.edu.cn/10.1007/978-3-319-99546-5_1</a>.</span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaynab</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fatimab</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abbasc</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharifd</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Umairc</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zafare</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Role of secondary metabolites in plant defense against pathogens</span><span class="source" tagx="source" title="source">Microb Pathog</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">124</span><span class="fpage" tagx="fpage" title="fpage">198</span><span class="lpage" tagx="lpage" title="lpage">202</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.micpath.2018.08.034">10.1016/j.micpath.2018.08.034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30145251">30145251</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gebhardt</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">The historical role of species from the Solanaceae plant family in genetic research</span><span class="source" tagx="source" title="source">Theor Appl Genet</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">2281</span><span class="lpage" tagx="lpage" title="lpage">2294</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00122-016-2804-1">10.1007/s00122-016-2804-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27744490">27744490</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">CastaÃ±eda-Ã�lvarez</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Haan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JuÃ¡rez</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khoury</span><span class="given-names" tagx="given-names" title="given-names">CK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Achicanoy</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sosa</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ex situ conservation priorities for the wild relatives of potato (<i>Solanum</i> L. section Petota)</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e0122599</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0122599">10.1371/journal.pone.0122599</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25923711">25923711</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McDonald</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Potato glycoalkaloids: chemistry, analysis, safety, and plant physiology</span><span class="source" tagx="source" title="source">Crit Rev Plant Sci</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">55</span><span class="lpage" tagx="lpage" title="lpage">132</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/07352689709701946">10.1080/07352689709701946</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morillo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rojas</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lequart</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamarti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Natural and synthetic derivatives of the steroidal glycoalkaloids of Solanum genus and biological activity</span><span class="source" tagx="source" title="source">Nat Prod Chem Res</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">371</span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shakya</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Navarre</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span></span><span class="mixed-article-title" title="mixed-article-title">LC-MS analysis of solanidane glycoalkaloid diversity among tubers of four wild potato species and three cultivars (<i>Solanum tuberosum</i>)</span><span class="source" tagx="source" title="source">J Agric Food Chem</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">6949</span><span class="lpage" tagx="lpage" title="lpage">6958</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/jf8006618">10.1021/jf8006618</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18616273">18616273</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roddick</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Potato Glycoalkaloids: some unanswered questions</span><span class="source" tagx="source" title="source">Trends Food Sci Technol</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">126</span><span class="lpage" tagx="lpage" title="lpage">131</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0924-2244(96)10013-3">10.1016/0924-2244(96)10013-3</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siddique</span><span class="given-names" tagx="given-names" title="given-names">MAB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunton</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">Food Glycoalkaloids: distribution, structure, cytotoxicity, extraction, and biological activity</span><span class="source" tagx="source" title="source">Alkaloids - their importance in nature and human life</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">13</span><span class="lpage" tagx="lpage" title="lpage">15</span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeandet</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ClÃ©ment</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Courot</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cordelier</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modulation of phytoalexin biosynthesis in engineered plants for disease resistance</span><span class="source" tagx="source" title="source">J Mol Sci</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">14136</span><span class="lpage" tagx="lpage" title="lpage">14170</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms140714136">10.3390/ijms140714136</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fewell</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roddick</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Interactive antifungal activity of the glycoalkaloids Î±-solanine and Î±-chaconine</span><span class="source" tagx="source" title="source">Phytochem.</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">323</span><span class="lpage" tagx="lpage" title="lpage">328</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0031-9422(93)85511-O">10.1016/0031-9422(93)85511-O</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SoÅ‚tys-Kalina</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murawska</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strzelczyk-Å»yta</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wasilewicz-Flis</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marczewski</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phytotoxic potential of cultivated and wild potato species (<i>Solanum</i> sp.): role of glycoalkaloids, phenolics and flavonoids in phytotoxicity against mustard (<i>Sinapis alba</i> L.)</span><span class="source" tagx="source" title="source">Acta Physiol Plant.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">55</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11738-019-2848-3">10.1007/s11738-019-2848-3</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukushima</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kusano</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Redestig</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arita</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saito</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Integrated omics approaches in plant systems biology</span><span class="source" tagx="source" title="source">Curr Op Chem Biol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">532</span><span class="lpage" tagx="lpage" title="lpage">538</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cbpa.2009.09.022">10.1016/j.cbpa.2009.09.022</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alexander</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cilia</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular tug-of-war: global plant proteome changes during viral infection</span><span class="source" tagx="source" title="source">Curr Plant Biol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">13</span><span class="lpage" tagx="lpage" title="lpage">24</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cpb.2015.10.003">10.1016/j.cpb.2015.10.003</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veilleux</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hybrid breakdown in the F1 between <i>Solanum chacoense</i> and <i>S. phureja</i> and gene transfer for leptine biosynthesis</span><span class="source" tagx="source" title="source">J Am Soc Hortic Sci</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">123</span><span class="fpage" tagx="fpage" title="fpage">854</span><span class="lpage" tagx="lpage" title="lpage">858</span><span class="pub-id"><a href="https://dx.doi.org/10.21273/JASHS.123.5.854">10.21273/JASHS.123.5.854</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinden</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lind</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deahl</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Segregation of leptine glycoalkaloids in <i>Solanum chacoense</i> bitter</span><span class="source" tagx="source" title="source">J Agric Food Chem</span><span class="year" tagx="year" title="year">1986</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">372</span><span class="lpage" tagx="lpage" title="lpage">377</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/jf00068a056">10.1021/jf00068a056</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deahl</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinden</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of wild tuber-bearing <i>Solanum</i> accessions for foliar glycoalkaloid level and composition</span><span class="source" tagx="source" title="source">Am Pot J</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">61</span><span class="lpage" tagx="lpage" title="lpage">69</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02848648">10.1007/BF02848648</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mweetwa</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunter</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poe</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harich</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ginzberg</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veilleux</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tokuhisa</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Steroidal glycoalkaloids in <i>Solanum chacoense</i></span><span class="source" tagx="source" title="source">Phytochemistry.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">32</span><span class="lpage" tagx="lpage" title="lpage">40</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phytochem.2011.12.003">10.1016/j.phytochem.2011.12.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22217745">22217745</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Joshi JR, Yao L, Charkowski AO, Heuberger AL. Metabolites from wild potato inhibit virulence factors of the soft rot and blackleg pathogen Pectobacterium brasiliense. MPMI. 2020; 10.1094/MPMI-08-20-0224-R.</span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukuhara</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kubo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation of steroidal glycoalkaloids from <i>Solanum incanum</i> by two countercurrent chromatographic methods</span><span class="source" tagx="source" title="source">Phytochem.</span><span class="year" tagx="year" title="year">1991</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">485</span><span class="lpage" tagx="lpage" title="lpage">687</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0031-9422(91)83753-8">10.1016/0031-9422(91)83753-8</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ni</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tai</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of glycoalkaloids from Solanum plants on cucumber root growth</span><span class="source" tagx="source" title="source">Phytochem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">1534</span><span class="lpage" tagx="lpage" title="lpage">1538</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phytochem.2010.06.002">10.1016/j.phytochem.2010.06.002</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ChowaÅ"ski</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adamski</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marciniak</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RosiÅ"ski</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ¼yÃ¼kgÃ¼zelet</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ¼yÃ¼kgÃ¼zel</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A review of bioinsecticidal activity of Solanaceae alkaloids</span><span class="source" tagx="source" title="source">Toxins (Basel).</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">60</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/toxins8030060">10.3390/toxins8030060</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4810205">4810205</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yencho</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalski</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kennedy</span><span class="given-names" tagx="given-names" title="given-names">GG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanford</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Segregation of leptine glycoalkaloids and resistance to Colorado potato beetle (<i>Leptinotarsa decemlineata</i> (Say)) in F2 <i>Solanum tuberosum</i> (4x) Ã—<i>S. chacoense</i> (4x) potato progenies</span><span class="source" tagx="source" title="source">Am J Pot Res.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">167</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02853941">10.1007/BF02853941</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Schreiber K. Steroid alkaloids, the Solanum group. In: The Alkaloids. Vol. X, Manske RHF, Holmes HL. (Eds). New York: AcademicPress; 1968. p. 1â€"192.</span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Itkin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heinig</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tzfadia</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhide</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shinde</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardenas</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biosynthesis of antinutritional alkaloids in solanaceous crops is mediated by clustered genes</span><span class="source" tagx="source" title="source">Science.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">341</span><span class="fpage" tagx="fpage" title="fpage">175</span><span class="lpage" tagx="lpage" title="lpage">179</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1240230">10.1126/science.1240230</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23788733">23788733</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakayasu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Umemoto</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohyama</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujimoto</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A dioxygenase catalyzes steroid 16Î±-hydroxylation in steroidal glycoalkaloid biosynthesis</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">175</span><span class="fpage" tagx="fpage" title="fpage">120</span><span class="lpage" tagx="lpage" title="lpage">133</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.17.00501">10.1104/pp.17.00501</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28754839">28754839</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C-H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">S-W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inheritance of steroidal glycoalkaloids in potato tuber flesh</span><span class="source" tagx="source" title="source">J Integr Agric</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">2255</span><span class="lpage" tagx="lpage" title="lpage">2263</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S2095-3119(19)62718-8">10.1016/S2095-3119(19)62718-8</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hain</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reif</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langebartels</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kindl</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vornam</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Disease resistance results from foreign phytoalexin expression in a novel plant</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">361</span><span class="fpage" tagx="fpage" title="fpage">153</span><span class="lpage" tagx="lpage" title="lpage">156</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/361153a0">10.1038/361153a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8421520">8421520</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biochemical mechanisms of disease resistance</span><span class="source" tagx="source" title="source">Annu Rev Plant Physiol</span><span class="year" tagx="year" title="year">1981</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">21</span><span class="lpage" tagx="lpage" title="lpage">81</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.pp.32.060181.000321">10.1146/annurev.pp.32.060181.000321</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morohashi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsushima</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Development of Î²-1,3-glucanase activity in germinated tomato seeds</span><span class="source" tagx="source" title="source">J Exp Bot</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">1381</span><span class="lpage" tagx="lpage" title="lpage">1387</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jexbot/51.349.1381">10.1093/jexbot/51.349.1381</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10944151">10944151</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beffa</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neuhaus</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meins</span><span class="given-names" tagx="given-names" title="given-names">F</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span></span><span class="mixed-article-title" title="mixed-article-title">Physiological compensation in antisense transformants: specific induction of an "ersatz" glucan endo-1,3-fB-glucosidase in plants infected with necrotizing viruses</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">8792</span><span class="lpage" tagx="lpage" title="lpage">8796</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.90.19.8792">10.1073/pnas.90.19.8792</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foss</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radulovic</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaffer</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruderfer</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedalov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodlett</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic basis of proteome variation in yeast</span><span class="source" tagx="source" title="source">Nat Genet</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">1369</span><span class="lpage" tagx="lpage" title="lpage">1375</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.2007.22">10.1038/ng.2007.22</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17952072">17952072</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vogel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marcotte</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Insights into the regulation of protein abundance from proteomic and transcriptomic analyses</span><span class="source" tagx="source" title="source">Nat Rev Genet</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">227</span><span class="lpage" tagx="lpage" title="lpage">232</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrg3185">10.1038/nrg3185</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22411467">22411467</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzales-Vigila</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bianchetti</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">GN</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howea</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Adaptive evolution of threonine deaminase in plant defense against insect herbivores</span><span class="source" tagx="source" title="source">PNAS.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">5897</span><span class="lpage" tagx="lpage" title="lpage">5902</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1016157108">10.1073/pnas.1016157108</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21436043">21436043</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wasilewko</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buraczewska</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemical composition including content of amino acids, minerals and alkaloids in seeds of three lupin species cultivated in Poland</span><span class="source" tagx="source" title="source">J Anim Feed Sci</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="https://dx.doi.org/10.22358/jafs/68803/1999">10.22358/jafs/68803/1999</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sawaya</span><span class="given-names" tagx="given-names" title="given-names">ACHF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abreu</span><span class="given-names" tagx="given-names" title="given-names">IN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andreazza</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eberlin</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazzafera</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pilocarpine and related alkaloids in Pilocarpus Vahl (<i>Rutaceae</i>)</span><span class="source" tagx="source" title="source">Alkaloids: Properties, applications and pharmacological effects</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">63</span><span class="lpage" tagx="lpage" title="lpage">80</span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Some miscellaneous alkaloids</span><span class="source" tagx="source" title="source">The biochemistry of alkaloids</span><span class="year" tagx="year" title="year">1981</span><span class="fpage" tagx="fpage" title="fpage">145</span><span class="lpage" tagx="lpage" title="lpage">147</span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hynes</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">The formamidase gene of <i>Aspergillus nidulans</i>: regulation by nitrogen metabolite repression and transcriptional interference by an overlapping upstream gene</span><span class="source" tagx="source" title="source">Genetics.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">157</span><span class="fpage" tagx="fpage" title="fpage">119</span><span class="lpage" tagx="lpage" title="lpage">131</span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiren</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abrams</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cutler</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">The relationship of drought-related gene expression in Arabidopsis thaliana to hormonal and environmental factors</span><span class="source" tagx="source" title="source">J Exp Bot</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">2991</span><span class="lpage" tagx="lpage" title="lpage">3007</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/ern155">10.1093/jxb/ern155</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18552355">18552355</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andreu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliva</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Distel</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daleo</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Production of phytoalexins, glycoalkaloids and phenolics in leaves and tubers of potato cultivars with different degrees of field resistance after infection with <i>Phytophthora infestans</i></span><span class="source" tagx="source" title="source">Am J Potato Res</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02360281">10.1007/BF02360281</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maurya</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Kr., quantitative analysis of total phenolic content in Adhatoda vasica Nees extracts. Int J pharm tech</span><span class="source" tagx="source" title="source">Res</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">2403</span><span class="lpage" tagx="lpage" title="lpage">2406</span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chern</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Estimation of total flavonoid content in propolis by two complementary colorimetric methods</span><span class="source" tagx="source" title="source">J Food Drug Anal</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">178</span><span class="lpage" tagx="lpage" title="lpage">182</span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bulked sample analysis in genetics, genomics and crop improvement</span><span class="source" tagx="source" title="source">Plant Biotech J</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">1941</span><span class="lpage" tagx="lpage" title="lpage">1955</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/pbi.12559">10.1111/pbi.12559</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ChomczyÅ"ski</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sacchi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction</span><span class="source" tagx="source" title="source">Anal Biochem</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">162</span><span class="fpage" tagx="fpage" title="fpage">156</span><span class="lpage" tagx="lpage" title="lpage">159</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0003-2697(87)90021-2">10.1016/0003-2697(87)90021-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2440339">2440339</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szajko</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strzelczyk-Å»yta</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marczewski</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of leaf proteomes of potato (<i>Solanum tuberosum</i> L.) genotypes with ER- and HR-mediated resistance to PVY infection</span><span class="source" tagx="source" title="source">Eur J Plant Pathol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">150</span><span class="fpage" tagx="fpage" title="fpage">375</span><span class="lpage" tagx="lpage" title="lpage">385</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10658-017-1284-8">10.1007/s10658-017-1284-8</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krohn</span><span class="given-names" tagx="given-names" title="given-names">RI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hermanson</span><span class="given-names" tagx="given-names" title="given-names">GT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallia</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gartner</span><span class="given-names" tagx="given-names" title="given-names">FH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Provenzano</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Measurement of protein using bicinchoninic acid</span><span class="source" tagx="source" title="source">Anal Biochem</span><span class="year" tagx="year" title="year">1985</span><span class="volume" tagx="volume" title="volume">150</span><span class="fpage" tagx="fpage" title="fpage">76</span><span class="lpage" tagx="lpage" title="lpage">78</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0003-2697(85)90442-7">10.1016/0003-2697(85)90442-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3843705">3843705</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lebecka</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kistowski</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debski</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szajko</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murawska</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marczewski</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Quantitative proteomic analysis of differentially expressed proteins in tubers of potato plants differing in resistance to <i>Dickeya solani</i></span><span class="source" tagx="source" title="source">Plant Soil</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">441</span><span class="fpage" tagx="fpage" title="fpage">317</span><span class="lpage" tagx="lpage" title="lpage">329</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11104-019-04125-7">10.1007/s11104-019-04125-7</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakun</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niemczyk</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domanski</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jazwiec</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perzanowska</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niemczyk</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Urine proteome of autosomal dominant polycystic kidney disease patients</span><span class="source" tagx="source" title="source">Clin Proteomics</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">13</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1559-0275-9-13">10.1186/1559-0275-9-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23228063">23228063</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malinowska</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kistowski</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakun</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rubel</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tkaczyk</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mierzejewska</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diffprot â€" software for non-parametric statistical analysis of differential proteomics data</span><span class="source" tagx="source" title="source">J Proteome</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">4062</span><span class="lpage" tagx="lpage" title="lpage">4073</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jprot.2012.05.030">10.1016/j.jprot.2012.05.030</a></span></span></li> </ul> </div> </div>  </body></html>