<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">BMC Plant Biol</span><span class="iso-abbrev" title="iso-abbrev">BMC Plant Biol</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">BMC Plant Biology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1471-2229</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7724791">7724791</a></span><span class="publisher-id" title="publisher-id">2745</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12870-020-02745-1">10.1186/s12870-020-02745-1</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Under fire-simultaneous volatilome and transcriptome analysis unravels fine-scale responses of tansy chemotypes to dual herbivore attack</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clancy</span><span class="given-names" tagx="given-names" title="given-names">Mary V.</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haberer</span><span class="given-names" tagx="given-names" title="given-names">Georg</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jud</span><span class="given-names" tagx="given-names" title="given-names">Werner</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niederbacher</span><span class="given-names" tagx="given-names" title="given-names">Bishu</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niederbacher</span><span class="given-names" tagx="given-names" title="given-names">Simon</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senft</span><span class="given-names" tagx="given-names" title="given-names">Matthias</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zytynska</span><span class="given-names" tagx="given-names" title="given-names">Sharon E.</span></span><a href="#Aff3">3</a><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weisser</span><span class="given-names" tagx="given-names" title="given-names">Wolfgang W.</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9825-867X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnitzler</span><span class="given-names" tagx="given-names" title="given-names">JÃ¶rg-Peter</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">jp.schnitzler@helmholtz-muenchen.de</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.4567.0</span><span class="institution-id" title="institution-id">0000 0004 0483 2525</span><span class="institution" title="institution">Helmholtz Zentrum MÃ¼nchen, Research Unit Environmental Simulation (EUS), </span><span class="institution" title="institution">Institute of Biochemical Plant Pathology, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.10711.36</span><span class="institution-id" title="institution-id">0000 0001 2297 7718</span><span class="institution" title="institution">Fundamental and Applied Research in Chemical Ecology (FARCE Lab), Institute of Biology, </span><span class="institution" title="institution">University of NeuchÃ¢tel, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.4567.0</span><span class="institution-id" title="institution-id">0000 0004 0483 2525</span><span class="institution" title="institution">Helmholtz Zentrum MÃ¼nchen, Plant Genome and Systems Biology, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.6936.a</span><span class="institution-id" title="institution-id">0000000123222966</span><span class="institution" title="institution">Terrestrial Ecology Research Group, Department of Ecology and Ecosystem Management, </span><span class="institution" title="institution">Technical University of Munich, School of Life Sciences Weihenstephan, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.10025.36</span><span class="institution-id" title="institution-id">0000 0004 1936 8470</span><span class="institution" title="institution">Department of Ecology, </span><span class="institution" title="institution">University of Liverpool, Evolution and Behaviour, Institute of Infection, Veterinary and Ecological Sciences, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-12-12</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-12-12</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">20</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">551</span><span class="history" title="history"><span class="received" title="received">received: 2020-7-21</span><span class="accepted" title="accepted">accepted: 2020-11-17</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1">Tansy plants (<i>Tanacetum vulgare</i> L.) are known for their high intraspecific chemical variation, especially of volatile organic compounds (VOC) from the terpenoid compound group. These VOCs are closely involved in plant-insect interactions and, when profiled, can be used to classify plants into groups known as chemotypes. Tansy chemotypes have been shown to influence plant-aphid interactions, however, to date no information is available on the response of different tansy chemotypes to simultaneous herbivory by more than one insect species.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par2">Using a multi-cuvette system, we investigated the responses of five tansy chemotypes to feeding by sucking and/or chewing herbivores (aphids and caterpillars; <i>Metopeurum fuscoviride</i> Stroyan and <i>Spodoptera littoralis</i> Boisduval). Herbivory by caterpillars following aphid infestation led to a plant chemotype-specific change in the patterns of terpenoids stored in trichome hairs and in VOC emissions. The transcriptomic analysis of a plant chemotype represents the first de novo assembly of a transcriptome in tansy and demonstrates priming effects of aphids on a subsequent herbivory. Overall, we show that the five chemotypes do not react in the same way to the two herbivores. As expected, we found that caterpillar feeding increased VOC emissions, however, a priori aphid infestation only led to a further increase in VOC emissions for some chemotypes.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par3">We were able to show that different chemotypes respond to the double herbivore attack in different ways, and that pre-treatment with aphids had a priming effect on plants when they were subsequently exposed to a chewing herbivore. If neighbouring chemotypes in a field population react differently to herbivory/dual herbivory, this could possibly have effects from the individual level to the group level. Individuals of some chemotypes may respond more efficiently to herbivory stress than others, and in a group environment these â€œlouderâ€� chemotypes may affect the local insect community, including the natural enemies of herbivores, and other neighbouring plants.</p> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>The online version contains supplementary material available at 10.1186/s12870-020-02745-1.</p> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001659</span><span class="institution" title="institution">Deutsche Forschungsgemeinschaft</span></div> </div><span class="award-id" tagx="award-id" title="award-id">SCHN 653/7-1</span><span class="award-id" tagx="award-id" title="award-id">WE 3080/25-1</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weisser</span><span class="given-names" tagx="given-names" title="given-names">Wolfgang W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnitzler</span><span class="given-names" tagx="given-names" title="given-names">JÃ¶rg-Peter</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par12">Nature is formed of complex communities in which plants play a central role. Interactions between plants and plant antagonists such as herbivores or pathogens result in ever-evolving defence mechanisms and corresponding efforts to overcome them [<a href="#CR1">1</a>]. Plants are sessile organisms, and as such have developed an arsenal of chemical defences in lieu of being able to flee. These plant secondary metabolites consist of diverse groups of compounds that vary considerably across species and families [<a href="#CR2">2</a>]. Different defence compounds challenge plant antagonists in different ways: direct defences include compounds that are actively detrimental to herbivores (for example digestibility reducers and toxins [<a href="#CR3">3</a>]), whereas indirect defences can function as a recruitment drive for attracting herbivore enemies [<a href="#CR4">4</a>]. One important group of compounds involved in indirect plant defence are volatile organic compounds (VOCs). Emission of VOCs by plants is often induced by herbivore attack on plants, but emissions can also be constitutive. VOCs are chemically highly diverse, also within plant species, and this can result in large variation in chemical profiles (i.e. chemotypes) both between and within plant populations [<a href="#CR5">5</a>, <a href="#CR6">6</a>]. Terpenoids, in particular mono- and sesquiterpenoids, are the largest and most structurally diverse class of VOCs found in the plant kingdom. Terpenoids are naturally occurring organic compounds derived from terpenes. Most terpenoid compounds are multicyclic structures with oxygen-containing functional groups. Although they are sometimes referred to as â€œterpenesâ€�, terpenoids contain additional functional groups, mostly containing oxygen [<a href="#CR7">7</a>]. The terpene synthase (TPS) gene family is responsible for this massive diversity of compounds, as a single TPS enzyme can catalyse up to 10 different structures from a single substrate [<a href="#CR8">8</a>, <a href="#CR9">9</a>]. VOCs can be emitted immediately following damage caused by a herbivore, or can be induced and emitted several hours later [<a href="#CR10">10</a>, <a href="#CR11">11</a>]. Traditional methods for assessing volatile emissions in plant-herbivore interactions continue to be improved upon, with more sensitive instruments coming to market [<a href="#CR12">12</a>, <a href="#CR13">13</a>].</p> <p id="Par13">Many studies have investigated the effect of attack of single species of antagonist on plant volatile emissions [<a href="#CR14">14</a>] and have found that the response of plants depends on the type of damage that is inflicted. For instance, chewing herbivores such as caterpillars generally induce the jasmonic acid (JA) pathway (which works in synergy with the volatile phytohormone ethylene), whereas sucking herbivores such as aphids typically induce the salicylic acid (SA) pathway, which is most commonly associated with biotrophic pathogens [<a href="#CR15">15</a>]. Chewing herbivores can inflict significant damage to plant tissues, leading to large changes in metabolism [<a href="#CR16">16</a>]. The level of damage caused to a plant by a chewing herbivore is dependent on several factors, including larval stage, specialism/ generalism of plant-herbivore interaction, infestation threshold, and species [<a href="#CR17">17</a>]. In contrast, sucking herbivores such as aphids, cause a less obvious form of damage, as they suck on plant sap and deplete the plant of nutrients rather than destroying leaf tissue. However, wide-ranging induction of defence pathways in local and systemic leaves is also observed [<a href="#CR18">18</a>] for both chewing and sucking herbivores. Interestingly, interactions between the SA and JA pathways are usually antagonistic, although additive or synergistic responses have been reported as well [<a href="#CR19">19</a>]. These phytohormone-mediated defence pathways result in comprehensive changes to the metabolome, transcriptome, and proteome. In nature, plants are often attacked by several different antagonists, and more and more information is becoming available on the VOC response of plants attacked by multiple species of attacking herbivores or other antagonists [<a href="#CR20">20</a>].</p> <p id="Par14">An increasing number of studies shows that attack by one herbivore or pathogen can induce a state of heightened defence capabilities in plants [<a href="#CR21">21</a>]. Changes may occur at the transcriptional, physiological, and metabolic levels, which effectively enable the plant to react more quickly or strongly to biotic and abiotic stresses. This defence priming is mediated by a range of chemicals, such as salicylic acid and pipecolic acid [<a href="#CR22">22</a>], jasmonic acid and ethylene [<a href="#CR23">23</a>], azealic acid [<a href="#CR24">24</a>], and various volatile compounds including, methyl salicylate and methyl jasmonate [<a href="#CR25">25</a>], green leaf volatiles [<a href="#CR26">26</a>], and monoterpenes [<a href="#CR27">27</a>]. VOCs are increasingly shown to be important for defence priming and systemic responses. For example, rapid responses of plants to wounding include emissions of lipoxygenase pathway (LOX) products, known as green leaf volatiles, comprising of different C6 compounds such as hexenals and other aldehydes [<a href="#CR28">28</a>]. These compounds are produced and emitted within seconds of wounding stress and are typically transient, however emissions can be maintained for longer periods of time following repeated wounding or herbivory. VOCs are also involved in plant induced systemic immunity (known as systemic acquired resistance, or SAR) that is triggered throughout a plant after it has been exposed to a local pathogen infection. SAR is characterised by high levels of accumulated SA throughout the plant and induced volatile monoterpenes are essential for within-plant SAR [<a href="#CR27">27</a>], but it has recently been suggested that these compounds can also act as signalling molecules for long-distance SAR induction between plants [<a href="#CR29">29</a>]. As the defence pathways induced by different types of antagonists such as chewing and sucking herbivores can be antagonistic or synergistic, the question arises of how attack by both feeding guilds affects induction of chemical defences. While studies have already described complex interactions including defence priming, we are far from understanding the complex response of plants to attack by multiple herbivores.</p> <p id="Par15"><i>Tanacetum vulgare</i> L. (Asteraceae), commonly known as tansy, is a highly aromatic herb that has extensive variation in terpenoid content stored in glands on the leaf surface [<a href="#CR30">30</a>â€"<a href="#CR32">32</a>]. Tansy plants can be grouped according to their terpenoid content, forming chemotypes [<a href="#CR6">6</a>, <a href="#CR33">33</a>]. Tansy chemotypes have been shown to significantly affect the colonisation of a specialised aphid species early in the season (during the main dispersal event when winged morphs are produced), however failed to have an effect on aphid colonisation later in the season, when aphids are unwinged and dispersal is via walking only [<a href="#CR6">6</a>]. One of these specialist aphids is <i>Metopeurum fuscoviride</i> Stroyan (Aphididae). Although <i>M. fuscoviride</i> is specialised to tansy, and causes minimally invasive damage, it can induce salicylic acid defence responses [<a href="#CR34">34</a>]. While tansy chemotypes affect aphid preference [<a href="#CR35">35</a>], and performance in the field [<a href="#CR6">6</a>, <a href="#CR34">34</a>], it is unclear if plants of different chemotypes also differ in their response to aphid attack. As tansy is attacked not only by sucking herbivores but also chewing ones, chemotypes may differ in how aphid priming affects the plantâ€™s response to subsequent attack by chewing herbivores.</p> <p id="Par16">In the present study we investigated how tansy chemotypes responded to attack by chewing herbivores when the plants had, or had not, been previously attacked by sucking aphids. VOCs can be emitted immediately following damage caused by a herbivore, or can be induced and emitted several hours later [<a href="#CR10">10</a>]. Here we looked at the effects of tansy chemotypes on the emission of VOCs triggered by two insect herbivores from differing feeding guilds; sucking aphids and chewing caterpillar larvae. To measure the real-time VOC emissions we used a platform introduced by Jud et al., 2018 [<a href="#CR12">12</a>], where we combine gas chromatography- mass spectroscopy (GC-MS) and proton transfer reaction time of flight mass spectroscopy (PTR-ToF-MS, [<a href="#CR36">36</a>]). We show that there are strong differences in VOC emissions upon herbivory across chemotypes. Furthermore, using RNA-Seq and de novo assembly of transcriptomes we examined transcriptional changes, i.e. induced/suppressed gene expression following aphid and caterpillar herbivory, with a view to simultaneously analyse metabolomic and transcriptomic changes to observe any inducible response of the plants to both sucking and chewing herbivores.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par17">In the central experiment of the work (Fig.Â <a href="#Fig1">1</a>b), the five terpenoid chemotypes were cultivated in a multi cuvette system (the scheme of the system is depicted in the additional Fig. S<a href="#MOESM1">1</a>) over seven days, continuously measuring the VOC emissions. At certain timepoints (days 4 and 7) leaf samples were also taken from all plants for chemical analyses, and the transcriptome study (only of plant chemotype 3). In order to select plants containing a wide range of compounds, we assessed the VOC profile of eight tansy plants using the hexane extraction method outlined in [<a href="#CR6">6</a>]. A total of 48 compounds were identified, and five chemotypes were assigned according to relative dominance of compounds (Fig. <a href="#Fig1">1</a>a). All plant chemotypes were propagated by splitting plants into nine daughter clones, which were further used as biological replicates as the chemotype is stable among clones [<a href="#CR6">6</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p><b>a</b> chemotype groupings of plants used in the cuvette experiment. Plant 3 was used for transcriptome analysis; <b>b</b> schematic of experimental timeline. Leaf material for RNA and liquid extraction was harvested on days 4 and 7. Leaf volatile emissions were continuously measured using PTR-ToF-MS, and collected daily using adsorbent Tenax and Carbopack cartridges</p>   </div> <p /> <div class="changesofterpenoidpatternsinthestoragepoolsoftansychemotypescausedbyaphidsandcaterpillars(hexaneextractions)" title="sec"> <div class="title" tagx="title" title="title">
Changes of terpenoid patterns in the storage pools of tansy chemotypes caused by aphids and caterpillars (hexane extractions)</div> <p id="Par18">A total of 64 compounds were identified across all chemotypes using GC-MS analysis of the hexane extracts, which were grouped into six compound classes (see Table S<a href="#MOESM5">1</a>). The compounds detected were mostly mono- and sesquiterpenes; the increased number of identified compounds in comparison to the initial chemotyping is due to higher overall emissions and herbivory induced stress compounds. Plant chemotypes 1 and 5 were partially dominated by (<i>Z</i>)-dihydrocarvone (&amp;gt;â€‰30%), while plant chemotype 2 was characterised by a slight dominance of myrtenol (~â€‰25%) followed by myrtenyl acetate (&amp;gt;â€‰10%). Plant chemotype 3 was strongly dominated by L-camphor (&amp;gt;â€‰70%), whereas plant chemotype 4 was slightly dominated by sabinene (~â€‰20%), with eucalyptol (~â€‰15%) and (E)-sabinene hydrate (&amp;gt;â€‰10%) also found in the mixture (Fig. <a href="#Fig1">1</a>a). FigureÂ <a href="#Fig2">2</a> shows clear differences between the five chemotypes. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Heatmap showing changes in concentration of VOCs measured using the hexane extraction method after each treatment. Each point in the heatmap represents the averaged data from three biological replicates (<i>n</i>â€‰=â€‰3 for each treatment) and two technical replicates. N: no aphid, no caterpillar, leaf material harvested on day 4; A: aphid, no caterpillar, leaf material harvested on day 7; C: no aphid, caterpillar, leaf material harvested on day 4; B: both aphid and caterpillar, leaf material harvested on day 7. The VOC concentrations for the heatmap are listed in supplemental Table S<a href="#MOESM6">2</a>. *Plant 3 was used for transcriptome analysis. MT: monoterpene, MT-acetate: monoterpene acetate, O-MT: oxygenated monoterpene, SQT: sesquiterpene, O-SQT: oxygenated sesquiterpene, SQT-lactone: sesquiterpene lactone</p>   </div> <p /> <p id="Par19">A two-tailed t-test analysis on the summed concentrations of all volatiles across plant chemotypes per treatment revealed no significant differences between treatment groups except between treatment groups N and B (both aphids and caterpillar; t-valueâ€‰=â€‰âˆ’â€‰3.659, dfâ€‰=â€‰4, <i>P</i>â€‰=â€‰0.022; Fig. S<a href="#MOESM2">2</a>). While there were little differences in total emission, there were significant differences among chemotypes in their response to herbivory of the two different species.</p> <p id="Par20">The terpenoid pattern of treatment group N plants (no aphid, no caterpillar) in Fig.Â <a href="#Fig2">2</a> represents the unstressed chemotype, with correspondingly lower concentrations of terpenoids detected in the hexane extracts compared to the other treatments (compound concentrations are listed Table S<a href="#MOESM6">2</a>). Treatment group A plants (aphid, no caterpillar) show the influence of aphid feeding on the profile of the stored (hexane extracted terpenoid compounds) chemotype. The aphid effect on the pattern and concentration of stored terpenoids was minimal; in plant chemotypes 1, 3, 4, and 5 the concentrations of terpenoids increased slightly, whereas in the chemotype 2 the concentrations were reduced.</p> <p id="Par21">While aphids are minimally invasive phloem suckers, and as such do not cause a great physically apparent stress on the plant, caterpillars are chewing herbivores and can cause severe tissue loss, resulting in huge physical trauma to the plant. Treatment group C plants (no aphid, caterpillar) show a varied response to caterpillar damage; with plant chemotypes 2 and 3 exhibiting a strong increase in terpenoid concentration. In plant chemotypes 1 and 5 very little increase in VOC concentration could be observed in treatment group C, whereas plant chemotype 4 appeared to show a decrease in terpenoid accumulation across all compounds apart from eucalyptol, the concentration of which was slightly increased. Treatment group B plants (both aphid and caterpillar) showed markedly increased concentrations of terpenoids in all chemotypes except in plant chemotypes 2 and 3.</p> <p id="Par22">The high concentration of detected terpenoids (in particular L-camphor) found in plant chemotype 3 after application of caterpillars (treatment group C) is reflected in the VOC emission pattern of this chemotype (Fig.Â <a href="#Fig3">3</a>). While the hexane extraction method gives an overview of all compounds that are synthesised and stored within the leaf structures, the collection of VOCs on Tenax/ Carbopack cartridges from the cuvette outlet air comprises only compounds that were actually released from the plant into the headspace of the cuvettes. The confirmation of high levels of VOCs detected in the headspace samples (see next section) were in line with the results obtained from the hexane extracts. This indicates that these compounds were synthesised and stored/released in response to herbivore feeding. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Heatmap showing changes in VOC emissions (logarithmic scale) after treatments using headspace analysis. VOCs were focused on Tenax/ Carbopack cartridges. Each point in the heatmap represents the averaged data from three biological replicates (<i>n</i>â€‰=â€‰3 for each treatment. N: no aphid, no caterpillar, average of data from days 1â€"4; A: aphid, no caterpillar, average of data from days 5â€"7; C: no aphid, caterpillar, average of data from days 1â€"4; B: both aphid and caterpillar, average of data from days 5â€"7. The VOC emission rates for the heatmap are listed in supplemental Table S<a href="#MOESM6">3</a>. *Plant 3 was used for transcriptome analysis. MT: monoterpene, MT-acetate: monoterpene acetate, O-MT: oxygenated monoterpene, SQT: sesquiterpene, O-SQT: oxygenated sesquiterpene. Compound concentrations can be found in Table S<a href="#MOESM6">3</a></p>   </div> <p /> </div> <div class="emissionpatternofterpenoidsfollowingaphidandcaterpillarherbivory(volatilemeasurementsfromfilters)" title="sec"> <div class="title" tagx="title" title="title">
Emission pattern of terpenoids following aphid and caterpillar herbivory (volatile measurements from filters)</div> <p id="Par23">To obtain an overview of the compounds emitted by each plant chemotype over a specific timeframe, emitted VOCs were collected on Tenax/ Carbopack cartridges for a period of 2â€"3â€‰h (see methods). Forty compounds were identified by GC-MS, and are listed in supplemental Table S<a href="#MOESM5">1</a>. Compounds found in the headspace analysis were mainly categorised as mono- and sesquiterpenoids. Emission patterns were clearly defined by the chemotype grouping (Fig. <a href="#Fig3">3</a>, supplemental Table S<a href="#MOESM6">2</a>), and were similar in composition to those seen in the initial chemotyping using the liquid extraction method. A main result was that responses of tansy to herbivore attack depended on plant chemotype.</p> <p id="Par24">Emission of plant chemotype 1 was dominated by (<i>Z</i>)-dihydrocarvone. Plants treated with caterpillars (treatment group C; no aphids, caterpillars) showed an increase in myrtenol and p-cymene emissions. Plant chemotype 2 was slightly dominated by sabinene and myrtenyl acetate. Plant chemotype 3 was strongly dominated by L-camphor. Plant chemotypes 1, 2, and 3 belonging to the treatment group C (no aphids, caterpillars) all emitted slightly higher levels of VOCs than plants pre-treated with aphids. Plant chemotype 4 emitted a more even blend of compounds with a slight dominance of sabinene, and eucalyptol. (<i>Z</i>)-dihydrocarvone was strongly emitted by plant chemotype 5, with a variety of other compounds including p-cymene and sabinene. In contrast, plant chemotypes 4 and 5 that were pre-treated with aphids (treatment group B) displayed higher VOC emissions than untreated plants (treatment group C). The emission patterns observed in the Tenax/ Carbopack measurements (Fig. <a href="#Fig3">3</a>) were also reflected in both the hexane extracts (Fig. <a href="#Fig2">2</a>) and PTR-ToF-MS analyses (see next section).</p> </div> <div class="dynamicemissionsoftansyvolatilesduringaphidandcaterpillarfeeding(continuousvolatileemissionmeasurements)" title="sec"> <div class="title" tagx="title" title="title">
Dynamic emissions of tansy volatiles during aphid and caterpillar feeding (continuous volatile emission measurements)</div> <p id="Par25">Tansy VOC emissions were continuously measured online using PTR-ToF-MS both before and after application of aphids and caterpillars. The time course of emissions demonstrated the effects of herbivore damage across all five chemotypes, and again pointed to chemotype-specific reactions to herbivore attack.</p> <p id="Par26">Diurnal variation in terpenoid emissions was observed across all plant chemotypes (e.g. sum of monoterpenes at <i>m/z</i> 137.133 and sesquiterpenes at <i>m/z</i> 205.196; see Fig.Â <a href="#Fig4">4</a>b and g respectively). High levels of emissions were seen at the onset of the experiment, when the plants were placed in the cuvettes. Except hexenal (mass <i>m/z</i> 99.081), the emission rates of all detectable mass features were elevated at the beginning of the measurements and declined within the first 24â€‰h. This indicates that no mechanical damage to the plants occurred when the plants were placed in the cuvettes. Emissions from the first 24â€‰h were not included in any data analysis as they were scattered and not representative of baseline volatile emissions from unstressed plants. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Time course of selected VOC emissions measured online using PTR-ToF-MS. Emission data from all chemotypes was averaged according to treatment. Plants that did not receive aphid treatment are represented by the blue lines. Plants that received aphid treatment are represented by the red lines. Where applicable, aphids were applied on day 1; caterpillars were applied on day 4. Leaf biomass was harvested on days 4 and 7. N: no aphid, no caterpillar, A: aphid, no caterpillar, C: no aphid, caterpillar, B: both aphid and caterpillar. <b>a</b><i>m/z</i> 135.116, representing monoterpenoids such as p-cymene; <b>b</b><i>m/z</i> 137.133, representing monoterpenes such as limonene; <b>c</b><i>m/z</i> 151.112, representing terpenoids such as carvone; <b>d</b> m/z 151.149, representing volatile compounds such as (<i>E</i>)-4,8-dimethyl-1,3,7-nonatriene (DMNT); <b>e</b><i>m/z</i> 153.128, representing oxygenated monoterpenoids such as L-camphor; <b>f</b><i>m/z</i> 155.144, representing oxygenated monoterpenoids such as eucalyptol; <b>g</b><i>m/z</i> 205.196, representing sesquiterpenoids such as germacrene D; <b>h</b><i>m/z</i> 99.081, representing hexenals, also known as green leaf volatiles (GLVs)</p>   </div> <p /> <p id="Par27">Application of aphids had no significant effect on the overall emission rates of the different compounds detected by PTR-ToF-MS. No obvious increase in emission occurred until day four, except a diurnal variation of the sum of monoterpenes (MTs; <i>m/z</i> 137.133; Fig. <a href="#Fig4">4</a>b) and three oxygenated monoterpenoids (O-MTs; <i>m/z</i> 135.116, <i>m/z</i> 151.112, <i>m/z</i> 153.128; see Fig. <a href="#Fig4">4</a>a, c, and e respectively; Supplemental Table S<a href="#MOESM6">3</a>). Addition of caterpillar larvae immediately changed tansy emissions, with all emission rates immediately increasing, whereas the diurnal variations can be seen in some mass features. It is not clear from the profiles whether the increase in emissions is merely a wounding effect or a rapid induction of the emissions after application of the chewing herbivores. A transient increase in the emission of dimethylnonatriene (DMNT, <i>m/z</i> 151.149; Fig. <a href="#Fig4">4</a>d), MTs, and the O-MTs was observed. In contrast, the emission of sesquiterpenes (SQTs; <i>m/z</i> 205.196) decreased after a sharp increase over time, while the emission of green leaf volatiles, exemplified by the hexenal signal at m/z 99.081, Fig. <a href="#Fig4">4</a>h) showed an initial increase, followed by a strong diurnal fluctuation, which did not increase over time. Plants that received aphid treatment first showed an overall tendency towards higher emission rates of MTs and O-MTs as well as the C11 homoterpene DMNT compared to plants that were not pre-treated by the sucking insects (Fig.Â <a href="#Fig5">5</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Heatmap showing changes in emissions of representative masses measured using PTR-ToF-MS. Each point in the heatmap represents the averaged data from three biological replicates (<i>n</i>â€‰=â€‰3 for each treatment group). For comparison with the off-line GC-Ms analysis of adsorbent cartridges (0.1â€‰Lâ€‰min<sup>âˆ’â€‰1</sup>; collection period of 3â€‰h around noon), PTR-ToF-MS data were averaged for the same time intervals. N: no aphid, no caterpillar, average of data from days 1â€"4; A: aphid, no caterpillar, average of data from days 5â€"7; C: no aphid, caterpillar, average of data from days 1â€"4; B: both aphid and caterpillar, average of data from days 5â€"7. The VOC emission rates for the heatmap are listed in supplemental Table S<a href="#MOESM7">5</a>. *Plant 3 was used for transcriptome analysis</p>   </div> <p /> <p id="Par28">A heatmap analysis (Fig. <a href="#Fig5">5</a>, supplemental Table S<a href="#MOESM6">4</a>) of the averaged on-line mass spectrometric data visually confirms the classification of the plant chemotypes done by GC-MS analysis of the hexane extracts (Fig. <a href="#Fig2">2</a>). For comparison with the off-line GC-MS analysis of VOCs collected on adsorbent cartridges (0.1â€‰Lâ€‰min<sup>âˆ’â€‰1</sup>; collection period of 3â€‰h around noon), PTR-ToF-MS data from the same time intervals were averaged. Treatment group N is the average of data collected from days 1 to 4 of plants that were not infested with aphids, while treatment group A is the average of the same time but of plants that were infested with aphids. Treatment group C is the average of data collected from days 4 to 7 of plants that were subjected to caterpillar feeding but not aphid infestation, while treatment group B is the average of the same time but of plants that were subjected to feeding by both aphids and caterpillars. Chemotypic profiles of the five plants are highlighted by the differences in signal strength between MTs (<i>m/z</i> 137.133; e.g. sabinene and Î³-terpinene) and O-MTs (<i>m/z</i> 153.128; e.g. L-camphor and (<i>Z</i>)-dihydrocarvone). After the addition of aphids to plant chemotypes 1 and 2, a decrease in MT and O-MT levels could be detected, whereas plant chemotypes 3, 4, and 5 showed an increase in these monoterpenoids. Each chemotype responded differently to the aphid treatment. While caterpillar feeding generally increased plant VOC emissions, prior feeding by aphids further increased this for plant chemotypes 3, 4, and 5, but reduced it for chemotypes 1 and 2.</p> </div> <div class="transcriptomechangesinchemotype3followingaphidandcaterpillarfeeding" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome changes in chemotype 3 following aphid and caterpillar feeding</div> <p id="Par29">Following RNA-Seq analysis of RNA extracted from plant chemotype 3, de novo assembly yielded a total of 52,765 plant genes that were surveyed for transcriptional changes resulting from the combinations of aphid and caterpillar treatments. Differences between treatment contrast groups are outlined as follows: A-Nâ€‰=â€‰aphid effect, no caterpillar (<i>a</i>), C-Nâ€‰=â€‰caterpillar effect, no aphid (<i>c</i>), B-Aâ€‰=â€‰caterpillar effect, with aphid (<i>ca</i>), (B-A) â€" (C-N)â€‰=â€‰aphid effect, with caterpillar (<i>d</i>) (see Fig. S<a href="#MOESM3">3</a>).</p> <p id="Par30">A total of 502 differentially expressed genes (DEGs) were found with a fold change greater than two whether up- or down-regulated (<i>P</i>â€‰<div class="â€‰0.05;_UNKNOWN" href="#MOESM7">
5). DEGs associated with various defence responses are detailed in supplemental Table S<a href="#MOESM8">6</a> and visualised in a heatmap in Fig.Â <a href="#Fig6">6</a>. Consistent with the metabolomics data above, the aphid treatment contrast (<i>a</i>) elicited only a minor transcriptional response. DEGs encoding a putative NAC transcription factor 56 (TanvuEGr019790, ortholog to AT2G41890.1, <i>Arabidopsis thaliana</i>; E value: 2.39e-17; 8.5 fold-change) and tobacco mosaic virus (TMV) resistance protein N (TanvuEGr041015, ortholog to OIT35319, <i>Nicotiana attenuata</i>; E-value: 2.6e-10; 7.3 fold-change) were upregulated. Considerably larger effects were observed for the infestation of plants with caterpillars. In treatment group contrasts involving caterpillar treatment, a series of Trypsin/chymotrypsin inhibitors orthologs TanvuEGr038583 (ortholog to AT1G73325, <i>A. thaliana</i>; E-value: 1.1e-3) TanvuEGr040924 (ortholog to AT1G73325, <i>A. thaliana</i>; E-value: 2.3e-5) TanvuEGr035344 (ortholog to AT1G73325, <i>A. thaliana</i>; E-value: 2.1e-3)) and putative abrin- and nigrin-like genes (impeding protein biosynthesis) were upregulated. Several genes were differentially regulated following caterpillar feeding only; a lipase (TanvuEGr016806, ortholog to AT3G04290.1, <i>A. thaliana</i>; E-value: 1.35e-4) was strongly upregulated (11.2 fold-change), while the receptor-like protein kinase FERONIA gene (TanvuEGr013486 (ortholog to AT3G51550.1, <i>A. thaliana</i>; E-value: 1.19e-2) was downregulated (âˆ’â€‰8.5 fold-change). Plants that were treated with both caterpillars and aphids (treatment contrast group (<i>ca</i>)) had the highest number of DEGs out of the four analysed treatment contrast groups. DEGs associated with cell wall processes including putative endochitinase EP3 (TanvuEGr027756, ortholog to AT3G54420.1, <i>A. thaliana</i>; E-value: 1.12e-11) and laccase-7 (TanvuEGr007097, ortholog to AT3G09220.1, <i>A. thaliana</i>; E-value: 1.28e-03) genes were strongly upregulated (10 and 11.2 fold-change, respectively). The same FERONIA gene, shown to be downregulated by caterpillar treatment only, was under the combined treatment highly upregulated (8.5 fold-change). An ortholog to AT4G27260 (<i>A. thaliana</i>; E-value: ~â€‰0), Indole-3-acetic acid-amido synthetase GH3.5 (TanvuEGr005980) was strongly downregulated (âˆ’â€‰7.1 fold-change), as were two putative G-type lectin S-receptor-like serine/threonine-protein kinases, SD2â€"5 (TanvuEGr003778, ortholog to AT4G32300.1, <i>A. thaliana</i>; E-value: 2.64e-02) and SD3â€"1 (TanvuEGr002669, ortholog to AT2G41890.1, <i>A. thaliana</i>; E-value: 2.39e-17) (âˆ’â€‰6.5 andâ€‰âˆ’â€‰9.6 fold-change respectively). Plants that were treated with both aphids and caterpillars, relative to plants that received only caterpillars (treatment contrast group (<i>d</i>)) did not exhibit many DEGs, however again the FERONIA gene mentioned before was very strongly upregulated (16.9 fold-change). </div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Heatmap visually representing differentially expressed genes related to defence processes from plant 3. Column A shows the effect of aphids with no caterpillars vs no aphids and no caterpillars (treatment groups A-N), C shows the caterpillar effect with no aphids vs no aphids and no caterpillars (treatment groups C-N), (<i>ca</i>) shows the effect of caterpillars with aphids vs caterpillars and no aphids (treatment groups B-A), and D shows the effects of caterpillars with aphids vs caterpillars with no aphids (treatment groups (B-A)-(C-N))</p>   </div> <p /> </div> <div class="phylogeneticanalysisoftpssfromtansyandotherplantspecies" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis of TPSs from tansy and other plant species</div> <p id="Par31">Since mono- and sesquiterpenes are basic components of the essential oils in the trichomes and they also dominate the volatile emissions of tansy, it is interesting to study the expression of the gene family on terpene synthases. The sequences of <i>TPS</i> genes from closely related plant species such as <i>Helianthus annuus</i> and <i>Artemisia annua</i>, were retrieved from online databases and aligned with the sequences of putatively annotated <i>TPS</i> genes from tansy. Phylogenetic analysis indicates that the tansy <i>TPSs</i> belong to the <i>TPS</i> subfamilies predicted (Fig. S<a href="#MOESM4">4</a>); for example, a putative (<i>E</i>)-Î²-ocimene synthase (TanvuEGr006575, ortholog to PWA70010.1, <i>A. annua</i>; E-value: 4e-147) and a putative sesquiterpene cyclase (TanvuEGr007220, ortholog to AAG24640.2, <i>A. annua</i>, E-value: 1e-124). The RNA-Seq data, however show that gene transcripts of putative <i>TPSs</i> are present, but only few show a change in their gene expression (Fig. <a href="#Fig6">6</a>, supplemental Table S<a href="#MOESM7">5</a>), e.g. two putative limonoid UDP-glucosyltransferases (TanvuEGr011661, (ortholog to AT4G15480.1, <i>A. thaliana</i>; E-value: 1.26e-04) and TanvuEGr028241 (ortholog to AT4G15480.1, <i>A. thaliana</i>; E-value: 3.49e-03)) and three <i>SQT</i> genes: a putative (âˆ’)-germacrene D synthase (TanvuEGr017925, ortholog to AT3G14490.1, <i>A. thaliana</i>; E-value: 2.73e-08) and two genes (TanvuEGr029614 and TanvuEGr007220) that are close orthologs to (<i>E</i>)-Î²-farnesene synthase (AT5G23960.1, <i>A. thaliana</i>; E-values: 1.02e-03 and 2.37e-03 respectively). While TanvuEGr029614 was upregulated (5.7 fold-change) under aphid treatment, the other a putative (<i>E</i>)-Î²-farnesene synthase only showed enhanced transcript level (2.8 fold-change) following aphid feeding and caterpillar attack.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par32">We combined a number of methods, lead extracts, cumulative headspace sampling and real-time headspace sampling using a multi-cuvette system [<a href="#CR12">12</a>], to investigate the response of five different tansy chemotypes to attack by a sucking and a chewing herbivore, and priming effects. Aphid feeding did not result in strong chemical changes, which might be related to a minimum infestation threshold of feeding aphids [<a href="#CR37">37</a>], whereas caterpillar feeding increased stored and emitted volatiles across each plant chemotype. Importantly, there were chemotype-specific responses to herbivore attack. We also found evidence for priming, but the strength was dependent on the chemotype. Feeding by aphids followed by caterpillars led to more varied effects on stored and emitted volatile compounds in each plant chemotype than if aphids had not been feeding on the plant before a caterpillar was introduced. Specifically, prior aphid feeding further increased concentrations of stored volatile compounds in plant chemotypes 1, 4, and 5, and increased concentrations of compounds emitted to the headspace in plant chemotypes 3, 4, and 5. We also demonstrated that the time point of measurements affects the results obtained, particularly in relation to plant handling and insect feeding patterns. Transcriptomic analysis of plant chemotype 3 represents the first de novo assembly of a tansy transcriptome.</p> <div class="differencesinresponsestoaphidsandcaterpillars:aphids" title="sec"> <div class="title" tagx="title" title="title">
Differences in responses to aphids and caterpillars: aphids</div> <p id="Par33">In this study, the different feeding mechanisms of the herbivores elicited very different defence responses in the tansy plants. Aphid feeding has been shown to exhibit variable effects on plant VOC emissions both alone and in conjunction with other herbivores. Schwartzberg and colleagues reported that exposure to the pea aphid (<i>Acyrthosiphon pisum</i>) did not induce any detectable changes in VOC emissions in its host plant <i>Vicia faba</i> [<a href="#CR38">38</a>], however contrastingly Du and colleagues showed that feeding by <i>A. pisum</i> on <i>V. faba</i> resulted in the induction and/or emission of several compounds [<a href="#CR39">39</a>]. Interestingly, Staudt and colleagues [<a href="#CR40">40</a>] found quantitative and qualitative differences in VOC emissions in peach cultivars (<i>Prunus</i> spp) exposed to <i>Myzus persicae</i>, with these differences being genotype-dependent. <i>Metopeurum fuscoviride</i> is specialised to tansy and may have co-evolved with the plant to minimise negative effects on each other, or the aphid may have evolved to avoid tansy defences, which could explain the minimal change in VOC emissions observed. It must be kept in mind that the intensity of aphid attack is also a key factor with aphid-induced volatiles; a minimum threshold of infestation of feeding aphids must be reached. Our previous work showed that aphid selection of host plants is non-random; early in the season aphids actively chose their host plant and colonised preferred chemotypes [<a href="#CR6">6</a>], and late in the season had almost exclusively colonised plants belonging to preferred metabotypes (irrespective of chemotype) [<a href="#CR34">34</a>]. Although aphids might select a host plant based on the volatile chemotype, the probability that they will successfully establish and maintain a colony is higher on certain metabotypes, and is related to other factors such as colony size and the presence of tending ants.</p> </div> <div class="differencesinresponsestoaphidsandcaterpillars:caterpillarsandpriming" title="sec"> <div class="title" tagx="title" title="title">
Differences in responses to aphids and caterpillars: caterpillars and priming</div> <p id="Par34">As expected, plants responded very strongly to caterpillar attack and an increase in monoterpene emissions was observed in every chemotype after the caterpillars were applied at day 4. We also found evidence of priming: the overall trend showed that the production of monoterpenes was highest in plants treated with both aphids and caterpillars, indicating an effect of the aphids on terpenoid biosynthesis. Measured in the lipophilic hexane extracts, the change in synthesis / storage of monoterpenes (compounds we measured, not including all known compounds often released following aphid feeding) after application of aphids is minimal, being slightly decreased in relation to the control (treatment group N) in plant chemotype 2, slightly increased in plant chemotypes 1, 4, and 5, and seemingly unchanged in plant chemotypes 3. Thus, chemotypes showed higher or lower emissions depending on whether they had been pre-treated with aphids. In plant chemotypes 2 and 3, the volatile response was higher in plants that were not treated with aphids, whereas plant chemotypes 1, 4, and 5 displayed a stronger volatile response when they were pre-treated with aphids., When all volatiles (measured in the hexane extracts) across all plants per treatment were summed, no significant differences were observed except between treatment groups N (no attack by any herbivore) and B (attack by both herbivores), showing that it is not the absolute amount of volatiles that differed but rather changes in the composition.</p> <p id="Par35">Different priming effects after aphid feeding have also been observed in other studies. Schwarzberg and colleagues [<a href="#CR38">38</a>] found that emission of caterpillar-induced VOCs was reduced when the plants were co-infested with aphids. Conversely, other studies have found that aphid feeding increases total emissions of volatile terpenoids when subsequently attacked by species of different feeding guilds [<a href="#CR41">41</a>, <a href="#CR42">42</a>]. Although there are differences in the experimental systems and specificity of aphid species, our observation that each chemotype responded differently to the herbivory treatments emphasises that there are chemoâˆ’/genotypic responses of plants to multiple herbivore attack.</p> </div> <div class="ptr-tof-msandothermethodspointstodisruptionoftrichomes" title="sec"> <div class="title" tagx="title" title="title">
PTR-ToF-MS and other methods points to disruption of trichomes</div> <p id="Par36">The use of PTR-ToF-MS allowed us to observe changes in VOC emissions in real time. Increases in emissions of terpenoid compounds upon exposure to caterpillars were seen at random times of the day and night, likely reflecting activity patterns of feeding caterpillars. Caterpillars generally consume all parts of the leaf tissue and consequently will disrupt glandular trichomes while feeding. Due to this is was not possible to discriminate whether compounds were released from trichomes, i.e. were stored, or originated from de novo biosynthesis of tansy plants. However, the strong fluctuations in the emissions and pronounced peaks suggest that the destruction of trichomes and the resulting immediate evaporation of VOCs is a major cause of the emission. Emission courses of plants without trichomes usually show stable, homogeneous emissions of terpenoids when under herbivore feeding attack [<a href="#CR43">43</a>].</p> <p id="Par37">The interpretation is reinforced by the observation that volatile emission composition patterns of the different chemotypes were not drastically changed by the application of the two herbivores, but instead the levels of emission of dominant compounds and other closely related compounds increased. This suggests that the destruction or mechanical/physical stress of the trichome hairs by the arthropod movements contributed significantly to the increase in VOC emissions. Closely related compounds such as myrtenol and myrtenyl acetate, or dihydrocarvone and neodihydrocarveol, tended to all increase simultaneously. This also implies that herbivores feeding on different chemotypes causes different volatile profiles, rather than always the same.</p> </div> <div class="localversusoverallresponses" title="sec"> <div class="title" tagx="title" title="title">
Local versus overall responses</div> <p id="Par38">Local and overall (systemic) volatile responses to both types of herbivory were investigated using two collection methods for GC-MS analysis. Hexane extraction, using control leaves and leaves from highly damaged sites, provided information on the local response to herbivory, while headspace analysis gave a view of the overall plant response. The high concentration of terpenoids (especially L-camphor) found in lipophilic extracts of plant chemotype 3 after aphid application is reflected in the emission patterns and rates of the plants. The increase in the concentration of stored terpenoids suggests an increased synthesis and storage of compounds, which then also contributes to an increased emission. In order to be able to clearly identify the origin of the emitted VOCs - from storage pools or from de novo biosynthesis - a <sup>13</sup>C labelling experiment would have to be carried out (e.g. Ghirardo et al. 2020 [<a href="#CR44">44</a>]), however this was beyond the scope of this study.</p> </div> <div class="geneexpression,identificationofgenes,differentialresponsestoherbivores" title="sec"> <div class="title" tagx="title" title="title">
Gene expression, identification of genes, differential responses to herbivores</div> <p id="Par39">Our transcriptome analysis of plant chemotype 3 confirmed that several genes involved in terpenoid synthesis and defence were differentially regulated depending on the treatment. In the transcriptomic analysis, treatment contrast group (<i>a</i>), detailing plants that were treated with aphids in relation to untreated plants, DEGs encoding tansy orthologs of NAC56 transcription factor and a TMV resistance protein N were upregulated. The NAC56 transcription factor (TF) is known to play a role in plant responses to abiotic stress and challenge by pathogens [<a href="#CR45">45</a>]. Chen and colleagues demonstrated that in <i>Brassica napus</i> (oilseed rape), <i>BnNAC56</i> was significantly induced by JA, SA, and ABA, indicating that it may play a central role in plant tolerance of biotic stresses. Although there is limited evidence that NAC TFs regulate hypersensitive response (HR), this study suggests that BnaNAC56 induces HR-like cell death and thus is a form of inducible defence. TMV resistance protein N is a disease resistance protein that when triggered can induce a hypersensitive response [<a href="#CR46">46</a>]. It is interesting that DEGs commonly associated with pathogen infection were upregulated in tansy upon aphid feeding, as it is known that plants mount a similar defence against both aphids and biotrophic pathogens [<a href="#CR47">47</a>]. In treatment contrast group (<i>c</i>), detailing plants that were treated with caterpillars in relation to untreated plants, the putative receptor-like protein kinase <i>TvFERONIA</i> was strongly downregulated. In response to chewing herbivory, the JA defence pathway is induced. As FERONIA regulates JA signalling in an antagonistic way [<a href="#CR48">48</a>], it is unsurprising that it was downregulated in tansy plants that received only caterpillar treatment.</p> <p id="Par40">Plants that had experienced both herbivores (treatment contrast group (<i>ca</i>), detailing plants that were treated with aphids and caterpillars in relation to plants that received only aphid treatment) had the highest number of DEGs out of all four treatment groups. Tansy orthologs of endochitinase <i>EP3</i> and <i>laccase-7</i> were both strongly upregulated in these plants. It has been previously shown that overexpression of the cotton laccase gene led to higher resistance against both fungal plant pathogens (<i>Verticillium dahlia)</i> and herbivorous pests (cotton bollworm <i>Helicoverpa armigera and the cotton aphid Aphis gosypii)</i> in cotton plants [<a href="#CR49">49</a>]. Indole-3-acetic acid-amino synthetase <i>GH3.5</i> belongs to an auxin responsive gene family, and is responsible for catalysing the synthesis of indole-3-acetic acid (IAA). IAA helps plants deal with the presence of excess auxin, and overexpression of <i>GH3.5</i> results in enhanced resistance to fungal pathogens in rice [<a href="#CR50">50</a>] via a mechanism that inhibits cell wall loosening and cell growth. It has been shown that aphid feeding results in significantly altered transcripts associated with cell wall metabolism [<a href="#CR51">51</a>].</p> <p id="Par41">Our transcriptomic analysis also showed high variability between the individual biological replicates, which made the evaluation difficult and limited the number of significant DEGs. Reasons for this could be that the interaction of the herbivores with the plants varied from experiment to experiment, which could lead to different time courses of gene expression. In addition, the leaves sampled for the analyses may not have been in exactly the same physiological state because of different light and temperature gradients within the plant canopy. The difference in DEGs after caterpillar feeding between tansy plants that were first attacked by aphids and those that were not was minimal. Due to the fluctuations in gene expression, we cannot exclude the possibility that we have overseen some possible synergistic interactions between aphid and caterpillar attack.</p> </div> <div class="transcriptome:synthases" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome: synthases</div> <p id="Par42">Tansy has a large chemical diversity of terpenoids. It is therefore not surprising that we were able to detect many transcripts with high similarity to terpene synthases in the de novo assembled transcriptome of plant chemotype 3. The phylogenetic analysis of putative terpene synthase genes shows that the tansy genes belong to the known <i>TPS</i> subgroups in dicotyledonous plants. Seven genes in group <i>TPS-b</i> which comprises monoterpene synthases, 16 in group <i>TPS-a</i>, which is characteristic for sesquiterpene synthases. Furthermore, we could detect one putative gene in the <i>TPS-f</i> group, which includes linalool synthases. The high homology of individual <i>TPS</i> genes to known and enzymatically characterised <i>TPS</i> does not imply that the enzymes annotated in tansy have the same biosynthesis products [<a href="#CR8">8</a>, <a href="#CR52">52</a>â€"<a href="#CR54">54</a>]. In order to make a conclusive statement about the enzyme activity of the putative <i>TPS</i>, we are aware that each gene must be expressed heterologously and subjected to biochemical function analysis [<a href="#CR55">55</a>â€"<a href="#CR58">58</a>].</p> </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par43">In conclusion, our work shows that there is a chemotype-specific response in plants to herbivore attack, and that this also extends to priming, i.e. the response of plants to a second herbivore attack by another species. This will be of relevance to researchers investigating plant chemical communication in the field. If neighbouring chemotypes in a field population respond differently to herbivory/dual herbivory (i.e. some chemotypes respond by emitting higher levels of VOCs than others), this could potentially have effects ranging from the individual level to the group level. Individuals of some chemotypes may respond more efficiently to herbivory stress than others, and in a group setting these â€œlouderâ€� chemotypes might influence the local insect community, including natural enemies of herbivores, and other neighbouring plants.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantandinsectmaterial" title="sec"> <div class="title" tagx="title" title="title">
Plant and insect material</div> <p id="Par44">Tansy seeds were collected with the consent of the property owner on a field site in southern Germany (N 48Â°25â€²1.51â€œ; E 11Â°46â€™1.19â€�), described earlier in [<a href="#CR6">6</a>] following the Convention on the Trade in Endangered Species of Wild Fauna and Flora. The parental plants were formally identified by S. Zytynska as <i>Tanacetum vulgare</i> (L.). Five different tansy genotypes were grown from seeds and propagated by cuttings in the greenhouse in 2015 (the same year the experiment was performed). Plants were grown to a size of approximately 30â€‰cm high in order to fit the size limitations of the glass cuvette system. The volatile terpenoid pattern of each genotype was assessed using the leaf hexane extraction method as outlined in [<a href="#CR6">6</a>]. Nine clonal daughters of each genotype were generated (via splitting), with the six healthiest plants selected for use in the experiment to serve as biological replicates.</p> <p id="Par45">All aphids (<i>Metopeurum fuscoviride</i> Stroyan) were kept under greenhouse conditions (21.7â€‰Â°C, 70% relative humidity, 16:8â€‰h light:dark) at DÃ¼rnast Experimental Station, Technical University Munich, Freising, Germany, and were reared on chemotypes not used in the experiment. (for further rearing details see [<a href="#CR35">35</a>]). As a chewing herbivore, we chose <i>Spodoptera littoralis</i> Boisduval, the African cotton leafworm, as it is a generalist caterpillar that can cause massive trauma to plants, oftentimes completely stripping a plant of its foliage. It has been shown that specific VOCs are released in response to <i>S. littoralis</i> feeding on tansy, with a similar VOC profile emitted upon application of <i>S. littoralis</i> oral secretions [<a href="#CR59">59</a>]. First instar larvae of <i>S. littoralis</i> were reared on commercial lettuce leaves at room temperature. All larvae were starved for 24â€‰h prior to application on the plants in order to ensure immediate feeding.</p> </div> <div class="initialselectionofterpenoidchemotypes" title="sec"> <div class="title" tagx="title" title="title">
Initial selection of terpenoid chemotypes</div> <p id="Par46">In order to select plants containing a wide range of compounds, we assessed the VOC profile of eight tansy plants using the hexane extraction method outlined in [<a href="#CR6">6</a>]. A total of 48 compounds were identified, and five chemotypes were assigned according to relative dominance of compounds (Fig. <a href="#Fig1">1</a>a). All plant chemotypes were propagated by splitting plants into nine daughter clones, which were further used as biological replicates as the chemotype is stable among clones [<a href="#CR6">6</a>].</p> </div> <div class="experimentaldesign" title="sec"> <div class="title" tagx="title" title="title">
Experimental design</div> <p id="Par47">In the central experiment, five tansy â€œchemotypesâ€� (Fig. <a href="#Fig1">1</a>a) with six biological replicates each, were exposed to caterpillars after they had been attacked or not attacked by aphids (Fig. <a href="#Fig1">1</a>b, six plant replicates per chemotype, three replicates not treated with aphids, three replicates treated with aphids, all replicates later treated with caterpillars). On day zero of the experiment five randomised tansy plants that were approximately 30â€‰cm tall were placed into cuvette bases within the system described shown in Fig. S<a href="#MOESM1">1</a>, with one cuvette kept empty to provide an empty background for statistical normalisation. It is important to note that when handling aromatic plants extreme care must be taken to avoid jostling the leaves so as to minimise disruption of oil storage cavities (trichomes). An appropriate amount of time must have passed for the plant to â€œsettle downâ€� after events such as applying the herbivores and placing the plant in the system. A large glass bulb was then carefully placed over each plant. The plants were cultivated with a day-night change of 16/8â€‰h at a light intensity of approximately 200â€‰Î¼mol photons m<sup>âˆ’â€‰2</sup>â€‰s<sup>âˆ’â€‰1</sup> (Agro Son-T 400â€‰W, Philips, Hamburg, Germany) above the plants. Temperature varied from 23â€‰Â°C at night to &amp;gt;â€‰30â€‰Â°C during the light phase. All plants were allowed to equilibrate for 24â€‰h. On day 1 of the experiment at midday, 100 unwinged adult aphids were carefully applied to the aphid treatment plants. On day 4 of the experiment two second-instar <i>S. littoralis</i> larvae were then carefully applied to each plant. The experiment concluded on day 7. All <i>S. littoralis</i> larvae were collected and destroyed in accordance with Council Directive 2000/29/EC.</p> <p id="Par48">We sampled our two-stage experiment twice at two sampling times (day 4, day 7), to look at the effect of aphid infestation only. In the first half of the experiments, plants not attacked by aphids (no aphids, no caterpillars) were labelled â€œNâ€�; and plants with only aphid treatment (aphids, no caterpillars) were labelled â€œAâ€�. In the second half of the experiment, plants that had previously not been attacked by aphids but received caterpillars (no aphids, caterpillars) were labelled â€œCâ€�, and aphid-treated plants that received caterpillars (both aphids and caterpillars) were labelled â€œBâ€�. FigureÂ <a href="#Fig1">1</a>a shows the VOC pattern of plants in treatment group N, i.e. the VOC emission of the different chemotypes when not attacked by a herbivore.</p> <p id="Par49">Leaf samples were collected from all plants first at day 4. We carefully avoided areas infested with aphids, and froze leaves immediately in liquid nitrogen, which were then stored at âˆ’â€‰80â€‰Â°C until further use. On day 7, all remaining leaves from all plants were harvested and immediately frozen in liquid nitrogen. Headspace volatiles were collected daily using mixed Tenax/Carbopack tubes (see Fig. <a href="#Fig1">1</a>b). All cuvettes were continuously monitored by PTR-ToF-MS in addition. Transcriptome analysis was performed on leaf material from plant chemotype 3.</p> </div> <div class="realtimeanalysisofemittedvocs" title="sec"> <div class="title" tagx="title" title="title">
Real time analysis of emitted VOCs</div> <p id="Par50">On-line analysis of emitted volatiles was conducted using the multiple cuvette system [<a href="#CR12">12</a>] comprising of six whole plant cuvettes coupled to a commercial PTR-ToF-MS, Ionicon Analytik GmbH, Innsbruck, Austria). The cuvettes consist of a glass bulb atop a gas tight cylindrical stainless-steel base using inert Viton rings to seal the joint (for details see Jud et al. [<a href="#CR12">12</a>]). The base contains connections for gas and irrigation tubing, as well as electrical connections. The cuvettes were flushed with air drawn from the experimental hall via a rotary vane compressor (DLT 40, Gardner Denver Schopfheim GmbH, Schopfheim, Germany). Air flow was set to 18â€‰Lâ€‰min<sup>âˆ’â€‰1</sup> from weeks one to three and reduced to 10â€‰Lâ€‰min<sup>âˆ’â€‰1</sup> from the beginning of week four for the remaining duration of the experiment. Air flow (~â€‰120â€‰mLâ€‰min<sup>âˆ’â€‰1</sup>) to the PTR-ToF-MS alternated between cuvettes, with switching time set to five minutes. The instrument was operated with an E/N of 115 Td (Eâ€‰=â€‰the electric field strength, Nâ€‰=â€‰the gas number density; 1 Tdâ€‰=â€‰10<sup>âˆ’â€‰17</sup>â€‰V cm<sup>2</sup>; drift tube pressureâ€‰=â€‰2.2â€‰mbar; drift voltageâ€‰=â€‰500â€‰V, drift tube temperatureâ€‰=â€‰60â€‰Â°C). Throughout the experiments, the ions H<sub>3</sub>OÂ·H<sub>2</sub>O<sup>+</sup>, O<sub>2</sub><sup>+</sup>, and NO<sup>+</sup> were kept below 10, 3, and 0.2% of the primary ions, respectively. The range of mass spectra was set to record up to <i>m/z 318</i>. The PTR-ToF-MS raw data were analysed using the routines described in [<a href="#CR12">12</a>, <a href="#CR60">60</a>]. Data were analysed as described in [<a href="#CR61">61</a>]. Briefly, calculated signals in counts per second were normalised to account for differences in the absolute humidity of each cuvette, giving the signals in normalised counts per second (ncps).</p> </div> <div class="gc-msanalysisofvocsfromabsorptiontubesandhexaneextraction" title="sec"> <div class="title" tagx="title" title="title">
GC-MS analysis of VOCs from absorption tubes and hexane extraction</div> <p id="Par51">For the VOC emission analysis by GC-MS from the cuvette measurements, a series of sorbent tubes containing 40â€‰mg Tenax TA /10â€‰mg Carbopack (both obtained from Sigma-Aldrich, Taufkirchen, Germany [<a href="#CR43">43</a>];) were coupled to the flow of air drawn (0.1â€‰Lâ€‰min<sup>âˆ’â€‰1</sup>; collection period of 3â€‰h around noon) from the outlet air of each cuvette on days four and seven (Fig. S<a href="#MOESM1">1</a>). After sampling, the Tenax/ Carbopack tubes were closed under gastight conditions and stored until analysis at 4â€‰Â°C. For GC-MS analysis the tubes were desorbed using a thermo-desorption unit (TDU; Gerstel GmbH, MÃ¼llheim an der Ruhr, Germany) coupled to a gas chromatography mass-spectrometer (GC-MS; GC: 7890A, MS: 5975C inert XL MSD with a triple axis detector, both Agilent Technologies, Palo Alto, CA, USA). The CIS vaporisation inlet (cooled injection system; Gerstel) was set at âˆ’â€‰50â€‰Â°C. The TDU was heated to 290â€‰Â°C from 37â€‰Â°C at a rate of 280â€‰Â°Câ€‰min<sup>âˆ’â€‰1</sup>. Samples were analysed splitlessly at a constant flow rate of helium at 1â€‰mLâ€‰min<sup>âˆ’â€‰1</sup>. After 0.2â€‰min the CIS was heated to 290â€‰Â°C at a rate of 12â€‰Â°Câ€‰min<sup>âˆ’â€‰1</sup> and held for 2â€‰min (method adapted from Ghirardo et al. 2012 [<a href="#CR43">43</a>]).</p> <p id="Par52">For the analysis of stored terpenoid compounds in leaves, hexane extracts were prepared from frozen leaf powder collected on days four and seven, and analysed by GC-MS. The extracts were prepared and analysed using the procedures and GC-MS methods as described in [<a href="#CR6">6</a>].</p> <p id="Par53">Identification and quantification of all VOCs was performed through comparison of obtained mass spectra with NIST 05 and Wiley library spectra, those of commercial standards (Sigma-Aldrich, Taufkirchen, Germany), and the Kovats retention index library [<a href="#CR62">62</a>].</p> </div> <div class="rnaextraction" title="sec"> <div class="title" tagx="title" title="title">
RNA extraction</div> <p id="Par54">On days 4 and 7, leaf tissue that had been badly damaged by <i>S. littoralis</i> or infested with <i>M. fuscoviride</i>, (however not containing any insect material) as well as control plants, was selected for analysis, with care being taken to avoid any aphids (<i>n</i>â€‰=â€‰3 for each treatment group, with 12 replicates in total). Leaf material was ground under liquid nitrogen. Total RNA extraction was performed using the innuPREP Plant RNA Kit from Analytik Jena AG (Jena, Germany) according to the manufacturerâ€™s instructions. RNA quality was confirmed using the Agilent 2100 Bioanalyzer with the RNA6000 Nano Lab Chip Kit (both Agilent Technologies, Palo Alto, CA, USA); the RNA Integrity Number (RIN) was between 7.5â€"8.</p> <p id="Par55">cDNA library construction and next-generation sequencing of transcripts were performed by Vertis Biotechnologie AG (Freising, Germany). Firstly, all samples were treated with DNase in order to remove any genomic DNA and then examined using a Shimadzu MultiNA microchip electrophoresis system (Shimadzu, Japan). Poly(A)â€‰+â€‰RNA was then isolated from the total RNA samples. First-strand cDNA was synthesised using an N6 randomised primer. Following fragmentation, the Illumina TruSeq sequencing adapters were ligated to the 5â€² and 3â€² ends of the cDNA fragments in a strand-specific manner. Finally, the cDNA was amplified with PCR (14â€"15â€‰cycles) using a proof-reading enzyme. cDNA was then pooled in approximately equimolar amounts, and eluted from a preparative agarose gel in the size range of 350â€"500â€‰bp. An aliquot of the size fractionated cDNA pool was analysed with capillary electrophoresis. The NGS library pool was paired-end sequenced with an Illumina NextSeq 500 system using 150â€‰bp (read 1) and 150â€‰bp (read 2) read length and a â€˜HIGH 300â€™ sequencing kit.</p> <p id="Par56">RNA-Seq data were assembled to transcript sequences applying two assemblers, the TRINITY platform [<a href="#CR63">63</a>] using default parameters and the entire read set, and Bridger [<a href="#CR64">64</a>] applying a kmer series (Kâ€‰=â€‰25,27,31) for combined replicates of each of the four samples separately. Resulting transcripts were merged into a consensus transcriptome assembly using the Evigene pipeline (<a href="http://arthropods.eugenes.org/EvidentialGene/">http://arthropods.eugenes.org/EvidentialGene/</a>) with default settings. Subsequently, reported transcripts were filtered for coding potential by Transdecoder v2.0.1 [<a href="#CR65">65</a>] including pfam searches and homology results. In total, 110,253 transcript/gene loci represented by 180,353 alternative splice variants were obtained. All transcripts were compared to an invertebrate protein dataset of NCBI. Sequences for which their top scoring hit matched a plant protein were retained in the final tansy transcriptome. This approach removed 69,613 transcripts with no protein homology and 16,578 insect related transcripts, resulting in 52,765 tansy transcript loci and 94,162 splice variants. Coding ORFs of the transcripts were annotated applying the AHRD pipeline (<a href="https://github.com/groupschoof/AHRD">https://github.com/groupschoof/AHRD</a>) and using default settings for the Interpro and Pfam searches, and homology comparisons to the <i>Arabidopsis thaliana</i>, Swissprot and Trembl databases.</p> <p id="Par57">Differential gene expression (DEG) analysis was performed using the R package <i>â€˜edgeR</i>â€™ v3.26 [<a href="#CR66">66</a>] applying a standard protocol. Briefly, salmon in quasi-mapping mode was applied to the tansy transcriptome for each sample and replicate to obtain a digital count matrix. To reduce data complexity and avoid overly stringent multiple hypothesis correction, we proceeded with mean counts of alternative splice variants for each transcript locus. Transcripts with low counts were filtered by the function â€˜<i>filterByExpr</i>â€™ and DEGs were derived by GLM (generalised linear models) method as provided in the <i>edgeR</i> package. After filtering, loci selected for DEG analysis totalled 31,928 loci. Four contrasts were analysed: A-N (with and without aphid treatment, <i>(a)</i> â€˜aphid effectâ€™), B-A and C-N (caterpillar treatment with and without pre-inoculation by aphids, <i>(ca)</i> and <i>(c)</i> respectively), and the combined contrast [(B-A) - (C-N)], <i>(d)</i>, the difference in response to caterpillars in the presence or absence of a pre-treatment by aphids; see Fig. S<a href="#MOESM3">3</a>.</p> <p id="Par58">Related TPS protein sequences from were obtained from NCBI, with an E-value cutoff of 1â€‰Ã—â€‰10<sup>âˆ’â€‰5</sup>. Protein sequences were aligned using MUSCLE [<a href="#CR67">67</a>]; poorly aligned positions were removed using Gblocks [<a href="#CR68">68</a>] and MaxAlign [<a href="#CR69">69</a>]. The maximum-likelihood phylogenetic tree was generated using RAxML [<a href="#CR70">70</a>] with 1000 bootstrap replications, and was drawn in Dendroscope [<a href="#CR71">71</a>].</p> </div> <div class="accessionnumbers" title="sec"> <div class="title" tagx="title" title="title">
Accession numbers</div> <p id="Par59">Raw and processed RNA-Seq data have been deposited at the NCBI Sequence Read Archive (SRA) with the BioProject number PRJNA646340.</p> </div> <div class="statistics" title="sec"> <div class="title" tagx="title" title="title">
Statistics</div> <p id="Par60">The metabolic profiles of all plant chemotypes and different treatments were analysed using MetaboAnalyst 3.0 [<a href="#CR72">72</a>]. Statistical analyses were performed using MetaboAnalyst 3.0 and SigmaPlot Version14 [<a href="#CR73">73</a>].</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2020_2745_MOESM1_ESM.tif">LINK</a> <p><b>Additional file 1:</b><b>Figure S1:</b> Schematic of cuvette platform and experimental setup.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2020_2745_MOESM2_ESM.tif">LINK</a> <p><b>Additional file 2:</b><b>Figure S2</b>: Two-tailed t-test analysis of summed concentrations of all VOCS (measured in hexane extracts) across chemotypes per treatment. A significant difference was found between treatment groups B and N (t-valueâ€‰=â€‰âˆ’â€‰3.659, dfâ€‰=â€‰4, <i>P</i>â€‰=â€‰0.022). N: no aphid, no caterpillar, leaf material harvested on day 4; A: aphid, no caterpillar, leaf material harvested on day 7; C: no aphid, caterpillar, leaf material harvested on day 4; B: both aphid and caterpillar, leaf material harvested on day 7.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2020_2745_MOESM3_ESM.tif">LINK</a> <p><b>Additional file 3:</b><b>Figure S3:</b> Visual representation of treatment groups for transcriptome analysis.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2020_2745_MOESM4_ESM.tif">LINK</a> <p><b>Additional file 4:</b><b>Figure S4:</b> Phylogenetic tree of <i>TPS</i> genes obtained from tansy and other related species. <i>TPS</i> subfamilies are coloured as follows, blue: green: <i>TPS-a, TPS-b</i>, yellow: <i>TPS-f</i>.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2020_2745_MOESM5_ESM.tif">LINK</a> <p><b>Additional file 5:</b><b>Table S1:</b> Compounds identified in hexane and SBSE extraction analysis methods.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2020_2745_MOESM6_ESM.xlsx">LINK</a> <p><b>Additional file 6:</b><b>Table S2</b>: Mean concentration of compounds of each plant chemotype found in hexane extractions at sampling days 4 and 7. <b>Table S3</b>: Mean VOC emission rates of each plant chemotype at sampling days 1â€"4 and 4â€"7. <b>Table S4</b>: Mean VOC emission rates of representative masses (m/z) measured using PTR-ToF-MS of different each plant chemotype at sampling days 1â€"4 and 4â€"7.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2020_2745_MOESM7_ESM.xlsx">LINK</a> <p><b>Additional file 7:</b><b>Table S5:</b> List of differentially expressed genes.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12870_2020_2745_MOESM8_ESM.xlsx">LINK</a> <p><b>Additional file 8:</b><b>Table S6:</b> List of differentially expressed genes associated with defence responses.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
VOCs</div> <div class="def" title="def"> <p id="Par4">Volatile organic compounds</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GC-MS</div> <div class="def" title="def"> <p id="Par5">Gas chromatography mass spectrometry</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PTR-ToF-MS</div> <div class="def" title="def"> <p id="Par6">Proton transfer reaction time-of-flight mass spectrometry</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TPS</div> <div class="def" title="def"> <p id="Par7">Terpene synthase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DEGs</div> <div class="def" title="def"> <p id="Par8">Differentially expressed genes</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SAR</div> <div class="def" title="def"> <p id="Par9">Systemic acquired resistance</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DMNT</div> <div class="def" title="def"> <p id="Par10">Dimethylnonatriene</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RNA-seq</div> <div class="def" title="def"> <p id="Par11">RNA next generation sequencing</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors wish to thank Robin Heinen for his comments on the manuscript.</p> </div> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>M.V.C., S.E.Z., M.S., W.W.W., and J.P.S. designed the study. M.V.C. performed the experiment. M.S. provided the invertebrate community. W.J., B.N., S.N., and M.V.C. ran and analysed PTR-ToF-MS measurements, while M.V.C. performed all GC-MS identification and quantification. G.H. processed and analysed transcriptome data. M.V.C. wrote the first draft of the manuscript, M.V.C. and W.J. and prepared the figures. All authors contributed to the manuscript drafts and approved the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was financially supported by the German Science Foundation (DFG grants SCHN 653/7â€"1 and WE 3080/25â€"1). The DFG was not involved in the design of the study, data analysis, and writing.</p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>The raw and processed transcriptomic datasets supporting the conclusions of this article have been deposited at the NCBI Sequence Read Archive (SRA) under bioproject number PRJNA646340 accessible at <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA646340">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA646340</a> or <a href="https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA646340">https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA646340</a>.The other datasets supporting the conclusion of this article are included within the article (and its additional files).</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par61">Tansy seeds were collected in the field with the permission of the property owner following the Convention on the Trade in Endangered Species of Wild Fauna and Flora.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p id="Par62">Not applicable.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par63">The authors declare that they have no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gleason</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foley</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thrall</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burdon</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plants versus pathogens: an evolutionary arms race</span><span class="source" tagx="source" title="source">Funct Plant Biol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">37</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">499</span><span class="lpage" tagx="lpage" title="lpage">512</span><span class="pub-id"><a href="https://dx.doi.org/10.1071/FP09304">10.1071/FP09304</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21743794">21743794</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kessler</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalske</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant secondary metabolite diversity and species interactions</span><span class="source" tagx="source" title="source">Annu Rev Ecol Evol Syst</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">49</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">115</span><span class="lpage" tagx="lpage" title="lpage">138</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-ecolsys-110617-062406">10.1146/annurev-ecolsys-110617-062406</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agrawal</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Specialist versus generalist insect herbivores and plant defense</span><span class="source" tagx="source" title="source">Trends Plant Sci</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">293</span><span class="lpage" tagx="lpage" title="lpage">302</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tplants.2012.02.006">10.1016/j.tplants.2012.02.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22425020">22425020</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heil</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Indirect defence via tritrophic interactions</span><span class="source" tagx="source" title="source">New Phytol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">178</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">41</span><span class="lpage" tagx="lpage" title="lpage">61</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1469-8137.2007.02330.x">10.1111/j.1469-8137.2007.02330.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18086230">18086230</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bustos-Segura</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dillon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keszei</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foley</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ¼lheim</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Intraspecific diversity of terpenes of <i>Eucalyptus camaldulensis</i> (Myrtaceae) at a continental scale</span><span class="source" tagx="source" title="source">Aust J Bot</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">65</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">257</span><span class="lpage" tagx="lpage" title="lpage">269</span><span class="pub-id"><a href="https://dx.doi.org/10.1071/BT16183">10.1071/BT16183</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clancy</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zytynska</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senft</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weisser</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnitzler</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemotypic variation in terpenes emitted from storage pools influences early aphid colonisation on tansy</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">38087</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep38087">10.1038/srep38087</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27892539">27892539</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christianson</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural and chemical biology of terpenoid cyclases</span><span class="source" tagx="source" title="source">Chem Rev</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">117</span><span class="issue" tagx="issue" title="issue">17</span><span class="fpage" tagx="fpage" title="fpage">11570</span><span class="lpage" tagx="lpage" title="lpage">11648</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.chemrev.7b00287">10.1021/acs.chemrev.7b00287</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28841019">28841019</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tholl</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bohlmann</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pichersky</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">The family of terpene synthases in plants: a mid-size family of genes for specialized metabolism that is highly diversified throughout the kingdom</span><span class="source" tagx="source" title="source">Plant J</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">66</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">212</span><span class="lpage" tagx="lpage" title="lpage">229</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-313X.2011.04520.x">10.1111/j.1365-313X.2011.04520.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21443633">21443633</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ro</span><span class="given-names" tagx="given-names" title="given-names">D-K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petri</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gershenzon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bohlmann</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pichersky</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of a root-specific arabidopsis terpene synthase responsible for the formation of the volatile monoterpene 1,8-cineole</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">135</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">1956</span><span class="lpage" tagx="lpage" title="lpage">1966</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.104.044388">10.1104/pp.104.044388</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15299125">15299125</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brilli</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loreto</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baccelli</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exploiting plant volatile organic compounds (VOCs) in agriculture to improve sustainable defense strategies and productivity of crops</span><span class="source" tagx="source" title="source">Front Plant Sci</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">264</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2019.00264">10.3389/fpls.2019.00264</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30941152">30941152</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bustos-Segura</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foley</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Foliar terpene chemotypes and herbivory determine variation in plant volatile emissions</span><span class="source" tagx="source" title="source">J Chem Ecol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">44</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">51</span><span class="lpage" tagx="lpage" title="lpage">61</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10886-017-0919-8">10.1007/s10886-017-0919-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29376212">29376212</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jud</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winkler</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niederbacher</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niederbacher</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnitzler</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Volatilomics: a non-invasive technique for screening plant phenotypic traits</span><span class="source" tagx="source" title="source">Plant Methods</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">109</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13007-018-0378-4">10.1186/s13007-018-0378-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30568721">30568721</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lubes</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodarzi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of volatile compounds by advanced analytical techniques and multivariate chemometrics</span><span class="source" tagx="source" title="source">Chem Rev</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">117</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">6399</span><span class="lpage" tagx="lpage" title="lpage">6422</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.chemrev.6b00698">10.1021/acs.chemrev.6b00698</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28306239">28306239</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Unsicker</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kunert</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gershenzon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Protective perfumes: the role of vegetative volatiles in plant defense against herbivores</span><span class="source" tagx="source" title="source">Curr Opin Plant Biol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">479</span><span class="lpage" tagx="lpage" title="lpage">485</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pbi.2009.04.001">10.1016/j.pbi.2009.04.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19467919">19467919</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beckers</span><span class="given-names" tagx="given-names" title="given-names">GJM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spoel</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fine-tuning plant defence signalling: salicylate versus jasmonate</span><span class="source" tagx="source" title="source">Plant biol (Stuttg)</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">01</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="pub-id"><a href="https://dx.doi.org/10.1055/s-2005-872705">10.1055/s-2005-872705</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16435264">16435264</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poelman</span><span class="given-names" tagx="given-names" title="given-names">EH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kessler</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Keystone herbivores and the evolution of plant defenses</span><span class="source" tagx="source" title="source">Trends Plant Sci</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">21</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">477</span><span class="lpage" tagx="lpage" title="lpage">485</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tplants.2016.01.007">10.1016/j.tplants.2016.01.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26832946">26832946</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lou</span><span class="given-names" tagx="given-names" title="given-names">Y-R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tzin</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jander</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Alteration of plant primary metabolism in response to insect herbivory</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">169</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">1488</span><span class="lpage" tagx="lpage" title="lpage">1498</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26378101">26378101</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tzin</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandez-Pozo</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richter</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmelz</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoettner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schafer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Dynamic maize responses to aphid feeding are revealed by a time series of transcriptomic and metabolomic assays</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">169</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">1727</span><span class="lpage" tagx="lpage" title="lpage">1743</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26378100">26378100</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devadas</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enyedi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raina</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Arabidopsis hrl1 mutation reveals novel overlapping roles for salicylic acid, jasmonic acid and ethylene signalling in cell death and defence against pathogens</span><span class="source" tagx="source" title="source">Plant J</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">467</span><span class="lpage" tagx="lpage" title="lpage">480</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-313X.2002.01300.x">10.1046/j.1365-313X.2002.01300.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12028576">12028576</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dicke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Loon</span><span class="given-names" tagx="given-names" title="given-names">JJA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soler</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemical complexity of volatiles from plants induced by multiple attack</span><span class="source" tagx="source" title="source">Nat Chem Biol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">317</span><span class="lpage" tagx="lpage" title="lpage">324</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nchembio.169">10.1038/nchembio.169</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19377458">19377458</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mauch-Mani</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baccelli</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luna</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flors</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title">Defense priming: an adaptive part of induced resistance</span><span class="source" tagx="source" title="source">Annu Rev Plant Biol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">68</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">485</span><span class="lpage" tagx="lpage" title="lpage">512</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-arplant-042916-041132">10.1146/annurev-arplant-042916-041132</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28226238">28226238</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernsdorff</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DÃ¶ring</span><span class="given-names" tagx="given-names" title="given-names">A-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gruner</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schuck</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BrÃ¤utigam</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeier</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pipecolic acid orchestrates plant systemic acquired resistance and defense priming via salicylic acid-dependent and -independent pathways</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">28</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">102</span><span class="lpage" tagx="lpage" title="lpage">129</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.15.00496">10.1105/tpc.15.00496</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26672068">26672068</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jia</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Involvement of jasmonic acid, ethylene and salicylic acid signaling pathways behind the systemic resistance induced by <i>Trichoderma longibrachiatum</i> H9 in cucumber</span><span class="source" tagx="source" title="source">BMC Genomics</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">20</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">144</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-019-5513-8">10.1186/s12864-019-5513-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30777003">30777003</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">HW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tschaplinski</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glazebrook</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenberg</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span></span><span class="mixed-article-title" title="mixed-article-title">Priming in systemic plant immunity</span><span class="source" tagx="source" title="source">Science.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">324</span><span class="issue" tagx="issue" title="issue">5923</span><span class="fpage" tagx="fpage" title="fpage">89</span><span class="lpage" tagx="lpage" title="lpage">91</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1170025">10.1126/science.1170025</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19342588">19342588</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryu</span><span class="given-names" tagx="given-names" title="given-names">C-M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evidence for volatile memory in plants: boosting defence priming through the recurrent application of plant volatiles</span><span class="source" tagx="source" title="source">Mol Cells</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">41</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">724</span><span class="lpage" tagx="lpage" title="lpage">732</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29991670">29991670</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Scala A, Allmann S, Mirabella R, Haring MA, Schuurink RC. Green leaf volatiles: a plant's multifunctional weapon against herbivores and pathogens. Int J Mol Sci. 2013;14(9):17781â€"811.</span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riedlmeier</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghirardo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wenig</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knappe</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koch</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Georgii</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Monoterpenes support systemic acquired resistance within and between plants</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1440</span><span class="lpage" tagx="lpage" title="lpage">1459</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.16.00898">10.1105/tpc.16.00898</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28536145">28536145</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Porta</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rocha-Sosa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant lipoxygenases. Physiological and molecular features</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">130</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.010787">10.1104/pp.010787</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12226483">12226483</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wenig</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghirardo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sales</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pabst</span><span class="given-names" tagx="given-names" title="given-names">ES</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breitenbach</span><span class="given-names" tagx="given-names" title="given-names">HH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antritter</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Systemic acquired resistance networks amplify airborne defense cues</span><span class="source" tagx="source" title="source">Nat Commun</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">3813</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-019-11798-2">10.1038/s41467-019-11798-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31444353">31444353</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benedek</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ¡lint</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salamon</span><span class="given-names" tagx="given-names" title="given-names">RV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KovÃ¡cs</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ã�brahÃ¡m</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fazakas</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Chemotype of tansy (<i>Tanacetum vulgare</i> L.) determines aphid genotype and its associated predator system</span><span class="source" tagx="source" title="source">Biol J Linn Soc</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">114</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">709</span><span class="lpage" tagx="lpage" title="lpage">719</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/bij.12445">10.1111/bij.12445</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keskitalo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pehu</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simon</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Variation in volatile compounds from tansy (<i>Tanacetum vulgare</i> L.) related to genetic and morphological differences of genotypes</span><span class="source" tagx="source" title="source">Biochem Syst Ecol</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">267</span><span class="lpage" tagx="lpage" title="lpage">285</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0305-1978(00)00056-9">10.1016/S0305-1978(00)00056-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11152946">11152946</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holopainen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiltunen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Schantz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">A study on tansy chemotypes</span><span class="source" tagx="source" title="source">Planta Med</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">53</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">284</span><span class="lpage" tagx="lpage" title="lpage">287</span><span class="pub-id"><a href="https://dx.doi.org/10.1055/s-2006-962707">10.1055/s-2006-962707</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17269022">17269022</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kleine</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MÃ¼ller</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Intraspecific plant chemical diversity and its relation to herbivory</span><span class="source" tagx="source" title="source">Oecologia.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">166</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">175</span><span class="lpage" tagx="lpage" title="lpage">186</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00442-010-1827-6">10.1007/s00442-010-1827-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21053017">21053017</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clancy</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zytynska</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moritz</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Witting</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmitt-Kopplin</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weisser</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metabotype variation in a field population of tansy plants influences aphid host selection</span><span class="source" tagx="source" title="source">Plant Cell Environ</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">41</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">2791</span><span class="lpage" tagx="lpage" title="lpage">2805</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/pce.13407">10.1111/pce.13407</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30035804">30035804</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senft</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clancy</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weisser</span><span class="given-names" tagx="given-names" title="given-names">WW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnitzler</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zytynska</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Additive effects of plant chemotype, mutualistic ants and predators on aphid performance and survival</span><span class="source" tagx="source" title="source">Funct Ecol</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">33</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">139</span><span class="lpage" tagx="lpage" title="lpage">151</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1365-2435.13227">10.1111/1365-2435.13227</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jordan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haidacher</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanel</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartungen</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MÃ¤rk</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seehauser</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A high resolution and high sensitivity proton-transfer-reaction time-of-flight mass spectrometer (PTR-TOF-MS)</span><span class="source" tagx="source" title="source">Int J Mass Spectrom</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">286</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">122</span><span class="lpage" tagx="lpage" title="lpage">128</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijms.2009.07.005">10.1016/j.ijms.2009.07.005</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guerrieri</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poppy</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powell</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tremblay</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pennacchio</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Induction and systemic release of herbivore-induced plant volatiles mediating in-flight orientation of <i>Aphidius ervi</i></span><span class="source" tagx="source" title="source">J Chem Ecol</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1247</span><span class="lpage" tagx="lpage" title="lpage">1261</span><span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1020914506782">10.1023/A:1020914506782</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwartzberg</span><span class="given-names" tagx="given-names" title="given-names">EG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ¶rÃ¶czky</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tumlinson</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pea aphids, <i>Acyrthosiphon pisum</i>, suppress induced plant volatiles in broad bean, <i>Vicia faba</i></span><span class="source" tagx="source" title="source">J Chemical Ecol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">37</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">1055</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10886-011-0006-5">10.1007/s10886-011-0006-5</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poppy</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powell</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pickett</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wadhams</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodcock</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of semiochemicals released during aphid feeding that attract parasitoid <i>Aphidius ervi</i></span><span class="source" tagx="source" title="source">J Chem Ecol</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">24</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">1355</span><span class="lpage" tagx="lpage" title="lpage">1368</span><span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1021278816970">10.1023/A:1021278816970</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Staudt</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El-aouni</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buatois</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lacroze</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PoÃ«ssel</span><span class="given-names" tagx="given-names" title="given-names">J-L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Volatile organic compound emissions induced by the aphid <i>Myzus persicae</i> differ among resistant and susceptible peach cultivars and a wild relative</span><span class="source" tagx="source" title="source">Tree Physiol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">1320</span><span class="lpage" tagx="lpage" title="lpage">1334</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/treephys/tpq072">10.1093/treephys/tpq072</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20739428">20739428</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moayeri</span><span class="given-names" tagx="given-names" title="given-names">HRS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashouri</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poll</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enkegaard</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Olfactory response of a predatory mirid to herbivore induced plant volatiles: multiple herbivory vs. single herbivory</span><span class="source" tagx="source" title="source">J Appl Entomol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">131</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">326</span><span class="lpage" tagx="lpage" title="lpage">332</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1439-0418.2007.01177.x">10.1111/j.1439-0418.2007.01177.x</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kroes</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weldegergis</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cappai</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dicke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Loon</span><span class="given-names" tagx="given-names" title="given-names">JJA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Terpenoid biosynthesis in Arabidopsis attacked by caterpillars and aphids: effects of aphid density on the attraction of a caterpillar parasitoid</span><span class="source" tagx="source" title="source">Oecologia.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">185</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">699</span><span class="lpage" tagx="lpage" title="lpage">712</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00442-017-3985-2">10.1007/s00442-017-3985-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29052769">29052769</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghirardo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heller</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fladung</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnitzler</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schroeder</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Function of defensive volatiles in pedunculate oak (<i>Quercus robur</i>) is tricked by the moth Tortrix viridana</span><span class="source" tagx="source" title="source">Plant Cell Environ</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">35</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">2192</span><span class="lpage" tagx="lpage" title="lpage">2207</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-3040.2012.02545.x">10.1111/j.1365-3040.2012.02545.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22632165">22632165</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ghirardo A, Lindstein F, Koch K, Buegger F, Schloter M, Albert A, et al. Origin of volatile organic compound emissions from subarctic tundra under global warming. Global Change Biology. 2020;n/a(n/a).</span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Oilseed rape NAC56 transcription factor modulates reactive oxygen species accumulation and hypersensitive response-like cell death</span><span class="source" tagx="source" title="source">Physiol Plant</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">160</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="lpage" tagx="lpage" title="lpage">221</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/ppl.12545">10.1111/ppl.12545</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28097691">28097691</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duncan</span><span class="given-names" tagx="given-names" title="given-names">GH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pradel</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carr</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oparka</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of the N gene hypersensitive response induced by a fluorescently tagged tobacco mosaic virus</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">123</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">1375</span><span class="lpage" tagx="lpage" title="lpage">1386</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.123.4.1375">10.1104/pp.123.4.1375</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10938355">10938355</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morkunas</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mai</span><span class="given-names" tagx="given-names" title="given-names">VC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GabryÅ›</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phytohormonal signaling in plant responses to aphid feeding</span><span class="source" tagx="source" title="source">Acta Physiol Plant</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">33</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">2057</span><span class="lpage" tagx="lpage" title="lpage">2073</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11738-011-0751-7">10.1007/s11738-011-0751-7</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Guo H, Nolan TM, Song G, Liu S, Xie Z, Chen J, et al. FERONIA receptor kinase contributes to plant immunity by suppressing jasmonic acid signaling in <i>Arabidopsis thaliana</i>. Curr Biol. 2018;28(20):3316â€"3324.e6.</span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Min</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Laccase GhLac1 modulates broad-spectrum biotic stress tolerance via manipulating phenylpropanoid pathway and jasmonic acid synthesis</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">176</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">1808</span><span class="lpage" tagx="lpage" title="lpage">1823</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.17.01628">10.1104/pp.17.01628</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29229698">29229698</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domingo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AndrÃ©s</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tharreau</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iglesias</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TalÃ³n</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Constitutive expression of OsGH3.1 Reduces auxin content and enhances defense response and resistance to a fungal pathogen in rice</span><span class="source" tagx="source" title="source">Mol Plant-Microbe Interact</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">22</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">201</span><span class="lpage" tagx="lpage" title="lpage">210</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/MPMI-22-2-0201">10.1094/MPMI-22-2-0201</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19132872">19132872</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foyer</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verrall</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hancock</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Systematic analysis of phloem-feeding insect-induced transcriptional reprogramming in Arabidopsis highlights common features and reveals distinct responses to specialist and generalist insects</span><span class="source" tagx="source" title="source">J Exp Bot</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">66</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">495</span><span class="lpage" tagx="lpage" title="lpage">512</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jxb/eru491">10.1093/jxb/eru491</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25540442">25540442</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Degenhardt</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ¶llner</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gershenzon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Monoterpene and sesquiterpene synthases and the origin of terpene skeletal diversity in plants</span><span class="source" tagx="source" title="source">Phytochemistry.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">70</span><span class="issue" tagx="issue" title="issue">15</span><span class="fpage" tagx="fpage" title="fpage">1621</span><span class="lpage" tagx="lpage" title="lpage">1637</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phytochem.2009.07.030">10.1016/j.phytochem.2009.07.030</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19793600">19793600</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ai</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Converting S-limonene synthase to pinene or phellandrene synthases reveals the plasticity of the active site</span><span class="source" tagx="source" title="source">Phytochemistry.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">137</span><span class="fpage" tagx="fpage" title="fpage">34</span><span class="lpage" tagx="lpage" title="lpage">41</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phytochem.2017.02.017">10.1016/j.phytochem.2017.02.017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28215610">28215610</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karunanithi</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zerbe</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Terpene synthases as metabolic gatekeepers in the evolution of plant terpenoid chemical diversity</span><span class="source" tagx="source" title="source">Front Plant Sci</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1166</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fpls.2019.01166">10.3389/fpls.2019.01166</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31632418">31632418</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudareva</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kish</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolosova</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorenstein</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FÃ¤ldt</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">(E)-Î²-Ocimene and myrcene synthase genes of floral scent biosynthesis in snapdragon: function and expression of three terpene synthase genes of a new terpene synthase subfamily</span><span class="source" tagx="source" title="source">Plant Cell</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">1227</span><span class="lpage" tagx="lpage" title="lpage">1241</span><span class="pub-id"><a href="https://dx.doi.org/10.1105/tpc.011015">10.1105/tpc.011015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12724546">12724546</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pichersky</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">The complete functional characterisation of the terpene synthase family in tomato</span><span class="source" tagx="source" title="source">New Phytol</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">226</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">1341</span><span class="lpage" tagx="lpage" title="lpage">1360</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/nph.16431">10.1111/nph.16431</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31943222">31943222</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borghi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">D-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of a monoterpene synthase gene from flowers of <i>Camelina sativa</i></span><span class="source" tagx="source" title="source">Planta.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">247</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">443</span><span class="lpage" tagx="lpage" title="lpage">457</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00425-017-2801-x">10.1007/s00425-017-2801-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29075872">29075872</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnee</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ¶llner</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gershenzon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Degenhardt</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">The maize gene <i>terpene synthase 1</i> encodes a sesquiterpene synthase catalyzing the formation of (<i>E</i>)-Î²-farnesene, (E)-nerolidol, and (<i>E</i>,<i>E</i>)-farnesol after herbivore damage</span><span class="source" tagx="source" title="source">Plant Physiol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">130</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">2049</span><span class="lpage" tagx="lpage" title="lpage">2060</span><span class="pub-id"><a href="https://dx.doi.org/10.1104/pp.008326">10.1104/pp.008326</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12481088">12481088</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mack</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gros</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burkhardt</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seifert</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Elicitors of tansy volatiles from cotton leafworm larval oral secretion</span><span class="source" tagx="source" title="source">Phytochemistry.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">96</span><span class="fpage" tagx="fpage" title="fpage">158</span><span class="lpage" tagx="lpage" title="lpage">169</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phytochem.2013.08.007">10.1016/j.phytochem.2013.08.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24011527">24011527</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mikoviny</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jud</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">D'Anna</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wisthaler</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">A new software tool for the analysis of high resolution PTR-TOF mass spectra</span><span class="source" tagx="source" title="source">Chemom Intell Lab Syst</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">127</span><span class="fpage" tagx="fpage" title="fpage">158</span><span class="lpage" tagx="lpage" title="lpage">165</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chemolab.2013.06.011">10.1016/j.chemolab.2013.06.011</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jud</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghirardo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antritter</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benz</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnitzler</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Sniffing fungi â€" phenotyping of volatile chemical diversity in <i>Trichoderma species</i></span><span class="source" tagx="source" title="source">New Phytol</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">227</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">244</span><span class="lpage" tagx="lpage" title="lpage">259</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/nph.16530">10.1111/nph.16530</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32155672">32155672</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucero</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Estell</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tellez</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fredrickson</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">A retention index calculator simplifies identification of plant volatile organic compounds</span><span class="source" tagx="source" title="source">Phytochem Anal</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">20</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">378</span><span class="lpage" tagx="lpage" title="lpage">384</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/pca.1137">10.1002/pca.1137</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19533596">19533596</a></span></span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grabherr</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yassour</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">JZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amit</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span><span class="source" tagx="source" title="source">Nat Biotechnol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">644</span><span class="lpage" tagx="lpage" title="lpage">652</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.1883">10.1038/nbt.1883</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a></span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashby</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bridger: a new framework for de novo transcriptome assembly using RNA-seq data</span><span class="source" tagx="source" title="source">Genome Biol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">30</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-015-0596-2">10.1186/s13059-015-0596-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25723335">25723335</a></span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papanicolaou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yassour</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grabherr</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blood</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowden</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis</span><span class="source" tagx="source" title="source">Nat Protoc</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">1494</span><span class="lpage" tagx="lpage" title="lpage">1512</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2013.084">10.1038/nprot.2013.084</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23845962">23845962</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCarthy</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smyth</span><span class="given-names" tagx="given-names" title="given-names">GK</span></span></span><span class="mixed-article-title" title="mixed-article-title">edgeR: a bioconductor package for differential expression analysis of digital gene expression data</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">26</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">139</span><span class="lpage" tagx="lpage" title="lpage">140</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp616">10.1093/bioinformatics/btp616</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308">19910308</a></span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span></span><span class="mixed-article-title" title="mixed-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">32</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">1792</span><span class="lpage" tagx="lpage" title="lpage">1797</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkh340">10.1093/nar/gkh340</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a></span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talavera</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castresana</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments</span><span class="source" tagx="source" title="source">Syst Biol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">56</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">564</span><span class="lpage" tagx="lpage" title="lpage">577</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/10635150701472164">10.1080/10635150701472164</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17654362">17654362</a></span></span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouveia-Oliveira</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sackett</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pedersen</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span></span><span class="mixed-article-title" title="mixed-article-title">MaxAlign: maximizing usable data in an alignment</span><span class="source" tagx="source" title="source">BMC Bioinform</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">312</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-8-312">10.1186/1471-2105-8-312</a></span></span></li> <li tag="ref"><a name="CR70" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">1312</span><span class="lpage" tagx="lpage" title="lpage">1313</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu033">10.1093/bioinformatics/btu033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623">24451623</a></span></span></li> <li tag="ref"><a name="CR71" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huson</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richter</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rausch</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dezulian</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rupp</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dendroscope: An interactive viewer for large phylogenetic trees</span><span class="source" tagx="source" title="source">BMC Bioinform</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">460</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-8-460">10.1186/1471-2105-8-460</a></span></span></li> <li tag="ref"><a name="CR72" /><span class="label" tagx="label" title="label">72.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Xia J, Wishart DS. Using MetaboAnalyst 3.0 for Comprehensive Metabolomics Data Analysis. Curr Protocols Bioinform. 2016;55(1):14.0.1â€".0.91.</span></li> <li tag="ref"><a name="CR73" /><span class="label" tagx="label" title="label">73.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Systat Software SJ, CA, USA. SigmaPlot Version 14. <a href="https://systatsoftware.com/products/sigmaplot/sigmaplot-how-to-cite-sigmaplot/">https://systatsoftware.com/products/sigmaplot/sigmaplot-how-to-cite-sigmaplot/</a>.</span></li> </ul> </div> </div>  </body></html>