<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">ACS Omega</span><span class="iso-abbrev" title="iso-abbrev">ACS Omega</span><span class="publisher-id" title="publisher-id">ao</span><span class="coden" title="coden">acsodf</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">ACS Omega</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2470-1343</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">American Chemical Society</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7726785">7726785</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1021/acsomega.0c04047">10.1021/acsomega.0c04047</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Antimicrobial Mechanism and Identification of the Proteins Mediated by Extracts from <i>Asphaltum punjabianum</i> and <i>Myrtus communis</i></div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aabed</span><span class="given-names" tagx="given-names" title="given-names">Kawther</span></span><a href="#aff1">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohammed</span><span class="given-names" tagx="given-names" title="given-names">Afrah E.</span></span><a href="#cor1">*</a><a href="#aff1">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benabdelkamel</span><span class="given-names" tagx="given-names" title="given-names">Hicham</span></span><a href="#aff2">â€¡</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masood</span><span class="given-names" tagx="given-names" title="given-names">Afshan</span></span><a href="#aff2">â€¡</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alfadda</span><span class="given-names" tagx="given-names" title="given-names">Assim A.</span></span><a href="#aff5">Â§</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alanazi</span><span class="given-names" tagx="given-names" title="given-names">Ibrahim O.</span></span><a href="#aff3">âˆ¥</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alnehmi</span><span class="given-names" tagx="given-names" title="given-names">Eman A.</span></span><a href="#aff4">âŠ¥</a></span><span class="citation_author_institution" id="aff1">[â€ ], <span class="institution" title="institution">Princess Nourah Bint Abdulrahman University</span><span class="country" tagx="country" title="country">Saudi Arabia</span></span><span class="citation_author_institution" id="aff2">[â€¡], <span class="institution" title="institution">King Saud University</span><span class="country" tagx="country" title="country">Saudi Arabia</span></span><span class="citation_author_institution" id="aff5">[Â§], <span class="institution" title="institution">King Saud University</span><span class="country" tagx="country" title="country">Saudi Arabia</span></span><span class="citation_author_institution" id="aff3">[âˆ¥], <span class="institution" title="institution">King Abdulaziz City for Science and Technology (KACST)</span><span class="country" tagx="country" title="country">Saudi Arabia</span></span><span class="citation_author_institution" id="aff4">[âŠ¥], <span class="institution" title="institution">King Saud University</span><span class="country" tagx="country" title="country">Saudi Arabia</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Email: <span class="email" tagx="email" title="email">AFAMohammed@pnu.edu.sa</span>.</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-12-12</span></span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">48</span><span class="fpage" tagx="fpage" title="fpage">31019</span><span class="lpage" tagx="lpage" title="lpage">31035</span><span class="history" title="history"><span class="received" title="received">received: 2020-08-21</span><span class="accepted" title="accepted">accepted: 2020-11-06</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">American Chemical Society</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article published under a Creative Commons Non-Commercial No Derivative Works (CC-BY-NC-ND) Attribution <a href="http://pubs.acs.org/page/policy/authorchoice_ccbyncnd_termsofuse.html">License</a>, which permits copying and redistribution of the article, and creation of adaptations, all for non-commercial purposes.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p content-type="toc-graphic"> </p><div class="graphic" id="ab-tgr1" /> <p /> <p><i>Myrtus communis</i> (â€œmyrtleâ€�) and <i>Asphaltum punjabianum</i> (â€œshilajeetâ€�) are a medicinal plant and a long-term-humified dead plant material, respectively. We studied their antibacterial and anticandidal activities against <i>Pseudomonas aeruginosa</i>, <i>Escherichia coli</i>, <i>Staphylococcus aureus</i>, and <i>Candida albicans</i>. The activities of the aqueous extracts of the studied materials were measured using agar-well diffusion methods. Furthermore, proteomic analysis of treated microbial cells was conducted to identify affected proteins. The results showed both antibacterial and anticandidal activities for the myrtle extract (ME), while the shilajeet extract (SE) showed antibacterial activity only. The highest antimicrobial activity was observed against <i>E. coli</i> among the microbes tested; therefore, it was taken as the model for the proteomic analysis to identify the antimicrobial mechanism of ME and SE using two-dimensional electrophoresis. Upregulation of expression of 42 proteins and downregulation of expression of 6 proteins were observed in <i>E. coli</i> treated with ME, whereas 12 upregulated and 104 downregulated proteins were detected in <i>E. coli</i> treated with SE, in comparison with the control. About 85% of identified expressed proteins were from the cytoplasm and 15% from microbial cell walls, indicating the penetration of extracts inside cells. A higher percentage of expressed proteins was recorded for enzymatic activity. Our findings suggest that the major targets of the antibacterial action were proteins involved in the outer membrane, oxidative stress, and metabolism. Our data might reveal new targets for antimicrobial agents.</p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="document-id-old-9" value="ao0c04047" /> <meta name="document-id-new-14" value="ao0c04047" /> <meta name="ccc-price" value="" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Proteomic investigations have increased knowledge and understanding of microbes at the molecular level. For instance, investigating proteins and regulation of their expression helps scientists to recognize how pathogenic microbes have adapted to the lethal dose of an antimicrobial agent.</p> <p>Infectious diseases remain the major cause of human death worldwide because of emergence of new pathogenic agents, pathogen transmission due to migration, and an increase in the resistance of pathogens to antibiotics.<sup><a href="#ref1">1</a></sup> Innovative antimicrobial and therapeutic agents are required immediately to mitigate and overcome infections by such pathogens, together with additional rapid and reliable analytical methods for describing resistant strains.</p> <p>Some plants have been employed as antimicrobial agents because of their medicinal properties. Medicinal plants for prevention/treatment of diseases (including treatment of infections) have been used in China, India, and the Near East for hundreds of years. Utilization of medicinal plants could also promote primary healthcare substantially in developing countries. Medicinal plants have enormous potential but have not been explored sufficiently.</p> <p>The capacity of compounds of the plant origin to treat and prevent diseases might be related to the different biomolecules present within them. These phytochemicals and active ingredients include phenolic compounds, flavonoids, tannins, and alkaloids. The phytochemicals from medicinal plants are noted for their different antimicrobial abilities. Therefore, such phytochemicals could be developed as antimicrobial drugs. Several scholars have evaluated the antimicrobial activities of various plant components in recent years.<sup><a href="#ref2">2</a>âˆ’<a href="#ref5">5</a></sup></p> <p>Antibiotics are sometimes associated with various adverse effects.<sup><a href="#ref6">6</a></sup> Development of drugs from plant sources and engagement of secondary metabolites with pharmacologic activity have become a â€œresearch hotspotâ€�.<sup><a href="#ref7">7</a></sup> Bacteria cannot develop resistance to these drugs readily. Assessment of the active ingredients isolated from plants has been done to discover new medications that could be utilized for the prevention and treatment of diseases.<sup><a href="#ref8">8</a></sup></p> <p><i>Asphaltum punjabianum</i> is known as â€œshilajeetâ€� and is removed from rocks in the Himalayas in India. It is a natural substance formed for centuries by the gradual decomposition of plants by the action of microorganisms. It is a form of mineral that drips from the cracks of rocks during hot weather. Many researchers have noted that shilajeet is most probably of the vegetable origin and dissimilar to â€œtar seepsâ€�.</p> <p>The common name of the medicinal plant <i>Myrtus communis</i> is â€œmyrtleâ€�. It is a species of flowering plants in the family Myrtaceae. It is an evergreen bush found in North Africa, western Asia, southern Europe, the Indian subcontinent, and Macaronesia.<sup><a href="#ref9">9</a></sup> The essential oil of myrtle may be helpful for the therapy of skin diseases resulting from microorganisms.<sup><a href="#ref5">5</a></sup> Few studies have focused on the antimicrobial ability of the essential oils of myrtle against pathogenic fungal and bacterial strains.</p> <p>We investigated the antibacterial and anticandidal activities of myrtle and shilajeet against some bacterial species (<i>Escherichia coli</i>, <i>Pseudomonas aeruginosa</i>, <i>Staphylococcus aureus</i>, and <i>Candida albicans)</i>. The mechanism of action of tested extracts on <i>E. coli</i> was investigated by proteomic analysis using two-dimensional (2D) gel electrophoresis.</p> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="antimicrobialactivityofextracts" title="sec"> <div class="title" tagx="title" title="title">
Antimicrobial Activity of Extracts</div> <p>Using an ecofriendly material to suppress microbial growth is a promising approach with no expected environmental impact and could be a great solution to treat microbes resistant to antibiotics. Different studies have shown the action of the myrtle extract (ME) and shilajeet extract (SE) against bacteria and fungi. Variation in the ability of extracts against the microbes studied was observed and indicated that <i>E. coli</i> was the most sensitive microbe (<a href="#fig1">Figures </a><a href="#fig1">1</a> and <a href="#fig2">2</a>). <i>E. coli</i> has also shown higher sensitivity than some Gram-positive microbes when the essential oil of myrtle was investigated.<sup><a href="#ref10">10</a></sup> The leaf extracts of myrtle have shown antibacterial and antifungal activities against some pathogenic bacterial and fungal strains.<sup><a href="#ref11">11</a></sup> Antifungal activities against <i>Bipolaris</i> species, <i>Alternaria</i> species, <i>Curvularia</i> species, <i>Fusarium</i> species, and <i>Helminthosporium</i> species have been noted for methanolic extracts of shilajeet.<sup><a href="#ref12">12</a></sup> However, the aqueous extract of shilajeet used in the present study showed no anticandidal activity, suggesting that its efficacy is highly dependent upon the extraction method and microbial species tested. The activity of an aqueous extract of myrtle leaves against test microbes was linked to its chemical composition (e.g., flavonols, terpineol, acetate, linalyl, linalool, cineol, and tannins).<sup><a href="#ref13">13</a>,<a href="#ref14">14</a></sup> Such components might be involved in different mechanisms against microbes, such as cell-wall and cytoplasmic-membrane degradation, alteration in fatty acids and phospholipids, impact on genetic materials, and protein translation.<sup><a href="#ref15">15</a></sup> Furthermore, the effect of SE could be related to its composition of benzoic acids and fulvic acids.<sup><a href="#ref16">16</a></sup> The latter are leading factors in the increase of membrane permeability, which enhances disturbances in cell osmolarity and, hence, cell lysis.<sup><a href="#ref17">17</a></sup></p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Antibacterial activity of aqueous extracts of <i>Asphaltum punjabianum</i> L. (shilajeet) against clinical pathogens (measured as the zone of growth inhibition in millimeter). Data are the mean Â± SD (<i>n</i> = 3 replicates).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Antibacterial activity of aqueous extracts of <i>Myrtus communis</i> against clinical pathogens (measured as the zone of growth inhibition in millimeter). Data are the mean Â± SE (<i>n</i> = 3 replicates).</p>   </div> <p>Commonly used antibiotics had activity against <i>E. coli</i> and <i>P. aeruginosa</i>, but no activity was noticed for bacteriocin (<a href="#fig3">Figure </a><a href="#fig3">3</a>). Compared with commonly used antibiotics, SE and ME showed 63.9% and 59.9% of ciprofloxacin activity against <i>E. coli</i> and &amp;gt;100% of tetracycline and cefixime activities against <i>E. coli</i>, respectively. Double the tetracycline activity was observed for SE and ME against <i>P. aeruginosa</i>. The efficacy of the extracts and antibiotics was assessed when they were combined. An antagonistic effect was observed against <i>E. coli</i> when SE was combined with tetracycline, but a clear effect was not observed when SE was combined with bacitracin, ciprofloxacin, or cefixime in relation to activity for the antibiotic alone. No activity of ciprofloxacin, tetracycline, or cefixime was observed against <i>P. aeruginosa</i>. However, when SE was combined with ciprofloxacin, high activity was observed (<a href="#fig4">Figure </a><a href="#fig4">4</a>). The combination of ME and tetracycline reduced the activity against <i>E. coli</i> compared with that observed with ME alone. However, upon combination with ciprofloxacin, higher activity was observed compared with that for ME, although it was lower than the antibiotic effect. With regard to <i>P. aeruginosa</i>, no activity was observed for ciprofloxacin or bacitracin, but when each was combined with ME, activity was clearly observed, but it was lower than that for ME alone. When tetracycline was combined with ME, the activity was higher than that for the antibiotic alone but lower than the activity of ME alone. No activity for bacitracin before and after combination with ME was noted (<a href="#fig5">Figure </a><a href="#fig5">5</a>). The essential oil of Myrtus in combination with each of the antibiotics polymixin B and ciprofloxacin showed a reduction in the antibioticâ€™s ability against <i>Acinetobacter baumannii</i> wound isolates.<sup><a href="#ref18">18</a></sup> Interestingly, ciprofloxacin and cefixime showed no activity against <i>P. aeruginosa,</i> but when extracts were added to the disks and then examined, a high antibacterial effect was observed.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Antibacterial activity of common antibiotics against clinical pathogens (measured as the zone of growth inhibition in millimeter). Data are the mean Â± SE (<i>n</i> = 3 replicates).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Antibacterial activity of common antibiotics against <i>E. coli</i> and <i>P. aeruginosa</i> in combination with aqueous extracts of shilajeet (antibiotic +). Data are the mean Â± SE (<i>n</i> = 3 replicates). Extract (EX), tetracycline (TE), ciprofloxacin (Cip), bacitracin (B), and cefixime (CFM).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Antibacterial activity of common antibiotics against <i>E. coli</i> and <i>P. aeruginosa</i> in combination with aqueous extracts of myrtle (antibiotic +). Data are the mean Â± SE (<i>n</i> = 3 replicates). Extract (EX), tetracycline (TE), ciprofloxacin (Cip), bacitracin (B), and cefixime (CFM).</p>   </div> </div> <div class="morphologyoftreatedbacteria" title="sec"> <div class="title" tagx="title" title="title">
Morphology of Treated Bacteria</div> <p>We tried to identify the possible mechanism of action of plant extracts against <i>P. aeruginosa</i> and <i>E. coli</i>. Hence, microbes were subjected to plant extracts, and then, after 2 h, scanning electron microscopy (SEM) was carried out to ascertain variations in cell morphology. Morphologic differences besides cell elongation were observed for <i>P. aeruginosa</i> and <i>E. coli</i> (<a href="#fig6">Figure </a><a href="#fig6">6</a>). Similar morphologic and membrane changes have been noted by SEM for <i>E. coli</i> and <i>S. aureus</i> when treated with a <i>Memecylon candidum</i> extract.<sup><a href="#ref19">19</a></sup> The antibacterial activity of plant extracts is incompletely understood but could be related to cellular oxidation due to reactive oxygen species (ROS) production because variation in cell morphology was observed. The change as cell enlargement appears from the increase in all dimensions might also be related to the increase in membrane permeability and the accumulation of fluids or influx of the plant extract inside the cell.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>SEM image for <i>P. aeruginosa</i> and <i>E. coli</i> treated and untreated controls. (A) SEM images of untreated <i>P. aeruginosa</i>. (B) SEM images of <i>P. aeruginosa</i> treated with an aqueous extract of shilajeet. (C) SEM images of <i>P. aeruginosa</i> treated with an aqueous extract of myrtle. (D) SEM images of untreated <i>E. coli</i>. (E) SEM images of treated <i>E. coli</i> with an aqueous extract of shilajeet. (F) <i>E. coli</i> with an aqueous extract of myrtle.</p>   </div> </div> </div> <div class="2d-digeandmalditof/tofms" title="sec"> <div class="title" tagx="title" title="title">
2D-DIGE and MALDI TOF/TOF MS</div> <p>Proteomic analysis of <i>E. coli</i> was carried out to discover the mechanism of action for the test extracts. 2D-difference gel electrophoresis (2D-DIGE) was employed to assess significant changes in protein abundance among <i>E. coli</i> treated with ME (<i>n</i> = 4), SE (<i>n</i> = 4), and the untreated control (<i>n</i> = 4). Representative profiles of fluorescent proteins using 2D-DIGE included the control labeled with Cy3 (<a href="#fig7">Figure </a><a href="#fig7">7</a>A), ME-treated microbes labeled with Cy5 (<a href="#fig7">Figure </a><a href="#fig7">7</a>B), SE-treated microbes labeled with Cy3 (<a href="#fig7">Figure </a><a href="#fig7">7</a>C), and the pooled internal control labeled with Cy2 (<a href="#fig7">Figure </a><a href="#fig7">7</a>D). The overlap 2D-DIGE comparison of Cy3/Cy5 of ME/control and SE/control Cy3/Cy5 is shown in <a href="#fig8">Figure </a><a href="#fig8">8</a>A,B, respectively. Upon mapping all spots on the gels, 1580 spots were identified. Of these, 172 were significantly different (<i>p</i> &amp;gt; 0.05 by ANOVA and a fold change â‰¥1.5) between the treatment group and control group (<a href="#fig9">Figure </a><a href="#fig9">9</a>). All gels reproduced spot patterns across them, resulting in alignment and further analyses. Cy2 labeling was used as an internal standard to allow normalization across the complete set of gels and for quantitative differential analysis of protein expression. Then, the 172 significant spots identified were excised manually from the preparative gel for protein identification using mass spectrometry (MS). Peptide mass fingerprinting (PMF) identified 119 out of the 172 protein spots, of which 81 spots were found to be unique protein sequences by matrix-assisted laser desorption/ionization-<i>time of flight mass</i> spectrometry (MALDI-TOF/MS) and were matched to entries in the SWISS-PROT database (<span class="uri" title="uri"><a href="http://www.uniprot.org/">www.uniprot.org/</a></span>) by Mascot with high confidence scores (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c04047/suppl_file/ao0c04047_si_001.pdf">Supporting Information S1, Supporting Information S2</a>). Proteins identified by PMFs had a sequence coverage ranging from 12 to 93%. Variants of the same protein were found at several locations on the gel at a few places (<a href="#tbl2">Table </a><a href="#tbl2">2</a>, <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c04047/suppl_file/ao0c04047_si_001.pdf">Supporting Information S2</a>, <a href="#fig9">Figure </a><a href="#fig9">9</a>). Among the 119 proteins identified, the expression of 42 proteins was upregulated and that of 6 proteins was downregulated in the ME-treated sample in comparison with that in the control; the expression of 12 proteins was upregulated and that of 104 proteins was downregulated in the SE-treated sample in comparison with the control group; the expression of 11 proteins was upregulated and that of 108 proteins was downregulated in the SE-treated sample in comparison with the ME-treated sample (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c04047/suppl_file/ao0c04047_si_001.pdf">Supporting Information S2</a>). The highest upregulated proteins were formate acetyltransferase 1 and CTP synthase in the ME-treated sample compared with the control; elongation factor Tu 2 and the DNA-binding protein HU-Î± in the SE-treated sample compared with the control; and the DNA-binding protein HU-Î± and POS ribosomal protein L10 in the SE-treated sample compared with the ME-treated sample. The proteins for which the expression was decreased were elongation factor Tu 2 and POS ribosomal protein L10 in the ME-treated sample compared with the control; 305 ribosomal protein S1 and elongation factor Ts in the SE-treated sample compared with the control; and formate acetyltransferase 1 and CTP synthase in the SE-treated sample compared with the ME-treated sample. A complete list of upregulated and downregulated proteins is provided in the <a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c04047/suppl_file/ao0c04047_si_001.pdf">Supporting Information</a> (S1 and S2). Among the identified proteins, proteins including adenosine triphosphate (ATP) synthase subunit beta, isocitrate dehydrogenase, and outer-membrane protein A were found in more than one spot on gels. These effects could be due to post-translational modifications, cleavage by enzymes, or the presence of different protein species.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Representative fluorescence protein profiles of 2D-DIGE containing (A) control labeled with Cy3, (B) myrtle-treated samples labeled with Cy5, (C) shilajeet-treated samples labeled with Cy3, and (D) pooled internal control labeled with Cy2.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <p>Representative overlay of Cy3/Cy5/Cy2 images of (A) myrtle-treated/control and (B) shilajeet-treated/control. Images were captured using a Typhoon 9400 system in the variable mode.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9</span></a></div>   <p>Representative image of protein spots from <i>E. coli</i> samples. Numbered spots indicate those that were identified to be differentially expressed (over 1.5-fold change, <i>p</i> &amp;lt; 0.05) and identified with MALDI-TOF/TOF.</p>   </div> <div class="principalcomponentanalysis,clusteranalysis,andheatmaps" title="sec"> <div class="title" tagx="title" title="title">
Principal component analysis, Cluster Analysis, and Heatmaps</div> <p>Principal component analysis (PCA) carried out on all 172 spot features demonstrated significant (<i>p</i> &amp;lt; 0.05 by ANOVA) changes in abundance, as identified by MS. Also, PCA revealed that the three groups clustered markedly from one another based on different proteins, with 82% as the cutoff score (<a href="#fig10">Figure </a><a href="#fig10">10</a>). Clusters of expression patterns were exhibited by differentially abundant spots based on hierarchical clustering analysis (<a href="#fig11">Figure </a><a href="#fig11">11</a>A,B). The clustering pattern showed that the change in protein intensity for selected spots between ME, SE, and the control sample was significantly different. A heatmap was generated using all the 119 significant proteins identified by MS. The heatmap (<a href="#fig12">Figure </a><a href="#fig12">12</a>) showed that most of the 119 identified proteins had upregulated expression patterns among the ME-treated and control samples when compared with the SE-treated sample, as indicated by shades of red for high expression or green for low expression.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 10</span></a></div>   <p>PCA plot of the two first principal components. Both together explained 82% of the variability of selected spots. Colored dots and numbers are the representation of gels and spots, respectively [treatment 1 (ME) and treatment (SE)].</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 11</span></a></div>   <p>Expression profiles separated into clusters of expression patterns, indicating the number of spots for each cluster.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 12</span></a></div>   <p>Heatmap representation of the differentially expressed protein spots from the control, treatment 1 (ME), and treatment 2 (SE).</p>   </div> </div> <div class="bioinformaticanalysis:functionalclassificationofproteins" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatic Analysis: Functional Classification of Proteins</div> <p>Bioinformatic analysis using STRING v11.0 provided the interaction network of the differently expressed proteins (<a href="#fig14">Figure </a><a href="#fig14">14</a>). The protein analysis through evolutionary relationships (PANTHER) system was used for the classification of identified proteins according to their molecular function (<a href="#fig13">Figure </a><a href="#fig13">13</a>A) and location (<a href="#fig13">Figure </a><a href="#fig13">13</a>B). The functional category showed that most of the differentially expressed proteins identified were transcriptional regulatory proteins (52%), followed by binding proteins (33%). Also, 85% of the identified proteins were located in the cytoplasm.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 13</span></a></div>   <p>Comparative depiction (%) of identified proteins categorized into groups according to their molecular function A and location B using the PANTHER classification system (<span class="uri" title="uri"><a href="http://www.pantherdb.org">www.pantherdb.org</a></span>).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 14</span></a></div>   <p>Proteinâ€"protein interaction network of the differentially expressed proteins between the control, ME-treated <i>E. coli</i>, and SE-treated <i>E. coli</i> using STRING v11.0 (<span class="uri" title="uri"><a href="https://string-db.org/">https://string-db.org/</a></span>). Many lines show a higher number of interactions, and a single line indicates one interaction.</p>   </div> <p>Proteins were separated on IPG strips (pH 3â€"11) in the first-dimension electrophoresis, followed by 12.5% PAGE in the second-dimension electrophoresis. Images were captured using a Typhoon 9400 system in the variable mode.</p> <p>Each line represents the standardized abundance of a spot across all gels and belongs to one of the clusters generated by hierarchical cluster analysis. (A) Spots with increased abundance indicate the 11 proteins upregulated in SE-treated samples in comparison with the control and ME-treated samples. (B) Spots with reduced abundance indicate the 108 downregulated proteins in SE-treated samples in comparison with control and ME-treated samples.</p> <p>Each column shows a different group of the study, and the rows show single-spot proteins. The increase and decrease in the abundance of spots are indicated based on a relative scale (âˆ’1 to 1), shown from red to green. Dark boxes show groups of spots with similar changes in abundance.</p> <div class="detailedproteomicanalysisofe.coliinresponsetoextractstested" title="sec"> <div class="title" tagx="title" title="title">
Detailed Proteomic Analysis of <i>E. coli</i> in Response to Extracts Tested</div> <p>About 119 different proteins involved in different molecular and biological functions were affected in <i>E. coli</i> in response to treatment with ME and SE. <a href="#tbl1">Table </a><a href="#tbl1">1</a> shows the identified proteins involved in response to extracts, which are reviewed below.</p> <div class="table-wrap_UNKNOWN" id="tbl1" position="float"><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Stress Response-Related Proteins<a href="#t1fn1">a</a></div>  <table frame="hsides" rules="groups" border="0"> <colgroup> <col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="char" char="." /></colgroup> <colgroup><col align="left" /></colgroup>  <thead> <tr> <th style="border:none;" align="center">sl no.</th> <th style="border:none;" align="center">accession no.</th> <th style="border:none;" align="center">protein name</th> <th style="border:none;" align="center">MASCOT ID</th> <th style="border:none;" align="center">ANOVA <i>p</i>-value</th> <th style="border:none;" align="center">fold T1/C</th> <th style="border:none;" align="center">EXP T1/C</th> <th style="border:none;" align="center" char=".">fold T2/C</th> <th style="border:none;" align="center">EXP T2/C</th> </tr> </thead> <tbody> <tr> <td style="border:none;" align="left">1</td> <td style="border:none;" align="left">P63284</td> <td style="border:none;" align="left">chaperone protein ClpB</td> <td style="border:none;" align="left">CLPB_ECOLI</td> <td style="border:none;" align="left">2.27Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.56</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">2</td> <td style="border:none;" align="left">P63284</td> <td style="border:none;" align="left">chaperone protein ClpB</td> <td style="border:none;" align="left">CLPB_ECOLI</td> <td style="border:none;" align="left">0.001</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">â€"2.19</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">3</td> <td style="border:none;" align="left">P0ACF0</td> <td style="border:none;" align="left">DNA-binding protein HU-Î±</td> <td style="border:none;" align="left">DBHA_ECOLI</td> <td style="border:none;" align="left">6.41Â Ã—Â 10<sup>â€"6</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">7.73</td> <td style="border:none;" align="left">UP</td> </tr> <tr> <td style="border:none;" align="left">4</td> <td style="border:none;" align="left">P0ACF8</td> <td style="border:none;" align="left">DNA-binding protein H-NS</td> <td style="border:none;" align="left">HNS_ECOLI</td> <td style="border:none;" align="left">0.01</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">â€"2.022</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">5</td> <td style="border:none;" align="left">P05055</td> <td style="border:none;" align="left">polyribonucleotide nucleotidyltransferase</td> <td style="border:none;" align="left">PNP_ECOLI</td> <td style="border:none;" align="left">3.56Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"3.10</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">6</td> <td style="border:none;" align="left">P0ACA3</td> <td style="border:none;" align="left">stringent starvation proteinÂ A</td> <td style="border:none;" align="left">SSPA_ECOLI</td> <td style="border:none;" align="left">0.006</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">â€"2.19</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">7</td> <td style="border:none;" align="left">P35340</td> <td style="border:none;" align="left">alkyl hydroperoxide reductase subunitÂ F</td> <td style="border:none;" align="left">AHPF_ECOLI</td> <td style="border:none;" align="left">0.017</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">â€"1.56</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">8</td> <td style="border:none;" align="left">P35340</td> <td style="border:none;" align="left">alkyl hydroperoxide reductase subunitÂ F</td> <td style="border:none;" align="left">AHPF_ECOLI</td> <td style="border:none;" align="left">0.023</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">DOWN</td> <td style="border:none;" align="char" char=".">â€"2.665</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">9</td> <td style="border:none;" align="left">P0AGD3</td> <td style="border:none;" align="left">superoxide dismutase [Fe]</td> <td style="border:none;" align="left">SODF_ECOLI</td> <td style="border:none;" align="left">0.033</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"1.52</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">10</td> <td style="border:none;" align="left">P13029</td> <td style="border:none;" align="left">catalase-peroxidase</td> <td style="border:none;" align="left">KATG_ECOLI</td> <td style="border:none;" align="left">0.006</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.32</td> <td style="border:none;" align="left">DOWN</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">a</span><p>T1= <i>E. coli</i> treated with ME, T2 = <i>E. coli</i> treated with SE, and C = control.</p> </div> </div> </div> <div class="oxidativestress-relatedproteins" title="sec"> <div class="title" tagx="title" title="title">
Oxidative Stress-Related Proteins</div> <p>Nine stress response-related proteins were identified. Of these, the expression of four was upregulated and the expression of one was downregulated in <i>E. coli</i> treated with ME, while others were not affected. The expression of nine proteins was downregulated and the expression of one was downregulated in <i>E. coli</i> treated with SE (<a href="#tbl1">Table </a><a href="#tbl1">1</a>). The direct interaction between plant extracts and their chemical compositions with the membranes of bacteria has been documented and leads to damage to cell components due to ROS generation as a result of oxidative stress.<sup><a href="#ref20">20</a>âˆ’<a href="#ref22">22</a></sup> Stress conditions (e.g., antimicrobial treatment) can lead to changes in protein expression.<sup><a href="#ref23">23</a></sup> Therefore, the expression of some proteins was upregulated, which might have been an attempt to overcome the stress conditions. An adaptive response appears following some antimicrobial applications [e.g., nanoparticles (NPs)], which leads to membrane damage and ROS production.<sup><a href="#ref24">24</a></sup> The level of the chaperone protein ClpB, polyribonucleotide nucleotidyl transferase, stringent starvation protein A, alkyl hydroperoxide reductase subunit F, and superoxide dismutase (SOD) was increased and might have been a defensive mechanism against such antimicrobial agents. Bacteria can produce some antioxidant enzymes for detoxification and maintaining cell growth.<sup><a href="#ref25">25</a></sup> This phenomenon might explain the increased expression of SOD in <i>E. coli</i> treated with ME, but in <i>E. coli</i> treated with SE, the SOD expression was downregulated. The SOD expression has been shown to be upregulated as a response to stress conditions, such as high temperature, in <i>E. coli</i>.<sup><a href="#ref26">26</a></sup> Furthermore, the combined activity of catalase and peroxidase is an attempt to degrade H<sub>2</sub>O<sub>2</sub> that might be formed in <i>E. coli</i> in response to stress conditions.<sup><a href="#ref27">27</a></sup> In <i>E. coli</i> treated with ME, increased catalase activity and increased peroxidase activity were noted in contrast with <i>E. coli</i> treated with SE. Furthermore, DNA-binding proteins can â€œwrapâ€� and stabilize DNA and, hence, protect them from denaturation as a response to stress conditions.<sup><a href="#ref28">28</a></sup> Such a protein was upregulated, indicating that its expression might be a tendency of the treated microbes to protect their DNA from being damaged as a result of ROS enhanced by extract application. The expression of alkyl hydroperoxide reductase subunit F has been shown to be upregulated in <i>Salmonella typhimurium</i> and <i>E. coli</i> as a response to oxidative stress.<sup><a href="#ref29">29</a></sup> Different patterns of protein expression were observed in our study, indicating different responses for <i>E. coli</i> treated with different extracts. Such a response indicates that different biological components present in the different test extracts might contribute to the different antimicrobial effects. The expressions of some stress-related proteins were not changed in <i>E. coli</i> treated with ME. These data suggested that ROS generation was not enhanced in substantial quantities, which helped the cell to regulate the expression of the responsible genes to react against ROS. Interestingly, the same proteins were identified from different locations, but different responses were observed, indicating that the effect was dependent upon the location.</p> </div> <div class="proteinsinvolvedintranscriptionandtranslation(proteinsynthesis)" title="sec"> <div class="title" tagx="title" title="title">
Proteins Involved in Transcription and Translation (Protein Synthesis)</div> <p>Under stress conditions such as antimicrobial agents, <i>E. coli</i> produces ROS, and many proteins are induced to help the cell adapt to such stress conditions. Because of changes in environmental conditions and stress, rapid changes in proteins occur to adjust cell development in response to unfavorable conditions.<sup><a href="#ref30">30</a></sup> In our study, 22 proteins involved in protein synthesis were detected showing different patterns, including 7 expressed proteins (5 upregulated and 2 downregulated) in <i>E. coli</i> treated with ME. However, all 22 proteins were expressed (7 upregulated and 15 downregulated) in <i>E. coli</i> treated with SE. Such expressed proteins were responsible for protein translation. The same proteins were detected from different locations and showed different expression patterns (<a href="#tbl2">Table </a><a href="#tbl2">2</a>).</p> <div class="table-wrap_UNKNOWN" id="tbl2" position="float"><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Translation- and Transcription (Protein Synthesis)-Related Proteins</div>  <table frame="hsides" rules="groups" border="0"> <colgroup> <col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="char" char="." /></colgroup> <colgroup><col align="left" /></colgroup>  <thead> <tr> <th style="border:none;" align="center">sl no.</th> <th style="border:none;" align="center">accession no.</th> <th style="border:none;" align="center">protein name</th> <th style="border:none;" align="center">MASCOT ID</th> <th style="border:none;" align="center">ANOVA <i>p</i>-value</th> <th style="border:none;" align="center">fold T1/C</th> <th style="border:none;" align="center">EXP T1/C</th> <th style="border:none;" align="center" char=".">fold T2/C</th> <th style="border:none;" align="center">EXP T2/C</th> </tr> </thead> <tbody> <tr> <td style="border:none;" align="left">1</td> <td style="border:none;" align="left">P0AG67</td> <td style="border:none;" align="left">30S ribosomal proteinÂ S1</td> <td style="border:none;" align="left">RS1_ECOLI</td> <td style="border:none;" align="left">0.012</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">â€"3.1</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">2</td> <td style="border:none;" align="left">P0AG67</td> <td style="border:none;" align="left">30S ribosomal proteinÂ S1</td> <td style="border:none;" align="left">RS1_ECOLI</td> <td style="border:none;" align="left">1.27Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"4.23</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">3</td> <td style="border:none;" align="left">P0A7R5</td> <td style="border:none;" align="left">30S ribosomal proteinÂ S10</td> <td style="border:none;" align="left">RS10_ECOLI</td> <td style="border:none;" align="left">6.22Â Ã—Â 10<sup>â€"5</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">3.66</td> <td style="border:none;" align="left">UP</td> </tr> <tr> <td style="border:none;" align="left">4</td> <td style="border:none;" align="left">P0A7V3</td> <td style="border:none;" align="left">30S ribosomal proteinÂ S3</td> <td style="border:none;" align="left">RS3_ECOLI</td> <td style="border:none;" align="left">2.45Â Ã—Â 10<sup>â€"6</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.89</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">5</td> <td style="border:none;" align="left">P0A7V8</td> <td style="border:none;" align="left">30S ribosomal proteinÂ S4Â (<b>2</b>)</td> <td style="border:none;" align="left">RS4_ECOLI</td> <td style="border:none;" align="left">0.002</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.51</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">6</td> <td style="border:none;" align="left">P02359</td> <td style="border:none;" align="left">30S ribosomal proteinÂ S7Â (<b>2</b>)</td> <td style="border:none;" align="left">RS7_ECOLI</td> <td style="border:none;" align="left">6.70Â Ã—Â 10<sup>â€"5</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.43</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left"><b>7</b></td> <td style="border:none;" align="left">P02359</td> <td style="border:none;" align="left">30S ribosomal proteinÂ S7Â (<b>2</b>)</td> <td style="border:none;" align="left">RS7_ECOLI</td> <td style="border:none;" align="left">0.016</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">â€"1.8</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">8</td> <td style="border:none;" align="left">P0A7J3</td> <td style="border:none;" align="left">50S ribosomal proteinÂ L10</td> <td style="border:none;" align="left">RL10_ECOLI</td> <td style="border:none;" align="left">3.56Â Ã—Â 10<sup>â€"8</sup></td> <td style="border:none;" align="left">â€"1.72</td> <td style="border:none;" align="left">DOWN</td> <td style="border:none;" align="char" char=".">4.86</td> <td style="border:none;" align="left">UP (12)</td> </tr> <tr> <td style="border:none;" align="left">9</td> <td style="border:none;" align="left">P0A7K6</td> <td style="border:none;" align="left">50S ribosomal proteinÂ L19</td> <td style="border:none;" align="left">RL19_ECOLI</td> <td style="border:none;" align="left">1.89Â Ã—Â 10<sup>â€"7</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">4.79</td> <td style="border:none;" align="left">UP</td> </tr> <tr> <td style="border:none;" align="left">10</td> <td style="border:none;" align="left">P60422</td> <td style="border:none;" align="left">50S ribosomal proteinÂ L2</td> <td style="border:none;" align="left">RL2_ECOLI</td> <td style="border:none;" align="left">0.012</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"1.82</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">11</td> <td style="border:none;" align="left">P0A7K2</td> <td style="border:none;" align="left">50S ribosomal proteinÂ L7/L12</td> <td style="border:none;" align="left">RL7_ECOLI</td> <td style="border:none;" align="left">0.005</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.08</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">12</td> <td style="border:none;" align="left">P0A7R1</td> <td style="border:none;" align="left">50S ribosomal proteinÂ L9</td> <td style="border:none;" align="left">RL9_ECOLI</td> <td style="border:none;" align="left">0.005</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"1.71</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">13</td> <td style="border:none;" align="left">P0A7Z4</td> <td style="border:none;" align="left">DNA-directed RNA polymerase subunitÂ Î±Â (<b>2</b>)</td> <td style="border:none;" align="left">RPOA_ECOLI</td> <td style="border:none;" align="left">0.011</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">2.03</td> <td style="border:none;" align="left">UP</td> </tr> <tr> <td style="border:none;" align="left">14</td> <td style="border:none;" align="left">P0A6M8</td> <td style="border:none;" align="left">elongation factorÂ GÂ (<b>3</b>)</td> <td style="border:none;" align="left">EFG_ECOLI</td> <td style="border:none;" align="left">0.006</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">â€"2.36</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">15</td> <td style="border:none;" align="left">P0A6M8</td> <td style="border:none;" align="left">elongation factorÂ GÂ (<b>3</b>)</td> <td style="border:none;" align="left">EFG_ECOLI</td> <td style="border:none;" align="left">0.009</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"1.5</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">16</td> <td style="border:none;" align="left">P0CE47</td> <td style="border:none;" align="left">elongation factorÂ TuÂ 1Â (<b>5</b>)</td> <td style="border:none;" align="left">EFTU1_ECOLI</td> <td style="border:none;" align="left">3.77Â Ã—Â 10<sup>â€"5</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.27</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">17</td> <td style="border:none;" align="left">P0CE47</td> <td style="border:none;" align="left">elongation factorÂ TuÂ 1Â (<b>3</b>)</td> <td style="border:none;" align="left">EFTU1_ECOLI</td> <td style="border:none;" align="left">2.46Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">â€"2.68</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">18</td> <td style="border:none;" align="left">P0CE48</td> <td style="border:none;" align="left">elongation factorÂ TuÂ 2Â (<b>2</b>)</td> <td style="border:none;" align="left">EFTU2_ECOLI</td> <td style="border:none;" align="left">6.54Â Ã—Â 10<sup>â€"5</sup></td> <td style="border:none;" align="left">â€"2.93</td> <td style="border:none;" align="left">DOWN</td> <td style="border:none;" align="char" char=".">7.83</td> <td style="border:none;" align="left">UP</td> </tr> <tr> <td style="border:none;" align="left">19</td> <td style="border:none;" align="left">P0A6P1</td> <td style="border:none;" align="left">elongation factorÂ TsÂ (<b>3</b>)</td> <td style="border:none;" align="left">EFTS_ECOLI</td> <td style="border:none;" align="left">0.011</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"1.83</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">20</td> <td style="border:none;" align="left">P0A800</td> <td style="border:none;" align="left">RNA polymerase-binding transcription factor DksA</td> <td style="border:none;" align="left">RPOZ_ECOLI</td> <td style="border:none;" align="left">0.005</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">2.41</td> <td style="border:none;" align="left">UP</td> </tr> <tr> <td style="border:none;" align="left">21</td> <td style="border:none;" align="left">P0ABS1</td> <td style="border:none;" align="left">RNA polymerase-binding transcription factor DksA</td> <td style="border:none;" align="left">DKSA_ECOL</td> <td style="border:none;" align="left">0.006</td> <td style="border:none;" align="left">1.56</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">2.7</td> <td style="border:none;" align="left">UP</td> </tr> <tr> <td style="border:none;" align="left">22</td> <td style="border:none;" align="left">P0A6F5</td> <td style="border:none;" align="left">60Â kDa chaperonin</td> <td style="border:none;" align="left">CH60_ECOLI</td> <td style="border:none;" align="left">1.30Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.78</td> <td style="border:none;" align="left">DOWN</td> </tr> </tbody> </table> </div> <p>The expression of five ribosomal proteins was increased: elongation factor G, T1, and S1; S7 from the ribosomal small subunit; and RNA polymerase-binding transcription factor DksA. Conversely, the expression of 50S ribosomal protein L10 and elongation factor Tu 2 was downregulated in <i>E. coli</i> treated with ME. The expression of some 30S and 50S ribosomal proteins besides elongation factor G and DNA-directed RNA polymerase subunit alpha was downregulated, suggesting suppression of protein synthesis as a response to SE. The expression of protein L7/L12 from 50 ribosomal proteins has been shown to be downregulated in bacteria treated with the antimicrobial agent Ag-MNP.<sup><a href="#ref31">31</a></sup></p> <p>Another translation elongation protein in <i>E. coli</i> is elongation factor G, which is highly sensitive to oxidation.<sup><a href="#ref32">32</a></sup> The expression of elongation factor G was increased in <i>E. coli</i> subjected to ME in contrast with <i>E. coli</i> treated with SE. Oxidation might show that ROS could enhance <i>E. coli</i> to produce elongation factor G because the latter is highly sensitive to oxidation and carbonylation in stressed <i>E. coli</i> after H<sub>2</sub>O<sub>2</sub> exposure.<sup><a href="#ref33">33</a>,<a href="#ref34">34</a></sup> However, different trends were detected for <i>E. coli</i> subjected to both extracts. A decrease in translational elongation processes in <i>E. coli</i> under hyperosmotic conditions has been observed, which suggests a reduction in protein synthesis under stress conditions.<sup><a href="#ref35">35</a></sup> Moreover, Tu is a component of the elongation of peptides.<sup><a href="#ref36">36</a></sup> The expression of the elongation factors Ts and T1 was upregulated and the expression of the elongation factor T2 was downregulated in <i>E. coli</i> treated by ME and SE, respectively. In growth media that contain an excess of zinc, the expression of the elongation factor Tu was downregulated as a stress-response mechanism.<sup><a href="#ref37">37</a></sup> The expression of the RNA polymerase-binding transcription factor DksA was upregulated in <i>E. coli</i> treated with both extracts separately, which might indicate that transcription was favored. An identical observation was recorded when <i>E. coli</i> was tested under alkaline conditions by Gonzales-Siles and colleagues.<sup><a href="#ref38">38</a></sup> The expression of 16 gene expression-related proteins was affected by treatment with SE (mostly downregulated), which suggested that the activity of the extract against <i>E. coli</i> could be related to protein suppression, which might not be the case after treatment with ME. Furthermore, FU reported that a 60-kDa chaperonin enhanced polypeptide folding and refolding of damaged proteins<sup><a href="#ref39">39</a></sup> and its expression was upregulated under different stress conditions.<sup><a href="#ref40">40</a></sup></p> </div> <div class="proteinsinvolvedinthetransportofelectronsandprotons(atpsynthesis)" title="sec"> <div class="title" tagx="title" title="title">
Proteins Involved in the Transport of Electrons and Protons (ATP Synthesis)</div> <p>The expression of the four proteins identified to be involved in ATP synthesis was downregulated in SE-treated <i>E. coli</i>, and no effect was noted in ME-treated microbes (<a href="#tbl3">Table </a><a href="#tbl3">3</a>). Three subunits of ATP synthase are important in oxidative respiration because they aid H<sup>+</sup> transport to the cytosol.<sup><a href="#ref41">41</a></sup> However, the expression of these three subunits was downregulated, suggesting that the intracellular level of ATP was reduced in response to treatment with SE. However, upregulation of the expression of ATP synthase in <i>E. coli</i> exposed to an alkaline pH and cadmium has been observed,<sup><a href="#ref38">38</a>,<a href="#ref42">42</a></sup> indicating that such expression is specific to stress.</p> <div class="table-wrap_UNKNOWN" id="tbl3" position="float"><span class="label" tagx="label" title="label">Table 3</span> <div class="title" tagx="title" title="title">
Proteins Involved in the Transport of Protons and Electrons</div>  <table frame="hsides" rules="groups" border="0"> <colgroup> <col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="char" char="." /></colgroup> <colgroup><col align="left" /></colgroup>  <thead> <tr> <th style="border:none;" align="center">sl no.</th> <th style="border:none;" align="center">accession no.</th> <th style="border:none;" align="center">protein name</th> <th style="border:none;" align="center">MASCOT ID</th> <th style="border:none;" align="center">ANOVA <i>p</i>-value</th> <th style="border:none;" align="center">fold T1/C</th> <th style="border:none;" align="center">EXP T1/C</th> <th style="border:none;" align="center" char=".">fold T2/C</th> <th style="border:none;" align="center">EXP T2/C</th> </tr> </thead> <tbody> <tr> <td style="border:none;" align="left">1</td> <td style="border:none;" align="left">P0ABB0</td> <td style="border:none;" align="left">ATP synthase subunitÂ Î±</td> <td style="border:none;" align="left">ATPA_ECOLI</td> <td style="border:none;" align="left">8.26Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"1.72</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">2</td> <td style="border:none;" align="left">P0ABB4</td> <td style="border:none;" align="left">ATP synthase subunitÂ Î²</td> <td style="border:none;" align="left">ATPB_ECOLI</td> <td style="border:none;" align="left">4.42Â Ã—Â 10<sup>â€"6</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.35</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">3</td> <td style="border:none;" align="left">P0ABB4</td> <td style="border:none;" align="left">ATP synthase subunitÂ Î²</td> <td style="border:none;" align="left">ATPB_ECOLI</td> <td style="border:none;" align="left">0.006</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"3.15</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">4</td> <td style="border:none;" align="left">P0AEZ3</td> <td style="border:none;" align="left">septum site-determining protein MinD</td> <td style="border:none;" align="left">MIND_ECOLI</td> <td style="border:none;" align="left">8.75Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.28</td> <td style="border:none;" align="left">DOWN</td> </tr> </tbody> </table> </div> </div> <div class="proteinsinvolvedinthesugarcatabolismandthetcacycle" title="sec"> <div class="title" tagx="title" title="title">
Proteins Involved in the Sugar Catabolism and the TCA Cycle</div> <p>A total of 29 differentially expressed proteins involved in the sugar metabolism and tricarboxylic acid (TCA) were identified by 2D-DIGE (<a href="#tbl4">Table </a><a href="#tbl4">4</a>), of which 15 proteins were expressed (13 were upregulated and 2 were downregulated) in ME-treated <i>E. coli</i>, and 28 proteins were expressed in <i>E. coli</i> treated with SE (26 downregulated and 2 upregulated). The expression of some TCA- and glycolytic-pathway enzymes was upregulated in <i>E.coli</i> treated with ME, and some enzymes related to the glycerol metabolism, glycolysis, or gluconeogenesis had a significant role in antimicrobial stress. One of the most sensitive pathways to ROS response is the TCA cycle.<sup><a href="#ref43">43</a></sup> TCA produces and scavenges ROS under oxidative stress, which might explain ROS upregulation under antimicrobial stress in the present study, thereby suggesting that some TCA pathways have a vital role in managing oxidative stress.<sup><a href="#ref44">44</a></sup> Upregulation of the expression of glycolysis enzymes and downregulation of the expression of TCA enzymes have been noted in <i>E. coli</i> under the stress of iron limitation.<sup><a href="#ref45">45</a></sup> Aconitate hydratase B is a TCA enzyme that appeared in three locations and showed different expression patterns in the present study. The expression of aconitate hydratase B was downregulated in all three locations in <i>E. coli</i> treated with SE but was upregulated in one location after treatment with ME. Aconitate hydratase B is sensitive, and SE appeared to damage it. The expression of aconitate hydratase B is downregulated in <i>E. coli</i> suffering from cadmium stress.<sup><a href="#ref42">42</a></sup> Glycerol kinase is an enzyme involved in glycerol uptake and lipolysis, and its expression was upregulated and downregulated in <i>E. coli</i> treated with ME and SE, respectively. Glycerol kinase participates in the energy metabolism, and the downregulation we observed might have occurred to reduce cell-energy consumption under stress conditions to conserve energy for cell survival. Pyruvate dehydrogenase is responsible for acetate formation, and its expression was upregulated after ME exposure in contrast with <i>E. coli</i> treated with SE. Downregulation of the expression of pyruvate dehydrogenase has also been reported for <i>E. coli</i> tested in an alkaline environment.<sup><a href="#ref38">38</a></sup> The expression of an enzyme involved in the electron-transport chain and TCA cycle, succinate dehydrogenase, was upregulated in <i>E. coli</i> treated with ME but downregulated in <i>E. coli</i> treated with SE. However, the expression of succinate dehydrogenase is enhanced in <i>E. coli</i> under zinc stress.<sup><a href="#ref46">46</a></sup> Malate dehydrogenase is vital for control of oxidative stress. Its expression was downregulated in SE-treated <i>E. coli</i> but was not affected significantly by ME. Several mechanisms are used by bacteria for carbohydrate uptake.<sup><a href="#ref47">47</a></sup> The important transport system for carbohydrates is the phosphotransferase system (PTS), and the enzyme involved in this system is phosphoenolpyruvate-protein phosphotransferase. The expression of the latter was not affected in <i>E. coli</i> treated with ME, but in <i>E. coli</i> treated with SE, it was upregulated. The expression of the glucose-specific EIIA component and galactofuranose transporter ATP-binding protein YtfR was downregulated and upregulated, respectively, in <i>E. coli</i> treated with SE. The expression of phosphoenolpyruvate-protein phosphotransferase is downregulated in <i>E. coli</i> under alkaline conditions.<sup><a href="#ref38">38</a></sup> The proteins involved in the nonoxidative part of the pentose phosphate pathway, transaldolase B and transaldolase B1, responded differently in <i>E. coli</i> to ME treatment compared with that upon SE treatment, with downregulation being observed in the latter treatment. Downregulation of the transketolase expression has been recorded for <i>E. coli</i> under alkaline conditions.<sup><a href="#ref38">38</a></sup> It appears that ME enhanced the tryptophanase expression for the tryptophan catabolism; this was in contrast to SE, which downregulated the tryptophanase expression and revealed no vital role for tryptophanase in the antimicrobial response to SE.</p> <div class="table-wrap_UNKNOWN" id="tbl4" position="float"><span class="label" tagx="label" title="label">Table 4</span> <div class="title" tagx="title" title="title">
Proteins Involved in the Sugar Catabolism and the TCA Cycle</div>  <table frame="hsides" rules="groups" border="0"> <colgroup> <col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup>  <thead> <tr> <th style="border:none;" align="center">sl no.</th> <th style="border:none;" align="center">accession no.</th> <th style="border:none;" align="center">protein name</th> <th style="border:none;" align="center">MASCOT ID</th> <th style="border:none;" align="center">ANOVA <i>p</i>-value</th> <th style="border:none;" align="center">fold T1/C</th> <th style="border:none;" align="center">EXP T1/C</th> <th style="border:none;" align="center">fold T2/C</th> <th style="border:none;" align="center">EXP T2/C</th> </tr> </thead> <tbody> <tr> <td style="border:none;" align="left">1</td> <td style="border:none;" align="left">P36683</td> <td style="border:none;" align="left">aconitate hydrataseÂ BÂ (2)</td> <td style="border:none;" align="left">ACNB_ECOLI</td> <td style="border:none;" align="left">0.003</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.42</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">2</td> <td style="border:none;" align="left">P36683</td> <td style="border:none;" align="left">aconitate hydrataseÂ B</td> <td style="border:none;" align="left">ACNB_ECOLI</td> <td style="border:none;" align="left">0.024</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.89</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">3</td> <td style="border:none;" align="left">P22259</td> <td style="border:none;" align="left">phosphoenolpyruvate carboxykinase (ATP)</td> <td style="border:none;" align="left">PCKA_ECOLI</td> <td style="border:none;" align="left">1.91Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.14</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">4</td> <td style="border:none;" align="left">P22259</td> <td style="border:none;" align="left">phosphoenolpyruvate carboxykinase (ATP)</td> <td style="border:none;" align="left">PCKA_ECOLI</td> <td style="border:none;" align="left">3.66Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.77</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">5</td> <td style="border:none;" align="left">P08839</td> <td style="border:none;" align="left">phosphoenolpyruvate-protein phosphotransferase</td> <td style="border:none;" align="left">PT1_ECOLI</td> <td style="border:none;" align="left">0.009</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">5.18</td> <td style="border:none;" align="left">UP</td> </tr> <tr> <td style="border:none;" align="left">6</td> <td style="border:none;" align="left">P69783</td> <td style="border:none;" align="left">PTS system glucose-specific EIIA component</td> <td style="border:none;" align="left">PTGA_ECOLI</td> <td style="border:none;" align="left">0.013</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"1.5</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">7</td> <td style="border:none;" align="left">Q6BEX0</td> <td style="border:none;" align="left">galactofuranose transporter ATP-binding protein YtfR</td> <td style="border:none;" align="left">YTFR_ECOLI</td> <td style="border:none;" align="left">0.004</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">1.61</td> <td style="border:none;" align="left">UP</td> </tr> <tr> <td style="border:none;" align="left">8</td> <td style="border:none;" align="left">P09373</td> <td style="border:none;" align="left">formate acetyltransferaseÂ 1</td> <td style="border:none;" align="left">PFLB_ECOLI</td> <td style="border:none;" align="left">0.033</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">DOWN</td> <td style="border:none;" align="left">â€"1.86</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">9</td> <td style="border:none;" align="left">P09373</td> <td style="border:none;" align="left">formate acetyltransferaseÂ 1</td> <td style="border:none;" align="left">PFLB_ECOLI</td> <td style="border:none;" align="left">0.046</td> <td style="border:none;" align="left">3.48</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.53</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">10</td> <td style="border:none;" align="left">P00363</td> <td style="border:none;" align="left">fumarate reductase flavoprotein subunit</td> <td style="border:none;" align="left">FRDA_ECOLI</td> <td style="border:none;" align="left">3.49Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.56</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">11</td> <td style="border:none;" align="left">P0A9S5</td> <td style="border:none;" align="left">glycerol dehydrogenaseÂ (2)</td> <td style="border:none;" align="left">GLDA_ECOLI</td> <td style="border:none;" align="left">0.019</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"1.55</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">12</td> <td style="border:none;" align="left">P0A6F3</td> <td style="border:none;" align="left">glycerol kinase</td> <td style="border:none;" align="left">GLPK_ECOLI</td> <td style="border:none;" align="left">5.58Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">1.77</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.9</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">13</td> <td style="border:none;" align="left">P0A799</td> <td style="border:none;" align="left">phosphoglycerate kinase</td> <td style="border:none;" align="left">PGK_ECOLI</td> <td style="border:none;" align="left">0.002</td> <td style="border:none;" align="left">1.61</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.41</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">14</td> <td style="border:none;" align="left">P0AFG8</td> <td style="border:none;" align="left">pyruvate dehydrogenase E1Â component</td> <td style="border:none;" align="left">ODP1_ECOLI</td> <td style="border:none;" align="left">0.01</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.54</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">15</td> <td style="border:none;" align="left">P0AFG8</td> <td style="border:none;" align="left">pyruvate dehydrogenase E1Â component</td> <td style="border:none;" align="left">ODP1_ECOLI</td> <td style="border:none;" align="left">0.013</td> <td style="border:none;" align="left">1.69</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.5</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">16</td> <td style="border:none;" align="left">P0A8G3</td> <td style="border:none;" align="left">uronate isomerase</td> <td style="border:none;" align="left">UXAC_ECOLI</td> <td style="border:none;" align="left">2.55Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"3.08</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">17</td> <td style="border:none;" align="left">P0A8G3</td> <td style="border:none;" align="left">uronate isomerase</td> <td style="border:none;" align="left">UXAC_ECOLI</td> <td style="border:none;" align="left">0.011</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.5</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">18</td> <td style="border:none;" align="left">P38489</td> <td style="border:none;" align="left">oxygen-insensitive NAD(P)H nitroreductase</td> <td style="border:none;" align="left">NFSB_ECOLI</td> <td style="border:none;" align="left">0.013</td> <td style="border:none;" align="left">â€"1.5</td> <td style="border:none;" align="left">DOWN</td> <td style="border:none;" align="left">â€"2.27</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">19</td> <td style="border:none;" align="left">P08200</td> <td style="border:none;" align="left">isocitrate dehydrogenase [NADP]Â (3)</td> <td style="border:none;" align="left">IDH_ECOLI</td> <td style="border:none;" align="left">2.10Â Ã—Â 10<sup>â€"5</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"3.8</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">20</td> <td style="border:none;" align="left">P37440</td> <td style="border:none;" align="left">oxidoreductase UcpA</td> <td style="border:none;" align="left">UCPA_ECOLI</td> <td style="border:none;" align="left">0.041</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.36</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">21</td> <td style="border:none;" align="left">P0AC41</td> <td style="border:none;" align="left">succinate dehydrogenase flavoprotein subunit</td> <td style="border:none;" align="left">SDHA_ECOLI</td> <td style="border:none;" align="left">0.025</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.64</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">22</td> <td style="border:none;" align="left">P61889</td> <td style="border:none;" align="left">malate dehydrogenase</td> <td style="border:none;" align="left">MDH_ECOLI</td> <td style="border:none;" align="left">2.59Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"3.21</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">23</td> <td style="border:none;" align="left">P0A998</td> <td style="border:none;" align="left">bacterial non-heme ferritin</td> <td style="border:none;" align="left">FTNA_ECOLI</td> <td style="border:none;" align="left">0.035</td> <td style="border:none;" align="left">1.78</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.5</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">24</td> <td style="border:none;" align="left">P26616</td> <td style="border:none;" align="left">NAD-dependent malic enzymeÂ (2)</td> <td style="border:none;" align="left">MAO1_ECOLI</td> <td style="border:none;" align="left">0.003</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.4</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">25</td> <td style="border:none;" align="left">P23538</td> <td style="border:none;" align="left">phosphoenolpyruvate synthase</td> <td style="border:none;" align="left">PPSA_ECOLI</td> <td style="border:none;" align="left">0.012</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"1.85</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">26</td> <td style="border:none;" align="left">P23538</td> <td style="border:none;" align="left">phosphoenolpyruvate synthase</td> <td style="border:none;" align="left">PPSA_ECOLI</td> <td style="border:none;" align="left">0.017</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> </tr> <tr> <td style="border:none;" align="left">27</td> <td style="border:none;" align="left">P0A870</td> <td style="border:none;" align="left">transaldolaseÂ B</td> <td style="border:none;" align="left">TALB_ECOLI</td> <td style="border:none;" align="left">0.002</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.64</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">28</td> <td style="border:none;" align="left">P27302</td> <td style="border:none;" align="left">transketolaseÂ 1</td> <td style="border:none;" align="left">TKT1_ECOLI</td> <td style="border:none;" align="left">0.002</td> <td style="border:none;" align="left">1.8</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.41</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">29</td> <td style="border:none;" align="left">P0A853</td> <td style="border:none;" align="left">tryptophanase</td> <td style="border:none;" align="left">TNAA_ECOLI</td> <td style="border:none;" align="left">0.011</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.2</td> <td style="border:none;" align="left">DOWN</td> </tr> </tbody> </table> </div> </div> <div class="biosynthesisandtransferoflipidsandaminoacids" title="sec"> <div class="title" tagx="title" title="title">
Biosynthesis and Transfer of Lipids and Amino Acids</div> <p>A total of 16 differentially expressed proteins involved in the synthesis of organic compounds in cells were identified in treated <i>E. coli</i>. Of these, the expression of nine was upregulated in <i>E. coli</i> treated with ME; the expression of 12 was downregulated and the expression of 1 was upregulated in SE-treated <i>E. coli</i> (<a href="#tbl5">Table </a><a href="#tbl5">5</a>). Amino acids have a vital role as the building blocks of proteins. Under stress, it has been suggested that the expression of some proteins might be upregulated to enable protein adaptation. Upregulation of the expression of several enzymes involved in amino-acid synthesis has been documented for <i>E. coli</i> under heat stress.<sup><a href="#ref26">26</a></sup> The antimicrobial ability of plant extracts might enhance the production of ROS, which damage the macromolecules in microbial cells such as proteins, lipids, and DNA.<sup><a href="#ref20">20</a></sup> Downregulation of the expression of the enzymes involved in the synthesis of proteins, lipids, and DNA was noted in SE-treated microbes.</p> <div class="table-wrap_UNKNOWN" id="tbl5" position="float"><span class="label" tagx="label" title="label">Table 5</span> <div class="title" tagx="title" title="title">
Biosynthesis of Lipids and Amino Acids and Transfer of Proteins</div>  <table frame="hsides" rules="groups" border="0"> <colgroup> <col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup>  <thead> <tr> <th style="border:none;" align="center">sl no.</th> <th style="border:none;" align="center">accession no.</th> <th style="border:none;" align="center">protein name</th> <th style="border:none;" align="center">MASCOT ID</th> <th style="border:none;" align="center">ANOVA <i>p</i>-value</th> <th style="border:none;" align="center">fold T1/C</th> <th style="border:none;" align="center">EXP T1/C</th> <th style="border:none;" align="center">fold T2/C</th> <th style="border:none;" align="center">EXP T2/C</th> </tr> </thead> <tbody> <tr> <td style="border:none;" align="left">1</td> <td style="border:none;" align="left">P0A715</td> <td style="border:none;" align="left">2-dehydro-3-deoxyphosphooctonate aldolase</td> <td style="border:none;" align="left">KDSA_ECOLI</td> <td style="border:none;" align="left">0.001</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.52</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">2</td> <td style="border:none;" align="left">P0A953</td> <td style="border:none;" align="left">3-oxoacyl-[acyl-carrier-protein] synthaseÂ 1</td> <td style="border:none;" align="left">FABB_ECOLI</td> <td style="border:none;" align="left">2.03Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.26</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">3</td> <td style="border:none;" align="left">P69441</td> <td style="border:none;" align="left">adenylate kinase</td> <td style="border:none;" align="left">KAD_ECOLI</td> <td style="border:none;" align="left">1.66Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.18</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">4</td> <td style="border:none;" align="left">P0A9Q7</td> <td style="border:none;" align="left">aldehydeâ€"alcohol dehydrogenase</td> <td style="border:none;" align="left">ADHE_ECOLI</td> <td style="border:none;" align="left">0.008</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"3.03</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">5</td> <td style="border:none;" align="left">P11875</td> <td style="border:none;" align="left">arginine--tRNA ligase</td> <td style="border:none;" align="left">SYR_ECOLI</td> <td style="border:none;" align="left">0.008</td> <td style="border:none;" align="left">1.69</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.63</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">6</td> <td style="border:none;" align="left">P22106</td> <td style="border:none;" align="left">asparagine synthetaseÂ B [glutamine-hydrolyzing]</td> <td style="border:none;" align="left">ASNB_ECOLI</td> <td style="border:none;" align="left">0.004</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.5</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">7</td> <td style="border:none;" align="left">P00509</td> <td style="border:none;" align="left">aspartate aminotransferase</td> <td style="border:none;" align="left">AAT_ECOLI</td> <td style="border:none;" align="left">0.015</td> <td style="border:none;" align="left">1.91</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">1.38</td> <td style="border:none;" align="left">UP</td> </tr> <tr> <td style="border:none;" align="left">8</td> <td style="border:none;" align="left">P00509</td> <td style="border:none;" align="left">aspartate aminotransferase</td> <td style="border:none;" align="left">AAT_ECOLI</td> <td style="border:none;" align="left">0.019</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"1.36</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">9</td> <td style="border:none;" align="left">P0AEK4</td> <td style="border:none;" align="left">enoyl-[acyl-carrier-protein] reductase [NADH] FabI</td> <td style="border:none;" align="left">FABI_ECOLI</td> <td style="border:none;" align="left">8.31Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.87</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">10</td> <td style="border:none;" align="left">P0A7E5</td> <td style="border:none;" align="left">CTP synthase</td> <td style="border:none;" align="left">PYRG_ECOLI</td> <td style="border:none;" align="left">0.031</td> <td style="border:none;" align="left">1.61</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.79</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">11</td> <td style="border:none;" align="left">P0A7E5</td> <td style="border:none;" align="left">CTP synthase</td> <td style="border:none;" align="left">PYRG_ECOLI</td> <td style="border:none;" align="left">0.039</td> <td style="border:none;" align="left">2.84</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.4</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">12</td> <td style="border:none;" align="left">P29208</td> <td style="border:none;" align="left"><i>o</i>-succinylbenzoate synthase</td> <td style="border:none;" align="left">MENC_ECOLI</td> <td style="border:none;" align="left">9.73Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.4</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">13</td> <td style="border:none;" align="left">P0A9M8</td> <td style="border:none;" align="left">phosphate acetyltransferase</td> <td style="border:none;" align="left">PTA_ECOLI</td> <td style="border:none;" align="left">0.011</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> </tr> <tr> <td style="border:none;" align="left">14</td> <td style="border:none;" align="left">P0A7D7</td> <td style="border:none;" align="left">phosphoribosylaminoimidazole-succinocarboxamide synthase</td> <td style="border:none;" align="left">PUR7_ECOLI</td> <td style="border:none;" align="left">0.023</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"1.83</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">15</td> <td style="border:none;" align="left">Q46803</td> <td style="border:none;" align="left">putative carbamoyltransferase YgeW</td> <td style="border:none;" align="left">YGEW_ECOLI</td> <td style="border:none;" align="left">0.007</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.27</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">16</td> <td style="border:none;" align="left">P0A8N3</td> <td style="border:none;" align="left">lysine--tRNA ligase</td> <td style="border:none;" align="left">SYK1_ECOLI</td> <td style="border:none;" align="left">5.55Â Ã—Â 10<sup>â€"5</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.8</td> <td style="border:none;" align="left">DOWN</td> </tr> </tbody> </table> </div> </div> <div class="envelopeandperiplasmicproteins" title="sec"> <div class="title" tagx="title" title="title">
Envelope and Periplasmic Proteins</div> <p>Outer-membrane proteins are important for the integrity of bacterial membranes. This is achieved via their connection with cell-wall peptidoglycans as well as their role in cell conjugation.<sup><a href="#ref48">48</a></sup> The expression of outer-membrane porins F and W as well as the putative outer-membrane porin protein NmpC was upregulated in <i>E. coli</i> treated with ME (<a href="#tbl6">Table </a><a href="#tbl6">6</a>). This upregulation suggested their role in defense mechanisms, and such a response has also been observed in <i>E. coli</i> responding to the antibiotic tetracycline.<sup><a href="#ref49">49</a></sup> However, Lok <i>et al.</i>(<a href="#ref50">50</a>) speculated that an increase in expression of the precursor of the envelope protein leads to weakening of the outer membrane of the cell and results in cell death as a response to the stress caused by the antimicrobial agents AgNPs. The expression of all outer-membrane porins was downregulated in SE-treated <i>E. coli</i>. An identical trend was recorded by Leung <i>et al.</i>(<a href="#ref51">51</a>) when <i>E. coli</i> was treated by antimicrobial MgO NPs, suggesting that downregulation of the expression of outer-membrane components is an indication of cell-membrane instability. The expression of membrane-related proteins could be a response to the adherence of biomolecules in extracts to the cell surface before entering the cell. Therefore, we might comment that bacterial cells are affected inside and outside the cell membrane. Some unknown proteins were detected (<a href="#tbl7">Table </a><a href="#tbl7">7</a>).</p> <div class="table-wrap_UNKNOWN" id="tbl6" position="float"><span class="label" tagx="label" title="label">Table 6</span> <div class="title" tagx="title" title="title">
Transport of Envelope Proteins and Periplasmic Proteins</div>  <table frame="hsides" rules="groups" border="0"> <colgroup> <col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup>  <thead> <tr> <th style="border:none;" align="center">sl no.</th> <th style="border:none;" align="center">accession no.</th> <th style="border:none;" align="center">protein name</th> <th style="border:none;" align="center">MASCOT ID</th> <th style="border:none;" align="center">ANOVA <i>p</i>-value</th> <th style="border:none;" align="center">fold T1/C</th> <th style="border:none;" align="center">EXP T1/C</th> <th style="border:none;" align="center">fold T2/C</th> <th style="border:none;" align="center">EXP T2/C</th> </tr> </thead> <tbody> <tr> <td style="border:none;" align="left">1</td> <td style="border:none;" align="left">P02931</td> <td style="border:none;" align="left">outer-membrane porinÂ F</td> <td style="border:none;" align="left">OMPF_ECOLI</td> <td style="border:none;" align="left">0.017</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">1.84</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">2</td> <td style="border:none;" align="left">P02931</td> <td style="border:none;" align="left">outer-membrane porinÂ F</td> <td style="border:none;" align="left">OMPF_ECOLI</td> <td style="border:none;" align="left">0.043</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> </tr> <tr> <td style="border:none;" align="left">3</td> <td style="border:none;" align="left">P0A910</td> <td style="border:none;" align="left">outer-membrane proteinÂ AÂ (2)</td> <td style="border:none;" align="left">OMPA_ECOLI</td> <td style="border:none;" align="left">1.98Â Ã—Â 10<sup>â€"5</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.17</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">4</td> <td style="border:none;" align="left">P0A915</td> <td style="border:none;" align="left">outer-membrane proteinÂ W</td> <td style="border:none;" align="left">OMPW_ECOLI</td> <td style="border:none;" align="left">7.31Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.29</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">5</td> <td style="border:none;" align="left">P0A915</td> <td style="border:none;" align="left">outer-membrane proteinÂ W</td> <td style="border:none;" align="left">OMPW_ECOLI</td> <td style="border:none;" align="left">0.039</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"1.55</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">6</td> <td style="border:none;" align="left">P0A917</td> <td style="border:none;" align="left">outer-membrane proteinÂ X</td> <td style="border:none;" align="left">OMPX_ECOLI</td> <td style="border:none;" align="left">0.002</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"2.69</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">7</td> <td style="border:none;" align="left">P02943</td> <td style="border:none;" align="left">maltoporin</td> <td style="border:none;" align="left">LAMB_ECOLI</td> <td style="border:none;" align="left">0.004</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.1</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">8</td> <td style="border:none;" align="left">P0ABP8</td> <td style="border:none;" align="left">purine nucleoside phosphorylase DeoD-type</td> <td style="border:none;" align="left">DEOD_ECOLI</td> <td style="border:none;" align="left">2.69Â Ã—Â 10<sup>â€"4</sup></td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.01</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">9</td> <td style="border:none;" align="left">P21420</td> <td style="border:none;" align="left">putative outer-membrane porin protein NmpC</td> <td style="border:none;" align="left">NMPC_ECOLI</td> <td style="border:none;" align="left">0.001</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="left">â€"2.24</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">10</td> <td style="border:none;" align="left">P21420</td> <td style="border:none;" align="left">putative outer-membrane porin protein NmpC</td> <td style="border:none;" align="left">NMPC_ECOLI</td> <td style="border:none;" align="left">0.036</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"1.5</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">11</td> <td style="border:none;" align="left">P06129</td> <td style="border:none;" align="left">vitaminÂ B12 transporter BtuB</td> <td style="border:none;" align="left">BTUB_ECOLI</td> <td style="border:none;" align="left">0.005</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="left">â€"1.5</td> <td style="border:none;" align="left">DOWN</td> </tr> </tbody> </table> </div> <p>In total, 119 proteins were identified and matched using the MASCOT PMFs to entries in the SWISS-PROT database (taxonomy: <i>E. coli</i>) with high confidence. Of these 119 proteins, the expression of 42 proteins was upregulated, and the expression of 6 was downregulated in ME-treated <i>E. coli</i> in comparison with the control, and the expression of 12 proteins was upregulated and the expression of 104 was downregulated in SE-treated <i>E. coli</i> in comparison with the control. Furthermore, the expression of 11 proteins was upregulated and that of 108 was downregulated in SE-treated <i>E. coli</i> in comparison with ME-treated <i>E. coli</i>.</p> <div class="table-wrap_UNKNOWN" id="tbl7" position="float"><span class="label" tagx="label" title="label">Table 7</span> <div class="title" tagx="title" title="title">
Unknown Proteins</div>  <table frame="hsides" rules="groups" border="0"> <colgroup> <col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="char" char="." /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="left" /></colgroup> <colgroup><col align="char" char="." /></colgroup> <colgroup><col align="left" /></colgroup>  <thead> <tr> <th style="border:none;" align="center">sl no.</th> <th style="border:none;" align="center">accession no.</th> <th style="border:none;" align="center">protein name</th> <th style="border:none;" align="center">MASCOT ID</th> <th style="border:none;" align="center" char=".">ANOVA <i>p</i>-value</th> <th style="border:none;" align="center">fold T1/C</th> <th style="border:none;" align="center">EXP T1/C</th> <th style="border:none;" align="center" char=".">fold T2/C</th> <th style="border:none;" align="center">EXP T2/C</th> </tr> </thead> <tbody> <tr> <td style="border:none;" align="left">1</td> <td style="border:none;" align="left">P39393</td> <td style="border:none;" align="left">putative uncharacterized protein YjiV</td> <td style="border:none;" align="left">YJIV_ECOLI</td> <td style="border:none;" align="char" char=".">0.01</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.03</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">2</td> <td style="border:none;" align="left">P27248</td> <td style="border:none;" align="left">aminomethyltransferase</td> <td style="border:none;" align="left">GCST_ECOLI</td> <td style="border:none;" align="char" char=".">0.047</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"1.5</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">3</td> <td style="border:none;" align="left">P0A9L3</td> <td style="border:none;" align="left">FKBP-type 22Â kDa peptidyl-prolylÂ cisâ€"trans isomerase</td> <td style="border:none;" align="left">FKBB_ECOLI</td> <td style="border:none;" align="char" char=".">0.006</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"1.77</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">4</td> <td style="border:none;" align="left">Q2M7M3</td> <td style="border:none;" align="left">uncharacterized lipoprotein YsaB</td> <td style="border:none;" align="left">YSAB_ECOLI</td> <td style="border:none;" align="char" char=".">0.022</td> <td style="border:none;" align="left">1.55</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">â€"2.92</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">5</td> <td style="border:none;" align="left">P64557</td> <td style="border:none;" align="left">uncharacterized protein YgfM</td> <td style="border:none;" align="left">YGFM_ECOLI</td> <td style="border:none;" align="char" char=".">0.009</td> <td style="border:none;" align="left">N.S.</td> <td style="border:none;" align="left">Â </td> <td style="border:none;" align="char" char=".">â€"2.01</td> <td style="border:none;" align="left">DOWN</td> </tr> <tr> <td style="border:none;" align="left">6</td> <td style="border:none;" align="left">P08331</td> <td style="border:none;" align="left">2â€²,3â€²-cyclic-nucleotide 2â€²-phosphodiesterase/3â€²-nucleotidase</td> <td style="border:none;" align="left">CPDB_ECOLI</td> <td style="border:none;" align="char" char=".">0.006</td> <td style="border:none;" align="left">1.5</td> <td style="border:none;" align="left">UP</td> <td style="border:none;" align="char" char=".">â€"1.66</td> <td style="border:none;" align="left">DOWN</td> </tr> </tbody> </table> </div> <p>About 85% of expressed proteins were from the cytoplasm and only 15% from microbial cell walls, which indicated the penetration of molecules from extracts into microbe cells. A higher percentage for expressed proteins was recorded for enzymatic activity (52%) (<a href="#fig13">Figure </a><a href="#fig13">13</a>). Our findings suggest that the expression of proteins involved in the outer membrane, oxidative stress, and metabolism was enhanced by the biomolecules within extracts.</p> </div> <div class="interactionsamongseparatedproteinsparticipatinginstress" title="sec"> <div class="title" tagx="title" title="title">
Interactions among Separated Proteins Participating in Stress</div> <p>The biological interpretation of proteins involved in stress conditions mediated by <i>E. coli</i> treated with ME and SE was assessed by STRING. 2D-DIGE was used to identify the critical proteins (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c04047/suppl_file/ao0c04047_si_001.pdf">Table S1</a>), which were charted in the protein network. There was high protein interaction within all expressed protein groups. A total of 15 networks were generated from the identified proteins whose expression changed significantly in <i>E. coli</i> after treatment with extracts: carbon metabolism, pyruvate metabolism, metabolic pathways, ribosome, microbial metabolism in diverse environments, citrate cycle (TCA cycle), biosynthesis of antibiotics, glycolysis/gluconeogenesis, biosynthesis of secondary metabolites, purine metabolism, pyrimidine metabolism, RNA polymerase, butanoate metabolism, propanoate metabolism, and oxidative phosphorylation. Connections among identified proteins offer new information about the reactions of bacterial cells under the stress conditions elicited by extracts. Metabolic pathways contained the most affected proteins from which the expression of 15 proteins was upregulated and the expression of 29 proteins was inhibited significantly in <i>E. coli</i> treated with ME and SE, respectively. In general, most expressed proteins showed an excess in microbes treated with ME and an inhibition in microbes treated with SE, which might suggest higher antimicrobial activity for SE. Furthermore, the last phase in glycolysis is the pyruvate metabolism (to provide acetyl-CoA) and the TCA cycle (to provide energy). The proton-motive force is a result of the enhancement of the TCA cycle via an increase in the level of nicotinamide adenine dinucleotide.<sup><a href="#ref52">52</a></sup> The proton-motive force enhances antibiotic uptake.<sup><a href="#ref53">53</a></sup> Our results showing variations in metabolic pathways (including the TCA cycle) could be an antibiotic-resistance approach which might explain upregulation of the expression of some metabolic-pathway proteins in extracts having a lower effect on <i>E. coli</i> in contrast to extracts showing higher activity even though the expression of their proteins was downregulated. Analysis of the comprehensive proteinâ€"protein network of <i>E. coli</i> under extract stress might extend our knowledge on the extract mechanism as antimicrobial agents (<a href="#fig15">Figure </a><a href="#fig15">15</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 15</span></a></div>   <p>Number of identified proteins according to their functions assessed by STRING.</p>   </div> </div> </div> </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>Using biological materials as antimicrobial agents is a promising approach. ME and SE showed good activities against different microbes, from which <i>E. coli</i> was the most affected. Morphologic changes and cell elongation were detected for microbes treated with ME and SE when compared with untreated controls that might be related to the increase in membrane permeability and the accumulation of fluids or influx of the plant extract inside the cell.</p> <p>Our study explores the underlying mode of action and the response of <i>E. coli</i> against toxic effects of ME and SE. 2D-DIGE for extract-treated <i>E. coli</i> indicated that the major systems in the antibacterial mode of action were proteins involved in the outer membrane, oxidative stress, and metabolism. Our data might reveal new targets for antimicrobial agents. Generally, the synergistic effect of extracts with antibiotics might help in the fight against antimicrobial resistance.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="descriptionofsamples" title="sec"> <div class="title" tagx="title" title="title">
Description of Samples</div> <p>Myrtle and shilajeet were obtained from a local market in Riyadh, Saudi Arabia. Before use, they were labeled on polythene bags and kept at 4 Â°C. The test sample was washed using distilled water, air-dried, and ground into fine powder with the aid of a milling machine (IKA Werke Laboratory Equipment, Staufen, Germany). The milled materials were stored in sealed plastic containers at room temperature for further extraction and analyses.</p> </div> <div class="preparationofaqueousextracts" title="sec"> <div class="title" tagx="title" title="title">
Preparation of Aqueous Extracts</div> <p>Aqueous extracts were prepared from each collected sample by addition of 10 g of the powder to 100 mL of water. The mixture was heated at 80 Â°C for 10 min for enzyme deactivation. Mixtures were filtered using Whatman #1 (pore size = 125 mm; Whatman, Maidstone, UK). Filtrates were kept at 4 Â°C for subsequent use.</p> <div class="bacterialstrainsandcultureconditions" title="sec"> <div class="title" tagx="title" title="title">
Bacterial Strains and Culture Conditions</div> <p>Pathogenic microbes that infect humans (<i>P. aeruginosa</i>, <i>E. coli</i>, <i>S. aureus</i>, and <i>C. albicans</i>) were isolated from patients. Isolates were suspended in 0.85% saline to produce a turbidity identical in number to the 0.5 McFarland turbidity standard. Incubation at 37 Â°C for 24 h was undertaken for prepared cultures after dilution to 1:10 to obtain a density of 1.5 Ã— 10<sup>8</sup> CFU/mL culture. Preparation of microbial cultures was undertaken at the Department of Biology, College of Science, Riyadh, Saudi Arabia.</p> </div> </div> <div class="antimicrobialactivity" title="sec"> <div class="title" tagx="title" title="title">
Antimicrobial Activity</div> <div class="determinationofthezoneofinhibition" title="sec"> <div class="title" tagx="title" title="title">
Determination of the Zone of Inhibition</div> <p>The antimicrobial ability of aqueous extracts was assessed by the agar-well diffusion method.<sup><a href="#ref54">54</a></sup> Muellerâ€"Hinton Agar (20 mL) was poured into sterilized Petri plates and maintained at room temperature. Then, 0.2 mL of each test strain (1.5 Ã— 10<sup>8</sup> CFU/mL) was cultured in nutrient broth (NB) for 24 h to prepare â€œbacterial lawns.â€� Four agar wells (4 mm) were prepared using a sterilized cork borer and filled with each extract. The reference negative control was sterile distilled water. Plates were incubated for 18â€"24 h at 37 Â°C for bacteria and for 48â€"96 h at 28 Â°C for <i>C. albicans</i>. After the incubation period, plates were examined for extract activity as evidenced by inhibition zones around the well as a clear area.<sup><a href="#ref55">55</a></sup> The diameter of each inhibition zone was measured (millimeters), and the mean value for each plate was recorded in three replicates for each microbe.</p> </div> <div class="minimuminhibitoryconcentration" title="sec"> <div class="title" tagx="title" title="title">
Minimum Inhibitory Concentration</div> <p>The minimum inhibitory concentration (MIC) was determined by a microdilution method in NB by addition of 0.2 mL of a microbial strain at a concentration of 1.5 Ã— 10<sup>8</sup> CFU/mL bacteria to 10 mL of NB, individually. Aqueous extracts at different concentrations were added to bacterial strains, and incubation for 24 h was allowed. After incubation, the MIC was assessed by examining the turbidity of bacterial growth. The lowest concentration that killed the test microbes completely was considered to be the MIC.<sup><a href="#ref56">56</a></sup> Additionally, the interaction of extracts with tested microbes, <i>E. coli</i> and <i>P. aeruginosa,</i> was evaluated 2 h after the treatment using field emission scanning electron microscopy.</p> </div> <div class="synergisticantibacterialactivityofaqueousextracts" title="sec"> <div class="title" tagx="title" title="title">
Synergistic Antibacterial Activity of Aqueous Extracts</div> <p>The synergistic effect of the aqueous extracts was determined upon mixture with antibiotics (bacitracin, ciprofloxacin, tetracycline, and cefixime). The synergistic potential of aqueous extracts as well as bacitracin, ciprofloxacin, tetracycline, and cefixime as standard antibiotics was determined against <i>S. aureus</i>, <i>P. aeruginosa</i>, <i>E. coli</i>, and <i>Candida</i> species by the standard disk-diffusion method.<sup><a href="#ref57">57</a></sup> The bacterial strains were cultured fresh on NB media (Becton Dickinson, Sparks Glencoe, MD, USA). The aqueous extracts (1 mg/mL) and standard antibiotics [bacitracin (10 Î¼g/mL), ciprofloxacin (10 Î¼g/mL), tetracycline (30 Î¼g/mL), cefixime (5 Î¼g/mL)] were mixed at a 1:1 ratio, applied to each microbial plate, and sonicated for 15 min at room temperature. The synergistic activity of the mixture of aqueous extracts/antibiotics was evaluated after 24 h of incubation at 37 Â°C in terms of inhibition zones around the filter paper disks (millimeters).</p> </div> </div> <div class="proteinextraction" title="sec"> <div class="title" tagx="title" title="title">
Protein Extraction</div> <p>Protein extraction was done as described previously with some modifications. Briefly, <i>E. coli</i> cells were collected by centrifugation at 12,000<i>g</i> for 10 min at 40 Â°C. The resulting pellet was washed twice with phosphate-buffered saline after discarding the supernatant. Following centrifugation, the protein pellets were suspended in the lysis buffer (0.5 mL; pH 8.8; 30 mM Tris buffer containing 7 M urea, 2 M thiourea, 2% Chaps, and the protease inhibitor cocktail; GE Healthcare, Chicago, IL, USA) for 30 min on ice. Sonication was carried out for 30 s with 3â€"4 pulses to obtain a clear solution. Unbroken or debris cells were removed by centrifugation at 10,000 rpm for 5 min at 4 Â°C. Subsequently, solubilized proteins in the supernatant were collected, and the protein concentrations were determined in triplicate using the 2D-Quant Kit according to the manufacturerâ€™s (GE Healthcare) instructions.</p> </div> <div class="fluorescencelabelingandproteomicanalysis[2d-digeandmalditandemtime-of-flightmassspectrometry(tof/tofms)]" title="sec"> <div class="title" tagx="title" title="title">
Fluorescence Labeling and Proteomic Analysis [2D-DIGE and MALDI Tandem Time-of-Flight Mass Spectrometry (TOF/TOF MS)]</div> <p>The protein extracted (50 Î¼g) from each sample underwent Cy3 labeling or Cy5 labeling. Also, a mixture of an equal amount of all samples was pooled, labeled with Cy2, and used as an internal standard as described previously.<sup><a href="#ref58">58</a>âˆ’<a href="#ref61">61</a></sup> During labeling, dye switching was applied to avoid a dye-specific bias (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c04047/suppl_file/ao0c04047_si_001.pdf">Table S1</a>). First- and second-dimension analytical gel electrophoresis was carried out as described previously.<sup><a href="#ref58">58</a>âˆ’<a href="#ref61">61</a></sup> Furthermore, a Typhoon 9410 scanner (GE Healthcare) was used for imaging the 2D-DIGE gels using excitation/emission wavelengths specific for Cy2 (488/520 nm), Cy3 (532/580 nm), and Cy5 (633/670 nm).</p> <p>The Coomassie Blue-stained gel was washed and digested from a preparatory gel according to procedures reported previously.<sup><a href="#ref58">58</a>âˆ’<a href="#ref60">60</a></sup> Spotting was carried out onto a MALDI target (384 MTP Anchorchip; 800 pm Anchorchip; Bruker Daltonics, Bremen, Germany) from a mixture of tryptic peptides (1 pL) derived from each protein. MALDI-MS(/MS) spectra were recorded using an UltrafleXtreme TOF mass spectrometer with a reflector voltage and a detector voltage of 21 and 17 kV, respectively, as reported.<sup><a href="#ref62">62</a>,<a href="#ref63">63</a></sup> The Mascot search algorithm v2.0.04 (updated on December 9, 2019; Matrix Science, London, UK) was used for searching peptide masses. The identified proteins were assessed for a Mascot score &amp;gt; 56 and <i>p</i> &amp;lt; 0.05.</p> </div> <div class="statisticalanalyses" title="sec"> <div class="title" tagx="title" title="title">
Statistical Analyses</div> <p>Statistical analyses involved uploading 2D-DIGE gel images into progenesis â€œSame Spotsâ€� software (Nonlinear Dynamics, Newcastle, UK), which were then analyzed applying an automated method for spot detection. Independent direct comparisons were made between SE-treated, ME-treated 2, and control <i>E. coli</i> groups, and fold differences and p-values were calculated using one-way ANOVA. All spots were prefiltered and manually checked before applying the statistical criteria (ANOVA test, <i>p</i> â‰¤ 0.05 and fold â‰¥ 1.5). PCA was carried out on log-transformed spot data. Furthermore, a heatmap was created using Heatmapper, a freely available web server <span class="uri" title="uri"><a href="http://heatmapper.ca">http://heatmapper.ca</a></span>.<sup><a href="#ref63">63</a></sup></p> </div> <div class="bioinformaticanalysisandfunctionalclassificationofproteins" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatic Analysis and Functional Classification of Proteins</div> <p>Network analysis was carried out by importing the quantitative data into the STRING v11.0 (<span class="uri" title="uri"><a href="https://string-db.org/">https://string-db.org/</a></span>) online software; this software aids in determining the functions and pathways that are most strongly associated with the protein list by overlaying the experimental expression data on networks constructed from published interactions. Functional classification involved classification of the identified proteins into different categories according to their molecular function and the biological processes in which they are involved using the PANTHER classification system (<span class="uri" title="uri"><a href="http://www.pantherdb.org">www.pantherdb.org</a></span>).</p> </div> </div> </div> <div class="back" title="back"> <div class="title" tagx="title" title="title">
Supporting Information Available</div> <p>The Supporting Information is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.0c04047?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acsomega.0c04047</a>. </p><ul> <li> <p>Mass spectrometry of significant differentially identified abundant proteins between control and treated <i>E. coli</i> using 2D-DIGE and identified proteins, with changes in abundance between samples using fold changes, one-way ANOVA (<i>p</i>-value &amp;lt; 0.05), and 2D-DIGE (analysis type: MALDI-TOF; database: SwissProt; taxonomy: <i>E. coli</i>) (<a href="http://pubs.acs.org/doi/suppl/10.1021/acsomega.0c04047/suppl_file/ao0c04047_si_001.pdf">PDF</a>)</p> </li> </ul> <p /> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="ao0c04047_si_001.pdf">LINK</a> <p>ao0c04047_si_001.pdf</p>  </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>K.A. designed and directed the project, supervised the experiments, and performed the analysis of the antimicrobial results. H.B., A.M., A.A.A., I.O.A., and E.A.A. performed the proteomic assessment and result analysis. A.E.M. interpreted, reviewed, and discussed the presented data as well as prepared the first draft. A.E.M. with K.A. and H.B. finalized writing the manuscript.</p> <p>This research was funded by the Deanship of Scientific Research at Princess Nourah Bint Abdulrahman University through the Fast-track Research Funding Program.</p> <p>The authors declare no competing financial interest.</p> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We would like to express special thanks and gratitude to Abeer Alqahtani, Arwa Safadi, Hessa AlTahiney, Ruaa Ayrout, and Walaa Alqishawi for their help in this study. Without their assistance and dedicated involvement in every step throughout the process, this research would have never been accomplished.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ref1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beceiro</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TomÃ¡s</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bou</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="mixed-article-title" title="mixed-article-title">Antimicrobial resistance and virulence: a successful or deleterious association in the bacterial world?</span>. <span class="source" tagx="source" title="source">Clin. Microbiol. Rev.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">185</span>â€"<span class="lpage" tagx="lpage" title="lpage">230</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/cmr.00059-12">10.1128/cmr.00059-12</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23554414">23554414</a></span></span></li> <li tag="ref"><a name="ref2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirwaikar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajendran</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bodla</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">C. D.</span></span><span class="mixed-article-title" title="mixed-article-title">Neutralization potential of Viper russelli russelli (Russellâ€™s viper) venom by ethanol leaf extract of Acalypha indica</span>. <span class="source" tagx="source" title="source">J. Ethnopharmacol.</span><span class="year" tagx="year" title="year">2004</span>, <span class="volume" tagx="volume" title="volume">94</span>, <span class="fpage" tagx="fpage" title="fpage">267</span>â€"<span class="lpage" tagx="lpage" title="lpage">273</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jep.2004.05.010">10.1016/j.jep.2004.05.010</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15325729">15325729</a></span></span></li> <li tag="ref"><a name="ref3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Govindarajan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jebanesan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reetha</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amsath</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pushpanathan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samidurai</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="mixed-article-title" title="mixed-article-title">Antibacterial activity of Acalypha indica L</span>. <span class="source" tagx="source" title="source">Eur. Rev. Med. Pharmacol. Sci.</span><span class="year" tagx="year" title="year">2008</span>, <span class="volume" tagx="volume" title="volume">12</span>, <span class="fpage" tagx="fpage" title="fpage">299</span>â€"<span class="lpage" tagx="lpage" title="lpage">302</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19024213">19024213</a></span></span></li> <li tag="ref"><a name="ref4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sathya</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kokilavani</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teepa</span><span class="given-names" tagx="given-names" title="given-names">K. S. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balakrishnan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">Biopotency of Acalypha indica Linn on membrane bound ATPases and marker enzymes urolithic rats</span>. <span class="source" tagx="source" title="source">Anc. Sci. Life</span><span class="year" tagx="year" title="year">2011</span>, <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">3</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22736883">22736883</a></span></span></li> <li tag="ref"><a name="ref5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Upadhyay</span><span class="given-names" tagx="given-names" title="given-names">O. P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tiwari</span><span class="given-names" tagx="given-names" title="given-names">R. K.</span></span><span class="mixed-article-title" title="mixed-article-title">Ethnobotanical study of skin treatment uses of medicinal plants of Bihar</span>. <span class="source" tagx="source" title="source">Pharm. Biol.</span><span class="year" tagx="year" title="year">1998</span>, <span class="volume" tagx="volume" title="volume">36</span>, <span class="fpage" tagx="fpage" title="fpage">167</span>â€"<span class="lpage" tagx="lpage" title="lpage">172</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1076/phbi.36.3.167.6339">10.1076/phbi.36.3.167.6339</a></span>.</span></li> <li tag="ref"><a name="ref6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chowdhury</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sayeed</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah Alam Bhuiyan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Astaq Mohal Khan</span><span class="given-names" tagx="given-names" title="given-names">G. R. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Antimicrobial activity and cytotoxicity of <i>Aerva lanata</i></span>. <span class="source" tagx="source" title="source">Fitoterapia</span><span class="year" tagx="year" title="year">2002</span>, <span class="volume" tagx="volume" title="volume">73</span>, <span class="fpage" tagx="fpage" title="fpage">92</span>â€"<span class="lpage" tagx="lpage" title="lpage">94</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/s0367-326x(01)00369-0">10.1016/s0367-326x(01)00369-0</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11864768">11864768</a></span></span></li> <li tag="ref"><a name="ref7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vetrichelvan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jegadeesan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Anti-diabetic activity of alcoholic extract of <i>Aerva lanata</i> (L.) Juss. ex Schultes in rats</span>. <span class="source" tagx="source" title="source">J. Ethnopharmacol.</span><span class="year" tagx="year" title="year">2002</span>, <span class="volume" tagx="volume" title="volume">80</span>, <span class="fpage" tagx="fpage" title="fpage">103</span>â€"<span class="lpage" tagx="lpage" title="lpage">107</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/s0378-8741(01)00412-3">10.1016/s0378-8741(01)00412-3</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12007698">12007698</a></span></span></li> <li tag="ref"><a name="ref8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agrawal</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sethiya</span><span class="given-names" tagx="given-names" title="given-names">N. K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mishra</span><span class="given-names" tagx="given-names" title="given-names">S. H.</span></span><span class="mixed-article-title" title="mixed-article-title">Antidiabetic activity of alkaloids of <i>Aerva lanata</i> roots on streptozotocin-nicotinamide induced type-II diabetes in rats</span>. <span class="source" tagx="source" title="source">Pharm. Biol.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">51</span>, <span class="fpage" tagx="fpage" title="fpage">635</span>â€"<span class="lpage" tagx="lpage" title="lpage">642</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3109/13880209.2012.761244">10.3109/13880209.2012.761244</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23527955">23527955</a></span></span></li> <li tag="ref"><a name="ref9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riya</span><span class="given-names" tagx="given-names" title="given-names">M. P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antu</span><span class="given-names" tagx="given-names" title="given-names">K. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pal</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandrakanth</span><span class="given-names" tagx="given-names" title="given-names">K. C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anilkumar</span><span class="given-names" tagx="given-names" title="given-names">K. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamrakar</span><span class="given-names" tagx="given-names" title="given-names">A. K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srivastava</span><span class="given-names" tagx="given-names" title="given-names">A. K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raghu</span><span class="given-names" tagx="given-names" title="given-names">K. G.</span></span><span class="mixed-article-title" title="mixed-article-title">Antidiabetic property of <i>Aerva lanata</i> (L.) Juss. ex Schult. is mediated by inhibition of alpha glucosidase, protein glycation and stimulation of adipogenesis</span>. <span class="source" tagx="source" title="source">J. Diabetes</span><span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">548</span>â€"<span class="lpage" tagx="lpage" title="lpage">561</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1753-0407.12216">10.1111/1753-0407.12216</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25224159">25224159</a></span></span></li> <li tag="ref"><a name="ref10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouzouita</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kachouri</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamdi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaabouni</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Antimicrobial activity of essential oils from Tunisian aromatic plants</span>. <span class="source" tagx="source" title="source">Flavour Fragrance J.</span><span class="year" tagx="year" title="year">2003</span>, <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">380</span>â€"<span class="lpage" tagx="lpage" title="lpage">383</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ffj.1200">10.1002/ffj.1200</a></span>.</span></li> <li tag="ref"><a name="ref11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Besufekad</span><span class="given-names" tagx="given-names" title="given-names">S. Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mekdes</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abebech</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delesa</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tekalign</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demitu</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birtukan</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="mixed-article-title" title="mixed-article-title">The antimicrobial activity of leaf extracts of <i>Myrtus communis</i></span>. <span class="source" tagx="source" title="source">J. Microb. Biochem. Technol.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">290</span>â€"<span class="lpage" tagx="lpage" title="lpage">292</span>. <span class="pub-id"><a href="https://dx.doi.org/10.4172/1948-5948.1000380">10.4172/1948-5948.1000380</a></span>.</span></li> <li tag="ref"><a name="ref12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srivastava</span><span class="given-names" tagx="given-names" title="given-names">S. R.</span></span><span class="mixed-article-title" title="mixed-article-title">Antifungal activity screening and hplc analysis of crude extract from <i>Tectona grandis</i>, shilajit, <i>Valeriana wallachi</i></span>. <span class="source" tagx="source" title="source">Electron. J. Environ., Agric. Food Chem.</span><span class="year" tagx="year" title="year">2009</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">218</span>â€"<span class="lpage" tagx="lpage" title="lpage">229</span>.</span></li> <li tag="ref"><a name="ref13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayder</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skandrani</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kilani</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouhlel</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelwahed</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ben Ammar</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahmoud</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghedira</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chekir-Ghedira</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="mixed-article-title" title="mixed-article-title">Antimutagenic activity of <i>Myrtus communis</i> L. using the <i>Salmonella</i> microsome assay</span>. <span class="source" tagx="source" title="source">S. Afr. J. Bot.</span><span class="year" tagx="year" title="year">2008</span>, <span class="volume" tagx="volume" title="volume">74</span>, <span class="fpage" tagx="fpage" title="fpage">121</span>â€"<span class="lpage" tagx="lpage" title="lpage">125</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.sajb.2007.10.001">10.1016/j.sajb.2007.10.001</a></span>.</span></li> <li tag="ref"><a name="ref14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taheri</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seyfan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jalalinezhad</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nasery</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="mixed-article-title" title="mixed-article-title">Antibacterial effect of Myrtus communis hydro-alcoholic extract on pathogenic bacteria</span>. <span class="source" tagx="source" title="source">Zahedan J. Res. Med. Sci.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">19</span>â€"<span class="lpage" tagx="lpage" title="lpage">24</span>.</span></li> <li tag="ref"><a name="ref15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shan</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Y.-Z.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooks</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corke</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="mixed-article-title" title="mixed-article-title">The in vitro antibacterial activity of dietary spice and medicinal herb extracts</span>. <span class="source" tagx="source" title="source">Int. J. Food Microbiol.</span><span class="year" tagx="year" title="year">2007</span>, <span class="volume" tagx="volume" title="volume">117</span>, <span class="fpage" tagx="fpage" title="fpage">112</span>â€"<span class="lpage" tagx="lpage" title="lpage">119</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijfoodmicro.2007.03.003">10.1016/j.ijfoodmicro.2007.03.003</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17449125">17449125</a></span></span></li> <li tag="ref"><a name="ref16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Rensburg</span><span class="given-names" tagx="given-names" title="given-names">C. E. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Straten</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dekker</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="mixed-article-title" title="mixed-article-title">An in vitro investigation of the antimicrobial activity of oxifulvic acid</span>. <span class="source" tagx="source" title="source">J. Antimicrob. Chemother.</span><span class="year" tagx="year" title="year">2000</span>, <span class="volume" tagx="volume" title="volume">46</span>, <span class="fpage" tagx="fpage" title="fpage">853</span>â€"<span class="lpage" tagx="lpage" title="lpage">854</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/jac/46.5.853">10.1093/jac/46.5.853</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11062218">11062218</a></span></span></li> <li tag="ref"><a name="ref17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kotb El-Sayed</span><span class="given-names" tagx="given-names" title="given-names">M.-I.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amin</span><span class="given-names" tagx="given-names" title="given-names">H.-K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Kaf</span><span class="given-names" tagx="given-names" title="given-names">A.-G.</span></span><span class="mixed-article-title" title="mixed-article-title">Anti-Microbial, anti-oxidant and anti-ulcerogenic effects of Shilajit on gastric ulcer in rats</span>. <span class="source" tagx="source" title="source">Am. J. Biochem. Biotechnol.</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">25</span>â€"<span class="lpage" tagx="lpage" title="lpage">37</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3844/ajbbsp.2012.25.37">10.3844/ajbbsp.2012.25.37</a></span>.</span></li> <li tag="ref"><a name="ref18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aleksic</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mimica-Dukic</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simin</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nedeljkovic</span><span class="given-names" tagx="given-names" title="given-names">N. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knezevic</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="mixed-article-title" title="mixed-article-title">Synergistic effect of Myrtus communis L. essential oils and conventional antibiotics against multi-drug resistant Acinetobacter baumannii wound isolates</span>. <span class="source" tagx="source" title="source">Phytomedicine</span><span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">1666</span>â€"<span class="lpage" tagx="lpage" title="lpage">1674</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phymed.2014.08.013">10.1016/j.phymed.2014.08.013</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25442275">25442275</a></span></span></li> <li tag="ref"><a name="ref19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yong</span><span class="given-names" tagx="given-names" title="given-names">A.-L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooh</span><span class="given-names" tagx="given-names" title="given-names">K.-F.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ong</span><span class="given-names" tagx="given-names" title="given-names">H.-C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chai</span><span class="given-names" tagx="given-names" title="given-names">T.-T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">F.-C.</span></span><span class="mixed-article-title" title="mixed-article-title">Investigation of antibacterial mechanism and identification of bacterial protein targets mediated by antibacterial medicinal plant extracts</span>. <span class="source" tagx="source" title="source">Food Chem.</span><span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">186</span>, <span class="fpage" tagx="fpage" title="fpage">32</span>â€"<span class="lpage" tagx="lpage" title="lpage">36</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.foodchem.2014.11.103">10.1016/j.foodchem.2014.11.103</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25976788">25976788</a></span></span></li> <li tag="ref"><a name="ref20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dakal</span><span class="given-names" tagx="given-names" title="given-names">T. C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Majumdar</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="mixed-article-title" title="mixed-article-title">Mechanistic basis of antimicrobial actions of silver nanoparticles</span>. <span class="source" tagx="source" title="source">Front. Microbiol.</span><span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">1831</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2016.01831">10.3389/fmicb.2016.01831</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27899918">27899918</a></span></span></li> <li tag="ref"><a name="ref21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Q. L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">G. Q.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">F. Z.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">T. N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J. O.</span></span><span class="mixed-article-title" title="mixed-article-title">A mechanistic study of the antibacterial effect of silver ions on <i>Escherichia coli</i> and <i>Staphylococcus aureus</i></span>. <span class="source" tagx="source" title="source">J. Biomed. Mater. Res.</span><span class="year" tagx="year" title="year">2000</span>, <span class="volume" tagx="volume" title="volume">52</span>, <span class="fpage" tagx="fpage" title="fpage">662</span>â€"<span class="lpage" tagx="lpage" title="lpage">668</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/1097-4636(20001215)52:4&lt;662::aid-jbm10&gt;3.0.co;2-3">10.1002/1097-4636(20001215)52:4&amp;lt;662::aid-jbm10&amp;gt;3.0.co;2-3</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11033548">11033548</a></span></span></li> <li tag="ref"><a name="ref22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">K. B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bard</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Interaction of silver (I) ions with the respiratory chain of Escherichia coli: an electrochemical and scanning electrochemical microscopy study of the antimicrobial mechanism of micromolar Ag+</span>. <span class="source" tagx="source" title="source">Biochemistry</span><span class="year" tagx="year" title="year">2005</span>, <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">13214</span>â€"<span class="lpage" tagx="lpage" title="lpage">13223</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/bi0508542">10.1021/bi0508542</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16185089">16185089</a></span></span></li> <li tag="ref"><a name="ref23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oz</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guvenek</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yildiz</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karaboga</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamer</span><span class="given-names" tagx="given-names" title="given-names">Y. T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mumcuyan</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ozan</span><span class="given-names" tagx="given-names" title="given-names">V. B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senturk</span><span class="given-names" tagx="given-names" title="given-names">G. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cokol</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeh</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toprak</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="mixed-article-title" title="mixed-article-title">Strength of selection pressure is an important parameter contributing to the complexity of antibiotic resistance evolution</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">2387</span>â€"<span class="lpage" tagx="lpage" title="lpage">2401</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msu191">10.1093/molbev/msu191</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24962091">24962091</a></span></span></li> <li tag="ref"><a name="ref24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poole</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial stress responses as determinants of antimicrobial resistance</span>. <span class="source" tagx="source" title="source">J. Antimicrob. Chemother.</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">67</span>, <span class="fpage" tagx="fpage" title="fpage">2069</span>â€"<span class="lpage" tagx="lpage" title="lpage">2089</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/jac/dks196">10.1093/jac/dks196</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22618862">22618862</a></span></span></li> <li tag="ref"><a name="ref25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiang</span><span class="given-names" tagx="given-names" title="given-names">S. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schellhorn</span><span class="given-names" tagx="given-names" title="given-names">H. E.</span></span><span class="mixed-article-title" title="mixed-article-title">Regulators of oxidative stress response genes in <i>Escherichia coli</i> and their functional conservation in bacteria</span>. <span class="source" tagx="source" title="source">Arch. Biochem. Biophys.</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">525</span>, <span class="fpage" tagx="fpage" title="fpage">161</span>â€"<span class="lpage" tagx="lpage" title="lpage">169</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.abb.2012.02.007">10.1016/j.abb.2012.02.007</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22381957">22381957</a></span></span></li> <li tag="ref"><a name="ref26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LÃ¼ders</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fallet</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franco-Lara</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="mixed-article-title" title="mixed-article-title">Proteome analysis of the <i>Escherichia coli</i> heat shock response under steady-state conditions</span>. <span class="source" tagx="source" title="source">Proteome Sci.</span><span class="year" tagx="year" title="year">2009</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">36</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1477-5956-7-36">10.1186/1477-5956-7-36</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19772559">19772559</a></span></span></li> <li tag="ref"><a name="ref27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S. R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imlay</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="mixed-article-title" title="mixed-article-title">How Escherichia coli tolerates profuse hydrogen peroxide formation by a catabolic pathway</span>. <span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">195</span>, <span class="fpage" tagx="fpage" title="fpage">4569</span>â€"<span class="lpage" tagx="lpage" title="lpage">4579</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/jb.00737-13">10.1128/jb.00737-13</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23913322">23913322</a></span></span></li> <li tag="ref"><a name="ref28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sidjabat</span><span class="given-names" tagx="given-names" title="given-names">H. E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gien</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kvaskoff</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashman</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaswani</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reed</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGeary</span><span class="given-names" tagx="given-names" title="given-names">R. P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paterson</span><span class="given-names" tagx="given-names" title="given-names">D. L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bordin</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schenk</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="mixed-article-title" title="mixed-article-title">The use of SWATH to analyse the dynamic changes of bacterial proteome of carbapanemase-producing Escherichia coli under antibiotic pressure</span>. <span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">3871</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-21984-9">10.1038/s41598-018-21984-9</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29497067">29497067</a></span></span></li> <li tag="ref"><a name="ref29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Storz</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacobson</span><span class="given-names" tagx="given-names" title="given-names">F. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tartaglia</span><span class="given-names" tagx="given-names" title="given-names">L. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morgan</span><span class="given-names" tagx="given-names" title="given-names">R. W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silveira</span><span class="given-names" tagx="given-names" title="given-names">L. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ames</span><span class="given-names" tagx="given-names" title="given-names">B. N.</span></span><span class="mixed-article-title" title="mixed-article-title">An alkyl hydroperoxide reductase induced by oxidative stress in <i>Salmonella typhimurium</i> and <i>Escherichia coli</i>: genetic characterization and cloning of ahp</span>. <span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">1989</span>, <span class="volume" tagx="volume" title="volume">171</span>, <span class="fpage" tagx="fpage" title="fpage">2049</span>â€"<span class="lpage" tagx="lpage" title="lpage">2055</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/jb.171.4.2049-2055.1989">10.1128/jb.171.4.2049-2055.1989</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2649484">2649484</a></span></span></li> <li tag="ref"><a name="ref30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duval</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korepanov</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuchsbauer</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fechter</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haller</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fabbretti</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choulier</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Micura</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klaholz</span><span class="given-names" tagx="given-names" title="given-names">B. P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romby</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title"><i>Escherichia coli</i> ribosomal protein S1 unfolds structured mRNAs onto the ribosome for active translation initiation</span>. <span class="source" tagx="source" title="source">PLoS Biol.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">11</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e1001731</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.1001731">10.1371/journal.pbio.1001731</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24339747">24339747</a></span></span></li> <li tag="ref"><a name="ref31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">S. B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">S. B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steadman</span><span class="given-names" tagx="given-names" title="given-names">C. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pechan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pechanova</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clemente</span><span class="given-names" tagx="given-names" title="given-names">H. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thirumalai</span><span class="given-names" tagx="given-names" title="given-names">R. V. K. G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Willard</span><span class="given-names" tagx="given-names" title="given-names">S. T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryan</span><span class="given-names" tagx="given-names" title="given-names">P. L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feugang</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="mixed-article-title" title="mixed-article-title">Silver-coated magnetic nanocomposites induce growth inhibition and protein changes in foodborne bacteria</span>. <span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2019</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">17499</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-019-53080-x">10.1038/s41598-019-53080-x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31767879">31767879</a></span></span></li> <li tag="ref"><a name="ref32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagano</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kojima</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hisabori</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayashi</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morita</span><span class="given-names" tagx="given-names" title="given-names">E. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanamori</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyagi</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ueda</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishiyama</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="mixed-article-title" title="mixed-article-title">Elongation factor G is a critical target during oxidative damage to the translation system of <i>Escherichia coli</i></span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">287</span>, <span class="fpage" tagx="fpage" title="fpage">28697</span>â€"<span class="lpage" tagx="lpage" title="lpage">28704</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.m112.378067">10.1074/jbc.m112.378067</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22773838">22773838</a></span></span></li> <li tag="ref"><a name="ref33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamarit</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cabiscol</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ros</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="mixed-article-title" title="mixed-article-title">Identification of the major oxidatively damaged proteins in <i>Escherichia coli</i> cells exposed to oxidative stress</span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">1998</span>, <span class="volume" tagx="volume" title="volume">273</span>, <span class="fpage" tagx="fpage" title="fpage">3027</span>â€"<span class="lpage" tagx="lpage" title="lpage">3032</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.273.5.3027">10.1074/jbc.273.5.3027</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9446617">9446617</a></span></span></li> <li tag="ref"><a name="ref34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cabiscol</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piulats</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Echave</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrero</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ros</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="mixed-article-title" title="mixed-article-title">Oxidative stress promotes specific protein damage in <i>Saccharomyces cerevisiae</i></span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2000</span>, <span class="volume" tagx="volume" title="volume">275</span>, <span class="fpage" tagx="fpage" title="fpage">27393</span>â€"<span class="lpage" tagx="lpage" title="lpage">27398</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M003140200">10.1074/jbc.M003140200</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10852912">10852912</a></span></span></li> <li tag="ref"><a name="ref35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dai</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warren</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balakrishnan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okano</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williamson</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fredrick</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hwa</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="mixed-article-title" title="mixed-article-title">Slowdown of translational elongation in Escherichia coli under hyperosmotic stress</span>. <span class="source" tagx="source" title="source">mBio</span><span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e02375</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/mbio.02375-17">10.1128/mbio.02375-17</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29440576">29440576</a></span></span></li> <li tag="ref"><a name="ref36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weijland</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harmark</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cool</span><span class="given-names" tagx="given-names" title="given-names">R. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anborgh</span><span class="given-names" tagx="given-names" title="given-names">P. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parmeggiani</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">Elongation factor Tu: a molecular switch in protein biosynthesis</span>. <span class="source" tagx="source" title="source">Mol. Microbiol.</span><span class="year" tagx="year" title="year">1992</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">683</span>â€"<span class="lpage" tagx="lpage" title="lpage">688</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2958.1992.tb01516.x">10.1111/j.1365-2958.1992.tb01516.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1573997">1573997</a></span></span></li> <li tag="ref"><a name="ref37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Easton</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crowder</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span><span class="mixed-article-title" title="mixed-article-title">Time-dependent translational response of <i>E. coli</i> to excess Zn (II)</span>. <span class="source" tagx="source" title="source">J. Biomol. Tech.</span><span class="year" tagx="year" title="year">2006</span>, <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">303</span>â€"<span class="lpage" tagx="lpage" title="lpage">307</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17122063">17122063</a></span></span></li> <li tag="ref"><a name="ref38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzales-Siles</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karlsson</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenny</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karlsson</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SjÃ¶ling</span><span class="given-names" tagx="given-names" title="given-names">Ã….</span></span><span class="mixed-article-title" title="mixed-article-title">Proteomic analysis of enterotoxigenic <i>Escherichia coli</i> (ETEC) in neutral and alkaline conditions</span>. <span class="source" tagx="source" title="source">BMC Microbiol.</span><span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">11</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12866-016-0914-1">10.1186/s12866-016-0914-1</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28061865">28061865</a></span></span></li> <li tag="ref"><a name="ref39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartl</span><span class="given-names" tagx="given-names" title="given-names">F. U.</span></span><span class="mixed-article-title" title="mixed-article-title">Molecular chaperones in cellular protein folding</span>. <span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1996</span>, <span class="volume" tagx="volume" title="volume">381</span>, <span class="fpage" tagx="fpage" title="fpage">571</span>â€"<span class="lpage" tagx="lpage" title="lpage">580</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/381571a0">10.1038/381571a0</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8637592">8637592</a></span></span></li> <li tag="ref"><a name="ref40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qamra</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mande</span><span class="given-names" tagx="given-names" title="given-names">S. C.</span></span><span class="mixed-article-title" title="mixed-article-title">Crystal structure of the 65-kilodalton heat shock protein, chaperonin 60.2, of Mycobacterium tuberculosis</span>. <span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">2004</span>, <span class="volume" tagx="volume" title="volume">186</span>, <span class="fpage" tagx="fpage" title="fpage">8105</span>â€"<span class="lpage" tagx="lpage" title="lpage">8113</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/jb.186.23.8105-8113.2004">10.1128/jb.186.23.8105-8113.2004</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15547284">15547284</a></span></span></li> <li tag="ref"><a name="ref41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasimoglu</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">S. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malek</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tseng</span><span class="given-names" tagx="given-names" title="given-names">C. P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gunsalus</span><span class="given-names" tagx="given-names" title="given-names">R. P.</span></span><span class="mixed-article-title" title="mixed-article-title">Transcriptional regulation of the proton-translocating ATPase (atpIBEFHAGDC) operon of Escherichia coli: control by cell growth rate</span>. <span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">1996</span>, <span class="volume" tagx="volume" title="volume">178</span>, <span class="fpage" tagx="fpage" title="fpage">5563</span>â€"<span class="lpage" tagx="lpage" title="lpage">5567</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/jb.178.19.5563-5567.1996">10.1128/jb.178.19.5563-5567.1996</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8824597">8824597</a></span></span></li> <li tag="ref"><a name="ref42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Isarankura-Na-Ayudhya</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thippakorn</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pannengpetch</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roytrakul</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Isarankura-Na-Ayudhya</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bunmee</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sawangnual</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prachayasittikul</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="mixed-article-title" title="mixed-article-title">Metal complexation by histidine-rich peptides confers protective roles against cadmium stress in <i>Escherichia coli</i> as revealed by proteomics analysis</span>. <span class="source" tagx="source" title="source">PeerJ</span><span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e5245</span><span class="pub-id"><a href="https://dx.doi.org/10.7717/peerj.5245">10.7717/peerj.5245</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30065864">30065864</a></span></span></li> <li tag="ref"><a name="ref43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">V. W. T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rothery</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiner</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span><span class="mixed-article-title" title="mixed-article-title">The iron-sulfur clusters in Escherichia coli succinate dehydrogenase direct electron flow</span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2006</span>, <span class="volume" tagx="volume" title="volume">281</span>, <span class="fpage" tagx="fpage" title="fpage">27662</span>â€"<span class="lpage" tagx="lpage" title="lpage">27668</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.m604900200">10.1074/jbc.m604900200</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16864590">16864590</a></span></span></li> <li tag="ref"><a name="ref44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimizu</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="mixed-article-title" title="mixed-article-title">Regulation systems of bacteria such as Escherichia coli in response to nutrient limitation and environmental stresses</span>. <span class="source" tagx="source" title="source">Metabolites</span><span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">35</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/metabo4010001">10.3390/metabo4010001</a></span>.</span></li> <li tag="ref"><a name="ref45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Folsom</span><span class="given-names" tagx="given-names" title="given-names">J. P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">A. E.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlson</span><span class="given-names" tagx="given-names" title="given-names">R. P.</span></span><span class="mixed-article-title" title="mixed-article-title">Physiological and proteomic analysis of Escherichia coli iron-limited chemostat growth</span>. <span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">196</span>, <span class="fpage" tagx="fpage" title="fpage">2748</span>â€"<span class="lpage" tagx="lpage" title="lpage">2761</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/jb.01606-14">10.1128/jb.01606-14</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24837288">24837288</a></span></span></li> <li tag="ref"><a name="ref46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Isarankura-Na-Ayudhya</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Isarankura-Na-Ayudhya</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Treeratanapiboon</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasikun</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thipkeaw</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prachayasittikul</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="mixed-article-title" title="mixed-article-title">Proteomic profiling of Escherichia coli in response to heavy metals stress</span>. <span class="source" tagx="source" title="source">Eur. J. Sci. Res.</span><span class="year" tagx="year" title="year">2009</span>, <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">679</span>â€"<span class="lpage" tagx="lpage" title="lpage">688</span>.</span></li> <li tag="ref"><a name="ref47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gunnewijk</span><span class="given-names" tagx="given-names" title="given-names">M. G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van den Bogaard</span><span class="given-names" tagx="given-names" title="given-names">P. T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veenhoff</span><span class="given-names" tagx="given-names" title="given-names">L. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heuberger</span><span class="given-names" tagx="given-names" title="given-names">E. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Vos</span><span class="given-names" tagx="given-names" title="given-names">W. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kleerebezem</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuipers</span><span class="given-names" tagx="given-names" title="given-names">O. P.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poolman</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="mixed-article-title" title="mixed-article-title">Hierarchical control versus autoregulation of carbohydrate utilization in bacteria</span>. <span class="source" tagx="source" title="source">J. Mol. Microbiol. Biotechnol.</span><span class="year" tagx="year" title="year">2001</span>, <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">401</span>â€"<span class="lpage" tagx="lpage" title="lpage">413</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11361071">11361071</a></span></span></li> <li tag="ref"><a name="ref48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ried</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henning</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="mixed-article-title" title="mixed-article-title">A unique amino acid substitution in the outer membrane protein OmpA causes conjugation deficiency in <i>Escherichia coli</i> K-12</span>. <span class="source" tagx="source" title="source">FEBS Lett.</span><span class="year" tagx="year" title="year">1987</span>, <span class="volume" tagx="volume" title="volume">223</span>, <span class="fpage" tagx="fpage" title="fpage">387</span>â€"<span class="lpage" tagx="lpage" title="lpage">390</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0014-5793(87)80324-1">10.1016/0014-5793(87)80324-1</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3311813">3311813</a></span></span></li> <li tag="ref"><a name="ref49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D.-f.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiang</span><span class="given-names" tagx="given-names" title="given-names">Z.-m.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S.-y.</span></span><span class="mixed-article-title" title="mixed-article-title">Functional characterisation of altered outer membrane proteins for tetracycline resistance in <i>Escherichia coli</i></span>. <span class="source" tagx="source" title="source">Int. J. Antimicrob. Agents</span><span class="year" tagx="year" title="year">2008</span>, <span class="volume" tagx="volume" title="volume">32</span>, <span class="fpage" tagx="fpage" title="fpage">315</span>â€"<span class="lpage" tagx="lpage" title="lpage">319</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijantimicag.2008.04.015">10.1016/j.ijantimicag.2008.04.015</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18620846">18620846</a></span></span></li> <li tag="ref"><a name="ref50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lok</span><span class="given-names" tagx="given-names" title="given-names">C.-N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">C.-M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Q.-Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">W.-Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tam</span><span class="given-names" tagx="given-names" title="given-names">P. K.-H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiu</span><span class="given-names" tagx="given-names" title="given-names">J.-F.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Che</span><span class="given-names" tagx="given-names" title="given-names">C.-M.</span></span><span class="mixed-article-title" title="mixed-article-title">Proteomic analysis of the mode of antibacterial action of silver nanoparticles</span>. <span class="source" tagx="source" title="source">J. Proteome Res.</span><span class="year" tagx="year" title="year">2006</span>, <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">916</span>â€"<span class="lpage" tagx="lpage" title="lpage">924</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/pr0504079">10.1021/pr0504079</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16602699">16602699</a></span></span></li> <li tag="ref"><a name="ref51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">Y. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">A. M. C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gethings</span><span class="given-names" tagx="given-names" title="given-names">L. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">M. T.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">C. M. N.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">M. Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">Y. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DjuriÅ¡iÄ‡</span><span class="given-names" tagx="given-names" title="given-names">A. B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">P. K. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">W. K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">L. H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">D. L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">A. P. Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">F. C. C.</span></span><span class="mixed-article-title" title="mixed-article-title">Mechanisms of antibacterial activity of MgO: non-ROS mediated toxicity of MgO nanoparticles towards Escherichia coli</span>. <span class="source" tagx="source" title="source">Small</span><span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">1171</span>â€"<span class="lpage" tagx="lpage" title="lpage">1183</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/smll.201302434">10.1002/smll.201302434</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24344000">24344000</a></span></span></li> <li tag="ref"><a name="ref52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">K.-S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J.-Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwon</span><span class="given-names" tagx="given-names" title="given-names">S.-Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H. J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">E.-K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shin</span><span class="given-names" tagx="given-names" title="given-names">M.-H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahn</span><span class="given-names" tagx="given-names" title="given-names">K. S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Um</span><span class="given-names" tagx="given-names" title="given-names">J. Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J.-H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jang</span><span class="given-names" tagx="given-names" title="given-names">H.-J.</span></span><span class="mixed-article-title" title="mixed-article-title">Comparative transcriptomic analysis of the multi-targeted effects of the herbal extracts against Escherichia coli O157: H7</span>. <span class="source" tagx="source" title="source">BioChip J.</span><span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">379</span>â€"<span class="lpage" tagx="lpage" title="lpage">390</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s13206-012-6410-2">10.1007/s13206-012-6410-2</a></span>.</span></li> <li tag="ref"><a name="ref53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">CrabbÃ©</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ostyn</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Staelens</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigauts</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Risseeuw</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhaenens</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daled</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Acker</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deforce</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Calenbergh</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coenye</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="mixed-article-title" title="mixed-article-title">Host metabolites stimulate the bacterial proton motive force to enhance the activity of aminoglycoside antibiotics</span>. <span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="year" tagx="year" title="year">2019</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e1007697</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1007697">10.1371/journal.ppat.1007697</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31034512">31034512</a></span></span></li> <li tag="ref"><a name="ref54" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perez</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pauli</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bazerque</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="mixed-article-title" title="mixed-article-title">An antibiotic assay by the agar well diffusion method</span>. <span class="source" tagx="source" title="source">Acta Biol. Med. Exp.</span><span class="year" tagx="year" title="year">1990</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">113</span>â€"<span class="lpage" tagx="lpage" title="lpage">115</span>.</span></li> <li tag="ref"><a name="ref55" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheesbrough</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="source" tagx="source" title="source">District Laboratory Practice in Tropical Countries</span>, low price ed.; <span class="publisher-name" tagx="publisher-name" title="publisher-name">The press syndicate of the University of Cambridge</span>: <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Trumpington Street Cambridge</span>, <span class="year" tagx="year" title="year">2000</span>; p. <span class="fpage" tagx="fpage" title="fpage">157</span>.</span></li> <li tag="ref"><a name="ref56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dash</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mandal</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adhikary</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chattopadhyay</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tripathy</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dey</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manna</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dey</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title">Green-synthesized silver nanoparticles kill virulent multidrug-resistant <i>Pseudomonas aeruginosa</i> strains: A mechanistic study</span>. <span class="source" tagx="source" title="source">BLDE Univ. J. Health Sciences</span><span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">1</span>, <span class="fpage" tagx="fpage" title="fpage">89</span>â€"<span class="lpage" tagx="lpage" title="lpage">101</span>. <span class="pub-id"><a href="https://dx.doi.org/10.4103/2468-838X.196087">10.4103/2468-838X.196087</a></span>.</span></li> <li tag="ref"><a name="ref57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naqvi</span><span class="given-names" tagx="given-names" title="given-names">S. Z.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiran</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">M. I.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jamal</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hameed</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="mixed-article-title" title="mixed-article-title">Combined efficacy of biologically synthesized silver nanoparticles and different antibiotics against multidrug-resistant bacteria</span>. <span class="source" tagx="source" title="source">Int. J. Nanomed.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">3187</span>â€"<span class="lpage" tagx="lpage" title="lpage">3195</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2147/ijn.s49284">10.2147/ijn.s49284</a></span>.</span></li> <li tag="ref"><a name="ref58" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alfadda</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benabdelkamel</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masood</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moustafa</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sallam</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bassas</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duncan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Proteomic analysis of mature adipo cytes from obese patients in relation to aging</span>. <span class="source" tagx="source" title="source">Exp. Gerontol.</span><span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">1196</span>â€"<span class="lpage" tagx="lpage" title="lpage">1203</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.exger.2013.07.008">10.1016/j.exger.2013.07.008</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23886751">23886751</a></span></span></li> <li tag="ref"><a name="ref59" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benabdelkamel</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masood</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almidani</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alsadhan</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bassas</span><span class="given-names" tagx="given-names" title="given-names">A. F.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duncan</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alfadda</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span><span class="mixed-article-title" title="mixed-article-title">Mature adipocyte proteome reveals differentially altered protein abundances between lean, overweight and morbidly obese human subjects</span>. <span class="source" tagx="source" title="source">Mol. Cell. Endocrinol.</span><span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">401</span>, <span class="fpage" tagx="fpage" title="fpage">142</span>â€"<span class="lpage" tagx="lpage" title="lpage">154</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mce.2014.11.021">10.1016/j.mce.2014.11.021</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25498962">25498962</a></span></span></li> <li tag="ref"><a name="ref60" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shevchenko</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilm</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vorm</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Mass spectrometric sequencing of proteins from silver-stained polyacrylamide gels</span>. <span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">1996</span>, <span class="volume" tagx="volume" title="volume">68</span>, <span class="fpage" tagx="fpage" title="fpage">850</span>â€"<span class="lpage" tagx="lpage" title="lpage">858</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ac950914h">10.1021/ac950914h</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8779443">8779443</a></span></span></li> <li tag="ref"><a name="ref61" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alfadda</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benabdelkamel</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masood</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jammah</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ekhzaimy</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">Differences in the plasma proteome of patients with hypothyroidism before and after thyroid hormone replacement: A proteomic analysis</span>. <span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">88</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms19010088">10.3390/ijms19010088</a></span>.</span></li> <li tag="ref"><a name="ref62" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benabdelkamel</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masood</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okla</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Naami</span><span class="given-names" tagx="given-names" title="given-names">M. Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alfadda</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span><span class="mixed-article-title" title="mixed-article-title">A proteomics-based approach reveals differential regulation of urine proteins between metabolically healthy and unhealthy obese patients</span>. <span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="year" tagx="year" title="year">2019</span>, <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">4905</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms20194905">10.3390/ijms20194905</a></span>.</span></li> <li tag="ref"><a name="ref63" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babicki</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arndt</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marcu</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grant</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maciejewski</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wishart</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span><span class="mixed-article-title" title="mixed-article-title">Heatmapper: web-enabled heat mapping for all</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">W147</span>â€"<span class="lpage" tagx="lpage" title="lpage">W153</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw419">10.1093/nar/gkw419</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27190236">27190236</a></span></span></li> </ul> </div> </div>  </body></html>