<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Genes (Basel)</span><span class="iso-abbrev" title="iso-abbrev">Genes (Basel)</span><span class="publisher-id" title="publisher-id">genes</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Genes</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2073-4425</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6826867">6826867</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/genes10100739">10.3390/genes10100739</a></span><span class="publisher-id" title="publisher-id">genes-10-00739</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Genomic Analyses Identify Novel Molecular Signatures Specific for the <i>Caenorhabditis</i> and other Nematode Taxa Providing Novel Means for Genetic and Biochemical Studies</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khadka</span><span class="given-names" tagx="given-names" title="given-names">Bijendra</span></span><a href="#af1-genes-10-00739">1</a><a href="#fn1-genes-10-00739">†</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-7049-3918</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chatterjee</span><span class="given-names" tagx="given-names" title="given-names">Tonuka</span></span><a href="#af1-genes-10-00739">1</a><a href="#fn1-genes-10-00739">†</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-8572-7054</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">Bhagwati P.</span></span><a href="#af2-genes-10-00739">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">Radhey S.</span></span><a href="#af1-genes-10-00739">1</a><a href="#c1-genes-10-00739">*</a></span></div><span class="citation_author_institution" id="af1-genes-10-00739">[1], <span class="email" tagx="email" title="email">khadkab@mcmaster.ca</span><span class="email" tagx="email" title="email">tinaburi02@hotmail.com</span></span><span class="citation_author_institution" id="af2-genes-10-00739">[2], <span class="email" tagx="email" title="email">guptab@mcmaster.ca</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">gupta@mcmaster.ca</span></div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">†</span><p>These authors contributed equally to this work.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-10</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">739</span><span class="history" title="history"><span class="received" title="received">received: 2019-7-26</span><span class="accepted" title="accepted">accepted: 2019-9-17</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The phylum Nematoda encompasses numerous free-living as well as parasitic members, including the widely used animal model <i>Caenorhabditis elegans</i>, with significant impact on human health, agriculture, and environment. In view of the importance of nematodes, it is of much interest to identify novel molecular characteristics that are distinctive features of this phylum, or specific taxonomic groups/clades within it, thereby providing innovative means for diagnostics as well as genetic and biochemical studies. Using genome sequences for 52 available nematodes, a robust phylogenetic tree was constructed based on concatenated sequences of 17 conserved proteins. The branching of species in this tree provides important insights into the evolutionary relationships among the studied nematode species. In parallel, detailed comparative analyses on protein sequences from nematodes (<i>Caenorhabditis</i>) species reported here have identified 52 novel molecular signatures (or synapomorphies) consisting of conserved signature indels (CSIs) in different proteins, which are uniquely shared by the homologs from either all genome-sequenced <i>Caenorhabditis</i> species or a number of higher taxonomic clades of nematodes encompassing this genus. Of these molecular signatures, 39 CSIs in proteins involved in diverse functions are uniquely present in all <i>Caenorhabditis</i> species providing reliable means for distinguishing this group of nematodes in molecular terms. The remainder of the CSIs are specific for a number of higher clades of nematodes and offer important insights into the evolutionary relationships among these species. The structural locations of some of the nematodes-specific CSIs were also mapped in the structural models of the corresponding proteins. All of the studied CSIs are localized within the surface-exposed loops of the proteins suggesting that they may potentially be involved in mediating novel protein–protein or protein–ligand interactions, which are specific for these groups of nematodes. The identified CSIs, due to their exclusivity for the indicated groups, provide reliable means for the identification of species within these nematodes groups in molecular terms. Further, due to the predicted roles of these CSIs in cellular functions, they provide important tools for genetic and biochemical studies in <i>Caenorhabditis</i> and other nematodes.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
genome sequences</div> <div class="kwd" title="kwd">
molecular markers (synapomorphies)</div> <div class="kwd" title="kwd">
phylogenetic trees</div> <div class="kwd" title="kwd">
conserved signature indels</div> <div class="kwd" title="kwd"><i>Caenorhabditis elegans</i></div> <div class="kwd" title="kwd"><i>Chromadorea</i></div> <div class="kwd" title="kwd">
structural analysis of <i>Caenorhabditis</i>/nematodes-specific indels</div> <div class="kwd" title="kwd">
evolutionary relationships among nematodes</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Animals of the phylum Nematoda represent a large and diverse group of eukaryotes present in various marine, freshwater, and terrestrial ecosystems [<a href="#B1-genes-10-00739">1</a>,<a href="#B2-genes-10-00739">2</a>,<a href="#B3-genes-10-00739">3</a>]. Of the &amp;gt;1 million nematode species that are indicated to exist, approximately 25000 species are currently recognized [<a href="#B3-genes-10-00739">3</a>,<a href="#B4-genes-10-00739">4</a>,<a href="#B5-genes-10-00739">5</a>]. Most nematodes are transparent and small organisms. However, some can grow to lengths of several meters [<a href="#B3-genes-10-00739">3</a>,<a href="#B6-genes-10-00739">6</a>]. There are two major types of nematodes in terms of their trophic ecology, i.e., those which are free-living and others which are parasitic [<a href="#B1-genes-10-00739">1</a>,<a href="#B5-genes-10-00739">5</a>,<a href="#B6-genes-10-00739">6</a>,<a href="#B7-genes-10-00739">7</a>]. Free-living nematodes are found in all types of environments and feed on bacteria, algae, or fungi [<a href="#B5-genes-10-00739">5</a>,<a href="#B6-genes-10-00739">6</a>,<a href="#B8-genes-10-00739">8</a>]. Parasitic nematodes occupy and obtain nutrients from various host organisms including animals, plants, and insects [<a href="#B1-genes-10-00739">1</a>,<a href="#B6-genes-10-00739">6</a>,<a href="#B8-genes-10-00739">8</a>,<a href="#B9-genes-10-00739">9</a>,<a href="#B10-genes-10-00739">10</a>,<a href="#B11-genes-10-00739">11</a>].</p> <p>Nematodes play significant roles in diverse environments and several parasitic species cause extensive damage to agricultural crops, harm livestock and are also a threat to humans [<a href="#B1-genes-10-00739">1</a>,<a href="#B2-genes-10-00739">2</a>,<a href="#B5-genes-10-00739">5</a>]. For example, root-knot nematodes from the genus <i>Meloidogyne</i> damage soybean, potato, and sugar beet crops, resulting in large losses to the agricultural industry [<a href="#B2-genes-10-00739">2</a>]. <i>Haemonchus contortus</i> and <i>Ascaris suum</i> are animal parasitic species known to infect sheep and pigs, respectively [<a href="#B6-genes-10-00739">6</a>]. Moreover, some nematode species belonging to the genera <i>Trichinella</i> and <i>Trichuris</i> infect humans and can cause severe gastrointestinal problems, which, in some cases, can result in death [<a href="#B12-genes-10-00739">12</a>,<a href="#B13-genes-10-00739">13</a>]. However, some nematode species, specifically <i>Caenorhabditis elegans</i> (<i>C. elegans</i>), have found wide-spread usage as important model organisms for studies related to cellular development, aging and other genetic, biochemical, and cell biological studies [<a href="#B14-genes-10-00739">14</a>,<a href="#B15-genes-10-00739">15</a>,<a href="#B16-genes-10-00739">16</a>,<a href="#B17-genes-10-00739">17</a>,<a href="#B18-genes-10-00739">18</a>,<a href="#B19-genes-10-00739">19</a>,<a href="#B20-genes-10-00739">20</a>]. Among the many advantages of <i>C. elegans</i> as a model system, it is a transparent microscopic organism with nervous system and all its neurons have been mapped. Further, it has a short life cycle and it is easy and inexpensive to maintain in a lab, and can be readily manipulated genetically [<a href="#B2-genes-10-00739">2</a>,<a href="#B3-genes-10-00739">3</a>,<a href="#B21-genes-10-00739">21</a>,<a href="#B22-genes-10-00739">22</a>]. More importantly, <i>C. elegans</i> contains a number of genes that are similar and homologous to the human disease genes, making it an ideal organism to study human diseases in an animal model [<a href="#B2-genes-10-00739">2</a>,<a href="#B3-genes-10-00739">3</a>,<a href="#B15-genes-10-00739">15</a>,<a href="#B17-genes-10-00739">17</a>,<a href="#B22-genes-10-00739">22</a>,<a href="#B23-genes-10-00739">23</a>,<a href="#B24-genes-10-00739">24</a>,<a href="#B25-genes-10-00739">25</a>,<a href="#B26-genes-10-00739">26</a>]. The global concerns of animal and plant-parasitic nematodes, as well as the medical and agricultural applications of other free-living and parasitic nematodes, underscore the need to understand the evolutionary relationships as well as novel characteristics of different groups of nematodes.</p> <p>Earlier studies on the classification of nematodes were based on morphological characteristics [<a href="#B1-genes-10-00739">1</a>,<a href="#B5-genes-10-00739">5</a>,<a href="#B6-genes-10-00739">6</a>,<a href="#B7-genes-10-00739">7</a>]. However, as most of the studied morphological traits were homoplasious (i.e., shared presence was not due to common ancestry), the resulting classification was misleading [<a href="#B1-genes-10-00739">1</a>,<a href="#B6-genes-10-00739">6</a>,<a href="#B27-genes-10-00739">27</a>]. In recent years, phylogenetic studies employing predominantly 18S and 28S ribosomal RNA (rRNA) genes and mitochondrial DNA have been used to examine the evolutionary relationships among nematodes [<a href="#B1-genes-10-00739">1</a>,<a href="#B2-genes-10-00739">2</a>,<a href="#B4-genes-10-00739">4</a>,<a href="#B5-genes-10-00739">5</a>,<a href="#B6-genes-10-00739">6</a>,<a href="#B21-genes-10-00739">21</a>,<a href="#B28-genes-10-00739">28</a>,<a href="#B29-genes-10-00739">29</a>,<a href="#B30-genes-10-00739">30</a>,<a href="#B31-genes-10-00739">31</a>,<a href="#B32-genes-10-00739">32</a>]. However, these studies are often unable to discriminate between species of higher-level nematode taxa [<a href="#B33-genes-10-00739">33</a>,<a href="#B34-genes-10-00739">34</a>]. Further, as the branching of species in phylogenetic trees is affected by large numbers of variables [<a href="#B35-genes-10-00739">35</a>,<a href="#B36-genes-10-00739">36</a>,<a href="#B37-genes-10-00739">37</a>,<a href="#B38-genes-10-00739">38</a>,<a href="#B39-genes-10-00739">39</a>], additional more reliable means for distinguishing different main groups of nematodes are needed. Currently, very few reliable molecular characteristics are known that are specific for the genus <i>Caenorhabditis</i> or other nematodes, which could be used to confidently discriminate important groups of nematodes in molecular terms.</p> <p>Genome sequences are currently available for 52 nematode species, providing good coverage of several important groups within the phylum Nematoda [<a href="#B40-genes-10-00739">40</a>]. These sequences serve as a valuable resource for a more reliable understanding of the evolutionary relationships amongst the species [<a href="#B41-genes-10-00739">41</a>] and for identifying novel molecular characteristics that are uniquely shared within specific groups/clades of nematodes. Additionally, the sequence data offers powerful means for genetic, biochemical studies, and other types of studies including identification of novel drug targets [<a href="#B9-genes-10-00739">9</a>,<a href="#B11-genes-10-00739">11</a>,<a href="#B30-genes-10-00739">30</a>,<a href="#B42-genes-10-00739">42</a>,<a href="#B43-genes-10-00739">43</a>,<a href="#B44-genes-10-00739">44</a>]. One important class of molecular markers whose discovery has been facilitated by genome sequence analyses is comprised of conserved signature indels (insertions/deletions) (CSIs) in gene/protein sequences that are uniquely shared by an evolutionarily related group of species [<a href="#B37-genes-10-00739">37</a>,<a href="#B38-genes-10-00739">38</a>,<a href="#B45-genes-10-00739">45</a>,<a href="#B46-genes-10-00739">46</a>]. The CSIs that are useful for evolutionary studies are generally of specific lengths, present at specific positions in particular genes/proteins, and they are flanked on both sides by conserved regions to ensure that they constitute reliable characteristics [<a href="#B37-genes-10-00739">37</a>,<a href="#B47-genes-10-00739">47</a>,<a href="#B48-genes-10-00739">48</a>,<a href="#B49-genes-10-00739">49</a>,<a href="#B50-genes-10-00739">50</a>]. The CSIs in gene/protein sequences generally result from rare genetic changes and they have provided important means for demarcation of different groups of organisms in molecular terms [<a href="#B37-genes-10-00739">37</a>,<a href="#B38-genes-10-00739">38</a>,<a href="#B45-genes-10-00739">45</a>,<a href="#B48-genes-10-00739">48</a>,<a href="#B51-genes-10-00739">51</a>]. Further, based upon their presence or absence in different species, important inferences regarding the evolutionary relationships among a given group of species can be derived [<a href="#B37-genes-10-00739">37</a>,<a href="#B38-genes-10-00739">38</a>,<a href="#B45-genes-10-00739">45</a>].</p> <p>In the present study, we have used the genome sequences of 52 nematode species to construct a phylogenetic tree for the nematodes based on concatenated sequences of 17 conserved proteins. This tree provides important insights into the evolutionary relationships amongst the nematodes, and a number of major groups/taxa within the phylum Nematoda are reliably resolved. More importantly, our comparative genomic analysis of the protein sequences of <i>Caenorhabditis</i> species has uncovered 52 molecular signatures comprising of CSIs in diverse proteins that are uniquely shared by either all sequenced <i>Caenorhabditis</i> species or by several higher taxa of nematodes encompassing this genus. Of these molecular markers, 39 CSIs in proteins involved in diverse functions are distinctive characteristics of homologs from all six genome-sequenced <i>Caenorhabditis</i> species. The described molecular markers, due to their exclusivity for the specific groups of nematodes, provide useful means for the development of novel diagnostics as well as for genetic and biochemical studies on this important group of organisms.</p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Methods</div> <div class="2.1.constructionofphylogenetictrees" title="sec"> <div class="title" tagx="title" title="title">
2.1. Construction of Phylogenetic Trees</div> <p>To construct a phylogenetic tree for 52 genome sequenced nematode species, sequences of 17 conserved proteins involved in a variety of cellular functions, which were present in a single copy in these genomes were identified (<a href="#app1-genes-10-00739">Table S1</a>). Sequences for four outgroup species viz. <i>Cryptosporidium muris, Plasmodium falciparum, Babesia sp. Xinjiang</i> and <i>Eimeria necatrix</i>, were used for the rooting of the tree. The phylogenetic tree construction was carried out using an internally developed pipeline described in our earlier work [<a href="#B52-genes-10-00739">52</a>]. Briefly, the CD-HIT program was used [<a href="#B53-genes-10-00739">53</a>] to identify protein families sharing a minimum of 50% in sequence identity and sequence length and which were found in at least 80% of the input genomes. The Clustal Omega [<a href="#B54-genes-10-00739">54</a>] algorithm was used to generate multiple sequence alignment (MSA) of these protein families. The aligned protein families were trimmed with TrimAl [<a href="#B55-genes-10-00739">55</a>] to remove poorly aligned regions [<a href="#B56-genes-10-00739">56</a>] before concatenation to the other proteins. This concatenated sequence alignment consisting of 10764 aligned amino acids positions was used for phylogenetic analysis. An approximate maximum likelihood (ML) tree based on this sequence alignment was initially constructed in FastTree 2 [<a href="#B57-genes-10-00739">57</a>] using the Whelan and Goldman model of protein sequence evolution [<a href="#B58-genes-10-00739">58</a>]. The resulting tree was then used as input for RAxML [<a href="#B59-genes-10-00739">59</a>], where the Le and Gascuel model of protein sequence evolution [<a href="#B60-genes-10-00739">60</a>] in RAxML 8 to optimize individual branch lengths and to identify the optimal maximum-likelihood topology. Optimization of the robustness of the tree was completed by conducting SH tests [<a href="#B61-genes-10-00739">61</a>] in RAxML 8 [<a href="#B59-genes-10-00739">59</a>]. The sequence alignment created by the above program was also used to construct an ML tree based on 100 bootstrap replicates in MEGA6 [<a href="#B62-genes-10-00739">62</a>] using Whelan and Goldman +Freq. model [<a href="#B58-genes-10-00739">58</a>] and JTT matrix-based model [<a href="#B62-genes-10-00739">62</a>]. </p> </div> <div class="2.2.identificationofconservedsignatureindels(csis)" title="sec"> <div class="title" tagx="title" title="title">
2.2. Identification of Conserved Signature Indels (CSIs)</div> <p>To identify potential CSIs specific for different groups within the phylum Nematoda, BLASTp searches were performed on &amp;gt;11800 proteins from <i>Caenorhabditis elegans</i> genome (from accession number NP_001033396.1 to NP_001343573.1) covering approximately 40% of the annotated proteins. Based on these blast searches, proteins for which high scoring homologs (E-values less than 1e<sup>−20</sup>) were present in multiple nematode species, as well as several non-nematode organisms, were identified and the sequences of these proteins from 15–25 species were retrieved. It was not essential that sequences from any Apicomplexa species (used to root the phylogenetic tree) be present among the sequences for non-nematode species. Multiple sequence alignments for the selected protein sequences were created using CLUSTAL_X 2.1 [<a href="#B63-genes-10-00739">63</a>] and these alignments were examined manually to identify insertions or deletions (indels), which were flanked by at least four to five conserved amino acid residues on both sides within the neighboring 40–50 residues [<a href="#B37-genes-10-00739">37</a>,<a href="#B47-genes-10-00739">47</a>,<a href="#B64-genes-10-00739">64</a>]. Indels which were not flanked by conserved regions were not further considered as they do not provide reliable molecular characteristics. Query sequences encompassing the indel and its flanking 40–50 amino acids were collected for all potential CSIs. Afterward, the query sequences underwent another BLASTp search carried out against the NCBI nr database. The resulting top 250–500 hits for all queries were examined to identify CSIs that are uniquely found in the nematode species as well as to evaluate the group specificities of these CSIs. Signature files for all useful CSIs, which were specifically found in the indicated nematode groups, were created using SIG_CREATE and SIG_STYLE programs described in our earlier work [<a href="#B47-genes-10-00739">47</a>] that are available on the GLEANS (Gleans.net) server. The CSIs reported here, unless otherwise indicated, are specific for all members of the indicated groups whose homologs were detected by BLASTp searches.</p> </div> <div class="2.3.homologymodellingandanalysisofproteinstructures" title="sec"> <div class="title" tagx="title" title="title">
2.3. Homology Modelling and Analysis of Protein Structures</div> <p>Homology models of some proteins which contain the CSIs were created for the <i>C. elegans</i> homolog to map the locations of the CSIs in the proteins’ structures. The homology models of the <i>C. elegans</i> Rab44 protein, poly ADP-ribose glycohydrolase protein and tRNA guanine N methyltransferase proteins were created using the solved structures of the following template proteins PDB ID: 2p5s (human), PDB ID: 6hmm (human) [<a href="#B65-genes-10-00739">65</a>] and PDB ID: 4jwg (from (<i>Schizosaccharomyces pombe</i>) [<a href="#B66-genes-10-00739">66</a>], respectively. Homology modeling was performed using MODELLER v9.15 [<a href="#B67-genes-10-00739">67</a>] and the top 500 models were ranked on the basis of their discrete optimized protein energy (DOPE) scores [<a href="#B68-genes-10-00739">68</a>]. The stereo-chemical properties of the final models were assessed using three independent servers: ERRAT, PROSA, and Verify3D [<a href="#B69-genes-10-00739">69</a>,<a href="#B70-genes-10-00739">70</a>,<a href="#B71-genes-10-00739">71</a>,<a href="#B72-genes-10-00739">72</a>]. These applications utilize a dataset of refined structures to evaluate the statistical significance of the models’ conformation, location, environment of each amino acid sequence and overall structural stability. Selected models were then refined using ModRefiner [<a href="#B73-genes-10-00739">73</a>]. These resultant models were then used to explore the structural changes associated with the insertion. The superimposition of the validated models with the template structures was carried out using PyMOL (<span class="uri" title="uri"><a href="http://www.pymol.org">http://www.pymol.org</a></span>) to examine the structure and location of identified CSIs in the modeled protein structures. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
3. Results</div> <div class="3.1.phylogeneticanalysisofnematodesbasedonconcatenatedsequencesofconservedproteins" title="sec"> <div class="title" tagx="title" title="title">
3.1. Phylogenetic Analysis of Nematodes Based on Concatenated Sequences of Conserved Proteins</div> <p>Evolutionary relationships of the nematodes species in the past have been mainly studied based on gene sequences for 18S or 28S rRNA and mitochondrial proteins [<a href="#B2-genes-10-00739">2</a>,<a href="#B4-genes-10-00739">4</a>,<a href="#B6-genes-10-00739">6</a>,<a href="#B34-genes-10-00739">34</a>]. Genome sequences are now available for 52 nematodes species covering a number of major groups/taxa within this phylum. These sequences can be used to examine the phylogenetic relationships among nematode species based on concatenated sequences for multiple conserved proteins. Hence, a maximum-likelihood (ML) phylogenetic tree was constructed for the 52 genome-sequenced nematodes species based on concatenated sequences of 17 conserved proteins. The proteins employed in these analyses, listed in <a href="#app1-genes-10-00739">Table S1</a>, are present in a single copy in the available nematodes genomes. The resulting bootstrapped tree, which was rooted using homologous sequences from representative Apicomplexa species, is shown in <a href="#genes-10-00739-f001">Figure 1</a>. In this tree, members from the two main classes within the phylum Nematoda, i.e., <i>Chromadorea</i> and <i>Enoplea</i>, were clearly separated from each other. </p> <p>Within the class <i>Chromadorea</i>, the species from the two suborders <i>Rhabditina</i> and <i>Spirulina</i> were also separated from each other. Additionally, species from a number of nematode genera for which sequences were available from multiple species viz. <i>Caenorhabditis, Ancyclostoma, Trichinella, Trichuris,</i> and <i>Brugia</i>, also formed monophyletic clades supporting the close relationships of species within these genera. However, in the constructed tree, species from the superfamilies <i>Rhabiditoidea, Strongyloidea</i>, <i>Trichostrongyloidea, Filarioidea,</i> and <i>Ascarioidea</i> exhibited polyphyletic branching within each other. Thus, these families cannot be reliably demarcated on the basis of constructed phylogenetic tree and the interrelationships as well as grouping of species within these families remains unclear at present. The branching patterns, as well as the interrelationships among different nematode species observed in our tree, are similar to that reported recently by Smythe et al. [<a href="#B41-genes-10-00739">41</a>] based on phylogenomic analysis using a conservative orthology inference strategy. We have also constructed ML trees based on our protein sequences using MEGA6 program employing two different amino acid substitution models, and the results obtained (<a href="#app1-genes-10-00739">Figure S1</a>) are very similar to that seen in <a href="#genes-10-00739-f001">Figure 1</a>. However, despite the noted limitations of the tree shown in <a href="#genes-10-00739-f001">Figure 1</a>, it provides a good phylogenetic framework for understanding and analyzing the results obtained from comparative genomic analysis, which are discussed below.</p> </div> <div class="3.2.identificationofconservedsignatureindelsspecificfordifferentnematodegroups" title="sec"> <div class="title" tagx="title" title="title">
3.2. Identification of Conserved Signature Indels Specific for Different Nematode Groups</div> <p>While the phylogenetic tree shown in <a href="#genes-10-00739-f001">Figure 1</a> allows some inferences to be drawn regarding the evolutionary relationships amongst the nematode species, it is important to confirm these inferences using other independent approaches that are also capable of providing further insights into the evolutionary relationships among nematode species. As noted in the introduction, CSIs in protein sequences that are uniquely shared by a given group of organisms provide an important class of molecular markers that have been proven very useful for evolutionary/taxonomic studies [<a href="#B45-genes-10-00739">45</a>,<a href="#B46-genes-10-00739">46</a>,<a href="#B48-genes-10-00739">48</a>,<a href="#B49-genes-10-00739">49</a>,<a href="#B50-genes-10-00739">50</a>,<a href="#B51-genes-10-00739">51</a>]. Due to the rare and discrete nature of the genetic changes that give rise to CSIs, the presence or absence of CSIs in different lineages (or proteins) is generally not affected by the factors that can confound or limit the reliability of inferences from phylogenetic trees [<a href="#B41-genes-10-00739">41</a>,<a href="#B45-genes-10-00739">45</a>,<a href="#B47-genes-10-00739">47</a>,<a href="#B48-genes-10-00739">48</a>,<a href="#B50-genes-10-00739">50</a>]. Hence, the CSIs provide powerful means for demarcating different groups of organisms in molecular terms and for understanding evolutionary relationships. Therefore, a major focus of the present study was to perform comprehensive genomic analysis of protein sequences from <i>Caenorhabditis</i> species to identify CSIs that are specific for this genus as well as other higher taxonomic groups/taxa of nematodes encompassing these organisms. The results of our analysis, reported below, have led to the identification of 52 novel molecular signatures in the form of CSIs that are uniquely shared by either all <i>Caenorhabditis</i> species or different nematodes groups belonging to this phylum. A brief description of the characteristics of the identified CSIs is provided below.</p> <p>Of the identified CSIs, 39 CSIs within proteins involved in diverse cellular functions are specifically found in the protein homologs of <i>Caenorhabditis</i> species, which form a strongly supported monophyletic clade in our phylogenetic tree. Two examples of such CSIs, one consisting of a 1 amino acid (aa) insertion in Rab44 protein (<i>C. elegans</i> gene number 4R79.2) and another comprising a 5 aa insertion in a poly ADP-ribose glycohydrolase protein (PARG-1) are shown in <a href="#genes-10-00739-f002">Figure 2</a>A,B, respectively. As seen from <a href="#genes-10-00739-f002">Figure 2</a>, both these CSIs are present in conserved regions of the proteins and they are commonly shared by the homologs of all six <i>Caenorhabditis</i> species with available genome sequences, but not found in the homologs from other nematodes or non-nematode organisms. Of the two proteins harboring these CSIs, Rab44 is a GTPase of Rab family (Ras superfamily). Although 4R79.2 is yet to be genetically characterized (<span class="uri" title="uri"><a href="www.wormbase.org">www.wormbase.org</a></span>), members of the Rab family act as molecular switches in vesicle trafficking and are known to interact with several other molecules at different trafficking stage [<a href="#B74-genes-10-00739">74</a>,<a href="#B75-genes-10-00739">75</a>,<a href="#B76-genes-10-00739">76</a>]. The protein PARG-1 is a member of poly ADP-ribose glycohydrolase (PARG) family. PARG is a primary enzyme responsible for hydrolyzing the poly(ADP-ribose) polymer synthesized by poly-(ADP-ribose) polymerases and is involved in a variety of nuclear processes such as DNA damage response, development, programmed cell death and aging [<a href="#B65-genes-10-00739">65</a>,<a href="#B77-genes-10-00739">77</a>]. </p> <p>In addition to the two CSIs shown in <a href="#genes-10-00739-f002">Figure 2</a>, our study has identified 37 other CSIs in different proteins, which are also specifically found in the homologs from <i>Caenorhabditis</i> species. Sequence information for these other CSIs is provided in <a href="#app1-genes-10-00739">Figures S3–S39</a> and some of their characteristics are summarized in <a href="#genes-10-00739-t001">Table 1</a>. For some proteins containing these CSIs (viz. an intermediate filament protein, <a href="#app1-genes-10-00739">Figure S39</a>), two homologs are present in <i>Caenorhabditis</i> species and the described CSI was found in only one of the two homologs. In such cases, it is likely that the two sets of homologs originated from a gene duplication event in a common ancestor of <i>Caenorhabditis</i> and the genetic change leading to the observed CSI occurred at this stage in the ancestor of one of the homologs. Due to the exclusive presence of different CSIs listed in <a href="#genes-10-00739-t001">Table 1</a> in the protein homologs for <i>Caenorhabditis</i> species, the described CSIs provide reliable molecular markers for distinguishing the members of this genus. The genetic changes responsible for these CSIs are postulated to have occurred in a common ancestor of the genus <i>Caenorhabditis</i> during its divergence from other nematodes.</p> <p>Our work has also identified 4 CSIs, which, in addition to the <i>Caenorhabditis</i> species, are also commonly shared by the species <i>Diploscapter pachys</i>. The species from both these genera are part of the family <i>Rhabditoidea</i> [<a href="#B3-genes-10-00739">3</a>]. One such CSI is a 2 aa insertion in a protein annotated as abnormal cell migration protein 13 (MIG-13), which is specifically found in the homologs from the family <i>Rhabditoidea</i> and it is not present in the homologous proteins from other nematodes or other species (see <a href="#genes-10-00739-f003">Figure 3</a>). </p> <p>Although the exact function of the abnormal cell migration protein MIG-13 has not been elucidated, cell migration and morphogenesis are key events in tissue development and organogenesis [<a href="#B18-genes-10-00739">18</a>,<a href="#B20-genes-10-00739">20</a>]. MIG-13 is an evolutionarily conserved transmembrane protein that has been shown to play an important role in cell migration in the Q neuroblast lineage [<a href="#B78-genes-10-00739">78</a>]. MIG-13 acts cell-autonomously to regulate the asymmetric distribution of the actin cytoskeleton in the leading edge of QR descendants [<a href="#B79-genes-10-00739">79</a>,<a href="#B80-genes-10-00739">80</a>]. Thus, the presence of a CSI in this protein, which is specific for the family <i>Rhabditoidea</i> is of much interest. Sequence information for the other three CSIs, which are also specific for the family <i>Rhabditoidea</i> is provided in <a href="#app1-genes-10-00739">Figures S41–S43</a> and information for them is summarized in <a href="#genes-10-00739-t002">Table 2</a>. These CSIs provide reliable evidence supporting a grouping of <i>Diploscapter pachys</i> with the <i>Caenorhabditis</i> species and they can be used to distinguish members of the family <i>Rhabditoidea</i> from other nematodes in molecular terms. </p> <p>The genus <i>Caenorhabditis</i> is embedded within the class <i>Chromadorea</i>, which constitutes one of the two main classes within the phylum Nematoda [<a href="#B3-genes-10-00739">3</a>,<a href="#B81-genes-10-00739">81</a>]. Our analysis has identified eight CSIs in different proteins that are uniquely shared by the homologs from different <i>Chromadorea</i> species but absent in nematodes belonging to the class <i>Enoplea</i> as well as other organisms. Of these eight CSIs, six CSIs are commonly shared in most cases by all genome-sequenced <i>Chromadorea</i> species, whereas in two of them, the described CSIs lack in the species <i>Strongyloides ratti</i> (belonging to the suborder <i>Tylenchina</i>) [<a href="#B30-genes-10-00739">30</a>], which in our phylogenetic tree branches are between the classes <i>Chromadorea</i> and <i>Enoplea.</i> One example of a CSI that is specific for the class <i>Chromadorea</i> is presented in <a href="#genes-10-00739-f004">Figure 4</a>. </p> <p>In the CSI shown in <a href="#genes-10-00739-f004">Figure 4</a>, which is specific for the class <i>Chromodorea</i>, a four aa insertion is present in a conserved region of tRNA (guanine-N(1)-)-methyltransferase, which is encoded by F46F11.10 gene in <i>C. elegans</i>. The human homolog of this protein plays an essential role in the methylation of specific guanine residues in tRNA molecules [<a href="#B82-genes-10-00739">82</a>]. This CSI is uniquely shared by different <i>Chromadorea</i> species, but it is absent in other nematodes as well as different other organisms. Sequence information for the other seven CSIs, which are also specific for the class <i>Chromadorea</i> is presented in <a href="#app1-genes-10-00739">Figures S44–S50</a> and information for them is summarized in <a href="#genes-10-00739-t002">Table 2</a>.</p> <p>Lastly, our analysis has also identified one CSI in a Na(+)/H(+) exchange regulatory factor protein NRFL-1 that appears to be specific for the phylum Nematoda. The <i>nrfl-1</i> gene is expressed in many cells and tissues including excretory cell, intestine, pharynx, and tail [<a href="#B83-genes-10-00739">83</a>]. NRFL-1 binds to an amino acid transporter AAT-6 to help retain localization of AAT-6 on the intestinal luminal membrane in older worms. Partial sequence alignment of NRFL-1 from nematodes species as well as representative outgroups species are shown in <a href="#genes-10-00739-f005">Figure 5</a>. Most nematodes species contain two homologs of <i>nrfl-1</i>. Of these two homologs, one contains a single aa insertion within a conserved region that is specifically found in all nematodes species (<a href="#genes-10-00739-f005">Figure 5</a>). This insert is absent in the other protein homolog as well as in the homologous protein from different outgroup species. The absence of this insertion in the outgroup species indicates that this indel is an insert and the genetic change leading to it was introduced in a common ancestor of the phylum Nematoda. More detailed information regarding the species distribution of this CSI is provided in <a href="#app1-genes-10-00739">Figure S51</a>. </p> </div> <div class="3.3.localizationsofthecsisinproteinstructures" title="sec"> <div class="title" tagx="title" title="title">
3.3. Localizations of the CSIs in Protein Structures</div> <p>Earlier work on CSIs in proteins shows that most of the studied CSIs in proteins are located on the surface exposed loops of proteins [<a href="#B84-genes-10-00739">84</a>,<a href="#B85-genes-10-00739">85</a>,<a href="#B86-genes-10-00739">86</a>]. The surface-exposed loops in proteins are known to play important roles in mediating novel protein–protein or protein–ligands interaction [<a href="#B84-genes-10-00739">84</a>,<a href="#B87-genes-10-00739">87</a>,<a href="#B88-genes-10-00739">88</a>]. In view of these earlier studies, we have also examined the locations of some of the nematodes-specific CSIs identified in the structures of the nematodes proteins. The mapping of the CSIs in protein structures was carried out for three different proteins. These proteins included Rab-44 (4R79.2) and poly ADP-ribose glycohydrolase (PARG-1), which contain one and five aa insertions, respectively, that are specific for the <i>Caenorhabditis</i> species (<a href="#genes-10-00739-f002">Figure 2</a>), and a four aa insertion in the protein tRNA (guanine-N(1)-)-methyltransferase (F46F11.10) (<a href="#genes-10-00739-f004">Figure 4</a>) that is specific for the class <i>Chormadorea</i>. The structural information for these proteins from <i>Caenorhabditis</i> or any other nematode species is presently lacking. However, the structures of their homologs from humans or other eukaryotic organisms exhibiting high sequence similarity to the <i>C. elegans</i> homologs are available [<a href="#B65-genes-10-00739">65</a>,<a href="#B66-genes-10-00739">66</a>] (see Materials and Methods). Using the available structures of these proteins as templates and by means of the homology modeling technique, the structures of the corresponding <i>C. elegans</i> proteins were constructed and validated as detailed in the Methods section. To visualize the locations of the identified CSIs in the structures of these proteins, structural overlaps of the modeled proteins containing the CSIs and the solved structures of the proteins lacking the CSIs were carried out. The results of these studies for the proteins Rab-44 (4R79.2), poly ADP-ribose glycohydrolase (PARG-1) and tRNA (guanine-N(1)-)-methyltransferase (F46F11.10) are presented in <a href="#genes-10-00739-f006">Figure 6</a>A–C, respectively. The locations of the CSIs in the protein structures are shown in red color in this figure. As seen from the presented structural overlaps, the CSIs in all three studied proteins are localized within the surface-exposed loops of these proteins, which is in accordance with the results of earlier studies [<a href="#B84-genes-10-00739">84</a>,<a href="#B85-genes-10-00739">85</a>,<a href="#B86-genes-10-00739">86</a>,<a href="#B87-genes-10-00739">87</a>].</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
4. Discussion</div> <p>Nematodes species are clinically, economically, and scientifically important organisms. In addition to their significance for human health and agricultural industry due to their animal and plant pathogenicity, they provide very useful model organisms for scientific research relevant to human [<a href="#B1-genes-10-00739">1</a>,<a href="#B2-genes-10-00739">2</a>,<a href="#B3-genes-10-00739">3</a>,<a href="#B5-genes-10-00739">5</a>,<a href="#B14-genes-10-00739">14</a>,<a href="#B22-genes-10-00739">22</a>,<a href="#B89-genes-10-00739">89</a>]. Thus, it is of much importance to understand their evolutionary relationships and identify reliable molecular means capable of clearly distinguishing different important groups among nematodes. In this study, we have used available genome sequences of 52 diverse nematode species to examine their evolutionary relationships and have performed a comparative analysis on their protein sequences to identify novel molecular markers that are distinctive characteristics of the <i>Caenorhabditis</i> species as well as other groups of nematodes.</p> <p>Phylogenetic trees based on concatenated sequences for multiple proteins are known to provide a more accurate depiction of the evolutionary relationships among a given group of species than trees based on a single gene/protein sequence [<a href="#B30-genes-10-00739">30</a>,<a href="#B39-genes-10-00739">39</a>,<a href="#B47-genes-10-00739">47</a>,<a href="#B90-genes-10-00739">90</a>,<a href="#B91-genes-10-00739">91</a>]. Hence, a phylogenetic tree for the genome-sequenced nematodes species was constructed in this work based on concatenated sequences of 17 conserved proteins. The tree shows a clear separation of the two main classes, i.e., <i>Chromadorea</i> and <i>Enoplea</i>, within the nematodes [<a href="#B5-genes-10-00739">5</a>]. Recently, Smythe et al. [<a href="#B41-genes-10-00739">41</a>] have also reported phylogenomic analysis of 108 nematodes using a conservative orthology inference strategy. Their analyses also indicated that the class <i>Enoplea</i> formed a sister taxon to the rest of the Nematoda [<a href="#B41-genes-10-00739">41</a>]. In the phylogenetic trees constructed in this work as well as by Smythe et al. [<a href="#B41-genes-10-00739">41</a>], species from a number of nematode genera viz. <i>Caenorhabditis, Ancyclostoma, Trichinella, Trichuris,</i> and <i>Brugia</i>, formed distinct clades supporting their expected close and specific groupings. However, in both these trees, the species from the superfamilies <i>Strongyloidea</i>, <i>Trichostrongyloidea,</i> and <i>Metastrongyloidea</i> were found to cluster closely together and exhibited polyphyletic branching within each other. Thus, the clades corresponding to these superfamilies are reliably discerned presently and their interrelationships are also not resolved.</p> <p>However, the main focus of the present work was on species from the genus <i>Caenorhabditis</i>, which formed a strongly supported monophyletic clade in the tree. Our comparative genomic analysis was aimed at identifying molecular markers that are commonly and uniquely shared by the members of this genus or other larger clades of nematodes which included <i>Caenorhabditis.</i> These studies have identified for the first time 52 novel molecular markers (or synapomorphies) consisting of conserved signature indels (CSIs) in proteins involved in various biological processes, which are uniquely shared by either all available <i>Caenorhabditis</i> species or other higher taxa of nematodes encompassing this genus, provide novel and important tools for studying these organisms. It should be mentioned that Mitreva and coworkers [<a href="#B43-genes-10-00739">43</a>,<a href="#B92-genes-10-00739">92</a>,<a href="#B93-genes-10-00739">93</a>] have previously carried out extensive work examining the presence of indels in nematode proteins. Although their work has identified large numbers of indels in nematode proteins, unlike the CSIs that are the focus of this work, the indels identified by these authors are not specific for a phylogenetically coherent group (i.e., species related by common ancestry), and in most cases, they were also not present in conserved regions. Extensive earlier work shows that only the indels of fixed lengths, which are flanked on both sides by conserved regions and are uniquely found in a monophyletic group of organisms, provide reliable molecular characteristics that are useful for evolutionary studies and for the demarcation of different groups of organisms in molecular terms [<a href="#B37-genes-10-00739">37</a>,<a href="#B46-genes-10-00739">46</a>,<a href="#B47-genes-10-00739">47</a>,<a href="#B48-genes-10-00739">48</a>,<a href="#B49-genes-10-00739">49</a>,<a href="#B50-genes-10-00739">50</a>,<a href="#B94-genes-10-00739">94</a>]. The other indels in protein sequences not meeting these criteria, although they provide valuable tools for genetic and biochemical studies [<a href="#B43-genes-10-00739">43</a>,<a href="#B92-genes-10-00739">92</a>], their utility for evolutionary studies is limited. </p> <p>A summary diagram showing the nematode groups’ specificities of different identified CSIs is presented in <a href="#genes-10-00739-f007">Figure 7</a>. Of the 52 CSIs identified in this work, 39 CSIs in different proteins are uniquely shared by all members of the genus <i>Caenorhabditis</i>. Four CSIs are specific for the family <i>Rhabditoidea</i>, which, in addition to the genus <i>Caenorhabditis,</i> also includes the genome-sequenced species <i>Diploscapter pachys</i> whereas eight CSIs in unrelated proteins are distinguishing characteristics of the different species from the class <i>Chromadorea</i>. In addition, we have identified one CSI in an Na(+)/H(+) exchange regulatory factor, NRFL-1, that appears to be a common and unique characteristic of different species from the phylum Nematoda. Some molecular features specific for the phylum Nematoda have also been reported by Yin et al. [<a href="#B95-genes-10-00739">95</a>]. However, our analysis did not identify any CSI that was specific for the <i>Strongyloidea</i> or <i>Trichostrongyloidea</i> superfamily, which also did not form well-resolved clades in our phylogenetic tree. Thus, the species distribution of the identified CSIs independently supports the different observed groupings of <i>Chromadorea</i> species in the phylogenetic tree. The specificities of the identified CSIs for different members of the indicated clades indicates that the genetic changes responsible for these CSIs initially occurred in the common ancestors of these groups and these genetic changes were then retained/inherited by various descendent species [<a href="#B47-genes-10-00739">47</a>]. </p> <p>The identified CSIs, due to their exclusive presence in the indicated groups of nematodes, provide novel and useful means for the identification of both known as well as novel species from these groups in molecular terms and for genetic, biochemical and evolutionary relationships. Extensive earlier work on CSIs for other groups of organisms strongly indicates that these molecular characteristics exhibit a high degree of constancy and predictive ability to be found in other members of the indicated groups [<a href="#B37-genes-10-00739">37</a>,<a href="#B45-genes-10-00739">45</a>,<a href="#B47-genes-10-00739">47</a>,<a href="#B96-genes-10-00739">96</a>]. It is expected that of the 39 CSIs identified in the present work which are specific for <i>Caenorhabditis</i> species, a large number of them should also be found in other non-genome sequenced or novel <i>Caenorhabditis</i> species. All of the described CSIs are present within conserved regions of the genes/proteins. Thus, based on the conserved regions encompassing these CSIs, the presence/absence of these CSIs in other nematodes/<i>Caenorhabditis</i> species could be readily examined by means of different commonly used experimental techniques viz. PCR-based, q-PCR-based, as well as by in silico BLAST searches examining the presence of these CSIs in genomic sequence data. The CSIs-based approaches have been used previously for developing novel and highly specific diagnostic tests for a number of important bacterial pathogens [<a href="#B97-genes-10-00739">97</a>,<a href="#B98-genes-10-00739">98</a>].</p> <p>The CSIs identified in this work are present in diverse proteins (see <a href="#genes-10-00739-t001">Table 1</a> and <a href="#genes-10-00739-t002">Table 2</a>) that are involved in important/essential functions in <i>C. elegans</i> that are likely to be conserved in other nematodes as well. Although the cellular functions of these CSIs are currently not known, earlier work on CSIs in other organisms has shown that these conserved molecular characteristics play important and often essential functions in the organisms where they are found [<a href="#B84-genes-10-00739">84</a>,<a href="#B99-genes-10-00739">99</a>]. Most of the studied CSIs in protein sequences are located in the surface loops of proteins, which are known to play important roles in mediating novel protein–protein or protein–ligand interactions that are essential or important for the CSI-containing organisms [<a href="#B84-genes-10-00739">84</a>,<a href="#B87-genes-10-00739">87</a>,<a href="#B99-genes-10-00739">99</a>]. In the present work, using homology modeling technique and structural overlaps of the CSIs-containing and CSIs-lacking proteins, we have mapped the locations of the CSIs in three proteins viz. Rab-44 (4R79.2), poly ADP-ribose glycohydrolase (PARG-1) and tRNA (guanine-N(1)-)-methyltransferase (F46F11.10) that contain CSIs specific for the <i>Caenorhabditis</i> or <i>Chormadorea</i> species (<a href="#genes-10-00739-f002">Figure 2</a> and <a href="#genes-10-00739-f004">Figure 4</a>). In all three cases, the CSIs in these proteins in the modeled structures from <i>C. elegans</i> were localized in the surface-exposed regions of the proteins (<a href="#genes-10-00739-f006">Figure 6</a>). </p> <p>As noted in the introduction, <i>C. elegans</i> is an important model organism for studying developmental process, for aging research, and for examining the cellular functions of different genes/proteins in eukaryotic organisms [<a href="#B2-genes-10-00739">2</a>,<a href="#B3-genes-10-00739">3</a>,<a href="#B22-genes-10-00739">22</a>,<a href="#B26-genes-10-00739">26</a>]. Further, as many genes in <i>C. elegans</i> (<i>Caenorhabditis</i> species) are homologous to human proteins, it has also used as a model for studying the role of homologous genes/proteins involved in human diseases [<a href="#B14-genes-10-00739">14</a>,<a href="#B15-genes-10-00739">15</a>,<a href="#B17-genes-10-00739">17</a>,<a href="#B23-genes-10-00739">23</a>,<a href="#B24-genes-10-00739">24</a>,<a href="#B100-genes-10-00739">100</a>]. One important advantage of <i>C. elegans</i> is that, in addition to its ease of growth, transparency, and well-studied developmental pathways, it can also be readily manipulated genetically. Thus, it should be possible to investigate in this system the functional significance of the CSIs that are specific for nematode groups [<a href="#B18-genes-10-00739">18</a>,<a href="#B19-genes-10-00739">19</a>,<a href="#B23-genes-10-00739">23</a>,<a href="#B89-genes-10-00739">89</a>,<a href="#B100-genes-10-00739">100</a>,<a href="#B101-genes-10-00739">101</a>,<a href="#B102-genes-10-00739">102</a>,<a href="#B103-genes-10-00739">103</a>]. Earlier work on CSIs has shown that these genetic characteristics are functionally important and often play essential roles in the organisms for which they are specific [<a href="#B84-genes-10-00739">84</a>,<a href="#B94-genes-10-00739">94</a>,<a href="#B99-genes-10-00739">99</a>]. Additionally, the conserved indels in protein sequences also provide potential drug targets [<a href="#B92-genes-10-00739">92</a>,<a href="#B104-genes-10-00739">104</a>]. In view of these considerations, further studies on understanding the functional significance of the CSIs which are specific for the <i>Caenorhabditis</i>/nematodes species should be of much interest and these could lead to the discovery of novel functional aspects of these important organisms.</p> </div> </div> <div class="back" title="back"> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>The following are available online at <span class="uri" title="uri"><a href="https://www.mdpi.com/2073-4425/10/10/739/s1">https://www.mdpi.com/2073-4425/10/10/739/s1</a></span>, Table S1: Name and accession numbers of proteins used for phylogenetic analysis, Figure S1: Maximum-likelihood trees for genome-sequenced nematode species based on concatenated sequences of 17 conserved proteins. The trees were constructed in MEGA6 using (A) Whelan and Goldman + Freq. and (B) JTT matrix-based models, Figure S2: Partial sequence alignments of (A) Rab-44 protein containing a 1 aa CSI and (B) poly ADP-ribose glycohydrolase protein containing a 5 aa insertion, which re specific for the genus <i>Caenorhabditis</i>, Figure S3: Partial sequence alignments of poly (ADP-ribose) polymerase 2 protein with 2 aa deletion which is specific for the <i>Caenorhabditis</i> genus, Figure S4: Partial sequence alignments of DnaJ-domain containing chaperone protein consisting of a 1 aa deletion which is specific for the <i>Caenorhabditis</i> genus, Figure S5: Partial sequence alignments of Cyclin dependent kinase 12 protein consisting of a 1 aa deletion that is specific for the <i>Caenorhabditis</i> genus, Figure S6: Partial sequence alignments of CRAL-TRIO domaincontaining Sec14 protein consisting of a 2 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S7: Partial sequence alignments of mammalian ZAK kinase homolog protein with 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S8: Partial sequence alignments of probable 3′,5′-cyclic phosphodiesterase pde-2 protein with a 2 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S9: Partial sequence alignments of nuclear hormone receptor protein with 1 aa deletion which is specific for the <i>Caenorhabditis</i> genus, Figure S10: Partial sequence alignments of SMAII-like (spore membrane assembly protein 2-like) protein with 2 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S11: Partial sequence alignments of glutathione transferase omega-1 protein with a 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S12: Partial sequence alignments of 26S proteasome regulatory protein subunit with a CSI consisting of a 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S13: Partial sequence alignments of serine/threonine protein phosphatase 2A regulatory protein subunit with a 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S14: Partial sequence alignments of failed axon connections like protein with a 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S15: Partial sequence alignments of NADH dehydrogenase 1 alpha sub-complex assembly factor 2 protein with a 13 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S16: Partial sequence alignments of disorganized muscle protein with a 1 aa deletion that is specific for the <i>Caenorhabditis</i> genus, Figure S17: Partial sequence alignments of ETS (E26 transformation-specific) class transcription factor protein with a 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S18: Partial sequence alignments of glycine-rich domain containing protein with a 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S19: Partial sequence alignments of a feat shock protein 70 protein consisting of a 2 aa deletion that is specific for the <i>Caenorhabditis</i> genus, Figure S20: Partial sequence alignments of a heat shock protein 70 protein consisting of a 1 aa deletion which is specific for the <i>Caenorhabditis</i> genus, Figure S21: Partial sequence alignments of abnormal cell migration protein 13 protein with a 1 aa deletion which is specific for the <i>Caenorhabditis</i> genus, Figure S22: Partial sequence alignments of regulatory associated protein of mTOR-like protein consisting of a 1 aa CSI that is specific for the <i>Caenorhabditis</i> genus, Figure S23: Partial sequence alignments of abnormal cell migration protein 13 protein consisting of a 3 aa deletion which is specific for the <i>Caenorhabditis</i> genus, Figure S24: Partial sequence alignments of abnormal cell migration protein 13 protein consisting of a 1 aa deletion which is specific for the <i>Caenorhabditis</i> genus, Figure S25: Partial sequence alignments of Plexin protein with a 1 aa CSI that is specific for the <i>Caenorhabditis</i> genus, Figure S26: Partial sequence alignments of a Piwi-like protein protein consisting of a 1 aa CSI that is specific for the <i>Caenorhabditis</i> genus, Figure S27: Partial sequence alignments of stomatin protein with a CSI consisting of a 1 aa deletion that is specific for the <i>Caenorhabditis</i> genus, Figure S28: Partial sequence alignments of Ral guanine nucleotide dissociation stimulator protein consisting of a 1 aa deletion that is specific for the <i>Caenorhabditis</i> genus, Figure S29: Partial sequence alignments of transglutaminase/protease homolog protein consisting of a 1 aa deletion that is specific for the <i>Caenorhabditis</i> genus, Figure S30: Partial sequence alignments of vacuolar protein sorting associated protein 41 homolog protein consisting of a 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S31: Partial sequence alignments of serine/arginine-rich splicing factor protein consisting of a 1 aa deletion that is specific for the <i>Caenorhabditis</i> genus, Figure S32: Partial sequence alignments of serine/threonineprotein phosphatase protein consisting of a 1 aa CSI that is specific for the <i>Caenorhabditis</i> genus, Figure S33: Partial sequence alignments of NEPrilysin metallopeptidase family protein consisting of a 1 aa deletion which is specific for the <i>Caenorhabditis</i> genus, Figure S34: Partial sequence alignments of DNA PRImase homolog protein consisting of a 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S35: Partial sequence alignments of a probable maleylacetoacetate isomerase protein with a 3 aa deletion that is specific for the <i>Caenorhabditis</i> genus, Figure S36: Partial sequence alignments of glutathione S-transferase protein consisting of a 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S37: Partial sequence alignments of CTD nuclear envelope phosphatase 1 homolog protein with a 1 aa CSI which is specific for the <i>Caenorhabditis</i> genus, Figure S38: Partial sequence alignments of Kelch-domain protein with a 6 aa CSI that is specific for the <i>Caenorhabditis</i> genus, Figure S39: Partial sequence alignments of intermediate filament protein consisting of a 2 aa deletion that is specific for the <i>Caenorhabditis</i> genus, Figure S40: Partial sequence alignments of cleavage factor IM (CFIm) homolog protein consisting of a 2 aa CSI which is specific for the <i>Rhabditoidea</i> suborder, Figure S41: Partial sequence alignments of Methyl-CpG-binding protein consisting of a 2 aa CSI which is specific for the <i>Rhabditoidea</i> suborder, Figure S42: Partial sequence alignments of abnormal cell migration protein 13 consisting of a 1 aa CSI which is specific for the <i>Rhabditoidea</i> suborder, Figure S43: Partial sequence alignments of PAX3- and PAX7 binding protein 1 protein with a 1 aa deletion that is specific for the <i>Rhabditoidea</i> suborder, Figure S44: Partial sequence alignments of palmitoyltransferase protein consisting of a 1 aa deletion that is specific for the <i>Chromadorea</i> class, Figure S45: Partial sequence alignments of palmitoyltransferase protein with a 2 aa deletion that is specific for the <i>Chromadorea</i> class, Figure S46: Partial sequence alignments of a battenin protein with a 3 aa CSI that is specific for the <i>Chromadorea</i> class, Figure S47: Partial sequence alignments of ETS (E26 transformation-specific) class transcription factor protein consisting of a 1 aa CSI that is specific for the <i>Chromadorea</i> class, Figure S48: Partial sequence alignments of heterogeneous nuclear ribonucleoprotein A1 protein with a 1 aa CSI that is specific for the <i>Chromadorea</i> class, Figure S49: Partial sequence alignments of heterogeneous nuclear ribonucleoprotein A1 protein consisting of a 5 aa deletion which is specific for the <i>Chromadorea</i> class, Figure S50: Partial sequence alignments of regulator of Gprotein signaling 7 protein consisting of a 1 aa CSI which is specific for the <i>Chromadorea</i> class, Figure S51: Partial sequence alignments of Na(+)/H(+) exchange regulatory factor protein with a 1 aa CSI which is specific for the entire Nematoda phylum.</p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="genes-10-00739-s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>B.K. and T.C. carried out identification of CSIs, phylogenetic analysis and homology modelling and localization of the CSIs in protein structures, and preparation of a draft manuscript. R.S.G., planning and supervision of the work, creation of sequence alignments of proteins for identification of the CSIs, obtained funding for the project and writing and finalizing of the manuscript. B.P.G., contributed to the editing of the manuscript and provided information regarding the functional significance of the proteins containing the CSIs. </p> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was supported by Research Grant number 249924 from the Natural Science and Engineering Research Council of Canada awarded to Radhey S. Gupta.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-genes-10-00739" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dorris</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Ley</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blaxter</span><span class="given-names" tagx="given-names" title="given-names">M.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular analysis of nematode diversity and the evolution of parasitism</span><span class="source" tagx="source" title="source">Parasitol. Today</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">188</span><span class="lpage" tagx="lpage" title="lpage">193</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0169-4758(99)01439-8">10.1016/S0169-4758(99)01439-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10322352">10322352</a></span></span></li> <li tag="ref"><a name="B2-genes-10-00739" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holterman</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Wurff</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van den Elsen</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Megen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bongers</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holovachov</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helder</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylum-wide analysis of SSU rDNA reveals deep phylogenetic relationships among nematodes and accelerated evolution toward crown Clades</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">1792</span><span class="lpage" tagx="lpage" title="lpage">1800</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msl044">10.1093/molbev/msl044</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16790472">16790472</a></span></span></li> <li tag="ref"><a name="B3-genes-10-00739" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blaxter</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Nematodes</i>: The worm and its relatives</span><span class="source" tagx="source" title="source">PLoS Biol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1001050</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/annotation/083d39ea-2269-4915-9297-bc6d9a9f7c58">10.1371/annotation/083d39ea-2269-4915-9297-bc6d9a9f7c58</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21526226">21526226</a></span></span></li> <li tag="ref"><a name="B4-genes-10-00739" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">G.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">D.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">X.Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The complete mitochondrial genomes of three parasitic nematodes of birds: A unique gene order and insights into nematode phylogeny</span><span class="source" tagx="source" title="source">BMC. Genom.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">14</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">414</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-14-414">10.1186/1471-2164-14-414</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23800363">23800363</a></span></span></li> <li tag="ref"><a name="B5-genes-10-00739" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meldal</span><span class="given-names" tagx="given-names" title="given-names">B.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debenham</span><span class="given-names" tagx="given-names" title="given-names">N.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Ley</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Ley</span><span class="given-names" tagx="given-names" title="given-names">I.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vanfleteren</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vierstraete</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bert</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borgonie</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moens</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tyler</span><span class="given-names" tagx="given-names" title="given-names">P.A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">An improved molecular phylogeny of the <i>Nematoda</i> with special emphasis on marine taxa</span><span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">622</span><span class="lpage" tagx="lpage" title="lpage">636</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2006.08.025">10.1016/j.ympev.2006.08.025</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17084644">17084644</a></span></span></li> <li tag="ref"><a name="B6-genes-10-00739" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van den Elsen</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holovachov</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karssen</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Megen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helder</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bongers</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mooyman</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A phylogenetic tree of nematodes based on about 1200 full-length small subunit ribosomal DNA sequences</span><span class="source" tagx="source" title="source">Nematology</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">927</span><span class="lpage" tagx="lpage" title="lpage">950</span><span class="pub-id"><a href="https://dx.doi.org/10.1163/156854109X456862">10.1163/156854109X456862</a></span></span></li> <li tag="ref"><a name="B7-genes-10-00739" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adamson</span><span class="given-names" tagx="given-names" title="given-names">M.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis of the higher classification of the <i>Nematoda</i></span><span class="source" tagx="source" title="source">Can. J. Zool.</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">1478</span><span class="lpage" tagx="lpage" title="lpage">1482</span><span class="pub-id"><a href="https://dx.doi.org/10.1139/z87-230">10.1139/z87-230</a></span></span></li> <li tag="ref"><a name="B8-genes-10-00739" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeates</span><span class="given-names" tagx="given-names" title="given-names">G.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bongers</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Goede</span><span class="given-names" tagx="given-names" title="given-names">R.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freckman</span><span class="given-names" tagx="given-names" title="given-names">D.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Georgieva</span><span class="given-names" tagx="given-names" title="given-names">S.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Feeding habits in soil <i>Nematode</i> families and genera-an outline for soil ecologists</span><span class="source" tagx="source" title="source">J. Nematol.</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">315</span><span class="lpage" tagx="lpage" title="lpage">331</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19279775">19279775</a></span></span></li> <li tag="ref"><a name="B9-genes-10-00739" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kikuchi</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eves-van den Akker</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">J.T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genome Evolution of Plant-Parasitic <i>Nematodes</i></span><span class="source" tagx="source" title="source">Annu. Rev. Phytopathol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">333</span><span class="lpage" tagx="lpage" title="lpage">354</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-phyto-080516-035434">10.1146/annurev-phyto-080516-035434</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28590877">28590877</a></span></span></li> <li tag="ref"><a name="B10-genes-10-00739" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blaxter</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koutsovoulos</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The evolution of parasitism in <i>Nematoda</i></span><span class="source" tagx="source" title="source">Parasitology</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">S26</span><span class="lpage" tagx="lpage" title="lpage">S39</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0031182014000791">10.1017/S0031182014000791</a></span></span></li> <li tag="ref"><a name="B11-genes-10-00739" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blaxter</span><span class="given-names" tagx="given-names" title="given-names">M.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nematoda: Genes, genomes and the evolution of parasitism</span><span class="source" tagx="source" title="source">Adv. Parasitol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">101</span><span class="lpage" tagx="lpage" title="lpage">195</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14711085">14711085</a></span></span></li> <li tag="ref"><a name="B12-genes-10-00739" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pozio</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darwin</span><span class="given-names" tagx="given-names" title="given-names">M.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Systematics and epidemiology of <i>Trichinella</i></span><span class="source" tagx="source" title="source">Adv. Parasitol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">367</span><span class="lpage" tagx="lpage" title="lpage">439</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17134656">17134656</a></span></span></li> <li tag="ref"><a name="B13-genes-10-00739" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rombout</span><span class="given-names" tagx="given-names" title="given-names">Y.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosch</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Der Giessen</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detection and identification of eight <i>Trichinella</i> genotypes by reverse line blot hybridization</span><span class="source" tagx="source" title="source">J. Clin. Microbiol.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">642</span><span class="lpage" tagx="lpage" title="lpage">646</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.39.2.642-646.2001">10.1128/JCM.39.2.642-646.2001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11158122">11158122</a></span></span></li> <li tag="ref"><a name="B14-genes-10-00739" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Litke</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boulanger</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fradin</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Caenorhabditis elegans</i> as a model organism for aging: Relevance, limitations and future</span><span class="source" tagx="source" title="source">Med. Sci.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">571</span><span class="lpage" tagx="lpage" title="lpage">579</span></span></li> <li tag="ref"><a name="B15-genes-10-00739" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ganner</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann-Haefelin</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic kidney diseases: <i>Caenorhabditis elegans</i> as model system</span><span class="source" tagx="source" title="source">Cell Tissue Res.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">369</span><span class="fpage" tagx="fpage" title="fpage">105</span><span class="lpage" tagx="lpage" title="lpage">118</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00441-017-2622-z">10.1007/s00441-017-2622-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28484847">28484847</a></span></span></li> <li tag="ref"><a name="B16-genes-10-00739" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nigon</span><span class="given-names" tagx="given-names" title="given-names">V.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felix</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">History of research on <i>C. elegans</i> and other free-living <i>Nematodes</i> as model organisms</span><span class="source" tagx="source" title="source">WormBook</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">2017</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">84</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28326696">28326696</a></span></span></li> <li tag="ref"><a name="B17-genes-10-00739" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kyriakakis</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markaki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tavernarakis</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Caenorhabditis elegans</i> as a model for cancer research</span><span class="source" tagx="source" title="source">Mol. Cell Oncol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">e975027</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/23723556.2014.975027">10.4161/23723556.2014.975027</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27308424">27308424</a></span></span></li> <li tag="ref"><a name="B18-genes-10-00739" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sommer</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bumbarger</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nematode model systems in evolution and development</span><span class="source" tagx="source" title="source">Wiley. Interdiscip. Rev. Dev. Biol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">389</span><span class="lpage" tagx="lpage" title="lpage">400</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/wdev.33">10.1002/wdev.33</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23801489">23801489</a></span></span></li> <li tag="ref"><a name="B19-genes-10-00739" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richman</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rashid</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prashar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mishra</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selvaganapathy</span><span class="given-names" tagx="given-names" title="given-names">P.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">B.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>C. elegans</i> MANF Homolog Is Necessary for the Protection of Dopaminergic Neurons and ER Unfolded Protein Response</span><span class="source" tagx="source" title="source">Front. Neurosci.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">544</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fnins.2018.00544">10.3389/fnins.2018.00544</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30147641">30147641</a></span></span></li> <li tag="ref"><a name="B20-genes-10-00739" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranawade</span><span class="given-names" tagx="given-names" title="given-names">A.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cumbo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">B.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Caenorhabditis elegans</i> histone deacetylase hda-1 is required for morphogenesis of the vulva and LIN-12/Notch-mediated specification of uterine cell fates</span><span class="source" tagx="source" title="source">G3 Genes Genomes Genet.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">1363</span><span class="lpage" tagx="lpage" title="lpage">1374</span></span></li> <li tag="ref"><a name="B21-genes-10-00739" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiontke</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gavin</span><span class="given-names" tagx="given-names" title="given-names">N.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raynes</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roehrig</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piano</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitch</span><span class="given-names" tagx="given-names" title="given-names">D.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Caenorhabditis</i> phylogeny predicts convergence of hermaphroditism and extensive intron loss</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">101</span><span class="fpage" tagx="fpage" title="fpage">9003</span><span class="lpage" tagx="lpage" title="lpage">9008</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0403094101">10.1073/pnas.0403094101</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15184656">15184656</a></span></span></li> <li tag="ref"><a name="B22-genes-10-00739" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nass</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merchant</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Caenohabditis elegans</i> in Parkinson’s disease drug discovery: Addressing an unmet medical need</span><span class="source" tagx="source" title="source">Mol. Interv.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">284</span><span class="lpage" tagx="lpage" title="lpage">293</span><span class="pub-id"><a href="https://dx.doi.org/10.1124/mi.8.6.6">10.1124/mi.8.6.6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19144901">19144901</a></span></span></li> <li tag="ref"><a name="B23-genes-10-00739" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elkabti</span><span class="given-names" tagx="given-names" title="given-names">A.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Issi</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Caenorhabditis elegans</i> as a Model Host to Monitor the <i>Candida</i> Infection Processes</span><span class="source" tagx="source" title="source">J. Fungi</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">123</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/jof4040123">10.3390/jof4040123</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30405043">30405043</a></span></span></li> <li tag="ref"><a name="B24-genes-10-00739" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinez</span><span class="given-names" tagx="given-names" title="given-names">B.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caldwell</span><span class="given-names" tagx="given-names" title="given-names">K.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caldwell</span><span class="given-names" tagx="given-names" title="given-names">G.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>C. elegans</i> as a model system to accelerate discovery for Parkinson disease</span><span class="source" tagx="source" title="source">Curr. Opin. Genet. Dev.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">102</span><span class="lpage" tagx="lpage" title="lpage">109</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gde.2017.02.011">10.1016/j.gde.2017.02.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28242493">28242493</a></span></span></li> <li tag="ref"><a name="B25-genes-10-00739" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lublin</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Link</span><span class="given-names" tagx="given-names" title="given-names">C.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Alzheimer's disease drug discovery: In vivo screening using <i>Caenorhabditis elegans</i> as a model for beta-amyloid peptide-induced toxicity</span><span class="source" tagx="source" title="source">Drug Discov. Today Technol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e115</span><span class="lpage" tagx="lpage" title="lpage">e119</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ddtec.2012.02.002">10.1016/j.ddtec.2012.02.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24050239">24050239</a></span></span></li> <li tag="ref"><a name="B26-genes-10-00739" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Culetto</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sattelle</span><span class="given-names" tagx="given-names" title="given-names">D.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A role for <i>Caenorhabditis elegans</i> in understanding the function and interactions of human disease genes</span><span class="source" tagx="source" title="source">Hum. Mol. Genet.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">869</span><span class="lpage" tagx="lpage" title="lpage">877</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/hmg/9.6.869">10.1093/hmg/9.6.869</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10767309">10767309</a></span></span></li> <li tag="ref"><a name="B27-genes-10-00739" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viney</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diaz</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phenotypic plasticity in nematodes: Evolutionary and ecological significance</span><span class="source" tagx="source" title="source">Worm</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">98</span><span class="lpage" tagx="lpage" title="lpage">106</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/worm.21086">10.4161/worm.21086</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24058831">24058831</a></span></span></li> <li tag="ref"><a name="B28-genes-10-00739" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callejon</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nadler</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Rojas</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zurita</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrasova</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cutillas</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular characterization and phylogeny of whipworm nematodes inferred from DNA sequences of cox1 mtDNA and 18S rDNA</span><span class="source" tagx="source" title="source">Parasitol. Res.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">112</span><span class="fpage" tagx="fpage" title="fpage">3933</span><span class="lpage" tagx="lpage" title="lpage">3949</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00436-013-3584-z">10.1007/s00436-013-3584-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24018707">24018707</a></span></span></li> <li tag="ref"><a name="B29-genes-10-00739" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aleshin</span><span class="given-names" tagx="given-names" title="given-names">V.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milyutina</span><span class="given-names" tagx="given-names" title="given-names">I.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kedrova</span><span class="given-names" tagx="given-names" title="given-names">O.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vladychenskaya</span><span class="given-names" tagx="given-names" title="given-names">N.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrov</span><span class="given-names" tagx="given-names" title="given-names">N.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogeny of <i>Nematoda</i> and <i>Cephalorhyncha</i> derived from 18S rDNA</span><span class="source" tagx="source" title="source">J. Mol. Evol</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">597</span><span class="lpage" tagx="lpage" title="lpage">605</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/PL00006416">10.1007/PL00006416</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9797410">9797410</a></span></span></li> <li tag="ref"><a name="B30-genes-10-00739" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunt</span><span class="given-names" tagx="given-names" title="given-names">V.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">I.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coghlan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reid</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holroyd</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foth</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tracey</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotton</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanley</span><span class="given-names" tagx="given-names" title="given-names">E.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beasley</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The genomic basis of parasitism in the <i>Strongyloides</i> clade of nematodes</span><span class="source" tagx="source" title="source">Nat. Genet.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">299</span><span class="lpage" tagx="lpage" title="lpage">307</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.3495">10.1038/ng.3495</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26829753">26829753</a></span></span></li> <li tag="ref"><a name="B31-genes-10-00739" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nadler</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hudspeth</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogeny of the <i>Ascaridoidea</i> (<i>Nematoda: Ascaridida</i>) based on three genes and morphology: Hypotheses of structural and sequence evolution</span><span class="source" tagx="source" title="source">J. Parasitol.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">380</span><span class="lpage" tagx="lpage" title="lpage">393</span><span class="pub-id"><a href="https://dx.doi.org/10.1645/0022-3395(2000)086[0380:POTANA]2.0.CO;2">10.1645/0022-3395(2000)086[0380:POTANA]2.0.CO;2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10780561">10780561</a></span></span></li> <li tag="ref"><a name="B32-genes-10-00739" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bik</span><span class="given-names" tagx="given-names" title="given-names">H.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambshead</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">W.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lunt</span><span class="given-names" tagx="given-names" title="given-names">D.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Moving towards a complete molecular framework of the <i>Nematoda</i>: A focus on the <i>Enoplida</i> and early-branching clades</span><span class="source" tagx="source" title="source">BMC. Evol. Biol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">353</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-10-353">10.1186/1471-2148-10-353</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21073704">21073704</a></span></span></li> <li tag="ref"><a name="B33-genes-10-00739" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiontke</span><span class="given-names" tagx="given-names" title="given-names">K.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felix</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ailion</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rockman</span><span class="given-names" tagx="given-names" title="given-names">M.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braendle</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Penigault</span><span class="given-names" tagx="given-names" title="given-names">J.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitch</span><span class="given-names" tagx="given-names" title="given-names">D.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A phylogeny and molecular barcodes for <i>Caenorhabditis</i>, with numerous new species from rotting fruits</span><span class="source" tagx="source" title="source">BMC Evol. Biol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">339</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-11-339">10.1186/1471-2148-11-339</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22103856">22103856</a></span></span></li> <li tag="ref"><a name="B34-genes-10-00739" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palomares-Rius</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cantalapiedra-Navarrete</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rchidona-Yuste</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subbotin</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castillo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The utility of mtDNA and rDNA for barcoding and phylogeny of plant-parasitic <i>Nematodes</i> from <i>Longidoridae</i> (<i>Nematoda</i>, <i>Enoplea</i>)</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">10905</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-11085-4">10.1038/s41598-017-11085-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28883648">28883648</a></span></span></li> <li tag="ref"><a name="B35-genes-10-00739" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smythe</span><span class="given-names" tagx="given-names" title="given-names">A.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanderson</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nadler</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Nematode</i> small subunit phylogeny correlates with alignment parameters</span><span class="source" tagx="source" title="source">Syst Biol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">972</span><span class="lpage" tagx="lpage" title="lpage">992</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/10635150601089001">10.1080/10635150601089001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17345678">17345678</a></span></span></li> <li tag="ref"><a name="B36-genes-10-00739" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felsenstein</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="source" tagx="source" title="source">Inferring Phylogenies</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Sinauer Associates, Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Sunderland, MA, USA</span><span class="year" tagx="year" title="year">2004</span></span></li> <li tag="ref"><a name="B37-genes-10-00739" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Impact of genomics on the understanding of microbial evolution and classification: The importance of Darwin’s views on classification</span><span class="source" tagx="source" title="source">FEMS Microbiol. Rev.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">520</span><span class="lpage" tagx="lpage" title="lpage">553</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/femsre/fuw011">10.1093/femsre/fuw011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27279642">27279642</a></span></span></li> <li tag="ref"><a name="B38-genes-10-00739" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldauf</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogeny for the faint of heart: A tutorial</span><span class="source" tagx="source" title="source">Trends Genet.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">345</span><span class="lpage" tagx="lpage" title="lpage">351</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0168-9525(03)00112-4">10.1016/S0168-9525(03)00112-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12801728">12801728</a></span></span></li> <li tag="ref"><a name="B39-genes-10-00739" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rokas</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carroll</span><span class="given-names" tagx="given-names" title="given-names">S.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genome-scale approaches to resolving incongruence in molecular phylogenies</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">425</span><span class="fpage" tagx="fpage" title="fpage">798</span><span class="lpage" tagx="lpage" title="lpage">804</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature02053">10.1038/nature02053</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14574403">14574403</a></span></span></li> <li tag="ref"><a name="B40-genes-10-00739" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: NCBI</span></span><span class="mixed-article-title" title="mixed-article-title">NCBI Completed Microbial Genomes</span><span class="comment">Available online: <a href="http://www.ncbi.nlm.nih.gov/PMGifs/Genomes/micr.html">http://www.ncbi.nlm.nih.gov /PMGifs/Genomes/micr.html</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 15 April 2019)</span></span></li> <li tag="ref"><a name="B41-genes-10-00739" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smythe</span><span class="given-names" tagx="given-names" title="given-names">A.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holovachov</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kocot</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Improved phylogenomic sampling of free-living <i>Nematodes</i> enhances resolution of higher-level nematode phylogeny</span><span class="source" tagx="source" title="source">BMC Evol. Biol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">19</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">121</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12862-019-1444-x">10.1186/s12862-019-1444-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31195978">31195978</a></span></span></li> <li tag="ref"><a name="B42-genes-10-00739" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Caenorhabditis elegans</i> as a model system for target identification and drug screening against neurodegenerative diseases</span><span class="source" tagx="source" title="source">Eur. J. Pharmacol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">819</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="lpage" tagx="lpage" title="lpage">180</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ejphar.2017.11.051">10.1016/j.ejphar.2017.11.051</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29208474">29208474</a></span></span></li> <li tag="ref"><a name="B43-genes-10-00739" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abubucker</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasser</span><span class="given-names" tagx="given-names" title="given-names">R.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitreva</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Systematic analysis of insertions and deletions specific to <i>Nematode</i> proteins and their proposed functional and evolutionary relevance</span><span class="source" tagx="source" title="source">BMC Evol. Biol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">23</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-9-23">10.1186/1471-2148-9-23</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19175938">19175938</a></span></span></li> <li tag="ref"><a name="B44-genes-10-00739" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coghlan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Nematode</i> genome evolution</span><span class="source" tagx="source" title="source">WormBook</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">2005</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">15</span><span class="pub-id"><a href="https://dx.doi.org/10.1895/wormbook.1.15.1">10.1895/wormbook.1.15.1</a></span></span></li> <li tag="ref"><a name="B45-genes-10-00739" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular signatures that are distinctive characteristics of the vertebrates and chordates and supporting a grouping of vertebrates with the tunicates</span><span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">383</span><span class="lpage" tagx="lpage" title="lpage">391</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ympev.2015.09.019">10.1016/j.ympev.2015.09.019</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26419477">26419477</a></span></span></li> <li tag="ref"><a name="B46-genes-10-00739" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Springer</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanhope</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madsen</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">W.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecules consolidate the placental mammal tree</span><span class="source" tagx="source" title="source">Trends Ecol. Evol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">430</span><span class="lpage" tagx="lpage" title="lpage">438</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tree.2004.05.006">10.1016/j.tree.2004.05.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16701301">16701301</a></span></span></li> <li tag="ref"><a name="B47-genes-10-00739" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of Conserved Indels that are Useful for Classification and Evolutionary Studies</span><span class="source" tagx="source" title="source">Methods Microbiol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">153</span><span class="lpage" tagx="lpage" title="lpage">182</span></span></li> <li tag="ref"><a name="B48-genes-10-00739" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rokas</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holland</span><span class="given-names" tagx="given-names" title="given-names">P.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rare genomic changes as a tool for phylogenetics</span><span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">454</span><span class="lpage" tagx="lpage" title="lpage">459</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0169-5347(00)01967-4">10.1016/S0169-5347(00)01967-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11050348">11050348</a></span></span></li> <li tag="ref"><a name="B49-genes-10-00739" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldauf</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palmer</span><span class="given-names" tagx="given-names" title="given-names">J.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Animals and fungi are each other’s closest relatives: Congruent evidence from multiple proteins</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">11558</span><span class="lpage" tagx="lpage" title="lpage">11562</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.90.24.11558">10.1073/pnas.90.24.11558</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8265589">8265589</a></span></span></li> <li tag="ref"><a name="B50-genes-10-00739" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Protein phylogenies and signature sequences: A reappraisal of evolutionary relationships among <i>Archaebacteria</i>, <i>Eubacteria</i>, and <i>Eukaryotes</i></span><span class="source" tagx="source" title="source">Microbiol. Mol. Biol. Rev.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">62</span><span class="fpage" tagx="fpage" title="fpage">1435</span><span class="lpage" tagx="lpage" title="lpage">1491</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9841678">9841678</a></span></span></li> <li tag="ref"><a name="B51-genes-10-00739" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Novel molecular synapomorphies demarcate different main groups/subgroups of <i>Plasmodium</i> and <i>Piroplasmida</i> species clarifying their evolutionary relationships</span><span class="source" tagx="source" title="source">Genes</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">490</span><span class="comment">(in press)</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/genes10070490">10.3390/genes10070490</a></span></span></li> <li tag="ref"><a name="B52-genes-10-00739" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adeolu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alnajar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naushad</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genome-based phylogeny and taxonomy of the ‘<i>Enterobacteriales</i>’: Proposal for Enterobacterales ord. nov. divided into the families <i>Enterobacteriaceae</i>, <i>Erwiniaceae</i> fam. nov., <i>Pectobacteriaceae</i> fam. nov., <i>Yersiniaceae</i> fam. nov., <i>Hafniaceae</i> fam. nov., <i>Morganellaceae</i> fam. nov., and <i>Budviciaceae</i> fam. nov</span><span class="source" tagx="source" title="source">Int. J. Syst. Evol. Microbiol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">5575</span><span class="lpage" tagx="lpage" title="lpage">5599</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27620848">27620848</a></span></span></li> <li tag="ref"><a name="B53-genes-10-00739" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">CD-HIT: Accelerated for clustering the next-generation sequencing data</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">3150</span><span class="lpage" tagx="lpage" title="lpage">3152</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bts565">10.1093/bioinformatics/bts565</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23060610">23060610</a></span></span></li> <li tag="ref"><a name="B54-genes-10-00739" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sievers</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilm</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dineen</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">T.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karplus</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McWilliam</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Remmert</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soding</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span><span class="source" tagx="source" title="source">Mol. Syst. Biol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">539</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/msb.2011.75">10.1038/msb.2011.75</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21988835">21988835</a></span></span></li> <li tag="ref"><a name="B55-genes-10-00739" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Capella-Gutierrez</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silla-Martinez</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gabaldon</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">TrimAl: A tool for automated alignment trimming in large-scale phylogenetic analyses</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1972</span><span class="lpage" tagx="lpage" title="lpage">1973</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp348">10.1093/bioinformatics/btp348</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505945">19505945</a></span></span></li> <li tag="ref"><a name="B56-genes-10-00739" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talavera</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castresana</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments</span><span class="source" tagx="source" title="source">Syst. Biol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">564</span><span class="lpage" tagx="lpage" title="lpage">577</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/10635150701472164">10.1080/10635150701472164</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17654362">17654362</a></span></span></li> <li tag="ref"><a name="B57-genes-10-00739" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">M.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehal</span><span class="given-names" tagx="given-names" title="given-names">P.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arkin</span><span class="given-names" tagx="given-names" title="given-names">A.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">FastTree 2—Approximately maximum-likelihood trees for large alignments</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e9490</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0009490">10.1371/journal.pone.0009490</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20224823">20224823</a></span></span></li> <li tag="ref"><a name="B58-genes-10-00739" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whelan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldman</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">691</span><span class="lpage" tagx="lpage" title="lpage">699</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a003851">10.1093/oxfordjournals.molbev.a003851</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11319253">11319253</a></span></span></li> <li tag="ref"><a name="B59-genes-10-00739" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">RAxML version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">1312</span><span class="lpage" tagx="lpage" title="lpage">1313</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu033">10.1093/bioinformatics/btu033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623">24451623</a></span></span></li> <li tag="ref"><a name="B60-genes-10-00739" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">S.Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gascuel</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">An improved general amino acid replacement matrix</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1307</span><span class="lpage" tagx="lpage" title="lpage">1320</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msn067">10.1093/molbev/msn067</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18367465">18367465</a></span></span></li> <li tag="ref"><a name="B61-genes-10-00739" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guindon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dufayard</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lefort</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anisimova</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hordijk</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gascuel</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: Assessing the performance of PhyML 3.0</span><span class="source" tagx="source" title="source">Syst. Biol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">307</span><span class="lpage" tagx="lpage" title="lpage">321</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syq010">10.1093/sysbio/syq010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638">20525638</a></span></span></li> <li tag="ref"><a name="B62-genes-10-00739" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Filipski</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MEGA6: Molecular Evolutionary Genetics Analysis version 6.0</span><span class="source" tagx="source" title="source">Mol. Biol. Evol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">2725</span><span class="lpage" tagx="lpage" title="lpage">2729</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst197">10.1093/molbev/mst197</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122">24132122</a></span></span></li> <li tag="ref"><a name="B63-genes-10-00739" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larkin</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blackshields</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">N.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chenna</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGettigan</span><span class="given-names" tagx="given-names" title="given-names">P.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McWilliam</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valentin</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallace</span><span class="given-names" tagx="given-names" title="given-names">I.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilm</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Clustal W and Clustal X version 2.0</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">2947</span><span class="lpage" tagx="lpage" title="lpage">2948</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btm404">10.1093/bioinformatics/btm404</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036">17846036</a></span></span></li> <li tag="ref"><a name="B64-genes-10-00739" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naushad</span><span class="given-names" tagx="given-names" title="given-names">H.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Conserved signature indels and signature proteins as novel tools for understanding microbial phylogeny and systematics: Identification of molecular signatures that are specific for the phytopathogenic genera <i>Dickeya</i>, <i>Pectobacterium</i> and <i>Brenneria</i></span><span class="source" tagx="source" title="source">Int. J. Syst. Evol. Microbiol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">366</span><span class="lpage" tagx="lpage" title="lpage">383</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/ijs.0.054213-0">10.1099/ijs.0.054213-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24505075">24505075</a></span></span></li> <li tag="ref"><a name="B65-genes-10-00739" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waszkowycz</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGonagle</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jordan</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Acton</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fairweather</span><span class="given-names" tagx="given-names" title="given-names">E.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griffiths</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamilton</span><span class="given-names" tagx="given-names" title="given-names">N.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamilton</span><span class="given-names" tagx="given-names" title="given-names">N.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hitchin</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cell-Active Small Molecule Inhibitors of the DNA-Damage Repair Enzyme Poly(ADP-ribose) <i>Glycohydrolase</i> (PARG): Discovery and Optimization of Orally Bioavailable Quinazolinedione Sulfonamides</span><span class="source" tagx="source" title="source">J. Med. Chem.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">10767</span><span class="lpage" tagx="lpage" title="lpage">10792</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jmedchem.8b01407">10.1021/acs.jmedchem.8b01407</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30403352">30403352</a></span></span></li> <li tag="ref"><a name="B66-genes-10-00739" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zuo</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Crystal structure of tRNA m1G9 methyltransferase Trm10: Insight into the catalytic mechanism and recognition of tRNA substrate</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">509</span><span class="lpage" tagx="lpage" title="lpage">525</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkt869">10.1093/nar/gkt869</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24081582">24081582</a></span></span></li> <li tag="ref"><a name="B67-genes-10-00739" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eswar</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webb</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marti-Renom</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madhushudan</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eramian</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">M.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pieper</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sali</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparative protein structure modelling using Modeller</span><span class="source" tagx="source" title="source">Curr. Protoc. Bioinform.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">5</span><span class="lpage" tagx="lpage" title="lpage">6</span></span></li> <li tag="ref"><a name="B68-genes-10-00739" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">M.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sali</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Statistical potential for assessment and prediction of protein structures</span><span class="source" tagx="source" title="source">Protein Sci.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">2507</span><span class="lpage" tagx="lpage" title="lpage">2524</span><span class="pub-id"><a href="https://dx.doi.org/10.1110/ps.062416606">10.1110/ps.062416606</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17075131">17075131</a></span></span></li> <li tag="ref"><a name="B69-genes-10-00739" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowie</span><span class="given-names" tagx="given-names" title="given-names">J.U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luthy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisenberg</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A method to identify protein sequences that fold into a known three-dimensional structure</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">1991</span><span class="volume" tagx="volume" title="volume">253</span><span class="fpage" tagx="fpage" title="fpage">164</span><span class="lpage" tagx="lpage" title="lpage">170</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1853201">10.1126/science.1853201</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1853201">1853201</a></span></span></li> <li tag="ref"><a name="B70-genes-10-00739" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colovos</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeates</span><span class="given-names" tagx="given-names" title="given-names">T.O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Verification of protein structures: Patterns of nonbonded atomic interactions</span><span class="source" tagx="source" title="source">Protein Sci.</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">1511</span><span class="lpage" tagx="lpage" title="lpage">1519</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/pro.5560020916">10.1002/pro.5560020916</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8401235">8401235</a></span></span></li> <li tag="ref"><a name="B71-genes-10-00739" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lovell</span><span class="given-names" tagx="given-names" title="given-names">S.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">I.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arendall</span><span class="given-names" tagx="given-names" title="given-names">W.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Bakker</span><span class="given-names" tagx="given-names" title="given-names">P.I.</span><span class="suffix" tagx="suffix" title="suffix">III</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Word</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prisant</span><span class="given-names" tagx="given-names" title="given-names">M.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">D.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure validation by Calpha geometry: Phi, psi and Cbeta deviation</span><span class="source" tagx="source" title="source">Proteins</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">437</span><span class="lpage" tagx="lpage" title="lpage">450</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.10286">10.1002/prot.10286</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12557186">12557186</a></span></span></li> <li tag="ref"><a name="B72-genes-10-00739" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luthy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowie</span><span class="given-names" tagx="given-names" title="given-names">J.U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisenberg</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Assessment of protein models with three-dimensional profiles</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">356</span><span class="fpage" tagx="fpage" title="fpage">83</span><span class="lpage" tagx="lpage" title="lpage">85</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/356083a0">10.1038/356083a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1538787">1538787</a></span></span></li> <li tag="ref"><a name="B73-genes-10-00739" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">G.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heo</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seok</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effective protein model structure refinement by loop modeling and overall relaxation</span><span class="source" tagx="source" title="source">Proteins</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">84</span><span class="fpage" tagx="fpage" title="fpage">293</span><span class="lpage" tagx="lpage" title="lpage">301</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.24858">10.1002/prot.24858</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26172288">26172288</a></span></span></li> <li tag="ref"><a name="B74-genes-10-00739" /><span class="label" tagx="label" title="label">74.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Segev</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ypt/rab gtpases: Regulators of protein trafficking</span><span class="source" tagx="source" title="source">Sci. STKE</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">2001</span><span class="fpage" tagx="fpage" title="fpage">re11</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/stke.2001.100.re11">10.1126/stke.2001.100.re11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11579231">11579231</a></span></span></li> <li tag="ref"><a name="B75-genes-10-00739" /><span class="label" tagx="label" title="label">75.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marlin</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rab family of GTPases</span><span class="source" tagx="source" title="source">Methods Mol. Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">1298</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">15</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25800828">25800828</a></span></span></li> <li tag="ref"><a name="B76-genes-10-00739" /><span class="label" tagx="label" title="label">76.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipatova</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hain</span><span class="given-names" tagx="given-names" title="given-names">A.U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nazarko</span><span class="given-names" tagx="given-names" title="given-names">V.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Segev</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ypt/Rab GTPases: Principles learned from yeast</span><span class="source" tagx="source" title="source">Crit Rev. Biochem Mol. Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">203</span><span class="lpage" tagx="lpage" title="lpage">211</span><span class="pub-id"><a href="https://dx.doi.org/10.3109/10409238.2015.1014023">10.3109/10409238.2015.1014023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25702751">25702751</a></span></span></li> <li tag="ref"><a name="B77-genes-10-00739" /><span class="label" tagx="label" title="label">77.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">St Laurent</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gagnon</span><span class="given-names" tagx="given-names" title="given-names">S.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dequen</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hardy</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desnoyers</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Altered DNA damage response in <i>Caenorhabditis elegans</i> with impaired poly (ADP-ribose) glycohydrolases genes expression</span><span class="source" tagx="source" title="source">DNA Repair</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">329</span><span class="lpage" tagx="lpage" title="lpage">343</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.dnarep.2006.10.027">10.1016/j.dnarep.2006.10.027</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17188026">17188026</a></span></span></li> <li tag="ref"><a name="B78-genes-10-00739" /><span class="label" tagx="label" title="label">78.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ou</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transmembrane protein MIG-13 links the Wnt signaling and Hox genes to the cell polarity in neuronal migration</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">110</span><span class="fpage" tagx="fpage" title="fpage">11175</span><span class="lpage" tagx="lpage" title="lpage">11180</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1301849110">10.1073/pnas.1301849110</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23784779">23784779</a></span></span></li> <li tag="ref"><a name="B79-genes-10-00739" /><span class="label" tagx="label" title="label">79.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masuda</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakamura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takata</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Itoh</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirose</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moribe</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mekada</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okada</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MIG-13 controls anteroposterior cell migration by interacting with UNC-71/ADM-1 and SRC-1 in <i>Caenorhabditis elegans</i></span><span class="source" tagx="source" title="source">FEBS Lett.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">586</span><span class="fpage" tagx="fpage" title="fpage">740</span><span class="lpage" tagx="lpage" title="lpage">746</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.febslet.2012.01.031">10.1016/j.febslet.2012.01.031</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22293500">22293500</a></span></span></li> <li tag="ref"><a name="B80-genes-10-00739" /><span class="label" tagx="label" title="label">80.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sym</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenyon</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MIG-13 positions migrating cells along the anteroposterior body axis of <i>C. elegans</i></span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">98</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0092-8674(00)80603-0">10.1016/S0092-8674(00)80603-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10412978">10412978</a></span></span></li> <li tag="ref"><a name="B81-genes-10-00739" /><span class="label" tagx="label" title="label">81.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sultana</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eom</span><span class="given-names" tagx="given-names" title="given-names">K.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">Y.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soonthornpong</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nadler</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">J.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The mitochondrial genome sequence of Enterobius vermicularis (Nematoda: Oxyurida)—An idiosyncratic gene order and phylogenetic information for Chromadorean Nematodes</span><span class="source" tagx="source" title="source">Gene</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">429</span><span class="fpage" tagx="fpage" title="fpage">87</span><span class="lpage" tagx="lpage" title="lpage">97</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18848867">18848867</a></span></span></li> <li tag="ref"><a name="B82-genes-10-00739" /><span class="label" tagx="label" title="label">82.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brule</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elliott</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Redlak</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zehner</span><span class="given-names" tagx="given-names" title="given-names">Z.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">W.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of the human tRNA-(N1G37) methyltransferase (TRM5) and comparison to the <i>Escherichia coli</i> TrmD protein</span><span class="source" tagx="source" title="source">Biochemistry</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">9243</span><span class="lpage" tagx="lpage" title="lpage">9255</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/bi049671q">10.1021/bi049671q</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15248782">15248782</a></span></span></li> <li tag="ref"><a name="B83-genes-10-00739" /><span class="label" tagx="label" title="label">83.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hagiwara</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagamori</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Umemura</span><span class="given-names" tagx="given-names" title="given-names">Y.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohgaki</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murata</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakagomi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nomura</span><span class="given-names" tagx="given-names" title="given-names">K.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kage-Nakadai</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitani</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">NRFL-1, the C. elegans NHERF orthologue, interacts with amino acid transporter 6 (AAT-6) for age-dependent maintenance of AAT-6 on the membrane</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e43050</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0043050">10.1371/journal.pone.0043050</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22916205">22916205</a></span></span></li> <li tag="ref"><a name="B84-genes-10-00739" /><span class="label" tagx="label" title="label">84.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khadka</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of a conserved 8 aa insert in the PIP5K protein in the <i>Saccharomycetaceae</i> family of fungi and the molecular dynamics simulations and structural analysis to investigate its potential functional role</span><span class="source" tagx="source" title="source">Proteins</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">1454</span><span class="lpage" tagx="lpage" title="lpage">1467</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.25306">10.1002/prot.25306</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28407364">28407364</a></span></span></li> <li tag="ref"><a name="B85-genes-10-00739" /><span class="label" tagx="label" title="label">85.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alnajar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khadka</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ribonucleotide reductases from <i>Bifidobacteria</i> contain multiple conserved indels distinguishing them from all other organisms: In silico analysis of the possible role of a 43 aa <i>Bifidobacteria</i>-specific insert in the Class III RNR homolog</span><span class="source" tagx="source" title="source">Front. Microbiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1409</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2017.01409">10.3389/fmicb.2017.01409</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28824557">28824557</a></span></span></li> <li tag="ref"><a name="B86-genes-10-00739" /><span class="label" tagx="label" title="label">86.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanda</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khadka</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Novel molecular, structural and evolutionary characteristics of the <i>Phosphoketolases</i> from <i>Bifidobacteria</i> and <i>Coriobacteriales</i></span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0172176</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0172176">10.1371/journal.pone.0172176</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28212383">28212383</a></span></span></li> <li tag="ref"><a name="B87-genes-10-00739" /><span class="label" tagx="label" title="label">87.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akiva</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Itzhaki</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Margalit</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Built-in loops allow versatility in domain-domain interactions: Lessons from self-interacting domains</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">13292</span><span class="lpage" tagx="lpage" title="lpage">13297</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0801207105">10.1073/pnas.0801207105</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18757736">18757736</a></span></span></li> <li tag="ref"><a name="B88-genes-10-00739" /><span class="label" tagx="label" title="label">88.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hashimoto</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panchenko</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mechanisms of protein oligomerization, the critical role of insertions and deletions in maintaining different oligomeric states</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">107</span><span class="fpage" tagx="fpage" title="fpage">20352</span><span class="lpage" tagx="lpage" title="lpage">20357</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1012999107">10.1073/pnas.1012999107</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21048085">21048085</a></span></span></li> <li tag="ref"><a name="B89-genes-10-00739" /><span class="label" tagx="label" title="label">89.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tejeda-Benitez</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olivero-Verbel</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Caenorhabditis elegans</i>, a Biological Model for Research in Toxicology</span><span class="source" tagx="source" title="source">Rev. Environ. Contam Toxicol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">237</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">35</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26613986">26613986</a></span></span></li> <li tag="ref"><a name="B90-genes-10-00739" /><span class="label" tagx="label" title="label">90.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rujan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richly</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cornelsen</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lins</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leister</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoebe</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasegawa</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Penny</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary analysis of <i>Arabidopsis</i>, cyanobacterial, and chloroplast genomes reveals plastid phylogeny and thousands of cyanobacterial genes in the nucleus</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">12246</span><span class="lpage" tagx="lpage" title="lpage">12251</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.182432999">10.1073/pnas.182432999</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12218172">12218172</a></span></span></li> <li tag="ref"><a name="B91-genes-10-00739" /><span class="label" tagx="label" title="label">91.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciccarelli</span><span class="given-names" tagx="given-names" title="given-names">F.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doerks</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Mering</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Creevey</span><span class="given-names" tagx="given-names" title="given-names">C.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snel</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bork</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Toward automatic reconstruction of a highly resolved tree of life</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">311</span><span class="fpage" tagx="fpage" title="fpage">1283</span><span class="lpage" tagx="lpage" title="lpage">1287</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1123061">10.1126/science.1123061</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16513982">16513982</a></span></span></li> <li tag="ref"><a name="B92-genes-10-00739" /><span class="label" tagx="label" title="label">92.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heizer</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosa</span><span class="given-names" tagx="given-names" title="given-names">B.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wildman</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janetka</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitreva</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of parasite-specific indels and their proposed relevance for selective anthelminthic drug targeting</span><span class="source" tagx="source" title="source">Infect. Genet. Evol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">201</span><span class="lpage" tagx="lpage" title="lpage">211</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2016.01.025">10.1016/j.meegid.2016.01.025</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26829384">26829384</a></span></span></li> <li tag="ref"><a name="B93-genes-10-00739" /><span class="label" tagx="label" title="label">93.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitreva</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smant</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helder</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Role of horizontal gene transfer in the evolution of plant parasitism among <i>Nematodes</i></span><span class="source" tagx="source" title="source">Methods Mol. Biol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">532</span><span class="fpage" tagx="fpage" title="fpage">517</span><span class="lpage" tagx="lpage" title="lpage">535</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19271205">19271205</a></span></span></li> <li tag="ref"><a name="B94-genes-10-00739" /><span class="label" tagx="label" title="label">94.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khadka</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Novel Molecular Signatures in the PIP4K/PIP5K Family of Proteins Specific for Different Isozymes and Subfamilies Provide Important Insights into the Evolutionary Divergence of this Protein Family</span><span class="source" tagx="source" title="source">Genes</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">312</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/genes10040312">10.3390/genes10040312</a></span></span></li> <li tag="ref"><a name="B95-genes-10-00739" /><span class="label" tagx="label" title="label">95.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abubucker</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyrwicz</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rychlewski</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCarter</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">R.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitreva</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular determinants archetypical to the phylum <i>Nematoda</i></span><span class="source" tagx="source" title="source">BMC Genomics</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">114</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-10-114">10.1186/1471-2164-10-114</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19296854">19296854</a></span></span></li> <li tag="ref"><a name="B96-genes-10-00739" /><span class="label" tagx="label" title="label">96.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Son</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenomics and Comparative Genomic Studies Robustly Support Division of the Genus <i>Mycobacterium</i> into an Emended Genus <i>Mycobacterium</i> and Four Novel Genera</span><span class="source" tagx="source" title="source">Front. Microbiol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">67</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2018.00067">10.3389/fmicb.2018.00067</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29497402">29497402</a></span></span></li> <li tag="ref"><a name="B97-genes-10-00739" /><span class="label" tagx="label" title="label">97.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmod</span><span class="given-names" tagx="given-names" title="given-names">N.Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">H.N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of a <i>Bacillus anthracis</i> specific indel in the yeaC gene and development of a rapid pyrosequencing assay for distinguishing <i>B. anthracis</i> from the B. cereus group</span><span class="source" tagx="source" title="source">J. Microbiol. Methods</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">278</span><span class="lpage" tagx="lpage" title="lpage">285</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mimet.2011.08.015">10.1016/j.mimet.2011.08.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21907250">21907250</a></span></span></li> <li tag="ref"><a name="B98-genes-10-00739" /><span class="label" tagx="label" title="label">98.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">S.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paschos</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schellhorn</span><span class="given-names" tagx="given-names" title="given-names">H.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Insertion/deletion-based approach for the detection of <i>Escherichia coli</i> O157: H7 in freshwater environments</span><span class="source" tagx="source" title="source">Environ. Sci. Technol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">11462</span><span class="lpage" tagx="lpage" title="lpage">11470</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/es502794h">10.1021/es502794h</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25166281">25166281</a></span></span></li> <li tag="ref"><a name="B99-genes-10-00739" /><span class="label" tagx="label" title="label">99.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Conserved inserts in the Hsp60 (GroEL) and Hsp70 (DnaK) proteins are essential for cellular growth</span><span class="source" tagx="source" title="source">Mol. Genet. Genom.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">281</span><span class="fpage" tagx="fpage" title="fpage">361</span><span class="lpage" tagx="lpage" title="lpage">373</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00438-008-0417-3">10.1007/s00438-008-0417-3</a></span></span></li> <li tag="ref"><a name="B100-genes-10-00739" /><span class="label" tagx="label" title="label">100.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lans</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vermeulen</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tissue specific response to DNA damage: <i>C. elegans</i> as role model</span><span class="source" tagx="source" title="source">DNA Repair</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">141</span><span class="lpage" tagx="lpage" title="lpage">148</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.dnarep.2015.04.025">10.1016/j.dnarep.2015.04.025</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25957488">25957488</a></span></span></li> <li tag="ref"><a name="B101-genes-10-00739" /><span class="label" tagx="label" title="label">101.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kammenga</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harvey</span><span class="given-names" tagx="given-names" title="given-names">S.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic variation in neurodegenerative diseases and its accessibility in the model organism <i>Caenorhabditis elegans</i></span><span class="source" tagx="source" title="source">Hum. Genom.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">12</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s40246-017-0108-4">10.1186/s40246-017-0108-4</a></span></span></li> <li tag="ref"><a name="B102-genes-10-00739" /><span class="label" tagx="label" title="label">102.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Norris</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grant</span><span class="given-names" tagx="given-names" title="given-names">B.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>C. elegans</i> as a model for membrane traffic</span><span class="source" tagx="source" title="source">WormBook</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">47</span><span class="pub-id"><a href="https://dx.doi.org/10.1895/wormbook.1.77.2">10.1895/wormbook.1.77.2</a></span></span></li> <li tag="ref"><a name="B103-genes-10-00739" /><span class="label" tagx="label" title="label">103.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranawade</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallick</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">B.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">PRY-1/Axin signaling regulates lipid metabolism in <i>Caenorhabditis elegans</i></span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">13</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0206540</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0206540">10.1371/journal.pone.0206540</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30403720">30403720</a></span></span></li> <li tag="ref"><a name="B104-genes-10-00739" /><span class="label" tagx="label" title="label">104.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nandan</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ban</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reiner</span><span class="given-names" tagx="given-names" title="given-names">N.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cherkasov</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Indel-based targeting of essential proteins in human pathogens that have close host orthologue(s): Discovery of selective inhibitors for <i>Leishmania donovani</i> elongation factor-1α</span><span class="source" tagx="source" title="source">Proteins</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">53</span><span class="lpage" tagx="lpage" title="lpage">64</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.21278">10.1002/prot.21278</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17243179">17243179</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Maximum-likelihood tree for 52 genome-sequenced nematode species. The tree was constructed based on the concatenated alignment of 17 orthologous proteins present in a single copy in these genomes as described in the Methods. Bootstrap scores for each node are indicated at the branch points. The bar indicates 0.2 changes per position. The major nematode groups at different phylogenetic levels are labeled. The tree was rooted using the outgroup species shown.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Partial sequence alignments of the proteins (<b>A</b>) Rab44 and (<b>B</b>) poly ADP-ribose glycohydrolase showing two CSIs (boxed) that are specific for the genus <i>Caenorhabditis</i>. Dashes (-) in these as well as all other alignments denote identity with the amino acid shown in the top sequence. Sequence information for only limited numbers of species is presented in this figure. More detailed alignments for these CSIs are shown in <a href="#app1-genes-10-00739">Figure S2</a>. Sequence information for 37 additional CSIs, which are also specific for the genus <i>Caenorhabditis</i> is provided in <a href="#app1-genes-10-00739">Figures S3–S39</a> and a summary of these CSIs is provided in <a href="#genes-10-00739-t001">Table 1</a>. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Partial sequence alignment of a conserved region from a protein annotated as abnormal cell migration protein 13 (MIG-13) containing a 2 aa insertion (boxed) which is specific for the family <i>Rhabditoidea</i>. This insertion is not present in the homologous proteins from other nematodes as well as other eukaryotic species. Sequence information for three additional CSIs, which are also specific for the family <i>Rhabditoidea</i> is provided in <a href="#app1-genes-10-00739">Figures S41–S43</a> and a summary of these CSIs is provided in <a href="#genes-10-00739-t002">Table 2</a>. Other details are the same as in the legend to <a href="#genes-10-00739-f002">Figure 2</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Excerpts from the sequence alignment of a conserved region of the protein tRNA (guanine-N(1)-)-methyltransferase protein containing a 4 aa CSI (boxed) which is specifically found in the homologs from the class <i>Chromadorea</i>. Sequence information for seven additional CSIs, which are also specific for the class <i>Chromadorea</i> is provided in <a href="#app1-genes-10-00739">Figures S44–S50</a> and a summary of these CSIs is provided in <a href="#genes-10-00739-t002">Table 2</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Partial sequence alignment from a conserved region of a Na(+)/H(+) exchange regulatory factor protein (NRFL-1) harboring a 1 aa insertion (boxed) which is specific for the phylum Nematoda. Most nematodes species contain two homologs of this protein and this CSI is specifically present in one of these two homologs. More detailed information regarding the species distribution of this CSI is provided in <a href="#app1-genes-10-00739">Figure S51</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Homology models of the <i>C. elegans</i> proteins (<b>A</b>) Rab-44, (<b>B</b>) poly ADP-ribose glycohydrolase and (<b>C</b>) tRNA (guanine-N(1)-)-methyltransferase showing the locations of the CSIs in the structures of these proteins. The CSIs are shown in red color in these figures. As seen from the presented structural overlap, the CSIs in all three studied proteins are localized within the surface-exposed loops of these proteins. More details regarding modeling of these structures are provided in the Methods section.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>A conceptual diagram summarizing the species specificities of different nematodes-specific CSIs identified in this work and the evolutionary relationships inferred from them and the constructed phylogenetic tree. The numbers of CSIs that are specific for different clades or species-groupings are noted on the respective nodes.</p>   </div> <div class="table-wrap_UNKNOWN" id="genes-10-00739-t001" orientation="portrait" position="float">
genes-10-00739-t001_Table 1<span class="label" tagx="label" title="label">Table 1</span> <p>Characteristics of the CSIs specific for the Genus <i>Caenorhabditis.</i></p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Protein Name</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><i>C. elegans</i> Gene Name</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Accession No.</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Figure No.</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Indel Size</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Indel Position</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Rab44</td> <td align="center" valign="top" rowspan="1" colspan="1">4R79.2</td> <td align="center" valign="middle" rowspan="1" colspan="1">AFP33163</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#genes-10-00739-f002">Figure 2</a>A, <a href="#app1-genes-10-00739">Figure S2A</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">233–263</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Poly ADP-ribose Glycohydrolase</td> <td align="center" valign="top" rowspan="1" colspan="1">parg-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001255324</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#genes-10-00739-f002">Figure 2</a>B, <a href="#app1-genes-10-00739">Figure S2B</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">5 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">411–454</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Poly (ADP-ribose) polymerase 2</td> <td align="center" valign="top" rowspan="1" colspan="1">parp-2</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001022057</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S3</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">2 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">389–420</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">DnaJ-domain containing chaperone protein</td> <td align="center" valign="top" rowspan="1" colspan="1">dnj-16</td> <td align="center" valign="middle" rowspan="1" colspan="1">OZF80352</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S4</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">186–207</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Cyclin-dependent kinase 12</td> <td align="center" valign="top" rowspan="1" colspan="1">cdk-12</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001254914</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S5</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">456–487</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">CRAL-TRIO domain-containing Sec14 protein</td> <td align="center" valign="top" rowspan="1" colspan="1">T23G5.2</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001040875</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S6</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">2 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">448–487</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Mammalian ZAK kinase homolog</td> <td align="center" valign="top" rowspan="1" colspan="1">zak-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001254942</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S7</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">80–109</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Probable 3',5'-cyclic phosphodiesterase</td> <td align="center" valign="top" rowspan="1" colspan="1">pde-2</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001022706</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S8</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">2 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">448–495</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Nuclear Hormone Receptor</td> <td align="center" valign="top" rowspan="1" colspan="1">nhr-68</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001256335</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S9</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">1–35</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">SMA2- like</td> <td align="center" valign="top" rowspan="1" colspan="1">sma-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001256383</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S10</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">2 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">1353–1393</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Glutathione transferase omega-1 *</td> <td align="center" valign="top" rowspan="1" colspan="1">C02D5.4</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001254962</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S11</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">65–103</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Probable 26S proteasome regulatory subunit</td> <td align="center" valign="top" rowspan="1" colspan="1">rpn-6.2</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001254973</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S12</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">46–90</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Serine/ Threonine protein phosphatase 2A Regulatory Subunit</td> <td align="center" valign="top" rowspan="1" colspan="1">pptr-2</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001256283</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S13</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">92–130</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Failed axon connections-like protein *</td> <td align="center" valign="top" rowspan="1" colspan="1">F53G12.9</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001293265</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S14</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">176–211</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">NADH dehydrogenase [ubiquinone] 1 alpha subcomplex assembly factor 2</td> <td align="center" valign="top" rowspan="1" colspan="1">Y116A8C.30</td> <td align="center" valign="middle" rowspan="1" colspan="1">XP_002632399</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S15</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">13 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">62–97</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Disorganized muscle protein 1</td> <td align="center" valign="top" rowspan="1" colspan="1">Cbn-dim-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">EGT45899</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S16</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">135–170</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">ETS (E26 transformation-specific) class transcription factor</td> <td align="center" valign="top" rowspan="1" colspan="1">ets-9</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001024482</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S17</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">54–78</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Glycine-rich domain-containing protein</td> <td align="center" valign="top" rowspan="1" colspan="1">F32B5.7</td> <td align="center" valign="middle" rowspan="1" colspan="1">EGT38541</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S18</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">430–466</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Heat shock protein 70</td> <td align="center" valign="top" rowspan="1" colspan="1">F11F1.1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001255199</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S19</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">2 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">364–399</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Heat shock protein 70</td> <td align="center" valign="top" rowspan="1" colspan="1">F11F1.1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001255199</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S20</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">437–481</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Abnormal cell migration protein 13</td> <td align="center" valign="top" rowspan="1" colspan="1">mig-13</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001024661</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S21</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">123–151</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Regulatory-associated protein of mTOR-like protein</td> <td align="center" valign="top" rowspan="1" colspan="1">daf-15</td> <td align="center" valign="middle" rowspan="1" colspan="1">XP_003089575</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S22</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">143–175</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Abnormal cell migration protein 13</td> <td align="center" valign="top" rowspan="1" colspan="1">mig-13</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001024661</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S23</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">3 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">141–170</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Abnormal cell migration protein 13</td> <td align="center" valign="top" rowspan="1" colspan="1">mig-13</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001024660</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S24</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">220–251</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Plexin</td> <td align="center" valign="top" rowspan="1" colspan="1">plx-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_500018</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S25</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">1460–1497</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Piwi-like protein *</td> <td align="center" valign="top" rowspan="1" colspan="1">ergo-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_503362</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S26</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">1020–1070</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Stomatin *</td> <td align="center" valign="top" rowspan="1" colspan="1">sto-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001123124</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S27</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">70–99</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Ral guanine nucleotide dissociation stimulator</td> <td align="center" valign="top" rowspan="1" colspan="1">rgl-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001123140</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S28</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">257–290</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Transglutaminase/ protease homolog</td> <td align="center" valign="top" rowspan="1" colspan="1">ltd-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001309573</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S29</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">261–290</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Vacuolar protein sorting-associated protein 41 homolog</td> <td align="center" valign="top" rowspan="1" colspan="1">vps-41</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001033544</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S30</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">209–242</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Serine/arginine-rich splicing factor</td> <td align="center" valign="top" rowspan="1" colspan="1">rsp-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001317731</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S31</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">13–36</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Serine/ Threonine-protein phosphatase PP1</td> <td align="center" valign="top" rowspan="1" colspan="1">Cni-W03D8.2</td> <td align="center" valign="middle" rowspan="1" colspan="1">PIC40784</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S32</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">159–191</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">NEPrilysin metallopeptidase *</td> <td align="center" valign="top" rowspan="1" colspan="1">nep-20</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001317749</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S33</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">761–804</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">DNA PRImase homolog</td> <td align="center" valign="top" rowspan="1" colspan="1">pri-2</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001251923</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S34</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">224–262</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Probable maleylacetoacetate isomerase</td> <td align="center" valign="top" rowspan="1" colspan="1">Y105E8A.21</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001252372</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S35</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">3 aa del</td> <td align="center" valign="middle" rowspan="1" colspan="1">56–91</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Glutathione S-transferase *</td> <td align="center" valign="top" rowspan="1" colspan="1">C25H3.7</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001254102</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S36</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">39–61</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">CTD nuclear envelope phosphatase 1 homolog</td> <td align="center" valign="top" rowspan="1" colspan="1">cnep-1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_001254124</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S37</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">32–52</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Kelch-domain protein</td> <td align="center" valign="top" rowspan="1" colspan="1">F53E4.1</td> <td align="center" valign="middle" rowspan="1" colspan="1">NP_506895</td> <td align="center" valign="middle" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S38</a></td> <td align="center" valign="middle" rowspan="1" colspan="1">6 aa ins</td> <td align="center" valign="middle" rowspan="1" colspan="1">206–248</td> </tr> <tr> <td align="center" valign="top" style="border-bottom:solid thin" rowspan="1" colspan="1">Intermediate filament protein *</td> <td align="center" valign="top" style="border-bottom:solid thin" rowspan="1" colspan="1">ifc-2</td> <td align="center" valign="top" style="border-bottom:solid thin" rowspan="1" colspan="1">NP_741705</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S39</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2 aa del</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">946-983</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>* Two isoforms of this protein are present in <i>Caenorhabditis</i> species.</p> </div> </div> </div> <div class="table-wrap_UNKNOWN" id="genes-10-00739-t002" orientation="portrait" position="float">
genes-10-00739-t002_Table 2<span class="label" tagx="label" title="label">Table 2</span> <p>Characteristics of the CSIs specific for the nematode suborder <i>Rhabditoidea</i> and class <i>Chromadorea.</i></p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Protein Name</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><i>C. elegans</i><br /><br />Gene Name</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Accession No.</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Figure (Fig. Sup) No.</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Indel Size</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Indel Position</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Specificity</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Cleavage Factor I<sub>m</sub> homolog</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">cfim-2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NP_001255355</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S40</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2 aa ins</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">87–130</td> <td rowspan="4" align="center" valign="middle" style="border-bottom:solid thin" colspan="1"><i>Rhabditoidea</i></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Methyl-CpG-binding protein</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">mbd-2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NP_001021012</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S41</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2 aa ins</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">158–200</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Abnormal cell migration protein 13</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">mig-13</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NP_001024660</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#genes-10-00739-f003">Figure 3</a><br /><br /><a href="#app1-genes-10-00739">Figure S42</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2 aa ins</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">71–105</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">PAX3- and PAX7 binding protein 1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">F43G9.12</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NP_001250840</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S43</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">126–164</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">tRNA (guanine-N(1)-)-methyltransferase</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">F46F11.10</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NP_491647</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#genes-10-00739-f004">Figure 4</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4 aa ins</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">632–669</td> <td rowspan="8" align="center" valign="middle" style="border-bottom:solid thin" colspan="1"><i>Chromadorea</i></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Palmitoyltransferase <sup>a</sup></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">spe-10</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">KHJ83757</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S44</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1 aa del</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">234–270</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Palmitoyltransferase</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">spe-10</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">KHJ83757</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S45</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2 aa del</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">255–282</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Battenin</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">cln-3.3</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">EGT30700</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S46</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">3 aa ins</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">162–194</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ETS (E26 transformation-specific) class transcription factor</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ets-5</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">KJH47557</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S47</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">122–155</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Heterogeneous nuclear ribonucleoprotein A1 *</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">H28G03.1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">KJH46562</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S48</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">93–122</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Heterogeneous nuclear ribonucleoprotein A1 *</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">H28G03.1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">XP_013302959</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S49</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">5 aa del</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">139–171</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Regulator of G-protein signaling 7 <sup>a</sup></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Cbn-rgs-7</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">EGT30339</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#app1-genes-10-00739">Figure S50</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">221–252</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Na(+)/H(+) Exchange Regulatory Factor *</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">nrfl-1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">NP_001294068 </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><a href="#genes-10-00739-f005">Figure 5</a><br /><br /><a href="#app1-genes-10-00739">Figure S51</a></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1 aa ins</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">210–245</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Nematoda</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>* Two isoforms of this protein are present in <i>Rhabitida</i> species. <sup>a</sup> These CSIs are not found in <i>Strongyloides ratti</i>, which branches deeply in comparison to the other <i>Chromadorea species</i>.</p> </div> </div> </div> </div>  </body></html>