<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Biomol Struct Dyn</span><span class="iso-abbrev" title="iso-abbrev">J Biomol Struct Dyn</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Biomolecular Structure &amp;amp; Dynamics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0739-1102</span><span class="issn-epub" tagx="issn" title="issn-epub">1538-0254</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7544953">7544953</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/07391102.2020.1808077">10.1080/07391102.2020.1808077</a></span><span class="publisher-id" title="publisher-id">1808077</span><div class="unknown" title="article-version" tag="article-version">
Version of Record</div> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title"><i>In-silico</i> screening of plant-derived antivirals against main protease, 3CL<sup>pro</sup> and endoribonuclease, NSP15 proteins of SARS-CoV-2</div> <div class="alt-title" title="alt-title">
A. Sharma et al.</div> <div class="alt-title" title="alt-title">
Journal of Biomolecular Structure and Dynamics</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">Aniket</span></span><a href="#AF0001">a</a><a href="#EN0002">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goyal</span><span class="given-names" tagx="given-names" title="given-names">Shubham</span></span><a href="#AF0001">a</a><a href="#EN0002">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">Arvind Kumar</span></span><a href="#AF0002">b</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">Pawan</span></span><a href="#AF0003">c</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-2610-3142</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">Lovely</span></span><a href="#AF0001">a</a><a href="#AN0001" /></span><span class="citation_author_institution" id="AF0001">[a], <span class="institution" title="institution">Amity Institute of Biotechnology, Amity University</span><span class="city" tagx="city" title="city">Noida</span><span class="state" tagx="state" title="state">Uttar Pradesh</span><span class="country" tagx="country" title="country">India</span></span><span class="citation_author_institution" id="AF0002">[b], <span class="institution" title="institution">Department of Biotechnology and Bioinformatics, Jaypee University of Information Technology</span><span class="city" tagx="city" title="city">Solan</span><span class="state" tagx="state" title="state">Himachal Pradesh</span><span class="country" tagx="country" title="country">India</span></span><span class="citation_author_institution" id="AF0003">[c], <span class="institution" title="institution">Bioinformatics Centre, National Institute of Immunology</span><span class="city" tagx="city" title="city">New Delhi</span><span class="country" tagx="country" title="country">India</span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""> <p>Supplemental data for this article can be accessed online at <a href="https://doi.org/10.1080/07391102.2020.1808077">https://doi.org/10.1080/07391102.2020.1808077</a>.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label"><sup>â€ </sup></span><p>Authors contributed equally in the research work.</p> </div> <div class="corresp" title="corresp"><b>CONTACT</b> Lovely Gupta <span class="email" tagx="email" title="email">lgupta@amity.edu</span><span class="institution" title="institution">Amity Institute of Biotechnology, Amity University</span>, <span class="city" tagx="city" title="city">Noida</span>, <span class="state" tagx="state" title="state">Uttar Pradesh</span><span class="postal-code" tagx="postal-code" title="postal-code">201301</span>, <span class="country" tagx="country" title="country">India</span></div> </div><span class="pub-date-" title="pub-date-">: <span>2020-9-9</span></span><span class="pub-date-" title="pub-date-">: <span>2020</span></span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">15</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Informa UK Limited, trading as Taylor &amp;amp; Francis Group</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the COVID-19 pandemic or until permissions are revoked in writing. Upon expiration of these permissions, PMC is granted a perpetual license to make this article available via PMC and Europe PMC, consistent with existing copyright protections.</span></span></div><a class="self-uri" href="TBSD_0_1808077.pdf" title="self-uri">TBSD_0_1808077.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Novel Coronavirus or SARS-CoV-2 outbreak has developed a pandemic condition all over the world. The virus is highly infectious and spreads by human to human local transmission mode. Till date, there is no vaccination or drugs been approved for the treatment by the World Health Organisation. Henceforth, the discovery of the potential drugs is an urgent and utmost requirement for the medical fraternity. Since, the side effects of plant-derived compounds will be lower compared to synthetic/chemical drugs. The Main protease (3CL<sup>pro</sup> or NSP5) and endoribonuclease (NSP15) proteins are necessity for viral replication and its survival in the host cell. In the present study, <i>in-silico</i> approach of drug development was used to search for potential antiviral plant-derived compounds as inhibitors against SARS-CoV-2 replication proteins. Eight plant-derived compounds of which the antiviral activity was known and available, and two reported drugs against SARS-CoV-2 selected for the molecular docking analysis. The docking results suggested that bisdemethoxycurcumin, demethoxycurcumin, scutellarin, quercetin and myricetin showed least binding energy, <i>i.e.,</i> greater than âˆ’6.5â€‰Kcal/mol against 3CL<sup>pro</sup> and endoribonuclease of SARS-CoV-2. Further studies of ADME-Tox and bioavailability of drugs were also performed that exhibited efficient parameters of drug likeness. Molecular dynamics simulation calculations were performed for the most negative binding affinity of the compound to evaluate the dynamic behavior,and stability of protein-ligand complex. Our findings suggest that these compounds could be potential inhibitors of SARSâ€�CoV-2 main protease and endoribonuclease. However, further <i>in-vitro</i> and pre-clinical experiments would validate the potential inhibitors of SARSâ€�CoVâ€�2 proteins.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
docking</div> <div class="kwd" title="kwd">
plant-derived compounds</div> <div class="kwd" title="kwd">
3CL<sup>pro</sup></div> <div class="kwd" title="kwd">
endoribonuclease</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>A mysterious breakdown of typical pneumonia by a novel coronavirus (COVID-19), created a pandemic situation all over the world by spreading the disease named as severe acute respiratory syndrome-related coronavirus-2 (SARS-CoV-2) by the World Health Organization (WHO). This disease is more contagious than previous outbreaks of severe acute respiratory syndrome (SARS) in the year 2002 and the Middle East respiratory syndrome (MERS) in 2012 (Dhama etÂ al., <a href="#CIT0010">2020</a>; Guo etÂ al., <a href="#CIT0018">2020</a>). By the end of the year 2019, an outbreak from COVID-19 occurred in Wuhan, Hubei Province of China. It was reported to infect over 3.3 million people and killed more than 0.23 million people in around 215 countries as of May 03, 2020 (<a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">https://www.who.int/emergencies/diseases/novel-coronavirus-2019</a>). According to WHO situation report-71, India reported 1071 positive cases of SARS-CoV-2 and after one month, WHO situation report-99 reported 29435 positive cases of coronavirus affected patients. Indiaâ€™s death rate of COVID-19 is still lower than the global death rate of around 3.4 percent, according to data and statistics website worldometer (<a href="https://www.worldometers.info/coronavirus/">https://www.worldometers.info/coronavirus/</a>).</p> <p>SARS-CoV-2 is considered to be a seventh known human coronavirus (hCoV) from the same family after 229E, NL63, OC43, HKU1, MERS-CoV and SARS-CoV that belongs to the Î²-coronavirus family (<i>coronaviridae</i>). It contains a single-stranded positive-sense (+) RNA encapsulated within the lipid membrane. It has been reported that the genome of SARS-CoV-2 was 96.2%, 79%, and 50% nucleotide identical with bat-CoV RaTG13, SARS-CoV, and MERS genome, respectively (Dhama etÂ al., <a href="#CIT0010">2020</a>; Guo etÂ al., <a href="#CIT0018">2020</a>). Therefore, the bat was suspected to be a natural host of virus origin that infected human via pangolins (<i>Manisjavanica)</i> as an intermediate host.</p> <p>The genome of SARS-CoV-2 is divided into two parts: open reading frames (ORFs) that encode accessory and structural proteins; and non-structural proteins (NSP). Structural protein involves spike (S) glycoprotein, small envelope (E) protein, matrix (M) protein, and nucleocapsid (N) protein (Boopathi etÂ al., <a href="#CIT0004">2020</a>). Sixteen NSPâ€™s are known to encode various enzymes required for replication and transcription process such as main protease (NSP5 and NSP3), RNA dependent-RNA polymerase (NSP12), a helicase/triphosphatase (NSP13), an exoribonuclease (NSP14), an endonuclease (NSP15) (Chan etÂ al., <a href="#CIT0005">2020</a>). Studies have proved that protein NSP15 of SARS-CoV-2 has 88% and âˆ¼50% sequence identity, 95% and 65% similarity with SARS-CoV and MERS, respectively (Kim etÂ al., <a href="#CIT0027">2020</a>). NSP15 protein, an endoribonuclease, one of the crucial NSP for viral replication and transcription. It is highly conserved in all known CoVs (Hackbart etÂ al., <a href="#CIT0020">2020</a>). It is a nidoviral RNA uridylate-specific (NendoU) endoribonuclease containing 346 amino acid and made up of 2 domains NSP-15A1 and NSP-15B. It is an interferon (IFN) antagonist and inhibits interferon-Î² production via an endoribonuclease activity-independent mechanism. (Liu etÂ al., <a href="#CIT0033">2019</a>)</p> <p>There are two proteases encoded in the SARS-CoV-2 RNA strand: papain-like protease (PL<sub>P</sub> or NSP3) and 3-Chymotrypsin-like protease (3CL<sup>pro</sup> or NSP5) co-translationally cleave the two polypeptides into mature NSPs (Lim etÂ al., <a href="#CIT0031">2000</a>). The 3CL<sup>pro</sup>, also known as M<sub>pro</sub>, has a dominant role in the post-translational processing of the replicase gene. Protease M<sub>pro</sub> has been reported to be a conserved protein, in favour of new drug target in (+) strand coronaviruses (Chandel etÂ al., <a href="#CIT0007">2020</a>). It has two homodimer NSP5A and NSP5B. It belongs to the cysteine protease class of protein and responsible for the cleavage of viral peptides into functional units for replication and packaging within the host cells (Liu etÂ al., <a href="#CIT0032">2020</a>). It is produced by ORF pp1ab only. Targeting M<sub>pro</sub> enzyme will inhibit the viral maturation and enhance the host innate immune response against COVID-19 (Elmezayen etÂ al., <a href="#CIT0013">2020</a>). Other studies suggested that the sequence similarity of NSP5 of SARS-CoV-2 reported nearly 95% with SARS-CoV (Chan etÂ al., <a href="#CIT0005">2020</a>). In the current study, two protein targets (3CL<sup>pro</sup> and endoribonuclease) of SARS-CoV-2 selected based on conserved sequence and their importance in virus replication according to the literature.</p> <p>For the treatment of COVID-19, doctors are prescribing antimalarial, anti-HIV, anti-influenza drugs and their combinations, but these drugs are not permanent cure of coronavirus infection (Joshi etÂ al., <a href="#CIT0025">2020</a>). Drugs such as Hydroxychloroquine, Chloroquine, Nelfinavir, Rhein, Withaferin A, Withanolide D, Enoxacin, Mercaptopurine and Aloe-emodin are tested against various target proteins of viruses (Muralidharan etÂ al., <a href="#CIT0038">2020</a>). Conversely, various phytochemicals reported in the literature to possess potential anti-viral activity that could be an alternative to restrict the coronavirus replication (Elfiky, <a href="#CIT0012">2020</a>). Natural compounds consist of high chemical diversity, lower cost of production than that of biotechnological products or compounds synthesised from combinatorial chemistry and also possess milder or inexistent side effects than chemical drugs (Kitazato etÂ al., <a href="#CIT0029">2007</a>). Therefore, in the present study attempt has been made to screen and evaluate the plant-derived compounds that have therapeutic applications against different genetically and functionally diverse viruses using molecular docking study.</p> </div> <div class="materials" title="materials"> <div class="title" tagx="title" title="title">
Material and method</div> <div class="retrievalofthetargetstructureandtheirpreparation" title="sec"> <div class="title" tagx="title" title="title">
Retrieval of the target structure and their preparation</div> <p>The protein structure of 3-chymotrypsin-like cysteine protease (3CL<sup>pro</sup>) enzyme (PDB ID: 6LU7) with 2.1Ã… resolution and endoribonuclease, non-structural protein 15 (NSP15) protein (PDB ID: 6VWW) with 2.2Ã… resolution of SARS-CoV- 2 were downloaded from RCSB protein databank (<a href="http://www.rcsb.org">http://www.rcsb.org</a>). Initially, different tasks were performed for protein preparation such as charge assignment, solvation parameters and fragmental volumes using SPDVB-4.10 version (Morris etÂ al., <a href="#CIT0036">2009</a>). The protein molecules were further optimized using AutoDock4 Tool for the molecular docking (Morris etÂ al., <a href="#CIT0036">2009</a>).</p> </div> <div class="secondarystructureprediction" title="sec"> <div class="title" tagx="title" title="title">
Secondary structure prediction</div> <p>The secondary structure of proteins was speculated using the Self-Optimized Prediction Method with Alignment (SOPMA) server at <a href="https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_sopma.html">https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_sopma.html</a> (Geourjon &amp;amp; DelÃ©age, <a href="#CIT0017">1995</a>). SOPMA predicts four conformational states including Helix, Beta sheets and bridges, Turns and Coils for each antigen.</p> </div> <div class="preparationofligands" title="sec"> <div class="title" tagx="title" title="title">
Preparation of ligands</div> <p>The natural compounds for the ligand dataset were selected from the previous literature (<a href="#t0002">Table 2</a>). Eight selected natural compounds and two drugs were obtained from the PubChem compound database (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702940/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702940/</a>) in InChI (Inch key) format. The two-dimensional (2D) ligand structures were generated using ACD/Chemsketch and save as mol file format. The three-dimensional (3D) structure of ligands were converted using OpenBabel tool (Oâ€™Boyle etÂ al., <a href="#CIT0039">2011</a>).</p> </div> <div class="moleculardocking" title="sec"> <div class="title" tagx="title" title="title">
Molecular docking</div> <p>The docking analysis was performed by molecular docking program AutoDock4 Tool (Morris etÂ al., <a href="#CIT0036">2009</a>) to explore the poses of potential inhibitors within the targets active site. For protein-ligand interactions, the Lamarckian genetic algorithm was utilized to perform the docking with the preset parameters. The total number of poses was set to 50. Poses were further clustered using all atom RMSD cutoff of 0.3â€‰Ã… to remove redundancy and on average 20 cluster representatives were kept. All other parameters for docking and scoring were used as default sets. The protein structure was kept rigid in all steps. All the docking poses and interaction analysis has been performed using the UCSF Chimera (Pettersen etÂ al., <a href="#CIT0040">2004</a>) and Discovery Studio Visualizer programs (<a href="http://accelrys.com/products/collaborative-science/biovia-discovery-studio/visualization-download.php">http://accelrys.com/products/collaborative-science/biovia-discovery-studio/visualization-download.php</a>). The amino acids residues that displayed interactions with the ligand were documented in <a href="#t0003">Table 3</a> and <a href="#F0001">Figure S1</a> and <a href="#F0002">S2</a> (Supplementary material).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Ribbon structure of (A) 3CL<sup>pro</sup> and (B) NSP15 protein. Crystal structure of (C) 3CL<sup>pro</sup> (PDB-ID:6LU7) and (D) NSP15 endoribonuclease protein (PDB-ID:6VWW). Source- PDB databank.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>The predicted secondary structure of the (A) 3CL<sup>pro</sup> and (B) NSP15 endoribonuclease using SOPMA software, whereas h: Alpha helix, e: Extended strand, t: Beta-turn and c: Random coil. The graphical representation of secondary elements in 3CL<sup>pro</sup> and NSP15 endoribonuclease protein (blue: alpha helix, red: extended, green: beta turn, yellow: random coil).</p>   </div> </div> <div class="adme-toxprediction" title="sec"> <div class="title" tagx="title" title="title">
ADME-Tox prediction</div> <p>The qualitative measurement of absorption, distribution, metabolism, excretion, and toxicity profile of selected compounds were predicted by using SwissADME program (<a href="http://www.swissadme.ch/index.php">http://www.swissadme.ch/index.php</a>) (Hoda etÂ al., <a href="#CIT0021">2019</a>). The major ADME-Tox properties parameters include Molecular weight, H-bond acceptor, H-bond donors, Predicted octanol/water partition coefficient (MLogP), Total Polar Surface Area (TPSA), Lipinski (drug likeness), GI absorption, BBB permeant, Bioavailability score.</p> </div> <div class="moleculardynamicssimulationsetupandanalysis" title="sec"> <div class="title" tagx="title" title="title">
Molecular dynamics simulation setup and analysis</div> <p>To analyze the conformational dynamics of the unbound and bound state of 3CL<sup>pro</sup> and endoribonuclease enzyme, molecular dynamics simulation has been performed using the Amber16 package (Gajula etÂ al., <a href="#CIT0016">2016</a>). After the preparation of the docked ligand molecules using the antechamber (Wang etÂ al., <a href="#CIT0055">2006</a>), tleap module is used to prepare the cubic simulation box of size 10â€‰Ã… considering the protein at the center, the box was filled with TIP3P water molecules (Jorgensen etÂ al., <a href="#CIT0024">1983</a>) for solvation under periodic boundary condition. Some water molecules were replaced by the counterions to neutralize the systems. The Generalized Amber Force Field (GAFF) for ligand (Wang etÂ al., <a href="#CIT0056">2004</a>) and FF14SB FF for protein (Maier etÂ al., <a href="#CIT0035">2015</a>) was used during the simulation setup. For all electrostatic interactions calculation, Particle Mesh Ewald (PME) method (Essmann etÂ al., <a href="#CIT0015">1995</a>) was used and 2 fs time step is used with the SHAKE algorithm (Ryckaert etÂ al., <a href="#CIT0045">1977</a>) to constrain the hydrogen bonds.</p> <p>Prepared simulation box was initially minimized for 500,000 steps (400,000SD +100,000CG) to relax the system. The minimized system was gradually heated from the 0 to 300â€‰K over the period of 6000ps. A 1â€‰ns simulation run was further used to equilibrate the system under NVT ensemble. The final MD production run of 50â€‰ns was performed with an integration time step of 2.0 fs. Simulation conformations were collected after every 50ps and all trajectories were analyzed for the RMSD, RMSF, SASA and hydrogen bond dynamics using the CPPTRAJ module (Roe &amp;amp; Cheatham, <a href="#CIT0044">2013</a>) of the Amber software.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p>To perform the target protein analysis, FASTA sequences and 3â€‰D structure of 3CL<sup>pro</sup> and endoribonuclease were retrieved from PDB databank (PDB ID: 6LU7 and 6VWW) (<a href="#F0001">Figure 1</a>). The secondary structure prediction indicated that protein 3CL<sup>pro</sup> consisted 29.08% Î±-helix (h), 27.12% extended strand (e), 11.44% beta-turn (t), 32.35% random coil (c) elements and NSP15 endoribonuclease comprised of 24.26% Î±-helix (h), 26.95% extended strand (e), 9.70% beta-turn (t), 39.08% random coil (c). There are two graphs shown in the result- 1) to visualize the prediction; 2) score curves for all predicted states (<a href="#F0002">Figure 2</a>). It also shows the parameters such as window width, number of states etc. that are used for the prediction.</p> <p>Molecular docking was performed to evaluate the interaction of plant-derived compounds, having reported pharmacological activities against two non-structural proteins of SARS-CoV-2. Non-structural proteins, 3CL<sup>pro</sup> and endoribonuclease were targeted and the docking pocket was formed at the catalytic sites of the respective proteins as mentioned in <a href="#t0001">Table 1</a>. Also, the binding affinity of interacted compounds at the active site were assessed. Known anti-viral drugs such as remdesivir and ribavirin were docked against 3CL<sup>pro</sup> as well as endoribonuclease proteins. Plant-derived active compounds, reported with their pharmacological properties were listed in <a href="#t0002">Table 2</a>.</p> <div class="table-wrap_UNKNOWN" id="t0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>Active site residues of 3CL<sup>pro</sup> and NSP15 endoribonuclease protein.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Protein Site</th> <th align="center">Residues</th> <th align="center">Amino Acids</th> <th align="center">Co-ordinates of amino acid residues</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">3CL<sup>pro</sup></td> <td align="left">Cys</td> <td align="char" char=".">145</td> <td align="left">x-axis: 3.032<br /><br />y-axis: âˆ’4.728<br /><br />z-axis: 39.052</td> </tr> <tr> <td align="left">Endoribonuclease (NSP15)</td> <td align="left">Thr</td> <td align="char" char=".">341</td> <td align="left">x-axis: âˆ’93.206<br /><br />y-axis: 14.072<br /><br />z-axis: âˆ’5.47</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="t0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> <p>Selected plant-derived compounds based on their known pharmacological properties.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">S.no.</th> <th align="center">Plant-derived compound</th> <th align="center">Reported activity</th> <th align="center">Disease</th> <th align="center">References</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1.</td> <td align="left">Curcumin</td> <td align="left">Anti-viral</td> <td align="left">Herpes virus, Ebola</td> <td align="left">(Kutluay etÂ al.,<a href="#CIT0030">2008</a>)</td> </tr> <tr> <td align="left">2.</td> <td align="left">Demethoxycurcumin</td> <td align="left">Anti-viral</td> <td align="left">Ebola</td> <td align="left">(Baikerikar,<a href="#CIT0002">2017</a>)</td> </tr> <tr> <td align="left">3.</td> <td align="left">Bisdemethoxycurcumin</td> <td align="left">Anti-viral</td> <td align="left">Ebola</td> <td align="left">(Baikerikar,<a href="#CIT0002">2017</a>)</td> </tr> <tr> <td align="left">4.</td> <td align="left">Scutellarin</td> <td align="left">Anti-viral</td> <td align="left">Covid-19 (Nsp-13)</td> <td align="left">(Yu etÂ al.,<a href="#CIT0058">2012</a>)</td> </tr> <tr> <td align="left">5.</td> <td align="left">Myricetin</td> <td align="left">Anti-inflammatory</td> <td align="left">Diabetes mellitus</td> <td align="left">(DormÃ¡n etÂ al., <a href="#CIT0011">2016</a>)</td> </tr> <tr> <td align="left">6.</td> <td align="left">Quercetin</td> <td align="left">Anti-viral</td> <td align="left">Mingo virus, Poliovirus, Herpes small virus-2</td> <td align="left">(Veckenstedt etÂ al.,<a href="#CIT0052">1987</a>)</td> </tr> <tr> <td align="left">7.</td> <td align="left">Bergapten</td> <td align="left">Anti-viral</td> <td align="left">Coxsackievirus B3, Herpes small virus-1</td> <td align="left">(Rajtar etÂ al.,<a href="#CIT0042">2017</a>)</td> </tr> <tr> <td align="left">8.</td> <td align="left">Isoflavone</td> <td align="left">Anti-viral</td> <td align="left">Herpes small Virus-1</td> <td align="left">(Akula etÂ al.,<a href="#CIT0001">2002</a>)</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="t0003" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3.</span> <p>Details of molecular docking analysis of Plant-based compounds against SARS-CoV-2 endoribonuclease and 3CL<sup>pro</sup> proteins.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th rowspan="2" align="left">Compound Name</th> <th rowspan="2" align="center">PubChem CID</th> <th rowspan="2" align="center">Structure</th> <th colspan="2" align="center">3CL<sup>Pro</sup><hr /> </th> <th colspan="2" align="center">Endoribonuclease <hr /> </th> </tr> <tr> <th align="center">Binding affinity</th> <th align="center">Amino acid residues</th> <th align="center">Binding affinity</th> <th align="center">Amino acid residues</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Demethoxycurcumin</td> <td align="center">5469424</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’7.02</td> <td align="center">His 163</td> <td align="center">âˆ’7.51</td> <td align="center">Lys 290; Gly 248; His 235; Thr 341; Glu 340</td> </tr> <tr> <td align="left">Curcumin</td> <td align="center">969516</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’6.04</td> <td align="center">Asn 119; His 163</td> <td align="center">âˆ’6.48</td> <td align="center">Lys 290</td> </tr> <tr> <td align="left">Bisdemethoxycurcumin</td> <td align="center">5315472</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’7.3</td> <td align="center">Cys 145;His 41; His 41; Phe 140; His 163</td> <td align="center">âˆ’6.56</td> <td align="center">Lys 290</td> </tr> <tr> <td align="left">Quercetin</td> <td align="center">5280343</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’6.58</td> <td align="center">Asn 142; Gly 143; Ser 144; Leu 141; Cys 145; His 41; Thr 26; Thr 26</td> <td align="center">âˆ’6.49</td> <td align="center">Lys 290; His 235;His 250; Glu 340</td> </tr> <tr> <td align="left">Myricetin</td> <td align="center">5281672</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’6.15</td> <td align="center">Gly 143; Leu 141; His 41; Thr 26</td> <td align="center">âˆ’6.52</td> <td align="center">Lys 290; His 235;His 250; Glu 340</td> </tr> <tr> <td align="left">Bergapten</td> <td align="center">2355</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’5.98</td> <td align="center">His 163; His 163; Phe 140</td> <td align="center">âˆ’5.92</td> <td align="center">Lys 290; His 235;Ser 294; Thr 341</td> </tr> <tr> <td align="left">Scutellarin</td> <td align="center">5281697</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’7.13</td> <td align="center">Gly 143; His 163</td> <td align="center">âˆ’6.97</td> <td align="center">Lys 290; His 250;Gly 248; Glu 340</td> </tr> <tr> <td align="left">Isoflavone</td> <td align="center">72304</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’5.69</td> <td align="center">Gly 143, Cys 145</td> <td align="center">âˆ’5.47</td> <td align="center">His 235; Thr 341</td> </tr> <tr> <td align="left">Remdesivir</td> <td align="center">121304016</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">196</td> <td align="center">Phe 140; His 164</td> <td align="center">âˆ’7.7</td> <td align="center">Glu 340</td> </tr> <tr> <td align="left">Ribavirin</td> <td align="center">37542</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’5.43</td> <td align="center">Ser 144; Ser 144; Gly 143; Thr 26; Thr 26; Thr 26; His 41; Cys 145</td> <td align="center">âˆ’5.84</td> <td align="center">Ser 294; Ser 294; Pro 344; Lys 290; His 235; His 235; Tyr 343; His 250; Thr 341</td> </tr> </tbody> </table> </div> <div class="interactionofplant-derivedcompoundswithactivesiteofprotein3clpro" title="sec"> <div class="title" tagx="title" title="title">
Interaction of plant-derived compounds with active site of protein 3CL<sup>pro</sup></div> <p>Bisdemethoxycurcumin and Demethoxycurcumin the analogues of curcumin, exhibited the docked score of âˆ’7.3Kcal/mol and âˆ’7.02Kcal/mol, and formed five hydrogen bonds with amino acid residues Lys-290, Gly-248, His-235, Thr-341, and Glu-340 and one hydrogen bond with His-163 amino acid residue, respectively (<a href="#F0003">Figure 3</a>). Scutellarin represented the binding affinity of âˆ’7.13â€‰Kcal/mol was found to form two hydrogen bonds with amino acid residues Gly-143 and His-163, followed by flavonoid quercetin with binding energy of âˆ’6.58Kcal/mol and hydrogen bonds formed with amino acids Asn-142, Gly-143, Ser-144, Leu-141, Cys-145, His-41, and Thr-26. Among all, four plant-derived compounds with binding affinity more than âˆ’6.5Kcal/mol were myricetin (-6.15Kcal/mol), curcumin (-6.04Kcal/mol), bergapten (-5.98Kcal/mol) and isoflavone (-5.69Kcal/mol) with amino acids present in the docking pocket of 3CL<sup>pro</sup> shown in <a href="#t0003">Table 3</a>. Furthermore, the anti-viral drugs remedesivir and ribavirin showed +196â€‰Kcal/mol and âˆ’5.43â€‰Kcal/mol binding affinity, respectively.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>Binding interactions of eight plant-derived compounds <b>(a-j)</b> with the active site of SARS-CoV-2 3CL<sup>pro</sup> protein target site (PDB ID: 6LU7).</p>   </div> </div> <div class="interactionofplant-derivedcompoundswithactivesiteofproteinendoribonuclease" title="sec"> <div class="title" tagx="title" title="title">
Interaction of plant-derived compounds with active site of protein endoribonuclease</div> <p>Compounds represented a similar kind of affinity with endoribonuclease as observed with 3CL<sup>pro</sup>. Based on binding affinities, three compounds were in common with both the protein targeted sites. Demethoxycurcumin with binding affinity of âˆ’7.51Kcal/mol and formed five hydrogen bonds with Lys-290, Gly-248, His-235, Thr-341, and Glu-340 amino acid residue. Another compound scutellarin, with binding affinity of âˆ’6.97Kcal/mol with Lys-290, His-250, Gly-248, and Glu-340 amino acids formed four hydrogen bonds within docking pocket followed by bisdemethoxycurcumin, binding affinity of âˆ’6.56Kcal/mol and one hydrogen bond formation with Lys-290. The flavonoid group of myricetin and quercetin exhibited negligible difference in their binding energies âˆ’6.52 and âˆ’6.49â€‰Kcal/mol, but hydrogen bond formed with same amino acids residues in the pocket- Lys-290, His-235, His-250, and Glu-340. Bergapten (-5.89Kcal/mol) and isoflavone (-5.69Kcal/mol) also showed the minimum binding energy and formed hydrogen bonds. The anti-viral drugs remedesivir docked with âˆ’7.7Kcal/mol with one hydrogen bond, whereas ribavirin binding energy (-5.84Kcal/mol) was found close to that of 3CL<sup>pro</sup> target site (<a href="#t0003">Table 3</a>; <a href="#F0004">Figure 4</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>Binding interactions of eight plant-derived compounds <b>(a-j)</b> with the active site of SARS-CoV-2 NSP15 endoribonuclease (PDB ID: 6VWW).</p>   </div> <p>All the plant-derived compounds studied were analysed for their physicochemical properties and ADME to screen bioavailability via oral route. The eight compounds listed in <a href="#t0004">Table 4</a> showed the five key criteria to qualify a compound to possess drug-like properties. The results explained that the plant-derived compounds studied would be more potent to be used as bioactive compounds as compared to the existing anti-viral drug remedesivir. All the compounds have molecular weight less than 500â€‰g/mol, lipophilicity (MlogP) value less than 5, hydrogen bond donor and acceptor are less than 5 and 10, respectively which corresponds to no violations of Lipinskiâ€™s rule of five. Thus, it suggested that all of these studied compounds have the potential ability to work effectively as novel inhibitors.</p> <div class="table-wrap_UNKNOWN" id="t0004" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4.</span> <p>Physicochemical analysis of potential inhibitors of 3CL<sup>pro</sup> and NSP15 endoribonuclease protein of SARS-CoV-2.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th colspan="2" align="left">Compounds</th> <th align="center">Molecular formula</th> <th align="center">ADME properties (Lipinkiâ€™s rule of five)</th> <th align="center">Values</th> </tr> </thead> <tbody valign="top"> <tr> <td colspan="2">Demethoxycurcumin</td> <td align="center">C<sub>20</sub>H<sub>18</sub>O<sub>5</sub></td> <td align="left">Molecular weight (&amp;lt;500g/mol)<br /><br />LogP (&amp;lt;5)<br /><br />H-bond donor (&amp;lt;5)<br /><br />H-bond acceptor (&amp;lt;10)<br /><br />Violation</td> <td align="center">338.4â€‰g/mol<br /><br />2.78<br /><br />2<br /><br />5<br /><br />0</td> </tr> <tr> <td colspan="2">Curcumin</td> <td align="center">C<sub>21</sub>H<sub>20</sub>O<sub>6</sub></td> <td align="left">Molecular weight (&amp;lt;500g/mol)<br /><br />LogP (&amp;lt;5)<br /><br />H-bond donor (&amp;lt;5)<br /><br />H-bond acceptor (&amp;lt;10)<br /><br />Violation</td> <td align="center">368.4â€‰g/mol<br /><br />3.27<br /><br />2<br /><br />6<br /><br />0</td> </tr> <tr> <td colspan="2">Bisdemethoxycurcumin</td> <td align="center">C<sub>19</sub>H<sub>16</sub>O<sub>4</sub></td> <td align="left">Molecular weight (&amp;lt;500g/mol)<br /><br />LogP (&amp;lt;5)<br /><br />H-bond donor (&amp;lt;5)<br /><br />H-bond acceptor (&amp;lt;10)<br /><br />Violation</td> <td align="center">308.3â€‰g/mol<br /><br />1.75<br /><br />2<br /><br />4<br /><br />0</td> </tr> <tr> <td colspan="2">Quercetin</td> <td align="center">C<sub>15</sub>H<sub>10</sub>O<sub>7</sub></td> <td align="left">Molecular weight (&amp;lt;500g/mol)<br /><br />LogP (&amp;lt;5)<br /><br />H-bond donor (&amp;lt;5)<br /><br />H-bond acceptor (&amp;lt;10)<br /><br />Violation</td> <td align="center">302.23â€‰g/mol<br /><br />1.63<br /><br />5<br /><br />7<br /><br />0</td> </tr> <tr> <td colspan="2">Myricetin</td> <td align="center">C<sub>15</sub>H<sub>10</sub>O<sub>8</sub></td> <td align="left">Molecular weight (&amp;lt;500g/mol)<br /><br />LogP (&amp;lt;5)<br /><br />H-bond donor (&amp;lt;5)<br /><br />H-bond acceptor (&amp;lt;10)<br /><br />Violation</td> <td align="center">318.23â€‰g/mol<br /><br />1.08<br /><br /><b>6</b><br /><br />8<br /><br />1</td> </tr> <tr> <td colspan="2">Bergapten</td> <td align="center">C<sub>12</sub>H<sub>8</sub>O<sub>4</sub></td> <td align="left">Molecular weight (&amp;lt;500g/mol)<br /><br />LogP (&amp;lt;5)<br /><br />H-bond donor (&amp;lt;5)<br /><br />H-bond acceptor (&amp;lt;10)<br /><br />Violation</td> <td align="center">216.19â€‰g/mol<br /><br />2.29<br /><br />0<br /><br />4<br /><br />0</td> </tr> <tr> <td colspan="2">Scutellarin</td> <td align="center">C<sub>15</sub>H<sub>10</sub>O<sub>6</sub></td> <td align="left">Molecular weight (&amp;lt;500g/mol)<br /><br />LogP (&amp;lt;5)<br /><br />H-bond donor (&amp;lt;5)<br /><br />H-bond acceptor (&amp;lt;10)<br /><br />Violation</td> <td align="center">286.24â€‰g/mol<br /><br />2.08<br /><br />4<br /><br />6<br /><br />0</td> </tr> <tr> <td colspan="2">Isoflavone</td> <td align="center">C<sub>15</sub>H<sub>10</sub>O<sub>2</sub></td> <td align="left">Molecular weight (&amp;lt;500g/mol)<br /><br />LogP (&amp;lt;5)<br /><br />H-bond donor (&amp;lt;5)<br /><br />H-bond acceptor (&amp;lt;10)<br /><br />Violation</td> <td align="center">222.24â€‰g/mol<br /><br />2.51<br /><br />0<br /><br />2<br /><br />0</td> </tr> <tr> <td colspan="2">Remdesivir</td> <td align="center">C<sub>27</sub>H<sub>35</sub>N<sub>6</sub>O<sub>8</sub>P</td> <td align="left">Molecular weight (&amp;lt;500g/mol)<br /><br />LogP (&amp;lt;5)<br /><br />H-bond donor (&amp;lt;5)<br /><br />H-bond acceptor (&amp;lt;10)<br /><br />Violation</td> <td align="center"><b>602.6â€‰g/mol</b><br /><br />3.24<br /><br />4<br /><br /><b>13</b><br /><br /><b>2</b></td> </tr> <tr> <td colspan="2">Ribavirin</td> <td align="center">C<sub>8</sub>H<sub>12</sub>N<sub>4</sub>O<sub>5</sub></td> <td align="left">Molecular weight (&amp;lt;500g/mol)<br /><br />LogP (&amp;lt;5)<br /><br />H-bond donor (&amp;lt;5)<br /><br />H-bond acceptor (&amp;lt;10)<br /><br />Violation</td> <td align="center">244.2â€‰g/mol<br /><br />0.13<br /><br />4<br /><br />7<br /><br />0</td> </tr> </tbody> </table> </div> </div> <div class="oralbioavailabilityandgastrointestinal(gi)absorptionofinteractedplant-derivedcompounds" title="sec"> <div class="title" tagx="title" title="title">
Oral bioavailability and gastrointestinal (GI) absorption of interacted plant-derived compounds</div> <p>The potential inhibitors of enzymes involved in replication and transcription of SARS-CoV-2 studied were analysed for their oral bioavailability as well as GI tract absorption parameters. The potent compounds extracted from various natural sources were listed in the <a href="#t0001">Table S1</a><b>(</b>Supplementary material). Natural sources such as herbs, oils have many active compounds: Curcumin, Quercetin, Bergapten etc. which possess to have potential as anti-SARS-CoV-2 activity. The bioavailability of all the plant-based compounds is equal to 0.55 whereas remdesivir has lower bioavailability of 0.17. The absorption of the compounds by GI tract is high except for myricetin while the drugs have lower absorption rate.</p> </div> <div class="moleculardynamicssimulation" title="sec"> <div class="title" tagx="title" title="title">
Molecular dynamics simulation</div> <p>Molecular dynamics simulation has been preformed for 50â€‰ns simulation time for four systems, two from 3CL<sup>pro</sup> (receptor only and with selected dock ligand complex) and two from endoribonuclease (receptor and docked ligand complex). For each case, receptor only dynamics is used as a reference to assess the stability dynamics of the ligand bound complex. RMSD (<a href="#F0005">Figure 5</a>), RMSF (<a href="#F0006">Figure 6</a>) and SASA (<a href="#F0007">Figure 7</a>) is calculated to assess the time dependent evolution of the systems and stability dynamics. At the same time, Hydrogen-bond analysis has been done to understand the engagement of the docked ligand with the corresponding receptor (<a href="#F0008">Figure 8</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>RMSD plot of the 3CL<sup>pro</sup> system (above) and endoribonuclease system (below). For both cases, protein-ligand complex is showing better stability compared to the receptor alone.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>RMSF analysis plot for 3CL<sup>pro</sup> system (above) and endoribonuclease system (below). As expected, except the binding site residues, nearly similar fluctuations are recorded for the both unbound and bound system. Binding site residues are less fluctuating in ligand complex compared to receptor only dynamics.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p>SASA plot for the 3CL<sup>pro</sup> system (above) and endoribonuclease system (below). For both systems, SASA are nearly consistent over time and reflects that after ligand binding receptor proteins do not undergo any large conformational change.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8.</span></a></div>   <p>Hydrogen bond dynamics between Bisdemethoxycurcumin-3CL<sup>pro</sup> complex (above) and Demethoxycurcumin-endoribonuclease complex (below). This analysis shows that selected ligands engage with receptors with nearly two hydrogen bonds and do not deviate much from the initial structure.</p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The rapidly spreading coronavirus disease SARS-CoV-2, also known as COVID-19, is declared as a pandemic outbreak by the WHO. It has affected the public as well as social health of more than 200 countries around the globe (<a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">https://www.who.int/emergencies/diseases/novel-coronavirus-2019</a>). Hence, itâ€™s the need of an hour that has demanded the scientific community from all over the world to collaborate and make an extraordinary effort for the rapid detection of an effective anti-COVID-19 drug.</p> <p>The NSPs, endoribonuclease (NSP15) and main protease enzyme encoded by 3CL<sup>pro</sup> have been sequenced and their X-ray crystallography 3-D structures are available in PDB databank (Jin etÂ al., <a href="#CIT0023">2020</a>; Kim etÂ al., <a href="#CIT0027">2020</a>) . Each monomeric unit of endoribonuclease protein is comprised of âˆ¼345 amino acids which are folded into three domains: N-terminal, middle and NendoU C-terminal catalytic domain. The NendoU C-terminal catalytic domain reported to possess two Î²-sheets in antiparallel direction (Sinha etÂ al., <a href="#CIT0049">2020</a>). This protein is required by the virus as it takes part in various viral mechanisms including the formation of the replicase and transcriptase complex (RTC) (Deng etÂ al., <a href="#CIT0009">2017</a>). Also, in innate immune response, therefore NSP15 considered to be essential for biological progression of coronavirus. Study has reported that the main active sites of Nsp15 protein, conserved among SARS-CoV-2, SARS-CoV and MERS-CoV are located in a shallow groove between the two Î²-sheets that comprise six amino acids (His235, His250, Lys290, Thr341, Tyr343, and Ser294) (Sinha etÂ al., <a href="#CIT0049">2020</a>).</p> <p>The main protease, 3CL<sup>pro</sup> protein is a homodimer with Cys-His catalytic dyad on the active site which is responsible for the protease activity and hence inhibiting the catalytic site present in enzyme would lead to the inhibition of activity of the protein(Islam etÂ al., <a href="#CIT0022">2020</a>; Lobo-Galo etÂ al., <a href="#CIT0034">2020</a>). The secondary structures of 3CL<sup>pro</sup>, has three domins: Domains I and II which have an antiparallel Î²-barrel structure and Domain III contains five Î±-helices arranged into a largely antiparallel globular cluster, and is connected to domain II by means of a long loop region. These features are similar to those of other M<sub>pro</sub> reported previously (Ren etÂ al., <a href="#CIT0043">2013</a>; Wang etÂ al., <a href="#CIT0054">2016</a>; Xue etÂ al., <a href="#CIT0057">2008</a>).</p> <p>At present, there are no approved medications from different drug administrations around the world for the treatment of novel coronavirus (SARS-CoV-2). Hence, the primary focus has been on clinical management which includes the prevention of infection, control measures and intensive supportive care. Recently, anti-malarial drugs like chloroquine, hydroxychloroquine have given positive results in the cell culture systems but has to proceed for the clinical trials to be established as an effective drug (Enmozhi etÂ al., <a href="#CIT0014">2020</a>). Several other anti-viral drugs such as remdesivir, ribavirin used in hepatitis C infection and Ebola, and many other vaccines are under rapid study to be used against coronavirus infection.</p> <p>An anti-viral activity of curcumin and its analogues demethoxycurcumin and bisdemethoxycurcumin was reported against several different viruses including hepatitis viruses, influenza viruses and emerging arboviruses like the Zika virus (ZIKV) (Mounce etÂ al., <a href="#CIT0037">2017</a>) or chikungunya virus (CHIKV) (von Rhein etÂ al., <a href="#CIT0053">2016</a>). These analogues have described role as a metabolite, an anti-neoplastic agent and an anti-inflammatory agent in PubChem databank. They possess much greater chemical stability at physiological pH. Scutellarin is a glycosyloxyflavone, a glucosiduronic acid, a trihydroxyflavone and a monosaccharide derivative. (Cheng etÂ al., <a href="#CIT0008">2013</a>) observed that it could inactivate porcine reproductive and respiratory syndrome virus (PRRSV) and restrict the early stage of PRRSV replication under <i>inÂ vitro</i> conditions. Similarly, Zhang etÂ al.(2005) demonstrated that it possessed anti-HIV-1 activity by inhibiting HIV-1 reverse transcriptase activity, HIV-1 particle attachment and cell fusion (Cheng etÂ al., <a href="#CIT0008">2013</a>; Sun etÂ al., <a href="#CIT0050">2015</a>). Quercetin is a polyphenol, known as flavonoids, its various pharmacological activities described in the literature such as anti-inflammatory, anti-oxidant, anti-enzymatic activities. (Polansky &amp;amp; Lori, <a href="#CIT0041">2020</a>) concluded that quercetin and its derivatives were reported to inhibit the SARS and MERS virus replication via protease inhibition (Cheng etÂ al., <a href="#CIT0008">2013</a>). Myricetin is structurally related to several well-known phenolic compounds namely quercetin, morin, kaempferol and fisetin. Myricetin demonstrated antiviral activity as a strong inhibitor of reverse transcriptase against Rauscher murine leukemia virus (RLV) and human immunodeficiency virus (HIV). Some studies reported that bergapten exhibited potent anti-HIV activity via inhibition of HIV-1 replication with therapeutic index value greater than 5.0 (Shikishima etÂ al., <a href="#CIT0048">2001</a>). According to literature, it has been reported that 5,7,3â€²,4â€²-tetrahydroxy-2â€²-(3,3-dimethylallyl) isoflavone used as an anti-leishmanial agent (Salem &amp;amp; Werbovetz, <a href="#CIT0046">2006</a>), and it is also described in traditional Chinese medicine records (Zhou etÂ al., <a href="#CIT0059">2011</a>).</p> <p>The preliminary screening of potential molecules having anti-viral activity via computational approach would help in providing the fast selection of promising compounds towards the development of therapeutics against SARS- CoV-2. To elucidate the binding affinity of plant-derived compounds with known properties of anti-viral, were docked against two NSPs of COVID-19 using AutoDock4 software (<a href="#t0002">Table 2</a>; <a href="#F0001">Figure 1</a> and 2). The binding scores from the docking study revealed the affinity of a specific ligand and strength by which a compound interacts with and binds to the pocket of a target protein. Docking results were successful with least binding scores and presence of significant hydrogen bonding with demethoxycurcumin, bisdemethoxycurcumin and a flavone scutellarin. Compounds having binding affinity of âˆ’6.5â€‰Kcal/mol or less are considered to be a good inhibitor of enzymatic activities (Shah etÂ al., <a href="#CIT0047">2020</a>). In the present study, five plant-derived compounds demonstrated binding affinity more negative to âˆ’6.5Kcal/mol with 3CL<sup>pro</sup>, whereas with NSP15 endoribonuclease, five different plant-derived compounds exhibited the same results. These compounds could be potent drugs for inhibition of two essential viral proteins and thus inhibiting the viral replication in the infected cells. Additionally, the <i>in-silico</i> experiment represented the inability of remdesivir to bind with 3CL<sup>pro</sup>, as binding affinity of remdesivir was +196â€‰Kcal/mol, which showed instability. In contrast, Remdesivir contributes âˆ’6.58â€‰kcal/mol to the binding free energy at the electrostatic S1 subsite of protease protein (Khan etÂ al., <a href="#CIT0026">2020</a>).</p> <p>Further, the physicochemical properties of these compounds were studied by ADME-Tox following the Lipinskiâ€™s rule of five for evaluating drug likeness. Lipinskiâ€™s rule determines the molecular properties which are important for a drugâ€™s pharmacokinetics in the human body such as absorption, distribution, metabolism, and excretion (ADME). Lipinskiâ€™s rule of five criteria for an ideal drug are (i) a molecular mass less than 500 Daltons (ii) no more than 5 hydrogen bond donors, (iii) no more than 10 hydrogen bond acceptors, (iv) an octanolâ€"water partition coefficient log P not greater than 5. Three or more than 3 violations of Lipinskiâ€™s rule do not fit into the criteria of drug likeliness via oral route administration (Benet etÂ al., <a href="#CIT0003">2016</a>). ADME-Tox studies of docked eight plant-derived compounds showed that all the compounds had virtual hits and were successful at passing through these ADME-Tox test filters, stating that they could be potential drugs which can be easily administered orally (<a href="#t0004">Table 4</a>).</p> <p>The other crucial experiments like bioavailability and GI absorptions are of detecting the low bioavailability is a major concern for drug design, as it may lead to failure of a novel drug in the clinical trials, in spite of the fact that the compounds have high efficacy in <i>in-vitro</i> and/or <i>in-vivo</i> tests (Gyebi etÂ al., <a href="#CIT0019">2020</a>). Therefore, oral bioavailability of a compound is essential to be taken into account when predicting the compound as a drug candidate. The compounds in the present study exhibited bioavailability of 0.55 with low as well as high GI absorption. The absorption into GI was found to be at a high rate in all compounds except that of Myricetin.</p> <p>Protein flexibility plays an essential role in the inhibitor designing process (Teague, <a href="#CIT0051">2003</a>) and as molecular docking has been performed using the single structure, molecular dynamic simulation can provide an in-depth understanding of the protein-ligand interaction information. 3CL<sup>pro</sup> and endoribonuclease proteins, along with selected compounds, were used for MDS analysis, which suggests that the ligand-bound receptor adopt stability over the receptor only dynamics (<a href="#F0005">Figure 5</a> and <a href="#F0007">7</a>). H-bond based interaction analysis suggests that both docked compounds with respective proteins utilize the available binding interactions and maintain these interactions throughout the simulation time (<a href="#F0008">Figure 8</a>). Apart from the receptor-based interaction, docked ligand also formed on average two H-bonds with the nearly water molecules which additionally provide the stability to the docked ligand (Supplementary material <a href="#F0003">Figure S3</a>).</p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>In the current scenario, there is an urgent requirement of the potential treatment of SARS-CoV-2 infection. Compounds extracted from natural sources might be a source in the development of new potent drugs which can improve the novel coronavirus treatment. Various plant-derived compounds have been reported for their anti-viral activity, may found to be effective against SARS-CoV-2. The bioinformatics approach could be a very useful and efficient tool to identify potent inhibitors against the specific targets of novel coronavirus. The docking study concluded that plant-derived compounds (Demethoxycurcumin, Bisdemethoxycurcumin, Scutellarin, Quercetin and Myricetin) exhibited potential drug candidates against two enzymes, main protease (3CL<sup>pro</sup>) and endoribonuclease (NSP15) proteins of SARS-CoV-2. Also, molecular dynamics and simulations of two best docked protein-ligand structures revealed the dynamics information of their stability in the biological system. Instability of these two enzyme activities would lead to inhibition of viral replication, which in response would restrict the infection in the host cell.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="TBSD_A_1808077_SM7578.jpg">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary_document......docx</div>  <div class="media" title="media"><a href="TBSD_A_1808077_SM2135.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors would like to thank Amity University Uttar Pradesh, India for providing facilities to carry out this work. The authors also would like to thank the researchers who generated and shared the sequencing and protein data of main protease (3CL<sup>pro</sup>) and endoribonuclease (NSP15) proteins of SARS-CoV-2 in Protein Data Bank website (<a href="https://www.rcsb.org/">https://www.rcsb.org/</a>). The authors would like to acknowledge Ms. Sapna Ratnakar, Ph.D. Scholar, School of Life Sciences, JNU for providing her support to carry out MDS. P.K. would like to thank DBT Project for supporting the research work and also thank Dr. Debabsisa Mohanty for providing the NII computational facility to carry out the simulation analysis.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>The authors declare no potential conflicts of interest to disclose.</p> </div> <div class="authorscontributions" title="sec"> <div class="title" tagx="title" title="title">
Authors contributions</div> <p>All authors conceived, conducted and analyzed the results of experiment(s) equally. L.G. performed docking studies, manuscript editing; A.S. and S.G. drafted manuscript, performed ADMET studies; A.K.Y. manuscript editing, result analysis and P.K. performed molecular dynamics and simulations.</p> </div> <div class="correction-statement" title="correction-statement"> <div class="title" tagx="title" title="title">
Correction Statement</div> <p>This article has been republished with minor changes. These changes do not impact the academic content of the article.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akula</span><span class="given-names" tagx="given-names" title="given-names">S. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurley</span><span class="given-names" tagx="given-names" title="given-names">D. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wixon</span><span class="given-names" tagx="given-names" title="given-names">R. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chase</span><span class="given-names" tagx="given-names" title="given-names">C. C. L.</span></span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Effect of genistein on replication of bovine herpesvirus type 1</span>. <span class="source" tagx="source" title="source"><i>American Journal of Veterinary Research</i></span>, (<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">1124</span>â€"<span class="lpage" tagx="lpage" title="lpage">1128</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2460/ajvr.2002.63.1124">10.2460/ajvr.2002.63.1124</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12171165">12171165</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baikerikar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Curcumin and natural derivatives inhibit Ebola viral proteins: An In silico approach</span>. <span class="source" tagx="source" title="source"><i>Pharmacognosy Research</i></span>, (<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">15</span><span class="pub-id"><a href="https://dx.doi.org/10.4103/pr.pr_30_17">10.4103/pr.pr_30_17</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benet</span><span class="given-names" tagx="given-names" title="given-names">L. Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hosey</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ursu</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oprea</span><span class="given-names" tagx="given-names" title="given-names">T. I.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">BDDCS, the Rule of 5 and drugability</span>. <span class="source" tagx="source" title="source"><i>Advanced Drug Delivery Reviews</i></span>, , <span class="fpage" tagx="fpage" title="fpage">89</span>â€"<span class="lpage" tagx="lpage" title="lpage">98</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.addr.2016.05.007">10.1016/j.addr.2016.05.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27182629">27182629</a></span></span></li> <li tag="ref"><a name="CIT0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boopathi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poma</span><span class="given-names" tagx="given-names" title="given-names">A. B.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolandaivel</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Novel 2019 coronavirus structure, mechanism of action, antiviral drug promises and rule out against its treatment</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, ,<span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">10</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758788">10.1080/07391102.2020.1758788</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">J. F. W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kok</span><span class="given-names" tagx="given-names" title="given-names">K. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">To</span><span class="given-names" tagx="given-names" title="given-names">K. K. W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">K. Y.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan</span>. <span class="source" tagx="source" title="source"><i>Emerging Microbes &amp;amp; Infections</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">221</span>â€"<span class="lpage" tagx="lpage" title="lpage">236</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/22221751.2020.1719902">10.1080/22221751.2020.1719902</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31987001">31987001</a></span></span></li> <li tag="ref"><a name="CIT0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">J. F.-W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kok</span><span class="given-names" tagx="given-names" title="given-names">K.-H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">To</span><span class="given-names" tagx="given-names" title="given-names">K. K.-W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yip</span><span class="given-names" tagx="given-names" title="given-names">C. C.-Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">R. W.-S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsoi</span><span class="given-names" tagx="given-names" title="given-names">H.-W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo</span><span class="given-names" tagx="given-names" title="given-names">S. K.-F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">K.-H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">V. K.-M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">W.-M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ip</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">J.-P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">V. C.-C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">C. K.-M.</span></span>, â€¦ <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">K.-Y.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: A study of a family cluster</span>. <span class="source" tagx="source" title="source"><i>The Lancet</i></span>, (<span class="issue" tagx="issue" title="issue">10223</span>), <span class="fpage" tagx="fpage" title="fpage">514</span>â€"<span class="lpage" tagx="lpage" title="lpage">523</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30154-9">10.1016/S0140-6736(20)30154-9</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandel</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raj</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rathi</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">In silico identification of potent COVID-19 main protease inhibitors from FDA approved antiviral compounds and active phytochemicals through molecular docking: A drug repurposing approach</span>. <i>7</i>(3), 1-9.<span class="pub-id"><a href="https://dx.doi.org/10.20944/preprints202003.0349.v1">10.20944/preprints202003.0349.v1</a></span></span></li> <li tag="ref"><a name="CIT0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">In vitro screening for compounds derived from traditional Chinese medicines with antiviral activities against porcine reproductive and respiratory syndrome virus</span>. <span class="source" tagx="source" title="source"><i>Journal of Microbiology and Biotechnology</i></span>, (<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">1076</span>â€"<span class="lpage" tagx="lpage" title="lpage">1083</span>. <span class="pub-id"><a href="https://dx.doi.org/10.4014/jmb.1303.03074">10.4014/jmb.1303.03074</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23727804">23727804</a></span></span></li> <li tag="ref"><a name="CIT0009" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hackbart</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mettelman</span><span class="given-names" tagx="given-names" title="given-names">R. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mielech</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kao</span><span class="given-names" tagx="given-names" title="given-names">C. C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">S. C.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Coronavirus nonstructural protein 15 mediates evasion of dsRNA sensors and limits apoptosis in macrophages</span>. <span class="source" tagx="source" title="source"><i>Proceedings of the National Academy of Sciences of the United States of America</i></span>, (<span class="issue" tagx="issue" title="issue">21</span>), <span class="fpage" tagx="fpage" title="fpage">E4251</span>â€"<span class="lpage" tagx="lpage" title="lpage">E4260</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1618310114">10.1073/pnas.1618310114</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28484023">28484023</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhama</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tiwari</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sircar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhat</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malik</span><span class="given-names" tagx="given-names" title="given-names">Y. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">K. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaicumpa</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonilla-Aldana</span><span class="given-names" tagx="given-names" title="given-names">D. K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez-Morales</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Coronavirus Disease 2019â€"COVID-19</span>. <span class="source" tagx="source" title="source"><i>Clinical Microbiology Reviews</i></span>, (<span class="issue" tagx="issue" title="issue">4</span>), e00028â€"20. <span class="pub-id"><a href="https://dx.doi.org/10.1128/CMR.00028-20">10.1128/CMR.00028-20</a></span></span></li> <li tag="ref"><a name="CIT0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DormÃ¡n</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flachner</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HajdÃº</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AndrÃ¡s</span><span class="given-names" tagx="given-names" title="given-names">C. D.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Target Identification and Polypharmacology of Nutraceuticals</span> In <span class="source" tagx="source" title="source"><i>Nutraceuticals</i></span> (pp. <span class="fpage" tagx="fpage" title="fpage">263</span>â€"<span class="lpage" tagx="lpage" title="lpage">286</span>). <span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/B978-0-12-802147-7.00021-8">10.1016/B978-0-12-802147-7.00021-8</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Natural products may interfere with SARS-CoV-2 attachment to the host cell</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, , <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">10</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1761881">10.1080/07391102.2020.1761881</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elmezayen</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Obaidi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Åžahin</span><span class="given-names" tagx="given-names" title="given-names">A. T.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">YelekÃ§i</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Drug repurposing for coronavirus (COVID-19): In silico screening of known drugs against coronavirus 3CL hydrolase and protease enzymes</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, , <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758791">10.1080/07391102.2020.1758791</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enmozhi</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raja</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sebastine</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joseph</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Andrographolide as a potential inhibitor of SARS-CoV-2 main protease: An in silico approach</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, , <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1760136">10.1080/07391102.2020.1760136</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Essmann</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perera</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berkowitz</span><span class="given-names" tagx="given-names" title="given-names">M. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darden</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pedersen</span><span class="given-names" tagx="given-names" title="given-names">L. G.</span></span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">A smooth particle mesh Ewald method</span>. <span class="source" tagx="source" title="source"><i>The Journal of Chemical Physics</i></span>, (<span class="issue" tagx="issue" title="issue">19</span>), <span class="fpage" tagx="fpage" title="fpage">8577</span>â€"<span class="lpage" tagx="lpage" title="lpage">8593</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1063/1.470117">10.1063/1.470117</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gajula</span><span class="given-names" tagx="given-names" title="given-names">M. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ijaq</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Protocol for molecular dynamics simulations of proteins</span>. <span class="source" tagx="source" title="source">Bio-Protocol</span>, (<span class="issue" tagx="issue" title="issue">23</span>), <span class="fpage" tagx="fpage" title="fpage">e2051</span>â€"<span class="lpage" tagx="lpage" title="lpage">e2051</span>. <span class="pub-id"><a href="https://dx.doi.org/10.21769/BioProtoc.2051">10.21769/BioProtoc.2051</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geourjon</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DelÃ©age</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">SOPMA: Significant improvements in protein secondary structure prediction by consensus prediction from multiple alignments</span>. <span class="source" tagx="source" title="source"><i>Computer Applications in the Biosciences: Cabios</i></span>, (<span class="issue" tagx="issue" title="issue">6</span>), <span class="fpage" tagx="fpage" title="fpage">681</span>â€"<span class="lpage" tagx="lpage" title="lpage">684</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/11.6.681">10.1093/bioinformatics/11.6.681</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8808585">8808585</a></span></span></li> <li tag="ref"><a name="CIT0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Q. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">Z. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">Y. Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">S. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">H. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">K. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D. Y.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">The origin, transmission and clinical therapies on coronavirus disease 2019 (COVID-19) outbreak - an update on the status</span>. <span class="source" tagx="source" title="source"><i>Military Medical Research</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">11</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s40779-020-00240-0">10.1186/s40779-020-00240-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32169119">32169119</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gyebi</span><span class="given-names" tagx="given-names" title="given-names">G. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogunro</span><span class="given-names" tagx="given-names" title="given-names">O. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adegunloye</span><span class="given-names" tagx="given-names" title="given-names">A. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogunyemi</span><span class="given-names" tagx="given-names" title="given-names">O. M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Afolabi</span><span class="given-names" tagx="given-names" title="given-names">S. O.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Potential inhibitors of coronavirus 3-chymotrypsin-like protease (3CLpro): An in silico screening of Alkaloids and Terpenoids from African medicinal plants</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, , <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">19</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1764868">10.1080/07391102.2020.1764868</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hackbart</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">S. C.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Coronavirus endoribonuclease targets viral polyuridine sequences to evade activating host sensors</span>. <span class="source" tagx="source" title="source"><i>Proceedings of the National Academy of Sciences of the United States of America</i></span>, (<span class="issue" tagx="issue" title="issue">14</span>), <span class="fpage" tagx="fpage" title="fpage">8094</span>â€"<span class="lpage" tagx="lpage" title="lpage">8103</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1921485117">10.1073/pnas.1921485117</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32198201">32198201</a></span></span></li> <li tag="ref"><a name="CIT0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoda</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agarwal</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raj</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vermani</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijayaraghavan</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Inhibition of aspergillus fumigatus biofilm and cytotoxicity study of natural compound cis-9-hexadecenal</span>. <span class="source" tagx="source" title="source"><i>Journal of Pure and Applied Microbiology</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">1207</span>â€"<span class="lpage" tagx="lpage" title="lpage">1216</span>. <span class="pub-id"><a href="https://dx.doi.org/10.22207/JPAM.13.2.61">10.22207/JPAM.13.2.61</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parves</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">A. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uddin</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mamun</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">M. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halim</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">A molecular modeling approach to identify effective antiviral phytochemicals against the main protease of SARS-CoV-2</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, , <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1761883">10.1080/07391102.2020.1761883</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">You</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, â€¦ <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span>. <span class="source" tagx="source" title="source">Nature</span>, (<span class="issue" tagx="issue" title="issue">7811</span>), <span class="fpage" tagx="fpage" title="fpage">289</span>â€"<span class="lpage" tagx="lpage" title="lpage">293</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2223-y">10.1038/s41586-020-2223-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32272481">32272481</a></span></span></li> <li tag="ref"><a name="CIT0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jorgensen</span><span class="given-names" tagx="given-names" title="given-names">W. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chandrasekhar</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madura</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Impey</span><span class="given-names" tagx="given-names" title="given-names">R. W.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klein</span><span class="given-names" tagx="given-names" title="given-names">M. L.</span></span></span> (<span class="year" tagx="year" title="year">1983</span>). <span class="mixed-article-title" title="mixed-article-title">Comparison of simple potential functions for simulating liquid water</span>. <span class="source" tagx="source" title="source"><i>The Journal of Chemical Physics</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">926</span>â€"<span class="lpage" tagx="lpage" title="lpage">935</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1063/1.445869">10.1063/1.445869</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joshi</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jagdale</span><span class="given-names" tagx="given-names" title="given-names">S. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bansode</span><span class="given-names" tagx="given-names" title="given-names">S. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shankar</span><span class="given-names" tagx="given-names" title="given-names">S. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tellis</span><span class="given-names" tagx="given-names" title="given-names">M. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandya</span><span class="given-names" tagx="given-names" title="given-names">V. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chugh</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giri</span><span class="given-names" tagx="given-names" title="given-names">A. P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulkarni</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Discovery of potential multi-target-directed ligands by targeting host-specific SARS-CoV-2 structurally conserved main protease</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, , <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">16</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1760137">10.1080/07391102.2020.1760137</a></span></span></li> <li tag="ref"><a name="CIT0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zia</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashraf</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uddin</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ul-Haq</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of chymotrypsin-like protease inhibitors of SARS-CoV-2 via integrated computational approach</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, , <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">10</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1751298">10.1080/07391102.2020.1751298</a></span></span></li> <li tag="ref"><a name="CIT0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jedrzejczak</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maltseva</span><span class="given-names" tagx="given-names" title="given-names">N. I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilamowski</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endres</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Godzik</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michalska</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joachimiak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</span>. <span class="source" tagx="source" title="source"><i>Protein Science</i></span>, (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">1596</span>â€"<span class="lpage" tagx="lpage" title="lpage">1605</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/pro.3873">10.1002/pro.3873</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32304108">32304108</a></span></span></li> <li tag="ref"><a name="CIT0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jedrzejczak</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maltseva</span><span class="given-names" tagx="given-names" title="given-names">N. I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilamowski</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endres</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Godzik</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michalska</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joachimiak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</span><span class="source" tagx="source" title="source">. <i>Protein Sci</i></span>, (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">1596</span>â€"<span class="lpage" tagx="lpage" title="lpage">1605</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/pro.3873">10.1002/pro.3873</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32304108">32304108</a></span></span></li> <li tag="ref"><a name="CIT0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitazato</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Viral infectious disease and natural products with antiviral activity</span>. <span class="source" tagx="source" title="source"><i>Drug Discoveries &amp;amp; Therapeutics</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">14</span>â€"<span class="lpage" tagx="lpage" title="lpage">22</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22504360">22504360</a></span></span></li> <li tag="ref"><a name="CIT0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kutluay</span><span class="given-names" tagx="given-names" title="given-names">S. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doroghazi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roemer</span><span class="given-names" tagx="given-names" title="given-names">M. E.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Triezenberg</span><span class="given-names" tagx="given-names" title="given-names">S. J.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Curcumin inhibits herpes simplex virus immediate-early gene expression by a mechanism independent of p300/CBP histone acetyltransferase activity</span>. <span class="source" tagx="source" title="source">Virology</span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">239</span>â€"<span class="lpage" tagx="lpage" title="lpage">247</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2007.11.028">10.1016/j.virol.2007.11.028</a></span></span></li> <li tag="ref"><a name="CIT0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">K. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">L. F. P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">D. X.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of a novel cleavage activity of the first papain-like proteinase domain encoded by open reading frame 1a of the coronavirus avian infectious bronchitis virus and characterization of the cleavage products</span>. <span class="source" tagx="source" title="source"><i>Journal of Virology</i></span>, (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">1674</span>â€"<span class="lpage" tagx="lpage" title="lpage">1685</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/jvi.74.4.1674-1685.2000">10.1128/jvi.74.4.1674-1685.2000</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10644337">10644337</a></span></span></li> <li tag="ref"><a name="CIT0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garner</span><span class="given-names" tagx="given-names" title="given-names">L. V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watkins</span><span class="given-names" tagx="given-names" title="given-names">S. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carter</span><span class="given-names" tagx="given-names" title="given-names">L. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smoot</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gregg</span><span class="given-names" tagx="given-names" title="given-names">A. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daniels</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jervey</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albaiu</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Research and development on therapeutic agents and vaccines for COVID-19 and related human coronavirus diseases</span>. <span class="source" tagx="source" title="source"><i>ACS Central Science</i></span>, (<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">315</span>â€"<span class="lpage" tagx="lpage" title="lpage">331</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acscentsci.0c00272">10.1021/acscentsci.0c00272</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32226821">32226821</a></span></span></li> <li tag="ref"><a name="CIT0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Porcine deltacoronavirus nsp15 antagonizes interferon-Î² production independently of its endoribonuclease activity</span>. <span class="source" tagx="source" title="source"><i>Molecular Immunology</i></span>, , <span class="fpage" tagx="fpage" title="fpage">100</span>â€"<span class="lpage" tagx="lpage" title="lpage">107</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molimm.2019.07.003">10.1016/j.molimm.2019.07.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31351410">31351410</a></span></span></li> <li tag="ref"><a name="CIT0034" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lobo-Galo</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Terrazas-LÃ³pez</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MartÃ­nez-MartÃ­nez</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DÃ­az-SÃ¡nchez</span><span class="given-names" tagx="given-names" title="given-names">Ã�. G.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">FDA-approved thiol-reacting drugs that potentially bind into the SARS-CoV-2 main protease, essential for viral replication</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>,Â <i>38,</i><span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">12</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1764393">10.1080/07391102.2020.1764393</a></span></span></li> <li tag="ref"><a name="CIT0035" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maier</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinez</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasavajhala</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wickstrom</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauser</span><span class="given-names" tagx="given-names" title="given-names">K. E.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmerling</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">ff14SB: Improving the accuracy of protein side chain and backbone parameters from ff99SB</span>. <span class="source" tagx="source" title="source"><i>Journal of Chemical Theory and Computation</i></span>, (<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">3696</span>â€"<span class="lpage" tagx="lpage" title="lpage">3713</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jctc.5b00255">10.1021/acs.jctc.5b00255</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26574453">26574453</a></span></span></li> <li tag="ref"><a name="CIT0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huey</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindstrom</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanner</span><span class="given-names" tagx="given-names" title="given-names">M. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belew</span><span class="given-names" tagx="given-names" title="given-names">R. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodsell</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">16</span>), <span class="fpage" tagx="fpage" title="fpage">2785</span>â€"<span class="lpage" tagx="lpage" title="lpage">2791</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.21256">10.1002/jcc.21256</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19399780">19399780</a></span></span></li> <li tag="ref"><a name="CIT0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mounce</span><span class="given-names" tagx="given-names" title="given-names">B. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cesaro</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrau</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vallet</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vignuzzi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Curcumin inhibits Zika and chikungunya virus infection by inhibiting cell binding</span>. <span class="source" tagx="source" title="source"><i>Antiviral Research</i></span>, , <span class="fpage" tagx="fpage" title="fpage">148</span>â€"<span class="lpage" tagx="lpage" title="lpage">157</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2017.03.014">10.1016/j.antiviral.2017.03.014</a></span></span></li> <li tag="ref"><a name="CIT0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muralidharan</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakthivel</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velmurugan</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gromiha</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Computational studies of drug repurposing and synergism of lopinavir, oseltamivir and ritonavir binding with SARS-CoV-2 protease against COVID-19</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <i>38,</i>Â <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1752802">10.1080/07391102.2020.1752802</a></span></span></li> <li tag="ref"><a name="CIT0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oâ€™Boyle</span><span class="given-names" tagx="given-names" title="given-names">N. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banck</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">C. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morley</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandermeersch</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hutchison</span><span class="given-names" tagx="given-names" title="given-names">G. R.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Open Babel: An open chemical toolbox</span>. <span class="source" tagx="source" title="source"><i>Journal of Cheminformatics</i></span>, , <span class="fpage" tagx="fpage" title="fpage">33</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1758-">10.1186/1758-</a></span>2946-3-33<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21982300">21982300</a></span></span></li> <li tag="ref"><a name="CIT0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pettersen</span><span class="given-names" tagx="given-names" title="given-names">E. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goddard</span><span class="given-names" tagx="given-names" title="given-names">T. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Couch</span><span class="given-names" tagx="given-names" title="given-names">G. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenblatt</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferrin</span><span class="given-names" tagx="given-names" title="given-names">T. E.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">UCSF Chimera-a visualization system for exploratory research and analysis</span>. <span class="source" tagx="source" title="source"><i>J Comput Chem</i></span>, (<span class="issue" tagx="issue" title="issue">13</span>), <span class="fpage" tagx="fpage" title="fpage">1605</span>â€"<span class="lpage" tagx="lpage" title="lpage">1612</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.20084">10.1002/jcc.20084</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15264254">15264254</a></span></span></li> <li tag="ref"><a name="CIT0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Polansky</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lori</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Coronavirus disease 2019 (COVID-19): first indication of efficacy of Gene-Eden-VIR/Novirin in SARS-CoV-2 infection</span>. <span class="source" tagx="source" title="source"><i>International Journal of Antimicrobial Agents</i></span>, (<span class="issue" tagx="issue" title="issue">6</span>), <span class="fpage" tagx="fpage" title="fpage">105971</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijantimicag.2020.105971">10.1016/j.ijantimicag.2020.105971</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32283177">32283177</a></span></span></li> <li tag="ref"><a name="CIT0042" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajtar</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skalicka-WoÅºniak</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ÅšwiÄ…tek</span><span class="given-names" tagx="given-names" title="given-names">Å�.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stec</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boguszewska</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Polz-Dacewicz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Antiviral effect of compounds derived from Angelica archangelica L. on Herpes simplex virus-1 and Coxsackievirus B3 infections</span>. <span class="source" tagx="source" title="source"><i>Food and Chemical Toxicology: An International Journal Published for the British Industrial Biological Research Association</i></span>, (<span class="issue" tagx="issue" title="issue">Pt 2</span>), <span class="fpage" tagx="fpage" title="fpage">1026</span>â€"<span class="lpage" tagx="lpage" title="lpage">1031</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.fct.2017.05.011">10.1016/j.fct.2017.05.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28487231">28487231</a></span></span></li> <li tag="ref"><a name="CIT0043" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lou</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">The newly emerged SARS-Like coronavirus HCoV-EMC also has an â€œAchilles' heelâ€�: current effective inhibitor targeting a 3C-like protease</span>. <span class="source" tagx="source" title="source"><i>Protein &amp;amp; Cell</i></span>, (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">248</span>â€"<span class="lpage" tagx="lpage" title="lpage">250</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s13238-013-2841-3">10.1007/s13238-013-2841-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23549610">23549610</a></span></span></li> <li tag="ref"><a name="CIT0044" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roe</span><span class="given-names" tagx="given-names" title="given-names">D. R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheatham</span><span class="given-names" tagx="given-names" title="given-names">T. E.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">PTRAJ and CPPTRAJ: Software for processing and analysis of molecular dynamics trajectory data</span>. <span class="source" tagx="source" title="source"><i>Journal of Chemical Theory and Computation</i></span>, (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">3084</span>â€"<span class="lpage" tagx="lpage" title="lpage">3095</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ct400341p">10.1021/ct400341p</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26583988">26583988</a></span></span></li> <li tag="ref"><a name="CIT0045" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryckaert</span><span class="given-names" tagx="given-names" title="given-names">J. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciccotti</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berendsen</span><span class="given-names" tagx="given-names" title="given-names">H. J. C.</span></span></span> (<span class="year" tagx="year" title="year">1977</span>). <span class="mixed-article-title" title="mixed-article-title">Numerical integration of the cartesian equations of motion of a system with constraints: Molecular dynamics of n-alkanes</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Physics</i></span>, (<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">327</span>â€"<span class="lpage" tagx="lpage" title="lpage">341</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0021-9991(77)90098-5">10.1016/0021-9991(77)90098-5</a></span></span></li> <li tag="ref"><a name="CIT0046" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salem</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Werbovetz</span><span class="given-names" tagx="given-names" title="given-names">K. A.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Isoflavonoids and other compounds from Psorothamnus arborescens with antiprotozoal activities</span>. <span class="source" tagx="source" title="source"><i>Journal of Natural Products</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">43</span>â€"<span class="lpage" tagx="lpage" title="lpage">49</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/np0502600">10.1021/np0502600</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16441066">16441066</a></span></span></li> <li tag="ref"><a name="CIT0047" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Modi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sagar</span><span class="given-names" tagx="given-names" title="given-names">S. R.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">In silico studies on therapeutic agents for COVID-19: Drug repurposing approach</span>. <span class="source" tagx="source" title="source"><i>Life Sciences</i></span>, , <span class="fpage" tagx="fpage" title="fpage">117652</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.lfs.2020.117652">10.1016/j.lfs.2020.117652</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32278693">32278693</a></span></span></li> <li tag="ref"><a name="CIT0048" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shikishima</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takaishi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Honda</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ito</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takfda</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kodzhimatov</span><span class="given-names" tagx="given-names" title="given-names">O. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashurmetov</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">K. H.</span></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Chemical constituents of Prangos tschiniganica; structure elucidation and absolute configuration of coumarin and furanocoumarin derivatives with anti-HIV activity</span><span class="source" tagx="source" title="source">. Chem. Pharm. Bull</span>, (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">877</span>â€"<span class="lpage" tagx="lpage" title="lpage">880</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1248/cpb.49.877">10.1248/cpb.49.877</a></span></span></li> <li tag="ref"><a name="CIT0049" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinha</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shakya</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prasad</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurav</span><span class="given-names" tagx="given-names" title="given-names">N. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prasad</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurav</span><span class="given-names" tagx="given-names" title="given-names">S. S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">An <i>in-silico</i> evaluation of different Saikosaponins for their potency against SARS-CoV-2 using NSP15 and fusion spike glycoprotein as targets</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, , <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1762741">10.1080/07391102.2020.1762741</a></span></span></li> <li tag="ref"><a name="CIT0050" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J. X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y. G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">J. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">Z. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">H. M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H. Q.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Antiviral activities of natural compounds derived from traditional chinese medicines against porcine circovirus type 2 (PCV2)</span>. <span class="source" tagx="source" title="source"><i>Biotechnology and Bioprocess Engineering</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">180</span>â€"<span class="lpage" tagx="lpage" title="lpage">187</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s12257-014-0520-8">10.1007/s12257-014-0520-8</a></span></span></li> <li tag="ref"><a name="CIT0051" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teague</span><span class="given-names" tagx="given-names" title="given-names">S. J.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Implications of protein flexibility for drug discovery</span>. <span class="source" tagx="source" title="source"><i>Nature Reviews. Drug Discovery</i></span>, (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">527</span>â€"<span class="lpage" tagx="lpage" title="lpage">541</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd1129">10.1038/nrd1129</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12838268">12838268</a></span></span></li> <li tag="ref"><a name="CIT0052" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veckenstedt</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ¼ttner</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ©lÃ¡di</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span> (<span class="year" tagx="year" title="year">1987</span>). <span class="mixed-article-title" title="mixed-article-title">Synergistic action of quercetin and murine alpha/beta interferon in the treatment of Mengo virus infection in mice</span>. <span class="source" tagx="source" title="source"><i>Antiviral Research</i></span>, (<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">169</span>â€"<span class="lpage" tagx="lpage" title="lpage">178</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0166-3542(87)90005-2">10.1016/0166-3542(87)90005-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3038013">3038013</a></span></span></li> <li tag="ref"><a name="CIT0053" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Rhein</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weidner</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HenÃŸ</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weber</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sliva</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnierle</span><span class="given-names" tagx="given-names" title="given-names">B. S.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Curcumin and Boswellia serrata gum resin extract inhibit chikungunya and vesicular stomatitis virus infections inÂ vitro</span>. <span class="source" tagx="source" title="source"><i>Antiviral Research</i></span>, , <span class="fpage" tagx="fpage" title="fpage">51</span>â€"<span class="lpage" tagx="lpage" title="lpage">57</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2015.11.007">10.1016/j.antiviral.2015.11.007</a></span></span></li> <li tag="ref"><a name="CIT0054" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Structure of Main Protease from Human Coronavirus NL63: Insights for Wide Spectrum Anti-Coronavirus Drug Design</span>. <span class="source" tagx="source" title="source"><i>Scientific Reports</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">22677</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep22677">10.1038/srep22677</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26948040">26948040</a></span></span></li> <li tag="ref"><a name="CIT0055" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollman</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span>. <span class="source" tagx="source" title="source"><i>Journal of Molecular Graphics &amp;amp; Modelling</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">247</span>â€"<span class="lpage" tagx="lpage" title="lpage">260</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jmgm.2005.12.005">10.1016/j.jmgm.2005.12.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16458552">16458552</a></span></span></li> <li tag="ref"><a name="CIT0056" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolf</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caldwell</span><span class="given-names" tagx="given-names" title="given-names">J. W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollman</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Development and testing of a general amber force field</span>. <span class="source" tagx="source" title="source">J Comput Chem</span>, (<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">1157</span>â€"<span class="lpage" tagx="lpage" title="lpage">1174</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.20035">10.1002/jcc.20035</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15116359">15116359</a></span></span></li> <li tag="ref"><a name="CIT0057" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartlam</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Structures of two coronavirus main proteases: Implications for substrate binding and antiviral drug design</span>. <span class="source" tagx="source" title="source"><i>Journal of Virology</i></span>, (<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">2515</span>â€"<span class="lpage" tagx="lpage" title="lpage">2527</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02114-07">10.1128/JVI.02114-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18094151">18094151</a></span></span></li> <li tag="ref"><a name="CIT0058" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chin</span><span class="given-names" tagx="given-names" title="given-names">Y. W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jee</span><span class="given-names" tagx="given-names" title="given-names">J. G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keum</span><span class="given-names" tagx="given-names" title="given-names">Y. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeong</span><span class="given-names" tagx="given-names" title="given-names">Y. J.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of myricetin and scutellarein as novel chemical inhibitors of the SARS coronavirus helicase, nsP13</span>. <span class="source" tagx="source" title="source"><i>Bioorganic &amp;amp; Medicinal Chemistry Letters</i></span>, (<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">4049</span>â€"<span class="lpage" tagx="lpage" title="lpage">4054</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bmcl.2012.04.081">10.1016/j.bmcl.2012.04.081</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22578462">22578462</a></span></span></li> <li tag="ref"><a name="CIT0059" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Volume 6 indexes</span> In <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span> (Eds.), <span class="source" tagx="source" title="source"><i>Encyclopedia of traditional Chinese medicines - molecular structures, pharmacological activities, natural sources and applications</i></span> (pp. <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">730</span>). <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-3-642-16744-7_1">10.1007/978-3-642-16744-7_1</a></span></span></li> </ul> </div> <div class="glossary" title="glossary"> <div class="def-list" title="def-list"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-item" title="def-item"> <div class="term" title="term">
2D</div> <div class="def" title="def"> <p>Two-dimensional</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
3CL<sup>pro</sup></div> <div class="def" title="def"> <p>3-Chymotrypsin-like protease</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
3D</div> <div class="def" title="def"> <p>Three-dimensionsal</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ADME</div> <div class="def" title="def"> <p>absorption, distribution, metabolism, and excretion</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BBB</div> <div class="def" title="def"> <p>Blood Brain Barrier</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
COVID-19</div> <div class="def" title="def"> <p>Coronavirus Disease</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GI</div> <div class="def" title="def"> <p>Gastrointestinal</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
hCoV</div> <div class="def" title="def"> <p>human coronavirus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HIV</div> <div class="def" title="def"> <p>human immunodeficiency virus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
IFN</div> <div class="def" title="def"> <p>interferon</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
InChI</div> <div class="def" title="def"> <p>Inch key</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MERS-CoV</div> <div class="def" title="def"> <p>Middle East respiratory syndrome- coronavirus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MlogP</div> <div class="def" title="def"> <p>lipophilicity</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Mpro</div> <div class="def" title="def"> <p>Main protease domain</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NSP</div> <div class="def" title="def"> <p>Non-structural proteins</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NSP15</div> <div class="def" title="def"> <p>endoribonuclease</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ORF</div> <div class="def" title="def"> <p>Open reading frame</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PDB</div> <div class="def" title="def"> <p>Protein Data Bank</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PL<sub>P</sub></div> <div class="def" title="def"> <p>papain-like protease</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RMSD</div> <div class="def" title="def"> <p>Root mean square deviation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RMSF</div> <div class="def" title="def"> <p>Root mean square fluctuation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SASA</div> <div class="def" title="def"> <p>Solvent accessible surface area</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RNA</div> <div class="def" title="def"> <p>Ribonucleic Acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SARS-CoV</div> <div class="def" title="def"> <p>Severe acute respiratory syndrome- coronavirus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SARS-CoV-2</div> <div class="def" title="def"> <p>Severe acute respiratory syndrome-related coronavirus 2</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SOPMA</div> <div class="def" title="def"> <p>Self-Optimized Prediction Method with Alignment</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TPSA</div> <div class="def" title="def"> <p>Total Polar Surface Area</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
WHO</div> <div class="def" title="def"> <p>World Health Organization</p> </div> </div> </div> </div> </div>  </body></html>